<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/7/1319" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/7/1319.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/7/1319.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/7/1319.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/7/1319.atom" />
<meta name="HW.pisa" content="ggg;4/7/1319" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Functional Conservation of the Pre-Sensor One Beta-Finger Hairpin (PS1-hp) Structures in Mini-Chromosome Maintenance Proteins of Saccharomyces cerevisiae and Archaea" />
<meta name="DC.Identifier" content="10.1534/g3.114.011668" />
<meta name="DC.Date" content="2014-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Ramey and Sclafani. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Mini-chromosome maintenance (MCM) proteins form complexes that are required for DNA replication and are highly conserved throughout evolution. The replicative helicase of eukaryotic organisms is composed of the six paralogs MCM2-7 , which form a heterohexameric ring structure. In contrast, the structure of the archaean replicative MCM helicase is a single Mcm protein that forms a homohexameric complex. Atomic structures of archaeal MCMs have identified multiple beta-finger structures in Mcm proteins whose in vivo function is unknown. In the present study, we have investigated the physiological role of the pre-sensor 1 beta - hairpin (PS1-hp) beta-fingers of Saccharomyces cerevisiae [Mcm4p][1] and [Mcm5p][2] in DNA replication initiation and elongation in vivo . The PS1-hp beta-finger mutant of [Mcm5p][2] ( [mcm5][2]-HAT K506A :: [URA3][3] ) has a growth defect at both 18° and 37°. Mutation of the [Mcm4p][1] PS1-hp beta-finger ( [mcm4][1]-HA K658A :: [TRP1][4] ) does not have a growth defect, indicating different functional contributions of the PS1-hp beta-finger structures of different MCM helicase subunits. Both [Mcm4p][1] and [Mcm5p][2] PS1-hp beta-finger mutants can coimmunoprecipitate [Mcm2p][5], indicating the formation of the hexameric MCM helicase complex. Both PS1-hp beta-finger mutants have a plasmid loss phenotype that is suppressible by origin dosage, indicating a defective replication initiation. Surprisingly, a defect in the binding of PS1-hp MCM mutants to origins of DNA replication was not found by chromatin immunoprecipitation, suggesting a novel interpretation in which the defect is in a subsequent step of DNA strand separation by the MCM helicase. The double mutant [mcm4][1]-HA K658A :: [TRP1][4] [mcm5][2]-HAT K506A :: [URA3][3] is lethal, displaying a terminal MCM mutant phenotype of large budded cells.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223
 [2]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264
 [3]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747
 [4]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414
 [5]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000119" />
<meta name="DC.Contributor" content="Christopher J. Ramey" />
<meta name="DC.Contributor" content="Robert A. Sclafani" />
<meta name="article:published_time" content="2014-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Functional Conservation of the Pre-Sensor One Beta-Finger Hairpin (PS1-hp) Structures in Mini-Chromosome Maintenance Proteins of Saccharomyces cerevisiae and Archaea" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Mini-chromosome maintenance (MCM) proteins form complexes that are required for DNA replication and are highly conserved throughout evolution. The replicative helicase of eukaryotic organisms is composed of the six paralogs &lt;i&gt;MCM2-7&lt;/i&gt;, which form a heterohexameric ring structure. In contrast, the structure of the archaean replicative MCM helicase is a single Mcm protein that forms a homohexameric complex. Atomic structures of archaeal MCMs have identified multiple beta-finger structures in Mcm proteins whose &lt;i&gt;in vivo&lt;/i&gt; function is unknown. In the present study, we have investigated the physiological role of the pre-sensor 1 beta&lt;b&gt;-&lt;/b&gt;hairpin (PS1-hp) beta-fingers of &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt;Mcm4p and Mcm5p in DNA replication initiation and elongation &lt;i&gt;in vivo&lt;/i&gt;. The PS1-hp beta-finger mutant of Mcm5p (&lt;i&gt;mcm5-HAT K506A&lt;/i&gt;::&lt;i&gt;URA3&lt;/i&gt;) has a growth defect at both 18° and 37°. Mutation of the Mcm4p PS1-hp beta-finger (&lt;i&gt;mcm4-HA K658A&lt;/i&gt;::&lt;i&gt;TRP1&lt;/i&gt;) does not have a growth defect, indicating different functional contributions of the PS1-hp beta-finger structures of different MCM helicase subunits. Both Mcm4p and Mcm5p PS1-hp beta-finger mutants can coimmunoprecipitate Mcm2p, indicating the formation of the hexameric MCM helicase complex. Both PS1-hp beta-finger mutants have a plasmid loss phenotype that is suppressible by origin dosage, indicating a defective replication initiation. Surprisingly, a defect in the binding of PS1-hp MCM mutants to origins of DNA replication was not found by chromatin immunoprecipitation, suggesting a novel interpretation in which the defect is in a subsequent step of DNA strand separation by the MCM helicase. The double mutant &lt;i&gt;mcm4-HA K658A&lt;/i&gt;::&lt;i&gt;TRP1mcm5-HAT K506A&lt;/i&gt;::&lt;i&gt;URA3&lt;/i&gt; is lethal, displaying a terminal MCM mutant phenotype of large budded cells.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/07/01" />
<meta name="citation_mjid" content="ggg;4/7/1319" />
<meta name="citation_id" content="4/7/1319" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/7/1319" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/7/1319.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/7/1319.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/7/1319.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.011668" />
<meta name="citation_pmid" content="24875627" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1319" />
<meta name="citation_lastpage" content="1326" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Christopher J. Ramey" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz Medical Campus, Aurora, Colorado 80045" />
<meta name="citation_author" content="Robert A. Sclafani" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz Medical Campus, Aurora, Colorado 80045" />
<meta name="citation_author_email" content="Robert.sclafani@ucdenver.edu" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=M. W. Adkins;citation_author=S. R. Howar;citation_author=J. K. Tyler;citation_title=Chromatin disassembly mediated by the histone chaperone Asf1 is essential for transcriptional activation of the yeast PHO5 and PHO8 genes.;citation_pages=657-666;citation_volume=14;citation_year=2004;citation_issue=5;citation_pmid=15175160;citation_doi=10.1016/j.molcel.2004.05.016" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=O. M. Aparicio;citation_author=A. M. Stout;citation_author=S. P. Bell;citation_title=Differential assembly of Cdc45p and DNA polymerases at early and late origins of DNA replication;citation_pages=9130-9135;citation_volume=96;citation_year=1999;citation_issue=16;citation_pmid=10430907;citation_doi=10.1073/pnas.96.16.9130" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harb. Perspect. Biol.;citation_author=S. D. Bell;citation_author=M. R. Botchan;citation_title=The Minichromosome Maintenance Replicative Helicase.;citation_volume=5;citation_year=2013;citation_pmid=23881943;citation_doi=10.1101/cshperspect.a012807" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Biochem.;citation_author=S. P. Bell;citation_author=A. Dutta;citation_title=DNA replication in eukaryotic cells.;citation_volume=71;citation_year=2002;citation_pmid=12045100;citation_doi=10.1146/annurev.biochem.71.110601.135425" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=M. L. Bochman;citation_author=A. Schwacha;citation_title=Differences in the single-stranded DNA binding activities of MCM2–7 and MCM467: MCM2 and MCM5 define a slow ATP-dependent step.;citation_volume=282;citation_year=2007;citation_pmid=17895243;citation_doi=10.1074/jbc.M703824200" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=M. L. Bochman;citation_author=A. Schwacha;citation_title=The Mcm Complex: Unwinding the Mechanism of a Replicative Helicase;citation_pages=652-683;citation_volume=73;citation_year=2009;citation_issue=4;citation_pmid=19946136;citation_doi=10.1128/MMBR.00019-09" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. S. Brewster;citation_author=G. Wang;citation_author=X. Yu;citation_author=W. B. Greenleaf;citation_author=J. M. Carazo;citation_title=Crystal structure of a near-full-length archaeal MCM: functional insights for an AAA+ hexameric helicase.;citation_volume=105;citation_year=2008;citation_pmid=19073923;citation_doi=10.1073/pnas.0808037105" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=S. Dalton;citation_author=B. Hopwood;citation_title=Characterization of Cdc47p-minichromosome maintenance complexes in Saccharomyces cerevisiae: identification of Cdc45p as a subunit;citation_pages=5867-5875;citation_volume=17;citation_year=1997;citation_issue=10;citation_pmid=9315644" />
<meta name="citation_reference" content="citation_journal_title=Nature structural biology;citation_journal_abbrev=Nat Struct Biol;citation_author=R. J. Fletcher;citation_author=B. E. Bishop;citation_author=R. P. Leon;citation_author=R. A. Sclafani;citation_author=C. M. Ogata;citation_title=The structure and function of MCM from archaeal M. Thermoautotrophicum.;citation_pages=160-167;citation_volume=10;citation_year=2003;citation_issue=3;citation_pmid=12548282;citation_doi=10.1038/nsb893" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=S. L. Forsburg;citation_title=Eukaryotic MCM Proteins: Beyond Replication Initiation;citation_pages=109-131;citation_volume=68;citation_year=2004;citation_issue=1;citation_pmid=15007098;citation_doi=10.1128/MMBR.68.1.109-131.2004" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. Frigola;citation_author=D. Remus;citation_author=A. Mehanna;citation_author=J. F. Diffley;citation_title=ATPase-dependent quality control of DNA replication origin licensing.;citation_pages=339-343;citation_volume=495;citation_year=2013;citation_pmid=23474987;citation_doi=10.1038/nature11920" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=Y. V. Fu;citation_author=H. Yardimci;citation_author=D. T. Long;citation_author=T. V. Ho;citation_author=A. Guainazzi;citation_title=Selective bypass of a lagging strand roadblock by the eukaryotic replicative DNA helicase.;citation_pages=931-941;citation_volume=146;citation_year=2011;citation_issue=6;citation_pmid=21925316;citation_doi=10.1016/j.cell.2011.07.045" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. Gai;citation_author=R. Zhao;citation_author=D. Li;citation_author=C. V. Finkielstein;citation_author=X. S. Chen;citation_title=Mechanisms of conformational change for a replicative hexameric helicase of SV40 large tumor antigen.;citation_pages=47-60;citation_volume=119;citation_year=2004;citation_issue=1;citation_pmid=15454080;citation_doi=10.1016/j.cell.2004.09.017" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=A. Gambus;citation_author=G. A. Khoudoli;citation_author=R. C. Jones;citation_author=J. J. Blow;citation_title=MCM2–7 form double hexamers at licensed origins in Xenopus egg extract.;citation_volume=286;citation_year=2011;citation_pmid=21282109;citation_doi=10.1074/jbc.M110.199521" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=M. L. Hoang;citation_author=R. P. Leon;citation_author=L. Pessoa-Brandao;citation_author=S. Hunt;citation_author=M. K. Raghuraman;citation_title=Structural changes in Mcm5 protein bypass Cdc7-Dbf4 function and reduce replication origin efficiency in Saccharomyces cerevisiae.;citation_volume=27;citation_year=2007;citation_pmid=17724082;citation_doi=10.1128/MCB.00997-07" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=E. Hogan;citation_author=D. Koshland;citation_title=Addition of extra origins of replication to a minichromosome suppresses its mitotic loss in cdc6 and cdc14 mutants of Saccharomyces cerevisiae.;citation_pages=3098-3102;citation_volume=89;citation_year=1992;citation_issue=7;citation_pmid=1557417;citation_doi=10.1073/pnas.89.7.3098" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harb. Perspect. Biol.;citation_title=Replication proteins and human disease.;citation_author=A. P. Jackson;citation_author=R. A. Laskey;citation_author=N. Coleman;citation_volume=6;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=K. Labib;citation_title=How do Cdc7 and cyclin-dependent kinases trigger the initiation of chromosome replication in eukaryotic cells?;citation_pages=1208-1219;citation_volume=24;citation_year=2010;citation_issue=12;citation_pmid=20551170;citation_doi=10.1101/gad.1933010" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=S. K. Lam;citation_author=X. Ma;citation_author=T. L. Sing;citation_author=B. H. Shilton;citation_author=C. J. Brandl;citation_title=The PS1 hairpin of Mcm3 is essential for viability and for DNA unwinding in vitro.;citation_pages=e82177;citation_volume=8;citation_year=2013;citation_pmid=24349215;citation_doi=10.1371/journal.pone.0082177" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. P. Leon;citation_author=M. Tecklenburg;citation_author=R. A. Sclafani;citation_title=Functional conservation of beta-hairpin DNA binding domains in the Mcm protein of Methanobacterium thermoautotrophicum and the Mcm5 protein of Saccharomyces cerevisiae.;citation_volume=179;citation_year=2008;citation_pmid=18660534;citation_doi=10.1534/genetics.108.088690" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harb. Perspect. Biol.;citation_author=A. C. Leonard;citation_author=M. Mechali;citation_title=DNA replication origins.;citation_volume=5;citation_year=2013;citation_pmid=23838439;citation_doi=10.1101/cshperspect.a010116" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=S. Loo;citation_author=C. A. Fox;citation_author=J. Rine;citation_author=R. Kobayashi;citation_author=B. Stillman;citation_title=The origin recognition complex in silencing, cell cycle progression, and DNA replication.;citation_pages=741-756;citation_volume=6;citation_year=1995;citation_issue=6;citation_pmid=7579692;citation_doi=10.1091/mbc.6.6.741" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. J. McCune;citation_author=L. S. Danielson;citation_author=G. M. Alvino;citation_author=D. Collingwood;citation_author=J. J. Delrow;citation_title=The temporal program of chromosome replication: genomewide replication in clb5Δ Saccharomyces cerevisiae.;citation_volume=180;citation_year=2008;citation_pmid=18832352;citation_doi=10.1534/genetics.108.094359" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=A. T. McGeoch;citation_author=M. A. Trakselis;citation_author=R. A. Laskey;citation_author=S. D. Bell;citation_title=Organization of the archaeal MCM complex on DNA and implications for the helicase mechanism.;citation_pages=756-762;citation_volume=12;citation_year=2005;citation_issue=9;citation_pmid=16116441;citation_doi=10.1038/nsmb974" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. K. Raghuraman;citation_author=E. A. Winzeler;citation_author=D. Collingwood;citation_author=S. Hunt;citation_author=L. Wodicka;citation_title=Replication Dynamics of the Yeast Genome;citation_pages=115-121;citation_volume=294;citation_year=2001;citation_issue=5540;citation_pmid=11588253;citation_doi=10.1126/science.294.5540.115" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=J. C. Randell;citation_author=J. L. Bowers;citation_author=H. K. Rodriguez;citation_author=S. P. Bell;citation_title=Sequential ATP hydrolysis by Cdc6 and ORC directs loading of the Mcm2-7 helicase.;citation_pages=29-39;citation_volume=21;citation_year=2006;citation_issue=1;citation_pmid=16387651;citation_doi=10.1016/j.molcel.2005.11.023" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=J. C. Randell;citation_author=A. Fan;citation_author=C. Chan;citation_author=L. I. Francis;citation_author=R. C. Heller;citation_title=Mec1 is one of multiple kinases that prime the Mcm2-7 helicase for phosphorylation by Cdc7.;citation_pages=353-363;citation_volume=40;citation_year=2010;citation_issue=3;citation_pmid=21070963;citation_doi=10.1016/j.molcel.2010.10.017" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. Remus;citation_author=F. Beuron;citation_author=G. Tolun;citation_author=J. D. Griffith;citation_author=E. P. Morris;citation_title=Concerted loading of Mcm2-7 double hexamers around DNA during DNA replication origin licensing.;citation_pages=719-730;citation_volume=139;citation_year=2009;citation_issue=4;citation_pmid=19896182;citation_doi=10.1016/j.cell.2009.10.015" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=A. Schwacha;citation_author=S. P. Bell;citation_title=Interactions between two catalytically distinct MCM subgroups are essential for coordinated ATP hydrolysis and DNA replication.;citation_pages=1093-1104;citation_volume=8;citation_year=2001;citation_issue=5;citation_pmid=11741544;citation_doi=10.1016/S1097-2765(01)00389-6" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=R. A. Sclafani;citation_author=T. M. Holzen;citation_title=Cell cycle regulation of DNA replication.;citation_pages=237-280;citation_volume=41;citation_year=2007;citation_pmid=17630848;citation_doi=10.1146/annurev.genet.41.110306.130308" />
<meta name="citation_reference" content="citation_journal_title=Sub-cellular biochemistry;citation_journal_abbrev=Subcell Biochem;citation_author=I. M. Slaymaker;citation_author=X. S. Chen;citation_title=MCM Structure and Mechanics: What We Have Learned from Archaeal MCM.;citation_pages=89-111;citation_volume=62;citation_year=2012;citation_pmid=22918582;citation_doi=10.1007/978-94-007-4572-8_6" />
<meta name="citation_reference" content="citation_journal_title=Nature Cell Biology;citation_journal_abbrev=Nature Cell Biology;citation_author=S. Tanaka;citation_author=J. F. Diffley;citation_title=Interdependent nuclear accumulation of budding yeast Cdt1 and Mcm2-7 during G1 phase.;citation_pages=198-207;citation_volume=4;citation_year=2002;citation_issue=3;citation_pmid=11836525;citation_doi=10.1038/ncb757" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Tanaka;citation_author=T. Umemori;citation_author=K. Hirai;citation_author=S. Muramatsu;citation_author=Y. Kamimura;citation_title=CDK-dependent phosphorylation of Sld2 and Sld3 initiates DNA replication in budding yeast.;citation_pages=328-332;citation_volume=445;citation_year=2007;citation_issue=7125;citation_pmid=17167415;citation_doi=10.1038/nature05465" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biochemistry;citation_journal_abbrev=Annu Rev Biochem;citation_author=B. K. Tye;citation_title=MCM PROTEINS IN DNA REPLICATION;citation_pages=649-686;citation_volume=68;citation_year=1999;citation_issue=1;citation_pmid=10872463;citation_doi=10.1146/annurev.biochem.68.1.649" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=H. Yan;citation_author=S. Gibson;citation_author=B. K. Tye;citation_title=Mcm2 and Mcm3, two proteins important for ARS activity, are related in structure and function.;citation_pages=944-957;citation_volume=5;citation_year=1991;citation_issue=6;citation_pmid=2044961;citation_doi=10.1101/gad.5.6.944" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. Zegerman;citation_author=J. F. Diffley;citation_title=Phosphorylation of Sld2 and Sld3 by cyclin-dependent kinases promotes DNA replication in budding yeast.;citation_pages=281-285;citation_volume=445;citation_year=2007;citation_issue=7125;citation_pmid=17167417;citation_doi=10.1038/nature05432" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Functional Conservation of the Pre-Sensor One Beta-Finger Hairpin (PS1-hp) Structures in Mini-Chromosome Maintenance Proteins of Saccharomyces cerevisiae and Archaea" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/7.cover-source.jpg" />
<meta name="twitter:description" content="Mini-chromosome maintenance (MCM) proteins form complexes that are required for DNA replication and are highly conserved throughout evolution. The replicative helicase of eukaryotic organisms is composed of the six paralogs MCM2-7 , which form a heterohexameric ring structure. In contrast, the structure of the archaean replicative MCM helicase is a single Mcm protein that forms a homohexameric complex. Atomic structures of archaeal MCMs have identified multiple beta-finger structures in Mcm proteins whose in vivo function is unknown. In the present study, we have investigated the physiological role of the pre-sensor 1 beta - hairpin (PS1-hp) beta-fingers of Saccharomyces cerevisiae [Mcm4p][1] and [Mcm5p][2] in DNA replication initiation and elongation in vivo . The PS1-hp beta-finger mutant of [Mcm5p][2] ( [mcm5][2]-HAT K506A :: [URA3][3] ) has a growth defect at both 18° and 37°. Mutation of the [Mcm4p][1] PS1-hp beta-finger ( [mcm4][1]-HA K658A :: [TRP1][4] ) does not have a growth defect, indicating different functional contributions of the PS1-hp beta-finger structures of different MCM helicase subunits. Both [Mcm4p][1] and [Mcm5p][2] PS1-hp beta-finger mutants can coimmunoprecipitate [Mcm2p][5], indicating the formation of the hexameric MCM helicase complex. Both PS1-hp beta-finger mutants have a plasmid loss phenotype that is suppressible by origin dosage, indicating a defective replication initiation. Surprisingly, a defect in the binding of PS1-hp MCM mutants to origins of DNA replication was not found by chromatin immunoprecipitation, suggesting a novel interpretation in which the defect is in a subsequent step of DNA strand separation by the MCM helicase. The double mutant [mcm4][1]-HA K658A :: [TRP1][4] [mcm5][2]-HAT K506A :: [URA3][3] is lethal, displaying a terminal MCM mutant phenotype of large budded cells.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223
 [2]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264
 [3]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747
 [4]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414
 [5]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000119" />
<meta name="og-title" property="og:title" content="Functional Conservation of the Pre-Sensor One Beta-Finger Hairpin (PS1-hp) Structures in Mini-Chromosome Maintenance Proteins of Saccharomyces cerevisiae and Archaea" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/7/1319" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Mini-chromosome maintenance (MCM) proteins form complexes that are required for DNA replication and are highly conserved throughout evolution. The replicative helicase of eukaryotic organisms is composed of the six paralogs MCM2-7 , which form a heterohexameric ring structure. In contrast, the structure of the archaean replicative MCM helicase is a single Mcm protein that forms a homohexameric complex. Atomic structures of archaeal MCMs have identified multiple beta-finger structures in Mcm proteins whose in vivo function is unknown. In the present study, we have investigated the physiological role of the pre-sensor 1 beta - hairpin (PS1-hp) beta-fingers of Saccharomyces cerevisiae [Mcm4p][1] and [Mcm5p][2] in DNA replication initiation and elongation in vivo . The PS1-hp beta-finger mutant of [Mcm5p][2] ( [mcm5][2]-HAT K506A :: [URA3][3] ) has a growth defect at both 18° and 37°. Mutation of the [Mcm4p][1] PS1-hp beta-finger ( [mcm4][1]-HA K658A :: [TRP1][4] ) does not have a growth defect, indicating different functional contributions of the PS1-hp beta-finger structures of different MCM helicase subunits. Both [Mcm4p][1] and [Mcm5p][2] PS1-hp beta-finger mutants can coimmunoprecipitate [Mcm2p][5], indicating the formation of the hexameric MCM helicase complex. Both PS1-hp beta-finger mutants have a plasmid loss phenotype that is suppressible by origin dosage, indicating a defective replication initiation. Surprisingly, a defect in the binding of PS1-hp MCM mutants to origins of DNA replication was not found by chromatin immunoprecipitation, suggesting a novel interpretation in which the defect is in a subsequent step of DNA strand separation by the MCM helicase. The double mutant [mcm4][1]-HA K658A :: [TRP1][4] [mcm5][2]-HAT K506A :: [URA3][3] is lethal, displaying a terminal MCM mutant phenotype of large budded cells.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223
 [2]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264
 [3]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747
 [4]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414
 [5]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000119" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/7.cover-source.jpg" />
    <title>Functional Conservation of the Pre-Sensor One Beta-Finger Hairpin (PS1-hp) Structures in Mini-Chromosome Maintenance Proteins of Saccharomyces cerevisiae and Archaea | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"KpLJRgsWS37n8kDoGyS5QbmIlzzphgu696b6cqpoEHs","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459915","apath":"\/ggg\/4\/7\/1319.atom","pisa":"ggg;4\/7\/1319","ac":{"\/ggg\/4\/7\/1319.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/7\/1319.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/7\/1319"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/7\/1319"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/7\/1319":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bXjHcBA200APdDAaYUSu5HVGcTfkAQwtCA8mtdEJnB4BXu71+OVCxBHQydpOLoQBO\\\\\\\/z\\\\\\\/rBl4Y7CKmP2CdvjGnhlCOOh6aSav0\\\\\\\/su6HxbkkXVKqb\\\\\\\/0yotf4YYjpiYfXWvRpUEJJFd2VHUyxx9tnXVJ3d+Sk\\\\\\\/qJYua+A9ec3pkoMgT0D29FMpMqtTlMXQzcf+eB9OU0ALUVg9LkEfExJoNip9ZNRfK7fr4GSsQtJrmYj7whI8S2IRzWqhLtwr1ZGFUQlUTt9RBE\\\\\\\/KnT5aQT1kySqxDb9fK\\\\\\\/CC7dS7l7evHkTEqejxHPA6hVs3GDpT83nesyrCGTZlvQ\\\\\\\/32TswJF9pdfqmirz+tQ\\\\\\\/WMrKi81FBP5UZyftrLZuE76MlDCOjAQxLlviqmmAMu\\\\\\\/UZOh67Nk+c3wA9sWk6JvUln6Jh\\\\\\\/AwCP0QtJYHYaPGZZmnQyy7wFBqAQ3rprXo5kSHZJDGvdWinMFqPsQvFtE+YAqojMzzSOMneguIwJ7aGvY0NK+yvg7EKgBIKi5wg93139UqzouFVHX30lJGXXhOfZRfZMZlvge\\\\\\\/UO3YR495c+YuDZwdZ7gczhY67mACG3CV3DyjcYasdQwkt2qVjrZ9tBL0q9oMMXmRoYGkxbUsye6K\\\\\\\/YuNDG5PFuKe77\\\\\\\/\\\\\\\/BX4sqyq3+3aLAHeQUgIbCMdndwjCFBtLOUdQcTteVs7qZzcrMu0q5jpLNpW3\\\\\\\/nuq2PPWsT1mRNlecnGsmLOF3zdytEA6QM6Jr\\\\\\\/nlOsu2u51zr\\\\\\\/AGC3w5d9FBiYmn0D0JOyQHz4F572Twt3186ie5sWet5mmftcSno3kkwxU5XqyySyuUKlkdur26Xz+kWxGlZmHJjx2aDRFAXNOcdIZdWZ4bEeug+tTLvGWqL4kKGCl\\\\\\\/75qjw7Q4Lcq0iNo2IDCcxBnXE7O9zpmKh6CALWWhhIp\\\\\\\/EUBgj13MIoO\\\\\\\/mHuKcWoYuGJ0yQyN97owcmuOImh\\\\\\\/6XirfE2Ow7DnQUyqEfGbk0lDiMwdcYVWmIs8b9kPbumoxiDSH7pqYBxiiZlrUyiInemtJJ5Y3W7IQCvZ9RujEaFze7i8KCYUvirvqJEtMIUzfpgfV4OSZaA4ntaPBBO7zb0TaQ6GUwSRr2B0ZtgsGT9nS62Mqd1Bgwkv1W6xVyPVyy7q0xqKaaLdcACCOxBfoVMv3B9\\\\\\\/kcnQPOiaJTjmTb6tAWCHaYEIIn\\\\\\\/EU7an8YzRupHK1rjzsxUYwIdTLoC9mCn\\\\\\\/tOzRp8S2spL42PTgtIcCfhkc7BdRbSLQ4WffIIZ3TV9ySBo8loPXkB\\\u0022,\\\u0022iv\\\u0022:\\\u0022KD5pkqspkH1kepkNcxkAxw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ca7152bc5f3d238f7e7ba9d1cb5490d36c6c477e50d9ffc799b1c0bcc573aaeb\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222hKxmQh9ULvJKFCMn+Jz+hhjGf\\\\\\\/csgBzQh\\\\\\\/vQGS1z0AJ+Tx2CwbRKCtLUKVOZ8Abqyb010ZCxPA8ttuoAHP1emacy+xns\\\\\\\/jmfp0yJe8uv62wNfnv+1x\\\\\\\/cbNwO3cK0hbfkmQ5SNtxUQoLXd58Ha76ts27rLpLxgGohST+1sYX9O2NZmoQk8W5hX+9zq\\\\\\\/UKxKk64vBfTIf67pDp1CBIuYVwJXG6\\\\\\\/6+vtrCMICt22MDi7nd58hPYZ4xfkLOmygp0sIc9Iq3SoWSs5DumEdZjYFs0JUAUq8APzF5PZZ6c43Jy4tQ9nj5o41KL3AXdbqgOYd+Bcu+u0xopD9sMnm1Rr4xFAaTouoKjEqrl4fJI2ALP0ozqtEs58d3eALJDkNdi2e2rvvlbx7MBap6AZtwPXYFHSjALXxN7h4CuQK+Y9JIAISKd04OA1Icf0+dqqrxiyTMboi4wMl1seTuZesnSdzZ9bPrdMVRGaxXSf3pA\\\\\\\/uWSVmjFUdLGIX4vHx7ue+CTsJVR6vBR5QJWHMDTqLrJ1Bj\\\\\\\/En6iUR1bYEQFrosPisH50gHmL\\\\\\\/26Qy\\\\\\\/Ye3IGuqD4Ra17pR52wZMRsIQiXNlKDD+stb5jGmmyYMXs87jxPBwaC34Onivsz99KcJ7PZOzxgJoy+qdGjxwEXjrO8Xwg3L3GDiFGEdYAF0mZ5rpi2qjA8x4t6NbxLC9PVFNFWQXEDIezaSR7sbn84uVulrOzT2YmCiGUhlOrDT4zfD\\\\\\\/FHqdsuHv7PpvRiRuq4ZwQ5DFbk2iSFgIxgvlfHDVJ4oz0RE+WwE1H40MyloJ15sVOZkUlAQa\\\\\\\/6AG6sK0AFpwtPVSfcIMuWu8pvSj35p9XamW2ATybJY\\\\\\\/mpAWP5ftg3Lb5yxiK3AuLvkFODgfSJZC3bFYhINp3QaIn50j4OBWHWVUus48JBezSaQ2lLuBPdgvcN60xWX5rNtyF8S4a6e10VLeSo4vAJUq0EkO3sdkv2JPbM4jQ0B8gOSqbLEiJFW2laBuWuHLJfEe3IHeXj5G23maRgRJcY1m9UUAO+JOEdAE+hQUJZhVKxk9n1D\\\\\\\/ZglUL+DgrOJ0YTO3n1iKGeFWrYDfkyAvct9sJkwZJ\\\\\\\/vLPysKs+royZGQ4xDsx40bs3vT9QVF5z5UsO7voLgIj9bec7kSFOmWrByDuKv2biMTTUF2hKV9GQCITbIQXRTfJzpQxY2bA4bXGhfg8nPfyg4\\\\\\\/M66r1dsucu0daaFrWNzaMpkQvEIrs20wRn\\\\\\\/w+\\\\\\\/\\\\\\\/QapcSwCxLWwOiwIYytsD3R1SYd+q7LIqb3pNmDZURbvMc3XQiriVdK+WZKHy7ffcYZKe1Lrs\\\\\\\/L\\\\\\\/PGyeLSylpEKkrrQQQ4l5LE+fDybzQGDJlZIhhOAL0LjBqa8RwqGyDFIIyTsaEfbPjfEkPRL2reDMGTZ2OTsFwTYgeTeawBpSR6aEVRzp3MI7cEeI37XB93MnVge15KLelAOQtUMfYThdYZzpxZ9dgSprne+FhImTb1pArx0FCt1njBs8El3GORBCsoHZctnU2ZswAPCI3RT7a44P+CWcC6pvjS1ZnHLaBTEunfk5RoqbJnLDipgAla986KVh0WUI4WZZItdP08F4\\\\\\\/B1drkA0tTWaCvMyICIGtomxswujLyFPV3\\\\\\\/yQl8EAOxYM\\\\\\\/Ll2z7AmJA00Lf85BD+X+hrePF75Lxt46YNMk1zP8j9Munw\\\\\\\/Um2RLlfT9Mwpp1947xjvhRhT8TxAmQvXDS19Vu+ZZVlMGyKd2v8mFUEJq6nOEJ4tpTk6Tc06wHFZrX09HhAPcs+VNXp2o9K0M8rqnBeqaNHjNgWxggG+sPSpRAKfvB\\\\\\\/At+Q+6IA2sW212ASrfunnLVFJdlAiRIz9IJMEbHvLznNXDNO0gZMtBWGA9o1UCwqVWhnCqyAYzvhgo31GhUbrUdO5VsDjQlYHOt4ZfMaypkmVL3fzm+KA5RtkaQR\\\\\\\/IQo1uZ3iXQzxrkPsHY0MZwLBq92M2IbFP2yK3JnGsqeGT69QudxFFb8QCClGksMiMqwrHenTmSifEAzuGznRD0Kp3wM9tUx9y93VL+pnSYRtQJ4W0SX2gxj0IvTW04T9kUFWvUUBD6U2xEml33KBmH7Y9eE+doKBbRBnK9QaXTJKUiqY5OU59eTtKHBEbPhECBUgL1aYUl6+breNeqSk1zC4QYg0NFmtmDRG2MkF0CcM6F7IDYNCNZ6Icpq0ldmsoJFuBdJXQ6HCbkJZsfaD5aD2zdbSjDtybdmZkUzrfmGRyxOTnh7QniX9sdvatXOJWCdtBjevvU3k8JXHR5SEAhr902ujcNAd3jwdx4rxH5dlHqe2qqxEgqD+CVrO8kEQuvYHIM\\\\\\\/WfocwP9NLthGkDL27GAIaIHi5s\\\\\\\/MMa+1Uo7h7qcrBzgPK\\\\\\\/qmXwhx0qkxwkppOzB65hkciJYTKQVEi+liwik\\\\\\\/cQTuwzy9lI60qYudxPD9hCUJ\\\\\\\/X+6dm96YKSWd8EXNjFk+7B48SISz9laxGVCpoUtr1tEG+UYuLJEseJW\\\\\\\/\\\\\\\/NDaotv1QmACDfx3UYq1V5bDbSHd1\\\\\\\/joeavXEYSK9fqJwXxUUDESuapCEmHQDlLfCzQXW9tN2DH\\\\\\\/rA5hRqVKOTAn0jLRoNQSwhlwKKhGktd05\\\\\\\/9N2PuwpZ7xD9HoQZh1QJlf4BNv4n+VaBBaiplwv6hr7gQ+OsHYI\\\\\\\/Vl9B+EgOXWc6gS\\\\\\\/3Emv5ivD3JzeETrMy1O\\\\\\\/FAB3rl9FTDvR1L4XS+5osMRUWfZJ8L1f+e\\\\\\\/rc0q8\\\\\\\/pMpcB15tpDU4MBtlATg2cycwfPE0yE+HLkJoRCyaWmAEsg\\\\\\\/YEdHT0s7rzjXqemN\\\\\\\/\\\\\\\/3kdU49OshKRkPZMDIXa7+oZ1GW1qzRZeapFZWePn\\\\\\\/j0tDytEV+fxVCjxG8oo4drLqAZmkTmHBsDAJBYMsDdexprmiIp8IEzZEs6tXk9IO7WXF96WP5G8LWJkq3WCdv2LXZT9tFQg1ojX5d0TOldyf9KpnKSeXinTMxQJnt84p7tAPMfZjtZnmElYY5DUKjH6kOCUefNYaFePZ+UUmlWVSM37dckhq0uCTsHxZEOjHFKyFHSHK4MJvvcJ\\\\\\\/fAUoBgBlCAgIvh0iNzbhQ7g5lyNcZnTpCXvgj2Y7bzFeUmILcqkiVXebt+NdPllFUmyoybvAM9tKsWSF72O\\\\\\\/V+7KXKndApszPlXRmxSqU64+tEpENCqCrBkUPhFRYPs9pVTpvL\\\\\\\/XBkLT79J1E2hhSDj\\\\\\\/qTvZYiDwRSLw8QQ5eUuMSmGo5o7IY7oysaxVwzG3VQxPASAx59jvLggx17kFLYX\\\\\\\/cwGjpsAnbvHPvsUOq5l3weiQGgbHYT4050M6MBZ\\\\\\\/OCU6JxldtCeoG4OSZtn6LWPi5poiC0WVO504rs5p4w\\\\\\\/GhzH93kLEz6KjVl6mjqpMtv4PNBTwwtO7hDXAgPd3Yon17q7MMBd+XqNoV\\\\\\\/SHS+OpFYf+PwYA\\\\\\\/K4+971IbCWcRO0xdh0axwRxp5hLiKX02y4DmIaATscSMTrCiQ95ReyBrNkcY1XUIWP0jMO6RBJL1Vb4OVVnydFG3ELTp99pVd2EZz45stCHbVnC1yWJRQ6PDI+6knzix+Dn+V+VpK+3OSTBbJPQpS8tZZFWvSAscddb+MbHXt6fPpKnrPPellRMzg0xRUxczE7ClkOQ1m6nDztwLRrFDeG2tXgOoX21Q7BSJzSvCiP2m7N8\\\\\\\/xqREIaD6WCBptFNPYJZCqxNP65k6t8S1YMGsfeudVAjw+RPfcVT708eYw2aJ1hWYWMbph8f\\\\\\\/8\\\\\\\/kpGt1bJKcWXiEmdRMgjvussb\\\\\\\/yYMD9p493GAMp8VtXvrU0rxNHp+deO5ZKzS5audMhcjy4Ny60w15ujXrCWOVbZpGw1hMid579XODZ52\\\\\\\/yOwUTO6xXYTKz9OfYF5456MdiifaRcNbaNMtuWTT71LodrO04cLWXHh9i0Pfag31dYAdtmL0I5jxi8IMUJbyhB\\\\\\\/MHNFzYu\\\\\\\/pvLBICsU8A3RG7DlnlOLY68zdW\\\\\\\/OWFejaKkDAkFwDsGfA3MncdxmwsXfnwApEDbhXR4WCyaK\\\\\\\/NCQ\\\\\\\/XeiqroL4Bd1dl0pT79cp96jFj3\\\\\\\/giZzLWPZ+zmsARZMMAl7qqfeNFKJtRfUMxFCfvw6S+KZShuuBjKXreyCsXa94UzI8fqD5OfBxJST9TTxQwZWDZvvBIj9L1QddTfBSloqBcdkf72gjAVng6hEFG5zykSs\\\\\\\/GBCR7\\\\\\\/grZm0BX3qsiguPAqvzNyAUdH43qihFGY90gaToq2hyPANnhV1MtdgSV7kj41Z3Qmh8JK1RMptSVikeVg663\\\\\\\/CNfPc8ruY2iQCn\\\\\\\/Ban7oyRPK+1vkVD2UQleDm5F7R2nSr78VYbM9Wz1G3Ju+Q1hZ8pvU2qg+oWilM58Nz3CXH5CKO+SpD8TLzbqgjleZDJtAowk6JrYzJ3vxgyvsiMC+SSpxOeDGnGYhsDeS62JThE8TJNyZSZZ2HozLzVH9POBXGQ0Yg2T2E3I\\\\\\\/9nH9ZmSlzfcFwQbYbX9pK\\\\\\\/vj44prRKoYZJ6v0NUz9wE4W8ty7Bx9cuIt29qRgagZQW1SkdTWxpj9bw\\\\\\\/KFTmmiQSuvWUgq2zbaocgEYilp\\\\\\\/BuhF7p9\\\\\\\/P7SyPvJ60jkqAPd\\\\\\\/WP7fqt7gfbosjLAosfIVuOGjuolLIADyAKF4lX09DWqqqZvETh8PJmLtD+8XN0EV7gXdByPY4puYf6gDjKzsYPhIfhN3Aqf0gH9i8PBNQI\\\\\\\/UBlYXCLTZrrH5SIG05JK8CeGZZ3KUAkAH0\\\\\\\/LwlW2A1mrxybVN7oyX5k+kEldk9DrSCqhhR37xX5iCrse\\\\\\\/37A+7kbRgrRM+39KgHdeeZ70mjcB8soUiNFqgj0CjvIHNPELTPhqBB+njItTFacuIrBoOvhd\\\\\\\/42tlJADkyYpsgIG298OSvTIYcousbilS+nl9GejEDKtfsmyy8QH80qAjZEYjDPcJZMABg5ptSs9MdAGOqq8ES9HjfuLIitrRslKysl5O4st9vngAlsbRH3TYWEJ4214QgRm9Mex3O1YlEK3O6NRUHhV2ONXqYA\\\\\\\/cB9P5YAafr+R9I33WI4rWj4nUvJp+wNI69QEPZlg6ZYEXtjbwIgTHaOgRGRNJqpXYlPmNnnDRcmIlIEqLMqaD8eWshdwQ0rljdtxTa0\\\\\\\/rgxC5qBirAs4fH2MgtaoV\\\\\\\/eJ06BZ0Wjf5WAUWZNI45ZJIrD9tLWLYZn2S1e44s6Ro3o16le3F8OF2HI52eHH8R1cKConY1JeYHBk2ROh17Yutca8uzkKVHk2mt5aLXt9VoEQ4zbIcAhvmKdcdf8magopDXRIp1GcGVlry6kO2nB80keCvtJl2RSnoiWZnQxpezLQE8KtJn7vPLFHcFYUzR6bizc3j\\\\\\\/5Z0OTWhFWzsJvCr4NLVV9DmFq1O6kcSAd6Zf1hxByWoAMt5RD0ewdwEXPOeJ8PQ4xOSM441fE\\\\\\\/BSUmvLhJ6eRSIBaz8Y8UBML0nxhe2YlrIj8RpfRt6\\\\\\\/aq6PsOLgDeLHDC9yOQMM2lnrTKhcEU7Zg9UY54L3E50VXTLFVaHK5OHEOIRqsw+DbCPfrwbpvJY6V2r2PNZ+o1nJK2\\\\\\\/1Lns0cRe9aFWfYgxdHG3Jq4MHxWXgH0SuOFMR8HsXKx5CPOzcs3g2cvilFqC1OUaMR\\\\\\\/IChm9jqRBfylkT6AHyk2kLPrl6UZg\\\\\\\/qXL6UBd+BYaKXNnlfpmAwp\\\\\\\/We0GcoSE8Iw9KS0v6zmWH5s7K\\\\\\\/EAtecGF0zphvC4LzmpeMBbnqmgcAS6EvfH72yCprqcGj79j2RS6MGeKrTLWYbtbaBL70rYojEYYiywLkAiKqLiGUymLzUfktk7posMMaZNL7G3Iw+MmirUFv8Tw5+fWA3v+NJMyrcXsaRub4+yy948GneXcxKXxGPyaLNZnBBf4qS+MHmihbJxbxtpYngjrjsEXK6tvAfIGvjq+V0HVbl\\\\\\\/wlBCVRQfFJk6+g4E+GcsPSOI70AaJCqh\\\\\\\/84bFOyl4laNYzzxybWn63QsjmJVFLWUEwW4V7lRJMIDmAsmD+2Q8E3nKxCx0mQgvAfkHkqmcNGwDCOKD6LMA1KH4pmIscfW28gOjb1S5T+zOEWAAQ8YCaMzt7nJw+pURyAFzOSXdPkhxDNv5K5Z1F6NFek23kbJWxMeLCgGytiZejW3UM3V0OM+p3bESRRgiYEewIeqftKjGMYs2bDKtXj05VX7r9A8\\\\\\\/CltujvhK4HzfYM8Qk5Uifo5pLdLyAOhO1H7cwebgMJThxmFgzPOPlQD8VrwAh6+KaMXYOmWT+Haax5dUxbEiz9GjRZq\\\\\\\/d9diHQtEJi\\\\\\\/D62GNeq7yEwsD5ihqN3H+Rz7xHhrxor8oJjWiLhmbAG4BqdnKoKW8JrFJATUqb\\\\\\\/UU4Vqg2k9V9ey0u+zxkz1dcsQPNfjmR\\\\\\\/Xj+t5NjujAs\\\\\\\/p+5x+2nrJpIoO2uaEr7ZO\\\\\\\/WzPzFjocPyxyjPPr1qLO2Lc3vuZxssy+\\\\\\\/X9eMoraIOrgH4wCtF4qc1TJLZmFMOb46Tnrvh3BfAJls15PdJvbc8gDd\\\\\\\/P664TSaPntFdtdOywu+eNo4W53lvECoYJe\\\\\\\/BtAAp\\\\\\\/03O8dsseTAjILX6\\\\\\\/r6FTLtqBmFbo8qUfYUMZbPu+AK5k1EJfH5Ip5bIgl8LXi+Ms1QhqmWy9WknUeNE2\\\\\\\/jqRPVrpbpZYshSALUnOSQayFxC7dk2yBpGVFkFPWfiMUkZh\\\\\\\/mw2b03tibxIX+9EqzFQUvpFbKzC2My7rPK5mDDHfd6ZpY1DngOTgtGkPNto\\\\\\\/pooFNxdwxry30m+HdeNl7tQjMxA3CIHwCN\\\\\\\/5WjjntKOWiJcFpBhvD2m47oxLSiqFeUJyEe9PHW72S4l9rQiDSKCtlvQShei5rETWLECV44kurvA9XSpsUsBTwU0bvb7h+B4hEHlhbxZ+Jnu1i2eBc6q\\\\\\\/1xGYZI+7DNjDAg7QjU2vCcWZgciufUButVp2xy99duUDKPpAIlwXB5lko5TIqe87S1StJurA4fpqTXG6L23LxxAVu3O8GEDZhHTHXHLz5G86or+pJw5V7kRMlolxnujGSHaIrSKPtXnnwRXl47W3NYqOpyxNjMKoDhql\\\\\\\/6uwemON3YntAJydqCplMDtv4atC3XDwjxihinvzEk03FxAJ\\\\\\\/OzKhFHnk3++FeJDPTH67wVS0nOigBhkLcD2VVbjS8sa9CPMdZqWXP6v9AI\\\\\\\/cJKFwCR0AAubSjym\\\\\\\/kHefFOoInuUYzvoysR1uirbUjnwjUiZVMkDKXstH3s4H86al\\\\\\\/cvIWMwOpQMNJHFwvHvo0GxYDh0C\\\\\\\/SXwmLI8mccaooVob4RAIobMPcgqEUnA5O17PqP0dVdllwkTClpzkMaz70OgkUd4DeMuN9DiEalUwO\\\\\\\/GoiRPtk8qjb26lCwcA2dVx0MqG1NiDLG+O31z8fmmCAz3X6mRwPmRBfxhq7jfOdRznL6h9\\\\\\\/mOtJ+LVAGnPioHCaIRXR1GBTgavk4fmpNyxznG4g9Xd\\\\\\\/AnkNp7R1PjDlXXqD8NkICi+xYTFfPlSKjzcRCLpjp2FtNsVfnQxg3+IBR7PGSOmLqfFlamwfdyQ0XWDHK8RFN3ENI5MelFrvYuG2A7qJdU48M2qjo6VmIxk0LDj59o3tUUp2TS7ew4RgOOtsNITmWoYzwTWMKBvob5OGbxH+M8TYvTs1TjzKXkGfUt\\\\\\\/JLlcDComx4sFRn+JUqDO36gBUekp+HgVEtbbx90xYE9RG7TYsfk1O1Z48bmPx9PRT7YfihdXOWHFsjhfh5ETeGDpQaNmUYmO08MXgT92jlnuMlrsAo0sZAShCqeEbkDWqIHun+ase9Rxkas2C9Fzl0DK1uxnmgqhLd4Nu64v1v3zY+XqXvQ+v4llu8vFc8ykduL9agjxDwH+yICNxFWXHthjVztx7zdn1naWmp25tMxhrUhziUeeAVLaY33uEQlZMg8RYmhoWFaug43akOMFVaL62dYlwpMVxi9\\\\\\\/FEieSaOVOSGU+2Xjd\\\\\\\/JJYV5jg2bU0BXBVtxxniWv\\\\\\\/719yjnAlYLcWb6GfghhuAj7TaHcUg20O36MRDG6++qW+1cejFR6EEtsOA3Js0LZ2dI3uUfMENf8+rgm9lQlJTBZWneIIsITK1k4F810ZrPozAwxq2jptI49sqt8ppzyJa0GzvvxGu\\\\\\\/pOsJjpaMDGVW5wrgNP8F1SEfWUnvD3TfuMtOb8C+q1KzL1jVDtda22t9LuQXyyUGTyWOHdPO7qUjVmkfgH+ipUMO+HM0AZIhrndTANLCBdFqRlqPtV8uVzThXR7ijLiJLAPGnZYo\\\\\\\/R\\\\\\\/ZmuZlzykmpihbwPkRKv4\\\\\\\/gdR848RW6Ra95P2fke0eqL1rX2ym8BgNKq5zmQjw+wAVtdbSg\\\\\\\/Yax7IBBUC16Uhjd\\\\\\\/+gEYNSJhBdpxay5m093jiTIFBsKLttnSIi7xDjgj+exXbmEJtdwNwejrzds7l6zg6Zl3F95vcCm+e0n01G3d70F7XUlwCAcqBthmveN1xyTnOuKNih\\\\\\\/4GlZHlY8NCoI6vGE0ZZO60PWgbFKao1FCuIe8JkKs9\\\\\\\/H5MF+V2n8fqWYt9kiu70cI7thfOcF\\\\\\\/J7vgCqSny\\\\\\\/uaixl3WcED+rINEkWvPwuzl5zCcEYppGtgrWkK1fTS\\\\\\\/R+XQa3bb9A19+KueUe\\\\\\\/7foXGqaTGnwCqyPWPrSEum5jsFOzfxZsbJFCHERBMziDSmGSOpHm0LRzdHc6DJYsoC+1j8B39f0p7c4pvFTjkPx7GrCVbwTqXmOTOLJhonLNIktw5guV1aPjBgg+1ud7amH9t4yjfzYROvVTOdjTbX7\\\\\\\/fvwBcwismfJVJyZG2QR3kwxT4nvsrrYTgqhWg\\\\\\\/XX3wiffbJRxDNkwibN9H+\\\\\\\/9Nrd503ucyZXEIQxK1RZZdz4rE48XRPg2HpLM1\\\\\\\/xM2\\\\\\\/UwMkNGxEVVUhYS0oAXwKnH6G5YDBo75yn9cqeV7a7GCgeYV9dV4HrgqiwEhAhVO5yUuEJS+gGEB\\\\\\\/FEpRitHbNTi2z8mpVmvd\\\\\\\/TPKXrbd5uoNHgjfFQIuphUwwnWxr872o8lYkg29zOuy+zkYLAvVTNdFXh79ZLU6UOA6D43KTzfCQEhXjNBEXUbWqAbdy\\\\\\\/5yOR1JqlLBN8iRS5\\\\\\\/kl6\\\\\\\/xLRe77Pd7rzveVPBwLFIz+H5v3eUwwzQ3QZFVZjBvweeLNKKQTOcOg0pz7+tO\\\\\\\/jCL1HaN7MGPDu+2GIZR841fNtRlXBMJNytJYB3PxuS7\\\\\\\/mWvzCAU3DvYkg68HmN21GSBBxtsl1gJ1clFNvFYJOrMmEJZl39JPnv+G7ij+x5+34EoDsRCqKyc5eqcruzMAMDdg551Vh7iHI8GL2qridJKbDuY9xGPbXKgLSw9voSXJtbIFA3aR4ZZuqNylduNLBXjFnSL6\\\\\\\/uWpXaWpeCUZsGieSZx1QiT38YMASxJAwshbla2OtKdBPBoxjsEkF4CKZ9ZXbW3GBSC0IcZ8vGOfwt67OBrFytYCq\\\\\\\/UariEHOyGx4D1ppxqzJgdSwqFfz0VOS+n4FVMqRV+vtMQ+Kqt7oNIaFBplUANbVgrunjMjYuPR6I5xVi5OxWGC7kIrHZeBzb\\\\\\\/KMtsBHOB1WZFCFsABxa+uFrI+ZfhilgTNJUCMzQuoGNohbwWmsiWVW\\\\\\\/3Rp9AOvxY+5VYaq6ShdCwTLzbjRLWEw+pefOJKOl\\\\\\\/jwvtHWjo3Ui7fNHQh0wk8LYQAQK3sPuc6y89882Laf9g1E6sBmGCGm5HboTpTuF3UyhG9\\\\\\\/qPAyUTzc\\\\\\\/zioaCaebAUutk6MeSetYKu3Fi8lSQiaffOeXOcewadvRiksdx70U7fSw6oPLemWckAOPufAfDyBqHbdx3U1BJze8VrAQvVyFE0Zwo4M8rPy5eGBaKWi4ae+mFAlq\\\\\\\/OA730rvXV6ecyeIXOwaxGWqsBPgAdAC1NISbO4OXu+uO8ErT62+ngj3zOiEksRsRr9cRGWhOsK4A5zYrAPt32k+QsXuVYdGDx\\\\\\\/xdxvgf7Oj4hZG99ym9gP1VnQwdnH0\\\\\\\/w52\\\\\\\/h6ia+e28Sh0BTD9Z1+KjvwfEd13dtFhfSufoB5Iwdf+IH1SaGzh\\\\\\\/FMCky3pp5ZVuHIChl1+FpSJrFkx2zFopVH43TRnxIjPOpft4+Gx+\\\\\\\/wnvxMWLLxHbKSue1cZpy1KcNp8kkey4a9XDD7BiQcwLrHeIh9XwY5pHHIL9\\\\\\\/OFVw8jnFoCzpEwerIQrxsQo0M8GK+LWkZVw39mnAmjk3c9c5nIUTA\\\\\\\/pwH2uA7bQh2QeXBUGU7OHHdJftg0AOYK1z8Cd4uJ4Of6Juf9j8l3RQpqt9cUF1soeLo8vLrkkBGNFzXhGMZOjrfZxUY4I8COXczAmXYCB8hxPRItwWik9qTObsSyr5j9crxF71uXv6u2HTMDu2JSnzsfl8ijbbtA+Ur0wvUWoamt\\\\\\\/MXSNuo1k\\\\\\\/dRUR8JjxX8Nf8y81vQIGW+18kS1rszw2JKUkkVx9X7FKeyOJBRSy+d9hhnNX8NdHB2WpqBeL2WwwK6BQiEAdXqqWdgqlDOYW+fJKnGwCy2A0vlp3+Q19wO8\\\\\\\/GsENgvnSJdKcxYdl1RX4KighZNxrg0qV1kt9qf3PRe6zNPjKpJym7ZVDJnXJ7f6IcqZ2DwdS0DG3pMZaQtXXJZiDhVfm4mGmmFkiz6EXS6HZTJZTIMBtu5R25SXPA6Qp7Vbc2ZA8mmJkfBF3fL6Su3vJCTnMy6sPlWXz8Ck99Y4I9hWAlNcvri8z0ogu8gnI7Iwy8J6HPPl8bSesRyaZt8mENoaZr0YbJ0pHf+eTqFUJ2h8MeElmrOCmr9TuNojAww1fQdX1OUeXDsZGuw3X3Y9dfBYTSUr8yzvLND5VOyWWcVwY6ri2g8fLg7OguWuOoPI3og2PYKlZ5GrNyeWMm98z9w0gMWIuQunPYZLmRw2kBlNkq2fFlPzpaDpKbrPyY3l8m\\\\\\\/ii66Dj7O6bYIq6v28bS0cd7JrCO06uIUFDySGA41vMNrP7itx10RZuxkWPbogz\\\\\\\/+HDMiryLwZoDpoJfSDFFiQ04xT\\\\\\\/yTkzn7qohkeukCr6TIi6smFjEQcqpt7tzYg2Ovp05UmhHYIiZyot0QF2BsJ\\\\\\\/29Djn2nY96ski09ku9TrWu9e8Li+kWTAINY+fZGpI1WXmhH3Dw9bgN2\\\\\\\/B30tQaxoyqc1SBeIEs+774k2kzqWgenBGYaUX5YquNJJVRujsg+uVRjeGRh0+9WJAvaYmLLjIqEKRmGzx6kJG2Ssv1h0aUTJXr+zub314ZsM1OPITCegmPS8S3RUG837gjp39FAWYpWZHirML50fbIve2OOxOvSAv683zdchBBPsOTlUTvvFWefVjckGBQWGBbU99b\\\\\\\/KuR4a4QkHpWAPMC7xqnuISpgAeTz1T6G2Q26tXqJMcYqCfIQPi3fWtPR+ZWnr6BbFV\\\\\\\/zEML0V5nh4AP4gld7WFKl6hlLEoif4HLwUDOGnkk891vfEZ8c9UCw2tDlq9snhblgEvqIUDCyGmga+zhafboZBun9cw6p5dkmHwZlKP7OssYioMIuzQoLxuZrg6ccE4ylY4V\\\\\\\/+2T7mCMQCTMCZSTZH3JjA3P8zCgOKMOvmGnwN8Hi5NGi4Bz9PLgRczYfzgrkW64MOiTEiR2x+AYNXCYXAu06mMRTmCGNvmA8Ao77EadnFa+4MXHftcDDxid6lwN8O4R2QsqWhQoJ748rPwg6DVvbmVXGJ0jWFPfdjBR00G9F1T8\\\\\\\/Sqr4rcy7V5QPHzS8xjBuAdZztXdSvmnQA0XgO81CVvgKmdIrCDOeGkqp1qbBzmwTqZSipxdWDX64GQ17nBXLaoH7nN+5OP3EGcvpMa\\\\\\\/R2PreqJwpSEvMAOMFhOqSYZfdOIqrF5TJB3br0udDNnzJyHJGsj3TIH5b79SPA7eAj9Q964VAuKdo3Ffc3R7MtH0VH4uRIxFbEydbho19SuVL8IWqD24wg6ZsmTx8E0m+xIPxTpEHfLDcBw9SGIpsYubSJX3rp2DfY+FfFulsxNj4hu7rf16sCbhcO\\\\\\\/okk8cpoHp9QoBVsWBKcNLQQOPEp9Y2E7a70+Bl8Gd7nuGQN6amz7Zf9tcy7OQdQLp4NuXW+8cf\\\\\\\/\\\\\\\/ykf69Oal8rcqs\\\\\\\/coly9eBCnf8m7R1+Bb8kbBZsGKGhdOoLyWuPtKMXRw0cbk+PA9WhXNWG\\\\\\\/wUvfsF2ZK4WJ7AvOS32vUhDn6whZIr3AEXlD5Ue0YpQLApResfYbvmiM269gB5ZNpO8kU6NHgZeRX0+JIbPR37DNDlOnSkztxfbKeZs8PVhLon03agsVOccBvPWA2FIE3YVt48psmyt3eUW4ppCJRyfF1MDj3dnAZWlet48BMWTNPywnoJhzHkbUOnlun8qKaeq0TYGEje+QCLHyhBYvBIalY+71coq3F43CKLIVK5id71Gi6qzy3g5T+3SgRj98r9EJhvtg6XXgmFYGVo+xIxgX+STTECKa2+Dtj3bsQhEsPhrDSH5S3PGl7jdw+XTagSzCv9rPlIVR6D1fghK6GtWpDIKMeUsogl33ye8fYGHLlsgrq3zenyMrWdNwyMamTJ3zfWTSQlHphVvMFbScNbyG8rWj9mUCe6+7rOLReQ24CiByS5k8J2Zuoytz3TBdxVqdDfl0ThrhkuYnX51qAILHFc6whGnKehwgx4IJPP92BxPAC8mjdA5MaiF0XdjJJF79wN9rWCzuLM6NxS11tcMqvBAubcZV7M4ruZm1+A9smwsx5N+3qqZCE7GQNb5FtuaxQhIIIGpzAQIm0z9pS2G7bT2bXrvSYue3KWIASkyyUW8n3RKrTZuN38S74dQubyrK+UiOxblcbGidLolRzw3TnvJPw1by6diI4XJAWbu7UMdaM+5CZYN5SoADQ5lebiuIPWE3JdNZgRsTqFiYIO5mRmyWqCCtk8tSXs7+RFc5pmK\\\\\\\/LrC9behc4JHMpHc3QrZenS+yGaZOPGtB2i2LuWsegrkIK+OHYMUtSp2vKUTl4ZpZymLNgCfQgCytEkSYosc4XWjKXQGwJfLww+AH\\\\\\\/yOHIL8TfkwcjnkJMrFBn3a+X6+dzZr8ABK4zIY9Ww8YxF5Be\\\\\\\/Q0TwwipNlFRS6yB65ZFXRxJbJvVtznm4b+UJYD2HI2Bk979UCgbYCvDXjav+hvoAeP5a70+8IXlSK2HG2oO+fHygVE3Y06vdxLh2YqBLxJP0cPzetUeS2osQBca\\\\\\\/KumAPuVfMUCUF0qjnFUWAHFlEtoe0RkACBCnL+DawQG+eBuj5rcbxKFinWw+UkMPc6uConT\\\\\\\/jltdh3C04yk7hZr32eJEl\\\\\\\/dm46wmZ2xs\\\\\\\/6D9bmvbEGHQYNWRVjRtngN3ThufUvOeoPXxrntHtIF9y62ayK+Yuz5a8iLmx2ySDeEDgzsROzeTPG1LJbIJOyijJscC847q9lp0ggWXvtkWxfbKJ65nkEQwGQAf75ReM3g87OP5fodoLULTNLjnvPhnMUN6LD7gVs5FTV84Hfp6jihbMdon\\\\\\\/GvPT1YiIadKI4MwZmI975\\\\\\\/Ena6a6ebBGj6vGTSh0ClVd7ofIH3KXMATdgz\\\\\\\/\\\\\\\/sAtsBECtOzVwCVfemtES6EES0vHL9HQ4z88IAu4Kxw43JaVZb1dzs7EVfLK7o8mOYKoZniEgvQ\\\\\\\/5TpVjglKXCujAlwgAJxCL8Ea\\\\\\\/XpAOp8gvcWhr7K1AA\\\\\\\/Lbwa65nrrFHH68aSoqG1KfF8FrLGoGCjci\\\\\\\/8DzMy98jgiGk69AzyCVvZeOlm4Cy\\\\\\\/C5wh9OZUXxfobzX2hf53HbfcCOlxckIvNwSSYjYz+P++p6grOHiFNxzlsvSL8spC1T\\\\\\\/hdh7\\\\\\\/bL05eEfZNpYwrLaHyyf4OZtF15RxQtyCQcqrtg3J0aMvxlLKOIysqy64TR39SQADGsWlbEgz76OvMIysvjoffbuPANqyrcWWcfybDVNW53LsdnHNZy7CySrL3bpfmXfF1paMrF559eq2PWHJN7+g7hXDbl5JSpkkdPm8CSB6r4fb81NllwuyQgb5Ev2WAaf0z1w+c5yQHu+5SnRZG0NMML1Ns7SJxX3IGF4E+KIuc56hKhvDCQu9W2P0aBU4HK1E0Ou0C1JNzxyghrRTuylxIgkgCtPBAZGaiRAsqkJdFh6iTLkOqA55fx6l4dfX3rJ1JINdX+gPmBAekgPOfaWk+nE0wfiCxiG00J+g\\\\\\\/T+dVukV3k+a2fv5m\\\\\\\/GK5hwo9UcMidhDEt5iZPodSamJuL8huX1JXnirS8Pvv9KGLjUTUzNDu\\\\\\\/wp7SdqUw47sc2oYhk8Z7O\\\\\\\/KTp\\\\\\\/Y0uh4BMQIPTP12F5loL7RsT1cnN2ckvXtCj9Y2QDR0ffp3BsZ8l8lK6azrELBYedDcMjfqMx2sPbQEatN6JhGYnGPN5Cq\\\\\\\/UuIYBlRppc0rTxBPB5SViQx7wAR6nvplcwy6FEgs8TCnlv\\\\\\\/GKq2EbUtyucc6g+duJYGYPsvKH2gc\\\\\\\/XPu8MKOPmtH\\\\\\\/BWRaTcDlgXzL84FxPj32+i3wF+tCrcYi1K\\\\\\\/w0Pu8eCdEjkPWPKE8jUbyhp9DeD82S97Lyp9m+U966r8LZz4OiWnr6\\\\\\\/7ZS3eRaxaBrchbyVE8DHhPYUizoITHwB6j8y+D6OtbKvbjwmS3wHxI8CgwbKefY9Hm18\\\\\\\/bAtBZOU4zJBpErcCPYQmTTDJQbkSn+Dh\\\\\\\/yTkx4ncjAngoDAcuoAJabi1JnCXDyC5fMm7n\\\\\\\/yXPrsp+lRQ267It4HM2yc+NAt9YC10WzSVb7HnxMD7QxNgDMCB7UhPAxvIHLLc2h5x6oV0b0hOTGiraak4GGp6w57qaQo0Mw870RsMjOSw\\\\\\\/xhpfzF577Uiaj7gJVPgVtBBAefkGEXqIPtfGefp5QgOXFlAiVpIrhy7RziJcX919bwmBHyzExsoy7\\\\\\\/\\\\\\\/ENC0gguSCCvQP\\\\\\\/Y4zFGK56q7zJolxGh6VRmHMPwuY6FQ0F7Em\\\\\\\/hZxrVHGkrpapxeZpwlims9fsIVIkCUfgwQf7FKIiZ9sX69REOLBNjR65VJi\\\\\\\/iLJJaZFUhg265SKOPnk7tbweNrvcyWfGx7nj+xvcubJ49Hp+X4QWSNcMacU8id16W0LTTIRUSferK3FS9Av1Wkz8ClyXZnbdDwP2CBz6ibQwicDDJRWCq3XhNKFAT33I9jM3wGiPkNuRZHoWuQNI9uoJD5WCpGDx45Em6sFNsotIohsdVi85FEPXedGxlaxzsJ2O+tTE735WuCCEKYGjVLwJ43aKdLbg5UmSEr9JvqKIKCF9K823RLKsMPTl1RAZIsF+bnDOj9KKLqbBaVc16RZNRiF6nbcP11y1O8yDHdO9b3A37LBIgA3NFm9Ypw36Ue9aTy6wpzA7OTUOFKtWVpWVtahL2GoQ+EtBVETVzYraPr7AwC4vsuYo3A26COhEPJOZm6isn2D6xgjHg2+x\\\\\\\/D2xrpIRZBWmzHLphfueQtDSF93oC7NXgPgzKi2gbbRpCI0bGIJIczB9jXtxDxv6NbjQy5oX8TeOn5JUR7q1xN5XG67Eqk6sq5XDL2MoscgaUcHvXSEVgXtPhzijFSkNHOVXBQMiCxgWKw\\\\\\\/aGMLbMPUru1j4BINMt0U0m9gw\\\\\\\/qqnVzO8mdsjDSmKRXSdeWzx0MRT1kvjR091H+Ew15nfuBmNwmQ0c5gVo\\\\\\\/cjHoQ+0OZgoMDZSiYMXGSQvtYWQp\\\\\\\/qAoiQ28NFnGhpKUQfoODdgZGzES9wOy+aQ5kjqD8C5NA5P1c6srEGB0c9ZMDedfPN\\\\\\\/cLERS7Q8dPFnidWnke7UdPfyVwKsW9U37CJ8S08b6CetA4R+F\\\\\\\/iXzbnwBnHVHdOlKq26o+vw55MhSKWXQGDzZjMCTYyS4ypUkyPJQUWs7vF7kU+CohGxIM+\\\\\\\/SZb3tuTa0oujDMPl1nXklQY60P6RXBh6ANzSr+m\\\\\\\/oZl5HVMmLSmigpIjLRO\\\\\\\/0LkF1CoPf2prETiM0Lgxq+bR9UqS7Pgf2XNsV77Yr8GcxZesK7+O9mPs04lb+Gm+c7PqJvntrSPsKSqdC6DuRJRfUONQqq93+kODeMK3MnunKzHvgGM9wvR0Lsjzj+EHEhKbykQO73zQBKuOIbIAQjkvms45GJ61KE3GA3jQjbzsVcnGFFfzMRSqFT+Ei+7pSTDH02HH2V+PvwKZ0fXjLDG6LKIebZJThZDohTqVKG7fyotab4WjAAJ3eIGIK7YXsh7Lue4a0Osuc+uTtSD5eBcauk3p7lMw9ZqBTDgEl3lZA1J1eoZleZEGF3V5RQOATw+IF1F7J9vlvouY9HdRBUxTWV+Kgpy39+dQygOabfJ3BAP3MezuYzG27g1Vymqgv3+LrDAIQlwWHjsHNmRiZ6wnQ8ozZ1ZFNmwd5Y5\\\\\\\/5iy1Af4vYeAhpRWJzyHAoyTIAG\\\\\\\/0051tZGQGCmmkofai22UCxfXpixLVFbHSkEF58Spj16g1W38geI+QOXffmZ3\\\\\\\/yI8\\\\\\\/b7YxwnJ\\\\\\\/JQR\\\\\\\/qiYi6hdV3FLotSEjcTLmlo1bK9YR9D\\\\\\\/T6KfdJEIbSAQmbYMI2BTeEeUKfzYyeceL\\\\\\\/CNVAxu1eSwMkY8m7EiFtEC34opGOCaExH+e8efmGeOqWOiK1N9CES6prul2puXWj64NZ2dUiWS6Ca4T\\\\\\\/N1+yUdY7AtcYWXQVHutT6MYkHi7Ktad6PncHxYcJXaEWdaS\\\\\\\/UnFR2m3QGYes1S3PtrJQOJhVaTa42NuZ7ZncRRcL4f8SrBZ3N\\\\\\\/\\\\\\\/0OiQ+vkea5icwMqH2XNabYOz0UNgtdyB1L60DPV4cWdmhomApPorsCH0WZ\\\\\\\/KMdtanX0ggivqxWay0bLJ\\\\\\\/unldOPLWNEYjiMdWo0IgywUoekA\\\\\\\/p8IGHyhyjY9fZjEF9oq+qSoJL5CZkFSUshLsc7cIW1oFYqUmB+WDlYS1P\\\\\\\/9UjzEqcP0aKHC1VRFE2gbq6+JTetUE6PXgjHTcMoKgPFwUmgkuGB4O1xm8o0R1LGc3vcEzf5m\\\\\\\/tbwBKPSMWYRxhYvZWR3qEo8PUK2w6v9krDOQvziXbXI8XZ\\\\\\\/v8Ez8RBwNdDgWdo+UNvR\\\\\\\/4sLEWDXP3wtjutrQKJ4hY1m+Fkow4i+fbQS8w\\\\\\\/7j2aUkDCftnFXI55G7bTAsV6GRsqXM4g5nCyk1tpc\\\\\\\/6bPHqNe2hscRYPP67MdYt9SpDVAyYvlhQZ++BAGUkbfUcQmC3mkX2VLyVizzNyeVQGhdnwh906jsik2KK5X9Z9Lu3p+3oeg1TWBuXw6FUyIOYgvivx0S1\\\\\\\/LfLegK1Eiv8gX34xg\\\\\\\/aLf9jKXdk8bkPjDczJmtoWtRdSqxCZVmQTZ856NJVUY\\\\\\\/Cq3L+6PYNiju4essECL2gGSiq9ALusSCigtfarGDlNhGEmjpfuJ\\\\\\\/OTf70jEKr7ojU2sZLV8P6vGMRkdagbYoGyBEdOVbSDHDxvi\\\\\\\/lyQj3vafl5dKgV0TGW59H4nKVeLBj4\\\\\\\/e3J1X7hAz9M0Zbj8QC124CuGX8pRu1vSj0guV1S7Wbjiw9FpvVHg8EUhZwv9\\\\\\\/qfj+637uWzWSku8APWPhQH\\\\\\\/MaIrWT+7QO9z8PZHXfKJObgGn5JD0miUVcYOz3bdhT9\\\\\\\/wB2xoqqh5Dc8AL0qv\\\\\\\/QjplEbkmz7a3iay2p8MgnODI6W\\\\\\\/F3qBYuuVmne7rqz03TlJv3O9oXsUl8Wx2aYsYomTOtf2Hz1QwckSfSgJOo6ISVRTlSW3VptlsuoCy3bLZ02XyGIrD7+J\\\\\\\/7+IvOqgFTSjofDhtdP1niDKRcHcDHQXk4\\\\\\\/aLpsArYEobm455X5DC0oV+BBKq7zDPR+rhSVHQBnG23uZ8pW473ujosZqki3Rl99mh8CRZh0ecoGG9KHBbTgohgC3VbRauOerStIJpVGZBfI\\\\\\\/uQBRVjKHvXnPdVV1nwUBI\\\\\\\/TfbRRXbov1ybHoXw6YCG0oRQ0cXNp1UV3D++QlHQVfC9g7Y9KdRRrp6\\\\\\\/HYHA4AtQCYGk1Z3fKjmmPX2Txz6jDmUTuFknAq31izSEkm7Mg0KLqlnVaoEe89YZbnLl9D1X79p+eOLZofsotxMvXgpAc74ra4x6BHd\\\\\\\/GEcpsoSGEppeucxx7lF8LZ\\\\\\\/1\\\\\\\/cKRHvhKhydgHk\\\\\\\/xtIH9cPmSk+g504Ql\\\\\\\/gCjg06khIwzJzeTtOHDkYQmxt25lDxAIbyO5iUfwhfx97OT1ZMDd29dT\\\\\\\/6IJA44vStQVo09BGsz8i5owL81vl0i4gMHosHMC0ywzp4i4cwRqLKM7B+rFw2OkR3wP7Yyhf0jCbO29xy2gh41YRjZMxuRUT5maR0AtveBFYKnpx6XkL331\\\\\\\/DInhUiGFLSkbQPSNJBvfx+W2yAAdAjI8zVQvM\\\\\\\/uh2fOwLoTfU3osiwdKXe2\\\\\\\/0r90wonUQGtGMXvBZV+t17DhakDmVuVjC3H4gOdHl\\\\\\\/TcGr5yUhN9hW3PaaJKCpIbXeWzkgB\\\\\\\/URK540FRNcYW\\\\\\\/rn1z0nx4zIO7uFI5oqeUjnT7O0CJbjEFSMoO8NinVGGpswn1iNhVS0vnm1tJELC9FbngKG8tM4y2xbTlJFTE\\\\\\\/TRklWIOFedsmX3WezrpLOATLmV8yY5Y9dsnPnPB2UifAJpG9pfLnUqZIodmyhGeA0H+m3WYpLVmGUwwDPDO+juLkFUFtPNt0ucnLW8qhZRh6b+tTpjCtzksoKAZVIvywihFtWjEjNpDKeocbngEEZEd++wWAvajdL32TxFjYso1qEL3M84uHpYSzIcxYkNn5A2wrZZVtwQ6bDI4rqviUDGLj4ZXbJ6S\\\\\\\/p2CuPeHi09h3Lho\\\\\\\/93BNMxb687UR4CGzbDxwYGqv+RXR0hlXCk8Q7zb7q4+YmiuvQZcmhRp\\\\\\\/seb8KE9N4zXju8iWcbq1rG2I5xA5VFnmifHlIWnmoQ3CWFqdyLPtPJRa1VeCv8G7cbXqdfmWltbiBiaydIyegOovi5x5qF\\\\\\\/BYABiN7lndTFdGK9Z6qqD85o7B\\\\\\\/e6rb7HUEF85FKcWfx7FJgxAb7W4fNC8J7F2vbqy0cj9i1jXkCA1TEZJ0R\\\\\\\/LiuiFQeOdLe46Ca1UlR11MVzjU6qQqRviq8V+J9gzduNDjn4OZ2Ps\\\\\\\/jED2gSFB0I4Z6N\\\\\\\/Y5\\\\\\\/Xksa2aboEwjJdBdeQexdAbHGf1wghdm5pJyFjQvrtBUjEJmv7XBsRacvbOfbzkc9JbSgU1iU3Iu4x7YgD+66YjDeMOF1b6IitSk+dzIjBuBwJHnXBCpYRQlngmkLN11tZvD5gvFObr5QSSCgZ8e2KDv8eEmFDOSu+o00b8aBiblD8XnJegekmxTcc+oLwKRlVF45mvFMzpRHqbgizQ+31CsCMEuSJvN0q+8OwrqzLdHlBShmsW7hMyG2MaEIbgcQY8uBJHt8ESvponCOqf2AnnqXM\\\\\\\/H5hzbWf5Rp\\\\\\\/DfILZ2TEnetj1sRzMguMg2GVgPOIVlqmwlurULygVkDaB+G03HkX7DJePYaYa2z4dwXnHH+x2iv9phwMIzoh71apccxPkAS2N4RckZf6HU0EmJkL4HojL2sksz3t4LCv6ALcnoYxJZThhDvzCJOezAyahRqA6NLKvuOf0YNJLA7M7izJPkIkIru\\\\\\\/9U6Vw9hPh7hdtwRXCr7O60pX7W5bWKDvw\\\\\\\/bbSrTDhcYhir9Jqb2OHmM95YUuR\\\\\\\/BvvsVQ5+Rfz3fJ\\\\\\\/v3WGOTOqZifpPYAYq+bLYlre1MBWXJLp93K35oB99LChwsEpjVrik0IVyUhFm8u+X7U+IHmNwHKLuhXWuS0F+08XoDuukkJksoBi67yJSChND9m4d+Tf31cK14gKPVnkag++oiWuAv4wqo0S1DusWYHoSG5CIz+wsNTchcWnXvZ7rv0MpXMdsb04fhOpYWD8+YTWIJwXtBsTcsb82J7edx7Lzri+EQ4JN7xTIR8whV5y1tYIl3BBSPUWHXuehDxRMYNK\\\\\\\/XaLnRMAiVF8GFj0rHBSw+65JcTQpxsu0HtXTIoeETZeLO25Ny6vAcPjz4E8oItQwT5n\\\\\\\/sDdPcnV\\\\\\\/ENrkKGCLC2ojw68YrVtNI3fs1+AN8kddNoUwHHaBzDSPcq0MjqvGhjqNcoWsfri7cZ21khZJUXqkWy6iqhSqEr\\\\\\\/sjFrkbtl59Ano6JLDVbcCuZDt5cj4T+ByKdrm\\\\\\\/Tm+LSKIlRR4TZzv+5lh4MK7thVGFzymKj94o2tlnBx5hyLBfSGZ5viIU\\\\\\\/ps5EL0uNN+qg\\\\\\\/OoKt3xmOn3vHvszdmpwi9j9yuDoBJtVNq+PMeW\\\\\\\/\\\\\\\/ZHpaegE2CWWNoao9AFepxX1avKWbi4tbR5OpMoiFYpGy3eRrYj6wjcG0+Ohs3aqN8ThV1hkdZ7chbCFNsDiZ5RfpjNgArt5o2bs7s7RPEbTpVumBpJLar5AOTCJCJfMyanoV4kuvv6BLCj+pLYJ1pt+9dlnovMWKCV45T8rKy\\\\\\\/zf3bLsatqntCFU+7BWdiMxX2vJLhQ7lhdsWR1KE7uRYLKynbETIxhItdi8ZnqMuz9RNym10y7NwNZ4djcYVklyo6mC9GSDBuPLDPiYjx3U1tjNVlNGiiqjHdK876RRC9RQ84npFUCSKWSbVrFJ4i7QoHHE3esF5OTpXLI7XFRptpEHryZvvipmf1RWds8ZL9IwA4+EcysqPsE0vM7Lm+3FjE1rh3K52WL0U8GwK5fXD2ERoWULS8T5x2TRu18qzMLMYpFB86wXs48S2HY+yuz7PgdMMhPRTP\\\\\\\/5x+tJKqzN3OrrJYIEvDF60lXTj6Ej5AqCNtq+6K4gpsSnb5tPR5JrEhLpD9tKl9s1tj3LnyWvrGmOo3Wuru8pdsZyj6uYHrNywKrp3EGoVAfwoFO5DI8ifipEnE0snuaPt0JFHRkC4UPIxrzQvEzbRzc9bn+gfA0s+hmKA\\\\\\\/slXXWllINbkeKW7fAWV3AG1sSTmrTsIBYS9ZurovxaptkVyuVysXbBevY+ChpR9LY8\\\\\\\/+gTUAH16ib+6bq9sinkoKuGHx+tkHDx5lZ4P5vTfeGMPiMnJT3XVipY\\\\\\\/V5q48xq+ejIE5v1oRIXt08+aizYai7t9fG1r\\\\\\\/+t3NN6jQSeKT4+jUlyTt3pNOR7fhBUy4O1O2toaqsp\\\\\\\/uarwphlG6kLN\\\\\\\/V5s2POH4iCKqkddJ+PD\\\\\\\/pcLZksr99THAk6dJW8Bmft199FGp8GjwmsdiZ5B8LkqaTvJSvs37EjOIGZF2clQlyGRP1QxU0nIcRq8y6NtGGmieNpfcz5AP0vTPUHN\\\\\\\/mTYYHhx5VuPECZVsAyze2Unqa9yaLKo7GRbJfcU3jW9PO76ZOaWWM04Qlo6VLoP6jzDxuhNZLoKa1StSmbRQ9U6mjDw6yLUjf7nJbgZMapSsTmftXg1olG0FzQNOAtj\\\\\\\/9resuKeKfJ1NSdRa4HfayVxLZhq6Ur19KFL1YoLFrIqR7JsPIQ8LLDXU75JIwkLpSfuMYuvmzCpTjfrOxXlDy50gf3upeHLX6gblELD1edhpE8AS1FNuDb9zY5H5RZZe8jF9mDgFciZ27XPVam9eyJK757DQpX5K6W9+QNTKCUNOA0zRT5Rm10m2XJ1W9RlevP1A97QGrGlYan0JAwpzklSoUf6mAafAvmdmxJsCdUYY54KU+FwWqzuOwQvi4wQ5bwzXS6Ss7ovBjnFBuiq7Y+sHpsNMsP2FKD48QpW50gAf5cuvi1PL2jqBL7WuScZgxiDziRtwf4bW7jvDWwqMCyEUiGAUbWmsGF3npMeyg242cwGZIEOCDp1xNj8sJ7GP1I4OcBxoMATfpt9O60Iv5C\\\\\\\/DmjcSsFtuuCWuij+m9XEbMnE2UrIpyoUl8trlsoFLqqZuh1x1dhtqIJcllOuaXzZVPMRZgNEihTrOIbEjoCald2oDJJBe56OzqWYzl6x7ngig7wKreYoIGobGY7f0eEZx7jBxi\\\\\\\/muJG3oT\\\\\\\/IBzaY0Km4ud5sAdh1wezIFlRvUcMxY5MjHWYITA7Z2iXHkJk2DGnLNIGIj8oKAJoEYpYhYZ8VZw3O7QDqaRiwu1cJ+lz1dX7o34M7U4OOCE7CcFb0lp65+Z+DhbsaXTRkr1vCpEj0qDPg6oybxmGKlsOFKbkthhEtohnmVNiH1IwOkHHOl+9IKX4lftZh5BThMfldkOAfQpFJ4yznrg6buMSLNtpHs2aD8Roml9nzTztZJSkoHkjvRHMTyTGKQhO9+1acuZYpBQ4uUmq5IbHeKkMArGG92zeJekFmZxQyFkpCun\\\\\\\/Pko2gdEx1+AZ0A1ivQDZsI3ndPp81QZxMYStFeb\\\\\\\/uZDGcWjJAW7Jz3JSJDDPno8k1mKmvlBmGKpHdfXb0bzYU4v6uStKW4JBPfwn9lfhSwGA3Jy+Pa43UiehFwPRRMw7JPaiUdRKaKZtwA0J2hDqQu4gh7bPn4JlFG7vVhz8VtWAybebrrovsA2gp+sOd9FLvI\\\\\\\/XPHQQwDZo5C1DMeyIZPM1ZB4VApclbDDg\\\\\\\/CFl4OglOBGpQCnji3ygrauYokdRsAr75t8llJGJYFKt3Cshjm3LfCpCkUmImbWc8CvT5NH7aQQxQB0dCcKrZCQv+J2FvPYJ92ZSPjNX6vi3\\\\\\\/RbQGw2c3bRlQsRMnMivx9t7fUwtNZJWKtpH9xJ7MzwxYRKlqHVGUkc8zWIe8lU9XkmuHjlcCY5mAF7uCgRTnFKzwBqs21\\\\\\\/R4uEui2BurPtd12QhNniWUaXFqsqBNVOa7T\\\\\\\/1wcuSBN67JcA3qJCMPlupDIa8F\\\\\\\/UPdeLygsyTR7Ba04Uu15I4cPiTDuPzrDrXiQpL7s1+OS\\\\\\\/AOPIV\\\\\\\/EaDh9+xYre1YgVstSQZg6ox+24p8QvWgOBSfrm0hfHgfgEvwxQYCA9qusqh1uND1wNMpQpHkvh9dcWfaE3e5SbGH4twkZrElARRLVuimg7vAzvxxf8Bb8IUrXnOL\\\\\\\/5htgqAPMqdEfOiv7lkfHVUWgD1J2joXsOiqFWTnOvjnXIScLMRD5C3dJxYMPynNBESvgp1vmC2xgytBoPyaH0jxP5vtszlDklLe\\\\\\\/TZFZkiozkehO8hv84g7D85y0vj\\\\\\\/Yg+gpaGZsNz6IGe6XDFTtuCE3d7n3VOxqpXBKDuGW0MbXFXwah3anPk6Wr6EpwUvr5Ze6eyI57\\\\\\\/xFdQ0eYTJxMswRyxTcbfrgFidZAAv5QyNnYgTIJTzDEkx2zH14UoLfYbpHnxCW4P9Mcd9vKzwguwOzgw4c+tPTh82AdQ2H0enqHXlqn\\\\\\\/js4Uj9vbs0eBjtj3xyL+qszKYUFSLwQTOaMIuOjTX71xqGII3JeX1eEaQaEf7g8aqpDTufTwHuXp9xaQVC+Mi099UMSdUcBvANrCCOyrEvufU2dpCeEyMj4o6Chxesp7yaaAbUgWUnWY720m6ZWGSXU9Rr0SGtED3NsaNIB3BXXubDyQatGC9paDl\\\\\\\/eP+0DKWzwxJRJWpO0tHAfJ1prRFutejLq5FSLaG8GzMip+yiXw\\\\\\\/2WRVRbQNQK7sDatcdiO23R\\\\\\\/scbwUCAU9W3Cp4b75jk0je+c0EquasoTK0ZJUeA3cmu6JoqNZEfCY8mLwisWahsyILqd00oealz4PLXUdkJcpkKCVc44E5fXufr\\\\\\\/hYMcTnNhZ3wHT4\\\\\\\/WhBZtNg\\\\\\\/n37xeIDWyNyqLdMx\\\\\\\/xlu5SIn6TsxDVD479Tq8dyb4Acqth\\\\\\\/497C5mJMJSP023tBwNIuC\\\\\\\/CGIJZq6TZGmSMOwbIh9Y6ZMpMiJxODc1zGEZ4ynm+sXjngLk6yRMQgBDAGxtERv9uSjZVtwhm3J7fw7UYC\\\\\\\/5BMJvARwwYs3d3nXbf1MT1\\\\\\\/rZ27LXnWewbPh1zaptstd+irRDB1HUaOv0e00CQzKM+\\\\\\\/3cDTeDgTrdLMPV1PPT+zOHsJp+iFVLgrYpO1R3vmVmcC5MtR2B+bPTwqIyf9NuWL9rZmF3rt3NXouvmF+\\\\\\\/l3sUQhZacTaa+xtdUkXO\\\\\\\/3L\\\\\\\/CsK45+HnZDMIY1rFnLZXhEger1a9PM+xcIGVCiZdWSgzm1FnYOt5TUW4sUiqTB9HCkYWdiTtL2l+1WbhdIHnSMK2E7j1WBp\\\\\\\/PZwbmkoRXCR\\\\\\\/t9BnCQlVDaxPvD+sIM\\\\\\\/DqTljLSYpKozfP321xgOL90A17xjZkt4gv+mJPWkFYxx2ACN1OSGKzDsQJj9sfCd8\\\\\\\/YRvOLKg\\\\\\\/wySCEuXVfGW\\\\\\\/JXDNtTY9Ly4rNXXdEO9GweqjWu0zrk\\\\\\\/tEBnEI1M306jBdyi3Xvrcnijqq4EtHRTgekl95t8\\\\\\\/Rh4fSfrWXP8NRQBTNMJayIjwaC6bMw1lpyNUWCWFjIGvXlDNOO7mdebWQt3aZbYOAZAdd+mQYAJ1nN2FjuCgmCRJBcQmUERyPFNMDLlo3jBK0kR8J3gO5H0yMmNOmnDfIHJX8kVKYq834+wG\\\\\\\/65Ijscg1HCh9+NoKp3Sualgdo\\\\\\\/2DW++Sm7WJmRNdzngnus+Z\\\\\\\/2y1yX+gwxXN\\\\\\\/fC9e8CI\\\\\\\/ZqUZVoxwQnzEpSFx+g2wgnbajEVsFKMgeIMvdTE3yu7zY8kxwHGizdnN9MOAw3Jt+tPHgAD8uCSazEx7ZV7JEZKgPW1gXc4OzPgWvwj0D0xk3Z1t50G44wm2ZwQYxpM4yB38M7YcxYeKAS+paW+pFNvX7zhWtpCFYSgZoUNxkK67CRrn71EvnGXyW0c+lr5OXBWwmZ\\\\\\\/4sRGqOjgUQy81QGQnwVk6re1moA\\\\\\\/pkDJSZEdiIfRq+MVk8myvLOvcitTc\\\\\\\/uxbz0U4pzWlSr1Bbhlnzc8+npz+4WyToqwXD4MxQIlCkPLCVlTm3tQvLaUpHFJcv6k4Y9W9Zy2OmWQHlZH0soTKJXqWOLN+DmvSgRCynsKZsEMBmwhICLi+0HFLQlsYlrjXAOhQbIHumsSE0rvzBoqSOi4IdgGNruRtxmfAu4\\\\\\\/SQOCSbIAXcMJF3mykVBA+ocVcOmc99LTbRI1CHs\\\\\\\/3R6Mk2pC67U9X+2tpBIgfzPVj8Z6TNRawsjubQbMgwEdMOY5e13lfp6izqltfcRFTboX5dIB95\\\\\\\/5EL4fkv4gOtdgACP0lQDXVR3FEkUquEbJwKlSejR72ETUGCg+Y+8srLp\\\\\\\/m63sZC3Nyc5Uh3+MngkY9pSpYADqi9G29l7OHB8agUI9y8QGWbr02rNMMBfEPxBVwqa39XgpRK2+81w4m+3oN\\\\\\\/PcAIaz9QK9QlbijhmjutbXbgiU+9885wnuNgv8nqWgeX4md2PMxmdIBw8j4+ypYzAUqvoI7qOU0qUkhc70VZdnWDh16wsibgD8ia6qchGHberd4tDFwbtx3fG6RUfjAq5HH4MlbnvKvy5zruFKHqFt8M34JgU+xfR6RS3Bw+XrUJ9IWBXNvzm4vkEtKmnK3kni5fggjyJcuBLndlfW8PXPNHz8Xn9L8+pWqvpkoT54x+BpgZp2jP4lzjgeZDo\\\\\\\/rIeMrVj8HolmveBJdIlFGLXlVoJrdM9T9ewvAYK7Kf5dsHJo62uMN0awDlJtGJfXaW2s8lpcGvFgA9nqLQ4amDXspiyISj5VO67APQ8QHerMoMSeEUPezhsryY6ddPuTD1PStOq\\\\\\\/QUDgI\\\\\\\/1KBIRpJvsQMgXcNp2CWwcbkNGlrgOFVAHue9tmPv3w9D0gH7oCr5z3mKr6Mr30k5FplNKZlxX\\\\\\\/adZ\\\\\\\/M+Mr6i7u+NzwYFp9srNoxrghKErNdiWMLpdXln3eE3zErk3f+MBRDhOqFm+SY\\\\\\\/kUscSI2GdgIc2v0OlWtWFZfeQ\\\\\\\/0nnekkarJBB+hTf7cke6tYuPYCVLqfZQgOeqJWsDNyXo\\\\\\\/35mgEichr+z5UfVqCD8ZmIjq1Ec\\\\\\\/NQbirFTnI0oDe7SRqPU++zyDYvc035FHLnzn\\\\\\\/s+c6FF6DaLf27sXqVZ1Ogo5V62qFZmKoPdCU3nWx67vmi\\\\\\\/6pQMlq59hAsI3VaNLsDEHU2of2F6b03LaMosbQ3iriN3kVkW9INS7aYuPL3MKmvGfUBNgg2H7iSpSZR\\\\\\\/GcpgNqqvOQakCE3pmqMePi5dKMfO9RKwi7HBe1\\\\\\\/lqWTzYEDFzLCBTjaV+9F7OjX2WWyKSUStKPu9d4gI7jOLufsz9okcR31\\\\\\\/T\\\\\\\/Vv1midEPd0OYXdQNhNmlAcY8XauDzaMHRiwEd5mZ\\\\\\\/V\\\\\\\/sEvVcbWdhSGEvdsjrxPXO3ZfPB8jDLkeZCps2Y9ZKvPyKrJeZVzFuzd83fqHXmd\\\\\\\/ESLA\\\\\\\/wcszejVatHnO\\\\\\\/fVyMUtUswOckfPCzQtIBjxsVEknWp\\\\\\\/8k+1p63zPw4Tw9Byu3\\\\\\\/o0HNbei87p2eOeAoDXb+yp2JJBp+rHHZA6WjuLtJU4wv\\\\\\\/augxjNq7nVduKpieUKOFPH7RUN7DaByARI4Z4YelqEzIDUx2XwI44+64Ho\\\\\\\/2zphwB1DbcF7cxRAtITSL50SuWh4TOi8ztyxgpAkcG3ZoEBEcYvXGlvpO+pFn2bWh5awWwxSE7yTrrADuNMZuMtsMmPu8Bxb08i7aw3DUpb2fRoTkkwP0SSUASEWGAZw2MK5pWR5Nz\\\\\\\/6gmsTlQQl88\\\\\\\/TNxZ5QfJuLh0vvWwGOLIWf0GaGmyKS9gIcx0CVzf+UsrLqw+4UKiDT3iS9PTfIoUnVGMIcn28Gx+++6Rd9mnC1g6qWfcJqHA+17hZigK9PmzggdhIJZxKQSkL3WG9dTldSKvHBlfhRey0ZUnv5fYZIQazQ+uUPcRSj6Eaf3dN6UMWkSK8CXY3nTjtvayqRnc3U+y85PZA6gwysB+CpaT+lUtNmrbGz82mCNHm3Jyt7ONr75hZ\\\\\\\/Dg9ZHLSCzV+E97ZzwF0+7MJ2y55y+c2KksWeKD1FXoeJez0Biohm0DGOR6IB8z+3H0Qdbq7lRu9CNpYLwr0Lipsh1lHlBFVuAePqNbaGDkDAsZfFW1wX5hkYbzgdOfUTcXhPjPOZ5Dl8gqqcn3\\\\\\\/hPOe\\\\\\\/bv9HElDP+A9AXzPmXJYYmdxklNunlSIo3yzumFqINAzS8ZEWd7\\\\\\\/eG4tHPsvUVwoDDYXzmNpINf6ZHwEU\\\\\\\/B3LCIc2zEm\\\\\\\/iov\\\\\\\/QeoHJopPp9m4nhjlwoeDrPo57XIqusbTE5AOyEenzqt6yA0rSKbo5K8f0UACH3Krms0KzfsN79Revd1yXgOVd5h6zUDAtkDaSmSwRbZ6xV0cpnPNmB3ZpUJuoi8K6K3XMt48WO7wE7gcyh9yOD36D2P6qBX2LeArjROpEGZHbOJ6m+2+itorrs5N5nVPnAIa1ztOBX23+Nb06t7Lzk7yFrUPMxzmNb3NeqhlKJ4bSyUxbP62fGPKHUoZ+sw8XtgeWDrpkxzs2xZpUSax320ut6Vss4PzzK3atUpowycpyKMaELlZEUkEbOZGj5jYTzDyjvVID8sl7cyzSvQZ1tTQQR+grOZh+GwyxZjfVTVi1vQQu0pjRB\\\\\\\/afpvL9npPW+PsTG9g6WiFDFav2pswf4ov5wbdJscUezkGFs+8WIwI5OVxJunctiMY9h6hDC9mtPzxbDzNPkxlgDxJp1WuQiMviLCteM24RYv\\\\\\\/tVq6mEBE6gkDABA6nPgG3rT+XjrIRrMq\\\\\\\/VYjEOFrAIxHOO8nzYA3\\\\\\\/wZ374EVqtLsEVlZx+fMPg07BQ3QWxXW4nrWOhnHuMOuuIzOdYxDcKNOuC+Ba0ouuMo+Qeo+xKH5U7cmZTj\\\\\\\/0Y9vQugm\\\\\\\/wSgjVFjobNOigBCjkE+6Iyot8k\\\\\\\/VG7TKCbVlL633zIqkFQ+j\\\\\\\/rih7zUVz6OkgmnD5NyemMgdBLQwb\\\\\\\/Im0F4iOdDEjibv4ud2mCyogc3B4V6++D5l5Smxv7777Z0Oa+ZqEUoBjWSRNM3X0Vs\\\\\\\/xCwprCyZzgyjuViF0tW6mgeYenLaEZCL0PIhb\\\\\\\/blrzQ70OfAKpzGX9obidc2mOZbiUtWnSuUOtbjt2vDzX2fB5p8sFvhOojnSmby4ZUBvqljoEKvyvYjE5EFka3JKUE1s1qKihTJEy\\\\\\\/GOGUjV4iA3VBOyG1HhXojyaTD8h\\\\\\\/gFIV54lXZ\\\\\\\/ggX4lKzCvwpYvbiEV9ozHpm7szJkPtaFpd\\\\\\\/rmr+jbdyU7cQPy4bPiZhnwbPGtw+TCR8vbFqY8uFdpnFIiHBJ\\\\\\\/GYD9RJIglwbpsd2baP8oXN7a2T73bXNK\\\\\\\/C4dOcxySUYNu\\\\\\\/JWKpb+U1MzTnU0lg4P2mtAt3eCnG4x2JHypVuRoyqvnv82EcBs7L6qX29NIN3K4w2k9QR5NOqQT1czyoDI7LlkSqQn5WMdgX9Ja9A8ZqGyY8hkI2iwYNSVstNvM7JS6IagfZW1RFbNhmaCpU4xsFO2foxcOqKtuZJHsr91Ax9Myu9ALCRXoqJMUSsQXflgSUZtopRXGYLvDHov6\\\\\\\/i4PI\\\\\\\/UXWV+GpFNccGUhyLqU8ZFmDg\\\\\\\/Kchxg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022xT2dSI8zdyxRZ\\\\\\\/vrjSOMrg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002266d9125c9c6f650ba46596160f0c97abe6760d87fe9a0f3aa7f7cd68be96fb81\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022BPi0UGPZnrv4yr0+mW+1amW8IZZVJB3nk065BH9NmONpCfXHZuWv1j7\\\\\\\/tqFoLvuYlMBuVkT6ZKp2yIsrhoaYhHXyvHIKjtJ+M0WgXTBQSJ1Bz4luaRSDHw+2ezyUrOjTtwVzbDhxXCuZVvScYi8OjM1s5QwJfYANeiT1onsON6Nu6azbiRjcFd3zvtBd+ErhiJsCQTDIvsh11c5DMpCsMe5tyzlPFMkGInBG9ziUDkPr7UKw8KwSxkhKBIXsoOkDxh08ucrSOdCHCFJcDcez\\\\\\\/SWBZc8VefmUWaXTZSyUCJfwBeIXTMCX\\\\\\\/KdWETQY9piPF5HSnSHg9BBWypUtF5HOV2lXrr5b4Zl4xH0mBjZJA7x1N4tfYDx50LkltfSk5IcF+aCGBIfaKfiu4kjS1kqf0NvT7DGE8gQlmyrTqkIWdDAL2cLGujLnzA9+vwzawiXik\\\\\\\/5HoZYLBAEK+pE+IVekku0RnGIFHfa20VgGOlLW\\\\\\\/P7z65C2TMGal4TzVe8y7eW83mlv265N7vBMNXrnVmuJ39bfuk2vhRS81cUuNt1OsymOQ6Np94hpsEQLKnFzyvTWdu5fvbspq8a0oc4ajA4qwPpDYIWzbmYckJz3EmfGOoK2ORAE1J7NxARn6X\\\\\\\/KqZx1Imz+uwYlQYjF\\\\\\\/A\\\\\\\/61Ef88AbGxa58oNl96FMFVvWBUc7f6brkqDMJMyE\\\\\\\/oAUaAlos7Sxsu0mmAfHZH8N1fe0gE0mCIC0I4nbSJjWn9RFDEKI\\\\\\\/RQPyJysg4U8q8Hs0eiLc\\\\\\\/OCagFYBZc3Mp6EP3ovZ5JwkW2IKZos44V45qvnVlomQhrQQ5utrGg75VX8+36JpgtjzuGPCIiKY1yZq0pqYNhfWSHjlbqTameKPwWRd4i65Oz7CHE5SMikj24sopDfdgQz6PHKAti\\\\\\\/RY9tsEFgzOu+94RXln1llkJQ71QkacqYtFl7JtidByBBPN0y2Hc9DdnTDlZlaySv\\\\\\\/nIvgoi55ToJfyAIWlm0GMD0AaZ6BS6VeA5DTxC\\\\\\\/gkzO2yrSNDG2Q4GdcAlPyY7qbRJiLrSkeTuwpyNJr9\\\\\\\/G+FFvgl7I99ywcBysGuBRGctNM0\\\\\\\/efXmHBjd4RaAIOOHO3Z6w4\\\\\\\/tEv7qzPGUPME2f42qiRnnFvL2e3ql5T6rRe5\\\\\\\/iQIJQjIFoFpHCJ5y74reX53xkkJlrrIRZJD2qmkRx7GkqUxVZLz2eZkw7KJEh1FIZOlUDUkVZgMX\\\\\\\/ByBOQHTGdkmjCELyKgi\\\\\\\/HJmAbdX\\\\\\\/T\\\\\\\/meESTDNG\\\\\\\/fvK1GzSoS55XRQXypZkJ3AVhJIGfSbseiY9NRXIjrl1GF0HFE7VIGm8LdH1WmBqROWAzcZUsQQwFNAnfme9hNOpzJLFSPYRVDt18lUDoT2u+K11MwV4hYa4iK7Zs4yEgbVAT\\\\\\\/cahL31rnOPObGJYKY8EnOjTDfsycKShg0VEBykVwpC0+I6xM1XPiCRDfOQig\\\\\\\/XQA9R8UKPJFG1m4u6ZLD7zGioy9MLD7RsAcTDuGMriGyHLl0IPb52U+7BJoYsA14xV6GwuWpgJ8oDzsu8aG7c2obgimL6bzXXI02+Ov2nj5cIiEdM8o3BE9bQa2X1HUzbbuoasck2\\\\\\\/RrjPNVrVEWrf7kXhbvt753UixeabF7cyPGgZ2YOtpfkm047fdGZeNfCZhRA7WrvamHDhql8Pcm+1H3II2cc5kEaOIjDaoDKW2DDamCTwEYUHYb9Br0W0awV2DesOC9X1K3WeKH8hRyASR79kYE\\\\\\\/skt4wD5zhRJgEWsWjekU3uVVqHnDiSljNM53EpqJVVaw2IXUGnq4kN+Wup8nFzPrksfHMF1ckEV1zwL8xk3MQdVylWzSe3V3TmmkYZREv3vparSdELUHJuqiPZIpEls3wwlNix3wsydJQR2A+k\\\\\\\/+JFYjv9DvbeGaObaxdUeoq\\\\\\\/PnPMOnt171OUJeIkru6CIevKQuSTNeqqzPNOhROA5ef0CBhjmcw9qUZ6POk9F03MFnquqBrUxSKX8bl1TdF6H7LU+bdyKyD9qKA==\\\u0022,\\\u0022iv\\\u0022:\\\u00228Rl7OndA6xrrwGWa5qei2w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221ef3b9c5281c0049a451e14577c61be6d69c3564dc85a5268539fc3b147d0fa4\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459915 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/7/1319" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ZqUgWWfAo9S64BzJqPvaOu5jo_9REihXSoCuuOof3LE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/7/1319" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-B2chd4U5wudNUEckUt7_d8ygveajkDrnFcSutDwdEXE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/7/1307" title="A Roadmap for Functional Structural Variants in the Soybean Genome" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/7/1327" title="Genetic Control of Maize Shoot Apical Meristem Architecture" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459915" id="node459915" data-pisa="ggg;4/7/1319" data-pisa-master="ggg;g3.114.011668" data-apath="/ggg/4/7/1319.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Functional Conservation of the Pre-Sensor One Beta-Finger Hairpin (PS1-hp) Structures in Mini-Chromosome Maintenance Proteins of <em>Saccharomyces cerevisiae</em> and Archaea</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Christopher J. Ramey</span> and <span class="highwire-citation-author" data-delta="1">Robert A. Sclafani</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1319-1326;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011668 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459915" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christopher J. Ramey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz Medical Campus, Aurora, Colorado 80045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BJ.%2BRamey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramey%20CJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristopher%2BJ.%2BRamey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert A. Sclafani </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz Medical Campus, Aurora, Colorado 80045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BA.%2BSclafani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sclafani%20RA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BA.%2BSclafani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Robert.sclafani@ucdenver.edu" class="" data-icon-position="" data-hide-link-title="0">Robert.sclafani@ucdenver.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/7/1319" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459915" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459915/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/7/1319.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459915" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459915/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/7/1319.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459915" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459915/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/7/1319.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459915" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459915/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/7/1319.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459915" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459915/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Mini-chromosome maintenance (MCM) proteins form complexes that are required for DNA replication and are highly conserved throughout evolution. The replicative helicase of eukaryotic organisms is composed of the six paralogs <em>MCM2-7</em>, which form a heterohexameric ring structure. In contrast, the structure of the archaean replicative MCM helicase is a single Mcm protein that forms a homohexameric complex. Atomic structures of archaeal MCMs have identified multiple beta-finger structures in Mcm proteins whose <em>in vivo</em> function is unknown. In the present study, we have investigated the physiological role of the pre-sensor 1 beta<strong>-</strong>hairpin (PS1-hp) beta-fingers of <em>Saccharomyces cerevisiae</em> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">Mcm4p</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">Mcm5p</a> in DNA replication initiation and elongation <em>in vivo</em>. The PS1-hp beta-finger mutant of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">Mcm5p</a> (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-HAT K506A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em>) has a growth defect at both 18° and 37°. Mutation of the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">Mcm4p</a> PS1-hp beta-finger (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">mcm4</a>-HA K658A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a></em>) does not have a growth defect, indicating different functional contributions of the PS1-hp beta-finger structures of different MCM helicase subunits. Both <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">Mcm4p</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">Mcm5p</a> PS1-hp beta-finger mutants can coimmunoprecipitate <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000119">Mcm2p</a>, indicating the formation of the hexameric MCM helicase complex. Both PS1-hp beta-finger mutants have a plasmid loss phenotype that is suppressible by origin dosage, indicating a defective replication initiation. Surprisingly, a defect in the binding of PS1-hp MCM mutants to origins of DNA replication was not found by chromatin immunoprecipitation, suggesting a novel interpretation in which the defect is in a subsequent step of DNA strand separation by the MCM helicase. The double mutant <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">mcm4</a>-HA K658A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-HAT K506A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> is lethal, displaying a terminal MCM mutant phenotype of large budded cells.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/replication" class="hw-term hw-article-keyword hw-article-keyword-replication" rel="nofollow">replication</a></li><li class="kwd"><a href="/keyword/helicase" class="hw-term hw-article-keyword hw-article-keyword-helicase" rel="nofollow">helicase</a></li><li class="kwd"><a href="/keyword/s-phase-0" class="hw-term hw-article-keyword hw-article-keyword-s-phase" rel="nofollow">S phase</a></li><li class="kwd"><a href="/keyword/cell-cycle" class="hw-term hw-article-keyword hw-article-keyword-cell-cycle" rel="nofollow">cell cycle</a></li><li class="kwd"><a href="/keyword/yeast" class="hw-term hw-article-keyword hw-article-keyword-yeast" rel="nofollow">yeast</a></li></ul><p id="p-4">DNA replication in eukaryotes is a tightly controlled process. Failure of DNA replication control mechanisms leads to chromosomal instability, mutations, and aneuploidy, which are hallmarks of human disease, including birth defects, aging, and cancer (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Jackson <em>et al.</em> 2014</a>). The exact duplication of the genome is an essential cellular process. DNA replication control mechanisms ensure that genomic information is replicated exactly once per cell cycle (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Sclafani and Holzen 2007</a>). DNA replication begins at DNA sequences termed “origins.” In higher eukaryotes, origins have no defined DNA sequence; however, in <em>Saccharomyces cerevisiae</em>, origins of DNA replication have a 14-bp consensus site termed the autonomous replicating sequence (ARS) (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Leonard and Mechali 2013</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Sclafani and Holzen 2007</a>). Origins are continually bound throughout the cell cycle by the origin recognition complex (ORC), which is a six-subunit ATP-dependent protein complex encoded in yeast by <em>ORC1-6</em> (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bell and Dutta 2002</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Leonard and Mechali 2013</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Sclafani and Holzen 2007</a>).</p><p id="p-5">A two-step process regulates DNA replication. The first step is termed “licensing of the origin,” characterized by the formation of the prereplicative complex (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Labib 2010</a>). Licensing consists of loading the mini-chromosome maintenance (MCM) helicase during the G1 phase of the cell cycle onto double-stranded DNA (dsDNA) by <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003807">Cdt1p</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003730">Cdc6p</a> in yeast (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Frigola <em>et al.</em> 2013</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Labib 2010</a>). The MCM helicase is composed of six paralogs <em>MCM2-7</em> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Forsburg 2004</a>) and is loaded as a double hexamer (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bell and Botchan 2013</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Frigola <em>et al.</em> 2013</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gambus <em>et al.</em> 2011</a>). Mcm2-7p is thought to be the replicative helicase responsible for DNA stand separation (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Bell and Botchan 2013</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Forsburg 2004</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Tye 1999</a>). The inactive MCM helicase loaded onto the origin of replication along with ORC forms the prereplicative complex. In the second step, the origin of DNA replication is activated in S phase by cyclin-dependent kinase (CDK) and Dbf4-dependent kinase (DDK) (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Sclafani and Holzen 2007</a>).</p><p id="p-6">DDK, which is composed of the catalytic <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002175">Cdc7p</a> and regulatory <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002459">Dbf4p</a> subunits, phosphorylates the N-terminal tails of Mcm 2/4/6. Phosphorylations are thought to induce structural changes that activate the MCM helicase (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hoang <em>et al.</em> 2007</a>; <a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Sclafani and Holzen 2007</a>). It has also been shown that S-phase CDK (S-CDK) phosphorylates the MCM helicase; however, the essential targets of S-CDK are <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001591">Sld2p</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003081">Sld3p</a> (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Randell <em>et al.</em> 2010</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Tanaka <em>et al.</em> 2007</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Zegerman and Diffley 2007</a>) Phosphorylation of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001591">Sld2p</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003081">Sld3p</a> allows for the binding of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003626">Dpb11p</a>, which leads to the stable association of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004093">Cdc45p</a> and the recruitment of the GINs complex and DNA polymerases, activating the origin of DNA replication (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Labib 2010</a>). The formation of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004093">Cdc45p</a>, GINs, and Mcm2-7p complex is known as the CMG complex and is thought to be the active form of the replicative helicase (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Bell and Botchan 2013</a>). The DNA replication origins are activated in a temporal fashion during S phase, with some origins activating (firing) early in S phase, whereas others fire in the middle and later parts of S phase (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">McCune <em>et al.</em> 2008</a>). Temporal control of origin activation is thought to be controlled by DNA replication factor binding mediated by the kinase activities of CDK and DDK.</p><div class="section other1" id="sec-1"><h2 class="">MCM Helicase Structure and Function</h2><p id="p-7">MCM genes are found in both eukaryotes and archaea (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bochman and Schwacha 2009</a>). The eukaryotic MCM helicase is composed of six paralogs <em>MCM 2-7</em> that belong to the highly conserved AAA+ ATPase family (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bochman and Schwacha 2009</a>). Mcm2-7p are thought to have diverged from a common ancestor and form a hexameric ring structure with a 1:1:1:1:1:1 stoichiometry (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brewster <em>et al.</em> 2008</a>). All Mcm2-7 proteins are required for DNA replication and are essential for viability (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Bochman and Schwacha 2009</a>). A single gene encodes the archeal MCMs in <em>Sulfolobus solfataricus</em> (<em>ssoMCM</em>) and <em>Methanothermobacter thermautotrophicus</em> (<em>mtMCM</em>) (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Brewster <em>et al.</em> 2008</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Fletcher <em>et al.</em> 2003</a>). Analysis of the structures of the N-terminal domain of mtMCM protein and the near full-length structure of ssoMCM protein has revealed interesting beta-hairpin structures of each subunit that are located in the central channel or side channels in the MCM hexamer (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Brewster <em>et al.</em> 2008</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Fletcher <em>et al.</em> 2003</a>). Previous work from our laboratory has demonstrated the importance and functional conservation of the N-terminal beta finger in <em>Saccharomyces cerevisiae</em>. The N-terminal beta finger of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">Mcm5p</a> is important for the MCM complex to bind to origins of DNA replication (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Leon <em>et al.</em> 2008</a>). The pre-sensor 1 beta<strong>-</strong>hairpin (PS1-hp) of MCM complexes is highly conserved (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) and a mutation in the PS1-hp of ssoMCMp greatly decreases helicase activity on both a 3′-tailed substrate and a Y-shaped substrate <em>in vitro</em> (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">McGeoch <em>et al.</em> 2005</a>). The PS1-hp is thought to have nucleotide dependent movement that promotes helicase activity (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Bell and Botchan 2013</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Slaymaker and Chen 2012</a>). The PS1-hp is conserved in the distantly related Superfamily 3 helicases, which includes SV40 T antigen, and has been shown to undergo significant movement during nucleotide binding, hydrolysis, and release cycle of the helicase (<a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Bell and Botchan 2013</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gai <em>et al.</em> 2004</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Slaymaker and Chen 2012</a>). In this current study, we present a molecular genetic analysis of mutations of the highly conserved lysine residue of the PS1-hp in <em>Saccharomyces cerevisiae</em> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">Mcm4p</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">Mcm5p</a> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/7/1319/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pre-sensor 1 beta-hairpin structure and growth of mutants. (A) Conservation of the PS1-hp amino acids between Sulfolobus solfataricus Mcm and Saccharomyces cerevisiae Mcm2-7p. (B) Structure prediction of scMcm4p using Phyre software using the ssoMcm structure as a template. Alignment of the ssoMcm(cyan) and scMcm4p (orange) using pymol software, PS1-hp is indicated with an arrow. (C) Structure prediction of scMcm5p using Phyre software using the ssoMcm structure as a template. Alignment of the ssoMcm(cyan) and scMcm5p (magenta) using pymol software, PS1-hp indicated with an arrow. (D) Serial dilution analysis of growth indicates no growth defect of the Mcm4p PS1-hp mutant compared with wild type (Mcm4p-HA). Yeast with the PS1-hp mutation in Mcm5p (mcm5-K506A::URA3) has a cold sensitivity phenotype at 18&#xB0; and a temperature sensitivity phenotype at 37&#xB0; as indicated by reduced growth compared with the wild type (Mcm5p-HA)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-48182889" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Pre-sensor 1 beta-hairpin structure and growth of mutants. (A) Conservation of the PS1-hp amino acids between &lt;em&gt;Sulfolobus solfataricus&lt;/em&gt; Mcm and &lt;em&gt;Saccharomyces cerevisiae&lt;/em&gt; Mcm2-7p. (B) Structure prediction of scMcm4p using Phyre software using the ssoMcm structure as a template. Alignment of the ssoMcm(cyan) and scMcm4p (orange) using pymol software, PS1-hp is indicated with an arrow. (C) Structure prediction of scMcm5p using Phyre software using the ssoMcm structure as a template. Alignment of the ssoMcm(cyan) and scMcm5p (magenta) using pymol software, PS1-hp indicated with an arrow. (D) Serial dilution analysis of growth indicates no growth defect of the Mcm4p PS1-hp mutant compared with wild type (Mcm4p-HA). Yeast with the PS1-hp mutation in Mcm5p (&lt;em&gt;mcm5-K506A&lt;/em&gt;::&lt;em&gt;URA3&lt;/em&gt;) has a cold sensitivity phenotype at 18&#xB0; and a temperature sensitivity phenotype at 37&#xB0; as indicated by reduced growth compared with the wild type (Mcm5p-HA).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/7/1319/F1.medium.gif" width="440" height="364"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/7/1319/F1.medium.gif" width="440" height="364"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/7/1319/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/7/1319/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464588" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child">Pre-sensor 1 beta-hairpin structure and growth of mutants. (A) Conservation of the PS1-hp amino acids between <em>Sulfolobus solfataricus</em> Mcm and <em>Saccharomyces cerevisiae</em> Mcm2-7p. (B) Structure prediction of scMcm4p using Phyre software using the ssoMcm structure as a template. Alignment of the ssoMcm(cyan) and scMcm4p (orange) using pymol software, PS1-hp is indicated with an arrow. (C) Structure prediction of scMcm5p using Phyre software using the ssoMcm structure as a template. Alignment of the ssoMcm(cyan) and scMcm5p (magenta) using pymol software, PS1-hp indicated with an arrow. (D) Serial dilution analysis of growth indicates no growth defect of the Mcm4p PS1-hp mutant compared with wild type (Mcm4p-HA). Yeast with the PS1-hp mutation in Mcm5p (<em>mcm5-K506A</em>::<em>URA3</em>) has a cold sensitivity phenotype at 18° and a temperature sensitivity phenotype at 37° as indicated by reduced growth compared with the wild type (Mcm5p-HA).</p><div class="sb-div caption-clear"></div></div></div></div><div class="section materials-methods" id="sec-2"><h2 class="">Materials and Methods</h2><div id="sec-3" class="subsection"><h3>Yeast strains, media, and plasmids</h3><p id="p-9">Yeast strains are listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and plasmids are listed in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. Yeast strains were grown using standard procedures as well as methods published previously (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Leon <em>et al.</em> 2008</a>). To produce integrating plasmids carrying <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">mcm4</a>-HA K506A</em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-HA K506A</em>, Quickchange mutagenesis (Stratagene) was used on plasmid pRS663 (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">MCM4</a>-HA</em>) and pRPL104 (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">MCM5</a>-HAT</em>) and mutations were marked by the addition of restriction sites <em>Pst</em>I (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">MCM4</a>-HA</em>) and <em>Pvu</em>II (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">MCM5</a>-HAT</em>). All mutations were verified by polymerase chain reaction (PCR), restriction digest, and by sequencing. Yeast strains were constructed using a plasmid shuffle strategy using a covering wild-type (WT) gene on a plasmid. Plasmids carrying either <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">MCM4</a>-HA</em> (pRS663) or <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">mcm4</a>-HA K658A</em> (pCJR101) were digested with <em>Nru</em>I and integrated at the endogenous <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">MCM4</a></em> locus, creating a partial duplication with the active copy containing a hemagglutinin (HA) tag or HA tag and the PS1-hp mutation. <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">MCM5</a>-HA</em> or <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-HA K506A</em> was constructed using pRPL104 (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">MCM5</a>-HAT</em>) or pRS734 (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-K506A-HAT</em>) digested with PpuMI to integrate at the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>-1 locus. <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">MCM5</a></em> at the endogenous locus was deleted with <em>KanMX4</em> and covered with a WT <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">MCM5</a></em> plasmid (pRS660) that was shuffled with the tagged version and the tagged PS1-hp mutant version to test for viability.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464688/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464688/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title"><em>S. cerevisiae</em> strains used</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464507/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464507/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Recombinant plasmids used</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Plasmid stability assay</h3><p id="p-10">To generate the strains used for plasmid loss assays, CRY106, CRY107, CRY213, and CRY214 were transformed with either pDK243 <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">LEU2</a></em> (1X-ARS site) or pDK368-7 <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">LEU2</a></em> (8X-ARS sites), and Leu<sup>+</sup> transformants were selected generating strains. WT and PS1-hp mutant strains containing either pDK243 <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">LEU2</a></em> (1X-ARS site) or pDK368-7 (8X-ARS sites) with a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">LEU2</a></em> marker were grown in synthetic complete media–leucine to stationary phase, then plated onto YEPD and synthetic complete media–leucine to determine plasmid stability under selection. Stationary cultures were then diluted to 2 × 10<sup>3</sup> cells/mL and grown in synthetic complete media + leucine (relaxed conditions) and allowed to grow to stationary phase. Stationary cultures were then plated onto yeast extract peptone dextrose and synthetic complete media – leucine to determine plasmid stability under relaxed selection. The plasmid loss rate was determined by the observed loss of the plasmid on synthetic complete–leucine plates compared with yeast extract peptone dextrose plates under relaxed selection as described previously (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Hogan and Koshland 1992</a>; <a id="xref-ref-20-11" class="xref-bibr" href="#ref-20">Leon <em>et al.</em> 2008</a>). At least three independent loss rates were calculated for each condition.</p><div id="sec-5" class="subsection"><h4>Co-immunoprecipitation (IP) and western blot analysis:</h4><p id="p-11">Protein extracts were prepared as described previously (<a id="xref-ref-20-12" class="xref-bibr" href="#ref-20">Leon <em>et al.</em> 2008</a>). In summary, 2 mg of protein extract were incubated with 30 μL of protein G-Sepharose beads (Sigma-Aldrich) blocked in lysis buffer supplemented with complete protease inhibitor tablets from Roche (cat. no. 05892970001) for 1 hr at 4°. IP was performed using 50 μg of anti-HA antibody (Roche). Negative controls were performed without antibody, containing only protein G-sepharose beads, and all reactions were incubated with end-over-end rotation at 4° for 3 hr. Samples were then centrifuged at 240<em>g</em> at 4° and the supernatant was collected and mixed with 5X sodium dodecyl sulfate (SDS) buffer to a final of 1X and boiled for 5 min. Pellets were washed three times in 500 μL of lysis buffer containing complete protease inhibitors tablets from Roche (cat. no. 05892970001) and were then resuspended in 50 μL of 2.5X SDS buffer. Western blot analysis used 10% of each fraction, whole-cell extract, supernatant, and pellet. Samples of IPs were resolved on a 7% SDS-polyacrylamide gel electrophoresis gel, transferred to a nitrocellulose membrane, and probed with anti-HA antibody (Roche) at a 1:2500 dilution, anti-Mcm2p antibody (Santa Cruz) at a 1:1000 dilution, and anti-Cdt1p (UM185) at a 1:15000 dilution. For HA blots, a secondary antimouse antibody conjugated to horseradish peroxidase (HRP; Jackson ImmunoResearch) was used at 1:3000 dilution. For <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000119">Mcm2p</a> blots, a secondary antigoat antibody conjugated to HRP (Santa Cruz Biotechnology) was used at a 1:3000 dilution. For <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003807">Cdt1p</a> blots, a secondary antirabbit antibody conjugated to HRP (Bio-Rad) was used at a 1:3000 dilution. Immunoblots were visualized on film or using a ChemiDoc MP scanner (Bio-Rad) employing an ECL chemiluminescence kit (Millipore).</p></div><div id="sec-6" class="subsection"><h4>Chromatin immunoprecipitation (ChIP) analysis:</h4><p id="p-12">ChIP analysis was performed as previously described (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adkins <em>et al.</em> 2004</a>) with the following modifications. Forty-five milliliters of alpha factor G1 arrested yeast cultures (2 × 10<sup>7</sup> cells/mL) were fixed with formaldehyde for 1 hr at room temperature. Cells were lysed using a mini bead buster (Biospec) for 3X 2-min intervals in ChIP lysis buffer 1 [50 mM HEPES, pH 7.5; 140 mM NaCl; 1 mM ethylenediaminetetraacetic acid (EDTA); 1% Triton X-100; 0.1% deoxychloic acid; complete protease inhibitor tablet (Roche cat nol. 05892970001)]. Cell lysis was sonicated using a Bioruptor (Diagenode) for 30 cycles of 30 sec on/off at 4°. Cell debris was removed with two 20,000<em>g</em> spins for 5 min. The chromatin fraction was precleared with the addition of 10 μL of Protein G sepharose beads (Sigma-Aldrich) with end-over-end rotation for 1 hr followed by centrifugation at 880<em>g</em> for 1 min. Then, 50 μL of cleared lysate was taken as an input sample with the addition of 30 μL of ChIP elution buffer (50 mM Tris-HCl, pH 7.5; 10mM EDTA; 1% SDS). IPs were set up with 350 μL of chromatin solution per IP reaction. For HA IPs, the chromatin fraction was incubated with 25 μg of ant-HA antibody (Roche). All IPs were incubated for 15 hr at 4°. Protein G sepharose beads (30 μL) were added to each reaction for 1 hr at 4°. Beads were washed 2X with ChIP lysis buffer 1 (50 mM HEPES, pH 7.5; 140mM NaCl; 1 mM EDTA; 1% Triton X-100; 0.1% deoxycholic acid), twice with lysis buffer 2 (50 mM HEPES, pH 7.5; 500 mM NaCl; 1 mM EDTA; 1% Triton X-100; 0.1% deoxycholic acid), twice with lysis buffer 3 (10 mM Tris, pH 7.9; 250 mM LiCl; 1 mM EDTA; 0.5% NP-50; 0.5% deoxycholic acid), and once with TE. Beads were resuspended in 80 μL of ChIP elution buffer with 20 μL of pronase at 20 mg/mL (Calbiochem), incubated at 42° for 2 hr, followed by 8 hr at 65° to reverse the crosslinks. Chromatin-associated DNA isolated by ChIP was subjected to quantitative PCR, using 2X Sybergreen master mix from (Roche) and a Roche Lightcycler 480 thermocycler. Chromatin associated DNA was subjected to PCR analysis with primers to the following DNA replication origins: <em>ARS1</em>, <em>ARS305</em>, <em>ARS306</em>, <em>ARS603</em>, <em>ARS607</em>, and <em>ARS1412</em>.</p></div></div><div id="sec-7" class="subsection"><h3>Determining whether Mcm4p Mcm5p PS1-hp double mutant requires a WT Mcm5p encoding plasmid for viability</h3><p id="p-13">Yeast strain CRY214 (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-HA K506A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em>) was transformed with pRS660 (WT <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">MCM5</a></em>). Next, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">mcm4</a>-HA K658A</em>::<em>TRP</em> was created by integration and partial duplication at the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">MCM4</a></em> locus. To test whether the viability of the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">Mcm4p</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">Mcm5p</a> PS1-hp double mutants requires the WT <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">Mcm5p</a> encoding plasmid, the loss rate of the WT <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">MCM5</a></em> plasmid under relaxed conditions was compared between the double PS1-hp mutant and the single mutant <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-HA K658A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em>.</p><div id="sec-8" class="subsection"><h4>Lethality of the Mcm4p Mcm5p PS1-hp double mutant by tetrad analysis:</h4><p id="p-14">CRY119 (<em>MATα <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">mcm4</a>-HA K658A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a></em>) was mated to RSY1345 (<em>MAT</em><strong>a</strong> <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-HA K506A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> integrated at <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>-1 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a></em>::<em>KanMX4</em>) (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Diploids were selected on synthetic complete –TRP –URA plates and sporulated in 3% potassium acetate. Ninety-one tetrads were dissected, and spores were germinated on YPD plates at 30°. Spores that were Ura<sup>+</sup> G418R were tested for the Trp<sup>+</sup> phenotype, indicating double mutant (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">mcm4</a>-HA K658A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-HA K658A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a></em>::<em>KanMX4</em>) viability. The terminal phenotype was determined by photographing spores know to be a double PS1-hp mutants.</p></div></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>PS1-hp mutations of <em>Mcm4</em> and <em>Mcm5</em> are viable</h3><p id="p-15">The PS1-hp beta-finger structure is conserved in both <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">Mcm4</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">Mcm5</a> proteins of yeast by both primary sequence and structural alignment (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>, A−C). Importantly, a conserved lysine is present at the tip of the beta-finger in both proteins. Yeast strains with integrated copies of the lysine (K) to alanine (A) PS1-hp mutations in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">Mcm5p</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">Mcm4p</a> were constructed to produce <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">mcm4</a> K658A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a> K506A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> strains CRY107 and RSY1345, respectively, as described in the <em>Materials and Methods</em>. These strains have a PS1-hp mutant <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">mcm4</a></em> or <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a></em> gene as the only copy of the gene in the cell, under the control of its native promoter. Because both temperature-sensitive and cold-sensitive <em>mcm</em> mutants have been described previously (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Tye 1999</a>), we tested for growth at a range of temperatures. Ten-fold serial-dilution analysis of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a> K506A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> strain indicates both a temperature-sensitive phenotype and a cold-sensitive phenotype, with reduced growth compared with the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">MCM4</a> or <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">MCM5</a></em> wt strain at 37° and 18° (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1D</a>). In contrast, the corresponding PS1-hp mutation on <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">Mcm4p</a> (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">mcm4</a> K658A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a></em>) does not display a temperature-sensitive or cold-sensitive phenotype (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1D</a>). To test for sensitivity of PS1-hp mutants to replication stress, 10-fold serial-dilution analysis of growth was performed on plates containing different amounts of hydroxyurea, but no hydroxyurea sensitivity was found (Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011668/-/DC1/FigureS2.pdf">Figure S2</a>) indicating that the checkpoint is intact in PS1-hp mutants. Introduction of the PS1-hp mutation into <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">Mcm4p</a> or <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">Mcm5p</a> does not have an effect on protein stability and both mutant proteins are expressed from their own promoters (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011668/-/DC1/FigureS1.pdf">Figure S1</a>).</p></div><div id="sec-11" class="subsection"><h3>The Mcm4p and Mcm5p PS1-hp mutant proteins are incorporated into the Mcm2-7p complex</h3><p id="p-16">To demonstrate that the PS1-hp mutants get incorporated into the Mcm2-7p hexamer, both WT and PS1-hp mutants of both <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">Mcm4p</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">Mcm5p</a> were tagged with an HA epitope. An anti-HA antibody was used to precipitate each of the following: Mcm4p-HA, mcm4p-HA K658A, Mcm5p-HA, and mcm5p-HA K658A. Immunoprecipitates were run on 7.5% SDS-polyacrylamide gel electrophoresis gels, transferred to nitrocellulose, and probed for <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000119">Mcm2p</a> using an anti-Mcm2p antibody to demonstrate Mcm2-7p complex formation. <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000119">Mcm2p</a> binds directly to <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">Mcm5p</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003169">Mcm6p</a> and only binds indirectly with <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">Mcm4p</a> in the hexameric MCM ring. Immunoprecipitates from both Mcm4p-HA and mcm4p-HA K658A strains contain similar amounts of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000119">Mcm2p</a>, indicating that the PS1-hp of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">Mcm4p</a> forms the hexameric MCM complex <em>in vivo</em> (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). Similar to <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">Mcm4p</a>, both Mcm5p-HA and mcm5p-HA K506A immunoprecipitates contain equal amounts of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000119">Mcm2p</a>, demonstrating <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">Mcm5p</a> binds <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000119">Mcm2p</a> in the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">Mcm5p</a> PS1-hp mutant (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>.). In addition to <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000119">Mcm2p</a>, mcm5p-HA K506A co-immunoprecipitates with <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003807">Cdt1p</a> at WT levels (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>), indicating that <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">Mcm5p</a> PS1-hp can interact with <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003807">Cdt1p</a>, which is important for MCM helicase loading at origins of DNA replication (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Tanaka and Diffley 2002</a>). Because <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003807">Cdt1p</a> is known to interact with the hexameric MCM helicase, these data indicate <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">Mcm5p</a> PS1-hp mutant can form the MCM complex. These results are consistent with results found <em>in vitro</em> when analyzing the MCM complex formation of the PS1-hp mutants of SsoMCM (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">McGeoch <em>et al.</em> 2005</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/7/1319/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Co-immunoprecipitation of the Mcm4p and Mcm5p PS1-hp mutants. (A) Immunoprecipitations of Mcm4p-HA, mcm4p-HA K658A, Mcm5p-HA, and mcm5p-HA K506A were performed using an anti-HA antibody or a no antibody negative control. Western blots were probed with anti-HA antibody for Mcm5p, an anti-Mcm2p antibody, and an anti-Ctd1p antibody. Supernatant and pellet fractions are represented by &#x201C;S&#x201D; and &#x201C;P,&#x201D; respectively. HA, hemagglutinin." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-48182889" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Co-immunoprecipitation of the Mcm4p and Mcm5p PS1-hp mutants. (A) Immunoprecipitations of Mcm4p-HA, mcm4p-HA K658A, Mcm5p-HA, and mcm5p-HA K506A were performed using an anti-HA antibody or a no antibody negative control. Western blots were probed with anti-HA antibody for Mcm5p, an anti-Mcm2p antibody, and an anti-Ctd1p antibody. Supernatant and pellet fractions are represented by &#x201C;S&#x201D; and &#x201C;P,&#x201D; respectively. HA, hemagglutinin.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/7/1319/F2.medium.gif" width="322" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/7/1319/F2.medium.gif" width="322" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/7/1319/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/7/1319/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464505" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-17" class="first-child">Co-immunoprecipitation of the Mcm4p and Mcm5p PS1-hp mutants. (A) Immunoprecipitations of Mcm4p-HA, mcm4p-HA K658A, Mcm5p-HA, and mcm5p-HA K506A were performed using an anti-HA antibody or a no antibody negative control. Western blots were probed with anti-HA antibody for Mcm5p, an anti-Mcm2p antibody, and an anti-Ctd1p antibody. Supernatant and pellet fractions are represented by “S” and “P,” respectively. HA, hemagglutinin.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Plasmid loss phenotype of Mcm4p and Mcm5p PS1-hp mutants</h3><p id="p-18">Yeast strains containing the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">mcm4</a>-K658A</em> PS1-hp mutation have a plasmid loss rate per generation of 4.1%, which is threefold higher than the loss rate of 1.2% per generation in wt cells (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). To test whether the plasmid loss was due to initiation of DNA replication or elongation, we examined the plasmid loss rate with a plasmid carrying eight origins (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Hogan and Koshland 1992</a>). The plasmid loss rate was decreased 2.5-fold in a Mcm4p-PS1-hp mutant compared to a 3-fold reduction in wt with the addition of seven ARS elements, consistent with a defect in replication initiation (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>). This result was unexpected and inconsistent with the published elongation defect seen for the ssoMCM PS1-hp mutant <em>in vitro</em> (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">McGeoch <em>et al.</em> 2005</a>). Next, we examined the plasmid loss rate of the PS1-hp mutant of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">Mcm5p</a>. The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-HA K506A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> PS1-hp mutant has an increased plasmid loss rate of 21.4%, which is 8-fold greater than the loss rate of 2.6% in wt cells. With the addition of seven origins, the loss rate was reduced about 30-fold for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-k506A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> compared with 3.5-fold in wt cells(<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). Thus, both <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">mcm4</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a></em> PS1-hp mutants had an increase in plasmid loss with a clear MCM phenotype that was partially suppressible by increased origin dosage. To further investigate this phenotype, we used ChIP analysis to determine Mcm2-7p helicase binding to chromatin at cellular origins of DNA replication.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/7/1319/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plasmid stability of Mcm4p and Mcm5p PS1-hp mutants. (A) Wild type or mcm4-HA K658A::TRP1 or (B) wild type or mcm5-HAT K506A::URA3 yeast strains were transformed with either pDK243 (1 origin of replication) or pDK368-7 (eight origins of replication), and plasmid stability was analyzed. Plasmid loss rates are shown in % loss per generation. The increase in stability with the addition of seven origins of replication is presented as fold increase in stability compared with one origin of replication." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-48182889" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plasmid stability of Mcm4p and Mcm5p PS1-hp mutants. (A) Wild type or &lt;em&gt;mcm4-HA K658A&lt;/em&gt;::&lt;em&gt;TRP1&lt;/em&gt; or (B) wild type or &lt;em&gt;mcm5-HAT K506A&lt;/em&gt;::&lt;em&gt;URA3&lt;/em&gt; yeast strains were transformed with either pDK243 (1 origin of replication) or pDK368-7 (eight origins of replication), and plasmid stability was analyzed. Plasmid loss rates are shown in % loss per generation. The increase in stability with the addition of seven origins of replication is presented as fold increase in stability compared with one origin of replication.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/7/1319/F3.medium.gif" width="359" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/7/1319/F3.medium.gif" width="359" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/7/1319/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/7/1319/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464530" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-19" class="first-child">Plasmid stability of Mcm4p and Mcm5p PS1-hp mutants. (A) Wild type or <em>mcm4-HA K658A</em>::<em>TRP1</em> or (B) wild type or <em>mcm5-HAT K506A</em>::<em>URA3</em> yeast strains were transformed with either pDK243 (1 origin of replication) or pDK368-7 (eight origins of replication), and plasmid stability was analyzed. Plasmid loss rates are shown in % loss per generation. The increase in stability with the addition of seven origins of replication is presented as fold increase in stability compared with one origin of replication.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">We used cells arrested in G1 phase of the cell cycle where Mcm2-7 protein binding to origin chromatin is maximal (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Aparicio <em>et al.</em> 1999</a>). Different origins were examined using ChIP to gain a broad understanding of the affects of the PS1-hp mutation in MCM complex binding to origins of DNA replication. We found no significant defect in origin binding by ChIP of MCM complexes containing the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">Mcm4p</a> PS1-hp mutation at all 5 origins examined (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>), which includes early (<em>ARS305</em>, <em>ARS607</em>), mid (<em>ARS1 = ARS416</em>), and late (<em>ARS603</em>, <em>1412</em>) origins (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Raghuraman <em>et al.</em> 2001</a>). Analysis of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">Mcm5p</a> PS1-hp mutants binding to four origins of DNA replication also showed no binding defects detectable by ChIP analysis in G1 arrested cultures (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/7/1319/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Chromatin immunoprecipitation (ChIP) analysis of Mcm4p and Mcm5p PS1-hp mutant proteins at origins of replication. (A) ChIP analysis of wild-type (Mcm4-HA) and PS1-hp mutant of Mcm4p (mcm4-K658A-HA) from G1-arrested cultures. Log-phase cultures were arrested in G1 with the addition of alpha factor for 3 hr and an anti-HA antibody was used for ChIP. Chromatin associated DNA was analyzed using quantitative polymerase chain reaction using primers specific to origins of DNA replication. (B) Similar ChIP analysis of wild type (Mcm5-HA) and PS1-hp mutant Mcm5p (mcm5-K506A-HA) strains." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-48182889" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Chromatin immunoprecipitation (ChIP) analysis of Mcm4p and Mcm5p PS1-hp mutant proteins at origins of replication. (A) ChIP analysis of wild-type (&lt;em&gt;Mcm4-HA&lt;/em&gt;) and PS1-hp mutant of Mcm4p (&lt;em&gt;mcm4-K658A-HA&lt;/em&gt;) from G1-arrested cultures. Log-phase cultures were arrested in G1 with the addition of alpha factor for 3 hr and an anti-HA antibody was used for ChIP. Chromatin associated DNA was analyzed using quantitative polymerase chain reaction using primers specific to origins of DNA replication. (B) Similar ChIP analysis of wild type (&lt;em&gt;Mcm5-HA&lt;/em&gt;) and PS1-hp mutant Mcm5p (&lt;em&gt;mcm5-K506A-HA&lt;/em&gt;) strains.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/7/1319/F4.medium.gif" width="440" height="169"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/7/1319/F4.medium.gif" width="440" height="169"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/7/1319/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/7/1319/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464628" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-21" class="first-child">Chromatin immunoprecipitation (ChIP) analysis of Mcm4p and Mcm5p PS1-hp mutant proteins at origins of replication. (A) ChIP analysis of wild-type (<em>Mcm4-HA</em>) and PS1-hp mutant of Mcm4p (<em>mcm4-K658A-HA</em>) from G1-arrested cultures. Log-phase cultures were arrested in G1 with the addition of alpha factor for 3 hr and an anti-HA antibody was used for ChIP. Chromatin associated DNA was analyzed using quantitative polymerase chain reaction using primers specific to origins of DNA replication. (B) Similar ChIP analysis of wild type (<em>Mcm5-HA</em>) and PS1-hp mutant Mcm5p (<em>mcm5-K506A-HA</em>) strains.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">To gain further insight into the plasmid loss defect of PS1-hp mutants, a conditional <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">Mcm4p</a> PS1-hp and a <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">Mcm5p</a> PS1-hp double mutant was constructed in a background containing the temperature-sensitive allele <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-461</em> (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dalton and Hopwood 1997</a>; <a id="xref-ref-20-13" class="xref-bibr" href="#ref-20">Leon <em>et al.</em> 2008</a>). At 37° the Mcm5-461 protein is unable to integrate into the Mcm2-7p complex (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Dalton and Hopwood 1997</a>; <a id="xref-ref-20-14" class="xref-bibr" href="#ref-20">Leon <em>et al.</em> 2008</a>), creating a double mcm4p mcm5p PS1-hp mutant cell. The following strains were used to observe S-phase progression by flow cytometry of the PS1-hp double mutant: RSY1148 (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-461</em>) and CRY209 (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-461 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-HAT K506A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">mcm4</a>-HA K658A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a></em>). Yeast strains were arrested in alpha factor for 3 hr at 22° then raised to 37° in alpha factor for 1 hr, then released into S-phase at 37° by the addition of pronase. The resolution of this flow cytometry experimental approach was unable to determine if PS1-hp double mutants progressed more slowly through S-phase when compared to either single PS1-hp mutant alone (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011668/-/DC1/FigureS3.pdf">Figure S3</a>). The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-461</em> temperature sensitive mutant is leaky because eventually cells progress through the cell cycle and will form colonies at 37° after several days (<a id="xref-ref-20-15" class="xref-bibr" href="#ref-20">Leon <em>et al.</em> 2008</a>). Therefore, we produced double <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">mcm4</a>-HA K658A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-HAT K506A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> mutants using either plasmid shuffling or yeast meiotic recombination techniques.</p></div><div id="sec-13" class="subsection"><h3>Mutation of the PS1-hp in both mcm4 and mcm5 is lethal</h3><p id="p-23">We tested the viability of a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">mcm4</a>-HA K658A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-HA K506A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> double mutant in two different ways. Yeast strains with an integrated <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-HA K506A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> mutation and a WT 2μ <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">LEU2</a> <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">MCM5</a></em> plasmid can easily lose the plasmid in that &gt;99% of the cells had lost the plasmid (n = 1836 colonies) after growth in nonselective media for &gt;10 doublings. In contrast, the double <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">mcm4</a>-HA K658A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-HA K506A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> mutant maintained the plasmid 100% of the time in that 0% of the cells (n = 2039 colonies; <em>P</em> &lt; 0.0025) had lost the plasmid under similar conditions, indicating a requirement of the plasmid <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">MCM5</a></em> gene for viability.</p><p id="p-24">Because the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">MCM5</a></em> gene was on plasmid pRAS690, it is possible that only the plasmid with a single ARS is unable to replicate as was found for the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-bob1</em> mutation in the absence of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002175">Cdc7p</a> function (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Hoang <em>et al.</em> 2007</a>). This creates an artificial situation in which the cells do not divide only because the essential plasmid, but not the genome, is unable to replicate. To confirm the synthetic lethality of the two PS1-hp mutants when both are integrated in the genome, CRY119 (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">mcm4</a>-HA K658A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a></em> integrated as a partial duplication at the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">mcm4</a></em> locus) was mated to RSY1345 (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-HA K658A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a> at the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>-1 locus</em>) that also had a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a></em>::<em>KanMX4</em> deletion to form diploids, which were then sporulated. Tetrads were dissected and spores that were Ura<sup>+</sup> G418R, which contain both the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a></em>::<em>KanMX4 and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-HA K658A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> mutations, were tested for the presence of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">mcm4</a>-HA K658A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a></em> allele. As <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">mcm4</a></em> are unlinked, we expected to find 50% Trp<sup>+</sup> colonies that carry the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">mcm4</a></em> PS1-hp mutation. With <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-HA K658A</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em>, no Trp<sup>+</sup> colonies were found (n = 44, <em>P</em> &lt; 0.005) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011668/-/DC1/FigureS4.pdf">Figure S4</a>A). Thus, the double <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">mcm4</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a></em> PS1-hp mutant is inviable in cells with the normal complement of cellular origins. The terminal phenotype of the germinated spore of a double <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">mcm4</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a></em> PS1-hp mutant is two large budded cells indicative of a helicase defect as originally described for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000119">mcm2</a>∆</em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000758">mcm3</a>∆</em> mutants (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011668/-/DC1/FigureS4.pdf">Figure S4</a>B)(<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Yan <em>et al.</em> 1991</a>).</p></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><p id="p-25">Using the archaeal structure of MCM as a model, we have confirmed the role of the SsoMCM PS1-hp structure that had been examined in <em>in vitro</em> experiments (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">McGeoch <em>et al.</em> 2005</a>) as also being important for downstream events after binding of DNA origins <em>in vivo</em> using the model system <em>Saccharomyces cerevisiae</em>. This is further <em>in vivo</em> evidence that structural models of the homohexameric structures of archaeal MCMs are a valuable tool in understanding the function of eukaryotic heterohexameric MCM complexes (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Brewster <em>et al.</em> 2008</a>; <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Fletcher <em>et al.</em> 2003</a>; <a id="xref-ref-20-16" class="xref-bibr" href="#ref-20">Leon <em>et al.</em> 2008</a>). Our results demonstrate that <em>in vivo</em> yeast PS1-hp mutants can form the heterohexameric MCM complex and bind origins of DNA replication, but fail to maintain plasmid stability, indicating a defect in the transition to an active MCM complex, or an MCM complex that is defective in efficient strand separation during replication elongation.</p><p id="p-26">The MCM helicase has been clearly shown to load onto dsDNA in the G1 phase of the cell cycle (<a id="xref-ref-20-17" class="xref-bibr" href="#ref-20">Leon <em>et al.</em> 2008</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Remus <em>et al.</em> 2009</a>). Our laboratory has previously shown the importance of N-terminal beta-fingers for the MCM complex to bind dsDNA (<a id="xref-ref-20-18" class="xref-bibr" href="#ref-20">Leon <em>et al.</em> 2008</a>). The circular MCM complex is loaded onto dsDNA using a break in the MCM ring between <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">Mcm5p</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000119">Mcm2p</a> (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bochman and Schwacha 2007</a>) by the actions of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003807">Ctd1</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003730">Cdc6p</a> (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Randell <em>et al.</em> 2006</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Remus <em>et al.</em> 2009</a>). One model of DNA strand separation by the MCM helicase is that of strand exclusion, in which the leading strand DNA remains in the central cavity of the complex bound to the β-hairpins, and the lagging strand is extruded and bound to the PS1 hairpin (<a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">Bell and Botchan 2013</a>; <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Bochman and Schwacha 2009</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Fu <em>et al.</em> 2011</a>; <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Slaymaker and Chen 2012</a>). One possible defect of Mcm PS1-hp mutants is the inability to undergo the transformation of binding dsDNA in the inactive form to excluding one strand of DNA in the active form, which would be consistent with our plasmid loss and ChIP results. During the review of this paper, we became aware of a new publication that demonstrates a PS1-hp mutation in yeast <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000758">Mcm3p</a> is lethal, although the MCM complex with the mutant <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000758">Mcm3p</a> is properly formed as interpreted from gel filtration experiments (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Lam <em>et al.</em> 2013</a>)</p><p id="p-27">We found that MCM helicase PS1-hp mutants bind origins of DNA replication at normal occupancy levels and propose PS1-hp mutants are defective in a transition step, which is recovered by the addition of multiple origins by mass action. In plasmid loss experiments, only mutants in initiation such as in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003730">cdc6</a></em> (<a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">Hogan and Koshland 1992</a>) or <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000264">orc2</a></em> (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Loo <em>et al.</em> 1995</a>) are suppressed by increased origin gene dosage. Similarly, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a></em> mutants in the N-terminal β-hairpins have a similar phenotype and displayed reduced binding to origins by ChIP (<a id="xref-ref-20-19" class="xref-bibr" href="#ref-20">Leon <em>et al.</em> 2008</a>). In contrast, plasmid loss rates are not suppressed in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005046">pol1</a></em> mutants, which are defective in elongation by DNA polymerase α (<a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">Hogan and Koshland 1992</a>). We propose that the Mcm PS1-hp mutant proteins can bind to origins normally but have increased dissociation due to a reduced ability to bind the extruded lagging strand. In support of this idea, the DNA helicase activity and the single-stranded DNA (ssDNA) binding stability of recombinant MCM helicases in an electromobility shift assay <em>in vitro</em> was lower for MCM2-7 complexes containing a PS1-hp mutation in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000758">Mcm3p</a>, possibly because the ssDNA-MCM complexes formed are unstable (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Lam <em>et al.</em> 2013</a>)</p><p id="p-28">In this scenario, this dissociation defect is suppressed by having more Mcm2-7p complexes at more origins with only one active helicase still bound to complete replication. Our study suggests a different interpretation of the plasmid stability assay when considering mutants of the Mcm helicase or of other proteins that load in G1 phase and are activated in S phase. This leads to a new understanding of the plasmid stability assay, such that multiple origins not only recover binding defects but also defects in transition from an inactive helicase form to an active replicative form.</p><p id="p-29">We found that the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">mcm4</a></em> PS1-hp mutants behave differently in that the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a></em> mutant was more deleterious with a higher plasmid loss defect (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>) and displays both temperature-sensitive and cold-sensitive conditional phenotypes (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>). This is similar to our previous analysis with the β-hairpin mutants (<a id="xref-ref-20-20" class="xref-bibr" href="#ref-20">Leon <em>et al.</em> 2008</a>). Perhaps, the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">Mcm5</a> protein is more important in stabilizing the channel in ssDNA strand exclusion.</p><p id="p-30">Also similar to our previous studies (<a id="xref-ref-20-21" class="xref-bibr" href="#ref-20">Leon <em>et al.</em> 2008</a>), lethality required mutations in two different <em>MCM</em> genes, which we propose reflects the fact that the six PS1-hp structures in the heterohexamer must cooperate for function and do not act independently, as also has been found for mutations in the ATPase site (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Schwacha and Bell 2001</a>). It is also possible that lethality results because the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">Mcm4</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">Mcm5</a> proteins lie opposite each other in the hexameric ring (<a id="xref-ref-3-7" class="xref-bibr" href="#ref-3">Bell and Botchan 2013</a>). In the recent study cited previously (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Lam <em>et al.</em> 2013</a>) all combinations of the other MCM helicase PS1-hp mutations (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000119">MCM2</a></em>, <em>4</em>, <em>5</em>, <em>6</em>, and <em>7</em>) were lethal except for the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">mcm4</a>-K568A</em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-K506A</em> combination, which had a slow growth phenotype. The later result contradicts our findings that the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006223">mcm4</a>-K568A</em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-K506A</em> combination is lethal. Furthermore, our <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a>-K506A</em> mutant has a conditional phenotype (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>), while theirs had no phenotype. One possible explanation is we used a different yeast strain background (A364a) compared to them (S288C). Nonetheless, our proposal in which important structures in MCM subunits cooperate and have similar functions is supported by both studies.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-31">We thank Karen Helm of the University of Colorado Comprehensive Cancer Center (P30CA046934) Flow Cytometry Shared Resources. We thank for Marisa Ruehle assisting with the structure-based alignments of yeast and archaeal Mcm proteins. We thank John Diller and Rebecca Ferguson for their critical reading of this manuscript. We also thank Stephanie Williams for the initial characterization of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004264">mcm5</a></em> mutant. This work was supported by a Public Health Service award to RAS (GM35078) from the NIH.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011668/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011668/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-32"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-04-16"><span class="received-label">Received </span>April 16, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-05-21"><span class="accepted-label">Accepted </span>May 21, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Ramey and Sclafani</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.1" data-doi="10.1016/j.molcel.2004.05.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adkins</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howar</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tyler</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Chromatin disassembly mediated by the histone chaperone Asf1 is essential for transcriptional activation of the yeast <em>PHO5</em> and <em>PHO8</em> genes.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">657</span>–<span class="cit-lpage">666</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DAdkins%26rft.auinit1%253DM.%2BW.%26rft.volume%253D14%26rft.issue%253D5%26rft.spage%253D657%26rft.epage%253D666%26rft.atitle%253DChromatin%2Bdisassembly%2Bmediated%2Bby%2Bthe%2Bhistone%2Bchaperone%2BAsf1%2Bis%2Bessential%2Bfor%2Btranscriptional%2Bactivation%2Bof%2Bthe%2Byeast%2BPHO5%2Band%2BPHO8%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2004.05.016%26rft_id%253Dinfo%253Apmid%252F15175160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2004.05.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15175160&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1319.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221915500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-2" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.2" data-doi="10.1073/pnas.96.16.9130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aparicio</span>  <span class="cit-name-given-names">O. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stout</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Differential assembly of Cdc45p and DNA polymerases at early and late origins of DNA replication.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">9130</span>–<span class="cit-lpage">9135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DAparicio%26rft.auinit1%253DO.%2BM.%26rft.volume%253D96%26rft.issue%253D16%26rft.spage%253D9130%26rft.epage%253D9135%26rft.atitle%253DDifferential%2Bassembly%2Bof%2BCdc45p%2Band%2BDNA%2Bpolymerases%2Bat%2Bearly%2Band%2Blate%2Borigins%2Bof%2BDNA%2Breplication%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.16.9130%26rft_id%253Dinfo%253Apmid%252F10430907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTYvMTYvOTEzMCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMzE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.3" data-doi="10.1101/cshperspect.a012807"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botchan</span>  <span class="cit-name-given-names">M. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The Minichromosome Maintenance Replicative Helicase.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Perspect. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">a012807</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarb.%2BPerspect.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a012807%26rft_id%253Dinfo%253Apmid%252F23881943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImNzaHBlcnNwZWN0IjtzOjU6InJlc2lkIjtzOjEyOiI1LzExL2EwMTI4MDciO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTMxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.4" data-doi="10.1146/annurev.biochem.71.110601.135425"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dutta</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">DNA replication in eukaryotic cells.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BBiochem.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.biochem.71.110601.135425%26rft_id%253Dinfo%253Apmid%252F12045100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.biochem.71.110601.135425&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12045100&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1319.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177352600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.5" data-doi="10.1074/jbc.M703824200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bochman</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwacha</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Differences in the single-stranded DNA binding activities of MCM2–7 and MCM467: MCM2 and MCM5 define a slow ATP-dependent step.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">33795</span>–<span class="cit-lpage">33804</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M703824200%26rft_id%253Dinfo%253Apmid%252F17895243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODIvNDYvMzM3OTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTMxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.6" data-doi="10.1128/MMBR.00019-09"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bochman</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwacha</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Mcm complex: unwinding the mechanism of a replicative helicase.</span> <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">652</span>–<span class="cit-lpage">683</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DBochman%26rft.auinit1%253DM.%2BL.%26rft.volume%253D73%26rft.issue%253D4%26rft.spage%253D652%26rft.epage%253D683%26rft.atitle%253DThe%2BMcm%2BComplex%253A%2BUnwinding%2Bthe%2BMechanism%2Bof%2Ba%2BReplicative%2BHelicase%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.00019-09%26rft_id%253Dinfo%253Apmid%252F19946136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI3My80LzY1MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMzE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.7" data-doi="10.1073/pnas.0808037105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brewster</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenleaf</span>  <span class="cit-name-given-names">W. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carazo</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Crystal structure of a near-full-length archaeal MCM: functional insights for an AAA+ hexameric helicase.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">20191</span>–<span class="cit-lpage">20196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0808037105%26rft_id%253Dinfo%253Apmid%252F19073923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzUxLzIwMTkxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzEzMTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dalton</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hopwood</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Characterization of Cdc47p-minichromosome maintenance complexes in <em>Saccharomyces cerevisiae</em>: identification of Cdc45p as a subunit.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">5867</span>–<span class="cit-lpage">5875</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DDalton%26rft.auinit1%253DS.%26rft.volume%253D17%26rft.issue%253D10%26rft.spage%253D5867%26rft.epage%253D5875%26rft.atitle%253DCharacterization%2Bof%2BCdc47p-minichromosome%2Bmaintenance%2Bcomplexes%2Bin%2BSaccharomyces%2Bcerevisiae%253A%2Bidentification%2Bof%2BCdc45p%2Bas%2Ba%2Bsubunit%26rft_id%253Dinfo%253Apmid%252F9315644%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxNy8xMC81ODY3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzEzMTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.9" data-doi="10.1038/nsb893"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fletcher</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leon</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sclafani</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogata</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The structure and function of MCM from archaeal <em>M. Thermoautotrophicum</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">160</span>–<span class="cit-lpage">167</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2Bbiology%26rft.stitle%253DNat%2BStruct%2BBiol%26rft.aulast%253DFletcher%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253D160%26rft.epage%253D167%26rft.atitle%253DThe%2Bstructure%2Band%2Bfunction%2Bof%2BMCM%2Bfrom%2Barchaeal%2BM.%2BThermoautotrophicum.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsb893%26rft_id%253Dinfo%253Apmid%252F12548282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsb893&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12548282&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1319.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181178500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.10" data-doi="10.1128/MMBR.68.1.109-131.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forsburg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Eukaryotic MCM proteins: beyond replication initiation.</span> <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DForsburg%26rft.auinit1%253DS.%2BL.%26rft.volume%253D68%26rft.issue%253D1%26rft.spage%253D109%26rft.epage%253D131%26rft.atitle%253DEukaryotic%2BMCM%2BProteins%253A%2BBeyond%2BReplication%2BInitiation%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.68.1.109-131.2004%26rft_id%253Dinfo%253Apmid%252F15007098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI2OC8xLzEwOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMzE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.11" data-doi="10.1038/nature11920"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frigola</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Remus</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mehanna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diffley</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">ATPase-dependent quality control of DNA replication origin licensing.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">495</span>: <span class="cit-fpage">339</span>–<span class="cit-lpage">343</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D495%26rft.spage%253D339%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11920%26rft_id%253Dinfo%253Apmid%252F23474987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11920&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23474987&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1319.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.12" data-doi="10.1016/j.cell.2011.07.045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yardimci</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">T. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guainazzi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Selective bypass of a lagging strand roadblock by the eukaryotic replicative DNA helicase.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">931</span>–<span class="cit-lpage">941</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DFu%26rft.auinit1%253DY.%2BV.%26rft.volume%253D146%26rft.issue%253D6%26rft.spage%253D931%26rft.epage%253D941%26rft.atitle%253DSelective%2Bbypass%2Bof%2Ba%2Blagging%2Bstrand%2Broadblock%2Bby%2Bthe%2Beukaryotic%2Breplicative%2BDNA%2Bhelicase.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.07.045%26rft_id%253Dinfo%253Apmid%252F21925316%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.07.045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21925316&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1319.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.13" data-doi="10.1016/j.cell.2004.09.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gai</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finkielstein</span>  <span class="cit-name-given-names">C. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Mechanisms of conformational change for a replicative hexameric helicase of SV40 large tumor antigen.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DGai%26rft.auinit1%253DD.%26rft.volume%253D119%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D60%26rft.atitle%253DMechanisms%2Bof%2Bconformational%2Bchange%2Bfor%2Ba%2Breplicative%2Bhexameric%2Bhelicase%2Bof%2BSV40%2Blarge%2Btumor%2Bantigen.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2004.09.017%26rft_id%253Dinfo%253Apmid%252F15454080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2004.09.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15454080&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1319.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224292600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.14" data-doi="10.1074/jbc.M110.199521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gambus</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khoudoli</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blow</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">MCM2–7 form double hexamers at licensed origins in Xenopus egg extract.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">286</span>: <span class="cit-fpage">11855</span>–<span class="cit-lpage">11864</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M110.199521%26rft_id%253Dinfo%253Apmid%252F21282109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODYvMTMvMTE4NTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTMxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.15" data-doi="10.1128/MCB.00997-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoang</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leon</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pessoa-Brandao</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghuraman</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Structural changes in Mcm5 protein bypass Cdc7-Dbf4 function and reduce replication origin efficiency in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">7594</span>–<span class="cit-lpage">7602</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00997-07%26rft_id%253Dinfo%253Apmid%252F17724082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNy8yMS83NTk0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzEzMTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-3" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.16" data-doi="10.1073/pnas.89.7.3098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hogan</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koshland</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Addition of extra origins of replication to a minichromosome suppresses its mitotic loss in <em>cdc6</em> and <em>cdc14</em> mutants of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">3098</span>–<span class="cit-lpage">3102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DHogan%26rft.auinit1%253DE.%26rft.volume%253D89%26rft.issue%253D7%26rft.spage%253D3098%26rft.epage%253D3102%26rft.atitle%253DAddition%2Bof%2Bextra%2Borigins%2Bof%2Breplication%2Bto%2Ba%2Bminichromosome%2Bsuppresses%2Bits%2Bmitotic%2Bloss%2Bin%2Bcdc6%2Band%2Bcdc14%2Bmutants%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.89.7.3098%26rft_id%253Dinfo%253Apmid%252F1557417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI4OS83LzMwOTgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTMxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laskey</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coleman</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Replication proteins and human disease.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Perspect. Biol.</abbr> <span class="cit-vol">6</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.18" data-doi="10.1101/gad.1933010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Labib</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">How do Cdc7 and cyclin-dependent kinases trigger the initiation of chromosome replication in eukaryotic cells?</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1208</span>–<span class="cit-lpage">1219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DLabib%26rft.auinit1%253DK.%26rft.volume%253D24%26rft.issue%253D12%26rft.spage%253D1208%26rft.epage%253D1219%26rft.atitle%253DHow%2Bdo%2BCdc7%2Band%2Bcyclin-dependent%2Bkinases%2Btrigger%2Bthe%2Binitiation%2Bof%2Bchromosome%2Breplication%2Bin%2Beukaryotic%2Bcells%253F%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1933010%26rft_id%253Dinfo%253Apmid%252F20551170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjI0LzEyLzEyMDgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTMxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.19" data-doi="10.1371/journal.pone.0082177"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sing</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shilton</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brandl</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The PS1 hairpin of Mcm3 is essential for viability and for DNA unwinding in vitro.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e82177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D8%26rft.spage%253De82177%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0082177%26rft_id%253Dinfo%253Apmid%252F24349215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0082177&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24349215&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1319.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.20" data-doi="10.1534/genetics.108.088690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leon</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tecklenburg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sclafani</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Functional conservation of beta-hairpin DNA binding domains in the Mcm protein of <em>Methanobacterium thermoautotrophicum</em> and the Mcm5 protein of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1757</span>–<span class="cit-lpage">1768</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.088690%26rft_id%253Dinfo%253Apmid%252F18660534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS80LzE3NTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTMxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.21" data-doi="10.1101/cshperspect.a010116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leonard</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mechali</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">DNA replication origins.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Perspect. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">a010116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarb.%2BPerspect.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a010116%26rft_id%253Dinfo%253Apmid%252F23838439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImNzaHBlcnNwZWN0IjtzOjU6InJlc2lkIjtzOjEyOiI1LzEwL2EwMTAxMTYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTMxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.22" data-doi="10.1091/mbc.6.6.741"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rine</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stillman</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">The origin recognition complex in silencing, cell cycle progression, and DNA replication.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">741</span>–<span class="cit-lpage">756</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DLoo%26rft.auinit1%253DS.%26rft.volume%253D6%26rft.issue%253D6%26rft.spage%253D741%26rft.epage%253D756%26rft.atitle%253DThe%2Borigin%2Brecognition%2Bcomplex%2Bin%2Bsilencing%252C%2Bcell%2Bcycle%2Bprogression%252C%2Band%2BDNA%2Breplication.%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.6.6.741%26rft_id%253Dinfo%253Apmid%252F7579692%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjc6IjYvNi83NDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTMxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.23" data-doi="10.1534/genetics.108.094359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCune</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danielson</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvino</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collingwood</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delrow</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The temporal program of chromosome replication: genomewide replication in <em>clb5Δ Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">1833</span>–<span class="cit-lpage">1847</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.094359%26rft_id%253Dinfo%253Apmid%252F18832352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MC80LzE4MzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTMxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.24" data-doi="10.1038/nsmb974"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McGeoch</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trakselis</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laskey</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Organization of the archaeal MCM complex on DNA and implications for the helicase mechanism.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">756</span>–<span class="cit-lpage">762</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DMcGeoch%26rft.auinit1%253DA.%2BT.%26rft.volume%253D12%26rft.issue%253D9%26rft.spage%253D756%26rft.epage%253D762%26rft.atitle%253DOrganization%2Bof%2Bthe%2Barchaeal%2BMCM%2Bcomplex%2Bon%2BDNA%2Band%2Bimplications%2Bfor%2Bthe%2Bhelicase%2Bmechanism.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb974%26rft_id%253Dinfo%253Apmid%252F16116441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb974&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16116441&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1319.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231691600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.25" data-doi="10.1126/science.294.5540.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raghuraman</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winzeler</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collingwood</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wodicka</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Replication dynamics of the yeast genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">294</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRaghuraman%26rft.auinit1%253DM.%2BK.%26rft.volume%253D294%26rft.issue%253D5540%26rft.spage%253D115%26rft.epage%253D121%26rft.atitle%253DReplication%2BDynamics%2Bof%2Bthe%2BYeast%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.294.5540.115%26rft_id%253Dinfo%253Apmid%252F11588253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyOTQvNTU0MC8xMTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTMxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.26" data-doi="10.1016/j.molcel.2005.11.023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Randell</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Sequential ATP hydrolysis by Cdc6 and ORC directs loading of the Mcm2–7 helicase.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DRandell%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D21%26rft.issue%253D1%26rft.spage%253D29%26rft.epage%253D39%26rft.atitle%253DSequential%2BATP%2Bhydrolysis%2Bby%2BCdc6%2Band%2BORC%2Bdirects%2Bloading%2Bof%2Bthe%2BMcm2-7%2Bhelicase.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2005.11.023%26rft_id%253Dinfo%253Apmid%252F16387651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2005.11.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16387651&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1319.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234764700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.27" data-doi="10.1016/j.molcel.2010.10.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Randell</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Francis</span>  <span class="cit-name-given-names">L. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heller</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mec1 Is one of multiple kinases that prime the Mcm2–7 helicase for phosphorylation by Cdc7.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">353</span>–<span class="cit-lpage">363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DRandell%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D40%26rft.issue%253D3%26rft.spage%253D353%26rft.epage%253D363%26rft.atitle%253DMec1%2Bis%2Bone%2Bof%2Bmultiple%2Bkinases%2Bthat%2Bprime%2Bthe%2BMcm2-7%2Bhelicase%2Bfor%2Bphosphorylation%2Bby%2BCdc7.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2010.10.017%26rft_id%253Dinfo%253Apmid%252F21070963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2010.10.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21070963&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1319.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284556300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.28" data-doi="10.1016/j.cell.2009.10.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Remus</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beuron</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tolun</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffith</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Concerted loading of Mcm2–7 double hexamers around DNA during DNA replication origin licensing.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">139</span>: <span class="cit-fpage">719</span>–<span class="cit-lpage">730</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DRemus%26rft.auinit1%253DD.%26rft.volume%253D139%26rft.issue%253D4%26rft.spage%253D719%26rft.epage%253D730%26rft.atitle%253DConcerted%2Bloading%2Bof%2BMcm2-7%2Bdouble%2Bhexamers%2Baround%2BDNA%2Bduring%2BDNA%2Breplication%2Borigin%2Blicensing.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2009.10.015%26rft_id%253Dinfo%253Apmid%252F19896182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2009.10.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19896182&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1319.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271747200016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.29" data-doi="10.1016/S1097-2765(01)00389-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwacha</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Interactions between two catalytically distinct MCM subgroups are essential for coordinated ATP hydrolysis and DNA replication.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1093</span>–<span class="cit-lpage">1104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DSchwacha%26rft.auinit1%253DA.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253D1093%26rft.epage%253D1104%26rft.atitle%253DInteractions%2Bbetween%2Btwo%2Bcatalytically%2Bdistinct%2BMCM%2Bsubgroups%2Bare%2Bessential%2Bfor%2Bcoordinated%2BATP%2Bhydrolysis%2Band%2BDNA%2Breplication.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252801%252900389-6%26rft_id%253Dinfo%253Apmid%252F11741544%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(01)00389-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11741544&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1319.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172312400018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.30" data-doi="10.1146/annurev.genet.41.110306.130308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sclafani</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holzen</span>  <span class="cit-name-given-names">T. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Cell cycle regulation of DNA replication.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">280</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DSclafani%26rft.auinit1%253DR.%2BA.%26rft.volume%253D41%26rft.spage%253D237%26rft.epage%253D280%26rft.atitle%253DCell%2Bcycle%2Bregulation%2Bof%2BDNA%2Breplication.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.41.110306.130308%26rft_id%253Dinfo%253Apmid%252F17630848%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.41.110306.130308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17630848&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1319.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252359500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.31" data-doi="10.1007/978-94-007-4572-8_6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slaymaker</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">MCM structure and mechanics: what we have learned from archaeal MCM.</span> <abbr class="cit-jnl-abbrev">Subcell. Biochem.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSub-cellular%2Bbiochemistry%26rft.stitle%253DSubcell%2BBiochem%26rft.aulast%253DSlaymaker%26rft.auinit1%253DI.%2BM.%26rft.volume%253D62%26rft.spage%253D89%26rft.epage%253D111%26rft.atitle%253DMCM%2BStructure%2Band%2BMechanics%253A%2BWhat%2BWe%2BHave%2BLearned%2Bfrom%2BArchaeal%2BMCM.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-94-007-4572-8_6%26rft_id%253Dinfo%253Apmid%252F22918582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-94-007-4572-8_6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22918582&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1319.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.32" data-doi="10.1038/ncb757"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diffley</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Interdependent nuclear accumulation of budding yeast Cdt1 and Mcm2–7 during G1 phase.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">198</span>–<span class="cit-lpage">207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.stitle%253DNature%2BCell%2BBiology%26rft.aulast%253DTanaka%26rft.auinit1%253DS.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253D198%26rft.epage%253D207%26rft.atitle%253DInterdependent%2Bnuclear%2Baccumulation%2Bof%2Bbudding%2Byeast%2BCdt1%2Band%2BMcm2-7%2Bduring%2BG1%2Bphase.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb757%26rft_id%253Dinfo%253Apmid%252F11836525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncb757&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11836525&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1319.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174209100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.33" data-doi="10.1038/nature05465"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Umemori</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirai</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muramatsu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kamimura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">CDK-dependent phosphorylation of Sld2 and Sld3 initiates DNA replication in budding yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">445</span>: <span class="cit-fpage">328</span>–<span class="cit-lpage">332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DTanaka%26rft.auinit1%253DS.%26rft.volume%253D445%26rft.issue%253D7125%26rft.spage%253D328%26rft.epage%253D332%26rft.atitle%253DCDK-dependent%2Bphosphorylation%2Bof%2BSld2%2Band%2BSld3%2Binitiates%2BDNA%2Breplication%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05465%26rft_id%253Dinfo%253Apmid%252F17167415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05465&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17167415&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1319.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243504700048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.34" data-doi="10.1146/annurev.biochem.68.1.649"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tye</span>  <span class="cit-name-given-names">B. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">MCM proteins in DNA replication.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">649</span>–<span class="cit-lpage">686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DTye%26rft.auinit1%253DB.%2BK.%26rft.volume%253D68%26rft.issue%253D1%26rft.spage%253D649%26rft.epage%253D686%26rft.atitle%253DMCM%2BPROTEINS%2BIN%2BDNA%2BREPLICATION%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.biochem.68.1.649%26rft_id%253Dinfo%253Apmid%252F10872463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.biochem.68.1.649&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10872463&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1319.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000082693200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.35" data-doi="10.1101/gad.5.6.944"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tye</span>  <span class="cit-name-given-names">B. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Mcm2 and Mcm3, two proteins important for ARS activity, are related in structure and function.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">944</span>–<span class="cit-lpage">957</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DYan%26rft.auinit1%253DH%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253D944%26rft.epage%253D957%26rft.atitle%253DMcm2%2Band%2BMcm3%252C%2Btwo%2Bproteins%2Bimportant%2Bfor%2BARS%2Bactivity%252C%2Bare%2Brelated%2Bin%2Bstructure%2Band%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.5.6.944%26rft_id%253Dinfo%253Apmid%252F2044961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6NzoiNS82Lzk0NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMzE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1319.36" data-doi="10.1038/nature05432"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zegerman</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diffley</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Phosphorylation of Sld2 and Sld3 by cyclin-dependent kinases promotes DNA replication in budding yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">445</span>: <span class="cit-fpage">281</span>–<span class="cit-lpage">285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DZegerman%26rft.auinit1%253DP.%26rft.volume%253D445%26rft.issue%253D7125%26rft.spage%253D281%26rft.epage%253D285%26rft.atitle%253DPhosphorylation%2Bof%2BSld2%2Band%2BSld3%2Bby%2Bcyclin-dependent%2Bkinases%2Bpromotes%2BDNA%2Breplication%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05432%26rft_id%253Dinfo%253Apmid%252F17167417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05432&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17167417&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1319.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243504700037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/7/1319.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/7/1307" title="A Roadmap for Functional Structural Variants in the Soybean Genome" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/7/1327" title="Genetic Control of Maize Shoot Apical Meristem Architecture" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/7"><span>Volume 4 Issue 7, July 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/7.cover-source.jpg?itok=MrsYFroL" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/7/1319" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-e2Tt-xBOnohepQmG_oe3PtYeTS1ljyhSK702l_aAeQ8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/7/1319" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/7/1319" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/7/1319" class="active" data-icon-position="" data-hide-link-title="0">Functional Conservation of the Pre-Sensor One Beta-Finger Hairpin (PS1-hp) Structures in Mini-Chromosome Maintenance Proteins of Saccharomyces cerevisiae and Archaea</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459915" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-bPSOijINHek_Je6xGTsB2VugLz38oJ-EOCnFKZIMiWk" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/7/1319.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/7/1319" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-2_obV5D0gwN9xM0aPfHJsM0BCIcqToaNHX7fGcJ48OM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/7/1319" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/7/1319.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459915" id="node459915--2" data-pisa="ggg;4/7/1319" data-pisa-master="ggg;g3.114.011668" data-apath="/ggg/4/7/1319.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Functional Conservation of the Pre-Sensor One Beta-Finger Hairpin (PS1-hp) Structures in Mini-Chromosome Maintenance Proteins of <em>Saccharomyces cerevisiae</em> and Archaea</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Christopher J. Ramey</span> and <span class="highwire-citation-author" data-delta="1">Robert A. Sclafani</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1319-1326;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011668 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459915--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christopher J. Ramey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz Medical Campus, Aurora, Colorado 80045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BJ.%2BRamey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramey%20CJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristopher%2BJ.%2BRamey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert A. Sclafani </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz Medical Campus, Aurora, Colorado 80045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BA.%2BSclafani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sclafani%20RA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BA.%2BSclafani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Robert.sclafani@ucdenver.edu" class="" data-icon-position="" data-hide-link-title="0">Robert.sclafani@ucdenver.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/7/1319" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/7/1319&amp;title=Functional%20Conservation%20of%20the%20Pre-Sensor%20One%20Beta-Finger%20Hairpin%20%28PS1-hp%29%20Structures%20in%20Mini-Chromosome%20Maintenance%20Proteins%20of%20Saccharomyces%20cerevisiae%20and%20Archaea" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/7/1319&amp;title=Functional%20Conservation%20of%20the%20Pre-Sensor%20One%20Beta-Finger%20Hairpin%20%28PS1-hp%29%20Structures%20in%20Mini-Chromosome%20Maintenance%20Proteins%20of%20Saccharomyces%20cerevisiae%20and%20Archaea" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/7/1319&amp;title=Functional%20Conservation%20of%20the%20Pre-Sensor%20One%20Beta-Finger%20Hairpin%20%28PS1-hp%29%20Structures%20in%20Mini-Chromosome%20Maintenance%20Proteins%20of%20Saccharomyces%20cerevisiae%20and%20Archaea" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/7/1319&amp;text=Functional%20Conservation%20of%20the%20Pre-Sensor%20One%20Beta-Finger%20Hairpin%20%28PS1-hp%29%20Structures%20in%20Mini-Chromosome%20Maintenance%20Proteins%20of%20Saccharomyces%20cerevisiae%20and%20Archaea" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/7/1319&amp;title=Functional%20Conservation%20of%20the%20Pre-Sensor%20One%20Beta-Finger%20Hairpin%20%28PS1-hp%29%20Structures%20in%20Mini-Chromosome%20Maintenance%20Proteins%20of%20Saccharomyces%20cerevisiae%20and%20Archaea" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/7/1319&amp;t=Functional%20Conservation%20of%20the%20Pre-Sensor%20One%20Beta-Finger%20Hairpin%20%28PS1-hp%29%20Structures%20in%20Mini-Chromosome%20Maintenance%20Proteins%20of%20Saccharomyces%20cerevisiae%20and%20Archaea" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/7/1319&amp;title=Functional%20Conservation%20of%20the%20Pre-Sensor%20One%20Beta-Finger%20Hairpin%20%28PS1-hp%29%20Structures%20in%20Mini-Chromosome%20Maintenance%20Proteins%20of%20Saccharomyces%20cerevisiae%20and%20Archaea" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/7/1319&amp;title=Functional%20Conservation%20of%20the%20Pre-Sensor%20One%20Beta-Finger%20Hairpin%20%28PS1-hp%29%20Structures%20in%20Mini-Chromosome%20Maintenance%20Proteins%20of%20Saccharomyces%20cerevisiae%20and%20Archaea" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459915/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459915" id="node459915--3" data-pisa="ggg;4/7/1319" data-pisa-master="ggg;g3.114.011668" data-apath="/ggg/4/7/1319.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Functional Conservation of the Pre-Sensor One Beta-Finger Hairpin (PS1-hp) Structures in Mini-Chromosome Maintenance Proteins of <em>Saccharomyces cerevisiae</em> and Archaea</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Christopher J. Ramey</span> and <span class="highwire-citation-author" data-delta="1">Robert A. Sclafani</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1319-1326;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011668 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459915--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christopher J. Ramey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz Medical Campus, Aurora, Colorado 80045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BJ.%2BRamey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramey%20CJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristopher%2BJ.%2BRamey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert A. Sclafani </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz Medical Campus, Aurora, Colorado 80045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BA.%2BSclafani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sclafani%20RA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BA.%2BSclafani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Robert.sclafani@ucdenver.edu" class="" data-icon-position="" data-hide-link-title="0">Robert.sclafani@ucdenver.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1319&amp;rft.epage=1326&amp;rft.atitle=Functional%20Conservation%20of%20the%20Pre-Sensor%20One%20Beta-Finger%20Hairpin%20%28PS1-hp%29%20Structures%20in%20Mini-Chromosome%20Maintenance%20Proteins%20of%20Saccharomyces%20cerevisiae%20and%20Archaea&amp;rft.volume=4&amp;rft.issue=7&amp;rft.date=2014-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Ramey%2C+Christopher+J.&amp;rft.au=Sclafani%2C+Robert+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459915/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459915/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459915/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459915/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459915/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459915/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459915/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459915/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459915/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459915/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459915/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459915/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/7/1319" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">MCM Helicase Structure and Function</a></li><li class="odd"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="even"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/7/1319.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/7/1319.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/7/1319.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/7/1319.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
