<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/7/1339" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/7/1339.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/7/1339.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/7/1339.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/7/1339.atom" />
<meta name="HW.pisa" content="ggg;4/7/1339" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Species-Level Deconvolution of Metagenome Assemblies with Hi-C–Based Contact Probability Maps" />
<meta name="DC.Identifier" content="10.1534/g3.114.011825" />
<meta name="DC.Date" content="2014-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Burton et al. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Microbial communities consist of mixed populations of organisms, including unknown species in unknown abundances. These communities are often studied through metagenomic shotgun sequencing, but standard library construction methods remove long-range contiguity information; thus, shotgun sequencing and de novo assembly of a metagenome typically yield a collection of contigs that cannot readily be grouped by species. Methods for generating chromatin-level contact probability maps, e.g. , as generated by the Hi-C method, provide a signal of contiguity that is completely intracellular and contains both intrachromosomal and interchromosomal information. Here, we demonstrate how this signal can be exploited to reconstruct the individual genomes of microbial species present within a mixed sample. We apply this approach to two synthetic metagenome samples, successfully clustering the genome content of fungal, bacterial, and archaeal species with more than 99% agreement with published reference genomes. We also show that the Hi-C signal can secondarily be used to create scaffolded genome assemblies of individual eukaryotic species present within the microbial community, with higher levels of contiguity than some of the species’ published reference genomes." />
<meta name="DC.Contributor" content="Joshua N. Burton" />
<meta name="DC.Contributor" content="Ivan Liachko" />
<meta name="DC.Contributor" content="Maitreya J. Dunham" />
<meta name="DC.Contributor" content="Jay Shendure" />
<meta name="article:published_time" content="2014-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Species-Level Deconvolution of Metagenome Assemblies with Hi-C–Based Contact Probability Maps" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Microbial communities consist of mixed populations of organisms, including unknown species in unknown abundances. These communities are often studied through metagenomic shotgun sequencing, but standard library construction methods remove long-range contiguity information; thus, shotgun sequencing and &lt;i&gt;de novo&lt;/i&gt; assembly of a metagenome typically yield a collection of contigs that cannot readily be grouped by species. Methods for generating chromatin-level contact probability maps, &lt;i&gt;e.g.&lt;/i&gt;, as generated by the Hi-C method, provide a signal of contiguity that is completely intracellular and contains both intrachromosomal and interchromosomal information. Here, we demonstrate how this signal can be exploited to reconstruct the individual genomes of microbial species present within a mixed sample. We apply this approach to two synthetic metagenome samples, successfully clustering the genome content of fungal, bacterial, and archaeal species with more than 99% agreement with published reference genomes. We also show that the Hi-C signal can secondarily be used to create scaffolded genome assemblies of individual eukaryotic species present within the microbial community, with higher levels of contiguity than some of the species’ published reference genomes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/07/01" />
<meta name="citation_mjid" content="ggg;4/7/1339" />
<meta name="citation_id" content="4/7/1339" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/7/1339" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/7/1339.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/7/1339.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/7/1339.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.011825" />
<meta name="citation_pmid" content="24855317" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1339" />
<meta name="citation_lastpage" content="1346" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Joshua N. Burton" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195-5065" />
<meta name="citation_author" content="Ivan Liachko" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195-5065" />
<meta name="citation_author" content="Maitreya J. Dunham" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195-5065" />
<meta name="citation_author_email" content="maitreya@uw.edu" />
<meta name="citation_author_email" content="shendure@uw.edu" />
<meta name="citation_author" content="Jay Shendure" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195-5065" />
<meta name="citation_author_email" content="maitreya@uw.edu" />
<meta name="citation_author_email" content="shendure@uw.edu" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=M. Albertsen;citation_author=P. Hugenholtz;citation_author=A. Skarshewski;citation_author=K. L. Nielsen;citation_author=G. W. Tyson;citation_title=Genome sequences of rare, uncultured bacteria obtained by differential coverage binning of multiple metagenomes.;citation_pages=533-538;citation_volume=31;citation_year=2013;citation_pmid=23707974;citation_doi=10.1038/nbt.2579" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. R. Baker;citation_author=B. B. Tuch;citation_author=A. D. Johnson;citation_title=Extensive DNA-binding specificity divergence of a conserved transcription regulator.;citation_volume=108;citation_year=2011;citation_pmid=21498688;citation_doi=10.1073/pnas.1019177108" />
<meta name="citation_reference" content="citation_journal_title=PeerJ;citation_author=C. W. Beitel;citation_author=L. Froenicke;citation_author=J. M. Lang;citation_author=I. F. Korf;citation_author=R. W. Michelmore;citation_title=Strain- and plasmid-level deconvolution of a synthetic metagenome by sequencing proximity ligation products.;citation_pages=415e;citation_volume=2;citation_year=2014;citation_doi=10.7717/peerj.415" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=C. Belloch;citation_author=E. Barrio;citation_author=M. D. Garcia;citation_author=A. Querol;citation_title=Inter- and intraspecific chromosome pattern variation in the yeast genus Kluyveromyces.;citation_pages=1341-1354;citation_volume=14;citation_year=1998;citation_issue=15;citation_pmid=9848227;citation_doi=10.1002/(SICI)1097-0061(199811)14:15&lt;1341::AID-YEA328&gt;3.0.CO;2-U" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=J. N. Burton;citation_author=A. Adey;citation_author=R. P. Patwardhan;citation_author=R. L. Qiu;citation_author=J. O. Kitzman;citation_title=Chromosome-scale scaffolding of de novo genome assemblies based on chromatin interactions.;citation_pages=1119-1125;citation_volume=31;citation_year=2013;citation_pmid=24185095;citation_doi=10.1038/nbt.2727" />
<meta name="citation_reference" content="citation_journal_title=PLoS Comp. Biol.;citation_title=Reconstructing the genomic content of microbiome taxa through shotgun metagenomic deconvolution.;citation_author=R. Carr;citation_author=S. S. Shen-Orr;citation_author=E. Borenstein;citation_volume=9;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=L. A. David;citation_author=C. F. Maurice;citation_author=R. N. Carmody;citation_author=D. B. Gootenberg;citation_author=J. E. Button;citation_title=Diet rapidly and reproducibly alters the human gut microbiome.;citation_pages=559-563;citation_volume=505;citation_year=2014;citation_pmid=24336217;citation_doi=10.1038/nature12820" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=J. Dekker;citation_author=M. A. Marti-Renom;citation_author=L. A. Mirny;citation_title=Exploring the three-dimensional organization of genomes: interpreting chromatin interaction data.;citation_pages=390-403;citation_volume=14;citation_year=2013;citation_pmid=23657480;citation_doi=10.1038/nrg3454" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_title=Community-wide analysis of microbial genome sequence signatures.;citation_author=G. J. Dick;citation_author=A. F. Andersson;citation_author=B. J. Baker;citation_author=S. L. Simmons;citation_author=A. P. Yelton;citation_volume=10;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. B. Eisen;citation_author=P. T. Spellman;citation_author=P. O. Brown;citation_author=D. Botstein;citation_title=Cluster analysis and display of genome-wide expression patterns;citation_pages=14863-14868;citation_volume=95;citation_year=1998;citation_issue=25;citation_pmid=9843981;citation_doi=10.1073/pnas.95.25.14863" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Frias-Lopez;citation_author=Y. Shi;citation_author=G. W. Tyson;citation_author=M. L. Coleman;citation_author=S. C. Schuster;citation_title=Microbial community gene expression in ocean surface waters.;citation_volume=105;citation_year=2008;citation_pmid=18316740;citation_doi=10.1073/pnas.0708897105" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. C. Howe;citation_author=J. K. Jansson;citation_author=S. A. Malfatti;citation_author=S. G. Tringe;citation_author=J. M. Tiedje;citation_title=Tackling soil diversity with the assembly of large, complex metagenomes.;citation_volume=111;citation_year=2014;citation_pmid=24632729;citation_doi=10.1073/pnas.1402564111" />
<meta name="citation_reference" content="citation_journal_title=Microbiome;citation_author=L. A. Hug;citation_author=C. J. Castelle;citation_author=K. C. Wrighton;citation_author=B. C. Thomas;citation_author=I. Sharon;citation_title=Community genomic analyses constrain the distribution of metabolic traits across the Chloroflexi phylum and indicate roles in sediment carbon cycling.;citation_pages=22;citation_volume=1;citation_year=2013;citation_pmid=24450983;citation_doi=10.1186/2049-2618-1-22" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. D. Huttenhower;citation_title=Structure, function and diversity of the healthy human microbiome.;citation_pages=207-214;citation_volume=486;citation_year=2012;citation_issue=7402;citation_pmid=22699609;citation_doi=10.1038/nature11234" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=V. Iverson;citation_author=R. M. Morris;citation_author=C. D. Frazar;citation_author=C. T. Berthiaume;citation_author=R. L. Morales;citation_title=Untangling Genomes from Metagenomes: Revealing an Uncultured Class of Marine Euryarchaeota;citation_pages=587-590;citation_volume=335;citation_year=2012;citation_issue=6068;citation_pmid=22301318;citation_doi=10.1126/science.1212665" />
<meta name="citation_reference" content="citation_journal_title=IEEE Trans. Comput.;citation_author=R. A. Jarvis;citation_author=E. A. Patrick;citation_title=Clustering using a similarity measure based on shared near neighbors.;citation_pages=1025-1034;citation_volume=C-22;citation_year=1973;citation_doi=10.1109/T-C.1973.223640" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=T. W. Jeffries;citation_author=I. V. Grigoriev;citation_author=J. Grimwood;citation_author=J. M. Laplaza;citation_author=A. Aerts;citation_title=Genome sequence of the lignocellulose-bioconverting and xylose-fermenting yeast Pichia stipitis.;citation_pages=319-326;citation_volume=25;citation_year=2007;citation_issue=3;citation_pmid=17334359;citation_doi=10.1038/nbt1290" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=N. Kaplan;citation_author=J. Dekker;citation_title=High-throughput genome scaffolding from in vivo DNA interaction frequency.;citation_pages=1143-1147;citation_volume=31;citation_year=2013;citation_pmid=24270850;citation_doi=10.1038/nbt.2768" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=T. B. K. Le;citation_author=M. V. Imakaev;citation_author=L. A. Mirny;citation_author=M. T. Laub;citation_title=High-resolution mapping of the spatial organization of a bacterial chromosome.;citation_volume=342;citation_year=2013;citation_pmid=24158908;citation_doi=10.1126/science.1242059" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. Lieberman-Aiden;citation_author=N. L. van Berkum;citation_author=L. Williams;citation_author=M. Imakaev;citation_author=T. Ragoczy;citation_title=Comprehensive Mapping of Long-Range Interactions Reveals Folding Principles of the Human Genome;citation_pages=289-293;citation_volume=326;citation_year=2009;citation_issue=5950;citation_pmid=19815776;citation_doi=10.1126/science.1181369" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. Mitra;citation_author=K. Forster-Fromme;citation_author=A. Damms-Machado;citation_author=T. Scheurenbrand;citation_author=S. Biskup;citation_title=Analysis of the intestinal microbiota using SOLiD 16S rRNA gene sequencing and SOLiD shotgun sequencing.;citation_pages=S16;citation_volume=14 (Suppl 5);citation_year=2013" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. Moustafa;citation_author=A. N. Evans;citation_author=D. M. Kulis;citation_author=J. D. Hackett;citation_author=D. L. Erdner;citation_title=Transcriptome profiling of a toxic dinoflagellate reveals a gene-rich protist and a potential impact on gene expression due to bacterial presence.;citation_pages=e9688-e9688;citation_volume=5;citation_year=2010;citation_issue=3;citation_pmid=20300646;citation_doi=10.1371/journal.pone.0009688" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_title=MetaVelvet: an extension of Velvet assembler to de novo metagenome assembly from short sequence reads.;citation_author=T. Namiki;citation_author=T. Hachiya;citation_author=H. Tanaka;citation_author=Y. Sakakibara;citation_volume=40;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_abbrev=ISME J;citation_author=P. Narasingarao;citation_author=S. Podell;citation_author=J. A. Ugalde;citation_author=C. Brochier-Armanet;citation_author=J. B. Emerson;citation_title=De novo metagenomic assembly reveals abundant novel major lineage of Archaea in hypersaline microbial communities.;citation_pages=81-93;citation_volume=6;citation_year=2012;citation_issue=1;citation_pmid=21716304;citation_doi=10.1038/ismej.2011.78" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Y. Peng;citation_author=H. C. M. Leung;citation_author=S. M. Yiu;citation_author=F. Y. L. Chin;citation_title=IDBA-UD: a de novo assembler for single-cell and metagenomic sequencing data with highly uneven depth.;citation_volume=28;citation_year=2012;citation_pmid=22495754;citation_doi=10.1093/bioinformatics/bts174" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. J. Qin;citation_title=A human gut microbial gene catalogue established by metagenomic sequencing.;citation_pages=59-65;citation_volume=464;citation_year=2010;citation_issue=7285;citation_pmid=20203603;citation_doi=10.1038/nature08821" />
<meta name="citation_reference" content="citation_journal_title=Applied Microbiology and Biotechnology;citation_journal_abbrev=Applied Microbiology and Biotechnology;citation_author=V. Renouf;citation_author=O. Claisse;citation_author=A. Lonvaud-Funel;citation_title=Inventory and monitoring of wine microbial consortia.;citation_pages=149-164;citation_volume=75;citation_year=2007;citation_issue=1;citation_pmid=17235561;citation_doi=10.1007/s00253-006-0798-3" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=C. Rinke;citation_author=P. Schwientek;citation_author=A. Sczyrba;citation_author=N. N. Ivanova;citation_author=I. J. Anderson;citation_title=Insights into the phylogeny and coding potential of microbial dark matter.;citation_pages=431-437;citation_volume=499;citation_year=2013;citation_pmid=23851394;citation_doi=10.1038/nature12352" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=I. Saeed;citation_author=S. L. Tang;citation_author=S. K. Halgamuge;citation_title=Unsupervised discovery of microbial population structure within metagenomes using nucleotide base composition.;citation_volume=40;citation_year=2012;citation_pmid=22180538;citation_doi=10.1093/nar/gkr1204" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=I. Sharon;citation_author=M. J. Morowitz;citation_author=B. C. Thomas;citation_author=E. K. Costello;citation_author=D. A. Relman;citation_title=Time series community genomics analysis reveals rapid shifts in bacterial species, strains, and phage during infant gut colonization.;citation_volume=23;citation_year=2013;citation_pmid=22936250;citation_doi=10.1101/gr.142315.112" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=M. A. Umbarger;citation_author=E. Toro;citation_author=M. A. Wright;citation_author=G. J. Porreca;citation_author=D. Bau;citation_title=The three-dimensional architecture of a bacterial genome and its alteration by genetic perturbation.;citation_pages=252-264;citation_volume=44;citation_year=2011;citation_issue=2;citation_pmid=22017872;citation_doi=10.1016/j.molcel.2011.09.010" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=J. C. Venter;citation_author=K. Remington;citation_author=J. F. Heidelberg;citation_author=A. L. Halpern;citation_author=D. Rusch;citation_title=Environmental genome shotgun sequencing of the Sargasso Sea.;citation_volume=304;citation_year=2004;citation_pmid=15001713;citation_doi=10.1126/science.1093857" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. E. Wood;citation_author=S. L. Salzberg;citation_title=Kraken: ultrafast metagenomic sequence classification using exact alignments.;citation_pages=R46;citation_volume=15;citation_year=2014;citation_pmid=24580807;citation_doi=10.1186/gb-2014-15-3-r46" />
<meta name="citation_reference" content="citation_journal_title=Mycological research;citation_journal_abbrev=Mycol Res;citation_author=G. Xin;citation_author=D. Glawe;citation_author=S. L. Doty;citation_title=Characterization of three endophytic, indole-3-acetic acid-producing yeasts occurring in Populus trees.;citation_pages=973-980;citation_volume=113;citation_year=2009;citation_issue=Pt 9;citation_pmid=19539760;citation_doi=10.1016/j.mycres.2009.06.001" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=E. Yaffe;citation_author=A. Tanay;citation_title=Probabilistic modeling of Hi-C contact maps eliminates systematic biases to characterize global chromosomal architecture.;citation_pages=1059-1065;citation_volume=43;citation_year=2011;citation_issue=11;citation_pmid=22001755;citation_doi=10.1038/ng.947" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Species-Level Deconvolution of Metagenome Assemblies with Hi-C–Based Contact Probability Maps" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/7.cover-source.jpg" />
<meta name="twitter:description" content="Microbial communities consist of mixed populations of organisms, including unknown species in unknown abundances. These communities are often studied through metagenomic shotgun sequencing, but standard library construction methods remove long-range contiguity information; thus, shotgun sequencing and de novo assembly of a metagenome typically yield a collection of contigs that cannot readily be grouped by species. Methods for generating chromatin-level contact probability maps, e.g. , as generated by the Hi-C method, provide a signal of contiguity that is completely intracellular and contains both intrachromosomal and interchromosomal information. Here, we demonstrate how this signal can be exploited to reconstruct the individual genomes of microbial species present within a mixed sample. We apply this approach to two synthetic metagenome samples, successfully clustering the genome content of fungal, bacterial, and archaeal species with more than 99% agreement with published reference genomes. We also show that the Hi-C signal can secondarily be used to create scaffolded genome assemblies of individual eukaryotic species present within the microbial community, with higher levels of contiguity than some of the species’ published reference genomes." />
<meta name="og-title" property="og:title" content="Species-Level Deconvolution of Metagenome Assemblies with Hi-C–Based Contact Probability Maps" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/7/1339" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Microbial communities consist of mixed populations of organisms, including unknown species in unknown abundances. These communities are often studied through metagenomic shotgun sequencing, but standard library construction methods remove long-range contiguity information; thus, shotgun sequencing and de novo assembly of a metagenome typically yield a collection of contigs that cannot readily be grouped by species. Methods for generating chromatin-level contact probability maps, e.g. , as generated by the Hi-C method, provide a signal of contiguity that is completely intracellular and contains both intrachromosomal and interchromosomal information. Here, we demonstrate how this signal can be exploited to reconstruct the individual genomes of microbial species present within a mixed sample. We apply this approach to two synthetic metagenome samples, successfully clustering the genome content of fungal, bacterial, and archaeal species with more than 99% agreement with published reference genomes. We also show that the Hi-C signal can secondarily be used to create scaffolded genome assemblies of individual eukaryotic species present within the microbial community, with higher levels of contiguity than some of the species’ published reference genomes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/7.cover-source.jpg" />
    <title>Species-Level Deconvolution of Metagenome Assemblies with Hi-C–Based Contact Probability Maps | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"iq8luB60z3y4lF-cSeu_Eu_P1c_Q-K9NS5_epatcroM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459908","apath":"\/ggg\/4\/7\/1339.atom","pisa":"ggg;4\/7\/1339","ac":{"\/ggg\/4\/7\/1339.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/7\/1339.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/7\/1339"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/7\/1339"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/7\/1339":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022pE9X5IhNJ2QjT3CfwS9FFJMrs3CVdEnWnN8yjn+O2SwFvSaYFLgZTHjb\\\\\\\/yqXGRZEevBIcju1d4Jj8U\\\\\\\/XsP6oUbCnjRGR\\\\\\\/8CunS7lCIxSzJ3GCu5P2MYkEuCkrRSBEUyEnRgADvaCdAg89Jdv3apWqaUX55cfABvk\\\\\\\/9gpTn\\\\\\\/YLWNkBS3PNuGV5Gbuf5waZD8UtTdLTyld3cq0KAeQWrDxu87iL5R0i3CLluCZieUJckCUlr1QsXDOK89k0tIUOCoaJ2CjStBcA27ddR1zTFCcLL4A8LpEWckyFEeehO\\\\\\\/tMddWWP4DeFA1i6UyuHcq1LGma5kGnbgrN\\\\\\\/sISE6lAGawCeRKhjf6+YRfAMxwVrYpVJJ7j0pVqO7SlVADwnPLQBQQkp\\\\\\\/aSFJa8G43u4Z4NHDr9mgTe2rwhT4J04tPkx4ibJ0OdcPRbu2C7\\\\\\\/bA5lk8xlbN5+2tftcWBcBY7yv949N++411cflEdFQPldIxyURrBZJ+zjktkTPdDosUnpxCFJ8EoaOzmPVTw3SixzIFiHAZISAtbQlbBnpvfHdLWyvwnqugfB6awEADKzTR0T+EQpIVusKMYhU3p8gz74Rj8CZUwio6Rq+YbK90F+WPoUjUJ\\\\\\\/lUiXKFtb4pY593xecyhDQjMMhuCJ1HEKMCEYxQoAH+m+Yrihd\\\\\\\/eOLzCXDkPrzvTPpUe6kIh2nGleFnw9SR2Ghb1A6B6y59lqRKPSiUNb2tdK5\\\\\\\/8dGi4ofhbQCKEiEdM8r6DZvrqA0d5y3yqKnb2cj5FtK9\\\\\\\/GXn9WX6A5H6He9Ld1MGSt0pwFiBdXkLKzr3L6sPVf6hr8pAZ8kU54qgiqEZMUtLo7TRU5nKZZbfe05BN1tyv5Q\\\\\\\/0\\\\\\\/jRj+ZWA8S9B6890NGMVUNyAyeA8UkRiaJqxm8WBdoaxe1tl1vc5P0pO1ihTUUpZGuqJ5WnjgaxuJFHd+SxmBwvgdfwndKs0aiguZQk\\\\\\\/nwhnEE6hjCVUQryXvE18aII1B93CXFLW+IQ1YVB\\\\\\\/P0H1MjU1hJJnMAGO2TSP4zr\\\\\\\/Ot9f8OFc8XMw82Ev7wKq0jFJgEoQa8WHsalypzT\\\\\\\/uXOuxBBvBkK389SFHTxyGJLUNcIDCr3M1wqgonVW4Mc0sN9Bta36LJmPyvWdQXNB6IP7vi7UH3BCJx1Ltoqxi+l3N6+QKnSldiUj4ePsZgnl2NjS6A4xiSOAaI09WAF9cCelmvcIQb1GR2Y3nkS+SeOZfTI0FOMbnE\\\\\\\/O5Ulm6TYHAk76QDmyMiwa5pV25oE+GDUAPMyPyHWTCGz\\\\\\\/B\\\\\\\/D\\\u0022,\\\u0022iv\\\u0022:\\\u0022i4wNsa686KDbv2HDiPFqbg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229564b59e4e3a2f51bd399a2a7b1ed1690cf855371457504d4081b2bee23c6aa9\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bigTARPQEl6QYKi5LuFq2zvHR5BN+vmsAiAgCQos8Gf2n4ZyRF0KHpH2rE1mFYwu8MpVCcZzM8Xu3pdm1sZEJSPHtJBsMZz6uH0fmmbAH7COjOTIIUs4ql\\\\\\\/C2ckxsy\\\\\\\/hNWM2SvUfXkn5520pzhm4MsfQZIAA8C+hmd1K+fziw5vAuUiE1baSefuqiD4jdoi2xA\\\\\\\/m1P11wuAvllUaQNQiF4+xXx5TH3Xuk8WHpl2MhIfcDQ8+Oa8GzLEkNEM11KOdp7DcLIUq14iSPgV+276rlA4Xo8fMLNESv1axfhhCnTUulVGY6qP1FkloCXt9ef2pnJ6WxGZ5CbDkvYky0urF\\\\\\\/8C1p5TwzmNL\\\\\\\/A4R+zDWk0h7oWg\\\\\\\/PPoNquqojworJ32eP3rmk3h+YXua+89u4nDXQ7EQFDaJYQXp\\\\\\\/MWZyPYlSeWvdHsCwW7cyhoMEckR0l8OUfTR7l5w9PIdPR+2zv6jzDX2Tl2XpQWDhD\\\\\\\/CR98vryj1sEWEr1G\\\\\\\/YYZKHndW5O4t\\\\\\\/M0ENJiyFNYwFEZOiQ2GSPhbUjDNQKDayZGrxSNPTjbDKenjwWszqpsLuLq2PjGKWCtnk6rK+D68ghum5dfyp0JT7hhrgPLHBjOrPMVkxq+qv4H0re\\\\\\\/jHNVPzqZ9HEwUxQ23xt4TH3ZB++wB6vmCRE8rP+aBwBOzMJdFlPj\\\\\\\/JsDc2OeBbR\\\\\\\/uweBLbKko62s\\\\\\\/tQ8NSsRe9xbokHhRUU6rjn6nqAj0pyNbYn+0g9JnhGpFSkD6klS1VaTHDDeC04QVTEbVPNd1DuKcLnSdoXjYHyPy9ocCXPlcyHQI+Pm6amEH8q\\\\\\\/lpqn\\\\\\\/hq0X9cnVv0XjAxoGrYE+ncT2Pulcu\\\\\\\/Mg9naLwpzUotgBi8OCkh60gyGwq6OMPZI48virnvOmS\\\\\\\/HxVtYDx+xCTKC9kq3S3dGDsvyjSJi0O8VuV+zJIY1JjtGscZmPUnwhHm3McJuAUKayc7KrS8CNmNpDlhPmv0VE4Aw\\\\\\\/7NLxL0XZMeW2RC12jpZivoA5PkRitnnMHTI9hm1+nlC0ETUQVV1eswssFtG7gZV53yQec1+q0Bpi5SCSzS1I0rynWkwc1ei3DPPws3JhXSj+7FsTDBvK68WSOI5E2hKaKee4iWofOnnf1TIjS8R8wh5XocqHix4l5a3\\\\\\\/8MZZYGw1zaYYCD7DANVm9rGPFKRRPW1Acqjn1CHYGwlLdM\\\\\\\/AIuK2WBByZseXaXdeNqpFFueo31q+9EZh4y\\\\\\\/uiPfLrebvd09MPtbj+jScgrmteN1L12TKI0MsUPS\\\\\\\/PXjhE7zHrCvqGAFDuifJamTwQIIfkGtDsjLc59h\\\\\\\/xUdGrumCdMl7JWZHrJEDDtYeF46GNZnglUGHk4Y43WPHLar\\\\\\\/lWmdF+K8nuMPPWruwdRwX07ZPlm56bm3H49LVZGCqxkWZjXdydbJKvjPd8ID3LXYERMjaeS3Pxe+F1AEGtRx0jqUeasTLGu0XyEC+YQsK3lpk+SlO+UWGnw3qNzXFRMSdgLTZCBTKjmM6qf\\\\\\\/GPmP\\\\\\\/EX3o1QDIShvWT5aFHiQbJHPHF8dFvpQs6YTxtPyBdy4TSzaVq1W77JQRiuHNu+09hmBcB7qihhL7Yg3\\\\\\\/43KvA9cQX5lp8mb0dD3JG0VfoWp287J3hQ1DVFMD\\\\\\\/uZ37gtXICCCz+Buh8XVft8HYMMqALB2uj7KlkCRVnC1imHsoNrqACmi0rkBS9S88h7KI6qkUmoFq3KevPJEeJQtuVoWMO5vVaBPzt3le0UAPDphUtl5N1zZzSbqRtAXz638ly6nX9Lnjl7vOpUsAL+yIdv94A7zTPjFPrL\\\\\\\/Z+XCwZ3ETyLwMHSfyjuh1hoTBgaPc+s0FCVYP9Eh4b+UGBNFWyVCQ66mWuk2aDxl3njpX437CUP7269cd5Zsk4ICEVh7tc76Iw3xNve7RlKp0b5bwEw5Jo2MsM8+MwDnoiQQRiA+OuyBRdmTVqR+TYiXXbSNplAKU\\\\\\\/eedsPM6BiocFwLPgXzTeE\\\\\\\/I7qvB2sidXV1bId0iPLhUFmb9QSWPPwhPyZkLl\\\\\\\/kXYXbos752x4aoT\\\\\\\/un5wRmdpB9lku\\\\\\\/iRgR8ndmSS2mPvh4my6zuJVNXzNJ5We\\\\\\\/DnCD961aKchxDefiDS79WPimTiBd3WZi8jNw4TvBNz76eetkFaSfP\\\\\\\/pnd4tuxla1H4lUXPxv3Ba0mmTgDGdS\\\\\\\/pSM8Ww\\\\\\\/GHquqaRfttjttyYlxxQy6BYEv\\\\\\\/ye2VRLtwrJtyFPoJidSywYsb63f08clfwTd\\\\\\\/Vg1ohxVlbHEEFHWI1mcwTy0rgAlmSvZ\\\\\\\/+7SpNAV6Q6TROA\\\\\\\/t+Nag2o2QFbXBqqggK3F53KMZhXMPfnfWuUhBOk+PXxQAdYOunq3jIOLv6C5PzV\\\\\\\/EVp03keWTpxOqPPfppHl8TMnGM1f1eRQdw\\\\\\\/U28EiLA1pZrxhdAw3I279Yi3x\\\\\\\/UMY7Xb5iXgiugZkdEgq4b7mFLbtsPoY\\\\\\\/twQ2nPo8bGTKQa3IHlidTrR6V+siVF4Ql083\\\\\\\/jRfmPULTkF\\\\\\\/SWdKJHGjir3EweIA4e1mB7b9RSBYWUU35QzSZOOqOy8O1Xte9ZwtIOd1Yf363oRhHep0d3VvhUfxO\\\\\\\/Q\\\\\\\/XcTv5yKYfBqBaI5ecj9XPtP5FaiyfMU6nHnfwoHmOBmyj2Z9yxcC9HtN5SnK7V224n9z2wV9UhBFC\\\\\\\/FGJtQdnOsOwY8s+76EtWMbBfRLvkpdEtp0pokQ+5wvDxXp4WHYct134ow997eLzj8SqQeyoJ\\\\\\\/mfCq0E+gyOWNLNJHkPnuSDfJ\\\\\\\/KoOlzbLWrYJf8YLZ+7c3xoYmTtV+UKIl0WyI\\\\\\\/H71+xCLs6POnaMn07UH5y6baO1PeVYpeX5DUnGM0MIUUV12Rjvd2vv5yme8sLBpJSJWgEKrinsZmWW1Ej40Rfgwt5NNmLejgh+t67jioKX45AAVKf7BwkoP8cxvIorEWSRPHOXZXmsvfQXRz4KjmviWIbH+b1zwsfZX7Dbs8ETekCfkK\\\\\\\/BvJMrt6z0BUTAYZ268mGMfH7VP0FSIaIYm08CJJKbhXZ07l4afPzIXRt36qsCjQ9bmSyePotdPa66jHkdy0bW4M521ZtAuiVGi4OdEUX8V0tOtyZib\\\\\\\/N7TgStiuwr4erKlQx+tPwVCZ+1aLyMpyq97188W8B1D6wkW3\\\\\\\/a+Ez\\\\\\\/OrUauPmbmCMYlaZc440punCD1Mu8HzU7KqvBQ+AsdTtsIDNXLq5R1hQo2y5EgzaviIFagZHFJzv4QvaHKXi5Ghm8EJUG3rlYzlBU4cXjfwdhvrkftxMD3wcoPiL5wJvw0JB24\\\\\\\/BtSVxLZAev6tRjFXYfHdSg4fYhp\\\\\\\/kY6THa92To3CueuzbnjfQ0lKfhM7YpbreGJi6uOf3mEqdvRjMC2ou7fNz6ZoEPZqo5t2yGCN7BkT6c58ShpOiinmvsRTGJSjXxbj5bEBwy7AmIZf+FbGQq6N7gyDIsq1iTu+TZ2CKULa+9CQBOGNflACP84JjB1BQ1oycqZe7q8dzxEkLbbnoGbBnKUWyAm5QjxnB80J3fAk9EsJ5Ki8P5f\\\\\\\/b+1t32Euw5ttaWhrbG65h+EqiOHRodV9AtLYwca8vbpYuA0f4yfUEONRkHvqXAFfTScG2Wf7g8VkL4syLzJ3IrjjQqXyz1QaG9FmjyUa1QjTz0BrZQyAMZOlkxMcoOheK+2Y+o4yX64hpeRvUcngiL\\\\\\\/GmViNptUHYdUx8Rw7Cr2\\\\\\\/f7zuWcL61GDTR7fARyAFwEC8qAm0uoX8dV4rOqCdpEGxmO5G\\\\\\\/vtA6ncxNDf8qXW5UMeRK2aXeCU62LvEDY6GBHrAs9fdkspxKl5Gj7J\\\\\\\/l1VMPb1xWJGIG7OFI4CFWwTFIKrhw2oXNys6WwhFaSys5KttFcoiOKjf3875fw\\\\\\\/rV0lre5y7YdxsF3NX48nhE9buzwh78K3vNXWjLQzrPUefqYkS8sKeBCX541c2XpGBjJ6YpocpoFl+J\\\\\\\/Tf9NZIIqJ5\\\\\\\/XEZN5UeiHDSN1qJuitM5zSBQ5GsOg7XyGIIIGwsFaNGkfp0PMnMbOUaw8oi7yzfbt2s0wEvEbaEuwlN7svDxjS4UupbiIyC4TQXb6jirESi508bOaX9\\\\\\\/7ZbMVPtZFvnRUY8e72+pyeezeI72obLZh1f4hfQ8UYs634wRfvabTQriRFiTRbOoqZjlATB1KoyEjWPuxrxXMmLIihXStNUTqvpVowQv4+FjF3+ADE1bhm+aF7N\\\\\\\/JYRFyhzVeR5uAly6pJbA\\\\\\\/4gmxjat0U7UaCfR74nFUW3LCm9ekx5Vs3KrbkFdz3kpbC+gLEUSHCHKe4SuukJgv2YljEKvxa8J9slAPotmFtia0Cvid3g59mWe4EcXZoXAFyF6NddEbvYan67DvZGmd\\\\\\\/kfnWLdTHG8h7ZktKLgwVGvZnvIPIi66+gRZ3\\\\\\\/iH4f5beR9gRj4qKPdnYbJtd6uNMaQO1PfBEV9bS\\\\\\\/yaAnaN\\\\\\\/mDIbgYy8k\\\\\\\/yJ56caRmYmDjh1sFAdEB5KgSzeppK\\\\\\\/b6Ff5e3s60W6uqgn9jaFYK6cJg4llMiOf0imp\\\\\\\/6jL0YrjU7xZhjEkkjG0aj8FVs1ypDfCJ0Uy7vw0VkbBxE3FuaCVt2p9FqeEDT8Z0DY3+n0P\\\\\\\/OUpeIa8JacuhEQ9QQqsh+v1as8gD+8nI0nVKwkaqbxjEjwgC0jupruDCjimsNNghyqQO27WCbe9d+tr3h9TA8RgQ0DbiKQ\\\\\\\/IX\\\\\\\/m4ZPablbPo59\\\\\\\/iaRnuHYcm6u2j5ZkpSMT9D2+5bvgnxCp9As5ExSh20jRnKR5hfiNoJUOW8lENJVcX\\\\\\\/pWFLA6wJO8avEeHGBZvSheCUduR8TQJ79bEvj48vaS1wb8TbKiPe2nauszzcjPVzvRgQ1UXB9D8HfDMEbInyPolCuaRwedn5aecytYjKiNfXVIM2qJU0qXsiArOPbTf3oYoLsTcF0+x6rJPCAtM2rzsqVd7RfKYbQhr1ufs8LX1YXx9kvavYKcpechh0Wz2caHWTs1imrUCFfWR45q46+4pJfP3KWXCRayySUHv06kjtpYeE+WtCgOinZa8Sxc6MX4pNqYRNm1SZaV3kuzjkTAbcr\\\\\\\/8cKgF7SXVIvSa3xq5Z0iXYdQuueQtspRO6+A2onRsU5b1j7UH8kg7p9FVkzSHyL2GzKgpEJy+ZJeU646UyyddYoQ2IH67lfbZsOYxVeHjSzdmmPOhBb4WLHshOzgVoviBenEwzJaxZK\\\\\\\/9MHa6JWxiY3BHyIzfvhWF2csGVyw28XOXj7DQiEfhLiHZ+Kw5COOUkO5GSfbkksmNKajTlSnioaMjK23qs+jpNGD4uHF1Ih85jDEAc62tdPUqIsvmKWHiDTj1MWLmqRSE+vXo2Xlm3g471CAfCYMGVlz6NvBgY7unP\\\\\\\/sLpPQjXqp9EXd\\\\\\\/V9nKlZ9I63yDwcIg0KzFiXmqQI+90WAGfxv4hNOUgxMvpBk570fBHEIIP6fARJQZcIY0E7ywg\\\\\\\/eU6C+fLZK6zLxMTWAel\\\\\\\/KTkAwu0YPxOi6STuCPhC5puq5G7f+d0S2zJmPYSVZhPLzFaqvU9l5va9oPsQplD6nUvxQDv2uDRaObCY6IxMN5FYc+8NTq5H+FSO\\\\\\\/RZgVAdEv\\\\\\\/2BVaStN05itSUV\\\\\\\/0+y40da3jzT23wzpobP9brumt7jOILftYSaO5tc8GjvYdJcb754t3RhRdLNSCvZU5TAMDgnW\\\\\\\/n5m58UbFGBwMu86iHjKAsCF93NFHBWUTD2Uk2dZYBg3dpoI2EAYp6j940kSZVPue8uYTopAw7RB+CdXumTepb65Xupb3UMYe1uEWXrsDZpFRoLttjDfl2oLLY\\\\\\\/iuCyqWsjDkFJv8JszleRQMeUdleFN\\\\\\\/Ke4TFbMaZeapgl96GX8+cDy8Xs+CTQoPF2nf9mr\\\\\\\/m\\\\\\\/7RCRGsfcgiWyH4yD6XSIXI1lCvvPqvao8XDEp9wxXjclG5BYWriBlpZUdUgv9h4HsrEwx8vGRKzO8cA7jbKm65DwVC2iiBCM4XJeI1JIU1Fn0x3nGwjtXL0aUvMhtl2ifAOu46SygsizvlPeUe1Ih9NVu9vUYQiUW\\\\\\\/U+rMk1lmeihev3L7aWq5xU349hs\\\\\\\/Cyz8Wl2iJAk\\\\\\\/3zG0gtja8FpX1FtqoYMVBmjfiTqDlI80h3JDZf+q6L2qHzmVYDsrIYKBVAaaXfVLYc\\\\\\\/FXXCUBJ7Cm0\\\\\\\/rwSNd4CSvaz\\\\\\\/x+ARcvVtuuR5WDJnlk0E7K26Xfr8c0RscySH\\\\\\\/1Z8oTx9ercVd\\\\\\\/wqWLkZY4XuTFECm7x\\\\\\\/fExVxm8ErEgPZ+VWvLMLY7\\\\\\\/YJBdT0NkVM+MwzpcerH3Cv2TT9Ew2RGBDWmCjPzmnOkQ49BDkeCf8iyXJzg8+tA4dWO00PZcZJ4OgY+Nu83GEgwzSDuZZmmlr2HectpXDUbBHmWxoKSBbQaVJ88SMu2gISmFefTWBhIfesxjjDgRzn8BLm27dSkZ\\\\\\\/cQcN\\\\\\\/Uc19tMISsKaeKWWC7vInycgU2UTG26iGt7As8eInQYJ7Mi9nUXCqKhnUsiLweEw826+Al+CDABhKSmH+OFq+b\\\\\\\/gFY4dUA3fAoWtKX\\\\\\\/hcrG00Pi4gy6NevhXZW2bPVrDAVztyqAgkUIu\\\\\\\/ErwX3MKUupyVaaQTHYFelQQgzQPJMe2+diG\\\\\\\/TbuMXy9aIxaMP3OjNNCja0qWzvi+mAycv2Ak4AavYl\\\\\\\/JHDccDdZ5xk\\\\\\\/OE785OF3AL95UDyEeIjchg3wMWALPEulLBN3ZnmA2G0RARwglEPsJFd25+esQdS936B93HJ5h034cXDOSFacOr5aGj1C5GLQi6hW92i31LLKW3HV10Keueo+PwZjPpm63YRX9BVD0b9VwZc3b3YOaCXTPmhFkUaUHXl0Kj3b9GVh7refAgNuLrVXoQ3yARvONE3gDMTgmZvpxiMuu1GP+\\\\\\\/zwZN3DcxILRGBDZABtyh3CzElwhRt0JYJkqD+V0fSo1KwwdWQWmRLO8S1ZImM4K705E8qznSyo6IS2U1h0ucbei4N864XPPy2o+gD4oqzqDWb\\\\\\\/xfmvMpG\\\\\\\/zarJT+LLr5lTCs\\\\\\\/h9qJljXhAKz3tt79N5xXeA082xEfvqJaqNoTUra\\\\\\\/SgRiAruJlb9cCgmQWKsQtw8J2irnzFTx0jNKuqGbS7eJIuRy6m\\\\\\\/7fAVDURNwJJNiYmGSK5OVs+tpZ1vH30AdjlT0iEG6w6UetKtO7tIBKFwysRsabB2LoSbnzOVr\\\\\\\/SACFqTSOLg51AT0YtCXeNbcLwTbrLyipqPWYrNXa0olZ1NmaQgfskuwNasAFJsRsLSEwkuseC2V0t6FtpwBUjWegpuhGyU56swK\\\\\\\/a8XUWm0U68hsrWV9M44Gqyzy4IxyMj4xNW9TPYUXLAtQTyQsezgYEuse8kUt3KgTD8My5v2MUq6H9BQoTSJC4LeXAKi+Ut8CRTZ9ADu+DftAdV2E21yphtaay1OUa5zbv4w19hoQlCN5dClQpj2r4YZKXRN5R7w7A6P1LOrQb5hc0MmxLKG4p40tvkVeyt65Rnv4zpk5VUkQa74QR0oE0FWY75FlclxJ6Oqhr\\\\\\\/uv3o+dkE7IJPZIXkikpb7nDKzwb9siz7alyX3fgkCSvJb5y14ncI7h8wBLNey2r1buZlZDvJrrytGkEsEkQ04WMj1qDZqaWiV2Nd2\\\\\\\/r9wSB+wbHiY2A+qSu\\\\\\\/fgxwi+p\\\\\\\/+ZbMzXsngMmT81J9MiF\\\\\\\/97RvcbX9YT\\\\\\\/vRlL1MT0XbXtWc6xZ73ArxHFyJipPuyif7rverOADt6I9AFGHO3fPwA8E1UWnyZl\\\\\\\/ROkaOEtqgnqQSoTkprNNsnZzcOIWYdzOVDb42p2xoK83VgaNg+3JcSWN0onVR7DYs3UlUg4ul4Yc0tUKsjAkj9k7TE5rwRxNOSYsNCrj+tRB8k+lAaH+ZuWS\\\\\\\/SPGMNJS105Rcf0Vtt3o\\\\\\\/bF+u3QgtUXwaf3L+sQKibvDHGYtx9pM1Soy6SgyyAg0mMWNzKOiHNhHWSCV8R58M1Xfuxpfg3Ga\\\\\\\/sb9aXhxEdbcZQyfctrA749vi\\\\\\\/Uu36\\\\\\\/0mO0euHjftijym+rnBHjMetvxKimykRP\\\\\\\/rWJZxN\\\\\\\/NK3cvRJv1b+TyUjWoyMOvTmC8v1fMJOv21kSvDCbY7eexnKWh6zHa61QW9GRTmDooD\\\\\\\/hTvoFEwXqKvZJ3oZfGrMicxwxSwcHCfIn\\\\\\\/me5vxFNi9eFkFiAQN2WWENVV2YllUu8BJj82jwq0Ux3p3XiBoH0xqAQr+aOz980StqE2kRa39A4\\\\\\\/kBs+pgjm0H+7bOblGNJFr57iwR0V2j9Pbck9dhK\\\\\\\/6H1SqFxIw8h+s9QTGuppi\\\\\\\/LXdIE9Qqxz75QF5eSqntdHWMtdZ3zjvjcIMun1Lk07qYtOQJbG7azpBTa9smYjS8zDNy\\\\\\\/D8h0eE5hiqmLMLVi5KAcnGfrGZo+30e7LtocQInzQ70cOe1r3QZ4XTJRbZ9UoEE3dBRmdSVYaNkp1vU9\\\\\\\/3aKTLKBT1aSbJaorPsLVW7M468YPWz5YAjb8QhkhddVrrakC4czubJ8xd9uypoRz64N7Wd64mfDrk8SdbcxdwNQd2SsI6jQy70AkYmme8lOEnJ7VQ511w9XmR+plWxQZ2iGP8bCQkqb6KW\\\\\\\/yhfQke0enKK9RIJrMiSEiO3sYIuQGVlHMpBtkgzLGYGGn9qHA2ewB7WoQsX8GY+L8yYq4UwUCOMYD\\\\\\\/p6mQw2Tp\\\\\\\/BzGIrNR4+LRVssLShiNpM7IMvpv2trVzYFL5zpxSVLiSOXnyylbwDSCaoadZBGKRqATKhAOyInAz2anA4M4c8e6UEuCQETLQNPeZaB4mK7XKlz2uftcsOa3jfC0Nkzr57yIacVYTzGcsS8+Cvdw9+7Z+ZFA42QSpLsljCOyq4iaLxhBoUIg+RzVRhFuGVyUAZjzCmX3XngcgP6Jsr78u8w5lkZuWrtsDLI0wm0xuSg59XP191fQmfp\\\\\\\/PM+W8w113Qu9VuC5HVapcPO3qO7\\\\\\\/iQIKc9Sq0F+poZWYd952jw449ZAmhtkMbXkEzj+aPX0LIUYJQnFDs8n+6kEnrxwfWKCJL+Kx0I4gc8QqPisX4WzQxHwfIVOilyhoyKkmxbNjVvsRL1JMCzlQDbfOJYa98edYNlFwK4HgXw95k7GXGG91lWvGDUma3cPusVQM4YLgi3Hvqqv591wJ7ZiVzeP0RHPc9ejo8tZQvXYhR1E50Wt6MZ+0j3O\\\\\\\/gbVQ47jEI7KfF+l\\\\\\\/loF75AoFuDFOXtgLmTxKmIfkFsEi2oThlW1Sucj8JthKX8Wkm\\\\\\\/l2BL3Ts4WewPPy2uJS6\\\\\\\/HBvt3dJ2gwa\\\\\\\/gyxkJAufGbSTiYGcgkuev8ByVk5bsDETQNskJmL5UpcsACHDlHTfWwz\\\\\\\/Fv+HpdajNNOkmPkVTIXa9imc9k4OkyXvYM8eWSiIiZurrPegwiFuNNKb+G4tE+1la7zw16DIIyf6Yh\\\\\\\/APX+pTbX+fFTzwkJ2knJPN605hYjtJAnq2QNHSSF+vJhqShRqgWJVKvHc7IlclTjF7pr+5YAXga6KV1G0jFyFVaYI5Ef7WRq5iQuig\\\\\\\/ZDoynatMp\\\\\\\/Fg6Ra1DfgQ7SYVDQGLX8YCgH8V033icaKiElU\\\\\\\/Uvm0nma8nBqwrtKfBoMF6+YF75GsfSnZmmANNNjafFRm4sVycziwaaYdjJB8sgzLsh29WA6HxAIh9sRXmpXPO\\\\\\\/oXcSipivWFQ7w0KtQKY4R+KZBfy9MjeJGjsyF8oADplRymWM8cSdNoehLMW9C4NhC0tSAkaaEJt1+2r\\\\\\\/nvOyQVFEFAGf0Nk5xFksVwffXZ6gNF+8rFDWNpMYFMc1Mqs3zmAMKr21G9+uf96lJ6j7gCX2O5hy45wrrZidkzK\\\\\\\/MG4Y1izPnGSODTudCjz90FvJyT1EcahERNaRCuJeGfkLKh7jNMqVrJPJT6cDMIkQDnbqfqPG07d9zCAQgbKXRAiCLSimvFa6WmfXESI+DySyA7Hu+aT+7mYfmhOtEFyBmplpQuLoPnIeWAg8bNutZWXsaYbOrUv8HC1qqIplSpHus1iQYKjtTcO2j7cEP1ie5o9zEoZIvyIjhr35LJj6HKHV11pCtUM+ffcQYlzJxiUxUMOI9Fq\\\\\\\/G1rh1Hl36XlnuT3lSIROVIfJYPj7I+rQ7+lxqb43qbA1BjosM2o8eSac0tZOlxxJhps0jJt\\\\\\\/mwfoZUOO9\\\\\\\/6HnHgx8phNLgVZ70ZY5fc1WXYy0HFFyYLPYYfkGXmY7dM63UNQ4elnCwfw\\\\\\\/Mhdy+Gd1ERxzIiplkTuhO27Iarn2VZjxDWemKLBMlT+hFChmSpCT42guEZ07xjBcMR8V15PIKPE8fcCEZxm9RfFl0NTTAEwHG5cQBt\\\\\\\/cmZuk2IExWO6f8Zh1BnZYicabMKKZKgAFxaRaojqfX2twjM01Zce3xLvpSUCLp+0Q3f8Zh92tJ28IWqvsXPO8RtRVvn96X8\\\\\\\/feZBRXyi+jG0r\\\\\\\/8Whsa1vG4cp3wQWJUD8r\\\\\\\/U5hYdBKSbAnAqRpOPyFHufphSaGdY2RdrEuzD\\\\\\\/iJSLk+\\\\\\\/w0g3LRQkZmezt137MsLYXalB\\\\\\\/YdJbTWArDFXs\\\\\\\/5CmjiwBcFBbJvHfTw5RGhbUhodkk9ZiKDsS6iaLiUGjPwK70TWkJtrpHkoO2YLtVP8QycQNuxdUimFLlypF8ECncjCUct7LCZFAhazyNVh7HsgtqTSWcfbNmfb4fkpwXGbmz8t3nfLupaHahha4WBBwlHc6TMaGf2ssFawlCAiKudkd8rEkBv8A20kV\\\\\\\/QWcy3nzym6QLsLJRZjtUHyKvUN73oVWgiUYDKAIyBGi96Al5nfObbtn0ocR3uJ+KbRnqenXWRBdUYg6XHBgBWhGJ60J8sAPM\\\\\\\/on1RnLo0rt04h5zx4rTouGzVWvL2739\\\\\\\/TGGu827f87GF4Uo1yj8wNVgK98FiP4vKyoKco1Y6eGVgiclZeGoxNDrVMLN1SS3Qk4dFxKbJfO+TlfM3vsYzmbKWNeMCe7\\\\\\\/z8YEUgLWj6UChkUUWCFwj1tG2e\\\\\\\/KH53QSTwRLZur6nGB7tDoB8RiZYPokVthmi5fKlNBvgEZ3J0ScCoaVxrVZXlmu82UCFyUm9ywA8vAxNq6sabC9hiSFj34AQtMKmDE0rgK2ErmA\\\\\\\/BHtPbQaJ52rooauaqv9+nCwFYGkYPgzZyTFyFbGZRFdrzKHOWRsy0ok81zrSqs25FtshUTg5UU4RwPXOZsPpLXNNfBRyikm3P\\\\\\\/ciMyouOGZlrDFYEve5QLVGuDTc8n1gvXv+CP5Ue+HIlKNNfnHAbkee\\\\\\\/u1Tg+85rQJkybl44InA0FhellZlb8\\\\\\\/QFPpdkWkJb4i7fYXj2JPUtXJnzO53htHW9gSHpV5rA4CvjHqvYsZn2IJujOyMLgQEi+d63bxyWu8OY1s+hg01tznRwgViqsvLx\\\\\\\/oBge1a1R34xF\\\\\\\/NIxk8KmC1EWtoqQCnUQjXu+cXnZJXdwyTuq5RBqKwrSwepz4MM2237OGLkL2h1BYJGcDm40jyvaoDXgM1obOHvVy2ujZ\\\\\\\/zU0hDNq3hbgCjnYxZmxS3rGkL48kNc4FYvpTEbgyFSGrWt4cpJ8DFJ3Z5D\\\\\\\/TGutytKlZOoZqXOLNFRb8aYEMyY9jiCM5Shz5p\\\\\\\/S1CKwJTy1S\\\\\\\/z3BkMVCTCX83bGyKrHDUUsVBV9JNIEaR9Mj1nIAjwUmwoe4OxOGSrw\\\\\\\/LsNYddOoSf1vgOEmznVYrVfoG0uAy9gglUmLZvV2z5w6SIc5qIAOX9keTaXrBErwSCDn1OBO+CeXfvp+hFvEn2+0OPycofbFPEa7HFQhYVooGmI1Jiutun4YO+jT1qfRQi+lffN0IybXdeMYsZY5xtJaZbx6XYdLkt+s\\\\\\\/IprK3ZMXgXcJsFXQedOjpshWHPIjy\\\\\\\/6b10VH5V7nEyKN4GCgnUfqkBOf\\\\\\\/vwKhoWOT004enURXxy4CrCtATdCJmRvZmPYW5gZr0UVUOmPblawNK6BnfN2YzrzjWaP3WrQfwPASJEEgep6QfrtQFMpPUu7HDij7qFacyFShJ\\\\\\\/RDHkdoOu48qgOGGT6MIfUQJ8fpL8fPLty5D4uzsBAicfts9g+le3VfaN1C0HuKSPRgHp1tbipHzJHP7SBKy1pv4Bnbnj6zhgaPle1TZ8W7ey2M2GzIFL2GDURjxStAXO6kjtz62Fgsq7O+M7DRSYbyNkOsvgQUIG+odMWpjFwlUrkzRdmJHApCD3swFPd9okcBssRNlntV\\\\\\\/P39I5qY+sNfQfLyGf5Av7Qbo2w1Hg\\\\\\\/id+RNcYXsTcEUWpVfDLVPPWoeVGtbSFudmYdghodFgkueR5ctsiJxFZ6hRQXOFA9CV2Dd5NHYjeUAiS0FuLHry3wyTwNjw1FtuycfxhucftTP6Hk6umKdIA7WmOvbfsEfG\\\\\\\/qKC8Q7wZ4mEDdQABGyBK45182\\\\\\\/3yFPQr3x1wYtN0LwfhPKYHYcIkrvYk\\\\\\\/y+38Ou\\\\\\\/QHFLoETpqvjOycFZQMJq+ZwzqYJ9VQOuiJHANWyuSVRSslCzIw2a5WUsOiqii77E6wgkWJqHtIlYjBO9kHrYUKN18tDwXO3QVLRoaXlXFdtTVDJ8nyqHKyrbMr+9sDXje\\\\\\\/rhyurQYlKNOPq8O0AM+BUoSn6Ba\\\\\\\/qfBcHH5Q4X6z8Yi0cPpr6e4TI1ORmIvCkrq6+TCIHVo1oJtHpZY05O1mSC6sk76USY\\\\\\\/V7j3s2NyrNpsLYBgKVsHNaQkYWQOwNvekOL\\\\\\\/lD3GyIcPOQGayK0\\\\\\\/XW21COl1iWGB7+Hu8I6XFWqTVlfkHNtQPz\\\\\\\/3\\\\\\\/OPAaG7QwMOT9bYyw4liVa1nTldbzYld4C1B49XjBNsqyIPSheEhhZTpzyMSx+URhLx1qoQd+N0iC42ELDzn+jcx6VudyFDzoKyfamU5zj8aXNLtwVarg54YcAo4AHX90maixM3bfo7rAnIGSuYDyWAau90U\\\\\\\/pvvf8ahDhA8VDsdN3GUJbLXFNIV9SOSxmhQiwVy6xlAys2BM79C4PZixC5\\\\\\\/aIQ5MTJVMpBFPhK1e68hwa5488cfk892Vh6w+l9+GFxy5t7EtH4zQqdpLbyVEChk+Ht1VvgrzLy+I2XUxXlTpkGVCuurl1oAIuCiw0ttBEpnPAGWfXoP2STZx4xGFcz3yabfPTk9f2G8kwuney7nxp7+HGie8Pg7vkhaG7iP1ZakPVN\\\\\\\/hvOTGIjPT+\\\\\\\/5kH\\\\\\\/47dW2FfvKbNfzdWTxW1TA6TM\\\\\\\/2Po+yQB+bALTehMTjHma1nMrLY7uUqWouxf0f77gm4AnR+HtniQ+EDmsDE6dQfCyIaslglcmAKdlv8edk2zm1AxVxvFNp1RQ6WJ1hu2ae1iOmRqFe6x3yngBth7H\\\\\\\/6g437c+xqET3N+sO96jzPmGs5u2Xvsae4sW6HHht6oqeH+thnonrtNuF9coxYdduvmPafLhzYCf8zGeeH0h\\\\\\\/LQD1Ez76zeXobG6EU0urwD1LfxQY3s9QtLSdfp2vGjKwHFFfnkxC99N\\\\\\\/09Ef+QGWucEXHE8kc7fhQLqVr2qWeSGtUzKB39LiG7rD7QCsBSC072aGte2A0WGiH0O8IhNmTuGhL99ZTGW8cuLV\\\\\\\/sgWBwnyA8\\\\\\\/ui\\\\\\\/t7GLw7DAhYnIb3mOzoaZM+XHN\\\\\\\/VIU6OltfIfmPXBnERLXcopuJlH6+PByvnN+Ai4taComewkj9qelzClPaRWCGnPNxl15P8mhiWLbFytTDcIrT\\\\\\\/YRxT5vK4fhQxsf8P5Xdkz23fbQymwAJZpq+S\\\\\\\/AHF7hdcFxg6ykgfYKRrcnjieOGev8YWtVc85erb9GJD\\\\\\\/H7awT+hjwuBVqqC2uzrp8t4BOw95ecEGJ07ovScnClVMZXQED\\\\\\\/88+5XDdoljosgKJ0x1M+exnNvOsr00\\\\\\\/xJrHK6M8dxUZWrCWUikOjCcEYOAa1CbMEJw3FX8GOUhlebC6WMXWsE7vfmtN9ImFNEhdvWsLDnueL9L+VZ0Qxd3Y7G4dG3bRhfaN2zrjt\\\\\\\/UeaS79dMtePs243yRCOpfyn9XvP8ABIHavNaHN2eb9UJxbSawUbJZj6hwR2p6xE0rTgFkZFghGjDFKbE6zXdmWO3EnQ7\\\\\\\/Wu0DUSRVI7ywOg5W9+55hrtkydkiSONGCuwCGnpzjFOt9Xw1tD5KE8F30SwTuEt2U8fa4NMbqxzov+i8CmKXexy5by\\\\\\\/yj0U63nPquWls0hBmx3SOFER7bFqjOxkneNevA9ZzAJTQZ59j\\\\\\\/+Xb76E\\\\\\\/a2KhuMv8KpJ7p64bTNqQsZFQT98sYlZ9zhxeVWYMe\\\\\\\/dsKzR4w++xocAVDS3rMk+7wGTmUtRUHXWPIA8oyM7V8qhPzFL0Np+1SiR\\\\\\\/Cbp0kYCQrmDnNiELBOC8HWW93kPR+yI064uN+Cqzi4JTGFngWP93PgVlYxhjf\\\\\\\/6qsQLAwMGH9KjYWsceaQki9GNwDKJcK1sCKR3R6M+\\\\\\\/5AxGX4WuohM+9tdGiVvDkKG4demEwbMk4tDogNvMk3N8o725N8VJX\\\\\\\/DDBagQryW9GCKpb16esRYsE6AdbD\\\\\\\/QF2vRucOxqn\\\\\\\/vSVb1z\\\\\\\/u3A5PJBWzsee+oil4lltQRpSsw+tInHeSuAvQsI6cXYyi8JALJIhUlvTD\\\\\\\/3l5wEPod822ZTOhcNeWQ\\\\\\\/wvxQ8zPzVR2QiWpOmfqe8uwqftBWlIfNyPQOT30KMbgB3jDuhbYL2alkCjcyfOXzzf9gbhVC5XZ6m6MARRbdnchMKG0kJIio\\\\\\\/iShD4n+PqicmIy\\\\\\\/g2QLyJuzHBe9HbyT6O8NuV65\\\\\\\/9ZuvXGvgW3SHp7xo+WXGncVFEuoZdWdtzteMNKeScem6oUnzkB6doVsGoUI+e7vl4y\\\\\\\/m+zDvvrrbpRcSMHwC0qWsrtbzXx+mwYImK+oYokbrbJFaJVH4dW3jsZnjAMo9tdGj373C74fvLj0JS5c65\\\\\\\/LQvlfCfLFJ0hec2QbYEKFGq3CmslNZ5K7L5oQmY3nc7MzwZVDUqz8dyC4klUgZ3AiUVw65LpoC6Jz9kwOX3fqNTxQUxESQhf2IFUcIK4qSEz14Z2I0JaZYvUkk6R61c8cHHBTX9IKRikdjMW3sLolJCzYnPVl\\\\\\\/8Rg6kB8D6E21Es09P5On3zaEoYuYnshXHT6ledyEKUVwiLqZlSjXyh6WPDWBin7DUssGE7FHR5t2aEmPNYnjDve6\\\\\\\/sY09JYgcAh3ZeeCPIcBJgBlRe4SOneOyuukfFHJy6Z5noVU4MnC3L6TtHOlH\\\\\\\/vVW57Q9\\\\\\\/knyJySyOeCch44zlyEHBFdk0+CL6zEamXKsT\\\\\\\/XtIddmzQWPRdhyJjuIQ4QEEVMSQcnYSR9TiaSTmXWGZTAzoTdn7PI9J4rRJKxJOW2Eweu\\\\\\\/JdHmWTVHE8RA5mkrVTGRJbm4tknkINPeqXA5v5diIPOsJBnQMFHLc4Md7Otv5\\\\\\\/FlNa82fU2LKIrkX8F4tISTdJZ1plCK2OcJ2e4uyAcdXLMg0SlHvGNjEEOveKGg3vSmBqVC6Ix2ff\\\\\\\/5nvKYERTioiGsedc8pIdQzCzOSIl3RWGegA8U1pnVAo+rURcDB\\\\\\\/FqxsYf6xqFvxz0dxbsueqhHxjamSpuj8T6rl5XCfhuq3RD4ath4AcKt9xuEIhr+9xinhpK2n3blBWmRpr\\\\\\\/ZF71GfI7lLbCESTDdL+Pt\\\\\\\/bQqfk\\\\\\\/vPvT8+TvoWdTMjlwNdG6nLxsOK1h6PdVDL+kIxjbm+iKIn8LofxCfPab7CpDpnPUwp7OxHnY1Z3q8l88atK3P+mVmSpOY3n98Lb+GbiZgxkdJbtX5dlhCRW7OtKo1hf1oB9hohVie8z5QVHhbHxpXIQ58pvYkEjA3lhpgj67QBZ+QjVU0WFq+c7km+eEEJAx+zQU+M3EE3cfMeDiMhsdCg8Zl0RsI0zEu98f8yxI+CrIDcscF\\\\\\\/9qSDGs5bnTT6OWkJ0NlCLOUT\\\\\\\/E7rctfQvrdHYVuqUmlyk8Zy+lk4XqemDK4DHOoJXiNzN1gYYaqLxqrzhgqh\\\\\\\/IELdGmsmCetCWcZLsTKZdEmYqZgTC78AlPH1r8Te2XgbkhQSS8JGS5nUOUcaNVTKrlbyrIP1dUbQgFuOEpVLykD5pAYDf4eUczYUsULmHktM95gSVVFCm++s\\\\\\\/A6WMhUS1HY04jFO0HXUG4rlw6qdoGPDf97VrC29Iu1FhZhy7vzHdk2Jiu0oq4IotkdU4WL7BQmUn5JjeE6zl4VN0sjSiAFwqZi8cv4OqQOu5+KzZUF+iRbvPdHJlp5x8X46ANTQv+luTdi52xxUa6Mod7j\\\\\\\/fmmCp+4s5gMJxzQtVJiViKAMmLNYHoqX6Q63X9\\\\\\\/16\\\\\\\/DXDU\\\\\\\/SSLSfyigQUyHQnoQaB5Ow6s8dik7LnU1HH7DGd\\\\\\\/SdtnUsmI++5tZnhfpjbN7ge3DPxSdmUOYGmlnnJvHuXrNDKYjJReA4EEM+mkfoUImkTjrhP8v1qyAyuzNic1KxngJGkn5mdK5tg12arLPfTg+4LqsHBVKLqa8mwq\\\\\\\/kbYHdckbZTQ8rgAfo36XiGlspVd+nQcIFp1PfRAXqj8ia4XvU0FJVfDatZ1EJUxBNDGzI3tquOhZdVyM+vel9CKj75gmmFnu\\\\\\\/4+3y61h1UBlVtUqgvXcXW7XsW3H5Khukf6jisG5KlsAUPxER+BC5kUNrI12azsZfwoIt0SuOr5Xgu+tpKHWFZ\\\\\\\/+O2qK66vJSkiHTytrNubyZBuvBZUnnAO83+9KVJJGSg9M4fv9jPYfcWSURjsVUdX6WIWbcMoSOhkgjMha1GK6KJVPtBVpd6L2ElxnMckgqb2XZEN2pfy3ijFRHX4QV5opypcoOV5zzS09sVKIb\\\\\\\/aIbysDkWtp7pkHBBmHroEalCJzdxJ7MeqZIIyTzbH\\\\\\\/vGjzEU7xa6kYINrBTWkmN9nZfvG2Fw59uze7djnR4hn01ASTKQAfra6YHLEhsUChmQG3kdXL7fn+htmJZp97K2loiEzOsVPK74bJIlVPLuXDkU4deCDnu2VnTOMeqSG7t+ymxTbdbdzV275s0bOcRbdxsup5iyt3tOAq0kO9k4jrhl7+hn+yIYojirkyvemth7Ze7\\\\\\\/6KYVjIkGSdPPOQBa8537yKTzLrO1be7IW5PZ1uGPLqaXODovOioE93EpviHavjFlKnbcNsLyD2nh2khF9Miv8RbezqY3N\\\\\\\/WVQi5USXP511hd05cDv+PycOci6CmELncQ2VuqF4aN1ogPAPZgJYHZgvjVbKdHW\\\\\\\/UGduhi4+MYa\\\\\\\/14gRb\\\\\\\/0qceG34nwGdeYyhM8W9RBBC8vY4NzdPYgSnCo3LpWH5GErMM93K3VOBTsuWB4PRgmoRZ3pEeRLlFyltj+ZmS6XR2fs5GWPsgESXL7fBDcA1Ik58dz04n+ZQBbFqjkmcEuwTqbhVz1O1FPVnqWBVc3b6JF9EJLBaQwpR+Hz1HToXRAnSvMaJ2CHocMHp7qeFuV8Hua2\\\\\\\/2mE\\\\\\\/99QoqKgzjlNsWtTEAwAwMEm0QH2Xbczw2+CI0ylInqQevtf2lyhxnOb\\\\\\\/Z65MXABvMdc2\\\\\\\/JIUE73q4hIRjHtXCtPbkVJW7ic5ohsXUutwYOq\\\\\\\/vkorokCN7W6JfFvX\\\\\\\/TU53ZtSoePfOs3Tt1DQn8Ij3a1x0bxH4ntqE1zQlxnVT0AqG5Wqbz4TVeqFHgvTIsLHe9xrVDitll0GiYUIeozqvTHWYbPK451P7ExvV3fgpOKGmcFVqrfF+IDex0jQ3Q9Wj\\\\\\\/jlDMxcYh72c1FXqFWTS\\\\\\\/TZ2EburuZv7HZ9EDIxsiJzmrMW9tmjH1j9Nl\\\\\\\/46wSRi6KkWMy73M7DybkVKHF7n1Ovqym1OKSNirxk\\\\\\\/Wcnjo51S9RQOxo0\\\\\\\/B0wYxjSSdGchUszEqP77EbPqRIYqcfCxKjEDeucu48uFJ8fhQEQFbxqPAydIT7jUGjfBLMYpbqa2dF3Ow7BfPzR8dNahGtvINCMlS44mNX+BBc3PJwJrZfwaao+xlL2iZU+NnEuk\\\\\\\/KRlUdMxwXZL7FPnmWLd6XAr7ROhWpmV3R\\\\\\\/b\\\\\\\/yWjwgkkw\\\\\\\/ZfpfP0muHG7ZUM2jkGem26GF6NenFiFolHGmhFVd9cE0E0ValP2LNrySbj6HR\\\\\\\/HF06HD5vpXbfZqfZCTaAi1kYzxVjm2KVGYvhhNduUZRFBfhRqNQ\\\\\\\/ZQHzWJN+lH+l1yjNAVnjGlbWM62qtIuwmfPdnczUu06JQ8IGK3vUFsABfZCAmmj52VuiMcoM9bWJEhYTCD2WDiaCmwd9pkcITTCajPYUMFbr9lo5T06Dg3bFSTUYvLvVq6nk+cZeZQuVPMMF8eeYCYNO+LjDPZUEtKaVwJId5s\\\\\\\/VYzF8dT8elh0NdNOErIUpWAMin9VmAGo3AlJbKA7LZamoMOdrTDiz5c8yDlaxOnRg\\\\\\\/IAhczZwjGpZ8dvZagyrGw7ut4NvrOEznCVFJn18bSaShve43YtyVe6JGwUU2AXFlClcZ1ewxSAXCevg9o74mi5t9y5UnpcfHDMpp+GbRo\\\\\\\/s7KGNWJKvTcB8q9VWyV+\\\\\\\/DmZiGcIhkuadrGw0HHcl09YoNgtULfPQvADWs3rC7dmgqQ0BNXqWSjxDh92yIPXEp1zVdpW2YZE0M7rAGtWWNwkJbWLmmccaZiMV+125KM1Zuq9KioZnqBngDLjkxi0KCxghhTvRqeUPyTUq7KIkBUTUMHiczegxtPZaOhq5OA0bzbEu6pPK1+FxHqCqRNb40fZGGa5obsNPQwnUMlKZG87PLZptovgM+LQYVrRMQSNS0E4RNzQvQhGyCQrD965axS9Vm+qChPiwY2oK2R1taaCMz0eNGhLhxgPBD9BE70SGKHwP5JdpCJkcf2x8oBBamv4M9OjZMJHh0B2aqNGqQOS9qs+TDledN56USV+75YIfq4msdG6hfnjr\\\\\\\/2l1L85B6CFEDrYU6kz54pKeZ8daQOfdci4fWkhucOsDnAr0tyypL5n3pC93hFVRHN9IW8VZZ1IUAZK6bHDuPRcLtuF\\\\\\\/5uKk6+TaoEcGovaRmWnK7UskacKhHd4r6DGLDRFEGAq7YWIiUzFoIxhJXXSAqLLEqsIWcUrp+wLYdUNpeiTUovO30RLfmGlzelIusXmjmcQWFTr8+DFiTwFYwzqS87ZWloZhz67+lX7pxk5C9zrQlMTRQjVRQrccvMNnxAMiUNW2B7WkDddX1WzlLLcZQVRuAsAPJHD3qa5oAGFsBWIMAB6s9lbuF0bF+Rd2ivjWlbPTJCQ2TJmP2DGj9k5Ztzjglz6KybXWeChZJcgEaTgafaR\\\\\\\/KO7rLVNz9I12kbba\\\\\\\/pnSkkTsf4NwXe2OFLf+9J0VGSwpKObP076MnpQM5wqSsbPabcGCORqoyTcWnBb2bVOn8bm+shX2RTfVVkIHwvTUIGmD4jH+dgXFP3M3erPLYdeJ04QDFH6ygsjws17WtY4yaCJz6CIISKNcqX+m5deQwuLlKhGyZ5czd\\\\\\\/05AkHnohVQJNn73avixOr460qdH1EbUHKaa1FhKUFXzVBfUw2Q0d4w9VjjbYv76AzusWyleopuHKrKgC3DRPeacbwRRWoGHeZ+t2LcEx3ElKKXkwbRXX0T50fTGBVOEE\\\\\\\/j6uXRjwLyTfk3WAwm0QpQGV09fVh47\\\\\\\/97564RlO56qr8fK+QFGPmqzJQyJBQfr1fx1ZoqDHbIROJQtDUPQsx6QxQFQytPNEdAxz4icq7PeLUwTwQLLOLBfYs0OO5Kb5QMlyMFtm6je\\\\\\\/XKaTcITnysYH98usiOMJBW+DxPa7Z2omTCwtuZ3y0FDI4iafZmFCaqkCE6cyQwETGcPWaI5EY2LYryO4nxDYacUPnaY6C4ixWJlaWOB5JonqAA0Rzw8y5iG3VKYsR9hjlwycDoXgTbI4DiyXKbF7rvcdH6zxzy6RfeLpc8lHMSTRD4VcgrTw+PdzgCv+Sz\\\\\\\/GUWHctOfpISTDECHWrKfvWy1UsILqu4o07q8Y\\\\\\\/6wpZ0gsmmTSG\\\\\\\/l+nPsae+B8jEm+2DH\\\\\\\/6Iz\\\\\\\/4pupaIaI8JsCSZmY4YQyIQq91ydZ7EeCklq3d7xyjlzvwFkrwbwv27quH5ppq8vZtkjMPoKYBsIV19nnkgjWp\\\\\\\/pcYKufIb61rtcUQvAQJn3m\\\\\\\/DqIu07H2a5XEjyLPxFa8FXaPZF2LP93qGvDCIlEE5S1bS3bksJThXFo435WdoXHUX+PU73f03DQvScDsJp\\\\\\\/beRETY1X386+A2URIHbzDmyoVSZztuv8OfWBK9REZ5iQ8dKo1xo1wAg3pr3FTza+oc1pYXJXN+x2b55+lNHTstj\\\\\\\/Zs3qIeMqZxE6zKpRmWeeoJikmg2xfoBHQsmom8H5pmVo5BSkRvVXxvubGhPmj5XEossfevN5v9oQcF5Zth5\\\\\\\/qWEdZyKV+PBy0eVOlDNDZ9duaWgBvlVyP269ZnDHpUe9d710rRAYw2MxJgMGZGpcKeZ1uDou8j+CIm8M2R4IiUwI+NnduPrngKFIl4krR2Yy\\\\\\\/ycYhiA0NGRbHliUAImv6vagBw4LWhDVsav83FrvNAAv4wgDZcqhveJSGdauqVh0PAlzj7KGwkyHGgpl9jXmk+RZPCLJd3AFFUdThIg0errn4GPq0\\\\\\\/MiaSH\\\\\\\/aDI\\\\\\\/YCGL1dyzxnBUGtyJ1a4yAKNFW3yAWelXvhHx2Em\\\\\\\/A24Y\\\\\\\/IBjY\\\\\\\/DB2TGEa\\\\\\\/mEjXVBRD0QObKMgQIUilKPHQY45yWQQTSGClLYYQcaQZfS2X9YXC5cTzSE81Tit9ck90s3MkHkm\\\\\\\/h4MfW9VIzdBQzyCJJELniGzriFOd4OeDzjdDJNri+JfcdC\\\\\\\/F74VjCRLhO4MlBsfWvEcXURgLLzbA89qbO480nMpSR6\\\\\\\/l95hiXALVrh868G2XIC2GOIkEYzqMc9vO7iJl5NGHJGiJWCW92ynRhT4JlJxlZoyqDgN6V\\\\\\\/Kw\\\\\\\/Q8tezLdK5lymOUzMCLNYw6JT2SvkgVyC3Dwc8zwHeIKFpcG\\\\\\\/lKWrKwiNj3zEgQ3Znrgyw5uqaxgDJXjJFQ\\\\\\\/8UjxQd1Lp\\\\\\\/xPFME+DMDbOeedEnHJ6\\\\\\\/hAtnPkCKo4ER\\\\\\\/WmxQp8rLugVcDvTJPWGtbNR7XZDKd5VovxXRuhTaqgkC5GZAwtNw8VGaf3YhDDeNGlhg7NVgoBENVfd6mBJKUIE3sqZpVJ2WLC\\\\\\\/ILIcatyHXBPLysmiY0\\\\\\\/7VCSqrtIiKcIl7rN2D\\\\\\\/+nRTJwL1\\\\\\\/xmyaxE1Ls1qfEca7I5ijAWrOd9nWwD0LH23xDErVV8n1s3B0OKUjnHip5xYx7wxSk1FCni2xeiZScGCIHwktR4pTb4xReitY0I7KUTIuttuVD8Qck3WMuq2te2sKmvlfgx3FHPvYoNUkw0Ek0qa6e5oBTO31rTMzSzg+xUqoso6Phv\\\\\\\/FymzwtDXIx5sAM2QLw2RWTLdUkSRG6BFe3VfOKXJyw06Y24aD+3xPmdRGTYidtI6hL5khcZM7ET035M3V4dOH9\\\\\\\/VspbCLJyrImz2WPyEdzc80v20u82kjZalSjR7fZsaUL\\\\\\\/SPhy0pVjMeIdBLDTFrgF21AfyBZibCCYTJs43O7jCcKOIyU+CsC5l1IchVANGB5yy5SacBvwLLaQDHvpDwfpZE4soIc50uf\\\\\\\/+uqUoPe7JrG61YKEhmKuq\\\\\\\/Kssk4IrvSVt8FBLSS2t9lmgrfCZgqTxbPd9aVlDmslojdj7Cu5AwJQAIddy3jc3bX61+7UkEtlXPdeIg1nn0Pt3ZVao13TJiWD8jGRcsFHCSrWIPyovdtQFi7CAw8IlcLW+pNrJWSWSfyD1Eu4BIe1S0fx0hjq0rycA7eSSY6z5f37zNKNdEsQEUP6eVyxao7aP5xoMLAz7n0vSaMQqW9B+kcBNuLp+9goUYvGYxIfhWzai9LoUe3RsjZlK05U8T4hxEl4sV+1nm3x5GI+CJcyOXsn0sML+npvtVN5Fq60oYupy6ueUojGSmR+qVSmzDav\\\\\\\/LqZsIrJnEVA1PpR5Kcy1nUmR0oaynG45O3fox5zep6OCoRPRpd2X7pJhOeglpQmEWweAJ4tun3GHxkLHOWQ67MQ5ISGgAqOnGKV8uKe5wF\\\\\\\/zUwIA8yt8fghoNARznnG7P6eLQVeOuLx52Z0BDTwfxRDXH8FE\\\\\\\/753k5zQC2URhRDLhOJEClfQC\\\\\\\/7dcdwjzoBj65ZVzb5SGTcVnLybAekJflehTVaTxTklGfDhuZX\\\\\\\/zP\\\\\\\/Plpnlq2BlnaUaxRk3aexs8MdSOZTGdGSPCVxS\\\\\\\/ttBtqiVRvmMLhRrcdKnbXcM59DZ9tXKSrQmlNg62sUhHKvc0DkzRb4PGdOJ1HZCLRQU2OeyiwLPbTM2yXKdqCEDaySYACfdnfYDoAihXWqqWq86v1GgG0JxOcWtfDAom6CfQL7NRgS5bphf2YA5RniRMyrkV42SLv0Ivwm1vZPKFxVUPaw5hcELiQWLsPjndC1MtKEjr7sfpEsuop5XzbmwyODdG8T5s0uf48ZEZGTInRPolSsCHU5C82Aie4elONiZSiWcDoBj9fNshdBg3HNF89FFP8dzM+GbQuli3FWrXo7zrIFc7NJp6Itk4ZG1tJ2PCwHO9dEJ2FFHc3tMU6sktwpBgwg\\\\\\\/dBA4X3UzLXuqw7Y\\\\\\\/WWRYx1nBc6vBbwpq4CaczbBXncp\\\\\\\/p\\\\\\\/192NYIW8NkozZEwcY2H7Wod8mWFZnD556uhMj2m9in5SGu8EhVgfp\\\\\\\/RsMFMrnTPVRF0FYfAm7vOzJfDiIJpVL3HM+D7MRT6RsUOmKioMiE2zPKk8w5qsC1FEVK19T02dH81VYRCAdon4FJ1xME9XyYpe6ya+wccFd4HdqOxsClAIuqhQsZfCe02OccCQvI7L1fkwPKYMp3q6oUVYzS8Ch42S6sgEmeExAQVKXIeAergc4L4DmP5ERc3g3fJNe5zvk9IE5RzBfs3YDGYLmJSiMaHioZXNfT8sgtfFsvFMKxl8xmKNBfx6ivm3dB4SaOJOgDzjuyPK7QdMTuXLFeBjOYkeIdZQZ7x8G+i8trlg1psmER6jk41rZlA+8cdKanFLAj1ZKSNCGMSziKzAl\\\\\\\/xwSHLHwcvaKHA42jAQ6YniitMwgz4rJdkcNA\\\\\\\/dbArWMwu2+TBy0DtI1+4MdEMCFFDzLctjuzfycQLyFw5jKZh\\\\\\\/sFbTeismIB9MRnMGE7lun1DjKMvNDmQbXgxUJ9SG5OvqcpX5ERvDjaPM+C3FlsLiF9FN3rfUR0P0lj1fvdLZmHoctnvL8S4Tvb55e0rawROl11Un0u5hyvb9B6StC8oPrVAmYDlNcFTioRaoVCiVtsTkJ4CA2MHQyhbmNpaAkPQ3h\\\\\\\/QzIFoFrSWKK3wK2KDM9mlyvGRS8ByqnLS8NXI53ZaiJIQzVACturg3cMEiZzcoN2zt49uWTXOF16PMuZxCJcBWdCicLd0+Az354+dBCgyZRA0aPNUQEZBSLGC\\\\\\\/SYeG4EuIB0MWdK7nABuzHmptoVPNjXppSG113\\\\\\\/EjF3SUU2pduDI\\\\\\\/hWa6+eJcx0h9TdtWqEeWxHDHE1vN0\\\\\\\/CULw6wzGH1lLniKMMA+M8beRUaWkwLD8d\\\\\\\/2yVQUoOvdvcCxuVJGdna2RV1FXLR5U\\\\\\\/pajm5tM0ZTmVWtPLWiCEzGiXC0supe\\\\\\\/DQXm+3AIpfhV8MPOvSAUMeP3GPSKiEWHz5HwMZrwtqlxAt9UxYtSVDSn4CXn70wazjJrxCUnmZBS42jmaU\\\\\\\/kAwA4UHtQPKbtbLv2uMjb1X7Rs2DuFHelj8fXn3y8X1VH9rEC4XdPOP38vN5VO3g00bJAqE5jS1NP3+HsYjKE3YddhzHyk2jjij8kftpDioZyQXapnhFCT4g1jOgs6oG\\\\\\\/CXIzLLW1MiUXcPzYkmHsPxQcKqzNRsd99PdeH4zPAkq9bJZTQJcXL6aS5rH3c+FInvTsoqhnwcL8zPqU1eEqYBPTtv5D6YEdtwFqgF+6rHCi0HaLFfNhjHUVoI328a5ZHzOTIBh9oghmYnt86h9adI+iRacDeWMDsACbko8+cl74d+VY+H3PDCl43+9uxJtlNH\\\\\\\/gDnooQN\\\\\\\/VPF2Dh64MGj8m454wBdvgQZzv4ayyYTSITnQsE3pyZL1yyxyjCtSa3v8qQYQmJycMkHGExLRSUzrcGi+KW0Em\\\\\\\/Oft34OzroSV6glHFQSFSOlKGRISFzUPlewgvWcaePVmUiUmjDAUFLy2Q4lhuoYo1IHTnyBh1kJeKpmdBv42kZn5k41xZcSzHFOrYKbe5wrgoxNpYoBT6bjHz9KCGJvYLEXILOkwBry2kW9kZzOCnVXIp6taVDD3q20DaS3WLgFNDBa8ahfBuiviN7jfjMy\\\\\\\/\\\\\\\/kO7QQub1JNm\\\\\\\/NXgEysl5qopJDiXhL4arsgbDOegyBCSXMHeoeIW6nnujtQ07yFnUFR9mmmFNfKC\\\\\\\/\\\\\\\/TQcd8UuhwMi5yn7ntCvzyJClmlrwZVgfkyNRU1dRYSnKM9pjoOPZUHKoL7HEzXdGp2d8RK+roxn24SNz9dXDnRUH3cdsmLhdl5Rmvu5yVdMdFd8I9dyO08UTbX\\\\\\\/CwrjZLJxSXfs4Nh\\\\\\\/e+\\\\\\\/VL48Tl9RPvm59AuhbIYH1jgTXXGrVu+iF8YIIqUAtC1G4B2PhoMzz+GGb27yDU2IaJ\\\\\\\/VBp+SwJupJUWjqOExo\\\\\\\/FBnThZ+0zAMNBrxKbaNeuwlp6Yf7xcdhWnLW2IhIJVZxosCKCuhUhvMkT01VueiLlR55D30G96ajnN6iVWL4fSff3pZRfxw5odLja+atGGStiZRE9T\\\\\\\/fUNDSKPGnTkBRUpB3x5KgLXJ4C90hHP2XH1WpB4eZ1+CdHedcolnHCUaK+Xwe1mTeHysreH76PUf\\\\\\\/70vvl5cIJA72GyQFW26SGK5CNFuRsv9H6\\\\\\\/04S9Ro7WMCjJd2sXdXKPARiaWS6pDnU6JTSzATbHZBf0Ji5HgWNHpCsanerWo\\\\\\\/DNnAzr9vCdN\\\\\\\/mlTIRuG2u8Xn7xTDi1dZMjdpIv1BGpVB9BoXAwkGacdgv9qGb677ymzHFuu+t2ez9stCvg8icWYinVFfvo8RvwxkdfwzmmZBFqQiHgymgXlSkiAyKvaOAaA0CjwI13EDODFW2u\\\\\\\/ThNY4VqcSREV\\\\\\\/jR\\\\\\\/2Jj+Otsjh+P4lDOhxUiKHRRrn0uHwrfdta8BUnr4f+ZV0K5CrWmigYaPMfpk\\\\\\\/2kxWVWEXFJzN\\\\\\\/gKR+ibqG4q9c13orS\\\\\\\/PyFID7JQKnndqDZOryt55EjoO8kxRECaxqqfSqWS7De66SNSNtbCAqcfDi04emMwoDAQWG5FtQgs8SXCy8F+A01vtzQhofIrAU7lZJSG7oiCSHdFYDQ8ureuWZfyj8eisdy6auLJtH\\\\\\\/xPReO4Sdg9wtnw8BSY4b1QMtGOAEOIBRjFzpt\\\\\\\/CBRvKvqW+9t3\\\\\\\/NWK0bW\\\\\\\/4MIGqx8i3isdW3uNSUn8d7fPDKeICd1\\\\\\\/3wbCsQhzBbDRNCEDTGbd9OkUXf8Fk22cKQz4hWw0t2b3DAKv1Kh\\\\\\\/FlM+4sQXcgSZYtaziAHHf0o\\\\\\\/s4xygFUKjZem6IfdbGHcy9Da6PTszs8Z5\\\\\\\/X9cS7bWXxoQFpb0AZsR8NHLc1FoG1uYkzIQJxC+LaHe1ziUIGTNEy0SZ9UuisaLqB4oiiJqxtqp6uKb4OoZbXNrVj1HJ9wPTZ2DFFskQoAlB+HwNOA0Fz7NDTvnMc2EIo+ffuI7Hzz\\\\\\\/iLp6Vakfm\\\\\\\/zYAfU2ot7iaWpAyDyaaCsYO82Llu\\\\\\\/peRHKnMC1YBxaZktemOeKKli8rqBGasXfSu\\\\\\\/p9pMcRhYklVLbBJ9ToEE59TVAM1fyEAgl0qWOErCD4tMPvRk0gTytsK23\\\\\\\/96U1JsE5XPqXaG0vx+pDEcQzUVU2cbhAYsnCmh+s8OlfYADTZ+5Xy9OGY9uKzxCzXh0diXQGAhDqpZMWB+tKlXx7I3sGSzs2c9E6Iiz3kvUJNJd1HJYxTiKVsTR0M4GUtDyjjufeDbSKyLWxJi8VlgpeU3csZfrXtduPhGac0jsQcbb6bddqnFCsBkkF0MqSyfPpRYv4g7aXND+jIbr8sDcv48poh5RyFJ3rJC5yv6NFql2woMLvxMUrrY4XeFO4K39xeRnHekcAicUjlRn\\\\\\\/Wis3KCsBUpMUmGTpRGEBKZqb1Qmd1uGb7mimNMlaNgLNoyYk35cmf57hDau+RcT\\\\\\\/uLyWDFZkK7S0PPnGv2rA5uqFvQhozCqgdeK8QPEoH4ua99C\\\\\\\/cyFIRuwyWau6YTXBvgfcas8JfujE16Pt2HeEmXhP8qFTv36kblowCS9VNOg2+HTAnDBgX5b51oIqKHSuZcAOfRl9oyN5TUQY4rKXd2OX4ERf8tTGkGJI1WOyAI9i3uhNlSuVzCc7OYTuD5AyugB01gDOgTH1Q2b8PEI41kMBOdhmLyGA6Ve+sjUyuZBHVNSwmqMJL9dQA3DQBMZRCm9letPnVo7Gn\\\\\\\/S3eMOzEZFlSz9fgTF9fjWiYuh2I1DAs\\\\\\\/w1HWoFpxQ4YMQjZIITwyLIperbBmeyFdH+NRt92dfTuT6496x8JRtkKjhVbn5xU6408QICgHpO8nmN7hKwgOIBAU9hKGnqFIAg9+AZbJmrMjFcz1AUXHCfgaAM0V\\\\\\\/dtTEHCVpMhgtdTykt+ufr+nTG+6a2K6RZvCAFXXTEpjlorCA93119EEk8J2\\\\\\\/yd+n9JXZySvHN7AnNA5dfempzoIFjFf+Le9ivWimq5FyE4gGKYd43OR0ubwyI8+euhBNVUMjnNrriF6NnGmI4JJQHdSsj6mBYbWZMBQdRZxIqRL858KgnSdGzUz39ICvSj\\\\\\\/\\\\\\\/gxeWAQ9GpMU8jpS51TN91bxovx\\\\\\\/fYR7la42BFOcKwpuMMbhjf+sNzmx7a5WZWDY4XhPP\\\\\\\/oPLnnGCeQJu8oe5KtqXIC0AGyk7lBexUwl5Y\\\\\\\/Oi6mgzkwJu4FAF+t7YYrzQIGdPvuiJMINi9oKe37OBV4zp2Fe4Hx+r9xgEUD89mHaFtyy9hUnjKockdZQ8r5fHKJhr6S7RhvcM0sUa\\\\\\\/LiSoioNzW7RSZBJh2WOCsqMpGgTmuJei5iq2dVEj7xaQ9lbvPL5UZPBz9IeeiSOz8sWOEigXfvtaanizh0q+qstfbQDLUJD6h7oB9twuJM3\\\\\\\/QBBEetZoxsdgE\\\\\\\/mXcWXNZxP26J5oMEBunDTSw5C+jOMjd20ZV1nK6nHLCiEb5u1A24Q\\\\\\\/WzqpfLYIYx90jM03deBYxOpf3WrTizNWRiRBzHswwJxVPRSyMV\\\\\\\/WI3A\\\\\\\/uxzt\\\\\\\/BlVRlTeAL2SvNIWXhddQkbMm+wALUgF\\\\\\\/gBVZarxfHyLwAu+SyLlui5f8fQI\\\\\\\/Ewmi+DiTXRltupMF4Ck2kFjkwkozMm3AQ6QypZhTLw0wzDFK6WhZVAwYTN8SjM7UmPypWcywfepRjgwB\\\\\\\/g2YFT+jQw3GzwdmKCdohLfDA2\\\\\\\/wOrx91CrJyfiI8MtQtxUpObjZlkLSrf2450WKcFgCttGFjK8fDwPKlK9zKiW44Y5guB+iHg+ExMB6xdSi1acbvMk1bpIqfkGH1jL2x4R3ph5gGnpiZ\\\\\\\/\\\\\\\/BXAXOX4W+8CmEBti7GkBQZN+oT4vb9UlD+zDzE0tpHqe43D8fzxUBqT5IAx1n7KC8+qdCSIgW2lkS\\\\\\\/HlzbuO2CiEwvIpvdNY2cYDizmHGip3hAIbkXmG4W6olWtRf+OYBJ17bm+wmAFlLktMDNJpRcpsPe0tDjbZnxt87HM5Oa+OI7HslxJ+PxEkRbRAjPpTj2WTTVpjRhvsYwedAT7n1U3QxV7gEhlAgh7zOAkBWDHIPX5fCHuLYFda3fqVta0Qp1p51XBGrUeDiKc40SDtIA0T5penliRDwGwa3n8RdnjIECsHMlXcobRYgr0U8MXXZmsv9pTIY0eHLFDhHjhFNdvAV0PEoMK4TbgBXSk5tT18v+VGwoRpK7rhWc5oFdpySmg\\\\\\\/7YqYId98lwnGuqRierqY0Zy8ugmWwRBGYMJTP9p3j7Y6xQjQAB87MeT6nDNj9FxlYLoA2zgD5u+oT3Y5G6Nqt9L9fdc3k7cB71hJVByWDvzhI03TcwDBvc83uciEUeNsxmzPmkJLjaSSrckRTIIHrRylLt0JlvtFa5+BM741h+gC4tGZSuJpM2pFfKHfgoPBPyv3rMZ3nY58xU9lYOkcQuEx3\\\\\\\/F+w9kEluWk0lld7N6aAaYWuf2OpunHzDqyEvWWAc1PPmgWUiUKvNUUfuhPAv+teAD\\\\\\\/7kk9E9I5IxzlAy4SCDR44lQeSlfX6LbubwUcl4SHUJddzZQ5FPz9X9zyIPWmsm0D4WGeMxuwL\\\\\\\/u2fqVfFSXL4TPkLWi2z6biORPWpj8Ej99hlNEnQt9pzNuGXe\\\\\\\/32Z402Kii2ZyGzn7rFQd0g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022egg6wlFSEdeOIqz8fMqMmQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f3ecc8eb9c7d12461419bd054c4bcfd71e7f9627881bcc88c6043c45f4f3f97d\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022cFB1afyr261EiUz5v9XCKkwc4i6eNWnt2dzwYYRR9e3NOI7dBIcfQpKu2d57\\\\\\\/yWjd9x6V\\\\\\\/9ZdVA5fUrIaTvqfEPfzTnr4SBQF3sjn2jzqxOlKdsn0EsvHKDF86tNSep5qwNowbEPIcXCH2x1HB4xlGFQ6BuGDLaXIG5QI4e2kcMGcVDIowHUiLCzv30CXTSPJqmMs4TJ71CxTmoaNd56XISSVYhqJE235VMGlb+x2NYASPOXIZXecn4qVxxhwPfIxjR8OFqqrsZl6yPXbvhXB31JPHyjFDihVL17iA7rsPyECCceBUH00vh9umxOB3hhw0GTb9Bn\\\\\\\/V07488j1Dmqc7KbFEjEWZ169107URwJ0Son89vWcgzqUOSa5fTF7Z9ZPS2UtssLGMUqP\\\\\\\/vcMa698eFMt\\\\\\\/R6lyyw+NWCoy8ITYIH8bAJ\\\\\\\/c\\\\\\\/g6PgH3dRoG9cRPuE21xSEey9kDZViDviC+UN5v4\\\\\\\/lQeIHpLe+83fv77fYLPfxalTvL08sF8s08pHv\\\\\\\/hFhCEiUW29GVK+RlAxwExNWmkFIfMcw9R9ZqMq\\\\\\\/3qK8CoXCsxU6HXmkdqFK07Srge9dFQOonSa\\\\\\\/aU\\\\\\\/LzZGn6h+KGOZb56CpU42DFd0CNfTLDW0NdTulTaljBg\\\\\\\/ivmGwAwrkOT6GRYcKriH7y99d2fMtzsXUQrUsziKtPPz\\\\\\\/4n8fsSSb57uBZ0AvueyZ+lGIeF0BHw\\\\\\\/GK6C\\\\\\\/1yse7HEA3iedRmYOa147cLtN\\\\\\\/ncIbCL64pexA370R5ZvUkhHBKN2cR1BFyEmxRKDhs2YqLLC32\\\\\\\/oyI38HDzg3fjMFwCiz2uZjE4vALZ7BPvK7z+HkChcY+M82sRbBb0lldgdxIDTlvfGtoypyDfFrWBSJnBrFhQ9c5xWA4QTSG5X7\\\\\\\/vaVKFDc2XIeKkalmYzxq9cNG6H050gyKGPFEznKulpr3eOJC1KNJdRYmIt6nVt0s+YgFck68uyZiyJ2zv\\\\\\\/\\\\\\\/u61FH1u1qJmJHM39EAzpr\\\\\\\/rCdGpWu7uCQAnlkixeV40OIc6+2L6+mFCCjFeQyjOWQeqXwVdXjoBifuSrKVqv0APQYzk6abvcMnhts2P2ysSJg\\\\\\\/U8vLp0jPUFr9bCGDznPfUKkEAHrPAmJ2pMGlf7WRADsEAG9WvxAsyklCztt\\\\\\\/uvw1XKtvNPndT6Yu6VXR9dlfV8JXR4DFKqnyL4b9i7lPVYSEPsy8S0gxLflGHjFTH529gp6C3dd\\\\\\\/vtq2s9fc2m6Mc5gJ+dRoZPgQVTrmsl\\\\\\\/Lj1dkaL5BLDFc2IeQRTlt\\\\\\\/z7tjtSW2I73Rw+k\\\\\\\/Qf6Rklk+qOJ9Bs0Cc5AOE9Xt4O+zXwaKjG+3535R7oEgOsPMnpElCin8PEkzqJ8aZpZRADE3HBiS3ppwQyWxmK8XCLDYw+hKFWhhJPuopFa7L4O4QjouQh4rcs4TUSj1ZDKKuPKRhiDzExhzY2i8lFUusbjVlogEzuXRWRivOLgYAb4nuXlecLFpQHJ2f4992lAqLGImE8YUVb40oHjheJsUDyRP0\\\\\\\/\\\\\\\/K2O1pZ6LnGL7XEt9ED7c8O+Sh0SMLh7JjdWda+W3J4QCwTdJXqqztvfpTnBX\\\\\\\/KWn7LJrOtupWmea6EaE3MACoMjxEUWA\\\\\\\/n+cODI+VkFJ6Ecf8UApL5AxmLZODHLLI6qK3HhVM4reSkI83oWkfeWwNO4\\\\\\\/Jgn1r\\\\\\\/sqgth2KUkCFLgxED6EQ4WPBMkUzPFVqaU11uIBAK8BG0LsMdGQYFj6UX8HyzS6xupa5+mmSxuwp1+d7O7kGCKB73FdLxGd3p3x00HccbjaD\\\\\\\/s+pclc9arfA94mnkLY6adK2BRGl9rjJn8Zg4xefTj\\\\\\\/NfD44eMVgbNK8o6lirE2oGy9ZuPqI2Q4VpH6giNpWBE3z4E4WkKlz9Mz753F1co5Vvek8+6C\\\\\\\/2EKNmavVjHhj37OMmRhD8O+kwsncI6rPLReysKiFFrFCiK\\\\\\\/21NgsT4fp2Ar9ObZcucdGB\\\\\\\/3\\\\\\\/AOChd2FKNlG95DNBgY5XUsTuLeUFH2FvJtOXm4XDmA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022r+6Gm63E6Zn+bokWQT\\\\\\\/PRg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e75310df669de188a546e2962371504134fb7c1630cc144394a2739348b44d2d\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459908 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/7/1339" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-t14YA9i7YsdB_EvsgOxYT_8Ss1fFVY3iipJPI9xqm54" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/7/1339" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-z66v40mL12NeE-IgmJB51wIW97ukxxziJKAmn7-rVKM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/7/1327" title="Genetic Control of Maize Shoot Apical Meristem Architecture" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/7/1347" title="Identification of a Novel Strong and Ubiquitous Promoter/Enhancer in the Silkworm Bombyx mori" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459908" id="node459908" data-pisa="ggg;4/7/1339" data-pisa-master="ggg;g3.114.011825" data-apath="/ggg/4/7/1339.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Species-Level Deconvolution of Metagenome Assemblies with Hi-C–Based Contact Probability Maps</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Joshua N. Burton</span>, <span class="highwire-citation-author" data-delta="1">Ivan Liachko</span>, <span class="highwire-citation-author" data-delta="2">Maitreya J. Dunham</span> and <span class="highwire-citation-author" data-delta="3">Jay Shendure</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1339-1346;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011825 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459908" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Joshua N. Burton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195-5065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BN.%2BBurton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burton%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoshua%2BN.%2BBurton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ivan Liachko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195-5065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ivan%2BLiachko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liachko%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIvan%2BLiachko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Maitreya J. Dunham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195-5065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maitreya%2BJ.%2BDunham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunham%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaitreya%2BJ.%2BDunham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:maitreya@uw.edu" class="" data-icon-position="" data-hide-link-title="0">maitreya@uw.edu</a>
<a href="mailto:shendure@uw.edu" class="" data-icon-position="" data-hide-link-title="0">shendure@uw.edu</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jay Shendure </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195-5065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jay%2BShendure%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shendure%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJay%2BShendure%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:maitreya@uw.edu" class="" data-icon-position="" data-hide-link-title="0">maitreya@uw.edu</a>
<a href="mailto:shendure@uw.edu" class="" data-icon-position="" data-hide-link-title="0">shendure@uw.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/7/1339" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459908" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459908/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/7/1339.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459908" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459908/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/7/1339.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459908" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459908/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/7/1339.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459908" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459908/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/7/1339.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459908" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459908/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Microbial communities consist of mixed populations of organisms, including unknown species in unknown abundances. These communities are often studied through metagenomic shotgun sequencing, but standard library construction methods remove long-range contiguity information; thus, shotgun sequencing and <em>de novo</em> assembly of a metagenome typically yield a collection of contigs that cannot readily be grouped by species. Methods for generating chromatin-level contact probability maps, <em>e.g.</em>, as generated by the Hi-C method, provide a signal of contiguity that is completely intracellular and contains both intrachromosomal and interchromosomal information. Here, we demonstrate how this signal can be exploited to reconstruct the individual genomes of microbial species present within a mixed sample. We apply this approach to two synthetic metagenome samples, successfully clustering the genome content of fungal, bacterial, and archaeal species with more than 99% agreement with published reference genomes. We also show that the Hi-C signal can secondarily be used to create scaffolded genome assemblies of individual eukaryotic species present within the microbial community, with higher levels of contiguity than some of the species’ published reference genomes.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/hi-c" class="hw-term hw-article-keyword hw-article-keyword-hi-c" rel="nofollow">Hi-C</a></li><li class="kwd"><a href="/keyword/metagenome-assembly" class="hw-term hw-article-keyword hw-article-keyword-metagenome-assembly" rel="nofollow">metagenome assembly</a></li><li class="kwd"><a href="/keyword/metagenomics" class="hw-term hw-article-keyword hw-article-keyword-metagenomics" rel="nofollow">metagenomics</a></li><li class="kwd"><a href="/keyword/clustering-algorithms" class="hw-term hw-article-keyword hw-article-keyword-clustering-algorithms" rel="nofollow">clustering algorithms</a></li></ul><p id="p-5">All ecosystems on this planet include communities of microbial organisms (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Howe <em>et al.</em> 2014</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Xin <em>et al.</em> 2009</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hug <em>et al.</em> 2013</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Venter <em>et al.</em> 2004</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Renouf <em>et al.</em> 2007</a>), including our own bodies (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Qin <em>et al.</em> 2010</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Huttenhower <em>et al.</em> 2012</a>). However, our understanding of microbial communities is limited by our ability to discern which microbial taxa they contain and how these taxa contribute to community-scale phenotypes. Most microbial taxa cannot be cultured independently of their native communities (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Rinke <em>et al.</em> 2013</a>) and therefore are not readily isolated for individual analysis, <em>e.g.</em>, by genome sequencing. Such unculturable taxa may be difficult to study even if they are abundant (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Iverson <em>et al.</em> 2012</a>). Consequently, many analyses of microbial communities must treat them as a single sample, for example, by shotgun sequencing of a metagenome (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Iverson <em>et al.</em> 2012</a>, <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Huttenhower <em>et al.</em> 2012</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Venter <em>et al.</em> 2004</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Howe <em>et al.</em> 2014</a>) or metatranscriptome (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Frias-Lopez <em>et al.</em> 2008</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">David <em>et al.</em> 2014</a>).</p><p id="p-6">A central challenge in analyzing a metagenome involves determining which sequence reads and/or sequence contigs originated from the same taxon (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Carr <em>et al.</em> 2013</a>). Many computational methods have been developed to deconvolute metagenomic assemblies by mapping reads or contigs to assembled microbial genomes (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Wood and Salzberg 2014</a>) or by analyzing base composition (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Saeed <em>et al.</em> 2012</a>) or gene abundance (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Hug <em>et al.</em> 2013</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Carr <em>et al.</em> 2013</a>). However, these strategies are handicapped by the remarkable variety of unculturable species in virtually all microbial communities and the fact that most of these species have not yet been sequenced in isolation (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Howe <em>et al.</em> 2014</a>). Individual microbial genomes have been deconvoluted from shotgun metagenome reads using methods such as mate-pair libraries (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Iverson <em>et al.</em> 2012</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Mitra <em>et al.</em> 2013</a>), lineage-specific probes (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Narasingarao <em>et al.</em> 2012</a>), single-cell sequencing (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Rinke <em>et al.</em> 2013</a>), neural networks (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dick <em>et al.</em> 2009</a>; <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Hug <em>et al.</em> 2013</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Sharon <em>et al.</em> 2013</a>), and differential coverage binning (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Sharon <em>et al.</em> 2013</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albertsen <em>et al.</em> 2013</a>). Some <em>de novo</em> assembly software has also been adapted to anticipate metagenomic shotgun sequence data (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Peng <em>et al.</em> 2012</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Namiki <em>et al.</em> 2012</a>). These methods have succeeded in isolating whole genomes from abundant organisms in some communities, but they are specific to the communities for which they have been devised and often require prior knowledge of the community’s composition (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Iverson <em>et al.</em> 2012</a>). Metagenomic analyses would benefit greatly from a more generalizable methodology that can identify the sequence content belonging to each taxon without any <em>a priori</em> knowledge of the genomes of these organisms, especially the genomes of low-abundance taxa. Related to the challenge of determining which contigs belong to the same species are the problems of how to further define and assemble the one or multiple chromosomes that comprise each species’ genome, and how to define and assign plasmid content to one or multiple species.</p><p id="p-7">To enable robust reconstruction of individual genomes from within a complex microbial community, additional information beyond standard shotgun sequencing libraries is required. We speculated that contact probability maps generated through chromosome conformation capture methods (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dekker <em>et al.</em> 2013</a>) might inform the species-level deconvolution of metagenome assemblies. One specific method for generating contact probability maps, Hi-C, uses proximity ligation and massively parallel sequencing to generate paired-end sequence reads that capture three-dimensional genomic interactions within a cell (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lieberman-Aiden <em>et al.</em> 2009</a>). We and others recently exploited the distance dependence of intrachromosomal interactions in Hi-C datasets to facilitate chromosome-scale <em>de novo</em> assembly of complex genomes (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Burton <em>et al.</em> 2013</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kaplan and Dekker 2013</a>). As an additional feature, because crosslinking occurs prior to cell lysis in the Hi-C protocol, each Hi-C interaction involves a pair of reads originating from within the same cell. We speculated that in the context of heterogeneous cell populations (<em>e.g.</em>, microbial communities), such pairings might inform the clustering of genome sequences originating from the same species. Importantly, the efficacy of the Hi-C protocol has recently been demonstrated in bacteria (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Umbarger <em>et al.</em> 2011</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Le <em>et al.</em> 2013</a>), implying that this method could be applicable to metagenome samples containing both prokaryotic and eukaryotic cells.</p><p id="p-8">Here, we provide experimental proof-of-concept for this strategy in several contexts while also describing an algorithm for this task, MetaPhase (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). We reconstruct the genomes of as many as 18 species from a single synthetic mixture of eukaryotes and/or prokaryotes, including some species with as much as 90% sequence identity to one another, and we generate high-contiguity <em>de novo</em> assemblies for individual eukaryotic species present within the synthetic microbial community. In the process, we also present the first demonstration of Hi-C in an archaeal species.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/7/1339/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of MetaPhase methodology. (A) Performing Hi-C on a mixed cell population. Shown are three microbial cells of different species (green, red, blue) with their genomes (thick colored lines or circles), which may or may not include multiple chromosomes or plasmids. A Hi-C library is prepared and sequenced from this sample. The Hi-C read pairs from this library (black lines) represent pairs of sequences that necessarily occur within the same cell. (B) Using Hi-C reads to deconvolute individual species&#x2019; genomes. A shotgun sequencing library from the same sample is used to create a draft de novo metagenome assembly, which contains contigs from all species (thick lines). The Hi-C reads are then aligned to this assembly. Because sequences connected by Hi-C links must appear in the same species, the contigs form clusters representing each species. Note that some sequences (e.g., blue/red dotted line) may appear in multiple species, confounding the clustering. (C) MetaPhase workflow. A single metagenome sample is used to create shotgun, Hi-C, and (optionally) mate-pair libraries, which are used together to create individual species assemblies." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-966264751" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Overview of MetaPhase methodology. (A) Performing Hi-C on a mixed cell population. Shown are three microbial cells of different species (green, red, blue) with their genomes (thick colored lines or circles), which may or may not include multiple chromosomes or plasmids. A Hi-C library is prepared and sequenced from this sample. The Hi-C read pairs from this library (black lines) represent pairs of sequences that necessarily occur within the same cell. (B) Using Hi-C reads to deconvolute individual species&#x2019; genomes. A shotgun sequencing library from the same sample is used to create a draft &lt;em&gt;de novo&lt;/em&gt; metagenome assembly, which contains contigs from all species (thick lines). The Hi-C reads are then aligned to this assembly. Because sequences connected by Hi-C links must appear in the same species, the contigs form clusters representing each species. Note that some sequences (&lt;em&gt;e.g.&lt;/em&gt;, blue/red dotted line) may appear in multiple species, confounding the clustering. (C) MetaPhase workflow. A single metagenome sample is used to create shotgun, Hi-C, and (optionally) mate-pair libraries, which are used together to create individual species assemblies.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/7/1339/F1.medium.gif" width="382" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/7/1339/F1.medium.gif" width="382" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/7/1339/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/7/1339/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464679" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-9" class="first-child">Overview of MetaPhase methodology. (A) Performing Hi-C on a mixed cell population. Shown are three microbial cells of different species (green, red, blue) with their genomes (thick colored lines or circles), which may or may not include multiple chromosomes or plasmids. A Hi-C library is prepared and sequenced from this sample. The Hi-C read pairs from this library (black lines) represent pairs of sequences that necessarily occur within the same cell. (B) Using Hi-C reads to deconvolute individual species’ genomes. A shotgun sequencing library from the same sample is used to create a draft <em>de novo</em> metagenome assembly, which contains contigs from all species (thick lines). The Hi-C reads are then aligned to this assembly. Because sequences connected by Hi-C links must appear in the same species, the contigs form clusters representing each species. Note that some sequences (<em>e.g.</em>, blue/red dotted line) may appear in multiple species, confounding the clustering. (C) MetaPhase workflow. A single metagenome sample is used to create shotgun, Hi-C, and (optionally) mate-pair libraries, which are used together to create individual species assemblies.</p><div class="sb-div caption-clear"></div></div></div><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Sample collection</h3><p id="p-10">Cultures of individual strains listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011825/-/DC1/011825SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011825/-/DC1/TableS1.pdf">Table S1</a> (M-Y) and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011825/-/DC1/TableS2.pdf">Table S2</a> (M-3D) were grown to saturation in rich media (YPD for yeasts, LB for bacteria, McCas media for <em>M. maripaludis</em>, PMsul media for <em>R. palustris</em>). Culture densities were measured by spectrophotometry and FACS. After mixing the strains, cultures were diluted with YPD media (M-Y) or with LB media (M-3D) to a final OD600 of 1.0 in a final volume of 500 mL. Formaldehyde was added to a final concentration of 1% and cultures were incubated at room temperature for 30 min. To quench the crosslinker, 5 g of glycine was added to each 500 mL of culture, and the cultures were incubated for 30 min at room temperature. Cultures were centrifuged to pellet all cells. Cell pellets were frozen at −20° until further processing.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464496/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464496/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Contents of the metagenome samples analyzed</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Shotgun and mate-pair libraries</h3><p id="p-11">Total DNA was isolated from cultures using a standard phenol/chloroform glass bead purification followed by ethanol precipitation and subsequent cleanup using the DNA Clean and Concentrator-5 Kit (Zymo Research). Shotgun libraries were prepared using the Nextera DNA Sample Preparation Kit (Illumina). Mate-pair libraries were constructed using the Nextera Mate Pair Sample Preparation Kit (Illumina).</p></div><div id="sec-4" class="subsection"><h3>Hi-C libraries</h3><p id="p-12">Cell pellets (∼100 μL volume each) were resuspended in 2 mL of 1× TBS buffer containing 1% Triton-X and Protease Inhibitors (cOmplete, EDTA-free; Roche) and split equally into two separate 2-mL tubes; 300–500 μL of 0.5-mm diameter glass beads were added to each tube and tubes were vortexed on the highest setting in four 5-min increments, each separated by 2-min incubations on ice. Lysate was transferred to fresh tubes. Crosslinked chromatin was recovered by centrifugation at 13 KRPM in an accuSpin Micro17 centrifuge (Fisher) and rinsed with 1× TBS buffer. Chromatin from each tube was digested overnight with 100 units of either <em>Hin</em>dIII or <em>Nco</em>I restriction endonuclease (NEB) at 37° in a total volume of 200 μL. To enrich for long-range interactions (M-3D library only), digested chromatin was centrifuged for 10 min at 13 KRPM, rinsed in 200 μL of 1× NEBuffer 2 (NEB), centrifuged again, and resuspended in 200 μL of 1× NEBuffer 2 (NEB). Restriction fragment overhangs were filled in using biotinylated dCTP (Invitrogen) and Klenow (NEB) as described (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">van Berkum <em>et al.</em> 2010</a>). DNA concentration within the chromatin suspension was quantitated using the QuBit fluorometer (Invitrogen), and for each sample an 8-mL ligation reaction was set-up at a final DNA concentration of 0.5 ng/μL using T4 DNA Ligase (NEB). Ligation reactions were incubated at room temperature for 4 hr and then overnight at 70° to reverse crosslinks. DNA was purified using a standard phenol/chloroform purification followed by ethanol precipitation and resuspended in 600 μL of water with 1× NEBuffer 2 (NEB) and 1× BSA (NEB). To remove biotin from unligated DNA ends, 20 units of T4 Polymerase (NEB) were added to each 600 μL DNA sample and incubated at 25° for 10 min followed by 12° for 1 hr. DNA was purified using the DNA Clean and Concentrator-5 Kit (Zymo Research). Illumina libraries were constructed as described (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">van Berkum <em>et al.</em> 2010</a>) using reagents from the Illumina Mate Pair Sample Preparation Kit. Paired-end sequencing was performed using the HiSeq and MiSeq Illumina platforms (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464526/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464526/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Sequencing libraries used in MetaPhase analyses</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Draft metagenome assembly for M-Y and M-3D</h3><p id="p-14">To create draft metagenome assemblies for the synthetic samples, we assembled the fragment library using the IDBA-UD assembler (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Peng <em>et al.</em> 2012</a>). We ran IDBA-UD with the –read option set to the fragment reads and the following additional parameters: ‘–pre_correction –mink 20 –maxk 60 –step 10’. We used the assembly in contig.fa rather than scaffold.fa to reduce the risk of false joins made at the scaffolding stage.</p></div><div id="sec-6" class="subsection"><h3>Aligning Hi-C reads</h3><p id="p-15">We aligned the Hi-C reads to the draft metagenome assembly in a multi-step process. First, the reads were aligned using BWA (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Li and Durbin 2009</a>) with the option ’-n 0’, requiring a perfect match of the entire 100-bp read. For read pairs in which an alignment was not found for both reads, the reads were trimmed from 100 bp to 75 bp and were aligned using ’-n 0’ again. For read pairs in which alignment was still not found for both reads, the reads were trimmed to 50 bp and aligned using ’-n 0’ again. All read pairs for which no alignment was found were discarded from further analysis. Read pairs were also discarded if the reads did not both align within 500 bp of a restriction site, as recommended by <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Yaffe and Tanay (2011)</a>.</p></div><div id="sec-7" class="subsection"><h3>Clustering contigs by species</h3><p id="p-16">To cluster the contigs of the draft metagenome assembly into individual species, we used a hybrid clustering algorithm. A graph was built, with each node representing one contig and each edge between nodes having a weight equal to the number of Hi-C read pairs linking the two contigs, normalized by the number of restriction sites on the contigs. Only the single largest component in the graph was used; the other components, generally comprising isolated contigs containing a small fraction of the total sequence length, were discarded and the contigs were not clustered. Within this component, the Jarvis-Patrick nearest-neighbor clustering algorithm (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Jarvis and Patrick 1973</a>) was applied with <em>k</em> = 100, removing some edges and reweighting all other edge weights by the frequency of their nodes’ shared nearest neighbors. This nearest-neighbor approach accounts for the likely possibility that the clusters representing each species will have different internal densities of Hi-C links due to species’ differing abundances in the sample or differing susceptibility to the cell lysis step of Hi-C. Finally, the nodes were merged together using hierarchical agglomerative clustering with an average-linkage metric (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Eisen <em>et al.</em> 1998</a>), which was applied until the number of clusters was reduced to the expected or predicted number of individual species (12 for M-Y, not including <em>P. pastoris</em>; 18 for M-3D).</p></div><div id="sec-8" class="subsection"><h3>Scaffolding of genomic content within individual clusters</h3><p id="p-17">To scaffold the individual species’ genomes represented in each cluster of contigs, we aligned the Hi-C reads to these contigs and ran them through our Lachesis software (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Burton <em>et al.</em> 2013</a>) to create chromosome-scale scaffolds. The number of chromosomes in each species [7 for <em>K. wickerhamii</em> (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Belloch <em>et al.</em> 1998</a>); 8 for <em>S. stipitis</em> (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Jeffries <em>et al.</em> 2007</a>)] was provided as an input to Lachesis.</p></div><div id="sec-9" class="subsection"><h3>Validation</h3><p id="p-18">To determine the true species identity of the contigs in the draft metagenome assembly, we aligned them to a combined reference genome that included the reference genomes of all strains known to be in the metagenome sample (16 strains for M-Y; 18 species for M-3D). The alignment was performed by BLASTn (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Altschul <em>et al.</em> 1990</a>) with the following stringent parameters: ’-perc_identity 95 -evalue 1e-30 -word_size 50’. A contig was defined as aligning to a species if any alignment of the contig to the species’ reference genome was found; the placement of the alignment was ignored.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Deconvoluting yeast genomes from a synthetic mixture</h3><p id="p-19">To evaluate the effectiveness of the proposed strategy, we first applied it to a sample of defined, exclusively eukaryotic composition. Specifically, we created a synthetic metagenome sample consisting of 16 yeast strains (“M-Y”) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>) The strains include four strains of <em>Saccharomyces cerevisiae</em> as well as 12 other species of Ascomycetes at varying genetic distances from <em>S. cerevisiae</em>, all of which have publicly available reference genomes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011825/-/DC1/TableS1.pdf">Table S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011825/-/DC1/FigureS1.pdf">Figure S1</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011825/-/DC1/FigureS2.pdf">Figure S2</a>). These strains were grown individually to saturation in YPD medium and mixed in approximately similar proportions (with the exceptions of <em>S. kudriavzevii</em> and <em>P. pastoris</em>, which were mixed in at a much lower proportion to test the sensitivity of this approach). The mixed cell culture was treated with the cross-linking agent, formaldehyde, immediately after mixing the individual strains. Total DNA was isolated from the mixed population culture and prepared for sequencing. This resulted in 92.1 M Illumina read pairs from one shotgun library, 9.2 M Illumina read pairs from one mate-pair library, and 81.0 M read pairs from one Hi-C library (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/7/1339/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MetaPhase clustering results on the M-Y draft metagenome assembly. (A) Using Hi-C links to cluster contigs into 12 clusters, one for every species with a substantial presence in the draft assembly. Each contig is shown as a dot, with size indicating contig length, colored by species. Edge widths represent the densities of Hi-C links between the contigs shown. Only 2400 contigs are shown: the 200 largest contigs that map uniquely to each species. (B) Validation. This heatmap indicates what fraction of the sequence in each MetaPhase cluster maps uniquely to each of the reference genomes of the 12 present yeast species. Note that not all sequence is expected to map uniquely to one species. x-axis: the 12 yeast species. y-axis: the MetaPhase clusters. (C and D) Lachesis (Burton et al. 2013) reconstruction of individual species&#x2019; genomes within the M-Y metagenome assembly. These heatmaps show the Hi-C link density among the contigs in the MetaPhase clusters corresponding to S. stipitis (C) and K. wickerhamii (D). The x-axis and y-axis show the clustering and ordering of contigs by Lachesis. Dotted black lines demarcate chromosomal clusters. Note the expected signals of enrichment within each chromosome and on the main diagonal. The assembly in (C) is similar to the S. stipitis reference genome (Figure S7), whereas the assembly in (D) has far higher chromosome-scale contiguity than the best available K. wickerhamii reference (Baker et al. 2011)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-966264751" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;MetaPhase clustering results on the M-Y draft metagenome assembly. (A) Using Hi-C links to cluster contigs into 12 clusters, one for every species with a substantial presence in the draft assembly. Each contig is shown as a dot, with size indicating contig length, colored by species. Edge widths represent the densities of Hi-C links between the contigs shown. Only 2400 contigs are shown: the 200 largest contigs that map uniquely to each species. (B) Validation. This heatmap indicates what fraction of the sequence in each MetaPhase cluster maps uniquely to each of the reference genomes of the 12 present yeast species. Note that not all sequence is expected to map uniquely to one species. &lt;em&gt;x&lt;/em&gt;-axis: the 12 yeast species. &lt;em&gt;y&lt;/em&gt;-axis: the MetaPhase clusters. (C and D) Lachesis (Burton &lt;em&gt;et al.&lt;/em&gt; 2013) reconstruction of individual species&#x2019; genomes within the M-Y metagenome assembly. These heatmaps show the Hi-C link density among the contigs in the MetaPhase clusters corresponding to &lt;em&gt;S. stipitis&lt;/em&gt; (C) and &lt;em&gt;K. wickerhamii&lt;/em&gt; (D). The &lt;em&gt;x&lt;/em&gt;-axis and &lt;em&gt;y&lt;/em&gt;-axis show the clustering and ordering of contigs by Lachesis. Dotted black lines demarcate chromosomal clusters. Note the expected signals of enrichment within each chromosome and on the main diagonal. The assembly in (C) is similar to the &lt;em&gt;S. stipitis&lt;/em&gt; reference genome (Figure S7), whereas the assembly in (D) has far higher chromosome-scale contiguity than the best available &lt;em&gt;K. wickerhamii&lt;/em&gt; reference (Baker &lt;em&gt;et al.&lt;/em&gt; 2011).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/7/1339/F2.medium.gif" width="440" height="368"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/7/1339/F2.medium.gif" width="440" height="368"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/7/1339/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/7/1339/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464672" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">MetaPhase clustering results on the M-Y draft metagenome assembly. (A) Using Hi-C links to cluster contigs into 12 clusters, one for every species with a substantial presence in the draft assembly. Each contig is shown as a dot, with size indicating contig length, colored by species. Edge widths represent the densities of Hi-C links between the contigs shown. Only 2400 contigs are shown: the 200 largest contigs that map uniquely to each species. (B) Validation. This heatmap indicates what fraction of the sequence in each MetaPhase cluster maps uniquely to each of the reference genomes of the 12 present yeast species. Note that not all sequence is expected to map uniquely to one species. <em>x</em>-axis: the 12 yeast species. <em>y</em>-axis: the MetaPhase clusters. (C and D) Lachesis (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Burton <em>et al.</em> 2013</a>) reconstruction of individual species’ genomes within the M-Y metagenome assembly. These heatmaps show the Hi-C link density among the contigs in the MetaPhase clusters corresponding to <em>S. stipitis</em> (C) and <em>K. wickerhamii</em> (D). The <em>x</em>-axis and <em>y</em>-axis show the clustering and ordering of contigs by Lachesis. Dotted black lines demarcate chromosomal clusters. Note the expected signals of enrichment within each chromosome and on the main diagonal. The assembly in (C) is similar to the <em>S. stipitis</em> reference genome (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011825/-/DC1/FigureS7.pdf">Figure S7</a>), whereas the assembly in (D) has far higher chromosome-scale contiguity than the best available <em>K. wickerhamii</em> reference (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baker <em>et al.</em> 2011</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">We used the shotgun and mate-pair (∼4 kb) read pairs to generate a draft <em>de novo</em> metagenome assembly using IDBA-UD (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Peng <em>et al.</em> 2012</a>) (see <em>Materials and Methods</em>). This assembly had 48,511 contigs with a total length of 136 Mb and an N50 contig length of 17.3 kb. Contigs from this assembly covered most of the reference genomes of all 13 yeast species present (average = 96.0%), with the exception of <em>P. pastoris</em> (13.7%), which also had a very low fraction of shotgun reads aligning to it (1.2%), confirming its low abundance in the sample (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011825/-/DC1/FigureS3.pdf">Figure S3</a>).</p><p id="p-22">We next aligned the Hi-C read pairs to the M-Y metagenome assembly, yielding a network of contigs joined by Hi-C links (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). Then, exploiting the fact that sequences connected by Hi-C links are overwhelmingly expected to derive from the same cell, we used the links to cluster these contigs, applying a novel algorithm that combines the steps of Jarvis-Patrick clustering (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Jarvis and Patrick 1973</a>) and agglomerative hierarchical clustering (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Eisen <em>et al.</em> 1998</a>) (see <em>Materials and Methods</em>). Our algorithm suggested the presence of 12 distinct clusters in the sample based on the metric of intracluster link enrichment (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011825/-/DC1/FigureS4.pdf">Figure S4</a>). It clustered the majority of the metagenome assembly (111 Mb or 82.2% of total sequence length) into these 12 clusters. Of the remaining 24.1 Mb of sequence not clustered, the majority (99.7%) belonged to contigs that contained no <em>Hin</em>dIII sites and thus are not expected to produce a Hi-C signal in this experiment. Bootstrapping tests confirmed the robustness of our clustering method (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011825/-/DC1/TableS3.pdf">Table S3</a>). The 12 clusters match closely with the 12 distinct species present in the draft assembly (excluding <em>P. pastoris</em>), and 99.2% of sequence was placed into the cluster representing a species to which it truly belongs (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011825/-/DC1/FigureS5.pdf">Figure S5</a>), allowing for the possibility of a given contig belonging to multiple species.</p><p id="p-23">Further analysis of the clusters demonstrated several strengths and limitations of our method. Some species had greater Hi-C link densities than others after correcting for differences in species abundances (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011825/-/DC1/FigureS6.pdf">Figure S6</a>). This suggests that some species’ cells are more susceptible to lysis during Hi-C than others, and MetaPhase is robust to these differences. However, distantly related species proved easier to separate than closely related species. For example, in the cluster representing <em>Scheffersomyces stipitis</em>, 99.88% of the contigs (by length) matched the <em>S. stipitis</em> reference genome; however, in the cluster representing <em>S. cerevisiae</em>, 3.3% of the contigs (by length) instead aligned uniquely to the genome of closely related <em>S. mikatae</em>. We also noted that the sequence content in the <em>S. cerevisiae</em> cluster included the contigs that aligned to any of the four <em>S. cerevisiae</em> strains’ reference genomes. This indicates that although our method is generally successful in merging closely related strains of the same species into a single cluster, genetic variation between strains causes fragmentation of the species’ sequence contigs in the metagenome assembly (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011825/-/DC1/FigureS3.pdf">Figure S3</a>), which in turn hampers our ability to delineate this cluster correctly because smaller contigs produce a weaker and noisier Hi-C signal. Separating this cluster into sub-clusters representing each <em>S. cerevisiae</em> strain represents an additional challenge that will require further algorithmic development.</p><p id="p-24">We next sought to scaffold the genomic content of individual yeast species from the clusters of contigs representing each species. We ran the contigs in each cluster through our Lachesis software (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Burton <em>et al.</em> 2013</a>) to create chromosome-scale scaffolds. With the <em>S. stipitis</em> contig cluster, this approach yielded a scaffold for each of the eight <em>S. stipitis</em> chromosomes, with a total scaffolded sequence length of 14.2 Mb (91.7% of the <em>S. stipitis</em> reference genome and 95.1% of the portion of the <em>S. stipitis</em> genome that appeared in the draft metagenome assembly) (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>). These scaffolds matched the reference <em>S. stipitis</em> genome assembly fairly well (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011825/-/DC1/FigureS7.pdf">Figure S7</a>). There were a number of clustering errors, including one chromosomal cluster containing telomeric sequence from four other chromosomes, but the local misassembly rates were quite low: 0.9% and 1.1% for ordering and orientation errors, respectively. We applied this same method to the contig cluster representing <em>K. wickerhamii</em>, producing chromosome-scale scaffolds for each of the seven <em>K. wickerhamii</em> chromosomes, with a total length of 9.4 Mb (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2D</a>). These scaffolds, although we emphasize they have not been thoroughly validated, may represent a draft assembly with far higher contiguity than the existing <em>K. wickerhamii</em> reference genome (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Baker <em>et al.</em> 2011</a>), which has an N50 contig size of only 36.7 kb. Thus, the MetaPhase approach can be combined with Lachesis to create high-contiguity <em>de novo</em> genome assemblies of individual eukaryotic species within metagenome samples.</p></div><div id="sec-12" class="subsection"><h3>Concurrently deconvoluting eukaryotic, bacterial, and archaeal genomes</h3><p id="p-25">We next asked whether MetaPhase could be applied to deconvolute a metagenome consisting of both eukaryotic and prokaryotic species. Toward a proof of concept, we gathered samples of 18 species including eight yeasts, nine bacteria, and one archaeon, thus representing all three domains of life (“M-3D”) (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011825/-/DC1/FigureS8.pdf">Figure S8</a>). The species were grown in appropriate rich media and mixed together in similar proportions. The proportions were estimated by a combination of spectrophotometric, flow sorting, and counting approaches and were later confirmed by sequence coverage (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011825/-/DC1/TableS2.pdf">Table S2</a>).</p><p id="p-26">We created a simulated draft <em>de novo</em> metagenome assembly for M-3D by splitting the reference genomes of the 18 species into 10-kb contigs. We also experimentally generated a Hi-C sequencing library for the M-3D sample (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>), aligned these reads to the simulated contigs of the draft assembly, and clustered the contigs using Hi-C link frequencies (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). Our algorithm predicted the presence of 18 distinct clusters, consistent with the actual content of the simulated draft assembly and experimental Hi-C data (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011825/-/DC1/FigureS4.pdf">Figure S4</a>). It clustered 89.1% of the simulated contigs into these 18 clusters; of the unclustered contigs, 85.8% contained no <em>Hin</em>dIII restriction sites and thus are not expected to produce a Hi-C signal in this experiment. The 18 clusters clearly matched the 18 species in the sample, with 99.6% of contigs clustered correctly (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011825/-/DC1/FigureS9.pdf">Figure S9</a>). The clusters corresponding to archaeal and bacterial species had a particularly high accuracy rate of 99.87%. Bootstrapping tests confirmed the robustness of our method (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011825/-/DC1/TableS3.pdf">Table S3</a>). Thus, our approach can simultaneously deconvolute the genomes of microbes belonging to all three domains of life, making it applicable to real and complex microbial communities.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/7/1339/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MetaPhase clustering results on the M-3D simulated contig assembly. (A) The reference genomes of the 18 species from the M-3D sample were split into 10-kb bins. Hi-C links from the metagenome sample were then used to divide the bins into 18 clusters, one for every species. The contigs are illustrated as in Figure 2A. Blue and green colors are yeast species; yellow is archaea; and red and orange are bacteria. (B) Validation. This heatmap has the same key as Figure 2B. (C) Heatmap of the M-3D Hi-C links aligned to the reference genome of Vibrio fischeri, one of the bacteria in the sample. The V. fischeri genome contains two chromosomes and a 46-kb plasmid, pES100 (demarcated by dotted black lines.) This heatmap has a resolution of 10 kb. (D) Applying Lachesis&#x2019; clustering algorithm to the V. fischeri clustered genome to deconvolute the pES100 plasmid from the V. fischeri chromosomes. The x-axis shows the 424 simulated contigs in the V. fischeri cluster derived in (A and B). The y-axis shows the four clusters derived by Lachesis. Due to the presence of strong chromatin domains on chromosome I, Lachesis was unable to merge this chromosome into a single cluster and required an input of N = 4." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-966264751" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;MetaPhase clustering results on the M-3D simulated contig assembly. (A) The reference genomes of the 18 species from the M-3D sample were split into 10-kb bins. Hi-C links from the metagenome sample were then used to divide the bins into 18 clusters, one for every species. The contigs are illustrated as in Figure 2A. Blue and green colors are yeast species; yellow is archaea; and red and orange are bacteria. (B) Validation. This heatmap has the same key as Figure 2B. (C) Heatmap of the M-3D Hi-C links aligned to the reference genome of &lt;em&gt;Vibrio fischeri&lt;/em&gt;, one of the bacteria in the sample. The &lt;em&gt;V. fischeri&lt;/em&gt; genome contains two chromosomes and a 46-kb plasmid, pES100 (demarcated by dotted black lines.) This heatmap has a resolution of 10 kb. (D) Applying Lachesis&#x2019; clustering algorithm to the &lt;em&gt;V. fischeri&lt;/em&gt; clustered genome to deconvolute the pES100 plasmid from the &lt;em&gt;V. fischeri&lt;/em&gt; chromosomes. The &lt;em&gt;x&lt;/em&gt;-axis shows the 424 simulated contigs in the &lt;em&gt;V. fischeri&lt;/em&gt; cluster derived in (A and B). The &lt;em&gt;y&lt;/em&gt;-axis shows the four clusters derived by Lachesis. Due to the presence of strong chromatin domains on chromosome I, Lachesis was unable to merge this chromosome into a single cluster and required an input of &lt;em&gt;N&lt;/em&gt; = 4.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/7/1339/F3.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/7/1339/F3.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/7/1339/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/7/1339/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464607" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">MetaPhase clustering results on the M-3D simulated contig assembly. (A) The reference genomes of the 18 species from the M-3D sample were split into 10-kb bins. Hi-C links from the metagenome sample were then used to divide the bins into 18 clusters, one for every species. The contigs are illustrated as in <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2A</a>. Blue and green colors are yeast species; yellow is archaea; and red and orange are bacteria. (B) Validation. This heatmap has the same key as <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2B</a>. (C) Heatmap of the M-3D Hi-C links aligned to the reference genome of <em>Vibrio fischeri</em>, one of the bacteria in the sample. The <em>V. fischeri</em> genome contains two chromosomes and a 46-kb plasmid, pES100 (demarcated by dotted black lines.) This heatmap has a resolution of 10 kb. (D) Applying Lachesis’ clustering algorithm to the <em>V. fischeri</em> clustered genome to deconvolute the pES100 plasmid from the <em>V. fischeri</em> chromosomes. The <em>x</em>-axis shows the 424 simulated contigs in the <em>V. fischeri</em> cluster derived in (A and B). The <em>y</em>-axis shows the four clusters derived by Lachesis. Due to the presence of strong chromatin domains on chromosome I, Lachesis was unable to merge this chromosome into a single cluster and required an input of <em>N</em> = 4.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">Finally, we sought to use Hi-C to scaffold the genomic content of prokaryotic species from clustered contigs. Consistent with previous findings (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Umbarger <em>et al.</em> 2011</a>), we observed in the M-3D sample that both bacterial and archaeal genomes contain a substantially weaker signal of genomic proximity in Hi-C data than do eukaryotic genomes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011825/-/DC1/FigureS10.pdf">Figure S10</a>). This suggests that in prokaryotic species, in sharp contrast with eukaryotic species, Hi-C is not very useful for ordering or orienting genomic content within chromosomes. However, Lachesis’ clustering algorithm can still be used to deconvolute chromosomes, including plasmids, inside prokaryotic cells. We applied this algorithm to the genome of <em>Vibrio fischeri</em> ES114, a bacterial strain present in M-3D that contains two chromosomes and one plasmid, pES100 (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). The chromosomal architecture of <em>V. fischeri</em> prevented a complete merging of its chromosome I, but chromosome II and pES100 both formed distinct clusters (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3D</a>). Thus, MetaPhase and Lachesis are capable of using Hi-C signal not only to deconvolute prokaryotic genomes but also to separate plasmid-derived sequence from chromosomal sequence within clusters corresponding to individual species.</p></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-29">Here, we demonstrate that contact probability maps such as those generated by Hi-C enable the deconvolution of shotgun metagenome assemblies and the reconstruction of individual genomes from mixed cell populations. Using only a single Hi-C library taken from a metagenome sample, we exploit two different signals inherent to Hi-C read pairing: the intracellularity of each pair, which enables species-level deconvolution, and the correlation of Hi-C linkage with chromosomal distance, which enables scaffolding of the <em>de novo</em> assemblies of at least eukaryotic species, as we have previously shown (<a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Burton <em>et al.</em> 2013</a>). All of the sequencing libraries used here were generated by <em>in vitro</em> methods and were sequenced on a single cost-effective sequencing platform.</p><p id="p-30">The MetaPhase method is straightforward enough to be applicable to any metagenome sample from which a sufficient number of intact microbial cells can be isolated (10<sup>5</sup>–10<sup>8</sup>). Furthermore, this approach can be applied to microbial communities containing both prokaryotes and eukaryotes. The application of MetaPhase to diverse microbial communities may permit the discovery and genome assembly of many unculturable and currently unknown microbial species. Additionally, the use of the intracluster enrichment metric (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011825/-/DC1/FigureS4.pdf">Figure S4</a>) permits a rough estimate of the species diversity within a draft metagenome assembly, a useful piece of information that is not easily measured. However, as with all shotgun metagenomic sequencing, low-abundance species—such as <em>P. pastoris</em> in our M-Y sample—will remain challenging to assemble into contigs without very deep sequencing. Additionally, in samples containing species such as dinoflagellates with unusually large genomes (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Moustafa <em>et al.</em> 2010</a>), even deeper sequencing of both shotgun and Hi-C libraries may be necessary.</p><p id="p-31">We note that as MetaPhase delineates genomic content corresponding to individual microbial species, it also informs the chromosome and plasmid structure of these genomes and, in the case of eukaryotic species, it is capable of facilitating high-contiguity draft genome assemblies. Thus, it makes new species immediately amenable to phylogenetic and functional analysis while concomitantly increasing the power of existing genome databases to classify metagenomic reads via non-<em>de novo</em> methods. This method need not be limited to metagenome samples, because any complex cell mixture may be deconvoluted into individual genomes assuming enough genomic diversity is present that reads can be accurately mapped.</p><div id="sec-14" class="subsection"><h3>Software availability</h3><p id="p-32">The computational portions of the MetaPhase method consist of software written in C++ using Boost (<a href="http://www.boost.org">http://www.boost.org</a>) with auxiliary scripts written in Perl and bash. It runs in a Unix environment. The source code has been uploaded to GitHub and is freely available for public download at <a href="https://github.com/shendurelab/MetaPhase">https://github.com/shendurelab/MetaPhase</a>.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-33">We thank Giang Ong and Charlie Lee for help with sequencing; Riza Daza and Ruolan Qiu for technical assistance; Elhanan Borenstein, Rogan Carr, Roie Levy, and members of the Dunham and Shendure labs for helpful discussions; Adam Gordon for suggesting the name “MetaPhase”; and the following individuals for contributing microbial strains: Houra Merrikh, Carrie Harwood, Peter Greenberg, Amy Schaefer, Tom Lie, John Leigh, Grace Alex Mason, Christine Queitsch, Nick Rhind, Susan Jaspersen, James Cregg, Duncan Greig, Mark Johnston, David Bartel, Bonita Brewer, and the USDA Agricultural Research Service. We thank C. Beitel and colleagues for helpful and ongoing discussions regarding these methods. This work was supported by National Institute of Health (NIH)/National Human Genome Research Institute (NHGRI) grant T32HG000035 (Joshua N. Burton), NIH/NHGRI grant HG006283 (Jay Shendure), NIH/National Institute of General Medical Sciences grant P41 GM103533 (Ivan Liachko and Maitreya J. Dunham), NSF grant 1243710 (I.L. and M.J.D.), and DOE-LBL-JGI grant 7074345/DE-AC02-05CH11231 (J.S.). M.J.D. is a Rita Allen Foundation Scholar and a Fellow in the Genetic Networks program at the Canadian Institute for Advanced Research. While this manuscript was in preparation, a preprint describing a related method appeared in <em>PeerJ PrePrints</em> (Beitel <em>et al.</em> 2014a). Note added in proof: this preprint was subsequently published (Beitel <em>et al.</em> 2014b). The authors have filed a provisional patent application on methods related to the work described here and in <a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">Burton <em>et al.</em> (2013)</a>.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011825/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011825/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">Sequencing datasets generated by this study have been deposited in the NCBI Short Read Archive under the accession SRP041431.</p></li><li class="fn" id="fn-5"><p id="p-35"><em>Communicating editor: J. Rine</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-04-25"><span class="received-label">Received </span>April 25, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-05-22"><span class="accepted-label">Accepted </span>May 22, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Burton <em>et al</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.1" data-doi="10.1038/nbt.2579"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albertsen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hugenholtz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skarshewski</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tyson</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome sequences of rare, uncultured bacteria obtained by differential coverage binning of multiple metagenomes.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">533</span>–<span class="cit-lpage">538</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D533%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2579%26rft_id%253Dinfo%253Apmid%252F23707974%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2579&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23707974&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1339.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.2" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1339.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.3" data-doi="10.1073/pnas.1019177108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuch</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Extensive DNA-binding specificity divergence of a conserved transcription regulator.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">7493</span>–<span class="cit-lpage">7498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1019177108%26rft_id%253Dinfo%253Apmid%252F21498688%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzE4Lzc0OTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTMzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-4.7.1339.4"><div class="cit-metadata unstructured">Beitel C. W., Froenicke L., Lang J. M., Korf I. F., Michelmore R. W., <em>et al.</em>, 2014. Strain- and plasmid-level deconvolution of a synthetic metagenome by sequencing proximity ligation products. <em>PeerJ PrePrints</em> <strong>2</strong>:e260v1. <a href="https://peerj.com/preprints/260v1/">https://peerj.com/preprints/260v1/</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-4.7.1339.5" data-doi="10.7717/peerj.415"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beitel</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Froenicke</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lang</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korf</span>  <span class="cit-name-given-names">I. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Strain- and plasmid-level deconvolution of a synthetic metagenome by sequencing proximity ligation products.</span> <abbr class="cit-jnl-abbrev">PeerJ</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeerJ%26rft.volume%253D2%26rft.spage%253D415e%26rft_id%253Dinfo%253Adoi%252F10.7717%252Fpeerj.415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7717/peerj.415&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.6" data-doi="10.1002/(SICI)1097-0061(199811)14:15&lt;1341::AID-YEA328&gt;3.0.CO;2-U"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Belloch</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrio</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Querol</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Inter- and intraspecific chromosome pattern variation in the yeast genus <em>Kluyveromyces</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1341</span>–<span class="cit-lpage">1354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DBelloch%26rft.auinit1%253DC.%26rft.volume%253D14%26rft.issue%253D15%26rft.spage%253D1341%26rft.epage%253D1354%26rft.atitle%253DInter-%2Band%2Bintraspecific%2Bchromosome%2Bpattern%2Bvariation%2Bin%2Bthe%2Byeast%2Bgenus%2BKluyveromyces.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%2528199811%252914%253A15%253C1341%253A%253AAID-YEA328%253E3.0.CO%253B2-U%26rft_id%253Dinfo%253Apmid%252F9848227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(199811)14:15&lt;1341::AID-YEA328&gt;3.0.CO;2-U&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9848227&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1339.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077266500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.7" data-doi="10.1038/nbt.2727"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adey</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patwardhan</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitzman</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Chromosome-scale scaffolding of <em>de novo</em> genome assemblies based on chromatin interactions.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1119</span>–<span class="cit-lpage">1125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D1119%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2727%26rft_id%253Dinfo%253Apmid%252F24185095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2727&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24185095&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1339.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carr</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen-Orr</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borenstein</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Reconstructing the genomic content of microbiome taxa through shotgun metagenomic deconvolution.</span> <abbr class="cit-jnl-abbrev">PLoS Comp. Biol.</abbr> <span class="cit-vol">9</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.9" data-doi="10.1038/nature12820"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurice</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carmody</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gootenberg</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Button</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Diet rapidly and reproducibly alters the human gut microbiome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">505</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">563</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D505%26rft.spage%253D559%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12820%26rft_id%253Dinfo%253Apmid%252F24336217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12820&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24336217&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1339.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000329995000042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.10" data-doi="10.1038/nrg3454"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dekker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marti-Renom</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mirny</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Exploring the three-dimensional organization of genomes: interpreting chromatin interaction data.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">390</span>–<span class="cit-lpage">403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D14%26rft.spage%253D390%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3454%26rft_id%253Dinfo%253Apmid%252F23657480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3454&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23657480&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1339.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dick</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simmons</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yelton</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Community-wide analysis of microbial genome sequence signatures.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: R85.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.12" data-doi="10.1073/pnas.95.25.14863"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spellman</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Cluster analysis and display of genome-wide expression patterns.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">14863</span>–<span class="cit-lpage">14868</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DEisen%26rft.auinit1%253DM.%2BB.%26rft.volume%253D95%26rft.issue%253D25%26rft.spage%253D14863%26rft.epage%253D14868%26rft.atitle%253DCluster%2Banalysis%2Band%2Bdisplay%2Bof%2Bgenome-wide%2Bexpression%2Bpatterns%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.95.25.14863%26rft_id%253Dinfo%253Apmid%252F9843981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTUvMjUvMTQ4NjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTMzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.13" data-doi="10.1073/pnas.0708897105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frias-Lopez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tyson</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coleman</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuster</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Microbial community gene expression in ocean surface waters.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">3805</span>–<span class="cit-lpage">3810</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0708897105%26rft_id%253Dinfo%253Apmid%252F18316740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA1LzEwLzM4MDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTMzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.14" data-doi="10.1073/pnas.1402564111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howe</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansson</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malfatti</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tringe</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiedje</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Tackling soil diversity with the assembly of large, complex metagenomes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">4904</span>–<span class="cit-lpage">4909</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1402564111%26rft_id%253Dinfo%253Apmid%252F24632729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTExLzEzLzQ5MDQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTMzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.15" data-doi="10.1186/2049-2618-1-22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hug</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castelle</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wrighton</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharon</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Community genomic analyses constrain the distribution of metabolic traits across the Chloroflexi phylum and indicate roles in sediment carbon cycling.</span> <abbr class="cit-jnl-abbrev">Microbiome</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiome%26rft.volume%253D1%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1186%252F2049-2618-1-22%26rft_id%253Dinfo%253Apmid%252F24450983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/2049-2618-1-22&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24450983&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1339.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.16" data-doi="10.1038/nature11234"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huttenhower</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth cit-collab">Human Microbiome Project Consortium, et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Structure, function and diversity of the healthy human microbiome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">486</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D486%26rft.issue%253D7402%26rft.spage%253D207%26rft.epage%253D214%26rft.atitle%253DStructure%252C%2Bfunction%2Band%2Bdiversity%2Bof%2Bthe%2Bhealthy%2Bhuman%2Bmicrobiome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11234%26rft_id%253Dinfo%253Apmid%252F22699609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11234&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22699609&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1339.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305189000025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.17" data-doi="10.1126/science.1212665"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iverson</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frazar</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berthiaume</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morales</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Untangling genomes from metagenomes: revealing an uncultured class of marine euryarchaeota.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">335</span>: <span class="cit-fpage">587</span>–<span class="cit-lpage">590</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DIverson%26rft.auinit1%253DV.%26rft.volume%253D335%26rft.issue%253D6068%26rft.spage%253D587%26rft.epage%253D590%26rft.atitle%253DUntangling%2BGenomes%2Bfrom%2BMetagenomes%253A%2BRevealing%2Ban%2BUncultured%2BClass%2Bof%2BMarine%2BEuryarchaeota%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1212665%26rft_id%253Dinfo%253Apmid%252F22301318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzUvNjA2OC81ODciO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTMzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.18" data-doi="10.1109/T-C.1973.223640"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarvis</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patrick</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Clustering using a similarity measure based on shared near neighbors.</span> <abbr class="cit-jnl-abbrev">IEEE Trans. Comput.</abbr> <span class="cit-vol">C-22</span>: <span class="cit-fpage">1025</span>–<span class="cit-lpage">1034</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2BTrans.%2BComput.%26rft.volume%253D22%26rft.spage%253D1025%26rft_id%253Dinfo%253Adoi%252F10.1109%252FT-C.1973.223640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/T-C.1973.223640&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.19" data-doi="10.1038/nbt1290"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jeffries</span>  <span class="cit-name-given-names">T. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grigoriev</span>  <span class="cit-name-given-names">I. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grimwood</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laplaza</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aerts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome sequence of the lignocellulose-bioconverting and xylose-fermenting yeast <em>Pichia stipitis</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DJeffries%26rft.auinit1%253DT.%2BW.%26rft.volume%253D25%26rft.issue%253D3%26rft.spage%253D319%26rft.epage%253D326%26rft.atitle%253DGenome%2Bsequence%2Bof%2Bthe%2Blignocellulose-bioconverting%2Band%2Bxylose-fermenting%2Byeast%2BPichia%2Bstipitis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1290%26rft_id%253Dinfo%253Apmid%252F17334359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1290&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17334359&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1339.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244748300024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.20" data-doi="10.1038/nbt.2768"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekker</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-throughput genome scaffolding from <em>in vivo</em> DNA interaction frequency.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1143</span>–<span class="cit-lpage">1147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D1143%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2768%26rft_id%253Dinfo%253Apmid%252F24270850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2768&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24270850&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1339.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.21" data-doi="10.1126/science.1242059"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Le</span>  <span class="cit-name-given-names">T. B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imakaev</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mirny</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laub</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-resolution mapping of the spatial organization of a bacterial chromosome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">342</span>: <span class="cit-fpage">731</span>–<span class="cit-lpage">734</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1242059%26rft_id%253Dinfo%253Apmid%252F24158908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDIvNjE1OS83MzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTMzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.22" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMzM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.23" data-doi="10.1126/science.1181369"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lieberman-Aiden</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Berkum</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imakaev</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ragoczy</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Comprehensive mapping of long-range interactions reveals folding principles of the human genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">326</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLieberman-Aiden%26rft.auinit1%253DE.%26rft.volume%253D326%26rft.issue%253D5950%26rft.spage%253D289%26rft.epage%253D293%26rft.atitle%253DComprehensive%2BMapping%2Bof%2BLong-Range%2BInteractions%2BReveals%2BFolding%2BPrinciples%2Bof%2Bthe%2BHuman%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1181369%26rft_id%253Dinfo%253Apmid%252F19815776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjYvNTk1MC8yODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTMzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mitra</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forster-Fromme</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Damms-Machado</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheurenbrand</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biskup</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Analysis of the intestinal microbiota using SOLiD 16S rRNA gene sequencing and SOLiD shotgun sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14 (Suppl 5)</span>: <span class="cit-fpage">S16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D145%26rft.spage%253DS16%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.25" data-doi="10.1371/journal.pone.0009688"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moustafa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulis</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erdner</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Transcriptome profiling of a toxic dinoflagellate reveals a gene-rich protist and a potential impact on gene expression due to bacterial presence.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e9688</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMoustafa%26rft.auinit1%253DA.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253De9688%26rft.epage%253De9688%26rft.atitle%253DTranscriptome%2Bprofiling%2Bof%2Ba%2Btoxic%2Bdinoflagellate%2Breveals%2Ba%2Bgene-rich%2Bprotist%2Band%2Ba%2Bpotential%2Bimpact%2Bon%2Bgene%2Bexpression%2Bdue%2Bto%2Bbacterial%2Bpresence.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0009688%26rft_id%253Dinfo%253Apmid%252F20300646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0009688&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20300646&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1339.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Namiki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hachiya</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakakibara</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">MetaVelvet: an extension of Velvet assembler to <em>de novo</em> metagenome assembly from short sequence reads.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: e155.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.27" data-doi="10.1038/ismej.2011.78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Narasingarao</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Podell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ugalde</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brochier-Armanet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emerson</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title"><em>De novo</em> metagenomic assembly reveals abundant novel major lineage of Archaea in hypersaline microbial communities.</span> <abbr class="cit-jnl-abbrev">ISME J.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DISME%2BJ%26rft.aulast%253DNarasingarao%26rft.auinit1%253DP.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253D81%26rft.epage%253D93%26rft.atitle%253DDe%2Bnovo%2Bmetagenomic%2Bassembly%2Breveals%2Babundant%2Bnovel%2Bmajor%2Blineage%2Bof%2BArchaea%2Bin%2Bhypersaline%2Bmicrobial%2Bcommunities.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fismej.2011.78%26rft_id%253Dinfo%253Apmid%252F21716304%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ismej.2011.78&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21716304&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1339.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299052500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.28" data-doi="10.1093/bioinformatics/bts174"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">H. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yiu</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">F. Y. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">IDBA-UD: a <em>de novo</em> assembler for single-cell and metagenomic sequencing data with highly uneven depth.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1420</span>–<span class="cit-lpage">1428</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts174%26rft_id%253Dinfo%253Apmid%252F22495754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTEvMTQyMCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMzM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.29" data-doi="10.1038/nature08821"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, R. Li, J. Raes, M. Arumugam, K. S. Burgdorf <em>et al</em>., <span class="cit-pub-date">2010</span> <span class="cit-article-title">A human gut microbial gene catalogue established by metagenomic sequencing.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DQin%26rft.auinit1%253DJ.%26rft.volume%253D464%26rft.issue%253D7285%26rft.spage%253D59%26rft.epage%253D65%26rft.atitle%253DA%2Bhuman%2Bgut%2Bmicrobial%2Bgene%2Bcatalogue%2Bestablished%2Bby%2Bmetagenomic%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08821%26rft_id%253Dinfo%253Apmid%252F20203603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08821&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20203603&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1339.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275117500033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.30" data-doi="10.1007/s00253-006-0798-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Renouf</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Claisse</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lonvaud-Funel</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Inventory and monitoring of wine microbial consortia.</span> <abbr class="cit-jnl-abbrev">Appl. Microbiol. Biotechnol.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">149</span>–<span class="cit-lpage">164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.stitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.aulast%253DRenouf%26rft.auinit1%253DV.%26rft.volume%253D75%26rft.issue%253D1%26rft.spage%253D149%26rft.epage%253D164%26rft.atitle%253DInventory%2Band%2Bmonitoring%2Bof%2Bwine%2Bmicrobial%2Bconsortia.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00253-006-0798-3%26rft_id%253Dinfo%253Apmid%252F17235561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00253-006-0798-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17235561&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1339.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246094200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.31" data-doi="10.1038/nature12352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rinke</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwientek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sczyrba</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ivanova</span>  <span class="cit-name-given-names">N. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Insights into the phylogeny and coding potential of microbial dark matter.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">499</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">437</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D499%26rft.spage%253D431%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12352%26rft_id%253Dinfo%253Apmid%252F23851394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23851394&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1339.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000322157900031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.32" data-doi="10.1093/nar/gkr1204"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saeed</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halgamuge</span>  <span class="cit-name-given-names">S. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Unsupervised discovery of microbial population structure within metagenomes using nucleotide base composition.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">e34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr1204%26rft_id%253Dinfo%253Apmid%252F22180538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjQwLzUvZTM0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzEzMzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.33" data-doi="10.1101/gr.142315.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharon</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morowitz</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costello</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Relman</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Time series community genomics analysis reveals rapid shifts in bacterial species, strains, and phage during infant gut colonization.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.142315.112%26rft_id%253Dinfo%253Apmid%252F22936250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIzLzEvMTExIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzEzMzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.34" data-doi="10.1016/j.molcel.2011.09.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Umbarger</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toro</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Porreca</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bau</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The three-dimensional architecture of a bacterial genome and its alteration by genetic perturbation.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">252</span>–<span class="cit-lpage">264</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DUmbarger%26rft.auinit1%253DM.%2BA.%26rft.volume%253D44%26rft.issue%253D2%26rft.spage%253D252%26rft.epage%253D264%26rft.atitle%253DThe%2Bthree-dimensional%2Barchitecture%2Bof%2Ba%2Bbacterial%2Bgenome%2Band%2Bits%2Balteration%2Bby%2Bgenetic%2Bperturbation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2011.09.010%26rft_id%253Dinfo%253Apmid%252F22017872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2011.09.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22017872&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1339.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296212100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-other" id="cit-4.7.1339.35"><div class="cit-metadata unstructured">van Berkum, N. L., Lieberman-Aiden E., Williams L., Imakaev M., Gnirke A., <em>et al.</em>, 2010 Hi-C: a method to study the three-dimensional architecture of genomes. <em>J Vis Exp</em>. 39. pii: 1869.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.36" data-doi="10.1126/science.1093857"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Venter</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Remington</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heidelberg</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halpern</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rusch</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Environmental genome shotgun sequencing of the Sargasso Sea.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">304</span>: <span class="cit-fpage">66</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1093857%26rft_id%253Dinfo%253Apmid%252F15001713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMDQvNTY2Ny82NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMzM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.37" data-doi="10.1186/gb-2014-15-3-r46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Kraken: ultrafast metagenomic sequence classification using exact alignments.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">R46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253DR46%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2014-15-3-r46%26rft_id%253Dinfo%253Apmid%252F24580807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2014-15-3-r46&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24580807&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1339.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.38" data-doi="10.1016/j.mycres.2009.06.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glawe</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doty</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Characterization of three endophytic, indole-3-acetic acid-producing yeasts occurring in <em>Populus</em> trees.</span> <abbr class="cit-jnl-abbrev">Mycol. Res.</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">973</span>–<span class="cit-lpage">980</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMycological%2Bresearch%26rft.stitle%253DMycol%2BRes%26rft.aulast%253DXin%26rft.auinit1%253DG.%26rft.volume%253D113%26rft.issue%253DPt%2B9%26rft.spage%253D973%26rft.epage%253D980%26rft.atitle%253DCharacterization%2Bof%2Bthree%2Bendophytic%252C%2Bindole-3-acetic%2Bacid-producing%2Byeasts%2Boccurring%2Bin%2BPopulus%2Btrees.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mycres.2009.06.001%26rft_id%253Dinfo%253Apmid%252F19539760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mycres.2009.06.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19539760&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1339.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1339.39" data-doi="10.1038/ng.947"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yaffe</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanay</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Probabilistic modeling of Hi-C contact maps eliminates systematic biases to characterize global chromosomal architecture.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1059</span>–<span class="cit-lpage">1065</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYaffe%26rft.auinit1%253DE.%26rft.volume%253D43%26rft.issue%253D11%26rft.spage%253D1059%26rft.epage%253D1065%26rft.atitle%253DProbabilistic%2Bmodeling%2Bof%2BHi-C%2Bcontact%2Bmaps%2Beliminates%2Bsystematic%2Bbiases%2Bto%2Bcharacterize%2Bglobal%2Bchromosomal%2Barchitecture.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.947%26rft_id%253Dinfo%253Apmid%252F22001755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.947&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22001755&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1339.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/7/1339.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/7/1327" title="Genetic Control of Maize Shoot Apical Meristem Architecture" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/7/1347" title="Identification of a Novel Strong and Ubiquitous Promoter/Enhancer in the Silkworm Bombyx mori" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/7"><span>Volume 4 Issue 7, July 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/7.cover-source.jpg?itok=MrsYFroL" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/7/1339" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-iY-XbKHC5fcBSz06bhrypHAeaeiBi3SqDRIP62URnw8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/7/1339" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/7/1339" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/7/1339" class="active" data-icon-position="" data-hide-link-title="0">Species-Level Deconvolution of Metagenome Assemblies with Hi-C–Based Contact Probability Maps</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459908" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-0scSQifxSMgQUH1tbFxuDLMdclgGf4iuetu0XlS3EUM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/7/1339.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/7/1339" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-j0INgi0wDIm5L8ESdJ8-3-Cn46861oBfbCeJjW6liE0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/7/1339" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/7/1339.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459908" id="node459908--2" data-pisa="ggg;4/7/1339" data-pisa-master="ggg;g3.114.011825" data-apath="/ggg/4/7/1339.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Species-Level Deconvolution of Metagenome Assemblies with Hi-C–Based Contact Probability Maps</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Joshua N. Burton</span>, <span class="highwire-citation-author" data-delta="1">Ivan Liachko</span>, <span class="highwire-citation-author" data-delta="2">Maitreya J. Dunham</span> and <span class="highwire-citation-author" data-delta="3">Jay Shendure</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1339-1346;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011825 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459908--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Joshua N. Burton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195-5065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BN.%2BBurton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burton%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoshua%2BN.%2BBurton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ivan Liachko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195-5065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ivan%2BLiachko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liachko%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIvan%2BLiachko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Maitreya J. Dunham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195-5065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maitreya%2BJ.%2BDunham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunham%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaitreya%2BJ.%2BDunham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:maitreya@uw.edu" class="" data-icon-position="" data-hide-link-title="0">maitreya@uw.edu</a>
<a href="mailto:shendure@uw.edu" class="" data-icon-position="" data-hide-link-title="0">shendure@uw.edu</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jay Shendure </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195-5065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jay%2BShendure%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shendure%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJay%2BShendure%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:maitreya@uw.edu" class="" data-icon-position="" data-hide-link-title="0">maitreya@uw.edu</a>
<a href="mailto:shendure@uw.edu" class="" data-icon-position="" data-hide-link-title="0">shendure@uw.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/7/1339" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/7/1339&amp;title=Species-Level%20Deconvolution%20of%20Metagenome%20Assemblies%20with%20Hi-C%E2%80%93Based%20Contact%20Probability%20Maps" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/7/1339&amp;title=Species-Level%20Deconvolution%20of%20Metagenome%20Assemblies%20with%20Hi-C%E2%80%93Based%20Contact%20Probability%20Maps" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/7/1339&amp;title=Species-Level%20Deconvolution%20of%20Metagenome%20Assemblies%20with%20Hi-C%E2%80%93Based%20Contact%20Probability%20Maps" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/7/1339&amp;text=Species-Level%20Deconvolution%20of%20Metagenome%20Assemblies%20with%20Hi-C%E2%80%93Based%20Contact%20Probability%20Maps" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/7/1339&amp;title=Species-Level%20Deconvolution%20of%20Metagenome%20Assemblies%20with%20Hi-C%E2%80%93Based%20Contact%20Probability%20Maps" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/7/1339&amp;t=Species-Level%20Deconvolution%20of%20Metagenome%20Assemblies%20with%20Hi-C%E2%80%93Based%20Contact%20Probability%20Maps" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/7/1339&amp;title=Species-Level%20Deconvolution%20of%20Metagenome%20Assemblies%20with%20Hi-C%E2%80%93Based%20Contact%20Probability%20Maps" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/7/1339&amp;title=Species-Level%20Deconvolution%20of%20Metagenome%20Assemblies%20with%20Hi-C%E2%80%93Based%20Contact%20Probability%20Maps" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459908/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459908" id="node459908--3" data-pisa="ggg;4/7/1339" data-pisa-master="ggg;g3.114.011825" data-apath="/ggg/4/7/1339.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Species-Level Deconvolution of Metagenome Assemblies with Hi-C–Based Contact Probability Maps</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Joshua N. Burton</span>, <span class="highwire-citation-author" data-delta="1">Ivan Liachko</span>, <span class="highwire-citation-author" data-delta="2">Maitreya J. Dunham</span> and <span class="highwire-citation-author" data-delta="3">Jay Shendure</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1339-1346;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011825 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459908--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Joshua N. Burton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195-5065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BN.%2BBurton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burton%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoshua%2BN.%2BBurton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ivan Liachko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195-5065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ivan%2BLiachko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liachko%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIvan%2BLiachko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Maitreya J. Dunham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195-5065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maitreya%2BJ.%2BDunham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunham%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaitreya%2BJ.%2BDunham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:maitreya@uw.edu" class="" data-icon-position="" data-hide-link-title="0">maitreya@uw.edu</a>
<a href="mailto:shendure@uw.edu" class="" data-icon-position="" data-hide-link-title="0">shendure@uw.edu</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jay Shendure </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195-5065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jay%2BShendure%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shendure%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJay%2BShendure%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:maitreya@uw.edu" class="" data-icon-position="" data-hide-link-title="0">maitreya@uw.edu</a>
<a href="mailto:shendure@uw.edu" class="" data-icon-position="" data-hide-link-title="0">shendure@uw.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1339&amp;rft.epage=1346&amp;rft.atitle=Species-Level%20Deconvolution%20of%20Metagenome%20Assemblies%20with%20Hi-C%E2%80%93Based%20Contact%20Probability%20Maps&amp;rft.volume=4&amp;rft.issue=7&amp;rft.date=2014-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Burton%2C+Joshua+N.&amp;rft.au=Liachko%2C+Ivan&amp;rft.au=Dunham%2C+Maitreya+J.&amp;rft.au=Shendure%2C+Jay"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459908/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459908/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459908/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459908/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459908/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459908/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459908/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459908/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459908/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459908/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459908/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459908/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/7/1339" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/7/1339.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/7/1339.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/7/1339.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/7/1339.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
