<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/7/1217" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/7/1217.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/7/1217.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/7/1217.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/7/1217.atom" />
<meta name="HW.pisa" content="ggg;4/7/1217" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Nonsynonymous Substitution Rate Heterogeneity in the Peptide-Binding Region Among Different HLA-DRB1 Lineages in Humans" />
<meta name="DC.Identifier" content="10.1534/g3.114.011726" />
<meta name="DC.Date" content="2014-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Yasukochi and Satta. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="An extraordinary diversity of amino acid sequences in the peptide-binding region (PBR) of human leukocyte antigen [HLA; human major histocompatibility complex (MHC)] molecules has been maintained by balancing selection. The process of accumulation of amino acid diversity in the PBR for six HLA genes ( HLA-A , B , C , DRB1 , DQB1 , and DPB1 ) shows that the number of amino acid substitutions in the PBR among alleles does not linearly correlate with the divergence time of alleles at the six HLA loci. At these loci, some pairs of alleles show significantly less nonsynonymous substitutions at the PBR than expected from the divergence time. The same phenomenon was observed not only in the HLA but also in the rat MHC . To identify the cause for this, DRB1 sequences, a representative case of a typical nonlinear pattern of substitutions, were examined. When the amino acid substitutions in the PBR were placed with maximum parsimony on a maximum likelihood tree based on the non-PBR substitutions, heterogeneous rates of nonsynonymous substitutions in the PBR were observed on several branches. A computer simulation supported the hypothesis that allelic pairs with low PBR substitution rates were responsible for the stagnation of accumulation of PBR nonsynonymous substitutions. From these observations, we conclude that the nonsynonymous substitution rate at the PBR sites is not constant among the allelic lineages. The deceleration of the rate may be caused by the coexistence of certain pathogens for a substantially long time during HLA evolution." />
<meta name="DC.Contributor" content="Yoshiki Yasukochi" />
<meta name="DC.Contributor" content="Yoko Satta" />
<meta name="article:published_time" content="2014-07-01" />
<meta name="article:section" content="Genetics of Immunity" />
<meta name="citation_title" content="Nonsynonymous Substitution Rate Heterogeneity in the Peptide-Binding Region Among Different HLA-DRB1 Lineages in Humans" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;An extraordinary diversity of amino acid sequences in the peptide-binding region (PBR) of human leukocyte antigen [HLA; human major histocompatibility complex (MHC)] molecules has been maintained by balancing selection. The process of accumulation of amino acid diversity in the PBR for six &lt;i&gt;HLA&lt;/i&gt; genes (&lt;i&gt;HLA-A&lt;/i&gt;, &lt;i&gt;B&lt;/i&gt;, &lt;i&gt;C&lt;/i&gt;, &lt;i&gt;DRB1&lt;/i&gt;, &lt;i&gt;DQB1&lt;/i&gt;, and &lt;i&gt;DPB1&lt;/i&gt;) shows that the number of amino acid substitutions in the PBR among alleles does not linearly correlate with the divergence time of alleles at the six &lt;i&gt;HLA&lt;/i&gt; loci. At these loci, some pairs of alleles show significantly less nonsynonymous substitutions at the PBR than expected from the divergence time. The same phenomenon was observed not only in the &lt;i&gt;HLA&lt;/i&gt; but also in the rat &lt;i&gt;MHC&lt;/i&gt;. To identify the cause for this, &lt;i&gt;DRB1&lt;/i&gt; sequences, a representative case of a typical nonlinear pattern of substitutions, were examined. When the amino acid substitutions in the PBR were placed with maximum parsimony on a maximum likelihood tree based on the non-PBR substitutions, heterogeneous rates of nonsynonymous substitutions in the PBR were observed on several branches. A computer simulation supported the hypothesis that allelic pairs with low PBR substitution rates were responsible for the stagnation of accumulation of PBR nonsynonymous substitutions. From these observations, we conclude that the nonsynonymous substitution rate at the PBR sites is not constant among the allelic lineages. The deceleration of the rate may be caused by the coexistence of certain pathogens for a substantially long time during HLA evolution.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/07/01" />
<meta name="citation_mjid" content="ggg;4/7/1217" />
<meta name="citation_id" content="4/7/1217" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/7/1217" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/7/1217.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/7/1217.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/7/1217.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.011726" />
<meta name="citation_pmid" content="24793785" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genetics of Immunity" />
<meta name="citation_firstpage" content="1217" />
<meta name="citation_lastpage" content="1226" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Yoshiki Yasukochi" />
<meta name="citation_author_institution" content="Molecular and Genetic Epidemiology, Faculty of Medicine, University of Tsukuba, Tsukuba, Ibaraki 305-8575, Japan" />
<meta name="citation_author_email" content="hyasukou@proof.ocn.ne.jp" />
<meta name="citation_author" content="Yoko Satta" />
<meta name="citation_author_institution" content="Department of Evolutionary Studies of Biosystems, the Graduate University for Advanced Studies (SOKENDAI), Hayama, Kanagawa 240-0193, Japan" />
<meta name="citation_reference" content="citation_journal_title=Frontiers in bioscience :  a journal and virtual library;citation_journal_abbrev=Front Biosci;citation_author=G. Andersson;citation_title=Evolution of the human HLA-DR region.;citation_pages=d739-d745;citation_volume=3;citation_year=1998;citation_pmid=9675159" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. H. Brown;citation_author=T. S. Jardetzky;citation_author=J. C. Gorga;citation_author=L. J. Stern;citation_author=R. G. Urban;citation_title=Three-dimensional structure of the human class II histocompatibility antigen HLA-DR1.;citation_pages=33-39;citation_volume=364;citation_year=1993;citation_issue=6432;citation_pmid=8316295;citation_doi=10.1038/364033a0" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Carrington;citation_author=G. W. Nelson;citation_author=M. P. Martin;citation_author=T. Kissner;citation_author=D. Vlahov;citation_title=HLA and HIV-1: Heterozygote Advantage and B*35-Cw*04 Disadvantage;citation_pages=1748-1752;citation_volume=283;citation_year=1999;citation_issue=5408;citation_pmid=10073943;citation_doi=10.1126/science.283.5408.1748" />
<meta name="citation_reference" content="citation_journal_title=European journal of immunology;citation_journal_abbrev=Eur J Immunol;citation_author=S. Demotz;citation_author=C. Barbey;citation_author=G. Corradin;citation_author=A. Amoroso;citation_author=A. Lanzavecchia;citation_title=The set of naturally processed peptides displayed by DR molecules is tuned by polymorphism of residue 86.;citation_pages=425-432;citation_volume=23;citation_year=1993;citation_issue=2;citation_pmid=7679644;citation_doi=10.1002/eji.1830230219" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. C. Doherty;citation_author=R. M. Zinkernagel;citation_title=Enhanced immunological surveillance in mice heterozygous at the H-2 gene complex.;citation_pages=50-52;citation_volume=256;citation_year=1975;citation_issue=5512;citation_pmid=1079575;citation_doi=10.1038/256050a0" />
<meta name="citation_reference" content="citation_title=PHYLIP (Phylogeny Inference Package) ver.3.69. Distributed by the author;citation_author=J. Felsenstein;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=The Journal of Immunology;citation_journal_abbrev=J. Immunol.;citation_author=R. Gongora;citation_author=F. Figueroa;citation_author=J. Klein;citation_title=Complex origin of the HLA-DR10 haplotype;citation_pages=6044-6051;citation_volume=159;citation_year=1997;citation_issue=12;citation_pmid=9550403" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=F. F. Gonzalez-Galarza;citation_author=S. Christmas;citation_author=D. Middleton;citation_author=A. R. Jones;citation_title=Allele frequency net: a database and online repository for immune gene frequencies in worldwide populations.;citation_volume=39;citation_year=2011;citation_pmid=21062830;citation_doi=10.1093/nar/gkq1128" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=P. K. Gregersen;citation_author=M. Shen;citation_author=Q. L. Song;citation_author=P. Merryman;citation_author=S. Degar;citation_title=Molecular diversity of HLA-DR4 haplotypes.;citation_pages=2642-2646;citation_volume=83;citation_year=1986;citation_issue=8;citation_pmid=3458223;citation_doi=10.1073/pnas.83.8.2642" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=M. Hasegawa;citation_author=H. Kishino;citation_author=T. Yano;citation_title=Dating of the human-ape splitting by a molecular clock of mitochondrial DNA.;citation_pages=160-174;citation_volume=22;citation_year=1985;citation_issue=2;citation_pmid=3934395;citation_doi=10.1007/BF02101694" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=T. Hertz;citation_author=D. Nolan;citation_author=I. James;citation_author=M. John;citation_author=S. Gaudieri;citation_title=Mapping the landscape of host-pathogen coevolution: HLA class I binding and its relationship with evolutionary conservation in human and viral proteins.;citation_volume=85;citation_year=2011;citation_pmid=21084470;citation_doi=10.1128/JVI.01966-10" />
<meta name="citation_reference" content="citation_title=Molecular evolution of the major histocompatibility complex;citation_title=HLA associations with malaria in Africa: some implications for MHC evolution;citation_author=A. V. S. Hill;citation_pages=403-419;citation_year=1991" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. L. Hughes;citation_author=M. Nei;citation_title=Pattern of nucleotide substitution at major histocompatibility complex class I loci reveals overdominant selection.;citation_pages=167-170;citation_volume=335;citation_year=1988;citation_issue=6186;citation_pmid=3412472;citation_doi=10.1038/335167a0" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=A. L. Hughes;citation_author=M. Nei;citation_title=Nucleotide substitution at major histocompatibility complex class II loci: evidence for overdominant selection.;citation_pages=958-962;citation_volume=86;citation_year=1989;citation_issue=3;citation_pmid=2492668;citation_doi=10.1073/pnas.86.3.958" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=A. L. Hughes;citation_author=M. Yeager;citation_title=NATURAL SELECTION AT MAJOR HISTOCOMPATIBILITY COMPLEX LOCI OF VERTEBRATES;citation_pages=415-435;citation_volume=32;citation_year=1998;citation_issue=1;citation_pmid=9928486;citation_doi=10.1146/annurev.genet.32.1.415" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=D. Jones;citation_author=W. Taylor;citation_author=J. Thornton;citation_title=The rapid generation of mutation data matrices from protein sequences;citation_pages=275-282;citation_volume=8;citation_year=1992;citation_issue=3;citation_pmid=1633570;citation_doi=10.1093/bioinformatics/8.3.275" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=J. Klein;citation_author=A. Sato;citation_author=N. Nikolaidis;citation_title=MHC, TSP, and the origin of species: from immunogenetics to evolutionary genetics.;citation_pages=281-304;citation_volume=41;citation_year=2007;citation_pmid=18076327;citation_doi=10.1146/annurev.genet.41.110306.130137" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. Kusaba;citation_author=T. Nishio;citation_author=Y. Satta;citation_author=K. Hinata;citation_author=D. Ockendon;citation_title=Striking sequence similarity in inter- and intra-specific comparisons of class I SLG alleles from Brassica oleracea and Brassica campestris: Implications for the evolution and recognition mechanism;citation_pages=7673-7678;citation_volume=94;citation_year=1997;citation_issue=14;citation_pmid=9207151;citation_doi=10.1073/pnas.94.14.7673" />
<meta name="citation_reference" content="citation_journal_title=Immunogenetics;citation_journal_abbrev=Immunogenetics;citation_author=K. Musolf;citation_author=Y. Meyer-Lucht;citation_author=S. Sommer;citation_title=Evolution of MHC-DRB class II polymorphism in the genus Apodemus and a comparison of DRB sequences within the family Muridae (Mammalia: Rodentia).;citation_pages=420-426;citation_volume=56;citation_year=2004;citation_issue=6;citation_pmid=15351919" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. Nowak;citation_author=K. Tarczy-Hornoch;citation_author=J. Austyn;citation_title=The optimal number of major histocompatibility complex molecules in an individual.;citation_pages=10896-10899;citation_volume=89;citation_year=1992;citation_issue=22;citation_pmid=1438295;citation_doi=10.1073/pnas.89.22.10896" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=M. K. Oliver;citation_author=S. Telfer;citation_author=S. B. Piertney;citation_title=Major histocompatibility complex (MHC) heterozygote superiority to natural multi-parasite infections in the water vole (Arvicola terrestris);citation_pages=1119-1128;citation_volume=276;citation_year=2009;citation_issue=1659;citation_pmid=19129114;citation_doi=10.1098/rspb.2008.1525" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=B. Ong;citation_author=N. Willcox;citation_author=P. Wordsworth;citation_author=D. Beeson;citation_author=A. Vincent;citation_title=Critical role for the Val/Gly86 HLA-DR beta dimorphism in autoantigen presentation to human T cells.;citation_pages=7343-7347;citation_volume=88;citation_year=1991;citation_issue=16;citation_pmid=1714600;citation_doi=10.1073/pnas.88.16.7343" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. J. Penn;citation_author=K. Damjanovich;citation_author=W. K. Potts;citation_title=MHC heterozygosity confers a selective advantage against multiple-strain infections.;citation_volume=99;citation_year=2002;citation_pmid=12177415;citation_doi=10.1073/pnas.162006499" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=P. A. Reche;citation_author=E. L. Reinherz;citation_title=Sequence variability analysis of human class I and class II MHC molecules: functional and structural correlates of amino acid polymorphisms.;citation_pages=623-641;citation_volume=331;citation_year=2003;citation_issue=3;citation_pmid=12899833;citation_doi=10.1016/S0022-2836(03)00750-2" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=J. Robinson;citation_author=M. J. Waller;citation_author=P. Stoehr;citation_author=S. G. E. Marsh;citation_title=IPD--the Immuno Polymorphism Database;citation_pages=D523-D526;citation_volume=33;citation_year=2005;citation_issue=suppl_1;citation_pmid=15608253;citation_doi=10.1093/nar/gki032" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Robinson;citation_author=K. Mistry;citation_author=H. McWilliam;citation_author=R. Lopez;citation_author=P. Parham;citation_title=The IMGT/HLA database.;citation_volume=39;citation_year=2011;citation_pmid=21071412;citation_doi=10.1093/nar/gkq998" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=N. Saitou;citation_author=M. Nei;citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees;citation_pages=406-425;citation_volume=4;citation_year=1987;citation_issue=4;citation_pmid=3447015;citation_doi=10.1099/ijs.0.01472-0" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=Y. Satta;citation_title=Intensity of natural selection at the major histocompatibility complex loci.;citation_pages=7184-7188;citation_volume=91;citation_year=1994;citation_issue=15;citation_pmid=8041766;citation_doi=10.1073/pnas.91.15.7184" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=S. Sawyer;citation_title=Statistical tests for detecting gene conversion;citation_pages=526-538;citation_volume=6;citation_year=1989;citation_issue=5;citation_pmid=2677599" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. W. Slade;citation_author=H. I. McCallum;citation_title=Overdominant vs. frequency-dependent selection at MHC loci.;citation_pages=861-864;citation_volume=132;citation_year=1992;citation_issue=3;citation_pmid=1468635" />
<meta name="citation_reference" content="citation_journal_abbrev=Front Zool;citation_author=S. Sommer;citation_title=The importance of immune gene variability (MHC) in evolutionary ecology and conservation.;citation_pages=16-16;citation_volume=2;citation_year=2005;citation_pmid=16242022;citation_doi=10.1186/1742-9994-2-16" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=L. G. Spurgin;citation_author=D. S. Richardson;citation_title=How pathogens drive genetic diversity: MHC, mechanisms and misunderstandings.;citation_volume=277;citation_year=2010;citation_pmid=20071384;citation_doi=10.1098/rspb.2009.2084" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=L. J. Stern;citation_author=J. H. Brown;citation_author=T. S. Jardetzky;citation_author=J. C. Gorga;citation_author=R. G. Urban;citation_title=Crystal structure of the human class II MHC protein HLA-DR1 complexed with an influenza virus peptide.;citation_pages=215-221;citation_volume=368;citation_year=1994;citation_issue=6468;citation_pmid=8145819;citation_doi=10.1038/368215a0" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=N. Takahata;citation_title=A simple genealogical structure of strongly balanced allelic lines and trans-species evolution of polymorphism.;citation_pages=2419-2423;citation_volume=87;citation_year=1990;citation_issue=7;citation_pmid=2320564;citation_doi=10.1073/pnas.87.7.2419" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=N. Takahata;citation_author=M. Nei;citation_title=Allelic genealogy under overdominant and frequency-dependent selection and polymorphism of major histocompatibility complex loci.;citation_pages=967-978;citation_volume=124;citation_year=1990;citation_issue=4;citation_pmid=2323559" />
<meta name="citation_reference" content="citation_journal_title=Sampling Errors in Phylogeny.;citation_author=N. Takahata;citation_author=F. Tajima;citation_pages=494-502;citation_volume=8;citation_year=1991" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=N. Takahata;citation_author=Y. Satta;citation_author=J. Klein;citation_title=Polymorphism and balancing selection at major histocompatibility complex loci.;citation_pages=925-938;citation_volume=130;citation_year=1992;citation_issue=4;citation_pmid=1582567" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Tamura;citation_author=D. Peterson;citation_author=N. Peterson;citation_author=G. Stecher;citation_author=M. Nei;citation_title=MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods.;citation_volume=28;citation_year=2011;citation_pmid=21546353;citation_doi=10.1093/molbev/msr121" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. R. Thursz;citation_author=H. C. Thomas;citation_author=B. M. Greenwood;citation_author=A. V. Hill;citation_title=Heterozygote advantage for HLA class-II type in hepatitis B virus infection.;citation_pages=11-12;citation_volume=17;citation_year=1997;citation_issue=1;citation_pmid=9288086;citation_doi=10.1038/ng0997-11" />
<meta name="citation_reference" content="citation_journal_title=Immunogenetics;citation_journal_abbrev=Immunogenetics;citation_author=F. A. Verreck;citation_author=A. van de Poel;citation_author=J. W. Drijfhout;citation_author=R. Amons;citation_author=J. E. Coligan;citation_title=Natural peptides isolated from Gly86/Val86-containing variants of HLA-DR1, -DR11, -DR13, and -DR52.;citation_pages=392-397;citation_volume=43;citation_year=1996;citation_issue=6;citation_pmid=8606061;citation_doi=10.1007/BF02199809" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. Vidović;citation_author=P. Matzinger;citation_title=Unresponsiveness to a foreign antigen can be caused by self-tolerance.;citation_pages=222-225;citation_volume=336;citation_year=1988;citation_issue=6196;citation_pmid=3143074;citation_doi=10.1038/336222a0" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. Vita;citation_author=L. Zarebski;citation_author=J. A. Greenbaum;citation_author=H. Emami;citation_author=I. Hoof;citation_title=The immune epitope database 2.0.;citation_volume=38;citation_year=2010;citation_pmid=19906713;citation_doi=10.1093/nar/gkp1004" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=B. Woelfing;citation_author=A. Traulsen;citation_author=M. Milinski;citation_author=T. Boehm;citation_title=Does intra-individual major histocompatibility complex diversity keep a golden mean?;citation_pages=117-128;citation_volume=364;citation_year=2009;citation_issue=1513;citation_pmid=18926972;citation_doi=10.1098/rstb.2008.0174" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Nonsynonymous Substitution Rate Heterogeneity in the Peptide-Binding Region Among Different HLA-DRB1 Lineages in Humans" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/7.cover-source.jpg" />
<meta name="twitter:description" content="An extraordinary diversity of amino acid sequences in the peptide-binding region (PBR) of human leukocyte antigen [HLA; human major histocompatibility complex (MHC)] molecules has been maintained by balancing selection. The process of accumulation of amino acid diversity in the PBR for six HLA genes ( HLA-A , B , C , DRB1 , DQB1 , and DPB1 ) shows that the number of amino acid substitutions in the PBR among alleles does not linearly correlate with the divergence time of alleles at the six HLA loci. At these loci, some pairs of alleles show significantly less nonsynonymous substitutions at the PBR than expected from the divergence time. The same phenomenon was observed not only in the HLA but also in the rat MHC . To identify the cause for this, DRB1 sequences, a representative case of a typical nonlinear pattern of substitutions, were examined. When the amino acid substitutions in the PBR were placed with maximum parsimony on a maximum likelihood tree based on the non-PBR substitutions, heterogeneous rates of nonsynonymous substitutions in the PBR were observed on several branches. A computer simulation supported the hypothesis that allelic pairs with low PBR substitution rates were responsible for the stagnation of accumulation of PBR nonsynonymous substitutions. From these observations, we conclude that the nonsynonymous substitution rate at the PBR sites is not constant among the allelic lineages. The deceleration of the rate may be caused by the coexistence of certain pathogens for a substantially long time during HLA evolution." />
<meta name="og-title" property="og:title" content="Nonsynonymous Substitution Rate Heterogeneity in the Peptide-Binding Region Among Different HLA-DRB1 Lineages in Humans" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/7/1217" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="An extraordinary diversity of amino acid sequences in the peptide-binding region (PBR) of human leukocyte antigen [HLA; human major histocompatibility complex (MHC)] molecules has been maintained by balancing selection. The process of accumulation of amino acid diversity in the PBR for six HLA genes ( HLA-A , B , C , DRB1 , DQB1 , and DPB1 ) shows that the number of amino acid substitutions in the PBR among alleles does not linearly correlate with the divergence time of alleles at the six HLA loci. At these loci, some pairs of alleles show significantly less nonsynonymous substitutions at the PBR than expected from the divergence time. The same phenomenon was observed not only in the HLA but also in the rat MHC . To identify the cause for this, DRB1 sequences, a representative case of a typical nonlinear pattern of substitutions, were examined. When the amino acid substitutions in the PBR were placed with maximum parsimony on a maximum likelihood tree based on the non-PBR substitutions, heterogeneous rates of nonsynonymous substitutions in the PBR were observed on several branches. A computer simulation supported the hypothesis that allelic pairs with low PBR substitution rates were responsible for the stagnation of accumulation of PBR nonsynonymous substitutions. From these observations, we conclude that the nonsynonymous substitution rate at the PBR sites is not constant among the allelic lineages. The deceleration of the rate may be caused by the coexistence of certain pathogens for a substantially long time during HLA evolution." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/7.cover-source.jpg" />
    <title>Nonsynonymous Substitution Rate Heterogeneity in the Peptide-Binding Region Among Different HLA-DRB1 Lineages in Humans | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"4trkpcwHvjIwghJsgjeVq92vFVft0--jt4CcJ5NejLE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459927","apath":"\/ggg\/4\/7\/1217.atom","pisa":"ggg;4\/7\/1217","ac":{"\/ggg\/4\/7\/1217.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/7\/1217.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/7\/1217"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/7\/1217"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/7\/1217":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kAk76fO4I+tT1eSim1DaIYq1G9HcmCvh5oI\\\\\\\/aKKr8ZkprVSpDoCynlJLvULDuk1goaT03\\\\\\\/yWd0Iohb64L7MrEc8PJzsV95M3tYGpXyniE\\\\\\\/a+i+3PDbp1TAbckeCZYNmCUuY2Pomt4Kcm9LjcROg2ulyUWP1WYCiMZeueC3FCyK7V9SPjppGpWts0hfYXusxn91718AMbibbjg3rfySaB6CkVESTp6U0zitVWFy94fDuaiVOH40nc0rd6RnTBQ5WukzZQw2XRQmXIyNEQCydih\\\\\\\/ySeTiRe\\\\\\\/zMwXtJRBzePdB+LQXM1c5IOJKFGqejVFIJ310YFxzeniWZC99q2TbqziIC4a0dcCgHM713J71lUTiYPsOZ+bGvTlqsImoyj8poaJrwetdV5gFJZUgxdvBBw52GZOR+nJuJVFfv5vu1qFEsY7OvIt+pEikGWkVvU64zlAERrUnWaxYXRyDFt5jokcTH5aGPma+fs\\\\\\\/a8G9+0w5NAIW4GKrXJUX2N107\\\\\\\/wJqDmaxpHxZqNoGDc6Id9ZW7hmW7HF44UyO55BDUKcax1cE06y9V8eBxc1S2srz2cWEtDnMIFQTToGV3R+eVxIE2TvywZGNQkTTH6pFbGw9nLmw+JzjeFOSxGiXUahiWbOOSUf8+KSyQUcDVFRGo9Ccw9VEZU18K3PfNGUBunOF2c\\\\\\\/2JKF5vMl\\\\\\\/koGNboKng\\\\\\\/3VW2JPKlfXOrfnkjShNk\\\\\\\/HY3xSFxQUNRhuUdSIbyT1m0+mXtdDw29Ii4kgiacwSgtdLiBghmFF4O3v8hDulo1tF0WHT2dHbH6QVqmWgOiW5M6m7zla6oeduFJ5oupifbh\\\\\\\/8S+VTav0\\\\\\\/o1W4MysTPSr03vmwFeRAfkUrcjrngC1pcPcr9bs5sGj5bIdVa8ZlBSST4vRRdJO5sgRZBMwf7q1cceklKKT1Sd\\\\\\\/XOz+kpMrWBLVLtvWP1nXQkVYyRBt8EaQU+R6RtBDVaa8phUkjQAzZEyZu+L\\\\\\\/d9jEw6ZZHKAD3mPsH5SljJz9OdHgBvO1RVXhqBFQnjy33z1IKG3GetBDMW5e+MQHQE+AxOm775Z5p4YyVhlZjWpfqG4SNisK3pmm5ktsQa4a8IpdRRttHxXzpjnQptmzOflNhzt3+Qp\\\\\\\/0rsJqyRbiL2DN25sDo+S2qUTs5vUbr3ocNN9AtXmwL70So8NwpjRkZivb63Gj6ZwOICllDwQhbjc53FWYrt+5klxTkMv971TUbpHLByIEwQ5h3Z7+q+4wVpV6gM6P7qNB4WUt0+xuu0XDkoqllM8eCiwi\\\u0022,\\\u0022iv\\\u0022:\\\u0022Su6FByBDfyAWPORUQmCqpA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227c6a058b0bccee8205b121d43ecdd5390b07cc55df565688dbdb201eba475c61\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002241BbbYc02eh5tec7xq7NCZkazIM4CpO\\\\\\\/sNnETUPvWSIJ28Tsjx0KbEhNXlj+kwzfvPya6PJ6sTtusUAiGmSVyA\\\\\\\/EE67jTVbEoOZJGJkrozS4klFx4YipFNP76+sLaoaltEBmiW0NMGlTBpLRtLpIiNLw+xrJqxVfnrtxu790PxG1045irUnYmY76YSEntnEDAE+BJEpcS9y1cYrOt868\\\\\\\/zCDJ9fHoyOOI8aZc+95x57SKRNWDBqwwXiDy7NTWWCDmD5omS8nadYOupPUcAUDEzR4ZYVy8UDj0SLhdPAjF8FtxX+uEhdXWu0NyHU88W93lsa7gknxEBLLDUPlouhOo93Xc1qytoE5n6FBmv\\\\\\\/6\\\\\\\/EK+jV431k0b1bM6UHifOco3FRJoq+NM1Q7zQoTs9GDOeZpPwJCFM+5F\\\\\\\/ZAs+hA5fe9BN8irSZqENKVr1j0XgJS\\\\\\\/H7Ww8LMdofXoJeYemJOs95VV+k8Y0WOSnOqn4ILFWWZyRZXmiGdZI8f06+ZA7nb9ilB\\\\\\\/IlZvelXhmFLrLOfJp9FQro0daSGv8b3cTQAJwsEPH88KrcygDbK2ecoM3oWtDQHopTdvgK5Qv7Ez1v0Ncaw335kWUg3OOHj3k0Sos6crlOYtd3lB9XUvzTthmYA+LSRsKQ2P9Quvyy\\\\\\\/bIDQpXqtYou+apF+LJiDhEdSjyLErYc9SAejWE60eK4FvfOFgKrH8VoNttDst9aFLdAavdcwjBbfQw9mgjfEnCgjujjAtHXXBl+PheVVQJHPSAYcgr\\\\\\\/uf9\\\\\\\/fUzDdRWxax7iRb54iJjtndK0J92nQtc2taJSPdaZrD\\\\\\\/OjWBReX0BFzJc4WxbqjIJMB+YHth4yxTifMQKmoaSIyW7HDOxgcEOZ5WJRwjIzJI03Eymx+M+ifdKZ3JS+s+SxkkniUskGcbCzjieICArGvhwDR1pRqEMf0p9ptrrZAP6GacXsbceMtbZnODKNM+3P5JDuEhtHKA6KN4HW8a\\\\\\\/AJQfDF3e+HQG0At7qu6LTP+oFw8rFTAHTQgpLNCnWvnEnSsS14MHLoNVWu3VH2oj3OHBgVA79VYUOu9YRcqLj+N3MKO45Dh1XPxOtwfHGUb01oI5ZwI2xEApAdtRexGPFJBFATboiGFyoeYAWYXt\\\\\\\/cWHcpEU+4HzwOlKqGAmCCg19TppYaGeBjHk8IWz2gjCQODVH4fzvpIHmqOQriu9nKslWFF9udpv3ckTwYsT1ZlCg0DnGmVs6JaQ\\\\\\\/ygI8nbfTsnZTYFDT1ei9jeXZ+XG+VWKFEHgEgMu0hLPfF+++9DDtz6EW+bLJt3iqABjNk7WLkRr\\\\\\\/034JmjCodnUrk6TOt9RIxJ4\\\\\\\/alf9FHcNzeLUnO3Rpkl0vmMihdxwGpx+MxpJbm4dpuz88a90KAcfpu+UovHAkKgD23fDMysplsHc47CO9QrXyIEX0Ay2b0nMNS1lWFJVLJD0GVW\\\\\\\/tvO3ShkxG4F6Q4JzN1xCcI0EX8gOvmsYELFeedw70YJ0i3DDNlAkckhiwn\\\\\\\/xRjZlrIQB9wgGC9PYyQHX1Dwc9wlgMBdbA1NF5\\\\\\\/FJ1RjChdhT41g3korGE5J6FcGdYls6Ns9ymA2mEf9TnLHrq0uJ\\\\\\\/tz62\\\\\\\/o37FlOeaT4nQayge\\\\\\\/yk4Nwc5IVw8ROjKi0bxS\\\\\\\/bnApRpgC\\\\\\\/w0VoNuPuDXm2RORLIdkN9SeKCseAkxmMAcfkjKq6Zl\\\\\\\/OKn52XZuVcWbuP3bEbHuZ7k2bsfPy5vFzR69wG1LZ5lhx4I6Wmk66NL58zk6Xv7RKSjUJP92s2QipINo5Ijr8IlVszApyxc\\\\\\\/2T7oTVRnQrxHMGjeqAAYpH43EUZ3p9X9RA5dI4gtUesjEmngLduScJq2h9Up7C1cr11i09qX77Zv4aIjQKM4GExRw2TiLjTpazCeke142GPztZRaagAigFWnWxpylt1PtwcSnORUjLbi6FRkHcJwcuXDKZsHFLbHzlQqEf4CJFziqEpGMe2tzhPRYDN\\\\\\\/87Y50GNGK4sZ2Bj2AFfi0mD9HTqHwpBQ9gn29hM3LH7G\\\\\\\/RTzQgAlfV\\\\\\\/+iC6wqEVIX1ZGIKkPNDWlUuj5OEfAy2vgqH2qb\\\\\\\/A3SfBovOg12KzblowK+v8pfaQYEwLt5VOb2JTJQ\\\\\\\/T4lhrxMIfRB6G9J8AJPkWcuT1KMMHgezbgBpdMR59Rt9\\\\\\\/eeBi8DmkX8eE6YAVTXhk5ecV9WdWu1eo1p79LrJTAyjuF9q0sGjvB3s8W4Kb8aUzAP9O4mEYwBtH\\\\\\\/nXdHMMM2e39Wpz1HOEx8px0j\\\\\\\/kt97cplOlwpaAAWap256S8sg0pD2hsdfbHHUTRiOgVTWI3M3ZnqbUXFFVJ0IPzfIeQrn20A1i2QB2q3o7FDwIy2yngrNfO16cjYhpcuGyQDk44Ir3s3bNIEhjrcW9ApOAeuyYEr\\\\\\\/X9DDuhx\\\\\\\/msdLZygBHdFW8tbi2ZncUVbQKjNWm1kH3JqA1f6qOD4lCc132Jjq2u5W+XD6a1yYSHodrRBHee\\\\\\\/8FWZUw7qv2qYEEWYRWOUcI0tXSnzvlXb02xMEBA87hl0bqTraQ9EGwJ2mXT3PBzg9xyu6+KaZZwGWb27h9v+YbFiTJv63mOZCMC1X\\\\\\\/OMX30+DiR3CVTzgPnjJnbuH\\\\\\\/xHW+X7NPwygPeRdthk+JpifuzLYDn6AYjaGqP2bFvpIQkjz\\\\\\\/iqlM7JTowLW3IXdszSJrK8HXg9DxW8nOw0\\\\\\\/srs0mNVNGFy4edovUGiAtdOmWRRtYHudNY0Im66wZoic0WFRvF6dd7QKB5\\\\\\\/ABIx2xFu6MXZSsAg2nPURYaggLXNDL9iSjrAhzVkfqrFkofD5McB2PQfFxAhSEqPx8Gv6I4\\\\\\\/+WtXbGRDB2O8EakR+9DnFKUGokFG\\\\\\\/x0\\\\\\\/3IPOBGPY7JURdOc7RJmPHSn7NM1OLKIQFVDgECCn5JBG9uhFxSK+r78d3RzeeFd4R0k3lktL\\\\\\\/x1v8u8L9G3u+R36okEgaVFkgaBgxYcIJeCc9eB2MaVkFmCjgz1U9JXIwtxydAuSp3HvMNhru96LP+nwG9l+8rMct6adtr8t4sCe2tIFCfthcJMYGzGbGRESh8DzSERXB2\\\\\\\/fzuNT1sGskw4KTBvu6gLXk1U6ysxu0sAgfdxPy8EWI5\\\\\\\/NvwlxI0Qr+sEws5Ga\\\\\\\/fV6mt9krTj9BuB6xz8hWtU6LmaGCnfoDdchCRc0Oqzwq4wcNQ\\\\\\\/r3zU8nMuPKX5bLzDMLwU0+pLDlUoeXf8+Wi8BlZQ6R4nVEktnj66UujRGXRibuPnwOahL2\\\\\\\/kJqfmbBvlzyB7sKxBFtZ8QZSGDuDp6bYxXtasFlK+GPxtTVS+SAXc1F6cGtmYbbNrGWpHL6YsCbFoE1F8LgKjpeNa\\\\\\\/qFzF7S28tOwTcJCjn0PquR42yaTOQRvrDR9P0Wid4i+FIjSGyaeFTpAb6IhNo+y5UIzEi6BlJqfeMo0ljaEI27ZCA3P0ldM7w6B07kDxN64di8fvUHzrwAqzzo2mmEqkvJ7CQkEdrOL+zlgj7eF6l79GA2a\\\\\\\/4BnM6bwiJnpXZYkMgvkS48TgNu1QVt+syMO6\\\\\\\/RGYM8wTJX\\\\\\\/cP4hHZfpv5qDWgwmEPTTR9MstwpT8hwrPMLp\\\\\\\/c0MpVg7ctTV4rGdnQoxq\\\\\\\/64PARbkfxPAenT9SZBJo8RuEJ7uA52OacEjqTT3J+CLlmau9p+\\\\\\\/M47x5MwYb6ooO8qDME6xDiLdBkEXofVO39qCo96LcF\\\\\\\/uSg4fI8fqmc9GvE7U+GufS7kOcV5gEtrofKwddeEdYO83c61ftlUhNREp5GdO8T0yP1U0jJ5dMcQ9hr4DeGLK67v623Zhh70f2I4nh2Nku7sq5mJbpCpvWe1ctLW1I5LDrPFq8bdq4g+fp3lhByn\\\\\\\/Oa98794dTYEbv25SO4WQ2iJF3juQCs1QxB13NTR1VdAsrG1FqpMb2oEJqx+qN1A+A0hjRMvH1ZOOGIMbr9AnPqngPdu2Ct+EttZcP8SecKdrpoQsABu8GUlyMFqJQMaBII00FaDxRR2BXz6KUiH9+p2LIEoM7ci1bfMLIpRKweHj9vfG\\\\\\\/1U9VfLzB9uxeaTG71E\\\\\\\/0VF6BcQ2q1hw7CW1ak1JOtt4peU82wxHyeXTl0vLk1xgAEnnOsPelXnn2YuFimfMTSMcpGtmPyM\\\\\\\/ptz3zK0s1eDcAfZdHmiNBTJN1rfyT1xCGUbTyyYQwYaurAsUwFgh2eqcMmj8ZOl5iWu27dGqcfqwlkD\\\\\\\/Z\\\\\\\/lcYXw4Z6Sc5Ikw59vyk2POzYGqyfrDoUqh64WCKyyMyibDv0qj3uEf7zdoKSvK8wJ9O9Eq0p4IF2mw+xQDzTUslS9AHIzU0z3YIaGvw1k9wtKNuXFcSn2PT7t8uqxaOfkethjb2gA4+w8fKUFa+VqkzlFH8O1H2Ih8FYiItpws8FKw67wS8nDQtrVd0Mca74NAQfPQyRxVvi8YcZdDnUmgt7RNWxXkL6HUmmOHrjhGtLUlHxzImnXdul8ItFZAuuzNEW7\\\\\\\/wKj1g\\\\\\\/B46JjzUclC6\\\\\\\/oyZmJP3Qp3Zv6OKwpPPB3nFN+ZfNeaeRsJefz37ve1k\\\\\\\/y4Cd1ASG57y9aPOVYpj9tuWKEf1rxzi3\\\\\\\/Tj7qu+d+kAVEJ8ta4q1ZpiJbQmcdO5e\\\\\\\/cSux8LxjOyjyIrv2i5PCwO9igYTE9HabnuZb7MaKjWE8CXaWBc4gQEwCzJwkHm5IAfL26dN2ZhPCNnInhswJb+xijiWHE2GJjbSso226p\\\\\\\/IWwHYdjzawoV+eXT6QVGjMEZkIdcWmWzKFYTAriqZfCBFZg3j6e60NqaFHECL+OYx2PjvtSqCM1J5sI9gwrc7ydrNtR3AAFsqnWJ16F6DXpW2UGtGquTG8I50p2MXjzYv8uEk1QWPa2omTsn9dEiadvkqVQxNEd9CD89mpY+Jv3KdVRK0X\\\\\\\/UN+aI1NoUEY+j3sPOGBYNpKlc8e2H6hzGnzGgTiv+6ZUixtEvM98ICj1\\\\\\\/EbmehYiRiWk6yqw8vzsktbIIajYuNS8ME5Dzi4C\\\\\\\/KDn8fln4hUhB2VZJZkHBBn4qw2q6R590f\\\\\\\/8gnQKaHBOppbCTJg273F4CMzdcquR3xB6m9QjRhGft4z0U0rg\\\\\\\/ZkyjZioNdKnf2GNUAXKfLEbq8fFn18DJc\\\\\\\/Q5EnWu5usZtp0xFZhbX2V6++UOKCUj2AdwJ4pO6UDXiDVFdSd6\\\\\\\/seZk7OXVo8ejocQDrTbqT9ncRVOYAXQlXt+WxTyaFIm+345vImCOeo3+YriBqrt3aEpugV0rxXRdYPysLqlLhsfguemVzDi6NM9ke6XMXCWcnw24Mpn06MYREOltSFZVU0+xLI3R9U5HWhgEB2nwF+0vocBMx08lULsF6HP8lsfHVih5KwE4p2wj27m9zQ+Q5kASA0Zsu5HYCeQD+dYQZuh1\\\\\\\/BIC5EXxSuXQWWyoWGSRAVlkB1ib82Hu6XeqEmPJ7UvfDec+wmM15I\\\\\\\/nAM55DZ46OpEcVsAHwwYCzhpZRYNMPTqPzd4fqTdAtFF8+W8L\\\\\\\/oD1cunVecxrtA68CSW1VQP6DF8VrQmdY+1wXjwRB7LA4xxM3y2TCIZEe2HJrszGXRAlJfzePRGoSzyC1UoP4VWdNMQ\\\\\\\/ibTcDMnaUlm9x+jeK\\\\\\\/Sy6Sv5lH003Y2ixiiaKzHok3TSuQYk+UH\\\\\\\/VwSxBuBqdTGF3T2\\\\\\\/snP2i61LzqL+in94M7Bb5K82xpUiCbTz8Zbxvk0L9ojCnKremqmv\\\\\\\/pt1l4NFPzDEQp3LmBuM59VtmjhNBnXNaombnI4JmV265gTcNAFCkkT3wBxsuaMmNHyvVLnONyIh5OIlmmOn6jooOKb6qI7M2y9FPnB29qQxkXx4TGnRYswM51BaBPXV5ERRsulqi4lNeyrxQcS3gvnztJ0Ki5VlzPHLEFi487C2qPyJCTzVX7sDBT\\\\\\\/3Xpkj8Cu1cwCfWvwAom6Rzn4FzzA69wNrMLGQlOwpZ5+dbOohD+3od5h5xEmfJ2ZGcO6puupIJ9+wr\\\\\\\/xjc0G0vYiaxEuNfUhPO\\\\\\\/LYt6JvhHBkrCEufsx32Ic+6zUuaeCE69z1QG2sETEqbFMsIQxu3pGF6STo7usi4oaH4eQ4369bdX2j7itm9hhoDPhAiNsPoNfP+IHh7H4Ig8TmZvIbVVxOXf4yjZzz93+17YmafuD6A\\\\\\\/AWouU+Jeoa44AwpRMTlz3VgT5kAcxcYI9qHv8d2LH0yMOd5c8t5jpcG5C8qajF6A2ky6HBfjU5BnFgmfucVk\\\\\\\/Eyyina8Zqslap4pv2wH5OnqLfw29M5SA6q+If1WoUfbZYlNH64bqZTEi3OrQ6Sg\\\\\\\/U6bbKgX19r3IKnJmrMpjj3lu+e9l\\\\\\\/ZLSTjaOY1POsdUtfEPDEenU3ThpyHbYWQnIWWawC32mc+Rce+PpzVv1ViuJCb81Ht7zeHssqFFYthFszwrBOCJDsJ6TsVfCJb2lSxysfj3s2Qzh9q1I+a3hGpQheTm92grzRc9lgneMOKjm8jIRw0HcfZ\\\\\\\/6ezGrDtxOt4VCaUDe71HQNTzdsSvsIT7wPuXtsDX5kfkX\\\\\\\/oRmIc4qfVyMwb\\\\\\\/XZCP2H713Iko2LV4z8Lbxvn+y3P1Urgm6apY0F9rdTMq0QdH43ifqFTQx6msoORHN\\\\\\\/vN7BWWdXIZpwvZCnMKaxXWGGuAKrFLx7Mqgb9qVoK1FZCgCnmfwsYkN9I3Zl+EQU61FHX76X2RnhW8pRkCWbWmJKLr73TrVY76IKru2BG0DzaSgvOXhdjaAZH0r3\\\\\\\/T7m\\\\\\\/UQ5tQ4p3IZxwo3MqkLSGj44nyZsJBaMZkCM3ARkZlsQcE\\\\\\\/YbiFJ0ljTefQkxIR43Ay9fqJpyOXuJe8lxdb5LdZuIe2dcxEF\\\\\\\/lF08dxie5C5XlHCRB8V\\\\\\\/nAgj1TQcvtJs9CNblgiNw0e6cggwOSMty\\\\\\\/MaAugfH\\\\\\\/CBQME78bbAkXtquQ6SvFBqWgA5jyaTHNbq3+KgIjHgBPnwc8YIZwAru9q3EUZow3Ea96mn175OAbfxB8FcOsMpi27IFD\\\\\\\/Bh5qOlkIYBfyCOFDzN6FuO72acYiiVIkzu4Kc5hNQ21\\\\\\\/40RDr8jO2dYxvXCI4xOqR1iHHleOkw\\\\\\\/NpTZGBOc9wHbwCLExFkLwd91LRZcSdl4SmgW6KCbX78HtyODZPwns04K2TfngGqLmURAAs5LHmUo+SOEKC9o1rcYLv73QX7JcOs8ah\\\\\\\/iUoMwDsW0kPG4+GQwiNsKP8jfbzX85uiiCnPxf4Qys6AVkz6qC0PVXsWll71GioriROzER8rRmdcqaivMVO48OftrLAzXrFVO68FkiZYDFaxXDgUQkvoPoniARjoxOj4eXbwr\\\\\\\/XnDJZ1q6A2vfmZ80H9mHFygN2N9yjv7dzxmIyFYiMpNDvkjWaG9O9bBs1AhI5bVGM4SZMpsrufkIUi6oF8pT\\\\\\\/XJ4J1LR4cmygYmpVQFouDAo8MLL6NZ4fKQ5F5xjt4bB59MdkkHAayHL7\\\\\\\/zTwu36Q1LeKzpnbeFZBFm4Gu9gfFYTNTZ+n+choogRFlg+hlnCaI1jrmIr7xK\\\\\\\/gx5DCHlSOkyozegJG8hpeMg42QlsrprgV5RpEkR08gKuT4qcQdLEJoQuMVxXVoMxoo0UGlD0NlevAMoMx5d2UFyWxx2zwLy4rEHX7Q\\\\\\\/YLkPTn9DnMe9D41b3JUADluS00scD+WSa5iD+ufYnaPX1TgElWDvEDwdZLfXzMFvPdXQ7xKa6me27Rh903GBKxg\\\\\\\/BoJnr+gP+MZE\\\\\\\/Kqomp\\\\\\\/Y6emCguZSHFTamBOREKwiCK4+3\\\\\\\/OPqN4eoh6rtYv2F8T\\\\\\\/9xTQct1IEedLgZ16AcFEgX+S5PkOaVPAjLo79poRLS8O81vs23V7wR+yNE0EFd34Kl2B+XgJqUjmbKB451fJ3pJSQ+lJI9i5acbFF92zH5eWNUvi0f9q5xPE7O1umJ0HH9aYf1PzhBh45UhaKERcnj7YqKFdzTNgaBcJyPo59TpD19kwnB2xzuU70+yDRuLYzDMnzROhj8vmXknW9lDao6FiK75GyzFrwbpvLFr2RoO4I7Idjm\\\\\\\/3sFcQW1PGmp2YMgTjO6pUTleDlC+3pqWmwDuyC0Xh\\\\\\\/ENdqKF5an3PirjSF2H9NVa5+p4cc+\\\\\\\/NSY0P5xaergo\\\\\\\/HLDVVzskQi+RsiybIEc1P9uxlf0X1\\\\\\\/1+ZDX\\\\\\\/oJobieuK23T4Di4odq925PFzYbZdk7nPHeCrCOk1wbcmR20W3mcUiz1vLL5zW1LkVpUlachgDdHFYGUjNMGf1\\\\\\\/UlNOTJO456eyTjmXWY8l\\\\\\\/mRtIvI4irMws\\\\\\\/Apxjiar5rOcMKnMIzmTKE0iFVrWiTIEEsr31cyPQk58\\\\\\\/nmF1BOzjq2tUbTApn\\\\\\\/w44abkRlFtc2nqWdKFfV8C0ly\\\\\\\/UUQlhSg2kJkGLPuC5o2FkAnHIkPiWRQIOzItk4szDFPagpV\\\\\\\/m6PGuc1MmnJCSivksuopU2Kh52kBA0akpfJs9AYAUKDxVfsaIw58HPWkM3Lw4+QcpZwctKWbSfw9UQeO8KQBqsQiJV6IVqRrVbHTMRiRX8a35Xv2j+D8lrXTr3mqZatgwoUVwAn0rKLJ6z34cLoaWkdVmLFMnwW5s2XaKDWzfmKVzkZ9PunQoPRCox5cTIOSg0O\\\\\\\/XcwmjgFMaPQiAsg+Q4EMtER\\\\\\\/LH60qOyqyRa9K4vl+HSphoj1Ax\\\\\\\/uYMPEqOBvg+h8YvgH1ZbeIl2WoWER7ujWkSHya9nANfER7uiG\\\\\\\/Qibx3A\\\\\\\/em+l3vzjN8FkW8+fZt3W8Sk+1O7tk0+P5VbcfLWRmpFzt3DESSupC7QAwA+LBBuKDo\\\\\\\/O2wGoR8C5Nwa+xL\\\\\\\/Z1Oh27K79QlVu457svm7pYcSYYEmefCtOjrCUJIkYWxoc2mapRCKyoIQ7EYmOChfxmwyjfCnPkScKHzvYw1WxiamBjHs11Rh4n2UDQo3+tpCJwh\\\\\\\/sG+R70sSLITboIXRiJo2y0TbD5wT0936VjkSbOSSfYnxrd0\\\\\\\/rag9+o9zU4M\\\\\\\/z0oh6PrVIhE1ekeS5lOD7xRnDWzQEt2vbKJomTYHNHO9xOLjDIE5EriAfHcz9OsVRVduGear9LaxCgTU\\\\\\\/YQE6YlJ9AKoBR9LeJ3Ah3Zua9ps5YuvdiGFrJt90NgWWlEjxy4XtXW4q5KnCg6czopETtCH6xbvkEnb813MPeYyCwgb2qkvH+TmYlDhXTB4tQNleCg7YHEu\\\\\\\/Uf90CS9B3Xb724WdQgodPpCLRskk\\\\\\\/2qqqfD4Be80vsLbaWo1QVMxfiPpGXlSYzkH1XptapWbQr40QS9Elu2I8qC3AmiXsdxe2U6HcZWo\\\\\\\/0J\\\\\\\/MLAkYJuyQ0i+00QJFnI61hbKiHtG21jTmcwp21BGoF+tzES37qShKYde78VRY2RdZTD5QVQ23ODb0yYR6ogL46Cy+ZQcj6KEFAeKF5WZDfNgfWrX827A2kMAkQrera5IP\\\\\\\/YnsOiUUpiGP1uEO8g9RmihlGAyZ03A6+2XeS4gqDgLyk4INu225\\\\\\\/ToweYP9j8L59yltUKeWXsm7QCjQUcZXJS6eoq1AgVqu7mWWTpbQO71LprOmgo2+tjE0w9eIazYdKiDAC67FypHDxLMEjcUb94CdsInB23+LyYv1RU5h4lqNbppJ3QfXnWxAq9qJyeZwye24Iz+gCBdRMZ6C+zUjYnn7ndLII9rzqv2Hj0i8sUr6culFomospPQPtkSnXBuO0QjLoSyg+PgZZJYiY4Qb\\\\\\\/QFdplwGoESA+nJfKSJF8P\\\\\\\/9fgkIvbp17P4FgExwxVCKc6FI10erjxV+q8j8+VTdx6ko\\\\\\\/LY2rcdXsHHtIRSpm\\\\\\\/BK8GSQN5tM1KduBWktL5vOe\\\\\\\/3GYjV1jT1cxydFLjd\\\\\\\/Ned8m+X0HT9mCatDXXLMVziX7L2j7EL80NaXvcJezIRuW1vzBvZYJvvm36AOZNkgXesWarXgMjI1ZIOC8QZdy8VPtBsrcuVpCqfODrL5i5uR\\\\\\\/uFRWogX6K17XqmioyGD8TmQp0zMyzyR+JBklhHNUQgSe1uZa\\\\\\\/Nc+gjPIATFv\\\\\\\/gXW5WQyZtzZXHrrSZTKBmEBDPo+wvlU3V9amMH20o3ryrzdW85Jg1Zaf6dJNVv3sP8lSXBDezzqZMFS5Xm9W7\\\\\\\/bEADRqUeeSsrCBbs83qMzgrPyoJLaBL6VuJCYuc3n9MCjBV\\\\\\\/aN2OkdJY8uqOyGG5EJtaU4w5N+gJIBRED\\\\\\\/BBjKPDY3RqZga8EmnhupP+6RPAHnb\\\\\\\/ZX0veJ5cbx7ShNuyt1w5uxpol3GF07vbVfhoZERp1lLr2ripcClV6RW89dI1QYW6m6jxlo1cOjzvB3evHmGjLmZTZCd0VTPJuLMdBL7TvK7EUyaFfAxa\\\\\\\/LLYV50iRkADj2VRivP6Xi9NkmBKkHznSjXu5MSsS8+2PuqSsdEVDJJZbBLKm3xRuvXjRGLmBn0Rrx2zjXmFWgMqVzxNWcJSy0oWtCFjaqJDtiifgislhvuz4EEiAGMYBSV0vEPUnCSas21NRv+9U8xfe+3NYnezXYjpu6EY\\\\\\\/UtBw4tRY98fgBDNG6\\\\\\\/s92sSN7PMVcOEuYybz1kozXgpUJvRxsp5UEHChzTCuuKVdd40P7oQuJtjPdqT2V1rtSTjBXALU1owS7unWmaayAL0R5K2f6+HYINpCDJGUbxMroV5ideaLHWAMOfkx0DKFIJTthuj0qy41ZAEZvcbu442SvojR0pPd75gqckCaE3HKtIv9ELJ+pzEPHItiQhWZk5gp\\\\\\\/5eylKzZHHlRNE\\\\\\\/Nx8j222ax2RXu+QdTu9Mybnhh0Z8v9L5g+179Q\\\\\\\/IwCTR12HvJGyK0WXGYzBk8wXHL2hcfyh13JgR6FLto3XW+bJZHK47LEYOtR0BU9byRP7BfLean9Lm9WCtUx6N+5uVI\\\\\\\/kCr\\\\\\\/YN7S3RjnYcBMsmsn4AnKYh0441b7u0+mVg0eq1lVRHTy7qlo17BC9M6CQq0JSybTivha54TzkyOWl7k6coEQHXsVMAfqUQM+CgGA7yWLBexRMOvZPEv5t0JsOgwySF+ObRIuukq7GP0G4iXVAe9D3aazdnfq61aYPABFJ3kw3E+MZUIQoSzBHg3Le+Ux4XPsKHby4GwAsEU+GvRCd3Jg6u6UX9w1kMOyG7b1RmUynJUDEE\\\\\\\/B0ONgd8pcJaULgXziU+WNKDKwgrnbkRa0Rfhuqa+zFL8li4bivO1YL8sqOCp79Qvi9oyHew4WTbJYn1SDRGypwJY+iUOv8rt97iJZbMzbDNnZM67YXgyibJKlc3SXOCoHwTpdg0vcSdoaGooTXK3pHOD+ezU5lFxq8HSaK0epYsqyqoco+JdJ4EqB\\\\\\\/zcJxUETne62VP4KUZ5Nx4gtB\\\\\\\/gJ\\\\\\\/Qt3jFc0f6MZhqmH\\\\\\\/KJ6K4b8nJBNuhRV20BErXm1jlltX0QjA6xpt6\\\\\\\/uQCh\\\\\\\/GgLCUGZCGJttHIcHLQGGqMnG1qtMqb4lv3bdHZOETivB9Q5ttpyRm289Uta\\\\\\\/NncJ2Q3FxsAnbDaaxrl4uBAOKcwOXH0L4RGxdRKp8K770hhnFjJBLEuoOEDc1fbXS0sytOJ\\\\\\\/73jwvHmWVxfytOEHAZQiYkyxqkzjIsIe4Y97x+ru4JDcENI+N+5o+\\\\\\\/7tm+RQ1QUXwNhwoW1RVXjugCPDJ8Ds871wh687vM5lqANuNoUwPdFPOSVXJ6C56SGuNbupRPwkZWmGiZVJsqYurDn3hxMvuG2XuaAQyB12JP1RGOBC+ODFuX\\\\\\\/eOKmPV2XIzqjU+UZo0Ta1OjkzANN6SUN67RUuJ0BQTOjjmRI+bCCtYEGznWWHWQgQ3YM765W+ZipKt69NM1EfLbXXJJ86MiIrD+crUdUWkVHQe0A4Dn7KatjQISbQ+xHMCTx60cjJiDl9PaYoEQPz5peZjOX5rWKkE7LIDbpn9rHeFQw4yXa1P8rO5RCecA9gsBG2DRBRJux8nxpJr2qPTZzg97r9fObNIjkjAl1bF5UuxkowlUu7v0HhG5xiTjRJDXI+9lD6KenKsm\\\\\\\/G140Ovla7Z9UY\\\\\\\/ufHx3f0rFqSc+ApbAcy\\\\\\\/o017vl1yeMXeApW97nagl6IbkprejJJhkKyaxhCQ1s2WR0+XsvSBwH2UNtx9vN7A37iMIa4E4a4HESNtUQ+FmT+Sosm4avhkCPBsLtZLN8kMNQxp5IegwBMJJkyXoxvOpZP5NMAVqwAWKKTh1pUKyGm3BoK8oq4RqvzOAT7\\\\\\\/VuW3bs3vDsCmaiclGX8Iva\\\\\\\/FM6VxLzevUNavESoPek3AFqQu9jBaqjPpMZ2PxHGzU0U\\\\\\\/hlvwjuKV3bsF7BYFrzZr2ynYkz3nBr8s3NxZUoTEmtniFVjJ5S8MK5zg8vbjdyMWmtF1Cat+12UXoA30Q25iVGo7XY+FbYQ2Xezlht5Cu+JTtQH7GWtt9zwdlfIkukw13CX2gSheZogSSpFXeYqYDBcGlyFm1d+CoZM0j9djIfIVgKqKvM4KHYYHsjfOD9CQRwYq2cE7s6H6KpiNVOJFbwwKJHktPunXIdo8uxdepFHMNY3RmE3tVe9otZ\\\\\\\/3oFHxt0VlMM3y6RyKCMuP0er7haQmEScGrw2GRDgLwjdYuSGHb6Bqpa4wqRDF93yLc79KfafLko3Htsbu1u2ZLAt\\\\\\\/O1F7c5x6DYha451Fx5HJlY4h68T2j8MBp3JmDuUWBEvOlSRU+muEb\\\\\\\/ByEIrnuAn8lP4KtTAwvFwNF0sFR5OkuT0gay2NnQk\\\\\\\/jC0CYF\\\\\\\/liOhc88BtTg+9D+vBThaq9WCX2\\\\\\\/DbmX3+NcCOUA4BQebDxBCu3tzhnvvrBp+28pxCahednheusbMcdn3gLabWUVEomL254AkOS3e0aWr4Zmzz4UQnPhSu+nevO7LmbU9tQyKLaha5pFOtHJv751XLUGX3qJBxtujP4XnleLW1xZNqAwde\\\\\\\/zUhFAdIc1pbD8ko1kL1iPDJRT1vv\\\\\\\/fdQxIsNwxAkWDugLBxJ6xusbNOpRV5zefJB39KflA+Su7aDCjegOYoW9YGLybqNZVi+Or0p0yXbsD3a35vpQjLkxQKf+3QQNhceuN33BQ546Gd5NsSrw2orl8OY1OBCPlnFmbYR2\\\\\\\/4dvPP9UF+NXjhzcWxjGLgPg3C2xjT\\\\\\\/Yedkd0JiTK318GOSP2FAKlH96nJww45Szj68lBr7M2OxCYq434b+8qd0CUwVzhmNqKkzWIBFxww0EZ8\\\\\\\/qyp0YAkFl9VYk1Ijo6jUR\\\\\\\/LGzHaY7PqqjTqe8AbZdVo+\\\\\\\/f3sPy4XmgNp3hCBIWgO4xnSjCct\\\\\\\/kroT7jbvaO9++GLETrvDZmVNbLWgN\\\\\\\/5QtRp0AXly5szL2D6im+H2L1H61eaulTHhrZY1lA7cn99U\\\\\\\/ktQwfCxKubHTnzbEdoujtJdDfUTC94vWLaNjgbLdGsbxLtpqAz4zRJs\\\\\\\/ekdzI+ucilOm0kLaBujwxNd8+90eVda2+5gMRXV1tYcefKTbIGs2nOilnfsGGqG5I+2KYpIeEkt3BWRrCHk8v36gAYTZRSfAmcLDX6ssZ8DEQLN6TUjP7zRgUeRnFptOo6fqhaqIsaOAacCBIvPH3TlZLDrSWDzxzgC2R2sXo1symLBJAq7xcZMR9qe5tTvKBB6ZR0E6oku6YwN4LgRVUhO0UzbInQ20R7wCwFJ2z6Q6vfpeLGbaqfc6hEjHYEtzr77qMukPxfsGeO7IRZcNirvUuEo+Xe2rck0NO7N9bUjqlyOzEuK7lPrN+tcO5WFFJ4Qojw4\\\\\\\/Lx0T08owrBHvdwERGfx7HK+1Zbdlg+kzBmgMO7O\\\\\\\/HVsvzfuNhpEchoTlLD5WkKpu0S7B1fAc242Cm3lc+SmdHzDbRT0jOyKHW7r6Jd4FFoMmEkxEqngJLh8a2x6eQ1Iz\\\\\\\/wH5yWdELvXW\\\\\\\/vT0O0XjBG4WfSNf\\\\\\\/U0dd9ICtkjj\\\\\\\/5AcGw7fCAA0W80wlhsDddDD0QhR1P3PadbG7sDUTGB+aUEkptzp3sVP6bIbz7+3kmoV6D0ysEUduEG56XplSMOPdr5FWXwPl3my7c\\\\\\\/bYkJuDBIRX5rOS0VE1NgoHY\\\\\\\/ew4SY8y\\\\\\\/etUHOUK+2vruksOg0IcvtCnzlF8Vhg4P0UfbX+2XtZvk2uoVgiAnKN0SoKH9Cx+EvCQB2tAj5uRhgxjT4\\\\\\\/wZ6eWcdcwzvnSDJjnPujXP8DyK7dl4f9lEF2JNXBwd6\\\\\\\/agnfEL8Dl\\\\\\\/CtMB5JYhM4No+YTDOmcSqEJ5kckbFuD+\\\\\\\/wmDggO3h1lRs91DsDi5TlOTLPNQz9aR98SxpJraosLDjS0CM3gg2Z6fjgy3aMK3yIQPJ3zhRLGDQo5Ae8kPZGrglyVxZraROj2O3Ld\\\\\\\/4jsiccMWibxQ+ImA0MLD3kl7pxML2zmuRN52SYXaP+1ivE5jcUPTtW0rzeJekwJYi6vG4h8PiIDsym397vodyNtcX8wHDg2\\\\\\\/D0D9CU8vVmUjQ1R\\\\\\\/hG+1BuwsyJCeewP9V+0bhg9w8d4GibY\\\\\\\/r7p87USyHHQtnbMIbRhklicXEd2OOGfNmToOOFW\\\\\\\/HiEAvEt3X38EoPcUnylebiR3lS9R2AREqec0QZgvFh3SPs3HP1H3qCWpxn1Yfo0lobOFX2oh7hbnOs33p+Fr\\\\\\\/5fJhQ8qstCwcA0xILRfqJvufK7FvfNPwbmpg0HwhDiKgz3uJRkxXz5Ki3V3UY16yZrG4vbmXSdMjuLZ+Yi3NO3C\\\\\\\/lQNahttuzyAj3boxSDCqT8YQJvmq1ztMkqyF3tPhqmywwoZIp3mo6RtR\\\\\\\/ChiMJA0M2vwcnMCZT7oIOW6LrjxyUvk+QDA7hsc5Z1bg7YVl\\\\\\\/BxzBwk7qf64mnbeinkTJ0jRJU7lN2ivlaFseV4yXoaAOo9KsUz\\\\\\\/CXYTXKbYo2i7JqzvogKQwsY3HgjiutK9AewBdjFUzYmkmmuFoC52uZYSUw7T3tF34P5Qg1\\\\\\\/0ahbnU2Jfo6a3JQ0Ir4EgZyW\\\\\\\/kBBFARTyKBcg1cXDaVQBFHMC4Swb1lUe3TnL\\\\\\\/xBjlilW+0sBhVuuj1OqTG5ht7oen61PczVSpKo5zXxs5FR9HzbHNbgdcF\\\\\\\/a\\\\\\\/J5RBGMEN4STzQO0inJ0R79ChFHau9rD7MyAKGiSp6T+\\\\\\\/ZGjVey1tHNLpumVDR\\\\\\\/IDexBLE\\\\\\\/PiXn5FhDehmFvb30oZbNfd+wp1bL70XbOxEYahXLT1jHXuYRdHG8U4363MNSyx6mSJTksf1OWgRGjFPOY5mx2Qu8BNM4nFuv6kYW19G2aF83rAy+n8UCID5darW0g+SR38+XGlwZR8cLppfpnJ5qpWiDs2Gl1uzAKFLi0nRcAB+jtArs4ggC9xhojdo0l6we\\\\\\\/ONeuxx+Dti7irWXmEyCevME70fhXgmEoUYwgCeY9i\\\\\\\/QPDIWKHnDSNaYB8GT4zlVyeoSWRDuIt2wJ5JjyLqNX5IWDgOOr4d7q0Rbn4NHp5XQH4vBr1eNuVu7\\\\\\\/3DYJJFdJD012Yxev4RnZcg0wk6r31tzv3Ofey9eXji0XGbOAdhBfKzKonEeXasZQ6aaXmztr1IWFplqxeEp5ZdSNag60zhHdmwiCQL3+SpdDQyqIh3ToJtwwr+X4Hc4cO3QjHpU0DJ6Pu+Q6zyxkZxnlnL8wdnO+0TSXo\\\\\\\/hEGytBIk2bgWE8dlKZ7BN7IUTdL\\\\\\\/uJjoMFKFb5fEJ47lNBRnwabO8nygDBJt82TlWG\\\\\\\/WjR0zKKzUNhpgsMvN0DK4HvSgJLZTcJnCXBcB6kWSaaekSpMcIr7z1AEycBJ+8hhpkTVR0gnFzhHrlvdAhsb39OQjX83ruq1HPRB6dTTQQeuduCXePs2QWfXmARr\\\\\\\/m9UQRMkhRyn8K3a3Uimdro2rcdIFP9LLV6Yy3EqW61ybkk2EopLe4cY1+IelX5HLVNQtJ5YhpE1mYLm78UHq8PTTseP4hTCenUD4yKn0efK\\\\\\\/i9h4p8nwfYI\\\\\\\/eZZdx7Kp4HE2vFSVRYMtiIFlJxjLnj68Pm\\\\\\\/+5UgtoEZZmiZQf5U9XX+Xj3zWgeSpxDdTOky4OZ\\\\\\\/NQnE\\\\\\\/1EbtJj0ZLyCR6Wdj7Wg+qCgHcCPC\\\\\\\/t4ubTbOFnfYVZeYCW4qWxs7i968j1xud9LgTZVVE+4C7jmzP9SrHE\\\\\\\/42AMlWXzeIl+kjEMIg+izq4kdIfFQDgstvfl5pd6x\\\\\\\/JenIMk9yUrjBGhRD+R\\\\\\\/sb2nHetpjxVeJ8eAtQ14U83+MRwQO5Wpm1KsHpp4h9qsMYMjxNkq8KJQBV8N4eFW0OhG2ds+jiXwH2gcDaj\\\\\\\/CoLKog0xLfo8pqkPDZrxK6KkbssaG2h+mzqdCl2ASw3bcLI3KwMHTavm+yPzjSboG9S\\\\\\\/HTcgjjc5tBVPZGNqIly9+9NKwk0to2ikerynbAcV88Z1TItI3nWrpU\\\\\\\/1kleJeKCYQa6HviYKuE\\\\\\\/ZS8rss5B2g9POcuyqaKnbliCnihHBG0A0ik2qxDUe3YX8WdIGeMLbp9r12Iyjbkz4T8PoAAIZs3raEvuwjrW8ykevTr\\\\\\\/587I9fi1SqGva2fVHXxfKslR1uE\\\\\\\/gHajilYUjAmHy5JlKrZ\\\\\\\/ManBsK9AiFbSS5yqiqD\\\\\\\/bR2NA7gEmwM8mL63kGWa\\\\\\\/NIHdkck2K6S\\\\\\\/RjblakyEanfMB199DtjURAQYPAQ1OKZBovJ5bNK\\\\\\\/TTdS1w29eEEIwujbGRLnj6lgZhbjjgbcy\\\\\\\/McuWpiVivvBdc5x58vMl\\\\\\\/5Nj7+3FFVA996Hr4HZ\\\\\\\/MWXgO7dRvPFfbVqGeS5tzBpVpEjYBNqdgznu7pj7rP\\\\\\\/lw0IKqlFro83XA9L1k8uPKXbUxsCDMAgzoYN\\\\\\\/bhsA+S3P3o0TArVvz4SeL6ZBvokWM8m1ke\\\\\\\/AUidkv+8MIVLIaUTzSQa0hRL0riF4nBaIN+A6QMTG5RUw4r9+W3TOsoZz5OIxetF63cbZbO82qwM7Sfojz7HTJ+r+P+L3ToJcxaA73FX24SVXShnJaq0zw6TET53qNHv+O5IylQ8KRLxmgfxQWkRBtK3v\\\\\\\/cM2VgUZBVwpTPFb9ocSr0sDWTCuM9pC56FkjxmCvNlnZGesJjkt6mMBhTEVMKfilbukbPgemNChiYmlhCNqSy0eev+oQs6Eb7BBDpI+MGGJp\\\\\\\/4x1liT\\\\\\\/zDFhfKtGI\\\\\\\/1Y+e+MolPK5Fied3fp1EaFyT3QHZuMNgmBPutj5TEPyINPfdrF0\\\\\\\/6QF5hgUDgOv9sQncOidSWUOkcVvsGCEgIVbxFLg4FrGccL1JNNDnc6fKKs4NrDEIEtaQ\\\\\\\/9mtWFdf9GewXjsZGDA90i3pBZa7U5nmVjhugi+BxOLBTIWhcdQnHBBD9ZNZ\\\\\\\/wg30XQEiRxVIdygh2vPxpU38zOg\\\\\\\/4yffqiuEBP0YOIfNUwGfDsxpeQLsJjz37xFlKcketlHJVsyWCD4xwTch4FYZ\\\\\\\/y9WyR3lLnOo08MI+MogI\\\\\\\/wQ659DxA1k1yXTSJGM\\\\\\\/jjSyMtwKMwYX40cDx7gF5xfcrJsTa+u6HpQkjDQ2I2tW8TH46PYfu1Rke1dZQd9Mx4APjo6C3hGdqtQJsyaTxFUoBYEsp+CfocJtd1oL3k4iLDWjyfbLh4FrsZ35sCu4czplZh24okvaMlrYbH9+iF95lgpiVw7pGzJ\\\\\\\/\\\\\\\/G3bKYDYA7QA04pUuFCh6pVdm1tQVQLIGTBFDRSg5lebpWYsnktTofdeGCG44Lo0m\\\\\\\/g8TMtEaZSNRhpu1zCLNaN5MVuh0BXM6OFom1ZL87yq36AfqXMiet6RUcEwO6ORXQqHFbM3Y\\\\\\\/YHaC8Wt3torEXkQmVUDdXv4qhnPR6NmjJTck39CyT5Np0Dwq9mW+jBIpCC3aaM6Ksp1jzovQaVefXfGaV4wt0ZRZtyEFa47mSPPrE7Zc5CMdth\\\\\\\/mKBt853mzUqX57lopAr7jemSjTFMltKJKyBsec5ZAtjdFXWeqOEfFE\\\\\\\/kHQgciGO+w6QktCT1OEMolcne39hVqqNVb5Sgc3+09v1OCw9Vov245L4hDjV49CrdBxD7H8xjQB97JM6bOs43koBe9qkyS3n13izKQ1WCjkkQYl42NimUw9ywB\\\\\\\/afq4+Udbk7JES6tzGW0GlbvV84ZDkpZzJo7pesXki8Ib3kneus8+wfLIPp\\\\\\\/76j+bRF5WNbSkXdi45sU3SBX8PTDWml+m\\\\\\\/ol7g+ifQOMelR4xH7dALth4HDDvt35rRB6SjTYGlaXL6b\\\\\\\/j9xKzOTgtjIVyp1XUgEqB3CHXYIskfUJ50b29VsSVttU5nejj2WNtvIbBOAhdQQ8\\\\\\\/8QLFXJYjnRsMDeqtOg1PxIIU+HWifwv6i3sCmhCljpUAwTrFtj6MfFrIa+d8FSvV2OyFVFLpQT1Jd35IoRs\\\\\\\/r+uVgBK4iYXgYIG2CmfUn5Dk1wvoBWEESt6wyKQHhQ6C5DENpgJc587eTgn5q2dhg+ISRbdww2JO+P7RpNfq3vZ7WzG3+igtg1RTmTuVyi38it\\\\\\\/ZrzYBAHRilvDYHdxFLWG82pWpu63kUcUi\\\\\\\/I2cMcii4SV0CyEOZmAimw28rxdmEnwU\\\\\\\/tR1NEMpgysIL3LLX3eB+AKoGKadi8sXD7NnsXeQKw5LUhFaXQ+E\\\\\\\/q3CPpxIWLIJ080SldJG+8UKwnALy69dzHVEKcCHq48EOukkiy2aQbdyAMS+SQDAKTtIop9WuNPF3ZgSdUMBhvjSlXCIaoHGytwa3vKdATVct0qAH6wxeiXpL2OEQXb2qfwwBm9Y\\\\\\\/0SnUhhcET0bIYdxsk02uflbN0x6MjeHJFKZoZJrcPpqQAPHrO1t9q\\\\\\\/aVs0EQW1e0JQgD0Iwr6uSWuBHakBWFu+USrmqqKkQoYRz\\\\\\\/sYgbJsbthGAVS9MetJesFNEa+y7I6tPcZcw1dZvuTP9n0SoeOHvYtlVxyMzIbSREuAQSMRB7DiynL1qhx+U95xK\\\\\\\/ZvFDkxR+tZEYgM0eVrsI2u3VSu3rq4rmNt7SQwO2zWaj+E9Wf2SBLXhgIG1wVTNFkwxhZQFzmkxBWUXzzgNSEhYgPPpnsXn7+cve0mBaxD+OvP0NNoxq+3pZMY6Vr9z0H8V9Ryro7yBtTYukoT9EKhnVKEBFSbPDO+IGqPK+ZIshJ+0MLC6as9dO\\\\\\\/aEzNFV5KLGyCUSU2LoboAlOwwjs1CF9z2mef8Grl\\\\\\\/QFp9tJTYhRM1MopPU4Qc4GzbH+ElPohMkKqYxHhgDc34ls9V++f5ktGfFlgW+opIGDB60OpEKKOIKjC5e6KfSnIaDfZhrLGrT66X2jsagOvOnRWXIqOAch48Gesg9GZUkDyoFfGzrTE8gjAJTmWhgpZ0Yz5J58CTBIcKmSmwbsq8j+lDolAjM+yuuWdCsXTTHEh4VxFdljE1ENWS2FnOB+AjLZEc6HeFc5q1WfB1+YvofVMFlhGTMjWpOTWUN7gKWq3KNJ9ApwVvrs64RSGTxb4QvGiLs61eXAhiYplZny1jIOZ2rr55aP0wHwLUQ6hOvRri3M6+BTGf7V3Oa\\\\\\\/2vVyJkE8S0EyiwDtf6GbpQ8P4F\\\\\\\/W0rn3ydJU7anltS19EsibxW248LuZV8qnFmi7htiGvl6XiXcW+o1DdpFeE+fHZ\\\\\\\/ShrjZhaK4zu\\\\\\\/+xWwPt1iYn0Ugkurb1Sci7GsSNV\\\\\\\/pQOs1OQaIytaCND22\\\\\\\/Sg1YiDJiG2nZctSuzW9Cav4RYFT56JeuM3S8CYEbeuPTCcUL4rn+iqjcChJYwr0uaN8qXzeEO2zjPhpKTA7xFDpGAETdlzHOoCGId6fT9ZyJh\\\\\\\/rJEWLfrlod0\\\\\\\/fkm99jzTGgf8trv4D2QyPkh+BJ3ek+H4LonL79LAX2uyKSxbTyGDFsKA1YGqJlSsqZIqiyZZcMpBpVvH2SlG9JYEGNF1Pa+3c0TvhznVzcyvq5qwo9S4gFXdMA2GqDEBbptK7AtzFCbn12HIv9Fz28hGiIQklP3LA3Y8R22yP5I5kRJrj6NRasA80RWu\\\\\\\/LCFpkRulhkHFayRbun45+iMOucWL9iyskxV\\\\\\\/aE5VY\\\\\\\/YH\\\\\\\/fOauojQ7R7E8\\\\\\\/DZVVOJdV\\\\\\\/P1ii\\\\\\\/LbRUhV5fHn5o1VoKbiVE0BeESYfMm0z7lu2K+HL52lY\\\\\\\/9GwA0FVNf7uMOulC0hDxN\\\\\\\/BF7B6o9nUdAs1dCk2Cm3VspXJ4v\\\\\\\/PlUuvgbkKu9nANaJbkAkPDUKhGOQ0fqzWsQsKcMtp8mqt\\\\\\\/B30WgsGKH839rxvxSh3fnwuYaKFZML5Kuy5vppRd9f8H4iIfpabJoWfiCmfGDzaBMTohHcb582ZcFcSbwGFHcCcgOvgfnLzijGB65z+y97F+BZswHlBSRDQJb4fzqPXqlBXeyuaOaCYtn767uJleX8DTF1HE3UnqBbKkIp\\\\\\\/p9iEMn1KTV5rgseGccLbMOL352jOs5Wl+vjH\\\\\\\/OdyJFEUUu3p88rOBUNwNfy23CyyFNGSM9PSEyqv8vG6xoyVl0Es2y\\\\\\\/hwLhezZ2tyAIDkuockW\\\\\\\/3Z+7RFideWHBWZ5HzbW2M+qBwyXQGwAaBxGubadgMRPhDewOCFR8RCB9HSaiwDgFGwm6IbnOPdZcuCSIV83zDGaSMbt0f6fA0LdpY9geKhLDAJBkCMvctfep6hskCSNE3MNuXptYZZSk2ogdTLGhIhJUnkVruQN4URS\\\\\\\/X7AAweldikAht41peyJfEyj\\\\\\\/20xqbUjIQfHEGLZhChJZ0Kz6vqAoQTiComnkjuvfCMPCZgWUJwMfNM1\\\\\\\/8rJzCeS\\\\\\\/60TFP5FpbvxAtG8BJykSivqR72d\\\\\\\/hEVqd8JcZR6KWRYktn\\\\\\\/OFBNTtnMkFEui9BnObMHQn9YvA3BQWTZvJlGUfy8bewZGGhO3uA3bX+vL8PzpGEZ96BRvWJ7P7HY9WVOxCdf1DwSPR0QJcoLBpBtb+nNKBawETuSRpsWFTS379lBJClrn1+t9pir3zE7gWXjVCTkYLmUZua9XyPURbtQGLezg4HGtqXxod64GSEELD8aFmMp5h23nNd4eX1VoWpPWLPvySsUz\\\\\\\/vAtkzCwnt\\\\\\\/jONskD\\\\\\\/uCFlp7OZxbzc86v57b02An9eWAVyg6OdrQh8H\\\\\\\/6Uco08Th8I\\\\\\\/ymsA4gqmNhKpRLD37eV4\\\\\\\/19kRKbJIrPlj7NUhwZxXz+13a9fr9fxx0EosZN9ynQsjKzg3\\\\\\\/h4Fg6fudEpeFdOJPne80xqFbmcZ8uInqbTmJ4g+Fosg6HkK9JoQluJVdZJ0jxiB\\\\\\\/nl5qYCIUGNsojD0FAoZ2KCjqmD1Sgn9N2\\\\\\\/9THpXdSyscyKQ2TelPc9ppct98vmVF37PPScb8TB2DxZDD8Bwt1p92+PzAqU2BLmXdnGl8ZUwXt6MJfXyscyfp64MMpwD1jYKOTzm3PAn0Ov6tvUKY37oaRWFaF6b8LKjDGDG0eKu4w1\\\\\\\/47GxyRrQD+27V3QN8efcqax4V1F7PO3qcIE9uPG\\\\\\\/B8KQm\\\\\\\/bM6NEImPj58ZWB3LynIs5fPelaSQqB7My7y6lTiBZ4M\\\\\\\/8w+cS2JDXlna5JUW0oxEPuNZ\\\\\\\/ec6jiRAAKvrJku+0tBSAfZamvjd+eH1U1ManRsQrlqZ5KlBwzAZmkfgoO1MaZM6\\\\\\\/t00pHZxEvju7h8fnqczlYQ3EQkzYDucg71Mc9m\\\\\\\/vpwssZcPgZpbdYovU8\\\\\\\/Hfy0aHvk8\\\\\\\/a+ctme91xpra7neSbOP2zYwr3gh3h7xFPJJncLnuPReTTGaZUMpq75E9SGAmHz3gisTreRtBAySNErXYQJI25P\\\\\\\/kdn477ICrJ8xL+QMGuqcLC7cbF+Ibngq70tChA7fc+rInpqxnHD5j9ol19W9juysR1VMxpxQ1s3j\\\\\\\/ZXdC+M62yuBdxZd7ZQA50FW8wzaaYZhpgpmgOrFmZ9WtVvrcMQYRapuB2KkMvkw122MVkH3mqNV1+yxzZuGguHKmOivhYXNaWU6f4yidvXyLJJ3rByh96FAaI5Yjh4PkU3IsYdsLxZcaVWibeqhMEZn\\\\\\\/haO3LaBWrZpJy+C8il+5blL5SiaXJCooIVL2vGAn8uh6CtJCnEBKiCMFBu4JNXmhaX2Sx9ABW7NFyT5C0hoUXNE8l8Ms4VGpok+VTJobX\\\\\\\/7bOqIgDdmfFbLZFaTa6Gyzv5+\\\\\\\/Jxlv2qwJEeQyMF+iM5kV5YPHkpH7eoHdRnODlee6RYObjW2sxKUTIT\\\\\\\/\\\\\\\/Y+zwfh97GhctRzwrDYq+lDhjQ70FKh3BvOSUEcfTMiQJcOo3pSbECdFiULOeVwl25WIlQpNg6HH1imbcGG1Y4Q2AChqbVGaSRsBzrYOONxbjHaJgman0KCBuob5\\\\\\\/2fNlLajnTGHtTJ2ePVPEBc6eGk\\\\\\\/Uph+pikKbT3pOLWOIAMqe1Oe2qwjjiMN8n5KuwepNXYxFfsshWc0MfwEJEbfhFt3IRPTM0RR52pKedJIxlvEcyOgvwYYEDbvoa9KgM0ykuFp2Hvg3BRv9ASm5bcm1hYZSDfRSFqZBlXMlln88iZHkTHhV7nj+ou5vmZrJuPUT\\\\\\\/PGoWpkgI1cm\\\\\\\/9hEtyT5chmk0M197jqXGO4IR8Ypqn+uQkj8uqsdh6JtyWCxiFl5y0t6LTrHSWTpUCWyyn2QQ4E5hmxG+p9iJnYI8AHe0FV6S2Py4dE28oFoPTyAXlK4ux7\\\\\\\/m+5ETo28axRquDU7QWMv7rjtXXjlinBsm7ta+HjShih+iU8cJQfNyrvw\\\\\\\/NEsfveoEMuEgPuLKsIsJU158Drcn8vELGzPwo7vtHeO12BhaHlFYAmTRIQ5Q9wKqeN1t+0SKurWv+\\\\\\\/pb4JKHL\\\\\\\/C4EiEDoo1LFaqmovZgx2i6mEnsqN1o8PzHSW1Ry5SeDho4tstaG2TESp9bQeU0A\\\\\\\/40823HPOgO4sm8F7OBjAaRrKj+UJUCwPo\\\\\\\/PFKYD36B7zE2FLCBUsYw9WymNicXBnwPwflmEcobSLpTCGhj9EHMQ6Z+Tb+4CU9CkIUVLXIPxt\\\\\\\/crQQjZ5Bk5JpnozOVg4THtz\\\\\\\/TauRB511kc0Ji+EkuxoJMuSeG1BpDuLQvZMcg1WsFJctBF7DhmuEgLDem7L+winJDDf\\\\\\\/P3afEIYmmP92dfXj9B2uIOkpuLOOVwQFuT9d1pnOJb9UMP0OETpK3O2mGzyCD+1Uw\\\\\\\/hx6DMuku74T+3ssY7wvgORtDaHXsnqUSi\\\\\\\/lWqwWJMP8pK8FUI4\\\\\\\/tWOzxFhiMLoXiBiYfzjH8b6QFoF93zx9WBL8HpvS5c\\\\\\\/44lPSA1MKR94sH5quljj6UNhk6Ylb\\\\\\\/snLEVZnq6s0v0OkivB6RYh81dqVVljN\\\\\\\/tY7EPcU9CmS1l1eUOvVjSSib3G62Cm+Tng0qjOxNoFz0NlOSKkrxsln\\\\\\\/JdgPXL0l5eZAOP6aGECZOM5EFFTzRpebmoOQQplzj90BIfQsoX8MUXK2XZ2z3GHhA6GoyQPR5czVkfKWSsRvvywFEy54BVgCaJjxH0hicluHFiqHILRxS\\\\\\\/+HOhZguF0BzdJiVH1Fo073iDwX8Cf05SibrZhKYK0KXN+zjGuFQe6CEMA4nFcACUrQbF+RUzTW1W0s0DpbAux6UMDq27\\\\\\\/A2jOxYzNC4l7SX\\\\\\\/k47wtOz6PV4xi5olSX5xDpkcz7akERYB6AKp1M9AzkI246+QJuBSYOw+SPzBOeUdgurq9C91iyR\\\\\\\/kvxQEu5TVG+QB9axJQ48q0q+EoRBKU+gM4WP74Ih11XkVXx4306\\\\\\\/Hb3PiIljcp8WgFA++yVFAOItdN7zQiwWsIs7kkuntRx6zO+B+78+YwZyGe8tRTmB2ouuBKcr9zsfYgmHqeLDDOsTw1ss48fQN+f7VhZAfHi263WhBk18pmSOj+iaFCEqqLjgYvp91VX1bKut18oJI62vWiV32BRnLGw68mIC8xOCeWcZ2SRl\\\\\\\/XA7M4csdrB1Ngxb7GcPQQ2Y\\\\\\\/R2poG0hyvUVY2j3NbpPtMW8yG+PeqJVfd1u0MHfI72Xrg4BQWxFr6DJ2EtysncERIak\\\\\\\/iuzq9hpXSUVlVxmla\\\\\\\/YfUg8YT4b9PlK4iWmnX8zWekEYYVQgqkipgm0N1nKkaSRsrH3tN8tgU236HV+VZFiJ7rHIAWICRO+Mo0eG7fx6TaiusFzCl\\\\\\\/r8t1bsT2byYPJ6i21RRqvCNSe1b7Yzowo7jYQ+yQucsTuxW+oWpPP3gv+tANycuw81kCA33gPsp3TW9hv2HTieIqKqnqeCeYr+mAM6xvrKtyfE1EzU9bVd5DfD3INe\\\\\\\/hXUEQAYoX0huoPH6h4zbKHISTQC+VgWyGRv\\\\\\\/mD3mjP9BmEqGHVdq+yxk2ZRdnW0rw8sZ070WBvwI4fbY+lmwGibFBWUx6EA43PNpNxKjz3gRmmoUwv2M0Gop5HdyF5lP5IKIaQLGPfVkXb2Lvtm3aoD2ILRI\\\\\\\/oOeb4Ql9xIV7si8kXgSGdN1iKMAg0iN0JLE1xjsJe5igczWDnQ8I8vn4NgippxJY62Q6PUphp03RroTVzSN7rMcTmZ7Ze5P2aWYwCzrj0v1QXJIkrFgmfzC176cPs3fzATyXAIH5T5sXSD9YPYRFZ28ASEzjflKPmqZC85n0D5FyDMExZMUUSNORXEFmu8Yy9TegfXszgarV3B0HaRz1h4LF4A\\\\\\\/vZChoGIA\\\\\\\/a7hHjj4mp5BHzwYQg2yq612Cmz26TnRQnCLkvwoZxxFMC4QllttqWF+erbhFtUOeO3Ujnv1Dj9hGNng2XmJEHiFTmNLCO6syCm53r2GNSm1ZPzqhT4mDTTRQS5de6Rc43c0+6Or6Ely0k3FzNM4VFztuziZJIk2eKV\\\\\\\/HAKk0Lu\\\\\\\/+uzLn6NZf4MufVkVUXjgzusNJY6tJZrBK0x11sUMdHxEw1yPyPaWwpGL8VpZ8O+72LKSW0cRlkMoxHc88q9VKoNoAzq0df+yFlgu4Ogn3kv9ouQ89PCxfhSyGXSD\\\\\\\/c8fuJBah8bajMMp85cXIswXQeK0HbKOcvX7UeW4OjyOs6OUzaKvwmqVxlq8GkwcX7xw5ibvguV1sF5RKgrbhVksEOkzeln2Pj77iWUYT\\\\\\\/RonKsydSMe7G4fsy6L6bs5TL36NvD3VbXvlBeS1uFu9tKhFH\\\\\\\/YzsVMmsyJCFx2ARBvRKNtzTlRGdux5Dkh7I6RkHXcECQG1FRUDarsXESOA7usRHDVZEHXkaU+gskZDiDO2Achodh5FUH9ZB9h5CCdcFcGXFmVUWGqcLGn1K7+3m1BSvH7mJjq9B26soQmOXRjFCHBIae7h04wiJj28XdahfcRj6P64z9gvnrB4+Dm8sg\\\\\\\/RsIdHUkTf3Dkaxbgu0TrE6sWXSzlNLqDzbEFJcLz8OcoBnXz20acOK9oCmREVe++x5t4g\\\\\\\/RHYJH0NG8apPHYsy\\\\\\\/+CWbiJgcD+Iu5G57euJtzhiElp9UqMHD15shMrcumyo5aWee74r8YK0c2tN7wI48I5IiDqmvrFLO\\\\\\\/ycbuHl28txudG4Jj\\\\\\\/8BkGm3dRDLJSogln6aJShvptxFuPpvBlrTQFu+zXjs7flPxMuXIQbBxEkhLFbsVI0F7dtDresDmOhD72J2KvpKW2Pmoixjv8XVIj1loZg7OdwVOWX\\\\\\\/nAJiYuDkUNzrv07f4GKzmwIFnM4vXFf7DoMhbHBGL+joDrQK+MyLQ0O58aeFRrEpvFYqhsSQcdo6biBxceCucbiP8NV08O4ofUkuL4oMKCk2cdT\\\\\\\/tVFTw+kpM1gxQjETPG0Ykl9NnXeOjZxYA0B9itC4iG5rdPopf8amPutFsbN3uE2yDcZoSHhUhteYYiI416kHwiQDbilp2kI\\\\\\\/5psd0b2qxJAjprsYH\\\\\\\/fLgkQhR2U5d8rtjTg9\\\\\\\/EYUsPo1EhbNAYlpxBoM6hJ24nWOi+P0NaqFtsdEWytjJTU78QWHDVtZW3dChnEXjnZCGcpaZeEudQxDTO03lYMo+Gu5mZS+7li4\\\\\\\/1qwgaZVjqrFmnzkRXhLqQfgKzsnxWU96cyhaw8o5RESmstkXbHMHXicxLTIL5XRQnpKA2PWu6Ejus7E9qi\\\\\\\/N+lUadiwgne9MvqMJi\\\\\\\/o6thRpeFVITnCRaKaWKYAivBbSkNQpSZn8x4Kb28KFK1sWs3lPkCVApurgPW0Zm3VniBtaJZlNkFagjOJKFa5EOpB5jAVd1\\\\\\\/toXf0+DfDp0zHRI8jh82cAWcl6+0kIgZBmfEp9UYEKcFdwQksF6JQishovRbSyHchvu7kB3CYIb8vUOYMhdDjyw+76+26iCufqTpVB7S53oy2lgTajPUP2h11YHQ+Uupq01e8ikgUcPkJPqY7QtaAAmPSu0TMmKAcGdrT2EyxH512t4T0\\\\\\\/4mQvqirsFVKlqQxj1gPPD791fUUPWzDvfhWnQglSwQtdHzFujVk8mDbFpqMl9TVczhmSiHlmgLbNwySaP4w5rLCHjn6fnVUlrTnl65awLVVrdacu786e6QZ+fk73QtClWW4E\\\\\\\/pimxaK1jZHUBCv83AcTCHvme26qkDw2LCoCcYhNIkBvnlPlqE\\\\\\\/DzeSbRVXGnjuSlQIugTXKBlA5LElW5XGEsdi6IGOwhh64kkV7I8QbGGYuczeTPJ5T12GoL1y7HCK+9WcSsB\\\\\\\/2SyuLmxLjplVqECiF1DJkmoo2SR5txkg2jY5yZorNQBTsovEl0FpnvBVJUlfxjNUVGCyrKdq2pOoNe0jvHv882rKc7HKDjbY5gzkbTX6RbfSegdq\\\\\\\/VVFjE0Op9x9OOgv1DHx+7PMta3nQ\\\\\\\/ek+24ltcIw6k0+pDrJVBMllkUlc+v\\\\\\\/\\\\\\\/rFKSRoZ6qMnrhmTUR+DbDs8fB+rzod6+auPTp5PMzWR5Pqh0WG5RZyHLlaHWW4kWM1Q1w63WstU9qfvQdqwOr3kx5IwYJ+9ftfZOXwYzrdDbw4jCMrIfyG+9NrCBtQ6yRpaVykt8nERL4kr+HvRCDgGQmx+8Vw3Dym1Z5ptlHxekacJ2mcjiPZ6sPDTxsxvjLEDFzk8Lj0mHqAxeHKM0RAYJbboycDeTaCXzoXJqurDK+OUplJFPsZV5yvUM+LmV1yM5ucaNRDUoQ7rY6FGr5421J3i6x8hFExakFPOy4rSOAyW9xCFPLsrgzBqdbZiZyhzq1UbEFy2mqn7SzUcuGvckXgVduy8WnnJXpDyh\\\\\\\/Er6BAqKCXsghG3hBMKWvzem4FlkTihsiAxTt2JyirVD1DzYNDEqg1qcsSsTutJ6G+2RAJRvi3NSkArZBHIByxOrUE8\\\\\\\/eCCQ57\\\\\\\/8vu3OZJjSQG3i6IJvNxvVk4gwFbQtcicjHyBsUemNwU3fY38Mcptq4u8do34ib7Cr9x6TlM37Vi43zRom0YmlNgCoGBgLdSu0HUO5WeIpT1fsVIWXDIfl0bKds+q+7qbNtkFUIrlDSwjkKbG7YrGSP\\\\\\\/AsOj7KK4vwp00+4G0iRPO+7WHRGB2rRVOHKKZql4yGl1nqKnirzG9ftR+6FtDm5LrPDbL5wSE\\\\\\\/8ylCRAjbpOusbdWtdr7yS84rXGunH3rAA\\\\\\\/B5IdE\\\\\\\/cnlOp1l\\\\\\\/rG8cyRfnWHvijkqF39gfCYPCRKfnm36BLxyl92U+pFn6GYIrFYd4qop0\\\\\\\/a+c5l77BW\\\\\\\/sGNkKWN4sM+6OOFgQmprFw6cYNNt3wuDSEotCMirEAWJVTv3xEdw0HAqrQvGcszu8i86HYmZG9r\\\\\\\/ykWBMWV15PL\\\\\\\/LVBKtIsld98ePpx\\\\\\\/Cne7LuB+ouwvQ10yfdZ9MjBCEtysNDOHU5c29r4zkMBZ7oA4mVuZAZ4sp4utDEgWqmIq\\\\\\\/Qs4LcBq9QTF9uwD7rKf6LhzGevXbdeQ6GOwIeu4oONixMLz2GcBLTheMKXNAmjTky6EAcgx80My03tJ23\\\\\\\/VSrq+VNmWGKMR0mr3FrGu2rONKUYhIg4rXIMy0c9jR4JGhlQXJJloDdmKLtsCNu0pjl3J0QyXmgXk2WThmRIdukAlwK\\\\\\\/zRJMCRlIx3u\\\\\\\/oHy44Rm5GjqdyPA8zWlnfN9kIpyqGNP2GNzC2ribf914Ds4x4+T43lXPsjmIX9zgLeb\\\\\\\/W0cqyU7y3HExUdXLJdmID0Tb7LMRhetNZwn1FpOPjoT\\\\\\\/89iQX2smuqkhALzMAZMXVL8VsjxjfKISuqRcmWaEyAvo3DETfcEtv1xeKiaWQm3yCoDUG6NY7P8tQTkPdryyeJnvlVnqvbtfURy6oYVPYWvhmjombZesuxj2SDdqnonTQOYMx7yYl3MdPEUXMi5QpXUvxychYY8Dzk6u9h4pyz5lQ+AySf43qx5TTKNNjez4sBkWVHLAXeaX+PFdyN55Uq6\\\\\\\/TCCPENbhJpLKjd3Eh3agJDxns338wm828+lHJ55dQMdom3b29Hmenjlt1A9P1L3m8yoLW0ZcBaWqPvc4e0+Wacapcw9q6WlOZbh\\\\\\\/zwx0DfUeU3gBjV26Wgf7oSCIraLK6rAyr4bNFKlhj7mf74ugI7kqjFc6Y95aaX5QDBLnhSZma+wHlxknIgzhAU\\\\\\\/YeSgspSFIs3sfAIvo6X17LNv8635fZ\\\\\\\/n\\\u0022,\\\u0022iv\\\u0022:\\\u0022fBGQ0N4cWEBzuhecmWUZ3Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fd5e6edec7e21aa3ffafb2a9bef890030f43b083479236136f7728eb7f8713f1\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022pJscZNupBztMfS2RV+0pRBF1wzogrh7uWaPc7uy1ACpjXcVTV6TpqeRBoa0jEleP7LNgtdek32DgtWv+sN1iibIR2mpdWjb8viOsce\\\\\\\/VIsXdHP9PH57F2DF9Sp87DCoRwkUT4zZS1hHmlZB9k9E0xuqEUVWHKDMo0X5NRKhzGFC0hMofmcAJSJTYmhF3tBooTJ5ahnZmd+rmYTdgcNaysv6xwdyHlQZLb2TqYBm6GTRIvgXFaLC4\\\\\\\/LtyxM5UGSvyu26BlLDfChSRWiZToCs6TaHhidjRDKMa3k0okwLYQu1h3FyB3w7S9Hp99nVxhyDgcupA60ySzm+0NQTZMvE4UXuvvF1mdPMls5zYk935\\\\\\\/BWydkl5Ts2tAtjPVACPhjZETG4IRoOc20j1xqWs8Pf330CwFLcBDZFm4b9F3jQAuSRcgOzn9\\\\\\\/jHTgkycCtHRxq0bon6HH9b9mOJye0ip4zmaRYtRkXuGfYt0F1q5Evn6EEedlAPM5+KerPrf6OpN1\\\\\\\/+TZkZ9JRm1zu9y6jR7ynpgRQki2F6zukDLVop1SCBXcltIeNczWDYnI4nRjUgB5aN8nYEzEp+LOR2iMdJLLuBRRvgZPmK8YMENwM0PWjMIfUkLQYQQ8hW\\\\\\\/wevyXt7FUhhhNsy\\\\\\\/L1Gj\\\\\\\/ctbaYAJHTJh8bbLV5K1wlV9xp9YMXp2oh4wi7h3Jadu8ENy5uKaSjbileHL4ChGA3+ULjEhLMmGpfrgxRrMcQPY1zhHweQGQwyPwoPPx0+jq8XKUuG1\\\\\\\/r3uZ81dcof+qC+IEG2KcUiXhdf8bXmweSxdGe57CBOziQqT\\\\\\\/pZ5XBKfzZHOLVI1XeXV4RMmMXLQwklVIQK66k+8z3YijdFM69OKcpCHw9PKaU\\\\\\\/wGcvXHg13Ha4Z4OQO1q6fEW1P83MSVkocex1VVk8MqueoWuzLk41rrDkCRFhfpf+ZpXmQ83vGeD9EVFXWQ+jiajHzhG3igdndtjEs9J13UxZgyMEm028lut8LjFvS92mLsc+hEI\\\\\\\/sktF3VikzLp7I4gMDjo3p4Iz+XfZDHQYmYH8YBhUlRoUUixgQ0qjyDr2l3USasM2VFhyxoHunXTGJuaecmra0evY+1Bhj7ReZkXgoR31NYbEfr4UVLlD6dDZI8cYarhgi9sG4KliUoaHG0nWDnRAbQVg5aXzfHy3f1VcU6fIrneIovXRtJnaHLVtuwvBlQiqIooJttrvabxD9v0EbDIl7nvVmHilu0UQ6Wg2U7+AkQs6VdeP2B+drDHZDcPx7b49WjoS4BP9kFrupVSUiSk59NIxzz5iW0j2epxbzQKol4kv8hd7fYFFDqmoPWSxOjyhg3o5pickz6D1MxdYZ4cm0lxUnDwcQMry9sv0Fc5hpv8RukvfROlyj\\\\\\\/gJ8rUGVMv+p940SR2n0SIonjnXBKggkCF1x6NRT9ov3NgB\\\\\\\/9xllf4efvA6MtSgZg\\\\\\\/60772Qe3JMhdLoYKqqBGCJaw6SyI6s9ha2CDYhV29k9cAhxGuw1sCOXJhxzG+X89RIUpx2xvuD5xDk1y85kh+Q9DSeQABgPZEUI+IFTyfNNJEmKT83m7ggX3CKO3ErkK4kJIs3dIcjXuImRqI76Kl\\\\\\\/e1LTZ87KtM1yGoxnjLfwhDAWvthIgJFIY+kgowjeZzXICkm8hiCw7J+cZxF3qHZDvnEj32V4f4aDKCNPXl10gxb2XZfPO8eY0GcBlY4EZ5brhzpS38KDr\\\\\\\/NBDg09ORgzn1JPC7L5WSTbB+PA0g5fFMlQlg3jb7\\\\\\\/aTogR+efc6BFXRm8LdB5jNogHnoW4BByuYzBu4qBiVEcK4P9A6kVeYKfwmYACfvNETmS1jWg3NniPVlNRY2vC1fiOMxM1r0+Pu24mrzr6XJilo5zLToiR3DG8ju0UbqkWfGeEmn40+SnPo\\\\\\\/ks1G7LdekMRscW7LgAdEn4sj69Gebag4Yoz8YPeCS53dhogzF+8kqzhOB1ExuceBOkH8p4zTD4ZBMi\\\\\\\/mmE6aWPHpT7PXARegyUtVxhF3VoumlExNMDXqqRtKftoc=\\\u0022,\\\u0022iv\\\u0022:\\\u0022ollDhChZUgBm92l2aZlFGg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c6fee6af9b60d7c68919efed467b31058391382d8e8714ffa3567101577f6c45\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459927 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genetics-of-immunity">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/7/1217" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-JdUyMTltVku7XDcrORT1ckzS6RCM_yb2drqSiQ1AqfE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/7/1217" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-UlfdlTjmtP6eTt1LFlw9KMZe1JwNuYVs8mjwfGJEwZM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/7/1205" title="Mapping Small Effect Mutations in Saccharomyces cerevisiae: Impacts of Experimental Design and Mutational Properties" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/7/1227" title="Infinium Monkeys: Infinium 450K Array for the Cynomolgus macaque (Macaca fascicularis)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459927" id="node459927" data-pisa="ggg;4/7/1217" data-pisa-master="ggg;g3.114.011726" data-apath="/ggg/4/7/1217.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Nonsynonymous Substitution Rate Heterogeneity in the Peptide-Binding Region Among Different <em>HLA-DRB1</em> Lineages in Humans</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yoshiki Yasukochi</span> and <span class="highwire-citation-author" data-delta="1">Yoko Satta</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1217-1226;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011726 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459927" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yoshiki Yasukochi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular and Genetic Epidemiology, Faculty of Medicine, University of Tsukuba, Tsukuba, Ibaraki 305-8575, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yoshiki%2BYasukochi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yasukochi%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYoshiki%2BYasukochi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hyasukou@proof.ocn.ne.jp" class="" data-icon-position="" data-hide-link-title="0">hyasukou@proof.ocn.ne.jp</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yoko Satta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Evolutionary Studies of Biosystems, the Graduate University for Advanced Studies (SOKENDAI), Hayama, Kanagawa 240-0193, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yoko%2BSatta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Satta%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYoko%2BSatta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/7/1217" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459927" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459927/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/7/1217.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459927" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459927/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/7/1217.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459927" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459927/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/7/1217.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459927" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459927/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/7/1217.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459927" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459927/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">An extraordinary diversity of amino acid sequences in the peptide-binding region (PBR) of human leukocyte antigen [HLA; human major histocompatibility complex (MHC)] molecules has been maintained by balancing selection. The process of accumulation of amino acid diversity in the PBR for six <em>HLA</em> genes (<em>HLA-A</em>, <em>B</em>, <em>C</em>, <em>DRB1</em>, <em>DQB1</em>, and <em>DPB1</em>) shows that the number of amino acid substitutions in the PBR among alleles does not linearly correlate with the divergence time of alleles at the six <em>HLA</em> loci. At these loci, some pairs of alleles show significantly less nonsynonymous substitutions at the PBR than expected from the divergence time. The same phenomenon was observed not only in the <em>HLA</em> but also in the rat <em>MHC</em>. To identify the cause for this, <em>DRB1</em> sequences, a representative case of a typical nonlinear pattern of substitutions, were examined. When the amino acid substitutions in the PBR were placed with maximum parsimony on a maximum likelihood tree based on the non-PBR substitutions, heterogeneous rates of nonsynonymous substitutions in the PBR were observed on several branches. A computer simulation supported the hypothesis that allelic pairs with low PBR substitution rates were responsible for the stagnation of accumulation of PBR nonsynonymous substitutions. From these observations, we conclude that the nonsynonymous substitution rate at the PBR sites is not constant among the allelic lineages. The deceleration of the rate may be caused by the coexistence of certain pathogens for a substantially long time during HLA evolution.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/allelic-lineage" class="hw-term hw-article-keyword hw-article-keyword-allelic-lineage" rel="nofollow">allelic lineage</a></li><li class="kwd"><a href="/keyword/balancing-selection" class="hw-term hw-article-keyword hw-article-keyword-balancing-selection" rel="nofollow">balancing selection</a></li><li class="kwd"><a href="/keyword/hla" class="hw-term hw-article-keyword hw-article-keyword-hla" rel="nofollow">HLA</a></li><li class="kwd"><a href="/keyword/pathogen" class="hw-term hw-article-keyword hw-article-keyword-pathogen" rel="nofollow">pathogen</a></li><li class="kwd"><a href="/keyword/peptide-binding-region" class="hw-term hw-article-keyword hw-article-keyword-peptide-binding-region" rel="nofollow">peptide-binding region</a></li><li class="kwd"><a href="/keyword/genetics-immunity" class="hw-term hw-article-keyword hw-article-keyword-genetics-of-immunity" rel="nofollow">genetics of immunity</a></li><li class="kwd"><a href="/keyword/innate-immunity" class="hw-term hw-article-keyword hw-article-keyword-innate-immunity" rel="nofollow">innate immunity</a></li><li class="kwd"><a href="/keyword/complex-genetics" class="hw-term hw-article-keyword hw-article-keyword-complex-genetics" rel="nofollow">complex genetics</a></li><li class="kwd"><a href="/keyword/tolerance" class="hw-term hw-article-keyword hw-article-keyword-tolerance" rel="nofollow">tolerance</a></li><li class="kwd"><a href="/keyword/complex-immunity" class="hw-term hw-article-keyword hw-article-keyword-complex-immunity" rel="nofollow">complex immunity</a></li><li class="kwd"><a href="/keyword/infection" class="hw-term hw-article-keyword hw-article-keyword-infection" rel="nofollow">infection</a></li><li class="kwd"><a href="/keyword/resistance" class="hw-term hw-article-keyword hw-article-keyword-resistance" rel="nofollow">resistance</a></li></ul><p id="p-4">The human leukocyte antigen (HLA) system, also known as the major histocompatibility complex (MHC) in nonhumans, codes for molecules that bind to both self and nonself peptides and presents them to T lymphocytes. In humans, six classical HLA molecules, three of class I (HLA-A, B, and C) and three of class II (HLA-DR, DQ, and DP), have been identified as important for the initiation of T cell–mediated immune response.</p><p id="p-5">The peptide-binding region (PBR) for each <em>HLA</em> is the most polymorphic coding region in the human genome, and this variation is thought to be primarily maintained by balancing selection (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hughes and Nei 1988</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">1989</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hughes and Yeager 1998</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Klein <em>et al.</em> 2007</a>). There are three major (nonmutually exclusive) mechanisms of balancing selection: heterozygote advantage (overdominant selection) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Doherty and Zinkernagel 1975</a>); rare allele advantage (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Slade and McCallum 1992</a>); and fluctuating selection (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hill 1991</a>) [see <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Spurgin and Richardson (2010)</a> for an in-depth review of these hypotheses].</p><p id="p-6">Of the different mechanisms proposed for balancing selection, there are several studies that have identified an important role of the heterozygote advantage in maintaining genetic variation of the HLA. Heterozygote advantage acts when individuals who are heterozygous at a given locus have a higher fitness than individuals who are homozygous because they can bind a larger suite of antigens and confer resistance to a broader range of pathogens (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Doherty and Zinkernagel 1975</a>). The first study to provide molecular evidence for overdominant selection was conducted by <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Hughes and Nei (1989)</a>. They found that the rate of nonsynonymous substitutions in the PBR exceeded that of synonymous substitutions in the entire gene. Studies examining human resistance to specific pathogens have also indicated a role for heterozygote advantage. For example, heterozygotes for <em>HLA</em> class II loci were more resistant to the hepatitis B virus than homozygotes (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Thursz <em>et al.</em> 1997</a>). Similarly, later onset and slower progression of acquired immunodeficiency syndrome (AIDS) have been observed in heterozygotes for <em>HLA</em> class I loci, rather than in homozygotes (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Carrington <em>et al.</em> 1999</a>). In addition, a review of MHC studies in free-ranging animal populations highlighted the functional importance of MHC variability in parasite resistance (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Sommer 2005</a>). Similar studies have also shown associations between MHC polymorphism and parasite resistance, although some studies did not support the heterozygote advantage hypothesis (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Penn <em>et al.</em> 2002</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Musolf <em>et al.</em> 2004</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Oliver <em>et al.</em> 2009</a>).</p><p id="p-7">In addition to balancing selection via amino acid substitutions in the PBR, evolutionary forces to the selection seem to be in operation, purifying selection due to functional (or structural) constraints. Consequently, nonsynonymous substitutions in the PBR show several characteristics. One is a frequent flip-flop substitution of Val and Gly at site 81 in the <em>HLA-DRB1</em> gene. This specific amino acid switching is believed to be due to functional or structural constraints for the peptide-binding groove (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Ong <em>et al.</em> 1991</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Demotz <em>et al.</em> 1993</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Verreck <em>et al.</em> 1996</a>). In addition, the extent of MHC diversity at the PBR appears to be limited, as too much diversity limits T lymphocyte repertoires as a result of extensive negative selection during the T cell development process in the thymus (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Vidović and Matzinger 1988</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Nowak <em>et al.</em> 1992</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Woelfing <em>et al.</em> 2009</a>).</p><p id="p-8">Despite the unusually large amount of variation found at the MHC PBR, the theory (allelic genealogy) of allelic lineage under symmetric balancing selection (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Takahata 1990</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Takahata <em>et al.</em> 1992</a>) predicts that nucleotide substitutions at the PBR sites are accumulated with the divergence time of the alleles. This is because symmetric balancing selection assumes that large numbers of alleles are equivalent to each other in their fitness, and thus the probability for producing descendants is also equal. Therefore, the genealogical characteristic is quite similar to the neutral case, except for its time scale (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Takahata 1990</a>). This time scale is determined by a “scaling factor” that is a function of the mutation rate, selection coefficient, and effective population size. For the molecular clock to operate for nonsynonymous substitutions in the PBR, the selection intensity for all the lineages must be constant. However, despite many studies that examined selective mechanisms of <em>HLA</em> diversity, no previous studies have elucidated the temporal aspects of nucleotide substitution patterns at the PBR nonsynonymous sites. To determine the temporal evolutionary trajectory of <em>HLA</em> diversity, amino acid substitution patterns at the PBR nonsynonymous sites were examined at six <em>HLA</em> loci (<em>HLA-A</em>, <em>B</em>, <em>C</em>, <em>DRB1</em>, <em>DQB1</em>, and <em>DPB1</em>), with special focus on <em>HLA-DRB1</em>. Here, we provide evidence for episodic amino acid substitutions in the PBR of humans, in primate evolution, suggesting significant fluctuation of selection intensity of the allelic lineage in <em>HLA-DRB1</em>. The extent of amino acid diversity in the PBR is directly linked to how many kinds of pathogens can be recognized by <em>HLA</em> molecules. Hence, tracing the transitional change of <em>HLA</em> diversity may allow us to describe the relationships between humans and pathogens in particular time periods.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Collection of nucleotide sequence data for six <em>HLA</em> loci</h3><p id="p-9">Nucleotide sequences for the six <em>HLA</em> loci (<em>HLA-A</em>, <em>B</em>, <em>C</em>, <em>DRB1</em>, <em>DQB1</em>, and <em>DPB1</em>) were obtained from the HLA/IMGT database (<a href="http://www.ebi.ac.uk/imgt/hla/">http://www.ebi.ac.uk/imgt/hla/</a>) (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Robinson <em>et al.</em> 2011</a>). The sequences of <em>HLA-A</em>, <em>B</em>, <em>C</em>, and <em>DRB1</em> containing the entire coding region (<em>HLA-A</em>/<em>-B</em>/<em>-C</em>, approximately 1100 bp; <em>HLA-DRB1</em>, approximately 800 bp) were used in this analysis. Because the number of alleles covering the whole sequence of the coding region in the <em>HLA-DQB1</em> and <em>HLA-DPB1</em> was limited, slightly shorter sequences were included in the analysis (<em>HLA-DQB1</em>, approximately 690 bp; <em>HLA-DPB1</em>, approximately 540 bp). Possible recombinant alleles were excluded by using the method described by <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Satta (1992)</a>. This method assumes that the relationship between the number of substitutions in a particular region and that in the entire region is binomially distributed. Additionally, the ratio of the number of these substitutions is proportional to the size in the corresponding regions (see <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kusaba <em>et al.</em> 1997</a> for more details). Consequently, the dataset used in this analysis included 50 <em>HLA-A</em> alleles, 143 <em>HLA-B</em>, 129 <em>HLA-C</em>, 56 <em>HLA-DRB1</em>, 55 <em>HLA-DQB1</em>, and 38 <em>HLA-DPB1</em>. <em>HLA-DRB1</em> alleles observed in a Japanese population were investigated using the Allele Frequency Net Database (<a href="http://www.allelefrequencies.net/">http://www.allelefrequencies.net/</a>) (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gonzalez-Galarza <em>et al.</em> 2011</a>).</p></div><div id="sec-3" class="subsection"><h3>Phylogenetic analyses</h3><p id="p-10">Multiple sequence alignments of <em>MHC-DRB1</em> nucleotide sequences and amino acid translations were performed using MEGA v5.10 (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Tamura <em>et al.</em> 2011</a>). A phylogenetic tree of nucleotide sequences in <em>DRB1</em> non-PBR was constructed based on the maximum-likelihood (ML) method using the Hasegawa-Kishino-Yano (HKY) substitution model (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Hasegawa <em>et al.</em> 1985</a>) implemented in MEGA. Nearest-neighbor-interchange (NNI) was applied as the ML heuristic method. Bootstrap values were obtained from 1000 replications. Amino acid distances at the PBR between alleles were calculated using the Jones-Taylor-Thornton (JTT) model (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Jones <em>et al.</em> 1992</a>) as per the ML tree topology for non-PBR using software in the PHYLIP 3.69 package (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Felsenstein 2009</a>). In the phylogenetic analysis of 56 <em>HLA-DRB1</em> alleles (described above), 6 <em>HLA-DRB3</em>, 4 <em>HLA-DRB4</em>, 2 <em>HLA-DRB5</em>, 11 chimpanzee (<em>Pan troglodytes</em>) <em>Patr-DRB1</em>, 22 rhesus monkey (<em>Macaca mulatta</em>) <em>Mamu-DRB1</em>, and 3 crab-eating macaque (<em>Macaca fascicularis</em>) <em>Mafa-DRB1</em> sequences were also included after possible recombinant alleles were removed using the method by <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Satta (1992)</a> (see above). Two <em>HLA-DQB1</em> sequences (<em>HLA-DQB1*02:01:01</em> and <em>HLA-DQB1*06:02:01</em>) were used for determining the root of the tree. The dataset of <em>HLA-DRB3</em>, <em>HLA-DRB4</em>, <em>HLA-DRB5</em>, and <em>HLA-DQB1</em> allele sequences was obtained from the HLA/IMGT database, whereas the nonhuman primate <em>DRB1</em> sequences were obtained from the IPD-MHC NHP database (<a href="http://www.ebi.ac.uk/ipd/mhc/nhp/">http://www.ebi.ac.uk/ipd/mhc/nhp/</a>) (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Robinson <em>et al.</em> 2005</a>).</p></div><div id="sec-4" class="subsection"><h3>Examination for temporal aspect of amino acid substitution pattern in the PBR</h3><p id="p-11">Sites 57, 67, and 90 were not identified as the PBR sites of HLA-DRB1 molecule by <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Brown <em>et al.</em> (1993)</a>. However, Brown and collaborators have subsequently shown that these sites are involved in peptide-binding pockets that are crucial for peptide capture (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Stern <em>et al.</em> 1994</a>). In addition, recent assays for the peptide-binding prediction of HLA molecules have re-identified these three sites as peptide-binding residues (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Reche and Reinherz 2003</a>). Therefore, they were included as PBR in this analysis, and 27 amino acids in total were regarded as PBR. To examine the nucleotide substitution rate, the mean number of nonsynonymous substitutions in PBR [<em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub>] among allelic pairs that had the particular number of substitutions (<em>m</em> = <em>K</em><sub>S</sub> + <em>K</em><sub>N</sub>, where <em>K</em><sub>S</sub> is the number of synonymous substitutions in the entire region and <em>K</em><sub>N</sub> is the number of nonsynonymous substitutions in non-PBR) at putative neutral sites (synonymous + non-PBR nonsynonymous sites) was examined. The molecular clock for <em>K</em><sub>N</sub> was tested using the ML method in MEGA.</p><p id="p-12">The expected <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> value was calculated on the basis of equation 12 described in <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Takahata <em>et al.</em> (1992)</a>, and its SE was estimated by maximum variance (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Takahata and Tajima 1991</a>). The variables necessary for the equation were selected such that the first several observations showed good agreement with expectations.</p></div><div id="sec-5" class="subsection"><h3>Estimation of the divergence time of <em>HLA-DRB1</em> alleles and amino acid substitution rates in the PBR</h3><p id="p-13">We calculated the divergence time of allelic pairs on the basis of neutral divergence, <em>d =</em> (<em>K</em><sub>S</sub> + <em>K</em><sub>N</sub>/<em>f</em>)/(<em>L</em><sub>S</sub> + <em>L</em><sub>N</sub>), where <em>L</em><sub>S</sub> and <em>L</em><sub>N</sub> are the number of synonymous and non-PBR nonsynonymous sites, respectively. The <em>K</em><sub>N</sub><em>/f</em> value is the number of converted neutral substitutions obtained by dividing by the functional constraint <em>f</em>, which is the mean ratio of nonsynonymous to synonymous substitution rates at the non-PBR of all allelic pairs. The divergence time (<em>T</em>) of each allele pair is given by the formula <em>T</em> = <em>d</em>/2<em>μ</em>, where <em>μ</em> is the neutral substitution rate of 10<sup>−9</sup> per site per year at the primate MHC loci (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Satta <em>et al.</em> 1994</a>).</p><p id="p-14">To evaluate the amino acid substitution rate in the PBR for each allelic lineage, the number of PBR amino acid substitutions on each branch of the non-PBR ML tree was compared with the ML estimate of the corresponding branch length. Here, the linear correlation between these two values was normally expected if the molecular clock worked in both PBR and non-PBR. Significant outliers were identified based on 95% confidence interval of the regression line. These outliers were further examined for whether the ML estimate of a branch length was significantly different from zero, based on a confidence interval of the estimates. In addition, internodal branches supported by more than 80% bootstrap values were selected. Finally, we identified allelic lineages or branches with a fast or slow PBR substitution rate compared with the average.</p></div><div id="sec-6" class="subsection"><h3>Computer simulation and prediction of peptide-binding specificity for <em>HLA-DRB1</em> alleles</h3><p id="p-15">The assumptions underlying the computer simulation were as follows. We supposed that there is a PBR phylogenetic relationship that shows the same topology of the non-PBR tree based on nucleotide sequences of 56 <em>HLA-DRB1</em> alleles. According to the expected time length (<em>T</em><sub>i</sub>) for a branch <em>i</em>, the number of PBR substitutions (<em>K</em><sub>i</sub>) on the branch follows a Poisson distribution with mean <em>λ</em> = <em>μT</em><sub>i</sub>, where <em>μ</em> is the substitution rate (=1/unit time). The <em>T</em><sub>i</sub> value is the number of neutral nucleotide substitutions corresponding to each branch length on the non-PBR tree. The <em>K</em><sub>i</sub> value on a slow-rate branch was determined by dividing the original <em>λ</em> value by 10, whereas that on a fast branch was calculated by multiplying the original <em>λ</em> value by 10. In this manner, <em>K</em><sub>i</sub> values for every branch were determined and the number of substitutions in a pair of alleles was obtained by summing corresponding branches.</p><p id="p-16">The dataset of epitopes or source organisms bound to HLA-DRB1 molecules was obtained from the Immune Epitope Database (IEDB) (<a href="http://www.immuneepitope.org">http://www.immuneepitope.org</a>) (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Vita <em>et al.</em> 2010</a>).</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Variation in amino acid diversity at the PBR</h3><p id="p-17">For each of the six <em>HLA</em> loci, the mean number of nonsynonymous substitutions in the PBR [<em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub>] among particular allele pairs with putative neutral substitutions of <em>m</em> (<em>m</em> = <em>K</em><sub>S</sub> + <em>K</em><sub>N</sub>) was calculated (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Since the <em>K</em><sub>N</sub> is the number of nonsynonymous substitutions at the non-PBR, we performed molecular clock tests for the substitutions. As a result, the homogeneous evolutionary rate of <em>K</em><sub>N</sub> in each <em>HLA</em> locus was statistically supported (<em>P</em> &lt; 0.05), with the exception of <em>HLA-B</em> and <em>HLA-C</em> loci. According to the theory previously described (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Takahata <em>et al.</em> 1992</a>), the value of <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> is expected to increase in proportion to <em>m</em>, the coalescent time (or divergence time) between alleles, and the value is then saturated with <em>m</em> of distantly related allele pairs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011726/-/DC1/011726SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011726/-/DC1/FigureS1.pdf">Figure S1</a>). However, the observed <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> value did not show the linear relationship with <em>m</em> for most loci (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). For the intermediate values of <em>m</em>, it appeared to decrease and cease the accumulation of the substitutions. After this plateau, the <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> value increased again. Although most of the <em>HLA</em> genes showed a generally similar pattern of <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> distribution, their detailed patterns were somewhat different for each locus. For instance, <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> values in <em>HLA-DQB1</em> decreased until <em>m</em> = 2, whereas those in <em>HLA-DPB1</em> decreased from <em>m</em> = 6 to <em>m</em> = 9. Interestingly, our preliminary research indicated that the rat (<em>Rattus norvegicus</em>) <em>DRB1</em> (<em>RT1-Db1</em>) also showed a similar pattern of <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> saturation (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011726/-/DC1/FigureS2.pdf">Figure S2</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/7/1217/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The level of amino acid diversity at the peptide-binding region among HLA allele pairs that share the same coalescence time. The ordinate axis indicates the mean number of nonsynonymous substitutions at the peptide-binding region (PBR) among allele pairs [KB(m)]. The abscissa axis indicates the number of substitutions at putative neutral sites (m = KS + KN). The gray dot represents the expected KB(m) value. The diamond represents the observed KB(m) value. The open diamond indicates the statistically significant difference between the observed and expected KB(m) values (P &lt; 0.05)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-53331287" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The level of amino acid diversity at the peptide-binding region among &lt;em&gt;HLA&lt;/em&gt; allele pairs that share the same coalescence time. The ordinate axis indicates the mean number of nonsynonymous substitutions at the peptide-binding region (PBR) among allele pairs [&lt;em&gt;K&lt;/em&gt;&lt;sub&gt;B(&lt;/sub&gt;&lt;em&gt;&lt;sub&gt;m&lt;/sub&gt;&lt;/em&gt;&lt;sub&gt;)&lt;/sub&gt;]. The abscissa axis indicates the number of substitutions at putative neutral sites (&lt;em&gt;m&lt;/em&gt; = &lt;em&gt;K&lt;sub&gt;S&lt;/sub&gt;&lt;/em&gt; + &lt;em&gt;K&lt;sub&gt;N&lt;/sub&gt;&lt;/em&gt;). The gray dot represents the expected &lt;em&gt;K&lt;/em&gt;&lt;sub&gt;B(&lt;/sub&gt;&lt;em&gt;&lt;sub&gt;m&lt;/sub&gt;&lt;/em&gt;&lt;sub&gt;)&lt;/sub&gt; value. The diamond represents the observed &lt;em&gt;K&lt;/em&gt;&lt;sub&gt;B(&lt;/sub&gt;&lt;em&gt;&lt;sub&gt;m&lt;/sub&gt;&lt;/em&gt;&lt;sub&gt;)&lt;/sub&gt; value. The open diamond indicates the statistically significant difference between the observed and expected &lt;em&gt;K&lt;/em&gt;&lt;sub&gt;B(&lt;/sub&gt;&lt;em&gt;&lt;sub&gt;m&lt;/sub&gt;&lt;/em&gt;&lt;sub&gt;)&lt;/sub&gt; values (&lt;em&gt;P&lt;/em&gt; &lt; 0.05).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/7/1217/F1.medium.gif" width="429" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/7/1217/F1.medium.gif" width="429" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/7/1217/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/7/1217/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464548" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-18" class="first-child">The level of amino acid diversity at the peptide-binding region among <em>HLA</em> allele pairs that share the same coalescence time. The ordinate axis indicates the mean number of nonsynonymous substitutions at the peptide-binding region (PBR) among allele pairs [<em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub>]. The abscissa axis indicates the number of substitutions at putative neutral sites (<em>m</em> = <em>K<sub>S</sub></em> + <em>K<sub>N</sub></em>). The gray dot represents the expected <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> value. The diamond represents the observed <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> value. The open diamond indicates the statistically significant difference between the observed and expected <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> values (<em>P</em> &lt; 0.05).</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">The comparison between the observed and expected <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> values showed that the observed <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> was quite lower than expected in all <em>HLA</em> loci, and the difference was statistically significant (Z test, <em>P</em> &lt; 0.05) (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). This result indicates that <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> fluctuation was not likely caused by stochastic error. Instead, nonsynonymous substitutions at the PBR may have been unexpectedly suppressed due to an unknown biological phenomenon.</p><p id="p-20">An acceleration of <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> was expected in order to cope with the rapid evolutionary change of pathogens; however, we observed a decline of <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub>, and the reason was unexplained. It was intriguing to understand why amino acid substitutions at the PBR were suppressed despite balancing selection. Therefore, we investigated why <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> did not increase in proportion to the divergence time of alleles (<em>m</em>). We explored <em>HLA-DRB1</em> alleles, which showed a typical graphical pattern and provided relatively rich experimental data for binding peptides. Depending on the extent of the <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> increment, we categorized the graph into four phases, I through IV (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). In phases I and III, <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> increase was proportional to <em>m</em>, the divergence time between alleles, whereas in phases II and IV, <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> values were more or less constant, irrespective of the <em>m</em> values. In phase IV, the accumulation of substitutions might have reached a ceiling because of an enhanced PBR nonsynonymous nucleotide substitution rate due to balancing selection. In phase II, the <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> values [<em>m</em> = 20 to 28, mean <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> = 12.7] appeared to be extensively decreased as compared to that [<em>m</em> = 19, <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> = 17.0] in phase I, suggesting that PBR amino acid substitutions of alleles in phase II were limited. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011726/-/DC1/TableS1.pdf">Table S1</a> shows the allelic pairs that constituted phase II. When we approximately estimated the divergence time of allele pairs in phase II (see <em>Materials and Methods</em>), the divergence time ranged from approximately 18 MYA to 24 MYA, suggesting that those allele pairs have diverged at least before speciation events in the subfamily Homininae (or Hominidae). The <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> decline was not observed in smaller <em>m</em>, suggesting that the decline had not occurred within the human lineage only.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/7/1217/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The mean number of nonsynonymous substitutions at the PBR among HLA-DRB1 allele pairs that share the same coalescence time. The ordinate axis represents the mean number of nonsynonymous substitutions at the PBR among allele pairs [KB(m)]. The abscissa axis represents the number of substitutions at putative neutral sites (m = KS + KN). The Roman numerals indicate the four phases that are classified as per variation in patterns of KB(m) values (see text). Error bars indicate the SD from the mean." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-53331287" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The mean number of nonsynonymous substitutions at the PBR among &lt;em&gt;HLA-DRB1&lt;/em&gt; allele pairs that share the same coalescence time. The ordinate axis represents the mean number of nonsynonymous substitutions at the PBR among allele pairs [&lt;em&gt;K&lt;/em&gt;&lt;sub&gt;B(&lt;/sub&gt;&lt;em&gt;&lt;sub&gt;m&lt;/sub&gt;&lt;/em&gt;&lt;sub&gt;)&lt;/sub&gt;]. The abscissa axis represents the number of substitutions at putative neutral sites (&lt;em&gt;m&lt;/em&gt; = &lt;em&gt;K&lt;/em&gt;&lt;sub&gt;S&lt;/sub&gt; + &lt;em&gt;K&lt;/em&gt;&lt;sub&gt;N&lt;/sub&gt;). The Roman numerals indicate the four phases that are classified as per variation in patterns of &lt;em&gt;K&lt;/em&gt;&lt;sub&gt;B(&lt;/sub&gt;&lt;em&gt;&lt;sub&gt;m&lt;/sub&gt;&lt;/em&gt;&lt;sub&gt;)&lt;/sub&gt; values (see text). Error bars indicate the SD from the mean.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/7/1217/F2.medium.gif" width="440" height="279"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/7/1217/F2.medium.gif" width="440" height="279"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/7/1217/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/7/1217/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464670" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-21" class="first-child">The mean number of nonsynonymous substitutions at the PBR among <em>HLA-DRB1</em> allele pairs that share the same coalescence time. The ordinate axis represents the mean number of nonsynonymous substitutions at the PBR among allele pairs [<em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub>]. The abscissa axis represents the number of substitutions at putative neutral sites (<em>m</em> = <em>K</em><sub>S</sub> + <em>K</em><sub>N</sub>). The Roman numerals indicate the four phases that are classified as per variation in patterns of <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> values (see text). Error bars indicate the SD from the mean.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Phylogenetic analysis for the <em>DRB1</em> locus</h3><p id="p-22">We constructed an ML tree for <em>DRB</em> alleles, including <em>HLA-DRB1</em>/<em>3</em>/<em>4</em>/<em>5</em>, <em>Patr-DRB1</em>, <em>Mamu-DRB1</em>, and <em>Mafa-DRB1</em>, based on total substitutions in only non-PBR nucleotide sequences. In the tree, some <em>HLA-DRB1</em> alleles formed a monophyletic group, whereas other alleles were polyphyletic with <em>DRB</em> alleles from nonhuman primates (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). This topology was also supported by the neighbor-joining (NJ) tree (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Saitou and Nei 1987</a>) method based on both nucleotide and amino acid sequences in the region, although bootstrap values of their groups were not high (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011726/-/DC1/FigureS3.pdf">Figure S3</a>). Here, we refer to these two groups as group A and group B, respectively (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Among the 14 known <em>HLA</em> allelic lineages, group A consisted of seven lineages, namely <em>DRB1*03</em>, <em>*08</em>, <em>*10</em>, <em>*11</em>, <em>*12</em>, <em>*13</em>, and <em>*14</em>. The remaining seven lineages, <em>DRB1*01</em>, <em>*02</em>, <em>*04</em>, <em>*07</em>, <em>*09</em>, <em>*15</em>, and <em>*16</em>, were in group B. The allelic lineages <em>HLA-DRB1*01</em> and <em>*10</em> of the same <em>DR1</em> haplotype were included in different groups, whereas other alleles of the same haplotype belonged to either group A or group B.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/7/1217/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic relationships between MHC DRB alleles (including those of humans, chimpanzees, and macaques) as determined by the maximum likelihood (ML) method on the basis of nucleotide sequences (690 bp) in the non-PBR. Only bootstrap values more than 80% are shown here. Two HLA-DQB1 sequences are used as the out-group. Evolutionary distances were computed using the Hasegawa-Kishino-Yano (HKY) model. Groups A and B indicate two major phylogenetic groups of HLA-DRB1 alleles. HLA, humans; Patr, chimpanzees; Mamu, rhesus monkeys; Mafa, crab-eating macaques. IMGT/HLA and IPD accession numbers are in parentheses." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-53331287" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogenetic relationships between &lt;em&gt;MHC DRB&lt;/em&gt; alleles (including those of humans, chimpanzees, and macaques) as determined by the maximum likelihood (ML) method on the basis of nucleotide sequences (690 bp) in the non-PBR. Only bootstrap values more than 80% are shown here. Two &lt;em&gt;HLA-DQB1&lt;/em&gt; sequences are used as the out-group. Evolutionary distances were computed using the Hasegawa-Kishino-Yano (HKY) model. Groups A and B indicate two major phylogenetic groups of &lt;em&gt;HLA-DRB1&lt;/em&gt; alleles. &lt;em&gt;HLA&lt;/em&gt;, humans; &lt;em&gt;Patr&lt;/em&gt;, chimpanzees; &lt;em&gt;Mamu&lt;/em&gt;, rhesus monkeys; &lt;em&gt;Mafa&lt;/em&gt;, crab-eating macaques. IMGT/HLA and IPD accession numbers are in parentheses.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/7/1217/F3.medium.gif" width="353" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/7/1217/F3.medium.gif" width="353" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/7/1217/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/7/1217/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464574" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-23" class="first-child">Phylogenetic relationships between <em>MHC DRB</em> alleles (including those of humans, chimpanzees, and macaques) as determined by the maximum likelihood (ML) method on the basis of nucleotide sequences (690 bp) in the non-PBR. Only bootstrap values more than 80% are shown here. Two <em>HLA-DQB1</em> sequences are used as the out-group. Evolutionary distances were computed using the Hasegawa-Kishino-Yano (HKY) model. Groups A and B indicate two major phylogenetic groups of <em>HLA-DRB1</em> alleles. <em>HLA</em>, humans; <em>Patr</em>, chimpanzees; <em>Mamu</em>, rhesus monkeys; <em>Mafa</em>, crab-eating macaques. IMGT/HLA and IPD accession numbers are in parentheses.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">When the non-PBR phylogeny (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>) based on total nucleotide substitutions was compared with the tree based on PBR amino acid substitutions, the phylogenetic position of allelic lineages in the non-PBR tree did not correspond to that in the PBR amino acids tree (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011726/-/DC1/FigureS4.pdf">Figure S4</a>). This means that when members of an allele pair are distantly related to each other in their non-PBR sequences, the amino acids in the PBR are often more closely related (and vice versa). When the allele pairs in phase II were placed on the tip of non-PBR tree, each allele for a particular pair was frequently found (73% of allele pairs in phase II) from either group A or group B.</p><p id="p-25">The constant accumulation of PBR nonsynonymous substitutions appears to be violated in certain allele pairs. The reasons for the observed nonlinearity of <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> includes one of the following possibilities: (1) saturation of transitions in nucleotide substitution precedes that of transversion, and this difference in saturation timing causes the phenomenon; (2) parallel substitutions in the PBR mask the linear relationship between <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> and <em>m</em>; or (3) slow-down of the PBR nonsynonymous substitution rate in particular allelic lineages skews the constant <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> accumulation.</p></div><div id="sec-10" class="subsection"><h3>Biased rate of transitions and transversions and parallel substitutions in the PBR among <em>HLA-DRB1</em> allele pairs</h3><p id="p-26">It is well-known that transitions (Ts) occur at a higher frequency than transversions (Tv) in both animal mitochondrial and nuclear DNA sequences. Because of this bias and the lower number of Ts than Tv states, Ts is likely to reach a plateau earlier than Tv. Thus, there is a possibility that excess of Ts in the <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> explains the observed accumulation pattern. To examine this possibility, we tested the relationship between numbers of Ts and Tv in the PBR (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011726/-/DC1/FigureS5.pdf">Figure S5</a>). In phase I, Ts values of 0.7 to 6.5 were observed with Tv of 0.6 to 11.0; in the phase II, Ts of 4.5 to 6.7 were seen with Tv of 7.7 to 11.0; in phase III, Ts of 5.5 to 7.3 were associated with Tv of 10.6 to 14.5; and in phase IV, Ts of 4.6 to 7.1 were found with Tv of 14.0 to 16.6. Although Ts values seem to be saturated when Tv = 14, the corresponding allele pairs were all in phase III or phase IV. Thus, the timing of Ts saturation did not explain the cessation of <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> accumulation in phase II.</p><p id="p-27">The non-PBR phylogenetic relationship among <em>HLA-DRB1</em> sequences was not consistent with the PBR phylogenetic relationship. This is likely due to parallel substitutions or recombination between alleles, and these events probably mask the linear relationship between <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> and <em>m</em>. Because we excluded the possibility of recombinants when filtering alleles used in this analysis (see <em>Materials and Methods</em>), the effect of recombination on the <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> accumulation pattern is quite small. Next, parallel substitutions in the PBR were investigated. An example of a parallel substitution is the substitution between Val and Gly at the site 81 (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gregersen <em>et al.</em> 1986</a>), which was frequently observed on several branches in the non-PBR tree. Dimorphism (Val and Gly) at the site has been shown to affect the conformation of the peptide-binding pocket (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Ong <em>et al.</em> 1991</a>) and allo-recognition of peptides (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Demotz <em>et al.</em> 1993</a>). This dimorphism appears to be associated with the selection of binding-peptide groups with different motifs (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Verreck <em>et al.</em> 1996</a>). However, this parallel substitution at site 81 does not explain the phase II substitution pattern in <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> (see below).</p><p id="p-28">We performed an extensive search for parallel substitutions at the PBR amino acid sites by manually placing the PBR amino acid substitutions on branches in the non-PBR tree (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011726/-/DC1/FigureS4.pdf">Figure S4</a>). The number of PBR substitutions was estimated by the maximum parsimonious method. Here, the same amino acid substitution on different branches was defined as parallel substitution. The number of such parallel substitutions was counted in each allele pair, not only in phase II but also in other phases. Consequently, it is interesting that a large number of parallel substitutions are observed even in closely related pairs of alleles as frequently as in distantly related allele pairs. Parallel substitutions at not only site 81 but also other PBR sites were widely observed across all phases (I–IV) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011726/-/DC1/TableS2.pdf">Table S2</a>). However, <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> values in phase I and phase III still increased with <em>m</em>, even though parallel substitutions were observed in these phases. Therefore, <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> saturation in phase II was unlikely to be affected by parallel substitutions.</p></div><div id="sec-11" class="subsection"><h3>Heterogeneity in the PBR substitution rate among allelic lineages</h3><p id="p-29">To evaluate the possibility of heterogeneity in the substitution rate, we tested whether the PBR amino acid substitution rate for each allelic lineage was constant by means of the comparison between the number of nonsynonymous substitutions at non-PBR and PBR sites on each branch of the non-PBR tree (see <em>Materials and Methods</em>). This analysis identified five and two branches with significantly slow and fast PBR substitution rates, respectively, as compared with the average (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Allelic lineages of <em>HLA-DRB1*04</em>, <em>*15</em>, and <em>*16</em> had a slow PBR substitution rate, and those of <em>*03</em>, and <em>*07</em> had a fast rate. As expected, alleles with slow PBR substitution rates were observed frequently (44%) in phase II, whereas in phases I, III, and IV the frequencies of such alleles were 1–6% (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/7/1217/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The relationship between the PBR and non-PBR genetic distances on each branch of the phylogenetic tree. The ordinate axis represents amino acid substitutions at the PBR among allele pairs on each branch of the non-PBR ML tree. The substitutions were estimated by the JTT model. The abscissa axis represents nucleotide substitutions at the non-PBR among allele pairs on each branch of the ML tree. The substitutions were estimated by the HKY model. The solid line represents a linear regression. The broken line represents the 95% confidential interval for the regression coefficient." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-53331287" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The relationship between the PBR and non-PBR genetic distances on each branch of the phylogenetic tree. The ordinate axis represents amino acid substitutions at the PBR among allele pairs on each branch of the non-PBR ML tree. The substitutions were estimated by the JTT model. The abscissa axis represents nucleotide substitutions at the non-PBR among allele pairs on each branch of the ML tree. The substitutions were estimated by the HKY model. The solid line represents a linear regression. The broken line represents the 95% confidential interval for the regression coefficient.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/7/1217/F4.medium.gif" width="440" height="355"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/7/1217/F4.medium.gif" width="440" height="355"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/7/1217/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/7/1217/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464684" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-30" class="first-child">The relationship between the PBR and non-PBR genetic distances on each branch of the phylogenetic tree. The ordinate axis represents amino acid substitutions at the PBR among allele pairs on each branch of the non-PBR ML tree. The substitutions were estimated by the JTT model. The abscissa axis represents nucleotide substitutions at the non-PBR among allele pairs on each branch of the ML tree. The substitutions were estimated by the HKY model. The solid line represents a linear regression. The broken line represents the 95% confidential interval for the regression coefficient.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/7/1217/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic relationships between HLA-DRB alleles as determined by the ML method on the basis of nucleotide sequences (690 bp) in the non-PBR. Only bootstrap values more than 80% are shown here. The values are shown by a bold character. Two HLA-DQB1 sequences are used as the out-group. Evolutionary distances were computed using the HKY model. Group A and group B indicate two major phylogenetic groups of HLA-DRB1 alleles. IMGT/HLA accession numbers are in parentheses. The numeric character shown in black on each branch of the tree represents nucleotide substitutions at the non-PBR among allele pairs. The numeric character shown in red on each branch of the tree represents amino acid substitutions estimated by the JTT model at the PBR among allele pairs. Branches and alleles with slow PBR amino acid substitutions are shown in green, whereas those with fast PBR substitutions are shown in pink." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-53331287" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogenetic relationships between &lt;em&gt;HLA-DRB&lt;/em&gt; alleles as determined by the ML method on the basis of nucleotide sequences (690 bp) in the non-PBR. Only bootstrap values more than 80% are shown here. The values are shown by a bold character. Two &lt;em&gt;HLA-DQB1&lt;/em&gt; sequences are used as the out-group. Evolutionary distances were computed using the HKY model. Group A and group B indicate two major phylogenetic groups of &lt;em&gt;HLA-DRB1&lt;/em&gt; alleles. IMGT/HLA accession numbers are in parentheses. The numeric character shown in black on each branch of the tree represents nucleotide substitutions at the non-PBR among allele pairs. The numeric character shown in red on each branch of the tree represents amino acid substitutions estimated by the JTT model at the PBR among allele pairs. Branches and alleles with slow PBR amino acid substitutions are shown in green, whereas those with fast PBR substitutions are shown in pink.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/7/1217/F5.medium.gif" width="414" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/7/1217/F5.medium.gif" width="414" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/7/1217/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/7/1217/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464615" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-31" class="first-child">Phylogenetic relationships between <em>HLA-DRB</em> alleles as determined by the ML method on the basis of nucleotide sequences (690 bp) in the non-PBR. Only bootstrap values more than 80% are shown here. The values are shown by a bold character. Two <em>HLA-DQB1</em> sequences are used as the out-group. Evolutionary distances were computed using the HKY model. Group A and group B indicate two major phylogenetic groups of <em>HLA-DRB1</em> alleles. IMGT/HLA accession numbers are in parentheses. The numeric character shown in black on each branch of the tree represents nucleotide substitutions at the non-PBR among allele pairs. The numeric character shown in red on each branch of the tree represents amino acid substitutions estimated by the JTT model at the PBR among allele pairs. Branches and alleles with slow PBR amino acid substitutions are shown in green, whereas those with fast PBR substitutions are shown in pink.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464681/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464681/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">The frequency of alleles with fast or slow PBR substitution rates in phase I to IV</span><div class="sb-div caption-clear"></div></div></div><p id="p-34">A computer simulation was performed to confirm whether the presence of such lineages with slow and fast substitution rates led to the nonlinearity of <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> accumulation using 56 fictitious alleles that had the same phylogenetic relationship as the non-PBR tree (for details of the simulation methods, see <em>Materials and Methods</em>). After the simulation was repeated 200 times, we examined the relationship between the divergence time of alleles and the number of substitutions in the 56 allele pairs (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). As expected, <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> values estimated in the computer simulation exhibited a similar pattern as the observation shown in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>. We also performed a similar computer simulation (100 replications) without a reduction or enhancement of the PBR substitution rate. The simulation showed that <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> values proportionally increased with the divergence time among the alleles (<em>m</em>). These results suggest that PBR nonsynonymous substitutions of <em>HLA-DRB1</em> alleles were decelerated or enhanced because of the heterogeneity of the PBR substitution rates among the allelic lineages.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/7/1217/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The K values of allele pairs that share the same coalescent time T. The ordinate axis represents the K value. The abscissa axis represents unit time T corresponding to the number of substitutions at neutral sites (m). The upper graph represents the accumulation of the mean number of nonsynonymous substitutions in the PBR among allele pairs (K) when PBR substitution rates of particular branches in a phylogenetic tree are slowed down or enhanced. The bottom graph represents the accumulation of K values without reduction or enhancement of PBR substitution rates. Error bars indicate the SD from the mean. The simulations for upper and bottom graphs were repeated 200 times and 100 times, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-53331287" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The &lt;em&gt;K&lt;/em&gt; values of allele pairs that share the same coalescent time &lt;em&gt;T&lt;/em&gt;. The ordinate axis represents the &lt;em&gt;K&lt;/em&gt; value. The abscissa axis represents unit time &lt;em&gt;T&lt;/em&gt; corresponding to the number of substitutions at neutral sites (&lt;em&gt;m&lt;/em&gt;). The upper graph represents the accumulation of the mean number of nonsynonymous substitutions in the PBR among allele pairs (&lt;em&gt;K&lt;/em&gt;) when PBR substitution rates of particular branches in a phylogenetic tree are slowed down or enhanced. The bottom graph represents the accumulation of &lt;em&gt;K&lt;/em&gt; values without reduction or enhancement of PBR substitution rates. Error bars indicate the SD from the mean. The simulations for upper and bottom graphs were repeated 200 times and 100 times, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/7/1217/F6.medium.gif" width="440" height="393"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/7/1217/F6.medium.gif" width="440" height="393"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/7/1217/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/7/1217/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464508" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-35" class="first-child">The <em>K</em> values of allele pairs that share the same coalescent time <em>T</em>. The ordinate axis represents the <em>K</em> value. The abscissa axis represents unit time <em>T</em> corresponding to the number of substitutions at neutral sites (<em>m</em>). The upper graph represents the accumulation of the mean number of nonsynonymous substitutions in the PBR among allele pairs (<em>K</em>) when PBR substitution rates of particular branches in a phylogenetic tree are slowed down or enhanced. The bottom graph represents the accumulation of <em>K</em> values without reduction or enhancement of PBR substitution rates. Error bars indicate the SD from the mean. The simulations for upper and bottom graphs were repeated 200 times and 100 times, respectively.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><div id="sec-13" class="subsection"><h3>The cessation of accumulation in amino acid variation at the PBR</h3><p id="p-36">Contrary to expectations, <em>HLA</em> loci showed a <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> decline in intermediate values of <em>m</em>. We estimated the <em>m</em> value on the basis of <em>K</em><sub>S</sub> and <em>K</em><sub>N</sub> in the analysis. It is possible that the <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> decline was caused by the heterogeneous evolutionary rate of <em>K</em><sub>N</sub>. Although constant non-PBR nonsynonymous substitution rates in the <em>HLA-B</em> and <em>HLA-C</em> were not supported by the molecular clock test, class II loci showed molecular clock in non-PBR nonsynonymous substitutions. Therefore, the unequal substitution rate of <em>K</em><sub>N</sub> values does not necessarily affect the <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> decline.</p><p id="p-37">The symmetric balancing selection model assumes that all <em>HLA</em> alleles are sampled from a panmictic population. To investigate the possibility that the biased sampling of sequences may result in the <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> accumulation pattern, we examined the relationship between <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> and <em>m</em> using <em>HLA-DRB1</em> alleles from a Japanese population, which is a putative panmictic population. The result showed that even if the <em>DRB1</em> alleles from a putative panmictic population were used, the pattern of <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> decline in intermediate values of <em>m</em> was still observed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011726/-/DC1/FigureS6.pdf">Figure S6</a>). Therefore, <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> reduction was unlikely to be caused due to the inappropriate sampling of <em>HLA</em> alleles in the present study.</p><p id="p-38">Although possible recombinant alleles were removed from the analyses of the present study by Satta’s method (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Satta 1992</a>), we further investigated for reciprocal recombination or gene conversion using the GENECONV program (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Sawyer 1989</a>) to dismiss the possibility that recombination caused <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> deceleration. Permutation test for the sequences used in this analysis indicated intragenic recombination between some pairs of alleles. However, the Bonferroni-corrected BLAST-like test showed no significant values. Even though possible recombinants (19 alleles) estimated by both of the permutation tests in GENECONV and Satta’s method were excluded from the examination of the relationship between <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> and <em>m</em>, <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> deceleration was still observed in 45 nonrecombinant alleles (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011726/-/DC1/FigureS7.pdf">Figure S7</a>). Therefore, filtering of recombinants in the present study was unlikely to skew the <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> distribution.</p><p id="p-39">Because <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> decline was also observed in the <em>K</em><sub>N</sub> and <em>K</em><sub>S</sub> comparison (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011726/-/DC1/FigureS8.pdf">Figure S8</a>), it was interesting to know whether such <em>K</em><sub>N</sub> saturation affected the <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> accumulation in proportion to <em>K</em><sub>S</sub> + <em>K</em><sub>N</sub>. This question was addressed in the following manner. We assumed that <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> values increase proportionately with <em>K</em><sub>S</sub>, but that the <em>K</em><sub>N</sub> value is somewhat saturated for intermediate <em>K</em><sub>S</sub> values (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011726/-/DC1/FigureS9.pdf">Figure S9</a>, A–C). Next, the <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> accumulation along with <em>K</em><sub>S</sub> + <em>K</em><sub>N</sub> was examined. We observed that even if <em>K</em><sub>N</sub> did not increase constantly with <em>K</em><sub>S</sub>, a linear relationship was observed between <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> and <em>K</em><sub>S</sub> + <em>K</em><sub>N</sub> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011726/-/DC1/FigureS9.pdf">Figure S9</a>D). This result suggests that <em>K</em><sub>N</sub> saturation did not affect our results, although the reason for <em>K</em><sub>N</sub> saturation against <em>K</em><sub>S</sub> remains unknown.</p><p id="p-40"><a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a> shows that allele pairs with <em>m</em> = 14 to 18 were not observed. This was due to a lack of allelic pairs with intermediate values of <em>K</em><sub>N</sub> (<em>K</em><sub>N</sub> = 7 and 8; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011726/-/DC1/FigureS10.pdf">Figure S10</a>A), whereas <em>K</em><sub>S</sub> values are continuously observed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011726/-/DC1/FigureS10.pdf">Figure S10</a>B). The absence of certain allele pairs may be due to gaps in the database information. Many sequences obtained from the database provided only exon 2 sequences, and there is a clear need for submitting complete <em>HLA-DRB1</em> coding sequences to the database in the future.</p></div><div id="sec-14" class="subsection"><h3>Phylogenetic relationships among <em>HLA-DRB1</em> alleles</h3><p id="p-41">The phylogenetic tree for <em>DRB</em> alleles of primates suggested that the group A <em>DRB1</em> alleles in humans formed a monophyletic group from other <em>DRB</em> alleles. However, the remaining <em>HLA-DRB1</em> alleles were polyphyletic. <em>HLA-DRB1*01</em> and <em>*10</em> belonged to the same DR1 haplotype, but only those alleles were assigned to different groups (group A and group B in <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andersson (1998)</a> showed that the <em>DR1</em> haplotype with <em>HLA-DRB1*01</em> or <em>HLA-DRB1*10</em> is likely derived from the <em>DR51</em> haplotype (<em>HLA-DRB1*15</em> and <em>HLA-DRB1*16</em>) in group B during recent primate evolution. The exon 2 sequence in <em>HLA-DRB1*10</em> appears to be similar to that of <em>HLA-DRB1*01</em> (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Gongora <em>et al.</em> 1997</a>). However, the configuration of repetitive elements in a segment encompassing the promoter region to upstream of exon 2 (including exon 1) showed higher similarity of <em>HLA-DRB1*10</em> with that of <em>HLA-DRB1*03</em> in <em>DR52</em> haplotype in group A than with <em>HLA-DRB1*01</em>. The assignment of <em>HLA-DRB1*10</em> to group A in the non-PBR tree may be affected by conversion of exon 1 and surrounding sequences from <em>HLA-DRB1*03</em> to <em>HLA-DRB1*10</em>, although the exon 1 sequence was quite short. Because the bootstrap value on the node leading to the group A branch was not high (33%), the clustering of group A still remains a matter of debate.</p></div><div id="sec-15" class="subsection"><h3>The cause of stagnation of <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> accumulation</h3><p id="p-42">A comparison of the relationship between non-PBR and PBR genetic distances on each branch of the non-PBR ML tree showed three allelic lineages (<em>HLA-DRB1*04</em>, <em>*15</em>, and <em>*16</em>) with a slow PBR substitution rate and two lineages (<em>HLA-DRB1*03</em> and <em>*07</em>) with a fast rate. The allelic lineages with the slow substitution rate were frequently included in phase II. This result suggests that a decline of the PBR substitution rate likely caused the stagnation of <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> accumulation. Interestingly, although the PBR substitution rate was reduced, most <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> values of those branches were higher than unity, suggesting that balancing selection is still active.</p><p id="p-43">The computer simulation supported the finding that the slow-down of amino acid substitution rate in the PBR caused the stagnation of <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> accumulation. The duration of the tentative plateaus was slightly different between the simulation and that observed. The extent of rate acceleration or deceleration was arbitrary in this simulation and branches with fast or slow rates may have been underestimated due to the strict criteria for heterogeneity identification (see <em>Materials and Methods</em>). This probably caused the difference in the timing of plateaus.</p><p id="p-44">In the symmetric balancing selection model, nonsynonymous substitutions at the PBR sites proportionally increased with divergence time and eventually reached a plateau (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Takahata <em>et al.</em> 1992</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011726/-/DC1/FigureS1.pdf">Figure S1</a>). However, in this study <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> values did not constantly increase with the divergence time of alleles. The theory of asymmetric balancing selection may explain this phenomenon because the fitness of heterozygotes or homozygotes is not equivalent as per this theory. However, asymmetric balancing selection does not fit the model of polymorphism for the actual data in the simulation (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Takahata and Nei 1990</a>). Thus, asymmetric balancing selection could not explain the skewed <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> distribution in the present study.</p><p id="p-45">We used PBR sites identified by <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Brown <em>et al.</em> (1993)</a> to estimate <em>K</em><sub>B(</sub><em><sub>m</sub></em><sub>)</sub> values. When PBR sites identified by recent assays for higher-order structures of the HLA molecule (S. Kusano and S. Yokoyama, personal communication) were used, the <em>K</em><sub>B</sub> saturation pattern was also observed in phase II. In addition, allele pairs with low PBR substitution rates were also assigned to phase II. The PBR definitions did not affect our results.</p></div><div id="sec-16" class="subsection"><h3>Biological significance of substitution rate heterogeneity</h3><p id="p-46">Slow nucleotide substitution rates, in particular allelic lineages (<em>HLA-DRB1*04</em>, <em>*15</em>, and <em>*16</em>), may be necessary to retain the binding affinity for specific peptides. In general, evolutionary rates of hosts and pathogens have been accelerated by virtue of their arms race. However, HLA molecules preferentially target evolutionarily conserved regions that are functionally important sites for pathogens. Among HLA alleles there appears to be a difference of correlation between the peptide-binding affinity and conservation of the targeted regions (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hertz <em>et al.</em> 2011</a>). If this is true, then these slower lineages must be highly specific to certain pathogens, which are the source of these specific peptides.</p><p id="p-47">Using the IEDB database, we examined peptides bound by <em>DRB1</em> allelic lineage with the slow PBR substitution rate (<em>HLA-DRB1*04</em>, <em>*15</em>, and <em>*16</em>) and pathogens from which the peptides were derived (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011726/-/DC1/TableS3.pdf">Table S3</a>). The prediction from the database showed that one pathogen was specifically recognized by the HLA molecule with a fast PBR substitutions rate, whereas 12 pathogens were specifically recognized by HLA molecules with a slow PBR substitution rate. These more slowly evolving HLA molecules recognized viruses and bacteria that cause disease specific to humans. In addition, HLA-DRB1*04:01 and HLA-DRB1*04:07 bound peptides from lactic bacteria, which are also specific to humans. To cope with these specific pathogens, the nucleotide substitution rate at particular <em>DRB1</em> alleles had possibly slowed down relative to those at other alleles. One may hypothesize that evolutionary rates of amino acids at sites 9, 11, 13, and 37 in the HLA-DRB1*04 lineage and at sites 11 and 13 in the DRB1*15/*16 lineages were decelerated, respectively, to retain the affinity of the conserved proteomic region in pathogens (green branches in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011726/-/DC1/FigureS4.pdf">Figure S4</a>). According to our estimate, the divergence time of allelic pairs in phase II is 18 MYA to 24 MYA. Humans and the pathogens described above might have coexisted for long periods of time, at least before speciation events of the subfamily Homininae.</p><p id="p-48">In summary, we observed the episodic fluctuations of the amino acid substitution rate in the PBR over the course of <em>HLA</em> evolution. This observation was not anticipated because we believed that nonsynonymous substitutions in the PBR increased with the divergence time of the alleles until the substitutions reached saturation. In the case of the <em>DRB1</em> gene, such fluctuation is probably due to a decrease of amino acid substitution rates at the PBR on the stem of particular allelic lineages. This suggests that a part of the peptide-binding repertoire at the <em>HLA-DRB1</em> locus has been limited over long periods of time due to the recognition of certain pathogens.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-49">We give special thanks to Dr. Naoyuki Takahata for providing valuable comments, and to Dr. Seisuke Kusano, Dr. Shigeyuki Yokoyama, and the Institute of Physical and Chemical Research (RIKEN). We thank Dr. John A. Eimes for the critical checking of the English language of this manuscript. This work was supported by a grant-in-aid for Scientific Research on Innovative Areas from the Ministry of Education, Culture, Sports, Science, and Technology of Japan (22133007).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011726/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011726/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-50"><em>Communicating editor: S. W. Scherer</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-12-25"><span class="received-label">Received </span>December 25, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-04-24"><span class="accepted-label">Accepted </span>April 24, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Yasukochi and Satta</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Evolution of the human HLA-DR region.</span> <abbr class="cit-jnl-abbrev">Front. Biosci.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">d739</span>–<span class="cit-lpage">d745</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2Bbioscience%2B%253A%2B%2Ba%2Bjournal%2Band%2Bvirtual%2Blibrary%26rft.stitle%253DFront%2BBiosci%26rft.aulast%253DAndersson%26rft.auinit1%253DG.%26rft.volume%253D3%26rft.spage%253Dd739%26rft.epage%253Dd745%26rft.atitle%253DEvolution%2Bof%2Bthe%2Bhuman%2BHLA-DR%2Bregion.%26rft_id%253Dinfo%253Apmid%252F9675159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9675159&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.2" data-doi="10.1038/364033a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jardetzky</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorga</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stern</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Urban</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Three-dimensional structure of the human class II histocompatibility antigen HLA-DR1.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">364</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBrown%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D364%26rft.issue%253D6432%26rft.spage%253D33%26rft.epage%253D39%26rft.atitle%253DThree-dimensional%2Bstructure%2Bof%2Bthe%2Bhuman%2Bclass%2BII%2Bhistocompatibility%2Bantigen%2BHLA-DR1.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F364033a0%26rft_id%253Dinfo%253Apmid%252F8316295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/364033a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8316295&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LK81800048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.3" data-doi="10.1126/science.283.5408.1748"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carrington</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kissner</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vlahov</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title"><em>HLA</em> and HIV-1: heterozygote advantage and <em>B*35-Cw*04</em> disadvantage.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">283</span>: <span class="cit-fpage">1748</span>–<span class="cit-lpage">1752</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCarrington%26rft.auinit1%253DM.%26rft.volume%253D283%26rft.issue%253D5408%26rft.spage%253D1748%26rft.epage%253D1752%26rft.atitle%253DHLA%2Band%2BHIV-1%253A%2BHeterozygote%2BAdvantage%2Band%2BB%2A35-Cw%2A04%2BDisadvantage%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.283.5408.1748%26rft_id%253Dinfo%253Apmid%252F10073943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODMvNTQwOC8xNzQ4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzEyMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.4" data-doi="10.1002/eji.1830230219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Demotz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbey</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corradin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amoroso</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanzavecchia</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">The set of naturally processed peptides displayed by DR molecules is tuned by polymorphism of residue 86.</span> <abbr class="cit-jnl-abbrev">Eur. J. Immunol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">425</span>–<span class="cit-lpage">432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuropean%2Bjournal%2Bof%2Bimmunology%26rft.stitle%253DEur%2BJ%2BImmunol%26rft.aulast%253DDemotz%26rft.auinit1%253DS.%26rft.volume%253D23%26rft.issue%253D2%26rft.spage%253D425%26rft.epage%253D432%26rft.atitle%253DThe%2Bset%2Bof%2Bnaturally%2Bprocessed%2Bpeptides%2Bdisplayed%2Bby%2BDR%2Bmolecules%2Bis%2Btuned%2Bby%2Bpolymorphism%2Bof%2Bresidue%2B86.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Feji.1830230219%26rft_id%253Dinfo%253Apmid%252F7679644%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/eji.1830230219&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7679644&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993KM03300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.5" data-doi="10.1038/256050a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doherty</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zinkernagel</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Enhanced immunological surveillance in mice heterozygous at the H-2 gene complex.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">256</span>: <span class="cit-fpage">50</span>–<span class="cit-lpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDoherty%26rft.auinit1%253DP.%2BC.%26rft.volume%253D256%26rft.issue%253D5512%26rft.spage%253D50%26rft.epage%253D52%26rft.atitle%253DEnhanced%2Bimmunological%2Bsurveillance%2Bin%2Bmice%2Bheterozygous%2Bat%2Bthe%2BH-2%2Bgene%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F256050a0%26rft_id%253Dinfo%253Apmid%252F1079575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/256050a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1079575&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975AG55100036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-4.7.1217.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">PHYLIP (Phylogeny Inference Package) ver.3.69. Distributed by the author</span>, <span class="cit-publ-name">Department of Genome Sciences, University of Washington</span>, <span class="cit-publ-loc">Seattle, USA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gongora</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Figueroa</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Complex origin of the HLA-DR10 haplotype.</span> <abbr class="cit-jnl-abbrev">J. Immunol.</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">6044</span>–<span class="cit-lpage">6051</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BImmunology%26rft.stitle%253DJ.%2BImmunol.%26rft.aulast%253DGongora%26rft.auinit1%253DR%26rft.volume%253D159%26rft.issue%253D12%26rft.spage%253D6044%26rft.epage%253D6051%26rft.atitle%253DComplex%2Borigin%2Bof%2Bthe%2BHLA-DR10%2Bhaplotype%26rft_id%253Dinfo%253Apmid%252F9550403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTE6IjE1OS8xMi82MDQ0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzEyMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.8" data-doi="10.1093/nar/gkq1128"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez-Galarza</span>  <span class="cit-name-given-names">F. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christmas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Middleton</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">A. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Allele frequency net: a database and online repository for immune gene frequencies in worldwide populations.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">D913</span>–<span class="cit-lpage">D919</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq1128%26rft_id%253Dinfo%253Apmid%252F21062830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzOS9zdXBwbF8xL0Q5MTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTIxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.9" data-doi="10.1073/pnas.83.8.2642"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gregersen</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merryman</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Degar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Molecular diversity of HLA-DR4 haplotypes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">2642</span>–<span class="cit-lpage">2646</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DGregersen%26rft.auinit1%253DP.%2BK.%26rft.volume%253D83%26rft.issue%253D8%26rft.spage%253D2642%26rft.epage%253D2646%26rft.atitle%253DMolecular%2Bdiversity%2Bof%2BHLA-DR4%2Bhaplotypes.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.83.8.2642%26rft_id%253Dinfo%253Apmid%252F3458223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI4My84LzI2NDIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTIxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.10" data-doi="10.1007/BF02101694"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hasegawa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kishino</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yano</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Dating of the human-ape splitting by a molecular clock of mitochondrial DNA.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">160</span>–<span class="cit-lpage">174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DHasegawa%26rft.auinit1%253DM.%26rft.volume%253D22%26rft.issue%253D2%26rft.spage%253D160%26rft.epage%253D174%26rft.atitle%253DDating%2Bof%2Bthe%2Bhuman-ape%2Bsplitting%2Bby%2Ba%2Bmolecular%2Bclock%2Bof%2Bmitochondrial%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02101694%26rft_id%253Dinfo%253Apmid%252F3934395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02101694&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3934395&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985AWB1600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.11" data-doi="10.1128/JVI.01966-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hertz</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">John</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaudieri</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mapping the landscape of host-pathogen coevolution: HLA class I binding and its relationship with evolutionary conservation in human and viral proteins.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">1310</span>–<span class="cit-lpage">1321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01966-10%26rft_id%253Dinfo%253Apmid%252F21084470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg1LzMvMTMxMCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMjE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-book" id="cit-4.7.1217.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">A. V. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">HLA associations with malaria in Africa: some implications for MHC evolution</span>, pp. <span class="cit-fpage">403</span>–<span class="cit-lpage">419</span> in <span class="cit-source">Molecular evolution of the major histocompatibility complex</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">D.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin, Heidelberg</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.13" data-doi="10.1038/335167a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Pattern of nucleotide substitution at major histocompatibility complex class I loci reveals overdominant selection.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">335</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHughes%26rft.auinit1%253DA.%2BL.%26rft.volume%253D335%26rft.issue%253D6186%26rft.spage%253D167%26rft.epage%253D170%26rft.atitle%253DPattern%2Bof%2Bnucleotide%2Bsubstitution%2Bat%2Bmajor%2Bhistocompatibility%2Bcomplex%2Bclass%2BI%2Bloci%2Breveals%2Boverdominant%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F335167a0%26rft_id%253Dinfo%253Apmid%252F3412472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/335167a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3412472&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988P964700056&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.14" data-doi="10.1073/pnas.86.3.958"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Nucleotide substitution at major histocompatibility complex class II loci: evidence for overdominant selection.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">958</span>–<span class="cit-lpage">962</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DHughes%26rft.auinit1%253DA.%2BL.%26rft.volume%253D86%26rft.issue%253D3%26rft.spage%253D958%26rft.epage%253D962%26rft.atitle%253DNucleotide%2Bsubstitution%2Bat%2Bmajor%2Bhistocompatibility%2Bcomplex%2Bclass%2BII%2Bloci%253A%2Bevidence%2Bfor%2Boverdominant%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.86.3.958%26rft_id%253Dinfo%253Apmid%252F2492668%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI4Ni8zLzk1OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMjE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.15" data-doi="10.1146/annurev.genet.32.1.415"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeager</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Natural selection at major histocompatibility complex loci of vertebrates.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">415</span>–<span class="cit-lpage">435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DHughes%26rft.auinit1%253DA.%2BL.%26rft.volume%253D32%26rft.issue%253D1%26rft.spage%253D415%26rft.epage%253D435%26rft.atitle%253DNATURAL%2BSELECTION%2BAT%2BMAJOR%2BHISTOCOMPATIBILITY%2BCOMPLEX%2BLOCI%2BOF%2BVERTEBRATES%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.32.1.415%26rft_id%253Dinfo%253Apmid%252F9928486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.32.1.415&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9928486&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000078222700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.16" data-doi="10.1093/bioinformatics/8.3.275"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">The rapid generation of mutation data matrices from protein sequences.</span> <abbr class="cit-jnl-abbrev">Comput. Appl. Biosci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">275</span>–<span class="cit-lpage">282</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DJones%26rft.auinit1%253DD.%2BT.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D275%26rft.epage%253D282%26rft.atitle%253DThe%2Brapid%2Bgeneration%2Bof%2Bmutation%2Bdata%2Bmatrices%2Bfrom%2Bprotein%2Bsequences%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F8.3.275%26rft_id%253Dinfo%253Apmid%252F1633570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo3OiI4LzMvMjc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzEyMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.17" data-doi="10.1146/annurev.genet.41.110306.130137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nikolaidis</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">MHC, TSP, and the origin of species: from immunogenetics to evolutionary genetics.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">281</span>–<span class="cit-lpage">304</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DKlein%26rft.auinit1%253DJ.%26rft.volume%253D41%26rft.spage%253D281%26rft.epage%253D304%26rft.atitle%253DMHC%252C%2BTSP%252C%2Band%2Bthe%2Borigin%2Bof%2Bspecies%253A%2Bfrom%2Bimmunogenetics%2Bto%2Bevolutionary%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.41.110306.130137%26rft_id%253Dinfo%253Apmid%252F18076327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.41.110306.130137&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076327&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252359500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.18" data-doi="10.1073/pnas.94.14.7673"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kusaba</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishio</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Satta</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinata</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ockendon</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Striking sequence similarity in inter- and intra-specific comparisons of class I <em>SLG</em> alleles from <em>Brassica oleracea</em> and <em>Brassica campestris</em>: implications for the evolution and recognition mechanism.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">7673</span>–<span class="cit-lpage">7678</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DKusaba%26rft.auinit1%253DM.%26rft.volume%253D94%26rft.issue%253D14%26rft.spage%253D7673%26rft.epage%253D7678%26rft.atitle%253DStriking%2Bsequence%2Bsimilarity%2Bin%2Binter-%2Band%2Bintra-specific%2Bcomparisons%2Bof%2Bclass%2BI%2BSLG%2Balleles%2Bfrom%2BBrassica%2Boleracea%2Band%2BBrassica%2Bcampestris%253A%2BImplications%2Bfor%2Bthe%2Bevolution%2Band%2Brecognition%2Bmechanism%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.94.14.7673%26rft_id%253Dinfo%253Apmid%252F9207151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTQvMTQvNzY3MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMjE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Musolf</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer-Lucht</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sommer</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Evolution of MHC-<em>DRB</em> class II polymorphism in the genus <em>Apodemus</em> and a comparison of <em>DRB</em> sequences within the family Muridae (Mammalia: Rodentia).</span> <abbr class="cit-jnl-abbrev">Immunogenetics</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">420</span>–<span class="cit-lpage">426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunogenetics%26rft.stitle%253DImmunogenetics%26rft.aulast%253DMusolf%26rft.auinit1%253DK.%26rft.volume%253D56%26rft.issue%253D6%26rft.spage%253D420%26rft.epage%253D426%26rft.atitle%253DEvolution%2Bof%2BMHC-DRB%2Bclass%2BII%2Bpolymorphism%2Bin%2Bthe%2Bgenus%2BApodemus%2Band%2Ba%2Bcomparison%2Bof%2BDRB%2Bsequences%2Bwithin%2Bthe%2Bfamily%2BMuridae%2B%2528Mammalia%253A%2BRodentia%2529.%26rft_id%253Dinfo%253Apmid%252F15351919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15351919&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.20" data-doi="10.1073/pnas.89.22.10896"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nowak</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarczy-Hornoch</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Austyn</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">The optimal number of major histocompatibility complex molecules in an individual.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">10896</span>–<span class="cit-lpage">10899</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DNowak%26rft.auinit1%253DM.%2BA.%26rft.volume%253D89%26rft.issue%253D22%26rft.spage%253D10896%26rft.epage%253D10899%26rft.atitle%253DThe%2Boptimal%2Bnumber%2Bof%2Bmajor%2Bhistocompatibility%2Bcomplex%2Bmolecules%2Bin%2Ban%2Bindividual.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.89.22.10896%26rft_id%253Dinfo%253Apmid%252F1438295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiODkvMjIvMTA4OTYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTIxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.21" data-doi="10.1098/rspb.2008.1525"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Telfer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piertney</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Major histocompatibility complex (MHC) heterozygote superiority to natural multi-parasite infections in the water vole (<em>Arvicola terrestris</em>).</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">276</span>: <span class="cit-fpage">1119</span>–<span class="cit-lpage">1128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DOliver%26rft.auinit1%253DM.K%26rft.volume%253D276%26rft.issue%253D1659%26rft.spage%253D1119%26rft.epage%253D1128%26rft.atitle%253DMajor%2Bhistocompatibility%2Bcomplex%2B%2528MHC%2529%2Bheterozygote%2Bsuperiority%2Bto%2Bnatural%2Bmulti-parasite%2Binfections%2Bin%2Bthe%2Bwater%2Bvole%2B%2528Arvicola%2Bterrestris%2529%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2008.1525%26rft_id%253Dinfo%253Apmid%252F19129114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMzoiMjc2LzE2NTkvMTExOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMjE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.22" data-doi="10.1073/pnas.88.16.7343"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ong</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willcox</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wordsworth</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beeson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vincent</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Critical role for the Val/Gly<sup>86</sup> HLA-DR<em>β</em> dimorphism in autoantigen presentation to human T cells.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">7343</span>–<span class="cit-lpage">7347</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DOng%26rft.auinit1%253DB.%26rft.volume%253D88%26rft.issue%253D16%26rft.spage%253D7343%26rft.epage%253D7347%26rft.atitle%253DCritical%2Brole%2Bfor%2Bthe%2BVal%252FGly86%2BHLA-DR%2Bbeta%2Bdimorphism%2Bin%2Bautoantigen%2Bpresentation%2Bto%2Bhuman%2BT%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.88.16.7343%26rft_id%253Dinfo%253Apmid%252F1714600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODgvMTYvNzM0MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMjE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.23" data-doi="10.1073/pnas.162006499"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Penn</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Damjanovich</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Potts</span>  <span class="cit-name-given-names">W. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">MHC heterozygosity confers a selective advantage against multiple-strain infections.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">11260</span>–<span class="cit-lpage">11264</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.162006499%26rft_id%253Dinfo%253Apmid%252F12177415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTkvMTcvMTEyNjAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTIxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.24" data-doi="10.1016/S0022-2836(03)00750-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reche</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinherz</span>  <span class="cit-name-given-names">E. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Sequence variability analysis of human class I and class II MHC molecules: Functional and structural correlates of amino acid polymorphisms.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">331</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DReche%26rft.auinit1%253DP.%2BA.%26rft.volume%253D331%26rft.issue%253D3%26rft.spage%253D623%26rft.epage%253D641%26rft.atitle%253DSequence%2Bvariability%2Banalysis%2Bof%2Bhuman%2Bclass%2BI%2Band%2Bclass%2BII%2BMHC%2Bmolecules%253A%2Bfunctional%2Band%2Bstructural%2Bcorrelates%2Bof%2Bamino%2Bacid%2Bpolymorphisms.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-2836%252803%252900750-2%26rft_id%253Dinfo%253Apmid%252F12899833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-2836(03)00750-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12899833&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184651500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.25" data-doi="10.1093/nar/gki032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waller</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoehr</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marsh</span>  <span class="cit-name-given-names">S. G. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">IPD-the Immuno Polymorphism Database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">D523</span>–<span class="cit-lpage">D526</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DRobinson%26rft.auinit1%253DJ.%26rft.volume%253D33%26rft.issue%253Dsuppl_1%26rft.spage%253DD523%26rft.epage%253DD526%26rft.atitle%253DIPD--the%2BImmuno%2BPolymorphism%2BDatabase%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki032%26rft_id%253Dinfo%253Apmid%252F15608253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzMy9zdXBwbF8xL0Q1MjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTIxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.26" data-doi="10.1093/nar/gkq998"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mistry</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McWilliam</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parham</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The IMGT/HLA database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">D1171</span>–<span class="cit-lpage">D1176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq998%26rft_id%253Dinfo%253Apmid%252F21071412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE2OiIzOS9zdXBwbF8xL0QxMTcxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzEyMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.27" data-doi="10.1099/ijs.0.01472-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saitou</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The neighbor-joining method: a new method for reconstructing phylogenetic trees.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">406</span>–<span class="cit-lpage">425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DSaitou%26rft.auinit1%253DN%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253D406%26rft.epage%253D425%26rft.atitle%253DThe%2Bneighbor-joining%2Bmethod%253A%2Ba%2Bnew%2Bmethod%2Bfor%2Breconstructing%2Bphylogenetic%2Btrees%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fijs.0.01472-0%26rft_id%253Dinfo%253Apmid%252F3447015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjc6IjQvNC80MDYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTIxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-other" id="cit-4.7.1217.28"><div class="cit-metadata unstructured">Satta, Y., 1992 Balancing selection at HLA loci, pp. 111–131 in <em>The Proceedings of the 17th Taniguchi Symposium</em>, edited by N. Takahata. Japan Science Society Press, Tokyo.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.29" data-doi="10.1073/pnas.91.15.7184"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Satta</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, C. O’hUigin, N. Takahata, and J. Klein., <span class="cit-pub-date">1994</span> <span class="cit-article-title">Intensity of natural selection at the major histocompatibility complex loci.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">7184</span>–<span class="cit-lpage">7188</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSatta%26rft.auinit1%253DY.%26rft.volume%253D91%26rft.issue%253D15%26rft.spage%253D7184%26rft.epage%253D7188%26rft.atitle%253DIntensity%2Bof%2Bnatural%2Bselection%2Bat%2Bthe%2Bmajor%2Bhistocompatibility%2Bcomplex%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.91.15.7184%26rft_id%253Dinfo%253Apmid%252F8041766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTEvMTUvNzE4NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMjE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sawyer</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Statistical tests for detecting gene conversion.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">526</span>–<span class="cit-lpage">538</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DSawyer%26rft.auinit1%253DS%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253D526%26rft.epage%253D538%26rft.atitle%253DStatistical%2Btests%2Bfor%2Bdetecting%2Bgene%2Bconversion%26rft_id%253Dinfo%253Apmid%252F2677599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjc6IjYvNS81MjYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTIxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slade</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCallum</span>  <span class="cit-name-given-names">H. I.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Overdominant <em>vs.</em> frequency-dependent selection at MHC loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">861</span>–<span class="cit-lpage">862</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSlade%26rft.auinit1%253DR.%2BW.%26rft.volume%253D132%26rft.issue%253D3%26rft.spage%253D861%26rft.epage%253D864%26rft.atitle%253DOverdominant%2Bvs.%2Bfrequency-dependent%2Bselection%2Bat%2BMHC%2Bloci.%26rft_id%253Dinfo%253Apmid%252F1468635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo5OiIxMzIvMy84NjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTIxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.32" data-doi="10.1186/1742-9994-2-16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sommer</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The importance of immune gene variability (MHC) in evolutionary ecology and conservation.</span> <abbr class="cit-jnl-abbrev">Front. Zool.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFront%2BZool%26rft.aulast%253DSommer%26rft.auinit1%253DS.%2BS.%26rft.volume%253D2%26rft.spage%253D16%26rft.epage%253D16%26rft.atitle%253DThe%2Bimportance%2Bof%2Bimmune%2Bgene%2Bvariability%2B%2528MHC%2529%2Bin%2Bevolutionary%2Becology%2Band%2Bconservation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1742-9994-2-16%26rft_id%253Dinfo%253Apmid%252F16242022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1742-9994-2-16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16242022&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.33" data-doi="10.1098/rspb.2009.2084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spurgin</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richardson</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">How pathogens drive genetic diversity: MHC, mechanisms and misunderstandings.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">277</span>: <span class="cit-fpage">979</span>–<span class="cit-lpage">988</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2009.2084%26rft_id%253Dinfo%253Apmid%252F20071384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMjoiMjc3LzE2ODQvOTc5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzEyMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.34" data-doi="10.1038/368215a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stern</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jardetzky</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorga</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Urban</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Crystal structure of the human class II MHC protein HLA-DR1 complexed with an influenza virus peptide.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">368</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DStern%26rft.auinit1%253DL.%2BJ.%26rft.volume%253D368%26rft.issue%253D6468%26rft.spage%253D215%26rft.epage%253D221%26rft.atitle%253DCrystal%2Bstructure%2Bof%2Bthe%2Bhuman%2Bclass%2BII%2BMHC%2Bprotein%2BHLA-DR1%2Bcomplexed%2Bwith%2Ban%2Binfluenza%2Bvirus%2Bpeptide.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F368215a0%26rft_id%253Dinfo%253Apmid%252F8145819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/368215a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8145819&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994NA87000046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.35" data-doi="10.1073/pnas.87.7.2419"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takahata</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">A simple genealogical structure of strongly balanced allelic lines and trans-species evolution of polymorphism.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">2419</span>–<span class="cit-lpage">2423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DTakahata%26rft.auinit1%253DN.%26rft.volume%253D87%26rft.issue%253D7%26rft.spage%253D2419%26rft.epage%253D2423%26rft.atitle%253DA%2Bsimple%2Bgenealogical%2Bstructure%2Bof%2Bstrongly%2Bbalanced%2Ballelic%2Blines%2Band%2Btrans-species%2Bevolution%2Bof%2Bpolymorphism.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.87.7.2419%26rft_id%253Dinfo%253Apmid%252F2320564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI4Ny83LzI0MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTIxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takahata</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Allelic genealogy under overdominant and frequency-dependent selection and polymorphism of major histocompatibility complex loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">967</span>–<span class="cit-lpage">978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTakahata%26rft.auinit1%253DN.%26rft.volume%253D124%26rft.issue%253D4%26rft.spage%253D967%26rft.epage%253D978%26rft.atitle%253DAllelic%2Bgenealogy%2Bunder%2Boverdominant%2Band%2Bfrequency-dependent%2Bselection%2Band%2Bpolymorphism%2Bof%2Bmajor%2Bhistocompatibility%2Bcomplex%2Bloci.%26rft_id%253Dinfo%253Apmid%252F2323559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI0LzQvOTY3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzEyMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takahata</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <abbr class="cit-jnl-abbrev">Sampling Errors in Phylogeny.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">494</span>–<span class="cit-lpage">502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSampling%2BErrors%2Bin%2BPhylogeny.%26rft.volume%253D8%26rft.spage%253D494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takahata</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Satta</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Polymorphism and balancing selection at major histocompatibility complex loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">925</span>–<span class="cit-lpage">938</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTakahata%26rft.auinit1%253DN.%26rft.volume%253D130%26rft.issue%253D4%26rft.spage%253D925%26rft.epage%253D938%26rft.atitle%253DPolymorphism%2Band%2Bbalancing%2Bselection%2Bat%2Bmajor%2Bhistocompatibility%2Bcomplex%2Bloci.%26rft_id%253Dinfo%253Apmid%252F1582567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTMwLzQvOTI1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzEyMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.39" data-doi="10.1093/molbev/msr121"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stecher</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2731</span>–<span class="cit-lpage">2739</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr121%26rft_id%253Dinfo%253Apmid%252F21546353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyOC8xMC8yNzMxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzEyMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.40" data-doi="10.1038/ng0997-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thursz</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenwood</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">A. V.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Heterozygote advantage for HLA class-II type in hepatitis B virus infection.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DThursz%26rft.auinit1%253DM.%2BR.%26rft.volume%253D17%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D12%26rft.atitle%253DHeterozygote%2Badvantage%2Bfor%2BHLA%2Bclass-II%2Btype%2Bin%2Bhepatitis%2BB%2Bvirus%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng0997-11%26rft_id%253Dinfo%253Apmid%252F9288086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng0997-11&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9288086&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XU72400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.41" data-doi="10.1007/BF02199809"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verreck</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van de Poel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drijfhout</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amons</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coligan</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Natural peptides isolated from Gly86/Val86-containing variants of HLA-DR1, -DR11, -DR13, and -DR52.</span> <abbr class="cit-jnl-abbrev">Immunogenetics</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">392</span>–<span class="cit-lpage">397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunogenetics%26rft.stitle%253DImmunogenetics%26rft.aulast%253DVerreck%26rft.auinit1%253DF.%2BA.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D392%26rft.epage%253D397%26rft.atitle%253DNatural%2Bpeptides%2Bisolated%2Bfrom%2BGly86%252FVal86-containing%2Bvariants%2Bof%2BHLA-DR1%252C%2B-DR11%252C%2B-DR13%252C%2Band%2B-DR52.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02199809%26rft_id%253Dinfo%253Apmid%252F8606061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02199809&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8606061&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UD86300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.42" data-doi="10.1038/336222a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vidović</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matzinger</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Unresponsiveness to a foreign antigen can be caused by self-tolerance.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">336</span>: <span class="cit-fpage">222</span>–<span class="cit-lpage">225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DVidovi%25C4%2587%26rft.auinit1%253DD.%26rft.volume%253D336%26rft.issue%253D6196%26rft.spage%253D222%26rft.epage%253D225%26rft.atitle%253DUnresponsiveness%2Bto%2Ba%2Bforeign%2Bantigen%2Bcan%2Bbe%2Bcaused%2Bby%2Bself-tolerance.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F336222a0%26rft_id%253Dinfo%253Apmid%252F3143074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/336222a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3143074&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1217.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.43" data-doi="10.1093/nar/gkp1004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vita</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zarebski</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenbaum</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emami</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoof</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The immune epitope database 2.0.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">D854</span>–<span class="cit-lpage">D862</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp1004%26rft_id%253Dinfo%253Apmid%252F19906713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzOC9zdXBwbF8xL0Q4NTQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTIxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1217.44" data-doi="10.1098/rstb.2008.0174"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woelfing</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Traulsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milinski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boehm</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Does intra-individual major histocompatibility complex diversity keep a golden mean?</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">364</span>: <span class="cit-fpage">117</span>–<span class="cit-lpage">128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DWoelfing%26rft.auinit1%253DB.%26rft.volume%253D364%26rft.issue%253D1513%26rft.spage%253D117%26rft.epage%253D128%26rft.atitle%253DDoes%2Bintra-individual%2Bmajor%2Bhistocompatibility%2Bcomplex%2Bdiversity%2Bkeep%2Ba%2Bgolden%2Bmean%253F%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2008.0174%26rft_id%253Dinfo%253Apmid%252F18926972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEyOiIzNjQvMTUxMy8xMTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTIxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/7/1217.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/7/1205" title="Mapping Small Effect Mutations in Saccharomyces cerevisiae: Impacts of Experimental Design and Mutational Properties" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/7/1227" title="Infinium Monkeys: Infinium 450K Array for the Cynomolgus macaque (Macaca fascicularis)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/7"><span>Volume 4 Issue 7, July 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/7.cover-source.jpg?itok=MrsYFroL" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/7/1217" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-SNwusJotE6sNiThZPGQ2nYp_TIISn1ZZrQ6ue6Ew_ro" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genetics-immunity" class="" data-icon-position="" data-hide-link-title="0">Genetics of immunity</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/7/1217" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/7/1217" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/7/1217" class="active" data-icon-position="" data-hide-link-title="0">Nonsynonymous Substitution Rate Heterogeneity in the Peptide-Binding Region Among Different HLA-DRB1 Lineages in Humans</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459927" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-M9DhbgSDiJ3aLRt4uBr9DsZ82ZAsz0hLI3bSK3xySuA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/7/1217.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/7/1217" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-BtjI1YtoQVeOYbHvWo5ztNFtkgv3-uC6poxqPnIlyoY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/7/1217" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/7/1217.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459927" id="node459927--2" data-pisa="ggg;4/7/1217" data-pisa-master="ggg;g3.114.011726" data-apath="/ggg/4/7/1217.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Nonsynonymous Substitution Rate Heterogeneity in the Peptide-Binding Region Among Different <em>HLA-DRB1</em> Lineages in Humans</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yoshiki Yasukochi</span> and <span class="highwire-citation-author" data-delta="1">Yoko Satta</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1217-1226;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011726 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459927--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yoshiki Yasukochi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular and Genetic Epidemiology, Faculty of Medicine, University of Tsukuba, Tsukuba, Ibaraki 305-8575, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yoshiki%2BYasukochi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yasukochi%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYoshiki%2BYasukochi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hyasukou@proof.ocn.ne.jp" class="" data-icon-position="" data-hide-link-title="0">hyasukou@proof.ocn.ne.jp</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yoko Satta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Evolutionary Studies of Biosystems, the Graduate University for Advanced Studies (SOKENDAI), Hayama, Kanagawa 240-0193, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yoko%2BSatta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Satta%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYoko%2BSatta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/7/1217" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/7/1217&amp;title=Nonsynonymous%20Substitution%20Rate%20Heterogeneity%20in%20the%20Peptide-Binding%20Region%20Among%20Different%20HLA-DRB1%20Lineages%20in%20Humans" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/7/1217&amp;title=Nonsynonymous%20Substitution%20Rate%20Heterogeneity%20in%20the%20Peptide-Binding%20Region%20Among%20Different%20HLA-DRB1%20Lineages%20in%20Humans" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/7/1217&amp;title=Nonsynonymous%20Substitution%20Rate%20Heterogeneity%20in%20the%20Peptide-Binding%20Region%20Among%20Different%20HLA-DRB1%20Lineages%20in%20Humans" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/7/1217&amp;text=Nonsynonymous%20Substitution%20Rate%20Heterogeneity%20in%20the%20Peptide-Binding%20Region%20Among%20Different%20HLA-DRB1%20Lineages%20in%20Humans" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/7/1217&amp;title=Nonsynonymous%20Substitution%20Rate%20Heterogeneity%20in%20the%20Peptide-Binding%20Region%20Among%20Different%20HLA-DRB1%20Lineages%20in%20Humans" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/7/1217&amp;t=Nonsynonymous%20Substitution%20Rate%20Heterogeneity%20in%20the%20Peptide-Binding%20Region%20Among%20Different%20HLA-DRB1%20Lineages%20in%20Humans" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/7/1217&amp;title=Nonsynonymous%20Substitution%20Rate%20Heterogeneity%20in%20the%20Peptide-Binding%20Region%20Among%20Different%20HLA-DRB1%20Lineages%20in%20Humans" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/7/1217&amp;title=Nonsynonymous%20Substitution%20Rate%20Heterogeneity%20in%20the%20Peptide-Binding%20Region%20Among%20Different%20HLA-DRB1%20Lineages%20in%20Humans" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459927/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459927" id="node459927--3" data-pisa="ggg;4/7/1217" data-pisa-master="ggg;g3.114.011726" data-apath="/ggg/4/7/1217.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Nonsynonymous Substitution Rate Heterogeneity in the Peptide-Binding Region Among Different <em>HLA-DRB1</em> Lineages in Humans</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yoshiki Yasukochi</span> and <span class="highwire-citation-author" data-delta="1">Yoko Satta</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1217-1226;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011726 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459927--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yoshiki Yasukochi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular and Genetic Epidemiology, Faculty of Medicine, University of Tsukuba, Tsukuba, Ibaraki 305-8575, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yoshiki%2BYasukochi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yasukochi%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYoshiki%2BYasukochi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hyasukou@proof.ocn.ne.jp" class="" data-icon-position="" data-hide-link-title="0">hyasukou@proof.ocn.ne.jp</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yoko Satta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Evolutionary Studies of Biosystems, the Graduate University for Advanced Studies (SOKENDAI), Hayama, Kanagawa 240-0193, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yoko%2BSatta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Satta%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYoko%2BSatta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1217&amp;rft.epage=1226&amp;rft.atitle=Nonsynonymous%20Substitution%20Rate%20Heterogeneity%20in%20the%20Peptide-Binding%20Region%20Among%20Different%20HLA-DRB1%20Lineages%20in%20Humans&amp;rft.volume=4&amp;rft.issue=7&amp;rft.date=2014-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Yasukochi%2C+Yoshiki&amp;rft.au=Satta%2C+Yoko"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459927/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459927/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459927/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459927/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459927/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459927/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459927/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459927/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459927/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459927/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459927/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459927/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496047" id="node496047" data-pisa="ggg;g3.120.401119v1" data-pisa-master="ggg;g3.120.401119" data-apath="/ggg/early/2020/03/24/g3.120.401119.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/24/g3.120.401119" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genes Encoding Recognition of the <em>Cladosporium fulvum</em> Effector Protein Ecp5 Are Encoded at Several Loci in the Tomato Genome</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495868" id="node495868" data-pisa="ggg;10/3/1137" data-pisa-master="ggg;g3.119.400734" data-apath="/ggg/10/3/1137.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1137" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Unique Antimicrobial Recognition and Signaling Pathways in Tardigrades with a Comparison Across Ecdysozoa</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495562" id="node495562" data-pisa="ggg;10/2/613" data-pisa-master="ggg;g3.119.400685" data-apath="/ggg/10/2/613.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/613" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">QTL Mapping of Intestinal Neutrophil Variation in Threespine Stickleback Reveals Possible Gene Targets Connecting Intestinal Inflammation and Systemic Health</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genetics%20of%20Immunity" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genetics of Immunity</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/7/1217" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/7/1217.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/7/1217.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/7/1217.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/7/1217.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
