<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/7/1193" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/7/1193.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/7/1193.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/7/1193.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/7/1193.atom" />
<meta name="HW.pisa" content="ggg;4/7/1193" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Two Genomic Regions Contribute Disproportionately to Geographic Differentiation in Wild Barley" />
<meta name="DC.Identifier" content="10.1534/g3.114.010561" />
<meta name="DC.Date" content="2014-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Fang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genetic differentiation in natural populations is driven by geographic distance and by ecological or physical features within and between natural habitats that reduce migration. The primary population structure in wild barley differentiates populations east and west of the Zagros Mountains. Genetic differentiation between eastern and western populations is uneven across the genome and is greatest on linkage groups 2H and 5H. Genetic markers in these two regions demonstrate the largest difference in frequency between the primary populations and have the highest informativeness for assignment to each population. Previous cytological and genetic studies suggest there are chromosomal structural rearrangements (inversions or translocations) in these genomic regions. Environmental association analyses identified an association with both temperature and precipitation variables on 2H and with precipitation variables on 5H." />
<meta name="DC.Contributor" content="Zhou Fang" />
<meta name="DC.Contributor" content="Ana M. Gonzales" />
<meta name="DC.Contributor" content="Michael T. Clegg" />
<meta name="DC.Contributor" content="Kevin P. Smith" />
<meta name="DC.Contributor" content="Gary J. Muehlbauer" />
<meta name="DC.Contributor" content="Brian J. Steffenson" />
<meta name="DC.Contributor" content="Peter L. Morrell" />
<meta name="article:published_time" content="2014-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Two Genomic Regions Contribute Disproportionately to Geographic Differentiation in Wild Barley" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genetic differentiation in natural populations is driven by geographic distance and by ecological or physical features within and between natural habitats that reduce migration. The primary population structure in wild barley differentiates populations east and west of the Zagros Mountains. Genetic differentiation between eastern and western populations is uneven across the genome and is greatest on linkage groups 2H and 5H. Genetic markers in these two regions demonstrate the largest difference in frequency between the primary populations and have the highest informativeness for assignment to each population. Previous cytological and genetic studies suggest there are chromosomal structural rearrangements (inversions or translocations) in these genomic regions. Environmental association analyses identified an association with both temperature and precipitation variables on 2H and with precipitation variables on 5H.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/07/01" />
<meta name="citation_mjid" content="ggg;4/7/1193" />
<meta name="citation_id" content="4/7/1193" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/7/1193" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/7/1193.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/7/1193.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/7/1193.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.010561" />
<meta name="citation_pmid" content="24760390" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1193" />
<meta name="citation_lastpage" content="1203" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Zhou Fang" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of Minnesota, St. Paul, Minnesota 55108" />
<meta name="citation_author" content="Ana M. Gonzales" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108" />
<meta name="citation_author" content="Michael T. Clegg" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of California, Irvine, California 92697" />
<meta name="citation_author" content="Kevin P. Smith" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108" />
<meta name="citation_author" content="Gary J. Muehlbauer" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of Minnesota, St. Paul, Minnesota 55108" />
<meta name="citation_author" content="Brian J. Steffenson" />
<meta name="citation_author_institution" content="Department of Plant Pathology, University of Minnesota, St. Paul, Minnesota 55108" />
<meta name="citation_author" content="Peter L. Morrell" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108" />
<meta name="citation_author_email" content="pmorrell@umn.edu" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Albrechtsen;citation_author=F. C. Nielsen;citation_author=R. Nielsen;citation_title=Ascertainment biases in SNP chips affect measures of population divergence.;citation_volume=27;citation_year=2010;citation_pmid=20558595;citation_doi=10.1093/molbev/msq148" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=A. H. D. Brown;citation_author=D. Zohary;citation_author=E. Nevo;citation_title=Outcrossing rates and heterozygosity in natural populations of Hordeum spontaneum Koch in Israel.;citation_pages=49-62;citation_volume=41;citation_year=1978;citation_doi=10.1038/hdy.1978.63" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. S. Caldwell;citation_author=J. Russell;citation_author=P. Langridge;citation_author=W. Powell;citation_title=Extreme population-dependent linkage disequilibrium detected in an inbreeding plant species, Hordeum vulgare.;citation_volume=172;citation_year=2006;citation_pmid=16219791;citation_doi=10.1534/genetics.104.038489" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=L. Cattivelli;citation_author=D. Bartels;citation_title=Molecular Cloning and Characterization of Cold-Regulated Genes in Barley;citation_pages=1504-1510;citation_volume=93;citation_year=1990;citation_issue=4;citation_pmid=16667647;citation_doi=10.1104/pp.93.4.1504" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=L. L. Cavalli-Sforza;citation_title=Population Structure and Human Evolution;citation_pages=362-379;citation_volume=164;citation_year=1966;citation_issue=995;citation_pmid=4379525;citation_doi=10.1098/rspb.1966.0038" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=T. J. Close;citation_author=P. R. Bhat;citation_author=S. Lonardi;citation_author=Y. Wu;citation_author=N. Rostoks;citation_title=Development and implementation of high-throughput SNP genotyping in barley.;citation_pages=582-582;citation_volume=10;citation_year=2009;citation_pmid=19961604;citation_doi=10.1186/1471-2164-10-582" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. I. Cohen;citation_author=W. C. Galinat;citation_title=Potential Use of Alien Germplasm for Maize Improvement;citation_pages=1011-1015;citation_volume=24;citation_year=1984;citation_issue=6;citation_doi=10.2135/cropsci1984.0011183X002400060002x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. Coop;citation_author=D. Witonsky;citation_author=A. Di Rienzo;citation_author=J. K. Pritchard;citation_title=Using environmental correlations to identify loci underlying local adaptation.;citation_volume=185;citation_year=2010;citation_pmid=20516501;citation_doi=10.1534/genetics.110.114819" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. M. Costa;citation_author=A. Corey;citation_author=P. M. Hayes;citation_author=C. Jobet;citation_author=A. Kleinhofs;citation_title=Molecular mapping of the Oregon Wolfe Barleys: a phenotypically polymorphic doubled-haploid population.;citation_pages=415-424;citation_volume=103;citation_year=2001;citation_doi=10.1007/s001220100622" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. DePristo;citation_author=E. Banks;citation_author=R. Poplin;citation_author=K. V. Garimella;citation_author=J. R. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=A. Eyre-Walker;citation_author=R. L. Gaut;citation_author=H. Hilton;citation_author=D. L. Feldman;citation_author=B. S. Gaut;citation_title=Investigation of the bottleneck leading to the domestication of maize;citation_pages=4441-4446;citation_volume=95;citation_year=1998;citation_issue=8;citation_pmid=9539756;citation_doi=10.1073/pnas.95.8.4441" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Falush;citation_author=M. Stephens;citation_author=J. K. Pritchard;citation_title=Inference of Population Structure Using Multilocus Genotype Data: Linked Loci and Correlated Allele Frequencies;citation_pages=1567-1587;citation_volume=164;citation_year=2003;citation_issue=4;citation_pmid=12930761" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. Fang;citation_author=T. Pyhäjärvi;citation_author=A. L. Weber;citation_author=R. K. Dawe;citation_author=J. C. Glaubitz;citation_title=Megabase-scale inversion polymorphism in the wild ancestor of maize.;citation_volume=191;citation_year=2012;citation_pmid=22542971;citation_doi=10.1534/genetics.112.138578" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=Z. Fang;citation_author=A. Eule-Nashoba;citation_author=C. Powers;citation_author=T. Y. Kono;citation_author=S. Takuno;citation_title=Comparative analyses identify the contributions of exotic donors to disease resistance in a barley experimental population.;citation_volume=3;citation_year=2013;citation_pmid=24048643;citation_doi=10.1534/g3.113.007294" />
<meta name="citation_reference" content="citation_journal_title=Plant Dis.;citation_author=T. G. Fetch;citation_author=B. J. Steffenson;citation_author=E. Nevo;citation_title=Diversity and sources of multiple disease resistance in Hordeum spontaneum.;citation_pages=1439-1448;citation_volume=87;citation_year=2003;citation_doi=10.1094/PDIS.2003.87.12.1439" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=E. Francia;citation_author=F. Rizza;citation_author=L. Cattivelli;citation_author=A. M. Stanca;citation_author=G. Galiba;citation_title=Two loci on chromosome 5H determine low-temperature tolerance in a 'Nure' (winter) x 'Tremois' (spring) barley map.;citation_pages=670-680;citation_volume=108;citation_year=2004;citation_issue=4;citation_pmid=14576984;citation_doi=10.1007/s00122-003-1468-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. M. Gattepaille;citation_author=M. Jakobsson;citation_title=Combining markers into haplotypes can improve population structure inference.;citation_volume=190;citation_year=2012;citation_pmid=21868606;citation_doi=10.1534/genetics.111.131136" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=S. J. Goodman;citation_title=RST Calc: a collection of computer programs for calculating estimates of genetic differentiation from microsatellite data and determining their significance.;citation_pages=881-885;citation_volume=6;citation_year=1997;citation_doi=10.1046/j.1365-294X.1997.00260.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Notes;citation_author=J. Goudet;citation_title=Hierfstat, a package for R to compute and test hierarchical F‐statistics.;citation_pages=184-186;citation_volume=5;citation_year=2005;citation_doi=10.1111/j.1471-8286.2004.00828.x" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=M. Grossi;citation_author=E. Giorni;citation_author=F. Rizza;citation_author=A. M. Stanca;citation_author=L. Cattivelli;citation_title=Wild and cultivated barleys show differences in the expression pattern of a cold-regulated gene family under different light and temperature conditions.;citation_pages=1061-1069;citation_volume=38;citation_year=1998;citation_issue=6;citation_pmid=9869412;citation_doi=10.1023/A:1006079916917" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=R. J. Haasl;citation_author=B. A. Payseur;citation_title=Multi-locus inference of population structure: a comparison between single nucleotide polymorphisms and microsatellites.;citation_pages=158-171;citation_volume=106;citation_year=2010;citation_pmid=20332809" />
<meta name="citation_reference" content="citation_journal_title=Biol. J. Linn. Soc. Lond.;citation_author=D. R. Harris;citation_title=Vavilov’s concept of centres of origin of cultivated plants: its genesis and its influence on the study of agricultural origins.;citation_pages=7-16;citation_volume=39;citation_year=1990;citation_doi=10.1111/j.1095-8312.1990.tb01608.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Genet. Resour. Newsl.;citation_author=R. J. Hijmans;citation_author=L. Guarino;citation_author=M. Cruz;citation_author=E. Rojas;citation_title=Computer tools for spatial analysis of plant genetic resources data: 1. DIVA-GIS.;citation_pages=15-19;citation_volume=127;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Climatol.;citation_author=R. J. Hijmans;citation_author=S. E. Cameron;citation_author=J. L. Parra;citation_author=P. G. Jones;citation_author=A. Jarvis;citation_title=Very high resolution interpolated climate surfaces for global land areas.;citation_pages=1965-1978;citation_volume=25;citation_year=2005;citation_pmid=19225523;citation_doi=10.1002/joc.1276" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=W. G. Hill;citation_author=A. Robertson;citation_title=Linkage disequilibrium in finite populations.;citation_pages=226-231;citation_volume=38;citation_year=1968;citation_pmid=24442307;citation_doi=10.1007/BF01245622" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=L. Y. Huynh;citation_author=D. L. Maney;citation_author=J. W. Thomas;citation_title=Chromosome-wide linkage disequilibrium caused by an inversion polymorphism in the white-throated sparrow (Zonotrichia albicollis).;citation_pages=537-546;citation_volume=106;citation_year=2011;citation_issue=4;citation_pmid=20571514;citation_doi=10.1038/hdy.2010.85" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=S. S. Jakob;citation_author=D. Rodder;citation_author=J. O. Engler;citation_author=S. Shaaf;citation_author=H. Ozkan;citation_title=Evolutionary history of wild barley (Hordeum vulgare subsp. spontaneum) analyzed using multilocus sequence data and paleodistribution modeling.;citation_volume=6;citation_year=2014;citation_pmid=24586028;citation_doi=10.1093/gbe/evu047" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Jakobsson;citation_author=N. A. Rosenberg;citation_title=CLUMPP: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure.;citation_volume=23;citation_year=2007;citation_pmid=17485429;citation_doi=10.1093/bioinformatics/btm233" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=H. Jones;citation_author=F. J. Leigh;citation_author=I. Mackay;citation_author=M. A. Bower;citation_author=L. M. J. Smith;citation_title=Population-based resequencing reveals that the flowering time adaptation of cultivated barley originated east of the Fertile Crescent.;citation_volume=25;citation_year=2008;citation_pmid=18669581;citation_doi=10.1093/molbev/msn167" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Kirkpatrick;citation_author=N. Barton;citation_title=Chromosome inversions, local adaptation and speciation.;citation_volume=173;citation_year=2006;citation_pmid=16204214;citation_doi=10.1534/genetics.105.047985" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=T. Konishi;citation_author=I. Linde-Laursen;citation_title=Spontaneous chromosomal rearrangements in cultivated and wild barleys.;citation_pages=237-243;citation_volume=75;citation_year=1988;citation_doi=10.1007/BF00303959" />
<meta name="citation_reference" content="citation_journal_title=Mol Ecol Resour;citation_author=T. Y. Kono;citation_author=K. Seth;citation_author=J. A. Poland;citation_author=P. L. Morrell;citation_title=SNPMeta: SNP annotation and SNP metadata collection without a reference genome.;citation_pages=419-425;citation_volume=14;citation_year=2014;citation_pmid=24237904;citation_doi=10.1111/1755-0998.12183" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. C. Lewontin;citation_author=J. Krakauer;citation_title=DISTRIBUTION OF GENE FREQUENCY AS A TEST OF THE THEORY OF THE SELECTIVE NEUTRALITY OF POLYMORPHISMS;citation_pages=175-195;citation_volume=74;citation_year=1973;citation_issue=1;citation_pmid=4711903" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. Z. Li;citation_author=T. G. Sjakste;citation_author=M. S. Röder;citation_author=M. W. Ganal;citation_title=Development and genetic mapping of 127 new microsatellite markers in barley.;citation_pages=1021-1027;citation_volume=107;citation_year=2003;citation_issue=6;citation_pmid=12879255;citation_doi=10.1007/s00122-003-1345-6" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. Z. Lin;citation_author=A. H. Brown;citation_author=M. T. Clegg;citation_title=Heterogeneous geographic patterns of nucleotide sequence diversity between two alcohol dehydrogenase genes in wild barley (Hordeum vulgare subspecies spontaneum);citation_pages=531-536;citation_volume=98;citation_year=2001;citation_issue=2;citation_pmid=11149938;citation_doi=10.1073/pnas.98.2.531" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=Q. Long;citation_author=F. A. Rabanal;citation_author=D. Meng;citation_author=C. D. Huber;citation_author=A. Farlow;citation_title=Massive genomic variation and strong selection in Arabidopsis thaliana lines from Sweden.;citation_pages=884-890;citation_volume=45;citation_year=2013;citation_pmid=23793030;citation_doi=10.1038/ng.2678" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=D. B. Lowry;citation_author=J. H. Willis;citation_title=A widespread chromosomal inversion polymorphism contributes to a major life-history transition, local adaptation, and reproductive isolation.;citation_pages=e1000500;citation_volume=8;citation_year=2010;citation_pmid=20927411;citation_doi=10.1371/journal.pbio.1000500" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=D. E. Matthews;citation_author=V. L. Carollo;citation_author=G. R. Lazo;citation_author=O. D. Anderson;citation_title=GrainGenes, the genome database for small-grain crops;citation_pages=183-186;citation_volume=31;citation_year=2003;citation_issue=1;citation_pmid=12519977;citation_doi=10.1093/nar/gkg058" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. F. Mayer;citation_author=R. Waugh;citation_author=J. W. Brown;citation_author=A. Schulman;citation_author=P. Langridge;citation_title=A physical, genetic and functional sequence assembly of the barley genome.;citation_pages=711-716;citation_volume=491;citation_year=2012;citation_issue=7426;citation_pmid=23075845" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. L. Morrell;citation_author=M. T. Clegg;citation_title=Genetic evidence for a second domestication of barley (Hordeum vulgare) east of the Fertile Crescent.;citation_volume=104;citation_year=2007;citation_pmid=17360640;citation_doi=10.1073/pnas.0611377104" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. L. Morrell;citation_author=K. E. Lundy;citation_author=M. T. Clegg;citation_title=Distinct geographic patterns of genetic diversity are maintained in wild barley (Hordeum vulgare ssp. spontaneum) despite migration.;citation_volume=100;citation_year=2003;citation_pmid=12963820;citation_doi=10.1073/pnas.1633708100" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. L. Morrell;citation_author=D. M. Toleno;citation_author=K. E. Lundy;citation_author=M. T. Clegg;citation_title=Low levels of linkage disequilibrium in wild barley (Hordeum vulgare ssp. spontaneum) despite high rates of self-fertilization.;citation_volume=102;citation_year=2005;citation_pmid=15699350;citation_doi=10.1073/pnas.0409804102" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. L. Morrell;citation_author=D. M. Toleno;citation_author=K. E. Lundy;citation_author=M. T. Clegg;citation_title=Estimating the contribution of mutation, recombination and gene conversion in the generation of haplotypic diversity.;citation_volume=173;citation_year=2006;citation_pmid=16624913;citation_doi=10.1534/genetics.105.054502" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=P. L. Morrell;citation_author=A. M. Gonzales;citation_author=K. K. T. Meyer;citation_author=M. T. Clegg;citation_title=Resequencing data indicates a modest effect of domestication on diversity in barley: a cultigen with multiple origins.;citation_volume=105;citation_year=2014;citation_pmid=24336926;citation_doi=10.1093/jhered/est083" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=M. Muñoz-Amatriaín;citation_author=M. J. Moscou;citation_author=P. R. Bhat;citation_title=An improved consensus linkage map of barley based on flow-sorted chromosomes and single nucleotide polymorphism markers.;citation_pages=238-249;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0023" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Nei;citation_author=T. Maruyama;citation_title=Lewontin-Krakauer test for neutral genes;citation_pages=395-395;citation_volume=80;citation_year=1975;citation_issue=2;citation_pmid=1132690" />
<meta name="citation_reference" content="citation_journal_title=Biol. J. Linn. Soc. Lond.;citation_author=E. Nevo;citation_author=A. Beiles;citation_author=D. Zohary;citation_title=Genetic resources of wild barley in the Near East: structure, evolution and application in breeding.;citation_pages=355-380;citation_volume=27;citation_year=1986;citation_doi=10.1111/j.1095-8312.1986.tb01742.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=R. Nielsen;citation_author=S. Williamson;citation_author=Y. Kim;citation_author=M. J. Hubisz;citation_author=A. G. Clark;citation_title=Genomic scans for selective sweeps using SNP data;citation_pages=1566-1575;citation_volume=15;citation_year=2005;citation_issue=11;citation_pmid=16251466;citation_doi=10.1101/gr.4252305" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=T. Ohta;citation_author=M. Kimura;citation_title=A model of mutation appropriate to estimate the number of electrophoretically detectable alleles in a finite population.;citation_pages=201-204;citation_volume=22;citation_year=1973;citation_issue=2;citation_pmid=4777279;citation_doi=10.1017/S0016672300012994" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=E. Paradis;citation_author=J. Claude;citation_author=K. Strimmer;citation_title=APE: Analyses of Phylogenetics and Evolution in R language;citation_pages=289-290;citation_volume=20;citation_year=2004;citation_issue=2;citation_pmid=14734327;citation_doi=10.1093/bioinformatics/btg412" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of Population Structure Using Multilocus Genotype Data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835412;citation_doi=10.1111/j.1471-8286.2005.01155.x" />
<meta name="citation_reference" content="citation_author=R Development Core Team;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. T. Ramage;citation_author=C. A. Suneson;citation_title=Translocation-Gene Linkages on Barley Chromosome 7;citation_pages=319-320;citation_volume=1;citation_year=1961;citation_issue=5;citation_doi=10.2135/cropsci1961.0011183X000100050005x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. Ramsay;citation_author=M. Macaulay;citation_author=S. Degli Ivanissevich;citation_author=K. MacLean;citation_author=L. Cardle;citation_title=A Simple Sequence Repeat-Based Linkage Map of Barley;citation_pages=1997-2005;citation_volume=156;citation_year=2000;citation_issue=4;citation_pmid=11102390" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=N. A. Rosenberg;citation_author=L. M. Li;citation_author=R. Ward;citation_author=J. K. Pritchard;citation_title=Informativeness of genetic markers for inference of ancestry.;citation_pages=1402-1422;citation_volume=73;citation_year=2003;citation_issue=6;citation_pmid=14631557;citation_doi=10.1086/380416" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Ross-Ibarra;citation_author=P. L. Morrell;citation_author=B. S. Gaut;citation_title=Plant domestication, a unique opportunity to identify the genetic basis of adaptation.;citation_volume=104;citation_year=2007;citation_pmid=17494757;citation_doi=10.1073/pnas.0700643104" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=J. Russell;citation_author=I. K. Dawson;citation_author=A. J. Flavell;citation_author=B. Steffenson;citation_author=E. Weltzien;citation_title=Analysis of &amp;gt;1000 single nucleotide polymorphisms in geographically matched samples of landrace and wild barley indicates secondary contact and chromosome-level differences in diversity around domestication genes.;citation_pages=564-578;citation_volume=191;citation_year=2011;citation_issue=2;citation_pmid=21443695;citation_doi=10.1111/j.1469-8137.2011.03704.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Saisho;citation_author=M. D. Purugganan;citation_title=Molecular phylogeography of domesticated barley traces expansion of agriculture in the Old World.;citation_volume=177;citation_year=2007;citation_pmid=17947416;citation_doi=10.1534/genetics.107.079491" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=K. Sato;citation_author=N. Nankaku;citation_author=K. Takeda;citation_title=A high-density transcript linkage map of barley derived from a single population.;citation_pages=110-117;citation_volume=103;citation_year=2009;citation_issue=2;citation_pmid=19455180;citation_doi=10.1038/hdy.2009.57" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Physiol.;citation_author=K. Sato;citation_author=T. J. Close;citation_author=P. Bhat;citation_author=M. Muñoz-Amatriaín;citation_author=G. J. Muehlbauer;citation_title=Single nucleotide polymorphism mapping and alignment of recombinant chromosome substitution lines in barley.;citation_volume=52;citation_year=2011;citation_pmid=21427110;citation_doi=10.1093/pcp/pcr024" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=P. Scheet;citation_author=M. Stephens;citation_title=A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase.;citation_pages=629-644;citation_volume=78;citation_year=2006;citation_issue=4;citation_pmid=16532393;citation_doi=10.1086/502802" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=I. Schmalenbach;citation_author=J. Léon;citation_author=K. Pillen;citation_title=Identification and verification of QTLs for agronomic traits using wild barley introgression lines.;citation_pages=483-497;citation_volume=118;citation_year=2009;citation_issue=3;citation_pmid=18979081;citation_doi=10.1007/s00122-008-0915-z" />
<meta name="citation_reference" content="citation_journal_title=Bioscience;citation_author=D. J. Schoen;citation_author=A. H. D. Brown;citation_title=The Conservation of Wild Plant Species in Seed Banks.;citation_volume=51;citation_year=2001;citation_doi=10.1641/0006-3568(2001)051[0960:TCOWPS]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=J. H. Shin;citation_author=S. Blay;citation_author=B. McNeney;citation_author=J. Graham;citation_title=LDheatmap: an R function for graphical display of pairwise linkage disequilibria between single nucleotide polymorphisms.;citation_pages=1-10;citation_volume=16;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=D. Shriner;citation_title=Investigating population stratification and admixture using eigenanalysis of dense genotypes.;citation_pages=413-420;citation_volume=107;citation_year=2011;citation_issue=5;citation_pmid=21448230" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Slatkin;citation_title=A measure of population subdivision based on microsatellite allele frequencies.;citation_pages=457-462;citation_volume=139;citation_year=1995;citation_issue=1;citation_pmid=7705646" />
<meta name="citation_reference" content="citation_journal_title=Aust. J. Agric. Res.;citation_author=B. J. Steffenson;citation_author=P. Olivera;citation_author=J. K. Roy;citation_author=Y. Jin;citation_author=K. P. Smith;citation_title=A walk on the wild side: mining wild wheat and barley collections for rust resistance genes.;citation_pages=532-544;citation_volume=58;citation_year=2007;citation_doi=10.1071/AR07123" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=N. Stein;citation_author=M. Prasad;citation_author=U. Scholz;citation_author=T. Thiel;citation_author=H. Zhang;citation_title=A 1,000-loci transcript map of the barley genome: new anchoring points for integrative grass genomics.;citation_pages=823-839;citation_volume=114;citation_year=2007;citation_issue=5;citation_pmid=17219208;citation_doi=10.1007/s00122-006-0480-2" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Z. A. Szpiech;citation_author=M. Jakobsson;citation_author=N. A. Rosenberg;citation_title=ADZE: a rarefaction approach for counting alleles private to combinations of populations.;citation_volume=24;citation_year=2008;citation_pmid=18779233;citation_doi=10.1093/bioinformatics/btn478" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=P. Szűcs;citation_author=V. C. Blake;citation_author=P. R. Bhat;citation_author=S. Chao;citation_author=T. J. Close;citation_title=An integrated resource for barley linkage map and malting quality QTL alignment.;citation_pages=134-140;citation_volume=2;citation_year=2009;citation_doi=10.3835/plantgenome2008.01.0005" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=S. Takeda;citation_author=M. Matsuoka;citation_title=Genetic approaches to crop improvement: responding to environmental and population changes.;citation_pages=444-457;citation_volume=9;citation_year=2008;citation_issue=6;citation_pmid=18475268;citation_doi=10.1038/nrg2342" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. W. Thomas;citation_author=M. Cáceres;citation_author=J. J. Lowman;citation_author=C. B. Morehouse;citation_author=M. E. Short;citation_title=The chromosomal polymorphism linked to variation in social behavior in the white-throated sparrow (Zonotrichia albicollis) is a complex rearrangement and suppressor of recombination.;citation_volume=179;citation_year=2008;citation_pmid=18562641;citation_doi=10.1534/genetics.108.088229" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. Thornton;citation_title=libsequence: a C++ class library for evolutionary genetic analysis;citation_pages=2325-2327;citation_volume=19;citation_year=2003;citation_issue=17;citation_pmid=14630667;citation_doi=10.1093/bioinformatics/btg316" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. M. Valdes;citation_author=M. Slatkin;citation_author=N. B. Freimer;citation_title=Allele frequencies at microsatellite loci: the stepwise mutation model revisited.;citation_pages=737-749;citation_volume=133;citation_year=1993;citation_issue=3;citation_pmid=8454213" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. K. Varshney;citation_author=T. C. Marcel;citation_author=L. Ramsay;citation_author=J. Russell;citation_author=M. S. Röder;citation_title=A high density barley microsatellite consensus map with 775 SSR loci.;citation_pages=1091-1103;citation_volume=114;citation_year=2007;citation_issue=6;citation_pmid=17345060;citation_doi=10.1007/s00122-007-0503-7" />
<meta name="citation_reference" content="citation_journal_title=Hereditas;citation_journal_abbrev=Hereditas;citation_author=S. Volis;citation_author=S. Mendlinger;citation_author=N. Orlovsky;citation_title=Variability in phenotypic traits in core and peripheral populations of wild barley Hordeum spontaneum Koch.;citation_pages=235-247;citation_volume=133;citation_year=2000;citation_issue=3;citation_pmid=11433968" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=S. Volis;citation_author=S. Mendlinger;citation_author=Y. Turuspekov;citation_author=U. Esnazarov;citation_title=Phenotypic and allozyme variation in Mediterranean and desert populations of wild barley, Hordeum spontaneum Koch.;citation_pages=1403-1415;citation_volume=56;citation_year=2002;citation_issue=7;citation_pmid=12206241;citation_doi=10.1554/0014-3820(2002)056[1403:PAAVIM]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Biol. J. Linn. Soc. Lond.;citation_author=S. Volis;citation_author=S. Mendlinger;citation_author=D. Ward;citation_title=Differentiation in populations of Hordeum spontaneum Koch along a gradient of environmental productivity and predictability: plasticity in response to water and nutrient stress.;citation_pages=301-312;citation_volume=75;citation_year=2002;citation_doi=10.1111/j.1095-8312.2002.tb02071.x" />
<meta name="citation_reference" content="citation_journal_title=Oecologia;citation_author=S. Volis;citation_author=S. Mendlinger;citation_author=D. Ward;citation_title=Adaptive traits of wild barley plants of Mediterranean and desert origin.;citation_pages=131-138;citation_volume=133;citation_year=2002;citation_doi=10.1007/s00442-002-0999-0" />
<meta name="citation_reference" content="citation_journal_title=Journal of evolutionary biology;citation_journal_abbrev=J Evol Biol;citation_author=S. Volis;citation_author=K. J. Verhoeven;citation_author=S. Mendlinger;citation_author=D. Ward;citation_title=Phenotypic selection and regulation of reproduction in different environments in wild barley.;citation_pages=1121-1131;citation_volume=17;citation_year=2004;citation_issue=5;citation_pmid=15312084;citation_doi=10.1111/j.1420-9101.2004.00738.x" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. von Korff;citation_author=H. Wang;citation_author=J. Leon;citation_author=K. Pillen;citation_title=AB-QTL analysis in spring barley. I. Detection of resistance genes against powdery mildew, leaf rust and scald introgressed from wild barley.;citation_pages=583-590;citation_volume=111;citation_year=2005;citation_issue=3;citation_pmid=15902395;citation_doi=10.1007/s00122-005-2049-x" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. von Korff;citation_author=H. Wang;citation_author=J. Leon;citation_author=K. Pillen;citation_title=AB-QTL analysis in spring barley: II. Detection of favourable exotic alleles for agronomic traits introgressed from wild barley (H. vulgare ssp. spontaneum).;citation_pages=1221-1231;citation_volume=112;citation_year=2006;citation_issue=7;citation_pmid=16477429;citation_doi=10.1007/s00122-006-0223-4" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=M. von Korff;citation_author=H. Wang;citation_author=J. Leon;citation_author=K. Pillen;citation_title=AB-QTL analysis in spring barley: III. Identification of exotic alleles for the improvement of malting quality in spring barley (H. vulgare ssp spontaneum).;citation_pages=81-93;citation_volume=21;citation_year=2008;citation_doi=10.1007/s11032-007-9110-1" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=C. Wang;citation_author=S. Zöllner;citation_author=N. A. Rosenberg;citation_title=A quantitative comparison of the similarity between genes and geography in worldwide human populations.;citation_pages=e1002886-e1002886;citation_volume=8;citation_year=2012;citation_issue=8;citation_pmid=22927824;citation_doi=10.1371/journal.pgen.1002886" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=B. S. Weir;citation_author=C. C. Cockerham;citation_title=Estimating F-statistics for the analysis of population structure.;citation_pages=1358-1370;citation_volume=38;citation_year=1984;citation_doi=10.2307/2408641" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. I. Wright;citation_author=I. V. Bi;citation_author=S. G. Schroeder;citation_author=M. Yamasaki;citation_author=J. F. Doebley;citation_title=The Effects of Artificial Selection on the Maize Genome;citation_pages=1310-1314;citation_volume=308;citation_year=2005;citation_issue=5726;citation_pmid=15919994;citation_doi=10.1126/science.1107891" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. H. Wright;citation_author=C. W. Tung;citation_author=K. Zhao;citation_author=A. Reynolds;citation_author=S. R. McCouch;citation_title=ALCHEMY: a reliable method for automated SNP genotype calling for small batch sizes and highly homozygous populations.;citation_volume=26;citation_year=2010;citation_pmid=20926420;citation_doi=10.1093/bioinformatics/btq533" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=R. C. Yang;citation_title=Estimating hierarchical F-statistics.;citation_pages=950-956;citation_volume=58;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=W. Y. Yang;citation_author=J. Novembre;citation_author=E. Eskin;citation_author=E. Halperin;citation_title=A model-based approach for analysis of spatial structure in genetic data.;citation_pages=725-731;citation_volume=44;citation_year=2012;citation_issue=6;citation_pmid=22610118;citation_doi=10.1038/ng.2285" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=S. J. Yun;citation_author=L. Gyenis;citation_author=E. Bossolini;citation_author=P. M. Hayes;citation_author=I. Matus;citation_title=Validation of Quantitative Trait Loci for Multiple Disease Resistance in Barley Using Advanced Backcross Lines Developed with a Wild Barley;citation_pages=1179-1186;citation_volume=46;citation_year=2006;citation_issue=3;citation_doi=10.2135/cropsci2005.08-0293" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=S. J. Yun;citation_author=L. Gyenis;citation_author=P. M. Hayes;citation_author=I. Matus;citation_author=K. P. Smith;citation_title=Quantitative Trait Loci for Multiple Disease Resistance in Wild Barley;citation_pages=2563-2572;citation_volume=45;citation_year=2005;citation_issue=6;citation_doi=10.2135/cropsci2005.0236" />
<meta name="citation_reference" content="citation_title=Domestication of plants in the Old World: the origin and spread of cultivated plants in West Asia, Europe, and the Nile Valley;citation_author=D. Zohary;citation_author=M. Hopf;citation_year=2000" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Two Genomic Regions Contribute Disproportionately to Geographic Differentiation in Wild Barley" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/7.cover-source.jpg" />
<meta name="twitter:description" content="Genetic differentiation in natural populations is driven by geographic distance and by ecological or physical features within and between natural habitats that reduce migration. The primary population structure in wild barley differentiates populations east and west of the Zagros Mountains. Genetic differentiation between eastern and western populations is uneven across the genome and is greatest on linkage groups 2H and 5H. Genetic markers in these two regions demonstrate the largest difference in frequency between the primary populations and have the highest informativeness for assignment to each population. Previous cytological and genetic studies suggest there are chromosomal structural rearrangements (inversions or translocations) in these genomic regions. Environmental association analyses identified an association with both temperature and precipitation variables on 2H and with precipitation variables on 5H." />
<meta name="og-title" property="og:title" content="Two Genomic Regions Contribute Disproportionately to Geographic Differentiation in Wild Barley" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/7/1193" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genetic differentiation in natural populations is driven by geographic distance and by ecological or physical features within and between natural habitats that reduce migration. The primary population structure in wild barley differentiates populations east and west of the Zagros Mountains. Genetic differentiation between eastern and western populations is uneven across the genome and is greatest on linkage groups 2H and 5H. Genetic markers in these two regions demonstrate the largest difference in frequency between the primary populations and have the highest informativeness for assignment to each population. Previous cytological and genetic studies suggest there are chromosomal structural rearrangements (inversions or translocations) in these genomic regions. Environmental association analyses identified an association with both temperature and precipitation variables on 2H and with precipitation variables on 5H." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/7.cover-source.jpg" />
    <title>Two Genomic Regions Contribute Disproportionately to Geographic Differentiation in Wild Barley | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Wlku7AG9uvsQwSxCVVG-4ZR9MsjRMbmnS13SoTK5Fvc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459925","apath":"\/ggg\/4\/7\/1193.atom","pisa":"ggg;4\/7\/1193","ac":{"\/ggg\/4\/7\/1193.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/7\/1193.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/7\/1193"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/7\/1193"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/7\/1193.abstract":true,"\/content\/4\/7\/1193":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CFVSjhrdTvwEi9EO+mPWe6OknTWjIgF0ldU7hsfUKJTUjGVLI3Juz2Xt5yiRR93yIDh0p2izXeNWYQ7TQWtV7PKWqV1rQc6zAZf3X+HXPZrb92FB0FGCaSvuUE2lRQWLyjGqtsWtKCQhqroF56O4rEtmduLTZQ0qOELc7b7iwjHDduOfls9494fO6+L0U8aPmML+3EOav5Pe2HNcFL9DXvxdami6d4HOntrs15BK5651YuIAEoBtxj3NzjKtAYr7mDbxbmFs4SCYHNHH4tvyPsvrw2ZTQ5wq9VrwSEMNFKaDiyvJSG+6tTgKzpzPQ0WeXulPqduJlxyIYk96H58o2hyh24SORV5ejX7U07AoPxpDzsleKAl6LngJSCmTAS80IsAdjsr0Sk943X6d5VcT0Ub2Bmn\\\\\\\/N8VSU1ZCTapU72\\\\\\\/nut\\\\\\\/+Xdj350riqiO3uu8IBgS5WTe8duO7DP6ZaUJDaHi3lISWdgnnN\\\\\\\/X9KA4gjRCC5srbdVzCHmlahUIJ09QlBZ\\\\\\\/5SZ4Ajopagfur3wol7WSOHqEYsgYaIRaw4qC3a1qUZ\\\\\\\/WXjXfIyxW6j3Zk0YitbMaE42aRMOuByexjPQFVEAUImupCntxnqsQCMvFPRSE1Vb4gv7rLxjmorOdHmtDNrmu3QsRX0QmkNCJqM8zdCpXk8KywTO0sHC\\\\\\\/aW6mbHhGvL4BPDhiA\\\\\\\/EMtL4ODMPUqy3sJikiQYInuaWAB7vwGTcj+E8wS96RY0Efi3Wrc4pU9fHNCrrhji147rYEM1MJCcNZ9+MzHwMh\\\\\\\/dl+lrc0s+pJ4v+HEzt5n8jwVC3YfeVrrk5wDS8SgdCmM4Svz714FIkgSZappVo0yXmNuMdPMi4WtD4dX5Ac13SNwPvYmuhMsuHmLTyLYXTzoR3lmxEyWedCsBo5Ams9blOqkY+rHV3AKUAjbcBS9HmliJu5Y8QturKQHPBSvn82ucIi+pGT38HCpM25Uhx6ewEvjD4xU7Ck6jYMpNNM83l9xSzrJJgCxaL6KsgVEyUJmjxXsaf6+K5HigeH62rJTMz7r9pIBnWx4qfim4NODgWGkrq\\\\\\\/hsNKZRYksau\\\\\\\/Yte9Jq+B0yPj5J+aGlauuVT+d5cRaOuzzHziXNzOixKD\\\\\\\/LNus1tQYOP2ke9uTMIbAy5Vzn7Mi34mOt5MoMjvYv63DOVbAhWYYqpuevJ1Wqs7xIdFbDxjgYSzQJkwPz8nRv0TBo8qxXfpWF2VTJiHSwCjPEFEHCBpTl56qZpUBa8S8N22cz\\\\\\\/8UOOcBRxHDU\\\\\\\/uKO\\\\\\\/yjrtyckg3j\\\u0022,\\\u0022iv\\\u0022:\\\u0022e4ZF2GQsc96BMZhbCmEo6Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002275df67f0f06d7c65fcc34d36de8ce6a296c603ace85942ae2a9a68a68905fe09\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022FgizvWkDXZIUnE+6stgDI4VgOEq\\\\\\\/S1jgzyDTnfJ8oNqvd2LwcZ+KhUvDQGQn3kR3LUGZONvGJI33qzzRmcaNNfjVP\\\\\\\/G8LMUePqOTR4cYNnrnoy0syM9wFD3Zvbi7TvlsFlvxqX0gZ\\\\\\\/JObVDSc5tJAi7VhOeymPAOSmy2qfpnQ7i0sxnCilRjbdRJ5T8LL7o\\\\\\\/8rpin4JRCI2yhMCONIX2boKylWjtuO+vWezox8gjUuT8h+HsiK\\\\\\\/rGZZjF\\\\\\\/wo9oqJQwE9r1iGXHkaN816co6aDihrzVEnGgDke3KNiBSNw55o2M3Jh5\\\\\\\/yxm3WPayDF\\\\\\\/hANMPfDoZLNJBvTzQxJxjSBXf2Uk3Oq6EB9Alg8Un0d+eNZMfKdTRUvm0VWSN21ATBq2lIdu0\\\\\\\/ng+BHGjChdKz+txEfbXfnG1moi1kzmbEyuDluAiV7TgYhGwYxbPl\\\\\\\/sk2GYstqJkS2M90+n1uvzorgBmCrmRX2EF3KfhOAvhDwP+qMEHEZqYGhTQC7cRyhzYR0s09+ZRHnKoPS4d6YJxoDNcEj9rfFzVFzou58sbq3VHM16kUaURzI21cULpgj156L6UhCXk3xBuJk1GuyBRDfXa+D1dP3BYXvZsJH9fHzvrI1xO+V\\\\\\\/VmQk2RnM0DhoW2AlQPg9ULwHqjyP4RcsZstqvvMUoky7Iqf4Vr+e1quF37H0JUgGjZKjes\\\\\\\/LosbiHi8xZ2XuOpkpUnVhVKMAogvMrZV5HWPdPXpOHjDa6tgTKmYvxiG7T8SDVelBz0HbdUJbUMYlgi6gGbthjT8R\\\\\\\/VdLDOm+kIXCig5HL8R+9CkKu4e8gks2IoCNmoY3Vq8KMB7x3RygLZuf6jX9nEoFSRirvsGzCId0V5f+stLx7PGN0km4b9zTFYw8P52+5W3KsFOOiuMIVtHkGN+WDRkKPZLlmF+prLxGWWIocMJ2CvH+5HyhaEHQwREx4+Ji\\\\\\\/8luUhG0uo8QgusWcUgZoWZVsHqbaywyNmITLFl7bDl\\\\\\\/7Qfo9ALwGdvMDFVUjWZhBvDFce43MrZQP5g9gMsKjR3HLvD5JlhI2wsMN3GzIBSKPxn2geDo8\\\\\\\/BCb1V+p0twy16moecE5C\\\\\\\/NhWtzHAonmXDmoFaiNWXA2Jh5DKBjHqgZnZ3z9nIkqqYB2KZxWuZ3F7v2Tjy3vI5nKEfcJ0rxHC+VrXg7JUnnQz17s18q3JZmeU+AvjSXUris3Y72MO7Xu6h3JZ13fd9BCKQWaszdP7y3SXCY5MXB2nlfCi0jmBowAAW8KXtajn\\\\\\\/vCmDQ4jxQKuKQFLQLWl5vwZ99X8XPoKCrB7SkABY5yn7N6SsMVYgSGIoHIUB7zWZHnVvLbsGBv1KrP5BBj\\\\\\\/nn9f5pqZWh6ERu8inc5CEVyq4Jf3\\\\\\\/pvGpxKhbiVatDb2lPUGfkTAFtDYWQjmHyUfVHLW80wBTlZTyayZEIpeLfKbvhRZ1KvjM3IOy7MhLd+mDYsRQQGREJrZ7vF7sWMFtHoG70qYelEsH9tQphylIKH\\\\\\\/8hPx733dGtHjvnxgNrZCtouVbsnOdXV2QnDj4jHYpE1N\\\\\\\/hKI2YByCcQI4I0r243uX6rC\\\\\\\/\\\\\\\/P0VXTHvgwenR\\\\\\\/cbyOlZwtYTWOxnsc0zI8OWbqJH6By7hIDmxoGCgvZJKOG\\\\\\\/3eEAwjeqGoYi9uqnm3Ytz0oEDZc9cWqXrK1LUhkn3oG1zY\\\\\\\/RVQDgXn10IlcCAObz3N9YqsSSD0CGpjLNmIhfKgrJEDWSyjLCVnyeYOkeAUK2Cr7s05bmI6iW6YWlkzK8O144FY25hbCGv\\\\\\\/RE\\\\\\\/jk6Asf7luapbb33d4KPUBn5oq5ryj3sNjMLZnt2p8p1jH8yO+9q\\\\\\\/GX\\\\\\\/w0GVOqM1Rt3WOjJ1seOUVgNJK2iW8YSK39W\\\\\\\/ae3uVyyb4weAjXYQIuib7MamS7D9uwAhe0Mza95I6g1OzNSjhaUFfWMy24Bv\\\\\\\/E4JqfwU8PoPNXHtmYolg+x2cSAS+VtNw2zj4MOx+AsOde+hceOmg7UF12KDyAdFh8Mxrot+tBshee43yoYTXfDvIqILuBZouxGXqMyAONPACpE2Ow4Zt9OhUuMYmTSWLP8PC2ZFZt2fZvzwdLUES0NUyA2\\\\\\\/QfeOJHjcHgiC+gbA6P39KlDzq51oT492TxJWNTuPJL\\\\\\\/rpbytV1a0qnbHwrhDdK+8f0CgAS0WCB99XibA3xROfqMAutHqw1wDF9ldgZTuVWWSPqoOxUqp6vRmcJaXuoa+CtDWYYl8WkvHxrDIpsjUJLpBUI\\\\\\\/roUWabkuV73vtpWuxs\\\\\\\/7Ek6KI2CRcMbZ3wcHHHfAn1VPmbO1QD2mPlsnk\\\\\\\/lfhprfxxSxT1Sl0XrCXFSCoGkOf1VHYr8FV0rLFE2gkL+lnMi1HTNsiw8pcEovJ5CKkcFFemMG1r+9QX1\\\\\\\/iunVuLBwAMABr24gpU7dSIt4nxB28RJZTI8zsjCMnJRIzU9TYmtupWrwOrLlAZKK1+126GU02LlWibPXs53UU8IJeS4KlQsioDI0L2jhYhxeV9jI0YfByFtKp58MEyBEOgD6to7bpV8fozugmnFEPIrTfe+PID8b0qS4MzTwrF4Ynb0a4RtJmLi8hqq7ev\\\\\\\/4JcVTStDLPVy1MW+LeWAroNCkK+VsHqY9cyvejoq6QZKTLw\\\\\\\/PS0k33NjJzBlASdwLKjEFIF+YjK71oj9Xq7CPadn49WVKq+MR7CAYbcFrgcOJm9Ta0ZUrn2\\\\\\\/P5weNB38H\\\\\\\/xFTB6EubSNlEupbZxPWXsZU6TCDFxf9NAuCRC9V5N5\\\\\\\/hkQP1cb0G61qF8amD0vITmSW3hCOpfyvaBHc1OYnsTiqA1qqkJeEQrHDjMl3VVHE5zvcAuN34\\\\\\\/Pz63I9Q87mouldlH1cvgSBlHtFuOBRjg5CskNWMQghpojFwAJqpZiYx8pvGgPMmH5PPtoCJlDGSQynPYCMsLSn2JLA44BtN6jPWcnpKlFHclCZymZZV4tZF13tSkMnzf9h2\\\\\\\/TMej8SUFy5zJOuh0BmXb1KKDoiDXLp1QPGqH7rNToRspFt\\\\\\\/JD1X8AQV7muAl1F6G0UPHpi1QgwVh\\\\\\\/0dBBtsZ0eS3MWTeN+0e3f2B2EevnVJcJMlqcTVM6FTVZMBqqRiW9tvYixWu\\\\\\\/aWFcoFZWD4Xq4BUuiPT5Luzf+aiNqdlHWfA\\\\\\\/G4vLJnyjOQ3Ny7v\\\\\\\/GxOp1kGYk8ti4DYhWG+hcWTtxVtrL5tGrfnXlca7\\\\\\\/FM\\\\\\\/s431uEoJ3fbGDLQkSXDtjgGfsL94GMg1SWxvMAkrdnCDGLyoU4IsI9QJdXbZx0k5UEa9GJrAE31Ei1JuPBRzAuKBO8Y9S+4dU2yaLm9WX0Zy7qCyyeZm8w2E1O1DPw\\\\\\\/5HJkg1mbmznPYk2iG9oWx94ZCd8KFY3IjlXcjwcQqaIc80G\\\\\\\/Cif4H0foaK+MoVXWRKIHEruJ1OXnpz0Jl7ZxhmKIqVzE8JUudWzHCiC0gloqxz6TJGwZXEe1+GRQ\\\\\\\/n+nS4RKR8fM\\\\\\\/aer+4YOVHOGOh3DVjmvcAiM3PjEhGtZH5CJIumjGd3pQ2LlGDtcX20H1KEaPKmd\\\\\\\/k9LERzqRm8DfILN+EP7dihUhENiG\\\\\\\/gkXu+N2aoU4KFvgwSYyHmu9Hxr8cyW+F+EeKq80swXYWXs6o+JO\\\\\\\/aL8IufDmqz\\\\\\\/C6hhYzWELonCpzVjGxBEsHO0r28u4nG6M9iZNwziU\\\\\\\/NVaQDzMczNbBV7ILbjU8FzNfNNWKBuEZ29pQsyzi84W0ZzRzdxVtwYd5CP+72z90MlKY\\\\\\\/S5gHHLMZylHdMWTznkdo2UEMDUGIRh9SiS6I1+3hs\\\\\\\/fO\\\\\\\/ky6WJg45GZqU\\\\\\\/gjeZbyaFBGRXgRT+kcg+v8HS+\\\\\\\/IrQS8BYEpaMpiZe3AExqHFmm+840X6LDhcMFrLYQszNcpAF98nJuUdG4L\\\\\\\/VhmM6zR9HuIvzuXLglafjYBav+fAU9jR6xKGzWms6uA3iNP9TT\\\\\\\/UpQsHDd9ZCpiug3Yf+ySPJCb6dokHe3E\\\\\\\/EsV7uOqI6iAzjdiGUqkQvqLc9LSaa1b3hptsfL6SpTv+0sCoBHF5w7n8W6PtBg59X17jgxs1lmv2D0JJMbgYDd6WqOFEU2WACvUq2OYpLxx31tTGR6bGGcRxHpIp5hGtbMp3t2Hr6b1\\\\\\\/FL7xB0II2jEnj6JaK6FUxC9LQBvhKLRMRgkv7A+7NTnnRSSuugym+JGRxtY8xkklDy8+JXnhZ89nWCs6z4Ay66l+LUMfI3w791R2pDaNmtngSfPr0gqoiARnlUezt+PBQJg6xuZl4nNlcq5\\\\\\\/MYCtOpeSahWBgseLxqCTOyVTHn8GALPFIZPv0s0B3x8rPq4yjrHtKJjvoo2aTbc\\\\\\\/gGjApHywwn+5055vehy7jmYFS\\\\\\\/qT4dd\\\\\\\/zKzFATKGY5D3gH6s2LdGYWzz9vsxfZATNt77PnnJGFTumNtZOTnzmwsvf9eVs83b8XyrqDmiojiG936vNSb6SElAEKMLyGJrseMPPsrWCgL2qJl75b9KTvqblo8\\\\\\\/tkOpMY7M7ac2oldWUCcflvL1OgDCY\\\\\\\/vyWUQCRh0oxcdpjecxLYtOHVdtp51C1rsPhGchti+qaAe6M+vcxDGHexqiNaVib7nSb8nHn60tCR\\\\\\\/DuYHbvhaFAZFjpVsmynZjy9mj2pG8BcJaXxj+9MUfHFCJV2huRfCeL6BciZxB0Q2NyahxebFKUou0P2+I6Rpa24fvqUyEmwsE8d7SiWT3BOkb\\\\\\\/1JUoOfJQ5BympRCXEZo7yiAUnll1v8UIVnofoL9BPxa2vkWGqBPMQqi2z04rgA011bVGRRh0qnHaL2nM349S84rXPKgbX\\\\\\\/yBXqqHjzHj4PAdZK0QweeJ4W403oi+fIxSff+yNqOqvJ\\\\\\\/YlPKNpiUjbWben7kNdVlDnCvQdMre2dMizgdpBxgznjHE+nX3saeA3QdDUbhZIC8n8lOEB15k7OO21NrQkTna8nBNHqCDPjWmDrVYbGp+f5EuKdBbSbWGzIB7rKZ\\\\\\\/QZXCqjUfakG9ow98Zpch\\\\\\\/tz\\\\\\\/5CqHuy7T1EBccT8MomdicGXYgCV+543VeaM8m5Z3JcBlRWOwVvQKE33vBmqw710\\\\\\\/EqbPpRMziqNIFi6f6W52u65CoTAW82BkSYf6fLrAdlakqzO29o7kQog50eNmBFM1EJwHaLn3wnaouKMy0WDlLaiXFvU5ZBt6TiJW6AElE3IYYNVBAIcsGygjPUCFJmFd5rut2YSgOvxr\\\\\\\/UK8GAo8N5WPX5JB+L1P2+iL5XtNhznAEE0jfK6M\\\\\\\/P\\\\\\\/vNykyIfVeSTFp0kDa9LU6cSGbCsSydtvWSe2b5yrHhPbKJjiJHnaeGV5FcT6M8HX2MD+f+91QM8EO6Tu6ydsc2XfSWiVQVxIf2bRYFa6Ijujj3PyI9XIqHvnipLSZxxzZrAM0zLCiw3wXffO27JWPsNcwKQTJINCTv8QIBIPJAhYYbg3nWOxekGGnH+pr0agADPOYcUvyyIBPnw0Ql3yPXsvPqx8iddO9syxdcFCj0a3vDr4WuAZI4RupHUkADDFs3GYdrwhQD\\\\\\\/WY\\\\\\\/jgLDLvMXxLZJFjX71QuqHJwpvzlSehEm6XtqFEzs\\\\\\\/HqsGtwBS5kd589I\\\\\\\/AamKtqPGdLsTSRAPbbN1xpbuzmXFuFF+Ky55WcTCy1+01HU6e2hpofiISf3SusUN3CIIIHfcb1ZE\\\\\\\/kR\\\\\\\/d1Yjm4sPjz\\\\\\\/\\\\\\\/65Am9wUf+65aB+q++d2DDEs2X33zcqLFViTBNZtk+EjHYC6M2cTRSpTcKByUS+gM5dpjF3HtdjkK3ShDMeSzJPg6TYxoJR+dK78lyFgvMjqBeqDqze3ztniBzzJre5YkdvNf0\\\\\\\/VNoVfqe6LQ3FwTUbb+3TVNj5Y7KliUKuGlkRtjeeBZYLxKjZYhbkv6kMuzK36znBkMjhoaRyI8AORBppSVSY4OfQlHGn0tLuJhQVCLPJy2SpccR4b\\\\\\\/9zYEAqq5HHvmtC3\\\\\\\/4+qK5S660z8KgZb37nR4jWPCYoiX7E+SLMawXcbxTpKLHF4RGyyNNb2nbn1uT2kOxRLD0vMZdMm7i6IOHt4Dlj4761DJX2tXQVKm03wFEHy204F+Txjli4AbVbd5CUZ2KmmgR9YIOE6vN5f644zL8HVRdfoPjIgR4dFJPNNMfojj0FjjlYqTOpHHfMJaE+rHU2NuazgCNFQu0hPIWeyego8B8rk8SFkfyqRfZ5JSw1krZdJDm7BC8NTzt6m81r+QBhtpjznM4QvYVCbcwBraiOQH8FWKtjvT6CGhVzClvhVS2FSEpSpb\\\\\\\/GfmjuSb\\\\\\\/QxxqaaifTFHLJ+QyapT0caQ9XEjbDroM4VZbureTqnFh9xaKQGHAQA94W3vl2rIfxXNJ2WY7xcciUiM09\\\\\\\/aGyiezp1bZepzLQ5LRJV3TADDxoUoDB5ZXkaXW0jFmixT+eLZkKpcr6l2v0LaacD36mslqdusyD4f+cVI6VKO7CwElRRFPF7xlVgcdOLdC1AFf86lUeTOnCYo5as+BCsIeO\\\\\\\/cgmMUGD\\\\\\\/JU9ZYP7a+L1hVxKCM+v3z0rTVPUiDyibtqUV1V8q+HTOy4FabginDstDBostFcLa5jQT2SUG\\\\\\\/Cg0rzuGHQvhNlCy4niKpWHMmAbKEuzNxweGmv6f6ZJxAzPD3Go1ljjV+ouIYs0ze41FdMs6GLYUTy6r6ivM846izjdioVNWrZhyIXz7+zGSR\\\\\\\/H2dY+K2yv0g5fGw778UxmdGwKQjCH0ydvIzdKHlNHEyIRlegYjTw4A23q9e52CgjQiSzn4whm\\\\\\\/TyR5qpKjwYBonWyMPdxj71\\\\\\\/4R1T7BJRtiWXtddC8NrlZD3PMDpxedXISTxTsBa4gFS2gx6y0YOhpKrtw2ceeYWJ1qCuEFPcuoX1suO+52+DcFqlTYFM7GMgKL4NKbKL1P2aQyy5TKg9dsnafsLBC4T8olCxck\\\\\\\/DehkIA0ifvlOX93vAd9A5j8GIdGKVZjJF1neIAxU6zw8YtieWU7buKsOoeU9xnb\\\\\\\/pIg3J68hLKjM\\\\\\\/IjIgcrPkBDM9Mi4kU7crKIE8DXZHxOplXwEZKkNMjx3PeV5YeZNOq7BMM45Iib8hWEn\\\\\\\/SS\\\\\\\/1VzdbgZBqhIocMqlJRQ+aY6JYBOjk8pVH0XA0Zm5uwJBSqvGRGVUQIUwfRsA8Yx3wSANP1YyptMI+9IIUvlpQyS6NaNPOJG7k0Lu20rrGOiHVpuOpW61KFCCcxbziI6hoRmSsHCCJjT804r0BE3INlCDOufPf+mNJeIZCuFXY+kq0B5TuSdWksnq8h\\\\\\\/P7\\\\\\\/aOFZTl6EeS8YffW5vEPmLbDBVeCUPztoo4KL+rFqlPN3ovDTqhaFPMmdRXS2D3eNn+JjnJySF4zEyK3DO6PYSEqYnfhLDlSjcZcZYuC0D8\\\\\\\/ZrFdVRqD56y8QoVmsJxHZklyt55gPw49+6zV6G49fXuhZiE4Gu6ZzQnKrJjCQJGQ+os1TViyTedLPDL4RFOZfEMfTnvpjqrT9eiQ4tKmDezyDkAvuhqFXoznbfTkMnuAXZML4OnzSTQrXe4oYmnCVYDSKgHGilQ3ZJg\\\\\\\/BQdAu1VYw+mlln6vHz6ki+66Io1DdKsb2+anHvCikslD6yAVhVOpx7WICjxkJacWz3M8Hws0ByPY94zjd7+d6cetuMjF6xTPF1cfTdTKTHNw53C6xWAc\\\\\\\/awYN\\\\\\\/79gk9bFriPTVwC9Zo5jB2rXw17fiB9Tz3oVKrX+4fFbzbNMby3qZ\\\\\\\/7TJECpfcJMmk1p7qyUhsAwemEAav6XnuAhOjl14ozjzLs7X5MbXsmS0FfGPGlRFgN7qRO\\\\\\\/CDeYaMEXbQ3dSZEPNj344Kh0dQUIPY7\\\\\\\/6YptBxRfiwYTiFxqczp8wv70OZOzSBF2EMBAcjquzHMThr4VlcKGD6xRM5n5h5T5CX0rWmWTdn5Rq8rFSgOHq1Fed0VoBVl4mCctnY1P88L1++3KTtDr0Mp8+FE2mk\\\\\\\/\\\\\\\/Ag\\\\\\\/4GNV5lZrNE36Avog8OTwwErJiGTCKyFHzeSA\\\\\\\/EhX8PVn2bjSQYMV2BRKKOWJgO8pDKV1Q14IpPQiN3rrFfLH8h+K9brt55gOiXkGMUQ\\\\\\\/jaiZNMHrVnpjY+PbvO9ptHgiglxz\\\\\\\/dwHUGliicK109gQW+UzgibnTd24rMwl3fiK9X86EfaXcloedMNlPLPJH8eogrHqXns4WKUKbJkDrHSvE7xdl7HhKypZBeW2CYQ6eG\\\\\\\/sxL762AKHdOgy+T7xXstTfq17KQ0DNqsjOT6WPImaQbuU5Z\\\\\\\/xOnNM55Lc3a\\\\\\\/cRGE0F87v7AcLwmg6XjRGww97DxZqjPhB4XWfcBfPH5+y++bSPYPfxqCLaGtoCrRyfJ523+Cpy1++1FOdGYOJkylrHjWcsrgha7\\\\\\\/93yZywOkUZN7FctV4Yi+Gbn20wy\\\\\\\/0RrHZMYA9tpaay90\\\\\\\/Erfz89XUrSYNThKifaJZP1ik9oCXbH0RoDi4a7JZCEIHpzkBWVP50aotrjjvyxjyH5fKpmXL\\\\\\\/HbzkeJycwcpf94wbexyUsR77tpzV8rIjAgp6\\\\\\\/3KXftqLRp0ejj7KN+mRt6UF1w0AAx\\\\\\\/rxIbGAZbZWPhknuDkcgiXpIZChFGiUqXip0psR\\\\\\\/KH0ZJCnQjNPMYhIbVFA\\\\\\\/2pCmHJX3IDnYhAnARCQWH7jzc+uuRPxddHnRWlzW8SuNFCjemubNBYTJjGb3hNQwgn992l9s9mNRvlyBXJ3IXAteRay0k43HylW2mISdIait\\\\\\\/7PKmSgF+QEfiBt2D2GP0vQc8W0lLJCy+RdgEkANemFTPq+9t6cuDGZK2NuJoqdAZxGCyRWBG0iLU5AgwVO7ZDa7zcpgbYUqnMCySVsuhKkQWHd7AkUU28WlRnn9jydsy+ZixtDPLOsxRP6+lKM\\\\\\\/PO768ood1fRkHWAiXcGGcsKLUZWNX6xsoFGp3mR5SE1JZtBDLeqhgGJhDl7qMZrMAIofo8ju4BVWfAKVy1vg\\\\\\\/GLZIztDktAgNfhFfcoiSInY1a+yZBsckXMponEZeVZxuiokNRguR4nyZjTXujX2iFyYCHp\\\\\\\/NMl8jXmfOueSd9isi2GkudDicwjLvePjX\\\\\\\/IOIEiFT8tkvz0W6XqbPTdQQJyzRthTlBhqARHrzojl9lug8dVWC1uvJbTHoeCVAf5BzmWDQpbz4JWZ+ttwupAEh7P+oWpHItYiX+9dEgNJnCs7qbSbpB+f7HLQnGsM9oIztCJCNFM7LNun0FVO6cvzC1KOMV3CYGO3eegEBEA2c9rf8VCeaZAfWIsMeFM1PhIawo\\\\\\\/o3iZACfQQBic2vdPqSIjYA+xVQpSuR13TMTsnch1KhxN9cRkzbcfi3m1+AZnKZej5YWBOWEm3J4C5w3suaQhwkiCcFGmh84oIr9MILOf0g2msYm4DzfFCX\\\\\\\/IOO1Wxd1kDkjlvnXD1cGOFQREn5RJ7W5YRtZTMPk6AL0e4Ht5wmjN5+TsoiGgT7aptfnYHIaac0jePr5X6tE9N7HWVwv28JLKvJYJH5SkH98WJNhD\\\\\\\/sdqta1\\\\\\\/hcTMesxOPwrR5VwM8u+xkLguKMpQLnVOqXLqwG4hxEfp4s5CQqFgAkdTyf6Bnzi0toIHWgs26ovaGaG4VNVNPgOGRuGAnyF1OT6i2g6iSR+WzUOR4gNIxvWRL\\\\\\\/n9a2G5X8JQiZO05Tlr58GWOsbfJg19my6mlsnHxeNZVAW1rIqdC5rGhbLlz2vBPYCBwO4sRMHeV05vGREowhxDmCu7Mw\\\\\\\/Xb2dKvF29ovibE60wOgf31K2xEKCWcoUhfh9hggbCdUXcgAiMcNf+iOPIhKNdibjHOELrZJSDdudMlTOGvcTy2qm8YJDk68U8+sYIarBMy\\\\\\\/9cgpDUk+MzibjItqcmF0xegBTLeBIDg34bgRjXDcasWJ8wGmDB6oQE6jojrzs5N2a+sKts\\\\\\\/E7ixuV3LBDzrfHDkQwIGUOA1ig2+fm\\\\\\\/XDhNgANUpoGL5aar\\\\\\\/6VOzj8cZEL7ZjMvaSOlfva1HR9bRxzTWidcRsWoDwhL0pTZd67gbEf0wMs6mDyB1\\\\\\\/wChsdbGbpnrFJr2Nrc5+0sFdFyKLH9gsUx0ZwG+jAay+VPMCWvrnB396xGuX5JfjnipaiwiiLK2LrnHgdva\\\\\\\/MDLWWKsyc9z49\\\\\\\/GZnSphJPH4V\\\\\\\/E+ppeooQjhs5TFiwN+LuFtQ6hFRFkBIdT9T1xGGR\\\\\\\/NnG+ZsmfhuHRzwU+mBqWDU8GM8FHb5oUclS9M5skw9zwpUeXuDzOCwMu6YqrjaWSRZqIH8Ei\\\\\\\/obWKFlWVy\\\\\\\/cTkxwkLLzEqi0K9C0jVtMqjJ11PbX0qKDey8j\\\\\\\/uK7V9UCwCeyQx9oxuQgVwtZEEGgP8urpMNqjsQmMsfOZN\\\\\\\/wL5i5T9bATuSM5k5327ca04gPqpylSBl28xPNYunLVmw31BslfT7hrcSj6rFoSVfSfbfLNIzMZJqEk+Xu3jgLI9slMb5GEfy+YDQ1csFo0BvYN\\\\\\\/1n\\\\\\\/5flX7qrv1v11KoEOxU6FBwJWNuuHJ9L4n6VGie34UZOvYSEB3ZYMO2LZkxGhOSJFrodr41BpXnMpJZJ276+X\\\\\\\/lEAbZOYXbttIxodlF+loPo3x4lv\\\\\\\/ZFCg0yfuyBdTA33bj6R9g28Nb82qeLoGUdtO8j8Wv9B6b\\\\\\\/KVjkW8\\\\\\\/s9keUSuzR4HmhbOcUix0SIUSMyjsQW1ryJijHC4D+ERx6gXqi\\\\\\\/Z0etzyoY2fpl05iuRL7JAWABAME\\\\\\\/10vOmjrITq0xTL9yI537bc80CcHmYqXbHkt9nvdiGaICrMld98do9LosLBY5vW2vNk\\\\\\\/Da13sBB+17zs4jLJkNNuf+\\\\\\\/ZQUrNL5saAIavLTR4+FWsN8DiyR09JiR\\\\\\\/Qxk6q0fNgIj\\\\\\\/vx8\\\\\\\/Zfuz0DSxrrYkKxqVoMt\\\\\\\/z7qMpWg3xWctxQJiNeMa6HLKnFyf9re7eWFYiTx+RKeDX00gWKPa5cAIfM6jWVh40fCaQS9z6orBfpudJ2CW4tpaN4OTweAFa8T\\\\\\\/1zuoHENnVcTQc6xz0EEDGPruxDvZ8mMi28sC5Qn4Ek6uTb2a\\\\\\\/TDGRH8APyhVJ9Az+I9\\\\\\\/L\\\\\\\/XKDSj\\\\\\\/fb2Uvr8Y+Jisz+l\\\\\\\/CW\\\\\\\/NZ6YaLa2PJgwtMx6avtkRuF2+Nrq9iHFZL8\\\\\\\/z\\\\\\\/b6x8ii1uUy3SmbzGqbqN5BlilgAD2afZo69edkgZZMGGvUPtcVou1nyZgVLrIDiMlQ3u+qRaVJ6X6BIdJn8CYLu99vfaM67O1IYeD7z+CzvpGP2TNypSaCfmIIW2fJohKw3rrx+YXEG12ssbkrTfA19RenclenNHVIvgDDaoKNxLGutb8zSarABuhWd5J987fwaxThqrYFHg\\\\\\\/EVA0SbejdhVcr56QCYSxhds\\\\\\\/TIfUViQ0YHX3F9nC\\\\\\\/\\\\\\\/wV\\\\\\\/wVloTV+GjoOCvNwH3CDm9dCgnxMBHi4svPCmPqm77MrNApqhYsdMRdIdgn3VsM1pmybmVDVDVeW+F9NdQSggabSHIstoN38\\\\\\\/jEVxp1zy9b2ZA5Tk9G5d1Qf2OzLM6D+7B\\\\\\\/fZca3FwOuDMdxPhv0VqiMQ7nwIvhEMqHNhgXtrw12q93M4eSgkknX2kK7UAiWA5sFQhLb7MPQjvQW2dmKRJSYfQcNC6ndAd4Pna3BSiPtQZkIL1chEqhjldwSUD8PipQmWaTbj5Pra14L5+d5A1Zc+PdVH0YrSyEjMIMcu66M1Z4VdVl3vKTWZPmCRL5VdrrVZ2YFIF7sfT3dhKn65DsMj+LQSVgUNWrYgoF\\\\\\\/JQ2x5RUI5XVbcVI1UOO7vUhtN8MyK7YccwpRruh5HxEhcVl5m3VEDMXk0NRpUoiofBvjdqP+5\\\\\\\/LEbGmktGNvVoTwB44d2QuehcOSwjupnfFBwO+joJ2NpsQg\\\\\\\/9erUu9\\\\\\\/6PftmT0EI0C9TS+8\\\\\\\/37xVH0OEcmDelkjI3RBea7tcDLveC1dGahxo6oSfzqVr47QkI78\\\\\\\/Dns4pXTb\\\\\\\/dkWJsZcQLlzQfJJEVEQKzMto7c1QKBFHjgg4jQrIPJypcACrwkql1ohK71nelY6X2gFOPgLzYibqzog5WfMp59KhcJzA6fG5rMfgyEdApNgJ4s2yImVc8kaX0b9Tos98yQWBftW+tzctIf6g0JQQ7O\\\\\\\/GVd4a5uVl7YGki6VAR8LcAYwxzvaREtrUu3mVPwZjzx7UmBNfOOks5\\\\\\\/qNoJ1g\\\\\\\/cBuM\\\\\\\/K2XLI+na0Y1mUPhoM+U1Fgjz8jBKcU7rEgrMn9Azy0lxc3PceJbth4m9UhZDhLb15EB6Fbm5c6Ac0sZvZv+kJOnWNvARf4Jo+\\\\\\\/NTkDx4Tv8mgCC+TgCZwBL1sRIenvpSb2lLuJebs1F7yGDfb9jcP+pQtKG8L\\\\\\\/54L5Er0XZTksqj4c8gzFBvq8t3+N2ej\\\\\\\/z+dwoO5UB0782zNCafpIlmxS0FXTkXHvlahXACa0QSvvro8tsGcajp6P4cKEDHcCOzWE0vGssN0ro0yK4qmPaSNDJDqkF4c6GAbl41cTjSlWlrYr6N4BgJbwEOvBV2N6iHskIr8TT7uIhX4Z8ZiXACpCX+aZhMyDTC+rWdKqAznfJp1hyRKIwtER32beQFr9OdtkyRuUA\\\\\\\/Ekr98ZCQuIfTsDW69kObJCF084BznW4uiQFD2PF51+a4+Cq9tMXLGxbf8tIXVWFspeWm9qoIYep5MNainbK2itVU59cN\\\\\\\/vmo6m6B6+kwOfsalw3Hwz75Q8VMcM7c6EIYenDMYUZTaA\\\\\\\/qytvZsjIuf8UedpFacX1DOfipVBdAkcDSFBNb\\\\\\\/CyySBPd6tJUyEJUSp87K1ujSwdx9iLpnXNATPPRTjPAdtHoO3kqThDyLkgZhoUqDkIgya5HtpMfjtq8oraQrKTprTZCNbIprVPTxeY8iGIAe9WqWVsyTlsymafiH4t4JOVCx3zGgZHzmFwQINXMoGpnTYdLEvf3qOe+9vCyWuD63M338788YgTJ0rRAKlT\\\\\\\/7B6NTU9Md7gN9E1SkgcyZrGE9j9sCZ4tXCg4pnWZknXYcHFIggtDR1jMSG8EUXVhJeRWqB0HkCB0H8s0RlRyvbtedcDWPGPeo2rhYk8ILkzEKBbI9B3zOObf\\\\\\\/7k8GsKzMglCMwm3+894kI9AVjTljjxd3znSRm5z\\\\\\\/n98ood778WXD+0qULwoDeYRk\\\\\\\/\\\\\\\/9QhZX\\\\\\\/XbxMPqBV2NjGRuoUSr+YUHB9uLOe1U7e\\\\\\\/1rTzyU2vaKkbIaJl0pfOWjRlladTkyz\\\\\\\/OxZ52PdduN\\\\\\\/TG1XmWaNsfc1d+c2k663\\\\\\\/gLv\\\\\\\/lbYlh7xjOIcpbJKJQ+XEaofqcZtBq+gdwPa4iYr1e+i6iInhAIeWaVVBlLpkEHPWD1ihyYtbTXxUE49J+98D2ItKnuKZzISP1FuaJtoIMCJS2OZpgR7xOjOpwjjQuQb0\\\\\\\/DEyviZ3x51rH9FRdxcv4kIHTV0NE4HXhyjcTIGULuHBsH6btHysc1dGTjL15SJQNWGg7JSh3Hk87ryv969jqQgInB\\\\\\\/LwNabIsIOFWwBEY15wJWVPWzvOaVnuoSJD+DmuWrjSHAOA5Mnza\\\\\\\/V65tvG+W0Lys0fV0XbdP+eWFtxqooC41psMeb6CGS6CKQUbXYbS0dUjRGeNAVMx4pb8to4hZf6SLXl6+ueOad8hKwvPRLhOM3oMfupVNBnAZlq0WS4jsB\\\\\\\/D3\\\\\\\/2AqHv6G\\\\\\\/w8n9aSuHbjB443oMYp1MFBUMz9+EW9uzX5dzHCBNhJCQ5qh8MTa3wgWSR4SPemXkl0q5Y5HC1BS7nw4lmLrKqMVUEWrF2DdmYFrYp1SiF8j+GSCqCl4IzrGZHXYZL7dbU\\\\\\\/JAB95rDxveZAb1v64sKQ8D2iwnqqo5Sm9+Ta07eBBumwOKppaguy1vEBhdzFenYdtcoKSHo6zw76IpvuQfnEwUjXibm94YGh+OG6+q0q9fZyvPrCBi4YBtZSmfCxjpRStjmLLoaktkGcndzhi7gvpOBb0HKqHOt7ihpsxcEai2uHwuxuYfTv3Pys+kqeQ\\\\\\\/bs\\\\\\\/zn+7fPWLqa0Geh6ApBmlr5We7if6m+bVwNzJSZBSL\\\\\\\/xUH4jJ5LhO6uQgXxKbnUrNWjJnqWQ8EAimKJDL4uK5sqKu24AoIEWIX8Kygi9txJMOlCKrfxQ2XauW3nol0r7P8juy1VSgwRfc1OM4jiJdgnnorY6srRcavs04VXAxUZABR1S2FqV3nWn18DOfaKBncUJtt7S41A3POfBmnpZ3wDn+y4MiTGQwRv28IRxifUExeReSZCZMD2locnmmYnh\\\\\\\/HLDy6nEOdtVSnw5t4gA1U7IuXIA1\\\\\\\/nzTF\\\\\\\/001Ehe6aVScJI6ig6yPAjkyHEFvf3cHRLsfxZYgfF6SmKHmbL+qZ0oSL9HriFIwS4h6+xaldaLQaMOArW05iU6FS3kY7DmxfCcp44SilUQo8scieaCKqZRp84pjcCROk63xAkjT7mvvZ7udcXm92JnI6oRjoVTlr1P6PtGHmv8Qej9Cq7X1Ewp\\\\\\\/gpv6btXFW+EosktImIJF0ahXh\\\\\\\/IBvU9l82ygNYuLmoJ0dqxaiRRX3PebAYMlMJL0hq5YLG0CWdyPuKJTQYz\\\\\\\/\\\\\\\/Wm0G0\\\\\\\/L4jUYeCpCbinK6lYs\\\\\\\/DnBcV5t6mUGGZCyljN0a1Wro5\\\\\\\/YddJor3Pf1ffVQYeHPYZiGKFfcrMpfT\\\\\\\/OmHv0DbVRJiD7HAH8Me78SfI+fG0UGlenZKUxDLds+sVPGrwkxE6seehffOqTinDXnV1jloxWGONT+EfSvsCSePOlRYn7DYUljKu\\\\\\\/73BvHO+nbU6Pj3hiSRBIwdgSZb6Mw9l3ZlMUi\\\\\\\/7m2tE4X8Fs90zGmfu7dUwi1P3LNTCGE8yNJRCPxf4Xnu55zZe68YhDk794xwcY1N7q7wlbF3tNol67pnQD6StUSKq4aMNsgsmr\\\\\\\/LGneA\\\\\\\/Sqaupn6amoGFK5e76Qt3e1SXMrOjZRjGlxbFARwNaYYrjFygmVWQ3PEv6BkFs0VREkv5Vq8r6\\\\\\\/QQDd1z\\\\\\\/2VRQ8iKGr+qNGZiSlNJufFdXUe0SKbPvib0QHsd7RayIu9fIS5U5y4gZ\\\\\\\/mYDOWy7zCVl58EDz6GB1SvzcT+31t4e\\\\\\\/GcnSpGDT5fMPVGbOzm1u6+hgPUQkkT2\\\\\\\/E+i81gcWTcUQRlll5l3mAB1SmN4N5wRc0NTVC8GUaYchuAePVwRbIQCjnT1UR\\\\\\\/yMXa0qWFLKXMOqyhTbF29qN96mFdEziBwOp1mxfeJpj9LbZSgIhk7yO4zQ6MudDaK4QxHqurbtxoWnTofK3WDzz9wm593BpqslqvAc5dgYhD8I\\\\\\\/xzJMryf8ggct3dffHeZZ0vEOg6pgN8hi5Am8PpoxQMRoNuWINJbia2DzsCyoPZsoGmL1cTrvYjXZCm7VJXQw3SOpyuD8dc+e8\\\\\\\/3\\\\\\\/P5cTL3iNNRQMTelTOS5TE4XNQjakDnAoVqRYc4tkBq5eHZWinlBYRwP\\\\\\\/ZGTHADfv3AQdnTa8XjrK7OzptJRqMvAdvok3YnrTuPINfPIUgZBvvyDHdtyDD3ZydyyjQwRmQZgc3OP07nAhuW5PejEoKdsXt7p5+MsqMVmvgi9JQL1pkgGk6r2\\\\\\\/AfI\\\\\\\/yTto34XU7Qtqwtfh0Q+Y2hVRblB2nmVv\\\\\\\/5w60inYQWj4Z+BLW4xeoo0sx0vZ3prRLTIJkJK6TmxBPAmKAPIL7CNl+PrO7X7KkUorvx3e7XgQGXw\\\\\\\/fst+8JgpdrLT8cGSo6CwqoUmrAmGmcxnfCmvgSMe7RzpTSw08HlbYez45PmA8TYZiHmhtq29RPz3fd7kaXlXEqzJTvCi0K09Zof7scLfVpHJMiRFjY3bvMwkrMM\\\\\\\/5g1jUcv5ZkPWnMia8ThQ2S1VPry\\\\\\\/tNBJUbOnrk3O76fI5D58w3XdByePz\\\\\\\/TpMmZYTQlVAZfeXiGasotRt\\\\\\\/2vHthadSKzcEtK1PlL8lLe\\\\\\\/K71ZqH1FLfeaN7bH3gBGJm86SjSVhL7mwt+eysCwkIJBqZ2DJcAi+i8UdmCimpHJ5rMiIlnE06Iw5fd\\\\\\\/4E7xabjXwXWtI8tNfrcKxbRHksOHu5TQDaI2jvz7MhdKarrWrffz+7Utfoj4dnGighvb0U\\\\\\\/IfULtIxi1OzS9Cb5wLtpVsztySGxlRGGPceTzah3F1PxJj1fhPWN62OceJWTITEgOubxafKZupsOIVfFUXbod8WlJXyqhJ1u1QpYkKEEs35fPTfFlodlVyCJPGk4+18AH5v\\\\\\\/STDK9Zox8L3ZFUcbtfvwqoDuIDbiU8WmSCwVXxfFvUOP8ndGv+IcQpez5p+BlYDf\\\\\\\/NbdisLwoB477p4oQwl41gIWKpoXb0GEeyfVIiuDmrpettVZueNUmKKFpmneHy9O6H5enYiqs0Tj+ZOc3\\\\\\\/\\\\\\\/xnY85l0jCrTYojSMiuW9l3z3zo6E2fvVf\\\\\\\/p\\\\\\\/H9z8MBf+NJqffqI7boHFf1gG+n1OPEoI9IsbOVwO1U0RNj4l94EM6jYhgq9xxinpAHUapSodaE77kPw2IAtlb29TxSZGUEbZcAEHL70dr9wrMttodFh\\\\\\\/moYjjvTuR+rDNu\\\\\\\/zgdDvOCp0OohNBkxhcC0aUYRHsqsa9TYpjKU1jM8U5iZhdmEayRInEqnrSD5eX2jo3tpjDXdV0vCnXtyP5H46Kt9Mr8Gq0ulU0SXjrjm3rVJp++zwLRGHTqo5CCnymVwkl3Kqlgu0O9OmjRRIyXp4kK9NevE+R\\\\\\\/s5KY\\\\\\\/DWth2lXt+MLRppvD1eWclbIwdCeNprkt8shyUkJcWAvsDkAG5NNhoJyAQdWRxx4ZdeVKRDJQPuN9M0Md1HbAt6BN4o54DbN2lIp1LcITcJRQKnsPgyRpxYMHDvQSU\\\\\\\/BAY0vnNTtY4zfD7sBH7m8eTeYsnmK7boMMLCjRgcVCXTc4fZwTNbe3E4JCc7JA9MQGl3FLVSQQukyyEy1ax4nQBB4yEc6mhvynWYZzcD+F5nEh4a8MlVVUFtZT18Tj7+3REpZ38ILla0jrAALCBhpfrKmtS8E7INonZZ9c\\\\\\\/Rv44zcQl6sR4EjWwtvwmuDi\\\\\\\/UzE1OODXEfNGXT9QoTf4F4GIIekKkhpMZbwTNFNZaqPGN5K+KcU5wga8pITVM0ZdG32hnM\\\\\\\/2c7V1glH\\\\\\\/77sqikPbX7NyyF6WR4EBiZgfyVi9VjmMd50x79IZwCmnJrgIKhxQELrITUG2JvH8hTZzzV5f2JUaYesR0fVt8ot6I+JVJZ1N0rT9cMpH2NBKdfX\\\\\\\/7UmiCumtnmHx4eZ+r0vx68bsCRZyNAsGIPJS5o5tNm6BKQxRwyxMBd4kTz90z3eieB7fqgukWO7K5JeKJz8oDt1pNjSQkvsAGbsJTeQwP5nESv\\\\\\\/0TvjSicmi9pqqCyfqNJhYDoNS5NgpwlCdWOfuWdJzlGpstTGBk1p\\\\\\\/reAL4epubPlmDl8j5YHOkqovrhRZxJpw\\\\\\\/b3o\\\\\\\/+hjHurvCmvX+F4c1+luhys6yKu2vqwS4tn8BEt+e8vfd+c+anrDd31KZNNR2tUP1xThsklI5ggGLs1bSV0aOmqeUik6rG9rSaeanqQbBfZXdwRJ9W9C6i1671x7yahRKXxQ0sp\\\\\\\/lzBmMkkAT2ps5lq334gG2Nx9i4PKrjVpVkmbZ1I7BmNVnkv2aKhDxVZ9QcNEGM79lBLKBgQfKupmHvetsog03kwYdElTMjjacHBgWZ4yt+4zuHFGw1QBQNFPPRBPTRPw0S8uo07yGLukk4O9PR2hFUvtB1XF7cWDMPySA0QZkKMJvlQyxGidLsdGjUJu7sRu8oi0ZoP1paC\\\\\\\/+lO1viZPkg9FhCAqrMqQWsolJ4wjie9A5VsR+jBznSOwLo5V+EgjtzfvCNehqXZ0K7+BZAs8tftNM5arFlF4T+C53iBFTJzSGbOGcq78KMpAY9P5JHDNSjFFQHdpGZQbgLsTm\\\\\\\/Cujo6J57YC8OodIDDcOQGZe4GFSo8889EqAvd3z6mykWnNVVolSZlpGQPNweqcltrAy0yT4J73nQLItpalhHOcl9AU9925yhPS1Vq9rMk3T1NXhPBk1p8+AtW5tVzIeDRABGxWezid1gv9NqpdJKtGLmA9VIy6+EEkd2qXXnaSqHxp0VdKhbetiILXDPXWZdhMLYpbSYkt4\\\\\\\/NE4WmnI0n2ST7tyazQsyJ8CxQI7kiKSGmxzWQsAdNt9g0omXrt0H0dbheM\\\\\\\/MKcxdomfU7x4tfzxRwS1UxYL2XNF7k\\\\\\\/8oPczpUx24mmMw9wm6XEu4qsj0+fTugeOXM9uXEkq6GjvKKJlciD33QsirN+36b19ZPoOk156vEbXje\\\\\\\/j21PshfZ\\\\\\\/KNxYFHj4UmcIF5Yvnj8YI4Z0DG6FCxO60J\\\\\\\/zcsd4f\\\\\\\/+RU9W+SNADo1H9TfxOXrAxG\\\\\\\/8PmDE33nYKsYgk\\\\\\\/SSZy5EP5M5dQ8FLbpkXWuVuu\\\\\\\/hNR0JDLiFfRuv3UlrTi5XkPlr\\\\\\\/Ydc5AqInjLQYz\\\\\\\/f9W+XtupmjwKVtVdgbfwgmQred4MQDLDYPFInSg8EvR2Gx23MuVNZoXnOwLRyD1IM77z3aA7auBxZ\\\\\\\/8gVQzCC2Bfy964NhrNEReYswa8MRr\\\\\\\/IUiutjbyWo5u0FYWhvH\\\\\\\/yJZLOgxOyfGoqLoFPPwBk6mfRPTAXmYQVvUBIuXcTusGeEWT5iYn1LoKa6g6tnO5p\\\\\\\/f697oCNXN0mXMr9UzwCRoBJCPRygGc5ntuKUZJrKw3TVSaITY3aJw3zuk\\\\\\\/9obvalesYxPWE8Ow0pZax+FRNvkEsmn0pdVv8IAd6tCbUnfeiZSDtkIw0DYjL9AWcok+vvZZnJresQp26R4FfHiZ0MkEdB1S+EEQm30o4lulRAfHnBy5ebIcNHNRjx2SmcYEaiTVh3rkxmegzVM+d9XVQCSdlo40nyiUZBOblvoKukw74MAuqDE0Rc\\\\\\\/Z7f566A68fdVvGPNoktzQUgqrVcQlua1fl31383uiZ0\\\\\\\/W5QH2L0Rqp7zlExtmN5V1WKUeY6E2FSz6P1\\\\\\\/y8vkfXk1FnRBlLPeygVRjSECdtFEAU+PrxxDqyBQiFOaF6JPI0kDSTotd5MNFA8rDYDe6BS9jbY156nVbEixMuNEKQ0B3mMfoLenCXbaSOjRqn2m5CAfZwfqT8Q4NN\\\\\\\/DN6iRH7tkpfu89f3GAGZQNGvXa1SqGrwrlATDqKKXpcLAg2yCKbSUdPgA0kgTNe6XqMFFyECL2HXKMWSRD3074tkF9iw1uHiV9SOItu+gHXpdFAdmU2rptFpblCRuFrmSL8S2jjDFH0GCLD6bXzE7llXOVbtRnXyxpJuwOHzHXbYawTlkl9j6Z\\\\\\\/4rhwZSpumWiTLjgjuh1cIsc38BUsBnvh4MFDl4ldupLSm1xZ+78bxIPy8yR3O4l14uZbIyNAMUzYWu01HhNxxGWYSYwYhA7Y5ZVFhfsMjEEhCRFgWBNFZWBRHkRR09OxnBoVAVVwbBSqawl2T1cv6zY0zmBlQyFsqOJgWV0ccXfpBSsx+IfRR1RDWRung1J2fRL9BFa9tn\\\\\\\/Q\\\\\\\/Ssd0iGdne5z4sWjA60pm0rNl6uI0Hvx40w+OlH11ALz6g1PjI3fRCC5O8ebSdZO6m5CYMMnLvTrzM1arLR6ZVQBxSi2VPlbs82vHz2+YQ4ihlixx5EANZn4xZLzUubGbAfFna4Q4EJGNUZEfUfrTuFjCkXuKRor4B1CkDVTcr7TL6rNxXiG9dXldhPOsT\\\\\\\/1FJmJhMvw3+qwcZRIhKBNM7PfMYLK1XT4sWfRYCPrl0RIENWA4PobXbWY7G7qWdc0z1ncVK\\\\\\\/S29DdTAbO21s2v08GcP1oFS7rGA2CJ49SwC0NYhyrqjWb5W93\\\\\\\/0F6uvXbryjnDEDEZuDfjhgGqQmrKwY9T6qRveXeeQaV5BemW4x2YWyCfE1aT+gwjGJbWRQYLuvNuo8Dmj5BqWuBlmUy5VkLH34OK\\\\\\\/N5Vj4PANmxwoQG4TC9MtTx9wkBfqfJ69POnYAldcHvS5B+RM6u18aBecP9bOiYcBvim64JOrh8WTmNVCXxr+A0pgdF7e9N+2N87JNQVqlLyPiVzgecoUUnNNZHI9CQkkVUZvRymBDPmiWz\\\\\\\/gPvLZL2Sw57VZAa+Ip2sTwnWsaDnqJC3n2rR1nqh7MJgWnk1qULs8Ufpczy3QBEEYiwCvXsqUMbr+7UMTNeYDpslzhYt1WD3CbbI9JeJnKi150BlMLKnA8dikv39fTkoiMlpEe+Yt\\\\\\\/Zph+q7\\\\\\\/iOpsoiXuqtYYhbSB8FIx0CE3NNlf5RoO3MnL2QRSGqK7I5GWLTtNI39wuLCQwrHsop0DpaTsGHV1LK3tW3vtBoqTgq5d0U+L0hlRrA22LFY\\\\\\\/21gd6VMugag4Flwrx\\\\\\\/MczUmnhsQXBwQp3GdBuR69Y5XgNJpnVIPoGh\\\\\\\/1hSSjpHYJEZfEhZ89l7UxHaevOM7MMyL8Wi9jKOQGflduPjGIWQidbx0ftetyjCECS8VjMEAi9LMSowhlPVmphhtThWFMSedA9MAGkdOEThSbHhYh8Y\\\\\\\/15lZlotbuCFoXVOVawtlEjlpCUThCJnoHEqhaI2GQsXDpZhB3hRy\\\\\\\/lUZ3dFJmBN4ioifh1Z4kVWE9kXnxJqmF8Sp1TDplqcyfzu71N2Hxel9ubfw5M9hJABhjAGTAVXHiT2qDexrQFYo\\\\\\\/pAPN0\\\\\\\/MoPHM5EfTO3wYEixi5k\\\\\\\/mNfuMA7vK2\\\\\\\/VlWAstSPAG+osDqaBZ5fNqM8HToCaTCOk37KJ+rvWcKsMOyxkn5zfkLpJeTY9hIWD9I6VWwrlhdAY6xnWUXfK02TK5hJRMdWCZrfMzgY+jCiMQpo4V6iMjPuFdYvfy1oQaonDbf9gZUImCGPPD25c4zy1XYYDsLzOY7MFwxz5syJy7zkQrPaqfDjVQyHQhiGPvUbNv6oksKUDkrtGS3WGpDhBH2O7qXGrG+nuV13TO4aiOgvA9xqfkHRO4z3mN0AKcjzOxAlnOITpU2kNe3ozxs6yHhABfJab78X+lLy4piN2sQhWFQlr1Jb4h30R3FpYUP+5zuTLGan8\\\\\\\/4nwGYoZKSaR8XqvDAA3j8M0gtILO4c7e55U+cu3WNa49oKhStb2VoQTgN29lfx5RmOL9R6vHWOVAzwkBGCHcVJkz67VR0p4CoTU\\\\\\\/i6tPb9j7pfyKtoNP7TJHKaIwps5hvBFbM2DZmAZFqhgpQdsWAT5hcgHjgDYTwVtm5NXmaXfJD3pOB2pWOz8uNqoP\\\\\\\/4KAGFAPgz4MC2rCZCmyEsH+N8X32dJJ92ff3wKtrA+RoFb7uQvaGbSrR0n8lQ\\\\\\\/rfk1DkGlcexrLbGgC3K5QP0BVjKXLDAcqllLZaryFdvYz5m48QBlN\\\\\\\/dCKZ62O9Ax94gRlAIUZpIOzrEjipXK0FiJ2j9W1tAJnAMHQbeZKu9YxqIwmStXSz5xrKl2QP3PQbGivp8QYllP3dg28FmPm\\\\\\\/6MWljc30UDXsYzraUKh33H30Z\\\\\\\/zwzrmU32OkdyILx2C5vT4S0xcb+DEqsOAOrY\\\\\\\/QyOrT7hcqsFSUTiGvlLWBRIzWaLloKgDuvHVMBS42BBdWjvbczVjI0j+Vz7ufJZL9kbqVfrjHXP1siTg8fTQN+aYJEFwfoPsKrFIiHRvC3ijdF0EgjOwBLfDQZdbWeIwWajX4uIpTwXM9fe5DUppdw1XGpUITT5EDEAYqVBpJFSUBE93P9rSSeHEpT7EVAEXH8lJ00ac255E+osrL0an\\\\\\\/5PcGNlASe6QEUvrJz5P\\\\\\\/7HVsP8rTllWhTe0z4vCqjh46zHSY33e4vPEfb1MS0bxDUSk6+XqHYfmOIS87tl3VRpzS7jfcsQAFtlsGCwXHpasZCw4uWRUmme4GBdUQkK8lYqTfZQO5UkXQE8NqLFToIZJw9OHcZ\\\\\\\/woaXSE43A3ldMMPrPSkaErKnPLpU0UhpewM7QJbx\\\\\\\/HvahRG16t5vyF1OZ8FgnWy3VJZ2id7nFBIFf6b5S3AsLnvcVUcAAEdBz6lTGGVFJG4Ecot2x7+FUQSlPtBLoF5Htc+qXvCFKKiOcCRb+TWlzHI626Lw1VxcgsufDL89pBnorsul7q4rG2398vy93LhIexI2mZwA1XoavO8OYyWOBNRag\\\\\\\/45nL7vAOP40CgLbgF1JYcDEEutFTy59eiURPn+2rJn\\\\\\\/BsKpXHXPTytAfTFt7ic651U8mwN1wiwFfjh68cT92osIRsvRVQbGYshv17efKgfbWRzVHluRClHgMMVGej4vZ5sCTjgrK\\\\\\\/nU1OEHi4BTUqREIjr9RKR33FlKEqrnlSp4CR72M95KUDD0OCjRrGINvl8b8kRvWh8gx3iasLPanhmWkdUnT1CpkksjJPz7PD3V2OZ89FlOGcO3x9BhBi7Vxm+Vqh+RshMi3CfGwU5ttO0BgN1jM2AzE7o7uXqh5hX0oCgAuuxZGagFP7yCMDgFfXhPCHpLSllCXtNmJPltjnwPeoNP8XFwll\\\\\\\/45DF3T6LP7RaXfA5Winht5BuvHqoLsobSum\\\\\\\/spzkj4UXNWCQKTMT12nUResyxSU7aw30Oj35kSO5ThmU8IVDzc1ALLHukOw36p9XVKqzl3t1Nd7fO5A3QRxUcQdhNQDwiRey3fzOo7trF+o5fcW0ZdbwRwHyWvcP9JEuk6zaLK5Qgmpnx5xGXpJvuIqyXE1eBiTRQu0JgDok\\\\\\\/ETeSmwIsyMEAyXAr4HMCcyGz9YHGCs7NeHP3Kgr6A90ofZ6asjxMR4EgypSjVsG8cII3gX\\\\\\\/t\\\\\\\/wsRBEuoPa+CRLIwH\\\\\\\/kmMHQMqQ6BmrG9UMqrGWedj20l80XpzBIjkQI2l2UPZvBlWxT\\\\\\\/r2RhRrQC6q7jfN4mtpNfmhLFTk42XNxVL6ZaQnPlglm8Ip\\\\\\\/E6UEyBeI14n1vrIfk9Rw\\\\\\\/c4J3aLMTH6++54Q4No4IK2MIyXoTCb1iXG9cg5Ypw15sCoaCjZWPFuFN+b+4mCOQp+TzOVPiXt03sccHYOsTW8uyy0CNU+2BIEDbPhPOZJBJabzieMce6kBcqx4ing8ZYCS3VZ4tnbLdU3L5p5WWTmXPuPuzXR0MPnqQao84zMrkDXWJHN8mlOtYIFDm6SJPFaZmPHkE5cJn\\\\\\\/kkm2WoK0OxWnLolPqSx6qt6u+RwWPVg9suhcr7avkxrRgks1xHJKGhGGAq1qP4mq1bLo1IhWZv2D6b2E0k6HfJLgXHpAMDCqw4xemt5bOpRB0glN5L0qWMwnPFSxWjujQFFEBfviGV23fO++Oa7NUUMxIOoLsmCihrgKuaFcaMsp9HNKEGPkn+wGbaAFZ1p0vVhiMD8MXPKiw0FooLWwfybxJj5ZsYtt9y\\\\\\\/U\\\\\\\/pVWmWINe+hUq3yEMBm0\\\\\\\/wuy9a1yurVvLB5qTiE82GaFXDhATxo0490mKR9XmoLHMNAHzpEFDIwywZsC\\\\\\\/lx6KkQ+4P6kCFgv+GQEgGhheHjwhU5g9tn57PqTjRczC6ywm9HoMjHiRoGDgKjj0OjAacu5YuBMdOZmSYox2Kl7xKHXHVLI5hliSNwnTGfaLviw\\\\\\\/8UJtIlKiXuTXylUwVOORXosJuQm9gm0TyNbCn5Dg7zhBg01QqxxvLFW4Nd2NhgjSVKGzAHjZ69CF4bda2L2DcNzYpatiGFB12UnOZjfLs+\\\\\\\/QNOuudhJrbO21n2HAAzkSaTOZ2l22iBZYqum2IS0DwE3tRssPbILPHdc+Jex7ZOS\\\\\\\/n4vRt1PMDuSLygDWc7Zwz8WXes2PrLl6uV0fsqTDVZa6Q6Gqagrh2INNDMb+4mG7oazUxMxb\\\\\\\/vwrQec5DZrBQdnEv2UPQ0KFdFMuv2F0m4v9ojP8e\\\\\\\/L+eh1j1rWRI6dvt\\\\\\\/g47HzN4jSrMR8gLFNVD09Ry49sBzw+jzD+rZLqad+G\\\\\\\/ZUCIMiyDc\\\\\\\/zASWylqWpQQTYCniDaum4Lh8wQvQGHEyy9ZQYxnckfCLmIl5nR4oQ4kdOGF5RvvRt\\\\\\\/o1ps9GsisH6gzrIW\\\\\\\/YB5d5HInBciQvThoejtGtVjxXpxqDW8fSJ37YpC6s0O7gnIsSTzzEodjqRVC8O4QqHizHKBWKx3SnqiTeuoly2sCmKmNLZrSTTnRrRFi8QQmrrhi7BaaRIHMqwSwu+2RUH37URhQ9HTN0P7KFjyYnbPvxk0ityvt3A2tdkmHywRb8S5P\\\\\\\/ybjCpE4bvI0rDx0x9ssOmRbiEwmIDZC6z6kWwtOZUXE87yzuMQCbdQbsh\\\\\\\/E7mNdZdVL99MPsLgmP0vb8C5XIsVDIMhy1fXiuNo6FI1JOW5f6gwgctB58g72KtEPHpagD+DD07ymSKbgdeSNz4JD7Q02Am\\\\\\\/7gItVPOwGCzDChO5aCAqH7wdXZvXPc4qhWgFuNlcUw+h93MPf27\\\\\\\/UC7PnMKQEwHwK2bkG3bktbhVaWDj3Lu8kaCGTayyOm579yaOefRyxTPE4MrnUxsu70raDPf1hkifUDm4xFVcjoD+sr3zb6czEu8Ddag8CD\\\\\\\/gG6b7Gvf7yQLyoU+prmh1kcQiZBqX\\\\\\\/g80xhrSDWlsv6a9TbXsd6Q8f1dRAleWExvEuGUF2VM8yZmwCrTKRgIzAqLkTGFyc08sg8H4IROKUkYGf15j+Z7CqCuA2hnR+c2x9X7ZbA5G\\\\\\\/1lcNdX5Tr9mAgfRnjxtUliQ6FFsET1AxuO9SMUfn0XNqjJJQf0Kw7\\\\\\\/EWAIZdA3rfP5S8D3gI7Gxtdd0DuhR2VMltLa875NaZks9GY8qNmMibdSaD3dSVFnah2qbSRuINWz7b\\\\\\\/rl+W6MNl\\\\\\\/QkjJD5deBM9xnCcaVjvl6Dv+I7VpNo1oEoAtSSuYxI8Dto1unqs1ZlP\\\\\\\/QW5fwVmyFecGpHGEiN6uy3MEV8KsFVl+YIeIiY1T2Ojo8QGNoFnetKv2pkw7c4TXKAGG2d\\\\\\\/GwL1BWRfC+KLXxhA8xbDXxU+drEivLcRCAbfRiZKrPvkE8AwO0tv0w+Ly84oSbe8\\\\\\\/6zxZN\\\\\\\/byruh\\\\\\\/Qv4X2Zk7lUbFgDf7tsVDX4FSar2UC2zGuQu3hgb5EcMsjVrQ+L15duHkmhD8Ku\\\\\\\/nD+c2VUHzohDXuMNZU9PvNlFxH8FBT3xu+PbKTsNU4vEJQDmSLnm99SHaXtpUAMeUescPH4MdOERSP5y9HMWkev4PCoTONjC9Be1aXa\\\\\\\/EU\\\\\\\/N87PjdzGMSN+pYAKZhATCCH\\\\\\\/QQtOl\\\\\\\/uYzRdtfKoVFC8STvhrUb+Ah8cSHS4gHt488hHIr2nGh6y2MyMReg16ySe3Kig5AH\\\\\\\/Dg5De2fPiIXNxEeIcuCnqt+DtaXj2jQRhxCuOQerM3rDIQEhSUVbIpNlkS33JvGhZtZG1JEgZXhoegtvykQwa9d6\\\\\\\/3kvmct41YgN3HBpQUVtd6DVrIcUKbFskcsNrI60HykF6HFhTZhSNOdLbPRjlFsbzWddZng100bXo+KXuYDaggwwilvXmbKomYuBKpsuCfPu7kgrnvYWgQWal06O2NQcVeMAjJz7iSv\\\\\\\/H4zKsLsRLeW3pYC2sg\\\\\\\/FA77Labg4zgjHLfzjOx3DgoRoNw\\\\\\\/MdYXLer6U2H0ap6MEMgT1H\\\\\\\/5hz8HFjXbGPcoTk9y6M5yyUoNN0be9F8Y4kgNuuPLEUFaKttt\\\\\\\/35f4Q2xudbEUPqZbp6LeAa+6ui7pBjES0aakQHLmGKOhe11c\\\\\\\/qe65pkFGhs6ufyjGci4GOmrpMORib5WuV0+0MYyvkB\\\\\\\/zvAdeps4sNTOu79fguoeLdxFrMSqRHjHLuxq1bei6NjyO74Db2SBRVIBozUDD8VFaA1oL6o39jgARm7cI8S3T7w+be6JwJmjTJLa+OmabRYzz+2A1X3ahekJArc934goI3Vic8iin5K9HZ+hU7noyv8IJh5sjncAePSbCy1oNPh7uj2sHm8iE2X6icIr68u0J7dJVtikDXDvKqk+Xr+DcvtqOtB+VUZGFO9eiy2bSKu4hRjIvNvJiF9VYrOeRVr3FuInrQ3eqwKOsyUfkPBL9EniKvI+LO\\\\\\\/Mj28EN9FCsgqfDBWytJgtonG1J2GRY3Mrs2iSsYBd+sQiIcciIJ75WJKzYHUjuAbyWGLRDxfSJHc5Si8WdT1WFn0ohhkf6j07u7IxP7w+m+Dd4ZRNgU611+Kff5+rpJEwguApllPrXA\\\\\\\/R+MdF7UAkqmF9KSNGrJec7F5UpFCRrw6mPqh5NlvWmVNa1gnh\\\\\\\/yVCxP6x6B1qcQbtIL+cXUG4RYo7r3lBmSG5cRHjlaPIVLv1AHrz0nkLCDQch2mnD\\\\\\\/0GbrR1sJjJRn8zK1n1uKEifkoQvWd1UTt0XyS8BbtuzSDQhElJeupuiUOER59ZqjurhvVZK00ZGcUIy6HzNqsdIM5vNMruaUjCmAqfPlcoD\\\\\\\/OOiOokX+DSxSURaQqcKwYj\\\\\\\/sw0JLYPCAdAWa8hN42E8CR0EIKqz9Z0lErZYZEgdZiWSvlZLtOhqRMOWDcsxSMfaOdTZZVHTRzU8Rij7iBZBI3pnrWiIE8SbBOTcpNdHL3t2YyOKPy9MeAnwWq5DS9FyuP\\\\\\\/MOnDxEFE7zhc7H7hwTGiqHVaF9JLBxJ6KGCzv6nM4Muk4sCl7aLh4\\\\\\\/j4vzLRT0t94oERY59tNzqfk2s4DQ4+Svus68Y\\\\\\\/nSDLiZyw63HfE0VFa4OtW\\\\\\\/T81ZASDEEwkGy1+B0MykokACr1JDtW\\\\\\\/iGotG29oaTYM5J3UhKGjQaFW0BiIccNePXzv9Z9\\\\\\\/XilRcA7V1Yn1V6PwNcov5PXsEx5iwCxwfFg4PY3ZW9\\\\\\\/KN8RWoRx83CkIgwhK7Gq9\\\\\\\/JfFjRFYoVJ3mYbCCDsF9I0i2rxO3sXvy77Wm2Hp4GgH3iN8rq0iWoOUAhdJwN42tTgXoH0\\\\\\\/GzVkOm1SNr\\\\\\\/Q23OAenFk6msHl8tP4TabCLvK9PoBs+57ld8874npozBF9kxCQMfg+yK+laRnMeqHr++pYDRCh\\\\\\\/tRG2xyRTIxN7CeeFJFQcHd\\\\\\\/WODZFgl+2bqM1KemNL2cXE3cCv\\\\\\\/nQHQjCVkuvhd0C5F5j8a4uUZyA\\\\\\\/TPUuFxxZX1lSTCibK4Z1Lkk\\\\\\\/a1ujDIMk8uLV\\\\\\\/xUW2M2taFB0jRCbvHvmde8dBcihWozSGOlkBDQEBkt3KPDu8vkia\\\\\\\/RvcaNhCA6L2WHUp7XFnFtvqg37\\\\\\\/Nmhr1zYSYsm2nFrv7nafilmXssILB2S5daF6SPy6Cz4CcdBTpM3Tqps5TwvEOixfxdGZlfqW9M1ibq1vVAZvC1jqvF+Cwvh3+DYuMIgbvo2gssVnD\\\\\\\/pnuV0cynZboG0jSeJBrOXD1oN1i5O5hd9uXRjmSDDRR4hLAgl8mtou449+CaBH7rhmb9NG8xPgza+EWs6yNKsw7gUDA8Tu8IPOr0q6K1ekWWyF01bK7LfjA80LfTL6\\\\\\\/0wBqY1Q7t5s5H47mdQ+aGdg6BMtHwHai5CCowwLbKS4F1w3\\\\\\\/8pUYmnmuMkOS7ok2CdqGgQi6ZBRP67zjY2zYMRLlwQ0Z0G0gk5SBGr7OrMacP7MMsJCzY70JS2mX1RHEExtzuq6L2lkGnKJc8o7KkbRlktx5Mk8TYBb\\\\\\\/6F3lR0oUQcxM0qZXwb+ksprHewzC9AXsYLC3DVg7Ul3FEGiI23lPTvBtGXWvw8Kw9pP3ApBeILeu3NtdBPhWwosioAIQfFxx7y9VeXXGfypnsKfFq44MFv0KYtwZeNVKhRXuB34X7vJXO3ssiMhNgP+2Zu84FbxPCvpdNCZHjtHhePt26qPTMKcHadEzOA+xiOAwIKrGGOLsE2\\\\\\\/5eK398FNHChS+CofnBfSzioaguFtn+9bdAmIFfaiCXyv4CrjHVIzKqyxJIyfE5iWAZ62f7G5b4\\\\\\\/PJjwkRKuXW3F0MuEZJSu\\\\\\\/S9\\\\\\\/dCnvkCu2jwgJKcWFYsGfgO48XZBDEE4lTxzmLg3qmGbTY9vYZwYzc\\\\\\\/4FzSL\\\\\\\/e\\\\\\\/GmQqXmyAIa9CwPJ8Z0cY0KotgJTxgEDOBGe+MhFT1BZJ2PHqR9UJiUU4AwIELLiwUeQV8tKP8ByyriQtEVHBCxsau+PcQG4mgvMXWdHaAVjDAECQnKUsBfNNzW9gN+J1TkIOr\\\\\\\/uqEmodhNUnUjpDDO9EICOtXObihcRyKMiAGYJlFxtqH6BN+2ORojOidIzlzwVxlQUtLfoijXTW5wiBNxElVR1sTZ\\\\\\\/b5wUQ3f4\\\\\\\/b09SaF32J0Xk1RDEXAI7XD9FEwQwlDbtcsPpE21\\\\\\\/82bcbK9s8ZtIjP2yqFBOK6dc9JVGCN+lsO2mTCNYXFKpd6VMggiM4maYcDCwZN1puJWOp6qP1Ds58BHt2kHL0K8cSUhABTp+ZHtoh0La6xfBi\\\\\\\/9lWiDOMWP\\\\\\\/0LZiBQUal1vmAHBa5EeJi6kqz3tm\\\\\\\/pXZLtJMGW8ykU4nGIlj\\\\\\\/+IOsXkGvZMhZYYkqto1XkexNywxF14uICKhLBAaD0fQaEw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022mJtYfZ9JQVXDWzzTWBPGzQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002289908908a7b92ed33b9cffd23c7548a934f67c160df993c5d20da32d5ef0424c\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00220aNNo4qpGzakOhZvBEBbPT\\\\\\\/C9dDX7Q3iR17KVFYCy4IU5G09eKTzwQVUTQtg2AQgLjCEBEwX5gUTw6a9iiLB4YC0dt07nGgLqDBUxkp2lu0ndtHHrc0tmpWdh0IniuvPq0BVmswKy6W8WAQQX3zWTwcBiiALgNyMuBAF+zeGFKKm3ohzNpHmFANCU\\\\\\\/NeFA\\\\\\\/jgi\\\\\\\/W46ENFMiUAtTCgLr4oFJpowYQnxlAOOdbcJuM3CMrwRnOy7WUv0tHDpOjZznHNr5QPElnvt\\\\\\\/\\\\\\\/PJoRa1jSwmi4nm10ftEE95RcGeULnX0jKkXFtdIorV2nPemnCBNqOmAP7+Gga01cOdYkYbYRqE1KfXynp4x\\\\\\\/rfkdTzwWFrzp3MbFzTtkOmVlAkMUK59DCc1rxTk2M1\\\\\\\/k8UsvfOEr6SnACDZM+wcL6T9pgZIl18X7\\\\\\\/EFumD14X1yJLprUcl3OMe4rqgQ8JojdmE1zsB40Fq0qIZONqtQdKJFDvx2XfFw9np55D838v6icx0+Fh8wR+VcAuRWvoR8e8dxw8aTNCw1A4vu2gGfCoxfpq86+at9o57KkFl1okrPJ6DyeweEnbR3cVsPtewEKbChsioC1eSkizPZZ5k7LeW8VTDmLZURaTU6N27C\\\\\\\/\\\\\\\/UWaYDs5Fk6eMV\\\\\\\/+0Cu81sE+0XAzLcxS6Cmuk9yP03ccbyUg4HMOefdwraCuhC9MvcFWNxc2yhy0fDuVHXnfzmtPF7K+br27sGddCe+lvwaH7REeDfOXJ5qiF2UOkm92kqGLHggdEB0LOUGXSf0joEQF9b3nXRQnQk4ORazd87zQFDs2AVj4cEm\\\\\\\/tlnJrqCrfRvtI4p0no0LUqWyf6Kk5M\\\\\\\/7JfQ16uCaTp+nKnAcbN\\\\\\\/6N\\\\\\\/8HuMKKidjmMI0HfYOxOOhKVQaej1DJQpniXF98L9ODi7p0uXg4IF6U9iP4ONlU6Qbg1tUxMR16XqIpYBTet3eWLl5UAQuCG+yuWEJKoZfmkpgZrSED1TYVYWjvrsII7rjdVrpQSDwU5CxMQEPLxJ\\\\\\\/rrtQ96lDRR0tcqgVEqahk0Y2kR0FJA5139tguufNcPSTnBEKhQCNRggjI6tEgp8NT4QYsbOjE9RgNC0BQZUfsuGJTKeJyEc+\\\\\\\/rUvacF6KCE8izqC0LzjoSNLh+LovWLjOOBNhS60z7nG2jKMghWTeFbDol\\\\\\\/eUcm6YqNiT1Rh2\\\\\\\/2W+ASXOJfpZDoyGXmTKTGujwZX0+dI+nhn7zti+d20RNWrDGe12x81MZcl8oSYHWUJmwCkYfhV1h7owFgN+rv0g49GIb7DtAwIyl91VVcm2w19cZ8ovSGtDBSXLdNJ6cbs1PYr1YivAmrUiSIGMyTmllEs5womGXtAZkCajhvxD+0lMuI\\\\\\\/Rbol3wCO1dqeuMmt+kQ6PzI8k\\\\\\\/LYQRK\\\\\\\/xz5C\\\\\\\/OJuq7hAx2dUF36nFCOCYtTU6iMUeBnfDMjlG94JEvTRfxGD+SMKoEKywAHm\\\\\\\/jOCxEjqNYGLBCPxY8FDJlLi7HynOnHbgQMpYlQ3tHEG9ih8QSHsylHRVGVX\\\\\\\/Hk\\\\\\\/NHa1aDAauq02\\\\\\\/PRogkIo1Wnefbx0nng9upISCIQ+ZRzFs3UHj4tFZpDa3KRqB4ADWv2nU8zZIyTbiRtwi5teNA2jcYjM6h+MbaS9Eo3SRATllVS\\\\\\\/UISBLEbWcCasqDKj9PpNZp8YyQkF7j6VH9OwJU5\\\\\\\/MGHEL8oXZRXGAfsg8n6cgGAbJ6cq\\\\\\\/bBWfJnORAM4iDt5UXBV5EdAIGaM5pbDbOHp0Bo+MAbej9InC67N4vTTj\\\\\\\/NLH0SQiLd\\\\\\\/c6b4O\\\\\\\/4NGWIMldgwExr9gL+lLthoCSfNaoXYAWyvlj68gOM5ojJ8OnIzkSQlC7qeJI2GtQqJTAKYj7BoAVmWhfo6PXMSwrdtf0RSbLrMUumLgpsxZ\\\\\\\/6sJVx9kOM3T+qSmRAoY\\\\\\\/lR+hJ6js4GdM9KYvg9L3sWoKPbork85P77sqzhG7aBAUOSK6zX6GK7T8s4jv8pxyZg6Jn07L7HyZaDowSnQrt5aJGANKYhQcd33cg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022DaJvni\\\\\\\/tkNptrYa5dkYkRA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022afd0ccd78a4d08731d46a9761e96521919883527f4978d30816e0187e4bee7bb\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459925 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/7/1193" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-h_2T2FSU6dx5KlspueZu3_icm3sTFZM_v3tTOxiBSrM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/7/1193" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Ri2IiECjJvKGxLJjfHIoiNVG58Nvyv-Sf28D2bseToY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/7/1183" title="The Insertion Green Monster (iGM) Method for Expression of Multiple Exogenous Genes in Yeast" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/7/1205" title="Mapping Small Effect Mutations in Saccharomyces cerevisiae: Impacts of Experimental Design and Mutational Properties" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459925" id="node459925" data-pisa="ggg;4/7/1193" data-pisa-master="ggg;g3.114.010561" data-apath="/ggg/4/7/1193.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Two Genomic Regions Contribute Disproportionately to Geographic Differentiation in Wild Barley</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhou Fang</span>, <span class="highwire-citation-author" data-delta="1">Ana M. Gonzales</span>, <span class="highwire-citation-author" data-delta="2">Michael T. Clegg</span>, <span class="highwire-citation-author" data-delta="3">Kevin P. Smith</span>, <span class="highwire-citation-author" data-delta="4">Gary J. Muehlbauer</span>, <span class="highwire-citation-author" data-delta="5">Brian J. Steffenson</span> and <span class="highwire-citation-author" data-delta="6">Peter L. Morrell</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1193-1203;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010561 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459925" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhou Fang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhou%2BFang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhou%2BFang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ana M. Gonzales </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ana%2BM.%2BGonzales%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gonzales%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAna%2BM.%2BGonzales%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michael T. Clegg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Ecology and Evolutionary Biology, University of California, Irvine, California 92697</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BT.%2BClegg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clegg%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BT.%2BClegg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kevin P. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BP.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20KP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKevin%2BP.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Gary J. Muehlbauer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BJ.%2BMuehlbauer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Muehlbauer%20GJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BJ.%2BMuehlbauer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Brian J. Steffenson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BJ.%2BSteffenson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steffenson%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrian%2BJ.%2BSteffenson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Peter L. Morrell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BL.%2BMorrell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morrell%20PL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BL.%2BMorrell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:pmorrell@umn.edu" class="" data-icon-position="" data-hide-link-title="0">pmorrell@umn.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/7/1193" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459925" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459925/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/7/1193.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459925" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459925/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/7/1193.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459925" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459925/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/7/1193.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459925" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459925/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/7/1193.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459925" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459925/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Genetic differentiation in natural populations is driven by geographic distance and by ecological or physical features within and between natural habitats that reduce migration. The primary population structure in wild barley differentiates populations east and west of the Zagros Mountains. Genetic differentiation between eastern and western populations is uneven across the genome and is greatest on linkage groups 2H and 5H. Genetic markers in these two regions demonstrate the largest difference in frequency between the primary populations and have the highest informativeness for assignment to each population. Previous cytological and genetic studies suggest there are chromosomal structural rearrangements (inversions or translocations) in these genomic regions. Environmental association analyses identified an association with both temperature and precipitation variables on 2H and with precipitation variables on 5H.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/environmental-association" class="hw-term hw-article-keyword hw-article-keyword-environmental-association" rel="nofollow">environmental association</a></li><li class="kwd"><a href="/keyword/local-adaptation" class="hw-term hw-article-keyword hw-article-keyword-local-adaptation" rel="nofollow">local adaptation</a></li><li class="kwd"><a href="/keyword/population-structure" class="hw-term hw-article-keyword hw-article-keyword-population-structure" rel="nofollow">population structure</a></li><li class="kwd"><a href="/keyword/chromosome-structural-variation" class="hw-term hw-article-keyword hw-article-keyword-chromosome-structural-variation" rel="nofollow">chromosome structural variation</a></li><li class="kwd"><a href="/keyword/wild-barley" class="hw-term hw-article-keyword hw-article-keyword-wild-barley" rel="nofollow">wild barley</a></li></ul><p id="p-4">The wild progenitors of major crops have long been recognized as a valuable genetic resource (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Harris 1990</a>). Natural populations of crop wild relatives have the potential to serve as a source of alleles that contribute to valuable agronomic traits, including cold or drought tolerance (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Volis <em>et al.</em> 2002a</a>, <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">2004</a>), improved disease resistance (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Fetch <em>et al.</em> 2003</a>), and yield increase (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cohen and Galinat 1984</a>). Plant germplasm repositories have made substantial investments in the preservation of accessions of crops and their wild relatives (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Schoen and Brown 2001</a>). Modern genetic approaches have increased the value of these resources as quantitative trait locus (QTL) mapping, association studies, and molecular population genetic studies have combined to uncover specific alleles associated with traits of potential value for crop improvement (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Ross-Ibarra <em>et al.</em> 2007</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Takeda and Matsuoka 2008</a>).</p><p id="p-5">Wild barley presents an especially valuable source of potentially useful genes because of its broad geographic distribution and ecological adaptation, spanning ∼3500 km east to west from the Levant (the eastern Mediterranean) and Anatolia (present day Turkey) to Central Asia, occurring across much of Southwestern Asia. <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Volis <em>et al.</em> (2002c)</a> identified four wild barley ecotypes that exhibit differences in their level and patterns of phenotypic plasticity. In common garden studies, water stress caused a greater plastic response in the desert ecotype than in the Mediterranean ecotype, whereas for nutrient stress, plasticity was higher in the Mediterranean ecotype than in the desert ecotype (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Volis <em>et al.</em> 2002b</a>). The observed differences in ecotype plasticity or local variation in phenotypic traits suggest environmentally induced local adaptation (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Volis <em>et al.</em> 2000</a>, <a id="xref-ref-82-2" class="xref-bibr" href="#ref-82">2002c</a>). Moreover, alleles from wild barley (<em>Hordeum vulgare</em> ssp. <em>spontaneum</em>) have been used in barley breeding programs for cultivated barley improvement (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Von Korff <em>et al.</em> 2005</a>, <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">2008</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Schmalenbach <em>et al.</em> 2009</a>). QTL analyses in an advanced backcross double haploid population derived from a cross between a barley cultivar and a wild barley accession demonstrates that wild barley harbors valuable alleles that can improve yield (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Von Korff <em>et al.</em> 2006</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Schmalenbach <em>et al.</em> 2009</a>) and malting quality traits (<a id="xref-ref-86-2" class="xref-bibr" href="#ref-86">Von Korff <em>et al.</em> 2008</a>) and strongly reduce disease symptoms (<a id="xref-ref-84-2" class="xref-bibr" href="#ref-84">Von Korff <em>et al.</em> 2005</a>). QTL analysis of a recombinant inbred line population and an advanced backcross population derived from crosses between a barley cultivar and a wild barley accession reveals that the wild barley accession contains alleles that confer resistance to multiple fungal pathogens (<a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Yun <em>et al.</em> 2005</a>, <a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">2006</a>). Recently, association and candidate gene resequencing studies have begun to uncover evidence that alleles contributing to agronomically important phenotypes, such as flowering time, may have been introduced from geographic regions outside the initial region of barley domestication, with locally adaptive variants contributing to successful cultivation at higher latitudes (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jones <em>et al.</em> 2008</a>).</p><p id="p-6">Examination of population structure in crop wild progenitors can result in a better understanding of the number and geographic region of domestication events (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Morrell and Clegg 2007</a>), which is fundamental to understanding the processes that drove human domestication of plants. The potential to fully exploit genetic variation in the wild relatives of a crop depends, in part, on determining which portions of the range of wild progenitors have and have not contributed to diversity in the domesticates (<a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Zohary and Hopf 2000</a>).</p><p id="p-7">A number of previous studies have examined wild barley genetic diversity (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Nevo <em>et al.</em> 1986</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Morrell and Clegg 2007</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Jakob <em>et al.</em> 2014</a>). Analysis of 27 allozyme loci in 2125 individuals sampled from Israel, Turkey, and Iran suggested geographic differentiation in wild barley populations (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Nevo <em>et al.</em> 1986</a>) but did not sample the complete geographic range of wild barley. Genetic assignment analysis based on resequencing of 18 loci in a sample of 25 to 45 wild barley individuals identified a primary geographic partition of samples into regions east and west of the Zagros Mountains (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Morrell and Clegg 2007</a>). The assignment analysis was limited by sample size but suggested that other geographically distinct wild barley subpopulations might be identified in a larger sample (<a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Morrell and Clegg 2007</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Saisho and Purugganan 2007</a>).</p><p id="p-8">The natural range of wild barley includes a variety of environmental conditions, from arid regions in Central Asia and the Syrian desert to coastal regions along the Mediterranean with relatively high rainfall, and from the cold environments in the Zagros Mountains, the western reaches of the Himalayas, and the Iranian plateau to relatively warm lowland Mediterranean coastal regions. By associating the geographic distribution of sequence polymorphisms in georeferenced samples with environmental variables, genomic regions and individual polymorphisms correlated with differences in drought or cold tolerance or other environmental factors may be identified for further investigation.</p><p id="p-9">The questions we seek to explore in this article are what ecological factors are most associated with observed population structure, how does the geographic distribution of populations relate to allele frequency differentiation genome-wide, and how can we use this information to improve conservation and utilization of wild barley genetic diversity? We report the examination of geographic structure and genetic differentiation using 3072 SNPs (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Close <em>et al.</em> 2009</a>) genotyped in a sample of 318 wild barley accessions. We detect two primary populations of wild barley separated by the Zagros Mountains and three subpopulations within each of the two primary populations. Comparison of the two primary populations reveals two pericentromeric regions on the long arms of 2H and 5H that are associated with a substantial fraction of the geographic differentiation in allele frequencies observed in wild barley. The genetic variation in these genomic regions suggests cryptic chromosomal structural rearrangements. Environmental association analyses reveal strong association between these genomic regions and precipitation or temperature.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant materials</h3><p id="p-10">The 318 sampled wild barley accessions are known as the Wild Barley Diversity Collection (WBDC) (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Steffenson <em>et al.</em> 2007</a>). WBDC accessions were selected to be representative of the geographic range of wild barley, accounting for multiple ecogeographic features (<em>e.g.</em>, latitude, elevation, temperature range, and rainfall). The majority of accessions (77.4%) are from the Fertile Crescent, with the balance from Central Asia (15.7%), North Africa (3.8%), and the Caucasus region (2.8%). Individual accessions were self-fertilized for three generations to create inbred lines. Accession numbers, latitude, and longitude information for each sample are in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010561/-/DC1/010561SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010561/-/DC1/TableS1.pdf">Table S1</a>.</p></div><div id="sec-3" class="subsection"><h3>Genotypic data</h3><p id="p-11">The WBDC accessions were genotyped using the Illumina Golden Gate Genotyping Assay with two Barley Oligo Pool Assay (BOPA) chips (BOPA1 and BOPA2), each including 1536 SNPs (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Close <em>et al.</em> 2009</a>). The SNPs were discovered by comparison of DNA sequence from expressed sequence tags and sequenced PCR amplicons, derived principally from one wild barley accession and eight malting barley cultivars, primarily from Europe and the United States (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Close <em>et al.</em> 2009</a>).</p><p id="p-12">The program ALCHEMY (<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Wright <em>et al.</em> 2010</a>) was used to generate machine-scored, automated genotype calls. The program incorporates estimated inbreeding coefficients for each sample to improve accuracy of genotype estimation. Unlike programs such as GenomeStudio, ALCHEMY does not assume Hardy-Weinberg Equilibrium (HWE) genotypic frequencies at each SNP. ALCHEMY is based on a Bayesian model of the raw intensity data and can accurately call genotypes. We used three approaches to verify the accuracy of genotype calls. First, WBDC355 (OUH602) has been genotyped separately, with variants segregating in a mapping population (OUH602 by the cultivar Haruna Nijo) (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Sato <em>et al.</em> 2009</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Muñoz-Amatriaín <em>et al.</em> 2011</a>). Second, genotypes from two lines (WBDC218 and WBDC228) were estimated from RNA-Seq (see below for details of RNA-Seq data processing), so these data were used for validation. Third, all SNPs on BOPA1 were also called manually in GenomeStudio. Only 5% of SNPs have posterior probability of genotype calls &lt;0.95 from ALCHEMY. We considered these SNPs as missing data.</p><p id="p-13">Before running ALCHEMY, SNPs in BOPA1 and BOPA2 with strong compression or multiple clusters were removed. Subsequent to initial SNP calling, the following quality control steps were applied to the genotyping data. First, we eliminated SNPs that were monomorphic in the wild barley sample. Second, we removed all SNPs that included ≥15% missing data based on the rationale that large amounts of missing data at a SNP could be associated with inaccurate genotypes. Finally, observed heterozygosity was used as an additional quality control measure. Wild barley is a highly self-fertilizing species (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Brown <em>et al.</em> 1978</a>) and the WBDC accessions have been subject to three rounds of inbreeding. SNPs with observed heterozygosity &gt;10% were removed on the rationale that the genotypes were likely in error. SNPs with centromeric genetic map positions were identified based on the consensus genetic map of <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Muñoz-Amatriaín <em>et al.</em> (2011)</a>. In the inbred WBDC lines, observed heterozygosity was extremely low (0.2%), thus we treat the data as haploid genotypes.</p><p id="p-14">SNPs were annotated using the program SNPMeta (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kono <em>et al.</em> 2014</a>). Annotation for each SNP included GenBank ID, gene short name, whether the SNP occurs in coding or noncoding sequence, the SNP position within a codon, and determination of whether the SNP is silent or induces an amino acid replacement. Among the 3072 BOPA SNPs, 2508 were annotated, with 338 annotations derived from named genes.</p><p id="p-15">Along with SNP data, we examined 29 microsatellite loci in all WBDC accessions (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Ramsay <em>et al.</em> 2000</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Li <em>et al.</em> 2003</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Varshney <em>et al.</em> 2007</a>). Microsatellites offer the advantage of reduced ascertainment bias, because the microsatellites are selected to be polymorphic, but the selection process is not conditional on the presence of an individual polymorphism (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Haasl and Payseur 2010</a>). Microsatellite locus names, repeat number, repeat unit length, total size, and heterozygosity for each microsatellite can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010561/-/DC1/TableS2.pdf">Table S2</a>. This information was obtained from GrainGenes: A Database for Triticeae and <em>Avena</em> (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Matthews <em>et al.</em> 2003</a>). The program <em>R</em><sub>ST</sub> Calc (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Goodman 1997</a>) was used to compare allele frequency differences among partitions of the sample, assuming a stepwise mutation model (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Slatkin 1995</a>).</p></div><div id="sec-4" class="subsection"><h3>RNA-Seq data processing</h3><p id="p-16">RNA-Seq reads from <em>Hordeum bulbosum</em> accession Cb2920/4 (used as outgroup to infer ancestral state) and wild barley WBDC218 and WBDC228 were trimmed of adapter contamination with Scythe (<a href="https://github.com/vsbuffalo/scythe">https://github.com/vsbuffalo/scythe</a>) and then aligned to the Morex draft sequence (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Mayer <em>et al.</em> 2012</a>) with Bowtie 2 (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Langmead and Salzberg 2012</a>). We adjusted read mapping parameters to accommodate the expected divergence between <em>H. vulgare</em> lines (∼1%) and between <em>H. vulgare</em> and <em>H. bulbosum</em> (∼3%) (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Morrell <em>et al.</em> 2014</a>). Alignments were processed with Samtools (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Li <em>et al.</em> 2009</a>) and the Genome Analysis Toolkit (GATK) (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">McKenna <em>et al.</em> 2010</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">DePristo <em>et al.</em> 2011</a>) according to GATK best practices (<a href="http://www.broadinstitute.org/gatk/guide/best-practices">http://www.broadinstitute.org/gatk/guide/best-practices</a>). For realignment around indels, we used a set of high-confidence indels reported in Sanger resequencing datasets (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Caldwell <em>et al.</em> 2006</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Morrell <em>et al.</em> 2006</a>; <a id="xref-ref-43-5" class="xref-bibr" href="#ref-43">Morrell and Clegg 2007</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Morrell <em>et al.</em> 2014</a>). We then extracted the base calls at each BOPA SNP location using tools from the GATK. For all SNPs where it can be inferred, the ancestral state is listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010561/-/DC1/TableS3.txt">Table S3</a>.</p></div><div id="sec-5" class="subsection"><h3>Geographic differentiation</h3><p id="p-17">We examined the geographic structure within the range of wild barley with Bayesian genetic assignment implemented in the program STRUCTURE (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Pritchard <em>et al.</em> 2000</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Falush <em>et al.</em> 2003</a>). We treated individual samples as haploid and explored both admixture and no admixture models and models with correlated and uncorrelated allele frequencies with <em>K</em> = 2 − 10 clusters. Because a model with no admixture and uncorrelated allele frequencies resulted in higher likelihoods, it was used for the final analyses. For each value of <em>K</em>, we used 10 replicate runs with a burn-in length of 100,000 iterations and a run length of 100,000 iterations.</p><p id="p-18">Haplotypes were defined based on five adjacent SNPs and used both for comparison of haplotype diversity and as input for genetic assignment analysis. Combining markers into haplotypes results in multi-allelic data that can improve inference of population structure (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Haasl and Payseur 2010</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gattepaille and Jakobsson 2012</a>). To infer missing data, we used the program fastPHASE (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Scheet and Stephens 2006</a>) with 20 random starts and 25 iterations of the Expectation-Maximization algorithm. The number of SNPs within each haplotype was determined based on optimal numbers from the simulation study of <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Gattepaille and Jakobsson (2012)</a> and the total number of SNPs in our samples. To deal with label switching (where cluster names change between replicate runs) and with true multimodality (where individual samples switch clusters in replicate runs), we used CLUMPP (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Jakobsson and Rosenberg 2007</a>) to summarize assignment results across replicate runs. The program Infocalc was used to calculate the informativeness for assignment (<em>In</em>) (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Rosenberg <em>et al.</em> 2003</a>) for each haplotype using the clusters identified by STRUCTURE. Informativeness for assignment identifies the information content for genetic assignment for markers based on the degree to which each locus (or haplotype) contributes to discrimination among populations (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Rosenberg <em>et al.</em> 2003</a>).</p><p id="p-19">The prcomp function in R (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">R Development Core Team 2012</a>) was used to perform principal component analysis (PCA). Each WBDC accession was assigned to clusters based on significant principal components (PCs) using the Ward clustering method in the R hclust function. To compare the similarity between genetic variation on a PCA plot and geographic maps of sample locations, we used Procrustes analysis to find a rotation that maximizes the similarity (<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Wang <em>et al.</em> 2012</a>).</p><p id="p-20">Within and among inferred clusters, we estimated hierarchical F-statistics (<a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Yang 1998</a>) using the Hierfstat package (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Goudet 2005</a>) in R. We calculated summary statistics, including number of segregating sites, and number of private alleles in each cluster using tools from the libsequence library (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Thornton 2003</a>) and estimated average pairwise SNP diversity within each cluster using the R package ape (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Paradis <em>et al.</em> 2004</a>). Rarefaction was used to analyze allelic diversity across populations while correcting for sample size differences using the program ADZE (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Szpiech <em>et al.</em> 2008</a>).</p><p id="p-21">Linkage disequilibrium (LD) as measured by <em>r</em><sup>2</sup> (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hill and Robertson 1968</a>) was calculated for all possible pairwise comparisons on each linkage group based on SNPs with minor allele frequency (MAF) &gt;5%. The LDheatmap package (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Shin <em>et al.</em> 2006</a>) was used to generate plots of LD relative to genetic distance.</p></div><div id="sec-6" class="subsection"><h3>Environmental–genetic correlations</h3><p id="p-22">To identify genomic segments potentially contributing to local adaptation, we divided samples into the two primary clusters (the Eastern and Western populations) and six clusters identified by genetic assignment in STRUCTURE and calculated <em>F</em><sub>ST</sub> (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Weir and Cockerham 1984</a>). The assumption is that SNPs linked to genomic regions contributing to local adaptation will show greater allele frequency divergence (higher <em>F</em><sub>ST</sub>) than those affected only by demography (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cavalli-Sforza 1966</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lewontin and Krakauer 1973</a>).</p><p id="p-23">Environmental variables, including altitude, monthly precipitation, monthly maximum and minimum temperatures, and 19 additional bioclimatic variables were downloaded from <a href="http://www.worldclim.org">www.worldclim.org</a> (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hijmans <em>et al.</em> 2005</a>). DIVA-GIS (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hijmans <em>et al.</em> 2001</a>) was used to extract climate data at 5 arc-min (∼10 km) resolution for each sample. We focused on measurements within the growing season for wild barley, from late autumn to spring, so we removed environmental variables related to summer temperature and precipitation. The environmental variables used are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010561/-/DC1/TableS4.pdf">Table S4</a>A. Environmental variables were scaled to a mean of 0 and SD of 1 and summarized into principal components. The significant PCs of environmental variables were used for Bayenv (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Coop <em>et al.</em> 2010</a>).</p><p id="p-24">We used Bayenv to identify the correlation between SNPs and environmental variables. Bayenv requires population information to account for population structure, making use of allele frequency within a set of samples representing a localized environment to correct for population structure in environmental association. We used PCA to group all samples into 17 clusters with a mean sample size of 17 accessions. The number of optimal stratifications in our data was determined using Velicer’s minimum average partial test (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Shriner 2011</a>). We used all SNPs to construct the covariance matrix. Two independent runs of 30,000 iterations were compared to control for convergence and the final covariance matrix is the mean of these two independent runs. Bayenv was used to estimate the Bayes factor for each SNP with each environmental variable using 50,000 iterations. SNPs were considered candidates contributing to local adaptation if they had an average Bayes factor above the 95<sup>th</sup> percentile genome-wide for five separate runs.</p><p id="p-25">Because the wild barley samples cover a large continuous geographic range, spatial ancestry analysis (SPA) (<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Yang <em>et al.</em> 2012</a>) was used to model allele frequency change for each SNP as a function of the location of the individual in geographic space. SNPs contributing to local adaptation potentially have larger gradients in allele frequency, reflected in a high SPA score (<a id="xref-ref-92-2" class="xref-bibr" href="#ref-92">Yang <em>et al.</em> 2012</a>). SNPs were considered candidates for local adaptation if their SPA score was above the 95<sup>th</sup> percentile genome-wide.</p><p id="p-26">SNPs that are outliers (above the 95<sup>th</sup> percentile genome-wide) in the <em>F</em><sub>ST</sub>, environmental association, or SPA analyses can be considered candidate variants either causative or, more likely, linked to loci involved in local adaptation. Enrichment analysis of candidates among genic <em>vs.</em> nongenic and nonsynonymous <em>vs.</em> synonymous was performed by resampling the number of SNPs in each candidate list randomly from the genome 1000 times. Enrichment analyses included sets of SNPs genome-wide, centromeric regions, and the two regions that are putative structural rearrangements (inversions or translocations). This generated a distribution of ratios of the number of genic to the number of nongenic SNPs and ratios of the number of nonsynonymous to the number of synonymous SNPs in each analysis, which can be compared to the observed ratios from the data.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><p id="p-27">Initial screening of relatedness among accessions identified 30 wild barley accessions either with a large genetic distance from the majority of accessions or that appear to be duplicated within the sample. The large genetic distance was also associated with a high degree of identity to barley landraces genotyped on the same platform. The increase in genetic distance appears to result from ascertainment bias due to the discovery of barley SNPs primarily among cultivated samples, which results in a larger number of segregating polymorphisms in barley landrace accessions than in wild barley (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Russell <em>et al.</em> 2011</a>). The 30 accessions were excluded from further analysis because they constitute duplicate accessions or because genotypic composition suggests they either could be feral barley accessions or were subject to recent introgression. Four accessions do not have known latitude and longitude of origin, so they were removed from analyses; thus, 284 wild barley accessions from the WBDC were used in this study (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010561/-/DC1/TableS1.pdf">Table S1</a>). After all quality control measures, 2330 SNPs were assayed in each accession (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010561/-/DC1/TableS5.pdf">Table S5</a>).</p><p id="p-28">Microsatellite loci were extremely polymorphic among wild barley accessions with an average of 20 alleles per locus. Allele size for individual microsatellites is normally distributed and thus accords with a stepwise mutation model (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Ohta and Kimura 1973</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Valdes <em>et al.</em> 1993</a>).</p><div id="sec-8" class="subsection"><h3>Population structure</h3><p id="p-29">A Mantel test identifies a positive correlation between geographic distance and genetic distance (Mantel statistic: 0.37; significance at 0.001), consistent with isolation by distance among these wild barley accessions.</p><p id="p-30">STRUCTURE results based on single-SNP and 5-SNP haplotypes for <em>K</em> = 2 identified populations east and west of the Zagros Mountains. Samples from a broad portion of the range, including Central Asia (particularly east of the Caspian Sea), the Iranian Plateau, and most samples in Northern Mesopotamia (modern Northern Iraq and Syria) form an Eastern population, and samples west of the Zagros Mountains, including samples from the Levant, and around the Mediterranean to North Africa form a Western population (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). There is also isolation by distance within the Eastern population (Mantel statistic: 0.31; significance at 0.001) and the Western population (Mantel statistic: 0.20; significance at 0.001). For genetic assignment based on individual SNPs, there are six accessions from Central Asia assigned to the Western population (data not shown). Thus, genetic assignment from haplotype data shows greater consistency with geographic location of origin. The broad-scale geographic patterns identified from SNP and 5-SNP haplotypes are not readily reflected in genetic assignment based on the 29 microsatellite loci (data not shown). The individual alleles are relatively rare and have low informativeness for assignment, an issue attributable to the high levels of polymorphism in these microsatellites (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010561/-/DC1/TableS2.pdf">Table S2</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/7/1193/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Population structure in wild barley. Each of the six colors represents one of six subpopulations. There are three subpopulations nested within both the Eastern and Western populations. (B) Procrustes-transformed PCA plot of genetic variation in wild barley." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-758118968" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Population structure in wild barley. Each of the six colors represents one of six subpopulations. There are three subpopulations nested within both the Eastern and Western populations. (B) Procrustes-transformed PCA plot of genetic variation in wild barley.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/7/1193/F1.medium.gif" width="440" height="431"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/7/1193/F1.medium.gif" width="440" height="431"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/7/1193/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/7/1193/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464660" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-31" class="first-child">(A) Population structure in wild barley. Each of the six colors represents one of six subpopulations. There are three subpopulations nested within both the Eastern and Western populations. (B) Procrustes-transformed PCA plot of genetic variation in wild barley.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">For STRUCTURE analysis based on haplotype data, when <em>K</em> = 3, a group of accessions from the Syrian desert region becomes an independent cluster from the Western population. As <em>K</em> increases to 4, the samples along the east coast of the Mediterranean split into two groups, one in the north (Northern Levant) and the other in the south (Southern Levant). The Eastern population begins to differentiate as <em>K</em> is increased to 5. The samples in Central Asia are separated from samples from Northern Mesopotamia and those from west of the Caspian Sea. With <em>K</em> = 6, the seven samples from the Caspian Sea become an independent cluster (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). For the present sample, <em>K</em> = 6 provides a clear distinction among populations and the genetic assignment is not constrained by very small sample size within individual clusters. The average informativeness for assignment for individual SNPs is 0.03 for <em>K</em> = 2 and 0.10 for <em>K</em> = 6, increasing to 0.14 for <em>K</em> = 2 and 0.47 for <em>K</em> = 6 when based on haplotypes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010561/-/DC1/FigureS1.pdf">Figure S1</a>).</p><p id="p-33">Among the six clusters, three are within the Eastern population (Central Asia, Caspian Sea, and Northern Mesopotamia) and three are in the Western population (Northern Levant, Syrian Desert, and Southern Levant), constituting hierarchical population structure nested within the major Eastern and Western populations. The comparative <em>F</em><sub>ST</sub> analyses identify both a significant effect of individuals within subpopulations (p-value = 0.01; 100 permutations; observed likelihood ratio statistics = 94,259.55) and a significant effect of subpopulations within populations (p-value = 0.04; 100 permutations; observed likelihood ratio statistics = 33,632.57). Population structure is best explained by a six-subpopulation model (variance components = 17.7%); in contrast, the two-population model provides a poorer fit to the data (variance components = 10.6%). <em>F</em><sub>ST</sub> values are also higher at the subpopulation level than the population level (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Within the two higher-level populations, 20.5% of the genetic differentiation can be explained by the three subpopulations within the Eastern population, whereas 13.2% can be explained by the three subpopulations within the Western population.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464553/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464553/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Hierarchical F-statistics comparing different levels of the hierarchical population structure</span><div class="sb-div caption-clear"></div></div></div><p id="p-35">In the PCA examination of population structure, the first PC separates all samples into the Eastern and Western populations. When adding the second PC, all six subpopulations are clearly differentiated. A PCA plot of genetic variation closely reflects geography in the population after a 42.44° counterclockwise rotation of the PCA plot using Procrustes analysis. The boundary of the Eastern and Western population approximately parallels the Zagros Mountains (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). The Syrian Desert subpopulation is noteworthy in showing greater PC distance within and among samples, possibly because of greater genetic drift within this subpopulation. A mean pairwise <em>F</em><sub>ST</sub> analysis indicates a higher mean <em>F</em><sub>ST</sub> value for this subpopulation (0.09) than for the other two Western subpopulations (0.05 and 0.04).</p></div><div id="sec-9" class="subsection"><h3>Population differentiation</h3><p id="p-36">The average genome-wide <em>F</em><sub>ST</sub> between the Eastern and Western populations is 0.07. The boundary of these two populations is close to the Zagros Mountains (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>), which forms a potential barrier to migration (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lin <em>et al.</em> 2001</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Morrell <em>et al.</em> 2003</a>). The most northerly portion of the Zagros Range is at ∼48° E longitude, trending from the northwest to the southeast, so we also compared the allele frequency differentiation between the two populations east and west of 48° E. For this geographic contrast, the average <em>F</em><sub>ST</sub> genome-wide is 0.06 and the correlation coefficient (<em>r</em>) between this partition based on the Zagros Mountains and the previous partition based on genetic assignment analysis is 0.473, which supports the hypothesis that the Zagros Mountains act as a natural barrier that bisects wild barley into the Eastern and Western populations. For the 29 microsatellite loci, the average <em>R</em><sub>ST</sub> between the Eastern and Western populations is 0.15.</p><p id="p-37">SNPs with high <em>F</em><sub>ST</sub> values between the Eastern and Western populations (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>) or among all the six subpopulations (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>) are most abundant in two genomic regions, one on linkage group 2H, from ∼67 to 74 cM, and the other on 5H, from ∼47 to 52 cM. For the 2H and 5H regions, the mean <em>F</em><sub>ST</sub>s are 0.20 (56 SNPs) and 0.17 (32 SNPs), respectively, <em>vs.</em> a genome-wide <em>F</em><sub>ST</sub> = 0.07.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/7/1193/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) FST between the Eastern and Western wild barley populations. (B) Pairwise FST based on all six subpopulations. The dotted line is the 95th percentile of FST genome-wide. (C) Bayes factors for correlation between allele frequencies and PC1. (D) Bayes factors for correlation between allele frequencies and PC2. (E) SPA score genome-wide from spatial analysis. The 95th percentile of the distribution of Bayes factors or SPA scores is indicated by a horizontal dashed line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-758118968" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;ST&lt;/sub&gt; between the Eastern and Western wild barley populations. (B) Pairwise &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;ST&lt;/sub&gt; based on all six subpopulations. The dotted line is the 95&lt;sup&gt;th&lt;/sup&gt; percentile of &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;ST&lt;/sub&gt; genome-wide. (C) Bayes factors for correlation between allele frequencies and PC1. (D) Bayes factors for correlation between allele frequencies and PC2. (E) SPA score genome-wide from spatial analysis. The 95&lt;sup&gt;th&lt;/sup&gt; percentile of the distribution of Bayes factors or SPA scores is indicated by a horizontal dashed line.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/7/1193/F2.medium.gif" width="440" height="434"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/7/1193/F2.medium.gif" width="440" height="434"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/7/1193/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/7/1193/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464521" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-38" class="first-child">(A) <em>F</em><sub>ST</sub> between the Eastern and Western wild barley populations. (B) Pairwise <em>F</em><sub>ST</sub> based on all six subpopulations. The dotted line is the 95<sup>th</sup> percentile of <em>F</em><sub>ST</sub> genome-wide. (C) Bayes factors for correlation between allele frequencies and PC1. (D) Bayes factors for correlation between allele frequencies and PC2. (E) SPA score genome-wide from spatial analysis. The 95<sup>th</sup> percentile of the distribution of Bayes factors or SPA scores is indicated by a horizontal dashed line.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">Many SNPs or haplotypes genome-wide with high informativeness for assignment are in these two high <em>F</em><sub>ST</sub> regions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010561/-/DC1/FigureS1.pdf">Figure S1</a>). Among all SNPs above the 95<sup>th</sup> and 99<sup>th</sup> percentiles (<em>In</em> = 0.31 and 0.42), 33% and 52% are in these two regions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010561/-/DC1/FigureS1.pdf">Figure S1</a>). However, when we perform genetic assignment analysis after masking these two high <em>F</em><sub>ST</sub> regions, the probability of assignment for each wild barley accession into the Eastern and Western population is nearly identical for all but one accession. This result reflects the relatively high informativeness for assignment observed for SNPs within pericentromeric regions on all linkage groups (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010561/-/DC1/FigureS1.pdf">Figure S1</a>). Therefore, population structure in these two high <em>F</em><sub>ST</sub> regions is similar to the genome-wide pattern and the genome-wide pattern is driven by a high degree of differentiation in pericentromeric regions.</p><p id="p-40">The joint unfolded site frequency spectrum demonstrates that there are more rare variants in the Western population than in the Eastern population (<a href="http://www.g3journal.orglookup/suppl/doi:10.1534/g3.114.010561/-/DC1/FigureS2.pdf">Figure S2</a>). Percent pairwise differences are lower in the Eastern population than in the Western population (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). There are more private SNPs in the Western population than in the Eastern population (430 <em>vs.</em> 86) (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Because the sample size is different between the Eastern and Western populations, we used rarefaction to correct for sample size. Despite the correction, both the mean number of distinct alleles per locus (<a href="http://www.g3journal.orglookup/suppl/doi:10.1534/g3.114.010561/-/DC1/FigureS3.pdf">Figure S3</a>A) and the mean number of private alleles per locus (<a href="http://www.g3journal.orglookup/suppl/doi:10.1534/g3.114.010561/-/DC1/FigureS3.pdf">Figure S3</a>B) are higher in the Western than the Eastern population. The Southern Levant subpopulation has the highest values for the number of segregating sites and number of private SNPs, whereas the Caspian Sea subpopulation has the lowest values for these summary statistics (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464582/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464582/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Diversity summary statistics for the two populations and six subpopulations</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Structural rearrangements</h3><p id="p-42">The high <em>F</em><sub>ST</sub> regions on 2H and 5H are potentially attributable to chromosomal structural variants. Population genetic variation, particularly patterns of LD, can suggest structural variation (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Huynh <em>et al.</em> 2011</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Long <em>et al.</em> 2013</a>). The average pairwise LD in the high <em>F</em><sub>ST</sub> region on 2H (<em>r</em><sup>2</sup> = 0.085) is higher than other regions of 2H (<em>r</em><sup>2</sup> = 0.018) and the adjacent regions with similar size left and right of the high <em>F</em><sub>ST</sub> region, where <em>r</em><sup>2</sup> = 0.021 for both regions. The average pairwise LD in the high <em>F</em><sub>ST</sub> region on 5H (<em>r</em><sup>2</sup> = 0.097) is also higher than other regions on 5H (<em>r</em><sup>2</sup> = 0.019) and the adjacent regions with similar size to the left and right of the high <em>F</em><sub>ST</sub> region (<em>r</em><sup>2</sup> = 0.017 and 0.020). The 5-SNP segment with the lowest haplotype number (4) on 2H is within the high <em>F</em><sub>ST</sub> region (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010561/-/DC1/FigureS4.pdf">Figure S4</a>). As on other linkage groups, the segment with the lowest haplotype number on 2H and 5H is within the centromeric region (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010561/-/DC1/FigureS4.pdf">Figure S4</a>). The centromeric region on 2H overlaps with the high <em>F</em><sub>ST</sub> region, whereas the centromere is distal to the high <em>F</em><sub>ST</sub> region on 5H (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010561/-/DC1/FigureS4.pdf">Figure S4</a>). The observed LD, <em>F</em><sub>ST</sub>, and haplotype number patterns are consistent with recent positive selection and/or chromosome structural rearrangements. There are more unique haplotypes in the Western population than in the Eastern population (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). In the Eastern population, both of the two regions on 2H and 5H are dominated by a few haplotypes (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>), which is potentially consistent with selection favoring these haplotypes.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/7/1193/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Diagram of haplotype diversity in the two putative chromosome structural rearrangements on 2H and 5H. Haplotypes are divided into the two primary populations identified by genetic assignment. Each SNP is represented by either the ancestral state (black) or the derived state (gray). The frequencies of each of the haplotypes from the Eastern population (top) and from the Western population (bottom) are shown on the right. FSTs for each SNP between these two populations are shown at the top of the haplotype diagram." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-758118968" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Diagram of haplotype diversity in the two putative chromosome structural rearrangements on 2H and 5H. Haplotypes are divided into the two primary populations identified by genetic assignment. Each SNP is represented by either the ancestral state (black) or the derived state (gray). The frequencies of each of the haplotypes from the Eastern population (top) and from the Western population (bottom) are shown on the right. &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;ST&lt;/sub&gt;s for each SNP between these two populations are shown at the top of the haplotype diagram.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/7/1193/F3.medium.gif" width="440" height="388"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/7/1193/F3.medium.gif" width="440" height="388"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/7/1193/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/7/1193/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464651" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-43" class="first-child">Diagram of haplotype diversity in the two putative chromosome structural rearrangements on 2H and 5H. Haplotypes are divided into the two primary populations identified by genetic assignment. Each SNP is represented by either the ancestral state (black) or the derived state (gray). The frequencies of each of the haplotypes from the Eastern population (top) and from the Western population (bottom) are shown on the right. <em>F</em><sub>ST</sub>s for each SNP between these two populations are shown at the top of the haplotype diagram.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Evidence for local adaptation</h3><p id="p-44">SNPs that occur as outliers in the <em>F</em><sub>ST</sub> analysis may indicate genomic regions involved in local adaptation. Annotation information for SNPs that are above the 95<sup>th</sup> percentile for <em>F</em><sub>ST</sub> between the Eastern and Western populations is listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010561/-/DC1/TableS6.pdf">Table S6</a>.</p><p id="p-45">Environmental association analysis was used to identify genetic polymorphisms potentially involved in local adaptation. PCA reveals two major clusters of environmental variables (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010561/-/DC1/FigureS5.pdf">Figure S5</a>). The first two PCs explain 80% of the total variance (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010561/-/DC1/FigureS5.pdf">Figure S5</a>). The first PC includes most temperature variables, whereas most precipitation variables and altitude are in the second PC (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010561/-/DC1/TableS4.pdf">Table S4</a>B).</p><p id="p-46">SNPs associated with both environmental PC1 and PC2 (above the 95<sup>th</sup> percentile) are distributed on all linkage groups (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2, C and D</a>). The high <em>F</em><sub>ST</sub> region on 2H is highly associated with both PC1 and PC2 (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2 C and D</a>). The high <em>F</em><sub>ST</sub> region on 5H is also associated with PC2 (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2D</a>). The annotation information for SNPs that are above the 95<sup>th</sup> percentile of association with PC1 and PC2 is listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010561/-/DC1/TableS7.pdf">Table S7</a>.</p><p id="p-47">SPA analysis reveals that ∼20% of the SNPs in the high <em>F</em><sub>ST</sub> regions on both 2H (11 out of 56) and 5H (7 out of 32) show strong geographic gradients in allele frequencies as their SPA scores are above the 95<sup>th</sup> percentile (2.72) (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2E</a>). The SNP with the highest SPA score (5.54) is in the high <em>F</em><sub>ST</sub> region on 2H. There are 24 SNPs with SPA score above the 99<sup>th</sup> percentile (3.75), and nearly half of these SNPs (11) are in these two high <em>F</em><sub>ST</sub> regions on 2H and 5H. The annotation information for the SNPs with SPA scores above the 95<sup>th</sup> percentile is in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010561/-/DC1/TableS8.pdf">Table S8</a>.</p><p id="p-48">Enrichment analysis reveals that the two putative chromosomal structural rearrangements are enriched for genic SNPs in the outliers of PC2 from environmental analysis and SPA analysis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010561/-/DC1/FigureS6.pdf">Figure S6</a>A). There is no evidence of enrichment for nonsynonymous SNPs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010561/-/DC1/FigureS6.pdf">Figure S6</a>B).</p></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><div id="sec-13" class="subsection"><h3>Hierarchical population structure</h3><p id="p-49">Wild barley shows strong hierarchical population structure, with primary structure east and west of the Zagros Mountains and three subpopulations identified in both the Eastern and Western populations (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1A</a>). Previous studies of sequence diversity in wild barley have identified population structure that strongly differentiates the Eastern and Western wild barley populations (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Lin <em>et al.</em> 2001</a>; <a id="xref-ref-43-6" class="xref-bibr" href="#ref-43">Morrell and Clegg 2007</a>; <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Saisho and Purugganan 2007</a>). The present study samples a much larger number of accessions and includes only 22 SNPs in common with those sampled in previous studies. Despite the limited overlap of sampled SNPs, genetic assignment with <em>K</em> = 2 uncovers a similar geographic pattern. Moreover, the results of this study go beyond previous work by revealing a division into six subpopulations that explains 7.1% more variance compared with the primary, two-population division.</p><p id="p-50">A nearly continuous geographic range represents a particular challenge for efforts to identify population structure and geographic discontinuities. However, populations of wild barley are much more common in the western portion of the range and below 1500 m (<a id="xref-ref-95-2" class="xref-bibr" href="#ref-95">Zohary and Hopf 2000</a>), thus the 3000-m to 4500-m peaks of the Zagros Mountains and the high elevation regions of the Iranian plateau are disruptions of an otherwise continuous range.</p><p id="p-51">The Western population is more diverse than the Eastern population (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>), at least in part because the SNP discovery panel was composed primarily of Western cultivars. It should be noted that the discovery panel also included OUH602 (WBDC355), which assigns to the Central Asia (Eastern) wild barley population. Estimates of diversity based on resequencing a more limited set of wild barley samples indicate moderately higher levels of diversity in the Western than in the Eastern wild barley populations (<a id="xref-ref-43-7" class="xref-bibr" href="#ref-43">Morrell and Clegg 2007</a>; <a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Morrell <em>et al.</em> 2014</a>). The effect of ascertainment bias on the frequency spectrum depends on the population in which SNPs were discovered (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albrechtsen <em>et al.</em> 2010</a>). Therefore, we observe more rare alleles in the Western population (<a href="http://www.g3journal.orglookup/suppl/doi:10.1534/g3.114.010561/-/DC1/FigureS2.pdf">Figure S2</a>). There is a small but clear effect of ascertainment bias, which leads to an increased estimate of diversity in the Western population, because the Western population is more similar to the discovery panel. The size of a discovery panel is less important than the composition, as long as it is not extremely small (&lt;4 chromosomes) (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Albrechtsen <em>et al.</em> 2010</a>). Based on coalescent simulations, a discovery panel of eight samples with a minimum allele count of three best-reflects the design parameters of the BOPA SNPs (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fang <em>et al.</em> 2013</a>).</p></div><div id="sec-14" class="subsection"><h3>Two putative chromosome rearrangements</h3><p id="p-52">Resequencing studies identified a large degree of heterogeneity in wild barley, in terms of both degree of population structure and levels of nucleotide sequence diversity (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Lin <em>et al.</em> 2001</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Morrell <em>et al.</em> 2003</a>). Using simple coalescent simulations, <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Morrell <em>et al.</em> (2003)</a> argued that stochastic variation alone was insufficient to explain intralocus heterogeneity and that selection, either through selective sweeps at some loci or through local adaptation at others, was necessary to explain observed patterns of diversity (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Wright <em>et al.</em> 2005</a>). Strong genetic differentiation between the Eastern and Western populations for the two regions on 2H and 5H suggest that some of the heterogeneity may result not from selection acting on individual loci, but rather on structural variants (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>). Structural variants are quickly lost due to drift and purifying selection unless they confer a locally adaptive advantage. A structural variant that captures two or more alleles adapted to the local environment has a selective advantage that can cause it to spread (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kirkpatrick and Barton 2006</a>).</p><p id="p-53">The high <em>F</em><sub>ST</sub> region on 2H occurs in the same approximate chromosomal location as a chromosomal rearrangement identified in an eastern wild barley accession based on meiotic pairing studies (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Konishi and Linde-Laursen 1988</a>). <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Konishi and Linde-Laursen (1988)</a> report a reciprocal translocation with 4H in a sample from Turkmenistan with the breakpoints of the translocation near the centromere on 2H. Using a three-point linkage test, <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Ramage and Suneson (1961)</a> identify evidence of both an inversion and translocations on the long arm of 5H.</p><p id="p-54">The two genomic regions with high <em>F</em><sub>ST</sub> also have above average levels of LD and low haplotype number (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010561/-/DC1/FigureS4.pdf">Figure S4</a>). Environmental association analysis identifies multiple SNPs in these regions associated with both temperature and precipitation (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a>). SPA analysis identifies half of the SNPs in these two regions as outliers with dramatic change in allele frequency gradients, as identified by SPA scores above the 99<sup>th</sup> percentile. The SPA method is particularly sensitive to SNPs that have steep geographic gradients in allele frequency and is scored based on individual accessions rather than populations (<a id="xref-ref-92-3" class="xref-bibr" href="#ref-92">Yang <em>et al.</em> 2012</a>).</p><p id="p-55">The large number of candidate SNPs in these regions identified in multiple analyses indicate these two regions may harbor variants that are locally adaptive, with selection altering the frequency of nearby SNPs through genetic hitchhiking (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Nielsen <em>et al.</em> 2005</a>). Given the density of SNPs assayed in the present study and the relatively rapid decay of LD in wild barley (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Morrell <em>et al.</em> 2005</a>, <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">2006</a>), the detection of multiple SNPs associated with environmental factors is likely to occur only in regions with suppressed recombination. These two high <em>F</em><sub>ST</sub> regions are 5 to 7 cM, potentially including hundreds of genes; thus, the patterns observed are likely due to chromosome structural variants that, through the inhibition of genetic exchange between chromosomal rearrangements, have the effect of slowing migration.</p><p id="p-56">The Eastern accessions occur in a region that, on average, is more arid than the region occupied by the Western accessions, where most samples were collected from populations along the Mediterranean coast. The precipitation pattern is reflected in the environmental association analysis. Both of the two putative chromosome rearrangements on 2H and 5H are associated with precipitation variables (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2</a>).</p><p id="p-57">Comparisons among genetic maps can provide confirmation of the presence of chromosomal rearrangements through the inference of alternative mapping order in populations where both mapping parents carry the alternative arrangement (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Thomas <em>et al.</em> 2008</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lowry and Willis 2010</a>) or through evidence of suppression of crossover when parents differ for the chromosomal arrangement (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fang <em>et al.</em> 2012</a>). It is possible to identify mapping parents that differ for chromosomal rearrangements based on the presence of variants (SNPs) that are private (or nearly private) to the putative rearrangement (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Fang <em>et al.</em> 2012</a>) or in cases where the geographic distribution of the putative rearrangement is well-defined, using parents from geographic regions where the rearrangement is more common. Both putative structural rearrangements in wild barley occur in the eastern portion of the wild range. Barley is a cultigen with multiple origins (<a id="xref-ref-43-8" class="xref-bibr" href="#ref-43">Morrell and Clegg 2007</a>), with a greater contribution of eastern wild barley ancestry among Asian landraces (<a id="xref-ref-43-9" class="xref-bibr" href="#ref-43">Morrell and Clegg 2007</a>; <a id="xref-ref-47-4" class="xref-bibr" href="#ref-47">Morrell <em>et al.</em> 2014</a>). Barley doubled haploid genetic mapping populations compared by <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Muñoz-Amatriaín <em>et al.</em> (2011)</a> include three populations that could potentially differ for the rearrangement: the Japanese malting barley cultivar Haruna Nijo by wild barley OUH602 (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Sato <em>et al.</em> 2009</a>); Haruna Nijo by food barley cultivar Akashinriki (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Sato <em>et al.</em> 2011</a>); and the Oregon Wolfe Barley (OWB) population (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Costa <em>et al.</em> 2001</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Stein <em>et al.</em> 2007</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Szűcs <em>et al.</em> 2009</a>).</p><p id="p-58">Examining the genomic regions on 2H and 5H with highest allele frequency differentiation and levels of LD among wild barley accessions, we identify an average of 24 SNPs on 2H and 17 SNPs on 5H segregating among mapping parents in these genetic maps. The two Haruna Nijo maps include a single inferred crossover on both 2H and 5H, whereas the OWB maps include a larger number of crossovers on 2H and two crossovers on 5H. Crossover number in these genetic maps does not differ dramatically from that observed in the same genomic regions in mapping populations for Morex × Barke or Morex × Steptoe. Morex, Barke, and Steptoe are western cultivated barleys and are likely to bear the standard arrangement. Given the uncertainty of the presence of the putative rearrangements in mapping parents and limited marker density within the putative rearranged genomic regions, the present genetic maps neither convincingly refute nor support the presence of structural rearrangements.</p><p id="p-59">We also noted that the two putative chromosome rearrangements are not differentiated by large numbers of private SNPs (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>) as observed at the largest inversion (<em>Inv1n</em>) in teosinte, the wild progenitor of maize (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Fang <em>et al.</em> 2012</a>). This may be attributable in the discovery of SNPs primarily among western cultivars, which are unlikely to carry the rearrangement. Moreover, the putative translocation on 2H incorporates the centromere and the putative rearrangement on 5H is close to the centromeric region. Therefore, an alternative explanation for the patterns observed in these two genomic regions could be exceptional effects arising from suppressed recombination in the centromeric regions (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Mayer <em>et al.</em> 2012</a>).</p></div><div id="sec-15" class="subsection"><h3>Useful wild barley alleles</h3><p id="p-60">Identification of functional variation that indicates local adaptation can contribute to sustained crop improvement. Crop wild progenitors have a history of adaptation to their local environments that is orders of magnitude older than the time since domestication, so wild populations have been exposed to natural selection for many more generations. Moreover, a domestication bottleneck decreases nucleotide diversity and causes the loss of valuable variants (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Eyre-Walker <em>et al.</em> 1998</a>). For these reasons, wild populations are expected to carry many novel nucleotide sequence variants and functional adaptations that are not present in domesticates.</p><p id="p-61">We used several approaches to identify nucleotide polymorphisms potentially involved in local adaptation. SNPs that are outliers in the <em>F</em><sub>ST</sub>, environmental association, and SPA analyses are potentially linked to loci contributing to local adaptation. The <em>F</em><sub>ST</sub> comparison assumes that genetic markers with extreme allele frequency differences among populations may contribute to local adaptation (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Lewontin and Krakauer 1973</a>), but this method has several limitations. First, the Lewontin and Krakauer approach suffers from a number of assumptions, including that all populations diverged at the same time (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Nei and Maruyama 1975</a>). Second, comparison of allele frequency differences with <em>F</em><sub>ST</sub> requires the prior identification of populations. Environmental association and SPA analyses complement and are consistent with the <em>F</em><sub>ST</sub> results. The results of both analyses support the conclusion that selection has contributed to the observed allele frequency differentiation likely driven by environmental factors or correlated selection pressures (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Coop <em>et al.</em> 2010</a>).</p><p id="p-62">A number of SNPs within previously characterized barley loci are outliers in one or more of the analyses reported here. The SNP (12_30850) from the gene <em>Cbf4</em> is above the 95<sup>th</sup> percentile of <em>F</em><sub>ST</sub> values for SNP frequencies compared between the Eastern and Western populations. <em>Cbf4</em> contributes to low-temperature tolerance in barley (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Francia <em>et al.</em> 2004</a>). From environmental association analysis, one SNP (11_11361) in <em>Cbf4</em> and another SNP (11_10989) in a cold-regulated gene <em>blt14</em> (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cattivelli and Bartels 1990</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Grossi <em>et al.</em> 1998</a>) are among outliers of PC1, which includes the temperature variables. SPA analysis reveals that SNP 12_30850 in <em>Cbf4</em> also shows a strong geographic gradient in allele frequency. These results establish the value of geographic analyses as a screen to identify potentially adaptive alleles.</p></div><div id="sec-16" class="subsection"><h3>Summary</h3><p id="p-63">Two large pericentromeric regions on 2H and 5H make strong contributions to the population structure observed in wild barley. These two genomic regions are putative chromosome structural rearrangements that harbor variants that appear to contribute to environmental adaptation. In particular, the SNPs in these chromosomal regions are shown to be associated with temperature and precipitation variables. It will be important to determine how specific genetic variants within these rearrangements are associated with local adaptation. Nevertheless, the identification of genomic regions associated with environmental adaptation suggests an opportunity for crop improvement.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-64">The authors thank Thomas Kono for assistance with determination of SNP ancestral state and helpful discussion, and Allison Haaning, Tanja Pyhäjärvi, Jeffrey Ross-Ibarra, and Jeremy Yoder for comments on an earlier version of the manuscript. This work was performed using computing resources at the University of Minnesota Supercomputing Institute. We acknowledge funding from the US Department of Agriculture National Institute for Food and Agriculture USDA NIFA 2011- 68002-30029 (to P.L.M.) and University of Minnesota Doctoral Dissertation Fellowship (to Z.F.) Support was also received from the Lieberman-Okinow Endowment at the University of Minnesota and the USAID-funded Cereals Comparative Genomics Initiative.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010561/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010561/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-65"><em>Communicating editor: D. Zamir</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-01-31"><span class="received-label">Received </span>January 31, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-04-22"><span class="accepted-label">Accepted </span>April 22, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Fang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.1" data-doi="10.1093/molbev/msq148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Ascertainment biases in SNP chips affect measures of population divergence.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2534</span>–<span class="cit-lpage">2547</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq148%26rft_id%253Dinfo%253Apmid%252F20558595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyNy8xMS8yNTM0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzExOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.2" data-doi="10.1038/hdy.1978.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">A. H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zohary</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nevo</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Outcrossing rates and heterozygosity in natural populations of <em>Hordeum spontaneum</em> Koch in Israel.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D41%26rft.spage%253D49%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1978.63%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1978.63&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1978FS72700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.3" data-doi="10.1534/genetics.104.038489"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caldwell</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langridge</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Extreme population-dependent linkage disequilibrium detected in an inbreeding plant species, <em>Hordeum vulgare</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">557</span>–<span class="cit-lpage">567</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.038489%26rft_id%253Dinfo%253Apmid%252F16219791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcyLzEvNTU3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzExOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.4" data-doi="10.1104/pp.93.4.1504"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cattivelli</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartels</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Molecular cloning and characterization of cold-regulated genes in barley.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">1504</span>–<span class="cit-lpage">1510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DCattivelli%26rft.auinit1%253DL.%26rft.volume%253D93%26rft.issue%253D4%26rft.spage%253D1504%26rft.epage%253D1510%26rft.atitle%253DMolecular%2BCloning%2Band%2BCharacterization%2Bof%2BCold-Regulated%2BGenes%2Bin%2BBarley%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.93.4.1504%26rft_id%253Dinfo%253Apmid%252F16667647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiI5My80LzE1MDQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTE5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.5" data-doi="10.1098/rspb.1966.0038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavalli-Sforza</span>  <span class="cit-name-given-names">L. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">Population structure and human evolution.</span> <abbr class="cit-jnl-abbrev">Proc. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">362</span>–<span class="cit-lpage">379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DCavalli-Sforza%26rft.auinit1%253DL.%2BL.%26rft.volume%253D164%26rft.issue%253D995%26rft.spage%253D362%26rft.epage%253D379%26rft.atitle%253DPopulation%2BStructure%2Band%2BHuman%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.1966.0038%26rft_id%253Dinfo%253Apmid%252F4379525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo3OiJyb3lwcnNiIjtzOjU6InJlc2lkIjtzOjExOiIxNjQvOTk1LzM2MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.6" data-doi="10.1186/1471-2164-10-582"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lonardi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rostoks</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Development and implementation of high-throughput SNP genotyping in barley.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">582</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DClose%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D10%26rft.spage%253D582%26rft.epage%253D582%26rft.atitle%253DDevelopment%2Band%2Bimplementation%2Bof%2Bhigh-throughput%2BSNP%2Bgenotyping%2Bin%2Bbarley.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-582%26rft_id%253Dinfo%253Apmid%252F19961604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-582&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19961604&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.7" data-doi="10.2135/cropsci1984.0011183X002400060002x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galinat</span>  <span class="cit-name-given-names">W. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Potential use of alien germplasm for maize improvement.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1011</span>–<span class="cit-lpage">1015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCohen%26rft.auinit1%253DJ.%2BI.%26rft.volume%253D24%26rft.issue%253D6%26rft.spage%253D1011%26rft.epage%253D1015%26rft.atitle%253DPotential%2BUse%2Bof%2BAlien%2BGermplasm%2Bfor%2BMaize%2BImprovement%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1984.0011183X002400060002x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1984.0011183X002400060002x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1984TS42700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.8" data-doi="10.1534/genetics.110.114819"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witonsky</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Di Rienzo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Using environmental correlations to identify loci underlying local adaptation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1411</span>–<span class="cit-lpage">1423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.114819%26rft_id%253Dinfo%253Apmid%252F20516501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS80LzE0MTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTE5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.9" data-doi="10.1007/s001220100622"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Costa</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corey</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jobet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleinhofs</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Molecular mapping of the Oregon Wolfe Barleys: a phenotypically polymorphic doubled-haploid population.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">415</span>–<span class="cit-lpage">424</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D103%26rft.spage%253D415%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220100622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220100622&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000170534600029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.10" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.11" data-doi="10.1073/pnas.95.8.4441"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hilton</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Investigation of the bottleneck leading to the domestication of maize.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">4441</span>–<span class="cit-lpage">4446</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DEyre-Walker%26rft.auinit1%253DA.%26rft.volume%253D95%26rft.issue%253D8%26rft.spage%253D4441%26rft.epage%253D4446%26rft.atitle%253DInvestigation%2Bof%2Bthe%2Bbottleneck%2Bleading%2Bto%2Bthe%2Bdomestication%2Bof%2Bmaize%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.95.8.4441%26rft_id%253Dinfo%253Apmid%252F9539756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5NS84LzQ0NDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTE5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falush</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">1567</span>–<span class="cit-lpage">1587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFalush%26rft.auinit1%253DD.%26rft.volume%253D164%26rft.issue%253D4%26rft.spage%253D1567%26rft.epage%253D1587%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%253A%2BLinked%2BLoci%2Band%2BCorrelated%2BAllele%2BFrequencies%26rft_id%253Dinfo%253Apmid%252F12930761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NC80LzE1NjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTE5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.13" data-doi="10.1534/genetics.112.138578"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pyhäjärvi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawe</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Megabase-scale inversion polymorphism in the wild ancestor of maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">883</span>–<span class="cit-lpage">894</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.138578%26rft_id%253Dinfo%253Apmid%252F22542971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkxLzMvODgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzExOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.14" data-doi="10.1534/g3.113.007294"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eule-Nashoba</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powers</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kono</span>  <span class="cit-name-given-names">T. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takuno</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparative analyses identify the contributions of exotic donors to disease resistance in a barley experimental population.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1945</span>–<span class="cit-lpage">1953</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.007294%26rft_id%253Dinfo%253Apmid%252F24048643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjMvMTEvMTk0NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.15" data-doi="10.1094/PDIS.2003.87.12.1439"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fetch</span>  <span class="cit-name-given-names">T. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steffenson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nevo</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Diversity and sources of multiple disease resistance in <em>Hordeum spontaneum</em>.</span> <abbr class="cit-jnl-abbrev">Plant Dis.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">1439</span>–<span class="cit-lpage">1448</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BDis.%26rft.volume%253D87%26rft.spage%253D1439%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPDIS.2003.87.12.1439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PDIS.2003.87.12.1439&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.16" data-doi="10.1007/s00122-003-1468-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Francia</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rizza</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cattivelli</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stanca</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galiba</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Two loci on chromosome 5H determine low-temperature tolerance in a ‘Nure’(winter) × ‘Tremois’(spring) barley map.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">670</span>–<span class="cit-lpage">680</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DFrancia%26rft.auinit1%253DE.%26rft.volume%253D108%26rft.issue%253D4%26rft.spage%253D670%26rft.epage%253D680%26rft.atitle%253DTwo%2Bloci%2Bon%2Bchromosome%2B5H%2Bdetermine%2Blow-temperature%2Btolerance%2Bin%2Ba%2B%2527Nure%2527%2B%2528winter%2529%2Bx%2B%2527Tremois%2527%2B%2528spring%2529%2Bbarley%2Bmap.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-003-1468-9%26rft_id%253Dinfo%253Apmid%252F14576984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-003-1468-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14576984&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000189207100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.17" data-doi="10.1534/genetics.111.131136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gattepaille</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Combining markers into haplotypes can improve population structure inference.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.131136%26rft_id%253Dinfo%253Apmid%252F21868606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzEvMTU5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzExOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.18" data-doi="10.1046/j.1365-294X.1997.00260.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodman</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">R<sub>ST</sub> Calc: a collection of computer programs for calculating estimates of genetic differentiation from microsatellite data and determining their significance.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">881</span>–<span class="cit-lpage">885</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D6%26rft.spage%253D881%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-294X.1997.00260.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-294X.1997.00260.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997XV25000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.19" data-doi="10.1111/j.1471-8286.2004.00828.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Hierfstat, a package for R to compute and test hierarchical F‐statistics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Notes</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">184</span>–<span class="cit-lpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BNotes%26rft.volume%253D5%26rft.spage%253D184%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2004.00828.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1471-8286.2004.00828.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000227598500059&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.20" data-doi="10.1023/A:1006079916917"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grossi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giorni</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rizza</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stanca</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cattivelli</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Wild and cultivated barleys show differences in the expression pattern of a cold-regulated gene family under different light and temperature conditions.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1061</span>–<span class="cit-lpage">1069</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DGrossi%26rft.auinit1%253DM.%26rft.volume%253D38%26rft.issue%253D6%26rft.spage%253D1061%26rft.epage%253D1069%26rft.atitle%253DWild%2Band%2Bcultivated%2Bbarleys%2Bshow%2Bdifferences%2Bin%2Bthe%2Bexpression%2Bpattern%2Bof%2Ba%2Bcold-regulated%2Bgene%2Bfamily%2Bunder%2Bdifferent%2Blight%2Band%2Btemperature%2Bconditions.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1006079916917%26rft_id%253Dinfo%253Apmid%252F9869412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1006079916917&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9869412&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077762100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haasl</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payseur</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Multi-locus inference of population structure: a comparison between single nucleotide polymorphisms and microsatellites.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">158</span>–<span class="cit-lpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D106%26rft.spage%253D158%26rft_id%253Dinfo%253Apmid%252F20332809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20332809&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.22" data-doi="10.1111/j.1095-8312.1990.tb01608.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Vavilov’s concept of centres of origin of cultivated plants: its genesis and its influence on the study of agricultural origins.</span> <abbr class="cit-jnl-abbrev">Biol. J. Linn. Soc. Lond.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">7</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BJ.%2BLinn.%2BSoc.%2BLond.%26rft.volume%253D39%26rft.spage%253D7%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1095-8312.1990.tb01608.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1095-8312.1990.tb01608.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1990CN43400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hijmans</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guarino</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cruz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rojas</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Computer tools for spatial analysis of plant genetic resources data: 1. DIVA-GIS.</span> <abbr class="cit-jnl-abbrev">Plant Genet. Resour. Newsl.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenet.%2BResour.%2BNewsl.%26rft.volume%253D127%26rft.spage%253D15%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.24" data-doi="10.1002/joc.1276"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hijmans</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parra</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarvis</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Very high resolution interpolated climate surfaces for global land areas.</span> <abbr class="cit-jnl-abbrev">Int. J. Climatol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1965</span>–<span class="cit-lpage">1978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BClimatol.%26rft.volume%253D25%26rft.spage%253D1965%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjoc.1276%26rft_id%253Dinfo%253Apmid%252F19225523%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/joc.1276&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19225523&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234519700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.25" data-doi="10.1007/BF01245622"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span> <span class="cit-article-title">Linkage disequilibrium in finite populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">226</span>–<span class="cit-lpage">231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D38%26rft.spage%253D226%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01245622%26rft_id%253Dinfo%253Apmid%252F24442307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01245622&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24442307&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.26" data-doi="10.1038/hdy.2010.85"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huynh</span>  <span class="cit-name-given-names">L. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maney</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Chromosome-wide linkage disequilibrium caused by an inversion polymorphism in the white-throated sparrow (<em>Zonotrichia albicollis</em>).</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">537</span>–<span class="cit-lpage">546</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHuynh%26rft.auinit1%253DL.%2BY.%26rft.volume%253D106%26rft.issue%253D4%26rft.spage%253D537%26rft.epage%253D546%26rft.atitle%253DChromosome-wide%2Blinkage%2Bdisequilibrium%2Bcaused%2Bby%2Ban%2Binversion%2Bpolymorphism%2Bin%2Bthe%2Bwhite-throated%2Bsparrow%2B%2528Zonotrichia%2Balbicollis%2529.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2010.85%26rft_id%253Dinfo%253Apmid%252F20571514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2010.85&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20571514&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288703900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.27" data-doi="10.1093/gbe/evu047"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jakob</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodder</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Engler</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaaf</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozkan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evolutionary history of wild barley (<em>Hordeum vulgare</em> subsp. <em>spontaneum</em>) analyzed using multilocus sequence data and paleodistribution modeling.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">685</span>–<span class="cit-lpage">702</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevu047%26rft_id%253Dinfo%253Apmid%252F24586028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjYvMy82ODUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTE5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.28" data-doi="10.1093/bioinformatics/btm233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">CLUMPP: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1801</span>–<span class="cit-lpage">1806</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm233%26rft_id%253Dinfo%253Apmid%252F17485429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjMvMTQvMTgwMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.29" data-doi="10.1093/molbev/msn167"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leigh</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bower</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">L. M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Population-based resequencing reveals that the flowering time adaptation of cultivated barley originated east of the Fertile Crescent.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2211</span>–<span class="cit-lpage">2219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn167%26rft_id%253Dinfo%253Apmid%252F18669581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xMC8yMjExIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzExOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.30" data-doi="10.1534/genetics.105.047985"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Chromosome inversions, local adaptation and speciation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.047985%26rft_id%253Dinfo%253Apmid%252F16204214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTczLzEvNDE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzExOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.31" data-doi="10.1007/BF00303959"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Konishi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linde-Laursen</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Spontaneous chromosomal rearrangements in cultivated and wild barleys.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">243</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D75%26rft.spage%253D237%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00303959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00303959&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.32" data-doi="10.1111/1755-0998.12183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kono</span>  <span class="cit-name-given-names">T. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seth</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrell</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">SNPMeta: SNP annotation and SNP metadata collection without a reference genome.</span> <abbr class="cit-jnl-abbrev">Mol Ecol Resour</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%2BResour%26rft.volume%253D14%26rft.spage%253D419%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12183%26rft_id%253Dinfo%253Apmid%252F24237904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12183&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24237904&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.33" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lewontin</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krakauer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Distribution of gene frequency as a test of the theory of the selective neutrality of polymorphisms.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLewontin%26rft.auinit1%253DR.%2BC.%26rft.volume%253D74%26rft.issue%253D1%26rft.spage%253D175%26rft.epage%253D195%26rft.atitle%253DDISTRIBUTION%2BOF%2BGENE%2BFREQUENCY%2BAS%2BA%2BTEST%2BOF%2BTHE%2BTHEORY%2BOF%2BTHE%2BSELECTIVE%2BNEUTRALITY%2BOF%2BPOLYMORPHISMS%26rft_id%253Dinfo%253Apmid%252F4711903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzQvMS8xNzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTE5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.35" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.36" data-doi="10.1007/s00122-003-1345-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sjakste</span>  <span class="cit-name-given-names">T. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Röder</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ganal</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Development and genetic mapping of 127 new microsatellite markers in barley.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">1021</span>–<span class="cit-lpage">1027</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DJ.%2BZ.%26rft.volume%253D107%26rft.issue%253D6%26rft.spage%253D1021%26rft.epage%253D1027%26rft.atitle%253DDevelopment%2Band%2Bgenetic%2Bmapping%2Bof%2B127%2Bnew%2Bmicrosatellite%2Bmarkers%2Bin%2Bbarley.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-003-1345-6%26rft_id%253Dinfo%253Apmid%252F12879255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-003-1345-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12879255&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185811300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.37" data-doi="10.1073/pnas.98.2.531"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clegg</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Heterogeneous geographic patterns of nucleotide sequence diversity between two alcohol dehydrogenase genes in wild barley (<em>Hordeum vulgare</em> subspecies <em>spontaneum</em>).</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">531</span>–<span class="cit-lpage">536</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DLin%26rft.auinit1%253DJ.-Z.%26rft.volume%253D98%26rft.issue%253D2%26rft.spage%253D531%26rft.epage%253D536%26rft.atitle%253DHeterogeneous%2Bgeographic%2Bpatterns%2Bof%2Bnucleotide%2Bsequence%2Bdiversity%2Bbetween%2Btwo%2Balcohol%2Bdehydrogenase%2Bgenes%2Bin%2Bwild%2Bbarley%2B%2528Hordeum%2Bvulgare%2Bsubspecies%2Bspontaneum%2529%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.98.2.531%26rft_id%253Dinfo%253Apmid%252F11149938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI5OC8yLzUzMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.38" data-doi="10.1038/ng.2678"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabanal</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farlow</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Massive genomic variation and strong selection in <em>Arabidopsis thaliana</em> lines from Sweden.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">884</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D884%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2678%26rft_id%253Dinfo%253Apmid%252F23793030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2678&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23793030&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.39" data-doi="10.1371/journal.pbio.1000500"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lowry</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A widespread chromosomal inversion polymorphism contributes to a major life-history transition, local adaptation, and reproductive isolation.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1000500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D8%26rft.spage%253De1000500%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000500%26rft_id%253Dinfo%253Apmid%252F20927411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000500&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20927411&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.40" data-doi="10.1093/nar/gkg058"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matthews</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carollo</span>  <span class="cit-name-given-names">V. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lazo</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">O. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">GrainGenes, the genome database for small-grain crops.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">183</span>–<span class="cit-lpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DMatthews%26rft.auinit1%253DD.%2BE.%26rft.volume%253D31%26rft.issue%253D1%26rft.spage%253D183%26rft.epage%253D186%26rft.atitle%253DGrainGenes%252C%2Bthe%2Bgenome%2Bdatabase%2Bfor%2Bsmall-grain%2Bcrops%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg058%26rft_id%253Dinfo%253Apmid%252F12519977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjMxLzEvMTgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzExOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mayer</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waugh</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langridge</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A physical, genetic and functional sequence assembly of the barley genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">491</span>: <span class="cit-fpage">711</span>–<span class="cit-lpage">716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMayer%26rft.auinit1%253DK.%2BF.%26rft.volume%253D491%26rft.issue%253D7426%26rft.spage%253D711%26rft.epage%253D716%26rft.atitle%253DA%2Bphysical%252C%2Bgenetic%2Band%2Bfunctional%2Bsequence%2Bassembly%2Bof%2Bthe%2Bbarley%2Bgenome.%26rft_id%253Dinfo%253Apmid%252F23075845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23075845&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311606000035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.42" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.43" data-doi="10.1073/pnas.0611377104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morrell</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clegg</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic evidence for a second domestication of barley (<em>Hordeum vulgare</em>) east of the Fertile Crescent.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">3289</span>–<span class="cit-lpage">3294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0611377104%26rft_id%253Dinfo%253Apmid%252F17360640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA0LzkvMzI4OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.44" data-doi="10.1073/pnas.1633708100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morrell</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lundy</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clegg</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Distinct geographic patterns of genetic diversity are maintained in wild barley (<em>Hordeum vulgare</em> ssp. <em>spontaneum</em>) despite migration.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">10812</span>–<span class="cit-lpage">10817</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1633708100%26rft_id%253Dinfo%253Apmid%252F12963820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAwLzE5LzEwODEyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzExOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.45" data-doi="10.1073/pnas.0409804102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morrell</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toleno</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lundy</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clegg</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Low levels of linkage disequilibrium in wild barley (<em>Hordeum vulgare</em> ssp. <em>spontaneum</em>) despite high rates of self-fertilization.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">2442</span>–<span class="cit-lpage">2447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0409804102%26rft_id%253Dinfo%253Apmid%252F15699350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAyLzcvMjQ0MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.46" data-doi="10.1534/genetics.105.054502"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morrell</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toleno</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lundy</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clegg</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Estimating the contribution of mutation, recombination and gene conversion in the generation of haplotypic diversity.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">1705</span>–<span class="cit-lpage">1723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.054502%26rft_id%253Dinfo%253Apmid%252F16624913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My8zLzE3MDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTE5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.47" data-doi="10.1093/jhered/est083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morrell</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzales</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">K. K. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clegg</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Resequencing data indicates a modest effect of domestication on diversity in barley: a cultigen with multiple origins.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">253</span>–<span class="cit-lpage">264</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fest083%26rft_id%253Dinfo%253Apmid%252F24336926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjk6IjEwNS8yLzI1MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.48" data-doi="10.3835/plantgenome2011.08.0023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muñoz-Amatriaín</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moscou</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">P. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">An improved consensus linkage map of barley based on flow-sorted chromosomes and single nucleotide polymorphism markers.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">238</span>–<span class="cit-lpage">249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D238%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000312661700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maruyama</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Lewontin-Krakauer test for neutral genes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNei%26rft.auinit1%253DM.%26rft.volume%253D80%26rft.issue%253D2%26rft.spage%253D395%26rft.epage%253D395%26rft.atitle%253DLewontin-Krakauer%2Btest%2Bfor%2Bneutral%2Bgenes%26rft_id%253Dinfo%253Apmid%252F1132690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI4MC8yLzM5NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.50" data-doi="10.1111/j.1095-8312.1986.tb01742.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nevo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beiles</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zohary</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Genetic resources of wild barley in the Near East: structure, evolution and application in breeding.</span> <abbr class="cit-jnl-abbrev">Biol. J. Linn. Soc. Lond.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">380</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BJ.%2BLinn.%2BSoc.%2BLond.%26rft.volume%253D27%26rft.spage%253D355%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1095-8312.1986.tb01742.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1095-8312.1986.tb01742.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1986C342800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.51" data-doi="10.1101/gr.4252305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genomic scans for selective sweeps using SNP data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1566</span>–<span class="cit-lpage">1575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D15%26rft.issue%253D11%26rft.spage%253D1566%26rft.epage%253D1575%26rft.atitle%253DGenomic%2Bscans%2Bfor%2Bselective%2Bsweeps%2Busing%2BSNP%2Bdata%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4252305%26rft_id%253Dinfo%253Apmid%252F16251466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMS8xNTY2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzExOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.52" data-doi="10.1017/S0016672300012994"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohta</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">The model of mutation appropriate to estimate the number of electrophoretically detectable alleles in a genetic population.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">201</span>–<span class="cit-lpage">204</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DOta%26rft.auinit1%253DT.%26rft.volume%253D22%26rft.issue%253D2%26rft.spage%253D201%26rft.epage%253D204%26rft.atitle%253DA%2Bmodel%2Bof%2Bmutation%2Bappropriate%2Bto%2Bestimate%2Bthe%2Bnumber%2Bof%2Belectrophoretically%2Bdetectable%2Balleles%2Bin%2Ba%2Bfinite%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300012994%26rft_id%253Dinfo%253Apmid%252F4777279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300012994&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4777279&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1973S206000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.53" data-doi="10.1093/bioinformatics/btg412"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paradis</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Claude</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strimmer</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">APE: analyses of phylogenetics and evolution in R language.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DParadis%26rft.auinit1%253DE.%26rft.volume%253D20%26rft.issue%253D2%26rft.spage%253D289%26rft.epage%253D290%26rft.atitle%253DAPE%253A%2BAnalyses%2Bof%2BPhylogenetics%2Band%2BEvolution%2Bin%2BR%2Blanguage%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg412%26rft_id%253Dinfo%253Apmid%252F14734327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyMC8yLzI4OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.54" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzExOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-web" id="cit-4.7.1193.55"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Development Core Team</span>, <span class="cit-pub-date">2012</span> R: A language and environment for statistical computing, version 2.15.1. R Foundation for Statistical Computing, Vienna. <a href="http://www.r-project.org">http://www.r-project.org</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.56" data-doi="10.2135/cropsci1961.0011183X000100050005x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramage</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suneson</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1961</span> <span class="cit-article-title">Translocation-gene linkages on barley chromosome 7.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DRamage%26rft.auinit1%253DR.%2BT.%26rft.volume%253D1%26rft.issue%253D5%26rft.spage%253D319%26rft.epage%253D320%26rft.atitle%253DTranslocation-Gene%2BLinkages%2Bon%2BBarley%2BChromosome%2B7%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1961.0011183X000100050005x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1961.0011183X000100050005x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramsay</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macaulay</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Degli Ivanissevich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacLean</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardle</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A simple sequence repeat-based linkage map of barley.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">1997</span>–<span class="cit-lpage">2005</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRamsay%26rft.auinit1%253DL.%26rft.volume%253D156%26rft.issue%253D4%26rft.spage%253D1997%26rft.epage%253D2005%26rft.atitle%253DA%2BSimple%2BSequence%2BRepeat-Based%2BLinkage%2BMap%2Bof%2BBarley%26rft_id%253Dinfo%253Apmid%252F11102390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ni80LzE5OTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTE5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.58" data-doi="10.1086/380416"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Informativeness of genetic markers for inference of ancestry.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">1402</span>–<span class="cit-lpage">1422</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DRosenberg%26rft.auinit1%253DN.%2BA.%26rft.volume%253D73%26rft.issue%253D6%26rft.spage%253D1402%26rft.epage%253D1422%26rft.atitle%253DInformativeness%2Bof%2Bgenetic%2Bmarkers%2Bfor%2Binference%2Bof%2Bancestry.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F380416%26rft_id%253Dinfo%253Apmid%252F14631557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/380416&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14631557&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187491100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.59" data-doi="10.1073/pnas.0700643104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrell</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Plant domestication, a unique opportunity to identify the genetic basis of adaptation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">8641</span>–<span class="cit-lpage">8648</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0700643104%26rft_id%253Dinfo%253Apmid%252F17494757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxNjoiMTA0L3N1cHBsXzEvODY0MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.60" data-doi="10.1111/j.1469-8137.2011.03704.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">I. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flavell</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steffenson</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weltzien</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Analysis of &gt;1000 single nucleotide polymorphisms in geographically matched samples of landrace and wild barley indicates secondary contact and chromosome‐level differences in diversity around domestication genes.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">564</span>–<span class="cit-lpage">578</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DRussell%26rft.auinit1%253DJ.%26rft.volume%253D191%26rft.issue%253D2%26rft.spage%253D564%26rft.epage%253D578%26rft.atitle%253DAnalysis%2Bof%2B%2526gt%253B1000%2Bsingle%2Bnucleotide%2Bpolymorphisms%2Bin%2Bgeographically%2Bmatched%2Bsamples%2Bof%2Blandrace%2Band%2Bwild%2Bbarley%2Bindicates%2Bsecondary%2Bcontact%2Band%2Bchromosome-level%2Bdifferences%2Bin%2Bdiversity%2Baround%2Bdomestication%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2011.03704.x%26rft_id%253Dinfo%253Apmid%252F21443695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2011.03704.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21443695&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292211800025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.61" data-doi="10.1534/genetics.107.079491"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saisho</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Purugganan</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Molecular phylogeography of domesticated barley traces expansion of agriculture in the Old World.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">1765</span>–<span class="cit-lpage">1776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.079491%26rft_id%253Dinfo%253Apmid%252F17947416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny8zLzE3NjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTE5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.62" data-doi="10.1038/hdy.2009.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nankaku</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takeda</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A high-density transcript linkage map of barley derived from a single population.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">110</span>–<span class="cit-lpage">117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DSato%26rft.auinit1%253DK.%26rft.volume%253D103%26rft.issue%253D2%26rft.spage%253D110%26rft.epage%253D117%26rft.atitle%253DA%2Bhigh-density%2Btranscript%2Blinkage%2Bmap%2Bof%2Bbarley%2Bderived%2Bfrom%2Ba%2Bsingle%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2009.57%26rft_id%253Dinfo%253Apmid%252F19455180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2009.57&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19455180&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268166000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.63" data-doi="10.1093/pcp/pcr024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muñoz-Amatriaín</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muehlbauer</span>  <span class="cit-name-given-names">G. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Single nucleotide polymorphism mapping and alignment of recombinant chromosome substitution lines in barley.</span> <abbr class="cit-jnl-abbrev">Plant Cell Physiol.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">728</span>–<span class="cit-lpage">737</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fpcp%252Fpcr024%26rft_id%253Dinfo%253Apmid%252F21427110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGNlbGxwaHlzIjtzOjU6InJlc2lkIjtzOjg6IjUyLzUvNzI4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzExOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.64" data-doi="10.1086/502802"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scheet</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">629</span>–<span class="cit-lpage">644</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DScheet%26rft.auinit1%253DP.%26rft.volume%253D78%26rft.issue%253D4%26rft.spage%253D629%26rft.epage%253D644%26rft.atitle%253DA%2Bfast%2Band%2Bflexible%2Bstatistical%2Bmodel%2Bfor%2Blarge-scale%2Bpopulation%2Bgenotype%2Bdata%253A%2Bapplications%2Bto%2Binferring%2Bmissing%2Bgenotypes%2Band%2Bhaplotypic%2Bphase.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F502802%26rft_id%253Dinfo%253Apmid%252F16532393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/502802&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16532393&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236755900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.65" data-doi="10.1007/s00122-008-0915-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmalenbach</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Léon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pillen</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Identification and verification of QTLs for agronomic traits using wild barley introgression lines.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSchmalenbach%26rft.auinit1%253DI.%26rft.volume%253D118%26rft.issue%253D3%26rft.spage%253D483%26rft.epage%253D497%26rft.atitle%253DIdentification%2Band%2Bverification%2Bof%2BQTLs%2Bfor%2Bagronomic%2Btraits%2Busing%2Bwild%2Bbarley%2Bintrogression%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0915-z%26rft_id%253Dinfo%253Apmid%252F18979081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0915-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18979081&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262577300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.66" data-doi="10.1641/0006-3568(2001)051[0960:TCOWPS]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schoen</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">A. H. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The Conservation of Wild Plant Species in Seed Banks.</span> <abbr class="cit-jnl-abbrev">Bioscience</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">960</span>–<span class="cit-lpage">966</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioscience%26rft_id%253Dinfo%253Adoi%252F10.1641%252F0006-3568%25282001%2529051%255B0960%253ATCOWPS%255D2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6ImJpb3NjaWVuY2UiO3M6NToicmVzaWQiO3M6OToiNTEvMTEvOTYwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzExOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.67"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shin</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blay</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McNeney</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">LDheatmap: an R function for graphical display of pairwise linkage disequilibria between single nucleotide polymorphisms.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D16%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shriner</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Investigating population stratification and admixture using eigenanalysis of dense genotypes.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">420</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DShriner%26rft.auinit1%253DD.%26rft.volume%253D107%26rft.issue%253D5%26rft.spage%253D413%26rft.epage%253D420%26rft.atitle%253DInvestigating%2Bpopulation%2Bstratification%2Band%2Badmixture%2Busing%2Beigenanalysis%2Bof%2Bdense%2Bgenotypes.%26rft_id%253Dinfo%253Apmid%252F21448230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21448230&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">A measure of population subdivision based on microsatellite allele frequencies.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">139</span>: <span class="cit-fpage">457</span>–<span class="cit-lpage">462</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSlatkin%26rft.auinit1%253DM.%26rft.volume%253D139%26rft.issue%253D1%26rft.spage%253D457%26rft.epage%253D462%26rft.atitle%253DA%2Bmeasure%2Bof%2Bpopulation%2Bsubdivision%2Bbased%2Bon%2Bmicrosatellite%2Ballele%2Bfrequencies.%26rft_id%253Dinfo%253Apmid%252F7705646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM5LzEvNDU3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzExOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.70" data-doi="10.1071/AR07123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steffenson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olivera</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roy</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A walk on the wild side: mining wild wheat and barley collections for rust resistance genes.</span> <abbr class="cit-jnl-abbrev">Aust. J. Agric. Res.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">532</span>–<span class="cit-lpage">544</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAust.%2BJ.%2BAgric.%2BRes.%26rft.volume%253D58%26rft.spage%253D532%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAR07123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AR07123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000247489400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.71" data-doi="10.1007/s00122-006-0480-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prasad</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scholz</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thiel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A 1,000-loci transcript map of the barley genome: new anchoring points for integrative grass genomics.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">823</span>–<span class="cit-lpage">839</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DStein%26rft.auinit1%253DN.%26rft.volume%253D114%26rft.issue%253D5%26rft.spage%253D823%26rft.epage%253D839%26rft.atitle%253DA%2B1%252C000-loci%2Btranscript%2Bmap%2Bof%2Bthe%2Bbarley%2Bgenome%253A%2Bnew%2Banchoring%2Bpoints%2Bfor%2Bintegrative%2Bgrass%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0480-2%26rft_id%253Dinfo%253Apmid%252F17219208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0480-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17219208&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244486600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.72" data-doi="10.1093/bioinformatics/btn478"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szpiech</span>  <span class="cit-name-given-names">Z. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">ADZE: a rarefaction approach for counting alleles private to combinations of populations.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2498</span>–<span class="cit-lpage">2504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn478%26rft_id%253Dinfo%253Apmid%252F18779233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjQvMjEvMjQ5OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.73" data-doi="10.3835/plantgenome2008.01.0005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szűcs</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blake</span>  <span class="cit-name-given-names">V. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">An integrated resource for barley linkage map and malting quality QTL alignment.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">134</span>–<span class="cit-lpage">140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D2%26rft.spage%253D134%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2008.01.0005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2008.01.0005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.74" data-doi="10.1038/nrg2342"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takeda</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsuoka</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic approaches to crop improvement: Responding to environmental and population changes.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">444</span>–<span class="cit-lpage">457</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DSlatkin%26rft.auinit1%253DM.%26rft.volume%253D9%26rft.issue%253D6%26rft.spage%253D444%26rft.epage%253D457%26rft.atitle%253DGenetic%2Bapproaches%2Bto%2Bcrop%2Bimprovement%253A%2Bresponding%2Bto%2Benvironmental%2Band%2Bpopulation%2Bchanges.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2342%26rft_id%253Dinfo%253Apmid%252F18475268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2342&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18475268&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255953500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.75" data-doi="10.1534/genetics.108.088229"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cáceres</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowman</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morehouse</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Short</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The chromosomal polymorphism linked to variation in social behavior in the white-throated sparrow (<em>Zonotrichia albicollis</em>) is a complex rearrangement and suppressor of recombination.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1455</span>–<span class="cit-lpage">1468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.088229%26rft_id%253Dinfo%253Apmid%252F18562641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8zLzE0NTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzcvMTE5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.76" data-doi="10.1093/bioinformatics/btg316"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Libsequence: a C++ class library for evolutionary genetic analysis.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">2325</span>–<span class="cit-lpage">2327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DThornton%26rft.auinit1%253DK.%26rft.volume%253D19%26rft.issue%253D17%26rft.spage%253D2325%26rft.epage%253D2327%26rft.atitle%253Dlibsequence%253A%2Ba%2BC%252B%252B%2Bclass%2Blibrary%2Bfor%2Bevolutionary%2Bgenetic%2Banalysis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg316%26rft_id%253Dinfo%253Apmid%252F14630667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMTkvMTcvMjMyNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valdes</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freimer</span>  <span class="cit-name-given-names">N. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Allele frequencies at microsatellite loci: the stepwise mutation model revisited.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">749</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DValdes%26rft.auinit1%253DA.%2BM.%26rft.volume%253D133%26rft.issue%253D3%26rft.spage%253D737%26rft.epage%253D749%26rft.atitle%253DAllele%2Bfrequencies%2Bat%2Bmicrosatellite%2Bloci%253A%2Bthe%2Bstepwise%2Bmutation%2Bmodel%2Brevisited.%26rft_id%253Dinfo%253Apmid%252F8454213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTMzLzMvNzM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzExOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.78" data-doi="10.1007/s00122-007-0503-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varshney</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcel</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramsay</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Röder</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A high density barley microsatellite consensus map with 775 SSR loci.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">1091</span>–<span class="cit-lpage">1103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DVarshney%26rft.auinit1%253DR.%2BK.%26rft.volume%253D114%26rft.issue%253D6%26rft.spage%253D1091%26rft.epage%253D1103%26rft.atitle%253DA%2Bhigh%2Bdensity%2Bbarley%2Bmicrosatellite%2Bconsensus%2Bmap%2Bwith%2B775%2BSSR%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-007-0503-7%26rft_id%253Dinfo%253Apmid%252F17345060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-007-0503-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17345060&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245362000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.79"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Volis</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mendlinger</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orlovsky</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Variability in phenotypic traits in core and peripheral populations of wild barley <em>Hordeum spontaneum</em> Koch.</span> <abbr class="cit-jnl-abbrev">Hereditas</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHereditas%26rft.stitle%253DHereditas%26rft.aulast%253DVolis%26rft.auinit1%253DS.%26rft.volume%253D133%26rft.issue%253D3%26rft.spage%253D235%26rft.epage%253D247%26rft.atitle%253DVariability%2Bin%2Bphenotypic%2Btraits%2Bin%2Bcore%2Band%2Bperipheral%2Bpopulations%2Bof%2Bwild%2Bbarley%2BHordeum%2Bspontaneum%2BKoch.%26rft_id%253Dinfo%253Apmid%252F11433968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11433968&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.80" data-doi="10.1554/0014-3820(2002)056[1403:PAAVIM]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Volis</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mendlinger</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turuspekov</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esnazarov</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span><span class="cit-comment">a</span> <span class="cit-article-title">Phenotypic and allozyme variation in Mediterranean and desert populations of wild barley, <em>Hordeum spontaneum</em> Koch.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">1403</span>–<span class="cit-lpage">1415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DVolis%26rft.auinit1%253DS.%26rft.volume%253D56%26rft.issue%253D7%26rft.spage%253D1403%26rft.epage%253D1415%26rft.atitle%253DPhenotypic%2Band%2Ballozyme%2Bvariation%2Bin%2BMediterranean%2Band%2Bdesert%2Bpopulations%2Bof%2Bwild%2Bbarley%252C%2BHordeum%2Bspontaneum%2BKoch.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F0014-3820%25282002%2529056%255B1403%253APAAVIM%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F12206241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/0014-3820(2002)056[1403:PAAVIM]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12206241&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177434400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.81" data-doi="10.1111/j.1095-8312.2002.tb02071.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Volis</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mendlinger</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span><span class="cit-comment">b</span> <span class="cit-article-title">Differentiation in populations of <em>Hordeum spontaneum</em> Koch along a gradient of environmental productivity and predictability: plasticity in response to water and nutrient stress.</span> <abbr class="cit-jnl-abbrev">Biol. J. Linn. Soc. Lond.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BJ.%2BLinn.%2BSoc.%2BLond.%26rft.volume%253D75%26rft.spage%253D301%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1095-8312.2002.tb02071.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1095-8312.2002.tb02071.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000174695400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.82" data-doi="10.1007/s00442-002-0999-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Volis</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mendlinger</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span><span class="cit-comment">c</span> <span class="cit-article-title">Adaptive traits of wild barley plants of Mediterranean and desert origin.</span> <abbr class="cit-jnl-abbrev">Oecologia</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOecologia%26rft.volume%253D133%26rft.spage%253D131%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00442-002-0999-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00442-002-0999-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000178825900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.83" data-doi="10.1111/j.1420-9101.2004.00738.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Volis</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verhoeven</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mendlinger</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Phenotypic selection and regulation of reproduction in different environments in wild barley.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1121</span>–<span class="cit-lpage">1131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.stitle%253DJ%2BEvol%2BBiol%26rft.aulast%253DVolis%26rft.auinit1%253DS.%26rft.volume%253D17%26rft.issue%253D5%26rft.spage%253D1121%26rft.epage%253D1131%26rft.atitle%253DPhenotypic%2Bselection%2Band%2Bregulation%2Bof%2Breproduction%2Bin%2Bdifferent%2Benvironments%2Bin%2Bwild%2Bbarley.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1420-9101.2004.00738.x%26rft_id%253Dinfo%253Apmid%252F15312084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1420-9101.2004.00738.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15312084&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223281200019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.84" data-doi="10.1007/s00122-005-2049-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">von Korff</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pillen</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">AB-QTL analysis in spring barley. I. Detection of resistance genes against powdery mildew, leaf rust and scald introgressed from wild barley.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">583</span>–<span class="cit-lpage">590</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253Dvon%2BKorff%26rft.auinit1%253DM.%26rft.volume%253D111%26rft.issue%253D3%26rft.spage%253D583%26rft.epage%253D590%26rft.atitle%253DAB-QTL%2Banalysis%2Bin%2Bspring%2Bbarley.%2BI.%2BDetection%2Bof%2Bresistance%2Bgenes%2Bagainst%2Bpowdery%2Bmildew%252C%2Bleaf%2Brust%2Band%2Bscald%2Bintrogressed%2Bfrom%2Bwild%2Bbarley.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-2049-x%26rft_id%253Dinfo%253Apmid%252F15902395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-2049-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15902395&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231346200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.85" data-doi="10.1007/s00122-006-0223-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">von Korff</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pillen</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">AB-QTL analysis in spring barley: II. Detection of favourable exotic alleles for agronomic traits introgressed from wild barley (<em>H. vulgare</em> ssp <em>spontaneum</em>).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">1221</span>–<span class="cit-lpage">1231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DKorff%26rft.auinit1%253DM.%26rft.volume%253D112%26rft.issue%253D7%26rft.spage%253D1221%26rft.epage%253D1231%26rft.atitle%253DAB-QTL%2Banalysis%2Bin%2Bspring%2Bbarley%253A%2BII.%2BDetection%2Bof%2Bfavourable%2Bexotic%2Balleles%2Bfor%2Bagronomic%2Btraits%2Bintrogressed%2Bfrom%2Bwild%2Bbarley%2B%2528H.%2Bvulgare%2Bssp.%2Bspontaneum%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0223-4%26rft_id%253Dinfo%253Apmid%252F16477429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0223-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16477429&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236585600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.86" data-doi="10.1007/s11032-007-9110-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">von Korff</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pillen</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">AB-QTL analysis in spring barley: III. Identification of exotic alleles for the improvement of malting quality in spring barley (<em>H. vulgare</em> ssp <em>spontaneum</em>).</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D21%26rft.spage%253D81%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-007-9110-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-007-9110-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.87" data-doi="10.1371/journal.pgen.1002886"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zöllner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A quantitative comparison of the similarity between genes and geography in worldwide human populations.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002886</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DC.%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253De1002886%26rft.epage%253De1002886%26rft.atitle%253DA%2Bquantitative%2Bcomparison%2Bof%2Bthe%2Bsimilarity%2Bbetween%2Bgenes%2Band%2Bgeography%2Bin%2Bworldwide%2Bhuman%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002886%26rft_id%253Dinfo%253Apmid%252F22927824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002886&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22927824&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.88" data-doi="10.2307/2408641"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cockerham</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Estimating F-statistics for the analysis of population structure.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1358</span>–<span class="cit-lpage">1370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D38%26rft.spage%253D1358%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1984TY40400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.89" data-doi="10.1126/science.1107891"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bi</span>  <span class="cit-name-given-names">I. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schroeder</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamasaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doebley</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The effects of artificial selection on the maize genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">308</span>: <span class="cit-fpage">1310</span>–<span class="cit-lpage">1314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWright%26rft.auinit1%253DS.%2BI.%26rft.volume%253D308%26rft.issue%253D5726%26rft.spage%253D1310%26rft.epage%253D1314%26rft.atitle%253DThe%2BEffects%2Bof%2BArtificial%2BSelection%2Bon%2Bthe%2BMaize%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1107891%26rft_id%253Dinfo%253Apmid%252F15919994%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDgvNTcyNi8xMzEwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC83LzExOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.90" data-doi="10.1093/bioinformatics/btq533"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tung</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCouch</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">ALCHEMY: a reliable method for automated SNP genotype calling for small batch sizes and highly homozygous populations.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2952</span>–<span class="cit-lpage">2960</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq533%26rft_id%253Dinfo%253Apmid%252F20926420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMjMvMjk1MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNy8xMTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.91"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Estimating hierarchical F-statistics.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">950</span>–<span class="cit-lpage">956</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D58%26rft.spage%253D950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.92" data-doi="10.1038/ng.2285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">W. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halperin</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A model-based approach for analysis of spatial structure in genetic data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">725</span>–<span class="cit-lpage">731</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DW.%2BY.%26rft.volume%253D44%26rft.issue%253D6%26rft.spage%253D725%26rft.epage%253D731%26rft.atitle%253DA%2Bmodel-based%2Bapproach%2Bfor%2Banalysis%2Bof%2Bspatial%2Bstructure%2Bin%2Bgenetic%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2285%26rft_id%253Dinfo%253Apmid%252F22610118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22610118&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F7%2F1193.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.93" data-doi="10.2135/cropsci2005.08-0293"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yun</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gyenis</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bossolini</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matus</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Validation of quantitative trait loci for multiple disease resistance in barley using advanced backcross lines developed with a wild barley.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1179</span>–<span class="cit-lpage">1186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DYun%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D46%26rft.issue%253D3%26rft.spage%253D1179%26rft.epage%253D1186%26rft.atitle%253DValidation%2Bof%2BQuantitative%2BTrait%2BLoci%2Bfor%2BMultiple%2BDisease%2BResistance%2Bin%2BBarley%2BUsing%2BAdvanced%2BBackcross%2BLines%2BDeveloped%2Bwith%2Ba%2BWild%2BBarley%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2005.08-0293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2005.08-0293&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000237375900021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-4.7.1193.94" data-doi="10.2135/cropsci2005.0236"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yun</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gyenis</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matus</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Quantitative trait loci for multiple disease resistance in wild barley.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">2563</span>–<span class="cit-lpage">2572</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DYun%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D45%26rft.issue%253D6%26rft.spage%253D2563%26rft.epage%253D2572%26rft.atitle%253DQuantitative%2BTrait%2BLoci%2Bfor%2BMultiple%2BDisease%2BResistance%2Bin%2BWild%2BBarley%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2005.0236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2005.0236&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000233147100053&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-book" id="cit-4.7.1193.95"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zohary</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hopf</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-source"><em>Domestication of plants in the Old World: the origin and spread of cultivated plants in West Asia</em>, <em>Europe</em>, <em>and the Nile Valley</em></span>, <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford, UK</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/7/1193.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/7/1183" title="The Insertion Green Monster (iGM) Method for Expression of Multiple Exogenous Genes in Yeast" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/7/1205" title="Mapping Small Effect Mutations in Saccharomyces cerevisiae: Impacts of Experimental Design and Mutational Properties" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/7"><span>Volume 4 Issue 7, July 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/7.cover-source.jpg?itok=MrsYFroL" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/7/1193.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-diSsQ6oCMVa-A1TXn1ZZpdmh6T-TyTkic95ljkaDxUQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/7/1193" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/7/1193" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/7/1193" class="active" data-icon-position="" data-hide-link-title="0">Two Genomic Regions Contribute Disproportionately to Geographic Differentiation in Wild Barley</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459925" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-sHpurRnyCBl54VqhC1zmAIct7MsTNHvKjQ4YM8pWB3k" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/7/1193.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/7/1193" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-PXzoRLD6RnGCx6i_uI1yvKNyd4bvfy5e4D2-ye5ronM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/7/1193" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/7/1193.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459925" id="node459925--2" data-pisa="ggg;4/7/1193" data-pisa-master="ggg;g3.114.010561" data-apath="/ggg/4/7/1193.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Two Genomic Regions Contribute Disproportionately to Geographic Differentiation in Wild Barley</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhou Fang</span>, <span class="highwire-citation-author" data-delta="1">Ana M. Gonzales</span>, <span class="highwire-citation-author" data-delta="2">Michael T. Clegg</span>, <span class="highwire-citation-author" data-delta="3">Kevin P. Smith</span>, <span class="highwire-citation-author" data-delta="4">Gary J. Muehlbauer</span>, <span class="highwire-citation-author" data-delta="5">Brian J. Steffenson</span> and <span class="highwire-citation-author" data-delta="6">Peter L. Morrell</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1193-1203;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010561 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459925--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhou Fang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhou%2BFang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhou%2BFang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ana M. Gonzales </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ana%2BM.%2BGonzales%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gonzales%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAna%2BM.%2BGonzales%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michael T. Clegg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Ecology and Evolutionary Biology, University of California, Irvine, California 92697</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BT.%2BClegg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clegg%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BT.%2BClegg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kevin P. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BP.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20KP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKevin%2BP.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Gary J. Muehlbauer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BJ.%2BMuehlbauer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Muehlbauer%20GJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BJ.%2BMuehlbauer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Brian J. Steffenson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BJ.%2BSteffenson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steffenson%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrian%2BJ.%2BSteffenson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Peter L. Morrell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BL.%2BMorrell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morrell%20PL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BL.%2BMorrell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:pmorrell@umn.edu" class="" data-icon-position="" data-hide-link-title="0">pmorrell@umn.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/7/1193" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/7/1193&amp;title=Two%20Genomic%20Regions%20Contribute%20Disproportionately%20to%20Geographic%20Differentiation%20in%20Wild%20Barley" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/7/1193&amp;title=Two%20Genomic%20Regions%20Contribute%20Disproportionately%20to%20Geographic%20Differentiation%20in%20Wild%20Barley" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/7/1193&amp;title=Two%20Genomic%20Regions%20Contribute%20Disproportionately%20to%20Geographic%20Differentiation%20in%20Wild%20Barley" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/7/1193&amp;text=Two%20Genomic%20Regions%20Contribute%20Disproportionately%20to%20Geographic%20Differentiation%20in%20Wild%20Barley" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/7/1193&amp;title=Two%20Genomic%20Regions%20Contribute%20Disproportionately%20to%20Geographic%20Differentiation%20in%20Wild%20Barley" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/7/1193&amp;t=Two%20Genomic%20Regions%20Contribute%20Disproportionately%20to%20Geographic%20Differentiation%20in%20Wild%20Barley" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/7/1193&amp;title=Two%20Genomic%20Regions%20Contribute%20Disproportionately%20to%20Geographic%20Differentiation%20in%20Wild%20Barley" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/7/1193&amp;title=Two%20Genomic%20Regions%20Contribute%20Disproportionately%20to%20Geographic%20Differentiation%20in%20Wild%20Barley" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459925/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459925" id="node459925--3" data-pisa="ggg;4/7/1193" data-pisa-master="ggg;g3.114.010561" data-apath="/ggg/4/7/1193.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Two Genomic Regions Contribute Disproportionately to Geographic Differentiation in Wild Barley</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhou Fang</span>, <span class="highwire-citation-author" data-delta="1">Ana M. Gonzales</span>, <span class="highwire-citation-author" data-delta="2">Michael T. Clegg</span>, <span class="highwire-citation-author" data-delta="3">Kevin P. Smith</span>, <span class="highwire-citation-author" data-delta="4">Gary J. Muehlbauer</span>, <span class="highwire-citation-author" data-delta="5">Brian J. Steffenson</span> and <span class="highwire-citation-author" data-delta="6">Peter L. Morrell</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1193-1203;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010561 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459925--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhou Fang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhou%2BFang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhou%2BFang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ana M. Gonzales </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ana%2BM.%2BGonzales%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gonzales%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAna%2BM.%2BGonzales%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michael T. Clegg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Ecology and Evolutionary Biology, University of California, Irvine, California 92697</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BT.%2BClegg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clegg%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BT.%2BClegg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kevin P. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BP.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20KP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKevin%2BP.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Gary J. Muehlbauer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BJ.%2BMuehlbauer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Muehlbauer%20GJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BJ.%2BMuehlbauer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Brian J. Steffenson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BJ.%2BSteffenson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steffenson%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrian%2BJ.%2BSteffenson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Peter L. Morrell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BL.%2BMorrell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morrell%20PL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BL.%2BMorrell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:pmorrell@umn.edu" class="" data-icon-position="" data-hide-link-title="0">pmorrell@umn.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1193&amp;rft.epage=1203&amp;rft.atitle=Two%20Genomic%20Regions%20Contribute%20Disproportionately%20to%20Geographic%20Differentiation%20in%20Wild%20Barley&amp;rft.volume=4&amp;rft.issue=7&amp;rft.date=2014-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Fang%2C+Zhou&amp;rft.au=Gonzales%2C+Ana+M.&amp;rft.au=Clegg%2C+Michael+T.&amp;rft.au=Smith%2C+Kevin+P.&amp;rft.au=Muehlbauer%2C+Gary+J.&amp;rft.au=Steffenson%2C+Brian+J.&amp;rft.au=Morrell%2C+Peter+L."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459925/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459925/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459925/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459925/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459925/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459925/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459925/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459925/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459925/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459925/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459925/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459925/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/7/1193" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/7/1193.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/7/1193.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/7/1193.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/7/1193.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
