<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/2/665" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/2/665.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/2/665.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/2/665.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/10/2/665/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/2/665.atom" />
<meta name="HW.pisa" content="ggg;10/2/665" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Quantitative Genomic Dissection of Soybean Yield Components" />
<meta name="DC.Identifier" content="10.1534/g3.119.400896" />
<meta name="DC.Date" content="2020-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Xavier, Rainey. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Soybean is a crop of major economic importance with low rates of genetic gains for grain yield compared to other field crops. A deeper understanding of the genetic architecture of yield components may enable better ways to tackle the breeding challenges. Key yield components include the total number of pods, nodes and the ratio pods per node. We evaluated the SoyNAM population, containing approximately 5600 lines from 40 biparental families that share a common parent, in 6 environments distributed across 3 years. The study indicates that the yield components under evaluation have low heritability, a reasonable amount of epistatic control, and partially oligogenic architecture: 18 quantitative trait loci were identified across the three yield components using multi-approach signal detection. Genetic correlation between yield and yield components was highly variable from family-to-family, ranging from -0.2 to 0.5. The genotype-by-environment correlation of yield components ranged from -0.1 to 0.4 within families. The number of pods can be utilized for indirect selection of yield. The selection of soybean for enhanced yield components can be successfully performed via genomic prediction, but the challenging data collections necessary to recalibrate models over time makes the introgression of QTL a potentially more feasible breeding strategy. The genomic prediction of yield components was relatively accurate across families, but less accurate predictions were obtained from within family predictions and predicting families not observed included in the calibration set." />
<meta name="DC.Contributor" content="Alencar Xavier" />
<meta name="DC.Contributor" content="Katy M. Rainey" />
<meta name="article:published_time" content="2020-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Quantitative Genomic Dissection of Soybean Yield Components" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Soybean is a crop of major economic importance with low rates of genetic gains for grain yield compared to other field crops. A deeper understanding of the genetic architecture of yield components may enable better ways to tackle the breeding challenges. Key yield components include the total number of pods, nodes and the ratio pods per node. We evaluated the SoyNAM population, containing approximately 5600 lines from 40 biparental families that share a common parent, in 6 environments distributed across 3 years. The study indicates that the yield components under evaluation have low heritability, a reasonable amount of epistatic control, and partially oligogenic architecture: 18 quantitative trait loci were identified across the three yield components using multi-approach signal detection. Genetic correlation between yield and yield components was highly variable from family-to-family, ranging from -0.2 to 0.5. The genotype-by-environment correlation of yield components ranged from -0.1 to 0.4 within families. The number of pods can be utilized for indirect selection of yield. The selection of soybean for enhanced yield components can be successfully performed via genomic prediction, but the challenging data collections necessary to recalibrate models over time makes the introgression of QTL a potentially more feasible breeding strategy. The genomic prediction of yield components was relatively accurate across families, but less accurate predictions were obtained from within family predictions and predicting families not observed included in the calibration set.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/02/01" />
<meta name="citation_mjid" content="ggg;10/2/665" />
<meta name="citation_id" content="10/2/665" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/2/665" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/2/665.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/2/665.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/2/665.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400896" />
<meta name="citation_pmid" content="31818873" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="665" />
<meta name="citation_lastpage" content="675" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Alencar Xavier" />
<meta name="citation_author_institution" content="Department of Agronomy, Purdue University, West Lafayette IN 47907 and" />
<meta name="citation_author_institution" content="Department of Biostatistics, Corteva Agrisciences, Johnston IA 50131" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5034-9954" />
<meta name="citation_author" content="Katy M. Rainey" />
<meta name="citation_author_institution" content="Department of Agronomy, Purdue University, West Lafayette IN 47907 and" />
<meta name="citation_author_email" content="xaviera@purdue.edu" />
<meta name="citation_author_email" content="krainey@purdue.edu" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. A. Ball;citation_author=L. C. Purcell;citation_author=E. D. Vories;citation_title=Short-Season Soybean Yield Compensation in Response to Population and Water Regime;citation_pages=1070-1078;citation_volume=40;citation_year=2000;citation_issue=4;citation_doi=10.2135/cropsci2000.4041070x" />
<meta name="citation_reference" content="Bates, D., M. Mächler, B. Bolker, and S. Walker, 2014 Fitting linear mixed-effects models using lme4. arXiv preprint arXiv:1406.5823." />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. L. Bernard;citation_title=Two Genes Affecting Stem Termination in Soybeans;citation_pages=235-239;citation_volume=12;citation_year=1972;citation_issue=2;citation_doi=10.2135/cropsci1972.0011183X001200020028x" />
<meta name="citation_reference" content="citation_title=Soybean Yield Formation: What controls it and how it can be improved. Soybean Physiology and Biochemistry;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. E. Board;citation_author=H. Modali;citation_title=Dry Matter Accumulation Predictors for Optimal Yield in Soybean;citation_pages=1790-1799;citation_volume=45;citation_year=2005;citation_issue=5;citation_doi=10.2135/cropsci2004.0602" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. E. Board;citation_author=M. S. Kang;citation_author=B. G. Harville;citation_title=Path Analyses Identify Indirect Selection Criteria for Yield of Late-Planted Soybean;citation_pages=879-884;citation_volume=37;citation_year=1997;citation_issue=3;citation_doi=10.2135/cropsci1997.0011183X003700030030x" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. E. Board;citation_author=Q. Tan;citation_title=Assimilatory Capacity Effects on Soybean Yield Components and Pod Number;citation_pages=846-851;citation_volume=35;citation_year=1995;citation_issue=3;citation_doi=10.2135/cropsci1995.0011183X003500030035x" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. E. Board;citation_author=B. G. Harville;citation_title=Soybean Yield Component Responses to a Light Interception Gradient during the Reproductive Period;citation_pages=772-777;citation_volume=33;citation_year=1993;citation_issue=4;citation_doi=10.2135/cropsci1993.0011183X003300040028x" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=V. Botta;citation_author=G. Louppe;citation_author=P. Geurts;citation_author=L. Wehenkel;citation_title=Exploiting SNP correlations within random forest for genome-wide association studies.;citation_volume=9;citation_year=2014;citation_doi=10.1371/journal.pone.0093379" />
<meta name="citation_reference" content="Carter, T. E., R. L. Nelson, C. H. Sneller, Z. Cui, H. R. Boerma, and J. E. Specht, 2004 Genetic diversity in soybean. Soybeans: Improvement, production, and uses." />
<meta name="citation_reference" content="citation_journal_title=Stat. Med.;citation_author=J. Chipman;citation_author=D. Braun;citation_title=Simpson’s paradox in the integrated discrimination improvement.;citation_pages=4468-4481;citation_volume=36;citation_year=2017;citation_doi=10.1002/sim.6862" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. Daetwyler;citation_author=M. P. Calus;citation_author=R. Pong-Wong;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_title=Genomic prediction in animals and plants: simulation of data, validation, reporting, and benchmarking.;citation_volume=193;citation_year=2013;citation_pmid=23222650;citation_doi=10.1534/genetics.112.147983" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. W. Diers;citation_author=J. Specht;citation_author=K. M. Rainey;citation_author=P. Cregan;citation_author=Q. Song;citation_title=Genetic Architecture of Soybean Yield and Agronomic Traits. G3: Genes, Genomes;citation_pages=3367-3375;citation_volume=8;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. W. Doerge;citation_author=G. A. Churchill;citation_title=Permutation Tests for Multiple Loci Affecting a Quantitative Character;citation_pages=285-294;citation_volume=142;citation_year=1996;citation_issue=1;citation_pmid=8770605" />
<meta name="citation_reference" content="citation_journal_title=Eur. J. Agron.;citation_author=D. B. Egli;citation_author=W. P. Bruening;citation_title=Temporal profiles of pod production and pod set in soybean.;citation_pages=11-18;citation_volume=24;citation_year=2006;citation_doi=10.1016/j.eja.2005.04.006" />
<meta name="citation_reference" content="citation_title=Introduction to quantitative genetics;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=C. Fang;citation_author=Y. Ma;citation_author=S. Wu;citation_author=Z. Liu;citation_author=Z. Wang;citation_title=Genome-wide association studies dissect the genetic networks underlying agronomical traits in soybean.;citation_pages=161;citation_volume=18;citation_year=2017;citation_pmid=28838319;citation_doi=10.1186/s13059-017-1289-9" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=W. R. Fehr;citation_author=C. E. Caviness;citation_author=D. T. Burmood;citation_author=J. S. Pennington;citation_title=Stage of Development Descriptions for Soybeans, Glycine Max (L.) Merrill;citation_pages=929-931;citation_volume=11;citation_year=1971;citation_issue=6;citation_doi=10.2135/cropsci1971.0011183X001100060051x" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=D. J. Garrick;citation_author=J. F. Taylor;citation_author=R. L. Fernando;citation_title=Deregressing estimated breeding values and weighting information for genomic regression analyses.;citation_pages=55-55;citation_volume=41;citation_year=2009;citation_pmid=20043827;citation_doi=10.1186/1297-9686-41-55" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=D. J. Garrick;citation_title=Genomic BLUP decoded: a look into the black box of genomic prediction.;citation_volume=194;citation_year=2013;citation_pmid=23640517;citation_doi=10.1534/genetics.113.152207" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=K. Kizilkaya;citation_author=D. J. Garrick;citation_title=Extension of the bayesian alphabet for genomic selection.;citation_pages=186-186;citation_volume=12;citation_year=2011;citation_pmid=21605355;citation_doi=10.1186/1471-2105-12-186" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=D. Hao;citation_author=H. Cheng;citation_author=Z. Yin;citation_author=S. Cui;citation_author=D. Zhang;citation_title=Identification of single nucleotide polymorphisms and haplotypes associated with yield and yield components in soybean (Glycine max) landraces across multiple environments.;citation_pages=447-458;citation_volume=124;citation_year=2012;citation_issue=3;citation_pmid=21997761;citation_doi=10.1007/s00122-011-1719-0" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=S. J. Herbert;citation_author=G. V. Litchfield;citation_title=Partitioning Soybean Seed Yield Components;citation_pages=1074-1079;citation_volume=22;citation_year=1982;citation_issue=5;citation_doi=10.2135/cropsci1982.0011183X002200050044x" />
<meta name="citation_reference" content="citation_journal_title=Breed. Sci.;citation_author=Z. Hu;citation_author=D. Zhang;citation_author=G. Zhang;citation_author=G. Kan;citation_author=D. Hong;citation_title=Association mapping of yield-related traits and SSR markers in wild soybean (Glycine soja Sieb. and Zucc.).;citation_pages=441-449;citation_volume=63;citation_year=2014;citation_doi=10.1270/jsbbs.63.441" />
<meta name="citation_reference" content="citation_journal_title=Agron. J.;citation_author=C. S. Kahlon;citation_author=J. E. Board;citation_author=M. S. Kang;citation_title=An analysis of yield component changes for new vs. old soybean cultivars.;citation_pages=13-22;citation_volume=103;citation_year=2011;citation_doi=10.2134/agronj2010.0300" />
<meta name="citation_reference" content="citation_journal_title=J. Crop Improv.;citation_author=C. S. Kahlon;citation_author=J. E. Board;citation_title=Growth dynamic factors explaining yield improvement in new vs. old soybean cultivars.;citation_pages=282-299;citation_volume=26;citation_year=2012;citation_doi=10.1080/15427528.2011.637155" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=A. S. Kaler;citation_author=J. D. Ray;citation_author=W. T. Schapaugh;citation_author=M. K. Davies;citation_author=C. A. King;citation_title=Association mapping identifies loci for canopy coverage in diverse soybean genotypes.;citation_pages=50;citation_volume=38;citation_year=2018;citation_doi=10.1007/s11032-018-0810-5" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. Lado;citation_author=I. Matus;citation_author=A. Rodríguez;citation_author=L. Inostroza;citation_author=J. Poland;citation_title=Increased genomic prediction accuracy in wheat breeding through spatial adjustment of field trial data. G3: Genes, Genomes;citation_pages=2105-2114;citation_volume=3;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=T. Langewisch;citation_author=H. Zhang;citation_author=R. Vincent;citation_author=T. Joshi;citation_author=D. Xu;citation_title=Major soybean maturity gene haplotypes revealed by SNPViz analysis of 72 sequenced soybean genomes.;citation_pages=e94150;citation_volume=9;citation_year=2014;citation_doi=10.1371/journal.pone.0094150" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Lehermeier;citation_author=N. Krämer;citation_author=E. Bauer;citation_author=C. Bauland;citation_author=C. Camisan;citation_title=Usefulness of Multiparental Populations of Maize (Zea mays L.) for Genome-Based Prediction;citation_pages=3-16;citation_volume=198;citation_year=2014;citation_issue=1;citation_pmid=25236445;citation_doi=10.1534/genetics.114.161943" />
<meta name="citation_reference" content="citation_journal_title=IEEE Access;citation_author=Y. Li;citation_author=J. Jia;citation_author=L. Zhang;citation_author=A. M. Khattak;citation_author=S. Sun;citation_title=Soybean Seed Counting Based on Pod Image Using Two-Column Convolution Neural Network.;citation_pages=64177-64185;citation_volume=7;citation_year=2019;citation_doi=10.1109/ACCESS.2019.2916931" />
<meta name="citation_reference" content="citation_journal_title=Oikos;citation_author=T. Malausa;citation_author=T. Guillemaud;citation_author=L. Lapchin;citation_title=Combining genetic variation and phenotypic plasticity in tradeoff modelling.;citation_pages=330-338;citation_volume=110;citation_year=2005;citation_doi=10.1111/j.0030-1299.2005.13563.x" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. A. Mikel;citation_author=B. W. Diers;citation_author=R. L. Nelson;citation_author=H. H. Smith;citation_title=Genetic Diversity and Agronomic Improvement of North American Soybean Germplasm;citation_pages=1219-1229;citation_volume=50;citation_year=2010;citation_issue=4;citation_doi=10.2135/cropsci2009.08.0456" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=I. Misztal;citation_title=Reliable computing in estimation of variance components.;citation_pages=363-370;citation_volume=125;citation_year=2008;citation_issue=6;citation_pmid=19134071;citation_doi=10.1111/j.1439-0388.2008.00774.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Meuwissen;citation_author=B. Hayes;citation_author=M. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=Field Crops Res.;citation_author=M. Nico;citation_author=D. J. Miralles;citation_author=A. G. Kantolic;citation_title=Natural post-flowering photoperiod and photoperiod sensitivity: Roles in yield-determining processes in soybean.;citation_pages=141-152;citation_volume=231;citation_year=2019;citation_doi=10.1016/j.fcr.2018.10.019" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=F. Ogut;citation_author=Y. Bian;citation_author=P. J. Bradbury;citation_author=J. B. Holland;citation_title=Joint-multiple family linkage analysis predicts within-family variation better than single-family analysis of the maize nested association mapping population.;citation_pages=552-563;citation_volume=114;citation_year=2015;citation_pmid=25585918;citation_doi=10.1038/hdy.2014.123" />
<meta name="citation_reference" content="citation_journal_title=Agronomy Journal;citation_journal_abbrev=Agron. J.;citation_author=P. Pedersen;citation_author=J. G. Lauer;citation_title=Response of Soybean Yield Components to Management System and Planting Date;citation_pages=1372-1381;citation_volume=96;citation_year=2004;citation_issue=5;citation_doi=10.2134/agronj2004.1372" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pérez;citation_author=G. de Los Campos;citation_title=Genome-wide regression and prediction with the BGLR statistical package.;citation_volume=198;citation_year=2014;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=J. Ping;citation_author=Y. Liu;citation_author=L. Sun;citation_author=M. Zhao;citation_author=Y. Li;citation_title=Dt2 is a gain-of-function MADS-domain factor gene that specifies semi-determinacy in soybean.;citation_volume=26;citation_year=2014;citation_pmid=25005919;citation_doi=10.1105/tpc.114.126938" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=K. Rincker;citation_author=R. Nelson;citation_author=J. Specht;citation_author=D. Sleper;citation_author=T. Cary;citation_title=Genetic improvement of US soybean in maturity groups II, III, and IV.;citation_pages=1419-1432;citation_volume=54;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Agronomy Journal;citation_journal_abbrev=Agron. J.;citation_author=A. P. Robinson;citation_author=S. P. Conley;citation_author=J. J. Volenec;citation_author=J. B. Santini;citation_title=Analysis of High Yielding, Early-Planted Soybean in Indiana;citation_pages=131-139;citation_volume=101;citation_year=2009;citation_issue=1;citation_doi=10.2134/agronj2008.0014x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Schopp;citation_author=D. Müller;citation_author=Y. C. Wientjes;citation_author=A. E. Melchinger;citation_title=Genomic prediction within and across biparental families: means and variances of prediction accuracy and usefulness of deterministic equations. G3: Genes, Genomes;citation_pages=3571-3586;citation_volume=7;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=E. J. Sedivy;citation_author=F. Wu;citation_author=Y. Hanzawa;citation_title=Soybean domestication: the origin, genetic architecture and molecular bases.;citation_pages=539-553;citation_volume=214;citation_year=2017;citation_doi=10.1111/nph.14418" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=A. K. Singh;citation_author=B. Ganapathysubramanian;citation_author=S. Sarkar;citation_author=A. Singh;citation_title=Deep learning for plant stress phenotyping: trends and future perspectives.;citation_pages=883-898;citation_volume=23;citation_year=2018;citation_doi=10.1016/j.tplants.2018.07.004" />
<meta name="citation_reference" content="Specht, J. E., B. W. Diers, R. L. Nelson, J. Francisco, F. de Toledo, J. A. Torrion, and P. Grassini, 2014 Soybean. Yield gains in major US field crops, 311–356." />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. E. Specht;citation_author=D. J. Hume;citation_author=S. V. Kumudini;citation_title=Soybean Yield Potential--A Genetic and Physiological Perspective;citation_pages=1560-1570;citation_volume=39;citation_year=1999;citation_issue=6;citation_doi=10.2135/cropsci1999.3961560x" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=Q. Song;citation_author=L. Yan;citation_author=C. Quigley;citation_author=B. D. Jordan;citation_author=E. Fickus;citation_title=Genetic characterization of the soybean nested association mapping population.;citation_volume=10;citation_year=2017;citation_doi=10.3835/plantgenome2016.10.0109" />
<meta name="citation_reference" content="citation_journal_title=Agron. J.;citation_author=J. J. Suhre;citation_author=N. H. Weidenbenner;citation_author=S. C. Rowntree;citation_author=E. W. Wilson;citation_author=S. L. Naeve;citation_title=Soybean yield partitioning changes revealed by genetic gain and seeding rate interactions.;citation_pages=1631-1642;citation_volume=106;citation_year=2014;citation_doi=10.2134/agronj14.0003" />
<meta name="citation_reference" content="citation_journal_title=Plant Mol. Biol.;citation_author=L. J. Qiu;citation_author=L. L. Xing;citation_author=Y. Guo;citation_author=J. Wang;citation_author=S. A. Jackson;citation_title=A platform for soybean molecular breeding: the utilization of core collections for food security.;citation_pages=41-50;citation_volume=83;citation_year=2013;citation_doi=10.1007/s11103-013-0076-6" />
<meta name="citation_reference" content="citation_journal_title=Comput. Electron. Agric.;citation_author=L. C. Uzal;citation_author=G. L. Grinblat;citation_author=R. Namías;citation_author=M. G. Larese;citation_author=J. S. Bianchi;citation_title=Seed-per-pod estimation for plant breeding using deep learning.;citation_pages=196-204;citation_volume=150;citation_year=2018;citation_doi=10.1016/j.compag.2018.04.024" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Xavier;citation_author=W. M. Muir;citation_author=K. M. Rainey;citation_title=bWGR: Bayesian Whole-Genome Regression.;citation_year=2019;citation_doi=10.1093/bioinformatics/btz794" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Xavier;citation_author=D. Jarquin;citation_author=R. Howard;citation_author=V. Ramasubramanian;citation_author=J. E. Specht;citation_title=Genome-Wide analysis of grain yield stability and environmental interactions in a multiparental soybean population. G3: Genes, Genomes;citation_pages=519-529;citation_volume=8;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=A. Xavier;citation_author=B. Hall;citation_author=S. Casteel;citation_author=W. Muir;citation_author=K. M. Rainey;citation_title=Using unsupervised learning techniques to assess interactions among complex traits in soybeans.;citation_pages=200;citation_volume=213;citation_year=2017;citation_doi=10.1007/s10681-017-1975-4" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Xavier;citation_author=B. Hall;citation_author=A. A. Hearst;citation_author=K. A. Cherkauer;citation_author=K. M. Rainey;citation_title=Genetic architecture of phenomic-enabled canopy coverage in Glycine max.;citation_volume=206;citation_year=2017;citation_pmid=28363978;citation_doi=10.1534/genetics.116.198713" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Xavier;citation_author=W. M. Muir;citation_author=K. M. Rainey;citation_title=Assessing predictive properties of genome-wide selection in soybeans. G3: Genes, Genomes;citation_pages=2611-2616;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Xavier;citation_author=S. Xu;citation_author=W. M. Muir;citation_author=K. M. Rainey;citation_title=NAM: association studies in multiple populations.;citation_volume=31;citation_year=2015;citation_pmid=26243017;citation_doi=10.1093/bioinformatics/btv448" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Xu;citation_title=Mapping quantitative trait loci by controlling polygenic background effects.;citation_volume=195;citation_year=2013;citation_pmid=24077303;citation_doi=10.1534/genetics.113.157032" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Watanabe;citation_author=Z. Xia;citation_author=R. Hideshima;citation_author=Y. Tsubokura;citation_author=S. Sato;citation_title=A map-based cloning strategy employing a residual heterozygous line reveals that the GIGANTEA gene is involved in soybean maturity and flowering.;citation_volume=188;citation_year=2011;citation_pmid=21406680;citation_doi=10.1534/genetics.110.125062" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Watanabe;citation_author=R. Hideshima;citation_author=Z. Xia;citation_author=Y. Tsubokura;citation_author=S. Sato;citation_title=Map-based cloning of the gene associated with the soybean maturity locus E3.;citation_volume=182;citation_year=2009;citation_pmid=19474204;citation_doi=10.1534/genetics.108.098772" />
<meta name="citation_reference" content="Wright, M. N., and A. Ziegler, 2015 Ranger: a fast implementation of random forests for high dimensional data in C++ and R. arXiv preprint arXiv:1508.04409." />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Z. Xia;citation_author=S. Watanabe;citation_author=T. Yamada;citation_author=Y. Tsubokura;citation_author=H. Nakashima;citation_title=Positional cloning and characterization reveal the molecular basis for soybean maturity locus E1 that regulates photoperiodic flowering.;citation_volume=109;citation_year=2012;citation_pmid=22619331;citation_doi=10.1073/pnas.1117982109" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Yang;citation_author=R. K. Ramamurthy;citation_author=X. Qi;citation_author=R. L. Fernando;citation_author=J. C. Dekkers;citation_title=Empirical Comparisons of Different Statistical Models To Identify and Validate Kernel Row Number-Associated Variants from Structured Multi-parent Mapping Populations of Maize. G3: Genes, Genomes;citation_pages=3567-3575;citation_volume=8;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. B. Zeng;citation_author=T. Wang;citation_author=W. Zou;citation_title=Modeling quantitative trait loci and interpretation of models.;citation_volume=169;citation_year=2005;citation_pmid=15654105;citation_doi=10.1534/genetics.104.035857" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=H. Zhang;citation_author=D. Hao;citation_author=H. M. Sitoe;citation_author=Z. Yin;citation_author=Z. Hu;citation_title=Genetic dissection of the relationship between plant architecture and yield component traits in soybean (Glycine max) by association analysis across multiple environments.;citation_pages=564-572;citation_volume=134;citation_year=2015;citation_doi=10.1111/pbr.12305" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=W. K. Zhang;citation_author=Y. J. Wang;citation_author=G. Z. Luo;citation_author=J. S. Zhang;citation_author=C. Y. He;citation_title=QTL mapping of ten agronomic traits on the soybean ( Glycine max L. Merr.) genetic map and their association with EST markers.;citation_pages=1131-1139;citation_volume=108;citation_year=2004;citation_issue=6;citation_pmid=15067400;citation_doi=10.1007/s00122-003-1527-2" />
<meta name="citation_reference" content="citation_year=2019" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Quantitative Genomic Dissection of Soybean Yield Components" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
<meta name="twitter:description" content="Soybean is a crop of major economic importance with low rates of genetic gains for grain yield compared to other field crops. A deeper understanding of the genetic architecture of yield components may enable better ways to tackle the breeding challenges. Key yield components include the total number of pods, nodes and the ratio pods per node. We evaluated the SoyNAM population, containing approximately 5600 lines from 40 biparental families that share a common parent, in 6 environments distributed across 3 years. The study indicates that the yield components under evaluation have low heritability, a reasonable amount of epistatic control, and partially oligogenic architecture: 18 quantitative trait loci were identified across the three yield components using multi-approach signal detection. Genetic correlation between yield and yield components was highly variable from family-to-family, ranging from -0.2 to 0.5. The genotype-by-environment correlation of yield components ranged from -0.1 to 0.4 within families. The number of pods can be utilized for indirect selection of yield. The selection of soybean for enhanced yield components can be successfully performed via genomic prediction, but the challenging data collections necessary to recalibrate models over time makes the introgression of QTL a potentially more feasible breeding strategy. The genomic prediction of yield components was relatively accurate across families, but less accurate predictions were obtained from within family predictions and predicting families not observed included in the calibration set." />
<meta name="og-title" property="og:title" content="Quantitative Genomic Dissection of Soybean Yield Components" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/2/665" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Soybean is a crop of major economic importance with low rates of genetic gains for grain yield compared to other field crops. A deeper understanding of the genetic architecture of yield components may enable better ways to tackle the breeding challenges. Key yield components include the total number of pods, nodes and the ratio pods per node. We evaluated the SoyNAM population, containing approximately 5600 lines from 40 biparental families that share a common parent, in 6 environments distributed across 3 years. The study indicates that the yield components under evaluation have low heritability, a reasonable amount of epistatic control, and partially oligogenic architecture: 18 quantitative trait loci were identified across the three yield components using multi-approach signal detection. Genetic correlation between yield and yield components was highly variable from family-to-family, ranging from -0.2 to 0.5. The genotype-by-environment correlation of yield components ranged from -0.1 to 0.4 within families. The number of pods can be utilized for indirect selection of yield. The selection of soybean for enhanced yield components can be successfully performed via genomic prediction, but the challenging data collections necessary to recalibrate models over time makes the introgression of QTL a potentially more feasible breeding strategy. The genomic prediction of yield components was relatively accurate across families, but less accurate predictions were obtained from within family predictions and predicting families not observed included in the calibration set." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
    <title>Quantitative Genomic Dissection of Soybean Yield Components | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"-Hdc8Mp4ruf7IjY54K1Kgo6AYFRtoyvuVzY4NRvUFM4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495558","apath":"\/ggg\/10\/2\/665.atom","pisa":"ggg;10\/2\/665","ac":{"\/ggg\/10\/2\/665.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/2\/665.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/2\/665"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/2\/665"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/2\/665":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022rDX+eYaUhf406nGG4pn8ldPt4BOU\\\\\\\/bU6W\\\\\\\/vBVj03LvPYq9KGuk8Db1Z8VZjtKsqRDjttQwLYrPc7snA1HLPzboOwKPv\\\\\\\/xvg2DNCkyVezB4vIp1fx8SEbA7mLSr7ZK6eX7Wd38W0VwkA34YZYguJcwK3v\\\\\\\/zd48HYrq7bF6wu9Dc9WH6kI6lwyzoW7I4gpJWv5eVKpweii74NPdBrPJlBYg\\\\\\\/\\\\\\\/x3NC5xdHkK8waUQYne1bZS9Sp9npOEHkRUvBkqK3U+m67j5AtHlavCRJPk2T41fGwIpdIvrxO91VwLxpbKXgWbOXDw9bHh2LeeUGbwpvN\\\\\\\/NQxT37BqlsEBXL2fk0TyBGuo99EpdwVnt820bmhHnMzHvyi2fMTkMbMb25pn1rq4i+WdjZqQIlU+D4f1NGT8jCwEF2evd0d\\\\\\\/+a3Sy99A0DN7AkNVi+mnMLAtus4rR76xHBRDZ4g5tfnmAcFtiQiH39EO5ud5Y7dC+5UDmAu3OdnzyjNyguw9CC3ZmPvs5bJqhJ6Yd5JnxsOD6mJ2kqUgq+xUHH7MxCOQYMrJ4rmE3Zob8fNHW8eUZpGpAP3h7zM+ixSa\\\\\\\/7LcxA4HcZthgzIEneSF28BSdXOiVVETEuEnzgbM5KezphkTFxnaFiKocZfLRPX9uect\\\\\\\/h5VY90ZdM30aXa5YtrL0zEJedXMNkIq3WUKeaVgOKyinjUyBmTy6744F5U8sXKjpVcE+VVj3aJUnU3q8p+cQqdQFP\\\\\\\/40UmgSPMXKr05Jgflj0lC07buDuPun3wtWSRJEKS0wbw6OSSZtS9O6Y7fT03kTf\\\\\\\/6F7KMtIwjjaMPCjpZetsDh2Q+BBBrr5CCkBN8ZGgaPl0V85jWBg3oixWwnDWH8TVtwpu1solpKhdiw5f688Pfzq2WgpzrVQVFH1GzJ\\\\\\\/Jczx4Tb+zvR0V0iEMXY8eVhKXRtNx6J\\\\\\\/1oYsFUNKkYW7CHVAy7TKp9cvPBULREjigaRotGRkB4pJqMSgHhbIe1zrQFhTTw98uemCwGvlaoaGTiFu+Xc0PBkiO8EX6AdJ2tk9eZ1z7Gb2ev\\\\\\\/9wId17JXc+1r97FiXCh0oqv6i40ykpimkGOgt\\\\\\\/+o03Bn\\\\\\\/x4pUZFs9IOl\\\\\\\/gh15fG2aycH232rlJowfprcJOOKtdtaMeJPdo\\\\\\\/kAusSxWrRMBoqy1xGJGvqvLtirsmvLHIckrZsuiMyG\\\\\\\/trlqftwaD5ukOIXIlOyJ5xCTlK29SydSD0yaaIoL1i1YHNYm\\\\\\\/91BbmlE8IZPUDo5nwX9+G0C467IU+4tjGaPbJ2W18UI\\\u0022,\\\u0022iv\\\u0022:\\\u0022jnsetku62ltROzDpzxKWNA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002266643fc0b79bb79f32fe95142dda671ba1e4dd8f6576f002d3894ea817837326\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022l\\\\\\\/vBXpklyfXF5gEPAIgZmT\\\\\\\/Mytfef003i9M9Lzn\\\\\\\/QcaM3EqBgL+rUx1YwffCAzf3VY0kgNcLEjadcd88K+W1\\\\\\\/vstuUeHvTQ30dpAoUOcp\\\\\\\/WvCzWqIbMW356sYdMVaFWyEoZnum3RnSucz+nUaIGUw9ejYMMOICKEjfZHhfDbNqDoREVzZQQB2nYmVTHiYDCLQO2G5bWWyvbi6RLLOlb2q1r3ZvUqf\\\\\\\/G5q3Bp2RaHnEPW1BQZ\\\\\\\/7fSTTmQf3XU4So75PpyjoUYPdZH1eS0ZiLeqwktyaW89THklc+ttm1WU7q7VXTCZCaaYy9J3SPD8dibQQXiMZmBSvSqXNJS97irvYFbTeShGpmpiiPnbGajLx9D2a+DeiaCAgAkD85LGw5H6ObRAhMCizIAJcKVvxEhfpp9KMq+waXIoldgyk1zlhkgD0VrnxY28Q23vWU+qPm\\\\\\\/uUPfLJR5gbXEziyvSQf05uO5Y3gptYvTdGilVuaEmjtVaJO8kNZoTZZXxMxTC7Z1ly6vwljHDIs8JVaIEsYdTRkwTPOUqjjOMWuZ6SMHY2Yekf2Ex7S7dmgeE2Uxx+2rQQMCMY+IfsVAZZdjCEWO7+rci090UdaIEz78z30bh5WjaSWJrQYwCEoh0vY8Y5mf6JqD804xUdj6OyNbitGF6QivR\\\\\\\/FeOdnTnp8ftRZ77VhDuQrd9d2JhqgeIJzftBQGr5DAFuqozwNjoRGFYH\\\\\\\/x4gJCkBdW87ehYbcyx9DJI1YsxvHhnTMR37crmXgXwpWcxGMMsbkFDz22i6v85rS90Colmp\\\\\\\/asQkpx2xZvcYfMKHkIqxGnAE63P15tHMtjEBNxTyEe0GGdRNS1ZhQbb6k+ytFGopT3YYKZjtKBf5hPZ5OINIFkdWf5tu4TQV4mA3qiqZ1ZlLDOpoRGMv4U2gLlADXRc67VO313aYa+dwpDqOYkD1pPe07AJfihRL7L+AH7rHPv4ygZxArkYqKETokHKUWuClRcagUqwtNMe46nFTiUl5m7PppRA4d\\\\\\\/HdXcAu7UMudBziKrzSv6W9FCrFPTZKTMHWkmKJzFASJWbKwzAYX95\\\\\\\/+OL\\\\\\\/bYb1lsuDx2iUWFqzGHm+QxOLi+WmtPM5JoQlfNYLGG4UFmER1pr8pM\\\\\\\/jyTOvYnNfYvJzpuh7Pz+AeBtjJChieyaBroRbegHy7d5QHTTkd3ssDCQ7JZ4Fb6JpxRBOAD\\\\\\\/yu1AiS1sFjJ7TbGj0rkbo6OJJ9iQHji1nSY+k6Lcx7uDh7XFhezz+3aEMl5RVM+kqAOvQ8UU8u6ncPBI0wL8FE4d69bMDG\\\\\\\/0b9OzduocOBndk9n3EaxWUcYlXUH90Mn81CRLnZ9AKwu0HGnfHkjeTEQc4UgvFQHo1qsRgjBxMkJn3xCYNjU2FPU3vRnw1cElVN+Tat1RajRaA7DkCWPfZ80cg\\\\\\\/24M2iqiNaOkvE7lPhGRRlRF5HWb0+pszt+VmvKZa3RVQCwxyn26Kf6vK5ufLckVgRU22R60WvQdHskBNU3kXN46\\\\\\\/\\\\\\\/wcofFWAHb++x8xBZv3InjoNnUNe1EiOxfi3wnHfBUhTREGsS5URPkdYxvzjiJXXt4d34Hms\\\\\\\/7JN4Z1AJpRVE\\\\\\\/WH0z3YPV6PAyz3Dqiqli+cEg3VupeWC6Yw6tmL\\\\\\\/tJ07pk0oLnBgy8TBYyh5ANnAzXnml+lI8LgIzGr2jLRd23NXKlfbP0JNDxtHizSSM4YHPPUcTDCCGkcbAwcb9GkzrayYiDoWROaD3BXhYph69nx\\\\\\\/iUAp+bF2sT1Bgps0QIjzxp7GhROfFYd7bsUaL95hoECU1gM+YbGPubcpUTHEmAWAXy2OwwR+xafYPmGbaQFjf78dWym3PCkU3ug0+M2EzVz9K7wm817NCkmBkYUMaoW69dP4fwD0xKgIuCweT8XXThpy+MWETbO5m0MplSwIiuMRbpsAMahbAWNlPITV3acjDCExLgLs2n1ZGuCycWlfiRn94uVm4fVSAchsJ3lzBaFZhsNw6qf9qmvWzfRITR8B3gyXf2K+UHgd+H15ymgBXHv9q3s0p9DlVDd9x54\\\\\\\/JpH4zoqbutYM9hpCwD+BluYXY\\\\\\\/0G6r01eZh\\\\\\\/Izi4VwuB6oA2caCddIDDQoqYEH7MndORLtDSIeAUp4jE06AOXOMldH7Y1XwqdnSzLM4K4kANZ5wm2kLuz3huwr8pogNoimgEdzlm4ZxjQS2oPr9BTVX+YLVmS+LPonsuC4rK5QGWhK6nPKsNRcWiLD9lSNnayFiWu5X\\\\\\\/VXwDRVZ1uUDCSLBCxXjP6RzV6lOeqXNKbgvl\\\\\\\/byP0fbQTpUteKWsAJwZZ81RxUKVNEdxJE55rXvLp2ER3znraMs7sLZpw84kRn5Fj4hYHHOllcAwFq+UteiG4TIse3eG6iq1WTZoAMbE5NPqn4uzCOK034ZjvA42SqEXHGmPmpPIIUu4GFUupMv2LaVwurG2eO0sREuMJMBINJTnbNvmSznnhH7vYq24twcfQ59NE8AjUJbS9eySwu5txqsa5qNDLH5p0090bcpQddugdAaWXYaW\\\\\\\/uWVkq2EHP7TGep2aja63i6d1yjdvQs4NTZJ7iPVF3yXMKSCtp9vx4lCsl+HI8yGGonNcThPuJ6VSaX1W9YHiapO71VPheqhjSVqbZjD2AGJm3yPKfmGscHRRkFWpVe5sxNzC6rFc0yh6UENQwZBJ\\\\\\\/V14EyUABF\\\\\\\/KMdfn5NHnWS+Ic82BoqqQDofiQbVQFW2jPhGuM1uaaPsxE4DpDaS6sJ8O13bE0YkD3aBpdKlTVFI2zRVQu\\\\\\\/Ssh7zNVWmLMELzQ9ULY+FDyQoNYXinU9Zj7+ljr+Tf6eW0sucPbGBhJrzDpJE0TEZO0o7VTUDrIrO6\\\\\\\/6xQFSIRQw+Qr458orwbxIiHoLA9u9OjX7TNMciJzmki4NNW5r6E5ME7kiAlhHIha436WGyf6+oTui9Em9dwrVLTiurzoErriEeLgiObYnnmVQRhkHnhCBfOulhB59foAES88mH\\\\\\\/r9IGgZ+IokmhIAHQjSiDU\\\\\\\/tXfy7P9HTlLTG5yVKEInSC0cpxJ+QXBypdnDUCudqV2ogF5WwFxlNO44mXSY18PaxgRg+PiziqHvEeGm7hbfJDaGsCuNEsjnXg9QEpC6X89zz+NRNLWFxHiEgzULgZhzp1\\\\\\\/nTjxMlJ381fDpTWgsJ7+P1pdDLyzwSppKpTaPJ7fQYwjdbjdCw2TAHepYiMizvii+endWGLaqbJPAJmuKAze7lkjGtKGD82yBT7bGVIBPzCC3BNy9Xd2Zqy6JEvVTAwRUrg0Tv\\\\\\\/voBu1xgA16G76sve0PHZV1eneOS+ikiRLJJF7Hj7lR1biOhNRPSCD4CDw8eSr1emwmKErjpmG07oOXqVKTaThb7ZljwJYwrvYB0n0qDFwUYrIjQr9z++mfybvBaJEVbHTYn05cbWGfvOtpFmWYnBGROO5PCou0zK7g9rfu5CA3l31a\\\\\\\/RfVMFJIwETfXRSUZdlPBJhCoUPNcYK7PmTXgnr\\\\\\\/Zl6TN4JmskXh8U\\\\\\\/iZIYV7u+vrd8ainPxc3aOAbrjvwRIQrTWOYm+s73stwikC5Vmr\\\\\\\/4vBIXeWIL5iJym2oyZA6hfl89rUA5m4arg4KmVaJ1hbb5g1bA1LMJm0TXxMTsA8uXywyhIdx0T31WhUUo1g+u8uz8Sfh4fzv5U1krEasnpOnrojVoLjXm8WfP+7xEDOkP3egwBe8Yg6Tx3P4pAjomGK3agN23NlRi73YrAi07TfLMmzdiTUyUTRP1KeXJC94o7JPNO8Juf9B1ysKDIlYMa72imsmpUB6SSlh4VzjH64hpjqC57Haq995uw77dqWT4uUqupHhtTKd1H6oqXNme0O3vzkTOqopMvXQib7KitLWCpTI46MYrb9wRrD92Xw0wHCOTFuCHtCGHvIKkfRK4Zj6DmROuShQLGNreJQSQx3qXly8Kn\\\\\\\/P4NA\\\\\\\/ItWgKw3BerpFzhygTV9j5mHKYIGCO5yTxREqMIhcUhaMDxSe8xDvQ3XFCcL++vpCzA4zzDyQ6OpW2xflUCq0ZH9tl38+4KduMGN5OTf0EgmuKdi4IowPE29ZxjIkfNnGWxfxpjd6PzFLSwTu4Vv3kWQX9uL4Ckri2N3B+mTdt1N4TSsxtb5UpvVcktBA9hBKzeE9DzHamBEGY+x\\\\\\\/SjBNjp2hTG+g8Zhx21qXp2B8nKbvuT\\\\\\\/yaB81V08kDn3iGMx2EVMxJhQDFkNXky0KC7cc+sLhWpNuBvFyGs066VGvnf\\\\\\\/t8Vu5gzMj7OXFg1DrR1Tjto7MKblGO\\\\\\\/P\\\\\\\/+8ZIqgUmFT5pYNntzrguuHtZ26d8qNbLDoAy9B9Q8kYdMtZB5YMl5WySpDuZymsMOkFxbN4Xz4FANIY5g+RFePVuNwc7WCmtpomL2nK\\\\\\\/nVIRTFCr\\\\\\\/wNLHOkc0zPJuJoZghUtyrW9wVeGTLRKtByVWMw3GRR+pwJ8GWxTAqbpT9WMoyg5gVM54qM+8APtgMV776TxLkzjBR30OMYk\\\\\\\/VO3LU0S\\\\\\\/SWLlE6mdWsk+hQzt5VYD8lODHT6u+PJ3pz8OQIEpwz8U4gomW4AY+Y31bmoglL9X9XxjeVsET6b58XSHnqGItlnpwqn2Kn2IsEe6zCzm9RQOIZThUhrbshIwdWuCCECOCfkDeCPj\\\\\\\/6a9le2sgoGmuTRdp2PaWoVl496MQn9o2yOtzN5vo2lbSwR8QLnz8lJC+ixytN96Qg2WhyNq0udMoh3rdQGD9+QkvqBXLWA49MqI2jKX3r7hR\\\\\\\/iOlISh3X3rkZy9gDtOBbI\\\\\\\/iKjNfrKzfkVGyHp9y1FHEBJAN3+7uqZai54XTvTTUOmR5B8C2Dt8EzzjmK3mO2Qxy1v9ZTOzEszSJn4ZThcwrUtleLEusM3RJHCm+8lyRhMmjFiZ\\\\\\\/IQCq8qySpsK7MBnyrHD8aU2t5C0MJfvuD1D\\\\\\\/fvIfyt7z1vIdGOxQzYZUHjQzeXlvSOr14qS3RfcwQUuY+8EFgHjipNsyAxdiaOm\\\\\\\/+aK8XSiSF7OJbNtWtA6UzJEoDO58uon05lCR0oQf0zW+7BpLIgeDi0vGMb90Y+Bz+EIweInH5nKsorUbGp7xpS3WqS\\\\\\\/4LvijTimTejdKuyssaCbNLEQTy7DHBJtiFYNE+N1m1r905f9odvvM1uyrNnGZJUPYhZDbqv\\\\\\\/tvRbhfWCYjA2Q7pg\\\\\\\/rHdyE\\\\\\\/+L0bf2\\\\\\\/81J17DRzDdj0mPc1CqfPOlXOAwl\\\\\\\/1hFO0s46JazJwHM+A+tt8m+Upwv1JQezvuvSK7cN53Ov93TFVKl9dJMOkU1342vvjz78D9ycy5F4n8cnpS9HpfVbeXZTlkK8VNzxy0uI7IERwQ6v4omyJQeBJP37EKew7TINfHTSmU88Yj3SbVFd86flnvRVQ2q+O4n8dnQIll7q4OiKXAQVXL6B5SpHu9Do5r942zdWF6bP+3w2WSjm+IOk9pxqOOTvIRNwrn\\\\\\\/jqtyTcOVkP27vPQ4qJsm\\\\\\\/LQ3hJU435DbJwfKN\\\\\\\/zG844XJsqaDX7TENkMXa761IAtU0rV\\\\\\\/PlnBPqLokofX0oFEWXSvfVvxtt4dRqx120PsBCeTQouWi7YNLJD+1immFateBT5dA1dtYKrsqYTrs3HEyW+Ahh6XHiLOIBt\\\\\\\/\\\\\\\/xCK2miAn7YWe2yxBKlKsuYGp4COepq0MZwYK5KYQQaZG4G05mFZSske9fb0HJyQ61eZmbKEjZ6utpWlQ3aqNIenG5q1vNCuuR2xv8dmPwOvqzFkwwcIYsY3EuVYtcg4KH\\\\\\\/MlTVviTyP9ATuNQgSCay4\\\\\\\/lf9aM0hjCrpz9jpzZ5sBWm7rdZDk8agnJE\\\\\\\/wggh12h3Zjm6poULYyM\\\\\\\/6P1mij5tUuQpPmABk4k7GHckGikzp\\\\\\\/FHQIHFHXvK09jiZE5HsH2BMY0rlHdAbKlBI8Yq0t0VQN1TAXGz6Pr6uf3fb6rSLIagH\\\\\\\/bFNP\\\\\\\/iUacgStwqSH2HlUJUjoAHELcutszH+dEQjWJLjk69wzH2linXX0bIr+cZcQQH33nQQyKMJ0i9U1n8t3mNQoOnYt6+dz2iXHv+2XaIh\\\\\\\/q+squhlmQDcawVfWKn6WmxXGF\\\\\\\/zJinwk344mpmN0EM02L\\\\\\\/2f89Q+GzDyfkz5PECp3h4\\\\\\\/T0i12CUt0MJhk4L+4mnNWdZWe\\\\\\\/vGez4HByR3sB7XYRAkhPnk+izixrmMkSvxu+z1rgosEWb6WHLSpTTtknHs6i35ObHpa7mo66vMRzsWVO\\\\\\\/uRigTWIRp4YQfajYUn\\\\\\\/BdHF6QEImMn2gF8fM6GuZzXmLoX\\\\\\\/cbgWVjNHtXZoveiAQgNSbF7B6r7KRmqRKC38z+\\\\\\\/xvR2r6kESznPkxJSgQlgR5Q\\\\\\\/kF\\\\\\\/mMdBq7IKtgBC\\\\\\\/hS2my\\\\\\\/RNjoi\\\\\\\/pBLHXnzLQDsF+QISiLmXVRfrB6nURgDlc1PFYP8vjMcIP3LUOU+61NWS5X3PfrKO96b6AFopILPWvIKIZclX6sCrL1EQq4jOYoOByrMKs3k81ScmrU1QvYkYUcz2vWYZmZJt5dRMLtWoBHTiVG0dZrVM46WDQm4nC6pIHX5cuZU9paaRUiODYvnut8i2wDkaErYlAtzTmT4IjaU9NJxgUlXAxAT0VPvePF2CTLyY+3jxLd95D5r7\\\\\\\/Lh\\\\\\\/lvIFFNykOetfdXOSf4SCILRlPk32bEE2grTFYSQG7Ld0i9uerX7ked5j4nGpA8u+epmBzkH9uClgk73p3gS+VRHF3xRK9ULye9ZMBFsG\\\\\\\/asIqSc6mGI4xDOoJBGxPPT\\\\\\\/gYhsbHR5g+6Y1+BuZX+2jgtUzpKe7uhDvLU0XGWN2pzOZdqxyFQGsrvikMv5YPWfv3VI8etcpv2sPH\\\\\\\/uigSP2rG1acJ84d5JgufMx\\\\\\\/yxuK2cXZnjBCz+zM\\\\\\\/Y6PKOXWp6RoWD96vr+KpPZe8fgEV\\\\\\\/ioou+DRAuWtPd1l6\\\\\\\/jUgFoPmFsUoCAtU5uYBhZGsSt4NnEwksANR7Eufuv18xuGXLJzvb6c3KQ9snJum5DeN00qLwoJyHuBQ1Fo59jGFU3dhRRXwFIIeLe3QMIwFa5mCY\\\\\\\/X+9xLABDvCKBSPBMgD0tfGRyvquhn+VFIjTn4H4\\\\\\\/SC4kE4D+JBH89RoirYUbB\\\\\\\/xRtYJ8A8p+Mr9Nid7RQWZsG3p7AqQxemz70VsrkK0JCyw4wzNAoXZD7KgkHXvG+LqMYf4oWh20rJN5uO\\\\\\\/BTfftmjOR0rOtpwLb5KDU+++WH8Og69DgtlVbzm4e99jWKPEzePxyPN0hS6ru30tysUpxGN\\\\\\\/egB+e10J4I8rKH9uv867qL\\\\\\\/JvvXo8pX8J4MwxJ7GmhlydjhiLptRcQUy5zN43mCcfpGPE+6\\\\\\\/d\\\\\\\/sj+H15mEp6Cub9h3tzgSW+HfUtFNzc68agJnpX9ELaW0Ph2iDYBCxerkNmbZVSs0+8Dv5gv4xNDcTNWHPi0jctZR7d2VdSaYDig+96+qkR+pBVD23Youqi3BRd1YzeGY5fqVBUegv9l2uuYYU3RLnbCxOFPpVvpRMpgaUYvVW6JwO1bFuuY\\\\\\\/MQUYyzsrawD3c+bEK2YUhYFQOdF3B0zPUx\\\\\\\/cW29+kEt91vf84DzGrKdj8vg4o2ytMv8c7BtlZwXRpo1ZPOggRHdRUlci0RnTp0x3FvlXx32PDZcbQ4JoaqboRFu8Eti72761F6kUmT\\\\\\\/2+9Cg70IfVVONFXjFSMLBtzG2oV+dz6hyGMbh+UcwY8Wl\\\\\\\/Hky2Ig+C4Ld9XCsgusReMoABsnttDahJYDL5M009TOP4X33a7uduv1dUV9S6kJPZ5Fm28fADxpvPOk6s1cgY9fano4Py95SQEUBO\\\\\\\/Fpf9lhS7KfnYa0jz0SIP+H0QXHsyER+48h5Q49k4pFdlGiLa+R+Pb60LPhHxS8iU83s1ecS5b8vrGhtZtJ+RO2sBIlmm12GI1qMYBhB9HUuJ3j3GmE+UGspld1PlxDRg8vzyBDUqUps7vhESD6KE1TIoZHVZSl7Dhm3\\\\\\\/0rdV8ENYkugWh0fqpBcPLuUC1uovMPek3INNHm1ghBzlQoi8l2ILt+0HgG5REK3iqONOzyJG7fOpTHz0ZVYJtrqQ9vuU4Lv2dwh6NO379iLMUj3WwoRooO4sC2N5SDm9NOESf4nAyap3U9GCctRjFedJl6510++l6GJVh43mFrSZ6M+ob46n6uLUCls5GeC6IUXHredxoN\\\\\\\/jojfRbd8G8ZeUSpw\\\\\\\/ZdTy5N56bVmhT2YRWVlbM1GysAUUATcPDAGUBZDOeYdZyvHpieZ\\\\\\\/CsTc+KKtFxf9IPf6qeVIQPkKOKJdWG32z0c8JjnTplbdl+xbkDn34PnzeorAKIiIyfxQa0vCLCY6vvzSvlEzP9CYr8+NJuqTVYp2JE9Am8pDVBxCPsahoyat7dCVNcDsiU1sZXG8btCkV5rHtcaM2XVqCXSdImM6TzuLZSpUNY6xXUbZqAtnI57VEBz2uEZUZCgrloeozj2bjfAdYAWmKK28IsMaV+Axv4ytCbP3lBqhKd2k58etF5GOyRfuDz2P18+BLLmAl1CiRAvlGdtH8YtbNA6VDvtJFAPcy9IwEElQzMUOwP3VmF5PvIMfaLcGvt75\\\\\\\/kGsSolAvdnpiQCDq6EQCzqmZUfEVMg1evhoYfz1hbE8VPqFT09pE1WJFa5tpklXYX0fdAKdVNaSxe3B3FmS0OdB6MzAt+4lXcTTBzYk5WSRtbs+0Ib3hj\\\\\\\/a80SbWxXCesv0jvSv0Z5oMvSokxvNgywZQURr3rPVBwVhgd+UyaHOniuMiQI30KkZNlMPA0Uo2q2zQPBgJPA1SSGms1GVoqgUwbtJkA3M2S5cvQeyBc7TYMuoxWApMZ0aq928\\\\\\\/ly8L5GzfYuDZ5sNy1d+IE48b7W\\\\\\\/JJz8igmThK0usevcY5B9j3Rb6bzyl+xGkD+oSm16Cuza5uLDdBY5ziHxB50VTeqM\\\\\\\/0y89xU+ahSgBBs8Yae40c3+cB+RQw+cDMAA\\\\\\\/JKy6RE53mWahtZeGyqb\\\\\\\/l3BjEeVArPzihtOKs\\\\\\\/fQywOzn7yjESZfjCy6dubcI\\\\\\\/Qoz9ppxv6JzOy41170nbiEcPpNj5pmk5DUeP8E2LuGY3xt5N2SZtRtKb312NWULBQA+MJjHP2CrMJTqb1osjUhXtD2rcygqi2GMti1kY4pklOWc9T6IFXOhrUON+StCAA1R5\\\\\\\/LtUPLtT2ryWUJjnU5APLJodUpiPjnykt\\\\\\\/hNjLUdwiBIsz+qUyEAFyHL6Ftbd9AHThJV+IJdSt\\\\\\\/MP5E0RwDElo65dolxWpGgEJgMjHJqvJ+bXb8kLrwNNM\\\\\\\/knSnFELMH2bOW9gtgxtTGvNq6YW8TJk8vHabqHxgph6dkv4uJW9Fw1KuTwjfs9U01GQvT\\\\\\\/MMzr+vYEhESDqCr9MVu3Tb1nvfv7\\\\\\\/c6lyRtnCXtA51ANUKYCe2c7QYLbuxfrYlSPgAIBfP3z\\\\\\\/l+23R5Kfht8dXMvgIR3L\\\\\\\/y1aCvhdoC\\\\\\\/V4\\\\\\\/hENxV8LYT7OnAo4REqksxmrA5JLuNQgA3m+F6ilb+vgjVVKui\\\\\\\/ebUWyYzByB2c4zNBS8KX2K92iOhVwB9op3G5sqxLhoUoJvDGkbHIIRBBwppBQGS3yOSKsGN7ocJzxpoMiVjvsQiQ+4hBgoPRGnpwq57lAH6tNW34Yh7pcMRYhBN0PjDkS+Xa3r\\\\\\\/Fg1ejr+MkMeOVCbJc+7hxJepCWLdv2HQr8XnhhgqU5kBRD7r8YUGJy3XPfc6mVOWYQ7ho7Bn4ODLwkOXNpwP+yhUakeWqOelNPqWhTIog9715enpbDZ3W8\\\\\\\/ypPu+BJ+PnvOE989fJ5bi5Ql2boWDo5zK4jFPj0ikwSm933RQu5437LPNETFziIGhJ+RJ8v1Vwk9W3CPyhep1ehiBuhu5j\\\\\\\/hMvf3Lu0UpGpWRYJ5WSocUUwV0bWV8zgkvajUtmX8WOrvPo0r9i+aBrg1fbT\\\\\\\/PYPgy5gGnrodX04HMVhMM8bXmJzunvHvtF7XM0OIb+6iGvveNUCI+6PbmXwenBASJiCh1R+iCQwpQ+Xw+7oBZ8L6Mpt0FKhI\\\\\\\/IEnCF771hWyPHaVt6t6icsya7oQQkCdeo2ZuK0gzr6KIbGqF3a2m9Vcwq8Nalk0+dNlmqWUeUVVKxvXBf9ucwEsE\\\\\\\/v6Cnw7ZTnFQ8JA7teNmfpDw+KWOaJfH6ZxOb7BGQ8ZPd2875VSXffjt1dT01F0nqqWwzV5SiFFPwyN\\\\\\\/zeGWQxQNzh+gfTL8hSGCN5nJ0Xn1KSseGZCLyBpCeBT4xyl27no351u91WREViJarzJqTtKUo7W0Ujy6pZlRs3Fhoq8iIIUf6o+b1ilvGohWzacnBAihbwLytFh3qw7gvy0EhktLOpW4FNG6okmBuHlrt10U6Z\\\\\\\/sLs02JQpJcddhSHCwyZ3rXkt+Dh4M8d9TUWHaW1\\\\\\\/usjoKu6+Tk5V5fHVVZMf0BzkAvyJ9MBXJVysT+puATGJLWNfkHNfPzDFSsLeUDG6rYUV7e6+4kEjvX4J3HCPXfOutewqsym4kgnu2D4Dl9CfnLtg\\\\\\\/RS7LtEV5izblma18E+48NeTa1ktvL4xb0hYNQJZBDV107I1vOaRXYlfgw9df10RTBE6uBoRK+JW0KnHmfCGSF5O2a2doeRQAViG24AtPkY\\\\\\\/8Y21kHYO1tmpP8YDnVg1OcEaPG3foW6JsaiioXNXQkrVpdfgHJMwbsEnDyr6cmOMfiouvcYeGcwr56qSVPLFMbFr0GEHPJQUxpf1T3kytBHlpqvO\\\\\\\/CW3npahymita0A\\\\\\\/gdwFzkXrFrmnGo7jtuW3ncsA\\\\\\\/p7HjfHLfnqmSnUwVqIDvTEnwhsqgdQEhVKQyLFHkNsTNYh5giRNkMFe6nAMkxfFv7QFWMSsrxLXCJQbpQLlJobYSaPUuWABR6VExyd0+MKgMuA+57I5WkYJqNdVcLpxywB9zwWOJtIbaP4sbZ+LMAvcQnzeLvuzuRmF1jR7cBE6bi6R9ZN9rrfD2m7BrIoltz\\\\\\\/jS+7dTHgp0z+O6eFUcYN48LQghx+RM8UY+82zNJj3aXtkvQZilbvf\\\\\\\/qkIc9noMg3FWsF52Kowdqa5iX24aqVr1u6IHPgeNgt3LYHR5ZJlY5Ux5VVquU6ex30KgjQeWRTR+l4SoP2XV87gwT0AbyJgFhqOfD3PwFDFzec5vEP6l0\\\\\\\/9KKbvMjFML+97N7JkrMxs9VVrYrpIlrz6hf0vsmqtD\\\\\\\/n35mKHKnMrCfJY8MWjM9hEDo0ZKsPAasjSb4GbeSYdJmMReOZle8ILzwpjhd15xgr5di\\\\\\\/kodhxaZxr2j\\\\\\\/T5hEpVm9n3IHOgmKeiZzT3qAsO04KDqv7MaZLhXRUuUAbLOPAwK1SuPYcQpSTkZ5sxHpQJ\\\\\\\/gdpBsitto4TsWjnhT8x8o4lfWgxb4pYPswsaeAxMa78kegSehvZCWJE230ZpZ4AdGBhafNGP7cd2kNCMFFG6rJA9G\\\\\\\/ioUBtMyT3\\\\\\\/h0PEqqCJI4tesKa5olBphufSPCXn3krXmaa2ffK+KnrcejmUBunSPn5owzMkxxW\\\\\\\/OsnghRKi3Qzdhgiv62Xn2HMwEgwIRc6mj0793NOiIO8NSloPGRPpH1734vyiFSfCy4uCdp69XNwRixj1IWCcWvXpTI9DUE9MUu4yaJdpWP7RZPtYz+h28BfaconBY0sL3K9Px79Fc4pKjg7IvH5WUn0UU35\\\\\\\/NLDuq3dWUHd74N3kXIUgvKSgUrLjzXJlv9TNB5iv0mTaOqD4AONySEgk8BvKntd\\\\\\\/Ki2TdyaF8rMPQtGH8\\\\\\\/WwTaApjox3zaE9K53OQBf4XSs1OSAaCSKZpyIhRIlRVH2j8uhIXTMo\\\\\\\/GCqiVHfjklUVGfIaaK3vNZbIjGPRPncd9sY1\\\\\\\/Fqg0zqT\\\\\\\/3\\\\\\\/Tq80NhZ8KnX5tyTkQ9XJd+wnJczVPoaa5Ezw3yF7gfeSn9UyYbtry48DmLgRXxg6rr8bk+mZhAlnLAgA+lGoIKLmiIo87YB7yF7H7oCd\\\\\\\/EZlCte7PFKvm1zuJ7B4pHZZbeJBR0mIKIb2BMeGEx93ryPUuzDW21tAde4Fz1N6afuudYKC3pqvNVJwkVRg4JgBUzGkas9W2mrvtA8D3kSCn6C26ibcWBfH8pZf8TGrH5HLNc4ZMvOj7Ffz4veYR9uumB3tcTYR4U8ENcNUOKdJuac3E\\\\\\\/yY8nCGTd4H7fiNwBrkCfi8HEf\\\\\\\/h5ls+BDH3PA9xpd2U3bcJRgou1eAPj5DmAs4P555VCyRA8PX9nep0Yqf+4EoPxzhB8ysnkPjr9e0G6y+WS3CCpZlmDOfojumXH9fZzQ8uqI7Ohev\\\\\\\/eFMiWEpuvFc5+etF8bz+MWDu\\\\\\\/8DgAT4kWhSrcQTJWsE0JR2y7Dmzh1vjJjG8LQR8GmJPNrzXssBksxFehCBmfdWGoe+yLwXhQKrWtnhK7ySx26HwFtn1O00iz4aBApd6PF2F84R2HsptwxjigDxkwpVKogaHuKi1Ag042Ul7d4b1bECQrMnHrTP6U5pdBRvcqorGKkcIxnT0qaqV7M+z9xIx5e\\\\\\\/G+m2VS7+uznRkph5K1nIuB1AjprJCxYhmgzfXGQRVbh\\\\\\\/Gj+fOX2jfn0fBeJcdDgKli1ICtvcAW01yGMY55I0dKvkfDORkHCKxoDfaBKL2raAjRKStzYTuWgEEFzWhf0MJVeauW0\\\\\\\/XdiiNEApto3xKyNljpLcwCbrzu65wIRu\\\\\\\/eP8fMZKOzwbLGf249bZo9hDFdnrJgZSKDbhFAvPv0YFiZtve8Rg3fnEmB37oyz\\\\\\\/GwhMMSu0\\\\\\\/Hvidjv1CM6JJ3ONApyDhE3NnLlq4tvGduBwqteriV2mKIy13m0GkTIFwgKiHd5FGOahy4uQ2L0yGGUJNjM246uYCWjv5B21R8EuLWp5eZmkuWIcCsIp+kURPtYauLPpM75mGFois01QCUiA4Bsf9EESzd79lRl8q9wq6xCFGXzlcuckQ97DoqQzYSkHl1wvCnk76NM+P4Bg1gJRrGJGUdW93NSqMjhAl0TyKB0MX0\\\\\\\/5LvOMXAr42zyDk2uVcqmpHUN5sddMHdqIGLWnUQHODzjb\\\\\\\/Q1idMhTah7YTVO\\\\\\\/XeJA0jI9OR+DznRCBbpiuVajWOfCu\\\\\\\/6KpD0DLy4Oj4s7p5PhvOs4ze0Oe4x1er21AtloGKzls8NE0ek\\\\\\\/3ZkJuREKXfDMPQTW6Sh4dWqX4GaKQwK+n5aZ2N8HXFaWEAvHwaJeybZpyrNFLQpR2ivmghK+eeNCWTz5tqPk+VTm3u9nqWhDE7+RrhSUfOvM8+n7ajULMwFtYu9L1rytkfQDkDiOnNRNmbaMoLvFL5QKYfDW9bc1\\\\\\\/BurP1eqvHbCqg1qNySvtzTFJbIUNVUNHapEWz9AhAh8\\\\\\\/wo75it+9X3yueU4LSZHNa5rstMGNEQUicnqIYgYwxNNNAHg2RjyT7jR2Po15TS514xSuJ6aEkU3hLQJmqqMILxfm2qbRLdoa35f3zgc12N2hVXjSZb8vJqr+0FwWsDif5lwfFukG8xow6EIcdSrDxl1cNutLUTSm3Ixf1AZmbQxyblENZGrKFca3w0Bjt6Va3JtK8XwCwi6oxgIE5GhblchnqIDbWzAWYKuyybfam4LXBrau3Zn8ZRbjXw5Lm2Nus\\\\\\\/JjqHsSO0YS5G7d2WHfp\\\\\\\/ZhyCOUEZSGvSdZTjbBFvqvKPTg6fWd6iaAxdjLJ89ZUhnOVdTo\\\\\\\/tLodLVFJlAqoUk516RdxaA4e90PN5W2YTwY5YSg9stdD4LST+ASYxABgiih2eJIsYOz3dVye74jPZzDizorPZHSfXyr5DQ2KmUrkooxjxUUEwaLpI1DQ9NfLpge9kEec6UwksluLuUY1rBCrnbzpytGcFGpu3N5zwgDkPoCQgwZ7V3UYhp8ya3xyffhbNuD4ot3vJBHKRU4kfB8vhIM1ZnMjNK8k6CjlabZZ0mS8\\\\\\\/QJdnRAR64TzRpRoMpJR6ExvB4\\\\\\\/H0D162Sf4iKOKPSQyMEyvWoY0qwKEkqy7LGlsAyiZLoGzV5kyHNPkAWskJrvkXuH+cTGoAsSM9VEcV51DTxc70lbY28EmlOH8msQpT5v7\\\\\\\/\\\\\\\/aoPczwVkfd0Gg4D0ej0Z2Bm87rzbRDiSzXyL6guc00fhk0JOvcJT+DM+sYBqb2Ucrw1ZhBwwAQGAy4tyoOE00cG6QtOakdLRUtBReeB\\\\\\\/nq5jkJ5YztC01A6A8\\\\\\\/lYJaSOL3u5Nsp5\\\\\\\/9DKcK16DSxtmfcwmfCKEJq8ebUgzFsfkUEnEkXqAvLkqASd0+5TRFP7ofsY8KKiXEnxh\\\\\\\/bIuCC3YlWHdpE3Sa+FzybqML8CicdOAJy9nXVjr3mDj0GqaaGjxLeph+ZGu0fWiAqn9KufNuOvP4R1l5EEFShT\\\\\\\/wnbg8CutRnlrZXJ+8WD3YB2qLe\\\\\\\/WDI97mib1Fxp7RCU45\\\\\\\/JblmljLORUMgYja6JSvHCRQgwteB0vHhhh0aAvZwOTbTvpNLiRGGg0jzrQehCTtsoVUtx07siCdWAzENHZpzEMcylUMq\\\\\\\/xvYwn5cLKjKnrOcilvABB4pBmY0V0tvU\\\\\\\/LCcJvDtxgfE04jGfRaCfvXpDGbhb5l35QEFgBAOxOP++sxFHc3YWiuQVlhMgnzkD4W4EV1zvLqbUP7g2amWACzHrTXtehZ\\\\\\\/pRj34kPUIPBuXUCkM0J17hJ+nzgQSAfDJUTKr5ucLlcs5ca9TcICMZyfxE9ch5wUh7YftpGk1+UV522nMdJYrefRVBRUH\\\\\\\/B2g8CWPpGo+OWHOgCQbOOA4fjdS65paVTafA82cx3ovzeC8zVTz4NImlp3sFsxpe7VQirI5gACUk9yyPl+SKwcipmYqbQbxFD\\\\\\\/weogJzEgsIgLzkGOhzNzQWrCrxdp1+YR4UVvSF2hAoRlInhbVxyBFTZsrejwIkf2k3PwQ8CQ2LNrsLziAELw+3MJEKPXy5RYpGrspjr7xuDE68vXao4bBOVtgcTk2oBbHGUfWXlwF5UV+XxvaaWq8wjW5YfDR9noxue+JGm8XCr7LEn8m+ngUzeDkkXPHsdhwRkLT5xFINfBzF0rXhVYyUN7\\\\\\\/iXZESkA4OzKnQnilNS3qLndkboEiABOQ\\\\\\\/yyVLYDuJx\\\\\\\/sY78q0A53U9a8VQ4aKhVlk9FU5PUGYBGSMa5b9Fj7MAMsoLY5+aYU\\\\\\\/6BU0XwYPINqf+YCMM8y+5yEauMdehwcmcI+FAb4LUcccp8k6p5MxDWeaTiDrVRyPblw+JSui3Jo2uQYGlYQ1qnhXugf927MFvs3laghbhKdmEyxm752+M5Ip9pXoNxY3gJmznZ3DzuxyDKSWk\\\\\\\/dxtB0VaNgejZKsRC1d\\\\\\\/szL0cW+7k68Lo5SltPBQ\\\\\\\/O9wfXO3qyVBFQhb2w6NZkB5d2DIDMBz3nMzJm1ycLMS\\\\\\\/PaNFezPmqhQYYk9ejPfu3CBgQl9sd34c0ls8irGNpyiKRbJjzI9qZKPKpz\\\\\\\/0MOilFyn4n22zG\\\\\\\/l4xmP0ZiOhGhiROcr412\\\\\\\/CUrSwfUnH4lzvslyYWxvo+JnWMA5IVN7EfFpxegEdADZxdFYdCwAY7Dx9LvucIygkq3aRWjjZhXZpUWc8c0RQuOzgdlFtmh7x2QbwYTOeRFD+tXIQ9A0HfJYtpJuJg4r+qwmSUeqlIme9ovfTR2MBLSOj5HzBTzC8sSGmxgdsNBjITNBZQYl8J3PQbv+rSKf5k3YPxqg91iPSTr2ssgEHCwec+HcWhUSanSNinidLvIw5GCJH4zmHfMHLpqZ4GTM49KXYg2ONUJNFMNCXYkXevn+jDpmKNkOqUVwPN372HkrEpCWwhOcyBwn\\\\\\\/2+7fbCQeoxvTFbe5\\\\\\\/ytGKwrf8wvMsA0kMJY47XvwFVmUkNIgC8dxBvNiY+YB80nxC6z\\\\\\\/slP6IoNtP88ZDY3\\\\\\\/nymzUHeChgC48p9FHw6btmSLAOlA3uq2g3MLjWSaNbAy2FiEKgXZG6XsoYkYyLMT3Dq3Kao8dnvuaJJCQqD+k6kWCXO6V1LCQWzeVnbTqR5SyGmpvuLgwXTr8qCAED9IeRFfPHPFfzV6xJyEXlih\\\\\\\/MYOKk1TPcCDsyhD0Ev8TFhDW3JYA2i6QN2\\\\\\\/y6bfYmCWMK1OGgUg1oyrzk\\\\\\\/mdWPDN9UK96j7Qj7mc8fC83KkHyzpiapE+GYS+63kWvmCrUGuLvDjaoodJaM+Idi5c0uEjMYkpX5QvI6HUI9UebdHMg2mLvJ5CztQXSPYxPAqcikXcz094Dct1m3voOOBNhYUInt\\\\\\\/mpUkwtTg0Bxi1NKTDnMPBwdeniiyed2+mIyXgsKB7FEPQXf26KM8d8y3RWwRqFMqefGkYm2hmIgsX2tSWC\\\\\\\/Zyp0nfo\\\\\\\/W+uGmRlyTBBckGHPksgTTMDwckyO9HhzC2HKAbLdAISMz\\\\\\\/sabXlKuR9ifsIkT+a2f5hiDNeTy+efEQPhvBD+RoV8LANlDQ9c8jxJEgkZrYxi+RvElhIZ+RU9e0SUb9eawm\\\\\\\/1mKNhQWnlej5B2hkzxZVBIqMP1W18\\\\\\\/7fuStBwBoHkOjNAG1dmNWGrsTb+CoT97WC9VHtdvp83trJr6vxX806s3zJzgz7jOgIpWDiCixylhnEL8vJpC20ki2l+PfIofC+D0MdzgLp\\\\\\\/VZfYJHggMGmf0H9YN6SohtflrfAjkAShcgl6Y4G5LccfxuU8uwN\\\\\\\/FVaeUUkfZNCmY1uNlBt+ToGy+Uq6p3RKb8Lv9xH65QlpvcoUulA3kxziuvyS741p0SEFsgz8XLLGz4uUpEzZBnFWSdhLLijDg6I5k20WSPOuEZSQSiX3crsBAQFeXeN1WYNQ0xTKLWPJjUF8nIIadyu0\\\\\\\/sjPA1wAAu5xIp6xUj8iCRmMUPFiFk5MEi3Mv6wsdpRL1G\\\\\\\/GdBQjfmsVDenOSmdxmRW45yKrGmYBpZbyaa4WtHL6kpmtmohEcuh2BF0r9kZFDeWNfLt0NdWqyMEV3h1T914CIBpchF0qIseQ2ejbC5Cm5jRT8kFMgf2Uy+3lxqr1H9j5xc\\\\\\\/v9opmTv1fLWo9Wc19JQ3\\\\\\\/sjYci7AVo9CvQd7hprjEVQM62bgSuKcLmJc245fSavE05igJ19KVcQ2BBaS8xpL6mmv7i8XAdyl7xTEbR0DGWZ2NIjTPiitu5A\\\\\\\/pGj17fXlVsj8Qg3dIOZ7Kkfqqe6ZGcNHWUg7T0Y\\\\\\\/ttWrIya9wBFYJsR62E6qdvpz9sED82\\\\\\\/DB83\\\\\\\/Ta\\\\\\\/vNVNMnqTV41vQdIG2BwJ5H\\\\\\\/SoLmrTBgI3W5unrk\\\\\\\/bbzMv2kN2+noqO7SsZ\\\\\\\/USGn2rfinCYijpHcyekTH7kQsHNx3vlOxXGfTKXRKpUGeo7zeBJZ5hViAu64k00pscsCtNWMFLTYSTpKgHc1gSikdO7a7gnylnAJjF9YX4G3o8I1sT52wHeI\\\\\\\/ngWo5QOnw8YXc88TXzlA0RcdxWSS4ucq3V32WUXJCyZwdryUg2sEYdAVO\\\\\\\/26+7BsrzUWx1WTo+MlJkUMztYiZeMqANocIPWvDc4dwHtZr0refsZrRbN5TXv+JBO28hsMFaUMH9i2EFK+lOnsowhHVPa46D6f0rLtkEotMh9T2MGpeiBLrHlB9VWBh12M0FqFBFG1xzD\\\\\\\/X6C7iXK3S1oHu1aPhw9aOrtZmlWKjXA+2sKxOB6l8oJKzeYhv0hh9R6yu+BoAZxTlkpbRxsFtcwddejTCf0nwRCt92yi3+8Sx4vaiocJ28Dpyo0W4iE8zdJGj4JTzCVxSS5EDCBaCRVmzLEazWA6C0UcktmFaxGkmjpQ4XLVzrRs0jvuCmDrNuG0xZC9SMWtZyjffYeF9K2hzOQqkzfk1Q4Iu+HhU9o6gDquXBLFdWRcHbQ+dgqlK9DG0HUEzBz1P335P5xJ0N9DqpjV5EfwQxTGI3VhIKJLySWs+GXDNMlR7nr5UMOXCDu0xF8jKW5fv1UM6avbPzMHZxDw1PjUubWt0mVWUX+WeZmEbcC8wYjlylbfd1hcmNYdUhwjpr8hTpXe5hzk+QgFrgC5Z7igtgoaP76CJM8Mxtd\\\\\\\/891TdhRhGSXWZGZy8yAHGTeYw0HuW2mT5P5Ko0rRXBZCowZP+o1GxcUe8+9XWDit1k22emZyVrCS+AUdUEJaHRogjI7Bc7IlH5wfVcabM1ABnT3qtI7QFR0KrQ16JXE3PEysIrmWPAfdrIp\\\\\\\/fMY8l5olFbcBYVxSAUviE+TjwjoEvJmfpwWETggQLjdaI0ADLg8Vf5wjuWBUsmbjTUrGnCAdzwelFgitqgjqYsL4aPCReqqJkFNkclnFUk\\\\\\\/9t0YSRYvJZRaGzKNh+fYaGqVYvRha5uLLN4hEugIjrRlunXwzzC+nXpPcscYQTLMozXvC6Yy6HfQMQQT9u9Y4AvVyVA6y2s0WTvgerGA6wV9NpLJmoaA5M1dYee92JLAIw93l2sTD9OMDJArnQUVNY8ZXH36IG54dsXdxurUYsFlCMOyYMJCL3O9yWbTfzOkc4+15Y+23qtqxINJA5Br9Z2rvy8AQCG1zmy5+XdozXC5fZPq69EHG2DXhFyP6G3Kfoq7VuWZvDW9mPTMXE4eVzdaejhnaHqGbsDZAY11S7oR\\\\\\\/GF+TPVx52pPVJ8X9kWRVl54Ub5aE3TuJ\\\\\\\/rsoBzCNG8i9ZpD67e2CQ0jme1dMeCSRv8f1RveuViV491NgfKrv4chptF\\\\\\\/PIA4ZOxcUA4FxUKVbr0WpddWfe\\\\\\\/EofArmGAV6s+2kz4vS3kHGcgdlv0HZqUAhLZ0f7F9u0Hij5f\\\\\\\/kR5dr4QKIvq19t2Uu9ipzu8QXCKelDmILCrP1fwVBnPNsO6+64qsUw63rcPM5z53Vv3MFOAqVHZKhg7bHc9AadBpw8h19WXoOTwtcuD8+9kZAShMxm0vb86+aW4PceVx4R63tPQ5GN8+plHhj0NogwUQyw9zsbljuH5kzwk0f8BdV28aVMo3BkQILBQzVb2MV+NCyGwxCBERCW0UM7EQHU0rkZThqhYQO+8VoNQvv\\\\\\\/6YiAhxa3649lz+RfIAK1rWILcTSGcmjtaGscIIbHUKvokshkv2UtpEzaX1zOvCTWlQCHAvbQ\\\\\\\/x4C+\\\\\\\/5gTNh54ckCd2FTPmU8h2TVZaAtuBpkeNoRqR89rIhByYcKcz90LNuuTS7hvOeytE4bnmhuKLlSxxUjUZK6Y2j8\\\\\\\/8ttDfIeRn4yBZ71eSZPDMsb7TJhX6h8Ucbc+PK6aNq9LELK2Or1xHZ+67moaQFv2DMn7394EK1kpJHAM4JT8fCa89ZEtYv+bxN99S5R1QmdYLl4za7NNlDtJxI3LJXtmXhkff+OzVu5aaTIZfS3wHZq\\\\\\\/O5bLTwZKhD9l6IRoSXuTK3uY7XS9EHpqEd2rvQT0r\\\\\\\/ieIVeMyxQrJf02uFlPoFAoneSj3xYA\\\\\\\/yk1evEwuZEITsXFsuE+1AkKPW4nCOhJRkuZLX8HORr3KM\\\\\\\/9bpx4A2GUUBtdlEg5bwWpaoJ+2cPrxwlCAFDwGLbZqOHZHsETFVI8Jg8422TrLwHk2Z76+j25Ynhf1Qo1d+Y\\\\\\\/66qAn0VIVREPL39vf+CvU2PkTTKHsaKBgqMD3zwU1yB9EY7By0NcTWBD1ZCasYpRsR5g6aFQXGX8OPiWDyDUKk9ExLRkB85FLTzHdOIXBkTVF7d1k6jaF50GDEdRcsozB\\\\\\\/Udz3xFHKu1psXA2CRZy21MP+EVheTWrjxU4FqdKbGg\\\\\\\/Y+iTysgLu5MXo+ILdpzZhmD0wL+QP\\\\\\\/TFK1tntk0OtUcnImzFoAZWWGUTGZG6aEvUsnYVT6LT5d3IOj3EKkzhuu6lBkzW1uer9EhL8wNYPLnfZHRDE0FcXhbpDoqEtCfiAanqFmuZfbI9t497bCG8DBLtqkZTrYsUAc9g01Iiejcw1dvnyh3iTDNDFBKkPtpILREnsYudKzmwPgzS8Y7zgCZKQtKHFB6F2X03UiZTJB9aNc6w7JKXpFyUzvkuTGbUoIbPuAtMFBvuwVG8JJMnjHfp9gwkmWzzG\\\\\\\/xfMLG90J9BlLDSGSuE1kiNH\\\\\\\/439g2Wg8gBOvj5GOBtCl6saBSmqDBqZUxCh3SubCGYyWo\\\\\\\/m1Dz7zmJx\\\\\\\/GckT47m6D9rwMcBQql4UG\\\\\\\/NQfx62J8hpVjXkxTZ0i2nqfQd0y2gZvF8qzl7ub07o+hLXnsIh9mVWY\\\\\\\/GwOYLxM6t6fD9uTdZd842FeKR+JT+1kJGq5oY8EjPrferpkwAJrLczulOwGWUE+P5ypeY5ddPGk6shM\\\\\\\/6hjhC3rvkMPYJUV40o4Oom0jke5zA5cjt5MzyzQHSZOtNd4bX+rXMGetLjr\\\\\\\/PX0V6JPGpIvQeyN3vVNH9qDYq26A7j0pzyyxBqbY3oq\\\\\\\/VeOW8iEmwTwNA5\\\\\\\/HUnMakDSJDS10gLRHMvIy1OvZGIBWoxn9nmvw3T2mEuhTutpQ9WUAhkIhAr3jEfccHqmPMvI5G2FicVgpTma87oKjqCvFTouMsbFov3VuBRK9jJRK0RFRCo9EeTLE+piaiq3r9\\\\\\\/i4WGYqGPdoO7OwjV6meh1ZMFpdpZZjZQx2hiDazOm+v+L8K1jfI\\\\\\\/9O0h7+bJQOYHvlznjwZPZ1S64IwpJxK2J1lLVv5aH5d8MMjplIVjwJIv8RXnV+j75bSSkl\\\\\\\/qU5CcypxMO7pO15Be5oqUUK8o1\\\\\\\/ImgOpfWSfKgUqs6phKTVaBHnXNqvxqWHY3GjSzfxIsnLuuZE6jSS0HY3IZe60dJaN9MKHMED\\\\\\\/tg+Rgoa6M4UONjR\\\\\\\/GIkaQLivM8bFuIj88hxC7DgUJTJnh1yCyvSG\\\\\\\/injZWJsOj0wXXMVBv185P99APBHByTKwK3HWrkOY+6Q7Su5n5n6xEoJwXttYLAk1hqU+Hu0VQp\\\\\\\/0q0MxITl2o5vjKt4j7wJv3LL6UzNgIuLzKaKIfaE7X4715uSeMGWMGdlD0PT796A2WapYzrO1647Ln3V\\\\\\\/pMMpiXwvokHllCKgrfmLfRjIWigLWZck+1kuIOvhIjyS3XanBrsVvzLZB72xapkRKwpcQmbJslAKhecHi\\\\\\\/0s+Iaea4oYstKPhUlmDj5zJ9cias5d2N+TOIvHNr7l8rIxc4f+xXLc1HLB3Tr9ALCJyH5MCBCVCf5SFhvabEIS\\\\\\\/yV4MyZb4zUjbpH1AJsi+vZzArB52UTrCt5TWk5jIgDfgq4rUUNUi+BMUwWGRy1yU4FSZchzT1lKNFhD1616hjmMRsh6aDrKpCyaE1ldWJhrM4q5w1nqykzpkcmvi3eiZMgBvEBKxtEX5SZuXwS12x8xma0UrXF7yhK73NUtnK9hpPmGEglJKLIS1irz\\\\\\\/vwVDjKVtTfe8DE03R7+4OhjTkIMdSIzVocnEev0hOy3P0TJRYvlIzkjnWEKyEoqPVHj+H71kxcr\\\\\\\/4JzaDKl4c5aesZ7XrV+oLRpb9mc8oRm6K0f9Nbc8S05NOsN4kq1l7qOl+ouR5DmQyhmjQ4jawPdMx\\\\\\\/Zwyi7YKlOp8buGuJTWNTofODOECBP20WpBXwRRA09PnIo1xnIlyBXXoaW9TDDb0CCzROZDD3l1WZG98xhsWW4QCqez6mxDr1HeAtN9PvVjfbiAvvRWj6gwgl53xSE+Cv+nD7DshVZkJhsXnxhnWx5jQ+6RYPrD2Nanr8mRy+TKC\\\\\\\/Vlu4fWBxwF73RCy8uqaAlS1XPXIWLEk64Nip03Rni6MIITfPKxx7APNFrAh3CMl0pEFd7hrm\\\\\\\/wKKi8yJM9y1J4mYR12oxI+BSQ5Gh7GU9qGN2YiE3tHlyZyrRgf6BYsvv9O4Nj9VnOj4elgjIFMzYPC9nh3Mb4zjzY8lKLHNj7KfIZw6vNGOIeak9gecxY8f9c5dfh3Q+n7f3mWPWoeAKD\\\\\\\/Sc4akYN3kJxH0nDE99vxFXheBDNI6qNtDlNXyyf0FRaU+Rn3x9Xnq5ltE44zcSPiw9UHn3JQBHHPZ6Skkw\\\\\\\/wsn59HuZ1lj55SFVikMteV8AcixWdtdKIK2cRCCT\\\\\\\/gpeg4ixIk0g4E56eCmFrsMu1LeToM6wulmCrz39EH9Plkx4ELt7beOh15v9NNXXEret\\\\\\\/LqauewdDYw6V68\\\\\\\/4AnROOX3uenmOVNEWLDGfsji1uZV6+0ETW9f7nO1AoEbw2ERsaB8yoFR9NsqC0x3sPHZ21Phvq+24PGxXCComALIxchy6S+fMWQNHg07\\\\\\\/Tbi+cj6ZfR+ItEAi2vaaUXnoB8Ia1i+sZt5lX6IPMT+OX7JaPbVZ7Oe6Bb0ikLtRiIPkldwwuWyF+XnjmE4Rd+kEkMenVtiOTCr1MI1WiQyApArBAUEGbZYt4rpI8tfxZUOxOGgLqeeuTCNObLabREpdF11BYMjpc6bKbfCcnA5T1RdMJOL+7TymAEVTjeQwoAN4VroA7XSBosgoikjLXydJ6vcu0fia6AuqmJL2UQYiN\\\\\\\/3EPifuHh7U3vrXAQFOnLrsdn+6+4RqvNpJuJbT\\\\\\\/xVfD7Zy+Ame3yedUpWxQBF0wuUoFBf+VZ2bEeF2bukgpFvUl2Jsby4PyaY9UfCro+7m3422YGHjfRTITei0Nc5wvXJkk8F6KMjvl41hLA5q6JbJrHT9xk5btZaGey4jqaHBjd8zBsjRBDZ+QqlmRWa3vpbiLcR8EH8bkRwMhE3f74R0sX4H5JLgVbDtNSIlKlyqZm+nhgb\\\\\\\/j8vyv903XS1szwpnq3ZEQNQdsI4lIf1RPUWMf9Xms7kJOy6uE+VeaSYJuGT6bvTjRz0S313acRa2otyn6gg+USP24j6nJGvD3936DtFlbCz8maqdviz3CxM4O6lETze2+ng8YL2Y6y2HMDWDWbsXLroFyDMUbogdwEifdFKg14YGMQMVSJHAsLsgndYnLCqhUe83KB4I9h2CNWhoH+ThVG0aYI7dZbODoW\\\\\\\/E6hlddnDcfqBNvjOfn18Gqr4a8MNqE08RsvsLJ\\\\\\\/Ec4\\\\\\\/a1UrZMFhOAqNm9HHrna13dYB82oVA30UeMRB+RVyFIEqyuPujzZVfdMHQCoYHoCpYikGKRhejPXJaDAEhCIcyQDlzv6aGbjy\\\\\\\/3qlhiVlc4tYZ4WfCYB8mq7IRSyRc2cS3vpj5VpeFWexAYw9EWpkaeIg0aUf+RRyKzAJl4I5sSMwytn6VqEMHXJW5ikp20W6BiIMXEue\\\\\\\/ik6jDRBLCzjnxGfV4oRpJqhH+FAOfZpxKDO6rBqOM4V2iNpO05+BJGyaXRi5uMMFZ0edoKt5p+tNMQTD3h0rVeMiUjQNphCFI5MtT1wIDtqQBsmeFmh5EuW1USet+ywuvAzHJtFUxF1P11kzW\\\\\\\/Ok7lZtq\\\\\\\/d786ok85YJOs2hVtq9XgalGy+edm2j9N3OwR0C\\\\\\\/lW9CJuRl2fxq16z+sQZhzYu0SCtseYxlzpXmrPzK8rbS1i7hedV\\\\\\\/QoN4eGU3eDgTFYjkYRXScdoqlMlr+WjXs0y5wXNK\\\\\\\/aOAjimU9yqecMVVyf56IdVGLFrik7xvkU\\\\\\\/XQ4r+cLaR1necew7L3rT+Nip+gCFlxYo1J+MSSIQf6ORZElNSsmYFY9txLbQVr305NrTlaVQt2rsUTrJJV+bQY1lx2vq7Xbrg\\\\\\\/7GXuEWSn3\\\\\\\/VUJ7nqXYaRJHi+ClTvicT9hgsIL38biRXyhsAj87QDIp70sPaDniyn4ztNC0QkgRlp5oXBOts1vNvS+pettELUoAxJml0qwayf69I7bzEfut77A6pxpdZlN5XlTRDbE+a14trwLV6Myalu4rceWH7\\\\\\\/l+zmjLkbepDTnTA7pvgXCMC2kwqEB3qEKl45Bp6Zt9QQ5eARzxh8pPWWD4KtKsdH4lcPZnXheSF660ECtMIhpgxN4LxWPC58mBbPHG7rqBw\\\\\\\/w9MGZTcqUkaGaj\\\\\\\/bNQcJBl6tfQCqCM1p3WxgN3IqzQU484kny10uvWelxPZaA6LqDaURkQb80kCSddGxfPH9fz3f36eV3tHf9v2JwEEal\\\\\\\/BdhbaxqMNUw09ExLeeJ0a9saM3cmLAZ7Xp9T7UD3q9Lp0RKnwTxtjm8ji39txoCCfJefKG2j\\\\\\\/eoL\\\\\\\/+DOlecpgjwhSYiHVsGv\\\\\\\/2SmUvhbVJD\\\\\\\/Z6nHZPGPDqj2m9B\\\\\\\/7821a3xnzcARnyKr3mePiF2SPzcXDa4pnkpxWabpYgHynbGOQTxOs0jF8sA2YleO6RRsLUuV4lIe4KgiZwaI\\\\\\\/ZVfpAVtsrYt9pcDz8us6ES4bpNhc5CmrCrHCMNS\\\\\\\/b5OW3oa8wczO+SlnYMIL342BUjRtTT9nQVLVEt8MDoGTyBkxRnOsR8kKFa9zY+vZcwOmT8BfOwY4f0szrZ+RbM5ktWaimAoohi+WxTAOoLZqoqNmH1cQwsnDaCCfntSeBcWWuzxwQ7ljpSBJMu67b7nHLooOg+dfVoCiO83X0CuqVuhtLsw34p3FbtNkxMMVzJyV+jFei4b\\\\\\\/SmriWIqGGN3Xz5P3voQR2XwCljYwZ33NPKLBqJ\\\\\\\/WLn2fDwj+mBH\\\\\\\/q6JSmQMlr7VAWNb8MnDN9zK8SycHWVynneL3xx121fT4KoRE7FYbwlRaNN\\\\\\\/W76EkbgWZGJ8M1RjqbtVC3qerk+a41G8GxIAamC+amvPaZf9jeOIJmWV79I1O\\\\\\\/ay5pPUGE0wqg0TbFpvwxTR3pUxvEWDoRf\\\\\\\/MkKht3wfHPv0irKgfS\\\\\\\/0Xl8CK5M8KF\\\\\\\/i2BHPhX3Df6gB3JAINqJiOSeAtBFaVmz4qkEoUIx6OEiCNVJ8hC++Zmxbg5Gpo+zmXrFKEPUQ3LJ2r3PiH+YmLD75Ki8imSqpYz17Mc8Na6hZcOkD2tfQQxoe22kRcjmpi7m1jom0TC1blyBZtF5LvGv9BSkWVpKj9essPB3R2C28IvQBiRY\\\\\\\/De82l6j35BAD7Uxqbd52353zjJDCF12ZCZpypzp1SMm3IN+wKxwSsjn6gSOz3WYTBaOnYiU\\\\\\\/nOCU9T4gXmVBTCQ0W5Yy7sDvAj5fw1PNlGaGEOISt1fUns+KHCY84LiipQKmjf5ZIWtfO90dxBMrhcJ3ow9rBGgBNUt3ZkfWfjnVkHWC0yL5AruhalbIGIT4cN8KH40zk79qQFAzBfV8g09WMwYj8RJ9eFhOu8GCMIfD7638iKmMCSHHCo5NeKzFVESuwFiy76FaB9UfZFC1y8gO0Kiet606cae52rnoguzqFv83jgVFEB2qJKw8OD\\\\\\\/ekRPYFnR\\\\\\\/\\\\\\\/EoauvzQoiGKoAOfDF1GcgA1gqFG7XeIbb+lO4NUZcvyC+wlGooI3GInaFV33PoPGv8GhC5YCAmCqg76Y3J+GvVLVhu1sODMbN0Cdd273587HjyW6dFVK3Noun4DffHyzOCptvWXYq70rE20WKHKJ4phypREppskCRVWnaoMK\\\\\\\/MJks2wkUEHf2Z\\\\\\\/AAmhIwcZauFzF7B4FDQ7BVVJ4LY0nNBfXB5eUp35iHjV9OLZXSQTsHy+1tURjVxNP3z9lzQvTtZFOhIgnGcrY8yV4iiDPa8U2AT3czX3S2NxDctw\\\\\\\/6WFxoPVXZelu2J2wmMZrlttcZN7y\\\\\\\/Orb+XLq7FG6KYPCBUJAtOn\\\\\\\/HTyJ4J786\\\\\\\/xEHtJ7Y+luMzLbhV6qS+maT9EilWszm5PHyuPTHQy7IJvUMUmg6TuncnnGH6+X5Ggakl7ILWc3tJGe9CZfOxXZrKxGix\\\\\\\/S7tqmHon5SA7lW8IG\\\\\\\/OrJ+PFVhafHEHnzykcHmD\\\\\\\/g1p13jvStIT2aL4FmmPZgz1VRYa+NOEHsGj8\\\\\\\/zRrhwBhhaMo9hE57boop6vNw9vdO3qINcG59EINMca4lJ5ieyO3CZI7d0D0ve3MoS0okrYa8kcOv+o6DwaeaisU0J\\\\\\\/\\\\\\\/7M9ZUBVitD2emAc6yyQZsHLaN8C3mvZjD8PFa2lKrzz2CPprKdeqjnHZaUKpJM10gj9rBKcrXzN+\\\\\\\/lkXycZ4XQQg+Nt6bStX9QRCCwOpKlWwjCjXXBfnGIGQfAt5sjIGImaBk3SzqRCGM8rgMi4Bc2USrOTs\\\\\\\/67dFrPpxJcPzFhlVw9wv+6FFMiBEE7\\\\\\\/iBD8oHNjRZJDaCj3\\\\\\\/J3BNaP921rW4SScNAijhEVYdJvgiK1WnM4TrRgOnf2P1UZcIwRGKUeG5rNqOBtLAbMzWAOxykLyN3z5ZM7iFBFZwvJleIYx7lZqQ4h9Tjom3X+CUyCpZCO7F0+UfY4slR3zCQqdSqlzShrwTWOGWVjeGjNMTuo8DM5qXMwc4nXyhJ8NJpkgouRolw5a46QYem3lagUEja7X4\\\\\\\/3LXobMKUBmGz9cgL26yxzPklE7vpMrwJQYeDhl0J4aNRtFR4xacoQl64cbu3OtRaBFZPMcSV0rucYYiyRxF+\\\\\\\/30q2E8rH5cHoCQAKNeUTp+ESqvo0XKKTnyOZeuQ40+FvWe47lzdf2G4CfQobmJ2AalqNXI6RL2cWGTylOW4YCqHJhRu+7LVoIyeam9Iv62kEwdba2yC6KW+rpobMDwrxaQ+JIZMD7SjpHthtK85SOYgLo\\\\\\\/ZkZvZVKO5kgHrmpJOjvPumF+UYPErGcvLbg+dWgD5cAyAw+sngZtsPnet2otw9LouWMVVwVS3cHOH562O+YvzM0sT58K8RkWEObrrtZ21LzYVC1Nu0rYUuONyHAjA5fIUmSK8dBULVLKElb2d\\\\\\\/BgTXl9\\\\\\\/6GEqpLx\\\\\\\/x6ZsDHpufbBWJn9qmWujuKTzmRYIfWKiY1qKN3bk6rFTHrv+JSZuX2X8tx46JoLywTQVw90KfsVFuWYbuiHxzV7TKiIQmB03AMlZWwGxDQB6Fz58jAWg4BH8BDR4S9EKs1Uekova3XJeNllZU40308GCTZzDL8x+GWMFSjk7wW2Lx77Fk5Sp0vbrAnE8\\\\\\\/vpo+B6SfJqPggLod0SzujmFvxSAwcz7gG\\\\\\\/BbBFNIRwOKFsxcPcdQ\\\\\\\/YETTtPeV7uulpfAI1v79qFy6Lh5Al+G6GSmu4DGdrccBurL9vgAoOVaiYfN7DdBvHjHbwzaIBi1Pk2I6ue4gtjhccjSnsWTUTZ0PoEPzSV4m0OmhVjGj9VAxctBSpbVt3eFrAf9nUt7z3+5rJzzMnIaEEfmOSNxVpeDJ2DB6k1uuYPlSKzFE15KVljJRbrOzoiHX2xjsZwG7Eew4J0qiPWQ94VIgiyvggj1Niv8Fq5YhMJlTnrJzjm\\\\\\\/3dok6V8IwTqVF9CY7H2cF1rm5BG\\\\\\\/QdKBMajX23yvdf03+W6NQLYb6Nk9CjHvYTAZaMdRBeL8HknUBHNAmUke6b3eyYt1wNAAV95KSdttSMOLw215hDxc7SuX9TyfK5XlJ1udBzXOT4pHi\\\\\\\/EpKnQduKGA1THrH5eYRhFBSTXvesYWfjhsu2lDplGQud+RHlFrjTt04Q1OowAwAMIT3rI1xJzjy2ipi7o6LIDlz2tsNBJ\\\\\\\/Eqa9BmmUGiAKZHm0EAfDME0hIW4EaGDhfsAd4Z\\\\\\\/4Lirb3lQDZefv+g01U3+KLCnIY0BLE0oGGoJSeVNKSjGaaP0wG211rDPNB5rwSSEgQliEUfJW8IG20fg03St+H4nwXheoI5dl0qrqHopZNpv5xyDgxcE0Z2cx6+hQlDPw9R3eAYwvzR7BG5w8TYE8LsalGLDOpbL0BomCcuMDrbM1t5ArJepz0T\\\\\\\/qUruotZB8iYXR4PcmQadDz6L2UwFTopfmPIs1ClYIddutUoMtmulT7vWJPBQ+mtDAw\\\\\\\/dtCToBDgQwVH\\\\\\\/AphArwQ2Rtp0UrtoZwOVSehkUBPsn5QMRSMs6LtKAmPqY3SQouD23s8iVn3fkP+e8eUpAhTfEXQacvUg\\\\\\\/tGHbi\\\\\\\/frMfdEgIv0tqOIOyFMy2sMndsMWi4v83dWF6EnJPksQUyj+do3a15IXv90KnmS8EHI4z7uRa0nrXFwOMmoHY17XXBAcSOPbAf59xbNq8wn\\\\\\\/inKShikGdLEnNFuKLiujjcezxcEiJE9TYE347BV0lJhZnor4cP1OxvLUl\\\\\\\/hzWGXuwH\\\\\\\/ZEdRpKtAvsKiiedHfZhSbgElnxQClOJs5BWCGXyLWpxph2vmihlNQxPXM5YR6xOubITHPIYNZGoC4LLRDZsK1oD11Gecqbt9irl5NAmxE6KQ9ZkXhWxlQtEwOQMQ+8F8HCAIDIpAR7ydVmZ2QeHicTUNX4s7ZzasSU5Rpi8YSCqhgfZQV6og9v3wwpmOXgyy8ZpwOiOQuZTB6vDsMoVg2cEzXM6DwhaRe2\\\\\\\/lUEkzK9tCX\\\\\\\/1HReGbHHQWMLE\\\\\\\/X5fZmbKaKS4H\\\\\\\/iSrdJVlag00UexgTeyeYfQo+vL5rij4hMmheBWuv0XIOd9nFSpkAq7Dq2iNU+BX+OnN6bXUiQYNtzEoDeCiKQEjLNrshjrEmFHCbQ+xu3ywTSz1q3dbJdYDJ\\\\\\\/CS7EUoEyZgieEmDg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022URszoq1wu4+Ax\\\\\\\/tkmCrdyQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229a3125b37ed9ba266751f3530b30317320d3cd9040dfbfed2f8dea7cf1e9b1cb\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022oxO85D3MxFsrFzaVP7qyvbxK5FWYuVr90Fqf7zYX19wmlwBrALFdik4Grd2MPqKX8OpA\\\\\\\/POs50fkycmmtIdnfckRF\\\\\\\/ynwOmnl3euTTipV2JTmJIvAbaQT17rygiPKhXiTA6\\\\\\\/fe6mCBdrtXtq0fhUGJEtUPvTQgnGN0KWeOHEOrshH6pkss0KoK5ZdFbksPfZ77BAV8KMal3gurGsUUOyUIdZufgpkwIMsE58x\\\\\\\/wLaE89+6UPwT63quAk2MvnLFhiqp\\\\\\\/OsLF745HL\\\\\\\/UIsDmcXLkW4axiHT6qQQ2hQjpmLUabbnSOuzegjkiB3WCoD7IUh5dZglzlL4lNQ9SuLZtPgsYnR3JtSu0BOdlh7jIXwVD682ZGuIo7QWqJh6RSY4LVjBYo7\\\\\\\/S1dVxcTbmXlnV\\\\\\\/EPmIunbGaxEQWKCSx9K1pB8Dye5MEnW2qH4Ocx2tbuFzrXGr7fP0lwn+xo26cifb5xhgH5luTnCvRpZqNsn8qCyk3r3QbgQNS1VVUxSgcq0EFz3hJfP28TB4u7\\\\\\\/Sew4jlwZP4JVdoHG73NsGc+jv5vrZE2DmxH1RQxLYHnEEwTcsRfTA2QAl2J6ulWxTGhfRh\\\\\\\/mCbWa030FhFOMuRMo+aW4zVivqHgDjhHExBhuXH4tEPwjKjUK6lp+01tdahG+CA5tRyU1WF9pZFgUTY0rR+9MQ10uE0HY\\\\\\\/ulvzkHB7RmAB8lkJVoks2wITixk3W\\\\\\\/HuGa8+UlI7QHJJl1Fcaony6wS8UXLmJnwgZV7eBS8am1xKsDcvJGBgExD\\\\\\\/kMpteYY9bprba20+YGb3mqQKeSmTCCQduwDYJpL1wri7gCTJSXw0z0+I1IqTjNuEtfwHd6TahpwFybW9KkWEppzdNMRL5UBd+8M6iFXRp3eAHqO3qnet7D+GHKKB9agoh8oqEfP3ccAs+2MpOhTm6otk9Me6iH3ZR\\\\\\\/fGKlhNaNhGQ8oEFsEAWmdk4RHZKwf3Y8Rt5C5xsq2geMy\\\\\\\/37UHGjf2iQpjWcpjPr3oaBzboRpfmqu9mWF1PCRmge8kxpix3RDQUM\\\\\\\/e7hZwEwPemwd9hzxmQPm9BG9d5yMWAfmKDLIRt\\\\\\\/R\\\\\\\/iQZPtfJQB3eRi\\\\\\\/Px7zlMkUErw0qx4qwHdPBTeDN9AuhR\\\\\\\/0ALtwgNmG\\\\\\\/wBM7ZKcpPafx\\\\\\\/vk6Sl56wYZIRDhVdWeTYKJMa118X3RNU0CAK+kSdX8Rv61Lt+XB5UgY19jxwLaBxhIMVunWucxxzA1rxkx5t3AFu3I0zr+VJg08jlF86zZmi1aEg3ssXFdul8SDgCOTSTl0EFWA0vKbnXBtF\\\\\\\/9Dnl29eRU3N8lnpDeW6P0O4C7Yi71cL4GCEk4mTHIZidJArosYNWfCHj3WmlYWwBa4Ht4g\\\\\\\/f3gQVZYmuNSWvrzINVGYSvhjyCmVpUGUvs2zfObQe6ot48KQtXbYrAkR5iclAP7IRDxrgogNfyigHTqJJPWEIXaMxuz0\\\\\\\/oS5w3XlwiQ2cMhcHqxG2SeRWZoXM0A9u4rS3EnlosSLOXOpfPYvF9P9MIdKz3FT3IfUsT8RT4bkbC8zhNqpYX8AfiRHIzvBBVSYI85ILSSJ2ZAOZXN0rfqZZ3CS3YgKOoOyPBp6BaLVg2+bNAIrKX66MkOv\\\\\\\/1s6coVdJpqjxTdw0GmrK\\\\\\\/ZRkr7qNUyj8KmCaiUyYmN4QTkPVEEZ9jGfc2zvihS3MvO09cH9vjTHEuVEU+5+D8SolBBt59aPlek8NCkjlf4nixiRiTPdY6aowwDNDgUZVQBOOD24PzR9+A\\\\\\\/FHGoooYhq\\\\\\\/tO03Ann69bV6luKDUl2oSiNT+z1G0lCDd9d992W10G4\\\\\\\/jOrU5LjdcYSoXjTUmYJ38oqFDnRxzes45r8aJhR7obWFM1zS1kKLOm8ZWbTNu84qil5+rJ8mL55hYqpTCTw664SeKiU\\\\\\\/9AIqPxz2nOgml12JU+1q8oDT9rA\\\\\\\/OBnwOcs2LKaPiYgOv1Bbj\\\\\\\/XAApJyNKj4MTbncHbKtWbz4fCBUmtFbkuglAT9JGqw0UWaEDzRkQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022sQJDzHmmkBvII0cI14t0Rw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b8fc86a33e4395a6390f6ce68f7c2224b63f15e99584695393b0c58dfb40a0f7\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495558 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/665" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CGvChduj_cgcd08dhyudlAvWd3TzIvM5HmJMJTEpUeE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/665" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-UpB3VonUk3IqNGEOL_s5vFB96tC9aqo5xNhn2PZoWa0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/2/645" title="A Comprehensive Genome-Wide and Phenome-Wide Examination of BMI and Obesity in a Northern Nevadan Cohort" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/2/677" title="Contingent Convergence: The Ability To Detect Convergent Genomic Evolution Is Dependent on Population Size and Migration" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495558" id="node495558" data-pisa="ggg;10/2/665" data-pisa-master="ggg;g3.119.400896" data-apath="/ggg/10/2/665.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Quantitative Genomic Dissection of Soybean Yield Components</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5034-9954" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alencar Xavier</span> and <span class="highwire-citation-author" data-delta="1">Katy M. Rainey</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">665-675;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400896 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495558" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alencar Xavier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, Purdue University, West Lafayette IN 47907 and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics, Corteva Agrisciences, Johnston IA 50131</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alencar%2BXavier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xavier%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlencar%2BXavier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5034-9954" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alencar Xavier</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Katy M. Rainey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, Purdue University, West Lafayette IN 47907 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katy%2BM.%2BRainey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rainey%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKaty%2BM.%2BRainey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:xaviera@purdue.edu" class="" data-icon-position="" data-hide-link-title="0">xaviera@purdue.edu</a>
<a href="mailto:krainey@purdue.edu" class="" data-icon-position="" data-hide-link-title="0">krainey@purdue.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/2/665" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495558" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495558/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/2/665.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495558" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495558/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/2/665.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495558" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495558/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/2/665.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495558" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495558/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Soybean is a crop of major economic importance with low rates of genetic gains for grain yield compared to other field crops. A deeper understanding of the genetic architecture of yield components may enable better ways to tackle the breeding challenges. Key yield components include the total number of pods, nodes and the ratio pods per node. We evaluated the SoyNAM population, containing approximately 5600 lines from 40 biparental families that share a common parent, in 6 environments distributed across 3 years. The study indicates that the yield components under evaluation have low heritability, a reasonable amount of epistatic control, and partially oligogenic architecture: 18 quantitative trait loci were identified across the three yield components using multi-approach signal detection. Genetic correlation between yield and yield components was highly variable from family-to-family, ranging from -0.2 to 0.5. The genotype-by-environment correlation of yield components ranged from -0.1 to 0.4 within families. The number of pods can be utilized for indirect selection of yield. The selection of soybean for enhanced yield components can be successfully performed via genomic prediction, but the challenging data collections necessary to recalibrate models over time makes the introgression of QTL a potentially more feasible breeding strategy. The genomic prediction of yield components was relatively accurate across families, but less accurate predictions were obtained from within family predictions and predicting families not observed included in the calibration set.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/soybean" class="hw-term hw-article-keyword hw-article-keyword-soybean" rel="nofollow">soybean</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">genomic prediction</a></li><li class="kwd"><a href="/keyword/gwas" class="hw-term hw-article-keyword hw-article-keyword-gwas" rel="nofollow">GWAS</a></li><li class="kwd"><a href="/keyword/gxe" class="hw-term hw-article-keyword hw-article-keyword-gxe" rel="nofollow">GxE</a></li><li class="kwd"><a href="/keyword/yield" class="hw-term hw-article-keyword hw-article-keyword-yield" rel="nofollow">yield</a></li><li class="kwd"><a href="/keyword/yield-components" class="hw-term hw-article-keyword hw-article-keyword-yield-components" rel="nofollow">yield components</a></li><li class="kwd"><a href="/keyword/heritability" class="hw-term hw-article-keyword hw-article-keyword-heritability" rel="nofollow">heritability</a></li><li class="kwd"><a href="/keyword/soynam" class="hw-term hw-article-keyword hw-article-keyword-soynam" rel="nofollow">SoyNAM</a></li></ul><p id="p-3">Soybean is a field crop of major importance due to its seed composition, containing approximately 40% protein and 20% oil. Its unique composition and scalable production make soy a key crop to world-wide food security (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Qiu <em>et al.</em> 2013</a>). However, soybean germplasm has narrow genetic basis (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Carter <em>et al.</em> 2004</a>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Mikel <em>et al.</em> 2010</a>) that has limited the rate of genetic gains of yield grain to 29 kg/ha/year in North America (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Rincker <em>et al.</em> 2014</a>). Better breeding strategies are needed to explore soybeans’ full genetic potential (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Specht <em>et al.</em> 1999</a>, <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">2014</a>), and a possible approach to increase grain yield is through trait dissection, breaking down yield into yield components. In fact, whereas modern cultivars have around 30 pods per plant (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kahlon <em>et al.</em> 2011</a>), some accessions have as many as 200 pods per plant (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Zhang <em>et al.</em> 2015</a>).</p><p id="p-4"><a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kahlon and Board (2012)</a> contrasted cultivars released over the past few decades and observed that grain yield increases may have been triggered by changes in yield components over time, particularly in pods and nodes. <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Suhre <em>et al.</em> (2014)</a> found that the number of nodes and pods per node have steadily increased in cultivars released from 1920 to 2010. The number of pods and nodes are key yield-driver (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Robinson <em>et al.</em> 2009</a>) that reflects the efficiency of the complex physiological process (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Board and Tan 1995</a>). These yield components can be increased at farming levels with good agronomic practices and high-end genetics (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Board and Kahlon 2011</a>, <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Kahlon <em>et al.</em> 2011</a>). However, the labor-intensive phenotyping of counting soybean pods and nodes can restrict the number of entries and most studies have been conducted with a small number of genotypes (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Egli and Bruening 2006</a>, <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Robinson <em>et al.</em> 2009</a>, <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Kahlon <em>et al.</em> 2011</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Nico <em>et al.</em> 2019</a>).</p><p id="p-5">The first large-scale genetic assessment of complex traits was performed in the SoyNAM population, where 5600 genotypes from 40 bi-parental families sharing a common parent were phenotyped for various agronomic traits (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Xavier <em>et al.</em> 2016</a>, <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Xavier <em>et al.</em> 2017a</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Diers <em>et al.</em> 2018</a>). Whereas soybeans have constrained genetic diversity (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Carter <em>et al.</em> 2004</a>), the SoyNAM is a relatively rich panel of locally adapted genotypes that represents an invaluable resource for the breeding community.</p><p id="p-6">From a preliminary analysis in the SoyNAM population, <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Xavier <em>et al.</em> (2017a)</a> found that grain yield presents strong genetic correlation to yield components, canopy development, and the length of the reproductive period. The latter is a function of days to flowering and days to maturity, both traits controlled by a few major genes (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Watanabe <em>et al.</em> 2009</a>, <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">2011</a>, <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Xia <em>et al.</em> 2012</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Langewisch <em>et al.</em> 2014</a>). The genetic architecture of canopy development has been recently described by <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Xavier <em>et al.</em> (2017b)</a> and <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kaler <em>et al.</em> (2018)</a>. However, the in-depth genetic architecture of yield components had not been characterized with sufficient power and resolutions.</p><p id="p-7">This study aims to conduct a set of quantitative genetic analyses performed with genome-wide markers to unravel the underlying architecture of yield components and assess potential breeding applications. Our evaluation approach includes comparing different strategies for genomic prediction within and across family; perform genomic covariance analysis to uncover the pleiotropy between yield and yield components, as well as the amount of genetic variation attributed to epistasis and genotype-by-environment interactions; and multi-approach association studies to identify regions containing QTL with the potential to be deployed for marker-assisted selection.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Population</h3><p id="p-8">The panel under evaluation is a nested association panel, namely the SoyNAM populations, where the standard parent IA3023 (Dairyland DSR365 x Pioneer P9381) was crossed to 40 founder parents that attempt to capture the diversity of public germplasm, each family comprising approximately 140 individuals. Among the 40 founder parents, 17 lines are U.S. elite public germplasm, 15 have diverse ancestry, and eight are planted introductions. The descriptions of parents are available <a href="https://www.soybase.org/SoyNAM/">https://www.soybase.org/SoyNAM/</a>. The population’s maturity ranged from late maturity group II to early maturity group IV. More details about the population composition are available in <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Diers <em>et al.</em> (2018)</a> and <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Xavier <em>et al.</em> (2018)</a>. After quality control based on segregation patterns, 5363 individuals were used for this study.</p></div><div id="sec-3" class="subsection"><h3>Experimental design</h3><p id="p-9">The experiment was conducted under a modified augmented design, with a 7:1 lines-to-check ratio, in two Purdue University research centers: Throckmorton-Purdue Agricultural Center (TPAC) located in Throckmorton, Indiana, and at the Agronomy Center for Research and Education (ACRE) in West Lafayette, Indiana. The experiments were planted during the third week of May in two-row plots (2.9m × 0.76m), at a density of approximately 36 plants m<sup>-2</sup>. The phenotypes were collected in 10 field blocks, these being distributed as 4 adjacent blocks in 2013, 4 adjacent blocks in 2014 and 2 field blocks in different locations in 2015. In 2013 and 2014 the experiments were conducted at the ACRE farm, where each field block contained all 40 families with 35 recombinant inbred lines (RIL) per family, that is, one-quarter of the total number of RILs. In 2013 and 2014 RILs were not replicated, but the same checks were used across fields. In 2015, the experiments were conducted on 6 of the 40 SoyNAM families in two locations, ACRE and TPAC, with two replicates per location.</p></div><div id="sec-4" class="subsection"><h3>Phenotyping</h3><p id="p-10">The number of pods and nodes was counted in the main stem, between phenological stages R5 and R7, averaging the counts of 3, 6 and 4 representative plants per plot in 2013, 2014 and 2015 respectively. The variable number of subsamples varied according to the resources available each year. The number of pods per node was obtained by the ratio. Grain yield was collected at harvest, converting the grain weight from individual plots to bushels per acre adjusted to 13% grain moisture. The number of days to maturity (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Fehr <em>et al.</em> 1971</a>) was collected by scoring the plots every 3 days from the time where the first mature plot was observed, using back-and-forth scoring to assign the plots that matured between scoring dates.</p></div><div id="sec-5" class="subsection"><h3>Genotyping</h3><p id="p-11">The genetic information was collected from Illumina SoyNAM BeadChip SNP array specially designed for SoyNAM, comprising 5305 SNP markers selected from the sequencing of all 41 parental lines (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Song <em>et al.</em> 2017</a>). Missing loci were imputed using a hidden Markov model and removed markers with minor allele frequency below 0.05 using the R package NAM (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Xavier <em>et al.</em> 2015</a>). A total of 4240 SNPs were used for genomic analysis.</p></div><div id="sec-6" class="subsection"><h3>Genetic merit</h3><p id="p-12">The genetic values were estimated as the best linear unbiased predictors (BLUP), as a random term of a mixed model. The mixed linear model was fitted with variance components based on restricted maximum likelihood (REML), computed using the R package lme4 (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bates <em>et al.</em> 2014</a>). The linear model used to model genetic values:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-1.gif"/></noscript></span></span>Where the response variable y was modeled as a function of an intercept μ, spatial covariate <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-2.gif"/></noscript></span></span> based on a moving-average of neighbor plots as described by <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lado <em>et al.</em> (2013)</a> implemented in the functions NNscr/NNcov of the R package NAM (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Xavier <em>et al.</em> 2015</a>), a random effect <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-3.gif"/></noscript></span></span> to capture the genetic effects of individual lines, namely the genetic effects, assumed to be normally distributed as <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-4.gif"/></noscript></span></span>, a nuisance random effect <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-5.gif"/></noscript></span></span> to capture the local environment effects, as normally distributed as <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-6.gif"/></noscript></span></span>, and a vector eof residuals, normally distributed <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-7.gif"/></noscript></span></span>. The inverse phenotypic variance was computed for each environment and used as observation weights to account for the heteroscedasticity among trials. Although the checks were not explicitly included in the genetic merit model, these were invaluable for the spatial correction of the field plot variation. Broad-sense heritability (H) was estimated from the REML variance components as:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-8.gif"/></noscript></span></span>Where r is the average number of replicates per entry. The reliability of the <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-9.gif"/></noscript></span></span> genotype (<span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-10.gif"/></noscript></span></span>) was used to deregress (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Garrick <em>et al.</em> 2009</a>) its corresponding BLUP (<span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-11.gif"/></noscript></span></span>) in order to obtain the genetic values in natural scale (<span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-12.gif"/></noscript></span></span>). This procedure of unshrink BLUPs precludes the downstream analyses to be performed upon a vector of phenotypes with heterogeneous degree of shrinkage, which may lead to biased results.</p><p id="p-13">The narrow-sense heritability estimates (<span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-13.gif"/></noscript></span></span>) were based on the following SNP-BLUP model:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-14.gif"/></noscript></span></span>Where y correspond to the genetic values, modeled as a function of an intercept μ, matrix with SNP information and marker effects (<span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-15.gif"/></noscript></span></span>), and the vector of residuals (ε). Both marker effects and residuals were assumed to be normally distributed with variances <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-16.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-17.gif"/></noscript></span></span>, respectively. The narrow-sense heritability was computed under two scenarios: 1) deploying all markers and 2) only with the markers found to be associated with yield components. The narrow-sense heritability was estimated as follows:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-18.gif"/></noscript></span></span></p></div><div id="sec-7" class="subsection"><h3>Polygenic epistasis</h3><p id="p-14">We performed a within-family variance component analysis to determine the amount of variability jointly explained by additive and additive-by-additive epistasis. For that, we fit a kernel-based model referred to as the G2A model (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Zeng <em>et al.</em> 2005</a>). Variance components were estimated using REML estimates (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Misztal 2008</a>). The analysis followed the linear model:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-19.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-20.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-21.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-22.gif"/></noscript></span></span>Where y correspond to the genetic values, modeled as a function of an intercept (μ), additive genetic values (ψ), additive epistatic value (ω), and the vector of residuals, (ε). The relationship matrices were built in accordance to <a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Zeng <em>et al.</em> (2005)</a> and <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Xu (2013)</a>. The additive genetic relationship matrix was obtained by the cross-product of the centralized marker matrix (M) with centralized trace, thus <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-23.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-24.gif"/></noscript></span></span>, and the additive epistatic relationship matrix was computed by the additive Hadamard product with centralized trace, thus <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-25.gif"/></noscript></span></span> with a normalizing factor <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-26.gif"/></noscript></span></span>.</p></div><div id="sec-8" class="subsection"><h3>Multivariate analysis of pleiotropy and stability</h3><p id="p-15">Multivariate analysis, namely genetic and additive genetic correlations, allows exploring the interaction between traits across years (pleiotropy) or within trait between years (stability or genotype-by-environment correlation). The genetic correlations within-family were obtained as the Pearson’s correlation between the BLUPs of yield and yield components for pleiotropy analysis, as well as the correlations of yield components from year to year for stability analysis. We estimated the additive genetic correlation between yield and yield components for pleiotropy analysis, and yield components across years for stability analysis, for each of the 40 families using a multivariate GBLUP model. The GBLUP model was fit with REML variance components. For the multivariate polygenic analysis, we fitted the following multi-trait model:<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-27.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-28.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-29.gif"/></noscript></span></span>Where, under multivariate settings, <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-30.gif"/></noscript></span></span> correspond to the genetic merits, modeled as a function of their corresponding intercepts, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-31.gif"/></noscript></span></span>, the additive genetic values, <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-32.gif"/></noscript></span></span>, and the residuals, <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-33.gif"/></noscript></span></span>. With respect to the model variances, Kis the relationship matrix defined in the previous model, the additive covariances <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-34.gif"/></noscript></span></span> is a dense <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-35.gif"/></noscript></span></span> matrix where the <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-36.gif"/></noscript></span></span> cell corresponds to the additive genetic covariance <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-37.gif"/></noscript></span></span> between <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-38.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-39.gif"/></noscript></span></span> traits, and the residual covariance was assumed to be diagonal <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-40.gif"/></noscript></span></span>. Additive genetic correlations were estimated from the covariance components as <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-41.gif"/></noscript></span></span>. From the genetic correlations and heritabilities, the efficiency of indirect selection (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Falconer and Mackay 1996</a>) using <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-42.gif"/></noscript></span></span> trait to select the <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-43.gif"/></noscript></span></span> trait was estimated as <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-44.gif"/></noscript></span></span>.</p></div><div id="sec-9" class="subsection"><h3>Association studies</h3><p id="p-16">Since various signal detection strategies may capture different QTL (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Yang <em>et al.</em> 2018</a>), three complementary methodologies of genome-wide association studies were deployed in this study: Single marker analysis, implemented in the R package NAM (<a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Xavier <em>et al.</em> 2015</a>), whole-genome regression BayesCpi (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Habier <em>et al.</em> 2011</a>) implemented in the R package bWGR (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Xavier <em>et al.</em> 2019</a>), and random forest implemented in the R package ranger (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Wright and Ziegler 2015</a>). A brief description of the methods is provided below.</p><div id="sec-10" class="subsection"><h4>Mixed Linear Model (MLM):</h4><p id="p-17">This method of an association study is based on the likelihood ratio between a model containing the marker of interest (full model) and a model without the marker (reduced model). Both models include a polygenic term that accounts for the population structure. The statistical model that describes this association study is tailored to NAM populations (<a id="xref-ref-60-4" class="xref-bibr" href="#ref-60">Xavier <em>et al.</em> 2015</a>) and follows the linear model:<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-45.gif"/></noscript></span></span>Where the genetic values (y) are modeled as a function of an intercept (μ), the matrix containing the interaction between the SNP information and family for the target marker under evaluation (X), the vector of marker effect within family <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-46.gif"/></noscript></span></span>, the vector of independent residuals, <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-47.gif"/></noscript></span></span>, and the polygenic term defined previously, <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-48.gif"/></noscript></span></span>, which parametrizes the genetic covariance among individuals through the full-ranking genomic relationship matrix K. Bonferroni thresholds were utilized to account for multiple testing and mitigate false-positives, yielding a two-sided threshold of -log10(0.025/4240)=5.23. The association model was fit with REML variance components.</p></div><div id="sec-11" class="subsection"><h4>Whole-genome regression (WGR):</h4><p id="p-18">Designed primarily for prediction, WGR methods fit all markers at once. The prior distribution of marker effects follows a mixture of distributions to perform feature selection. The association statistics are based on the posterior probability of each marker to be included in the model, or “model frequency”. The model of choice, BayesCpi, assumes each marker has a probability π of being included in the model, where the parameter π is estimated in each MCMC iteration. Markers reached statistical significance if 1-π was smaller than a two-sided threshold of <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-49.gif"/></noscript></span></span>, which translates into a threshold for the Manhattan plot of -log10(0.025)=1.6. The linear model that describes BayesCpi is the following:<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-50.gif"/></noscript></span></span>Where y correspond to the genetic values, modeled as a function of an intercept (μ), the matrix containing the all SNP information (M) and the vector of all marker effects jointly estimated (a), which followed a mixture of distributions, having probability π of having null effect and probability <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-51.gif"/></noscript></span></span> or being normally distributed as <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-52.gif"/></noscript></span></span>, and the vector of independent residuals, <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-53.gif"/></noscript></span></span>. The marker and residual variances were assumed to follow an inverse scaled chi-squared distribution, <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-54.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-55.gif"/></noscript></span></span>, assuming <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-56.gif"/></noscript></span></span> prior degrees of freedom and shape parameters computed assuming prior heritability of 0.5 (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Pérez and de Los Campos 2014</a>), thus <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-57.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-58.gif"/></noscript></span></span>. The model was fit with 20000 MCMC iterations, discarding the initial 2000 iterations, and no thinning, such that the posterior means were computed by averaging 18000 MCMC iterations.</p></div><div id="sec-12" class="subsection"><h4>Random forest regression (RFR):</h4><p id="p-19">Random forest is a non-parametric regression derived from the bootstrapping aggregation of decision trees built from subsets of data and parameters. The association statistics of RFR is based on feature importance (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Botta <em>et al.</em>, 2014</a>). The forest was grown with 10000 decision trees. The trees were built having as starting point <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-59.gif"/></noscript></span></span> SNPs sampled at random with replacement. The metric of variable importance was the ‘impurity’ index, which is a measure of the out-of-bad explained variance. Because there is no objective way of defining an association threshold for significant SNPs, we estimated the global empirical threshold (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Doerge and Churchill 1996</a>) based on 1000 permutations (<span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-60.gif"/></noscript></span></span>), thus making no assumptions about the distribution of the associations.</p></div></div><div id="sec-13" class="subsection"><h3>Cross-validation studies</h3><p id="p-20">Cross-validations were performed for each yield component. Due to the known population structure of the SoyNAM, three types of cross-validations were performed: (1) within-family, (2) across-family, and (3) leave-family-out. Within- and across-family validations were performed as fivefold cross-validation, randomly selecting 80% of the data as a calibration set, and using the remaining 20% as a prediction target. The sampling and prediction procedure is repeated 25 times. Leave-family-out validation use 39 families to predict the family left out, and the procedure is performed to all 40 families. The prediction statistic is the predictive ability (PA), as the correlation between predicted and observed values.</p><p id="p-21">The cross-validation was performed using the functions <em>emCV</em> of the R package bWGR (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Xavier <em>et al.</em> 2019</a>). In accordance with the genomic prediction benchmark proposed by <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Daetwyler <em>et al.</em> (2013)</a>, two statistical models evaluated in this study were GBLUP (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">VanRaden 2008</a>) and BayesB (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Meuwissen <em>et al.</em> 2001</a>). The GBLUP model was fitted as a ridge regression with REML variance components, and the BayesB assumes that markers effects follow a mixture of distribution, where the <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-61.gif"/></noscript></span></span> marker had probability <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-62.gif"/></noscript></span></span> of having null effect and probability <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-63.gif"/></noscript></span></span> of being normally distributed as <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-64.gif"/></noscript></span></span>, variances were assumed to follow an inverse scaled chi-squared distribution, <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-65.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-66.gif"/></noscript></span></span>, assuming <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-67.gif"/></noscript></span></span> prior degrees of freedom and shape parameters computed as <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-68.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-69.gif"/></noscript></span></span>.</p></div><div id="sec-14" class="subsection"><h3>Data availability</h3><p id="p-22">All phenotypic and genotypic data are available in the R package SoyNAM available on CRAN. To access the data, install the SoyNAM package (<a href="http://CRAN.R-project.org">CRAN.R-project.org</a>/package=SoyNAM), then load the Indiana dataset with the following command in R: data(soyin, package=′SoyNAM′).</p></div></div><div class="section results" id="sec-15"><h2 class="">Results</h2><p id="p-23">The SoyNAM provided reasonable variation for the three yield components. The phenotypic distributions of the yield components for each of the SoyNAM families is presented in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. The mean and standard deviation across families is provided in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, alongside the broad- and narrow-sense heritability estimated across families.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/665/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phenotypic distribution of the pod number (top), node number (center) and pods per node (bottom). Families had elite (2-23), diverse (2-39) and exotic (40-64) genetic background." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-558496342" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Phenotypic distribution of the pod number (top), node number (center) and pods per node (bottom). Families had elite (2-23), diverse (2-39) and exotic (40-64) genetic background.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/665/F1.medium.gif" width="440" height="355"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/2/665/F1.medium.gif" width="440" height="355"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/665/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/665/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495759" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-24" class="first-child">Phenotypic distribution of the pod number (top), node number (center) and pods per node (bottom). Families had elite (2-23), diverse (2-39) and exotic (40-64) genetic background.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495664/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495664/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Trait distribution (mean and standard deviation) and genetic metrics: broad-sense heritability (H), narrow-sense heritability (<span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/T1/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/T1/embed/mml-math-70.gif"/></noscript></span></span>) estimated using all SNPs and the subset of significant SNPs</span><div class="sb-div caption-clear"></div></div></div><p id="p-25">The broad-sense heritability of the number of pods and nodes was slightly higher than the broad-sense heritability of yield, however, the narrow-sense heritability of yield was almost twice as large and the number of nodes, and almost three times higher than the narrow-sense heritability of the number of pods. The narrow-sense heritability estimated from the 18 markers found associated with yield components recovered almost entirely the narrow-sense heritability of the number of pods, but just a third of the heritability of the number of nodes and grain yield. And, surprisingly, the narrow-sense heritability of the ratio of pods per node was higher when only the significant markers were used.</p><div id="sec-16" class="subsection"><h3>Association analysis</h3><p id="p-26">The genome-wide screening for segments associated to yield components is presented in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. Regions associated with the number of pods were located in chromosomes 3, 5, 14 and 19; significant associations for node number were observed in chromosomes 2, 3, 5, 6, 14, 18 and 19; and regions associated with pods per node were detected in chromosomes 3, 7, 12 and 19. The summary of the associated regions is presented in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, alongside the impact of each significant marker on the yield components, grain yield and days to maturity. With the exception of the association between the marker Gm02_6396340 and the number of nodes, our study did not find any other consensus QTL detected by all three association methods for any of the yield components. All three yield components had significant associations in chromosomes 3 and 19, and the marker Gm19_1587494 was associated with all three traits. From the associated markers, Gm13_14346156 had the highest impact on grain yield, potentially increasing yield as much as 0.6 bushels per acre.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/665/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide association studies of pod number (A,B,C), node number (D,E,F) and pods per node (G,H,I), performed through three methodologies: WGR whole-genome regression (A,D,G), RFR random forest regression (B,E,H), and MLM mixed linear model (C,F,I). RFR significance is defined by permutation threshold; MLM significance is adjusted for multiple testing with Bonferroni threshold; WGR does not require adjustment for multiple testing." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-558496342" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genome-wide association studies of pod number (A,B,C), node number (D,E,F) and pods per node (G,H,I), performed through three methodologies: WGR whole-genome regression (A,D,G), RFR random forest regression (B,E,H), and MLM mixed linear model (C,F,I). RFR significance is defined by permutation threshold; MLM significance is adjusted for multiple testing with Bonferroni threshold; WGR does not require adjustment for multiple testing.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/665/F2.medium.gif" width="440" height="194"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/2/665/F2.medium.gif" width="440" height="194"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/665/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/665/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495762" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-27" class="first-child">Genome-wide association studies of pod number (A,B,C), node number (D,E,F) and pods per node (G,H,I), performed through three methodologies: WGR whole-genome regression (A,D,G), RFR random forest regression (B,E,H), and MLM mixed linear model (C,F,I). RFR significance is defined by permutation threshold; MLM significance is adjusted for multiple testing with Bonferroni threshold; WGR does not require adjustment for multiple testing.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495770/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495770/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary of association studies: SNP at the peak of each QTL; corresponding trait and method from which the QTL was identified, and the least squared effect of the SNP for each yield components, yield and days to maturity. Negative values indicate the desirable allele is inherited from founder parents</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Polygenic architecture</h3><p id="p-28">The proportion of variance explained by additivity and epistasis for individual families is presented in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. The additive fraction of the genetic variance computed using G2A kernels is comparable to the narrow-sense heritability estimated across families (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). All three yield components presented similar average polygenic architecture, having the additive and epistatic components ranging from 0 to approximately 50%, but the estimates where highly variable from family to family. The additive component averaged 7.46%, 9.03%, and 6.18%; the epistatic component averaged 7.92%, 7.02%, and 7.77%, and the total genomic heritability (additive + epistatic components) averaged 15.38%, 16.05%, and 13.95% for the number of pods, nodes, and pods per node, respectively. Many families provided near-zero genetic control for yield components, in agreement with the low within-family predictive ability (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/665/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Barplot of the proportion of variance explained by different genetic components of pod number (left), node number (center) and pods per node (right) by family. Additive (black), epistatic (gray) and residual (white) variances. Families had elite (2-23), diverse (24-39) and exotic (40-64) genetic background." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-558496342" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Barplot of the proportion of variance explained by different genetic components of pod number (left), node number (center) and pods per node (right) by family. Additive (black), epistatic (gray) and residual (white) variances. Families had elite (2-23), diverse (24-39) and exotic (40-64) genetic background.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/665/F3.medium.gif" width="440" height="229"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/2/665/F3.medium.gif" width="440" height="229"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/665/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/665/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495701" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-29" class="first-child">Barplot of the proportion of variance explained by different genetic components of pod number (left), node number (center) and pods per node (right) by family. Additive (black), epistatic (gray) and residual (white) variances. Families had elite (2-23), diverse (24-39) and exotic (40-64) genetic background.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/665/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Boxplot of predictive ability of pod number (left), node number (center) and pods per node (right), where two prediction models (BayesB and GBLUP) tested three cross-validations strategies: across-family (green), leave-family-out (blue) and within-family (red). The three cross-validations schemes provide an insight on across-family selection (across family), prediction and selection of individuals from unobserved family (leave family out), and within family selection that capture only QTL segregating in the family under evaluation (within family)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-558496342" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Boxplot of predictive ability of pod number (left), node number (center) and pods per node (right), where two prediction models (BayesB and GBLUP) tested three cross-validations strategies: across-family (green), leave-family-out (blue) and within-family (red). The three cross-validations schemes provide an insight on across-family selection (across family), prediction and selection of individuals from unobserved family (leave family out), and within family selection that capture only QTL segregating in the family under evaluation (within family).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/665/F4.medium.gif" width="440" height="143"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/10/2/665/F4.medium.gif" width="440" height="143"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/665/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/665/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495777" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-30" class="first-child">Boxplot of predictive ability of pod number (left), node number (center) and pods per node (right), where two prediction models (BayesB and GBLUP) tested three cross-validations strategies: across-family (green), leave-family-out (blue) and within-family (red). The three cross-validations schemes provide an insight on across-family selection (across family), prediction and selection of individuals from unobserved family (leave family out), and within family selection that capture only QTL segregating in the family under evaluation (within family).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Prediction analysis</h3><p id="p-31">The outcome of the prediction analysis is presented in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>. Predictions within-family provided lower correlations than leave-family-out, and across-family predictions yielded the most predictive scenario. All three yield components had similar heritabilities (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>) and, consequently, similar prediction accuracies. For the different cross-validation scenarios, correlations around 0.05, 0.08 and 0.21 were observed for predictions within-family, leave-family-out, and across-families, respectively. BayesB provided a slightly higher predictive ability than GBLUP across cross-validation scenarios, providing an increase in predictability of as much as 0.02. However, the differences in predictive ability were negligible, in agreement with previous results (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Xavier <em>et al.</em> 2016</a>). The slightly advantageous performance of BayesB suggests that some QTL contribute to the prediction of yield components, but a polygenic model captures most of the genomic signal.</p></div><div id="sec-19" class="subsection"><h3>Genetic correlations and indirect selection</h3><p id="p-32">The within-family genetic and additive genetic correlations between yield components and yield, as well as yield components stability, are presented in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>. Whereas the average correlations between yield components and yield are relatively small (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>), there is a large variation from family to family, which indicates that some families could benefit from the selection of yield components. From the three yield components, the number of pods was the only trait with the efficiency of indirect selection that departed from zero (data not presented), so we broke down the efficiency of indirect selection based on pod counts by the genetic background of the SoyNAM founder (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a>). Families with non-elite genetic backgrounds are more likely to benefit, and the indirect selection based on pods was more effective than on yield itself in 10 families (<span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/665/embed/mml-math-73.gif"/></noscript></span></span>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/665/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pleiotropy yield and yield component (A), genotype-by-environment correlation (B) and efficiency of indirect selection (C). Boxplot displaying the dispersion of within-family genetic and additive-genetic correlation between yield components and grain yield (A); the within-family genetic and additive genetic genotype-by-environment correlation (B), where more means more stable across years; the efficiency of indirect selection to yield using pods, breakdown by germplasm background (C)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-558496342" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Pleiotropy yield and yield component (A), genotype-by-environment correlation (B) and efficiency of indirect selection (C). Boxplot displaying the dispersion of within-family genetic and additive-genetic correlation between yield components and grain yield (A); the within-family genetic and additive genetic genotype-by-environment correlation (B), where more means more stable across years; the efficiency of indirect selection to yield using pods, breakdown by germplasm background (C).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/665/F5.medium.gif" width="440" height="119"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/10/2/665/F5.medium.gif" width="440" height="119"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/665/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/665/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495696" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-33" class="first-child">Pleiotropy yield and yield component (A), genotype-by-environment correlation (B) and efficiency of indirect selection (C). Boxplot displaying the dispersion of within-family genetic and additive-genetic correlation between yield components and grain yield (A); the within-family genetic and additive genetic genotype-by-environment correlation (B), where more means more stable across years; the efficiency of indirect selection to yield using pods, breakdown by germplasm background (C).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-20"><h2 class="">Discussion</h2><p id="p-34">The dissection of yield components using multiple quantitative genetic approaches using genomic information provides an insight on how such traits can be utilized for breeding purposes. For that, we performed a wide range on analysis, including checking the heritability in broad- and narrow-sense, whether there were major genes involved, whether these genes are captured by different approaches of association analysis, whether the genetic control is influenced by epistatic factors, the trait stability across years, and genomic predictive ability in different settings, different models, within and across families. The collective interpretation of these analysis contributes to the construct the big picture of the genetic architecture of these traits.</p><div id="sec-21" class="subsection"><h3>Brief overview of the architecture</h3><p id="p-35">The number of pods and nodes, as the ratio pods per node, are key yield components in soybean (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Herbert and Litchfield 1982</a>) reported to be yield drivers (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Kahlon and Board 2012</a>, <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Suhre <em>et al.</em> 2014</a>). Understanding how such traits work may provide insight into better strategies to increase yield and yield stability (<a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Xavier <em>et al.</em> 2017a</a>). In soybeans, we found that these yield components have low heritability, both in the broad and narrow sense, and have partially oligogenic architecture, where the genomic control is jointly explained by a set of QTL and polygenic terms (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figures 2</a> and <a id="xref-fig-3-2" class="xref-fig" href="#F3">3</a>). In addition, within-family analysis indicates that some populations display more epistatic than additive control under the polygenic model (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>), whereas other families presented no genetic control whatsoever.</p></div><div id="sec-22" class="subsection"><h3>QTL</h3><p id="p-36">Successful mapping of markers associated with complex traits relies on the size and variability of the mapping population. Our study was conducted on the SoyNAM, a large population designed to optimize power and resolution. Yet, only a small number of QTL were detected. Previous mapping studies on yield components have relied on non-experimental panels with a highly diverse genetic background. The studies of <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hao <em>et al.</em> (2012)</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hu <em>et al.</em> (2014)</a>, <a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Zhang <em>et al.</em> (2015)</a> and <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Fang <em>et al.</em> (2017)</a> assessed 191, 113, 219 and 809 genotypes, respectively, including landraces and wild accessions. Among the studies on diverse backgrounds, <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Fang <em>et al.</em> (2017)</a> found a QTL for pod and node numbers in close proximity to our QTL peak on chromosome 06, marker Gm06_47199506. The pod number QTL detected by <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Hu <em>et al.</em> (2014)</a> were located in chromosomes 3, 5 and 6, in overlapping regions to signals Gm03_2382974, Gm03_46533591, Gm05_914933, Gm05_3661638, Gm06_47199506, and Gm07_7868756.</p><p id="p-37">The significant markers found from this study do not overlap with the signals found for grain yield (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Diers <em>et al.</em> 2018</a>) and yield stability (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Xavier <em>et al.</em> 2018</a>) in the SoyNAM population. However, markers Gm02_639640, Gm07_7868756 and Gm12_2838455 are in close proximity to seed size QTL reported by <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Diers <em>et al.</em> (2018)</a>. Two markers, Gm19_1587494 and Gm18_57370051, were found to be associated with important traits from previous studies. The marker Gm19_1587494 was also found to be the key association to canopy coverage (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Xavier <em>et al.</em> 2017b</a>), which means that canopy coverage could be associated with the three yield components. The marker Gm18_57370051 is linked to the stem termination gene Dt2 (Bernard <em>et al.</em> 1972), which has been previously detecting in NAM families by <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Ping <em>et al.</em> (2014)</a>. In previous studies, Hao <em>et al.</em> (2011) and <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Fang <em>et al.</em> (2017)</a> found that Dt2 is an influential gene on the number of pods and nodes. The Dt2 gene is also believed to have played a role in the soybean domestication (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Sedivy <em>et al.</em> 2017</a>).</p><p id="p-38">The markers that were found to be associated to yield components in this study had little to no impact in maturity, which can be a major limiting factor to their use in breeding as most QTL that improve yield often increase the number of days to maturity (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). However, the QTL peaks also had a limited impact on grain yield across family, with effects ranging from -0.46 to 0.62 bu/ac.</p><p id="p-39">It is important to point out that <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> presents an average effect of allele substitution for simplicity. However, two association methods deployed in this study do not directly estimate the allele effects: The MLM utilized in this study computes the significance from within-family effects, hence capturing signal in different linkage phases between marker and QTL. The RFR also does not necessarily provide an allele effect, instead it computes recursive decision trees that would capture QTL with additive, dominant or epistatic effect. Therefore, the intend of this study was mostly focused on tracking which markers are likely associated to the yield components rather than inferring from which parent the desirable alleles are inherited from.</p></div><div id="sec-23" class="subsection"><h3>Genomic selection</h3><p id="p-40">Markers are informative in two levels for genomic predictions: they can inform the relationship and detect markers linked to, or under linkage disequilibrium with, the quantitative trait loci (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Habier <em>et al.</em> 2007</a>). Within-family predictive ability solely relies on the linkage disequilibrium (LD) between markers and QTL, as the relationship among individuals is constant. The predictions of families not included in the training set (leave-family-out) can yield mixed results since the training set often holds families with shared ancestry. Of course, the controlled ancestry is a key property of NAM populations since all families share a common parent and, therefore, the outcome predictive ability is higher than the non-experimental population where neither parent has offspring in the calibration set. Predictions performed across family are presumably the most likely to be accurate, as they capture relationships among families and disequilibrium between markers and QTL.</p><p id="p-41"><a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a> depicts well the expected predictive ability, as within-family predictions hold a high degree of uncertainty, with correlations averaging from 0.064 across yield components, followed by leave-family-out predictions, with an average correlation of 0.092, and the most predictive was across-family predictions, with average correlations above 0.224. Predictive abilities computed from leave-family-out and within-family can be penalized from the fact that some families presented had near-zero heritability and hence no variation for yield components. Within-family predictions may be further penalized due to the small population size to calibrate the genomic models. However, across family predictions are relatively more accurate as those capture both relationship and LD information, and the lower dispersion of the predictions can be attributed to the fact that the prediction model is large, containing a large number of full- and half-siblings. Results from the enhancement in predictive ability due to the joint availability of LD and relationship information have been previously presented from a theoretical standpoint by <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Habier <em>et al.</em> (2013)</a> and <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Schopp <em>et al.</em> (2017)</a>, and similar results in real data were reported by <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Ogut <em>et al.</em> (2015)</a> in the maize NAM population. In hybrid maize study, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lehermeier <em>et al.</em> (2014)</a> claimed 375 half-siblings to provide the same predictive ability of 50 full-siblings, but emphasized the degree of relatedness among families would also play a key role in the predictive ability.</p><p id="p-42">Prediction accuracies estimated across families can also have a misleading interpretation, as these are subject to the Simpson paradox (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chipman and Braun 2017</a>), where the model is able to detect large differences across families, but the predicted families may display negative correlation within-family. Such limitations could be addressed if the cross-validations across-family were performed sampling 20% of individuals from each family and training with the remaining set comprising all families, then estimating the average within-family predictions. However, across-family validations have two advantages: (1) these indicate the predictive potential of selections performed across populations and (2) provide results that can be more easily compared to other literature reports, as most studies performed cross-validations disregarding within-population studies.</p><p id="p-43">The difference in predictive ability between GBLUP and BayesB, which translated into an average improvement of 0.02 going from GBLUP to BayesB, is due the larger flexibility the BayesB model, which is more likely to capture large effects and perform variable selection (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Meuwissen <em>et al.</em> 2001</a>, <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Habier <em>et al.</em> 2011</a>, <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Pérez and de los Campos 2014</a>, <a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Xavier <em>et al.</em> 2016</a>). Having a comparison between GBLUP and BayesB can provide an insight into the genetic architecture of the trait under evaluation (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Daetwyler <em>et al.</em> 2013</a>). In this study, we expected BayesB to outperform GBLUP since we uncover a partially oligogenic architecture from the association analysis, but a key piece of information that the genomic prediction analysis provides is discrepancy between GBLUP and BayesB, which inform the degree to which the genetic architecture of the traits under evaluation depart from a polygenic architecture.</p><p id="p-44">Note that the advantage provided by changing the model from GBLUP to BayesB is nowhere comparable to the difference in predictability between cross-validation methods (<em>i.e.</em>, within-family, leave-family-out, and across-family). The reason why this phenomenon occurs is that different methods may improve how well the model detects the genetic architecture, but different types of cross-validation provide different information. Thus, gains associated to the choice of a prior are often considered negligible in comparison to increases in population size, better experimental practices, or more representative calibration sets (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">de los Campos <em>et al.</em> 2013</a>, <a id="xref-ref-59-4" class="xref-bibr" href="#ref-59">Xavier <em>et al.</em> 2016</a>). A possible way of capturing more information for genomic prediction is the explicit modeling of other sources of genetic information, such as dominance and epistasis (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Xu 2013</a>). As presented in this study, yield components in some populations have a greater influence of epistasis than the additive background and, on average, the within-family variance decomposition indicates that additive genetics explains as much of the yield components phenotypes as epistasis (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>).</p></div><div id="sec-24" class="subsection"><h3>Stability and plasticity</h3><p id="p-45">When assessing genotype-by-environment, the total genetic correlation was larger the additive genetic correlation for all yield components (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>), approximately ranging from 0 to 0.4, whereas the additive genetic correlations ranged from 0 to 0.25. The discrepancy between genetic and additive genetic correlations is attributed to the genetic control due to QTL and non-additive polygenic genetic background.</p><p id="p-46">For the families with near-zero genotype-by-environment correlation, performing selections with a single year of data may not reflect into observable genetic gains in the coming years, and that collecting data from more environments may not necessarily increase the predictive ability of the yield components. Particularly for yield components, low genotype-by-environment correlations is not necessarily bad since the soybean yield plasticity relies on reallocating resources among yield components, which serves as a physiological response to mitigate yield losses under stress (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Board and Tan 1995</a>, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Board <em>et al.</em> 1997</a>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Pedersen and Lauer 2004</a>, <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Zhang <em>et al.</em> 2004</a>). Whereas yield components are mainly responsible for the yield formations, these are not necessarily the best linear yield predictors (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Board and Modali 2005</a>). For example, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Board and Harville (1993)</a> showed that the number of pods serves as the mechanism by which seed production increases in response to greater light interception.</p><p id="p-47">Our previous study (<a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">Xavier <em>et al.</em> 2017a</a>) assessed the association among soybean agronomic traits and yield components in the SoyNAM population-based on undirected graphical models. The graphical models depicted genetic and environmental interdependence among yield components. That means that interactions among yield components occur due to genetic forces as well as a response to environmental stimuli and agronomic practices. Such a phenomenon is also described in a summary of agronomic studies on soybean yield components authored by <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Board and Kahlon (2011)</a>. The interactions among yield components play a key role in the redistribution of resources and yield stability (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ball <em>et al.</em> 2000</a>). It is possible that breeding any given yield component toward extreme values may result in a compromised ability of soybeans to compensate yield under stress (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Malausa <em>et al.</em> 2005</a>).</p></div><div id="sec-25" class="subsection"><h3>Yield increases</h3><p id="p-48">From the standpoint of trait decomposition, the breaking down of grain yield into pods and nodes does not seem to be an effective approach since there is no strong evidence that these yield components are more heritable than yield (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>) or strong genetic correlation to yield (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5A</a>) that would justify the selection based on yield components. With the exception of a few families, yield components are not good proxies for grain yield (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5C</a>). It is possible that the genetic architecture of the yield components under evaluation is just as complex as grain yield itself, not justifying predicting yield components instead of yield <em>per se</em>.</p><p id="p-49">In our previous genomic prediction study (<a id="xref-ref-59-5" class="xref-bibr" href="#ref-59">Xavier <em>et al.</em> 2016</a>), we assessed how a variety of different genomic prediction models would predict the agronomic traits and yield components under the following scenario: within year and across-population. Even though that study did not provide in-depth insight into the genetic architecture of yield components, it was found that genomic prediction models that can jointly account for large effect QTL and epistasis were advantageous over simpler prediction approaches. That study also found that predicting yield is easier than predicting yield components. Those results were further confirmed by the current study, where we assessed the architecture of yield components with more data and under different approaches.</p></div><div id="sec-26" class="subsection"><h3>Phenotyping</h3><p id="p-50">A major challenge of working with yield components is the data collection as the counting is highly subjective to human error, lowering the trait heritability and affecting the signal detection in downstream analysis. As deep learning methods for computer vision become increasingly population for phenotyping morphological traits (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Singh <em>et al.</em> 2018</a>), the current limitations with data collection could be addressed by an automated high-throughput phenotyping instead of human counts, that would likely increase both accuracy and scalability of the process. A recent study by <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Zhang <em>et al.</em> (2019)</a> provides a procedure using computer vision for counting soybean pod under experimental settings that would address the phenotypic limitation of this study. Similarly, <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Uzal <em>et al.</em> (2018)</a> and <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Li <em>et al.</em> (2019)</a> recently proposed an imagery system for counting seeds directly from images of soybean pods, yet another yield component limited by the challenging phenotyping. Technologies that enable better, faster and cheaper data collection remain a key limiting factor for the research in yield components.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-51">We thank the SoyNAM collaborators for their contributions to the experiment. William Beavis for experimental design, Qijan Song and Perry Cregan for genotyping, and Jim Specht and Brian Diers for creating the germplasm resource. We thank Chris Hoagland and Curtis Brackett for managing the experiments and contributed to collect the phenotypes. We thank the United Soybean Board for funding the field experiment in 2013 and Dow AgroSciences (now Corteva Agrisciences) for funding the data collection in 2013 and 2014, and the field experiment in 2015.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-52"><em>Communicating editor: A. Lipka</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-09-23"><span class="received-label">Received </span>September 23, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-12-06"><span class="accepted-label">Accepted </span>December 6, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Xavier, Rainey</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.1" data-doi="10.2135/cropsci2000.4041070x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ball</span>,  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. C.</span> <span class="cit-name-surname">Purcell</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. D.</span> <span class="cit-name-surname">Vories</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Short-season soybean yield compensation in response to population and water regime.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1070</span>–<span class="cit-lpage">1078</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2000.4041070x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBall%26rft.auinit1%253DR.%2BA.%26rft.volume%253D40%26rft.issue%253D4%26rft.spage%253D1070%26rft.epage%253D1078%26rft.atitle%253DShort-Season%2BSoybean%2BYield%2BCompensation%2Bin%2BResponse%2Bto%2BPopulation%2Band%2BWater%2BRegime%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2000.4041070x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2000.4041070x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000089410600021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-web" id="cit-10.2.665.2"><div class="cit-metadata unstructured">Bates, D., M. Mächler, B. Bolker, and S. Walker, 2014 Fitting linear mixed-effects models using lme4. arXiv preprint arXiv:1406.5823.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.2.665.3" data-doi="10.2135/cropsci1972.0011183X001200020028x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernard</span>,  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Two Genes Affecting Stem Termination in Soybeans 1.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">239</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1972.0011183X001200020028x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernard%26rft.auinit1%253DR.%2BL.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253D235%26rft.epage%253D239%26rft.atitle%253DTwo%2BGenes%2BAffecting%2BStem%2BTermination%2Bin%2BSoybeans%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1972.0011183X001200020028x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1972.0011183X001200020028x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1972M322900028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.665.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Board</span>,  <span class="cit-name-given-names">J. E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. S.</span> <span class="cit-name-surname">Kahlon</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-source">Soybean Yield Formation: What controls it and how it can be improved. Soybean Physiology and Biochemistry</span>, <span class="cit-publ-name">INTECH Open Access Publisher</span>, <span class="cit-publ-loc">Rijeka, Croatia</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.5" data-doi="10.2135/cropsci2004.0602"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Board</span>,  <span class="cit-name-given-names">J. E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Modali</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Dry matter accumulation predictors for optimal yield in soybean.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1790</span>–<span class="cit-lpage">1799</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2004.0602</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBoard%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D45%26rft.issue%253D5%26rft.spage%253D1790%26rft.epage%253D1799%26rft.atitle%253DDry%2BMatter%2BAccumulation%2BPredictors%2Bfor%2BOptimal%2BYield%2Bin%2BSoybean%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2004.0602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2004.0602&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000231764400016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.6" data-doi="10.2135/cropsci1997.0011183X003700030030x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Board</span>,  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Kang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. G.</span> <span class="cit-name-surname">Harville</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Path analyses identify indirect selection criteria for yield of late-planted soybean.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">879</span>–<span class="cit-lpage">884</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1997.0011183X003700030030x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBoard%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D37%26rft.issue%253D3%26rft.spage%253D879%26rft.epage%253D884%26rft.atitle%253DPath%2BAnalyses%2BIdentify%2BIndirect%2BSelection%2BCriteria%2Bfor%2BYield%2Bof%2BLate-Planted%2BSoybean%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1997.0011183X003700030030x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1997.0011183X003700030030x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.7" data-doi="10.2135/cropsci1995.0011183X003500030035x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Board</span>,  <span class="cit-name-given-names">J. E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Tan</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Assimilatory capacity effects on soybean yield components and pod number.</span> <abbr class="cit-jnl-abbrev">Crop</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">846</span>–<span class="cit-lpage">851</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1995.0011183X003500030035x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBoard%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D35%26rft.issue%253D3%26rft.spage%253D846%26rft.epage%253D851%26rft.atitle%253DAssimilatory%2BCapacity%2BEffects%2Bon%2BSoybean%2BYield%2BComponents%2Band%2BPod%2BNumber%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1995.0011183X003500030035x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1995.0011183X003500030035x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995QW74800033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.8" data-doi="10.2135/cropsci1993.0011183X003300040028x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Board</span>,  <span class="cit-name-given-names">J. E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. G.</span> <span class="cit-name-surname">Harville</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Soybean yield component responses to a light interception gradient during the reproductive period.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">772</span>–<span class="cit-lpage">777</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1993.0011183X003300040028x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBoard%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D33%26rft.issue%253D4%26rft.spage%253D772%26rft.epage%253D777%26rft.atitle%253DSoybean%2BYield%2BComponent%2BResponses%2Bto%2Ba%2BLight%2BInterception%2BGradient%2Bduring%2Bthe%2BReproductive%2BPeriod%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1993.0011183X003300040028x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1993.0011183X003300040028x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.9" data-doi="10.1371/journal.pone.0093379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Botta</span>,  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Louppe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Geurts</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Wehenkel</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Exploiting SNP correlations within random forest for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e93379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0093379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0093379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0093379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.10"><div class="cit-metadata unstructured">Carter, T. E., R. L. Nelson, C. H. Sneller, Z. Cui, H. R. Boerma, and J. E. Specht, 2004 Genetic diversity in soybean. Soybeans: Improvement, production, and uses.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.11" data-doi="10.1002/sim.6862"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chipman</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Braun</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Simpson’s paradox in the integrated discrimination improvement.</span> <abbr class="cit-jnl-abbrev">Stat. Med.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">4468</span>–<span class="cit-lpage">4481</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/sim.6862</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BMed.%26rft.volume%253D36%26rft.spage%253D4468%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fsim.6862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/sim.6862&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.12" data-doi="10.1534/genetics.112.147983"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>,  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Calus</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Pong-Wong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Hickey</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction in animals and plants: simulation of data, validation, reporting, and benchmarking.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">365</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.147983</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147983%26rft_id%253Dinfo%253Apmid%252F23222650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi82NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.13" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Hickey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Pong-Wong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. D.</span> <span class="cit-name-surname">Daetwyler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Calus</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.143313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi82NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Diers</span>,  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Specht</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Rainey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Cregan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Song</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genetic Architecture of Soybean Yield and Agronomic Traits. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">3367</span>–<span class="cit-lpage">3375</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D8%26rft.spage%253D3367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>,  <span class="cit-name-given-names">R. W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. A.</span> <span class="cit-name-surname">Churchill</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Permutation tests for multiple loci affecting a quantitative character.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">142</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDoerge%26rft.auinit1%253DR.%2BW.%26rft.volume%253D142%26rft.issue%253D1%26rft.spage%253D285%26rft.epage%253D294%26rft.atitle%253DPermutation%2BTests%2Bfor%2BMultiple%2BLoci%2BAffecting%2Ba%2BQuantitative%2BCharacter%26rft_id%253Dinfo%253Apmid%252F8770605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQyLzEvMjg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi82NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.16" data-doi="10.1016/j.eja.2005.04.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Egli</span>,  <span class="cit-name-given-names">D. B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. P.</span> <span class="cit-name-surname">Bruening</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Temporal profiles of pod production and pod set in soybean.</span> <abbr class="cit-jnl-abbrev">Eur. J. Agron.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.eja.2005.04.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEur.%2BJ.%2BAgron.%26rft.volume%253D24%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.eja.2005.04.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.eja.2005.04.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000234069500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.665.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>,  <span class="cit-name-given-names">D. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. F. C.</span> <span class="cit-name-surname">Mackay</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source">Introduction to quantitative genetics</span>, <span class="cit-publ-name">Longman Group</span>, <span class="cit-publ-loc">Essex, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.18" data-doi="10.1186/s13059-017-1289-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Ma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Wu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Wang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome-wide association studies dissect the genetic networks underlying agronomical traits in soybean.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">161</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-017-1289-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D18%26rft.spage%253D161%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-017-1289-9%26rft_id%253Dinfo%253Apmid%252F28838319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-017-1289-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28838319&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.19" data-doi="10.2135/cropsci1971.0011183X001100060051x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fehr</span>,  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. E.</span> <span class="cit-name-surname">Caviness</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. T.</span> <span class="cit-name-surname">Burmood</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Pennington</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">Stage of development descriptions for soybeans, Glycine Max (L.) Merrill 1.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">929</span>–<span class="cit-lpage">931</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1971.0011183X001100060051x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DFehr%26rft.auinit1%253DW.%2BR.%26rft.volume%253D11%26rft.issue%253D6%26rft.spage%253D929%26rft.epage%253D931%26rft.atitle%253DStage%2Bof%2BDevelopment%2BDescriptions%2Bfor%2BSoybeans%252C%2BGlycine%2BMax%2B%2528L.%2529%2BMerrill%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1971.0011183X001100060051x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1971.0011183X001100060051x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1971L246700051&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.20" data-doi="10.1186/1297-9686-41-55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>,  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. F.</span> <span class="cit-name-surname">Taylor</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Deregressing estimated breeding values and weighting information for genomic regression analyses.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">55</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-41-55</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DGarrick%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D41%26rft.spage%253D55%26rft.epage%253D55%26rft.atitle%253DDeregressing%2Bestimated%2Bbreeding%2Bvalues%2Band%2Bweighting%2Binformation%2Bfor%2Bgenomic%2Bregression%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-55%26rft_id%253Dinfo%253Apmid%252F20043827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20043827&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.21" data-doi="10.1534/genetics.113.152207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Garrick</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic BLUP decoded: a look into the black box of genomic prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">597</span>–<span class="cit-lpage">607</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.152207</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152207%26rft_id%253Dinfo%253Apmid%252F23640517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi82NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.22" data-doi="10.1186/1471-2105-12-186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kizilkaya</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Garrick</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Extension of the Bayesian alphabet for genomic selection.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">186</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-186</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D186%26rft.epage%253D186%26rft.atitle%253DExtension%2Bof%2Bthe%2Bbayesian%2Balphabet%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-186%26rft_id%253Dinfo%253Apmid%252F21605355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21605355&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.23" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Dekkers</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.081190</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzY2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.24" data-doi="10.1007/s00122-011-1719-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Cheng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Yin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Cui</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Zhang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Identification of single nucleotide polymorphisms and haplotypes associated with yield and yield components in soybean (Glycine max) landraces across multiple environments.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">447</span>–<span class="cit-lpage">458</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-011-1719-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DHao%26rft.auinit1%253DD.%26rft.volume%253D124%26rft.issue%253D3%26rft.spage%253D447%26rft.epage%253D458%26rft.atitle%253DIdentification%2Bof%2Bsingle%2Bnucleotide%2Bpolymorphisms%2Band%2Bhaplotypes%2Bassociated%2Bwith%2Byield%2Band%2Byield%2Bcomponents%2Bin%2Bsoybean%2B%2528Glycine%2Bmax%2529%2Blandraces%2Bacross%2Bmultiple%2Benvironments.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1719-0%26rft_id%253Dinfo%253Apmid%252F21997761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1719-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21997761&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.25" data-doi="10.2135/cropsci1982.0011183X002200050044x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Herbert</span>,  <span class="cit-name-given-names">S. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. V.</span> <span class="cit-name-surname">Litchfield</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Partitioning Soybean Seed Yield Components 1.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1074</span>–<span class="cit-lpage">1079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1982.0011183X002200050044x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHerbert%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D22%26rft.issue%253D5%26rft.spage%253D1074%26rft.epage%253D1079%26rft.atitle%253DPartitioning%2BSoybean%2BSeed%2BYield%2BComponents%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1982.0011183X002200050044x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1982.0011183X002200050044x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.26" data-doi="10.1270/jsbbs.63.441"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Kan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Hong</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Association mapping of yield-related traits and SSR markers in wild soybean (Glycine soja Sieb. and Zucc.).</span> <abbr class="cit-jnl-abbrev">Breed. Sci.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">441</span>–<span class="cit-lpage">449</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1270/jsbbs.63.441</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBreed.%2BSci.%26rft.volume%253D63%26rft.spage%253D441%26rft_id%253Dinfo%253Adoi%252F10.1270%252Fjsbbs.63.441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1270/jsbbs.63.441&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.27" data-doi="10.2134/agronj2010.0300"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kahlon</span>,  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Board</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Kang</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">An analysis of yield component changes for new <em>vs.</em> old soybean cultivars.</span> <abbr class="cit-jnl-abbrev">Agron. J.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2134/agronj2010.0300</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgron.%2BJ.%26rft.volume%253D103%26rft.spage%253D13%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj2010.0300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj2010.0300&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.28" data-doi="10.1080/15427528.2011.637155"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kahlon</span>,  <span class="cit-name-given-names">C. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Board</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Growth dynamic factors explaining yield improvement in new <em>vs.</em> old soybean cultivars.</span> <abbr class="cit-jnl-abbrev">J. Crop Improv.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">282</span>–<span class="cit-lpage">299</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/15427528.2011.637155</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCrop%2BImprov.%26rft.volume%253D26%26rft.spage%253D282%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15427528.2011.637155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/15427528.2011.637155&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.29" data-doi="10.1007/s11032-018-0810-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaler</span>,  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Ray</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. T.</span> <span class="cit-name-surname">Schapaugh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. K.</span> <span class="cit-name-surname">Davies</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">King</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Association mapping identifies loci for canopy coverage in diverse soybean genotypes.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">50</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11032-018-0810-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D38%26rft.spage%253D50%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-018-0810-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-018-0810-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lado</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Matus</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Rodríguez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Inostroza</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Poland</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Increased genomic prediction accuracy in wheat breeding through spatial adjustment of field trial data. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">2105</span>–<span class="cit-lpage">2114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D3%26rft.spage%253D2105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.31" data-doi="10.1371/journal.pone.0094150"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langewisch</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Vincent</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Joshi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Xu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Major soybean maturity gene haplotypes revealed by SNPViz analysis of 72 sequenced soybean genomes.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e94150</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0094150</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De94150%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0094150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0094150&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.32" data-doi="10.1534/genetics.114.161943"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Krämer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Bauer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Bauland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Camisan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Usefulness of multiparental populations of maize (Zea mays L.) for genome-based prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">16</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.161943</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLehermeier%26rft.auinit1%253DC.%26rft.volume%253D198%26rft.issue%253D1%26rft.spage%253D3%26rft.epage%253D16%26rft.atitle%253DUsefulness%2Bof%2BMultiparental%2BPopulations%2Bof%2BMaize%2B%2528Zea%2Bmays%2BL.%2529%2Bfor%2BGenome-Based%2BPrediction%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.161943%26rft_id%253Dinfo%253Apmid%252F25236445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiMTk4LzEvMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvNjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.33" data-doi="10.1109/ACCESS.2019.2916931"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Khattak</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sun</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Soybean Seed Counting Based on Pod Image Using Two-Column Convolution Neural Network.</span> <abbr class="cit-jnl-abbrev">IEEE Access</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">64177</span>–<span class="cit-lpage">64185</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1109/ACCESS.2019.2916931</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2BAccess%26rft.volume%253D7%26rft.spage%253D64177%26rft_id%253Dinfo%253Adoi%252F10.1109%252FACCESS.2019.2916931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/ACCESS.2019.2916931&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.34" data-doi="10.1111/j.0030-1299.2005.13563.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malausa</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Guillemaud</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Lapchin</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Combining genetic variation and phenotypic plasticity in tradeoff modelling.</span> <abbr class="cit-jnl-abbrev">Oikos</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">330</span>–<span class="cit-lpage">338</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.0030-1299.2005.13563.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOikos%26rft.volume%253D110%26rft.spage%253D330%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0030-1299.2005.13563.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0030-1299.2005.13563.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.35" data-doi="10.2135/cropsci2009.08.0456"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mikel</span>,  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. W.</span> <span class="cit-name-surname">Diers</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Nelson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. H.</span> <span class="cit-name-surname">Smith</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic diversity and agronomic improvement of North American soybean germplasm.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1219</span>–<span class="cit-lpage">1229</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2009.08.0456</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DMikel%26rft.auinit1%253DM.%2BA.%26rft.volume%253D50%26rft.issue%253D4%26rft.spage%253D1219%26rft.epage%253D1229%26rft.atitle%253DGenetic%2BDiversity%2Band%2BAgronomic%2BImprovement%2Bof%2BNorth%2BAmerican%2BSoybean%2BGermplasm%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.08.0456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.08.0456&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000279138200012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.36" data-doi="10.1111/j.1439-0388.2008.00774.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>,  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Reliable computing in estimation of variance components.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">363</span>–<span class="cit-lpage">370</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0388.2008.00774.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DMisztal%26rft.auinit1%253DI.%26rft.volume%253D125%26rft.issue%253D6%26rft.spage%253D363%26rft.epage%253D370%26rft.atitle%253DReliable%2Bcomputing%2Bin%2Bestimation%2Bof%2Bvariance%2Bcomponents.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2008.00774.x%26rft_id%253Dinfo%253Apmid%252F19134071%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2008.00774.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19134071&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261063900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.37" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Hayes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Goddard</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzY2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.38" data-doi="10.1016/j.fcr.2018.10.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nico</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Miralles</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. G.</span> <span class="cit-name-surname">Kantolic</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Natural post-flowering photoperiod and photoperiod sensitivity: Roles in yield-determining processes in soybean.</span> <abbr class="cit-jnl-abbrev">Field Crops Res.</abbr> <span class="cit-vol">231</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">152</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fcr.2018.10.019</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DField%2BCrops%2BRes.%26rft.volume%253D231%26rft.spage%253D141%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fcr.2018.10.019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fcr.2018.10.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.39" data-doi="10.1038/hdy.2014.123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ogut</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Bian</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Bradbury</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Holland</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Joint-multiple family linkage analysis predicts within-family variation better than single-family analysis of the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">552</span>–<span class="cit-lpage">563</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2014.123</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D114%26rft.spage%253D552%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2014.123%26rft_id%253Dinfo%253Apmid%252F25585918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2014.123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25585918&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.40" data-doi="10.2134/agronj2004.1372"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>,  <span class="cit-name-given-names">P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. G.</span> <span class="cit-name-surname">Lauer</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Response of soybean yield components to management system and planting date.</span> <abbr class="cit-jnl-abbrev">Agron. J.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">1372</span>–<span class="cit-lpage">1381</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2134/agronj2004.1372</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgronomy%2BJournal%26rft.stitle%253DAgron.%2BJ.%26rft.aulast%253DPedersen%26rft.auinit1%253DP.%26rft.volume%253D96%26rft.issue%253D5%26rft.spage%253D1372%26rft.epage%253D1381%26rft.atitle%253DResponse%2Bof%2BSoybean%2BYield%2BComponents%2Bto%2BManagement%2BSystem%2Band%2BPlanting%2BDate%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj2004.1372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj2004.1372&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000224037800020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.41" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>,  <span class="cit-name-given-names">P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de Los Campos</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide regression and prediction with the BGLR statistical package.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.164442</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi82NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.42" data-doi="10.1105/tpc.114.126938"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ping</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Sun</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Zhao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Li</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Dt2 is a gain-of-function MADS-domain factor gene that specifies semi-determinacy in soybean.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2831</span>–<span class="cit-lpage">2842</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.114.126938</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.114.126938%26rft_id%253Dinfo%253Apmid%252F25005919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI2LzcvMjgzMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvNjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rincker</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Nelson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Specht</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Sleper</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Cary</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic improvement of US soybean in maturity groups II, III, and IV.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1419</span>–<span class="cit-lpage">1432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D1419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.44" data-doi="10.2134/agronj2008.0014x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>,  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. P.</span> <span class="cit-name-surname">Conley</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Volenec</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Santini</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Analysis of high yielding, early-planted soybean in Indiana.</span> <abbr class="cit-jnl-abbrev">Agron. J.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">139</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2134/agronj2008.0014x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgronomy%2BJournal%26rft.stitle%253DAgron.%2BJ.%26rft.aulast%253DRobinson%26rft.auinit1%253DA.%2BP.%26rft.volume%253D101%26rft.issue%253D1%26rft.spage%253D131%26rft.epage%253D139%26rft.atitle%253DAnalysis%2Bof%2BHigh%2BYielding%252C%2BEarly-Planted%2BSoybean%2Bin%2BIndiana%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj2008.0014x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj2008.0014x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schopp</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Müller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. C.</span> <span class="cit-name-surname">Wientjes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Melchinger</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic prediction within and across biparental families: means and variances of prediction accuracy and usefulness of deterministic equations. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">3571</span>–<span class="cit-lpage">3586</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D7%26rft.spage%253D3571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.46" data-doi="10.1111/nph.14418"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sedivy</span>,  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Wu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Hanzawa</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Soybean domestication: the origin, genetic architecture and molecular bases.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">214</span>: <span class="cit-fpage">539</span>–<span class="cit-lpage">553</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.14418</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D214%26rft.spage%253D539%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.14418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.14418&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.47" data-doi="10.1016/j.tplants.2018.07.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>,  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Ganapathysubramanian</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sarkar</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Singh</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Deep learning for plant stress phenotyping: trends and future perspectives.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">883</span>–<span class="cit-lpage">898</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2018.07.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D23%26rft.spage%253D883%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2018.07.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2018.07.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.48"><div class="cit-metadata unstructured">Specht, J. E., B. W. Diers, R. L. Nelson, J. Francisco, F. de Toledo, J. A. Torrion, and P. Grassini, 2014 Soybean. Yield gains in major US field crops, 311–356.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.49" data-doi="10.2135/cropsci1999.3961560x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Specht</span>,  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Hume</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. V.</span> <span class="cit-name-surname">Kumudini</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Soybean yield potential: a genetic and physiological perspective.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1560</span>–<span class="cit-lpage">1570</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1999.3961560x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DSpecht%26rft.auinit1%253DJ.E.%26rft.volume%253D39%26rft.issue%253D6%26rft.spage%253D1560%26rft.epage%253D1570%26rft.atitle%253DSoybean%2BYield%2BPotential--A%2BGenetic%2Band%2BPhysiological%2BPerspective%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1999.3961560x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1999.3961560x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000084787700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.50" data-doi="10.3835/plantgenome2016.10.0109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Yan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Quigley</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. D.</span> <span class="cit-name-surname">Jordan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Fickus</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genetic characterization of the soybean nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.10.0109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.10.0109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.10.0109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.51" data-doi="10.2134/agronj14.0003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suhre</span>,  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. H.</span> <span class="cit-name-surname">Weidenbenner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. C.</span> <span class="cit-name-surname">Rowntree</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. W.</span> <span class="cit-name-surname">Wilson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Naeve</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Soybean yield partitioning changes revealed by genetic gain and seeding rate interactions.</span> <abbr class="cit-jnl-abbrev">Agron. J.</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">1631</span>–<span class="cit-lpage">1642</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2134/agronj14.0003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgron.%2BJ.%26rft.volume%253D106%26rft.spage%253D1631%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj14.0003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj14.0003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.52" data-doi="10.1007/s11103-013-0076-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qiu</span>,  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. L.</span> <span class="cit-name-surname">Xing</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Guo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Jackson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A platform for soybean molecular breeding: the utilization of core collections for food security.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">50</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11103-013-0076-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMol.%2BBiol.%26rft.volume%253D83%26rft.spage%253D41%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11103-013-0076-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11103-013-0076-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.53" data-doi="10.1016/j.compag.2018.04.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uzal</span>,  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. L.</span> <span class="cit-name-surname">Grinblat</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Namías</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. G.</span> <span class="cit-name-surname">Larese</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Bianchi</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Seed-per-pod estimation for plant breeding using deep learning.</span> <abbr class="cit-jnl-abbrev">Comput. Electron. Agric.</abbr> <span class="cit-vol">150</span>: <span class="cit-fpage">196</span>–<span class="cit-lpage">204</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.compag.2018.04.024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BElectron.%2BAgric.%26rft.volume%253D150%26rft.spage%253D196%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.compag.2018.04.024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.compag.2018.04.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.54" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>,  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.55" data-doi="10.1093/bioinformatics/btz794"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xavier</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. M.</span> <span class="cit-name-surname">Muir</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Rainey</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">bWGR: Bayesian Whole-Genome Regression.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-fpage">1</span>–<span class="cit-lpage">2</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btz794</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtz794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btz794&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xavier</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Jarquin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Howard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Ramasubramanian</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Specht</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genome-Wide analysis of grain yield stability and environmental interactions in a multiparental soybean population. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">519</span>–<span class="cit-lpage">529</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D8%26rft.spage%253D519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.57" data-doi="10.1007/s10681-017-1975-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xavier</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Hall</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Casteel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Muir</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Rainey</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">a</span> <span class="cit-article-title">Using unsupervised learning techniques to assess interactions among complex traits in soybeans.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">213</span>: <span class="cit-fpage">200</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10681-017-1975-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D213%26rft.spage%253D200%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-017-1975-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-017-1975-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.58" data-doi="10.1534/genetics.116.198713"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xavier</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Hall</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Hearst</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Cherkauer</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Rainey</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">b</span> <span class="cit-article-title">Genetic architecture of phenomic-enabled canopy coverage in Glycine max.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">1081</span>–<span class="cit-lpage">1089</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.198713</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.198713%26rft_id%253Dinfo%253Apmid%252F28363978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNi8yLzEwODEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzY2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xavier</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. M.</span> <span class="cit-name-surname">Muir</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Rainey</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Assessing predictive properties of genome-wide selection in soybeans. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2611</span>–<span class="cit-lpage">2616</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D6%26rft.spage%253D2611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.60" data-doi="10.1093/bioinformatics/btv448"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xavier</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Xu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. M.</span> <span class="cit-name-surname">Muir</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Rainey</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">NAM: association studies in multiple populations.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3862</span>–<span class="cit-lpage">3864</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv448%26rft_id%253Dinfo%253Apmid%252F26243017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv448&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26243017&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.61" data-doi="10.1534/genetics.113.157032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mapping quantitative trait loci by controlling polygenic background effects.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">1209</span>–<span class="cit-lpage">1222</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.157032</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.157032%26rft_id%253Dinfo%253Apmid%252F24077303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS80LzEyMDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzY2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.62" data-doi="10.1534/genetics.110.125062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Xia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Hideshima</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Tsubokura</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sato</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A map-based cloning strategy employing a residual heterozygous line reveals that the GIGANTEA gene is involved in soybean maturity and flowering.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">395</span>–<span class="cit-lpage">407</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.125062</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.125062%26rft_id%253Dinfo%253Apmid%252F21406680%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzIvMzk1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi82NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.63" data-doi="10.1534/genetics.108.098772"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Hideshima</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Xia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Tsubokura</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sato</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Map-based cloning of the gene associated with the soybean maturity locus E3.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">1251</span>–<span class="cit-lpage">1262</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.108.098772</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.098772%26rft_id%253Dinfo%253Apmid%252F19474204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Mi80LzEyNTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzY2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-web" id="cit-10.2.665.64"><div class="cit-metadata unstructured">Wright, M. N., and A. Ziegler, 2015 Ranger: a fast implementation of random forests for high dimensional data in C++ and R. arXiv preprint arXiv:1508.04409.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.65" data-doi="10.1073/pnas.1117982109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Watanabe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Yamada</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Tsubokura</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Nakashima</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Positional cloning and characterization reveal the molecular basis for soybean maturity locus E1 that regulates photoperiodic flowering.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">E2155</span>–<span class="cit-lpage">E2164</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1117982109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1117982109%26rft_id%253Dinfo%253Apmid%252F22619331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzMyL0UyMTU1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi82NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. K.</span> <span class="cit-name-surname">Ramamurthy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Qi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Dekkers</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Empirical Comparisons of Different Statistical Models To Identify and Validate Kernel Row Number-Associated Variants from Structured Multi-parent Mapping Populations of Maize. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">3567</span>–<span class="cit-lpage">3575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D8%26rft.spage%253D3567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.67" data-doi="10.1534/genetics.104.035857"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>,  <span class="cit-name-given-names">Z. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Wang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Zou</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Modeling quantitative trait loci and interpretation of models.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">1711</span>–<span class="cit-lpage">1725</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.035857</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.035857%26rft_id%253Dinfo%253Apmid%252F15654105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS8zLzE3MTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzY2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.68" data-doi="10.1111/pbr.12305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Hao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. M.</span> <span class="cit-name-surname">Sitoe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Yin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Hu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic dissection of the relationship between plant architecture and yield component traits in soybean (Glycine max) by association analysis across multiple environments.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">564</span>–<span class="cit-lpage">572</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbr.12305</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D134%26rft.spage%253D564%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbr.12305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbr.12305&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.665.69" data-doi="10.1007/s00122-003-1527-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. J.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. Z.</span> <span class="cit-name-surname">Luo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. Y.</span> <span class="cit-name-surname">He</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">QTL mapping of ten agronomic traits on the soybean (Glycine max L. Merr.) genetic map and their association with EST markers.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">1131</span>–<span class="cit-lpage">1139</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-003-1527-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DW.%2BK.%26rft.volume%253D108%26rft.issue%253D6%26rft.spage%253D1131%26rft.epage%253D1139%26rft.atitle%253DQTL%2Bmapping%2Bof%2Bten%2Bagronomic%2Btraits%2Bon%2Bthe%2Bsoybean%2B%2528%2BGlycine%2Bmax%2BL.%2BMerr.%2529%2Bgenetic%2Bmap%2Band%2Btheir%2Bassociation%2Bwith%2BEST%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-003-1527-2%26rft_id%253Dinfo%253Apmid%252F15067400%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-003-1527-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15067400&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220655000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-confproc" id="cit-10.2.665.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Z.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Ghosal</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> A deep vision based approach to real-time detection and counting of soybean pods. Presented at the Machine Learning and Cyber Agriculture Symposium, <span class="cit-publ-loc">Iowa</span> <span class="cit-publ-name">State University</span>. Available: <a href="http://www.register.extension.iastate.edu/mlcas2019">www.register.extension.iastate.edu/mlcas2019</a></cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/2/665.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/2/645" title="A Comprehensive Genome-Wide and Phenome-Wide Examination of BMI and Obesity in a Northern Nevadan Cohort" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/2/677" title="Contingent Convergence: The Ability To Detect Convergent Genomic Evolution Is Dependent on Population Size and Migration" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/2"><span>Volume 10 Issue 2, February 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/2.cover-source.jpg?itok=LSawHOSr" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/665" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-_JhNOPw4KlVY3DGM2r_EO8yROw46txttUkagQIGfzVE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/2/665" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/2/665" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/2/665" class="active" data-icon-position="" data-hide-link-title="0">Quantitative Genomic Dissection of Soybean Yield Components</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495558" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-y1q1HtuQ3HRJLDrf0wu6BDptU2CQQPiTh6l_eAY68Jc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/2/665.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/2/665" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-bNNAYt3ZxX7EbUDpBC5JJ2SRPxJpQeIlR6Ta6XkhJbM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/2/665" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/2/665.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495558" id="node495558--2" data-pisa="ggg;10/2/665" data-pisa-master="ggg;g3.119.400896" data-apath="/ggg/10/2/665.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Quantitative Genomic Dissection of Soybean Yield Components</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5034-9954" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alencar Xavier</span> and <span class="highwire-citation-author" data-delta="1">Katy M. Rainey</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">665-675;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400896 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495558--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alencar Xavier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, Purdue University, West Lafayette IN 47907 and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics, Corteva Agrisciences, Johnston IA 50131</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alencar%2BXavier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xavier%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlencar%2BXavier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5034-9954" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alencar Xavier</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Katy M. Rainey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, Purdue University, West Lafayette IN 47907 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katy%2BM.%2BRainey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rainey%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKaty%2BM.%2BRainey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:xaviera@purdue.edu" class="" data-icon-position="" data-hide-link-title="0">xaviera@purdue.edu</a>
<a href="mailto:krainey@purdue.edu" class="" data-icon-position="" data-hide-link-title="0">krainey@purdue.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/2/665" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/2/665&amp;title=Quantitative%20Genomic%20Dissection%20of%20Soybean%20Yield%20Components" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/2/665&amp;title=Quantitative%20Genomic%20Dissection%20of%20Soybean%20Yield%20Components" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/2/665&amp;title=Quantitative%20Genomic%20Dissection%20of%20Soybean%20Yield%20Components" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/2/665&amp;text=Quantitative%20Genomic%20Dissection%20of%20Soybean%20Yield%20Components" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/2/665&amp;title=Quantitative%20Genomic%20Dissection%20of%20Soybean%20Yield%20Components" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/2/665&amp;t=Quantitative%20Genomic%20Dissection%20of%20Soybean%20Yield%20Components" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/2/665&amp;title=Quantitative%20Genomic%20Dissection%20of%20Soybean%20Yield%20Components" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/2/665&amp;title=Quantitative%20Genomic%20Dissection%20of%20Soybean%20Yield%20Components" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495558/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495558" id="node495558--3" data-pisa="ggg;10/2/665" data-pisa-master="ggg;g3.119.400896" data-apath="/ggg/10/2/665.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Quantitative Genomic Dissection of Soybean Yield Components</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5034-9954" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alencar Xavier</span> and <span class="highwire-citation-author" data-delta="1">Katy M. Rainey</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">665-675;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400896 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495558--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alencar Xavier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, Purdue University, West Lafayette IN 47907 and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics, Corteva Agrisciences, Johnston IA 50131</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alencar%2BXavier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xavier%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlencar%2BXavier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5034-9954" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alencar Xavier</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Katy M. Rainey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, Purdue University, West Lafayette IN 47907 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katy%2BM.%2BRainey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rainey%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKaty%2BM.%2BRainey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:xaviera@purdue.edu" class="" data-icon-position="" data-hide-link-title="0">xaviera@purdue.edu</a>
<a href="mailto:krainey@purdue.edu" class="" data-icon-position="" data-hide-link-title="0">krainey@purdue.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=665&amp;rft.epage=675&amp;rft.atitle=Quantitative%20Genomic%20Dissection%20of%20Soybean%20Yield%20Components&amp;rft.volume=10&amp;rft.issue=2&amp;rft.date=2020-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Xavier%2C+Alencar&amp;rft.au=Rainey%2C+Katy+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495558/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495558/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495558/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495558/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495558/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495558/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495558/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495558/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495558/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495558/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495558/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495558/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/2/665" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/2/665.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/2/665.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/2/665.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
