<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/2/569" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/2/569.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/2/569.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/2/569.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/10/2/569/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/2/569.atom" />
<meta name="HW.pisa" content="ggg;10/2/569" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Simulation-Based Evaluation of Three Methods for Local Ancestry Deconvolution of Non-model Crop Species Genomes" />
<meta name="DC.Identifier" content="10.1534/g3.119.400873" />
<meta name="DC.Date" content="2020-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Cottin et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Hybridizations between species and subspecies represented major steps in the history of many crop species. Such events generally lead to genomes with mosaic patterns of chromosomal segments of various origins that may be assessed by local ancestry inference methods. However, these methods have mainly been developed in the context of human population genetics with implicit assumptions that may not always fit plant models. The purpose of this study was to evaluate the suitability of three state-of-the-art inference methods (SABER, ELAI and WINPOP) for local ancestry inference under scenarios that can be encountered in plant species. For this, we developed an R package to simulate genotyping data under such scenarios. The tested inference methods performed similarly well as far as representatives of source populations were available. As expected, the higher the level of differentiation between ancestral source populations and the lower the number of generations since admixture, the more accurate were the results. Interestingly, the accuracy of the methods was only marginally affected by i) the number of ancestries (up to six tested); ii) the sample design ( i.e. , unbalanced representation of source populations); and iii) the reproduction mode ( e.g. , selfing, vegetative propagation). If a source population was not represented in the data set, no bias was observed in inference accuracy for regions originating from represented sources and regions from the missing source were assigned differently depending on the methods. Overall, the selected ancestry inference methods may be used for crop plant analysis if all ancestral sources are known." />
<meta name="DC.Contributor" content="Aurélien Cottin" />
<meta name="DC.Contributor" content="Benjamin Penaud" />
<meta name="DC.Contributor" content="Jean-Christophe Glaszmann" />
<meta name="DC.Contributor" content="Nabila Yahiaoui" />
<meta name="DC.Contributor" content="Mathieu Gautier" />
<meta name="article:published_time" content="2020-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Simulation-Based Evaluation of Three Methods for Local Ancestry Deconvolution of Non-model Crop Species Genomes" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Hybridizations between species and subspecies represented major steps in the history of many crop species. Such events generally lead to genomes with mosaic patterns of chromosomal segments of various origins that may be assessed by local ancestry inference methods. However, these methods have mainly been developed in the context of human population genetics with implicit assumptions that may not always fit plant models. The purpose of this study was to evaluate the suitability of three state-of-the-art inference methods (SABER, ELAI and WINPOP) for local ancestry inference under scenarios that can be encountered in plant species. For this, we developed an R package to simulate genotyping data under such scenarios. The tested inference methods performed similarly well as far as representatives of source populations were available. As expected, the higher the level of differentiation between ancestral source populations and the lower the number of generations since admixture, the more accurate were the results. Interestingly, the accuracy of the methods was only marginally affected by i) the number of ancestries (up to six tested); ii) the sample design (&lt;i&gt;i.e.&lt;/i&gt;, unbalanced representation of source populations); and iii) the reproduction mode (&lt;i&gt;e.g.&lt;/i&gt;, selfing, vegetative propagation). If a source population was not represented in the data set, no bias was observed in inference accuracy for regions originating from represented sources and regions from the missing source were assigned differently depending on the methods. Overall, the selected ancestry inference methods may be used for crop plant analysis if all ancestral sources are known.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/02/01" />
<meta name="citation_mjid" content="ggg;10/2/569" />
<meta name="citation_id" content="10/2/569" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/2/569" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/2/569.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/2/569.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/2/569.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400873" />
<meta name="citation_pmid" content="31862786" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="569" />
<meta name="citation_lastpage" content="579" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Aurélien Cottin" />
<meta name="citation_author_institution" content="CIRAD, UMR AGAP, F-34398 Montpellier, France," />
<meta name="citation_author_institution" content="AGAP, Univ. Montpellier, CIRAD, INRAE, Montpellier SupAgro, Montpellier, France, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0862-784X" />
<meta name="citation_author" content="Benjamin Penaud" />
<meta name="citation_author_institution" content="CIRAD, UMR AGAP, F-34398 Montpellier, France," />
<meta name="citation_author_institution" content="AGAP, Univ. Montpellier, CIRAD, INRAE, Montpellier SupAgro, Montpellier, France, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8881-9253" />
<meta name="citation_author" content="Jean-Christophe Glaszmann" />
<meta name="citation_author_institution" content="CIRAD, UMR AGAP, F-34398 Montpellier, France," />
<meta name="citation_author_institution" content="AGAP, Univ. Montpellier, CIRAD, INRAE, Montpellier SupAgro, Montpellier, France, and" />
<meta name="citation_author" content="Nabila Yahiaoui" />
<meta name="citation_author_institution" content="CIRAD, UMR AGAP, F-34398 Montpellier, France," />
<meta name="citation_author_institution" content="AGAP, Univ. Montpellier, CIRAD, INRAE, Montpellier SupAgro, Montpellier, France, and" />
<meta name="citation_author_email" content="mathieu.gautier@inra.fr" />
<meta name="citation_author_email" content="nabila.yahiaoui@cirad.fr" />
<meta name="citation_author" content="Mathieu Gautier" />
<meta name="citation_author_institution" content="INRAE, UMR CBGP, F-34988 Montferrier-sur-Lez Cedex, France" />
<meta name="citation_author_email" content="mathieu.gautier@inra.fr" />
<meta name="citation_author_email" content="nabila.yahiaoui@cirad.fr" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. H. Alexander;citation_author=J. Novembre;citation_author=K. Lange;citation_title=Fast model-based estimation of ancestry in unrelated individuals.;citation_volume=19;citation_year=2009;citation_pmid=19648217;citation_doi=10.1101/gr.094052.109" />
<meta name="citation_reference" content="Barrett, S. C. H., R. Arunkumar, and S. I. Wright, 2014 The demography and population genomics of evolutionary transitions to self-fertilization in plants. Philos. Trans. R. Soc. B Biol. Sci. 369: 20130344. 10.1098/rstb.2013.0344" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=Y. Brandvain;citation_author=T. Slotte;citation_author=K. M. Hazzouri;citation_author=S. I. Wright;citation_author=G. Coop;citation_title=Genomic Identification of Founding Haplotypes Reveals the History of the Selfing Species Capsella rubella.;citation_pages=e1003754;citation_volume=9;citation_year=2013;citation_pmid=24068948;citation_doi=10.1371/journal.pgen.1003754" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=S. R. Browning;citation_author=B. L. Browning;citation_title=Haplotype phasing: existing methods and new developments.;citation_pages=703-714;citation_volume=12;citation_year=2011;citation_issue=10;citation_pmid=21921926;citation_doi=10.1038/nrg3054" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=C. Burgarella;citation_author=A. Barnaud;citation_author=N. A. Kane;citation_author=F. Jankowski;citation_author=N. Scarcelli;citation_title=Adaptive Introgression: An Untapped Evolutionary Mechanism for Crop Adaptation.;citation_pages=4;citation_volume=10;citation_year=2019;citation_doi=10.3389/fpls.2019.00004" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=L.-Y. Chen;citation_author=R. VanBuren;citation_author=M. Paris;citation_author=H. Zhou;citation_author=X. Zhang;citation_title=The bracteatus pineapple genome and domestication of clonally propagated crops.;citation_pages=1549-1558;citation_volume=51;citation_year=2019;citation_doi=10.1038/s41588-019-0506-8" />
<meta name="citation_reference" content="citation_journal_title=Biodivers. Conserv.;citation_author=P. Christelová;citation_author=E. De Langhe;citation_author=E. Hřibová;citation_author=J. Čížková;citation_author=J. Sardos;citation_title=Molecular and cytological characterization of the global Musa germplasm collection provides insights into the treasure of banana diversity.;citation_pages=801-824;citation_volume=26;citation_year=2017;citation_doi=10.1007/s10531-016-1273-9." />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=R. Corbett-Detig;citation_author=R. Nielsen;citation_title=A Hidden Markov Model Approach for Simultaneously Estimating Local Ancestry and Admixture Time Using Next Generation Sequence Data in Samples of Arbitrary Ploidy.;citation_pages=e1006529;citation_volume=13;citation_year=2017;citation_doi=10.1371/journal.pgen.1006529" />
<meta name="citation_reference" content="Cornejo, O. E., M.-C. Yee, V. Dominguez, M. Andrews, A. Sockell et al., 2018 Population genomic analyses of the chocolate tree, Theobroma cacao L., provide insights into its domestication process. Commun. Biol. 1: 167." />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. Cornille;citation_author=P. Gladieux;citation_author=M. J. M. Smulders;citation_author=I. Roldán-Ruiz;citation_author=F. Laurens;citation_title=New insight into the history of domesticated apple: secondary contribution of the European wild apple to the genome of cultivated varieties.;citation_pages=e1002703-e1002703;citation_volume=8;citation_year=2012;citation_issue=5;citation_pmid=22589740;citation_doi=10.1371/journal.pgen.1002703" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=F. Curk;citation_author=G. Ancillo;citation_author=F. Ollitrault;citation_author=X. Perrier;citation_author=J.-P. Jacquemoud-Collet;citation_title=Nuclear Species-Diagnostic SNP Markers Mined from 454 Amplicon Sequencing Reveal Admixture Genomic Structure of Modern Citrus Varieties.;citation_pages=e0125628;citation_volume=10;citation_year=2015;citation_doi=10.1371/journal.pone.0125628" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot.;citation_author=F. Curk;citation_author=F. Ollitrault;citation_author=A. Garcia-Lor;citation_author=F. Luro;citation_author=L. Navarro;citation_title=Phylogenetic origin of limes and lemons revealed by cytoplasmic and nuclear markers.;citation_volume=117;citation_year=2016;citation_pmid=26944784;citation_doi=10.1093/aob/mcw005" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=T. Dias-Alves;citation_author=J. Mairal;citation_author=M. G. B. Blum;citation_title=LOTER: A SOFTWARE PACKAGE TO INFER LOCAL ANCESTRY FOR A WIDE RANGE OF SPECIES.;citation_pages=2318-2326;citation_volume=35;citation_year=2018;citation_doi=10.1093/molbev/msy126" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=M. El Baidouri;citation_author=F. Murat;citation_author=M. Veyssiere;citation_author=M. Molinier;citation_author=R. Flores;citation_title=Reconciling the evolutionary origin of bread wheat (Triticum aestivum).;citation_volume=213;citation_year=2017;citation_doi=10.1111/nph.14113" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Falush;citation_author=M. Stephens;citation_author=J. K. Pritchard;citation_title=Inference of Population Structure Using Multilocus Genotype Data: Linked Loci and Correlated Allele Frequencies;citation_pages=1567-1587;citation_volume=164;citation_year=2003;citation_issue=4;citation_pmid=12930761;citation_doi=PubMed PMID: 88" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. Frichot;citation_author=F. Mathieu;citation_author=T. Trouillon;citation_author=G. Bouchard;citation_author=O. François;citation_title=Fast and Efficient Estimation of Individual Ancestry Coefficients.;citation_volume=196;citation_year=2014;citation_pmid=24496008;citation_doi=10.1534/genetics.113.160572" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=E. Geza;citation_author=J. Mugo;citation_author=N. J. Mulder;citation_author=A. Wonkam;citation_author=E. R. Chimusa;citation_title=A comprehensive survey of models for dissecting local ancestry deconvolution in human genome.;citation_pages=1709-1724;citation_volume=20;citation_year=2019;citation_doi=10.1093/bib/bby044" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Guan;citation_title=Detecting structure of haplotypes and local ancestry.;citation_volume=196;citation_year=2014;citation_pmid=24388880;citation_doi=10.1534/genetics.113.160697" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. Hellenthal;citation_author=G. B. J. Busby;citation_author=G. Band;citation_author=J. F. Wilson;citation_author=C. Capelli;citation_title=A Genetic Atlas of Human Admixture History;citation_pages=747-751;citation_volume=343;citation_year=2014;citation_issue=6172;citation_pmid=24531965;citation_doi=10.1126/science.1243518;citation_georef=2014024187" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=D. Hui;citation_author=Z. Fang;citation_author=J. Lin;citation_author=Q. Duan;citation_author=Y. Li;citation_title=LAIT: a local ancestry inference toolkit.;citation_pages=83;citation_volume=18;citation_year=2017;citation_doi=10.1186/s12863-017-0546-y" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=D. J. Lawson;citation_author=G. Hellenthal;citation_author=S. Myers;citation_author=D. Falush;citation_title=Inference of population structure using dense haplotype data.;citation_pages=e1002453-e1002453;citation_volume=8;citation_year=2012;citation_issue=1;citation_pmid=22291602;citation_doi=10.1371/journal.pgen.1002453" />
<meta name="citation_reference" content="citation_journal_title=Hum. Genomics;citation_author=Y. Liu;citation_author=T. Nyunoya;citation_author=S. Leng;citation_author=S. A. Belinsky;citation_author=Y. Tesfaigzi;citation_title=Softwares and methods for estimating genetic ancestry in human populations.;citation_pages=1;citation_volume=7;citation_year=2013;citation_pmid=23289408;citation_doi=10.1186/1479-7364-7-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P.-R. Loh;citation_author=M. Lipson;citation_author=N. Patterson;citation_author=P. Moorjani;citation_author=J. K. Pickrell;citation_title=Inferring Admixture Histories of Human Populations Using Linkage Disequilibrium.;citation_volume=193;citation_year=2013;citation_pmid=23410830;citation_doi=10.1534/genetics.112.147330" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=B. K. Maples;citation_author=S. Gravel;citation_author=E. E. Kenny;citation_author=C. D. Bustamante;citation_title=RFMix: A Discriminative Modeling Approach for Rapid and Robust Local-Ancestry Inference.;citation_pages=278-288;citation_volume=93;citation_year=2013;citation_pmid=23910464;citation_doi=10.1016/j.ajhg.2013.06.020" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=A. J. Miller;citation_author=B. L. Gross;citation_title=From forest to field: Perennial fruit crop domestication.;citation_volume=98;citation_year=2011;citation_pmid=21865506;citation_doi=10.3732/ajb.1000522" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=S. G. Milner;citation_author=M. Jost;citation_author=S. Taketa;citation_author=E. R. Mazón;citation_author=A. Himmelbach;citation_title=Genebank genomics highlights the diversity of a global barley collection.;citation_volume=51;citation_year=2019;citation_doi=10.1038/s41588-018-0266-x" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=A. Moreno-Estrada;citation_author=S. Gravel;citation_author=F. Zakharia;citation_author=J. L. McCauley;citation_author=J. K. Byrnes;citation_title=Reconstructing the Population Genetic History of the Caribbean.;citation_pages=e1003925;citation_volume=9;citation_year=2013;citation_pmid=24244192;citation_doi=10.1371/journal.pgen.1003925" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=B. Padhukasahasram;citation_title=Inferring ancestry from population genomic data and its applications.;citation_pages=204;citation_volume=5;citation_year=2014;citation_pmid=25071832;citation_doi=10.3389/fgene.2014.00204" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=B. Paşaniuc;citation_author=S. Sankararaman;citation_author=G. Kimmel;citation_author=E. Halperin;citation_title=Inference of locus-specific ancestry in closely related populations.;citation_volume=25;citation_year=2009;citation_pmid=19477991;citation_doi=10.1093/bioinformatics/btp197" />
<meta name="citation_reference" content="citation_journal_title=Ethnobot. Res. Appl.;citation_author=X. Perrier;citation_author=F. Bakry;citation_author=F. Carreel;citation_author=C. Jenny;citation_author=J.-P. Horry;citation_title=Combining Biological Approaches to Shed Light on the Evolution of Edible Bananas.;citation_pages=199;citation_volume=7;citation_year=2009;citation_doi=10.17348/era.7.0.199-216" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=X. Perrier;citation_author=E. De Langhe;citation_author=M. Donohue;citation_author=C. Lentfer;citation_author=L. Vrydaghs;citation_title=Multidisciplinary perspectives on banana (Musa spp.) domestication.;citation_volume=108;citation_year=2011;citation_pmid=21730145;citation_doi=10.1073/pnas.1102001108" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of Population Structure Using Multilocus Genotype Data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835412;citation_doi=10.1111/j.1471-8286.2005.01155.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=S. J. Puechmaille;citation_title=The program structure does not reliably recover the correct population structure when sampling is uneven: subsampling and new estimators alleviate the problem.;citation_pages=608-627;citation_volume=16;citation_year=2016;citation_doi=10.1111/1755-0998.12512" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Purcell;citation_author=B. Neale;citation_author=K. Todd-Brown;citation_author=L. Thomas;citation_author=M. A. R. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Salter-Townshend;citation_author=S. Myers;citation_title=Fine-Scale Inference of Ancestry Segments Without Prior Knowledge of Admixing Groups.;citation_volume=212;citation_year=2019;citation_pmid=31123038;citation_doi=10.1534/genetics.119.302139" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Sankararaman;citation_author=S. Sridhar;citation_author=G. Kimmel;citation_author=E. Halperin;citation_title=Estimating local ancestry in admixed populations.;citation_pages=290-303;citation_volume=82;citation_year=2008;citation_issue=2;citation_pmid=18252211;citation_doi=10.1016/j.ajhg.2007.09.022" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot.;citation_author=J. Sardos;citation_author=X. Perrier;citation_author=J. Doležel;citation_author=E. Hřibová;citation_author=P. Christelová;citation_title=DArT whole genome profiling provides insights on the evolution and taxonomy of edible Banana (Musa spp.).;citation_volume=118;citation_year=2016;citation_pmid=27590334;citation_doi=10.1093/aob/mcw170" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=P. Scheet;citation_author=M. Stephens;citation_title=A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase.;citation_pages=629-644;citation_volume=78;citation_year=2006;citation_issue=4;citation_pmid=16532393;citation_doi=10.1086/502802" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Semon;citation_author=R. Nielsen;citation_author=M. P. Jones;citation_author=S. R. McCouch;citation_title=The Population Structure of African Cultivated Rice Oryza glaberrima (Steud.): Evidence for Elevated Levels of Linkage Disequilibrium Caused by Admixture with O. sativa and Ecological Adaptation.;citation_volume=169;citation_year=2005;citation_pmid=15545652;citation_doi=10.1534/genetics.104.033175" />
<meta name="citation_reference" content="citation_title=The Evolution of the Bananas;citation_year=1962" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. R. Staab;citation_author=D. Metzler;citation_title=Coala: an R framework for coalescent simulation.;citation_volume=32;citation_year=2016;citation_pmid=27153679;citation_doi=10.1093/bioinformatics/btw098" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. R. Staab;citation_author=S. Zhu;citation_author=D. Metzler;citation_author=G. Lunter;citation_title=scrm: efficiently simulating long sequences using the approximated coalescent with recombination.;citation_volume=31;citation_year=2015;citation_pmid=25596205;citation_doi=10.1093/bioinformatics/btu861" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=M. Stift;citation_author=F. Kolář;citation_author=P. G. Meirmans;citation_title=Structure is more robust than other clustering methods in simulated mixed-ploidy populations.;citation_pages=429-441;citation_volume=123;citation_year=2019;citation_doi=10.1038/s41437-019-0247-6" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=H. Tang;citation_author=M. Coram;citation_author=P. Wang;citation_author=X. Zhu;citation_author=N. Risch;citation_title=Reconstructing genetic ancestry blocks in admixed individuals.;citation_pages=1-12;citation_volume=79;citation_year=2006;citation_issue=1;citation_pmid=16773560;citation_doi=10.1086/504302" />
<meta name="citation_reference" content="The 3,000 rice genomes project, 2014 The 3,000 rice genomes project. GigaScience 3: 7." />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=A haplotype map of the human genome.;citation_pages=1299-1320;citation_volume=437;citation_year=2005;citation_issue=7063;citation_pmid=16255080;citation_doi=10.1038/nature04226" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls.;citation_pages=661-678;citation_volume=447;citation_year=2007;citation_issue=7145;citation_pmid=17554300;citation_doi=10.1038/nature05911" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=B. S. Weir;citation_author=C. C. Cockerham;citation_title=Estimating F-Statistics for the Analysis of Population Structure.;citation_pages=1358;citation_volume=38;citation_year=1984;citation_pmid=28563791;citation_doi=10.2307/2408641" />
<meta name="citation_reference" content="citation_title=PLoS One;citation_author=E.-M. Willing;citation_author=C. Dreyer;citation_author=C. van Oosterhout;citation_title=Estimates of genetic differentiation measured by FST do not necessarily require large sample sizes when using many SNP Markers.;citation_pages=e42649;citation_volume=7;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=G. A. Wu;citation_author=S. Prochnik;citation_author=J. Jenkins;citation_author=J. Salse;citation_author=U. Hellsten;citation_title=Sequencing of diverse mandarin, pummelo and orange genomes reveals complex history of admixture during citrus domestication.;citation_pages=656-662;citation_volume=32;citation_year=2014;citation_pmid=24908277;citation_doi=10.1038/nbt.2906" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=G. A. Wu;citation_author=J. Terol;citation_author=V. Ibanez;citation_author=A. López-García;citation_author=E. Pérez-Román;citation_title=Genomics of the origin and evolution of Citrus.;citation_pages=311-316;citation_volume=554;citation_year=2018;citation_doi=10.1038/nature25447" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=K. Zhao;citation_author=M. Wright;citation_author=J. Kimball;citation_author=G. Eizenga;citation_author=A. McClung;citation_title=Genomic diversity and introgression in O. sativa reveal the impact of domestication and breeding on the rice genome.;citation_pages=e10780-e10780;citation_volume=5;citation_year=2010;citation_issue=5;citation_pmid=20520727;citation_doi=10.1371/journal.pone.0010780" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=Q. Zhou;citation_author=L. Zhao;citation_author=Y. Guan;citation_title=Strong Selection at MHC in Mexicans since Admixture.;citation_pages=e1005983;citation_volume=12;citation_year=2016;citation_doi=10.1371/journal.pgen.1005847" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Simulation-Based Evaluation of Three Methods for Local Ancestry Deconvolution of Non-model Crop Species Genomes" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
<meta name="twitter:description" content="Hybridizations between species and subspecies represented major steps in the history of many crop species. Such events generally lead to genomes with mosaic patterns of chromosomal segments of various origins that may be assessed by local ancestry inference methods. However, these methods have mainly been developed in the context of human population genetics with implicit assumptions that may not always fit plant models. The purpose of this study was to evaluate the suitability of three state-of-the-art inference methods (SABER, ELAI and WINPOP) for local ancestry inference under scenarios that can be encountered in plant species. For this, we developed an R package to simulate genotyping data under such scenarios. The tested inference methods performed similarly well as far as representatives of source populations were available. As expected, the higher the level of differentiation between ancestral source populations and the lower the number of generations since admixture, the more accurate were the results. Interestingly, the accuracy of the methods was only marginally affected by i) the number of ancestries (up to six tested); ii) the sample design ( i.e. , unbalanced representation of source populations); and iii) the reproduction mode ( e.g. , selfing, vegetative propagation). If a source population was not represented in the data set, no bias was observed in inference accuracy for regions originating from represented sources and regions from the missing source were assigned differently depending on the methods. Overall, the selected ancestry inference methods may be used for crop plant analysis if all ancestral sources are known." />
<meta name="og-title" property="og:title" content="Simulation-Based Evaluation of Three Methods for Local Ancestry Deconvolution of Non-model Crop Species Genomes" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/2/569" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Hybridizations between species and subspecies represented major steps in the history of many crop species. Such events generally lead to genomes with mosaic patterns of chromosomal segments of various origins that may be assessed by local ancestry inference methods. However, these methods have mainly been developed in the context of human population genetics with implicit assumptions that may not always fit plant models. The purpose of this study was to evaluate the suitability of three state-of-the-art inference methods (SABER, ELAI and WINPOP) for local ancestry inference under scenarios that can be encountered in plant species. For this, we developed an R package to simulate genotyping data under such scenarios. The tested inference methods performed similarly well as far as representatives of source populations were available. As expected, the higher the level of differentiation between ancestral source populations and the lower the number of generations since admixture, the more accurate were the results. Interestingly, the accuracy of the methods was only marginally affected by i) the number of ancestries (up to six tested); ii) the sample design ( i.e. , unbalanced representation of source populations); and iii) the reproduction mode ( e.g. , selfing, vegetative propagation). If a source population was not represented in the data set, no bias was observed in inference accuracy for regions originating from represented sources and regions from the missing source were assigned differently depending on the methods. Overall, the selected ancestry inference methods may be used for crop plant analysis if all ancestral sources are known." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
    <title>Simulation-Based Evaluation of Three Methods for Local Ancestry Deconvolution of Non-model Crop Species Genomes | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__49R9wsCpiKJS6e6ZOiXRsC0989bO7FF1XdNTikSRKGM__k1m2vpL1Nu9n43bJnV24EQ0Z_UWo-Iq0zZ2FkQLL-iU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"uouCffk7NtrRPv2efhPWXuU2VVvtz7MBUzApFtJlU7Y","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495549","apath":"\/ggg\/10\/2\/569.atom","pisa":"ggg;10\/2\/569","ac":{"\/ggg\/10\/2\/569.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/2\/569.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/2\/569"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/2\/569"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/2\/569.figures-only":true,"\/content\/10\/2\/569":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022FC077qgPEvbRms4y3BdR\\\\\\\/BI+eGTTKS2LsmlW2qiEtGnD6SKie8bT9wenIOt02lzTMtz0OukANvw\\\\\\\/DZBwIB+OGUMyU3dwRGg2CpJe4BFTDYm7V2tpnP6e+lq0Tj6F5Uk+ye3j6vgwlSrlaO\\\\\\\/RDJeFb4fOQnO3plfFE9b02qRlyLFlMddBKPAK3q\\\\\\\/TwltdqS7seCxFuc\\\\\\\/nhf32XVA7pl421YPrwU06RtzKCdz2aBDIvSCFinTevXflnea7Q12kBoFHgpeOAP4Wsm+6YFiFEk0ePvu\\\\\\\/EmjCm7RMBZy\\\\\\\/iLGWFBbJm3M\\\\\\\/YOfk5t8tfa2Z1k2718QLINmLaRlCt2TcQbjRC7HgSI3RlACttsZXsaOTSlPATOPlnKEdpPAbu42bt1TGCZjH8gNjwazYNImGyPbyA6hPusfMcdmj5PzzAx8I2+TCeoDBdp+TAuHgPBq6YC5KdhFSNQv1KIAUvenkECOA06b48\\\\\\\/Iv2TLH0wRTeUFlgDKXPERh08d7KhhatTAgprpaawXVHl\\\\\\\/GH5b2ZT9uXk11LrsUxayPHfDSJ+T3NmewvNbCq5PippKtk0SBInnQ41FNv2lfPCsFmdne+NR4MDY3yS35nqr9U9WWVfIgEq8biOl59uzyfjfK\\\\\\\/zJTDkHHP\\\\\\\/jFpdFyrXlbhRvU63wv5Ue\\\\\\\/IpCbI7qtRNSu+qqHGzdwxTi4HmxlW52Pbi00OqaDnKXPtsC93cMLCncbaTMaH\\\\\\\/jIaeSMNmlKnm\\\\\\\/Uzf5RNt4Dn1WPz1Kdv\\\\\\\/ZAfBhqISFjJMvsJSTLaR0yU+lG6JRjAI9HKHUdYfVknQermQu8LZyV0rH3it7OggoNKlP+3\\\\\\\/1C8hghHd7Yk3GUz5aw5P+vslH4vqyRhBS9fdCrhWd0y88zEkdDOV7NOZIbR9NacDGFZAdUbwTs4QYrbn1p+wnpoSQr8UJxYoKcMFyVGH\\\\\\\/wTX\\\\\\\/f5qd9nxYKxhvvq7ejjXMazZfr3D1i0X4K04l8RER28RnR1Owzysv8gfA9KsGKQecN7B0B+AzTPvV4YcaICYsjsc58iT5TM1aGHLilQx9ayZPC8aSKuIAiUxRSRfTZvMPdZuk0LiWT1Zpp\\\\\\\/xxYtSfjrn7g9zK14kXFFKaLWvLVhXsJNHwC1z+E9SP413v6BlM0z2BupaUNshkXeJpiosRpiaQhPN8A3gBRvz9hdpmP2OR\\\\\\\/VBYdokwR\\\\\\\/urZdP1GtBjaz7Q6Sq\\\\\\\/Xl6L01u5K+Ib+m8xrSqAUnK+MvD02G5wti8yLMSrDnujBKU5g8CwCaDdcqwXObR18MiThdlDeW9uCd9BL\\\u0022,\\\u0022iv\\\u0022:\\\u0022kH45+MmBXtUNTLehdovibw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e5cd249f34d8a45363698b155a97829fe358129358bf011b266d393f900c4684\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DMTqHHLPcYL4+PGI0ZgCN0OD9Kccj2PCn8P\\\\\\\/2ZigD1NK0Jne6Ur165tc9\\\\\\\/xVG9RO7Uax+00s40wsmtLfKEtL6QcM2JzoGiNeNg2hX6OAHrYiIQVyVvkxmZNqhrN\\\\\\\/\\\\\\\/a3r492V6pWOeZMSoT2qJpaww9o42Vv5jz9h48K9rXC6lL+SA5xAkTc35yi4meT1Kav2mqZhqW98sbyN66L3dcUWgEsNsKQoZI6QqMtWkxcSi5PAFAmeC1gdG\\\\\\\/EQfn35DR+cjofNsyi1J0n713q5GLocOyy97K8tBkbrt5uRnaWfWGgQqvqVSVmHgbiAB69KGqIberJqeDSeYRanI2zGzOPKEt\\\\\\\/uF3lxllB9oTJW9HPMpa8VOKuHsHruFeB8WOqKSMIYVTLVRXpW9ch+lugvdP69blq8KDPQw4Sblr2t4R8pHuJ+t8p0plukMyZTYfJs6YTKcz6j151vUYYwxzS2gX0BP+sczeOzhT0hHlfok2gn6vPHhC+qqCTj4GNQph9yszkZT75sGBxsWy8lHdhOLWe+Vm72uPEbkf1FxYGuyUiRDI2U1M+HREXmwLcwL4xsHEBlhFJK+oPn87EhTr6lwe+exqBzkT+a2DZvXJlPQFeaJx8hNGU3r9zFD80BhRGhxPrMHEP5SayvsI1mMNKshh5X4Yf3wLKsEr\\\\\\\/9Lwmtq+gzUdujilY4bxoTpIskdkunIJGAOrfo\\\\\\\/3dwQakH71RK6YY0YyMvsncIQHw1CM+o56g7FOvvdDZDLXrEmNT2bi6z3QkAtWSjSyfpb4uyBpdVemBZMZwjbXzirroaBIgx3hUa+kcj12b2xc3pFfa1OgZ8eplH+tt8u6YloauXA+DzZWxiLQkNxTvVfnhBH98fzn3714YLRMbpjIcjWZWd4Z8e4Hf6gS3FafbnujXuPy3vld9SomDtTWB9xnA\\\\\\\/ydtLQfrGTkJYVPsqmpXVQemF\\\\\\\/gVJP4\\\\\\\/KEa5C75ygbDoloI6NKv1PN4V60s1Zecx5JFzANoE\\\\\\\/L8scN9BVYbTljewnHs6AvsfUuv8OXdNwwVvxYPOyepR2dbTm8VdMZSvLMZom207sDuqstdLnSJx9VwAEuL1ddHu7g7rVBiHBNuWdRbIEp3Kfd9T\\\\\\\/pYsFIqUNtLYWaGo3dkcdrOxN3fyeVfykUrztyGFEzunMIyRu6z\\\\\\\/E9o9fL+jdriZVrXWTfffaMeg7NTVhU\\\\\\\/UgxcwJEfwcoFtXO3Rf3kf8\\\\\\\/zxHGxQP48Fe4qIHgRypBd7Y3sfsZ8nxn3Yyt7mU\\\\\\\/MFOqDXrG1hPiJ9Hb9peAkUr2FgvucjRDXcPN7Kd0zkzEykSVVsCtyT1g6AxlZfWS+bVrsIHNc2oBTdrl049u7Y9SzKS2ytqXFr2q2rKLKAx3E9E0u5QmA97+UabWu4V0XkFDLEpJlowgkXIs8v3D5kVH3xqvs6VhixCYUM6IPeb\\\\\\\/SkpbPPpvq09FlwA6az9FSePBqgYiL63SvK3GomvMTiMD9RnoNFSTOTYudM64W37nGP4N4KSmFB2X9TXHtqG+VGD966TA7HA1bs9Wzq6gD1A8SjDkN91mszgUND2zQZWhzKrOlsJhlfBg9azZ\\\\\\\/Kiy0MKReJgWiIs5DkY+t3Xc0LCGTfvYug3HNA9f4QDIdju+IwHbNiQkPpdwQ41RX94+0pixb9TNS27Ez51rKiigd9jDtH4m6tTBVtdmHi28TWg1+cn046rMcoOAMcnIT8NPnqYCPRuPgNBA9II4OAvrzRzzjvzpphhTZWssyowYL3GgdD6iQ417l5XLFpIi658Q+5xZq63mXX0EyD+ug2lJSoJGhnH\\\\\\\/vTcTmtL+APWQxMFXpruAFDHNiI1JJ7GkiCuDgIzV5ppsfi8kemffl0PijcuZmj3kLoHCD6RQrXaCXW4VglqBJM5RmbRXbdMEr0DTOcFFuEPiSvzalZFM95UD0Zb\\\\\\\/dbeXNNCSNwwtPazfINNn6TatB5VP81C0KHaP1nbLs9HAe2u6wpKEovrZMTa7vHdvWwgoHiQT3eu4cnxtS7T\\\\\\\/k6oZUJHvMAXbc\\\\\\\/My\\\\\\\/yVhFbslWCB21f4lCfqLgsWkUUO1HYisMA9wB8CGiwhEkZTDUbBQ9qvq2R3q0h\\\\\\\/AUZrJ5bPbBUE6egtv+Sz57xzhzm9DXDVrhC7Indm2C5vZ+JWlgnmqLW2f++ErxtL\\\\\\\/lJXHI30Ba7G1QyzXF6A92w+15qh1+1jZNUoX6NDZp6JFYfSl1uqeYyocze1fgtg2rmkrtu79igBW63DewXtujBMKnon6CDwyqfKucvcktKJdAOHc+bxVmd3fabTdCzL8SA5Mtfkr5aP6vWg8EOTfwSYLHvm1b5TRhEKE+II+ExqJnSwYBS1APP2VbpYhjW4WhHE81dvkw9pffx94awnAI+dWedb3LLeT1euhVuKMEGKxsdEN76nKkyGzHQOibMEVls2MWM46kd22i8X+9DgV0SS3nAlnRe6Atc34UHXrr\\\\\\\/5QnFXMRJnSeax5ebC9zqH4YiIQyH7MlgzTAYDfL5Ks4PxgjpNrh3VNHKYeJIOIG9ga+2lMRUoRaLGJNH89FZ07pBBKNlkbwTQNe0EA2H8nVc3VwJvOZnysrsOeKGKSNcwlCLbr\\\\\\\/e91yZe4eI98Q7p2wiLN4GD4ImEfmdWaeWkR5iVs1Op2NNyBwsy5tj0Cbo0Cq\\\\\\\/eX4Vt8c6Xrfd6h1afSw+CtErWAUpewG2WazXURF7DptjYjJkM3bWq4ewi9rR1nj0bj1VxiMjzp8bNoB66OraUhQiZUWWRX32PSR+HxMvmCqjN\\\\\\\/GQ+VdcIGi112JDpETg553IkjFQ5Vp33F\\\\\\\/zzc70m1XuLo+OfEzLKYV4Ywtt\\\\\\\/PvJRRB2VNpsnljU4R2aQFvinaRquROVktFcTx+sChccmMoyeRsyftStMajgZNQRKtwb1s02nPvgms4DAy8k+MXxq1XcBimor0RyLSt87nJi4DjO0DDTE68VBlHzqYHtIQVo51FK9q6t0WUyIbSUHy3NGWB3yfXHMc0KTOeVxYXKKQfEwSKDfJp65cqiIdSTcthaC69Ffw9YomWiknWLDz3+2ByvJz\\\\\\\/cggl3H7iQhBHfHvJhdA0kwbqqVjIo\\\\\\\/JrOGmKJh++pDezb2+QuZ9sDOvlcidxGPOE9ocID6Ih1WkEbli7gguRM1hYe\\\\\\\/Z93PCU03GcBGrAi+LB18VyZ00SLC7Hkp+Ewcju7WXaAxjOtLZRqbmdLvoOXh\\\\\\\/CLLOtwhSS+LTAkMEUU8yyXy0o5cH81mxOZfj4Eptv2aokPZn65dM6Km4f4EAJSBXDsnhmfavhcAN+itjK9RBOPoVRJXLYoqxCmhAkHzVGufauCEW0aNLs8zKoGjKjw1A9\\\\\\\/ZPw9S5RfjpSF+YBV7LZtg9CjZFBeSD4OtUE0S00Hlb7EuNuTy+6Y8P4CYvraHRTQeWnKWukjYk919lhVarIbJ9kGNqVEkgYNFQrkh2k9AI8QAHa\\\\\\\/Pk1guUzqeqrR0UfpoWSOqBHinRl75qKgcoCNAlLczV6G73MZs+9L\\\\\\\/gQsTQaWMRioneOIdrPR3c5qQg4P0XxVNL+k9dJCoFQSYecU8xGeZ1Fp1T65AZckx8LDkZW5a6qlbk4CPb06nUw+SepVAn0Ux4uKYOeEPN+5H4VFZAK0eBBVaGYEqTn3Sd24alu0yjW8917dKD8EZTr5E\\\\\\\/M20FctLDJkFhFCIQEha69+Tz4IO0gqyaNlH7CnLzePhIlyaYoEvIbw0zmFLEmzNwzWc3kr6GrYiI\\\\\\\/HdTJMxWtIuDQOXvgb8xF4APHW5mUANqic6+VEitig8PZ7vnfBIapzjStD4PcV8DXvAfnJ1P6DBUKgS7u+4i1f8AEXsXsIao6jSggUDi0x\\\\\\\/\\\\\\\/kPQAR7jErQIi\\\\\\\/c26FmlcbufBI02o0PW5HOOWB3V\\\\\\\/JY\\\\\\\/K7tRT2C+zM6Qx4vIboA6Tteh7Lf0LXEu\\\\\\\/WWTbdQKU6PHFw36Dn1veCeA5Z6gmJ5FqZ0EHQB0eN7+CnnN8pNIRIeoB5wQ30gpHY+9JNfe\\\\\\\/DABr4bxVeKQZbPyvqkXKkzFB+Kccv1BfLja7\\\\\\\/Hbz4BGKg1EwlMnYZn7aVKkBs2fnjJICn79VE+hMFXT9lLkTLSUShg4rwwhmBTLXL7CbPX7dgR9zWBkEas7JxJgEPHLdQ2yunfE0\\\\\\\/2rN+ebHrNMr3nc2zhzV4uPdHjrUzXKTWmzn\\\\\\\/Swfd21Zlrh1uOSlASGWAFDzTKlMYvXU7TI1WDcveqcUP+Go0bMBYu9cwAsrGTEo1dLGxclAVdCAEcEV0HXQLOWSzCvBoN7Hn9o72\\\\\\\/kIvNnf3654Le9PtEC5CVAbird8CF+vBw3X4fQNZYQDTVJ897wOAU9EBnBSj6GYQO07J4JNSt9iDzSwzcGnNx3Wyp8kwaIK81eeUz4p46zIjJ\\\\\\\/9Peaks4cVEuNGDUvV8j4BMbFxI8sLtn7itx2zNxqOW9nZ84G65unT5i4MQHPBpdzQCNCPNlTUMTtlJM994OrYRIpeyVqZh1chUfKIx1qmRoCaSH21ClXRy78HTdoQ9sfHErsMMZgWLrGexI1IVXh71TUQWcqYfcMppwyfCyqII07Pzq+++HxDs89J56ypgN91U7wlKX5oqYB9poUU1Fk7hDCGVs94fNrFsuQs\\\\\\\/I0S3l7Ef8AmcEngZV5RZMvbdvtg3K9BLnnqecc8BaZeLUYPCwtcT45nLvmG+9hgRYgsOBw95+OlwoxdbOQQZH5gDL2doDVHVwl\\\\\\\/U1VWbZgpQZZY3EbFxWwYpoXTLQ8qF1YqeTq5iwUVVA4\\\\\\\/8guYa+z50zsJsOamATT\\\\\\\/BBtLhMNDuPRsYZs3Zn9o4URd73bNMvEZenLOH7ORuFM2Za4mlDHeAPYFaxWhaEqrRJjF1jQF9+A60EABFy64++2yqBT50Y8cKgDJZanHFuQPZ14HnD3VtBTyEssEZTmIbuPqgRX3\\\\\\\/84fL2e2bxEUSzitHu11vPNrClYFUUldgB7sH\\\\\\\/m+msOl5ze\\\\\\\/SVNSMWlUaQuVrb+EcY2aPbBojPOa5eiMxdwrWcRi0yF+epnrw01oanWwK8ZoyjypNB5bva8lBuLqxpEBRJnH4tqNNWA1S3Ie8+CqWfGtunbkoJnV9oUUeio0POHfVJu2zPgpV63JHklbEJoU1zZoqKWC++Ope4t5ddS8L2jmgyMW+P9Yc8ccSRRarsyPR4lJeG3e6uZMDcH2voNxOe+55e6DllBZejVikey74zZATNv8DG0ZUR7Lxn\\\\\\\/mOawQWKXYbNwJkaXQGDEJVydJGnXiqlXoNcBPSgW\\\\\\\/6i5TOyX0DEZsgiR\\\\\\\/GOAFlEvIUYfD0M2nufJWjBahRH4QVa6chQYErdJGz1RH9ssY+G1w23AwwQaQlXwvRq+WkQgvDrQ6YaD6zmJZQxuIqHULiZgLww5k6pE7QFr+TUoudRIq3jHaw1kqic3Bf\\\\\\\/D72WDH8DAradipKWoy2RuIMjEaAFJ34pIRVaiz\\\\\\\/PL2xtSMayfzEwE6rYj5ydz7OWwsGq497+Vx3JhwJ64cgy\\\\\\\/RuC4X4APMXUX7sSn0CrnrstGFs6Kx+2SJuGK3EXD1AJxlRB1RbcVVfgqrh76\\\\\\\/cfkiQrPcN9AzTrWH7JKZWC9EF3a0FlrKApSXY4nTalN7ByRzHk\\\\\\\/RtlfTupxyxot8YSRD+\\\\\\\/J6G6WQeM+pzy1dGV7QN7LYdHzxyjdJikVW9wk52Vjg0YK44Iee6J3o787tCw8hZbEaYm7KN+XHzAI+2MVf5Aj6cOod4S6r5ek3GXfeYuHYwhS45i19hIUI4Z5p+a9GDelve7AI8oLtCF1mii0RIzcew6VEE8cRZphO9GZ\\\\\\\/ESXKv0Uz3gJ85WCczydhzadi\\\\\\\/np\\\\\\\/VqBZNRVne6VAaye90GpeGpr5C5mjUHn87aHcSL7v8Sh8p4l77HbMudfwzdTOgYhfCgmWUW2yL80aJnVrTysAGyOwpX7EdJDK3+t05H3a\\\\\\\/6uxYlzZzqhOaqwgl1VSb96si8e+VP8CjF5j4eZ3jfbcg8s9+1oTUaULO+h10EC6QLnfDrGK3jGd5Lu\\\\\\\/ZzOpRrusoHNTASdTxMTG17eRH0Le\\\\\\\/GLNudpcE3ZY5pw7ae3DhjK9YAhWSMht3fPGpYhZ1V+AfnNwk6tLp+J0mUXNIc0mP\\\\\\\/9jorE8XqeKiPVZe61xtCXZdo73h8vrVtWt\\\\\\\/xjMj3SbC0pG+byLEdVTPgAFLAKcpPszO3Rb4wPNgpepVsIJ+J5qfSfqiCrwGmtAl\\\\\\\/nSTm6m09rWhjguPpuRpSZBShYUqNZO4elDy9pvUiWf0vYlskDBFvPivpn3tZYbx7mVUT1K\\\\\\\/63+zzQlRg6uKY6TqM2P5rPcOitwbYkmCDhEdItQ+QH7\\\\\\\/R2ZLqf8wNkfa6DMDlop0tCz7jkZDbA8ZBYIszkgoI70WCqpx6jQgZ1XekUBZW5YQ6pVAUUOMSYUxEfEhrr1pQ0yvpQaXzJnSGMe9RCKcF\\\\\\\/64s7Ma9+yM7fLT8tR6z4dsIGsOtdRKgOnWvgUQtFlaaa01\\\\\\\/qxQ43b8wRSrv\\\\\\\/HvP\\\\\\\/2qIXL0ZMgZghCJP5dBOWxA8pdJSU3iR5A1E\\\\\\\/0GzTjBkQ4n9GvDUvCVUum2cNqrzSW9bIb8rHTz3XGIG80YKpMLV8lPyFfqB6yYfmSfX6SCeyIwVDLiJenhcnVrLAdjdoC+lGcF6XKotr0UWFd9AmXRcYOPmwxQL3KYc67590FV4kw+AQmnZGQwNX40gaRgw2Svq9plyxmYJqUsaXfyL5\\\\\\\/25ceLnDHlAr2ekK\\\\\\\/i8G89YN4UjSYE6RdmMRD7hn5KqdEjkRdY1Me3dH5o4ar4C+yiwDs2Dkh\\\\\\\/LeRZmTFVvXpCMPROu3v2CBGNR1RclV5MVLYijfmgvZvVC\\\\\\\/XPekpnUq4uSkd1an4qGug9BG14k17ytsgmSNOgMyvvxvFkYrOPI2D5s2hEm64NYMQuJYqUyOG0IgAFjS2USyBTTvGV8lknty1fiMf0qD58iH0FXOYvXkruCrsbJt5N3R8w26t9\\\\\\\/4Cbk9LeWPS38lEKMR5\\\\\\\/EGSWJTqF20wPRoHdrdrgIZ\\\\\\\/yaMspY\\\\\\\/xLHcRAY9V6QLkYImlzZ2D7ejfa+gl22byBe\\\\\\\/WKmUVcGJvXIzpsB4c4urGS\\\\\\\/OhFYxdmyf\\\\\\\/1VLV9MmT4gEskCA3lzEKY4M22WRVGtInkNXw+4pQQrT1LIL8psXSIieaEleV3pdBYaPDM82Nt\\\\\\\/fubukGaYcYtauhYRgtrxLIWgad4eKYpwRfvTLLgmODzENfYl9jcMjw6NkB\\\\\\\/ktFxmNcSlPLqWjcpqdrdEEwvK5QMA834lLuDT5dYKMuBsa9vwlZan8aNNDrMLEr2+KLcFY399Hg1K+LUkmvySEYajnvkvCVwKey8HRacz20zr3nhzHEH4w3BSMTebOvvixZa5TWv6KCnAvVpIa37EncTYswGBrFV94lPkIQnIDf3Cqkv501XJjKAcBjU+bwdi29v0GFfhoRWxMhJUXHnFzsD+rhJ\\\\\\\/pH7J2aFxJg57Ch5nWdZyXJsdiArny0H2LKDK8nZjv5hXg2SXHCRgfypn7cpjS0+sJ\\\\\\\/heGXFWruCYmkEYR6Owk+ZxkVc5sio2LmExQC49preVtCpNFUy45yeIqi3yzodQjoGDt1R29\\\\\\\/5BL8c3F8tRF9QUzlRrZGO5U0oxR+6iGqjAIlJFcIuGulQ4gH4YD4gtqctzcXf8ytQSOHpSKF7+6HDMKF8brnQVGTQuJSf7kIlRa6H5m+LYOY6jcUiqxwnQ5+uOXP7\\\\\\\/xdEjmoUoIvvbJv40t+ojp9QbiFtw174qxRw9nFYslCLvMK9yuMx6wZbes5fh8mXD8q1oC\\\\\\\/SCrZNzSnhgUM5wIf1YPgcNvj4pntKLvsuH6CMXguz3iG7sqUlWyZqY9LIAynz7JLA9ftR7QcNGAhTYD28JyNVGAmyODZY+WbuEbRmJmcCt28V4l6p7dn\\\\\\\/oePAna7ws\\\\\\\/KsnxH2xWtwDkuffv3qZ62XirStfdNE9jE1MAcXj4RqYTLnKkq\\\\\\\/czH+zer5IX9u\\\\\\\/+2RGXG1FPieS42W8EBsA+edg0v0dukzAeUWH9Ax\\\\\\\/Z+bo3IYbv0bIgWrfMQOJkGRzS8y2sSDgGNzyIYZ74KlsYxwR9982f14mA3D74uz550vGhgkw3W2JaIg4J3G3CM9wmKWRlwN1l3eANkph6ZAO5D1ZaVvCNkhExyY3HvP6SANeIGqvKJIUEeQzb\\\\\\\/wPdMrs1uCJ3IkGTLJ8XcgxeyZ33vj+a4f10QAnBHHtNJ3e+bqKHiDcr2gbC1VevKiYZTPs4N9s8buTki0O3t+iG8wJs5jp0hTV+eyWms\\\\\\\/ajZb7QbtIrhmPdUCqAzfzOj4Ejw37zQH\\\\\\\/flo9Is2A\\\\\\\/pr6aaqcJJ3xJbtwDmo8rYEoLD\\\\\\\/IpXZHoQbBMtCYFHQC3RvTszXcyNM5u2yxrpRfxSBe+LbR6DTH5eW1Io9cdq+prNiYh6HQz\\\\\\\/hCqP17evE\\\\\\\/W2k46+MCqe8hQ6Cp041WlIaL\\\\\\\/gJJ70\\\\\\\/6ABGxcTdpODjnwjM\\\\\\\/bQ71adSdeRU+PA5iutJJfdE6HNVp1QEL8qgK2xhVk63LWt7MexA5MKnTXOhoIxbn8bs\\\\\\\/Mgxxa0k5s4bcHyPjsCF6bQI7terLLPb+Sc0oOqr4gRFakrfHh6wrHNEHQqGGmnIO\\\\\\\/59qaMTQ3jhDn0AyYFCKfPjuuVlGsBwN0qDNsrFpdxWyZ0FI2UZwODs+\\\\\\\/29vlDzypHeLMjXRJ7iPp03YK2ynv0Bx2H3YTknuEfJ8WSGAt+q\\\\\\\/Q5uvHe9XwminicXRusCh6Hx2mHvoaBYjPG++2SN9l8OihkaiVQqEU9BM3H1P+9ya3PWnvqJc9DVA+lxxSfuObCwMtKzZm9biXUD\\\\\\\/jhpOBai\\\\\\\/Yjgz4H5m16WPAgqULGk3Sn7J\\\\\\\/TQuwt7f4pvYPMZEoHvk2FukhniS3EorF+qd79B9ckfSGJnMnDB5p2b7\\\\\\\/Yn9g\\\\\\\/kir9fwN9++NVOgKEe7\\\\\\\/T97OsNxGamsVnOvPp3YFlxoT+vFkEU91KuV8LYVyKxo9jCQrvT9oCoWjP+HynmCBMXi5+8kmSTjHJmsQZ2CFjeBEmZwtrNGSUeFzN8YljPY4XcZQbCozOVfORiS3T7SmqdeffcOQXZSbijsNpZu0bwUfu0AOpGKQtkHgoh5QYDv5gBS9hnBVDOfbk9EMNAYOFiJqENzda\\\\\\\/nr29i8RkZE11YiwSLBfxPVtiM0faIkkiwzMR43Wo1DTVCd+Byc4B1PBbLskm5CFPDZP9nHX8Elr3B5+ZdxAJa2+VNCheSjX\\\\\\\/zUZraEIkuUONARNMkQKu8tFa3N7i7ZYewJZV6TOvGKUYcjKOEYAQRBPCHXSeha0Olx89RtFrNfG\\\\\\\/iN+dwkSMN5HsVb1uWwG8d2e8MxjJERefrJVoQNqEf1uPYztCdQ8G\\\\\\\/wB5z16G6mKoBxHbzU1n\\\\\\\/tcKV5W6DTXRqOhGQ3jT2Bt6WO6fXXS6jjSzirKU7gSczeniwkpArOLPTzxBtwHjlR7V3LmOTjm8iqJpqOK5lWDMPS+Smtw7qHObTuNtaKdqK\\\\\\\/5HGRnbCPs\\\\\\\/Y6l9hUw0E+w9h\\\\\\\/LxCxkQE37NIrpA60zNDnWenSeojG60gy\\\\\\\/fkyLbEO6fwI2uvarYjenTFJEzz1lLdgsGTOt1JbRxzld+kGF3ejvtWynx1jgsj8\\\\\\\/x7vkN59rApelqVCELwwuea22aehfBdjcUVTMheHTn+4pIiGWS\\\\\\\/MTABlLD9xrMZOoZD+C\\\\\\\/nMOUUsXpQ1BNhBtCQeV5RYi4JoV5zGv9WIofCZB02dbteW0PNO0oNyxJNiQiUYj\\\\\\\/mLms+DNNBbW1FhPPPNz0\\\\\\\/D18h3yBGqK+QTa0uxkyd\\\\\\\/O+rZgUMCgpClhF82WAhP0aHLrzbrLMJQ1iVMvnAXV588lUZYLZ4dm56CMIOKNPF7Y9f1fZWHIbIHRGSL8U3ckjbB72S+r81S51fizU9YRvbF6+FxAhq1nkjyeTF8Q7pm74yKO3YdLvyKOpILh2dEXiOPEGSSN38tgzYXL3p6IOxrdVX\\\\\\\/TGe7HscpYHkvMqm1phjf10eLEuIqh6LLAs9V5gFqLpRx\\\\\\\/wOQ7iC2uauSU8mr7Gjht8nv4XmKhVoDNvgHmov0JKFVbU9EcPB1lwPmxu7ITJ8EMt82dV0U2Ra\\\\\\\/7p0rZezgK+gTWHIbiJ8PZu7SgxOIzvhYy8uH+exxgiiUi132MfYpXAaAeW00WZspUx931vkAbEs+9Rxz4SSRTL6nCoQmS2nsT5TpPCDwxbmA9OUzE7oQO9RGYzMbFOdp+f1vzXSCACW7N7CN1dVuqI4qqF6fNtCnXiL0O9YH2Tv29camxecdenjQpl9\\\\\\\/jyAMk1Vz\\\\\\\/WnY\\\\\\\/icODG2BQv9aH\\\\\\\/1O5E\\\\\\\/XPIms\\\\\\\/0MdVbPdmV2dlD0rcT30+RI8arvbmuhVBp4\\\\\\\/9VuOlaclqbt8uad3fwITDgs1HeLRVXqrz6HoL\\\\\\\/ndAe+5hG3PJTfR17yTWJ874RA3ive0nRcgUhiOhJY0qWiNdY8dFK0Mr3VlsMcg+b0h+0vLBux3\\\\\\\/XPRJ3GUvqdYJomwb2Vs03dLo+wDOehAjFw4nIEA1WhC8DCx4Q5+YVUWeBO39UdY9lSLGps77xDE1NBTNLJqVjpzP0kReXg8iIa4Tq0K+W3HCx8hzyFK25XlaNkLUHhVB9HKJD2dBxHJ\\\\\\\/ijj6VIhY6LaNpuWEUl450FQtoCxLzNl5doUUgiZGuDCL\\\\\\\/spd4aYqQWDaeWG00mRqwZbOCZPOWgiwCl+8o1u4C3m\\\\\\\/qfrfBDMovupq10UDyS5e2P7c\\\\\\\/nbNhheBbRIHkSw8JWs\\\\\\\/HlrFoA58mhUQPQU8Qv1N+oKUdQVT7yf4JYZxlR5oIYvRC0U8G\\\\\\\/9ugvTW4SNUTLpEltwlJCFDn1tsAC7bhFOYNnBOkkQ+LS0qbyAFb8JuKrtak3BY0tVjy+vpp8QPDEYGdKVkxzPaifurXDaRyqfBQpt7qfBdaF2G2d2vIfv4g9JuTzJ8sBLVdOnyvNTUHqSA+sznPc9Ag0sgPdJOh\\\\\\\/OQT9VWIojAxthptbBANlfeJSI8V6HYfQ6h8ISgfhcnK1FeSA5ftf3X624wVon+crd4VHxH93LAq+zqkmmcxCCOFKh8Wat1OibgUok\\\\\\\/CYLpnxTYHMdX6K\\\\\\\/AMj+ZrBVcRr1nQCqEdD3I9gLxAa3\\\\\\\/Yig+38Zwlu94grxHaSaAGFSFK29mm0IG2i5LjP2e1mep36OhL+oEBP29NOkcYsRZRU\\\\\\\/Q3FSyAvV0VuZmfh2ORq\\\\\\\/en\\\\\\\/gPP7qWNl49T\\\\\\\/vwaijuOSRZs7WY0\\\\\\\/dMQNMqzXbBVWSPIGTDIOTnoURJjVOdng2Gwq2ILLhuTWdfjRdZvc+E8\\\\\\\/r7GUEB+Q9SPr7S7WPAPB6rmGnMUPOtbuTqKqHLNUFKIsV7kM4vC6vEYHmjrcnc7fZe7aaouDnxBRU7aN3S5taKkpioM1XiQ3XiCz7xslP3YPzMWHsN+YXOgDhrnAhF2XLUmXN0xMe8ez7tqTJKW7YVl1yrCQo0qBG5iMjM6DbvSIUOtkAHjzGXxueBcCXLIY3HY99Dle23sfuDy5F1\\\\\\\/DeaRYebYlPIM+RYU44wE3zoej9QWuzL1pbt59kbYnqe8o7oa8qd6PEHltGV9q+Pfj5Cfsf2d4D+RK\\\\\\\/B3LO5as8rgayOFqFMljKtMxErXfHAhtjUq0O+YOT3X+B1iyhpMvJQ8sI3siIyU1KtsK7x6ZL4gDOduauc3SDP40FLnWBSqQ3nVbTWt2PyaHkkjBmyazU7ymlpenRcvjnlBOZMJuW918von+V1SVlPGJbV1AsqzK3zBXSsiostcM2rXUtV\\\\\\\/Q1Fywy6HW306G6Z\\\\\\\/2Fp75MnkR5J1r1poqIFRz5dn5bDrpwOL12a59oNtrAXZvgh5v0UZY5X23oL1WSVJi9uTX5fKEvJ+\\\\\\\/6Q3K12qRvZaqN3An5qxmnpmQsX2dBlFmzjXnZI7lkOmMqcio3rMPb2JWdj0xpQFpqS76g108hRY1rTajlzxKMQgXLZDJ3JyjUnx+WmWpOBHWgz1JMJyEX0hstV476Ng69B3mpy\\\\\\\/3xroa+aNNO02IrsuwpagOVVMHVdHdvpwXNIOcfmdlB+9Rojo1Ag\\\\\\\/jsQPZx2jc5+Vc4kFuwRPeoAfm4TvmrrWugQ4emKDpixHw9kJkrIyrYcHIcqyZUCy8cvwGHfosYsSVVCPlL8acuEUtLqkoTUq5Ai1O90TX1dk9NZRL9Gcy6hempaemO\\\\\\\/UTtBi5bObsQl2YMeDp\\\\\\\/20Ci113\\\\\\\/iU435Z1eQ1p7c5CQVtbl5MZzIwlymmxHmPVojiufjJ3tml4DgC1cW+g9Y0jTtVCIx01mgXmCeQlSy++6\\\\\\\/CZt0zBEjcYvjaKH38I5oHEXHrWZ0fmt0PIIOcX\\\\\\\/5mSO\\\\\\\/G2HC+1NuSAT7CZkmyYfkH258IXzvUtlzlF57zd3M+9FTT7IHT8rIWAXvoIi0Xw7cAzhdoKt3hB5KBnR\\\\\\\/oNO2P\\\\\\\/E0XAjipgbIeTmpE0oFQiXCbZ4klMN4Tna7Zzoy4JNFIs8D88EhxBY\\\\\\\/MTqF7C5I0rE56lnXF2XBV3IVUlhSgu\\\\\\\/3fF4XPBHGWwYGq9AxNISNCoQ0Mwpb7TJKapFKBAVpLDAwPfE9WhGmjE6CkRIfKCFhomnoT30qOSbz1eigvTHVVurts95YuOvJHhbDySFC0RZpKHpiEyp9TqwuUqRWXjVt0M2aHm3U+qiqc9Z1C8Klixw+ER\\\\\\\/PvjE8q8oZupC7o5bK0rN9C7S1S2L8AIXQ4UqWwJnGQOrSMFejUXd89UCLQB93gVwn0y51jqmgOe71V1kfmy+3OG0x\\\\\\\/SPbihCQSrHAQfBNZ69mig0fDEKAqsU7fWNWCYFNbCBi7u9upTcTgdhd90CuMKyeletx03sdSXSKvjYxsWDutzlXdFVPRM4LKfFhW8NDern8+L+\\\\\\\/GTNYKdWlk03TDS87apgBER3Urj9R6ArdkR7gmih6ZL\\\\\\\/6zfTjkfHSr\\\\\\\/Ldn5rLQgzO1O9Uf\\\\\\\/IOQJnFfCeqJYxT4Ph1UfYWD5xCrNUf7O0+SGGtj0P3++Z1VlhxNIcQpV8Ibm6jsHvpw\\\\\\\/RLhKd9mPIUJRvvgdiuISY2hce7U7LOeo4b9Xn2xxSZHPEjEzIEzlDmrJX8no\\\\\\\/Xi2mavoEciUkG1EgaeAbg2IYmAJa2yY+LRjiYQpU2SEXXgSnuLb9UhuHX2q24HlgaPGKTLzlSSDJu3jm6CWUC6qs7HDnPhKzp0W4qqm5lmOw7\\\\\\\/RiWTtp\\\\\\\/O\\\\\\\/tSG2ii0NO\\\\\\\/tcRuJEES27uHhttzxepFWB1VazXnvtUMSaizzqD1CPqA2DdzjyNo3DCNNXO6GpUVaCBO7rd6MsxeXTwS1gghLgP+wQbvcD\\\\\\\/spF4TV6UTBiNMiAyMRnNcL6zc3hhBWsYTWJ9QSQnwgTmgWlYXqHI0tEbfZ+zLzMK2CTEEvNYaCU3GKwGBl\\\\\\\/iZtlPol+vj6nkh7MFDTLzv8S8Z2xcaf2hQ8p0QQR3UN0U\\\\\\\/UQ5vQWDa8J+kirJfNJCEmyVcXBOZqGJyGlNGa\\\\\\\/nD97O8zSod7vaD+9E0qe27Cc7LRE3rVigxICvmHeENjP3Tr2A6+D\\\\\\\/JqwhLnayw1KlK0o\\\\\\\/V8vlM4CXR9ksM87yIGxtO3UXzPqEx0vCdRXZhSXRTsTEb\\\\\\\/x6qHxsrRRPIYD0aKSulfhy2XZPkNux79TWVvEUyZMnyyk74XpNdQzUbnUZsxpfMPYDWuAh0Ezpxj9LiCeID1sH3gJoAv7cK8ASnCx2efB+3vLgioW6HT2GB\\\\\\\/PFhn0zaJt7h6ZLmeG6TC16xhIw3W0MNQ2ql0qSc1ArihWtJv9txXLePLUqKQwroHJ1addbQetyYFJAT9hsUF8EGvNSrzj+28IEZL579LDvhFLKDo2ue9gZ\\\\\\\/1dIYiY+LDIjAPbg5hEdTd3abui4uxNQRjlBJte0fznD1fTq0YnNdHjoueyDJD6UWw+4kR2\\\\\\\/KHEq\\\\\\\/LzFMSb\\\\\\\/tOvqK0XUvn7LIWVJ0oRWk8jMEIFLr03u0lUXdmXgpewPUp09FUWS8JjAArJBr7RQwwifk\\\\\\\/VSOzJbazPT5shdag+t54oN\\\\\\\/FTYqpASs7H7BSAVeTk3yHMX2jl0xgLN4952ob\\\\\\\/q4Oo8aly\\\\\\\/TPx13t6ne0l4ToxT8xpYRmtMvqv6RCsS8WYxbF2t3qeX60Aulhhss\\\\\\\/jCt98DTpyYxancVR5kTrTRo3JCjODtXtmpzhtUoHT0LZW\\\\\\\/z1br\\\\\\\/OxypaZcd3gldbk+CfzW8WWzaF4JABl+kF85vUPWU84SH+plMUW3YAm7hMrkJkIVVCFn77vSr3sQuHLEcCvUr9Y00RKu2pZG+Ly6Co5dnEpT3ivieKVF1MN1ah4ZhIVXMOONO0fZEd5abbhY8HhnABFcTfQJqr7fcHVLzhMMaELXaZC0e15ZFUKJiZ0Gn752FsOit\\\\\\\/Nu7HBdNiWmBgpSKtf9IDA6vIh4HleDUw6bD+ZE6XJNZEqWmL88JOqEswwb5Q8bjc+kKArR\\\\\\\/4jQsVw4TufSwCEK4POU4\\\\\\\/0wXzljo6eSu4\\\\\\\/t5Zq26R2+evF8FHs34vtqNDd9t46K4Xhd0Nucn+byPD1XxIkdK58jZhMXBrBMNtwE1kC3Tb3r+ycPI2FEWWlHZv08r6+OvSmwLc0rSn6RW6DaF\\\\\\\/xSIvySgvRXcoHBCG+t14ygqtDI8MHdSRamYGosGjt9xDseWLR+vpJVVBddHoufjFU0TBinSXbeyYPt2wBw4dvG3OUYlyp+QTOoGqSg7ylvCSirjD5YF\\\\\\\/OcaBPRhDoOGqsb9M9qH6+YwS2ckdvoIpURoW98tPjhTSldHhxZcBtGQDEMbBsjXPzdf\\\\\\\/A+zBTMLu+jzZwHk2H5VBix5Q5wgOadLMjTFVIGxA1e3hsW4\\\\\\\/f7A2V6cLwdT9822pC6vEvU9ik9WfIxKLbw3bzTOqL\\\\\\\/6oy5zMWw2eyvc4NyeAhfeMLjJipPsqLVPzOwX2SHZv8bpvRftTJi1TXHb2wOmMiHKNs2s45h2Nl1yeV81fsYPT8rJUBLdTiyCIflVQziwt8AGD\\\\\\\/5vIrAdYL3zMkeRI9Y89cB6q+QbIT+RI6pGa02G1nscVz+0Pn1MECYtmxfDqARZA4gNbgVIH4o7bgMc594p14LFxA9FLBND\\\\\\\/bopumKGYiKRdPrjHa+b1Cbd6xymhAkoLsisOqeXrl6OIqS\\\\\\\/\\\\\\\/OQ45hG93c+jh2PHRR1lDCWLKTl7KBHdaUbrHAdNJwW6v7G6kGjdBxzJbYGo8npKMrpM71C7VnnVoIQpNat6TaLnfa30VWrlRj08LcROUffF5zHR2qxlYEf0CJIwRL+IDUTjzHrvbBYI7ghshh8+lIaeKQMFIQy+YIUk3wc4XE0o3xIkY3XP7CLbx9I9AedM+c0Ef59jtyMhud9Nop0Ypzbq5D4f6pWihY8DLCcthjCT2HegAj+99GV0dMnb904Uc9\\\\\\\/m7gDurcDuV+73RbV0Tyu1Sk6JzTTt6uShEnQdbEPDEUQaLkJq\\\\\\\/ksCogLBWGf94Caxa4uMy\\\\\\\/Gz63VFgvQiyotYkPVJhLvWLYO49QhfNcmNCIAIZWTd7\\\\\\\/cHJKAAb7R+HGXqX+8SkRzkA9+9YtTR781ODY+cIyRCR5XUl+qf3NyT4Y5MjCVLNxzmoIoJoMtYefjAfHInliaSDGuCy2ik5zVmZ7w0cqjCkf1CvqoWrE8DKd7xTYbsCRXZ8pVCElzCUY9bBeV8NizVQ6dCszqJ9jeNZnINDekv4t9EBYY2gMYnVxCwL7sZZS5WYwhLh1JmzRdq56OqnBH+BQUNfVgOpsMZgr7woZVAz47GNAz8YXHrbNxLmKrW6HliplgDQCmPlEJS\\\\\\\/bnwGRUNp\\\\\\\/hQrpuKt0Go5KqB7XJPdhIwkHi\\\\\\\/DytSgEE3ELBIuiiB5luqmDxVGyuvU8SGOgPTvmoqIEj3smin7ZyznHKXDpYS2O6bMsw\\\\\\\/9Dpw45N5h2QZp1y+Y4Tnv5glLdVUbceZC1I+NoUIkYs3SF\\\\\\\/qivoSrMXsg80qXBYrvaYQe65tCZFeI3FUlfVH7b9SKztkovguYS4QFD3GCXI+bhyssw73f0xSoimd2hHBIDvPxNafAOnSOCo\\\\\\\/pTscJULQKk1NAoRjKcF5\\\\\\\/fr7eOlWquqnwmBdIIUWGmFfFcdLelERt9b58ehehDnHWy62QIiGRyRrXLAVS6EJU+jZ\\\\\\\/DVby226RbynQwTjC2LBZRi7nJD2n6XHRv5zy+cYZF+GkucjmX\\\\\\\/uVUOW1PeLainEenE24XN22UkuGZnSvZKPMkxoC05Sdlvg6h\\\\\\\/9j3Ljof0vmeWtZ8vcRvVyQq2uQVDVcKFQziUMMtcf3CVTmKFju0ZVeHgZRz7seYIZIUY81dt\\\\\\\/hiUTPoKtQ5LXhwWD4dMk5OSx3ARiMT5hyQ\\\\\\\/xAoXnahyE+bXE189u4d5JE10UzZOdu2FRworhA0IadSISvzwwpdGO14LjRRB3ZpE8k\\\\\\\/ySb2IFhQ\\\\\\\/Ui04JC6\\\\\\\/lFgLVFF3v48CZA78v5B0ehcd+7GvWz3YiFnmjggXDF9yyPRpbZ+uOwzpJn5mazf2NMJPnTJWpa8MOiIdQqBZWxHNk3qL0FdSnTAFgcJrdGvLJ2sSrNwXaCAzTaEnzKTqmE0Ob8wO9ZeGKc4MO9NVDp3IPB\\\\\\\/LghAaZpqtE1nGzainhoqU7xrQfcoD78RfWelh8OAeupIRjLzYsRqRnJFl64Wl16M+inm0iXn+P2\\\\\\\/mUBWH7hZZYoA+21OXMYWK50oHHQA7CHYK+JutNmsgKZUtdWPAnDWd3io1E42\\\\\\\/RTgJEp\\\\\\\/vBMVWJFG7oNwOr+Im9LCf+Lo1bmzzwe04leCHoEnjcQ6YCZ3xEQ7i1qn8ILPq9TUe9CqyKzeiqheEEyYmfCm1P1OO0KTHHdJXBuUnCAlcRb+a54SALfxIWQByB+lZ3AJmFDqSt4PdF0kGJxJvyxsS6huvY8seyoAQZeacTzJxai4aczkBnnLsLmtPcTB4hbpJGopBGJ25ftv6VplUvUT5o9eL8ddfIiEnO6XKcFMEyJg42SnEbQaQ5uP6VHDqNl40p4jqlUYFzrJgTrAxeYbudqcx0N2WHKi\\\\\\\/XjH\\\\\\\/F1Pui4SgXlOh2YAbnl0YqQ1qgWkVtEhgElLvVKkhtZ5EcagZR9v8qrTQxd1dAPjY+pq9SQmCFzt0gURsTpzI8urtj9uGXmWsoOfZVlOzTfWYhDv\\\\\\\/6MYizfHDBKeh1RyBj+BzihWn2E1rhx7MY0hA7vuHZGBQQ9lQH4wTLNFp8BIlHNn526CHn2SgLHc5qpLF70LaU\\\\\\\/qLJGETtbcP7VElUDrvs4Z3FLRtYwegxbb8Zn+Bpn640KcVgDVFi7JPkQZPsDf7jIrbt7zmDPCUm5ZnCBGOaYfUd8M0CHYZnwN0jwnAqMzof0rGpFOfo+S1ZzMjJL2VuUkkHswot0ZFRC9hLw7n7kLmKGHQv2LrYm683KcMO48SdBGSqShMUenjmm+fSZyG1VeCJAh25bZtRRa3NDwM3HLKG7wfbALo8MyjVzFX\\\\\\\/ZzU2tPnmQLDJ5Ef7Jy3BJFfav7U7k9h5jRkUIcAkFrJLuEUCBryFFAz9xBRPuASZIslfcFEl3xLejADtRMCctuBc8fpI0gYgk2c9R8AqMNpyUrTkPugn3kdFHAAAxtiA4ZhOG531HDdJwCvW1PeEv2JJDQLcL\\\\\\\/2t81kWom89kLyTOWw6+5ypA9Tqy5G6UMs7thNvJNvVyoI9E2oAF5rPeEmKwFJUQjQ3kllFVQswI+xRctqeUjBKpvIihYS5MlvCwrqJwcVl5Qsact+5v8BR0GWmmpTKTi88rvD5qncUFqIFVlwwkX26wxUb4mo75Ja8FianDBwZ8K3cZy4CYrjEb03HSjtdyISYa3vxzYwxH\\\\\\\/gXRiYRtbjZzR4xcNqTJdGlrEo2ChN7Zyn5IZVcV98o3Y95gnEKBUbA\\\\\\\/+l9AWCML0dXu3+NIVYUbfSJkd\\\\\\\/08yakaMDqeyWXPZthWnEIxxsjqfn8fmX9QMgDvFQZ0fnW4z1eSZXuacMnTKlZS2OzK3u5M1IVPdz+MDHNOw\\\\\\\/fLZNEWjgzODLX1dnWcXI91lwMy8hTB05bXNw5JDbS629eBLn8CM8SgyxOJurSSTfzsb93VybDwRVpKbMPM7ald0SrQl67eCr5XmuEd2ZDrKk1bPqxomubjmx1Pgj3p\\\\\\\/CzLIMtkmaLAyaMGWpeAplXxBd1QebwtZTmtcmO+wP6ry\\\\\\\/bvgHIh+aH1xevtBzInQivaL21n8PtH5Qtwkv0NR7nJEs5\\\\\\\/kVJTIlIGXpsCKewq0uRej9kKoV\\\\\\\/\\\\\\\/sXUfTu8Gdk7+Jgbc9UvYpIfZEHCUnLikG+YOfiMMF6xPI1G4bcnmP96DDxwfPzR0WPC2P2KFzcAZQglnPumJcyIq21zsa5NrMHSDImCcR1n0NWh3BYMtXRVIhHsHYu4I3EZdW91889U\\\\\\\/AsSsH0iiQT+Zsgg33dORQpOTlFaZKVtVpjbT\\\\\\\/yPFOao67Po+DPzK1pt7dyaRTe\\\\\\\/7PH3KAGbUMbNUmZFDJRdH\\\\\\\/vwEmIdKFrE\\\\\\\/NELRkj8gvci0QLRmKjWFvPp3CGUa7smWsS6JsCZ8Q7EVHI8P6jFpI9bdK2iqQtc5jBjaKDpvjvMVooemNAnZOFoh3+41kibHqJaSsb6vkblGWnBMSN\\\\\\\/rcs5Z7vvr55nKjrlI\\\\\\\/tHbh3pQPRcOSaVBs9G2tlOuCwOQaNzIc4VGc1N8iE62atKrw6rh5\\\\\\\/ucVeWp0wYp+bdmKawNP3EFuUIsTfX+2zv45lh2HN58UBvAwWKqUFsYlwFnl4SCWviQIqciWbF48+OatlZPttTpH28DBLkpdrl7kyi7o48q\\\\\\\/FjaLZwgUKd6QWpNTxVm\\\\\\\/AY6GA2Raq4TO+meM4fuFJs2vwTeb1EKeTe587QhyJjL37xLHVn1Nc3Jdp+mxBAskfaWp5gvTQcZbTHkJMwGIMOZ5kdHsYYXtxXUBpg7JRZ\\\\\\\/b8cg7DwraLT+t2ICu579EsdmdQfshbb7Ao502bxsHVhBZnRtV2BIsGfnUn4+Vsq6eCPx501W12cGe5gzWTB+e2RMmV1Fd9Ya7Vf1pTwjC+RCNBwz6mPzuK4sx6sfaXR6iMDDu0cC0Rs1F2KHbuVXQEDOyceK8GpZGQ6BChwUPitmN3SSUvIoYu0Y1a6VyBQ2b+B9J39W5h0drwYb7cCuYZGbjguRvD2AzWMUYv5yhrIId2h9EE\\\\\\\/GkqRiLuuYjIzdMBEge4JxcBOSv6p0XXL3vK5POKlI4OaQWOGcSwtwnEfPRllO8nndpWcrc\\\\\\\/sVHguqXvc5pgRhJct3H42+Aotk5HD+WgyN0ZBthYM9hsY3PHo51OLy23GeAk8QW50cdngv\\\\\\\/yW+nuVrGTgF\\\\\\\/jI13Kr+aQAJl1M+INWL0\\\\\\\/jQ9p6Sz4eobQ3ayRWhRsAgMnsIuck7KxklYzNbbbskU9ykzdcWL7bmb0kYeaxD\\\\\\\/nebUlDFF8UwKEUYYi9uzRYnt53QXAFoQV391fJ2+M8taM4t0fghr91Pb6yEvheLRqMPThTRc9XjIU7XY+8PujXBnvIUDgMS5O+Ts3OYHAhXW9YU+3FTaBI9N5QhieV8Ylw1CcL4j\\\\\\\/A10YZfTPVnFX\\\\\\\/ajINlQYZ8EfFJeElpMVbc9YiGNez1tbBQGzH92lqPiO7WTOZBu\\\\\\\/Ajkn4tp9hp5gXY0tSj\\\\\\\/JZY15fHoA1bDjxE\\\\\\\/Ujrx8LVQOhZ8bqbYzWg4X95S2lm9LwAS\\\\\\\/lotn5x8l5pQ9tMeYVee0vtYwXGixe4GuwLEG+AJUZ2QiEvqIGlzh8ZFKCDg71lpDAEbdansGUGU\\\\\\\/Brvq2ks27Sr3Wx6xBa0jgmhHQkmSkx2jbCgtk+kN+VKcitjFFktDuVE2lhsILyedJogZ0W+DRbzFnJjigf4k2yIJpirHFRpUxEMCvixtfC8y3khKS17VWePAvpjH0TA3O5S\\\\\\\/Ty4i55k1Eqt5sh1+GOXUQiKDO7Tafb3AEVn301mIsMtK7f6UX7KzVgUYkxoA0LGsPIui\\\\\\\/bGTqJs5cjBl2G6Xd7KPsnH6NgS6eNs076p3a6XU0OsYbunpmtje4gBXlcLfdDs2A4I\\\\\\\/Q0r9w5ZTn5V33C\\\\\\\/2aMTfOU3VQR\\\\\\\/4Zkn1ak1FTZ+7iSQ88dyplssUS7xPoK8xO5BKK7RNp0ckBUAAtaaLvFC6XWq2AiEt5FS5aTTfx3u3tS3WXN9qKUmj8DoIgBOwtgj1UG9cx7dnrXZ64yLaupfubgdyJJh786iggB+pFK9\\\\\\\/eQqInuE6qTp36MDyhPPij7Z+ww1oVAucViwIIzxk3H752zSBo55\\\\\\\/f9zrgW12gjP2BKA3bvNCYhSUxm46M7b\\\\\\\/RWAhG4OcsdMG4Dp0ltnKuvWPyIHaRmhRRWkAfpwuoXDUHCxSt3ZX+TMCWaIzhEBFtFdH\\\\\\\/oeyx5tTjelW\\\\\\\/WdB\\\\\\\/pFUypQX9ZGW5uBoyRtaV21GKdTClbmnjjABjUaseWho1BTGBOPZulxydmt2jC626R9zpguUYDdsD6Ai0HjcE7WN9UWjpDvMn8F3of74Zy8i+7+3cf38AYHlYBPd40rgdvCb0CpxDNVa5ba9rnha9jLIOjWBkYqFEKa54\\\\\\\/yrTdaLIYXRAnmgcnTR0fkf65PZvtZxlRZF0FPg+OZ1NYH9UkqM+7pQncDCpJz6UGSIrXix\\\\\\\/XrZ9zQPLMj7bCGrAWQBFJURxm7SnB3M0QjYv+\\\\\\\/Qh2BJfWdxvVsAAwlRXd9iR5Ld27hiBo86fBHHcmdLu9q2fb3hJ9EqBW5Vt3gOUUILoQMcwxZMFDRlGke62UylSgCiZ6vF03bSAntrCYumwdkhV0nLrds8Rg9aBXql7jO4lLJQF2UGZr9h43h7Fh8te\\\\\\\/fhcNQR9BJP4g5Y0f0NxBeK0VoRaxWAA816IKfl9GKvYewdMpbHG3Elaij3ACHCYqLbBKm91NmHN93WKQzr35QhpWSqwxBsnUJRviM3JOWDMzJreKErKsnarS7Wkzy+hA\\\\\\\/NER8oMl\\\\\\\/4k\\\\\\\/WjSVHUiLNYKGFSMLJS7lr\\\\\\\/zOhh51o+lUMKhEzJGTlkaWh27LU952o+HsHPTGsdGpL8BNauw0yqTNZwFNxiekPYdCA6dbvxXH4XKgupe118FRirunYH9Vlrh+SUZPBlfkFcn0a7QPxwJkjdytOgYPjCsII3CeFjCJYG5gelRbJWcG040ewRKu8Pqceg9LPGAxqVky0VMhFfqThZhrjBFzHJwzZsQFka4uf\\\\\\\/kzfzTOBF7UsfnPBMUB2uf3f0\\\\\\\/JzlJVW8e7YfD\\\\\\\/+H7qCXGXwBiqN1BmkYxNs7\\\\\\\/Zil+0\\\\\\\/9CvHjLdgAzQsCWXH1UIiaQzCNznqfpMD5FSd1hUEi7o01k+u4at8xdPDpnNRUhshEJXHEbfimxs5LQO3WG6PmYMwiAZEWXz5CzbCP8zNxZ6boF7MPFPP7V16FTof7o1\\\\\\\/Aqj0U71TCG81r4rcyTg+TGMjVdTLKbjX0KiJOq4avuXJs+IGLD8wHAqLTgsCnQWYfhh7MNip8XWnQaXb7lk55YpWKU+v7PWYJPLY80WqM6fhOLDMNPwiPtqWy\\\\\\\/ccs8Z601iWXlA2Q50dJTLPuV4kex6vVULn0eQSjUvcOzDbB7o3Z+6UwBZJTJQtwlZ2sj0Myls8gxBCoMoBXEKDwvmYZOmKQbQDTeBPg\\\\\\\/fSH\\\\\\\/TKWmyhHB5Ubtjxdl4FhAEKTKuC4rJdhV6Y59hN4QOc+q9NExfFzOei860R5S5tosTX9MahwQ6Q350ygrApIxY9ycyyBrncSNoXkW3Y8RnM3BesVlN1qjmvjWMTPD4hyH8QC3aJye0722e0aBJxpjXlTF8skW8jt65v4yYIkq1G2kWUARuCVCjHXag\\\\\\\/esfyCOhdA3sGIdfWFHB70GNCy8G6DN+S9uCZ1BYyygk9uvDQDElsG\\\\\\\/fZCL3fgepiN9tTdzHBoamm3kRYFKH3x00tUfmxYn0g\\\\\\\/bxn9brXTIQzi5f7wns0CyG7c2fMA706Vdbari5G5ftZUiOX4yd1tEPWIlRD8WABrQptPieVEI3vSgZKjFD0F5GlShKje9FTuNSzfATfoZtrvfRwjB9OYaXwB9XDD\\\\\\\/qU7bgGw1uY7UPecxcVY3lrJ9eZ85edkAcM3tyDsMeWTqJlsFkqMWwz2h51PA71cHojol51aVxrwuVjvrf9SjkP3WcLKr+IlkF7Nrp+e28zOk2V1mpN4wRVXg+uZwRGxAH7fSne7FpOz\\\\\\\/CxyNZiuZ7vlH7SdZV4g21BCemn20AUDcdgy7J0ZkS2bCZ\\\\\\\/qxHxz5GABirzhCT3KCd5PVDfx02CNF+Rb5fko1YOMluG7fAcyMcy5jDFos5IqF4hP0Hc3tIzB6phNLtoBfeZ3tae4CxXlBMpJJhss1b4PrIVzJJnVQ7UwJJNIKd8GjFOwAwUawEurXiiCk1r8IQipoGEkS1QqOtTwQlFD0GuZ61zhUTGv7Z8j91434BsgUSwWqQP9w3f7DSxRLESsEj16c2oeL17vJnt7sO3EJYUpMj5dtu6Htc04RQ8wfDyUeptSOIjKHTEHNCJKOSbtcXJq5ToP5F0yJxn080AIfNszNBm10fDrEqoWBbv7iMVraTmScZ6Z0ruh2QJIDzzTX8Y7YaS9Q0+01dmPpVkisLaUVav6Ejyi999NBC3525ZRVSQKk1ev4ULaJOxtfbPDxrorkD9HlKES7LPpnRqNcd9icNf4PnQnYHRlIq1Duz5aQ07SdPOQOW3p6Hera6PkeZ4Dlwob2mFsatLhEuuMXQ\\\\\\\/QrdyLiZo\\\\\\\/3ir7y8HTqniaUWJmwIAFIVlonFqjk9tV4C3L2GPH2gM7YNIsDJBP+hMLNUACpRLJARLuq\\\\\\\/7xQagmwY89xVcFNJiU879tuMUnRGwwW+bEOKZ1fZzIzwPM6\\\\\\\/a2\\\\\\\/lPIAZ\\\\\\\/qA5bV3YZTkfufkGs4z+lge8\\\\\\\/gNi594f5hqpb2YDxkPVrbMbletRfwIpVbif42MncxMXcGOGYC8HuiIKGYBBbMeEPqMXJ8FasGpHt4xZEYidbY8sej+eXsmEm\\\\\\\/PwZ8OlrRK3r9cvts7ClAfRoqzyxPY\\\\\\\/O9VmTfOuiJICWjp5baJ8yxvKoLQzy48GxEBL0wxCLN+hk0niqijCOoUWQDxnW7ixlL6mhtE11M2ReDGljBJpPQ0uKVwUm9JmVPZCmSaTwaJmdKmxCNn9ECBoOj7W7AAA626UE8XXx+j2vyFJVRheOZl9XqeSBUTxCg+nueXi2gp87DxqipMEJD\\\\\\\/31QYhBtu7q7U2MV1J72B84k5QXWwzbVvfeuAsUwTL8Y5EmnDaxKCjwAr+cshDDK0Cq\\\\\\\/XtAibIDL9v\\\\\\\/Q4MEp5B8r4h615rJ6PxrzIKkM7J\\\\\\\/owXrKwHuMghbeKwkMlW7vLo6gZzEEevgF7KhYHv8sKVTFqqCRsFaFIcW6K5abAvCj+08tD8jh5DeZXWJrhRY9MRKLu2Q0DXdpLgQRkyEmWnN5C\\\\\\\/nA9DvTjpWJFOxc8DlnlDnt+ALWKjh7d3eSLUK\\\\\\\/fKKNzR2zpGDeZPnwv\\\\\\\/ZYqTcVxKW3be0gPJNkvp6ahQeANsKp6eI5lGNfm5p8UV85r09rEkkj+7X2jV+PsV\\\\\\\/4wiC8BBg4rMsNfsozU93TscwNfKAJCF1W0CeKZwDERgFLxrB2nYgGzRqTJdVIi2lm2XPdVGJND1YsMzuitT6R9+6SHRCItzUBMntofcNul3wDmR3N2PZkmHHHMXgbSocA9GCSiwmP0VXIypJ8RH3vToX4N+YeNmjsSWsgk7V3gbLoo\\\\\\\/dRZIsBPOd2Jy858waycG0T5u4mrhroai5uTC9KRCT3IgXCoxx4dZ5E548YSOvgtsSDfb8tlE+1imPIPaLPVpQwIXVY0\\\\\\\/C9YR+YMHTkR1ml6NcDT0riFccBne809hpN1Shiv5chCzzjxlyy\\\\\\\/lXtTmwXXz36ooQSkxODNQOpOXj0AxSHm7f4VIlLHUAE1P\\\\\\\/UOGhuij4wPyHZy5Oinq7H70C+zEKKiX3W4F03q8Luu+79mBZot415jEFwFgcYYGB8OylFGvP6\\\\\\\/HHEVIK6v5xC7kBZF1mw3KfeAgWvnhs6p1p1eitdPRTJkfaYQgN83x38TViJMSdkyEWmit5jAK0JYDq80emmvOFfpO7XO0vdV0mqXN3CmzMGisSf\\\\\\\/gzmOTbxPk7nWHND9wfNM+Jl6iYl4jvX1nrv6m\\\\\\\/cwnMfQq20ZVTLh07j6WD29zdT1aD6aYZR8umW\\\\\\\/6wW7vqT9COeQIRFvOilYUEcWZYKHN+9BXWKLRSFQcBuMy2zWobiVi0OeZX5Qx\\\\\\\/UxuPT5NIq+dYxd2MAzXRcI7ctRjXLLZlYpRtqssIb9oIteUH6vzxqR0sxyXBX1lifhjh8QyeyyQipvkA89Vohyvl7cUv571uHmSW7iO6\\\\\\\/hKxv\\\\\\\/\\\\\\\/C3ngDrNT5cm+fahJpkt8QyhAskmwYps3AgPTotU4m+cmMYBBINJeL9TT8PjaIQpSOL4j+4exLa\\\\\\\/L9jkvC5hGsCWttprAkW2NcGOMDu925Y1X\\\\\\\/EfTyPXhLF2a5GnSjgrQvioOVU4OTT9nDUSQJfTXXYuhIdEweOUinAbjzUbJ2kzZqOopl9\\\\\\\/azk4smZhXVRiBpKn4D+VKVFwEgRFe+Pl9X0OurEJhjgyc\\\\\\\/Lpk6AFx3q7EOcHgWX0jQA82IHvQVKeJZGRxQrLNQyqr3K7A0E+S09oZ7Rsv+xABWTeE0kdZXXRadVmv3KfUtkVqGQOx521+1HB7nhULQ70RYwwl6z67RW0DBPE1gJVAhhVTr9USu8AV9yqJzebcVLXpdnQwX7vKjkOCT1cYV7nu6j1zZXi5yRfiWcrCvG3fdC+0eGJnwTdCYyB93XtQ0lz\\\\\\\/bLvp9ucNRutTdjKflMfbrs\\\\\\\/8McbvztKjr5ZsZ2Lsa6sw9e4f6iCRfMHQ\\\\\\\/DoEkeYiFu+27yJjCBN9AnxFHwfqN4Mn5+Su9Z9Nutku+RzgctbSkfuOnnOwUlT2CJetcLQxrDSEhmg5vDlpeiWL6ndGfq+V5m3uvnT6XAgu6vZ0sYJheXEPGC+JoJ2iBVxxQtRnPIrk7CImzcPiiPTezjgNWV95d5ihENDDrC7fbttON3S8LlVo0k1yDBFCfPfG6mTyCAOgNKQEbSmdoGc7pXUk8h7wGOFfSIjb8CgWj8xf8mE43x4t7fz530paPpZCA6XUviPk5iU5Nbtbvp8ZVBx4GhGgE1yvYCatrAGV4VSYRsQR9+IMEvT2\\\\\\\/HF8DgnMdyClXyffZFwvdJg8N50G5PNCwWUFHOL4pTYx8MGN8TiwOlTCvmx23qpGGoZ01AU2poqmzq9IP3xRzhdfmVxgE0E2uTzynwbL413\\\\\\\/UnkhmZ+0afoDNKqtb5Ef9nE2wOYVNlvjtu7sBjpF7S3RQ8fPI44mNP87TgJfGZTaoNlvmaqKGWVbxg3d17zj+apJXqWHQv+ss6l7l+OrqWERe8fVkO1bRGNiJ4uwQF97pO7Ph8tuHY3KMw05aXlBq9f+eGUd3PyWjivnkF4I6LuP7jLddCBDcyKtAmMsIQErUBGQN56bp02bS+Y8nR5QPIOo1XbgLU1KFfezH6DufvEnt\\\\\\\/9ZHNnBJBd0wN6NcuD+BCFH2+UOG2XVN4VRLZ4CktMcK4\\\\\\\/RRBAbFMgtvWNQi2AoLye4ohme1E7Qg7YKSPkRZhl8cZqxkyVaGiq+jojNotHmFSFx+1Q0pylMYNaM68lggWLGpqNqjjZANEZFLWOlnEsxe2G6Pq6EgzbpxxEd7O+lS\\\\\\\/wvrtSWvz8IUdW7VvebFXGtw\\\\\\\/RC1pPxnIE7cCxMjWIduOxyArJJP4lM+igDjo4vJWNltwtSBPtW0hsi8u0KC8q2eE4ssPt7pvrMiNEhQ37+lSYbxNqSLddAy2r84bSdRe7OE\\\\\\\/C9TaiGzvys58mNLOsnDu+bonKaesqu87zz1uti+dwUuDqhUsqzor64FJpSnxnXLZh9s8gREu4WN0zrRYgy8sgjcrIa0SXMu4g+kYYU0HhQ8N6Gtt9dWZl984ykLPvzJN58lmIs\\\\\\\/j6mPHRyUxVVFmVVTqUfKcS2v7G8abw6z8hC2NwKqaYslwuvGiyHKR6phQg5+PkyHbf4iT27M9i5HvC5FITaPa2JRmaqlv7tZXSIRuVcRPYeSF7BovYrIe8Vkd9p7tUfGu5wRFT1efeCB7YgCKP2QqNPFRX63nam2dlsmUv\\\\\\\/eI4uE49y8lQvPXG5elai5dq3sB5j1taGC7\\\\\\\/\\\\\\\/f3v1X3pHpSXDhi6ck5XOWxwsDGAaPoNP2+Q4f+SppHkwRDX\\\\\\\/ZkoG0g5FUI040covZAPIXZumI0kzBG1KHvQc8W3+GCFVKsXm7c4KQWOO235ohhqfO4CQj6AWk78x09BooQ0Y18TelNzZSLtwA+XliP7SsymewgEhIRD2nUdaC5H6jTnUzqOV0suAjphL9zqmVRzPh6ne5K0mJWOu5AHl2aGTG+JD8CzMrRDFUPrPCn9nHQZyYxU9kMfV2zaOLJCeMVT3hDm7mSC\\\\\\\/N\\\\\\\/et3W9KcoS3AVxbmeQVzQ5oHXZCqXDpt3u+lqEav6PdO8dZden36p9LETdlYbqYvY05qd8l4HXfar3zTj9u1+KfIyObRylg3HP68i6tvHCbia6c6btAuyjAv0dU+Cf9Z2KQDAAC4wnDUqHJtNUN0kRIyw9jJPiledpC8jUi0YR+yiZhVSX9e771YqL50Gi\\\\\\\/TSngeJ0d6NrewtPpXF3i0nkvdf2qtjFs1XFIYWVmpG+vaC1JQUMPXAeDZRiljlCjZTevsKR0hgF5h\\\\\\\/zUk4wXn\\\\\\\/yHR8BJRP6TUNL8R\\\\\\\/Fg3qpi34UddXvy9GJ6v4oYejq2rb1W822oiKsXS\\\\\\\/ArM4ypheXVMgEYr57ph0Rd8LaugFTQwLEE6yaQIiLNaEOYdZl1ut7v4zF4HVdohShsTQA32e1ZSRAlt8GstfY8nSyiAATRSW\\\\\\\/usNoX1HIdiEU8OAAyqZmFw5sq6tu6sc56am4iouJCbXvRE368hr3Co6f0K\\\\\\\/tDIuMoAt9rEJFgIeLp3vbRVqrkSS3cmChzVQxBk2VMa5azr6L\\\\\\\/p1H5v9f4npvK7idOtCY1lXU2Ek7t9ml\\\\\\\/tkYmiKnbt2gDAI4ovpbMHl5QNy05Zoiy4U1wWLPtrljxBl4Wt1AWqi9QjKMkNv6zKEsgaOl389Pv7AI3WuKyt8eOr23p\\\\\\\/3Y73aia\\\\\\\/Q6Qj7W\\\\\\\/zbBLUgMu4K9AKfC41OZf9MlmZDyVvJWuL0VZ25RK3c7QuOJS8bw33WXJLX4u0jQKvplT8wkFMfOmw2tON3mvpdTMPQwji\\\\\\\/KyB4mS2M3CkKCwmSQPJdk2OSyjFcalis9TO2GRPf8T2flut588pEW4BTu543EnMCAjuCKOPDkSvOwojA5kyFzvUiLVRIaqgKt+pvG\\\\\\\/1zDC5KQ6xeCXVJ8LCKyf6aNHIVmIipTzrzq1fXprq5OrsiMj7JWStsy0M9XuKZVvIwPycB6c85Ww4j4X7qDvmR5MQHsM\\\\\\\/hmObtNy5fWyujngyzKJQV8VYYbMc+57Q80cDn0ptWeedS1P00cfkh8I9ShLPeRH7PuNaWeNcEmS4cPRSIMUO3snsQs9wC0w1WMR6XXeMnscfmgojCrkafVd8G3JJqvvKqzKsHzLwbSULwlyX2avGoEpswz0iSTb3MTscwXdpi5s5sW1ArK2WkDqHaXXxTeEGYT5tJsAC9a1QSA10repx7idGZexDG3WyG2G9mFO9hexaNin6H8EDvZxHhFK5dQt8Hq2R0Od1WaqumDKtTLIB99hIiUjZmzRZ5juQ7ZC1ctPbYyRzxquBuh0CQpW7pIZGBBte2mgS9odccm3fFZUc1WLocJY+AxJvfV+xAg88B41ZsjLW2BxRhaGPITJTXwhvsGhdnl9eJgD8M1X1wTJzBA\\\\\\\/A\\\\\\\/lwEBkUESxYNqBexwth2EplER8RN4pmBvou1kH5DNfg6dNLlR0cLdQ0ZkmU1Rw6G9zAAOVA\\\\\\\/a1lRm+qzDL7EXYE7vA3zk\\\\\\\/o5p5oxGoWtrg0zo\\\\\\\/pCbEEHhshOAfWSneLuaTAR7gZxp38pMovsTVCvfXx6TdOn3tCVzXSLTRzKP1aSpN+IjoqXMnMlt6UfZUBiKifzA\\\\\\\/8XuMim09poCDg59lGCkG4W1C9fQHrf4N2HAw6Qtycfn7f4+d\\\\\\\/HZinhmkeGEax3SzHYBZbaXUgwOVen8FLKPSnyvq1v6at6KUBUuChhZaeNICUCr7DlHALxOxJSszpmbsWvf8WSeZ+wvM1ZFCld2wcb+Usms18VR+GDcmCeYM+9R8WIhzy0UWnrUBTT4Bj6PwnKTqPptUjO5MHhuOMGzKVC7cvW9dknskUKLL0OQn0FR34fXVRITk5mtpqp7AEsdo95vQpJw1lalSwdn5zkRmM6U\\\u0022,\\\u0022iv\\\u0022:\\\u0022zl6pwP+zoesJiSsZzi4I8w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222c3f7a0aae1a5a2fef589758572893e21b2e1db23ccfd78e13367029751efab6\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bnazZkDdYr\\\\\\\/Uu3IG51SceF4lP1KAAeCuS9jIer78mnYngqn6lS2MGZIWGMD+FP85a5kU6Jy8+Un7GwR1e8Pl5hQE\\\\\\\/u9SBliYivTwS9tMFyciKDXUE7d18KQSUHbvYP8o+Yei\\\\\\\/24VGu\\\\\\\/Mw0Y0nK51GTTUQlM2kLgXc3p+d3lIsUOhvPApmFLDfaH+Hp8UuYNEjLrNQNHd\\\\\\\/iyhFlrnK3vRGaAjNtLJU0hQHF0DwwOCi6sXBo3BxXfU\\\\\\\/0nVv5y7LCyec7nGojCOJmVuf\\\\\\\/CsqB8LNLvPGeqy+rQDHA0GiDS2NFYb2KN9obA0ZCCBdRonI12ID\\\\\\\/I4d+nTbA+stGIetMNf\\\\\\\/hIEsz7jFJa8zPq8F6kPS01UetgrzZmQ1u9udh8THJTMDRf4I2cGlmM0p7vRc34aRZNpS\\\\\\\/2mo370GJME1BKSw1JX00d0HfiNRV0secNfiOQo6up8mwPB7DZZcXHKxKy8xTY\\\\\\\/fUjZ+Q84MpM2iLVQKXbeVA6rvIyvZKEv7aebcUfjPJr4LnLdVPb9nSIk90nSPHA+Po7m\\\\\\\/4PkfWxpr\\\\\\\/evBMNM5\\\\\\\/aDe\\\\\\\/Pf2oet34Bw\\\\\\\/IFhhq1gwbBkTbc7CMeWr6K2xpVMfIA\\\\\\\/4mKKy4IlP8vmsb5YUspOo0nMuD8wpZZ1Ql39kf7J4OO2+Ykz7stAVRlfqP9k6qhrcvQMN0qcCaEVqDr3KRf4hhEi4OxlCVwuVWLs6frx2Z1tjd8aypxRNiPuIN+tSoRf0gpFxKONoyi\\\\\\\/YrWe6h4c4STrPaP+N74GKatsn2GLA+GhVqVYR7D7au150dhvUeBWympXkSZ3aH6Tbllb7FWh8PhZJfiWC\\\\\\\/x2nXLnhYoumhPlBqf8hQ1SLoOYA6nxSJdc2N2gLfSwasKnpeXVELPnzaiwEes6kZlscUyXKIUbXZLj04YNpI5+5KJS7APPaq38rU+ghMFncZLlwg0sDO\\\\\\\/ESF9hfaplFW5UEZcPUgF3VKXuY\\\\\\\/eDP1ZwrPmGVyuva7BKbyobdQAWiuuIKgX23sppxnslB1Xk7OYDi0KS2nVPXYESstfBnsI0Od6JWCN+Kay6OdxvZk7VoW6T1icrfMD8p01mFJ2HnoSmvxa+lv65IVUZB5BpbfYliHsVPHDac\\\\\\\/eOOKCJyKarDSnd5Tzq4MAZ\\\\\\\/zunj0HXErciXMRC2ziTpG31NgWDY\\\\\\\/CrfibL45c6gyKHV327EwZlIFs4uRXKmmTJwKhuz7vqJERY5a7mNL0RmBoicp1d9DNMq1R9rsaqjeQG5LGXzgxHK6aBw3bvd7kdgZatN4c1xERW9WEDHsOusoro51oyBcPcvD7Qxs6G55fDZ0ihdqa0SFUbi1SO0PTtO07DlFiRfDI4z8g5+q+bZTw6yX5loCUe5A\\\\\\\/W+bkzZuwtYRzvyol06IefqU6nw9x97ojt\\\\\\\/qdLTaga5CVSaO\\\\\\\/tQzTCXt2kQHvBPbp5OppIaAjeSAfL3hWjlBm4dQ+jZLbjGSOP+9oBm4y9mBhUubUL27rQ7w\\\\\\\/z3M\\\\\\\/hZtLZ5XsxExNO+tzIIugi5lbDOMJp87gOqB+6UWWf4IBqzQFLmCMPXtQfAk+AdmegHEOmwOi27OwjFQ8MHM02EZEsIIX83j8RVGP5jvi4\\\\\\\/ctldBRupl9rdaOOs9NR2n880e54GV9EA\\\\\\\/nAMBtdaq+WtWrl53ge27VvmpEaBnrO1jHIMzX3YhvLU38W1P6oGYagnvJ0MjyuzoICF\\\\\\\/NT9e2SI2fQ8vOSFH9lk4IxZPAaaBEUzgF2wXDeTg0Fl2rWkLANSPaZFDxbtkzNgaduJuB16vyRuyMCUO1TXlJyCVLDU+DQ2csPPCSVbSGNGuf1esjRgQ7gctWdnciBcZbZg8wTmq9Lv3NE6oiitFwLfTjJMobX5SQqOMqaGXVgd5JRyxAqzX1DZ8x3CvaDHhv7+R6vHnH0w7zd\\\\\\\/omcYOhdIcvxoXVgsuIUzFmK3EfXppf62\\\\\\\/rREgX+G1S3WSZWQoczJO+aptji7SdbJz8bb8gPsnnDppqzHKXSjNvJ5L\\\\\\\/ANFWsY\\\\\\\/87RdkYPiPTgtiLPqNh72bQBUamQGbG+YVD19iMZFYV3Y454Fgm\\\u0022,\\\u0022iv\\\u0022:\\\u0022O9uMx458\\\\\\\/ej9hQp8JTYtMA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228534eeff27a6543cae0fa11afbff5eaf4111f8dd53afc502a5199b9a364bdcbf\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495549 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/569" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-7aO0nsNgwfcIyySM-eFr4EBlvaB-8oZXYJLIEXg7iAc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/569" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8WM8u1puiorWySX-1Vy6xW_Lt-5gk5-18pGr0gLxsQI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/2/555" title="Inflammation-Induced Alternative Pre-mRNA Splicing in Mouse Alveolar Macrophages" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/2/581" title="Optimizing Low-Cost Genotyping and Imputation Strategies for Genomic Selection in Atlantic Salmon" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495549" id="node495549" data-pisa="ggg;10/2/569" data-pisa-master="ggg;g3.119.400873" data-apath="/ggg/10/2/569.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Simulation-Based Evaluation of Three Methods for Local Ancestry Deconvolution of Non-model Crop Species Genomes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0862-784X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aurélien Cottin</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8881-9253" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Benjamin Penaud</span>, <span class="highwire-citation-author" data-delta="2">Jean-Christophe Glaszmann</span>, <span class="highwire-citation-author" data-delta="3">Nabila Yahiaoui</span> and <span class="highwire-citation-author" data-delta="4">Mathieu Gautier</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">569-579;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400873 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495549" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Aurélien Cottin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CIRAD, UMR AGAP, F-34398 Montpellier, France,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>AGAP, Univ. Montpellier, CIRAD, INRAE, Montpellier SupAgro, Montpellier, France, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aur%C3%A9lien%2BCottin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cottin%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAur%25C3%25A9lien%2BCottin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0862-784X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aurélien Cottin</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Benjamin Penaud </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CIRAD, UMR AGAP, F-34398 Montpellier, France,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>AGAP, Univ. Montpellier, CIRAD, INRAE, Montpellier SupAgro, Montpellier, France, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BPenaud%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Penaud%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABenjamin%2BPenaud%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8881-9253" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Benjamin Penaud</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jean-Christophe Glaszmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CIRAD, UMR AGAP, F-34398 Montpellier, France,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>AGAP, Univ. Montpellier, CIRAD, INRAE, Montpellier SupAgro, Montpellier, France, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Christophe%2BGlaszmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Glaszmann%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Christophe%2BGlaszmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Nabila Yahiaoui </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CIRAD, UMR AGAP, F-34398 Montpellier, France,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>AGAP, Univ. Montpellier, CIRAD, INRAE, Montpellier SupAgro, Montpellier, France, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nabila%2BYahiaoui%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yahiaoui%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANabila%2BYahiaoui%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mathieu.gautier@inra.fr" class="" data-icon-position="" data-hide-link-title="0">mathieu.gautier@inra.fr</a>
<a href="mailto:nabila.yahiaoui@cirad.fr" class="" data-icon-position="" data-hide-link-title="0">nabila.yahiaoui@cirad.fr</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Mathieu Gautier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>INRAE, UMR CBGP, F-34988 Montferrier-sur-Lez Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mathieu%2BGautier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gautier%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMathieu%2BGautier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mathieu.gautier@inra.fr" class="" data-icon-position="" data-hide-link-title="0">mathieu.gautier@inra.fr</a>
<a href="mailto:nabila.yahiaoui@cirad.fr" class="" data-icon-position="" data-hide-link-title="0">nabila.yahiaoui@cirad.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/2/569" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495549" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495549/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/2/569.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495549" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495549/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/2/569.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495549" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495549/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/2/569.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495549" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495549/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Hybridizations between species and subspecies represented major steps in the history of many crop species. Such events generally lead to genomes with mosaic patterns of chromosomal segments of various origins that may be assessed by local ancestry inference methods. However, these methods have mainly been developed in the context of human population genetics with implicit assumptions that may not always fit plant models. The purpose of this study was to evaluate the suitability of three state-of-the-art inference methods (SABER, ELAI and WINPOP) for local ancestry inference under scenarios that can be encountered in plant species. For this, we developed an R package to simulate genotyping data under such scenarios. The tested inference methods performed similarly well as far as representatives of source populations were available. As expected, the higher the level of differentiation between ancestral source populations and the lower the number of generations since admixture, the more accurate were the results. Interestingly, the accuracy of the methods was only marginally affected by i) the number of ancestries (up to six tested); ii) the sample design (<em>i.e.</em>, unbalanced representation of source populations); and iii) the reproduction mode (<em>e.g.</em>, selfing, vegetative propagation). If a source population was not represented in the data set, no bias was observed in inference accuracy for regions originating from represented sources and regions from the missing source were assigned differently depending on the methods. Overall, the selected ancestry inference methods may be used for crop plant analysis if all ancestral sources are known.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/local-ancestry-inference" class="hw-term hw-article-keyword hw-article-keyword-local-ancestry-inference" rel="nofollow">local ancestry inference</a></li><li class="kwd"><a href="/keyword/crops" class="hw-term hw-article-keyword hw-article-keyword-crops" rel="nofollow">crops</a></li><li class="kwd"><a href="/keyword/admixture" class="hw-term hw-article-keyword hw-article-keyword-admixture" rel="nofollow">admixture</a></li><li class="kwd"><a href="/keyword/simulation-analysis" class="hw-term hw-article-keyword hw-article-keyword-simulation-analysis" rel="nofollow">simulation analysis</a></li></ul><p id="p-4">Inter-(sub)-specific hybridizations have shaped the genomes of many crop species as for example in wheat (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">El Baidouri <em>et al.</em> 2017</a>), rice (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Zhao <em>et al.</em> 2010</a>), citrus (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wu <em>et al.</em> 2014</a>, <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">2018</a>), banana (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Perrier <em>et al.</em> 2011</a>) or apple (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cornille <em>et al.</em> 2012</a>). They can be a consequence of germplasm transport by humans bringing together plants from related but differentiated species, subspecies or populations, or of gene flow between cultivated plants and neighboring wild relatives (<em>e.g.</em>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Semon <em>et al.</em> 2005</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Perrier <em>et al.</em> 2011</a>). At the genome level, such admixture events can result in a mosaic pattern of chromosomal segments of various origins. The complexity of the mosaic will depend on the demographic characteristics of the populations (<em>e.g.</em>, number of source ancestries and the timing of admixture events). As already demonstrated in other species such as humans, characterizing the genome mosaic may in turn provide valuable insights into the genetic history of populations (<em>e.g.</em>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Moreno-Estrada <em>et al.</em> 2013</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hellenthal <em>et al.</em> 2014</a>), and in the context of domesticated plant species, it may lead to a better understanding of crop domestication and diversification history. It might also help identifying the origin of introgressed variants underlying agricultural traits of interest (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Burgarella <em>et al.</em> 2019</a>) and possibly support breeding strategies to produce improved hybrids.</p><p id="p-5">Over the past twenty years, the development of high-density genotyping and sequencing technologies has promoted the development of accurate approaches to infer genetic ancestry of individuals based on genotyping data. Historically, the first proposed methods aimed at characterizing individual ancestries on a genome-wide scale by estimating the relative contributions of a given number of underlying ancestries. The most popular of these methods are based on unsupervised clustering approaches as introduced by <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Pritchard <em>et al.</em> (2000)</a> in the Structure software, where clusters were interpreted as proxies for ancestries. An extension of the Pritchard’s work by <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Falush <em>et al.</em> (2003)</a>, further allowed to perform local ancestry inference (LAI), <em>i.e.</em>, to infer the ancestral origin at a local chromosome scale in individual genomes. Since then and as reviewed in <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Geza <em>et al.</em> (2019)</a>, more than 20 LAI methods have been published extending, in particular, this pioneering work by scaling up to high throughput genotyping data or by leveraging phased data for more accurate inferences.</p><p id="p-6">Most LAI approaches have been developed in the context of human genetics studies for which their properties have been extensively characterized (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Liu <em>et al.</em> 2013</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Padhukasahasram 2014</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hui <em>et al.</em> 2017</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Geza <em>et al.</em> 2019</a>). Human studies relying on LAI approaches usually aim at assessing admixture between two or three populations and may benefit from a rich amount of genetics resources (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">The International HapMap Consortium 2005</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">The Wellcome Trust Case Control Consortium 2007</a>) with, in particular, dense haplotype data for reference populations and/or admixed samples. For plant species, large-scale sequencing or genotyping resources are also increasingly available for some crops such as rice (The 3,000 rice genomes project 2014) or barley (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Milner <em>et al.</em> 2019</a>). However, for many other species of interest such resources remain scarce which implies that haplotype data may not yet be fully accessible, particularly for non-autogamous species. In addition, ancestries may be multiple as exemplified by the cacao tree (<em>Theobroma cacao</em>) germplasm which is composed of 10 major genetically differentiated groups with up to six-way admixed individuals (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cornejo <em>et al.</em> 2018</a>) or pineapple (<em>Ananas comosus</em>) with up to four-way admixed individuals between cultivar groups and varieties (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chen <em>et al.</em> 2019</a>). Moreover, populations representative of contributing ancestries may be unavailable or represented by only a few individuals. To that respect, the case for banana is particularly illustrative since hybridization events involving well-differentiated <em>Musa acuminata</em> subspecies are predicted to be involved in the formation of some major cultivars (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Perrier <em>et al.</em> 2009</a>, <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">2011</a>). Yet, some of these subspecies are represented only by a few individuals (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Christelová <em>et al.</em> 2017</a>) or some contributors may not be represented in available germplasm (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sardos <em>et al.</em> 2016</a>). Moreover, it remains unclear how some features, regarding reproduction modes (<em>e.g.</em>, selfing or vegetative reproduction) that may be encountered in plant models, may affect the performance of LAI methods. Hence, in fruit crops such as citruses or banana, individuals resulting from inter(sub)specific hybridizations were further multiplied by vegetative propagation (also termed clonal propagation). Thus, they do not form a population but rather a collection of individuals sometimes of different origins with ancestry mosaics of relatively large blocs depending on the number of sexual generations they may have undergone. Datasets of vegetatively propagated individuals may thus be heterogeneous in terms of ancestry structure and in terms of time in generations since admixture events. The number of sexual generations since admixture is a parameter that is often required by LAI programs (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Geza <em>et al.</em> 2019</a>) and it can be difficult to correctly estimate it for plants that have been vegetatively propagated sometimes since hundreds of years. On the other hand, high selfing rates result in increased levels of homozygosity and generally in reduced diversity levels compared to outcrossing species (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Brandvain <em>et al.</em> 2013</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Barrett <em>et al.</em> 2014</a>) while introducing additional levels of structuring of haplotype diversity when selfing and outcrossing populations are analyzed together. Finally, polyploidy that is a feature of many crop plants (<em>e.g.</em>, wheat, sugarcane, potato, and major banana cultivars) is still a complex case to handle for LAI as genotypes are difficult to infer.</p><p id="p-7">The purpose of this study was to evaluate the accuracy of LAI approaches to perform ancestry deconvolution, based on genotyping data simulated under scenarios that can be representative of plant species models. Given the lack of available methods dealing with polyploids, we only considered diploid individuals. Among the 22 LAI approaches recently reviewed by <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Geza <em>et al.</em> (2019)</a>, we chose to evaluate three methods – SABER (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Tang <em>et al.</em> 2006</a>); ELAI (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Guan 2014</a>) and WINPOP (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Paşaniuc <em>et al.</em> 2009</a>) because they do not require prior phasing of the data and they could cope with more than two ancestries. We developed an R package to perform simulations and we focused our evaluation on the influence of i) the level of divergence between the source populations; ii) the number of generations since admixture for the admixed populations; iii) the number of contributing ancestries and their representation in the analyzed data sets; and iv) the mode of reproduction such as selfing (in a source representative population) or vegetative propagation (in the admixed population).</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and Methods</h2><div id="sec-2" class="subsection"><h3>Simulation tool</h3><p id="p-8">We developed an R package (named <em>plmgg</em> for plant-like mosaic genome generator) to simulate individual chromosome-wide genotyping data from an arbitrary number of populations P deriving from S differentiated source populations under scenarios that may include hybridization events and modes of reproduction representative of plant model evolution (<em>i.e.</em>, selfing or vegetative propagation). The simulation approach is depicted in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and consisted of the three following successive steps i) coalescent simulation of a sample of founder chromosomes from S differentiated source populations; ii) forward in-time simulation of P populations deriving from the S source populations with complex demographic scenarios involving various modes of reproduction and admixture; and iii) sampling of individuals from the P populations to generate the genotyping data sets. For coalescent simulations, we relied on the <em>scrm</em> algorithm (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Staab <em>et al.</em> 2015</a>) implemented in the R package <em>coala</em> (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Staab and Metzler 2016</a>) to simulate <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-1.gif"/></noscript></span></span> founder chromosomes (<em>i.e.</em>, haploid (h) individuals) from S predefined source populations (where <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-2.gif"/></noscript></span></span> is the number of chromosomes from source population s). Sources were assumed to derive from a single ancestral population under a pure-drift model of divergence with a star-shaped history. The divergence scenario of the source populations was specified with three parameters: i) the divergence time τ measured in units of <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-3.gif"/></noscript></span></span> (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-4.gif"/></noscript></span></span> where t is the number of generations since the ancestral population and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-5.gif"/></noscript></span></span> is the haploid effective source population size assumed to be the same for the S source populations); ii) the scaled mutation rate <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-6.gif"/></noscript></span></span> (where µ is the mutation rate per site and per generation); and iii) the scaled recombination rate <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-7.gif"/></noscript></span></span> (where r is the recombination rate per site and per generation). For the purpose of this study, τ was varied to control the level of differentiation among the source populations (see below) while both θ and ρ were set equal to 10<sup>−4</sup> (as obtained for instance if one assumes <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-8.gif"/></noscript></span></span> mutation and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-9.gif"/></noscript></span></span> recombination per site and per generation in a population of haploid size <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-10.gif"/></noscript></span></span>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/569/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of the admixture simulation process with plmgg. The coalescent step produces S source populations (here, three sources are represented in blue, red and yellow) that differentiated at &#x3C4;. In the forward step, source-representative populations and admixed populations are generated from sampling of the source populations. Then, each population follows for a number of generations g, a user-defined reproduction process that allows to select and combine reproduction modes (within population random mating, across population random mating, selfing, vegetative reproduction). In the last step, a sampling is performed on each population of the forward step to generate a data set for analysis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1436731740" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Overview of the admixture simulation process with &lt;em&gt;plmgg&lt;/em&gt;. The coalescent step produces &lt;em&gt;S&lt;/em&gt; source populations (here, three sources are represented in blue, red and yellow) that differentiated at &#x3C4;. In the forward step, source-representative populations and admixed populations are generated from sampling of the source populations. Then, each population follows for a number of generations g, a user-defined reproduction process that allows to select and combine reproduction modes (within population random mating, across population random mating, selfing, vegetative reproduction). In the last step, a sampling is performed on each population of the forward step to generate a data set for analysis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/569/F1.medium.gif" width="344" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/2/569/F1.medium.gif" width="344" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/569/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/569/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495615" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-9" class="first-child">Overview of the admixture simulation process with <em>plmgg</em>. The coalescent step produces <em>S</em> source populations (here, three sources are represented in blue, red and yellow) that differentiated at τ. In the forward step, source-representative populations and admixed populations are generated from sampling of the source populations. Then, each population follows for a number of generations g, a user-defined reproduction process that allows to select and combine reproduction modes (within population random mating, across population random mating, selfing, vegetative reproduction). In the last step, a sampling is performed on each population of the forward step to generate a data set for analysis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-10">In the second simulation step, <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-11.gif"/></noscript></span></span> diploid individuals belonging to P populations (where <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-12.gif"/></noscript></span></span> is the number of diploid individuals from population p) were first generated by randomly sampling two chromosomes (without recombination and with replacement) among the <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-13.gif"/></noscript></span></span> founder ones according to the S pre-defined source contributions (in a <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-14.gif"/></noscript></span></span> ancestry proportion matrix). The <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-15.gif"/></noscript></span></span> individuals were further reproduced over G generations, in a forward-in-time process, by specifying for each generation g (in a <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-16.gif"/></noscript></span></span> matrix) the proportions of the four following possible population-specific modes of reproduction: i) within population random mating; ii) across population random mating; iii) selfing; and iv) vegetative reproduction (consisting of randomly reproducing an individual’s chromosome pair from one generation to the next). For sexual reproduction events (<em>i.e.</em>, random mating and selfing), parental gametes were generated by randomly distributing one crossing-over between the two parental chromosomes which amounts to assume a 1 Morgan length chromosome map. In addition, mutations that only affected existing variant positions (switch to the alternate SNP allele) were introduced at each generation at the rate µ defined above (whatever the reproduction mode). In other words, no new segregating sites appeared after the initial coalescent phase of the simulation (first simulation step described above).</p><p id="p-11">In the third and last step of the simulation, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-17.gif"/></noscript></span></span> diploid individuals belonging to the P populations were sampled to generate the data set to be analyzed (where <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-18.gif"/></noscript></span></span> represents the number of genotyped diploid individuals from population p that are randomly sampled with replacement from the corresponding <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-19.gif"/></noscript></span></span> individuals available at generation G). After filtering out monomorphic SNPs, the simulation output consists of i) the genotyping data set in vcf (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Danecek <em>et al.</em> 2011</a>) and <em>plink</em> ped (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Purcell <em>et al.</em> 2007</a>); ii) the true local ancestry for each individual at each SNP position which may be displayed with plotting functions; and iii) summary statistics including pairwise population <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-20.gif"/></noscript></span></span> (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Weir and Cockerham 1984</a>), population heterozygosities and ancestry block sizes.</p></div><div id="sec-3" class="subsection"><h3>Simulated scenarios</h3><p id="p-12">Six scenarios detailed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, each replicated 50 times, were considered for this study. The number of founder chromosomes was set to <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-21.gif"/></noscript></span></span> for each source population (thereby mimicking bottlenecks involved by the domestication process from a small number of wild relatives) and the number of diploid individuals was set to <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-22.gif"/></noscript></span></span> for all the populations (<em>i.e.</em>, the source and admixed populations). Forward simulations were run for <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-23.gif"/></noscript></span></span> generations maintaining <em>S</em> non-admixed populations as source population proxies and two populations originating from an admixture event between three or more ancestries that occurred from <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-24.gif"/></noscript></span></span> to 50 generations ago. Unless otherwise stated, the sampled data set consisted of <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-25.gif"/></noscript></span></span> diploid individuals for each ancestry representative population and <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-26.gif"/></noscript></span></span> individuals for each admixed population. The scenarios were split into three groups to investigate the effect of i) the ancestry representative sample size; ii) the number of sources; and iii) the reproduction modes. First, the DiffGenSam scenarios (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) aimed at evaluating the impact of the amount of differentiation between <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-27.gif"/></noscript></span></span> source populations (with τ varying from 0.05 to 0.40); the number of generations since admixture for the two admixed populations (from <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-28.gif"/></noscript></span></span> to 50); and the sample size (from <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-29.gif"/></noscript></span></span> to 40) of each of the three ancestry-representative populations. Five other scenarios were subsequently considered to address specific points while setting <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-30.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-31.gif"/></noscript></span></span> (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). The SamBal scenario aimed at evaluating the impact of unbalanced sample sizes among three ancestry representative populations (<em>i.e.</em>, two with 20 sampled individuals and the remaining with 2 to 20 sampled individuals). We also considered two scenarios to address the impact of the number of source populations (SrcNum with <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-32.gif"/></noscript></span></span> to 6 source populations equally contributing to the admixed populations) or the presence of a non-sampled source population contributing to the admixed individuals (SrcMiss). In the latter case, <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-33.gif"/></noscript></span></span> source populations were simulated, but only three of them had representatives in the final data set. The contribution of the “missing” source population to the admixed populations varied from 0.05 to 0.15, the three other sources having equal contributions. Finally, the SrcSelf and AdmxVegProp scenarios aimed at investigating the impact of alternative modes of reproduction. In the SrcSelf scenario, we assumed that one of the three source populations was reproducing with a selfing rate varying from 0 (<em>i.e.</em>, no selfing) to 0.99. The AdmxVegProp scenario modeled 10 admixed populations (with <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-34.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-35.gif"/></noscript></span></span> for each admixed population) that switched from exclusive within population random mating to exclusive vegetative propagation <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-36.gif"/></noscript></span></span> generations ago, with <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-37.gif"/></noscript></span></span> varying from 0 (<em>i.e.</em>, no vegetative propagation) to 45 for the 10 populations. Note that the realized number of SNPs (after filtering steps) in the different simulated data sets ranged from <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-38.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-39.gif"/></noscript></span></span> (File S1).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495791/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495791/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of simulation parameters</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>LAI methods</h3><p id="p-16">As mentioned in the introduction, we retained the three LAI methods respectively implemented in the programs SABER, WINPOP and ELAI, that do not require prior phasing of the data and that could cope with more than two ancestries. The two methods SABER (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Tang <em>et al.</em> 2006</a>) and ELAI (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Guan 2014</a>) rely on Hidden Markov Models (HMM) with an explicit modeling of LD, while WINPOP (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Paşaniuc <em>et al.</em> 2009</a>) relies on a model-based LD-free approach.</p><p id="p-17">More precisely, SABER (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Tang <em>et al.</em> 2006</a>) extended the HMM by <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Falush <em>et al.</em> (2003)</a>, to account for background LD existing in ancestral populations by modeling the joint distribution of alleles from consecutive markers within each ancestral population. In addition, SABER allows modeling an arbitrary number of ancestral groups that may admix at different times estimated by a Likelihood Maximization algorithm (<em>saberML</em> function, here initialized with the simulated values). Each individual SNP-specific ancestry estimates were calculated as the posterior probability obtained with the forward-backward algorithm implemented in the <em>pipeline</em> function.</p><p id="p-18">ELAI (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Guan 2014</a>) implements a two layers HMM to model two different scales of LD: the admixture LD (between alleles from different source populations) and a shorter ranged LD existing between alleles within each source population. This is achieved by introducing a local structuring of haplotypes into i) upper-layer clusters that represent different groups (interpreted as source populations); and ii) lower-layer clusters that represent group-specific haplotypes. We here set the number of upper clusters to the number <em>S</em> of simulated sources; the number of lower clusters to <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-52.gif"/></noscript></span></span> as recommended; and the time since admixture (also required by ELAI) to the corresponding simulated one. Model fitting was carried out with the default Expectation-Maximization (EM) algorithm.</p><p id="p-19">WINPOP, included in LAMP &gt;=2.3 (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Sankararaman <em>et al.</em> 2008</a>), is a model-based LD-free method that focuses on ancestry informative markers (AIM) to assign local haplotype blocks to their originating source populations (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Paşaniuc <em>et al.</em> 2009</a>). WINPOP works with variable-size overlapping windows along the chromosomes, and uses a clustering method to assign ancestries in each window, based on estimates of global ancestry proportions. WINPOP was used with the simulated recombination rate and default parameters for the configuration files including a LD pruning cutoff of r<sup>2</sup> = 0.1 and a fraction of sliding window overlap of 20%.</p><p id="p-20">Both WINPOP and SABER required estimates of global ancestry proportion. These were obtained by running the default unsupervised hierarchical clustering algorithm implemented in the ADMIXTURE software (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alexander <em>et al.</em> 2009</a>) setting the number of clusters to <em>S</em>, the number of simulated source populations.</p></div><div id="sec-5" class="subsection"><h3>Evaluation of the performance of LAI methods</h3><p id="p-21">To evaluate the performance of the LAI methods, we defined an accuracy metric α to quantify the overall differences between simulated and inferred local ancestries. Let <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-53.gif"/></noscript></span></span> represent the simulated proportion of ancestry <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-54.gif"/></noscript></span></span> at SNP position <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-55.gif"/></noscript></span></span>; M being the number of SNPs) for individual i (<span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-56.gif"/></noscript></span></span>0, 0.5 or 1 since individuals are diploid). Similarly, let <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-57.gif"/></noscript></span></span> represent the inferred proportion of ancestry. The SNP-specific accuracy for individual i over the S ancestries was then defined as <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-58.gif"/></noscript></span></span>. Note that <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-59.gif"/></noscript></span></span> is bound between 0 (when <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-60.gif"/></noscript></span></span> for all s) and 2 (when <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-61.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-62.gif"/></noscript></span></span> for all s since <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-63.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-64.gif"/></noscript></span></span>), hence the division of this sum by 2 in the definition of the accuracy <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-65.gif"/></noscript></span></span> to keep <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-66.gif"/></noscript></span></span> between 0 and 1. The overall accuracy metric α was defined as the average of <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-67.gif"/></noscript></span></span> over the M markers and the individuals belonging to admixed populations (<em>i.e.</em>, excluding individuals belonging to source representative populations). For the particular case of SrcMiss simulations (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>) in which one source representative population was missing, SNP positions with the corresponding missing ancestry were excluded from the computation of α. According to our definition, α always lies between 0 and 1 (the higher the α value, the more accurate the inference). For calibration purposes, we also computed a minimal value of α as would be obtained by randomly inferred local ancestries under the assumptions of equal contribution of the sources (<em>i.e.</em>, setting <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-68.gif"/></noscript></span></span> for all s). Alternative metrics, such as the coefficient of determination (<em>i.e.</em>, sample correlation coefficient between the inferred and true local ancestries) or mean square errors were also evaluated but were not presented since they lead to the same conclusions regarding the ranking of LAI methods.</p><p id="p-22">We finally evaluated computational efficiency of the different LAI programs by recording for each run of analysis on our computer grid, both the memory usage and the system computing time (<em>max_vmem</em> and <em>ru_wallclock</em>, respectively) available from the Sun Grid Engine user notification.</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-23">The R package <em>plmgg</em> is available at <a href="https://gitlab.southgreen.fr/acottin/plmgg">https://gitlab.southgreen.fr/acottin/plmgg</a>. Scripts used to run simulations, LAI programs and inference comparison are available at <a href="https://gitlab.southgreen.fr/acottin/lai-comparison">https://gitlab.southgreen.fr/acottin/lai-comparison</a>. Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.10266149">https://doi.org/10.25387/g3.10266149</a>.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Source differentiation and number of generations Since admixture</h3><p id="p-24">The impact of the level of differentiation among the sources and the number of generations since admixture on the performance of the three LAI methods was assessed with the DiffGenSam scenarios (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). The analysis of the generated data sets showed that both the level of differentiation among sources and the number of generations since admixture had a strong impact on the performance of LAI methods (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, File S2). Indeed, the accuracy α decreased with an increasing number of generations after admixture (<em>i.e.</em>, when ancestry block sizes became smaller) and with decreasing levels of differentiation between source populations (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Although, the three evaluated LAI approaches performed overall similarly, at the lowest levels of differentiation (<span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-69.gif"/></noscript></span></span>), ELAI and WINPOP were more accurate than SABER for more recent admixture events (<span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-70.gif"/></noscript></span></span>) (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, File S2). In the most favorable situations of high differentiation among the source populations (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-71.gif"/></noscript></span></span>), the accuracy α tended toward 1 (<em>i.e.</em>, no error) with decreasing time since admixture for all the three LAI methods.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/569/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Accuracy of LAI methods with varying levels of differentiation and number of generations (DiffGenSam simulation). The accuracy (&#x3B1;) of the LAI methods (y-axis) is plotted for different levels of differentiation that vary from 0.05 to 0.4 (vertical tiles) and a number of generations after admixture that varies from 5 to 50 (x-axis). The sample size is set to 20 for the sources and the admixed populations. Each dot is the mean value of 50 repetitions of each simulation. Error bars indicate the standard deviation. ELAI, WINPOP and SABER scores are plotted in blue, red and yellow, respectively. Accuracy of random inference (proportion of ancestry fixed at 1/3) is plotted in gray." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1436731740" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Accuracy of LAI methods with varying levels of differentiation and number of generations (DiffGenSam simulation). The accuracy (&#x3B1;) of the LAI methods (y-axis) is plotted for different levels of differentiation that vary from 0.05 to 0.4 (vertical tiles) and a number of generations after admixture that varies from 5 to 50 (x-axis). The sample size is set to 20 for the sources and the admixed populations. Each dot is the mean value of 50 repetitions of each simulation. Error bars indicate the standard deviation. ELAI, WINPOP and SABER scores are plotted in blue, red and yellow, respectively. Accuracy of random inference (proportion of ancestry fixed at 1/3) is plotted in gray.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/569/F2.medium.gif" width="440" height="175"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/2/569/F2.medium.gif" width="440" height="175"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/569/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/569/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495718" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child">Accuracy of LAI methods with varying levels of differentiation and number of generations (DiffGenSam simulation). The accuracy (α) of the LAI methods (y-axis) is plotted for different levels of differentiation that vary from 0.05 to 0.4 (vertical tiles) and a number of generations after admixture that varies from 5 to 50 (x-axis). The sample size is set to 20 for the sources and the admixed populations. Each dot is the mean value of 50 repetitions of each simulation. Error bars indicate the standard deviation. ELAI, WINPOP and SABER scores are plotted in blue, red and yellow, respectively. Accuracy of random inference (proportion of ancestry fixed at 1/3) is plotted in gray.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Number of individuals from the source representative populations</h3><p id="p-26">The impact of the number of sampled representative individuals for each of the three source populations was also evaluated within the DiffGenSam scenarios (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>). As shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, for a given time since admixture (here <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-72.gif"/></noscript></span></span>, see Figure S1 and File S2 for alternative <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-73.gif"/></noscript></span></span> values) decreasing the number of individuals representative of the source populations (<em>e.g.</em>, from <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-74.gif"/></noscript></span></span> as in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a> to <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-75.gif"/></noscript></span></span>) had a higher impact on accuracy for ELAI compared to WINPOP and SABER. Conversely, except for the highest level of differentiation among source populations, increasing the number of source representative individuals improved ELAI performances.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/569/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Accuracy of LAI methods with varying levels of differentiation and source-representative sample size (DiffGenSam simulation). The accuracy (&#x3B1;) of the LAI methods (y-axis) is plotted for different levels of differentiation that vary from 0.05 to 0.4 (vertical tiles) and the size of source-representative sample that varies from 5 to 40 individuals (x-axis). The source sample size is set to 20. Each dot is the mean value of 50 repetitions of each simulation. Error bars indicate the standard deviation. ELAI, WINPOP and SABER scores are plotted in blue, red and yellow, respectively. Accuracy of random inference (proportion of ancestry fixed at 1/3) is plotted in gray." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1436731740" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Accuracy of LAI methods with varying levels of differentiation and source-representative sample size (DiffGenSam simulation). The accuracy (&#x3B1;) of the LAI methods (y-axis) is plotted for different levels of differentiation that vary from 0.05 to 0.4 (vertical tiles) and the size of source-representative sample that varies from 5 to 40 individuals (x-axis). The source sample size is set to 20. Each dot is the mean value of 50 repetitions of each simulation. Error bars indicate the standard deviation. ELAI, WINPOP and SABER scores are plotted in blue, red and yellow, respectively. Accuracy of random inference (proportion of ancestry fixed at 1/3) is plotted in gray.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/569/F3.medium.gif" width="440" height="174"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/2/569/F3.medium.gif" width="440" height="174"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/569/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/569/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495583" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">Accuracy of LAI methods with varying levels of differentiation and source-representative sample size (DiffGenSam simulation). The accuracy (α) of the LAI methods (y-axis) is plotted for different levels of differentiation that vary from 0.05 to 0.4 (vertical tiles) and the size of source-representative sample that varies from 5 to 40 individuals (x-axis). The source sample size is set to 20. Each dot is the mean value of 50 repetitions of each simulation. Error bars indicate the standard deviation. ELAI, WINPOP and SABER scores are plotted in blue, red and yellow, respectively. Accuracy of random inference (proportion of ancestry fixed at 1/3) is plotted in gray.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">We evaluated the robustness of the three LAI methods to unbalanced sample sizes of source-representative populations by analyzing data sets simulated under the SamBal scenarios where the number of samples was reduced for one of the three sources (Figure S2, File S3). The accuracy of ELAI was lower than both the WINPOP and SABER when sampling was reduced for the third source (<em>e.g.</em>, for 2 representatives instead of 20, accuracy of 0.720 for ELAI <em>vs.</em> 0.815 for WINPOP and 0.806 for SABER, File S3) but it increased when sampling was more balanced reaching accuracy of 0.870 for a completely balanced setting. For WINPOP and SABER the accuracy was only marginally improved, reaching up to 0.850.</p><p id="p-29">According to the results above, to allow better discrimination of the LAI methods in relatively challenging conditions, we chose to perform the remaining evaluations with a number of generations after admixture set to 50, a level of differentiation among sources of <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-76.gif"/></noscript></span></span> and 20 individuals per source representative population.</p></div><div id="sec-10" class="subsection"><h3>Number of source populations and absence of source representative individuals</h3><p id="p-30">With the SrcNum scenarios (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>), data sets were simulated for admixture events involving up to six source populations. The analysis of LAI results showed that the accuracy decreased with increasing numbers of sources for all three evaluated LAI approaches (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). However, the magnitude of decrease in accuracy from <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-77.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-78.gif"/></noscript></span></span> source populations remained moderate with rates equal to 2.7%, 8.7% and 11% for ELAI, WINPOP and SABER respectively (to be compared with the 45% decrease observed with the random inference) (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). We further assessed the impact of the absence of individuals from one out of four source representative populations using data sets simulated under the SrcMiss scenario (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>). Different proportions of this unrepresented source to admixed populations were tested (5, 10 and 15%) and accuracy was measured by excluding regions contributed by the missing source population. As shown in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>, the accuracy for all methods was stable in regions without the unknown ancestry, whatever the global proportions (at a data set level) of unknown ancestry. This suggested that the absence of individuals from a source representative population in the analyzed data sets did not introduce biases in inferring local ancestries of the represented source populations. Visual inspection of local ancestries inferred in regions containing the missing ancestry did not reveal any particular pattern (<em>e.g.</em>, like a higher switching rate among the other represented ancestries). As an example, <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> shows the inferred local ancestry mosaic of one individual from a simulated data set with a 10% contribution of the unrepresented source population. In general, the chromosomal regions originating from the missing source population tended to be assigned to different represented ancestries, the assignation also varying according to the LAI method used.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495763/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495763/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Accuracy of LAI methods with varying number of source populations (SrcNum scenario)</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495514/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495514/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Accuracy of LAI methods with varying proportions of an unknown source population in admixed populations (SrcMiss scenario)</span><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/569/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="LAI results for a simulated admixed individual of the SrcMiss simulation. The x-axis indicates simulated SNPs positions and the y-axis represents the stacked ancestry proportions (between 0 and 1). (A) represents the true ancestries. (B), (C) and (D) represent the inference from ELAI, WINPOP and SABER, respectively. The three known sources are shown in blue, red and yellow and the unknown source in gray." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1436731740" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;LAI results for a simulated admixed individual of the SrcMiss simulation. The x-axis indicates simulated SNPs positions and the y-axis represents the stacked ancestry proportions (between 0 and 1). (&lt;strong&gt;A&lt;/strong&gt;) represents the true ancestries. (&lt;strong&gt;B&lt;/strong&gt;), (&lt;strong&gt;C&lt;/strong&gt;) and (&lt;strong&gt;D&lt;/strong&gt;) represent the inference from ELAI, WINPOP and SABER, respectively. The three known sources are shown in blue, red and yellow and the unknown source in gray.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/569/F4.medium.gif" width="440" height="295"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/10/2/569/F4.medium.gif" width="440" height="295"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/569/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/569/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495608" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-33" class="first-child">LAI results for a simulated admixed individual of the SrcMiss simulation. The x-axis indicates simulated SNPs positions and the y-axis represents the stacked ancestry proportions (between 0 and 1). (<strong>A</strong>) represents the true ancestries. (<strong>B</strong>), (<strong>C</strong>) and (<strong>D</strong>) represent the inference from ELAI, WINPOP and SABER, respectively. The three known sources are shown in blue, red and yellow and the unknown source in gray.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Selfing and vegetative propagation</h3><p id="p-34"><a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> gives the accuracy of the different LAI approaches on data sets simulated under the SrcSelf simulation (<a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table 1</a>) in which the third source representative population reproduced with a varying extent of selfing. For the three LAI approaches, increased proportions of selfing in the third source representative population resulted in a decrease of accuracy, to a small extent. Indeed the decrease in accuracy between rates of selfing of 0 and 99% was equal to 0.92%, 6.6% and 3.5% for ELAI, WINPOP and SABER, respectively.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495687/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495687/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Accuracy of LAI methods with varying proportions of selfing in a source-representative population (SrcSelf simulation)</span><div class="sb-div caption-clear"></div></div></div><p id="p-36"><a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> plots the accuracies of LAI approaches estimated on data sets simulated under the AdmxVegProp scenarios (<a id="xref-table-wrap-1-10" class="xref-table" href="#T1">Table 1</a>) consisting of individuals from three source representative populations and 10 admixed populations that switched to an exclusive vegetative propagation mode <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-85.gif"/></noscript></span></span> generations ago (<span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-86.gif"/></noscript></span></span> varying from 0 to 45 for the different populations). Note that, the larger <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-87.gif"/></noscript></span></span>, the larger the ancestry block sizes (since the smaller the number of post-admixture recombinations). For both WINPOP and ELAI based inference, the accuracy increased for increasing values of <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-88.gif"/></noscript></span></span> as expected given larger ancestry block sizes. However, the accuracy of SABER, being very similar for individuals with <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-89.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-90.gif"/></noscript></span></span>, was mostly not influenced by <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-91.gif"/></noscript></span></span><sub>,</sub> although a slight decrease was observed at <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-92.gif"/></noscript></span></span>. As this decrease appeared for higher numbers of generations of vegetative propagation, it may be linked to the fact that SABER performs its own estimation of time since admixture. To investigate this, a second run of SABER was performed without using the time since admixture estimation method (<em>saberML</em> function), but with a time since admixture fixed at <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-93.gif"/></noscript></span></span> as for WINPOP and ELAI (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). SABER accuracy was found higher with this fixed number of generations but a decrease at <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-94.gif"/></noscript></span></span> was still observed.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/569/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Accuracy of LAI methods with varying number of generations of vegetative propagation (AdmxVegProp simulation). The accuracy of the LAI methods (y-axis) is plotted for different numbers of generations of vegetative propagation after admixture () that vary from 0 to 45 (x-axis). The source sample size is set to 20, the differentiation set to 0.2 and the total number of generations after the admixture event set to 50. Each dot is the mean value of 50 repetitions of each simulation. Error bars indicate the standard deviation. ELAI, WINPOP and SABER scores are plotted in blue, red and yellow, respectively. SABER score with fixed number of generations after admixture is plotted in darker yellow. Accuracy of random inference (proportion of ancestry fixed at 1/3) is plotted in gray." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1436731740" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Accuracy of LAI methods with varying number of generations of vegetative propagation (AdmxVegProp simulation). The accuracy of the LAI methods (y-axis) is plotted for different numbers of generations of vegetative propagation after admixture (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-95&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) that vary from 0 to 45 (x-axis). The source sample size is set to 20, the differentiation set to 0.2 and the total number of generations after the admixture event set to 50. Each dot is the mean value of 50 repetitions of each simulation. Error bars indicate the standard deviation. ELAI, WINPOP and SABER scores are plotted in blue, red and yellow, respectively. SABER score with fixed number of generations after admixture is plotted in darker yellow. Accuracy of random inference (proportion of ancestry fixed at 1/3) is plotted in gray.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/569/F5.medium.gif" width="440" height="335"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/10/2/569/F5.medium.gif" width="440" height="335"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/569/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/569/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495669" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-37" class="first-child">Accuracy of LAI methods with varying number of generations of vegetative propagation (AdmxVegProp simulation). The accuracy of the LAI methods (y-axis) is plotted for different numbers of generations of vegetative propagation after admixture (<span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/F5/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/F5/embed/mml-math-95.gif"/></noscript></span></span>) that vary from 0 to 45 (x-axis). The source sample size is set to 20, the differentiation set to 0.2 and the total number of generations after the admixture event set to 50. Each dot is the mean value of 50 repetitions of each simulation. Error bars indicate the standard deviation. ELAI, WINPOP and SABER scores are plotted in blue, red and yellow, respectively. SABER score with fixed number of generations after admixture is plotted in darker yellow. Accuracy of random inference (proportion of ancestry fixed at 1/3) is plotted in gray.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Computational performances of LAI methods</h3><p id="p-38">Computational performance was measured for all the analyses performed on the simulated data sets. For the DiffGenSam scenario (<span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-96.gif"/></noscript></span></span> sources), memory consumption for the different methods ranged from 0.5Gb to 2Gb of RAM and was not highly variable across scenario variations (Figure S3). WINPOP was the fastest of the three LAI methods with a mean running time ranging from 20 s to 60 s in the DiffGenSam data sets (with three source representative populations) while SABER runs lasted between 30 min and 60 min and ELAI runs between 50 min and 4h (Figure S4). The analysis of data sets simulated under the SrcNum scenarios showed that the number of sources had the most significant impact on resource consumption (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>), particularly for ELAI that used up to 10GB and 30h with <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-97.gif"/></noscript></span></span> sources. This corresponded to a 20-fold memory and a 38-fold computing time increases as compared with <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-98.gif"/></noscript></span></span> sources, (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>) whereas the overall number of individuals (70 <em>vs.</em> 55) and the number of SNPs remained similar. Although memory usage increased steadily for WINPOP and SABER (from 0.7GB to 3.75GB), the computing time remained low for WINPOP (20s to 3min20s) and intermediary (up to 5h) for SABER.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/569/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Memory usage and computation time of LAI methods with varying number of sources (SrcNum simulation). (A) Memory usage of LAI methods in giga bytes (y-axis). (B) Wallclock time of the LAI methods in minutes (y-axis). The x-axis represents the number of simulated source populations in the SrcNum scenario. Each dot is the mean value of 50 repetitions of each simulations. Error bars indicate the standard deviation. ELAI, LAMP and SABER performance in memory (A) and time (B) are plotted in blue, red and yellow, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1436731740" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Memory usage and computation time of LAI methods with varying number of sources (SrcNum simulation). (&lt;strong&gt;A&lt;/strong&gt;) Memory usage of LAI methods in giga bytes (y-axis). (&lt;strong&gt;B&lt;/strong&gt;) Wallclock time of the LAI methods in minutes (y-axis). The x-axis represents the number of simulated source populations in the SrcNum scenario. Each dot is the mean value of 50 repetitions of each simulations. Error bars indicate the standard deviation. ELAI, LAMP and SABER performance in memory (&lt;strong&gt;A&lt;/strong&gt;) and time (&lt;strong&gt;B&lt;/strong&gt;) are plotted in blue, red and yellow, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/569/F6.medium.gif" width="440" height="159"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/10/2/569/F6.medium.gif" width="440" height="159"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/569/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/569/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495581" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-39" class="first-child">Memory usage and computation time of LAI methods with varying number of sources (SrcNum simulation). (<strong>A</strong>) Memory usage of LAI methods in giga bytes (y-axis). (<strong>B</strong>) Wallclock time of the LAI methods in minutes (y-axis). The x-axis represents the number of simulated source populations in the SrcNum scenario. Each dot is the mean value of 50 repetitions of each simulations. Error bars indicate the standard deviation. ELAI, LAMP and SABER performance in memory (<strong>A</strong>) and time (<strong>B</strong>) are plotted in blue, red and yellow, respectively.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-40">The approaches evaluated in this study (implemented in the SABER, WINPOP and ELAI programs) were mostly developed for applications in human populations. The purpose of our study was to carry out a detailed evaluation of the accuracy of these three LAI approaches on data simulated under scenarios with features that may be encountered in studies of plant domestication or diversification involving admixture. For instance, the three methods we considered here were originally tested on data simulated by resampling haplotypes from two to three human populations in scenarios consisting of two-way or three-way admixture with up to a few tens generations post-admixture and including from 100 to 200 genotyped individuals per source representative populations in the analysis (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Tang <em>et al.</em> 2006</a>; <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Paşaniuc <em>et al.</em> 2009</a>; <a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Guan 2014</a>).</p><p id="p-41">We developed an R package (<em>plmgg</em>) to simulate genotyping data under a wider range of scenarios and sample designs that include plant-like features. Even if this simulator has some limitations (it does not simulate recombination hotspots, multiple recombination per chromosomes nor selection), it allowed us to assess the influence on LAI accuracy of the level of differentiation, of multiway admixture with up to six ancestries and of limited sampling of source populations. In addition, the impact of two plant reproduction modes was also evaluated: selfing (in a source representative population) and vegetative propagation (in the admixed population).</p><p id="p-42">Overall, the two main factors that contributed to improve accuracy of all the three tested LAI approaches were the level of divergence between source populations (the higher, the better) and the number of generations since admixture (the smaller, the better) which was not surprising given their expected influence on the complexity of genome mosaics. Indeed, due to both mutations and recombination, divergence between source populations leads to increased differences among their originating haplotypes that facilitates their discrimination. Similarly, increasing the number of generations since admixture, results in shorter ancestral chromosome segment tracks, which are then more difficult to identify. However, it should be noticed that in scenarios with the most extreme level of differentiation among the source populations we considered here (<span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-99.gif"/></noscript></span></span> which corresponds to a <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-100.gif"/></noscript></span></span> in the pure-drift model of divergence we simulated), LAI accuracy remained acceptable even for the oldest admixture events (50 generations since admixture). In Citrus, average <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-101.gif"/></noscript></span></span> values of 0.44 up to 0.85 were found between the four ancestral taxa depending on studies or marker types (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Curk <em>et al.</em> 2015</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">2016</a>). In the cacao tree or in pineapple, pairwise <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-102.gif"/></noscript></span></span> ranges between genetic groups were of 0.16 to 0.65 (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Cornejo <em>et al.</em> 2018</a>) and 0.28 to 0.94 (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Chen <em>et al.</em> 2019</a>), respectively. The lowest part of these ranges are covered in our simulations and higher values of <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/569/embed/mml-math-103.gif"/></noscript></span></span> will actually facilitate LAI even with older admixture events. For closely related source populations, LAI approaches only performed well if admixture events were very recent (<em>i.e.</em>, below 10 generations). The three methods tested behaved roughly similarly, although WINPOP tended to be superior when source populations were more closely related whereas for more differentiated sources and between 20 and 50 generations after admixture, ELAI tended to be more accurate. This result was consistent with the WINPOP paper (<a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Paşaniuc <em>et al.</em> 2009</a>) that showed that WINPOP performed well with closely related populations, with its improved modeling of recombination and adaptive window length that takes into account local genetic distances between ancestral populations. As for ELAI, its two-layer HMM model helps resolving short ancestry segments that can result from increasing generation numbers after admixture (<a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Guan 2014</a>). In practice, differentiation among the source populations may be estimated with genotyping data available in the source representative individuals even when few individuals are available (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Willing <em>et al.</em> 2012</a>).</p><p id="p-43">The timing of admixture events, required by both ELAI and WINPOP, may also represent in practice a parameter difficult to provide, especially for populations reproducing with vegetative propagation. Also, as we fixed this parameter to its true simulated value when running ELAI and WINPOP programs, our evaluation of these two methods may be overly optimistic. Yet, results obtained on the AdmxVegProp scenarios that include several generations of vegetative propagations suggests that both ELAI and WINPOP remain robust to (at least) upwardly biased estimates of the timing of admixture. In practice however, it may be valuable to check the sensitivity of the results obtained with these methods to a biologically sound range of (exponentially) varying values for this parameter. On the other hand, the timing of admixture events may also be estimated as proposed in the SABER framework. We nevertheless observed that in our settings the SABER estimations were inaccurate (see Figure S5) which suggests in turn that LAI relying on SABER is also robust to biased estimates of the timing of admixture events. Other approaches may thus be preferable to that end, for example those modeling LD decay on a whole genome basis providing sampling allows it (<em>e.g.</em>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Loh <em>et al.</em> 2013</a>). Recently, Chen <em>et al.</em> (2019) estimated an average of 37 generations since the onset of admixture events for 22 (primarily) vegetatively propagated pineapple (var. <em>comosus</em>) hybrids, with a range of 21-55 generations.</p><p id="p-44">Interestingly, we found that selfing (in a source representative population) or vegetative propagation (in the admixed population) had only a small impact on the inference accuracy. Selfing in a source population is of particular interest for banana as one of the <em>M. acuminata</em> subspecies contributing to banana hybrids is predicted to be frequently self-pollinated (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Simmonds 1962</a>). Reproduction by vegetative propagation is favored for many fruit tree crops (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Miller and Gross 2011</a>). Depending on the number of generations of sexual reproduction after admixture, vegetative propagation of admixed individuals can result in different levels of fragmentation of the mosaic structures. As mentioned above, this type of setting, with an overestimation of the generation number parameter had a minor impact on both ELAI and WINPOP, but a more notable impact on SABER inference for individuals where the overestimation was the highest.</p><p id="p-45">Increasing the number of source populations (up to six tested) only marginally affected the accuracy of the tested LAI methods, particularly for ELAI. Nevertheless, this also increased the computational burden that became substantial for the ELAI program, presumably due to the higher number of model parameters. <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Hui <em>et al.</em> (2017)</a> developed a tool (LAIT) to run four LAI methods including WINPOP and ELAI on a data set. They used LAIT to compare LAI methods on two-way and three-way admixture, and showed that ELAI performed better than WINPOP at the cost of increased resources consumption, which is consistent with our results.</p><p id="p-46">Our results also showed that LAI methods perform similarly well for moderate to high levels of differentiation among source populations, even when the number of source representative individuals is small, which may have favorable practical consequences as it is not always possible to have access to large numbers of source representatives. Yet the three different methods behaved differently given an unbalanced data set, with a minor impact on SABER and WINPOP compared to ELAI. This may be explained by the two layers models of ELAI that ties haplotypes structure to ancestries, so that clustering will be hindered by low haplotypic variability. More generally, and in practice, assessing the number of source populations and assigning individuals to them might not be an easy task. Unsupervised clustering approaches (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Pritchard <em>et al.</em> 2000</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Alexander <em>et al.</em> 2009</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Frichot <em>et al.</em> 2014</a>) might be viewed as a reference choice (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Stift <em>et al.</em> 2019</a>) provided the source populations are differentiated enough and evenly represented in the data set (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Puechmaille 2016</a>). The Chromopainter method (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lawson <em>et al.</em> 2012</a>) allows to determine ancestry sources without individuals assigned as source-representatives, provided that phased data are available.</p><p id="p-47">A most critical issue regarding LAI performances was the absence of representative individuals for a given source. The results obtained on the SrcMiss simulations showed no particular bias in attributing the missing population to known ancestries. This result may come from the fact that in our simulation the population tree between the four sources is star shaped. In practice, a star shaped tree is uncommon, one known population may be closely related to the missing population and bias cannot be excluded in this case. Some empirical and specific sampling procedures have been proposed to circumvent the absence of source representatives, in the case of large proportions of unrepresented ancestry in admixed populations (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Zhou <em>et al.</em> 2016</a>). Recently, a promising and more generic alternative has been developed in the MOSAIC model of <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Salter-Townshend and Myers (2019)</a> for haplotype data, which allows for extracting information on source populations from related (and possibly admixed) individuals. Yet, phased data that we purposely kept out of consideration may not be accessible for many crop species. Moreover, it has been shown that switch errors that can occur with statistical phasing (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Scheet and Stephens 2006</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Browning and Browning 2011</a>) reduce LAI accuracy (<em>e.g.</em>, <a id="xref-ref-19-6" class="xref-bibr" href="#ref-19">Guan 2014</a>). However, haplotype-based LAI approaches such as RFMix (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Maples <em>et al.</em> 2013</a>), LOTER (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Dias-Alves <em>et al.</em> 2018</a>) and MOSAIC (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Salter-Townshend and Myers 2019</a>) that included switch error modeling demonstrated that, if properly modeled, inaccurate phasing is becoming less of a threat for LAI accuracy.</p><p id="p-48">LAI on phased data may also be particularly well suited to deal with polyploidy, ploidy being highly variable in crop species (<em>e.g.</em>, pineapple 2x, cacao tree 2x, banana 2x and 3x, citrus up to 4x, sugarcane up to 12x) although statistical phasing might be challenging. Alternatively, HMM-based methods such as those proposed by (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Corbett-Detig and Nielsen 2017</a>) for Pool-Seq data may also be of value.</p><p id="p-49">The evaluation of LAI methods accuracy and performance with the <em>plmgg</em> R package, showed that LAI methods are usable in the scope of crops genetics, with caution particularly in case of a missing source population. The software WINPOP seems suited when source populations are close and admixture events recent. ELAI could be particularly adapted for well differentiated and relatively well represented sources, in case of selfing in source populations, for vegetative propagation settings, and multiway admixture although for the latter, computational performance might be a limiting factor. Other parameters more specific to different plant/crop models might be evaluated using the <em>plmgg</em> package.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-50">This work was supported by funding from the Agropolis Fondation ‘GenomeHarvest’ project (ID 1504-006) through the French ‘Investissements d’avenir’ program (Labex Agro:ANR-10-LABX-0001-01) and by CIRAD. We thank Franck Curk, Gilles Costantino, Benjamin Heuclin, Joao D Santos, Guillaume Martin for fruitful discussions and the PIs of the ‘GenomeHarvest’ project, Manuel Ruiz and Angélique D’Hont. We also thank the two anonymous reviewers for their constructive comments. This work was also supported by the CIRAD—UMR AGAP HPC Data Center of the South Green Bioinformatics platform (<a href="http://www.southgreen.fr/">http://www.southgreen.fr/</a>).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-8"><p id="p-51">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.10266149">https://doi.org/10.25387/g3.10266149</a>.</p></li><li class="fn" id="fn-9"><p id="p-52"><em>Communicating editor: A. Kern</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-11-08"><span class="received-label">Received </span>November 8, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-12-12"><span class="accepted-label">Accepted </span>December 12, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Cottin <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.1" data-doi="10.1101/gr.094052.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>,  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Novembre</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Lange</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast model-based estimation of ancestry in unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1655</span>–<span class="cit-lpage">1664</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.094052.109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.094052.109%26rft_id%253Dinfo%253Apmid%252F19648217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTY1NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvNTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.2" data-doi="10.1098/rstb.2013.0344"><div class="cit-metadata"><cite>Barrett, S. C. H., R. Arunkumar, and S. I. Wright, 2014 The demography and population genomics of evolutionary transitions to self-fertilization in plants. Philos. Trans. R. Soc. B Biol. Sci. 369: 20130344.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rstb.2013.0344</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2013.0344%26rft_id%253Dinfo%253Apmid%252F24958918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2013.0344&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24958918&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.3" data-doi="10.1371/journal.pgen.1003754"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brandvain</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Slotte</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Hazzouri</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. I.</span> <span class="cit-name-surname">Wright</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Coop</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic Identification of Founding Haplotypes Reveals the History of the Selfing Species Capsella rubella.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e1003754</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003754</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003754%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003754%26rft_id%253Dinfo%253Apmid%252F24068948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24068948&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.4" data-doi="10.1038/nrg3054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>,  <span class="cit-name-given-names">S. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. L.</span> <span class="cit-name-surname">Browning</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Haplotype phasing: existing methods and new developments.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">703</span>–<span class="cit-lpage">714</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3054</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DS.%2BR.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253D703%26rft.epage%253D714%26rft.atitle%253DHaplotype%2Bphasing%253A%2Bexisting%2Bmethods%2Band%2Bnew%2Bdevelopments.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3054%26rft_id%253Dinfo%253Apmid%252F21921926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21921926&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.5" data-doi="10.3389/fpls.2019.00004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgarella</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Barnaud</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Kane</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Jankowski</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Scarcelli</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Adaptive Introgression: An Untapped Evolutionary Mechanism for Crop Adaptation.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">4</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2019.00004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D10%26rft.spage%253D4%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2019.00004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2019.00004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.6" data-doi="10.1038/s41588-019-0506-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">L.-Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">VanBuren</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Paris</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Zhou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">The bracteatus pineapple genome and domestication of clonally propagated crops.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">1549</span>–<span class="cit-lpage">1558</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41588-019-0506-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D51%26rft.spage%253D1549%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-019-0506-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41588-019-0506-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.7" data-doi="10.1007/s10531-016-1273-9."><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christelová</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">De Langhe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Hřibová</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Čížková</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sardos</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Molecular and cytological characterization of the global <em>Musa</em> germplasm collection provides insights into the treasure of banana diversity.</span> <abbr class="cit-jnl-abbrev">Biodivers. Conserv.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">801</span>–<span class="cit-lpage">824</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10531-016-1273-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiodivers.%2BConserv.%26rft.volume%253D26%26rft.spage%253D801%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10531-016-1273-9.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10531-016-1273-9.&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.8" data-doi="10.1371/journal.pgen.1006529"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corbett-Detig</span>,  <span class="cit-name-given-names">R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Nielsen</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A Hidden Markov Model Approach for Simultaneously Estimating Local Ancestry and Admixture Time Using Next Generation Sequence Data in Samples of Arbitrary Ploidy.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-elocation-id">e1006529</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1006529</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D13%26rft.spage%253De1006529%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006529&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.9"><div class="cit-metadata unstructured">Cornejo, O. E., M.-C. Yee, V. Dominguez, M. Andrews, A. Sockell <em>et al.</em>, 2018 Population genomic analyses of the chocolate tree, Theobroma cacao L., provide insights into its domestication process. Commun. Biol. 1: 167.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.10" data-doi="10.1371/journal.pgen.1002703"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cornille</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Gladieux</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J. M.</span> <span class="cit-name-surname">Smulders</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Roldán-Ruiz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Laurens</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">New Insight into the History of Domesticated Apple: Secondary Contribution of the European Wild Apple to the Genome of Cultivated Varieties.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e1002703</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002703</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DCornille%26rft.auinit1%253DA.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253De1002703%26rft.epage%253De1002703%26rft.atitle%253DNew%2Binsight%2Binto%2Bthe%2Bhistory%2Bof%2Bdomesticated%2Bapple%253A%2Bsecondary%2Bcontribution%2Bof%2Bthe%2BEuropean%2Bwild%2Bapple%2Bto%2Bthe%2Bgenome%2Bof%2Bcultivated%2Bvarieties.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002703%26rft_id%253Dinfo%253Apmid%252F22589740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002703&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22589740&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.11" data-doi="10.1371/journal.pone.0125628"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Curk</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Ancillo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Ollitrault</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Perrier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-P.</span> <span class="cit-name-surname">Jacquemoud-Collet</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Nuclear Species-Diagnostic SNP Markers Mined from 454 Amplicon Sequencing Reveal Admixture Genomic Structure of Modern Citrus Varieties.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-elocation-id">e0125628</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0125628</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0125628%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0125628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0125628&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.12" data-doi="10.1093/aob/mcw005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Curk</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Ollitrault</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Garcia-Lor</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Luro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Navarro</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Phylogenetic origin of limes and lemons revealed by cytoplasmic and nuclear markers.</span> <abbr class="cit-jnl-abbrev">Ann. Bot.</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">583</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/aob/mcw005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcw005%26rft_id%253Dinfo%253Apmid%252F26944784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/aob/mcw005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26944784&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.13" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Auton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Abecasis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Albers</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Banks</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr330</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653522&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.14" data-doi="10.1093/molbev/msy126"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dias-Alves</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Mairal</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. G. B.</span> <span class="cit-name-surname">Blum</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Loter: A Software Package to Infer Local Ancestry for a Wide Range of Species.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">2318</span>–<span class="cit-lpage">2326</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msy126</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D35%26rft.spage%253D2318%26rft.atitle%253DLOTER%253A%2BA%2BSOFTWARE%2BPACKAGE%2BTO%2BINFER%2BLOCAL%2BANCESTRY%2BFOR%2BA%2BWIDE%2BRANGE%2BOF%2BSPECIES.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsy126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msy126&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.15" data-doi="10.1111/nph.14113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">El Baidouri</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Murat</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Veyssiere</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Molinier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Flores</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Reconciling the evolutionary origin of bread wheat (<em>Triticum aestivum</em>).</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">213</span>: <span class="cit-fpage">1477</span>–<span class="cit-lpage">1486</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.14113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.14113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.14113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.16" data-doi="PubMed PMID: 88"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falush</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Stephens</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Pritchard</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">1567</span>–<span class="cit-lpage">1587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFalush%26rft.auinit1%253DD.%26rft.volume%253D164%26rft.issue%253D4%26rft.spage%253D1567%26rft.epage%253D1587%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%253A%2BLinked%2BLoci%2Band%2BCorrelated%2BAllele%2BFrequencies%26rft_id%253Dinfo%253Adoi%252FPubMed%2BPMID%253A%2B88%26rft_id%253Dinfo%253Apmid%252F12930761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NC80LzE1NjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzU2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.17" data-doi="10.1534/genetics.113.160572"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frichot</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Mathieu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Trouillon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Bouchard</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">François</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Fast and Efficient Estimation of Individual Ancestry Coefficients.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">973</span>–<span class="cit-lpage">983</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.160572</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.160572%26rft_id%253Dinfo%253Apmid%252F24496008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzQvOTczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi81NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.18" data-doi="10.1093/bib/bby044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Geza</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Mugo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. J.</span> <span class="cit-name-surname">Mulder</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wonkam</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. R.</span> <span class="cit-name-surname">Chimusa</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">A comprehensive survey of models for dissecting local ancestry deconvolution in human genome.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1709</span>–<span class="cit-lpage">1724</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bib/bby044</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft.volume%253D20%26rft.spage%253D1709%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbby044%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bby044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.19" data-doi="10.1534/genetics.113.160697"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>,  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Detecting structure of haplotypes and local ancestry.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">625</span>–<span class="cit-lpage">642</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.160697</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.160697%26rft_id%253Dinfo%253Apmid%252F24388880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzMvNjI1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi81NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.20" data-doi="10.1126/science.1243518"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hellenthal</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. B. J.</span> <span class="cit-name-surname">Busby</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Band</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. F.</span> <span class="cit-name-surname">Wilson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Capelli</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A Genetic Atlas of Human Admixture History.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">343</span>: <span class="cit-fpage">747</span>–<span class="cit-lpage">751</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1243518</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHellenthal%26rft.auinit1%253DG.%26rft.volume%253D343%26rft.issue%253D6172%26rft.spage%253D747%26rft.epage%253D751%26rft.atitle%253DA%2BGenetic%2BAtlas%2Bof%2BHuman%2BAdmixture%2BHistory%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1243518%26rft_id%253Dinfo%253Apmid%252F24531965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDMvNjE3Mi83NDciO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzU2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.21" data-doi="10.1186/s12863-017-0546-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hui</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Fang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Lin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Duan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Li</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">LAIT: a local ancestry inference toolkit.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">83</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12863-017-0546-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D18%26rft.spage%253D83%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12863-017-0546-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12863-017-0546-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.569.22" data-doi="10.1371/journal.pgen.1002453"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawson</span>,  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Hellenthal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Myers</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Falush</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Inference of population structure using dense haplotype data.</span> <span class="cit-source">PLoS Genet.</span> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002453</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002453</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLawson%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253De1002453%26rft.epage%253De1002453%26rft.atitle%253DInference%2Bof%2Bpopulation%2Bstructure%2Busing%2Bdense%2Bhaplotype%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002453%26rft_id%253Dinfo%253Apmid%252F22291602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002453&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22291602&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.23" data-doi="10.1186/1479-7364-7-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Nyunoya</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Leng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Belinsky</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Tesfaigzi</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Softwares and methods for estimating genetic ancestry in human populations.</span> <abbr class="cit-jnl-abbrev">Hum. Genomics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1479-7364-7-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BGenomics%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1479-7364-7-1%26rft_id%253Dinfo%253Apmid%252F23289408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1479-7364-7-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23289408&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.24" data-doi="10.1534/genetics.112.147330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loh</span>,  <span class="cit-name-given-names">P.-R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lipson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Patterson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Moorjani</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Pickrell</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Inferring Admixture Histories of Human Populations Using Linkage Disequilibrium.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">1233</span>–<span class="cit-lpage">1254</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.147330</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147330%26rft_id%253Dinfo%253Apmid%252F23410830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5My80LzEyMzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzU2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.25" data-doi="10.1016/j.ajhg.2013.06.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maples</span>,  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Gravel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. E.</span> <span class="cit-name-surname">Kenny</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. D.</span> <span class="cit-name-surname">Bustamante</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">RFMix: A Discriminative Modeling Approach for Rapid and Robust Local-Ancestry Inference.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">278</span>–<span class="cit-lpage">288</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2013.06.020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D93%26rft.spage%253D278%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2013.06.020%26rft_id%253Dinfo%253Apmid%252F23910464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2013.06.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23910464&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.26" data-doi="10.3732/ajb.1000522"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">A. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. L.</span> <span class="cit-name-surname">Gross</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">From forest to field: Perennial fruit crop domestication.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">1389</span>–<span class="cit-lpage">1414</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3732/ajb.1000522</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.1000522%26rft_id%253Dinfo%253Apmid%252F21865506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjk6Ijk4LzkvMTM4OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvNTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.27" data-doi="10.1038/s41588-018-0266-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Milner</span>,  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Jost</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Taketa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. R.</span> <span class="cit-name-surname">Mazón</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Himmelbach</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Genebank genomics highlights the diversity of a global barley collection.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">326</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41588-018-0266-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-018-0266-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41588-018-0266-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.28" data-doi="10.1371/journal.pgen.1003925"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moreno-Estrada</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Gravel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Zakharia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">McCauley</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Byrnes</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Reconstructing the Population Genetic History of the Caribbean.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e1003925</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003925</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003925%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003925%26rft_id%253Dinfo%253Apmid%252F24244192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003925&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24244192&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.29" data-doi="10.3389/fgene.2014.00204"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Padhukasahasram</span>,  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Inferring ancestry from population genomic data and its applications.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">204</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2014.00204</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D5%26rft.spage%253D204%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2014.00204%26rft_id%253Dinfo%253Apmid%252F25071832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2014.00204&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25071832&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.30" data-doi="10.1093/bioinformatics/btp197"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paşaniuc</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sankararaman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Kimmel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Halperin</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Inference of locus-specific ancestry in closely related populations.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">i213</span>–<span class="cit-lpage">i221</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp197</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp197%26rft_id%253Dinfo%253Apmid%252F19477991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp197&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19477991&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266498300027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.31" data-doi="10.17348/era.7.0.199-216"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perrier</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Bakry</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Carreel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Jenny</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-P.</span> <span class="cit-name-surname">Horry</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Combining Biological Approaches to Shed Light on the Evolution of Edible Bananas.</span> <abbr class="cit-jnl-abbrev">Ethnobot. Res. Appl.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">199</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.17348/era.7.0.199-216</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEthnobot.%2BRes.%2BAppl.%26rft.volume%253D7%26rft.spage%253D199%26rft_id%253Dinfo%253Adoi%252F10.17348%252Fera.7.0.199-216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.17348/era.7.0.199-216&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000274569400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.32" data-doi="10.1073/pnas.1102001108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perrier</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">De Langhe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Donohue</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lentfer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Vrydaghs</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Multidisciplinary perspectives on banana (<em>Musa spp</em>.) domestication.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">11311</span>–<span class="cit-lpage">11318</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1102001108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1102001108%26rft_id%253Dinfo%253Apmid%252F21730145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzI4LzExMzExIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi81NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.33" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>,  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Stephens</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Donnelly</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi81NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.34" data-doi="10.1111/1755-0998.12512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Puechmaille</span>,  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The program structure does not reliably recover the correct population structure when sampling is uneven: subsampling and new estimators alleviate the problem.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">608</span>–<span class="cit-lpage">627</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12512</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D16%26rft.spage%253D608%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.35" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Neale</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Todd-Brown</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Thomas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A. R.</span> <span class="cit-name-surname">Ferreira</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PLINK: A Tool Set for Whole-Genome Association and Population-Based Linkage Analyses.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">575</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/519795</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.36" data-doi="10.1534/genetics.119.302139"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salter-Townshend</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Myers</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Fine-Scale Inference of Ancestry Segments Without Prior Knowledge of Admixing Groups.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">212</span>: <span class="cit-fpage">869</span>–<span class="cit-lpage">889</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.119.302139</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.119.302139%26rft_id%253Dinfo%253Apmid%252F31123038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjEyLzMvODY5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi81NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.37" data-doi="10.1016/j.ajhg.2007.09.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sankararaman</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sridhar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Kimmel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Halperin</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Estimating Local Ancestry in Admixed Populations.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">290</span>–<span class="cit-lpage">303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2007.09.022</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DSankararaman%26rft.auinit1%253DS.%26rft.volume%253D82%26rft.issue%253D2%26rft.spage%253D290%26rft.epage%253D303%26rft.atitle%253DEstimating%2Blocal%2Bancestry%2Bin%2Badmixed%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2007.09.022%26rft_id%253Dinfo%253Apmid%252F18252211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2007.09.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18252211&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253223900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.38" data-doi="10.1093/aob/mcw170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sardos</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Perrier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Doležel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Hřibová</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Christelová</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">DArT whole genome profiling provides insights on the evolution and taxonomy of edible Banana (<em>Musa</em> spp.).</span> <abbr class="cit-jnl-abbrev">Ann. Bot.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">1269</span>–<span class="cit-lpage">1278</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/aob/mcw170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcw170%26rft_id%253Dinfo%253Apmid%252F27590334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/aob/mcw170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27590334&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.39" data-doi="10.1086/502802"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scheet</span>,  <span class="cit-name-given-names">P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Stephens</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A Fast and Flexible Statistical Model for Large-Scale Population Genotype Data: Applications to Inferring Missing Genotypes and Haplotypic Phase.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">629</span>–<span class="cit-lpage">644</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/502802</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DScheet%26rft.auinit1%253DP.%26rft.volume%253D78%26rft.issue%253D4%26rft.spage%253D629%26rft.epage%253D644%26rft.atitle%253DA%2Bfast%2Band%2Bflexible%2Bstatistical%2Bmodel%2Bfor%2Blarge-scale%2Bpopulation%2Bgenotype%2Bdata%253A%2Bapplications%2Bto%2Binferring%2Bmissing%2Bgenotypes%2Band%2Bhaplotypic%2Bphase.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F502802%26rft_id%253Dinfo%253Apmid%252F16532393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/502802&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16532393&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236755900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.40" data-doi="10.1534/genetics.104.033175"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Semon</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Nielsen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Jones</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">McCouch</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The Population Structure of African Cultivated Rice <em>Oryza glaberrima</em> (Steud.): Evidence for Elevated Levels of Linkage Disequilibrium Caused by Admixture with <em>O. sativa</em> and Ecological Adaptation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">1639</span>–<span class="cit-lpage">1647</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.033175</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.033175%26rft_id%253Dinfo%253Apmid%252F15545652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS8zLzE2MzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzU2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.569.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simmonds</span>,  <span class="cit-name-given-names">N. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1962</span> <span class="cit-source">The Evolution of the Bananas</span>, <span class="cit-publ-name">Longmans</span>, <span class="cit-publ-loc">London, United Kingdom</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.42" data-doi="10.1093/bioinformatics/btw098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Staab</span>,  <span class="cit-name-given-names">P. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Metzler</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Coala: an R framework for coalescent simulation.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1903</span>–<span class="cit-lpage">1904</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btw098</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw098%26rft_id%253Dinfo%253Apmid%252F27153679%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw098&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27153679&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.43" data-doi="10.1093/bioinformatics/btu861"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Staab</span>,  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zhu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Metzler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Lunter</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">scrm: efficiently simulating long sequences using the approximated coalescent with recombination.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1680</span>–<span class="cit-lpage">1682</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu861</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu861%26rft_id%253Dinfo%253Apmid%252F25596205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu861&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25596205&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.44" data-doi="10.1038/s41437-019-0247-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stift</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Kolář</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. G.</span> <span class="cit-name-surname">Meirmans</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Structure is more robust than other clustering methods in simulated mixed-ploidy populations.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">429</span>–<span class="cit-lpage">441</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41437-019-0247-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D123%26rft.spage%253D429%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41437-019-0247-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41437-019-0247-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.45" data-doi="10.1086/504302"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Coram</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Risch</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Reconstructing Genetic Ancestry Blocks in Admixed Individuals.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/504302</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DTang%26rft.auinit1%253DH.%26rft.volume%253D79%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DReconstructing%2Bgenetic%2Bancestry%2Bblocks%2Bin%2Badmixed%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F504302%26rft_id%253Dinfo%253Apmid%252F16773560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/504302&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16773560&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238341200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.2.569.46"><div class="cit-metadata unstructured">The 3,000 rice genomes project, 2014 The 3,000 rice genomes project. GigaScience 3: 7.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.47" data-doi="10.1038/nature04226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The International HapMap Consortium</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A haplotype map of the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">437</span>: <span class="cit-fpage">1299</span>–<span class="cit-lpage">1320</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature04226</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAltshuler%26rft.auinit1%253DD.%26rft.volume%253D437%26rft.issue%253D7063%26rft.spage%253D1299%26rft.epage%253D1320%26rft.atitle%253DA%2Bhaplotype%2Bmap%2Bof%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04226%26rft_id%253Dinfo%253Apmid%252F16255080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16255080&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232829100044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.48" data-doi="10.1038/nature05911"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The Wellcome Trust Case Control Consortium</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">447</span>: <span class="cit-fpage">661</span>–<span class="cit-lpage">678</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature05911</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D447%26rft.issue%253D7145%26rft.spage%253D661%26rft.epage%253D678%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2B14%252C000%2Bcases%2Bof%2Bseven%2Bcommon%2Bdiseases%2Band%2B3%252C000%2Bshared%2Bcontrols.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05911%26rft_id%253Dinfo%253Apmid%252F17554300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05911&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17554300&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247030700035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.49" data-doi="10.2307/2408641"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>,  <span class="cit-name-given-names">B. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Cockerham</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Estimating F-Statistics for the Analysis of Population Structure.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1358</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D38%26rft.spage%253D1358%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408641%26rft_id%253Dinfo%253Apmid%252F28563791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28563791&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984TY40400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.569.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Willing</span>,  <span class="cit-name-given-names">E.-M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Dreyer</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">van Oosterhout</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Estimates of genetic differentiation measured by F<sub>ST</sub> do not necessarily require large sample sizes when using many SNP Markers.</span> <span class="cit-source">PLoS One</span> <span class="cit-vol">7</span>: <span class="cit-fpage">e42649</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D7%26rft.spage%253De42649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.51" data-doi="10.1038/nbt.2906"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Prochnik</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jenkins</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Salse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Hellsten</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Sequencing of diverse mandarin, pummelo and orange genomes reveals complex history of admixture during citrus domestication.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">656</span>–<span class="cit-lpage">662</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.2906</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D32%26rft.spage%253D656%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2906%26rft_id%253Dinfo%253Apmid%252F24908277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2906&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24908277&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.52" data-doi="10.1038/nature25447"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Terol</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Ibanez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">López-García</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Pérez-Román</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genomics of the origin and evolution of <em>Citrus</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">554</span>: <span class="cit-fpage">311</span>–<span class="cit-lpage">316</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature25447</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D554%26rft.spage%253D311%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature25447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature25447&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.53" data-doi="10.1371/journal.pone.0010780"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Wright</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Kimball</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Eizenga</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">McClung</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic Diversity and Introgression in <em>O. sativa</em> Reveal the Impact of Domestication and Breeding on the Rice Genome.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-elocation-id">e10780</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0010780</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DZhao%26rft.auinit1%253DK.%26rft.volume%253D5%26rft.issue%253D5%26rft.spage%253De10780%26rft.epage%253De10780%26rft.atitle%253DGenomic%2Bdiversity%2Band%2Bintrogression%2Bin%2BO.%2Bsativa%2Breveal%2Bthe%2Bimpact%2Bof%2Bdomestication%2Band%2Bbreeding%2Bon%2Bthe%2Brice%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0010780%26rft_id%253Dinfo%253Apmid%252F20520727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0010780&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20520727&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.569.54" data-doi="10.1371/journal.pgen.1005847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Zhao</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Guan</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Strong Selection at MHC in Mexicans since Admixture.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-elocation-id">e1005983</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005847</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1005983%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/2/569.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/2/555" title="Inflammation-Induced Alternative Pre-mRNA Splicing in Mouse Alveolar Macrophages" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/2/581" title="Optimizing Low-Cost Genotyping and Imputation Strategies for Genomic Selection in Atlantic Salmon" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/2"><span>Volume 10 Issue 2, February 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/2.cover-source.jpg?itok=LSawHOSr" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/569.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-_ZrZfm76KlLmRrBTPtrRvWOlukGUmFk6qXKhyo2BOHw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/2/569" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/2/569" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/2/569" class="active" data-icon-position="" data-hide-link-title="0">Simulation-Based Evaluation of Three Methods for Local Ancestry Deconvolution of Non-model Crop Species Genomes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495549" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-5sWBsyB3_6ao3a8HUzOLy8KMQH8byi5x8ZQrc8FRivI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/2/569.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/2/569" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-g9k-w4vT15v9jxhIusXjksj7cWQZ8bpPdJS3P0JvMSI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/2/569" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/2/569.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495549" id="node495549--2" data-pisa="ggg;10/2/569" data-pisa-master="ggg;g3.119.400873" data-apath="/ggg/10/2/569.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Simulation-Based Evaluation of Three Methods for Local Ancestry Deconvolution of Non-model Crop Species Genomes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0862-784X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aurélien Cottin</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8881-9253" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Benjamin Penaud</span>, <span class="highwire-citation-author" data-delta="2">Jean-Christophe Glaszmann</span>, <span class="highwire-citation-author" data-delta="3">Nabila Yahiaoui</span> and <span class="highwire-citation-author" data-delta="4">Mathieu Gautier</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">569-579;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400873 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495549--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Aurélien Cottin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CIRAD, UMR AGAP, F-34398 Montpellier, France,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>AGAP, Univ. Montpellier, CIRAD, INRAE, Montpellier SupAgro, Montpellier, France, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aur%C3%A9lien%2BCottin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cottin%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAur%25C3%25A9lien%2BCottin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0862-784X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aurélien Cottin</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Benjamin Penaud </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CIRAD, UMR AGAP, F-34398 Montpellier, France,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>AGAP, Univ. Montpellier, CIRAD, INRAE, Montpellier SupAgro, Montpellier, France, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BPenaud%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Penaud%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABenjamin%2BPenaud%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8881-9253" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Benjamin Penaud</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jean-Christophe Glaszmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CIRAD, UMR AGAP, F-34398 Montpellier, France,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>AGAP, Univ. Montpellier, CIRAD, INRAE, Montpellier SupAgro, Montpellier, France, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Christophe%2BGlaszmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Glaszmann%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Christophe%2BGlaszmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Nabila Yahiaoui </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CIRAD, UMR AGAP, F-34398 Montpellier, France,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>AGAP, Univ. Montpellier, CIRAD, INRAE, Montpellier SupAgro, Montpellier, France, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nabila%2BYahiaoui%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yahiaoui%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANabila%2BYahiaoui%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mathieu.gautier@inra.fr" class="" data-icon-position="" data-hide-link-title="0">mathieu.gautier@inra.fr</a>
<a href="mailto:nabila.yahiaoui@cirad.fr" class="" data-icon-position="" data-hide-link-title="0">nabila.yahiaoui@cirad.fr</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Mathieu Gautier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>INRAE, UMR CBGP, F-34988 Montferrier-sur-Lez Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mathieu%2BGautier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gautier%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMathieu%2BGautier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mathieu.gautier@inra.fr" class="" data-icon-position="" data-hide-link-title="0">mathieu.gautier@inra.fr</a>
<a href="mailto:nabila.yahiaoui@cirad.fr" class="" data-icon-position="" data-hide-link-title="0">nabila.yahiaoui@cirad.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/2/569" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/2/569&amp;title=Simulation-Based%20Evaluation%20of%20Three%20Methods%20for%20Local%20Ancestry%20Deconvolution%20of%20Non-model%20Crop%20Species%20Genomes" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/2/569&amp;title=Simulation-Based%20Evaluation%20of%20Three%20Methods%20for%20Local%20Ancestry%20Deconvolution%20of%20Non-model%20Crop%20Species%20Genomes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/2/569&amp;title=Simulation-Based%20Evaluation%20of%20Three%20Methods%20for%20Local%20Ancestry%20Deconvolution%20of%20Non-model%20Crop%20Species%20Genomes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/2/569&amp;text=Simulation-Based%20Evaluation%20of%20Three%20Methods%20for%20Local%20Ancestry%20Deconvolution%20of%20Non-model%20Crop%20Species%20Genomes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/2/569&amp;title=Simulation-Based%20Evaluation%20of%20Three%20Methods%20for%20Local%20Ancestry%20Deconvolution%20of%20Non-model%20Crop%20Species%20Genomes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/2/569&amp;t=Simulation-Based%20Evaluation%20of%20Three%20Methods%20for%20Local%20Ancestry%20Deconvolution%20of%20Non-model%20Crop%20Species%20Genomes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/2/569&amp;title=Simulation-Based%20Evaluation%20of%20Three%20Methods%20for%20Local%20Ancestry%20Deconvolution%20of%20Non-model%20Crop%20Species%20Genomes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/2/569&amp;title=Simulation-Based%20Evaluation%20of%20Three%20Methods%20for%20Local%20Ancestry%20Deconvolution%20of%20Non-model%20Crop%20Species%20Genomes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495549/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495549" id="node495549--3" data-pisa="ggg;10/2/569" data-pisa-master="ggg;g3.119.400873" data-apath="/ggg/10/2/569.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Simulation-Based Evaluation of Three Methods for Local Ancestry Deconvolution of Non-model Crop Species Genomes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0862-784X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aurélien Cottin</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8881-9253" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Benjamin Penaud</span>, <span class="highwire-citation-author" data-delta="2">Jean-Christophe Glaszmann</span>, <span class="highwire-citation-author" data-delta="3">Nabila Yahiaoui</span> and <span class="highwire-citation-author" data-delta="4">Mathieu Gautier</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">569-579;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400873 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495549--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Aurélien Cottin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CIRAD, UMR AGAP, F-34398 Montpellier, France,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>AGAP, Univ. Montpellier, CIRAD, INRAE, Montpellier SupAgro, Montpellier, France, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aur%C3%A9lien%2BCottin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cottin%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAur%25C3%25A9lien%2BCottin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0862-784X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aurélien Cottin</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Benjamin Penaud </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CIRAD, UMR AGAP, F-34398 Montpellier, France,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>AGAP, Univ. Montpellier, CIRAD, INRAE, Montpellier SupAgro, Montpellier, France, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BPenaud%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Penaud%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABenjamin%2BPenaud%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8881-9253" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Benjamin Penaud</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jean-Christophe Glaszmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CIRAD, UMR AGAP, F-34398 Montpellier, France,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>AGAP, Univ. Montpellier, CIRAD, INRAE, Montpellier SupAgro, Montpellier, France, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Christophe%2BGlaszmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Glaszmann%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Christophe%2BGlaszmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Nabila Yahiaoui </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CIRAD, UMR AGAP, F-34398 Montpellier, France,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>AGAP, Univ. Montpellier, CIRAD, INRAE, Montpellier SupAgro, Montpellier, France, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nabila%2BYahiaoui%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yahiaoui%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANabila%2BYahiaoui%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mathieu.gautier@inra.fr" class="" data-icon-position="" data-hide-link-title="0">mathieu.gautier@inra.fr</a>
<a href="mailto:nabila.yahiaoui@cirad.fr" class="" data-icon-position="" data-hide-link-title="0">nabila.yahiaoui@cirad.fr</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Mathieu Gautier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>INRAE, UMR CBGP, F-34988 Montferrier-sur-Lez Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mathieu%2BGautier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gautier%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMathieu%2BGautier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mathieu.gautier@inra.fr" class="" data-icon-position="" data-hide-link-title="0">mathieu.gautier@inra.fr</a>
<a href="mailto:nabila.yahiaoui@cirad.fr" class="" data-icon-position="" data-hide-link-title="0">nabila.yahiaoui@cirad.fr</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=569&amp;rft.epage=579&amp;rft.atitle=Simulation-Based%20Evaluation%20of%20Three%20Methods%20for%20Local%20Ancestry%20Deconvolution%20of%20Non-model%20Crop%20Species%20Genomes&amp;rft.volume=10&amp;rft.issue=2&amp;rft.date=2020-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Cottin%2C+Aur%C3%A9lien&amp;rft.au=Penaud%2C+Benjamin&amp;rft.au=Glaszmann%2C+Jean-Christophe&amp;rft.au=Yahiaoui%2C+Nabila&amp;rft.au=Gautier%2C+Mathieu"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495549/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495549/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495549/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495549/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495549/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495549/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495549/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495549/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495549/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495549/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495549/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495549/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/2/569" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/2/569.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/2/569.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/2/569.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
