<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/2/769" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/2/769.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/2/769.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/2/769.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/10/2/769/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/2/769.atom" />
<meta name="HW.pisa" content="ggg;10/2/769" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum" />
<meta name="DC.Identifier" content="10.1534/g3.119.400759" />
<meta name="DC.Date" content="2020-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 dos Santos et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The ability to connect genetic information between traits over time allow Bayesian networks to offer a powerful probabilistic framework to construct genomic prediction models. In this study, we phenotyped a diversity panel of 869 biomass sorghum ( Sorghum bicolor (L.) Moench) lines, which had been genotyped with 100,435 SNP markers, for plant height (PH) with biweekly measurements from 30 to 120 days after planting (DAP) and for end-of-season dry biomass yield (DBY) in four environments. We evaluated five genomic prediction models: Bayesian network (BN), Pleiotropic Bayesian network (PBN), Dynamic Bayesian network (DBN), multi-trait GBLUP (MTr-GBLUP), and multi-time GBLUP (MTi-GBLUP) models. In fivefold cross-validation, prediction accuracies ranged from 0.46 (PBN) to 0.49 (MTr-GBLUP) for DBY and from 0.47 (DBN, DAP120) to 0.75 (MTi-GBLUP, DAP60) for PH. Forward-chaining cross-validation further improved prediction accuracies of the DBN, MTi-GBLUP and MTr-GBLUP models for PH (training slice: 30-45 DAP) by 36.4–52.4% relative to the BN and PBN models. Coincidence indices (target: biomass, secondary: PH) and a coincidence index based on lines (PH time series) showed that the ranking of lines by PH changed minimally after 45 DAP. These results suggest a two-level indirect selection method for PH at harvest (first-level target trait) and DBY (second-level target trait) could be conducted earlier in the season based on ranking of lines by PH at 45 DAP (secondary trait). With the advance of high-throughput phenotyping technologies, our proposed two-level indirect selection framework could be valuable for enhancing genetic gain per unit of time when selecting on developmental traits." />
<meta name="DC.Contributor" content="Jhonathan P. R. dos Santos" />
<meta name="DC.Contributor" content="Samuel B. Fernandes" />
<meta name="DC.Contributor" content="Scott McCoy" />
<meta name="DC.Contributor" content="Roberto Lozano" />
<meta name="DC.Contributor" content="Patrick J. Brown" />
<meta name="DC.Contributor" content="Andrew D.B. Leakey" />
<meta name="DC.Contributor" content="Edward S. Buckler" />
<meta name="DC.Contributor" content="Antonio A. F. Garcia" />
<meta name="DC.Contributor" content="Michael A. Gore" />
<meta name="article:published_time" content="2020-02-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The ability to connect genetic information between traits over time allow Bayesian networks to offer a powerful probabilistic framework to construct genomic prediction models. In this study, we phenotyped a diversity panel of 869 biomass sorghum (&lt;i&gt;Sorghum bicolor&lt;/i&gt; (L.) Moench) lines, which had been genotyped with 100,435 SNP markers, for plant height (PH) with biweekly measurements from 30 to 120 days after planting (DAP) and for end-of-season dry biomass yield (DBY) in four environments. We evaluated five genomic prediction models: Bayesian network (BN), Pleiotropic Bayesian network (PBN), Dynamic Bayesian network (DBN), multi-trait GBLUP (MTr-GBLUP), and multi-time GBLUP (MTi-GBLUP) models. In fivefold cross-validation, prediction accuracies ranged from 0.46 (PBN) to 0.49 (MTr-GBLUP) for DBY and from 0.47 (DBN, DAP120) to 0.75 (MTi-GBLUP, DAP60) for PH. Forward-chaining cross-validation further improved prediction accuracies of the DBN, MTi-GBLUP and MTr-GBLUP models for PH (training slice: 30-45 DAP) by 36.4–52.4% relative to the BN and PBN models. Coincidence indices (target: biomass, secondary: PH) and a coincidence index based on lines (PH time series) showed that the ranking of lines by PH changed minimally after 45 DAP. These results suggest a two-level indirect selection method for PH at harvest (first-level target trait) and DBY (second-level target trait) could be conducted earlier in the season based on ranking of lines by PH at 45 DAP (secondary trait). With the advance of high-throughput phenotyping technologies, our proposed two-level indirect selection framework could be valuable for enhancing genetic gain per unit of time when selecting on developmental traits.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/02/01" />
<meta name="citation_mjid" content="ggg;10/2/769" />
<meta name="citation_id" content="10/2/769" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/2/769" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/2/769.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/2/769.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/2/769.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400759" />
<meta name="citation_pmid" content="31852730" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="769" />
<meta name="citation_lastpage" content="781" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jhonathan P. R. dos Santos" />
<meta name="citation_author_institution" content="Plant Breeding and Genetics Section, School of Integrative Plant Science," />
<meta name="citation_author_institution" content="Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, SP, Brazil," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9030-702X" />
<meta name="citation_author" content="Samuel B. Fernandes" />
<meta name="citation_author_institution" content="Department of Crop Science," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8269-535X" />
<meta name="citation_author" content="Scott McCoy" />
<meta name="citation_author_institution" content="Institute for Genomic Biology," />
<meta name="citation_author" content="Roberto Lozano" />
<meta name="citation_author_institution" content="Plant Breeding and Genetics Section, School of Integrative Plant Science," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0760-4977" />
<meta name="citation_author" content="Patrick J. Brown" />
<meta name="citation_author_institution" content="Section of Agricultural Plant Biology, Department of Plant Sciences, University of California Davis, 95616, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1332-711X" />
<meta name="citation_author" content="Andrew D.B. Leakey" />
<meta name="citation_author_institution" content="Department of Crop Science," />
<meta name="citation_author_institution" content="Institute for Genomic Biology," />
<meta name="citation_author_institution" content="Department of Plant Biology, University of Illinois at Urbana Champaign, 61801," />
<meta name="citation_author" content="Edward S. Buckler" />
<meta name="citation_author_institution" content="Plant Breeding and Genetics Section, School of Integrative Plant Science," />
<meta name="citation_author_institution" content="United States Department of Agriculture, Agricultural Research Service, R. W. Holley Center, Ithaca, New York 14853" />
<meta name="citation_author_institution" content="Institute for Genomic Diversity, Cornell University, Ithaca, New York 14853," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3100-371X" />
<meta name="citation_author" content="Antonio A. F. Garcia" />
<meta name="citation_author_institution" content="Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, SP, Brazil," />
<meta name="citation_author_email" content="augusto.garcia@usp.br" />
<meta name="citation_author_email" content="mag87@cornell.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0634-3277" />
<meta name="citation_author" content="Michael A. Gore" />
<meta name="citation_author_institution" content="Plant Breeding and Genetics Section, School of Integrative Plant Science," />
<meta name="citation_author_email" content="augusto.garcia@usp.br" />
<meta name="citation_author_email" content="mag87@cornell.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6896-8024" />
<meta name="citation_reference" content="Akdemir, D. and O. U. Godfrey, 2018 EMMREML: Fitting Mixed Models with Known Covariance Structures (Version 3.1)." />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=H. Bae;citation_author=S. Monti;citation_author=M. Montano;citation_author=M. H. Steinberg;citation_author=T. T. Perls;citation_title=Learning Bayesian Networks from Correlated Data.;citation_pages=25156;citation_volume=6;citation_year=2016;citation_doi=10.1038/srep25156" />
<meta name="citation_reference" content="citation_journal_title=J. Field Robot.;citation_author=Y. Bao;citation_author=L. Tang;citation_author=M. W. Breitzman;citation_author=M. G. Salas Fernandez;citation_author=P. S. Schnable;citation_title=Field-based robotic phenotyping of sorghum plant architecture using stereo vision.;citation_pages=397-415;citation_volume=36;citation_year=2019;citation_doi=10.1002/rob.21830" />
<meta name="citation_reference" content="citation_journal_title=Phil Trans R Soc A;citation_author=C. M. Bishop;citation_title=Model-based machine learning.;citation_pages=20120222;citation_volume=371;citation_year=2013;citation_doi=10.1098/rsta.2012.0222" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. W. Brenton;citation_author=E. A. Cooper;citation_author=M. T. Myers;citation_author=R. E. Boyles;citation_author=N. Shakoor;citation_title=A genomic resource for the development, improvement, and exploitation of sorghum for bioenergy.;citation_volume=204;citation_year=2016;citation_pmid=27356613;citation_doi=10.1534/genetics.115.183947" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=B. Browning;citation_author=S. Browning;citation_title=Genotype imputation with millions of reference samples.;citation_pages=116-126;citation_volume=98;citation_year=2016;citation_pmid=26748515;citation_doi=10.1016/j.ajhg.2015.11.020" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=E. S. Buckler;citation_author=D. C. Ilut;citation_author=X. Wang;citation_author=T. Kretzschmar;citation_author=M. A. Gore;citation_title=rampseq: Using repetitive sequences for robust genotyping.;citation_year=2016;citation_doi=10.1101/096628" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueño;citation_author=G. de los Campos;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Genomic Prediction of Breeding Values when Modeling Genotype × Environment Interaction using Pedigree and Dense Molecular Markers.;citation_pages=707-719;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0299" />
<meta name="citation_reference" content="citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=T. Caliński;citation_author=S. Czajka;citation_author=Z. Kaczmarek;citation_author=P. Krajewski;citation_author=W. Pilarczyk;citation_title=Analyzing multi-environment variety trials using randomization-derived mixed models.;citation_pages=448-455;citation_volume=61;citation_year=2005;citation_issue=2;citation_pmid=16011691;citation_doi=10.1111/j.1541-0420.2005.00334.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=M. P. Calus;citation_author=R. F. Veerkamp;citation_title=Accuracy of multi-trait genomic selection using different methods.;citation_pages=26-26;citation_volume=43;citation_year=2011;citation_pmid=21729282;citation_doi=10.1186/1297-9686-43-26" />
<meta name="citation_reference" content="citation_journal_title=Plant Direct;citation_author=M. Campbell;citation_author=H. Walia;citation_author=G. Morota;citation_title=Utilizing random regression models for genomic prediction of a longitudinal trait derived from high-throughput phenotyping.;citation_pages=00080e;citation_volume=2;citation_year=2018;citation_doi=10.1002/pld3.80" />
<meta name="citation_reference" content="citation_journal_title=Articles;citation_author=B. Carpenter;citation_author=A. Gelman;citation_author=M. Hoffman;citation_author=D. Lee;citation_author=B. Goodrich;citation_title=Stan: A probabilistic programming language. Journal of Statistical Software;citation_pages=1-32;citation_volume=76;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. W. Davey;citation_author=P. A. Hohenlohe;citation_author=P. D. Etter;citation_author=J. Q. Boone;citation_author=J. M. Catchen;citation_title=Genome-wide genetic marker discovery and genotyping using next-generation sequencing.;citation_pages=499-510;citation_volume=12;citation_year=2011;citation_issue=7;citation_pmid=21681211;citation_doi=10.1038/nrg3012" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=K. O. G. Dias;citation_author=S. A. Gezan;citation_author=C. T. Guimarães;citation_author=A. Nazarian;citation_author=L.S. da Costa E.;citation_title=Improving accuracies of genomic predictions for drought tolerance in maize by joint modeling of additive and dominance effects in multi-environment trials.;citation_pages=24-37;citation_volume=121;citation_year=2018;citation_doi=10.1038/s41437-018-0053-6" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=J. P. R. dos Santos;citation_author=L. Paulo;citation_author=M. Pires;citation_author=R. Coelho;citation_author=D. C. Vasconcellos;citation_title=Genomic selection to resistance to Stenocarpella maydis in maize lines using DArTseq markers.;citation_pages=86;citation_volume=17;citation_year=2016;citation_doi=10.1186/s12863-016-0392-3" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. P. R. dos Santos;citation_author=R. C. De Castro Vasconcellos;citation_author=L. P. M. Pires;citation_author=M. Balestre;citation_author=R. G. Von Pinho;citation_title=Inclusion of dominance effects in the multivariate GBLUP model.;citation_pages=e0152045;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0152045" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with r package rrblup.;citation_pages=250-255;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. B. Fernandes;citation_author=K. O. G. Dias;citation_author=D. F. Ferreira;citation_author=P. J. Brown;citation_title=Efficiency of multi-trait, indirect, and trait-assisted genomic selection for improvement of biomass sorghum.;citation_pages=747-755;citation_volume=131;citation_year=2018;citation_doi=10.1007/s00122-017-3033-y" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=L. F. V. Ferrão;citation_author=R. G. Ferrão;citation_author=M. A. G. Ferrão;citation_author=A. Fonseca;citation_author=P. Carbonetto;citation_title=Accurate genomic prediction of Coffea canephora in multiple environments using whole-genome statistical models.;citation_volume=122;citation_year=2018;citation_doi=10.1038/s41437-018-0105-y" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Foley;citation_author=N. Ramankutty;citation_author=K. Brauman;citation_author=E. S. Cassidy;citation_author=J. S. Gerber;citation_title=Solutions for a cultivated planet.;citation_pages=337-342;citation_volume=478;citation_year=2011;citation_issue=7369;citation_pmid=21993620;citation_doi=10.1038/nature10452" />
<meta name="citation_reference" content="citation_journal_title=The Plant Phe. J;citation_author=J. L. Gage;citation_author=E. Richards;citation_author=N. Lepak;citation_author=N. Kaczmar;citation_author=C. Soman;citation_title=In-field whole plant maize architecture characterized by latent space phenotyping.;citation_pages=1-21;citation_volume=2;citation_year=2019" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=A. F. Garcia;citation_author=M. Mollinari;citation_author=T. G. Marconi;citation_author=O. R. Serang;citation_author=R. R. Silva;citation_title=SNP genotyping allows an in-depth characterisation of the genome of sugarcane and other complex autopolyploids.;citation_pages=3399;citation_volume=3;citation_year=2013;citation_pmid=24292365;citation_doi=10.1038/srep03399" />
<meta name="citation_reference" content="citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_title=Priors in whole-genome regression: the bayesian alphabet returns.;citation_volume=194;citation_year=2013;citation_pmid=23636739;citation_doi=10.1534/genetics.113.151753" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_author=G. de los Campos;citation_author=W. G. Hill;citation_author=E. Manfredi;citation_author=R. Fernando;citation_title=Additive genetic variability and the Bayesian alphabet.;citation_volume=183;citation_year=2009;citation_pmid=19620397;citation_doi=10.1534/genetics.109.103952" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_author=G. de los Campos;citation_author=M. A. Toro;citation_author=H. Naya;citation_author=C.-C. Schön;citation_title=Do molecular markers inform about pleiotropy?;citation_volume=201;citation_year=2015;citation_pmid=26205989;citation_doi=10.1534/genetics.115.179978" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. C. Glaubitz;citation_author=T. M. Casstevens;citation_author=F. Lu;citation_author=J. Harriman;citation_author=R. J. Elshire;citation_title=Tassel-gbs: A high capacity genotyping by sequencing analysis pipeline.;citation_pages=e90346;citation_volume=9;citation_year=2014;citation_doi=10.1371/journal.pone.0090346" />
<meta name="citation_reference" content="citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed. Rev.;citation_author=J. Hamblin;citation_author=M. O. Zimmermann;citation_title=Breeding common bean for yield in mixtures.;citation_pages=245-272;citation_volume=4;citation_year=1986" />
<meta name="citation_reference" content="citation_journal_title=BMC Syst. Biol.;citation_author=B. Han;citation_author=X. Chen;citation_author=Z. Talebizadeh;citation_author=H. Xu;citation_title=Genetic studies of complex human diseases: characterizing SNP-disease associations using Bayesian networks.;citation_pages=S14;citation_volume=6;citation_year=2012;citation_doi=10.1186/1752-0509-6-S2-S14" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. R. Harlan;citation_author=J. M. J. de Wet;citation_title=A Simplified Classification of Cultivated Sorghum;citation_pages=172-176;citation_volume=12;citation_year=1972;citation_issue=2;citation_doi=10.2135/cropsci1972.0011183X001200020005x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=C. R. Henderson;citation_author=R. L. Quaas;citation_title=Multiple Trait Evaluation Using Relatives' Records;citation_pages=1188-1197;citation_volume=43;citation_year=1976;citation_issue=6;citation_doi=10.2527/jas1976.4361188x" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=J. L. Jannink;citation_author=M. E. Sorrels;citation_title=Perspectives for Genomic Selection Applications and Research in Plants.;citation_pages=1-12;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2014.03.0249" />
<meta name="citation_reference" content="citation_journal_title=J. Mach. Learn. Res.;citation_author=M. D. Hoffman;citation_author=A. Gelman;citation_title=The No-U-Turn Sampler: Adaptively Setting Path Lengths in Hamiltonian Monte Carlo.;citation_pages=1593-1623;citation_volume=15;citation_year=2014" />
<meta name="citation_reference" content="citation_title=Estimating and interpreting heritability for plant breeding: An update.;citation_pages=9-112;citation_volume=22;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=H. Hung;citation_author=C. Browne;citation_author=K. Guill;citation_author=N. Coles;citation_author=M. Eller;citation_title=The relationship between parental genetic or phenotypic divergence and progeny variation in the maize nested association mapping population.;citation_pages=490-499;citation_volume=108;citation_year=2012;citation_issue=5;citation_pmid=22027895;citation_doi=10.1038/hdy.2011.103" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Jia;citation_author=J.-L. Jannink;citation_title=Multiple-trait genomic selection methods increase genetic value prediction accuracy.;citation_volume=192;citation_year=2012;citation_pmid=23086217;citation_doi=10.1534/genetics.112.144246" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=C. J. Lawrence;citation_author=V. Walbot;citation_title=Translational Genomics for Bioenergy Production from Fuelstock Grasses: Maize as the Model Species.;citation_volume=19;citation_year=2007;citation_pmid=17660357;citation_doi=10.1105/tpc.107.053660" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=H. Li;citation_author=A. Rasheed;citation_author=L. T. Hickey;citation_author=Z. He;citation_title=Fast-forwarding genetic gain.;citation_pages=184-186;citation_volume=23;citation_year=2018;citation_doi=10.1016/j.tplants.2018.01.007" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_title=A complete bacterial genome assembled de novo using only nanopore sequencing data.;citation_volume=12;citation_year=2015" />
<meta name="citation_reference" content="citation_title=Genetics and analysis of quantitative traits;citation_author=M. Lynch;citation_author=B. Walsh;citation_pages=537-550;citation_volume=Vol. 1;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=E. S. Mace;citation_author=S. Tai;citation_author=E. K. Gilding;citation_author=Y. Li;citation_author=P. J. Prentis;citation_title=Whole-genome sequencing reveals untapped genetic potential in Africa’s indigenous cereal crop sorghum.;citation_pages=2320;citation_volume=4;citation_year=2013;citation_pmid=23982223;citation_doi=10.1038/ncomms3320" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=O. A. Montesinos-López;citation_author=A. Montesinos-López;citation_author=J. Crossa;citation_author=F. H. Toledo;citation_author=O. Pérez-Hernández;citation_title=A genomic bayesian multi-trait and multi-environment model. G3: Genes, Genomes;citation_pages=2725-2744;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=G. P. Morris;citation_author=P. Ramu;citation_author=S. P. Deshpande;citation_author=C. T. Hash;citation_author=T. Shah;citation_title=Population genomic and genome-wide association studies of agroclimatic traits in sorghum.;citation_volume=110;citation_year=2013;citation_pmid=23267105;citation_doi=10.1073/pnas.1215985110" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=J. Mullet;citation_author=D. Morishige;citation_author=R. McCormick;citation_author=S. Truong;citation_author=J. Hilley;citation_title=Energy sorghum–a genetic model for the design of C4 grass bioenergy crops.;citation_volume=65;citation_year=2014;citation_pmid=24958898;citation_doi=10.1093/jxb/eru229" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=M. M. Muraya;citation_author=J. Chu;citation_author=Y. Zhao;citation_author=A. Junker;citation_author=C. Klukas;citation_title=Genetic variation of growth dynamics in maize (zea mays l.) revealed through automated non-invasive phenotyping.;citation_pages=366-380;citation_volume=89;citation_year=2017;citation_doi=10.1111/tpj.13390" />
<meta name="citation_reference" content="citation_title=Machine learning: a probabilistic perspective;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Cancer Inform.;citation_author=R. Neapolitan;citation_author=D. Xue;citation_author=X. Jiang;citation_title=Modeling the altered expression levels of genes on signaling pathways in tumors as causal bayesian networks.;citation_pages=77-84;citation_volume=13;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=U. G. Okeke;citation_author=D. Akdemir;citation_author=I. Rabbi;citation_author=P. Kulakow;citation_author=J.-L. Jannink;citation_title=Accuracies of univariate and multivariate genomic prediction models in african cassava.;citation_pages=88;citation_volume=49;citation_year=2017;citation_doi=10.1186/s12711-017-0361-y" />
<meta name="citation_reference" content="citation_journal_title=Biometrika;citation_journal_abbrev=Biometrika;citation_author=H. D. Patterson;citation_author=R. Thompson;citation_title=Recovery of inter-block information when block sizes are unequal;citation_pages=545-554;citation_volume=58;citation_year=1971;citation_issue=3;citation_doi=10.1093/biomet/58.3.545" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=D. Pauli;citation_author=S. C. Chapman;citation_author=R. Bart;citation_author=C. N. Topp;citation_author=C. J. Lawrence-Dill;citation_title=The quest for understanding phenotypic variation via integrated approaches in the field environment.;citation_volume=172;citation_year=2016;citation_pmid=27482076;citation_doi=10.1104/pp.16.00592" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Raj;citation_author=M. Stephens;citation_author=J. K. Pritchard;citation_title=faststructure: Variational inference of population structure in large snp data sets.;citation_volume=197;citation_year=2014;citation_pmid=24700103;citation_doi=10.1534/genetics.114.164350" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=B. Ratcliffe;citation_author=O. G. El-Dien;citation_author=J. Klapste;citation_author=I. Porth;citation_author=C. Chen;citation_title=A comparison of genomic selection models across time in interior spruce (Picea engelmannii × glauca) using unordered SNP imputation methods.;citation_pages=547-555;citation_volume=115;citation_year=2015;citation_pmid=26126540;citation_doi=10.1038/hdy.2015.57." />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=O. Serang;citation_author=M. Mollinari;citation_author=A. A. F. Garcia;citation_title=Efficient exact maximum a posteriori computation for bayesian SNP genotyping in polyploids.;citation_pages=e30906-e30906;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22363513;citation_doi=10.1371/journal.pone.0030906" />
<meta name="citation_reference" content="citation_journal_title=BioData Min.;citation_author=C. Su;citation_author=A. Andrew;citation_author=M. R. Karagas;citation_author=M. E. Borsuk;citation_title=Using Bayesian networks to discover relations between genes, environment, and disease.;citation_pages=6;citation_volume=6;citation_year=2013;citation_doi=10.1186/1756-0381-6-6" />
<meta name="citation_reference" content="Team, S. D., 2018 PyStan: the Python interface to Stan, Version 2.17.1.0." />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Valluru;citation_author=E. E. Gazave;citation_author=S. B. Fernandes;citation_author=J. N. Ferguson;citation_author=R. Lozano;citation_title=Deleterious Mutation Burden and Its Association with Complex Traits in Sorghum (Sorghum bicolor);citation_pages=1075-1087;citation_volume=211;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_abbrev=J Integr Plant Biol;citation_author=W. Vermerris;citation_title=Survey of genomics approaches to improve bioenergy traits in maize, sorghum and sugarcane.;citation_pages=105-119;citation_volume=53;citation_year=2011;citation_issue=2;citation_pmid=21205186;citation_doi=10.1111/j.1744-7909.2010.01020.x" />
<meta name="citation_reference" content="citation_journal_title=Nat. Plants;citation_author=A. Watson;citation_author=S. Ghosh;citation_author=M. J. Williams;citation_author=W. S. Cuddy;citation_author=J. Simmonds;citation_title=Speed breeding is a powerful tool to accelerate crop research and breeding.;citation_pages=23-29;citation_volume=4;citation_year=2018;citation_doi=10.1038/s41477-017-0083-8" />
<meta name="citation_reference" content="citation_journal_title=Methods Ecol. Evol.;citation_author=M. E. Wolak;citation_title=nadiv: an R package to create relatedness matrices for estimating non-additive genetic variances in animal models.;citation_pages=792-796;citation_volume=3;citation_year=2012;citation_doi=10.1111/j.2041-210X.2012.00213.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Xu;citation_title=Genetic mapping and genomic selection using recombination breakpoint data.;citation_volume=195;citation_year=2013;citation_pmid=23979575;citation_doi=10.1534/genetics.113.155309" />
<meta name="citation_reference" content="citation_journal_title=Nat. Plants;citation_author=X. Yu;citation_author=X. Li;citation_author=T. Guo;citation_author=C. Zhu;citation_author=Y. Wu;citation_title=Genomic prediction contributing to a promising global strategy to turbocharge gene banks.;citation_pages=16150;citation_volume=2;citation_year=2016;citation_doi=10.1038/nplants.2016.150" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=X. Zheng;citation_author=D. Levine;citation_author=J. Shen;citation_author=S. M. Gogarten;citation_author=C. Laurie;citation_title=A high-performance computing toolset for relatedness and principal component analysis of SNP data.;citation_volume=28;citation_year=2012;citation_pmid=23060615;citation_doi=10.1093/bioinformatics/bts606" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
<meta name="twitter:description" content="The ability to connect genetic information between traits over time allow Bayesian networks to offer a powerful probabilistic framework to construct genomic prediction models. In this study, we phenotyped a diversity panel of 869 biomass sorghum ( Sorghum bicolor (L.) Moench) lines, which had been genotyped with 100,435 SNP markers, for plant height (PH) with biweekly measurements from 30 to 120 days after planting (DAP) and for end-of-season dry biomass yield (DBY) in four environments. We evaluated five genomic prediction models: Bayesian network (BN), Pleiotropic Bayesian network (PBN), Dynamic Bayesian network (DBN), multi-trait GBLUP (MTr-GBLUP), and multi-time GBLUP (MTi-GBLUP) models. In fivefold cross-validation, prediction accuracies ranged from 0.46 (PBN) to 0.49 (MTr-GBLUP) for DBY and from 0.47 (DBN, DAP120) to 0.75 (MTi-GBLUP, DAP60) for PH. Forward-chaining cross-validation further improved prediction accuracies of the DBN, MTi-GBLUP and MTr-GBLUP models for PH (training slice: 30-45 DAP) by 36.4–52.4% relative to the BN and PBN models. Coincidence indices (target: biomass, secondary: PH) and a coincidence index based on lines (PH time series) showed that the ranking of lines by PH changed minimally after 45 DAP. These results suggest a two-level indirect selection method for PH at harvest (first-level target trait) and DBY (second-level target trait) could be conducted earlier in the season based on ranking of lines by PH at 45 DAP (secondary trait). With the advance of high-throughput phenotyping technologies, our proposed two-level indirect selection framework could be valuable for enhancing genetic gain per unit of time when selecting on developmental traits." />
<meta name="og-title" property="og:title" content="Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/2/769" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The ability to connect genetic information between traits over time allow Bayesian networks to offer a powerful probabilistic framework to construct genomic prediction models. In this study, we phenotyped a diversity panel of 869 biomass sorghum ( Sorghum bicolor (L.) Moench) lines, which had been genotyped with 100,435 SNP markers, for plant height (PH) with biweekly measurements from 30 to 120 days after planting (DAP) and for end-of-season dry biomass yield (DBY) in four environments. We evaluated five genomic prediction models: Bayesian network (BN), Pleiotropic Bayesian network (PBN), Dynamic Bayesian network (DBN), multi-trait GBLUP (MTr-GBLUP), and multi-time GBLUP (MTi-GBLUP) models. In fivefold cross-validation, prediction accuracies ranged from 0.46 (PBN) to 0.49 (MTr-GBLUP) for DBY and from 0.47 (DBN, DAP120) to 0.75 (MTi-GBLUP, DAP60) for PH. Forward-chaining cross-validation further improved prediction accuracies of the DBN, MTi-GBLUP and MTr-GBLUP models for PH (training slice: 30-45 DAP) by 36.4–52.4% relative to the BN and PBN models. Coincidence indices (target: biomass, secondary: PH) and a coincidence index based on lines (PH time series) showed that the ranking of lines by PH changed minimally after 45 DAP. These results suggest a two-level indirect selection method for PH at harvest (first-level target trait) and DBY (second-level target trait) could be conducted earlier in the season based on ranking of lines by PH at 45 DAP (secondary trait). With the advance of high-throughput phenotyping technologies, our proposed two-level indirect selection framework could be valuable for enhancing genetic gain per unit of time when selecting on developmental traits." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
    <title>Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Gq1_dI2Xa-1mC6xiUQUIzmK-gagRG8gP4Hz5LUuGtoQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495543","apath":"\/ggg\/10\/2\/769.atom","pisa":"ggg;10\/2\/769","ac":{"\/ggg\/10\/2\/769.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/2\/769.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/2\/769"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/2\/769"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/2\/769":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022k\\\\\\\/+G9p3k4gHXEih6uRZlRO\\\\\\\/e5CZnkxelQOpx1iUSnnrh7MVttfLZI6TyJxEYwsseEn39chX7S8iL5EMxuGWHpBTwK2JMwh0onVmHELNvfRDpSJCc0EId27zRScsP0NuHcwZaxM5kQYvdtGPeFyn3aTS\\\\\\\/HdMR8njcq7RGqGXqdA2QGfSNcaZ4aW2hpdUjM1r63P2y8DsESLRdBDALZBP23Cv3YKfr\\\\\\\/zmRwsr6rCUPvtUrAbEN9+2+KVNkHT8ZK+aszt+dleWDua70GWBvVqjh9W8Ja8QEUwvkHzbHeOTkx6eHNaMWKfsDC8Me8nv59SQqL\\\\\\\/X\\\\\\\/uZJLbHb+wCPEcH8YA535FF8t5nrQEiVp8Uguhhm2QyYBT3re0x99x8eR6Om9It5g+K+lNKpt5dumtrjxiG8gFEL1mugQbLmqx8tC3EP447wjccWKTWt2c3zsDaHPfTaFqVgK19tscsaSz6BavnRIx6W3q7y5w3\\\\\\\/088RGx\\\\\\\/AZz2OmN6cejgjSbq7r+W6IKTpIkNLQR3xTHcHO7xWqEoYreOJDqfWi8skBRyU2x\\\\\\\/R7Pgnah0eUUX2yUHRC1JNSBTrb0RrGwtPZJ4aq5zl5vVGF6BJ4F3JQOnMGtciScSWHJuZEPk4yYV8tk1KZUZx+bFkAN\\\\\\\/LbdS1FJ8Pmr7t9JvIdqgq7I0w55Kow+uBLkOulPFv2liqfhImuB3H314Kq0N5kf66aALBKfVh7DBrsgWJGJREsu\\\\\\\/HT1E8mDSL9l2BbdkCMEG9Lo4bUhROm\\\\\\\/PaQaYTWfI\\\\\\\/etasU89zvTgl1oLtm\\\\\\\/ThHuLWYSTMqwa5Zo1KWK7TSlaMsFFgP6GTJkN8Fh4kvC8f2iNtiDypdAO5wgxDRWuez8oXqUgRkOln3MbNSamcNKDzgrT7TC+EEYeoNV0LhUCtgKZVoYSS0y1Khi32RBUHBZansqdc\\\\\\\/QyB1D4Wt+o2tQZwO1Rf4p9ASi4o7AFt1hxDyPdQZSxOcun8J7J0nG\\\\\\\/zcemoSWQA6ZYBvJISGJm5zP+ul5AhnJm+v+BZGrjnmTw5IlCeS+UgfWXgycfmE6782UPjAy8F9Oi6NL+fZSdqdq4gMMwHjPvdKjN6aPED1rQhkgsfqeOitsINzS0p5IiA4KQw9wQq8hK8cE1BJofyP7U4Kux\\\\\\\/bOLJNjxwX602B7CH6IMUhfCOlsR4atF6widFx+pIPzsBPYidR4LIdr+bBhvpBPVGeMTZDPQlPuzqowDzJbQnDJE4ji8QYQetGYMQv0ZmLIO3dgAXznu8dReispdCW4o3swFLuJoOT\\\u0022,\\\u0022iv\\\u0022:\\\u0022pMxcwLFIEJDUaEWA6iLdWw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022931dd135f089c52f6fd800b03bd38fda4ef64803ebf37415cd317e0fe5e4756a\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mwV5M9FW1jxkRGfVtcBcKLOEl+xEwfZhYqQOcJ\\\\\\\/o40Im7cFzM\\\\\\\/5M8W6MrGy\\\\\\\/vWcq4hQ85e8f9l8ocH35GLYQuw\\\\\\\/3HIxQgncLLwvdMBEzXAhhqYhxlKOUM\\\\\\\/xGrFI45LO1ypVwuV7pf9Df0JEj1pW+oTFh3maO9eivYPHsO1TmW4h+KmuRmrzZf7jEFs80HJX8bhZvq1\\\\\\\/7Ay69v8Z6vpmMQCObo7s+\\\\\\\/0Ujz1s8fsz9cflpCb2TpXpLDOuYI2K31ygJvj\\\\\\\/cbJ1quRSL\\\\\\\/elfHT4dltVo7Tt4F1xpClScW5VBXOtuOh4Si4p4IzXByX1Ep7LWZukMv24XUbK4iB2Wfe+cyijLu+Zg0rclcsxnFMsOnrbxb4LVTJhLls16UxlHQHcFVPEVNn2VwGXyAzG89MOmwM49AdHdZ25g2BjgWzyDKXc49jI443rjwLGuySy6oKcDMeIgBp3uJQq78rCDrV5HH0JFwmMgn9C3VzSCE4QDfyi0xBo1rrYn6Ygg8Aa6RMnzUGkOZpc890POm27w8LZNp43M6myXsaByZUFolGXZ3fzIglPsZvM+D+WhA1xa95w1D9Gp9lieVY0\\\\\\\/dih3UNCKMyUdHw4tTBfE4ZDdqGBywtfbXcyt4OsLXXwKLFekWocJQCxW9UwqFYNUHa9lNViXd6vrAF4DIWufNQ4JQiumWsU0pFp4JVhb0dK4EWYEB82QxaIMdVwDEvXFQuogsAgnYaXqUBKWNpc9UsMgMWj7hMVYSsq3Pc8gr+X6P3CrkNhzHgz\\\\\\\/tAcmZPuey2uvH+0cYTMPKZSCm43s\\\\\\\/aNoaGoUMJLog8\\\\\\\/otxkF8tC3OIcqqW6dUBLiOq0WqoDH1l3pMrDBzW0XHDFc1zcD23wyxPzfnwwrS3xHksdadfgdQZm1gzklV\\\\\\\/XBnNzeL085pQQd+0yvXwBID4OkJ0Y6OfWBUJj\\\\\\\/iI8ivQ4\\\\\\\/5HjenZ9O9KxDfsr2PIVIUWtIXBeLYxefOKYNkCDBWDzBm2x\\\\\\\/xCFXG21yKdhcIkVvNhrGN1P+ul+A6hdjtdx4H61ZaAYqmO7\\\\\\\/57+CxyOYzUFmM60Ioh3FFBC8Rmj82sKrK1vDibwM6HWfcWzo605gzMQfWzzxsEbGanj32hM4kPV9v0C+SWDqEskScZu7bOxtqVvkaraYqSRfwzH6HWQT70qlWqkdOsvzM+0N9fhI9lDdLYDYXNiCCnEO71GdRe4PF3IcBirBN8YsjAoNxCLRpHTuLanx\\\\\\\/v4+APOSt8wmAqHABP5wDuJNstJ2rP60rjdsIdPF2G32yyfKgEvGsB39jgByX5l6SnTxA+OppyLlS7zdHnafGQRUgzG+LGurozOU4RB9Hv8ceQK5mgPXd5lKuWt9U2CC7laLElb9WW55Q\\\\\\\/abOKr3cR4fnvLNHwsMad3UC0q1QvftXAIJ1qmTSKH2oDsbphvo6RMZyWh6ZMVixf9LBikB0WduEToeMcmO6hidrkSX+WeZlxqh9mF2ov3WRRBsuQZ+Qvqz7D+SjMkSJ2oiUJVT6e3Y8K\\\\\\\/g+k2zhsunuk2MbSIfbiEdwAdSluoZjhRWjrdQU6W9A\\\\\\\/vYwCVw83r9l9wxw9REQ3CNKBNqWAW9Tr8MEI\\\\\\\/XfQ+P1j\\\\\\\/OrAHlT+tqFX65Bc0Kd6g\\\\\\\/v26JpCWLB2zQaBuhw+UMkqT8W56odQwJauKgO1OOj9pz+LKYZVNaEt\\\\\\\/z6eP\\\\\\\/w6HiUxaKE3QCSH8b6UplPGYbUt7Mw8i3iCBz3Yo7RLS2Mmv3NMzZyWyPeWpFyUEfaz7sKyd97KR0ib\\\\\\\/Nya84APIc0O95usG+++J9dAl+F2pazgDa7OxoWgY8rrqYKLA\\\\\\\/tu0le7dD3cYg85r4Y2xSSounaFseJCMZYKzHJOyeKE18vwSFVNwOAkozQqv9reGaXdjmMbRkR211JVUc2p3TRU3Lc75o4CvZSvbjiPEvZddTaSiC8URrfZV0xTQf1n14fU\\\\\\\/WsJ34o1qRWXVPXiW5Oq0b+1agbxCBa2avslmB3Q9y49hw\\\\\\\/FBg5ddMR0kuVegNW0au1p9U8YrVUZb\\\\\\\/6BzAyhPqCEi90i4QwEr3J2W4AhfTecK8hp8VY\\\\\\\/O15U9L\\\\\\\/1SGPYMnUGLbWdcSBjD2kmQ9vJWYkl4vUfZ8QXnrxfn8nJ42nyyecXcH2mANfdZ6QXd4uNrgQIHhrgbZGqK0LYaOXTpO3HbRX+5AO4yOjjSKmIc8EVLNGZUwBhxQt8ZdXTDCqRh72CY55x9RZphRKS8iD56gtXJ\\\\\\\/niXPyHT71Q7vr0osrGoqzBPbU6p9wBU6oLVW5QHKXVCBySQbc3s3osHkJ0TgCLpPiEC\\\\\\\/kj00IhiKzpMx2tgfpQwMDfCEpikRdSWhDjIQYim71\\\\\\\/LbwOBNLmvFdCE5R\\\\\\\/J0ylE9ARPM6L1gUA8DgDReOvo8Wblxo7Eb1KnE9z5CY3F6zvdVOCziKB3c95aE9BU6OXVxfe6VvyfQ+\\\\\\\/2x22xBJyhbYW55tajX4ZGar89ebffCkHHHtXXEnhoigvjJYJ5ZasRLhQ5a0i6A18An65s1buEoIP1lo9OVtQaTXDZY+Y1IDeFy\\\\\\\/lyZW\\\\\\\/LfxCwY2GFH0o3y9YkmCIzHSceTWDpOVaee4KbS1ox9h2Hs4Ww5Q\\\\\\\/yGNv01+gsmpiWL+rMduKzwXN7RWIkkNMo9beHS20jY7j+MJ55zMds2oeqV3lVh69fWjochFDcerB3+3AN9JVKpbSx1HUETRZrAuLbxXOsY7x+mn2DDQB+xs67AL8+Vv6UNcx2oM7VUL4RlQWqeD6WPFA1TBzq34g\\\\\\\/4jroMMgbw8qx6wtQB8XxXxLvZpcUKdwnHbcszW5Kzcm9d+SCNSuYLDlClrBn8krjOMC0YhBm0UsvmkrPXd6qCU6mPxTTfR285RCQOn5IDm1Ff7LO2\\\\\\\/YrIjhM8AxHQbIQJ4tA42ZCV4nEU+KLA48leszvraREm8zHofZo0cSU2SCAXykLtpxtuROesYpJ8WLrwOy+GMKI\\\\\\\/FzOdmvjIhJj39Vp9vSCXpXMIlH\\\\\\\/Nb1YLJn97FMBZ+LgrQVu0d5WIRSCdEb0tpkt0ZM57iWk8wa8FYPAdAV0jDvF0NCDh9igf+wrQufxn1ADIVJ71gaCfaATKYRFof\\\\\\\/+ZwqAyAyiGnXSZdUhjnYqcKe9UYA6nyLkSlTYQIoJhAHB3WOSWb6XugtiVlK34RvDeVnacGYs69uy4DAn\\\\\\\/ytRqBGRRq6IIKAB7Go1AlGGy1zzZvFi+GVPu4yZPMRwiFQTsjp2NyRUXj7O7ncdau7p81avqG1Z7szBRZmXlXns3hNz\\\\\\\/JDfc5\\\\\\\/Sql6hcWMfoBjdhHS29+ieA6J87421H5i\\\\\\\/bPsx8Ch2+iM+lFEM282EtElhSfl2+I0r2yf0pbfel2Fnd6woAwb\\\\\\\/Pn0bZ7EnSTzbzeBcGmVwzbQMg9XFt3A9zADWZC681qlnPpTA2UIYhSgUMBLOgFvFbfYIhXn1inW46KmgBUL9J66Btn73irE92PG7hj72IDEOVCuOfosd9IThYBBzuEPNaLjqF0dtpeAFnDpMxp5bxl5n8R0kdLXGwNdzWX8SjvZ9yoTmlO8o4qpb\\\\\\\/XuHvJ+5R76SdMJftjCQeJGIEBlA2\\\\\\\/DSCMfV6+N2Ai7VgzcnKhufCeSIRfQtFPQSPNrwL4Fm8Xl2suBWrD\\\\\\\/pWHP\\\\\\\/s8RvvpPb1rJTYztF4LZNrQQ2WLH\\\\\\\/51ziPCKgRZbxs1zFKFTk3j+1hn+VtyBUITlPQtZkwkP8wVFCxPuhu5X7MkUBCCPCV3moI7mXFgtEMEd5V3yVLivPWxsmE6iJC3Rp4A\\\\\\\/5g25Eox7KV7RzS5sQw2N\\\\\\\/9Nc77IagV4XbtcFZbIFuYa8z8NgSZOEaQTRp1YLyKT8PdxfuYToKdhk8dhJzE+DtP\\\\\\\/WGweecFeeT1K7YOyNpbnhW2DdQC2ZaNB+pJVTxPQtdZA5kQp\\\\\\\/+WzNj6TrlgFDhneW8rF0k6vzQgqCC7nTE1thTy\\\\\\\/uazUTBIqHyaEfsQyH40SoyH6Vmk7i0i1DzO7VUWEFSgYi1Av6UWwxfU7jgaPBcAX7SsRfbJIbtho+yoi\\\\\\\/eEYSqExToqzFtg3OJWbLvdxKpabz2NCtWDfYTGAo2UB9t56WK0B5IzfssMzI+THEI40vAfkWF5nydaOiCVPFcL6csSmEtZhkBO8f63Rjw8K7BRRDoqsibi7sZvmeSJ8x2bFW2M4xawl+kf0bU4CNlDUDzWf6gVuZ4UT3SXYbs4d+3PFbxDzCYkJVzC5DgTqhHn\\\\\\\/anFNH18r3wOSoa\\\\\\\/eJibRjRYh4qYVG53wb3Ec7WOUaPGj\\\\\\\/mwrwUrShm6\\\\\\\/UxA0OVxgmjhWUXao7P\\\\\\\/AjZVjWeM9QXBpI6guu9eH5QukK+2gSotALFxhRka7Hy4a+uwdthKwXcLQ3Su0fATcm9gk2AQkkvt2T1dmRSXN6OG327PZSApzpZdUu\\\\\\\/jgV\\\\\\\/3dt8N2O6Gp2vDPqSDIB\\\\\\\/p96+2SeZEfTOm6RcJgWuGAE0dCPEP\\\\\\\/\\\\\\\/M+KROulJ1RzLr7z2ZLa2\\\\\\\/8ofZdp8XCc8O9h1N0HBT7HlrpUPx0mMh2PnIk3GBUqCq\\\\\\\/kL5VB1+QMHk64su9IS2DI2AW+0nqcfO6W6FMwYdZ8gnMRpIdw0qI2O5vtghzt8e8fYSsA6FJw4vAF8TMY3jOpIUHQYl7M8wBcOaoHojH3rvwO1RQR1GY1JgCwSvfFHX1BjHbLsv0gqXnc7vMFymJ160qpQMB+niJygijxs3EZlLXWwm5znKDMbsXjwALHXNmJ2lNpJevnuw5yPoeRGSkHlgt\\\\\\\/87M1v9AhrZSH38DCi1IpJy9avVkxxwUqIOUzU9w19vaO1N++HzVDqGsnYWim41FderA4b9zStw8RFsux2j4uKSouug9pES9mYCTjYt7r5X0ki1sQuvc+6217Psn8IHW5qDnftmeHTmjnB5Rc8vLlABuBhRRya8blnCMSpy7q7nYaOdZ4YYxfIxOa3ZuOpghb2FiFeflGJxOzu7tNLe3SHJT1yFV\\\\\\\/rhSe+J0d9bN1vhEYvR9DzT1nVxGDNK06yAFn+ewhoPB+ywMrl6SerfF1FQb1RPLWGfE3iyMH33hI+YAf1xBo9KRRgg1\\\\\\\/z3s3AUW4rZmFe2\\\\\\\/rSVlD7zYFsTpnKbWk34946HAUkP2Nf0bGcKZdB7IIV0H84t6RihAu2LrsHOA\\\\\\\/XQg9bYl05oTF\\\\\\\/Wb4nRSD1JHDOB+OfnTZ4kahz1pGAiwoNymm9fufbruQSlaIlur9aSyE\\\\\\\/\\\\\\\/gYG5JqbrSlvEeRLSxBITRM8YiIgTkB2\\\\\\\/i3OJpF5j7Qb03hXazufRlnJG0MfBZLFhwGc1eEmvguq9EIqF8r2RpYZ96NcSVJCcrp66vHuXtowkvWZWB1XrFEzjCs2ynx5qQDu\\\\\\\/Ymz5LmgJxcZRvL2odQpaEreihCur47OLO\\\\\\\/1hMa6ZviZ5n0TB01D7BHzIUOpYNeQ9wyUQAnK4H+OLDopltXCY38KIPnb7DN\\\\\\\/VeDcoAnWiA6K\\\\\\\/4jI8Lg3PXryRGZxbbYP3KHXD1qHw75VuwCUhFg6FFUnvb1RmrTRTMbRDGfNjRgXk5aUld5Tr4ZKtL+ZTwlG5JzJoqwfA8igrHfBdZSNLjdymWBu2c0qWqO6euFHGlvnJ0sPPHxwpUcAd\\\\\\\/bTP0AjQ6LNXNG3Oxmk5IMQF3RVVqZy7xG62SC2PoyOuybxQiygee\\\\\\\/dk+FnZ996LI5zCNN0UMzGNPxI3P9RsC0Jops2zg\\\\\\\/DrJdjwSrB1av8fcmBNkcqwLasJmNy9u5KfQ1q7Z3foI22eYwgJSVbTAofCxJqtL8KHzJp4ZRMNUhzkZb6Yn6mnz5OAyFZOQNEvHCXcrlEznqGetX+nrW7OxCnFz\\\\\\\/3qw+i8UP3lj\\\\\\\/bXZcuPMenjJ7bQRKIu+6IK4oPldUYfUqjcgFUjaT97TyeXuSVXNWzyUSwGdRAsUOHcbdGOiYpCoEAxLY+aBiOdHrE9Jfzpt5KFnM+w92MUH28HVpqKmHArJAESBBeByJ+gj5geaLhscM1cqIrWrqsWpIYUso1\\\\\\\/dKv7iiVj9WjrdGnsSiMophd0aDputmVqD8P75mNakDSbBbKYv4ouZ\\\\\\\/v5RIobKh5dCN1iF8LnvnA9bORr\\\\\\\/Mz22FCyqdWJWYnEacdHN6IGc24wDdaMz82WVg37m6ySmDGAgi97duiurD3UJoPDT10iMYCFND3HnmdnauCj8V7zyqLKtt2AE4IvNAnHKDhaTun+LMSr5aoMTWkLfzX\\\\\\\/gmxRnJwAUsI5JSvW01I3tqw\\\\\\\/SNhgxGcLYxtJYwDq2s+Rl\\\\\\\/vP8oQ7LOR1qdWG6HAKXs1Xq\\\\\\\/j8FMJinPnIkoaIQ\\\\\\\/GMuhL1bB1MWbeBOm2RxBHw5WMMc1igWIBxOc+cQARmNOgoG7T6\\\\\\\/42zzavtkTCtGNyUxYtZ5svjXKTCM\\\\\\\/JPRt93Yv8AGoMHy\\\\\\\/P\\\\\\\/zDJ+tupfJot7\\\\\\\/+zPy\\\\\\\/k1jsfEWyJgWiKHq84t4L0n\\\\\\\/m2TzihFOA6gQN+8NpoDwR\\\\\\\/4qVq7aVc+uQOggN+7epdcg+iuWqtL0pUTuSaI\\\\\\\/MFdGq9lHfbIg2gO60kBUrda2+Hue\\\\\\\/Gbzzpy\\\\\\\/CQhSJnYjvyLuvHzH\\\\\\\/h\\\\\\\/J\\\\\\\/MfJ6d4LgdoANbmBtK6J8VsrVKoFaVM0+XPwmaU+HK0Ze3jhdc9uDzEY6Q0UG3KhyguqNufzp2GGQ44h1xBetMtE2YkBQjj7p3E1Zudn530tlj6o6pbUbQWZNIWXhkRnzLdSuuiiDV\\\\\\\/3\\\\\\\/5MnmdCvkOoiaXeJ1aa7edlensuBwqowp8VA0AULDS3OMJdCIsgco+Pwp\\\\\\\/MHjfEQ12t4E1h1Ov9+ldb7bE\\\\\\\/RafJ\\\\\\\/THOREsyRl2BIQPYJvSbafyO7qW60oWuiZlCT9LdS8CrIF6sOjqnZLzYIJqnOfBuAMP4VkwBZ1ja1TKStjv0GoajnqfoBjfZqp0qcw5GvAtI7W4JmXj4\\\\\\\/Z6GZua9+6TPbq5GJks5mFIDHTRKho1DtrVWbJuYHUzrPIL6Lxod3lUJeQtUu9YExDeon2eQ5l79lEVY7cTrnZl8FZGaLxLQas2L9Et35e0OMGFdwbOS0IWsEbEwr505lz2HNtEtzGxBQzt0zcrNrHw\\\\\\\/0zpkxgdQVrU4NnZmMla2m\\\\\\\/sdprQdiDS9naC9p3w2ILTts23Z9MIb2QCyx2DYA6ePI3rsBSfpn3Ehsz\\\\\\\/IX6SqeQheT5po15f\\\\\\\/U5YRfiBqO24rxYOw+Qz4w4Dn8R31mGx2PZYB5Rxt+s47fvBuozEWMNy\\\\\\\/o5ait2L39kNHliNwMkqgwI54PdzHR40DovOHIXS3\\\\\\\/YjxpFVy3EKu4nkgWuUoQg6gYeVSlqVOS4GeekpTf1jB\\\\\\\/nUBB5deWm2S4pAqht49wS6nPW8qBF2wRxBE2Tfmu\\\\\\\/ngM47jrYXob6OhwuwL6XXIKdCBZuOZcd01g24YqTKA9wBupIazHQnPSxmE734fMDdiQQo9ffRxLQOAQFxxnaf\\\\\\\/v+ivJOnKzwc59Y1bLBF3fri9\\\\\\\/bwyaMRb+\\\\\\\/jhwZnMBzAOXbTCVngFzxCpqLy\\\\\\\/Lc\\\\\\\/a4v76rxaLhluvO4RCpO\\\\\\\/hzT9q8O5xxxzhe5HMfVefFf1anc0YOVh6yQOTvsCDEnrXVOnPuVipWFhX5WSlujufRen7w2ppxW1EHifRmTqXJLsm3Nan27JTMu46IQraQ5qJUJ5VQW3oAU8tb8dPlmB6isK2L070nhH6yrjnmRi2B5PcWfWpQIUTjfuicNxgtydDm797m6M7bFK419MIoLhvmZUdr9ZwSLYbJ125O6Xnkm2qhEB4ShBHnkDaIjAbEcjlUqrC7cVlG0p\\\\\\\/tXu8j8gMb5gNoFV6wL8K\\\\\\\/i66GMI2lcryxvSlWBRTHScFRCTsHkOmY5g3+74VZVvkbAR\\\\\\\/cGUvEbTL07ySZFaBzGNNx4GmA7cLOexLiit9K9S7N0o5djb\\\\\\\/wF9a7BzRLStBbkmXWOXbOtqLfjr3jEpoBgNTfKRUpFf+rp\\\\\\\/ywGQ0fLnL8Ctns0jvJmAlmrEr8zR8GR6N7BEazwwGRf4734incbzv2ZKozqaGh2uF16PStYkpadotU3zJeGEuy3JHeSn62tp4zB69fvZOaipM683xs9iIBgZ2slunwRSOCqQ6bcSEsiIZ7N4lKe7XGxHtsEhr70\\\\\\\/2OBIQuPfZq+rR54GI+8Oi+dUyP5rul62KZSjOmgtYkAPzUQJb0YtFVACDJxm5pT\\\\\\\/BNvTJeXYxF2TtSWw2\\\\\\\/kyDM0+bna8EcA2kpATesUPSKcyNQuFRenAI7aNu6bEiVHhBHwmoi+jdI85Z81FPTcu5ZXYecIDJ0YKnGiJFHOyIIBar98gtRemNb1FXqI2ic5nFBmxUnarO1az2FiBRYnFbetv62C3f\\\\\\\/\\\\\\\/9oeJlVD3oHBWPZ+XJtNyH5XqUeSYdFtvAJSF5HK1NVkPWHpg27d+rewUp97orxcB32eVYSd7tsqHUSxYJI6KpiQPXJDXXIHCWXuqrbZagKejGNpuQJw2XCs4SiMuRHhn6m0dry4gKph7ngNAHBkvy7PRGTkJFztEHy8rnBRZ1cL7Z3jjkmWu72GOV4kQLtAKTNetmNNQa44vlRfj0GDctr35SMwlfvhGtMLV8+m5EwsqbA2m4sKc8OHLlEN11QQcMWIffkRqoNra9A5qNKIQVsMjQPy0o7EAIroMZY7D\\\\\\\/jcLcuVZW1VTPHzuw9xXHE3cLOEcsemRrsLedyNOLGLVIK0YquZ8MOj1yBl3mkfBNKr6+4cbTP\\\\\\\/g4gjohg+YQUPw4TAEdykVQyA2it\\\\\\\/KJMSnk4hrgPZPjos8u0PGeQtTLyCpZ+lknapOEuxT+42YSaAOd42RCLG4hNfmIH14OWW2iQ+O8bs1Nb8z2g1GzDYaCmv5pxZuvl7E3tMEXoakJYjGTcvsn6Vna2cKZCMx13tH9mP7GtlkVIGX37RutjjeSOfpBzp05B30Uf7DhYlU3wjs1h0hhHQlIfOVjtcoGCtfDcc93MDVYunoxFNztgH5UmHka+zB1u7EOm2uJ1JhKAUhqS+pPzWOGA+vEwwxXRDhVU6SwsvhlIktGCinsiqqEbANbGjZ8P9NOU9\\\\\\\/4fZVkcMxWBjaa5mhkSMaDUS9IEGjNljV20835WFEhFFfcwYcAL35vHcgYMWARkZnN1DVg2E7m5Nt0Sc0E9NDzxuFAqhDsxNflRN7d15aQpVbLe91APMIB0Dkdxr4tdYvAC1mo4641XRaMyYeuB9k8OhVbg4aVUebDGX53+bqigxxMBhi5pFKOYVn8YT69Zex\\\\\\\/FXlDRf7QOc7wcpX7EWbySCmk2aFSqWw4LmwYyb\\\\\\\/pS\\\\\\\/9BmGMvVW7GrTmUWYhgpil5khyY6yd39nOJ\\\\\\\/Rijko2twxYuh+0oHConGf5f24UqqUB5QRRbUDwMEkjPPQQ0264VfEqsxcn4fNGaOG4J4lVxwuMdsdGcz43vVAvhjR2Z4+0Kp6V2AyXmdN0G4nfL4zKmRXMbM\\\\\\\/ntkcXZ+iwZ2YHuHznL4fmdhxv3Zog7gQSN98VaguzfuqC9094mkGdn82mSsjpxg5zzAk\\\\\\\/oFEMQS3dRR3\\\\\\\/T5bm2uxuwGI\\\\\\\/v2Bksme2mjWWF5mnEeNeAtHlP1hooUfDVLBtthf+9M\\\\\\\/NJisho60lEkpeO+l32nR2w5Yh6bzFDrAwjc3c45m9fMpvSzIFD3bIDWmLMIotEwAH7HiM8EL7rftNsoHS8DieSg1B7Mv+D54EpSYTgO1pwn19nm3hiTz\\\\\\\/IWk0vIppV9sRr4mK9b82tBzFFyQslqN6dFWNVh++icJnwYGrfw9vGmdaIK7OYSx7zJly9wo0H0Mi66RELJeBLgWM1yPGIgkj61\\\\\\\/RbIPKOQHhw1qBoI+P5kI5MPbtNZTuIbuphk38xsZhoyTIwSwFnxH16x7OA\\\\\\\/fJrYiave9\\\\\\\/dz4hKnb9ZxP8Aqqu0x8DlNHjT\\\\\\\/GgWqJ5fn\\\\\\\/0AffhkGNo5qCAU5eLlR2lOhxWzvnG7gkXU6t+92PdYDzJ6MgH0diZ4fB+erNA5siBQ\\\\\\\/Y\\\\\\\/vEFn8BulG9gMge7L4BmPk+0G5aKJVdM0XgyrW6lFphWfCLW8g\\\\\\\/PMapZZVH3GartjOKf0VT1KIjUVCsTbQ6PjY4lZ48+sT49DUkOvAhfpY1kaOSPigFtdni8SVfjZM+yPm+ICCuLkWkX5SeOxMULS7HbBy1tP2JF1qGPKvefynJm+7HLNnwQwGxf4actDtt1mESmF3XbFGcOrufSHCa88FjgmFZhoLVA8z2hnDvBiEB4e4+OWviTvDOhGRNvSCLB5BHTqc4KbHPT+F4JaOQd+hw\\\\\\\/I4zaLe2XOXafoVNHmzUBr0xLOiD+QfqD9j4+xandxfNYFJZJ3z4nSen4FnREWS2SIsFeqwP\\\\\\\/Tb9WVR\\\\\\\/fYnz\\\\\\\/QmNWlzatzrV67BSHcPfqioWS43HNRiZ86DgoPP+65T2YA+\\\\\\\/YwsbyndNTjRWwBzuJx6OWRzlI0KOjruDTFQ0ADyPqIJlP6Q8PqBdfa8MrOREEzbdAwlW7F2gww7U8IPazyWA9M6CxK3q8d7Hkk3ZcLzkHxAXzS4J39lMSLYzU6IuIUW2AJfwwgdvpvJWvS4ngm11RCxKvYTW+SMYoh5A+ZUgMc9j36nJXfmXSVmwk0brJHm0J28cm27YtRXV28u53ZXIawPPFhPnz18l5Scyjzk9DAoB4+stgdk+2OZK2QkvA+WlY6Z5Zj+QXMlDplWPHp8QIJNmmNUl+kQoBv++o9qtBd\\\\\\\/R0rOVc52yF8LkY19+T9N1JhMp96LtwwUIrgcAD5OuQkqnsKfXNxt1P4QWqq7ls4P7i7qXr6cWc9UUFofJeIWBohkE0aoKEHwN9EFHroxuqSFRifYkmFRwFXSTx5OWnybzN35\\\\\\\/1yu+8aP5ZaRMiarxwEJNbLS1\\\\\\\/2\\\\\\\/gG+hhXR8jNZAlWr5VDXVV6l++qUlGfFWNsQf7v13WXzDVGQU19wWA1L0to8fydeTDCZcHYDCZ1+PzDwxwlYiwLTWyk3BlZuh9A3liGzi+GzHFLkfULyUPiN+YwJxbLP2EXPh0JN5T4+m21hKISjLyG47Giu8jBRB49aH2nSHUC2aJ7OIQw14qAa6k9AEGD0qWqAKIVhmwrrFHovDBi54c4HBMVBkZNrydyWx542K1ls50SI6cFzalJhwNzRXiW6ZLWmgWVnc6ODzRI2T\\\\\\\/V+pVGevRMpGCfGjKmGQFnFMbA1aTAQ\\\\\\\/5iWcC2Io1GpCWJnFNbxZBVYRL9kl1WQ\\\\\\\/FTG2yFiIajcUzcvl1e4fyYazVFzSsPkaX5u95JPoG9REmtliqrGSmbh0ZmnlfEy4Tdva58fgZ2VXL2bpZGJ2AvBfwkEzuc60fQT\\\\\\\/zgYzmLIhYusWp4RxnZ5kslskxmgKWlY+ShIGowDLb6a4GgJD3HOrsFvLRW7UnBHppAXH7YyrAhnpHHnoxy56Qdq+VOn+NVDsT+VRuB4oGK8NGAYGmILvslx6l4DajX7P4go8PdbJoF09S2+\\\\\\\/itjYgd8OZMAQOfugT7fDd4\\\\\\\/7eZrkw\\\\\\\/S+9jjtH2EaHtpgug+PNK\\\\\\\/SQDaKbAIPUtPE+eeqbIVH1rT4EqIsUgLdu2MYb2Drv8kw1IBNY2HbEiK8S+rJ3Bta3ug5Rd+AXPRYW8ncNiZWI9DYeNcG\\\\\\\/2izZc6nmgE55BklvFpYGhLWwJJIl+\\\\\\\/HgI35EcmpyeNnxQvuxq4K+mtiYnV4ZPZqlCQg7xX1Lyp2OR+wlDY4MTrdRpuPZhkYwLnR7cUjqm7y27gmWOic7Ib0o00j96bx0R2AneU38mlRqSbb4Z5jKpcKwepgn\\\\\\\/pxuOJ3QwGrXynBpER77AGwXA91W\\\\\\\/5DMEGUCwvgR4Y4SaFUZ6RSJw\\\\\\\/Ad+R3jZIiQ0GQWr0PrspAvnQA\\\\\\\/iIckT9nuTuvNr1EjiF\\\\\\\/XLKR0vxyY4IgtmGKW7X74e+qLdce\\\\\\\/RRX9jOzJ2oHtkhhTS0PVLKzTHz4Y6cJji5ysG\\\\\\\/VeV94kiaqLZiQWmBxrmcTxSmcstSOIVts1rh7uMnVenNicpuTe+iszVj6yeOoSacPv0YU24h\\\\\\\/XL8iWq+52Zy8pQu41IY9gzlAuzjY\\\\\\\/in5iCfuSx3bbqIUucY\\\\\\\/yXjqDDhv18+epYoBS6U02iYwrzU7\\\\\\\/9TE5LAAUYQ5SG0AdXZdMusKPBa4hJDdb11LIiAbx1dddotgcTwuWYlI\\\\\\\/G4FuQa57aZXQH0xDLF0pvMdeMNeigrJ7anpl\\\\\\\/Em2X6sJikO2y5A3UuZ3NULPNzGMGdQm8iSWttYI0I8oHFTC7JPRyoEQd8x3ZCs6MJ\\\\\\\/kMXUE2mJcqH9inMYlISRmbT4nFtRDLQjeka88S8Hiwrl6NPQJZIiYortn\\\\\\\/QYQVKVmhZwR1qxcsL2ZpOvXV14NcC8HYnuAywWwvEGAwPaoRE0jsxr1JAU5NRZZ5Nfpt996nYJALDOX4uySNth8K23taSWYFKjM+20gt07l92nFQNtPNJuBj8zgoAIO8TeZLWkAw8Wn\\\\\\\/r6mvRVMYZiBz8zkX6twyXAMQI\\\\\\\/oJPYaPt9XuVLlatwX2n1uC1KwRyC4l6VXsJxaRM1SvlsmMZUKB\\\\\\\/OiBnt0THKDJNyszyzHwzAGJpbvqmdQvBZODy0jpG09HSxjCQAQWq9RT41cfW4BKX7aUXaiez6nxC8LxZ32PxDNDVs9sTtWILjXY0goJvv82pblHJWJ4NiftEgZqpstD\\\\\\\/ioR6hCn\\\\\\\/fKYbzG4oEA9K23t4xJbjY2Q74mWwb2arNl3KN+TbxC9itTIEdAfuG2vXR1R6B5umysn8wDbG+HunAGikFSt81cnhPjVykAJBJqSG6olyjAbYPiDgpDdIRcsCTEjhpkpMXstSM3BnNUy4iObW9M4N9gNAqdLsoqe9m2WtAsmU3hMLsqKwNtZ0SCURWlcPkg9cntTZwqj8KV9B9Wf4+YzNX12ss6uxEcaaWVGc4HT7l7dlZ7Q1LJYSgmtHwG+CcIqTuT+2uzEA+ec5E+nv4Vt7sRnqaGVu\\\\\\\/rMhpW2WdeabRDvLhDSf369Ryog7esHALhbA7rKawINGx+m3DhuQbT1Kz37Ow0mYW571ZYI9hZ0fY\\\\\\\/AHHnXm\\\\\\\/AeSokNJg7v++uoXCmWrw4G9c\\\\\\\/4DV+ddqYHVX6tLHSrRgPc7CnPmALdAgLA1wNSaL6uR7A\\\\\\\/AjMQFYhpqQtrViGiuKeJrEkVtUeb2tCuQiBikNwwFwiIcvPMCt2vB7eCaYFQPJpGNwELMw\\\\\\\/GlOwBe6YvQGdrItHC4z+vKnp7yRXr\\\\\\\/vG0yT0b2MCntZ1UreOtvK26\\\\\\\/8R7SZ907nRncad3dpjsPHwjU\\\\\\\/6XgBZ8mu0FfcUFhw83VwFeI\\\\\\\/NRb34kottgoZ9asNmZ9L3JRhvrVShItdzCP694weqwDRw+V7+I1f0cbpHrAXzBkX5abDC+Fa0nye8IXPFHdt2gDHnT\\\\\\\/PLoiIo78Sish2Q0YKtZftOPmUSAl7VSWLuf7vN0e5MsUkAvNDcUfyanW\\\\\\\/ioA3ernNTaAzw\\\\\\\/BhvzPKrnox7a4azmIaXIoZay+ig62536mJODIuP9Xk8uu7g9vnQCVXqHC1wbe56OZbYpND\\\\\\\/KNdGYeP5NQLw6riUGXOfcYmiaMXdyYJ2CVkKQ7XPE0o01y+G5xJ3f8SJ1tp9PNbjKsHFY\\\\\\\/vNuaMVQnmtqtRjReoBPFegnSjL5jaQv1AVlE4VXdBFZbQirg7tiKq1PBxopz7Q8pvLOAINZKFZjDOfjSA+No5bXWSYm5Y9xXARFp76l8peY3Bh47cRqhk8pqoT+PEofVW8urx5G6o20oiIZjcL2V32pKVyw3XwcEtMKQ0MCB7TF1+tDwjY2YOhCXosWfcwvfCMDHy9fd0uqlq5re8zOa9KL5QKZwwvtg7mMiONEhkr+bFRLddpYqxZtv\\\\\\\/Rx6xKvSRtaRVZ02eCkF3c7KxUc5pv1jyHJi7+GzIviIlji30SHBnOnfvhWM7j1DgbiHiiWBnCkwuC\\\\\\\/LjDPoSN0rTCKfxwfUJWXSCgvYIzPhZcaBcSauVqRc69M\\\\\\\/82aGgoaAgkSQo9B8Qnx1XUp3kGut\\\\\\\/pYW7LjliiH35gr3yEpmk5tK+cj6kVB84OrCBCN\\\\\\\/UEuShfC6KE09ixH3BJO0i3sZCVvqtn0CpPQNNd+bz5+FPkF6lLDiIFLbsa8dA3TVPW78eY1WFW6rN8srsQ+8yyqsqLINGIJiBjYrHY2xDAsjYUdWEnjSarJ9AXdqqhcb88wfW2mwGp8fKOEEM9NVgMZr5ETGb9Ezk5VkyVh31YCgCExXgq2up8zFlYG5W92Lo9y31cBwfhRMwaZ1PvDC1lCFaibRmZR5k1aih9s3GNd7t36Mi6R2nglycjGK6bJywkBYva+3e3TykelJT0LfT9bcEo3NU+QWdbVFIgRX9uPATSKqG29jtO42rfaJSwCTZC8sFhr8dCULzEnUzKlLQ+BBjdBpPiUkVgHlpEzXjflozBbr8dSROw\\\\\\\/8LUt0x0NwNdbT+IBzm42+a+J5IYfHOCQIjFEwQKVDGytSmPSPcnLvqkMslVI0JUWBktTvBlH7Hu8RtlkvbRhcVGrmFvNb6J\\\\\\\/lVcw4SR1fXbcyymNy+bAaSGCmUSDPwcJMbpHKcUlX1MebGz9\\\\\\\/FR6UdlZEDnLEZ3DdXF0W8D6SMwXwp43eFTaNOzyBCzECYFZ8hbGjjVGKh0yd8DEmgprQm90TWpbynjljE3e+llvHFcpFbtrLRhsmTyC1Tduuni7UsGICvb0t8RYawc5S8svjxuBUAWxsC7wfgfz77puo8LQZ2WuyAKvvKWmYkBPDwLORV11FwDSJbAqbc1DKVF1PYy3qg\\\\\\\/wjiYGVp0F8Xll1Boe6p6jOa7UIa\\\\\\\/6J4i7bwP2zRVaaJ0gapkVB6Ek7W4Yo9napKq7k7xp\\\\\\\/ClxbysAKFCfS7eKShQiN9g2d4yncO9imGCtp1uN9dGwFinhvMpW65T8uTF77HLQjPcp9XcPnEHOV3LRxoyW0tpeOLX932LvX4o\\\\\\\/cqNuz7wsyTqxsw2DDd\\\\\\\/oabkcMa6feT3vHk07Fzx4XG1IF7+m8+irexwwDIEMwZsWEmSyQLyBKWtKtbWl2rqXiC3s4fgeN0xefqDcZ7uiCUYpZDG26nyZE99YRSmhUIVfI\\\\\\\/Uhn\\\\\\\/FOLnXnP2PHf\\\\\\\/N0ERkfFyygzZSeFbI7cAKmONQyDtKRPt1IIysnCSB1ZZr4ElkarY12lZiwD68qF7hWEsiIiOeX5juDvYNozrnY495GW422nGh603aDn2+CTf8p4eO0Tt9WXL4n71zu1m2VLWo6MH22r7XWcCBCCUzrpTlExtg\\\\\\\/pryl4hC+40xhkMdn+xGvMJRoRulwaG5u2bd6tWIwTJOIIJxlBAuhqwuN8XJBjZeZ0AtVbtPYNQLHzEvAp3eOyA+ukAVG2Prpv7\\\\\\\/ko8siQt+AU1K8SD8QCUihQPaYH8uXU2\\\\\\\/a47XnFEI3N9F3f0co7+LlrY+qQzV99x3A8M2xr3wM3J2kXhKkSSl7VmG7WIcQeAqdL6iuCssD7dG\\\\\\\/iByxHB0B0iBs57mp1Fs4P2tudaWLs3QD2AJvu9Dzv1ewBzp3ZOCLk4u4CjWQgbess\\\\\\\/WN08DBZ9wyNaKLjRDc+rCNH11AvFszt+8qSpSPM99sdRUYYmEXUmZ2WUaT8NFKmWkjrNKZsrisNuytL28p9vMr3g2rV9\\\\\\\/5Z\\\\\\\/Z0U4GEEbS6fdkE9S1XsAIpH8TImPiwY5fEjrRypXJIyOHXtquX0YBYzXTf77zg4CABVsy41f3lMeZsVIBjSeRPZ\\\\\\\/suHguOQKKL69aNWezIHNpEv+HPDOnFDU65HxuB7ny4z2zyvE9SeajoBIuqk9gvRn6\\\\\\\/T2TEnsLHEJdBgdIXiuzYrVpjzV0I5psqiW212EdEXnUp+gmRpm249h6OBnuBRR8TvaFNg4fjFtVTgevlUp62qPN0c7YHJ10R8paemcp\\\\\\\/YrIJ3eP1FIxPBL7PcGOT78dezu5fox4V9XDRuXvh9zs1o9oKbl8zUYej5W5FK9iKgtGeqKgOQBvXO5Cqwdw1eOcu83jV836Cra5805Rvy38oIxAFk9OU2mLO8k5lgZs\\\\\\\/V2PSDuLeHVUEHcRD8vMMFx3BUGSJWu8UMPdzQmu4pEF95SVyZNv9ZggDOnxKahoEGUUSWumBWNsxADKJggTmjD\\\\\\\/m9Ep5\\\\\\\/FzYQUOvoawIEbuCQdOsMyKvZ3GALC7h+\\\\\\\/\\\\\\\/HEu4Hn03sqOImHIQDGhoK+nPdPsURXcUOD3+VpKvf8stKvryMrptsdfufK3dSwq6mgKdlMYfolEPnDTb5fwZ8VdWYaoGjlnXYZAZ9uZzyB\\\\\\\/683z6vgq1hqUf6JTCnpo\\\\\\\/g7RWeElfdbLuLghWP8jvKfJRyaGD6Cg72DtJNpewp5oFpp1qewM3kvgziKrlk1CW3dL02unj3jjc4t8AJnfYJcPBxK6XKfeBk+VvsfA8Q7iOhxXRoukmqdElSaqhRlYYoE5\\\\\\\/OEUXlmMVnca\\\\\\\/5sN4jIx80mJQmpBl2AKWRp4dUU7jINMneeIBzHCkZ3InR43HDqDKdzip54MNitI+YsV3G4jqXEYAv6uT+kXSuo25+3FokvLpQD31DpNGJROhXTXSiBpeXzY\\\\\\\/d11aMYVVt7cEUNQR7WuEyFtRRclDZOXscv1TZaFGI0unQLjH1CvMYD2\\\\\\\/ZKdeBl5HQaU+dv9JFSUJHP6cO47fp2LbIovi3rKKpcxEhXTO6C2G0ty8Lc69IPy3kj+OnKYIvwbJGH54RHVxRnSmwVR97OXNA1IC0\\\\\\\/xIdGXUt+VedKsd0rMPNryXspyt0Y4DoDMjCjvwWUhhXskazQuEpqWsGkWyhoTOjl\\\\\\\/BAPJaOgpNSme9aqdZfmMPBcCbjQ+F3KfC6iFN\\\\\\\/QHY6gqZA4VTqtHje\\\\\\\/Ec2fuwsBvPN+RdeYOCRO0csyF2ICbh6CL5b7Fe1DcYitnt+Ifi40zE9Obd1WVcQzC2PaCkjcpjzk+1c1ZclmffnV+jG1zxONU0Kcv2w9QcMV3BrmCnvSy0siray4P0wVvXwqHRASbXhy9xrbxuM34GKm7serKPrttC49MveORQjQhyIAqL0qEzS0DDIgczobHmYJ6DJmifaxUMDlLBUpy62nVEBtYRPNOrvTS1r2PJLYcEjVKYF10jkIkV99m4Mo5JIvUFPyDYWs6vMU\\\\\\\/IpfRnl2SVXCvJf8CSuhm+KnMXmvbKP6NcX3EqFLS\\\\\\\/PUjWZ\\\\\\\/7yZ6p6movPaBB6GoKG1S8JLJ61SfLJ3X\\\\\\\/6Nun3\\\\\\\/53TBB+\\\\\\\/9SUM5Jl8qFg\\\\\\\/VXVbDDWsClKlxrU53g9euMIDIUq32N72gfx+iHX2ABIe0ldWC5Yw6gVBcrZEb+PMx4dCDyOliI1A5UUECbMO7CjVC5dIC0E6GGHjACXkXHtZIEqJfSpHrpbKYO1cDSAIW9XMGtmfEobrxl9TY\\\\\\\/Qa1Zoo1M\\\\\\\/DN66AhK0ytGY9HzCNzLVfQt4+8g8qQBTltkG0N2PHzIlZCJf+jIzv+rOVziH95MUcCBlMs7WlCgKrqui1yNgOBBhFbseZzv0TKYKwhdD+FyJVLwwFgc+TZmswvEg7uFtR48w0fkCFEhWW6zcResTlgMk10TRRp9M3Qs6uiua7RUxYne5dJbNncMeaxEjSWVORtIDeApxnSZjaRhhAE5kIO8PkLPYK9jlkzxCHLnmpXIws4wGqUYPME9krS6HQ6TvJrvKivMmb78zjoP+wjdlsRtPGa7avmj5yBW++tB\\\\\\\/8r+imGRStKC9rFfsqY\\\\\\\/8GYIaNJzriEi7x7Uku0\\\\\\\/jvyxsnWfXlw8iQ9rmOV2j3HScg2nI0AHB1LjIF389vNYGwwyeRBQxB0CyDOyFgFVdoxsO4aqVdAJA2Sqy\\\\\\\/Jaa0HBsuWJWLcqJQ1sP2+KgyYAGEcRAbJb4mjem3YD0uLffXq4vsFg2n5inNPgRwayJDrO8G+xpUylpFdcsmQ12j9ZMtPgFlQUA0bSLpwgXKClMdVVrnc49V114PQUVY0WJylX3+RB3gecfD1vJqdOg0qAHyzJKdBGepGIPrsDFKVvvQ7+9h1JMH86n3CIFg6Lo5IHhqy+bu1flCFQWj5Gw5VIh5EyZ1EFUKAIRHoLCWVcGZRR9sGqspKNM10c9QqFjSTSSLLmE0plYX+ecEtW\\\\\\\/rQiRzv\\\\\\\/v9+5SxS0K30a89+UHCGjDrgZwtMQTLuHa24aS7qvhdVZGIUzSi2Bt5u9R3GFIJPWhX3QgOwsHhgPP7mCDEWYY9NLI9XGkLzstD15UcoTdktKNjZ0cz2RBweuq7+TJaCo+waG6\\\\\\\/r4wefNnnZ2WtyXkhZnbFxvISSjbKiXp321SLQnm8C5gbosDiie3VA+yj0ABQmmj27SVte29U4TUpjkHIpMPnPPLGMjiFKItv5UZYH4gMGBbWmGNQACHOFRPpwBfnLdDDi8wYx85e6Pd1HURQslntoSlyCix8DiFDCTA9fjFQSZhcrRW06axPgUh8Hb3vvWmtefKuz5CEK+L\\\\\\\/IgdngASVWzqsxlFBZ+zC3H+MXGNMEtdpx3gzwCnO3LRfBJXQjozvCDP2R+sOfEcuEjsLOpc1gZT1NGxqZ0E4vT+fNFKnI4kq66kb+yqw\\\\\\\/cBPjEnCYbXhNCvwh9YzRorJRZJSpgGJJDXqyAa0xr1jIh6AZhz7HTIi+UpssAq1sgFl3z33msv8tMFdD0Hm3SsLAh8FBMCPNqzt7roFDbT3NURLj4Xfs6243zQ9CTkQCw9V6HkSMU8ve5MCM3rjFl+MGC2i6+RtXRcQH0zsCpZY3Xks+E8znf9\\\\\\\/+hFaPphJMKgH8t9my+c3Ql+dcVxPmqyHPRSZtbkuEbB+YieFjJyNHknW1vc8q3882+46\\\\\\\/nd54DaXC2WKnWJv80mO8VUKdRBAwU5wyRARfMKG5o\\\\\\\/MqkL4GlrQ1MaZfmlQ6sOAAeh85oULIm1O00BJSQnxdklwq0y2hJDJIpwboVDhOSaKBHQw\\\\\\\/nVBkvU4pqjW1TE9tp2wTqthE8fA5PYNgVsI5UOP8xW0MMEHmXlo7owoJNW2HpVob\\\\\\\/U2j7alZbrKM4xSGpOiupMSLUrJmmbhB6vU9lLDzeeMFFSv8oLxRdW3koU3bAprZ\\\\\\\/cEN8qmFtPuPjjTLHeeQmyGVqVUQv1Nq++Q3dD0Hg8dtjJaseUiJB0lIyXkHLm4dZS81Q0mb3nEqM4IHnCGal4PtmsnABfyAW1YnXddLp97jVeKkFLQUt0i0fL3zOcprEYbc8fvFd65WjQpA1+5XW8K2ImIO+iSPBwfllU1BYfIinsz6DemGgvqOEYl4Y1ZfmessuKK8ZYQvC2jTWod7cPYeavMN+HxU2BWJFL7blqfXmqGjnvUAQmqguggTTmP7vkTSaCK+01f4hy9L3jd54P10Yu3WkNdPjbJM3kMsTWbWo4VVhkFQxuSch7qs86O8CRMcLSRKs1Gt67cx6Pqgv4Z6URoLye2pr1rKYca4Z\\\\\\\/t49FS+pC2VEQzKJreA+kQIwViKze1k3Z0SbJbjzMx6VEgc4Qr+2w5sKa2wfAdmWAT2S3AS3AexA+RJl1L9b9xZQMq5lOZEsggSnFoXFtGMvMMobSWEIi\\\\\\\/pn6C+ewFkUkYg9QLp+KHEH1LBm0SAKe\\\\\\\/KI\\\\\\\/GGjDZBhd15jNrZ8JmpZZgnnAv0tS7kbQD0uZu2pWgDDfkV8\\\\\\\/DXzWp0z2JEN3\\\\\\\/TgWRhBKiFVCKRGXyUfKOwCu7x+KwzWtC9XkizEfOfw3gZqYyJ4uC5woqQ048JvrMEetFBcO5Vmq0S7MO+moPby2unUI9Ku023QzKRSbnFLrh\\\\\\\/OwJ\\\\\\\/3eEp+42jfbgoE8+1+ViP1l8WOZspU1kcMLDJS2SoZEh64KyG6CqKXxoNDa5a3MfqQzSCfUGockOdrIgLHDPbPDBXLW8j4EVWmkA3USZhi+1FaNCHY6\\\\\\\/WZR7I7ebN0DoEjsv5Fg5s+ZYXu\\\\\\\/wYZmleXJCDlErkE1iR\\\\\\\/ogoRX0aC4w+YP1p4\\\\\\\/qKMyU5yEIrAAuwIS4WDNx7wQtf5p2CO6EHVaV\\\\\\\/o\\\\\\\/5tijMqsistI4yzMO4UwVDElI7iVmwq7rELRWn1ZzJdsbo65paqYart8C30dk5dCT2lGxkcwf3P+0sCBbr6foz7BsR4dbbLbRfy2qSeiaBqoAmD\\\\\\\/BXcjSMSIdZDUS9XRmcltHbJpavnO9FIiOR0cK3VP07ftFlTve4rw6lUHcGAhzim\\\\\\\/L3T2AbAb5kaXPMWvlySFP59zOi0+2L2s7JfWD\\\\\\\/SGxCbii3biUPNKp5tSc9IrRIE1IYSvFxcBFyjKLZl6\\\\\\\/RiCaywfsGkPpZGj2zIpJBZRT9nO2oDx+xdzApE83oGmO5Q2zdR55G5OjMoTc+VV4qKY1NpHgCRHkdfBYebRP4W\\\\\\\/efEewR0JbnNvM2scCZWlpsD8x1JjWcvs82Y62eJOs0F\\\\\\\/+MgUOVgNBl9VuqkqvOaOD9O5CcYbvaP3lrBahbjvUGKyjILl7o1zllciCRZXl5yY5k6CziyQx7UJlGo5BWlYMWjamNOhETjKsgE4Ug82iJ1ZzR9pFFrVuPsASBAjOkcL1eDglrjIwIOALvQ2gz48O+bR3Errx4a168AKwIP90YZ1dK+sQabhu+SmHB8et1Tw53qx0806jokmsWFMh96VAtH6wtaluRuKX3jcxyLAt\\\\\\\/7z3nvrsimABLgKu5mmzOWNQHUFrOT4+cC\\\\\\\/QMtxz9dWGShIJ3U1EDmbUX2lOwZR75MUZHCBimEkbDqmg9WUPAHMBCEZbvbJxKqZb9RTMRb2aMPc4+Hs43EEdd5bzY0iQQstNU7QytTylBDvEkKuna2E8\\\\\\\/xtH79ufSZlgRWRVsbkYfZphBW8eoD8RYCSg9lBHx6ubzWGbpqHGdBfB4ZmioBccHj3B+l0sYXanoCKke8MZuoa20PmoUkBbn\\\\\\\/8iaw+XKoxOqJsQWVpcALB2Hnv8klwtzNO4d9OqbUpskRSn13v\\\\\\\/p+dLsg0yhRxNM3Yr5AqpyBRKjkUvPP1K+GXmRxZmMO5hjo8GDcVXqVMRaC\\\\\\\/UQ7Hc4bdCMrIlpctqtA+U6lupwFq5U8Lst4fo3C9D5F7MxbOATVIeqdqX0mKwxcooy2eoGZeyMM8W0aAnpLJFTA2tzuew+lJNooXpqB9Ml5QLkeLvEY1YOkKMH7NDhyx\\\\\\\/SqdkbKWUXCEikhbt0UNKHhk+9HD15YTb\\\\\\\/EfZXPhNAj+Ue+W2jX5lSEGLRAUmhVcE5lnJ72flwbU+6+gXCLuiNZfKZ5TVSaxXf0+MLkcifHjv6Wa6woQDEs4yNO2UIHeXxV\\\\\\\/MpDaDVLWb53Tf+Pfww5tVV9zYd\\\\\\\/klgXG7yau1AvT9h+oVqrQ2o3v+6WKd6\\\\\\\/U2+W7RVWjo5l4XvHnZoWHUbZdoHari8Ay11qRR004pnmo4p1zfbN0E+NgZMge54VRP+x1n0MHh67xofMjb9BgWU0aeTX+uPzZtpHy3xfCwDosXDkeRnc\\\\\\\/or2WkQECiU3fUrxRAoQcYxAXNHdNCvX38qmudvYGkwkc+h0wq6SaUK\\\\\\\/4zDICgyvqDsbAx2oxkMl4hnYb9KYkKIjGbwXOYQgVS4YQWjIEb8Bj1oJn+QxXgk4OqmTh4MyWgrbajkoO7U8RaA\\\\\\\/i\\\\\\\/FvtXEuwQtBvfAVCBh63cvhaF6WrCytvitEZigodZc0kOnUiG3bBmNKiO3i6y1NdCKuX3A9zXBAPGb012URFMR9jIbYd105hNSkD8zlfaYXvQZrkkoxKcuj0vtvyc79b0oaX6XLUPFNfaDJ9R2af6u\\\\\\\/6MaoJcYT\\\\\\\/SQzhR3zvBbo3UwEjUHvq+gd7KLeZHGnsy448b5jGBJ3c+oN0kEZd02o3wTNRRFIgK5oS0DqQH+ZcUERf6KwPfclD7iZN3rR560DWt6QN2DxapmRddaI5mK0j4S02+IsOP5cw1RvgnHY3ipZ+3S\\\\\\\/GkWryVZ4mpD2HgwqwJ2pDvGd\\\\\\\/7YFNTHbwYxl84aBw486HwaBegYVE9gsVfDqNkhzL4yiziJp1DBW6AV2pCrFSgp7ERMY376Iu2ogQYEH+ydqyNufzc5Eb3\\\\\\\/43KqqrfJvat\\\\\\\/bYfxOMRUhtGSNDxqIXxiCHstrt6N\\\\\\\/MdMZ+\\\\\\\/75ins2D5P4NxipIYcdQHmXVKwg6OEn\\\\\\\/dYp+ghE89HclGkVElCTYKuJMzmq\\\\\\\/WME1nxaCF6FJlIDO\\\\\\\/XS8imkwAJfdzHXjrwBswwb1cdzq67C5oMVH6xfVhJ5ToLqWy9q07sF+3bvuYGO6jFwZrA+6kzaZerLHFAn4J4T4JPGCEjr\\\\\\\/XOsCVFsbKVO6pi62aEtB9eaAoGs982iwIdK\\\\\\\/JVav2SSvKpDZaWQVSG9OD7mGK7\\\\\\\/\\\\\\\/B7o\\\\\\\/4MV5BuGmSQsutti\\\\\\\/o\\\\\\\/+jaxf7Pyh2d1q6ar0dRfBAz4Y\\\\\\\/SMQJb\\\\\\\/DeefnA6ZRyoNxk+i+7esX6uFua2BAgfBu3xTCfjXPjbiXxD2S7msZ2+EhwpbXqIs\\\\\\\/y57tr3d2n6uQjwtQWDdipPhRzyYGRqX7H7SDPAQS41RhesZg3wb59n\\\\\\\/dps6ycAVx1\\\\\\\/7r9z9p1kjYwwEZ2wSuiOyfcHbt6\\\\\\\/gkjprYbzeyr7L\\\\\\\/HIFzafNYXjeVDImkT2jvHDPoHc6MAq620ruMUeIApmOjUZeXZA3grT\\\\\\\/MD6CXJlhigvBWZnhkTqWmmyZBqRqqI5IieB4Qu+LPx\\\\\\\/2sm+4bRkjrCwX2Tc\\\\\\\/aJd+aEGSodo+hUNBokTeFHnD9A71w5BQXbojOc\\\\\\\/a3bu20MlmO\\\\\\\/cqpRhPAbjr7cDSjexj2M6O8wKsc9xFmO+bVvAWIinPZG8Xgd3uh\\\\\\\/IlhewrV1swJNyrVkLYWjBKqdQtyzIqQ5MPTz9O6Vtb+hmhXxtF2nk6DyR9dwOTqIiZL0IHpS6fBQ1+Ja9sKt9XdU317XYIStUx\\\\\\\/zGt9xKro4GHn83L\\\\\\\/+EVjlL8fjAA8jvlQQZQzDzxLpB3V9TzdWKlWgpHb0pfxTXkIjPxYTmRBxBev3HFHmBH6oHzTeosj7dfCc+6m8XZMOYw33OhVTPaYdgomdIpdJpa6ZxWrhNTMa5gq1iszUNh3nAGVKbI8whKf6nx6\\\\\\\/DG8liqKoQGbKw3PsYw8\\\\\\\/9jZexsq\\\\\\\/JbDAHba3i9HK4kzTtAT3b0HIECr5AxYOxsPjRg5e03AMrxEWRuX4KA\\\\\\\/8v0Ah+lf404Wg+WMJT6aKEl13Kx\\\\\\\/TxZEjGMlMKhKpHxda2ud0QRtdet367lwmb2nPYH0CgbfclrfyJGG1an186Bu21A3Dh39yWSwTgKjHZxfQZRLZektemvyg12uHCTcXGLU9UkkIvFsQ09\\\\\\\/5GnMR4q7sb0yKLiGjJ93DPn\\\\\\\/8D88HT6UhcQLHp\\\\\\\/Vbf0r51SbUEpTo\\\\\\\/QV+ATu+kyj7WRFTnY+neSH3ISPKXQ3gkH0+viwYUpEvLQWHjHx2kKeHgdMMKqnwcTs6S15NKAvGVNf\\\\\\\/mhCRs\\\\\\\/hkU\\\\\\\/vvctvm0FLCBxTUFgerjp5mr3i2mBDMGdxh5ul876RWDphXjXhyq8ZCPZ9gbIP3JnsDSpb7VogwG1\\\\\\\/k7cApV4eFGJaISnlfpElFV9wAsfzdALiM2BE4TyS44FcO23DD+u5Fd0g2tgSZnLn0kHNSpFf0aiE131HUSIV17+xYeawBpU0BT0NRX5bDGLdxkau7Lws4JMEUBh35fZf9ZEgZtCixf+lHx7r0XbQPoPxjQ4Hv5ToYqwntICT0zZrgVi5++jkRC1SVaDLH9XlXTPNL8icEtf9kbOJI3zir0wn2guDpSwzScfQRZHq1tRSKw9\\\\\\\/8SeKPNwJJRurik1jzVBkNyPRoLwWkXS\\\\\\\/gAxtYzW+\\\\\\\/kA3+231c3LvMbOqAcPTsrfZZdmi\\\\\\\/ZRVxuiaWnMaes9KB1elNy1AbWDHNOrKp3Rz+7JbSqkawIKF\\\\\\\/Ay8E38AM\\\\\\\/UI5duQzcvIvIN7qqsazmltuPRVKPw0QjT+vm2gj9fU0Bnefwtbf0CzTk7Vjuafcchk1K96\\\\\\\/YjrEj6pwYiwSAcNF5m4HUsjnwUzgOLR1zubYX7uMsPa3DGBoIJwaVXOzDN6kIXFHQzPXOmKdHe7sbk6naI4M8TDTBQPZ1HmAYxJ1C11Zhd0OpkfQT28vXxS3FKcdzQnjauL31DrLRDjIxkTgzWAPHqfFEvSz721fHpbzx+uUCUWWEt9nv2gi16jFN+snX59s1JD8wUHrxuAC8RqzhoVkt6n9716dfFdLDQXXaK4V9rCe8hLwhab47e3GEAdLxh1IGPMQoRkvyh\\\\\\\/ME6Hsv\\\\\\\/ERIv++gKQTOQqwV9zPOg9H4S4wMoMqXvI3bcPfYiKUtu7Olzfyiut1+guJ5SrGs1fjwxtNw9jDZN+mheoxdOOMGih2LtW2A\\\\\\\/yh3h\\\\\\\/dP0RHOw0gGrG9te+JVnNMGmmo1gjkr3dZrJCxht8OCvhWFFZEOvX1c6TsT8IryhdIeCRo9gJxRu\\\\\\\/\\\\\\\/JpuZ+OzhpnxCdYxOPIkf4ZwykyhqUhodk+ALv1XdKJjvAwU21usviYIwhtT1BqyEFaJE8nib8jUjzNZMfVS1tZSxB5bXkYxQDpBj9WFhI3mclzzEQi0L06mZFSuISXin9DsEOa7OW0e00Ttnb13AjYOc2Lj2piPhEurt5dwLPUhby76gGB1UyNuYxnlu9o9W8\\\\\\\/mVgGuLvG79Cm1bXoruStMQiQ\\\\\\\/baecGnFNiNIs+gpI504ebQpCo5ijx0KQBRF7TuwqLnrUwlulwLibS13ZZVfZDSZKljv2WNHhkXyyGPxE2In2CSomOetjndxrSnkO0nE\\\\\\\/o4exXvtPN7FlU4tYOx\\\\\\\/pi+eozC+HmBPmu5j25nSKipDCFgw+dAd6FIZNgu2vUAVm+B3cXlkR0kr38TrsJrMgovGIYvovbCLZMeKFNSxaA4bJq64YwcgdTn9ygeAyGQ3BqDRiRCr3cJrYgBWTkIRjPQWHHvcXzAiyEtCUQoLcsYLveetvExu7AJLk8CUZUiIXIMHHKntyR6WkjglLSAnHz2uGFaa4jBFzF4Bb2ysTeh8qlUb4FhiL3qiHJaAR4XSOie+hYx4bRXoTP\\\\\\\/QWtSXWh3w4NjvDwb9RBoX7AmP6QtB9GXv6XqnnDg\\\\\\\/x0cAcRV0af66SF\\\\\\\/ZJjjJJBLxEaPBaCdQNxSzRWEVhD6z8cZ41118dapDYOIhuAA0mgESMsomvdeBPTsGEsy5yiB6GRM38iv1K985\\\\\\\/HuZoIx4nA60koGlVT1WL01TJy1PAujZMXo1YOlia\\\\\\\/GPghXK9k8+heu\\\\\\\/h+RCQukBzO0a03SSDYQFQbT7d0wiXtTbvAls4MCiOx46B4ITtcpzLtjIM1OIg03oKD+IHBTUcGCotQxoDa1leYFmmBgzvbfyJb7Nd0lbErfrVspG0MF82WyE4d2yxC2N\\\\\\\/Hfb0oLBNyD1\\\\\\\/UiekqIPRecTQWs\\\\\\\/LfT2WZg0PzFKYZ0NiTgP3TY7piQpm4SSwyCRwxH9OT\\\\\\\/3w47Fgc7TJ4qs6iwD8\\\\\\\/5bbchty8t6lKXiBZUWzm72Ks3myF9KPRWtx7ylVmlFjTgcuZmupiSczAAHui1lACVshTI9cGz0NrSTkACRMBAWGH9VN5tTTW6XjZefxd8moEXp4gLzjpBnysSjRWPp5NmFMDJxWzqmlmwGJIomZoTsHBS+oqB+7wMIdebPxhyIxXx4CD3fW8bplaPvMSdnoxcCodKv1kUn2hwt1czHgNZvyYTip7AAFQF5sLBW51o1z4DEiOdoEyQmb8IITFW1n15TDbJCc8pxYw\\\\\\\/VPr63Q2Mq8oX6zgZzYptfejHBNPD2VaVUi28WL5Wee9E1bH042K9mugwkjGb6FxKcWS45gfMhhNWwWRCD6H4dyatWn0GRWvWQgN093L791K6UEEoLn\\\\\\\/OpumebGNv\\\\\\\/zPKvbmwbU0l1bSyWPFt4x\\\\\\\/ng2tlndAZXIsStROBecNc6Z88+HCdxO7WE1WxpnHcP0YW9DenNJ1PJEjVyPdY5O61GOf5WqsjbUSPzDr9faSgNd\\\\\\\/As3lavtUnSz0I7+Pxi\\\\\\\/tG\\\\\\\/Ml+UF7NcS6nvAdAy96FxBU8yXyWh5lehNTlzlYYATqZJa5qQZsE\\\\\\\/cRzSieAYen\\\\\\\/luE++l25zDQDq77HB+ghdYpHME+ZQkVo3leDPntnMUdKJEwSIlPkBfA1WRdgx4bzoVy8gtxu\\\\\\\/Kmqr8JLJuYRDsdvio5j40Lxc3NuJqyTp\\\\\\\/CTHfMMjTLUAqQQg3f5EKzsz79mNbZXL5u0eCDy9FBv5Czfp+44gBns8ZrmFVap+RIJ1pokZw9W8vHb\\\\\\\/0j7QQ2dg2T587fYJPGV9kWJ4QZzluoJ31gQrGeO9HzPUr4D\\\\\\\/6rdDVPMhJFKwWZYpZUb+qSvPO2AHWbryjCqKbWO2ms200pZGqKqEdlgGM+Km9JoLHQ4dRoMuYkO12bKVUrCGp2+FTG2WfLllJEGXhdklrVCUD32bR4USMuBuUP\\\\\\\/pa4iaugmw2iHtm2EwYBFGsioh7zAuwbIBqOYzqCGd1M6\\\\\\\/CL\\\\\\\/koZRrmKfYSf1izrs\\\\\\\/2eFqNvmXrRskDzHHVL2FiOmKrTO+tN6+cnZV\\\\\\\/n6GMx9t20N4tkZ4xJf\\\\\\\/HQkLD8dFT8m8n1TGv0FrD7w2l6AFCI4xH83XU9kO15TQwU6GI\\\\\\\/Bf63FCLawN2db8PvjrFQTqC8uW57BHnukluHx2BUEHl4YhINlQuNFnBvr0F8Ln4ktNGfyhVxxBC8\\\\\\\/RPrGaMLEgVebHBEtwPQ0i\\\\\\\/AM6LONno1tQ6OEZRlkWZo8DQ+Rcy+DpLqG1LaqwJQF8kepQlmhHsFYSSE0Sf16L4Kojl2tSzNTFY3CUWLJFK2QzRNCmu9GWt1yYp5LDDkIiQdoo7qVoaJTg6Ai6fGTwOORriNP4RvKo0wCZ4pK4STXTZ0KgpS+R+4CShT9ZWs3jqk\\\\\\\/\\\\\\\/Kt3AeiL8lA3kowGYP3P5SjWtqPPlL91ni68OasaY\\\\\\\/m\\\\\\\/1i1iSqweSIZCr+87LsnJzwYJRAOymh1tqL0zlcIXSmWqgJj7av4jjdH5dDEprlMJvP3BDP6UNhoYynj\\\\\\\/mk1UWffvtu1cEstmY0d3hGf+ZffkO1uvwRqOIIA8zGI4N0b\\\\\\\/NDJcrMHb98roohTHbvj1j\\\\\\\/rRESgqhxMUDHkrmbfrSxyCPVBbQfaQEfb\\\\\\\/JqLRKw8rBeAgVccxGlbqNxqjzfAbQqjKFpD8+ELRA+D+oX87AhQ\\\\\\\/Dru96fduVnYVwm0AHDwRjGw94tGriBe5DIcCeV3q7nLBRaDIs7nrLYp5TyfQZ1BBxsYufTgPo78Ur3T0L80zT5bxd1xZHn165MYJ8lKHjKoeu9vl4BfG+1Fa8XFGgXLLnft1ePO38nN1p2h\\\\\\\/mVR5usSGqfSkpWQ3gj\\\\\\\/saAHhuavXrGdog1tU5uNw5dpYQK6aB1WP11nC8YoazFn8xGHoDWr642XMNL34HkMPs3lXiZGs4Z+HSPr0WZpxIvtmI9YcUrMKxFH8ytLIGtsbCONTBRUqZ+QCSM9P46ITbcdJaL1Lwm4nrMzXt3YudvCaD157+3qXcEzMjyqgXFlRqzywDmpYB+mqCKNh4AtXOptqa3qjBG47O3vxTghTn5iSCpS42ht5eHIurVicp0EScwTG6EtE+54z1OK32xRZCPY2d0niRd7mSPelBJeF88HWLtf5uh\\\\\\\/FBHteiTEM2mIPjJ4f64tN2OTBeVGjp1rY4m6KfedyYWui0Er\\\\\\\/oPQYg9IwM3LlcEzmMx3TdqhUuYSo+6gu4fV7it9AQyhXpWIy844TGPAoXNrAcnYCseXzqFF7HzAWiNqILlz60pKmkIz0+pJ8H82N0bG6+0n\\\\\\\/30jNNDai2jydOiaGhbzsDD00oivPA6CRIgYwys9raG3PkxmCbQdVxHBndx\\\\\\\/VUUjhzX9XvYTW9fefE6aWnWZCfvIlsJfwXS4jS1FAPPuZ95ffKApctZYdczuphXlWaqh9F4tU30hLy9YVFgDJJ6MZAPeUXnItXectoAIfhAmxxOmo5SUvQ3Zlo+Ga2j8XZEeMpAJvc7ty0ym1AV29E8qCghGgy1b5azb2ufkd5v\\\\\\\/bp+lNwiIY+LfByTnV3oFNHGdFZTKej07nANhUqkg5oHvb195jJ560o5JSuFaFBzFPnTQFvLqk7Dc5j0wqlo4bOArZNVLAEUj2jUNKwSv\\\\\\\/\\\\\\\/TrXT1ZvFsx87FTgowgTOiiLpet7kCDfCtfiyPMyhWhotUWZ97leYrqMCxYqGG54txBez+N8LYDCluXCe2uCdCNRiyA+bk8io212VAgzNYGGEW1oxmvcx0QySdflGsY8c6QR7zuSW2KZkD\\\\\\\/VdL4nq\\\\\\\/43gFvYD1BmdT2G2MZ8p\\\\\\\/VkLGjdseRQsEcpif6ja8y66aWSy95QNESwZfcxmFG7rMhaYpc07KgFPYSshJ7pLhiT64jSLUUI46ha6rf0lkHophp51ghdd6r\\\\\\\/NP2kJRDoqXsBu\\\\\\\/vg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022hDDPkBIqRwNymQHYCkdUrg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022237a9ae6bd6cc66d7da01aa0a35d33586a49c3a67ddcc0588bbe32765d3ac401\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222ANfR6wTKlFz\\\\\\\/ltX5f16TDvZFkUMV\\\\\\\/LdqWs8aBbm1cctfv7wfgvaJ7TmS9nP80DIOe8JsTLxlI9kET5viZlmX7+qQAxXWRXzhl9dTdiLaE9v1E5PI+SQ8FYNFyW61AV3aByNQf4foRJHCc79dpSL9NFrMJYlTjIJL3anvdva6Z8vUZhRxKQYQULYM0h\\\\\\\/wmnbkYU1fH+46ZJqd3ah9w937TUrpAKMRqdgGVbgrcDxmWRc09mfIp1eWyiQIaXJcUnYSdLMTdWr7MiqMSO6s5MPALYnYi2J2um\\\\\\\/VDSIM1GVqSP8Y1ZN7B9sqqPrgqQ3MIcslE3NBK48NJkd7V\\\\\\\/alZsNrj5i0EBiR38Wg8yLmM8riXY9HTc8vjx\\\\\\\/k+INGXj+7tmSNKxqV9VTovI92g23AwsK3lPnL3Npg6TA8sEtTddq0t8Qyy9hwP7m8tIkL2XOBykvewjQDGaEhKtGZ8K+MYFABevh1sRgAnjh67hM+FbPTZz\\\\\\\/w4b7pcCWdI18YNUNY4QQE4gENuEYjRPu8tutpg7yrJeRqb9KdduScMxFfVJuqbfmK6euC7pHMhu5DjCXVJH9SPt+6r\\\\\\\/C7zKMT4pT+RaXYX2F814KNH10LKRctavkMISsEeusg1UeKc0eKI69om6L0ks50EffP9BfBtssf\\\\\\\/QubxaQ8GwI5C6Ncjh4QKJnm\\\\\\\/0Mt3+DLU2n2tOdz4Ebq30kWiChbIdYV4KxT5WCAGcpoWcPReTAYT18WBwZqg0u43pqnvZfAMCoMubetQsUGbYTVVWvAcBsiNjXsVP0qKzn0yz6dyTgTJwwSozbBBPON8q3hdR2hmO+5dSvRd+cFgdzFfAb9EQafsBCVMmhuoVRx2MhdfoNhLBxb5aKoCHUoco4\\\\\\\/RSuTSHH3jjJ1\\\\\\\/pb7Afb5m7RifzC1qp3M4y3TPaYnMUchkBdoQ15zI7MHZ\\\\\\\/sZYq+arjMNQxpWA6GYwhwqUDcHGHXbmt8t4N3xqXoKYNcWB+HwrJlX6l0g+BXowynmoKI+xutr86obtyp78rJz7WhoQnp34eeJWBg\\\\\\\/O8IyFGPqNOQK1ft6MtovEWydyCDcXNujx9OkuCdxZxv00hoD3mtnkUEDXP+KmrPZlHwWFQmqaQoJt6jU4HCEaUNjUktEQlckuS6tNnwWmCzbYajlGT9Lb7mBp9Hl2Gm\\\\\\\/+oZQRmJXCYH1xli4cDCJg2R61Ar6KvKAFk3Gjw9AHSCzhGM7w23RSx3QIjQDVYdP+K2FjM6vrRuN+JKbaISj+zUmvS93kk\\\\\\\/jhOm52+VkiJe6KFP7Kw1KbMx7MWzMXIS6vlsKu6XfMXR4q7cxnIjMOJYx2HxVtzKH7Uhg\\\\\\\/Jt4CO9x54vmV9YFpoi1q7CI7W+15YcohWSNcACrWO7SQY8VlGTCKl5HIndSZj0zBSHz\\\\\\\/b8tHS7Quggi6p1l6G4yab06qMuEn7D97dpglkWPuYsTT6KoxVD0GV1L5X0sfUEtKJIZwaYp452VL09+GLXC+dhSLnC8Mcu2KZUKUthgJ8xGVCDQLflvpMr1tCieOcyn6fuuIMGj0aqsaZ4C54qSHVg3X\\\\\\\/nTFQudxk7gk7Yc3+6WhDZ8dXzdWpO3mebAKVexBHuvfakglLtDGHN2yifyfNkX433+Dmhca8ygCqCYL99dK7XzwEuU3EQBqHBvAWUGqh4v\\\\\\\/Q8avS+wd4R15zYFzlbx2hFtApdMyiqAoHIngcU8VbDFhOQ8Ihy\\\\\\\/\\\\\\\/kt\\\\\\\/d4sXo\\\\\\\/vvSIIWa0aSyKy1HUylV+rsk\\\\\\\/RWG5yJJZ6qNq8xfAq4oCoj5DNHO8S4D6XFDWtj3yKmOxENx20BwTMEmRbVY6c7XN0ZX\\\\\\\/74\\\\\\\/ToMYVziDt83sPC0h0Dlv4RupuAHvVxbu8V0poKABZK6bJqXRMnhr+gR7+KN\\\\\\\/ZwP3Rvp1qW8RSGuTPH7VfjkbbNp7BRWHbsobIjexWZFZyBV4uGq1eu79ezxMaHsJYVyHo6637NUvDPL1wGKLXAr2wgtDT5leiGZui3DR+ZVaSbSI17HZTGUyQZmzc9wTRhgw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022XVP7U5Re5FSqVs7CxJvMAA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022105c6e2a206cf4143d401b742e1007ac3346b2176841727543c92e1c42367237\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495543 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/769" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-MmcicjwKKivIIkCPRND0GOYzKziqVkU-TPJTYgODGuA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/769" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DWocT5P6eHfp9HEgyfJxJ3h_UM1cfGQ0NA23sc6aGE4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/2/755" title="Origin and Evolution of Two Independently Duplicated Genes Encoding UDP- Glucose: Glycoprotein Glucosyltransferases in Caenorhabditis and Vertebrates" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/2/783" title="The Impact of Non-additive Effects on the Genetic Correlation Between Populations" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9030-702X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jhonathan P. R. dos Santos</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8269-535X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Samuel B. Fernandes</span>, <span class="highwire-citation-author" data-delta="2">Scott McCoy</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-0760-4977" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Roberto Lozano</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-1332-711X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick J. Brown</span>, <span class="highwire-citation-author" data-delta="5">Andrew D.B. Leakey</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-3100-371X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Edward S. Buckler</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-0634-3277" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Antonio A. F. Garcia</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael A. Gore</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">769-781;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400759 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495543" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jhonathan P. R. dos Santos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, SP, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jhonathan%2BP.%2BR.%2Bdos%2BSantos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=dos%20Santos%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJhonathan%2BP.%2BR.%2Bdos%2BSantos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9030-702X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jhonathan P. R. dos Santos</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Samuel B. Fernandes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop Science,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Samuel%2BB.%2BFernandes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fernandes%20SB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASamuel%2BB.%2BFernandes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8269-535X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Samuel B. Fernandes</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Scott McCoy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Institute for Genomic Biology,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BMcCoy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCoy%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AScott%2BMcCoy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Roberto Lozano </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roberto%2BLozano%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lozano%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoberto%2BLozano%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0760-4977" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Roberto Lozano</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Patrick J. Brown </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Section of Agricultural Plant Biology, Department of Plant Sciences, University of California Davis, 95616, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BJ.%2BBrown%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brown%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BJ.%2BBrown%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1332-711X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick J. Brown</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Andrew D.B. Leakey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop Science,</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Institute for Genomic Biology,</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Plant Biology, University of Illinois at Urbana Champaign, 61801,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BD.B.%2BLeakey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Leakey%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BD.B.%2BLeakey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Edward S. Buckler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science,</div><div class='author-affiliation'><span class='nlm-sup'>**</span>United States Department of Agriculture, Agricultural Research Service, R. W. Holley Center, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Institute for Genomic Diversity, Cornell University, Ithaca, New York 14853,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edward%2BS.%2BBuckler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buckler%20ES&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEdward%2BS.%2BBuckler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3100-371X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Edward S. Buckler</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Antonio A. F. Garcia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, SP, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonio%2BA.%2BF.%2BGarcia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garcia%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntonio%2BA.%2BF.%2BGarcia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0634-3277" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Antonio A. F. Garcia</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:augusto.garcia@usp.br" class="" data-icon-position="" data-hide-link-title="0">augusto.garcia@usp.br</a>
<a href="mailto:mag87@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">mag87@cornell.edu</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Michael A. Gore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BA.%2BGore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gore%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BA.%2BGore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael A. Gore</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:augusto.garcia@usp.br" class="" data-icon-position="" data-hide-link-title="0">augusto.garcia@usp.br</a>
<a href="mailto:mag87@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">mag87@cornell.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/2/769" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495543" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495543/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/2/769.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495543" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495543/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/2/769.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495543" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495543/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/2/769.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495543" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495543/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The ability to connect genetic information between traits over time allow Bayesian networks to offer a powerful probabilistic framework to construct genomic prediction models. In this study, we phenotyped a diversity panel of 869 biomass sorghum (<em>Sorghum bicolor</em> (L.) Moench) lines, which had been genotyped with 100,435 SNP markers, for plant height (PH) with biweekly measurements from 30 to 120 days after planting (DAP) and for end-of-season dry biomass yield (DBY) in four environments. We evaluated five genomic prediction models: Bayesian network (BN), Pleiotropic Bayesian network (PBN), Dynamic Bayesian network (DBN), multi-trait GBLUP (MTr-GBLUP), and multi-time GBLUP (MTi-GBLUP) models. In fivefold cross-validation, prediction accuracies ranged from 0.46 (PBN) to 0.49 (MTr-GBLUP) for DBY and from 0.47 (DBN, DAP120) to 0.75 (MTi-GBLUP, DAP60) for PH. Forward-chaining cross-validation further improved prediction accuracies of the DBN, MTi-GBLUP and MTr-GBLUP models for PH (training slice: 30-45 DAP) by 36.4–52.4% relative to the BN and PBN models. Coincidence indices (target: biomass, secondary: PH) and a coincidence index based on lines (PH time series) showed that the ranking of lines by PH changed minimally after 45 DAP. These results suggest a two-level indirect selection method for PH at harvest (first-level target trait) and DBY (second-level target trait) could be conducted earlier in the season based on ranking of lines by PH at 45 DAP (secondary trait). With the advance of high-throughput phenotyping technologies, our proposed two-level indirect selection framework could be valuable for enhancing genetic gain per unit of time when selecting on developmental traits.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/bayesian-networks" class="hw-term hw-article-keyword hw-article-keyword-bayesian-networks" rel="nofollow">Bayesian networks</a></li><li class="kwd"><a href="/keyword/biomass-sorghum" class="hw-term hw-article-keyword hw-article-keyword-biomass-sorghum" rel="nofollow">biomass sorghum</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">genomic prediction</a></li><li class="kwd"><a href="/keyword/indirect-selection" class="hw-term hw-article-keyword hw-article-keyword-indirect-selection" rel="nofollow">indirect selection</a></li><li class="kwd"><a href="/keyword/probabilistic-programming" class="hw-term hw-article-keyword hw-article-keyword-probabilistic-programming" rel="nofollow">probabilistic programming</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">The development of renewable energy resources from biomass crops is a key step toward the establishment of a sustainable agroecosystem (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Foley <em>et al.</em> 2011</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Mace <em>et al.</em> 2013</a>). Among the plant species amenable to bioenergy production, sorghum [<em>Sorghum bicolor</em> (L.) Moench] is a prominent candidate for genetic improvement because it is diploid (2<em>n</em> = 2<em>x</em> = 20), fixes carbon through the <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-1.gif"/></noscript></span></span> pathway, predominantly autogamous, and resilient against biotic and abiotic stresses (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Vermerris 2011</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lawrence and Walbot 2007</a>). Although substantially more economic investments have been made in crops such as maize, which resulted in improvements of up to fourfold in grain yield over the last century, proportional gains in yield might be achievable in biomass sorghum (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Mullet <em>et al.</em> 2014</a>). Currently, sorghum has an average biomass yield of 12-15 dry Mg ha<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-2.gif"/></noscript></span></span> under rainfed conditions, with a predicted potential yield of 55-60 dry Mg ha<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-3.gif"/></noscript></span></span> for ideotypes (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Mullet <em>et al.</em> 2014</a>). Because of its biology, sorghum has the potential to become a model organism to understand the genetic basis of growth traits related to biomass production (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brenton <em>et al.</em> 2016</a>).</p><p id="p-4">Genomic prediction (GP) is a statistical approach that predicts the unobserved phenotypes of individuals using genomic information (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Meuwissen <em>et al.</em> 2001</a>). Because of its potential to enrich for promising selection candidates, GP is increasingly becoming an important component of plant breeding and genetic resources conservation programs. In sorghum, <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Yu <em>et al.</em> (2016)</a> showed that GP can optimize the management and evaluation of accessions from gene banks through the prediction of different traits. Briefly, the GP procedure involves two steps: (i) phenotyping and genotyping a reference population (training set) to train statistical models, and (ii) genotyping of unevaluated individuals (test set) for predicting their unobserved phenotypes with the trained models (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Heslot <em>et al.</em> 2015</a>). To support this procedure, plant breeding programs collect phenotypes from training set individuals evaluated in multi-environment trials. In parallel, high density single-nucleotide polymorphism (SNP) markers are scored on individuals in the training and test sets with skim sequencing or SNP arrays (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Davey <em>et al.</em> 2011</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Buckler <em>et al.</em> 2016</a>).</p><p id="p-5">Mixed linear models, hierarchical Bayesian models with informative priors, kernel methods, and neural nets are modeling approaches used for GP, but minimal differences in predictive performance are typically seen across these approaches (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">de los Campos <em>et al.</em> 2013</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Heslot <em>et al.</em> 2015</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">dos Santos <em>et al.</em> 2016a</a>). This outcome may be explained by the high density of covariates (SNP markers) compared to the population size used for training the models. This scenario is known as the large <em>p</em> and small <em>n</em> problem (<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-4.gif"/></noscript></span></span>) (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Gianola <em>et al.</em> 2009</a>). In statistical models, this may lead to the problem of multicollinearity, <em>i.e.</em>, multiple covariates with redundant information. As it relates to GP models, markers in complete or near complete linkage disequilibrium provide redundant information and will not contribute to enhancing statistical power (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Gianola 2013</a>). Dimensionality reduction techniques, such as the artificial bins approach, may help circumvent the challenge of multicollinearity with minimal information loss, as well as mitigate the computational cost often associated with GP (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Xu 2013</a>).</p><p id="p-6">The vast majority of GP studies conducted in crop species have only tested models for predicting individual traits. However, recent studies have shown the advantages of combining multiple correlated traits in a GP model (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Calus and Veerkamp 2011</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Jia and Jannink 2012</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Fernandes <em>et al.</em> 2018</a>), allowing genetic correlations among secondary traits to be leveraged for improving predictions of a target trait (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">dos Santos <em>et al.</em> 2016b</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Okeke <em>et al.</em> 2017</a>). Most of these efforts used multi-trait GBLUP—a type of multivariate mixed linear model that incorporates a genomic relationship matrix (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Gianola <em>et al.</em> 2015</a>). Despite the advances obtained so far, the use of genetic models that exploit information between traits using other parametrizations beyond those reliant on genetic correlations under multivariate normal distribution assumptions have yet to be addressed. Indeed, novel genetic models with parametrizations to partition genetic effects influencing only a single trait from those acting on multiple traits (<em>i.e.</em>, pleiotropy) may help to better understand the genetic architecture of correlated traits.</p><p id="p-7">There have been significant advances in field-based high-throughput phenotyping (HTP) technologies for the rapid measurement of plant traits over the growing season (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bao <em>et al.</em> 2019</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Pauli <em>et al.</em> 2016</a>). Measuring phenotypes at multiple time points over the life cycle of a plant can better describe the progression of growth and development (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Muraya <em>et al.</em> 2017</a>). Having collected phenotypic information on a time axis may help to identify key environmental stress events during the growing season, which might be masked if phenotypic data are only obtained at harvest (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Campbell <em>et al.</em> 2018</a>). Furthermore, the underlying genetic signals of these phenotypic responses are additional sources of information to more powerfully predict and dissect the genetic architecture of developmental plant traits (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Muraya <em>et al.</em> 2017</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Campbell <em>et al.</em> 2018</a>). Statistical models that exploit temporal genetic trends are especially needed for longitudinal (repeated measure) data collected by field-based HTP systems. Such models could be used to reduce generation time and prioritize which breeding populations to evaluate.</p><p id="p-8">Among the models available for analyzing traits in a time series, probabilistic graphical models (PGMs) offer a versatile, efficient, and intuitive approach for drawing inferences (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Murphy 2013</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bishop 2013</a>). Popular PGMs include directed graphical models or Bayesian networks (BNs), undirected graphical models or Markov random fields, chain models, and factor graphs (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hamelryck 2012</a>). In particular, BNs provide the flexibility to model repeated measure and correlated trait data, as would be important for the study of developmental traits. A BN is defined as a structured graphical representation of joint distributions factored into a set of conditional probability distributions, where shaded and unshaded nodes represent known and unknown variables, respectively, and arrows showing dependence between them (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bishop 2013</a>). The Markov condition is a key property of the BN, ensuring that a variable (child) is only dependent on the information of its parents in the network (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Su <em>et al.</em> 2013</a>; <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Bishop 2013</a>). Through their ability to connect joint probability distributions, BNs enable the aggregation of advantages from multiple machine learning approaches under a directed acyclic graph structure. Notably, BNs have been diversely applied in genetic and genomic studies (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Loman <em>et al.</em> 2015</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Serang <em>et al.</em> 2012</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Garcia <em>et al.</em> 2013</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Han <em>et al.</em> 2012</a>; <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Su <em>et al.</em> 2013</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Neapolitan <em>et al.</em> 2013</a>), but to our knowledge have never been used for modeling trends of genetic effects considering repeated measures and correlated traits.</p><p id="p-9">There are several features of BNs that enable them to recover information from correlated data types such as multiple correlated traits scored at a single time point or the repeated measurement of a single trait across multiple time points (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bae <em>et al.</em> 2016</a>). Several different GP models could be unified for leveraging pleiotropy or temporal genetic effects in a single BN to improve prediction accuracies. This is because these genetic effects can be modeled with a BN through connections between likelihood functions. Also, BNs offer the possibility to use general Markov chain Monte Carlo (MCMC) methods to obtain solutions for complex time series and multiple trait models that otherwise would have been mathematically intractable to derive analytically. Furthermore, the posterior samples of genomic estimated breeding values (GEBVs) may be used to create indices for understanding the uncertainty of selecting promising lines either earlier in the season or through indirect selection based on the ranking of the lines at other measurement time points or with correlated traits.</p><p id="p-10">With sorghum as a model biomass crop, we developed PGMs for the GP of developmental traits in a sorghum diversity panel of nearly 900 lines. Herein, we aimed to (i) develop PGMs for the GP of plant height (PH) and dry biomass yield (DBY) traits by connecting genetic effects across multiple developmental time points and traits, and (ii) describe growth dynamics based on the change of the ranking of lines across multiple time points and correlated traits to design novel breeding strategies to genetically improve biomass sorghum.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant material, field experiments and phenotypic data</h3><p id="p-11">In this study, we evaluated a biomass sorghum diversity panel consisting of 869 lines (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Valluru <em>et al.</em> 2018</a>). The diversity panel was grown at three field locations only a few km from the main campus of the University of Illinois Urbana-Champaign in 2016 (Fisher and Energy Farms) and 2017 (Maxwell and Energy Farms). Each of the four environments had one complete replication of the field experiment that contained 960 four-row plots laid out in an 40-row by 24-column arrangement. The experimental field design consisted of 16 incomplete blocks, and each block was augmented with a common set of six lines (four shared between years): Pacesetter, PI276801, PI148089, PI524948, NSL50748, and PI148084 in 2016 and Pacesetter, PI276801, PI148089, PI524948, PI525882, and PI660560 in 2017. Plots were 3 m in length, with a 1.5 m alley at the end of each plot. Plots had a spacing between rows of 0.76 m. The plant population had a targeted density of 270,368 plants <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-5.gif"/></noscript></span></span>. Experiments were planted in late May and harvested in early October. PH was measured in centimeters (cm) from the soil line to the topmost leaf whorl. A single plant was measured in each plot on a biweekly basis from 30 to 120 days after planting (DAP). Plots were harvested for above ground biomass using a four-row Kemper head attached to a John Deere 5830 tractor. Wet weight of total biomass (lbs) and biomass moisture (%) in the center two rows of each 4-row plot were measured using a plot sampler that had a near infrared sensor (model 130S, RCI engineering). DBY in dry ton per hectare was calculated as follows: dry t <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-6.gif"/></noscript></span></span> = total plot wet weight (kg) × (1-plot moisture) / (plot area in square meters/10,000).</p></div><div id="sec-3" class="subsection"><h3>Phenotypic data analysis</h3><p id="p-12">Phenotypic measurements for DBY (dry t <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-7.gif"/></noscript></span></span>; one measurement at harvest) and PH (cm; one measurement on each of seven plant developmental stages) were analyzed individually with the following mixed linear model:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-8.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where y (<span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-9.gif"/></noscript></span></span>) represents the phenotypic vector with <em>n</em> entries, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-10.gif"/></noscript></span></span> a unit vector, <em>μ</em> a scalar to map the population mean, <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-11.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-12.gif"/></noscript></span></span>) the design matrix of the <em>q</em> fixed genetic effects (number of lines), <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-13.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-14.gif"/></noscript></span></span>) the design matrix of the <em>l</em> random block within environment effects, <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-15.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-16.gif"/></noscript></span></span>) the design matrix of the <em>s</em> random environment (location x year combination) effects, <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-17.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-18.gif"/></noscript></span></span>) the design matrix of the <em>m</em> random genotype-by-environment effects; and g, b, e, and ge are column vectors mapping the design matrices effects, respectively, and ϵ (<span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-19.gif"/></noscript></span></span>) the vector of errors. The model random effects b, e, ge, and ϵ were assumed to follow a <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-20.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-21.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-22.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-23.gif"/></noscript></span></span>, respectively.</p><p id="p-13">Empirical estimates of variance components were obtained by the maximization of the restricted (residual or reduced) maximum likelihood (REML) function (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Patterson and Thompson 1971</a>). For each phenotype, the adjusted mean for each line was derived by the sum of the population mean with the fixed genetic effect of the <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-24.gif"/></noscript></span></span> line (<span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-25.gif.backup.1581003573.4718"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-25.gif.backup.1581003573.4718"/></noscript></span></span>).</p><p id="p-14">Heritability on an line-mean basis was estimated for each phenotype. Variance component estimates were obtained by refitting model (1) with all terms as random effects in ASReml-R version 3.0 (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Butler <em>et al.</em> 2009</a>). The variance component estimates from each model for a phenotype were used to estimate heritability on a line-mean basis as the ratio of genetic variance to phenotypic variance following (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Holland <em>et al.</em> 2003</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Hung <em>et al.</em> 2012</a>). Standard errors of the heritability estimates were calculated with the delta method (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Lynch <em>et al.</em> 1998</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Holland <em>et al.</em> (2003)</a> in the <em>nadiv</em> R package (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Wolak 2012</a>). The Pearson’s correlation coefficient (<em>r</em>) was used to assess the degree of relationship between adjusted means for each pair of traits.</p></div><div id="sec-4" class="subsection"><h3>Genotypic data</h3><p id="p-15">We genotyped the sorghum diversity panel using the genotyping-by-sequencing (GBS) procedure (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Elshire <em>et al.</em> 2011</a>) based on the <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-26.gif.backup.1581003573.489"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-26.gif.backup.1581003573.489"/></noscript></span></span>I-HF/<span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-27.gif.backup.1581003573.5116"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-27.gif.backup.1581003573.5116"/></noscript></span></span>P1I and <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-28.gif.backup.1581003573.5337"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-28.gif.backup.1581003573.5337"/></noscript></span></span>I-HF/<span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-29.gif.backup.1581003573.5647"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-29.gif.backup.1581003573.5647"/></noscript></span></span>I restriction enzymes. A total of 367 million sequence reads were generated (100 bp length) on a HiSeq 4000 sequencer. Sequence reads were aligned to the <em>Sorghum bicolor</em> genome v3.1 (<a href="http://www.phytozome.jgi.doe.gov">www.phytozome.jgi.doe.gov</a>) using Bowtie2 (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Langmead and Salzberg 2012</a>). The TASSEL 3 GBS pipeline (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Glaubitz <em>et al.</em> 2014</a>) was then used to call variants. Only biallelic SNPs were retained. Additionally, lines with &gt;80% missing data (sample call rate) and SNPs with &gt;60% missing data (SNP call rate) were removed. Also, SNPs with a minor allele frequency less than 5% were discarded. Missing genotypes of SNP markers were imputed using <em>Beagle 4.1</em> (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Browning and Browning 2016</a>) with default parameters and an Ne of 150,000. In total, 100,435 SNP markers were scored and converted to dosage format (0,1,2). Of the 869 total lines, 839 had both phenotypic and genotypic data; therefore, the genetic analyses focused on only these 839 lines.</p></div><div id="sec-5" class="subsection"><h3>Population structure analysis</h3><p id="p-16">To examine population structure, we first analyzed the filtered set of 100,435 SNPs with a variational Bayesian framework implemented in fastSTRUCTURE version 1.0 (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Raj <em>et al.</em> 2014</a>). When conducting the analysis in fastSTRUCTURE, we varied the number of ancestral populations (K) between 1 and 15 using the simple prior. Next, the same genotype matrix was used to perform a principal component analysis (PCA) implemented in SNPRelate version 1.18.1 (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Zheng <em>et al.</em> 2012</a>). The results from fastSTRUCTURE and PCA were combined to select K = 4 as the number of subpopulations. The labeling of subpopulations according to sorghum racial groups (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Harlan and de Wet 1972</a>) was determined by the membership of previously classified lines from the germplasm resources information network (GRIN, <a href="https://www.ars-grin.gov/">https://www.ars-grin.gov/</a>) that were included in the sorghum diversity panel (Figure S1A). The simple prior approach in fastSTRUCTURE was used to determine the subpopulation composition for each of the 839 lines with K = 4 (Figure S1B). Lines were assigned to subpopulations (1, 2, 3, or 4) if having assignment values of Q ≥ 0.80, whereas lines with assignment values of Q &lt; 0.80 for all four subpopulations were designated as admixed (Figure S1C and Supplemental Table 1).</p></div><div id="sec-6" class="subsection"><h3>Artificial bins</h3><p id="p-17">Due to the high dimensionality of the SNP marker matrix (100, 435 loci), we developed a strategy similar to that proposed by <a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Xu (2013)</a> for obtaining artificial bins. In our approach, after centering (subtracting) the marker scores 2 (MM), 1 (Mm), and 0 (mm) by <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-30.gif"/></noscript></span></span>, instead of averaging the columns from equally sized slices of the marker matrix, we conducted a PCA to calculate the first PC of each artificial bin (Figure S2). The procedure was based on the following steps: (i) subdivision of the centered marker matrix into 1000 column-slices, with each slice comprising ∼100 columns; (ii) singular value decomposition of each matrix column slice into singular vectors and values; and (iii) construction of each artificial bin as the first PC coordinate of its respective matrix column slice, given by <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-31.gif.backup.1581003573.589"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-31.gif.backup.1581003573.589"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-32.gif.backup.1581003573.6248"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-32.gif.backup.1581003573.6248"/></noscript></span></span> is the first left singular vector of the matrix slice and <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-33.gif.backup.1581003573.6474"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-33.gif.backup.1581003573.6474"/></noscript></span></span> its first singular value. This procedure resulted in 1,000 artificial bins. This number of artificial bins was selected as a balance between model run time and predictive performance for the computationally intensive Bayesian models. In theory, the first PC (artificial bin) will retain as much information as possible from the matrix slice in one dimension in a least-square reconstruction error sense (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Goodfellow <em>et al.</em> 2016</a>).</p></div><div id="sec-7" class="subsection"><h3>Probabilistic graphical models</h3><p id="p-18">We developed three different Bayesian models for genomic prediction of the PH and DBY traits (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The first model, Bayesian network (BN), neither recovered information between traits nor time points. The BN has the following conditional normal likelihood form:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-61.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-62.gif"/></noscript></span></span> is the adjusted mean related to the <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-63.gif.backup.1581003573.7059"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-63.gif.backup.1581003573.7059"/></noscript></span></span> line evaluated in the <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-64.gif.backup.1581003573.7236"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-64.gif.backup.1581003573.7236"/></noscript></span></span> time point (or DAP), <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-65.gif.backup.1581003573.7422"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-65.gif.backup.1581003573.7422"/></noscript></span></span> the unknown population mean in the <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-66.gif.backup.1581003573.7597"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-66.gif.backup.1581003573.7597"/></noscript></span></span> time point, <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-67.gif.backup.1581003573.7788"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-67.gif.backup.1581003573.7788"/></noscript></span></span> the known row vector (<span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-68.gif.backup.1581003573.8006"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-68.gif.backup.1581003573.8006"/></noscript></span></span>) of the <em>p</em> artificial bins of the <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-69.gif.backup.1581003573.8217"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-69.gif.backup.1581003573.8217"/></noscript></span></span> line, <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-70.gif.backup.1581003573.8412"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-70.gif.backup.1581003573.8412"/></noscript></span></span> the column vector (<span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-71.gif.backup.1581003573.8587"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-71.gif.backup.1581003573.8587"/></noscript></span></span>) of the unknown <em>p</em> artificial bins effects within in the <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-72.gif.backup.1581003573.8856"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-72.gif.backup.1581003573.8856"/></noscript></span></span> time point and <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-73.gif"/></noscript></span></span> the unknown residual standard deviation mapping the uncertainty around the expected value in the <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-74.gif"/></noscript></span></span> time point.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/769/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of the Bayesian models. Bayesian network (BN), pleiotropic Bayesian network (PBN), and dynamic Bayesian network (DBN) probabilistic graphical models. k: number of time points; : number of lines within a time point; p: number of artificial bins; , , : adjusted means for the  line evaluated in the  time point, which can be for trait 1 (Tr1) or trait 2 (Tr2); , , : population means; : row vector with artificial bins; , , , : column vector with artificial bin effects; , : pleiotropic  bin effect; : standardized pleiotropic  bin effect; , : pleiotropic means hyperparameters; : bin effects between the current and previous time points; , : pleiotropic standard deviations hyperparameters; , , : standard deviations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-78870331" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Overview of the Bayesian models. Bayesian network (BN), pleiotropic Bayesian network (PBN), and dynamic Bayesian network (DBN) probabilistic graphical models. &lt;em&gt;k&lt;/em&gt;: number of time points; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-34&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;: number of lines within a time point; &lt;em&gt;p&lt;/em&gt;: number of artificial bins; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-35&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-36&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-37&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;: adjusted means for the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-38&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; line evaluated in the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-39&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; time point, which can be for trait 1 (Tr1) or trait 2 (Tr2); &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-40&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-41&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-42&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;: population means; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-43&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;: row vector with artificial bins; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-44&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-45&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-46&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-47&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;: column vector with artificial bin effects; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-48&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-49&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;: pleiotropic &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-50&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; bin effect; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-51&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;: standardized pleiotropic &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-52&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; bin effect; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-53&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-54&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;: pleiotropic means hyperparameters; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-55&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;: bin effects between the current and previous time points; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-56&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-57&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;: pleiotropic standard deviations hyperparameters; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-58&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-59&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-60&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;: standard deviations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/769/F1.medium.gif" width="440" height="203"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/2/769/F1.medium.gif" width="440" height="203"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/769/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/769/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495738" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child">Overview of the Bayesian models. Bayesian network (BN), pleiotropic Bayesian network (PBN), and dynamic Bayesian network (DBN) probabilistic graphical models. <em>k</em>: number of time points; <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-34.gif"/></noscript></span></span>: number of lines within a time point; <em>p</em>: number of artificial bins; <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-35.gif.backup.1581003573.9678"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-35.gif.backup.1581003573.9678"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-36.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-37.gif"/></noscript></span></span>: adjusted means for the <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-38.gif.backup.1581003574.0714"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-38.gif.backup.1581003574.0714"/></noscript></span></span> line evaluated in the <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-39.gif"/></noscript></span></span> time point, which can be for trait 1 (Tr1) or trait 2 (Tr2); <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-40.gif.backup.1581003574.098"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-40.gif.backup.1581003574.098"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-41.gif.backup.1581003574.1266"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-41.gif.backup.1581003574.1266"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-42.gif.backup.1581003574.1564"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-42.gif.backup.1581003574.1564"/></noscript></span></span>: population means; <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-43.gif.backup.1581003574.2077"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-43.gif.backup.1581003574.2077"/></noscript></span></span>: row vector with artificial bins; <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-44.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-45.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-46.gif.backup.1581003574.2522"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-46.gif.backup.1581003574.2522"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-47.gif.backup.1581003574.2729"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-47.gif.backup.1581003574.2729"/></noscript></span></span>: column vector with artificial bin effects; <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-48.gif.backup.1581003574.2946"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-48.gif.backup.1581003574.2946"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-49.gif.backup.1581003574.3173"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-49.gif.backup.1581003574.3173"/></noscript></span></span>: pleiotropic <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-50.gif.backup.1581003574.3462"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-50.gif.backup.1581003574.3462"/></noscript></span></span> bin effect; <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-51.gif"/></noscript></span></span>: standardized pleiotropic <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-52.gif.backup.1581003574.3731"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-52.gif.backup.1581003574.3731"/></noscript></span></span> bin effect; <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-53.gif.backup.1581003574.4174"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-53.gif.backup.1581003574.4174"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-54.gif.backup.1581003574.4434"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-54.gif.backup.1581003574.4434"/></noscript></span></span>: pleiotropic means hyperparameters; <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-55.gif.backup.1581003574.4719"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-55.gif.backup.1581003574.4719"/></noscript></span></span>: bin effects between the current and previous time points; <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-56.gif.backup.1581003574.4921"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-56.gif.backup.1581003574.4921"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-57.gif.backup.1581003574.5112"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-57.gif.backup.1581003574.5112"/></noscript></span></span>: pleiotropic standard deviations hyperparameters; <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-58.gif.backup.1581003574.5371"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-58.gif.backup.1581003574.5371"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-59.gif.backup.1581003574.5588"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-59.gif.backup.1581003574.5588"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-60.gif.backup.1581003574.5804"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/F1/embed/mml-math-60.gif.backup.1581003574.5804"/></noscript></span></span>: standard deviations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">The BN has an unnormalized joint posterior distribution (hyperpriors omitted from <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> for simplicity),<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-75.gif.backup.1581003574.6017"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-75.gif.backup.1581003574.6017"/></noscript></span></span><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-76.gif.backup.1581003574.627"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-76.gif.backup.1581003574.627"/></noscript></span></span><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-77.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-78.gif.backup.1581003574.6563"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-78.gif.backup.1581003574.6563"/></noscript></span></span><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-79.gif.backup.1581003574.6766"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-79.gif.backup.1581003574.6766"/></noscript></span></span>where <em>k</em> is the total number of time points, <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-80.gif.backup.1581003574.7038"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-80.gif.backup.1581003574.7038"/></noscript></span></span> is the total number of lines in the <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-81.gif.backup.1581003574.7302"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-81.gif.backup.1581003574.7302"/></noscript></span></span> time point, <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-82.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-83.gif.backup.1581003574.7621"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-83.gif.backup.1581003574.7621"/></noscript></span></span> denotes the normal probability density function, and Cauchy probability density function truncated to the real positive space (<span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-84.gif"/></noscript></span></span>) of the random variable <em>θ</em> (general notation), respectively, parametrized by the mean (<span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-85.gif.backup.1581003574.789"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-85.gif.backup.1581003574.789"/></noscript></span></span>) and standard deviation (<span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-86.gif.backup.1581003574.8082"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-86.gif.backup.1581003574.8082"/></noscript></span></span>). The joint distribution was parameterized as second (<span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-87.gif.backup.1581003574.8301"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-87.gif.backup.1581003574.8301"/></noscript></span></span>) and third (<span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-88.gif"/></noscript></span></span>) level scale hyperparameters. The known global hyperparameter was defined by <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-89.gif.backup.1581003574.8716"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-89.gif.backup.1581003574.8716"/></noscript></span></span>, resulting in weakly informative second-level hyperpriors that eliminated the subjectiveness to define hyperparameters when choosing first-level prior hyperparameters (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gelman <em>et al.</em> 2014</a>). This same approach was used for the next set of described models.</p><p id="p-21">The second Bayesian model, pleiotropic Bayesian network (PBN), exploited information between PH and DBY (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). This model has two conditionally dependent normal likelihood functions that characterized the observed adjusted means distribution for each trait as follows:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-90.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-91.gif"/></noscript></span></span>where all variables are the same from the previous model, except the column vectors <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-92.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-93.gif"/></noscript></span></span>) and <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-94.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-95.gif"/></noscript></span></span>), that represent the pleiotropic effects of known bins with continuous space corrected by the transformation of an unknown pleiotropic standardized random variable <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-96.gif"/></noscript></span></span> for the <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-97.gif.backup.1581003575.011"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-97.gif.backup.1581003575.011"/></noscript></span></span> bin,<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-98.gif.backup.1581003575.0331"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-98.gif.backup.1581003575.0331"/></noscript></span></span><span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-99.gif.backup.1581003575.053"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-99.gif.backup.1581003575.053"/></noscript></span></span>with <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-100.gif.backup.1581003575.0736"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-100.gif.backup.1581003575.0736"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-101.gif.backup.1581003575.0996"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-101.gif.backup.1581003575.0996"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-102.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-103.gif.backup.1581003575.1255"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-103.gif.backup.1581003575.1255"/></noscript></span></span> being unknown random variables. The PBN model has an unnormalized joint posterior density function (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>),</p><span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-104.gif.backup.1581003575.1511"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-104.gif.backup.1581003575.1511"/></noscript></span></span><span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-105.gif.backup.1581003575.1799"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-105.gif.backup.1581003575.1799"/></noscript></span></span><span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-106.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-107.gif.backup.1581003575.2087"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-107.gif.backup.1581003575.2087"/></noscript></span></span><span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-108.gif.backup.1581003575.2302"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-108.gif.backup.1581003575.2302"/></noscript></span></span><span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-109.gif.backup.1581003575.2666"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-109.gif.backup.1581003575.2666"/></noscript></span></span><span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-110.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-111.gif.backup.1581003575.3046"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-111.gif.backup.1581003575.3046"/></noscript></span></span><span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-112.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-113.gif.backup.1581003575.3445"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-113.gif.backup.1581003575.3445"/></noscript></span></span><span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-114.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-115.gif.backup.1581003575.3792"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-115.gif.backup.1581003575.3792"/></noscript></span></span><span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-116.gif.backup.1581003575.416"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-116.gif.backup.1581003575.416"/></noscript></span></span><p id="p-22">The third Bayesian model, dynamic Bayesian network (DBN), recovered information from PH measurements across multiple time points (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>). This network architecture has a specific conditionally, dependent normal likelihood function for each time point as follows:<span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-117.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-118.gif.backup.1581003575.4432"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-118.gif.backup.1581003575.4432"/></noscript></span></span>where the column vector <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-119.gif.backup.1581003575.4663"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-119.gif.backup.1581003575.4663"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-120.gif.backup.1581003575.4913"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-120.gif.backup.1581003575.4913"/></noscript></span></span>) is the known artificial bins effects at time <em>t</em>, that are a linear combination of the <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-121.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-122.gif.backup.1581003575.5171"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-122.gif.backup.1581003575.5171"/></noscript></span></span>) known artificial bins effects displayed in the previous time point (<span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-123.gif.backup.1581003575.5441"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-123.gif.backup.1581003575.5441"/></noscript></span></span>) plus the unknown <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-124.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-125.gif"/></noscript></span></span>) random noise mapping the bin effect between the current and previous time points, such that genetic information is propagated over time. The artificial bins effects were treated as unknown random variables only at the first time point. The DBN model has an unnormalized joint posterior distribution (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>),</p><span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-126.gif.backup.1581003575.6069"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-126.gif.backup.1581003575.6069"/></noscript></span></span><span class="disp-formula" id="disp-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-127.gif.backup.1581003575.6296"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-127.gif.backup.1581003575.6296"/></noscript></span></span><span class="disp-formula" id="disp-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-128.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-129.gif.backup.1581003575.664"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-129.gif.backup.1581003575.664"/></noscript></span></span><span class="disp-formula" id="disp-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-130.gif.backup.1581003575.6904"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-130.gif.backup.1581003575.6904"/></noscript></span></span><span class="disp-formula" id="disp-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-131.gif"/></noscript></span></span><p id="p-23">The joint distributions of the BN, PBN, and DBN models were integrated using the No-U-Term sampler algorithm available in the probabilistic programming language Stan (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Hoffman and Gelman 2014</a>). We used the implementation available in the Python package <em>pystan 2.17.1.0</em> (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Team 2018</a>). Stan compiles the probabilistic programming code in C++ and has a user interface within the Python environment. The probabilistic programming language saved time during customization of the C++ code, allowed rapid implementation during model design and training, as well as facilitated the manipulation of posterior draws after fitting the model in Python (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Carpenter <em>et al.</em> 2017</a>). We set up the No-U-Term sampler to iterate 400 times and used as warm up 50% of the samples from four Markov chains. The number of iterations (400) was selected in consideration of runtime and predictive performance.</p></div><div id="sec-8" class="subsection"><h3>Multivariate GBLUP model</h3><p id="p-24">For comparison to the three Bayesian models, we also evaluated two different formulations of the multivariate GBLUP model (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Henderson and Quaas 1976</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">dos Santos <em>et al.</em> 2016b</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Fernandes <em>et al.</em> 2018</a>) that recovered information between traits and/or time points. In the first formulation, only PH measurements across time points were used (MTi-GBLUP). In the second, DBY and PH measurements across time points (MTr-GBLUP) were jointly analyzed. Both formulations share the same linear model as follows:<span class="disp-formula" id="disp-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-132.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-133.gif.backup.1581003575.7559"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-133.gif.backup.1581003575.7559"/></noscript></span></span> corresponds to the adjusted mean related to the <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-134.gif"/></noscript></span></span> line evaluated for the <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-135.gif.backup.1581003575.7794"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-135.gif.backup.1581003575.7794"/></noscript></span></span> time point or trait, <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-136.gif.backup.1581003575.806"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-136.gif.backup.1581003575.806"/></noscript></span></span> is the fixed population mean effect for the <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-137.gif.backup.1581003575.8362"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-137.gif.backup.1581003575.8362"/></noscript></span></span> time point and/or trait, <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-138.gif.backup.1581003575.8637"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-138.gif.backup.1581003575.8637"/></noscript></span></span> is the genomic estimated breeding value (GEBV) of the <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-139.gif"/></noscript></span></span> line evaluated for the <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-140.gif.backup.1581003575.8962"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-140.gif.backup.1581003575.8962"/></noscript></span></span> time point and/or trait, and <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-141.gif.backup.1581003575.9218"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-141.gif.backup.1581003575.9218"/></noscript></span></span> is the residual.</p><p id="p-25">Considering the structure of multivariate GBLUP models with stacked trait and/or time subvectors like <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-142.gif.backup.1581003575.9431"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-142.gif.backup.1581003575.9431"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-143.gif.backup.1581003575.9667"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-143.gif.backup.1581003575.9667"/></noscript></span></span>, in which <em>k</em> is the number of traits and/or time points, we assume that <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-144.gif.backup.1581003575.9916"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-144.gif.backup.1581003575.9916"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-145.gif.backup.1581003576.0206"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-145.gif.backup.1581003576.0206"/></noscript></span></span>), where A (<span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-146.gif.backup.1581003576.0415"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-146.gif.backup.1581003576.0415"/></noscript></span></span>) is the additive relationship matrix (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">VanRaden 2008</a>) between <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-147.gif.backup.1581003576.0663"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-147.gif.backup.1581003576.0663"/></noscript></span></span> lines, G (<span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-148.gif.backup.1581003576.094"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-148.gif.backup.1581003576.094"/></noscript></span></span>) and R (<span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-149.gif.backup.1581003576.1221"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-149.gif.backup.1581003576.1221"/></noscript></span></span>) are unstructured variance-covariance matrices for genetic and residual effects, respectively. The A matrix was constructed using the 100,435 SNP markers with the A.mat function in the R package <em>rrBLUP 4.6</em> (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Endelman 2011</a>). Spectral decomposition was performed to transform the A matrix into positive definite. The procedure was based on the singular value decomposition of the A matrix, substitution of the negative values by a decreasing small constant (<span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-150.gif.backup.1581003576.1462"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-150.gif.backup.1581003576.1462"/></noscript></span></span>), and reconstruction of the A matrix. Additional details on the spectral decomposition procedure are available in <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Caliński <em>et al.</em> (2005)</a>; <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">dos Santos <em>et al.</em> (2016b)</a>. The MTi-GBLUP and MTr-GBLUP models were fitted using the R package <em>EMMREML 3.1</em> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akdemir and Godfrey 2018</a>).</p></div><div id="sec-9" class="subsection"><h3>Cross-validation schemes</h3><p id="p-26">Two different cross-validation (CV) schemes were used to evaluate the predictive accuracy of the GP models. The first scheme used was stratified fivefold CV for each individual trait (<em>i.e.</em>, DBY or PH measured at a single time point). This procedure was based on stratifying the phenotypic and genotypic data of the lines into five non-overlapping folds, training the model with four folds (training set), and predicting the phenotypes of lines in the fold not included for training (test set) with only their genotypes as predictors in the trained model. This procedure was repeated until phenotypes from all five folds were predicted. We accounted for the presence of population structure in the sorghum diversity panel by employing a stratified sampling approach within each of the identified subpopulations in fivefold CV, allowing for each fold to be representative of the subpopulation composition (1, 2, 3, 4, and admixed) in the entire panel. Forward-chaining CV was used as a second scheme. In this scheme, data were split into time point subsets. The initial training set of five total was based on data from the first two time points (30 and 45 DAP), with the remaining time points (60, 75, 90, 105, and 120 DAP) comprising the test set. This procedure was repeated four more times to build new training sets until all time points except the last one (120 DAP) were included in the training set. The forward-chaining CV scheme was used to assess the accuracy of GP models to predict and identify the best set of lines for PH (tallest) prior to harvest.</p><p id="p-27">The correlation (Pearson’s <em>r</em>) of adjusted means with predicted values was used to estimate predictive accuracy in both CV schemes. In the forward-chaining CV scheme, the predicted values were always obtained with the artificial bins effects from the previous time point used for training the DBN model. For the MTi-GBLUP and MTr-GBLUP models, the predicted values from the previous time point used for training were used as predicted values. For the BN and PBN models, which do not share information across time, the effects of artificial bins for each time point were used to compute the predicted values.</p></div><div id="sec-10" class="subsection"><h3>Coincidence index</h3><p id="p-28">For the five GP models, coincidence indices (CIs) were constructed to evaluate the capacity for selecting the top 20% best performing lines for DBY when considering the rank of the PH adjusted mean values at each time point. The posterior values of the CI were calculated as the rate of successes between the top 20% best lines for PH at each time point and DBY. The CI was computed by assigning a ‘1’ to lines in the top 20% best lines for PH and DBY, or ‘0’ otherwise, then dividing the total number of successes (sum of ‘1’s) by the total number of lines at each posterior sample.</p></div><div id="sec-11" class="subsection"><h3>Coincidence index based on lines</h3><p id="p-29">For the DBN model, we constructed a coincidence index based on lines (CIL) that used posterior samples from the adjusted mean values of PH across the seven developmental time points. This CIL was used to determine how early selection could be performed within-season to optimally reduce the length of the breeding cycle. Calculation of the CIL was based on the following steps: (i) identify the top 20% best lines for each posterior sample of the PH adjusted mean values at each time point; (ii) create for each posterior sample a one-hot vector encoding, assigning a ‘1’ to the best lines in the top 20% in the evaluated time point and at the end of the season (120 DAP), or ‘0’ otherwise; and (iii) compute the CIL as the total number of successes divided by the total number of posterior samples that each line appeared in the top 20% for the predicted (evaluated time points) and observed adjusted mean values (end of season).</p></div><div id="sec-12" class="subsection"><h3>Data availability</h3><p id="p-30">Genotypic and phenotypic data are available at CyVerse (<a href="http://datacommons.cyverse.org/browse/iplant/home/shared/GoreLab/dataFromPubs/dosSantos_BayesianNetworks_2019">http://datacommons.cyverse.org/browse/iplant/home/shared/GoreLab/dataFromPubs/dosSantos_BayesianNetworks_2019</a>). Scripts used in this study are available on GitHub (<a href="https://github.com/GoreLab/sorghum-multi-trait">https://github.com/GoreLab/sorghum-multi-trait</a>). Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.9891119">https://doi.org/10.25387/g3.9891119</a>.</p></div></div><div class="section results" id="sec-13"><h2 class="">Results</h2><div id="sec-14" class="subsection"><h3>Phenotypic variation</h3><p id="p-31">We used a mixed linear model that accounted for the influence of environment and genotype-by-environment interaction to generate adjusted means for end-of-season DBY and PH measured at seven developmental time points over the growing season. Both DBY and the multiple PH measures had moderately high estimates of heritability on a line-mean basis (<span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-151.gif"/></noscript></span></span>) across the four environments (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The distribution of adjusted mean values for DBY was slightly skewed toward the left tail, with values centered on 23.55 t <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-152.gif"/></noscript></span></span> (std = 5.79). In comparison, the adjusted mean values for PH showed an expected growth pattern across the seven time points, with the population mean for PH changing from 25.6 (std = 3.7) at 30 DAP to 350.7 cm (std = 48.0) at 120 DAP. Indicative of an autoregressive trend of correlation over time, the weakest correlation (<em>r</em>) was observed between PH measures collected at 30 and 120 DAP (<em>r</em> = 0.40), while 90 and 105 DAP were the two most strongly correlated time points (<em>r</em> = 0.96). Correlations between DBY and PH varied from 0.10 to 0.31 across time points, suggesting an opportunity for recovering information across time points and/or between traits to improve the predictive accuracy of GP models.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/769/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary statistics of evaluated phenotypes. Correlations between adjusted means, heritabilities, and distributions of adjusted means." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-78870331" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Summary statistics of evaluated phenotypes. Correlations between adjusted means, heritabilities, and distributions of adjusted means.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/769/F2.medium.gif" width="440" height="386"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/2/769/F2.medium.gif" width="440" height="386"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/769/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/769/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495741" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-32" class="first-child">Summary statistics of evaluated phenotypes. Correlations between adjusted means, heritabilities, and distributions of adjusted means.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Predictive accuracies from stratified fivefold CV</h3><p id="p-33">We evaluated the accuracy of the BN, PBN, DBN, MTr-GBLUP, and MTi-GBLUP models for predicting DBY and PH measured throughout the growing season with a stratified fivefold CV scheme. Prediction accuracies (0.46-0.49) of DBY were nearly identical for the BN, PBN, and MTr-GBLUP models (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). When predicting PH at each of the seven developmental stages with the BN, PBN, MTi-GBLUP and MTr-GBLUP models, we found that accuracies gradually increased from 30 to 60 DAP, peaked at 60 DAP, and incrementally decreased from 60 to 120 DAP. Of these four models, MTi-GBLUP (0.57-0.75) and MTr-GBLUP (0.57-0.74) had prediction accuracies comparable to each other and slightly higher than those of BN (0.54-0.72) and PBN (0.53-0.71). Comparatively, the DBN model showed a randomly fluctuating trend of relatively slightly lower prediction accuracies for PH at all seven time points. The minor variation in predictive accuracy, especially after 45 DAP, suggests that early season prediction could be possible for PH.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495709/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495709/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Prediction accuracies obtained from the fivefold cross-validation scheme by training the Bayesian network (BN), pleiotropic Bayesian network (PBN), dynamic Bayesian network (DBN), multi time GBLUP (MTi-GBLUP) and multi trait GBLUP (MTr-GBLUP) models to predict dry biomass yield (DBY) collected at harvest and plant height (PH) measured across different days after planting (DAP). The standard deviation of the prediction accuracy obtained by each Bayesian model is reported within parentheses.</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Predictive accuracies from forward-chaining cross-validation</h3><p id="p-34">We performed a forward-chaining CV procedure to evaluate the accuracy of the five models to predict PH at unobserved time points. In general, the models showed high accuracy to predict the phenotypic values of the lines observed at the last time point (120 DAP) even when trained only with data from both 30 and 45 DAP (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The BN and PBN models had similar prediction accuracies across all scenarios, ranging from 0.42 (BN, training: 45 DAP; predicting: 120 DAP) to 0.86 (PBN, training: 60 DAP; predicting: 75 DAP). In contrast, the DBN, MTi-GBLUP, and MTr-GBLUP models had substantially higher predictive accuracies compared to the BN and PBN models. The DBN model showed predictive accuracies varying from 0.60 (training slice: 30-45 DAP, predicting: 120 DAP) to 0.95 (training slice: 30-60 DAP; predicting: 75 DAP). The MTi-GBLUP prediction accuracies ranged from 0.63 (training slice: 30-45 DAP; predicting: 120 DAP) to 0.94 (training slice: 30-90 DAP; predicting: 105 DAP), and comparably, the MTr-GBLUP varied from 0.64 (training slice: 30-45 DAP; predicting: 120 DAP) to 0.94 (training slice: 30-90 DAP; predicting: 105 DAP). These results did not suggest any advantage for modeling dependence between PH and DBY in the PBN and MTr-GBLUP models; however, the results did suggest that the dependence between time points accounted for in the DBN, MTi-GBLUP and MTr-GBLUP models improved the prediction accuracy of PH.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/769/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic prediction of plant height with forward chaining cross-validation. Prediction accuracies were obtained from prediction models exploiting single (Bayesian Network and Pleiotropic Bayesian Network) or multiple time points (Dynamic Bayesian Network, Multi Time GBLUP, and Multi Trait GBLUP). The horizontal axis represents the slice (:) of the time interval used for training the models with multiple time points and the vertical axis the testing data. The &#x2019;*&#x2019; symbol denotes the days after planting (DAP) time point used to obtain the adjusted means." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-78870331" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genomic prediction of plant height with forward chaining cross-validation. Prediction accuracies were obtained from prediction models exploiting single (Bayesian Network and Pleiotropic Bayesian Network) or multiple time points (Dynamic Bayesian Network, Multi Time GBLUP, and Multi Trait GBLUP). The horizontal axis represents the slice (:) of the time interval used for training the models with multiple time points and the vertical axis the testing data. The &#x2019;*&#x2019; symbol denotes the days after planting (DAP) time point used to obtain the adjusted means.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/769/F3.medium.gif" width="440" height="256"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/2/769/F3.medium.gif" width="440" height="256"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/769/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/769/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495674" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-35" class="first-child">Genomic prediction of plant height with forward chaining cross-validation. Prediction accuracies were obtained from prediction models exploiting single (Bayesian Network and Pleiotropic Bayesian Network) or multiple time points (Dynamic Bayesian Network, Multi Time GBLUP, and Multi Trait GBLUP). The horizontal axis represents the slice (:) of the time interval used for training the models with multiple time points and the vertical axis the testing data. The ’*’ symbol denotes the days after planting (DAP) time point used to obtain the adjusted means.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Coincidence indexes</h3><p id="p-36">The high predictive performance of the DBN, MTri-GBLUP, and MTr-GBLUP models, which exploited multiple PH measurements over initial growth stages, incentivized us to investigate how the rank of the lines varied across the different time points. To that end, we evaluated how PH measures over time (secondary traits) could be informative for performing indirect selection of DBY (target trait) through the calculation of coincidence indices (CIs). The posterior distribution of the CIs showed an overlapping pattern over time for the three Bayesian models (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>), with most of them ranging from 0.18 to 0.34. This implied that the ranking of the lines for PH did not change significantly from early to late growth stages. Additionally, the MTi-GBLUP CI (first-level target trait: DBY; second-level target trait: PH) ranged from 0.25 (training slice: 30-105 DAP) to 0.27 (training slice: 30-45 DAP), and MTr-GBLUP varied from 0.26 (training slice: 30-105 DAP) to 0.28 (training slice: 30-45 DAP). These results suggest that the initial growth stage ranging from 30 to 45 DAP could be an optimal stage of development for the early selection of PH and indirect selection of DBY based on the ordering of the PH adjusted means over the growing season.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/769/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The calculated coincidence index (CI) at multiple developmental stages. The CIs for selecting the top 20% for dry biomass yield at specific developmental stages were calculated using as reference the adjusted mean values obtained by training the Bayesian network (BN), pleiotropic Bayesian network (PBN), and dynamic Bayesian network (DBN) models with the plant height time series. The CIs from the multi time GBLUP (MTi-GBLUP) and multi trait GBLUP (MTr-GBLUP) models were not plotted because their point estimates do not include confidence intervals (point estimates reported in the text). The &#x2019;*&#x2019; symbol denotes the time point estimates used to obtain the adjusted means as expected values for indirect selection. For the DBN model that leveraged multiple time points, the &#x2019;*&#x2019; symbol denotes the last time point used for training with the earlier time points also considered in the model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-78870331" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The calculated coincidence index (CI) at multiple developmental stages. The CIs for selecting the top 20% for dry biomass yield at specific developmental stages were calculated using as reference the adjusted mean values obtained by training the Bayesian network (BN), pleiotropic Bayesian network (PBN), and dynamic Bayesian network (DBN) models with the plant height time series. The CIs from the multi time GBLUP (MTi-GBLUP) and multi trait GBLUP (MTr-GBLUP) models were not plotted because their point estimates do not include confidence intervals (point estimates reported in the text). The &#x2019;*&#x2019; symbol denotes the time point estimates used to obtain the adjusted means as expected values for indirect selection. For the DBN model that leveraged multiple time points, the &#x2019;*&#x2019; symbol denotes the last time point used for training with the earlier time points also considered in the model.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/769/F4.medium.gif" width="440" height="330"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/10/2/769/F4.medium.gif" width="440" height="330"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/769/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/769/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495641" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-37" class="first-child">The calculated coincidence index (CI) at multiple developmental stages. The CIs for selecting the top 20% for dry biomass yield at specific developmental stages were calculated using as reference the adjusted mean values obtained by training the Bayesian network (BN), pleiotropic Bayesian network (PBN), and dynamic Bayesian network (DBN) models with the plant height time series. The CIs from the multi time GBLUP (MTi-GBLUP) and multi trait GBLUP (MTr-GBLUP) models were not plotted because their point estimates do not include confidence intervals (point estimates reported in the text). The ’*’ symbol denotes the time point estimates used to obtain the adjusted means as expected values for indirect selection. For the DBN model that leveraged multiple time points, the ’*’ symbol denotes the last time point used for training with the earlier time points also considered in the model.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">To gain more insight and empirical evidence to support the hypothesis of early selection for PH with measures from 30 and 45 DAP, we developed a coincidence index based on lines (CIL) using the posterior values from the DBN model that achieved optimal performance among the Bayesian models tested in the forward-chaining CV. The CIL allowed us to better understand phenotypic plasticity through assessing how the expected rank of the lines at the end of the season agreed with their ranking at earlier growth stages. The closer that the CIL is to one, the more likely the line is expected to be at the top 20% for PH at the end of the season. We plotted lines with CIL &gt; 0.5, fixed their ordering (training slice: 30-45 DAP), and displayed the CILs from other time slices in the same order (Figure S3). The CILs showed the expected trend of increasing the chance of lines to be in the top 20% after 45 DAP, which indicated that the ranking of the top lines had not majorly changed over time.</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-39">Biomass sorghum is a promising bioenergy feedstock because of its extensive genetic diversity, high biomass yield potential, and strong tolerance to environmental stress. Sorghum is evolutionarily related to key bioenergy grasses, including maize, sugarcane, switchgrass, and <em>Miscanthus</em> spp., making it a potentially important diploid model to inform the genetic improvement of these other bioenergy crops (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Morris <em>et al.</em> 2013</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Brenton <em>et al.</em> 2016</a>). Despite sorghum’s appealing features as both a crop and model species, few studies have focused on genetically modeling its growth patterns and leveraging this information for breeding optimization. In this study, we investigated a diverse panel of 839 sorghum lines genotyped with 100,435 SNP markers that was evaluated for a PH time series and DBY in four environments. With these collected data, we evaluated several GP models for exploiting genetic information over time and/or between correlated traits to improve prediction accuracies compared to models that assumed independence. Our implemented Bayesian models allowed us to estimate the level of uncertainty in determining optimal time points when developing breeding strategies for early selection of PH within season, as well as the indirect selection of DBY in combination with the repeated measures of PH as secondary traits.</p><p id="p-40">To conduct GP of DBY and PH, we used both PGM and multivariate mixed linear model approaches to better model growth dynamics (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Bishop 2013</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Henderson and Quaas 1976</a>; <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">dos Santos <em>et al.</em> 2016b</a>). Due to the high computational cost of the PGM approach, we modified the artificial bins method of <a id="xref-ref-68-3" class="xref-bibr" href="#ref-68">Xu (2013)</a> to reduce the dimensionality of the SNP marker matrix through a PCA. This modified approach reduced the number of parameters needed to train the different PGM architectures by a 100-fold. Also, this procedure in other scenarios has the flexibility for predictions even when the number of loci pooled is different between the training and testing sets. Indicative of minimal information loss, there were negligible differences in prediction accuracies achieved by the MTi-GBLUP and MTr-GBLUP models that used the 100,000 SNP markers to compute the relationship matrix compared to those of the BN and PBN models reliant on the 1,000 artificial bins. Also, the artificial bins approach did not compromise the results from the DBN model, as indicated by the similarity of its obtained prediction accuracies with those from the multivariate GBLUP models in the forward-chaining CV scheme.</p><p id="p-41">We initiated a model-based machine learning approach for GP analysis by first defining the baseline of PGMs (<a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Bishop 2013</a>). Several studies have shown a similar level of predictive performance between PGMs that assume either a common or specific normal prior for each marker effect (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">de los Campos <em>et al.</em> 2013</a>; <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Heslot <em>et al.</em> 2015</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Ferrão <em>et al.</em> 2018</a>). Therefore, we parsimoniously used a common normal prior for the effects of all artificial bins, resulting in a BN model that had less unknown parameters but a slower MCMC process to obtain posterior draws. The BN model can be considered a non-conjugate form of the Bayesian linear regression model (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">de los Campos <em>et al.</em> 2013</a>) that automatically learns the hyperparameters of priors from the data. This model also has Cauchy priors truncated to the <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-153.gif.backup.1581003576.2147"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-153.gif.backup.1581003576.2147"/></noscript></span></span> on the scale components, which avoids sampling implausible standard deviation values (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Gelman <em>et al.</em> 2014</a>). Although the main disadvantage of the BN model formulation is that its architecture cannot recover information between traits or time points, the BN model can be useful for analyzing highly unbalanced data as is frequently observed for large-scale field trials in the plant breeding industry.</p><p id="p-42">To improve the performance of the BN model, we novelly developed the PBN model by connecting two representations of the BN model with hidden variables (nodes in the graph representing artificial bins effects influencing both traits), allowing a conditional relationship between the likelihood functions of PH and DBY to be established. Despite our efforts to estimate pleiotropic effects with the PBN model, the interpretation of the effects as pleiotropic should be carefully interpreted, especially due to the challenge of differentiating between pleiotropy and tight linkage. <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Gianola <em>et al.</em> (2015)</a> theorized that the linkage disequilbrium (LD) between markers and quantitative trait loci (QTL), LD between QTL controlling different traits, or LD between markers linked to QTL controlling different traits can make it difficult to partition pleiotropic effects in GP models. The presence of such complex LD structure could explain the minor difference in predictive accuracy between the BN and PBN models when attempting to partition genetic effects influencing single <em>vs.</em> multiple traits. This interpretation is supported by the finding that prediction accuracies obtained by the MTi-GBLUP (PH time series) and MTr-GBLUP (DBY and PH time series) models were similar to each other. Additionally, the lower than expected performance of the PBN model relative to the BN model could be attributed to the modest correlations observed between DBY and the multiple PH traits (<em>r</em> = 0.10 - 0.31). Simulation studies have shown that genetic correlations weaker than 0.5 do not provide marked improvements to the prediction accuracy of GP models used for multiple traits (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Calus and Veerkamp 2011</a>; <a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">dos Santos <em>et al.</em> 2016b</a>). Further studies analyzing traits having strong genetic correlations with DBY are needed to better understand how to further improve the prediction accuracy of the PBN model.</p><p id="p-43">The DBN model, a variant of hidden Markov models that is intended for modeling continuous variables (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Murphy 2013</a>), was used to exploit the genetic effects of artificial bins from the prediction of PH at earlier time points. This connection of genetic information between time points was crucial for dramatically improving the performance of the Bayesian framework in the forward-chaining CV scheme. Moreover, the predictions using the posterior mean of the DBN model allowed us to obtain predictions as precise as the point estimates from the MTi-GBLUP model and construct indices with posterior samples to identify optimal time points for indirect selection. The strong correlation between multiple time points for PH is quite possibly the main factor that favored the improvement of prediction accuracy for the DBN model compared to the BN and PBN models that did not leverage information over time. In contrast to the high predictive performance achieved in the forward-chaining CV scheme, relatively lower predictive accuracies were observed for the DBN model in the fivefold CV scheme, especially when removing lines across all time points to split the data into training and testing sets. This is because the splitting did not allow the DBN model to learn with precision the effects of the artificial bins between time points and added noise to the artificial bin estimates. Despite the sharing of genetic signals over time, these findings suggest that the DBN model should not be used when lines are completely unobserved across all time points (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Burgueño <em>et al.</em> 2012</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Dias <em>et al.</em> 2018</a>; <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Fernandes <em>et al.</em> 2018</a>). This reduced level of prediction performance because of unobserved lines has also been previously reported for multivariate GP models tested with a fivefold CV scheme (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Burgueño <em>et al.</em> 2012</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Dias <em>et al.</em> 2018</a>; <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Fernandes <em>et al.</em> 2018</a>).</p><p id="p-44">Even though modeling growth dynamics is an important area of research, there have been a limited number of studies in plants that have analyzed longitudinal phenotypes related to growth rate with GP models. In a greenhouse study of 357 diverse rice (<em>Oryza sativa</em> L.) accessions, <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Campbell <em>et al.</em> (2018)</a> developed GP models with a first- or second-order Legendre polynomial to predict the sum of “plant pixels” from image-based phenotyping as a daily estimate of shoot biomass during initial growth stages (13 to 33 days after transplant), resulting in an 11.6% improvement in prediction accuracies relative to a single time point analysis. Despite the advantage of fitting a nonlinear random regression model with phenotypic data that showed an exponential curve over 20 days of early vegetative growth, this procedure is limited to only early developmental stage phenotypes such as shoot biomass that follow an exponential growth curve. In our study, PH was measured throughout the entire growing season without a focus on the early vegetative stage, thus the collected PH data did not have an exponential shape. The modeling of genetic effects as either a linear additive function over time or though the <em>G</em> variance-covariance matrix might be the main factor causing the 36.4–52.4% (training slice: 30-45 DAP; predicting: 120 DAP) improvements in prediction accuracy of the DBN, MTi-GBLUP, and MTr-GBLUP models relative to the BN and PBN models. When analyzing repeated measures of height collected from an interior spruce (<em>Picea engelmannii</em> x <em>glauca</em>) population of 769 trees at six sparse time points over a period of 37 years, <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Ratcliffe <em>et al.</em> (2015)</a> observed minor differences in prediction performance among the evaluated BayesC<em>π</em>, ridge regression (rrBLUP), and generalized ridge regression models. These findings are contrary to our evaluated PGMs that recovered genetic information between time points that showed substantial improvement compared to the BN model—a Bayesian formulation of the rrBLUP model.</p><p id="p-45">The Bayesian multi-trait and multi-environment (BMTME) method for GP enables a unified model for multiple traits and multiple environments (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Montesinos-López <em>et al.</em> 2016</a>). This method models the correlation structure of the phenotypic data between traits or environments at a population level under the assumption of a multivariate normal distribution, with a Gibbs sampler used for obtaining the posterior distribution of model parameters. In contrast, our PBN and DBN models allow for genetic effects to be shared across different likelihoods, enabling estimated genetic parameters to be leveraged between traits and/or time points. Such parameterizations could especially be helpful with the advance of field-based high-throughput phenotyping, where time points over the growing season could potentially be at the scale of hundreds. This would result in high-dimensional variance-covariance structures that could be computationally prohibitive to estimate with simpler methods such as the Gibbs sampler. Additionally, the architectures of the BN, PBN and DBN models are designed with a non-conjugate Bayesian formulation, allowing solutions to be obtained using a state-of-the-art No-U-Term sampler algorithm (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Hoffman and Gelman 2014</a>) that more rapidly converges to high-dimensional target distributions than the Gibbs sampler.</p><p id="p-46">The implemented PGMs provided a powerful modeling framework to infer uncertainty based on well-established probability theory (<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Murphy 2013</a>; <a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">Bishop 2013</a>), allowing us to define optimal time points for early within-season selection. To this end, the posterior distribution of the CIs was used to show that the rank order of the sorghum lines changed minimally after 45 DAP, which suggested an opportunity to indirectly select for DBY based on early season PH measures as secondary traits. The Bayesian CIs allowed the overlapping pattern of posterior values to be assessed for identifying optimal time points for selection contrary to the point estimate approach of <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Hamblin and Zimmermann (1986)</a> applied to genomic prediction of a PH time series and DBY in sorghum by <a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Fernandes <em>et al.</em> (2018)</a>. The Bayesian CIs also did not require resampling to build the index. In addition, we used the DBN model to evaluate the phenotypic plasticity of PH with the CILs at the population level. The pattern of phenotypic plasticity shown by the CILs confirmed the general findings of the CIs—the ranking of lines by PH changed minimally from early (45 DAP) to late (120 DAP) time points. Early within-season selection (<em>i.e.</em>, prior to flowering) for PH could help to realize greater potential for genetic gain in that cycle, because it would then allow only the chosen superior plants to be intermated in a genomic selection scheme. Considering the Breeder’s equation <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-154.gif.backup.1581003576.2361"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/769/embed/mml-math-154.gif.backup.1581003576.2361"/></noscript></span></span> (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Li <em>et al.</em> 2018</a>), reducing the length of the generation (L) for a given selection intensity (i) could also provide a new avenue for increasing genetic gain per unit of time through early indirect selection of PH and DBY at 45 DAP in these tested environments.</p><p id="p-47">Given that the order of lines ranked by PH minimally changed across the measured plant developmental stages and had a moderate coincidence with rankings based on DBY, we propose a two-level indirect selection framework: (i) fit the DBN model, which of the Bayesian models had the best performance for predicting future measures, to learn posterior values of the GEBVs for PH repeated measures in initial growth stages and obtain a precise estimate of the ranking of each line at the end of the season; (ii) compute CIs and CILs to evaluate the extent to which the rank order of lines change; (iii) use GEBVs of the last time point used for training (<em>e.g.</em>, 45 DAP for our tested environment) as a secondary trait; (iv) perform indirect selection for PH at the end of the season as a first-level target trait; and (v) perform indirect selection for DBY as the second-level target trait. This selection approach together with the trait-assisted genomic selection approach proposed by <a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">Fernandes <em>et al.</em> (2018)</a> may allow the end-of-season rank order of observed and eventually unobserved lines to be accurately predicted early in the growing season. When integrated with high-throughput phenotyping platforms, the two-level indirect selection framework has the potential to further accelerate selection cycles and support a larger number of evaluated families. This could be accomplished by deploying low-cost ground rovers (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gage <em>et al.</em> 2019</a>) for the early season measurement of height and other traits genetically correlated with DBY on field-grown plants in combination with off-season winter nurseries and greenhouses with automated phenotyping systems (<em>e.g.</em>, Lemnatec, <a href="http://www.lemnatec.com">http://www.lemnatec.com</a>; Photon Systems Instruments, <a href="http://www.psi.cz">http://www.psi.cz</a>) and optimized growing conditions for speed breeding (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Watson <em>et al.</em> 2018</a>).</p></div><div class="section conclusions" id="sec-19"><h2 class="">Conclusion</h2><p id="p-48">We analyzed phenotypic measures over time for PH and DBY at the end of the season to design a novel indirect selection scheme. To that end, we developed and evaluated novel Bayesian networks for GP that were used to better model and understand phenotypic plasticity of PH across different plant developmental stages. The GP models showed minor differences in prediction accuracies for the fivefold CV scheme. In stark contrast, in the forward-chaining CV scheme, we observed a 36.4–52.4% improvement in prediction accuracy of the DBN and multivariate GBLUP models (train on 30-45 DAP, predict 120 DAP) compared to the BN and PBN models that assumed independence over time. The Bayesian models were used to show that the ranking of lines changed minimally after 45 DAP based on the CI and CIL, serving as novel approaches to understand ranking dynamics with repeated measures. These results suggest that in these environments 45 DAP is an optimal developmental stage for imposing a two-level indirect selection framework for biomass sorghum. Therefore, indirect selection for end of season PH (first-level target trait) and DBY (second-level target trait) could be performed based on the ranking of lines by PH at 45 DAP (secondary trait).</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-49">We thank Tim Mies, Derek Lewis, and Trace Elliott for technical assistance with crop planting and harvesting. This work was supported by FAPESP (Fundaҫão de Amparo á Pesquisa do Estado de São Paulo) Grant 2017/03625-2 and 2017/25674-5. CAPES (Coordenaҫão de Aperfeiҫoamento de Pessoal de Nível Superior) Finance Code 001. CNPq (Conselho Nacional de Desenvolvimento Científico e Tecnológico). The information, data, or work presented herein was funded in part by the Advanced Research Projects Agency-Energy (ARPA-E), U.S. Department of Energy, under Award Number DE-AR0000598 and DE-AR0000661. This work was also supported by the USDA-ARS. The views and opinions of authors expressed herein do not necessarily state or reflect those of the United States Government or any agency thereof.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-50">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.9891119">https://doi.org/10.25387/g3.9891119</a>.</p></li><li class="fn" id="fn-2"><p id="p-51"><em>Communicating editor: P. Morrell</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-09-22"><span class="received-label">Received </span>September 22, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-12-15"><span class="accepted-label">Accepted </span>December 15, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 dos Santos <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.1"><div class="cit-metadata unstructured">Akdemir, D. and O. U. Godfrey, 2018 <em>EMMREML: Fitting Mixed Models with Known Covariance Structures (Version 3.1)</em>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.2" data-doi="10.1038/srep25156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bae</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Monti</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Montano</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. H.</span> <span class="cit-name-surname">Steinberg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. T.</span> <span class="cit-name-surname">Perls</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Learning Bayesian Networks from Correlated Data.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">25156</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep25156</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D25156%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep25156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep25156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.3" data-doi="10.1002/rob.21830"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Tang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Breitzman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. G.</span> <span class="cit-name-surname">Salas Fernandez</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. S.</span> <span class="cit-name-surname">Schnable</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Field-based robotic phenotyping of sorghum plant architecture using stereo vision.</span> <abbr class="cit-jnl-abbrev">J. Field Robot.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">397</span>–<span class="cit-lpage">415</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/rob.21830</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BField%2BRobot.%26rft.volume%253D36%26rft.spage%253D397%26rft_id%253Dinfo%253Adoi%252F10.1002%252Frob.21830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/rob.21830&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.4" data-doi="10.1098/rsta.2012.0222"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>,  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Model-based machine learning.</span> <abbr class="cit-jnl-abbrev">Phil Trans R Soc A</abbr> <span class="cit-vol">371</span>: <span class="cit-elocation-id">20120222</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rsta.2012.0222</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhil%2BTrans%2BR%2BSoc%2BA%26rft.volume%253D371%26rft.spage%253D20120222%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsta.2012.0222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rsta.2012.0222&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.5" data-doi="10.1534/genetics.115.183947"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brenton</span>,  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Cooper</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. T.</span> <span class="cit-name-surname">Myers</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. E.</span> <span class="cit-name-surname">Boyles</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Shakoor</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A genomic resource for the development, improvement, and exploitation of sorghum for bioenergy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">204</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">33</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.183947</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.183947%26rft_id%253Dinfo%253Apmid%252F27356613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMjA0LzEvMjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzc2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.6" data-doi="10.1016/j.ajhg.2015.11.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Browning</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genotype imputation with millions of reference samples.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">116</span>–<span class="cit-lpage">126</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2015.11.020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D98%26rft.spage%253D116%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2015.11.020%26rft_id%253Dinfo%253Apmid%252F26748515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2015.11.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26748515&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.7" data-doi="10.1101/096628"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>,  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. C.</span> <span class="cit-name-surname">Ilut</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Kretzschmar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Gore</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">rampseq: Using repetitive sequences for robust genotyping.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1101/096628</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F096628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIwOTY2Mjh2MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvNzY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.8" data-doi="10.2135/cropsci2011.06.0299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Weigel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic Prediction of Breeding Values when Modeling Genotype × Environment Interaction using Pedigree and Dense Molecular Markers.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">719</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2011.06.0299</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>,  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. R.</span> <span class="cit-name-surname">Cullis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Gilmour</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Gogel</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <em>ASReml-R reference manual</em>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.10" data-doi="10.1111/j.1541-0420.2005.00334.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caliński</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Czajka</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Kaczmarek</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Krajewski</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Pilarczyk</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Analyzing multi-environment variety trials using randomization-derived mixed models.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">448</span>–<span class="cit-lpage">455</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1541-0420.2005.00334.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DCali%25C5%2584ski%26rft.auinit1%253DT.%26rft.volume%253D61%26rft.issue%253D2%26rft.spage%253D448%26rft.epage%253D455%26rft.atitle%253DAnalyzing%2Bmulti-environment%2Bvariety%2Btrials%2Busing%2Brandomization-derived%2Bmixed%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1541-0420.2005.00334.x%26rft_id%253Dinfo%253Apmid%252F16011691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1541-0420.2005.00334.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16011691&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.11" data-doi="10.1186/1297-9686-43-26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>,  <span class="cit-name-given-names">M. P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. F.</span> <span class="cit-name-surname">Veerkamp</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy of multi-trait genomic selection using different methods.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-43-26</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DCalus%26rft.auinit1%253DM.%2BP.%26rft.volume%253D43%26rft.spage%253D26%26rft.epage%253D26%26rft.atitle%253DAccuracy%2Bof%2Bmulti-trait%2Bgenomic%2Bselection%2Busing%2Bdifferent%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-26%26rft_id%253Dinfo%253Apmid%252F21729282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-26&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21729282&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.12" data-doi="10.1002/pld3.80"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Walia</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Morota</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Utilizing random regression models for genomic prediction of a longitudinal trait derived from high-throughput phenotyping.</span> <abbr class="cit-jnl-abbrev">Plant Direct</abbr> <span class="cit-vol">2</span>: <span class="cit-elocation-id">e00080</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/pld3.80</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BDirect%26rft.volume%253D2%26rft.spage%253D00080e%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpld3.80%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/pld3.80&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gelman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hoffman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Lee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Goodrich</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Stan: A probabilistic programming language. Journal of Statistical Software</span>. <abbr class="cit-jnl-abbrev">Articles</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArticles%26rft.volume%253D76%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.14" data-doi="10.1038/nrg3012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>,  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">Hohenlohe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. D.</span> <span class="cit-name-surname">Etter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. Q.</span> <span class="cit-name-surname">Boone</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Catchen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide genetic marker discovery and genotyping using next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">510</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDavey%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D510%26rft.atitle%253DGenome-wide%2Bgenetic%2Bmarker%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3012%26rft_id%253Dinfo%253Apmid%252F21681211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21681211&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.15" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Hickey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Pong-Wong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. D.</span> <span class="cit-name-surname">Daetwyler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P. L.</span> <span class="cit-name-surname">Calus</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.143313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi83NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.16" data-doi="10.1038/s41437-018-0053-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dias</span>,  <span class="cit-name-given-names">K. O. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Gezan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. T.</span> <span class="cit-name-surname">Guimarães</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Nazarian</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.S.</span> <span class="cit-name-surname">da Costa E.</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Improving accuracies of genomic predictions for drought tolerance in maize by joint modeling of additive and dominance effects in multi-environment trials.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">37</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41437-018-0053-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D121%26rft.spage%253D24%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41437-018-0053-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41437-018-0053-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.17" data-doi="10.1186/s12863-016-0392-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">dos Santos</span>,  <span class="cit-name-given-names">J. P. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Paulo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Pires</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Coelho</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. C.</span> <span class="cit-name-surname">Vasconcellos</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">a</span> <span class="cit-article-title">Genomic selection to resistance to Stenocarpella maydis in maize lines using DArTseq markers.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">86</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12863-016-0392-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D17%26rft.spage%253D86%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12863-016-0392-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12863-016-0392-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.18" data-doi="10.1371/journal.pone.0152045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">dos Santos</span>,  <span class="cit-name-given-names">J. P. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. C.</span> <span class="cit-name-surname">De Castro Vasconcellos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. P. M.</span> <span class="cit-name-surname">Pires</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Balestre</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. G.</span> <span class="cit-name-surname">Von Pinho</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">b</span> <span class="cit-article-title">Inclusion of dominance effects in the multivariate GBLUP model.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e0152045</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0152045</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0152045%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0152045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0152045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.19" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>,  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Glaubitz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Sun</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Poland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kawamoto</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A Robust, Simple Genotyping-by-Sequencing (GBS) Approach for High Diversity Species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e19379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0019379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.20" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>,  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge regression and other kernels for genomic selection with r package rrblup.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2011.08.0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.21" data-doi="10.1007/s00122-017-3033-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernandes</span>,  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. O. G.</span> <span class="cit-name-surname">Dias</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. F.</span> <span class="cit-name-surname">Ferreira</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Brown</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Efficiency of multi-trait, indirect, and trait-assisted genomic selection for improvement of biomass sorghum.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">747</span>–<span class="cit-lpage">755</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-017-3033-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D131%26rft.spage%253D747%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-017-3033-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-017-3033-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.22" data-doi="10.1038/s41437-018-0105-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferrão</span>,  <span class="cit-name-given-names">L. F. V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. G.</span> <span class="cit-name-surname">Ferrão</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A. G.</span> <span class="cit-name-surname">Ferrão</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Fonseca</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Carbonetto</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Accurate genomic prediction of Coffea canephora in multiple environments using whole-genome statistical models.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">261</span>–<span class="cit-lpage">275</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41437-018-0105-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41437-018-0105-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41437-018-0105-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.23" data-doi="10.1038/nature10452"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foley</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Ramankutty</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Brauman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. S.</span> <span class="cit-name-surname">Cassidy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Gerber</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Solutions for a cultivated planet.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">478</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">342</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature10452</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DFoley%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D478%26rft.issue%253D7369%26rft.spage%253D337%26rft.epage%253D342%26rft.atitle%253DSolutions%2Bfor%2Ba%2Bcultivated%2Bplanet.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10452%26rft_id%253Dinfo%253Apmid%252F21993620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10452&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21993620&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296021100036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gage</span>,  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Richards</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Lepak</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Kaczmar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Soman</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">In-field whole plant maize architecture characterized by latent space phenotyping.</span> <abbr class="cit-jnl-abbrev">The Plant Phe. J</abbr>. <span class="cit-vol">2</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BPhe.%2BJ%26rft.volume%253D2%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.25" data-doi="10.1038/srep03399"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>,  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Mollinari</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. G.</span> <span class="cit-name-surname">Marconi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. R.</span> <span class="cit-name-surname">Serang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. R.</span> <span class="cit-name-surname">Silva</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">SNP genotyping allows an in-depth characterisation of the genome of sugarcane and other complex autopolyploids.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">3399</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep03399</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D3%26rft.spage%253D3399%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep03399%26rft_id%253Dinfo%253Apmid%252F24292365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep03399&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24292365&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gelman</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Carlin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. S.</span> <span class="cit-name-surname">Stern</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. B.</span> <span class="cit-name-surname">Rubin</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <em>Bayesian Data Analysis</em>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.27" data-doi="10.1534/genetics.113.151753"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>,  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Priors in whole-genome regression: the bayesian alphabet returns.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">596</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.151753</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.151753%26rft_id%253Dinfo%253Apmid%252F23636739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi83NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.28" data-doi="10.1534/genetics.109.103952"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. G.</span> <span class="cit-name-surname">Hill</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Manfredi</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Fernando</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Additive genetic variability and the Bayesian alphabet.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">363</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.103952</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.103952%26rft_id%253Dinfo%253Apmid%252F19620397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgzLzEvMzQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi83NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.29" data-doi="10.1534/genetics.115.179978"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Toro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Naya</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.-C.</span> <span class="cit-name-surname">Schön</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Do molecular markers inform about pleiotropy?</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">29</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.179978</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.179978%26rft_id%253Dinfo%253Apmid%252F26205989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMjAxLzEvMjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzc2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.30" data-doi="10.1371/journal.pone.0090346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>,  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. M.</span> <span class="cit-name-surname">Casstevens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Lu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Harriman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Elshire</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Tassel-gbs: A high capacity genotyping by sequencing analysis pipeline.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e90346</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0090346</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De90346%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-web" id="cit-10.2.769.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodfellow</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Bengio</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Courville</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <em>Deep Learning</em>. MIT Press, <a href="http://www.deeplearningbook.org">http://www.deeplearningbook.org</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. O.</span> <span class="cit-name-surname">Zimmermann</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Breeding common bean for yield in mixtures.</span> <abbr class="cit-jnl-abbrev">Plant Breed. Rev.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%2BRev.%26rft.volume%253D4%26rft.spage%253D245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamelryck</span>,  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <em>Bayesian Methods in Structural Bioinformatics</em>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-642-27225-7</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.34" data-doi="10.1186/1752-0509-6-S2-S14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Talebizadeh</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Xu</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic studies of complex human diseases: characterizing SNP-disease associations using Bayesian networks.</span> <abbr class="cit-jnl-abbrev">BMC Syst. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">S14</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1752-0509-6-S3-S14</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BSyst.%2BBiol.%26rft.volume%253D6%26rft.spage%253DS14%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1752-0509-6-S2-S14%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1752-0509-6-S2-S14&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.35" data-doi="10.2135/cropsci1972.0011183X001200020005x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harlan</span>,  <span class="cit-name-given-names">J. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M. J.</span> <span class="cit-name-surname">de Wet</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">A simplified classification of cultivated sorghum.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">172</span>–<span class="cit-lpage">176</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1972.0011183X001200020005x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHarlan%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253D172%26rft.epage%253D176%26rft.atitle%253DA%2BSimplified%2BClassification%2Bof%2BCultivated%2BSorghum%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1972.0011183X001200020005x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1972.0011183X001200020005x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1972M322900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.36" data-doi="10.2527/jas1976.4361188x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>,  <span class="cit-name-given-names">C. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Quaas</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Multiple trait evaluation using relatives’ records.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1188</span>–<span class="cit-lpage">1197</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas1976.4361188x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DHenderson%26rft.auinit1%253DC.%2BR.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D1188%26rft.epage%253D1197%26rft.atitle%253DMultiple%2BTrait%2BEvaluation%2BUsing%2BRelatives%2527%2BRecords%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas1976.4361188x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas1976.4361188x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1976CP41700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.37" data-doi="10.2135/cropsci2014.03.0249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Jannink</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Sorrels</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Perspectives for Genomic Selection Applications and Research in Plants.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2014.03.0249</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.03.0249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.03.0249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffman</span>,  <span class="cit-name-given-names">M. D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gelman</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The No-U-Turn Sampler: Adaptively Setting Path Lengths in Hamiltonian Monte Carlo.</span> <abbr class="cit-jnl-abbrev">J. Mach. Learn. Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1593</span>–<span class="cit-lpage">1623</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMach.%2BLearn.%2BRes.%26rft.volume%253D15%26rft.spage%253D1593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.769.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Nyquist</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Cervantes</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Estimating and interpreting heritability for plant breeding: An update.</span> plant breeding reviews <span class="cit-vol">22</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">112</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.40" data-doi="10.1038/hdy.2011.103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hung</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Browne</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Guill</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Coles</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Eller</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The relationship between parental genetic or phenotypic divergence and progeny variation in the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">490</span>–<span class="cit-lpage">499</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2011.103</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHung%26rft.auinit1%253DH.%2BY.%26rft.volume%253D108%26rft.issue%253D5%26rft.spage%253D490%26rft.epage%253D499%26rft.atitle%253DThe%2Brelationship%2Bbetween%2Bparental%2Bgenetic%2Bor%2Bphenotypic%2Bdivergence%2Band%2Bprogeny%2Bvariation%2Bin%2Bthe%2Bmaize%2Bnested%2Bassociation%2Bmapping%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2011.103%26rft_id%253Dinfo%253Apmid%252F22027895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2011.103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22027895&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.41" data-doi="10.1534/genetics.112.144246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multiple-trait genomic selection methods increase genetic value prediction accuracy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1513</span>–<span class="cit-lpage">1522</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.144246</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.144246%26rft_id%253Dinfo%253Apmid%252F23086217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzE1MTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzc2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.42" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Salzberg</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1923</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.43" data-doi="10.1105/tpc.107.053660"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawrence</span>,  <span class="cit-name-given-names">C. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Walbot</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Translational Genomics for Bioenergy Production from Fuelstock Grasses: Maize as the Model Species.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">2091</span>–<span class="cit-lpage">2094</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.107.053660</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.107.053660%26rft_id%253Dinfo%253Apmid%252F17660357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE5LzcvMjA5MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvNzY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.44" data-doi="10.1016/j.tplants.2018.01.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Rasheed</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. T.</span> <span class="cit-name-surname">Hickey</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">He</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Fast-forwarding genetic gain.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">184</span>–<span class="cit-lpage">186</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2018.01.007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D23%26rft.spage%253D184%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2018.01.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2018.01.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loman</span>,  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Quick</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. T.</span> <span class="cit-name-surname">Simpson</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A complete bacterial genome assembled de novo using only nanopore sequencing data.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-vol">12</span>: <span class="cit-elocation-id">015552</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.769.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Walsh</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source">Genetics and analysis of quantitative traits</span>, <span class="cit-vol">Vol. 1</span>. <span class="cit-publ-name">Sinauer Sunderland</span>, <span class="cit-publ-loc">MA</span>. <span class="cit-vol">17</span>: <span class="cit-fpage">537</span>–<span class="cit-lpage">550</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%2Band%2Banalysis%2Bof%2Bquantitative%2Btraits%26rft.volume%253D17%26rft.spage%253D537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.47" data-doi="10.1038/ncomms3320"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mace</span>,  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Tai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. K.</span> <span class="cit-name-surname">Gilding</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Prentis</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-genome sequencing reveals untapped genetic potential in Africa’s indigenous cereal crop sorghum.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">2320</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms3320</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D4%26rft.spage%253D2320%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms3320%26rft_id%253Dinfo%253Apmid%252F23982223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms3320&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23982223&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.48" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>,  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzc2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>,  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Montesinos-López</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. H.</span> <span class="cit-name-surname">Toledo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Pérez-Hernández</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A genomic bayesian multi-trait and multi-environment model. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2725</span>–<span class="cit-lpage">2744</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D6%26rft.spage%253D2725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.50" data-doi="10.1073/pnas.1215985110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>,  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Ramu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. P.</span> <span class="cit-name-surname">Deshpande</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. T.</span> <span class="cit-name-surname">Hash</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Shah</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Population genomic and genome-wide association studies of agroclimatic traits in sorghum.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">453</span>–<span class="cit-lpage">458</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1215985110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1215985110%26rft_id%253Dinfo%253Apmid%252F23267105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMTAvMi80NTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzc2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.51" data-doi="10.1093/jxb/eru229"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mullet</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Morishige</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">McCormick</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Truong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hilley</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Energy sorghum–a genetic model for the design of C4 grass bioenergy crops.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">3479</span>–<span class="cit-lpage">3489</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jxb/eru229</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Feru229%26rft_id%253Dinfo%253Apmid%252F24958898%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/eru229&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24958898&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.52" data-doi="10.1111/tpj.13390"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muraya</span>,  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Chu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Junker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Klukas</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genetic variation of growth dynamics in maize (zea mays l.) revealed through automated non-invasive phenotyping.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">366</span>–<span class="cit-lpage">380</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13390</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D89%26rft.spage%253D366%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13390&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.769.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>,  <span class="cit-name-given-names">K. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source">Machine learning: a probabilistic perspective</span>, <span class="cit-publ-name">MIT Press</span>, <span class="cit-publ-loc">Cambridge, Mass.</span> [<span class="cit-comment">u.a.</span>].</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neapolitan</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Xue</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Jiang</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Modeling the altered expression levels of genes on signaling pathways in tumors as causal bayesian networks.</span> <abbr class="cit-jnl-abbrev">Cancer Inform.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BInform.%26rft.volume%253D13%26rft.spage%253D77%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.55" data-doi="10.1186/s12711-017-0361-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Okeke</span>,  <span class="cit-name-given-names">U. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Akdemir</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Rabbi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Kulakow</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Accuracies of univariate and multivariate genomic prediction models in african cassava.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">88</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-017-0361-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D49%26rft.spage%253D88%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-017-0361-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-017-0361-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.56" data-doi="10.1093/biomet/58.3.545"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>,  <span class="cit-name-given-names">H. D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Thompson</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">Recovery of inter-block information when block sizes are unequal.</span> <abbr class="cit-jnl-abbrev">Biometrika</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">545</span>–<span class="cit-lpage">554</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/biomet/58.3.545</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft.stitle%253DBiometrika%26rft.aulast%253DPATTERSON%26rft.auinit1%253DH.%2BD.%26rft.volume%253D58%26rft.issue%253D3%26rft.spage%253D545%26rft.epage%253D554%26rft.atitle%253DRecovery%2Bof%2Binter-block%2Binformation%2Bwhen%2Bblock%2Bsizes%2Bare%2Bunequal%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252F58.3.545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biomet/58.3.545&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1971L054700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.57" data-doi="10.1104/pp.16.00592"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pauli</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. C.</span> <span class="cit-name-surname">Chapman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Bart</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. N.</span> <span class="cit-name-surname">Topp</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Lawrence-Dill</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The quest for understanding phenotypic variation via integrated approaches in the field environment.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">622</span>–<span class="cit-lpage">634</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.16.00592%26rft_id%253Dinfo%253Apmid%252F27482076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNzIvMi82MjIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzc2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.58" data-doi="10.1534/genetics.114.164350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raj</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Stephens</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Pritchard</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">faststructure: Variational inference of population structure in large snp data sets.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">589</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.164350</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164350%26rft_id%253Dinfo%253Apmid%252F24700103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk3LzIvNTczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi83NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.59" data-doi="10.1038/hdy.2015.57."><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ratcliffe</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. G.</span> <span class="cit-name-surname">El-Dien</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Klapste</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Porth</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Chen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A comparison of genomic selection models across time in interior spruce (Picea engelmannii × glauca) using unordered SNP imputation methods.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">547</span>–<span class="cit-lpage">555</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2015.57</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D115%26rft.spage%253D547%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2015.57.%26rft_id%253Dinfo%253Apmid%252F26126540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2015.57.&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26126540&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.60" data-doi="10.1371/journal.pone.0030906"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Serang</span>,  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Mollinari</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A. F.</span> <span class="cit-name-surname">Garcia</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Efficient exact maximum a posteriori computation for Bayesian SNP genotyping in polyploids.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e30906</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0030906</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSerang%26rft.auinit1%253DO.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De30906%26rft.epage%253De30906%26rft.atitle%253DEfficient%2Bexact%2Bmaximum%2Ba%2Bposteriori%2Bcomputation%2Bfor%2Bbayesian%2BSNP%2Bgenotyping%2Bin%2Bpolyploids.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0030906%26rft_id%253Dinfo%253Apmid%252F22363513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0030906&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22363513&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.61" data-doi="10.1186/1756-0381-6-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Su</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Andrew</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Karagas</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Borsuk</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Using Bayesian networks to discover relations between genes, environment, and disease.</span> <abbr class="cit-jnl-abbrev">BioData Min.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">6</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1756-0381-6-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioData%2BMin.%26rft.volume%253D6%26rft.spage%253D6%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1756-0381-6-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1756-0381-6-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-web" id="cit-10.2.769.62"><div class="cit-metadata unstructured">Team, S. D., 2018 <em>PyStan: the Python interface to Stan</em>, <em>Version 2.17.1.0</em>..</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valluru</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. E.</span> <span class="cit-name-surname">Gazave</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. B.</span> <span class="cit-name-surname">Fernandes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. N.</span> <span class="cit-name-surname">Ferguson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Lozano</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Deleterious Mutation Burden and Its Association with Complex Traits in Sorghum (<em>Sorghum bicolor</em>)</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>. <span class="cit-vol">211</span>: <span class="cit-fpage">1075</span>–<span class="cit-lpage">1087</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D211%26rft.spage%253D1075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.64" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>,  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.65" data-doi="10.1111/j.1744-7909.2010.01020.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vermerris</span>,  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Survey of Genomics Approaches to Improve Bioenergy Traits in Maize, Sorghum and Sugarcane.</span> <abbr class="cit-jnl-abbrev">J. Integr. Plant Biol.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">119</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1744-7909.2010.01020.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BIntegr%2BPlant%2BBiol%26rft.aulast%253DVermerris%26rft.auinit1%253DW.%26rft.volume%253D53%26rft.issue%253D2%26rft.spage%253D105%26rft.epage%253D119%26rft.atitle%253DSurvey%2Bof%2Bgenomics%2Bapproaches%2Bto%2Bimprove%2Bbioenergy%2Btraits%2Bin%2Bmaize%252C%2Bsorghum%2Band%2Bsugarcane.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1744-7909.2010.01020.x%26rft_id%253Dinfo%253Apmid%252F21205186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1744-7909.2010.01020.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21205186&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.66" data-doi="10.1038/s41477-017-0083-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watson</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Ghosh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Williams</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. S.</span> <span class="cit-name-surname">Cuddy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Simmonds</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Speed breeding is a powerful tool to accelerate crop research and breeding.</span> <abbr class="cit-jnl-abbrev">Nat. Plants</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">29</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41477-017-0083-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BPlants%26rft.volume%253D4%26rft.spage%253D23%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41477-017-0083-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41477-017-0083-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.67" data-doi="10.1111/j.2041-210X.2012.00213.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolak</span>,  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">nadiv: an R package to create relatedness matrices for estimating non-additive genetic variances in animal models.</span> <abbr class="cit-jnl-abbrev">Methods Ecol. Evol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">792</span>–<span class="cit-lpage">796</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.2041-210X.2012.00213.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol.%2BEvol.%26rft.volume%253D3%26rft.spage%253D792%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.2041-210X.2012.00213.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.2041-210X.2012.00213.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.68" data-doi="10.1534/genetics.113.155309"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic mapping and genomic selection using recombination breakpoint data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">1103</span>–<span class="cit-lpage">1115</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.155309</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.155309%26rft_id%253Dinfo%253Apmid%252F23979575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS8zLzExMDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzc2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.69" data-doi="10.1038/nplants.2016.150"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Guo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Zhu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction contributing to a promising global strategy to turbocharge gene banks.</span> <abbr class="cit-jnl-abbrev">Nat. Plants</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">16150</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nplants.2016.150</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BPlants%26rft.volume%253D2%26rft.spage%253D16150%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnplants.2016.150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nplants.2016.150&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.769.70" data-doi="10.1093/bioinformatics/bts606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Levine</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Shen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Gogarten</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Laurie</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A high-performance computing toolset for relatedness and principal component analysis of SNP data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">3326</span>–<span class="cit-lpage">3328</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts606</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts606%26rft_id%253Dinfo%253Apmid%252F23060615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts606&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23060615&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312105300026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/2/769.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/2/755" title="Origin and Evolution of Two Independently Duplicated Genes Encoding UDP- Glucose: Glycoprotein Glucosyltransferases in Caenorhabditis and Vertebrates" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/2/783" title="The Impact of Non-additive Effects on the Genetic Correlation Between Populations" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/2"><span>Volume 10 Issue 2, February 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/2.cover-source.jpg?itok=LSawHOSr" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/769" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-yCb7TKkSuZFJAVhgpXdHlcRzpfIU9WLztGAHQqjfMRo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/2/769" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/2/769" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/2/769" class="active" data-icon-position="" data-hide-link-title="0">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495543" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-jyBYjPKOd24C-1gDmBUPay9__2uJaMsZVwzLsv6CFOY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/2/769.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/2/769" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-WmPz030mKtc3mutepocp_7-2gLbGdwYxYd1HDu3XCIw" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/2/769" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/2/769.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543--2" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9030-702X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jhonathan P. R. dos Santos</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8269-535X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Samuel B. Fernandes</span>, <span class="highwire-citation-author" data-delta="2">Scott McCoy</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-0760-4977" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Roberto Lozano</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-1332-711X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick J. Brown</span>, <span class="highwire-citation-author" data-delta="5">Andrew D.B. Leakey</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-3100-371X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Edward S. Buckler</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-0634-3277" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Antonio A. F. Garcia</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael A. Gore</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">769-781;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400759 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495543--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jhonathan P. R. dos Santos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, SP, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jhonathan%2BP.%2BR.%2Bdos%2BSantos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=dos%20Santos%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJhonathan%2BP.%2BR.%2Bdos%2BSantos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9030-702X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jhonathan P. R. dos Santos</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Samuel B. Fernandes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop Science,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Samuel%2BB.%2BFernandes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fernandes%20SB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASamuel%2BB.%2BFernandes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8269-535X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Samuel B. Fernandes</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Scott McCoy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Institute for Genomic Biology,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BMcCoy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCoy%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AScott%2BMcCoy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Roberto Lozano </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roberto%2BLozano%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lozano%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoberto%2BLozano%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0760-4977" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Roberto Lozano</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Patrick J. Brown </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Section of Agricultural Plant Biology, Department of Plant Sciences, University of California Davis, 95616, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BJ.%2BBrown%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brown%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BJ.%2BBrown%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1332-711X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick J. Brown</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Andrew D.B. Leakey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop Science,</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Institute for Genomic Biology,</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Plant Biology, University of Illinois at Urbana Champaign, 61801,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BD.B.%2BLeakey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Leakey%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BD.B.%2BLeakey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Edward S. Buckler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science,</div><div class='author-affiliation'><span class='nlm-sup'>**</span>United States Department of Agriculture, Agricultural Research Service, R. W. Holley Center, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Institute for Genomic Diversity, Cornell University, Ithaca, New York 14853,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edward%2BS.%2BBuckler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buckler%20ES&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEdward%2BS.%2BBuckler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3100-371X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Edward S. Buckler</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Antonio A. F. Garcia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, SP, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonio%2BA.%2BF.%2BGarcia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garcia%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntonio%2BA.%2BF.%2BGarcia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0634-3277" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Antonio A. F. Garcia</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:augusto.garcia@usp.br" class="" data-icon-position="" data-hide-link-title="0">augusto.garcia@usp.br</a>
<a href="mailto:mag87@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">mag87@cornell.edu</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Michael A. Gore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BA.%2BGore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gore%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BA.%2BGore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael A. Gore</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:augusto.garcia@usp.br" class="" data-icon-position="" data-hide-link-title="0">augusto.garcia@usp.br</a>
<a href="mailto:mag87@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">mag87@cornell.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/2/769" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/2/769&amp;title=Novel%20Bayesian%20Networks%20for%20Genomic%20Prediction%20of%20Developmental%20Traits%20in%20Biomass%20Sorghum" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/2/769&amp;title=Novel%20Bayesian%20Networks%20for%20Genomic%20Prediction%20of%20Developmental%20Traits%20in%20Biomass%20Sorghum" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/2/769&amp;title=Novel%20Bayesian%20Networks%20for%20Genomic%20Prediction%20of%20Developmental%20Traits%20in%20Biomass%20Sorghum" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/2/769&amp;text=Novel%20Bayesian%20Networks%20for%20Genomic%20Prediction%20of%20Developmental%20Traits%20in%20Biomass%20Sorghum" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/2/769&amp;title=Novel%20Bayesian%20Networks%20for%20Genomic%20Prediction%20of%20Developmental%20Traits%20in%20Biomass%20Sorghum" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/2/769&amp;t=Novel%20Bayesian%20Networks%20for%20Genomic%20Prediction%20of%20Developmental%20Traits%20in%20Biomass%20Sorghum" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/2/769&amp;title=Novel%20Bayesian%20Networks%20for%20Genomic%20Prediction%20of%20Developmental%20Traits%20in%20Biomass%20Sorghum" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/2/769&amp;title=Novel%20Bayesian%20Networks%20for%20Genomic%20Prediction%20of%20Developmental%20Traits%20in%20Biomass%20Sorghum" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495543/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543--3" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9030-702X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jhonathan P. R. dos Santos</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8269-535X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Samuel B. Fernandes</span>, <span class="highwire-citation-author" data-delta="2">Scott McCoy</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-0760-4977" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Roberto Lozano</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-1332-711X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick J. Brown</span>, <span class="highwire-citation-author" data-delta="5">Andrew D.B. Leakey</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-3100-371X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Edward S. Buckler</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-0634-3277" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Antonio A. F. Garcia</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael A. Gore</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">769-781;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400759 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495543--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jhonathan P. R. dos Santos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, SP, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jhonathan%2BP.%2BR.%2Bdos%2BSantos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=dos%20Santos%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJhonathan%2BP.%2BR.%2Bdos%2BSantos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9030-702X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jhonathan P. R. dos Santos</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Samuel B. Fernandes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop Science,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Samuel%2BB.%2BFernandes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fernandes%20SB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASamuel%2BB.%2BFernandes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8269-535X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Samuel B. Fernandes</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Scott McCoy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Institute for Genomic Biology,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BMcCoy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCoy%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AScott%2BMcCoy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Roberto Lozano </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roberto%2BLozano%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lozano%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoberto%2BLozano%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0760-4977" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Roberto Lozano</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Patrick J. Brown </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Section of Agricultural Plant Biology, Department of Plant Sciences, University of California Davis, 95616, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BJ.%2BBrown%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brown%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BJ.%2BBrown%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1332-711X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick J. Brown</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Andrew D.B. Leakey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop Science,</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Institute for Genomic Biology,</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Plant Biology, University of Illinois at Urbana Champaign, 61801,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BD.B.%2BLeakey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Leakey%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BD.B.%2BLeakey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Edward S. Buckler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science,</div><div class='author-affiliation'><span class='nlm-sup'>**</span>United States Department of Agriculture, Agricultural Research Service, R. W. Holley Center, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Institute for Genomic Diversity, Cornell University, Ithaca, New York 14853,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edward%2BS.%2BBuckler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buckler%20ES&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEdward%2BS.%2BBuckler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3100-371X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Edward S. Buckler</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Antonio A. F. Garcia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, SP, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonio%2BA.%2BF.%2BGarcia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garcia%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntonio%2BA.%2BF.%2BGarcia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0634-3277" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Antonio A. F. Garcia</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:augusto.garcia@usp.br" class="" data-icon-position="" data-hide-link-title="0">augusto.garcia@usp.br</a>
<a href="mailto:mag87@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">mag87@cornell.edu</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Michael A. Gore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BA.%2BGore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gore%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BA.%2BGore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael A. Gore</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:augusto.garcia@usp.br" class="" data-icon-position="" data-hide-link-title="0">augusto.garcia@usp.br</a>
<a href="mailto:mag87@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">mag87@cornell.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=769&amp;rft.epage=781&amp;rft.atitle=Novel%20Bayesian%20Networks%20for%20Genomic%20Prediction%20of%20Developmental%20Traits%20in%20Biomass%20Sorghum&amp;rft.volume=10&amp;rft.issue=2&amp;rft.date=2020-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=dos+Santos%2C+Jhonathan+P.+R.&amp;rft.au=Fernandes%2C+Samuel+B.&amp;rft.au=McCoy%2C+Scott&amp;rft.au=Lozano%2C+Roberto&amp;rft.au=Brown%2C+Patrick+J.&amp;rft.au=Leakey%2C+Andrew+D.B.&amp;rft.au=Buckler%2C+Edward+S.&amp;rft.au=Garcia%2C+Antonio+A.+F.&amp;rft.au=Gore%2C+Michael+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495543/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495543/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495543/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495543/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495543/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495543/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495543/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495543/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495543/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495543/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495543/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495543/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="last even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/2/769" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/2/769.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/2/769.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/2/769.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
