<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/2/525" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/2/525.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/2/525.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/2/525.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/2/525.atom" />
<meta name="HW.pisa" content="ggg;10/2/525" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Synaptonemal Complex-Deficient Drosophila melanogaster Females Exhibit Rare DSB Repair Events, Recurrent Copy-Number Variation, and an Increased Rate of de Novo Transposable Element Movement" />
<meta name="DC.Identifier" content="10.1534/g3.119.400853" />
<meta name="DC.Date" content="2020-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Miller. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genetic stability depends on the maintenance of a variety of chromosome structures and the precise repair of DNA breaks. During meiosis, programmed double-strand breaks (DSBs) made in prophase I are normally repaired as gene conversions or crossovers. DSBs can also be made by other mechanisms, such as the movement of transposable elements (TEs), which must also be resolved. Incorrect repair of these DNA lesions can lead to mutations, copy-number changes, translocations, and/or aneuploid gametes. In Drosophila melanogaster , as in most organisms, meiotic DSB repair occurs in the presence of a rapidly evolving multiprotein structure called the synaptonemal complex (SC). Here, whole-genome sequencing is used to investigate the fate of meiotic DSBs in D. melanogaster mutant females lacking functional SC, to assay for de novo CNV formation, and to examine the role of the SC in transposable element movement in flies. The data indicate that, in the absence of SC, copy-number variation still occurs and meiotic DSB repair by gene conversion occurs infrequently. Remarkably, an 856-kilobase de novo CNV was observed in two unrelated individuals of different genetic backgrounds and was identical to a CNV recovered in a previous wild-type study, suggesting that recurrent formation of large CNVs occurs in Drosophila. In addition, the rate of novel TE insertion was markedly higher than wild type in one of two SC mutants tested, suggesting that SC proteins may contribute to the regulation of TE movement and insertion in the genome. Overall, this study provides novel insight into the role that the SC plays in genome stability and provides clues as to why the sequence, but not structure, of SC proteins is rapidly evolving." />
<meta name="DC.Contributor" content="Danny E. Miller" />
<meta name="article:published_time" content="2020-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Synaptonemal Complex-Deficient Drosophila melanogaster Females Exhibit Rare DSB Repair Events, Recurrent Copy-Number Variation, and an Increased Rate of de Novo Transposable Element Movement" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genetic stability depends on the maintenance of a variety of chromosome structures and the precise repair of DNA breaks. During meiosis, programmed double-strand breaks (DSBs) made in prophase I are normally repaired as gene conversions or crossovers. DSBs can also be made by other mechanisms, such as the movement of transposable elements (TEs), which must also be resolved. Incorrect repair of these DNA lesions can lead to mutations, copy-number changes, translocations, and/or aneuploid gametes. In &lt;i&gt;Drosophila melanogaster&lt;/i&gt;, as in most organisms, meiotic DSB repair occurs in the presence of a rapidly evolving multiprotein structure called the synaptonemal complex (SC). Here, whole-genome sequencing is used to investigate the fate of meiotic DSBs in &lt;i&gt;D. melanogaster&lt;/i&gt; mutant females lacking functional SC, to assay for &lt;i&gt;de novo&lt;/i&gt; CNV formation, and to examine the role of the SC in transposable element movement in flies. The data indicate that, in the absence of SC, copy-number variation still occurs and meiotic DSB repair by gene conversion occurs infrequently. Remarkably, an 856-kilobase &lt;i&gt;de novo&lt;/i&gt; CNV was observed in two unrelated individuals of different genetic backgrounds and was identical to a CNV recovered in a previous wild-type study, suggesting that recurrent formation of large CNVs occurs in Drosophila. In addition, the rate of novel TE insertion was markedly higher than wild type in one of two SC mutants tested, suggesting that SC proteins may contribute to the regulation of TE movement and insertion in the genome. Overall, this study provides novel insight into the role that the SC plays in genome stability and provides clues as to why the sequence, but not structure, of SC proteins is rapidly evolving.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/02/01" />
<meta name="citation_mjid" content="ggg;10/2/525" />
<meta name="citation_id" content="10/2/525" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/2/525" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/2/525.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/2/525.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/2/525.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400853" />
<meta name="citation_pmid" content="31882405" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="525" />
<meta name="citation_lastpage" content="537" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Danny E. Miller" />
<meta name="citation_author_institution" content="Division of Medical Genetics, Department of Medicine, and" />
<meta name="citation_author_institution" content="Division of Genetic Medicine, Department of Pediatrics, University of Washington, Seattle, Washington 98105, and" />
<meta name="citation_author_institution" content="Seattle Children’s Hospital, Seattle, Washington 98105" />
<meta name="citation_author_email" content="danny.miller@seattlechildrens.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6096-8601" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. F. Altschul;citation_author=T. L. Madden;citation_author=A. A. Schäffer;citation_author=J. Zhang;citation_author=Z. Zhang;citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.;citation_volume=25;citation_year=1997;citation_pmid=9254694;citation_doi=10.1093/nar/25.17.3389" />
<meta name="citation_reference" content="citation_author=M. Bétermier;citation_author=P. Bertrand;citation_author=B. S. Lopez;citation_pages=e1004086;citation_volume=10;citation_year=2014;citation_pmid=24453986;citation_doi=10.1371/journal.pgen.1004086" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. K. Bishop;citation_author=D. Park;citation_author=L. Xu;citation_author=N. Kleckner;citation_title=DMC1: a meiosis-specific yeast homolog of E. coli recA required for recombination, synaptonemal complex formation, and cell cycle progression.;citation_pages=439-456;citation_volume=69;citation_year=1992;citation_issue=3;citation_pmid=1581960;citation_doi=10.1016/0092-8674(92)90446-J" />
<meta name="citation_reference" content="citation_year=2012" />
<meta name="citation_reference" content="citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=C. B. Bridges;citation_title=TRIPLOID INTERSEXES IN DROSOPHILA MELANOGASTER;citation_pages=252-254;citation_volume=54;citation_year=1921;citation_issue=1394;citation_pmid=17769897;citation_doi=10.1126/science.54.1394.252" />
<meta name="citation_reference" content="citation_year=1972" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. Cardoso-Moreira;citation_author=R. J. Arguello;citation_author=A. G. Clark;citation_title=Mutation spectrum of Drosophila CNVs revealed by breakpoint sequencing.;citation_pages=R119;citation_volume=13;citation_year=2012;citation_pmid=23259534;citation_doi=10.1186/gb-2012-13-12-r119" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. Cardoso-Moreira;citation_author=J. Emerson;citation_author=A. G. Clark;citation_author=M. Long;citation_title=Drosophila duplication hotspots are associated with late-replicating regions of the genome.;citation_pages=e1002340-e1002340;citation_volume=7;citation_year=2011;citation_issue=11;citation_pmid=22072977;citation_doi=10.1371/journal.pgen.1002340" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=M. M. Cardoso-Moreira;citation_author=M. Long;citation_title=Mutational bias shaping fly copy number variation: implications for genome evolution.;citation_pages=243-247;citation_volume=26;citation_year=2010;citation_issue=6;citation_pmid=20416969;citation_doi=10.1016/j.tig.2010.03.002" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. A. Collins;citation_author=J. R. Unruh;citation_author=B. D. Slaughter;citation_author=Z. Yu;citation_author=C. M. Lake;citation_title=Corolla is a novel protein that contributes to the architecture of the synaptonemal complex of Drosophila.;citation_volume=198;citation_year=2014;citation_pmid=24913682;citation_doi=10.1534/genetics.114.165290" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=J. M. Cridland;citation_author=K. R. Thornton;citation_title=Validation of Rearrangement Break Points Identified by Paired-End Sequencing in Natural Populations of Drosophila melanogaster.;citation_volume=2;citation_year=2010;citation_pmid=20333226;citation_doi=10.1093/gbe/evq001" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Curtis;citation_author=S. Clark;citation_author=A. Chovnick;citation_author=W. Bender;citation_title=Molecular analysis of recombination events in Drosophila.;citation_pages=653-661;citation_volume=122;citation_year=1989;citation_issue=3;citation_pmid=2503419" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Dakshinamurthy;citation_author=K. M. Nyswaner;citation_author=P. J. Farabaugh;citation_author=D. J. Garfinkel;citation_title=BUD22 Affects Ty1 Retrotransposition and Ribosome Biogenesis in Saccharomyces cerevisiae.;citation_volume=185;citation_year=2010;citation_pmid=20498295;citation_doi=10.1534/genetics.110.119115" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. de Massy;citation_title=Initiation of Meiotic Recombination: How and Where? Conservation and Specificities Among Eukaryotes.;citation_pages=563-599;citation_volume=47;citation_year=2012;citation_doi=10.1146/annurev-genet-110711-155423" />
<meta name="citation_reference" content="citation_journal_abbrev=Mob DNA;citation_author=M. T. Eickbush;citation_author=T. H. Eickbush;citation_title=Retrotransposition of R2 elements in somatic nuclei during the early development of Drosophila.;citation_pages=11-11;citation_volume=2;citation_year=2011;citation_issue=1;citation_pmid=21958913;citation_doi=10.1186/1759-8753-2-11" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=J. Emerson;citation_author=M. Cardoso-Moreira;citation_author=J. O. Borevitz;citation_author=M. Long;citation_title=Natural Selection Shapes Genome-Wide Patterns of Copy-Number Polymorphism in Drosophila melanogaster.;citation_volume=320;citation_year=2008;citation_pmid=18535209;citation_doi=10.1126/science.1158078" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. G. Faust;citation_author=I. M. Hall;citation_title=SAMBLASTER: fast duplicate marking and structural variant read extraction.;citation_volume=30;citation_year=2014;citation_pmid=24812344;citation_doi=10.1093/bioinformatics/btu314" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=E. Feldmann;citation_author=V. Schmiemann;citation_author=W. Goedecke;citation_author=S. Reichenberger;citation_author=P. Pfeiffer;citation_title=DNA double-strand break repair in cell-free extracts from Ku80-deficient cells: implications for Ku serving as an alignment factor in non-homologous DNA end joining;citation_pages=2585-2596;citation_volume=28;citation_year=2000;citation_issue=13;citation_pmid=10871410;citation_doi=10.1093/nar/28.13.2585" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Fraune;citation_author=M. Alsheimer;citation_author=J.-N. Volff;citation_author=K. Busch;citation_author=S. Fraune;citation_title=Hydra meiosis reveals unexpected conservation of structural synaptonemal complex proteins across metazoans.;citation_volume=109;citation_year=2012;citation_pmid=23012415;citation_doi=10.1073/pnas.1206875109" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Fraune;citation_author=C. Brochier-Armanet;citation_author=M. Alsheimer;citation_author=R. Benavente;citation_title=Phylogenies of Central Element Proteins Reveal the Dynamic Evolutionary History of the Mammalian Synaptonemal Complex: Ancient and Recent Components.;citation_volume=195;citation_year=2013;citation_pmid=24026100;citation_doi=10.1534/genetics.113.156679" />
<meta name="citation_reference" content="citation_journal_title=Chromosoma;citation_author=J. Fraune;citation_author=C. Brochier-Armanet;citation_author=M. Alsheimer;citation_author=J.-N. Volff;citation_author=K. Schücker;citation_title=Evolutionary history of the mammalian synaptonemal complex.;citation_pages=355-360;citation_volume=125;citation_year=2016;citation_doi=10.1007/s00412-016-0583-8" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. L. Goldberg;citation_author=J.-Y. Sheen;citation_author=W. J. Gehring;citation_author=M. Green;citation_title=Unequal crossing-over associated with asymmetrical synapsis between nomadic elements in the Drosophila melanogaster genome;citation_pages=5017-5021;citation_volume=80;citation_year=1983;citation_issue=16;citation_pmid=16593354;citation_doi=10.1073/pnas.80.16.5017" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=T. Goldfarb;citation_author=M. Lichten;citation_title=Frequent and efficient use of the sister chromatid for DNA double-strand break repair during budding yeast meiosis.;citation_pages=e1000520-e1000520;citation_volume=8;citation_year=2010;citation_issue=10;citation_pmid=20976044;citation_doi=10.1371/journal.pbio.1000520" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Zool.;citation_author=J. W. Gowen;citation_title=Meiosis as a genetic character in Drosophila melanogaster.;citation_pages=83-106;citation_volume=65;citation_year=1933;citation_doi=10.1002/jez.1400650106" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=B. M. Green;citation_author=K. J. Finn;citation_author=J. J. Li;citation_title=Loss of DNA Replication Control Is a Potent Inducer of Gene Amplification;citation_pages=943-946;citation_volume=329;citation_year=2010;citation_issue=5994;citation_pmid=20724634;citation_doi=10.1126/science.1190966" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_title=What makes transposable elements move in the Drosophila genome?;citation_volume=108;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Guirouilh-Barbat;citation_author=E. Rass;citation_author=I. Plo;citation_author=P. Bertrand;citation_author=B. S. Lopez;citation_title=Defects in XRCC4 and KU80 differentially affect the joining of distal nonhomologous ends.;citation_volume=104;citation_year=2007;citation_pmid=18093953;citation_doi=10.1073/pnas.0708541104" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Hall;citation_title=CHROMOSOME SEGREGATION INFLUENCED BY TWO ALLELES OF THE MEIOTIC MUTANT c(3)G IN DROSOPHILA MELANOGASTER;citation_pages=367-400;citation_volume=71;citation_year=1972;citation_issue=3;citation_pmid=4624918" />
<meta name="citation_reference" content="citation_journal_title=Drosoph. Inf. Serv.;citation_author=J. Hall;citation_title=Recombination influenced by two alleles of the meiotic mutant c(3)G.;citation_pages=143-144;citation_volume=52;citation_year=1977" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. Handel;citation_author=J. C. Schimenti;citation_title=Genetics of mammalian meiosis: regulation, dynamics and impact on fertility.;citation_pages=124-136;citation_volume=11;citation_year=2010;citation_issue=2;citation_pmid=20051984;citation_doi=10.1038/nrg2723" />
<meta name="citation_reference" content="citation_journal_title=BMC Evol. Biol.;citation_author=L. W. Hemmer;citation_author=J. P. Blumenstiel;citation_title=Holding it together: rapid evolution and positive selection in the synaptonemal complex of Drosophila.;citation_pages=91;citation_volume=16;citation_year=2016;citation_doi=10.1186/s12862-016-0670-8" />
<meta name="citation_reference" content="citation_journal_title=Chromosoma;citation_journal_abbrev=Chromosoma;citation_author=A. Hernández-Hernández;citation_author=H. Rincón-Arano;citation_author=F. Recillas-Targa;citation_author=R. Ortiz;citation_author=C. Valdes-Quezada;citation_title=Differential distribution and association of repeat DNA sequences in the lateral element of the synaptonemal complex in rat spermatocytes.;citation_pages=77-87;citation_volume=117;citation_year=2008;citation_issue=1;citation_pmid=17968579;citation_doi=10.1007/s00412-007-0128-2" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Hilliker;citation_author=G. Harauz;citation_author=A. Reaume;citation_author=M. Gray;citation_author=S. Clark;citation_title=Meiotic gene conversion tract length distribution within the rosy locus of Drosophila melanogaster.;citation_pages=1019-1026;citation_volume=137;citation_year=1994;citation_issue=4;citation_pmid=7982556" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Hinton;citation_title=ENHANCEMENT OF RECOMBINATION ASSOCIATED WITH THE c3G MUTANT OF DROSOPHILA MELANOGASTER;citation_pages=157-164;citation_volume=53;citation_year=1966;citation_issue=1;citation_pmid=5900602" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=W. Huang;citation_author=L. Li;citation_author=J. R. Myers;citation_author=G. T. Marth;citation_title=ART: a next-generation sequencing read simulator.;citation_volume=28;citation_year=2012;citation_pmid=22199392;citation_doi=10.1093/bioinformatics/btr708" />
<meta name="citation_reference" content="citation_year=2016" />
<meta name="citation_reference" content="citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. K. Jeffress;citation_author=S. L. Page;citation_author=S. K. Royer;citation_author=E. D. Belden;citation_author=J. P. Blumenstiel;citation_title=The formation of the central element of the synaptonemal complex may occur by multiple mechanisms: the roles of the N- and C-terminal domains of the Drosophila C(3)G protein in mediating synapsis and recombination.;citation_volume=177;citation_year=2007;citation_pmid=17947423;citation_doi=10.1534/genetics.107.078717" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=R. D. Johnson;citation_author=M. Jasin;citation_title=Sister chromatid gene conversion is a prominent double-strand break repair pathway in mammalian cells;citation_pages=3398-3407;citation_volume=19;citation_year=2000;citation_issue=13;citation_pmid=10880452;citation_doi=10.1093/emboj/19.13.3398" />
<meta name="citation_reference" content="citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=E. B. Kabotyanski;citation_author=L. Gomelsky;citation_author=J.-O. Han;citation_author=D. B. Roth;citation_author=T. D. Stamato;citation_title=Double-strand break repair in Ku86- and XRCC4-deficient cells.;citation_volume=26;citation_year=1998;citation_pmid=9826756;citation_doi=10.1093/nar/26.23.5333" />
<meta name="citation_reference" content="citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Kidwell;citation_author=J. Kidwell;citation_author=J. Sved;citation_title=HYBRID DYSGENESIS IN DROSOPHILA MELANOGASTER: A SYNDROME OF ABERRANT TRAITS INCLUDING MUTATION, STERILITY AND MALE RECOMBINATION;citation_pages=813-833;citation_volume=86;citation_year=1977;citation_issue=4;citation_pmid=17248751" />
<meta name="citation_reference" content="citation_journal_title=Annual Review of Physiology;citation_journal_abbrev=Annual Review of Physiology;citation_author=C. M. Lake;citation_author=S. R. Hawley;citation_title=The molecular control of meiotic chromosomal behavior: events in early meiotic prophase in Drosophila oocytes.;citation_pages=425-451;citation_volume=74;citation_year=2012;citation_pmid=22335798;citation_doi=10.1146/annurev-physiol-020911-153342" />
<meta name="citation_reference" content="citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_year=2009" />
<meta name="citation_reference" content="citation_title=The Genome of Drosophila melanogaster;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=E. A. Manheim;citation_author=K. S. McKim;citation_title=The Synaptonemal complex component C(2)M regulates meiotic crossing over in Drosophila.;citation_pages=276-285;citation_volume=13;citation_year=2003;citation_issue=4;citation_pmid=12593793;citation_doi=10.1016/S0960-9822(03)00050-2" />
<meta name="citation_reference" content="citation_journal_title=Chromosome research :  an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology;citation_journal_abbrev=Chromosome Res;citation_author=E. Marcon;citation_author=T. Babak;citation_author=G. Chua;citation_author=T. Hughes;citation_author=P. Moens;citation_title=miRNA and piRNA localization in the male mammalian meiotic nucleus.;citation_pages=243-260;citation_volume=16;citation_year=2008;citation_issue=2;citation_pmid=18204908;citation_doi=10.1007/s10577-007-1190-6" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. Mehrotra;citation_author=K. S. McKim;citation_title=Temporal analysis of meiotic DNA double-strand break formation and repair in Drosophila females.;citation_pages=e200-e200;citation_volume=2;citation_year=2006;citation_issue=11;citation_pmid=17166055;citation_doi=10.1371/journal.pgen.0020200" />
<meta name="citation_reference" content="citation_title=A whole-chromosome analysis of meiotic recombination in Drosophila melanogaster;citation_pages=249-260;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. E. Miller;citation_author=K. R. Cook;citation_author=N. Kazemi;citation_author=C. B. Smith;citation_author=A. J. Cockrell;citation_title=Rare recombination events generate sequence diversity among balancer chromosomes in Drosophila melanogaster.;citation_volume=113;citation_year=2016;citation_pmid=26903656;citation_doi=10.1073/pnas.1601232113" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. E. Miller;citation_author=C. B. Smith;citation_author=N. Kazemi;citation_author=A. J. Cockrell;citation_author=A. V. Arvanitakis;citation_title=Whole-Genome Analysis of Individual Meiotic Events in Drosophila melanogaster Reveals That Noncrossover Gene Conversions Are Insensitive to Interference and the Centromere Effect.;citation_volume=203;citation_year=2016;citation_pmid=26944917;citation_doi=10.1534/genetics.115.186486" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=S. Page;citation_author=R. Hawley;citation_title=c(3)G encodes a Drosophila synaptonemal complex protein;citation_pages=3130-3143;citation_volume=15;citation_year=2001;citation_issue=23;citation_pmid=11731477;citation_doi=10.1101/gad.935001" />
<meta name="citation_reference" content="citation_journal_title=Fly (Austin);citation_author=S. L. Page;citation_author=R. J. Nielsen;citation_author=K. Teeter;citation_author=C. M. Lake;citation_author=S. Ong;citation_title=A germline clone screen for meiotic mutants in Drosophila melanogaster.;citation_pages=172-181;citation_volume=1;citation_year=2007;citation_pmid=18820465;citation_doi=10.4161/fly.4720" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=E. G. Pasyukova;citation_author=S. V. Nuzhdin;citation_title=Doc and copia instability in an isogenic Drosophila melanogaster stock.;citation_pages=302-306;citation_volume=240;citation_year=1993;citation_issue=2;citation_pmid=8394986;citation_doi=10.1007/BF00277071" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. Pearlman;citation_author=N. Tsao;citation_author=P. Moens;citation_title=Synaptonemal complexes from DNase-treated rat pachytene chromosomes contain (GT)n and LINE/SINE sequences.;citation_pages=865-872;citation_volume=130;citation_year=1992;citation_issue=4;citation_pmid=1582563" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=E. Rass;citation_author=A. Grabarz;citation_author=I. Plo;citation_author=J. Gautier;citation_author=P. Bertrand;citation_title=Role of Mre11 in chromosomal nonhomologous end joining in mammalian cells.;citation_pages=819-824;citation_volume=16;citation_year=2009;citation_issue=8;citation_pmid=19633668;citation_doi=10.1038/nsmb.1641" />
<meta name="citation_reference" content="citation_journal_title=JNCI Monographs;citation_journal_abbrev=J Natl Cancer Inst Monogr;citation_author=L. Sandler;citation_title=The meiotic mechanics of ring chromosomes in female Drosophila melanogaster.;citation_pages=243-273;citation_volume=18;citation_year=1965;citation_pmid=5863584" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=A. Schwacha;citation_author=N. Kleckner;citation_title=Interhomolog bias during meiotic recombination: meiotic functions promote a highly differentiated interhomolog-only pathway.;citation_pages=1123-1135;citation_volume=90;citation_year=1997;citation_issue=6;citation_pmid=9323140;citation_doi=10.1016/S0092-8674(00)80378-5" />
<meta name="citation_reference" content="Smit, A. F. A., R. Hubley, and P. Green RepeatMasker. RepeatMasker Open-4.0. 2013–2015. http://www.repeatmasker.org." />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=H. Thorvaldsdóttir;citation_author=J. T. Robinson;citation_author=J. P. Mesirov;citation_title=Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration.;citation_volume=14;citation_year=2013;citation_pmid=22517427;citation_doi=10.1093/bib/bbs017" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=C. Treiber;citation_author=S. Waddell;citation_title=Resolving the prevalence of somatic transposition in Drosophila.;citation_volume=6;citation_year=2017;citation_pmid=28742021;citation_doi=10.7554/eLife.28297" />
<meta name="citation_reference" content="citation_journal_abbrev=Epigenetics;citation_author=G. W. van der Heijden;citation_author=A. Bortvin;citation_title=Transient relaxation of transposon silencing at the onset of mammalian meiosis.;citation_pages=76-79;citation_volume=4;citation_year=2009;citation_issue=2;citation_pmid=19252417;citation_doi=10.4161/epi.4.2.7783" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=A. M. Villeneuve;citation_author=K. J. Hillers;citation_title=Whence meiosis?;citation_pages=647-650;citation_volume=106;citation_year=2001;citation_issue=6;citation_pmid=11572770;citation_doi=10.1016/S0092-8674(01)00500-1" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=H. Wang;citation_author=A. Perrault;citation_author=Y. Takeda;citation_author=W. Qin;citation_author=H. Wang;citation_title=Biochemical evidence for Ku-independent backup pathways of NHEJ;citation_pages=5377-5388;citation_volume=31;citation_year=2003;citation_issue=18;citation_pmid=12954774;citation_doi=10.1093/nar/gkg728" />
<meta name="citation_reference" content="citation_journal_title=Cancer Research;citation_journal_abbrev=Cancer Res.;citation_author=H. Wang;citation_author=B. Rosidi;citation_author=R. Perrault;citation_author=M. Wang;citation_author=L. Zhang;citation_title=DNA Ligase III as a Candidate Component of Backup Pathways of Nonhomologous End Joining;citation_pages=4020-4030;citation_volume=65;citation_year=2005;citation_issue=10;citation_pmid=15899791;citation_doi=10.1158/0008-5472.CAN-04-3055" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Watanabe;citation_author=A. Takahashi;citation_author=M. Itoh;citation_author=T. Takano-Shimizu;citation_title=Molecular spectrum of spontaneous de novo mutations in male and female germline cells of Drosophila melanogaster.;citation_volume=181;citation_year=2009;citation_pmid=19114461;citation_doi=10.1534/genetics.108.093385" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=K. Ye;citation_author=M. H. Schulz;citation_author=Q. Long;citation_author=R. Apweiler;citation_author=Z. Ning;citation_title=Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads.;citation_volume=25;citation_year=2009;citation_pmid=19561018;citation_doi=10.1093/bioinformatics/btp394" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Zalevsky;citation_author=A. MacQueen;citation_author=J. Duffy;citation_author=K. Kemphues;citation_author=A. Villeneuve;citation_title=Crossing Over During Caenorhabditis elegans Meiosis Requires a Conserved MutS-Based Pathway That Is Partially Dispensable in Budding Yeast;citation_pages=1271-1283;citation_volume=153;citation_year=1999;citation_issue=3;citation_pmid=10545458" />
<meta name="citation_reference" content="citation_journal_title=Csh Perspect Biol;citation_title=Recombination, Pairing, and Synapsis of Homologs during Meiosis.;citation_volume=7;citation_year=2015" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Synaptonemal Complex-Deficient Drosophila melanogaster Females Exhibit Rare DSB Repair Events, Recurrent Copy-Number Variation, and an Increased Rate of de Novo Transposable Element Movement" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
<meta name="twitter:description" content="Genetic stability depends on the maintenance of a variety of chromosome structures and the precise repair of DNA breaks. During meiosis, programmed double-strand breaks (DSBs) made in prophase I are normally repaired as gene conversions or crossovers. DSBs can also be made by other mechanisms, such as the movement of transposable elements (TEs), which must also be resolved. Incorrect repair of these DNA lesions can lead to mutations, copy-number changes, translocations, and/or aneuploid gametes. In Drosophila melanogaster , as in most organisms, meiotic DSB repair occurs in the presence of a rapidly evolving multiprotein structure called the synaptonemal complex (SC). Here, whole-genome sequencing is used to investigate the fate of meiotic DSBs in D. melanogaster mutant females lacking functional SC, to assay for de novo CNV formation, and to examine the role of the SC in transposable element movement in flies. The data indicate that, in the absence of SC, copy-number variation still occurs and meiotic DSB repair by gene conversion occurs infrequently. Remarkably, an 856-kilobase de novo CNV was observed in two unrelated individuals of different genetic backgrounds and was identical to a CNV recovered in a previous wild-type study, suggesting that recurrent formation of large CNVs occurs in Drosophila. In addition, the rate of novel TE insertion was markedly higher than wild type in one of two SC mutants tested, suggesting that SC proteins may contribute to the regulation of TE movement and insertion in the genome. Overall, this study provides novel insight into the role that the SC plays in genome stability and provides clues as to why the sequence, but not structure, of SC proteins is rapidly evolving." />
<meta name="og-title" property="og:title" content="Synaptonemal Complex-Deficient Drosophila melanogaster Females Exhibit Rare DSB Repair Events, Recurrent Copy-Number Variation, and an Increased Rate of de Novo Transposable Element Movement" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/2/525" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genetic stability depends on the maintenance of a variety of chromosome structures and the precise repair of DNA breaks. During meiosis, programmed double-strand breaks (DSBs) made in prophase I are normally repaired as gene conversions or crossovers. DSBs can also be made by other mechanisms, such as the movement of transposable elements (TEs), which must also be resolved. Incorrect repair of these DNA lesions can lead to mutations, copy-number changes, translocations, and/or aneuploid gametes. In Drosophila melanogaster , as in most organisms, meiotic DSB repair occurs in the presence of a rapidly evolving multiprotein structure called the synaptonemal complex (SC). Here, whole-genome sequencing is used to investigate the fate of meiotic DSBs in D. melanogaster mutant females lacking functional SC, to assay for de novo CNV formation, and to examine the role of the SC in transposable element movement in flies. The data indicate that, in the absence of SC, copy-number variation still occurs and meiotic DSB repair by gene conversion occurs infrequently. Remarkably, an 856-kilobase de novo CNV was observed in two unrelated individuals of different genetic backgrounds and was identical to a CNV recovered in a previous wild-type study, suggesting that recurrent formation of large CNVs occurs in Drosophila. In addition, the rate of novel TE insertion was markedly higher than wild type in one of two SC mutants tested, suggesting that SC proteins may contribute to the regulation of TE movement and insertion in the genome. Overall, this study provides novel insight into the role that the SC plays in genome stability and provides clues as to why the sequence, but not structure, of SC proteins is rapidly evolving." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
    <title>Synaptonemal Complex-Deficient Drosophila melanogaster Females Exhibit Rare DSB Repair Events, Recurrent Copy-Number Variation, and an Increased Rate of de Novo Transposable Element Movement | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"SZFguDvKBiejtLkY4ocD36dN_b014niS6af-0mSiTYg","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495552","apath":"\/ggg\/10\/2\/525.atom","pisa":"ggg;10\/2\/525","ac":{"\/ggg\/10\/2\/525.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/2\/525.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/2\/525"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/2\/525"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/2\/525":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wUhjgkE9txO0YsP9c3S33CIOTRN3eIfHhnYviF+8oGoPgyzr0\\\\\\\/BGsTyrhqg2kxwXwtN3j8fa+j9bVQfGucSx5bPPkC6HzzIKo2EnG+LRl4xlOsYa0B44DUKzk4MiGsLrNoSW+JLmNn4y+xV27rHYHkQ1zCu5zzAwL6L7GNOviqymSrfSoShgiVXiZZvSISpWgdqPsq+kp0fZQnZQt2\\\\\\\/5SLFgmSxrAtJyQprhCOLlrx63p20zRpC+qOdrwzChUVs39L0RQ4q9ET5qhrsgjKXouRj0euxBQU80Nqmil+7oA9i7Ml4zWNbLPB6S\\\\\\\/fjQqj+4WuQTtfreqF\\\\\\\/JQlEvMwA1PYnFJTLUCTy8ffw6AQmUvypakmZbl3cQwsDBK+tRMVya\\\\\\\/3V3XMNgb0zP6jG5ID4Z97F0lg09ko6eQ21jPOaDtY3mdHhTd0YMwwIt7N1oUP7iTh7DWVUq1leAPtTmqHZB2AP6ZsnXhZnw8sheDtTpxD38Ko4EL6BrMC7Nip+hqGPRgZqo5PGCs\\\\\\\/pjwsKNmHoEPMRXjEq6T5uWhuf32D8FiJY8PneQehnH7116MFlncB2cnR6RV1ZhBEkiCIhFAFi3IPlLXLnZyXsAYCYlqsaKtpNXUMJzTKk77gAVio0\\\\\\\/dqjtCl2Cj49hrDveCdJeVMf4qs3DBV70QBCdOrMvjy2ygjlzHm31QdSeAl+cnOa+aRUe\\\\\\\/E2NfRCxpNfY2SkakYaTvdyyABSORoEaSjS4Yn5uddmbEMj1SJlTnk5XZJ3rv+pghqUjmIYX8E8fvcAOwbkoXRGIweNHPbmjc+7k3hZ5M+B8pvfeUCrwgzkNvnE8BfqY65vdp\\\\\\\/vdynK0vvzuh\\\\\\\/xdH4k1nS5t4o3TQFYGgo7rJAo1Ihs\\\\\\\/UBBEprcX1eJdIMI+i0KWsdF6tSHEB\\\\\\\/fAAXUfwJ3W3XUNYS3U3YJP3UKNMDD2v9ZiYiv4g5WYS76VOo4H5a55SrmhpKRSn0Y24J+hy0JhQzyUeT3byLDVWtBU+N+g1kCJ2uApPpy5kosI3u8m1bxRZFRgoQEitrh9Q4nY6P+wPTdLIVSIZbLK6TIzOAtn4IBQg0uIvaEg2jCWZgFTHNZBTRoPVa0\\\\\\\/tpSc6c+6fkQ96RYQblA72Gfd5gKP7hN4+27R78\\\\\\\/4FTLMyIuPYZIj5ttm25EcNksK5x3oSdyTULAVJ\\\\\\\/iZXaR4n3Rhz7BECjYh8qw8i9bsYSWkdlSaun6GAemY2VJFtmR3hBy4cQgeAlC7MxQue4WCD+0KKeQguRBAFuf7s8XGnyv6k4hLjQiY\\\u0022,\\\u0022iv\\\u0022:\\\u00224Vxf6ep2+hQ0mnqVPiOwuA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b8fb51d40712337e9a9a09cc481f5c2408cace46925cce0a2c4944fb521dbbda\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qZQM8ejdqJ5WkCBq3s+zcxluR9u4+SyarZ1\\\\\\\/glSpQ4IN47BcrfSIzTti+CgUekDuvzegYTa9TVGxccvuj4hr9F96vBrQKjGvJDnvB7oux77wA3u8a1T5RXJl7TbhuMq58MbcAee\\\\\\\/b7y09GvTFDYZwZb7YKbq87VHuKAPCNWjnwPiWghA8j2iM6ztJwrTym66Jl7q9Nma6O35r++3Tf+totL9+xTI5pURVn7hgYhmXmBqSOAFpHlr3rikIaCleQliKDgQHwW0kr8BelT1+o7GVs\\\\\\\/dqOURc2iEz1kXC1ojCnglmKp6v\\\\\\\/RGSUO0\\\\\\\/KLA4UAw3ovx8tUAgdiL0XYWeIUG2RPCbLWoMYXgP2UPs5v8uqpeJBT8I261OTGIfaPxBLjh2sgN\\\\\\\/Fxs66OZoE9JHBS1yK0ENAbCUrGtl1pFBMbVsrqvMec4u89BayKIm6AnSITGM7itjFNyK6DA1vfEry2yQaZznNF8LAwbwjLbkRHklKjZSzGwYcTR7wxRHyklugKvfdDpAp9ZsDtcTmfxuBC7ruoGRCcdjzeW6GahHqmHNOCxRTEdaImkSmWfGj45l4KXqnP1qHnET0DBFRPeokmO0NzHB87si7rwl2lSuoYd63pPvTW3gOG3Elp9uiR6A9jvDDFL5Xc+Y1FkWKdSVvLJp9Nj7BVSO4QAGPARxB0fP3QHe9iWzOOfQdHdC8hK3l0a4mTPXRJjBmqX8v4enS1\\\\\\\/uc9HTvyZ\\\\\\\/DtQT\\\\\\\/piA88b+7rRIae9B7BWIX2FT1xU9io7X5ZrAgX+\\\\\\\/ZmqjOqoKBSZh1BdB+WA4DI4LDwTTWddlDFuqTAfjnbc5p1bL8Dt\\\\\\\/OgCaXUnbNjYGEIPFnCtSqTa49ekxim4B2YoNAtGilo6+eyQApBxqrpX5YTx02F6bITWU3RM7HEqyawKVof\\\\\\\/kKX5FGzgdJZnHldJplx6C24mK2Jbv\\\\\\\/c3VrHgXsFMhUrvU8YK3gonSZ9br3OUxQ3b4hK0U97Kp\\\\\\\/WhlVbrHAO4cVV\\\\\\\/CUfW\\\\\\\/iDV3Q\\\\\\\/ljv7JAj4XJ6LR6\\\\\\\/K31p0Hk7knfbV\\\\\\\/YPVCs3p2BDhKtHwjIE+6m68xJkFVxZACkMeBe3ufcvGHNIgbPeycHGAgORVxFa08yPPC0TCFJzBcvnyD4K4NYaXDOtoqNRCfTfjdmDILjrHG8PDjo7oDdjxViWv77plLU1DgfIEDVQrxsrOeneIHTdlhUuWBmHqUxPPD4R0KQ8UjApiEK8XY0IgFRwgX\\\\\\\/N\\\\\\\/i2v8mjFcFGAzw5WJuoXHYTtWYnZIcOwv9\\\\\\\/2sij1uNjbQNQGqYcSvAVusQSsBISFobDhBlt2IrRHjPCq1EQQwvKgO59RarobouavfvJT6IdgZKv1Hx77goesJCajzduQisAslSoMmlFL5\\\\\\\/69FUlWzNgB6OZ2q6FmHZ9ESjgwM4UX95urYhZqW2jYEyNWcXvjhjb3Ub6xS9DNTP+GbVCwdBxeC\\\\\\\/PeZV98+rCF5j8MoSRQMF5Ij7BC\\\\\\\/GaZZhswF9068fdeigIV0WxgMLWIOW+doZe4GvbMjDSeb9QTJmrafux2sdEwkmSDtLGhzZDRqttsbClzc2AZ5FTZ8sjf8pcIc0vLtZj+1FIfXi3aY6XauFLYHPD7E5pfahKErDLCosaVGTplAYEgrJWAczR0JjGXPrGk2UOm5y70Z\\\\\\\/QXazTvWwIiGa8rNYrVcjHDuFQ6kYOsOw5jFZtHjruOJBq\\\\\\\/5UPjEWhDGFeI6\\\\\\\/BjEZiCYh5TDd0TvwwEq9cmXVHfOfsSVb2HOwQTvElQgsTdIx2p2orazWctzYxCss0bsZ\\\\\\\/toUSmbjtt5SoNFfOCluz+GhGA6WUQWyGEdTf0CE34c3ZLIFov3lXW6WHVejKXOXo42ySh72fAfE4JX+AC4CCkM9T5sFuX3Ro+3tCWbfvSVO3Cn2Yc8r+Hsz668pFNMwUJX8gpeJPlaTXg\\\\\\\/nUXff86\\\\\\\/8vg5wTO8ZCsoKSN5IMa5Ks3zw2qx08VvIoI+8e+8huCojlOIVRMJnHB6\\\\\\\/dXthPCMpNrL9BpRm3cdZXDnzjsyI8edwANQr4Pw40jXKUpWMJd+5f4Z8ThM8NtR1CQMAYqiXtqNx9vNKBC7A8YO2ZCDJXTJB9V58lAQPqzx6oNbCuprNqmDN1RRPDEgV4KlJny46vXKoAb794qYZa+sw8m+d2znoz9PiHIetuE+44ShUThg\\\\\\\/q5zoGBar9w8R\\\\\\\/BA7eDfrtRIrqdPKrbgCrlQzQ24sXiciQQ3lUIGVU\\\\\\\/KrTnZQ0EOy85eIOdIpRTw9BnMTOrIeKRMy1UzqdZuprYiBWBl8PiH2LpCCMz+R3wf6xhoF1qivgxHXRgi6ebu3hu3+MGqLwOo1uM3Am8zQKubAuofGnGxufF42qijOTvhcpp+GksTyN8p8Kc9YRBEieHONdQZzXBs7wJXgGy7GRjGgxAdJT1+8XeUiOMh6r0Aygzh7bWAy+BL8Q83fnRQ4i\\\\\\\/2jtu0mRDKjrrLwhrUmtFZVdFSqKG+XMiB8q8cL2rN9RHUi5RkKu8IOpik5Y3S6I9FI87NpBxR8Ph5UD+WGNFhx+DKGJ9GuLiw933CLToNI7AViUTBQjgvHG+uQqGow2s81wr5poYqDrs74eEonU6oSS0\\\\\\\/iLqZjIu8YP2qu8hD4H68tBI8AOPEj2lflWXiDuzTWbFeUw1c7gN6rYWW8XODliKU1SIsMNlGqCn9cTFOBFEUttC064+NPEi7qfh99ijwGEuJMSSKwNYdlwl9679bpk14cgsMqOwDlCVIqAJv+d1WhNDKN9TRSMQP8iX8NQ4cinq9tgwEdXQgFRCwyp83pN2O2j8NKDqARiVKl9vGuE5BbE\\\\\\\/GkGbIuREnYwpI8n7QDDVqbUU9CqkCn5naVe5NuQ+t7E5KYbcETCd\\\\\\\/\\\\\\\/fow6OetfW2PJV4iSZ\\\\\\\/M+B1oQMLiF6Fg9ZqD9jQR+QYV0UTMPQ+NfmFSmYYqD8xMvXJesvpSYDh978DZNDMY8rsyMxZFx67\\\\\\\/easDlaHUOAj4nCCk2e8eDUOUXDvfu3qMg3OXrhA7HYOKO67DbK7a8ORHVp3SAhyM8MzFV5edcqeBLoqFb9gApF8rNbtmTwMB\\\\\\\/Y7YPPEWYT\\\\\\\/oYiCNAwLVDkbTV+ezEYc75Fuqajo7SNXeGqIuFXHKxnem53W\\\\\\\/atNjTKj0wWWRCm7+hE3pQwdBpy2k1kyhPt9A0\\\\\\\/4lol1FLoIF1EUMNhEuunTJ3X\\\\\\\/\\\\\\\/2NjqXggG5NgfNPurNcwCbtx3EVv9a8IKzZeI3Xr1\\\\\\\/EI3rzEGlcTll5Y7d83eB+GMMz4p5MjAjaPI5LPoll+7urEFZLtuwzWt6merpOjymZwuJ\\\\\\\/VueIn23nHtm3GA+kjqwDMZ3iYnLzkecZWP9i3z6vlzBagu0alNpPaseH\\\\\\\/2I80mVCATWEYtSUoGzX0HW0cgMBiCP8SDoxNuWiBXkLEmxzoMu5Ty6LVbJwvk40iillN+QQ03hMnUnKVL\\\\\\\/9loEPwW4Cm3ckkM0nRdfM90VrQN69FYtNlj2JJ5Euc34Sylqwb+RS2Hm70dSjDNzknojMVgmfo1lQofud9PJyP95u9BvMZvTefg7FRZhwg4HC2LJ465Ur23MNWkcQj1mR5X37fEt\\\\\\\/uvv5u0MZ49jE3ofo9zFm2WqmGGLBUE7K+kUDbCDlpJsySbdF8sQPLjxU++BANAdMrnUOY23j6DX3zDN1fXJ2YkwA1+7N\\\\\\\/\\\\\\\/5HBwVF\\\\\\\/D6b\\\\\\\/YyDyeY4Sy8ASFA1UXn3erwvqkSLPVQyjV+pfwz9D9lxBBwvj9FDKG+3KposU4i\\\\\\\/hvXpVx3B5kGm4s5fIh1mntn+rOm4CGsBZ+nrcarBxlVOmmLLfpQcKFkdWfhvG9t6bkf1EtHqW0i\\\\\\\/ElGBcQizuVL41qy3VlGDRjCIjqS7CotS35bQLS48CL1jT0fmY7RxnsAbuqHKNE2JjuUKZ9tZ+EBoMYXqrGSWf+XaQvtcDKQ7PL\\\\\\\/8N4ZIjMZ0bJ1Ia7I2ZDl2sgTuV9Fy7g2jwFxyMtl3RGvINtd+eLiEAmStsVdSNcl7ywgMTdGW18gNxhKg7SLtq80Henur1eSsWxZF5jF+eboFPJ1QMKR6ebVgWkipjvo0SWZ2GJGxaUByOSXp40TeA1xmyLw0nxtZMtRD4mZhuzKPjbX8tN7SCTvZ2aKu7rJcCo8CA90nhr9CUMWKrTSUOuI9ngI+KXuB2BJWVqBNuAqWIyj617no26gG9Gym\\\\\\\/lS853BsWFGsFXlLdmA5wshBWvzDM5Bq9yLVQOaPp8ds5pexCjHCr9k1gGThFPpATLLOFG3TpDLtAXIosgadel4TJmGWRBjzgcY7n7NB9JdDEYdC6DvRqS8BN3W4UrhyDyLjixxXmCuMJK0SEa8Hk4HpqFeO7Jdx4qlJD0lwCkg9ZZktwGB0QfVf+ZG5nJKCMntRxsBfKgPUnMfDz8p1z4XuDhhEqjPMHTnRhpITX2+g0g7TPYqRDlzdaDb8DONEWMXrxIhw5kZI9KCXrP9CO8hidmxImWZMbe4P2vTTc6OcAsZ\\\\\\\/ece08PVIV6tQjKOSeZqECAyhHqDmYKFx9Qby8qVW5AD6FKO36khBBF2JBELL6y1dtpP7TjAGbS5PM4ZvgT+BuRjsW8iANe6y6er1Z6D9WmUiuH7eT5D4C3xj3G7HiAUxV+BC0corQ+HIDHtiX2e0uje1KygbVI4YR05yVYV\\\\\\\/nsyn59b\\\\\\\/7yKl03JAU9\\\\\\\/oeSwzz+q6KtjiEuQRxbot38CT10T1kOoYtiIIusIq6qtdN6yy\\\\\\\/6vg8siraaO+M3QOTbU6DTxQLYOOhqG+nBR8kwmX8ZT3Fds6MoLSmNSZZZL7c9NeyQXiSro8KP\\\\\\\/1kDimtCQxZSmxjBKjN1TFLewYKkr+7xTcaLstLx2xAfwIl5XPFw3PWShR40AMkOedlEItbAOOjAHg7\\\\\\\/gxaC6FuH8ZumJjVnF+3CGBXSUmWVbUPTs4zVy3A7CnsBR2zT20VHjed1ek40IbakxbjD4uzFc1nFoaWShN3RAwY1dTGCFcyqvJZyuxHoN6P4i0TEQiFeVFckxHW4jsptKW62FnFTWjvjDun5lFOxUTRDWin173s0wT4ECBOJNX2bNNfAwt7RPyF3Sz\\\\\\\/Xdgrp\\\\\\\/4taEiP3n71H39OkcjnVEGrObAynzJvhvs2tX1ReVsznW8ZEcaL6viWyM7Gh0j14PBhRbebs03QFL9G12S+MlyuvBDiIPCXZZBShITInolzdjgPgOn7488PrFCj3BxuRt3LfwJ8lY0zrGIRgD7JMrrHXMA4pAx\\\\\\\/6vp3054MmZvpLE8bhz\\\\\\\/nxlmcWYIDdKUDByA\\\\\\\/AtmSV6h4qD94xS8HH8VzVIUpmndmFpHQiZ8EbNhc2oxXERu9E3ExpSCLGcgYwSN4HDt8A+hZ8Xptr5Z7QIzsTSnsvMVWFk1cMXG4qqxQMNZtbEPB338v7QUnzwDHDSjHz05wjjFaxzlhWad1Ltzk57pJRVmj4Muq3+3F2haA9j3YGOUhnGyUTexmt2cRGb6V0VJi5h5xtZbm8karXoDvKIK\\\\\\\/JVJcDxjKGrSWAT2RnREfm5w5uxLILtqfAEpmutSB1TbIIx\\\\\\\/R64uGAK5GvhVXTPlSYO7upAKz8c24C4AemC+gzmhBvhQ8Z1riJwMQoBrpd65PCgMRF3MBAaVA5eyI4Pdq6YX9No6VJ+IajFk2sha38OxQz3lIW3cjLW\\\\\\\/3gd1oYSS4pebW\\\\\\\/Al0KWqAJUnbJPAqeTluSRVOko83BQv2d6oCOcOhIEBS8jn+fZHcVX7IY54C1kGek1PtqMxCzN4a32ubs+g5+UJuYFVgpScydANzY77cHGkjXCBOxsrY97ca4uTmlWEk9zSeAwdsITY6H+eelgfPOBdC2pddmfbey\\\\\\\/6i1XiXHA7V637MwFcJL9GGrD7o5YJi4r7Cc7osmAFGdsYfngiZWfRQg0raG9ZRlOpiTdwq1oEOtQrBXa4PzLJMb2cv8VJqgMP474FFSJEs8JnyAQNZab41ZhF2sFMBlcJepGrwMzgFDiRquSpIYGQOZ8W594mAySZi\\\\\\\/Ub1RithXtFFRaDQraPf8fbqJhtQHnWaU1eHQdFWn4escZBBYyG3gW5V2CbShX6D8KonXz33uCXbElypN7j\\\\\\\/hm9EibnOom6FvfJBVqqxUeTJAu87ktc1V+PLntmqOt5fvWAWyrmw2jUnE3YV2lVGnzEcDW5KYhBbjp6w3jaYWRSLYFxtVyqG9FdNfU9ntysVNJwBwe4frBM+J\\\\\\\/c2hPCyhYSwvgU4UTTU8a7jOxaa\\\\\\\/T1lnZmdoVtWlTOnogFR7ozFI7ACVJ6wM4iPulENs29Ucy3tXvUdHoB4RIv0obc+AvU74qukrknOMpYcUwnA5AnPoCpSiV7zaSnWAtxCdf0DczuMrAV2vkOZW9Rf5oqK2OhnRdzqqwECN8qp01IiPYoH4mg+sxf18Zla41aUvkJRQkjNcXKLX3G+FRyegSyULq6ch6Fql2U0mp2WkfWFjiznMxjunOw2szwCPBbbj6fOm1ahkYXzRjNAj+Y\\\\\\\/zuTSEzt3Uo2h9Y285J3Jv7yFmrNZOrKrOnTq8S4ynBmQNTHeqi2nmfLHwd6mA\\\\\\\/UTn6IYoI6aT\\\\\\\/ZHaoMdwU8+xpPdBOPiQrA+pGS+MlEN\\\\\\\/Ra7Uzk7JAiXR5OdquEWAdvPecQQvKeeDhM+dJinDs37iGO5pmMGCx2Sfh6zA7xbzhkMK+iM5O6eJCAmLZunYdw0xPUtSG353T119Trj5UEA8hKxkVmDD3z8GkaA6QNDNJjIzbKAvKSTb4CkJQTW6pIzE\\\\\\\/3pACJq9t6D7w+5PsmTAjIeEsn+JPM4CirmXkXEfIVFJhlxzzzl2D4yYXyf+lLPKcSHat1Mm2xbpWZ62QVZrFTJNDPBxet7UkNY3MnT3Z74hoQ2jVwaT30z0ibWBQlhNfGlUNTHIk0BLIMdtP3ndfQXMrJWr\\\\\\\/sZ34T2ac6PQLVKQ1XAMrTGNzvM8Any3zczdZQpE9eWp\\\\\\\/tV3Rs0eBatBZhc9DbED8++BP\\\\\\\/cw370RmmCSdwI3r96inPhsiXoN4S1eylLOJ0Ge0KnTRgRFGE7Fk\\\\\\\/0RvJoIxJCPzKdygbpvUa5kf2pLpLXJwqphiwBznvvLiG+vwtACybvxiwBN9y45irzRGNw+ucu4CW581blSgQJugGR5lxcOd6uA4uulsSeWoyuUdse9AyBob91iylCXx62PVfxRWngTzXEUCiqh88\\\\\\\/Ejeeq0alQEx5Fx7HkZadYIH0uKxPdLzsp4FdcDnC0JCXbCScD643DrAoqgjaRwjabwdHLuf7V28DtPIfN5mDhkZIuEPpyGusGILrtvt5mblua9KloMTG\\\\\\\/NpFCKKx4uohVNFwXkT6Hx5T4TJB3BdZQgwVzOvOFcArNVKQHhSAC7oKlu6JAu93oui7rt2SPIJxB+lZBVH1ymFLyvgu5E7OS7VAYQ6Jkmnlf1AAakkHSUrPlV38vVrs8xMMrDhctk8i+1qGjyDDow+l7\\\\\\\/hboG9j882HZmKZHD9C4TyXgvP2Q+PDafWlkFUJWGYzyi7R\\\\\\\/ijA0GrZhNuNP2tY7kvXqHiOXeb162lXm4cYaPZ9Wc6kGsQi6BAjzBHhvl9Gp47\\\\\\\/fxUe8MAdGvzGttCkVLnPL\\\\\\\/N4e5+uQNtxZVC8EUN+wucdl9O1g7aXS5ihCUsukdUrYGU2De3MoCYXv6rYbCZksgJmECBwymLwLPhJgwViNMCIVTc1mqYFSdf2XGq7cMFV1bJfZDsaScnpvqYWBRmqz9CzVSflRHbLKfuEa7sgVqbhFUbi94sgG4a+WeOMDqaa45\\\\\\\/x+HDAEx2z9RRlvwN35cNLm7zqHyltb0Jy2gfi8cr2i1z\\\\\\\/B3sp3jpb+TynoeULDh1BgcmkW3VhdipDQmZONKaFbePJPjqlJ9XhS5ZjKqrakYvd3VrxDIgoYwJt9fkggLRVL9lNks7qru1Qghq9wvVCjzx1S97YNxeWQEQXmaorjjUbUxypeSoaOMKWboK4ntiE6nCn4TuNIGMBTxrDWzSwZzkC0VFCPKiwhc5bJoSJGMqoml9XeV65qyXB6m9cb+t1i9fVecwW4+KcTULwMnmFdWhL8MdwmN9RAXEdUT1U2YoJMR40+GL2H8SfDezrpsLpVehENBJOlAwmjs0p0gN9s92L6fa9l0coTF7GvH2O8se6DdMeEKTgndryesx+AwAsYSfBY4amIKMZ7nJjvag4xemLg3s31N3\\\\\\\/qPHaq4cH\\\\\\\/0KEJbCP+rRiyXtlvTherkejxvVOeGXB\\\\\\\/Y4e1GkgY7jhv6J7ckOhnUUrr46veWRfRHGl3jaGr31qCHJC8K+SeY9G9nmZf\\\\\\\/AjRqD5Sn\\\\\\\/e\\\\\\\/m5X3T6epjABi8v8S2SfP0yk0R912zedA14jAFPFn9FU6lIh5dCL2KAy157c0n\\\\\\\/thMdjE0Vr17nzht6mRNbjKokXhWqjSh\\\\\\\/3bQ+V15TN6y\\\\\\\/gYGo\\\\\\\/wZEnDR2PtFcse5Db+ixcVHHhPG+ecLVdpHgegPPzETa85LNKnbRJBll0nvSBuU\\\\\\\/ZYpFor+jNxeXeJvWFskooohegM8hPUi+y8MzYtVEsgjaM6IWGsChfQ7s8NK\\\\\\\/xCqfJGsPg7bP8aJhZwuTdFkFtqW3yKxAh4VAmtHD+KKbl9td6Qxx2a7fw7x+BtfxDMdapdq0mldN6gsZbUd\\\\\\\/HMg4mOVXRubSRSiFKJDgxFX1suzr3Cq41ULIEFFEYXFORozZUsRXtxxGCXtp2Q1rzGmNR7F78IMsq6e6PFhKed6oWkQs3AfFBBX\\\\\\\/4ADh2D3dgLHeebtI1s0oXvlRsNwBni+8im3SO68vKd0WJQxcbq9B6Vn\\\\\\\/wBiuMCBeqiHbHPP2PCc1evhAST6y0l4QH7Y0JDOgwZ\\\\\\\/kIwCi191if5Z8ZfP1x+8pGGJQW6WNdRfFkECxcFScs\\\\\\\/8+h6n1MaoxPxjp9EUbvCvfuhXlATY8eZKHm3afji9amE0q2bM1gUj8Xt7z0zB+6ynPw6jKF+HMLjmj1ri2TjUwN75SYKr1Hsm1HK9sdSNbk3CA6uKz3cf\\\\\\\/bp4A667rLhpUU+nshoYENjQFshJQaQt2fVU9VwVzgGRMBFe\\\\\\\/cbdT1xmWi0oB0nEjEMyFvkwO3mbHR819FW6z2hchTQm3K6oSYesw34ONLZNqVe5D4QRMDrc9dbCIUyOJEg7D\\\\\\\/Rkg1xTj4D9KYADCf1FqRKZ6xKSr5g48tDou768fjn5kqxyr3CB\\\\\\\/B8XMhVr\\\\\\\/DjuUHTufubV47bwYnOU9YDUp8MuFvON6Q5cLjEFqbh4VHhss3ntFjI0C6LE9Y3I30hDBKAMAGcKx8lAm1RVoSpQ7SPTwldT6EDF6YD0Eanu92GZGb6l2miGBlOtFHBHxNAyfKZTs2Km6rIJeRAnD8G4hPc3VCL8EvrzZ7LL0Usk4ofOpHBI0JzJbf8\\\\\\\/FJlhZql+DVI5fDwdGlV\\\\\\\/6jCb78mJKyndDSy2\\\\\\\/NHXT2jdQob2PMRwEvDvaGQ5Iaj0KJ8btJRXpk4EQRBEnS7b5oILxEuu9rkLI3HZm1tMstMfHwh1xtweMZnKd4lIkQo20mVbUixAiGJAToPdV2y2kNg01Cf1UakqujkJKabS1AR14yEh+uP83vX4Ik\\\\\\\/Pl1aJVpU4iHiZHlueFkUKmJN5WNKq4xEud8vykAjCIMhZIAh0\\\\\\\/8a11UtY\\\\\\\/YX5KCiV1i2XNw2LSjiXPQepvfZKIbon\\\\\\\/Mjzu0UGKLIVxJg+3WZvrAr2E\\\\\\\/tZYNd7VIqxBt0+lMpHTA4Qjvy\\\\\\\/6MbcugM4ZXMlQNSpiV0RMLLYdt1mtZID1yCWw0DfUn0E+zJTrxs68qR6DACj9UTyVVNBEo4udO\\\\\\\/84KE9HHgT61V1FYzcFLgUdeBmLlpk7kKBKFczJZDAPvvMmKBqkyvGAn5MrR0NVHHMSNa2Pqw29XueQiaUxUSwDeR8sBPW0wZjdA8MBtygxnDuvs6voMWfXY6\\\\\\\/KNQLoNka4jGUmU5ZWOIaKdybGobNbgDHb48GCGHG2eenZkp\\\\\\\/edFA\\\\\\\/zlYP58CQGXADltZ3YpueP7jVCMGo1XLvM5WZwntkU9NguEiYRjI7n8T1crkG7WwMkCvKoS7nJUnI50U+XiEN4pay6TYLgGgW2nvZbytxXv+7+uQmoH5Fpok0nqKdhR9bFmFdz6cpM4B5w5vI0H+SnFfHhMyMGui7tHxdd1WZJbRTShtLsOcW3ZJF5TAJMNO0pheL\\\\\\\/wvX1sOVyrf9fVIfPKZY5L9vlYe7nKCNSg9JLbwU3gBFzmE4lHFK5VLip8kBsjZ9c8fzy12fZ0eR06lJjUgb9G\\\\\\\/vH3CA2uj09bGEuLUTRK9yIy\\\\\\\/VoFZAtL4f2l3A4MRK9Lipan+IoL6znK\\\\\\\/HK9U\\\\\\\/yrWA1QLQcVGzBpxb\\\\\\\/wHbgMJ6L+f\\\\\\\/VZ+Yuq4DztsSxh+oca04rzKBFtRbvxP3r+iEeCjRNYPTE9Kg96yQ9Si1v+F90gbI9kHjLu19HDxWbQGAVf5V+Wtce\\\\\\\/aLIf8xZuHFWq5rBnzhnKPB6HuGN04Qhuc7Z4hi6\\\\\\\/V\\\\\\\/HCn\\\\\\\/NGB8vbDvKq6y54L9COnA9+jZSwefN0\\\\\\\/iP0Nmx2PddxQ7MXrdG0Tc\\\\\\\/NkkalIzPOGcx5z5PHlSL8FkvfaDValpviGROCDC4B55rsFAIAEP9HAhyYBZVeVF9JB6X161\\\\\\\/hw\\\\\\\/LgApymMSabX3XLObve02GeTxWradYFCz\\\\\\\/G6Er3IBHMelGo3glAF7yW1OlP08EHLG5bJ6vAyMm6BRKtV+E6BkBWhR9lTV2wOzV34IljIW33xinKf1ZFGBLX9Z\\\\\\\/D\\\\\\\/8SNibsihVD363nhVxjVhF5VE9je\\\\\\\/ecf9auLOjpIzZgSYUoY+W0meZ44x10v1tN3byN7rCwgWOBTpKpQEfBWrMclyvc9ZP5iXKcL3IPvMuBdJWKkXmIhcqSYvRVH62NMCQWOSX5FVStD5w6dIF0UE\\\\\\\/3NhOZbKsj9XJdX2dSuKUwjXDkwo7ZPmWuJy9r+GH+DKmINJtoAB8lP6I+CAIbrkLwTdpmsfkuEhQqpNn+WFiIQE34MpE11mgJq1Rzfj3gqjHUthYYe1qRcWMGth2uUh7Xr7YcNqda3G1LJBaI1Df+VPPookeQmTmfndu9Jbq9fd\\\\\\\/QOqPUc1MINlCkrpAsaLWAvSR3AQJ4QL2VBZSZezQFaP5xyxTphGjDKN1FwOEOfoGJotr9K67EYsY0bYnVh60h26x2UitPYUmkWVMoNpEBUCYV8A8Xs4V6xCNGomtQYSUB1g3UU40N6wtKL8Jt2Edq8r0TcEJC+eoTSIPZbZClijbH87k9Dk+Qi\\\\\\\/PlUGEif98GalY0eU\\\\\\\/7+75eTmJwIrzvR6bv7AzeO7nm3ffifaRy54P50ytvMCYKTy4h2ehkBVdMnfh46qTzWDIE8ItznwObX7d+32ik1MzuPLEHf4li4rDsIggwwjcSXNMy7LLdVJRnbixV2Zf6dUZnrnBSX3LPTA8K8Ut3ccDU84da7ZlvAkWcMRHZSNXl0I0QdwPJZVCrNJD31PeYK80J2glysnSQv5yiyAK3DDsN2ut4wS1nDRaIrp5z9wSzvYvCSLOfTmmnVt7g5uZVZANWzrPizfyJUnDFKtvnXl07GGctphuFFweRgQLC0pkq+SiGe+NatR5Ypn0CaxdDbf1X9uM54f6Gj2TWKRpfXuJl6DdztqWERaayGmtFA2D2SEK6GaxUrIC3CKdHD2M5K0zdW9B8EQJIXtKmoptOnzCKN14qjRu9PL+pWm4wHYdkYIJ9puuCu\\\\\\\/VcTL6Y1RUayEX5j\\\\\\\/FX\\\\\\\/8oNQ0xbAjqjxSkkUorV\\\\\\\/D4FiYobub1nmU\\\\\\\/fXTNUjhhXPSbWFt\\\\\\\/+t3kzl4pHjpq7Ip1UAngwqXXAiYuwTePtZmHUslMSXwsTvEf+dsSStXt5xpXr3ZU4NoyjdCVj6eJpYktBtml7htV3LroaZAZFXXP0u5\\\\\\\/t3V+fyFXMgXaryab54pMuMvFy1sJ16P4sN6J3EPNmsSLUrdkYSpaX1wuyaUkuAGZPmSpaYgilNjf2BfELaFhjByW2AfOjgOSie9cXjp+oDn0EFDLiH\\\\\\\/udkmvKF47haLuFOIMLhAUNDvN\\\\\\\/OVA8qw0OQStQhzRKCE8lG0y5p0h255TXFmrTwC0Rk8nyezFPvU+sHrN4R4e+5yAmhQxn7JjnhtzSz8aPaDuV+64kOcoW6TW8t9AgG\\\\\\\/hXxvkUVCcvcDOHrrRfxlbvrN4yYBXv\\\\\\\/AJZUQLQPBT5IEymhIQTshoeLl6ksbUEgtd61rbP9MiqbNnbdRZlB8+vTfh7H+O42qBDmGnaGAwLnqnD2dLnP\\\\\\\/1hO4cFBk9cHx3vu0KjMITRP\\\\\\\/u6HrGbOf5L0sojFsHraykIvWGQuX7zpEmYUzU5cQk2M3IeexykjC\\\\\\\/s5BPOSOsUgmLGYc7eYkcrSufwCDv1s3ibStBEACzOqWGCgAob28IYdfU1pLN7OcY7ihs5fT0gc9+S2wl1d3DX67Bf9znpYW3SQeP+B2QrCl3ldY76UKgfqXn84gPoGnblYgxWOxMTwXXQ46rQeJAlptsppHL4ajlj\\\\\\\/dBh4bKVfK4Ilgj419qroJeh1jtHB7onrnq+wKHXQYELH4IxmxkoX\\\\\\\/XZj+BmFY+0v5HFUW7JBbvBOTQA0FqTiMGoff3lAwhyWA9gSTcTL\\\\\\\/eTww13tY8FGpOZbkMFDO4VFNsx0aaalJaq0UcU4fqhdKrKngg8ROwkjL8HZaCN8LqdNFvkTb3QXXz79hFmI4ROMKwqZxU7J2HTOmpheKUUZtPvJoOH9g\\\\\\\/RqRGypnMgpQFFwlvrmjFlD28jCaANq7bRbCxfL9QVFJ28f4K4ZkZNb7SaztLV8PKQz7BFJybFdmbSWrd5TW8hSRjPg3wZWAksGFiQuAMxDkuAPCk+7HrYG0RIEOambG5XfxEH04K5YtG6CwvTryz\\\\\\\/EBj\\\\\\\/dCKhAg1RMZuCoSfn91ZjRzzfZTYsFW8vFjSvlrYF595tVhmL1OzPyKS\\\\\\\/za1kFjXf6\\\\\\\/DdXBVy5Xf2nzmD8GeWQ3oCaGM8uyMeaP+To965R4LJ+rl3pan+Z3QworAKsaMGGA7l14vCzOMKESdNokAwzaA+qZRQRHz\\\\\\\/Wn2+jxalEQzM9a7JgUytnBJFWT5COOmDYRJewkwy36lqvg4T4yYsli2oTHBb6Xbn8CF6qAZnemKOxnXJP91WEnhV0Uz5VkU0Pj98BNlaeiMB\\\\\\\/J\\\\\\\/iLkAaecG42v7DpJTLU4A21fEDghqWT5j5iaN9L35m6FSlAmGAvXjUwbluKboZeZUAPGwd0tE8iuaoE2LOIhclVUpNIa1SX+XjREMGVxph1GqnBs24cGkzPgaODcfBAlbRCYbOe7BUuBn8bVgu8g5tUOqgu48I6pyjBwTyExPNaPxlXxSvryRgTbwd0Rj+DwS988mZhM8dyerBIrEuFt4oFA888c80OkBCSFTI7UJzbiyHR7FJUFMCHLa5K0wWAIYxZEfaR1HHYPBQCCBSPWYJFy+r66ACIw5pgyrqy1TO2VXatkT5MR0HNWqAqvl7pQTArTx\\\\\\\/aI9hc0uvaD3kPPDHi2nLhfAXRqhckr5pxQ4dUxlrIoAnt39\\\\\\\/hHWfHvRo1HMPCIHkFpJE2UnxD3F0HhsMohhNBu2BtOQmXr6xDsG0qIbP8dhWI2AyTw9ksRPoPu0HTvqlKX6BA4OL22cEVfVz\\\\\\\/H3C64BuxMH+MvgoNH3zS1hTUqopBlKSoWcxfbYjXA6GRyKWndJlQc6OcI5XpGlWPCCETUDvHBS0ClaZ5zzi\\\\\\\/H+guFcohasYal2bjwCuiU3dzmIPv5sbMhcmBofRLaqCBQKHI95NjT6S5Q5RNEmVX1G8cWHFlUHc40yXdCjlDh+1VArEn1VxODaYOaDuXSz6iJfNa9kp2rymZGO82Zsexyaear830Mp5AJvgELeilnu9ai\\\\\\\/3PGtLkJ9\\\\\\\/ppVNM4++xK\\\\\\\/quzzx6BF12JDqSJQbhF344VgWKRECstjQyvdMNM8yxEulNaiO2WRdWX3EurqfMaTc9JeJUsSYX8cMh2DTJHZlsOsfPj3hicSiclB5y5lnK1tLmX\\\\\\\/aZN8uz\\\\\\\/GtSQ2CJoTmmbEJKF6gZRW0d+Gs22Yy9\\\\\\\/XAaU83ZAMQepSn09YblEubSGOhO7KISHJ55Tw07ibP5DUaBBDA\\\\\\\/VQEDr+Kdh95IcpFn\\\\\\\/Wyy4Bu2OKa9XwZ7Ljadde+7c9dtH\\\\\\\/d36TBkYwLuRoRC\\\\\\\/PKFtXDDUN8hL+Vct\\\\\\\/7MDJ3S2FeLCzIMKLzglUdL0xqgpKOnS0CTIc04WDuX24fWjqwNUNdXLNkz7QP\\\\\\\/fHmJEfqfBW69Zz5hTsID+9An58CidiCFpggk\\\\\\\/jkK0nO\\\\\\\/2ckHeageGINzuVpI6+jeEeLs75vTYEBXKNMES1dauk4H29v0tRKInsDMdMZdkTKRpg3LKPS\\\\\\\/nuXwf9K6XeeQCJNu\\\\\\\/hG1qo0zDJTPE3nt9B1tuSlTGu2EPCJeASG0Kf1lh5\\\\\\\/JPGi2JkyQVtVYLPIG+1AyfBzsqtjYaKkSN39eedMPwErI0v9Bj49WB0LoJ7zTvdBqPZbDateZbnCVvcNtDUKcWNfG9kB1+vqQ1HQ3p5vPogbObUYM\\\\\\\/1DJHyCriFF4UQszElp02DTfkczB+MyxjcI980ESftDi4uBFJJ1CiN+b1V5VEs3E4f5wT39OL6OZv6CIhNGZTbpJSQy\\\\\\\/Je1j4VFw83OmteL3zxf0EdSiLE61MhEUfOEDmbgeyx9jAKtZdYNv1ehMVvfts179M\\\\\\\/BvZSgLP23Xr5Ul+8tpIH6LaMTUpwQoB4gy4h+bUA9zIMXB6Gkz29O58RbSAM3h2Eo1ZAjo\\\\\\\/kt\\\\\\\/D8KnIJXjdVbWTxk+xQVyXSzOd\\\\\\\/ECBv1WUbMGdEGZq\\\\\\\/OKo2pYa\\\\\\\/uEzqa3fOfNmICP9kNOIXnlFSpMUWUIGDiLGI6a3KInVqI44x+qlFDwDXYYiQ1Y2G0\\\\\\\/lFXWr90eO5pguQzsK47KUYe9vYx3T+NYlZnweeBO3pXOiqCYqu2Insen7Oc4B8xlweiRTfxRb8NuWjg191QWIoYuNn2Yq5wGp9ldhN1kh80KwbtzQOQBv8hy1qAC+D95bJlIdxbmWvDf84RqXa14QkTiGOVcPT\\\\\\\/KAkYcet15Y8ZNOzpbpA2oyIIrndhc58w4yEM6AEfwSgqd0Ca58QLxNyv6fhduwhuMc+2dTlq0cZW3UChrxxQsolXiM\\\\\\\/3m6scYtyXHVeD6WquHvW9ZuYzdpOsqGZ3FrQoN\\\\\\\/ufesll6oHPZ5I7pE8R+n7pD7Vju16pKSejtXFLZxyB49hYmmoc3vwzqE5bdzxaq0wktNnpmTP3DuYB+uyuLs7\\\\\\\/1q5rMBq4PzQsiJy5WnhvlFdeUluo5rLAvcTNatCA0GOG4wA1Fi5FIhCT5LIBpr27zs4rricLIzrwCENBtgT7Fw4WL6OcLqHprOQUKVMbOO0GAJlUWnMGhBWGZBi9Y1z4B8wuW7MrMuJxI0cww4doI3svqhmbgvWrBwMzNxPREvVTsJYnJ0y8m5g1WWX9UmB5H+wE4w\\\\\\\/z9BllSQPMoQkb4\\\\\\\/enTjO25uXX2WjOx+XRUG5PEsvVCE+scK7\\\\\\\/xVTMLCRfo8RMD4sqqY96S4HTozmb\\\\\\\/oG6mtax2BJ2polh2zxLD54kCDeNuQh+B7yxe9BkP40nKNH+e3kVEKu1icTKyUeXod21QUvAEt48VLtpF5HNHrc4raQuN9ws\\\\\\\/cYP2Gtx0HY01ddmri4TmYOwSR1BSwyc2IoByecEAPgXmoJqlp3wu8HO2QryZcvLaO6jOJw8QtTUgwUiVt4+ghYw9FQtpXk4ao3lsyxW1pIEWOMTdei1V7FoP5sBkzco21L5Xgnxhf3yuZpEnFxgfwMsYp9p\\\\\\\/sGiOb+HC97aitDxMxuLDEFhIVY+mxqBoS4+LvhZFAl4ptjzu6Sl\\\\\\\/bNHDqQF65kI+DsmxlftV9vbpTKC4KxCeXz\\\\\\\/1sM\\\\\\\/fYmw8dBBknmgDQe\\\\\\\/EQdfr47Pm\\\\\\\/aWMvsq7jnOFVxOe4S0bDpt4l+9OBMD2rwbsW7w0DKUw3D3zIvzOmctAYNQbJG0k0Sd3BR\\\\\\\/XGFeEG9nbz4cDbWp0VBC7veQUAyYqzRxwQK\\\\\\\/22o3YBRYMdvt1cy18QjYU+VjVxnc87g88sowav77T7COCJi6Zfdsg0x9RF8xPIDRsc2iPU82mg65r0cREV7gMqao4sCtFcK5mg7hCEFThd5LkbL9O0UaQfEB832zTF3ywrPueXDKVpMdUqtxeBBkSBSRXxAJ3HaT4cddSlSAOe6bHCeronO9Nol7aRiJbFXz2eebXvx3OpqVdBIIAEN1RXT7iq8x4zb5Hov14vKz9w6xHhx5BEyfUOY844wIbXScKrFEUO9JTsAQiM4wH577rEJYlvzdPSUhWz9VfwUuVCBBfu0QzCcVEGUAaY+z3yEYvYuvSt3HJnORsWMtDyB5ZS8gAtHdkggqsMRyivGYfIQnSFkpUNZ17EQL4W70O2fPqEhuv19DJtgh5UQ1uTKKwLUPuKpc4DQlzT\\\\\\\/jA4c\\\\\\\/WGmucJKpziupj6ZzSVdzi\\\\\\\/pBCu+Vq2c6gCtXMwUDcoRW5jZmLE72I0dinsUUbluRQFXNPftOKJMPmgTezNILMd2yNlefHhoxDwWIgWGcKHhVavis7gsQscFa\\\\\\\/sp6loRFznlX7+19eXy1eR3\\\\\\\/5XcH3zTr7XtuuCf3dqXnkp2BAktUst4oq0gnA\\\\\\\/g4IeGtausm3GgZhrWjMO96jyQBeOVx29zbghopq2bVEYX904tyASBwQWqDm0fqRC5+v8UIT53tf1dlDOf9oUdFxlbrn88i1w4F1SvR\\\\\\\/zM3t5xxy8C5MTuaNoJhLndXx6b4tRsL+WO7CCJcK1iyu99PyCuKEGmqLXJPPZZkk5LIAgXHzSOONSeBPhqc8P\\\\\\\/ISkb6fS4i1I2LVTqeJMpPL+BXm1h+TrOwQLT1rb28XPSojsLVc8Ti+z7GshbRmkfSsxsgd4bGWuka23KScKgI7L7oZYm76UfO+UMO6sUUmEqRkd5euvlP4ye6SZtLU6hjyEJfiHD092ZBjRwUOHsyM3zivaQ5mxG41zNAL\\\\\\\/co1mcvdfrTo5ZpYvAV4NmEm\\\\\\\/2pJxcVXuVv0DP5REc+p8VKn8MSUixZZQahWDPRBYYc\\\\\\\/gw3GTNx7myHq99f08wALYaMhGXzpbeykANmQZQ8t4TOmOvY\\\\\\\/3iIL3WxaJj6w+y9BBj7Cl5kxfoRA6Fm6AuSaKmt\\\\\\\/CVr8eABvWqxXTca26SntKwVuilTVXbetxtyyuYbC57MfLaX3UhMxGilam9cR9kTcZtZHgkmWXAnCak1Uy9atycY5vi7HRaCmYtyvF24i4rkduBOmZDITsEceI+FHKggx9546LZJpio1YSCNLHeHX7g777JgyCGvS6wSEtrR8SHk\\\\\\\/7pkTWroBPFW3C2GCqTOi09l4fTDjfbegN+Be5tL\\\\\\\/a8EXmpQ\\\\\\\/HzsUqp4pqwVAEO3jv2zPw7inzBOxub4mvGg1T\\\\\\\/jye5Aa98v5jBwOPrzNDv7f0HhGp1O7aub0NUKMvr1Qv0ACqGW\\\\\\\/Jpt4F4uib942X32+I37rbwao3PwOl9TyUJH1E\\\\\\\/KNK4J7Ne3z1uG4k1HNOmU0kQ8uTTXVC+uniAJXktRRf4WhZJXJHY+QVaNheZLNGkLXcKBvYxqXfqJGjVJXzXdxaatJzGfgiS93dxj+mjZ7aD0Yhb8DEjjy2G+1ujeBqEf9PDd6BNaisuvJvufxzOJhJ2BjGPkYRJMwVpTVUGJApYRpH4NGRZ9T7n9zTgoO+yL+f3\\\\\\\/dTcPHdfiXTyVXna6IpvP+\\\\\\\/EETo9v9I3HGYeAKIxC4hy8ZbJFy1NTbUEkbg9QCtDvQT606QTJAEP6uC0rjLQXYHBhL35tzxStQWu\\\\\\\/0rerZ2Mi9NNvd35s4HDgmVxjsrRBNNpuCyw45Re4+iU+wxWy0b1x6nr0bGUdmsgrbQl8hKwRGTTxHAt2prbx1itWLtmKpM4rQVJiSgpU9VRo5KxeRO4QoRVe3fQsx0p56KBdtBvc2TzNBNYdfAO2pMqVe23rD7rN4Ayw3CPOCkSLdApNrp6HNsAeW8482yQGfEmjWr\\\\\\\/BKT7Rsy8prb6alE9Ml79W2WTYwvvJAmSCi3CNBPjSzLt38QrCj3HlXxiz57S5aDjDC8W4vlCVZ0UC0FOpWCmKrhCxxIEbVnDXavnCmK167MnIYWF3Pyf8CuBl7F0Sz35yK80q2d+XQzWJQZC4baquG2aAEad68c8WsJdlp3l+EsAUK\\\\\\\/Lu9pxopg9Y2Rx+QFq6DfCMyunSGH+VcqFshJLYS2BE8SZwZtI8aPuIxBrNr9psjH8rxnuKLKnyE6Ql1E9goRd31bHnpj2WAqHRMA0EyaDQz3QZgMcF1MrJ235T1JCkvQCfBbsqlzJFaE7fpZm\\\\\\\/x0NldtdWkZ\\\\\\\/CCHgDhL5xTZR8JNqvjCjvpZytZQpIcxktN0I2x03Lz3ddfSefrMbnLRJUieIDyMBiveVDMbZPwU3wTf\\\\\\\/WAo008PbdRT3SNdmHnKB0NdWq10k95B06x0\\\\\\\/EFqqI41DS+QCg5BLY3b+Sk6HwiCybIDBZ\\\\\\\/5qkYzxVFQL1c+trnNIxQ3z5GlKh682bryw7B904ZSEi7BnBOoZR46ROFPeNhDN0KbKZ+RIUVLlQaXKb7r\\\\\\\/u2hMWMkZVj3KDJqCmdfU6fpO+IhLsMhtZXgG1uXovdealF3ikFXGV1EyqGZBzjS9yjezSY2J8T7w7Na8wDPu8oMDCG3GqqaPM2DmBHlS+ynqBXtq7I0Zvs2Q+n6QXut2wKyAbAMQBXXQ3ssBeIa\\\\\\\/UjuHNrS+3IZfSBuhS\\\\\\\/sED\\\\\\\/9vP1ERdLUQsEL\\\\\\\/Pq+buyCBfro5OdtOcu6NbMrCAkcVRvpNHUPbZuzHRFceAHWU4LqsYfZZ9in\\\\\\\/Mc1BFW8WQ0f1saRbU4obOWAtT8bwOfCmHoWTHnR6QJnm\\\\\\\/wlfIwA93ehdQ6mlod4jXz9ZAgdNQEuQTJJVW4u3jg0AJw7WZxjfA2y6oQXldyTtGnr8cIll3+DP7g1Hz1BPkXFp8UTsrRq9hTAqYYOR8eLzSOwylr83eEBcBBZrKQv7mZu\\\\\\\/CyCjCvR46Bddb8OGYBJ2bQ12mzpyyl5RbkIq344J41r55mT\\\\\\\/lPJnoj8oKyl\\\\\\\/iZxVXNpsOmFewLKXeWX5QjLqiSzxwTSHjjPBSUWm1N2EGfCrWC1QqL3kFopd+DoiGtZrrJuMh+jUc1qVQRBYdk+YsVRcHLa6ua295gGXrxub1QvSElCLMD\\\\\\\/Am2SnvJNHopKsED4DvzhZoY5K1\\\\\\\/e9FGSP2\\\\\\\/iOD2sArbgzewcyRPlDh1HHAuvFP7uRWUfOvi3WpD2H29Xnido4ZcKDF0gNonn94W6pfrdoZhEM7FSMxnOJAdwJjwr5kDr5b5+jsCe3HRsw77ih8C9NY2Yz0sxhvst1gNUgaBEH\\\\\\\/aAjZ3aoVfkTYYG\\\\\\\/nusQceal7neu0Q8cqNUjUcUdXYLhx5ZlzBD3KgWGl+rhuIKTNeyMndxu3B230O1T8kTh8QXYau7bL0GQBhjU92wdpK9mJHkN5zCrFDqeKfAN1xJcoUIqY2RURLE4+c3Jv2DoUe3QK10kJXFj2fF17MKPE+KsAgEnL+3RiDr2D3CHS4XLesZr9rh+wg66ZYhXUcnmHnPEssx5z1uRQxl5ZEdb8GPhdI1Bi4xZbFuVDT\\\\\\\/D+Q5w2vwtw7Yn+guDObgKn2BJI5F68IVcqpIH3lA0go\\\\\\\/76ScVISQG5onqhOZcY9lML7gskO3sIN9v9EUgjdmT4Zffh9ZJEBA6fzWRrTKx0XWqfzhHhvYiijt6TOqP\\\\\\\/JJS28QXuViIeUTrs9umomEZu+WzmWYTs1xp\\\\\\\/Euiw\\\\\\\/APm6bwVpZWCspQtb7bHl5OQyVky2v+GQYuR70zMW5ZBSK039VuCTzyH559nc8rsztz6RCITJ00QzZZI3fwyAoygF+H9K9FNmD\\\\\\\/9ObnrXz7PmocrrASQRSvloNWsbwIiF9C5RBVfk8QJxyS2oaDPkCbI0jVPf5czvBtCb4JEdfRtwp82BeBABcfS4E\\\\\\\/yOTzolaBT8Dq0aWjwb89wgF4a9S3aUjxdLyCB+SKZZEwc6ipy\\\\\\\/6CbojXxQF3lYoxpBaY0kmqvdp8oPkUxkm6BnOHT9w9JDR0nCQQUTLXw9mE4DFrspIsV8ukxer5TSG\\\\\\\/LpiSqmcL3WiMHUAVf\\\\\\\/\\\\\\\/tiwQQQbm259oND\\\\\\\/rx\\\\\\\/A3OmBTre6iYWaBmWe25q\\\\\\\/xPE9MNQ2nDI6H2Cwsp83LjWStC7bCVdboq1ThPYqCRhBFotIizlPHP1iMPtqzTp5KGHomWr8NDzZJWhS0yEk1FGNhaqOo7Z6oN7gP8xC6w0vJ4hMk2qDtbG1qV+cr1S4M8XT+G30d2xSOEoRSVuRs5JzDnlgwbLHPozzQBhhpiW6QYZoidY6RlhhpXkRDqL0s0PH2YvcYxM8fQDw3Tq\\\\\\\/oiEwSUa7yfP25qxTt7D9D8vGSg0FOghxWHiltBh7Vs2tFr6HGnSiZ\\\\\\\/CcvwCZgcGfud9fTekE3DqgSLSW+0ehg28YqPVwdcehxx7VTn9VqF\\\\\\\/oSD1zcGwgu6VnqlN87fDLsCOevHWOXu1A\\\\\\\/YvE80cgmL4CFp+CPPD1x0gcuHLTgGYoNzwivslv2TJFPq1Stv86SYhk+3FminFfD0k8RRbRJVcT0G80n1OPE8YHYybp3FGeiJry1h3TIsKzzXCAtQGNCOtrSw6J6QdpQtnHsC1fOTLYqfP8aTTTwH\\\\\\\/aVzxeo3Xb46iE1PNZDMI0HOXhJV7876FJ67MZRo78Sq9b+Eu3cC0ymKbzeUg5adtBv6uTH1nn0l0EizF10nW87i25YFLKYs74n9sCNzd3xl9NWgmJ8dcXGvW2UEHDaW1IoEo7FUVzdSJ+87Teqb5zXPm7+biuceJpEHKA93SWoCZiYAFtwrH1AZM+I2ciK9SE1yWquoRKwZRU2dy8Q77pKU5ISNSiO0bqZVAddDJfz0tbQ5voOL6r4Zq\\\\\\\/ztOu6l+B9DAHrnc5pdy1Vch3AkRR0Kruxsrx7413nn6+Y\\\\\\\/UahtnxMYM6Qr0nM7gEHTncBeKjndC64iR8IW7jWtjH+Zt5xaxoflt1S03t8NHo1dPf1KB10KfQ0Vmn6XGKDt7KHRKl2Hfu83o14n8iovESF1PAZA7OvhNfzcJvce94gLnjfwjDxxaNGWmp55C9LEVULLvRmM4GMqamXFkt5mWQ76UsX+RC46IPnFYR57EnnaQilTA25lqVhNSbIFyuWbqgow\\\\\\\/XJA9NY0UT\\\\\\\/FRQj4kzI5kmWEu\\\\\\\/WJcDq\\\\\\\/bebDsRQwCd6ud9BpN57F\\\\\\\/p\\\\\\\/3fvD5zMy+\\\\\\\/k0hykXUJsI8\\\\\\\/O7wmASxfuXtgYDnPWXT7dCN68tG5cCyHDG1kKkHN2hrxclH8NJ5YMFjSLloz4g5NcnrxUJO0e5CIIrSWYA9zA8tfp7sS3ikMaB5JMqvJvOTgKbfaMZ19RlM26N6SkYWfAam0ap+lgXrJas61QFwLP5Kx\\\\\\\/QKtLTz3\\\\\\\/+VS2cSpOqcdOKWRnLEI\\\\\\\/jHiMitTi0iDWi2GgUEzwmWfirH5f08w9NI2FNRLxR9SjGUh3aZmMdPrMhGm+MkDORfErqeAaW\\\\\\\/joy5VWW61fk+4bmUCQgmJ5uuoVrephSdsctEUB392O70qvWL5DUShjelUWp2HfoWxAsa7jhwNN8TIVHr7tH1tmBCsALZJaFAyPIlGUdaVe349jJTfAjqNoa0ZaaYqE99GgmAziINx3AR7jlGpYQAWV+Gj7TtHDWMoVvGaA6t2bDrLZ4SdpjbSlgf2Cu9VKLVSemMOlAGppSL10DzO7yZzlAxBbwV2Nu72SYWy35G6SrLPNYuoszKEmPupJRvnYLZSAixSMHpmcrUoE2Pb0XSPDSkStDuxgjsZ+x0jrtW81R5rBOSDJPXV1neLl9SpRFkK4ul8CpVUbsyCjWC2+na7ohGikg5Q+9LON8dmlxXb0r+BQAJatpEKQq9oXkS2UXff9tX8tlXJCM8YTpiEJS6THkhRrmm6KAyCO292pOCK+i02IXyrDqfnsFxERpeLTAF+pGccPjWD\\\\\\\/lo\\\\\\\/h5jo\\\\\\\/S\\\\\\\/WLrWNAiNMYG0xzmPY1f59Gj82mi+37VNNhON2G5DD2itj\\\\\\\/4VIzSU0gWKZqj8E2wqRtdfLSRMLFpyyLIgGTQ6zK52uJAv4pQ+vk1t1A8+WkaCf0qPW1S0BwOI60UK6MlJxUp19mNbxvIkYcnyf8txDUz4xZSUO4BWTF5cP5LW6vCAIp4KoFR52TbniI+Zzx9mdwumQwkJ83vG1qYyNyc+ggqe30XOqJ5v7nCLJYvFVXIUSAHDOtrOBDf2CJswQc2bXc+Hn1OYenOk8Z21qghUnpumRD7rSZuVGndAr6wvTwyhjGRcgQB4rhndbheeDsc16kT7lm9X6OnCOERaJ+urNfvR9DlI9pHAaT4SVb\\\\\\\/qKgxwyQN4lL+fr\\\\\\\/4PA28j2EWj96u1jnJeYeTvlrymOd1KmJyKTde17OM0FkPYiDSUDHg1KF\\\\\\\/wbJFIOkQCOObmjpyAKu5zVvzi28cMy992ajJg4vL9AWLQqOkDN2kRgPrKd4NMZ6QrxCm2xvDZ\\\\\\\/+3la8XfkhB7JctAaUxyjZ8cOlkLtjYPlqloo8eI3WEFJD8mCWk7AIHQLEpR8uCilB+Uw8593Jej+uI73dAiELePKhzKB\\\\\\\/5t6BQXdGncT2CtXr6FEmybnvFt4qEAq9JlrXfDIPQP+2\\\\\\\/DQu7oHDH1zdby9l83qU6gqAcyzdFPdHWCdwiajPyhAWgwbLuhihKFlxzsBXT83JBVMaHcPhEHivqyFa6NKMQnReXhzdt5IU4P\\\\\\\/tZRJL1b0wtIbh451uHo5dX8qM6M+s9d01e\\\\\\\/5DQJHE0t7anLRREZn0OWSR3tG07HwP9NAzLvJX0fo\\\\\\\/4KygoePAYzPU4bOmXKF+aQQtYamg0VAi3X6xVVFdnkhjNBogAyYtU33yyEJKHN5AI9ZYg66qCFbeKAoO7Blu9kRkcoxnczc\\\\\\\/i4te6q9iVxxXbJJsnSlzw8mxFXVKzvB96U9OKERiGRmnCqNV70tG4Dk6014Me\\\\\\\/DtmHhVd6BG48BzH5ow0ZTEoquK8BWaeMsblOSVJqtDh3hV+zthdTJlPC3s+Ky+MTUfs25Wkbae4S1Ix8RnbHAZkIiwkVNjk4qKO02YQ6KHNGG8ucDjZumyoozC6XA8RsdqyILNo7BI6NbxXFNGlrCZwIxJd6h+dbLl5u30SAJSfAQ3hcBr2\\\\\\\/3zdD1VLVElDcNEDyBuO2XhfsUcECT\\\\\\\/h4zIkETZPo7DgGvQRyef5hVRWScPBdIUi7z5B3e18DtrorBHpL7nZtH+ujDoYtzzTJbXY6hV6B0njUypG4er4XuXDaoVMURPJ2ZhwZHb7HDo2Oq4GdqDx9fvJcxHfmzlSomwWyXvhDCxfBcgcoq6EhoVc7oQU\\\\\\\/Xmjksp+p0i8EQvjIf0avZAMbsCny1bci5H1jPP6wltlPEuUTzijxuCiv1TegIWAUzXLLtPy7qd6dp11mEV2op8PjHDfu6JNrvvHXihxhw3Fh94vaKcpdqamxvav7xdVLd1CIYOrAulZ\\\\\\\/TufWUubxVuxJQY0vC51tqRrwxhOQtKkN2T3SkwA4Au\\\\\\\/5YzOUvZwgPLbNtuwz+PpVgg621eRmLy6g9hZfgG4GwcxhcjYL0p5dQkOqtc\\\\\\\/+KTZCop1Tjf4VQUEtPThXxP\\\\\\\/WpVsnTdSZ+\\\\\\\/COGpqc73zqXITgM5AD\\\\\\\/GLvq9y+tlIlvWWWl8AQt6uTBALoTNS+kmfHXD1wzPCXg5Yd5VsNHQSarNJy7kgChNeMTGfcemak8eWm7WCBsQzDsCwG+f7L4aO4KJVIfUQ2vul2x8MtmCEuCeolKi36TBdsSBbmfhjXqVdEBhAB\\\\\\\/5Sh4MhK\\\\\\\/R7mDLeHC8A+MwJdp2CuSd803YIPlGc48aCDJ\\\\\\\/lKuQJHD8BTSLoImJLD6uwXDuA8yRmdkI4efvctKJKQIAxMHtoD0FYjt52I9iVE2f5F439rsZBoyi7orESenR7NfmdhKzBRPoPzJGJCgkzH19bBN9D17qeObrRJw+tqKujwtaO01ijpk8YP1W50857NMCv+2IAgGKNrPOq1hTCnRjntfkU45pR4yjweoPnf9UvYx41247D5pU9NB\\\\\\\/pVWTW4l3IO3DON2L0NgYmMK7DBbaCui6vr4QJ+nGdq+8Wlqv6dqJDkA5jMh595HnR+PKyU2YX53\\\\\\\/PDkQDxPIy+Owh9ghGWdGGik8zKPU0xYE3EihyksOAdeH5oYYDPTrLcnzp7di8jRZg0VAWxY8PxShZZm26OQNFu+tWn4u7jgufsepl9HU1ZpbLxU2kvSH+oJHot5UW2DmhXXHw5DrEk6\\\\\\\/umLDLjP\\\\\\\/W0Ha7TgSUM7Nldd\\\\\\\/TJtLhlhnucytLyC2rGg1LI0YJTEDe42lGwAIUFG7pyHxGglpUbNMnyV3RvECQUCmSvIgeGt7FcVFY6uNxaiB1FK+lXfXpVv6woTNeCUPF26sGDX3OwQilv6vyKLBt1YFtteqS4uhVbjcC2RPfAkaCRWhNZaW+AytC1gBzSOR07uEREOj3ToVrPyVCgw5EQpayzWvRPLWmuzD60+xtMIE5bxdw2sNydK\\\\\\\/bV\\\\\\\/BqpX7RuJqW+wNSYkYoCXu\\\\\\\/pxPIiJVwV9B1jFwyeVtU7GKZfNBMtgjF58hmjTpNuzAFcHPaiG3XDFS8gd4eSfEbI9jv7EamyHwA9DJ55lSv+zjMWw+QsbY8ZYkOgc9ZIbS+GCNqttUgu5x7q3mB4HBzjXKsjECW2y6ARhVfiRjYzlN1JpqJF7hdNb6IBGvBCy6+rgPNN0D3Q0bjg9DWpOV2wQoeoG6NN5D5HBn14lVdJQYpZM02VEXjNwL1EEHIAYbktPbvAkzIX3IphjQCj1a3lxU3ABJ4D9WOhZuZv\\\\\\\/+OJGAxQT95NDoh\\\\\\\/ShOCvvvyay5uSj\\\\\\\/yvjXibUO9K3JHPNVBCIzJJO\\\\\\\/aGbmWBJNSFnRNfQvKX8gvLpIaiB4TvwfvTJWqnMFxi\\\\\\\/vDLoxBI7E3QOuRTQPDH4SLI1kAHggyRkr7mqh5Qp\\\\\\\/tMXqXC5FSJwnRJFB5rEqrh8bGnTCSO3DJGpbikfydb\\\\\\\/HMdDJPGhwzZg+MaHzmt5MhPQweoQxax\\\\\\\/PMikjTZ2\\\\\\\/UTVqJCM1q9IyPZFtnhTjiS7bZREhsIdkEF\\\\\\\/fY4eR6os4XM5KMSzVnt+VGWE+gE1HqEZ4y9rHOK+fPnvKfUT5wAK5UQKhATjrIsFL4\\\\\\\/7C366y5trJ7yeSgYtJHU\\\\\\\/1mtlMbmxG5wC\\\\\\\/lbWHW16p7Rtx8U+6PdIGOSdNvyyo\\\\\\\/8PdjAvStkbVcAxJ7SAfpFUKK1fv79dCC35Skzj4\\\\\\\/i844g08kTe8\\\\\\\/\\\\\\\/EwEebdBf7R+g2sxg6YKgXvacG+m4YsbVTN2u9lLje2O7F6MED+t542Fa\\\\\\\/xxt63ESLmcnsc1khi+chKIcLM18rpPPdvMtUx38x8nLc25MgsHQhuSg\\\\\\\/XjgTXR531JBGgDzb259SnuiI8nb4KheMuil99mglMDOZi11iLdtyWlCNQia7Un\\\\\\\/7KI2y52MkO0WtPr53UFR8JnEe3BIB\\\\\\\/6roJ6Yp1vaYB\\\\\\\/QWHqhrp+CVpPOMNLBQhkQPGcSFp5KLhPYwel7i\\\\\\\/qGhzvua6oIAm1uEa6KJVvUtyPv83XksildSbnoQQIuGtDPu55Mt+tVk76cMCyn54+J8XC0Hs34FkUx9ZqQQyTg1tdX+BF4zNoGzEmSvdiVJ8BlkhzK\\\\\\\/+XfpuK2FsgaROXh3xR7+bczjFJXOpnqjTxKTwL6kxcbtneQATaP64Y3FDAs2TSqsobQqvmZOCtEmfYSiL2zK71C6cFqO2zYaPLHmQJAdyDML2gv9car1RhchA6uPyCJ5BgOlB3hZLWQFXHS9zjj4dkcm51kdCSMUFzoTexge+\\\\\\\/FvbtFW4ZO2yPAPBtw9zpEW\\\\\\\/+8Kg9inUPbCEMpX\\\\\\\/vgPxUqrpYXGaloo2fLpayn9xK2aD85vg+9bSj+53B+Bh0DXAAcd3jucUJjzDj42y8WARhPwlyhIrGzwmSuEDoGsJQwM2lRvc7Z1jz1VI5R80YoSQNqi2OPYzSgf3yySvwsNF50sOXqMsNWDc84xZc2MMqMxnX3XKFPxFLMm5dmMapLuA8VQ2a6s7F+nm10GhjS5yodj5DmE5eUY2iyiK\\\\\\\/vIo0pWU\\\\\\\/GS5m0mdrNW6AnagW6lTRyl9DGeC9BWJjD8N9vSwvghRP+j3YLIVx0oKMO\\\\\\\/oRLih7oxkja\\\\\\\/VaB8OGdeFPYDzgwGGSIZCw4\\\\\\\/z6mC5CxUhdeBnPBFAVRB9UWYBzhk9eV+tzFXWEl6q48alATyTjL5eNWKKMmbrw\\\\\\\/NyNL16UwZsEp346jQBcFk+ap5JDqa8k2K2k7qic7G9aGCFUsn95aAmUMlW3HjjTOZB9w290BQH8Bbs8Gt5J+WUnSTo\\\\\\\/TDTwOPJp0Bizh6Gb1GwF673ls83c\\\\\\\/3u5yPBf\\\\\\\/xMyIa+nZjR4tr7E1ZPmzVrnZ8UyDQK24dw3AesZhnmnmYWsxlaDaKkbEP2\\\\\\\/SKK\\\\\\\/us3Ctd\\\\\\\/ki1hh8g6NKc4AEIoRXRAyPsWXDK954qgj7ggd\\\\\\\/qsN25XEavXbacIrow5twkwENKpYGByMqyf8t8JLpLdmt5XoEgvCUiiwvorIHXKQntzDUed2zIMyLzdJajW7nOQhhyziOj5M0JtwRZpKFVXJ17ObEzPRd8Khuwv7SzV2M2D1VfcnRQJ3eKfenVp8fAXU4OOttJceyk6X5zROg4QvbxBOTn2b65v8gpLJ+busy\\\\\\\/3rt2AS9PuG05xXCPI6MsIU\\\\\\\/r8tTyEm0zvOXlO6jhzVEPQynyOw7+ZDUFd\\\\\\\/x7ikpPAqv6k\\\\\\\/KGKHKwhZaFTYbuW8jVxDKrRG1IjBYwmGiZQzKhlTSWGzsvpyrPtEHkO3p2rFmvR7x5rCCAzqMnrWprgjJUekNAcSFQpQh8DD2nWjJ94MRXQpuudGVcBSUyo5dYD8PymbU9knrbJp+FN3b0qX3JBOCKtL27xxsy9Wlyq38+WbF2gQHMRYmf9r0gBNQScSEnTSbl2WR8SJl1s3PM3ezLTxMDcV\\\\\\\/JmzFzr9ckoxoqg+wAh6+TqGEWzU05vYKLtMt0Vow2WR2OUBdJZeJJW6STh1WNIFlo7o+xkMLoTY9PpalTth1+culhCnUF2aEUSGGrAvQ+cTzb00vFvVNs0ydMoWtPRep13Abm0fFhF+fk7Oz+4wKZwgmO3OhXNhE+UuIb\\\\\\\/CZ1xl4uuNvc3CXfKfqTDcyvDyI51wgQn4nrVX13G1ho0OttEOtm5CNKNZpnD1u9s85XWofZADvnQs3pXvOSJQI1JSajVWzzs1BrtQbZi8uHcRkC0dLLMrYoY0OfVRCwNfwgj0G7AnrsQ7ZX1X9Q1vnSfiw\\\\\\\/twYr9+7z0agE0qHtrqRXitrPjxcdD8w0bREpG1zpzRzEMPTZZpegM6Ii4CzcuyzB1Z2LQ+jX3hHgSt7j2PPgep1ltvEeK9Y+o99RFM+u+GyPx\\\\\\\/gtF1ea++GFbBeimD1fBYzNJJH7Jdehr7RVD\\\\\\\/wipIaWuzYd+wtOka8M0\\\\\\\/7aRt9BQ\\\\\\\/hjZMdQLJx42VS9l5SGMbkkI2VnnaKwxe1lyf0Hfh6XZ0jcyxFMpZfaLh1hLw4Q2s2SizInC2K893tdTHJs3rf5Yac1SoP\\\\\\\/nHByct4PCwdKbLjI5dgTjw6a7+QpDR1nmPXFr92vdG7yLsrRsc21yqFDkiByTAvwP\\\\\\\/anyHSPMKCaqEnFVXGZ+puIZqEJTJ3hJZNfyqQrbUfiervKmeCVeNtDOffdR6JjF35Q2NFBWvCKQ\\\\\\\/a1RS3uUj68H9c4Ikbal\\\\\\\/WssHY7J\\\\\\\/IpIFrj6DUz\\\\\\\/7b+HaX+syPKq8MgKSX+TLwK1bcatFx2pN0CMuk\\\\\\\/G47I3Z3lUDQbQvgJ4T201ORtGzsiTPgzakaiPi8LOGETysAZOwDiciaSvh3nBK21lGZk+dW3arfu8Z\\\\\\\/OVH26bdT+EfK9tcxk8ESwuka9OW1x1TRl1fcFhqUY\\\\\\\/4gcVo9ZVNo3BgQquG12St4MOXJwi5DAsChr8PfeVI2\\\\\\\/8DK3JdSr\\\\\\\/cHwLbDdkrGfVdfdN\\\\\\\/RDsZ++QZNFceMtq7KwGr8BN3xQCftCEJsLv9l5EZpWSRBgs9j5APSUdhqWXce0YFrG1jmpzB7lrG4uC+f81FNLnzA7bx2Ue2BuLztJ3Gn8\\\\\\\/q2O+hVP13drlrWVMhaSmExW\\\\\\\/CxUKrjWVpI\\\\\\\/nuAKGS0RVoYaTLXiP5eUNZQ3V0sO7tSC5ChTuIsPCI6ejCwJC7bJOlAM6INX75ScGuUj0rwWrBqwaxw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022WV+N9qs3DEL7jriEHNhPrw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229983ff7ec0395b9b84f25b122fed0cd8975ff4b6fbc7ebbaa158566e87c21dd3\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022OPreUGZh0vDcIzf137AdXjhg77DHcCiXf\\\\\\\/JAW\\\\\\\/3X8L6a0GeYhKselo0HaDL8JhoQuP+kVM7grVVzBsmqyA7gTRKQ9fD2o3pxLIc0qI9VYJ5Y4jvc5q9jnMt+sm\\\\\\\/l5cnRjz8U47CDYSYtoOD2TEt7quq2RwvwPZrL2A7Bu+6T5hFIARSL6I1oVc4INRGZfRESik7\\\\\\\/nQtKa7fSVFu8Ztim4yAVic7xtucdoZj853JyvnAH6ZMH60Sk5fUGAw54hezH94FHgKJK+yxnWK0TEhhLxsSshLnQDSpl0lMvlX2u13oZ4OpxYtJBp47AhtC0w9M4kzXgMaT9WhnHdemz+eX0DwB7bsyblxJGkyTtuUXoc0fD4gB54luJ8nhE9BNWachVNpsPkKE0XHvbjpJp02yGETlbnTs7a\\\\\\\/ffP1eOwsPho0blySiNDaJAp\\\\\\\/So4CehyXec5NyPMqb67OxYXZcMMWELTJLXqIAPE2RYgbwlfyXkRKO1KZo1SXK4UQ+O\\\\\\\/6D4owzLRmsNCTk3V8oQsZQKZXtnotQMQLBGjyI+DbYBXaYchbIaWwojfjwZ7c2WIBjwbWCcBu9DiKnCJ5+HFpdOJw\\\\\\\/FkYqXHMfmHqgguMTy8sRZCxTMMhrLbMQvPV\\\\\\\/9NOfiOkPFjpoWzxwSuhVP2z4+BkybYFSie2+xiGAiCC2ZH+8aOu6ohqkFoB+Xlo7QB1IFFvawyVkX\\\\\\\/0mfeB+BXEuzJ\\\\\\\/B3SeO3dXq4aJ5KGRTXTtxlXgfbkj1XmZyKoGA0EpK60BPJXNNIaPnaqC9qIc+FPdg9dRd0sjlmETnfml+7RGm\\\\\\\/Ot9NlSGl+IkFRxmcjzpnVVeT08LzfXhd5FZVKbTDAeiVSKiehaTUAXByM7whRiMZVhtmj3dhFkfwuuXuGtSDroLAH8E2i15mCz9FwyIdLQl+DhB2FRa0kNGpqJwHG8ce6I6i8T1MZiTVjv+rvJ+YxzkOrFcciKGlYK7j+TyelXlrCrNL7E7YaN1XEU7gle2JQ7K9eJ8wMXSIVdSSvLHhG5kNVKWJu\\\\\\\/m0+qZZRdqZA3ptAF9LvZbGqeBmdFHyTbbVXmj3te9rEi8s9UJf4Jd2rxghs3ZSmT4u5q8dJbYNndt+rI4jvawehh5IPqc6sY4XJYJ1Aj7pY\\\\\\\/LNMGIuIkT2wj5YLGVwGGmZJsWcrJJhEwLu3ke4q6qx+G4U6kbcEcm4GZMPSW9OasxKmRhQ3NgOKCNSsHQCZRY752+d5f7YX1R7RV5u+cTkiCcPw3N1f0fDyg90vRTeF8T+DTUjnGkgmbWL0FekXDlmF6H3O4mDUX8PvjS58t\\\\\\\/q1TwUXd4suKyxQ5L3O7g131aoHzDMY2YR6HN79UREBAJ9Wg79SD8xTdjSzg6VsH9IbOMcGIJ8w\\\\\\\/mL6GAQqS5\\\\\\\/JA+8KB4bUA8nyKsrNYkz2Zg+zcF7xHEtXomKoTeJQWVQmH\\\\\\\/hWK9hQEwA+QpIZV7kSIR5vf4sZhbgcE0x7tiC7fXxea5aEttJq78KiIArXIPm633R\\\\\\\/NCENKNCe4pyG7pRtVsddsjlnhmUYLxZoRIHzTVmVNAlOwaLwgEEi00gaY9dD6NPS6J1VRkjkzBJA3FSrmtawBTsSQHXaTUMoSqThiv2+\\\\\\\/wqJovB0SRtGHn5oHzYs+H+Kxq9yhQ0fSA\\\\\\\/iHAihsS9yrcnDeQ6Hpb4qMunMjU1iSf6AzzsZXGtcn5yqafPsTCk3afnCZJ1lKNX2+wGsWuRrf9HwvV4dXEjxUzPb9wDK7yQsd2DRz5k0FHoVkHx\\\\\\\/3g+NzYFMmrYLOkKKNshh8J7Xmej45QX7JnDkDbixVoGewB83QSWQ80gQzpQdrk+twH\\\\\\\/foYeTmSdKXNOzCguco\\\\\\\/MKCFCq5KQXSUkadICQmWxRzNrBSk+kU9DhzSEfvG9wu9273oUnK\\\\\\\/MZpJ\\\\\\\/dyEYvwJcirr375pfyJ1aDgxKIGXDGq1nyBdPnBuqery5cUNmE1LR1t7bJKvz\\\\\\\/kbPDUiPwo\\\\\\\/yUm6aJjdgP6LJ4S1WvjjJuVXFi48ebD99yw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022GLmUAntShnThF60zQ2JnDw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022822200d5b609455014a15c9403e6ea9e59a17d776fea995417de42564b6508b8\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495552 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/525" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-zYVnqDc-bG_SWKy0oyiAJSXlnukqKCPzL3prdKoXTQg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/525" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-OyrHqeEklTdlt_RsG9xG0lpnPWdYsINBbcWmNZvQqJU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/2/515" title="Accelerated Evolution of Limb-Related Gene Hoxd11 in the Common Ancestor of Cetaceans and Ruminants (Cetruminantia)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/2/539" title="Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495552" id="node495552" data-pisa="ggg;10/2/525" data-pisa-master="ggg;g3.119.400853" data-apath="/ggg/10/2/525.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Synaptonemal Complex-Deficient <em>Drosophila melanogaster</em> Females Exhibit Rare DSB Repair Events, Recurrent Copy-Number Variation, and an Increased Rate of <em>de Novo</em> Transposable Element Movement</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6096-8601" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Danny E. Miller</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">525-537;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400853 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495552" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Danny E. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Medical Genetics, Department of Medicine, and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Genetic Medicine, Department of Pediatrics, University of Washington, Seattle, Washington 98105, and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Seattle Children’s Hospital, Seattle, Washington 98105</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danny%2BE.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20DE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADanny%2BE.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6096-8601" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Danny E. Miller</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:danny.miller@seattlechildrens.org" class="" data-icon-position="" data-hide-link-title="0">danny.miller@seattlechildrens.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/2/525" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495552" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495552/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/2/525.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495552" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495552/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/2/525.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495552" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495552/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/2/525.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495552" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495552/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genetic stability depends on the maintenance of a variety of chromosome structures and the precise repair of DNA breaks. During meiosis, programmed double-strand breaks (DSBs) made in prophase I are normally repaired as gene conversions or crossovers. DSBs can also be made by other mechanisms, such as the movement of transposable elements (TEs), which must also be resolved. Incorrect repair of these DNA lesions can lead to mutations, copy-number changes, translocations, and/or aneuploid gametes. In <em>Drosophila melanogaster</em>, as in most organisms, meiotic DSB repair occurs in the presence of a rapidly evolving multiprotein structure called the synaptonemal complex (SC). Here, whole-genome sequencing is used to investigate the fate of meiotic DSBs in <em>D. melanogaster</em> mutant females lacking functional SC, to assay for <em>de novo</em> CNV formation, and to examine the role of the SC in transposable element movement in flies. The data indicate that, in the absence of SC, copy-number variation still occurs and meiotic DSB repair by gene conversion occurs infrequently. Remarkably, an 856-kilobase <em>de novo</em> CNV was observed in two unrelated individuals of different genetic backgrounds and was identical to a CNV recovered in a previous wild-type study, suggesting that recurrent formation of large CNVs occurs in Drosophila. In addition, the rate of novel TE insertion was markedly higher than wild type in one of two SC mutants tested, suggesting that SC proteins may contribute to the regulation of TE movement and insertion in the genome. Overall, this study provides novel insight into the role that the SC plays in genome stability and provides clues as to why the sequence, but not structure, of SC proteins is rapidly evolving.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/meiosis" class="hw-term hw-article-keyword hw-article-keyword-meiosis" rel="nofollow">meiosis</a></li><li class="kwd"><a href="/keyword/whole-genome-sequencing" class="hw-term hw-article-keyword hw-article-keyword-whole-genome-sequencing" rel="nofollow">whole-genome sequencing</a></li><li class="kwd"><a href="/keyword/crossing-over" class="hw-term hw-article-keyword hw-article-keyword-crossing-over" rel="nofollow">crossing over</a></li><li class="kwd"><a href="/keyword/noncrossover-gene-conversion" class="hw-term hw-article-keyword hw-article-keyword-noncrossover-gene-conversion" rel="nofollow">noncrossover gene conversion</a></li><li class="kwd"><a href="/keyword/c3g" class="hw-term hw-article-keyword hw-article-keyword-c3g" rel="nofollow">c(3)G</a></li><li class="kwd"><a href="/keyword/corolla" class="hw-term hw-article-keyword hw-article-keyword-corolla" rel="nofollow">corolla</a></li><li class="kwd"><a href="/keyword/synaptonemal-complex" class="hw-term hw-article-keyword hw-article-keyword-synaptonemal-complex" rel="nofollow">synaptonemal complex</a></li><li class="kwd"><a href="/keyword/transposable-element" class="hw-term hw-article-keyword hw-article-keyword-transposable-element" rel="nofollow">transposable element</a></li><li class="kwd"><a href="/keyword/copy-number-variation-0" class="hw-term hw-article-keyword hw-article-keyword-copy-number-variation" rel="nofollow">copy-number variation</a></li><li class="kwd"><a href="/keyword/sister-chromatid-exchange" class="hw-term hw-article-keyword hw-article-keyword-sister-chromatid-exchange" rel="nofollow">sister chromatid exchange</a></li></ul><p id="p-3">Programmed double-stranded DNA breaks (DSBs) made during prophase of meiosis I are a critical step in the formation of healthy gametes, yet they are potentially catastrophic events for cells. The meiotic break repair machinery must therefore accurately resolve DSBs as either crossovers (COs) or noncrossover gene conversions (NCOGCs). Because more DSBs are made than will be repaired as COs, the majority of DSBs are repaired as NCOGCs, which are nonreciprocal exchange events that result in the 3:1 segregation of alleles.</p><p id="p-4">Crossing over is essential to ensure the proper segregation of homologous chromosomes during subsequent meiotic divisions. Crossovers occur within the context of a large, multiprotein structure called the synaptonemal complex (SC), which forms between homologous chromosomes. In most organisms, DSBs must be made before SC formation can occur, and functional SC is required for proper DSB repair (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">de Massy 2012</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Zickler and Kleckner 2015</a>). However, in <em>Drosophila melanogaster</em>, the SC is necessary for both robust DSB formation and DSB repair (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Lake and Hawley 2012</a>); in the absence of functional SC, DSBs are made at about 20–40% of the wild-type level (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Mehrotra and McKim 2006</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Collins <em>et al.</em> 2014</a>).</p><p id="p-5">The Drosophila SC protein C(3)G is functionally homologous to the transverse filament proteins SYCP-1 in mammals and ZIP1 in budding yeast (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Page and Hawley 2001</a>). While females heterozygous for a loss-of-function <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> allele appear to build normal SC, homozygous females do not build SC and are thus unable to resolve into COs those DSBs that do occur (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Page and Hawley 2001</a>). A previous study examining NCOGC events at a single locus in Drosophila recovered no events from <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> homozygous females but did not report the number of progeny scored (Carlson 1972), thus whether DSBs can be repaired as NCOGCs in females lacking functional SC is unknown. Like C(3)G, the Drosophila SC protein Corolla is also required for SC formation. Mutants in <a href="https://identifiers.org/bioentitylink/FB:FBgn0267967?doi=10.1534/g3.119.400853"><em>corolla</em></a> exhibit phenotypes typical of Drosophila SC mutants, including a reduced number of DSBs (∼40% as assayed by γH2AV foci) and increased levels of chromosome segregation defects (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Collins <em>et al.</em> 2014</a>). As with <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> homozygous females, it is not known how DSBs are repaired in <a href="https://identifiers.org/bioentitylink/FB:FBgn0267967?doi=10.1534/g3.119.400853"><em>corolla</em></a> homozygotes.</p><p id="p-6">While it is evident the SC plays a vital role in resolving DSBs into COs, its role in other meiotic processes is less obvious. For example, there is some evidence for a link between SC formation and the position of transposable elements (TEs), but the data are not definitive (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Pearlman <em>et al.</em> 1992</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hernández-Hernández <em>et al.</em> 2008</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Marcon <em>et al.</em> 2008</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">van der Heijden and Bortvin 2009</a>). Transposable elements are mobile genetic elements active during different stages of gametogenesis. They can be divided into two classes: Class 1, or retrotransposons, replicate using a copy-and-paste method to insert copies of themselves into new locations in the genome, while Class 2, or DNA transposons, use a cut-and-paste method to move from one position in the genome to another. While the tripartite structure of the SC is highly conserved among many species, the genes that make up the SC are evolving rapidly at the sequence level, making their identification even within a species group challenging (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Fraune <em>et al.</em> 2012</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Hemmer and Blumenstiel 2016</a>). The reason for the rapid sequence evolution of the SC is unknown, but it could be to counter the effects of TE movement during meiosis. In Drosophila female meiosis, the rate at which TE movement occurs and whether the SC has any role in facilitating or limiting TE movement remains unclear.</p><p id="p-7">In the current study, whole-genome sequencing was used to investigate individual meiotic events in male offspring from females heterozygous or homozygous for a loss-of-function allele of <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a>. While the number and distribution of CO and NCOGC events in individuals from females heterozygous for <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> was similar to wild type, in progeny arising from <em>c(3)G</em> homozygous mothers (which do not build SC), no COs and only one likely NCOGC event were recovered. The recovery of a single presumed NCOGC event suggests that while repair of DSBs via NCOGC may be possible in females lacking functional SC, it is not a common mechanism of DSB repair. Consistent with the high levels of chromosome missegregation observed in SC mutants, <em>X0</em> males lacking a <em>Y</em> chromosome, males with <em>4<sup>th</sup></em> chromosome gain or loss, and flies that appeared phenotypically male were also recovered.</p><p id="p-8">These data also provide information on what role, if any, the SC components C(3)G and Corolla play in facilitating or inhibiting TE movement during meiosis. In the current study of SC-defective mutants, novel TE insertions were curiously significantly elevated in <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> homozygotes but similar to wild type in <a href="https://identifiers.org/bioentitylink/FB:FBgn0267967?doi=10.1534/g3.119.400853"><em>corolla</em></a> homozygotes. Shared and novel large-scale TE-mediated CNVs were also identified in progeny from all genotypes. Remarkably, one of these CNVs was observed in three unrelated individuals—two from this study and one from a separate study of individual meiotic events in wild type (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Miller <em>et al.</em> 2016b</a>)—suggesting that, similar to humans (Itsara <em>et al.</em> 2009), recurrent CNVs may be a common occurrence in Drosophila. Overall, this work helps further our understanding of how meiotic cells cope with DNA breaks and maintain genetic stability.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Fly Stocks and husbandry</h3><p id="p-9">The loss-of-function allele <em>c(3)G<sup>68</sup></em> (<a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Page and Hawley 2001</a>) was crossed into stocks isogenic for either <em>w<sup>1118</sup></em> or Canton-S strain polymorphisms (Miller <em>et al.</em> 2012). Females homozygous for Canton-S <em>X</em> and <em>2<sup>nd</sup></em> chromosomes and heterozygous for the <em>c(3)G<sup>68</sup></em> loss-of-function allele were crossed to <em>w<sup>1118</sup></em> males to generate females heterozygous for Canton-S and <em>w<sup>1118</sup></em> strain polymorphisms. These heterozygous females were then crossed again to isogenic <em>w<sup>1118</sup></em> males and individual male progeny were isolated for sequencing (Figure S1). Females heterozygous for <em>w<sup>1118</sup></em> and Canton-S <em>X</em> and <em>2<sup>nd</sup></em> chromosomes and homozygous for <em>c(3)G<sup>68</sup></em> were crossed to isogenic <em>w<sup>1118</sup></em> males and individual male offspring were isolated for sequencing (Figure S1). Progeny from corolla<sup><em>129</em></sup> homozygous females were generated by crossing virgin corolla<sup><em>129</em></sup> females to sibling males and collecting both male and female progeny (Figure S1). All crosses were done using a single male and female, and females were allowed to lay eggs for 7 days before being removed from a vial. Male offspring used for sequencing were collected between days 12 and 15 post-fertilization. All flies were kept on standard cornmeal-molasses media and maintained at 25°.</p></div><div id="sec-3" class="subsection"><h3>DNA preparation and sequencing</h3><p id="p-10">For all flies, DNA was prepared from single adults using the Qiagen DNeasy Blood &amp; Tissue Kit. All flies were starved for 4 hr before freezing at -80° for at least 1 hr. One ng of DNA from each was fragmented to 250-bp fragments by adjusting the treatment time to 85 sec using a Covaris S220 sonicator (Covaris Inc.). Libraries were prepared using a Nextera DNA Sample Prep Kit and Bioo Scientific NEXTflex DNA Barcodes. The resulting libraries were purified using Agencourt AMPure XP system (Beckman Coulter) then quantified using a Bioanalyzer (Agilent Technologies) and a Qubit Fluorometer (Life Technologies). Samples from <em>c(3)G<sup>68</sup></em> homozygous females were run on a HiSeq 2500 in rapid mode as either 100-bp paired-end or 125-bp paired-end samples using HiSeq Control Software 1.8.2 and Real-Time Analysis (RTA) version 1.17.21.3. Samples from the <em>c(3)G<sup>68</sup></em> heterozygous and corolla<sup><em>129</em></sup> homozygous experiments were run as 150-bp paired-end on a HiSeq 2500 in rapid mode using HiSeq Control Software 2.2.58 and RTA version 1.18.64. Secondary Analysis version CASAVA-1.8.2 was run to demultiplex reads and generate FASTQ files. Per-sample sequencing and alignment statistics can be found in Table S1.</p></div><div id="sec-4" class="subsection"><h3>DNA alignment, SNP calling, and identification of CO and NCOGC events</h3><p id="p-11">Alignment to the Drosophila reference genome (dm6) was performed using bwa version 0.7.7-r441 using default parameters (Li and Durbin 2009). Single nucleotide and insertion or deletion polymorphisms were identified using SAMtools version 1.9 (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Li <em>et al.</em> 2009</a>). Candidate CO and NCOGC events were identified as described in <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Miller <em>et al.</em> (2016b)</a>.</p></div><div id="sec-5" class="subsection"><h3>Depth-of-coverage calculations</h3><p id="p-12">Depth of coverage for each chromosome arm was calculated by summing the total read depth for each base position then dividing by the length of the entire chromosome arm. Because of the repetitive nature of the <em>Y</em> chromosome, analysis was limited to <em>chrY</em>:332,000–510,000 (Table S1).</p></div><div id="sec-6" class="subsection"><h3>Validation of NCOGCs by PCR</h3><p id="p-13">Nine candidate NCOGC events were identified in 93 males from <em>c(3)G<sup>68</sup></em> females and examined by PCR and Sanger sequencing; Phusion polymerase (NEB) was used according to the manufacturer’s instructions. Only one of the nine putative conversion events validated as real in male c3g-hom-6.4. All primers used can be found in Table S2.</p></div><div id="sec-7" class="subsection"><h3>Calculation of expected NCOGC events</h3><p id="p-14">The number of NCOGCs expected to be recovered from 93 individuals from <em>c(3)G<sup>68</sup></em> females if all DSBs on the <em>X</em> and <em>2<sup>nd</sup></em> chromosomes were repaired as NCOGCs was estimated by performing 100,000 trials of randomly distributing an estimated number of DSBs among the <em>X</em> and <em>2<sup>nd</sup></em> chromosomes using the SNP density of a <em>w<sup>1118</sup></em>/Canton-S heterozygote. Given that DSBs in <em>c(3)G<sup>68</sup></em> females are made at 20% of the wild-type rate of 18–20 DSBs per meiosis (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Mehrotra and McKim 2006</a>), a per-arm number of DSBs was estimated as 0–2 per meiosis. Each break was randomly assigned to a chromosome arm, then to a random chromatid. A random chromatid was then selected to be recovered. NCOGC tract length was assumed to be a minimum of 250 bp and a maximum of 1000 bp (<a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Miller <em>et al.</em> 2016b</a>). An NCOGC was predicted to be recoverable if the tract involved at least one high-quality SNP that differentiated the <em>w<sup>1118</sup></em> and Canton-S genotypes. The estimate of the number of NCOGCs that should be recovered from individual offspring of <em>c(3)G<sup>68</sup></em>/+ females was calculated by multiplying the wild-type per-arm NCOGC rate of 0.3 (<a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Miller <em>et al.</em> 2016b</a>) by 120, the number of arms studied.</p></div><div id="sec-8" class="subsection"><h3>Identification of novel deletion polymorphisms</h3><p id="p-15">Novel deletions were identified using two approaches. Deletions smaller than 30 bp were identified using SAMtools (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Li <em>et al.</em> 2009</a>). For each class of progeny (wild type, <em>c(3)G<sup>68</sup></em>, <em>c(3)G<sup>68</sup></em>/+, and corolla<sup><em>129</em></sup>) a custom script identified any deletion, regardless of quality score, from all vcf files that did not overlap repetitive regions as defined by Repeatmasker (Smit <em>et al.</em>). Novel deletions were those with quality scores over 200 (as determined by SAMtools) that did not fall within 100 bp of another deletion on a different offspring. Candidate novel deletions were validated visually using IGV (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Thorvaldsdóttir <em>et al.</em> 2013</a>). Data for both wild type and <em>c(3)G<sup>68</sup></em> were also analyzed using GATK HaplotypeCaller (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">McKenna <em>et al.</em> 2010</a>), but no deletions not identified by SAMtools were found, thus the remainder of the analysis was completed with SAMtools. Larger deletions were identified using Pindel (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Ye <em>et al.</em> 2009</a>). For each class of progeny, Pindel was run using default settings with an average insert size of 200 bp. Output files for each class of progeny were analyzed as a group and candidate novel deletions were visually validated using IGV.</p></div><div id="sec-9" class="subsection"><h3>Construction of synthetic genomes and sequencing reads</h3><p id="p-16">To determine what percentage of small or large <em>de novo</em> deletion polymorphisms would be identified by SAMtools and Pindel, synthetic genomes were computationally modified with deletions of varying sizes then analyzed using the approach described above. Two classes of genomes were generated: 100 with deletions 1–10 bp long, and 100 with deletions 1–1,000 bp long. For each individual, two genomes were generated: one with an <em>X</em> and without a <em>Y</em> chromosome, and one with a <em>Y</em> and without an <em>X</em>. For each of these genomes, a single nucleotide was randomly changed approximately once every 500 nucleotides to a randomly selected A, G, C, or T. Next, for each genome with an <em>X</em> and without a <em>Y</em> chromosome 2–6 DSBs (approximately 20% of the 18–20 DSBs expected in wild-type (<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Mehrotra and McKim 2006</a>)) were randomly placed on one of four haplotypes in a euchromatic location in the genome. Each of these DSBs was randomly determined to have a deletion between either 1–10 bp or 1–1,000 bp beginning at the site of the DSB. One haplotype of these four was then randomly chosen as the genome for the individual. For each individual, ART was used to generate synthetic reads for both genomes with a read depth of approximately 10x (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Huang <em>et al.</em> 2012</a>). FASTQ files were then combined into a single forward and a single reverse file—and thus represented data from an <em>XY</em> individual with 20x depth of coverage—that were then aligned to the <em>D. melanogaster</em> reference genome as above. SNPs, insertion/deletion polymorphisms, and larger deletions were identified as described above with SAMtools (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Li <em>et al.</em> 2009</a>) and Pindel (<a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Ye <em>et al.</em> 2009</a>). Deletions generated per individual genome can be found in Table S3.</p></div><div id="sec-10" class="subsection"><h3>Identification of transposable element insertions</h3><p id="p-17">To identify TE insertions, split and discordant read pairs were isolated from alignment files using SAMBLASTER (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Faust and Hall 2014</a>). BLAST (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1997</a>) was then used to annotate individual split or discordant reads using the <em>D. melanogaster</em> canonical TE set (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Kaminker <em>et al.</em> 2002</a>). Split and discordant clusters that contained more than five reads aligning to a specific TE family were considered candidate TE insertion sites. Novel insertions were detected by a custom script that compared insertions in one population or stock to related stocks or populations and were visually validated using IGV (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Thorvaldsdóttir <em>et al.</em> 2013</a>). Mosaic insertions on the <em>X</em> chromosome were identified visually using IGV.</p></div><div id="sec-11" class="subsection"><h3>Identification of CNV events</h3><p id="p-18">CNV events were identified as described in <a id="xref-ref-56-5" class="xref-bibr" href="#ref-56">Miller <em>et al.</em> (2016b)</a>. Briefly, average depth of coverage for each individual chromosome arm was determined, then the log<sub>2</sub> depth of coverage for 5-kb nonoverlapping windows was calculated and plotted to reveal large regions of deletions or duplications.</p></div><div id="sec-12" class="subsection"><h3>Data availability</h3><p id="p-19">Illumina data generated for this project are available at the National Center for Biotechnology Information (<a href="https://www.ncbi.nlm.nih.gov/">https://www.ncbi.nlm.nih.gov/</a>). Data for individuals from <em>c(3)G<sup>68</sup></em> females can be found under project PRJNA565835, data for males from <em>c(3)G<sup>68</sup></em> heterozygous females are under project PRJNA565834, and data for males and females from corolla<sup><em>129</em></sup> females are under project PRJNA565794. Wild-type data used in this study were obtained from project PRJNA307070 (<a id="xref-ref-56-6" class="xref-bibr" href="#ref-56">Miller <em>et al.</em> 2016b</a>). Original data underlying this manuscript can be accessed from the Stowers Original Data Repository at <a href="http://www.stowers.org/research/publications/libpb-1476">http://www.stowers.org/research/publications/libpb-1476</a>. All code used in this project is available at GitHub (<a href="https://github.com/danrdanny/c3g-corolla-project/">https://github.com/danrdanny/c3g-corolla-project/</a>). Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.10006634">https://doi.org/10.25387/g3.10006634</a>.</p></div></div><div class="section results" id="sec-13"><h2 class="">Results</h2><div id="sec-14" class="subsection"><h3>Analysis of individual meiotic events from c(3)G<sup>68</sup> heterozygous and homozygous females</h3><p id="p-20">While in many organisms DSBs are made in the absence of SC (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">de Massy 2012</a>; <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Zickler and Kleckner 2015</a>), Drosophila is unique in that SC is required for robust DSB formation (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Lake and Hawley 2012</a>). <em>D. melanogaster</em> females homozygous for loss-of-function alleles of SC genes make DSBs at a rate approximately 20–40% that of wild type (<a id="xref-ref-53-4" class="xref-bibr" href="#ref-53">Mehrotra and McKim 2006</a>; <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Collins <em>et al.</em> 2014</a>), and it remains unclear how these DSBs are repaired. Studies using visual markers in Drosophila have shown that repair of DSBs by crossing over is substantially reduced or completely abolished in females unable to construct full-length SC, and it is not known if these DSBs can be repaired by other pathways, such as NCOGC or nonhomologous end-joining (NHEJ) (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gowen 1933</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hall 1972</a>; <a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">Page and Hawley 2001</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Manheim and McKim 2003</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Jeffress <em>et al.</em> 2007</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Page <em>et al.</em> 2007</a>; <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Collins <em>et al.</em> 2014</a>).</p><p id="p-21">To better understand this process, whole-genome sequencing was performed on individual male progeny from mothers heterozygous for wild-type Canton-S and <em>w<sup>1118</sup> X</em> and <em>2<sup>nd</sup></em> chromosomes and either homozygous or heterozygous for the loss-of-function allele <em>c(3)G<sup>68</sup></em> on chromosome <em>3</em>. Male progeny from <em>c(3)G<sup>68</sup></em> homozygous mothers (96 males from 10 females) represent the experimental group lacking SC and will hereafter be referred to as <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> offspring, and male progeny from <em>c(3)G<sup>68</sup></em> heterozygous mothers (40 males from two females) represent the control group with functional SC and will be referred to as <em>c(3)G/+</em> offspring (Figure S1).</p><p id="p-22">While Drosophila males normally have an <em>X</em> and a <em>Y</em> chromosome, sex is determined by the ratio of <em>X</em> chromosomes to autosomes rather than the presence of a <em>Y</em>, thus <em>X0</em> flies are male. This is seen when <em>X</em> chromosome missegregation (nondisjunction) leads to no maternal sex chromosome contribution, with a paternally inherited <em>X</em>. Triploid flies carrying three copies of each autosome and two <em>X</em> chromosomes may also be phenotypically male and are known as intersex males (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bridges 1921</a>). To assay for <em>X</em> and <em>4<sup>th</sup></em> chromosome nondisjunction and the presence of triploid flies, depth of coverage was calculated for each chromosome arm as a percentage of one of the autosomes (Table S1). Males carrying the expected number of <em>X</em> chromosomes should have <em>X</em> and <em>Y</em> chromosome depth of coverage half that of an autosome and <em>4<sup>th</sup></em> chromosome depth of coverage equal to an autosome. As expected, all 40 male offspring from the <em>c(3)G/+</em> control group were diploid with an <em>X</em> and a <em>Y</em> chromosome as well as two copies of the <em>4<sup>th</sup></em> chromosome. Meanwhile, among the offspring from the SC-deficient <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> experimental group, 25 were found to be <em>X0</em> males carrying an <em>X</em> chromosome with the <em>w<sup>1118</sup></em> haplotype and six were found to have three <em>4<sup>th</sup></em> chromosomes (Table S1, Figure S2). These 25 <em>X</em> chromosomes were most likely paternally inherited, since nondisjunction of the <em>X</em> or <em>Y</em> chromosome in wild-type Drosophila males has been shown to occur in fewer than 1 in 5,000 meioses (Boschi <em>et al.</em> 2006); however, because the mother was heterozygous for a <em>w<sup>1118</sup></em> chromosome, there is a small, albeit unlikely, possibility that these <em>X</em> chromosomes were maternally derived. The high levels of nondisjunction in <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> homozygous females (42% <em>X</em> and 12% <em>4<sup>th</sup></em>) was expected and is similar to previous work using the <em>c(3)G<sup>68</sup></em> allele (39% <em>X</em> and 27% <em>4<sup>th</sup></em>) (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Hall 1972</a>).</p><p id="p-23">Three of the 96 <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> offspring had an <em>X</em> chromosome depth of coverage approximately 67% that of chromosomes <em>2</em> and <em>3</em>, with two of these three also carrying a <em>Y</em> chromosome (Figure S2, Table S1). Allele frequency for each SNP on each chromosome arm was calculated and revealed that all three were triploid, with one <em>XX:222:333</em> and two <em>XXY:222:333</em> offspring (Figure S3). The <em>XX:222:333</em> intersex offspring was also mosaic for loss of a <em>4<sup>th</sup></em> chromosome, with 75% depth of coverage of the <em>4<sup>th</sup></em> compared to chromosome <em>2L</em>, suggesting post-meiotic loss of the <em>4<sup>th</sup></em> in <em>XX:222:333:444</em> cells (Figure S2). The recovery of intersex individuals was not surprising as previous studies have noted an increase in the number of triploid individuals recovered from <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> mutants (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Gowen 1933</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Lindsley and Zimm 1992</a>). These three offspring were excluded from subsequent analysis.</p><p id="p-24">CO and NCOGC events were then identified on the <em>X</em> and <em>2<sup>nd</sup></em> chromosomes in both <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> and <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a>/+ male offspring through changes in polymorphisms in each fly. (CO and NCOGC events were not analyzed on the <em>3<sup>rd</sup></em> because <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> lies on this chromosome, nor were they analyzed on the 25 paternally inherited <em>X</em> chromosomes carried by <em>X0 c(3)G</em> offspring.) A total of 41 single COs and 7 double COs were identified in <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a>/+ offspring (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>, Table S4), with a frequency of exchange similar to previous observations in wild type for all three arms (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). A total of 32 NCOGCs were also identified (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>, Table S5), not significantly different than the 36 expected to be recovered based on wild-type rates (<a id="xref-ref-56-7" class="xref-bibr" href="#ref-56">Miller <em>et al.</em> 2016b</a>). Previous work has shown rates of crossing over similar to wild type for the <em>c(3)G<sup>68</sup></em> allele when heterozygous (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hall 1977</a>), but higher rates of crossing over for <em>c(3)G<sup>17</sup></em> as a heterozygote (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Hinton 1966</a>). While the <em>c(3)G<sup>68</sup></em> allele is a known point mutation, the <em>c(3)G<sup>17</sup></em> allele (also historically known as <em>c(3)G<sup>1</sup></em>) is a TE insertion that disrupts the function of the gene (<a id="xref-ref-57-5" class="xref-bibr" href="#ref-57">Page and Hawley 2001</a>), and the reason for the difference in exchange between these two alleles is not clear.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/525/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="CO and NCOGC events recovered in this study (details are in Table S4 and Table S5). (A) Individual NCOGC, single crossover (SCO), and double crossover (DCO) events recovered per chromosome arm in c(3)G68/+ females. No DCOs were recovered on 2R in c(3)G68/+, and no COs of any type were recovered in c(3)G68 homozygotes. Gray shading indicates heterochromatic regions; black circles represent centromeres. (B) Coefficient of exchange for all 55 CO events from c(3)G68/+ females recovered in this study compared to wild-type data from Miller et al. (2016b). Points represent observed frequency; the line is a best-fit line through the data. (C) Coefficient of exchange for all 32 NCOGC events recovered from c(3)G68/+ females compared to the same wild-type data as in (B). (D) Detail of the only CO-associated GC recovered in this study. The CO could have occurred at one of two positions on chromosome 2L, either between SNPs at positions 2,863,597 and 2,867,093 with the CO-associated GC being the heterozygous tract between positions 2,867,448 and 2,867,730. Alternatively, the CO may have occurred between 2,867,730 and 2,869,931 with the CO-associated GC defined by the 4 SNPs between 2,867,093 and 2,867,282. No similar CO-associated GC events were recovered in a previous analysis of 196 individual meiotic events from wild-type females (Miller et al. 2016b) but have been reported in previous studies in Drosophila (Curtis et al. 1989). (E) Structure of the single NCOGC event recovered from a homozygous c(3)G68 female in this study. This NCOGC, validated by PCR and Sanger Sequencing, was defined by a 4-bp deletion on one side and a SNP on the other, both from the w1118 line. The NCOGC has a maximum possible tract length of 2,507 bp and a minimum tract length of 180 bp." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-677979155" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;CO and NCOGC events recovered in this study (details are in Table S4 and Table S5). (A) Individual NCOGC, single crossover (SCO), and double crossover (DCO) events recovered per chromosome arm in &lt;em&gt;c(3)G&lt;sup&gt;68&lt;/sup&gt;/+&lt;/em&gt; females. No DCOs were recovered on &lt;em&gt;2R&lt;/em&gt; in &lt;em&gt;c(3)G&lt;sup&gt;68&lt;/sup&gt;/+&lt;/em&gt;, and no COs of any type were recovered in &lt;em&gt;c(3)G&lt;sup&gt;68&lt;/sup&gt;&lt;/em&gt; homozygotes. Gray shading indicates heterochromatic regions; black circles represent centromeres. (B) Coefficient of exchange for all 55 CO events from &lt;em&gt;c(3)G&lt;sup&gt;68&lt;/sup&gt;/+&lt;/em&gt; females recovered in this study compared to wild-type data from Miller &lt;em&gt;et al.&lt;/em&gt; (2016b). Points represent observed frequency; the line is a best-fit line through the data. (C) Coefficient of exchange for all 32 NCOGC events recovered from &lt;em&gt;c(3)G&lt;sup&gt;68&lt;/sup&gt;/+&lt;/em&gt; females compared to the same wild-type data as in (B). (D) Detail of the only CO-associated GC recovered in this study. The CO could have occurred at one of two positions on chromosome &lt;em&gt;2L&lt;/em&gt;, either between SNPs at positions 2,863,597 and 2,867,093 with the CO-associated GC being the heterozygous tract between positions 2,867,448 and 2,867,730. Alternatively, the CO may have occurred between 2,867,730 and 2,869,931 with the CO-associated GC defined by the 4 SNPs between 2,867,093 and 2,867,282. No similar CO-associated GC events were recovered in a previous analysis of 196 individual meiotic events from wild-type females (Miller &lt;em&gt;et al.&lt;/em&gt; 2016b) but have been reported in previous studies in Drosophila (Curtis &lt;em&gt;et al.&lt;/em&gt; 1989). (E) Structure of the single NCOGC event recovered from a homozygous &lt;em&gt;c(3)G&lt;sup&gt;68&lt;/sup&gt;&lt;/em&gt; female in this study. This NCOGC, validated by PCR and Sanger Sequencing, was defined by a 4-bp deletion on one side and a SNP on the other, both from the &lt;em&gt;w&lt;sup&gt;1118&lt;/sup&gt;&lt;/em&gt; line. The NCOGC has a maximum possible tract length of 2,507 bp and a minimum tract length of 180 bp.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/525/F1.medium.gif" width="340" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/2/525/F1.medium.gif" width="340" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/525/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/525/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495597" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-25" class="first-child">CO and NCOGC events recovered in this study (details are in Table S4 and Table S5). (A) Individual NCOGC, single crossover (SCO), and double crossover (DCO) events recovered per chromosome arm in <em>c(3)G<sup>68</sup>/+</em> females. No DCOs were recovered on <em>2R</em> in <em>c(3)G<sup>68</sup>/+</em>, and no COs of any type were recovered in <em>c(3)G<sup>68</sup></em> homozygotes. Gray shading indicates heterochromatic regions; black circles represent centromeres. (B) Coefficient of exchange for all 55 CO events from <em>c(3)G<sup>68</sup>/+</em> females recovered in this study compared to wild-type data from <a id="xref-ref-56-8" class="xref-bibr" href="#ref-56">Miller <em>et al.</em> (2016b)</a>. Points represent observed frequency; the line is a best-fit line through the data. (C) Coefficient of exchange for all 32 NCOGC events recovered from <em>c(3)G<sup>68</sup>/+</em> females compared to the same wild-type data as in (B). (D) Detail of the only CO-associated GC recovered in this study. The CO could have occurred at one of two positions on chromosome <em>2L</em>, either between SNPs at positions 2,863,597 and 2,867,093 with the CO-associated GC being the heterozygous tract between positions 2,867,448 and 2,867,730. Alternatively, the CO may have occurred between 2,867,730 and 2,869,931 with the CO-associated GC defined by the 4 SNPs between 2,867,093 and 2,867,282. No similar CO-associated GC events were recovered in a previous analysis of 196 individual meiotic events from wild-type females (<a id="xref-ref-56-9" class="xref-bibr" href="#ref-56">Miller <em>et al.</em> 2016b</a>) but have been reported in previous studies in Drosophila (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Curtis <em>et al.</em> 1989</a>). (E) Structure of the single NCOGC event recovered from a homozygous <em>c(3)G<sup>68</sup></em> female in this study. This NCOGC, validated by PCR and Sanger Sequencing, was defined by a 4-bp deletion on one side and a SNP on the other, both from the <em>w<sup>1118</sup></em> line. The NCOGC has a maximum possible tract length of 2,507 bp and a minimum tract length of 180 bp.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">Among 93 <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> homozygous offspring, no CO events were recovered on the <em>X</em> or <em>2<sup>nd</sup></em> chromosomes, but a single NCOGC event in male c3g-hom-6.4 was identified and validated by PCR and Sanger sequencing. This event occurred on a chromosome with the Canton-S haplotype, which could have occurred only in the heterozygous <em>w<sup>1118</sup></em>/Canton-S mother and thus was not contributed by the isogenic <em>w<sup>1118</sup></em> father. This NCOGC was minimally defined by a 4-bp deletion on the 5′ side (<em>2R</em>:23,350,969–23,350,972, release 6 coordinates) and a single polymorphism on the 3′ side (<em>2R</em>:23,351,148) (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1E</a>, Table S5). Because it was defined by these two closely located polymorphisms that created two changes identical to the other haplotype used in this study, it is unlikely the event was the result of <em>de novo</em> somatic mutation. Moreover, the average depth of coverage within the 1-kb interval surrounding the two polymorphisms was 54x, similar to the average depth of coverage for chromosome <em>2R</em> for this individual, making it unlikely that this NCOGC was due to a deletion or duplication of this interval. Additionally, the minimum and maximum possible widths of this NCOGC are 180 bp and 2,507 bp, respectively, well within ranges observed in wild type (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Hilliker <em>et al.</em> 1994</a>; <a id="xref-ref-56-10" class="xref-bibr" href="#ref-56">Miller <em>et al.</em> 2016b</a>). Because homologous chromosomes pair prior to meiotic onset, this NCOGC could be the result of DSB repair in a pre-meiotic cell (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bosco 2012</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Joyce <em>et al.</em> 2012</a>). Unfortunately, there are no reliable estimates of the rate at which pre-meiotic NCOGC occurs, making the likelihood difficult to assess.</p><p id="p-27">Females homozygous for <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> loss-of-function alleles make DSBs at ∼20% the level of wild type (<a id="xref-ref-53-5" class="xref-bibr" href="#ref-53">Mehrotra and McKim 2006</a>). To estimate the number of NCOGCs that should have been recovered in the <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> dataset if DSB repair as NCOGCs occurred frequently, a simulation was performed. This model randomly distributed DSBs among 68 <em>X</em> and 93 <em>2<sup>nd</sup></em> chromosome arms as if they occurred at a rate 20% that of wild type. The model estimated that 37–62 NCOGCs should have been recovered if all DSBs that occurred were repaired as NCOGCs (since COs do not occur in <em>c(3)G<sup>68</sup></em> homozygotes). The recovery of a single candidate NCOGC event is significantly less than the 37–62 expected NCOGCs (<em>P</em> &amp;amp;lt; 0.001, Fisher’s exact), indicating that like COs, full-length SC is crucial for the repair of DSBs into NCOGCs. Indeed, using these data, the rate of NCOGC in an SC-deficient female can be estimated as approximately 1x10<sup>−10</sup> per bp per meiosis, markedly lower than the wild-type rate of 1.9x10<sup>−8</sup> NCOGCs per bp per meiosis (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Hilliker <em>et al.</em> 1994</a>; <a id="xref-ref-56-11" class="xref-bibr" href="#ref-56">Miller <em>et al.</em> 2016b</a>). This raises the obvious question, which will be considered next: if they are rarely repaired as COs or NCOGCs, what <em>is</em> the fate of DSBs that occur in SC-deficient flies?</p></div><div id="sec-15" class="subsection"><h3>Most DSBs in SC-deficient females are repaired by an error-free process</h3><p id="p-28">In addition to meiotic CO or NCOGC, other potential mechanisms exist for repair of meiotic DSBs. One such mechanism, NHEJ, is often described as an error-prone process resulting in deletions (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bétermier <em>et al.</em> 2014</a>), but data suggest the canonical NHEJ pathway is a higher-fidelity system than previously believed (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Kabotyanski <em>et al.</em> 1998</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Feldmann <em>et al.</em> 2000</a>). Alternatively, single-strand annealing (SSA), alternative end-joining (alt-EJ), and microhomology-mediated end-joining (MMEJ) are pathways that may result in small deletions that could be detected as novel deletion polymorphisms in whole-genome sequencing data (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Wang <em>et al.</em> 2003</a>, <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">2005</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Guirouilh-Barbat <em>et al.</em> 2007</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Rass <em>et al.</em> 2009</a>). Finally, repair of DSBs using the sister chromatid as a template may occur and leave little or no evidence detectable by whole-genome sequencing.</p><p id="p-29">Gene conversion with the sister chromatid has been shown to be a significant repair pathway in both <em>S. cerevisiae</em> (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Goldfarb and Lichten 2010</a>) and mammalian cells (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Johnson and Jasin 2000</a>), thus it is reasonable to assume it may be active during Drosophila female meiosis as well. Indeed, ring chromosome assays have shown a decrease in the recovery of ring chromosomes in the absence of <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a>, suggesting breaks in <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> homozygous females can be repaired by intersister recombination (see Discussion for a description of the ring chromosome assay) (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Sandler 1965</a>). Furthermore, in both <em>FM7/+</em>; <em>c(3)G<sup>68</sup></em> and <em>FM7/+</em> females with functional SC, the recovery of <em>Bar</em> revertants through unequal exchange between sister chromatids supports the hypothesis that sister chromatid exchange does occur in flies (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Hall 1977</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Miller <em>et al.</em> 2016a</a>).</p><p id="p-30">To determine if DSB repair in SC-deficient females occurs by an error-prone process, novel deletion polymorphisms were identified in the three previously described classes of progeny (wild type, <em>c(3)G<sup>68</sup></em>/+, and <em>c(3)G<sup>68</sup></em>) plus an additional class unable to repair DSBs by crossing over. Females carrying loss-of-function mutations of the SC gene <a href="https://identifiers.org/bioentitylink/FB:FBgn0267967?doi=10.1534/g3.119.400853"><em>corolla</em></a> are unable to construct full-length SC, and thus have a high rate of nondisjunction, yet still make DSBs at a rate approximately 40% of wild-type (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Collins <em>et al.</em> 2014</a>), similar to the phenotype observed in <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> loss-of-function mutations. 50 individual males and females from three females homozygous for a nonsense mutation in the SC protein <a href="https://identifiers.org/bioentitylink/FB:FBgn0267967?doi=10.1534/g3.119.400853"><em>corolla</em></a> (<em>corolla<sup>129</sup></em>) were sequenced. Of these 50 individuals, 11 were the result of <em>X</em> chromosome nondisjunction, with 3 <em>X0</em> males and 8 <em>XXY</em> females; 9 were triplo-4; 2 were nondisjunctional for both the <em>X</em> and <em>4<sup>th</sup></em> chromosomes; and no <em>X</em> or <em>4<sup>th</sup></em> chromosome mosaics were observed (Table S1). The genetic background of the <em>2<sup>nd</sup></em> and <em>3<sup>rd</sup></em> chromosomes of females homozygous for <a href="https://identifiers.org/bioentitylink/FB:FBgn0267967?doi=10.1534/g3.119.400853"><em>corolla</em></a> was not controlled, thus candidate CO and NCOGC events could not be identified, but previous studies have shown a nearly complete absence of exchange in <a href="https://identifiers.org/bioentitylink/FB:FBgn0267967?doi=10.1534/g3.119.400853"><em>corolla</em></a> homozygous females (<a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Collins <em>et al.</em> 2014</a>).</p><p id="p-31">In all four classes of progeny (wild type, <em>c(3)G<sup>68</sup></em>/+, <em>c(3)G<sup>68</sup></em>, and corolla<sup><em>129</em></sup>), <em>de novo</em> deletions were searched for using two different approaches (see Methods). First, vcf files generated by SAMtools (<a id="xref-ref-47-4" class="xref-bibr" href="#ref-47">Li <em>et al.</em> 2009</a>) were analyzed for deletion polymorphisms (these are generally less than 20 bp), and second, larger deletions were identified using Pindel (<a id="xref-ref-72-3" class="xref-bibr" href="#ref-72">Ye <em>et al.</em> 2009</a>). Separately, the output of GATK HaploType caller (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">McKenna <em>et al.</em> 2010</a>) was compared to SAMtools and was found to produce similar results, thus only data from SAMtools was analyzed. Both approaches identified a similar number of <em>de novo</em> deletions per fly in all four classes of progeny. Specifically, SAMtools identified 11 deletions 1–11 bp in size in previously published data from 196 wild-type males, a single 21-bp deletion in 40 <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a>/+ offspring, 8 deletions ranging from 1–14 bp from 93 <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> offspring, and one 3-bp deletion from 50 <a href="https://identifiers.org/bioentitylink/FB:FBgn0267967?doi=10.1534/g3.119.400853"><em>corolla</em></a> individuals (Table S6). Pindel identified only one large novel deletion among all genotypes, a complex 17-bp deletion in a <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> homozygous male.</p><p id="p-32">Because DSBs are made at different rates in wild type and SC-deficient mutants, comparing the absolute number of deletions among progeny is not informative. A more accurate measure of deletion frequency is deletions per DSB. While approximately 18–20 DSBs are made during wild-type meiosis, only 4–8 are made in the SC mutants studied here (<a id="xref-ref-53-6" class="xref-bibr" href="#ref-53">Mehrotra and McKim 2006</a>; <a id="xref-ref-11-7" class="xref-bibr" href="#ref-11">Collins <em>et al.</em> 2014</a>). Although the number of DSBs made in <em>c(3)G<sup>68</sup></em> heterozygous females is unknown, it is assumed to be similar to wild type because the landscape of CO and NCOGC events is similar to wild type. Using these estimates, the rate of deletions per DSB per meiosis (assuming only 25% of DSBs are recovered in the offspring) can be calculated as approximately 1% for wild-type and <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a>/+ females, 3% for <a href="https://identifiers.org/bioentitylink/FB:FBgn0267967?doi=10.1534/g3.119.400853"><em>corolla</em></a> females, and 10% for <em>c(3)G</em> females. The rate for <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> females is significantly higher than wild type (<em>P =</em> 0.008, Student’s 2-tailed T-test) but the rate for <a href="https://identifiers.org/bioentitylink/FB:FBgn0267967?doi=10.1534/g3.119.400853"><em>corolla</em></a> females is not (<em>P</em> = 0.6, Student’s 2-tailed T-test).</p><p id="p-33">Thus, while there is a subtle but significant increase in the rate of <em>de novo</em> deletions recovered in offspring from <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> mothers, this does not sufficiently explain what happens to the majority of DSBs made during meiosis. It does suggest that the majority of DSBs are repaired by a non-error-prone process between the homolog or with the sister chromatid. Of course, secondary alignment and/or analysis errors may make <em>de novo</em> deletions difficult to detect.</p><p id="p-34">To test whether the analysis approach was robust enough to detect both large and small deletions, 200 <em>D. melanogaster</em> genomes with novel random single nucleotide and deletion polymorphisms were generated computationally. Two different classes of genomes were created, 100 with deletions 1–10 bp in size, and 100 with deletions 1–1,000 bp in size (Table S3). Synthetic reads were generated based on these genomes and aligned and analyzed using the same steps as the experimental samples. A total of 713 synthetic deletions were generated, with 339 1–10 bp deletions and 374 1–1,000 bp deletions. SAMtools identified 86% of 1–10 bp deletions on the <em>X</em>, <em>2<sup>nd</sup></em>, and <em>3<sup>rd</sup></em> chromosomes. Pindel recovered 57% of synthetic 1–1000 bp deletions (213 of the 374) with the highest fraction of deletions recovered on chromosome <em>2L</em> (72%) and the fewest on chromosome <em>2R</em> (44%) (Table S7). These models indicate that had deletions occurred at a higher rate, the additional small and large deletion polymorphisms created by error-prone repair mechanisms should have been detected in offspring from the <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> or <a href="https://identifiers.org/bioentitylink/FB:FBgn0267967?doi=10.1534/g3.119.400853"><em>corolla</em></a> females.</p></div><div id="sec-16" class="subsection"><h3>De novo transposable element insertions are more frequent in c(3)G<sup>68</sup> homozygous females</h3><p id="p-35">While the SC is essential for repair of DSBs as CO and NCOGCs, it is unknown if the SC regulates other molecular events such as the movement of TEs. Absence of the yeast <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> homolog <em>Zip1</em> has been shown to result in a decreased insertion rate of the retrotransposon <em>Ty1</em>, suggesting a role for the SC in TE movement in some organisms (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Dakshinamurthy <em>et al.</em> 2010</a>). The rate at which TE movement occurs during Drosophila meiosis is unknown, therefore to determine the baseline transposition rate, previously published data were used to identify 44 novel TE insertions from the <em>X</em>, <em>2<sup>nd</sup></em> and <em>3<sup>rd</sup></em> chromosomes from 196 wild-type individuals (<a id="xref-ref-56-12" class="xref-bibr" href="#ref-56">Miller <em>et al.</em> 2016b</a>) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, Table S8). In this dataset, a single novel insertion on the <em>4<sup>th</sup></em> chromosome was observed but is not included in the rate calculations (Table S8). Seven of the 44 insertions occurred close enough to a polymorphism to confirm through linkage that they could only have been maternally inherited. For example, male cs13.13 carries a novel <em>roo</em> insertion on the <em>w<sup>1118</sup> X</em> chromosome that is not seen in the 11 other male siblings that also inherited the <em>w<sup>1118</sup></em> haplotype from the same female. It is not possible to definitively determine which parent the remaining 37 events were inherited from due to low SNP density. Using these data, a per arm rate of <em>de novo</em> euchromatic TE insertion can be estimated as 0.18 insertions per arm per meiosis [(44 events × 4 haploid meiotic products) / (196 meiosis × 5 arms)], meaning that while a novel euchromatic TE insertion occurs in approximately 1 in every 5 meioses, it would only be recovered in approximately 1 in every 20 progeny.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/525/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Novel TE insertions identified after a single round of meiosis for all four classes of offspring analyzed in this study. Details about insertion position and class of TE inserted can be found in Table S8." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-677979155" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Novel TE insertions identified after a single round of meiosis for all four classes of offspring analyzed in this study. Details about insertion position and class of TE inserted can be found in Table S8.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/525/F2.medium.gif" width="440" height="213"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/2/525/F2.medium.gif" width="440" height="213"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/525/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/525/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495689" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-36" class="first-child">Novel TE insertions identified after a single round of meiosis for all four classes of offspring analyzed in this study. Details about insertion position and class of TE inserted can be found in Table S8.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">This same approach was then applied to the <em>X</em>, <em>2<sup>nd</sup></em>, and <em>3<sup>rd</sup></em> chromosomes from <em>c(3)G/+</em> and <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> offspring. In the 40 <em>c(3)G/+</em> offspring, 9 novel insertion events were identified—2 on the <em>X</em> chromosome, 3 on chromosome <em>2R</em>, 1 on chromosome <em>3L</em>, and 3 on chromosome <em>3R</em> (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, Table S8). Recovery of 9 novel insertion events in 40 individuals when surveying 5 chromosome arms gives a per arm <em>de novo</em> rate of transposition of 0.18 insertions per arm per meiosis, identical to the rate observed in wild type.</p><p id="p-38">In <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> homozygotes, <em>de novo</em> transposition events were identified on all <em>2<sup>nd</sup></em> and <em>3<sup>rd</sup></em> chromosomes as well as the 68 maternally inherited <em>X</em> chromosomes from the 93 non-intersex males. Among the 440 maternally inherited chromosomes from these 93 males [68 <em>X</em> chromosomes + (93 offspring × 4 autosomes)], 64 novel transposition events were identified—12 on the <em>X</em> chromosome, 10 on <em>2L</em>, 15 on <em>2R</em>, 13 on <em>3L</em>, and 14 on <em>3R</em> (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, Table S8). Analysis of sibling <em>X</em> chromosome haplotypes confirmed that all 12 maternally inherited <em>X</em> chromosome insertions were <em>de novo</em>. Considering the 440 maternally inherited chromosomes together, the per arm per meiosis rate of novel transposon insertion events in <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> offspring was 0.58 insertions per arm per meiosis for the <em>X</em>, <em>2<sup>nd</sup></em>, and <em>3<sup>rd</sup></em> chromosomes, which is significantly higher than wild type (<em>P</em> &amp;amp;lt; 0.001, Chi-square test). There was not a significant difference in the per-arm per meiosis insertion rates of 0.71, 0.54, and 0.58 for the maternally inherited <em>X</em>, <em>2<sup>nd</sup></em>, and <em>3<sup>rd</sup></em> chromosomes, respectively (<em>P</em> = 0.36, one-way ANOVA). The 25 paternally inherited <em>X</em> chromosomes were analyzed separately and 4 <em>de novo</em> TE insertions were observed resulting in a <em>de novo</em> transposition rate of 0.64 insertions per arm per meiosis. This rate was not statistically different from that observed for maternally inherited chromosomes (<em>P</em> = 0.58, Student’s T-test) and was also significantly higher than wild type (<em>P</em> = 0.002, Chi-square test).</p><p id="p-39">To help delineate whether the increase in <em>de novo</em> transposition events is a general property of SC-deficient females or specific to <em>c(3)G<sup>68</sup></em> homozygous females, novel TE insertions were identified in offspring from the previously described <a href="https://identifiers.org/bioentitylink/FB:FBgn0267967?doi=10.1534/g3.119.400853"><em>corolla</em></a> mutant females. Using the same approach as above, 12 <em>de novo</em> transposon insertions were identified on the <em>X</em>, <em>2<sup>nd</sup></em>, and <em>3<sup>rd</sup></em> chromosomes of 50 individual offspring (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>, Table S8). Of the 12 novel insertions identified, none occurred on the <em>X</em> in a male with a paternally inherited <em>X</em> chromosome, and one occurred on the <em>X</em> chromosome of an <em>XXY</em> female carrying two maternally inherited <em>X</em> chromosomes. The distribution of events was similar to that observed in wild type. These 12 events were recovered from all five chromosome arms, giving a rate of 0.19 insertions per arm per meiosis in <a href="https://identifiers.org/bioentitylink/FB:FBgn0267967?doi=10.1534/g3.119.400853"><em>corolla</em></a> mutants, similar to the rate of 0.18 observed in both wild type and <em>c(3)G<sup>68</sup>/+</em> heterozygous females but significantly less than the observed <em>de novo</em> TE insertion rate in <em>c(3)G<sup>68</sup></em> homozygous females. This suggests that the increase in <em>de novo</em> TE insertions may not be a general property of SC-deficient mutants, but may be specific to <em>c(3)G<sup>68</sup></em> homozygotes.</p><p id="p-40">Previous work has shown that the <em>de novo</em> movement of TEs in somatic cells contributes to cellular diversity in Drosophila (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Eickbush and Eickbush 2011</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Treiber and Waddell 2017</a>). While relatively low depth of coverage and difficulty with phasing due to limited SNP density make identifying post-meiotic somatic TE insertions on the autosomes challenging, it is easier to identify mosaic insertions on the single <em>X</em> chromosome carried by males. None of the 6 <em>de novo X</em> chromosome insertions among males from wild type, <em>c(3)G/+</em>, and <a href="https://identifiers.org/bioentitylink/FB:FBgn0267967?doi=10.1534/g3.119.400853"><em>corolla</em></a> females appeared to be somatic events. In <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> offspring, however, two of the 16 <em>de novo X</em> chromosome insertions identified were likely somatic events (males c3g-hom-7.14 and c3g-hom-9.7, Table S8). Both of these insertions were <em>Doc</em> elements, similar to somatic events identified by <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Treiber and Waddell (2017)</a>, and are present in approximately 50% of reads, suggesting they happened after the first but before the second mitotic division. It is likely that additional somatic insertions occurred in later mitotic divisions that are not detectable in this dataset given the depth of coverage in this experiment and challenges in identifying these events using short reads.</p></div><div id="sec-17" class="subsection"><h3>De novo copy-number variation occurs in the absence of full-length SC</h3><p id="p-41">Copy-number variation is a significant source of genetic variability within populations (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Kaminker <em>et al.</em> 2002</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Lee and Langley 2010</a>). CNVs may be beneficial or deleterious to an individual and may involve a large or small number of genes. Previous studies in <em>D. melanogaster</em> have revealed surprisingly high rates of <em>de novo</em> CNV both in single offspring and shared among several siblings (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Watanabe <em>et al.</em> 2009</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cardoso-Moreira and Long 2010</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cardoso-Moreira <em>et al.</em> 2012</a>; <a id="xref-ref-56-13" class="xref-bibr" href="#ref-56">Miller <em>et al.</em> 2016b</a>). Large CNVs have been found to frequently form between sister chromatids and are flanked by transposons, suggesting that TEs may play a key role in <em>de novo</em> CNV formation (<a id="xref-ref-56-14" class="xref-bibr" href="#ref-56">Miller <em>et al.</em> 2016b</a>). Smaller CNVs that average 50–100 bp have also been identified in Drosophila, and these are associated with replication timing, but not with TEs (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cardoso-Moreira <em>et al.</em> 2011</a>). It is not known whether functional SC is critical for CNV formation either between sisters or homologs.</p><p id="p-42">Here, large CNVs were identified by plotting depth of coverage for all chromosome arms for all individuals from <em>c(3)G<sup>68</sup>/+</em>, <em>c(3)G<sup>68</sup></em>, or corolla<sup><em>129</em></sup> females and revealed 5 total events. No <em>de novo</em> CNV events were observed in offspring from <em>c(3)G<sup>68</sup></em>/+ females, 4 events were recovered in offspring from <em>c(3)G<sup>68</sup></em> females, and 1 event was seen in an offspring of a corolla<sup><em>129</em></sup> female (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, Table S9). Among the 4 events recovered from the <em>c(3)G<sup>68</sup></em> females, one was shared among multiple siblings—a 223-kb deletion of chromosome <em>2R</em> involving 27 genes, which was recovered from 14 male offspring from females #4, 5, and 6 (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>). That this event was observed in individuals from multiple crosses of individual males and females makes it likely that the deletion occurred at least two generations prior and did not have a significant impact on the reproductive fitness of those individuals carrying it.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/525/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Copy-number variants recovered in this study (details are in Table S9). Gray shading indicates heterochromatic regions and &#xA9; represents the centromere. (A) 223-kb deletion shared among 14 males from 3 different females that likely occurred at least two generations prior. (B) A complex 27-kb duplication and 13-kb deletion at the white locus that was recovered in a single offspring and is likely to be de novo based on sibling haplotypes lacking the rearrangement. (C, D) An 856-kb duplication identified in a single male from a c(3)G68 female has identical start and end coordinates as an 856-kb deletion recovered in a single male from a corolla129 female and is identical to an 856-kb duplication recovered in a single male from a wild-type female from a prior study (Miller et al. 2016b). (E) A large 4.17-Mb duplication observed in a single individual that is likely mosaic based on its lower log2 ratio of 0.25." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-677979155" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Copy-number variants recovered in this study (details are in Table S9). Gray shading indicates heterochromatic regions and &#xA9; represents the centromere. (A) 223-kb deletion shared among 14 males from 3 different females that likely occurred at least two generations prior. (B) A complex 27-kb duplication and 13-kb deletion at the &lt;em&gt;white&lt;/em&gt; locus that was recovered in a single offspring and is likely to be &lt;em&gt;de novo&lt;/em&gt; based on sibling haplotypes lacking the rearrangement. (C, D) An 856-kb duplication identified in a single male from a &lt;em&gt;c(3)G&lt;sup&gt;68&lt;/sup&gt;&lt;/em&gt; female has identical start and end coordinates as an 856-kb deletion recovered in a single male from a corolla&lt;sup&gt;&lt;em&gt;129&lt;/em&gt;&lt;/sup&gt; female and is identical to an 856-kb duplication recovered in a single male from a wild-type female from a prior study (Miller &lt;em&gt;et al.&lt;/em&gt; 2016b). (E) A large 4.17-Mb duplication observed in a single individual that is likely mosaic based on its lower log&lt;sub&gt;2&lt;/sub&gt; ratio of 0.25.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/525/F3.medium.gif" width="426" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/2/525/F3.medium.gif" width="426" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/525/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/525/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495646" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-43" class="first-child">Copy-number variants recovered in this study (details are in Table S9). Gray shading indicates heterochromatic regions and © represents the centromere. (A) 223-kb deletion shared among 14 males from 3 different females that likely occurred at least two generations prior. (B) A complex 27-kb duplication and 13-kb deletion at the <a href="https://identifiers.org/bioentitylink/FB:FBgn0003996?doi=10.1534/g3.119.400853"><em>white</em></a> locus that was recovered in a single offspring and is likely to be <em>de novo</em> based on sibling haplotypes lacking the rearrangement. (C, D) An 856-kb duplication identified in a single male from a <em>c(3)G<sup>68</sup></em> female has identical start and end coordinates as an 856-kb deletion recovered in a single male from a corolla<sup><em>129</em></sup> female and is identical to an 856-kb duplication recovered in a single male from a wild-type female from a prior study (<a id="xref-ref-56-15" class="xref-bibr" href="#ref-56">Miller <em>et al.</em> 2016b</a>). (E) A large 4.17-Mb duplication observed in a single individual that is likely mosaic based on its lower log<sub>2</sub> ratio of 0.25.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">The remaining CNVs recovered were only observed in single individuals and, based on haplotype analysis, were likely <em>de novo</em> events. One event, a complex <em>de novo</em> CNV involving both a deletion and a duplication was identified at the <a href="https://identifiers.org/bioentitylink/FB:FBgn0003996?doi=10.1534/g3.119.400853"><em>white</em></a> (<em>w</em>) locus on chromosome <em>X</em> in one male from a <em>c(3)G<sup>68</sup></em> homozygous female (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). This event was mediated by unequal crossing over between <em>roo</em> elements. Previous work describing ectopic recombination in <em>D. melanogaster</em> focused on unequal exchange between <em>roo</em> elements at the <em>w</em> locus, similar to the event observed here (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Goldberg <em>et al.</em> 1983</a>).</p><p id="p-45">Two <em>de novo</em> CNVs (one duplication and one deletion) occurred at the exact same position on chromosome <em>2R</em> in two individuals. The duplication was recovered from the offspring of a <em>c(3)G<sup>68</sup></em> female, while the deletion was recovered from the offspring of a corolla<sup><em>129</em></sup> female (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C,D</a>). This 856-kb event includes 107 genes and is flanked on both sides by a <em>hobo</em> element. Remarkably, a CNV with the exact same breakpoints was identified in a previous study of individuals from wild-type females (<a id="xref-ref-56-16" class="xref-bibr" href="#ref-56">Miller <em>et al.</em> 2016b</a>). All three of these CNVs were <em>de novo</em> events validated using the haplotypes of the siblings that did not carry the CNV. All three crosses were between individual males and females and multiple genetic backgrounds are involved (<em>w<sup>1118</sup></em>, Canton-S, and the undefined corolla<sup><em>129</em></sup> background), thus these CNVs are not variants segregating at low frequency in the population and are recurrent <em>de novo</em> events.</p><p id="p-46">The final CNV observed, a 4.2-Mb duplication on chromosome <em>2L</em> not flanked by a TE or low-complexity sequence, was recovered in a single male from a <em>c(3)G<sup>68</sup></em> female (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3E</a>). Analysis of read pairs revealed that this was a tandem duplication, with reads mapping to the proximal end of the duplication linked to reads mapping to the distal end of the duplication. The log<sub>2</sub> depth-of-coverage ratio for this interval is 1.25, 0.25 higher than expected for a diploid and less than the log<sub>2</sub> depth-of-coverage ratio of 1.5 that would be seen in an autosomal duplication occurring before the first mitotic division. Thus, this duplication is present in half the cells in the individual sequenced and likely occurred during the first mitotic division, possibly as a consequence of a re-replication event that was then repaired by recombination between the duplicated segments (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Green <em>et al.</em> 2010</a>). It is notable that the fly was able to tolerate such a large duplication, involving 513 genes, present in half of all cells. Although the possibility that there was selection against cells carrying the large duplication cannot be excluded, a log<sub>2</sub> depth-of-coverage ratio of 1.25 does strongly suggest there was limited selection against those cells with the duplication. If selection was acting strongly on these cells the log<sub>2</sub> ratio would fall below 1.25 and perhaps become undetectable.</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-47">In this study, whole-genome sequencing was used to evaluate individual offspring from females either heterozygous or homozygous for loss-of-function alleles of two different SC components, <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> and <a href="https://identifiers.org/bioentitylink/FB:FBgn0267967?doi=10.1534/g3.119.400853"><em>corolla</em></a>. Loss-of-function variants in either gene prevent the construction of full-length SC and reduce the number of meiotic DSBs to approximately 20% of wild type in <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> and 40% of wild type in <a href="https://identifiers.org/bioentitylink/FB:FBgn0267967?doi=10.1534/g3.119.400853"><em>corolla</em></a> (<a id="xref-ref-53-7" class="xref-bibr" href="#ref-53">Mehrotra and McKim 2006</a>; <a id="xref-ref-11-8" class="xref-bibr" href="#ref-11">Collins <em>et al.</em> 2014</a>). Previous studies have shown that these DSBs are rarely repaired as COs (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Gowen 1933</a>; <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Hall 1972</a>; <a id="xref-ref-11-9" class="xref-bibr" href="#ref-11">Collins <em>et al.</em> 2014</a>), therefore how DSBs are repaired in the absence of SC remains an open question.</p><p id="p-48">While evaluating the results of this study, several possible confounding factors should be kept in mind. First, as not all experiments were done at the same time, subtle differences in food, humidity, or temperature could affect these results. Next, the number of females from which offspring were collected from was not balanced. For example, while 96 individuals were taken from 10 <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> females, 40 males were taken from 2 <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a><em>/+</em> females. This could result in subtle female-specific effects being amplified in the results. Finally, offspring for sequencing were collected between days 12 and 15 post-fertilization and flies which would have eclosed after day 15 were not collected. Previous work has shown that maternal age affects CO events (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Hunter <em>et al.</em> 2016</a>), and it is not unreasonable to assume that age may have an impact on other factors such as the formation of CNVs and movement of TEs.</p><p id="p-49">CO and NCOGC events were identified in males from <em>c(3)G<sup>68</sup></em>/+ females and compared to wild type (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>). In agreement with previous studies of crossing over in <em>c(3)G<sup>68</sup></em>/+ (<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Hall 1972</a>), the frequency and distribution of events identified in this study appears similar to wild type, although the small sample size may mask subtle differences in the heterozygous genotype. Next, among 93 males from <em>c(3)G<sup>68</sup></em> homozygous females, no CO events and a single NCOGC event were identified. Recovering no CO events was not unexpected based on prior studies of <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> homozygous females (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Gowen 1933</a>; <a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">Hall 1972</a>). The single NCOGC recovered was defined by two polymorphisms and validated using PCR and Sanger sequencing. While this event is likely real and not an artifact of sequencing or alignment, it is not possible to definitively determine whether this event occurred during meiosis or was instead a pre-meiotic conversion event.</p><p id="p-50">To determine the fate of DSBs made in females unable to construct full-length SC, novel deletion polymorphisms were identified in wild type, <em>c(3)G<sup>68</sup></em>/+, <em>c(3)G<sup>68</sup></em>, and corolla<sup><em>129</em></sup> offspring. Comparing the absolute number of deletions between genotypes is not accurate as DSBs in SC mutants are made at a fraction of the wild-type rate, thus a better comparator is deletions per DSB. When analyzed in this way, approximately 10% of DSBs made in individuals from <em>c(3)G<sup>68</sup></em> homozygous females resulted in a novel deletion—significantly more than the 1% observed in the wild-type and <em>c(3)G<sup>68</sup></em>/<em>+</em> datasets—while only 3% of DSBs in <a href="https://identifiers.org/bioentitylink/FB:FBgn0267967?doi=10.1534/g3.119.400853"><em>corolla</em></a> homozygous females were recovered as deletions, an observation not different from wild type.</p><p id="p-51">The increased frequency of <em>de novo</em> deletions in offspring from <em>c(3)G<sup>68</sup></em> homozygous females does not explain the fate of the remaining 90% of DSBs made in these flies. <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Sandler (1965)</a> used a ring chromosome assay in a <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> background to determine whether these DSBs are repaired by sister chromatid exchange. In that assay, females carrying one ring <em>X</em> chromosome and one normal rod <em>X</em> chromosome are constructed. The absence of any exchange yields an equal number of progeny with a ring or a rod chromosome. If exchange occurs between homologs (between the ring and the rod chromosome), single CO events will result in acentric and dicentric chromosomes that will not segregate properly, and because both chromosomes will be equally affected, an equal but decreased number of ring and rod chromosomes will be recovered. In the absence of homologous recombination but in the presence of sister chromatid recombination, rod chromosomes will be transmitted at a normal level, since sister chromatid exchange events on a rod chromosome will not affect their structure or segregation. Single sister chromatid exchange events on a ring chromosome will result in a large dicentric ring that will not segregate properly, thus the number of progeny inheriting a ring will be lower than the number of rod chromosomes recovered.</p><p id="p-52">Sandler found that in the <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> mutant background, rings are recovered less frequently than the homologous rod <em>X</em> chromosome, demonstrating that sister chromatid exchange is occurring in the absence of C(3)G (<a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">Sandler 1965</a>). Separately, <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Hall (1977)</a> was able to recover <em>Bar</em> revertants from <em>FM7/+</em>; <em>c(3)G<sup>68</sup></em> females which, in the absence of homologous recombination, occur through unequal exchange between sister chromatids (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Hall 1977</a>). Thus, taken together, it is most likely that the majority of DSBs in SC-deficient flies are repaired either by a high-fidelity repair process such as canonical NHEJ or by sister chromatid repair.</p><p id="p-53">These results also help clarify the role the SC plays in determining which chromatid is used for DSB repair. During mitosis, an intersister bias exists as the sister chromatid is preferred for repair. Alternatively, there is interhomolog bias during meiosis, as the homolog is preferred for DSB repair. How the cell determines which chromatid to use for repair is an area of active research. In <em>S. cerevisiae</em>, DMC1 (the <em>E. coli</em> RecA homolog) promotes interhomolog DSB repair and Dmc1-deficient yeast only undergo intersister DSB repair during meiosis (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bishop <em>et al.</em> 1992</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Schwacha and Kleckner 1997</a>). Both <em>D. melanogaster</em> and <em>C. elegans</em> lack an ortholog of DMC1, thus it is unclear how interhomolog bias is established during meiosis. One hypothesis is that these species no longer need DMC1 because the SC is built prior to the initiation of recombination. The data presented here, along with prior studies showing that sister chromatid repair occurs in the absence of SC, support the hypothesis that the SC plays a critical role in establishing interhomolog bias in Drosophila and that in the absence of SC, repair from the sister chromatid is possible.</p><p id="p-54">Whole-genome sequencing of individual meiotic products allowed for the characterization of other meiotic events as well, such as whether the absence of full-length SC affects the movement of TEs. The observed rate of novel TE insertions in this study was significantly higher in offspring from <em>c(3)G<sup>68</sup></em> females when compared to the other three classes of progeny studied (wild type, <em>c(3)G<sup>68</sup>/+</em>, and corolla<sup><em>129</em></sup>). Surprisingly, the elevated rate in maternally derived <em>c(3)G<sup>68</sup></em> chromosomes was similar to the rate from paternally derived <em>X</em> chromosomes, which came from males with two wild-type copies of <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a>. The increased rate of <em>de novo</em> transposition in an SC mutant may provide new clues as to other roles that SC components might play in facilitating or preventing the movement of TEs.</p><p id="p-55">This increased rate of novel TE insertion in <em>c(3)G<sup>68</sup></em> mutants could be explained by a model in which C(3)G prevents mobilized TEs from inserting into genomic DNA. In the absence of C(3)G, a greater number of TEs may be available to insert into nuclear DNA. A higher number of active TEs may also explain why the rate of TE insertions was similar on <em>X</em> chromosomes derived from wild-type males, but this would require that TE insertions occur post-fertilization.</p><p id="p-56">Alternatively, the increased rate of TE insertion in <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> females could be explained by differences in genetic background leading to an increased rate of transposition (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Kidwell <em>et al.</em> 1977</a>). Previous studies have reported “bursts” of TE insertions from a specific TE class and attributed the observation to differences in genetic background (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Pasyukova and Nuzhdin 1993</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Page <em>et al.</em> 2007</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Guerreiro 2011</a>). It is worth noting that 20 of the novel insertions identified in the homozygous <em>c(3)G<sup>68</sup></em> dataset were <em>doc</em> elements, which seems to support the idea that different genetic backgrounds might exhibit different rates of transposition. Although, the genetic background in these experiments was somewhat controlled as females both heterozygous and homozygous for <em>c(3)G<sup>68</sup></em> were heterozygous for the same <em>w<sup>1118</sup></em> and Canton-S <em>X</em> and <em>2<sup>nd</sup></em> chromosomes, which were from the same stocks used in the wild-type experiment (<a id="xref-ref-56-17" class="xref-bibr" href="#ref-56">Miller <em>et al.</em> 2016b</a>). These two stocks differed in that females heterozygous for <em>c(3)G<sup>68</sup></em> carried one copy of a <em>w<sup>1118</sup> 3<sup>rd</sup></em> chromosome, while those homozygous for <em>c(3)G<sup>68</sup></em> did not. Although the fact that these chromosomes have been used in previous experiments might reduce the contribution of genetic background to the elevated TE insertion rate, it may not completely eliminate it. The background of <a href="https://identifiers.org/bioentitylink/FB:FBgn0267967?doi=10.1534/g3.119.400853"><em>corolla</em></a> mutants was not controlled. A unifying explanation may be that <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> itself plays a previously unappreciated role in the prevention of TE movement and that this is separate from the role, if any, played by fully functional SC.</p><p id="p-57">Analysis of reads spanning <em>de novo</em> TE insertion sites revealed that two of the 16 total <em>de novo</em> TE insertions identified on the <em>X</em> chromosome were mosaic. Approximately 50% of reads at the TE insertion site spanned the junction and did not contain TE sequence, suggesting that the insertion occurred after the first, but before the second mitotic cell division. Both of the likely mosaic insertions were <em>doc</em> elements, with one occurring on a maternally inherited <em>X</em> chromosome and the other on a paternally inherited <em>X</em>. Removing both of the insertions does not significantly alter the overall rate of TE insertions per arm per meiosis calculated using the observed data, but does decrease the rate of insertions observed on the paternally inherited <em>X</em> chromosome from 0.64 to 0.48, suggesting a <em>de novo</em> insertion on the paternal <em>X</em> chromosome would be recovered approximately 1 in every 8 meioses instead of 1 in every 6.</p><p id="p-58">The increased rate of TE movement in <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> homozygotes suggests that SC components may play a role in mediating TE movement. While the tripartite structure of the SC is widely conserved across species, the amino acid sequence of the components themselves are not (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Handel and Schimenti 2010</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Fraune <em>et al.</em> 2012</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">2013</a>). Indeed, the rate of the amino acid change of SC components makes it challenging to find homologs of the transverse filament protein <a href="http://flybase.org/reports/FBgn0000246"><em>c(3)G</em></a> and other previously identified <em>D. melanogaster</em> SC components even in other species of Drosophila (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Hemmer and Blumenstiel 2016</a>). At a larger evolutionary scale, there is little sequence homology among lateral element, transverse filament, or central element components of <em>C. elegans</em>, <em>D. melanogaster</em>, and <em>M. musculus</em>, suggesting the rapid rate of evolution is not limited to the Drosophila species group (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Fraune <em>et al.</em> 2016</a>). This is unlike components of the recombination machinery, which associate with the SC during meiosis and are highly conserved at the amino acid level (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Zalevsky <em>et al.</em> 1999</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Villeneuve and Hillers 2001</a>). Thus, it is reasonable to posit that the rapid rate of evolution observed for SC components across species and organisms may in part be a response to the ongoing evolution of TEs.</p><p id="p-59">The recovery of large TE-mediated CNVs in females unable to construct SC demonstrates that these types of CNVs can occur independent of normal meiotic synapsis and DSB formation, perhaps depending only on the presence of a chromosome axis. Of course, it is also possible the events recovered here, including the events identified in the wild-type dataset, occurred during mitosis. While this study only sought to identify large CNVs, it is likely that many smaller CNVs occurred which were not identified. Indeed, previous studies in Drosophila using both microarray and whole-genome sequencing have shown that both small and large CNVs are segregating in wild populations (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Emerson <em>et al.</em> 2008</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cridland and Thornton 2010</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Cardoso-Moreira <em>et al.</em> 2011</a>, <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">2012</a>), however these studies did not evaluate the frequency of TE-mediated events because they were done using microarrays or filtered reads that partially mapped to TE sequences.</p><p id="p-60">That two different TE-mediated CNV events (a deletion and a duplication) in two different genetic backgrounds were recovered at the exact same coordinates as a duplication observed in wild-type individuals was surprising and suggests that the rate of CNV formation and the ability to tolerate a large CNV may not be uniform across the genome. As would be expected in mutants with defective homologous chromosome pairing, all four TE-mediated CNV events recovered in this study appear, based on allele frequency and TE positioning, to be events between sister chromatids and not between homologous chromosomes.</p><p id="p-61">Overall, this work demonstrates the wide range of insights that can be gained by analysis of individual meiotic products from both wild-type and mutant backgrounds. From this work, additional testable questions arise. For example, what happens to DSBs in SC-deficient backgrounds in the absence of the NHEJ machinery? And is an increased rate of TE movement observed in Drosophila SC mutants other than the transverse filament protein C(3)G, such as C(2)M or Cona? If it is simply the result of a deficiency of C(3)G, then the question arises of whether homologs of C(3)G, such as ZIP1 and SYCP1, also play a role in mediating TE movement during meiosis and through what mechanism. Future work may address these questions at a larger scale with additional detail as the cost of sequencing continues to fall and tools and techniques for analyzing large datasets such as this improve.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-62">I thank Angela Miller for editorial and figure preparation assistance; the Stowers Institute Molecular Biology core for expert assistance with DNA sequencing; Nazanin Yeganeh Kazemi and Clarissa Smith for assistance with DNA preparation and NCOGC validation; Justin Blumenstiel, Cori Cahoon, and Talia Hatkevich for helpful discussion and comments; two anonymous reviewers for helpful comments and suggestions; and R. Scott Hawley for critical feedback and invaluable mentorship. The DNA sequence data described here were supported by funding to Scott Hawley from the Stowers Institute for Medical Research during my work in his lab.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-63">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.10006634">https://doi.org/10.25387/g3.10006634</a>.</p></li><li class="fn" id="fn-2"><p id="p-64"><em>Communicating editor: H. Salz</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-10-19"><span class="received-label">Received </span>October 19, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-12-23"><span class="accepted-label">Accepted </span>December 23, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Miller</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.1" data-doi="10.1093/nar/25.17.3389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>,  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. L.</span> <span class="cit-name-surname">Madden</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Schäffer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zhang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3389</span>–<span class="cit-lpage">3402</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/25.17.3389</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/25.17.3389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9254694&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XU79300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.2" data-doi="10.1371/journal.pgen.1004086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bétermier</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Bertrand</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. S.</span> <span class="cit-name-surname">Lopez</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> Is Non-Homologous End-Joining Really an Inherently Error-Prone Process? Plos Genet 10: e1004086 9.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004086</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D10%26rft.spage%253De1004086%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004086%26rft_id%253Dinfo%253Apmid%252F24453986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004086&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24453986&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.3" data-doi="10.1016/0092-8674(92)90446-J"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>,  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Park</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Xu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Kleckner</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">DMC1: A meiosis-specific yeast homolog of E. coli recA required for recombination, synaptonemal complex formation, and cell cycle progression.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">439</span>–<span class="cit-lpage">456</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0092-8674(92)90446-J</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBishop%26rft.auinit1%253DD.%2BK.%26rft.volume%253D69%26rft.issue%253D3%26rft.spage%253D439%26rft.epage%253D456%26rft.atitle%253DDMC1%253A%2Ba%2Bmeiosis-specific%2Byeast%2Bhomolog%2Bof%2BE.%2Bcoli%2BrecA%2Brequired%2Bfor%2Brecombination%252C%2Bsynaptonemal%2Bcomplex%2Bformation%252C%2Band%2Bcell%2Bcycle%2Bprogression.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252892%252990446-J%26rft_id%253Dinfo%253Apmid%252F1581960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(92)90446-J&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1581960&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HT07800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bosco</span>,  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> Chromosome Pairing: A Hidden Treasure No More. Plos Genet 8: e1002737 3.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.2.525.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boschi</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth">, <span class="cit-name-surname">M.</span>  <span class="cit-name-given-names">Belloni</span></span>, and </li><li><span class="cit-auth"><span class="cit-name-surname">L. G.</span>  <span class="cit-name-given-names">Robbins</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> Genetic evidence that nonhomologous disjunction and meiotic drive are properties of wild-type <em>Drosophila melanogaster</em> male meiosis. Genetics 172: 305–316.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.6" data-doi="10.1126/science.54.1394.252"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bridges</span>,  <span class="cit-name-given-names">C. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1921</span> <span class="cit-article-title">Triploid Intersexes in Drosophila Melanogaster.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">252</span>–<span class="cit-lpage">254</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.54.1394.252</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBridges%26rft.auinit1%253DC.%2BB.%26rft.volume%253D54%26rft.issue%253D1394%26rft.spage%253D252%26rft.epage%253D254%26rft.atitle%253DTRIPLOID%2BINTERSEXES%2BIN%2BDROSOPHILA%2BMELANOGASTER%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.54.1394.252%26rft_id%253Dinfo%253Apmid%252F17769897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJzY2kiO3M6NToicmVzaWQiO3M6MTE6IjU0LzEzOTQvMjUyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi81MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.2.525.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>,  <span class="cit-name-given-names">P. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> The effects of inversions and the <em>c(3)G</em> mutation on intragenic recombination in <em>Drosophila</em>. Genetical Research 19: 129–132. </cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.8" data-doi="10.1186/gb-2012-13-12-r119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cardoso-Moreira</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Arguello</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. G.</span> <span class="cit-name-surname">Clark</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mutation spectrum of Drosophila CNVs revealed by breakpoint sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">R119</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2012-13-12-r119</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D13%26rft.spage%253DR119%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2012-13-12-r119%26rft_id%253Dinfo%253Apmid%252F23259534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2012-13-12-r119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23259534&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.9" data-doi="10.1371/journal.pgen.1002340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cardoso-Moreira</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Emerson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. G.</span> <span class="cit-name-surname">Clark</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Long</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Drosophila Duplication Hotspots Are Associated with Late-Replicating Regions of the Genome.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e1002340</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002340</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DCardoso-Moreira%26rft.auinit1%253DM.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De1002340%26rft.epage%253De1002340%26rft.atitle%253DDrosophila%2Bduplication%2Bhotspots%2Bare%2Bassociated%2Bwith%2Blate-replicating%2Bregions%2Bof%2Bthe%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002340%26rft_id%253Dinfo%253Apmid%252F22072977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22072977&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.10" data-doi="10.1016/j.tig.2010.03.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cardoso-Moreira</span>,  <span class="cit-name-given-names">M. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Long</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mutational bias shaping fly copy number variation: implications for genome evolution.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">247</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2010.03.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DCardoso-Moreira%26rft.auinit1%253DM.%2BM.%26rft.volume%253D26%26rft.issue%253D6%26rft.spage%253D243%26rft.epage%253D247%26rft.atitle%253DMutational%2Bbias%2Bshaping%2Bfly%2Bcopy%2Bnumber%2Bvariation%253A%2Bimplications%2Bfor%2Bgenome%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2010.03.002%26rft_id%253Dinfo%253Apmid%252F20416969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2010.03.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20416969&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278704300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.11" data-doi="10.1534/genetics.114.165290"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>,  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Unruh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. D.</span> <span class="cit-name-surname">Slaughter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Yu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Lake</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Corolla is a novel protein that contributes to the architecture of the synaptonemal complex of Drosophila.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">219</span>–<span class="cit-lpage">228</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.165290</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.165290%26rft_id%253Dinfo%253Apmid%252F24913682%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzEvMjE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi81MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.12" data-doi="10.1093/gbe/evq001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cridland</span>,  <span class="cit-name-given-names">J. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. R.</span> <span class="cit-name-surname">Thornton</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Validation of Rearrangement Break Points Identified by Paired-End Sequencing in Natural Populations of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">101</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evq001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevq001%26rft_id%253Dinfo%253Apmid%252F20333226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evq001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20333226&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Curtis</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Clark</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Chovnick</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Bender</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Molecular analysis of recombination events in Drosophila.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">653</span>–<span class="cit-lpage">661</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCurtis%26rft.auinit1%253DD.%26rft.volume%253D122%26rft.issue%253D3%26rft.spage%253D653%26rft.epage%253D661%26rft.atitle%253DMolecular%2Banalysis%2Bof%2Brecombination%2Bevents%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Apmid%252F2503419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIyLzMvNjUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi81MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.14" data-doi="10.1534/genetics.110.119115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dakshinamurthy</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Nyswaner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Farabaugh</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Garfinkel</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BUD22 Affects Ty1 Retrotransposition and Ribosome Biogenesis in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1193</span>–<span class="cit-lpage">1205</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.119115</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.119115%26rft_id%253Dinfo%253Apmid%252F20498295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS80LzExOTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzUyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.15" data-doi="10.1146/annurev-genet-110711-155423"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Massy</span>,  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Initiation of Meiotic Recombination: How and Where? Conservation and Specificities Among Eukaryotes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">563</span>–<span class="cit-lpage">599</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-genet-110711-155423</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D47%26rft.spage%253D563%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genet-110711-155423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genet-110711-155423&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.16" data-doi="10.1186/1759-8753-2-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eickbush</span>,  <span class="cit-name-given-names">M. T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. H.</span> <span class="cit-name-surname">Eickbush</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Retrotransposition of R2 elements in somatic nuclei during the early development of Drosophila.</span> <abbr class="cit-jnl-abbrev">Mobile Dna-uk</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1759-8753-2-11</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMob%2BDNA%26rft.aulast%253DEickbush%26rft.auinit1%253DM.%2BT.%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D11%26rft.atitle%253DRetrotransposition%2Bof%2BR2%2Belements%2Bin%2Bsomatic%2Bnuclei%2Bduring%2Bthe%2Bearly%2Bdevelopment%2Bof%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1759-8753-2-11%26rft_id%253Dinfo%253Apmid%252F21958913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1759-8753-2-11&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21958913&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.17" data-doi="10.1126/science.1158078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Emerson</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cardoso-Moreira</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. O.</span> <span class="cit-name-surname">Borevitz</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Long</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Natural Selection Shapes Genome-Wide Patterns of Copy-Number Polymorphism in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>: <span class="cit-fpage">1629</span>–<span class="cit-lpage">1631</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1158078</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1158078%26rft_id%253Dinfo%253Apmid%252F18535209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjAvNTg4My8xNjI5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi81MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.18" data-doi="10.1093/bioinformatics/btu314"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Faust</span>,  <span class="cit-name-given-names">G. G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. M.</span> <span class="cit-name-surname">Hall</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">SAMBLASTER: fast duplicate marking and structural variant read extraction.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2503</span>–<span class="cit-lpage">2505</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu314</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu314%26rft_id%253Dinfo%253Apmid%252F24812344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu314&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24812344&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000342912400059&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.19" data-doi="10.1093/nar/28.13.2585"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feldmann</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Schmiemann</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Goedecke</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Reichenberger</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pfeiffer</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">DNA double-strand break repair in cell-free extracts from Ku80-deficient cells: implications for Ku serving as an alignment factor in non-homologous DNA end joining.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2585</span>–<span class="cit-lpage">2596</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/28.13.2585</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DFeldmann%26rft.auinit1%253DE.%26rft.volume%253D28%26rft.issue%253D13%26rft.spage%253D2585%26rft.epage%253D2596%26rft.atitle%253DDNA%2Bdouble-strand%2Bbreak%2Brepair%2Bin%2Bcell-free%2Bextracts%2Bfrom%2BKu80-deficient%2Bcells%253A%2Bimplications%2Bfor%2BKu%2Bserving%2Bas%2Ban%2Balignment%2Bfactor%2Bin%2Bnon-homologous%2BDNA%2Bend%2Bjoining%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.13.2585%26rft_id%253Dinfo%253Apmid%252F10871410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/28.13.2585&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10871410&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088070700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.20" data-doi="10.1073/pnas.1206875109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fraune</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Alsheimer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-N.</span> <span class="cit-name-surname">Volff</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Busch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Fraune</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Hydra meiosis reveals unexpected conservation of structural synaptonemal complex proteins across metazoans.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">16588</span>–<span class="cit-lpage">16593</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1206875109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1206875109%26rft_id%253Dinfo%253Apmid%252F23012415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzQxLzE2NTg4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi81MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.21" data-doi="10.1534/genetics.113.156679"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fraune</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Brochier-Armanet</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Alsheimer</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Benavente</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Phylogenies of Central Element Proteins Reveal the Dynamic Evolutionary History of the Mammalian Synaptonemal Complex: Ancient and Recent Components.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">781</span>–<span class="cit-lpage">793</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.156679</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.156679%26rft_id%253Dinfo%253Apmid%252F24026100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzMvNzgxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi81MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.22" data-doi="10.1007/s00412-016-0583-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fraune</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Brochier-Armanet</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Alsheimer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-N.</span> <span class="cit-name-surname">Volff</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Schücker</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Evolutionary history of the mammalian synaptonemal complex.</span> <abbr class="cit-jnl-abbrev">Chromosoma</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">360</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00412-016-0583-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosoma%26rft.volume%253D125%26rft.spage%253D355%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00412-016-0583-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00412-016-0583-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.23" data-doi="10.1073/pnas.80.16.5017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldberg</span>,  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-Y.</span> <span class="cit-name-surname">Sheen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. J.</span> <span class="cit-name-surname">Gehring</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Green</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Unequal crossing-over associated with asymmetrical synapsis between nomadic elements in the <em>Drosophila melanogaster</em> genome.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">5017</span>–<span class="cit-lpage">5021</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.80.16.5017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DGoldberg%26rft.auinit1%253DM.%2BL.%26rft.volume%253D80%26rft.issue%253D16%26rft.spage%253D5017%26rft.epage%253D5021%26rft.atitle%253DUnequal%2Bcrossing-over%2Bassociated%2Bwith%2Basymmetrical%2Bsynapsis%2Bbetween%2Bnomadic%2Belements%2Bin%2Bthe%2BDrosophila%2Bmelanogaster%2Bgenome%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.80.16.5017%26rft_id%253Dinfo%253Apmid%252F16593354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODAvMTYvNTAxNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvNTI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.24" data-doi="10.1371/journal.pbio.1000520"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldfarb</span>,  <span class="cit-name-given-names">T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lichten</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Frequent and efficient use of the sister chromatid for DNA double-strand break repair during budding yeast meiosis.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e1000520</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.1000520</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DGoldfarb%26rft.auinit1%253DT.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253De1000520%26rft.epage%253De1000520%26rft.atitle%253DFrequent%2Band%2Befficient%2Buse%2Bof%2Bthe%2Bsister%2Bchromatid%2Bfor%2BDNA%2Bdouble-strand%2Bbreak%2Brepair%2Bduring%2Bbudding%2Byeast%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000520%26rft_id%253Dinfo%253Apmid%252F20976044%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000520&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20976044&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.25" data-doi="10.1002/jez.1400650106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gowen</span>,  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1933</span> <span class="cit-article-title">Meiosis as a genetic character in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">J. Exp. Zool.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">106</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/jez.1400650106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BZool.%26rft.volume%253D65%26rft.spage%253D83%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjez.1400650106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jez.1400650106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.26" data-doi="10.1126/science.1190966"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Green</span>,  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. J.</span> <span class="cit-name-surname">Finn</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Li</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Loss of DNA replication control is a potent inducer of gene amplification.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">329</span>: <span class="cit-fpage">943</span>–<span class="cit-lpage">946</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1190966</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGreen%26rft.auinit1%253DB.%2BM.%26rft.volume%253D329%26rft.issue%253D5994%26rft.spage%253D943%26rft.epage%253D946%26rft.atitle%253DLoss%2Bof%2BDNA%2BReplication%2BControl%2BIs%2Ba%2BPotent%2BInducer%2Bof%2BGene%2BAmplification%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1190966%26rft_id%253Dinfo%253Apmid%252F20724634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjkvNTk5NC85NDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzUyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guerreiro</span>,  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">What makes transposable elements move in the Drosophila genome?</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">108</span>: <span class="cit-elocation-id">hdy201189</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.28" data-doi="10.1073/pnas.0708541104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guirouilh-Barbat</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Rass</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Plo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Bertrand</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. S.</span> <span class="cit-name-surname">Lopez</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Defects in XRCC4 and KU80 differentially affect the joining of distal nonhomologous ends.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">20902</span>–<span class="cit-lpage">20907</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0708541104</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0708541104%26rft_id%253Dinfo%253Apmid%252F18093953%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzUyLzIwOTAyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi81MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Chromosome segregation influenced by two alleles of the meiotic mutant c(3)G in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">400</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHall%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D71%26rft.issue%253D3%26rft.spage%253D367%26rft.epage%253D400%26rft.atitle%253DCHROMOSOME%2BSEGREGATION%2BINFLUENCED%2BBY%2BTWO%2BALLELES%2BOF%2BTHE%2BMEIOTIC%2BMUTANT%2Bc%25283%2529G%2BIN%2BDROSOPHILA%2BMELANOGASTER%26rft_id%253Dinfo%253Apmid%252F4624918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzEvMy8zNjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzUyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Recombination influenced by two alleles of the meiotic mutant c(3)G.</span> <abbr class="cit-jnl-abbrev">Drosoph. Inf. Serv.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">143</span>–<span class="cit-lpage">144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDrosoph.%2BInf.%2BServ.%26rft.volume%253D52%26rft.spage%253D143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.31" data-doi="10.1038/nrg2723"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Handel</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Schimenti</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetics of mammalian meiosis: regulation, dynamics and impact on fertility.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">124</span>–<span class="cit-lpage">136</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2723</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DHandel%26rft.auinit1%253DM.%2BA.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D124%26rft.epage%253D136%26rft.atitle%253DGenetics%2Bof%2Bmammalian%2Bmeiosis%253A%2Bregulation%252C%2Bdynamics%2Band%2Bimpact%2Bon%2Bfertility.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2723%26rft_id%253Dinfo%253Apmid%252F20051984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2723&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20051984&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273688200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.32" data-doi="10.1186/s12862-016-0670-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hemmer</span>,  <span class="cit-name-given-names">L. W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Blumenstiel</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Holding it together: rapid evolution and positive selection in the synaptonemal complex of Drosophila.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">91</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12862-016-0670-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol.%2BBiol.%26rft.volume%253D16%26rft.spage%253D91%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12862-016-0670-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12862-016-0670-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.33" data-doi="10.1007/s00412-007-0128-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hernández-Hernández</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Rincón-Arano</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Recillas-Targa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Ortiz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Valdes-Quezada</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Differential distribution and association of repeat DNA sequences in the lateral element of the synaptonemal complex in rat spermatocytes.</span> <abbr class="cit-jnl-abbrev">Chromosoma</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">87</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00412-007-0128-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosoma%26rft.stitle%253DChromosoma%26rft.aulast%253DHernandez-Hernandez%26rft.auinit1%253DA.%26rft.volume%253D117%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D87%26rft.atitle%253DDifferential%2Bdistribution%2Band%2Bassociation%2Bof%2Brepeat%2BDNA%2Bsequences%2Bin%2Bthe%2Blateral%2Belement%2Bof%2Bthe%2Bsynaptonemal%2Bcomplex%2Bin%2Brat%2Bspermatocytes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00412-007-0128-2%26rft_id%253Dinfo%253Apmid%252F17968579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00412-007-0128-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17968579&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hilliker</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Harauz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Reaume</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Gray</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Clark</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Meiotic gene conversion tract length distribution within the rosy locus of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">1019</span>–<span class="cit-lpage">1026</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHilliker%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D137%26rft.issue%253D4%26rft.spage%253D1019%26rft.epage%253D1026%26rft.atitle%253DMeiotic%2Bgene%2Bconversion%2Btract%2Blength%2Bdistribution%2Bwithin%2Bthe%2Brosy%2Blocus%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Apmid%252F7982556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNy80LzEwMTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzUyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hinton</span>,  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">Enhancement of recombination associated with the c3G mutant of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">157</span>–<span class="cit-lpage">164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHinton%26rft.auinit1%253DC.%2BW.%26rft.volume%253D53%26rft.issue%253D1%26rft.spage%253D157%26rft.epage%253D164%26rft.atitle%253DENHANCEMENT%2BOF%2BRECOMBINATION%2BASSOCIATED%2BWITH%2BTHE%2Bc3G%2BMUTANT%2BOF%2BDROSOPHILA%2BMELANOGASTER%26rft_id%253Dinfo%253Apmid%252F5900602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI1My8xLzE1NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvNTI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.36" data-doi="10.1093/bioinformatics/btr708"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Myers</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. T.</span> <span class="cit-name-surname">Marth</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">ART: a next-generation sequencing read simulator.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">593</span>–<span class="cit-lpage">594</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr708</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr708%26rft_id%253Dinfo%253Apmid%252F22199392%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr708&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22199392&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300490500023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>,  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Robinson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. L.</span> <span class="cit-name-surname">Aylor</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. D.</span> <span class="cit-name-surname">Singh</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> Genetic Background, Maternal Age, and Interaction Effects Mediate Rates of Crossing Over in <em>Drosophila melanogaster</em> Females. G3 Genes Genomes Genetics 6: 1409–1416.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.2.525.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Itsara</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. M.</span> <span class="cit-name-surname">Cooper</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Baker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Girirajan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Li</span></span></li></ol><cite> <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2009</span> Population analysis of large copy number variants and hotspots of human genetic disease. Am J Hum Genetics 84: 148–161.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.39" data-doi="10.1534/genetics.107.078717"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jeffress</span>,  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Page</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. K.</span> <span class="cit-name-surname">Royer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. D.</span> <span class="cit-name-surname">Belden</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Blumenstiel</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The formation of the central element of the synaptonemal complex may occur by multiple mechanisms: the roles of the N- and C-terminal domains of the Drosophila C(3)G protein in mediating synapsis and recombination.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2445</span>–<span class="cit-lpage">2456</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.078717</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.078717%26rft_id%253Dinfo%253Apmid%252F17947423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzI0NDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzUyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.40" data-doi="10.1093/emboj/19.13.3398"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>,  <span class="cit-name-given-names">R. D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Jasin</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Sister chromatid gene conversion is a prominent double‐strand break repair pathway in mammalian cells.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">3398</span>–<span class="cit-lpage">3407</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/emboj/19.13.3398</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DJohnson%26rft.auinit1%253DR.%2BD.%26rft.volume%253D19%26rft.issue%253D13%26rft.spage%253D3398%26rft.epage%253D3407%26rft.atitle%253DSister%2Bchromatid%2Bgene%2Bconversion%2Bis%2Ba%2Bprominent%2Bdouble-strand%2Bbreak%2Brepair%2Bpathway%2Bin%2Bmammalian%2Bcells%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F19.13.3398%26rft_id%253Dinfo%253Apmid%252F10880452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMTkvMTMvMzM5OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvNTI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Joyce</span>,  <span class="cit-name-given-names">E. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. R.</span> <span class="cit-name-surname">Williams</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Xie</span></span></li></ol><cite>, and C. ting Wu, <span class="cit-pub-date">2012</span> Identification of Genes That Promote or Antagonize Somatic Homolog Pairing Using a High-Throughput FISH–Based Screen. Plos Genet 8: e1002667 20.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.42" data-doi="10.1093/nar/26.23.5333"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kabotyanski</span>,  <span class="cit-name-given-names">E. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Gomelsky</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-O.</span> <span class="cit-name-surname">Han</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. B.</span> <span class="cit-name-surname">Roth</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. D.</span> <span class="cit-name-surname">Stamato</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Double-strand break repair in Ku86- and XRCC4-deficient cells.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">5333</span>–<span class="cit-lpage">5342</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/26.23.5333</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F26.23.5333%26rft_id%253Dinfo%253Apmid%252F9826756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/26.23.5333&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9826756&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077487400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaminker</span>,  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Bergman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Kronmiller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Carlson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Svirskas</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> The transposable elements of the <em>Drosophila melanogaster</em> euchromatin: a genomics perspective. Genome Biol 3: RESEARCH0084.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kidwell</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Kidwell</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sved</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Hybrid Dysgenesis in <em>Drosophila melanogaster</em>: A Syndrome of Aberrant Traits Including Mutation, Sterility and Male Recombination.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">813</span>–<span class="cit-lpage">833</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKidwell%26rft.auinit1%253DM.%2BG.%26rft.volume%253D86%26rft.issue%253D4%26rft.spage%253D813%26rft.epage%253D833%26rft.atitle%253DHYBRID%2BDYSGENESIS%2BIN%2BDROSOPHILA%2BMELANOGASTER%253A%2BA%2BSYNDROME%2BOF%2BABERRANT%2BTRAITS%2BINCLUDING%2BMUTATION%252C%2BSTERILITY%2BAND%2BMALE%2BRECOMBINATION%26rft_id%253Dinfo%253Apmid%252F17248751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiODYvNC84MTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzUyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.45" data-doi="10.1146/annurev-physiol-020911-153342"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lake</span>,  <span class="cit-name-given-names">C. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Hawley</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The Molecular Control of Meiotic Chromosomal Behavior: Events in Early Meiotic Prophase in Drosophila Oocytes.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Physiol.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">425</span>–<span class="cit-lpage">451</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-physiol-020911-153342</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BPhysiology%26rft.stitle%253DAnnual%2BReview%2Bof%2BPhysiology%26rft.aulast%253DLake%26rft.auinit1%253DC.%2BM.%26rft.volume%253D74%26rft.spage%253D425%26rft.epage%253D451%26rft.atitle%253DThe%2Bmolecular%2Bcontrol%2Bof%2Bmeiotic%2Bchromosomal%2Bbehavior%253A%2Bevents%2Bin%2Bearly%2Bmeiotic%2Bprophase%2Bin%2BDrosophila%2Boocytes.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-physiol-020911-153342%26rft_id%253Dinfo%253Apmid%252F22335798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-physiol-020911-153342&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22335798&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000301794000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">Y. G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. H.</span> <span class="cit-name-surname">Langley</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> Transposable elements in natural populations of <em>Drosophila melanogaster</em>. Philosophical Transactions Royal Soc B Biological Sci 365: 1219–1228.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.47" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Handsaker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wysoker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Fennell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ruan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.2.525.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Durbin</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> Fast and accurate short read alignment with Burrows–Wheeler transform. Bioinformatics 25: 1754–1760.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.525.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindsley</span>,  <span class="cit-name-given-names">D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Zimm</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-source">The Genome of Drosophila melanogaster</span>, <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">Cambridge, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.50" data-doi="10.1016/S0960-9822(03)00050-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manheim</span>,  <span class="cit-name-given-names">E. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. S.</span> <span class="cit-name-surname">McKim</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The Synaptonemal Complex Component C(2)M Regulates Meiotic Crossing over in Drosophila.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">276</span>–<span class="cit-lpage">285</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0960-9822(03)00050-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DManheim%26rft.auinit1%253DE.%2BA.%26rft.volume%253D13%26rft.issue%253D4%26rft.spage%253D276%26rft.epage%253D285%26rft.atitle%253DThe%2BSynaptonemal%2Bcomplex%2Bcomponent%2BC%25282%2529M%2Bregulates%2Bmeiotic%2Bcrossing%2Bover%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0960-9822%252803%252900050-2%26rft_id%253Dinfo%253Apmid%252F12593793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0960-9822(03)00050-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12593793&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181075300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.51" data-doi="10.1007/s10577-007-1190-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marcon</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Babak</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Chua</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Hughes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Moens</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">miRNA and piRNA localization in the male mammalian meiotic nucleus.</span> <abbr class="cit-jnl-abbrev">Chromosome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">260</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10577-007-1190-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosome%2Bresearch%2B%253A%2B%2Ban%2Binternational%2Bjournal%2Bon%2Bthe%2Bmolecular%252C%2Bsupramolecular%2Band%2Bevolutionary%2Baspects%2Bof%2Bchromosome%2Bbiology%26rft.stitle%253DChromosome%2BRes%26rft.aulast%253DMarcon%26rft.auinit1%253DE.%26rft.volume%253D16%26rft.issue%253D2%26rft.spage%253D243%26rft.epage%253D260%26rft.atitle%253DmiRNA%2Band%2BpiRNA%2Blocalization%2Bin%2Bthe%2Bmale%2Bmammalian%2Bmeiotic%2Bnucleus.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10577-007-1190-6%26rft_id%253Dinfo%253Apmid%252F18204908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10577-007-1190-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18204908&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253753800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.52" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hanna</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Banks</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sivachenko</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Cibulskis</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: 1297–1303.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.107524.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvNTI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.53" data-doi="10.1371/journal.pgen.0020200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mehrotra</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. S.</span> <span class="cit-name-surname">McKim</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Temporal analysis of meiotic DNA double-strand break formation and repair in Drosophila females.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e200</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.0020200.eor</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DMehrotra%26rft.auinit1%253DS.%26rft.volume%253D2%26rft.issue%253D11%26rft.spage%253De200%26rft.epage%253De200%26rft.atitle%253DTemporal%2Banalysis%2Bof%2Bmeiotic%2BDNA%2Bdouble-strand%2Bbreak%2Bformation%2Band%2Brepair%2Bin%2BDrosophila%2Bfemales.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020200%26rft_id%253Dinfo%253Apmid%252F17166055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17166055&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.2.525.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Takeo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Nandanan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Paulson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. M.</span> <span class="cit-name-surname">Gogol</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A whole-chromosome analysis of meiotic recombination in <em>Drosophila melanogaster</em></span>. G3 (Bethesda). <span class="cit-vol">2</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">260</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.55" data-doi="10.1073/pnas.1601232113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. R.</span> <span class="cit-name-surname">Cook</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Kazemi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. B.</span> <span class="cit-name-surname">Smith</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Cockrell</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">a</span> <span class="cit-article-title">Rare recombination events generate sequence diversity among balancer chromosomes in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">E1352</span>–<span class="cit-lpage">E1361</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1601232113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1601232113%26rft_id%253Dinfo%253Apmid%252F26903656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEzLzEwL0UxMzUyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi81MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.56" data-doi="10.1534/genetics.115.186486"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. B.</span> <span class="cit-name-surname">Smith</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Kazemi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Cockrell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. V.</span> <span class="cit-name-surname">Arvanitakis</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">b</span> <span class="cit-article-title">Whole-Genome Analysis of Individual Meiotic Events in <em>Drosophila melanogaster</em> Reveals That Noncrossover Gene Conversions Are Insensitive to Interference and the Centromere Effect.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">203</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">171</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.186486</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.186486%26rft_id%253Dinfo%253Apmid%252F26944917%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAzLzEvMTU5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi81MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.57" data-doi="10.1101/gad.935001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Page</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Hawley</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">c<em>(3)G</em> encodes a Drosophila synaptonemal complex protein.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">3130</span>–<span class="cit-lpage">3143</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.935001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DPage%26rft.auinit1%253DS.%2BL.%26rft.volume%253D15%26rft.issue%253D23%26rft.spage%253D3130%26rft.epage%253D3143%26rft.atitle%253Dc%25283%2529G%2Bencodes%2Ba%2BDrosophila%2Bsynaptonemal%2Bcomplex%2Bprotein%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.935001%26rft_id%253Dinfo%253Apmid%252F11731477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE1LzIzLzMxMzAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzUyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.58" data-doi="10.4161/fly.4720"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Page</span>,  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Nielsen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Teeter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Lake</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Ong</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A germline clone screen for meiotic mutants in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Fly (Austin)</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">172</span>–<span class="cit-lpage">181</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4161/fly.4720</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFly%2B%2528Austin%2529%26rft.volume%253D1%26rft.spage%253D172%26rft_id%253Dinfo%253Adoi%252F10.4161%252Ffly.4720%26rft_id%253Dinfo%253Apmid%252F18820465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/fly.4720&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18820465&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.59" data-doi="10.1007/BF00277071"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pasyukova</span>,  <span class="cit-name-given-names">E. G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. V.</span> <span class="cit-name-surname">Nuzhdin</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Doc and copia instability in an isogenic <em>Drosophila melanogaster</em> stock.</span> <abbr class="cit-jnl-abbrev">Mol Gen Genetics Mgg</abbr> <span class="cit-vol">240</span>: <span class="cit-fpage">302</span>–<span class="cit-lpage">306</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00277071</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DPasyukova%26rft.auinit1%253DE.%2BG.%26rft.volume%253D240%26rft.issue%253D2%26rft.spage%253D302%26rft.epage%253D306%26rft.atitle%253DDoc%2Band%2Bcopia%2Binstability%2Bin%2Ban%2Bisogenic%2BDrosophila%2Bmelanogaster%2Bstock.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00277071%26rft_id%253Dinfo%253Apmid%252F8394986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00277071&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8394986&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LR35700020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pearlman</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Tsao</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Moens</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Synaptonemal complexes from DNase-treated rat pachytene chromosomes contain (GT)n and LINE/SINE sequences.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">865</span>–<span class="cit-lpage">872</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPearlman%26rft.auinit1%253DR.%2BE.%26rft.volume%253D130%26rft.issue%253D4%26rft.spage%253D865%26rft.epage%253D872%26rft.atitle%253DSynaptonemal%2Bcomplexes%2Bfrom%2BDNase-treated%2Brat%2Bpachytene%2Bchromosomes%2Bcontain%2B%2528GT%2529n%2Band%2BLINE%252FSINE%2Bsequences.%26rft_id%253Dinfo%253Apmid%252F1582563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTMwLzQvODY1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi81MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.61" data-doi="10.1038/nsmb.1641"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rass</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Grabarz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Plo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Gautier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Bertrand</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Role of Mre11 in chromosomal nonhomologous end joining in mammalian cells.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">819</span>–<span class="cit-lpage">824</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nsmb.1641</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DRass%26rft.auinit1%253DE.%26rft.volume%253D16%26rft.issue%253D8%26rft.spage%253D819%26rft.epage%253D824%26rft.atitle%253DRole%2Bof%2BMre11%2Bin%2Bchromosomal%2Bnonhomologous%2Bend%2Bjoining%2Bin%2Bmammalian%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.1641%26rft_id%253Dinfo%253Apmid%252F19633668%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.1641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19633668&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268738700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sandler</span>,  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1965</span> <span class="cit-article-title">The meiotic mechanics of ring chromosomes in female <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Natl. Cancer Inst. Monogr.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">273</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJNCI%2BMonographs%26rft.stitle%253DJ%2BNatl%2BCancer%2BInst%2BMonogr%26rft.aulast%253DSandler%26rft.auinit1%253DL.%26rft.volume%253D18%26rft.spage%253D243%26rft.epage%253D273%26rft.atitle%253DThe%2Bmeiotic%2Bmechanics%2Bof%2Bring%2Bchromosomes%2Bin%2Bfemale%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Apmid%252F5863584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=5863584&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.63" data-doi="10.1016/S0092-8674(00)80378-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwacha</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Kleckner</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Interhomolog Bias during Meiotic Recombination: Meiotic Functions Promote a Highly Differentiated Interhomolog-Only Pathway.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">1123</span>–<span class="cit-lpage">1135</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0092-8674(00)80378-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSchwacha%26rft.auinit1%253DA.%26rft.volume%253D90%26rft.issue%253D6%26rft.spage%253D1123%26rft.epage%253D1135%26rft.atitle%253DInterhomolog%2Bbias%2Bduring%2Bmeiotic%2Brecombination%253A%2Bmeiotic%2Bfunctions%2Bpromote%2Ba%2Bhighly%2Bdifferentiated%2Binterhomolog-only%2Bpathway.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252980378-5%26rft_id%253Dinfo%253Apmid%252F9323140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)80378-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9323140&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XX76800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-10.2.525.64"><div class="cit-metadata unstructured">Smit, A. F. A., R. Hubley, and P. Green RepeatMasker. RepeatMasker Open-4.0. 2013–2015. <a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.65" data-doi="10.1093/bib/bbs017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdóttir</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. T.</span> <span class="cit-name-surname">Robinson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Mesirov</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">192</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bib/bbs017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbs017%26rft_id%253Dinfo%253Apmid%252F22517427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbs017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22517427&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.66" data-doi="10.7554/eLife.28297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Treiber</span>,  <span class="cit-name-given-names">C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Waddell</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Resolving the prevalence of somatic transposition in Drosophila.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">6</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7554/eLife.28297</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.28297%26rft_id%253Dinfo%253Apmid%252F28742021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.28297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28742021&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.67" data-doi="10.4161/epi.4.2.7783"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van der Heijden</span>,  <span class="cit-name-given-names">G. W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bortvin</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Transient relaxation of transposon silencing at the onset of mammalian meiosis.</span> <abbr class="cit-jnl-abbrev">Epigenetics</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">76</span>–<span class="cit-lpage">79</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4161/epi.4.2.7783</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEpigenetics%26rft.aulast%253Dvan%2Bder%2BHeijden%26rft.auinit1%253DG.%2BW.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253D76%26rft.epage%253D79%26rft.atitle%253DTransient%2Brelaxation%2Bof%2Btransposon%2Bsilencing%2Bat%2Bthe%2Bonset%2Bof%2Bmammalian%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fepi.4.2.7783%26rft_id%253Dinfo%253Apmid%252F19252417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/epi.4.2.7783&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19252417&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264745200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.68" data-doi="10.1016/S0092-8674(01)00500-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Villeneuve</span>,  <span class="cit-name-given-names">A. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. J.</span> <span class="cit-name-surname">Hillers</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Whence Meiosis?</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">647</span>–<span class="cit-lpage">650</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0092-8674(01)00500-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DVilleneuve%26rft.auinit1%253DA.%2BM.%26rft.volume%253D106%26rft.issue%253D6%26rft.spage%253D647%26rft.epage%253D650%26rft.atitle%253DWhence%2Bmeiosis%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252801%252900500-1%26rft_id%253Dinfo%253Apmid%252F11572770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(01)00500-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11572770&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171213000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.69" data-doi="10.1093/nar/gkg728"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Perrault</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Takeda</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Qin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Wang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Biochemical evidence for Ku‐independent backup pathways of NHEJ.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">5377</span>–<span class="cit-lpage">5388</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkg728</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DWang%26rft.auinit1%253DH.%26rft.volume%253D31%26rft.issue%253D18%26rft.spage%253D5377%26rft.epage%253D5388%26rft.atitle%253DBiochemical%2Bevidence%2Bfor%2BKu-independent%2Bbackup%2Bpathways%2Bof%2BNHEJ%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg728%26rft_id%253Dinfo%253Apmid%252F12954774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkg728&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12954774&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185353200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.70" data-doi="10.1158/0008-5472.CAN-04-3055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Rosidi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Perrault</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Zhang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">DNA Ligase III as a Candidate Component of Backup Pathways of Nonhomologous End Joining.</span> <abbr class="cit-jnl-abbrev">Cancer Res.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">4020</span>–<span class="cit-lpage">4030</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1158/0008-5472.CAN-04-3055</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BResearch%26rft.stitle%253DCancer%2BRes.%26rft.aulast%253DWang%26rft.auinit1%253DH.%26rft.volume%253D65%26rft.issue%253D10%26rft.spage%253D4020%26rft.epage%253D4030%26rft.atitle%253DDNA%2BLigase%2BIII%2Bas%2Ba%2BCandidate%2BComponent%2Bof%2BBackup%2BPathways%2Bof%2BNonhomologous%2BEnd%2BJoining%26rft_id%253Dinfo%253Adoi%252F10.1158%252F0008-5472.CAN-04-3055%26rft_id%253Dinfo%253Apmid%252F15899791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjEwOiI2NS8xMC80MDIwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi81MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.71" data-doi="10.1534/genetics.108.093385"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Takahashi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Itoh</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Takano-Shimizu</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Molecular spectrum of spontaneous de novo mutations in male and female germline cells of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">181</span>: <span class="cit-fpage">1035</span>–<span class="cit-lpage">1043</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.108.093385</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.093385%26rft_id%253Dinfo%253Apmid%252F19114461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MS8zLzEwMzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzUyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.72" data-doi="10.1093/bioinformatics/btp394"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. H.</span> <span class="cit-name-surname">Schulz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Long</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Apweiler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Ning</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2865</span>–<span class="cit-lpage">2871</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp394</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp394%26rft_id%253Dinfo%253Apmid%252F19561018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp394&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19561018&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F525.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271106800026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.73"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zalevsky</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">MacQueen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Duffy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kemphues</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Villeneuve</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Crossing over during <em>Caenorhabditis elegans</em> meiosis requires a conserved MutS-based pathway that is partially dispensable in budding yeast.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">1271</span>–<span class="cit-lpage">1283</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DZalevsky%26rft.auinit1%253DJ.%26rft.volume%253D153%26rft.issue%253D3%26rft.spage%253D1271%26rft.epage%253D1283%26rft.atitle%253DCrossing%2BOver%2BDuring%2BCaenorhabditis%2Belegans%2BMeiosis%2BRequires%2Ba%2BConserved%2BMutS-Based%2BPathway%2BThat%2BIs%2BPartially%2BDispensable%2Bin%2BBudding%2BYeast%26rft_id%253Dinfo%253Apmid%252F10545458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1My8zLzEyNzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzUyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.525.74"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zickler</span>,  <span class="cit-name-given-names">D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Kleckner</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Recombination, Pairing, and Synapsis of Homologs during Meiosis.</span> <abbr class="cit-jnl-abbrev">Csh Perspect Biol</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">a016626</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/2/525.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/2/515" title="Accelerated Evolution of Limb-Related Gene Hoxd11 in the Common Ancestor of Cetaceans and Ruminants (Cetruminantia)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/2/539" title="Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/2"><span>Volume 10 Issue 2, February 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/2.cover-source.jpg?itok=LSawHOSr" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/525" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qSy-zEbz0tVqLmYi7Q4mAy9eMWxOKxOBMvzPD0AOHt8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/2/525" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/2/525" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/2/525" class="active" data-icon-position="" data-hide-link-title="0">Synaptonemal Complex-Deficient Drosophila melanogaster Females Exhibit Rare DSB Repair Events, Recurrent Copy-Number Variation, and an Increased Rate of de Novo Transposable Element Movement</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495552" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-kKuD-E-rNUFwMkOOf7niIINvr3rvW_XbRppNH7orwPQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/2/525.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/2/525" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ehjl9yHTpt6fXMJUznCMaYfcP2P_-vePbjWyhugIbi0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/2/525" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/2/525.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495552" id="node495552--2" data-pisa="ggg;10/2/525" data-pisa-master="ggg;g3.119.400853" data-apath="/ggg/10/2/525.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Synaptonemal Complex-Deficient <em>Drosophila melanogaster</em> Females Exhibit Rare DSB Repair Events, Recurrent Copy-Number Variation, and an Increased Rate of <em>de Novo</em> Transposable Element Movement</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6096-8601" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Danny E. Miller</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">525-537;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400853 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495552--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Danny E. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Medical Genetics, Department of Medicine, and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Genetic Medicine, Department of Pediatrics, University of Washington, Seattle, Washington 98105, and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Seattle Children’s Hospital, Seattle, Washington 98105</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danny%2BE.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20DE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADanny%2BE.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6096-8601" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Danny E. Miller</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:danny.miller@seattlechildrens.org" class="" data-icon-position="" data-hide-link-title="0">danny.miller@seattlechildrens.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/2/525" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/2/525&amp;title=Synaptonemal%20Complex-Deficient%20Drosophila%20melanogaster%20Females%20Exhibit%20Rare%20DSB%20Repair%20Events%2C%20Recurrent%20Copy-Number%20Variation%2C%20and%20an%20Increased%20Rate%20of%20de%20Novo%20Transposable%20Element%20Movement" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/2/525&amp;title=Synaptonemal%20Complex-Deficient%20Drosophila%20melanogaster%20Females%20Exhibit%20Rare%20DSB%20Repair%20Events%2C%20Recurrent%20Copy-Number%20Variation%2C%20and%20an%20Increased%20Rate%20of%20de%20Novo%20Transposable%20Element%20Movement" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/2/525&amp;title=Synaptonemal%20Complex-Deficient%20Drosophila%20melanogaster%20Females%20Exhibit%20Rare%20DSB%20Repair%20Events%2C%20Recurrent%20Copy-Number%20Variation%2C%20and%20an%20Increased%20Rate%20of%20de%20Novo%20Transposable%20Element%20Movement" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/2/525&amp;text=Synaptonemal%20Complex-Deficient%20Drosophila%20melanogaster%20Females%20Exhibit%20Rare%20DSB%20Repair%20Events%2C%20Recurrent%20Copy-Number%20Variation%2C%20and%20an%20Increased%20Rate%20of%20de%20Novo%20Transposable%20Element%20Movement" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/2/525&amp;title=Synaptonemal%20Complex-Deficient%20Drosophila%20melanogaster%20Females%20Exhibit%20Rare%20DSB%20Repair%20Events%2C%20Recurrent%20Copy-Number%20Variation%2C%20and%20an%20Increased%20Rate%20of%20de%20Novo%20Transposable%20Element%20Movement" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/2/525&amp;t=Synaptonemal%20Complex-Deficient%20Drosophila%20melanogaster%20Females%20Exhibit%20Rare%20DSB%20Repair%20Events%2C%20Recurrent%20Copy-Number%20Variation%2C%20and%20an%20Increased%20Rate%20of%20de%20Novo%20Transposable%20Element%20Movement" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/2/525&amp;title=Synaptonemal%20Complex-Deficient%20Drosophila%20melanogaster%20Females%20Exhibit%20Rare%20DSB%20Repair%20Events%2C%20Recurrent%20Copy-Number%20Variation%2C%20and%20an%20Increased%20Rate%20of%20de%20Novo%20Transposable%20Element%20Movement" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/2/525&amp;title=Synaptonemal%20Complex-Deficient%20Drosophila%20melanogaster%20Females%20Exhibit%20Rare%20DSB%20Repair%20Events%2C%20Recurrent%20Copy-Number%20Variation%2C%20and%20an%20Increased%20Rate%20of%20de%20Novo%20Transposable%20Element%20Movement" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495552/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495552" id="node495552--3" data-pisa="ggg;10/2/525" data-pisa-master="ggg;g3.119.400853" data-apath="/ggg/10/2/525.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Synaptonemal Complex-Deficient <em>Drosophila melanogaster</em> Females Exhibit Rare DSB Repair Events, Recurrent Copy-Number Variation, and an Increased Rate of <em>de Novo</em> Transposable Element Movement</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6096-8601" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Danny E. Miller</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">525-537;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400853 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495552--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Danny E. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Medical Genetics, Department of Medicine, and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Genetic Medicine, Department of Pediatrics, University of Washington, Seattle, Washington 98105, and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Seattle Children’s Hospital, Seattle, Washington 98105</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danny%2BE.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20DE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADanny%2BE.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6096-8601" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Danny E. Miller</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:danny.miller@seattlechildrens.org" class="" data-icon-position="" data-hide-link-title="0">danny.miller@seattlechildrens.org</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=525&amp;rft.epage=537&amp;rft.atitle=Synaptonemal%20Complex-Deficient%20Drosophila%20melanogaster%20Females%20Exhibit%20Rare%20DSB%20Repair%20Events%2C%20Recurrent%20Copy-Number%20Variation%2C%20and%20an%20Increased%20Rate%20of%20de%20Novo%20Transposable%20Element%20Movement&amp;rft.volume=10&amp;rft.issue=2&amp;rft.date=2020-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Miller%2C+Danny+E."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495552/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495552/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495552/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495552/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495552/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495552/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495552/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495552/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495552/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495552/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495552/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495552/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/2/525" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/2/525.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/2/525.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/2/525.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
