<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/2/721" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/2/721.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/2/721.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/2/721.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/2/721.atom" />
<meta name="HW.pisa" content="ggg;10/2/721" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Prevalence and Implications of Contamination in Public Genomic Resources: A Case Study of 43 Reference Arthropod Assemblies" />
<meta name="DC.Identifier" content="10.1534/g3.119.400758" />
<meta name="DC.Date" content="2020-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Francois et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Thanks to huge advances in sequencing technologies, genomic resources are increasingly being generated and shared by the scientific community. The quality of such public resources are therefore of critical importance. Errors due to contamination are particularly worrying; they are widespread, propagate across databases, and can compromise downstream analyses, especially the detection of horizontally-transferred sequences. However we still lack consistent and comprehensive assessments of contamination prevalence in public genomic data. Here we applied a standardized procedure for foreign sequence annotation to 43 published arthropod genomes from the widely used Ensembl Metazoa database. This method combines information on sequence similarity and synteny to identify contaminant and putative horizontally-transferred sequences in any genome assembly, provided that an adequate reference database is available. We uncovered considerable heterogeneity in quality among arthropod assemblies, some being devoid of contaminant sequences, whereas others included hundreds of contaminant genes. Contaminants far outnumbered horizontally-transferred genes and were a major confounder of their detection, quantification and analysis. We strongly recommend that automated standardized decontamination procedures be systematically embedded into the submission process to genomic databases." />
<meta name="DC.Contributor" content="Clementine M. Francois" />
<meta name="DC.Contributor" content="Faustine Durand" />
<meta name="DC.Contributor" content="Emeric Figuet" />
<meta name="DC.Contributor" content="Nicolas Galtier" />
<meta name="article:published_time" content="2020-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Prevalence and Implications of Contamination in Public Genomic Resources: A Case Study of 43 Reference Arthropod Assemblies" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Thanks to huge advances in sequencing technologies, genomic resources are increasingly being generated and shared by the scientific community. The quality of such public resources are therefore of critical importance. Errors due to contamination are particularly worrying; they are widespread, propagate across databases, and can compromise downstream analyses, especially the detection of horizontally-transferred sequences. However we still lack consistent and comprehensive assessments of contamination prevalence in public genomic data. Here we applied a standardized procedure for foreign sequence annotation to 43 published arthropod genomes from the widely used Ensembl Metazoa database. This method combines information on sequence similarity and synteny to identify contaminant and putative horizontally-transferred sequences in any genome assembly, provided that an adequate reference database is available. We uncovered considerable heterogeneity in quality among arthropod assemblies, some being devoid of contaminant sequences, whereas others included hundreds of contaminant genes. Contaminants far outnumbered horizontally-transferred genes and were a major confounder of their detection, quantification and analysis. We strongly recommend that automated standardized decontamination procedures be systematically embedded into the submission process to genomic databases.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/02/01" />
<meta name="citation_mjid" content="ggg;10/2/721" />
<meta name="citation_id" content="10/2/721" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/2/721" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/2/721.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/2/721.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/2/721.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400758" />
<meta name="citation_pmid" content="31862787" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="721" />
<meta name="citation_lastpage" content="730" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Clementine M. Francois" />
<meta name="citation_author_institution" content="UMR 5554, Institut des Sciences de l’Evolution; CNRS, University of Montpellier, IRD, EPHE, Montpellier, France" />
<meta name="citation_author_email" content="clementine.francois@univ-lyon1.fr" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7781-8781" />
<meta name="citation_author" content="Faustine Durand" />
<meta name="citation_author_institution" content="UMR 5554, Institut des Sciences de l’Evolution; CNRS, University of Montpellier, IRD, EPHE, Montpellier, France" />
<meta name="citation_author" content="Emeric Figuet" />
<meta name="citation_author_institution" content="UMR 5554, Institut des Sciences de l’Evolution; CNRS, University of Montpellier, IRD, EPHE, Montpellier, France" />
<meta name="citation_author" content="Nicolas Galtier" />
<meta name="citation_author_institution" content="UMR 5554, Institut des Sciences de l’Evolution; CNRS, University of Montpellier, IRD, EPHE, Montpellier, France" />
<meta name="citation_reference" content="citation_journal_title=Insect Biochem. Mol. Biol.;citation_author=S. J. Ahn;citation_author=W. Dermauw;citation_author=N. Wybouw;citation_author=D. G. Heckel;citation_author=T. Van Leeuwen;citation_title=Bacterial origin of a diverse family of UDP-glycosyltransferase genes in the Tetranychus urticae genome.;citation_pages=43-57;citation_volume=50;citation_year=2014;citation_pmid=24727020;citation_doi=10.1016/j.ibmb.2014.04.003" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=J. Alneberg;citation_author=B. S. Bjarnason;citation_author=I. De Bruijn;citation_author=M. Schirmer;citation_author=J. Quick;citation_title=Binning metagenomic contigs by coverage and composition.;citation_pages=1144-1146;citation_volume=11;citation_year=2014;citation_pmid=25218180;citation_doi=10.1038/nmeth.3103" />
<meta name="citation_reference" content="citation_journal_title=Biol. Lett.;citation_author=B. Altincicek;citation_author=J. L. Kovacs;citation_author=N. M. Gerardo;citation_title=Horizontally transferred fungal carotenoid genes in the two-spotted spider mite Tetranychus urticae.;citation_volume=8;citation_year=2012;citation_pmid=21920958;citation_doi=10.1098/rsbl.2011.0704" />
<meta name="citation_reference" content="citation_journal_title=Cellular and molecular life sciences :  CMLS;citation_journal_abbrev=Cell Mol Life Sci;citation_author=J. O. Andersson;citation_title=Lateral gene transfer in eukaryotes.;citation_pages=1182-1197;citation_volume=62;citation_year=2005;citation_issue=11;citation_pmid=15761667;citation_doi=10.1007/s00018-005-4539-z" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Andorf;citation_author=D. Dobbs;citation_author=V. Honavar;citation_title=Exploring inconsistencies in genome-wide protein function annotations: a machine learning approach.;citation_pages=284-284;citation_volume=8;citation_year=2007;citation_pmid=17683567;citation_doi=10.1186/1471-2105-8-284" />
<meta name="citation_reference" content="citation_journal_title=Environ. Microbiol.;citation_author=I. I. Artamonova;citation_author=T. Lappi;citation_author=L. Zudina;citation_author=A. R. Mushegian;citation_title=Prokaryotic genes in eukaryotic genome sequences: when to infer horizontal gene transfer and when to suspect an actual microbe.;citation_pages=2203-2208;citation_volume=17;citation_year=2015;citation_doi=10.1111/1462-2920.12854" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=I. I. Artamonova;citation_author=A. R. Mushegian;citation_title=Genome sequence analysis indicates that the model eukaryote Nematostella vectensis harbors bacterial consorts.;citation_volume=79;citation_year=2013;citation_pmid=23995941;citation_doi=10.1128/AEM.01635-13" />
<meta name="citation_reference" content="citation_journal_title=BMC Biol.;citation_author=M. Ballenghien;citation_author=N. Faivre;citation_author=N. Galtier;citation_title=Patterns of cross-contamination in a multispecies population genomic project: detection, quantification, impact, and solutions.;citation_pages=25;citation_volume=15;citation_year=2017;citation_doi=10.1186/s12915-017-0366-6" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Binns;citation_title=Contamination of DNA database sequence entries with Escherichia coli insertion sequences;citation_pages=779-779;citation_volume=21;citation_year=1993;citation_issue=3;citation_pmid=8382807;citation_doi=10.1093/nar/21.3.779" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. Borner;citation_author=T. Burmester;citation_title=Parasite infection of public databases: a data mining approach to identify apicomplexan contaminations in animal genome and transcriptome assemblies.;citation_pages=100;citation_volume=18;citation_year=2017;citation_doi=10.1186/s12864-017-3504-1" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=L. Boto;citation_title=Horizontal gene transfer in the acquisition of novel traits by metazoans.;citation_volume=281;citation_year=2014;citation_pmid=24403327;citation_doi=10.1098/rspb.2013.2450" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=F. P. Breitwieser;citation_author=M. Pertea;citation_author=A. Zimin;citation_author=S. L. Salzberg;citation_title=Human contamination in bacterial genomes has created thousands of spurious proteins.;citation_volume=29;citation_year=2019;citation_pmid=31064768;citation_doi=10.1101/gr.245373.118" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Bryon;citation_author=A. H. Kurlovs;citation_author=W. Dermauw;citation_author=R. Greenhalgh;citation_author=M. Riga;citation_title=Disruption of a horizontally transferred phytoene desaturase abolishes carotenoid accumulation and diapause in Tetranychus urticae.;citation_volume=114;citation_year=2017;citation_pmid=28674017;citation_doi=10.1073/pnas.1706865114" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=B. Buchfink;citation_author=C. Xie;citation_author=D. H. Huson;citation_title=Fast and sensitive protein alignment using DIAMOND.;citation_pages=59-60;citation_volume=12;citation_year=2014;citation_pmid=25402007;citation_doi=10.1038/nmeth.3176" />
<meta name="citation_reference" content="citation_title=Structural approaches to sequence evolution: Molecules, networks, populations, Biological and Medical Physics, Biomedical Engineering;citation_author=D. Charif;citation_author=J. Lobry;citation_title=SeqinR 1.0–2: a contributed package to the R project for statistical computing devoted to biological sequences retrieval and analysis;citation_year=2007;citation_doi=10.1007/978-3-540-35306-5_10" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=L. Cornet;citation_author=L. Meunier;citation_author=M. Van Vlierberghe;citation_author=R. R. Leonard;citation_author=B. Durieu;citation_title=Consensus assessment of the contamination level of publicly available cyanobacterial genomes.;citation_pages=e0200323;citation_volume=13;citation_year=2018;citation_doi=10.1371/journal.pone.0200323" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=A. Crisp;citation_author=C. Boschetti;citation_author=M. Perry;citation_author=A. Tunnacliffe;citation_author=G. Micklem;citation_title=Expression of multiple horizontally acquired genes is a hallmark of both vertebrate and invertebrate genomes.;citation_pages=50;citation_volume=16;citation_year=2015;citation_pmid=25785303;citation_doi=10.1186/s13059-015-0607-3" />
<meta name="citation_reference" content="citation_journal_title=PeerJ;citation_author=T. O. Delmont;citation_author=A. M. Eren;citation_title=Identifying contamination with advanced visualization and analysis practices: metagenomic approaches for eukaryotic genome assemblies.;citation_pages=1839e;citation_volume=4;citation_year=2016;citation_doi=10.7717/peerj.1839" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=G. J. Dick;citation_author=A. F. Andersson;citation_author=B. J. Baker;citation_author=S. L. Simmons;citation_author=B. C. Thomas;citation_title=Community-wide analysis of microbial genome sequence signatures.;citation_pages=R85-R85;citation_volume=10;citation_year=2009;citation_issue=8;citation_pmid=19698104;citation_doi=10.1186/gb-2009-10-8-r85" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. A. Drăgan;citation_author=I. Moghul;citation_author=A. Priyam;citation_author=C. Bustos;citation_author=Y. Wurm;citation_title=GeneValidator: identify problems with protein-coding gene predictions.;citation_volume=32;citation_year=2016;citation_pmid=26787666;citation_doi=10.1093/bioinformatics/btw015" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=T. Driscoll;citation_author=J. J. Gillespie;citation_author=E. K. Nordberg;citation_author=A. F. Azad;citation_author=B. W. Sobral;citation_title=Bacterial DNA sifted from the Trichoplax adhaerens (Animalia: Placozoa) genome project reveals a putative rickettsial endosymbiont.;citation_volume=5;citation_year=2013;citation_pmid=23475938;citation_doi=10.1093/gbe/evt036" />
<meta name="citation_reference" content="citation_title=An R Companion to Applied Regression;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=L. Fu;citation_author=B. Niu;citation_author=Z. Zhu;citation_author=S. Wu;citation_author=W. Li;citation_title=CD-HIT: accelerated for clustering the next-generation sequencing data.;citation_volume=28;citation_year=2012;citation_pmid=23060610;citation_doi=10.1093/bioinformatics/bts565" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. P. Gauthier;citation_author=Y. Outreman;citation_author=L. Mieuzet;citation_author=J. C. Simon;citation_title=Bacterial communities associated with host-adapted populations of pea aphids revealed by deep sequencing of 16S ribosomal DNA.;citation_pages=e0120664;citation_volume=10;citation_year=2015;citation_doi=10.1371/journal.pone.0120664" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Grbić;citation_author=T. Van Leeuwen;citation_author=R. M. Clark;citation_author=S. Rombauts;citation_author=P. Rouzé;citation_title=The genome of Tetranychus urticae reveals herbivorous pest adaptations.;citation_pages=487-492;citation_volume=479;citation_year=2011;citation_issue=7374;citation_pmid=22113690;citation_doi=10.1038/nature10640" />
<meta name="citation_reference" content="citation_journal_abbrev=J Gen Appl Microbiol;citation_author=H. Harada;citation_author=H. Oyaizu;citation_author=Y. Kosako;citation_author=H. Ishikawa;citation_title=Erwinia aphidicola, a new species isolated from pea aphid, Acyrthosiphon pisum.;citation_pages=349-354;citation_volume=43;citation_year=1997;citation_issue=6;citation_pmid=12501306;citation_doi=10.2323/jgam.43.349" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=T. Hashimoto;citation_author=D. D. Horikawa;citation_author=Y. Saito;citation_author=H. Kuwahara;citation_author=H. Kozuka-Hata;citation_title=Extremotolerant tardigrade genome and improved radiotolerance of human cultured cells by tardigrade-unique protein.;citation_pages=12808;citation_volume=7;citation_year=2016;citation_pmid=27649274;citation_doi=10.1038/ncomms12808" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=J. C. D. Hotopp;citation_author=M. E. Clark;citation_author=D. C. Oliveira;citation_author=J. M. Foster;citation_author=P. Fischer;citation_title=Widespread lateral gene transfer from intracellular bacteria to multicellular eukaryotes.;citation_volume=317;citation_year=2007;citation_pmid=17761848;citation_doi=10.1126/science.1142490" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_title=Genome sequence of the pea aphid Acyrthosiphon pisum.;citation_pages=e1000313-e1000313;citation_volume=8;citation_year=2010;citation_issue=2;citation_pmid=20186266;citation_doi=10.1371/journal.pbio.1000313" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Katoh;citation_author=D. M. Standley;citation_title=MAFFT multiple sequence alignment software version 7: improvements in performance and usability.;citation_volume=30;citation_year=2013;citation_pmid=23329690;citation_doi=10.1093/molbev/mst010" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=P. J. Kersey;citation_author=J. E. Allen;citation_author=A. Allot;citation_author=M. Barba;citation_author=S. Boddu;citation_title=Ensembl Genomes 2018: an integrated omics infrastructure for non-vertebrate species.;citation_pages=D802-D808;citation_volume=46;citation_year=2017;citation_doi=10.1093/nar/gkx1011" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=L. Klasson;citation_author=N. Kumar;citation_author=R. Bromley;citation_author=K. Sieber;citation_author=M. Flowers;citation_title=Extensive duplication of the Wolbachia DNA in chromosome four of Drosophila ananassae.;citation_pages=1097;citation_volume=15;citation_year=2014;citation_pmid=25496002;citation_doi=10.1186/1471-2164-15-1097" />
<meta name="citation_reference" content="citation_journal_title=F1000 Res.;citation_author=E. V. Koonin;citation_title=Horizontal gene transfer: essentiality and evolvability in prokaryotes, and roles in evolutionary transitions.;citation_volume=5;citation_year=2016;citation_doi=10.12688/f1000research.8737.1" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=G. Koutsovoulos;citation_author=S. Kumar;citation_author=D. R. Laetsch;citation_author=L. Stevens;citation_author=J. Daub;citation_title=No evidence for extensive horizontal gene transfer in the genome of the tardigrade Hypsibius dujardini.;citation_volume=113;citation_year=2016;citation_pmid=27035985;citation_doi=10.1073/pnas.1600338113" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Syst. Evol. Microbiol.;citation_author=W. K. Kwong;citation_author=N. A. Moran;citation_title=Cultivation and characterization of the gut symbionts of honey bees and bumble bees: description of snodgrassella alvi gen. nov., sp. nov., a member of the family neisseriaceae of the betaproteobacteria, and gilliamella apicola gen. nov., sp. nov., a member of orbaceae fam. nov., orbales ord. nov., a sister taxon to the order ‘enterobacteriales’ of the gammaproteobacteria.;citation_volume=63;citation_year=2013;citation_pmid=23041637;citation_doi=10.1099/ijs.0.044875-0" />
<meta name="citation_reference" content="citation_journal_title=F1000 Res.;citation_author=D. R. Laetsch;citation_author=M. L. Blaxter;citation_title=Blobtools: Interrogation of genome assemblies.;citation_pages=1287;citation_volume=6;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=E. D. Lamperti;citation_author=J. M. Kittelberger;citation_author=T. F. Smith;citation_author=L. Villa-Komaroff;citation_title=Corruption of genomic databases with anomalous sequence;citation_pages=2741-2747;citation_volume=20;citation_year= 1992;citation_issue=11;citation_pmid=1614861;citation_doi=10.1093/nar/20.11.2741" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=S. Laurin-Lemay;citation_author=H. Brinkmann;citation_author=H. Philippe;citation_title=Origin of land plants revisited in the light of sequence contamination and missing data.;citation_pages=R593-R594;citation_volume=22;citation_year=2012;citation_issue=15;citation_pmid=22877776;citation_doi=10.1016/j.cub.2012.06.013" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Syst. Evol. Microbiol.;citation_author=I. Lee;citation_author=M. Chalita;citation_author=S. M. Ha;citation_author=S. I. Na;citation_author=S. H. Yoon;citation_title=ContEst16S: an algorithm that identifies contaminated prokaryotic genomes using 16S RNA gene sequences.;citation_pages=2053-2057;citation_volume=67;citation_year=2017;citation_doi=10.1099/ijsem.0.001872" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. S. Longo;citation_author=M. J. O’Neill;citation_author=R. J. O’Neill;citation_title=Abundant human DNA contamination identified in non-primate genome databases.;citation_pages=e16410-e16410;citation_volume=6;citation_year=2011;citation_issue=2;citation_pmid=21358816;citation_doi=10.1371/journal.pone.0016410" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=V. G. Martinson;citation_author=T. Magoc;citation_author=H. Koch;citation_author=S. L. Salzberg;citation_author=N. A. Moran;citation_title=Genomic features of a bumble bee symbiont reflect its host environment.;citation_volume=80;citation_year=2014;citation_pmid=24747890;citation_doi=10.1128/AEM.00322-14" />
<meta name="citation_reference" content="citation_journal_title=PeerJ;citation_author=S. Merchant;citation_author=D. E. Wood;citation_author=S. L. Salzberg;citation_title=Unexpected cross-species contamination in genome sequencing projects.;citation_pages=675e;citation_volume=2;citation_year=2014;citation_doi=10.7717/peerj.675" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=V. Miele;citation_author=S. Penel;citation_author=L. Duret;citation_title=Ultra-fast sequence clustering from similarity networks with SiLiX.;citation_pages=116-116;citation_volume=12;citation_year=2011;citation_pmid=21513511;citation_doi=10.1186/1471-2105-12-116" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Mistry;citation_author=R. Greenlee;citation_author=K. Fong;citation_title=Database contamination;citation_pages=605-606;citation_volume=260;citation_year=1993;citation_issue=5108;citation_pmid=8480169;citation_doi=10.1126/science.8480169" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=E. Nováková;citation_author=N. A. Moran;citation_title=Diversification of genes for carotenoid biosynthesis in aphids following an ancient transfer from a fungus.;citation_volume=29;citation_year=2011;citation_pmid=21878683;citation_doi=10.1093/molbev/msr206" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=R. W. Nowell;citation_author=P. Almeida;citation_author=C. G. Wilson;citation_author=T. P. Smith;citation_author=D. Fontaneto;citation_title=Comparative genomics of bdelloid rotifers: Insights from desiccating and nondesiccating species.;citation_pages=e2004830;citation_volume=16;citation_year=2018;citation_doi=10.1371/journal.pbio.2004830" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=H. Ochman;citation_author=J. G. Lawrence;citation_author=E. A. Groisman;citation_title=Lateral gene transfer and the nature of bacterial innovation.;citation_pages=299-304;citation_volume=405;citation_year=2000;citation_issue=6784;citation_pmid=10830951;citation_doi=10.1038/35012500" />
<meta name="citation_reference" content="Oksanen, J., F. G. Blanchet, M. Freindly, R. Kindt, P. Legendre, et al., 2018 vegan: Community Ecology Package. R package version 2.5–3." />
<meta name="citation_reference" content="citation_journal_title=Annual review of entomology;citation_journal_abbrev=Annu Rev Entomol;citation_author=K. M. Oliver;citation_author=P. H. Degnan;citation_author=G. R. Burke;citation_author=N. A. Moran;citation_title=Facultative symbionts in aphids and the horizontal transfer of ecologically important traits.;citation_pages=247-266;citation_volume=55;citation_year=2010;citation_pmid=19728837;citation_doi=10.1146/annurev-ento-112408-085305" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. H. Parks;citation_author=M. Imelfort;citation_author=C. T. Skennerton;citation_author=P. Hugenholtz;citation_author=G. W. Tyson;citation_title=CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes.;citation_volume=25;citation_year=2015;citation_pmid=25977477;citation_doi=10.1101/gr.186072.114" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. T. Pride;citation_author=R. J. Meinersmann;citation_author=T. M. Wassenaar;citation_author=M. J. Blaser;citation_title=Evolutionary implications of microbial genome tetranucleotide frequency biases.;citation_volume=13;citation_year=2003;citation_pmid=12566393;citation_doi=10.1101/gr.335003" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=mSphere;citation_journal_abbrev=mSphere;citation_author=K. Raymann;citation_author=A. H. Moeller;citation_author=A. L. Goodman;citation_author=H. Ochman;citation_title=Unexplored Archaeal Diversity in the Great Ape Gut Microbiome;citation_pages=e00026-17-e00026-17;citation_volume=2;citation_year=2017;citation_issue=1;citation_pmid=28251182;citation_doi=10.1128/mSphere.00026-17" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=B. M. Sadd;citation_author=S. M. Barribeau;citation_author=G. Bloch;citation_author=D. C. De Graaf;citation_author=P. Dearden;citation_title=The genomes of two key bumblebee species with primitive eusocial organization.;citation_pages=76;citation_volume=16;citation_year=2015;citation_pmid=25908251;citation_doi=10.1186/s13059-015-0623-3" />
<meta name="citation_reference" content="citation_journal_title=BMC Biol.;citation_author=S. J. Salter;citation_author=M. J. Cox;citation_author=E. M. Turek;citation_author=S. T. Calus;citation_author=W. O. Cookson;citation_title=Reagent and laboratory contamination can critically impact sequence-based microbiome analyses.;citation_pages=87;citation_volume=12;citation_year=2014;citation_pmid=25387460;citation_doi=10.1186/s12915-014-0087-z" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=S. L. Salzberg;citation_title=Horizontal gene transfer is not a hallmark of the human genome.;citation_pages=85;citation_volume=18;citation_year=2017;citation_doi=10.1186/s13059-017-1214-2" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. L. Salzberg;citation_author=J. C. D. Hotopp;citation_author=A. L. Delcher;citation_author=M. Pop;citation_author=D. R. Smith;citation_title=Serendipitous discovery of Wolbachia genomes in multiple Drosophila species.;citation_pages=R23-R23;citation_volume=6;citation_year=2005;citation_issue=3;citation_pmid=15774024;citation_doi=10.1186/gb-2005-6-3-r23" />
<meta name="citation_reference" content="citation_journal_title=BioEssays;citation_author=G. Schönknecht;citation_author=A. P. Weber;citation_author=M. J. Lercher;citation_title=Horizontal gene acquisitions by eukaryotes as drivers of adaptive evolution.;citation_pages=9-20;citation_volume=36;citation_year=2014;citation_pmid=24323918;citation_doi=10.1002/bies.201300095" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. Schmieder;citation_author=R. Edwards;citation_title=Fast identification and removal of sequence contamination from genomic and metagenomic datasets.;citation_pages=e17288-e17288;citation_volume=6;citation_year=2011;citation_issue=3;citation_pmid=21408061;citation_doi=10.1371/journal.pone.0017288" />
<meta name="citation_reference" content="citation_journal_title=BMC Biol.;citation_author=P. Simion;citation_author=K. Belkhir;citation_author=C. M. Francois;citation_author=J. Veyssier;citation_author=J. C. Rink;citation_title=A software tool ‘CroCo’detects pervasive cross-species contamination in next generation sequencing data.;citation_pages=28;citation_volume=16;citation_year=2018;citation_doi=10.1186/s12915-018-0486-7" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Stamatakis;citation_title=RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.;citation_volume=30;citation_year=2014;citation_pmid=24451623;citation_doi=10.1093/bioinformatics/btu033" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. J. Stanhope;citation_author=A. Lupas;citation_author=M. J. Italia;citation_author=K. K. Koretke;citation_author=C. Volker;citation_title=Phylogenetic analyses do not support horizontal gene transfers from bacteria to vertebrates.;citation_pages=940-944;citation_volume=411;citation_year=2001;citation_issue=6840;citation_pmid=11418856;citation_doi=10.1038/35082058" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Starcevic;citation_author=S. Akthar;citation_author=W. C. Dunlap;citation_author=J. M. Shick;citation_author=D. Hranueli;citation_title=Enzymes of the shikimic acid pathway encoded in the genome of a basal metazoan, Nematostella vectensis, have microbial origins.;citation_volume=105;citation_year=2008;citation_pmid=18268342;citation_doi=10.1073/pnas.0707388105" />
<meta name="citation_reference" content="citation_journal_title=Environmental Microbiology (Print);citation_journal_abbrev=Environmental Microbiology (Print);citation_author=H. Teeling;citation_author=A. Meyerdierks;citation_author=M. Bauer;citation_author=R. Amann;citation_author=F. O. Glöckner;citation_title=Application of tetranucleotide frequencies for the assignment of genomic fragments.;citation_pages=938-947;citation_volume=6;citation_year=2004;citation_issue=9;citation_pmid=15305919;citation_doi=10.1111/j.1462-2920.2004.00624.x" />
<meta name="citation_reference" content="citation_journal_title=ISME J.;citation_author=K. Tennessen;citation_author=E. Andersen;citation_author=S. Clingenpeel;citation_author=C. Rinke;citation_author=D. S. Lundberg;citation_title=ProDeGe: a computational protocol for fully automated decontamination of genomes.;citation_pages=269-272;citation_volume=10;citation_year=2016;citation_pmid=26057843;citation_doi=10.1038/ismej.2015.100" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=I. Uchiyama;citation_author=M. Mihara;citation_author=H. Nishide;citation_author=H. Chiba;citation_title=MBGD update 2015: microbial genome database for flexible ortholog analysis utilizing a diverse set of genomic data.;citation_volume=43;citation_year=2014;citation_pmid=25398900;citation_doi=10.1093/nar/gku1152" />
<meta name="citation_reference" content="citation_journal_title=Ecology;citation_journal_abbrev=Ecology;citation_author=D. I. Warton;citation_author=F. K. Hui;citation_title=The arcsine is asinine: the analysis of proportions in ecology.;citation_pages=3-10;citation_volume=92;citation_year=2011;citation_issue=1;citation_pmid=21560670;citation_doi=10.1890/10-0340.1" />
<meta name="citation_reference" content="citation_journal_title=DNA sequence :  the journal of DNA sequencing and mapping;citation_journal_abbrev=DNA Seq;citation_author=E. Willerslev;citation_author=T. Mourier;citation_author=A. J. Hansen;citation_author=B. Christensen;citation_author=I. Barnes;citation_title=Contamination in the draft of the human genome masquerades as lateral gene transfer.;citation_pages=75-76;citation_volume=13;citation_year=2002;citation_issue=2;citation_pmid=12180348;citation_doi=10.1080/10425170290023392" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=C. G. Wilson;citation_author=R. W. Nowell;citation_author=T. C. Barraclough;citation_title=Cross-contamination explains “inter and intraspecific horizontal genetic transfers” between asexual bdelloid rotifers.;citation_volume=28;citation_year=2018;citation_doi=10.1016/j.cub.2018.05.070" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. D. Wu;citation_author=C. K. Watanabe;citation_title=GMAP: a genomic mapping and alignment program for mRNA and EST sequences.;citation_volume=21;citation_year=2005;citation_pmid=15728110;citation_doi=10.1093/bioinformatics/bti310" />
<meta name="citation_reference" content="citation_journal_title=Insect biochemistry and molecular biology;citation_journal_abbrev=Insect Biochem Mol Biol;citation_author=N. Wybouw;citation_author=V. Balabanidou;citation_author=D. J. Ballhorn;citation_author=W. Dermauw;citation_author=M. V. Grbić;citation_title=A horizontally transferred cyanase gene in the spider mite Tetranychus urticae is involved in cyanate metabolism and is differentially expressed upon host plant change.;citation_pages=881-889;citation_volume=42;citation_year=2012;citation_issue=12;citation_pmid=22960016;citation_doi=10.1016/j.ibmb.2012.08.002" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. R. Zerbino;citation_author=P. Achuthan;citation_author=W. Akanni;citation_author=M. R. Amode;citation_author=D. Barrell;citation_title=Ensembl 2018.;citation_pages=D754-D761;citation_volume=46;citation_year=2017;citation_pmid=29155950;citation_doi=10.1093/nar/gkx1098" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Prevalence and Implications of Contamination in Public Genomic Resources: A Case Study of 43 Reference Arthropod Assemblies" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
<meta name="twitter:description" content="Thanks to huge advances in sequencing technologies, genomic resources are increasingly being generated and shared by the scientific community. The quality of such public resources are therefore of critical importance. Errors due to contamination are particularly worrying; they are widespread, propagate across databases, and can compromise downstream analyses, especially the detection of horizontally-transferred sequences. However we still lack consistent and comprehensive assessments of contamination prevalence in public genomic data. Here we applied a standardized procedure for foreign sequence annotation to 43 published arthropod genomes from the widely used Ensembl Metazoa database. This method combines information on sequence similarity and synteny to identify contaminant and putative horizontally-transferred sequences in any genome assembly, provided that an adequate reference database is available. We uncovered considerable heterogeneity in quality among arthropod assemblies, some being devoid of contaminant sequences, whereas others included hundreds of contaminant genes. Contaminants far outnumbered horizontally-transferred genes and were a major confounder of their detection, quantification and analysis. We strongly recommend that automated standardized decontamination procedures be systematically embedded into the submission process to genomic databases." />
<meta name="og-title" property="og:title" content="Prevalence and Implications of Contamination in Public Genomic Resources: A Case Study of 43 Reference Arthropod Assemblies" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/2/721" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Thanks to huge advances in sequencing technologies, genomic resources are increasingly being generated and shared by the scientific community. The quality of such public resources are therefore of critical importance. Errors due to contamination are particularly worrying; they are widespread, propagate across databases, and can compromise downstream analyses, especially the detection of horizontally-transferred sequences. However we still lack consistent and comprehensive assessments of contamination prevalence in public genomic data. Here we applied a standardized procedure for foreign sequence annotation to 43 published arthropod genomes from the widely used Ensembl Metazoa database. This method combines information on sequence similarity and synteny to identify contaminant and putative horizontally-transferred sequences in any genome assembly, provided that an adequate reference database is available. We uncovered considerable heterogeneity in quality among arthropod assemblies, some being devoid of contaminant sequences, whereas others included hundreds of contaminant genes. Contaminants far outnumbered horizontally-transferred genes and were a major confounder of their detection, quantification and analysis. We strongly recommend that automated standardized decontamination procedures be systematically embedded into the submission process to genomic databases." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
    <title>Prevalence and Implications of Contamination in Public Genomic Resources: A Case Study of 43 Reference Arthropod Assemblies | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Itnv9hkEi8C0oIysIGXwi6OTM_K095jby-aa2D_FvF4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495529","apath":"\/ggg\/10\/2\/721.atom","pisa":"ggg;10\/2\/721","ac":{"\/ggg\/10\/2\/721.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/2\/721.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/2\/721"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/2\/721"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/2\/721.figures-only":true,"\/content\/10\/2\/721":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022j49Fip222JdSmkqPx4GLrP3T9ygl9JOPfwJuGDZYWKsQmbG+FXFYNzkSCDxMFFzTBWWEdlyJwcl1YYIpJagJTuYSF7lEjCbpES8i9B52D2nnbhxWqGnZ67wZrwZsLuqhud5\\\\\\\/O\\\\\\\/gvBBtLRNZwgryPLnMNT8g71hdymiDXsw3PGQKrgLkD890PiBatVF8FiFffP3Xf8AsKW3M8F7nEvWsfWn98FsZU0nxlXMYKx7+xFIxHC7DwZqOkR6ovaNHE\\\\\\\/dTjvT1a9tTM1ZUEpgjR8KPlYwx5guIF8STiGNwhxGPYFKgDRXvjj7CkFGAIQON7amPuK93PTDHRTtPRqlXOy2gElEgLOqVx8u2HQcpHKKa3jA4gSIlWROSORIikpVxJUwsHjccNkAUX04\\\\\\\/IkINxOvwH3Ve95B2811jIo3b8\\\\\\\/bsAuZaItD1Np\\\\\\\/QMiieGiJC3aQsgqy1i3R+iicekGIvNiTO\\\\\\\/JhKd3LKmvkdPA0Qjr0geL0DfWh0NHoN2o7G2rTQCn8bahJbUV3PbbBb9a5R5H5CUdZVlDFfF4dOAsOuBYLRYgFlaVrIxVzGST0y8VJSmrv\\\\\\\/hbICs25+5kTlgSKDKZlfIloF\\\\\\\/PvxBMW6mfO3wRJ+P4+XN3NXS32Jgxx\\\\\\\/RUPAW2C9UdC52tiGaNhDXTdqdk2cfvyGlmVALD7JsG4uZViFRuoPuScamlbyvvwJRuhhHjf7sDjJywHM47GoKY6hZVQIRk6Ni0LFBeky18QGTs6PibzVmGawx27uN58Da2n1+SennKS2fODxXesEg658KTSfqXaFgnaRVambJahLviS\\\\\\\/2ZKUGH0clKzXX2rYED+il6x4J0nMWw79ilIFDjIROTyXH50lp3pPm4BK5YqW9\\\\\\\/XU+1VYH7SNuQwAKKDYso0Ptbl\\\\\\\/SowUv0WAgDhPK8PeXdTJZzXCsSXBMVH6Esr4iXFuQ3p1z+nWCr78j1IQ1c3G+R0q5EknlQUn2ZszgkfJCHmu2dhLOpxuXV5f34crdjA9lqjCuYj7te+hiXczIlY3mJrxilDhAo4OuH09yP+pN8NgGmHGmaFhh4h+G+kaQshtV8uOGszLmmrwAKM5ExHj0R9LHkX0w4Pv5\\\\\\\/ikRYoVa+QqwpX4Wxk5R4Cdc16gWGO6EoJICz2LtX+rrYKeWcjBBpw2ASRfy4628oZUWqhhuizByBBU23ztCsPAbzrcNMk9HHg2pfigftgvn5DYJD\\\\\\\/mmFvJv499yx5n+9DaEQiATWUjpvFG59QVrZ0XY32D2CYBA6ZwKoGiAGmACT9tHQ8sPWeAa\\\u0022,\\\u0022iv\\\u0022:\\\u0022mRO+32j8O3LzwjgeLY7ZLA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226a90a14180d2a84b1089e9e6f72ab691e0307278d9c2d7bda57149b7174d4fab\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022SnBTgYCZNfeNh4egvh9Yvd8Sho5Cu28OkCzi7H+s0jaimwTQv0o9rRTbE9QjEAen5Tfc4ujhs0IYTE1fzZ5frhrcMBGFv5z79Oo0jWlTUqpVSEBdgBd9Eo4WS7yG8PaBBKm0XEW6pHhY8L03wveREOEZkjOVeb31O4FMf45itZldhE3o8TwcdFnNjXZX\\\\\\\/7sTSgW5IbtW0KzGv0gvZGu9M1K0xAGZcB9Jhjo5uMHR9Gr+WShmhXp7rnc7aUUVIDiyma+0toJHdVfrfKMShD3UmMxIdT6jCpzDWNJD6utND9WWG31396HAUAULUriqROHIAH8uLpcqYZpaaJPysK6GNxm3m8co+0fCl84hZ9r9ympaEcGgi0Vw0weeF1Cyg7uL+cBv+o5KCazHJtgdcpsSwEqqtcurl4aVMvtYIGIIGKFsezvpYzdJwy6KRtMinOpp2kuM0BOuBSwdiXkcM4aqNV0YBKLCM8e5uHpMQ1K3R1qG9c1refvD5QlwO2dyzcU0rC1ZC+RXieh18v1\\\\\\\/WsoQimqK\\\\\\\/dPgjvoAEsuTpa+G+Jax07v+v0+xGR\\\\\\\/hpc+OuhXLGVpAlZFtdkjwHqhQgpMFJ\\\\\\\/zhRp\\\\\\\/JB4QUvo2uzLo1dq3h4yOETGmpe1Lf9acDHO+73y9vl1yzqy5sRykib\\\\\\\/ha+\\\\\\\/Wxj+nPDH9u76SXssACGvXI+WfyMmTw7zD+oNMJi\\\\\\\/G5OkfnpcCII08rWdwWGxlt8fEC2USyqudxJjFQkj6GTZUB9tIx9syRi571nap6ZnNlikTCQ+1gp37TNUjYlaxbbL1N5UM6h4a90Gwmq7zuQUFpjhCysh887reF98C0cQFVowL2O\\\\\\\/R+Q89az3F5UgoNFs9HiZEpMH8r7Y0SX6bdkH5vW\\\\\\\/YLiswL0a\\\\\\\/0CpiDQ8DbNuP303As7JIzTDkGaPuYEaEBEwyn6jXOi+anIOPKhE+9jsROrPIFQVwPhtBiio6I8nZ9nYnOEXKhmUgQLgxWiyuJPTW72Hl08LGaRLH0GiZB3TxcRqlJov37Va7KfymsOxKPVMqOVJ0SycuZSI7go9GCwgDjFOypb9PtThQaIWR6gPRe9ZkM3OpdXv4Wosh+HhLG6LZhF2AzLyQ+d+e9CM2EqzZT1O1YjY\\\\\\\/fL3VdlZ+8T1GJfkxNegLpiNjGizn5+gCZb1L\\\\\\\/in7MEyj6XyBlwALTb6K+De\\\\\\\/\\\\\\\/jNADGHG\\\\\\\/VSLQGz+kzX7cVRsi0BvIj4cLbRQBQzBln30Qeh0Z7S7KMGD+YCDsnwnCna9JkIDx\\\\\\\/Z5j\\\\\\\/baNj\\\\\\\/TIX2X\\\\\\\/FcgdJW7q6Qo\\\\\\\/WQeYSFBcpVtve2Ys2Q++Z\\\\\\\/6SD5penw7uLRDrvLra1q6uOArVjnJGk26BsRUWEEjKYsc1IJEHkssBbrsuxCkuzX+vjc+qVBmoNj2yw7+MOwdmtGk\\\\\\\/jmqfa5LGb0UzMmcfIJHepR44USxNOiT94z7lIFq8uxxZMiwO+Q51u5m5H+sZ06LVv9Wc9Cxz4NvTx8aLTWBr46lyXPH54CSWflrVboW5BHqbAo8ex1AhbEk1xn7J+Yjk\\\\\\\/rzniLXOXHiTTusu2P573R\\\\\\\/NzvvK9T\\\\\\\/0WIiCq1Ws6Ip6wsCEQtQQejkBYBaUWEaAGruWr0vnhghggXOP1xW3MSP9ntCUjii3X2pEVbBHjVUzn4Iq7sKvszTpfH54hQoztNCcgW\\\\\\\/P1R5qMSF8bCPyPxJMQIwN0BBZ+0aizq3\\\\\\\/PismolO22OaG+WO8Zlu61wJXUbSd\\\\\\\/osIfRiUMUMQOOM94Gljm4D6GzfHFXlXsJC27g6HfES0lujky2jmWXsAhmaUxbO+\\\\\\\/Ziao59IE+kQA1O4XbRufjxwRfbeKYPG2GIRsPm0mftDAw\\\\\\\/u8d+ipJ5u9kxRp2f0wyh9oH8zA66C2++D\\\\\\\/jVGOz0Br38RmC3NNYGkvIE6DaaDpYdgr+8MhV5XgolY\\\\\\\/VzagAvfvoG152fWHjpiIpe972pTfktnoQ+SdiAqok+tb4V3865G4SffFvKIgfhCuY1ckol1aHnHnDa0iv\\\\\\\/erQKTRMQd1LZl576\\\\\\\/viKZIszaM++eLqmkeQQ5WGzr+umT\\\\\\\/Hjd7bsUv0wmqU4TrMYADnfNArJoshlc6Q9kbag5misqZLeweGAe9ftih4mPQ7HiWZWIvICLSKRj2A+\\\\\\\/mdRSBhG34myA47fmRNuVKiBR96utXKlIURkJYUHq7PbREA0ijnYF1SciNvq+2prvChsVzfh6OBi2KsMrppP9Jo+cg+qV09rOBEtO1Eo9G99RrHS6vj2alRVR8e8mCnrZDOpqxrlHBVcINk\\\\\\\/I0SV\\\\\\\/uM45QVR2Fhi9ELWHKsmV8xexIy+EmboZ49pC4kF4iKGvKY1MRey4sOQKqyL2PTv5FUxOEtP9fGoHZRMb\\\\\\\/mPbYh9lpNartATSJhKoQgvqO\\\\\\\/igIFu7ZsjeU4WnHIS1dtY6ze6biCSMYq3hdna3\\\\\\\/j6Ra1O+xHTVHJQ1occgeMd3icChUG+D\\\\\\\/zPLKumLzdeeujmTk+\\\\\\\/sv3zHElaCdI5G6B6n5wP0q1tAcpiZUkyWjfUckzbMwAn+\\\\\\\/0DWCwz++JBIR4jelEmqjNP4v68t6k0743lK9ct4VJdHPSb75vACogRTqhBb4vCbibfkJhTEl5C\\\\\\\/HXhoUiPAc8QBQaMCmNKFQeBcoawRPyzHB1\\\\\\\/Wp0HBRMLmdDoq2kx4+RCae7pdbSBwKQIAIYac7w4a\\\\\\\/dRn6hxA4lZpBsgeFA3SUeBu8AS9zMYOjRBoDKhN4DIQSE4c9mBGfGTixgNaSx58EkBgwcm\\\\\\\/hHEAz4Xo7hA2N3d+0bVtVSNfhafILCf3slkVUpR08ByoaV8Fq+WUjz4qdHACs6\\\\\\\/BSgmLPRCxSwJuZi4YlygIA51hsKtN+LJ4JF2XWkrZeK4VV+01FVMPs2XNHZ\\\\\\\/XG3J7zSrIHZB7w7HxD987Ivols4GEbL1k\\\\\\\/IzWGwBaWAuKNEejtukrumQ3Lwyylas8++2\\\\\\\/tLjqry2+pDLzI3s7FIAhwhyy2O0VYLyyJaOF9mK1pl+Sn4nKPpgb8QAaA06gaCJZeGh2v9FYw6wiBWMD+fxoETdwNnclvG50ekwIrXSJGS\\\\\\\/9KkULxMsJ1eZ0A\\\\\\\/YJ\\\\\\\/MAp83HzOV521iESzwQ\\\\\\\/e+eiuSmcRax7gAhkS+c+9X8VEBQaiRobvvrC0mMwliHaK9x\\\\\\\/yWoka9cKMTfPkYaMcTKQlTpIRtYP1zHCJWy5aAR8BCenCI6hKhQyNgcIJJ+Zanq\\\\\\\/yT5WmYlPO675MbAb\\\\\\\/66odfwM7fVfcUf68DG\\\\\\\/rlWTTTsMUcKHnRqVWBnrLwg7YjP+yPNNEeTlXigs08pS4US\\\\\\\/0D2Dnj5HbSFy9YewI\\\\\\\/dwD6vAS\\\\\\\/HNcIVCArVwH4uG+Bbj\\\\\\\/sUIGtVM6X2KF4mfgifHk5J+6g49Yt2k5R+cQ57avDg+ijXNYimlvGt1cXvkoBxkXHGgotbxaq0T8\\\\\\\/ZJiQUCbH1HXosDJfERYGEvd2famJ7uybU\\\\\\\/UcQDzw89nZKbhQvf29Yu+QRL4EkKIXW22UOvgyqnn27mZOMCECfMLBpNkE1hVmbDXsfhiKi4fA1+BGFSBk89u3YYcf+p26ebukEWQ6GCZojzyzkOMKZ6fT1zNTsPm9TpzKnpq9O6TPl0\\\\\\\/NpK\\\\\\\/0\\\\\\\/2xA7XqRsgrGdHKIM\\\\\\\/h456eUCFQXUGkvRi5ulf2yEoAmgmro5rQdcIaZvAGtImVlfX3AWG3IWYpRJ9BZYtoC\\\\\\\/B6hT\\\\\\\/DCCYjNwLhMExcK7l9R+E9JcZOX\\\\\\\/4snAsX6bMkWmTJYO0LNTx5kwELJJKbNBcVR6Qfm8ypEF2tyZBPah8eJLzkVLMgv5PA9GzjhXlZYRSxWiMQ9BVtUt8j+9y5U6DxGkDPVr1Gsm3KILDdSphFGzQgMSIh+Smk3z1rLvwEbpHOoMDbcXtAZ\\\\\\\/skRhgC3fyS4STQPABlspNAZw0bzlsNkMnpBt4e0VhiJpUKCmsgYNlKvLy\\\\\\\/X6WlaAObcBT9H5K7TBy2V\\\\\\\/NJH3gX7eygKyetmXnfn+fAM3jnhhxU9N+E\\\\\\\/vbEOmi78j2+9Wq7oDqlYWh6uEoJM0fnegK4\\\\\\\/OuxkmEitKkQf1K9EPKAzkKEpXb8pyAbB1xMn0enGUXbS98fc9VOsCfl6x6CuICuizGMmJkRx7s3jc3FSZFlFvjSwr2aEf8Ctl94cyZjxA3OoYrVJY0Pi4UX4F89wHwcoO8Bvwe+y0saGC2SIJLI2tMeji\\\\\\\/B1Q6pM49oNEZs0Q2jO8BGmIRUPDKhXcUjaEG7emi7IlxZEG5mT6eE6M5IR5A3U2WvRrgbx5ZaSSOJj7T7P3bcQrfcf5jefEItSAzeScSEFyTuoG3+0euRMR9SaS6VtTKFvE0yAMnS9awLV10ulhP8yCPYBOJ3XT8JSsCbF6i7tkxoGu+YKYfcVQQOTs9lBIei8jrNmCOutv\\\\\\\/O9zpXAjQd8hFNtdqdKi\\\\\\\/VJaPlb451hMvs+bO+7dSKWpNSa8f156WMbW9I73NSD4kMvWiUaOKKmuC1+3Vks1pEdNwm9GMS\\\\\\\/WnSIUZLhr2coGuaMDNsC5466rDNfpSAQV3TGJxBOsxbeBMf7WhPN1Lfswl3pDu\\\\\\\/AYTDAEAdFVhPqG714OkI3vapVKl75hH851elIfAuFHYLqbjC81p9C6ntt2q8Bvtuy37CEel7PpN1IBDBz5r1QHfLLfC5aotRI9fopVjyv\\\\\\\/ipcS6yzwPdPpQ5rZHx7z64Nk3k3rycHckJNaJNOiQJMh1Lc3K4JG9xjgUo1fs9\\\\\\\/\\\\\\\/AVQs9AodhiOxa6mn0sARHI+rZCZknTAqmJTZxF2TzxGVuhXXI9mrlAGKJJSfbVGSniBe689qFgB5ujgyo8x2C+Z7LSrIzLdqzLnrZjoWP29sPbLNmxgdJXljHE0J+fMoBP3V+h39L\\\\\\\/0eWOFYKImEYMJ0mqAoCzjh46Dae1Sp6diC3oRY0eBByGgBE0UWR0Z2Fyx8IBarsHlGYIBU11aFW7eC9KbLeq3qM88vorw9l2YgvfOkhoyNpNhJd+OiZeTYEAqoAP1t6r11i6dp\\\\\\\/OAKX1ji\\\\\\\/HEMTLpanSnTQSlf65Ia2G8x2p\\\\\\\/qxItKCyaQhK8oiefOvwPGWYmIXCwJjhUokG6nVTGX7bFfUdr4IiIlK8YeIMifzXoeg5vrdnkYnx9bDS6BCdF9ZiZ0N16iyEmJiGmr6uVHAilsIF\\\\\\\/u+2JAW6gR1O2+oAtZoixZsgKIj4j59z9PEcsVMCXv7bKSKjygI7GcqB8wU3D0FbEpXsRLAuT44WP6yP3UELhW3hvVIQcTq5ayciSeKCtJe1r9yP0n0\\\\\\\/T1Ls1b7tmc2Q\\\\\\\/fBFeB\\\\\\\/AUY+D4Gsxs1LeZmjeEhT\\\\\\\/t4ellrd8FARqbBufvSM2+e4KhH0Eyl8kdP\\\\\\\/FcluZUc5OLrAseGjbdZf5Eo00A71aXXW77c3nfg2KRc50z8ApjUXxFRkCwN3Nmxsb5vlz+mco3rRqn\\\\\\\/YoAgL3XL9p3gKeIVONa8KyMrG8\\\\\\\/Edn9DRxwRKr5pjehKj6ZIQ9SIPVoqJ2R2ot\\\\\\\/oVv1EalPXWaKEncOWoauapeS3QMdFeWoas2eysGhtLnzNjxlor3MOJiU7bkumsx60vJrLRPJSvUuQhWFp6GrO0X5RCHdraQd60aNTir3Nglm3qm4lRG0KrYuc3\\\\\\\/hERYxNHkcYVqfQ+YDYgD8jRr4XPjUihOrhxYWQbNYsmrzwibSQoH2X7FK95NkHB7rAnLp1H5EC+bDxmhFxRylJ2qZqcUC6L3Igai2xEKbz1ZrpNP+WidfnE8BIhTEh8N3o7w4RY9lOcTbOS\\\\\\\/p5Cilycs6l8Sj3KyW+YjcFds286oA46j2+8fjYq0mmsvuRZV0FkeVteIQzrOuaoUMKqUoIF8ZSbdEzZKi1wLeymJSwOe5VFyiWPFSl2FF8E+eU9FWJDeQoYEgPO1dJilR1YdKYFk4uow7UxsQfdqxu7GCZS5qxLtKyP1ycVYvDlMA0xzUUUt9OGEDoeY9OgktfPzLf7+V1418rDbGJG2hhwP5uhrxgdz1fO+5WQ9nRD5o4nvU86gdUi5VFmZxJ9ZxhnxEeFGVHv8ZTe3fLY0102ck9Oc8Gga7H4IxQcvQLsSZBang8RQ6TV11uHQkFni0LD9kpSswyMwirKo6BRdyistw9GhwRQYSw49KyxWTrdndOuNvTT0+Qr5VEVXz8JloYSKBBHa1dcxLItDDd2fwz8NYcdYJ29qZT2OCHugGb\\\\\\\/l3NGTA6rYMFSPQPJzKbVq+8lwJbjroReAsCqN+tJeMGRL0dHku0ATFLHO87eOmJ0hI\\\\\\\/XQ1bYXb5EZO+ZJOyt3i8NXrKG+a\\\\\\\/HFtMJ7RHf5iXlwU1JhC2q2lfp2IiwitI98qzwZN4PP+CKs9W\\\\\\\/RwKouRwT14CXbemL9CU+FhIPBzfnQLKPDL92XaoAF2PrfDZP9Otx00BU1AB7gfRs3LBqVhJLRmfGzGuI0XAvPM0UxnfinpI0CDdZhh\\\\\\\/mhELJXdPxx4uK0x4+z0g5MKSSJL\\\\\\\/oiF5Lf2AzWiOK9xqc6NlhrUM3JwhETjYnByD1HJGbpAaxIUe7QRP1bjp0QwuqXZBmIb8kEk0R2LSFOEvjAV5SiJ06B3f0814eM\\\\\\\/VfV6PIl5+4eklikZFTfrQt4hYFyHDIov9WB1kQt\\\\\\\/53duPbSFyLGQdTji+BR44dkkcHhC8uj0FKeciq4qFhAIG0O8wg5ILP+36q29IFG73mtvyqdkkWPzWbMHrp8fWMgmMCwYzPHzcpSrFY0BWRfdD4H5mOUPvi+efbH0zyqSlMkUu3eOsz2nwU+HA5IMOauxLir+ZcoSmKoCX9DV64B8yibKxUM62lg6E4uMVhwqfqwLHJYCb47sB5whXf9WG5ri0XDubL5VDia\\\\\\\/Y7+JX6Os0cbNkyrQuIT6bx0WWHPMN0boo7xVapalz\\\\\\\/NKxC464wGY0O0ujrM1gm4F6Ht0YWl3cOL7aPySxeRy3BCNvbv4o0cqGRSqyGVs7vqghO5Wtpeqr07apR182L8i+\\\\\\\/Cxy2QApClAJSnelgNv8+ax0T04bcsCtwj5JFDmqynSyIamQxgCvuNgD0n2DfFrfUcqWRQk+167MHri7DAsdpFNCJWJO7I7HeQx7sqPkfa3gm11vEEfasRzYXwyf8hMNUQIjghl68zOHgl2SVFiRWvX8F6GCQZ4xVK4BpFAosPJSlBXZXWGiIy9JxYyQYnDw3NhrYyq2sbSkOc9CCk4NrdtiMUEA4cu0HBoFN6YkCE7jB6jUgEzmCk3bJ752MTDn7qFpI2Pc09CyonakqvXLDBhkLEGIdAX4w4nclSU5toQJhfjj\\\\\\\/Cfv7t4eGZ6tJ52xZUAw5ZPIasaVGoXILxEYff1+hVh+6njoGxPE+geG0rO8OUNz2MjY5pyxPexlwZgEgxI\\\\\\\/Q9lUUX6hCay5cHeAzkoOZ++3o0DQgJ\\\\\\\/QJVWDLqxspXSX2hLaFDe7QV1M5wfBhA\\\\\\\/m8aEPVt6p\\\\\\\/oeEFX6AqD0jtoTar5ChT4Eb+tVNCHvs7NJrZRJe5is+9bMH0FJpJ1Wi9mWsWCifAZRam1CGypUX0yfIzgum5Zx\\\\\\\/P2hVVlHX0BHbTMvQSzz\\\\\\\/yIxQ2fy4jz+ycOQH45LdprbIydpSpBdccBDFou4nSziUrHNc9lLhLrhv+WcXa0F4GNIcrSodgWU9SWbyRgPIFxdr5yL\\\\\\\/fEl1PUCg5Q6W0P2X5VlWEReGtKc\\\\\\\/LW5nqUOGVyREjczOQUqP4zHyuAXzZ9fyYYl7Bc4sCkg98i4SgeWM5\\\\\\\/k6eKVgpAvVxkGWWGbExm+qDWmRHVLH\\\\\\\/gp61D\\\\\\\/ngGhp+gOpSMd15Kb6mqgX0wHYg5hX\\\\\\\/z5hiXseXplb1uyNFbjEo0\\\\\\\/4eKl1P0m3aTSQ3WC2G7xOyl\\\\\\\/tvB0EsyQVvEFp85h\\\\\\\/c7zcYAnBWf30iCWu7kXJuGCCAqsMUVdQFE2plJ+z3PjaR+zv56N3KzV0kYqV\\\\\\\/kTs9nKuLFXNAGSjqig1i8bd8UKebrbjEaTWofsVYbbEO5YWX0PZ3v1yskm11TFaV789jpjVFdwqLdIMLamWxXv4thtvrzinf0BvxO6r4vwNMps\\\\\\\/13eXSUTa9albTnj\\\\\\\/NXwWlI\\\\\\\/HcoAa+LyNzUpJvet3uadRDOJaaB\\\\\\\/V2fdsW2Fir1lJjK\\\\\\\/Yk415ns2byIGhSooRm1yUNM2VoEv4kKHxY2xpi37sIXZnSchqV3tPoBYdWoHDfuv2uba09OLjZbSE\\\\\\\/6nytwGX5sTG2Wsy1qSQJ2wZ8oONdHCamsl7aZDrwxQgr5ARX9ikJOsOUtWNUlWbew+9CoA3Rr\\\\\\\/OFXy40xu185EUaBCvSx+ei3rVHIbdHZI6A5hDrSuOkZPMM4yVDNAmrocyFKzIGRRoh1qXkt5Qqp\\\\\\\/\\\\\\\/IcKh2wMvdExS8fqBejH2uCMR4H0fY3jgPBPxX5XrRZIEhI17wUVgIKMDt5EqNUuVh6z5qN9KCijQ82zwYFM2EiS4WfeUPasVQMZJb6BLGN6g7qfp71Kif4MT+86\\\\\\\/GB\\\\\\\/1RkAP5DqnXmRxqsCOFD600Az0TqBmdzMYe2b1mG7C+LRv1ycGsY0czyzDgakywnRG8eK02pdVjw46OyMyVr\\\\\\\/mfFTJCX9QzQdhmivIWTWVP2QmfK947aQj3iEJr2E6au0Z0QaUbL2tLzJxFkFjVPLBTSBwKG4zxDAJB+AiLp6kd+QZsMVBMlb+XGul\\\\\\\/nxxhH9+QYgI3g74qc+kJkbR6NlEOxxthKiFHey43pf8Cki4mSvR7doCJD\\\\\\\/QMw1B9cXtBozmNFUhc6geNktvYGBmHhDFHhovXidikSM6WidjBU6267ijeRXZrkX1MArP3Om8yop4uTjSoxZXDvG9lBQYU52l8N71C8WQ6v6RiWWelgktEtHshk3nTumDIJOzj7Oyqbs8bCb3dg22fg4Vl42uindRG7aijpn+sfFOr8Rc3Vl53THPiYvaQ5IEg5coMOW+fyLaRYcy4bigCucbKgP15gOzLhK3g2ikhBr5gWyRQ6KMhi2XAtur14NR5kSuCuQojhe9yEegf0zKXXMjEJhhuA1ee9IcNRK3Itk8iLz5p5ad6aLfBcjgku64c4WtKeE3eePf9mKkS4SrZ9GSiNOZ6pjvBeHbb5zcJZfTxEUrxBY0nwHU5bAoWbSsA5BMTyt1fk3n2qfHDAFTTJEdr8ICofnukZlefArM00ZyluNfdVlRBWxGM5EbPqAeIj1nAmU5CZQ8JqdUpfpGTOOMMrYePsSEwm1pikB7D9046Fe9P5KVZO6Glt49MfyfI22Oyywf\\\\\\\/c2vAuFxXxEWXiVShPUPjjBdkZdd2WyS4pvabNSO4DeoqB5R\\\\\\\/JGQiJ1PjMrgaVT4oPSnmVyaJftvBFcROo5ELGCA36tLDhuj47OpsGXnzYUv8+0H9zcHwAazq3sSchSFPag1mTnZK8XNxM4\\\\\\\/h5XHmCMJX0uBWG\\\\\\\/BTqp8JUknaY4EbJQSlyNs34ZE6PVmj3kfZD4MF8tEvn8ZEbzJUQlzDIBtbv4C6uggOHAnanPzWYls\\\\\\\/OFGWR6BMoEMJ8p0YrNidsPCTx4UBZBhMMokjZTJ71u3D3Me6cenncKJTMPqDf2Svh0gRWy39sifz0vjw0x5i\\\\\\\/B+vsST0X3iAOed\\\\\\\/cR1zx91wS0XYftb9GrcoW8xE2rF41HvNxHo2a92KkcgU4+tF4M2c7erZE042aF1c5yG0IOPznMVjOMzEAF11LYv5C44lIw0liIeEl9\\\\\\\/n0442eTEdzjmER+scsdSn+GPUVRH2IQFOiKV9zfuNRhhJrVYu3ZQXq3tG6ygfR0SC9BV1GklnIIWIwVGfdJ+sozDo0vXYq3XkRiqWgWamN\\\\\\\/wizlTxty4z04jq+nqoqts7Hq0HlbDtBVyPMenE+q2TzcE6lnwUgL+BDCOQ+8qT7MFVCEl0drDWt6JDqN4CJcJC1rs5E8zYKCwIjagwwTGIaKu2fJe6mwYgfWouqNgV1sjoy8fX3Iu1LomQUXPUchQOsrPty9YJAcE7p83HS1Civ2skz4NNfvQ6obTNSgyq+fmgBFUADU8JKyzDnngJr87R+hxHhRUq5jELtgx+WOLmmkk0fqazENtv4uVDbsKVx3pVQnE6ej+REveSGmTDSg9dUcWwl3YJeIhFgaBd1SHJ\\\\\\\/lvK2YjfTPqNnePUABOxyRSU6CUiwAMOmaxHY22qrmZTD31\\\\\\\/5Ar2WpHDHUhDk3oK1WeIdDyyuYekGCPzNlQoldUDa74eItHR3I4gT7x4CWotTIQSFa6KcDD5+rByEH73\\\\\\\/eANN3xRSYgN4mjtLouUUd95QAwDZs+OTuwHG8emM22ex6RcDCrf84JgtVrC54SRHsRWVpJyqYCCO5QpHeunE24fo5+kW4H6AsJ2Q4I5zUawVAWyaSPSe1GmwdxAKJoNplMPA6km0Xm1L0NlQQLKighkprvu5RrcF93X17aY25TN9x9UgXJgrz60mkjdUMhWY8bKCpDUcrUOHHs9qIE7Hrts2HbOQPtWIxxhZQyC8IDv5m1nTIThw1r2IxIibYiyr2Kl4MuQWlIOiWF65MybKM98Qc5mWmrurjUN04n6XzhdumLqe\\\\\\\/HCG\\\\\\\/Kdr9uPo7k4\\\\\\\/5vHgnmAolUETgBTfyy6y1V1qZmDzhEq1IaMvwS+jgAxqrU\\\\\\\/WDThcpEmVUxmU4rZ5aIuybWHYeKv9KLbYALoXRjfJbo2twXBx4CRy1wwS3\\\\\\\/a\\\\\\\/NEc7qgNNIdbKrVAcJx6OvtVMrVEFwEDjQjvG+WJxhz7Cs0BajlTMkTjhTlFrg4+9cvubam9F51Ov0plVYzTvFnPyq45VJlC9dzwF2a8WTpGOxFIhy6YJMcasdP6U\\\\\\\/y377WjBrjG0LNGaDYngbFkG5rYCWM4yFLP48rNojHFUF9sMvI2sg2HLgNIYg2klxgBiPhPT043YOqK6fwFTgSx1p9fXZ0n43dHPz2\\\\\\\/Mhk7yLcDF910eTOeSA4sgdbFF\\\\\\\/5y1EkkKMd7hug44S\\\\\\\/4dZPlL99qEfw3g5m\\\\\\\/WbL2KwY43KOR6HVCneRoWz8yp44csFJrzPhPO8ZTgcYHBYzamWxetTYBiVezUQx7LHN+DO22DCyzF\\\\\\\/hADBRllgLmC6CHb\\\\\\\/6+\\\\\\\/P0rmqx5GSw0ZZi820lp9m\\\\\\\/1bk7C\\\\\\\/eZqWCFvrfPdR4wsID9dejaOPPQGSUZsNvro8iqERgD4x\\\\\\\/M0oP89ebHWWUBRY2e8zuGalxZuXNZrsVl3ohQ8mYcM7Ogv\\\\\\\/shGNNxZArZ75UpxZ0N3lGWScFDvxa0e8OfYB+2Bg6pUbtCqPh56Fndt\\\\\\\/tzyYBVGk+gM82HsObn8iDcw0N0HFMepOd4TDq3VLmdLMnJPPxqYJg+IcqxpxARRjk7Wlfi4\\\\\\\/sqM4NclVelJonxo7EXJUrzRLQxFwgTqs5LnQ0oAqTKKOweqXXYZ2fXFr\\\\\\\/UH8XPDWzjLpdoWyADO0mMz291V+whUfdtfFvL1lr+e8u7tn++HmZsok3EAUsO6gg1POyNyBOvU7dCG7n0VgA\\\\\\\/o31DZWV8XsmQhs59ZlGEbAuNArNoLR8F87sdvtwcq56Ej8Z\\\\\\\/kPiCPGH6fKLQ30ELADjGTq03w5CAs4Wj3KUDbP\\\\\\\/SM\\\\\\\/Dp3rAloxrmunGlLroTzjawLbSf\\\\\\\/wORdy+FLazeLoEK6yfes3mm3VzKrGCQypp2DBH+95IYhql\\\\\\\/qaqcSMm3rkH675wAQUCzkvDk7qCN3VNb9cBXosiai4OBilI55NhM\\\\\\\/bVsTzqX0qNPgnokmsfNfhkfkAFNlzpcUaJUJ\\\\\\\/o1iUSnPE0g2a7TJMsPv\\\\\\\/VhXJ3WDHFWFikPXTNTRdUPkv2sXN1u9YwdZZFYfNi18KuvvhhzMdy8C+kbOsIN17\\\\\\\/usfbbze6fOfnoUCIsSA+UY4NyXVmLuK9MFt5VBtKknVi5O0Q4peV9QyPWsqPtwkmsVY+hZd2u23ynTo7Yq4jXzLyKDWUtSqhJA4U7+kzG3xyd8cuuUiJk0ymgOi4Y8j7SdZL8\\\\\\\/U9szxRYWf7\\\\\\\/hNg2HsTq5ko76TgyF2PC96EyUOcsYS5ipoH75s27Rbrr461QosG5CQxVxrKn01Fd1LPFiv3QS1N3SEPEHctYzTCzJIOOI9azoVODD8wMrNZMesz2bnPAh\\\\\\\/ALVIZ8d6MH7uPZvrYW\\\\\\\/K+YrHvwT886BoNlLGDxLn1kK6RrXvYfTLPTyPAhKJcBL7NbykXkTUeilXV6XpD4bzSSbnoe1bAYfWRPRCBHxMxNdzw7vexi6g+dfa8x1aw69jnPk2\\\\\\\/iYpSzbMYWvMU5zSQFFwLeXFQQrqlig6P7QUZIBnhecT3iOVvlQomV6FYuJcuoP\\\\\\\/p1OOdWJMgWYqeRz5Ijg3YwXDZrUtSCc6tJ4I14qFTPqEz6CZZX3OSkinR9vN0hFj3TdHgSqVJZ0oqDtjUNqNxg7jnTFyoXiMlQazwSR98RhwrggutYWEiokm9IBcPKC+Gx4Vqj75umMdRlEm5sNYo7nrszumf6ocqzFx2O3kUoMNIbDl4RNqjPAGatN2umdXF5b0DRvQx7iHjZc2s0fXXJLRdbwrWqdmIUg060y\\\\\\\/R0NRCy3RmXW0zANBjsaDxC+\\\\\\\/IANt9pYAMyxuSsszr13uqOihDoDovm3447rXmoomIfLY5\\\\\\\/xvsIt8rZ+rjnH8naw+1LHlQ0hhXtQAh23dJD0HaEYGWtFKedDYswNDJBqthZedamhHx+xjM7lGvOb2ye7tKgsSt2UL5bBJTGRJFViQjfs\\\\\\\/dSM1A6qBk8G+P2rKdn\\\\\\\/DxD2\\\\\\\/Fu54+aFth2VXZrKFoczSIDHF7vA7qWavqrpVVKwfCgQdRt8TBuBUHb7Ks0PHeSXJtK6JQ7RN6VawMqVCgweWHAPWIIc70cSVRpKY2EAC7Ic08dkqtQWeOHHm9YR5+07sC5LKKJanpKzIdG+HAXUwyR8zIBRXUCNoypP+NOatqpg3xMOXT00NkBph4jsiyXnHX7IgIu5+rYjLsjPlmtCsCtBCQqCmgzmCATgIMs7I11ohEPvjZw1GBHXsOS+D6SSuQT9xf17GEc+amn4ZsWSEpxKdXWXnYr6pSPxDV5mz\\\\\\\/WQASR3tv9fhRNR27x8ZFwzYLYuU8TAXvVbCiHiYK7qgvYM7sR7+XRsjwSl7x1Prox44SptFDx\\\\\\\/vditNsqG+zEyWCsrKgqQz7PkDiB\\\\\\\/7jDnf2tqgHSoa4yOUMRe1cS64ohp2XSYzGK1O70XIOzEpJQzsLFCo3Uv8PxDEuyy5tOr\\\\\\\/tsEwtx6338k1OuFjJ\\\\\\\/D4wOzEZMQhfRcpsnAjN24f528OSVYGlbOfZfveVAJ9R9u264rg+aY3+j62BSrgEzbRbnTBXk3ML\\\\\\\/768a37S2ZP1+K0UkjeGKbjwnGTISt3+F2ghc1Pp6qKK\\\\\\\/\\\\\\\/GAtubFb9e9AOUU7hmyvAAq4bYYpnyEbOpT2hXzjvhxtTa1SQMGBYKJ2ERfOnWnTS3N05KkR4gGGfsrqGe7q9BZhh\\\\\\\/bI4XhBenVTS3v3O5HRvjNIvF3WVmsrFoKm48KLYWvKF8MeslHS09JfKMB7X087gYJnTlhmGTMzO+RRP3k0hfGPjCUbb2NY1M5ASc7pTTlrwMJtmFFTDvLfEhOpjxGXvFDfFgO7fpteJblWXkfVabHKgaxFX41DpIb1FP0nOQX70y3a2BCxhS\\\\\\\/GpMLzrcbg8oDkjuN2sNqLRrdZwej\\\\\\\/NUh\\\\\\\/inmk6JGHu+7BKk3xg1QEGyUeds8UlgyqktL8eVeiCAq\\\\\\\/M+oEn9xfyM4TvSokJAwC\\\\\\\/U3DBYnel9KJTbgGOCH26CMV+wROddzjf46k5cwq4gyfmFx1QMlIkk0kw+6OdpV3+0XiyMkP+4YGX8HwOXa2Z2F6OZu6wwv+\\\\\\\/+tRhXC48yZofRH+w7L9rrAwVlHIlfea3MVJ8M2L\\\\\\\/rgxoF6OP0rTNrLQwiZybtKuj3K7rGeGlXPDxTiqvK7PnZ8vnO4U8lpVNhrJKbWmqSIbAOg9RZ62iUIHZKF1xOgfXPnUZTqq\\\\\\\/bpFCHVwKMXlsztcr+wDcJ\\\\\\\/TmjwiNzq1nTCMVTqmHUsbgVG+XLpVsigsSCOCwXDqqQ24nnAeo96E5mPmw4ncTZ1phcprq\\\\\\\/hSonjRWpfM9KHmoM0vIEI05ZiVZ0qEy3Y68\\\\\\\/K07x0wa3W02+Slg0B7YiCMBuVhSBS5SnP0arcTIhuDU6OZHfalK1WvP6SzKuHjMqPGu2vhKVTOMgGwcAHsCS6fokd4OeRuworPVdJXIWvcHo+fAQ38cZ\\\\\\\/kU8mlhYJlSrTMfrz6ljc7fstGA5uOBvHzDW5WzlwKjHhNBZKFmkxlmBMjoMPQ8XVVi9T0vCxOcHvfsWN7LS0huojooNvAODKBb1YSDHQwQjiogHk9T77nUeVMRKFC+CFkHWJ+oqAGGRabtANfGh9TRhSNJ5aX09XFODp9oFy0GvyWtVYeBDJ+4\\\\\\\/70kt0K0pTU0u+nq2IpsxxisdQumwIvamAV82KlF+SQNHZAMnVlorm\\\\\\\/fz\\\\\\\/t6G6IT417SzKT78PdtfOdioTfX4jmmd3A\\\\\\\/Fn78S7jt7eJ6RJOUehE3WalSYPsl9r1Tpa+UaIITr24al3XZ3W\\\\\\\/tPZmleweQeTk4AzGoz5oPY9Idu6KEx8fhiySt7A0jKuCBiPk+t7ezSA7Wb5D8rFbO\\\\\\\/oIhmL0iOwqL6N4BZtoK6IYfXLzz\\\\\\\/VjDuLjf+I5qNdr5Vpz7542QuCZ7fdMl9R+yo+HRCXod6Az4dfT0zcAyR06CJLuVIV3PtMxuczbqWLhN+S1ZFH8abncoPSrAxkyCvSYm+bF7uoUPe2vgoZ5KnlBUudKghiRQBTyqX7JChSmTeClSOpG7j\\\\\\\/ELiVsFu5Qu0H7SqgpAsEV7LJSQFY2rK4R5qtqQVHCZI46xz7geHQWieTUSwUVzHO0jmpvz8TNnn0IRoJxc\\\\\\\/S0Gp0q0fsCa+oBCWoDVBveK+H\\\\\\\/sr01vO+a+4hZu\\\\\\\/GVXLVPzoWOzob3Ke6Q2F3zyTp9yJDGk9y5cGMpTfk79EAoGBrLsYs5lQgkZrqGP1I9MNyrtOmuE3H6bAcHNWVC7FXyIUKGrYXt5n225frTxQ33J1HXknmtIjUqJJpkMFYU\\\\\\\/6O++Ow4j4\\\\\\\/11emqadIHba09PVpqsnkkf4cGNPBr+vjPlqUoJExtFLAkWZs8v43LVQwMUjkjik3IGywLZeNQvXAHAcAVlP8\\\\\\\/9+fEdXCm3AVcBpWSAniV0Q+kjMbXuYNXQwobDiTPM9qEOOaSQg6k6IJkMmvN9WwLQbCTDm9CtThcawEjCAzURJzsRGYJzijzVWtninTtW7htnE92lsywqX0VjVxjnOpmdkmfF8X50i8aPLzGcoqjycKXIulxDM9qm0kG+PJGN7sKyJn6vBv+nw5tG9X8XpyvONjiFNCqHwbyH77Z4eChGDcyVAXIcAYYaPVjEDlRLBpxh4U0lL\\\\\\\/oTJJ2TQfltIfzSOadDRNF\\\\\\\/g3MMDsvQP4XByEoK6PtuKpfs6kjImaCMf\\\\\\\/86qlmc69uNgeB5zXOx+7s5mKzktRUpL7ksNvfI3JZUk1WATtbCWUIrh\\\\\\\/+jdwS+YdI84\\\\\\\/WAE+2MBTU5tmtl4c1AU3hrJAEKcu7jVpg9YegSw0xDya\\\\\\\/3\\\\\\\/d+tw59jFznQeiB3aLZwA1pJ6TG5ybAnEcFyN+DPDQLaq7W6WTX+DhXje9IWiLdqSLj2JjIcwIuIji\\\\\\\/HJDgabzaUN8GzelIbFhAvWErZ4Mf2ggGoGuwaKUJt3ZWpTP5Jk\\\\\\\/q\\\\\\\/YCfTVeO5Jf5exVt4KVz2M1cCPVAQQVBDpGsSvl9S5x49GmmpiUXMKlmylVJQgqcbRtldcyKNtyZ1YJzITLqYtPfNJJp+bKNJl6uljGob7NwyplEIrKeStdd8aF4a7PFHxiGLICpZ9\\\\\\\/oR0acnRgPJQjGXgFx4CRmQ\\\\\\\/OlrQs1gVDRdF4KrlXA+4bxQUOWxpn6qUzmOcl1VEsayop2S3TfTOsudggyANBObxYffENOslpw\\\\\\\/jbxOBFZAbm\\\\\\\/r7z3cZRrqnt4s0AZOnwlynov\\\\\\\/aImjqhfh\\\\\\\/2daiQUsJYCE95FsHwFyKNw\\\\\\\/FEgu0Kj4iYBaN5bWbg6SkKt8iB\\\\\\\/2B06PF\\\\\\\/ATr+xWek+4m4T5NW\\\\\\\/gNkL3LyHCEiuoloE7SmlnsXg75bDAND4R26HG9zdSHNIk\\\\\\\/qiqGIllU7WThQ9g0V3egVmpEf9TZbojVAHuIOf91x281LlWZiavA7nzeMXj2THYE2bqPvQ7jWN6dFr845\\\\\\\/d1anYGLczVb6vbQUgoxQsZxHLYTGc7qRUBeH9hFngoLcEy3xeDsybcinwpXSVn\\\\\\\/Dd1wiX\\\\\\\/dO\\\\\\\/0qS9K63kA2+cAZY5p+chVQ0wb074Tp6R0xfs1pVVqnEQX6W3VaDO+gopLytt8\\\\\\\/SK51rWaDRvRzqH3Gdg6qU5QDrs2ifJwdXIPpritOBsJcVcw0\\\\\\\/FAQc10JsQOkNcZyZuGkCtWNPZy\\\\\\\/sQxkUPIgwrmloaszKUDS0kH+NAVOUzGJYPFNg\\\\\\\/S2t17zdrlgkcK2fpgamYu4lteajbJZD7pCf6ZEY6PamXNM0wnniG56+MEWEwHT1g9KuqyxrPbXXDOFxXczNzRnLddKE3Y0hhN4xVeFrts2Lr38wI+agTDEzLcnMpE9OeQuYOLbmj\\\\\\\/7oSgIHnlfQb1QzPNctLtFoPON8en9f7FKaewwukMDjBum6ddTLcdj57vzNZnxKHN+v3Ka19dV9AsgGhd+CNNAdnMG8eUsSZT8qoT76QII22tIXrnjPAcdQbkrB7veeor47tshTLFHOp5mreV7WMEqgDaIhJFRp0MPk1HWnPbBcHOWvu0\\\\\\\/czy6SYwm+altvZQq1kSgDd344t1H3\\\\\\\/7qQ47pY76EMJGiqW8ZYaqrMepj9QtCBEVybLuD12L26oDxA7QXlLpbD3mtxfmpHTHO+rW\\\\\\\/G1YwEbxHizEkWTlp\\\\\\\/u3xpK+8DM1NXrhLOQZrbSMnjRxWxmvLHUOIttC3uc9WgCVPKK430fcrwdkY+Suhn40SaA6DW+56kOXyeZDQjJDnLGwtqcFZlkE+aexz0cLBKFf6FPBQ+OHX0NRlZTIQqENdJg+8Q1M2w0V231WVmRD6nL6yzqYpfpkHVSpYAgfxttOP4Bnxgj3plHpIAC+YUURYUoTljkO51O2Qp4Iky4rPmAdVLlNn1jR5KI\\\\\\\/zzWIeyw+XIQrzSK7WXjqEXAQGLzg\\\\\\\/1haEsbUbSi1JGrIkBPqSwDxx5TwdSUgYbzETZYqOyEr+dK1Op8mNXjXPmAND6ACLYf3Na2rJ4XZNhfAWJ8TmltBJrrRnuBDyl\\\\\\\/zA3ydu8y5C5sa9Wa4HRa22H8B0mnmP8J6ThkAMgLDZgDhg1zpvwMfKe3bwVg2DI875UnGufl0qsYcslg3\\\\\\\/mLG5Fzbrk9G0Nd+8jgLelgCQScSs8WStSO6CtmmHDObm2nwyaVKk1biXPj0Lfy4bHm3Ad1Xir0hOe3SssHUcJr8+Tk5mdOsJ2xyLVLIW5ay\\\\\\\/uJjbsbn1KVXxOqMDbjmhp2tdO1OsRDUEVK+z12Fb7IUGiC2wwaA1Z+g7Zwixl7JessoiE4Looq51KR+9EqpXwVN8PrAdC0VDqMGtEUXAQ4mkEBv5BXB3WBh+UE8Nlv5E7D9rT\\\\\\\/CAFZkq0LboxjvENUJsVOr8W+QQqUZl1PwjxCndo8bb9tYXESrpIKylU1+ybQZnc\\\\\\\/SB9qSK7D45zGhpG1q6A5amb2BFd2XlPsPqKNJvjf4Q7+nFvPzepn6OepXIyNf+\\\\\\\/PpYWi\\\\\\\/H9H0iUOmySZhCrLE\\\\\\\/683RsyDPANTOrhJxJZSXYIUMrgYxlJ5izInhrtKV\\\\\\\/Fj\\\\\\\/JPw761vecxb074bmDsSVlMCHZZTfIClvO2a+\\\\\\\/2Xj9rzOQWDb5pMQnl8xzaw6v9+g8hefCXRAAvx5bbkvyHnTlaV9q6br9bgOD9pw5741SAE+8UbdsTe8gjBZ48qUDp8jsxXj7dA6SleIAOC2uS6Vu1V+9dX4U\\\\\\\/cFiEXeOHF0oqC\\\\\\\/etxjPf+seIIAHIY0+O0MMhp7xedQx5fs170kn4yKrLEQXo0H6CFolBOOfIiCxKakeC26TgoriAR7HdZbvo+I05YzzYKqtS+\\\\\\\/5KScphHidGnZZB0jpFUazz7kzI8xMQSxq8wNc8DkC9kI8hFle9snFxti6fZDu2uehh5xQI\\\\\\\/5nKKrh07TiF4DzIilqUdnRCfFh8zTfzrZqZ103EgaO6Auu5G5klQ8Ld9QaXMstwYC6pbn5u0UmS8X\\\\\\\/AF3HgpfyODgeu\\\\\\\/YfxxCJfv4Zwm\\\\\\\/00Jt9Rbm5N5nF7lwfZ5QoJKaNfVqWXzO7tH1BNppI11jwga0Dph9w5yeNH0A\\\\\\\/NuU78YD5xlclnIaC99OTYofziuUFcrqXLExZ9bMSzhW5iV+90UK9isvvCd59vV5sdWmzIcgf00xB\\\\\\\/Y0SaLgKYFB3NFiV7bsL48TkRljERl+RNx\\\\\\\/rIbTpWhyrst\\\\\\\/ine+XrTP0nqHDDCgoBNt6Gac1rsvTLyWCPEVSMn0z\\\\\\\/sNppY5xyItlKiqqBQw0IFPz3ePCWH26xIuoYjiV4\\\\\\\/oxmSvjpqeUbXM7HR6ktX\\\\\\\/IO+UPJsRgAS2sSaGofL++TZq+ExmRJHUQxD01mPSuDSQ0MuZvByFspPrIAdfr7SfkPXoGyIl05uwmPQB2Q6qGnnT4UVzYfCxbV5lVA2luRndIEnaSKDc89ZyiHup1pEZ3jW46wYmgZ+kqW892DBG4V111ijUivk1oyiGy8zMulAeTdtHpzqAQUdTlaKeUNHJNCwlQi6PY75DOQbVhNhmeLzbfYR0pJb4+b80loDMVyJboFZMm3ob7Td+DwHm6OpjDTXjQUBCq2KfjG1SyBEATcPMynaR1GwV28b2ivzLv8oFnn86G6haX0zP2bI+O18zSCq1ZgU7b3zn2h9W9mfMPIfB\\\\\\\/Qj3d3kYFatrGsdDnhd6cngbWd0dFMqgK\\\\\\\/M+Ng8uNd\\\\\\\/Ajk+I6uC6B6vfdEpNIIPvXVe0eWc5zBwkWssscRHGwSAcQOumLNmoJ9X7\\\\\\\/+rDTlZHYbdRBum11GWhmdnPtG81OkSW+2gMcHFj90zOO+gHVqAReGea\\\\\\\/Qq7Wg9bqAzx2RpV2A6roKNOYjPRLoVbuisQfT+9sEL2qWpoYg\\\\\\\/pfRt+4pAFm92veAWTz46SJ1r1kR1zVcul4deksGzMVduqxxQncWlXsHO2uncZyZ\\\\\\\/1tFlkJcxbW6jLiBSbvGCNFlbn1iBfdO4oJ+KiBkWPCQRsDEt3roM\\\\\\\/c0oREzejvettGSF4CxrR\\\\\\\/oiRA7gizpnv5gChBBCFeuldAkGltvDtZRKV2Oc4MHuapOx\\\\\\\/9St8xEqGc19EhXoOVA2XtZbCFTWVY6iNEhnj\\\\\\\/6Zb1dT01t6Q3KbD3dXzWI6klYwMjtqYTni8wYwW\\\\\\\/0iV9OS+nypZRsAcwIjRsFd9iZAMAqnM17ukbWtnXYw47PVSBtn+PVyVjwxp9uoq4jPh4EVbvSiAhu7mUOxZDCbH+bnVSysyBM4suOVPEcqIP54LoRw\\\\\\\/2ZroHR36\\\\\\\/97sq1nEjIJCYFn6JYz5nm0OBY82R5vieHPOuUq\\\\\\\/fcnY2rVkhT9jU1hbLmCMeIVeL9iU\\\\\\\/x9y\\\\\\\/cHTG9vzeFNRQO0FhYnPGFNwia\\\\\\\/3Kl850UUrLbqsoEc\\\\\\\/aeJZvqPdhEPZYmt2kycuoOUBFAUyyK9t3nlK8fxl4Q+SRLzQBwqqfbLrk72PTaUCA8j+Sa+bangTN2Gnxf3iuNrtLiYxmd8Zh9WQWswLwiSyKWT7yZs8uE+O2yleMLORsrVnMhEzs0y7mrFD4pVCfg7BHB3fLDG0a3XyCouNH4exPMo5WISHS1eaBFF2mgm6YP8OghlJyGdgW1S84olbqvkRAv4yltKT7yJ6scGZwZWiIBSTleOJvdlOpG8BBEVlWX6YmGI8jVC4I3MyipwJlOxFTTJjHmk9h07vlEdCcIlWjrRKJ4eDIeut8o9rVpJpzB1MK7\\\\\\\/q6OpeEf7z+4D97OPwVIRd2cSeqU\\\\\\\/wMOzW6cb3asZQqDZp+OeeLm0SHfLkX4\\\\\\\/wEWLXyRBBvhsYXtvHk+GXu6A\\\\\\\/Nz0LJ31DxnMup3mjj\\\\\\\/6c7i6W9NcooGyfvYExfAFZ7XoaEyPzrrzB11Lz1t0EpXvH0cLZar6uNSl87mgkJRpLuAqYLVuVMns0+HyOmcvcl5LorUzl\\\\\\\/SRTfKTFLy+KK6rTlo7qKh++Waib6LlFKl5vFcqfE0dSAk3lOzPjje9ZQ2pjpRSwBtfkT5ydR4+RUSBdaiA8xffjFXxK254pyQZCpEwncUlYxOTr6WEYMiNNXvZj\\\\\\\/fDJGgCqOSrAEmMZEKxWELptmMq2jgoiV50dRFWliRtoLOZ\\\\\\\/pbMIhakdMC7effZi3vg8lCXTS5xRC2Siz+zbwWjDudG9YVYT\\\\\\\/5vjPxHIDt3NUTXTnWN6fr8Cn+JAyu6ByqoDceqaYEs1IBOXgl3nX2RST1KNON7tjoIMRLy+DvZRVn1jOliBkq5CAa3tGyqwCw8dyqDTJa3W+sSLuAvFOwxCvp3vnHqxrkZZcnSCZuBj0bFdOvT4PiptbbXpoMltCdDC1TR7F6wFJ6zbli8TULsQS0qiRHfQBSh4fvWmOet2uwqQWwgNUsY3iNqWjN3VHZg0FaRUFU6NAxLgTjD1XRl\\\\\\\/EsUYQ+d8knlk5\\\\\\\/sGq2P5\\\\\\\/9uTW5fRy7RZXXNj0CEvvV5EoXdr1gR4Xz23zXVelGcUB13S2jGFblKnJOGZou02fOatD6yUpFrguHYEVxr5NHUOwHozL8eoZJw3sD8UrcgsXHoiKA+UdXxU1xATao8Jhk40bo9HR2WSNSRtvWfyEr\\\\\\\/hTwCxwb6ZT4gEYk5drwr7BXQQZY8orxZPp2JCNU2juta70C7uN7gli9BtE324aF0x3d3\\\\\\\/\\\\\\\/6R6LuJTqOPAGstYF60ZLLIzGGX4YBBLWVNye3GQMBDmaN9GouPVdFE8\\\\\\\/bNxUS2RfrangPykNVPzTpBVdfDh8FBKsE3PgClhL\\\\\\\/rbzrzCBa+XBVbEft+qaXlhz2JP6tP1bIB79emlVrKerXgyeG6qiauEHIsH6VtTjrn1VNMiovPJm2MI4\\\\\\\/4NcMW1lYnEmUjFUEwaxdd5p5aIvYYEc9q9XUuUlZwKcsyv\\\\\\\/ZYAzYJ2wSb6mOcWdcPeG6g2kTGtoZ3jAx8bRlmtFdELTx1snWwJXhFMhlVe0oOj7Wn\\\\\\\/KKgxHcZJEOs7YmpSJivoY6t0WftP5MBcW2QeX+NvRl1EVrZtlotVqDOo2pJzU1dc9XPyn0xMzfI8FJANm7P0CfverWyrxsmV8dUTqot97cVAZoCfRfoJbGo+f0aZSGz7x9B+W6WwM7iwRSqZaPNkHSQ2G3y6botZh2l7k3RHc7GjBoXH8IgstbtGJnN3HE03GoyOTisvpBOp+zPklBbYf3Tt00DaWjpjXvR6o5OPlHzM1VsdxpZRYT5OvJHW0aaDIJpDPubAtTR1La69nNtGbBFlUBYF0GF\\\\\\\/GGSUyW71sFMvxHUEPPhD3LvEACaG2sdlFZodVyHsdCJ3+kpmlzU3z2woHimesjvFHZc2AXXJMFP3x26rR3hFvA3\\\\\\\/dt7kJXuPFuiAI6HHLEQ9ru2I5IJYq5SiD+B23rmMkDRtRsa9TbzOz4CYF1Jgn5j5jrKJvHVM8Ka+AgkuurT2O+ZQPeHfr2m2Q1ZOjgcjMcm4wu3+97OB6K53RmlJXcQ2a7029gP9nAZa6YEjPJKf5OtQe7G6\\\\\\\/Q8v04pDeFORIFAXroGbDwukDwBs3P\\\\\\\/r1XgxuuuXgQ1qJZQVtX39i+D8eCAfdVSLsQDCMZEqabQ9IQgDlOxcLEFD7w7YzWt6r1+Fzh97Wl0H9s0XgQTKIGkevR\\\\\\\/ZtJVkQeW5qDHRkpPC9xt9nS1BStGTLVdyoLZAsio61GmaA\\\\\\\/TiA2UIuq\\\\\\\/kFTIRjr0imWtaB8859AwCvW8HA2FklNWIjPEke\\\\\\\/U1OxuDxXweBNJb8GALCWITbtxUyE34uQBJ07lr5kwNdSTmqVMi5FFbgMmgB88kvtYPJbZrSepy5hDGJWX8fiQ7mlV6MZiN9WE3VI4bumpHt3YJCtvwPGEPVj2dknZylMawAiFok3EdrZGhnmny6sNOr\\\\\\\/xXxs4\\\\\\\/BHtL7wNGSkh4i1Hx+0af15Py\\\\\\\/FOFjt49e\\\\\\\/hojZBzCKREf8ImI5xyFdkeSLyuvue2CEh+1EC94ZBnSa3NzMCwjOzRwaRcFeCm5Ke9Ux\\\\\\\/svi9fTSVqz2fu5P6K3fkm7u+X0pg+e1HebpT3UWjFAnYcrqduVSzx72I2kWSu1Xcph\\\\\\\/LhUOkF1juwdCfbcctF4EePu8Z6KrLvoIAvbmxJcV\\\\\\\/YI\\\\\\\/kTDQiweIRi2aVtUMqfxu0XRTZzzi78vGDFyWCq+CsF38xLcIFbYAUN7jvJPLI1DQkjn9UKy5PpuHm+tdLNJu1oLL8CLAtAx0rh4i2zQMl7YQc0Ra5VUCLatnVz2nNrOaaDcTWXDNRVCj4IZ39+oooElnJJkYsadqG5YxbAhP3QADCi4P9v+W4jhL5Mo0Ua2Ks6OFQvIZ\\\\\\\/9SLUXAZdAaet346GNcutBv6LeV8UoMuiYgSyfXux2c0xW9VRahYbB6T2i2Kv2Ogv2Zc91b4y2v7RrOHaZqAE2MsysB5pznYdjs3fRrAFAI3DHbam8h41BhmSpH5iuyq+XKpKh6oSCKEo1qt0pO8TZU68wuTOCQTMHY2KAWnGRA9oOQg4ndsLF4Q5n02SYQPgNfWTuzIOabFLn718CnTm2Xojyo9jFyCfmDwCcKSnqWah7w0P5jt5QUbAr7upzcLvQ0Z3IwtP4i+An6DTYjH8MbMYMfXKrENv\\\\\\\/b6B+sIICAgU8sumVEHVlRJA3VdpdNluh4vLrq151s9SzdjL4roPyCmN7mNLElqxyA48oxKWr8DaIsJckSu8ePXqUGV08XYdEV9KW07Fmp\\\\\\\/JcQJdEu61PkemmnAUmXfmJMArgojmlkQ9\\\\\\\/yk6NbxGVft3mlNhs5FJmE6M75rTEDTUZx90tsSbW7MK+1aIJeEqGm9mM1sLvJnKPAZ6ZuE4OnEIKvn2gcIdT4mlGjANvYIyJqNB+AVuftLfro1JBsCeP7y\\\\\\\/7oFSBE5P0oKdiizwiCmV3mhLCVf8krhWaGFe7nSFf97cXIaFBoDgR6t1vPePl\\\\\\\/5k1uwz9CX46GtV+wz1DsxffgEeHmcOW15S25cZuDlQtE3IIKt62lzPVzgVRTN5cSeDmlw1RASo3JokgxksldypxHvh2PuFfRaqjhkSWVKwatTUWHJUeaEmsOWE9ZDn4DMbyZ7UtcPDsryieAR\\\\\\\/2cQxe2lFVoxtA11jJsbnPI6v76WXHEbvoXd+TkhpecLJDvJPmZPND84G3kOSmc9cT1Wh1i5S6Ehy5nhdANyV2UPd0\\\\\\\/IUNMZw93zZWj\\\\\\\/NYj5tkKQOm1PGjCHJespZX7kLiPtRqCR5YG68qoW6ZVBX38oYC27ULwGIs9XlEEaUMpp8ceUTey80HGG3a+GQnjeUY1+2eccDc1TvZ9cpXty3xRiO3DcUoTVkCpsI1mMZyw3ROVlx62K3u+cL4sXbJrAzXjfJ41HJ8CF4HynCbyL3OASUf5UqFfTJhMV3RzDjpX523CJPquBSIGRef\\\\\\\/UcW2IokLFM6oCPP8qswRNW9RnF1oCE2cJ9j\\\\\\\/0IwluZFQ1XHVawHDo52QTMD2T5gKpNDSijWQStalBfoXAfgovP5kHYnycsmEwS2gMYeRlCfAUKpGVaFAk7epIGtGmIK9oEWbba6FiF5nIbyCRZjdhv+YkDZazwUxyUmAgGHeGSXcR2IW0B9LR1qOsuwdueLHwclMfTGFw1D5B6BHibIzBJHXt7bvEK\\\\\\\/ZYfC+1I+XmWTlHeEJ4jQLm4pggOuFZ6+Ri1E9vUxbO2a4d5ZAMMydnwm\\\\\\\/aC7THajsFCw7LWRBJ7kLa+7nSlJmen9Gex8ZeC5NJJ\\\\\\\/VpSmneAzM69VTdbmLiRzm4K\\\\\\\/SofBrRX1K2hx\\\\\\\/Zl94LCry8gbd8\\\\\\\/R8qgYeG3E6siLVa33yjqKJG5Je4SL4YN5Sa0XwP1vly0iLNJntcEIedltnPa8HwsKEs7Fv9otg20QAuRnN9xZpwvSfQPqASpEm\\\\\\\/OesGCuISP1Xvo5hn27WbMcAieBHNnuST3Lg2wxzlgonzhn3luxv7qySUeOXWcw482wVX\\\\\\\/absFQLQLqbLai0hKZRWZx5a007JT2rQttu43BCoOB7mLy789neVfV8zSiaL9vMV30KDqYtp04l4bU+Hz4xN1cNRJZ\\\\\\\/I7bjYtKhdbCfPwudeamxDHqXwJOUnm\\\\\\\/Cf1q767FeQT6fBHw4f6TTJRmzYfNZM+zN87Y4g\\\\\\\/T4ZGutuZRAuwLW47pV7+t5jH5JU3c2F2Rq54LtWNnGtPe9\\\\\\\/KRwKeCNOHzCg2giSxggLa2VxeoOhjKJ6iq\\\\\\\/gzG5I4VndO3CnlxRUi5qKwfFI3hz1jVccPu\\\\\\\/wNpfU7kaza0R4jwk\\\\\\\/bpqVM7GxLUcVwd+3mvqIo1b+BYczdeLE86038jS\\\\\\\/DIiEfzkEh0dQ4enbLm61fXL0EB4+QGI5QhBg1eeZZARsogmt14Ia0KxJb1IBFBse55ctyGxeg4VelZnnB8PvmsSgzIAAlYVB7exMQmL0z3UR83Usl35ydGmTh5znYwrQ76418zoyONKN2+TZT9XzbXup47M3Y7nHA1EvV4\\\\\\\/5Vbj1Om5HMkw\\\\\\\/GqlykU+uM\\\\\\\/D2A77SyZ4uFgSH8Vu1hcF5FxmS1esTUgvY\\\\\\\/BesqwtyV0lIeCAWZpeFIbvbeDJIUutYMGyo8t\\\\\\\/1CS+\\\\\\\/3nOV69QMEJ\\\\\\\/NqFtJCkjvLu6s+6\\\\\\\/XW9XW8YOeCPu0ep\\\\\\\/QfixnOR+aSMXSWugDS5YqsPSs1FsxLrcYY5i3QnTHC1Fc0GbAXxqLy1dPaphL0lUgDEdTxPhPSqZDgoF6WCOKFx2Eo5wYLexzyad\\\\\\\/li18LYAPfXGxZAyeHdcJQYijxqtohxJ4JOpvWhdc3mcIT537vvJuG24TZOH57Z3NvYnIC9zh1zLFZ7ReZo+B\\\\\\\/u9jAA\\\\\\\/xxEq\\\\\\\/4DwI7cglijz6f165HBfcWhk0LsHwMk0qMz8y7p55STnCjCZJ+W9HI9NSIkg++3ZyTANYKVlJ8m4SVqhfvREFiqFLIttxaVe11jT1UIrQcWfUpb3LWVJJmSruRcz\\\\\\\/kBTLvaKFweuQfe55Qw0nyFZRZD45lQ9di6GOzuj0xEjWjknM1kdMnDtk32PJihCBkwLmltWc+ny+1EEjV7wf1Ksoi8VX3hXnOn7WGmFXMuTq9igmtPPMhUbOhUnsIT5RgCSpSy38l7+NezzCRLpXs2jWgc6jgHMQlTjGiDCVgNFsYQbSmbG6anHP+Vk0JSpPRA1c14MsjWxuYpmv1T8JDes2tWAkPeWfO0oI6TISEKuJ\\\\\\\/2j3YS+B\\\\\\\/KGF3pPDavXTEdVDa7wK8WWHCN1SXGREg3fH59hkdnM1ZP938jJi2xUiXyszeK7Ihu8BbtdkftIYwRafhPISD8KqhTDNkbLCFuty+ai015VvWf4DBZqfc4OO8HFpwDhTvKMqtjX+OSYyq675TwJpLW4wUbXWphmOYuJ15I\\\\\\\/Wb3w6XqsKT82ycpgxbvBg5NivuY2+8U4VqyfaCX3pCnqE+rwguoOgiMRR4vp8+GZ7z6vznWmiPJJLbPnLXdOfd8cWjFt7FgJTwKBIiQD9W8ydXcwExS1AaDduWGueFQEkNalIxFRpa7BCZt+SalmWm97X4y\\\\\\\/hKDHJQRAaowj8gzXVzwPIWSb8mmphLyKIp0Lxs385SzFzagBBdqRviU+2qWW0b5e5uNCPioNlibi0EPqTice16fMC48QAe4iN63XdH0p+EJVed4KB6gPkxlaH8rAMYNvt4BAdyTnSPIW8DC5bj1OKYH\\\\\\\/nEVt\\\\\\\/77E0H7obF24ZHJavB2FM3FtN8GlbYGBe4AV+9hmzvADEAX00XarlfTteWBJh\\\\\\\/md+AnU5UAHCk\\\\\\\/j7W0sbvcT68mcU0IVz9PcH1XlQf5r6C9Y8OjAaZWPsn9CqCkyGtM+B2ZkGSipXuaJH9IbavmQblK1OfDDm3WyVZQY+uwIGJCBeq9XaQM4Y4fIsDNNoyk+VaD5XG7m1xSPXmb4fc3dGtHU5f2T6xALjT9uKJMGD\\\\\\\/wsaNF1Tdqz8Sl\\\\\\\/0rERkLEV+EbiBshyPvXWSTZHqSIY\\\\\\\/xDqZlzJygwE\\\\\\\/wVTvN5bZAwaVuyJEwua6Boo7\\\\\\\/paAdc3541\\\\\\\/f2gXFLiJmeMC7cdUXCPHwARySvHxjVW1cHGRpVTHnJo8sVgkscjGpLPod+Tyz7pwAlJEfN3juGP+DF1jO67OieAVINm7EI2X6iIJMtKdS84WEOtipwb3avmTRpmoB22185KRli+X5RYXGQM0nGPczapweOr2WT5rGg8sjThiZUj87P9kZ5bpB4QGl4yjBwa6qIdQajn4WS+9bvxAbnMOFmPnMHHDk1Y41iHc\\\\\\\/yAzsa+wB32DBG\\\\\\\/UAZqosfgTfDgF0bo6L99qPTmT7ODIcZgkNHKmQwyr3UVqDMin+szZ11ShhuwW0XHSp4F+p2BasoQEzTjM32qKmyN\\\\\\\/gMx0WAwkcezv0alc+EtKanokD2lEVKCXeZwlj3i6XINIVve7H0QZtTasbVG8XZ7wPzuRMmqfh5yV8M8dk9VaGvewOzX7yOIdFicte\\\\\\\/vVlCYuC\\\\\\\/edG7VrTrYkeNU8Q9qxCwyV3y5TP3zWFYoNLEIXzSUNlE1beWKqIJExTCDVve1bMzD7yFJZU8nF\\\\\\\/fGmMtXjBpZB6wQ4if47FRObDa8nu5tfGnifGZ+Yr1XP4Iq2VuAzOD5s5Qmmmh5W0tphHstQIqhVvWq\\\\\\\/JAFxk5D6kMrwuDSAqTvD\\\\\\\/v\\\\\\\/JDX8k2fWGJ08wTWxgRMgjFzmm5pPvzBvoPvIna4yKfw8HQp+sDCUTy+rcZDwT2HYfgPgpm\\\\\\\/OA8lPpwv\\\\\\\/BUJWKZ3pgrnPt3lTzwIV\\\\\\\/xs0tt6xq0KsdqTNdAAgE3UQMn850mYB3SwMDSuXs0xJpQWcJGPWnfR49TesLH3HzmfudWetr+2BOhXmvDKrEUbYedUt4vN8pCVjbCqqVFv94pvSvYWX7wcqXN0oYUhWSlz1psclgJNu5MIK1yhCAslheajuCmfIcbiFb2+GvJ5E1LKXGJozwiGkwr7TaTEBvs+XOY389SNi41+MhLP0ThlDogJSfL528woJTvBylc7O\\\\\\\/OyWRJ34voF6C4WstjcyuLtcWADn1vj2x7iW6LhEQJ8KFqhkQwEU6mPtB6y0Nl3LNPOi3wErvtbYfLNGecI0mxAZUxaaGOoZ+x3j2sbb3bhBFaWQtnQxS3jmHXCnA6qB7b2qC0BADtm2w5WFAG1wYaWUfJ8PWThfm+dorpEW0kRpJzIrlJAgXiJNH2mTSqUIO0aZIKKAo7D59MWmA+3fnyVV83gm6vZ0rUJnm969nM4x6sPVyhuHgqB7IM1SiAVaqz5QMM8FW7V0eLrYzP+LEcdNi8xlc7gTIYgMN+F\\\\\\\/xoCbpo3iUVuGqJWZrDC7dRvuow5bOb+VGZQAwQiywulyoHJTYC1nqdbd3uTNka+yw2bZA9qCgqmm6kRgK8ka0L3FB9oUB6ni\\\\\\\/DDC1ouMLH+qOlpBnl1GvYma5ms3nv1ySygVOx5BRr5DzRH0QX4xE2P6yr0oU5RykxLXFyTlTJUMAdfPpW6evFLmkrv\\\\\\\/7NQNmO3QQw6aIHb7w6gNPANB4RmrZTHOPuGZpD2UpxaUw1r6TicXXz7fYTbRR\\\\\\\/byeCAp8Cd1XRTMRckpc\\\\\\\/ljPiAW3ep1xHqI4Sin+eMksbk7\\\\\\\/CrJ0vkhQMjMQp87x+HYoYBRUCtGLcm52ath9JZvMHEsmkzjCTXgDxz5SE95+4\\\\\\\/jToV6IcdNEogBsaShsTlnLzbLz0KpwGRUbjVL2p4v1V4n1QB2HUln2+TMU0DK9gYjlBcaTHM3Me17RmY\\\\\\\/bO4WEb9FVhPIdJ1B7TzCCVfdov4YNaST6+4EAbUC423SNwXKCDkvFom+s1X5gCPzOtYvbLU\\\\\\\/zHTXImCeo712belREL\\\\\\\/xyJXygnr04Z2ihwdYnQlElsBXH5sLzFJN\\\\\\\/ZTUrSq\\\\\\\/\\\\\\\/eicK98XuKwYW3LuJ3e1EihnAwlusKYnzRbi\\\\\\\/vw5uvKX4TvwehJm8lKpvHv1oiXlJgNqouUm6skMb+QUNSEigRLrkYCDSuosLf7l737lYPRBvqtBpvwP4QEC2+vFNtGjMi+L5ezaT5Pi2x51ZawnalW+\\\\\\\/YYweJbhHlR2aReu8QMLOSprJOODNSyc1bApBRT3F4uB7hFO0B6PHz1CgQInWdTPyiNoEYGAYXboWRqJ\\\\\\\/fRUMdg2kcVmbvufVUb0nwcYm8SeECWZwtjf\\\\\\\/+g2W4KvusD+u43WsPOztk\\\\\\\/6HSDYReC0KlYpRn6DJSgduWq+f16j0I0RPGpJpeKEDir01eXFISc\\\\\\\/sd\\\u0022,\\\u0022iv\\\u0022:\\\u0022HEOhhRiNUJgXHDKuEuU+xQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022daea95b294012fcfc076e41db9b57697fef948bead0b1234c8ab636f5dcf94d9\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002245V3tVd6GhK75X2tV3P9hnEcajiX+bCKVe\\\\\\\/9DZMiNY7WG8DhjwXv3IoJWzCRRxRfynf72NPxxaQD0sz8b1D5R2PZs19tVqv41G6rE+p3hO30RfWbtFGmw1gN5dVmpNCc2gDcx0KxzIOUtyOiZEpyZSs9oNfu0WE4kkkfKCjZiDpB3yvLwycfsuKaOMrPKqEstiqinWcH9oNuFfn\\\\\\\/5pk4C4ZD42fBmGaFhN1OEsVYfFC2WWbTyez7WecFbuHrERVOrKfjSJYtO2vQAMavDHqsempP4zPuxC2qzoHUrPYL950c7nUBbycnomdqq8hHVu94yFMFk\\\\\\\/JHPAVZCBl8w3vsAkJhHTyISuf1P1LFEbXkyQVJAXAqH81dCDGqvlGQzKLfwPtpNJbFhi1IYN5p7PGlDUSMsFw9KESFEGJuSFWEFf2qwoPzjDEbXqScFQPC9Bxa2c4V1pRR7T9HfvOT0K27Pn7idVpEPhtjRpRgsyCWrQRpdBmsm\\\\\\\/k1ZuMyn4bQic\\\\\\\/LwqMzEpbb3Nt7TYSZbG8voWyIHmJOa+ikoUp9WCUWVdIo80D1PKR1WDHeNX\\\\\\\/lwV6Q8D2MsRLHYOmFNVF33VH5vrPD+2W5oOhrBa8YE6gsIAMOQkhBF5roqnoXQW\\\\\\\/Fy48O6bXm1Rro44X7SyRTeoJBfdyx8bvxUpMrTaQ2ZnIbER+GJnxAU\\\\\\\/Abmqd81rd\\\\\\\/BxpVqd09Vfqq7vyVItMqQFGXH8L62mXxpAV+VcAZMbBBumDUd7\\\\\\\/wHug89sqoHpeD\\\\\\\/zZiok9985W1MyXXj19K7RuxrrjfwDR6dMLhx511yTxhMyOYWhgvCMWFyGn+oW5e0uRFB8TUBtzlgtW+ieyRb6oO3EFFNmlJeaIj1\\\\\\\/T+DdvHFMZwSk39oDe+eV\\\\\\\/jFghT47E\\\\\\\/5JEQOnEFjkJtCbMZ21Fpu65mVHzRVPSLgOZXee+W1FiHe0epN3KdiD8wC9vtyzx7s8gZyjyGLAay28ugXhFNDtRmwKEM4F\\\\\\\/mCCGiGea6cwb6JdBE9cScwzePdnIn9\\\\\\\/4M9ocsBJKJXjCZTjJy6fYKA4AFRgbzuNz70dQdPvmvG+mFEhyzR9wAIGcBqFJCN68iQZKXSWY2e9iSWpAQiVEp+WQb\\\\\\\/NZBtsI5e8GaEO3SMY7mDls+DYy9d9uNubwR0dyteGkrNIgO1jqPCc\\\\\\\/y544j7tbt2zdcyK3yINpo6b4dm5x60G8MEUNKDIFbGMnJeUlxENOQJRowKmjV1pkgzx6X1dQoa\\\\\\\/JRLyv20PHOAfoR5tYM\\\\\\\/YwQ\\\\\\\/VWylQYw3nSn7RqomfT6eTY95RE1AGWt\\\\\\\/g91dNUGvg825Hh+bgZe0C2EsIdfqjzA5aeABR30ReMoUTxVlYw1VEnnn8OzJZNxe3ToiPzxFk4i9vn3CA3zJgNj4p\\\\\\\/ol0LlgojvCQNucYIAVBzm2qrvij8PUQghKSuAZxSRVJ5GDb2tMLEIEDucDrpcoE964nQb\\\\\\\/JXscr45S8aD8Unv1aIAO6kxzjRySqM02KwEd6CAOyAgg5wt6E6Vj+Xw+u5ZUIXNW5e7Xejn0qc1pXamlJeyqIHUHbLxoLB\\\\\\\/MfOS9xdNYwQPh86SRC5QQSYy4rrnOaK2YSFnJzLVLUEjmZv4BLzY1+33\\\\\\\/ciME1yUlzPHQBp3zV964bSd\\\\\\\/Vqe9Gf1uvcUCMWcoJyJ0e\\\\\\\/kGcWxSMFecP0vwwVzPuDicsluXDU1vVW61TJkG6q8eQw1z84keyqC9ZI+MK4pKauoDqxTPfolY0\\\\\\\/L+QG3tjBTyIMPYbNiSPeSwzCeudwZ\\\\\\\/DL\\\\\\\/gE1XSwLwk+\\\\\\\/8c1d35gzg0yW2iG\\\\\\\/mR8keGxWaNbx97SMbXhbG42xkdsua+3psYTp6pfMAlsgfJsALWrJiosCUZfNNOxQ5vgja2TsEe3MmRv6DQv4pDJW\\\\\\\/Yb8PFsNX5P2yeBBC3TETAPmOk3AscKQADWKOUbJnQmuD9jhdH3zSJBE\\\\\\\/tgj0ckucaTzIn\\\\\\\/dP\\\\\\\/zZcnN83HcLR+1ejpsU3GwIWmB8j5WIrifFWpFfg02feUsvu0dk0Bk+cPOLbqAYjy5svt64OK4egB8ZclpBauO1ggUiKowch\\\u0022,\\\u0022iv\\\u0022:\\\u002231zCfw\\\\\\\/XCiOssZSVLZtMyw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220e061626dd5a65e2b6fa264ab7ea45fe5a613177d6d80b10185e0a6e6d20b588\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495529 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/721" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qtTZdrS6KT6herAQJC5xfHNhu9ijzcErb4YW-Ceqrf8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/721" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CU-xeOVAV-kHz81KughGNGcl7YdtsL-LEaJtBrPBOn0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/2/709" title="Horizontal Transfer and Gene Loss Shaped the Evolution of Alpha-Amylases in Bilaterians" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/2/731" title="Genetic Analysis of the Transition from Wild to Domesticated Cotton (Gossypium hirsutum L.)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495529" id="node495529" data-pisa="ggg;10/2/721" data-pisa-master="ggg;g3.119.400758" data-apath="/ggg/10/2/721.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Prevalence and Implications of Contamination in Public Genomic Resources: A Case Study of 43 Reference Arthropod Assemblies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7781-8781" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Clementine M. Francois</span>, <span class="highwire-citation-author" data-delta="1">Faustine Durand</span>, <span class="highwire-citation-author" data-delta="2">Emeric Figuet</span> and <span class="highwire-citation-author" data-delta="3">Nicolas Galtier</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">721-730;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400758 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495529" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Clementine M. Francois </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>UMR 5554, Institut des Sciences de l’Evolution; CNRS, University of Montpellier, IRD, EPHE, Montpellier, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clementine%2BM.%2BFrancois%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Francois%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AClementine%2BM.%2BFrancois%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7781-8781" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Clementine M. Francois</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:clementine.francois@univ-lyon1.fr" class="" data-icon-position="" data-hide-link-title="0">clementine.francois@univ-lyon1.fr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Faustine Durand </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>UMR 5554, Institut des Sciences de l’Evolution; CNRS, University of Montpellier, IRD, EPHE, Montpellier, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Faustine%2BDurand%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Durand%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFaustine%2BDurand%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Emeric Figuet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>UMR 5554, Institut des Sciences de l’Evolution; CNRS, University of Montpellier, IRD, EPHE, Montpellier, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emeric%2BFiguet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Figuet%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmeric%2BFiguet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Nicolas Galtier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>UMR 5554, Institut des Sciences de l’Evolution; CNRS, University of Montpellier, IRD, EPHE, Montpellier, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BGaltier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Galtier%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicolas%2BGaltier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/2/721" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495529" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495529/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/2/721.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495529" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495529/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/2/721.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495529" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495529/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/2/721.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495529" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495529/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Thanks to huge advances in sequencing technologies, genomic resources are increasingly being generated and shared by the scientific community. The quality of such public resources are therefore of critical importance. Errors due to contamination are particularly worrying; they are widespread, propagate across databases, and can compromise downstream analyses, especially the detection of horizontally-transferred sequences. However we still lack consistent and comprehensive assessments of contamination prevalence in public genomic data. Here we applied a standardized procedure for foreign sequence annotation to 43 published arthropod genomes from the widely used Ensembl Metazoa database. This method combines information on sequence similarity and synteny to identify contaminant and putative horizontally-transferred sequences in any genome assembly, provided that an adequate reference database is available. We uncovered considerable heterogeneity in quality among arthropod assemblies, some being devoid of contaminant sequences, whereas others included hundreds of contaminant genes. Contaminants far outnumbered horizontally-transferred genes and were a major confounder of their detection, quantification and analysis. We strongly recommend that automated standardized decontamination procedures be systematically embedded into the submission process to genomic databases.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/contaminant-sequences" class="hw-term hw-article-keyword hw-article-keyword-contaminant-sequences" rel="nofollow">contaminant sequences</a></li><li class="kwd"><a href="/keyword/horizontal-gene-transfer" class="hw-term hw-article-keyword hw-article-keyword-horizontal-gene-transfer" rel="nofollow">horizontal gene transfer</a></li><li class="kwd"><a href="/keyword/automated-detection-pipeline" class="hw-term hw-article-keyword hw-article-keyword-automated-detection-pipeline" rel="nofollow">automated detection pipeline</a></li><li class="kwd"><a href="/keyword/curation-genomic-databases" class="hw-term hw-article-keyword hw-article-keyword-curation-of-genomic-databases" rel="nofollow">curation of genomic databases</a></li></ul><p id="p-4">Scientists typically re-use sequence data generated by others, and are therefore dependent on the reliability of the available genomic resources. For this reason, the problem of public data quality in molecular biology has long been identified as a crucial issue (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lamperti <em>et al.</em> 1992</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Mistry <em>et al.</em> 1993</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Binns 1993</a>). The problem is even more acute nowadays with the advent of high-throughput sequencing technologies, when most datasets generated in genomic research are simply not amenable to manual curation by humans. This brings a new challenge to current methodologies in genomic sciences, namely, the development of automated approaches to the detection and processing of errors (<em>e.g.</em>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Andorf <em>et al.</em> 2007</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Schmieder and Edwards 2011</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Parks <em>et al.</em> 2015</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Delmont and Eren 2016</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Drăgan <em>et al.</em> 2016</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Tennessen <em>et al.</em> 2016</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Laetsch and Blaxter 2017</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lee <em>et al.</em> 2017</a>).</p><p id="p-5">Data quality issues in genome sequences include sequencing errors, assembly errors and contamination, among other things. Errors due to contamination are particularly worrying for several reasons. First, they can lead to serious mis-interpretations of the data, as illustrated by recent, spectacular examples. Potential problems include mis-characterization of gene content and related metabolic functions (<em>e.g.</em>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Koutsovoulos <em>et al.</em> 2016</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Breitwieser <em>et al.</em> 2019</a>), improper inference of evolutionary events (<em>e.g.</em>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Laurin-Lemay <em>et al.</em> 2012</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Simion <em>et al.</em> 2018</a>), and biases in genotype calling and population genomic analyses (<em>e.g.</em>, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Ballenghien <em>et al.</em> 2017</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Wilson <em>et al.</em> 2018</a>). Second, contamination is suspected to be widespread. It occurs naturally in most sequencing projects due to foreign DNA initially present in the raw biological material (<em>e.g.</em>, symbionts, parasites, ingested food; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Salzberg <em>et al.</em> 2005</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Starcevic <em>et al.</em> 2008</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Artamonova and Mushegian 2013</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Driscoll <em>et al.</em> 2013</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Martinson <em>et al.</em> 2014</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Cornet <em>et al.</em> 2018</a>), or entering the process in wet labs and sequencing centers (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Longo <em>et al.</em> 2011</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Salter <em>et al.</em> 2014</a>; <a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Wilson <em>et al.</em> 2018</a>). Third, contamination errors easily propagate across databases in a self-reinforcing vicious circle. If a DNA sequence from species A is initially assigned to the wrong species B due to a contamination of B by A, it is likely to keep its incorrect status for a while, and may even be identified as a contamination of A by B when the genome of A is eventually sequenced (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Merchant <em>et al.</em> 2014</a>). Despite all the possible problems stemming from contamination in genomic resources, most studies addressing this issue so far have focused on one particular genome (<em>e.g.</em>, tardigrades) and/or one particular source of contaminants (<em>e.g.</em>, humans). Only two studies that we are aware of have consistently screened more than one genome assembly. <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Merchant <em>et al.</em> (2014)</a> focused on the bovine genome but also applied their pipeline to eight randomly drawn draft genomes (five animals, two plants, one fungus), with contrasted results. <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Cornet <em>et al.</em> (2018)</a> analyzed 440 genomes of Cyanobacteria and uncovered a substantial level of contamination in &gt;5% of these. There is obviously a need for further assessment of the problem of contamination in publicly available genomic data.</p><p id="p-6">Probably the research goal most sensitive to contamination is the detection of horizontally-transferred genes – nothing resembles a transferred sequence more than a contaminant does. Horizontal gene transfer (HGT) between species is a pervasive process in prokaryotes, which dramatically affects gene phylogenies and species ability to adapt to environmental changes (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Ochman <em>et al.</em> 2000</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Koonin 2016</a>). Whether it substantially influences genome evolution also in large eukaryotes is a matter of debate (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Andersson 2005</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Boto 2014</a>). A number of examples are documented (<em>e.g.</em>, <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Schönknecht <em>et al.</em> 2014</a>), but a quantitative assessment of the prevalence of HGT in eukaryotes is difficult, and many HGT candidates were subsequently shown to result from contamination. Controversies over the confusion between HGT and contaminants have concerned the human genome (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Willerslev <em>et al.</em> 2002</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Salzberg 2017</a>), the <em>Nematostella vectensis</em> sea anemone genome (<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Starcevic <em>et al.</em> 2008</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Artamonova <em>et al.</em> 2015</a>), and the <em>Hypsibius dujardini</em> tardigrade genome, among others. In <em>H. dujardini</em>, the initial estimate of 17% of genes being of foreign origin was revisited to 1% when contamination was properly accounted for (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hashimoto <em>et al.</em> 2016</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Koutsovoulos <em>et al.</em> 2016</a>).</p><p id="p-7">A straightforward way to identify contamination in a newly sequenced genome is to compare the assembled sequences to existing databases using BLAST-like algorithms. If a sequence’s best match is assigned to a species that is phylogenetically distant from the target organism, then the sequence is annotated as a contaminant. There are several problems with this simple strategy. First, this does not allow one to distinguish contaminants from HGT. Second, this approach is entirely dependent on the correctness of the reference database. A best-BLAST-hit survey can only propagate, not correct, pre-existing taxonomic mis-assignments, as discussed above. Third, such an approach is also dependent on the completeness of the reference database, and on the phylogenetic position of the target organism. If the reference database is imbalanced and dominated by one or a few particular taxa (typically model organisms), then its power to properly discriminate genuine sequences from contaminants will be maximal for newly sequenced organisms closely related to the dominant taxa, and much lower for organisms distantly related to the dominant taxa.</p><p id="p-8">Solutions to these problems exist, and include (i) considering multiple BLAST hits, not just the “best” one, (ii) using an appropriately balanced reference database, (iii) incorporating information on synteny (<em>i.e.</em>, physical co-localization of loci on the same scaffold), and ultimately phylogeny, in addition to sequence similarity. Here we collated these ideas in an integrated framework aiming at properly quantifying the prevalence of contamination in genomic data based on reliable, existing tools. We applied this pipeline to 43 published genomes of arthropods distributed in the Ensembl database. We report that data quality is highly heterogeneous across species in this widely used database, some genomes being heavily affected by contamination. Our results also show that a careful annotation of contaminant sequences is mandatory in any subsequent attempt to detect HGT.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials &amp; Methods</h2><div id="sec-2" class="subsection"><h3>Foreign sequence annotation</h3><p id="p-9">We developped a dedicated pipeline for the simultaneous detection of contaminants and HGT candidates in published genome assemblies. This pipeline was optimized and benchmarked in arthropods, but can be applied to any other taxa, provided that an adequate reference database is available. The outline of the pipeline is presented in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. It takes as input a genome assembly and a set of predicted coding sequences (CDS). It returns a set of CDS annotations with the following categories: genuine arthropod gene, HGT candidate, contaminant candidate, orphan gene, uncertain. Five non-metazoan taxonomic groups are considered as potential sources of contaminants and HGT: eubacteria, archaea, fungi, viridiplantae and ‘protists’. Each investigated genome is processed independently and without any <em>a priori</em> on the source(s) of contamination. As discussed below, the power of this pipeline to detect foreign sequences depends on the level of fragmentation of the considered assembly.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/721/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A simplified flow diagram of the pipeline developed for this study. Each species assembly is evaluated independently through this pipeline, which requires the set of coding sequences (CDS) as well as the genomic scaffolds of each species, and an appropriate reference database. In this diagram, boxes referring to &#x2018;data&#x2019;, &#x2018;reference database&#x2019; and &#x2018;tools&#x2019; are colored in blue, green, and red, respectively. See the main text for detailed explanations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490007601" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A simplified flow diagram of the pipeline developed for this study. Each species assembly is evaluated independently through this pipeline, which requires the set of coding sequences (CDS) as well as the genomic scaffolds of each species, and an appropriate reference database. In this diagram, boxes referring to &#x2018;data&#x2019;, &#x2018;reference database&#x2019; and &#x2018;tools&#x2019; are colored in blue, green, and red, respectively. See the main text for detailed explanations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/721/F1.medium.gif" width="378" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/2/721/F1.medium.gif" width="378" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/721/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/721/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495523" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">A simplified flow diagram of the pipeline developed for this study. Each species assembly is evaluated independently through this pipeline, which requires the set of coding sequences (CDS) as well as the genomic scaffolds of each species, and an appropriate reference database. In this diagram, boxes referring to ‘data’, ‘reference database’ and ‘tools’ are colored in blue, green, and red, respectively. See the main text for detailed explanations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">The first step of the pipeline is a preliminary taxonomic assignment of CDS based on sequence similarity. Using DIAMOND BLASTP (v0.8.22, “more-sensitive” mode, otherwise default parameters; Buchfink <em>et al.</em> 2015), each CDS was blasted against a custom protein reference database (see below). Hits with identity greater than 40%, alignment length greater than 75 amino-acids and E-value lower than 10<sup>−10</sup> were retained. A minimum of two such hits to two different species was required for taxonomic assignment. CDS not matching this criterion were regarded as orphan genes (‘no reliable taxonomic assignment’) and not considered further. For each CDS, the 10 hits with the smallest E-values were considered – or less if less than 10 hits had an e-value below 10<sup>−10</sup>. A CDS was assigned to a given taxonomic group (<em>i.e.</em>, eubacteria; archaea; viridiplantae; fungi; protists) if at least 70% of its best hits fell within this group. These were called “foreign CDS candidates”. In addition, a CDS was assigned to the “confident-arthropod” group if 100% of its best hits were to a species of Metazoa, among which at least 70% to a species of Arthropoda. Finally, a CDS was assigned to “other metazoa” if at least 70% of its best hits were to species of non-arthropod metazoa, and none to a species of arthropods. CDS not matching any of these criteria were considered taxonomically unassigned. Using the 10 best hits instead of just the best one provides a robust way to account for potential contaminations and other sources of taxonomic mis-assignment in the reference database. The 70% threshold was empirically determined as providing a reasonable trade-off between sensitivity and specificity.</p><p id="p-12">The second step of the pipeline is a test of synteny. All foreign CDS candidates as well as the “confident-arthropod” CDS were mapped onto the species genomic scaffolds using GMAP (v2017-04-24; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Wu and Watanabe 2005</a>) with the option “-npaths=0”. To account for variable fragmentation of genome assemblies (<em>i.e.</em>, N50), we allow for “chimeric alignments” (<em>i.e.</em>, CDS whose 5 and 3 ends map to different scaffolds). We required a minimum alignment length of 100bp and a minimum identity of 95%. A foreign CDS candidate was considered as a HGT candidate if it was physically linked to (<em>i.e.</em>, mapped to the same scaffold as) at least one “confident-arthropod” CDS. A foreign CDS candidate was considered as a contaminant candidate if it mapped to a scaffold to which no “arthropod-confident” CDS mapped, and at least another non-metazoa CDS mapped. A foreign CDS candidate was considered as “uncertain” if it did not reliably map to any scaffold or if it was the only CDS to map to a given scaffold. When present, the “confident-arthropod” tag was propagated across all scaffolds linked by chimeric alignments. This synteny-based step can also be performed at the contig scale in case of doubts regarding the scaffolding process; this should increase the proportion of foreign candidates classified as “uncertain”.</p><p id="p-13">The corresponding script is available on GitHub (<a href="https://github.com/ClementineFrancois/Foreign-CDS-detection">https://github.com/ClementineFrancois/Foreign-CDS-detection</a>). The analysis of the 43 arthropod assemblies of this study took around 48 hr to run on 50 CPU.</p></div><div id="sec-3" class="subsection"><h3>Evaluated genome assemblies</h3><p id="p-14">The 43 arthropod genomes available in Ensembl Metazoa (Release 37, as of October 2017; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kersey <em>et al.</em> 2017</a>) were investigated using our dedicated pipeline. This included 36 insects, two crustaceans, four chelicerates and one myriapod (see Supplementary Table S1). For each species, the set of masked genomic scaffolds (“dna_rm.toplevel”) as well as the set of all predicted coding sequences (“cds.all”) were retrieved from Ensembl Metazoa. Depending on the species, the set of annotated CDS was either generated by Ensembl or imported from other reference databases relying on different annotation pipelines. Scaffolds shorter than 200 bp were discarded. The longest transcript was selected for each gene. Coding sequences shorter than 150 bp were discarded.</p></div><div id="sec-4" class="subsection"><h3>Custom reference database</h3><p id="p-15">A custom protein reference database was built to cover all domains of life and included 937 species (4,622,809 sequences). The proteomes of 100 eukaryotic species were retrieved from Ensembl (Release 90; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Zerbino <em>et al.</em> 2017</a>) and Ensembl Metazoa (Release 37; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Kersey <em>et al.</em> 2017</a>). These included 40 metazoa (of which 20 arthropods), 20 fungi (of which 10 fungi known to infect arthropods), 20 Viridiplantae and 20 ’protists’. The proteomes of 837 prokaryotic species, of which 748 eubacteria and 89 archae, were retrieved from the Microbial Genome Database for Comparative Analysis (mbgd_2016-01; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Uchiyama <em>et al.</em> 2014</a>) selecting one species per genus. An additional 11 known symbionts of arthropods were subsequently included. Within each proteome, redundant sequences (&gt;90% identity) were removed using CD-HIT (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Fu <em>et al.</em> 2012</a>). Information on the content of the custom reference database is provided in Supplementary Table S2.</p></div><div id="sec-5" class="subsection"><h3>Validation of the contaminant candidates</h3><p id="p-16">In two species of interest, the tetranucleotide (4-mer) frequencies of candidate contaminant CDS were visually compared to those of “confident-arthropod” CDS using a Principal Components Analysis (PCA). PCA was performed in R using the ‘prcomp’ function and results were plotted using the ‘pca3d’ package.</p></div><div id="sec-6" class="subsection"><h3>Validation of the HGT candidates</h3><p id="p-17">We took a phylogenetic approach to validate / invalidate HGT candidates in one species of interest, the pea aphid <em>Acyrthosiphon pisum</em>. All HGT candidates detected in the pea aphid assembly were clustered into families with Silix (v1.2; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Miele <em>et al.</em> 2011</a>), requiring a minimum of 60% of identity (default parameters otherwise). For each family, a protein alignment of the candidate HGT sequence(s) and its (their) 50 best BLAST hits in the custom reference database was generated with MAFFT (v7; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Katoh and Standley 2013</a>). Only BLAST hits with identity greater than 40%, alignment length greater than 75 amino-acids and E-value lower than 10<sup>−10</sup> were considered. The alignements were cleaned using HMMcleaner (stringency parameter = 12). Phylogenetic trees were inferred using RAxML (v8.2; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Stamatakis 2014</a>) with the model ‘PROTGAMMALGX’ of amino-acid substitution and 100 bootstrap replicates. Phylogenetic trees were inspected by eye.</p></div><div id="sec-7" class="subsection"><h3>Statistical analyses</h3><p id="p-18">According to the recommendations of <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Warton and Hui (2011)</a>, all proportion data were logit-transformed prior to statistical analyses, using the ‘car’ R package (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Fox and Weisberg 2011</a>). The normality of the residuals was checked for all models reported in this article. All data analyses were performed with R 3.4 software (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">R Core Team 2018</a>) using the vegan (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Oksanen <em>et al.</em> 2018</a>) and seqinr (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Charif and Lobry 2007</a>) packages.</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-19">This study is based on publicly available data from the Ensembl database (the accession numbers are listed in Table S1).</p><p id="p-20">Table S1 describes the genomic features of the 43 arthropod species from EnsemblMetazoa investigated in this study. Table S2 details the composition of the custom reference database. Table S3 describes the categorization of all CDS in the 43 arthropod genomes. Table S4 indicates the inferred function and potential donor for the six validated HGT families in the pea aphid assembly.</p><p id="p-21">Figure S1 shows the correlation between the log-transformed N50 of each genome assembly and the percentage of foreign CDS candidates initially identified in the 1<sup>st</sup> similarity-based step of the pipeline which were subsequently considered as uncertain in the 2<sup>nd</sup> synteny-based step. Figure S2 shows the number of contaminant and HGT candidates detected in each of the 43 arthropod genomes, according to the assembly N50. Figure S3 displays the Principal Components Analysis of CDS tetranucleotide frequencies in the pea aphid and bumblebee assemblies. Figure S4 shows the distribution of the number of contaminant CDS per contaminant scaffold. Figure S5 contains the RAxML phylogenies inferred for the six validated HGT families in the pea aphid assembly.</p><p id="p-22">Bioinformatic scripts are available on GitHub (<a href="https://github.com/ClementineFrancois/Foreign-CDS-detection">https://github.com/ClementineFrancois/Foreign-CDS-detection</a>). Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.9890894">https://doi.org/10.25387/g3.9890894</a>.</p></div></div><div class="section results-discussion" id="sec-9"><h2 class="">Results &amp; Discussion</h2><div id="sec-10" class="subsection"><h3>Overview of the 43 arthropod genomes: contamination</h3><p id="p-23">We applied our newly introduced contamination/HGT annotation pipeline to 43 assemblies from Ensembl Metazoa. Detailed results are displayed in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and Suppl. Tables S1 &amp; S3. Out of 43 arthropod assemblies, 28 were completely devoid of non-metaozan contamination (including the 12 <em>Drosophila</em> species), while 4 of them contained more than 150 contaminant CDS. The number of predicted contaminant CDS per assembly ranged from 0 to 827 among species, representing 0–5% of all CDS, and 0–8% of the CDS for which a taxonomy assignment was possible – which is probably the most meaningful measure of the prevalence of contamination (Suppl. Table S1). The most contaminated assemblies were those of the bumblebee (<em>Bombus impatiens</em>) and the pea aphid (<em>Acyrthosiphon pisum</em>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/721/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prevalence of contaminant and HGT candidates in the 43 arthropod genomes. Contaminant CDS are classified according to their taxonomic group (i.e., originating from eubacteria, archaea, viridiplantae, fungi and &#x2018;protists&#x2019;). Images courtesy of PhyloPic." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490007601" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Prevalence of contaminant and HGT candidates in the 43 arthropod genomes. Contaminant CDS are classified according to their taxonomic group (&lt;em&gt;i.e.&lt;/em&gt;, originating from eubacteria, archaea, viridiplantae, fungi and &#x2018;protists&#x2019;). Images courtesy of PhyloPic.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/721/F2.medium.gif" width="367" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/2/721/F2.medium.gif" width="367" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/721/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/721/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495506" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-24" class="first-child">Prevalence of contaminant and HGT candidates in the 43 arthropod genomes. Contaminant CDS are classified according to their taxonomic group (<em>i.e.</em>, originating from eubacteria, archaea, viridiplantae, fungi and ‘protists’). Images courtesy of PhyloPic.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">The number of contaminant scaffolds (<em>i.e.</em>, containing at least two contaminant CDS and no genuine arthropod CDS) varied from 0 to 202 across assemblies (Suppl. Table S1). The contaminant CDS were either scattered across many small scaffolds (<em>e.g.</em>, 448 contaminant CDS distributed across 202 scaffolds in the pea aphid) or carried by just a few long contaminant scaffolds (<em>e.g.</em>, 827 contaminant CDS in 30 scaffolds in the bumblebee). The size of contaminant scaffolds ranged from 602 bp (in the barley midge <em>Mayetiola destructor</em>) to 793,321 bp (in the deer tick <em>Ixodes scapularis</em>), and their cumulative length represents up to 2,497,466 bp in the pea aphid <em>Acyrthosiphon pisum</em>.</p><p id="p-26">As an evaluation of the reliability of our results, we tested the taxonomic consistency of the contaminant scaffolds detected in our analyses. Indeed, all CDS encoded on a given contaminant scaffold are expected to derive from the same organism, thus to be assigned to the same non-metazoan group (<em>e.g.</em>, eubacteria). Out of 408 detected contaminant scaffolds, only one was taxonomically inconsistent. This 20kb scaffold from the <em>Lucilia cuprina</em> (blowfly) assembly encoded one eubacterial and two fungal CDS. It could be a chimera between two contaminant sequences.</p><p id="p-27">The great majority of detected contaminations originated from eubacteria (1,796 out of 1,849 contaminant candidates for the 43 species), except in blowfly <em>Lucilia cuprina</em> which was mostly contaminated by fungal sequences (41 CDS; <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and Suppl. Table S3). The fact that no archaeal contamination was detected in any assembly (Suppl. Table S3) might at least in part reflect a taxonomic gap in the public reference databases. This problem has already been evidenced from the study of mammalian gut microbiota (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Raymann <em>et al.</em> 2017</a>) and likely impacts all database-dependent studies.</p><p id="p-28">In summary, out of 43 published genome assemblies, 15 (<em>i.e.</em>, 35%) presented at least some traces of non-metazoan contamination, including four which were substantially contaminated. These figures are likely an underestimation of the actual prevalence of contamination because of the limitation due to incompleteness of reference genomic databases, as discussed above. Moreover, the overall prevalence of contamination is expected to be even higher as we did not consider metazoan contaminants. Yet contamination from wet lab technicians as well as from model organisms extensively used in research facilities (<em>e.g.</em>, mouse, zebrafish, …) is likely to occur in any sequencing project. Our results are consistent with recent analyses which uncovered similar level of contamination in published genome assemblies (<em>e.g.</em>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Borner and Burmester 2017</a>). In particular, Bemm <em>et al.</em> (BioRxiv: <a href="https://doi.org/10.1101/122309">https://doi.org/10.1101/122309</a>) reported from 0 to ca. 5% of bacterial contamination in Ensembl Metazoa genome and identified the bumblebee <em>Bombus impatiens</em> as one of the most highly contaminated assemblies. In addition, our analyses focused on CDS, which are among the most conserved and easy to annotate sequences of a genome, <em>i.e.</em>, probably most easily filtered for contamination by assembly pipelines. Therefore the situation regarding contamination is probably even worse as far as non-coding sequences are concerned.</p></div><div id="sec-11" class="subsection"><h3>Overview of the 43 arthropod genomes: HGT candidates</h3><p id="p-29">In this study, potential HGT candidates were detected at very low level in all genome assemblies. Across the 43 investigated species, the number of CDS suspected to derive from an HGT event ranged from 2 to 81 per assembly, with a median of 12 HGT candidates (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, Suppl. Tables S1 and S3). The HGT candidates represented up to 1.25% of the taxonomically-assigned CDS for a given species (in the spider mite <em>Tetranychus urticae</em>). These HGT candidates have yet to be validated. These results are consistent with several recent studies on this species which evidenced an unexpectedly high level of putative HGT from bacteria and fungi (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Grbić <em>et al.</em> 2011</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ahn <em>et al.</em> 2014</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Altincicek <em>et al.</em> 2012</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Wybouw <em>et al.</em> 2012</a>). The proportion of HGT candidates was rather variable among species, and substantially lower than 1% in a large majority of species (median = 0.14%). Around a third of all detected HGT candidates likely originated from eubacterial donors, and another third from viridiplantae ones (respectively 286 and 295 candidates out of 756; Suppl. Table S3). Similarly to contaminant candidates, very few putative archaeal HGT were detected.</p><p id="p-30">These preliminary results should be considered with a high degree of caution as HGT candidates have not been validated through a phylogenetic approach or an experimental confirmation via PCR or re-sequencing, so the prevalence of HGT in arthropod genomes is likely over-estimated. Still these preliminary results can be compared to previous HGT studies on Metazoa. In a review including 8 metazoan species (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Schönknecht <em>et al.</em> 2014</a>), the number of phylogenetically-supported HGT ranged from 12 to 198 genes across species (with the repeatedly documented exception of the bdelloid rotifers containing 2,700 HGT; see <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Nowell <em>et al.</em> 2018</a>). In another study, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Crisp and colleagues (2015)</a> analyzed 26 metazoan genomes and identified from 2 to 100 HGT across species. Both studies evidenced the same order of HGT prevalence in metazoans as we preliminary did on arthropods.</p></div><div id="sec-12" class="subsection"><h3>Influence of the fragmentation level of the assembly</h3><p id="p-31">Our ability to detect contaminants and HGT decreases with the fragmentation of genome assembly. Indeed, highly fragmented assemblies contain many small scaffolds which are more likely to encode a single CDS. If detected as suspicious in the first step of the pipeline, such CDS (<em>i.e.</em>, alone on their scaffold) would then be considered as uncertain in the second synteny-based step, thus decreasing the power of our pipeline.</p><p id="p-32">The N50 was highly variable across the 43 arthropod assemblies, ranging from 2.3 kb (in the fly <em>Megaselia scalaris</em>) to 41.5 Mb (in the mosquito <em>Anopheles gambiae</em>), with a median at 742 kb (Suppl Table S1). We found a negative correlation between genome assembly N50 and the percentage of foreign CDS candidates classified as “uncertain” at the second step of the pipeline (linear model, p-value = 0.0002, R<sup>2</sup> = 0.2854; Suppl. Figure S1). This indicates that the actual prevalence of contamination was underestimated in our study. Of note, despite the decreased power to detect contaminants and HGT in fragmented assemblies, our pipeline identified high amounts of putative contaminants and HGT in some low-N50 genomes (Suppl. Figure S2). As a matter of fact, the highest contamination levels were identified in low-N50 assemblies (Bombus impatiens, N50 = 1.3 Mb; Acyrthosiphon pisum, N50 = 431 kb).</p></div><div id="sec-13" class="subsection"><h3>Detailed investigation in three species</h3><p id="p-33">Further analyses were performed in three species of interest: the pea aphid (<em>Acyrthosiphon pisum</em>), the bumblebee (<em>Bombus impatiens</em>) and the fruit fly (<em>Drosophila ananassae</em>). We assessed the reliability of our sets of contaminant / HGT candidates and discussed their origin through the analyses of their tetranucleotide content, across-scaffold distribution, and phylogeny.</p><div id="sec-14" class="subsection"><h4>The case of the pea aphid (Acyrthosiphon pisum):</h4><p id="p-34">The genome assembly of the pea aphid showed one of the highest number of predicted contaminant and horizontally-transferred CDS (Suppl. Table S1). We thus investigated in more details the sets of contaminant and HGT candidates.</p><p id="p-35">The phylogenetic signal repeatedly described in tetranucleotide (4-mer) frequencies of CDS means that such frequency patterns convey information about the evolutionary history of the sequence (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Pride <em>et al.</em> 2003</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Teeling <em>et al.</em> 2004</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Dick <em>et al.</em> 2009</a>) and should theoretically enable to discriminate between contaminant and arthropod sequences, similarly to the rationale behind the Blobtools suite (which considers the scaffold %GC; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Laetsch and Blaxter 2017</a>) or the algorithm CONCOCT (for the automated binning of metagenomic contigs; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alneberg <em>et al.</em> 2014</a>). The 4-mer frequencies of the contaminant candidates identified in the pea aphid assembly, as well as those of the ‘confident-arthropod’ CDS, were visualized using a PCA (plotting the three principal components). Almost all contaminant candidates fall outside of the cluster of resident arthropod genes (Suppl. Figure S3a), supporting the reliability of the set of contaminants identified in the pea aphid assembly.</p><p id="p-36">In this assembly, the 448 predicted contaminant CDS derived from 202 scaffolds. Contaminant CDS were scattered across many small scaffolds harboring only a few CDS each, a pattern similar to most of the screeened assemblies (Suppl. Figure S4). 99.5% of the contaminant scaffolds (201 out of 202) were from eubacterial origin. An examination of the taxonomy of BLAST hits indicated that a vast majority of contaminant sequences originated from donors of the order Enterobacterales, and showed closest matches to species of the families Enterobacteriaceae and Erwiniaceae. Interestingly, these two families contain several well-described bacterial symbionts of aphids, such as the obligate endosymbiont <em>Buchnera aphidicola</em> or the facultative symbionts <em>Hamiltonella defensa</em> and <em>Serratia symbiotica</em> (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Oliver <em>et al.</em> 2010</a>). However, none of the detected contaminant CDS blasted reliably on the genome of <em>Buchnera aphidicola</em> nor on the genomes of common aphid secondary symbionts (<em>Hamiltonella defensa</em>, <em>Serratia symbiotica</em>, <em>Spiroplasma</em>, <em>Cardinium</em> and <em>Rickettsia</em>), although these species were represented in our reference database. Symbiont-derived sequences were likely present in the raw dataset and subsequently removed from the assembly, which is a common procedure in sequencing projects (<em>e.g.</em>, see <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">International Aphid Genomics Consortium 2010</a>). This targeted cleaning approach can only be applied to well-known symbionts of the focal organism. The remaining contaminant sequences might thus correspond to less studied aphid symbionts, such as species of the genera <em>Pantoea or Erwinia</em>, which showed strong BLAST matches with contaminant CDS and have been identified as aphid gut symbionts (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Harada <em>et al.</em> 1997</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gauthier <em>et al.</em> 2015</a>).</p><p id="p-37">The 75 HGT candidates detected in the pea aphid assembly clustered in 70 gene families, from which gene phylogenies were constructed. The 70 trees were inspected by eye, and only six of them were considered as reliably supporting an instance of HGT (Suppl. Figure S5). The other 64 trees were disregarded mainly because the terminal branch leading to the putatively-transferred sequence was too long for a reliable phylogenetic placement. Five HGT likely originated from eubacterial donors (Suppl. Table S4), including a transposase gene. The remaining HGT concerned four CDS, which were likely acquired from a fungus (Suppl. Figure S5, Suppl. Table S4). The functional annotations of the best BLASTP hits (NR database) suggest that the horizontally-transferred genes of fungal origin encode a phytoene desaturase, an enzyme involved in carotenoid biosynthesis. This result is congruent with previous studies in aphids wich indicated that the phytoene desaturase gene had undergone several duplication events after its transfer from a fungal donor (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Nováková and Moran 2011</a>). This HGT event seems to be ancient and shared with the red spider mite T<em>etranychus urticae</em> (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Altincicek <em>et al.</em> 2011</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Grbić <em>et al.</em> 2011</a>), which is consistent with the phylogenetic tree we inferred for this gene family (<em>cf</em>. Suppl. Figure S5). Carotenoid pigments can confer many essential benefits (<em>e.g.</em>, protection from oxidative damage, light detection, photoprotection, signaling) and are acquired by most animals from their diet. HGT events enabling an organism to <em>de novo</em> synthetise carotenoids could confer a substantial adaptive advantage to the recipient species (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Bryon <em>et al.</em> 2017</a>).</p><p id="p-38">Of note, only a minority of the suspected HGT (six out of 70) were confirmed via our phylogenetic analysis. This confirms that evidence solely based on sequence similarity are not sufficient to demonstrate the existence of an HGT event, far from it. A phylogeny-based validation is required. For example the controversy on human genome demonstrated that most, if not all, putative horizontally-transferred sequences initially identified through a BLAST approach, actually originated from classical vertical descent (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Stanhope <em>et al.</em> 2001</a>).</p></div><div id="sec-15" class="subsection"><h4>The case of the bumblebee (Bombus impatiens):</h4><p id="p-39">The genome assembly of the bumblebee represents a particularly striking example of host genome contamination by symbiont sequences. In this assembly, the 827 predicted contaminant CDS derived from only 30 scaffolds.</p><p id="p-40">Using the same approach as described above in the pea aphid, the 4-mer frequencies of contaminant candidates and ‘confident-arthropod’ sequences were visualized using a PCA, which clearly separated the two sets of CDS (Suppl. Figure S3b). This pattern supported the reliability of the set of contaminants identified in the bumblebee assembly.</p><p id="p-41">The 827 contaminant CDS were concentrated in only 30 contaminant scaffolds harboring up to 108 CDS each, a pattern strikingly different from the other assemblies we analyzed (Suppl. Figure S4). All contaminant sequences were of eubacterial origin, and ca. 97% (799 out of 827) consistently showed high sequence similarity with two species of the Orbaceae family present in our reference database, namely <em>Gilliamella apicola</em> and <em>Frischella perrara</em>. These 799 Orbaceae CDS correspond to just 25 contaminant scaffolds, the lengths of which sum up to 2,157,077 bp. <em>Gilliamella apicola</em> is known to be a gut symbiont of bumblebees and its genome size is ∼2.2 Mb (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kwong and Moran 2013</a>), suggesting at first sight that the whole genome of this species could be included in the bumblebee assembly. However, <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Martinson <em>et al.</em> (2014)</a> described a new bumblebee gut symbiont sequenced concurrently with the genome of its host. This symbiont, <em>Candidatus</em> Schmidhempelia bombi, is another good candidate as it was not present in our reference database, has a genome size of at least 2 Mb, and shares significant sequence similarity with <em>Frischella perrara</em> and <em>Gilliamella apicola</em>. All contaminant CDS were blasted against the three assemblies available in NCBI (<em>Candidatus</em> Schmidhempelia bombi str. Bimp; <em>Gilliamella apicola</em> str. WkB30; <em>Frischella perrara</em> str. PEB0191). 769 CDS out of 827 showed 100% nucleotide similarity with sequences of <em>Candidatus</em> Schmidhempelia bombi. The maximum sequence similarity with <em>Frischella perrara</em> and <em>Gilliamella apicola</em> were 95% et 87%, respectively. We conclude that almost the entire genome of <em>Candidatus</em> Schmidhempelia bombi is present in the bumblebee assembly distributed by Ensembl Metazoa, although this symbiont was described and its genome sequence published in 2014 (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Martinson <em>et al.</em> 2014</a>).</p><p id="p-42">The bumblebee assembly is therefore a textbook example of a complete symbiont genome accidentally sequenced alongside the focal organism and mistakenly incorporated into the primary assembly (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Sadd <em>et al.</em> 2015</a>). As of today, while both NCBI and the European Nucleotide Archive have twice updated the bumblebee assembly since March 2018 (exclusion of bacterial sequences; BIMP_2.2, GCA_000188095.4), EnsemblMetazoa is still distributing the first version of the assembly (BIMP_2.0) which includes the endosymbiont sequences, with obvious implications regarding downstream analyses.</p></div><div id="sec-16" class="subsection"><h4>The case of Drosophila ananassae:</h4><p id="p-43">We focused on <em>Drosophila ananassae</em> because several studies demonstrated widespread HGT from <em>Wolbachia</em> into the genome of this species (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hotopp <em>et al.</em> 2007</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Klasson <em>et al.</em> 2014</a>). However, only three eubacterial HGT candidates were detected by our pipeline, even though four <em>Wolbachia</em> strains were present in our reference database. Besides, none of these HGT candidates showed any good match with <em>Wolbachia</em> sequences when blasted against NR NCBI. This discrepancy could have been explained if these HGT occurred a long time ago, causing horizontally-transferred sequences to degenerate beyond the point where they would be recognized as CDS, and thus would not have been screened in our pipeline. However, at least 28 of these <em>Wolbachia</em> horizontally-transferred sequences seem to be expressed at low abundance in <em>D. ananassae</em> (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Hotopp <em>et al.</em> 2007</a>), suggesting that they are not too degenerate to be transcribed. Another explanation would be an excessive cleaning of <em>Drosophila ananassae</em> assembly causing all foreign (HGT and contaminant) sequences to be systematically removed, regardless of their physical integration into the fly genome. This would explain why none of the previously described HGT was detected. This hypothesis is supported by the fact that no contaminant sequence was detected in any of the 12 <em>Drosophila</em> assemblies (Suppl. Table S1). A last hypothesis would be that horizontally-transferred sequences are still functional and present on the genomic scaffolds, but were somehow filtered out during the annotation step (prediction of CDS; imported from FlyBase for all Drosophila species) and thus were not screened by our pipeline. This hypothesis is supported by the fact that 239 proteins of the <em>Wolbachia</em> reference assembly (ASM367136v1, NCBI) showed good BLASTX matches with the genomic scaffolds of <em>Drosophila ananassae</em> (E-value &lt; 10<sup>−30</sup>). It should be noted that Ensembl has updated the gene set of <em>D. ananassae</em> since our analyses, adding thousands of CDS (release dana_r1.05 from FlyBase), including some with high similarity to Wolbachia sequences.</p><p id="p-44">This example illustrates the potential downstream impacts of the cleaning and annotation procedures implemented in genome sequencing projects, which can result in <em>bona fide</em> genes of interest being discarded, and therefore taken away from the genomic databases and literature. Moreover, the lack of specific documentation on the procedures implemented for each assembly makes the (frequent) successive versions changes hardly tractable for the users, although these can have substantial impact on the distributed genomic data.</p></div></div></div><div class="section conclusions" id="sec-17"><h2 class="">Conclusions</h2><p id="p-45">Identifying genes of foreign origin in a genome is a goal of major biological interest, which is required to properly account for the problem of contamination in published genome assemblies. Applying a consistent, automated, reproducible foreign sequence annotation pipeline, we revealed considerable heterogeneity among arthropod genomes from the Ensembl Metazoa database in terms of prevalence of contaminants. Of the 43 arthropod assemblies we analyzed, 28 were completely devoid of contaminant sequences (including the 12 <em>Drosophila</em> species), 11 included a few, while four of them were heavily affected (&gt; 150 contaminant CDS). The highest level of contamination was detected in the bumblebee assembly which contained 827 contaminant CDS likely originating from a single endosymbiont. This disparity between entries of a single, widely used database is worth noting, beyond the heterogeneity of annotation procedures among genome assemblies. Some of the Ensembl Metazoa assemblies were “cleaned” to the point that previously documented HGT have been removed, whereas others included hundreds of contaminant genes. Most of the detected foreign sequences proved to be contaminants, while very few HGT were confirmed. Therefore any analysis of HGT solely based on existing gene annotations would presumably yield results of little, if any, biological relevance. Contamination is in large part unavoidable and a major confounder of all downstream genomic analyses. While researchers should be accountable for the cleaning of their NGS datasets prior to distribution, there is inevitably some heterogeneity among labs and consortiums in terms of procedures and scientific goals. Thus we recommend that reproducible decontamination procedures (<em>e.g.</em>, <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Tennessen <em>et al.</em> 2016</a>; <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Laetsch and Blaxter 2017</a>; this study) be systematically embedded into the submission process to genomic databases.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-46">These analyses largely benefited from the Montpellier Bioinformatics Biodiversity computing cluster platform. The authors would particularly like to thank Dr. I. Uchiyama for his help in extracting microbial genomes from the MBGD database. We are grateful to Céline Scornavacca, Paul Simion and Yoann Anselmetti for valuable help with bioinformatic scripts. We thank the editors and the anonymous reviewers for their valuable comments on an earlier version of this manuscript. This work was supported by Swiss SNF grant number PP00P3_170627.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-47">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.9890894">https://doi.org/10.25387/g3.9890894</a>.</p></li><li class="fn" id="fn-3"><p id="p-48"><em>Communicating editor: R. Kulathinal</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-09-23"><span class="received-label">Received </span>September 23, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-12-10"><span class="accepted-label">Accepted </span>December 10, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Francois <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.1" data-doi="10.1016/j.ibmb.2014.04.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ahn</span>,  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Dermauw</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Wybouw</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. G.</span> <span class="cit-name-surname">Heckel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Van Leeuwen</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Bacterial origin of a diverse family of UDP-glycosyltransferase genes in the Tetranychus urticae genome.</span> <abbr class="cit-jnl-abbrev">Insect Biochem. Mol. Biol.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">57</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ibmb.2014.04.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInsect%2BBiochem.%2BMol.%2BBiol.%26rft.volume%253D50%26rft.spage%253D43%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ibmb.2014.04.003%26rft_id%253Dinfo%253Apmid%252F24727020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ibmb.2014.04.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24727020&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.2" data-doi="10.1038/nmeth.3103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alneberg</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. S.</span> <span class="cit-name-surname">Bjarnason</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">De Bruijn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Schirmer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Quick</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Binning metagenomic contigs by coverage and composition.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1144</span>–<span class="cit-lpage">1146</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.3103</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D11%26rft.spage%253D1144%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3103%26rft_id%253Dinfo%253Apmid%252F25218180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25218180&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000344580600021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.3" data-doi="10.1098/rsbl.2011.0704"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altincicek</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Kovacs</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. M.</span> <span class="cit-name-surname">Gerardo</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Horizontally transferred fungal carotenoid genes in the two-spotted spider mite <em>Tetranychus urticae</em>.</span> <abbr class="cit-jnl-abbrev">Biol. Lett.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">253</span>–<span class="cit-lpage">257</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rsbl.2011.0704</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BLett.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsbl.2011.0704%26rft_id%253Dinfo%253Apmid%252F21920958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rsbl.2011.0704&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21920958&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.4" data-doi="10.1007/s00018-005-4539-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>,  <span class="cit-name-given-names">J. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Lateral gene transfer in eukaryotes.</span> <abbr class="cit-jnl-abbrev">Cell. Mol. Life Sci.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">1182</span>–<span class="cit-lpage">1197</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00018-005-4539-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCellular%2Band%2Bmolecular%2Blife%2Bsciences%2B%253A%2B%2BCMLS%26rft.stitle%253DCell%2BMol%2BLife%2BSci%26rft.aulast%253DAndersson%26rft.auinit1%253DJ.%2BO.%26rft.volume%253D62%26rft.issue%253D11%26rft.spage%253D1182%26rft.epage%253D1197%26rft.atitle%253DLateral%2Bgene%2Btransfer%2Bin%2Beukaryotes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00018-005-4539-z%26rft_id%253Dinfo%253Apmid%252F15761667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00018-005-4539-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15761667&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232496800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.5" data-doi="10.1186/1471-2105-8-284"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andorf</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Dobbs</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Honavar</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Exploring inconsistencies in genome-wide protein function annotations: a machine learning approach.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">284</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-8-284</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DAndorf%26rft.auinit1%253DC.%26rft.volume%253D8%26rft.spage%253D284%26rft.epage%253D284%26rft.atitle%253DExploring%2Binconsistencies%2Bin%2Bgenome-wide%2Bprotein%2Bfunction%2Bannotations%253A%2Ba%2Bmachine%2Blearning%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-8-284%26rft_id%253Dinfo%253Apmid%252F17683567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-8-284&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17683567&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.6" data-doi="10.1111/1462-2920.12854"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Artamonova</span>,  <span class="cit-name-given-names">I. I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Lappi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Zudina</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Mushegian</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Prokaryotic genes in eukaryotic genome sequences: when to infer horizontal gene transfer and when to suspect an actual microbe.</span> <abbr class="cit-jnl-abbrev">Environ. Microbiol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">2203</span>–<span class="cit-lpage">2208</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1462-2920.12854</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnviron.%2BMicrobiol.%26rft.volume%253D17%26rft.spage%253D2203%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1462-2920.12854%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1462-2920.12854&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.7" data-doi="10.1128/AEM.01635-13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Artamonova</span>,  <span class="cit-name-given-names">I. I.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Mushegian</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome sequence analysis indicates that the model eukaryote <em>Nematostella vectensis</em> harbors bacterial consorts.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">6868</span>–<span class="cit-lpage">6873</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/AEM.01635-13</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.01635-13%26rft_id%253Dinfo%253Apmid%252F23995941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI3OS8yMi82ODY4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi83MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.8" data-doi="10.1186/s12915-017-0366-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ballenghien</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Faivre</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Galtier</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Patterns of cross-contamination in a multispecies population genomic project: detection, quantification, impact, and solutions.</span> <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">25</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12915-017-0366-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBiol.%26rft.volume%253D15%26rft.spage%253D25%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12915-017-0366-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12915-017-0366-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.9" data-doi="10.1093/nar/21.3.779"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Binns</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Contamination of DNA database sequence entries with <em>Escherichia coli</em> insertion sequences.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">779</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/21.3.779</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DBinns%26rft.auinit1%253DM.%26rft.volume%253D21%26rft.issue%253D3%26rft.spage%253D779%26rft.epage%253D779%26rft.atitle%253DContamination%2Bof%2BDNA%2Bdatabase%2Bsequence%2Bentries%2Bwith%2BEscherichia%2Bcoli%2Binsertion%2Bsequences%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F21.3.779%26rft_id%253Dinfo%253Apmid%252F8382807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/21.3.779&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8382807&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.10" data-doi="10.1186/s12864-017-3504-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borner</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Burmester</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Parasite infection of public databases: a data mining approach to identify apicomplexan contaminations in animal genome and transcriptome assemblies.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">100</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-017-3504-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D18%26rft.spage%253D100%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-017-3504-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-017-3504-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.11" data-doi="10.1098/rspb.2013.2450"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boto</span>,  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Horizontal gene transfer in the acquisition of novel traits by metazoans.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">281</span>: <span class="cit-elocation-id">20132450</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rspb.2013.2450</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2013.2450%26rft_id%253Dinfo%253Apmid%252F24403327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2013.2450&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24403327&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.12" data-doi="10.1101/gr.245373.118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breitwieser</span>,  <span class="cit-name-given-names">F. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Pertea</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Zimin</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Salzberg</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Human contamination in bacterial genomes has created thousands of spurious proteins.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">954</span>–<span class="cit-lpage">960</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.245373.118</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.245373.118%26rft_id%253Dinfo%253Apmid%252F31064768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI5LzYvOTU0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi83MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.13" data-doi="10.1073/pnas.1706865114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bryon</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. H.</span> <span class="cit-name-surname">Kurlovs</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Dermauw</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Greenhalgh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Riga</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Disruption of a horizontally transferred phytoene desaturase abolishes carotenoid accumulation and diapause in <em>Tetranychus urticae</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">E5871</span>–<span class="cit-lpage">E5880</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1706865114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1706865114%26rft_id%253Dinfo%253Apmid%252F28674017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzI5L0U1ODcxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi83MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.2.721.14" data-doi="10.1038/nmeth.3176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buchfink</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Xie</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. H.</span> <span class="cit-name-surname">Huson</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Fast and sensitive protein alignment using DIAMOND.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.3176</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D12%26rft.spage%253D59%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3176%26rft_id%253Dinfo%253Apmid%252F25402007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3176&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25402007&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.721.15" data-doi="10.1007/978-3-540-35306-5_10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charif</span>,  <span class="cit-name-given-names">D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Lobry</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">SeqinR 1.0–2: a contributed package to the R project for statistical computing devoted to biological sequences retrieval and analysis</span>, pp. <span class="cit-fpage">207</span>–<span class="cit-lpage">232</span> in <span class="cit-source"><em>Structural approaches to sequence evolution: Molecules</em>, <em>networks</em>, <em>populations</em>, <em>Biological and Medical Physics</em>, <em>Biomedical Engineering</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Bastolla</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Porto</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Roman</span></span>, and <span class="cit-ed"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Vendruscolo</span></span>. <span class="cit-publ-name">Springer Verlag</span>, <span class="cit-publ-loc">New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-540-35306-5_10</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStructural%2Bapproaches%2Bto%2Bsequence%2Bevolution%253A%2BMolecules%252C%2Bnetworks%252C%2Bpopulations%252C%2BBiological%2Band%2BMedical%2BPhysics%252C%2BBiomedical%2BEngineering%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-540-35306-5_10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-540-35306-5_10&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.16" data-doi="10.1371/journal.pone.0200323"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cornet</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Meunier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Van Vlierberghe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. R.</span> <span class="cit-name-surname">Leonard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Durieu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Consensus assessment of the contamination level of publicly available cyanobacterial genomes.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">13</span>: <span class="cit-elocation-id">e0200323</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0200323</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D13%26rft.spage%253De0200323%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0200323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0200323&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.17" data-doi="10.1186/s13059-015-0607-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crisp</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Boschetti</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Perry</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Tunnacliffe</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Micklem</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Expression of multiple horizontally acquired genes is a hallmark of both vertebrate and invertebrate genomes.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">50</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-015-0607-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D50%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0607-3%26rft_id%253Dinfo%253Apmid%252F25785303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0607-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25785303&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.18" data-doi="10.7717/peerj.1839"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Delmont</span>,  <span class="cit-name-given-names">T. O.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Eren</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Identifying contamination with advanced visualization and analysis practices: metagenomic approaches for eukaryotic genome assemblies.</span> <abbr class="cit-jnl-abbrev">PeerJ</abbr> <span class="cit-vol">4</span>: <span class="cit-elocation-id">e1839</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7717/peerj.1839</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeerJ%26rft.volume%253D4%26rft.spage%253D1839e%26rft_id%253Dinfo%253Adoi%252F10.7717%252Fpeerj.1839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7717/peerj.1839&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.19" data-doi="10.1186/gb-2009-10-8-r85"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dick</span>,  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. F.</span> <span class="cit-name-surname">Andersson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Baker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Simmons</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. C.</span> <span class="cit-name-surname">Thomas</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Community-wide analysis of microbial genome sequence signatures.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R85</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2009-10-8-r85</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DDick%26rft.auinit1%253DG.%2BJ.%26rft.volume%253D10%26rft.issue%253D8%26rft.spage%253DR85%26rft.epage%253DR85%26rft.atitle%253DCommunity-wide%2Banalysis%2Bof%2Bmicrobial%2Bgenome%2Bsequence%2Bsignatures.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-8-r85%26rft_id%253Dinfo%253Apmid%252F19698104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-8-r85&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19698104&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.20" data-doi="10.1093/bioinformatics/btw015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drăgan</span>,  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Moghul</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Priyam</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Bustos</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wurm</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">GeneValidator: identify problems with protein-coding gene predictions.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1559</span>–<span class="cit-lpage">1561</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btw015</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw015%26rft_id%253Dinfo%253Apmid%252F26787666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26787666&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.21" data-doi="10.1093/gbe/evt036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Driscoll</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Gillespie</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. K.</span> <span class="cit-name-surname">Nordberg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. F.</span> <span class="cit-name-surname">Azad</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. W.</span> <span class="cit-name-surname">Sobral</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Bacterial DNA sifted from the <em>Trichoplax adhaerens</em> (Animalia: Placozoa) genome project reveals a putative rickettsial endosymbiont.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">621</span>–<span class="cit-lpage">645</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evt036</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevt036%26rft_id%253Dinfo%253Apmid%252F23475938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evt036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23475938&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.721.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fox</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Weisberg</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-source">An R Companion to Applied Regression</span>, <span class="cit-edition">Ed. 2nd</span>. <span class="cit-publ-name">Sage</span>, <span class="cit-publ-loc">Thousand Oaks, CA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.23" data-doi="10.1093/bioinformatics/bts565"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Niu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zhu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Wu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Li</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">CD-HIT: accelerated for clustering the next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">3150</span>–<span class="cit-lpage">3152</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts565</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts565%26rft_id%253Dinfo%253Apmid%252F23060610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts565&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23060610&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311902700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.24" data-doi="10.1371/journal.pone.0120664"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gauthier</span>,  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Outreman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Mieuzet</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Simon</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Bacterial communities associated with host-adapted populations of pea aphids revealed by deep sequencing of 16S ribosomal DNA.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-elocation-id">e0120664</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0120664</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0120664%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0120664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0120664&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.25" data-doi="10.1038/nature10640"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grbić</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Van Leeuwen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Clark</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Rombauts</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Rouzé</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The genome of <em>Tetranychus urticae</em> reveals herbivorous pest adaptations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">479</span>: <span class="cit-fpage">487</span>–<span class="cit-lpage">492</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature10640</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGrbic%26rft.auinit1%253DM.%26rft.volume%253D479%26rft.issue%253D7374%26rft.spage%253D487%26rft.epage%253D492%26rft.atitle%253DThe%2Bgenome%2Bof%2BTetranychus%2Burticae%2Breveals%2Bherbivorous%2Bpest%2Badaptations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10640%26rft_id%253Dinfo%253Apmid%252F22113690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10640&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22113690&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000297285600041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.26" data-doi="10.2323/jgam.43.349"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harada</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Oyaizu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Kosako</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Ishikawa</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title"><em>Erwinia aphidicola</em>, a new species isolated from pea aphid, <em>Acyrthosiphon pisum</em>.</span> <abbr class="cit-jnl-abbrev">J. Gen. Appl. Microbiol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">354</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2323/jgam.43.349</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BGen%2BAppl%2BMicrobiol%26rft.aulast%253DHarada%26rft.auinit1%253DH.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D349%26rft.epage%253D354%26rft.atitle%253DErwinia%2Baphidicola%252C%2Ba%2Bnew%2Bspecies%2Bisolated%2Bfrom%2Bpea%2Baphid%252C%2BAcyrthosiphon%2Bpisum.%26rft_id%253Dinfo%253Adoi%252F10.2323%252Fjgam.43.349%26rft_id%253Dinfo%253Apmid%252F12501306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2323/jgam.43.349&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12501306&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.27" data-doi="10.1038/ncomms12808"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hashimoto</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. D.</span> <span class="cit-name-surname">Horikawa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Saito</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Kuwahara</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Kozuka-Hata</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Extremotolerant tardigrade genome and improved radiotolerance of human cultured cells by tardigrade-unique protein.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">12808</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms12808</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D7%26rft.spage%253D12808%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms12808%26rft_id%253Dinfo%253Apmid%252F27649274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms12808&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27649274&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.28" data-doi="10.1126/science.1142490"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hotopp</span>,  <span class="cit-name-given-names">J. C. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Clark</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. C.</span> <span class="cit-name-surname">Oliveira</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Foster</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Fischer</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Widespread lateral gene transfer from intracellular bacteria to multicellular eukaryotes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">317</span>: <span class="cit-fpage">1753</span>–<span class="cit-lpage">1756</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1142490</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1142490%26rft_id%253Dinfo%253Apmid%252F17761848%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTcvNTg0NS8xNzUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi83MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.29" data-doi="10.1371/journal.pbio.1000313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">International Aphid Genomics Consortium</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome sequence of the pea aphid <em>Acyrthosiphon pisum</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e1000313</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.1000313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253De1000313%26rft.epage%253De1000313%26rft.atitle%253DGenome%2Bsequence%2Bof%2Bthe%2Bpea%2Baphid%2BAcyrthosiphon%2Bpisum.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000313%26rft_id%253Dinfo%253Apmid%252F20186266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000313&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20186266&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.30" data-doi="10.1093/molbev/mst010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katoh</span>,  <span class="cit-name-given-names">K.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Standley</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">772</span>–<span class="cit-lpage">780</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/mst010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst010%26rft_id%253Dinfo%253Apmid%252F23329690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23329690&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000317002300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.31" data-doi="10.1093/nar/gkx1011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kersey</span>,  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Allen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Allot</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Barba</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Boddu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Ensembl Genomes 2018: an integrated omics infrastructure for non-vertebrate species.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">D802</span>–<span class="cit-lpage">D808</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkx1011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D46%26rft.spage%253DD802%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx1011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.32" data-doi="10.1186/1471-2164-15-1097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klasson</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Kumar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Bromley</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Sieber</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Flowers</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Extensive duplication of the <em>Wolbachia</em> DNA in chromosome four of <em>Drosophila ananassae</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1097</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-1097</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D1097%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-1097%26rft_id%253Dinfo%253Apmid%252F25496002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-1097&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25496002&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.33" data-doi="10.12688/f1000research.8737.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koonin</span>,  <span class="cit-name-given-names">E. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Horizontal gene transfer: essentiality and evolvability in prokaryotes, and roles in evolutionary transitions.</span> <abbr class="cit-jnl-abbrev">F1000 Res.</abbr> <span class="cit-vol">5</span>: 1805.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.12688/f1000research.8737.1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DF1000%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.12688%252Ff1000research.8737.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.12688/f1000research.8737.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.34" data-doi="10.1073/pnas.1600338113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koutsovoulos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kumar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Laetsch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Stevens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Daub</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">No evidence for extensive horizontal gene transfer in the genome of the tardigrade <em>Hypsibius dujardini</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">5053</span>–<span class="cit-lpage">5058</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1600338113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1600338113%26rft_id%253Dinfo%253Apmid%252F27035985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEzLzE4LzUwNTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzcyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.35" data-doi="10.1099/ijs.0.044875-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kwong</span>,  <span class="cit-name-given-names">W. K.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Moran</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Cultivation and characterization of the gut symbionts of honey bees and bumble bees: description of snodgrassella alvi gen. nov., sp. nov., a member of the family neisseriaceae of the betaproteobacteria, and gilliamella apicola gen. nov., sp. nov., a member of orbaceae fam. nov., orbales ord. nov., a sister taxon to the order ‘enterobacteriales’ of the gammaproteobacteria.</span> <abbr class="cit-jnl-abbrev">Int. J. Syst. Evol. Microbiol.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">2008</span>–<span class="cit-lpage">2018</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1099/ijs.0.044875-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BSyst.%2BEvol.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fijs.0.044875-0%26rft_id%253Dinfo%253Apmid%252F23041637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/ijs.0.044875-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23041637&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000322348700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laetsch</span>,  <span class="cit-name-given-names">D. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Blaxter</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Blobtools: Interrogation of genome assemblies.</span> <abbr class="cit-jnl-abbrev">F1000 Res.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1287</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DF1000%2BRes.%26rft.volume%253D6%26rft.spage%253D1287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.37" data-doi="10.1093/nar/20.11.2741"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lamperti</span>,  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Kittelberger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. F.</span> <span class="cit-name-surname">Smith</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Villa-Komaroff</span></span></li></ol><cite>,<span class="cit-pub-date"> 1992</span> <span class="cit-article-title">Corruption of genomic databases with anomalous sequence.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">2741</span>–<span class="cit-lpage">2747</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/20.11.2741</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DLamperti%26rft.auinit1%253DE.%2BD.%26rft.volume%253D20%26rft.issue%253D11%26rft.spage%253D2741%26rft.epage%253D2747%26rft.atitle%253DCorruption%2Bof%2Bgenomic%2Bdatabases%2Bwith%2Banomalous%2Bsequence%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F20.11.2741%26rft_id%253Dinfo%253Apmid%252F1614861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/20.11.2741&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1614861&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HZ67300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.38" data-doi="10.1016/j.cub.2012.06.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laurin-Lemay</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Brinkmann</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Philippe</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Origin of land plants revisited in the light of sequence contamination and missing data.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">R593</span>–<span class="cit-lpage">R594</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cub.2012.06.013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DLaurin-Lemay%26rft.auinit1%253DS.%26rft.volume%253D22%26rft.issue%253D15%26rft.spage%253DR593%26rft.epage%253DR594%26rft.atitle%253DOrigin%2Bof%2Bland%2Bplants%2Brevisited%2Bin%2Bthe%2Blight%2Bof%2Bsequence%2Bcontamination%2Band%2Bmissing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2012.06.013%26rft_id%253Dinfo%253Apmid%252F22877776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2012.06.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22877776&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.39" data-doi="10.1099/ijsem.0.001872"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Chalita</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Ha</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. I.</span> <span class="cit-name-surname">Na</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. H.</span> <span class="cit-name-surname">Yoon</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">ContEst16S: an algorithm that identifies contaminated prokaryotic genomes using 16S RNA gene sequences.</span> <abbr class="cit-jnl-abbrev">Int. J. Syst. Evol. Microbiol.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">2053</span>–<span class="cit-lpage">2057</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1099/ijsem.0.001872</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BSyst.%2BEvol.%2BMicrobiol.%26rft.volume%253D67%26rft.spage%253D2053%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fijsem.0.001872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/ijsem.0.001872&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.40" data-doi="10.1371/journal.pone.0016410"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Longo</span>,  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">O’Neill</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">O’Neill</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Abundant human DNA contamination identified in non-primate genome databases.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e16410</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0016410</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLongo%26rft.auinit1%253DM.%2BS.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253De16410%26rft.epage%253De16410%26rft.atitle%253DAbundant%2Bhuman%2BDNA%2Bcontamination%2Bidentified%2Bin%2Bnon-primate%2Bgenome%2Bdatabases.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0016410%26rft_id%253Dinfo%253Apmid%252F21358816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0016410&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21358816&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.41" data-doi="10.1128/AEM.00322-14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martinson</span>,  <span class="cit-name-given-names">V. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Magoc</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Koch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Salzberg</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Moran</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic features of a bumble bee symbiont reflect its host environment.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">3793</span>–<span class="cit-lpage">3803</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/AEM.00322-14</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.00322-14%26rft_id%253Dinfo%253Apmid%252F24747890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI4MC8xMy8zNzkzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi83MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.42" data-doi="10.7717/peerj.675"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Merchant</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">Wood</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Salzberg</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Unexpected cross-species contamination in genome sequencing projects.</span> <abbr class="cit-jnl-abbrev">PeerJ</abbr> <span class="cit-vol">2</span>: <span class="cit-elocation-id">e675</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7717/peerj.675</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeerJ%26rft.volume%253D2%26rft.spage%253D675e%26rft_id%253Dinfo%253Adoi%252F10.7717%252Fpeerj.675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7717/peerj.675&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.43" data-doi="10.1186/1471-2105-12-116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miele</span>,  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Penel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Duret</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ultra-fast sequence clustering from similarity networks with Silix.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">116</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DMiele%26rft.auinit1%253DV.%26rft.volume%253D12%26rft.spage%253D116%26rft.epage%253D116%26rft.atitle%253DUltra-fast%2Bsequence%2Bclustering%2Bfrom%2Bsimilarity%2Bnetworks%2Bwith%2BSiLiX.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-116%26rft_id%253Dinfo%253Apmid%252F21513511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21513511&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.44" data-doi="10.1126/science.8480169"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mistry</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Greenlee</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Fong</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Database contamination.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">260</span>: <span class="cit-fpage">605</span>–<span class="cit-lpage">606</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.8480169</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMistry%26rft.auinit1%253DA%26rft.volume%253D260%26rft.issue%253D5108%26rft.spage%253D605%26rft.epage%253D606%26rft.atitle%253DDatabase%2Bcontamination%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.8480169%26rft_id%253Dinfo%253Apmid%252F8480169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJzY2kiO3M6NToicmVzaWQiO3M6MTI6IjI2MC81MTA4LzYwNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvNzIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.45" data-doi="10.1093/molbev/msr206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nováková</span>,  <span class="cit-name-given-names">E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Moran</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Diversification of genes for carotenoid biosynthesis in aphids following an ancient transfer from a fungus.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">323</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msr206</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr206%26rft_id%253Dinfo%253Apmid%252F21878683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msr206&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21878683&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298383900036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.46" data-doi="10.1371/journal.pbio.2004830"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nowell</span>,  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Almeida</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. G.</span> <span class="cit-name-surname">Wilson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. P.</span> <span class="cit-name-surname">Smith</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Fontaneto</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Comparative genomics of bdelloid rotifers: Insights from desiccating and nondesiccating species.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-elocation-id">e2004830</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.2004830</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D16%26rft.spage%253De2004830%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.2004830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.2004830&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.47" data-doi="10.1038/35012500"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ochman</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. G.</span> <span class="cit-name-surname">Lawrence</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Groisman</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Lateral gene transfer and the nature of bacterial innovation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">405</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">304</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/35012500</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DOchman%26rft.auinit1%253DH.%26rft.volume%253D405%26rft.issue%253D6784%26rft.spage%253D299%26rft.epage%253D304%26rft.atitle%253DLateral%2Bgene%2Btransfer%2Band%2Bthe%2Bnature%2Bof%2Bbacterial%2Binnovation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35012500%26rft_id%253Dinfo%253Apmid%252F10830951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35012500&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10830951&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087085700035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.48"><div class="cit-metadata unstructured">Oksanen, J., F. G. Blanchet, M. Freindly, R. Kindt, P. Legendre, <em>et al.</em>, 2018 vegan: Community Ecology Package. R package version 2.5–3.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.49" data-doi="10.1146/annurev-ento-112408-085305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>,  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. H.</span> <span class="cit-name-surname">Degnan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. R.</span> <span class="cit-name-surname">Burke</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Moran</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Facultative symbionts in aphids and the horizontal transfer of ecologically important traits.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Entomol.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">247</span>–<span class="cit-lpage">266</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-ento-112408-085305</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bentomology%26rft.stitle%253DAnnu%2BRev%2BEntomol%26rft.aulast%253DOliver%26rft.auinit1%253DK.%2BM.%26rft.volume%253D55%26rft.spage%253D247%26rft.epage%253D266%26rft.atitle%253DFacultative%2Bsymbionts%2Bin%2Baphids%2Band%2Bthe%2Bhorizontal%2Btransfer%2Bof%2Becologically%2Bimportant%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-ento-112408-085305%26rft_id%253Dinfo%253Apmid%252F19728837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-ento-112408-085305&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19728837&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273712100014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.50" data-doi="10.1101/gr.186072.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parks</span>,  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Imelfort</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. T.</span> <span class="cit-name-surname">Skennerton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Hugenholtz</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. W.</span> <span class="cit-name-surname">Tyson</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1043</span>–<span class="cit-lpage">1055</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.186072.114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.186072.114%26rft_id%253Dinfo%253Apmid%252F25977477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI1LzcvMTA0MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvNzIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.51" data-doi="10.1101/gr.335003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pride</span>,  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Meinersmann</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. M.</span> <span class="cit-name-surname">Wassenaar</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Blaser</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Evolutionary implications of microbial genome tetranucleotide frequency biases.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.335003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.335003%26rft_id%253Dinfo%253Apmid%252F12566393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEzLzIvMTQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi83MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.721.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.53" data-doi="10.1128/mSphere.00026-17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raymann</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. H.</span> <span class="cit-name-surname">Moeller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Goodman</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Ochman</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Unexplored archaeal diversity in the great ape gut microbiome.</span> <abbr class="cit-jnl-abbrev">MSphere</abbr> <span class="cit-vol">2</span>: e00026-17.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/mSphere.00026-17</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DmSphere%26rft.stitle%253DmSphere%26rft.aulast%253DRaymann%26rft.auinit1%253DK.%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253De00026-17%26rft.epage%253De00026-17%26rft.atitle%253DUnexplored%2BArchaeal%2BDiversity%2Bin%2Bthe%2BGreat%2BApe%2BGut%2BMicrobiome%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmSphere.00026-17%26rft_id%253Dinfo%253Apmid%252F28251182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibXNwaCI7czo1OiJyZXNpZCI7czoxMzoiMi8xL2UwMDAyNi0xNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvNzIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.54" data-doi="10.1186/s13059-015-0623-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sadd</span>,  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Barribeau</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Bloch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. C.</span> <span class="cit-name-surname">De Graaf</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Dearden</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The genomes of two key bumblebee species with primitive eusocial organization.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">76</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-015-0623-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D76%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0623-3%26rft_id%253Dinfo%253Apmid%252F25908251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0623-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25908251&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.55" data-doi="10.1186/s12915-014-0087-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salter</span>,  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Cox</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Turek</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. T.</span> <span class="cit-name-surname">Calus</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. O.</span> <span class="cit-name-surname">Cookson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Reagent and laboratory contamination can critically impact sequence-based microbiome analyses.</span> <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">87</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12915-014-0087-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBiol.%26rft.volume%253D12%26rft.spage%253D87%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12915-014-0087-z%26rft_id%253Dinfo%253Apmid%252F25387460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12915-014-0087-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25387460&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.56" data-doi="10.1186/s13059-017-1214-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>,  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Horizontal gene transfer is not a hallmark of the human genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">85</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-017-1214-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D18%26rft.spage%253D85%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-017-1214-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-017-1214-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.57" data-doi="10.1186/gb-2005-6-3-r23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>,  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C. D.</span> <span class="cit-name-surname">Hotopp</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Delcher</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Pop</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Smith</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Serendipitous discovery of Wolbachia genomes in multiple drosophila species.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">R23</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2005-6-3-r23</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DSalzberg%26rft.auinit1%253DS.%2BL.%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253DR23%26rft.epage%253DR23%26rft.atitle%253DSerendipitous%2Bdiscovery%2Bof%2BWolbachia%2Bgenomes%2Bin%2Bmultiple%2BDrosophila%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2005-6-3-r23%26rft_id%253Dinfo%253Apmid%252F15774024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2005-6-3-r23&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15774024&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.58" data-doi="10.1002/bies.201300095"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schönknecht</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Weber</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Lercher</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Horizontal gene acquisitions by eukaryotes as drivers of adaptive evolution.</span> <abbr class="cit-jnl-abbrev">BioEssays</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">20</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/bies.201300095</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%26rft.volume%253D36%26rft.spage%253D9%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.201300095%26rft_id%253Dinfo%253Apmid%252F24323918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.201300095&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24323918&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000328391600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.59" data-doi="10.1371/journal.pone.0017288"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmieder</span>,  <span class="cit-name-given-names">R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Edwards</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Fast identification and removal of sequence contamination from genomic and metagenomic datasets.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e17288</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0017288</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSchmieder%26rft.auinit1%253DR.%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253De17288%26rft.epage%253De17288%26rft.atitle%253DFast%2Bidentification%2Band%2Bremoval%2Bof%2Bsequence%2Bcontamination%2Bfrom%2Bgenomic%2Band%2Bmetagenomic%2Bdatasets.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0017288%26rft_id%253Dinfo%253Apmid%252F21408061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0017288&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21408061&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.60" data-doi="10.1186/s12915-018-0486-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simion</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Belkhir</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Francois</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Veyssier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Rink</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">A software tool ‘CroCo’detects pervasive cross-species contamination in next generation sequencing data.</span> <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">28</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12915-018-0486-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBiol.%26rft.volume%253D16%26rft.spage%253D28%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12915-018-0486-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12915-018-0486-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.61" data-doi="10.1093/bioinformatics/btu033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>,  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1312</span>–<span class="cit-lpage">1313</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24451623&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000336095100024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.62" data-doi="10.1038/35082058"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stanhope</span>,  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Lupas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Italia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. K.</span> <span class="cit-name-surname">Koretke</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Volker</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Phylogenetic analyses do not support horizontal gene transfers from bacteria to vertebrates.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">411</span>: <span class="cit-fpage">940</span>–<span class="cit-lpage">944</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/35082058</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DStanhope%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D411%26rft.issue%253D6840%26rft.spage%253D940%26rft.epage%253D944%26rft.atitle%253DPhylogenetic%2Banalyses%2Bdo%2Bnot%2Bsupport%2Bhorizontal%2Bgene%2Btransfers%2Bfrom%2Bbacteria%2Bto%2Bvertebrates.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35082058%26rft_id%253Dinfo%253Apmid%252F11418856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35082058&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11418856&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169386200042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.63" data-doi="10.1073/pnas.0707388105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Starcevic</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Akthar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. C.</span> <span class="cit-name-surname">Dunlap</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Shick</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Hranueli</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Enzymes of the shikimic acid pathway encoded in the genome of a basal metazoan, <em>Nematostella vectensis</em>, have microbial origins.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">2533</span>–<span class="cit-lpage">2537</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0707388105</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0707388105%26rft_id%253Dinfo%253Apmid%252F18268342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA1LzcvMjUzMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvNzIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.64" data-doi="10.1111/j.1462-2920.2004.00624.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teeling</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Meyerdierks</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Bauer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Amann</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. O.</span> <span class="cit-name-surname">Glöckner</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Application of tetranucleotide frequencies for the assignment of genomic fragments.</span> <abbr class="cit-jnl-abbrev">Environ. Microbiol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">938</span>–<span class="cit-lpage">947</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1462-2920.2004.00624.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2BMicrobiology%2B%2528Print%2529%26rft.stitle%253DEnvironmental%2BMicrobiology%2B%2528Print%2529%26rft.aulast%253DTeeling%26rft.auinit1%253DH.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253D938%26rft.epage%253D947%26rft.atitle%253DApplication%2Bof%2Btetranucleotide%2Bfrequencies%2Bfor%2Bthe%2Bassignment%2Bof%2Bgenomic%2Bfragments.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1462-2920.2004.00624.x%26rft_id%253Dinfo%253Apmid%252F15305919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1462-2920.2004.00624.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15305919&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223254700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.65" data-doi="10.1038/ismej.2015.100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tennessen</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Andersen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Clingenpeel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Rinke</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. S.</span> <span class="cit-name-surname">Lundberg</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">ProDeGe: a computational protocol for fully automated decontamination of genomes.</span> <abbr class="cit-jnl-abbrev">ISME J.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">272</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ismej.2015.100</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DISME%2BJ.%26rft.volume%253D10%26rft.spage%253D269%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fismej.2015.100%26rft_id%253Dinfo%253Apmid%252F26057843%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ismej.2015.100&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26057843&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.66" data-doi="10.1093/nar/gku1152"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uchiyama</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Mihara</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Nishide</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Chiba</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">MBGD update 2015: microbial genome database for flexible ortholog analysis utilizing a diverse set of genomic data.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">D270</span>–<span class="cit-lpage">D276</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gku1152</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1152%26rft_id%253Dinfo%253Apmid%252F25398900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1152&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25398900&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.67" data-doi="10.1890/10-0340.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warton</span>,  <span class="cit-name-given-names">D. I.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. K.</span> <span class="cit-name-surname">Hui</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The arcsine is asinine: the analysis of proportions in ecology.</span> <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1890/10-0340.1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.stitle%253DEcology%26rft.aulast%253DWarton%26rft.auinit1%253DD.%2BI.%26rft.volume%253D92%26rft.issue%253D1%26rft.spage%253D3%26rft.epage%253D10%26rft.atitle%253DThe%2Barcsine%2Bis%2Basinine%253A%2Bthe%2Banalysis%2Bof%2Bproportions%2Bin%2Becology.%26rft_id%253Dinfo%253Adoi%252F10.1890%252F10-0340.1%26rft_id%253Dinfo%253Apmid%252F21560670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/10-0340.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21560670&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289552200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.68" data-doi="10.1080/10425170290023392"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Willerslev</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Mourier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Hansen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Christensen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Barnes</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Contamination in the draft of the human genome masquerades as lateral gene transfer.</span> <abbr class="cit-jnl-abbrev">DNA Seq.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">75</span>–<span class="cit-lpage">76</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/10425170290023392</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2Bsequence%2B%253A%2B%2Bthe%2Bjournal%2Bof%2BDNA%2Bsequencing%2Band%2Bmapping%26rft.stitle%253DDNA%2BSeq%26rft.aulast%253DWillerslev%26rft.auinit1%253DE.%26rft.volume%253D13%26rft.issue%253D2%26rft.spage%253D75%26rft.epage%253D76%26rft.atitle%253DContamination%2Bin%2Bthe%2Bdraft%2Bof%2Bthe%2Bhuman%2Bgenome%2Bmasquerades%2Bas%2Blateral%2Bgene%2Btransfer.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10425170290023392%26rft_id%253Dinfo%253Apmid%252F12180348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10425170290023392&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12180348&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.69" data-doi="10.1016/j.cub.2018.05.070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>,  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. W.</span> <span class="cit-name-surname">Nowell</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. C.</span> <span class="cit-name-surname">Barraclough</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Cross-contamination explains “inter and intraspecific horizontal genetic transfers” between asexual bdelloid rotifers.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2436</span>–<span class="cit-lpage">2444.e14</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cub.2018.05.070</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2018.05.070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2018.05.070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.70" data-doi="10.1093/bioinformatics/bti310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">T. D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. K.</span> <span class="cit-name-surname">Watanabe</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">GMAP: a genomic mapping and alignment program for mRNA and EST sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1859</span>–<span class="cit-lpage">1875</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bti310</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti310%26rft_id%253Dinfo%253Apmid%252F15728110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15728110&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228783000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.71" data-doi="10.1016/j.ibmb.2012.08.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wybouw</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Balabanidou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Ballhorn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Dermauw</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. V.</span> <span class="cit-name-surname">Grbić</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A horizontally transferred cyanase gene in the spider mite <em>Tetranychus urticae</em> is involved in cyanate metabolism and is differentially expressed upon host plant change.</span> <abbr class="cit-jnl-abbrev">Insect Biochem. Mol. Biol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">881</span>–<span class="cit-lpage">889</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ibmb.2012.08.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInsect%2Bbiochemistry%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DInsect%2BBiochem%2BMol%2BBiol%26rft.aulast%253DWybouw%26rft.auinit1%253DN.%26rft.volume%253D42%26rft.issue%253D12%26rft.spage%253D881%26rft.epage%253D889%26rft.atitle%253DA%2Bhorizontally%2Btransferred%2Bcyanase%2Bgene%2Bin%2Bthe%2Bspider%2Bmite%2BTetranychus%2Burticae%2Bis%2Binvolved%2Bin%2Bcyanate%2Bmetabolism%2Band%2Bis%2Bdifferentially%2Bexpressed%2Bupon%2Bhost%2Bplant%2Bchange.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ibmb.2012.08.002%26rft_id%253Dinfo%253Apmid%252F22960016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ibmb.2012.08.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22960016&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.721.72" data-doi="10.1093/nar/gkx1098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zerbino</span>,  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Achuthan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Akanni</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Amode</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Barrell</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Ensembl 2018.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">D754</span>–<span class="cit-lpage">D761</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkx1098</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D46%26rft.spage%253DD754%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1098%26rft_id%253Dinfo%253Apmid%252F29155950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx1098&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29155950&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/2/721.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/2/709" title="Horizontal Transfer and Gene Loss Shaped the Evolution of Alpha-Amylases in Bilaterians" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/2/731" title="Genetic Analysis of the Transition from Wild to Domesticated Cotton (Gossypium hirsutum L.)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/2"><span>Volume 10 Issue 2, February 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/2.cover-source.jpg?itok=LSawHOSr" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/721.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1gpcEcQmRwy1-6fcn4trPijZWxBEyB82kn7auPdE8cU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/2/721" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/2/721" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/2/721" class="active" data-icon-position="" data-hide-link-title="0">Prevalence and Implications of Contamination in Public Genomic Resources: A Case Study of 43 Reference Arthropod Assemblies</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495529" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-DIqBu_6ZkMXWJ_X1cx5-3mwQRQE4dn_2ZDcKjLRBzyw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/2/721.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/2/721" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-M6gVVtHsBLXJzS5poNibl3ebLGkB-3skL-_K-tkhVpU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/2/721" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/2/721.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495529" id="node495529--2" data-pisa="ggg;10/2/721" data-pisa-master="ggg;g3.119.400758" data-apath="/ggg/10/2/721.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Prevalence and Implications of Contamination in Public Genomic Resources: A Case Study of 43 Reference Arthropod Assemblies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7781-8781" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Clementine M. Francois</span>, <span class="highwire-citation-author" data-delta="1">Faustine Durand</span>, <span class="highwire-citation-author" data-delta="2">Emeric Figuet</span> and <span class="highwire-citation-author" data-delta="3">Nicolas Galtier</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">721-730;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400758 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495529--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Clementine M. Francois </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>UMR 5554, Institut des Sciences de l’Evolution; CNRS, University of Montpellier, IRD, EPHE, Montpellier, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clementine%2BM.%2BFrancois%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Francois%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AClementine%2BM.%2BFrancois%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7781-8781" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Clementine M. Francois</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:clementine.francois@univ-lyon1.fr" class="" data-icon-position="" data-hide-link-title="0">clementine.francois@univ-lyon1.fr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Faustine Durand </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>UMR 5554, Institut des Sciences de l’Evolution; CNRS, University of Montpellier, IRD, EPHE, Montpellier, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Faustine%2BDurand%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Durand%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFaustine%2BDurand%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Emeric Figuet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>UMR 5554, Institut des Sciences de l’Evolution; CNRS, University of Montpellier, IRD, EPHE, Montpellier, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emeric%2BFiguet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Figuet%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmeric%2BFiguet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Nicolas Galtier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>UMR 5554, Institut des Sciences de l’Evolution; CNRS, University of Montpellier, IRD, EPHE, Montpellier, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BGaltier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Galtier%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicolas%2BGaltier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/2/721" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/2/721&amp;title=Prevalence%20and%20Implications%20of%20Contamination%20in%20Public%20Genomic%20Resources%3A%20A%20Case%20Study%20of%2043%20Reference%20Arthropod%20Assemblies" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/2/721&amp;title=Prevalence%20and%20Implications%20of%20Contamination%20in%20Public%20Genomic%20Resources%3A%20A%20Case%20Study%20of%2043%20Reference%20Arthropod%20Assemblies" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/2/721&amp;title=Prevalence%20and%20Implications%20of%20Contamination%20in%20Public%20Genomic%20Resources%3A%20A%20Case%20Study%20of%2043%20Reference%20Arthropod%20Assemblies" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/2/721&amp;text=Prevalence%20and%20Implications%20of%20Contamination%20in%20Public%20Genomic%20Resources%3A%20A%20Case%20Study%20of%2043%20Reference%20Arthropod%20Assemblies" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/2/721&amp;title=Prevalence%20and%20Implications%20of%20Contamination%20in%20Public%20Genomic%20Resources%3A%20A%20Case%20Study%20of%2043%20Reference%20Arthropod%20Assemblies" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/2/721&amp;t=Prevalence%20and%20Implications%20of%20Contamination%20in%20Public%20Genomic%20Resources%3A%20A%20Case%20Study%20of%2043%20Reference%20Arthropod%20Assemblies" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/2/721&amp;title=Prevalence%20and%20Implications%20of%20Contamination%20in%20Public%20Genomic%20Resources%3A%20A%20Case%20Study%20of%2043%20Reference%20Arthropod%20Assemblies" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/2/721&amp;title=Prevalence%20and%20Implications%20of%20Contamination%20in%20Public%20Genomic%20Resources%3A%20A%20Case%20Study%20of%2043%20Reference%20Arthropod%20Assemblies" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495529/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495529" id="node495529--3" data-pisa="ggg;10/2/721" data-pisa-master="ggg;g3.119.400758" data-apath="/ggg/10/2/721.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Prevalence and Implications of Contamination in Public Genomic Resources: A Case Study of 43 Reference Arthropod Assemblies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7781-8781" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Clementine M. Francois</span>, <span class="highwire-citation-author" data-delta="1">Faustine Durand</span>, <span class="highwire-citation-author" data-delta="2">Emeric Figuet</span> and <span class="highwire-citation-author" data-delta="3">Nicolas Galtier</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">721-730;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400758 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495529--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Clementine M. Francois </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>UMR 5554, Institut des Sciences de l’Evolution; CNRS, University of Montpellier, IRD, EPHE, Montpellier, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clementine%2BM.%2BFrancois%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Francois%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AClementine%2BM.%2BFrancois%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7781-8781" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Clementine M. Francois</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:clementine.francois@univ-lyon1.fr" class="" data-icon-position="" data-hide-link-title="0">clementine.francois@univ-lyon1.fr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Faustine Durand </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>UMR 5554, Institut des Sciences de l’Evolution; CNRS, University of Montpellier, IRD, EPHE, Montpellier, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Faustine%2BDurand%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Durand%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFaustine%2BDurand%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Emeric Figuet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>UMR 5554, Institut des Sciences de l’Evolution; CNRS, University of Montpellier, IRD, EPHE, Montpellier, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emeric%2BFiguet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Figuet%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmeric%2BFiguet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Nicolas Galtier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>UMR 5554, Institut des Sciences de l’Evolution; CNRS, University of Montpellier, IRD, EPHE, Montpellier, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BGaltier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Galtier%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicolas%2BGaltier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=721&amp;rft.epage=730&amp;rft.atitle=Prevalence%20and%20Implications%20of%20Contamination%20in%20Public%20Genomic%20Resources%3A%20A%20Case%20Study%20of%2043%20Reference%20Arthropod%20Assemblies&amp;rft.volume=10&amp;rft.issue=2&amp;rft.date=2020-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Francois%2C+Clementine+M.&amp;rft.au=Durand%2C+Faustine&amp;rft.au=Figuet%2C+Emeric&amp;rft.au=Galtier%2C+Nicolas"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495529/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495529/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495529/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495529/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495529/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495529/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495529/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495529/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495529/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495529/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495529/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495529/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/2/721" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials &amp; Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results &amp; Discussion</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/2/721.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/2/721.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/2/721.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
