<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/2/783" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/2/783.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/2/783.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/2/783.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/10/2/783/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/2/783.atom" />
<meta name="HW.pisa" content="ggg;10/2/783" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The Impact of Non-additive Effects on the Genetic Correlation Between Populations" />
<meta name="DC.Identifier" content="10.1534/g3.119.400663" />
<meta name="DC.Date" content="2020-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Duenk et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Average effects of alleles can show considerable differences between populations. The magnitude of these differences can be measured by the additive genetic correlation between populations (![Formula][1]&lt;/img&gt;). This ![Formula][2]&lt;/img&gt; can be lower than one due to the presence of non-additive genetic effects together with differences in allele frequencies between populations. However, the relationship between the nature of non-additive effects, differences in allele frequencies, and the value of ![Formula][3]&lt;/img&gt; remains unclear, and was therefore the focus of this study. We simulated genotype data of two populations that have diverged under drift only, or under drift and selection, and we simulated traits where the genetic model and magnitude of non-additive effects were varied. Results showed that larger differences in allele frequencies and larger non-additive effects resulted in lower values of ![Formula][4]&lt;/img&gt;. In addition, we found that with epistasis, ![Formula][5]&lt;/img&gt; decreases with an increase of the number of interactions per locus. For both dominance and epistasis, we found that, when non-additive effects became extremely large, ![Formula][6]&lt;/img&gt; had a lower bound that was determined by the type of inter-allelic interaction, and the difference in allele frequencies between populations. Given that dominance variance is usually small, our results show that it is unlikely that true ![Formula][7]&lt;/img&gt; values lower than 0.80 are due to dominance effects alone. With realistic levels of epistasis, ![Formula][8]&lt;/img&gt; dropped as low as 0.45. These results may contribute to the understanding of differences in genetic expression of complex traits between populations, and may help in explaining the inefficiency of genomic trait prediction across populations.

 [1]: /embed/mml-math-1.gif
 [2]: /embed/mml-math-2.gif
 [3]: /embed/mml-math-3.gif
 [4]: /embed/mml-math-4.gif
 [5]: /embed/mml-math-5.gif
 [6]: /embed/mml-math-6.gif
 [7]: /embed/mml-math-7.gif
 [8]: /embed/mml-math-8.gif" />
<meta name="DC.Contributor" content="Pascal Duenk" />
<meta name="DC.Contributor" content="Piter Bijma" />
<meta name="DC.Contributor" content="Mario P. L. Calus" />
<meta name="DC.Contributor" content="Yvonne C. J. Wientjes" />
<meta name="DC.Contributor" content="Julius H. J. van der Werf" />
<meta name="article:published_time" content="2020-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="The Impact of Non-additive Effects on the Genetic Correlation Between Populations" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Average effects of alleles can show considerable differences between populations. The magnitude of these differences can be measured by the additive genetic correlation between populations (&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me2&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mi&gt;r&lt;/mi&gt;&lt;mi&gt;g&lt;/mi&gt;&lt;/msub&gt;&lt;/mrow&gt;&lt;/math&gt;). This &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me3&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mi&gt;r&lt;/mi&gt;&lt;mi&gt;g&lt;/mi&gt;&lt;/msub&gt;&lt;/mrow&gt;&lt;/math&gt; can be lower than one due to the presence of non-additive genetic effects together with differences in allele frequencies between populations. However, the relationship between the nature of non-additive effects, differences in allele frequencies, and the value of &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me4&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mi&gt;r&lt;/mi&gt;&lt;mi&gt;g&lt;/mi&gt;&lt;/msub&gt;&lt;/mrow&gt;&lt;/math&gt; remains unclear, and was therefore the focus of this study. We simulated genotype data of two populations that have diverged under drift only, or under drift and selection, and we simulated traits where the genetic model and magnitude of non-additive effects were varied. Results showed that larger differences in allele frequencies and larger non-additive effects resulted in lower values of &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me5&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mi&gt;r&lt;/mi&gt;&lt;mi&gt;g&lt;/mi&gt;&lt;/msub&gt;&lt;/mrow&gt;&lt;/math&gt;. In addition, we found that with epistasis, &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me6&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mi&gt;r&lt;/mi&gt;&lt;mi&gt;g&lt;/mi&gt;&lt;/msub&gt;&lt;/mrow&gt;&lt;/math&gt; decreases with an increase of the number of interactions per locus. For both dominance and epistasis, we found that, when non-additive effects became extremely large, &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me7&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mi&gt;r&lt;/mi&gt;&lt;mi&gt;g&lt;/mi&gt;&lt;/msub&gt;&lt;/mrow&gt;&lt;/math&gt; had a lower bound that was determined by the type of inter-allelic interaction, and the difference in allele frequencies between populations. Given that dominance variance is usually small, our results show that it is unlikely that true &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me8&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mi&gt;r&lt;/mi&gt;&lt;mi&gt;g&lt;/mi&gt;&lt;/msub&gt;&lt;/mrow&gt;&lt;/math&gt; values lower than 0.80 are due to dominance effects alone. With realistic levels of epistasis, &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me9&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mi&gt;r&lt;/mi&gt;&lt;mi&gt;g&lt;/mi&gt;&lt;/msub&gt;&lt;/mrow&gt;&lt;/math&gt; dropped as low as 0.45. These results may contribute to the understanding of differences in genetic expression of complex traits between populations, and may help in explaining the inefficiency of genomic trait prediction across populations.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/02/01" />
<meta name="citation_mjid" content="ggg;10/2/783" />
<meta name="citation_id" content="10/2/783" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/2/783" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/2/783.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/2/783.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/2/783.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400663" />
<meta name="citation_pmid" content="31857332" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="783" />
<meta name="citation_lastpage" content="795" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Pascal Duenk" />
<meta name="citation_author_institution" content="Animal Breeding and Genomics, Wageningen University and Research, P.O. Box 338, 6700 AH Wageningen, the Netherlands and" />
<meta name="citation_author_email" content="pascal.duenk@wur.nl" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8403-968X" />
<meta name="citation_author" content="Piter Bijma" />
<meta name="citation_author_institution" content="Animal Breeding and Genomics, Wageningen University and Research, P.O. Box 338, 6700 AH Wageningen, the Netherlands and" />
<meta name="citation_author" content="Mario P. L. Calus" />
<meta name="citation_author_institution" content="Animal Breeding and Genomics, Wageningen University and Research, P.O. Box 338, 6700 AH Wageningen, the Netherlands and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3213-704X" />
<meta name="citation_author" content="Yvonne C. J. Wientjes" />
<meta name="citation_author_institution" content="Animal Breeding and Genomics, Wageningen University and Research, P.O. Box 338, 6700 AH Wageningen, the Netherlands and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0681-2902" />
<meta name="citation_author" content="Julius H. J. van der Werf" />
<meta name="citation_author_institution" content="School of Environmental and Rural Science, University of New England, Armidale 2351 NSW, Australia" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. M. Álvarez-Castro;citation_author=Ö. Carlborg;citation_title=A Unified Model for Functional and Statistical Epistasis and Its Application in Quantitative Trait Loci Analysis.;citation_volume=176;citation_year=2007;citation_pmid=17409082;citation_doi=10.1534/genetics.106.067348" />
<meta name="citation_reference" content="citation_title=Mendel’s principles of heredity.;citation_author=W. Bateson;citation_author=G. Mendel;citation_year=1909;citation_doi=10.5962/bhl.title.44575" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=J. Bennewitz;citation_author=T. H. E. Meuwissen;citation_title=The distribution of QTL additive and dominance effects in porcine F2 crosses.;citation_pages=171-179;citation_volume=127;citation_year=2010;citation_issue=3;citation_pmid=20536634;citation_doi=10.1111/j.1439-0388.2009.00847.x" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=B. C. Brown;citation_author=C. J. Ye;citation_author=A. L. Price;citation_author=N. Zaitlen;citation_title=Transethnic Genetic-Correlation Estimates from Summary Statistics.;citation_pages=76-88;citation_volume=99;citation_year=2016;citation_pmid=27321947;citation_doi=10.1016/j.ajhg.2016.05.001" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=O. Carlborg;citation_author=S. Kerje;citation_author=K. Schutz;citation_author=L. Jacobsson;citation_author=P. Jensen;citation_title=A Global Search Reveals Epistatic Interaction Between QTL for Early Growth in the Chicken;citation_pages=413-421;citation_volume=13;citation_year=2003;citation_issue=3;citation_pmid=12618372;citation_doi=10.1101/gr.528003" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=A. J. Carter;citation_author=J. Hermisson;citation_author=T. F. Hansen;citation_title=The role of epistatic gene interactions in the response to selection and the evolution of evolvability.;citation_pages=179-196;citation_volume=68;citation_year=2005;citation_issue=3;citation_pmid=16122771;citation_doi=10.1016/j.tpb.2005.05.002" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. M. Cheverud;citation_author=E. J. Routman;citation_title=Epistasis and its contribution to genetic variance components.;citation_pages=1455-1461;citation_volume=139;citation_year=1995;citation_issue=3;citation_pmid=7768453" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=J. M. Cheverud;citation_author=E. J. Routman;citation_title=Epistasis as a source of increased additive genetic variance at population bottlenecks.;citation_pages=1042-1051;citation_volume=50;citation_year=1996;citation_doi=10.2307/2410645" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. C. Cockerham;citation_title=AN EXTENSION OF THE CONCEPT OF PARTITIONING HEREDITARY VARIANCE FOR ANALYSIS OF COVARIANCES AMONG RELATIVES WHEN EPISTASIS IS PRESENT;citation_pages=859-882;citation_volume=39;citation_year=1954;citation_issue=6;citation_pmid=17247525" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=T. R. de Candia;citation_author=S. H. Lee;citation_author=J. Yang;citation_author=B. L. Browning;citation_author=P. V. Gejman;citation_title=Additive genetic variation in schizophrenia risk is shared by populations of African and European descent.;citation_pages=463-470;citation_volume=93;citation_year=2013;citation_pmid=23954163;citation_doi=10.1016/j.ajhg.2013.07.007" />
<meta name="citation_reference" content="citation_year=2019" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. P. de Roos;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Reliability of genomic predictions across multiple populations.;citation_volume=183;citation_year=2009;citation_pmid=19822733;citation_doi=10.1534/genetics.109.104935" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=J. Ertl;citation_author=A. Legarra;citation_author=Z. G. Vitezica;citation_author=L. Varona;citation_author=C. Edel;citation_title=Genomic analysis of dominance effects on milk production and conformation traits in Fleckvieh cattle.;citation_pages=40;citation_volume=46;citation_year=2014;citation_doi=10.1186/1297-9686-46-40" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=H. Esfandyari;citation_author=M. Henryon;citation_author=P. Berg;citation_author=J. R. Thomasen;citation_author=P. Bijma;citation_title=Response to Selection in Finite Locus Models with Nonadditive Effects.;citation_pages=318-327;citation_volume=108;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=H. Esfandyari;citation_author=A. C. Sorensen;citation_author=P. Bijma;citation_title=A crossbred reference population can improve the response to genomic selection for crossbred performance.;citation_pages=76;citation_volume=47;citation_year=2015;citation_doi=10.1186/s12711-015-0155-z" />
<meta name="citation_reference" content="citation_title=Introduction to quantitative genetics;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=D. S. Falconer;citation_title=The Problem of Environment and Selection.;citation_pages=293-298;citation_volume=86;citation_year=1952;citation_doi=10.1086/281736" />
<meta name="citation_reference" content="citation_journal_title=Trans. R. Soc. Edinb.;citation_author=R. A. Fisher;citation_title=The Correlation between Relatives on the Supposition of Mendelian Inheritance.;citation_pages=399-433;citation_volume=52;citation_year=1918;citation_doi=10.1017/S0080456800012163" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=S. K. G. Forsberg;citation_author=J. S. Bloom;citation_author=M. J. Sadhu;citation_author=L. Kruglyak;citation_author=Ö. Carlborg;citation_title=Accounting for genetic interactions improves modeling of individual quantitative trait phenotypes in yeast.;citation_pages=497-503;citation_volume=49;citation_year=2017;citation_pmid=28250458;citation_doi=10.1038/ng.3800" />
<meta name="citation_reference" content="citation_journal_title=Livest. Prod. Sci.;citation_author=M. E. Goddard;citation_title=The validity of genetic models underlying quantitative traits.;citation_pages=117-127;citation_volume=72;citation_year=2001;citation_doi=10.1016/S0301-6226(01)00272-X" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=J. Hallander;citation_author=P. Waldmann;citation_title=The effect of non-additive genetic interactions on selection in multi-locus genetic models.;citation_pages=349-359;citation_volume=98;citation_year=2007;citation_issue=6;citation_pmid=17327874;citation_doi=10.1038/sj.hdy.6800946" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=T. F. Hansen;citation_title=Why epistasis is important for selection and adaptation.;citation_pages=3501-3511;citation_volume=67;citation_year=2013;citation_pmid=24299403;citation_doi=10.1111/evo.12214" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. C. Chamberlain;citation_author=K. Verbyla;citation_author=M. E. Goddard;citation_title=Accuracy of genomic breeding values in multi-breed dairy cattle populations.;citation_pages=51-51;citation_volume=41;citation_year=2009;citation_pmid=19930712;citation_doi=10.1186/1297-9686-41-51" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=W. G. Hill;citation_title=Is Continued Genetic Improvement of Livestock Sustainable?;citation_pages=877-881;citation_volume=202;citation_year=2016;citation_issue=3;citation_pmid=26953266;citation_doi=10.1534/genetics.115.186650" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=W. G. Hill;citation_title=“Conversion” of epistatic into additive genetic variance in finite populations and possible impact on long-term selection response.;citation_pages=196-201;citation_volume=134;citation_year=2017;citation_doi=10.1111/jbg.12270" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=W. G. Hill;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=Data and theory point to mainly additive genetic variance for complex traits.;citation_pages=e1000008-e1000008;citation_volume=4;citation_year=2008;citation_issue=2;citation_pmid=18454194;citation_doi=10.1371/journal.pgen.1000008" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=O. O. M. Iheshiulor;citation_author=J. A. Woolliams;citation_author=X. Yu;citation_author=R. Wellmann;citation_author=T. H. E. Meuwissen;citation_title=Within- and across-breed genomic prediction using whole-genome sequence and single nucleotide polymorphism panels.;citation_pages=15;citation_volume=48;citation_year=2016;citation_doi=10.1186/s12711-016-0193-1" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=T. Johnson;citation_author=N. Barton;citation_title=Theoretical models of selection and mutation on quantitative traits;citation_pages=1411-1425;citation_volume=360;citation_year=2005;citation_issue=1459;citation_pmid=16048784;citation_doi=10.1098/rstb.2005.1667" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=R. Joshi;citation_author=J. A. Woolliams;citation_author=T. Meuwissen;citation_author=H. M. Gjoen;citation_title=Maternal, dominance and additive genetic effects in Nile tilapia; influence on growth, fillet yield and body size traits.;citation_volume=120;citation_year=2018;citation_doi=10.1038/s41437-017-0046-x" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=S. Karoui;citation_author=M. J. Carabaño;citation_author=C. Díaz;citation_author=A. Legarra;citation_title=Joint genomic evaluation of French dairy cattle breeds using multiple-trait models.;citation_pages=39;citation_volume=44;citation_year=2012;citation_pmid=23216664;citation_doi=10.1186/1297-9686-44-39" />
<meta name="citation_reference" content="citation_pages=103-113;citation_year=1954" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Kimura;citation_author=J. F. Crow;citation_title=THE NUMBER OF ALLELES THAT CAN BE MAINTAINED IN A FINITE POPULATION;citation_pages=725-738;citation_volume=49;citation_year=1964;citation_issue=4;citation_pmid=14156929" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Le Rouzic;citation_author=J. M. Alvarez-Castro;citation_author=O. Carlborg;citation_title=Dissection of the genetic architecture of body weight in chicken reveals the impact of epistasis on domestication traits.;citation_volume=179;citation_year=2008;citation_pmid=18622035;citation_doi=10.1534/genetics.108.089300" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=M. S. Lopes;citation_author=J. W. M. Bastiaansen;citation_author=L. Janss;citation_author=E. F. Knol;citation_author=H. Bovenhuis;citation_title=Genomic prediction of growth in pigs based on a model including additive and dominance effects.;citation_pages=180-186;citation_volume=133;citation_year=2016;citation_doi=10.1111/jbg.12195." />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=M. S. Lund;citation_author=A. P. W. de Roos;citation_author=A. G. de Vries;citation_author=T. Druet;citation_author=V. Ducrocq;citation_title=A common reference population from four European Holstein populations increases reliability of genomic predictions.;citation_pages=43-43;citation_volume=43;citation_year=2011;citation_pmid=22152008;citation_doi=10.1186/1297-9686-43-43" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=T. F. C. Mackay;citation_title=Epistasis and quantitative traits: using model organisms to study gene–gene interactions.;citation_pages=22-33;citation_volume=15;citation_year=2014;citation_pmid=24296533;citation_doi=10.1038/nrg3627" />
<meta name="citation_reference" content="citation_author=T. F. C. Mackay;citation_title=Epistasis for Quantitative Traits in Drosophila.;citation_pages=47-70;citation_volume=1253;citation_year=2015;citation_doi=10.1007/978-1-4939-2155-3_4" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Mäki-Tanila;citation_author=W. G. Hill;citation_title=Influence of gene interaction on complex trait variation with multilocus models.;citation_volume=198;citation_year=2014;citation_pmid=24990992;citation_doi=10.1534/genetics.114.165282" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=N. Moghaddar;citation_author=J. H. J. van der Werf;citation_title=Genomic estimation of additive and dominance effects and impact of accounting for dominance on accuracy of genomic evaluation in sheep populations.;citation_pages=1-10;citation_volume=00;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=P. J. Monnahan;citation_author=J. K. Kelly;citation_title=Epistasis Is a Major Determinant of the Additive Genetic Variance in Mimulus guttatus.;citation_pages=e1005201;citation_volume=11;citation_year=2015;citation_pmid=25946702;citation_doi=10.1371/journal.pgen.1005201" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. Paixão;citation_author=N. H. Barton;citation_title=The effect of gene interactions on the long-term response to selection.;citation_volume=113;citation_year=2016;citation_pmid=27044080;citation_doi=10.1073/pnas.1518830113" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. Pettersson;citation_author=F. Besnier;citation_author=P. B. Siegel;citation_author=O. Carlborg;citation_title=Replication and explorations of high-order epistasis using a large advanced intercross line pedigree.;citation_pages=e1002180-e1002180;citation_volume=7;citation_year=2011;citation_issue=7;citation_pmid=21814519;citation_doi=10.1371/journal.pgen.1002180" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=L. Porto-Neto;citation_author=W. Barendse;citation_author=J. Henshall;citation_author=S. McWilliam;citation_author=S. Lehnert;citation_title=Genomic correlation: harnessing the benefit of combining two unrelated populations for genomic selection.;citation_pages=84;citation_volume=47;citation_year=2015;citation_doi=10.1186/s12711-015-0162-0" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=B. Raymond;citation_author=A. C. Bouwman;citation_author=C. Schrooten;citation_author=J. Houwing-Duistermaat;citation_author=R. F. Veerkamp;citation_title=Utility of whole-genome sequence data for across-breed genomic prediction.;citation_pages=27;citation_volume=50;citation_year=2018;citation_doi=10.1186/s12711-018-0396-8" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=B. Raymond;citation_author=A. C. Bouwman;citation_author=Y. C. J. Wientjes;citation_author=C. Schrooten;citation_author=J. Houwing-Duistermaat;citation_title=Genomic prediction for numerically small breeds, using models with pre-selected and differentially weighted markers.;citation_pages=49;citation_volume=50;citation_year=2018;citation_doi=10.1186/s12711-018-0419-5" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Sargolzaei;citation_author=F. S. Schenkel;citation_title=QMSim: a large-scale genome simulator for livestock.;citation_volume=25;citation_year=2009;citation_pmid=19176551;citation_doi=10.1093/bioinformatics/btp045" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=A. K. Sonesson;citation_author=T. H. Meuwissen;citation_title=Testing strategies for genomic selection in aquaculture breeding programs.;citation_pages=37-37;citation_volume=41;citation_year=2009;citation_pmid=19566932;citation_doi=10.1186/1297-9686-41-37" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=X. Sun;citation_author=R. H. Mumm;citation_title=Method to represent the distribution of QTL additive and dominance effects associated with quantitative traits in computer simulation.;citation_pages=73;citation_volume=17;citation_year=2016;citation_doi=10.1186/s12859-016-0906-z" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. G. Vitezica;citation_author=A. Legarra;citation_author=M. A. Toro;citation_author=L. Varona;citation_title=Orthogonal Estimates of Variances for Additive, Dominance and Epistatic Effects in Populations.;citation_volume=206;citation_year=2017;citation_pmid=28522540;citation_doi=10.1534/genetics.116.199406" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=Z. G. Vitezica;citation_author=A. Reverter;citation_author=W. Herring;citation_author=A. Legarra;citation_title=Dominance and epistatic genetic variances for litter size in pigs using genomic models.;citation_pages=71;citation_volume=50;citation_year=2018;citation_doi=10.1186/s12711-018-0437-3" />
<meta name="citation_reference" content="citation_title=Maintenance of Quantitative Genetic Variation in Evolution and Selection of Quantitative Traits;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=M. Wei;citation_author=H. A. M. van der Steen;citation_author=J. H. J. van der Werf;citation_author=E. W. Brascamp;citation_title=Relationship between purebred and crossbred parameters.;citation_pages=253-261;citation_volume=108;citation_year=1991;citation_doi=10.1111/j.1439-0388.1991.tb00183.x" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=W.-H. Wei;citation_author=G. Hemani;citation_author=C. S. Haley;citation_title=Detecting epistasis in human complex traits.;citation_pages=722-733;citation_volume=15;citation_year=2014;citation_pmid=25200660;citation_doi=10.1038/nrg3747" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=R. Wellmann;citation_author=J. Bennewitz;citation_title=The contribution of dominance to the understanding of quantitative genetic variation.;citation_pages=139-154;citation_volume=93;citation_year=2011;citation_doi=10.1017/S0016672310000649" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=Y. C. J. Wientjes;citation_author=M. P. L. Calus;citation_title=BOARD INVITED REVIEW: The purebred-crossbred correlation in pigs: A review of theory, estimates, and implications.;citation_pages=3467-3478;citation_volume=95;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=Y. C. J. Wientjes;citation_author=R. F. Veerkamp;citation_author=P. Bijma;citation_author=H. Bovenhuis;citation_author=C. Schrooten;citation_title=Empirical and deterministic accuracies of across-population genomic prediction.;citation_pages=5;citation_volume=47;citation_year=2015;citation_pmid=25885467;citation_doi=10.1186/s12711-014-0086-0" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=A. Wolc;citation_author=J. Arango;citation_author=P. Settar;citation_author=J. E. Fulton;citation_author=N. P. O’Sullivan;citation_title=Persistence of accuracy of genomic estimated breeding values over generations in layer chickens.;citation_pages=23-23;citation_volume=43;citation_year=2011;citation_pmid=21693035;citation_doi=10.1186/1297-9686-43-23" />
<meta name="citation_reference" content="citation_year=2018" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="The Impact of Non-additive Effects on the Genetic Correlation Between Populations" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
<meta name="twitter:description" content="Average effects of alleles can show considerable differences between populations. The magnitude of these differences can be measured by the additive genetic correlation between populations (![Formula][1]&lt;/img&gt;). This ![Formula][2]&lt;/img&gt; can be lower than one due to the presence of non-additive genetic effects together with differences in allele frequencies between populations. However, the relationship between the nature of non-additive effects, differences in allele frequencies, and the value of ![Formula][3]&lt;/img&gt; remains unclear, and was therefore the focus of this study. We simulated genotype data of two populations that have diverged under drift only, or under drift and selection, and we simulated traits where the genetic model and magnitude of non-additive effects were varied. Results showed that larger differences in allele frequencies and larger non-additive effects resulted in lower values of ![Formula][4]&lt;/img&gt;. In addition, we found that with epistasis, ![Formula][5]&lt;/img&gt; decreases with an increase of the number of interactions per locus. For both dominance and epistasis, we found that, when non-additive effects became extremely large, ![Formula][6]&lt;/img&gt; had a lower bound that was determined by the type of inter-allelic interaction, and the difference in allele frequencies between populations. Given that dominance variance is usually small, our results show that it is unlikely that true ![Formula][7]&lt;/img&gt; values lower than 0.80 are due to dominance effects alone. With realistic levels of epistasis, ![Formula][8]&lt;/img&gt; dropped as low as 0.45. These results may contribute to the understanding of differences in genetic expression of complex traits between populations, and may help in explaining the inefficiency of genomic trait prediction across populations.

 [1]: /embed/mml-math-1.gif
 [2]: /embed/mml-math-2.gif
 [3]: /embed/mml-math-3.gif
 [4]: /embed/mml-math-4.gif
 [5]: /embed/mml-math-5.gif
 [6]: /embed/mml-math-6.gif
 [7]: /embed/mml-math-7.gif
 [8]: /embed/mml-math-8.gif" />
<meta name="og-title" property="og:title" content="The Impact of Non-additive Effects on the Genetic Correlation Between Populations" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/2/783" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Average effects of alleles can show considerable differences between populations. The magnitude of these differences can be measured by the additive genetic correlation between populations (![Formula][1]&lt;/img&gt;). This ![Formula][2]&lt;/img&gt; can be lower than one due to the presence of non-additive genetic effects together with differences in allele frequencies between populations. However, the relationship between the nature of non-additive effects, differences in allele frequencies, and the value of ![Formula][3]&lt;/img&gt; remains unclear, and was therefore the focus of this study. We simulated genotype data of two populations that have diverged under drift only, or under drift and selection, and we simulated traits where the genetic model and magnitude of non-additive effects were varied. Results showed that larger differences in allele frequencies and larger non-additive effects resulted in lower values of ![Formula][4]&lt;/img&gt;. In addition, we found that with epistasis, ![Formula][5]&lt;/img&gt; decreases with an increase of the number of interactions per locus. For both dominance and epistasis, we found that, when non-additive effects became extremely large, ![Formula][6]&lt;/img&gt; had a lower bound that was determined by the type of inter-allelic interaction, and the difference in allele frequencies between populations. Given that dominance variance is usually small, our results show that it is unlikely that true ![Formula][7]&lt;/img&gt; values lower than 0.80 are due to dominance effects alone. With realistic levels of epistasis, ![Formula][8]&lt;/img&gt; dropped as low as 0.45. These results may contribute to the understanding of differences in genetic expression of complex traits between populations, and may help in explaining the inefficiency of genomic trait prediction across populations.

 [1]: /embed/mml-math-1.gif
 [2]: /embed/mml-math-2.gif
 [3]: /embed/mml-math-3.gif
 [4]: /embed/mml-math-4.gif
 [5]: /embed/mml-math-5.gif
 [6]: /embed/mml-math-6.gif
 [7]: /embed/mml-math-7.gif
 [8]: /embed/mml-math-8.gif" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
    <title>The Impact of Non-additive Effects on the Genetic Correlation Between Populations | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"zJfZwZ1cWe4d4Di50gw0n9Icm1ByfXgmGyZNRKnOu-M","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495560","apath":"\/ggg\/10\/2\/783.atom","pisa":"ggg;10\/2\/783","ac":{"\/ggg\/10\/2\/783.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/2\/783.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/2\/783"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/2\/783"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/2\/783.figures-only":true,"\/content\/10\/2\/783":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022SJoUyPgRD2ezGC5QFujnUiyTbc116uX30BqEJiVyjCTi5+PbQG94R1VnkfDJhqmO22niUHgO\\\\\\\/B+CNXQU1WAckCGstcafW2GRQWxfil4vq6yiT6UHZRmZasPBukN34e3Xy14tJv5ipAMA7bZCwpsQLRM4+VUAH14sCrrhtQyDQpGZ+boFhEAbNuhgbnmtxJD\\\\\\\/T0jjhCW4wWXlqvt6qw+dZ8UnB+Hw6xG8EngBicltKH6XSGKmPb7jo1354TI\\\\\\\/ROiVtAevefprAjp\\\\\\\/Dp2TmYk\\\\\\\/PYKWoRqdUTcIlTCas2sNEcbL0P3QgKeeSKJcGynLsXBl54dGhwpBpGzSTwy4tMhAh5p9scY8RuiTGr87QAuKTf3bE65OG8n+oX6HD0CHtpdhWszvKVzEN01yvPMu4ZzfqjEZ18++c1Gvi2fGeOwWPrfrcuQEdi7dANC1jnUW4O4z4gWB\\\\\\\/C6FYnuTYyEnaJPn6AvwBF\\\\\\\/6byuJdx5LDZpeVxhuJONdo6bhmIad3RgA0IEerT\\\\\\\/b\\\\\\\/epHbBfgNOs227emVza7jTf9MO72QEd3DDEJGs\\\\\\\/Wdc78kYwS9bUvt5DpSbUa9Wi6fTgwO3bsfT1vnEgzZEHcBLK6LMro7HnEb5T4ous6pGS4fXcWbk8Xnpfzo4+kS2xmZgkS6GehT9myvv2GWcrbXa\\\\\\\/yjmMr7jc859R56nB59I3D+GRvNIveNBkHHTcciPguTzLMYXB6LIxLZ\\\\\\\/WAa4yHTPNmIO62gzd5AehoTwg800oKE9w5J1AA8u+CG2fgIlt8reEpmlR7E\\\\\\\/1lcca8paYX8XHDdKUd68yiUK+Z5hw0G+uN6xGNk1azc+u9yHrzUwtTWs1EyK6nAFv+eBzMmlAyLNwP9d3siCGVn4e\\\\\\\/JfJzUlO4sLV5kXrmh6CFv2MgWYUbq6Dq\\\\\\\/fMfKG46O2d2ASMx9pwIApd8BrT7XPTDLvZGtkgDf892Zd6f0H\\\\\\\/D0m4I9PNJUl+k3TOacjb7HXQHYXM2sMMIcA23ZJiNzDJybdqV\\\\\\\/W3AGp\\\\\\\/gY02G6Y8KMTfc78uoQfWfLo2lqMEivnpUlPBDqGS27TvZKhNGQwYlMTko6G3OyH4YnaRh6Jwr4SN9MiVinqHEiRSWpNDc7vl6qLCw2miXuK2Pk1OOmLGxw06Dl9kTE0G9ZYiE2mRBQ5xQw\\\\\\\/bwGw9fBV8PY2\\\\\\\/VjB\\\\\\\/45jONYVYpKDpxgzwmY8V2bEgtiFOetuAcrEe7745nJaTb4Od5Bey5lURVyCBnGGgwEEZkmMEvHKoeDSTZ\\\\\\\/2C0shEafgVIvvCEAJdGUsprnw2u\\\u0022,\\\u0022iv\\\u0022:\\\u0022pzF8V0YL4ETZd\\\\\\\/91KxA1nQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e3df7e6ec3907c38f303c5039818bfe048d10765890323860df0ce317cbaa867\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022VcPsA88kwFbLdtkp\\\\\\\/53QfxBGjQ8lZwQLQ3EImVjWTY\\\\\\\/6+6cRz6v6eegiZBnEh5jDOd5Xu2ss0Rfe\\\\\\\/ZgU9PIA96LXwuLqBA7fLdf5S0mEnKrKnv1tOj4FX7\\\\\\\/Gku0u4B2W+6xmjKoPXrZp8elznVxRwH8Jvfn2FNB1CS9MX2ZQTV5aInv3IsqHawyN0blEjCYhj0lKb+7C44qIuwJu34NMg8ciKvKv32GDqxWzrbGNIXxgDM0xIvd6XXdEHWhq3ypcPx0x4IVMFECvSHOUUQiAzdkRzBOVDgrc1LVuaaowf7mXwElf6t6sExe\\\\\\\/KRGLN2Wwjg\\\\\\\/wrbVwAsJCHMQtjupcLEWNWpQ6XweXvjsA0r902kTHPLJo6zv2+Np2o1rhxOg3uCUMy\\\\\\\/90ff+cvvq8LpAYijv73WaZiQtWlafjuMj7a1yCkJCflyhCWhNMj+J2SAxURY6\\\\\\\/DPr2rsNqrvMb9EsJxxmu4SNFTmTttIhiiZFQAcSWSHQuQbJal4zl\\\\\\\/duVjQEbEgbB6D5qDgqNNR+7uGi9EO8cYO4Ico3i29sHFK0x1M3Zdx5vzYuyRCIkwfR13phYSgsWA4bB1s4p7av0iww2QCSYdAeaRMENApqjXkg5JY1xNjcjYeUn7ZYo8LvtvcuNnTDXJpX9mDDD18kGJguyNWizIras7WGYbA0utukkspz4czHlnQe\\\\\\\/H9IvSOcBDNDrrnuN5jOiNzr\\\\\\\/k+V2oZW4GZZ\\\\\\\/L5V3ppGTfVMgoRbPJxoDgaqcoZem+xR56AOxqZ9HfdzzTt3PceB8wzYWbLpRRQpmZrCYw3aS0c0pv8NavLP850AFBiPkR2OiDjVRxV0jQvSDViuZcvFRF90vwgrKEdlTcMXKcEfADDeDPRbE23uikm4wZjlXMqsXx4NOOlpy\\\\\\\/3CX3pq+d2J04ea0hFFzDZ9l6QkNa9UKwQ1y8sA3qVP4xvXbSAmQZUzt79bSOHASoRLkTz7IygDzOHjA04NOcRGJlzaeu5G5bIyHWPxm8OzzaOjxGuSnJ\\\\\\\/rBybIVUeooL+kI7EmEhWclBxzom+++cDdcRzkqAUO7mAOAcbPQ9UM\\\\\\\/bih2AK7ou6mfmjMpIJMKHuTFGKKRQA5LKTU0Ii1XhUCwjEdOfG74dYgfYxDwcMHt9AVam0fq2gcBDH6vXWpRx3o+\\\\\\\/pQKAr0Oasqr6bsUHr\\\\\\\/uIqbUEEifV8Nc0htEhrDNvgIuJWKL4ArUXMFQsv2QfcI2nxxatpy5KdTjW4VT1W2NPN4Vv6cLqN2\\\\\\\/WtdqRnGlaXfC9f7hvMdZx7Z8n70hqh6YXjpi48u+1rlmiXFjpXXon\\\\\\\/7AjQlzgWMMKdgcKYQ9BkUl7XxPY5JMA0vEVILsne+qh+ahASG8ofQ5jr\\\\\\\/8Q40at0VnOOp6ouTs5VErQIWE\\\\\\\/4fWw0vY\\\\\\\/cLu6+8fglGvdKggF6ZhfF\\\\\\\/Ypj5GYZS8KWO9l7Mp68Qjmtf09PQYLV\\\\\\\/BLyGni6wKswbc3GhfmNqOB3VGl5QGHR7bOlplTAAiAkiwoIAWWOgZbdNhE\\\\\\\/l1p6nb2wlpNNzZ1rS9nyU4IrmqmtViqHOhgtNcYf3XyLDzksq+G2pt6yKGRn311HHSg6ti2bDVfpoO7M4nnJgcck1jc0dpYWiXdbIXRbDY9\\\\\\\/FpUsGScZUrMAabElMewmgJ\\\\\\\/Oy0qMlr2gXJWrwVI4hoso2VfFN+QpPaMDBI3+PWxV2Z+mTxG3xgAOFutj8kxJR0wN\\\\\\\/PU7StrBHkUBhnQeOFY90HjdXFKCJnG\\\\\\\/FNEEhAy6xLn0aLBvctULR5kqcQwQoJt97On4nhz5yrVZI6a1g\\\\\\\/djRkXdx3L8RdAfJdJgOuodpzUVWtF1v7P+LHmV4phK8k31Q3cHhTRIkf3nUT9TM69r2RkojGhOPYlksCA+6KkQ8TgWv52PnF\\\\\\\/3Ty\\\\\\\/x1b\\\\\\\/0ciKG9OLOYOMd0ffNREm6KDQQ3w5QhOqBuh0WdF5l4He6T\\\\\\\/MkgGx7b0uKBoxLi480UdK8cO9K8AXNLq9dmS0BCi60TUsTMnFFrkx0qTIKwms7uFsDEJVhat8Q0emBjZOR+9JY6LO0TLFxd2hLs69gmf2eiG0EOOigfyJqixXKuDpW22z\\\\\\\/6qbKm0VB7+1RwD5CcKu7dIe7URMg1FI+81RSKa+qiQBVO+29nnQ6Ewdsp5wZiTR5qQrihD74Vgc25hSnlW9avZQcTmy568InmRbOpRNvmC9Y+DtaTsNEMEU71sSt1yALZp4tn0tHaPK+qhgvT7BvAkndN5JHIrd4PL\\\\\\\/2vLtFTlqO0KUxek9W31Vw9mNJEbqcVG4V295rt+It5Hlyqzu\\\\\\\/1UMNXd+jWEyrT5KpoUqI0j5SWQanu4AlUa7t2GqHnuelLKMkisEr\\\\\\\/ABe\\\\\\\/JLXhpUby9dqg4DkkonT4ygPTsvMGh0z4ixYGfSPQTSPfvEe1TMd9sTPUvT5JiD5ZPi6NdDlqe8WdvQk8Nwd75Y9gFxcR3JE+vr2CJ5JzpSmMXNpxekwD7mjEVj6F5\\\\\\\/tXJoaPVksRZ07ANlvRXZCYo25TxN1jAuAFp36M\\\\\\\/WAJGpgnmKJNVwlaQDiGxFzzQyp53T5mvlQa7FfKgLIIFABhLbZxD9kC69pKGNUSSj4FRBH7Y+2dQUYbm3uEEMP5jTxva65Imv4yq9UGvn+8w7YSXoGvxi4n\\\\\\\/gDxy5h2gb+NOW8LVTVEnR1ej+OhLLX8LcfA6QY2wEHZtrvNVc2O8+f0y+1CMOgaf3c9mh9wK\\\\\\\/x0uWaVuqWLl3Rbhbk0RhlMVzPnSksXrRKZ4NA+Jwxt+7o5hrc7a5fsRl89nk1KOZqASfR6jO8tXQCswkRjr\\\\\\\/Z70qyhkpmf9P18x\\\\\\\/7ABo9ET4BMdflfhtcmykzqg95QaH2twqw95ljkG8ucW3XNDuZxWqBQhO1gguWfaEHqDF1lCAjyT9e2aDvl50S4haE7uMU4A257+GjsgRd3U0F\\\\\\\/nQ42i5nLH6UjetCGX6Uuf30kWyFdLvjUZ4TQ2Ex\\\\\\\/ZkR+gfSzI+VycTtfVZriPV4\\\\\\\/\\\\\\\/+j0kMlJnNlZP+Do0PGpFhlJxW5Hao9jX50H+Wi8gRk8lCzt\\\\\\\/Zh+tst8JsQIZFetyrpnNJ+6SYh7VmM8FpB\\\\\\\/ufsSg5UFdUoVcqBvDHKkq8cArI5mqle1O7K2H9AVKUqxEr7n9bgqbhu\\\\\\\/WDkMt3ZaMHgBvYxXO3Mk35mwf6EFOoMxR1elCbw0CYkcKFBz1ObSMQrXJU71B1VeJuymzidsBjs55pB4Dus2vZmAcBYcPD0SCGFguPIxr8Dltyq8+xOyBbqbnB+nkDbb+7kgtdNosYWjaewElwk\\\\\\\/AcSl1R0ypkJeKUQMkF8R8YGM9jKnAEZ\\\\\\\/n9MHpCqYe4yfu4s\\\\\\\/oE0awJTgyh38jZvRi9RcrrumZcik28ZtrRNAabQtysgnTu7k1l8BUPQRiaQ96OxRWTc+Kd+gtlL77tg\\\\\\\/w\\\\\\\/fs7CELS5ZRL3NQ9nXHao0EQGZoITVFRJ5bGEX1fGNEPt8k1VB4qiqJpN2zrbAU2APppucjV8Ra6VUf\\\\\\\/T87IGg7ygQEj0dZqpfepL2hPSx2RvdaaTptc1bkHp3eu2M4Py7SkNzp9V04aSj4vLYk\\\\\\\/auBqYfphO\\\\\\\/TgNOGay6vMeFHjxyitCzafJ+He1t8PTFXEChbqdkS3c3lYD6rEsVsTga6Sbaoj6rZzOtgeruRc1bHnlOkM1fE3oZGY\\\\\\\/LGRAVYSm3SsyRmeW6U9oNegvcla5L5yD6SDoap6Vh210RNfySzBb5E94\\\\\\\/VBtVoyFOSHrel4yIWLbUBrltiSx2IfpIEQmqxlOFWSUM6zYrS0utFmy7DYGnMcBv9HR3RHqUoHqi954AY6S1UYTGxRl1ZId32sWLPpKVzQXMnilGDj8fuUIHf0mppFYEh2usOszwP\\\\\\\/BsI+zkVuQaV1LDgEdwZBCMIXMSQnDJhvC7cPIfBWhW4rr80WAsP4EgASD9zoDHs\\\\\\\/9b9D5p0bMI5Ovchn6Fbm751+qqKawfAjrMSpW2aZB22Zi3H45m4RqxEdOQ1QssrKMEGEm4HLSBBtP7W0fCen7UrzSHeX93MDOn9VNd8sKSlQaOTfoxsyHovzgQgyLP\\\\\\\/6fNlat\\\\\\\/k53QxyoJjGLKbcDvxqJHgOI1gVjbApQnFv6cQgvx5yBnqgFNpLR+wQTd00PZnTNL6Rty8gzXegxON+NPoEmuEtakO5q1VYjuRPCMNsFFfmOlZdtxIo9cx53ECalKHvz9UJBtuPMi8+OAOzaSkI+FTwM3C9K7F1YZcvVjCRsQeZuw\\\\\\\/kGouTLSkM+jJcckHupCGJzbh\\\\\\\/nmrRsBQioZBJI0we1ceSannYRO3M6rWTiJgOGlncehzhrMumpTw8Q4A1O70GBW\\\\\\\/uBE9Mte8d8kpbu91uibqgW\\\\\\\/VeB2E74Y9YKxaDIY9vLPl4vpnt8c5uDnRwfwCBWXiGC6Ak4cVmk9COy3IslydYZzyRZIBA5WFTqPnFkoWF1UKD+3oZP6qviP\\\\\\\/3OROKvV2C+\\\\\\\/uKyekgTJfK33rn\\\\\\\/fjdmmZNrvm9kj4ohKkkBfJgXIHukI+tprV3KjuKJ\\\\\\\/S0GSDNZBri7Syowhg2L0X942ENYEljG166T7msXqquL\\\\\\\/FobVCBGhvaGwW8UXJBVr1dLSwG\\\\\\\/Cq0rUAZKus7LvCkPmbZ1ixLvGHXnAXYzaAkt6s6v11otLbBxWZFSIBGNqfa7eijaSDLRaGgax+s91trQM7AHp5EGjj9mR79u5H2mvce6cGjAuiSRd5Jje+SHjG2gzjy18v9X00YMINBk3T7zUr8JE8TRHPjx3hYoYCCrYOI0jWQDHLVk9e\\\\\\\/8GxyRMbwslJrnSqZJ8F1XoHF\\\\\\\/lfzBTb7Zujy+CczMb0kkLsdxRm2TioCYLWkhHsDUyyrGgbIuJfOKG3ZYF+yiGB0mYFjNc46IxU94o54VYPEkvzCiHGi2HdVON9iarwPi7fAVr+0FafbbOpMSYjD4O5p3uIMNdRuQt7X07V6ZjQnhPQt\\\\\\\/gxuJOCAkGOQBEpIudXKTLWAxJfAuJuD4iNGZrwe0zkRchmm3TKPxJVeSOlToaOPvyfjq6SNQyKcTQNeX4YQZaXhodwFaAudlQdTfNzgvSjHd0sB4S1paAAj+lUA8COXx0UQ8RK15QQhzxe20O0tekKn3pbtYVOCHrkkIhi7ykeN+UVzxPH5wqm2c0PlvevIl5MLb8ZAaHi3FqwYmbYQaPENo8LVEElVUO8GDJq0ksOeKuAQvhElnJYeaEPYlmFIHWCC0L+sp\\\\\\\/OFUizOCBD0\\\\\\\/hpmMvxAOarHOHwfH6EtD3PEVGuGHPtALEQ2HMS6XdvkMnLTSTf78trv3C7XmV0itlgWwTCygWe+NaoC9IxZ0j7lyQXXiBBhcLl880nvq0\\\\\\\/L6bE2BmqXBAH1LmtsvblNoTdaeHzkFMUtBaMFXf3f1Sxx9jB4pDGGJg1R21AZG16\\\\\\\/Irdo4hPv\\\\\\\/3SbYj2AWPNLwKUbOQql2B3eJCqn67578x9rI+RjgJEW17Qy82vG\\\\\\\/+YhYwb3PsHSq0soY9uS89xvZRn5ardvjdAEZuY0zgLcqS4P8sXG14LfEedj9SlojuakM4LQ+1Vkop1rKoPBhoHMcHBL49ATKRFC+sBKlhxCvqprmO\\\\\\\/arX1UXTV23oele2j3QXJsX3lz0aHUJBh1MFPdeq\\\\\\\/FuB\\\\\\\/9If7qSA5E9g+XJsZStp9gzlphC8EUZqWPrMuHujpRJLHDRjqlyTVBEsyMrKoh9D4KR3ayjKPLqorBtWJCerqY7Jo\\\\\\\/rg4ZsZNUXH2FkKv0N+T6h8VJlB+Y5bEKXnF4P+fMZBM8oAxFbC3dwa2ko16PxgzArc5FvoOBVj+4OPMStj0u+I5M4sI1UtGS7rgEQLW1oGy1VfhmAO7CmM9lM2CmS1nZCA1EbxMI5iTL71m1mDi4PKD3LfbMTMZGWXNTprk8v00y3j6yJXPAR3AuoP6tWtH9G5yDycyS0UwtqZeN\\\\\\\/4FDq6cGIo7t2ECKClc5XT6RDuvDNXCcF\\\\\\\/aWF9\\\\\\\/8QW05jliMzzYx2E\\\\\\\/9j3z2bAco8Kvos1F48SzXhMLkcMsLBeaQpeto4v+HBv2X27j6i0fzkoPeChva+yhXlyTm00d8eoU4DZ4wNnuDExtwcQf1UDdImzO\\\\\\\/akcQW2wq1SwcxaEb7VTGA44PK3+0cp3vsE6LFxdOeWHm6ODkbHuZzATCKcGKQ1MZgMaj1ACXSUq4mwulRDq+4S4vu6R2bzZCUT\\\\\\\/vQy4VvaHJ3sn2NZTuVQZG7fftQ3kze+jQmmE+aJYUXdMU7nlnLthuX+uw9hB4m6fb1aqNmUK4ar9GobzzQ08ZnqOdI\\\\\\\/HZaW95Lpdd2DcBw5Bn\\\\\\\/cPlWAtaZbY2ciFHF9ClVKNQjbCfOLllK1\\\\\\\/L38zhEGPtPVhF8qw0nVbfFrZgU+2zIYXAji8SboQkJlz2Z0VLIFvpIiuNH9+yipfCowaN2bxHKwxFzIg2DaF8i98p8NLEd7e8kp+KodWhXHoLCV5pIqvTXXN9cmtcrjU\\\\\\\/hGXlS8yepC9KM29rXQBP9EafnYfc\\\\\\\/VNWwPyrdbm3CWVOICrf9IJ+WdM0JAyZOO7KfsO\\\\\\\/rtsKRdXtOKWTVDt8kEnkDA6DWxKhOppDNAdRVqfrTIUWAygWdL0y7RO2IpLRYClsEiazQ3WC1om3SAgBXpp17r43yk0\\\\\\\/feokwL9gruk3x1bek+hdLIEUwWJRoL49oEBYe1Xs7m0c9NAvXPnvSaM2WtQ7lYQInfGX66ALsiGGXk7SjSx0yibna7L502ov6mcgRVJsGmKrzyYWAMRW+Whg+sO+wMV34eUSFFdLpt+dmoRgN54G5onXibyghOSHNkSsa2NoaGEd99hHQXYoHPW2yLy3Ze1gtApS4KoiZrOKraNNTwvt9SngEAECDzu\\\\\\\/2hYpyNt8zqv7EZJdkC+8cnd5ozPpY06JrWKvQ+w23iwbcl9dbAu\\\\\\\/8jHh2afpSqF7qO8f8lD+Ia4eogdx+DM83CSJF\\\\\\\/+5avR\\\\\\\/KjNro150ikeCv5lpNWEwbnzUzgGc+2jLbHD2lD\\\\\\\/ZtwCN937t4ZeW2wL6xXoXcCN6YJK0AHJ+Qf8muBYhpXz+ex650AWCjbBlr25KDeHsi7rzWJlu5ucOEVdzHs9VjnADSo+2OVFrx2x4oYh7Yjqqon0dSV53ZLqwBuIoSHClhgX17+6z7LRlHwqtXMwZWfXzpekubM2KGda8YMQQfHCJUmEcWdVqrKF0jFg0KcTlHdvLaFu66sVyKSNp7l4yHEwEg6TZ6tERXs37AcfQr7PBRTO4T6k6GhYpJBnME\\\\\\\/\\\\\\\/txhV0kZLPpAZZR7MiViIMtnBUFH\\\\\\\/3wT0k13SWcW6GdxGfiofq97VEXXbJSFC5y461GwAzecXMpTdzHCdONUWl\\\\\\\/8h5FR4thdSYpLI26UBVzVZzn+t9kKDFJSvQcXqLYyQ7aC1pMpJSqFipCQSbmkVllkSqAmrNkE2wPFuWzbVQvkLTY9CNqisydbH1WMD76Zn5hsGZLNbqQYOaXuWKq0+zlyqI4vqkIppNVk3OyKzyakuOyN\\\\\\\/IjLnLc3ryOycCYOVyKbz2arcVraX9Ft82KgD10FyVzLwlsGwKXzUmQsdTxOO+2koDEUv4EAy12GM0qUdDWl3mNlJ\\\\\\\/0qTYpM9AwknE7VHRkHch7GQ\\\\\\\/JeSkjeq4gDiUkzT+bVV2aw5OOROi+OQE7LOiySI5qDYRBg3jKhNjiIXO30HKcT8OpOsvMmScOCq4zdnMSrNhYZfp1Gx7cOc4U6Z+2huzOTmD3+6uFa9pm9ceJxbaW+xexkSySkzRwzug7Exmmz90cLcmbFBzb86DT5LEqPmQg8+tmzJq\\\\\\\/XcBlwHOgYY5cex0xaNLX+dyNcuVgNgX3xBkNXKoi3awrz0VskK8\\\\\\\/RhT0lakb4\\\\\\\/A5\\\\\\\/A5kqM0xQWjwfp9TijZCZrOCDO7ZYDCtBk70tQYX+oMRq063X7wdI8VnYjrDIyD2xKr+w6rM31rwP15bqx7hq8Rzm96c91d1w4F1TkIjjlBS8tWtkrjVNN9q30cAabvVa2rE2qFba62drtgHMuHGnkUqiD\\\\\\\/BvBTScwf9Xu36eKki+0irsLWatQij3zhy3qUkw7VzlBwZ9j0kLGh1M0MEpclGZ4opi380m2LfXDBdMM2Ee67uLaIVq3kRGBku2kFijo2ArD5y4UmcN3Rft\\\\\\\/W+pl\\\\\\\/SZ08DMQQeh6ZXUDc27X+Icop61zgenbFz+V5NQXy+eAuJZGMh3lfFq2DaHb8RtsbyaTYplRq5aTeUcwSoJp2FWMOLJzCcxJ1GEB1hymBrSSwmEc57O8HBOElsntvCFDQv\\\\\\\/\\\\\\\/RH0UlKM1H5tVThAk6zmnZKQ5+ziqiYs0R6F79mu0Cq11BznqQxx7tVQbBBN5eYzfAWDeCsSx9ULzoeXR5ptOygQBHH4fof94qNJY07\\\\\\\/E5Do7T1A\\\\\\\/MQVUcOhCMztpt5bOFFAJTqzsZ2Pykp3tkI0rtNPhg7Aiv3bMPM+4F9jmfS5ERy7gWhW+TRN\\\\\\\/LjvCWLj4PCFKaAdHbZhPlH8HKmBGrmAUyHoNhnNXLpF5TpFfro695KKheX5cJ1bQA08wmM2cizsD44d8zDBqSzpYY0ibk\\\\\\\/FU7mOOtOQ93JbLa8b4xCWil0UHotG03oDcZjR6CrWFq\\\\\\\/ZMj0d4y5yHadpCkFDEaHcYS+QXnvJ35R\\\\\\\/2iJQn90vMepObFdQjabQhG9M83FuGfZktJpiXgwnzokBF4CAz0SIQTQmgJRMvn40WIIOWumqJnM7nf8W4h8gWeK\\\\\\\/7pofpm5BXFE9VqHpRj0ycGTvqdghcQ+xpZtsbNQdR1rl58yMfhVI4zAw62GQGrEbAIgcSNPwsANbW4ZXi\\\\\\\/iADnNLrz2c7XNhS\\\\\\\/O\\\\\\\/12X9Y9C8f7XY8o5vN+INElfAN79tSBn+XKHgSR1QMs4BAIJcwmQVEJm4z6gzm5s1WtULvuY6QJJOUD8RPD8EwNaCRAT1s2qUXQmj0Hv9\\\\\\\/KBqd7g\\\\\\\/P\\\\\\\/sWVEIji+x3ZjDLcIRgAuF7RH4tgnurHC1OVC5kGUgsLKoqQ1L6uZViWKAvXm0vFryS2cHvGyLcxt44kNYZBws7oEQuNZdjmODmRsZy+C5icQRIPNOUop6bavrdAqkqKt2gDAgVPOnAbg9fVbqoH+LrsCyNdVNYgo8mMQDbLHbxA3BVMJcaGIIZDr9NlCbyqAaydyc3eF3Xys9LXUxKWVqx7HsI81zMt6FiCQcE3KUKNxBr1fIyoVRmWTggr1R5BVXJc1BJ\\\\\\\/mhBSmgHS5+Syf2PIt7INmtB7gUzCi6H17o1QSa3fK\\\\\\\/Al8z7IhrPN9qNwBeP+dQISox5uOpDMC51h7xF5kh4DFwPlgrUQbHamP3mk1y8elUw5CkcMICBF81yHHuMUpUetDJKt58s\\\\\\\/OhaYOdYbvJt7qswXK7YjW0boDPfBJovW\\\\\\\/1u1c\\\\\\\/dehpvuqiPlY1IRlAvXrecwhticlrTOkRB86gwpySmHxmDucp+5d3CHpgQFZXJniRrBU5\\\\\\\/NzGgy\\\\\\\/9XxnLjUPbLxktOB6iC5K\\\\\\\/aG06SY0WyVwbu+\\\\\\\/gOS3OG6LBkMPwaSVFOX6JcdWdsikOZvlaoLbggDVPCSd15oAXdnVo1\\\\\\\/drL\\\\\\\/E6RoTDRevqK1T27hdJLPCPHuKf6\\\\\\\/9Fc7ZULDiygCR9W+j\\\\\\\/POYWfXBpbpiJoNdGsJqb65LbhwILNPXBHYJlb1iwH3OM1n76+W3zGwffyG4d\\\\\\\/EC5WggNd1NDNQ65pnVAk52BlspLSuy2k060k5lHGaks\\\\\\\/Q3pZBddBStDAsrvO09dYNAqqkZna7Htt0kZJNuQ9nl6u4NnpyHYFdLEW+24TcqZKQP7PHJP4GkbKq5Wa4uh4fBxLAVZiSsrjFQ6\\\\\\\/6GHUE9840D0IiLbXsCFfznI1lzI18AFs3DwklwNmV26u30iDlkhL\\\\\\\/tprhPj1COThoaUQrBDrbXDSFP5xnOzVe02KYt9aV9LjLbtRSLHKmLa7jtD9PMDHxqtTnzG39U66Igrfh3SgBJXOV7iuP+5SKPCDv2AYWnvK2baFQLz\\\\\\\/cD0e+tP86gcrV3oEapL9xdCH3lyC9eB7FYg0Htwb909osr7UrsMAgg1w+yn9Zq3U086h2cF\\\\\\\/n8PMuQZtCGz5n8rHuxg88B6xRYhH71H1qq8q5Jqk3sfhRPKuCqKNNmTrWgLVUPeJJ8+3Ube1ZJhdkjEia4aG43BQUlLV5RKr4pCkp9MskiT6K4O7tPTiOR+Rx\\\\\\\/qUqRjlcxPjPuXFOoGELVI8HiwqShcV67aCHS3tV9EfPOQIphmn6gsfoEiJu0fWajyVK0hoqOPwD+3X9Ji15bkdO1wq+Y+KDxcwXSaBBqgUR35zJsZZMiLwnwyUvBzzmb0osbEkD0tEExw1t+35\\\\\\\/CuIzZTE5LHycjx50mU5dnJxikNRKQwrj1NiaGkfPjJPwgzk8vrdAdKOPVJsdEFwswHySwlP4tyQ+mRF8CJeVKMZ4iTB9Kg21ko2dMpFNks8PbuZHT8jzkav24AQED088pmHjZkuz7EZhg+ME6DqSXaxt7+cO7G8Y2F9EkWsBAN26anaGasLUPXNH8VI86wfvYjRmNapKNE1qYtIN57xbAirh9eH+VQT43mMht6PkWspEuVAX+OCzKhNf9olmThRNaDFkl2YwvOcMsqrQASiF9GVen2IMDu\\\\\\\/MktmkcxdvHwssxLvekmiK5mJFNAf\\\\\\\/9TLYDMBhTan9l6Aupr\\\\\\\/c7qDJwt0kSG2zy7iwiQOATiYOWx2kRY\\\\\\\/8PEFv+9wIV+EMCD17xblFbsNLOrotC\\\\\\\/no5sCxGFCqoXNT2TuvRsXc6RcRywIMDkEyP2Oooge8ZReuIdWX0nQELBQYwNCAHmsfXAxrkiznXSeNUA\\\\\\\/ESco8NkaZoMHdDWEFEcw1W0tYJZCJzPVVi1Pfdyguj8vl5QOevEEJnH48C3cpS4u+pU2IpxtaL1ad4L+p5og337THR1THymtg6tX24JMI+pk1ckSmrnazYZDaWx1iF7MTzRsFhdw72jiO2UU2jO71EcsNct83c6LgKkBz93U1y+A\\\\\\\/AkgWgtIJIxW+VYf2xCHeYAUnm1UZovocq1OYo3bnEizDmqhOoj7c0uoUARQfMvyE7goi12tJnJHJDVl0tCXgFv5iZ\\\\\\\/jAxSRY\\\\\\\/5s4MPRIrMd77j4EeD16STNN7qIfgIwKLhG9wX0Smt2BNXZ\\\\\\\/608yRnTgoYc+pIonWcRdhLInX1VE0\\\\\\\/j0XLrhF5fZpFYhu9oGiI9baAsQn67ZleR\\\\\\\/QbHM0Rb1Mw79JiCzWxS5raz68Shj9ZDl+If+JOXzqCsl28I4vjbWaXG2sjOjplasRrvDb92XmUv7RwMZXUnjjrIlFqBAi8sjuwEZjgBilbk3qsWMuc5w0kIJjfDJ\\\\\\\/j8CRxc7bHS0+H5pzAZdL2zR2Knva5hiAMCz\\\\\\\/bxdlCgxKnyjIzINhWvq0zw5SK3vdKco4141Vgfkt7mW0YWe96Y8kCdxnluDNb1ezFkTLtXZV\\\\\\\/omFrwKaq7uWrdSP51SCpF4+kn2deVtuI+tibK0Qq2mkCNIeLAzUodMin5z8ZyG6KBEiGgbG\\\\\\\/3zfLfZm2gaGJrXJ7Fny9\\\\\\\/Tc6a7czci7YcUqEkYPpEkXXtXMEr53P\\\\\\\/yYSoKtVT\\\\\\\/hlTShT02QeQfzTRb9a4wOpNYAPUyxl+e2bZlczYCSX8HbpFpaeAhOyj\\\\\\\/Xg7w1euuWhEkspYY9psdfQLq12Vgvnt67OtilLX4DGqedcSEzM2lgBLnNQqBcI+E\\\\\\\/naZoQEWATg2UJTcNq\\\\\\\/c8iOEWP2q6ccNgBDKto1ngSN4h0V58zlq2HQ4lvngfT5wmjKMjm56GJ1\\\\\\\/\\\\\\\/FHLv4H6I92WGV3wHvRkYO7mDnlCaQ8nOhzA9iVcN54mZz3w2iLmQvpWPQqXGZ8GZ7HDDn2xWp3msbLhmPHbB7NIPwwPO+y5MuMkQmxoGbT8w3uiR40BFVUdeoQG0XzhqVDiSaCqKX4ydz5\\\\\\\/z4yeM+sn9xnPTrFSKSRn3jqZYVQ6XlKoCcQluI4HlRBVo89O3A+N4+ulQ9Jqbbf6A3OT9HoDbnU1\\\\\\\/Hzw03Do9NYZTJ2Df4VZYY\\\\\\\/Nb9lpdFWfVBA6jil3eZpcWWSMTup\\\\\\\/6l0T4jzxxp9R\\\\\\\/dCS4LPMsX0XZOs1I7LeDfgoN7jD4kb8lFjJCN0lCeYNlaijgGg814c9vX1Y0XsVCPRuNlIH+NOiCUKyUH\\\\\\\/VLQx59\\\\\\\/ESNwyPskaJAoTFaFl2sSNJf54l3Nsuw1nrWkIUIvDJnRh7lG1r6SQRrDA47B3bqDb6LJPR+Ej8KVNAFWSfgakUJSEFRgf0AZnoy+4wjmoFZ2GyWV8d\\\\\\\/ih+jLT2oAwJaSnQ6OyKJ8LRrxGQSfqVeyoBDu9\\\\\\\/warfik94gV2PSeVI6\\\\\\\/bxNCmzIp+ckn9w84H0GsS1WJcrw5YPxw6bi7SAY6jLLty6NpKyLQRJwGZNwOa6jV5f3SvVygw8xvqHjEJ25juDn1RKSxH6BzSLUacBwS9jhAEZmXdvzKBMkV9RYVBgxAR1mwGhe5sv4tPULsnsLajbq1yXsdbviJ88tGHfKIDqgK1I6R362cSPisX5xRcmqnmEi9IKl4bhdmLZQeLLJEwrgpa7qWIMjkX6tHjCVMPZsO4netNneQxGVuzzC3qB5oxPCR5wSIxAwlY3vufje5bVbD1\\\\\\\/lNg5UJbZO4ibKvvVZeY6M9R2QeivToSAM7Do6COLZtRjwCGtmdNotDz6OWVqTH9njS4AvjKufxEPpLykXQsXJd5fYeTg928c3NMDgMnQerW0XCgsjbGgK275\\\\\\\/06Xgji5mvFjwOBqdCQQ5Lnk\\\\\\\/GmO5gifZNlrAobQ2os8\\\\\\\/eWIXDCRErJ3rF1250PUMs0p+IvZDQrlHUBtsDINdqMFMYAJYo+M8oTCwDcNyRfXjFPHROP0Bp35LQBzZWQNv5msqaV+9kXX9R5RETrsSGkG6dDk6+1Wp\\\\\\\/6XOnGGPpywNzmQDc9ewt7oJZmvFGf\\\\\\\/1Kiu\\\\\\\/+GfIeAwSgARb4ldIPjK2XZ0y78LwbwRj24FNpA07wR4sJ8I9zaupBF0ik1HaqCNUvEBOHu8y\\\\\\\/tIX\\\\\\\/ihUMSZM9r\\\\\\\/s4ikjOCvma8B7pihW4ULAtuh2t2LSr6XmuKbJnV0QO7hgsowzZ6DZKr+d2e4LHGxrKJuZTIaYQYUUL2WFC3KAf5IsbYtDImHATAmocbBAGnQsHKIsvKY9GMtGnu9bsIO0zsMJ9yHa\\\\\\\/hJKSeXWlq7Uz9ccVCSy1oTPkEGtuc2IVrtI2fupCZqq26FmidV7xAfonIB669TEM1LDy5knMSPxGyIuGePoYTeQg7I9flIEhzHJJzzOGq6QMAn9E2u5fcSUcEHvlrfkzn7UPbs5+XEmlnYEaIEHRH3SyqfZRoM1bTOXYtahauNaLSsH\\\\\\\/mWm4NNGrbjjANKrXoyoRxCe2PT35KnDsrhhJooSKDzbv6MfR0TbXWXKBVejJ2XS+45IGLqb+cH\\\\\\\/ZFcwoLrpKdUFhTBDV3Yrl0SOyegK90B0cdeYLdmtaTu2tqbVDtXNj\\\\\\\/p8DAI7Qxuq8B75hZ9VD\\\\\\\/sEMDa08b+kK4Xe5GG6jEwwhBGNOImaa9BcaBBMkzW5Ghide0kiVHRGh68WxKHreJA0PBvfToXp4jtDCdgoetwPjge2hqpSMkxMx1mTtcm76laYFNgOvj\\\\\\\/g9\\\\\\\/bywdQrPJNk2KXL3HvnoRpzVkO3BdVseD9yVJ5wwl+83Q8hiXP0bFlddPSqarjSdKkhrON3X5b6tULB0eRDEoDpedw7gm1ijZYhHC3f7dUnFC0wx+lGzruxxE8IZrZ7L9t2NdndKnG\\\\\\\/LAuvV6IZ+5f\\\\\\\/4YZj0FIxldFq+mKyl9LCmUG8WAki36WhBNRyyNmxz6Qps1HrbQZOgbRErs6LUluk0jtvQCoysuTBN1VyHq6075cCauf86jMDKXoHBlb59N\\\\\\\/NUA4s9vq8puXgksu3aKStwcE2NnYCn9J1PDkWMvUW71MlaLlClsb6vr2c4AAAICXiqnsFjbHM5KmRiRNc8DaMzL9UNUrMJRoVnuSSHuMV7XYXFDY6G+jtiXCjsSPkN4gBorxF\\\\\\\/PC48rToOvD91EymN7qvpLBgi3IKEPwkWOzXutzNBkoZviFe9F5BWjYpsreC+uktNFbZAqx3YKzyaB\\\\\\\/bgYPOUAgtdIpCbqvNVBLlZ3ZdloexN\\\\\\\/McAcl5\\\\\\\/0flPue4zJ8WDpzbr380q4zESgJ5S+0X8EskR0KfupDZE3jQodU9pWtNeu608Eh6ZMwH19ndFeTCbLP85YsOWyfeglPrywuP4NOLRVxHRfdo\\\\\\\/XSLxv0UhPbWQqSbd7yIMjw2aJBHWRqPwTafmTZoSInhLGqeW\\\\\\\/K0M0VbzdcABfOmZc64b+ig+Kw\\\\\\\/aKTFP1C3RNqQ07v98SIsjTSm54X3YSvXKDiWfMFafSXjgPu9B+CWOEjxILyPjdzGMqgPdWY2DI7OhyW\\\\\\\/k98zhQY8WorSsHZj+dqNYvUZfDTBJb58C4alpZwQpSLlN+oqp792LBcAmTkHmQAY4XeYGlyDO8Jhqx1a\\\\\\\/8AM4klB+VMbvUz8t2Sdu+TAbyXyC\\\\\\\/wQx8THqSjjK8BnKs8SvEgQ4TsF7DDEKAddiAOiEOtxFpJ1p+jojMc\\\\\\\/\\\\\\\/jCPSVeiDpck0ItlnOzOj34jPd\\\\\\\/hArkszm3VxnTCleBBojGUjag9PgNU7\\\\\\\/u4xyptWfaepLiIj22fu1DPevO80te6MnZhmY9z1o+51Hzc556+RYkF1gZvX\\\\\\\/vaGEKOZt3mY4HXZF2WLU5JB\\\\\\\/KA42gTKFPcoUjx+lFtXVx+57HBFhYS4hMFsTXS6ba9FZfdmNzMkND2s\\\\\\\/m3SXmFPcNkTw\\\\\\\/WkI\\\\\\\/F1zDLVM5UBySU7j+BA7EBPh08qcQIoSotld+CubtG+AO89mRy3EnW10d21P5PpQELLr9\\\\\\\/7Wq3EcN4pCrrAFhzK6LLHMnS7hmpaOA9m+9yh+CGjsf0WEnJSHhwz2u\\\\\\\/7ODZpbH9PVKSipxdcgJ\\\\\\\/zJRBBmvNpw8N6gRjCs2mF84i\\\\\\\/6vaGV94s0XkM3jWLNFEYgthrIF+OSF62XjReyrbOGe+p1iQWkt3ReETMMfrHIWnlKbRFFSdKcyNh+U1tRCIb3Va9zMrHDsNZJuoRyZDkv6m0AjQY0HenVkWdYGjCC7zk8rUPDS7RN4\\\\\\\/oDJXIjLlfsjDGqpOQO\\\\\\\/KmG+Jnwgrd9iHYWKQhLOLLvp2q9IDsVXHchoMoCWPnWcOZlh4w1yAJk1upytKkrsYK0KeeR5HceZMzNMGLGtvAj9zGNZQaSscGme+Lf4NMlcOE3\\\\\\\/qlQLPaKb6iDEqHdSjlpDm1X5sAegUUssJ6x6iXVwjf4FdcrhHlvrHpZQU6+D4RjHrKdfWZTSpFPkb4J+wccRrR7DzJ8XaUJx5R2KczLtWO\\\\\\\/TWp+W6kLGg4lk29A5e8i8HAdnKFLiQCqMCih1ReQiljLxpHipRxqFC+ayPemv9p6Dj5wux6Q9olHn6BSU60W86Ua4gct0bXPHPJKuKIWXshNxfRy8ae0i+A5y7UY+chKiB8qN8yDp9OTVhzohuN7vYk6QvywgmkbmDSLrgSJGkzyqPxvv+HeEr8ZWlmxgkO5B2wVnj1GdLWj+Xbho0TgXHc8DVHeyRjlx4NMmkHljLMhiTxsfQHQvglZWtKjUEW8dUtx2XQ9gOX92B+ZuiVG2Kd8xu\\\\\\\/xWwuOD1tB+M1BH+AYmNYQLoOhDffDaK5CgVvx+F\\\\\\\/z4PmUUCaLXzqpGW81iXdRQC+5CqEwWY0oIhlHWzFXHqwEZZ\\\\\\\/CqkhbIldEYcOeEtnGKDevg9RQrYBKko8FoEbzzDa9T7PqOU0mu1zhX7kS8sMM+wkKYfEX007EFq3WR6it9sJE9gA6GmdBS3oIeD6Zu2XPeaXZ3+5dKCVcxwh1rY1CJj44JgKuhkCHXyYI0i3S9+OSPg9994KEwwFPCaF2QJr1CDGjCPTQXOnjW+rnhSufllR6SQAPKeH9bT9QJdF9x2xSWLw85T0Vt0+uBryLK8GQGaivSG2isW9XN2Fcxo3ZooUF\\\\\\\/KYVswMpFdryFLFqXTxh7kue+sS1vvrzDibA64WdI8agzHtgJwDhA75pBnrWluIvrfpnvIT3aUvzqkUGDJRkJSezp\\\\\\\/ihQX5Dcak0ukeT5lcwwnhjuyhuhV+jHscmOT0BQRP4flNGBMQOfQjFr9bwTVIebryzOf4M4qjVrdEPlHY5AwM9x6vqMtKCiJ9OBcGGqwIrxfU7kXC29LIx2MF6cPQ9Ee4\\\\\\\/tz54bNDetPinI1SFPA6Sf+ele9v2tyznCtnZCKJ1ZVLEKd9EHf0JIA1BBfZXrmGaRMxkdoaAh+IxBkxtxqdUM3QLn+YJmlWTrASmawW5wQMvDC0jxcVD3s3PISnZsLMBAYyr0DJ68R4flYOiOBlVwWPxOMGeKbcIVnmWuFsz4krFzNrBJ9ovx2zwvayoY2SBNOZuE6YlvqCajXM3qHieyg+hN+0RVRJbhhBQOMAj6\\\\\\\/i+o\\\\\\\/HOx7wTHSucFBWDJijVyrPa90brh8FHRc6BIG\\\\\\\/QrQoJ2qx00dB+z5Tnk7+sGdkb7lwUOm2RVQlfJDkkxpkFE4bqqRUfAScsPWhkwQgaURK5\\\\\\\/oM+N9W\\\\\\\/t8KoUFS0tf8tgD2s10lZZYM9z76h1sVju8BvpkQDNUqkZ6sk7mFTkl\\\\\\\/AHjla+yasGUcfixuljQbwEPyjaW+WBwW1Bkm9pmE7\\\\\\\/ab0ntyvWWyovsB779pnZFrJx+zx+qNccCQmv76qy3LokdcAtBdRwIqIpa1koqpMQx1A6tQISOA0chEFL+OSim\\\\\\\/yj+b0wED5ygotzkhaLX\\\\\\\/GwOWkWhRxov\\\\\\\/I3+MP7Nsuv\\\\\\\/b6+LSYtO4ho4\\\\\\\/XfDeKDkMPw2TfRpdjH+rF0V7NIiqtF13oXrNig47D7BZYOcoDtKiS5E5sL6kiuFfP7Afgdh3H7D9c4dIFY2VRzMf2j1SNYMtSR6omM5+6rVogZ1sUdc8l\\\\\\\/zNwRf0+d5VY6A\\\\\\\/MP8wiUyFTxFTPKKHLwvRGqPU8VPnF7z9pCbLwsnvzQ1icAy9K8LSKEyM\\\\\\\/6QWx7vMj4qJpEN1Ci83YhnmNZMnK\\\\\\\/RTWc3+bppe4ePLhE07pi24fJujML7cXSfkWmXYONDA4dSCy4\\\\\\\/mehZ3CsEq7gqA3o1WdXCJelie3iV8DMiFDrVHM6n4UAa0FdOsiV3QJCxa\\\\\\\/5jv1J\\\\\\\/tm6MZ1f7Lc\\\\\\\/cSHPcwrGL\\\\\\\/SXDhaYa+u9ZHoJTwnZyrlSBTPT1lAGT0qb1j4E53J8fhRYVBQj4Ylr6h8LSUJ4l9r3NueI\\\\\\\/fLbotO3XHNi\\\\\\\/P6eTtpaj0vf2mgY6xQR3tQ0McsrsCnoUH\\\\\\\/YqRv6RjIVowixJEiLeHZGRjsLC7chjCM9A\\\\\\\/BuImEqnE6Ab6TsSow0q3mtUVTi31xizBCENi04zrcOZP+BV0JuzjX1B2NBkT9AIH6cT9dvAXEk218vSoHk3IwAfBvWL+FxYpLUuGvdLPxOdAgINeclljg\\\\\\\/pC\\\\\\\/dNa8GWMH\\\\\\\/DaLQ\\\\\\\/7tJ5rYZPXNas7Yr0QUljOE+2bf7MLmcsR\\\\\\\/WRXonbigScE4aMutUn3b9uueKiJMmFCs80MUCqUy4BwxiSZydQ5EzoCVzv5\\\\\\\/0I8nYBAUIdKCGagGamb3sJBQtevvHjbtBDwF6U49I1uarISfphgg59DOu3ZyAN9MoEQ7j2r+q3hQXkUCU85nkiABXQbInlJWVN5UDeJeU4hURfmQN3h5ShyxjHgldmjw1\\\\\\\/a1x9JXi46L7uY6MZ\\\\\\\/RhJ0uybN27mgOXUUAvTKS64j1BZrMYNKLGeeyDVPEiMVcHYg4jVSKqDx0SKju0+93DHjlZzWy656nDRQIzeRIRrY9xPZpN4bhlUtTuLpDEbFBrxzcjMHmCjfEj7UQS1GSrQN51W9\\\\\\\/p0W4cftJT8+N\\\\\\\/PrHITlpMU0QbUTUbWqITxq767mgxhIn6vH+4bwQBLD58PoQvhPvx8TZq4TU3zlpJssdIIjJpPIqdfb5K5qQ2uIIicOQCutsKJjMOJmVJHoA1HAcdyJN87fuuDk4jmoZ\\\\\\\/yrUUKwq698Pl9wNGlUAvG74F11Z6iCroMJmkFXwcw1m7OQhKQ1lG5naylnc0jEdRwUSijpyDlOfcvJ778C2bq5oahEtmVwZGQheFKEBF2Ac4pr4q7477nNx39y4oBM+U3Lmt+fgY0UnSm56otTi7eMtFiHnm4lsaI\\\\\\\/B0h91ESBLz+z6XNHZCKIvZyLj2wMbhUJhu4FMM+udeMnE6T1pj35ZQhNwypoHIBk8W\\\\\\\/lb51vITvmNbsvBARy1x\\\\\\\/EGN+CK8Tk6xpRmwHkL\\\\\\\/ES4T6\\\\\\\/KhZpdL8i9PZxgVAUw9FlMSJI5g3jkmuHt3xK+Sd0nc9PdRzHFXQmgpxDRITvE2KpCS9V1yTDhedOy8XWD+zctkeNFYyLhDaJ2vBB\\\\\\\/5ZVLLltCpk1Fyeegp9fE05FLUV82ZuHYpTbsQ8bIEK0T9exFra5f5DW3agtIRHz6+mn6y1rSpa+pCVM\\\\\\\/vE4VdVzLIykhHRCpFeeTfysXVwGqpx7vb70Swihqd3EgFCgC20md7Lu2P76lMYGKYW87j+l2um+xtvj5bIecq1tcdH+hlW3QQpslLZ3DtUaKDakJvzy7NWZPXu3fl2cJAlqx572jDYZ2qfO\\\\\\\/c+Ty0bB5YglWaSNPt3KsUySB7j0cSaVj5rrhlXBzemYi\\\\\\\/N0swC+C2oa9RvbjBK4PNJqZ6flr30RhK\\\\\\\/mzSVg5vDybqWQ5hnxwtyaSBC7\\\\\\\/u6eyDKxHS9oLZl5MqJ8fLKIcNQe1Gjf9vLOKIakDy2zcFBHAZ6NtlUX+HHQkIajC08Gvom+hYDQrqRZ6hFBYbYUED1mb4+zesZ0N9nVobiuO6OWHdJSdkX\\\\\\\/9zlc2L2ziqRM\\\\\\\/alWyzVSD4QLt1L52gZEaErd1BYffY7udoqjov\\\\\\\/O\\\\\\\/aAqeS9IPkKbQLyN8fP7z8PXFOvTRB46q2E2fzArbpYmMhlxcaYwEUzObKtzNufDX4t\\\\\\\/YNekLDGLYmhXXAjaawTQmepibpx6\\\\\\\/3BUj3mXNr4Q4y35KWKiffTOvZjj773VZPbTJTPuZYUFLLST3kIZJCdJzdmuqDtamVWyrNLqQaNHIj5+skoFEAsH5o1RJdzYpLNWXvSexyEwmodoBfHc97i1E1KBUwA9njNMOy5E3maaw8vmAzUEMKgyMEUHYQtQD5VuWTDumvEck4duhWsIxq+Tm8axSh3UmyZF3OJY4AXGfXKdtzBc+jES1qVPhGNAaypoKdzJx9gvPD1DqtF77BHmkUXMu5bY+2mKmybVnYIYDtAY6iPomMXHe9oUSijPCjepLtB1GX0\\\\\\\/h7IJSG\\\\\\\/w0TkqX47VWvioMOK\\\\\\\/4V\\\\\\\/ZiPa\\\\\\\/oecKpne5mryIyXf1zM7Vz3CDByXmpL+hwhU7CaSYKFJ7YF62xUfoeOqyMnKfiAr7KdXF4nypHiyfMWbDHXzvgt85x2x0fI49Cr8BPlu57TT8zbI4jbELvgEL4BLCt8e0ENOSgPTFPUY53D\\\\\\\/JujEpLZ1YO\\\\\\\/aNyHILeoAZMsAScQN2QnTeFJ8SjifMzy\\\\\\\/6TYjUMR1radvYssUhm8seoVWrI+Nx0LEnDocd+Z6iAQ\\\\\\\/d8fzD+p9Ph7zaJGtCb8xxKyh7OgxYAeMi+Sknsz9xvY9MboYFTqAyRVDWm9Q7NqzhUH+vSNz+VfSo6yPlMecCgzbqFBtUlK9eezPmNiNT4sImbuJe7wU87\\\\\\\/UVStNAqkXC30AhWiUya5gR4YvrIbfmgPfls3GqJa7DV2PLtYCvTNsC6BjEy2gQPUv+mRvfHCDbn2YYQRN+LLHoFJZq\\\\\\\/xvFoyx4V92OHPwuM6aK+pKrEiEvuu+SJvVkVfwWVSb0bwVokTG3VkXl3mIcw4wxdqno1CDWJejPa\\\\\\\/JuVvI2VTLutPdhdelDHzWDyTvru1qoG3lrTRC\\\\\\\/lS0NuYOGCT34rGKPLRPSpRN8rMwehycSFPwnUATRhNS3Q0E5dZT1iSyR8cVlzkzKfXfflkWni0Z8SsIGflq9ehAtdafCA5fIc79CWiPaiZlST1oKyi+5rH7EmamNB7I7z\\\\\\\/pzHXRMz\\\\\\\/VUY57F\\\\\\\/6ghyIT48+gE38sKHGL5U8+WC\\\\\\\/lN6SkkbAvWVtwdIPJkGxj8Nv3w19o\\\\\\\/oT7mByk6ir1+TpdcU9EcsKSatO0AWYTrBTqQd0qsvshk9eAteNtFvUVGQgxxglNWz+fO1ap9ZpiwtJs1796nqVMV7ffC4fNTWj+GQutlmOr5\\\\\\\/mLAAhsbUtF77w3B9LiT04JnlaFHf1BrNB8dgHKdrwlgi+IMMOyDcmWUD8Nnw8b2ARyyX25gvBz0Wuk0uKMCjZUZNfiMARg4U3tLUqzo0NNx934uthq4QVOxhqpldBX\\\\\\\/Z9wqcv1OHnzRfMp1ZYxpbotV57sUW0W39BxOGFkQU6Zalm14HaK5sPdz0rESP9Gc1EXIyEXPskhIKaXzL8maUH8mpBlG6CUTKfvUa4pliky2We0WJXW5OzpAgOlBbaJkBA8p\\\\\\\/+BRtCqbFiRhKosGUyINYporhQZZ4HwPMU7Jx6YYntJBSgs4Ii1bjnrCSsC9lZrf61uC3eQ3PDHdDNtDkmyo0tRYQdDIW9Vylpz6stdZAKDZqC+lhh64W\\\\\\\/nAtwSOw5k2v9crJ0R\\\\\\\/ojW7LQbRhFuw\\\\\\\/2RjikNfn2NFo1nplf\\\\\\\/ocscp0n30eqLtIB5gDgS359VBwdNikDHRuNY5BA1SJQ+sRziBzstyEnBaigTkJTv+8t72\\\\\\\/Kfa3kynKv+qHXJJv1Y3KgMlDVPeJNzdYJhywVTuJdl+gfKx3L0dttS25qtROgwV+MsBJtDZ+72X3\\\\\\\/ZPS1A+yYSNy9fPuSCJwMtz8EedE+0q\\\\\\\/Dpu0Tk8Pwb3CNkMwVgXM3aJlwbNh2\\\\\\\/8hrmSQhj6h51vy6gNaSXxyB8OcCjlBzacdgi\\\\\\\/dKIGWJIVZxHuzJMSRYtvDntVRDKvHbSfgtFRnJT5IcmDr\\\\\\\/Cj3Kg+uT2MEiaRD2WYe+iKFrmxd2WlYpy+WWHkov\\\\\\\/zvf0g\\\\\\\/JUC9GJcIl\\\\\\\/1MT7HzafM0QwPeWpQDwgMOsYlN6ejUJebEVa\\\\\\\/ksY+qa1hjPhn9Sw59tgV8dbTjh2efumfy94W\\\\\\\/pwUqwEmrQq9pDY7KAiDGby8CAs+1pKF+fyvgnPf1cVUD9TI+YAZ5Pr0T6AYi4J\\\\\\\/8UNIFkDXxPRjnqt6DeZoJeKTP+wc+2ae4NOLFSNjpXH0kgKmPMEIqTRCYjTFM44NOWU1UVySvyBX4hkgG\\\\\\\/oMc5ZyrfqoLfMPkwNSxo3jXdLz1ZhIfys6E+eOyx35RLq5nVxpWhDUik2UenWQfyqxQRAt6KyMMdQVwY6wRpf\\\\\\\/jqNWK9w8xEXlm1MKbEyduxt77ZpLADRBBx9QJtPFh+mWIXtEO3mitgUCbv10x+hzITtqP\\\\\\\/CHtdvcdGhNpT+W+wiKSgEkaLWaKslfJleMM7uPGm5A\\\\\\\/9iYP05NEunhkxIBWO2oLjL4cuIjtBAtnH7clNaplSJAoxbVEY1B8r7Ou8pL8pe8qIzgpRYetb\\\\\\\/\\\\\\\/7VLHHPmKWnz45RMa7Ll4Mg1mX5pleL4w9zf233YBGWRhe\\\\\\\/6xfyzwWGUXCRX5JSHDhDzmAYPkksfxKlJewqjpRrv0eDNDqhDKpyTWjwXTrxY61bI+QFKLsSJS30SI4dNiRK9Nml9nJ5n+oFHdhpA6aIC8v2S8nu4nAYXAc0T\\\\\\\/HHAqm0\\\\\\\/hpFK7I+YuSQ5wGeT378G2jgt4RcDXLlO6GOB8N++qxE+9JAlsoSXmuSSspPkiaVyak0h6UiF\\\\\\\/AXv5M0PHdKqRE22EKhdQ1HuFCfRXwcohlrhjKbn035PslwDkmlPh\\\\\\\/VrZbIO9mx9JbwbC238T7ZKyKaMU45XHVKsUrt4F1TFEDQAnMNwTrOuhYZXveoDIB3kINP8CuYgnE0\\\\\\\/Qspa+3e5IMd2jx7NvEk0arjMhGc1vqozzsgsajTSbUzofJM1DY+AyFRF9JVD0rQpw2XfBBBFqvHvlNH3gEGxyUyq3zqjEfpewuxC0hMz\\\\\\\/jzQ\\\\\\\/2l3yOqtnSsKwOCx+VpHu5y7qXmyYuhDwMoVfGaQTFBds1mO0KRWIQmaTMDc3+bozos+Tz1LUbxvVsTTM5SbGqTgngGcigCtfKBfBO+0YOw94Z69Lv04nqGhLYbNi7o7vaggeM2b+G9CKLojBLhJHO44Lb16JZDHLodRyG12CawFKHnSypmrdYxw7IBnUlv3wv0xsBnsl12u7SNUc6zstoJrNc15SjGpAVrrCiM9dDdOzeCCZIw3UspK6WvW+JC240IX8rzUQ7jkUAZCSmsJHvZG4pxLL4QHdFshohAcGG4Vqlm1lFjUxuAa2kp2MlAFOKnLwcHVocY4HQiJOEjMECI5qtxGztpDSW5\\\\\\\/YyHU4xbB31jKAnJ78zK+5oo994pRYQxWKvtMUPjJmT0iq3aDVBCY9TBKWVzLKYRBCYNUEQVvl5vc627HJBDCGaunUsm1lkCIGBVQZiEO9hGNDs9deKVq6kSOSZtLEV6m2Z+ifQ2piXzJaEmqbS7I4TyqG\\\\\\\/yxbry5KmPIKnx9G0Rk44G5IOCY6ai3HdhPeXthNb\\\\\\\/TT979PmvUEyoYr1D56iIwK6oK9ThtzT28u7B0OVfYUUZwQKdWA4OOFBMZ9x4jLXRaxDZAeh\\\\\\\/waP3LvH1valOHxNyIZIDuQqp7zFMEs5dtGXke\\\\\\\/vprzcZXFXU7mJu03as73P+5\\\\\\\/2zB4uxtWkYj928DB3drNIjYuOzkJ3SttIiTTQAkmxyJcSnDMx9vknkTXeVsuxPahL5Hfg5g3LKZmi1kjzrvwpxfkkCDDR4SDtj+lsTpmRYgFfVST4Z+ARHIQQxQ7w7132FbgYfxIlq\\\\\\\/XxqDIctWMhTELiWGdFJbC5ueytjDQC99\\\\\\\/xa5P2HRIypEr21JU8vskaPICNkPEe6QhaYeCeQYUbtm2VCYhuoCO6vmNy83Hs4pd3lF\\\\\\\/F7R\\\\\\\/2SIlIhsF77NHZ3fX+oHzZ5WotWF7NDwQ0AsQhVE0760zbyPxs2CxUvWdx1DCZHflA2uM4zqzMkbpnd6JFZ6S7FvGanfw0xNbMtmZwWVVEi4TcvkvPMNnM+PCWcP8xoWEIr7fEL2t5AM8KrAeOjZGK9tROFd4\\\\\\\/A87nsCNYz9evGNiBfCR74Aa7nQdq00257vGPXIO5CCZ9phpUv7Y0l1q1RaWfE1iusLo9nzwMNbIF3qPUd7fcGkU4IRxOzPtXGiWfzKtqZqbAp9384mocqMTzQOl2Xa6R4lHOWcLgXigCuVoGiIluFt3vk9dLGIDwsBB7QgUr5Ht48+ibeA2M1ymzxMRfF6M2C6KWRAwDTzLfjnTFmkJpxxLLKD26JWYtTf5ngd4wucWD6A8jPOPA4zgXiGyZu9+O0OT\\\\\\\/BHKXkMqEbM6JV+7nS00jfAYA2NmSlK7riDG6P0cXJ36yPqVkrQXLTaEEJUwdMElAR+yMJRhrkPOt0fmzdVHoykexRROFCBwIPMLSvLkGBVzZ\\\\\\\/Hp4HvQ\\\\\\\/z8hkQxy1\\\\\\\/1wtRDYmg4QRTs4hxUuDbl4UEXCbySeVgJmSvQ3u7gzmNKPG3RKlijxjeo4wrwbc3TeKOrMfjOOt7OhQwUvUb4k\\\\\\\/TJcAWBDT5N15rBO+P3AYVLtobOzUF0VKqG1OeZzZ1fdzHr35cIG2NHOHPPJCHZJaisq9WwO2CUvYKU7LK00WNYibNRbrgX8JHfBEXzrQ7HZrNoOBotq1HVHwusfc02vGKb4Kj+6gS+DUmScoV0exapZFxMYTD+CTar+oeiX64GD45fk9IJnw6BT\\\\\\\/cb0RnBsuVqHeFCF6sXFDPMcqBXbIqh5mKxdkWc2AOiYmMTZfrHS00sYYLoGuXhFqDbo1\\\\\\\/Evi9hC39CXIkfslV3NF1+nsIsnBrLReZeT6eOC7Fc5o\\\\\\\/CIr2sPFJG4Yv7O\\\\\\\/MBXMTT\\\\\\\/5pX+Lc0t9g4IzZdCzxuzfAufJCdhBx2LLwlCqrDLTNKRH\\\\\\\/oAVVTFwAkdhI\\\\\\\/AKXMWaU+GIxkot1V6k9LbpXmm2oge5FuVTNe+0YpIL+KCbxhU\\\\\\\/WpN6eZ4BbpdKrCxjMZneN\\\\\\\/k9iNLe9Gs3VRryNMgg+XQajDIJUkuRewbnqRB9L+WmBdWD5XiQMYB0lwmGioHYts0exT0oAXDqYebVCZ5+\\\\\\\/+q4JcYqmbKZUYnGoxK3zwkxxDOsZ4OQQfzpogXCs+sHtCd6nMVUYccHuiA3kIYd3RvfQ0yz5tgmq\\\\\\\/BtaBOXe0wPllnTdMVaSw2ZWBLKb1wwJ8MRO0XZEEcXNvG6IhQW2ZDRyrXUjcl1xPK4I9jTRXHMeMdIaLZqliTLKtGPLB6ubtf3zXYKDCBJgjmdcBKvA1MkAlgmnWqRQgeQNtVpk4OkcX9yQUX7IijUqy+tBP8CQDg5YN6Nk+wTYaYUPcrWjS6thxvGLcF7aSwWGoG1rBtK1\\\\\\\/mBYG1ibKUCUT+t0xn8vkKSi+VZ6XVxnCAttyYNopuIS9ylempHiFPX4Yl05282y0Gv+dG4XfpIFwRE08vZZLII3AhLYJ+oIXWlfL8JE1wq+9GuUzntVwPUoy\\\\\\\/GehOB6YE9ndOKSKa8Kgelzz68K8glc3hVyn2naJ4wZY3cgL98tbR+1BVwuqOi39fSUuMDuE2s0sKwz330kM5QtZq+8JmCadMumTUkLeqpMP30gC0i6NNN24m7viw4LCjtLQLaEiWhpQqRwV97oId+OCJpiQVoa7NYwjGctpdIvcr4TWYZVmj2tqaaNamrv\\\\\\\/gdBVkG99MwBrp7D9stHmZ8cCadIBmHyHWgSe7jzq1sjr2FfGEXpwGlpb8815n9C9hzH64mhbDDr7jaU2KJkfDxlDVVyGuzUskFRuUuQ2UKkOY1nZVSyjndVnrfwYb2X\\\\\\\/9oDAX3P431Qk156fRtbaeGJe7YpHK\\\\\\\/dR1vCDl59EkWIdvUvHdYkT\\\\\\\/AFjd6WmC9FO\\\\\\\/u4JzzT\\\\\\\/5Tz\\\\\\\/6o0zRsQ5aQT2tNFX9SZBlcdergeVH0x1peo0V56CcojGpXdkvVL70p\\\\\\\/wdsrABb5rOhfD+dmeeOFOsOzRtdfUjYwLgRgcQXTOMIl\\\\\\\/3RJJwRJHpfRIWkJJvFEqrNe60a1zdJuWiDxI1SthfkJBIXRzJjLXPEcUVXqSY0jKL\\\\\\\/ch7Ao2JhA8AjgkrHq4zYIxP2P1KmGNRib3oMWaurSqGya77UdQDji4RRq8o1EhqU4vICjMU\\\\\\\/Gi18+wSHKsymmIubIG\\\\\\\/FacHoZQGkKmyaf8uurPziHOhkvew49K91MP6LjYCKCd5\\\\\\\/sh7zfW00aSMmUT4IDKL2HuqEQqdB5M4hS+JzasoIEQyygbhbCQMPHMX33EJtHxuEpVQ5dleTv8XMbNpRzSof23z8kmSar0WBnSsb1NuQiMDtg+U3f2nXZTq6Iw0zPl5sHjQ+U\\\\\\\/hRsGSzi8F1xCAXTo23IM+oGF4cwBPn8tXHZnB+VC2VaH3BB1TtxyDuDnyzRq1quoW+JWPt3VCcdldEoEJJHerB2fAzAKE5ykxVrb3or4NOpcVkaRM576M7mjm4fVuEjGVR86uLbfO8v1467tdgBEQiwfFv8EuAyhqzqVUCkVtlV51CcwRSjzZvgmB+cUBdwMC9lT+QI48yNqjstfLUS8YKKY34U6CeFVwcFGmeTwe3WyAgflS46GGE89CJxjuua5OvZ7+GBdWWpsK41590LAulddqawdYv4wQNnQSRVGYGeNki\\\\\\\/Y8gJnDNXZRGacu8b7sHoKQmRnCIaU+KyAdlZlVXpBu98\\\\\\\/Qp2igT\\\\\\\/JFvxRolGTFhkE1iXYish1xjwpEHVHwgGRnuoDUIq7QAKISaeQ26tLBVBjT+Jm2+D6o4rVS5GTqYcTJePE+JXyPafqfCez96TCLIAanSI2d9dLxKhgoerRxzErw3ObtZOxWCW57NHqKCchFI1Lgtkb59fu+HFYhIwh9Rp3g7H56cN58TMrP50gJTBexYnd\\\\\\\/UIFCdEuFpKVW4Xuw\\\\\\\/L+VqZ8OLQqIcDpF+vS+TbMXoFskDWnnCjDDKc+bXPSQaRi4Monrzh3Ihv\\\\\\\/MO1CWEJ+ArvbIP88+ciSNQ+zryyYouk+NqTZvf4ybKV+hgxqRi2rcadkdUD2fAEgLZfZLKCmK0pwoWMGBehTCx8OFFVzAiyAPPLGcmNVb0oSGy6pIlluR78w33eLSdzEy+CdL3zWJwQqfNp\\\\\\\/E\\\\\\\/RT1ml2INboRE4yuWY+nUqY0PqEGn5yiJCqhZTnTi\\\\\\\/AfoFfS\\\\\\\/MFcrLF9fx\\\\\\\/Ea\\\\\\\/+X3J4hylf784iRZQpBCWvLnqjBC1+B17Segm7b61M1SIkvqyihY6nWxZ8tAvN0ltEXAmx+Cph\\\\\\\/MxdF7h88JW3uZ91xCRlNo90SkRpihp4BZsqx+W8uQ6AREsEfwOfBAod6IR3tK2gkPhLJuqsnVxXpsismo3lWEH5LGGBIvqFDVuMJnw+vMsFIgqEKt5hzdecsQAxMgPFDn\\\\\\\/czD7sN8i6z7aiXn8r6y+8t4zeEIo8YAftWjZQMR4e3qESmPTVUJdog5K8RBNY7rw97AWpZ4KNfx1Jl9Zqmgzy39FC+xseecg2ZF6hwTX1TUIb2S1e0GxqwNZWxa2Z2sXw7XL6xf0RxGQiuxIWTa02vE\\\\\\\/GaX9OsH\\\\\\\/9ZfjpiNgFzRgfwPoxwmDNKXlGzaD+spwa6hQrKGb+LWxRvDf2OPS33iBMg\\\\\\\/s+fTj0m8MC+WiXwJtbdU8G0NP7F87KA\\\\\\\/uMsz4+2DOa9oPnQ1TefCoQ6e3apFmPBtEWkTaBSSE8IEgtbupwWiiTufIgAmexQCVTpLqpyzuHKAokuyT9V3XliijmnZMm7bCfvLNMWMQcjZe8QagGSVwOI6nm9MHIgRS0eCIBvo9Mh8LYZrx9lHoTlT05jwYnRK9aAxPBLMyi5MKkEL30RDI7mZftP3euAMtDRgKlF0NbbAiEnAYXmyLwI0VNaXmVZ\\\\\\\/wfNRrhWBaT+FvyrzUxWhjnyFDYwkafoqbvx120yq1O2MPx\\\\\\\/jxLXAMwAefxc59oqWNtmGIy1gu51ewBb3tu++REC7JcCZKU5Euyx++Qtl3KklwMV9L+F2fPpOPWzimfDAaUvn117w46ODiiynq2NEmtYF0yMckyHDaxYoZWVr5Fs7lJr8gFmu63XUEUzycJABpwdkvPRciEaS4uKNPvchTaAhvo6ISdVy1yiOFbHH3W58qNYraaE7OPt8T2qKy4eF1KPlCfp+FjI7q7Uuq7ZjsHROfVoFS7KDkViupNpPEYgusJImk+c3mxME2puAasMwAHwAaqB7N6U+fSSKjcHNA7gVLZJz+hgmUZw0D8iAKG+75SmzK6zU7YCEsHHRl79KfRlx6nJ4SX\\\\\\\/hyJOacWdi7anluYMabVw45tpY2NhRJNrqLkFg+yINBu5O\\\\\\\/jTh5hI5A5Nmdu5yDMAFQo8\\\\\\\/ehUJxyVYvosgRqAC1hnraTfpOuxJvGaYBDysNvjJU6wkvD0fK75Ad9PEGLd0XLxiBAbL3mDLuVQZfzmAL6d3gD4Q5zZzWtomrcPB9MSpNgH+sfcp0jGnBU7XAfRam1majPgdorbgx30x59OpPX4LfWbqC6p0\\\\\\\/+IA1nzPsOsCoiPmcpxBLx\\\\\\\/q+uP5N9ZQt\\\\\\\/s4MPv2PgdQmqhG8m27\\\\\\\/FCp5vVc5OxujrP\\\\\\\/D9CI4oL+on48L\\\\\\\/l2DqtjMerRHU0Y2N6d6ZckCmNIqYD6a4QBjU30kYCZQZN5a\\\\\\\/6pTZ+kVaeRIdg72yv231T\\\\\\\/0pZ1D7Hu8qITp3eBwTL+nQ+sUL703l60V3B2gMCiJTo5kfiu3zI+se9asEmmoNzYm23MoAbBgzzxCBFxIb\\\\\\\/DOn9YUNdi0H1UGkBDqmEaZ7B0byLlAzmao5N17R+\\\\\\\/9UAoeLn6Lb\\\\\\\/KY3YHDWLcD4vb\\\\\\\/JQBb5SKDSur\\\\\\\/GXKrcOAeTZ+DQQ2H8U\\\\\\\/6atJvBdB\\\\\\\/KzZh042+nDUQXXwVa6n9BofFGL3U7wcxcQh49ngCtJ6ZuezoNVO5G+kjSUoHp5lyWIZonmMiTjSp9y1GtYCw30ryFsJo8Q3Bei2eLN3GLFGK5aRxRF0VtphNYmYRJ8xEIAlfSKOUmp4CJQV\\\\\\\/TIPp8QKLxalM3yLdMXNiLm4PegH8ouWkw90TVepsk6lv0SPvVDfiS1tTeS6GwJ+sQ0GXOw0PijU0nRPa30c51qRH86Wup8bV+nIEyrYfzS6C0xfGwh6Qf3EaqvlGwZIS5cQYwc1RiXa0eYMybpuWkvlczp5MW0bHIhx0SgyOyksgNoZIrvRUr\\\\\\\/+mVOBgjdqCFM3GOLAytvxoSSp2338lPHl+sBUGMQytO+Xq53L8\\\\\\\/ptX80fdy9+93yCldBFa7x8w57iNpCW829OAQrtCoZLeuoKRBaW012Txlw\\\\\\\/nJ9P4i+jeA36ElYKG4lfKAZrFP\\\u0022,\\\u0022iv\\\u0022:\\\u0022EAN6cazBzfsO4xmy+Ma0kg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227988f2a99cff8bd9f437afdc80acfff1ad8442a39b9152455a5110b31d25f71e\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022XjB7qiap9tccYaUeyF1L1I3dHiUjMQaUQ4TjzYoMgCq6ptv1tL05OCC8Pd1o6\\\\\\\/pkCBryQUWVOVr2DoVMp9v68B+VdolBnFYU9TFx\\\\\\\/XVn\\\\\\\/PIk7ge\\\\\\\/PZotyCxEPPTr887WqyMNXhQzSjaG8dy9uCM2ErJD3trZMVnU8petQ77BGn9kVxo\\\\\\\/ixj38TgLOVMe7pi7mlNEMVa16OI9\\\\\\\/wedWREaye8fHxgWkeVucmVZiFLRSlB5MK1GC5UROs7HfYth+hXSY6NLOnno7JT1IWSqqWQjzggUY93pT8qls44EWB+Y1gE76jqxxwHMTqBPZaAgmV\\\\\\\/aiMbucF4PepQAOsacovIPWdnG7jeQ8qmuzEHuKiXC5qgUmiZ60e7CmY74GwH3AGmMbM9hkDrJRLMh3LnbiMm0ytm24DvWE0aOVSU7mLBQme8mva9vlJuu7QdmLLLL534E9oEo1abhvTqv\\\\\\\/2XjK2SXQB\\\\\\\/9LPTs53Bg3iBn6Tu6kI6RHQO\\\\\\\/OkbAMDspvIQx424Q2WdNSIH6PFQlb7PVGH+pphrXpVUqk2zHSDLM+3N2UjvxkbYQUfUJlp9InnTii4ZC9\\\\\\\/or7ql4qpZy9jL+\\\\\\\/FSXTIeRi9lLwp7ecWDBm\\\\\\\/NuFBdYdUISBe6ZHfPBUkldd47VTdzXJxPUyFLHXnwezl7sggadLdODOPH0JkfopTaZaRJBihhOBRKExcZEJIWj5I5jg\\\\\\\/RIC9aOntbqULVD9UA17XQvGIKHzhAGKvNYBxKtXzuLZcRt\\\\\\\/3GaPN+Myx2RYdbi0bUfZDorkmwHJQiCiF2WfayofgTbRcNX2idpwWqWs44pxxjLSJ9NJb\\\\\\\/6y+\\\\\\\/rejd5aJnhKapRoiWEi0WuPxk2LlWsHcKrLPNXesMKvhad7cmLi\\\\\\\/VNnOU2WYnBfwTsjit5cAqvsuIkjUKVf29uMExYbiSsYdNoYMjrlY1PjGKvt7I9emkhLaBSvAetqVwTGWQa0ucICSPPxKVv6BPtPPpwx47dzc1Y18p6bLxWmhWmr93mi1369qyrHzLUy8zU36QgiC2DPoMnA6XbOBy2ktddBlzfR+ja5ctEynGMhskESBAYBeT09fs9GBDpmjMxkxjiMrdKpjKBqB97pBMEIr2mb3HI4WavSKqk17fKPFZnQgWS2s\\\\\\\/\\\\\\\/HM0R2xlbZP9OdmG3brKqZFoo68HoSP7\\\\\\\/ezN2K9AW2dpEZdQNzpvsVthlLfR7JoLIJOk5DfVWunrRkNn4m\\\\\\\/JQoSqRFHF7WbJaZaHUPdC728HEYJrDbAaF76hjV41tCZsvXzF\\\\\\\/tohqYkw1FCDM7TuVfuIgo5hsh+p541wpHSGEdhfodEq8SqoSPMn3k3cIVl6pb88JqD9lrU08rwYL\\\\\\\/7irRk969aHi01uiqcLxioLppaliKaEiV4Cs92cz\\\\\\\/\\\\\\\/L3yv8wEu+iQN7D4Z+XVWWxNggiT0cTSTIDbtsilMBA5BjTz9zgAnrrx3adC6wNuTA1EcQztLAaY5JbE9fK\\\\\\\/K4IwXdUsXUNnt+YnWCGrgV0412RLtyjwMuoNyWfuzLtngGDfValHXVjazXDfKImealJkjZmiAcubzzHQUNjhvNdCdS2j9LR4ZjG5VY0WAvhD8k\\\\\\\/91\\\\\\\/tYG+HcR38bl65pvFNSELJt2ONpUFvmem8DZE\\\\\\\/nfimEl0+Umr3gGzza6DVwE5ESMNFXEYF9aiXbQMuZB1Me1Gs1xO3ZMM7ERvGpet1SOG8tOcK4hq4klkA44q46ueCgiNXHchpUuawpmIfuWx1xR9G0lnUxfKswCXmjRYWSc4u+dqbR250hG898uCaJHJ3VQTDwA3pRIfxuCJI6w\\\\\\\/e+GITCqomtt\\\\\\\/JlOwWN6K2\\\\\\\/kcp3DSkmroFSvNcF50Ky486AiR646TpR3GbdPvPe2AJNyLekQqqsIe1j4H\\\\\\\/ZdiKDGON57WZ399g+tyhFM0AioG\\\\\\\/Ty7NNVUO1eOTIZxB+IOD32rzSApcCLwvKxDBz\\\\\\\/X4NhBHmEQ2V6UmMd\\\\\\\/L2RvLzumG\\\\\\\/gfgRtqiy9nNQ7xvS\\\\\\\/3PORaDMB4USccN1\\\\\\\/PWZ2p7Q8nrj8v+Xjm3Dngl1qZo67eYVEQ1Ci9IMXfdZckNWhSI\\\u0022,\\\u0022iv\\\u0022:\\\u002290IYqjWu6UnNUiB7PG\\\\\\\/FKA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002284f3f09b1fc9746ba3c902fc58182270d8b54bd081c2b02ccf7617dcbbf5c80a\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495560 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/783" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-eaDCCy5VLWMd7uwnH-7AQdBNjAsRgFROfsRSn6oFSTc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/783" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Ca0wCn2Z41RS6-iFajIGlYftQBjMZJesylKAFB14kSk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/2/769" title="Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/2/797" title="Interaction Between Induced and Natural Variation at oil yellow1 Delays Reproductive Maturity in Maize" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495560" id="node495560" data-pisa="ggg;10/2/783" data-pisa-master="ggg;g3.119.400663" data-apath="/ggg/10/2/783.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Impact of Non-additive Effects on the Genetic Correlation Between Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-8403-968X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pascal Duenk</span>, <span class="highwire-citation-author" data-delta="1">Piter Bijma</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-3213-704X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mario P. L. Calus</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-0681-2902" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yvonne C. J. Wientjes</span> and <span class="highwire-citation-author" data-delta="4">Julius H. J. van der Werf</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">783-795;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400663 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495560" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Pascal Duenk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Animal Breeding and Genomics, Wageningen University and Research, P.O. Box 338, 6700 AH Wageningen, the Netherlands and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pascal%2BDuenk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Duenk%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APascal%2BDuenk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8403-968X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pascal Duenk</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:pascal.duenk@wur.nl" class="" data-icon-position="" data-hide-link-title="0">pascal.duenk@wur.nl</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Piter Bijma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Animal Breeding and Genomics, Wageningen University and Research, P.O. Box 338, 6700 AH Wageningen, the Netherlands and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Piter%2BBijma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bijma%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APiter%2BBijma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mario P. L. Calus </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Animal Breeding and Genomics, Wageningen University and Research, P.O. Box 338, 6700 AH Wageningen, the Netherlands and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mario%2BP.%2BL.%2BCalus%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Calus%20MP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMario%2BP.%2BL.%2BCalus%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3213-704X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mario P. L. Calus</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yvonne C. J. Wientjes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Animal Breeding and Genomics, Wageningen University and Research, P.O. Box 338, 6700 AH Wageningen, the Netherlands and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yvonne%2BC.%2BJ.%2BWientjes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wientjes%20YC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYvonne%2BC.%2BJ.%2BWientjes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0681-2902" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yvonne C. J. Wientjes</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Julius H. J. van der Werf </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Environmental and Rural Science, University of New England, Armidale 2351 NSW, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julius%2BH.%2BJ.%2Bvan%2Bder%2BWerf%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20der%20Werf%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJulius%2BH.%2BJ.%2Bvan%2Bder%2BWerf%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/2/783" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495560" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495560/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/2/783.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495560" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495560/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/2/783.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495560" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495560/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/2/783.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495560" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495560/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Average effects of alleles can show considerable differences between populations. The magnitude of these differences can be measured by the additive genetic correlation between populations (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-1.gif"/></noscript></span></span>). This <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-2.gif"/></noscript></span></span> can be lower than one due to the presence of non-additive genetic effects together with differences in allele frequencies between populations. However, the relationship between the nature of non-additive effects, differences in allele frequencies, and the value of <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-3.gif"/></noscript></span></span> remains unclear, and was therefore the focus of this study. We simulated genotype data of two populations that have diverged under drift only, or under drift and selection, and we simulated traits where the genetic model and magnitude of non-additive effects were varied. Results showed that larger differences in allele frequencies and larger non-additive effects resulted in lower values of <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-4.gif"/></noscript></span></span>. In addition, we found that with epistasis, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-5.gif"/></noscript></span></span> decreases with an increase of the number of interactions per locus. For both dominance and epistasis, we found that, when non-additive effects became extremely large, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-6.gif"/></noscript></span></span> had a lower bound that was determined by the type of inter-allelic interaction, and the difference in allele frequencies between populations. Given that dominance variance is usually small, our results show that it is unlikely that true <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-7.gif"/></noscript></span></span> values lower than 0.80 are due to dominance effects alone. With realistic levels of epistasis, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-8.gif"/></noscript></span></span> dropped as low as 0.45. These results may contribute to the understanding of differences in genetic expression of complex traits between populations, and may help in explaining the inefficiency of genomic trait prediction across populations.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/dominance" class="hw-term hw-article-keyword hw-article-keyword-dominance" rel="nofollow">dominance</a></li><li class="kwd"><a href="/keyword/epistasis" class="hw-term hw-article-keyword hw-article-keyword-epistasis" rel="nofollow">epistasis</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">genomic prediction</a></li><li class="kwd"><a href="/keyword/average-effects" class="hw-term hw-article-keyword hw-article-keyword-average-effects" rel="nofollow">average effects</a></li><li class="kwd"><a href="/keyword/population-divergence" class="hw-term hw-article-keyword hw-article-keyword-population-divergence" rel="nofollow">population divergence</a></li></ul><p id="p-3">Populations can differ considerably in the average effects of loci (<em>i.e.</em>, α, the difference between average effects of the two alleles, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Falconer and Mackay (1996)</a>). For a given genotype (<em>i.e.</em>, individual), differences in α between two populations lead to differences in the additive genetic values of that genotype, as expressed in both populations. The magnitude of these differences can be measured by the additive genetic correlation between populations (<span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-9.gif"/></noscript></span></span>), defined as the correlation between the additive genetic values of a genotype expressed in population 1 and population 2. In reality, a single genotype cannot belong to two populations at the same time. This means that a trait expressed in two populations can be seen as a pair of traits that cannot be measured on the same individual, analogous to <em>e.g.</em>, age at sexual maturity in males and females (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Falconer and Mackay 1996</a>). Although no phenotypic correlation exists between such pairs of traits, they can nevertheless be genetically correlated.</p><p id="p-4">The <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-10.gif"/></noscript></span></span> can be lower than one due to genotype by environment interaction (GxE) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Falconer 1952</a>), or due to non-additive genetic effects (GxG-interaction) together with differences in allele frequencies between populations (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Fisher 1918</a>). Knowledge of this correlation contributes to the understanding of the genetic architectures of polygenic traits (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">de Candia <em>et al.</em> 2013</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Brown <em>et al.</em> 2016</a>). Such understanding may lead to improved knowledge of genetics and can facilitate accurate prediction of traits, such as disease risk in humans and yield traits in crops (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Forsberg <em>et al.</em> 2017</a>). Furthermore, understanding the genetic mechanisms that determine <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-11.gif"/></noscript></span></span> may help in explaining the inefficiency of trait prediction across populations (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Wientjes <em>et al.</em> 2015</a>).</p><p id="p-5">Following <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Falconer (1952)</a>, we can interpret a metric trait expressed in two populations as two different, genetically correlated traits. The additive genetic value of individual i for the trait expressed in the population that i belongs to (say, population 1) is<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-12.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-13.gif"/></noscript></span></span> is a column vector of additive genotypes (measured as allele counts, minus the mean allele count in the population) of individual i at quantitative trait loci (QTL), and <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-14.gif"/></noscript></span></span> is a column vector of average effects at those QTL in population 1. The additive genetic value of individual i for another population (say, population 2) is<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-15.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-16.gif"/></noscript></span></span> is a column vector of average effects in population 2. Conceptually, this <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-17.gif"/></noscript></span></span> can be thought of as the additive genetic value for an individual in population 2 that has the same genotype as individual i. Here we define the additive genetic correlation between population 1 and population 2 (<span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-18.gif"/></noscript></span></span>) as the correlation between both additive genetic values for the individuals in population 1,<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-19.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>In other words, the <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-20.gif"/></noscript></span></span> is defined for individuals coming from population 1, which may be different from the <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-21.gif"/></noscript></span></span> defined for individuals coming from population 2 (See Discussion).</p><p id="p-6">Equation (1) illustrates that the value of <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-22.gif"/></noscript></span></span> depends on the differences in average effects between populations. With non-additive effects, average effects depend on the allele frequencies in the population, and, therefore, larger differences in allele frequencies between populations are expected to result in lower values of <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-23.gif"/></noscript></span></span>.</p><p id="p-7">Note that <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-24.gif"/></noscript></span></span> is the correlation between the additive genetic values, not the genotypic values (<em>i.e.</em>, additive plus non-additive genetic values). In the absence of GxE-interaction, the genotypic correlation between both populations is equal to one irrespective of the presence of GxG-interactions, because the genotypic value of a genotype (<em>i.e.</em>, individual) is the same in both populations. The additive genetic correlation <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-25.gif"/></noscript></span></span> may, however, be smaller than one because the partitioning of genotypic values into additive genetic values, dominance deviations and epistatic deviations depends on the allele frequencies (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Fisher 1918</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cockerham 1954</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kempthorne 1954</a>).</p><p id="p-8">A deeper understanding of the relationship between non-additive genetic effects, allele frequencies and <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-26.gif"/></noscript></span></span> may help geneticists to predict the value of <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-27.gif"/></noscript></span></span> based on the importance of dominance and epistasis in the expression of the trait, and the genetic distance between populations. <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Wei <em>et al.</em> (1991)</a> studied the impact of dominance on the additive genetic correlation between a purebred and crossbred population, known as <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-28.gif"/></noscript></span></span>. Using a two-locus model, they showed that <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-29.gif"/></noscript></span></span> indeed depends on both the magnitude of the dominance effect (d), and on the difference in allele frequencies between the populations. We are not aware of any theoretical studies that investigated the relationship between the importance of dominance and <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-30.gif"/></noscript></span></span> between two purebred populations.</p><p id="p-9">With epistasis, <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-31.gif"/></noscript></span></span> is also expected to depend on the magnitude of epistatic effects and on the difference in allele frequencies between populations. Epistasis in the functional (<em>i.e.</em>, biological) sense means that the genotypic values of individuals depend on interactions between alleles or genotypes at different loci (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bateson and Mendel 1909</a>), and there is substantial evidence for the existence of functional epistasis across species (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Carlborg <em>et al.</em> 2003</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Le Rouzic <em>et al.</em> 2008</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Pettersson <em>et al.</em> 2011</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Mackay 2015</a>). Epistasis in the statistical sense is measured as the deviation of multi-locus genotypic values from the sum of the marginal effects (<em>i.e.</em>, average and dominance effects) of the individual loci (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Fisher 1918</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Cockerham 1954</a>). Although functional epistatic interactions do not necessarily lead to substantial statistical epistasis (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cheverud and Routman 1995</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hill <em>et al.</em> 2008</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Mäki-Tanila and Hill 2014</a>), epistasis can contribute significantly to the additive genetic variance because average effects of individual loci may capture a substantial part of the functional epistasis (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Hill <em>et al.</em> 2008</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Mäki-Tanila and Hill 2014</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Monnahan and Kelly 2015</a>). Furthermore, epistatic variance may be ‘converted’ into additive genetic variance due to genetic drift or due to selection (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cheverud and Routman 1996</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hill 2017</a>). Thus, epistatic interactions modify average effects of individual loci when allele frequencies change, and may therefore play an important role in the value of <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-32.gif"/></noscript></span></span> and its change over time.</p><p id="p-10">In summary, the <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-33.gif"/></noscript></span></span> between populations is affected by non-additive effects in combination with differences in allele frequencies between populations. For populations in the same environment (<em>i.e.</em>, in the absence of GxE), <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-34.gif"/></noscript></span></span> is equal to 1 in the absence of non-additive effects or in the absence of allele frequency differences. So far, the relationship between the nature and magnitude of non-additive effects, differences in allele frequencies, and the value of <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-35.gif"/></noscript></span></span> remains unclear. Our objective was therefore to investigate the impact of non-additive effects on <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-36.gif"/></noscript></span></span> for populations that have diverged either under drift only, or under both drift and selection.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><p id="p-11">We aimed to investigate the relationship between non-additive effects and the additive genetic correlation between populations (<span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-37.gif"/></noscript></span></span>) with small effective size, as observed in livestock. For this purpose, we simulated genotypes of quantitative trait loci (QTL) for two populations that have diverged for a number of generations under either pure drift, or under drift and selection. The populations were assumed to be kept in the same environment, so there was no GxE. We simulated traits following several scenarios that differed in the type (<em>i.e.</em>, genetic model) and the magnitude of non-additive effects (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495628/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495628/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Overview of scenarios with the genetic architecture of the trait, and their parameters for distributions of sampled dominance coefficients and epistatic coefficients</span><div class="sb-div caption-clear"></div></div></div><p id="p-12">We considered six genetic models; a basic model with additive effects only (A), which served as a basis for comparison, and five alternative models with non-additive effects: one with only dominance effects (D), and four with only epistatic effects. With epistasis, we simulated interactions between pairs of loci that followed one of the configurations presented in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. We chose these genetic models so that there were scenarios with only dominance variance (D), scenarios with only additive by additive epistatic variance (E<sub>AA</sub> and E<sub>M</sub>), and scenarios with all types of non-additive variance (E<sub>C</sub> and E<sub>DD</sub>). For each genetic model, we considered three magnitudes of non-additive effects, labeled as small, intermediate, and large.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/783/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Epistatic contrasts for four biological epistatic configurations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1925584831" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Epistatic contrasts for four biological epistatic configurations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/783/F1.medium.gif" width="440" height="130"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/2/783/F1.medium.gif" width="440" height="130"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/783/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/783/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495516" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-13" class="first-child">Epistatic contrasts for four biological epistatic configurations.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-2" class="subsection"><h3>Simulation</h3><p id="p-14">We simulated genotypes of two livestock populations (1 and 2) that diverged for 50 generations (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). For divergence, we considered two situations: one where the populations diverged due to drift only, and one where the populations diverged also due to selection in population 1 and drift in population 2.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/783/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of the simulated population structure." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1925584831" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Overview of the simulated population structure.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/783/F2.medium.gif" width="440" height="434"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/2/783/F2.medium.gif" width="440" height="434"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/783/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/783/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495745" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-15" class="first-child">Overview of the simulated population structure.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-3" class="subsection"><h4>Populations:</h4><p id="p-16">We simulated a historical population with QMSim (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Sargolzaei and Schenkel 2009</a>) by randomly mating 100 males and 500 females starting in generation -3001. From generation -3000 to generation -2501, we simulated a bottleneck by gradually decreasing population size to 150 (25 males and 125 females) to create initial linkage disequilibrium (LD), and population size was gradually increased again to 600 during the next 100 generations. The population size remained constant from generation -2400 until -1 to allow for the development of mutation-drift equilibrium. To provide a sufficient number of individuals for the development of populations 1 and 2, we doubled the number of individuals in the last historical generation (generation 0) to 200 males and 1,000 females. This simulation resulted in an average effective population size (<span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-53.gif"/></noscript></span></span>) of ∼285 generation 0, calculated as the harmonic mean of <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-54.gif"/></noscript></span></span> in each preceding generation, where <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-55.gif"/></noscript></span></span> is the number of males and <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-56.gif"/></noscript></span></span> is the number of females that become parents in a generation (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Falconer and Mackay 1996</a>).</p><p id="p-17">After simulating the historical population, we simulated two current populations (1 and 2). We randomly sampled 100 males and 500 females from the last historical generation to become founders of population 1. The remaining 100 males and 500 females were the founders of population 2. We will refer to the generation of founders as generation 0. Within each population, simulation continued for 50 generations by randomly mating 100 selected males with 500 selected females. Each mating resulted in 5 offspring, resulting in a total of 2,500 offspring (exactly 1,250 males and 1,250 females) in each generation. Generations were non-overlapping, meaning that in each generation, the parents were selected from the previous generation only. In the drift scenario, animals in both populations were randomly selected to become parents of the next generation. Effective population size (<span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-57.gif"/></noscript></span></span>) in the drift scenario was ∼285 in the two populations. In the selection-drift scenario, animals in population 1 were selected based on their own phenotype (mass selection), while in population 2, selection was random. In this scenario, effective population size (<span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-58.gif"/></noscript></span></span>) in population 1 was ∼250, which was calculated as <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-59.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-60.gif"/></noscript></span></span> is the inbreeding rate estimated from the pedigree (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Falconer and Mackay 1996</a>). We simulated selection only in population 1 to reduce computation time.</p></div><div id="sec-4" class="subsection"><h4>Genome:</h4><p id="p-18">The simulated genome consisted of 10 chromosomes of 1 Morgan that each had 200 randomly positioned bi-allelic loci. In the first historical generation (generation -3001), we randomly sampled the allele frequencies of loci from a uniform distribution. Mutation rate was 2.5*10<sup>−5</sup> during the historical generations. In generation 0, the distribution of allele frequencies had evolved to a U-shape, and we randomly selected 500 segregating loci to become QTL, which resulted in low linkage disequilibrium between QTL. There was no mutation from generation 0 to 50, because the QMSim software does not allow for mutation after the last historical generation.</p></div><div id="sec-5" class="subsection"><h4>Functional genetic effects:</h4><p id="p-19">Additive effects (a) of all 500 QTL were sampled from <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-61.gif"/></noscript></span></span>. We assumed that the size of the dominance and epistatic effects were proportional to the additive effects of the QTL involved in the interaction (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wellmann and Bennewitz 2011</a>). We therefore sampled dominance coefficients (δ) for all QTL from <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-62.gif"/></noscript></span></span>, from which dominance effects (d) were computed as <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-63.gif"/></noscript></span></span>. Similarly, we sampled epistatic coefficients (γ) for all pairwise epistatic interactions from <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-64.gif"/></noscript></span></span>, from which functional epistatic effects (ϵ) were computed as <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-65.gif"/></noscript></span></span> where k and l denote the QTL involved in the interaction. Each QTL had an epistatic interaction with 5 randomly sampled other QTL, resulting in a total of 1250 pairwise interactions.</p><p id="p-20">For both dominance and epistasis, we considered 3 magnitudes of effects: small, intermediate, and large. For all magnitudes, the mean dominance coefficient (<span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-66.gif"/></noscript></span></span>) was 0.2, and the mean epistatic coefficient (<span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-67.gif"/></noscript></span></span>) was 0.0. The magnitude of dominance and epistatic effects were controlled by changing the standard deviation of dominance coefficients (<span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-68.gif"/></noscript></span></span>) or epistatic coefficients (<span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-69.gif"/></noscript></span></span>). For dominance, <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-70.gif"/></noscript></span></span> was 0.3 with small effects, 0.7 with intermediate effects, and 1.5 with large effects (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The mean and standard deviation of small dominance coefficients were chosen based on empirical results of <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bennewitz and Meuwissen (2010)</a> and <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Sun and Mumm (2016)</a>. For epistasis, <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-71.gif"/></noscript></span></span> was scaled such that the total functional epistatic variance was comparable to the total functional dominance variance in the scenario with the same magnitude. To this end, <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-72.gif"/></noscript></span></span> was computed as <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-73.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-74.gif"/></noscript></span></span> is the number of epistatic interactions per QTL, and <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-75.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-76.gif"/></noscript></span></span> are the squared mean and variance of dominance effects in the scenario with the corresponding magnitude. For example, with small epistatic effects, <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-77.gif"/></noscript></span></span> was computed as <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-78.gif"/></noscript></span></span> (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>).</p></div><div id="sec-6" class="subsection"><h4>From functional dominance and epistatic effects to statistically orthogonal effects:</h4><p id="p-21">We simulated dominance and epistasis by introducing functional dominance and epistatic effects that are independent of allele and genotype frequencies. Our interest, however, is in statistical average, dominance and epistatic effects of QTL, which do depend on genotype frequencies (<a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Fisher 1918</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Cheverud and Routman 1995</a>). We will describe the general procedure to obtain these statistical effects, for a general situation where there can be dominance, epistasis, or both. Note, however, that our scenarios had either dominance or epistasis, but never both. After obtaining statistical effects, we describe how we computed the additive genetic value, genotypic value and phenotype for each individual. Although genotypic values themselves are independent of genotype frequencies, the partitioning of these genotypic values into additive, dominance, and epistatic components does depend on genotype frequencies. Additive genetic values of individuals in population 1 were needed to compute <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-79.gif"/></noscript></span></span>, and genotypic values and phenotypes were needed because selection in population 1 was based on own performance. In the following, we describe the procedure to obtain the average effects and dominance effects in population 1 (<span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-80.gif"/></noscript></span></span>). The procedure to obtain these effects in population 2 (<span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-81.gif"/></noscript></span></span>) follows naturally by replacing the genotype and allele frequencies of population 1 with the frequencies in population 2.</p><p id="p-22">The procedure starts by applying the natural and orthogonal interactions (NOIA) model (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Álvarez-Castro and Carlborg 2007</a>) for each epistatic interaction between two QTL. First, functional epistatic values for the 9 possible two-locus genotypes at QTL k and l were collected in a vector <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-82.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-83.gif"/></noscript></span></span> is a scalar representing the functional epistatic effect between QTL k and l, and t is a 9 × 1 vector of epistatic contrasts for the 9 two-locus genotypes, ordered as (WWYY, WwYY, wwYY, WWYy, ..., wwyy). The simulated epistatic contrasts in t followed one of four configurations: additive x additive (E<sub>AA</sub>), dominance x dominance (E<sub>DD</sub>), complementary (E<sub>C</sub>), or multiplicative (E<sub>M</sub>) (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). The contrasts in t were centered and scaled to a standard deviation of one, so that the contrasts were comparable between configurations. We then used genotype frequencies of QTL k and l to partition the functional epistatic values in <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-84.gif"/></noscript></span></span> into 9 statistical genetic effects (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Álvarez-Castro and Carlborg 2007</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Vitezica <em>et al.</em> 2017</a>)<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-85.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-86.gif"/></noscript></span></span> is a 9x9 diagonal matrix with each of the nine genotype frequencies in the same order as in t. Matrix <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-87.gif"/></noscript></span></span>, where ⊗ denotes the Kronecker product, and <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-88.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-89.gif"/></noscript></span></span> are constructed as<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-90.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where columns relate to orthogonal contrasts for the mean (1), average effect (<span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-91.gif"/></noscript></span></span>), and dominance effect (<span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-92.gif"/></noscript></span></span>) of QTL X and where <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-93.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-94.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-95.gif"/></noscript></span></span> are the genotype frequencies of QTL X. The resulting vector of statistical genetic effects is<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-96.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-97.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-98.gif"/></noscript></span></span> are the terms that contribute to average effects of QTL k and l. The other terms in <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-99.gif"/></noscript></span></span> contribute to dominance effects (<span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-100.gif"/></noscript></span></span>) of individual QTL and to epistatic effects of interacting QTL (<span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-101.gif"/></noscript></span></span>).</p><p id="p-23">We repeated this procedure of partitioning functional epistatic effects into statistical genetic effects for all pairwise interactions between QTL. Each QTL was involved in 5 epistatic interactions and therefore has 5 terms that contribute to its average effect. Following this reasoning, the average effect of QTL k in population 1 with epistasis is<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-102.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-103.gif"/></noscript></span></span> is the frequency of the counted allele of QTL k in population 1, ℤ is the set of loci that QTL k interacts with, and <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-104.gif"/></noscript></span></span>. Note the difference between “additive effect” (a) and “average effect” (α); the additive effect a is half the difference in genotypic value between both opposing homozygotes, whereas the average effect (α) is the (statistical) marginal effect of the QTL. Throughout this manuscript, we will use the term “functional additive effect” to refer to a, and “average effect” (<em>i.e.</em>, statistical substitution effect) to refer to α.</p><p id="p-24">In our simulations, we needed to compute phenotypes of selection candidates in each generation, for which we needed the statistical dominance effect (<span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-105.gif"/></noscript></span></span>) of each QTL as well. The dominance effect of QTL k in population 1 with epistasis is</p><span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-106.gif"/></noscript></span></span></div><div id="sec-7" class="subsection"><h4>Additive genetic values and phenotypes:</h4><p id="p-25">We computed additive genetic values (v) of selection candidates in population 1 for the trait expressed in both population 1 and 2. Their genotypic values (g) and phenotypes were only computed for the trait expressed in population 1. The additive genetic value of individual i for the trait expressed in population 1 (2) were computed as <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-107.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-108.gif"/></noscript></span></span>), and genotypic values for the trait expressed in population 1 were computed as<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-109.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-110.gif"/></noscript></span></span> is a column vector of additive genotype indicators for individual i, and <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-111.gif"/></noscript></span></span> is column a vector of dominance genotype indicators for individual i. These indicators were coded following the NOIA parameterization as denoted in the rows of <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-112.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-113.gif"/></noscript></span></span> (Equation 1) for genotypes <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-114.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-115.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-116.gif"/></noscript></span></span>, respectively. Phenotypes with a broad sense heritability of 0.5 were computed as <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-117.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-118.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-119.gif"/></noscript></span></span> was equal to the variance of genotypic values (<span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-120.gif"/></noscript></span></span>).</p></div></div><div id="sec-8" class="subsection"><h3>Computing parameters of interest</h3><p id="p-26">The parameters of interest were (1) the genetic correlation between the trait in population 1 and the trait in population 2 (<span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-121.gif"/></noscript></span></span>), and (2) the average absolute difference in allele frequencies between populations (<span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-122.gif"/></noscript></span></span>). For each generation, we computed <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-123.gif"/></noscript></span></span> as the Pearson correlation between the additive genetic values of individuals in population 1 for the trait expressed in the two populations (equation (1)). Effectively, this <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-124.gif"/></noscript></span></span> is a weighted correlation between <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-125.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-126.gif"/></noscript></span></span>, where the weights depend on the allele frequencies in population 1. Hence, the <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-127.gif"/></noscript></span></span> computed as the correlation of additive genetic values of individuals in population 2 may give different results because the genotypes sampled from population 2 result in different weights than those sampled from population 1 (see Discussion). For each generation, we computed <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-128.gif"/></noscript></span></span> as <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-129.gif"/></noscript></span></span>. We chose this parameter as a measure for population divergence, because we expect that there is a linear relationship between <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-130.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-131.gif"/></noscript></span></span>. These parameters were computed for generation 1 to 5, and for every 5<sup>th</sup> generation after generation 5, to limit computation time.</p></div><div id="sec-9" class="subsection"><h3>Replicates</h3><p id="p-27">We ran the simulation with drift 50 times, resulting in 50 sets of genotypes (<em>i.e.</em>, replicates). For each of those replicates, we computed <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-132.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-133.gif"/></noscript></span></span> for each of the scenarios (<em>i.e.</em>, genetic model and magnitude). We ran the simulations with both selection and drift for each scenario separately, because the selection of parents in population 1 depended on the genetic model. To limit computation time, we used 20 replicates for each scenario with selection.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-28">The data used in this study can be reproduced with the files and seeds in the following GitHub repository: <a href="https://git.wageningenur.nl/duenk002/rg-and-non-additive-effects">https://git.wageningenur.nl/duenk002/rg-and-non-additive-effects</a>. Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.10252856">https://doi.org/10.25387/g3.10252856</a>.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><p id="p-29">First, for each scenario with selection, we show the change in mean genotypic value (<span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-134.gif"/></noscript></span></span>) and the change of additive genetic variance (V<sub>A</sub>) in population 1 across generations, to illustrate how population 1 evolved over time. Second, we report realized fractions of additive, dominance and epistatic variance in generation 1 and 50. Third, for scenarios with small non-additive effects, we show the effects of the genetic model and of applying selection on the additive genetic correlation (<span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-135.gif"/></noscript></span></span>) and the difference in allele frequency (<span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-136.gif"/></noscript></span></span>) between populations. Fourth, for each genetic model with selection, we investigate the impact of the magnitude of non-additive effects and the number of generations since divergence. Finally, we investigate the relationship between <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-137.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-138.gif"/></noscript></span></span> across genetic models and within genetic models. All results presented refer to generation 50 and to scenarios with small non-additive effects, unless otherwise stated.</p><div id="sec-12" class="subsection"><h3>Mean genotypic value and variance components</h3><p id="p-30">With all scenarios, the mean genotypic value expressed in genetic standard deviations (<span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-139.gif"/></noscript></span></span>) in population 1 increased due to selection (Figure S 1). With all genetic models, the increase in <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-140.gif"/></noscript></span></span> was smaller when the magnitude of non-additive effects was larger. This result was expected, because the marginal effects of alleles may change over time in the presence of non-additive effects, reducing the effectiveness of selection. The increase in <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-141.gif"/></noscript></span></span> was largest with model A, and it was smallest with model E<sub>DD</sub> and large non-additive effects. There were only small differences in <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-142.gif"/></noscript></span></span> between models D, E<sub>AA</sub>, E<sub>C</sub>, and E<sub>M</sub>.</p><p id="p-31">The additive genetic variance in population 1 (V<sub>A</sub>) decreased due to selection with all scenarios (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and Figure S 2). With genetic model A, E<sub>AA</sub>, E<sub>C</sub> and E<sub>M</sub>, about 95–98% of V<sub>A</sub> was lost after 50 generations of selection, whereas with D and E<sub>DD</sub>, 88–95% of V<sub>A</sub> was lost. A change in magnitude of non-additive effects did not substantially affect the decrease in V<sub>A</sub>, except with genetic models D and E<sub>DD</sub>, where more additive genetic variance was preserved with larger non-additive effects. In the drift scenario, the average loss of V<sub>A</sub> was about 7% for all scenarios (results not shown).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495705/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495705/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Fractions of additive (V<sub>A</sub>), dominance (V<sub>D</sub>), and epistatic (V<sub>I</sub>) variances with respect to the total genetic variance in generation 1 and generation 50 with selection. Reported values are averages of 20 replicates</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">In generation 1, scenarios that had only additive genetic (V<sub>A</sub>) and epistatic variance (V<sub>I</sub>), V<sub>A</sub> accounted for the largest, and V<sub>I</sub> for the smallest fraction of the total genetic variation (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). The largest fraction of V<sub>I</sub> was realized with genetic model E<sub>AA</sub> (max. 0.048), followed by E<sub>DD</sub> (max. 0.033), E<sub>C</sub> (max. 0.024) and E<sub>M</sub> (max. 0.017). The largest fraction of dominance variance (V<sub>D</sub>) was realized with model E<sub>DD</sub> (max. 0.364), followed by D (max. 0.298) and E<sub>C</sub> (max. 0.105). With genetic models D, E<sub>DD</sub> and E<sub>C</sub>, the fraction V<sub>D</sub> increased and V<sub>A</sub> decreased across generations, especially with intermediate or large effects (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>, generation 50). The fraction V<sub>I</sub> remained relatively constant across generations with all scenarios.</p></div><div id="sec-13" class="subsection"><h3>Effect of genetic model and of selection on <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-143.gif"/></noscript></span></span></h3><p id="p-33">For all genetic models and small non-additive effects, <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-144.gif"/></noscript></span></span> was lower with selection than with drift only (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). With drift only, <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-145.gif"/></noscript></span></span> was between 0.99 and 1 for all genetic models. After 50 generations of selection, average <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-146.gif"/></noscript></span></span> was lowest with genetic model E<sub>DD</sub> (0.65), followed by E<sub>AA</sub> (0.75), D (0.83), E<sub>C</sub> (0.83) and finally E<sub>M</sub> (0.94). There was a tendency that scenarios with the largest non-additive variance in generation 1 had the smallest <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-147.gif"/></noscript></span></span> in generation 50 (Figure S 3). Note that the <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-148.gif"/></noscript></span></span> was always equal to 1 with the additive model (A) (results not shown).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/783/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of genetic model on  with small non-additive effects, under drift only, or under drift and selection." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1925584831" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Effect of genetic model on &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-149&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; with small non-additive effects, under drift only, or under drift and selection.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/783/F3.medium.gif" width="440" height="142"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/2/783/F3.medium.gif" width="440" height="142"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/783/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/783/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495710" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-34" class="first-child">Effect of genetic model on <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/F3/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/F3/embed/mml-math-149.gif"/></noscript></span></span> with small non-additive effects, under drift only, or under drift and selection.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">As expected, <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-150.gif"/></noscript></span></span> was larger with selection than with drift, and was the same across all genetic models with drift (0.05; <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). With selection, <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-151.gif"/></noscript></span></span> with non-additive models was very similar (around 0.20) to the value with an additive model.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/783/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of genetic model on the difference in allele frequencies between populations, under drift only, or under drift and selection." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1925584831" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Effect of genetic model on the difference in allele frequencies between populations, under drift only, or under drift and selection.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/783/F4.medium.gif" width="440" height="142"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/10/2/783/F4.medium.gif" width="440" height="142"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/783/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/783/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495658" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-36" class="first-child">Effect of genetic model on the difference in allele frequencies between populations, under drift only, or under drift and selection.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Effect of the magnitude of non-additive effects</h3><p id="p-37">For all genetic models and with selection, <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-152.gif"/></noscript></span></span> decreased with increasing magnitude of non-additive effects (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). With genetic model D, <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-153.gif"/></noscript></span></span> dropped about 31% from small to intermediate, and about 27% from intermediate to large dominance effects. With all epistatic models, the drop in <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-154.gif"/></noscript></span></span> with increasing magnitude was smaller (16–23%) than with D.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/783/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of magnitude of non-additive effects on  for all genetic models, where population 1 was selected and population 2 was not selected." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1925584831" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Effect of magnitude of non-additive effects on &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-155&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for all genetic models, where population 1 was selected and population 2 was not selected.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/783/F5.medium.gif" width="440" height="209"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/10/2/783/F5.medium.gif" width="440" height="209"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/783/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/783/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495784" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-38" class="first-child">Effect of magnitude of non-additive effects on <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/F5/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/F5/embed/mml-math-155.gif"/></noscript></span></span> for all genetic models, where population 1 was selected and population 2 was not selected.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">For all genetic models with selection, the average absolute difference in allele frequency between lines (<span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-156.gif"/></noscript></span></span>) decreased with increasing magnitude of non-additive effects, especially with D and E<sub>DD</sub> (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). With model D, <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-157.gif"/></noscript></span></span> was 0.18 with intermediate dominance effects, and 0.141 with large effects. With E<sub>DD</sub>, <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-158.gif"/></noscript></span></span> was 0.162 with intermediate epistatic effects, and 0.130 with large effects. With the other epistatic models (E<sub>AA</sub>, E<sub>C</sub> and E<sub>M</sub>), the effect of an increase in magnitude on <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-159.gif"/></noscript></span></span> was much smaller (∼0.19 with intermediate and ∼0.18 with large effects).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/783/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of magnitude of non-additive effects on the difference in allele frequencies between populations under selection. In each subplot, the additive scenario (A) was included for reference (i.e., magnitude &#x201C;none&#x201D;)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1925584831" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Effect of magnitude of non-additive effects on the difference in allele frequencies between populations under selection. In each subplot, the additive scenario (A) was included for reference (&lt;em&gt;i.e.&lt;/em&gt;, magnitude &#x201C;none&#x201D;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/783/F6.medium.gif" width="440" height="213"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/10/2/783/F6.medium.gif" width="440" height="213"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/783/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/783/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495682" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-40" class="first-child">Effect of magnitude of non-additive effects on the difference in allele frequencies between populations under selection. In each subplot, the additive scenario (A) was included for reference (<em>i.e.</em>, magnitude “none”).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Effect of number of generations since divergence</h3><p id="p-41">With all scenarios, <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-160.gif"/></noscript></span></span> decreased with the number of generations since divergence, and the rate of decrease was relatively small during the first five generations (<span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-161.gif"/></noscript></span></span>), especially when the non-additive effects were small (<span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-162.gif"/></noscript></span></span>) (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). After the first five generations, the rate of decrease in <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-163.gif"/></noscript></span></span> differed across genetic models. There was a considerable difference between genetic models, the E<sub>M</sub> model showed the smallest decline of <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-164.gif"/></noscript></span></span> over time, and the E<sub>DD</sub> model showed the largest decline. With large non-additive effects, models E<sub>M</sub> and E<sub>AA</sub> tended to show an accelerated decrease in <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-165.gif"/></noscript></span></span> across generations, whereas models D, E<sub>C</sub> and E<sub>DD</sub> tended to show a decelerated decrease in <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-166.gif"/></noscript></span></span> (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/783/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of number of generations since divergence on  for all genetic models with small (left) or large (right) non-additive effects." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1925584831" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Effect of number of generations since divergence on &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-167&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for all genetic models with small (left) or large (right) non-additive effects.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/783/F7.medium.gif" width="440" height="215"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/10/2/783/F7.medium.gif" width="440" height="215"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/783/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/783/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495750" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-42" class="first-child">Effect of number of generations since divergence on <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/F7/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/F7/embed/mml-math-167.gif"/></noscript></span></span> for all genetic models with small (left) or large (right) non-additive effects.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">With all scenarios, the average absolute difference in allele frequency between lines (<span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-168.gif"/></noscript></span></span>) increased with the number of generations since divergence (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>). In contrast to the result of the genetic correlation with small non-additive effects (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7</a>), <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-169.gif"/></noscript></span></span> was remarkably similar between the genetic models (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8</a>). With large effects, models D and E<sub>DD</sub> showed a smaller <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-170.gif"/></noscript></span></span> than models E<sub>M</sub>, E<sub>AA</sub>, and E<sub>C</sub>.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/783/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of number of generations since divergence on the difference in allele frequencies between populations, for all genetic models and small (left) or large (right) non-additive effects. Dotted line represents additive trait." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1925584831" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Effect of number of generations since divergence on the difference in allele frequencies between populations, for all genetic models and small (left) or large (right) non-additive effects. Dotted line represents additive trait.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/783/F8.medium.gif" width="440" height="214"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/10/2/783/F8.medium.gif" width="440" height="214"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/783/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/783/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495662" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-44" class="first-child">Effect of number of generations since divergence on the difference in allele frequencies between populations, for all genetic models and small (left) or large (right) non-additive effects. Dotted line represents additive trait.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">In summary, for each genetic model, <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-171.gif"/></noscript></span></span> was smallest with selection, large non-additive effects and many generations since divergence. Overall, the smallest realized value of <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-172.gif"/></noscript></span></span> after 50 generations of divergence was achieved with genetic model D or E<sub>DD</sub> (<span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-173.gif"/></noscript></span></span> for both).</p></div><div id="sec-16" class="subsection"><h3>Relationship Between <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-174.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-175.gif"/></noscript></span></span></h3><p id="p-46">For all genetic models, there was a clear negative relationship between <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-176.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-177.gif"/></noscript></span></span> (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a>), and the relationship was strongest for genetic models showing the strongest decline of <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-178.gif"/></noscript></span></span> with time (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7</a>). This result suggests that differences between genetic models in the decline of <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-179.gif"/></noscript></span></span> over time originate from different impacts of <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-180.gif"/></noscript></span></span> on <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-181.gif"/></noscript></span></span>, and not from differences in <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-182.gif"/></noscript></span></span> <em>per se</em>. For example, with small non-additive effects and after 50 generations of divergence, the value of <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-183.gif"/></noscript></span></span> was different between genetic models, whereas the realized <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-184.gif"/></noscript></span></span> was very similar (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9</a>). In other words, <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-185.gif"/></noscript></span></span> is a function of <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-186.gif"/></noscript></span></span> and of genetic architecture.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/783/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between  and the difference in allele frequencies between populations for all scenarios." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1925584831" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Relationship between &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-187&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and the difference in allele frequencies between populations for all scenarios.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/783/F9.medium.gif" width="440" height="213"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/10/2/783/F9.medium.gif" width="440" height="213"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/783/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/783/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495632" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-47" class="first-child">Relationship between <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/F9/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/F9/embed/mml-math-187.gif"/></noscript></span></span> and the difference in allele frequencies between populations for all scenarios.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-48">Our objective was to investigate the relationship between non-additive effects, differences in allele frequencies between populations (<span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-188.gif"/></noscript></span></span>), and the genetic correlation between populations (<span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-189.gif"/></noscript></span></span>. We simulated genotype data of two populations that have diverged for a number of generations under drift only, or drift and selection, and we simulated traits where the genetic model and magnitude of non-additive effects were varied.</p><p id="p-49">We computed <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-190.gif"/></noscript></span></span> as the correlation between additive genetic values of individuals in population 1, for the trait expressed in population 1 and 2. Effectively, this <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-191.gif"/></noscript></span></span> is a weighted correlation between average effects in population 1 (<span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-192.gif"/></noscript></span></span>) and 2 (<span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-193.gif"/></noscript></span></span>), where the weights depend on the sample of genotypes that were used to compute the additive genetic values. This suggests that different values of <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-194.gif"/></noscript></span></span> could have been obtained when using the additive genetic values of individuals in population 2, because of differences in genotype frequencies between populations. We chose, however, to focus on population 1 because we were also interested in the change of allele frequencies over time due to selection. This approach leads to values of <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-195.gif"/></noscript></span></span> that indicate whether information from an unselected population (population 2) can be used to predict additive genetic values in a selected population (population 1).</p><div id="sec-18" class="subsection"><h3>Realized variance components</h3><p id="p-50">Because little is known about the quantity and magnitude of dominance and epistatic effects in reality, we considered a range of functional non-additive effect sizes and epistatic configurations. Realized proportions in our simulations (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>) did not always match with those observed in real data. For example, with large dominance effects, the fraction of dominance variance was 30%, which is uncommon in real data (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Ertl <em>et al.</em> 2014</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lopes <em>et al.</em> 2016</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Moghaddar and van der Werf 2017</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Joshi <em>et al.</em> 2018</a>). Similarly, scenario E<sub>DD</sub> also resulted in more dominance variance than expected in real populations, especially with large epistatic effects (33%). Empirical studies on livestock (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Bennewitz and Meuwissen 2010</a>) and crops (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Sun and Mumm 2016</a>) found that approximately 0.3% of loci show overdominance, which is comparable to our scenario with small dominance effects (0.5% overdominance). Furthermore, the scenario with small dominance effects resulted in a small proportion of dominance variance, and might therefore be most realistic for actual populations.</p><p id="p-51">In contrast to our realized proportions of dominance variance, proportions of epistatic variance were lower (max. 5%) than estimates from an empirical study on litter size in pigs (about 26%) (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Vitezica <em>et al.</em> 2018</a>), though the standard error of that estimate was large (about 22%). Further evidence of statistical epistatic effects is scarce, probably because methods used for the detection of statistical epistasis are frequently underpowered (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Wei <em>et al.</em> 2014</a>). Furthermore, it has been suggested that incomplete LD between genomic markers and QTL may create the illusion of epistasis, making inference about the importance of epistasis from genome-wide regression studies difficult (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Wei <em>et al.</em> 2014</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Zan <em>et al.</em> 2018</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">de los Campos <em>et al.</em> 2019</a>). In contrast to the lack of evidence of statistical epistasis, there is substantial evidence that physiological epistasis is abundant in several classes of organisms (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Carlborg <em>et al.</em> 2003</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Le Rouzic <em>et al.</em> 2008</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Pettersson <em>et al.</em> 2011</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Mackay 2015</a>). Nevertheless, large epistatic effects between pairs of loci are believed to be unlikely (<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Wei <em>et al.</em> 2014</a>), and the contribution of epistatic variance to the total genetic variance is expected to be small (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Hill <em>et al.</em> 2008</a>).</p><p id="p-52">In summary, among the scenario’s we studied here, scenarios D and E<sub>DD</sub> with small effects, and scenarios E<sub>AA</sub>, E<sub>C</sub> and E<sub>M</sub> are probably most realistic, because these scenarios always resulted in little dominance (max. 7%) and epistatic (max. 5%) variance.</p></div><div id="sec-19" class="subsection"><h3>Effect of genetic model on <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-196.gif"/></noscript></span></span></h3><p id="p-53">For the dominance model (D), we observed that <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-197.gif"/></noscript></span></span> decreased with increasing size of dominance effects and with increasing difference of allele frequencies between populations. In some cases, the <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-198.gif"/></noscript></span></span> can be negative due to dominance alone, as shown for a two-locus model (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Wei <em>et al.</em> 1991</a>). Such low values of <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-199.gif"/></noscript></span></span> were, however, only obtained with scenarios where both loci showed substantial overdominance, and where the difference in allele frequencies between the two populations was at least 0.3 for one of the loci. In our study, we considered many loci and the distributions of dominance effects was based on empirical results (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Bennewitz and Meuwissen 2010</a>; <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Sun and Mumm 2016</a>). These distributions resulted in only a fraction of loci showing overdominance (<em>i.e.</em>, 0.5% for small effects, 16% for intermediate effects, and 51% for large effects). Furthermore, our simulations resulted in U-shaped distributions of allele frequencies in the last generation of the historical population, which agrees with expectations based on neutral theory (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kimura and Crow 1964</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Goddard 2001</a>). After the two populations separated, allele frequency differences between populations were a result of drift and/or selection. We therefore believe that our simulations represent a more realistic model of quantitative traits and population divergence than those in <a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Wei <em>et al.</em> (1991)</a>. In conclusion, given that dominance variance is usually small and overdominance does not occur frequently, our results show that it is unlikely that true <span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-200.gif"/></noscript></span></span> values lower than 0.80 are due to dominance effects alone.</p><p id="p-54">In another simulation study, where the fraction of loci showing overdominance was 12%, realized <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-201.gif"/></noscript></span></span> was 0.78 (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Esfandyari <em>et al.</em> 2015</a>). Although the fraction of loci showing overdominance in that study was comparable to our scenario with intermediate dominance effects, our realized <span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-202.gif"/></noscript></span></span> in that scenario was much lower (0.57). This difference is likely due to the smaller number of generations that populations diverged in the study of <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Esfandyari <em>et al.</em> (2015)</a>.</p><p id="p-55">With epistasis, <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-203.gif"/></noscript></span></span> decreased with increasing size of epistatic effects and with increasing difference of allele frequencies between populations, and the value of <span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-204.gif"/></noscript></span></span> depended on the nature of the epistatic interaction (<em>i.e.</em>, configuration). In addition, there was a tendency for configurations that resulted in large initial non-additive variance to result in smaller values of <span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-205.gif"/></noscript></span></span> (Figure S 3). Even though large epistatic effects are unlikely and epistatic variance is expected to be small, <span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-206.gif"/></noscript></span></span> could be as low as 0.45 for supposedly realistic epistatic scenarios.</p><p id="p-56">To our knowledge, the relationship between the nature of epistasis and <span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-207.gif"/></noscript></span></span> has not been studied before. The mechanism behind differences in <span class="inline-formula" id="inline-formula-199"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-208.gif"/></noscript></span></span> between epistatic models can be illustrated with an example of two interacting loci. Suppose that both loci have an additive effect (a) of 1, an epistatic coefficient (γ) of 0.5, and the allele frequency at locus 1 (<span class="inline-formula" id="inline-formula-200"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-209.gif"/></noscript></span></span>) is the same in both populations (here we use 0.10). Then, we study the effect of allele frequency difference between populations at locus 2 (<span class="inline-formula" id="inline-formula-201"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-210.gif"/></noscript></span></span>) on the difference in average effects between populations (<span class="inline-formula" id="inline-formula-202"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-211.gif"/></noscript></span></span>) for locus 1 and 2. Results show that E<sub>AA</sub> and E<sub>M</sub> interactions only affect the α of the locus with fixed p (locus 1), whereas E<sub>DD</sub> and E<sub>C</sub> interactions affect the α at both loci (Figure S 4). Note that this result was the same with different values for a, γ, or <span class="inline-formula" id="inline-formula-203"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-212.gif"/></noscript></span></span>. This shows that, in general, E<sub>AA</sub> and E<sub>M</sub> interactions create a dependency of α at a locus on the allele frequency of all loci it interacts with, whereas E<sub>DD</sub> and E<sub>C</sub> interactions also create a dependency of α on the allele frequency of the locus itself. These mechanisms may contribute to the differences in <span class="inline-formula" id="inline-formula-204"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-213.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-213.gif"/></noscript></span></span> between genetic models, because the interplay between differences in allele frequencies and <span class="inline-formula" id="inline-formula-205"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-214.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-214.gif"/></noscript></span></span> depends on the genetic model.</p></div><div id="sec-20" class="subsection"><h3>Effect of magnitude of non-additive effects on <span class="inline-formula" id="inline-formula-206"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-215.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-215.gif"/></noscript></span></span></h3><p id="p-57">As expected, an increase in magnitude of dominance effects resulted in a lower <span class="inline-formula" id="inline-formula-207"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-216.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-216.gif"/></noscript></span></span>, which is in line with results from <a id="xref-ref-52-4" class="xref-bibr" href="#ref-52">Wei <em>et al.</em> (1991)</a>. Similarly, an increase in magnitude of epistatic effects also resulted in a lower <span class="inline-formula" id="inline-formula-208"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-217.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-217.gif"/></noscript></span></span>. An important question is whether this decrease of <span class="inline-formula" id="inline-formula-209"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-218.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-218.gif"/></noscript></span></span> due to an increase in magnitude continues until the theoretical limit of <span class="inline-formula" id="inline-formula-210"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-219.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-219.gif"/></noscript></span></span> is reached. Additional analyses revealed that <span class="inline-formula" id="inline-formula-211"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-220.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-220.gif"/></noscript></span></span> appears to asymptote with increasing magnitude of non-additive effects. In these analyses, we repeated our original simulations of genetic models D and E<sub>AA</sub>, using non-additive effects that were multiplied by 100 for all magnitudes. Results from those simulations showed that the difference in <span class="inline-formula" id="inline-formula-212"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-221.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-221.gif"/></noscript></span></span> between “small”, “intermediate”, or “large” effects had indeed disappeared (Figure S 5), and that the lower bound of realized values for <span class="inline-formula" id="inline-formula-213"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-222.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-222.gif"/></noscript></span></span> was ∼0.25 with scenario D and ∼0.36 with scenario E<sub>AA</sub>.</p><p id="p-58">To show the mechanism behind this result, we again consider a two-locus model where, like before, both loci have an additive effect (a) of 1, the allele frequency of locus 1 (<span class="inline-formula" id="inline-formula-214"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-223.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-223.gif"/></noscript></span></span>) is 0.10 in both populations and <span class="inline-formula" id="inline-formula-215"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-224.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-224.gif"/></noscript></span></span>. We studied the effect of the magnitude of the epistatic effect (γ) on the absolute difference in average effects between populations, relative to the absolute value of α in population 1 (<span class="inline-formula" id="inline-formula-216"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-225.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-225.gif"/></noscript></span></span>). We observed that for all epistatic models, especially for larger values of γ, both <span class="inline-formula" id="inline-formula-217"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-226.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-226.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-218"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-227.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-227.gif"/></noscript></span></span> increase roughly linearly with γ, and that therefore <span class="inline-formula" id="inline-formula-219"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-228.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-228.gif"/></noscript></span></span> stops increasing with large values of γ (Figure S 6). Note that the same mechanism was observed with dominance when <span class="inline-formula" id="inline-formula-220"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-229.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-229.gif"/></noscript></span></span> was the same in both populations and <span class="inline-formula" id="inline-formula-221"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-230.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-230.gif"/></noscript></span></span>. Hence, a change in magnitude equally affects the variance of α’s in the two populations, and the covariance between them. As a result, <span class="inline-formula" id="inline-formula-222"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-231.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-231.gif"/></noscript></span></span> is unaffected by a change in size of non-additive effects when non-additive effects are already large. In conclusion, when non-additive effects are very large, <span class="inline-formula" id="inline-formula-223"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-232.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-232.gif"/></noscript></span></span> no longer depends on the magnitude of non-additive effects relative to the magnitude of functional additive effects. At that point, there is a lower bound of <span class="inline-formula" id="inline-formula-224"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-233.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-233.gif"/></noscript></span></span> that is determined by the nature of the non-additive effects (<em>i.e.</em>, type of inter-allelic interaction) and by the difference in allele frequencies between populations.</p></div><div id="sec-21" class="subsection"><h3>Number of epistatic interactions</h3><p id="p-59">In the epistatic scenarios, we assumed that each locus interacted with 5 other loci. Because little is known about the number of interactions per locus (<span class="inline-formula" id="inline-formula-225"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-234.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-234.gif"/></noscript></span></span>) in reality, we tested whether our results were sensitive to a change in <span class="inline-formula" id="inline-formula-226"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-235.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-235.gif"/></noscript></span></span>. For that purpose, we repeated all simulations of epistatic scenarios with <span class="inline-formula" id="inline-formula-227"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-236.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-236.gif"/></noscript></span></span>. Note that the total functional epistatic variance with <span class="inline-formula" id="inline-formula-228"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-237.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-237.gif"/></noscript></span></span> was the same as with <span class="inline-formula" id="inline-formula-229"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-238.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-238.gif"/></noscript></span></span>, because the epistatic coefficients were scaled with <span class="inline-formula" id="inline-formula-230"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-239.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-239.gif"/></noscript></span></span>, so that the product <span class="inline-formula" id="inline-formula-231"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-240.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-240.gif"/></noscript></span></span> is constant. This analyses resulted in values of <span class="inline-formula" id="inline-formula-232"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-241.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-241.gif"/></noscript></span></span> that were very similar to those of our original simulations (results not shown), suggesting that, in our simulations, the value of <span class="inline-formula" id="inline-formula-233"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-242.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-242.gif"/></noscript></span></span> depends on the level of total functional epistatic variance, which scales similarly with <span class="inline-formula" id="inline-formula-234"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-243.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-243.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-235"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-244.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-244.gif"/></noscript></span></span>.</p></div><div id="sec-22" class="subsection"><h3>Effect of selection on <span class="inline-formula" id="inline-formula-236"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-245.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-245.gif"/></noscript></span></span></h3><p id="p-60">Non-additive effects and selection create a complex interplay between average effects, the difference in allele frequencies between populations (<span class="inline-formula" id="inline-formula-237"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-246.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-246.gif"/></noscript></span></span>) over time, and their effects on <span class="inline-formula" id="inline-formula-238"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-247.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-247.gif"/></noscript></span></span>. For a trait with small dominance effects under selection, we observed that <span class="inline-formula" id="inline-formula-239"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-248.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-248.gif"/></noscript></span></span> was almost the same as for an additive trait (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8</a>). We expected, however, that directional dominance would reduce <span class="inline-formula" id="inline-formula-240"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-249.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-249.gif"/></noscript></span></span>, because the average effect at a locus can become smaller or even switch sign when the frequency of the favorable dominant allele increases (<a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">Falconer and Mackay 1996</a>). This change in average effects would affect the change in allele frequencies over time due to selection in population 1, because the selection pressure at loci may change. A reduction in <span class="inline-formula" id="inline-formula-241"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-250.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-250.gif"/></noscript></span></span> with small dominance effects was not observed, probably because only a small fraction of loci showed full- or over-dominance. Indeed, with large dominance coefficients (so that the fraction of loci showing over-dominance was much larger compared to with small dominance coefficients) <span class="inline-formula" id="inline-formula-242"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-251.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-251.gif"/></noscript></span></span> was smaller (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). In real data, however, we do not expect a large fraction of loci that show full- or over-dominance (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Wellmann and Bennewitz 2011</a>). It is therefore unlikely that dominance significantly affects the change in allele frequencies over time due to selection, compared to a purely additive trait.</p><p id="p-61">For a trait with epistatic effects under selection, we observed that <span class="inline-formula" id="inline-formula-243"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-252.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-252.gif"/></noscript></span></span> was a bit smaller than that for a trait with only additive effects (<a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 8</a>). Similar to the models with only dominance effects, this reduction in <span class="inline-formula" id="inline-formula-244"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-253.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-253.gif"/></noscript></span></span> was expected because the average effect at a locus can become smaller or switch sign over time in the presence of epistasis. How epistasis affects the change in allele frequencies due to selection depends on the directionality of the epistatic interaction effect. Theory suggests that, compared to pure additivity, positive interactions (<em>i.e.</em>, in the same direction as the additive effects) will promote the selection of favorable alleles, whereas negative interactions (<em>i.e.</em>, in the opposite direction from the additive effects) will suppress the selection of favorable alleles (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Carter <em>et al.</em> 2005</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hansen 2013</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Paixão and Barton 2016</a>). We chose to simulate both positive and negative interactions with equal probabilities, because empirical studies suggest that epistatic interactions are not biased in being either positive or negative (<em>i.e.</em>, they are non-directional) (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Mackay 2014</a>). Our results showed that, for a trait with intermediate epistatic effects, the net effect of having both positive and negative interactions was a decrease in fixation rate of favorable alleles (<em>i.e.</em>, with a positive a), and an increase in fixation rate of unfavorable alleles (<em>i.e.</em>, with a negative a) compared to an additive trait (Figure S 7). Similar results were found by <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Esfandyari <em>et al.</em> (2017)</a>. In conclusion, epistatic effects may affect <span class="inline-formula" id="inline-formula-245"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-254.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-254.gif"/></noscript></span></span> through two related mechanisms. First, with an epistatic model and when selection takes place in one of the populations, the difference in allele frequencies between populations may be smaller compared to an additive model. This reduction occurs because negative interactions decrease the fixation rates of favorable alleles, and increase those of unfavorable alleles. Second, for given allele frequency differences, the value of <span class="inline-formula" id="inline-formula-246"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-255.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-255.gif"/></noscript></span></span> depends on the nature of the epistatic interaction.</p></div><div id="sec-23" class="subsection"><h3>Loss of additive genetic variance</h3><p id="p-62">Selection experiments in Drosophila, maize, and <em>Escherichia coli</em> have shown that additive genetic variation (V<sub>A</sub>) can be maintained for at least 100 generations (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hill 2016</a>). Some researchers suggested that this preservation of V<sub>A</sub> may be due to the conversion of non-additive genetic variance to additive genetic variance (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Cheverud and Routman 1996</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hallander and Waldmann 2007</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Hill 2017</a>). Simulation studies, however, have failed to show a preservation of V<sub>A</sub> due to this conversion (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Carter <em>et al.</em> 2005</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Esfandyari <em>et al.</em> 2017</a>). Similarly, our simulations showed little conversion of non-additive genetic variance to V<sub>A</sub> with genetic models E<sub>AA</sub> and E<sub>M</sub>, and no conversion with other genetic models (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). As a result, almost all additive genetic variance was lost after 50 generations (Figure S 2).</p><p id="p-63">The large loss of additive genetic variance in our simulations may be due to two reasons. First, there was little epistatic variance in generation one that could be ‘converted’ to V<sub>A</sub> in subsequent generations (<a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Hill <em>et al.</em> 2008</a>; <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Mäki-Tanila and Hill 2014</a>). This was largely because the allele frequency distribution was strongly U-shaped in generation one. Second, mutational variance was zero because there were no mutations simulated after the historical generation. Even though these mechanisms may explain some of the loss of <span class="inline-formula" id="inline-formula-247"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-256.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-256.gif"/></noscript></span></span> in our simulations, the issue still remains that, to date, simulations have failed to convincingly reproduce the conservation of <span class="inline-formula" id="inline-formula-248"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-257.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-257.gif"/></noscript></span></span> observed in reality (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Johnson and Barton 2005</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Walsh and Lynch 2018</a>).</p></div><div id="sec-24" class="subsection"><h3>Practical relevance</h3><p id="p-64">In our simulations, there was selection in only one of the populations, while the other population was unselected. In reality, populations may have been divergently selected (<em>e.g.</em>, Friesian Holstein <em>vs.</em> Angus cattle), resulting in larger differences in allele frequencies than simulated here. Hence, <span class="inline-formula" id="inline-formula-249"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-258.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-258.gif"/></noscript></span></span> between divergently selected populations may be smaller than observed in our simulations.</p><p id="p-65">In this study, we assumed that there were no genotype x environment interactions (GxE), so that <span class="inline-formula" id="inline-formula-250"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-259.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-259.gif"/></noscript></span></span> values smaller than one were only due to non-additive effects. In reality, both non-additive effects and GxE may contribute to <span class="inline-formula" id="inline-formula-251"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-260.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-260.gif"/></noscript></span></span> values being smaller than one. The relative importance of non-additive effects and GxE can be inferred from the difference between estimated <span class="inline-formula" id="inline-formula-252"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-261.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-261.gif"/></noscript></span></span>, from a design where the populations were tested in different environments, and from a design where one of the populations was tested in the environment of the other population. This approach is similar to what was proposed by <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wientjes and Calus (2017)</a> to dissect the components of the genetic correlation between purebred and crossbred performance. However, to our knowledge, there are no studies that have used this approach to disentangle the effects of non-additive effects and GxE on <span class="inline-formula" id="inline-formula-253"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-262.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-262.gif"/></noscript></span></span>. This study shows that, even without GxE, the <span class="inline-formula" id="inline-formula-254"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-263.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-263.gif"/></noscript></span></span> can be substantially smaller than one, and sometimes even close to zero.</p><p id="p-66">Estimated genetic correlations between two populations (<span class="inline-formula" id="inline-formula-255"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-264.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-264.gif"/></noscript></span></span>) may differ across traits (<em>e.g.</em>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lund <em>et al.</em> 2011</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Karoui <em>et al.</em> 2012</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Porto-Neto <em>et al.</em> 2015</a>). For example, in dairy cattle, <span class="inline-formula" id="inline-formula-256"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-265.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-265.gif"/></noscript></span></span> of fertility traits tended to be lower than those of fat yield and milk production (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Karoui <em>et al.</em> 2012</a>). The results from the present study suggest that such differences in <span class="inline-formula" id="inline-formula-257"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-266.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-266.gif"/></noscript></span></span> may indicate differences in the underlying genetic model between traits (<em>i.e.</em>, in the importance of non-additive effects). Although this may be the case, differences in <span class="inline-formula" id="inline-formula-258"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-267.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-267.gif"/></noscript></span></span> between traits can arise through other mechanisms as well. First, <span class="inline-formula" id="inline-formula-259"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-268.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-268.gif"/></noscript></span></span> often include a component due to GxE interactions. Such GxE interactions may be more important for some traits than for others, resulting in differences in <span class="inline-formula" id="inline-formula-260"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-269.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-269.gif"/></noscript></span></span> between traits. Second, different traits are influenced by (at least partly) different QTL, and some traits may have been under stronger selection than others. As a result, the differences in allele frequencies at QTL between populations may vary across traits. These mechanisms may result in differences in <span class="inline-formula" id="inline-formula-261"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-270.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-270.gif"/></noscript></span></span> between traits, even when the underlying genetic models of those traits are similar. It is therefore questionable whether inferences can be made about differences in genetic model among traits, based on differences in <span class="inline-formula" id="inline-formula-262"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-271.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-271.gif"/></noscript></span></span>.</p><p id="p-67">The results in this study may be relevant for the prediction of additive genetic values across populations using genomic information. In this strategy, termed across-population genomic prediction, average effects at markers are estimated in one population, and used to compute additive genetic values in another population (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">de Roos <em>et al.</em> 2009</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hayes <em>et al.</em> 2009</a>). It has been suggested that the inefficiency of across-population genomic prediction is partly due to differences in linkage disequilibrium between markers and QTL. This insight has inspired the use of whole-genome sequence (WGS) data, because in WGS data, genotypes of the QTL themselves are included (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Iheshiulor <em>et al.</em> 2016</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Raymond <em>et al.</em> 2018a</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Raymond <em>et al.</em> 2018b</a>). The results of the current study suggest, however, that even when QTL genotypes are known and their average effects are accurately estimated in one population, across-population genomic prediction may be inefficient, because <span class="inline-formula" id="inline-formula-263"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-272.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-272.gif"/></noscript></span></span> can differ considerably from one, even when genetic variance is mostly additive. This view is supported by the results of <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Raymond <em>et al.</em> (2018b)</a>, who reported that although the <span class="inline-formula" id="inline-formula-264"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-273.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-273.gif"/></noscript></span></span> estimated from putative QTL was higher than the estimate from regular marker data, it was still lower than one.</p><p id="p-68">Similar to across populations, genomic prediction from current to future generations may be inefficient because of changes in allele frequencies, and the subsequent changes in average effects at QTL. In other words, two different generations can be considered as two populations that have a genetic correlation between them that may be lower than unity. The results of this study may therefore partly explain the need for frequent retraining of genomic prediction models to achieve constant accuracy across generations (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Sonesson and Meuwissen 2009</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Wolc <em>et al.</em> 2011</a>). We expect, however, that the change in allele frequency at a single QTL is relatively small across a few (4-5) generations, especially for traits that are highly polygenic. As a result, <span class="inline-formula" id="inline-formula-265"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-274.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-274.gif"/></noscript></span></span> may be relatively high across a few generations. Nevertheless, the relative contribution of non-additive effects to the decline of genomic prediction accuracy across generations is currently unknown, and would be an interesting topic for future research.</p></div><div id="sec-25" class="subsection"><h3>Conclusion</h3><p id="p-69">Our findings show that the genetic correlation between populations (<span class="inline-formula" id="inline-formula-266"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-275.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-275.gif"/></noscript></span></span> is partly determined by the difference in allele frequencies between populations and the magnitude of non-additive effects. Large differences in allele frequencies and large non-additive effects resulted in low values of <span class="inline-formula" id="inline-formula-267"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-276.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-276.gif"/></noscript></span></span>. For both dominance and epistasis, when non-additive effects become extremely large, <span class="inline-formula" id="inline-formula-268"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-277.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-277.gif"/></noscript></span></span> has a lower bound that is determined by the nature of non-additive effects, and the difference in allele frequencies between populations. In addition, we found that with epistasis, <span class="inline-formula" id="inline-formula-269"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-278.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-278.gif"/></noscript></span></span> depends on the level of total functional epistatic variance, which is a function of epistatic effect size and the number of interactions per locus. Given that dominance variance is usually small and there is not much overdominance, we expect that it is unlikely that values of <span class="inline-formula" id="inline-formula-270"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-279.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-279.gif"/></noscript></span></span> below 0.8 are due to dominance alone. With supposedly realistic epistasis, <span class="inline-formula" id="inline-formula-271"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-280.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/783/embed/mml-math-280.gif"/></noscript></span></span> could be as low as 0.45. These results may contribute to the understanding of differences in genetic expression of complex traits between populations, and may help in explaining the inefficiency of genomic prediction across populations.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-70">The authors thank the Netherlands Organization of Scientific Research (NWO) and the Breed4Food consortium partners Cobb Europe, CRV, Hendrix Genetics, and Topigs Norsvin for their financial support. We thank the Wageningen Institute of Animal Science for a scholarship that allowed PD to visit the University of New England in Armidale, Australia. The use of the HPC cluster has been made possible by CAT-AgroFood (Shared Research Facilities Wageningen UR). The authors also thank Mehdi Sargolzaei for providing an improved version of QMSim.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-71">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.10252856">https://doi.org/10.25387/g3.10252856</a>.</p></li><li class="fn" id="fn-2"><p id="p-72"><em>Communicating editor: J.-L. Jannink</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-08-25"><span class="received-label">Received </span>August 25, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-12-18"><span class="accepted-label">Accepted </span>December 18, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Duenk <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.1" data-doi="10.1534/genetics.106.067348"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Álvarez-Castro</span>,  <span class="cit-name-given-names">J. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Ö.</span> <span class="cit-name-surname">Carlborg</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A Unified Model for Functional and Statistical Epistasis and Its Application in Quantitative Trait Loci Analysis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">1151</span>–<span class="cit-lpage">1167</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.067348</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.067348%26rft_id%253Dinfo%253Apmid%252F17409082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni8yLzExNTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzc4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.783.2" data-doi="10.5962/bhl.title.44575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bateson</span>,  <span class="cit-name-given-names">W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Mendel</span></span></li></ol><cite>, <span class="cit-pub-date">1909</span> <span class="cit-source"><em>Mendel’s principles of heredity</em>.</span> <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">Cambridge, UK</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.5962/bhl.title.44575</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMendel%253Fs%2Bprinciples%2Bof%2Bheredity.%26rft_id%253Dinfo%253Adoi%252F10.5962%252Fbhl.title.44575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5962/bhl.title.44575&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.3" data-doi="10.1111/j.1439-0388.2009.00847.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bennewitz</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. H. E.</span> <span class="cit-name-surname">Meuwissen</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The distribution of QTL additive and dominance effects in porcine F2 crosses.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">179</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0388.2009.00847.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DBennewitz%26rft.auinit1%253DJ.%26rft.volume%253D127%26rft.issue%253D3%26rft.spage%253D171%26rft.epage%253D179%26rft.atitle%253DThe%2Bdistribution%2Bof%2BQTL%2Badditive%2Band%2Bdominance%2Beffects%2Bin%2Bporcine%2BF2%2Bcrosses.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2009.00847.x%26rft_id%253Dinfo%253Apmid%252F20536634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2009.00847.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20536634&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.4" data-doi="10.1016/j.ajhg.2016.05.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>,  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Ye</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Price</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Zaitlen</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Transethnic Genetic-Correlation Estimates from Summary Statistics.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">76</span>–<span class="cit-lpage">88</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2016.05.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D99%26rft.spage%253D76%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2016.05.001%26rft_id%253Dinfo%253Apmid%252F27321947%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2016.05.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27321947&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.5" data-doi="10.1101/gr.528003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>,  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kerje</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Schutz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Jacobsson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Jensen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A global search reveals epistatic interaction between QTL for early growth in the chicken.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">421</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.528003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DCarlborg%26rft.auinit1%253DO.%26rft.volume%253D13%26rft.issue%253D3%26rft.spage%253D413%26rft.epage%253D421%26rft.atitle%253DA%2BGlobal%2BSearch%2BReveals%2BEpistatic%2BInteraction%2BBetween%2BQTL%2Bfor%2BEarly%2BGrowth%2Bin%2Bthe%2BChicken%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.528003%26rft_id%253Dinfo%253Apmid%252F12618372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEzLzMvNDEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi83ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.6" data-doi="10.1016/j.tpb.2005.05.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>,  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hermisson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. F.</span> <span class="cit-name-surname">Hansen</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The role of epistatic gene interactions in the response to selection and the evolution of evolvability.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">179</span>–<span class="cit-lpage">196</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tpb.2005.05.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DCarter%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D68%26rft.issue%253D3%26rft.spage%253D179%26rft.epage%253D196%26rft.atitle%253DThe%2Brole%2Bof%2Bepistatic%2Bgene%2Binteractions%2Bin%2Bthe%2Bresponse%2Bto%2Bselection%2Band%2Bthe%2Bevolution%2Bof%2Bevolvability.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tpb.2005.05.002%26rft_id%253Dinfo%253Apmid%252F16122771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tpb.2005.05.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16122771&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232911200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheverud</span>,  <span class="cit-name-given-names">J. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. J.</span> <span class="cit-name-surname">Routman</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Epistasis and its contribution to genetic variance components.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">139</span>: <span class="cit-fpage">1455</span>–<span class="cit-lpage">1461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCheverud%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D139%26rft.issue%253D3%26rft.spage%253D1455%26rft.epage%253D1461%26rft.atitle%253DEpistasis%2Band%2Bits%2Bcontribution%2Bto%2Bgenetic%2Bvariance%2Bcomponents.%26rft_id%253Dinfo%253Apmid%252F7768453%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzOS8zLzE0NTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzc4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.8" data-doi="10.2307/2410645"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheverud</span>,  <span class="cit-name-given-names">J. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. J.</span> <span class="cit-name-surname">Routman</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Epistasis as a source of increased additive genetic variance at population bottlenecks.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1042</span>–<span class="cit-lpage">1051</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.1996.tb02345.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D50%26rft.spage%253D1042%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2410645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2410645&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996UY55500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cockerham</span>,  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1954</span> <span class="cit-article-title">An extension of the concept of partitioning hereditary variance for analysis of covariances among relatives when epistasis is present.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">859</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCockerham%26rft.auinit1%253DC.%2BC.%26rft.volume%253D39%26rft.issue%253D6%26rft.spage%253D859%26rft.epage%253D882%26rft.atitle%253DAN%2BEXTENSION%2BOF%2BTHE%2BCONCEPT%2BOF%2BPARTITIONING%2BHEREDITARY%2BVARIANCE%2BFOR%2BANALYSIS%2BOF%2BCOVARIANCES%2BAMONG%2BRELATIVES%2BWHEN%2BEPISTASIS%2BIS%2BPRESENT%26rft_id%253Dinfo%253Apmid%252F17247525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIzOS82Lzg1OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvNzgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.10" data-doi="10.1016/j.ajhg.2013.07.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Candia</span>,  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. H.</span> <span class="cit-name-surname">Lee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. L.</span> <span class="cit-name-surname">Browning</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. V.</span> <span class="cit-name-surname">Gejman</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Additive genetic variation in schizophrenia risk is shared by populations of African and European descent.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">463</span>–<span class="cit-lpage">470</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2013.07.007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D93%26rft.spage%253D463%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2013.07.007%26rft_id%253Dinfo%253Apmid%252F23954163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2013.07.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23954163&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Sorensen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Toro</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> Imperfect Linkage Disequilibrium Generates Phantom Epistasis (&amp;Perils of Big Data). G3 (Bethesda). 9: 1429–1436.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.12" data-doi="10.1534/genetics.109.104935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Roos</span>,  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Reliability of genomic predictions across multiple populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">1545</span>–<span class="cit-lpage">1553</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.104935</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.104935%26rft_id%253Dinfo%253Apmid%252F19822733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4My80LzE1NDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzc4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.13" data-doi="10.1186/1297-9686-46-40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ertl</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Legarra</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. G.</span> <span class="cit-name-surname">Vitezica</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Varona</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Edel</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic analysis of dominance effects on milk production and conformation traits in Fleckvieh cattle.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">40</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-46-40</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D46%26rft.spage%253D40%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-46-40%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-46-40&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Esfandyari</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Henryon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Berg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Thomasen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Bijma</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Response to Selection in Finite Locus Models with Nonadditive Effects.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">318</span>–<span class="cit-lpage">327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft.volume%253D108%26rft.spage%253D318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.15" data-doi="10.1186/s12711-015-0155-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Esfandyari</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Sorensen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Bijma</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A crossbred reference population can improve the response to genomic selection for crossbred performance.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">76</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-015-0155-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D76%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-015-0155-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-015-0155-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.783.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>,  <span class="cit-name-given-names">D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Mackay</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source">Introduction to quantitative genetics</span>, <span class="cit-publ-name">Pearson Education Limited</span>, <span class="cit-publ-loc">Harlow</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.17" data-doi="10.1086/281736"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>,  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1952</span> <span class="cit-article-title">The Problem of Environment and Selection.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">293</span>–<span class="cit-lpage">298</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/281736</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D86%26rft.spage%253D293%26rft_id%253Dinfo%253Adoi%252F10.1086%252F281736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/281736&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1952UY00500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.18" data-doi="10.1017/S0080456800012163"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>,  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1918</span> <span class="cit-article-title">The Correlation between Relatives on the Supposition of Mendelian Inheritance.</span> <abbr class="cit-jnl-abbrev">Trans. R. Soc. Edinb.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">433</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0080456800012163</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrans.%2BR.%2BSoc.%2BEdinb.%26rft.volume%253D52%26rft.spage%253D399%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0080456800012163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0080456800012163&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.19" data-doi="10.1038/ng.3800"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forsberg</span>,  <span class="cit-name-given-names">S. K. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Bloom</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Sadhu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Kruglyak</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Ö.</span> <span class="cit-name-surname">Carlborg</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Accounting for genetic interactions improves modeling of individual quantitative trait phenotypes in yeast.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">497</span>–<span class="cit-lpage">503</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3800</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D49%26rft.spage%253D497%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3800%26rft_id%253Dinfo%253Apmid%252F28250458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3800&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28250458&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.20" data-doi="10.1016/S0301-6226(01)00272-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>,  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The validity of genetic models underlying quantitative traits.</span> <abbr class="cit-jnl-abbrev">Livest. Prod. Sci.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">117</span>–<span class="cit-lpage">127</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0301-6226(01)00272-X</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BProd.%2BSci.%26rft.volume%253D72%26rft.spage%253D117%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0301-6226%252801%252900272-X%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0301-6226(01)00272-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.21" data-doi="10.1038/sj.hdy.6800946"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hallander</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Waldmann</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The effect of non-additive genetic interactions on selection in multi-locus genetic models.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/sj.hdy.6800946</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHallander%26rft.auinit1%253DJ.%26rft.volume%253D98%26rft.issue%253D6%26rft.spage%253D349%26rft.epage%253D359%26rft.atitle%253DThe%2Beffect%2Bof%2Bnon-additive%2Bgenetic%2Binteractions%2Bon%2Bselection%2Bin%2Bmulti-locus%2Bgenetic%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800946%26rft_id%253Dinfo%253Apmid%252F17327874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800946&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17327874&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.22" data-doi="10.1111/evo.12214"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>,  <span class="cit-name-given-names">T. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Why epistasis is important for selection and adaptation.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">3501</span>–<span class="cit-lpage">3511</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/evo.12214</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D67%26rft.spage%253D3501%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.12214%26rft_id%253Dinfo%253Apmid%252F24299403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.12214&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24299403&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327572400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.23" data-doi="10.1186/1297-9686-41-51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Bowman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Chamberlain</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Verbyla</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Accuracy of genomic breeding values in multi-breed dairy cattle populations.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">51</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-41-51</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D41%26rft.spage%253D51%26rft.epage%253D51%26rft.atitle%253DAccuracy%2Bof%2Bgenomic%2Bbreeding%2Bvalues%2Bin%2Bmulti-breed%2Bdairy%2Bcattle%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-51%26rft_id%253Dinfo%253Apmid%252F19930712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-51&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19930712&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.24" data-doi="10.1534/genetics.115.186650"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>,  <span class="cit-name-given-names">W. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Is Continued Genetic Improvement of Livestock Sustainable?</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">877</span>–<span class="cit-lpage">881</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.186650</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D202%26rft.issue%253D3%26rft.spage%253D877%26rft.epage%253D881%26rft.atitle%253DIs%2BContinued%2BGenetic%2BImprovement%2Bof%2BLivestock%2BSustainable%253F%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.186650%26rft_id%253Dinfo%253Apmid%252F26953266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAyLzMvODc3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi83ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.25" data-doi="10.1111/jbg.12270"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>,  <span class="cit-name-given-names">W. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">“Conversion” of epistatic into additive genetic variance in finite populations and possible impact on long-term selection response.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">196</span>–<span class="cit-lpage">201</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jbg.12270</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D134%26rft.spage%253D196%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjbg.12270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jbg.12270&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.26" data-doi="10.1371/journal.pgen.1000008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>,  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Visscher</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Data and theory point to mainly additive genetic variance for complex traits.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-elocation-id">e1000008</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253De1000008%26rft.epage%253De1000008%26rft.atitle%253DData%2Band%2Btheory%2Bpoint%2Bto%2Bmainly%2Badditive%2Bgenetic%2Bvariance%2Bfor%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000008%26rft_id%253Dinfo%253Apmid%252F18454194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18454194&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.27" data-doi="10.1186/s12711-016-0193-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iheshiulor</span>,  <span class="cit-name-given-names">O. O. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Woolliams</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Yu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Wellmann</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. H. E.</span> <span class="cit-name-surname">Meuwissen</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Within- and across-breed genomic prediction using whole-genome sequence and single nucleotide polymorphism panels.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-016-0193-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D48%26rft.spage%253D15%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-016-0193-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-016-0193-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.28" data-doi="10.1098/rstb.2005.1667"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>,  <span class="cit-name-given-names">T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Barton</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Theoretical models of selection and mutation on quantitative traits.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">360</span>: <span class="cit-fpage">1411</span>–<span class="cit-lpage">1425</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rstb.2005.1667</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DJohnson%26rft.auinit1%253DT.%26rft.volume%253D360%26rft.issue%253D1459%26rft.spage%253D1411%26rft.epage%253D1425%26rft.atitle%253DTheoretical%2Bmodels%2Bof%2Bselection%2Band%2Bmutation%2Bon%2Bquantitative%2Btraits%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2005.1667%26rft_id%253Dinfo%253Apmid%252F16048784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2005.1667&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16048784&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.29" data-doi="10.1038/s41437-017-0046-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Joshi</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Woolliams</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Meuwissen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. M.</span> <span class="cit-name-surname">Gjoen</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Maternal, dominance and additive genetic effects in Nile tilapia; influence on growth, fillet yield and body size traits.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">452</span>–<span class="cit-lpage">462</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41437-017-0046-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41437-017-0046-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41437-017-0046-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.30" data-doi="10.1186/1297-9686-44-39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karoui</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Carabaño</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Díaz</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Legarra</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Joint genomic evaluation of French dairy cattle breeds using multiple-trait models.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">39</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-44-39</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D44%26rft.spage%253D39%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-44-39%26rft_id%253Dinfo%253Apmid%252F23216664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-44-39&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23216664&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-confproc" id="cit-10.2.783.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kempthorne</span>,  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">1954</span> The Correlation between Relatives in a Random Mating Population. Proc R Soc Lond Ser B Biol sci 143: <span class="cit-fpage">103</span>–<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. F.</span> <span class="cit-name-surname">Crow</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">The number of alleles that can be maintained in a finite population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">725</span>–<span class="cit-lpage">738</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKimura%26rft.auinit1%253DM.%26rft.volume%253D49%26rft.issue%253D4%26rft.spage%253D725%26rft.epage%253D738%26rft.atitle%253DTHE%2BNUMBER%2BOF%2BALLELES%2BTHAT%2BCAN%2BBE%2BMAINTAINED%2BIN%2BA%2BFINITE%2BPOPULATION%26rft_id%253Dinfo%253Apmid%252F14156929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI0OS80LzcyNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvNzgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.33" data-doi="10.1534/genetics.108.089300"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Le Rouzic</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Alvarez-Castro</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Carlborg</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Dissection of the genetic architecture of body weight in chicken reveals the impact of epistasis on domestication traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1591</span>–<span class="cit-lpage">1599</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.108.089300</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.089300%26rft_id%253Dinfo%253Apmid%252F18622035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8zLzE1OTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzc4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.34" data-doi="10.1111/jbg.12195."><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>,  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. W. M.</span> <span class="cit-name-surname">Bastiaansen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Janss</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. F.</span> <span class="cit-name-surname">Knol</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Bovenhuis</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction of growth in pigs based on a model including additive and dominance effects.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">180</span>–<span class="cit-lpage">186</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jbg.12195</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D133%26rft.spage%253D180%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjbg.12195.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jbg.12195.&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.35" data-doi="10.1186/1297-9686-43-43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>,  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. P. W.</span> <span class="cit-name-surname">de Roos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. G.</span> <span class="cit-name-surname">de Vries</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Druet</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Ducrocq</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A common reference population from four European Holstein populations increases reliability of genomic predictions.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">43</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-43-43</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DLund%26rft.auinit1%253DM.%2BS.%26rft.volume%253D43%26rft.spage%253D43%26rft.epage%253D43%26rft.atitle%253DA%2Bcommon%2Breference%2Bpopulation%2Bfrom%2Bfour%2BEuropean%2BHolstein%2Bpopulations%2Bincreases%2Breliability%2Bof%2Bgenomic%2Bpredictions.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-43%26rft_id%253Dinfo%253Apmid%252F22152008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-43&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22152008&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.36" data-doi="10.1038/nrg3627"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>,  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Epistasis and quantitative traits: using model organisms to study gene–gene interactions.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">22</span>–<span class="cit-lpage">33</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3627</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3627%26rft_id%253Dinfo%253Apmid%252F24296533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3627&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24296533&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.783.37" data-doi="10.1007/978-1-4939-2155-3_4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>,  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Epistasis for Quantitative Traits in Drosophila.</span> Epistasis: Methods and Protocols <span class="cit-vol">1253</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">70</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-4939-2155-3_4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D1253%26rft.spage%253D47%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4939-2155-3_4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4939-2155-3_4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.38" data-doi="10.1534/genetics.114.165282"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mäki-Tanila</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. G.</span> <span class="cit-name-surname">Hill</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Influence of gene interaction on complex trait variation with multilocus models.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">367</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.165282</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.165282%26rft_id%253Dinfo%253Apmid%252F24990992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzEvMzU1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi83ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moghaddar</span>,  <span class="cit-name-given-names">N.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H. J.</span> <span class="cit-name-surname">van der Werf</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic estimation of additive and dominance effects and impact of accounting for dominance on accuracy of genomic evaluation in sheep populations.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">00</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D00%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.40" data-doi="10.1371/journal.pgen.1005201"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Monnahan</span>,  <span class="cit-name-given-names">P. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Kelly</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Epistasis Is a Major Determinant of the Additive Genetic Variance in Mimulus guttatus.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e1005201</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005201</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005201%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005201%26rft_id%253Dinfo%253Apmid%252F25946702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005201&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25946702&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.41" data-doi="10.1073/pnas.1518830113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paixão</span>,  <span class="cit-name-given-names">T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. H.</span> <span class="cit-name-surname">Barton</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The effect of gene interactions on the long-term response to selection.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">4422</span>–<span class="cit-lpage">4427</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1518830113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1518830113%26rft_id%253Dinfo%253Apmid%252F27044080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEzLzE2LzQ0MjIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzc4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.42" data-doi="10.1371/journal.pgen.1002180"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pettersson</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Besnier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. B.</span> <span class="cit-name-surname">Siegel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Carlborg</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Replication and explorations of high-order epistasis using a large advanced intercross line pedigree.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e1002180</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002180</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPettersson%26rft.auinit1%253DM.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De1002180%26rft.epage%253De1002180%26rft.atitle%253DReplication%2Band%2Bexplorations%2Bof%2Bhigh-order%2Bepistasis%2Busing%2Ba%2Blarge%2Badvanced%2Bintercross%2Bline%2Bpedigree.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002180%26rft_id%253Dinfo%253Apmid%252F21814519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002180&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21814519&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.43" data-doi="10.1186/s12711-015-0162-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Porto-Neto</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Barendse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Henshall</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">McWilliam</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Lehnert</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic correlation: harnessing the benefit of combining two unrelated populations for genomic selection.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">84</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-015-0162-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D84%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-015-0162-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-015-0162-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.44" data-doi="10.1186/s12711-018-0396-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raymond</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Bouwman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Schrooten</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Houwing-Duistermaat</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. F.</span> <span class="cit-name-surname">Veerkamp</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">a</span> <span class="cit-article-title">Utility of whole-genome sequence data for across-breed genomic prediction.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">27</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-018-0396-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D50%26rft.spage%253D27%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-018-0396-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-018-0396-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.45" data-doi="10.1186/s12711-018-0419-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raymond</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Bouwman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. C. J.</span> <span class="cit-name-surname">Wientjes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Schrooten</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Houwing-Duistermaat</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">b</span> <span class="cit-article-title">Genomic prediction for numerically small breeds, using models with pre-selected and differentially weighted markers.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">49</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-018-0419-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D50%26rft.spage%253D49%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-018-0419-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-018-0419-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.46" data-doi="10.1093/bioinformatics/btp045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sargolzaei</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. S.</span> <span class="cit-name-surname">Schenkel</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">QMSim: a large-scale genome simulator for livestock.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">680</span>–<span class="cit-lpage">681</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp045</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp045%26rft_id%253Dinfo%253Apmid%252F19176551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19176551&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.47" data-doi="10.1186/1297-9686-41-37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sonesson</span>,  <span class="cit-name-given-names">A. K.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. H.</span> <span class="cit-name-surname">Meuwissen</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Testing strategies for genomic selection in aquaculture breeding programs.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">37</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-41-37</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DSonesson%26rft.auinit1%253DA.%2BK.%26rft.volume%253D41%26rft.spage%253D37%26rft.epage%253D37%26rft.atitle%253DTesting%2Bstrategies%2Bfor%2Bgenomic%2Bselection%2Bin%2Baquaculture%2Bbreeding%2Bprograms.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-37%26rft_id%253Dinfo%253Apmid%252F19566932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-37&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19566932&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.48" data-doi="10.1186/s12859-016-0906-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">X.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. H.</span> <span class="cit-name-surname">Mumm</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Method to represent the distribution of QTL additive and dominance effects associated with quantitative traits in computer simulation.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">73</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-016-0906-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D17%26rft.spage%253D73%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-016-0906-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-016-0906-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.49" data-doi="10.1534/genetics.116.199406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vitezica</span>,  <span class="cit-name-given-names">Z. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Legarra</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Toro</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Varona</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Orthogonal Estimates of Variances for Additive, Dominance and Epistatic Effects in Populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1307</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.199406</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.199406%26rft_id%253Dinfo%253Apmid%252F28522540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNi8zLzEyOTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzc4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.50" data-doi="10.1186/s12711-018-0437-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vitezica</span>,  <span class="cit-name-given-names">Z. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Reverter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Herring</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Legarra</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Dominance and epistatic genetic variances for litter size in pigs using genomic models.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">71</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-018-0437-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D50%26rft.spage%253D71%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-018-0437-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-018-0437-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.783.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lynch</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-source">Maintenance of Quantitative Genetic Variation in Evolution and Selection of Quantitative Traits</span>, <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.52" data-doi="10.1111/j.1439-0388.1991.tb00183.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. A. M.</span> <span class="cit-name-surname">van der Steen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H. J.</span> <span class="cit-name-surname">van der Werf</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. W.</span> <span class="cit-name-surname">Brascamp</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Relationship between purebred and crossbred parameters.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">253</span>–<span class="cit-lpage">261</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0388.1991.tb00183.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D108%26rft.spage%253D253%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.1991.tb00183.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.1991.tb00183.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.53" data-doi="10.1038/nrg3747"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>,  <span class="cit-name-given-names">W.-H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Hemani</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. S.</span> <span class="cit-name-surname">Haley</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Detecting epistasis in human complex traits.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">722</span>–<span class="cit-lpage">733</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3747</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D722%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3747%26rft_id%253Dinfo%253Apmid%252F25200660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25200660&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.54" data-doi="10.1017/S0016672310000649"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wellmann</span>,  <span class="cit-name-given-names">R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bennewitz</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The contribution of dominance to the understanding of quantitative genetic variation.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">154</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672310000649</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D93%26rft.spage%253D139%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000649&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wientjes</span>,  <span class="cit-name-given-names">Y. C. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P. L.</span> <span class="cit-name-surname">Calus</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">BOARD INVITED REVIEW: The purebred-crossbred correlation in pigs: A review of theory, estimates, and implications.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">3467</span>–<span class="cit-lpage">3478</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft.volume%253D95%26rft.spage%253D3467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.56" data-doi="10.1186/s12711-014-0086-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wientjes</span>,  <span class="cit-name-given-names">Y. C. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. F.</span> <span class="cit-name-surname">Veerkamp</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Bijma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Bovenhuis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Schrooten</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Empirical and deterministic accuracies of across-population genomic prediction.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">5</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-014-0086-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-014-0086-0%26rft_id%253Dinfo%253Apmid%252F25885467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-014-0086-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25885467&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.57" data-doi="10.1186/1297-9686-43-23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolc</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Arango</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Settar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Fulton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. P.</span> <span class="cit-name-surname">O’Sullivan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Persistence of accuracy of genomic estimated breeding values over generations in layer chickens.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">23</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-43-23</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DWolc%26rft.auinit1%253DA.%26rft.volume%253D43%26rft.spage%253D23%26rft.epage%253D23%26rft.atitle%253DPersistence%2Bof%2Baccuracy%2Bof%2Bgenomic%2Bestimated%2Bbreeding%2Bvalues%2Bover%2Bgenerations%2Bin%2Blayer%2Bchickens.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-23%26rft_id%253Dinfo%253Apmid%252F21693035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-23&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21693035&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.783.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zan</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.K.G.</span> <span class="cit-name-surname">Forsberg</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Ö.</span> <span class="cit-name-surname">Carlborg</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> On the Relationship Between High-Order Linkage Disequilibrium and Epistasis. G3 (Bethesda). 8: 2817–2824.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/2/783.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/2/769" title="Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/2/797" title="Interaction Between Induced and Natural Variation at oil yellow1 Delays Reproductive Maturity in Maize" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/2"><span>Volume 10 Issue 2, February 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/2.cover-source.jpg?itok=LSawHOSr" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/783.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YdosIG7PTc5z3P8YuyiSBwqJiEigbK9_xw40QzjrWJI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/2/783" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/2/783" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/2/783" class="active" data-icon-position="" data-hide-link-title="0">The Impact of Non-additive Effects on the Genetic Correlation Between Populations</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495560" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-NhULmtTT331fWgfqaUoegtv--azsBU_capgGWWrHGfU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/2/783.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/2/783" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-PXTiuzZKYeYdVSPg8q_cYkEOLJdzM_k5NKQHVOfcwcM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/2/783" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/2/783.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495560" id="node495560--2" data-pisa="ggg;10/2/783" data-pisa-master="ggg;g3.119.400663" data-apath="/ggg/10/2/783.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Impact of Non-additive Effects on the Genetic Correlation Between Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-8403-968X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pascal Duenk</span>, <span class="highwire-citation-author" data-delta="1">Piter Bijma</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-3213-704X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mario P. L. Calus</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-0681-2902" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yvonne C. J. Wientjes</span> and <span class="highwire-citation-author" data-delta="4">Julius H. J. van der Werf</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">783-795;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400663 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495560--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Pascal Duenk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Animal Breeding and Genomics, Wageningen University and Research, P.O. Box 338, 6700 AH Wageningen, the Netherlands and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pascal%2BDuenk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Duenk%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APascal%2BDuenk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8403-968X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pascal Duenk</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:pascal.duenk@wur.nl" class="" data-icon-position="" data-hide-link-title="0">pascal.duenk@wur.nl</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Piter Bijma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Animal Breeding and Genomics, Wageningen University and Research, P.O. Box 338, 6700 AH Wageningen, the Netherlands and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Piter%2BBijma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bijma%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APiter%2BBijma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mario P. L. Calus </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Animal Breeding and Genomics, Wageningen University and Research, P.O. Box 338, 6700 AH Wageningen, the Netherlands and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mario%2BP.%2BL.%2BCalus%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Calus%20MP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMario%2BP.%2BL.%2BCalus%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3213-704X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mario P. L. Calus</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yvonne C. J. Wientjes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Animal Breeding and Genomics, Wageningen University and Research, P.O. Box 338, 6700 AH Wageningen, the Netherlands and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yvonne%2BC.%2BJ.%2BWientjes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wientjes%20YC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYvonne%2BC.%2BJ.%2BWientjes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0681-2902" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yvonne C. J. Wientjes</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Julius H. J. van der Werf </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Environmental and Rural Science, University of New England, Armidale 2351 NSW, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julius%2BH.%2BJ.%2Bvan%2Bder%2BWerf%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20der%20Werf%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJulius%2BH.%2BJ.%2Bvan%2Bder%2BWerf%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/2/783" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/2/783&amp;title=The%20Impact%20of%20Non-additive%20Effects%20on%20the%20Genetic%20Correlation%20Between%20Populations" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/2/783&amp;title=The%20Impact%20of%20Non-additive%20Effects%20on%20the%20Genetic%20Correlation%20Between%20Populations" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/2/783&amp;title=The%20Impact%20of%20Non-additive%20Effects%20on%20the%20Genetic%20Correlation%20Between%20Populations" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/2/783&amp;text=The%20Impact%20of%20Non-additive%20Effects%20on%20the%20Genetic%20Correlation%20Between%20Populations" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/2/783&amp;title=The%20Impact%20of%20Non-additive%20Effects%20on%20the%20Genetic%20Correlation%20Between%20Populations" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/2/783&amp;t=The%20Impact%20of%20Non-additive%20Effects%20on%20the%20Genetic%20Correlation%20Between%20Populations" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/2/783&amp;title=The%20Impact%20of%20Non-additive%20Effects%20on%20the%20Genetic%20Correlation%20Between%20Populations" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/2/783&amp;title=The%20Impact%20of%20Non-additive%20Effects%20on%20the%20Genetic%20Correlation%20Between%20Populations" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495560/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495560" id="node495560--3" data-pisa="ggg;10/2/783" data-pisa-master="ggg;g3.119.400663" data-apath="/ggg/10/2/783.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Impact of Non-additive Effects on the Genetic Correlation Between Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-8403-968X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pascal Duenk</span>, <span class="highwire-citation-author" data-delta="1">Piter Bijma</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-3213-704X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mario P. L. Calus</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-0681-2902" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yvonne C. J. Wientjes</span> and <span class="highwire-citation-author" data-delta="4">Julius H. J. van der Werf</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">783-795;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400663 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495560--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Pascal Duenk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Animal Breeding and Genomics, Wageningen University and Research, P.O. Box 338, 6700 AH Wageningen, the Netherlands and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pascal%2BDuenk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Duenk%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APascal%2BDuenk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8403-968X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pascal Duenk</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:pascal.duenk@wur.nl" class="" data-icon-position="" data-hide-link-title="0">pascal.duenk@wur.nl</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Piter Bijma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Animal Breeding and Genomics, Wageningen University and Research, P.O. Box 338, 6700 AH Wageningen, the Netherlands and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Piter%2BBijma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bijma%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APiter%2BBijma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mario P. L. Calus </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Animal Breeding and Genomics, Wageningen University and Research, P.O. Box 338, 6700 AH Wageningen, the Netherlands and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mario%2BP.%2BL.%2BCalus%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Calus%20MP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMario%2BP.%2BL.%2BCalus%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3213-704X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mario P. L. Calus</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yvonne C. J. Wientjes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Animal Breeding and Genomics, Wageningen University and Research, P.O. Box 338, 6700 AH Wageningen, the Netherlands and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yvonne%2BC.%2BJ.%2BWientjes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wientjes%20YC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYvonne%2BC.%2BJ.%2BWientjes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0681-2902" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yvonne C. J. Wientjes</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Julius H. J. van der Werf </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Environmental and Rural Science, University of New England, Armidale 2351 NSW, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julius%2BH.%2BJ.%2Bvan%2Bder%2BWerf%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20der%20Werf%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJulius%2BH.%2BJ.%2Bvan%2Bder%2BWerf%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=783&amp;rft.epage=795&amp;rft.atitle=The%20Impact%20of%20Non-additive%20Effects%20on%20the%20Genetic%20Correlation%20Between%20Populations&amp;rft.volume=10&amp;rft.issue=2&amp;rft.date=2020-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Duenk%2C+Pascal&amp;rft.au=Bijma%2C+Piter&amp;rft.au=Calus%2C+Mario+P.+L.&amp;rft.au=Wientjes%2C+Yvonne+C.+J.&amp;rft.au=van+der+Werf%2C+Julius+H.+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495560/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495560/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495560/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495560/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495560/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495560/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495560/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495560/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495560/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495560/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495560/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495560/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/2/783" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature cited</a></li></ul></div></li><li class="odd"><a href="/content/10/2/783.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/2/783.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/2/783.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
