<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/2/811" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/2/811.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/2/811.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/2/811.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/2/811.atom" />
<meta name="HW.pisa" content="ggg;10/2/811" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Screen for Gene Paralogies Delineating Evolutionary Branching Order of Early Metazoa" />
<meta name="DC.Identifier" content="10.1534/g3.119.400951" />
<meta name="DC.Date" content="2020-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Erives and Fritzsch. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The evolutionary diversification of animals is one of Earth’s greatest marvels, yet its earliest steps are shrouded in mystery. Animals, the monophyletic clade known as Metazoa, evolved wildly divergent multicellular life strategies featuring ciliated sensory epithelia. In many lineages epithelial sensoria became coupled to increasingly complex nervous systems. Currently, different phylogenetic analyses of single-copy genes support mutually-exclusive possibilities that either Porifera or Ctenophora is sister to all other animals. Resolving this dilemma would advance the ecological and evolutionary understanding of the first animals and the evolution of nervous systems. Here we describe a comparative phylogenetic approach based on gene duplications. We computationally identify and analyze gene families with early metazoan duplications using an approach that mitigates apparent gene loss resulting from the miscalling of paralogs. In the transmembrane channel-like (TMC) family of mechano-transducing channels, we find ancient duplications that define separate clades for Eumetazoa (Placozoa + Cnidaria + Bilateria) vs. Ctenophora, and one duplication that is shared only by Eumetazoa and Porifera. In the Max-like protein X (MLX and MLXIP) family of bHLH-ZIP regulators of metabolism, we find that all major lineages from Eumetazoa and Porifera (sponges) share a duplicated gene pair that is sister to the single-copy gene maintained in Ctenophora. These results suggest a new avenue for deducing deep phylogeny by choosing rather than avoiding ancient gene paralogies." />
<meta name="DC.Contributor" content="Albert Erives" />
<meta name="DC.Contributor" content="Bernd Fritzsch" />
<meta name="article:published_time" content="2020-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Screen for Gene Paralogies Delineating Evolutionary Branching Order of Early Metazoa" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The evolutionary diversification of animals is one of Earth’s greatest marvels, yet its earliest steps are shrouded in mystery. Animals, the monophyletic clade known as Metazoa, evolved wildly divergent multicellular life strategies featuring ciliated sensory epithelia. In many lineages epithelial sensoria became coupled to increasingly complex nervous systems. Currently, different phylogenetic analyses of single-copy genes support mutually-exclusive possibilities that either Porifera or Ctenophora is sister to all other animals. Resolving this dilemma would advance the ecological and evolutionary understanding of the first animals and the evolution of nervous systems. Here we describe a comparative phylogenetic approach based on gene duplications. We computationally identify and analyze gene families with early metazoan duplications using an approach that mitigates apparent gene loss resulting from the miscalling of paralogs. In the transmembrane channel-like (TMC) family of mechano-transducing channels, we find ancient duplications that define separate clades for Eumetazoa (Placozoa + Cnidaria + Bilateria) &lt;i&gt;vs.&lt;/i&gt; Ctenophora, and one duplication that is shared only by Eumetazoa and Porifera. In the Max-like protein X (MLX and MLXIP) family of bHLH-ZIP regulators of metabolism, we find that all major lineages from Eumetazoa and Porifera (sponges) share a duplicated gene pair that is sister to the single-copy gene maintained in Ctenophora. These results suggest a new avenue for deducing deep phylogeny by choosing rather than avoiding ancient gene paralogies.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/02/01" />
<meta name="citation_mjid" content="ggg;10/2/811" />
<meta name="citation_id" content="10/2/811" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/2/811" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/2/811.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/2/811.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/2/811.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400951" />
<meta name="citation_pmid" content="31879283" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="811" />
<meta name="citation_lastpage" content="826" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Albert Erives" />
<meta name="citation_author_institution" content="Department of Biology, University of Iowa, Iowa City, IA, 52242" />
<meta name="citation_author_email" content="albert-erives@uiowa.edu" />
<meta name="citation_author_email" content="bernd-fritzsch@uiowa.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7107-5518" />
<meta name="citation_author" content="Bernd Fritzsch" />
<meta name="citation_author_institution" content="Department of Biology, University of Iowa, Iowa City, IA, 52242" />
<meta name="citation_author_email" content="albert-erives@uiowa.edu" />
<meta name="citation_author_email" content="bernd-fritzsch@uiowa.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4882-8398" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=D. Arendt;citation_title=The enigmatic xenopsins.;citation_pages=1-3;citation_volume=6;citation_year=2017;citation_pmid=28112643;citation_doi=10.7554/eLife.20010" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=D. Arendt;citation_author=J. M. Musser;citation_author=C. V. Baker;citation_author=A. Bergman;citation_author=C. Cepko;citation_title=The origin and evolution of cell types.;citation_pages=744-757;citation_volume=17;citation_year=2016;citation_pmid=27818507;citation_doi=10.1038/nrg.2016.127" />
<meta name="citation_reference" content="citation_journal_title=Science Translational Medicine;citation_journal_abbrev=Sci Transl Med;citation_author=C. Askew;citation_author=C. Rochat;citation_author=B. Pan;citation_author=Y. Asai;citation_author=H. Ahmed;citation_title=Tmc gene therapy restores auditory function in deaf mice;citation_pages=295ra108-295ra108;citation_volume=7;citation_year=2015;citation_issue=295;citation_pmid=26157030;citation_doi=10.1126/scitranslmed.aab1996" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=D. L. Ayres;citation_author=A. Darling;citation_author=D. J. Zwickl;citation_author=P. Beerli;citation_author=M. T. Holder;citation_title=BEAGLE: an application programming interface and high-performance computing library for statistical phylogenetics.;citation_volume=61;citation_year=2012;citation_pmid=21963610;citation_doi=10.1093/sysbio/syr100" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=A. Ballesteros;citation_author=C. Fenollar-Ferrer;citation_author=K. J. Swartz;citation_title=Structural relationship between the putative hair cell mechanotransduction channel TMC1 and TMEM16 proteins.;citation_volume=7;citation_year=2018;citation_doi=10.7554/eLife.38433" />
<meta name="citation_reference" content="citation_title=Genetics of mechanoreceptor evolution and development;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_author=C. Bökel;citation_author=S. Dass;citation_author=M. Wilsch-Bräuninger;citation_author=S. Roth;citation_title=Drosophila Cornichon acts as cargo receptor for ER export of the TGFalpha-like growth factor Gurken.;citation_volume=133;citation_year=2006;citation_pmid=16396907;citation_doi=10.1242/dev.02219" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=M. L. Borowiec;citation_author=E. K. Lee;citation_author=J. C. Chiu;citation_author=D. C. Plachetzki;citation_title=Extracting phylogenetic signal and accounting for bias in whole-genome data sets supports the Ctenophora as sister to remaining Metazoa.;citation_pages=987;citation_volume=16;citation_year=2015;citation_pmid=26596625;citation_doi=10.1186/s12864-015-2146-4" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. J. Brown;citation_author=M. D. Cole;citation_author=A. J. Erives;citation_title=Evolution of the holozoan ribosome biogenesis regulon.;citation_pages=442-442;citation_volume=9;citation_year=2008;citation_pmid=18816399;citation_doi=10.1186/1471-2164-9-442" />
<meta name="citation_reference" content="citation_journal_title=Journal of Cell Science;citation_journal_abbrev=J. Cell Sci.;citation_author=C. P. Castro;citation_author=D. Piscopo;citation_author=T. Nakagawa;citation_author=R. Derynck;citation_title=Cornichon regulates transport and secretion of TGF{alpha}-related proteins in metazoan cells;citation_pages=2454-2466;citation_volume=120;citation_year=2007;citation_issue=14;citation_pmid=17607000;citation_doi=10.1242/jcs.004200" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=J. C. Corbo;citation_author=A. Erives;citation_author=A. Di Gregorio;citation_author=A. Chang;citation_author=M. Levine;citation_title=Dorsoventral patterning of the vertebrate neural tube is conserved in a protochordate;citation_pages=2335-2344;citation_volume=124;citation_year=1997;citation_issue=12;citation_pmid=9199360" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=C. D. Cox;citation_author=C. Bae;citation_author=L. Ziegler;citation_author=S. Hartley;citation_author=V. Nikolova-Krstevski;citation_title=Removal of the mechanoprotective influence of the cytoskeleton reveals PIEZO1 is gated by bilayer tension.;citation_pages=10366;citation_volume=7;citation_year=2016;citation_pmid=26785635;citation_doi=10.1038/ncomms10366" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Physiol.;citation_author=C. D. Cox;citation_author=N. Bavi;citation_author=B. Martinac;citation_title=Bacterial mechanosensors.;citation_pages=71-93;citation_volume=80;citation_year=2018;citation_doi=10.1146/annurev-physiol-021317-121351" />
<meta name="citation_reference" content="citation_journal_title=Biophys. J.;citation_author=C. D. Cox;citation_author=N. Bavi;citation_author=B. Martinac;citation_title=Cytoskeleton-associated proteins modulate the tension sensitivity of Piezo1.;citation_pages=111a;citation_volume=114;citation_year=2018;citation_doi=10.1016/j.bpj.2017.11.641" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=P. Delmas;citation_author=B. Coste;citation_title=Mechano-gated ion channels in sensory systems.;citation_pages=278-284;citation_volume=155;citation_year=2013;citation_pmid=24120130;citation_doi=10.1016/j.cell.2013.09.026" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=S. Durinck;citation_author=Y. Moreau;citation_author=A. Kasprzyk;citation_author=S. Davis;citation_author=B. De Moor;citation_title=BioMart and Bioconductor: a powerful link between biological databases and microarray data analysis;citation_pages=3439-3440;citation_volume=21;citation_year=2005;citation_issue=16;citation_pmid=16082012;citation_doi=10.1093/bioinformatics/bti525" />
<meta name="citation_reference" content="citation_journal_title=Dev. Genes Evol.;citation_author=A. J. Erives;citation_title=Genes conserved in bilaterians but jointly lost with Myc during nematode evolution are enriched in cell proliferation and cell migration functions.;citation_pages=259-273;citation_volume=225;citation_year=2015;citation_pmid=26173873;citation_doi=10.1007/s00427-015-0508-1" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. J. Erives;citation_author=J. S. Fassler;citation_title=Metabolic and chaperone gene loss marks the origin of animals: evidence for Hsp104 and Hsp78 chaperones sharing mitochondrial enzymes as clients.;citation_pages=e0117192;citation_volume=10;citation_year=2015;citation_doi=10.1371/journal.pone.0117192" />
<meta name="citation_reference" content="Feuda, R., M. Dohrmann, W. Pett, H. Philippe, O. Rota-Stabelli, et al., 2017 Improved modeling of compositional heterogeneity supports sponges as sister to all other animals. Curr Biol 27: 3864–3870 e4. 10.1016/j.cub.2017.11.008" />
<meta name="citation_reference" content="citation_title=Proc. Natl. Acad. Sci. USA;citation_author=R. Feuda;citation_author=S. C. Hamilton;citation_author=J. O. McInerney;citation_author=D. Pisani;citation_title=Metazoan opsin evolution reveals a simple route to animal vision.;citation_volume=109;citation_year=2012;citation_pmid=23112152;citation_doi=10.1073/pnas.1204609109" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=W. M. Fitch;citation_author=E. Margoliash;citation_title=Construction of Phylogenetic Trees;citation_pages=279-284;citation_volume=155;citation_year=1967;citation_issue=3760;citation_pmid=5334057;citation_doi=10.1126/science.155.3760.279" />
<meta name="citation_reference" content="citation_journal_title=Neuroreport;citation_journal_abbrev=Neuroreport;citation_author=B. Fritzsch;citation_author=K. W. Beisel;citation_author=N. A. Bermingham;citation_title=Developmental evolutionary biology of the vertebrate ear: conserving mechanoelectric transduction and developmental pathways in diverging morphologies.;citation_pages=R35-R44;citation_volume=11;citation_year=2000;citation_issue=17;citation_pmid=11117521;citation_doi=10.1097/00001756-200011270-00013" />
<meta name="citation_reference" content="citation_journal_title=Dev. Biol.;citation_author=B. Fritzsch;citation_author=K. L. Elliott;citation_title=Gene, cell, and organ multiplication drives inner ear evolution.;citation_pages=3-15;citation_volume=431;citation_year=2017;citation_doi=10.1016/j.ydbio.2017.08.034" />
<meta name="citation_reference" content="citation_journal_title=Cell Tissue Res.;citation_author=B. Fritzsch;citation_author=I. Jahan;citation_author=N. Pan;citation_author=K. L. Elliott;citation_title=Evolving gene regulatory networks into cellular networks guiding adaptive behavior: an outline how single cells could have evolved into a centralized neurosensory system.;citation_pages=295-313;citation_volume=359;citation_year=2015;citation_pmid=25416504;citation_doi=10.1007/s00441-014-2043-1" />
<meta name="citation_reference" content="citation_journal_title=Brain research;citation_journal_abbrev=Brain Res;citation_author=B. Fritzsch;citation_author=S. Pauley;citation_author=K. W. Beisel;citation_title=Cells, molecules and morphogenesis: the making of the vertebrate ear.;citation_pages=151-171;citation_volume=1091;citation_year=2006;citation_issue=1;citation_pmid=16643865;citation_doi=10.1016/j.brainres.2006.02.078" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. Guo;citation_author=Y. Wang;citation_author=W. Zhang;citation_author=S. Meltzer;citation_author=D. Zanini;citation_title=Transmembrane channel-like (Tmc) gene regulates Drosophila larval locomotion.;citation_volume=113;citation_year=2016;citation_pmid=27298354;citation_doi=10.1073/pnas.1606537113" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Haider;citation_author=B. Ballester;citation_author=D. Smedley;citation_author=J. Zhang;citation_author=P. Rice;citation_title=BioMart Central Portal–unified access to biological data.;citation_volume=37;citation_year=2009;citation_pmid=19420058;citation_doi=10.1093/nar/gkp265" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Cell Biol.;citation_author=E. Havula;citation_author=V. Hietakangas;citation_title=Sugar sensing by ChREBP/Mondo-Mlx-new insight into downstream regulatory networks and integration of nutrient-derived signals.;citation_pages=89-96;citation_volume=51;citation_year=2018;citation_pmid=29278834;citation_doi=10.1016/j.ceb.2017.12.007" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=E. Havula;citation_author=M. Teesalu;citation_author=T. Hyotylainen;citation_author=H. Seppala;citation_author=K. Hasygar;citation_title=Mondo/ChREBP-Mlx-regulated transcriptional network is essential for dietary sugar tolerance in Drosophila.;citation_pages=e1003438;citation_volume=9;citation_year=2013;citation_pmid=23593032;citation_doi=10.1371/journal.pgen.1003438" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=G. Hemmrich;citation_author=T. C. Bosch;citation_title=Compagen, a comparative genomics platform for early branching metazoan animals, reveals early origins of genes regulating stem-cell differentiation.;citation_pages=1010-1018;citation_volume=30;citation_year=2008;citation_issue=10;citation_pmid=18800383;citation_doi=10.1002/bies.20813" />
<meta name="citation_reference" content="citation_journal_title=Neuron;citation_author=B. E. Herring;citation_author=Y. Shi;citation_author=Y. H. Suh;citation_author=C. Y. Zheng;citation_author=S. M. Blankenship;citation_title=CORNICHON PROTEINS DETERMINE THE SUBUNIT COMPOSITION OF SYNAPTIC AMPA RECEPTORS;citation_pages=1083-1096;citation_volume=77;citation_year=2013;citation_pmid=23522044;citation_doi=10.1016/j.neuron.2013.01.017" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.;citation_author=P. W. Holland;citation_author=F. Marlétaz;citation_author=I. Maeso;citation_author=T. L. Dunwell;citation_author=J. Paps;citation_title=New genes from old: asymmetric divergence of gene duplicates and the evolution of development.;citation_pages=20150480;citation_volume=372;citation_year=2017;citation_doi=10.1098/rstb.2015.0480" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=J. P. Huelsenbeck;citation_author=F. Ronquist;citation_title=MRBAYES: Bayesian inference of phylogenetic trees;citation_pages=754-755;citation_volume=17;citation_year=2001;citation_issue=8;citation_pmid=11524383;citation_doi=10.1093/bioinformatics/17.8.754" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. Iizuka;citation_author=R. K. Bruick;citation_author=G. Liang;citation_author=J. D. Horton;citation_author=K. Uyeda;citation_title=Deficiency of carbohydrate response element-binding protein (ChREBP) reduces lipogenesis as well as glycolysis.;citation_volume=101;citation_year=2004;citation_pmid=15118080;citation_doi=10.1073/pnas.0401516101" />
<meta name="citation_reference" content="citation_journal_title=Evodevo;citation_author=G. Jékely;citation_author=J. Paps;citation_author=C. Nielsen;citation_title=The phylogenetic position of ctenophores and the origin(s) of nervous systems.;citation_pages=1;citation_volume=6;citation_year=2015;citation_pmid=25905000;citation_doi=10.1186/2041-9139-6-1" />
<meta name="citation_reference" content="citation_title=Neuron;citation_author=Y. Jia;citation_author=Y. Zhao;citation_author=T. Kusakizako;citation_author=Y. Wang;citation_author=C. Pan;citation_title=TMC1 and TMC2 proteins are pore-forming subunits of mechanosensitive ion channels.;citation_year=2019;citation_doi=10.1016/j.neuron.2019.10.017" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. D. Jones;citation_author=I. Forn;citation_author=C. Gadelha;citation_author=M. J. Egan;citation_author=D. Bass;citation_title=Discovery of novel intermediate forms redefines the fungal tree of life.;citation_pages=200-203;citation_volume=474;citation_year=2011;citation_issue=7350;citation_pmid=21562490;citation_doi=10.1038/nature09984" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=G. Keresztes;citation_author=H. Mutai;citation_author=S. Heller;citation_title=TMC and EVER genes belong to a larger novel family, the TMC gene family encoding transmembrane proteins.;citation_pages=24-24;citation_volume=4;citation_year=2003;citation_issue=1;citation_pmid=12812529;citation_doi=10.1186/1471-2164-4-24" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Kumar;citation_author=G. Stecher;citation_author=K. Tamura;citation_title=MEGA7: Molecular evolutionary genetics analysis version 7.0 for bigger datasets.;citation_volume=33;citation_year=2016;citation_pmid=27004904;citation_doi=10.1093/molbev/msw054" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. Kung;citation_title=A possible unifying principle for mechanosensation.;citation_pages=647-654;citation_volume=436;citation_year=2005;citation_issue=7051;citation_pmid=16079835;citation_doi=10.1038/nature03896" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=A. B. Lassar;citation_author=R. L. Davis;citation_author=W. E. Wright;citation_author=T. Kadesch;citation_author=C. Murre;citation_title=Functional activity of myogenic HLH proteins requires hetero-oligomerization with E12/E47-like proteins in vivo.;citation_pages=305-315;citation_volume=66;citation_year=1991;citation_issue=2;citation_pmid=1649701;citation_doi=10.1016/0092-8674(91)90620-E" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Lynch;citation_author=M. O’Hely;citation_author=B. Walsh;citation_author=A. Force;citation_title=The Probability of Preservation of a Newly Arisen Gene Duplicate;citation_pages=1789-1804;citation_volume=159;citation_year=2001;citation_issue=4;citation_pmid=11779815" />
<meta name="citation_reference" content="citation_journal_title=Genesis (New York, N.Y. :  2000);citation_journal_abbrev=Genesis;citation_author=M. A. Michellod;citation_author=F. Forquignon;citation_author=P. Santamaria;citation_author=N. B. Randsholt;citation_title=Differential requirements for the neurogenic gene almondex during Drosophila melanogaster development.;citation_pages=113-122;citation_volume=37;citation_year=2003;citation_issue=3;citation_pmid=14595834;citation_doi=10.1002/gene.10233" />
<meta name="citation_reference" content="citation_journal_title=Brain research bulletin;citation_journal_abbrev=Brain Res Bull;citation_author=M. A. Michellod;citation_author=N. B. Randsholt;citation_title=Implication of the Drosophila beta-amyloid peptide binding-like protein AMX in Notch signaling during early neurogenesis.;citation_pages=305-309;citation_volume=75;citation_year=2008;citation_issue=2-4;citation_pmid=18331889;citation_doi=10.1016/j.brainresbull.2007.10.060" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=L. L. Moroz;citation_author=K. M. Kocot;citation_author=M. R. Citarella;citation_author=S. Dosung;citation_author=T. P. Norekian;citation_title=The ctenophore genome and the evolutionary origins of neural systems.;citation_pages=109-114;citation_volume=510;citation_year=2014;citation_pmid=24847885;citation_doi=10.1038/nature13400" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=C. Murre;citation_author=P. S. McCaw;citation_author=D. Baltimore;citation_title=A new DNA binding and dimerization motif in immunoglobulin enhancer binding, daughterless, MyoD, and myc proteins.;citation_pages=777-783;citation_volume=56;citation_year=1989;citation_issue=5;citation_pmid=2493990;citation_doi=10.1016/0092-8674(89)90682-X" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=C. Murre;citation_author=A. Voronova;citation_author=D. Baltimore;citation_title=B-cell- and myocyte-specific E2-box-binding factors contain E12/E47-like subunits.;citation_pages=1156-1160;citation_volume=11;citation_year=1991;citation_issue=2;citation_pmid=1990271;citation_doi=10.1128/MCB.11.2.1156" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=S. E. Murthy;citation_author=A. E. Dubin;citation_author=T. Whitwam;citation_author=S. Jojoa-Cruz;citation_author=S. M. Cahalan;citation_title=OSCA/TMEM63 are an evolutionarily conserved family of mechanically activated ion channels.;citation_pages=1-17;citation_volume=7;citation_year=2018;citation_pmid=30247123;citation_doi=10.7554/eLife.34110" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=L. A. Neuhold;citation_author=B. Wold;citation_title=HLH forced dimers: tethering MyoD to E47 generates a dominant positive myogenic factor insulated from negative regulation by Id.;citation_pages=1033-1042;citation_volume=74;citation_year=1993;citation_issue=6;citation_pmid=7691411;citation_doi=10.1016/0092-8674(93)90725-6" />
<meta name="citation_reference" content="citation_journal_title=Evolution &amp; development;citation_journal_abbrev=Evol Dev;citation_author=C. Nielsen;citation_title=Six major steps in animal evolution: are we derived sponge larvae?;citation_pages=241-257;citation_volume=10;citation_year=2008;citation_issue=2;citation_pmid=18315817;citation_doi=10.1111/j.1525-142X.2008.00231.x" />
<meta name="citation_reference" content="citation_journal_title=BMC Evol. Biol.;citation_author=C. Nielsen;citation_title=Life cycle evolution: Was the eumetazoan ancestor a holopelagic, planktotrophic gastraea?;citation_pages=171;citation_volume=13;citation_year=2013;citation_pmid=23957497;citation_doi=10.1186/1471-2148-13-171" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=E. Noutahi;citation_author=M. Semeria;citation_author=M. Lafond;citation_author=J. Seguin;citation_author=B. Boussau;citation_title=Efficient gene tree correction guided by genome evolution.;citation_pages=e0159559;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0159559" />
<meta name="citation_reference" content="citation_title=Evolution by gene duplication.;citation_author=S. Ohno;citation_year=1970;citation_doi=10.1007/978-3-642-86659-3" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=I. V. Pacentine;citation_author=T. Nicolson;citation_title=Subunits of the mechano-electrical transduction channel, Tmc1/2b, require Tmie to localize in zebrafish sensory hair cells.;citation_pages=e1007635;citation_volume=15;citation_year=2019;citation_doi=10.1371/journal.pgen.1007635" />
<meta name="citation_reference" content="Pan, B., N. Akyuz, X. P. Liu, Y. Asai, C. Nist-Lund, et al., 2018 TMC1 forms the pore of mechanosensory transduction channels in vertebrate inner ear hair cells. Neuron 99: 736–753 e6. 10.1016/j.neuron.2018.07.033" />
<meta name="citation_reference" content="citation_journal_title=Neuron;citation_author=B. Pan;citation_author=G. S. Geleoc;citation_author=Y. Asai;citation_author=G. C. Horwitz;citation_author=K. Kurima;citation_title=TMC1 and TMC2 are components of the mechanotransduction channel in hair cells of the mammalian inner ear.;citation_pages=504-515;citation_volume=79;citation_year=2013;citation_pmid=23871232;citation_doi=10.1016/j.neuron.2013.06.019" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=W. Pett;citation_author=M. Adamski;citation_author=M. Adamska;citation_author=W. R. Francis;citation_author=M. Eitel;citation_title=The role of homology and orthology in the phylogenomic analysis of metazoan gene content.;citation_pages=643-649;citation_volume=36;citation_year=2019;citation_doi=10.1093/molbev/msz013" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. Pisani;citation_author=W. Pett;citation_author=M. Dohrmann;citation_author=R. Feuda;citation_author=O. Rota-Stabelli;citation_title=Genomic data do not support comb jellies as the sister group to all other animals.;citation_volume=112;citation_year=2015;citation_pmid=26621703;citation_doi=10.1073/pnas.1518127112" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=N. H. Putnam;citation_author=M. Srivastava;citation_author=U. Hellsten;citation_author=B. Dirks;citation_author=J. Chapman;citation_title=Sea Anemone Genome Reveals Ancestral Eumetazoan Gene Repertoire and Genomic Organization;citation_pages=86-94;citation_volume=317;citation_year=2007;citation_issue=5834;citation_pmid=17615350;citation_doi=10.1126/science.1139158" />
<meta name="citation_reference" content="citation_journal_title=Front. Cell. Neurosci.;citation_author=X. Qiu;citation_author=U. Muller;citation_title=Mechanically gated ion channels in mammalian hair cells.;citation_volume=12;citation_year=2018;citation_doi=10.3389/fncel.2018.00100" />
<meta name="citation_reference" content="citation_journal_title=Neuron;citation_author=S. S. Ranade;citation_author=R. Syeda;citation_author=A. Patapoutian;citation_title=Mechanically activated ion channels.;citation_pages=1162-1179;citation_volume=87;citation_year=2015;citation_doi=10.1016/j.neuron.2015.08.032" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=D. J. Richter;citation_author=P. Fozouni;citation_author=M. B. Eisen;citation_author=N. King;citation_title=Gene family innovation, conservation and loss on the animal stem lineage.;citation_volume=7;citation_year=2018;citation_doi=10.7554/eLife.34226" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=F. Ronquist;citation_author=J. P. Huelsenbeck;citation_title=MrBayes 3: Bayesian phylogenetic inference under mixed models;citation_pages=1572-1574;citation_volume=19;citation_year=2003;citation_issue=12;citation_pmid=12912839;citation_doi=10.1093/bioinformatics/btg180" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=F. Ronquist;citation_author=M. Teslenko;citation_author=P. van der Mark;citation_author=D. L. Ayres;citation_author=A. Darling;citation_title=MrBayes 3.2: efficient Bayesian phylogenetic inference and model choice across a large model space.;citation_volume=61;citation_year=2012;citation_pmid=22357727;citation_doi=10.1093/sysbio/sys029" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. F. Ryan;citation_author=K. Pang;citation_author=C. E. Schnitzler;citation_author=A. D. Nguyen;citation_author=R. T. Moreland;citation_title=The Genome of the Ctenophore Mnemiopsis leidyi and Its Implications for Cell Type Evolution;citation_pages=1242592-1242592;citation_volume=342;citation_year=2013;citation_issue=6164;citation_pmid=24337300;citation_doi=10.1126/science.1242592" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=C. L. Sans;citation_author=D. J. Satterwhite;citation_author=C. A. Stoltzman;citation_author=K. T. Breen;citation_author=D. E. Ayer;citation_title=MondoA-Mlx Heterodimers Are Candidate Sensors of Cellular Energy Status: Mitochondrial Localization and Direct Regulation of Glycolysis;citation_pages=4863-4871;citation_volume=26;citation_year=2006;citation_issue=13;citation_pmid=16782875;citation_doi=10.1128/MCB.00657-05" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=P. Sarkar;citation_author=E. Bosneaga;citation_author=M. Auer;citation_title=Plant cell walls throughout evolution: Towards a molecular understanding of their design principles.;citation_volume=60;citation_year=2009;citation_pmid=19687127;citation_doi=10.1093/jxb/erp245" />
<meta name="citation_reference" content="citation_journal_title=Traffic;citation_author=E. Sauvageau;citation_author=M. D. Rochdi;citation_author=M. Oueslati;citation_author=F. F. Hamdan;citation_author=Y. Percherancier;citation_title=CNIH4 interacts with newly synthesized GPCR and controls their export from the endoplasmic reticulum.;citation_pages=383-400;citation_volume=15;citation_year=2014;citation_pmid=24405750;citation_doi=10.1111/tra.12148" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Schwenk;citation_author=N. Harmel;citation_author=G. Zolles;citation_author=W. Bildl;citation_author=A. Kulik;citation_title=Functional Proteomics Identify Cornichon Proteins as Auxiliary Subunits of AMPA Receptors;citation_pages=1313-1319;citation_volume=323;citation_year=2009;citation_issue=5919;citation_pmid=19265014;citation_doi=10.1126/science.1167852" />
<meta name="citation_reference" content="citation_journal_title=Nat. Ecol. Evol.;citation_author=A. Sebé-Pedrós;citation_author=E. Chomsky;citation_author=K. Pang;citation_author=D. Lara-Astiaso;citation_author=F. Gaiti;citation_title=Early metazoan cell type diversity and the evolution of multicellular gene regulation.;citation_pages=1176-1188;citation_volume=2;citation_year=2018;citation_pmid=29942020;citation_doi=10.1038/s41559-018-0575-6" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_journal_abbrev=Genetica;citation_author=M. P. Shannon;citation_title=Characterization of the female-sterile mutant almondex of Drosophila melanogaster.;citation_pages=244-256;citation_volume=43;citation_year=1972;citation_issue=2;citation_pmid=4625164;citation_doi=10.1007/BF00123632" />
<meta name="citation_reference" content="citation_journal_title=The Journal of experimental zoology;citation_journal_abbrev=J Exp Zool;citation_author=M. P. Shannon;citation_title=The development of eggs produced by the female-sterile mutant almondex of Drosophila melanogaster.;citation_pages=383-400;citation_volume=183;citation_year=1973;citation_issue=3;citation_pmid=4633255;citation_doi=10.1002/jez.1401830312" />
<meta name="citation_reference" content="citation_journal_title=Nat. Ecol. Evol.;citation_author=X. X. Shen;citation_author=C. T. Hittinger;citation_author=A. Rokas;citation_title=Contentious relationships in phylogenomic studies can be driven by a handful of genes.;citation_pages=126;citation_volume=1;citation_year=2017;citation_doi=10.1038/s41559-017-0126" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=S. B. Shibata;citation_author=P. T. Ranum;citation_author=H. Moteki;citation_author=B. Pan;citation_author=A. T. Goodwin;citation_title=RNA interference prevents autosomal-dominant hearing loss.;citation_pages=1101-1113;citation_volume=98;citation_year=2016;citation_doi=10.1016/j.ajhg.2016.03.028" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=P. Simion;citation_author=H. Philippe;citation_author=D. Baurain;citation_author=M. Jager;citation_author=D. J. Richter;citation_title=A large and consistent phylogenomic dataset supports sponges as the sister group to all other animals.;citation_pages=958-967;citation_volume=27;citation_year=2017;citation_pmid=28318975;citation_doi=10.1016/j.cub.2017.02.031" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=D. Smedley;citation_author=S. Haider;citation_author=B. Ballester;citation_author=R. Holland;citation_author=D. London;citation_title=BioMart--biological queries made easy.;citation_pages=22-22;citation_volume=10;citation_year=2009;citation_pmid=19144180;citation_doi=10.1186/1471-2164-10-22" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=S. Sogabe;citation_author=W. L. Hatleberg;citation_author=K. M. Kocot;citation_author=T. E. Say;citation_author=D. Stoupin;citation_title=Pluripotency and the origin of animal multicellularity.;citation_pages=519-522;citation_volume=570;citation_year=2019;citation_doi=10.1038/s41586-019-1290-4" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=I. Sørensen;citation_author=D. Domozych;citation_author=W. G. T. Willats;citation_title=How have plant cell walls evolved?;citation_volume=153;citation_year=2010;citation_pmid=20431088;citation_doi=10.1104/pp.110.154427" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Srivastava;citation_author=E. Begovic;citation_author=J. Chapman;citation_author=N. H. Putnam;citation_author=U. Hellsten;citation_title=The Trichoplax genome and the nature of placozoans.;citation_pages=955-960;citation_volume=454;citation_year=2008;citation_issue=7207;citation_pmid=18719581;citation_doi=10.1038/nature07191" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Srivastava;citation_author=O. Simakov;citation_author=J. Chapman;citation_author=B. Fahey;citation_author=M. E. Gauthier;citation_title=The Amphimedon queenslandica genome and the evolution of animal complexity.;citation_pages=720-726;citation_volume=466;citation_year=2010;citation_issue=7307;citation_pmid=20686567;citation_doi=10.1038/nature09201" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Biol.;citation_author=A. J. Swafford;citation_author=T. H. Oakley;citation_title=Multimodal sensorimotor system in unicellular zoospores of a fungus.;citation_volume=221;citation_year=2018;citation_pmid=29170260;citation_doi=10.1242/jeb.163196" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=J. S. Taylor;citation_author=J. Raes;citation_title=Duplication and divergence: the evolution of new genes and old ideas.;citation_pages=615-643;citation_volume=38;citation_year=2004;citation_pmid=15568988;citation_doi=10.1146/annurev.genet.38.072902.092831" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. J. Vilella;citation_author=J. Severin;citation_author=A. Ureta-Vidal;citation_author=L. Heng;citation_author=R. Durbin;citation_title=EnsemblCompara GeneTrees: Complete, duplication-aware phylogenetic trees in vertebrates.;citation_volume=19;citation_year=2009;citation_pmid=19029536;citation_doi=10.1101/gr.073585.107" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. B. Walsh;citation_title=How often do duplicated genes evolve new functions?;citation_pages=421-428;citation_volume=139;citation_year=1995;citation_issue=1;citation_pmid=7705642" />
<meta name="citation_reference" content="citation_journal_title=Neuron;citation_author=X. Wang;citation_author=G. Li;citation_author=J. Liu;citation_author=J. Liu;citation_author=X. Z. Xu;citation_title=TMC-1 mediates alkaline sensation in C. elegans through nociceptive neurons.;citation_pages=146-154;citation_volume=91;citation_year=2016;citation_pmid=27321925;citation_doi=10.1016/j.neuron.2016.05.023" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=N. V. Whelan;citation_author=K. M. Kocot;citation_author=L. L. Moroz;citation_author=K. M. Halanych;citation_title=Error, signal, and the placement of Ctenophora sister to all other animals.;citation_volume=112;citation_year=2015;citation_pmid=25902535;citation_doi=10.1073/pnas.1503453112" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=S. Whelan;citation_author=N. Goldman;citation_title=A General Empirical Model of Protein Evolution Derived from Multiple Protein Families Using a Maximum-Likelihood Approach;citation_pages=691-699;citation_volume=18;citation_year=2001;citation_issue=5;citation_pmid=11319253;citation_doi=10.1093/oxfordjournals.molbev.a003851" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=C. R. Woese;citation_author=G. E. Fox;citation_title=Phylogenetic structure of the prokaryotic domain: The primary kingdoms;citation_pages=5088-5090;citation_volume=74;citation_year=1977;citation_issue=11;citation_pmid=270744;citation_doi=10.1073/pnas.74.11.5088" />
<meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.;citation_author=Z. Wu;citation_author=N. Grillet;citation_author=B. Zhao;citation_author=C. Cunningham;citation_author=S. Harkins-Perry;citation_title=Mechanosensory hair cells express two molecularly distinct mechanotransduction channels.;citation_pages=24-33;citation_volume=20;citation_year=2017;citation_doi=10.1038/nn.4449" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. Yamashita;citation_author=M. Takenoshita;citation_author=M. Sakurai;citation_author=R. K. Bruick;citation_author=W. J. Henzel;citation_title=A glucose-responsive transcription factor that regulates carbohydrate metabolism in the liver.;citation_volume=98;citation_year=2001;citation_pmid=11470916;citation_doi=10.1073/pnas.161284298" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=H. Yoshimura;citation_author=S. B. Shibata;citation_author=P. T. Ranum;citation_author=R. J. Smith;citation_title=Enhanced viral-mediated cochlear gene delivery in adult mice by combining canal fenestration with round window membrane inoculation.;citation_pages=2980;citation_volume=8;citation_year=2018;citation_doi=10.1038/s41598-018-21233-z" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. L. Young;citation_author=D. Diolaiti;citation_author=M. Conacci-Sorrell;citation_author=I. Ruiz-Trillo;citation_author=R. N. Eisenman;citation_title=Premetazoan ancestry of the Myc-Max network.;citation_volume=28;citation_year=2011;citation_pmid=21571926;citation_doi=10.1093/molbev/msr132" />
<meta name="citation_reference" content="citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=Y. Zhao;citation_author=J. Vinther;citation_author=L. A. Parry;citation_author=F. Wei;citation_author=E. Green;citation_title=Cambrian sessile, suspension feeding stem-group ctenophores and evolution of the comb jelly body plan.;citation_pages=1112-1125.e2;citation_volume=29;citation_year=2019;citation_doi=10.1016/j.cub.2019.02.036" />
<meta name="citation_reference" content="citation_journal_title=Journal of Theoretical Biology;citation_journal_abbrev=Journal of Theoretical Biology;citation_author=E. Zuckerkandl;citation_author=L. Pauling;citation_title=Molecules as documents of evolutionary history.;citation_pages=357-366;citation_volume=8;citation_year=1965;citation_issue=2;citation_pmid=5876245;citation_doi=10.1016/0022-5193(65)90083-4" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Screen for Gene Paralogies Delineating Evolutionary Branching Order of Early Metazoa" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
<meta name="twitter:description" content="The evolutionary diversification of animals is one of Earth’s greatest marvels, yet its earliest steps are shrouded in mystery. Animals, the monophyletic clade known as Metazoa, evolved wildly divergent multicellular life strategies featuring ciliated sensory epithelia. In many lineages epithelial sensoria became coupled to increasingly complex nervous systems. Currently, different phylogenetic analyses of single-copy genes support mutually-exclusive possibilities that either Porifera or Ctenophora is sister to all other animals. Resolving this dilemma would advance the ecological and evolutionary understanding of the first animals and the evolution of nervous systems. Here we describe a comparative phylogenetic approach based on gene duplications. We computationally identify and analyze gene families with early metazoan duplications using an approach that mitigates apparent gene loss resulting from the miscalling of paralogs. In the transmembrane channel-like (TMC) family of mechano-transducing channels, we find ancient duplications that define separate clades for Eumetazoa (Placozoa + Cnidaria + Bilateria) vs. Ctenophora, and one duplication that is shared only by Eumetazoa and Porifera. In the Max-like protein X (MLX and MLXIP) family of bHLH-ZIP regulators of metabolism, we find that all major lineages from Eumetazoa and Porifera (sponges) share a duplicated gene pair that is sister to the single-copy gene maintained in Ctenophora. These results suggest a new avenue for deducing deep phylogeny by choosing rather than avoiding ancient gene paralogies." />
<meta name="og-title" property="og:title" content="A Screen for Gene Paralogies Delineating Evolutionary Branching Order of Early Metazoa" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/2/811" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The evolutionary diversification of animals is one of Earth’s greatest marvels, yet its earliest steps are shrouded in mystery. Animals, the monophyletic clade known as Metazoa, evolved wildly divergent multicellular life strategies featuring ciliated sensory epithelia. In many lineages epithelial sensoria became coupled to increasingly complex nervous systems. Currently, different phylogenetic analyses of single-copy genes support mutually-exclusive possibilities that either Porifera or Ctenophora is sister to all other animals. Resolving this dilemma would advance the ecological and evolutionary understanding of the first animals and the evolution of nervous systems. Here we describe a comparative phylogenetic approach based on gene duplications. We computationally identify and analyze gene families with early metazoan duplications using an approach that mitigates apparent gene loss resulting from the miscalling of paralogs. In the transmembrane channel-like (TMC) family of mechano-transducing channels, we find ancient duplications that define separate clades for Eumetazoa (Placozoa + Cnidaria + Bilateria) vs. Ctenophora, and one duplication that is shared only by Eumetazoa and Porifera. In the Max-like protein X (MLX and MLXIP) family of bHLH-ZIP regulators of metabolism, we find that all major lineages from Eumetazoa and Porifera (sponges) share a duplicated gene pair that is sister to the single-copy gene maintained in Ctenophora. These results suggest a new avenue for deducing deep phylogeny by choosing rather than avoiding ancient gene paralogies." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
    <title>A Screen for Gene Paralogies Delineating Evolutionary Branching Order of Early Metazoa | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"bBS0-JaIW-XpJwjDZZOioGkLjg47L_gNYhagzu3EExE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495499","apath":"\/ggg\/10\/2\/811.atom","pisa":"ggg;10\/2\/811","ac":{"\/ggg\/10\/2\/811.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/2\/811.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/2\/811"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/2\/811"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/2\/811":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022I9dRT4GZt4mvTcIPpNAiAIau4zlItQKiJHznyZJkby+X8alqddfmhIhOa3E710aIrGC34zUDI7iIgiYk3651pXDTd\\\\\\\/So5+gO98QL0\\\\\\\/ErrnIloaFRnNQPkKN9QT\\\\\\\/e0Ef8AOlXTonF4ZCOvHGbk50Bbgu9qrxTGO7Qye\\\\\\\/TxqzbqwWJpJjpSNnOYKK+\\\\\\\/0ZTaAUuu\\\\\\\/nuOHik+GI1KHo8rg8A5Gh2gfw4oBCu9joxV68KdwPNlFMnH41R8c7tJqNUZAv6iYEdsrkhE\\\\\\\/q61FhDmt5tjoVb74vY3KXGS5q86xBtLnSx\\\\\\\/hjSACmmWAg\\\\\\\/Sb4V6dxpsVNQccQCENdPvUrQcD4+LFFcoWqHJmq7j3+qvwEODdTrnBaWiZTLvGP1+mEY4xjeUKxBi7AEu5WaP3XDgE63T+zCK1THZpunxLmCtojoOvRPXK6fYuuVesee+aAILe3zavuM9HeqrgUOE2H4Xqwbm7a8fKrfVnO2DXYGNv4I\\\\\\\/88hTVQ1L+kNDJqsiLgkOwwchSmc3Pn3QHhiTHXnAIRbfkZXvLiFYzrFRJFRlZQ66rFb2MEd\\\\\\\/Lk8+paLDH1Hj\\\\\\\/fdsP2u35rRRnFFGjLOK1PwNxovvBJGerClQ97Xvt2VlUxYxuo0jRLpW1K4iDa54SubKcaLO92b\\\\\\\/3aLeXdo4S1U2vcigNVGBrbEkIoZvaVjA12lCGDnERi7CeMUQYE4WdbS0kywudDjUhIWs+GzZbkMQcme+c+BNUP6lORMrspLeT9OMXrqstbsad12UzS2NjSbbX32+NFLmAzibephOeV6vw6eMAvHnTHjAGpo14AfbfytGRg6GiuEkPsVwdCyKrXdRWTSwy9nnUrCME6GD4bOWSU2sN8MBCujEUgT6Pc24nm2jvzspUQcBD1MYYwUEMJLdUxyvSy7B1akKTHHnAuqcpTL++R9eHfkgDwk1bHbUDqSLliO4c7KoGK3U5\\\\\\\/i0GvxbDCo+0UUQjeR0qhgDPwO6BYAYpW\\\\\\\/XJXmjGj48oX287plblUTuUWev6ngrMAak4w8s93x0+aMO+fuHd\\\\\\\/nuRYZxGtmXVQXnduoqhdvRv18vIyYukYg7dSoaMXizo24gWCGeJ9AEnyCUcNgZblK3W0roIpbllyJopA8sx9lShsQ6NmRK7RDzTitdJt0UpAEvcoblsUo1W6nwLu2MbUGONejCRdh3gtlpvvmIS7uia0TQriiqmU99ncJF3piT+cr6QBO8Q64NRwjgb1FqkVrTIcHt2RJDWWUDm1GN7Ml\\\\\\\/FOtyab+QVwx+PGt1QqCXj\\\\\\\/Qkj6G\\\u0022,\\\u0022iv\\\u0022:\\\u0022MABbWK5M15\\\\\\\/e0OB1TIamxg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022eaf18b64a3b9e853356b7836b2b50bc472951c16da53fae2894f80b4d0eda8c9\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022sYKRqiWIEF47GRuPz+Z\\\\\\\/YYOk9DUfkIYLbTVPhsfc76mdyiskyULJFQ9dYzqt9AvToEJ5YIeRh8gUzw6m0lG8zsPT30EzNjZlgWDIW4ynIOv5YmouSed6kh\\\\\\\/j3gorwYieix9U+Yas\\\\\\\/pTeRDMjhEef8Z2heltv7URpruSsfLR2YtUU2nw9BItxKEtwA7aCyF5jg4fQbBaadCucdkI1x2v1lMH+XmQnB53q9TXCiQZXkUpUOFsvzLMcsgMStJ7HPGIdGOJ3XCL2sAi50ighXdius76oweuDeH8AErCma8mXmSX1zUfSoaWXp4wFNYYbGmIqB20aOpIw4TnQf1N+QQflFAoFCB6ixqL2\\\\\\\/rTB3gpfpoefJm2+Kl\\\\\\\/RD9t0TFyRcOHt2EHBC45OhkssutzUzscBJXbhNKqq5mi7Hvy6hEh0ij6up1\\\\\\\/naZNRQM\\\\\\\/EDtJ+4UyXA4gZ29pt\\\\\\\/L4dgoVR\\\\\\\/Gii8SFdvTQk4gc+36CMMMPZrwz1oHqPsRBqVzaFgxaO0BTyT\\\\\\\/WcDeRxk4cWYpkymxnUV7vjZtbG4dgbWa05p\\\\\\\/sdlpfWfCNomZqWwl8S6qn\\\\\\\/GBpp0GjFs+DY61b+JlheWSxcq6w7zTJd6k0C8HTgV3P7YDPWRGwLDw68H92fXjEtLA5ISVXZ0dYGYk6GTlfn7CBFhyZAqacNocXElhhq99v6NssHin5QgNNjM2KwJRSe2zjvYv7r6DEKmTqRIPbQlMD0iMqd\\\\\\\/N0Ivtyzf9shEgcy8e6dqGN\\\\\\\/fjSG\\\\\\\/ropixt5KAIp5oLrrS3B55q\\\\\\\/JLM1ut3\\\\\\\/GCiT85y1YLnEncLoDqdyYDcforFws2IAacO\\\\\\\/5dfZhl4Nbjjqmdx0JHH\\\\\\\/sStMl8HoTEy0+3ZDPaWdTX2fQRxLOu4DJH3MKefyrOOEhhUMFF7Y4R\\\\\\\/tuV85cYo43\\\\\\\/inx7k1i1B5G946Hx2n4eD1vyYKi7gTUpuEXTMUctCTYwDRORrtJEl6r+P\\\\\\\/vyY2R3eQaAZZDHTQKuQCPWSgJGqO5oTW\\\\\\\/P+HksYTHsv8SwNda0dX7TrOfRz5YH9UnA84TAoYu0AYR4M\\\\\\\/eJ6ZnNrn4+BYqWkgzc6cBJnEvjocY\\\\\\\/4c1yz0E+CqS4IAYt8D50mqFUOl7yngzuso+iLdOsMbEM4FaPY0fY8a1z+seSKX6Cc5wtRAU5ml0DIou3RfIUZrlryIKN35u6zJSldLTibhMB5AcdjOdNeCXqfTgUYW0BJOGVHBMz9kuujTGdQ7SxR3ihw6Gcf\\\\\\\/kJXBgsE3\\\\\\\/2NkSr2mfSf9u5k6m8qUH+HEQRasYea5aOaslHp34mblDb1RzsasOeg2pHu8HLdAnG+d\\\\\\\/q9BZy\\\\\\\/S\\\\\\\/jv7izetNtzzQgvt5BVbGhcWuf82Mkt2leSpwnVV\\\\\\\/XI9ZPJbR2LcIyXyx7E2+90yh7AZpqjNuMMpbrz0S1J4h5bcN4Asm+u84I30RLUT6ib3Ai1AieTdj67K29T\\\\\\\/v8LcaJcII2K0Zq1Uu8fUaoNCITJKfwuxjlWsIJc9FhTYqr5PGzUM2hmhX1RIhCA42wUH0d\\\\\\\/UXdfktycxfK0XDovytGh8OqZJ2NxpN4ZHJ2bJi+84PCx2SkVza3cnYS5zk\\\\\\\/4A8xy1Nd4bwk1IBupUwqqll4Ni0BngPOK7FUMRvrSOia8gaEVGnj1gXsklNxzGWA9SMuzSIhDHOxIMfrrMIq\\\\\\\/rVh47KMxyznxhqU2kLOHQc+PedV7eCOUkpzPK2hFRmnE4QWzpbXnzRzQG93aENjPlibIwmf1NgaFXv92wHlyBg9DgazDzCff4MSXGNfoFo6ef\\\\\\\/bOR28XbCyNXDYR1rSrQyK5HkbZ+Bdcfi+VqTr32jyhhtiYVdjNfUUExm6zGYOqksGR679Lewo\\\\\\\/cSINH2IDtPhUWDA6elBCnuSiysuhQVwwhpU4MwLOXOwbOeUKnGJ7eF81q39aF6YC0TxqxRnH1zUSkw7WqPTwAvw1ampop5Ad3MzXBc5NAzMAKwypcXx7pb5fgZt4nl6YBWu+X49aeSDcdIkE42yPAkgfF6PB1m23OpngorhEEhDGhQ8KGopdNDRMZMLDTzjf3STi92uq4ZXCGVeQrB6bSPuf4yqfCjHwc5Hp5SexqMwJgAcckHoH12bQb34fVLOnTsSw\\\\\\\/PczjZI\\\\\\\/jotNJuqk\\\\\\\/rKFr2ZHPEa9JULlv2djxZMpf3XUw5EQFwEWx06zaW5tsqHaWaRlSzQQVCxMEXgZCeNF902en\\\\\\\/QykYH8seHiw0uStAGIZSw1032UXFXCpJMy1nlKn2VWBM8nGiYOx2rJYfFJKeFv9kD2kAwGrMnHlkqXoVDUbv1vM7q2InXzItRlANPW8bqqGzAeGz1Yz8b8dmPpAUlLp0wI025caIiDe8sZSGBf3UJEy8CfoHtZw5EqBCtCAyx5vkcUsc6GMQakwujeBZAd8xZ7o66fDwCk5\\\\\\\/OGSMgDqt+UloI4IoBMgKp73SvZcBIoKOZtILAP2gfhl868iAOBe794qZ88fpWRcLtfjID3lHsdUEmpfRRlXlz3MlYqT+E5qodS4TnF1zA38R7zkIZTHuVphkFVsZLHXc5g3Ql2h78jnBkriF4RA1OcHXJ1WV24OM9DzAuB2YMLqcaWYW2UsfnbkJySI7XYW+3qq7dsY\\\\\\\/\\\\\\\/3E2PqhkMKbBfEYRXNtWS+x6M8lh3dPolUhkJCzcYkudVDe22GrkppvXlTUqXNlGuWCI4vXy+KHN7bpZpxbggNe2ZvYGKkXS9HozxrzjYfaS7Vb4VxlXaF0z\\\\\\\/zGr2B1W8bX\\\\\\\/KK1YBFbONLITSjxb0L3IhEQmaqYjPT4irYbPjGTr5nU1vHlquykBTsMkEDP4r5hfbOIH0HzbtEokGvZdKJ+aoMp+zBOtBwd2FDWVe328SVb74ctfykf9w7spRKpWCLxDzyy1H1Hdf5vNZ5QmvRpcxRiHf3GjYxHnC\\\\\\\/d5\\\\\\\/cRW\\\\\\\/sKQjUOt73OcUKHOwRyuZjQCjOBRn6frwAEz68MBosC7rBoeu0DMDymh949A3u1NGrcwsFRg0ygTGhF8RdQoZz9liZxvXHNG+mzUlwj6bJM0ONOJYIuKU\\\\\\\/+uEQw0NqYYHqUvrckREWqiAx21RNsHpU4nv5oJ2BPquchpRtO89fVD4zdJ5n4E5ZM3MvHCXCDRmAfP6855wv5R\\\\\\\/yAR08BouEqAD1j6tm4Djmr4tFfwParwkrqmGT8g4avJPqR9xUkF35jQ3XT\\\\\\\/1YLoBOYizX+u5oT7srBp0LqmVq8mzOrIKm8m6UqRG1\\\\\\\/wYYnl\\\\\\\/LFknx5zZ\\\\\\\/MYFQBA33oto\\\\\\\/8EMPQWrfxFqeKmXz0AfaniUbM03UhgKjJ0+CrFrvLGE0IFbjykCHTX2BzEbXe7KYCtXZli4bJv705qBKPyxLw08y5639ymEhb6glq\\\\\\\/VigliON06r65i7qvWbv5O5bssFgzAo2fVvqVmmPOeIXHmeUaktZItTtd5VybJXIm1nBozUy2AHh8VTApKFa8omLRBzjZCRGXq3bsEkpm\\\\\\\/Wsaj\\\\\\\/0BSb1pENO8rEW+daJKB18N5iYS1w4tUCTMxA6mKppyFqVeST1bN2USllO7DTwJxEiUQyMooToTow+fQLZO\\\\\\\/BdpC9IYojq0+6K72ZYPYUI+fG\\\\\\\/TSNM0oTF0d5wuhEnO\\\\\\\/mNbGy8rwa5zzZHWAL0OaVVv0HgoqsyX9hNaLdFJ517O4CYNDDiy037p7n7ChW+URlGXat55FZqO0XKDFkF7m5ioJlwdBKku6IRTWYJGGEW1mSOSgPkRv0AV4d9BPuev4ACANmalyT3JhC6hFik88B\\\\\\\/AVRuoZ+b22CzZfgNvtUs1eZ2j57qw\\\\\\\/WrB8dp6VcoT8cAoRPKxFpPPMgVOgY3tZWX3sAXnjJqijsbEmphv2fD8GDiQfqzZEEhbgVXxzYBa2sXopbE1fALFqCgY8eXmLUG01GbPJmmcqVm8yoiOJnLhrLJkaf\\\\\\\/3TbEqmrqIKor0t3TYqEyMlf0v9roXdaLdr5dkB8vNK0aAxbo0LPDHAsfK59aQRb+DIsOv\\\\\\\/RuEBY2bqbvHEVGLQ7Zs3+xi56qED\\\\\\\/XA5EpmEIHpugcGSebhroi8z9wRLZ0zX2eYKbWKe+Go9HAnixOFva\\\\\\\/L8bDHXw\\\\\\\/VNPThNi0ay4MTlsJ5DU2MM0ZhG9CjsmKQsoJ3gLJl0eSJ8lW2rt26G5ggWJl4lrG0a2XTerDTy59u7IVFM86yirYNRr49PwG5OCZadJ0llsXwlpmBkHZRfV\\\\\\\/SvRpfenWYq6+S8coql4Z6XY8h2GaQg7N1pHlHbh3q0170pImNxzlpeNmBcyoOTtOoxS1UkKeYk+r5S4\\\\\\\/bF\\\\\\\/tb1FQseYdqv5Zu046iQgs2FhQXP7g4\\\\\\\/K602\\\\\\\/XXn3E6eNOPtrH1q5iLwUMWSvSnmfWfeI\\\\\\\/ExCtM0IbaDPUlHQ0eASPRG45SoeubzHHt+fP8aes2yrrMBY8E2357jPtug6mTKNGlbaK4IPo35TX7mlJ4\\\\\\\/NcnSXQhXCvAtHPTuNP86Thp13wJbwyplexlZOPgS0Fww7FVVISgNiUMGJWJrC08HJmPvEx6QDVSG1FET9+ZLtr3MwRap0dIa4uvT\\\\\\\/V\\\\\\\/E8S+9Iv9++sDmAqCgVGWjDjUs4zH\\\\\\\/iUwbFJqKQZ\\\\\\\/uZGf3OLVbIDgos6NdLLaaD2vXgKg8ePKRk\\\\\\\/qNQmEUv6fbDoodURUYogGIQxOx2RmXbFxF4oHh0XZ999PGDdfN21s+thSCEq0g0FQKQQsqoYwPloGWfjnufOEdxifkSCtXr0heaihYfFLUoDWnyDaYMcMG4FALTjFbEEQ2D9LxfSGgz6q3vINrbZNyDHpUPJVAI7lnKO9kDiuP3I4sI4OjfU+n1plYd8lQCbB9b07Qpzo12AjtseGjkqsBtl5datOyCd5OWZhya1MowVMILENxY6VswdAqoMGgy2\\\\\\\/Hv5XI4aYzMXv77vl\\\\\\\/AwCeJsGB4VlXPwKspHBXAYvjgzExXZywI7ZN8O2hVWb\\\\\\\/NM7dFJ3T36OcR+eWb2T+oXXnIidJ6EKAimloOa+zYMXem83d+pZbT+fhnTuNyoENwVC8JJtS7sp0bNuk\\\\\\\/kT4McKpnuCS2HSRNRUxxXk2lQZuf0jrz\\\\\\\/QMAacUMi\\\\\\\/HoScrtDqJVMkfwjGrmcx8Nvdb8XzyG+YkBrYQtUFRwa67KJoPYUm36MIKhFlWv\\\\\\\/Y6FM\\\\\\\/EMBvgpriPwBnxEWRdzlpCRvimz+anTmupr\\\\\\\/Xq11pcmt9RZPxC9hd5WV02AVL65kEUNfpYKgUARbV9whr2yASNWgq3K72g72+nffGnfLp4wLWKBw9bplAUZhCYmpHJ8aG6B6XIKuttbLBbq80x+lTDgla9KOaivbLypYpdV96bnM8jDyMR2M1IX8ooJ8WmeDXnmzPO0WeFLRTABS1YOyjtLd4zvS67XS7nCQb1oVamRdZEOPvCi7ZlF1Kz5\\\\\\\/kJrcxLnEh\\\\\\\/WKyaN0fxR\\\\\\\/T2Vbm2\\\\\\\/uVNjNZ4sMHPvuH33WCSOXKiBBcNAhqb1ehkA490KOSQRnvxuOxo9J57rRX9ukpHkG4rn7Ow9cc2a2w+Uub+BMLEXWVQ2lYf55tumMrcIi3A6OfcqiqXhKWIJbpkHsL1UhIsSIWhuT7PsMawX7vaBSLny+39Ft8l4zJg\\\\\\\/IORxeAGEo9nkpamGC8eQQMzBhPbJqR4I2jjFwVOshWgSd1tSKjVzEjltr0dVSe5UKOIAXvYgoA97iGa1gfIJdGP6NMLypk89BCyohbLWpCCmKrRvqhIagbkCvIIieJpbOipyUOvSlO3c57rDmG2A9UjmYtuWxIfUxt2oz+bAcNVZWvZFxNTE9T4qxdEkrhnQnj5kNJHA8dsC2NWsWwhQMvQ8\\\\\\\/EAw8Ijsm0R2sC\\\\\\\/2SRoVYkn4RzlL2Ew2CQvQcf6JAXyQyqLGVnghof8Unj\\\\\\\/wq2LVdaek8Pm1Is7J8zBNnCftmMhkoXDiAgwUJX1Wt6QOi7jncPiVszoMdPoaXM+Ml5Hrq90RiyZm7jSpBOk6gWqgriXVonNa2Ktc9PQXMoat5RMOxZNIETFpAGm1foHrXFTaOl8Junzia1eXfa0GRcEbzOFY3ck8J78rond2NwVwh4OfNyWCWvAOrVHeBQbPg4O39rtaRXDW\\\\\\\/PH+fUyF6bYLwWsosKZ67mf7RAF38U8DhtnyClaAtPqxw7z6WLKir4RHg2vFADavlicxgdqvRdmxCgHSB1u97HLion1\\\\\\\/17GZehqGVZQmgm2uc3P5meiTYcjSe8OtnjKwil5PMLi9bKji7KzIyuBItZbQbwZDI7O8AiD4qmE9hp3Tg5gBHrg0iHgKyws5WVduuYDszokodaPVLbmRE33+R0duAOR8TIIkPZDYWdV+N75eza4cSGD\\\\\\\/v7LRCRNsDmdb3J2Titm65MmWJ9e+csiGcfmqv7utY6tGVxXyyr9zollHVMUODEOXUd5WXBXIM7+ilK9t8aBYIhUVr9BoPcYhQC8Qt3RB04Axe6sdcDtS2\\\\\\\/KTs2XCmCClNt2b5A41140emhXePgqlrETzib5ea6viSSggYdDlTCUDCdUt9IWVlnhs5QPbqAIYKP3KSgsAZPqpf1YIYciOX31gjCHMCfY\\\\\\\/9wgEWTTeESc6L9GsOxJk91rihFkl8bpEo\\\\\\\/zq\\\\\\\/kfwqOFgh5gICpiJ4ZcOLy+Fc8MW1vHf\\\\\\\/r2JaFP1+pTzrCMxeM5T6u2YGc8\\\\\\\/vf3eHF2mJG0q8L1+Fa\\\\\\\/GpVEJHiBtGzMY7tPQ3Pl2W6HeCoNkWeXnTfCOPv2WF7J3\\\\\\\/p9fX9AwXGDfZa70kWzyYlnhG1xZ1YFrObsG1r7mZ3zbssCMZHwGOay2YLFPUGb8kZZIl94sSiqzaGu3Fdy5ySIUwI1d2cjqy3ckXNxqIGDXy4\\\\\\\/9eg3Qpl1vFr9uBNsCHl4ZQi3d9F0K8Za+YLNtuUxrVkOONokJlGBO1pzg8QYGEfuQ2L765Bqzen5xKSnNJYxiittd0wxtndUVplp1RPjIFDAkjZ4iHT8Ue8CSuceKUhBQ4KDZ+FmxMvCeokdjgpqteHjEsEHyhe3GkrkAMQ+7zgW91VSVXG2UHSK6paUDtSkl3WLtqQBt6zeNk2OyBPodjtzBU+qOciAi5Z8fR6EqIYZGPphKwWUu3BrrRQXp\\\\\\\/gKlKmCC0sFYFcj5drB\\\\\\\/42f00pCvdQC1ETkgBXUfowIwlGFX8f0zIPAT44y4RM9jw3fD\\\\\\\/KmuV4Mn\\\\\\\/AOSEDrW330Y48XDnKWe282fAfHOhYyucuaQ9KsnOjEol4arH12tiwggfbwI4SGlVk2+rWdfLXrZlfvHENQ6Ui5JoWnsP1\\\\\\\/XLNnA+9jURq4lJP9xESY0tgRcK7TK\\\\\\\/vP62y+Xgz\\\\\\\/zFAFd06aj8Ut10TTwTmH6QJmY3bj5KXFBkH06OWmxlrj+fkn5S4RBoDuKLEb4IQHkFrkWZKoG9UW9qvsuqoqWvQwvhko+xZeatKSS1U4GAVuT1XHk5O0k6R92xcFCnVkI4lXrRNcszTfIAaz4XXCzkLfiNXybrlomSdhOWUQ6bVwRkZLY9lRfeHWqNxgdh\\\\\\\/jW9Yoz+YLLcqUVRuRPI4XG4zBydisM+kjrilFn7WYK7G\\\\\\\/l5OUYwqoGlxKQWn4B1\\\\\\\/Olm1HqnD3nxOndfi5NfoqIXtuQ\\\\\\\/H7WAxHETgDaYHUQ7C+zO0sIRkHYkMOgREN7ZluxT2cQmAxih8HINUO7qT0VqetXAXRDJrr20k9dBPDBAp0NSPxLjEIiY7ehGKr7xjx3hrH4gQ0GV5AKgUOZkUjcaQHK2XANLP4+LfrSDufGUlZmkpJxkUQ3R4+XKsEDCRaT3SFnQxaiSmQx0LCStAEOSpKNzMMfbIQkNrlRH\\\\\\\/6MKI5NqXS\\\\\\\/7oYiIsygTelBpIbQkWrA6iwrdiRhdQkFQHYTA74v+FsKeVv3DsTb7ONkmPDeGQPBCQjP9G7PjoH2OaKfjzBpDIm7F7VJpThBPTOjNSQNFSp9cE85s3P6HFon\\\\\\\/9P9z9r1TPKeV5tJCsNxEHRhN68aEVneg\\\\\\\/G1GU5jd\\\\\\\/FDeo0TAgJlr2aIjyQbHKRmxwLdR7zQY22KqTT16kQyeFo8n8jbpYTYLUUxHcUI+qvlmRaq81OfK9fQGuraKzkCU\\\\\\\/GyvD+EUPQGnqB7bb+gzfZi+F9T6WlUckid2TNdRvgw3BUB\\\\\\\/w7VkLlmDoiGUGeML1Gxte2Z3c2u\\\\\\\/4JESdvQQuf5AzBzN17OD0mZjH0iqm0Otws07QBUgDVKHjsFvHkK0QCHKZ+qQiHGnLSdKcSJfyW26Yy3fWUZx\\\\\\\/3Z2SXr1AtxApjRgz23n1M2ENQUimP5rg+sFKWo1p7Z9VQ0iKYrd80GZJhuXQPM4bgehoQi7YCWS46ykGRPVEqp2By9w\\\\\\\/WlG\\\\\\\/VPzLrdUhr+dXB01TQJr1Qg52vMDhCn5pAPUpKpMAqz5kbUvJkAcucmSda1rKIm\\\\\\\/SFX8N\\\\\\\/Mv8ewmN6jZlTU47U3NORigrs11cAovGUBOsS+DbIpX8YdDLntlvw2\\\\\\\/ln\\\\\\\/zQatw1+vocWOPpuaMhz13qqmvoQtiJGpNuqPB1sZ7kZued1sdQVxv6yyi1OU45rwNVQ5vt2qslDxqQMjqsLNKOmqmDcnh70jd3UcHbfZitAtJvL2I0gk\\\\\\\/LzMZyB\\\\\\\/yliAfmlBgc3vS7x8jvseEqJsLT59ADOXRcmDMKj4uiEK8vcCO1ZoV0OcSYUsik1qaAyhlwyUld8iNGknNArp7oc9lHgtUANJVgKIMafVuM3j+Gn+PNnotp6Jed6Kva\\\\\\\/dYyPjuZNgzQQun7mTqUzQs8Fz\\\\\\\/N8thotWJ4gYwDlx0v5QYEcccLE0jq4F2AH7feIp++Y\\\\\\\/ElkEk8oZ9\\\\\\\/rIPT17UC0loY3MmIo6paA32rLLrm3iHZpNkm1CYghq6MCbIb+eC4SF+ZogO\\\\\\\/GpvEO+S3803jMB05SYhqGf\\\\\\\/lk0GvFLYJo+Eb8AMDBTnVC2yTnoUNKl3xm+n7r+agRn08yh6NbX5KnzkEe6xXNm8PCmkiB2vMbTTG4zl6B9kbgtauAAEqt04txPyuLFTBUZz4ZqdepddSgi+r6dozjybnrzsclRUDpCmdJ9YDOmHHQ1A6XV8sElRhH8YwDHZ12AJbZ0ku1JshFVpwI49Y6UGZmN0gs\\\\\\\/48OAkonlS\\\\\\\/hNHAO5aBt5b8DpvIQC4HpMVD3aSlx\\\\\\\/KEGfiLSBKrSnIwkSV6Hh5AYxXMQKNkoMdEpO8Rkx0TSakX64MM8ccSIaZTGAZb84sYYLVun9+ASIa439WQYcxPttEpQ7ve+QA6abJRC1ga+MPOBwQ0rVcMeLDKFW0KRZ0vF\\\\\\\/YfjfyNoc2B5AMbnKIiOEcVgQktYPU+M23JvynJ+2URoEdM\\\\\\\/ScEImf3EJDl5a+mw2uRimL9\\\\\\\/vUs4OHKt0AqTqpVV0UaSLq+bpKGkNHlzv7+ZYDCYixZ96S6irFil5Ie0qOz+pdvZV8dO9ZyNdHj9Mk9CUHiCwFHcDCOMvVjJDsy59Lvs+Y4y2THgQc0qsCdXt5Z\\\\\\\/4vZ5I1OvgndNa4rgu72Wtd+KvqcRnyDNtWTblhW8wV0lSK7eiMGRtfUpoWJHi7xuJDY7KUJJmpfN6UuMVuXwR+nnlCjTBCmCkk0fxPybz8RbmSCgpApnf+kunjKL\\\\\\\/SgTYuV\\\\\\\/nUo1jCjCY9EpATfg6ZJLEaeMsfc1ATsgsmeObU\\\\\\\/GllQ2MOcxcFqpG5NM6umsLdMhyi6CsDYI13UhxREzGQ4y+DSD5rTRGEq8aUPbVDt5JDFvcArpva6+U3C9wGli\\\\\\\/5kV23K6x+Q\\\\\\\/byARqaFgBzoxxvQTBGqJZT\\\\\\\/oexFjXx2txYopQ08rVIAUaMHW95Obd28mubUcm6XfUfoNUni519SoLZHfp2Q7yz1nidoV47FqJEeEsSCu9DB1oYizhwMDydUeDBhAdzK5yY1TFBHuZhdb4VE7CmdOrByPruM7DFObkCmxS6xvcRLTLu7ufiLfoUrJkZe4p7p2lwgKgzckjm\\\\\\\/O25LI+qAspesnAI4gzVfY1QnaK6Q670Lc81YBxegXJ97ZIrryeUf1TK\\\\\\\/Zd3kbDjPDeLjt9HI3cZ8bhej7omtfzoef7AnckP7r6fjy0xSetTB+D6aRSZQhYmdyQqcGmc956wD+5w\\\\\\\/Se3G8D97ZkbaliA2kOhPmnPfXS8nmIZf8ojrPKVsEa8cM4QcbyAAV2ZCy1axU1sB+95rFxdBs6k3d0N3j8Q6iFnbqlz\\\\\\\/AL9m1fqGqNwLxtoPLZy\\\\\\\/toEcIUO0B8DFlG641e89j21RF8AQxZ7Jl5lM84Do7x+CQBL3D8kyvpvo0RxswxmFMxNsGNrT7sdMRxrYuCh53LpkOBMGQ5BnjtYCO09wJ7ItLM9N2rB1HoFOmsvuGzN60LNHhOOA149YgSFH3GXpV+VwHO7PvCIKwgf2cuIodjZfBUSvzxwDCpMs6oOi+98XFLlym67dWamWIUat7gOwEwB4nTzBMpxgLjoyY\\\\\\\/tFGTLa7sQXVlUzBOEFjmfrZ+lMFtzOgAFwwu\\\\\\\/4WnUx07OYNeC5\\\\\\\/ZIsjjSREqGokkHW1JMlIQbjxWMkEUs\\\\\\\/0yYdeQS\\\\\\\/GqVu74yDaW8MxWCTZdfoidh0Gso08YO37lNegVwQH\\\\\\\/FZQL5a9BdbcV92h0NL7OlZdbA3MNnDct5y0bkF5hyIaYdFda5UoBb0CacdzhMtlSDHoYzp+8Cv0tS2KBBl65eQZZYjCLICrOAAZuAe9a2U0z5+dZJQezfHK4wb78QI7GhPgUQSkJ2EUFzqnDRyyXECLNm0QWA7ai57GcpjxYP1zlUGv1B3ax05ZF4YkFX84aWPcFwnV1bE2BQMxfJYXoRAKa\\\\\\\/cYe1RYcKdHibfW2Bvb91ZYQDzPbHklZ654hAaVMjrner02KY07sO1uU9jo1OclkFj8N5rAT2sJqoaJ2\\\\\\\/dhLV9W938FutPtyBXD2aip2eeWiHikVVieIVR5mZCqHBOOOgH3bCMr6BAJcW0bZuk8y9vFIjPAXZrJ51KCE+xUNz8zkBxte4PFQtMAfIBJhF6GPmEtuQuejhK\\\\\\\/EdaY7hZsnXNF2AD5OEL\\\\\\\/Z\\\\\\\/cvbktOLrYqtqPyNvfYVUtmHKz\\\\\\\/4CVe50N0dSFtSc8ThjaNL8Dh9wNX4GJWG9Q9\\\\\\\/90cVOZoyOTjdfLa9Jaq5jtP+ApAfmwTktySwHjyJsw\\\\\\\/qbcuweKWma3y0gSi1\\\\\\\/3mTvlN03rMxT4YwxcSXMCrSDRGQhc0JSNxT3UScdd5BLTGfmTpN5pGzSkYZV5f5RWn7CkbfgO+r1xVI1ENcTC6xN40DM4Vzq3v785d9gbViiXHs51oXCFZ7hAvMblm30+7amEN+FsnVR2Dx0lJduVAVt\\\\\\\/\\\\\\\/voWcTpCzWkWDKtAuo\\\\\\\/CPa2n2bW\\\\\\\/C+CAmh22XT5jbDSuHXfSat8MpEYlkrk0N6SNDctztG9W1st7\\\\\\\/SZJUxuGS52DPGPtkhST\\\\\\\/4bEAN3Uc\\\\\\\/rQ6jz94jCZSxTiMAE+aU2RJNIPJR11KXodpC1cIgyFcn\\\\\\\/bkySkk2yf+dTBuTZb19kvyIgo2pgvua2YNUzzb3BYbECqgpZzEDhp+x2mkKNOhHO+mEE\\\\\\\/lK0JB1sTpu9U+mCzb3UZeJNhk4jDixoyK7J6Dwe2Np5lz4LhOL2AxDzlEwspSnBDdf0GpAVsB1R\\\\\\\/ZvjuqlWzaaqdh\\\\\\\/N5oT7Dmn+XhA3y8HfcyOSZPP8vF1SW21EGxMX+irfORELWf1+CrlOhw\\\\\\\/pmGzRISj1jXgie2eQ5qUOzNRr5RnWWnHUjqAdPkRnpA5wUQUuO9okVTsN3vTthNtdihxa1fxx3E+z0rRSfKn9P2SQlCX9U5haHL+0iMDDbw0y4bZH4LOFNtaDKcstZTzprI8pjUeH4Yfw2mFhZPyQLpv7MEMm1c53ovIN7O4VP3vfpKc9poXAI4BxpZ5KEJgQqU+YHzOIvQNp5bomTDg7pLH8wb5hEIHPNXaMqtfTmHJes7Mxn3aLEynvWABmKRMzhYO400yazDuBCpdVdzh6udfE4H+hJDZfQ+wOFRtAATWpEY1\\\\\\\/\\\\\\\/tvjLXr8olfLKk+M2cLwvltXEkmBYZcltIFF2L5A0OvttPG+kpJlHAF4yrci77BoPOcMcO3DhfucsGXyhIqtx2LG0SDaM9IFaKoPFAjO2sVZjXY28JEULKvvDTb2tnPZ9nM+LttT0BVkSo8i2K80I7x36YI8OolhspcISvq9fzNs\\\\\\\/bBhJ\\\\\\\/f\\\\\\\/dRcJQ4GFiTb5ESHVI\\\\\\\/+8P\\\\\\\/KDLI1tL5ZhkF0oBmVjgQ4Ralah1TrzaWR\\\\\\\/1ecgBFhPXhu8hr5EGpVYVUogoH6M1v6Wi2mzVCsJLr4D9Ee1SdUgER7qE3I37udW+J2oW4LrGsbj9r6vB8KsaQZ9WhUccUXbbtSxezUHuQO\\\\\\\/kjRR3E2ImhpMMSYR2ADx2jZ0Tr6a0gflWKByL6KiW1EKD0qWM3EqdsehQbS0g1t6GFnh+mhUBW2lhIVvzILuQa7FdmJVmBRI+QbKogUWaVeXhPMhtHwcuPGZccSbFl1AVDSnomAk6CaCM6lJLr58z68tDayaDNrAtZIIGtdGiFCO8aTlOWMtUOupdVEg7miBfLSgRXOVJcQ3p1y+XEHT2kQDyd1+lcnQYURUo\\\\\\\/fhTsv9u\\\\\\\/GTMZC3YGeInLD7+uY86yqUHYlWa9KW9OWb6dSorBYyeG0GSDfWJBdr6eYUs0qY+8I6dE81ukyIerOCCEd6zBFklssDzql54J7Yocgz2\\\\\\\/3kIeyt4kIpi2fRdUcz2jEoKRSxm9oTNq1ejeowa+ypapmL8UxB9oi5+oZcg3aVTGyg0snVhRtiLHxzjU1R5VaFVmEuwiXrHyNbMXMjoHo5ztc0SbV\\\\\\\/4d5ucvzL3gMsXsLZj9wPNya0R2svN6AZw3AqqQ+XFY0RWiG0vAHukqySm0ikncNOXNI0MtJ5b0DLeq\\\\\\\/Q8nGsmAkOad1u65q7nEyI91Yx2aYwiphoQ\\\\\\\/O9egnR3KS\\\\\\\/rDKN1bhjEqAEDpM\\\\\\\/E4Cn0TtjvtgEXHT+FZE7mWKqli4yXYDU6Ag9jJpy5csg\\\\\\\/45k+A1pEcRs8yauWGeT7iqbHDP72y12LogHVFtXovJDrByi0o+eorhTmtGWFwHQ1hY2ur4W9+9waaU3r\\\\\\\/cgNivY4q1VKBmY\\\\\\\/B40WvAJZe19WwnYAGqTTHh69eSF8AuvANNCdu2wiJiunBUGaR\\\\\\\/i3N3sHHaC6uo\\\\\\\/Xa4x3KKEBhNIfFWAoXvCUOuUjLcaHdBT4\\\\\\\/STRLqIAqlWrSbbwdB5OrXzuUWUHP7eB4P6p8QvL+soZ1KT\\\\\\\/1blj+sAlkwrvcQcYCPbItpFcRuLLuH4OSNz2QXBODqI9oF5RdOKRLz4hiylr1Z+fOLEHrNJEZOEQ154CUCD2e5SSNO+vA8Ho8rvhqGtfrAR3ostmDAkSUsqv+xI\\\\\\\/q\\\\\\\/TMVOVJM\\\\\\\/OIRYQvw9wncYbk2dMUbXP2QCRohHa\\\\\\\/W52FceXf\\\\\\\/aujz+5HCQhz2Jz84rjdPH0xzlXoMh3Y6eL0wFZ5ANRYbExuYJ7veGoo65Wm2IHjpWSrtudBpMiEKyL6UjVlIeJ\\\\\\\/9m1knHLEJPRxOWGGftWjImmozOab+ZlzI1QcEC0ZS6oBtaLQKObI5plylyHN+MmpRKAluzN8gqyqn86JHUp5tJjSW6QDZaWQq7Gauzh1wSJ8xvGa7Cx7x8oajcbmhen+oWS4OBkHGMPJxZM1QBK+BKe9jCON7MzIIH7\\\\\\\/Jqh2zdjPIryZV4MFizKjagb9aY5v1gSFXS9QtoUlcBnB6i0ClyrXtsjFDPMZMvjpXSYK\\\\\\\/U0TjooBvhoN+3evZJZomc2pPXoOJile40yn8nXH7G\\\\\\\/SXoGsX2XmhM4XUlqVK40+BhZkAdIFBd7rRzmrJUOks1rvVygYAtsn9vZHHDt4PTCAhO2mJieeUKCc0Yuo+ucjWcuy8nJyASI+SpzfAvZe+amUnmtl1KHyRnP2Wqa13yV6ixJyC1cZ6YW1t32uffbS3FF0c474KS\\\\\\\/u\\\\\\\/16uKdOL4HPWPYTo7LHE\\\\\\\/S0Q3rUdpdNd7NhAGFlVwMLnBl0AEPs9B0zepmx24sa1JcYp39HSTRUe+YVYHXBvjy3\\\\\\\/b0Mt8wSsZ0CzJRSndmWXndMIVnLdpx3Z0fGBQ\\\\\\\/VsQiXu1lmsOzeDpfkIcC4MDxj5ww7DHt\\\\\\\/8GYdm52KNMAszwtVYjDKnOjjgQjC4gXX6XSrAmSVkIWLxaTXIgJynHlH5oe+dT6p+AR4\\\\\\\/rsXjnJ19b3oLyX3VegQr7wympGKSu+QhTylOjHn+Er3JTg8oguL2oTLOIEHKVs0UAyfnmJ+PsXTGpVF8tuy9Izhae8Zl3bG3pSxPJ3vW0N1tJ33R+4j0LZ1o9tqn7bcyRf4LOWltRb8jPRKPG9cOY\\\\\\\/OZ81YdQtQU55WbkBmq7ktjgFuq7Z2jPd3awABWgT1yH60j1EK+uWHvv27BWtuvB65ps7Q5QpanyztqVd42dMwB3Ui2sb1PNpgfQTBm90kEkzB2g00U3k5DvTBMfD0iSH7jQ0Hrns+FvraeQe1PQ74rz7YX+h0zVtvk1AH3veV66pl45qyWGOp1LBUOsQBRjLGMjyDC24FaozfoREwSDl83YR9S4OrpZ9z5T7ZpD8\\\\\\\/nxSurIicsQQZyGKkfJvm\\\\\\\/IrHQ5bfrz0MtUkm3gXwI6PCddoFRGPqS291lyR2PwpY+YW4dLYLJE0\\\\\\\/JV17wMaYKazxvhx85ffdvrNizFXvMgndBYiPrtWzUQckhoNYNLpTFMgvPpIIKyysNfzKtaBUUR0Rf+jeG6KOLbvOPizLzr6qua7rip7\\\\\\\/W7BqMqlfwdXz98arcEx9lZezY3FTSazwyVNwEjtRWLP8S1jL77LsZLEex++vXESfOQv7hq4gamLeM3HIwpoe9qjZmiCK31kg1JnmhP+2+VB44mCYMdAAsMzv2XxzOaqKOH+bciBKzpQbhQyHejyi83Re9W836ABA9IHXepoVAPwuKrglZISIM0wFpi8icOZX9AWx4XEKbB4lo5etWJfeK014NnQd02Ngl4mn0YK9sii56ZJxbk+RLQ+EXL8Sh0mqIj3chQvbdEAR41sG+sdEkGmHrBf8DUrE\\\\\\\/3sgbNzn465DWz2iUKQzWADFW8oE1Yn+JKOI8pR5b95SoAVKX5JKsZ+zHSJ64Q\\\\\\\/l81uaJk5LCHa5Y8eLFKfCeuDiwNjz9TukfJk8ndCXxaz\\\\\\\/h1\\\\\\\/0newx5cgcXw\\\\\\\/uWBDTlwnL\\\\\\\/ripeSy17aVkh8TuSx+A4Yv4Z9uRYyWL4+9Qp3aVRBpDS4NKQ1VVl0WkOMlda3EppnO+09kyrDY\\\\\\\/OqPXgk5qSb\\\\\\\/foyRNQN0GDf6ToYZCh2pCuv6ymsZuj9Mc6TJOfZYpeMhs0RjydYNklZAhWhZ3wf1dx3Aze2GE2wNL0k2C7gGcobarrd+tkgOCDQ6dYh6dzjoQCfvmQve0fguldvadO8tN5VCbXFKN5PCazVI37bifv4nifvE2IUFXHBtX9NCoZKfCq3XVprSe+oHma2w8lCF4JZqLH\\\\\\\/oR9VmTgAg\\\\\\\/tcrUNs5dVop99P9QStVPvh2NYDOJWOPeOAKfbWjUvyn14ma65bB9+VY70E93A7vGrai4sLAu9quxYJoKepsNL7NS7RpaKPR+Ne7z2rJtyF+KglnoXGXEOG\\\\\\\/545Ia6cRcpy7SuphpnmF6GLIjWkOdb6htwdeliTJWL5Ycqwxpx7XJQoBRiXo2rJqHzjdTbL1hb1+nmpkcnyisOrMA6kbWyK1fdi4QIJOxKRUcjM5jAG2WFVhqt2sc7YNbl2D9dVB7ScqiPDPdKYJzoNWfB1PP5ibnFgtFRsfPL3MQJHXItF4BmWQC35gwos7iAbJ40C74+3IiMHQf8EQoxMtcziHsORajsIXkYwwXhkFwd\\\\\\\/eO5Jg3m99QkV1SIwZp3IxOhpM4tVQ3IqlpqmKSZmJ7JxEk1vddl3qEdBWHXPPvZ3uB5tvrHwtskY7rf+goSo2HCwbMCEyQ5lJAgObrZzshYVLRmNbySkwkd2+NyySmIx5r3pWYdBvXEs6ove2Acqmnbpnt17JdlOZ+C2ZibZkfmQQQAXw9nRgT9ikIqfmeeYi6E1gmwYsZzRzZLG+uIz5xLFu9LC2eilrFnvKoBOsxzRtAJnUeIM9PCseNK5u0jwl7vmbVoKlVZl40Vp1mWqv1Q0RklWX1vEfQCQbkwIPcm2YB8D7949rZ3DcApZAK2CsCrl3U7cUMILa88dfKO8WHDEa1P9+45OJo8geila\\\\\\\/evuyEXepZxo3eLGVCOLm5m8+Mm9971B9j05B0k8Y1K6aRL5JrZ+YCujIEgK4srmz5lN2Vq7D8V+qwZRpXIzM2Kco08pVPnjo+aBZX8aILJHxs5gqjKJ3C8s4L1deLkW63anlMqYmaJX41lJ0iBaOQplnYJbBcWjTp4I2XdFW\\\\\\\/6S\\\\\\\/zQ4+vJVJv633TFE\\\\\\\/HBUaCCUYVuZ4sg7e7oqlVTGhwEsOLUdArmrxTvNR+ynTNhVxMToCfOMFsAFlAd1Ip9+K6ZePzMDQ6KZ1nY\\\\\\\/Bsh7iK4VQG6VE93+DIDgMq5UkZs2LruGD86JeUrcDvSFRqSx\\\\\\\/dp8DV6ppcBRfKKZN6bbg188fbExSrlZwVai5lm2OYQFM6r8BR1kLROAv0acSEpeQT1hQ4wqcjrUsL1SWSK06hbXLDVyzva3qJKp1iqo0xJ1qDMh68CspQg4NTYTV4pM8jKgrY2IiObSJqWBa\\\\\\\/Xfpv7xpp2I9BtR4mVvxPmPs16T6JqxxuWHlQX3flYMIQMyWc8UCcc19WZ\\\\\\\/UCmHkaXqoeIMc82YoVsdiUXpF7aoShkWvFeI4zulObqNIDSNbuHoaoDFCuLsATxKY296Nu+hHNzGdPVBlntNVpyp2ClFkS2KcnJe2vISnVjVnmB+GOfw0ZvpsxKy+yGre\\\\\\\/i7DZEfc1X8cxRbpeEylyupVLbe5MmKi9hnDgYrcUvyiCIrpC2FJ1nao\\\\\\\/17G19dQvvJmx71d73DkFSUNace+T0SN6Zi\\\\\\\/oPYSH9TmeB0NX10KkVlUksKwXh4xxgho8DYq\\\\\\\/G7MoJNR5fqVe7K3VsUMbwfYHylb6JLXGxw5fKypkgAr91ew4LyVO3CVAMXHo7aWfO3BV0OQBamjR3Twf1InxtfhtgRBnGcpYZWjICQG84ALycVUBLxgXmkM9bHnK53K2+oba5zojDjuMh5tH34Nq6jQoh1mvgX6c9DfKfKEPc5jJFPaHQ979ew5VQmicfcdRkdGMi5jTJ8IXbZtIUDXFRspe8H8Y0PZbyXbHeSZKbH6B7NozJnTlOOwlVBc07dvdwFJQ6KZ9Kx\\\\\\\/UxCcrygLZ7ZwMX6zBnnXkJCeZrRmfFINxR0XEb25aR4mGFepZoa3wZNK\\\\\\\/XEMmb8cDuRXx58jx\\\\\\\/NXlST50TJyneIsn8lBe0lqZmWyV5vIXeZonWmar\\\\\\\/+Ozy6vnBjUZR84WJcaZcrAgvmPAacXztCRFI4mfo9BuNfe9kLoisk3Bia+9IJ4NnhqK\\\\\\\/kThQrv+fvchmOMhJ1SZsiQyrRSG8JZbMBkPpFbCosJ\\\\\\\/B2SvCOu8S83l3EF9TlK6sHDZ\\\\\\\/Gd3KveChifWWa9GJSJAIa\\\\\\\/zOxckzsvjZ\\\\\\\/vxOLtHADargabtzM3rmjC9Ff5aEWeDltSQMjd62Qh5oL5CEarCN8TUC0h5rfop\\\\\\\/5MFHZoa0Skq6yzoqwkf9vubWio180xlhoc\\\\\\\/1JepsxHBIv9\\\\\\\/PwO1wlP+YHb5zvnfJihvmL29OWx6d3kVWk5YVrZ8p7KVGq2eoR52wZGeaToXbulOlCw1zQMzHyOy2f0T1YYdKIfkSswML8oc2mR6GuIjA93NdpGlgwv7syh3jtbMbYoN0Rq3uDxqCwyduL5m0nAGzqcxSCvPLkJrOCkbKph7jOVCJsYKp8HKgJKIr\\\\\\\/EHjiEbATUtFCQ3UYSJ2k+R6nIHXP2mijr7oczDDusk25vE8fsGUzRPUgHMyAtVAm+LEE\\\\\\\/AxbMT5diVVlvSWx+HBkWHLv0w6C8sgKlExOL2MeK6B5FqLmDAdSHoVFUrNDVephV69LhLj4ioUfxO0cYIAZ38cWKctl\\\\\\\/Teru7Kg8eFdo+\\\\\\\/3rjnaKYblKk0Hv+dEw0BUhSg8OJs\\\\\\\/rgsM7Hu0CNWpkLSSbZS7t+LlN8x5ZFOHbzSrpAcOuSwNcgkLUSGk3l6QTL+ivZjlwEnI\\\\\\\/saJ95RllkPi9JCw0M\\\\\\\/nHYKIrs27PBMHUAL3PzWA66Fk\\\\\\\/FnK6pUtmtQfM4CmjG4pAXKXJ6ln2E92W\\\\\\\/ShN54o8mZ1AyEphSkCav\\\\\\\/6X1dFUrmtj0MG3e+XQq8MHJrfLX+phf\\\\\\\/M93+LYPv09hSmeWmVB1KJNMp+gyPEhTTcWNWy9YFMoO6p8VO1dtTiimK4Zj7m\\\\\\\/Xigqa0cW408q4fwuU6iNkH5rFvHVyBD42gI+KiNuTTllqGB\\\\\\\/M6HK+AYAFo80rR4kUY7ktrxntCkLL+9VETUDqhYUNOxDJ3nzOsB0OtnIY6X5K0UL18Mt8wfo3oaQi5dLADbUfMFR5jxuF8pgPhXCEN63X\\\\\\\/ANMf9zyB80SSF1dgZxuKiLM1SmoTEdDLRnw7EvB7d2BkdxNfhjshvFvhwW63Cah0oB5t82lWVsQIDa\\\\\\\/\\\\\\\/0\\\\\\\/fcZHKy0jL0L6JjFC6RCfQgK1tPZFpWhyLhdFJT6YdfOkBX9zkiIo2T9hNc7DghlFRIwJyL\\\\\\\/gSC37mQMylr0XZFFOe5Y+nlmPGsXf8BZynKxnscM9Yea3cl5CftilNPrO3nDxmKqg8o7+D2aS1VGCobtW46PZkA9p+SL+sbOxa50v2PHHCQYazIqV0Ha1PN7bhyoxRFO+UR7\\\\\\\/aTXczofCsehjClX5GqGjlv2qldJILzBy3clN0Bm43HveIj0iTEGnRKI+Gy+h9iYn\\\\\\\/h3AwvNpGrdmhZZbC8n5zMGBZBXZtMaOq8gBa06R3iVZYz31cmLX+Ea8JqkPKjnfDcnAz1BHEs47aQjLIpRTC10bE3Z\\\\\\\/yxggIqXJHclR7aXX02zs1vCIy\\\\\\\/Sh8XmraLBYQBWEGBxHjOy3ShZ+8tqtktKpdGHPZ9tcucnIbhAEczXd0ZzTjuIhK4ytEOO2oX6aJobKqOFLZEQU+abE2EcFrknAoQ\\\\\\\/D6+OYvqGViuo83NjzaUlE5eVErw3ch4u2Apvp9QkF+LfSecXTwS\\\\\\\/t18WenfNem95IJ+gGELdFX\\\\\\\/58ewAta2Xbe\\\\\\\/U3tJNb6eVv5aUOvNErNo4bmhDvCoU3dSzWISM8DYydBHy5CFgc6zqYkxK11nkOgQd1xmxSTTqZJ9w7xkHmDWFgsZB6+AyBdAdCI2DRCvCXel0tfaDljSJToUTQAiVPuD07V3\\\\\\\/Id\\\\\\\/tHonhckQ4OYetV7VF0fhd97bffmfi4LBOcyK5FHtz4FKsfnl1dmxHRIHVZVaFnRecdeLYzBAnAvGxrWAQ3HDssmEQiRoIANMC\\\\\\\/21qwjVIbAqqR3IHSVg85RuscwoM3cB22vviQ0cSoxxhngQ9ZFqUpzoO\\\\\\\/wUIuQ9nTv6JaIlF6q3TpPLVKmALbk4lTbsZBGCslxzLIUmw1J2ikKa6kM+ARxAwgRf6uNdkR1bl64KRobeunqU441TvqaLPFE+FB59rBYQYHKNgsZpIBB9pW9eK\\\\\\\/MyrmzjhqNxJc7cajZVqcMRz9eYRyYkKTPdfcUew43CBHZ1EkB0VB+ZD9X\\\\\\\/VKHc2utn3dXVtLX3khGtRFtN5e6XXSxo0TBsHJG+rOUwbTWJlnu+TdrhZAgoNKlI7Lpj0Z6\\\\\\\/BixiFHLevnVqWmLzHQzvlE0BXfV4\\\\\\\/AklfF6nU8gfNf8pTgxJVW7xkXsz7Aa75Hj4JbI07kAc5YuFiyGaUvqD53CGg83qDKPQlqMb5nYC0Dzb4LxQqL6Wo+sgRGdZwH1Yj3bQMqCUKRXhjrf6gfqCMimUK8Fkhz8HMwc158G3iybENZzpCG1m5418c+0kj7B63p8JZO85AuWwC9iouijln1hlYD1hV+L66VYZiwvKEF48RoETGdBzVkwoRC0D3QQvYMEmd29l9d8ILb1XfoyZiGgyc0JpVovGy\\\\\\\/dWbVk9I4dmH9Q\\\\\\\/j4oLdq01pr4GcGS41Mujd0Mw2Oxaamu\\\\\\\/N6R48ZVTARMjT18a5ZgzKnXnfBMNNp6DqNSZM4sw1qFdRgeMO\\\\\\\/u80PRdAvn49Dw4BRYW6r1sB8DjnO+4SSQZDUuAjNfXhIwUl5Qu8l8G1kCY+JSZa4WIsq6Au8UdwofOZZfgDlTrQIJdadNOk+jPqMYjC93fXw3\\\\\\\/W+woIhHCPEtmTE\\\\\\\/w6OpOSlrY2kQwTBqyKmvAx72fqaZjt\\\\\\\/20i5h8GhLKIL5fh2vbrUGQy+xx7FEozRZgiUSjGB3MfDkgPButCLqx8KWABSPqUJszQ7U5bvbjTqnfwvUncVTH6woXYLhRrqkbZWrht73JrIh6hVxmSZzqzSQWO\\\\\\\/FKxv1Z9ceWjLDKRGOCgyDGaZKIUu96p46I3jNb9s6H5tHVC9kGX6LYNdR8RI1635Frgu5\\\\\\\/uCLcu8BfSh7dimHUGaAtVb28tDmMZV\\\\\\\/xc\\\\\\\/PDPW2wUzpEe8Gmdm3v1OJxtPtbkEMSv15pb4UXlOy+jtUFP2qvfkXLywqd\\\\\\\/Rplj2G+nUF9FmOn57AMSzoPhGP9t2WxEZ6jFNPuECYTLUOaPDf6XfyYrk1Xlm8F2r40KGnicwWV1iQhLaLjIcff+YaTDLfF0m6k4wbCt0ef\\\\\\\/DLDyTAVksSAv3D0ZEnEx83TQkmKAXb5NG1zjEg3l8O3WLLLIPIdPPYC0WBaKrYSHx1p4VBXKWijHpXFiWMgoTuzDIqzWBrjvoDnNfKyHWOgpLvJyaI7ZAf1t9DsGyUp4E\\\\\\\/qAXW75A34CvifKk0IZ7QDDg6JFFfap34pDEhblZMnV7\\\\\\\/6fWd4Odo+7fIkb9a5vYDtUXYNvIHa5so2S9dQvNHs62y3SzjdwycDuMblZ9kQLYHKkXF7C+2WKY\\\\\\\/Yx0qNb\\\\\\\/781s29rIOEqy7KOWJYSn8GGLQo7aC6+icW9S65hFE\\\\\\\/gmhdXy8T6ToAtjPGVUmkTVa5f59FI3I6UkXL\\\\\\\/njceqC\\\\\\\/8NTz3f71dUjDKlJ+5SZU1lrE283bp\\\\\\\/afrfjJvF7cmtmPapPghffkH5ww\\\\\\\/UeTeWxAu48RYBpokO8kGPY0qw9Emc9avriBxvfYYhg4P8+ghiNyZy4\\\\\\\/\\\\\\\/bhe8\\\\\\\/lEdT0N6QWy3QyKqdColWHkH3\\\\\\\/NLVmzlmyowtYlYtRM\\\\\\\/2gn3CYi5ANI5\\\\\\\/DMAYDW3lzInMVAo4yaSz820NeiZow7ylHM0AX5B1BXkY26HCt3b4DAK0K+ZbqvlIW2LN0Y3iRxQSDqK7MoikwXbruhNtUf2qZTM4574i1fDiwT9GSTg834YeMgfOvfzd1QPAPWbwGVZyhtPND3aFheViWTwlsmgm+Vgm346U\\\\\\\/XAKXsqly1KIYIscrqd1+9SktFT+AQOjOh01wE19XtnaK7rQt+5IRBg1wy9ESVbirB+xCjgO4GXa00ysrTBvrjlu3vmWWwbStxbbPSsXF+990ZoDTYUZeqL8QFtlhyxTyjU+UrNgmK\\\\\\\/cRRVXsI4bRXUW9ANIwPS7RdeqCfdH9ZNNmRdHi1cl28UC3M8fiijQYf\\\\\\\/uVFmJrf9whoqsMcqJS+4y+e9Xu74LF16\\\\\\\/XJ91mgchx\\\\\\\/7Z6tlRKNcmThGH3glNrXGSjECdgqmm0YE779aojofRL\\\\\\\/fJ6pAfjqOsKegRZTa7\\\\\\\/zBLUZ\\\\\\\/C5IR4n1XuxpwE4oZsR5vrvrfz4xItC6wIDa+0My5AcOvXdYoqSsyx3weiaMvg3VbkQZvh5u9zi0prA+HwSuZbM3Pnk5ABUWGYVA64lYN6Uvn4wwNbHqOJ\\\\\\\/Q1lrRQEBeAyzDk1HkQsb3ZPxWBrMS+U5VT6j03tXyldNl60INBv0Ke7s00AOU6IW+LL7ODw7FZeFTDpxqMXPqm5zUI2slXzqMryy8JL2WRNd913xBK8E\\\\\\\/6uzDQ9g7+Fv5uDIrL\\\\\\\/6OEQwVuENg7VkAXxSRqDkFvHImunQI0XFmJ2W9U9I10fG9ZeM+U59yoWpNzW5W\\\\\\\/qPNN2ree\\\\\\\/0j49LCzgc1ot9bfjJkvoSqV1CS+0Lxj9C0UvRwlf7eSUG47W5F\\\\\\\/yAP\\\\\\\/ksoLI0+6wsbzWjpaX+IK3XNIAAYvZvlZwrGtNqGFDkOs8meBPutGc6I4HBn2DjhN8vS6n0+pW02PmHqhSxKAKMmHAXN2ki+24n9FoTcY3Luu0BObbAmYt99WXe\\\\\\\/T46vovInQYjb56O+mGoL2oUBH28SSL4DLyerOmM9EOMtPXwbm8ENt1g6HgJ2SL\\\\\\\/7n9LGWMSvC3LMTozHlLItO+r\\\\\\\/oElwc+xQEEabE\\\\\\\/C01sJShD3H6mec0EUg6aG15xg8iMr7AmiZSEV+HTJX6PoEwTeOSkdoMRW7eWzr0AOYY+ZNS5jA+rULJ7EFmNbxVoKCkn7nzOPZHeLhqb012jxIGAuyzN5vl+A8JviYj0GldpTvwGvB1hVuGUV6pGXx+Nt7nMHefUnCROWLJ4YEBjNMoLL+zqKzDo8ZGOyH6G209C1eEG0ukC67Coh0l5BonYYBDpHcWhA4rRknPb+0ojl5FVpoDYVVaVOSCrg7nvfwoHMlksP5XUm+f5kj6NK0iimhO9vcscG\\\\\\\/ldvs1GoKn4p0fQBok+H6\\\\\\\/8ouO2aX1cX0eOTxRkouX+qr5t91WzME9eb6GyIJwqd6I7+6Nsk7josi9H6W1FCgJ3z0Lbvwv8VUEPY6paCHdrmdgVdKHyKcfAKUWwQL5nA+OcmyPEZE827eJ5RDfAFKhfhz4CB4emPSmi5Og2Nm6IDRDw2v2l72bKFUc5m0YyZtHu4ap2OJ\\\\\\\/O2RHD5vYEATgB5PZSOeWIK92fJJGjGIPFXT1gKBKq3LQhVBFePPBtZ28Nt6LeEh1g+4fBK5Qmja9hKMLO04nHF5Lj8AcUt84D923OFZuIWTUUcMpEDebe2bnaGqdKLIZFYWTssgdT4cqXVKUjRQg5XgUxYfxAXagBuM8Xm6DAL+YEQ9xj1LsMUOPG0aAWLDzsS1oFA5AD7w7SE1WnWDD27KPfZ8PlsMHCjULH23Utxq5vIdl9gwO8SMnSi0qbOB3YAxTqG+qpkeBLnYe1LhdIxyZlYSDttPRejX3S7M007\\\\\\\/wjuSiGRadORQcsTuVWmQdGV3Rlmtl2cqYhGKS0axtQtbTip\\\\\\\/SxQZcVPoRSTe8ufPPsvo\\\\\\\/8yKeOczlt44BDWkALykaFvfWm9rFpShVtGVx9hoiwg9krEHmGmnXujYe249YMtFCEkJNIZlitEoq+5rDlnY2zcnAR35809EAht1T5gtrtFK+ZuQvuGTxti\\\\\\\/A+mMuw9ALclMkGKPIjcFfA2u\\\\\\\/icQAhOXzZODkDy2h1bNBWhu5ei1RVpTVWnBVr0J0P96gPEdBIVK\\\\\\\/7ZofIa8u4ziDr8+db4epPdUmrF38nNpD8TQzJkf15oytrTZnC\\\\\\\/\\\\\\\/TLw4T63o1yTf2s1fQNABr+etz4UR8P+vf6kguecbT8FwRRe40YU8bDQNu4HqxTaw3c4VOTy2bTxP3KC3As57myZokBfYo5KAIbqGBLyCJsZsVNw\\\\\\\/zgMaYomw0jay6uyZNqePiWmrc3Sn9Efe9\\\\\\\/hgsO8M74JnuobO7lrgqSGK0anu5M7xvzrhgbEKAWfPg6NuJyXH8AyO+nPCrKpRJ5ipX8ZiXDULeKMqcdfKNwU2LIDCR7\\\\\\\/YjRu1OwtxO4V38K+A0ATIepJUpy286kEGHAEJEWEGfuG4DVrxoMxZbaQBD6iGCu8tjCQZEs9ij\\\\\\\/rIxg7NyT\\\\\\\/dQ\\\\\\\/eDh+DmBIzdYYcfR4zYnpDVJtNuKEZbaqPL95J7k+ELUbWw+Y5jjeX5omGJGn4YsfbrMuHOD84idFkaU9FkXHwqpGuZjWeJ1cg9yn9nDyTFwYCZsghqM9J7IQvtdp4XiskXxfE\\\\\\\/LVsRisqFlHs7QEaXBlSIw0n1yQNGtuwJSG3Wa71lWFvl6zRjxKaQUs94Kk7PFZOeJCHRTwDKXIuurIuhceII+BTcyhSOljYtf6865KZUFDIwHybqvD75OOykox6QrjN2RCbgiWZX\\\\\\\/saTKfOuk3tynrnOrx24mTrKEr294GnDzmW1S8AmjXW6Zp1ThAWNHBsimAnI4O1Zmt1kJ682GlribY701VPxvVm32gX\\\\\\\/26bpq2xdBu2Gck09DiLvvc72s0ajxfbV08r8JxuvBT1u4q2JcnoXpX6gAsmCyJzAF1So9kl8gAFuvNHptXvLmnppp8fVVNr9ExT5I00YHts143bWPFDURP4k7GLa1S3pVc\\\\\\\/YRp8kvyG2U5u1URm4F6H8M6Ykvg5HU8j2BItV20PszLg91zhV+fQoCm+SVHP+\\\\\\\/f\\\\\\\/uCmlEts6vu0Wt0ROUx7Tn4bli7t5UOxDO76Qmm14i244G5ajlawdgPeoAfIsu1cN98saL50tRu4fd1w0IUgz2Sx1OfQi1PV7GEdcF6PpyiLShQoQhkdfVi0YZ+lXlNmQMSrYOQ2WjmX4MowQ1HWEEmucMfBd91SmYvk\\\\\\\/nnMcItpG\\\\\\\/eSnfwVGM8gakuvpEOSfy034aRmlPby9sREROfNZZcn+Zxjul4Rv3Wo3DkIFaAkrRMg2vCieAZcJwcbomAjHjxu7cZ5bckA2HjdlHswWKtVLE3g2cKO8mLpy14b2x+q3iinMyfixJjyHHE1YQqRkEsYfkPZptSB354xs\\\\\\\/wf04qKn0MVBKWHAq5rO9Zrq0ny4kBtJwSA\\\\\\\/3x3hwPNQG1UXCvlJePm1mAaVDlVk8o+jdowUoZtdcu\\\\\\\/oP0Gr844SwNiGqWdmfgQ4bpDuQsesX+WWLH9nwArckxa8rmrOYDBrMzUz9xXgBLVbmSScWMsBk5lPEa7uAVcErTSham3Gu94\\\\\\\/n8cdHExsGR8ZQvdyMGZYzVFjs22mfutVl3bZ916whX\\\\\\\/tirygvf5nZkTX09CkrRN3fSu1hg6ebTGoOtBnxytotf+c+FR0Jc3YsVEacUe6gzs7w19ruGL5jhAD667ym+oxmg1OzPwIgtZr86DHgmHxfGxqaYhgu1JCcC3295DIz974Pk5nN8cwGuUbR1VquqrjHJ5JPqCY6GhrnVWq9JPYhaKbnGNpiVarArpwtZpxU+WKJexXDH2+cwT6FuKO7PgrwtP4XSIIlIFiRsQcGXtyQCYmAOsIbLV2PhkBGJp\\\\\\\/x2J55gmytcZXFc\\\\\\\/aQ82t16YoVdO13NnY9qsTHcmMIJdMFPvD8Y3L\\\\\\\/lEbOpDXPoIV17TvVvpYssiIrE1ohw9iqiSpf\\\\\\\/FA4RGs0ehqbpnBWvQBZQSR8VHYTUb1HYCUpPYEQuVxIB7\\\\\\\/JVc3kucPMY8npgcD2Sz1FjOYE+UxbniBb7S09UMyaN3TWyMnxnAFDoFJrsI7u8Jc1Og0JZKIDVlvCE6PpG73QFBgsIaFqrEX85+Iw3K6tZLCZIzji\\\\\\\/LljQRWYF3PLhPymyVEXWfz6guTs4DIUibSw6OLOgivAdtBSR\\\\\\\/RSb7CzO5zH3yzLKSSKe9k06GW0OhvAAWOQ4TLkhkjrJFWy3bNd4HVsoO9hVmEH31DlgFFBKAZXbTESOcvAC5IBdzpYrhpet+tv57QKmuDz0T+47XmLIvxeEusUiZ1zagvwgr62i2xJW2e0T1Mm2cNa\\\\\\\/FEDOQGK9\\\\\\\/YjChTUR4pQgCUJHxvXk1RQaCrKHZKN9CECI3aR6kqqiVBoPcZ77\\\\\\\/2Pfns4fO4+wiBphtgyLZY9WFgy6BzbTRcQSzhZCO4VSlbCDEH3mvctY8wdsyaPPce3vj9zhk6kGxldwJBQ+qVJBnCFYEfZPtk\\\\\\\/5RLauuBkDJ6Rt8cO9hpSJkbzPozYQKo7Qpxy8iHPvb3MwUdJ3\\\\\\\/h5W8jBD3J8bPCbqX5iDcsa+ryxZPK8zI1L\\\\\\\/GUwaxTtkl9f4lcalXX6e6YZIVtE9zCg3YjH3oQ82kAz12RbJ\\\\\\\/BwVLOeY5eiWgQik8Sw0\\\\\\\/k7rYcMS4+QikwXT0B90C46MCefLrKbeA5Yg60THwcc1qSVcAF2NICBUwgQWmED\\\\\\\/NTUYjmUQPzQIfhylLoxu1rJNlKfG7IHEwICAZxpfpiD2upNPpsQL0ZnnN7+Y0nKC3f19\\\\\\\/FLYN627kBKPANCq7U5byKWRsm6FIbC7JEWMWHfLEVK77EmyLfvmEKPcXlYUkrLa6BsajDt2OT1nfnRQ9aZb0xKnSeq4gQC8HkYVMPpYI9f4VKsduMH4oGO1ltGkDQPfan16abnBiPORz5E2gm8qVfY8LKZ3\\\\\\\/tv7ud1yIpuohDw5iTei8Ih7iZDKzvyuk8Rjk0Ou36vV+X6PML3BY\\\\\\\/AdpsQBBiw20HtAHUAzmpUYxmOheMx16MzlRh9rnlYOAT0f9XFqovnfd3kgeR\\\\\\\/fbOBWDhM1TlUzUCAlVGZfyjHaqp3kMfPtbCcdwHCVno0qhxTW8gx2YBpafcerh6pu0s20Fmo\\\\\\\/rg5EmuBkMaBVLSqVjQVxYWzeoZTGrDiW+fYN2j0jwzdTCancDDwW0Lu\\\\\\\/z+BTl9TmXrglB0yDzXwAYm333DPR2X67yzJz55FgoVASlYbcGHTVg0sF9d0iJoHpoXvJDysmnFTJMFI3QVQ1juL4vJB0YTCxHoKo9SAdNDEjw+WlJK6y0KZMGW7kLM\\\\\\\/IKPDqYWphLILlCdupbpRhMnyQIeEeVh0MDdQjvcKfO6WPOwScSdLQVNsm8eoQgC69UvgJa+z6CyxcJukrfJyPC8Raxm6e0trs7zwFupMIX7MhVObJJvR7sDY39Q3drecxJf+lioIBzOLtW7zyHbBwSa5gUj4qmjLBD3ctJA7VA7Kpdn67juDbHMYcYgRDR3xDfvldi4K5E2Pbse\\\\\\\/gniDIiHy1qdryK4fFdudTHfYqW52Bw5hFAK7tgiT7+hb8kQdnI0EV+gRXaMUbf35oYoT4TQJiVuwlp4zJWj2gSxd50ZAAlXKL2OWPMmnxXlt5xcD6RQ9lWRZzC6jj5vCNgK1+3YFoQ7sQNa6sHjvod\\\\\\\/qD4NX5LGlYwUwKVWUx8SZS+5OJvBLCWsKFUgoAKTwtmJbh4ozCBIMcUpHhIeZrZlQWHE\\\\\\\/\\\\\\\/QeRx9xvU9SqEq7AbNOl\\\\\\\/AIa8NAul\\\\\\\/Zf2Wpqjg\\\\\\\/XNNVaX7PT\\\\\\\/iAxlf6NILBK4nvOdcU2IjdmLjAc6JQhgp76qUI9BgtneOO2j7ZLaLK1OV\\\\\\\/\\\\\\\/zqSSuFz6qtml06SEL+3aK4t8DNfkG0E5knPC3UmSU8pgpqXPcc5BYaTykXTLG7\\\\\\\/4FCmolV6m0XoL5eyvWmaLVg0EaQRxsgQq21Dz\\\\\\\/rTGyK1\\\\\\\/RuzpH6V9\\\\\\\/oN4Yh\\\\\\\/RC5f68ivstAlAakE1E97Zyv6wwZTBcH8+F\\\\\\\/GcpxZ6ZP+t2mmP5b9nsP0j7bSotLPFMr2IRI+SKaFQghg7NgdS2NDiRrbDCOj52Op2j1nEaXpj9nhNCToYYgYEbWMVLk8yCEhpN4c1NN6oU\\\\\\\/rjweA0QLCc0C\\\\\\\/JfPFVY7G9iKEcL8tLa\\\\\\\/z0j2Qp1yw01arYnvdLPs1xlE3g\\\\\\\/Wv\\\\\\\/jHpM0FoCf5dbomz3a4SZcXph4ULKtq7fNZz0qGKoVeDgf3AjNV06XurmrMYdBNVBs5gMOR+\\\\\\\/Y+V8qZihQ1pLXk1eiJWrWlL9pdh4EtC0ZXHddp+FQjILuJ8c10gqfTOg2DlD9pEpPRCMwXKxUZJMukiXJC\\\\\\\/R3yAAAzL1FKwyQ91UoY99eZfMffJ1bp6wuRFYUuWFi2dNIyGSglnWQHl5nemehVNjpYlwjcYz9DdZsBpV3sDtyZo\\\\\\\/S2WrgeqBC2G0lfAL\\\\\\\/GdbmNhNkQSazSHKrdJEQYpqu5MoWHhHIc4CmHTaIr8yq9WLbjDyG2lmktApVPgDiqX\\\\\\\/zEhmGxJwrlzPLeQd5\\\\\\\/KWiFqWkP0xdmiR0ihZWJmo5GdU55SCfiG4cax+vurmDmIxYy5D4nJ97wUZcCqOvOCn+d\\\\\\\/S4fbDjuxNNPjcJThMfVHKkQ0ESEUaU+A06MpOJWSSM3N1wp+Ck1Q6iZnM6FFuFtVQzsSs98aViLbNlCR1wogTVqCnMX4TtSdFaCCeEEm3HqUKCTTtjP6Og2\\\\\\\/YgWj+PqppgnNZPeUxxirPp6Q8g1YivmO25adTRQFnFnnNWdIokcpLl\\\\\\\/SwfUrx\\\\\\\/AGLecKZH9VR9OhApJoEBVegma8Zj2tLvr0Kd7yJfslkNofJd3s68x54ty8HslOMi4mPb9NkzaHHwcHpVKbfYsgEdA08rv0BOEy1vv79b2YLcfQGfh+4oUjRLs2vGP2CPUZPS0QQbQ3KVAuSeyQQegUPHMbU\\\\\\\/8LnRANBu9cUi+6S0lJz7RMftA53U8nDtKO6esCwTD1QBu+xdYK05PIEwoQkGAtBVySyx+na3QRaBQI3+20A\\\\\\\/zgFrbkAwS1FbYWyW434EpRpARQpv++\\\\\\\/CAvxg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022IjrQAGb257Yfmf5yRX8TGA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002208b61438faa733ec41e9970558507f51898e4f82c48e8495a29db510559baa8d\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022oUcwvTFlFBCHGkif9KeEGT+3ZEvCK\\\\\\\/d\\\\\\\/Xhsr8HsplUsFpztzdhl3n1EXd00OSKQ71Y+dFaxVfojkVkOokZf5nYu3P2wNyKAMBUQc7iOxWNsrQ5p+2iQCXYqyRVyILtDd2gfHs2gQU1WEDZusLVEfMcrZddW0LJaKh1JMDzR8nXC7o2gGyMWhOLAwpvrap0AG61dWUIr\\\\\\\/66SuwskwWvzOH2WrhUrK4mXlFqfvjzqyFcXctT2U90bsSePXCMwSfEzc2P2sUWRh0DkybZk6orPEmazO\\\\\\\/7igAsD8z9qpCijv\\\\\\\/ri8Ez2gk3pEDzZL5OSPw\\\\\\\/jrb8LxrxNy8zYV\\\\\\\/fW5ezlDDqYj1YeKxT7\\\\\\\/UjHudAD02Lh9hiHHpwYJuiALo5IdOdJfflb+oMduF066qgzMasNIwH\\\\\\\/6Km4vV\\\\\\\/2tM+2S89Nx19HugzcNE\\\\\\\/qdoWXh72oXRy2EwpX8dnQg1cD0K7sbSogdpjg3\\\\\\\/HqqPQpFwdP5Qxh1rt7BEtfZ7uWbPo1RlE8fuDSGTTx+cde9+pfMs+CcvtyP339Zs+yBtHS5FAN8NuNABHTmDjsDRr34sQ7QMRDljJcFvlNR3uTokSQu9qs7lfCnS0pU6e0iQ03C811+Z78FR3DLjQ+h8mVPnAklG8umvT7RSD9GYSCeDizmg+0ar44ATqf7KB\\\\\\\/n3ktAeWzdh6o0X4hujsyn9f5klrzUf8\\\\\\\/ydUhV9iyhPqlCgWF5fdcMLZyGoP6aO4FxeDIhO\\\\\\\/5\\\\\\\/\\\\\\\/IVEt2LnTAAhXABpmrUE1lb0zfVMrkkZfJ3DTN94YPvoWzbdjzGhLA0bviRS1rn6nqrX+p2\\\\\\\/FGM3GkDIg90QXsxI18TLPpzWgxCaDqizU2D9TrQkgG6Ay9PZ4hOdF1KS+hfsP+H8nlvYoRr19T4X7W4QQtm4PsRznWqDWgStbyoJ0cRB4iLR4E72T4qUnsnJslAJrcKz7G7RmiaXIrGIDmGxeqrQOf9hZnrHGEAqBW77QsA3eFIQgckPthSwneE3ywlh\\\\\\\/hlSlF51Cq\\\\\\\/GbhGWe\\\\\\\/QcQJ\\\\\\\/EE\\\\\\\/9gV0qx9VfAnVBd4z4\\\\\\\/x10WlH1dcPCqUvhINsk805hZ6d46KNs489cSIQfkh1jt2eSBshov\\\\\\\/NOQuaHrkPaj6gH1M4JlL+ikOOZGsDALLTGPv6iELQUnaMLDEXCSJlxNt\\\\\\\/Ouj9HTwVoC5\\\\\\\/Dt1YsA2iHxTZUjEeoSPGOzs+qtsrO9FUWciwSIQjRoD3Tgqtuwj5hUXgvFeGzyQYXqGGAIK33IaUDthvQtpyX8HurxfLXHMFMhGuyt8gxzgvnS9XptFoMrIbja4e66Ygz1c9Dop9spNVcRBofe2Xck20Es7wjpIPmGz9oj1WAT47+gdpozoNqApIWAyzYZoH0noPkrJ14io9SibUs2ay0cmV6Nzi\\\\\\\/cCI53Iedeqmbuix9MCa6OvCA2UdU8tVJLTWJqdQleOMA3OsKN3RmOB4+j6EUaYRz4IewieJB1pjeDRxi3\\\\\\\/KPT4v+3uNztQIQSnNvTIXG\\\\\\\/nH71z52d38CL3llzKAUWwCIExWrJr2l85Kc22t1xKf8Wt8ebhCkkKlzGaC+um6t6KbUwc26YBJGlSQ6Px+r4TzSjxguTY\\\\\\\/YVgr4\\\\\\\/xPNJ99dhvneWz\\\\\\\/XIOugpINig9Hnprdyzc4X7lk3ISduVk1tODdata4r6F3sqztV9mnh3PzwzRrj3Z\\\\\\\/E+H3FAagp4AjBS8KEdlzO9fHosRZwZxhh70GDDlya6nMOAcPrHJDC4F6U3w+Xr\\\\\\\/gq2Gx8D28D5ENP7UuKpXD52+V\\\\\\\/QCF8tdp1ZLICl3FYDvJGzj+aUvSAKcUK31nCcoHrnzbeJvvoT5LIyeZuSB64BznLyK7i5gAw9QpemnQmRvII6lprNvk20v\\\\\\\/KY2Qa8xAwJJsQM7RjfWw6LttBWNVzkIzX3g9DLT2EB7SckUTiX1WIvejvO9fJmkK277wYe6TJsHVbE7l3oxrZVYn+Yf2A3yygDIBaVSSimtcTBUruJFs1\\\\\\\/BOWCZGNv5t\\\\\\\/5QsCsUvUz2YKCDJRNltQA0Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022PSxul7eszJne0FuElWLpbQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f810457fb273efef8be5b14e1ef5fdf189015407d5fc217d26de0c4f0a41609d\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495499 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/811" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Oa4kF8T8hBUDUOmorQAvf_UwIxv34DMWxcw1kcod9-o" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/811" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-K0PYc41Zd3sc1I3IE2D6imiBfePh9g1S4wtzFhuQBKA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/2/797" title="Interaction Between Induced and Natural Variation at oil yellow1 Delays Reproductive Maturity in Maize" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/2/827" title="Assessment of a Split Homing Based Gene Drive for Efficient Knockout of Multiple Genes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495499" id="node495499" data-pisa="ggg;10/2/811" data-pisa-master="ggg;g3.119.400951" data-apath="/ggg/10/2/811.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Screen for Gene Paralogies Delineating Evolutionary Branching Order of Early Metazoa</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7107-5518" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Albert Erives</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-4882-8398" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bernd Fritzsch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">811-826;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400951 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495499" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Albert Erives </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Iowa, Iowa City, IA, 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Albert%2BErives%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Erives%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlbert%2BErives%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7107-5518" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Albert Erives</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:albert-erives@uiowa.edu" class="" data-icon-position="" data-hide-link-title="0">albert-erives@uiowa.edu</a>
<a href="mailto:bernd-fritzsch@uiowa.edu" class="" data-icon-position="" data-hide-link-title="0">bernd-fritzsch@uiowa.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bernd Fritzsch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Iowa, Iowa City, IA, 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bernd%2BFritzsch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fritzsch%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABernd%2BFritzsch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4882-8398" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bernd Fritzsch</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:albert-erives@uiowa.edu" class="" data-icon-position="" data-hide-link-title="0">albert-erives@uiowa.edu</a>
<a href="mailto:bernd-fritzsch@uiowa.edu" class="" data-icon-position="" data-hide-link-title="0">bernd-fritzsch@uiowa.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/2/811" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495499" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495499/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/2/811.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495499" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495499/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/2/811.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495499" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495499/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/2/811.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495499" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495499/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The evolutionary diversification of animals is one of Earth’s greatest marvels, yet its earliest steps are shrouded in mystery. Animals, the monophyletic clade known as Metazoa, evolved wildly divergent multicellular life strategies featuring ciliated sensory epithelia. In many lineages epithelial sensoria became coupled to increasingly complex nervous systems. Currently, different phylogenetic analyses of single-copy genes support mutually-exclusive possibilities that either Porifera or Ctenophora is sister to all other animals. Resolving this dilemma would advance the ecological and evolutionary understanding of the first animals and the evolution of nervous systems. Here we describe a comparative phylogenetic approach based on gene duplications. We computationally identify and analyze gene families with early metazoan duplications using an approach that mitigates apparent gene loss resulting from the miscalling of paralogs. In the transmembrane channel-like (TMC) family of mechano-transducing channels, we find ancient duplications that define separate clades for Eumetazoa (Placozoa + Cnidaria + Bilateria) <em>vs.</em> Ctenophora, and one duplication that is shared only by Eumetazoa and Porifera. In the Max-like protein X (MLX and MLXIP) family of bHLH-ZIP regulators of metabolism, we find that all major lineages from Eumetazoa and Porifera (sponges) share a duplicated gene pair that is sister to the single-copy gene maintained in Ctenophora. These results suggest a new avenue for deducing deep phylogeny by choosing rather than avoiding ancient gene paralogies.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/animal-evolution" class="hw-term hw-article-keyword hw-article-keyword-animal-evolution" rel="nofollow">animal evolution</a></li><li class="kwd"><a href="/keyword/gene-duplication" class="hw-term hw-article-keyword hw-article-keyword-gene-duplication" rel="nofollow">gene duplication</a></li><li class="kwd"><a href="/keyword/max-x-mlx" class="hw-term hw-article-keyword hw-article-keyword-max-like-x-mlx" rel="nofollow">Max-like X (MLX)</a></li><li class="kwd"><a href="/keyword/mlx-interacting-protein-mlxip" class="hw-term hw-article-keyword hw-article-keyword-mlx-interacting-protein-mlxip" rel="nofollow">MLX interacting protein (MLXIP)</a></li><li class="kwd"><a href="/keyword/transmembrane-channel-proteins-tmcs" class="hw-term hw-article-keyword hw-article-keyword-transmembrane-channel-like-proteins-tmcs" rel="nofollow">transmembrane channel-like proteins (TMCs)</a></li></ul><p id="p-3">The branching order of the major metazoan lineages has received much attention due to its importance in understanding how animals and their sensory and nervous systems evolved (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Jékely <em>et al.</em> 2015</a>). To date, most phylogenetic analyses have used single-copy orthologs, with different genes and approaches finding support for either Ctenophora (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Ryan <em>et al.</em> 2013</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Moroz <em>et al.</em> 2014</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Borowiec <em>et al.</em> 2015</a>; <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Whelan <em>et al.</em> 2015</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Shen <em>et al.</em> 2017</a>) or Porifera (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Pisani <em>et al.</em> 2015</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Feuda <em>et al.</em> 2017</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Simion <em>et al.</em> 2017</a>) as the sister taxon of all other animals. In contrast to sequence-based phylogenies, comparative analysis of single-cell transcriptomes from different lineages and cell types is consistent with an independent origin of neuron-like cells in ctenophores (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Sebé-Pedrós <em>et al.</em> 2018</a>). Another recent study that does not rely directly on sequence analysis provides evidence that the sponge choanocyte does not correspond to the sponge cell type most similar transcriptomically to the choanoflagellate cell type (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Sogabe <em>et al.</em> 2019</a>). Thus to date, few studies (<a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Sebé-Pedrós <em>et al.</em> 2018</a>; <a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Sogabe <em>et al.</em> 2019</a>) have addressed the question of early animal branching without using single-copy genes for phylogenetic analysis.</p><p id="p-4">Single-copy genes are preferred for phylogenetic inference of lineage branching order for several reasons (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Fitch and Margoliash 1967</a>; <a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Woese and Fox 1977</a>). For example, single-copy genes more closely approximate clock-like divergence (<a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Zuckerkandl and Pauling 1965</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Fitch and Margoliash 1967</a>; <a id="xref-ref-89-2" class="xref-bibr" href="#ref-89">Woese and Fox 1977</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Pett <em>et al.</em> 2019</a>) compared to duplicated genes, which frequently experience neofunctionalization and/or uneven subfunctionalization and evolutionary rate asymmetries (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Walsh 1995</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lynch <em>et al.</em> 2001</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Holland <em>et al.</em> 2017</a>). An organismal tree of animals can be constructed from a single-copy gene, or from a set of concatenated single-copy genes, provided orthologous outgroup genes are included in the analysis as an aid for rooting.</p><p id="p-5">Likewise, single-copy genes (strict orthologs across all lineages) offer a practical advantage in eliminating the ambiguity associated with gene duplications (paralogs). Some paralogs represent recent lineage-specific duplications while others stem from deeper duplications contributing to a larger gene super-family. Nonetheless, a gene tree of a pair of genes produced by a duplication in the stem-metazoan lineage can depict lineage-branching order doubly so, once in each paralog’s subtree. Furthermore, a gene tree of paralogs offers a unique advantage unavailable in single-copy gene trees: a tree of paralogs captures the duplication itself and unites lineages sharing the duplication relative to outgroup lineages with a single gene.</p><p id="p-6">A majority of gene orthologs are part of larger super-families (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Ohno 1970</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Taylor and Raes 2004</a>) as has been well documented for the Hox gene family (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Holland <em>et al.</em> 2017</a>). Thus, many “single-copy” genes are only ostensibly so because they can be evaluated separately from their ancient paralogs; and because in principle single-copy genes diverged lineally from a single ancestral gene present in the latest common ancestor (LCA) of a taxonomic clade. However, the choice of homologs is a poorly examined aspect of modern phylogenetic analysis even though various ortholog-calling errors associated with ancient paralogy have been noted (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Noutahi <em>et al.</em> 2016</a>).</p><p id="p-7">Here we identify candidate gene paralogies established prior to the evolution of Eumetazoa (Bilateria + Cnidaria + Placozoa) for the purpose of determining early animal branching order. The majority of these paralogies predate the metazoan LCA and/or experienced apparent gene losses in either candidate first animal sister lineage (Porifera or Ctenophora) and are not informative (see Table S1). We also find a smaller number of paralogies that possibly support a proposed clade of “Benthozoa” (Porifera + Eumetazoa) while we have found none that support the traditional grouping of Ctenophora with Eumetazoa. The benthozoic hypothesis is premised on the latest common ancestor (LCA) of Choanozoa (Choanoflagellatea + Metazoa) being holopelagic, and the LCA of Benthozoa having evolved a biphasic pelagic larva and a benthic adult form. Other alternative life cycle scenarios have been proposed that correspond to different branching patterns (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Nielsen 2008</a>, <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">2013</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Jékely <em>et al.</em> 2015</a>), some of which are based on fossil interpretation (<a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Zhao <em>et al.</em> 2019</a>).</p><p id="p-8">Among the most intriguing of our findings are the recently identified family of multimeric transmembrane mechanosensitive channel proteins, the transmembrane channel-like (TMC) proteins (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Keresztes <em>et al.</em> 2003</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Ballesteros <em>et al.</em> 2018</a>), in which family we find definitive independent duplications in Eumetazoa (<em>Tmc<span class="underline">487</span></em> + <em>Tmc<span class="underline">56</span></em>, where “<em>Tmc<span class="underline">487</span></em>” and “<em>Tmc<span class="underline">56</span></em>” represent the ancestral sister genes eventually giving rise to the five genes <em>TMC4</em>–<em>TMC8</em> in jawed vertebrates for example), Ctenophora (<em>Tmc-α</em> + <em>Tmc-β</em> + <em>Tmc-γ</em> + <em>Tmc-δ</em>), and possibly Benthozoa (<em>Tmc<span class="underline">48756</span></em> + a neofunctionalized <em>Tmc<span class="underline">123</span></em> clade). In summary, our identification and analysis of genes duplicating and diversifying in the stem-benthozoic lineage will help to outline the extent of a shared biology for Benthozoa and the nature of independent evolutionary neuralization in Eumetazoa and Ctenophora.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Comparative genomic orthology counting and screening</h3><p id="p-9">To identify sets of candidate gene paralogies for further phylogenetic analysis, we used the BioMart query tool (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Durinck <em>et al.</em> 2005</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Haider <em>et al.</em> 2009</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Smedley <em>et al.</em> 2009</a>) and the EnsemblCompara orthology calls (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Vilella <em>et al.</em> 2009</a>) for MetazoaEnsembl Genomes Release 41. We also used these same tools to identify 2146 unique protein-coding genes in the placozoan <em>Trichoplax adhaerens</em>, which share the following properties: (1) these genes all can be grouped into a smaller number of paralogy groups; (2) these genes all have homologs in the cnidarian <em>Nematostella vectensis</em>; (3) the cnidarian homologs can also all be grouped into a smaller number of paralogy groups; (4) these genes all have homologs in the molluscan genome of <em>Lottia gigantea</em>, representing Lophotrochozoa, as well as in <em>Drosophila melanogaster</em>, representing Ecdysozoa; and (5) these genes all have homologs in the sponge <em>Amphimedon queenslandica</em> (a sponge in class Demospongiae). To ensure that our results would not be skewed by errors in gene annotation and curation, we focused on genes for which we could identify homologs in other cnidarians (the anthozoan <em>Stylophora pistillata</em> and the hydrozoan <em>Hydra vulgaris</em>), another sponge (the homoscleromorphan sponge <em>Oscarella carmela</em>), and throughout Bilateria. We then constructed phylogenetic trees for several different gene families from this list. An additional but similar search strategy is described in the text.</p></div><div id="sec-3" class="subsection"><h3>Sequence alignment</h3><p id="p-10">We identified and curated sequences only from representative taxa with whole-genome sequence assemblies. We obtained initial (pre-full-length curation) sequences from NCBI’s non-redundant protein database using the BLAST query tool with taxonomic specification (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1990</a>), and/or from the Ensembl Release 97 and Metazoa Ensembl Release 45 databases using the ComparaEnsembl orthology calls (<a id="xref-ref-84-2" class="xref-bibr" href="#ref-84">Vilella <em>et al.</em> 2009</a>). For additional ctenophore sequences from the <em>Pleurobrachia</em> and <em>Beroë</em> genomes, we queried the Neurobase transcriptome databases using BLAST (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Moroz <em>et al.</em> 2014</a>). For additional sequences and transcripts from cnidarian and sponge genomes we also queried the Compagen databases (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hemmrich and Bosch 2008</a>). In addition to BLASTP queries of protein or translated transcriptome databases, we also used the TBLASTN tool to search translated genomic sequences using a protein sequence. We did this for two reasons. First, we used TBLASTN to verify that certain gene absences were not simply due to a failure to annotate a gene. Second, we used TBLASTN on several occasions when curating missing exons. We identified several genes from genome assemblies that were initially predicted by computational annotation and for which we then hand-curated for this study, typically to identify missing terminal exons. These are indicated in the tree figures and in the FASTA headers (Supplementary files) by the accession numbers with “CUR” appended. Alignment of protein-coding sequence was conducted using MUSCLE alignment option in MEGA7 (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Kumar <em>et al.</em> 2016</a>). The multiple sequence alignment (MSA) was conducted using default parameters that were adjusted as follows: The gap existence parameter was changed to -1.6 and the gap extension parameter was changed to -0.01. Excessively long protein sequences were trimmed at the N- and C- termini so that they began and ended on either side of the ten transmembrane domains. Lengthy, fast-evolving, loop segments and/or repetitive amino acid sequences occurring in between transmembrane domains were trimmed. Supplementary Files for curated data sets are provided as explained under “Data availability”.</p></div><div id="sec-4" class="subsection"><h3>Phylogenetic analyses</h3><div id="sec-5" class="subsection"><h4>Candidate gene family screening:</h4><p id="p-11">To screen through a large number of candidate gene trees, we first constructed draft trees using Maximum Parsimony and Neighbor-Joining using sequences from the principal animal lineages and from choanoflagellates when available using MEGA7 (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Kumar <em>et al.</em> 2016</a>). For potentially informative trees, we then selected sequences from additional lineages, performed more extensive curation, and constructed additional more detailed tree versions. These early versions were typically of sufficient quality to identify whether or not genes were present in the correct lineages to be informative to this study’s focus on animal lineage branching.</p></div><div id="sec-6" class="subsection"><h4>Bayesian inference of phylogeny:</h4><p id="p-12">To conduct metropolis-coupled Markov chain Monte Carlo (MC3) Bayesian phylogenetic analysis we used the MrBayes (version 3.2) software package (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Huelsenbeck and Ronquist 2001</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Ronquist and Huelsenbeck 2003</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Ronquist <em>et al.</em> 2012</a>). All runs used two heated chains (“nchains = 2”, “temperature = 0.08”) that were run for at least 1.2 M generations with a burn-in fraction of 20–25%. Initial runs for all gene families sampled all substitution models, but we always found 100% percent posterior probability assigned to the WAG substitution model (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Whelan and Goldman 2001</a>). Subsequently all finishing runs used the WAG model with invariant-gamma rates modeling. Double precision computing was enabled using BEAGLE toolkit (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Ayres <em>et al.</em> 2012</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Ronquist <em>et al.</em> 2012</a>).</p><p id="p-13">All trees were computed multiple times during the process of sequence curation and annotation. The final <em>cornichon</em>/<em>cornichon-related</em> gene family tree finished with 0.009 average standard deviation of split frequencies from two heated chains. The final <em>Tmc</em> gene family tree finished with 0.005 average standard deviation of split frequencies from the two heated chains. The final <em>MLX</em>/<em>MLXIP</em> gene family tree finished with 0.007 average standard deviation of split frequencies from the two heated chains. The final version of the <em>MAX</em> super-family tree finished at ∼0.015 average standard deviation of split frequencies after 3.2 M generations. Tree graphics were rendered with FigTree version 1.4.4 and annotated using Adobe Photoshop software.</p></div></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-14">All data files, including sequence files (<em>*.fas</em>), multiple sequence alignment files (<em>*.masx</em> and <em>*.nexus</em>), and curation documents, are provided with the Supporting Information as a zipped archive. Four files are provided for each of the three gene families shown for a total of 12 files. Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.11444400">https://doi.org/10.25387/g3.11444400</a>.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Gene duplications from the early metazoan radiation</h3><p id="p-15">Given the preponderance and constancy of gene duplication (and gene loss) throughout evolution, one should in principle be able to find a gene duplication shared by all major animal lineages except the one true sister animal lineage. Therefore, ever since the elucidation of the first non-bilaterian animal genomes, mainly those from Cnidaria (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Putnam <em>et al.</em> 2007</a>), Placozoa (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Srivastava <em>et al.</em> 2008</a>), Porifera (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Srivastava <em>et al.</em> 2010</a>), and Ctenophora (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Ryan <em>et al.</em> 2013</a>; <a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Moroz <em>et al.</em> 2014</a>), we have sought to identify gene duplications that definitively order early metazoan phylogeny. For example, to identify a diagnostic gene family with a signature duplication occurring just after the early metazoan radiation, we have searched for <em>Amphimedon</em> (sponge) or <em>Mnemiopsis</em> (ctenophore) genes with predicted 1-to-Many relationships with the other metazoan genomes. Remarkably, this strategy has not been productive, and at best has led to the identification of gene families with many possible root choices consistent with either a sponge-early or ctenophore-early model. A good example of this root intractability with an increasing number of deep gene duplications is the phylogeny of the metazoan WNT gene family (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/811/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Early metazoan branching and the root of the WNT gene family. Example gene tree for the metazoan WNT ligand in which sponge and ctenophore genes exist in 1-to-Many relationship with other metazoan genes. A major problem in extracting a phylogenetic signal from a large gene family defined by many paralogs is the increasing ambiguity in identifying the true root of the tree, particularly when lacking a non-metazoan outgroup gene. For the WNT family tree shown here, there are four ctenophore-early (up arrowheads) and three sponge-early (down arrowheads) choices for rooting. In addition, there are other root choices for either model in combination with inferred gene losses. For example, here this tree is rooted between a weakly supported clade (40%) containing only sponge and ctenophore genes and the clade containing all the remaining genes. If this was the correct root, then there would have been a loss of a metazoan WNT paralog in the stem-eumetazoan lineage regardless of whether the true tree is a ctenophore-early or sponge-early tree. This tree was constructed using a distance-based method (Neighbor-Joining) with 100 boot strap replicate samples. Numbers indicate boot strap supports for values &#x2265; 40%. The same color scheme is used throughout this study as follows: violet = Bilateria, cyan = Cnidaria, magenta = Placozoa, blue = Porifera (sponges), and mustard/yellow/orange = Ctenophora (different hues for different species)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-780785390" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Early metazoan branching and the root of the WNT gene family. Example gene tree for the metazoan WNT ligand in which sponge and ctenophore genes exist in 1-to-Many relationship with other metazoan genes. A major problem in extracting a phylogenetic signal from a large gene family defined by many paralogs is the increasing ambiguity in identifying the true root of the tree, particularly when lacking a non-metazoan outgroup gene. For the WNT family tree shown here, there are four ctenophore-early (up arrowheads) and three sponge-early (down arrowheads) choices for rooting. In addition, there are other root choices for either model in combination with inferred gene losses. For example, here this tree is rooted between a weakly supported clade (40%) containing only sponge and ctenophore genes and the clade containing all the remaining genes. If this was the correct root, then there would have been a loss of a metazoan WNT paralog in the stem-eumetazoan lineage regardless of whether the true tree is a ctenophore-early or sponge-early tree. This tree was constructed using a distance-based method (Neighbor-Joining) with 100 boot strap replicate samples. Numbers indicate boot strap supports for values &#x2265; 40%. The same color scheme is used throughout this study as follows: violet = Bilateria, cyan = Cnidaria, magenta = Placozoa, blue = Porifera (sponges), and mustard/yellow/orange = Ctenophora (different hues for different species).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/811/F1.medium.gif" width="440" height="406"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/2/811/F1.medium.gif" width="440" height="406"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/811/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/811/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495739" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-16" class="first-child">Early metazoan branching and the root of the WNT gene family. Example gene tree for the metazoan WNT ligand in which sponge and ctenophore genes exist in 1-to-Many relationship with other metazoan genes. A major problem in extracting a phylogenetic signal from a large gene family defined by many paralogs is the increasing ambiguity in identifying the true root of the tree, particularly when lacking a non-metazoan outgroup gene. For the WNT family tree shown here, there are four ctenophore-early (up arrowheads) and three sponge-early (down arrowheads) choices for rooting. In addition, there are other root choices for either model in combination with inferred gene losses. For example, here this tree is rooted between a weakly supported clade (40%) containing only sponge and ctenophore genes and the clade containing all the remaining genes. If this was the correct root, then there would have been a loss of a metazoan WNT paralog in the stem-eumetazoan lineage regardless of whether the true tree is a ctenophore-early or sponge-early tree. This tree was constructed using a distance-based method (Neighbor-Joining) with 100 boot strap replicate samples. Numbers indicate boot strap supports for values ≥ 40%. The same color scheme is used throughout this study as follows: violet = Bilateria, cyan = Cnidaria, magenta = Placozoa, blue = Porifera (sponges), and mustard/yellow/orange = Ctenophora (different hues for different species).</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">As a possible explanation for our failure to find unambiguous, early metazoan, gene paralogies, we hypothesized the existence of a methodological bias associated with the various ortholog-calling pipelines on which we were relying. This hypothesis of an inherent bias in resolving ancient gene paralogy motivated an alternative comparative genomic screen that we present here. We begin by illustrating the problems that can arise by not accounting for ancient paralogy.</p><p id="p-18">We consider the consequences of a rooting procedure defined by a sponge-early model given an early-stem metazoan gene duplication event in either a true sponge-early world (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>) or a true ctenophore-early world (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). These arguments are meant to illustrate that human and machine-predictions of orthology will encounter unique problems specific to the true animal sister lineage of all other animals particularly when phylogenies are constructed as “single copy” genes. (Many single-copy genes are actually members of much larger super-families and so are single-copy only in the sense that an analysis restricts itself to a sub-clade of genes).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/811/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dangers and promise of ancient paralogy for deducing the deep phylogeny of animals. (A) Given a true sponge-early phylogeny (left organismal tree with animal icons), a true gene tree for a gene duplicated in the stem-metazoan lineage (red bar) and maintained in each lineage without any gene loss is composed of two separate gene clades (blue clade one and magenta clade two), depicted here as evolving at different clade-specific rates. The principal metazoan lineages correspond to Bilateria (B), Cnidaria (Cn), Placozoa (Pl), Ctenophora (Ct), and Porifera (Po, sponges). Eumetazoa is defined here as Bilateria + Cnidaria + Placozoa. The true gene tree is rooted easily between the two clades and recapitulates the sponge-early model in each clade (Po1 and Po2 are the deepest branching lineages). (B&#x2013;D) Depicted below the horizontal line are the consequences of forcing gene trees from a ctenophores-first world into a sponge-early model. (B) Shown is the competing ctenophore-early model in which Ctenophora is sister to all other animals (here referred to as Benthozoa as explained in the text). For the same stem-metazoan gene duplication depicted in A, we now have a corresponding true gene tree in which the ctenophore genes (Ct1 and Ct2) are the deepest branching lineages given the true ctenophore-early organismal tree. Asymmetric rates (faster or slower) are depicted in the gene phylogram, typical after divergent functionalization of paralogs. (C) The true gene tree in B is redrawn here with a different rooting procedure that isolates the more slowly-evolving paralog clade two (magenta clade in inset box) such that the sponge (Po2) lineage appears as the outgroup lineage within that subclade. This sponge-early re-rooting procedure maintains the topology of the true gene tree. The dotted line anticipates the analyses of the improperly partitioned subclades as separate &#x201C;single-copy&#x201D; gene clades. The partial clade two tree (inset) is missing the Ct2 ctenophore gene and is associated with a false-positive signature of gene loss. (D) The sponge-early model is applied to the remaining genes as shown. This tree includes the ctenophore Ct2 gene that was previously flipped into the faster evolving clade one, which also has a higher propensity for long branch attraction than clade two. In contrast to the false-positive gene loss associated with the slowly-evolving clade two, the faster evolving clade one manifests false-positive heterogeneity of evolutionary rates and amino acid content due to the mixture of ctenophore paralogs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-780785390" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Dangers and promise of ancient paralogy for deducing the deep phylogeny of animals. (A) Given a true sponge-early phylogeny (left organismal tree with animal icons), a true gene tree for a gene duplicated in the stem-metazoan lineage (red bar) and maintained in each lineage without any gene loss is composed of two separate gene clades (blue clade one and magenta clade two), depicted here as evolving at different clade-specific rates. The principal metazoan lineages correspond to Bilateria (B), Cnidaria (Cn), Placozoa (Pl), Ctenophora (Ct), and Porifera (Po, sponges). Eumetazoa is defined here as Bilateria + Cnidaria + Placozoa. The true gene tree is rooted easily between the two clades and recapitulates the sponge-early model in each clade (Po1 and Po2 are the deepest branching lineages). (B&#x2013;D) Depicted below the horizontal line are the consequences of forcing gene trees from a ctenophores-first world into a sponge-early model. (B) Shown is the competing ctenophore-early model in which Ctenophora is sister to all other animals (here referred to as Benthozoa as explained in the text). For the same stem-metazoan gene duplication depicted in A, we now have a corresponding true gene tree in which the ctenophore genes (Ct1 and Ct2) are the deepest branching lineages given the true ctenophore-early organismal tree. Asymmetric rates (faster or slower) are depicted in the gene phylogram, typical after divergent functionalization of paralogs. (C) The true gene tree in B is redrawn here with a different rooting procedure that isolates the more slowly-evolving paralog clade two (magenta clade in inset box) such that the sponge (Po2) lineage appears as the outgroup lineage within that subclade. This sponge-early re-rooting procedure maintains the topology of the true gene tree. The dotted line anticipates the analyses of the improperly partitioned subclades as separate &#x201C;single-copy&#x201D; gene clades. The partial clade two tree (inset) is missing the Ct2 ctenophore gene and is associated with a false-positive signature of gene loss. (D) The sponge-early model is applied to the remaining genes as shown. This tree includes the ctenophore Ct2 gene that was previously flipped into the faster evolving clade one, which also has a higher propensity for long branch attraction than clade two. In contrast to the false-positive gene loss associated with the slowly-evolving clade two, the faster evolving clade one manifests false-positive heterogeneity of evolutionary rates and amino acid content due to the mixture of ctenophore paralogs.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/811/F2.medium.gif" width="302" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/2/811/F2.medium.gif" width="302" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/811/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/811/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495788" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-19" class="first-child">Dangers and promise of ancient paralogy for deducing the deep phylogeny of animals. (A) Given a true sponge-early phylogeny (left organismal tree with animal icons), a true gene tree for a gene duplicated in the stem-metazoan lineage (red bar) and maintained in each lineage without any gene loss is composed of two separate gene clades (blue clade one and magenta clade two), depicted here as evolving at different clade-specific rates. The principal metazoan lineages correspond to Bilateria (B), Cnidaria (Cn), Placozoa (Pl), Ctenophora (Ct), and Porifera (Po, sponges). Eumetazoa is defined here as Bilateria + Cnidaria + Placozoa. The true gene tree is rooted easily between the two clades and recapitulates the sponge-early model in each clade (Po1 and Po2 are the deepest branching lineages). (B–D) Depicted below the horizontal line are the consequences of forcing gene trees from a ctenophores-first world into a sponge-early model. (B) Shown is the competing ctenophore-early model in which Ctenophora is sister to all other animals (here referred to as Benthozoa as explained in the text). For the same stem-metazoan gene duplication depicted in A, we now have a corresponding true gene tree in which the ctenophore genes (Ct1 and Ct2) are the deepest branching lineages given the true ctenophore-early organismal tree. Asymmetric rates (faster or slower) are depicted in the gene phylogram, typical after divergent functionalization of paralogs. (C) The true gene tree in B is redrawn here with a different rooting procedure that isolates the more slowly-evolving paralog clade two (magenta clade in inset box) such that the sponge (Po2) lineage appears as the outgroup lineage within that subclade. This sponge-early re-rooting procedure maintains the topology of the true gene tree. The dotted line anticipates the analyses of the improperly partitioned subclades as separate “single-copy” gene clades. The partial clade two tree (inset) is missing the Ct2 ctenophore gene and is associated with a false-positive signature of gene loss. (D) The sponge-early model is applied to the remaining genes as shown. This tree includes the ctenophore Ct2 gene that was previously flipped into the faster evolving clade one, which also has a higher propensity for long branch attraction than clade two. In contrast to the false-positive gene loss associated with the slowly-evolving clade two, the faster evolving clade one manifests false-positive heterogeneity of evolutionary rates and amino acid content due to the mixture of ctenophore paralogs.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">In a sponge-early world, the sponge-early rooting procedure is simple and defines two gene clades (blue and magenta clades in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>). The same is true if we use a ctenophore-early rooting procedure given a ctenophore-early world (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>). We now describe the consequences of a forced sponge-early rooting given a ctenophore-early world, which has been the subject of much debate (<a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">Ryan <em>et al.</em> 2013</a>; <a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Moroz <em>et al.</em> 2014</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Borowiec <em>et al.</em> 2015</a>; <a id="xref-ref-87-2" class="xref-bibr" href="#ref-87">Whelan <em>et al.</em> 2015</a>; <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Shen <em>et al.</em> 2017</a>; <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Jékely <em>et al.</em> 2015</a>). For ease of reference, and for reasons explained in the Discussion, we will refer to the hypothetical sister clade composed of Eumetazoa + Porifera as “Benthozoa” (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a>).</p><p id="p-21">By choosing the most closely-related homologs to the more conserved member of a pair of duplicated genes (the more slowly-evolving magenta subclade two of <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>), or by internally re-rooting the true gene tree so that the subclade in question fits the standard model in which Porifera is sister to all other animals (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2C</a>), we end up with an isolated subclade with a false-positive gene loss (inset box in <a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2C</a>). Furthermore, the ctenophoran true outgroup sequence (“Ct2”) is easily collected into the more divergent sub-clade (blue subclade one in <a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2C</a>). When the fast-evolving gene sub-clade is rooted separately with sponge as outgroup (“Po1”), the inherent topology unites both ctenophore paralogs in an apparent ctenophore-specific duplication (bottom phylogram in <a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2D</a>). This second fast-evolving subclade would feature false-positive rate heterogeneity and compositional heterogeneity. False-positive compositional heterogeneity is consistent with recent attributions of (true) evolutionary sequence bias in ctenophores (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Feuda <em>et al.</em> 2017</a>).</p><p id="p-22">As predicted by the ctenophore-early hypothesis (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2B</a>) and deep paralogy-induced miscalling of orthologs (<a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2C–D</a>), we find that relatively fewer metazoan orthologs are called in the ctenophore <em>Mnemiopsis leidyi</em> relative to sponge (<em>Amphimedon queenslandica</em>), placozoan (<em>Trichoplax adhaerens</em>), cnidarian (<em>Nematostella vectensis</em>), and bilaterian (the lophotrochozoans <em>Lottia gigantea</em> and <em>Capitella teleta</em>) genomes when orthologies are predicted according to a sponge-early model (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). This finding supports the ctenophore-early hypothesis and is consistent with a similar approach in estimating deep animal phylogeny using gene content (<a id="xref-ref-66-4" class="xref-bibr" href="#ref-66">Ryan <em>et al.</em> 2013</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/811/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Deficit of metazoan homologs in ctenophores. Consistent with the predicted false-positive gene losses depicted in Fig. 2C, comparative genomic orthology data sets based on a sponge-early model result in far fewer orthologs called in a ctenophore relative to other metazoans. Shown are the percentages of each organism&#x2019;s protein-coding gene set that have orthologs called in a cnidarian (Nematostella, x-axis) or a sponge (Amphimedon, y-axis) as inferred from the Metazoa ComparaEnsembl orthology calls (see Materials and Methods). Dotted box shows the distance from ctenophore value to the lowest value of the other animals along each axis (solid lines)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-780785390" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Deficit of metazoan homologs in ctenophores. Consistent with the predicted false-positive gene losses depicted in Fig. 2C, comparative genomic orthology data sets based on a sponge-early model result in far fewer orthologs called in a ctenophore relative to other metazoans. Shown are the percentages of each organism&#x2019;s protein-coding gene set that have orthologs called in a cnidarian (Nematostella, &lt;em&gt;x&lt;/em&gt;-axis) or a sponge (Amphimedon, &lt;em&gt;y&lt;/em&gt;-axis) as inferred from the Metazoa ComparaEnsembl orthology calls (see Materials and Methods). Dotted box shows the distance from ctenophore value to the lowest value of the other animals along each axis (solid lines).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/811/F3.medium.gif" width="440" height="234"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/2/811/F3.medium.gif" width="440" height="234"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/811/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/811/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495501" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-23" class="first-child">Deficit of metazoan homologs in ctenophores. Consistent with the predicted false-positive gene losses depicted in <a id="xref-fig-2-13" class="xref-fig" href="#F2">Fig. 2C</a>, comparative genomic orthology data sets based on a sponge-early model result in far fewer orthologs called in a ctenophore relative to other metazoans. Shown are the percentages of each organism’s protein-coding gene set that have orthologs called in a cnidarian (Nematostella, <em>x</em>-axis) or a sponge (Amphimedon, <em>y</em>-axis) as inferred from the Metazoa ComparaEnsembl orthology calls (see Materials and Methods). Dotted box shows the distance from ctenophore value to the lowest value of the other animals along each axis (solid lines).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>A model-agnostic screen for early metazoan duplications</h3><p id="p-24">Based on the above observations and rationale, we devised a comparative genomic screen in which orthology calls to the ctenophore <em>Mnemiopsis</em> and the sponge <em>Amphimedon</em> are not considered during the initial candidate paralogy group screen. In other words, we relinquished our previous reliance on 1-to-Many predictions for sponge or ctenophore genes relative to other metazoans. We began with the 16,590 protein-coding genes from the beetle <em>Tribolium castaneum</em> (Tcas5.2 genes), which we chose as a model bilaterian that has not experienced extensive gene loss as in nematodes (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Erives 2015</a>), nor extensive gene duplications as in vertebrates. The <em>Tribolium</em> gene set becomes 8,733 genes if we only consider those that exist in paralogous relationship(s) with other <em>Tribolium</em> genes. Of these only 2,617 have orthologs called in the mosquito <em>Aedes aegypti</em>, the lophotrochozoan <em>Capitella teleta</em>, and the placozoan <em>Trichoplax adhaerens</em>. Then we discard those paralogous <em>Tribolium</em> genes which are related by a common ancestor in more recent taxonomic group such as Cucujiformia (an infraorder of beetles), Holometabola, Hexapoda, Mandibulata, Arthropoda, Pancrustacea, Protostomia, Bilateria, and Eumetazoa. We then grouped 2,038 candidate paralogous beetle genes into families of various sizes.</p><p id="p-25">We considered the optimal size for a gene family to be used in resolving early animal branching order. First and foremost, having more duplications is likely to increase the probability that some of the duplications occurred during the early metazoan radiation that produced the extant animal phyla. However, choosing extremely large gene families containing many paralogs introduces other problems in a classic trade-off scenario. For example, large gene families often present intractable root choice problems, particularly when genes are absent in the outgroup lineage of choanoflagellates, as is the case for animal-only genes (<em>e.g.</em>, <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Extremely large gene families seem to also have a greater rate of gene loss. This ‘easily-duplicated, easily-lost’ pattern further exacerbates the choice of root. We thus concluded that it is more effective to screen for gene paralogies defined by the smallest possible number of early metazoan gene duplications (two or three).</p><p id="p-26">To avoid the added complexity from greatly expanded gene families, we set aside 56% of the beetle genes from the 153 largest paralogy groups, containing 4 to 41 genes each (totalling 1,146/2038 genes). We were then left with 892 genes grouped in &lt; 382 small paralogy groups containing two to three genes each (Supporting Table S1). We sampled 16% (60 gene families) from these small candidate paralogy groups to find gene families that had at least one ortholog called in <em>Amphimedon queenslandica</em> and at least one in <em>Mnemiopsis leidyi</em>. We note that we cannot rule out the possibility that metazoan-relevant paralogies were excluded because true orthologs failed to be computationally called in sponges and ctenophores. Many of the candidate paralogy groups have paralogs called in sponges and ctenophores, indicating that the duplications occurred prior to the latest common ancestor (LCA) of Metazoa.</p><p id="p-27">We constructed draft phylogenies using different approaches, including an explicit evolutionary model (Maximum Parsimony) and a distance-based method (Neighbor-Joining). If either of these initial trees indicated a possible early metazoan duplication, we then constructed trees using a more sophisticated and informative approach with metropolis-coupled-MCMC Bayesian phylogenetic inference in conjunction with more extensive gene curation of unannotated exons (see Materials and Methods). As detailed below, this new approach begins to identify choanozoan genes with early metazoan duplications. We present our first results for several gene families as demonstration of the value of our approach and in anticipation of finding more examples as additional genome assemblies from diverse metazoans become available.</p></div><div id="sec-11" class="subsection"><h3>Duplication of cornichon-like transmembrane chaperones</h3><p id="p-28">As expected, many of the candidate gene families, which were identified without regard to presence or absence in <em>Amphimedon</em> or <em>Mnemiopsis</em>, were either: (<em>i</em>) missing in both ctenophores and sponges, or (<em>ii</em>) present in both ctenophores and sponges. The latter sometimes occurred when the ctenophore <em>Mnemiopsis</em> did not actually have the full set of duplications but a related ctenophore did. For example, in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a> we show one of our initial draft trees for the Band 7/Stomatin and Stomatin-like paralogs. These trees are consistent with a signature of gene absence for Stomatin-like in the ctenophore <em>Mnemiopsis</em> (Order Lobata). However, subsequent follow-up shows that a ctenophore from a different order (Order Cydippida) has both paralogs (double asterisks). The clear presence of each paralog in at least one ctenophore means that this candidate could be set aside as being an uninformative duplication that occurred in an earlier choanozoan ancestor. The Stomatin/Stomatin-like example (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>) also shows that our approach is likely to improve as additional genome assemblies become available for sponges and ctenophores. Nonetheless, other trees show from our screen, which was based on an agnostic preference for gene homologs in sponges and ctenophores, were in range to start being informative.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/811/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Example choanozoan duplications: Stomatin/Stomatin-like, and cornichon/cornichon-related. (A) Shown is the draft NJ tree for the Band 7/Stomatin and Stomatin-like paralogs, which were identified in our screen due to the signature of gene loss for Stomatin-like in the ctenophore Mnemiopsis (Order Lobata), which was indicated to have only a single Stomatin gene (single asterisk). Subsequent follow-up showed that a ctenophore from a different order (Order Cydippida) has both paralogs (double astrerisks). The clear presence of each paralog in at least one ctenophore means that this candidate can be set aside as being an uninformative duplication. Boot strap support is shown for values &#x2265; 40% from 500 replicates. (B) Shown is another example gene tree from our screen to identify candidate duplications based on a signature of gene loss (either true or false-positive gene loss). This gene family encodes a eukaryotic chaperone of transmembrane receptors. This tree is equivocal for two scenarios: (1) loss of one of a pair of paralogs in choanoflagellates and ctenophores (see cnir part of tree) as shown, or (2) a duplication occurring later than is shown in the stem-benthozoan lineage with sufficient divergence of the cnir progenitor such that this clade is misplaced in the tree. Tree was generated from a multiple sequence alignment composed of 167 alignment columns and is rooted with Holomycota (Fungi) as outgroup." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-780785390" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Example choanozoan duplications: &lt;em&gt;Stomatin&lt;/em&gt;/&lt;em&gt;Stomatin-like&lt;/em&gt;, and &lt;em&gt;cornichon&lt;/em&gt;/&lt;em&gt;cornichon-related&lt;/em&gt;. (A) Shown is the draft NJ tree for the Band 7/Stomatin and Stomatin-like paralogs, which were identified in our screen due to the signature of gene loss for Stomatin-like in the ctenophore &lt;em&gt;Mnemiopsis&lt;/em&gt; (Order Lobata), which was indicated to have only a single Stomatin gene (single asterisk). Subsequent follow-up showed that a ctenophore from a different order (Order Cydippida) has both paralogs (double astrerisks). The clear presence of each paralog in at least one ctenophore means that this candidate can be set aside as being an uninformative duplication. Boot strap support is shown for values &#x2265; 40% from 500 replicates. (B) Shown is another example gene tree from our screen to identify candidate duplications based on a signature of gene loss (either true or false-positive gene loss). This gene family encodes a eukaryotic chaperone of transmembrane receptors. This tree is equivocal for two scenarios: (1) loss of one of a pair of paralogs in choanoflagellates and ctenophores (see &lt;em&gt;cnir&lt;/em&gt; part of tree) as shown, or (2) a duplication occurring later than is shown in the stem-benthozoan lineage with sufficient divergence of the &lt;em&gt;cnir&lt;/em&gt; progenitor such that this clade is misplaced in the tree. Tree was generated from a multiple sequence alignment composed of 167 alignment columns and is rooted with Holomycota (Fungi) as outgroup.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/811/F4.medium.gif" width="300" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/10/2/811/F4.medium.gif" width="300" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/811/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/811/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495650" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-29" class="first-child">Example choanozoan duplications: <em>Stomatin</em>/<em>Stomatin-like</em>, and <em>cornichon</em>/<em>cornichon-related</em>. (A) Shown is the draft NJ tree for the Band 7/Stomatin and Stomatin-like paralogs, which were identified in our screen due to the signature of gene loss for Stomatin-like in the ctenophore <em>Mnemiopsis</em> (Order Lobata), which was indicated to have only a single Stomatin gene (single asterisk). Subsequent follow-up showed that a ctenophore from a different order (Order Cydippida) has both paralogs (double astrerisks). The clear presence of each paralog in at least one ctenophore means that this candidate can be set aside as being an uninformative duplication. Boot strap support is shown for values ≥ 40% from 500 replicates. (B) Shown is another example gene tree from our screen to identify candidate duplications based on a signature of gene loss (either true or false-positive gene loss). This gene family encodes a eukaryotic chaperone of transmembrane receptors. This tree is equivocal for two scenarios: (1) loss of one of a pair of paralogs in choanoflagellates and ctenophores (see <em>cnir</em> part of tree) as shown, or (2) a duplication occurring later than is shown in the stem-benthozoan lineage with sufficient divergence of the <em>cnir</em> progenitor such that this clade is misplaced in the tree. Tree was generated from a multiple sequence alignment composed of 167 alignment columns and is rooted with Holomycota (Fungi) as outgroup.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">We find that the <em>cornichon</em> (<em>cni</em>/<em>CNIH1–CNIH3</em>) and <em>cornichon-related</em> (<em>cnir</em>/<em>CNIH4</em>) paralogy groups, which encode eukaryotic chaperones of transmembrane receptors (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Herring <em>et al.</em> 2013</a>), was either duplicated in the stem-choanozoan lineage with subsequent gene loss only in choanoflagellates and ctenophores, or else duplicated in the stem-benthozoan lineage with subsequent neofunctionalization. We find that both <em>cni</em> and <em>cnir</em> clades exist only in Porifera, Placozoa, Cnidaria, and Bilateria but not Ctenophora and Choanoflagellatea (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>). This tree is based on a protein that is only ∼150 amino acids long and suggests that the ancestral <em>cni</em>/<em>cnir</em> gene was duplicated in the stem-choanozoan lineage with two subsequent independent losses in the stem-ctenophore lineage and the stem-choanoflagellate lineage (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a>).</p><p id="p-31">The alternative interpretation of the <em>cni</em>/<em>cnir</em> pair is that the progenitor gene was duplicated in the stem-benthozoic lineage with the <em>cnir</em> clade undergoing neofunctionalization to an extent promoting artifactual basal branching. In this interpretation there is no need to invoke independent gene losses for Ctenophora and Choanoflagellatea. Furthermore, key differences in client proteins for the Cornichon chaperones (CNIH1/2/3) <em>vs.</em> the Cornichon-related chaperone (CNIH4) support the neo-functionalization interpretation for the evolution of the CNIH4 paralog. Vertebrate CNIH1 and <em>Drosophila</em> Cornichon were found to mediate chaperone-like export of the transforming growth factor alpha (TGF-<em>α</em>)/Gurken precursor from the endoplasmic reticulum (ER) to the plasma membrane (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Castro <em>et al.</em> 2007</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bökel <em>et al.</em> 2006</a>), while the other vertebrate homologs of <em>Drosophila</em> Cornichon, CNIH2 and CNIH3, were found to mediate similar chaperone roles for AMPA receptor subunits (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Schwenk <em>et al.</em> 2009</a>). Both the TGF-<em>α</em> precursor and the individual AMPA receptor subunits have single transmembrane passes. In contrast, the vertebrate ortholog of Cornichon-related, CNIH4, has been found to mediate chaperone-like ER exit roles for certain G-protein coupled receptors (GPCRs), which possess seven transmembrane passes (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Sauvageau <em>et al.</em> 2014</a>). Thus, the Cornichon homologs, which are also present in choanoflagellates, may have evolved to help ER export of clients with only single transmembrane passes, while the Cornichon-related homolog CNIH4 may represent a neofunctionalization specialized for plasma membrane-bound proteins with multiple transmembrane passes.</p><p id="p-32">In either case the progenitor <em>cni</em>/<em>cnir</em> gene was present as a single copy gene in the LCA of Holozoa as shown by the basal branching of the single gene from the filozoan <em>Capsaspora owczarzaki</em>, which branches before the duplication (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a>).</p></div><div id="sec-12" class="subsection"><h3>TMC sub-clades define Eumetazoa, Porifera, and Ctenophora</h3><p id="p-33">We find that a phylogenetic analysis of gene duplications of the transmembrane channels (TMCs), a large family of ion leak channels with roles in mechanotransduction (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Delmas and Coste 2013</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Pan <em>et al.</em> 2013</a>, <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">2018</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Qiu and Muller 2018</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Ballesteros <em>et al.</em> 2018</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Jia <em>et al.</em> 2019</a>), excludes ctenophores from a eumetazoan super-clade composed of Bilateria, Cnidaria, and Placozoa (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). We describe this gene family’s origin and diversification beginning with the deepest duplications in this family and then proceeding to those most relevant to early metazoan evolution.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/811/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A phylogenetic tree of the mechanotransducing transmembrane channel-like proteins (TMCs) resolves early metazoan diversification as a branching of only three lineages: Eumetazoa (magenta = Placozoa, teal = Cnidaria, and violet = Bilateria), Porifera (blue), and Ctenophora (mustard shades). Each of the eight TMC ortho-groups specific to gnathostomes are indicated on the right. This tree is based on a trimmed data set composed of 777 alignment columns and is rooted between unikonts (labeled &#x201C;Opisthokonta&#x201D; due to the apparent loss in Amoebozoa) and bikonts. Eumetazoan lineages share a duplication corresponding to the Tmc487 and Tmc56 TMC clades, while ctenophores share independent duplications of an ancestral Tmc48756 gene. Ctenophores share with choanoflagellates the more ancient Tmc-X paralog." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-780785390" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A phylogenetic tree of the mechanotransducing transmembrane channel-like proteins (TMCs) resolves early metazoan diversification as a branching of only three lineages: Eumetazoa (magenta = Placozoa, teal = Cnidaria, and violet = Bilateria), Porifera (blue), and Ctenophora (mustard shades). Each of the eight TMC ortho-groups specific to gnathostomes are indicated on the right. This tree is based on a trimmed data set composed of 777 alignment columns and is rooted between unikonts (labeled &#x201C;Opisthokonta&#x201D; due to the apparent loss in Amoebozoa) and bikonts. Eumetazoan lineages share a duplication corresponding to the Tmc487 and Tmc56 TMC clades, while ctenophores share independent duplications of an ancestral Tmc48756 gene. Ctenophores share with choanoflagellates the more ancient Tmc-X paralog.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/811/F5.medium.gif" width="223" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/10/2/811/F5.medium.gif" width="223" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/811/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/811/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495671" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-34" class="first-child">A phylogenetic tree of the mechanotransducing transmembrane channel-like proteins (TMCs) resolves early metazoan diversification as a branching of only three lineages: Eumetazoa (magenta = Placozoa, teal = Cnidaria, and violet = Bilateria), Porifera (blue), and Ctenophora (mustard shades). Each of the eight TMC ortho-groups specific to gnathostomes are indicated on the right. This tree is based on a trimmed data set composed of 777 alignment columns and is rooted between unikonts (labeled “Opisthokonta” due to the apparent loss in Amoebozoa) and bikonts. Eumetazoan lineages share a duplication corresponding to the Tmc<span class="underline">487</span> and Tmc<span class="underline">56</span> TMC clades, while ctenophores share independent duplications of an ancestral Tmc<span class="underline">48756</span> gene. Ctenophores share with choanoflagellates the more ancient Tmc-X paralog.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-13" class="subsection"><h4>Early eukaryotic origin as a single-copy Tmc gene:</h4><p id="p-35">In our TMC phylogeny, we identify genes from both Unikonta and Bikonta, which are the main branches of the eukaryotic tree (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). An affinity of the TMC proteins with OSCA/TMEM proteins has been proposed (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Murthy <em>et al.</em> 2018</a>) despite low sequence similarity (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Ballesteros <em>et al.</em> 2018</a>), but we find that the percent identity between predicted TMC proteins is higher than between TMC proteins and TMEM. For example, the gap-normalized percent amino acid identity in a pairwise alignment of human Tmc1 with the single <em>Naegleria gruberi</em> Tmc sequence is 17.3%, <em>vs.</em> 12.7% (∼5% less) with human Ano3/TMEM16C (Supporting File S2). Moreover the identity between Tmc1 and TMEM16C goes away in a multiple sequence alignment suggesting it is inflated by the extra degree of freedoms allowed in a gapped pair-wise alignment.</p></div><div id="sec-14" class="subsection"><h4>Single-copy Tmc in Fungi and gene loss in clades with cell walls:</h4><p id="p-36">In the context of this ancient eukaryotic provenance of the TMC family, we find an intriguing and suggestive pattern of TMC gene loss that is likely relevant to animal evolution. We find that the Tmc gene exists as a single copy in the genome of the non-amoebozoan slime mold <em>Fonticula alba</em> and a few early-branching fungal lineages, altogether representing the clade of Holomycota. One of these fungal lineages, <em>Rozella allomycis</em>, belongs to the Cryptomycota, a phylum notable for its absence of a chitinous cell wall typically present in most other fungi (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Jones <em>et al.</em> 2011</a>). The absence of a cell wall allows the Cryptomycota to maintain a phagotrophic lifestyle (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Jones <em>et al.</em> 2011</a>). The only other fungal lineages with a TMC gene are from Blastocladiomycota, suggesting that TMC genes were lost in all of Ascomycota, Basidiomycota, and Zygomycota, for which many genomes have been sequenced.</p></div><div id="sec-15" class="subsection"><h4>Single-copy Tmc in Viridiplantae and gene loss With increasing complexity of cell walls:</h4><p id="p-37">Physiological incompatibility of Tmc function with certain cell walls is further supported by a similar distribution of Tmc genes in Viridiplantae (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a> green clade). We find that a single Tmc gene can be found in a chlorophyte (<em>Coccomyxa subellipsoidea</em>), a charophyte (<em>Klebsormidium nitens</em>), and in the multicellular plants of a liverwort (<em>Marchantia polymorpha</em>), a moss (<em>Physcomitrella patens</em>), and a lycophyte (<em>Selaginella moellendorfii</em>). The moss <em>Physcomitrella</em> and the lycophyte <em>Selaginella</em> represent a nonvascular land plant and a vascular land plant most closely-related to the clade composed of ferns, gymnosperms, and angiosperms, which correspond to the crown group having evolved lignin-based secondary cell walls (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Sarkar <em>et al.</em> 2009</a>). In addition, both the <em>Physcomitrella</em> and <em>Selaginella</em> genomes lack many of the cellulose synthase (CesA) genes responsible for the synthesis of plant cell wall components (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Sørensen <em>et al.</em> 2010</a>). Thus these peculiar <em>TMC</em> gene distributions across plants and fungi suggest that rigid cell walls potentially interfere with physical environmental coupling to TMCs.</p></div><div id="sec-16" class="subsection"><h4>Choanozoan duplications of Tmc:</h4><p id="p-38">While we find <em>Tmc</em> genes as a single copy in some lineages of the fungal and plant kingdoms, in <em>Naegleria gruberi</em>, and in the filozoan <em>Capsaspora owczarzaki</em>, we find that these genes underwent duplication only during the holozoan radiation (or if earlier only with corresponding losses in fungi). In short, definitive <em>Tmc</em> gene duplications can be found only within Choanozoa (choanoflagellates + Metazoa) (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>). The choanoflagellates <em>Monosiga brevicollis</em> and <em>Salpingoeca rosetta</em> have at least three <em>Tmc</em> genes each, which we have labeled “<em>Tmc-X</em>”, “<em>Tmc-Y</em>”, and “<em>Tmc-Z</em>” (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>). Remarkably, of all the animals, only ctenophores appear to have genes from the <em>Tmc-X</em> clade, which is a basally-branching clade that is sister to all of the remaining Tmc genes from all of Opisthokonta. Only choanoflagellates and Holomycota have genes in the <em>Tmc-Y</em> clade. Last, except <em>Capsaspora</em>, whose single <em>Tmc</em> gene is of uncertain affinity to the well supported <em>Tmc-X</em>/<em>Y</em>/<em>Z</em> clades, all of the remaining <em>Tmc</em> genes are choanozoan genes from the <em>Tmc-Z</em> clade.</p><p id="p-39">The <em>Tmc</em> duplications within Choanozoa are informative for early choanozoan and metazoan branching. <em>Tmc-Z</em> apparently underwent a key duplication (“<em>Tmc<span class="underline">123</span></em>” + “<em>Tmc<span class="underline">48756</span></em>”) in the stem-metazoan lineage with a corresponding loss of one of the paralogs (“<em>Tmc<span class="underline">123</span></em>”) in ctenophores. Alternatively, this duplication is a stem-benthozoan duplication that occurred after ctenophores split off from the rest of Metazoa. These two <em>Tmc-Z</em> subclades are so named here for their relationship to the vertebrate <em>TMC1</em>/<em>2</em>/<em>3</em> (“<em>Tmc-Z1</em>”) and <em>TMC4</em>/<em>8</em>/<em>7</em>/<em>5</em>/<em>6</em> (“<em>Tmc-Z2</em>”) paralogy groups. We find that this <em>Tmc-Z2</em> gene (<em>Tmc<span class="underline">48756</span></em>) underwent a Eumetazoa-defining duplication that unites Placozoa, Cnidaria, and Bilateria (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5</a>). Thus, in this stem-eumetazoan lineage we see that the <em>Tmc-Z2</em> bifurcates into the sister-clades of <em>Tmc<span class="underline">487</span></em> and <em>Tmc<span class="underline">56</span></em>. This duplication makes it unlikely that Porifera or Ctenophora are more closely related to any single lineage within Eumetazoa. This contrasts with the stem-ctenophore lineage, in which the single <em>Tmc-Z2</em> gene independently duplicated several-fold to produce four ctenophore-specific duplications (<em>Tmc-α</em>, <em>Tmc-β</em>, <em>Tmc-γ</em>, and <em>Tmc-δ</em>) (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5</a>, lineages in shades of mustard). This ctenophore specific repertoire is present in two different classes and three different orders of ctenophores. In Class Tentaculata, we have four <em>Tmc<span class="underline">48756</span></em> genes from each of <em>Mnemiopsis leidyi</em> (Order Lobata) and <em>Pleurobrachia bachei</em> (Order Cydippida). In class Nuda, so named because of a complete (derived) loss of tentacles, we have only three <em>Tmc<span class="underline">48756</span></em> genes from <em>Beroë abyssicola</em> (Order Beroida) because a representative gene from the <em>Tmc-γ</em> clade could not be identified.</p><p id="p-40">Many metazoan sub-clades (<em>e.g.</em>, Ctenophora, Lophotrochozoa, and Vertebrata) can be defined by clade-specific duplications. For example, lophotrochozoans share a duplication of <em>Tmc<span class="underline">487</span></em> into <em>Tmc<span class="underline">487</span>a</em> + <em>Tmc<span class="underline">487</span>b</em>, while gnathostomes share a duplication of vertebrate <em>Tmc<span class="underline">12</span></em> into <em>TMC1</em> + <em>TMC2</em> and vertebrate <em>Tmc<span class="underline">48</span></em> into <em>TMC4</em> + <em>TMC8</em>. If we include the cyclostomes (hagfish + lamprey), then all vertebrates share the duplication of <em>Tmc<span class="underline">123</span></em> into <em>Tmc<span class="underline">12</span></em> + <em>TMC3</em>, and <em>Tmc<span class="underline">487</span></em> into <em>Tmc<span class="underline">48</span></em> + <em>TMC7</em>. Thus, the “canonical” vertebrate repertoire of eight genes represented by <em>TMC1–TMC8</em> is more correctly characterized as a gnathostome-specific <em>TMC</em> repertoire because cyclostomes are united in sharing only some of the duplications seen in humans (see <a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5</a>).</p><p id="p-41">Considering the <em>Tmc</em> gene duplications that are ancestral to Choanozoa and those specific to metazoan phyla, this phylogeny, based on the large TMC protein spanning ten transmembrane domains, unites Placozoa, Cnidaria, and Bilateria into a single Eumetazoa in the following ways. First, of greatest significance, is the shared derived eumetazoan duplication of <em>Tmc<span class="underline">48756</span></em> into <em>Tmc<span class="underline">487</span></em> and <em>Tmc<span class="underline">56</span></em>. Second, is the close placement of Porifera as the sister group of Eumetazoa in both the <em>Tmc<span class="underline">123</span></em> and <em>Tmc<span class="underline">48756</span></em> subclades. Third is the unique situation that ctenophores possess TMC genes from an ancestral <em>Tmc-X</em> clade that were apparently lost in a stem-benthozoan lineage while also possessing an expansive repertoire via ctenophore-specific duplications (<em>Tmc-α</em>, <em>Tmc-β</em>, <em>Tmc-γ</em>, and <em>Tmc-δ</em>). These latter duplications are most parsimonious if they occurred in a metazoan lineage that itself branched off prior to the eumetazoan duplication within the <em>Tmc-Z2</em> clade. In the Discussion, we speculate on the significance of the evolution of mechanotransduction in connection with the evolution of diverse ’body plans’ during the metazoan radiation.</p></div></div><div id="sec-17" class="subsection"><h3>A bHLH-ZIP duplication unites Benthozoa</h3><p id="p-42">We find that a new gene family from the bHLH-ZIP superfamily originated in the stem-metazoan lineage most likely from a duplication of the more distantly-related <em>Max</em> gene (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). This gene occurs as one gene copy in all ctenophores but as a pair of duplicated genes in Porifera and Eumetazoa (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A</a>). The pair of paralogous genes corresponds to <em>Max-like protein X</em> (<em>MLX</em>)/<em>bigmax</em> and <em>MLX Interacting Protein</em> (<em>MLXIP</em>)/<em>Mondo</em>. Mondo paralogs are also known as the carbohydrate response element binding protein (ChREBP) (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Iizuka <em>et al.</em> 2004</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Havula and Hietakangas 2018</a>; <a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Yamashita <em>et al.</em> 2001</a>). The progenitor gene likely encoded a bHLH-ZIP homodimeric transcription factor (TF) in the stem-metazoan lineage, continuing as such into modern ctenophores, but evolving into the MLX:MLXIP heterodimer found in all other animals (Bigmax:Mondo in <em>Drosophila</em>, MLX:MondoA or MLX:MondoB in gnathostomes). To identify the origin of the MLX and MLXIP bHLH-ZIP paralogs, we then conducted phylogenetic analyses with the most closely-related bHLH-ZIP sequences, which correspond to the MAX proteins. We find that the MLX family likely originated from an earlier stem-metazoan duplication of MAX that is absent in choanoflagellates and other holozoans (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a>). The Myc:Max gene regulatory network is known to have evolved in an early choanozoan stem-lineage (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Brown <em>et al.</em> 2008</a>; <a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Young <em>et al.</em> 2011</a>). When we root with choanoflagellate MAX as the out-group, we see that the MLX and MLXIP paralogs emerge from an earlier and presumably neofunctionalized duplication within the MAX family (see “stem-metazoan MAX duplication” in <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6B</a>). However, this duplication appears to have occurred after the lineage leading to modern ctenophores branched away from all other animals as ctenophores appear to have only the progenitor Max-like protein (see “stem-benthozoan duplication” in <a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6B</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/811/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MLX/bigmax and MLXIP/Mondo encode a bHLH-ZIP heterodimer originating in a stem-benthozoan gene duplication. (A) Phylogenetic analysis of a MLX/MLXIP bHLH-ZIP gene family originating in the stem-metazoan lineage (possibly from a duplication of the distantly-related Max gene) followed by a second duplication after the divergence of the ctenophore lineage. Color coding of lineages follows Figures 4 and 5 except when topology precludes coloring sister stem lineages. This tree is rooted with ctenophores as the outgroup lineage. This tree is based on a data set composed of 263 alignment columns. (B) Shown is a phylogenetic tree constructed by Bayesian MC-MCMC and including sequences from the most closely-related and presumed progenitor MAX family (207 alignment columns). This tree places the Max-like sequences within the Max superfamily and furthermore puts the MLX and MLXIP duplications as occurring in a stem-benthozoan lineage that is sister to the single Max-like gene of ctenophores. While the Max-like/MLX/MLXIP sub-family likely originated as a stem-metazoan duplication, its sequences have diverged sufficiently to pull the sponge MAX sequences in a long-branch attraction (LBA) artifact (asterisks). This LBA is not the result of root choice because rooting between the choanozoan MAX sub-clade and the remaining sequences would result in sponges diverging prior to choanoflagellates, which would also correspond to LBA." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-780785390" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;MLX&lt;/em&gt;/&lt;em&gt;bigmax&lt;/em&gt; and &lt;em&gt;MLXIP&lt;/em&gt;/&lt;em&gt;Mondo&lt;/em&gt; encode a bHLH-ZIP heterodimer originating in a stem-benthozoan gene duplication. (A) Phylogenetic analysis of a &lt;em&gt;MLX&lt;/em&gt;/&lt;em&gt;MLXIP&lt;/em&gt; bHLH-ZIP gene family originating in the stem-metazoan lineage (possibly from a duplication of the distantly-related &lt;em&gt;Max&lt;/em&gt; gene) followed by a second duplication after the divergence of the ctenophore lineage. Color coding of lineages follows Figures 4 and 5 except when topology precludes coloring sister stem lineages. This tree is rooted with ctenophores as the outgroup lineage. This tree is based on a data set composed of 263 alignment columns. (B) Shown is a phylogenetic tree constructed by Bayesian MC-MCMC and including sequences from the most closely-related and presumed progenitor MAX family (207 alignment columns). This tree places the Max-like sequences within the Max superfamily and furthermore puts the &lt;em&gt;MLX&lt;/em&gt; and &lt;em&gt;MLXIP&lt;/em&gt; duplications as occurring in a stem-benthozoan lineage that is sister to the single Max-like gene of ctenophores. While the &lt;em&gt;Max-like&lt;/em&gt;/&lt;em&gt;MLX&lt;/em&gt;/&lt;em&gt;MLXIP&lt;/em&gt; sub-family likely originated as a stem-metazoan duplication, its sequences have diverged sufficiently to pull the sponge MAX sequences in a long-branch attraction (LBA) artifact (asterisks). This LBA is not the result of root choice because rooting between the choanozoan MAX sub-clade and the remaining sequences would result in sponges diverging prior to choanoflagellates, which would also correspond to LBA.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/811/F6.medium.gif" width="341" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/10/2/811/F6.medium.gif" width="341" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/811/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/811/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495605" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-43" class="first-child"><em>MLX</em>/<em>bigmax</em> and <em>MLXIP</em>/<em>Mondo</em> encode a bHLH-ZIP heterodimer originating in a stem-benthozoan gene duplication. (A) Phylogenetic analysis of a <em>MLX</em>/<em>MLXIP</em> bHLH-ZIP gene family originating in the stem-metazoan lineage (possibly from a duplication of the distantly-related <em>Max</em> gene) followed by a second duplication after the divergence of the ctenophore lineage. Color coding of lineages follows <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figures 4</a> and <a id="xref-fig-5-9" class="xref-fig" href="#F5">5</a> except when topology precludes coloring sister stem lineages. This tree is rooted with ctenophores as the outgroup lineage. This tree is based on a data set composed of 263 alignment columns. (B) Shown is a phylogenetic tree constructed by Bayesian MC-MCMC and including sequences from the most closely-related and presumed progenitor MAX family (207 alignment columns). This tree places the Max-like sequences within the Max superfamily and furthermore puts the <em>MLX</em> and <em>MLXIP</em> duplications as occurring in a stem-benthozoan lineage that is sister to the single Max-like gene of ctenophores. While the <em>Max-like</em>/<em>MLX</em>/<em>MLXIP</em> sub-family likely originated as a stem-metazoan duplication, its sequences have diverged sufficiently to pull the sponge MAX sequences in a long-branch attraction (LBA) artifact (asterisks). This LBA is not the result of root choice because rooting between the choanozoan MAX sub-clade and the remaining sequences would result in sponges diverging prior to choanoflagellates, which would also correspond to LBA.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">We find that the bHLH domain from ctenophores shares residues with both subfamilies (pink <em>vs.</em> black background in <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). In the Discussion we speculate on a possible role for this gene duplication in a life cycle synapomorphy for the proposed clade of Benthozoa.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/811/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Shown are a subset of alignment columns in which the ctenophore residues more closely resemble either the MLX sequences (top pink) or the MLXIP/Mondo sequences (bottom dark gray). The MLXIPL/MondoB representative sequence is shown for humans." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-780785390" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Shown are a subset of alignment columns in which the ctenophore residues more closely resemble either the MLX sequences (top pink) or the MLXIP/Mondo sequences (bottom dark gray). The MLXIPL/MondoB representative sequence is shown for humans.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/811/F7.medium.gif" width="440" height="276"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/10/2/811/F7.medium.gif" width="440" height="276"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/811/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/811/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495591" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-45" class="first-child">Shown are a subset of alignment columns in which the ctenophore residues more closely resemble either the MLX sequences (top pink) or the MLXIP/Mondo sequences (bottom dark gray). The MLXIPL/MondoB representative sequence is shown for humans.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-46">By identifying and phylogenetically analyzing small paralogy groups that predate a eumetazoan super-clade composed of Bilateria, Cnidaria, and Placozoa, we find we can identify gene duplications that are consistent with Ctenophora being the sister metazoan lineage to all other animals (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figures 4</a>, <a id="xref-fig-5-10" class="xref-fig" href="#F5">5</a>, <a id="xref-fig-6-6" class="xref-fig" href="#F6">6</a>). We discuss the significance of the different gene families that we identified and the possible relevance to metazoan biology.</p><div id="sec-19" class="subsection"><h3>Mechanosensation for animal body plans</h3><p id="p-47">We find that the transmembrane channel-like (TMC) gene family continued to duplicate and diversify for several phyla throughout metazoan evolution. For example, we see that the cyclostomes (hagfish and lampreys) branched early in the vertebrate tree prior to a number of duplications that occurred only in the stem-gnathostome lineage (<em>Tmc<span class="underline">12</span></em> → <em>TMC1</em> + <em>TMC2</em>; <em>Tmc<span class="underline">48</span></em> → <em>TMC4</em> + <em>TMC8</em>; <em>Mondo</em> → <em>MondoA</em>/<em>MLXIP</em> + <em>MondoB</em>/<em>MLXIPL</em>; and <em>CNIH<span class="underline">23</span></em> → <em>CNIH2</em> + <em>CNIH3</em>). Nonetheless, cyclostomes and gnathostomes share the vertebrate-specific duplications corresponding to <em>Tmc<span class="underline">12</span></em>, and <em>Tmc<span class="underline">48</span></em> + <em>TMC7</em>. Thus, like the vertebrate clade and its subclades, many other metazoan phyla can be defined solely on the basis of <em>Tmc</em> gene duplications (<a id="xref-fig-5-11" class="xref-fig" href="#F5">Figure 5</a>). We therefore propose that the evolutionary diversification of the Tmc channels throughout Metazoa, including the independent diversification within Ctenophora (<a id="xref-fig-5-12" class="xref-fig" href="#F5">Figure 5</a>), must have been under selection of two principal forces. The first is the changing physical constraints and sensorial opportunities associated with the evolutionary diversification of animal body plans themselves. The second factor is the evolutionary diversification of specialized cell types within their epithelial sensoria.</p><p id="p-48">Mechanosensory, chemosensory and photosensory responses are universal among single and multicellular organisms and can be related to the evolution of specific proteins enabling already single cells to respond to mechanical, chemical and photic stimuli. For example, opsin proteins evolved in single-celled ancestors of metazoans (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Arendt 2017</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Feuda <em>et al.</em> 2012</a>) and many single cell organisms can sense light, gravity and several chemical stimuli with dedicated sensors (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Swafford and Oakley 2018</a>). While the history of chemical and photic senses and the formation of specialized cell types and integration into appropriate organs in metazoans has been driven by the molecular insights into the molecular transducers (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Arendt <em>et al.</em> 2016</a>), mechanosensory transduction has seen less progress due to uncertainty of consistent association of a specific mechanotransducer channel across phyla (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Beisel <em>et al.</em> 2010</a>). On the one hand, mechanical sensation is clearly present in all single cell organisms to function as safety valves to release intracellular pressure sensed as tension in the lipid bilayer and was proposed as a possibly unifying principle of mechanosensation (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Kung 2005</a>). Follow up work showed a multitude of channels associated with mechanosensation (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Beisel <em>et al.</em> 2010</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Delmas and Coste 2013</a>) arguing against a single unifying evolution of mechanotransduction. Indeed, several families of mechanosensory channels have been identified whereby pores open as a function of lipid stretch or tethers attached to extra-or intracellular structures (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Cox <em>et al.</em> 2018a</a>,<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">b</a>).</p><p id="p-49">Simply speaking, no single molecule has been identified to be associated with all mechanotransduction across phyla such as the ecdysozoan TRP channels also found in bony fish but absent in mammals (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Beisel <em>et al.</em> 2010</a>; <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Qiu and Muller 2018</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Cox <em>et al.</em> 2018a</a>). Likewise, the ubiquitous Piezo mechanotransduction channels in Merkel cells (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Delmas and Coste 2013</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Ranade <em>et al.</em> 2015</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cox <em>et al.</em> 2016</a>) were hypothesized to be the mechanotransduction channel of hair cells (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Arendt <em>et al.</em> 2016</a>) but have meanwhile been found not to be directly associated with the mechanotransduction process (<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Wu <em>et al.</em> 2017</a>). Molecular analysis of mammalian mutations meanwhile has focused on the family of TMC (transmembrane channel-like proteins) as possibly involved in hair cell mechanotransduction (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Delmas and Coste 2013</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Pan <em>et al.</em> 2013</a>) and replacement of mutated <em>TMC</em> can restore hearing (<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Yoshimura <em>et al.</em> 2018</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Shibata <em>et al.</em> 2016</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Askew <em>et al.</em> 2015</a>).</p><p id="p-50">More recently, molecular analysis has established that the TMC forms a mechanosensory pore as a homodimer with each subunit having ten predicted transmembrane domains (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Pan <em>et al.</em> 2018</a>). However, the detailed transmembrane protein dimer is unclear as other proteins are needed to transport the TMC channels to the tip (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Pacentine and Nicolson 2019</a>) and the entire complex of the vertebrate mechanosensory channel and its attachment to intra- and extracellular tethers remains unresolved (<a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Qiu and Muller 2018</a>). To what extent TMC family channel evolution aligns with mechanosensory cell and organ evolution remains to be seen but is already indicating unique features of ctenophores at every level (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Fritzsch <em>et al.</em> 2015</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">2006</a>).</p><p id="p-51">Recent work also establishes that ecdysozoan <em>Tmc<span class="underline">123</span></em> paralogs function in body kinesthesia (proprioception), sensory control of locomotion or egg-laying behavior via membrane depolarization, and nociception (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Guo <em>et al.</em> 2016</a>; <a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Yue <em>et al.</em> 2018</a>; <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Wang <em>et al.</em> 2016</a>). In summary, our study further lays the groundwork for understanding the molecular history of a sensory channel family in the context of the evolution of developmental gene regulatory circuits in different animal lineages (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Beisel <em>et al.</em> 2010</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Fritzsch <em>et al.</em> 2000</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Fritzsch and Elliott 2017</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Corbo <em>et al.</em> 1997</a>; <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Cox <em>et al.</em> 2018a</a>).</p></div><div id="sec-20" class="subsection"><h3>Regulation of metabolism in larval and adult morphologies</h3><p id="p-52">We speculate on the possible connection of the <em>MLX</em>/<em>MLXIP</em> gene duplication (<a id="xref-fig-6-7" class="xref-fig" href="#F6">Figure 6</a>) to an evolutionary transition from a holopelagic to a biphasic pelago-benthic life cycle in the stem-benthozoan lineage. This finding adds to a growing picture of metabolic and chaperone gene loss and gene innovation in early animal evolution (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Erives and Fassler 2015</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Richter <em>et al.</em> 2018</a>). Basic-helix-loop-helix (bHLH) TFs function as obligate dimers for DNA-binding (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Murre <em>et al.</em> 1989</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lassar <em>et al.</em> 1991</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Murre <em>et al.</em> 1991</a>) and their combinatorial dimerizations are a dominant aspect of their regulatory interactions (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Murre <em>et al.</em> 1989</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Neuhold and Wold 1993</a>) and of their evolution (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Brown <em>et al.</em> 2008</a>). Thus, it seems unlikely that a second MLX-related gene encoding a heterodimeric partner to the single gene found in ctenophores is artifactually missing in multiple sequenced genomes and transcriptomes (<a id="xref-ref-66-5" class="xref-bibr" href="#ref-66">Ryan <em>et al.</em> 2013</a>; <a id="xref-ref-46-5" class="xref-bibr" href="#ref-46">Moroz <em>et al.</em> 2014</a>). It is possible then that the single MLX-like gene corresponds to an evolutionary intermediate state in which a single gene encodes a homodimeric bHLH TF. But this idea needs to be tested.</p><p id="p-53">The MLX:MLXIP/MLXIPL heterodimeric TF acts as a transcriptional regulator of metabolic pathways (<em>e.g.</em>, lipogenesis genes) in response to variation in intracellular sugar concentrations (<a id="xref-ref-91-2" class="xref-bibr" href="#ref-91">Yamashita <em>et al.</em> 2001</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Iizuka <em>et al.</em> 2004</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Sans <em>et al.</em> 2006</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Havula <em>et al.</em> 2013</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Havula and Hietakangas 2018</a>). In this regard it is interesting to speculate that the duplication evolved in connection with the evolution of a biphasic pelago-benthic life cycle featuring a pelagic feeding larva and a benthic feeding adult (whether it was a motile planula or sessile adult). Pelagic larval and benthic adult feeding forms would have distinct nutritional intakes associated with dissimilar feeding strategies and dissimilar nominal parameters. This bimodal variation would have evolved on top of variation associated with just a single feeding strategy. Thus, a stem-benthozoic ancestor may have demanded additional complexity in metabolic regulation that was subsequently afforded by duplicated paralogs to expand regulation of life cycle cell forms into differentiation of different cell types (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Fritzsch <em>et al.</em> 2015</a>).</p></div><div id="sec-21" class="subsection"><h3>Methodological pitfalls in using paralogy to infer phylogeny</h3><p id="p-54">While we may have identified some of the best candidates to date depicting early metazoan branching via gene duplications, we want to point out one possible pitfall in this approach. One of our candidate paralogies encodes the Almondex (Amx) superfamily, which persisted in our screen through the stages where we required a missing orthology call for either the sponge <em>Amphimedon</em> or the ctenophore <em>Mnemiopsis</em> (see gene family 23 in size group three of Table S1). Almondex is a neurogenic TM2-domain containing protein characterized as a genetic modifier of Notch signaling in <em>Drosophila</em> (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Shannon 1972</a>, <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">1973</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Michellod <em>et al.</em> 2003</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Michellod and Randsholt 2008</a>). Thus, as a neurogenic locus this candidate gene family could have made biological sense if it linked Ctenophora as a sister-group to Eumetazoa in the proposed Neuralia clade (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Nielsen 2008</a>). However, when we analyzed the Amx super-family in more depth, we found that one of the three paralogs was missing in the ctenophore <em>Mnemiopsis</em> (see the <em>CG11103</em> clade in <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>), while all three paralogs were present in the sponge <em>Amphimedon</em> (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8</a>).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/811/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The neurogenic Almondex superfamily. Shown is a gene phylogeny composed of three paralogy groups (labeled by their Drosophila gene names), which originated in a choanozoan ancestor given the presence in both choanoflagellates (red asterisks) and animals. Given this ancient provenance, this tree may be rooted with any one of the three sub-clades as out-group. As explained in the text, ctenophores appear to be missing some of these paralogs (e.g., CG11103), and were it not for the presence of each paralog in at least one choanoflagellate, alternate tree rooting strategies suggestive of gene duplications could have been possible. This tree was constructed using Neighbor-Joining (distance-based) with 1000 boot-strap replicates sampled from an alignment with 207 alignment columns. Boot-strap supports are shown only for values &#x2265; 40" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-780785390" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The neurogenic Almondex superfamily. Shown is a gene phylogeny composed of three paralogy groups (labeled by their &lt;em&gt;Drosophila&lt;/em&gt; gene names), which originated in a choanozoan ancestor given the presence in both choanoflagellates (red asterisks) and animals. Given this ancient provenance, this tree may be rooted with any one of the three sub-clades as out-group. As explained in the text, ctenophores appear to be missing some of these paralogs (&lt;em&gt;e.g.&lt;/em&gt;, CG11103), and were it not for the presence of each paralog in at least one choanoflagellate, alternate tree rooting strategies suggestive of gene duplications could have been possible. This tree was constructed using Neighbor-Joining (distance-based) with 1000 boot-strap replicates sampled from an alignment with 207 alignment columns. Boot-strap supports are shown only for values &#x2265; 40&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/811/F8.medium.gif" width="440" height="323"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/10/2/811/F8.medium.gif" width="440" height="323"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/811/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/811/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495676" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-55" class="first-child">The neurogenic Almondex superfamily. Shown is a gene phylogeny composed of three paralogy groups (labeled by their <em>Drosophila</em> gene names), which originated in a choanozoan ancestor given the presence in both choanoflagellates (red asterisks) and animals. Given this ancient provenance, this tree may be rooted with any one of the three sub-clades as out-group. As explained in the text, ctenophores appear to be missing some of these paralogs (<em>e.g.</em>, CG11103), and were it not for the presence of each paralog in at least one choanoflagellate, alternate tree rooting strategies suggestive of gene duplications could have been possible. This tree was constructed using Neighbor-Joining (distance-based) with 1000 boot-strap replicates sampled from an alignment with 207 alignment columns. Boot-strap supports are shown only for values ≥ 40</p><div class="sb-div caption-clear"></div></div></div><p id="p-56">To rule out <em>Mnemiopsis</em>-specific gene losses, we also searched for orthologs of each paralogy group in other ctenophores, and were indeed able to find Amx in the ctenophore <em>Pleurobrachia</em> (middle clade in <a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8</a>). Superficially this tree began to resemble the MLX-MLXIP tree (<a id="xref-fig-6-8" class="xref-fig" href="#F6">Figure 6B</a>) except in its important relation to choanoflagellates. Unlike the single choanoflagellate MAX gene (<a id="xref-fig-6-9" class="xref-fig" href="#F6">Figure 6B</a>), the Amx superfamily tree shows that the choanoflagellate <em>Salpingoeca rosetta</em> has each of the three paralogs (red asterisks in <a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 8</a>). This result implies that ctenophores have lost the <em>CG11103</em> ortholog, and have either lost or are missing the <em>CG10795</em> ortholog (<em>Pleurobrachia</em>) or else possess a divergent version of this gene (<em>Mnemiopsis</em>).</p><p id="p-57">In the case of the <em>Amx</em> superfamily, the evolutionary maintenance of <em>Amx</em> paralogs in at least one choanoflagellate is critical to ruling out duplicated genes, which would be possible with alternate root-choices. With the absence of paralogs for the choanoflagellate <em>Salpingoeca</em> in the <em>Amx</em> and <em>CG11103</em> sub-clades, which already appear to be lost in the choanoflagellate <em>Monosiga</em>, we may justifiably have rooted the tree in <a id="xref-fig-8-5" class="xref-fig" href="#F8">Figure 8</a> so that the choanoflagellate <em>CG10795</em> clade is the outgroup. A tree rooted in this way (in the hypothetical absence of the two lone <em>Salpingoeca</em> sequences) allows the possibility that <em>CG11103</em> could be a Benthozoa-specific duplication. Thus, in using gene duplications to infer phylogenetic branching it is important to be mindful of unchecked misinterpretations facilitated by true gene loss. If this type of error is ubiquitous, that we should be able to find examples showing duplications shared by ctenophores and bilaterians even if they are false-positive duplications caused by unconstrained root choices. Further examples will be necessary to see if this is the case. But in summary, our comparative genomic screen designed to identify candidate families duplicated prior to the evolution of Eumetazoa has only identified examples consistent with a clade of <em>Benthozoa</em> that unites Porifera and Eumetazoa as the sister-clade to Ctenophora.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-58">Funding support from the NIH (R01 AG060504 to BF) and an anonymous Iowa Foundation grant (to AE).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-59">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.11444400">https://doi.org/10.25387/g3.11444400</a>.</p></li><li class="fn" id="fn-2"><p id="p-60"><em>Communicating editor: R. Kulathinal</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-07-15"><span class="received-label">Received </span>July 15, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-12-19"><span class="accepted-label">Accepted </span>December 19, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Erives and Fritzsch</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.1" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>,  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Gish</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Miller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. W.</span> <span class="cit-name-surname">Myers</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Lipman</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0022-2836(05)80360-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.2" data-doi="10.7554/eLife.20010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arendt</span>,  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The enigmatic xenopsins.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">3</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7554/eLife.31781</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D6%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.20010%26rft_id%253Dinfo%253Apmid%252F28112643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.20010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28112643&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.3" data-doi="10.1038/nrg.2016.127"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arendt</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Musser</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. V.</span> <span class="cit-name-surname">Baker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bergman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Cepko</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The origin and evolution of cell types.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">744</span>–<span class="cit-lpage">757</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg.2016.127</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D17%26rft.spage%253D744%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg.2016.127%26rft_id%253Dinfo%253Apmid%252F27818507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg.2016.127&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27818507&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.4" data-doi="10.1126/scitranslmed.aab1996"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Askew</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Rochat</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Pan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Asai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Ahmed</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title"><em>Tmc</em> gene therapy restores auditory function in deaf mice.</span> <abbr class="cit-jnl-abbrev">Sci. Transl. Med.</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">295ra108</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/scitranslmed.aab1996</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BTranslational%2BMedicine%26rft.stitle%253DSci%2BTransl%2BMed%26rft.aulast%253DAskew%26rft.auinit1%253DC.%26rft.volume%253D7%26rft.issue%253D295%26rft.spage%253D295ra108%26rft.epage%253D295ra108%26rft.atitle%253DTmc%2Bgene%2Btherapy%2Brestores%2Bauditory%2Bfunction%2Bin%2Bdeaf%2Bmice%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscitranslmed.aab1996%26rft_id%253Dinfo%253Apmid%252F26157030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6InNjaXRyYW5zbWVkIjtzOjU6InJlc2lkIjtzOjE0OiI3LzI5NS8yOTVyYTEwOCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvODExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.5" data-doi="10.1093/sysbio/syr100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ayres</span>,  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Darling</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Zwickl</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Beerli</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. T.</span> <span class="cit-name-surname">Holder</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">BEAGLE: an application programming interface and high-performance computing library for statistical phylogenetics.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">170</span>–<span class="cit-lpage">173</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/sysbio/syr100</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyr100%26rft_id%253Dinfo%253Apmid%252F21963610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syr100&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21963610&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.6" data-doi="10.7554/eLife.38433"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ballesteros</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Fenollar-Ferrer</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. J.</span> <span class="cit-name-surname">Swartz</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Structural relationship between the putative hair cell mechanotransduction channel TMC1 and TMEM16 proteins.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7554/eLife.38433</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.38433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.38433&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.811.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beisel</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">He</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Hallworth</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Fritzsch</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source">Genetics of mechanoreceptor evolution and development</span>, <span class="cit-publ-name">Elsevier Inc.</span>, <span class="cit-publ-loc">Amsterdam</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.8" data-doi="10.1242/dev.02219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bökel</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Dass</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Wilsch-Bräuninger</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Roth</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title"><em>Drosophila</em> Cornichon acts as cargo receptor for ER export of the TGFalpha-like growth factor Gurken.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">459</span>–<span class="cit-lpage">470</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1242/dev.02219</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.02219%26rft_id%253Dinfo%253Apmid%252F16396907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czo5OiIxMzMvMy80NTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzgxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.9" data-doi="10.1186/s12864-015-2146-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borowiec</span>,  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. K.</span> <span class="cit-name-surname">Lee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Chiu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. C.</span> <span class="cit-name-surname">Plachetzki</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Extracting phylogenetic signal and accounting for bias in whole-genome data sets supports the Ctenophora as sister to remaining Metazoa.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">987</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-2146-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D987%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2146-4%26rft_id%253Dinfo%253Apmid%252F26596625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2146-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26596625&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.10" data-doi="10.1186/1471-2164-9-442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>,  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Cole</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Erives</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Evolution of the holozoan ribosome biogenesis regulon.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">442</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-9-442</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DBrown%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D9%26rft.spage%253D442%26rft.epage%253D442%26rft.atitle%253DEvolution%2Bof%2Bthe%2Bholozoan%2Bribosome%2Bbiogenesis%2Bregulon.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-442%26rft_id%253Dinfo%253Apmid%252F18816399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-442&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18816399&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.11" data-doi="10.1242/jcs.004200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Castro</span>,  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Piscopo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Nakagawa</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Derynck</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Cornichon regulates transport and secretion of TGFalpha-related proteins in metazoan cells.</span> <abbr class="cit-jnl-abbrev">J. Cell Sci.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">2454</span>–<span class="cit-lpage">2466</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1242/jcs.004200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BCell%2BScience%26rft.stitle%253DJ.%2BCell%2BSci.%26rft.aulast%253DCastro%26rft.auinit1%253DC.%2BP.%26rft.volume%253D120%26rft.issue%253D14%26rft.spage%253D2454%26rft.epage%253D2466%26rft.atitle%253DCornichon%2Bregulates%2Btransport%2Band%2Bsecretion%2Bof%2BTGF%257Balpha%257D-related%2Bproteins%2Bin%2Bmetazoan%2Bcells%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjcs.004200%26rft_id%253Dinfo%253Apmid%252F17607000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiam9jZXMiO3M6NToicmVzaWQiO3M6MTE6IjEyMC8xNC8yNDU0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi84MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corbo</span>,  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Erives</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Di Gregorio</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Chang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Levine</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Dorsoventral patterning of the vertebrate neural tube is conserved in a protochordate.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">2335</span>–<span class="cit-lpage">2344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DCorbo%26rft.auinit1%253DJ.%26rft.volume%253D124%26rft.issue%253D12%26rft.spage%253D2335%26rft.epage%253D2344%26rft.atitle%253DDorsoventral%2Bpatterning%2Bof%2Bthe%2Bvertebrate%2Bneural%2Btube%2Bis%2Bconserved%2Bin%2Ba%2Bprotochordate%26rft_id%253Dinfo%253Apmid%252F9199360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTI0LzEyLzIzMzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzgxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.13" data-doi="10.1038/ncomms10366"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>,  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Bae</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Ziegler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Hartley</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Nikolova-Krstevski</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Removal of the mechanoprotective influence of the cytoskeleton reveals PIEZO1 is gated by bilayer tension.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">10366</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms10366</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D7%26rft.spage%253D10366%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms10366%26rft_id%253Dinfo%253Apmid%252F26785635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms10366&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26785635&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.14" data-doi="10.1146/annurev-physiol-021317-121351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>,  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Bavi</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Martinac</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">a</span> <span class="cit-article-title">Bacterial mechanosensors.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Physiol.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">93</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-physiol-021317-121351</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BPhysiol.%26rft.volume%253D80%26rft.spage%253D71%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-physiol-021317-121351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-physiol-021317-121351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.15" data-doi="10.1016/j.bpj.2017.11.641"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>,  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Bavi</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Martinac</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">b</span> <span class="cit-article-title">Cytoskeleton-associated proteins modulate the tension sensitivity of Piezo1.</span> <abbr class="cit-jnl-abbrev">Biophys. J.</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">111a</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.bpj.2017.11.641</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiophys.%2BJ.%26rft.volume%253D114%26rft.spage%253D111a%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bpj.2017.11.641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bpj.2017.11.641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.16" data-doi="10.1016/j.cell.2013.09.026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Delmas</span>,  <span class="cit-name-given-names">P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Coste</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mechano-gated ion channels in sensory systems.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">278</span>–<span class="cit-lpage">284</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2013.09.026</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D155%26rft.spage%253D278%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.09.026%26rft_id%253Dinfo%253Apmid%252F24120130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.09.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24120130&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000325719800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.17" data-doi="10.1093/bioinformatics/bti525"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durinck</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Moreau</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kasprzyk</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Davis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">De Moor</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Biomart and bioconductor: a powerful link between biological databases and microarray data analysis.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">3439</span>–<span class="cit-lpage">3440</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bti525</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DDurinck%26rft.auinit1%253DS.%26rft.volume%253D21%26rft.issue%253D16%26rft.spage%253D3439%26rft.epage%253D3440%26rft.atitle%253DBioMart%2Band%2BBioconductor%253A%2Ba%2Bpowerful%2Blink%2Bbetween%2Bbiological%2Bdatabases%2Band%2Bmicroarray%2Bdata%2Banalysis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti525%26rft_id%253Dinfo%253Apmid%252F16082012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti525&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16082012&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231360600020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.18" data-doi="10.1007/s00427-015-0508-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Erives</span>,  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genes conserved in bilaterians but jointly lost with <em>Myc</em> during nematode evolution are enriched in cell proliferation and cell migration functions.</span> <abbr class="cit-jnl-abbrev">Dev. Genes Evol.</abbr> <span class="cit-vol">225</span>: <span class="cit-fpage">259</span>–<span class="cit-lpage">273</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00427-015-0508-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BGenes%2BEvol.%26rft.volume%253D225%26rft.spage%253D259%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00427-015-0508-1%26rft_id%253Dinfo%253Apmid%252F26173873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00427-015-0508-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26173873&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.19" data-doi="10.1371/journal.pone.0117192"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Erives</span>,  <span class="cit-name-given-names">A. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Fassler</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Metabolic and chaperone gene loss marks the origin of animals: evidence for Hsp104 and Hsp78 chaperones sharing mitochondrial enzymes as clients.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-elocation-id">e0117192</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0117192</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0117192%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0117192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0117192&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.20" data-doi="10.1016/j.cub.2017.11.008"><div class="cit-metadata"><cite>Feuda, R., M. Dohrmann, W. Pett, H. Philippe, O. Rota-Stabelli, <em>et al.</em>, 2017 Improved modeling of compositional heterogeneity supports sponges as sister to all other animals. <em>Curr Biol</em> <strong>27</strong>: 3864–3870 e4.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cub.2017.11.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D27%26rft.spage%253Dp3864%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2017.11.008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2017.11.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-confproc" id="cit-10.2.811.21" data-doi="10.1073/pnas.1204609109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feuda</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. C.</span> <span class="cit-name-surname">Hamilton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. O.</span> <span class="cit-name-surname">McInerney</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Pisani</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Metazoan opsin evolution reveals a simple route to animal vision.</span> <span class="cit-source">Proc. Natl. Acad. Sci. USA</span> <span class="cit-vol">109</span>: <span class="cit-fpage">18868</span>–<span class="cit-lpage">18872</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1204609109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1204609109%26rft_id%253Dinfo%253Apmid%252F23112152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzQ2LzE4ODY4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi84MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.22" data-doi="10.1126/science.155.3760.279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fitch</span>,  <span class="cit-name-given-names">W. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Margoliash</span></span></li></ol><cite>, <span class="cit-pub-date">1967</span> <span class="cit-article-title">Construction of phylogenetic trees.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">279</span>–<span class="cit-lpage">284</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.155.3760.279</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFitch%26rft.auinit1%253DW.%2BM.%26rft.volume%253D155%26rft.issue%253D3760%26rft.spage%253D279%26rft.epage%253D284%26rft.atitle%253DConstruction%2Bof%2BPhylogenetic%2BTrees%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.155.3760.279%26rft_id%253Dinfo%253Apmid%252F5334057%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJzY2kiO3M6NToicmVzaWQiO3M6MTI6IjE1NS8zNzYwLzI3OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvODExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.23" data-doi="10.1097/00001756-200011270-00013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fritzsch</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. W.</span> <span class="cit-name-surname">Beisel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Bermingham</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Developmental evolutionary biology of the vertebrate ear: conserving mechanoelectric transduction and developmental pathways in diverging morphologies.</span> <abbr class="cit-jnl-abbrev">Neuroreport</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R35</span>–<span class="cit-lpage">R44</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1097/00001756-200011270-00013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuroreport%26rft.stitle%253DNeuroreport%26rft.aulast%253DFritzsch%26rft.auinit1%253DB.%26rft.volume%253D11%26rft.issue%253D17%26rft.spage%253DR35%26rft.epage%253DR44%26rft.atitle%253DDevelopmental%2Bevolutionary%2Bbiology%2Bof%2Bthe%2Bvertebrate%2Bear%253A%2Bconserving%2Bmechanoelectric%2Btransduction%2Band%2Bdevelopmental%2Bpathways%2Bin%2Bdiverging%2Bmorphologies.%26rft_id%253Dinfo%253Adoi%252F10.1097%252F00001756-200011270-00013%26rft_id%253Dinfo%253Apmid%252F11117521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1097/00001756-200011270-00013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11117521&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165569800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.24" data-doi="10.1016/j.ydbio.2017.08.034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fritzsch</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. L.</span> <span class="cit-name-surname">Elliott</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Gene, cell, and organ multiplication drives inner ear evolution.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">431</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ydbio.2017.08.034</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BBiol.%26rft.volume%253D431%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2017.08.034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2017.08.034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.25" data-doi="10.1007/s00441-014-2043-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fritzsch</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Jahan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Pan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. L.</span> <span class="cit-name-surname">Elliott</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Evolving gene regulatory networks into cellular networks guiding adaptive behavior: an outline how single cells could have evolved into a centralized neurosensory system.</span> <abbr class="cit-jnl-abbrev">Cell Tissue Res.</abbr> <span class="cit-vol">359</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">313</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00441-014-2043-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BTissue%2BRes.%26rft.volume%253D359%26rft.spage%253D295%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00441-014-2043-1%26rft_id%253Dinfo%253Apmid%252F25416504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00441-014-2043-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25416504&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.26" data-doi="10.1016/j.brainres.2006.02.078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fritzsch</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Pauley</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. W.</span> <span class="cit-name-surname">Beisel</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Cells, molecules and morphogenesis: the making of the vertebrate ear.</span> <abbr class="cit-jnl-abbrev">Brain Res.</abbr> <span class="cit-vol">1091</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">171</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.brainres.2006.02.078</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrain%2Bresearch%26rft.stitle%253DBrain%2BRes%26rft.aulast%253DFritzsch%26rft.auinit1%253DB.%26rft.volume%253D1091%26rft.issue%253D1%26rft.spage%253D151%26rft.epage%253D171%26rft.atitle%253DCells%252C%2Bmolecules%2Band%2Bmorphogenesis%253A%2Bthe%2Bmaking%2Bof%2Bthe%2Bvertebrate%2Bear.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.brainres.2006.02.078%26rft_id%253Dinfo%253Apmid%252F16643865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.brainres.2006.02.078&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16643865&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239351500018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.27" data-doi="10.1073/pnas.1606537113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Meltzer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Zanini</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title"><em>Transmembrane channel-like</em> (<em>Tmc</em>) gene regulates <em>Drosophila</em> larval locomotion.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">7243</span>–<span class="cit-lpage">7248</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1606537113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1606537113%26rft_id%253Dinfo%253Apmid%252F27298354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEzLzI2LzcyNDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzgxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.28" data-doi="10.1093/nar/gkp265"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haider</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Ballester</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Smedley</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Rice</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">BioMart Central Portal–unified access to biological data.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">W23</span>–<span class="cit-lpage">W27</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkp265</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp265%26rft_id%253Dinfo%253Apmid%252F19420058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkp265&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19420058&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267889100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.29" data-doi="10.1016/j.ceb.2017.12.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Havula</span>,  <span class="cit-name-given-names">E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Hietakangas</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Sugar sensing by ChREBP/Mondo-Mlx-new insight into downstream regulatory networks and integration of nutrient-derived signals.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Cell Biol.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">96</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ceb.2017.12.007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BCell%2BBiol.%26rft.volume%253D51%26rft.spage%253D89%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ceb.2017.12.007%26rft_id%253Dinfo%253Apmid%252F29278834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ceb.2017.12.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29278834&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.30" data-doi="10.1371/journal.pgen.1003438"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Havula</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Teesalu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Hyotylainen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Seppala</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Hasygar</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mondo/ChREBP-Mlx-regulated transcriptional network is essential for dietary sugar tolerance in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e1003438</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003438</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003438%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003438%26rft_id%253Dinfo%253Apmid%252F23593032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003438&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23593032&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.31" data-doi="10.1002/bies.20813"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hemmrich</span>,  <span class="cit-name-given-names">G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. C.</span> <span class="cit-name-surname">Bosch</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Compagen, a comparative genomics platform for early branching metazoan animals, reveals early origins of genes regulating stem-cell differentiation.</span> <abbr class="cit-jnl-abbrev">BioEssays</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1010</span>–<span class="cit-lpage">1018</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/bies.20813</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DHemmrich%26rft.auinit1%253DG.%26rft.volume%253D30%26rft.issue%253D10%26rft.spage%253D1010%26rft.epage%253D1018%26rft.atitle%253DCompagen%252C%2Ba%2Bcomparative%2Bgenomics%2Bplatform%2Bfor%2Bearly%2Bbranching%2Bmetazoan%2Banimals%252C%2Breveals%2Bearly%2Borigins%2Bof%2Bgenes%2Bregulating%2Bstem-cell%2Bdifferentiation.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.20813%26rft_id%253Dinfo%253Apmid%252F18800383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.20813&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18800383&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259959700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.32" data-doi="10.1016/j.neuron.2013.01.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Herring</span>,  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Shi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. H.</span> <span class="cit-name-surname">Suh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. Y.</span> <span class="cit-name-surname">Zheng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Blankenship</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Cornichon proteins determine the subunit composition of synaptic AMPA receptors.</span> <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">1083</span>–<span class="cit-lpage">1096</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.neuron.2013.01.017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D77%26rft.spage%253D1083%26rft.atitle%253DCORNICHON%2BPROTEINS%2BDETERMINE%2BTHE%2BSUBUNIT%2BCOMPOSITION%2BOF%2BSYNAPTIC%2BAMPA%2BRECEPTORS%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2013.01.017%26rft_id%253Dinfo%253Apmid%252F23522044%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2013.01.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23522044&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.33" data-doi="10.1098/rstb.2015.0480"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>,  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Marlétaz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Maeso</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. L.</span> <span class="cit-name-surname">Dunwell</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Paps</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">New genes from old: asymmetric divergence of gene duplicates and the evolution of development.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">372</span>: <span class="cit-elocation-id">20150480</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rstb.2015.0480</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BLond.%2BB%2BBiol.%2BSci.%26rft.volume%253D372%26rft.spage%253D20150480%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2015.0480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2015.0480&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.34" data-doi="10.1093/bioinformatics/17.8.754"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huelsenbeck</span>,  <span class="cit-name-given-names">J. P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Ronquist</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">MRBAYES: Bayesian inference of phylogenetic trees.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">754</span>–<span class="cit-lpage">755</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/17.8.754</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DHuelsenbeck%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D17%26rft.issue%253D8%26rft.spage%253D754%26rft.epage%253D755%26rft.atitle%253DMRBAYES%253A%2BBayesian%2Binference%2Bof%2Bphylogenetic%2Btrees%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F17.8.754%26rft_id%253Dinfo%253Apmid%252F11524383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/17.8.754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11524383&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171021000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.35" data-doi="10.1073/pnas.0401516101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iizuka</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. K.</span> <span class="cit-name-surname">Bruick</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Liang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Horton</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Uyeda</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Deficiency of carbohydrate response element-binding protein (ChREBP) reduces lipogenesis as well as glycolysis.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">7281</span>–<span class="cit-lpage">7286</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0401516101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0401516101%26rft_id%253Dinfo%253Apmid%252F15118080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzE5LzcyODEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzgxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.36" data-doi="10.1186/2041-9139-6-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jékely</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Paps</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Nielsen</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The phylogenetic position of ctenophores and the origin(s) of nervous systems.</span> <abbr class="cit-jnl-abbrev">Evodevo</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/2041-9139-6-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvodevo%26rft.volume%253D6%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F2041-9139-6-1%26rft_id%253Dinfo%253Apmid%252F25905000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/2041-9139-6-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25905000&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.811.37" data-doi="10.1016/j.neuron.2019.10.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Kusakizako</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Pan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">TMC1 and TMC2 proteins are pore-forming subunits of mechanosensitive ion channels.</span> <span class="cit-source">Neuron</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.neuron.2019.10.017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2019.10.017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2019.10.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.38" data-doi="10.1038/nature09984"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Forn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Gadelha</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Egan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Bass</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Discovery of novel intermediate forms redefines the fungal tree of life.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">474</span>: <span class="cit-fpage">200</span>–<span class="cit-lpage">203</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature09984</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJones%26rft.auinit1%253DM.%2BD.%26rft.volume%253D474%26rft.issue%253D7350%26rft.spage%253D200%26rft.epage%253D203%26rft.atitle%253DDiscovery%2Bof%2Bnovel%2Bintermediate%2Bforms%2Bredefines%2Bthe%2Bfungal%2Btree%2Bof%2Blife.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09984%26rft_id%253Dinfo%253Apmid%252F21562490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09984&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21562490&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291397800047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.39" data-doi="10.1186/1471-2164-4-24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keresztes</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Mutai</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Heller</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">TMC and EVER genes belong to a larger novel family, the TMC gene family encoding transmembrane proteins.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">24</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-4-24</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DKeresztes%26rft.auinit1%253DG.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D24%26rft.epage%253D24%26rft.atitle%253DTMC%2Band%2BEVER%2Bgenes%2Bbelong%2Bto%2Ba%2Blarger%2Bnovel%2Bfamily%252C%2Bthe%2BTMC%2Bgene%2Bfamily%2Bencoding%2Btransmembrane%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-4-24%26rft_id%253Dinfo%253Apmid%252F12812529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-4-24&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12812529&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.40" data-doi="10.1093/molbev/msw054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Stecher</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Tamura</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">MEGA7: Molecular evolutionary genetics analysis version 7.0 for bigger datasets.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1870</span>–<span class="cit-lpage">1874</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msw054</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw054%26rft_id%253Dinfo%253Apmid%252F27004904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27004904&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.41" data-doi="10.1038/nature03896"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kung</span>,  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A possible unifying principle for mechanosensation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">436</span>: <span class="cit-fpage">647</span>–<span class="cit-lpage">654</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature03896</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKung%26rft.auinit1%253DC.%26rft.volume%253D436%26rft.issue%253D7051%26rft.spage%253D647%26rft.epage%253D654%26rft.atitle%253DA%2Bpossible%2Bunifying%2Bprinciple%2Bfor%2Bmechanosensation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03896%26rft_id%253Dinfo%253Apmid%252F16079835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03896&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16079835&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230964500032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.42" data-doi="10.1016/0092-8674(91)90620-E"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lassar</span>,  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Davis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. E.</span> <span class="cit-name-surname">Wright</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Kadesch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Murre</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Functional activity of myogenic HLH proteins requires hetero-oligomerization with E12/E47-like proteins in vivo.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">305</span>–<span class="cit-lpage">315</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0092-8674(91)90620-E</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLassar%26rft.auinit1%253DA.%2BB.%26rft.volume%253D66%26rft.issue%253D2%26rft.spage%253D305%26rft.epage%253D315%26rft.atitle%253DFunctional%2Bactivity%2Bof%2Bmyogenic%2BHLH%2Bproteins%2Brequires%2Bhetero-oligomerization%2Bwith%2BE12%252FE47-like%2Bproteins%2Bin%2Bvivo.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252891%252990620-E%26rft_id%253Dinfo%253Apmid%252F1649701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(91)90620-E&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1649701&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FY71200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">O’Hely</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Walsh</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Force</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The probability of preservation of a newly arisen gene duplicate.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">1789</span>–<span class="cit-lpage">1804</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D159%26rft.issue%253D4%26rft.spage%253D1789%26rft.epage%253D1804%26rft.atitle%253DThe%2BProbability%2Bof%2BPreservation%2Bof%2Ba%2BNewly%2BArisen%2BGene%2BDuplicate%26rft_id%253Dinfo%253Apmid%252F11779815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1OS80LzE3ODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzgxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.44" data-doi="10.1002/gene.10233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michellod</span>,  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Forquignon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Santamaria</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. B.</span> <span class="cit-name-surname">Randsholt</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Differential requirements for the neurogenic gene <em>almondex</em> during <em>Drosophila melanogaster</em> development.</span> <abbr class="cit-jnl-abbrev">Genesis</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">122</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gene.10233</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenesis%2B%2528New%2BYork%252C%2BN.Y.%2B%253A%2B%2B2000%2529%26rft.stitle%253DGenesis%26rft.aulast%253DMichellod%26rft.auinit1%253DM.%2BA.%26rft.volume%253D37%26rft.issue%253D3%26rft.spage%253D113%26rft.epage%253D122%26rft.atitle%253DDifferential%2Brequirements%2Bfor%2Bthe%2Bneurogenic%2Bgene%2Balmondex%2Bduring%2BDrosophila%2Bmelanogaster%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgene.10233%26rft_id%253Dinfo%253Apmid%252F14595834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gene.10233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14595834&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.45" data-doi="10.1016/j.brainresbull.2007.10.060"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michellod</span>,  <span class="cit-name-given-names">M. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. B.</span> <span class="cit-name-surname">Randsholt</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Implication of the <em>Drosophila</em> beta-amyloid peptide binding-like protein AMX in Notch signaling during early neurogenesis.</span> <abbr class="cit-jnl-abbrev">Brain Res. Bull.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">305</span>–<span class="cit-lpage">309</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.brainresbull.2007.10.060</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrain%2Bresearch%2Bbulletin%26rft.stitle%253DBrain%2BRes%2BBull%26rft.aulast%253DMichellod%26rft.auinit1%253DM.%2BA.%26rft.volume%253D75%26rft.issue%253D2-4%26rft.spage%253D305%26rft.epage%253D309%26rft.atitle%253DImplication%2Bof%2Bthe%2BDrosophila%2Bbeta-amyloid%2Bpeptide%2Bbinding-like%2Bprotein%2BAMX%2Bin%2BNotch%2Bsignaling%2Bduring%2Bearly%2Bneurogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.brainresbull.2007.10.060%26rft_id%253Dinfo%253Apmid%252F18331889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.brainresbull.2007.10.060&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18331889&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.46" data-doi="10.1038/nature13400"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moroz</span>,  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Kocot</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Citarella</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Dosung</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. P.</span> <span class="cit-name-surname">Norekian</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The ctenophore genome and the evolutionary origins of neural systems.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">510</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">114</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature13400</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D510%26rft.spage%253D109%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13400%26rft_id%253Dinfo%253Apmid%252F24847885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13400&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24847885&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000336768900038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.47" data-doi="10.1016/0092-8674(89)90682-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murre</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. S.</span> <span class="cit-name-surname">McCaw</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Baltimore</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">A new DNA binding and dimerization motif in immunoglobulin enhancer binding, <em>daughterless</em>, <em>MyoD</em>, and <em>myc</em> proteins.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">777</span>–<span class="cit-lpage">783</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0092-8674(89)90682-X</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMurre%26rft.auinit1%253DC.%26rft.volume%253D56%26rft.issue%253D5%26rft.spage%253D777%26rft.epage%253D783%26rft.atitle%253DA%2Bnew%2BDNA%2Bbinding%2Band%2Bdimerization%2Bmotif%2Bin%2Bimmunoglobulin%2Benhancer%2Bbinding%252C%2Bdaughterless%252C%2BMyoD%252C%2Band%2Bmyc%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252889%252990682-X%26rft_id%253Dinfo%253Apmid%252F2493990%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(89)90682-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2493990&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989T686200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.48" data-doi="10.1128/MCB.11.2.1156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murre</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Voronova</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Baltimore</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">B-cell- and myocyte-specific E2-box-binding factors contain E12/E47-like subunits.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1156</span>–<span class="cit-lpage">1160</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MCB.11.2.1156</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DMurre%26rft.auinit1%253DC%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D1156%26rft.epage%253D1160%26rft.atitle%253DB-cell-%2Band%2Bmyocyte-specific%2BE2-box-binding%2Bfactors%2Bcontain%2BE12%252FE47-like%2Bsubunits.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.11.2.1156%26rft_id%253Dinfo%253Apmid%252F1990271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjExLzIvMTE1NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvODExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.49" data-doi="10.7554/eLife.34110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murthy</span>,  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Dubin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Whitwam</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Jojoa-Cruz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Cahalan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">OSCA/TMEM63 are an evolutionarily conserved family of mechanically activated ion channels.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">17</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7554/eLife.41844</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.34110%26rft_id%253Dinfo%253Apmid%252F30247123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.34110&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30247123&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.50" data-doi="10.1016/0092-8674(93)90725-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neuhold</span>,  <span class="cit-name-given-names">L. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Wold</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">HLH forced dimers: tethering MyoD to E47 generates a dominant positive myogenic factor insulated from negative regulation by Id.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">1033</span>–<span class="cit-lpage">1042</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0092-8674(93)90725-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DNeuhold%26rft.auinit1%253DL.%2BA.%26rft.volume%253D74%26rft.issue%253D6%26rft.spage%253D1033%26rft.epage%253D1042%26rft.atitle%253DHLH%2Bforced%2Bdimers%253A%2Btethering%2BMyoD%2Bto%2BE47%2Bgenerates%2Ba%2Bdominant%2Bpositive%2Bmyogenic%2Bfactor%2Binsulated%2Bfrom%2Bnegative%2Bregulation%2Bby%2BId.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252893%252990725-6%26rft_id%253Dinfo%253Apmid%252F7691411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(93)90725-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7691411&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LY49300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.51" data-doi="10.1111/j.1525-142X.2008.00231.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Six major steps in animal evolution: Are we derived sponge larvae?</span> <abbr class="cit-jnl-abbrev">Evol. Dev.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">257</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1525-142X.2008.00231.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2B%2526%2Bdevelopment%26rft.stitle%253DEvol%2BDev%26rft.aulast%253DNielsen%26rft.auinit1%253DC.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D241%26rft.epage%253D257%26rft.atitle%253DSix%2Bmajor%2Bsteps%2Bin%2Banimal%2Bevolution%253A%2Bare%2Bwe%2Bderived%2Bsponge%2Blarvae%253F%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1525-142X.2008.00231.x%26rft_id%253Dinfo%253Apmid%252F18315817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1525-142X.2008.00231.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18315817&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253707300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.52" data-doi="10.1186/1471-2148-13-171"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Life cycle evolution: Was the eumetazoan ancestor a holopelagic, planktotrophic gastraea?</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">171</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2148-13-171</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol.%2BBiol.%26rft.volume%253D13%26rft.spage%253D171%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-13-171%26rft_id%253Dinfo%253Apmid%252F23957497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-13-171&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23957497&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.53" data-doi="10.1371/journal.pone.0159559"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noutahi</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Semeria</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lafond</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Seguin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Boussau</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Efficient gene tree correction guided by genome evolution.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e0159559</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0159559</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0159559%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0159559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0159559&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.811.54" data-doi="10.1007/978-3-642-86659-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohno</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-source"><em>Evolution by gene duplication</em>.</span> <span class="cit-publ-name">Allen Unwin; Springer-Verlag</span>, <span class="cit-publ-loc">London, New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-642-86659-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2Bby%2Bgene%2Bduplication.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-642-86659-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-642-86659-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.55" data-doi="10.1371/journal.pgen.1007635"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pacentine</span>,  <span class="cit-name-given-names">I. V.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Nicolson</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Subunits of the mechano-electrical transduction channel, Tmc1/2b, require Tmie to localize in zebrafish sensory hair cells.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-elocation-id">e1007635</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1007635</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D15%26rft.spage%253De1007635%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1007635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1007635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.56" data-doi="10.1016/j.neuron.2018.07.033"><div class="cit-metadata"><cite>Pan, B., N. Akyuz, X. P. Liu, Y. Asai, C. Nist-Lund, <em>et al.</em>, 2018 TMC1 forms the pore of mechanosensory transduction channels in vertebrate inner ear hair cells. <em>Neuron</em> <strong>99</strong>: 736–753 e6.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.neuron.2018.07.033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D99%26rft.spage%253D736%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2018.07.033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2018.07.033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.57" data-doi="10.1016/j.neuron.2013.06.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. S.</span> <span class="cit-name-surname">Geleoc</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Asai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. C.</span> <span class="cit-name-surname">Horwitz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kurima</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">TMC1 and TMC2 are components of the mechanotransduction channel in hair cells of the mammalian inner ear.</span> <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">504</span>–<span class="cit-lpage">515</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.neuron.2013.06.019</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D79%26rft.spage%253D504%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2013.06.019%26rft_id%253Dinfo%253Apmid%252F23871232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2013.06.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23871232&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.58" data-doi="10.1093/molbev/msz013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pett</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Adamski</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Adamska</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. R.</span> <span class="cit-name-surname">Francis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Eitel</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">The role of homology and orthology in the phylogenomic analysis of metazoan gene content.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">643</span>–<span class="cit-lpage">649</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msz013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D36%26rft.spage%253D643%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsz013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msz013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.59" data-doi="10.1073/pnas.1518127112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pisani</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Pett</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Dohrmann</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Feuda</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Rota-Stabelli</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic data do not support comb jellies as the sister group to all other animals.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">15402</span>–<span class="cit-lpage">15407</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1518127112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1518127112%26rft_id%253Dinfo%253Apmid%252F26621703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEyLzUwLzE1NDAyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi84MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.60" data-doi="10.1126/science.1139158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Putnam</span>,  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Srivastava</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Hellsten</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Dirks</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Chapman</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Sea anemone genome reveals ancestral eumetazoan gene repertoire and genomic organization.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">317</span>: <span class="cit-fpage">86</span>–<span class="cit-lpage">94</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1139158</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DPutnam%26rft.auinit1%253DN.%2BH.%26rft.volume%253D317%26rft.issue%253D5834%26rft.spage%253D86%26rft.epage%253D94%26rft.atitle%253DSea%2BAnemone%2BGenome%2BReveals%2BAncestral%2BEumetazoan%2BGene%2BRepertoire%2Band%2BGenomic%2BOrganization%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1139158%26rft_id%253Dinfo%253Apmid%252F17615350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMTcvNTgzNC84NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvODExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.61" data-doi="10.3389/fncel.2018.00100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qiu</span>,  <span class="cit-name-given-names">X.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Muller</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Mechanically gated ion channels in mammalian hair cells.</span> <abbr class="cit-jnl-abbrev">Front. Cell. Neurosci.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">100</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fncel.2018.00100</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BCell.%2BNeurosci.%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffncel.2018.00100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fncel.2018.00100&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.62" data-doi="10.1016/j.neuron.2015.08.032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ranade</span>,  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Syeda</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Patapoutian</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Mechanically activated ion channels.</span> <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">1162</span>–<span class="cit-lpage">1179</span>. Erratum: 433.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.neuron.2015.08.032</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D87%26rft.spage%253D1162%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2015.08.032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2015.08.032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.63" data-doi="10.7554/eLife.34226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Richter</span>,  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Fozouni</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. B.</span> <span class="cit-name-surname">Eisen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">King</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Gene family innovation, conservation and loss on the animal stem lineage.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">43</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7554/eLife.34226</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.34226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.34226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.64" data-doi="10.1093/bioinformatics/btg180"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ronquist</span>,  <span class="cit-name-given-names">F.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Huelsenbeck</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">MrBayes 3: Bayesian phylogenetic inference under mixed models.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1572</span>–<span class="cit-lpage">1574</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btg180</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DRonquist%26rft.auinit1%253DF.%26rft.volume%253D19%26rft.issue%253D12%26rft.spage%253D1572%26rft.epage%253D1574%26rft.atitle%253DMrBayes%2B3%253A%2BBayesian%2Bphylogenetic%2Binference%2Bunder%2Bmixed%2Bmodels%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg180%26rft_id%253Dinfo%253Apmid%252F12912839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg180&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12912839&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184878700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.65" data-doi="10.1093/sysbio/sys029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ronquist</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Teslenko</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">van der Mark</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. L.</span> <span class="cit-name-surname">Ayres</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Darling</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">MrBayes 3.2: efficient Bayesian phylogenetic inference and model choice across a large model space.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">539</span>–<span class="cit-lpage">542</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/sysbio/sys029</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsys029%26rft_id%253Dinfo%253Apmid%252F22357727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/sys029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22357727&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.66" data-doi="10.1126/science.1242592"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ryan</span>,  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Pang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. E.</span> <span class="cit-name-surname">Schnitzler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Nguyen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. T.</span> <span class="cit-name-surname">Moreland</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The genome of the ctenophore <em>Mnemiopsis leidyi</em> and its implications for cell type evolution.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">342</span>: <span class="cit-elocation-id">1242592</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1242592</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRyan%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D342%26rft.issue%253D6164%26rft.spage%253D1242592%26rft.epage%253D1242592%26rft.atitle%253DThe%2BGenome%2Bof%2Bthe%2BCtenophore%2BMnemiopsis%2Bleidyi%2Band%2BIts%2BImplications%2Bfor%2BCell%2BType%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1242592%26rft_id%253Dinfo%253Apmid%252F24337300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNDIvNjE2NC8xMjQyNTkyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi84MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.67" data-doi="10.1128/MCB.00657-05"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sans</span>,  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Satterwhite</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Stoltzman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. T.</span> <span class="cit-name-surname">Breen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">Ayer</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">MondoA-Mlx heterodimers are candidate sensors of cellular energy status: Mitochondrial localization and direct regulation of glycolysis.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">4863</span>–<span class="cit-lpage">4871</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MCB.00657-05</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DSans%26rft.auinit1%253DC.%2BL.%26rft.volume%253D26%26rft.issue%253D13%26rft.spage%253D4863%26rft.epage%253D4871%26rft.atitle%253DMondoA-Mlx%2BHeterodimers%2BAre%2BCandidate%2BSensors%2Bof%2BCellular%2BEnergy%2BStatus%253A%2BMitochondrial%2BLocalization%2Band%2BDirect%2BRegulation%2Bof%2BGlycolysis%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00657-05%26rft_id%253Dinfo%253Apmid%252F16782875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xMy80ODYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi84MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.68" data-doi="10.1093/jxb/erp245"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sarkar</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Bosneaga</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Auer</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Plant cell walls throughout evolution: Towards a molecular understanding of their design principles.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">3615</span>–<span class="cit-lpage">3635</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jxb/erp245</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferp245%26rft_id%253Dinfo%253Apmid%252F19687127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/erp245&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19687127&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269609800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.69" data-doi="10.1111/tra.12148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sauvageau</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Rochdi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Oueslati</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. F.</span> <span class="cit-name-surname">Hamdan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Percherancier</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">CNIH4 interacts with newly synthesized GPCR and controls their export from the endoplasmic reticulum.</span> <abbr class="cit-jnl-abbrev">Traffic</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">383</span>–<span class="cit-lpage">400</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tra.12148</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTraffic%26rft.volume%253D15%26rft.spage%253D383%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftra.12148%26rft_id%253Dinfo%253Apmid%252F24405750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tra.12148&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24405750&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.70" data-doi="10.1126/science.1167852"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwenk</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Harmel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Zolles</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Bildl</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kulik</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Functional proteomics identify cornichon proteins as auxiliary subunits of AMPA receptors.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">323</span>: <span class="cit-fpage">1313</span>–<span class="cit-lpage">1319</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1167852</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSchwenk%26rft.auinit1%253DJ.%26rft.volume%253D323%26rft.issue%253D5919%26rft.spage%253D1313%26rft.epage%253D1319%26rft.atitle%253DFunctional%2BProteomics%2BIdentify%2BCornichon%2BProteins%2Bas%2BAuxiliary%2BSubunits%2Bof%2BAMPA%2BReceptors%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1167852%26rft_id%253Dinfo%253Apmid%252F19265014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjMvNTkxOS8xMzEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi84MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.71" data-doi="10.1038/s41559-018-0575-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sebé-Pedrós</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Chomsky</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Pang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Lara-Astiaso</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Gaiti</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Early metazoan cell type diversity and the evolution of multicellular gene regulation.</span> <abbr class="cit-jnl-abbrev">Nat. Ecol. Evol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1176</span>–<span class="cit-lpage">1188</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41559-018-0575-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BEcol.%2BEvol.%26rft.volume%253D2%26rft.spage%253D1176%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41559-018-0575-6%26rft_id%253Dinfo%253Apmid%252F29942020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41559-018-0575-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29942020&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.72" data-doi="10.1007/BF00123632"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shannon</span>,  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Characterization of the female-sterile mutant <em>almondex</em> of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">244</span>–<span class="cit-lpage">256</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00123632</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.stitle%253DGenetica%26rft.aulast%253DShannon%26rft.auinit1%253DM.%2BP.%26rft.volume%253D43%26rft.issue%253D2%26rft.spage%253D244%26rft.epage%253D256%26rft.atitle%253DCharacterization%2Bof%2Bthe%2Bfemale-sterile%2Bmutant%2Balmondex%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00123632%26rft_id%253Dinfo%253Apmid%252F4625164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00123632&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4625164&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.73" data-doi="10.1002/jez.1401830312"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shannon</span>,  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">The development of eggs produced by the female-sterile mutant <em>almondex</em> of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">J. Exp. Zool.</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">383</span>–<span class="cit-lpage">400</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/jez.1401830312</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bexperimental%2Bzoology%26rft.stitle%253DJ%2BExp%2BZool%26rft.aulast%253DShannon%26rft.auinit1%253DM.%2BP.%26rft.volume%253D183%26rft.issue%253D3%26rft.spage%253D383%26rft.epage%253D400%26rft.atitle%253DThe%2Bdevelopment%2Bof%2Beggs%2Bproduced%2Bby%2Bthe%2Bfemale-sterile%2Bmutant%2Balmondex%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjez.1401830312%26rft_id%253Dinfo%253Apmid%252F4633255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jez.1401830312&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4633255&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.74" data-doi="10.1038/s41559-017-0126"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">X. X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. T.</span> <span class="cit-name-surname">Hittinger</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Rokas</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Contentious relationships in phylogenomic studies can be driven by a handful of genes.</span> <abbr class="cit-jnl-abbrev">Nat. Ecol. Evol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">126</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41559-017-0126</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BEcol.%2BEvol.%26rft.volume%253D1%26rft.spage%253D126%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41559-017-0126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41559-017-0126&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.75" data-doi="10.1016/j.ajhg.2016.03.028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shibata</span>,  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. T.</span> <span class="cit-name-surname">Ranum</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Moteki</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Pan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. T.</span> <span class="cit-name-surname">Goodwin</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">RNA interference prevents autosomal-dominant hearing loss.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">1101</span>–<span class="cit-lpage">1113</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2016.03.028</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D98%26rft.spage%253D1101%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2016.03.028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2016.03.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.76" data-doi="10.1016/j.cub.2017.02.031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simion</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Philippe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Baurain</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Jager</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Richter</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A large and consistent phylogenomic dataset supports sponges as the sister group to all other animals.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">958</span>–<span class="cit-lpage">967</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cub.2017.02.031</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D27%26rft.spage%253D958%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2017.02.031%26rft_id%253Dinfo%253Apmid%252F28318975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2017.02.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28318975&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.77" data-doi="10.1186/1471-2164-10-22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smedley</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Haider</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Ballester</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Holland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">London</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">BioMart–biological queries made easy.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-10-22</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DSmedley%26rft.auinit1%253DD.%26rft.volume%253D10%26rft.spage%253D22%26rft.epage%253D22%26rft.atitle%253DBioMart--biological%2Bqueries%2Bmade%2Beasy.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-22%26rft_id%253Dinfo%253Apmid%252F19144180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-22&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19144180&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.78" data-doi="10.1038/s41586-019-1290-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sogabe</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. L.</span> <span class="cit-name-surname">Hatleberg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Kocot</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. E.</span> <span class="cit-name-surname">Say</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Stoupin</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Pluripotency and the origin of animal multicellularity.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">570</span>: <span class="cit-fpage">519</span>–<span class="cit-lpage">522</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41586-019-1290-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D570%26rft.spage%253D519%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-019-1290-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-019-1290-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.79" data-doi="10.1104/pp.110.154427"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sørensen</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Domozych</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. G. T.</span> <span class="cit-name-surname">Willats</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">How have plant cell walls evolved?</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">366</span>–<span class="cit-lpage">372</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.110.154427</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.110.154427%26rft_id%253Dinfo%253Apmid%252F20431088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNTMvMi8zNjYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzgxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.80" data-doi="10.1038/nature07191"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Begovic</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Chapman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. H.</span> <span class="cit-name-surname">Putnam</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Hellsten</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The <em>Trichoplax</em> genome and the nature of placozoans.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">454</span>: <span class="cit-fpage">955</span>–<span class="cit-lpage">960</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature07191</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSrivastava%26rft.auinit1%253DM.%26rft.volume%253D454%26rft.issue%253D7207%26rft.spage%253D955%26rft.epage%253D960%26rft.atitle%253DThe%2BTrichoplax%2Bgenome%2Band%2Bthe%2Bnature%2Bof%2Bplacozoans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07191%26rft_id%253Dinfo%253Apmid%252F18719581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18719581&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258591000030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.81" data-doi="10.1038/nature09201"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Simakov</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Chapman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Fahey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Gauthier</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The <em>Amphimedon queenslandica</em> genome and the evolution of animal complexity.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">466</span>: <span class="cit-fpage">720</span>–<span class="cit-lpage">726</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature09201</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSrivastava%26rft.auinit1%253DM.%26rft.volume%253D466%26rft.issue%253D7307%26rft.spage%253D720%26rft.epage%253D726%26rft.atitle%253DThe%2BAmphimedon%2Bqueenslandica%2Bgenome%2Band%2Bthe%2Bevolution%2Bof%2Banimal%2Bcomplexity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09201%26rft_id%253Dinfo%253Apmid%252F20686567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09201&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20686567&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280562500031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.82" data-doi="10.1242/jeb.163196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swafford</span>,  <span class="cit-name-given-names">A. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. H.</span> <span class="cit-name-surname">Oakley</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Multimodal sensorimotor system in unicellular zoospores of a fungus.</span> <abbr class="cit-jnl-abbrev">J. Exp. Biol.</abbr> <span class="cit-vol">221</span>: <span class="cit-elocation-id">jeb163196</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1242/jeb.163196</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjeb.163196%26rft_id%253Dinfo%253Apmid%252F29170260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4YmlvIjtzOjU6InJlc2lkIjtzOjE1OiIyMjEvMi9qZWIxNjMxOTYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzgxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.83" data-doi="10.1146/annurev.genet.38.072902.092831"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>,  <span class="cit-name-given-names">J. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Raes</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Duplication and divergence: the evolution of new genes and old ideas.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">615</span>–<span class="cit-lpage">643</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.genet.38.072902.092831</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.volume%253D38%26rft.spage%253D615%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.38.072902.092831%26rft_id%253Dinfo%253Apmid%252F15568988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.38.072902.092831&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15568988&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226244600019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.84" data-doi="10.1101/gr.073585.107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vilella</span>,  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Severin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ureta-Vidal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Heng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Durbin</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">EnsemblCompara GeneTrees: Complete, duplication-aware phylogenetic trees in vertebrates.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">335</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.073585.107</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.073585.107%26rft_id%253Dinfo%253Apmid%252F19029536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzIvMzI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi84MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.85"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>,  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">How often do duplicated genes evolve new functions?</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">139</span>: <span class="cit-fpage">421</span>–<span class="cit-lpage">428</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWalsh%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D139%26rft.issue%253D1%26rft.spage%253D421%26rft.epage%253D428%26rft.atitle%253DHow%2Boften%2Bdo%2Bduplicated%2Bgenes%2Bevolve%2Bnew%2Bfunctions%253F%26rft_id%253Dinfo%253Apmid%252F7705642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM5LzEvNDIxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi84MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.86" data-doi="10.1016/j.neuron.2016.05.023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Liu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">X. Z.</span> <span class="cit-name-surname">Xu</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">TMC-1 mediates alkaline sensation in <em>C. elegans</em> through nociceptive neurons.</span> <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">146</span>–<span class="cit-lpage">154</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.neuron.2016.05.023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D91%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2016.05.023%26rft_id%253Dinfo%253Apmid%252F27321925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2016.05.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27321925&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.87" data-doi="10.1073/pnas.1503453112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whelan</span>,  <span class="cit-name-given-names">N. V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Kocot</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. L.</span> <span class="cit-name-surname">Moroz</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Halanych</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Error, signal, and the placement of Ctenophora sister to all other animals.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">5773</span>–<span class="cit-lpage">5778</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1503453112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1503453112%26rft_id%253Dinfo%253Apmid%252F25902535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzE4LzU3NzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzgxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.88" data-doi="10.1093/oxfordjournals.molbev.a003851"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whelan</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Goldman</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A general empirical model of protein evolution derived from multiple protein families using a maximum-likelihood approach.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">691</span>–<span class="cit-lpage">699</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/oxfordjournals.molbev.a003851</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DWhelan%26rft.auinit1%253DS.%26rft.volume%253D18%26rft.issue%253D5%26rft.spage%253D691%26rft.epage%253D699%26rft.atitle%253DA%2BGeneral%2BEmpirical%2BModel%2Bof%2BProtein%2BEvolution%2BDerived%2Bfrom%2BMultiple%2BProtein%2BFamilies%2BUsing%2Ba%2BMaximum-Likelihood%2BApproach%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a003851%26rft_id%253Dinfo%253Apmid%252F11319253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a003851&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11319253&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168427400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.89" data-doi="10.1073/pnas.74.11.5088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woese</span>,  <span class="cit-name-given-names">C. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. E.</span> <span class="cit-name-surname">Fox</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Phylogenetic structure of the prokaryotic domain: the primary kingdoms.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">5088</span>–<span class="cit-lpage">5090</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.74.11.5088</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DWoese%26rft.auinit1%253DC.%2BR.%26rft.volume%253D74%26rft.issue%253D11%26rft.spage%253D5088%26rft.epage%253D5090%26rft.atitle%253DPhylogenetic%2Bstructure%2Bof%2Bthe%2Bprokaryotic%2Bdomain%253A%2BThe%2Bprimary%2Bkingdoms%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.74.11.5088%26rft_id%253Dinfo%253Apmid%252F270744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiNzQvMTEvNTA4OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvODExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.90" data-doi="10.1038/nn.4449"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Grillet</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Zhao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Cunningham</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Harkins-Perry</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Mechanosensory hair cells express two molecularly distinct mechanotransduction channels.</span> <abbr class="cit-jnl-abbrev">Nat. Neurosci.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">33</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nn.4449</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BNeurosci.%26rft.volume%253D20%26rft.spage%253D24%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnn.4449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nn.4449&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.91" data-doi="10.1073/pnas.161284298"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamashita</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Takenoshita</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Sakurai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. K.</span> <span class="cit-name-surname">Bruick</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. J.</span> <span class="cit-name-surname">Henzel</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A glucose-responsive transcription factor that regulates carbohydrate metabolism in the liver.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">9116</span>–<span class="cit-lpage">9121</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.161284298</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.161284298%26rft_id%253Dinfo%253Apmid%252F11470916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTgvMTYvOTExNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvODExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.92" data-doi="10.1038/s41598-018-21233-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yoshimura</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. B.</span> <span class="cit-name-surname">Shibata</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. T.</span> <span class="cit-name-surname">Ranum</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Smith</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Enhanced viral-mediated cochlear gene delivery in adult mice by combining canal fenestration with round window membrane inoculation.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2980</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41598-018-21233-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D8%26rft.spage%253D2980%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-018-21233-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-018-21233-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.93" data-doi="10.1093/molbev/msr132"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Young</span>,  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Diolaiti</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Conacci-Sorrell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Ruiz-Trillo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. N.</span> <span class="cit-name-surname">Eisenman</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Premetazoan ancestry of the Myc-Max network.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2961</span>–<span class="cit-lpage">2971</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msr132</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr132%26rft_id%253Dinfo%253Apmid%252F21571926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msr132&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21571926&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295184200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.94"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yue</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Fan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Duan</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> TMC proteins modulate egg laying and membrane excitability through a background leak conductance in <em>C. elegans</em>. Neuron <strong>97</strong>: 571–585 e5.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.95" data-doi="10.1016/j.cub.2019.02.036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Vinther</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. A.</span> <span class="cit-name-surname">Parry</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Wei</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Green</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Cambrian sessile, suspension feeding stem-group ctenophores and evolution of the comb jelly body plan.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1112</span>–<span class="cit-lpage">1125.e2</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cub.2019.02.036</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D29%26rft.spage%253D1112%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2019.02.036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2019.02.036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.811.96" data-doi="10.1016/0022-5193(65)90083-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zuckerkandl</span>,  <span class="cit-name-given-names">E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Pauling</span></span></li></ol><cite>, <span class="cit-pub-date">1965</span> <span class="cit-article-title">Molecules as documents of evolutionary history.</span> <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">366</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0022-5193(65)90083-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DZuckerkandl%26rft.auinit1%253DE.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D357%26rft.epage%253D366%26rft.atitle%253DMolecules%2Bas%2Bdocuments%2Bof%2Bevolutionary%2Bhistory.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-5193%252865%252990083-4%26rft_id%253Dinfo%253Apmid%252F5876245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-5193(65)90083-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5876245&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A19656369100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/2/811.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/2/797" title="Interaction Between Induced and Natural Variation at oil yellow1 Delays Reproductive Maturity in Maize" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/2/827" title="Assessment of a Split Homing Based Gene Drive for Efficient Knockout of Multiple Genes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/2"><span>Volume 10 Issue 2, February 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/2.cover-source.jpg?itok=LSawHOSr" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/811" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ojX0Q18jOcJSpwTrxW5u70xi-LMUgKGmV2m4zOENZDI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/2/811" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/2/811" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/2/811" class="active" data-icon-position="" data-hide-link-title="0">A Screen for Gene Paralogies Delineating Evolutionary Branching Order of Early Metazoa</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495499" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-SvVcO_84a0q0EroLcJUAPyO5RO2vRZTJFgKJ6Wv9kQE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/2/811.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/2/811" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form--3arLjRsfq3y3oHbN0WLXMlBgnfcDG6fKH0aDlxvP6U" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/2/811" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/2/811.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495499" id="node495499--2" data-pisa="ggg;10/2/811" data-pisa-master="ggg;g3.119.400951" data-apath="/ggg/10/2/811.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Screen for Gene Paralogies Delineating Evolutionary Branching Order of Early Metazoa</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7107-5518" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Albert Erives</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-4882-8398" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bernd Fritzsch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">811-826;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400951 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495499--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Albert Erives </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Iowa, Iowa City, IA, 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Albert%2BErives%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Erives%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlbert%2BErives%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7107-5518" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Albert Erives</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:albert-erives@uiowa.edu" class="" data-icon-position="" data-hide-link-title="0">albert-erives@uiowa.edu</a>
<a href="mailto:bernd-fritzsch@uiowa.edu" class="" data-icon-position="" data-hide-link-title="0">bernd-fritzsch@uiowa.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bernd Fritzsch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Iowa, Iowa City, IA, 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bernd%2BFritzsch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fritzsch%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABernd%2BFritzsch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4882-8398" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bernd Fritzsch</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:albert-erives@uiowa.edu" class="" data-icon-position="" data-hide-link-title="0">albert-erives@uiowa.edu</a>
<a href="mailto:bernd-fritzsch@uiowa.edu" class="" data-icon-position="" data-hide-link-title="0">bernd-fritzsch@uiowa.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/2/811" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/2/811&amp;title=A%20Screen%20for%20Gene%20Paralogies%20Delineating%20Evolutionary%20Branching%20Order%20of%20Early%20Metazoa" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/2/811&amp;title=A%20Screen%20for%20Gene%20Paralogies%20Delineating%20Evolutionary%20Branching%20Order%20of%20Early%20Metazoa" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/2/811&amp;title=A%20Screen%20for%20Gene%20Paralogies%20Delineating%20Evolutionary%20Branching%20Order%20of%20Early%20Metazoa" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/2/811&amp;text=A%20Screen%20for%20Gene%20Paralogies%20Delineating%20Evolutionary%20Branching%20Order%20of%20Early%20Metazoa" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/2/811&amp;title=A%20Screen%20for%20Gene%20Paralogies%20Delineating%20Evolutionary%20Branching%20Order%20of%20Early%20Metazoa" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/2/811&amp;t=A%20Screen%20for%20Gene%20Paralogies%20Delineating%20Evolutionary%20Branching%20Order%20of%20Early%20Metazoa" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/2/811&amp;title=A%20Screen%20for%20Gene%20Paralogies%20Delineating%20Evolutionary%20Branching%20Order%20of%20Early%20Metazoa" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/2/811&amp;title=A%20Screen%20for%20Gene%20Paralogies%20Delineating%20Evolutionary%20Branching%20Order%20of%20Early%20Metazoa" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495499/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495499" id="node495499--3" data-pisa="ggg;10/2/811" data-pisa-master="ggg;g3.119.400951" data-apath="/ggg/10/2/811.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Screen for Gene Paralogies Delineating Evolutionary Branching Order of Early Metazoa</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7107-5518" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Albert Erives</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-4882-8398" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bernd Fritzsch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">811-826;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400951 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495499--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Albert Erives </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Iowa, Iowa City, IA, 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Albert%2BErives%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Erives%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlbert%2BErives%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7107-5518" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Albert Erives</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:albert-erives@uiowa.edu" class="" data-icon-position="" data-hide-link-title="0">albert-erives@uiowa.edu</a>
<a href="mailto:bernd-fritzsch@uiowa.edu" class="" data-icon-position="" data-hide-link-title="0">bernd-fritzsch@uiowa.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bernd Fritzsch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Iowa, Iowa City, IA, 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bernd%2BFritzsch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fritzsch%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABernd%2BFritzsch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4882-8398" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bernd Fritzsch</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:albert-erives@uiowa.edu" class="" data-icon-position="" data-hide-link-title="0">albert-erives@uiowa.edu</a>
<a href="mailto:bernd-fritzsch@uiowa.edu" class="" data-icon-position="" data-hide-link-title="0">bernd-fritzsch@uiowa.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=811&amp;rft.epage=826&amp;rft.atitle=A%20Screen%20for%20Gene%20Paralogies%20Delineating%20Evolutionary%20Branching%20Order%20of%20Early%20Metazoa&amp;rft.volume=10&amp;rft.issue=2&amp;rft.date=2020-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Erives%2C+Albert&amp;rft.au=Fritzsch%2C+Bernd"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495499/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495499/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495499/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495499/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495499/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495499/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495499/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495499/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495499/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495499/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495499/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495499/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/2/811" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/2/811.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/2/811.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/2/811.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
