<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/2/695" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/2/695.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/2/695.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/2/695.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/10/2/695/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/2/695.atom" />
<meta name="HW.pisa" content="ggg;10/2/695" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Predictive Ability for Foliar Nutritive Traits in Perennial Ryegrass" />
<meta name="DC.Identifier" content="10.1534/g3.119.400880" />
<meta name="DC.Date" content="2020-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Arojju et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Forage nutritive value impacts animal nutrition, which underpins livestock productivity, reproduction and health. Genetic improvement for nutritive traits in perennial ryegrass has been limited, as they are typically expensive and time-consuming to measure through conventional methods. Genomic selection is appropriate for such complex and expensive traits, enabling cost-effective prediction of breeding values using genome-wide markers. The aims of the present study were to assess the potential of genomic selection for a range of nutritive traits in a multi-population training set, and to quantify contributions of family, location and family-by-location variance components to trait variation and heritability for nutritive traits. The training set consisted of a total of 517 half-sibling (half-sib) families, from five advanced breeding populations, evaluated in two distinct New Zealand grazing environments. Autumn-harvested samples were analyzed for 18 nutritive traits and maternal parents of the half-sib families were genotyped using genotyping-by-sequencing. Significant ( P &lt; 0.05) family variance was detected for all nutritive traits and genomic heritability ( h2g ) was moderate to high (0.20 to 0.74). Family-by-location interactions were significant and particularly large for water soluble carbohydrate (WSC), crude fat, phosphorus (P) and crude protein. GBLUP, KGD-GBLUP and BayesCπ genomic prediction models displayed similar predictive ability, estimated by 10-fold cross validation, for all nutritive traits with values ranging from r = 0.16 to 0.45 using phenotypes from across two locations. High predictive ability was observed for the mineral traits sulfur (0.44), sodium (0.45) and magnesium (0.45) and the lowest values were observed for P (0.16), digestibility (0.22) and high molecular weight WSC (0.23). Predictive ability estimates for most nutritive traits were retained when marker number was reduced from one million to as few as 50,000. The moderate to high predictive abilities observed suggests implementation of genomic selection is feasible for most of the nutritive traits examined." />
<meta name="DC.Contributor" content="Sai Krishna Arojju" />
<meta name="DC.Contributor" content="Mingshu Cao" />
<meta name="DC.Contributor" content="M. Z. Zulfi Jahufer" />
<meta name="DC.Contributor" content="Brent A. Barrett" />
<meta name="DC.Contributor" content="Marty J. Faville" />
<meta name="article:published_time" content="2020-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genomic Predictive Ability for Foliar Nutritive Traits in Perennial Ryegrass" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Forage nutritive value impacts animal nutrition, which underpins livestock productivity, reproduction and health. Genetic improvement for nutritive traits in perennial ryegrass has been limited, as they are typically expensive and time-consuming to measure through conventional methods. Genomic selection is appropriate for such complex and expensive traits, enabling cost-effective prediction of breeding values using genome-wide markers. The aims of the present study were to assess the potential of genomic selection for a range of nutritive traits in a multi-population training set, and to quantify contributions of family, location and family-by-location variance components to trait variation and heritability for nutritive traits. The training set consisted of a total of 517 half-sibling (half-sib) families, from five advanced breeding populations, evaluated in two distinct New Zealand grazing environments. Autumn-harvested samples were analyzed for 18 nutritive traits and maternal parents of the half-sib families were genotyped using genotyping-by-sequencing. Significant (&lt;i&gt;P&lt;/i&gt; &amp;lt; 0.05) family variance was detected for all nutritive traits and genomic heritability (&lt;i&gt;h&lt;sup&gt;2&lt;/sup&gt;&lt;sub&gt;g&lt;/sub&gt;&lt;/i&gt;) was moderate to high (0.20 to 0.74). Family-by-location interactions were significant and particularly large for water soluble carbohydrate (WSC), crude fat, phosphorus (P) and crude protein. GBLUP, KGD-GBLUP and BayesCπ genomic prediction models displayed similar predictive ability, estimated by 10-fold cross validation, for all nutritive traits with values ranging from &lt;i&gt;r&lt;/i&gt; = 0.16 to 0.45 using phenotypes from across two locations. High predictive ability was observed for the mineral traits sulfur (0.44), sodium (0.45) and magnesium (0.45) and the lowest values were observed for P (0.16), digestibility (0.22) and high molecular weight WSC (0.23). Predictive ability estimates for most nutritive traits were retained when marker number was reduced from one million to as few as 50,000. The moderate to high predictive abilities observed suggests implementation of genomic selection is feasible for most of the nutritive traits examined.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/02/01" />
<meta name="citation_mjid" content="ggg;10/2/695" />
<meta name="citation_id" content="10/2/695" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/2/695" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/2/695.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/2/695.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/2/695.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400880" />
<meta name="citation_pmid" content="31792009" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="695" />
<meta name="citation_lastpage" content="708" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Sai Krishna Arojju" />
<meta name="citation_author_institution" content="AgResearch Ltd, Grasslands Research Centre, PB 11008, Palmerston North, New Zealand" />
<meta name="citation_author_email" content="sai.arojju@agresearch.co.nz" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4769-9782" />
<meta name="citation_author" content="Mingshu Cao" />
<meta name="citation_author_institution" content="AgResearch Ltd, Grasslands Research Centre, PB 11008, Palmerston North, New Zealand" />
<meta name="citation_author" content="M. Z. Zulfi Jahufer" />
<meta name="citation_author_institution" content="AgResearch Ltd, Grasslands Research Centre, PB 11008, Palmerston North, New Zealand" />
<meta name="citation_author" content="Brent A. Barrett" />
<meta name="citation_author_institution" content="AgResearch Ltd, Grasslands Research Centre, PB 11008, Palmerston North, New Zealand" />
<meta name="citation_author" content="Marty J. Faville" />
<meta name="citation_author_institution" content="AgResearch Ltd, Grasslands Research Centre, PB 11008, Palmerston North, New Zealand" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3129-6540" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=P. Annicchiarico;citation_author=N. Nazzicari;citation_author=X. Li;citation_author=Y. Wei;citation_author=L. Pecetti;citation_title=Accuracy of genomic selection for alfalfa biomass yield in different reference populations.;citation_pages=1020;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12864-015-2212-y" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=S. K. Arojju;citation_author=P. Conaghan;citation_author=S. Barth;citation_author=D. Milbourne;citation_author=M. D. Casler;citation_title=Genomic prediction of crown rust resistance in Lolium perenne.;citation_pages=35;citation_volume=19;citation_year=2018;citation_doi=10.1186/s12863-018-0613-z" />
<meta name="citation_reference" content="citation_title=Feeding value evaluation in grass and legume breeding and variety testing: Report of a debate;citation_pages=307-311;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Bellot;citation_author=G. de los Campos;citation_author=M. Pérez-Enciso;citation_title=Can deep learning improve genomic prediction of complex human traits?;citation_volume=210;citation_year=2018;citation_pmid=30171033;citation_doi=10.1534/genetics.118.301298" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=E. Biazzi;citation_author=N. Nazzicari;citation_author=L. Pecetti;citation_author=E. C. Brummer;citation_author=A. Palmonari;citation_title=Genome-wide association mapping and genomic selection for alfalfa (Medicago sativa) forage quality traits.;citation_pages=e0169234;citation_volume=12;citation_year=2017;citation_doi=10.1371/journal.pone.0169234" />
<meta name="citation_reference" content="citation_title=ASReml-R reference manual. The State of Queensland;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=S. L. Byrne;citation_author=P. Conaghan;citation_author=S. Barth;citation_author=S. K. Arojju;citation_author=M. Casler;citation_title=Using variable importance measures to identify a small set of SNPs to predict heading date in perennial ryegrass.;citation_pages=3566;citation_volume=7;citation_year=2017;citation_doi=10.1038/s41598-017-03232-8" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=S. L. Byrne;citation_author=I. Nagy;citation_author=M. Pfeifer;citation_author=I. Armstead;citation_author=S. Swain;citation_title=A synteny‐based draft genome sequence of the forage grass Lolium perenne.;citation_pages=816-826;citation_volume=84;citation_year=2015;citation_pmid=26408275;citation_doi=10.1111/tpj.13037" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. Casler;citation_author=K. P. Vogel;citation_title=Accomplishments and Impact from Breeding for Increased Forage Nutritional Value;citation_pages=12-20;citation_volume=39;citation_year=1999;citation_issue=1;citation_doi=10.2135/cropsci1999.0011183X003900010003x" />
<meta name="citation_reference" content="citation_journal_title=Grass Forage Sci.;citation_author=D. Chapman;citation_author=J. Bryant;citation_author=M. Olayemi;citation_author=G. Edwards;citation_author=B. Thorrold;citation_title=An economically based evaluation index for perennial and short‐term ryegrasses in New Zealand dairy farm systems.;citation_pages=1-21;citation_volume=72;citation_year=2017;citation_doi=10.1111/gfs.12213" />
<meta name="citation_reference" content="citation_journal_title=Anim. Prod. Sci.;citation_author=D. Chapman;citation_author=G. Edwards;citation_author=A. Stewart;citation_author=M. McEvoy;citation_author=M. O’Donovan;citation_title=Valuing forages for genetic selection: what traits should we focus on?;citation_pages=869-882;citation_volume=55;citation_year=2015;citation_doi=10.1071/AN14838" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=N. Cogan;citation_author=K. Smith;citation_author=T. Yamada;citation_author=M. Francki;citation_author=A. Vecchies;citation_title=QTL analysis and comparative genomics of herbage quality traits in perennial ryegrass (Lolium perenne L.).;citation_pages=364-380;citation_volume=110;citation_year=2005;citation_issue=2;citation_pmid=15558228;citation_doi=10.1007/s00122-004-1848-9" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Crossa;citation_author=G. de los Campos;citation_author=M. Maccaferri;citation_author=R. Tuberosa;citation_author=J. Burgueño;citation_title=Extending the marker× environment interaction model for genomic-enabled prediction and genome-wide association analysis in durum wheat.;citation_volume=56;citation_year=2016;citation_doi=10.2135/cropsci2015.04.0260" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=J. Crossa;citation_author=P. Pérez-Rodríguez;citation_author=J. Cuevas;citation_author=O. Montesinos-López;citation_author=D. Jarquín;citation_title=Genomic selection in plant breeding: methods, models, and perspectives.;citation_pages=961-975;citation_volume=22;citation_year=2017;citation_doi=10.1016/j.tplants.2017.08.011" />
<meta name="citation_reference" content="citation_journal_title=N. Z. J. Agric. Res.;citation_author=J. Crush;citation_author=S. Woodward;citation_author=J. Eerens;citation_author=K. Macdonald;citation_title=Growth and milksolids production in pastures of older and more recent ryegrass and white clover cultivars under dairy grazing.;citation_pages=119-135;citation_volume=49;citation_year=2006;citation_doi=10.1080/00288233.2006.9513702" />
<meta name="citation_reference" content="citation_journal_title=N. Z. J. Agric. Res.;citation_author=J. R. Crush;citation_author=J. M. Lee;citation_author=G. P. Cosgrove;citation_author=L. Rossi;citation_author=D. F. Chapman;citation_title=Foliar macronutrient concentrations of eight perennial ryegrass (Lolium perenne L.) cultivars grown in four regions in New Zealand.;citation_pages=285-300;citation_volume=61;citation_year=2018;citation_doi=10.1080/00288233.2017.1418397" />
<meta name="citation_reference" content="citation_journal_title=N. Z. J. Agric. Res.;citation_author=J. R. Crush;citation_author=J. M. Lee;citation_author=G. P. Cosgrove;citation_author=L. Rossi;citation_author=D. F. Chapman;citation_title=Foliar micronutrient concentrations of eight perennial ryegrass (Lolium perenne L.) cultivars grown in four regions in New Zealand.;citation_pages=301-311;citation_volume=61;citation_year=2018;citation_doi=10.1080/00288233.2018.1426020" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. Daetwyler;citation_author=M. P. Calus;citation_author=R. Pong-Wong;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_title=Genomic prediction in animals and plants: simulation of data, validation, reporting, and benchmarking.;citation_volume=193;citation_year=2013;citation_pmid=23222650;citation_doi=10.1534/genetics.112.147983" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. Daetwyler;citation_author=R. Pong-Wong;citation_author=B. Villanueva;citation_author=J. A. Woolliams;citation_title=The impact of genetic architecture on genome-wide evaluation methods.;citation_volume=185;citation_year=2010;citation_pmid=20407128;citation_doi=10.1534/genetics.110.116855" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=G. de los Campos;citation_author=D. Sorensen;citation_author=D. Gianola;citation_title=Genomic heritability: what is it?;citation_pages=e1005048;citation_volume=11;citation_year=2015;citation_pmid=25942577;citation_doi=10.1371/journal.pgen.1005048" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=K. G. Dodds;citation_author=J. C. McEwan;citation_author=R. Brauning;citation_author=R. M. Anderson;citation_author=T. C. Stijn;citation_title=Construction of relatedness matrices using genotyping-by-sequencing data.;citation_pages=1047;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12864-015-2252-3" />
<meta name="citation_reference" content="citation_journal_title=Aust. J. Agric. Res.;citation_author=H. Easton;citation_author=A. Mackay;citation_author=J. Lee;citation_title=Genetic variation for macro-and micro-nutrient concentration in perennial ryegrass (Lolium perenne L.).;citation_pages=657-666;citation_volume=48;citation_year=1997;citation_doi=10.1071/A96086" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the New Zealand Grassland Association;citation_author=H. Easton;citation_author=A. Stewart;citation_author=T. Lyons;citation_author=M. Parris;citation_author=S. Charrier;citation_title=Soluble carbohydrate content of ryegrass cultivars.;citation_pages=161-166;citation_volume=71;citation_year=2009" />
<meta name="citation_reference" content="Easton, S., J. Amyes, N. Cameron, R. Green, G. Kerr et al., 2002 Pasture plant breeding in New Zealand: where to from here? Proceedings of the conference-New Zealand grassland association: 173–180." />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the New Zealand Grassland Association;citation_author=G. R. Edwards;citation_author=A. Parsons;citation_author=S. Rasmussen;citation_author=R. H. Bryant;citation_title=High sugar ryegrasses for livestock systems in New Zealand.;citation_pages=161-171;citation_volume=69;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with R package rrBLUP.;citation_pages=250-255;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. B. Endelman;citation_author=J.-L. Jannink;citation_title=Shrinkage estimation of the realized relationship matrix. G3: Genes, Genomes;citation_pages=1405-1413;citation_volume=2;citation_year=2012;citation_doi=10.1534/g3.112.004259" />
<meta name="citation_reference" content="citation_title=Introduction to quantitative genetics;citation_year=1960" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the New Zealand Grassland Association;citation_author=M. Faville;citation_author=K. Richardson;citation_author=M. Gagic;citation_author=W. Mace;citation_author=X. Sun;citation_title=Genetic improvement of fibre traits in perennial ryegrass.;citation_pages=71-78;citation_volume=72;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. J. Faville;citation_author=S. Ganesh;citation_author=M. Cao;citation_author=M. Z. Jahufer;citation_author=T. P. Bilton;citation_title=Predictive ability of genomic selection models in a multi-population perennial ryegrass training set using genotyping-by-sequencing.;citation_pages=703-720;citation_volume=131;citation_year=2018;citation_doi=10.1007/s00122-017-3030-1" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=D. Fè;citation_author=B. H. Ashraf;citation_author=M. G. Pedersen;citation_author=L. Janss;citation_author=S. Byrne;citation_title=Accuracy of genomic prediction in a commercial perennial ryegrass breeding program.;citation_pages=1-12;citation_volume=9;citation_year=2016;citation_doi=10.3835/plantgenome2016.05.0047" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. B. Fernandes;citation_author=K. O. Dias;citation_author=D. F. Ferreira;citation_author=P. J. Brown;citation_title=Efficiency of multi-trait, indirect, and trait-assisted genomic selection for improvement of biomass sorghum.;citation_pages=747-755;citation_volume=131;citation_year=2018;citation_doi=10.1007/s00122-017-3033-y" />
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol.;citation_author=J. D. Fiedler;citation_author=C. Lanzatella;citation_author=S. J. Edmé;citation_author=N. A. Palmer;citation_author=G. Sarath;citation_title=Genomic prediction accuracy for switchgrass traits related to bioenergy within differentiated populations.;citation_pages=142;citation_volume=18;citation_year=2018;citation_doi=10.1186/s12870-018-1360-z" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=R. Fristche-Neto;citation_author=D. Akdemir;citation_author=J.-L. Jannink;citation_title=Accuracy of genomic selection to predict maize single-crosses obtained through different mating designs.;citation_pages=1153-1162;citation_volume=131;citation_year=2018;citation_doi=10.1007/s00122-018-3068-8" />
<meta name="citation_reference" content="citation_journal_title=BioEnergy Res.;citation_author=J. A. Gallagher;citation_author=L. B. Turner;citation_author=A. J. Cairns;citation_author=M. Farrell;citation_author=J. A. Lovatt;citation_title=Genetic differentiation in response to selection for water-soluble carbohydrate content in perennial ryegrass (Lolium perenne L.).;citation_pages=77-90;citation_volume=8;citation_year=2015;citation_doi=10.1007/s12155-014-9491-z" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. García-Ruiz;citation_author=J. B. Cole;citation_author=P. M. VanRaden;citation_author=G. R. Wiggans;citation_author=F. J. Ruiz-López;citation_title=Changes in genetic selection differentials and generation intervals in US Holstein dairy cattle as a result of genomic selection.;citation_volume=113;citation_year=2016;citation_pmid=27354521;citation_doi=10.1073/pnas.1519061113" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=D. J. Garrick;citation_author=J. F. Taylor;citation_author=R. L. Fernando;citation_title=Deregressing estimated breeding values and weighting information for genomic regression analyses.;citation_pages=55-55;citation_volume=41;citation_year=2009;citation_pmid=20043827;citation_doi=10.1186/1297-9686-41-55" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. C. Glaubitz;citation_author=T. M. Casstevens;citation_author=F. Lu;citation_author=J. Harriman;citation_author=R. J. Elshire;citation_title=TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline.;citation_pages=e90346;citation_volume=9;citation_year=2014;citation_doi=10.1371/journal.pone.0090346" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=M. E. Goddard;citation_author=B. J. Hayes;citation_author=T. H. Meuwissen;citation_title=Using the genomic relationship matrix to predict the accuracy of genomic selection.;citation_pages=409-421;citation_volume=128;citation_year=2011;citation_issue=6;citation_pmid=22059574;citation_doi=10.1111/j.1439-0388.2011.00964.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. M. González-Camacho;citation_author=L. Ornella;citation_author=P. Pérez-Rodríguez;citation_author=D. Gianola;citation_author=S. Dreisigacker;citation_title=Applications of machine learning methods to genomic selection in breeding wheat for rust resistance.;citation_pages=1-15;citation_volume=11;citation_year=2018;citation_doi=10.3835/plantgenome2017.11.0104" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=N. F. Grinberg;citation_author=A. Lovatt;citation_author=M. Hegarty;citation_author=A. Lovatt;citation_author=K. P. Skøt;citation_title=Implementation of genomic prediction in Lolium perenne (L.) breeding populations.;citation_pages=133;citation_volume=7;citation_year=2016;citation_doi=10.3389/fpls.2016.00133" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=K. Kizilkaya;citation_author=D. J. Garrick;citation_title=Extension of the bayesian alphabet for genomic selection.;citation_pages=186-186;citation_volume=12;citation_year=2011;citation_pmid=21605355;citation_doi=10.1186/1471-2105-12-186" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=H. Hahn;citation_author=W. Huth;citation_author=W. Schöberlein;citation_author=W. Diepenbrock;citation_title=Detection of endophytic fungi in Festuca spp. by means of tissue print immunoassay.;citation_pages=217-222;citation_volume=122;citation_year=2003;citation_doi=10.1046/j.1439-0523.2003.00855.x" />
<meta name="citation_reference" content="citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=B. Hayes;citation_author=J. Panozzo;citation_author=C. Walker;citation_author=A. Choy;citation_author=S. Kant;citation_title=Accelerating wheat breeding for end-use quality with multi-trait genomic predictions incorporating near infrared and nuclear magnetic resonance-derived phenotypes.;citation_pages=2505-2519;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-017-2972-7" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=B. J. Hayes;citation_author=N. O. Cogan;citation_author=L. W. Pembleton;citation_author=M. E. Goddard;citation_author=J. Wang;citation_title=Prospects for genomic selection in forage plant species.;citation_pages=133-143;citation_volume=132;citation_year=2013;citation_doi=10.1111/pbr.12037" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. L. Heffner;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic Selection for Crop Improvement;citation_pages=1-12;citation_volume=49;citation_year=2009;citation_issue=1;citation_doi=10.2135/cropsci2008.08.0512" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=Perspectives for genomic selection applications and research in plants.;citation_pages=1-12;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2014.03.0249" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed. Rev.;citation_author=J. B. Holland;citation_author=W. E. Nyquist;citation_author=C. T. Cervantes-Martínez;citation_title=Estimating and interpreting heritability for plant breeding: an update.;citation_pages=9-111;citation_volume=22;citation_year=2003;citation_doi=10.1002/9780470650202.ch2" />
<meta name="citation_reference" content="citation_journal_title=Grass Forage Sci.;citation_author=M. Humphreys;citation_title=Water‐soluble carbohydrates in perennial ryegrass breeding.;citation_pages=237-244;citation_volume=44;citation_year=1989;citation_doi=10.1111/j.1365-2494.1989.tb01932.x" />
<meta name="citation_reference" content="citation_journal_title=Grass Forage Sci.;citation_author=M. Humphreys;citation_title=Water‐soluble carbohydrates in perennial ryegrass breeding: III. Relationships with herbage production, digestibility and crude protein content.;citation_pages=423-430;citation_volume=44;citation_year=1989;citation_doi=10.1111/j.1365-2494.1989.tb01942.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Environ.;citation_author=M. G. Hunt;citation_author=S. Rasmussen;citation_author=P. C. Newton;citation_author=A. J. Parsons;citation_author=J. A. Newman;citation_title=Near‐term impacts of elevated CO2, nitrogen and fungal endophyte‐infection on Lolium perenne L. growth, chemical composition and alkaloid production.;citation_pages=1345-1354;citation_volume=28;citation_year=2005;citation_doi=10.1111/j.1365-3040.2005.01367.x" />
<meta name="citation_reference" content="citation_journal_title=Ir. J. Agric. Food Res.;citation_author=A. Jafari;citation_author=V. Connolly;citation_author=A. Frolich;citation_author=E. Walsh;citation_title=A note on estimation of quality parameters in perennial ryegrass by near infrared reflectance spectroscopy.;citation_pages=293-299;citation_volume=42;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Ir. J. Agric. Food Res.;citation_author=A. Jafari;citation_author=V. Connolly;citation_author=E. Walsh;citation_title=Genetic analysis of yield and quality in full-sib families of perennial ryegrass (Lolium perenne L.) under two cutting managements.;citation_pages=275-292;citation_volume=42;citation_year=2003" />
<meta name="citation_reference" content="citation_title=Environmental and genetic variation for water soluble carbohydrate content in cool season forage grasses. In Carbohydrates-Comprehensive Studies on Glycobiology and Glycotechnology. IntechOpen;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=M. Jahufer;citation_author=D. Luo;citation_title=DeltaGen: A Comprehensive Decision Support Tool for Plant Breeders.;citation_pages=1118-1131;citation_volume=58;citation_year=2018;citation_doi=10.2135/cropsci2017.07.0456" />
<meta name="citation_reference" content="citation_journal_title=Brief. Funct. Genomics;citation_author=J.-L. Jannink;citation_author=A. J. Lorenz;citation_author=H. Iwata;citation_title=Genomic selection in plant breeding: from theory to practice.;citation_volume=9;citation_year=2010;citation_pmid=20156985;citation_doi=10.1093/bfgp/elq001" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M. Jermyn;citation_title=A new method for determining ketohexoses in the presence of aldohexoses.;citation_pages=38-39;citation_volume=177;citation_year=1956;citation_doi=10.1038/177038a0" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=C. Jia;citation_author=F. Zhao;citation_author=X. Wang;citation_author=J. Han;citation_author=H. Zhao;citation_title=Genomic Prediction for 25 Agronomic and Quality Traits in Alfalfa (Medicago sativa).;citation_pages=1220;citation_volume=9;citation_year=2018;citation_doi=10.3389/fpls.2018.01220" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Jia;citation_author=J.-L. Jannink;citation_title=Multiple-trait genomic selection methods increase genetic value prediction accuracy.;citation_volume=192;citation_year=2012;citation_pmid=23086217;citation_doi=10.1534/genetics.112.144246" />
<meta name="citation_reference" content="citation_year=1991" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=H. Jung;citation_author=M. Allen;citation_title=Characteristics of plant cell walls affecting intake and digestibility of forages by ruminants;citation_pages=2774-2790;citation_volume=73;citation_year=1995;citation_issue=9;citation_pmid=8582870;citation_doi=10.2527/1995.7392774x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Kearse;citation_author=R. Moir;citation_author=A. Wilson;citation_author=S. Stones-Havas;citation_author=M. Cheung;citation_title=Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data.;citation_volume=28;citation_year=2012;citation_pmid=22543367;citation_doi=10.1093/bioinformatics/bts199" />
<meta name="citation_reference" content="citation_journal_title=Neth. J. Agric. Sci.;citation_author=A. Kemp;citation_author=M. L. T Hart;citation_title=Grass tetany in grazing milking cows.;citation_pages=4-17;citation_volume=5;citation_year=1957" />
<meta name="citation_reference" content="citation_journal_title=BioData Min.;citation_author=W. B. Langdon;citation_title=Performance of genetic programming optimised Bowtie2 on genome comparison and analytic testing (GCAT) benchmarks.;citation_pages=1;citation_volume=8;citation_year=2015;citation_doi=10.1186/s13040-014-0034-0" />
<meta name="citation_reference" content="citation_journal_title=Anim. Prod. Sci.;citation_author=C. Leddin;citation_author=J. Jacobs;citation_author=K. Smith;citation_author=K. Giri;citation_author=B. Malcolm;citation_title=Development of a system to rank perennial ryegrass cultivars according to their economic value to dairy farm businesses in south-eastern Australia.;citation_pages=1552-1558;citation_volume=58;citation_year=2018;citation_doi=10.1071/AN17815" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=X. Li;citation_author=Y. Wei;citation_author=A. Acharya;citation_author=J. L. Hansen;citation_author=J. L. Crawford;citation_title=Genomic prediction of biomass yield in two selection cycles of a tetraploid alfalfa breeding population.;citation_volume=8;citation_year=2015;citation_doi=10.3835/plantgenome2014.12.0090" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. E. Lipka;citation_author=F. Lu;citation_author=J. H. Cherney;citation_author=E. S. Buckler;citation_author=M. D. Casler;citation_title=Accelerating the switchgrass (Panicum virgatum L.) breeding cycle using genomic selection approaches.;citation_pages=e112227;citation_volume=9;citation_year=2014;citation_doi=10.1371/journal.pone.0112227" />
<meta name="citation_reference" content="citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=A. J. Lorenz;citation_author=K. P. Smith;citation_author=J.-L. Jannink;citation_title=Potential and optimization of genomic selection for Fusarium head blight resistance in six-row barley.;citation_pages=1609-1621;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.09.0503" />
<meta name="citation_reference" content="citation_journal_title=Crop Pasture Sci.;citation_author=D. G. Masters;citation_author=H. C. Norman;citation_author=D. T. Thomas;citation_title=Minerals in pastures—are we meeting the needs of livestock?;citation_year=2019;citation_doi=10.1071/CP18546" />
<meta name="citation_reference" content="citation_journal_title=Plant Soil;citation_author=R. W. McDowell;citation_author=A. N. Sharpley;citation_author=J. R. Crush;citation_author=T. Simmons;citation_title=Phosphorus in pasture plants: potential implications for phosphorus loss in surface runoff.;citation_pages=23-35;citation_volume=345;citation_year=2011;citation_doi=10.1007/s11104-010-0687-5" />
<meta name="citation_reference" content="McEvoy, M., N. McHugh, M. O’Donovan, D. Grogan, and L. Shalloo, 2014 Pasture profit index: updated economic values and inclusion of persistency. EGF at 50: The future of European grasslands. Proceedings of the 25th General Meeting of the European Grassland Federation, Aberystwyth, Wales, 7–September 11, 2014: 843–845." />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=M. McEvoy;citation_author=M. O’Donovan;citation_author=L. Shalloo;citation_title=Development and application of an economic ranking index for perennial ryegrass cultivars.;citation_pages=1627-1639;citation_volume=94;citation_year=2011;citation_issue=3;citation_pmid=21338830;citation_doi=10.3168/jds.2010-3322" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=Hortic. Res.;citation_author=H. Muranty;citation_author=M. Troggio;citation_author=I. B. Sadok;citation_author=M. Al Rifaï;citation_author=A. Auwerkerken;citation_title=Accuracy and responses of genomic selection on key traits in apple breeding.;citation_pages=15060;citation_volume=2;citation_year=2015;citation_doi=10.1038/hortres.2015.60" />
<meta name="citation_reference" content="citation_title=Breeding Strategies for Sustainable Forage and Turf Grass Improvement;citation_title=Genetic diversity for cell wall digestibility in a diverse Lolium perenne collection;citation_pages=261-267;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Norman;citation_author=J. Taylor;citation_author=J. Edwards;citation_author=H. Kuchel;citation_title=Optimising genomic selection in wheat: effect of marker density, population size and population structure on prediction accuracy. G3: Genes, Genomes;citation_pages=2889-2899;citation_volume=8;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=T. Ostersen;citation_author=O. F. Christensen;citation_author=M. Henryon;citation_author=B. Nielsen;citation_author=G. Su;citation_title=Deregressed EBV as the response variable yield more reliable genomic predictions than traditional EBV in pure-bred pigs.;citation_pages=38-38;citation_volume=43;citation_year=2011;citation_pmid=22070746;citation_doi=10.1186/1297-9686-43-38" />
<meta name="citation_reference" content="citation_title=GenStat for windows introduction;citation_pages=204;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=L. W. Pembleton;citation_author=C. Inch;citation_author=R. C. Baillie;citation_author=M. C. Drayton;citation_author=P. Thakur;citation_title=Exploitation of data from breeding programs supports rapid implementation of genomic selection for key agronomic traits in perennial ryegrass.;citation_pages=1891-1902;citation_volume=131;citation_year=2018;citation_doi=10.1007/s00122-018-3121-7" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pérez;citation_author=G. de Los Campos;citation_title=Genome-wide regression and prediction with the BGLR statistical package.;citation_volume=198;citation_year=2014;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="citation_journal_abbrev=Biom J;citation_author=H. P. Piepho;citation_author=J. Moehring;citation_author=T. Schulz‐Streeck;citation_author=J. O. Ogutu;citation_title=A stage-wise approach for the analysis of multi-environment trials.;citation_pages=844-860;citation_volume=54;citation_year=2012;citation_issue=6;citation_pmid=23007738;citation_doi=10.1002/bimj.201100219" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Poland;citation_author=J. Endelman;citation_author=J. Dawson;citation_author=J. Rutkoski;citation_author=S. Wu;citation_title=Genomic selection in wheat breeding using genotyping-by-sequencing.;citation_pages=103-113;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.06.0006" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=B. Raymond;citation_author=A. C. Bouwman;citation_author=C. Schrooten;citation_author=J. Houwing-Duistermaat;citation_author=R. F. Veerkamp;citation_title=Utility of whole-genome sequence data for across-breed genomic prediction.;citation_pages=27;citation_volume=50;citation_year=2018;citation_doi=10.1186/s12711-018-0396-8" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=J. E. Rutkoski;citation_author=E. L. Heffner;citation_author=M. E. Sorrells;citation_title=Genomic selection for durable stem rust resistance in wheat.;citation_pages=161-173;citation_volume=179;citation_year=2011;citation_doi=10.1007/s10681-010-0301-1" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=T. Schulz-Streeck;citation_author=J. O. Ogutu;citation_author=H.-P. Piepho;citation_title=Comparisons of single-stage and two-stage approaches to genomic selection.;citation_pages=69-82;citation_volume=126;citation_year=2013;citation_pmid=22903736;citation_doi=10.1007/s00122-012-1960-1" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=H. Shinozuka;citation_author=N. O. Cogan;citation_author=G. C. Spangenberg;citation_author=J. W. Forster;citation_title=Quantitative Trait Locus (QTL) meta-analysis and comparative genomics for candidate gene prediction in perennial ryegrass (Lolium perenne L.).;citation_pages=101-101;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=23137269;citation_doi=10.1186/1471-2156-13-101" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Sci.;citation_author=A. Smith;citation_author=P. Lim;citation_author=B. R. Cullis;citation_title=The design and analysis of multi-phase plant breeding experiments.;citation_pages=393-409;citation_volume=144;citation_year=2006;citation_doi=10.1017/S0021859606006319" />
<meta name="citation_reference" content="citation_journal_title=Aust. J. Agric. Res.;citation_author=K. Smith;citation_author=G. Rebetzke;citation_author=H. Eagles;citation_author=M. Anderson;citation_author=H. Easton;citation_title=Genetic control of mineral concentration and yield in perennial ryegrass (Lolium perenne L.), with special emphasis on minerals related to grass tetany.;citation_pages=79-86;citation_volume=50;citation_year=1999;citation_doi=10.1071/A98070" />
<meta name="citation_reference" content="citation_journal_title=Grass Forage Sci.;citation_author=K. Smith;citation_author=K. Reed;citation_author=J. Foot;citation_title=An assessment of the relative importance of specific traits for the genetic improvement of nutritive value in dairy pasture.;citation_pages=167-175;citation_volume=52;citation_year=1997;citation_doi=10.1111/j.1365-2494.1997.tb02347.x" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. Sun;citation_author=J. A. Poland;citation_author=S. Mondal;citation_author=J. Crossa;citation_author=P. Juliana;citation_title=High-throughput phenotyping platforms enhance genomic selection for wheat grain yield across populations and cycles in early stage.;citation_volume=132;citation_year=2019;citation_doi=10.1007/s00122-019-03309-0" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Sun;citation_author=J. E. Rutkoski;citation_author=J. A. Poland;citation_author=J. Crossa;citation_author=J.-L. Jannink;citation_title=Multitrait, random regression, or simple repeatability model in high-throughput phenotyping data improve genomic prediction for wheat grain yield.;citation_pages=1-12;citation_volume=10;citation_year=2017;citation_doi=10.3835/plantgenome2016.11.0111" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=L. B. Turner;citation_author=A. J. Cairns;citation_author=I. P. Armstead;citation_author=J. Ashton;citation_author=K. Skøt;citation_title=Dissecting the regulation of fructan metabolism in perennial ryegrass (Lolium perenne) with quantitative trait locus mapping.;citation_pages=45-57;citation_volume=169;citation_year=2006;citation_issue=1;citation_pmid=16390418;citation_doi=10.1111/j.1469-8137.2005.01575.x" />
<meta name="citation_reference" content="citation_journal_title=Grass Forage Sci.;citation_author=F. van Parijs;citation_author=C. Van Waes;citation_author=B. Vandecasteele;citation_author=G. Haesaert;citation_author=I. Roldán‐Ruiz;citation_title=The optimal lignin quantification method to breed for an improved cell wall digestibility in perennial ryegrass.;citation_pages=101-111;citation_volume=73;citation_year=2018;citation_doi=10.1111/gfs.12293" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=PROCEEDINGS OF THE CONFERENCE-NEW ZEALAND GRASSLAND ASSOCIATION;citation_author=G. Waghorn;citation_title=What is dietary metabolisable energy?;citation_pages=153;citation_volume=69;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=New Zealand Veterinary Journal;citation_journal_abbrev=New Zealand Veterinary Journal;citation_author=G. Waghorn;citation_author=D. Clark;citation_title=Feeding value of pastures for ruminants.;citation_pages=320-331;citation_volume=52;citation_year=2004;citation_issue=6;citation_pmid=15768132;citation_doi=10.1080/00480169.2004.36448" />
<meta name="citation_reference" content="citation_journal_title=Crop Pasture Sci.;citation_author=J. Wang;citation_author=N. Cogan;citation_author=L. Pembleton;citation_author=J. Forster;citation_title=Variance, inter-trait correlation, heritability and trait-marker association of herbage yield, nutritive values, and morphological characteristics in Italian ryegrass (Lolium multiflorum Lam.).;citation_pages=973-984;citation_volume=66;citation_year=2015;citation_doi=10.1071/CP15070" />
<meta name="citation_reference" content="citation_journal_title=Grass Forage Sci.;citation_author=J. Wheeler;citation_author=J. Corbett;citation_title=Criteria for breeding forages of improved feeding value: results of a Delphi survey.;citation_pages=77-83;citation_volume=44;citation_year=1989;citation_doi=10.1111/j.1365-2494.1989.tb01912.x" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=P. Wilkins;citation_title=Breeding perennial ryegrass for agriculture.;citation_pages=201-214;citation_volume=52;citation_year=1991;citation_doi=10.1007/BF00029397" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Sci.;citation_author=P. Wilkins;citation_author=M. Humphreys;citation_title=Progress in breeding perennial forage grasses for temperate agriculture.;citation_pages=129-150;citation_volume=140;citation_year=2003;citation_doi=10.1017/S0021859603003058" />
<meta name="citation_reference" content="citation_journal_title=N. Z. J. Agric. Res.;citation_author=W. Williams;citation_author=H. Easton;citation_author=C. Jones;citation_title=Future options and targets for pasture plant breeding in New Zealand.;citation_pages=223-248;citation_volume=50;citation_year=2007;citation_doi=10.1080/00288230709510292" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=Y. Zhao;citation_author=M. Gowda;citation_author=W. Liu;citation_author=T. Würschum;citation_author=H. P. Maurer;citation_title=Accuracy of genomic selection in European maize elite breeding populations.;citation_pages=769-776;citation_volume=124;citation_year=2012;citation_issue=4;citation_pmid=22075809;citation_doi=10.1007/s00122-011-1745-y" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Zhong;citation_author=J. C. Dekkers;citation_author=R. L. Fernando;citation_author=J.-L. Jannink;citation_title=Factors affecting accuracy from genomic selection in populations derived from multiple inbred lines: a barley case study.;citation_volume=182;citation_year=2009;citation_pmid=19299342;citation_doi=10.1534/genetics.108.098277" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Predictive Ability for Foliar Nutritive Traits in Perennial Ryegrass" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
<meta name="twitter:description" content="Forage nutritive value impacts animal nutrition, which underpins livestock productivity, reproduction and health. Genetic improvement for nutritive traits in perennial ryegrass has been limited, as they are typically expensive and time-consuming to measure through conventional methods. Genomic selection is appropriate for such complex and expensive traits, enabling cost-effective prediction of breeding values using genome-wide markers. The aims of the present study were to assess the potential of genomic selection for a range of nutritive traits in a multi-population training set, and to quantify contributions of family, location and family-by-location variance components to trait variation and heritability for nutritive traits. The training set consisted of a total of 517 half-sibling (half-sib) families, from five advanced breeding populations, evaluated in two distinct New Zealand grazing environments. Autumn-harvested samples were analyzed for 18 nutritive traits and maternal parents of the half-sib families were genotyped using genotyping-by-sequencing. Significant ( P &lt; 0.05) family variance was detected for all nutritive traits and genomic heritability ( h2g ) was moderate to high (0.20 to 0.74). Family-by-location interactions were significant and particularly large for water soluble carbohydrate (WSC), crude fat, phosphorus (P) and crude protein. GBLUP, KGD-GBLUP and BayesCπ genomic prediction models displayed similar predictive ability, estimated by 10-fold cross validation, for all nutritive traits with values ranging from r = 0.16 to 0.45 using phenotypes from across two locations. High predictive ability was observed for the mineral traits sulfur (0.44), sodium (0.45) and magnesium (0.45) and the lowest values were observed for P (0.16), digestibility (0.22) and high molecular weight WSC (0.23). Predictive ability estimates for most nutritive traits were retained when marker number was reduced from one million to as few as 50,000. The moderate to high predictive abilities observed suggests implementation of genomic selection is feasible for most of the nutritive traits examined." />
<meta name="og-title" property="og:title" content="Genomic Predictive Ability for Foliar Nutritive Traits in Perennial Ryegrass" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/2/695" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Forage nutritive value impacts animal nutrition, which underpins livestock productivity, reproduction and health. Genetic improvement for nutritive traits in perennial ryegrass has been limited, as they are typically expensive and time-consuming to measure through conventional methods. Genomic selection is appropriate for such complex and expensive traits, enabling cost-effective prediction of breeding values using genome-wide markers. The aims of the present study were to assess the potential of genomic selection for a range of nutritive traits in a multi-population training set, and to quantify contributions of family, location and family-by-location variance components to trait variation and heritability for nutritive traits. The training set consisted of a total of 517 half-sibling (half-sib) families, from five advanced breeding populations, evaluated in two distinct New Zealand grazing environments. Autumn-harvested samples were analyzed for 18 nutritive traits and maternal parents of the half-sib families were genotyped using genotyping-by-sequencing. Significant ( P &lt; 0.05) family variance was detected for all nutritive traits and genomic heritability ( h2g ) was moderate to high (0.20 to 0.74). Family-by-location interactions were significant and particularly large for water soluble carbohydrate (WSC), crude fat, phosphorus (P) and crude protein. GBLUP, KGD-GBLUP and BayesCπ genomic prediction models displayed similar predictive ability, estimated by 10-fold cross validation, for all nutritive traits with values ranging from r = 0.16 to 0.45 using phenotypes from across two locations. High predictive ability was observed for the mineral traits sulfur (0.44), sodium (0.45) and magnesium (0.45) and the lowest values were observed for P (0.16), digestibility (0.22) and high molecular weight WSC (0.23). Predictive ability estimates for most nutritive traits were retained when marker number was reduced from one million to as few as 50,000. The moderate to high predictive abilities observed suggests implementation of genomic selection is feasible for most of the nutritive traits examined." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
    <title>Genomic Predictive Ability for Foliar Nutritive Traits in Perennial Ryegrass | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"DiRW6beMG3O_R7kdNvEaYAARFXfNDqp5cElkyGphI7M","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495555","apath":"\/ggg\/10\/2\/695.atom","pisa":"ggg;10\/2\/695","ac":{"\/ggg\/10\/2\/695.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/2\/695.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/2\/695"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/2\/695"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/2\/695":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jWgFbrdAmM+5LUvLIvMgUpaN6Wz1eSAJeK7ydpY0JxuAhxcNmvOt6Dbheu74RlK6DAU6wuYlgoSy0kJNuQucGMtqxuDiO0L83EqiU9W8nj+hFhifs1J2M9mFSlMprpS1z6x\\\\\\\/bikmxjVvKna2+MmsLYalug0r3SbjpZb6Cr83E+Pn+DtaNUoTVBdEsJ8w2DHkwPo0KGhB7ziARI+jqEzOewT\\\\\\\/wfu7kdlOJawrcO7RbA8fB00L6QZC022hFq1VxPqA91U6A9KSSWtKSmH2RXiTC4c0m\\\\\\\/T8aGQsMFLKY17i42OcykpdfjSdArnNMR1Y1R3zRxtpDCFT5IJv0txKCldkGqkDcjhKgumQeriXm1C4j\\\\\\\/7xrD9Y5gjm8Puv5Sp71qhZLN52rtkGuRSCr+4ia7ZkXdk92kiIdnXkFlA32LmVcSKeRbSM3xmGP3KwvxMsZxjoL3hxRcCRQz\\\\\\\/Ga\\\\\\\/cpvC4d34lDXac039EZrmA\\\\\\\/Q6pQE1O4hUzxG4npv16EN1P3uKD87bY7tyQFcOcela5pVCfeI7quWmT9rQI+SBLi\\\\\\\/0uoreF6Skr9oXTpp1t8Bsx3EOU6rzubpKVcAa8VJppKijwwr2Wr9pZu6XkM48ISN29AxdDdyQRjZMC1RhGVd7SxIRvjRgsLfCKFW15jhOOlS78h5wyYf5ZJrjQsuk0rX5jUnvoNl2FGrqJDICm2abTlKRl1JOAfXmAhwLPQhv4kvk7oeRV77cuAn58CqudyiipcI6lqeD9LP9f\\\\\\\/zs4W8cOlDyGLJH4O30Xjdtg7p8seUgdPSGzrc3zha\\\\\\\/I6WCxKtGSyCZpqNwta8X4Ul0OEuoo+qTPSORJNd2NAyOrLkTqY4Tcyku2Ibd+hKtQQ\\\\\\\/zEuPfOJUkkrqmmIHLZff4C3grpkJulay+3WoCB+dU2yRBUTpH+5NJi\\\\\\\/93VKwawB48hft\\\\\\\/y0fx99PagiwAnarSMHbhB4AWFkS8y41b+ztdRk2Azv0xGFQNWmsRCZpt8gR9FCQoHHbI0XxU+ZSPgy1gUbFaetSojaj14gXwv\\\\\\\/yPDl\\\\\\\/ijkou8dcGq95Nmw296bQaXbDrzT4JtWSIwESrNHdeHZ8tRxVreiB22nDJ\\\\\\\/JFQjVZT4vQq\\\\\\\/1kkUoSNaRH1Pm2Rj6kH3d6tEIKwpMWo3cCacrhs2bJa5NTqu2aO09EB6CZ+FSyFnNfnyN3CsyZdAVt9hn4DksdMBZ5te1iWqpdHvZ4xTgMIJkUIQDcsD5SGuUfcta0r4iSgvsOgzG3k6eDvt0\\\\\\\/U7+EKasE4fYwajcV69zYKLIKRxB\\\u0022,\\\u0022iv\\\u0022:\\\u0022C4px\\\\\\\/OYx4yH+lO+QHJiIWQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229ab7c2aad034449826fd4bc4f089a0ef4a59fe9917dc23a906e32376cd991202\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022M3GCBSlmjMmzw\\\\\\\/hzTHIl8\\\\\\\/PNUa5PBm4JXb6amrM8YNaLWKVAAtl0+Kj66SIuPs93Z6rFM+EZs8bVcTpTr0ZO\\\\\\\/YAQ\\\\\\\/DDTuc05kO8FQShVePk\\\\\\\/YoRBvu\\\\\\\/\\\\\\\/hwqycuG1\\\\\\\/OjvAUwbXdcsVYiTHTdDWHpx2LGVkJW+MnzUUCoawMuCTq+voV7Iu7LslEzdPV6A8DeMFN7P69kyBDn9r6OvOQSyORnZv9LeWP9cfCb5vUMI6kyIPtaewHPU3agOMsP3nSMBogjqlBUL\\\\\\\/ksEN9\\\\\\\/HEygne1BXQ37R2eJtr5LUs0LE1Bt08V27qK1eZV1sVmBTP0lrG8k0sZvCblbnVb59Z+OZj5QvXc1PftUfwwW9rcR0HG1hXnBuUplur+K5l22oVAl3lPau4mh3HBrbiOyD7wUnCJhH2rw7UKSOKiLvNcCGUYDPSAz4o3d2lNU0j0kd9wtMpUYVD1nxDcvY0b6ykiUJPlNMntcyFI2ZKWws4TXBa6nW68f52tGD8zlzOwUR\\\\\\\/K3dBEBkbWadLbO2xfIDKXY0wK8zIFsiDu7SnzRSYqgJF\\\\\\\/DVOBvcuBR3mFjXtBbHI5\\\\\\\/eiblApQ1Ce9lUB1NSxCEzuHOf53aOTocW5wR+OdNv5tvhBnWjrk79Nn\\\\\\\/nDw4\\\\\\\/3D7fDxJJKr\\\\\\\/GOFzrDYHs3qEdUNdxc+PMWP0QsxED9yTbzHEtZeG3ekjhUIksVelhwWEbFshcaR0ylftwbzpjg\\\\\\\/Lpsu0zaV10aLFCYX\\\\\\\/cwKu2LOLiixCl6ASN8ziHEcU5lrHHevLDC+8DKBVxqDwf1UKPZiHM40p9zGWyMaqLKGp\\\\\\\/rXDOENbfSCet7C\\\\\\\/XJ94JWdaNn3HTqmNL8UVWVLlZpgSPa5A2nlhjrTHemix0Epn8rSxZBvKG9jiZPoK0htsDIFYoNm0sBpPMT1JLAEZwN1wUvvQo0\\\\\\\/U2HIJXDKo9J2yxOYuRB5lIp8j617FoK\\\\\\\/xXE8jnwEN0r11p3UNBMdhjOv6jcR7c4q9dg0gJhbmYHdOiVDjzHt83P6vQMRLH5dYwz9op8qMC39HKoXNNsUR6LFIv17K1i4kXqiarqmiMxAUM8UyPtOWJkJj0icD6vzKfQuk26GbtI1q3yp8OYgrBU+IHcdp9eIKxZgIYPzHVo11Eh3OXxJz7q\\\\\\\/lrvCkrc0R9rKi20ecUghmLJzXeVhCuloTTbP8hEXDrJ1pAwuL1SA\\\\\\\/R4inlDpop3rEk0Do5edG4mIN5CmsgAlK4PJJp6y6xfU6MOd3138rm9BBadx9kp9SC+\\\\\\\/YspUtaQHzK6\\\\\\\/Gc0HgnCVWmqgg4h362EcWRhRoot2cu9W1kcqX48a6M0gtmD2aEdBsytgLuj\\\\\\\/GojCmJez4072cA6IAxgjTEUUheozu96MTjbyg9QBrE9aqUu6eWx\\\\\\\/LQp9QFMDOPYDXGnhQa7xp8YQm\\\\\\\/nDH1zl\\\\\\\/VNVZVpyOwyZfU8zbzCmDXRnSxXgPKWsUU9+MsVTePr98FxI0CUiy92yC9D7ESIQf2UvTZ6tFJ5pSwKqawI3nvWBxCledq8fQNYL5IfJOEqPCIFMwplpsbz\\\\\\\/PIbSHaA5+9YClCRv+Yv8HNL6Y4HGg98SZGzwHshHyelrej1NLVpgoKoEHIo0e\\\\\\\/rOPmbEedFnsZ1q3RTESSF4yUVzVT4ny2Jpl3wpiR6nP\\\\\\\/znfvXB77TvvLfs2lFec29zivG7CzE9dMmDrrX9ldhwMFwHJXERcYi6wVnov3TbZ9BVFhPNGkyiGccaRvNqsUehICuh76daGXbZe\\\\\\\/OBOtrmcBK\\\\\\\/Aq7HeGhGihON4Bta8FKWNN\\\\\\\/8LKC68\\\\\\\/P0+Tf04FzgIAlt6Khh0FLuj7KQlnrWSdO7561gc3Oea4KasgGYrCOf7iZ9FBa213MbOe2FqrEXq4wkrUwsFAW2ek6v+7FqhxPZ\\\\\\\/xAp+GwM\\\\\\\/0gfVC4T8k1PvDyHS7eJXVGN3\\\\\\\/etg2AfgdVtwn\\\\\\\/ZytVmDAVV\\\\\\\/nuM+y7lQp\\\\\\\/oHmYEn9hy7FdiVg3fHCJLrPZu+K5CblMo0sV1NFY\\\\\\\/Kr0s7EFMQKuRpliPPBzym1yO4fWx\\\\\\\/eyLIOVAOlO+43kjtVMeTx6CDmZvQaMLDJfI3E\\\\\\\/kiH4Bd8afXUUimn0rjpO9+LoUDwLLX\\\\\\\/QyTcIOikgfHsynA\\\\\\\/jf1OBGpaVc6wvVM\\\\\\\/o65NtcBybT\\\\\\\/YpGOvgt862N3SpJJtLQhQBR3E6d+qMutUQ9UO7VnJjBX+qUwevumQAh+WTiccPk9Mgafmnhq+9QC6Yea9E61NmbKpfDC+381y9Ct8+nIFAdcOplSdY5Lt5p6C13yJ+8bR\\\\\\\/+4BW2u1PQ\\\\\\\/dotifgz1rAvogVdzwVP+SwAWYJFUuCzZj+0QEqSWLm4H+JWcn4+lp+rm3\\\\\\\/lwTbM948bgPOXMXVupN9v9bWWBj50ql6bCz+FA6X2O6Pt62Cij\\\\\\\/GRUxiF\\\\\\\/svEs7Im8C6N6Dz2+Loh3iXT4d5XOYYGaUsACjmYD\\\\\\\/LJzxilldXXKh0Eyg+Rw9i8wc21nAUqgKUxUdb3FwToPYlWvuyAr\\\\\\\/4xFIHZYTrlR8eo1eT1d1t4ae0wHjfZnOErchHgqW\\\\\\\/XLiYz0rh8DMDMwtyEjmOrufg\\\\\\\/EeihRK8t97ESsY3R6zPN2D9Te9Bd6LxQ75a5Eirx1FQvJdnr6PsRR+0YDYRnZxlBaR4Oy\\\\\\\/PDNU+Dejzirv8GSAc0vCsIBkDJL6tTwUdmtUlELz2AF9yCq6ggDEdKNTdAxN4O6LB08epRkiO7C6Y5icl\\\\\\\/HIM8bmB2kxGEP76EmgDHwA0eM9uEViqoSxLs0LLkRp9QBN+kd8EExWGuyE5kXzr4PAD2Irv+UgHO2qdmNAZ5fKBfasd6weUPjFGhe75LrJCME+N3YLsAWAIOh9x5\\\\\\\/oxcRcLkfuHWpbwGYQTQVa+w439YrJ1y038CBeXABDSdgU43hHUNKPmwe2uZlnR\\\\\\\/qYc7mrfFbJIspremPXOiBTAdJJhlszT8LZDFu\\\\\\\/+e8X19A+OZVw03b9kqUQkhwCZyaloApjJ5+KZjRUcZIWReKjyqtTpoVpJGKYqXfkFY6fmXEWXBbfvlcDQtIu4mkV+pNS9F8bNgilbXmtYoeIvE9RB+Rm6Zp4mZOX+7T+C4dfwe\\\\\\\/zMRD2vzeLOrNBWqRToSG4sj12JBeSQpn3dJLitwsZofLanamUiuG1k8YB1JxEWrF7n2lPCPFZqyY2YFX241cE1Tvv3NagutoD1+rkQ6TSG47OvBp2NyXKp6J83AZV9kw4xNOzgH85uAqoKEEdb8n413EGIPzWPH+\\\\\\\/DCw9hLNzQMQrrQlCu7QGuECp2cPJsU921nkwBk8IEIqCI1NJfcjlUrWECYTNZ8KTer3oC+PFG2QskyUEk68\\\\\\\/M1j3hmAeCdTjyg3wNRkBuMvBQLfbFxIN6trgJpgijXrihJ5I1\\\\\\\/kVNRQhxryVx4p2AJL\\\\\\\/lKISy6YlMLPFmlQuHS6TCb0SlthbBPcrZjNuTqJff5g5OQghapu571fUEMZrHugOXwmu+Aiwj58eGb3vzrPahwOHpxlz9AUfiZxqYyLu\\\\\\\/+3oT+okVzeZJCklFdvrDibuTZoimigs6RkbV\\\\\\\/HiFZ6cdXUTU8PHMvPTRoNxFtqeQTHnL\\\\\\\/febEZUrDxW+zJ3F74Fdv5vE0A45du\\\\\\\/mVtDThKGnHcDs4vwagHloba9Ja\\\\\\\/7BSkhnlAyic2QATY4EWajOo3DKZZFH6iElo1BwXq3HDrHOLqKXKBoTMYPipZo1caiXgzuaPeWJC85bvtz6roFVjvzd4K36OJR9WlFC1cBtJ8XpxsSUeE4On53xk44fCseYHwg8HEvzgXAn0ks76sFtnCpFCtmkW4VdKVAkD66Yg0XHtk2AnIL7W34iBPxiUXqaxT3sw1hZQ09uWLSuVlmsf6s8WtWV0SrkBoAVu2OSdDN0In42ryVbtf4Np14e8ZhJFz6jqm3pdc\\\\\\\/CaBcAsSpcYbd3+3iWhQD8v\\\\\\\/GNkdTrAM0M0WjByDsm6wxDqWUYXpplSW1CpxNGLK+YS+Cz+626fxtVp8E8r3mPs6gjq8NK0fECzHmvoS8prpL1sL0yg4+dW+9EJCypl37k7Mn5dyVwiGG\\\\\\\/IRMP5kg2185sfJ0+\\\\\\\/I3mlOdeNNNOgCDUELCgBYTUfAXpBdLlFb6MvIgFZurGhq9SXKEf7rdqe+i3tgmEpHUmyhWQICwrhAIsXa5+T+AWDpFmWBiazCwdzb0MVJ3gMVlZPkFoMJ\\\\\\\/1ut6wLFliLjFAM8G7PmiuZB4HZBx08PvCmlgYFo1NAFhJWckMJM7eMaAlOGiYAPcgOrQnK6jSTPLP+ahbcfO0HzmsDmlUA+dAnJXYyNJBntUJYcHrug6EsMB25uQZF4MvInrtBlzV3m52eWo6BlPBF85qPjoS5E8gPBZ5ivi9ZfT8L6BUOGyE3eJ4y5GpCm5fQ+h01Tbmt3g+KI5Gud8MuWq2F6FGBl03DPfCP2F774I6YR5oo5qAAqEwybPFG82KXR\\\\\\\/ZjjNorigg4fbscsGCvoRmpID6X5lxwkgIPLKigfBS3kTBQ6\\\\\\\/OfngyFGh54L\\\\\\\/ch2cnCcZVzBFDfnZ2LJk+5L3lsuS6dOUx3NaJvkFbtJvWJzvy4xfc6DXWIOIqIfbzz55+rZ1d6b25mcGfVnc4OzaBk7Y0mZABveopotjv\\\\\\\/Ezgk9cpYtg7Qzb+Dg6i2CKbFd28xnuaw1xVB04nS60Vliu9UIsJ4UaVuR6TYvjc6QshjCp4WylRzVapN1bF9b9O5NL06NMy0sh7a42rGcTrNDBy50Lyad28fJGe\\\\\\\/eFOwyil+CpRg7dCruwrwr11nUBNvkLRoELSdW1wKUgVNah5m\\\\\\\/kv0CqVzqytOg8q0TGVggh8AjrQbAGU7tw6NyI+jyVN0HLwOdxTKOwuP\\\\\\\/wYBFPanM6OI85we93+qKr7KV1OS5FbJ0tNT\\\\\\\/8f8dgZFlerW0QHZhpB34Zlrnx7QvRh419GiqUzNeD05QQhZ9w23yuDmmQTlZgsy+uqXfJZgz9XOmtI7C3\\\\\\\/uA2oYw\\\\\\\/8C3oLd1vSuVqYEvMNx4nwhiiYZ3sdV\\\\\\\/FhyeI\\\\\\\/uXCxuj0XK50EFwwY2HqYkep1ZpRKPyWq4ItgFSN6OTg15quqM\\\\\\\/Lf9aOIGpH8Dk1kS+4e10jspvjKnU+HPWx\\\\\\\/r0ppVQlHYq0XjjnxG8\\\\\\\/\\\\\\\/gMNavHhYL7ASWPpXl4C15CW5VQA1iQwAQ5xU2K4u4UuFtgpFJvgq2j5isL22Oq8Q8ngitTmIF8Rrg37q3bVo65qdWxzxt7iTCxTS+jfZS\\\\\\\/em1ZaMHbPm2UhlJXSwJDX1HJq2Asf0j3cVYBLDTYN4FakY1CnhoRgwUHraFCaVMyDiNF0mABIU82PFsGJiAvjXWHkQtwXjQGvvj9CzOake2l0FOWEnyGnp3OfJGRobiYv0pCwE\\\\\\\/8krb2KBQ6CP7JcW0EKb9lf5Xz3uzTkOVgf+YwqvDrQ+OE7lUsPFIiqkQaHuJ4AmqWRDEjNqIWUyFdX5O0bPrb+8AmKKG\\\\\\\/Bz+PpiOCKgTqyPe+fia3rJLN20Sw6qJ4pVPRhZBJS7ZJlW0hASolo4zGeVjTYJAkdj3t89dJcFgZxytJsUTPJhuGMJp\\\\\\\/0hDY+EsszecfXfqJj3B8x0ozkBVOQvtfhgmApM4Kies7kfdbVHvuM0uh0NHDjvInq0Soz75+aQG7yaBqm2IYDs7u8aYp7NTEcRemmN8vAC+QaiCDHg+XUmLMBTeslVHpbvJNcxXXGyrrEMaG98yDimSSeDPBufnSyREovrXsOoAfHQbqhcc7VrINrhTqY1JTTYTOyv9Xfzy10YRGwCGQinBF\\\\\\\/j19G8s3XZzHzBaXyPYo0ikAMi7mcLTK\\\\\\\/dgoUfbASedsgnAwezOCv8lo4SvfAnX0fnAKOcmD8fc6dVY5+JH8w7jaeYnT9AIDYXFx+j1N5a6McLU7tBEa12ZDkqEfJ1W0zKqWtWs+8k9kW5bQY0h4puzvnKEwnyKuY4SXa9guW7xvWk14+7wR7SluuTKn02lWcIrmhZ7+m61Cx+l0N+1rqQ8apwATKaRYjgNZjbxmAdm8xbhv\\\\\\\/At1roPmS9\\\\\\\/yB1R6CNvI1O92Dom8vYclnJfZB7TXlL28EOMv7aILAktF4+NZQ6LMjiTWATS4CGIZn9zR48BDn1dGygTlnbnlRM6E\\\\\\\/SEf3g5oWzdE25OGBkLCbKZn07o+CYRMICXDrZ8TyX+M3i5cBsF+aNKfXZ5hG8CN+5M2w45ClRejThx0r4CrfPJ9BT02J8jjtfeyoYgCb3sFiwBIL\\\\\\\/L3AGNsClO6PdXImWvYi9RLk0v9r04C6KCBY178U2NIwxpCOXNOe+EspJQ7h0EadEKqDr5thyFn8Kfo9N\\\\\\\/9ybuwZXlO7AYueDvSGjuiojWjObmTqu4NWM9zOf9xK++bSRkcv1jhDvHW5sHuXAk4tpYVl6wcGmpy0OPR022nKakL2a32bf6dl30E1zBNMtSkZqf9saa1JurjUELjZoBl0O9A88ERrszRLZ4gOnzWp8ujYxXu+fLEFhubyBzwkpZRnXznO01eBQ4gMAd98H+Djj3Qrb9UyR0v8x7DyQF5D+jjfImw3jTdWWlnhtxwF8uZDfA8oEJHv0r4PVbgA3AuffIsF\\\\\\\/JjnJMl02x0M1qZG6HZ2m4GdSbPV\\\\\\\/sCoMeTkeL1pnYUIjI4xaPe+ILzhTBHZ7UxNGGw42MmSrsx6GycJxSHy5o2wJxlf3MD6gu+pmo2P0kmfBoiDHmlwrtrYutjFnbwjxcpQ0\\\\\\\/WnsmNR+HyBUxoI+RA24uJ1zWevxEpycUgpd4EOhipUTrqOOGkOW6ytLMSIVn+akp00jffaV7lP7DPpxO\\\\\\\/3EoHVgrpOgLv4VUorjVRI22ThUQjlOnFeIbLqTcH1N4pMkEh7qNkwKGJtqqCvUFaCWJv0154UIf22HYMfytPD+vN12czWMUZnt+1q4xLJOReujz0JIbjAICE20V7A8WGdFMtnO+tLjcAKksVBRWgke8p3PiFHS5r8i9HgWLCqXckq+7tPuSXfDpkcE0100O5mkPdBnxVr9EmNE8Za9q3sTUqgnfia\\\\\\\/mokihRE71TGRsogB80FpKsMRM+9XWlUi1VYAsGGcm5p2qSeLIGs9VaGTshC7HxHa9nPwyjqU46u\\\\\\\/hvvxfsNpag\\\\\\\/rTKoeYMWAMIqP1dpNOyHq7WsRNmi0tHZoXRmUCB2fVflg3q9x\\\\\\\/Bv4BuTrS\\\\\\\/A3tG0jKKu34UdyYwbo45fFXX\\\\\\\/S\\\\\\\/ZKvbvjCH3neO4GNXoWm5TRmXag1zUZUqHMFNk3loq8f\\\\\\\/ICOPsBLwYBxWALRqiJcdw8kTST0F9hrBZmpkJGe8990TpxNtu5Uc\\\\\\\/o8NISPtcwSVjWHCH7amPTBvCiCoca6PKCnlrBR9l5hRC6gQdx8LPamZGxVirLwPO6yifjKVGNFGS0bKWZnYv8a4SBAM1pfjcpsVES3GQMwqsKDMOrUDWlNzma74\\\\\\\/bSQR4Bb2U28I4+7QUko+\\\\\\\/d0QwEP\\\\\\\/eAcAuGGosIEQrycGA6RZqhM\\\\\\\/mpBPmJcVKN+u0+fCDLVCxqMbD16UETeSD8tGPytKZrZnW\\\\\\\/AhZ0yGBytjjd3dJfXCW9huJqvNuNjFkrjYr7oGZGn6pKBPHi7AhwiWeYxfT+gPN813nocKULRftvWgPL0yeXxCkXPl2Hze0KOqmabWoo+fEw67l0Re4BZ24sprH5qhasSKEmKK4yK5831N5DzxlPWiybepGVNKFrHPlnWjVQQ\\\\\\\/qiQvFYLPk20SkJghAelUHRkntBvOtdjLMzO+AfXwPgJk8MGWK3HiwOpk1GNj7BL05pmEu5rQ1dRySMu5nofkXn2ygNDCCPmgwuxGcdZTjmyddXiO3MoCM6UpLQW3DaxnwbJHQLZ2RT96sNzbJcasuw296zDjz3lAvRjMdcESLjgSqbiGbrLNxbJTyhwMNXqcOlPMdV4d68OKzGlTUihhzIXy2nCO4GbyzsYfHJNnZk8MTzTCzCJdx8ih+FXGw8TKZHi5tNRsSE3wpYoF74dvtmaKU0Ubq\\\\\\\/DgOX1A3awxzAcO0KBxiFnfteCFwMZiYl7eHII\\\\\\\/dthi5Td9WNqL8ODaNnpU4Wq3Ra+arxRBP0C+woCpSTL0tvyy2ziCWKzAp5wAMUQnVAlrNsAyFwCZ3ljgpKE0KpOnURNsOXvJAJn6XMBiJnsaA449uaDSwj\\\\\\\/x8++RodVIxbRC1D98JBVnZ0dSqP+ke7XCDhLu4d3rDBeKQrGjPLQIG6pEdo+Nrv1oyOyEMkVJeh0J8Ny\\\\\\\/HLc9Tg15pJMkUG7pAQwzOsodUfzBXnihrQIRmFXQpVTK6y\\\\\\\/OUiQpUl5Crc73dnHrTyOOnBtDnKpxRToVF1TU5qcLtrkTXlbsztiHumDWWUwHZ0Ts3YijNhdiPJ0ryzKTBeBeYOL2\\\\\\\/wgft8vysBNduIP+cuKYS6vpNM\\\\\\\/J+jhI2bk8ohJBkjsBsp+mavlqSLGxfiq05GvFa5ij0HPv6GUvzZjC5X9f0xMVxSBDTjvjEIgdCCaolwAFMZSVPlvXnBdSPzdSkooZxjsBabsp5ddqmGlsVMNw0KInL7dv2lnJG2hHS3pKTEMqTC+TN9oVJdCw4aHwj2wKXCq8iblWpepQtVPJiqUcBc8CwKTl2eV1PgEI25t7eM4beodMmbDYwGxp5+XtJxjK802x9ZYzP1Jq6VfEUD1li1SDliVSD4Gmc5h3maRJHysdTvdVWwraOfDrcx0THlJ0bEkQ3MzLvHrrNvjy4jFZtqS+q3qdP8tQoKS1FXAOW438dx1l3qIaY9YVbdwgjUGqfeFbnIoV85HVeHuRNBoS4sNjhv2QzfaUn46iieDfEiryKoj5iPjuWG\\\\\\\/zWIzgHHL7RGN3pA9iw4spwh89TZv5kAI22zC93M\\\\\\\/bc3xU97B8dA9gDuM4b8hcX+5EQUOLl5g\\\\\\\/V6\\\\\\\/FsPAEBTf+HQX3FI7fOoMbcHwguT6xuppdKm86BiUifL\\\\\\\/lAePPAHYXlxwS25etCe02UNdHMdrsNmrqAEBtrCrLzw1UFGijoixp6A3Rs\\\\\\\/ufoOqmcgfzkW4ElIEol7jQZC5mEJKIE523K3S+IbyAuyyGOyqWdvXzR2OvjRiYuyRLSyBH+ZBBJL8piMv7Lx5SnPIsbjiwvTp4BIypW7aSKXh\\\\\\\/RAr7NjJ5xlWV9qDsvpYIDjwhMAADkPiOooKQN8qkjV1eYedAibaLf4vO5XLJgpQe\\\\\\\/D+1czvQeI35LdIklzVQ8L9jUea3VQZHVY8sAN60mlcIcaeV\\\\\\\/0dLX63kn2SU3jtME8oCNu1J3seEMjO0P2vvBUgDpm9Ky8hPhvTRD19CD6QYtquNebGcPtJy2UsTZ9FtklBYySbtvIm+TJO2xshJJlRWj4l+\\\\\\\/4+rdlyWHNxjeW749sgGQRVsgpR36247Ba+YZVsmDPDY7RF3+\\\\\\\/AkU4s0pynNJt8MdEd2L1pac1K5RrsK4pxu2q3hEaW0hGgGYSu35gXrQdf1bUx1uEP7MBia1gzZ19630+A7AEEBo8MyljQrSVSuiZkW6raJjIV+8k4VO+j7U4MmQu\\\\\\\/OyaECQMplwT1enIk4yp8wSyy5kBfiyiEnnWHXqzHu2nlbIZi6oJMxBf42cUtPS47F5\\\\\\\/hyen8hFwkR3Ym\\\\\\\/RBTrDj1bvV1tPWcMbqpODe4HHuSR0ZfQqq4p5w+8HGLbX3dtn3c2bQQpIeY5UNix3eZ2U5GDiKWwHkd\\\\\\\/MKryXXcn+LHD9tGIKErCWDTqrawK4k4Ui1So3sTrUiF4N5GIhRi+WbDhK7GebbNEj+MHEi7rvkgroNwb2u4Zaiuy601+rGhfYQHt+YD2tfMk7+dPMabnu7LcS1m\\\\\\\/FGor6PJm9lg+j+NtfYyKj1rXPOg5okq\\\\\\\/HgQZwCqFB2h1Wv7aUW0S6tTce20boTZUJOBV2O7CCmfzyYqSQR6LcUDFJsBmbCzQoaUoNxXt1j96kQ2PaIQN+3Lh2kTZYVxNvvj9gn46Vf+oAR9\\\\\\\/BpriN+q3NoU7X+ls\\\\\\\/z1zptItw2lDHlBRXFL9zs3FCgJq\\\\\\\/+2whhdJLWN6fm\\\\\\\/dm2itgk4QGzaIKpTKBYd\\\\\\\/PEOyAk3BAQ9w+W2PL5CUn\\\\\\\/F4OxcRu2TZXS\\\\\\\/v\\\\\\\/vxfOibGE0rnLhmeTo6LUSKyxsy9MVDbrQ5IQa5BIwK0G2GDCYHb5A6SsHlU+p4ffvCOvui\\\\\\\/JZeVSO2U7dNpRP4rYqjU3WeuC40NGyxl90HZCuILZStyHD8zdyCXzncRfBVbAHFKj8fTfa7pZ3UfcejkytYE+RttPVCdoomruZosLzlN1wnuKFfrNRBtUmXcv3aBlQtz8rfHDETbb9LP9NQ7awYwzyTM31V9bcAs2L7SvMoc\\\\\\\/QzaLpxc5mBfnPeWs6hlxY72ee7nwMEFdeNKPWW8\\\\\\\/yvC0AQVkkiaJJ+\\\\\\\/tMuKGqZHlsZg2rSTVX0WVJuatGkDjVYOdTdSX9mh1ksHQTWbioyYZCQYi5PYlEMsqJMUxpB4uew+P\\\\\\\/Grg1t3gknFSNteWoNMzfQdqjIzOS41dDgYtc6zH4KyN8Heg4gEz29kLj6UlmAONFOWOGd\\\\\\\/PzEcaWSH0VqwisrTFGg\\\\\\\/SYRyn9M9fs2nyXm+u1vaWcN3zvYjt5vmpwcs5Req1Q94t33Q1C5MK3zOENtK9lVFIxf\\\\\\\/LL9BT0tr7cDIffLH3h8oId3EX9UlTqOZnYxu8nuHEmSK1EZFPlu\\\\\\\/vca1C9MZ8uZzEWtbjpJOE6IRr+880XFWgRqmZdSWzirxothsPTWTwu+1CpZ0V8qwX26aokJydXDU8IhFgqTDGS\\\\\\\/qC+falVFro406sjufeRml1mBm2Y4VPJg0U7U0oFGTnmsyDQ7PS3cPXDG3u2JFKeXm+c09XxI+yfUGIsSAt0spo93DJZWmVOxTzFYUcILaSnNVBneymRoYcAwtOPHwIPQAs52OSrSihiUJD6eGBkbaon3Pnw118bngVsX1DcXg16WTpPAzoVbQ8l+mkQEAFY1Iz8tPXhXvU2qhe8WLjYyg9J1+aKjslKLQ3a02xm+uyroXBshIy50INIIs5PNJiELgWrqBfIRRYuvmTeQoAzsktz3v3WZBFWOVI7NZkjQClAF2ZFjRO3ZSs+jMNxVGz2+yYgs88zqhTr3zttPtrgznhYxFnb97ZkJ6iqn7uzfci1w6bpOa7KDLjlrbqYMJNCVRiwZ5lu18EXpztbkQfsG+vads8T33RjsuKwxZHI8DYBabJlB8xffnXHU4iHaPj7pvjanZOq1j0zk9SVdllCa4V0CrBdn9ihkMNxm1eBfdO\\\\\\\/hzGdpcuGzqVrK3NmALLW+H\\\\\\\/+2kK6P2xQI6RiEF5ePjYshNkRN8LF7zJ\\\\\\\/Diru4dvgQCGNpxHtvrD6YmoWfwg9kbRAx+MMe+urwj8CRcOOSPDOYHcsNpJXPTgtjFe8lnfOoqso4lzRq\\\\\\\/eQEgYJhP9XMAbxdUmvRD1sLR5Si0T1bHvsguIB+5enPhRiVdhMR5N130pDTveYfxSHVVnEhTViXW3xDnCaNYplrsJ0HXRW1ibZaFLK6KK1mp58grOkNuUuQPgGbi819dcaGpFUao9B88DbN1tFE2cSk9P6jARjk\\\\\\\/tfh\\\\\\\/L6vbDE+MUWOKYZpUJPeMQ\\\\\\\/T1Xt9huoIDd24ptAMAtQVb2K1fSciBaQVw5+3cJqg5NHgOX0+T8\\\\\\\/EzwDJKl0bZSvvSl1eeYPPvg7DNopYToQtsq\\\\\\\/\\\\\\\/\\\\\\\/He32rGf3yj9+fB3oRQI3wOL5wIC3YR\\\\\\\/w2fwEP5azRqve4IE00pDdZewfo00kWw3lj4J6Tm2O9GDFx0OCNgWkL+m9HfqnoclHaQgjC4YVCoWZWUIvRAg+aFEBIhdJ32JUyITkOZgYbtWEWuBWrNRvCHZVbSTGDLfPt3MSKM57vkQrFIFkjvG1MZjVaUgD7fhVsAgxKjKNZOZqTgz9KXyjtTpsEy7TB3OtqgpwvoNAeFL2d8sUF\\\\\\\/rdA1FSfV\\\\\\\/wJFxK7Vg4NGNbhgMTcHO\\\\\\\/DDmrRA6z1C0c8SP4XtzyD\\\\\\\/n3aJUx1y73HhRmIsQJ8RihspBAdJD2+R7GleW+fZCdwnddNcltxfDz5QhPoTQ7J97dEen0BcFWZjpfDf3h97G7F+s02XQ7+r9rKs2gqn6h32SlsXXZZpzd3wY2V5MPKfsQl5mkuvsx9tOHGrhuPxN4koalPPtV7mpDmb\\\\\\\/52pIilnqWZWQH\\\\\\\/v7IHeMxSkTxqRgk1rOvfx8\\\\\\\/WkULMPn7lKE43N4PyJ401n57V3hch1C9blVFz50VwOGkXbdvg2dknPv5lY3UJEDAKA+6NH0NeuDhnGotDD6H+YkQ2eDeLzK\\\\\\\/wwlbsIjwwdVPpd15\\\\\\\/slKWDj12kutrRyBDoQrEIcE0e5O8I5kW\\\\\\\/aZvbOTHGVps2OvLGBi2Uzj9NO8F1mgWJGPccuY792UQGxL4nZ1mdha+FCpMrYnRj6TguzfS\\\\\\\/UkIcTKzA9JIqgGXCTkmddTiYff9cHEtBSoDQcShRKcSriXmI70jhAiwXX1UB7kK\\\\\\\/JtS6hMcdpmZW674Ef0tRZYV8pElhtnuUOdVE1syjHw+bLZ0wZrBoZVRXrMF29BYDMeMWk9W08HM+\\\\\\\/6tqmLRul9k\\\\\\\/e3ZXn3svrXE9ghzh8RFfiQQUvEUUWDq0XdG1WvO4H63MDaCqTelo0kSRTlwadsFoyZKpsLpwJIhmn2uQQnpM6KzQn7levlOjpoZsWGceh13y0prDCu\\\\\\\/t0EcEGrMotC9ZqZleWmsUxXEVQ0gR1skXzIrVT0zfQtZE1akk4+k9urTvYW5AEm0Mmc7oIjo40JXyKjNTIGrlmPC4qRmq33vx6kGmzxE8FhKwTBhaJqA0OLxKNpvZhaAuMzv4G0gN+KoRH4xXBegLkIytG\\\\\\\/1+JWep4hHUzSEFVQfXpoq\\\\\\\/BZqOy++RDIwuWy2ghdZMsr0GtcIxI0RkdtJRXrVBK9AhaY0XGA6\\\\\\\/JNGMHKysRukwPSnQLz1Q6cmUTS+wM0c9uAOJ1urXuQvjNBpY14wJuqoBAPSTALYvsdUkvAApfXuqtXi+MOiBFt4HseRtqqgdpvs0OLuw5TVTd4mzhjYSt+3j4+4uZdgV2nPut5R+RxjThcmorCrICYLIIORl68WdUKS4yrEaMG2ygcvCBkFVXFblfrHau2ICBnj3HKVG23f9ejOzkn41wymdKauktufnLz8iUseW93Wk2m7bXk+VNhK1B8ZEbPhULekSrlVHcZZJP\\\\\\\/nENtTSzkbKY0Kp9178KjiYYpTJRaZGEEuIkyfytUJF3riiW5XndgOdPMw6QCxZapQLKh7VWhyRIAWoToD8inQg0mhKw\\\\\\\/xrCDUlIOTn6KkLkutMleLFkseo5\\\\\\\/lec+neldAfDmsQmkwMuMQwrJOeqJ5pgS3BYI\\\\\\\/Tgig1ooQe7eJw3bNeYcBJC1Nj9k5BBT2\\\\\\\/YbN1thrtRGZwESYzjLm7KT2WC0rR92VgFTIYTbYrlfD5aqpmqzC\\\\\\\/RN03PyjZZDxLCzTAisZsqN28L4qT1W8aYIJjZRVCNWhK9EYstenEp0tjXAj+F6+KieLYFsXidZWPh2fqjQRIuucYW8Bu9a9gRgC+n7RBMs97LHTHqKO9q7gCRv9JMiIsBy3CabQ0thWOTUFStpNc90FCNBBtbOpd8Or05aheDVkOpoSicAdx0PNH+YL2zvP01yO0xZEsV0i0xlGMZczWx7l5kdLq90cyf50uaX\\\\\\\/f1GwXGJFkE2WVpGJU3KRSYMxB+gYZGfyp\\\\\\\/sacapotH3Vy9f1Ng357U1eUKPQMKmFp+Eu0m0+cIH8DHTWftcp8lkEfRmaBFtwpnoz\\\\\\\/mPB6l6yMu+QTTOBDnwYb3JnvGRevkkzLIvcRZ5zBm6FXQdar7qivzKZi0KJGbI7IuDDPpwUSwG3Bac81PUBGdKVN4Cs3p4hIB5o7GD4y4xhpz1ko3pexeUQPrLpRjSPftrtGCIP2Q\\\\\\\/nU+Hc5c8IRwBCDDpcjlzJSTNNlk8U08irn3hJXJx9grIqMRK\\\\\\\/MNwB\\\\\\\/m1gUY4rolMA+Hzmq5Sd+2m15XJFZNdxWIZTIkzJEKMQ+4z11JPYHoY8l6wyRl2dpLvKElb+9bY852KmyyzaLW3\\\\\\\/V2vT60Q1wbuMHiHJ9YIXeUbT0aEigcx+Jng9bGColxA9zCUQSjZKstZfvq7QscKHM69Cq0Yar7ZiGaHNb8lSTrCcbo\\\\\\\/Oe6v8aY7tDzS0fkWghs5NvGnBo6KPZ\\\\\\\/TzH9inURNai02SGFyjJGQwJHFVR2mstioOVJIg3602h1JdRTvzD+bt0m0odfQFKIO364oLB7C1Z3s311iK+d87t+MQsnRz\\\\\\\/xOp9n0OsKb8hl+IHjZgpOusbEHQEjKjoZ42Y\\\\\\\/jyyqpOmGiJevUS63Jo7fOAgmNwcBgHgvwyBRzBlWcNXsHXUVYToYzlfMHbfJysYMEzlt2kJDa5DS6Di5cdrB6V+64WZPeIGa\\\\\\\/9umpeD7XV\\\\\\\/d1GJ0rnc7+IoVAgvD\\\\\\\/Tuto9pEO\\\\\\\/jF7vpm+Qr5TBlU3EaQi5mMg1Oy4MIesKKcogMVbtF93oTVvtjbP4OZv1OhQx1\\\\\\\/x1j09wHfeyzUUL8bPgbai95hQg42fPUuhUVQNGSageE8GAVOiX9JizhGs1Z8lQCYM6\\\\\\\/9mCokdR\\\\\\\/JtdqR4+y7m0VmFKFLAQu2N5OAmHQ5F8v+oUAduQGT73oI0iAsFurjk1y4cQFFFpFer8NLMXPy91VhUI\\\\\\\/Nqf4WUP3N9VOWNQ+byKQdhQaWufD+qNZS84tTJ4akh0fmySxuc73kTHc+JvU205XUM4I7Bz27EQiYm1HjODcrlUAntDWwW5GAjOwUhz7CWL78Q1qUG7ZgFwwTqeuE+5d8B6oc0nqogogCtgb9tvTbnakE2f0Z8v31bucG+RN+McowI9JMwmRp68pulV+hNCLtM02aS1F9pjh1Z0evdeKfWcVFjw71OM0PrBcxkEmkOQsDEc8sqgy3EBo\\\\\\\/djW62jsWPXgHI6TMcr2gmzN42xTZPe6YmHiFNn4+dq3pqnRJRW8XD07TYgOJorszUVyOX895BM32BiZeqyn4cmlP7PQ9P0iJaiKmkdi\\\\\\\/QN6n2w13nL40PWE\\\\\\\/+2Yw40MoqRAyiROL9XJB+EcPf3B+cP2TeikCs06wgMeqST3bQWuRKh7g4dolkffvAhFKQGiASap0NsRFd+qT\\\\\\\/p9rAyOCjSPjSP2vwR3V51i7fNQJdCTarMrbh+K5dPpEg4w8m7UCLrVASW78T0iyIlMLTOwM\\\\\\\/QtsLR1uJkbWoAzFaHfBl22LiD3FIc61QwGbnKjL3T88ecNdjGd5n2wNPcQyL4j\\\\\\\/UWejjgpP4rF\\\\\\\/WMfH4P+QB9kuR8SvkXqrBUSp9Qw1oE6B8pJsI8O+62ZB110NttCWgFpVA3YaN1ttOQeaRIrQq9SXmF68Izbgz\\\\\\\/EQfU9kXFwxp+CZWwRzp7CuhUv0zjlUUTjBIIJwao4AmAxi+TXezx+sjvwRoZq3+GKhXw\\\\\\\/XWsNOPUSgxezoygQ530Ys7FLKLQbAF0jyVbElNKCOFlt6lO8jexgV0RUCXlY6I745i7oLjzCiniQm7lEMx5lJUeJYO1bZizMkSYRdB5dhMjSDCUnnMXoegjkU2gU5cO+eImxNquFL0Ss1OVdEb5Pgrs+oULVvGzpPGeO0rly2wmRNwSQycDngia7Lg+qKtGpqScEM2hWmoXUv3LsWj8JLJfX3z1raeFqs0QlHKMCEHUXtcjAQnmwJaFcq0Bgz7fey2b4np68b0cQfEpHXGMk5Cqz9g5hqFX3famMZsaH2UxjgcKjEv\\\\\\\/WbH9NkFHN5b2WveIgbqvKsAY62LqpalQuEZ+NZUADC1WIcqabK7vcM71JVWhC\\\\\\\/DB8DBFz93UdKD5EulGKAqmhhNXxnCXa6oyqj7iqMDT4uQabPw5yJVdCwhoWWL0XizdT\\\\\\\/QVrFD6fxBL5LRvBASXDA9S4J21vc6F\\\\\\\/yn3sNPj6LrxPoKQYxSTA2qeu79uuNyBxCJVOuhytAmLDeT7dLXlJHq5YXjrMxu0qPC+RnnK7mkm4YYwZp7c\\\\\\\/RmNaKAlHVqq3rpp6KaPJmm8Z5vqO3PM\\\\\\\/hV9dJDz0CmaYlUjn\\\\\\\/drXuEhU59H5WyDKvSs3nfDsEjbxyvFFBgmd6e1PSoFOnpsH2Ka50eg\\\\\\\/WEHSUytaNrK4TXDB9UH29CMpJSK\\\\\\\/j60tX33k5MFuPOBenQlEP2fyiN5azJ5hUqIlBApxpnvujRXCxLXKNFNyuElXWL9xp12+TCk0FG9SksKQ3LA5gl9ttZ7Kp7mSC+9uNJJDHfO4amqDlA2wRbWctIisbWo0jBQaFsshzHhdeun+uIfKv44IeBtqwA4kGaAQFwu9KGhurZCbKghzBtCFb\\\\\\\/7eCtz1PMx5GEcRq5pb+udPbec5LfQl8ctnHqk+Myfz\\\\\\\/pfLDNoa+KRnNhWrdDtJFTJQCVYKdn6Jz3GipmAJc2trn8iU2VRXzDjPE7A2IpzmfDWWs\\\\\\\/B035ZFxApDqs01Yo+ObXoKV+pOqIu8go7Objcy398siM+M8KaS7flD5qHkiSuxpi2v9vXXhj12Q06huhWu7VmYbmH1jhO5yo1Psrf3lmOHBsU3IRvO53czBDoqFyAZS2FBzrmfdk\\\\\\\/iXNx7A3WYPf0PlEOu05pg3Jp+gZ3LPR01Ps6btVk0VOuCBHqZ\\\\\\\/vaNMtLA4K\\\\\\\/hhUfMETNQV0pQ9IVOJOiAcShYKA2fFRQ8Q6LH7I3kdx9oYv4EtniQy0Xu4NMRtr1vA8YVMTkJPxizo0djB3q4Oa4t9offWoHIMKVYElLe5lWnDufNg++bkaNUFU+g\\\\\\\/LnUjP0SSrgXe65Yd9sdiqwbNyTuP7S7GYxJcea8Q6VTF2WB7F\\\\\\\/vrCXUoYiqttcr08FY1BZpK3\\\\\\\/+gjVh0s8L\\\\\\\/Ql6S7BTwIumVDmg12Zth+77ub0cJTgch4TZ9sKJ5IlrGfQ1LY\\\\\\\/Op+30sucFMDO9z3iTkP9cKIjmNlRM83V4PvSqaPDlZ4HckpJkmFobSyt9WPuodGVJi\\\\\\\/rq+iP4A7dgO3dycqWaMgfSCCh1Fy+SvkNTAuMi0QYH0pP3c73KF9MYOwZxN0UDfIbZVxA0L82MBT0bcMTth6A2EHHo8Ne0rdY4AYXMO6H+K7oa3ra7Pep5aUmZn3OgLXp4+twGjTacqM0xbuIdL3CO5RZ08E\\\\\\\/DXJx2Kzz\\\\\\\/lh8Zbjhwg1oKrfzTrN+naoIJ6IaI96nv\\\\\\\/2QrYNsXSi+5oEvE4i\\\\\\\/DccVYkhUYuIiJDNAeKeyzp8\\\\\\\/ox\\\\\\\/q4BcDl34loSMnRrpQFzkgfaHC4i184RM3sczijpMIFnDnIiAX3+eGqlFUPaK3It2ckHIEA\\\\\\\/B\\\\\\\/\\\\\\\/eXVrX3WoA57NekFfnt54FJsZwQkE+1vq+yy5YSC\\\\\\\/HTPLn4DZ0plvpGXCDVpH8IhJuMKtdVqRtMcLXmGS5wydPNhF6bwLvqz6ZOUpEGx2txm0t9w8uX3ygrQsd9+v4wlg9rVIHT4tU16Tg60Lr26Fhex6pPFBP21n45+WtP8m2RPQPK3nsWzLdKMCGBsCbS8czXZlPpGC1ZTRjlRkiE45Dvhn9vcJB0GRaXzS+iRCPtRj0KZreODHpvekTJ+SGxeGBAgseSAqx0u\\\\\\\/TJ0wL+QjDcz3cyMfvj01mr1W5llFA\\\\\\\/unp45UHWqCM6sn6WaC2uPy2fg4FmGVzDu6C5G7QSZ9NvryaFm3mI6yxglTsQVF+qhQ0f49FxmpU1XubGN5L7yDyFpA2jJT4l08tlBrfpQMONKY\\\\\\\/IzZhyHcbLTZWPWAp\\\\\\\/+JwFBZtMQwchgFuLSvfx7eQjP6a4occaCTKE6k+ROk+fPQW35+nuzM7PWCWpaYWPZL2jwCiPHloHf\\\\\\\/cqckSKIl83GDozmYNRSSLTIogNhdB9N\\\\\\\/gb7RTvfceoS8+CkUoRkTHQ\\\\\\\/76qQ0+flQEnn4Ehi2Z9Kl9Uz3Ws6aZypumsVsjD8NLjW0t1tN6l4jp1AMus+wlQ2sd8vHKHg+J2ehNzHfF5TJ0AHF+gOzJtX68fJKRwmqvW2KOQ8Uh\\\\\\\/CaMbVi1bsSF3RM9\\\\\\\/7OoVI\\\\\\\/lBJrbbgVLlbDbvivgtaO71gkJoykphbOI0713dgYbIfwipWEOxrXHQSSiBbhlajdS5hN9TNXWxi5vus+CAaOWBMS5lCWsIdlJUZFSQC+\\\\\\\/788Q7y5woWXdQllo14uTzZ1XsEHcWh0F7wcXI0\\\\\\\/PfHp+cRuSKpoFAKSJXW7nYf22Ob3losxfJaiho18t2Et1+2jqr\\\\\\\/SNrQGlWlcPr7qeZ0B1asEvPWBL+jxZ3Fs7VGfn0M21p3mY9NWH1S5tf8z8kjDu2AwvV3D796VbEGb4bti74zmdAMY5gW3YlUkolbszFxn0Ukm5CtlYmwzjz8u6ZiKS1IpDWzQQGPfY3R1tiRa20KVaV0R80R6UnpvXdowWQu7SgvkvXJVQP2dEJODSzPWyHEGCD+UOhU596IcAPUf\\\\\\\/VFtaj3DL85lOf6IN7LGL89YiQylpTf7\\\\\\\/EOpjg9ArHKybJNwOqIDTCTgjStW4vfKbWjVebLR7mZB+dRKuwGi48iVjXZxyWR2PqVzIT61U\\\\\\\/0nOUFHFxQNkRy1oqReEhQ2f26mLKgcGPUhiVUuYJtDBlcO2t4Az6b1Hy4HZoeZHIhRzunp3KPYsH9bkHI1Kq2L9C7HiqMTvtc1s0guAy5xT2ZJ2urQmwg8FovIUJ3nTy2QAGW+8fDn3AxEHme\\\\\\\/7TIgfx6w37SCYvXfuHapYuBEKXadgkc9vmrXiDnMWPCXqxEoz8dGKYDjnsSep0Y+DZmFU2M+UnttXyKrRWKT\\\\\\\/CjrzK76hymxIIyObS7O9ZQ0S1xPkhLlOGecvtsWH6XZNRtGF3vCokQKM0Cil9Eu2F7W7lUJpiL2zxToHnbxT0pdzBAs5zU\\\\\\\/0Ia96EppsBauNA9\\\\\\\/f2lK+C+0yZTSrVt9DQbjIJDvYl8hpug6wh72nMWugbCfV2fhr0k4Hlbf01xnfnhKXtEopkFX3JtbNG\\\\\\\/CKJK10I+ozAYBE3AIlLoE4+1m9BOpU9DQ0BXVnYJSGn+T4762kIPtdfTNvZARkeA3drwVKfYS8wrD3mYVChaJ+\\\\\\\/zIiIOmTJZw8ggb3hrqPLtoKVbD64CeTiq0XXM2JEyGcUTZVdx8cfLtj7RBVy0i2EomYseQqC+M93sEfqeJ50LBazX1rANxBVo+F2\\\\\\\/Zd7x85\\\\\\\/Kv1P8lKAPLdBjwnJ78KH2AGRlAWl\\\\\\\/uxRS7P1D\\\\\\\/Ajb8k0BsN5ULg95mv5p3MZHgpgl5trW0+muOwZzj27QXgERiqLGTu6hi3DBpStE5GwPPG9JCpKcOzfvBnbtQHBDIxfscT1aOBt0rTjNOB2BJDwDer6XX6coWIv+4vsn6qSagMLKV+FXX8AL2zkQlnVW9r0Dbfcuuw0UWu8CUjG7FtjKq8pEoMODYVKj1ARFWoKBMnoyzQ6gnlrPCUSzuG9L16SkPm9qBd4\\\\\\\/0\\\\\\\/SoqWB0TT9qRvZxYk5X951Hoc3WOC6ktCFGVRoiSYWJ3IH6SDBnSTHTDw4wfpw6jHwad\\\\\\\/45A\\\\\\\/Xv0l+SdtuVkayqkOUpkKCCh+VbfleZVWWhL6M2tVZFaakbNBM6Jz5Q1PtYIoVMCPo9izOXOxZryUy37TePklz1QY88D6XUQWpEqucatJxP8iwPYnPsN8Ifpbg2iFLecZHuGSV5R\\\\\\\/6IA1wayNuPoGxLEEEvGieqR9d1fg1XfjKWpdqqZ6XpvqZLsDXf+9e8G320w2GzsUCyL2CVRnBn2o1bOzQffmdPdf03nV1YrAdyry1uFJazyvV+vLIaeyIzYo8\\\\\\\/SPGdjwPY90YzDxz45SZHp0Rs\\\\\\\/u3Ozes7YxSnpAFYQP7\\\\\\\/GsEoaUhHUd6T5LwM8iVUCGR2PziTMnN\\\\\\\/M6C8WKOeZGH8yot\\\\\\\/3yx61M6kEvV93yiDkNHCRk2qu4VWsxywIfU4tDdJVwgZlyhCLFn3kvtt60vHn3n+\\\\\\\/SJ7TIfmMYmRWX3\\\\\\\/BWNtlUTxmhI72r+x\\\\\\\/RPDpiWU+02cxXOKtvC+AzsL38Sfm\\\\\\\/jqv7tYGnBjAQNkO7jfatexon8QIiwXdTN6D1sLvI2U3BG6XlcazsYSpndGZzHwNIWAHKzYNET2vGWdahTxOMmR8+EoXYWpVc7q3LFQTDotdhZhohNpYO70x6U4Fi7pWjGHi+VpafihmUwJm6i3MrImlM7qadFfN3Yh8GATAFRQhcwEzfzKDMXq3Nlr6zIKOAOyJHz0th0Iugcko\\\\\\\/BDcPxAz57M7hgZjQAM\\\\\\\/Ju8jkEAQisjHVuDVjyHTpia\\\\\\\/DAWXDAenGrSYi0HPaKJQSXmbkCsKEMjwCv2HG5O\\\\\\\/8Hmd8M2DS\\\\\\\/RCRdS8ODy0\\\\\\\/bOyubRw7WGtUD6lqpWeXyijzfeVPd+B\\\\\\\/3SnPuZ3Piw0p74KsbJUA6GFsu6DAsR1oejrSYXElXIjZkKgIaPGm2biZK96\\\\\\\/E+Rf7DF3b5myqbD3ZPLstB8uUSXh1bwImhGFae+M\\\\\\\/sX9gO0FcdPa0cwbObWRscCgwEAuopiW8k8Ht2s7WDJBTK8skWUh99WAxTYmm4mhLUpe0JEwM5KNAkhbMI6kzB2GCO9b7tgxoqSVEE18qSNO1WiE+9nbAjS1q9uVBd+nClc16ugSMlRbtFdOrQCb8WOSTsQvY8JByzH0myxJljqcSdc6fXLblBY\\\\\\\/I8yl28KwZcz3GNhSQy+GWsauHYnt3sflAfQtygEz1qLIlV2IzQuR9D6n\\\\\\\/jAtZ5ryQrO2OQ7ACAHwLHIDdcPT\\\\\\\/7QKSQliDfaSScZZsDwmfl+7o\\\\\\\/OlJaEF+JZjfwrMrjh0KVklqBvwja7PbYSsRlxjhb9SQF2yOX18L9vPmMjAxPwmMhdwDPYuV5qL3SGWwr8ordjX2VVcGZBXztcOLU+VwMZ\\\\\\\/At7LQ0rNRwGfd5Ajg9baiSe6gOnATLh1B0sBMG1SlwJsV97MmsM15Akwt+80PVoteXqhM5qMW5GSuTaR28msp4UvSPM5D6SHIKwSxHbmSxr6cu4xtDyR57L+wvfCBStm4I0xdXYZBg1zg1XATrCP0jaj8wT3fB1mtaVVCbAgjxjl6LH++Ye5RZYaxoQRVK1u3PwK1F5Fg7wbTOGFzRGSgvs12IlVkBhU7cEsHFeMyEB4VjE2bbAm3sUCPbThoEjQ1OpGoXu3q20EkqspGxJtAkgF5xWmuRyPCDbr2FnHq0QNw3f2pO5qgcjPJKqLMT9oQzvZ8w0UA0G36LdoJA44yhZePViJeOtkw09dAcDD0mvwu4J9zv0S+6ipvMwilrEljV3cOOWdWtSPA+O8S+EMvp6W2n97JBe6M4ZdY\\\\\\\/SrMqHR7YOxoNaIxRemrnync8XE8WF0RoitrLbgf9M6+J7y2oaDhTrJcvMmikxYRKFZtc94UzT2y7MDRhadf6AsxmID789rR9dqyLX2dnZWlFPhbe3yFHdcVT0nQQu+vGDugiN2HaP5f62EQOBS9pfX6E9NGlvKypIloOKuh3LzUkZ8jItLVNMsgNwpSmfUbV5tudHsDkqIg9Cv++NBa1aesUNPDZWDSvjYvHJoh\\\\\\\/a4o9atF44gbvUKumUUBxWr3zITJtliPBqrqtKrIweRGVWa1C3b+03Kw4BmExqW5jun9g\\\\\\\/eJA8ZmDFJDHftvDzvZXS4Ho1kZbFwK+ICJeceY4U7cTrFx+njhWxjTsBCd+Im\\\\\\\/Tss3WPQxSKLLzRQlgpFPN4zxIPIOdukW0FO0Y8cql9Q4PED7QFKg8mf2otRO2YPF3dI8rdHXUkmP7EFZzMWDOEeTFln3xbm2JRzWtHaEL7kmVVQTaWvjPFpHpjcgXgG52rMksxDcME0Cc2ldqy2Vq\\\\\\\/4uu76vkClYdBh6mjK++B+9IX19A+rpS6bLTl0KfastDaJbyrPIbS0EtykXaZ7ivW\\\\\\\/\\\\\\\/CbLxx50gP64WEbRl\\\\\\\/M1N0+ctRUjJ5gJMUJKVzWhhDQtbNHMUUTS3WRTF4kndBOs\\\\\\\/LlorZuX3BqK6TOT27OQBbmFviWD9IYZtBoNe1Y9X+p\\\\\\\/1rF78RPINsx7+d+qYZNxwbFhSdm7Rzpuz5x3kPN6\\\\\\\/ojRQMRmQ4xrwBoYcnyG30ie\\\\\\\/Y5mbGHEETua2PkvHqWLJoz2kuML71LbwqIMUg+lZh5zlCx2zKVSwJBMfrqMXnDipn25sxlPEX+KdYtWna4PWjjB5UN8nxo5tBxOKqAid7PUvLEHWWDlnAqyfzCti31hliOSxnMfMqzOuBOEXMGMSr+c0UWLVxXmpszYkwBvFkUmTxhR5rYbzpFUI61Rs2c1\\\\\\\/eBhjAuiogZS9f\\\\\\\/B9FfWFDKsrbc3arjFzQZZ14o\\\\\\\/RuI4G\\\\\\\/fMIENZsJpq6hyB9nOoj1rmSyr9bsQThRX1QG1Af70I3UClGWmCJaYPlpDv0+v9kvUK9SU0TPio2svFhF57KX43KB+3T5H2pN\\\\\\\/qJTjssEmfLosYMnCmwXDPFN4O\\\\\\\/eksEQIazSMo6aZt7ibX3O7Jvf6DRLk3dY4l2couEf9FfiZR2SP5+9ex4Uj2CdcTpjHs0zftx\\\\\\\/BhJLXKlvSTOv\\\\\\\/GAkz1Z8vVQZLKxv7CKEABL1rZaX1\\\\\\\/YAKvwAhkJXdStHwwMGhyNMfuQIGYkyi9EeSz9HkPcVkW4qmvdl6tUFs4vcuSZaOL8+FdPL5x5BbgBjTZI0E7zl6qIdKtcesZO7553mDuQtgZuF\\\\\\\/aWm7e7lVYTiRSEoen6PjoSYo6wJZSJHZdoJ0M6zIltgoac\\\\\\\/DaiSpX\\\\\\\/X\\\\\\\/1rpGUc3pXS40UMv+OvgpQA0XJHgg1YUBHk3Ay3Q78NEoPb1B9DmWxuvyb1M5Jd59O0BYhwI3itffQeD4fNkeBcemP+qHJ11Xn4TdrkSyXLQQ+uo8O96avL5JWpGNxJ4eEzO6H4EMIkNE46cNXQePUY+y6JFDjj6l017JGIuxqPKNVxTDsZKJ+tfIRI04KAammy7rxi7\\\\\\\/ARViuUN813qmHWccKB0cajCyyRwWBRSuLJa2tlOFX2HHhIDGw1nDzf\\\\\\\/EK0rFgaNgAC8Jgs6gpvsOzoo9S3ewWYt9h5u2iuZ2e9RTceu3BvwzBuuxiLP7s50x3KxvK7xx83AMmUQCb26siUw1UUk2MK\\\\\\\/2FO7zV4qTl+O5tLaJK2R7HnkgRuo8GbbYoQBUt04ssgseRD9kA7rPUtPMeF\\\\\\\/ZxxFvK8OOuSlJyhQm3p8DqDHvp2RTiz1zunNh3ZDoQ8Lij74mS0Vfyoc+lns8FhMp1b3a6C1T4NAt1MediAp3Jsix63mVx9+k8M1Jd0j6b6KzHu8c3u7ydki2xdsp8MZeBp9R0yJTm9NfEhiE2y5ZHEfSAdlTFcPvo2q26ycSKDYaYWEFBhxdqUN6WgOwGldB6G8Iy7DqriqcY4AZNNGMweV\\\\\\\/B0gt699ptPy2G4Aqv+I5PCOFwtmYHDzjIDvtJGFVCXiOWeHNljB2GRsFHj7isYDQtehRPDJ+Y3Uwxk3YG\\\\\\\/0IBuEwoUjSAJAOXc\\\\\\\/ymnpWO0oE\\\\\\\/FNGghaDqYPaLbHNm7y8JMWCfN7Z\\\\\\\/A0LhAxYgW1Jk0Dm6RIcROoUsfW+o\\\\\\\/iBzGWijwhPatYl5xYaGMXRDuXHAovw8zUgODBJ1kPYdAGaAFGPu\\\\\\\/H6q6knnmycGRbqoqAuf7HSAqERp9YGcgah+UCWlr4xvkN8c8nIPj3cUk73aZKyzgi4zBGWcPKasJPtBARQ7Q+uKw7rZIHzyG3dOma16oNl3cjAkvPk9VwXOTrEXVg7FII3rNOazRFCttWVsZM1\\\\\\\/+MU7JILfR3Zzbfcg+ug65QHZ\\\\\\\/KWlrRWtkw3xyje5vpdRIbNd4xw6VSxrphyuvsMj9obG77e4N0eRN42UGGTxlujGS9gkr1w9wTqsupCdEH6knEJ5cByyibaX9vhjzVobAHZJE81IkvNhbRAKT1j3GQqysughy51PJ7iXv+BsvIY8yCQseoL1Hgr+SQsgUrHKN1E04e7UnZsqUlQ7Enn7+Syzx0IjbHJdT5ucIGM1sXhFkm7\\\\\\\/u7btvnNYMpc9gru0Rj2MwlfWvx4qfBwsDjoYFq4TaFYgXDbnd1iBgIG+7BovCfJW2fX3Ytqax6Oy8uxo4GZMii2I5fYwXj5hfMnpZUzzFM9tIKpWLrHMZfuImTR1Z\\\\\\\/y0rIcdm3etF1OAqhz6EUl4EIRFptRglfv5hXPBaX+UD2rfCQxteTazsZrEwvr6jxTB6Pl4l5qsJtKOdG9lwQzBkEFt6T9yf7S\\\\\\\/ozoeo4nwKtarsPyy4VTxr5dT6Co+tbsVV9YmJvHOvwtfWFU1Ef6ZCi9iArT150JHXB9QxCCL8ZBgK1R08jJFgliAZJaHC0Zu77lwyzWKTrGnT5rH3sjWpz1HyF\\\\\\\/kx7k\\\\\\\/CRr+GLRp8NwA2KhPlRo6NJsr2Sbal6MuRRCvatQTYzblXFH3LclZVQidMLy1W0\\\\\\\/EKGmZJrcF9aOwS0sCnmQUG5ROKN9JwEwfLYhpiGgrTtCTArTsZYeqVRIy++CpeEL7u\\\\\\\/xe7lR1hKMAyQje\\\\\\\/1fHs6FRxcDdpOqbgYHkiwFyL0L\\\\\\\/291P9hOSXgNsbhk8tFbj8JlIbb0uoB9QTeq2syxMMpJqtYzuh7jOxF8MmrI5CNcUyKQAQpDozoWeNPOFlZKuefD4z22uZM0hpWEJGhCaW1O3mzM1amVdzo7AyUdIJcroOePFZp6sb73CDb\\\\\\\/tRBiXEf5O4msKUsmudvKzejAjVHksbPKEvP\\\\\\\/IjNCdyJJcbqpNf3w1YY1g4GTnn+95OmJ+loMBD3Rt8cSAR6x412ReV6Gbt5X18hNZMPa\\\\\\\/BoPXMD2wBnFU0ndaUMRyLKd4gYTCgpTrLoyVKf1lOi4ygn+Wi7ZMNcZmE5y0Ike20srVhAYRpYkKEirNKnb1lHbvzlp7IcBd1C7QTWo2b9q8NW\\\\\\\/xpGQEkkxB1mOaqSvjMh6xPpC1+JbaTFHSNUSy0GqCvyF7CZkkE1COSxv1StkRXEauZ\\\\\\\/WguXpU4KeFReN6SzRIzhnVOg+ItatLupRTOYvAudyboAC1JalPvXsBm3eqqLy7ehKZfgXnf6pIzkpAfU75Dq+qsATwxn\\\\\\\/6IACEg0U1+t55aAssULEDNmyyDIWd6IdK5I2pr\\\\\\\/6tQgWvge4PnL0uwzZAB589fTtrYLNXPstLK3\\\\\\\/N52UbY06ZAlAk2xibS5kIi2edHL8RIGs4ba+ylkDw96SJibo588uVcU+y3DFyZRhrdieDShfVsUQfdk0kXJLcq+m+17razGWDWeZlWGAKotDhjSBt7rs3Qy2mDgtjMng1DM+moKsmCeAC8P183wvhFAp5wmWnWwxMoaQ\\\\\\\/rP93CpR3\\\\\\\/8CraXipZjW3a222a5Lr682wqk7dGq7Py1p6p89TSgmBdQ\\\\\\\/kBEko4Pcav1JuuARNnJVN+0m336NyM2ypUxjYodij\\\\\\\/ka0xjtk3vaMwweiuwsuVzTR6s6d9CHrH3vFZ6X4q+CT1GuA8NC1uJx\\\\\\\/ZSbRPj+lj8CfDT43yPCO0vehpyxrEWEE7QCNZLe7boiVbD207wc5nTruVIB2LMYaPxvmR\\\\\\\/pOHx9gRYS+w0D\\\\\\\/YORD1PxzZHpPtWzUgQNRwTspI0HsngM53AViy6DsMj9RDFjLmnqdTwaaXvvK3WWzTXxmrFyo5rcg489VmmgZ1OjXsBD9UVMZXFod8tvFXeuKiYI1YI+54LUZxdP8EaWaKFZsVqzy0CzaFTh8VA7CeQ3UnZ7fl5bnlcczj871Zrx4JykEIFbtnRq4V9PMaYKLVzRZ6vXfTlej8GnmReByXXNl27mxUdSqCbR694D4nHlAOGyIBoVJopitjKY1tUgBB7NXK2ABnn+i2LLgZEs8Xifcbc+lRd7BUnECbTkvLNj0X9pJIZOmGF193a8FMowc4DWF4\\\\\\\/GC1IM9h0BJURZay1FuQrDwIOLVDxpJW0pvDAeM3fgF4mu2t+UTNI4KQtL3+Of+QMycWClPT0J6s0Ed4a8ad94vlN0SG0+S\\\\\\\/5aNSCRGEeLV4zK0F4x8fCmdUyQfAwoNWjg8el4quNnSZA9FnBS15DKGlftnr4M\\\\\\\/AKDDLJlxH3BcBjQwQhoLp4Up4S4wS2fLn9+0PNnskGXEghW+vxhxP7+ONzhtfaU4VHTV+Mggu+Af2xqTPPVrn9HQ8+pmYOX8WpzP+5Czbo\\\\\\\/g+5iXeb0T+3kvEQt0AodYaDgV+X+iQ1QvixJhdmUaHO6DYVUH9YBSANXjqzGvndMQQ1Jt+LWD+53TUrbOk4kmHwToSIvVqU3vuFyOue\\\\\\\/Fhx+1OQLkw8sLBIkevYo3eJGmHoi0HLyq2QZc27Vmi8jYotmhM5PLfaCXhXPC4VU0rIuaNkUsrk9BcGaG8Ug+A4X6g5bD80M0SqzlDx4iO325LF1rliJnEJGdE1gZZ\\\\\\\/4Can9y4DXZjmxc4cKRsCoehf7pJVq0UVrr7AOS5USJcbmf4VgaTgeF2qFm07ZskdlXxPRAhARXlO8y0LP6ng7OAMjsiqcWj6twNZtuMC6wW4tcRXAIFQIQ8u2qfukFMUppUytCeeBXxTsqqalXVzgFNXmgfFdBd6bnoMY1+oNZ3n7Vl6Zw5Px\\\\\\\/xaOoTjkFqjzwBnGOknOEbE+cN5NWiSAeVyGELFXLUdu9rYJHB52IEXlimhPVFb4Y6GZmbT0oMWizyM0YiRjrQs1LmidjeVUV7e2hIQInCUNPPgsDucLrD02m364V3Jke9vdFGGKgW3eBPJ9S4Y2uciGJW934Wfu8pdmNoazs13QJJt6uW2NS7bDXxO2+O8ymHSomnZbaYiy+mXyRRJpAXLBA4FVKOZOVMeR7otgLxFBTZxuXwY3OHLXtvWEHPk9yd5nDiulk76LJWFiKPR3MsrpxVZmkBNZDQnHe\\\\\\\/Q75F0c\\\\\\\/dMbBmcMPxlLQxDDzUao5mLKziaLz9cIttAa0zaHw6tsway6X+ZgQrUr7RWa\\\\\\\/NktfOx78GM+\\\\\\\/a2nrX\\\\\\\/kh5zrBxHW2ztyPcDul4iToRR3hmxleV1iI9ifvLqGpe6S4cPwcOSn4QlfrpSeZocX7Z4cbhaqWaMfDdI4LyJNZ2l1GTvPXLXknIsjmwQ809O3oI7FWSGMnG5ZuiziAkfvet8BNt71G6ChXYdQt4GA1buM1cJByJWM02GNxqXdj3jVdoSLZbcUrsPN8brffw2EoM9xbHG0cgQ66hUDKS4\\\\\\\/wr\\\\\\\/wGjYi3LUBX6szi75jHg7sZRB5XpgGWYc3czmeCO5lH80JyK4bapGAwUeGHczvIaU5AjzfbCQ1n6FTOazea5rsueP3llw4+7VgXcoaw\\\\\\\/+WOckTW18Q+bgksFltqDSh56My94yiG42LsDb8Iqsm3QtxdWZ8lHf5SCSnXzEnof6GVjgGmy0tWl+BWHvgR3KgLlzGNj37\\\\\\\/v4bkcrlao18Wvr6NP1sJVG9eVGEfI+BzW156jMZTiIfRxZ\\\\\\\/1Da13\\\\\\\/QYB1ptHl0TByT0bbF9wwwjAxNEPghBTK8UsNsGQTRNkaV8KZEq8nCx47yASQpPeVLxO5nICze8u9CK\\\\\\\/02B+Yc2kGfVbaVpeknp03vf8Bwykr065T6IqT5YzfC6sdF18hENTLaoyF3VABOVNTsA44ObFMIHHqH\\\\\\\/XvwUxP2Cvf3G1Lbv7bfumuWcU8QcHIC+4UwKuZ6lDfaiMz5T1\\\\\\\/bqgsUQ+R3UsB\\\\\\\/JTmcUcDvKCaPGDZNPN8frJY+VbPinlwO\\\\\\\/pyoJojwmM13x+zqydPujy9FNc+lVBC3gnmm4R0U+pu\\\\\\\/17ZVG+DcGPFLFpgxMbflfvDnXeMVTfcVwiGc2+KBNPootCa5bJ9ZwkklIeia3hKtUVvDu\\\\\\\/1dBgWk9pC0\\\\\\\/bIDUASBcbqC7kKrgGzwxBBaIf5u6fT9YvdUAJP4k3Snc7ZoqRPyrG10lEl4pRDrRj3uc6UftlwGfAmHpYRUAEluqQPJ8iTMN3KZKPOUiD4HAZ8jDBqBek5PC7PeUEYRKGGYTzff5rl\\\\\\\/8SzFwtWwRz3nqn8OmCoPU2LI9OB5gbjtl+c\\\\\\\/oM6waPbtUiRttzusjRiZ9f5AGf0Fu0hqnEXJn\\\\\\\/csZuOcX6UkAxu09Xh4\\\\\\\/JsdMe4iTW1IEACF25iUd0RLuDyNRqB4zXHAUzWkbdNF8crltrn0+zWsf44wdISNJCA9aOVYR\\\\\\\/Im\\\\\\\/KygpLB48SBjKLWYAj+MziVyIutPA74lrSjFYT13BhUHWq79eQCdN7rMcDM5Ga1c95iNDrzpgo63VVEcWSpLnXSURyzhmZHFaedOTeCicq7Yn4Qqlpr+lHJaYbbnUg7hrnf\\\\\\\/Sqh7+YN8Mb8lef663WyYGlJloiPlnQq4jxlgSwakhW9kAOX3UwEwSL7bkeC4hQA0fJlhLV2FMWlJb7UmM7WNPbaRQ\\\\\\\/L8Wx3GZopMd52pt1D2puJNg6pYM8yGvw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022WsfknB+I8mfjKtyUe59rMg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224c7aeff4910e9ade970e5a090336673072f8960ecf057f7370468cb2681ca5b4\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022UjEHO4\\\\\\\/aEaj4Rea\\\\\\\/T3kA54E6inf\\\\\\\/RNljLM9h11IqwFMGl2u\\\\\\\/\\\\\\\/BINl0gvQw+gr9eZq0T\\\\\\\/H\\\\\\\/KsFDy34ULU3st+69wmXwozE+adApSQEWabRcgGGYEnBcf0qsznDye0g+787PIVNE3DfTNXEn0VyQ6eZqrpys61cIZrE8h+1uAfY88v2EeUjGLdQnrOwtx1sks4\\\\\\\/kU9iyQ+wpBz61pFtvGUU4baZs2JWtw0SPg0VHHPT9uifl0puIPBC8H6qcOdkNb3juXqcaD8ZAY9eFBH7C73tfp0zaomC9U3QO294VQDShWYVXaGfR\\\\\\\/\\\\\\\/EomsFmBw3\\\\\\\/6wGTpgQloBcarerLpc6ewPq3+VuLtbiLyTZm6jPpGxDvFEsGiN2\\\\\\\/oRvAl+es9vIIPvXmyMJCUhh8+Da7Hy22G+YmJsnWk1QJf9IiKqJEkjkxQKxuugactkYGBxBu78lFUHWLvRs\\\\\\\/uUenvM4DfVjUz\\\\\\\/4PzDmjlg6fjs4gPQUYlM5RIJ5WU4d7WXYqsh5PHCCA0PqzM59J06hVDZoBe3YVpJcV3enN\\\\\\\/ddnG64VI9uh9JxQBhsvR9rVFQOXBNXffqfXC+fE2PT7U8lbG5eXcq4cdQFG90OT\\\\\\\/pJrpIE5acpPjyu8IgVwTSP9HmTCtKK4tcmgMf9Uw\\\\\\\/XdVWtLCGPUnHi61UwUK8xPpteO9WE5y9rtPiayBYjdfA6SPTfMjKmkt2S\\\\\\\/iM6KGHo5m2rKn1vE6tFvmauUFMealXSZSn22r5lEIlPIw0wtu5swK6qbxcrc\\\\\\\/RCwgsKYLy4aKiujfxtS1\\\\\\\/IUh9d+IX1+t6CMDZ4A\\\\\\\/byeN9WgXhOHyQt28S7YvZciaTCrqfRNqtv5J\\\\\\\/9foPkBKrFW3XpqRvdjPaYFARkXC4Hkq9ifQNRP9gARs6P2D35\\\\\\\/6dwwy\\\\\\\/KxIBFVCLbN+S3Zw+KtedgqByousD5DtUa9OWo8UcLqWGiL8eKqBj\\\\\\\/Krvp8nmVN+02H\\\\\\\/QjQiqi3v9MgnJ+Bgq\\\\\\\/5eXxZNdyeY5386olpLAd73lE4bgFYkQ+Dq\\\\\\\/c8HNbPXp+M0L\\\\\\\/xy6OhGe1JrCHbgMgWPCgZWTbcrm+LtoQiuZet+WEHcGa7uO0BdFvBhFqxXmhTR\\\\\\\/HQEMHPG6paL1MOyYj1lxKXMH+UtJuLks77CgLLtLCaeSqHNEka5eeWMsnlt0BlEBC3eA5E4lfbXAxUobml1OcVzvsTjMkKOvhhd9pw6hPQkvqwxaZpnBIzt+NXo4aE1GlwgaKPdLviscjAzXpzkwT5Oq6tvwik3JSdn42rZDIg4wYv8cqQ+NcAzwCaSIbFnpYRSu+aut5AqydQDDbxUW5WiU0xIFmDu4e7uxtWtFocKHRuOdxsIJA48lFyrN+d+hf19TWHOgH2u\\\\\\\/uP21ngvIPIa+1JYNKxBohoGkxLvno76fAWmpHIEMw38wSPq4s8ooIvMHr4TQasvtd1lDbNNVUH55epxPfvfsV3fhLwN+bTkWkynNvS0t3kBhxcmev2EhzSBMBeLFyoRhvl7KFz4a\\\\\\\/qN34YzkkZJcoGJGuAVZsainc0kclhEztcHRS+5Q2UTe66GZRz5s\\\\\\\/2eFS42KgEgc+Yhwv4auiYuQEDWf4alD90BFK66CIjZnJgEIy2+cKK1XLbUiRunHLCOMWOpe7Rj1098L3+BEvsmoik\\\\\\\/9boiu9+NeN3YxY6zyltEAUZPgZ3n0YtiSyO5qqLLzABYRgDhxFcGPpOLJCsdGwxi8lCCNdt79stFFKbZfM+dQNcTseervSEwYbd3bAgVPu8mqaqVHVZG7ukMJmkwNtnzxgoCBRK0O8IoFBmx4+FTmv2C5fjPGT+FkoyyoGxIDATBigkfZ\\\\\\\/CdZKF6NbXs0pFDNk8irVrnUK6pm\\\\\\\/qoBi2paAP4N2q2MgAGYU9KXb6xfCXfaqmlfzf0me4aU5Bq33AlVgwc0MzdOtP4mZWkNRoDH4qkLMKt2cNrgBUExNZ7KYqv5b96qo0RnpiMB9IdgcwuxTrlV6kjcuYNFtC76mN0h1zJl7jH+0PyTbGb\\\\\\\/J+WTUmyatQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022G7x0lAw54dO9TA9tN1S6KQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221e2cfa74219c8f5b3cf7e5c6b5f1461261d25f780d7c6f0d5643fd9b23310c21\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495555 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/695" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3y83lfwtYPQG275aY7SEjxVjZgV-9-cDLW971pTfxrs" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/695" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FFzmwVE-EewnU4jEIjADtDrMw84u2jXLXYIhFNVjKwI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/2/677" title="Contingent Convergence: The Ability To Detect Convergent Genomic Evolution Is Dependent on Population Size and Migration" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/2/709" title="Horizontal Transfer and Gene Loss Shaped the Evolution of Alpha-Amylases in Bilaterians" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495555" id="node495555" data-pisa="ggg;10/2/695" data-pisa-master="ggg;g3.119.400880" data-apath="/ggg/10/2/695.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Predictive Ability for Foliar Nutritive Traits in Perennial Ryegrass</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4769-9782" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sai Krishna Arojju</span>, <span class="highwire-citation-author" data-delta="1">Mingshu Cao</span>, <span class="highwire-citation-author" data-delta="2">M. Z. Zulfi Jahufer</span>, <span class="highwire-citation-author" data-delta="3">Brent A. Barrett</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-3129-6540" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marty J. Faville</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">695-708;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400880 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495555" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sai Krishna Arojju </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch Ltd, Grasslands Research Centre, PB 11008, Palmerston North, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sai%2BKrishna%2BArojju%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arojju%20SK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASai%2BKrishna%2BArojju%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4769-9782" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sai Krishna Arojju</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sai.arojju@agresearch.co.nz" class="" data-icon-position="" data-hide-link-title="0">sai.arojju@agresearch.co.nz</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mingshu Cao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch Ltd, Grasslands Research Centre, PB 11008, Palmerston North, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mingshu%2BCao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cao%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMingshu%2BCao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">M. Z. Zulfi Jahufer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch Ltd, Grasslands Research Centre, PB 11008, Palmerston North, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BZ.%2BZulfi%2BJahufer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zulfi%20Jahufer%20MZ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM.%2BZ.%2BZulfi%2BJahufer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Brent A. Barrett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch Ltd, Grasslands Research Centre, PB 11008, Palmerston North, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brent%2BA.%2BBarrett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barrett%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrent%2BA.%2BBarrett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Marty J. Faville </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch Ltd, Grasslands Research Centre, PB 11008, Palmerston North, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marty%2BJ.%2BFaville%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Faville%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarty%2BJ.%2BFaville%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3129-6540" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marty J. Faville</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/2/695" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495555" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495555/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/2/695.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495555" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495555/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/2/695.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495555" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495555/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/2/695.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495555" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495555/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Forage nutritive value impacts animal nutrition, which underpins livestock productivity, reproduction and health. Genetic improvement for nutritive traits in perennial ryegrass has been limited, as they are typically expensive and time-consuming to measure through conventional methods. Genomic selection is appropriate for such complex and expensive traits, enabling cost-effective prediction of breeding values using genome-wide markers. The aims of the present study were to assess the potential of genomic selection for a range of nutritive traits in a multi-population training set, and to quantify contributions of family, location and family-by-location variance components to trait variation and heritability for nutritive traits. The training set consisted of a total of 517 half-sibling (half-sib) families, from five advanced breeding populations, evaluated in two distinct New Zealand grazing environments. Autumn-harvested samples were analyzed for 18 nutritive traits and maternal parents of the half-sib families were genotyped using genotyping-by-sequencing. Significant (<em>P</em> &lt; 0.05) family variance was detected for all nutritive traits and genomic heritability (<em>h<sup>2</sup><sub>g</sub></em>) was moderate to high (0.20 to 0.74). Family-by-location interactions were significant and particularly large for water soluble carbohydrate (WSC), crude fat, phosphorus (P) and crude protein. GBLUP, KGD-GBLUP and BayesCπ genomic prediction models displayed similar predictive ability, estimated by 10-fold cross validation, for all nutritive traits with values ranging from <em>r</em> = 0.16 to 0.45 using phenotypes from across two locations. High predictive ability was observed for the mineral traits sulfur (0.44), sodium (0.45) and magnesium (0.45) and the lowest values were observed for P (0.16), digestibility (0.22) and high molecular weight WSC (0.23). Predictive ability estimates for most nutritive traits were retained when marker number was reduced from one million to as few as 50,000. The moderate to high predictive abilities observed suggests implementation of genomic selection is feasible for most of the nutritive traits examined.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/heritability" class="hw-term hw-article-keyword hw-article-keyword-heritability" rel="nofollow">heritability</a></li><li class="kwd"><a href="/keyword/nutritive-traits" class="hw-term hw-article-keyword hw-article-keyword-nutritive-traits" rel="nofollow">nutritive traits</a></li><li class="kwd"><a href="/keyword/perennial-ryegrass" class="hw-term hw-article-keyword hw-article-keyword-perennial-ryegrass" rel="nofollow">perennial ryegrass</a></li><li class="kwd"><a href="/keyword/water-soluble-carbohydrates-0" class="hw-term hw-article-keyword hw-article-keyword-water-soluble-carbohydrates" rel="nofollow">water soluble carbohydrates</a></li></ul><p id="p-3">Perennial ryegrass (<em>Lolium perenne</em> L.) (2n = 2x = 14) from permanent pasture is the major feed component for ruminant production systems in temperate regions of the world. Historically, improvement of annual and seasonal dry matter yield (DMY) have been significant objectives for perennial ryegrass breeding (<a id="xref-ref-107-1" class="xref-bibr" href="#ref-107">Wilkins and Humphreys 2003</a>; <a id="xref-ref-108-1" class="xref-bibr" href="#ref-108">Williams <em>et al.</em> 2007</a>; <a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">Van Parijs <em>et al.</em> 2018</a>). Today, seasonal distribution of DMY features as the major component of economic ranking indices developed for this species in New Zealand (Forage Value Index, FVI) (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chapman <em>et al.</em> 2017</a>), Australia (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Leddin <em>et al.</em> 2018</a>) and Ireland (Pasture Profit Index, PPI) (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Mcevoy <em>et al.</em> 2011</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Mcevoy <em>et al.</em> 2014</a>). Nutritive traits in forages are also important for livestock productivity, maintenance of body weight and for supporting reproduction and health in the grazing animals (<a id="xref-ref-103-1" class="xref-bibr" href="#ref-103">Waghorn and Clark 2004</a>). Although there is existing information that demonstrates the importance of nutritive value traits and the potential economic returns from trait improvement, the overall breeding effort for nutritive traits in ryegrass has received considerably less attention than for DMY (<a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Smith <em>et al.</em> 1997</a>). Increased breeding effort for nutritive traits, with validated outcomes for animal productivity, would provide enhanced on-farm value to farmers (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Jafari <em>et al.</em> 2003a</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Chapman <em>et al.</em> 2017</a>).</p><p id="p-4">Compared to other forage grass species, perennial ryegrass is regarded as having relatively high nutritive value, providing a cost effective, nutrient rich feed for ruminant livestock (<a id="xref-ref-106-1" class="xref-bibr" href="#ref-106">Wilkins 1991</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baert and Muylle 2016</a>). Breeding for improved nutritive value in this species has focused principally on higher <em>in vitro</em> dry matter (DM) digestibility to enhance energy availability and voluntary intake from grazed pasture (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Jung and Allen 1995</a>). This is a key selection criterion in many ryegrass breeding schemes (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Casler and Vogel 1999</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Easton <em>et al.</em> 2002</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Muylle <em>et al.</em> 2013</a>), particularly in Europe, where <a id="xref-ref-107-2" class="xref-bibr" href="#ref-107">Wilkins and Humphreys (2003)</a> reported genetic improvement of approximately 10g kg<sup>-1</sup> per decade for DM digestibility. Breeding to increase water-soluble carbohydrate (WSC) content in ryegrass herbage, one of few reported studies of successful breeding for a nutritive trait in perennial ryegrass (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Humphreys 1989a</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Jones and Roberts 1991</a>; <a id="xref-ref-96-2" class="xref-bibr" href="#ref-96">Smith <em>et al.</em> 1997</a>), has been a major contributor to genetic improvement of digestibility (<a id="xref-ref-107-3" class="xref-bibr" href="#ref-107">Wilkins and Humphreys 2003</a>; <a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">Muylle <em>et al.</em> 2013</a>). More recently, there has been increased emphasis on addressing digestibility through the improvement of fiber degradability <em>per se</em>, by targeting changes in the biochemical composition of the cell wall (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Faville <em>et al.</em> 2010</a>; <a id="xref-ref-100-2" class="xref-bibr" href="#ref-100">Van Parijs <em>et al.</em> 2018</a>).</p><p id="p-5">Minerals and trace elements are essential elements for plant growth and are critical to various biological functions of the plant. In forages, these macro- and micronutrients are also important components of nutritive quality, critical for maintaining livestock health (<a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">Waghorn 2007</a>). For example, metabolic disorders can be caused or contributed to by mineral imbalances in the diet, such as hypomagnesaemia (grass tetany) which is caused by insufficient magnesium and calcium in the diet. Earlier studies have identified genetic variation among families (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Easton <em>et al.</em> 1997</a>; <a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Smith <em>et al.</em> 1999</a>) or genotypic variation among cultivars (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Crush <em>et al.</em> 2018a</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Crush <em>et al.</em> 2018b</a>) for micro- and macronutrients, indicating that breeding for mineral content is a realistic opportunity.</p><p id="p-6">The reduced emphasis on breeding for nutritive traits in forages is affected by a number of factors, including a lack of consensus on specific breeding targets (<a id="xref-ref-105-1" class="xref-bibr" href="#ref-105">Wheeler and Corbett 1989</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chapman <em>et al.</em> 2015</a>), ambiguous evidence for the impact of specific nutritive traits on animal production outcomes (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Easton <em>et al.</em> 2002</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Edwards <em>et al.</em> 2007</a>; <a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Mcevoy <em>et al.</em> 2011</a>), the confounding influence of environment and genotype x environment (G x E) interactions, and the significant additional resources needed in a breeding program to undertake nutritive trait measurements in large panels of selection candidates (<a id="xref-ref-96-3" class="xref-bibr" href="#ref-96">Smith <em>et al.</em> 1997</a>).</p><p id="p-7">Genomic selection (GS), where breeding value for a trait may be cost-effectively predicted for selection candidates using genome-wide markers, was initially proposed for animal breeding by <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Meuwissen <em>et al.</em> (2001)</a>. In GS, a training population combining phenotypic and genotypic information is used to develop a model that can subsequently be used to predict genomic estimated breeding values (GEBVs) for individuals in a test or selection population that have been genotyped only. In essence, GS replaces the need to phenotype the target trait in selection candidates for one or more cycles of recurrent selection. GS has been demonstrated in dairy cattle breeding, where the rate of genetic gain was doubled by reducing generation interval from 7 to 2.5 years or from 4 to 2.5 years, depending upon selection strategy (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">García-Ruiz <em>et al.</em> 2016</a>). Over the last decade the declining cost of genotyping single nucleotide polymorphisms (SNPs), largely through reduced representation sequencing approaches such as genotyping-by-sequencing (GBS) (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Elshire <em>et al.</em> 2011</a>), has made this tool feasible for plant breeding. GS is now being applied in major crop species, including wheat (<a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Rutkoski <em>et al.</em> 2011</a>; <a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Poland <em>et al.</em> 2012</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Lopez-Cruz <em>et al.</em> 2015</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Hayes <em>et al.</em> 2017</a>), maize (<a id="xref-ref-109-1" class="xref-bibr" href="#ref-109">Zhao <em>et al.</em> 2012</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Fristche-Neto <em>et al.</em> 2018</a>) and barley (<a id="xref-ref-110-1" class="xref-bibr" href="#ref-110">Zhong <em>et al.</em> 2009</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Lorenz <em>et al.</em> 2012</a>) and is under adoption in forage species, including perennial ryegrass (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Fè <em>et al.</em> 2016</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Grinberg <em>et al.</em> 2016</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Byrne <em>et al.</em> 2017</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Arojju <em>et al.</em> 2018</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Faville <em>et al.</em> 2018</a>; <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Pembleton <em>et al.</em> 2018</a>), and alfalfa (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Annicchiarico <em>et al.</em> 2015</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Li <em>et al.</em> 2015</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Biazzi <em>et al.</em> 2017</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Jia <em>et al.</em> 2018</a>), principally in terms of assessing the influence of training set size and composition, trait characteristics and genotyping approaches on predictive ability. In perennial ryegrass, so far only two studies have evaluated the possibility of using genome-wide markers to predict GEBVs for nutritive quality traits. <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Grinberg <em>et al.</em> (2016)</a> reported predictive ability of 0.45 and 0.32 for WSC and N and <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Fè <em>et al.</em> (2016)</a> reported predictive ability for NDF (0.68) and HMW WSC (0.45). A comprehensive study on a range of nutritive traits is still lacking.</p><p id="p-8">GS can accelerate genetic gain particularly for complex traits, which are controlled by many genes with small effects and for traits which are difficult to measure and expensive (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Heslot <em>et al.</em> 2015</a>). GS is therefore a very attractive tool for nutritive traits, given the barriers, described above, to routine integration of nutritive traits into forage breeding programs. The success of GS primarily depends on predictive ability, which is influenced by trait heritability (<em>h<sup>2</sup><sub>n</sub></em>), training population size, marker density, extent of linkage disequilibrium (LD) and relatedness between training and test population (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Daetwyler <em>et al.</em> 2013</a>). While the heritability of a trait and the extent of LD in a training population cannot be easily optimized, the density of markers and the size and composition of the training population are two factors that can be controlled. Several methods have been developed for genomic prediction and can be broadly classified as whole-genome regression methods (discussed by <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">De Los Campos <em>et al.</em> (2013)</a>) or machine learning methods (outlined by <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">González-Camacho <em>et al.</em> (2018)</a>). Based on simulation and empirical results, <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Daetwyler <em>et al.</em> (2013)</a> concluded that genomic best linear unbiased predictor (GBLUP) and Bayesian variable selection methods (BayesB and BayesCπ) were the benchmark for genomic prediction, as these methods are appropriate for a range of genetic architectures, from traits which are controlled by many genes with small effects (infinitesimal model) to traits with large SNP effects (variable selection model).</p><p id="p-9">The principle aim of the current study was to assess genomic predictive ability for 18 nutritive quality traits, measured in a large multi-population training set in two key New Zealand grazing environments, and to investigate the impact of marker density and of genomic prediction models with different prior assumptions regarding the distribution of SNP effects. The study also provided an opportunity to assess the magnitude of genetic variation and to estimate heritability for a large range of nutritive traits under New Zealand grazing environments.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant material and experimental design</h3><p id="p-10">The half-sibling (half-sib) families used in this study were derived from five different advanced breeding populations (Pop I – Pop V) which are part of the Grassland Innovation Ltd breeding program. From each population, 102 to 117 plants that tested positive for endophyte infection (<em>Epichloё festucae</em> var <em>lolli</em>) by immunoblotting (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Hahn <em>et al.</em> 2003</a>), were polycrossed in isolation during spring 2012 in Palmerston North, New Zealand (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Faville <em>et al.</em> 2018</a>). Polycrosses were performed separately for each population, without admixing, and seeds from the maternal parents were harvested and cleaned. In total 543 half-sib families were harvested for seed, however only 517 families had sufficient seed (≥ 3.6g) for sowing field trials. Further details regarding population development are presented in Table S1.</p><p id="p-11">A total of six trials were sown for measuring DMY (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Faville <em>et al.</em> 2018</a>), of which two were used for the current study. These were trials established at Lincoln (Canterbury region, southern New Zealand, 43.38°S 172.62°E; Wakanui silt loam) and Aorangi (Manawatu region, central New Zealand, 40.34°S 175.46°E; Kairanga sandy loam), during the autumn of 2013. The experimental design at each site was row-column with three replicates. Within each trial, populations were blocked, and families randomized in three replicates within these blocks. Multiple repeated checks were also randomly allocated within and across the replicated blocks. Half-sib families were evaluated as a 1m row of plants (referred to from now as plots), by sowing 0.2 g of seed (which is equivalent to 14 kg ha<sup>-1</sup>, if a sward was sown at 7 rows m<sup>-1</sup>). Space between the plots was 25-30 cm and 30-50 cm separated the ends of neighboring plots. Nitrogen and phosphate fertilizer was applied at the rate of 15-30 kg N ha<sup>-1</sup> and 8.8 kg P ha<sup>-1</sup>, in late autumn each year (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Faville <em>et al.</em> 2018</a>). Climate data regarding rain fall, sunshine hours and air temperature of the two locations during the trial period was is provided as Figure S1.</p></div><div id="sec-3" class="subsection"><h3>Phenotypic measurements</h3><p id="p-12">Plot harvests were undertaken at Lincoln starting April 14, 2014 (24 days post-defoliation by sheep grazing) and at Aorangi starting April 29, 2014 (25 days post-defoliation), during the southern hemisphere autumn. At each site a single harvest was undertaken over three days, between 10:30 <span class="sc">am</span> and 3:00 <span class="sc">pm</span> on each day to minimize the influence of diurnal variation on levels of measured constituents. Split harvesting of populations or replicate blocks over two days was avoided. Plots were cut to a height of approximately 5 cm, above the pseudostem, meaning that only leaf lamina material was harvested. Harvested foliage was placed into micro-perforated plastic bread bags and immediately snap frozen in liquid nitrogen. Samples were subsequently maintained at ca. -80° on frozen CO<sub>2</sub> to preserve labile components and then freeze-dried at one of two commercial facilities - Genesis Bio-Laboratory Ltd (Christchurch, New Zealand) or Horowhenua Freeze-Dry (Levin, New Zealand). Freeze-dried samples were milled to powder through a 1mm sieve and thoroughly mixed to homogenize the sample. Sub-samples were weighed out and transferred to Hill Laboratories (Hamilton, New Zealand) for near-infrared spectroscopy (NIRS) and minerals analysis and to AgResearch (Palmerston North, New Zealand) for analysis of water‐soluble carbohydrate (WSC). A total of 3082 samples (n = 1476 from Lincoln and n = 1606 from Manawatu) were provided for analysis. Hill Laboratories provided NIRS data for a range of nutritional traits, as outlined in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. Data for mineral concentrations (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) were based on inductively coupled plasma-optical emission (ICP-OES) analysis of plant material digested with nitric acid: hydrogen peroxide (2:1). Grass tetany ratio was calculated as [K/(Mg + Ca)] using the data provided for the individual minerals. WSC was extracted and quantified as described by <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Hunt <em>et al.</em> (2005)</a>. Briefly, 25 mg of milled leaf material was extracted twice with 1mL of 80% ethanol (low‐molecular‐weight fraction; LMW WSC) and then twice with 1 mL water (high‐molecular‐weight fraction; HMW WSC), for 30 min at 65°. Extracts were centrifuged, and supernatants of the respective fractions were analyzed using anthrone as a colorimetric reagent (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Jermyn 1956</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495765/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495765/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Trait family (σ<sup>2</sup><sub>g</sub>), family-by-location interaction (σ<sup>2</sup><sub>gl</sub>) and residual error (σ<sup>2</sup><sub>ε</sub>) variance components and their associated standard errors (SE), repeatability (<em>R</em>) and genomic heritability (<em>h<sup>2</sup><sub>g</sub></em>), estimated for the range of nutritive traits, among 517 half-sib families of perennial ryegrass evaluated across the two locations (Lincoln and Aorangi). All σ<sup>2</sup><sub>g</sub> for nutritive traits were significant (<em>P</em> &lt; 0.05).</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Statistical models and variance components</h3><p id="p-14">Data analyses were performed across the five populations, for individual locations and across the two locations, using the restricted maximum likelihood (REML) method, by fitting a linear mixed model in GenStat (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Payne <em>et al.</em> 2009</a>). Analyses were also performed on the five populations individually, by fitting linear mixed models in DeltaGen (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Jahufer and Luo 2018</a>). Family, family-by-location interaction, replicates, rows and columns were considered as random effects, whereas location, population and repeated checks were considered as fixed effects. Three different mixed linear models were used: (i) Model 1, to estimate family variance components, pooling all five populations, all 517 families together, within individual locations; (ii) Model 2, for estimating family variance components and interactions of family and location, pooling all five populations, across locations; and (iii) Model 3, for estimating family variance and family-by-location interactions, among half-sib families within individual populations, across locations.</p><p id="p-15">Model 1: Mixed model for individual locations.<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span><span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-2.gif"/></noscript></span></span> is the phenotypic value measured on half-sib family i in row j and column k of replicate l nested within population n, and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-3.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-4.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-5.gif"/></noscript></span></span> and p are half-sib families, rows, columns, replicates and populations respectively. Where, μ is the overall mean; <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-6.gif"/></noscript></span></span> is the random effect of half-sib family i, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-7.gif"/></noscript></span></span> is the fixed effect of population n; <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-8.gif"/></noscript></span></span> is the random effect of replicate l in population n, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-9.gif"/></noscript></span></span> is the random effect of row j within replicate l of population <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-10.gif"/></noscript></span></span> is the random effect of column k within replicate l of population n, <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-11.gif"/></noscript></span></span> is the residual effect of half-sib family i in row r and column c of replicate b of population n, <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-12.gif"/></noscript></span></span>.</p><p id="p-16">Model 2: Mixed model for across locations.<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-13.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span><span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-14.gif"/></noscript></span></span> is the phenotypic value measured on half-sib family i in row j and column k of replicate l nested in location m within population n, and <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-15.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-16.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-17.gif"/></noscript></span></span> and p are half-sib families, rows, columns, replicates, locations and populations respectively. In the equation, μ is the overall mean; <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-18.gif"/></noscript></span></span> is the random effect of half-sib family <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-19.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-20.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-21.gif"/></noscript></span></span> is the fixed effect of location m; <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-22.gif"/></noscript></span></span> is the random effect of interaction between half-sib family i and location m, <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-23.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-24.gif"/></noscript></span></span> is the fixed effect of population n; <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-25.gif"/></noscript></span></span> is the random effect of replicate l within location m in population <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-26.gif"/></noscript></span></span> is the random effect of row j within replicate l in location m of population n, <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-27.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-28.gif"/></noscript></span></span> is the random effect of column k within replicate l in location m of population n, <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-29.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-30.gif"/></noscript></span></span> is the residual effect of half-sib family i in row r and column c of replicate b in location m of population n, <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-31.gif"/></noscript></span></span>.</p><p id="p-17">Model 3: Mixed model for individual populations.<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-32.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span><span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-33.gif"/></noscript></span></span> is the phenotypic value measured on half-sib family i in row j and column k of replicate l nested in location m. In the equation, μ is the overall mean; <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-34.gif"/></noscript></span></span> is the random effect of half-sib family i, <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-35.gif"/></noscript></span></span> is the fixed effect of location m; <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-36.gif"/></noscript></span></span> is the random effect of interaction between half-sib family i and location m, <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-37.gif"/></noscript></span></span> is the fixed effect of population n; <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-38.gif"/></noscript></span></span> is the random effect of replicate l within location m, <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-39.gif"/></noscript></span></span>; <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-40.gif"/></noscript></span></span> is the random effect of row j within replicate l in location <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-41.gif"/></noscript></span></span>; <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-42.gif"/></noscript></span></span> is the random effect of column k within replicate l in location m, <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-43.gif"/></noscript></span></span>; <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-44.gif"/></noscript></span></span> is the residual effect of half-sib family i in row r and column c of replicate b in location m, <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-45.gif"/></noscript></span></span>.</p><p id="p-18">The variance components estimated based on the mixed model analysis were used to calculate repeatability (Model 2) (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Falconer 1960</a>) and narrow sense heritability (Model 3) for each trait. Repeatability was based on family variance estimated across five populations, whereas narrow-sense heritability is based on additive genetic variance among half-sib families within each population. Repeatability and narrow sense heritability, on a family mean basis, were estimated using the equation:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-46.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>Where, R and <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-47.gif"/></noscript></span></span> are repeatability and narrow-sense heritability. For repeatability, <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-48.gif"/></noscript></span></span> was the family variance among all the 517 half-sib families. In the estimation of narrow-sense heritability, <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-49.gif"/></noscript></span></span> was the estimated additive genetic variation among half-sib families within a specific population, <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-50.gif"/></noscript></span></span> is the variance associated with family-by-location interaction and <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-51.gif"/></noscript></span></span> is the variance of residuals.</p></div><div id="sec-5" class="subsection"><h3>Genotypic and phenotypic correlation</h3><p id="p-19">The genotypic correlation among traits was estimated as proposed by <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Falconer (1960)</a>. Multivariate analysis of variance (MANOVA) was performed in DeltaGen (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Jahufer and Luo 2018</a>), using the multi-location trait data from half-sib families, to estimate variance and covariance among traits:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-52.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>Where, <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-53.gif"/></noscript></span></span> is the genotypic covariance between trait x and y; <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-54.gif"/></noscript></span></span> is the variance associated with trait x, and <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-55.gif"/></noscript></span></span> is the variance associated with trait y. Phenotypic correlation was performed in DeltaGen (<a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Jahufer and Luo 2018</a>) using the best linear unbiased predictors (BLUPS) estimated based on Model 2.</p></div><div id="sec-6" class="subsection"><h3>Genotyping and genomic heritability</h3><p id="p-20">All maternal parents of the 517 half-sib families were genotyped using a GBS approach described in <a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Faville <em>et al.</em> (2018)</a>, following the protocol proposed by <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Elshire <em>et al.</em> (2011)</a>. Briefly, a reference ryegrass genome assembly was constructed using scaffolds from a published ryegrass assembly (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Byrne <em>et al.</em> 2015</a>). Scaffolds were aligned to the barley genome using Lastz version 7.0.1 (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Harris 2007</a>) from Geneious 8 (<a href="https://www.geneious.com/">https://www.geneious.com/</a>, (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Kearse <em>et al.</em> 2012</a>)) with default parameters. Demultiplexing of sequencing reads was performed using the TASSEL 5.0 GBS pipeline (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Glaubitz <em>et al.</em> 2014</a>) and initial quality control was based on read count statistics. The quality GBS tags were aligned to the reference genome using Bowtie2 (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Langdon 2015</a>). Genotype calling was performed using the TASSEL GBS pipeline to obtain 1,093,464 SNPs, after filtering for maximum missing SNPs per site (50%), minor allele frequency (&gt; 0.05) and read depth (&gt; 1) using VCF tools (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Danecek <em>et al.</em> 2011</a>). Genotyped 1,093,464 SNPs were exported and filtered for Hardy-Weinberg disequilibrium (HWdiseq &gt; -0.05). The resulting 1,023,011 SNPs, with a mean read depth of 2.98, were used to compute a genomic relationship matrix (GRM) based on the kinship using depth adjustment (KGD) protocol proposed described by <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Dodds <em>et al.</em> (2015)</a> and implemented by <a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">Faville <em>et al.</em> (2018)</a>. The KGD method enables the unbiased estimation of relatedness using low depth sequence data. In the KGD approach, the data are considered to be a set of allele reads rather than genotypes <em>per se</em>. In addition, the elements of the GRM are calculated using only those SNPs which have reads in both individuals. This allows the GRM to be calculated without needing to impute missing values. The KGD matrix was used for genomic predictive modeling. The 1,093,464 genotyped SNPs were also mean imputed using A.mat function in rrBLUP package (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Endelman 2011</a>) and the GRM was calculated using these imputed SNPs based on method proposed by <a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">Vanraden (2008)</a>.</p><p id="p-21">Genomic heritability (<em>h<sup>2</sup><sub>g</sub></em>) was calculated using Equation 4, based on variance components estimated using the mixed model proposed in Equation 2. In the model, the KGD matrix was fitted as variance-covariance among genotypes (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">De Los Campos <em>et al.</em> 2015</a>) and the genetic variance was calculated as proportion of variance explained by regressing markers on phenotypes. The model was fitted in ASreml-R (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Butler <em>et al.</em> 2009</a>).</p></div><div id="sec-7" class="subsection"><h3>Genomic prediction modeling</h3><p id="p-22">Three whole-genome regression methods, with two different prior assumptions regarding the distribution of marker effects, were used for generating GEBVs. The first method was a univariate linear mixed model, called GBLUP (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Goddard <em>et al.</em> 2011</a>) in which markers effects were assumed to have equal variance. The linear model can be expressed follows:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-56.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>Where y is the vector of BLUP values of the trait, β is the vector of grand mean, Z is the design matrix associated with random marker effects μ, with <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-57.gif"/></noscript></span></span>, in which G is the additive genetic relationship matrix, and <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-58.gif"/></noscript></span></span>, in which I is the identity matrix. The G matrix was calculated based on the method proposed by <a id="xref-ref-101-2" class="xref-bibr" href="#ref-101">Vanraden (2008)</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Endelman and Jannink (2012)</a> using A.mat function in rrBLUP package (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Endelman 2011</a>).</p><p id="p-23">The second method is a variant of GBLUP method with KGD matrix as G in the linear mixed model. The GBLUP and KGD-GBLUP models were fitted using the rrBLUP package (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Endelman 2011</a>), implemented through R programming language (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">R Core Team 2017</a>). The equations used to calculate the KGD <em>G</em> matrix are detailed in <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Dodds <em>et al.</em> (2015)</a>.</p><p id="p-24">The third method was BayesCπ (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Habier <em>et al.</em> 2011</a>), in which markers effects can depart from normality, that is, large variances are allowed for markers with larger effects.</p><p id="p-25">The model is expressed as follows:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-59.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>Where y is the vector of BLUP values of the trait, β is the vector of grand mean, k is the number of makers, <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-60.gif"/></noscript></span></span> is the vector of genotypes at marker <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-61.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-62.gif"/></noscript></span></span> is the additive effect of the marker, and ε is the vector of residual effects with a normal distribution <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-63.gif"/></noscript></span></span>. The BayesCπ model was implemented through R programming using the BGLR package (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Pérez and De Los Campos 2014</a>), with the number of burn-ins set to 2,000, total number of iterations set to 10,000, and other parameters set to default (<a id="xref-ref-86-2" class="xref-bibr" href="#ref-86">Pérez and De Los Campos 2014</a>).</p><p id="p-26">The predictive ability of the models based on data from the composite training population was assessed by a ten-fold cross validation approach. For each cross validation, randomized data were divided into ten equal parts, of which nine parts (training set) were used to train the model and to predict GEBVs in the remaining one part of the data (test set). Randomization of the complete data set was repeated five times and the mean of the five iterations was reported as the predictive ability of the model (<a id="xref-ref-33-7" class="xref-bibr" href="#ref-33">Faville <em>et al.</em> 2018</a>).</p></div><div id="sec-8" class="subsection"><h3>Evaluating predictions in individual populations</h3><p id="p-27">As the overall training population is a composite of 517 individuals and their corresponding half-sib families from five discrete breeding populations, the predictive ability of the prediction models was also assessed within each individual population using KGD-GBLUP. A random 50% of individuals was selected from within each population (Pop I – Pop V; total = 255 individuals) as a training set in order to represent each population equally. Using this set of 255 individuals to train the model, GEBVs were then predicted in the remaining 50% of Pop I and the mean correlation of 500 iterations was considered as the predictive ability for this population. This approach was likewise extended to each of the other four populations.</p></div><div id="sec-9" class="subsection"><h3>Optimizing marker density</h3><p id="p-28">To evaluate the minimum number of markers needed to achieve maximum predictive ability for each nutritive trait, a random set of markers ranging from 1,093,464 (100%, unfiltered) to 1,093 (0.1%) in 10 steps were obtained from the training population. Using each set of randomly selected markers, a G matrix was computed based on the method proposed by <a id="xref-ref-101-3" class="xref-bibr" href="#ref-101">Vanraden (2008)</a> using the rrBLUP package (<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Endelman 2011</a>). Considering the computational load, KGD method was not extended to randomly selected markers, to construct G matrix. <a id="xref-ref-33-8" class="xref-bibr" href="#ref-33">Faville <em>et al.</em> (2018)</a> reported broadly similar predictive ability for DMY in this training population, when G matrices based on <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Dodds <em>et al.</em> (2015)</a> and <a id="xref-ref-101-4" class="xref-bibr" href="#ref-101">Vanraden (2008)</a> were compared. The G matrix was used in a GBLUP model to estimate predictive ability for each randomly chosen marker set. The predictive ability was assessed via Monte-Carlo cross validations with 500 iterations, where 80% of the data were used to train the model (training set) and 20% to predict the GEBVs (test set).</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-29">Supplementary material file contains Tables S1 - S11 and Figure S1 generated in the current study to draw conclusions. File S1 contains genome-wide SNP data, File S2 is the genomic relationship matrix calculated using the method proposed by <a id="xref-ref-101-5" class="xref-bibr" href="#ref-101">VanRaden (2008)</a>, File S3 is the KGD relationship matrix calculated using method proposed by <a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Dodds <em>et al.</em> (2015)</a> and File S4 contains phenotypic data used for the analysis. Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.10074323">https://doi.org/10.25387/g3.10074323</a>.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Variance components, repeatability, and genomic heritability</h3><p id="p-30">There was significant (<em>P</em> &lt; 0.05) variance among 517 half-sib families from five populations for all traits, based on mean performance across the two locations, Lincoln and Aorangi (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>, Table S2, S3 and S4). There were also significant (<em>P</em> &lt; 0.05) family-by-location interactions for all the traits, indicating a relative change in ranking among the 517 half-sib families between the two locations. There was a high genotypic correlation (<em>r</em> = 0.93) between <em>R</em> and <em>h<sup>2</sup><sub>g</sub></em> in the across-location dataset and these ranged from a low of 0.26 (<em>R</em>) and 0.22 (<em>h<sup>2</sup><sub>g</sub></em>) for traits N and P to a high of 0.75 (<em>R</em>) and 0.74 (<em>h<sup>2</sup><sub>g</sub></em>) for Na (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>) across the two locations. Genotypic correlation between <em>R</em> and <em>h<sup>2</sup><sub>g</sub></em> was slightly lower in Aorangi (<em>r</em> = 0.85) compared with Lincoln (<em>r</em> = 0.93). Because of the high correlation between <em>R</em> and <em>h<sup>2</sup><sub>g</sub></em> and because <em>h<sup>2</sup><sub>g</sub></em> captures marker-based additive variance, from here on results for <em>h<sup>2</sup><sub>g</sub></em> only are reported and discussed. Overall, <em>h<sup>2</sup><sub>g</sub></em> estimated within a location was substantially higher at the Aorangi site than Lincoln (mean of all traits <em>h<sup>2</sup><sub>g</sub></em> = 0.62 and 0.43, respectively) (Table S3 and S4), with values from the across-location analysis (<em>h<sup>2</sup><sub>g</sub></em> = 0.42) lying between those of Lincoln and Aorangi. Traits with low <em>h<sup>2</sup><sub>g</sub></em> tended to have relatively large family-by-location interactions, whereas those with high <em>h<sup>2</sup><sub>g</sub></em> had a low family-by-location interactions (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). Variance component analysis within the two locations (Lincoln and Aorangi) indicated significant (<em>P</em> &lt; 0.05) family variance for all 18 nutritive traits. Differences in family variance were observed for the same trait among the five populations in the across location dataset (Table S5-S9). For example, family variance was non-significant (<em>P</em> &gt; 0.05) for ADF, NDF and DOMD in Pop I &amp; II, but was significant for these traits in Pop III – V (Table S5-S9). Similar observations can be made for all of the analyzed traits, with no population showing significant (<em>P</em> &lt; 0.05) family variance component for all 18 traits. Among the five populations, Pop I had significant family variance for only 42% of traits (8 traits out of 19) while for Pop V that number was 84%, with the remaining populations intermediate to these at 58–68% (Table S5-S9).</p></div><div id="sec-13" class="subsection"><h3>Correlation among traits</h3><p id="p-31">Genotypic and phenotypic correlation coefficients for all nutritive quality traits are shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Tables 2</a> and S10, respectively. Strong, positive genotypic correlation was observed between fiber measures ADF and NDF and these in turn were negatively correlated with energy traits including ME, DOMD and WSC (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Tables 2</a> and Table S10). A positive genotypic correlation was estimated for both LMW WSC and total WSC with DOMD, however, a weak positive correlation was found between HMW WSC and DOMD. A strong negative genotypic correlation was observed for both ADF and NDF with both LMW WSC and total WSC. A moderate genotypic correlation was observed between fiber traits (ADF and NDF) and minerals traits including K, Mg and Mn (positive), P and Ca (negative).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495723/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495723/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Genotypic correlations among a range of nutritive quality traits measured from 517 half-sib families, estimated using data from across two locations (Lincoln and Aorangi)</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Genotyping-by-sequencing</h3><p id="p-32">The results of GBS were described in detail by <a id="xref-ref-33-9" class="xref-bibr" href="#ref-33">Faville <em>et al.</em> (2018)</a>. Population structure was analyzed using multi-dimensional scaling based on genomic relationship matrix (see <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> in <a id="xref-ref-33-10" class="xref-bibr" href="#ref-33">Faville <em>et al.</em> (2018)</a>), revealing structure describing the five individual populations used in the training set and, with two exceptions, no structure within each population. The exceptions were the presence of eight apparently outlying individuals associated with Pop V and six with Pop II; these were confirmed as being true to their respective populations (<a id="xref-ref-33-11" class="xref-bibr" href="#ref-33">Faville <em>et al.</em> 2018</a>) and so retained for genomic prediction analyses. The percentage of missing SNPs in each population were between 22–24%, with no particular population skewed toward higher missing rate. The missing SNPs were further investigated to see if they were population specific and found that it was not the case in the current study.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/695/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predictive ability (Pearson correlation coefficient between observed and predicted values) for nutritive traits and their associated standard deviation, assessed using three genomic prediction models (BayesC&#x3C0;, KGD-GBLUP and GBLUP), based on adjusted means (BLUP&#x2019;s) measured among five populations across two locations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-718683525" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Predictive ability (Pearson correlation coefficient between observed and predicted values) for nutritive traits and their associated standard deviation, assessed using three genomic prediction models (BayesC&#x3C0;, KGD-GBLUP and GBLUP), based on adjusted means (BLUP&#x2019;s) measured among five populations across two locations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/695/F1.medium.gif" width="440" height="226"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/2/695/F1.medium.gif" width="440" height="226"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/695/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/695/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495684" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-33" class="first-child">Predictive ability (Pearson correlation coefficient between observed and predicted values) for nutritive traits and their associated standard deviation, assessed using three genomic prediction models (BayesCπ, KGD-GBLUP and GBLUP), based on adjusted means (BLUP’s) measured among five populations across two locations.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Predictive ability for nutritive traits</h3><p id="p-34">Predictive ability for all nutritive traits was evaluated using GBLUP, KGD-GBLUP and BayesCπ genomic prediction models, and the results are summarized in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> as the Pearson correlation coefficient between observed (adjusted means) and predicted values. There were no significant differences (<em>P</em> &gt; 0.05) in terms of predictive ability between GBLUP, KGD-GBLUP and BayesCπ across all nutritive traits (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). Although slight differences can be noted from the <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>, no single statistical approach consistently gave higher predictive ability for all nutritive traits. Because the results from the three models were largely indistinguishable, from here on results from KGD-GBLUP only are reported and discussed. Using the adjusted phenotypic trait means (BLUPs) estimated across both locations, predictive ability for all traits was positive and was strongly correlated with <em>h<sup>2</sup><sub>g</sub></em> (<em>r</em> = 0.65). The highest predictive ability observed was for Na and S (both <em>r</em> = 0.45), followed by CFAT (0.38) (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>). The lowest predictive ability was noted for P (0.16), followed by DOMD with a value of 0.22 (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>). The slope of the regression model (bias) for all nutritive traits was around 1, meaning unbiased estimates were obtained by regressing GEBVs on adjusted means (BLUPs) (Table S11).</p><p id="p-35">Predictive ability of models based on phenotypic means from Lincoln only (location-specific predictive ability) was negative to low and showed a very high correlation with <em>h<sup>2</sup><sub>g</sub></em> (<em>r</em> = 0.93) (Table S3). The highest predictive ability was obtained for Na (0.35), similar to the across locations analysis, and the lowest predictive ability was for ADF with a negative accuracy of -0.06. Predictive ability of models using phenotypic data from Aorangi were generally higher than both the Lincoln and across-location models (Table S4) and the correlation between <em>h<sup>2</sup><sub>g</sub></em> and predictive ability was 0.67. In this dataset the highest predictive ability was for HMW WSC (0.56) and lowest predictive ability was for Ca (0.16) (Table S3).</p><p id="p-36">In terms of different trait categories, for the measures of fiber content, ADF and NDF, predictive ability of the across-location models was moderate, at 0.24 and 0.36 respectively. There was a strong effect of location on these traits, with moderate predictive ability at Aorangi (ADF = 0.29 and NDF = 0.35) whereas at Lincoln, the predictive ability was almost zero for NDF (0.02) and negative for ADF (-0.06) (Table S3 and S4).</p><p id="p-37">The traits DOMD, CFAT, WSC (LMW, HMW and total) and ME were grouped as energy traits in this study. Predictive ability for energy traits in the across location analysis was generally low to moderate, with CFAT (0.38) and LMW WSC (0.34) the highest, and DOMD (0.22) and HMW WSC (0.23) low (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>). As with the fiber traits, the ranking of predictive ability for CFAT varied by environment and was highest in Lincoln and in across-location analysis, whereas predictive ability for CFAT ranked fourth highest in Aorangi. By contrast, the predictive ability estimated for DOMD was ranked similarly (fifth highest) for Lincoln and Aorangi.</p><p id="p-38">The predictive ability of genomic prediction models for mineral traits assessed in this study was generally high, with Mg, Na and S consistently ranked highest in terms of predictive ability within the two locations (Lincoln and Aorangi) and in across-location analysis. The lowest <em>h<sup>2</sup><sub>g</sub></em> was observed for P, which was reflected in the predictive ability of prediction models for Lincoln and across-location analysis. Models for tetany ratio ([K/(Ca+Mg)]), a predictor of hypomagnesaemia risk in livestock, had a predictive ability of 0.34 across locations, 0.29 at Lincoln and 0.18 at Aorangi.</p><p id="p-39">The measures CP and N are both indicative of protein content, with crude protein a derivate of measured N, obtained by multiplying N by a conversion factor of 6.25 (<a id="xref-ref-102-2" class="xref-bibr" href="#ref-102">Waghorn 2007</a>), hence predictive ability estimated within and across locations was highly similar for both the traits. Predictive ability for these traits was low to moderate, at 0.28 (CP) and 0.26 (N) in the across location analysis, 0.14 for both traits at Lincoln and 0.20 and 0.21 for CP and N at Aorangi.</p><p id="p-40">Genotyping efficiency impacts the design and overall cost of implementing GS in a breeding program. For all nutritive traits, a steady decline in predictive ability was observed from 100% (1,093,464) to 0.5% (5,467) markers and a rapid decrease in predictive ability was noted from 0.5 to 0.1% (1,093) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and Table S11). Overall, reducing the marker number to 5% (54,673) of the total available SNPs had minimal impact on overall predictive ability (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and Table S11). Further reductions in marker number resulted in losses in predictive ability, the extent of which varied by trait (Table S11). For example, with 10,934 markers (1% of the total dataset) the predictive ability for LMW WSC, HMW WSC and total WSC decreased by 3%, 7% and 4%, respectively compared to the total dataset (100%) (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). At 1,093 markers (0.1%) the predictive ability for these traits declined further although the absolute values were still positive, at 0.31 for LMW WSC, 0.18 for HMW WSC and 0.26 for total WSC (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). The decay in predictive ability was typically highest for those traits which had low <em>h<sup>2</sup><sub>g</sub></em> and low predictive ability under the full SNP dataset. For example, between the highest and lowest marker number datasets there was a 36% decrease in predictive ability for P (<em>h<sup>2</sup><sub>g</sub></em> = 0.22), while for S (<em>h<sup>2</sup><sub>g</sub></em> = 0.53) there was a 14% decrease in predictive ability (Table S11).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/695/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Random subsets of markers ranging from 0.1% (1,093) to 100% (1,093,464) of the marker set, used in GBLUP model to estimate predictive ability for HMW WSC, LMW WSC and Total WSC." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-718683525" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Random subsets of markers ranging from 0.1% (1,093) to 100% (1,093,464) of the marker set, used in GBLUP model to estimate predictive ability for HMW WSC, LMW WSC and Total WSC.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/695/F2.medium.gif" width="343" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/2/695/F2.medium.gif" width="343" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/695/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/695/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495519" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-41" class="first-child">Random subsets of markers ranging from 0.1% (1,093) to 100% (1,093,464) of the marker set, used in GBLUP model to estimate predictive ability for HMW WSC, LMW WSC and Total WSC.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">The training population used in this study is a composite of five different breeding populations, with differing genetic relationships (see <a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1</a> in <a id="xref-ref-33-12" class="xref-bibr" href="#ref-33">Faville <em>et al.</em> (2018)</a>). The predictive ability of a model constructed based on a composite training set, for each of the individual populations is therefore an important consideration. Cross-validations were conducted within the individual populations using the protocol reported by <a id="xref-ref-33-13" class="xref-bibr" href="#ref-33">Faville <em>et al.</em> (2018)</a>. Predictive ability varied among the populations (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). For example, predictive ability for ADF ranged from 0.13 to 0.24 among the five populations (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). The majority of predictions were positive across all populations, with the exception of K for Pop I, and only LMW WSC and P in Pop II had notably poor predictive ability (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). No population was superior for genomic prediction of all nutritive traits. However, Pop V returned the highest predictive ability overall (mean predictive ability of Pop V = 0.29, compared with 0.30 in the training set, TP), followed by Pop III, Pop I, Pop IV, and Pop II (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/695/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predictive ability for 18 nutritive traits in each individual population (Pop I &#x2013; Pop V) and in complete training population (TP). For each population, predictive ability was estimated based on genomic prediction model built using complete training population (TP). Predictive ability is the mean of 500 iterations and error bars represents the standard deviation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-718683525" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Predictive ability for 18 nutritive traits in each individual population (Pop I &#x2013; Pop V) and in complete training population (TP). For each population, predictive ability was estimated based on genomic prediction model built using complete training population (TP). Predictive ability is the mean of 500 iterations and error bars represents the standard deviation.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/695/F3.medium.gif" width="440" height="227"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/2/695/F3.medium.gif" width="440" height="227"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/695/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/695/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495772" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-43" class="first-child">Predictive ability for 18 nutritive traits in each individual population (Pop I – Pop V) and in complete training population (TP). For each population, predictive ability was estimated based on genomic prediction model built using complete training population (TP). Predictive ability is the mean of 500 iterations and error bars represents the standard deviation.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-44">Nutritive quality traits in forages are important for animal productivity and for maintaining livestock health and are therefore important targets for genetic improvement in perennial ryegrass. Nutritive traits can be expensive to measure and are labor-intensive, hindering the improvement of these traits by conventional breeding methods. Genomic selection (GS), the use of genome-wide molecular markers for the prediction of breeding values in selection candidates, is well suited for traits that are costly and difficult to phenotype (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Heffner <em>et al.</em> 2009</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Jannink <em>et al.</em> 2010</a>) and therefore represents a promising approach for enabling cost-effective improvement of nutritive traits in forages. In this study we demonstrate that GS is a strong prospect for improvement of nutritive quality traits as assessed by cross-validation predictive abilities estimated for 18 nutritive traits in a multi-population training set. Furthermore, the extensive phenotypic dataset, collected from two contrasting environments, has enabled the contribution of family, location, and family-by-location variance components to be estimated across a large range of nutritive traits.</p><p id="p-45">Several methods for GS have been proposed for both plant and animal breeding, including GBLUP, Bayesian alphabets (BayesA, BayesB and BayesCπ), Ridge Regression (RR) BLUP, Random Forest, Support Vector Machine and deep learning through Multilayer Perceptron’s and Convolutional Neural Networks (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">De Los Campos <em>et al.</em> 2013</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Crossa <em>et al.</em> 2017</a>). Both simulations and empirical data suggests that linear models are superior in terms of predicting GEBVs at higher accuracy (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Daetwyler <em>et al.</em> 2010</a>; <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">De Los Campos <em>et al.</em> 2013</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Byrne <em>et al.</em> 2017</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bellot <em>et al.</em> 2018</a>; <a id="xref-ref-33-14" class="xref-bibr" href="#ref-33">Faville <em>et al.</em> 2018</a>). In this study, we compared three linear models characterized by two different assumptions with respect to the distribution of variance for marker effects. In GBLUP and KGD-GBLUP all marker effects are shrunk equally, assuming the predicted trait is controlled by many markers with small effect (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Goddard <em>et al.</em> 2011</a>), whereas BayesCπ assumes that the trait is a mixture of distributions with large and small effect markers (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Habier <em>et al.</em> 2011</a>). Even with different prior assumptions, <a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1</a> illustrates the similarity in predictive ability among the three methods for all nutritive traits, with only minor differences (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1</a>). Through simulation and empirical data, <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">De Los Campos <em>et al.</em> (2013)</a> pointed out that the superiority of Bayesian variable selection models can be illustrated when applied to a trait with large effect quantitative trait loci (QTL). The lack of improvement in predictive ability under the BayesCπ model observed here may reflect a complex genetic architecture for the nutritive traits studied, which are likely controlled by many genes with small effects. For instance, QTL studies in perennial ryegrass reported 25 loci for WSC (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cogan <em>et al.</em> 2005</a>; <a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">Turner <em>et al.</em> 2006</a>; <a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Shinozuka <em>et al.</em> 2012</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Gallagher <em>et al.</em> 2015</a>), however genetic variation explained by the multiple QTL was no more than 20%, suggesting that genetic control of WSC may tend toward an infinitesimal model.</p><p id="p-46">The success of GS primarily depends on the predictive ability of the genomic prediction model, which is influenced by <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-64.gif"/></noscript></span></span>, training population size, linkage disequilibrium (LD), genetic diversity within the training population and relatedness between training and test set (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Daetwyler <em>et al.</em> 2013</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Crossa <em>et al.</em> 2017</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Arojju <em>et al.</em> 2018</a>). Traits with low <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-65.gif"/></noscript></span></span> need a larger training population to achieve the same level of predictive ability as a trait with higher <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-66.gif"/></noscript></span></span>. Results from our study indicate that predictive ability estimated by cross-validation and <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-67.gif"/></noscript></span></span> will not be a limiting factor for implementing GS for nutritive traits in perennial ryegrass, as predictive ability and various measures of heritability (<em>R</em>, <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-68.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-69.gif"/></noscript></span></span>) were moderate to high for most traits (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>, Table S3-S9 and <a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1</a>). A strong positive correlation was observed between predictive ability and <em>h<sup>2</sup><sub>g</sub></em> for traits at the individual locations (Aorangi and Lincoln) and in the across-location analysis, confirming previous findings (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Crossa <em>et al.</em> 2017</a>) and suggesting that genomic prediction can be more accurate for highly heritable traits. A positive correlation between predictive ability and heritability was also previously observed for nutritive traits in switchgrass (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Fiedler <em>et al.</em> 2018</a>) and alfalfa (<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Jia <em>et al.</em> 2018</a>), as well as for crown rust and heading date in perennial ryegrass (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Arojju <em>et al.</em> 2018</a>) and for fruit quality traits in apple (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Muranty <em>et al.</em> 2015</a>).</p><p id="p-47">For most traits, <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-70.gif"/></noscript></span></span> at Aorangi was consistently higher compared to Lincoln, and consequently higher predictive abilities were observed. This difference between locations was due to a combination of the family variance component estimated at Aorangi being higher and estimates of trait-associated experimental error being higher at Lincoln (Table S3 and S4). While it is not possible to conclusively determine the basis of this disparity in experimental error, it may be explained by greater within-environment variability at Lincoln, due to factors such as climatic variations over the sampling period (Figure S1) soil heterogeneity or operator-to-operator variations. The predictive ability for a given trait also varied among each of the individual populations. For some traits, a positive correlation was observed between family variance and predictive ability but for the majority of traits there is no such relationship (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a> and Table S5 - S9). Another factor that may have affected prediction outcomes was the experimental design. While every practical effort was made to control for diurnal variation in levels of measured constituents, the relatively wide daily sampling window (4.5 hr) used will affected the accurate estimation of breeding values (BLUP’s). The application of a two-phase design (<a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Smith <em>et al.</em> 2006</a>) accommodating both field and laboratory sources of variation in the experiment, may also have been beneficial in terms of the accuracy of the estimated breeding values and should be considered for future studies of this nature. Finally, another contributing factor could be the inability of the marker dataset to capture differences in allelic effects between the populations. A recent study has shown that across cattle breeds (Holstein and Jersey) genomic predictive ability can be improved, using a pre-selected marker set based on GWAS performed on combined breeds using whole-genome sequencing data (<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Raymond <em>et al.</em> 2018</a>). This approach could be extended to improve across population prediction ability in perennial ryegrass. Finally, in the current study, a two-stage GS approach was undertaken, whereby in stage one adjusted means (BLUPs) were calculated from field trial data and, in stage two, these BLUPs were used as a dependent variable in the model to predict GEBVs based on markers. The two-stage GS approach is a commonly-used procedure, in part because it is a computationally -efficient method for analyzing large datasets from multi-environment trials (<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Piepho <em>et al.</em> 2012</a>). This approach has consequently been widely used across many plant species for GS application (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Lipka <em>et al.</em> 2014</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Annicchiarico <em>et al.</em> 2015</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Fernandes <em>et al.</em> 2018</a>; <a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">Sun <em>et al.</em> 2019</a>). However, the double shrinkage imposed by the two-stage GS system is considered a major limitation, which may lead to less reliable GEBVs if not de-regressed (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Ostersen <em>et al.</em> 2011</a>). As opposed to two-stage, in a single-stage GS method the adjusted means are shrunk only once and this is hence considered the ‘gold standard’ approach (<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Schulz-Streeck <em>et al.</em> 2013</a>). Implementing single-stage GS has its own limitations, as it cannot be adopted for models beyond general and generalized linear mixed models, and these models are computationally demanding with increased complexity of the dataset. In two-stage GS, to overcome the limitation of double shrinkage, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Garrick <em>et al.</em> (2009)</a> used de-regressed BLUPs in the genomic prediction model. However, <a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">Sun <em>et al.</em> (2017)</a>; <a id="xref-ref-97-2" class="xref-bibr" href="#ref-97">Sun <em>et al.</em> (2019)</a> suggested that, when lines are balanced across replicates in each environment (which was the case in our study), differential shrinkage of BLUPs will not be an issue, when used as a dependent variable in genomic prediction models. The use of single-stage approaches should be considered for future application of GS in ryegrass, with trials carefully designed to accommodate this.</p><p id="p-48">In contrast to switchgrass (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Fiedler <em>et al.</em> 2018</a>) and alfalfa (<a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">Jia <em>et al.</em> 2018</a>), prior to this study genomic predictive ability for nutritive traits has been evaluated in perennial ryegrass for only a limited set of traits. <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Fè <em>et al.</em> (2016)</a> reported high predictive abilities of 0.68 for NDF and 0.45 for fructan in a large training set of 1918 F<sub>2</sub> families, evaluated at multiple environments. In another study, <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Grinberg <em>et al.</em> (2016)</a> reported similarly high predictive abilities for WSC (0.59), DMD (0.41) and N (0.31) from prediction models applied in F<sub>14</sub> generation families after training using a set of 364 families from earlier generations, phenotyped at a single location. Predictive ability for nutritive traits in the present study were overall lower compared to those reported by <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Fè <em>et al.</em> (2016)</a> and <a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Grinberg <em>et al.</em> (2016)</a> with predictive abilities of 0.35, 0.29 and 0.22 for NDF, total WSC and HMW WSC (fructan), respectively. The lower predictive ability was likely affected by the smaller training population used in this study compared to <a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">Fè <em>et al.</em> (2016)</a>, as well as its composite nature. Overall, the values in the current study, based on a relatively small, composite training set were sufficiently high to support prediction of GEBVs and implementation of GS to accelerate genetic gain for nutritive traits across environments in perennial ryegrass.</p><p id="p-49">Determining the magnitude and genetic basis of G x E interactions for a trait is important, as it can assist in making appropriate breeding design decisions for the development of cultivars that are adapted to a broad range of target environments. In the current study family-by-location interactions were significant for all nutritive quality traits. The majority of traits displayed a family-by-location interaction that was small in comparison to family variance, when nutritive traits were evaluated at two distinct locations (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>). This was reflected in the ratios of σ<sub>g</sub> to σ<sub>gs</sub>, which was &gt; 1 for 60% of the traits, indicating that the family variance was predominant. However, the ratio for CFAT, total WSC, LMW WSC, HMW WSC, P and N were &lt; 1, indicating a greater influence of family-by-location interactions. The identification of high G x E interactions for WSC contrasts with results reported by <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Easton <em>et al.</em> (2009)</a> and are at variance with propositions by <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Casler and Vogel (1999)</a> and <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Jafari (2012)</a>, that G x E for WSC is minimal to negligible. Our results are based on relatively large populations of half-sib families, compared to previous studies and may therefore be a more accurate reflection of the influence of family-by-location interactions on these traits, particularly in New Zealand environments. However, it should be noted that the family-by-location interactions estimated here were based on only two locations, and a more robust estimation would be derived if based on a larger number of locations, representing the full target population of environments. The presence of G x E interactions may negatively influence ability to improve these traits for broad adaptation and represents a challenge during selections (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Holland <em>et al.</em> 2003</a>).</p><p id="p-50">Where family-by-location interaction effects are large and significant, genetic improvement for a trait may only be achieved through selection based on multi-year, multi-environment evaluation. Considering the relatively high costs associated with phenotyping of nutritive quality traits, this approach might not always be feasible, and decisions will be based on available resources. Genomic selection, however, represents a promising approach to more directly tackle family-by-location interactions. Models such as marker-by-environment interactions proposed by <a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Lopez-Cruz <em>et al.</em> (2015)</a> and further developed by <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Crossa <em>et al.</em> (2016)</a>, can be used to identify genomic regions that are stable across environments and other regions that are associated with specific environments that contribute to G x E interactions. These marker effects can be fixed in GS models to assist the selection of stable genotypes. However, these models were primarily developed for wheat, and a detailed investigation is needed to assess models perform in outcrossing species such as perennial ryegrass.</p><p id="p-51">Traits with high family-by-location interactions displayed both lower <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-71.gif"/></noscript></span></span> and comparatively low predictive abilities (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>, Table S1-S2 and <a id="xref-fig-1-12" class="xref-fig" href="#F1">Figure 1</a>). For such traits multi-trait genomic prediction models (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Jia and Jannink 2012</a>) may be one way of improving predictive ability and thereby genetic gain. The concept of multi-trait genomic prediction approaches is to improve the predictive ability of a primary target trait (which may be difficult and expensive to phenotype) by utilizing the genetic correlation with a secondary trait which is highly heritable and significantly less expensive to phenotype. Heritability and genotypic correlation data generated in the current study may assist in designing multi-trait prediction models for key nutritive traits. For example, a negative genetic correlation was observed between fiber and WSC traits, as reported previously in Italian ryegrass (<a id="xref-ref-104-1" class="xref-bibr" href="#ref-104">Wang <em>et al.</em> 2015</a>), and a positive genetic correlation was observed between DOMD and WSC traits as described previously by <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Humphreys (1989b)</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Jafari <em>et al.</em> (2003b)</a> (Table 4). These secondary traits (ADF, NDF and DOMD) are measured routinely and relatively inexpensively by NIRS and may therefore be useful in multi-trait genomic prediction models to more accurately predict WSC traits that are most accurately measured using more expensive wet chemistry methodologies.</p><p id="p-52">Mineral composition of forages is of interest from a perspective of livestock health and, as with nutritive traits overall, there has been little or no emphasis on selection for mineral composition in forage breeding programs (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Masters <em>et al.</em> 2019</a>). Significant family variation was observed for all minerals in this study, with relatively low influence of family-by-location interactions, moderate to high heritability and genomic prediction models with predictive abilities high in comparison to the other nutritive quality traits assessed (<a id="xref-fig-1-13" class="xref-fig" href="#F1">Figure 1</a>). This indicates that selective breeding for levels of micro- and macro-minerals is feasible and that genomic selection represents a strong option for pursuing improvement in these traits. In general, ryegrass cultivars that grow well under low soil P will compete less for P in the sward, increasing P availability for uptake to support legume growth (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Easton <em>et al.</em> 1997</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">McDowell <em>et al.</em> 2011</a>). For instance, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Crush <em>et al.</em> (2006)</a>, reported that in a mixed sward of ryegrass and clover (18% clover content), net annual flux of P into ryegrass was 4.7 times higher compared to clover. A small improvement in ryegrass phosphate use efficiency (PUE), can significantly change these proportions and may have large environmental and economic benefits (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Crush <em>et al.</em> 2018a</a>). In the current dataset predictive ability for P was very low (0.13), underpinned by a significant family-by-location interaction component to total phenotypic variation. This indicates that breeding for this P levels in perennial ryegrass foliage needs to be designed to account for family-by-location interaction effects. Alternatively, moderate to high genetic correlation with high <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-72.gif"/></noscript></span></span> traits, such as Mg (genotypic correlation -0.62), might support an indirect multi-trait genomic selection strategy, as discussed earlier.</p><p id="p-53">Hypomagnesaemia or grass tetany is a metabolic disorder in ruminants, caused by inadequate supply of Ca and Mg. This is often described in terms of a tetany index ([K/Ca+Mg]), for which values exceeding 2.2 (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Kemp and T Hart 1957</a>) are associated with increased risk of the disorder. We observed a moderate predictive ability for the ratio and the magnitude of family-by-location interaction was low compared to family variation, suggesting that tetany ratio could be used successfully as a selection criterion for developing cultivars with reduced potential for the incidence of hypomagnesaemia. This is in contrast to the results of <a id="xref-ref-95-2" class="xref-bibr" href="#ref-95">Smith <em>et al.</em> (1999)</a>, who reported large G x E variance for the tetany ratio evaluated at two locations in Australian environments and suggested the use Mg alone as a selection criteria to improve tetany ratio. Results from the current study showed a high predictive ability for Mg, making GS a viable strategy for this trait. Although, increasing Mg concentration alone may be sufficient to decrease the incidence of hypomagnesaemia, the presence of a positive correlation between Mg and K observed in the current study (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>) and reported by <a id="xref-ref-95-3" class="xref-bibr" href="#ref-95">Smith <em>et al.</em> (1999)</a>, suggests that selections based on Mg concentration alone should be monitored and might not always give the expected outcome.</p><p id="p-54">Using approximately 50k random markers the predictive ability of genomic prediction models for all nutritive traits was similar to using the full dataset of ca. 1M markers (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a> and Table S11). Previously, <a id="xref-ref-33-15" class="xref-bibr" href="#ref-33">Faville <em>et al.</em> (2018)</a> showed that reducing SNP number, by filtering out sites based on different missing data thresholds (1%, 10% and 50%), did not significantly affect genomic predictive ability for the traits herbage accumulation or days-to-heading in this training set. Similarly, filtering GBS data based on a read depth threshold (&gt;7) did not improve predictive ability for those traits. To investigate the minimum number of SNP markers needed to achieve maximum predictive ability within the current dataset, without introducing bias in terms of data missing-ness, random marker sets with varying numbers of SNPs were used to build genomic prediction models for all nutritive traits, using the across locations dataset. Below the 50k marker number there was a decrease in predictive ability, and this was particularly evident for traits with low <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-73.gif"/></noscript></span></span>. Considering the low levels of LD (<em>r<sup>2</sup></em> decaying to 0.25 after 366-1750 base pairs (<a id="xref-ref-33-16" class="xref-bibr" href="#ref-33">Faville <em>et al.</em> 2018</a>)) observed in the component populations of the training set, the major proportion of predictive ability is likely a result of capturing relationship among individuals, rather than historical LD with QTL. In perennial ryegrass, to capture genetic variance associated with all causative QTL a very large number of markers and a large training population are needed, due to rapid decay of LD as a result of a very large past effective population size (<span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/695/embed/mml-math-74.gif"/></noscript></span></span>) (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Hayes <em>et al.</em> 2013</a>; <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Fiedler <em>et al.</em> 2018</a>). Predictive ability based on relatedness between training and selection population can deteriorate after a few selection cycles (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Habier <em>et al.</em> 2007</a>), and to maintain adequate predictive ability, either the training population should be very large and highly diverse or some form of relatedness should exit between training and selection population (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Hayes <em>et al.</em> 2013</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Norman <em>et al.</em> 2018</a>).</p><p id="p-55">In conclusion, family variation and family-by-location interactions were significant for all nutritive quality traits evaluated in two distinct New Zealand environments. The predictive ability of genomic prediction models reported in this study for most of the traits would be sufficient to implement GS for nutritive traits in perennial ryegrass. Although a major proportion of this predictive ability is the result of capturing relatedness among individuals, maintaining relatedness between training and selection population would be an option to implement GS in perennial ryegrass. Predictive ability for most of the nutritive traits was retained even with as few as 50,000 markers. A next step would be to simulate a cost-benefit analysis to study the implications of manipulating marker number for cost-effective GS. For traits with low G x E interactions, single-trait genomic prediction models can be considered and for traits with large G x E, and consequently lower heritability, multi-trait approaches may be useful to explore as a method for obtaining high levels of prediction accuracy. This appears to be particularly important for WSC, which is considered to be one of the primary constituents of nutritive value for forages.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-56">The research was supported by Pastoral Genomics (PSTG1501), a joint venture co-funded by DairyNZ, Beef+Lamb New Zealand, Dairy Australia, AgResearch Ltd, Barenbrug Agriseeds Ltd, Grasslands Innovation Ltd, and the Ministry of Business, Innovation and Employment, New Zealand (MBIE).</p><p id="p-57">The authors thank Dr. Alan Stewart, Keith Saulsbury and Tom Lyons of Grasslands Innovation Ltd for providing accesses to the ryegrass populations used in this study. We are grateful to Jason Trethewey, Anthony Hilditch, Lee Sutherland, Ivan Baird, Keith Widdup, Phil Rolston, Casey Flay, Jana Schmidt, Won Hong, Alieu Sartie, Aimee Avery, Anna Larking, Prue Taylor, Michelle Ebbett, Craig Anderson, Jessica O’Connor and many others for their assistance in trial harvests and sample processing. Special thanks to Dr. Andrew Griffiths, Dr. Marcelo Carena, Dr. Valerio Hoyos-Villegas and anonymous reviewers for their suggestions to improve this manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-58">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.10074323">https://doi.org/10.25387/g3.10074323</a>.</p></li><li class="fn" id="fn-3"><p id="p-59"><em>Communicating editor: P. Ingvarsson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-10-31"><span class="received-label">Received </span>October 31, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-11-25"><span class="accepted-label">Accepted </span>November 25, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Arojju <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.1" data-doi="10.1186/s12864-015-2212-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Annicchiarico</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Nazzicari</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wei</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Pecetti</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Accuracy of genomic selection for alfalfa biomass yield in different reference populations.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1020</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-2212-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D1020%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2212-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2212-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.2" data-doi="10.1186/s12863-018-0613-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arojju</span>,  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Conaghan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Barth</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Milbourne</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Casler</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genomic prediction of crown rust resistance in <em>Lolium perenne</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">35</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12863-018-0613-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D19%26rft.spage%253D35%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12863-018-0613-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12863-018-0613-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.695.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baert</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Muylle</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Feeding value evaluation in grass and legume breeding and variety testing: Report of a debate</span>, pp. <span class="cit-fpage">307</span>–<span class="cit-lpage">311</span> in <span class="underline">Breeding in a World of Scarcity</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin, Germany</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.4" data-doi="10.1534/genetics.118.301298"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bellot</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Pérez-Enciso</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Can deep learning improve genomic prediction of complex human traits?</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">210</span>: <span class="cit-fpage">809</span>–<span class="cit-lpage">819</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.301298</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.301298%26rft_id%253Dinfo%253Apmid%252F30171033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjEwLzMvODA5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi82OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.5" data-doi="10.1371/journal.pone.0169234"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Biazzi</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Nazzicari</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Pecetti</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Brummer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Palmonari</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome-wide association mapping and genomic selection for alfalfa (<em>Medicago sativa</em>) forage quality traits.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">12</span>: <span class="cit-elocation-id">e0169234</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0169234</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D12%26rft.spage%253De0169234%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0169234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0169234&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.695.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. R.</span> <span class="cit-name-surname">Cullis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gilmour</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Gogel</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">ASReml-R reference manual. The State of Queensland</span>, <span class="cit-publ-name">Department of Primary Industries and Fisheries</span>, <span class="cit-publ-loc">Brisbane</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.7" data-doi="10.1038/s41598-017-03232-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>,  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Conaghan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Barth</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. K.</span> <span class="cit-name-surname">Arojju</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Casler</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Using variable importance measures to identify a small set of SNPs to predict heading date in perennial ryegrass.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">3566</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41598-017-03232-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D7%26rft.spage%253D3566%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-017-03232-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-017-03232-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.8" data-doi="10.1111/tpj.13037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>,  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Nagy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Pfeifer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Armstead</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Swain</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A synteny‐based draft genome sequence of the forage grass <em>Lolium perenne</em>.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">816</span>–<span class="cit-lpage">826</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13037</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D84%26rft.spage%253D816%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13037%26rft_id%253Dinfo%253Apmid%252F26408275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26408275&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.9" data-doi="10.2135/cropsci1999.0011183X003900010003x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. P.</span> <span class="cit-name-surname">Vogel</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Accomplishments and impact from breeding for increased forage nutritional value.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">12</span>–<span class="cit-lpage">20</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1999.0011183X003900010003x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCasler%26rft.auinit1%253DM.%2BD.%26rft.volume%253D39%26rft.issue%253D1%26rft.spage%253D12%26rft.epage%253D20%26rft.atitle%253DAccomplishments%2Band%2BImpact%2Bfrom%2BBreeding%2Bfor%2BIncreased%2BForage%2BNutritional%2BValue%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1999.0011183X003900010003x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1999.0011183X003900010003x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000078747300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.10" data-doi="10.1111/gfs.12213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bryant</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Olayemi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Edwards</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Thorrold</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">An economically based evaluation index for perennial and short‐term ryegrasses in New Zealand dairy farm systems.</span> <abbr class="cit-jnl-abbrev">Grass Forage Sci.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/gfs.12213</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGrass%2BForage%2BSci.%26rft.volume%253D72%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fgfs.12213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/gfs.12213&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.11" data-doi="10.1071/AN14838"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Edwards</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Stewart</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">McEvoy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">O’Donovan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Valuing forages for genetic selection: what traits should we focus on?</span> <abbr class="cit-jnl-abbrev">Anim. Prod. Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">869</span>–<span class="cit-lpage">882</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1071/AN14838</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BProd.%2BSci.%26rft.volume%253D55%26rft.spage%253D869%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAN14838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AN14838&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.12" data-doi="10.1007/s00122-004-1848-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cogan</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Smith</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Yamada</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Francki</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Vecchies</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">QTL analysis and comparative genomics of herbage quality traits in perennial ryegrass (<em>Lolium perenne</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">364</span>–<span class="cit-lpage">380</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-004-1848-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DCogan%26rft.auinit1%253DN.%2BO.%26rft.volume%253D110%26rft.issue%253D2%26rft.spage%253D364%26rft.epage%253D380%26rft.atitle%253DQTL%2Banalysis%2Band%2Bcomparative%2Bgenomics%2Bof%2Bherbage%2Bquality%2Btraits%2Bin%2Bperennial%2Bryegrass%2B%2528Lolium%2Bperenne%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-004-1848-9%26rft_id%253Dinfo%253Apmid%252F15558228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-004-1848-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15558228&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226553900019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.13" data-doi="10.2135/cropsci2015.04.0260"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Maccaferri</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Tuberosa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Burgueño</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Extending the marker× environment interaction model for genomic-enabled prediction and genome-wide association analysis in durum wheat.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">2193</span>–<span class="cit-lpage">2209</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2015.04.0260</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2015.04.0260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2015.04.0260&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.14" data-doi="10.1016/j.tplants.2017.08.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez-Rodríguez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Cuevas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Montesinos-López</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Jarquín</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic selection in plant breeding: methods, models, and perspectives.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">961</span>–<span class="cit-lpage">975</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2017.08.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D22%26rft.spage%253D961%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2017.08.011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2017.08.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.15" data-doi="10.1080/00288233.2006.9513702"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crush</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Woodward</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Eerens</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Macdonald</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Growth and milksolids production in pastures of older and more recent ryegrass and white clover cultivars under dairy grazing.</span> <abbr class="cit-jnl-abbrev">N. Z. J. Agric. Res.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">135</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/00288233.2006.9513702</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BZ.%2BJ.%2BAgric.%2BRes.%26rft.volume%253D49%26rft.spage%253D119%26rft_id%253Dinfo%253Adoi%252F10.1080%252F00288233.2006.9513702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/00288233.2006.9513702&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.16" data-doi="10.1080/00288233.2017.1418397"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crush</span>,  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Lee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. P.</span> <span class="cit-name-surname">Cosgrove</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Rossi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. F.</span> <span class="cit-name-surname">Chapman</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">a</span> <span class="cit-article-title">Foliar macronutrient concentrations of eight perennial ryegrass (<em>Lolium perenne</em> L.) cultivars grown in four regions in New Zealand.</span> <abbr class="cit-jnl-abbrev">N. Z. J. Agric. Res.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">300</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/00288233.2017.1418397</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BZ.%2BJ.%2BAgric.%2BRes.%26rft.volume%253D61%26rft.spage%253D285%26rft_id%253Dinfo%253Adoi%252F10.1080%252F00288233.2017.1418397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/00288233.2017.1418397&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.17" data-doi="10.1080/00288233.2018.1426020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crush</span>,  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Lee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. P.</span> <span class="cit-name-surname">Cosgrove</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Rossi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. F.</span> <span class="cit-name-surname">Chapman</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">b</span> <span class="cit-article-title">Foliar micronutrient concentrations of eight perennial ryegrass (<em>Lolium perenne</em> L.) cultivars grown in four regions in New Zealand.</span> <abbr class="cit-jnl-abbrev">N. Z. J. Agric. Res.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">311</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/00288233.2018.1426020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BZ.%2BJ.%2BAgric.%2BRes.%26rft.volume%253D61%26rft.spage%253D301%26rft_id%253Dinfo%253Adoi%252F10.1080%252F00288233.2018.1426020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/00288233.2018.1426020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.18" data-doi="10.1534/genetics.112.147983"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>,  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Calus</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Pong-Wong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Hickey</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction in animals and plants: simulation of data, validation, reporting, and benchmarking.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">365</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.147983</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147983%26rft_id%253Dinfo%253Apmid%252F23222650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi82OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.19" data-doi="10.1534/genetics.110.116855"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>,  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Pong-Wong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Villanueva</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Woolliams</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The impact of genetic architecture on genome-wide evaluation methods.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1021</span>–<span class="cit-lpage">1031</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.116855</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116855%26rft_id%253Dinfo%253Apmid%252F20407128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS8zLzEwMjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzY5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.20" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Auton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Abecasis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Albers</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Banks</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr330</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653522&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.21" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Hickey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Pong-Wong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. D.</span> <span class="cit-name-surname">Daetwyler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Calus</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.143313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi82OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.22" data-doi="10.1371/journal.pgen.1005048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Sorensen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic heritability: what is it?</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e1005048</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005048</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005048%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005048%26rft_id%253Dinfo%253Apmid%252F25942577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25942577&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.23" data-doi="10.1186/s12864-015-2252-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dodds</span>,  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">McEwan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Brauning</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Anderson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. C.</span> <span class="cit-name-surname">Stijn</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Construction of relatedness matrices using genotyping-by-sequencing data.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1047</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-2252-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D1047%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2252-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2252-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.24" data-doi="10.1071/A96086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Easton</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Mackay</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Lee</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Genetic variation for macro-and micro-nutrient concentration in perennial ryegrass (<em>Lolium perenne</em> L.).</span> <abbr class="cit-jnl-abbrev">Aust. J. Agric. Res.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">657</span>–<span class="cit-lpage">666</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1071/A96086</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAust.%2BJ.%2BAgric.%2BRes.%26rft.volume%253D48%26rft.spage%253D657%26rft_id%253Dinfo%253Adoi%252F10.1071%252FA96086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/A96086&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Easton</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Stewart</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Lyons</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Parris</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Charrier</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Soluble carbohydrate content of ryegrass cultivars.</span> <abbr class="cit-jnl-abbrev">Proceedings of the New Zealand Grassland Association</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">161</span>–<span class="cit-lpage">166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNew%2BZealand%2BGrassland%2BAssociation%26rft.volume%253D71%26rft.spage%253D161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.26"><div class="cit-metadata unstructured">Easton, S., J. Amyes, N. Cameron, R. Green, G. Kerr <em>et al.</em>, 2002 Pasture plant breeding in New Zealand: where to from here? Proceedings of the conference-New Zealand grassland association<strong>:</strong> 173–180.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>,  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Parsons</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Rasmussen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. H.</span> <span class="cit-name-surname">Bryant</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">High sugar ryegrasses for livestock systems in New Zealand.</span> <abbr class="cit-jnl-abbrev">Proceedings of the New Zealand Grassland Association</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">161</span>–<span class="cit-lpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNew%2BZealand%2BGrassland%2BAssociation%26rft.volume%253D69%26rft.spage%253D161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.28" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>,  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Glaubitz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Sun</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Poland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kawamoto</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e19379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0019379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.29" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>,  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge regression and other kernels for genomic selection with R package rrBLUP.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2011.08.0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.30" data-doi="10.1534/g3.112.004259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>,  <span class="cit-name-given-names">J. B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Shrinkage estimation of the realized relationship matrix. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1405</span>–<span class="cit-lpage">1413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D2%26rft.spage%253D1405%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.004259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.112.004259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.695.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>,  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1960</span> <span class="cit-source">Introduction to quantitative genetics</span>, <span class="cit-publ-name">Oliver And Boyd</span>, <span class="cit-publ-loc">Edinburgh, London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Faville</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Richardson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Gagic</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Mace</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Sun</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic improvement of fibre traits in perennial ryegrass.</span> <abbr class="cit-jnl-abbrev">Proceedings of the New Zealand Grassland Association</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNew%2BZealand%2BGrassland%2BAssociation%26rft.volume%253D72%26rft.spage%253D71%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.33" data-doi="10.1007/s00122-017-3030-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Faville</span>,  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Ganesh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. Z.</span> <span class="cit-name-surname">Jahufer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. P.</span> <span class="cit-name-surname">Bilton</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Predictive ability of genomic selection models in a multi-population perennial ryegrass training set using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">703</span>–<span class="cit-lpage">720</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-017-3030-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D131%26rft.spage%253D703%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-017-3030-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-017-3030-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.34" data-doi="10.3835/plantgenome2016.05.0047"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fè</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. H.</span> <span class="cit-name-surname">Ashraf</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. G.</span> <span class="cit-name-surname">Pedersen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Janss</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Byrne</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Accuracy of genomic prediction in a commercial perennial ryegrass breeding program.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2015.11.0110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D9%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.05.0047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.05.0047&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.35" data-doi="10.1007/s00122-017-3033-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernandes</span>,  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. O.</span> <span class="cit-name-surname">Dias</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. F.</span> <span class="cit-name-surname">Ferreira</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Brown</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Efficiency of multi-trait, indirect, and trait-assisted genomic selection for improvement of biomass sorghum.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">747</span>–<span class="cit-lpage">755</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-017-3033-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D131%26rft.spage%253D747%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-017-3033-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-017-3033-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.36" data-doi="10.1186/s12870-018-1360-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fiedler</span>,  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lanzatella</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Edmé</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Palmer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Sarath</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genomic prediction accuracy for switchgrass traits related to bioenergy within differentiated populations.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">142</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12870-018-1360-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol.%26rft.volume%253D18%26rft.spage%253D142%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12870-018-1360-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12870-018-1360-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.37" data-doi="10.1007/s00122-018-3068-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fristche-Neto</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Akdemir</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Accuracy of genomic selection to predict maize single-crosses obtained through different mating designs.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">1153</span>–<span class="cit-lpage">1162</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-018-3068-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D131%26rft.spage%253D1153%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-018-3068-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-018-3068-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.38" data-doi="10.1007/s12155-014-9491-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gallagher</span>,  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. B.</span> <span class="cit-name-surname">Turner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Cairns</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Farrell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Lovatt</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic differentiation in response to selection for water-soluble carbohydrate content in perennial ryegrass (<em>Lolium perenne</em> L.).</span> <abbr class="cit-jnl-abbrev">BioEnergy Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">90</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s12155-014-9491-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEnergy%2BRes.%26rft.volume%253D8%26rft.spage%253D77%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12155-014-9491-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12155-014-9491-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.39" data-doi="10.1073/pnas.1519061113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">García-Ruiz</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Cole</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">VanRaden</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. R.</span> <span class="cit-name-surname">Wiggans</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. J.</span> <span class="cit-name-surname">Ruiz-López</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Changes in genetic selection differentials and generation intervals in US Holstein dairy cattle as a result of genomic selection.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">E3995</span>–<span class="cit-lpage">E4004</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1519061113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1519061113%26rft_id%253Dinfo%253Apmid%252F27354521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEzLzI4L0UzOTk1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi82OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.40" data-doi="10.1186/1297-9686-41-55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>,  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. F.</span> <span class="cit-name-surname">Taylor</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Deregressing estimated breeding values and weighting information for genomic regression analyses.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">55</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-41-55</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DGarrick%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D41%26rft.spage%253D55%26rft.epage%253D55%26rft.atitle%253DDeregressing%2Bestimated%2Bbreeding%2Bvalues%2Band%2Bweighting%2Binformation%2Bfor%2Bgenomic%2Bregression%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-55%26rft_id%253Dinfo%253Apmid%252F20043827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20043827&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.41" data-doi="10.1371/journal.pone.0090346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>,  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. M.</span> <span class="cit-name-surname">Casstevens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Lu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Harriman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Elshire</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e90346</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0090346</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De90346%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.42" data-doi="10.1111/j.1439-0388.2011.00964.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>,  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. H.</span> <span class="cit-name-surname">Meuwissen</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Using the genomic relationship matrix to predict the accuracy of genomic selection.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">409</span>–<span class="cit-lpage">421</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0388.2011.00964.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BE.%26rft.volume%253D128%26rft.issue%253D6%26rft.spage%253D409%26rft.epage%253D421%26rft.atitle%253DUsing%2Bthe%2Bgenomic%2Brelationship%2Bmatrix%2Bto%2Bpredict%2Bthe%2Baccuracy%2Bof%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2011.00964.x%26rft_id%253Dinfo%253Apmid%252F22059574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2011.00964.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22059574&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.43" data-doi="10.3835/plantgenome2017.11.0104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Camacho</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Ornella</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez-Rodríguez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Dreisigacker</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Applications of machine learning methods to genomic selection in breeding wheat for rust resistance.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2017.11.0104</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D11%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2017.11.0104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2017.11.0104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.44" data-doi="10.3389/fpls.2016.00133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grinberg</span>,  <span class="cit-name-given-names">N. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Lovatt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hegarty</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Lovatt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. P.</span> <span class="cit-name-surname">Skøt</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Implementation of genomic prediction in <em>Lolium perenne</em> (L.) breeding populations.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">133</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2016.00133</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D7%26rft.spage%253D133%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2016.00133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2016.00133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.45" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Dekkers</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.081190</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzY5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.46" data-doi="10.1186/1471-2105-12-186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kizilkaya</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Garrick</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Extension of the Bayesian alphabet for genomic selection.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">186</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-186</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D186%26rft.epage%253D186%26rft.atitle%253DExtension%2Bof%2Bthe%2Bbayesian%2Balphabet%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-186%26rft_id%253Dinfo%253Apmid%252F21605355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21605355&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.47" data-doi="10.1046/j.1439-0523.2003.00855.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Huth</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Schöberlein</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Diepenbrock</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Detection of endophytic fungi in <em>Festuca</em> spp. by means of tissue print immunoassay.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">222</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1439-0523.2003.00855.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D122%26rft.spage%253D217%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1439-0523.2003.00855.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1439-0523.2003.00855.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>,  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> Improved Pairwise Alignmnet of Genomic DNA.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.49" data-doi="10.1007/s00122-017-2972-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Panozzo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Walker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Choy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kant</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Accelerating wheat breeding for end-use quality with multi-trait genomic predictions incorporating near infrared and nuclear magnetic resonance-derived phenotypes.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">2505</span>–<span class="cit-lpage">2519</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-017-2972-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D2505%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-017-2972-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-017-2972-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.50" data-doi="10.1111/pbr.12037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. O.</span> <span class="cit-name-surname">Cogan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. W.</span> <span class="cit-name-surname">Pembleton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Prospects for genomic selection in forage plant species.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">143</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbr.12037</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D132%26rft.spage%253D133%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbr.12037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbr.12037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.51" data-doi="10.2135/cropsci2008.08.0512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>,  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Sorrells</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection for crop improvement.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2008.08.0512</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHeffner%26rft.auinit1%253DE.%2BL.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DGenomic%2BSelection%2Bfor%2BCrop%2BImprovement%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000263344300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.52" data-doi="10.2135/cropsci2014.03.0249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Sorrells</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Perspectives for genomic selection applications and research in plants.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2014.03.0249</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.03.0249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.03.0249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.53" data-doi="10.1002/9780470650202.ch2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>,  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. E.</span> <span class="cit-name-surname">Nyquist</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. T.</span> <span class="cit-name-surname">Cervantes-Martínez</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Estimating and interpreting heritability for plant breeding: an update.</span> <abbr class="cit-jnl-abbrev">Plant Breed. Rev.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%2BRev.%26rft.volume%253D22%26rft.spage%253D9%26rft_id%253Dinfo%253Adoi%252F10.1002%252F9780470650202.ch2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/9780470650202.ch2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.54" data-doi="10.1111/j.1365-2494.1989.tb01932.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Humphreys</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span><span class="cit-comment">a</span> <span class="cit-article-title">Water‐soluble carbohydrates in perennial ryegrass breeding.</span> <abbr class="cit-jnl-abbrev">Grass Forage Sci.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">244</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2494.1989.tb01932.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGrass%2BForage%2BSci.%26rft.volume%253D44%26rft.spage%253D237%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2494.1989.tb01932.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2494.1989.tb01932.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.55" data-doi="10.1111/j.1365-2494.1989.tb01942.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Humphreys</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span><span class="cit-comment">b</span> <span class="cit-article-title">Water‐soluble carbohydrates in perennial ryegrass breeding: III. Relationships with herbage production, digestibility and crude protein content.</span> <abbr class="cit-jnl-abbrev">Grass Forage Sci.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">430</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2494.1989.tb01942.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGrass%2BForage%2BSci.%26rft.volume%253D44%26rft.spage%253D423%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2494.1989.tb01942.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2494.1989.tb01942.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1989CC34700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.56" data-doi="10.1111/j.1365-3040.2005.01367.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>,  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Rasmussen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. C.</span> <span class="cit-name-surname">Newton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Parsons</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Newman</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Near‐term impacts of elevated CO2, nitrogen and fungal endophyte‐infection on <em>Lolium perenne</em> L. growth, chemical composition and alkaloid production.</span> <abbr class="cit-jnl-abbrev">Plant Cell Environ.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1345</span>–<span class="cit-lpage">1354</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-3040.2005.01367.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BEnviron.%26rft.volume%253D28%26rft.spage%253D1345%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-3040.2005.01367.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-3040.2005.01367.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jafari</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Connolly</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Frolich</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Walsh</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span><span class="cit-comment">a</span> <span class="cit-article-title">A note on estimation of quality parameters in perennial ryegrass by near infrared reflectance spectroscopy.</span> <abbr class="cit-jnl-abbrev">Ir. J. Agric. Food Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">293</span>–<span class="cit-lpage">299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIr.%2BJ.%2BAgric.%2BFood%2BRes.%26rft.volume%253D42%26rft.spage%253D293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jafari</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Connolly</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Walsh</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span><span class="cit-comment">b</span> <span class="cit-article-title">Genetic analysis of yield and quality in full-sib families of perennial ryegrass (<em>Lolium perenne</em> L.) under two cutting managements.</span> <abbr class="cit-jnl-abbrev">Ir. J. Agric. Food Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">275</span>–<span class="cit-lpage">292</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIr.%2BJ.%2BAgric.%2BFood%2BRes.%26rft.volume%253D42%26rft.spage%253D275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.695.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jafari</span>,  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Environmental and genetic variation for water soluble carbohydrate content in cool season forage grasses. In Carbohydrates-Comprehensive Studies on Glycobiology and Glycotechnology. IntechOpen</span>. <a href="https://www.intechopen.com/books/carbohydrates-comprehensive-studies-on-glycobiology-and-glycotechnology/environmental-and-genetic-variation-for-water-soluble-carbohydrate-content-in-cool-season-forage-gra">https://www.intechopen.com/books/carbohydrates-comprehensive-studies-on-glycobiology-and-glycotechnology/environmental-and-genetic-variation-for-water-soluble-carbohydrate-content-in-cool-season-forage-gra</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.60" data-doi="10.2135/cropsci2017.07.0456"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jahufer</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Luo</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">DeltaGen: A Comprehensive Decision Support Tool for Plant Breeders.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">1118</span>–<span class="cit-lpage">1131</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2017.07.0456</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D58%26rft.spage%253D1118%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2017.07.0456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2017.07.0456&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.61" data-doi="10.1093/bfgp/elq001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>,  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Lorenz</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Iwata</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic selection in plant breeding: from theory to practice.</span> <abbr class="cit-jnl-abbrev">Brief. Funct. Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">177</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bfgp/elq001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BFunct.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felq001%26rft_id%253Dinfo%253Apmid%252F20156985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bfgp/elq001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20156985&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276191200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.62" data-doi="10.1038/177038a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jermyn</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1956</span> <span class="cit-article-title">A new method for determining ketohexoses in the presence of aldohexoses.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">38</span>–<span class="cit-lpage">39</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/177038a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D177%26rft.spage%253D38%26rft_id%253Dinfo%253Adoi%252F10.1038%252F177038a0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/177038a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.63" data-doi="10.3389/fpls.2018.01220"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Zhao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Han</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Zhao</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genomic Prediction for 25 Agronomic and Quality Traits in Alfalfa (<em>Medicago sativa</em>).</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1220</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2018.01220</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D9%26rft.spage%253D1220%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2018.01220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2018.01220&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.64" data-doi="10.1534/genetics.112.144246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multiple-trait genomic selection methods increase genetic value prediction accuracy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1513</span>–<span class="cit-lpage">1522</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.144246</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.144246%26rft_id%253Dinfo%253Apmid%252F23086217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzE1MTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzY5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.65"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Roberts</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> A note on the relationship between palatability and water-soluble carbohydrates content in perennial ryegrass. Irish Journal of Agricultural Research<strong>:</strong> 163–167.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.66" data-doi="10.2527/1995.7392774x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>,  <span class="cit-name-given-names">H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Allen</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Characteristics of plant cell walls affecting intake and digestibility of forages by ruminants.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">2774</span>–<span class="cit-lpage">2790</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/1995.7392774x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DJung%26rft.auinit1%253DH.%2BG.%26rft.volume%253D73%26rft.issue%253D9%26rft.spage%253D2774%26rft.epage%253D2790%26rft.atitle%253DCharacteristics%2Bof%2Bplant%2Bcell%2Bwalls%2Baffecting%2Bintake%2Band%2Bdigestibility%2Bof%2Bforages%2Bby%2Bruminants%26rft_id%253Dinfo%253Adoi%252F10.2527%252F1995.7392774x%26rft_id%253Dinfo%253Apmid%252F8582870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/1995.7392774x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8582870&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RV36400033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.67" data-doi="10.1093/bioinformatics/bts199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kearse</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Moir</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wilson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Stones-Havas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cheung</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1647</span>–<span class="cit-lpage">1649</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts199</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts199%26rft_id%253Dinfo%253Apmid%252F22543367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22543367&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305419800052&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kemp</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">T Hart</span></span></li></ol><cite>, <span class="cit-pub-date">1957</span> <span class="cit-article-title">Grass tetany in grazing milking cows.</span> <abbr class="cit-jnl-abbrev">Neth. J. Agric. Sci.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">4</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeth.%2BJ.%2BAgric.%2BSci.%26rft.volume%253D5%26rft.spage%253D4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.69" data-doi="10.1186/s13040-014-0034-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langdon</span>,  <span class="cit-name-given-names">W. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Performance of genetic programming optimised Bowtie2 on genome comparison and analytic testing (GCAT) benchmarks.</span> <abbr class="cit-jnl-abbrev">BioData Min.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13040-014-0034-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioData%2BMin.%26rft.volume%253D8%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13040-014-0034-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13040-014-0034-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.70" data-doi="10.1071/AN17815"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leddin</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jacobs</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Smith</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Giri</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Malcolm</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Development of a system to rank perennial ryegrass cultivars according to their economic value to dairy farm businesses in south-eastern Australia.</span> <abbr class="cit-jnl-abbrev">Anim. Prod. Sci.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">1552</span>–<span class="cit-lpage">1558</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1071/AN17815</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BProd.%2BSci.%26rft.volume%253D58%26rft.spage%253D1552%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAN17815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AN17815&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.71" data-doi="10.3835/plantgenome2014.12.0090"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wei</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Acharya</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Hansen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Crawford</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic prediction of biomass yield in two selection cycles of a tetraploid alfalfa breeding population.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2014.12.0090</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2014.12.0090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2014.12.0090&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.72" data-doi="10.1371/journal.pone.0112227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>,  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Lu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Cherney</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. S.</span> <span class="cit-name-surname">Buckler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Casler</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Accelerating the switchgrass (Panicum virgatum L.) breeding cycle using genomic selection approaches.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e112227</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0112227</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De112227%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0112227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0112227&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.73"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lopez-Cruz</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Bonnett</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Dreisigacker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Poland</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> Increased prediction accuracy in wheat breeding trials using a marker× environment interaction genomic selection model. G3:(Bethesda). 114.016097.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.74" data-doi="10.2135/cropsci2011.09.0503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>,  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. P.</span> <span class="cit-name-surname">Smith</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Potential and optimization of genomic selection for Fusarium head blight resistance in six-row barley.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">1609</span>–<span class="cit-lpage">1621</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2011.09.0503</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D1609%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.09.0503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.09.0503&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000305804900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.75" data-doi="10.1071/CP18546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Masters</span>,  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. C.</span> <span class="cit-name-surname">Norman</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. T.</span> <span class="cit-name-surname">Thomas</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Minerals in pastures—are we meeting the needs of livestock?</span> <abbr class="cit-jnl-abbrev">Crop Pasture Sci.</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1071/CP18546</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BPasture%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1071%252FCP18546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/CP18546&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.76" data-doi="10.1007/s11104-010-0687-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McDowell</span>,  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. N.</span> <span class="cit-name-surname">Sharpley</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Crush</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Simmons</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Phosphorus in pasture plants: potential implications for phosphorus loss in surface runoff.</span> <abbr class="cit-jnl-abbrev">Plant Soil</abbr> <span class="cit-vol">345</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">35</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11104-010-0687-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSoil%26rft.volume%253D345%26rft.spage%253D23%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11104-010-0687-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11104-010-0687-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-confproc" id="cit-10.2.695.77"><div class="cit-metadata unstructured">McEvoy, M., N. McHugh, M. O’Donovan, D. Grogan, and L. Shalloo, 2014 Pasture profit index: updated economic values and inclusion of persistency. EGF at 50: The future of European grasslands. Proceedings of the 25th General Meeting of the European Grassland Federation, Aberystwyth, Wales, 7–September 11, 2014: 843–845.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.78" data-doi="10.3168/jds.2010-3322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McEvoy</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">O’Donovan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Shalloo</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Development and application of an economic ranking index for perennial ryegrass cultivars.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">1627</span>–<span class="cit-lpage">1639</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2010-3322</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DMcEvoy%26rft.auinit1%253DM.%26rft.volume%253D94%26rft.issue%253D3%26rft.spage%253D1627%26rft.epage%253D1639%26rft.atitle%253DDevelopment%2Band%2Bapplication%2Bof%2Ban%2Beconomic%2Branking%2Bindex%2Bfor%2Bperennial%2Bryegrass%2Bcultivars.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2010-3322%26rft_id%253Dinfo%253Apmid%252F21338830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2010-3322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21338830&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.79" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>,  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzY5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.80" data-doi="10.1038/hortres.2015.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muranty</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Troggio</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. B.</span> <span class="cit-name-surname">Sadok</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Al Rifaï</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Auwerkerken</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Accuracy and responses of genomic selection on key traits in apple breeding.</span> <abbr class="cit-jnl-abbrev">Hortic. Res.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">15060</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hortres.2015.60</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHortic.%2BRes.%26rft.volume%253D2%26rft.spage%253D15060%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhortres.2015.60%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hortres.2015.60&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.695.81"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muylle</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Van Waes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Van Parijs</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Obianugba</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Baert</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic diversity for cell wall digestibility in a diverse Lolium perenne collection</span>, pp. <span class="cit-fpage">261</span>–<span class="cit-lpage">267</span> in <span class="cit-source">Breeding Strategies for Sustainable Forage and Turf Grass Improvement</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin, Germany</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.82"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Norman</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Taylor</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Edwards</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Kuchel</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Optimising genomic selection in wheat: effect of marker density, population size and population structure on prediction accuracy. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2889</span>–<span class="cit-lpage">2899</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D8%26rft.spage%253D2889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.83" data-doi="10.1186/1297-9686-43-38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ostersen</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. F.</span> <span class="cit-name-surname">Christensen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Henryon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Nielsen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Su</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Deregressed EBV as the response variable yield more reliable genomic predictions than traditional EBV in pure-bred pigs.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">38</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-43-38</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DOstersen%26rft.auinit1%253DT.%26rft.volume%253D43%26rft.spage%253D38%26rft.epage%253D38%26rft.atitle%253DDeregressed%2BEBV%2Bas%2Bthe%2Bresponse%2Bvariable%2Byield%2Bmore%2Breliable%2Bgenomic%2Bpredictions%2Bthan%2Btraditional%2BEBV%2Bin%2Bpure-bred%2Bpigs.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-38%26rft_id%253Dinfo%253Apmid%252F22070746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-38&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22070746&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.695.84"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Payne</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Murray</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Harding</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Baird</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Soutar</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>, p. <span class="cit-fpage">204</span> in <span class="cit-source">GenStat for windows introduction</span>, <span class="cit-publ-name">VSN International</span>, <span class="cit-publ-loc">Hemel Hempstead</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.85" data-doi="10.1007/s00122-018-3121-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pembleton</span>,  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Inch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. C.</span> <span class="cit-name-surname">Baillie</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Drayton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Thakur</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Exploitation of data from breeding programs supports rapid implementation of genomic selection for key agronomic traits in perennial ryegrass.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">1891</span>–<span class="cit-lpage">1902</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-018-3121-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D131%26rft.spage%253D1891%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-018-3121-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-018-3121-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.86" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>,  <span class="cit-name-given-names">P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de Los Campos</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide regression and prediction with the BGLR statistical package.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.164442</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi82OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.87" data-doi="10.1002/bimj.201100219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>,  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Moehring</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Schulz‐Streeck</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. O.</span> <span class="cit-name-surname">Ogutu</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A stage‐wise approach for the analysis of multi‐environment trials.</span> <abbr class="cit-jnl-abbrev">Biom. J.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">844</span>–<span class="cit-lpage">860</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/bimj.201100219</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBiom%2BJ%26rft.aulast%253DPiepho%26rft.auinit1%253DH.%2BP.%26rft.volume%253D54%26rft.issue%253D6%26rft.spage%253D844%26rft.epage%253D860%26rft.atitle%253DA%2Bstage-wise%2Bapproach%2Bfor%2Bthe%2Banalysis%2Bof%2Bmulti-environment%2Btrials.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbimj.201100219%26rft_id%253Dinfo%253Apmid%252F23007738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bimj.201100219&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23007738&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.88" data-doi="10.3835/plantgenome2012.06.0006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Endelman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Dawson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Rutkoski</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Wu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in wheat breeding using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">113</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2012.06.0006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D5%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.06.0006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.06.0006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.695.89"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.90" data-doi="10.1186/s12711-018-0396-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raymond</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Bouwman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Schrooten</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Houwing-Duistermaat</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. F.</span> <span class="cit-name-surname">Veerkamp</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Utility of whole-genome sequence data for across-breed genomic prediction.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">27</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-018-0396-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D50%26rft.spage%253D27%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-018-0396-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-018-0396-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.91" data-doi="10.1007/s10681-010-0301-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>,  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. L.</span> <span class="cit-name-surname">Heffner</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Sorrells</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection for durable stem rust resistance in wheat.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">161</span>–<span class="cit-lpage">173</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10681-010-0301-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D179%26rft.spage%253D161%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-010-0301-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-010-0301-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000289305300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.92" data-doi="10.1007/s00122-012-1960-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schulz-Streeck</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. O.</span> <span class="cit-name-surname">Ogutu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.-P.</span> <span class="cit-name-surname">Piepho</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparisons of single-stage and two-stage approaches to genomic selection.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">82</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-012-1960-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D69%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1960-1%26rft_id%253Dinfo%253Apmid%252F22903736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1960-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22903736&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.93" data-doi="10.1186/1471-2156-13-101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shinozuka</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. O.</span> <span class="cit-name-surname">Cogan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. C.</span> <span class="cit-name-surname">Spangenberg</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Forster</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Quantitative Trait Locus (QTL) meta-analysis and comparative genomics for candidate gene prediction in perennial ryegrass (<em>Lolium perenne</em> L.).</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">101</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2156-13-101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DShinozuka%26rft.auinit1%253DH.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D101%26rft.epage%253D101%26rft.atitle%253DQuantitative%2BTrait%2BLocus%2B%2528QTL%2529%2Bmeta-analysis%2Band%2Bcomparative%2Bgenomics%2Bfor%2Bcandidate%2Bgene%2Bprediction%2Bin%2Bperennial%2Bryegrass%2B%2528Lolium%2Bperenne%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-13-101%26rft_id%253Dinfo%253Apmid%252F23137269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-13-101&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23137269&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.94" data-doi="10.1017/S0021859606006319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Lim</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. R.</span> <span class="cit-name-surname">Cullis</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The design and analysis of multi-phase plant breeding experiments.</span> <abbr class="cit-jnl-abbrev">J. Agric. Sci.</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">393</span>–<span class="cit-lpage">409</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0021859606006319</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BSci.%26rft.volume%253D144%26rft.spage%253D393%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0021859606006319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0021859606006319&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.95" data-doi="10.1071/A98070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Rebetzke</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Eagles</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Anderson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Easton</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Genetic control of mineral concentration and yield in perennial ryegrass (<em>Lolium perenne</em> L.), with special emphasis on minerals related to grass tetany.</span> <abbr class="cit-jnl-abbrev">Aust. J. Agric. Res.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">86</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1071/A98070</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAust.%2BJ.%2BAgric.%2BRes.%26rft.volume%253D50%26rft.spage%253D79%26rft_id%253Dinfo%253Adoi%252F10.1071%252FA98070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/A98070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.96" data-doi="10.1111/j.1365-2494.1997.tb02347.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Reed</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Foot</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">An assessment of the relative importance of specific traits for the genetic improvement of nutritive value in dairy pasture.</span> <abbr class="cit-jnl-abbrev">Grass Forage Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">175</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2494.1997.tb02347.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGrass%2BForage%2BSci.%26rft.volume%253D52%26rft.spage%253D167%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2494.1997.tb02347.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2494.1997.tb02347.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997XP84300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference  in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.97" data-doi="10.1007/s00122-019-03309-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Poland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Mondal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Juliana</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">High-throughput phenotyping platforms enhance genomic selection for wheat grain yield across populations and cycles in early stage.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">1705</span>–<span class="cit-lpage">1720</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-019-03309-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-019-03309-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-019-03309-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference  in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.98" data-doi="10.3835/plantgenome2016.11.0111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Rutkoski</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Poland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Multitrait, random regression, or simple repeatability model in high-throughput phenotyping data improve genomic prediction for wheat grain yield.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.11.0111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D10%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.11.0111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.11.0111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference  in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.99" data-doi="10.1111/j.1469-8137.2005.01575.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>,  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Cairns</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. P.</span> <span class="cit-name-surname">Armstead</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ashton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Skøt</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Dissecting the regulation of fructan metabolism in perennial ryegrass (<em>Lolium perenne</em>) with quantitative trait locus mapping.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">45</span>–<span class="cit-lpage">58</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-8137.2005.01575.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DTurner%26rft.auinit1%253DL.%2BB.%26rft.volume%253D169%26rft.issue%253D1%26rft.spage%253D45%26rft.epage%253D57%26rft.atitle%253DDissecting%2Bthe%2Bregulation%2Bof%2Bfructan%2Bmetabolism%2Bin%2Bperennial%2Bryegrass%2B%2528Lolium%2Bperenne%2529%2Bwith%2Bquantitative%2Btrait%2Blocus%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2005.01575.x%26rft_id%253Dinfo%253Apmid%252F16390418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2005.01575.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16390418&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233530400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference  in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.100" data-doi="10.1111/gfs.12293"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Parijs</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Van Waes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Vandecasteele</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Haesaert</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Roldán‐Ruiz</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The optimal lignin quantification method to breed for an improved cell wall digestibility in perennial ryegrass.</span> <abbr class="cit-jnl-abbrev">Grass Forage Sci.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">111</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/gfs.12293</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGrass%2BForage%2BSci.%26rft.volume%253D73%26rft.spage%253D101%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fgfs.12293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/gfs.12293&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference  in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.101" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>,  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference  in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waghorn</span>,  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">What is dietary metabolisable energy?</span> <abbr class="cit-jnl-abbrev">PROCEEDINGS OF THE CONFERENCE-NEW ZEALAND GRASSLAND ASSOCIATION</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPROCEEDINGS%2BOF%2BTHE%2BCONFERENCE-NEW%2BZEALAND%2BGRASSLAND%2BASSOCIATION%26rft.volume%253D69%26rft.spage%253D153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-103-1" title="View reference  in text" id="ref-103">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.103" data-doi="10.1080/00480169.2004.36448"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waghorn</span>,  <span class="cit-name-given-names">G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Clark</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Feeding value of pastures for ruminants.</span> <abbr class="cit-jnl-abbrev">N. Z. Vet. J.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">320</span>–<span class="cit-lpage">331</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/00480169.2004.36448</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BZealand%2BVeterinary%2BJournal%26rft.stitle%253DNew%2BZealand%2BVeterinary%2BJournal%26rft.aulast%253DWaghorn%26rft.auinit1%253DG.%2BC.%26rft.volume%253D52%26rft.issue%253D6%26rft.spage%253D320%26rft.epage%253D331%26rft.atitle%253DFeeding%2Bvalue%2Bof%2Bpastures%2Bfor%2Bruminants.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F00480169.2004.36448%26rft_id%253Dinfo%253Apmid%252F15768132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/00480169.2004.36448&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15768132&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-104-1" title="View reference  in text" id="ref-104">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.104" data-doi="10.1071/CP15070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Cogan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Pembleton</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Forster</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Variance, inter-trait correlation, heritability and trait-marker association of herbage yield, nutritive values, and morphological characteristics in Italian ryegrass (<em>Lolium multiflorum</em> Lam.).</span> <abbr class="cit-jnl-abbrev">Crop Pasture Sci.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">973</span>–<span class="cit-lpage">984</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1071/CP15070</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BPasture%2BSci.%26rft.volume%253D66%26rft.spage%253D973%26rft_id%253Dinfo%253Adoi%252F10.1071%252FCP15070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/CP15070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-105-1" title="View reference  in text" id="ref-105">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.105" data-doi="10.1111/j.1365-2494.1989.tb01912.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Corbett</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Criteria for breeding forages of improved feeding value: results of a Delphi survey.</span> <abbr class="cit-jnl-abbrev">Grass Forage Sci.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">83</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2494.1989.tb01912.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGrass%2BForage%2BSci.%26rft.volume%253D44%26rft.spage%253D77%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2494.1989.tb01912.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2494.1989.tb01912.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-106-1" title="View reference  in text" id="ref-106">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.106" data-doi="10.1007/BF00029397"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilkins</span>,  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Breeding perennial ryegrass for agriculture.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">201</span>–<span class="cit-lpage">214</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00029397</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D52%26rft.spage%253D201%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00029397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00029397&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-107-1" title="View reference  in text" id="ref-107">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.107" data-doi="10.1017/S0021859603003058"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilkins</span>,  <span class="cit-name-given-names">P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Humphreys</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Progress in breeding perennial forage grasses for temperate agriculture.</span> <abbr class="cit-jnl-abbrev">J. Agric. Sci.</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">129</span>–<span class="cit-lpage">150</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0021859603003058</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BSci.%26rft.volume%253D140%26rft.spage%253D129%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0021859603003058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0021859603003058&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-108-1" title="View reference  in text" id="ref-108">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.108" data-doi="10.1080/00288230709510292"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Easton</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Jones</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Future options and targets for pasture plant breeding in New Zealand.</span> <abbr class="cit-jnl-abbrev">N. Z. J. Agric. Res.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">223</span>–<span class="cit-lpage">248</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/00288230709510292</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BZ.%2BJ.%2BAgric.%2BRes.%26rft.volume%253D50%26rft.spage%253D223%26rft_id%253Dinfo%253Adoi%252F10.1080%252F00288230709510292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/00288230709510292&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-109-1" title="View reference  in text" id="ref-109">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.109" data-doi="10.1007/s00122-011-1745-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Gowda</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Würschum</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. P.</span> <span class="cit-name-surname">Maurer</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Accuracy of genomic selection in European maize elite breeding populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">769</span>–<span class="cit-lpage">776</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-011-1745-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DZhao%26rft.auinit1%253DY.%26rft.volume%253D124%26rft.issue%253D4%26rft.spage%253D769%26rft.epage%253D776%26rft.atitle%253DAccuracy%2Bof%2Bgenomic%2Bselection%2Bin%2BEuropean%2Bmaize%2Belite%2Bbreeding%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1745-y%26rft_id%253Dinfo%253Apmid%252F22075809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1745-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22075809&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-110-1" title="View reference  in text" id="ref-110">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.695.110" data-doi="10.1534/genetics.108.098277"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Dekkers</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Factors affecting accuracy from genomic selection in populations derived from multiple inbred lines: a barley case study.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">364</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.108.098277</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.098277%26rft_id%253Dinfo%253Apmid%252F19299342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzU1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi82OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/2/695.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/2/677" title="Contingent Convergence: The Ability To Detect Convergent Genomic Evolution Is Dependent on Population Size and Migration" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/2/709" title="Horizontal Transfer and Gene Loss Shaped the Evolution of Alpha-Amylases in Bilaterians" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/2"><span>Volume 10 Issue 2, February 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/2.cover-source.jpg?itok=LSawHOSr" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/695" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9oAZ_mb0J0GRofJw3MhoBAqRdUNEJcXiNo3oFncGNK8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/2/695" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/2/695" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/2/695" class="active" data-icon-position="" data-hide-link-title="0">Genomic Predictive Ability for Foliar Nutritive Traits in Perennial Ryegrass</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495555" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-lsfxymjxHWbyYjcSrOLwiK7DMt5nClPTZBglTyjyHNg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/2/695.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/2/695" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-fq1wIaHQaFh7Xih43l9bSSQco-Ew7ufhQSBh3xDFkLU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/2/695" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/2/695.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495555" id="node495555--2" data-pisa="ggg;10/2/695" data-pisa-master="ggg;g3.119.400880" data-apath="/ggg/10/2/695.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Predictive Ability for Foliar Nutritive Traits in Perennial Ryegrass</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4769-9782" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sai Krishna Arojju</span>, <span class="highwire-citation-author" data-delta="1">Mingshu Cao</span>, <span class="highwire-citation-author" data-delta="2">M. Z. Zulfi Jahufer</span>, <span class="highwire-citation-author" data-delta="3">Brent A. Barrett</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-3129-6540" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marty J. Faville</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">695-708;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400880 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495555--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sai Krishna Arojju </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch Ltd, Grasslands Research Centre, PB 11008, Palmerston North, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sai%2BKrishna%2BArojju%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arojju%20SK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASai%2BKrishna%2BArojju%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4769-9782" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sai Krishna Arojju</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sai.arojju@agresearch.co.nz" class="" data-icon-position="" data-hide-link-title="0">sai.arojju@agresearch.co.nz</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mingshu Cao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch Ltd, Grasslands Research Centre, PB 11008, Palmerston North, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mingshu%2BCao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cao%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMingshu%2BCao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">M. Z. Zulfi Jahufer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch Ltd, Grasslands Research Centre, PB 11008, Palmerston North, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BZ.%2BZulfi%2BJahufer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zulfi%20Jahufer%20MZ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM.%2BZ.%2BZulfi%2BJahufer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Brent A. Barrett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch Ltd, Grasslands Research Centre, PB 11008, Palmerston North, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brent%2BA.%2BBarrett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barrett%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrent%2BA.%2BBarrett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Marty J. Faville </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch Ltd, Grasslands Research Centre, PB 11008, Palmerston North, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marty%2BJ.%2BFaville%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Faville%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarty%2BJ.%2BFaville%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3129-6540" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marty J. Faville</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/2/695" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/2/695&amp;title=Genomic%20Predictive%20Ability%20for%20Foliar%20Nutritive%20Traits%20in%20Perennial%20Ryegrass" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/2/695&amp;title=Genomic%20Predictive%20Ability%20for%20Foliar%20Nutritive%20Traits%20in%20Perennial%20Ryegrass" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/2/695&amp;title=Genomic%20Predictive%20Ability%20for%20Foliar%20Nutritive%20Traits%20in%20Perennial%20Ryegrass" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/2/695&amp;text=Genomic%20Predictive%20Ability%20for%20Foliar%20Nutritive%20Traits%20in%20Perennial%20Ryegrass" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/2/695&amp;title=Genomic%20Predictive%20Ability%20for%20Foliar%20Nutritive%20Traits%20in%20Perennial%20Ryegrass" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/2/695&amp;t=Genomic%20Predictive%20Ability%20for%20Foliar%20Nutritive%20Traits%20in%20Perennial%20Ryegrass" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/2/695&amp;title=Genomic%20Predictive%20Ability%20for%20Foliar%20Nutritive%20Traits%20in%20Perennial%20Ryegrass" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/2/695&amp;title=Genomic%20Predictive%20Ability%20for%20Foliar%20Nutritive%20Traits%20in%20Perennial%20Ryegrass" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495555/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495555" id="node495555--3" data-pisa="ggg;10/2/695" data-pisa-master="ggg;g3.119.400880" data-apath="/ggg/10/2/695.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Predictive Ability for Foliar Nutritive Traits in Perennial Ryegrass</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4769-9782" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sai Krishna Arojju</span>, <span class="highwire-citation-author" data-delta="1">Mingshu Cao</span>, <span class="highwire-citation-author" data-delta="2">M. Z. Zulfi Jahufer</span>, <span class="highwire-citation-author" data-delta="3">Brent A. Barrett</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-3129-6540" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marty J. Faville</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">695-708;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400880 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495555--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sai Krishna Arojju </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch Ltd, Grasslands Research Centre, PB 11008, Palmerston North, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sai%2BKrishna%2BArojju%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arojju%20SK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASai%2BKrishna%2BArojju%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4769-9782" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sai Krishna Arojju</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sai.arojju@agresearch.co.nz" class="" data-icon-position="" data-hide-link-title="0">sai.arojju@agresearch.co.nz</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mingshu Cao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch Ltd, Grasslands Research Centre, PB 11008, Palmerston North, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mingshu%2BCao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cao%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMingshu%2BCao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">M. Z. Zulfi Jahufer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch Ltd, Grasslands Research Centre, PB 11008, Palmerston North, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BZ.%2BZulfi%2BJahufer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zulfi%20Jahufer%20MZ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM.%2BZ.%2BZulfi%2BJahufer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Brent A. Barrett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch Ltd, Grasslands Research Centre, PB 11008, Palmerston North, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brent%2BA.%2BBarrett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barrett%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrent%2BA.%2BBarrett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Marty J. Faville </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch Ltd, Grasslands Research Centre, PB 11008, Palmerston North, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marty%2BJ.%2BFaville%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Faville%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarty%2BJ.%2BFaville%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3129-6540" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marty J. Faville</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=695&amp;rft.epage=708&amp;rft.atitle=Genomic%20Predictive%20Ability%20for%20Foliar%20Nutritive%20Traits%20in%20Perennial%20Ryegrass&amp;rft.volume=10&amp;rft.issue=2&amp;rft.date=2020-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Arojju%2C+Sai+Krishna&amp;rft.au=Cao%2C+Mingshu&amp;rft.au=Zulfi+Jahufer%2C+M.+Z.&amp;rft.au=Barrett%2C+Brent+A.&amp;rft.au=Faville%2C+Marty+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495555/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495555/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495555/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495555/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495555/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495555/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495555/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495555/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495555/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495555/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495555/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495555/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/2/695" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/2/695.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/2/695.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/2/695.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
