<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/2/581" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/2/581.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/2/581.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/2/581.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/10/2/581/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/2/581.atom" />
<meta name="HW.pisa" content="ggg;10/2/581" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Optimizing Low-Cost Genotyping and Imputation Strategies for Genomic Selection in Atlantic Salmon" />
<meta name="DC.Identifier" content="10.1534/g3.119.400800" />
<meta name="DC.Date" content="2020-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Tsairidou et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genomic selection enables cumulative genetic gains in key production traits such as disease resistance, playing an important role in the economic and environmental sustainability of aquaculture production. However, it requires genome-wide genetic marker data on large populations, which can be prohibitively expensive. Genotype imputation is a cost-effective method for obtaining high-density genotypes, but its value in aquaculture breeding programs which are characterized by large full-sibling families has yet to be fully assessed. The aim of this study was to optimize the use of low-density genotypes and evaluate genotype imputation strategies for cost-effective genomic prediction. Phenotypes and genotypes (78,362 SNPs) were obtained for 610 individuals from a Scottish Atlantic salmon breeding program population (Landcatch, UK) challenged with sea lice, Lepeophtheirus salmonis . The genomic prediction accuracy of genomic selection was calculated using GBLUP approaches and compared across SNP panels of varying densities and composition, with and without imputation. Imputation was tested when parents were genotyped for the optimal SNP panel, and offspring were genotyped for a range of lower density imputation panels. Reducing SNP density had little impact on prediction accuracy until 5,000 SNPs, below which the accuracy dropped. Imputation accuracy increased with increasing imputation panel density. Genomic prediction accuracy when offspring were genotyped for just 200 SNPs, and parents for 5,000 SNPs, was 0.53. This accuracy was similar to the full high density and optimal density dataset, and markedly higher than using 200 SNPs without imputation. These results suggest that imputation from very low to medium density can be a cost-effective tool for genomic selection in Atlantic salmon breeding programs." />
<meta name="DC.Contributor" content="Smaragda Tsairidou" />
<meta name="DC.Contributor" content="Alastair Hamilton" />
<meta name="DC.Contributor" content="Diego Robledo" />
<meta name="DC.Contributor" content="James E. Bron" />
<meta name="DC.Contributor" content="Ross D. Houston" />
<meta name="article:published_time" content="2020-02-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Optimizing Low-Cost Genotyping and Imputation Strategies for Genomic Selection in Atlantic Salmon" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomic selection enables cumulative genetic gains in key production traits such as disease resistance, playing an important role in the economic and environmental sustainability of aquaculture production. However, it requires genome-wide genetic marker data on large populations, which can be prohibitively expensive. Genotype imputation is a cost-effective method for obtaining high-density genotypes, but its value in aquaculture breeding programs which are characterized by large full-sibling families has yet to be fully assessed. The aim of this study was to optimize the use of low-density genotypes and evaluate genotype imputation strategies for cost-effective genomic prediction. Phenotypes and genotypes (78,362 SNPs) were obtained for 610 individuals from a Scottish Atlantic salmon breeding program population (Landcatch, UK) challenged with sea lice, &lt;i&gt;Lepeophtheirus salmonis&lt;/i&gt;. The genomic prediction accuracy of genomic selection was calculated using GBLUP approaches and compared across SNP panels of varying densities and composition, with and without imputation. Imputation was tested when parents were genotyped for the optimal SNP panel, and offspring were genotyped for a range of lower density imputation panels. Reducing SNP density had little impact on prediction accuracy until 5,000 SNPs, below which the accuracy dropped. Imputation accuracy increased with increasing imputation panel density. Genomic prediction accuracy when offspring were genotyped for just 200 SNPs, and parents for 5,000 SNPs, was 0.53. This accuracy was similar to the full high density and optimal density dataset, and markedly higher than using 200 SNPs without imputation. These results suggest that imputation from very low to medium density can be a cost-effective tool for genomic selection in Atlantic salmon breeding programs.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/02/01" />
<meta name="citation_mjid" content="ggg;10/2/581" />
<meta name="citation_id" content="10/2/581" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/2/581" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/2/581.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/2/581.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/2/581.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400800" />
<meta name="citation_pmid" content="31826882" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="581" />
<meta name="citation_lastpage" content="590" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Smaragda Tsairidou" />
<meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Edinburgh, EH25 9RG, United Kingdom," />
<meta name="citation_author_email" content="Smaragda.Tsairidou@roslin.ed.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0792-9421" />
<meta name="citation_author" content="Alastair Hamilton" />
<meta name="citation_author_institution" content="Hendrix Genetics Aquaculture BV/ Netherlands Villa ’de Körver’, Spoorstraat 695831 CK Boxmeer, The Netherlands, and" />
<meta name="citation_author" content="Diego Robledo" />
<meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Edinburgh, EH25 9RG, United Kingdom," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9616-5912" />
<meta name="citation_author" content="James E. Bron" />
<meta name="citation_author_institution" content="Institute of Aquaculture, University of Stirling, FK9 4LA, United Kingdom" />
<meta name="citation_author" content="Ross D. Houston" />
<meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Edinburgh, EH25 9RG, United Kingdom," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1805-0762" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. Amin;citation_author=C. M. Van Duijn;citation_author=Y. S. Aulchenko;citation_title=A genomic background based method for association analysis in related individuals.;citation_pages=e1274-e1274;citation_volume=2;citation_year=2007;citation_issue=12;citation_pmid=18060068;citation_doi=10.1371/journal.pone.0001274" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Y. S. Aulchenko;citation_author=S. Ripke;citation_author=A. Isaacs;citation_author=C. M. Van Duijn;citation_title=GenABEL: an R library for genome-wide association analysis.;citation_volume=23;citation_year=2007;citation_pmid=17384015;citation_doi=10.1093/bioinformatics/btm108" />
<meta name="citation_reference" content="citation_title=A Colour Atlas of Salmonid Diseases Springer Netherlands;citation_author=D. W. Bruno;citation_author=P. A. Noguera;citation_author=T. T. Poppe;citation_year=2013;citation_doi=10.1007/978-94-007-2010-7" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=P. Cáceres;citation_author=A. Barría;citation_author=K. A. Christensen;citation_author=L. N. Bassini;citation_author=K. Correa;citation_title=Genome-scale comparative analysis for host resistance against sea lice between Atlantic salmon and rainbow trout.;citation_year=2019;citation_doi=10.1101/624031" />
<meta name="citation_reference" content="citation_journal_title=Journal of fish diseases;citation_journal_abbrev=J Fish Dis;citation_author=M. J. Costello;citation_title=The global economic cost of sea lice to the salmonid farming industry.;citation_pages=115-118;citation_volume=32;citation_year=2009;citation_issue=1;citation_pmid=19245636;citation_doi=10.1111/j.1365-2761.2008.01011.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=H. D. Daetwyler;citation_author=B. Villanueva;citation_author=J. A. Woolliams;citation_title=Accuracy of predicting the genetic risk of disease using a genome-wide approach.;citation_pages=e3395-e3395;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852893;citation_doi=10.1371/journal.pone.0003395" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=P. Dufflocq;citation_author=M. Pérez-Enciso;citation_author=J. P. Lhorente;citation_author=J. M. Yáñez;citation_title=Accuracy of genomic predictions using different imputation error rates in aquaculture breeding programs: A simulation study.;citation_pages=225-230;citation_volume=503;citation_year=2019;citation_doi=10.1016/j.aquaculture.2018.12.061" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. M. Edwards;citation_author=I. F. Sørensen;citation_author=P. Sarup;citation_author=T. F. C. Mackay;citation_author=P. Sørensen;citation_title=Genomic Prediction for Quantitative Traits Is Improved by Mapping Variants to Gene Ontology Categories in Drosophila melanogaster.;citation_volume=203;citation_year=2016;citation_pmid=27235308;citation_doi=10.1534/genetics.116.187161" />
<meta name="citation_reference" content="Gilmour, A., B. Gogel, B. Cullis, and Thompson, 2009 R. ASReml User Guide Release 3.0. VSN International Ltd. Hemel Hempstead, HP1 1ES, UK." />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=B. Gjerde;citation_author=J. Ødegård;citation_author=I. Thorland;citation_title=Estimates of genetic variation in the susceptibility of Atlantic salmon (Salmo salar) to the salmon louse Lepeophtheirus salmonis.;citation_pages=66-72;citation_volume=314;citation_year=2011;citation_doi=10.1016/j.aquaculture.2011.01.026" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=H. D. Daetwyler;citation_author=J. W. Kijas;citation_author=J. H. J. Van Der Werf;citation_title=Accuracy of genotype imputation in sheep breeds.;citation_pages=72-80;citation_volume=43;citation_year=2012;citation_issue=1;citation_pmid=22221027;citation_doi=10.1111/j.1365-2052.2011.02208.x" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=R. D. Houston;citation_author=J. B. Taggart;citation_author=T. Cezard;citation_author=M. Bekaert;citation_author=N. R. Lowe;citation_title=Development and validation of a high density SNP genotyping array for Atlantic salmon (Salmo salar).;citation_pages=90;citation_volume=15;citation_year=2014;citation_pmid=24524230;citation_doi=10.1186/1471-2164-15-90" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=Y. Li;citation_author=C. Willer;citation_author=S. Sanna;citation_author=G. Abecasis;citation_title=Genotype imputation.;citation_pages=387-406;citation_volume=10;citation_year=2009;citation_pmid=19715440;citation_doi=10.1146/annurev.genom.9.081307.164242" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=S. Lien;citation_author=B. F. Koop;citation_author=S. R. Sandve;citation_author=J. R. Miller;citation_author=M. P. Kent;citation_title=The Atlantic salmon genome provides insights into rediploidization.;citation_volume=533;citation_year=2016;citation_pmid=27088604;citation_doi=10.1038/nature17164" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=C. Palaiokostas;citation_author=M. Kocour;citation_author=M. Prchal;citation_author=R. D. Houston;citation_title=Accuracy of Genomic Evaluations of Juvenile Growth Rate in Common Carp (Cyprinus carpio) Using Genotyping by Sequencing.;citation_pages=82;citation_volume=9;citation_year=2018;citation_doi=10.3389/fgene.2018.00082" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=C. Palaiokostas;citation_author=T. Vesely;citation_author=M. Kocour;citation_author=M. Prchal;citation_author=D. Pokorova;citation_title=Optimizing Genomic Prediction of Host Resistance to Koi Herpesvirus Disease in Carp.;citation_pages=543;citation_volume=10;citation_year=2019;citation_doi=10.3389/fgene.2019.00543" />
<meta name="citation_reference" content="citation_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Purcell;citation_author=B. Neale;citation_author=K. Todd-Brown;citation_author=L. Thomas;citation_author=M. A. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_title=R: A language and environment for statistical computing.;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. Robledo;citation_author=O. Matika;citation_author=A. Hamilton;citation_author=R. D. Houston;citation_title=Genome-Wide Association and Genomic Selection for Resistance to Amoebic Gill Disease in Atlantic Salmon.;citation_volume=8;citation_year=2018;citation_pmid=29420190;citation_doi=10.1534/g3.118.200075" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=M. Sargolzaei;citation_author=J. P. Chesnais;citation_author=F. S. Schenkel;citation_title=A new approach for efficient genotype imputation using information from relatives.;citation_pages=478;citation_volume=15;citation_year=2014;citation_doi=10.1186/1471-2164-15-478" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=M. Scutari;citation_author=I. Mackay;citation_author=D. Balding;citation_title=Using Genetic Distance to Infer the Accuracy of Genomic Prediction.;citation_pages=e1006288;citation_volume=12;citation_year=2016;citation_doi=10.1371/journal.pgen.1006288" />
<meta name="citation_reference" content="Sevatdal, S. and T. Horsberg, 2003 Determination of reduced sensitivity in sea lice (Lepeophtheirus salmonis KrØyer) against the pyrethroid deltamethrin using bioassays and probit modelling." />
<meta name="citation_reference" content="citation_journal_title=J Anim Sci.;citation_author=W. M. Snelling;citation_author=R. A. Cushman;citation_author=J. W. Keele;citation_author=C. Maltecca;citation_author=M. G. Thomas;citation_title=Breeding and Genetics Symposium: networks and pathways to guide genomic selection.;citation_volume=91;citation_year=2013;citation_pmid=23097404;citation_doi=10.2527/jas.2012-5784" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=H.-Y. Tsai;citation_author=A. Hamilton;citation_author=A. E. Tinch;citation_author=D. R. Guy;citation_author=J. E. Bron;citation_title=Genomic prediction of host resistance to sea lice in farmed Atlantic salmon populations.;citation_pages=47;citation_volume=48;citation_year=2016;citation_doi=10.1186/s12711-016-0226-9" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=H.-Y. Tsai;citation_author=A. Hamilton;citation_author=A. E. Tinch;citation_author=D. R. Guy;citation_author=K. Gharbi;citation_title=Genome wide association and genomic prediction for growth traits in juvenile farmed Atlantic salmon using a high density SNP array.;citation_pages=969;citation_volume=16;citation_year=2015;citation_pmid=26582102;citation_doi=10.1186/s12864-015-2117-9" />
<meta name="citation_reference" content="citation_year=2017" />
<meta name="citation_reference" content="Tsairidou, S., 2019 CVrepGPAcalc. [Online], Available: https://github.com/SmaragdaT/CVrep/tree/master/CVrepGPAcalc." />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=S. Tsairidou;citation_author=J. A. Woolliams;citation_author=A. R. Allen;citation_author=R. A. Skuce;citation_author=S. H. Mcbride;citation_title=Genomic prediction for tuberculosis resistance in dairy cattle.;citation_pages=e96728;citation_volume=9;citation_year=2014;citation_doi=10.1371/journal.pone.0096728" />
<meta name="citation_reference" content="citation_journal_title=Aquacult. Res.;citation_author=O. Tully;citation_author=Y. Mcfadden;citation_title=Variation in sensitivity of sea lice [Lepeophtheirus salmonis (Krøyer)] to dichlorvos on Irish salmon farms in 1991–92.;citation_pages=849-854;citation_volume=31;citation_year=2000;citation_doi=10.1046/j.1365-2109.2000.00518.x" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=R. L. Vallejo;citation_author=T. D. Leeds;citation_author=G. Gao;citation_author=J. E. Parsons;citation_author=K. E. Martin;citation_title=Genomic selection models double the accuracy of predicted breeding values for bacterial cold water disease resistance compared to a traditional pedigree-based model in rainbow trout aquaculture.;citation_pages=17;citation_volume=49;citation_year=2017;citation_doi=10.1186/s12711-017-0293-6" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=G. M. Yoshida;citation_author=R. Carvalheiro;citation_author=J. P. Lhorente;citation_author=K. Correa;citation_author=R. Figueroa;citation_title=Accuracy of genotype imputation and genomic predictions in a two-generation farmed Atlantic salmon population using high-density and low-density SNP panels.;citation_pages=147-154;citation_volume=491;citation_year=2018;citation_doi=10.1016/j.aquaculture.2018.03.004" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=G. M. Yoshida;citation_author=J. P. Lhorente;citation_author=K. Correa;citation_author=J. Soto;citation_author=D. Salas;citation_title=Genome-Wide Association Study and Cost-Efficient Genomic Predictions for Growth and Fillet Yield in Nile Tilapia (Oreochromis niloticus).;citation_volume=9;citation_year=2019;citation_pmid=31171566;citation_doi=10.1534/g3.119.400116" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=K. R. Zenger;citation_author=M. S. Khatkar;citation_author=D. B. Jones;citation_author=N. Khalilisamani;citation_author=D. R. Jerry;citation_title=Genomic Selection in Aquaculture: Application, Limitations and Opportunities With Special Reference to Marine Shrimp and Pearl Oysters.;citation_pages=693;citation_volume=9;citation_year=2019;citation_doi=10.3389/fgene.2018.00693" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Optimizing Low-Cost Genotyping and Imputation Strategies for Genomic Selection in Atlantic Salmon" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
<meta name="twitter:description" content="Genomic selection enables cumulative genetic gains in key production traits such as disease resistance, playing an important role in the economic and environmental sustainability of aquaculture production. However, it requires genome-wide genetic marker data on large populations, which can be prohibitively expensive. Genotype imputation is a cost-effective method for obtaining high-density genotypes, but its value in aquaculture breeding programs which are characterized by large full-sibling families has yet to be fully assessed. The aim of this study was to optimize the use of low-density genotypes and evaluate genotype imputation strategies for cost-effective genomic prediction. Phenotypes and genotypes (78,362 SNPs) were obtained for 610 individuals from a Scottish Atlantic salmon breeding program population (Landcatch, UK) challenged with sea lice, Lepeophtheirus salmonis . The genomic prediction accuracy of genomic selection was calculated using GBLUP approaches and compared across SNP panels of varying densities and composition, with and without imputation. Imputation was tested when parents were genotyped for the optimal SNP panel, and offspring were genotyped for a range of lower density imputation panels. Reducing SNP density had little impact on prediction accuracy until 5,000 SNPs, below which the accuracy dropped. Imputation accuracy increased with increasing imputation panel density. Genomic prediction accuracy when offspring were genotyped for just 200 SNPs, and parents for 5,000 SNPs, was 0.53. This accuracy was similar to the full high density and optimal density dataset, and markedly higher than using 200 SNPs without imputation. These results suggest that imputation from very low to medium density can be a cost-effective tool for genomic selection in Atlantic salmon breeding programs." />
<meta name="og-title" property="og:title" content="Optimizing Low-Cost Genotyping and Imputation Strategies for Genomic Selection in Atlantic Salmon" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/2/581" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomic selection enables cumulative genetic gains in key production traits such as disease resistance, playing an important role in the economic and environmental sustainability of aquaculture production. However, it requires genome-wide genetic marker data on large populations, which can be prohibitively expensive. Genotype imputation is a cost-effective method for obtaining high-density genotypes, but its value in aquaculture breeding programs which are characterized by large full-sibling families has yet to be fully assessed. The aim of this study was to optimize the use of low-density genotypes and evaluate genotype imputation strategies for cost-effective genomic prediction. Phenotypes and genotypes (78,362 SNPs) were obtained for 610 individuals from a Scottish Atlantic salmon breeding program population (Landcatch, UK) challenged with sea lice, Lepeophtheirus salmonis . The genomic prediction accuracy of genomic selection was calculated using GBLUP approaches and compared across SNP panels of varying densities and composition, with and without imputation. Imputation was tested when parents were genotyped for the optimal SNP panel, and offspring were genotyped for a range of lower density imputation panels. Reducing SNP density had little impact on prediction accuracy until 5,000 SNPs, below which the accuracy dropped. Imputation accuracy increased with increasing imputation panel density. Genomic prediction accuracy when offspring were genotyped for just 200 SNPs, and parents for 5,000 SNPs, was 0.53. This accuracy was similar to the full high density and optimal density dataset, and markedly higher than using 200 SNPs without imputation. These results suggest that imputation from very low to medium density can be a cost-effective tool for genomic selection in Atlantic salmon breeding programs." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
    <title>Optimizing Low-Cost Genotyping and Imputation Strategies for Genomic Selection in Atlantic Salmon | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"JvoFIZSI9Kg04Nb6YKZSQ5LNjnhjpGRKdQR44K6S9zc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495563","apath":"\/ggg\/10\/2\/581.atom","pisa":"ggg;10\/2\/581","ac":{"\/ggg\/10\/2\/581.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/2\/581.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/2\/581"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/2\/581"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/2\/581":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00226I9tmcYSvDuk\\\\\\\/Ot9R3faz6HxWhrYlBPA+\\\\\\\/Ln6P7k9fAoOfDDQZ4pymSb2d0VxE9+JUffxZHrhErVnxZBEgtKVDaQcm91hwpBfN8qepNCAEgNu\\\\\\\/O3niUaC37lbQ65GjmYWlpNQH\\\\\\\/BaPHLGi9XA9HKmSquAtDAwU01ha5qKaCgicJ3SfagRO\\\\\\\/MtMOg\\\\\\\/ickp9ytY3lGG930Pz9qJl0JvCdJckG9DgUXkwK\\\\\\\/RcMDg0wI58l9F5Yy5f+TnHtB5d9d8vW3NogGag5j3K0cYc64pnCCZMpg1exmEqa1H2weDvURZHFDLrPq2Dihut9k1XQJCH3nxUqrmOyCBzWg\\\\\\\/YHaKjGmAVNX7ynpHm6Yz5e8QEdxsxI2LcXIGyUJm2SC4bqvDmu9dGSOaxSWhTi5xoC8WKsehHzH+bwta1IWL\\\\\\\/PxV0c\\\\\\\/FcTkObiA7SoJ5ZpRsJz\\\\\\\/256BVM6rOWHbFr+0smRuUUjzKtVROa5P2JHo4klHp47leaBJWOVh7IlV9DgHbG\\\\\\\/ywFWWfqOiimYY8U6mfg9eXaIrfkDynjNw+5JuUICk3au1IipEEiQMjiQ+CzaFL6tr9S\\\\\\\/0\\\\\\\/H7M2aRQ0x4Zs0IzNsJ9sb1EvkS+QO2fLnaYfga4g+CYEEhzeH2aIdUjTfrYGsrvvBlq32I\\\\\\\/HC0RGv41PiarTpgT5x5WqZC8FYCHtTVq8H+F5oSEk0oHpbVx20A82DvbW0w3aFZD4w5RkGXWzjQ8XOA0TAbe9cpKTzke8HsjydpBjnJ21Xo+SUpyxJQFszeRHG3eT06fBZ8ZlinlCJWOKKxWjqK7UXH7NN3PfvTZdJmEZfwLnx1mwEltADUDTsibjd3FOS2kNEZmdYkNOraqNHmnQeiWz+AA9zWsqP18n5r3q2eeod0W9ACxt2jLaT48+\\\\\\\/zDDpzmBAvU0IWfeGqEJKz688eNruwDwbylzVWdxUDQR4Su1rXxRvcHEFqENiDTAwopwzVTJhGQN4qVO69YWrW3wPV6PKQlBxr5EDZaeFVUdUlElQR02njXMp+sLWUl1ONdbrbJ7ajyBNxU6UqkovHFLn+EyxX8bU7sh6Mmq1RirQOTFFJbu+IPso8kZht4+fwJRnDGt4tUIMnzhI05nxI2SQN8f10By8oe0cj58cCwUwBNEecZ+4Ux1w3K5Dcae0ZvY5\\\\\\\/0ukyQLSsGW9Wq8yCUbjmXNY8S83N8EtYlS0g9uqTd2YWb7swtUvY5LAPoyvkmvgyItgU\\\\\\\/\\\\\\\/9Q6l7EzX1WS6DO51fNte5P1vJ\\\\\\\/7Vnw3jm1q\\\\\\\/3qFt07i+H\\\\\\\/65Dy4\\\u0022,\\\u0022iv\\\u0022:\\\u0022cPm7a23wvqwX\\\\\\\/Y6kS4thqQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b59b6eac9242c6a038a0fcc2f32368754a4071c40b9c13b38bcb3e1b7f23551b\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022X3Al1CXQiYW\\\\\\\/cJG7X7PKlxQ3ISASr19h\\\\\\\/hYd2ebO3guytzuubTo4zu2xgNomx\\\\\\\/gn1\\\\\\\/M\\\\\\\/LzgKU\\\\\\\/tVvYb5wI10LVh3MfcJZMEYPgJffm908ruFD6AE3sx5UsCqCd\\\\\\\/7ednedMyRIhU+wMxSL5B0mMAzyhzzqbsZsT3gcjaFtMcalHzEg5S+z08KQsWf7TqZmdPbvydxOz6R\\\\\\\/0C+Xiy6LpulnP3F+jrzn7WmNXAh7ux9r5UKOlK6GaqOWHueKJemOh41zk1DTrlYBgMpOQmXs+s+hPE0ooDsbZd+YB\\\\\\\/84SeQUT9WT+aK4DvZ1nPY8rs+U4ILWw5rHUulLds8orRdmSFv4fHqCPYH8HcQWLoaDKODvqRR029l3+nQhpoRp16+OUOo+rQUwZkZiiPJ9IyDqnaDDQVFuIlk4Kzr2v6eqpANMmKthx0c\\\\\\\/vamF5KfLKi0GbD7q0uuogdWTIb1fG9uDok5wJnVsyD1aKu\\\\\\\/+DSfsd1m9Szz+EITECUJ7RQfqCcbtTuWMGpS5uet9neI\\\\\\\/4Zbsun\\\\\\\/nL69tPSImbnCFLuv7lu+Bj9YHOXK00SpchX6CYtZ6tHWZM+HyXPbeirT2FE7\\\\\\\/rl2xelCADHmDVM4fottAl8j0sdDeNZwm5zTWa7NMKfe3n6nylJdU3sgqHMwl9v8OpCdmqgHNY6BRuY6AHNvkjPD561LijtxYq8ES+yxlQtAl4\\\\\\\/KaE8nkzpEWKX2jEvH5BiBYf0KCcEjHFqpUWt+m7pfVleIXPb2SGUMvy0iZXZ9UupDA8M0BxhCwo3ZdloLnjHRr8uOh0JvlxgyiOQPZTqy7QvBGh9fEl06N0VetBBEveev7qp3R4cHJYGUvhUY7xGdHcJHktjbdJQqSj+3FfXVqQgoVv4A3exrxWKeJNUBCOzu9sWpYW2di7lN+O9kuFoz3zYL3r1XFjJit4P\\\\\\\/NbEgFjPfy0GIn0ASNmwlpnMCFCbnnQNB5764WGGQCImlE1YLGZ4chLWnIQ6k8hpkaZNi3jSVUebmWBM7pyN7oFx5ROcq+ovEsFQ8QU3zCEsfeXTBrubT1ntRp6nRgV45oZ5wFpQIZN\\\\\\\/\\\\\\\/Pmy2Zzvb+nZea9CDJnLkKcrE0SOVcCWX5vvYXiXhhuK2HhmWxwofAWzksh2JQcFIr0sMUlR3dMJHP8q3m\\\\\\\/WbkxbOzeJhMbbL2fzdvPdLadKlqjMKE9bolHnMLsNOP+J1JLBRc\\\\\\\/QzPjmEv7\\\\\\\/T\\\\\\\/vN0HgFZtrlIHXJT5OOZNaJ0VizyF\\\\\\\/OgLIAIAaIzcJqkW\\\\\\\/VutqMe9MCTVnlKsAdGR6+ZnT5Bbe1qei8L8J9KMi8FHhPRutoW8QHxDSdyrBEaDHXUQBPHxvYwHjxRXjlU4KUKLbkhRiaq0w11gC3KNQetHcRZvYgltTm6qC0Oe48GtXJdMRo6VoLAd6+\\\\\\\/uf5s5bvh\\\\\\\/bBgbNTCkdkkjDtKdMEOMdph1aXmPuaEj5ZC9OVnTvuAkMQuLueCKpGUlD43uStzy4puLKGTNFW37jnoLP09VbTAO28YcIFMl928LVAD3q8l4rJlsSfYj1X0J9ASLTE2VP34\\\\\\\/vCc1F6kf3GrhmO597t9y2rxfhG6qxdpirT015GB8U7u\\\\\\\/dGGZeoy9cHxcQeGVpon3Be+ElsNo2J1Hvh\\\\\\\/pXxlEObvCQBHp9D+VkMwfeftUeR21f2CHDMNayQRjkw\\\\\\\/3svHXTLWHnn\\\\\\\/Iurap8Y1rvGUSaZpupB4Vj6Y0apAVIvKQB8VtvO1vL03FnyVKqcSdajmoNyb3RzZm0itrNFI6CO43J6Hs1v7FODY3tRDaBpSxvO2Q+ZVKQ9qTFrdS4wiir0m+wygw\\\\\\\/Nj8duLPziYo\\\\\\\/i8ZvEdL7ukeataJgINDpewm9SKq9\\\\\\\/H2Df8w4\\\\\\\/sWoU4kfP1XYyACwxXHyFSkwvW8Ud8AORDO2N6u77y0OTnr2DFNkHuVI8DXabv7V41HQmNWkSZRZ2XT6xRHObdZaj56go2oXe+AodBLZgz07rATKYceyDCINH3dZh1PwZ\\\\\\\/pkP0dYLnKPmMEQ9tR39hz7Vo2MMyB2sSK1w7xHHcbqXu6x8cPXPFdpeoThN03kcHnUHhNwGxf8EmQwoj73lOGz9s7tYC\\\\\\\/nhqsyZQ5OmpxQZ+LnSZ+YmGEtSh9TyK0Mvd5apYel29zHkoHB\\\\\\\/k7+XPAienjJvJIJwsA2r4euYpzToMdr5HWKWVPkgu\\\\\\\/XGddc4vmhjO5EoUpljv++ltQdJlGXPOV0xgMm5j4iiIy\\\\\\\/jIhNm9nxMWzB4ShEUGwqODlQHJj8K4tQID\\\\\\\/nJSiouicf4\\\\\\\/sY\\\\\\\/0xSWO2A4\\\\\\\/5x2VohSusea+mG5nJsK5Jq1fvT6z78U4O5jX8e\\\\\\\/WxNdWWJ5FmYxeymF+h3Dv\\\\\\\/jptmah86Voy4BOWTEmideI3CiM3g0gBhVDCT+S5XrrKgqWwS3EavUzjeN3JDzTFW5ni0uoSXgWN\\\\\\\/FQeuFPW1Qe+UFLxJ\\\\\\\/xfhRfj7dOKVwn6StYSJLg\\\\\\\/hfcPEi9HRQ7ugv7mvyp1\\\\\\\/C8YNky87QjKLdXiFL0is1wbB09W5b3CDUaaadW+Pt\\\\\\\/rEC6qOhwwZHGUsIGb10JXsw8adxVzy\\\\\\\/cRrYG9\\\\\\\/dCtV4tSQ70RqJKTf0yqwBvSBse5EN+aVa4aEFhmjR6culBlEUDkXYxSlitaWajJxUkdPGDz4wgwoStdX0\\\\\\\/8kp3zd+b4niswnncQ\\\\\\\/6rey3E7PeN2zduOBo0q\\\\\\\/TKXSpFUc55J8TATiGwK3OlUHU53LcFtEZPn63aHLRp3nN0XYel0DX0Lq22SjXnZ4BWHBgsnFolyF8wDC2dcdxxhqrTMbeLFAbRkUQaqb7RSnSmPTvt3EmRh6gXDp3lWwwOZ6vQoSAOrFNXPMv2FMMjocHbC5H3tYYCg5zDClueE+NfKEW2esVxjFjlVeWUeaDs588SN4yn+b6GAn5y\\\\\\\/WUgvMi7isTpvNKcg2t1ReZvVnqZeUlcRu8tdMl8rIR6im7i2oPyekn49qQi+BNqiYDAgBAmoz8+sYZ5vRMo1Ea2t9Nvv\\\\\\\/UWOz8hj6+1PJA8qg1OaedLV0hBb0T5qMHt555mxdndxifBFtwdmQvLY33Y5mjDxn8p1QdZAbf4zwF1nzKa39mQM0694F5Jw\\\\\\\/jQ+ymziGf\\\\\\\/u15ZdBt1VjdURy+4+F+q8H4RE4XdHcwV1oBtxo1g1ygF6sACJnzMQj53l+GhealxauuDjSQT0713xOHSZqdriOIohUEPsq1dlTSVv2gDEoYMA+UkWZ9SIWLlGqqNzar+edp9186BUUdUrQqZA4Wz2Vm0WzXLFTZnh8qEJQBgd\\\\\\\/JeJRVVrU6ioNdn1YSsTTBl8t+is26pdBpGb\\\\\\\/uKgsZBizoQNFAUI8+pGEMOC+TFP9tSi4XcBXq06iohFZ71MqyyadeMSR2PjJQWmpi4LIIqAgPyVqChh035mw3SgQLsvRO6++oQ+zM49r7ojW31oEz4WJEodye6zXKrRm7cVzfsF84iVYQyVWzoH7IBbfpLDTXshjtOy+KtjC8OzHmQu6cWw+CW+ArR72mtGU\\\\\\\/CEzNVroLgGfopZ\\\\\\\/bEvZPxOTpVEw2+HOiwmrw3FJgdKs6a6fofzKu6EeSXzEKlPbkDsSvkq3vTNDt086ZvPb4bEXSiOp5ZDG4oGLz+K6ng54E8W2AQZ32YKi8e+HCrQBl+hy0fBx4l2vgqC7kH08eEfrJF4FS8pS4mIi5BwtahS1iV7orbmtf0lt4U7OBWYXOESa2ZA2Nsfdxr2wBMiicNs04A4R+Z99HPYwDj05Yk6oZGHxfyaSFpgG7DmC+e5F4YGkxS8knDlBUHaUiYbhkXZDYxElla6Kq2OdJYMAJm9YOW+6g\\\\\\\/hO05JL5tNm34W2fG2xc2jugGc9+VxCoP\\\\\\\/3rYcB2kKI3xaT2CLkp1oFHMsAgD4sY4q8jbARf5srnxyKeRNtSd8Mfq1N6Td6e346oQH+h2dJep5bgYPi3mYJLBrRiur7bs5Tqdm4J1e8Rdzf0D2Nd5q41dkY9Rf8ZkDstwIEIbmhlw1oYNQc+bIXozPj6mZ\\\\\\\/d6AuGo0x7FQhtvesZT4plKcyu7eSgBUzQTJjTVkWNcf5nX+GwR1Qf2Fv4vOTSSUULP7ABCUilXodW+8KTOwFExWNTY6u1JSG8JVuOPCnrbjvG3JHfgwfFsRFcybx2+99mHzM\\\\\\\/NgcfrarxkYFR\\\\\\\/t8tp2F++gdzCUA51WpRSRiBA2GilVlbjFIRHoiL+GrxyxKvwfWxtjI0IW4JMj+QsFA2UmMI2Hd1LklB4neq0utQ1N+jngzV6xg+cNn1AtFjmZEFTUGPxrPnGsR3+bafN2qviE3qQ7rt6jV93gPuFn5mWMSrAFIYKQePmbf5n9Rd0fMM7TK8ZdHN3kuNJ7DTTccR7vIQIs0DlIS7EvKCr6ojJcU2wu\\\\\\\/8BxvtgPQ9bsBoJS21gKFbjt5uPCUPCWC1EpjRo6IlTP6KfBpbpxOewzmb2LoFbzrmLlaJ89vipkMDTjErxTKY4\\\\\\\/dbI145aWKDofj8sJL0W9nIp+vfTIlPZp8op2pKYO+F6OuVdqkXZdxtTeKlDHeplhjewwYY3ZrJoDkDyc8GCdtY9k7piZ3Qb3ot\\\\\\\/ghslp0JR986y3osVYSIvuWPJZIk3nPUmIiQpTK1Sao5noglHpBd+vOFpGqR027D8SAUaaKSOcqiLU2GF4SbiIP9W9hhLHjGX9uRbX1xiI9tSvNMwmFBoCZIcNI3+bfij9bapd+JwSFacoG2QOAgoKAwVx\\\\\\\/TZp\\\\\\\/Cp4xfIk4jko19Qlwv+uOwMK3zmj4ogUEybyygiPWiTZvNMXqy3hYzsshgncC2i9HcUOnaO+1xSmAvaS7BoMlmgmgQaEmsBxoZUHZY8vhTbWkgTgAPAjoX6zf492ccl7b5Eiu6fF3P9QYffAfBQQjEWGBr1OUleHu1phEh83hNm79+IHqZUF8qhYC+ylr2dv1Qp4KM0g61MToZbYwGmFSTInkC2tHRoaevJtQN3Jdd3EAnCyBSSFj1Cu8bWDh+kjPRIKXrDHntfjOIkLA5kno0LrZaf45I2QobhcNT929LCq1SPsSrm6JSAsu7ikBJeBcfkbU4AxEtiYVR8cgr+psiZ6nI+xN93htZyoQG05pfTyGfiZ8\\\\\\\/NNam3HdFCexJIw\\\\\\\/1HzksGin49BpJWJvMOgZrGFOeuqT5fVmtVvqevcVTigu3LvFPHAv26+u9OMpC55EuOu\\\\\\\/RlSP1XycUhPHY6QmJPE+tc5IU4CkbISoG2LN5QS7m+GBIOq680X5+6\\\\\\\/0+jWRr\\\\\\\/ruHdOxTL8OZiRdku6vQV5e1itGO17RL6+gsWtb7pEB5mGxwaw\\\\\\\/pQ6F4Gq8UGx6sd3ETc\\\\\\\/e0\\\\\\\/ZL72gii82D58QMxUt0zccdm\\\\\\\/L5jrq9EeAUZcLj5MNTnVcYBPP4rWVrQaO+DxrZMVhVMWsv6SNqGlc5l\\\\\\\/5fzAFID4EJYtFwaGzYSM1+uzz0dn7kw5qD45nomboakYO4uA7v31QDoFRpsmOBmHgxl4nWLFFHivFriDu5Gz5LSz1644pHBatR1qYFua4k\\\\\\\/ItzHwdKr0NBZc0QuHUoWZy52qLGUSy5cb1jNxX8XGImtNz6mTBLLMoGbeZvsGs6Ll4Doa2Iw3FCOLgYgT7gg67HCN1sqwmE56JG99OnkPtaJcq9eVox44t0\\\\\\\/DpBlOBy6q7gatViDMWXf4MpmYeIarJRMwsZGxVtGzji2gKIL9\\\\\\\/zZDGTmqVlQbPlS2orKMSP8E6V1Z1q3iPMII8RB8KJiaOparHJoxwHjp+NrVjJZHFi7I47D0yyeR6uENpYOiBTGwPzgJsoYiU6hI20wKmmD81amsV1lkPxVDXeRb5Q0EjXFzxnLnDFH++3judBCrhea8+G9PROVBtgc7GIcGx4ejt88AIdm6aEBvq5eWORpGPXFivu6gR1Nwxq71Xu4sHRkXHD0mQYT\\\\\\\/PS5vcGvXD8u7wDylz27++UH8MnYx5Gt8OijXxBUHWa1gjkZF2xsAt5epAfJiRRLh0WT2yCQf2k9gHKU6l7DVyH6tkcSuXfMOsuhZgw8Hpj8ZJpV5jpaiqkskdGt6iq4XOkQAwsez6w4dIHOe3sG+nPH4XEU9EWxHTT9yvDEFNlLWt4aHJtDGOwyfkIOOFIsUHGSyNYpJnOtkX4Zo0SxlTMVSAl3C8vzbQ3JLN5Dhl8mQLl6QQC47D95Ul8PJ5eH1UzwyAN1ado1oQljNID9Pdk471X7HfXrl5GzmsnhwGdsEdXrEewpW3BojK3uQ8HcTvTojQbAD2OEheVrrCBl3ibN\\\\\\\/qmsvh0OBziDXumJt05WdkIekBCCkUcYMcLxmiTrfYPBXz4ptLYNwQVz+gm2YcJEFpmAmAbdovnA2eqzGFw7qUSG5ao\\\\\\\/K9AxRJUApvXDMQ4ON6TXqSsvG+tSTC8oao3Hs1NXZBtgvUpQ2Tx5WBZ6sD0I8YIBej9e7bPy2r3YHfiDHTwJPvB54Ya8tnFDPboBeCg9vizJP0Kya5j1S8Fd3G1udAzm+tEQ36POm1TJ2B\\\\\\\/uLDs65JnLH\\\\\\\/4nftRdpD1+wUADLk1MWm2SnNFabHca81pQqCMRF0PiwzDPQjc5gT2PyQLzuaaNLE6ODGFLiixDdcCg8bVLVtn8SdD34ZNcbs76T5eCm+zvejWvK5L0VRKDnIz3kbA6VdxaZbIqR0D0otAzx2f0Nas7FiHtxpu90joP\\\\\\\/8NlLpPpSR\\\\\\\/3xZkHEWMeVQCCyeXkAbZ9GeTgXn\\\\\\\/naAHlCiscbm\\\\\\\/OEP9oCpuMH+G4KC+Johm7kvzz43g7BmSSWbde7VtfCMKqYWB4UV+zGzsVZlw4YkdoBSb7+i076Y3QkDcDHtTcrPmE+N1aWBNH6+uPHPtR07FpGkXzSJYsdQvUSqzbng4ukVScsS15v0pzzI0eLXYIXGBwFCIW6RKGQD4r9voPDLefWCrsbAH\\\\\\\/okWIkfkwqr2FMDmCQ\\\\\\\/OB1nwsfOGW3oaA4UV4YcnnmhAi7TxfDh1EdLUt1c76XPmEwincOuN7tM3d0AqnLuBagtbngPbCD0Pxy2gaYIknUUBxAlKUwfHpPRouAgGYxYvo6GMNOqzwdajM1z+NvwIcIo8mQv1gdHUIPZKd2sTiX8RaYV+R8lAvu7O6UjSN1dcY\\\\\\\/lzqmzoKwuAUN8Q0oLlWzqOOtc9qx\\\\\\\/OC0N0Ba5FaLTaPh0he7PNDx7g3yzgT3Y0FAmTaYdT0jTjtNFS\\\\\\\/bqLrpQ1YPHiYP\\\\\\\/DrgJUvtb5bnb96W2cYvav4dUcu6gTJBJPp+oCWbI6EsLqfGM7fGt8wATIiZfdz4gc9H2HE0UHlXbrGEN2dgjcnPJQySFwTUidgq6sn1xon5XC45cdieMFEIraQ54avdP5yuPiDEbdEigoq23LRj0Dt7bBk2QV3s46vK6ZVUc+Kmq9k8Fs0uQcmqWm579HfAUpfcB5YbRl0C+qHSFgMccxcX4kUTSXGjoFKMVI0NhxaqMS1IY\\\\\\\/i8nxMOZ9YRwuxWTb1CqQWrbdkREU8aQzk+aQaChkYwnUPeE2vSpHCNqR+kS7+ddFTiOMdd5NkjMT6Lz5nEK+azHpx1nkfaZpRF8zJ0uiy3az+bGShyWD06ddpw5pwrCg+pC0jO+xgTkDnI94BJnGJVDwyR1qWOJeuW\\\\\\\/nHK0mSqbQLGh1apAYDFgWScoPjyRCrsfBi69sRyFVxJkBz5kDRVrw94f6+tNHkU5ofHHJukosam0g+12uILqr8ccQR9zmN5fAIZpYVpp4OCSZVDHImue6Yl9yjCRr5TSHbtNLcF9BafDjNh\\\\\\\/FlzbrZlTDR+EQLLaZ8yLhvJiDsiiua88iMfUlSpGFpsdkG8QWTrDu56uCWAQ14PCq8wdF\\\\\\\/1+vF5R81ZaEBKtbjcl4eNu7wuHKVkoqS+NGO6GEWnbPZZSt1rdJ928XB7Urd7cdKUHKkIneqdD45yYbzni4hS7sfWjM1oVOLpThk0G0mS1CRl+O1doP9j2y4iTfwAD9NfirhjSpgVw0nCRIq80qUeyNBhLcCDrHZxVBFKOdTkuwHx9VddjEg2pat6tiHG8BIH7t96SjFHeDte3Oas4jLvkeD5tIq7W+F4wxlyVKtPU51tbfIWCXLla9XqPhwiytXSLK3pfmMFHdaBesiVunBCzbwO58BUUCjBd0eX3TZ5YofYTyiLgU4a\\\\\\\/QFAJq7Pheekq9JwuWih9wBewUutSDx4qQ5ub1wmTEbLCl5B3ASv5IR3ps8RzWjG0pMkGvpm1RPrqXrotJ8RxiLzzUCxOjYhrFCdCajrC601cAWfEhV4ncbwsoDFyaKYalUVcs5QkQOx3f0wlUQmm8AT\\\\\\\/+HdY83alh8ytZOgs2OX1mnZOgyZL3LxvgEi4Ti0FCiarpqTIZVq0qYHMAQPC9jPAHLFY8keQdFpq0i3Qh6GXuCV4AnrGvS4ijeepKh\\\\\\\/CxKlwGMIm57n1USKndybLzF1SS1kxyqLM1UqU5LLTW7oeuhEQXjNkxhND978BC7+mrnMXZWx9drF8qRSzNyD2xIcAhyUq+AO696N7qTPeTLZ9QmoXiPsnNSc64j5+HgHCNst9iLGBWUlWxVNRxHVJycugZlwp3\\\\\\\/rI8FhBIzTBEpwzgeEwNH6gC6fBT+TaGGOvBrEmzE6HWA9Kgd71ECXB23uaF5uqRWGhH\\\\\\\/7NxfUkE+AIExn6beLZmtbAc+K5owhWwFEKhp3l13JkeQMqGHKg9OnO\\\\\\\/FN0+tqavhFGvUUYTSbIOCODS6H5OQvuHxJADc\\\\\\\/PGU6Hio0hlHlrK8vqa2vA3cUGT90pd9eUFGg\\\\\\\/ymwmlz3\\\\\\\/605vdkHolmPKho0Zsl+7T0f\\\\\\\/tDQ1hHJYNPt3f0NbiVc3Cn\\\\\\\/inimy3o5C4PE+J0TbXEeWz+fDChbjR3uOvPRSLvSiNZjWHzLge4Pwj0i1i8im7GpKC4+IpzyY2EziGcnbLUcgC2gcG4EnA1Sdgoje3ilBM2vTdxwvQBMjPofxyStyrjQT7OkKMMCjPiEQmJ7B6Tj\\\\\\\/J8y14PvfmN8AKUEoDNfGwM2w4sUtR22xmnfbyB2o9RSicM01QpEoQw4K2Xpk\\\\\\\/cIELEyqShA4WWQcGiiJRGCSJKOUtEwLnVMGNQBzfuOYNu\\\\\\\/3uGpog5Z2bSCJk63wBn73L2xi\\\\\\\/I10JpwfS8nlQ4MAsbmnV45FIzPpiTVzjj2HOPZT5xliLKC6qPT8EvC7oyQhwA4PTmMq3hEmKpCWKJhfjKkHfdMcN4YI1kGLdnLiEZrr2iB9Z6F8s0SdLodfvyLOz19xZV8cJMJAtN1NJ2R\\\\\\\/JxZwUupirxyIUaFn9BqG1OeWrmI0nurhA4j8aWq7VsE5U57JQh77XiFBn\\\\\\\/iTDBu764IyDUPBXzZb3Ku3614WtfJlu1yCy\\\\\\\/jiH6hFNvOqvCcYxik2F3sRqn1pvIcK0GHz9DeBrCLWEeXv1y4EEMg2tmgseS0Ke6oofiRyj\\\\\\\/imApZkjHVYRd5y6ZDm6+VgENJ6LVtmJBnddD8KDGe6S9kdLKYSQhkN1WUqbq8oe0pJXinzgET9tTcY84JHGPt17jGEKebXCIxAvHcguWfVxMw5b7PFNYILevwZUsDIRJiPcI2RzSwh8kHVdG65U8SJk+sCa9YLAGkyzHdZLqvFzusYBlEwSgXMvqve\\\\\\\/cmozOJ8KlFkanxk2O1kpRTj+GRwYLIclpgBJoF+ni5S+LWjifMhsBAU+Iq0ASXhBl9WuqIy79IfV4RWneO0DHnjiB3aCnWnTfOwEHKFmwquVdphpP8UMjKwN8gKNX8g889NSyQtITjdyYYiiMXOU79sMGkuas5RqrABX1f98hHneIkO2OCE\\\\\\\/2G4ugfyy7meHPkf59h9v+vYvrN+uKKRhu0Odbc9qGfjKZP3mXsWZjXF6RjqEZvzyWU7PMLrObo1L5xGnWmvpU9OQ8m9tDuaS+kzWiA7DJPt4fjR2U17CbZtO7O9oVq2dDVRPObHWcggdz+evRuuLSpLeS5B\\\\\\\/bEmsAYfQwgbjNf+J9yeR4nFR7Z8hCcTrWeUe5il9cfdWX7xuJgNCQKBJqJ1aQcNjtXGY2Ed+F23FFeOlxh3BI+Ow5oGorBh5Yib9C4Arr68kuhBsJmG43pL+TAud3Em9E+jbtsA7xN9PyEOygjOR9Okz7U1C4EO9nLnRFKTUz+ReHakJusog3E+I4kd7kaumo9E2VcAHWKg3quNDIdVxcDjJVLrkHdeiMVPmlFZspnLyugELUx12bpkaV\\\\\\\/\\\\\\\/r6AekLjKRE\\\\\\\/mHK2dYeCOwC1BjKzMtIzBRLepEhAC2gey4Kne8FkQ2RYrICSOo+bRbRIZ7UusroeS8lsDa7cMOdvFG+4j4PB4Z6jyZaGe24BJu6KADPasiNRHNoGn6BD\\\\\\\/TPPnEBPvsWvt5D2p\\\\\\\/1FbHvlxh5rdtxNQL2+CUCSPLmuCof+uYDhyC9YY8dPQuhuMzlJxRqhA9jonmK32sST9rR5bdnDKAHarJAk6J40NJw0BhrQNlATsAyGF3MS5j3reWLG6IE7cGAqAqgl+iyGM315pp+aEXGoE9Ijg8JcK2M8aoMwSwixQTRWCgBDK0BhVr92hCmYXkcWZoFClMVf2IME4bOjns6zKlc\\\\\\\/grAjSyC8I3BquwY\\\\\\\/aAfCoqJN4\\\\\\\/\\\\\\\/9BREvs\\\\\\\/tVY4tJiQB6JzRmsGFwiYliZMpM0gyKlTrjV3mYr6WfX+YGShoGGJLFL8IiG5U64EK8XPuG6jTm9GcccX1joQVogAtESuM9l1EOpOixx4dNsYYOo7pyuIkvhYg+wc+hYZtAsFo9b3wjbzptxZ4STKy5B9T10NNPaIXf+lLubv+0l7LCIZbyy\\\\\\\/gsP6w6Q+fS9gbiBsi0z97HpdAnqVzRot9bJwhTQViq564k2pRW91WHJJ1va4u8FNeGRTdpHLkOzxyCJr5MWA0WQ0pMscxLEmm0ROBczsIROHaymxmMlPf7RkkwbMohCn8IlI7MYUkebTa\\\\\\\/mxWOgvcKW33ySzLH2ilK4CD8BAcbpIG9qZNIDsoTD+BlY3iPNc+3lwr4\\\\\\\/20US\\\\\\\/\\\\\\\/iRzSN+BVRJPV2EMBesa5TmeKbmOy84TTpUBwshVGVZfduTNZMCPnJQxipCjZ3crBlsOrOfetTSUueIecGIMUz5WrGVKlhkak+OKQFIBSYt9JPuTsnSzHWNTDtalhqUbIAy9+EoFMxn259GbhlMOCYrUBjPewxRlA3kYoxt1eRqSSJ51UU3jEcsxKDceuEiIpLKQSdcOj81xaVlmvMjABN+Eogc6nmguLf8mt1zmN23yjkUohy\\\\\\\/8EPP5cgdVwrbHOcnZnmwKVk2wUq8cezz2h63BbTlLvxnGL2a54dRTVcv72BtVjFcL83tx2XUXE2PlmBBJ5QyJ3eOVvomfDA5A6jXN+gMDhSuLJuGqrI1RCgOxGv6YN9DbDjISEyYVkiLQ2StubcJ2qfM3meVGuldwrcCWeyazABagcGEMsVGPUYYso9jo8+6ILzI3tIGk2WaXjJgV+T2\\\\\\\/RjrzM4J3Vb5fDvzKWXeRvba4gU0GtJYm72IPKVuKa7JIrf4DCnKCl4sKbq3T93KWoZbZrdtbHcxySQBYTSq+Yv65ih2j2DIu++dlB89sHpfPQKN256rBhwtUdtEyNbxk\\\\\\\/5bjBFWR9Wgvsg+AUqdFC8kCx\\\\\\\/u1D1USAd6qyKu4GtKgOwfztdX+gpWXIHrc2pSTMsnIgM4Yc+7tSnA68KOnNfpz3tLPYbWtEiopqvRnmHrXzGcs5Ay9LL77NlMazzocWLXk+vn3KLogHrInYlbJb3eyAMrcGLtDbUOxnWQnSTDad2EewkqjXmYKc9lf\\\\\\\/Dk0xgqIjB2Ua5hlek6f0mW7Tc7jKJ9I15kqhy0KbcUpmH8ndUoXj7AduXSdDPjgtEf6mot+si9H2b+SgaO2+B2p8XhuGNf9RuFs8ZXnyBU0dWgz2i7SY3qro7nJ9QZoWePu6Dlko0r0ovex+WUV6qrnDpqJm8M3uBz9ZUyoblI8lOOcIHIzm0CPm9P7iP4Agl\\\\\\\/SyDnV0XmtRnMawGtqwzmlDZznppgyeVH8z2716zMPA5\\\\\\\/2hMUw+BwWDKvSzTutiEWF52fzb6y2BNj4a9K5pu\\\\\\\/5FUsyXXp6TSfMczCbnPGYVnkPVxN1E2Kh8jSUydNmh19XHJpaUqzi1Su1ZbbcVlRjnjHgaH9gXcKKMCRiMsrIppqFKSWdFkk3nimFhu0ycm4rH3x+w6r\\\\\\\/iJq9vtglq5ei\\\\\\\/PAnS5m8Es3+xL1GlnDJagxqwto\\\\\\\/8voR3mPxrWinIBFmQ6IMIckBvR3TSw29tL6mqcA5QYRE8p1itrcxHJaQDC8dPBAdoWfU4xIg4A3sKSolC8NJmfso0fCcpQNDmYzgvws\\\\\\\/1wWX89ASq\\\\\\\/c\\\\\\\/Hi8UgwCpNtqvH9gbLrhjV7qNpkq5BnP8TQEQXv9vv5Q8ycw3ZsIif8lTeF7Qu3AKTy7TiujKfpP8o6pYsrHDrsrw1ydshzVnX7g3CWobDn\\\\\\\/AjHaq+FEpI2f7UpkEd1M3jDRitekjpBPY7mg0fYmPT7PW\\\\\\\/YiI8wTvd1D\\\\\\\/KS2Iic1qvwHtzOVCEgT1ti0LpfmKBFFQNF9z77SYKHvrOIgwRo4wi36BbTHIW8yjGe+dW+T9vA7G3avXyDDrZs009a9sjd4P8j+J\\\\\\\/S6IrC\\\\\\\/XNIputOHpH5XBGW\\\\\\\/nHKHGSLaEqshTtlP\\\\\\\/q+B94GyDZZ52bFlwJsiTS9oSLIZnmcAJIcxd\\\\\\\/KMEFHO3fnM96WSsmG9PMJTknabRpaomjLD89MK0VQ5oRc0TvbZC80Insn8m+GNaNAlz13GE5kh0tsbzEjjjA\\\\\\\/2oda03vufNuvClK3dGvKVrxjxyE6h5kC5DSOZqOq8W8Vt1gyENNGSrrz1Q7knuOHhmBvfQ2Gj+LGkU9xvQK2qCiNPHpKuyq9iruiXb\\\\\\\/JV9YHH7VkngR954pmKTdVKRrQezH65Ytd6SgbzYZny4OAfFaVQRXYQa9BmdTbv2amlkntskU+vv8lwNQ9VaA915XeOxDZW8hZcbDsl8d0+5JyAsl50r1NtR0nr1Ugmf\\\\\\\/E7TnRZPkdqiaFwvoZK\\\\\\\/Rx8rV0IMPMT\\\\\\\/llTwPyEUnzZQGTzIGZHAbJ7PqleYhbXzUSpAKrejwkNwHqr6rR9Esp5a0xDeGm6DTtgoNaeYq6ptsz0p+9uN1tNru3LIV3Wy2YLDnEM\\\\\\\/0UyBQiYCUj22gXKr7ig\\\\\\\/uYZ4D450t49jcDI9C4gJFQOFYBC1OJl78a+y0CrjUh\\\\\\\/CYtz7U4Pt6bIF\\\\\\\/dMD6QSu7u9OckI1qT5hr6kq79iM+iLghi5Bc82e935rg4hzB\\\\\\\/X6FyyiS3nQCDf0McjqhA6tihRM6zH85reeIXfktRCttpfyg9GPfwWfuOcZkplHCOACfD\\\\\\\/rNjc0fm8x5Uv+\\\\\\\/6IXUs3fxdXaoMqxidSAomc5WUkbaJYWoFcugaAuF3k9EtiKmOXHhdCzMCYn0bPpmC8XE\\\\\\\/ofLp1iaXc0hV0WFJeK87Wu8wMyD09V2HC05hrSoP+UWYza6k3BjSCsjfLmVdRdZaicMAjNHmSoEZvuWTrZwWjmkHPQIQnSz2jVOSAh0fj4ka6u2IpNUvMrFuRvxYiJwATZmJ+8rNi08yJWSVEwPlayCm4IiRpHPg1Exhmu7Q\\\\\\\/wvaTtrb5kX28FIW3lXGzqW0zZQKWE1GTByHozM0C0EMYVJ56cu7Uc1PMkE86mZ5Yhm6pO9kuzV9DjeH5Vi9PV3\\\\\\\/F\\\\\\\/mdAIm3ZNbG1ElLkaOTslysEz5U2qv62QlUQRllMKymXt91hOAtn9nzam6OyxiKXW7XwMkXI8k6BzlzWfNoxXmboF3MTUUBqvt0\\\\\\\/oNOsgwS4c4TFKI7NK2OnLTotC\\\\\\\/zTpIgopN2MAQ76OBx+Fcqi4ye4PYE0wYmf+R2w44VGtCqxZcBPkse3AMvQi8jg5yl7aSjaJbcBzIzODTJwacp9vK78OJnZ9S8MmVqF0tm3kkpckRH\\\\\\\/UfNf+F61wHAU9ACcsZlswUeODDxF3wDvB89FWVFEJFzO3e3uENThIrim5TWx6SJv8httUHGBX5JSAXCaSeOI6Ws6WffOy\\\\\\\/OzxETqMBNhEsK2gfUjOiOKzaB8qncxE4KHApaktGb6Ln2XrNUDlFdfQBeFa\\\\\\\/iDy\\\\\\\/N+82CNZWHc80wkDM7gdXz83gyUxU+98S8InF4qzxSGomEJfO7H6keex+C0QEv65Iv1mh1YdV7aW6a9W6AxDQ3VSIU4eTfXD\\\\\\\/AMGXXDB2zQufOWr6nawqGN9ZuyKYH6cFy42CYrIFQ3z8dtfpdvPA6jSncHopAG8TzVsNrzqh0H\\\\\\\/5IEUuzCj\\\\\\\/vRahl\\\\\\\/Hk9GAIOQX4DkBQF5wCIrrfCUTr56Xrveo\\\\\\\/Cv1dtMmcZ7jfLRjTBPxpXTqE9XbAbsSszmMJd\\\\\\\/TqqlciRb5K4cpbhQmgqM6mvf7KX+DCQDEteZPL8Cdb\\\\\\\/sR7zyC9OxXYRQvuM5JInaQepX7NEkIhmcfRUCBcZHRnw6ilMHLoYD35w4gh2arXfcL9g5\\\\\\\/WmzUgfyah4lpjqY6INSmapbl8AeUmsLqR8tuJjagNga03CqRhloCet7Av0ReXwN8JkWEu4iHIGxS4mCqk6KiBfdMZIK4EW9c3uMhZbnbClWyWXs2Ve67AonOZ579U9SjrHtAaW4QppA4zvHOM3vdAkyr\\\\\\\/zsC6XehFWgGgIqJqR5kR+JCV0mMBJqjb8TpY+jhdbwM8gZB5mkZHxkvfN48gg\\\\\\\/H5S8YnbbH+qq5uG2CGjUJAgq4pFYcaotAJEvtwhi+Wo\\\\\\\/WMIXwl8BBEVGVsTh3zTAao9jSuU9IyeAHbzX7FSKSu6Cyyzx9t\\\\\\\/I+\\\\\\\/PFPp7UYgKKKRv25c0ncOVZWBJFOcurXaZihEGvChH2m8vX56D+XwxtT3CxLQ4DOBtLEQvUa1MZ2N6CnCgUyv1hqMMoQX8rp15c7tZ0PT9KMWIbbH0\\\\\\\/8UXDvvpTYAQL\\\\\\\/P9zj\\\\\\\/q2BhrXZ9zjjdgmXtTH1xyfb6B1EzwfIX+wepoOMVWecPwgVYd7\\\\\\\/+xbmPXdZnqAnJLeRk11+li\\\\\\\/umkA7juSOSzUB7ujx5dAP3C4bf7zVx62NjSqRgNmNYFt2mpLCcx3JdLBe\\\\\\\/uiRX\\\\\\\/4\\\\\\\/drfkJF6utRyGvJIwPOdTllpnZix0DfJZcx6CpFk68w7LsNMEEqnw1Yh9MhTe\\\\\\\/ESc0TTDujVz96WbdnrJ+TQZPL9j\\\\\\\/NmhHm4ceYXbJH2Z0jqKHHBT0SPzpRcCcxWiPPFKdO9uJLsNH1CptXYqZ9Q9TxD1aArt5wmhaiy71a1E0tLMSpj0ngC5eqpZH67m3Cq8V7slHzRJ6rYXSV\\\\\\\/dF74IMpnIAfgW28\\\\\\\/WUwFt\\\\\\\/EwLbnVxXnLSaZZQfXt22gGp5SwEeWDPlOJu62q1wh\\\\\\\/2hSijcNKGvsuVPfDtfxaVCJYutLnT217YRQnkjO7rb2C2aSjtxbVXa4tZAK1GyzRlFiXyKsGassagHR0Yg887muI910Tkib3jgi2SVlea+4QKkwqLwe6TY4BF9wjoYuBvBAnxXJuNuvitzMdNqMCJF03qSxCBfucYwS1\\\\\\\/e45obj0S\\\\\\\/CNVwcs5OdcshZ7qbiC\\\\\\\/wlHARvNS719RDvd\\\\\\\/SwLkAOsls1ax\\\\\\\/Tivkfe\\\\\\\/hUZqhESY4Rf1g8pjCNzwt7u3EVaY3fBUVu0IRPlX7QT7wENvOZVwfiDOcvJCamqPrlZmsdPjOLXJOk4Upd4svpGo6cnaVoQhW\\\\\\\/LHKzf+i3cb7kE5P3iK\\\\\\\/T9rBCcedtVKbI+jQo62DjFZmxesgvYc7ekegCY8almoxkEb\\\\\\\/IGGKnUoSIY96sXm3sZNkPWrA+ytsutn70b4x7ITU8EdBQ2KZGN\\\\\\\/8Mlk3\\\\\\\/4D3BSDNVjv33cgnyfC7\\\\\\\/OkbKOFFKbGyqE3AWRptnaQwnorqR\\\\\\\/nVMcFt9SK\\\\\\\/9s+BOHRXqVyKUlyQTMve+DriA3KbP2L5JgbehZQEHdQ8k6btfLTv2AdFRe5fs8WSIp7TytWF+2AMNJIHATPcWgaP7RJq6kHXQXwpFcDnCRU4oz\\\\\\\/snK1Y55\\\\\\\/sYpprMBUnlCRPsqK3xlJtPmd2QFpzmwS5OtYDb2h3FryQXa4TSimXTL96zAoWgDWdgUsYit3vNbznyU06C\\\\\\\/eHVDYTfLCF88JckcxCIp9geBNq44fJUMdIi1NMTWLjlaCpQQ8kzi7TD1dD1Y81\\\\\\\/lth5VPxjP+Q+\\\\\\\/dbq7sW76m0EJkTpmdVNJn1\\\\\\\/yRIiqTiYFxh4lN3vaaRLwexwqEbmC0ZrqMYKPlKDmMVR+Asgsi1RBVeSA9BwdmbOxBxBWObLShKYQS8wcPwBY1O1yjWtCoIt2SD6W4JNAJhkssFMRXD1szWl35o8vOd6wxhKEfd5xGMUAzHSK1w2YDufjN3SpIgASLZ7bXub\\\\\\\/3sO3smjU9lrcIazvQ+1KylZaxJWWdYJhq4d23rJCpA8QIw6EGtEW6OEzjlSvWpJyc268nq\\\\\\\/\\\\\\\/KnN\\\\\\\/b5QEO7z+rY9cxLG0dj6Wz83jg7PSCsRR67EMBnnEDA9CPQ92z7MMLdgwc2fIQL6dMZE2CF4jKpXp3p+fzcBfx4\\\\\\\/QvWCCXxkz3xLhmq\\\\\\\/tknzgR0OQHF3Gzqi9Co9\\\\\\\/hqqG4FHvIC0lrSKc3uZ7RvusED\\\\\\\/S6rnSN54qC4+g1rI5JPveAEFJOloBY8DGBEGbnR71fr9hJmaozrgrW9RFJElC4ZRUNRvcEPlrrC4hmeUK6Q1urb393ee\\\\\\\/GS9z04rvqSs5LYDgfYw0Id03E8KkupYP6UmPJivwqNfitcbqRmfTIim9qy5vAQbCOdSCt6UB+WcRbrlrzH6v6PlEZz46hakOI+OrM39Zjo7cFFUUCRu1FH+y0H5xnOXhddR+4MKBHkXcXu2inDx01UdArFO8hG+hKt8pyxf44muBYWHUd5X4b2FronwCma5HsoXL1qbD8PXy5QybbVWkLjm8Os9h9LK1eEvrXK4iNW0m0NY3PZJmu8pUYCRwNUFw+2ehzYM9g4UaJlwDUqhERI6dzhaeQ4CiLkQ\\\\\\\/aOwvlIUSRhmqXfCJrd+Fgh4LRrbTF9uE8rcD+47F8DomqL4H53ogiWKdL8v0WmN9aQhwUGCo9PV0BA5JQXVv9nEwwhUIz+hGP7uYTx9pLEztCawlk9XDOu8E4M6l3bNSsHDGjrvqOepeYvd7gV8A5lEG31LFntozSIo3wuPFr0tkjZUXifUNo\\\\\\\/uvyX67\\\\\\\/Q8wupnAY8Bryefar78fvuMrC\\\\\\\/Ostrn52iM3u53a0DHSlqO1FvbIQeN4skuu63FiSSldLaV3CKMhzrqoz7nCYHM2mynNRTrMkB+g2HfnF6iGPpgC8\\\\\\\/wd7EvOzNef+iV3EjnX8CqsdCYx+NEpbydhnJXorYNWkSjI7xhhyLFyFlckae3n5c8z20WGTryD2WjQ3HbJCT6amDBpIf8qmhqXMhQs1q3\\\\\\\/tG4Jmd\\\\\\\/+gZxjYWWvZp+hdL91PJnmZXnUOaTUpr6OXvetMHhqtJYYULjGkYxjApkGDKEc6kBy5sXoErT75x1ILDOWJkygYHdezK0pYaZC2WpcNGYiNvEA3COHmsDHLBgwbiukELNensCDb90gS4v6\\\\\\\/9oznutMDGUNYokG4iearI0rk5Fgt8RL7W5xX647rSWOUJhzVDKURj+\\\\\\\/wgT23XY0CXXKMZSO2m\\\\\\\/uwd\\\\\\\/+zoxICt5KTeodrBTqux\\\\\\\/qaQv+tqvfzkEF6BpAxCjQwU6DTT7qE1e6zDRe2yGnA5046Zo6mTxhZVh+XnwCwm9RqChTEC\\\\\\\/6hBC17dN\\\\\\\/IZ4vnDn7KOOTX\\\\\\\/E3c7sM5DIpA0PIFbm21IxilAbdKjBDnQPCIwoy2BD6DvXVMOxVur0EoLYS2NqKExdOHwjhPxs7sOigUJ8rhonkGNCzG+bHV6TTOG7k\\\\\\\/ML+LTOA0deX3U3MEx3r9ibqeG+ZvG6uNsAimpilZI6f+eZ+McyBxTLi2jXRm0r48M1SKkbauxIIpmtyJxeto881kk92NQpaCzz9zzKHcrPEoUTWEPxnfayXuQyUQuruznyWbPoBMrn3RUxkFUwLJNnGiXFX03QF0lO5FD+LQo4ShSSE7VP\\\\\\\/cjcyANQJcvV\\\\\\\/VMEmYgcGr7DsjkbuXnZ7u5LrRkcSQaG1goXQTyhk6DXgv8a1ihFGYMqPH9oM\\\\\\\/xDJkGktcfCB4p+81or7MVHEXvigYYVTh8iizwIMyMF3yS5OGxJ9Jcg18J1bIU6noUvRruHvGeHQxQ036xsUvSZF2soMn2aZ0fEfSqDJBcp6apMSRshPsHAOKMl1vGPMFUaLQB6I+geuKM5IPPH2LUdJtOzV9Hg4R3LF1alW9W5mnNUsd8+fMZnb0LnUBGDoXxD4PYQ\\\\\\\/7+lwcp7TJqiUCftJzdsRVnGCBDRwAItHCSZz4rg6Or98M19U1w5\\\\\\\/tFhwxVYg5D28SeHj3NR2QD4OB7fBFNdIuCbOBvMWtVDsMlxW31PWN5\\\\\\\/7x7LDixFpRGxGb5UkrCkV9zTyluWYKmJK11h0g\\\\\\\/kMerZnlA1acRN5Jr+b\\\\\\\/SK9n3bj\\\\\\\/DWhAH8fEbpctbq8f0YORvH7rZLosLbHt4lQQrkO+aXWPQHwriEp7bf5IE1w1cgyzY98\\\\\\\/ShTcu6WuaNYIcecJWDg3hFUm6mHXIR76lKUqGxeDwD8Ns4FMSYXgOc493dZG0k9jWgj1BLcXF7khIBiG873isi7qMAMXKtzr0st+y6VcWcA7TFjFHqyCWK\\\\\\\/qVUaYIR6tkn1aPbIFsa1MBRe\\\\\\\/bz2pHTrVcLB7tnJHfbu6vIlsfgZKCtDOm\\\\\\\/LcvqfZP6\\\\\\\/VRBjo64j5C\\\\\\\/y1LyOFUIO+Eo6aTnfjn7PYAca3ZLmCyoDAioO7m00mYohCu+y1GQ50s4CJhDepFZXBRaRasGuHXmQJ2DOlgORKudcP2sJ+YEjp3HOLL7LpaqeJSF\\\\\\\/lA6Ij1488Kqdv1Lq6lfIQJpbs6BTt9DPnnnIwjVmd4+B0cROjLjmKqIClGUOs7Xzr+D3KebZlcbDm\\\\\\\/iKjDcHOJQW4DTNXB\\\\\\\/JescvoYK8sgbskb\\\\\\\/Yc5PoTYBmNxy+UxyGmWbHwr\\\\\\\/f3MqlVuUmyfx89qSDWYEQVYPVGA6uTIm6J4Q\\\\\\\/8bJRki5KrjITKhlPFxdsKvuw4OJigddLPj0dHz8B8uaWFojSvjDMYHIqtBHCzIm\\\\\\\/aUrJRYDNeLkDAh2jQZm0uvcJQebKJme1hOHOT\\\\\\\/tbsrLrMlSNgOPhTjleOnRuvQWgv2uwiy7rB9lpU21iobcDhfarirOTeHqjXcI8bDbLYo0AiwLtGAuyv2BKY5tCKMvrBPWcMwtgFS3NvqgqmnPF9fww5R779SocuOnl+bqgwYyZWWvdld7\\\\\\\/rhXHjuHV6oGQzimFhriy\\\\\\\/PKEi7HkGTImPQC3cQnSX7oL8fI5nzlkVdakxxe2He0rxVuEqcYr4QTuRT9TRa9hXpVQYizwOssODTHA5MnpWPZ97GGhXynsv7QFOCCX+OJQ2iyp4d8aauPgcsPEEhCVj8MoX7BlgdQYeoBSVb0tp3F773aBP26A\\\\\\\/StFiPFw2gpozwkjvxFdOJg4a29UAiZ3ywsFaMJcVMKP1p2\\\\\\\/GiUd7eL1dJDD+SgAXHLEzVIgjjFdQABiPqfFOOMwZa\\\\\\\/hbi2lja8Vj+A49tWFm7Wba1DER0kyG9lmfhd55v1T4T9GHWYkDOwdB+KvOSc4SpeaGq3i+6LOn3dfoDqDfOw2x5ImSmOze64WkY9sig0g1LxS8KvhCDh3ocoHU+FAhUd4fNY+CJX7MDDENnvbkFH4uLplPoPufnKgNyXo3u72SCAXkGJL6jyi30NxtR2kpX5k3fkUcvAr0AwQ73NqazQz+Z6rMcYr+TR4QlGAz2HUQP8J6F\\\\\\\/xC6YF3iIdRY74ea13lSgyWgJzfmwWH8Z3n1Z6LVgb2uoaRgrprV\\\\\\\/\\\\\\\/A6w36AD0WzeFE\\\\\\\/MFeolWmnGXs+xmy4SM5fJJi3T1d1bdwsQuMPGL0hUYTDtTCKf7MGZn40nmo1VUCa2b3RUWCT546GtemiML7XRVIfCAn14NFk3nBwkYOnlcmaxMYfheilz8ACeLs9x0l8NoF+l0eRtiWv4SpNLM4+WXhnLFRMkrorsAQJz6CuIqrRMNQtr1Mz2eMRy6LOUuUudwijFNo7mgNkeMiSaFrLRpc2J+d0f\\\\\\\/Ip55kmInMgsWNsjQ6+YYlHrzYmLI7APR5GoSD+G1OWKOOLHGM3MArgX+146Knc\\\\\\\/Ehskilbbi3gzbvm0MiTu97YaGMk9ue\\\\\\\/C9Na1lsTPdzIdbXE11u2sWRmfTSdJ0Mc\\\\\\\/Gc1vzr03Vo0feBa4AYKNtyouXzYcMvoIV+ZcfhoZfShpPXoSzEBPVJ+QcSgtUMj2R0dpoDIYqlipvfQq2LdLcDXVFrdiKVRaAGig+vbYYtaa0krPZALGMNsRaZhd2iYtbv3lxYmul2xa2J+WND6zRid2mEMQiigYoHEP\\\\\\\/Zc8pvxte+lXSRXLiCBINu5VQp2cP5iVE0evrO9tJGEjflvTkPLdaaUrWLuA02yYaAuuEJw8Mvp9xuO4N8Ke15878gKLpP4Fo2PfiZZknCRMSK6ZNrX9wIyn7JP9rg7L5PbVQcvCUQ3K4LbzUrAJzoo3QVi7MZqqf6OxpOTrAP+i8U8QYEISdYgEIRWzelddmQrNOeKZWQomasn6oPmt8XCpB\\\\\\\/Itq6y79NXsiYa3cAGSTAZQzflRUF4NPWJvqhUwEDaRuim9jc8srhFfdpwnUpamcefrcQchpd4FK4onknH0nXcr4GPhdH7bUx8OT9fAxqbLj+ICAn5zW+vecbLpmokpTN30uRjIF3umc0jiOLGLYyEKjEBstyg9UNPlr\\\\\\\/EUNt64cbGM69hwONjHx06Zfvgpgn+vN8xteIVRkzMVFOXK5tE38z6wta7m8znHLgUOelGguFDA1AQytrsCqeD\\\\\\\/JO3kzvxzd2JHKjxThRuknDD+BLJGrMtfiMHl4o38Mg08F5liQ\\\\\\\/iQRj2lKJOEOHNhwP9uK43w82jKzZhVNtfZ8lS2UhshMv9vio8wpS0\\\\\\\/6A8mYYk3s\\\\\\\/LjDp3JhVCb90jwWoBPq3RFPYaSfPp85f7NCK2bMIVrLjuokvQ7zkPJeZzzllZAttK0RTDkZKCD6l7BB8HIT5Pxp5l4eo2Q3s6dsyYxDHUY7gBoKyGuL6e83zrPJF5Ybm8ot6QpubeG46BR5DxPhRjUQDbeTGmYitZC2dP9Dkqz7bEoTKq6XI8FhtT4BJvFegrTrLDan4q3mWhwnfHAWkcAVGPGyBL0f+8oKH\\\\\\\/pUbrCMcUYdlmLjO96S8VX4CQJYmUW+VonDWViXNfxdOGK6Q9j6wW5jVjmnNbaFyLcCMGpHb1t7ZcBgfrnz4kPISkQjnB21w4S2xfbTVGn0N5bXhVCGTR9IMNzn7Sg3TqIgnYv3D+GJH8QgfWJDquPs5jyOlquJxhG7kiweffRWPR68b4W6U8NCcwAVxM6HvKp19hZcauCPaXhaw47xuMoqdSrgyur4icx47uFcduUtd8kWGyiJvMygqtHg7YyoCh2ZB4tV9lS9f\\\\\\\/sydthmVPwCPSTciT89RbI+F7ernhItgaKTOdfm3cons9S6RZgfTqMpYlM0YNKjy38UfaRENRdSjMY4jxtanqeLJIuinsMa3ruoX9szG1CzIFX531oaCqEK2Hh54vQMAhxjF2MW2MDhRzK05ialsGboFLL5UuX5xgNx4v5U+vI98MRwd3ll+bu0JDahjPdF2\\\\\\\/3N8vhsJ3y0BdUYHft6zrBOeACRCW7WBHkxbMvJj3tyfMnfaVz5LlK9ba8cmDUetejDcpGpHoZ4vj8x+NI\\\\\\\/FUb1EKGmLNmK8V4profW\\\\\\\/hQGzDUYIyx5fwLp4BnXyT9ebI2cEG1NfhGV7ccVgcAvUCwhigelHO5PPyB6uvWf4EC4Gmj4Yp3Egved6N+Wli7J8iqnNHtMjo2g4YA8tbOrgIa2CsabdZV7TjKf55PxRx5PSOK6Cf3CglzXhnTWhYys9cUVGtpZ+P3k5hp7KKQcO1JA1qGYxt7TMbNZ2gBHUuK84aKHodGcQmbrpEDUO+\\\\\\\/RvWSKojyMkminQXX4M8jZasY9\\\\\\\/XuRs3yumlmm4g8RUr28HMuvBbSxDyzkEd6H9YnowjBu74X8F84rpwxsmmxY2cEJ8uOtAfXMxdER9mq9JL7vx7tHJYYymeoPwM\\\\\\\/fI5628CosYIMkWAiPO0IgYXVJqiBspS7nUUyb3PF9lUy0h\\\\\\\/n+RVhkSv\\\\\\\/t\\\\\\\/ppgTj8LEDAdqrZPsuCczf58CnFndFgadTa3rXZFPNhynaLwrXtuPvUq2bCxJWpOLXgiC2ixOzGYXNBYTMdHV8kn\\\\\\\/LkEXe4Cq6BRWdZrwvEZX8hvf4ZfIbbg2VTNTBtCJDLLN8xyTk3MamGMoFM4FeBgpocJuQR7MCPvK2XNGCPxYUjuvsET7WDfph29IRFrVmy778p7\\\\\\\/qNeaO4Z20wGTVvomt88iuBfWQnlQgpLexcQ+Hz8fZkbJT1+wKg5RtWJLZnuKr9kKG3tjGmDXFTUdbrY3\\\\\\\/Gs0J4KYxAZOkE4gg6cnEXjNZWD1a61Ds2578Hgvg0cPijbxUQUl7+TS+Mn\\\\\\\/sbaUxU4hfBKWW+319ceULxgTQ4vTUBhUapgfSr6tunb8BUbxTgtWwG1orj2nYyYx2+oye8skBa+BErEAj2U7WnphuLPT0g2w0gmKtQCElgrk\\\\\\\/2p9N\\\\\\\/z2hnTz4OD5wXvSarNPOp8eujVbuN6ZUbLkOVU1Cvbrn04qN23x5fAmJtxDs1cpuuR4PAS39FKvPMbNkQxkcUiLxVUIqLgqizrJQ6XqG3KdEo0XcYzhg1aqHFwNN0Up5wGk6gBsALNEEehDnUkq+nTCslteMll5kUTSPw\\\\\\\/IagrXaeXLGgcLM7syAni7QtMBzB4Ffv\\\\\\\/mGAcynXZSHmuBz2wGzTJim5jM3hTc+Hvu0rsnDL3RlW5NRZuYGSwgB7ISNsoDrA0rDnJD3yQ\\\\\\\/RGf9rVR3RF\\\\\\\/Wo1Jl4df1arPgw7hEE1on\\\\\\\/oXngJiF7UDM0IBAj4F2XpdSDOEc3QmkRa6qpbccKK\\\\\\\/g9o9PmXsKu7PPFBvlMwFmjsllae8A0gIPA42XVm9IaNQ6KQYpW72EelYeVsPqkfP7ulPjoQc2kMqAXewcAwobrjcwoP6Gvl2VcvhYjIpg3JbS6aaPVg0kM0zKfEYg6Wqvx0vr7WUaoxLPyYsQ5aN0PbLdztRcJlZmd0lFwVctkNhedZuH8gDTxfVSKef81DYwNq0DrgYdvNKkpnNRruBZK6XSZheBN70\\\\\\\/bUiXdg4grytWUf91ijTScdacDAk5zX65\\\\\\\/uEM+HFy3K0r8PtBYQRHjRLdmDIcH5ng0333FrHnD5WJ6GmTNYwOoSs4viSFFAv3UIOuxH9eKKbnkh\\\\\\\/NvsHzK\\\\\\\/wMUh12wEwkdKSkpAq3gvCdEvT2kerEqauzwhL42wmXjJlYJzWWxG2+n97zg3L2\\\\\\\/ZpzdKgswrIIUXuODYBvOfKfXBgkkJ6\\\\\\\/mpq9w5gjCRGpZUWpkKphyX5iV6DYogW21BV639N384IY5oY6KnXWe+OJt+JxXOrk8WhEeUZCIRW21+nuxnOztB1XXRk\\\\\\\/VIuiRCloY\\\\\\\/cBkHCR+BnnN9IJt1e7Y81d6hBm8VUXUbGi\\\\\\\/9HNVldn1pvFj72XLFPAue2jxoCzdS97lsMD0S3lvvKlMH0iuvmjxh+G41Fb6OuHmKNWV5cMH7ueApbvZ\\\\\\\/3Wm15rvT8JDriS1E\\\\\\\/fOIjubTxClNld3jt7s37SoOh57U2MVab5FTHTXrBBVs3k6QlfLpq+uSUWmsK\\\\\\\/NQH154jB2q\\\\\\\/i+MS5jD8eYZBHEEk8T5pfOpW3uytlYNKIoO7jsiEEz1b\\\\\\\/QZfi1VcBnCq4rz80NFJdLI63ARrBVxqwbPNQqS2ZldM0VdF+psV5hLt20bvecfIrrxPNJTc4Eh3GEoOLQ0a7WRQx3h8hvBwSwq0AyAOj204S53\\\\\\\/KejkFNt9WrvgQuf7PYOo+rW7QcfwJkmRulZTYRlgrinP83uDF3KBZdqMSbMmNgrOjacrc7hWxttW\\\\\\\/7uZaCk0ySE6w0KSIl1QHfUPV\\\\\\\/vikebdRUtRVc09YurzLLHUS8dG\\\\\\\/3iDGVOGNHXfDES8TZFw+44+xWE4eon\\\\\\\/WC0u6KgxFf0qnnDOR8CtFFEfxjbZQQ7GZKCgLHKVQGPEGhF7vDjry8jHrOYpddAqTSoxvDxbDLolUOXKLxs8d8dmv013P8gJBfu2uTHh1Ni66X6ikhevwPRkKZdDdt2YUBlwCDatXU0OHOqjtgUUX8QyXDLVJ5NjUPw7m9icCOkxFZzkRoCKORn7+C8tTNLKp4wvdUF6A8\\\\\\\/CEqgBLQYRulGrwX+MS9bU2kRgW4WnHJPxZvmJMzeGtZPx6WS3VNKyD5mYtN\\\\\\\/VPYDgsfAQrTHlEBEXLb1QCcq76M03hMFWGxMOE+CHsqLK3GcRFDKsy8oFn4CQe6Y09plKscxHrHKsa0+UX\\\\\\\/7phtuRyq0gsJ7NCG0G+RhJidoBpVqBLKtukUpXah2K4ouJsAdYlD0MslCa9aXtByMElHVONATLD1Wd3XyDeoSW+vRYLEr5tkdn+uvMLpwMoyzbhMpinwF\\\\\\\/Js\\\\\\\/\\\\\\\/HdH8QbUbRLkGWDp6BP\\\\\\\/cYkp4BUIdcujzn9iXFOo5KDulwWZSTKJHBqlEvhiMp+aA3EXdSSg4yIQLlNzK7obCL2zW8deS+vCXm98tSHwXLe7gM0Qyk5taN+sR6OcXeSaKbrF2tkHaf32GZn2LWd55+Zb9EKbpPADjbs6fZQvoaYdukhCVHw3S7esLcoAXh9f4on78a9UqDc33igejBq9iVws3kodi1NgsDiAi0ai7ZWOmhuje3q9P8gW\\\\\\\/X1Zj9yGQMElo3s+sj+A2AI\\\\\\\/7PjHaxAkbSwvj7X2de+xIJQeNNpJlcXU\\\\\\\/9SyqMf\\\\\\\/mjy\\\\\\\/QxCFh40M4ratD921pUn\\\\\\\/WR4oT2GYiZQhX4a9x3PD6qc0dkfhPzV3SJQWl7p11xo2D8NIAz3J4eutqTzAgNi9PPNPFDZ1R2mzmpXZtGCWRbtzwJ1cKmA\\\\\\\/EI\\\\\\\/hKYPhFjSDJM70mJtJ6+8fw3IwKweZjWOxHTDWao3zcWbkZCH9AdZjg\\\\\\\/T9zTETB20s+9rzwa6Zeh9Q9o4c8txnLODwzNQL33IDX0RhMt44EYjbhstvX7yD1bB0YtjRFRNi+8Hq4OdqbTY9jvUMEOKlsbG59Cvpy3km0Z4+DsPqpTaVaQ5p6H9nPZXt\\\\\\\/XG\\\\\\\/JMId6LP6MUVpneaSJ9dhAmRUDe\\\\\\\/HMT6WJoBJb7Y5V8Q+T42co6DdNXPs2b07z\\\\\\\/qtXMxaCzgs0yOUuBfTvBG7AEubA0H6mLWMx31a7yff4eQUCMVVfPV2T0aW9GZZRrvG9kczZAgXOE0f5FfGfpSBOs0GtO08pjrcMIiw\\\\\\\/6McPPiaG2FWNPh7mHnHKFLG0EFqFcXYLo0\\\\\\\/HciJon\\\\\\\/N1eHPMVxFFEhS2LsN2F+kaJ5hy28I1kafX9i7Nrsb2At+ct54dvsQrUNADBZ2H0PCN0hTyhnneWtkyXlnIFmQfHfdj8AA4y\\\\\\\/UMny68Ah3ue5TuAX6JgdqiIsSp72\\\\\\\/XXw5zcOlvamlMlPc1E2CLlBbaR\\\\\\\/PB3gcEyxs8lMrDRpoIOUUey9Kq2MbSP9rQ1n8e0wJ\\\\\\\/zNkoR2fWWDerDxkPd\\\\\\\/MKR+bKAsJkZPOLT\\\\\\\/Ha5rQ\\\\\\\/ODpoxPezYPNK\\\\\\\/2k+6Pnk9DBFB9wd9CsWA+ACCNt7Y9NHZkyX40CerPV8YeiUpLGc4mVxbyNagCupvFjXCWui8DXv0kOMVWgS9KXeOPx7fB\\\\\\\/rjCCeiGbMh9PsPJ37IAVWXchaAqjnz7qsBjqNBb8njUvfwUNxMpAA4qBjOYZibfL\\\\\\\/FBnBiAxlEkgl+6eiHMdatu0MCnuKWvxI0Qk00kq7zCT3+tDbRn3u1DUZTV8RS5+ITIFm\\\\\\\/BHbfWmYIlNPq7yA1ldatQ\\\\\\\/rru5dsB0OQLpN9KnIL6QVzeLqEaGY5Rkn5DDLNgA8aa4GsjZ9b4bxI1OvyQCq2DTLjFyettNZ89W2x2Lw6JeL7FU5XBF1ngvGIPaxfvi8Tm7QrtKB2+eSLmtJUJWecEVxmbp4Hd0u0lIZhXjXBcZftInS6n6iex8WhjkSm7Ud09wd4pfFBtt4AryMwuZi0GU\\\\\\\/BHgutVa57emcP5H8MPT6hu0eH+j5CfF\\\\\\\/STDy1v8\\\\\\\/Mlk7SGPe4hpUnPhiNLQVVhZk3IcEAbwIDzGzM0+8csn101rJJZ7HGD+PXBPLmX0nwQseX3ZfaxMFH8HuwPjqMobfZhk\\\\\\\/wGSTacoxJi2BC33AW2+73d9Joiu0k\\\\\\\/soc1zgicSGWlSVh9\\\\\\\/Znpz8lCWMtF69g5Y2rGs+AWDATRnUzbDcYso4quXFaVW+l9b9vZbOh4FaGXl7wlPAssGFor9idsyXcoNFiXWpWhpcIU8Z0J\\\\\\\/2jlATHUEMYk3uFl6+OyVExHQeFS858Trp8OvMfo3SoeKZLtdprTsJA85VTJb\\\\\\\/Dn06INLGBEtw8gYGi9cubd6sLnqJcj85bwB1Je81yF4+avIqje1b+a8SjqMNSQL79XqSQrJc0jRWjdZKqpUjJBhcvvGV1zwFyA0ovMqYd1ZMRLNrIr4swvSzTYXFvncNWlVh5hlT3Hpahrx5Mz0MBkO31BSUo3RyjYFPSWToAyV5QVHMZAN0xFz+zmRz8v2OJWy1aX4MCLx\\\\\\\/8w\\\\\\\/2Gpf7zsg8Yj8ADMY3m0j4AZfjOk3jPWkYOoLYlqvRAKBO9NYFTEdF7akkXYQ2NEkK66jCzIQKWgqztEVLaO5Byg8VO3PLmKaWFPBVCmA\\\\\\\/8TllENbVHy126OdEu3DNaBrEbPm7UZxrMTajj8rFWG5noXgIA1ac6ItNt6Sb3b\\\\\\\/As4Z2HMkL9zNbLnpn07drT117NNR9oV1GzLnJEDIp\\\\\\\/8Tm06qLk6J9tlwSK7VsiAWEHepSE2tDTnDGuc4WPpWjFo0gWPRRgHhR5pUCGE76Qox\\\\\\\/E2ILYHRWjOtiPpEqlxgewMzrkBu9VXIXP9PiP65fomJtolbX2abvUCWKf2\\\\\\\/MZJKoYQm2Xk\\\\\\\/U3FhPdstTzD2KB4H\\\\\\\/VgIZtgYPv8xChi\\\\\\\/\\\\\\\/uXqlWeQc\\\\\\\/9ZVWPrpE3yvPNfBviHEDa3zLtWWyrxEE4Bu1hY17Ksn2zoEzIpDG+EPaRYFqpfR25FzlOrCtbd7x22\\\\\\\/pNxzCCa4RkQPVI2HjqVPiD6FPUmkNNpdFE2m2E0cH\\\\\\\/WrvYuQpgCZUbB3GdrYCPRqP3S2b06pA4kGtUcNXDDYjSrCG5DIjF0XU9zU7KA5anO6hl2qZGU4qjqUjxRhVt2DaNS\\\\\\\/Zleip1OeCGO274rcN4dk3kDX3MWmV9iux8SuWuAqKKd\\\\\\\/mw+zKNuWWJAGMT3TcckpyqqTVHTxXp0AqlB4mdsVG3\\\\\\\/wyPxWmisRVxzK3mj6rW9bGqKnDFGGirHLckiLlZW0r1vMvS35tSDgF9F4fUPfhACoWoMw27OPJAnmFq4gsaJINdoexLBrdke0yzUyQOzvT4eya1ZwOj4SLLUhXqhYIV5oQ8NhfTws5MrWAsKamaLwVEtIZzWqU+tnuMMjYDzI67kN+CJ3pmgldvfcmGRmogge0AmpfMVT5H4iX3UdudikXi8bWF\\\\\\\/YoUGZme9qoAiTPqeySnDqC9AjBcH4\\\\\\\/Z76qg9C2ka1qUs6k4VCC5E8p4ViQBxQJzhu8WkOf1PoCq6zAoNDZRw95ruc3nTd5IjvCzOTVs\\\\\\\/iRcvPGmhetpY57PDDEk3hYhoXNf3\\\\\\\/nUu\\\\\\\/4YQL2WnMFMctyZ1D5Ry4SVyYdJToqR5lsSavT0hp5gQBStqtEobJveOcA4VMGBOK3RPAi3N\\\\\\\/pDHKVQ8ias8zRtxPt5rzsut4PAvY3q0WBGgWwFML0xNHrn8yJ+jiRPQoJeNs5vlCmELIgxSgSep2GfavweLlXtrGKZKGr\\\\\\\/\\\\\\\/yl8p+6ZqCn0Ing5P2Tvyortc8uGmXGJr74beI3mzWcL9BKc3WjgXcR\\\\\\\/O7rQLLbWRcfZ7eC8EQhHQdG8bJOb\\\\\\\/ifpaSiIeyzMef4E3zelJ0y01BKk1g5Otfjwj99zUIIsXsOGEaC3HNKeOXSEnT+vyovxL8aI+A\\\\\\\/ic+\\\\\\\/4GWFR0vbtuWJbNHeTDO9ZxyNbBwkPX+H9Snh9FUclNamsjL+Ty+LfCTGock34lpALfNT0Vy6ib4YhDXh5LsHWtdbh2w2mVK8m2\\\\\\\/EtP10Z33C7NTWLeFtV63rVAUNK8ZjxzueIN\\\\\\\/sTbd1LzsoawH6Az7EONhQUMuOw2FJOV0Pixdni6td7ctC2dblvubXFI0KqCr38U0IUf\\\\\\\/oqxj1EZM31IBJ0QyobBRC22stmnXmYDeDmarFGD+Lw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022V\\\\\\\/+Fkrd6iMOpsgwvTbdehw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221ebbcc5a641eae87ab3254f1fa674b77904ce814fb8350a298c6432950e6ae8f\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HInlxKNPPmVP6rZByTSKwHbfUKxtOyTlczczGq4Qc3bWZtqx0KyfB7N9SftoxwKEGddh3QxwKACJUkzRG5UW0XaoCL2R9tpJtXdfcyb8drO76WeK5oeJ90Tnb0tWr35TVZZS2hk7KB+b7zAKEJmPnquBivph2iqh55Tne5Eznz198pAybGseRA2zfNbbGstn6Jdbnk1VuA0Ib4\\\\\\\/gWtOkTh5QUmONWrB9oIHGeMu2HnJk0QAYRVQ5OCmLBkK8pm3X4tAZCz01IoB2ttFhor8com90G0xF9oMsG1MiYlpkw\\\\\\\/LbJ4r5DfsJydD0RChQgowxDniGkM8wpkzbDdpaE+vWxvILJgzyISN8U9zUxwS3uHaT98S\\\\\\\/l7w9M56yiIoRPsL52MzEhpTzyNO2AB90a1aua0NPAtL\\\\\\\/VJGcaHvZFY2sse7zZ\\\\\\\/TbDeLeSxARgF19SFZWNE3ZMPmMmlXODtE7OBULk\\\\\\\/nicZIkqGGBWgKD8SbdalZWLkh0fNssqkYdaOyTGMs8GdU48icdvDq7RXwN3Xsr6MatpkTqweDNEPqtQveigHWbgP6QnM8vTKwsZdIuxUx0Blul9rJHKJrVDwRGndAC3J3PvzMgOjqexk7twsEW34HxeKM9DLrXQsk3q3x22E5hXlp1deGzBjgifyAPgoRCotU\\\\\\\/+yhW0cA\\\\\\\/0UlS2cSuccTI6zs4rgXEAqubHbS0s5qQrUo8dApO1kXVaaN6UZCJGSiWsrq\\\\\\\/bqUF3g3VkmjhiUWHSmJNy5ulhjn9k5M8srEQ6kkDWwAGhS4kI3okqC\\\\\\\/OMi18fZAo0B5WvYDHrG62NLKEUjyEKKcAMxmxwkXGXFgCF2KfBwHqr197\\\\\\\/NAKOB+In3ZR7oc7CqhlYBtMYTH2K9jXwP8EyRaycasAZNA+jw6FCogh8a4tz59D0AxmuwyMbaCI76zp+UQySYvbExkKHZSxz9LkCeartYh08LWeXUaN3CnY7CZ3UpYdsYyKRVxN+BEEZgtuMweCcc2G8GpES9HSUhSuDRkBL+LDDJN7mbgfx1kZniXA9O8Q0SJetZW9w0PhN18\\\\\\\/Gejdo9hY6tVhujdm84NV7MIn4ouqMSBXcKnaQhRLp2x2C8Xwfag4hdo23\\\\\\\/1jOkMZ0PWp03Q1Q9qDjDaFSq+7NNZYUZ67Zcs1PihpnXtZdFst5fEutQ9XtUywoAPh6OIxj6r5ptibY8VXrcq7UxbG3\\\\\\\/s48AJa684aSXHBNT85wRFlumeBt6dziaor\\\\\\\/9MhWfUI2P5JO3NZbX7kDVrzNKB3600uxjftSI7P\\\\\\\/NZsgzirtvnZimnZDKZjqQpUW+hLKWdjS1SfFPIrwcCp7BMExNnH7I5AhJH\\\\\\\/2Prh\\\\\\\/r8TpTdIeUOBtgz8rkRntL4K1tilBwAuDtbAV7VXVlwQwnUKEtUkngDNrUVtszCPj+Hlw9hNxISFnt7amqBX4nRss\\\\\\\/ExV\\\\\\\/P63debqG2AGmsT6MxZxKg8IfozhACq8Do3yxz844wUuYP8IwZmgJOTCEtT39FwYaagC3lf+\\\\\\\/N44bkQ9KBrRZL4GwHewtnpUqwo0IiRziQkR3NoPBhc7ji5fKv8Teffbmhpjy+8eGCrW8yT0r5z26gLRjFG2Pf5xakFxjgHoz7mhQjRSVQdUEti7cNHshPfieja2uBycl4etZ+0biqTUp5q0rsdrmFgzXSKlc4CL4jfOzy0JKJCoZL6Q2A1ffC+rl6LyEiYZq+f1YbYEjj5gaU41Zp\\\\\\\/wGKOY36742nmrCwARPI\\\\\\\/zoHSPIJ7EaqnJ1GlZaX2i1CtJiP+Qg\\\\\\\/QuSitq0HCua4ODHlFq4oIsufv0DKT9J4eV2I2+EluwDwy89rLACBHcV2x6Z5B8+dlFG\\\\\\\/Dj8C5YoExdEupzE2u8BM+2+Virtx6jZCizEiJ5i4P\\\\\\\/sggVV1q3daLxjO+8joK1vPMvXCxixzJI9bwW3BP3d01sB2fN2qW34pKRZqNGvo9s24JgSzp8nSkMQhryQGZNEgygDwC7WtXDccxOiw5FIB\\\\\\\/bgPDl6BGJ4EV8zXw+Z1s8w==\\\u0022,\\\u0022iv\\\u0022:\\\u00222\\\\\\\/TYpgxJYwwr14RQ3PN5TQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228f248f995e5c83a7e2b59e2db084a8ce2a6b88089bf16a2e0192d509c696e7b1\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495563 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/581" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jN7HAMn5ho2sUqBICO9iA_w5bg_Dg-vkRxD5WQrVHVQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/581" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ekA33KMKsgwkQbeCgFcV5y6Y4ctMysJg4fJ10x4-BLs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/2/569" title="Simulation-Based Evaluation of Three Methods for Local Ancestry Deconvolution of Non-model Crop Species Genomes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/2/591" title="Evaluation of Saccharomyces cerevisiae Wine Yeast Competitive Fitness in Enologically Relevant Environments by Barcode Sequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495563" id="node495563" data-pisa="ggg;10/2/581" data-pisa-master="ggg;g3.119.400800" data-apath="/ggg/10/2/581.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Optimizing Low-Cost Genotyping and Imputation Strategies for Genomic Selection in Atlantic Salmon</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0792-9421" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Smaragda Tsairidou</span>, <span class="highwire-citation-author" data-delta="1">Alastair Hamilton</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-9616-5912" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Diego Robledo</span>, <span class="highwire-citation-author" data-delta="3">James E. Bron</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-1805-0762" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ross D. Houston</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">581-590;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400800 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495563" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Smaragda Tsairidou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Edinburgh, EH25 9RG, United Kingdom,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Smaragda%2BTsairidou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tsairidou%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASmaragda%2BTsairidou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0792-9421" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Smaragda Tsairidou</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Smaragda.Tsairidou@roslin.ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">Smaragda.Tsairidou@roslin.ed.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alastair Hamilton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Hendrix Genetics Aquaculture BV/ Netherlands Villa ’de Körver’, Spoorstraat 695831 CK Boxmeer, The Netherlands, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alastair%2BHamilton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hamilton%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlastair%2BHamilton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Diego Robledo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Edinburgh, EH25 9RG, United Kingdom,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BRobledo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Robledo%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADiego%2BRobledo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9616-5912" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Diego Robledo</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">James E. Bron </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Aquaculture, University of Stirling, FK9 4LA, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BE.%2BBron%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bron%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BE.%2BBron%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ross D. Houston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Edinburgh, EH25 9RG, United Kingdom,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ross%2BD.%2BHouston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Houston%20RD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoss%2BD.%2BHouston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1805-0762" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ross D. Houston</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/2/581" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495563" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495563/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/2/581.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495563" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495563/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/2/581.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495563" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495563/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/2/581.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495563" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495563/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genomic selection enables cumulative genetic gains in key production traits such as disease resistance, playing an important role in the economic and environmental sustainability of aquaculture production. However, it requires genome-wide genetic marker data on large populations, which can be prohibitively expensive. Genotype imputation is a cost-effective method for obtaining high-density genotypes, but its value in aquaculture breeding programs which are characterized by large full-sibling families has yet to be fully assessed. The aim of this study was to optimize the use of low-density genotypes and evaluate genotype imputation strategies for cost-effective genomic prediction. Phenotypes and genotypes (78,362 SNPs) were obtained for 610 individuals from a Scottish Atlantic salmon breeding program population (Landcatch, UK) challenged with sea lice, <em>Lepeophtheirus salmonis</em>. The genomic prediction accuracy of genomic selection was calculated using GBLUP approaches and compared across SNP panels of varying densities and composition, with and without imputation. Imputation was tested when parents were genotyped for the optimal SNP panel, and offspring were genotyped for a range of lower density imputation panels. Reducing SNP density had little impact on prediction accuracy until 5,000 SNPs, below which the accuracy dropped. Imputation accuracy increased with increasing imputation panel density. Genomic prediction accuracy when offspring were genotyped for just 200 SNPs, and parents for 5,000 SNPs, was 0.53. This accuracy was similar to the full high density and optimal density dataset, and markedly higher than using 200 SNPs without imputation. These results suggest that imputation from very low to medium density can be a cost-effective tool for genomic selection in Atlantic salmon breeding programs.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/salmon-breeding" class="hw-term hw-article-keyword hw-article-keyword-salmon-breeding" rel="nofollow">Salmon breeding</a></li><li class="kwd"><a href="/keyword/genotype-imputation" class="hw-term hw-article-keyword hw-article-keyword-genotype-imputation" rel="nofollow">genotype imputation</a></li><li class="kwd"><a href="/keyword/aquaculture" class="hw-term hw-article-keyword hw-article-keyword-aquaculture" rel="nofollow">aquaculture</a></li><li class="kwd"><a href="/keyword/sea-lice-resistance" class="hw-term hw-article-keyword hw-article-keyword-sea-lice-resistance" rel="nofollow">sea lice resistance</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">genomic prediction</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">Genomic selection is increasingly applied in aquaculture breeding to expedite genetic gain for key production and disease resistance traits (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Zenger <em>et al.</em> 2019</a>). Genomic selection exploits both between- and within-family genetic information, and therefore can predict the breeding values of individuals more accurately than using traditional pedigree-based approaches. However, routine and effective application of genomic selection in breeding programs depends on large training datasets of phenotypes and genotypes, which can be expensive. In livestock and plant breeding, genomic information on large populations can be achieved in a very cost-effective manner by means of genotype imputation. Imputation exploits the presence of haplotypes that are shared between related individuals (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Li <em>et al.</em> 2009</a>), such that high density (HD) imputed genotypes can be obtained from low density (LD) SNP panels. This can facilitate a major reduction in the cost of genotyping, as direct genotyping at HD is required only for a subset of individuals, <em>i.e.</em>, the reference panel. However, applications of genotype imputation in aquaculture breeding are at a formative stage, and optimal strategies need to be defined.</p><p id="p-4">In typical aquaculture breeding programs large full-sibling families are available, and performance testing is routinely performed on siblings of selection candidates. Full-siblings share large genomic segments, which results in relatively high genomic prediction accuracy with low density SNP genotypes, when compared to the SNP densities typically required in other farmed animal species (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Tsai <em>et al.</em> 2016</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Vallejo <em>et al.</em> 2017</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Palaiokostas <em>et al.</em> 2018</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Robledo <em>et al.</em> 2018</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Yoshida <em>et al.</em> 2018</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Yoshida <em>et al.</em> 2019</a>). Further, high density SNP arrays have been developed for Atlantic salmon (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Houston <em>et al.</em> 2014</a>), and a high quality reference genome assembly is available to map those SNPs and determine their order (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Lien <em>et al.</em>, 2016</a>). Therefore, given the family structure within a typical aquaculture setting, a first step is to identify the minimum SNP panel density required to achieve maximum genomic prediction accuracy. Furthermore, optimizing the composition of these SNP panels needs to be explored to assess impact on the prediction accuracy. Previous genotype imputation studies have highlighted the potential of harnessing medium or high density genotyping of parents and low density genotyping of offspring to reduce costs of genomic prediction (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Tsai <em>et al.</em> 2017</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Yoshida <em>et al.</em> 2018</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Dufflocq <em>et al.</em> 2019</a>). In these studies, the genomic prediction accuracy that could be achieved when using LD genotypes imputed to HD was found to be comparable to the prediction accuracy when using the full HD panel in Atlantic salmon (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Tsai <em>et al.</em> 2017</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Yoshida <em>et al.</em> 2018</a>), and in simulated data for rainbow trout (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Dufflocq <em>et al.</em> 2019</a>). However, a systematic analysis of the optimal combination of low and high density panels, and of the effects of SNP panel composition, <em>i.e.</em>, which SNPs are selected for the reduced density panels, has not been yet performed.</p><p id="p-5">One of the most important target traits for genetic improvement in salmon breeding is host resistance to infectious diseases such as sea lice. Sea lice are parasitic marine copepods of the family of Caligidae, endemic in all major salmon-producing countries (<em>Lepeophtheirus salmonis</em> in Europe and North America, and <em>Caligus rogercresseyi</em> in Chile), and it is the most costly disease-related problem for the global salmon industry (total cost of sea lice control is estimated to be €305 million in 2006 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Costello 2009</a>)). Sea lice infestation has a severe negative impact on salmon health and production, causing substantial financial losses due to treatment costs, associated labor and increased morbidity and mortality due to infestation or secondary bacterial and fungal infections (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Gjerde <em>et al.</em> 2011</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bruno <em>et al.</em> 2013</a>). In addition, sea lice show variability in their sensitivity to common treatments, and in some instances, they express high levels of resistance to commonly-used therapeutants (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Tully and Mcfadden 2000</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Sevatdal and Horsberg 2003</a>). Encouragingly, host resistance to sea lice has shown significant heritabilities of 0.22-0.33 (<em>e.g.</em>, <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Gjerde <em>et al.</em> 2011</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Tsai <em>et al.</em> 2016</a>). Another economically important production trait is body weight, which reflects growth performance. Body weight has been reported to have a high heritability of 0.5-0.6 (<em>e.g.</em>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Tsai <em>et al.</em> 2015</a>). In addition, both sea lice resistance and body weight are known to have a polygenic genetic architecture (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Tsai <em>et al.</em> 2015</a>; <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Tsai <em>et al.</em> 2016</a>). Hence, these traits lend themselves to improvement by genomic selection approaches, and cost-efficient genotyping approaches employing genotype imputation may encourage widespread commercial adoption (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Tsai <em>et al.</em> 2017</a>).</p><p id="p-6">The aim of this study was to systematically evaluate and develop optimal genotype imputation strategies for Atlantic salmon breeding programs. Using data from a typical commercial breeding program, several genotype imputation strategies for genomic selection were tested by (a) identifying high density SNP panels with optimal density and composition; (b) testing the imputation accuracy when parents were genotyped at high density, and offspring at a range of low densities (c) comparing the genomic prediction accuracies of each of the datasets, and (d) assessing the cost-benefit of imputation for genomic prediction by quantifying the trade-off between the cost of genotyping at a given density and the achieved improvement in genomic prediction accuracy.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Phenotypes and data transformations</h3><p id="p-7">The data used for testing genotype imputation and genomic prediction were collected from an Atlantic salmon commercial population (Landcatch, UK), as described in (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Tsai <em>et al.</em> 2015</a>). Briefly, the data comprised of 520 phenotyped and genotyped post-smolt salmon (267 males and 253 females), offspring of 29 sires and 57 dams. Each sire was mated to two dams (except for one sire mated to one dam), so that the offspring were full-sibs / paternal half-sibs, derived from 57 nuclear families with 4 - 14 offspring per family.</p><p id="p-8">The population was challenged with sea lice (<em>Lepeophtheirus salmonis</em>) as part of a trial conducted at the Marine Environmental Research Laboratory (Machrihanish, UK) in 2007, as described in (<a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Tsai <em>et al.</em> 2015</a>). Phenotypic records of sea lice counts (SLC) were collected for all offspring. SLC ranged from 1 to 81 lice per animal, and had a positively skewed distribution with a mean of 25.5 and median 23.5 (Supplementary Information 1). SLC was logarithmically transformed and log<sub>e</sub>(SLC), with a mean of 3.11 and median of 3.16, was used in all subsequent analysis (Supplementary Information 1). The weight of the offspring ranged from 52 to 203 g with a mean of 111.8 g. The weight data were normally distributed and therefore no transformation was required.</p></div><div id="sec-3" class="subsection"><h3>Genotypes and quality control</h3><p id="p-9">All samples were genotyped with the Affymetrix Axiom 132K Atlantic salmon SNP chip (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Houston <em>et al.</em> 2014</a>) such that genotypes were available for all animals for 78,362 high quality, mapped and ordered SNP markers. Families with &gt; 5% and individuals with &gt; 10% Mendelian errors were excluded using Plink/1.90-beta4.1 (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Purcell <em>et al.</em> 2007</a>). Further quality control was conducted for the offspring using the GenABEL package (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aulchenko <em>et al.</em> 2007</a>) on R/3.1.2 (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">R Development Core Team 2008</a>) with the following criteria: SNPs with a minor allele frequency (MAF) &lt; 5%, call rates &lt; 95%, or Hardy-Weinberg Equilibrium P-value &lt; 10<sup>−5</sup> were excluded. Individuals with a percentage of missing genotypes &gt; 3% were also excluded. In total, 76,488 SNPs (hereafter referred to as the HD SNP panel), and 520 offspring passed all criteria, and were used to calculate the genomic relationship matrices for each SNP panel.</p></div><div id="sec-4" class="subsection"><h3>Calculating genomic relationships</h3><p id="p-10">The identity by state (IBS) genomic relationship matrix (<strong>G</strong>) was calculated from the GenABEL/R (“ibs” function) kinship matrix (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Amin <em>et al.</em> 2007</a>) multiplied by two, so that the genomic relationship between animals <em>i</em> and <em>j</em> was given by:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/581/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/581/embed/mml-math-1.gif"/></noscript></span></span>Where <em>n</em> is the number of loci used for estimating relationships; <em>x<sub>ik</sub></em> is the count of alternative alleles (0, 1 or 2) of individual <em>i</em> at SNP locus <em>k</em> where the reference allele is arbitrarily chosen; and, <em>p<sub>k</sub></em> is the frequency of the reference allele in the data.</p><p id="p-11">The diagonal elements were given by <em>g<sub>ii</sub> = 1+F<sub>i</sub></em>, where F<sub>i</sub> is the inbreeding coefficient for individual <em>i</em>, calculated using SNP genotypes as follows: <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/581/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/581/embed/mml-math-2.gif"/></noscript></span></span>, where <em>H<sub>E,k</sub></em> is the expected heterozygosity at locus <em>k</em> assuming Hardy‐Weinberg equilibrium (HWE); and <em>H<sub>ik</sub></em> is the observed heterozygosity for animal <em>i</em>. Subsequently <strong>G</strong> was inverted (R “solve” function) to be used in the ASReml (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gilmour <em>et al.</em> 2009</a>) analysis described below.</p><p id="p-12">Structure exploration using classical multi-dimensional scaling analysis (function “cmdscale” in package “stats”/R/3.1.2) did not identify any sub-clustering in the data (Supplementary Information 1).</p></div><div id="sec-5" class="subsection"><h3>Statistical Analysis</h3><p id="p-13">The SNP panels used throughout the analysis were defined as follows: ‘<em>HD SNP panel</em>’ was the full high density panel containing 76,488 SNPs after quality control; ‘<em>reduced SNP panels</em>’ were the reduced density panels generated from the HD SNP panel to test genomic prediction at lower SNP densities; ‘<em>optimal SNP panel</em>’ (medium density) was the selected 5K SNP panel (see ‘Identification of optimal SNP panels’ section); and, ‘<em>imputation SNP panels</em>’ were the low density SNP panels assumed to be genotyped on the offspring, selected from the optimal SNP panel to test imputation.</p><p id="p-14">The analyses comprised three main parts: (a) identification of optimal SNP panels in terms of density and SNP composition; (b) assessment of the imputation accuracy of a range of lower densities of imputation SNP panels, when imputed to the optimal SNP panel identified in (a); and, (c) estimation of genomic prediction accuracies of imputed datasets and comparison with genomic prediction accuracies of datasets without imputation. All analyses were performed via a custom pipeline that was developed integrating: (i) PLINK /1.90-beta4.1 (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Purcell <em>et al.</em> 2007</a>), R/3.1.2 and Shell for selection of SNP panels, and calculation of genomic relationship matrices; (ii) FImpute/2.2 (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Sargolzaei <em>et al.</em> 2014</a>) for genotype imputation; and (iii) ASReml/3.0 for estimating breeding values. The code used for the construction of SNP panels and for the multiple cross-validation procedure can be accessed on GitHub (<a href="https://github.com/SmaragdaT/CVrep">https://github.com/SmaragdaT/CVrep</a>) and is available as an R package (CVrepGPAcalc v1.0 / R version 3.1.2) under GNU General Public License v3.0 (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Tsairidou 2019</a>; CVrepGPAcalc v1.0).</p><div id="sec-6" class="subsection"><h4>Identification of optimal SNP panels:</h4><p id="p-15">Previous research has shown that genomic prediction accuracy has little or no reduction as SNP panel density is reduced from 132K down to approximately 5K (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Tsai <em>et al.</em> 2016</a>). The focus of the selection of optimal SNP panels was on identifying an approximate minimal SNP density which could be considered the asymptote of prediction accuracy. From the HD SNP panel SNP, reduced panels were generated for densities of 60,000, 10,000, 5,000, 2,000, and from 1,000 down to 200 SNPs at 100 SNP density intervals, using in-house built software which is available as an R/3.1.2 package (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Tsairidou 2019</a>; CVrepGPAcalc v1.0). Two different methods for selecting SNPs were used: (i) SNPs were sampled randomly without replacement across the entire genome; and (ii) SNPs were sampled randomly, without replacement, within each chromosome, and the number of SNPs sampled from each chromosome was proportional to the physical length of the chromosome as calculated using the <em>salmo salar</em> reference genome assembly (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Lien <em>et al.</em> 2016</a> Genbank accession GCA_000233375.4). For method (ii), in some instances, the total number of SNPs selected across the genome was allowed to be slightly higher than the target density due to rounding. Furthermore, for higher densities, the number of SNPs available on some chromosomes could be smaller than the required number, therefore the total number of SNPs selected across the genome was allowed to be smaller than the intended density (Supplementary Information 2; Table S1).</p><p id="p-16">The process was replicated 10 times so that for each density, 10 SNP panels were generated, which were allowed to partly overlap by chance (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). For each density and each replicate, the genomic relationship matrices were calculated for the offspring as described above using the SNPs that passed quality control.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/581/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Experimental design." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1942647353" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Experimental design.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/581/F1.medium.gif" width="440" height="228"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/2/581/F1.medium.gif" width="440" height="228"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/581/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/581/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495724" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-17" class="first-child">Experimental design.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">An additional method for SNP selection was trialled whereby SNPs were chosen based on even physical distance intervals, but results were similar to the two random selection methods described above, and are reported in Supplementary Information 3 (Figure S3). Briefly, SNPs were selected so that they would be on average equally spaced within each chromosome with a ‘step’ of 0.5, 1, 3, 5, 7, 9, and 11 Mbp between them. The first and last SNPs on each chromosome were always selected. If given the ‘step’ and physical position of the subsequent SNP, the same SNP happened to be selected twice, then the next SNP was included in the panel instead. The larger the ‘step’ the lower the density (Supplementary Information 3; Table S3).</p><div id="sec-7" class="subsection"><h5>Cross validation to test genomic prediction accuracy</h5><p id="p-19">Genomic prediction accuracy in the offspring was assessed through cross validation. The offspring were randomly partitioned into five groups of equal size, each time masking the phenotypes of one of the groups (validation set, n<sub>v</sub> = 104). Subsequently, using the phenotypes of the remaining four groups (training set n<sub>t</sub> = 416) and the genomic relationship matrix, breeding values (BVs) were predicted for each of the validation sets fitting the following mixed model in ASReml/3.0:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/581/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/581/embed/mml-math-3.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>Where, <strong>y</strong> is the response variable (Log<sub>e</sub>(SLC) or body weight); <strong>m</strong> is the overall mean, <strong>b</strong> is the vector of fixed effects (sex and weight for the Log<sub>e</sub>(SLC) model; sex for the body weight model) associated with the incidence matrix <strong>X</strong>; <strong>a</strong> is the vector of random animal effects with <strong>a</strong>∼<strong>MVN</strong>(0,<strong>G</strong>σ<sub>a</sub><sup>2</sup>) associated with the incidence matrix <strong>Z,</strong> where σ<sub>a</sub><sup>2</sup> is the additive genetic variance; and, <strong>e</strong> is the residual error with <strong>e</strong>∼<strong>MVN</strong>(0,<strong>I</strong>σ<sub>e</sub><sup>2</sup>). The mean correlation between the predicted BVs (<em>ŷ</em>) and the phenotypes (<em>y</em>) was calculated for the five validation sets.</p><p id="p-20">In order to assess the effects of random sampling on the prediction accuracy (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Tsairidou <em>et al.</em> 2014</a>), the cross-validation process was replicated fifty times, where for each replicate a new randomization of individuals into groups was performed. For each high density SNP panel, the overall mean correlation and standard deviation were obtained over the 50 replicates (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p><p id="p-21">For each SNP panel and for each replicate, prediction accuracy was calculated as the mean correlation divided by the square root of the heritability <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/581/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/581/embed/mml-math-4.gif"/></noscript></span></span>, for Log<sub>e</sub>(SLC) and for weight. This heritability was calculated from the whole dataset, all 520 offspring and with <strong>G</strong> calculated on all SNP markers after quality control with the same criteria as above (76,488 SNPs), and using model (1). Then for each SNP density, the average accuracy (and standard deviation) was obtained over the 10 SNP panel replicates.</p></div></div><div id="sec-8" class="subsection"><h4>Imputation SNP panels and assessment of imputation accuracy:</h4><p id="p-22">A range of imputation SNP panels were constructed as a subset of the optimal SNP panel identified in (a) (<em>i.e.</em>, the lowest density SNP panel that rendered genomic prediction accuracies for both traits similar to those obtained with the HD SNP panel). For the imputation panels, SNPs were selected randomly within each chromosome (proportional to chromosome length as previously described). Due to rounding, it was possible that the total number of selected SNPs could be slightly larger than the intended density (Supplementary Information 2; Table S2). The optimal SNP panel was assumed to be genotyped on all parents (n = 86) and the imputation SNP panels of varying densities were assumed to be genotyped for all offspring (n = 520), so that the offspring were imputed to the optimal SNP panel density (parents’ genotypes). Imputation was conducted using FImpute/2.2 (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Sargolzaei <em>et al.</em> 2014</a>). Imputation accuracy was calculated as the correlation between imputed and observed genotypes in the offspring (<a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Yoshida <em>et al.</em> 2018</a>). Then the average imputation accuracy across individuals was calculated for each of the low density imputed SNP panels.</p></div><div id="sec-9" class="subsection"><h4>Genomic prediction accuracies of imputed datasets:</h4><p id="p-23">Estimated breeding values for log<sub>e</sub>(SLC) and body weight, and cross-validated prediction accuracies were estimated in ASReml/3.0 using model (1), with the difference that here the <strong>G</strong> matrices were calculated from both directly genotyped and imputed SNP genotypes for the offspring. Quality control analysis described in section ‘Genotypes and quality control’ was repeated for the imputed genotypes. The multiple cross-validation procedure was used as described above, and for each imputed dataset, the average prediction accuracy was calculated over 50 replicates.</p></div></div><div id="sec-10" class="subsection"><h3>Cost-benefit analysis using imputed data</h3><p id="p-24">A cost-benefit analysis was performed to evaluate the economic benefit of genotyping the selection candidates at lower densities and performing genotype imputation considering the genomic prediction accuracy obtained. The scenarios tested were (i) all animals (parents and offspring) genotyped at medium density with a 5K SNP panel, and, (ii) parents genotyped at 5K and offspring at 200 SNPs and imputed to 5K. The annual cost of genotyping was estimated as follows: for a salmon breeding program assuming 300 parents and 10,000 offspring, for scenario (i) a cost of $15 per individual was assumed; for scenario (ii) 300 parents genotyped at 5K were assumed to cost $30 per SNP array (the price per array increases with lower volume of samples), and 10,000 offspring genotyped at 200 SNPs were assumed to cost $5 per individual to genotype the SNP panel.</p></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-25">The code used in the current study can be accessed on GitHub (<a href="https://github.com/SmaragdaT/CVrep">https://github.com/SmaragdaT/CVrep</a>), and is available as an R package (CVrepGPAcalc v1.0 / R version 3.1.2) under GNU General Public License v3.0 (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Tsairidou 2019</a>; CVrepGPAcalc v1.0). To install in RStudio use: install.packages(“devtools”); library(devtools); install_github(“SmaragdaT/CVrep”, subdir=”CVrepGPAcalc”); library(CVrepGPAcalc). The genotype and phenotype data can be accessed at <a href="https://www.g3journal.org/content/7/4/1377.supplemental">https://www.g3journal.org/content/7/4/1377.supplemental</a> (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Tsai <em>et al.</em> 2017</a>). Supplementary Information 1: sea lice count distribution and data structure exploration using classical multi-dimensional scaling; Supplementary Information 2: number of SNPs selected when SNPs were sampled randomly within each chromosome; Supplementary Information 3: genomic prediction accuracies when SNPs were selected based on physical distance; Supplementary Information 4: correlations between phenotypes and cross-validated predicted BVs for the different HD SNP panel densities; Supplementary Information 5: distributions of imputation accuracy values across individual animals; Supplementary Information 6: genomic prediction accuracy after removing SNPs with minor allele frequency less than 0.3. Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.9929945">https://doi.org/10.25387/g3.9929945</a>.</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Genetic parameters and testing of genomic prediction with reduced density panels</h3><p id="p-26">The genomic heritability using the HD SNP panel was found to be 0.19 (SE = 0.07) for sea lice resistance, and 0.57 (SE = 0.07) for body weight which was generally in agreement with findings from earlier studies, either based on the same (<em>e.g.</em>, <a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">Tsai <em>et al.</em> 2015</a>; <a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">Tsai <em>et al.</em> 2016</a>) or different populations (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cáceres <em>et al.</em> 2019</a>).</p><p id="p-27">For the different reduced SNP panel densities the correlations between phenotypes and cross-validated predicted BVs reflected the increase in information with increasing SNP density (Supplementary Information 4). Reducing the SNP panel density from the HD SNP panel (76,488 SNPs) to 200 SNPs resulted in a 14.5% decrease in genomic prediction accuracy for sea lice resistance, and a 27.9% decrease for body weight (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>; SNPs selected randomly within chromosome). Across all SNP densities, prediction accuracies for body weight were higher than those for sea lice resistance. Predictions for weight appeared to benefit slightly more from using higher SNP densities compared to sea lice resistance (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>), which may reflect that body weight is potentially a more polygenic trait than sea lice resistance. Both methods of selecting SNPs to construct reduced SNP panels (randomly across the entire genome or within each chromosome) showed similar patterns performed similarly with regards to genomic prediction accuracy.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/581/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracy for sea lice resistance and body weight for the HD and the reduced SNP panels, when SNPs were sampled (i) randomly across the genome, and (ii) randomly within each chromosome, proportionally to chromosome length. Vertical bars represent the standard deviations over 10 SNP panel replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1942647353" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Prediction accuracy for sea lice resistance and body weight for the HD and the reduced SNP panels, when SNPs were sampled (i) randomly across the genome, and (ii) randomly within each chromosome, proportionally to chromosome length. Vertical bars represent the standard deviations over 10 SNP panel replicates.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/581/F2.medium.gif" width="440" height="388"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/2/581/F2.medium.gif" width="440" height="388"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/581/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/581/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495692" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-28" class="first-child">Prediction accuracy for sea lice resistance and body weight for the HD and the reduced SNP panels, when SNPs were sampled (i) randomly across the genome, and (ii) randomly within each chromosome, proportionally to chromosome length. Vertical bars represent the standard deviations over 10 SNP panel replicates.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">Regardless of the trait, prediction accuracy started to decrease between 2,000 and 5,000 SNPs, therefore the medium SNP density of 5,000 was tested as the optimal SNP panel for the imputation analyses. This medium SNP density was used rather than the HD SNP panel because it was assumed that a 5K SNP panel would be cheaper to genotype in most circumstances.</p></div><div id="sec-14" class="subsection"><h3>Variability of cross-validated prediction accuracy</h3><p id="p-30">While the reduction in genomic prediction accuracy with the reduced SNP densities was modest, the standard deviations (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>) and the variances (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>) were substantially larger at these lower densities. In other words, there was substantial variability in accuracy between SNP panel replicates at lower densities, which means the accuracy depends on the SNPs included on the panel. In contrast, variability over SNP panel replicates diminished for SNP densities of 5K of larger. This phenomenon was more pronounced for sea lice resistance than for body weight (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/581/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Variance of prediction accuracy of the reduced SNP panels, for sea lice resistance and weight, when SNPs were sampled (i) randomly across the genome, and (ii) randomly within each chromosome, proportionally to chromosome length." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1942647353" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Variance of prediction accuracy of the reduced SNP panels, for sea lice resistance and weight, when SNPs were sampled (i) randomly across the genome, and (ii) randomly within each chromosome, proportionally to chromosome length.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/581/F3.medium.gif" width="440" height="388"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/2/581/F3.medium.gif" width="440" height="388"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/581/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/581/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495640" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-31" class="first-child">Variance of prediction accuracy of the reduced SNP panels, for sea lice resistance and weight, when SNPs were sampled (i) randomly across the genome, and (ii) randomly within each chromosome, proportionally to chromosome length.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">Another source of variation in the genomic prediction accuracy is due to the random sampling of individuals into the cross-validation groups, which will depend to some extent on sample size. Interestingly, a single prediction accuracy from one cross-validation can be a major over- or underestimate of the prediction accuracy. For example, for one of the medium density 5000 SNP panels, the prediction accuracy for sea lice resistance ranged from 0.40 to 0.62 over the 50 cross-validation replicates, with a mean of 0.52 (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Similarly for weight and for the same 5000 SNP panel, the prediction accuracy ranged from 0.55 to 0.64, with a mean of 0.59 (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Therefore, in the present study a multiple cross-validation procedure was followed, where individuals were re-allocated into 5 cross-validation groups 50 times. A larger sampling size would be expected to make the distribution of the prediction accuracies narrower and minimize the random sampling effects. This may be good practice for future studies of genomic selection in aquaculture breeding to avoid sampling bias.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/581/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracy across 50 cross-validation repeats for the 200 and 5000 SNP panels, for sea lice resistance and body weight." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1942647353" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Prediction accuracy across 50 cross-validation repeats for the 200 and 5000 SNP panels, for sea lice resistance and body weight.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/581/F4.medium.gif" width="440" height="309"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/10/2/581/F4.medium.gif" width="440" height="309"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/581/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/581/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495686" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-33" class="first-child">Prediction accuracy across 50 cross-validation repeats for the 200 and 5000 SNP panels, for sea lice resistance and body weight.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Imputation accuracy</h3><p id="p-34">Imputation accuracy increased as the density of the imputation SNP panels increased (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>) which was as expected based on previous studies (<em>e.g.</em>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hayes <em>et al.</em> 2012</a>). For imputation to 5000 SNPs, the accuracy ranged from 0.72 (SD = 0.03) for 200 SNPs, to 0.94 (SD = 0.02) for the SNP panel comprising 1000 SNPs, reaching &gt; 90% imputation accuracy for 700 SNPs or more. The standard deviation of the imputation accuracies among individuals reduced for higher density imputation panels (SD = 0.034 for 200 SNPs, and SD = 0.016 for 1000 SNPs; <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). Imputation accuracy per individual ranged from 0.61 to 0.82 when imputing from 200 SNPs, and from 0.86 to 0.96 when imputing from 1000 SNPs (Supplementary Information 5).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/581/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Imputation accuracy for the imputation SNP panels, when imputed to the optimal SNP panel. Vertical bars represent the standard deviations over 520 offspring." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1942647353" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Imputation accuracy for the imputation SNP panels, when imputed to the optimal SNP panel. Vertical bars represent the standard deviations over 520 offspring.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/581/F5.medium.gif" width="440" height="198"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/10/2/581/F5.medium.gif" width="440" height="198"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/581/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/581/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495683" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-35" class="first-child">Imputation accuracy for the imputation SNP panels, when imputed to the optimal SNP panel. Vertical bars represent the standard deviations over 520 offspring.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Genomic prediction accuracy of imputed datasets</h3><p id="p-36">Despite the higher imputation accuracy of higher density imputation SNP panels (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>), increasing the density of the imputation SNP panel for sea lice resistance resulted in an only marginal increase in genomic prediction accuracy achieved using imputed genotypes (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). For example, 200 SNPs imputed to 5,000 provided a prediction accuracy of 0.53, while the prediction accuracy for 1000 SNPs imputed to 5,000 was 0.56, and with 5,000 true genotypes was 0.54. For body weight, genomic prediction accuracies using SNPs imputed to 5,000 were slightly higher when higher imputation SNP panel densities were used (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/581/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracy of imputation SNP panels for sea lice resistance and body weight, for (i) the imputation SNP panel densities with range from 200 to 1,000 SNPs imputed to the optimal 5,000 SNP panel (Imputed_5000), (ii) the imputation SNP panel densities with range from 200 to 1,000 SNPs alone without imputation (LD_only), and, (iii) the optimal 5,000 SNP panel directly genotyped without imputation (HD_5000)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1942647353" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Prediction accuracy of imputation SNP panels for sea lice resistance and body weight, for (i) the imputation SNP panel densities with range from 200 to 1,000 SNPs imputed to the optimal 5,000 SNP panel (Imputed_5000), (ii) the imputation SNP panel densities with range from 200 to 1,000 SNPs alone without imputation (LD_only), and, (iii) the optimal 5,000 SNP panel directly genotyped without imputation (HD_5000).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/581/F6.medium.gif" width="440" height="388"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/10/2/581/F6.medium.gif" width="440" height="388"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/581/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/581/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495512" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-37" class="first-child">Prediction accuracy of imputation SNP panels for sea lice resistance and body weight, for (i) the imputation SNP panel densities with range from 200 to 1,000 SNPs imputed to the optimal 5,000 SNP panel (Imputed_5000), (ii) the imputation SNP panel densities with range from 200 to 1,000 SNPs alone without imputation (LD_only), and, (iii) the optimal 5,000 SNP panel directly genotyped without imputation (HD_5000).</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">While the influence of the SNP density of the imputation panel on genomic prediction accuracy might be minor, the benefit of imputation <em>vs.</em> using the low-density SNP panel alone (without imputation) varied according to SNP density. As a general pattern, genomic prediction accuracies using genotypes imputed to the medium density 5,000 SNPs optimal panel were very close to those using true genotypes for the same density (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figures 2</a> and <a id="xref-fig-6-3" class="xref-fig" href="#F6">6</a>). The benefits of imputation were highest at the lowest imputation SNP panel densities. For example, genomic prediction accuracy using 200 SNPs without imputation for sea lice resistance was 0.45 (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-6-4" class="xref-fig" href="#F6">6</a>), whereas with imputation to 5,000 SNPs it was 0.53, which corresponds to an increase of 17.5% (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6</a>). In contrast, for a density of 700 SNPs, the benefit of imputation <em>vs.</em> no imputation was approximately 6.83%, and at 1,000 SNPs the benefit was negligible. In summary, the use of low density imputation SNP panels (<em>e.g.</em>, 200 SNPs) with imputation to medium density (5,000 SNPs) resulted in prediction accuracies comparable to using the optimal SNP panel (5,000 SNPs) on all animals, and significantly outperformed low density panels alone without imputation.</p></div><div id="sec-17" class="subsection"><h3>Cost-benefit analysis</h3><p id="p-39">In scenario (i) where all animals (parents and offspring) were genotyped with a medium density SNP panel (5000 SNPs), the total cost of genotyping was estimated at $154,500. However in scenario (ii) where only the parents were genotyped for medium density and the offspring were genotyped for 200 SNPs, the cost was estimated to reduce to $59,000. This corresponds to a 62% cost reduction between scenario (i) and (ii), with virtually no loss of genomic prediction accuracy for the traits measured.</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-40">This study explores developing optimal methods for cost-effective genomic selection in aquaculture, with a focus on an Atlantic salmon sib-testing breeding program. In this study, genetic relationships were incorporated through the genomic relationship matrix which was computed from the SNP genotype data and captures identity by state (IBS). While family-based selection using phenotypes and pedigrees can only capture between-family variation, genomic selection using genetic markers can also capture within-family variation. This allows to distinguish between the genetic merit of full-siblings by predicting and utilizing the Mendelian segregation term. Previous studies have shown that using genomic relationships improves the predictive ability of the BVs compared to pedigree-based predictions (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Habier <em>et al.</em> 2007</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Zenger <em>et al.</em> 2019</a>). Genomic selection presents further advantages over pedigree-based selection as it allows to reduce emphasis on particular families and therefore facilitates improved control of the rate of inbreeding (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Daetwyler <em>et al.</em> 2008</a>).</p><div id="sec-19" class="subsection"><h3>Genomic prediction using low density SNP panels</h3><p id="p-41">The results presented herein confirmed findings from earlier studies of genomic selection in aquaculture species where relatively low density SNP panels were found to be as effective as full high density panels for the prediction of breeding values (<a id="xref-ref-25-6" class="xref-bibr" href="#ref-25">Tsai <em>et al.</em> 2016</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Vallejo <em>et al.</em> 2017</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Palaiokostas <em>et al.</em> 2018</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Robledo <em>et al.</em> 2018</a>; <a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">Yoshida <em>et al.</em> 2018</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Yoshida <em>et al.</em> 2019</a>). Aquaculture settings are typically characterized by full-sib testing schemes, and full-siblings share long genomic segments which can be adequately captured by fewer SNPs. Therefore, the existence of close relationships between training and validation animals is likely to be the reason why fewer SNPs are sufficient to capture relationships between individuals and provide near-maximal prediction accuracy. However, the current study has highlighted high variability in prediction accuracies with lower density SNP panels, reflecting the importance of the sampling effect in selecting the SNP panel. In contrast, for densities &gt; 5K, the risk of losing accuracy due to the chance inclusion of less informative SNP markers diminished. Non-random SNP selection, based on knowledge of the causal mutations affecting the trait of interest, is expected to enhance genomic prediction accuracy and assist particularly with cross-population prediction (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Edwards <em>et al.</em>, 2016</a>). Such knowledge will allow including the causative variant itself rather than relying on linkage disequilibrium between the QTL and the SNP marker, and prioritizing functional SNPs into genomic prediction.</p><p id="p-42">It should be noted that the high prediction accuracies using relatively modest SNP marker densities is likely to be restricted to sibling-testing schemes or similar. In situations where the relationship between the training and validation populations is more distant (such as use of one population as training set and another population as validation set), prediction accuracy is likely to be much lower, and higher SNP marker density may be more advantageous (<a id="xref-ref-25-7" class="xref-bibr" href="#ref-25">Tsai <em>et al.</em>, 2016</a>). This is because variation in linkage disequilibrium patterns between SNP markers and QTL across populations compromises across-population prediction accuracy (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Snelling <em>et al.</em> 2013</a>). Therefore, the results in the present study may be to some extent population and trait specific. Another factor that may affect the prediction accuracy is the minor allele frequency, however, rarer alleles may be important for capturing haplotypes for specific families. Removing SNPs with minor allele frequency less than 0.3 was not found to reduce the variability of prediction accuracy values obtained across the SNP panel replicates (Supplementary Information 6).</p><p id="p-43">Selecting SNPs randomly across the genome or randomly within each chromosome performed similarly in terms of genomic prediction accuracy. The method of random sampling within each chromosome was preferred for the selection of the optimal SNP panel, as this method ensures a more even distribution of SNPs across the genome, and ensures that SNPs from all chromosomes are included in the calculation of the genomic relationship matrices. In the present study, SNPs were selected at random within each chromosome, with the number of SNPs corresponding to the physical length of the chromosome. SNP panels were also constructed by selecting SNPs based on even physical distance within each chromosome (Supplementary Information 3). Selecting SNPs based on genetic distance was not tested, and may be expected to impact on prediction accuracy due to major variation in recombination rate across the genome of salmon, particularly in males. However, given that reducing SNP density from the highest density to the optimal density panel did not reduce prediction accuracy, it is unlikely that genetic distance or regions of high recombination have a major impact.</p></div><div id="sec-20" class="subsection"><h3>Heritability estimation and prediction accuracy</h3><p id="p-44">Heritabilities estimated using lower density SNP panels (where the reduced SNP panels were used to calculate the <strong>G</strong> matrix) were not found to be reliable, with large standard deviations that often were larger than the estimates. Hence, while the correlation between predicted BV and phenotype reduced for lower densities as expected, the accuracies calculated using the heritability estimates from lower density SNP panels appeared to remain high (preliminary analysis; results not shown), which was an artifact introduced by the spurious heritability estimates. This is likely to be common in studies of genomic prediction accuracies with reduced marker densities if the reduced panel is used to calculate the h<sup>2</sup> estimate that underlies the transformation from correlation between phenotypes and EBVs to the accuracy value <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/581/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/581/embed/mml-math-5.gif"/></noscript></span></span>. Therefore, for the calculation of the accuracies presented here, the heritability calculated from the HD SNP panel was used, which is expected to be closer to the true heritability in the population as a greater proportion of the genetic variance is captured by the HD SNP markers. The heritability estimated using the HD SNP panel was consistent with the heritability estimated using pedigree data for the same population (pedigree-based heritability of 0.23 (SE = 0.08) for sea lice resistance).</p><p id="p-45">Genomic prediction accuracy is affected by the genetic distance between training and test populations (<em>e.g.</em>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Scutari <em>et al.</em> 2016</a>), and the rapid drop in accuracy when the training and test sets do not include close relatives has been observed in both salmon (<a id="xref-ref-25-8" class="xref-bibr" href="#ref-25">Tsai <em>et al.</em> 2016</a>) and common carp (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Palaiokostas <em>et al.</em> 2019</a>). The variation observed in the accuracy across the cross-validation repeats is due to the random sampling of individuals to construct the training and validation sets; the genetic distance between the training and validation sets varies by chance and affects the prediction accuracy. Hence in the present study average prediction accuracies are reported over 50 replicates of cross-validation, where in each replicate, individuals are re-allocated in training and test sets. Therefore, where possible in aquaculture breeding programs, close relatives (such as full-siblings) to the selection candidates should be included in reference populations to achieve high accuracy levels for genomic prediction.</p></div><div id="sec-21" class="subsection"><h3>Imputation and cost-benefit analysis</h3><p id="p-46">Results from using imputed genotypes for genomic prediction were in agreement with previous studies (<a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Tsai <em>et al.</em> 2017</a>; <a id="xref-ref-32-6" class="xref-bibr" href="#ref-32">Yoshida <em>et al.</em> 2018</a>), where the authors observed that genomic prediction accuracies using imputed genotypes were very close to those of true genotypes. For very low imputation SNP panel densities (<em>e.g.</em>, 200 SNPs), although the imputation accuracy was lower, the genomic prediction accuracy using the imputed genotypes was not greatly reduced. The high imputation accuracies observed in this study can be explained by the inclusion of close relatives in the reference group. A trade-off was observed between genotyping cost and loss of prediction accuracy, but using low density imputation panels with imputation to medium density appears to be a highly accurate cost-effective option. A reduction in prediction accuracy means a reduction in genetic gain, and this has a cost implication. However, this cost is difficult to quantify, as it will be specific to the trait, the breeding goal, and the company. Furthermore, it is unclear where this cost would be incurred, for example to the breeding company or to the producer. In addition, it should be noted that this is a small-scale trial and the imputation and genomic prediction performance is likely to be more consistent in larger commercial operations, and therefore needs to be tested for different populations and traits.</p></div></div><div class="section conclusions" id="sec-22"><h2 class="">Conclusion</h2><p id="p-47">High genotype imputation accuracy can be achieved in the context of a salmon breeding program by genotyping parents at medium density and offspring at low density. When offspring are genotyped at very low density (<em>e.g.</em>, 200 SNPs), and the parents are genotyped at medium density (<em>e.g.</em>, 5000 SNPs), this is sufficient to obtain near-maximal genomic prediction accuracy. Since genotyping for very low density panels is considerably cheaper than genotyping for medium or high density panels, this genotype imputation approach may be a cost effective tool for genomic prediction in Atlantic salmon.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-48">The authors would like to thank Dr Ricardo Pong-Wong for stimulating discussions on cross-validation techniques. The authors gratefully acknowledge funding from the Scottish Aquaculture Innovation Centre (Grant number SL_2017_09), and BBSRC Institute Strategic Program Grants to the Roslin Institute (BB/P013732/1, BB/P013740/1, BB/P013759/1).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-49">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.9929945">https://doi.org/10.25387/g3.9929945</a>.</p></li><li class="fn" id="fn-2"><p id="p-50"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-10-03"><span class="received-label">Received </span>October 3, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-12-03"><span class="accepted-label">Accepted </span>December 3, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Tsairidou <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.1" data-doi="10.1371/journal.pone.0001274"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amin</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Van Duijn</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. S.</span> <span class="cit-name-surname">Aulchenko</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A genomic background based method for association analysis in related individuals.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">2</span>: <span class="cit-elocation-id">e1274</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0001274</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DAmin%26rft.auinit1%253DN.%26rft.volume%253D2%26rft.issue%253D12%26rft.spage%253De1274%26rft.epage%253De1274%26rft.atitle%253DA%2Bgenomic%2Bbackground%2Bbased%2Bmethod%2Bfor%2Bassociation%2Banalysis%2Bin%2Brelated%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001274%26rft_id%253Dinfo%253Apmid%252F18060068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0001274&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18060068&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F581.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.2" data-doi="10.1093/bioinformatics/btm108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aulchenko</span>,  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Ripke</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Isaacs</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Van Duijn</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">GenABEL: an R library for genome-wide association analysis.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1294</span>–<span class="cit-lpage">1296</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btm108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm108%26rft_id%253Dinfo%253Apmid%252F17384015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm108&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17384015&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F581.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247348300017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.581.3" data-doi="10.1007/978-94-007-2010-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bruno</span>,  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">Noguera</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. T.</span> <span class="cit-name-surname">Poppe</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source">A Colour Atlas of Salmonid Diseases Springer Netherlands</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin, Germany</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-94-007-2010-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DA%2BColour%2BAtlas%2Bof%2BSalmonid%2BDiseases%2BSpringer%2BNetherlands%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-94-007-2010-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-94-007-2010-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.4" data-doi="10.1101/624031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cáceres</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Barría</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Christensen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. N.</span> <span class="cit-name-surname">Bassini</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Correa</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Genome-scale comparative analysis for host resistance against sea lice between Atlantic salmon and rainbow trout.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/624031</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F624031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI2MjQwMzF2MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvNTgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.5" data-doi="10.1111/j.1365-2761.2008.01011.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Costello</span>,  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The global economic cost of sea lice to the salmonid farming industry.</span> <abbr class="cit-jnl-abbrev">J. Fish Dis.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">118</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2761.2008.01011.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bfish%2Bdiseases%26rft.stitle%253DJ%2BFish%2BDis%26rft.aulast%253DCostello%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D32%26rft.issue%253D1%26rft.spage%253D115%26rft.epage%253D118%26rft.atitle%253DThe%2Bglobal%2Beconomic%2Bcost%2Bof%2Bsea%2Blice%2Bto%2Bthe%2Bsalmonid%2Bfarming%2Bindustry.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2761.2008.01011.x%26rft_id%253Dinfo%253Apmid%252F19245636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2761.2008.01011.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19245636&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F581.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263521000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.6" data-doi="10.1371/journal.pone.0003395"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>,  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Villanueva</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Woolliams</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Accuracy of Predicting the Genetic Risk of Disease Using a Genome-Wide Approach.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-elocation-id">e3395</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0003395</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DDaetwyler%26rft.auinit1%253DH.%2BD.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3395%26rft.epage%253De3395%26rft.atitle%253DAccuracy%2Bof%2Bpredicting%2Bthe%2Bgenetic%2Brisk%2Bof%2Bdisease%2Busing%2Ba%2Bgenome-wide%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003395%26rft_id%253Dinfo%253Apmid%252F18852893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003395&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852893&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F581.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.7" data-doi="10.1016/j.aquaculture.2018.12.061"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dufflocq</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Pérez-Enciso</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Lhorente</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Yáñez</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Accuracy of genomic predictions using different imputation error rates in aquaculture breeding programs: A simulation study.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">503</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">230</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.aquaculture.2018.12.061</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D503%26rft.spage%253D225%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2018.12.061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2018.12.061&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.8" data-doi="10.1534/genetics.116.187161"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>,  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. F.</span> <span class="cit-name-surname">Sørensen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Sarup</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. F. C.</span> <span class="cit-name-surname">Mackay</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Sørensen</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic Prediction for Quantitative Traits Is Improved by Mapping Variants to Gene Ontology Categories in Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">203</span>: <span class="cit-fpage">1871</span>–<span class="cit-lpage">1883</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.187161</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.187161%26rft_id%253Dinfo%253Apmid%252F27235308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMy80LzE4NzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzU4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-web" id="cit-10.2.581.9"><div class="cit-metadata unstructured">Gilmour, A., B. Gogel, B. Cullis, and Thompson, 2009 R. ASReml User Guide Release 3.0. VSN International Ltd. Hemel Hempstead, HP1 1ES, UK.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.10" data-doi="10.1016/j.aquaculture.2011.01.026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gjerde</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ødegård</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Thorland</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Estimates of genetic variation in the susceptibility of Atlantic salmon (Salmo salar) to the salmon louse Lepeophtheirus salmonis.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">314</span>: <span class="cit-fpage">66</span>–<span class="cit-lpage">72</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.aquaculture.2011.01.026</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D314%26rft.spage%253D66%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2011.01.026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2011.01.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.11" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Dekkers</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.081190</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzU4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.12" data-doi="10.1111/j.1365-2052.2011.02208.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Bowman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. D.</span> <span class="cit-name-surname">Daetwyler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Kijas</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H. J.</span> <span class="cit-name-surname">Van Der Werf</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Accuracy of genotype imputation in sheep breeds.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">72</span>–<span class="cit-lpage">80</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2052.2011.02208.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D43%26rft.issue%253D1%26rft.spage%253D72%26rft.epage%253D80%26rft.atitle%253DAccuracy%2Bof%2Bgenotype%2Bimputation%2Bin%2Bsheep%2Bbreeds.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2052.2011.02208.x%26rft_id%253Dinfo%253Apmid%252F22221027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2052.2011.02208.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22221027&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F581.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.13" data-doi="10.1186/1471-2164-15-90"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Houston</span>,  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Taggart</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Cezard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Bekaert</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. R.</span> <span class="cit-name-surname">Lowe</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Development and validation of a high density SNP genotyping array for Atlantic salmon (Salmo salar).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">90</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-90</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D90%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-90%26rft_id%253Dinfo%253Apmid%252F24524230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-90&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24524230&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F581.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.14" data-doi="10.1146/annurev.genom.9.081307.164242"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Willer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sanna</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Abecasis</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genotype imputation.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">406</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.genom.9.081307.164242</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DY.%26rft.volume%253D10%26rft.spage%253D387%26rft.epage%253D406%26rft.atitle%253DGenotype%2Bimputation.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genom.9.081307.164242%26rft_id%253Dinfo%253Apmid%252F19715440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genom.9.081307.164242&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19715440&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F581.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270313200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.15" data-doi="10.1038/nature17164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lien</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. F.</span> <span class="cit-name-surname">Koop</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Sandve</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Miller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Kent</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Atlantic salmon genome provides insights into rediploidization.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">533</span>: <span class="cit-fpage">200</span>–<span class="cit-lpage">205</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature17164</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature17164%26rft_id%253Dinfo%253Apmid%252F27088604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature17164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27088604&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F581.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.16" data-doi="10.3389/fgene.2018.00082"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palaiokostas</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kocour</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Prchal</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. D.</span> <span class="cit-name-surname">Houston</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Accuracy of Genomic Evaluations of Juvenile Growth Rate in Common Carp (Cyprinus carpio) Using Genotyping by Sequencing.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">82</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2018.00082</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D9%26rft.spage%253D82%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2018.00082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2018.00082&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.17" data-doi="10.3389/fgene.2019.00543"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palaiokostas</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Vesely</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kocour</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Prchal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Pokorova</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Optimizing Genomic Prediction of Host Resistance to Koi Herpesvirus Disease in Carp.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">543</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2019.00543</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D10%26rft.spage%253D543%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2019.00543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2019.00543&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-web" id="cit-10.2.581.18" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Neale</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Todd-Brown</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Thomas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Ferreira</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> PLINK: a tool set for whole-genome association and population-based linkage analyses. Available: <a href="http://pngu.mgh.harvard.edu/purcell/plink/">http://pngu.mgh.harvard.edu/purcell/plink/</a>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/519795</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F581.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.581.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Development Core Team</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-source"><em>R: A language and environment for statistical computing</em>.</span> <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.20" data-doi="10.1534/g3.118.200075"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robledo</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Matika</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Hamilton</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. D.</span> <span class="cit-name-surname">Houston</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genome-Wide Association and Genomic Selection for Resistance to Amoebic Gill Disease in Atlantic Salmon.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1195</span>–<span class="cit-lpage">1203</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.118.200075</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.118.200075%26rft_id%253Dinfo%253Apmid%252F29420190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjgvNC8xMTk1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi81ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.21" data-doi="10.1186/1471-2164-15-478"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sargolzaei</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Chesnais</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. S.</span> <span class="cit-name-surname">Schenkel</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A new approach for efficient genotype imputation using information from relatives.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">478</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-478</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D478%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-478&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.22" data-doi="10.1371/journal.pgen.1006288"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scutari</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Mackay</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Balding</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Using Genetic Distance to Infer the Accuracy of Genomic Prediction.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-elocation-id">e1006288</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1006288</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1006288%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006288&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.23"><div class="cit-metadata unstructured">Sevatdal, S. and T. Horsberg, 2003 Determination of reduced sensitivity in sea lice (Lepeophtheirus salmonis KrØyer) against the pyrethroid deltamethrin using bioassays and probit modelling.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.24" data-doi="10.2527/jas.2012-5784"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Snelling</span>,  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Cushman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Keele</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Maltecca</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. G.</span> <span class="cit-name-surname">Thomas</span></span></li></ol><cite>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Breeding and Genetics Symposium: networks and pathways to guide genomic selection.</span> <abbr class="cit-jnl-abbrev">J Anim Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">537</span>–<span class="cit-lpage">552</span>. <a href="https://www.ncbi.nlm.nih.gov/pubmed/23097404">https://www.ncbi.nlm.nih.gov/pubmed/23097404</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BAnim%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2012-5784%26rft_id%253Dinfo%253Apmid%252F23097404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2012-5784&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23097404&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F581.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.25" data-doi="10.1186/s12711-016-0226-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>,  <span class="cit-name-given-names">H.-Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Hamilton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Tinch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Guy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Bron</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction of host resistance to sea lice in farmed Atlantic salmon populations.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">47</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-016-0226-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D48%26rft.spage%253D47%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-016-0226-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-016-0226-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.26" data-doi="10.1186/s12864-015-2117-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>,  <span class="cit-name-given-names">H.-Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Hamilton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Tinch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Guy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Gharbi</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome wide association and genomic prediction for growth traits in juvenile farmed Atlantic salmon using a high density SNP array.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">969</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-2117-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D969%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2117-9%26rft_id%253Dinfo%253Apmid%252F26582102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2117-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26582102&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F581.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>,  <span class="cit-name-given-names">H.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O.</span>  <span class="cit-name-given-names">Matika</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">S. M.</span>  <span class="cit-name-given-names">Edwards</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">R.</span>  <span class="cit-name-given-names">Antolín–Sánchez</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">A.</span>  <span class="cit-name-given-names">Hamilton</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> Genotype Imputation To Improve the Cost-Efficiency of Genomic Selection in Farmed Atlantic Salmon. G3: (Bethesda) 7: 1377–1383.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-web" id="cit-10.2.581.28"><div class="cit-metadata unstructured">Tsairidou, S., 2019 CVrepGPAcalc. [Online], Available: <a href="https://github.com/SmaragdaT/CVrep/tree/master/CVrepGPAcalc">https://github.com/SmaragdaT/CVrep/tree/master/CVrepGPAcalc</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.29" data-doi="10.1371/journal.pone.0096728"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsairidou</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Woolliams</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Allen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Skuce</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. H.</span> <span class="cit-name-surname">Mcbride</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic prediction for tuberculosis resistance in dairy cattle.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e96728</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0096728</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De96728%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0096728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0096728&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.30" data-doi="10.1046/j.1365-2109.2000.00518.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tully</span>,  <span class="cit-name-given-names">O.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Mcfadden</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Variation in sensitivity of sea lice [Lepeophtheirus salmonis (Krøyer)] to dichlorvos on Irish salmon farms in 1991–92.</span> <abbr class="cit-jnl-abbrev">Aquacult. Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">849</span>–<span class="cit-lpage">854</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1365-2109.2000.00518.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquacult.%2BRes.%26rft.volume%253D31%26rft.spage%253D849%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2109.2000.00518.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2109.2000.00518.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000165393000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.31" data-doi="10.1186/s12711-017-0293-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vallejo</span>,  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. D.</span> <span class="cit-name-surname">Leeds</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Gao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Parsons</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. E.</span> <span class="cit-name-surname">Martin</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic selection models double the accuracy of predicted breeding values for bacterial cold water disease resistance compared to a traditional pedigree-based model in rainbow trout aquaculture.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">17</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-017-0293-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D49%26rft.spage%253D17%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-017-0293-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-017-0293-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.32" data-doi="10.1016/j.aquaculture.2018.03.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yoshida</span>,  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Carvalheiro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Lhorente</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Correa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Figueroa</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Accuracy of genotype imputation and genomic predictions in a two-generation farmed Atlantic salmon population using high-density and low-density SNP panels.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">491</span>: <span class="cit-fpage">147</span>–<span class="cit-lpage">154</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.aquaculture.2018.03.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D491%26rft.spage%253D147%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2018.03.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2018.03.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.33" data-doi="10.1534/g3.119.400116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yoshida</span>,  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Lhorente</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Correa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Soto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Salas</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Genome-Wide Association Study and Cost-Efficient Genomic Predictions for Growth and Fillet Yield in Nile Tilapia (Oreochromis niloticus).</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">2597</span>–<span class="cit-lpage">2607</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.119.400116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.119.400116%26rft_id%253Dinfo%253Apmid%252F31171566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjkvOC8yNTk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi81ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.581.34" data-doi="10.3389/fgene.2018.00693"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zenger</span>,  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Khatkar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. B.</span> <span class="cit-name-surname">Jones</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Khalilisamani</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Jerry</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Genomic Selection in Aquaculture: Application, Limitations and Opportunities With Special Reference to Marine Shrimp and Pearl Oysters.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">693</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2018.00693</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D9%26rft.spage%253D693%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2018.00693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2018.00693&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/2/581.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/2/569" title="Simulation-Based Evaluation of Three Methods for Local Ancestry Deconvolution of Non-model Crop Species Genomes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/2/591" title="Evaluation of Saccharomyces cerevisiae Wine Yeast Competitive Fitness in Enologically Relevant Environments by Barcode Sequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/2"><span>Volume 10 Issue 2, February 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/2.cover-source.jpg?itok=LSawHOSr" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/581" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vWEmO3MkTR8oeZvuNV8RxyCmgC4AYL-zfSHvniDZYUU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/2/581" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/2/581" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/2/581" class="active" data-icon-position="" data-hide-link-title="0">Optimizing Low-Cost Genotyping and Imputation Strategies for Genomic Selection in Atlantic Salmon</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495563" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-L2IQD_AcHBy0-5Xw-O_ijWu7MTV52zhmsWggecUa0-8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/2/581.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/2/581" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-OcIYM4r9brVBYKSMEzaQNYXwPhs03wtz0vvlA3hXcVA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/2/581" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/2/581.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495563" id="node495563--2" data-pisa="ggg;10/2/581" data-pisa-master="ggg;g3.119.400800" data-apath="/ggg/10/2/581.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Optimizing Low-Cost Genotyping and Imputation Strategies for Genomic Selection in Atlantic Salmon</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0792-9421" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Smaragda Tsairidou</span>, <span class="highwire-citation-author" data-delta="1">Alastair Hamilton</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-9616-5912" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Diego Robledo</span>, <span class="highwire-citation-author" data-delta="3">James E. Bron</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-1805-0762" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ross D. Houston</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">581-590;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400800 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495563--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Smaragda Tsairidou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Edinburgh, EH25 9RG, United Kingdom,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Smaragda%2BTsairidou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tsairidou%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASmaragda%2BTsairidou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0792-9421" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Smaragda Tsairidou</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Smaragda.Tsairidou@roslin.ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">Smaragda.Tsairidou@roslin.ed.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alastair Hamilton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Hendrix Genetics Aquaculture BV/ Netherlands Villa ’de Körver’, Spoorstraat 695831 CK Boxmeer, The Netherlands, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alastair%2BHamilton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hamilton%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlastair%2BHamilton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Diego Robledo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Edinburgh, EH25 9RG, United Kingdom,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BRobledo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Robledo%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADiego%2BRobledo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9616-5912" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Diego Robledo</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">James E. Bron </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Aquaculture, University of Stirling, FK9 4LA, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BE.%2BBron%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bron%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BE.%2BBron%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ross D. Houston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Edinburgh, EH25 9RG, United Kingdom,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ross%2BD.%2BHouston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Houston%20RD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoss%2BD.%2BHouston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1805-0762" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ross D. Houston</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/2/581" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/2/581&amp;title=Optimizing%20Low-Cost%20Genotyping%20and%20Imputation%20Strategies%20for%20Genomic%20Selection%20in%20Atlantic%20Salmon" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/2/581&amp;title=Optimizing%20Low-Cost%20Genotyping%20and%20Imputation%20Strategies%20for%20Genomic%20Selection%20in%20Atlantic%20Salmon" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/2/581&amp;title=Optimizing%20Low-Cost%20Genotyping%20and%20Imputation%20Strategies%20for%20Genomic%20Selection%20in%20Atlantic%20Salmon" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/2/581&amp;text=Optimizing%20Low-Cost%20Genotyping%20and%20Imputation%20Strategies%20for%20Genomic%20Selection%20in%20Atlantic%20Salmon" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/2/581&amp;title=Optimizing%20Low-Cost%20Genotyping%20and%20Imputation%20Strategies%20for%20Genomic%20Selection%20in%20Atlantic%20Salmon" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/2/581&amp;t=Optimizing%20Low-Cost%20Genotyping%20and%20Imputation%20Strategies%20for%20Genomic%20Selection%20in%20Atlantic%20Salmon" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/2/581&amp;title=Optimizing%20Low-Cost%20Genotyping%20and%20Imputation%20Strategies%20for%20Genomic%20Selection%20in%20Atlantic%20Salmon" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/2/581&amp;title=Optimizing%20Low-Cost%20Genotyping%20and%20Imputation%20Strategies%20for%20Genomic%20Selection%20in%20Atlantic%20Salmon" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495563/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495563" id="node495563--3" data-pisa="ggg;10/2/581" data-pisa-master="ggg;g3.119.400800" data-apath="/ggg/10/2/581.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Optimizing Low-Cost Genotyping and Imputation Strategies for Genomic Selection in Atlantic Salmon</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0792-9421" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Smaragda Tsairidou</span>, <span class="highwire-citation-author" data-delta="1">Alastair Hamilton</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-9616-5912" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Diego Robledo</span>, <span class="highwire-citation-author" data-delta="3">James E. Bron</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-1805-0762" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ross D. Houston</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">581-590;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400800 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495563--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Smaragda Tsairidou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Edinburgh, EH25 9RG, United Kingdom,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Smaragda%2BTsairidou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tsairidou%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASmaragda%2BTsairidou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0792-9421" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Smaragda Tsairidou</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Smaragda.Tsairidou@roslin.ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">Smaragda.Tsairidou@roslin.ed.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alastair Hamilton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Hendrix Genetics Aquaculture BV/ Netherlands Villa ’de Körver’, Spoorstraat 695831 CK Boxmeer, The Netherlands, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alastair%2BHamilton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hamilton%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlastair%2BHamilton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Diego Robledo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Edinburgh, EH25 9RG, United Kingdom,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BRobledo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Robledo%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADiego%2BRobledo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9616-5912" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Diego Robledo</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">James E. Bron </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Aquaculture, University of Stirling, FK9 4LA, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BE.%2BBron%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bron%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BE.%2BBron%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ross D. Houston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Edinburgh, EH25 9RG, United Kingdom,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ross%2BD.%2BHouston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Houston%20RD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoss%2BD.%2BHouston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1805-0762" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ross D. Houston</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=581&amp;rft.epage=590&amp;rft.atitle=Optimizing%20Low-Cost%20Genotyping%20and%20Imputation%20Strategies%20for%20Genomic%20Selection%20in%20Atlantic%20Salmon&amp;rft.volume=10&amp;rft.issue=2&amp;rft.date=2020-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Tsairidou%2C+Smaragda&amp;rft.au=Hamilton%2C+Alastair&amp;rft.au=Robledo%2C+Diego&amp;rft.au=Bron%2C+James+E.&amp;rft.au=Houston%2C+Ross+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495563/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495563/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495563/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495563/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495563/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495563/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495563/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495563/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495563/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495563/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495563/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495563/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/2/581" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-22" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/2/581.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/2/581.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/2/581.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
