<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/2/455" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/2/455.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/2/455.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/2/455.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/2/455.atom" />
<meta name="HW.pisa" content="ggg;10/2/455" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="De Novo Genome Assembly of Populus simonii Further Supports That Populus simonii and Populus trichocarpa Belong to Different Sections" />
<meta name="DC.Identifier" content="10.1534/g3.119.400913" />
<meta name="DC.Date" content="2020-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Wu et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Populus simonii is an important tree in the genus Populus , widely distributed in the Northern Hemisphere and having a long cultivation history. Although this species has ecologically and economically important values, its genome sequence is currently not available, hindering the development of new varieties with wider adaptive and commercial traits. Here, we report a chromosome-level genome assembly of P. simonii using PacBio long-read sequencing data aided by Illumina paired-end reads and related genetic linkage maps. The assembly is 441.38 Mb in length and contain 686 contigs with a contig N50 of 1.94 Mb. With the linkage maps, 336 contigs were successfully anchored into 19 pseudochromosomes, accounting for 90.2% of the assembled genome size. Genomic integrity assessment showed that 1,347 (97.9%) of the 1,375 genes conserved among all embryophytes can be found in the P. simonii assembly. Genomic repeat analysis revealed that 41.47% of the P. simonii genome is composed of repetitive elements, of which 40.17% contained interspersed repeats. A total of 45,459 genes were predicted from the P. simonii genome sequence and 39,833 (87.6%) of the genes were annotated with one or more related functions. Phylogenetic analysis indicated that P. simonii and Populus trichocarpa should be placed in different sections, contrary to the previous classification according to morphology. The genome assembly not only provides an important genetic resource for the comparative and functional genomics of different Populus species, but also furnishes one of the closest reference sequences for identifying genomic variants in an F1 hybrid population derived by crossing P. simonii with other Populus species." />
<meta name="DC.Contributor" content="Hainan Wu" />
<meta name="DC.Contributor" content="Dan Yao" />
<meta name="DC.Contributor" content="Yuhua Chen" />
<meta name="DC.Contributor" content="Wenguo Yang" />
<meta name="DC.Contributor" content="Wei Zhao" />
<meta name="DC.Contributor" content="Hua Gao" />
<meta name="DC.Contributor" content="Chunfa Tong" />
<meta name="article:published_time" content="2020-02-01" />
<meta name="article:section" content="Genome Report" />
<meta name="citation_title" content="De Novo Genome Assembly of Populus simonii Further Supports That Populus simonii and Populus trichocarpa Belong to Different Sections" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;&lt;i&gt;Populus simonii&lt;/i&gt; is an important tree in the genus &lt;i&gt;Populus&lt;/i&gt;, widely distributed in the Northern Hemisphere and having a long cultivation history. Although this species has ecologically and economically important values, its genome sequence is currently not available, hindering the development of new varieties with wider adaptive and commercial traits. Here, we report a chromosome-level genome assembly of &lt;i&gt;P. simonii&lt;/i&gt; using PacBio long-read sequencing data aided by Illumina paired-end reads and related genetic linkage maps. The assembly is 441.38 Mb in length and contain 686 contigs with a contig N50 of 1.94 Mb. With the linkage maps, 336 contigs were successfully anchored into 19 pseudochromosomes, accounting for 90.2% of the assembled genome size. Genomic integrity assessment showed that 1,347 (97.9%) of the 1,375 genes conserved among all embryophytes can be found in the &lt;i&gt;P. simonii&lt;/i&gt; assembly. Genomic repeat analysis revealed that 41.47% of the &lt;i&gt;P. simonii&lt;/i&gt; genome is composed of repetitive elements, of which 40.17% contained interspersed repeats. A total of 45,459 genes were predicted from the &lt;i&gt;P. simonii&lt;/i&gt; genome sequence and 39,833 (87.6%) of the genes were annotated with one or more related functions. Phylogenetic analysis indicated that &lt;i&gt;P. simonii&lt;/i&gt; and &lt;i&gt;Populus trichocarpa&lt;/i&gt; should be placed in different sections, contrary to the previous classification according to morphology. The genome assembly not only provides an important genetic resource for the comparative and functional genomics of different &lt;i&gt;Populus&lt;/i&gt; species, but also furnishes one of the closest reference sequences for identifying genomic variants in an F&lt;sub&gt;1&lt;/sub&gt; hybrid population derived by crossing &lt;i&gt;P. simonii&lt;/i&gt; with other &lt;i&gt;Populus&lt;/i&gt; species.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/02/01" />
<meta name="citation_mjid" content="ggg;10/2/455" />
<meta name="citation_id" content="10/2/455" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/2/455" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/2/455.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/2/455.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/2/455.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400913" />
<meta name="citation_pmid" content="31806765" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genome Report" />
<meta name="citation_firstpage" content="455" />
<meta name="citation_lastpage" content="466" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Hainan Wu" />
<meta name="citation_author_institution" content="Key Laboratory of Forest Genetics &amp; Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2890-9702" />
<meta name="citation_author" content="Dan Yao" />
<meta name="citation_author_institution" content="Key Laboratory of Forest Genetics &amp; Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5937-3467" />
<meta name="citation_author" content="Yuhua Chen" />
<meta name="citation_author_institution" content="Key Laboratory of Forest Genetics &amp; Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4530-8653" />
<meta name="citation_author" content="Wenguo Yang" />
<meta name="citation_author_institution" content="Key Laboratory of Forest Genetics &amp; Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5191-828X" />
<meta name="citation_author" content="Wei Zhao" />
<meta name="citation_author_institution" content="Key Laboratory of Forest Genetics &amp; Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7130-7670" />
<meta name="citation_author" content="Hua Gao" />
<meta name="citation_author_institution" content="Key Laboratory of Forest Genetics &amp; Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China" />
<meta name="citation_author" content="Chunfa Tong" />
<meta name="citation_author_institution" content="Key Laboratory of Forest Genetics &amp; Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China" />
<meta name="citation_author_email" content="tongchf@njfu.edu.cn" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9795-211X" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Conesa;citation_author=S. Götz;citation_author=J. M. García-Gómez;citation_author=J. Terol;citation_author=M. Talón;citation_title=Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research.;citation_volume=21;citation_year=2005;citation_pmid=16081474;citation_doi=10.1093/bioinformatics/bti610" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Bairoch;citation_author=B. Boeckmann;citation_title=The SWISS-PROT protein sequence data bank.;citation_volume=19;citation_year=1991;citation_pmid=2041811;citation_doi=10.1093/nar/19.suppl.2247" />
<meta name="citation_reference" content="citation_journal_title=Mob. DNA;citation_author=W. Bao;citation_author=K. K. Kojima;citation_author=O. Kohany;citation_title=Repbase Update, a database of repetitive elements in eukaryotic genomes.;citation_pages=11-11;citation_volume=6;citation_year=2015;citation_pmid=26045719;citation_doi=10.1186/s13100-015-0041-9" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=H. D. Bradshaw;citation_author=M. Villar;citation_author=B. D. Watson;citation_author=K. G. Otto;citation_author=S. Stewart;citation_title=Molecular genetics of growth and development in Populus. III. A genetic linkage map of a hybrid poplar composed of RFLP, STS, and RAPD markers.;citation_pages=167-178;citation_volume=89;citation_year=1994;citation_doi=10.1007/BF00225137" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=B. Buchfink;citation_author=C. Xie;citation_author=D. H. Huson;citation_title=Fast and sensitive protein alignment using DIAMOND.;citation_pages=59-60;citation_volume=12;citation_year=2015;citation_pmid=25402007;citation_doi=10.1038/nmeth.3176" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. A. Campbell;citation_author=B. J. Haas;citation_author=J. P. Hamilton;citation_author=S. M. Mount;citation_author=C. R. Buell;citation_title=Comprehensive analysis of alternative splicing in rice and comparative analyses with Arabidopsis.;citation_pages=327-327;citation_volume=7;citation_year=2006;citation_pmid=17194304;citation_doi=10.1186/1471-2164-7-327" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Capella-Gutiérrez;citation_author=J. M. Silla-Martínez;citation_author=T. Gabaldón;citation_title=trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses.;citation_volume=25;citation_year=2009;citation_pmid=19505945;citation_doi=10.1093/bioinformatics/btp348" />
<meta name="citation_reference" content="citation_journal_title=Nat. Plants;citation_author=J. Chen;citation_author=Z. Hao;citation_author=X. Guang;citation_author=C. Zhao;citation_author=P. Wang;citation_title=Liriodendron genome sheds light on angiosperm phylogeny and species–pair differentiation.;citation_pages=18-25;citation_volume=5;citation_year=2019;citation_doi=10.1038/s41477-018-0323-6" />
<meta name="citation_reference" content="citation_journal_title=Plant Mol. Biol. Report.;citation_author=J. Chen;citation_author=Y. Song;citation_author=Z. He;citation_author=D. Zhang;citation_title=Genome-wide analysis of gene expression in response to drought stress in Populus simonii.;citation_pages=946-962;citation_volume=31;citation_year=2013;citation_doi=10.1007/s11105-013-0563-6" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=C. S. Chin;citation_author=D. H. Alexander;citation_author=P. Marks;citation_author=A. A. Klammer;citation_author=J. Drake;citation_title=Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data.;citation_pages=563-569;citation_volume=10;citation_year=2013;citation_pmid=23644548;citation_doi=10.1038/nmeth.2474" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=C. S. Chin;citation_author=P. Peluso;citation_author=F. J. Sedlazeck;citation_author=M. Nattestad;citation_author=G. T. Concepcion;citation_title=Phased diploid genome assembly with single-molecule real-time sequencing.;citation_pages=1050-1054;citation_volume=13;citation_year=2016;citation_pmid=27749838;citation_doi=10.1038/nmeth.4035" />
<meta name="citation_reference" content="citation_journal_title=Cell Res.;citation_author=X. Dai;citation_author=Q. Hu;citation_author=Q. Cai;citation_author=K. Feng;citation_author=N. Ye;citation_title=The willow genome and divergent evolution from poplar after the common genome duplication.;citation_pages=1274-1277;citation_volume=24;citation_year=2014;citation_pmid=24980958;citation_doi=10.1038/cr.2014.83" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. De Bie;citation_author=N. Cristianini;citation_author=J. P. Demuth;citation_author=M. W. Hahn;citation_title=CAFE: a computational tool for the study of gene family evolution.;citation_volume=22;citation_year=2006;citation_pmid=16543274;citation_doi=10.1093/bioinformatics/btl097" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=A. J. Drummond;citation_author=A. Rambaut;citation_title=BEAST: Bayesian evolutionary analysis by sampling trees.;citation_pages=214-214;citation_volume=7;citation_year=2007;citation_pmid=17996036;citation_doi=10.1186/1471-2148-7-214" />
<meta name="citation_reference" content="citation_title=Biology of Populus and its implications for management and conservation;citation_title=Systematics and evolution of Populus, pp;citation_pages=7-32;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=P. P. Edger;citation_author=T. J. Poorten;citation_author=R. VanBuren;citation_author=M. A. Hardigan;citation_author=M. Colle;citation_title=Origin and evolution of the octoploid strawberry genome.;citation_pages=541-547;citation_volume=51;citation_year=2019;citation_doi=10.1038/s41588-019-0356-4" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. M. Emms;citation_author=S. Kelly;citation_title=OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy.;citation_pages=157;citation_volume=16;citation_year=2015;citation_pmid=26243257;citation_doi=10.1186/s13059-015-0721-2" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. M. Emms;citation_author=S. Kelly;citation_title=STRIDE: Species tree root inference from gene duplication events.;citation_volume=34;citation_year=2017;citation_doi=10.1093/molbev/msx259" />
<meta name="citation_reference" content="citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=L. Guo;citation_author=T. Winzer;citation_author=X. Yang;citation_author=Y. Li;citation_author=Z. Ning;citation_title=The opium poppy genome and morphinan production.;citation_volume=362;citation_year=2018;citation_pmid=30166436;citation_doi=10.1126/science.aat4096" />
<meta name="citation_reference" content="citation_journal_title=Nat. Protoc.;citation_author=B. J. Haas;citation_author=P. Alexie;citation_author=Y. Moran;citation_author=G. Manfred;citation_author=P. D. Blood;citation_title=De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis.;citation_pages=1494-1512;citation_volume=8;citation_year=2013;citation_pmid=23845962;citation_doi=10.1038/nprot.2013.084" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=H. Heberle;citation_author=G. V. Meirelles;citation_author=F. R. da Silva;citation_author=G. P. Telles;citation_author=R. Minghim;citation_title=InteractiVenn: a web-based tool for the analysis of sets through Venn diagrams.;citation_pages=169-175;citation_volume=16;citation_year=2015;citation_pmid=25994840;citation_doi=10.1186/s12859-015-0611-3" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Holt;citation_title=MAKER2: an annotation pipeline and genome-database management tool for second-generation genome projects.;citation_pages=491-491;citation_volume=12;citation_year=2011;citation_pmid=22192575;citation_doi=10.1186/1471-2105-12-491" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Itkin;citation_author=R. Davidovich-Rikanati;citation_author=S. Cohen;citation_author=V. Portnoy;citation_author=A. Doron-Faigenboim;citation_title=The biosynthetic pathway of the nonsugar, high-intensity sweetener mogroside V from Siraitia grosvenorii.;citation_volume=113;citation_year=2016;citation_pmid=27821754;citation_doi=10.1073/pnas.1604828113" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Mistry;citation_author=R. D. Finn;citation_author=S. R. Eddy;citation_author=B. Alex;citation_author=P. Marco;citation_title=Challenges in homology search: HMMER3 and convergent evolution of coiled-coil regions.;citation_volume=41;citation_year=2013;citation_pmid=23598997;citation_doi=10.1093/nar/gkt263" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=H. Jung;citation_author=C. Winefield;citation_author=A. Bombarely;citation_author=P. Prentis;citation_author=P. Waterhouse;citation_title=Tools and strategies for long-read sequencing and de novo assembly of plant genomes.;citation_pages=700-724;citation_volume=24;citation_year=2019;citation_doi=10.1016/j.tplants.2019.05.003" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=I. Kalvari;citation_author=J. Argasinska;citation_author=N. Quinones-Olvera;citation_author=E. P. Nawrocki;citation_author=E. Rivas;citation_title=Rfam 13.0: shifting to a genome-centric resource for non-coding RNA families.;citation_pages=D335-D342;citation_volume=46;citation_year=2018;citation_pmid=29112718;citation_doi=10.1093/nar/gkx1038" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Katoh;citation_author=D. M. Standley;citation_title=MAFFT multiple sequence alignment software version 7: improvements in performance and usability.;citation_volume=30;citation_year=2013;citation_pmid=23329690;citation_doi=10.1093/molbev/mst010" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=I. Korf;citation_title=Gene finding in novel genomes.;citation_pages=59-59;citation_volume=5;citation_year=2004;citation_pmid=15144565;citation_doi=10.1186/1471-2105-5-59" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. Kurtz;citation_author=A. Phillippy;citation_author=A. L. Delcher;citation_author=M. Smoot;citation_author=M. Shumway;citation_title=Versatile and open software for comparing large genomes.;citation_pages=R12-R12;citation_volume=5;citation_year=2004;citation_issue=2;citation_pmid=14759262;citation_doi=10.1186/gb-2004-5-2-r12" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. Lagesen;citation_author=P. Hallin;citation_author=E. A. Rødland;citation_author=H.-H. Staerfeldt;citation_author=T. Rognes;citation_title=RNAmmer: consistent and rapid annotation of ribosomal RNA genes.;citation_volume=35;citation_year=2007;citation_pmid=17452365;citation_doi=10.1093/nar/gkm160" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=S. Lamichhaney;citation_author=G. Fan;citation_author=F. Widemo;citation_author=U. Gunnarsson;citation_author=D. S. Thalmann;citation_title=Structural genomic changes underlie alternative reproductive strategies in the ruff (Philomachus pugnax).;citation_volume=48;citation_year=2016;citation_doi=10.1038/ng.3430" />
<meta name="citation_reference" content="citation_journal_title=Genes;citation_author=C. Li;citation_author=F. Lin;citation_author=D. An;citation_author=W. Wang;citation_author=R. Huang;citation_title=Genome sequencing and assembly by long reads in plants.;citation_volume=9;citation_year=2017;citation_issue=6;citation_doi=10.3390/genes9010006" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows–Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment-Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y.-C. Lin;citation_author=J. Wang;citation_author=N. Delhomme;citation_author=B. Schiffthaler;citation_author=G. Sundström;citation_title=Functional and evolutionary genomic inferences in Populus through genome and population sequencing of American and European aspen.;citation_volume=115;citation_year=2018;citation_pmid=30373829;citation_doi=10.1073/pnas.1801437115" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=D. Liu;citation_author=J. Gong;citation_author=W. Dai;citation_author=X. Kang;citation_author=Z. Huang;citation_title=The genome of Ganoderma lucidum provides insights into triterpenes biosynthesis and wood degradation [corrected].;citation_pages=e36146-e36146;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22567134;citation_doi=10.1371/journal.pone.0036146" />
<meta name="citation_reference" content="citation_journal_title=Sci. China Life Sci.;citation_author=Y.-J. Liu;citation_author=X.-R. Wang;citation_author=Q.-Y. Zeng;citation_title=De novo assembly of white poplar genome and genetic diversity of white poplar population in Irtysh River basin in China.;citation_pages=609-618;citation_volume=62;citation_year=2019;citation_doi=10.1007/s11427-018-9455-2" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=Y. Liu;citation_author=J. Wang;citation_author=W. Ge;citation_author=Z. Wang;citation_author=Y. Li;citation_title=Two highly similar poplar paleo-subgenomes suggest an autotetraploid ancestor of Salicaceae plants.;citation_pages=571-581;citation_volume=8;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. M. Lowe;citation_author=S. R. Eddy;citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence.;citation_volume=25;citation_year=1997;citation_pmid=9023104;citation_doi=10.1093/nar/25.5.955" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=J. Ma;citation_author=D. Wan;citation_author=B. Duan;citation_author=X. Bai;citation_author=Q. Bai;citation_title=Genome sequence and genetic transformation of a widely distributed and cultivated poplar.;citation_volume=17;citation_year=2019;citation_doi=10.1111/pbi.12989" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=T. Ma;citation_author=J. Wang;citation_author=G. Zhou;citation_author=Z. Yue;citation_author=Q. Hu;citation_title=Genomic insights into salt adaptation in a desert poplar.;citation_pages=2797-2806;citation_volume=4;citation_year=2013;citation_pmid=24256998;citation_doi=10.1038/ncomms3797" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. Marçais;citation_author=C. Kingsford;citation_title=A fast, lock-free approach for efficient parallel counting of occurrences of k-mers.;citation_volume=27;citation_year=2011;citation_pmid=21217122;citation_doi=10.1093/bioinformatics/btr011" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Stanke;citation_author=O. Keller;citation_author=I. Gunduz;citation_author=A. Hayes;citation_author=S. Waack;citation_title=AUGUSTUS: ab initio prediction of alternative transcripts.;citation_pages=W435-W439;citation_volume=34;citation_year=2006;citation_issue=Web Server issue;citation_pmid=16845043;citation_doi=10.1093/nar/gkl200" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=T. P. Michael;citation_author=F. Jupe;citation_author=F. Bemm;citation_author=S. T. Motley;citation_author=J. P. Sandoval;citation_title=High contiguity Arabidopsis thaliana genome assembly with a single nanopore flow cell.;citation_pages=541-548;citation_volume=9;citation_year=2018;citation_doi=10.1038/s41467-018-03016-2" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=A. Minio;citation_author=M. Massonnet;citation_author=R. Figueroa-Balderas;citation_author=A. Castro;citation_author=D. Cantu;citation_title=Diploid genome assembly of the wine grape Carménère.;citation_volume=9;citation_year=2019;citation_pmid=30923135;citation_doi=10.1534/g3.119.400030" />
<meta name="citation_reference" content="citation_journal_title=BioData Min.;citation_author=J. G. Monroe;citation_author=Z. A. Allen;citation_author=P. Tanger;citation_author=J. L. Mullen;citation_author=J. T. Lovell;citation_title=TSPmap, a tool making use of traveling salesperson problem solvers in the efficient and accurate construction of high-density genetic linkage maps.;citation_pages=38;citation_volume=10;citation_year=2017;citation_doi=10.1186/s13040-017-0158-0" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=M. Mousavi;citation_author=C. Tong;citation_author=F. Liu;citation_author=S. Tao;citation_author=J. Wu;citation_title=De novo SNP discovery and genetic linkage mapping in poplar using restriction site associated DNA and whole-genome sequencing technologies.;citation_pages=656-667;citation_volume=17;citation_year=2016;citation_doi=10.1186/s12864-016-3003-9" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=B. Nystedt;citation_author=N. R. Street;citation_author=A. Wetterbom;citation_author=A. Zuccolo;citation_author=Y.-C. Lin;citation_title=The Norway spruce genome sequence and conifer genome evolution.;citation_pages=579-584;citation_volume=497;citation_year=2013;citation_pmid=23698360;citation_doi=10.1038/nature12211" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=H. Ogata;citation_author=S. Goto;citation_author=K. Sato;citation_author=W. Fujibuchi;citation_author=H. Bono;citation_title=KEGG: Kyoto Encyclopedia of Genes and Genomes.;citation_pages=29-34;citation_volume=27;citation_year=2000;citation_doi=10.1093/nar/27.1.29" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=I. Paolucci;citation_author=M. Gaudet;citation_author=V. Jorge;citation_author=I. Beritognolo;citation_author=S. Terzoli;citation_title=Genetic linkage maps of Populus alba L. and comparative mapping analysis of sex determination across Populus species.;citation_pages=863-875;citation_volume=6;citation_year=2010;citation_doi=10.1007/s11295-010-0297-7" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. K. Patel;citation_author=J. Mukesh;citation_title=NGS QC Toolkit: a toolkit for quality control of next generation sequencing data.;citation_pages=e30619-e30619;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22312429;citation_doi=10.1371/journal.pone.0030619" />
<meta name="citation_reference" content="citation_journal_title=Plant Mol. Biol. Report.;citation_author=S. Porebski;citation_author=L. G. Bailey;citation_author=B. R. Baum;citation_title=Modification of a CTAB DNA extraction protocol for plants containing high polysaccharide and polyphenol components.;citation_pages=8-15;citation_volume=15;citation_year=1997;citation_doi=10.1007/BF02772108" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=M. J. Sanderson;citation_title=r8s: inferring absolute rates of molecular evolution and divergence times in the absence of a molecular clock;citation_pages=301-302;citation_volume=19;citation_year=2003;citation_issue=2;citation_pmid=12538260;citation_doi=10.1093/bioinformatics/19.2.301" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=J. Shi;citation_author=X. Ma;citation_author=J. Zhang;citation_author=Y. Zhou;citation_author=M. Liu;citation_title=Chromosome conformation capture resolved near complete genome assembly of broomcorn millet.;citation_pages=464;citation_volume=10;citation_year=2019;citation_doi=10.1038/s41467-018-07876-6" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. A. Simão;citation_author=R. M. Waterhouse;citation_author=I. Panagiotis;citation_author=E. V. Kriventseva;citation_author=E. M. Zdobnov;citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.;citation_volume=31;citation_year=2015;citation_pmid=26059717;citation_doi=10.1093/bioinformatics/btv351" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=G. S. C. Slater;citation_author=E. Birney;citation_title=Automated generation of heuristics for biological sequence comparison.;citation_pages=31-31;citation_volume=6;citation_year=2005;citation_pmid=15713233;citation_doi=10.1186/1471-2105-6-31" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant;citation_author=C. Song;citation_author=Y. Liu;citation_author=A. Song;citation_author=G. Dong;citation_author=H. Zhao;citation_title=The Chrysanthemum nankingense genome provides insights into the evolution and diversification of chrysanthemum flowers and medicinal traits.;citation_pages=1482-1491;citation_volume=11;citation_year=2018;citation_doi=10.1016/j.molp.2018.10.003" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Rep.;citation_author=Y. Song;citation_author=Q. Chen;citation_author=D. Ci;citation_author=D. Zhang;citation_title=Transcriptome profiling reveals differential transcript abundance in response to chilling stress in Populus simonii.;citation_pages=1407-1425;citation_volume=32;citation_year=2013;citation_pmid=23652820;citation_doi=10.1007/s00299-013-1454-x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Stamatakis;citation_title=RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.;citation_volume=30;citation_year=2014;citation_pmid=24451623;citation_doi=10.1093/bioinformatics/btu033" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=H. Tang;citation_author=X. Zhang;citation_author=C. Miao;citation_author=J. Zhang;citation_author=R. Ming;citation_title=ALLMAPS: robust scaffold ordering based on multiple maps.;citation_pages=3-17;citation_volume=16;citation_year=2015;citation_pmid=25583564;citation_doi=10.1186/s13059-014-0573-1" />
<meta name="citation_reference" content="Tarailo-Graovac, M., and N. Chen, 2009 Using RepeatMasker to identify repetitive elements in genomic sequences. Current Protocols in Bioinformatics 25: 4.10.11–14.10.14. 10.1002/0471250953.bi0410s25" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. L. Tatusov;citation_author=D. A. Natale;citation_author=I. V. Garkavtsev;citation_author=T. A. Tatusova;citation_author=U. T. Shankavaram;citation_title=The COG database: new developments in phylogenetic classification of proteins from complete genomes;citation_pages=22-28;citation_volume=29;citation_year=2001;citation_issue=1;citation_pmid=11125040;citation_doi=10.1093/nar/29.1.22" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=V. Ter-Hovhannisyan;citation_author=A. Lomsadze;citation_author=Y. O. Chernoff;citation_author=M. Borodovsky;citation_title=Gene prediction in novel fungal genomes using an ab initio algorithm with unsupervised training.;citation_volume=18;citation_year=2008;citation_pmid=18757608;citation_doi=10.1101/gr.081612.108" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. Tong;citation_author=H. Li;citation_author=Y. Wang;citation_author=X. Li;citation_author=J. Ou;citation_title=Construction of high-density linkage maps of Populus deltoides × P. simonii using restriction-site associated DNA sequencing.;citation_pages=e0150692;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0150692" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. A. Tuskan;citation_author=S. Difazio;citation_author=S. Jansson;citation_author=J. Bohlmann;citation_author=I. Grigoriev;citation_title=The genome of black cottonwood, Populus trichocarpa (Torr. &amp;amp; Gray).;citation_pages=1596-1604;citation_volume=313;citation_year=2006;citation_issue=5793;citation_pmid=16973872;citation_doi=10.1126/science.1128691" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=R. VanBuren;citation_author=C. M. Wai;citation_author=M. Colle;citation_author=J. Wang;citation_author=S. Sullivan;citation_title=A near complete, chromosome-scale assembly of the black raspberry (Rubus occidentalis) genome.;citation_pages=1-9;citation_volume=7;citation_year=2018;citation_pmid=29300887;citation_doi=10.1093/gigascience/gix120" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. W. Vurture;citation_author=F. J. Sedlazeck;citation_author=M. Nattestad;citation_author=C. J. Underwood;citation_author=H. Fang;citation_title=GenomeScope: fast reference-free genome profiling from short reads.;citation_pages=2202-2204;citation_volume=33;citation_year=2017;citation_pmid=28369201;citation_doi=10.1093/bioinformatics/btx153" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=B. J. Walker;citation_author=T. Abeel;citation_author=T. Shea;citation_author=M. Priest;citation_author=A. Abouelliel;citation_title=Pilon: An integrated tool for comprehensive microbial variant detection and genome assembly improvement.;citation_pages=e112963;citation_volume=9;citation_year=2014;citation_pmid=25409509;citation_doi=10.1371/journal.pone.0112963" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Z. S. Wang;citation_author=S. H. Du;citation_author=S. Dayanandan;citation_author=D. S. Wang;citation_author=Y. F. Zeng;citation_title=Phylogeny reconstruction and hybrid analysis of Populus (Salicaceae) based on nucleotide sequences of multiple single-copy nuclear genes and plastid fragments.;citation_pages=e103645;citation_volume=9;citation_year=2014;citation_pmid=25116432;citation_doi=10.1371/journal.pone.0103645" />
<meta name="citation_reference" content="citation_journal_title=Plant Mol. Biol. Report.;citation_author=Z. Wei;citation_author=Q. Du;citation_author=J. Zhang;citation_author=B. Li;citation_author=D. Zhang;citation_title=Genetic diversity and population structure in Chinese indigenous poplar (Populus simonii) populations using microsatellite markers.;citation_pages=620-632;citation_volume=31;citation_year=2013;citation_doi=10.1007/s11105-012-0527-2" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_title=The genome of the tegu lizard Salvator merianae: combining Illumina, PacBio, and optical mapping data to generate a highly contiguous assembly.;citation_volume=7;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Y. Wu;citation_author=P. R. Bhat;citation_author=T. J. Close;citation_author=S. Lonardi;citation_title=Efficient and accurate construction of genetic linkage maps from the minimum spanning tree of a graph.;citation_pages=e1000212-e1000212;citation_volume=4;citation_year=2008;citation_issue=10;citation_pmid=18846212;citation_doi=10.1371/journal.pgen.1000212" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=C.-Q. Xu;citation_author=H. Liu;citation_author=S.-S. Zhou;citation_author=D.-X. Zhang;citation_author=W. Zhao;citation_title=Genome sequence of Malania oleifera, a tree with great value for nervonic acid production.;citation_pages=giy164;citation_volume=8;citation_year=2019;citation_doi=10.1093/gigascience/giy164" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=N. Yang;citation_author=J. Liu;citation_author=Q. Gao;citation_author=S. Gui;citation_author=L. Chen;citation_title=Genome assembly of a tropical maize inbred line provides insights into structural variation and crop improvement.;citation_pages=1052-1059;citation_volume=51;citation_year=2019;citation_doi=10.1038/s41588-019-0427-6" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=W. Yang;citation_author=K. Wang;citation_author=J. Zhang;citation_author=J. Ma;citation_author=J. Liu;citation_title=The draft genome sequence of a desert tree Populus pruinosa.;citation_pages=1-7;citation_volume=6;citation_year=2017;citation_pmid=29092041;citation_doi=10.1093/gigascience/giw015" />
<meta name="citation_reference" content="citation_journal_title=Hortic. Res.;citation_author=X. Yang;citation_author=Y. Yue;citation_author=H. Li;citation_author=W. Ding;citation_author=G. Chen;citation_title=The chromosome-level quality genome provides insights into the evolution of the biosynthesis genes for aroma compounds of Osmanthus fragrans.;citation_pages=72-84;citation_volume=5;citation_year=2018;citation_doi=10.1038/s41438-018-0108-0" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=D. Yao;citation_author=H. Wu;citation_author=Y. Chen;citation_author=W. Yang;citation_author=H. Gao;citation_title=gmRAD: an integrated SNP calling pipeline for genetic mapping with RADseq across a hybrid population.;citation_year=2018;citation_doi=10.1093/bib/bby114" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=R. Yasodha;citation_author=R. Vasudeva;citation_author=S. Balakrishnan;citation_author=A. R. Sakthi;citation_author=N. Abel;citation_title=Draft genome of a high value tropical timber tree, Teak (Tectona grandis L. f): insights into SSR diversity, phylogeny and conservation.;citation_pages=409-419;citation_volume=25;citation_year=2018;citation_doi=10.1093/dnares/dsy013" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Ye;citation_author=Y. Zhang;citation_author=H. Cui;citation_author=J. Liu;citation_author=Y. Wu;citation_title=WEGO 2.0: a web tool for analyzing and plotting GO annotations, 2018 update.;citation_pages=W71-W75;citation_volume=46;citation_year=2018;citation_pmid=29788377;citation_doi=10.1093/nar/gky400" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=T. Yin;citation_author=X. Zhang;citation_author=M. Huang;citation_author=M. Wang;citation_author=Q. Zhuge;citation_title=Molecular linkage maps of the Populus genome.;citation_pages=541-555;citation_volume=45;citation_year=2002;citation_issue=3;citation_pmid=12033623;citation_doi=10.1139/g02-013" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=E. M. Zdobnov;citation_author=R. Apweiler;citation_title=InterProScan - an integration platform for the signature-recognition methods in InterPro;citation_pages=847-848;citation_volume=17;citation_year=2001;citation_issue=9;citation_pmid=11590104;citation_doi=10.1093/bioinformatics/17.9.847" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=X. Zeng;citation_author=X. Li;citation_author=L. Bai;citation_author=Y. Wang;citation_author=T. Xu;citation_title=Improved high-quality genome assembly and annotation of Qingke, Tibetan hulless barley.;citation_year=2018;citation_doi=10.1101/409136" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=B. Zhang;citation_author=C. Tong;citation_author=T. Yin;citation_author=X. Zhang;citation_author=Q. Zhuge;citation_title=Detection of quantitative trait loci influencing growth trajectories of adventitious roots in Populus using functional mapping.;citation_pages=539-552;citation_volume=5;citation_year=2009;citation_doi=10.1007/s11295-009-0207-z" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=D. Zhang;citation_author=Z. Zhang;citation_author=K. Yang;citation_author=B. Li;citation_title=Genetic mapping in (Populus tomentosa x Populus bolleana) and P. tomentosa Carr. using AFLP markers.;citation_pages=657-662;citation_volume=108;citation_year=2004;citation_issue=4;citation_pmid=14564399;citation_doi=10.1007/s00122-003-1478-7" />
<meta name="citation_reference" content="citation_journal_title=Hortic. Res.;citation_author=L. Zhang;citation_author=X. Cai;citation_author=J. Wu;citation_author=M. Liu;citation_author=S. Grob;citation_title=Improved Brassica rapa reference genome by single-molecule sequencing and chromosome conformation capture technologies.;citation_pages=50-60;citation_volume=5;citation_year=2018;citation_doi=10.1038/s41438-018-0071-9" />
<meta name="citation_reference" content="citation_journal_title=Forests;citation_author=J. Zhu;citation_author=J. Tian;citation_author=J. Wang;citation_author=S. Nie;citation_title=Variation of traits on seeds and germination derived from the hybridization between the sections Tacamahaca and Aigeiros of the genus Populus.;citation_pages=516-530;citation_volume=9;citation_year=2018;citation_doi=10.3390/f9090516" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=D. Zong;citation_author=P. H. Gan;citation_author=A. P. Zhou;citation_author=Y. Zhang;citation_author=X. L. Zou;citation_title=Plastome sequences help to resolve deep-level relationships of Populus in the Family Salicaceae.;citation_pages=5;citation_volume=10;citation_year=2019;citation_doi=10.3389/fpls.2019.00005" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Zwaenepoel;citation_author=Y. Van de Peer;citation_title=wgd—simple command line tools for the analysis of ancient whole-genome duplications.;citation_pages=2153-2155;citation_volume=35;citation_year=2019;citation_doi=10.1093/bioinformatics/bty915" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="De Novo Genome Assembly of Populus simonii Further Supports That Populus simonii and Populus trichocarpa Belong to Different Sections" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
<meta name="twitter:description" content="Populus simonii is an important tree in the genus Populus , widely distributed in the Northern Hemisphere and having a long cultivation history. Although this species has ecologically and economically important values, its genome sequence is currently not available, hindering the development of new varieties with wider adaptive and commercial traits. Here, we report a chromosome-level genome assembly of P. simonii using PacBio long-read sequencing data aided by Illumina paired-end reads and related genetic linkage maps. The assembly is 441.38 Mb in length and contain 686 contigs with a contig N50 of 1.94 Mb. With the linkage maps, 336 contigs were successfully anchored into 19 pseudochromosomes, accounting for 90.2% of the assembled genome size. Genomic integrity assessment showed that 1,347 (97.9%) of the 1,375 genes conserved among all embryophytes can be found in the P. simonii assembly. Genomic repeat analysis revealed that 41.47% of the P. simonii genome is composed of repetitive elements, of which 40.17% contained interspersed repeats. A total of 45,459 genes were predicted from the P. simonii genome sequence and 39,833 (87.6%) of the genes were annotated with one or more related functions. Phylogenetic analysis indicated that P. simonii and Populus trichocarpa should be placed in different sections, contrary to the previous classification according to morphology. The genome assembly not only provides an important genetic resource for the comparative and functional genomics of different Populus species, but also furnishes one of the closest reference sequences for identifying genomic variants in an F1 hybrid population derived by crossing P. simonii with other Populus species." />
<meta name="og-title" property="og:title" content="De Novo Genome Assembly of Populus simonii Further Supports That Populus simonii and Populus trichocarpa Belong to Different Sections" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/2/455" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Populus simonii is an important tree in the genus Populus , widely distributed in the Northern Hemisphere and having a long cultivation history. Although this species has ecologically and economically important values, its genome sequence is currently not available, hindering the development of new varieties with wider adaptive and commercial traits. Here, we report a chromosome-level genome assembly of P. simonii using PacBio long-read sequencing data aided by Illumina paired-end reads and related genetic linkage maps. The assembly is 441.38 Mb in length and contain 686 contigs with a contig N50 of 1.94 Mb. With the linkage maps, 336 contigs were successfully anchored into 19 pseudochromosomes, accounting for 90.2% of the assembled genome size. Genomic integrity assessment showed that 1,347 (97.9%) of the 1,375 genes conserved among all embryophytes can be found in the P. simonii assembly. Genomic repeat analysis revealed that 41.47% of the P. simonii genome is composed of repetitive elements, of which 40.17% contained interspersed repeats. A total of 45,459 genes were predicted from the P. simonii genome sequence and 39,833 (87.6%) of the genes were annotated with one or more related functions. Phylogenetic analysis indicated that P. simonii and Populus trichocarpa should be placed in different sections, contrary to the previous classification according to morphology. The genome assembly not only provides an important genetic resource for the comparative and functional genomics of different Populus species, but also furnishes one of the closest reference sequences for identifying genomic variants in an F1 hybrid population derived by crossing P. simonii with other Populus species." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
    <title>De Novo Genome Assembly of Populus simonii Further Supports That Populus simonii and Populus trichocarpa Belong to Different Sections | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"z2cURYbekeISSUWePQo6-q3KWmYgbLlhRtSAr0w6kn4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495551","apath":"\/ggg\/10\/2\/455.atom","pisa":"ggg;10\/2\/455","ac":{"\/ggg\/10\/2\/455.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/2\/455.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/2\/455"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/2\/455"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/2\/455":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/Vs9EHnuHU\\\\\\\/IzvP51dTnhERGK0SP9hxLGMIzsXNq2MoqMMLdxVIzLzCwkk0pZawVdST7x52HxHCkQkNh+kzeNWgVEtRBPL0E6Z1xpeeo\\\\\\\/CdA1ra8UJk05CDnTMAdwBmWb0cHb4R+KwV\\\\\\\/v9hqI37WB3NUOJL2Efc5t1zyDQFloRtS+dR88GOBTA3saV+t49C6TG5BZlUB1Waujg8R4tR\\\\\\\/oBIA63LHj9o09UNMPjkvTdraQ5NkSFZSTJD5qsr4t4OQxgG53whwL4VXOD6WbtuuYleeyOKK+KMJ04LRPiA9Wraw3L0s3SVEDiA6XBQGgpwBTB8u8fuIewuS1+kPCxMpRgSVF1eBzOsedK4jOfTCU2lvBKaT0VjYjKmG8fFlinZ04qwe3PWy1g99nMg8cxxlhi7EULLMooQm6wA8wC\\\\\\\/ckiauImEA2Qnocj6uyan9SXrXQB1NdBt\\\\\\\/ot0jzcOalTg5okiu2P487YNcB9mb2qxxnm77PzeO5PGwk2hCYwXS4NPF2BZgZg+8kyXe53fqSsorisvJlwcpAe5IjO1kVESsnxuBN98Exkbk\\\\\\\/2TtcRjwTRsMJh5VSRoHsS5hUCOQru\\\\\\\/jA9s\\\\\\\/k0674karuDh4tuwmunAsy4ogHEcSeUc1QzrDDeFgSuih+fjkCjxA1\\\\\\\/YtjK9uSfOItKf+L7R8AxGSTFyShMFuFA4x7tphhrRkPnEZMYRS1LngZO2AUgwjXPyQ5hF2Aswg\\\\\\\/F8+WhYJSGavcoD8dhnqAO9YtdSTMUnwIVjfjWZs\\\\\\\/wrVW80nqPnnvA77S52oiCBeOtqyHGAlXHs\\\\\\\/D+WMZFnS25oQqC0EX2n2Tb6oP1dYPDl8QJzl4dv5PIJo7DkfkYSgoThYT+JqSYXprH7F9eiqvNxcSzb\\\\\\\/VAK5jNOlMtEOiZKNI1K79NDwR9+aoLPW3MVCo2dw794WMzZRx9V8gUi6VHCloOFLgqDY4\\\\\\\/QcWBXGqnofSjyXGsSjv9gHcgZAjvDccBuFEw6el5T071dm4FAi8u\\\\\\\/UXlGKTB5VNAs3iuUHE\\\\\\\/3zDfeflrTAjbZlvJGv1ZzmGxGIqP0khVebDhZNJjM32yyqCLTBzRz64dkBsurBofPPpO1RYzXeBbuynZWMZtwp7kMaj45gkttsq8h3VSwz\\\\\\\/lx5qFRnhx73Al8niDA1xatVwH+m4g0ReLlW7aqfS+Lxc2GDRTQ0\\\\\\\/HT44nk2xC+6iNpc8tflB3rO\\\\\\\/cJDdY\\\\\\\/ZtjoM1b2aCNP8uVjpjobS+rOr3\\\\\\\/f4V7icTzjHG5usduNfwMEnA83p0ghPB3A7XEKv\\\u0022,\\\u0022iv\\\u0022:\\\u0022StjLqWxwYzguhHasuRkrzA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224642fc7d1c025ffcaf4ce913c46f9c37000953c4e5be9bad34a61c1d7c83ac18\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022uHBS6DMoMLrW4XtpS2TzXf\\\\\\\/8GTcyJA+sRWg3OIGHv0tGraueNQJ07\\\\\\\/48k93C4LjmQbYZed+r\\\\\\\/5nqIgxco2SBPgnBqc1zKjrCzLkterVNlfOptmw54OCoa4ytyueEFRbU4LUKpLZXCKplwgGloc1TnWnDbWLszvPjFhwZddmPLqn+AUTE\\\\\\\/NnxIO6jR2pZNfsPUvgYAHbksxPbg34gIVjaPWqw5nOueUKkn0gtaIxvhwC41KQvRGKEYaXb4WfBiIB6oQXh6YvE+PBGGPnTvNcTKSrssnumpskqCLfT3s4uL01RdCfZeRwP4ZMVohr3xvFXTuJ6Vg33p7SxrwtLXJ8D8nFcpHi2e\\\\\\\/GFBRmVutH\\\\\\\/JGSIZaaaU64HvaKW6jmzPtX\\\\\\\/RR\\\\\\\/KMdIHJPEnvcgW562TX1b2XnHB27+3g68gKDwsbjydPQUH6pVEAXRsrtdzuUvEFgJCijUZGDSW+7Qc+FY53d8zRHoCDLCjIx1dxrwsTMgS\\\\\\\/i4G+69uAtL1YRCoipqSDDqj7OSbgzXD1X+Bb0zsL5W72GSzDmTkpKeEhtzDXuYChMGx+B1sYl946WyzbwzeWQ2lrr+2pu8xxUyvMHtaswXD\\\\\\\/NQE8V\\\\\\\/+NmAszkqvX7Xyeyh9r0s43LAoQHCp67aVokIioZtB0tI2goRXIXAVB11whxUDUOKRMmLDMCh08I8ytN7qtE+mx9EFcZW\\\\\\\/Ij8wy4hztdLRFmgWRLgbZIOkYcWcS62qB5wJtjtN8ulG67qw0ffYQ8J+LNz2e5sBJZ8n0bx4qjvg8GgTTzGPYLg1kna8rdAZAG6ArE2geFqkpxTfv7f4Hkduyc9wB3p\\\\\\\/aBc5Bslsrdjmart0wJZXVJSIHWHgVlEIxuRYW65r4JdxQSaJwkhF6YdnOD2p8TVcAgpyXiejMob2OXDoCAYMoNfASVsD\\\\\\\/d8zZ6DCYJzMJSKIfZBaGHBnU3kwAHHXAL1jBR2YeF+QnenDL8eMkUuyMz6xpo9DndoluAhe\\\\\\\/yPT4dFaEBuSfaSDwroQeNpKVYtD1kMTqnlytGm1MFNfEMk0kjh9BXFojRN6XUzD57wQbuhaUpUnkzPGD9zPw8\\\\\\\/LKvwYa0klpuqg6jsQlqrfZHBZQoI83he+cdgEKEHuQM7BFo5HBMDlbZf5K8IizRo\\\\\\\/ix1lXVzQTGGaWCvyTi3Y9GrOS32mp06LGioICGcLMSRz034RFz9dZBKxOIg1SDeI2cIpVCfgOr2Mp0R1PFAZRr2dV1DFuqPNQvaJo1V4xpSb0Nsm1kmDBDsKKrsFifWgnM6EmT1f30CfVjn9MMa2o0Zv2hYxMEhezXQJ0LjK9v5ff7K9K297ZtmmZ\\\\\\\/V5F+urzk2yugL6enCTnrfqHo7f1gtc\\\\\\\/8DYWltbwed\\\\\\\/BBeCRosED0CjYRKjE4+2MdkGDDRTgf\\\\\\\/9ddX0l42ucWhRyM2zGM3Gbb8R4DaX2SqBuMQpMMF5dPcGb29R2xVnd6gFXtEFeg74tzjPcVRnc3GV+vmSuu6E2QzjhrhzjyodD4tw3Hake\\\\\\\/HmTGFH58CbqcoeGG6w0vBbvLkAECw2HKKp4QHjWCxfFMCjeKqyf0v59VgbVfwaUjb3h4DphL6aA8vr3TNYfu8O0Vzak9rdkRu1F93bSvrqIV5rUcylo68WyrbKQT7y\\\\\\\/CS2RPoWpNBW7+dlCdPZy0qOrsfENMMuGaW7pvwG8ped+EOQrxwGAu5R8RipoMJdRUBvlCgXjTbVp0J8y\\\\\\\/kcIuq\\\\\\\/q\\\\\\\/afqLH4P0fTX+KQ9QrxkzQlfAYTUPFuJ0kNHQWFG3m65NGtu1cdwOYG3b92c8tuQg4GXNBpaYwfZ1Cy17PsISUQYspCGw8lEwk3VTTRAc50+0X25f2ejkTyXJqvMQ8z8RlYJUCwKccBMcM16Gy\\\\\\\/AwPIjftc1VOBsHuSW15aWLHimDT3\\\\\\\/1HDaf+bqZL\\\\\\\/bvVlNI7G796SD8xCY3FcqzWdhBbEbJD1VLAtJiTWEUlYU3cmF4ak5s0YG8gs1fZ+4HVhufPVwc8U8ZbtueK7Bdp99IEBWlFP1pY0OtsJ78ZWDi2gyIc2Pf9BoIidEg2udNWk0IoFCWtRWYPrgiNZ+kLM5h2ZTJG\\\\\\\/oz6+Ctk3P0mGtKEqPJeV0+KCM6GggcLnnVBn6OZMzr0HsruItRzIAEhKrqyitIBS9j4dFcmg5sz8B1g7MN4RCh9a8cte5\\\\\\\/BJVHXwjNudG7T9Ckql18sJv8HmRkgOvbjkDg58tLJyBQ5hpBXUCTyel1d8t3RHfEKuCQePF1dUHgsfO3TlES+IPWi4ABkBtfLycjmNeqRbnK1L2N93BW86b5Ro\\\\\\\/xLtlFBcVavcQc4r3GkTfz3s2OnV3lYtBEry11Hc48CfpDpO4p6T1PjXG3qGyG0CsSE60ulWWjuJ5Qgw4LCI1E6hSfLBd4H5ho9LWi\\\\\\\/GPTnVxRW65Rlg+y0iQgSlMNkusI279DkiqcQaoJ4w+SHsDm\\\\\\\/bOg9XC2hxQFVxzWfZ+tEdLr5WtdDEIQGtfFr63h6LFaUZZlOxISw6qKZszRtQGW9eT2gUwsf88+PC\\\\\\\/hvy2VBRr4EexvsSVJBApO96+lP9yVwm56BqhC\\\\\\\/aVczeTS0ssX6hNqaSH\\\\\\\/a58Yz\\\\\\\/0vhU4RSvqwKcc88YZJxs3PwNkx\\\\\\\/rifvL80vBL5Cfs07pt7nr3jqhNifXyF6JOcOSZQiQ6d+eBgT9Cx5Y86MS67JDi\\\\\\\/vIRIvpqvn\\\\\\\/wVn5PH6FhGYuZ8IDSM7buDOnTijQb\\\\\\\/Np\\\\\\\/6Fr9IxCq6jdKCTPV0Xc5Y2duD1\\\\\\\/CbPCP1Zqfxb53ii8akdSB3j2wLIcBEdpRHFqIs+\\\\\\\/8dBL+eRu2qbxcOb9HTF5mBZ9yEO5p5gTWkPVXoD+HsKZQ6l9WQcu6qhgwZaHEdbbAYFHtc0qQfM0FsXGHW+kz8+vHALZZvGTFxjcPosKL\\\\\\\/5dnB74cwaPlic4gNzxMTdJLA4oSP9xzZmaDCJMTmrUPtRzMDnWHWr09RUrXjJNV\\\\\\\/A3wE4LAeojKnAa6puVb4wOjH6AocKOYgo1L20Bha3HF5hWsRMPbYgoWjF2\\\\\\\/8djBsMii6DFGkUfCC9omqAABa9ZaestBosjHZWLWgrX0rA8AM5BTjgRSqucgDNEtdhO8+nlIWcN1SulXvw1e25fqKRK+xZNndjjQPC41nLUFlJuMeuvwFWGNdXQMWpp1msu5M6buFkPuj63Qdc8sBK6N9QzmNe3N5modlZreumrpO8Ui4iXZb8q6zcfJpIiu8HoCjDkS3n4ZLtrm80dkOrwxRS06\\\\\\\/RHFWzQ9+TS+c35KuATFEZ+aYm+UigCCu+8+qPqd2A0NKGQaz39l2KW17s0KN9SKSj7Yn7RV6YTW1MFPfxrT6ikQl0g\\\\\\\/Kizph8S5+SVQ6Pn3R133UsF\\\\\\\/RyiNxbC+W0U6o9PV7XXBKsHQh0WO+27aEW2dHgCwPNhjAJ2XHI5GMVhprBnYXAim3BsrGNbE\\\\\\\/mzKAoZ9wYo8\\\\\\\/BcL2Ko0YrE775vVR7rvJsrLUFxQfizT7+LjvVm\\\\\\\/1BbhyEJSKxufZA71tEudSRpNujPBZKervzHt8MyTa6C0t5VObnOy3mFUQP9rndQWsPhMQo\\\\\\\/+ZXL1W72VwPifkbZsjdXGu3Nf3zRIcy971gxv2eOVcyyByabZSOEqBvM\\\\\\\/V3VjDQnIrvEeRRSGFWMNAmO6fmTxYgD\\\\\\\/aDtwG9uTpTanseMJNEdC3nRoJdGNVwwkych2vkUefAhw+23aS4u1xX7kLKxnhAqgDFhrVB41R4Kjah9uyx4EQrQezbTM5c5dlwV5zG2QU74Q8YPMllRy\\\\\\\/cI+0GPh\\\\\\\/iLeylHfhK0mlREN4b6zaPBfcNOOdiKohfSomuUxijoqWqF86OrDvkVEhENv9jkN+SjNYYhw9nS+peqAUbGLETS5kYAtbd03xX9ul\\\\\\\/Ms0+WMcvhLTpkF5kUI2I2oGQReVFzouYBpsRDlVwWajeAcrEOifonelwoEGQ5McVw8KNyGgEzZxKW8151cA3LrmtxHFUkILbwkof1qPcurG6xMpM\\\\\\\/BFdJSqFcGcz8Za6cj0jHzEczULgFED4YanMJSVW+sVsjgOZmNAotoXCMj7fqtHL18C2ofir6am9wYzDzG9mGTCEq7WxuN0i98srlqkUg6AH18gTJoZC+w6J+tfGPCaEFeKuPTD+d7TTS0Ur2zl111UJs9NBY0klF97rASoRJhKy4WndERCw5+LIm3soUgn5sFouyuAG3l3hQZARzjRf9YCmWlD+Avy\\\\\\\/NND2+AOM+MeSv4uI+XvgSsGtFWFxKQRmkvqRZT6KHiF6uUFXgk4H9IigoUrfnyqmNqu7OhVxc2BpvKNztY3097Zftqlfh+KaqjlqjHoe1HujquGfazJQt8lD1Y\\\\\\\/RZ2RAwuHLcNzuHuvCQxRfLgu+pCgmvyAorG0Wx6RMyb9d12J8npBxC7DMCI60avSM6SnSEW8oTrbCcxwVeG8xhtS3EIAM1fLRCz3hjO\\\\\\\/MHFNqpq0vwJjVGEX5J3JEDtiKjycVGIgYaqSh4ZqK36+4loDSd2EAACr3nwVycgGbQh0Mle8nDF9+zPvkphASCHLEflIoGOuB5mxnYk3geE2Pyv6sD60rNum8Z6g\\\\\\\/6OtY2bsOvdkuk14jOvU9cO7lCBwLFeovfdIvLdT5VBvlOvWIOkrZ0jRoHe5HkkbwIRDZaY3r6qr\\\\\\\/ZpTWgTg9Ivt0COS9CSn57d\\\\\\\/4Q0dx5cWa2NXSmWajlg2vd2Acj8t1N2oICojWuYxZ\\\\\\\/s9x2RTCEhhO1VuLM+RnOrevH9Svw3MbEZPRyzuioH+6CpJiWgcM7Tl4GEYkgK9ejOxxOW956HOql\\\\\\\/yWAgrusASm81mlXENHn+G8QZxxdSz4f+Dzri1Emso625VdWoRli4EUjsILFD+Q9iwr89ghRnmprLuoyJm4Fr7eG+SStmRRePC3mOJuf1WQa86oExLuaxckcpeK\\\\\\\/mYFviFJKNHzKzzCVoXqxOHo2DLZL5gwn1X04cmuE528LxQkLoP6c1rMoh98zNB4Vkf4DGrCDSOY2uJvkARH7qs7EJRqucfvrPJDJ+RCpbDL\\\\\\\/Kn1zmtrElDjKzBZGmRzGatG8g\\\\\\\/+5JbCyPToHJdrsl\\\\\\\/ttD0YeQB6UR2GOZdssNR39V\\\\\\\/o2KVzuVuLWmW4\\\\\\\/IsCi+8\\\\\\\/fqRMX3FuJ11XZxHSHVJL4rKmTVkPjfT8dECb4orUeUxqKK3nvfng72+Tzx7HdpU9VWDSKg+uP8xPjfWxDDcOU1r0pqM3e8cI4foiibGTni7pEThtzd5cYBtn6WLsUwUPZQOqRBVH0BgGcniShvzgk9LgU\\\\\\\/sZvnsLnFTzZ32uuta6hC4p2fh+8i1xdFNkjqWyusCeQhTKmSCClY5sAff7OnVy1yhU32dk2P1Sc1+dzjlI\\\\\\\/mzkbI2D1dqTNDxoJMnrBEzyJ74BsoVAu5ITzhwvEez94Luq44L9FhjI+RIqDRixrxbL43qmdkVIvTgbq8bvJTd76j7hpxjPcXG+K7aY5Qrjmyk3oy5Q+dlarn7QTwBuOKpffUu8YRgUf8AK8FTxAhXjxKGGCeESeWfiamJn1ElsRFrAK7ikplGaNYWK21H2gKVWBsUUCgzRmmkje6HBEpg4TdoCuusklWPEn8kvnddmZnYs6i4V0rcAPDPzQ516FXW9i5zeas748Q07fxCZQH0cv6De2LNdpa\\\\\\\/wmR5wq7HSHrRFFs1iQ+m+v0dL3M+DSW\\\\\\\/M2i9BXrBkfdBf0AsR8nAp2j5vp9Ujh52rz3Q5ALxa73cVO4cwhzpBznwY12527XIw91kl\\\\\\\/qFHdKl9Y5M9RXx\\\\\\\/xl8oeneVrtB5p+eeVa6DWw4YZtDy5p6kF24i9Knyt5ACXz8peeh830\\\\\\\/e1gIdIfKEZ6x2IHa2oPM5wJTn4FPXRov5AbLNS91BIwh181ko17u3ca6dvv8gJ7NZqsFcNq\\\\\\\/xM9j5ByqScxeXSm6MaqzmfHuiqblMnb2zawjmAAX1gtBsKByUsFQXKc+SRluip3x\\\\\\\/2TMZWy+Ecw9Wa\\\\\\\/JBkcZMHJ+7zWw939F7fmO7o\\\\\\\/vrjlZRjTegwyGvGzp4TVjXyqDoK8eZcMSKGVaeRQVTHfi0Ilqx\\\\\\\/4E7uaWfsXZ748G693Mx9JLM92Kln0D3CV8WP36f+4LwGcyiOAp9tvYz6\\\\\\\/40tdyLsFLwVUJWvpDGyBk0V9KnvpJIk2U1CXearr59rdw5kn2VGtNiGgPEyF\\\\\\\/6Yd9CW6G99K5UIdMfgZP1dBFi5XUyofBJP1phoUFbX1WsoHYxGcpzj4b3xhnaUr0apwCEOA1nlap44knbpGkwyirBc01fJllIyueTbeuzeUhr\\\\\\\/9azQHPjZ8WzY8J+nyc2KSEKu2uB4LmZJAr8WgMtc0uLdyOlRZmbQJABM+DRrvTY+zIomJpNASbnOlXMZMkCViwKPvffi3gcFmyqb11XlTdwbWD8EYbDseOrbwVNCLZMUiFVa9+6FZJdFWrX7fCH8Vc7tGJkDMkkivxbEjs9UthMe0sYKXDob+lUAh9b4oKXNxuGXf2fW1Ki5zvjMtPFKOQfcMqo\\\\\\\/M6JxU4LGkvin5dpAxWClr7T0eEoYrS3tvk7dCaGw8BKhQB6\\\\\\\/2UYd2bbJAFQ4hFtWMhfg+Bjcqfu\\\\\\\/i0uXLuekXFaBtbATCakV9r49UEys8R7oDT4hgJSg7ulMRn4XrI7As1JmqN0W2yE8YfCu8wkUDb\\\\\\\/Y3kH\\\\\\\/enm5QEV\\\\\\\/SpyB3vxoYwjSoHPVfn2aN2WhNk1eDRTqFjV9UIZUU9lQBEzL0L\\\\\\\/Y49A8uiCfsYoYSlKJQyNfnube4m3jqJVumI1vGWPssXQ4f95aODYOibGldpo1taB4bpp8D7YebgQuy8dUeDlWavGA6vdAkf9MAG9kVYbFNsLbE5LFwoR83DSlXOMWdWrW9CthjQ\\\\\\\/RgjHdzlsk4gBm38YJBZInbrDb5iIfdULKZSP8ttErgaVw7+cVTZEhxdgGnZBg39ozfNSqy+GtALLqSKRmY5xtEVMxYsMdfOi7bKcA7bH4eCVwqdJy82ZWO4zDz1iO6JuXFbTJ3WXV50Fm\\\\\\\/jxxPGxF5yWjdwWbTI9kud8pEiwzod2OTZ3djGGioVdCe141ZCbN05xKK7wZqttS7vguMGPWxHNTuJJ7yWR3JiPrJCH3SFy5ynY0xC1fMKytfK\\\\\\\/swkT\\\\\\\/IEqxVOuV+Jx83PQSGQbJkJZCHB8CwZZOKWeDV5VHa\\\\\\\/gEZvcjBQ9AmMOyvypB1WKoVk8Lok3+A8SrqoPZ6cOqP6EXIZcfwHP42\\\\\\\/6rlNxjrF\\\\\\\/KKVDYbG3aLp\\\\\\\/Sbr6Fe0fToOFRyPnCq\\\\\\\/SoOQSaoV0MIeaDefB3GEWmXYd65Ot6r708HKurHxO7HsDdG3VSwtgfmEHrN3Up4JUSoa7k\\\\\\\/ukFAf1qNsh0Nce2aW4hhYoA53NxpWFYEMdsXLGGMRVTXkYfNVU+YwQz3KbULB4PIn4CE6e+BTDaE43wpB7\\\\\\\/oooGHyqeCVWqxb8zzfIaCdtaqfdftkxq2nYMe66D+sm9DbRGGiG7Gx5s0zYKdXv+vIkbGGcWnO4zj0sfxTd3ySf3spYCsUfpIOD9h0dOp0X22Cgk+Ad6Pmb\\\\\\\/bpN+oivBAEQcC6ZZUdX+XQ\\\\\\\/9FJ1F\\\\\\\/vSYCo3lYOFjf0pe1PqSDROytxPZGPja\\\\\\\/lt6J8YIL34jMLoRkS+uMpCVVOOPK4ydzKukacOiD\\\\\\\/sbNnsI0BTmSz\\\\\\\/lqSzosrMCEd7rGf2htjnFDEWRqzE9KNPkslqeE162E3Tc8ex3XuVbVc1yxf5Px3buAjSMbuMMhjcaOvD9hONHB7CmZ0r1gWT6OIeM6vlNmZZfPvQp72kIooPko0EBE\\\\\\\/QQPPYRldMDxj6ik2IL00LVSZ2XCgsC\\\\\\\/1+HYqFR12wY4L5icIK0SGNJeWOO8T4J\\\\\\\/FprSJ3242vLrGDNjEcf7SgeDzL74vZHKqAqR88+i4KM\\\\\\\/Ou7JsfPIU3+b0z3TwqSkyifh\\\\\\\/1CDB4\\\\\\\/ugTDoa\\\\\\\/4hgiJv4KfTt2vwN\\\\\\\/6I3J5q40dE6qJOja1KavPXy5TLjgmLONAOb5JNTLgPOn6Wymw3kPtM5QYJGJXmHzgeAGTYssc2K8EZmizhvUFOfsRa2mmsL8Ks0dCMrrR7\\\\\\\/efii5FEAafZT5LSc3Q1q0tql\\\\\\\/iETJse2wUPj9RDmz38eZqQq4d0Rm+L7WHszAwAlDhMjnqCGdaja2dzaSnQXk1FsIRTpiBulbWLucpwryNwiu4TBYk3IwbpQ2gleKef6bW+KKhFbAdsEqOuSiNNCftxGH7yUkR57bCmk5i2r18NAJbzklQ+IolxAD0WdzLipn84+KCvyp6rndX3nSjT1nZQXnaRV\\\\\\\/zbzEIAaFOi7ASgtmok4PVV9X2ZVJnkJ1S5\\\\\\\/RTH\\\\\\\/n2bdOEtuXIU8+EL\\\\\\\/FyHiWhAzjdUUS0hb8gJKRe1IaOiFjYhS3QCWokSXZAknMH9h6tlvYngbB\\\\\\\/jeiB5TRLHUcYMPaL2Oluit1OKxTSqcFGhKaeOW8uIMMA+FIPHLS2J3pDkP4vkvMaifhmlewVpX3g8gsgMJlMdphCklAePBiikHViVTBnrAOVruKuGkp3fGImOyToe+X6clgNaRl7LJmXrmEgXhK\\\\\\\/B4I64aMaBKqlqtePLMVl8rDTGxdirToUg+7kDQXD11aE2BrWmtiygLkxIBSQgZwH\\\\\\\/gCBtAtdXEZ6p9Z8Gzu5Ng\\\\\\\/ooXVSkNxaswhFYK1\\\\\\\/1Kt2pbYfgcktcpFTNdbAhd9hoMtkKhwwtQtmtwBbiCv7z6D3oRQAJ5F4GNrtdXCJVgf0MHU2W5YXbMIBnmaT48erWcG6\\\\\\\/Vw4Mme8kwqu4UiRPDQo56I9BZQaECBe3jGz0SGvg+fdHkTe95CzadqOMSnwWOPtZoBnZ+Iqmn6Ramo+QSR34ZZ3sUq9Iu8qrVhjNEszACOTyEPGrqDI75K0GFEdQhVN1nAEbR2VMmj4ExQlzfQZaLNWLbAVQl5kcZ0gcYSsi23gCUDOd2gwTEyu5UUKVE8S++k3dNy6BaMQh6WxUMn+XMf8MDhsWFGStVNvd4Ifk82Rnn8D++aMIlVNDR46+yhpGAjdl69NtJuHal0giy4WzfpJUQ4ZSXMxo3Zy7eQP5T1DNJlsM55SO1D2awyVVY+h59ZDcBk18ocSbG6SjUG2HX4Kgi0tweKbHF+6qijeKOWGmK+\\\\\\\/U6F\\\\\\\/rLX9zvJinHkMbwMhdlm2b43aXqEATXpXtkivI2L58U52WkLpoK+DUu7Ax\\\\\\\/SOq0hfFhtT6I8tOrdsOH6HuVNZiEpChsZAGkCBSk4v7ZUz9J2rCuoBGg4ILz4OvU4sJ6tFsBRHHa4Cki+AAp48mqWAdSCNkfYKAoxv\\\\\\\/uyK2aRBgTGt0hXtxZ30FW5CM+SsKDmxz\\\\\\\/FI5yd9lXq3JpW3kwn\\\\\\\/P5nVlAaWKuxGfBqyIht8sQrge7oFsMGyrTDX53T5WuXFAwFaqVUX9oD7dhw2G4f7XVIdQDO9RzGEy2BPBs0zPNFM0r\\\\\\\/LKGfnUTkbcpddUPgDZg7nJqF8e5gwAQ\\\\\\\/00y59TY13PDBgRZ0eSFzzYafWOC\\\\\\\/msjJeoVYV\\\\\\\/JW4mPBCpFMD2OSmWwVxhPB\\\\\\\/Ta4vct0dd52SunNwyE6Wa8pgccRiYv2avuyulvaKwAfQrc72EUXC78ZQ4IwOCihrox6TYv0F7MAcKmN7Vt24kolIRWWzOi1D+N9l7rNHZS7X3eOLfHpBlBeARCETENNs29X2bgDrP+KPkG4zvh5QQqClFTWafzok2jqbreRxI8gLwfwoJmI1ByyiHxdj0gAd5xc+1snrHwzh39aHBmTtaKPwmLY47Q5DahS0sVU8BG1mr9NilmX+WYBgZBaIT3\\\\\\\/UN5PDWSDs2ohGZygQUQO1nCIoBpTK1D1p2K5AreXv7hpLhiY+7a+HbxYso6BT32pjUj2wVBXRnp0Ls0ElGdi6Yx0JICythdxdt4vNxshg6qUBbuzMCYOGt8CSbaFNprJ8UZmcaUhzqox0w3DZECAhhNsT7XfhVzdFIKmPKZXZypj6K\\\\\\\/bYHtxqxL80gBh8jzPHH51p5hzruWU7SD+eZHJ\\\\\\\/Q3dXPI1Yi3dG2lXW6pQWcE06dfTzRvNcruf9EAtHlvbObq5bPlZQmOS7ROyudgcZrNwuySxNrK8\\\\\\\/DLFgPVlGDmnRPKwDXOvR2cXAEGmkUjYhjfZxc3KBkTaVupmyb8lgVrz6oOt3FzQGF25ykoXNQbNEQIjWNWR8LZe\\\\\\\/8g3UohmXPFwq039HLSe1ADHTt14bNiU9CxTjBjBky6AEDMIRr2TtKvlus7i\\\\\\\/G6Wfyaj9BMSJ78lHPoPavh6u\\\\\\\/AH8BO8bavAqA+bcNDUsykHsIp5EOXDeCCwjydFV1TMj6ivocfipjLIpuDvH+VqjVpNYU8JOZzL\\\\\\\/124k8\\\\\\\/hpvBEODrOMhp8Ol6WVSB2+mR1PCg\\\\\\\/xctthvdG2CLY2EgaA01Oiv5rq21F\\\\\\\/Xvumglsk2qN4z129CSiNzIhwMWZUpFvvvrSkE2p3dDVlG2w6Gx1xURSD7SispvfgXpdRfwA1A8bAWBCTIZKwWmgMSo\\\\\\\/saweEBM4bvjnPGhtiE46EcD5CF4PwscEdCXQttkZLpGCeNPf9cs2bIwNaNK\\\\\\\/9YMdZnGv6TOa5Zc9AHa9YBPNznJUx7SWnQ\\\\\\\/gB\\\\\\\/j7zlJ5BOIfNU2abIQQbh080YutLdCcHsE9qdQxKVyjkV0cYrAoVnqT0\\\\\\\/dVkQ26rTxGOnunK+C2T7EGnBiwYoLOsMVKBKGxlaUhsmjLaQOifsACowhdgsTvdBxMacImF24UK5U\\\\\\\/Drvaofgz\\\\\\\/OjIOrt7Ibt+HIYOe834+7q1XztSQvQFTYCJeFAxxLrKedd6Kjm2rIA2guerjQXmHRKST9DPPnLtoKNZ7MyuNWig2MA7oTnPB\\\\\\\/gc+9phhl+g\\\\\\\/84IwE\\\\\\\/gNoU06+aKQRwkzSqqcMZJqEnz9hX1+P6w3qQ+MMWccqodVHzZOBQCWrd6Mfyv7yuvgOtQS9Nq7SAsypfMiDaFZ2539vetc4m\\\\\\\/Ar2kICE8zl7QX4X8eo1heMs6LbUqhPt3jwsU8HBz8XahTpF1vSSnaBR5Y\\\\\\\/d8DbgjFEiB9c8IdeRzxU16FtOTTEy\\\\\\\/QF+v\\\\\\\/fs35PYqjGFMqMkMgyYcs4T2xO1bat2eeER+4J+l7X8\\\\\\\/LAzkdrwBkA\\\\\\\/1WaAKNS3+HDrFFNJcUpgvKfwy8hAM+PwKHkkM\\\\\\\/tk9nGPCq9xZGhjVFzDAy7B+NbMUy+EMIUnRp\\\\\\\/E8SmdcaWNjYhMESVXB0bVIcAlOoZwdv3SWRCulhZYKVd+gqHLAWIIK8d4\\\\\\\/pNCiQ10Sxn+WIU9pJVGfyq71yi436eUHUfuFc\\\\\\\/EewqJDjEx35MlPFxbVAVZ\\\\\\\/NDSr7gYugB8f+VNGcwcctGAbLyao7Qy8k\\\\\\\/1aTJtlJbkDipMS41fBt7D2arKzR8fSjpKHzNerpgYWBEm1\\\\\\\/tl+9YEBKlXWbPc90TesQTDqrp+IYUyezSNSCbSMgMM\\\\\\\/dQX3ExWtDwgDFUTStDEBKd5ObytzfL04BwSKq\\\\\\\/adNI307Z1UJtvF+MspUXV4dX8gyVHlTycbOVsCWNxTDXYbBNdwmLatuxtICZp6DCUYWudNFB97tg7NIITcwposInozmrwsCIf7FJv0SU044vfvRHNMtonkEpeHmalQHlHPIPMWlWeflMXgV2olesPMYuNUikxQ7PdkQhkCspaP+OIoeEYS6722eg8tfKwZo\\\\\\\/8aumWeiMx7fndiyfQa50+lTEAI1UswLSzvRKF4M7f3KXZZyc8L3lxVLXoMZ4OD+xCXngwC4+D9q1QtkrI4HN0WBt65jzUZUz3Ap0vNiRG4Enaje9F9vIcKYS4oS8AU1N2Su\\\\\\\/NGuhrB\\\\\\\/mX2LlxUFGRhyBDSd4qoGBuMQAQwr4Do3PITxqImZ2VoZrsB3h6h4a1vYA3U98CS6AULxqXgmEhBJuVo9jhAoGpg6OwUC7GdmNBYiBcA05mvFtrrrvbMLdeAKQiujbuqlpjFgYO6e7PcfC08Ozp0hON0xpx19um0MS\\\\\\\/p3YJSgaAORrJM6zYDfgK+UpVjJe+TQI8SomxbgkL6k2jN5bukME0S8ySy36uW+roWJ6wWk51\\\\\\\/nN0DKHG5uS17ZQ\\\\\\\/NOJ867Px5X3LZcxCDazz2SlVRPdSOoL6YB4eblZICITefADWQ07ikKpU6jg63YLQBAbE0e3jFlRiXo7784ZOI+5wVk+n\\\\\\\/A2CWOJ9d+vq7rSByntMoNBCEEeWpx1bJ3houVgflC8h2Gj3fSISZGrEKs4j1dffRqmnZbRAJ3havCygZ2dfZHvvAv\\\\\\\/avC3fflGs4PJGtI0yfDAjHJwRG20bLkAek0xyPlMRhAyBc1snIpD+EhJ4G0OtQHvnwdHaIOalKjUhAzi+2Tsm8doBozg0Yrz6defkmzrNRthUBUIZFuA\\\\\\\/PHL0y+wlawIJoW1rK4A+idAlKa1ZuWbgF5SMUlibcMWtn5YipWI9JDZIZuCHJFkN3AHuyH2ELYOaGZYs4CvQq1M5uU2BgocScUBdqlu5CikGhwpFJ7kUtJx1snXT0E4PY4iwHa4LntdpqNOdPmTOdjTYCWYx0QVjn9F6xwPPg41yzsxvEcBRCI0tsQdeN4ITOzDV1PuF2QSz3S64mWUsT2ubEa7VJAmNCsj\\\\\\\/RZfKLJhbgEK4x7+dr9G8v0jlbSxy25EajYNYxXM85dpe+2q0pRX03a9ryjwAPXv+C6TvhF8\\\\\\\/tKgs+jvIroMGEXi\\\\\\\/mdDfjrdzYYpLs76owk\\\\\\\/ph08XlFkLpC1ykSduTL+It3SAm8dmtlpvJYTnvXGb\\\\\\\/bVrwbhsRPVOuUtLEydfBsq4IyIVfjqQqU6XfiKzM9It\\\\\\\/+rUna8i0wzUjrMkOtB\\\\\\\/BQPA+P5Pi9j0v0ieDhiUzT4vFwmc\\\\\\\/WKPbrESOpsYgJRt+aG5yNqc78FpbpQD8pIJC\\\\\\\/YRPyIHlZ7h6RwKoTnaj+YcCjqfp+x52Ux10a3EcXvMt59EEKkAnK7\\\\\\\/LlrpVUsgSzfC1rg\\\\\\\/BdnMyeGE4hYiVr+xiuWEvWSz7XX\\\\\\\/BfIAv5wNCHRWiqLak8bzBW8BNGvE9tgq+9lCX7uqEjNDOh20TLlpnRD0yez1psgPsgKsSYxIKxyX\\\\\\\/OO5C9IJTTPu4TFX4+We2MQ\\\\\\\/\\\\\\\/kb2BVhNyvPBNsmq5Z5lguLbjLlHQSZ6WWw9Ue3jU0fIqW8\\\\\\\/EsR19N1sHPoMIsThhAg5XdT9K3wbznhmJPQEyXgNqXzaOdMHvEfh84I5varkBRYkFmQgHKhQWGL1cJVki7RWqn4J2P+icOcovDm7pbHNKvYIs53jc9a6BdGf+KpqalGcrwUwzcE5HdWQLCFA0h7bQD9fYjOPEqtCY60SxU88Et9TnY2biL7P17odaNmfcJ95b6qCHpRSAQkKPa8Sg3BIUdPaDD3ExqXTHc98mBjwH5zMoqIUGzh+gNLxZ\\\\\\\/PYwnXrYXTcYh9dO1qX4Xo6eSVt1WZPawBrlMXVe+33afWmJE0ZG4ZqKqtzJC8CnO+cmeUgK3bzrZ0L24zve2\\\\\\\/rhFz3S9ZmlFrnMawqnO8emc+K8\\\\\\\/lM91Sb0UAb7ZGqGjKHvv1qMsdKSyCN\\\\\\\/ZyFRTo+MTt4zvHuBkYDEOnELu51BRla5DHmqrJSkbc4dRyi25TlmfxUdejf\\\\\\\/cIMth\\\\\\\/9i+kMJWdZ6s0nb4\\\\\\\/lBfbH\\\\\\\/pFw70VlEwioYzyGkc3y1BQiswPp9lpTTDIpD+0nVXs3RsMo0KLZoiVLIoM8yLgaVS0Kk74MFWLOIAiKYqP9jjRPXXQsETPwc\\\\\\\/N2W\\\\\\\/\\\\\\\/sVAvJXLsGEa\\\\\\\/NOFJ4ZZPeDJlv2C4vQeyeZ8xkK8uqXq\\\\\\\/dqJSrttjHAsPs+eOqv3PBQFkEibk7Y79PYIaDVUPAUrV4JGyr26nCmU5EX4Kp50GlJiuHq11FktXO+Krw+Xb99L2p150h95jvqMgIcji5hoFZ\\\\\\\/RQe0G13F3btfwoVv2XSTNY3mMOg1ofuSy3uxFrPYRkhZl5qATVd+A+bMp0zZ1dtXHj67+iPbs5+IGpGfbN6KrlPFGwtJH6wjijSnJGX8kveslx\\\\\\\/GcETK\\\\\\\/S7i+7MeZBJt1S90Dp4X4E+YbHHcjdiwBgktRGc8jjODocI8qd++U+K0tU8GLw4oZ22OHNVZX78Gzj\\\\\\\/02hSdtndfO7VP+DrsCN0auLemC75ugz+xgSyoHMcx2oprP6p4keTj2WNY5\\\\\\\/wxhHMywZe52gTSwmNTM7hYf0M+Gw5RMXcswcPrEBq3ydYivbLpnz0MPndklF2V\\\\\\\/LoBQ8s+7WNoSpfQmu0vdCyU9qqD0tk4nwBXFf4eIgnYPlaYwtPGe3I++akjQzBdNKOFU\\\\\\\/GWiClc7Gru\\\\\\\/DDVNIq9MYmtewKVtIalm+9X1cPW+xTe2MtxEck9D+Ljd7lEdLUup9OiVAyRZdlL8Y6gRQTWkr7zs7Zn48Mo9yD1qrraE+Rq577KlmBcADLAkxLY8oBKf3Qry\\\\\\\/Ae3zIZQVKbGitX5PaHZ2\\\\\\\/Z1g2UKfxN7yOr0KpoGLClnKju\\\\\\\/HSNwoGlXhhpO4LNKe4O7iceaGNCw+R6uuBeK786W+xysFl2ONuUmxjVnYs9vMdPXlP2zf9hqO6o\\\\\\\/NzwMKWAcpBZeA0lxJhXmm+KojwNdOUwwTQnWsXBGNaID93WhAh+3MTuY4AZX\\\\\\\/qV0KKiPNlytSY5\\\\\\\/y\\\\\\\/UN8AK1iIkt9AxiFFZs6QD5Bd9978QtiGXTwjzV7BsTQvXQxjWOYcT9MTIhctZSwyWfrIK8xHJo0XXhY+sXGNko5kSRjCVoFgd3O5\\\\\\\/TAojuydn3RhKSf\\\\\\\/gEHpg8YPUrcfrBL37E5g3hnpMmFfMRjX\\\\\\\/mk3tUiQY0n7FGWQA0rZWJpFmX4PHoxmCmH4RUnMZaag6LjKJo7IhUGA3c0QhG\\\\\\\/3xi3mbz6yJ6nrSnFAzl17FGC6fRLLWs6SkVnmjQEJENpEu69Jml\\\\\\\/3ZGyNLpQxRF6KQ++LRecFrOSiwOLLCIomVCP0zNsinEoEHGajMxh\\\\\\\/FCw13wJoNIV3iXSRwjBtRv8iYGPn+4tQyVKmNz2fNxFmzAF8Nqhy34HN6qN42kfY2Py2vx7VNZ48mbr6UfYqwOHE9E14ehzuGJiXVyky9g0ep+1wHnXNqdU7lCref30Q5kICaMlqyR8VlhY4B9GKndOcbLaicWX1YkZvSDVlKGQuF+ou5jpCNsrMt52kS\\\\\\\/pF3wR6zVAwKOBz0RW3EMfyvon3gyhOjduEe5Lgoi8jyIg2OXIJfYGf2Db268h0pvgojFIjNAs4o1dOgikvqJv+VzIwnuFkaupfnZhi5D2a5EILg7+VzpyPFY+Ck69sN3ltZts4QJnvDfw6ZWI5IaBE3QZsZ3CG8Zk0UDJyxz2+bUhGkHozZjf7vNsb3Qder\\\\\\\/rxMQ4czlkPfKiteZ5yO+kcaLj\\\\\\\/KEv11sKHd7aC8tpuFlWTgFQiR6GyO\\\\\\\/q8Cvf1357V31Fcnuk5ai7ZKD5ZjE\\\\\\\/QZyFG9li13Ohd2zPFIXIB0kUMJyKmG+FaE6g2m4iYKBvRIHjjY1pNI+HD5g5h+belm+cKvssRtyshsYmRIldwT9un0fYaZur1qea9DrOGSy262nrZfwG0YWlyB+24n4bkmwnlONiBWLgd1rPRdWcdXuYXoHmirKyEip8\\\\\\\/WpQ2vQUl3K+I2kF94ofQUmZ2Dz22A9OpGXKXNODweFiKZSDOmzklZAPpUQ0yuor\\\\\\\/A4jmOcwEOznpItj6zmMElwcZWTFxRFUmDaPUhOIugMeiaYsq8T49iZFFbymu20fTTMU336r5e15yhs3csipd8BpU4aflQ3fixfwGxNAnji3DvC2tWixGyk36YmgmKNNtzJOOrmzfaSNmhdHvthvz12bOcm84zD5AYNheLUNRsm4kIEVmJtjD964K1ECKNb27BM5ytBOej2jDuEg3w\\\\\\\/5BHvZQzfK8kcsBJrS6FdAlIj5QIPaOEutGRITxvBDvR7JzMj2wXAsNjHYRIyJu0N+tjl7cihMiABZ6iZcSkD4eD9CBG9D4A4eUZDEqbKalBDAqI5pjXcEwxaxVWouNklaTuTtPokh4dwmdLCbsLHICnmfIJfjpOW5tqkZiFc5FxwUqD9D68ydEX4d1Cgn6uQCLTFtquJ32KnlnZ42JQGL8Y\\\\\\\/q+NFjMIJbd+jrxNNuteH3nA4y1x6Bxr71sSinSl604oVESUang62fimZPS2zqOUGPGuzqMgNvDjgOdkLb5ikihaN7v0mRpqzFiViBLw\\\\\\\/M\\\\\\\/is6XAPmWaIepSoCLyUznW3JZoJYWhaZe1w5+YwJ19wNQfwkT8JtzAGzf6+sezEf\\\\\\\/3qc07Ok9PLERmC1E8cEV5GvBXZ6Tk1+QiJEp5B2swISi\\\\\\\/apU9SQDkE4fr+ydVxTXdUEG+s7LqdTwlmU7UbAvLd3IaDd9jF3awKGR8NmpqzwwHawFa7JDoj5Ncu2lEMwbidEUp6wvex39\\\\\\\/oqb87b93HV\\\\\\\/khQ9ocCmawCy2oAuOs\\\\\\\/ARbTF3ZHFSGlJCxoyhg62qrHI\\\\\\\/R6RBFjDoZul1SvhYtDqUB2Hh2b1wWCrLK\\\\\\\/+0Um51+Xl1kialnsznRxmoYT766uI9MY7fdUj6IHbjstkGIqGzpUpXaugBVjrpFq+eMxTcvZu56guplmY+9g6IqDhHlh49ncdfIyGtL1WwwuILEBVu9DcR8sdlwFt1VIdjaZb2U8F5U+irIxVcfJvMbHkSAjDb\\\\\\\/tkPip1vUvrNgNHcpwDRnATflbHUExR8Fym0ClWJW+zkyuxV1vtDaqN8YSHRbvf96cTrYfIHFI8zkaZHxaCSQ8\\\\\\\/1DgCPWKIMwLTkldV+1iBPWdQ3Sk85akgj5gTm1MLmHNEnYFHCCInxey9nIYPaYX8GHhAiuN2AxNcolBAToA9mJd+K7PBck0ULzELXgBBEi2kBQ6fWmy15\\\\\\\/EkZFSagZYOpze3P7\\\\\\\/q+V1vN4LbehXYQy7DR2EzOLoUdxBjTRcnxN\\\\\\\/JFpAYMFssil\\\\\\\/Jfk39hjKhdQDI2i\\\\\\\/HJTxq2oVkm5131MKmvtmMcIcYOToNuKxi5xj9N+MK0+\\\\\\\/OrLWTL8NuFoFeuP1imDQIo2+w8q3h6ylDJzbp1+sktpafJDTvwyGdOFUGFz5xZ\\\\\\\/HulYykmtwRHDv\\\\\\\/7phTR3w3tPRGVjMczQLQG7BpUEasYVjXlow8wG6Y3brvgvtQ\\\\\\\/GsQziUtmeOzscl3v0oT91Ks0A3hoaLUGMAKq9dzOgxes\\\\\\\/n33cU2zgal\\\\\\\/iHQWKQrx4DoRoOnc3GoY4u5M\\\\\\\/AMLBvy8N8EspWEQqPfx6VhBsHt2CxYuIq\\\\\\\/H00OiE\\\\\\\/5L7Ooctwt8FZo8ozguOUE\\\\\\\/rRHiWhQdbl+Lr2tp8eyGbgF5vTfPGkW1si8aQNEv3MMjKFztKYZr9yb5R8cgm1YeNnjP28Jm+qSB1mYoNcCIco4wj+etBtX1GM+GcDyp7sMcnl415\\\\\\\/JL6ys\\\\\\\/WHMMPjlrPb2L5CLy+pNwgVyJXxBgqmBmCPUo3zZFeqpCCLaI7tXR3FUe5E\\\\\\\/77Qjc+WHZCXWKaj+WsFrbXG3MPbzO+fgXuaxuoc21zy8JOSnx4\\\\\\\/fqcdbzM2siBuagELItO9\\\\\\\/xYdnVMZHV\\\\\\\/aiTs2L0svCVvSkdHr4xGba6FfK2\\\\\\\/wdPZYtVUw0J1Bah8l2CH\\\\\\\/8dwdWlYT62nx7zMruIvrpwVOkaB8MeCyp+byAFDB6tYE2JRv2Z4M33gl5bTOTTuwpC2fOdId0oEQ0F3E1ZRN\\\\\\\/ToLTHcCklfGYQ7yMtaJ22m6AOKGs0tUhukNUS2bkcsK\\\\\\\/+LnLKbPsOpg66uQvyuN+x3Deufs93\\\\\\\/pvmQlWJwKPmu5fThdMtWpiI7zExYSufj5y2bn9SecpOueWgWtFiLueEg6bHDEVRFC659+CyJ81n9P2QwInQkErMqVehVpn4WcCTkT+M9OsO3iyuYtXdZS5qeAAMDJ7zruo4UefbPnp4b3xesvoWBI0sLa1Rv1P9ETdpuZpeBd8+SYjM31ZXdUHnJnLzXdf76EyFQbBq3Wmudl+nXSSxEbMZo9fsQYog5+VCcYRpKnrlZmrJkD+\\\\\\\/crLc3rxFGQr3YRLAQUuq7StuJ10\\\\\\\/8eKzibmgjl8l+uESRT4OoI5vslcnpuNSYx8pPmiA9l7wL88y3n2omxuULUweT8XfRIVaUnLuCI\\\\\\\/0qt2\\\\\\\/knGPkprsTD2rYjqoSpexH0DTAH8dP2mMKWsJ48qejDFYWQnCIZHfca+1TsN2PsGYZdmPkn6weiw\\\\\\\/oLeOXX62SdIUylRnFnv4Yuvn\\\\\\\/wjnVFTstoAPWZN5RV4MM8NW8gWaOUMkxAqZHckhmmhrMandQRCsIwzZQVQY+ebD365AEmZYKfa\\\\\\\/G2jqW8UzM2TBUinDlsUIsQQOQJyphwc6ab0B7a1M4jd9cVjhIVkVndt\\\\\\\/5dpuXx6K0F9rUm+gjKlNT8VULSqVnMj6nAUqaijDizn5REFAWskhyRB0xigEZSZcba6SUcfsAiyS7vyZd\\\\\\\/WeA\\\\\\\/qQcB96LiL4IgD+QA6i0RJQ45Xq5CGrhjdXa44FA\\\\\\\/u\\\\\\\/deKmFNGxtul88iYVZx9kspQedlD8H7vK4EpKvmQEHQUPAInL5qrXvaipLibf4XDmEEBS90R59MxVfK+ZzYzp08a0nGgqJ3v7q2tKqyhLEicThbjzGNFRiVtcivTVGVgkXONPn1ZfxZoVG17RnQJDG5z8CQFDQmi0uRA9NNZz8x64yLeOv9Oqoc8xzmMISXvXb6WVBSk1Egn2dneHaN5b0sT1\\\\\\\/MWck6YsONXxHp1qAZZJfBSwKl\\\\\\\/zN9c4Y55ELhMlDgCWaUqu2qwrMh7PY7xgRfp+rU+3677A\\\\\\\/dh7fKcIiEaynkd\\\\\\\/NJXn+\\\\\\\/Kp8\\\\\\\/ZiIJTJlQ5Mw6wQukP14bKVN5Ma6zdLXRWME2FzIJ2UMhdMgIhn04u2u72964NSX+gSWwvEPZls7PZnlFMWyXyH+iMxOiLnls1XP2XZUFpB3e1QVFU3LU5WzPlvey9odHpTgbWtRK4GpMfRJzJhiU6QEOBZooluKColkTzI6X1r8i4g+IuCHUvWmg5srN0BuCBeTh9yV6+IA64JGr5BLKnPHb55tg3a3wUE3IZKP+DQtOFP88mdOAbUBCDZ5VgXv4cKSDllUJItILEO0v6UbxMsr9Sgh0QAHiTB5\\\\\\\/qB4KXyD7gULkiFE++dbYiZUQ10TuoE0QG7NNdRV\\\\\\\/h+bTftD81A3u3z\\\\\\\/2alDFbGxFs6eFCbrGIhg6btVbJX2GwNva3OAKzjbsUwoj+KKN+9u6az9Vk7BMaa62cCg\\\\\\\/Lpsu3vhoBV16L1\\\\\\\/5HigGSotq7u866uYuP2HXJbhFDiv0lJeaAxqzHMUQJaaS2HEJgZnzYFqPVxAFG0+j8AXDSJrEKvZ9FgxsIiXh3SPLPYRwONX2eybVV4nY+dsAkZ7taC1nKerDVM0FV9jVEnNfmsVql3aASfFcT3htBTBukBfAU9gFGZ+ctcszXDKhAlc8gS+DyFha1xroZoP\\\\\\\/u6trlddZ9rtaTvhDS7zAJyBFYwINSKtTgRDupNM60orJ033W7RbBj2TpkIiaSIhR9HP97pngtq\\\\\\\/lPu7DmhevZaVV6orSmfAZLnkHQFD77o1cw4qP4GepALaDb2PWX85S1aUsognEZhx1m50UN0cuqjDZTCSxdQc3NLSu4ViZ4AbsCt1+kBh\\\\\\\/Pp2LrLqcRRjxPwFX07x3mGN6V8PlSrIQmOXWmef\\\\\\\/m2rFiVfpeMuRC9G1a3t0mbd0zwBIxU3PD64vr1OHBNdSk26pZXqz\\\\\\\/j\\\\\\\/g3YaPUTkDChprK+eGpVdgAAPuDwuBQS\\\\\\\/FdpEYyfT18bgPRQdf3tMGvNt0XPuNMWuCnQYYj1N8sOY1p5nFyozAQa0ljAx9kW7mJuXsSB1rCl8InLmJfcvZxuhPFZxKGuvFC9T4H0RyMEHAUokJjsjasMd497mslzih64leAkGd8zk3WJgUyzvYiz4e3Bc5mHfDjZtq09cbFBVl3IBZCxGpIvD3umdgM+fWJkkL8utvp9Qg4SCH2Zcfh7Yrzuqj5njbEjUdVPc8o\\\\\\\/gEIvmMhpOY2K4VH2mTAKMF\\\\\\\/EMISZ7izLe\\\\\\\/i7r2WqOU\\\\\\\/WXIZt4FlszBtXH1gDb5NVZr58HJcTiivDLVs1wGT6AklbyaNQX5g2tZ5XJihpC5lQonnEvLoFtvyvJYYUItaC1zlWkdD6EfZE6RDw9M7f1HC2xl2cLgwJtS21H8Iot\\\\\\\/N+r6Aj5wzRWDAjNAhvPeF1t1i9yeLfPj582uIrE+1Re58n4TThcly\\\\\\\/MAYbhs6flJcHg6oObsTs7n2Ix3gvhcTfGTrQanQl1l5cvkDeZ9b4JXIeiYKI7YhQKst350e\\\\\\\/gfjbUbEY0uTprKSUjrpsmumczmfD6TshHTj\\\\\\\/FKlEzV9BnkeSuCCAKfvqWZAp38tR7ciYestQIKZytT7jP2HAYkdCJuxlpud9+J20fh6+dTrBvVt6n0sbZFWyufmPicWzWqw81XiXEgLKkeepkB9bmx1pZ6T\\\\\\\/KECZQ+NtzU9gXAKnQd4DuuzQDvw9gusO+w9A0laY2qLZcrU+f8+Qw6U5pOqYSwkAG6\\\\\\\/0L8MLuYOWJyBx4+EEMEQvF26vzediM+OlI1W\\\\\\\/\\\\\\\/1b7WmzVqJlpzbY5Qu2jGQ9\\\\\\\/1TOcVwFSALWKkQMFVf046EBpvW4JEdhxVartCWkbL3\\\\\\\/qqt0UaF5D424DJ5OiP+K2bHr6alrkpW6gTLjspvY1kHDkpExHNgDx9wWz+h5pHqVu6zzYjilMtEkUKGFEfIv+DMtCCubNSyi9W7fiQETZ\\\\\\\/fEZS726CHDCnQsiUFTIumNg+NPff1aG4rMuYz\\\\\\\/cSdwDCZj+iM6nOODuOm17G8o2cNl1Dv4j6auR1HWLONweMhlfZ6j5Lwtl9KTzjMiKa2x5tIh4465US5WwM2KXsugDs5mCG27ukFRrM\\\\\\\/I5PqyGb5+RN2fnkzhWdOZB0j\\\\\\\/6YMOCfGvdYWQeJPBgYed6gKtsbmuXerDRizpi14YI0biu0CdKpJGVQM3KcvSQbtaVPiLWkv7gLpxGFtVeYLG2rNHIZ2DcKNvBxUKK0pxpG1qujstnFKI3x19vVCWf2bJch8N5ax+oVPFn7AvElz6Tg42Ek1LwEp0JO4gFDSrS77b2Hq1jMz8W7ZcD0SctU+Bxi5PKtW42OBtcmAwjdb8nYx4AFyAeGBRY4wz1ab8DT2lM5v19skDn+3Hi4RzVhgVQLXMBzG95u9vJGF9mD89BRN\\\\\\\/Lv2ACYbilwTrrHX\\\\\\\/BtTj9ZkRlP8JCLdtLXJ43GUtDmI9jDFDA5A\\\\\\\/4irVl9bPjskdLdJVnesSBhfQzWdbb5lit5\\\\\\\/SgI5TCnYZR\\\\\\\/9c+Qr7JwHP+yclSHk68blVY4XvdKZMLrBNIA27AU0akCl0XQdTzDh7rxhFdL9M7B02QXCsTCS5ZDHasa+4JT\\\\\\\/NMYNx79TTS2KPtvfpHOJ6BPWADdfA5+sTwie5bPrMft+0hcLEe5WU2061WxE3SIA62K3\\\\\\\/eFncQ9wK4ooxosVvhFexfFNuFAVDW\\\\\\\/XYJlhQQ2NJfD1mmZIeq2vANouTXgNLZuUr8Sep36zbcx3MY8BMAlm4rStlbQuiZazIwUADKW7NK0Kb\\\\\\\/9Ow\\\\\\\/oWm0rBm4pL6ECKz2CMBhi6lRVktQf1zZDys2hTr16xsXK\\\\\\\/jTOlb06Cz+y290KZ+J+qCOzcclfAuMAiTyj0I5pnQrImYvSoIa\\\\\\\/Gb687fb+2H1tkhccTTTAriu+OmGwedLSmTFAP9chtIYGWl23RmpDrZAs715MZMaCbzPGintwQBp06af6eY3CwOau8umgR2PPQvX6t2CryUThsyGrhtZgQDDlUh195ZwC6JRO7jK7xxU25CDiBw2AYUEvyLYKG0e+sjymxk8u0Krp66tPpTZL7lJkCKWcgGZD2\\\\\\\/C1L2\\\\\\\/kRXcUNegTTtqi0wrHjTi6Wq4\\\\\\\/AVLN4fIUIokPt0UtzyZEBi\\\\\\\/1BX2vIfybreMLzCTiGEuZ7jtguN940fx25pkWo0jBOB3s242mX4s3KzqZCFakwaHiC4VfEVXMaZGnCCcFFNHivQxUpPwBUEbh5s3INcH55PgoLAOITBp6skJkokLhPjaEGoIxf6CU95LQq5ELJ5JHEXCv3Zn+c7\\\\\\\/Xzm3jtPwtAVTA5H9sIfHn41fPVZzMnAvA\\\\\\\/PMy6\\\\\\\/TYbtUVwmLtBTqac33VBAlJwvMEgP67lgw\\\\\\\/9frgVjm4pKaNPcHtC+DDqu1SM3RXyZ8lGEMSveeabvqg2uvX8cvNz99RZDxe1WgxHa38orbGnwfruz1A5pzQzVlronAZt1wru6v+VwGvF9wrzH5+rNwG0FAO1pqhTnmjW6YgGiYng8bleCvdYTgJvpP\\\\\\\/5XTIZR4E3RNzJwrJE5fEvZiMJvosOeCtEZ4WzJQLptm8Wv0q\\\\\\\/G\\\\\\\/YQs3VStT0wouiN8plgs93sLyh6B5OcZ0dj6C1A3p5O5LUj6atP8Z4Kg37hlU9oW\\\\\\\/ny8D0mZMDLXmqHtx7MiWlBPU+KWImnfcZ9CnfyS7VfHCBPm4YkGPzjs3d0rC0LFAVi3ijWoCHpOm6hCSbF6JljGHpGQ2mDQmGHW\\\\\\\/J6r5BA6LYJ0iXzIFPs\\\\\\\/zBx5FMQ7K2DcZt8ZHKVy23YYzeaVDD625cB\\\\\\\/yy\\\\\\\/WwyIqmFRuKciqYrvUQPoW\\\\\\\/TfJS7TYf1+kp3hmOC+V+YrxPpu1vU7GXTiEzvW+PLsmu70fUmwu4TdhZe9fdIxC0hv3DvlpyDKGqwKJXLD8yBEsoNHDYVhd0ytGUbrVLsiM3AIwCN0Y2aOJLT6sYClrW1Cw7WGbpSclRi3fEPZkiFXwv1jqldQ5T6mVPNtQqWbjttjXMCty+pXvbHhXQJKVKcJJBPE14GR3jq6Yjfad6da4N+qx8IGKNnarzC3Jw+trVjstrJdpsYXDCk6EoQazYsSJWo3p1PZWNopEGkVDBJ8TyhV6BBE0qdCfdTjsJHt30gReoDUaVkJxzelVPDiA21GE1xS6dOJ6FQvFzlQ9EcOA4pxtuaYvTJPi\\\\\\\/Z6vP3tEgnEHM8BEg06W4KErOsePTLGf\\\\\\\/5P+UnG7eJu3FJnGF+9pkA4ssF94a6BsC65aKCb6pjAEc0tRjXXLg8hnMNIcNK5gzaqKNtoA2q1ntKRHT+rnL5TW+Gc+2jM0W1+17SW0cJ\\\\\\\/uBrbuFBfVTShrd+UMh\\\\\\\/T3J0LVLzum8w\\\\\\\/MrEqDBbTy82wN7No+QwZshafAyxzxMrJBTJCannNZHgV8vdAomRIEDSXrPmxeY1Fg0Tn5M4pSW37w+VnyKuadioh1232PyYjLzplfbFIV1dMVY6wrZEUup9K1P82FiSwHqJF3yUKDAtJ5eAn2ZvS9yRnjXdFCNNSrZHW0cq3ZFjArgOQqm3KwxJCw2zq2hAogtw+AA0yIMzy2hRg5dlvlswnXyb29uMJNdDMP6ar2u1nl3U4f+hpFl3Vpvct+\\\\\\\/UoNDmLgoLOSS1RpyrJWml6ajQoY8UebFuIGE\\\\\\\/Zzg642Z0kPLqKPcbdnijLofg5PeZo\\\\\\\/Z42jlPvEqVeGQni8HyhiQwRm1Wjacumrw25X9yAlY6ms9A1dMiyjncXIf9Q\\\\\\\/uSy0GQzjSu2iGT61swqpsrQ8KF7Bt+566Vv62o0vTKUM1fKWY9Rrqd+tX3guc1I4l3vSAZiTVem+MiZWB0xZEXfVKhrtWJEQbtxBbteKqO+IQr6AzlRK+BQIVnpIcVheTY4q8qlN+YForDZEOVjMTogmNBGKRXHv0KuvaOvr+sVinTvFZLEkPofe7u1Ztco1\\\\\\\/blluzpSuKcEq7DbcSdnthVxYBLh2soJqMbZQlaWxyOisNFdriiq3LzdmJ2KQLrMVeNnj5EttJZYvnjUofmq+L5XCckq7kPdHIqHMOHOhV8TatVDjo\\\\\\\/LvS4jL2laaHJaM1fhjerVz8UOXwN4pJnLzA1cTGK7dF1XzfJQrRhpZoL2ewMphA8M\\\\\\\/WbK5\\\\\\\/69fpIeTPLwpuWHnx0qvFmXdVyMZCfNyGVkJ1\\\\\\\/b0B4FUH9VrZ3lMvqTrVzRf9V+TjRFa3CyrHHpfr9gJ92iyqlXn1FuBwMty8pbMKSuQoNCcsX9cQRAP2T9BB\\\\\\\/u3CcwVkS8w7nUpy+SS2fhrBcb7ov0jUk6NMb2B3Eugzkcbfd4Gw6qdVMFKgA4RwOauj8HxHSi31mr4V1Lw7ylqoghJKAV0HxsN6cdqDX+WAQihVQES0DSwsUf6YSEyALpgspyOex6Qj1D3LDLK3KeGQXxwVmCp\\\\\\\/6q6KP2U\\\\\\\/oW7+xswWCd7cL1zWlR+bj0Hjm0JfQzI08aTCCosKLzAACqJ2qYjkbv+YUKWX2ZcxergNP2xlDlmcPS4LcGQz6mdnEycswR5cjLXfAOuOk3TeF1migP4bsgx0PPb4cf45ARjq\\\\\\\/DbnP9JYDIFPc+3L\\\\\\\/Mtiw6p5RDsHJady7qqtJYDwjxbCCc0ov+F8Dx+q3jRv2JTBYr0OpMpK\\\\\\\/+eE1wl8vCY8lsiWiRUQJp8hbMF8KXdYdOmt\\\\\\\/8d+VSc0T9zFORkpByP9LzDMk16pasjBgw0E73JFExPDBVlFlWqieqLQpga+xxToGZ8\\\\\\\/pUlqsgntNdAledmlWwPmb4S5QLqDae3h0GNfJwY4Olzt7luGw2KCzIfqnZRIfL0uWQTUnwk6F\\\\\\\/M81FxFgZc1UJM318y5eL9TwjPyJqNogGyUisXNKiL4RQJB5wtsjry3RC\\\\\\\/qPz3kwWIjRFs6ojibku4rlN+8Wv3P4IIUoj5VT4WKKFJG9y+3uPCRi28Rrb+zWHxrDWqQrEEREyYl9SBba0evtPpeh0ueZO72NCtdOYFwTT6a7FrYvJnAWwKhvUVjritpi4hPgRz3iZWZqOf0johXIGSIWublUDC5uz4JDCF876UPInmNfRltvNjGrINTZzd\\\\\\\/2YuqBzcAwkVcuGhIZIWB9FuNn6TVvAbsHUaXXxK9c7D\\\\\\\/LNYDdAnPdeNqNaHXdEuVvVI0bdCqV7yI7+lRsTHx1oqJGG4MDhZtOcJNtQBFI\\\\\\\/9raWOfJTD4m25UMhSxFUsPQ6ebpvhW4x\\\\\\\/V4\\\\\\\/mFMofqRTZ2ZbAFAqMNebKy+CSnS7fCjj+5XHN9STimYs2vMepmxyJV0gwbkT4gP3lDNwTGvmXPlbw4Z6s+EOuev5Gl8xGsHEVyLG9Ckfc22WEz1kHCYc7xGQ1grYgwiZMZj4Tku8odFOK9kd4VETsQg1a5Rkoo4HqKcQu\\\\\\\/ZvNQR\\\\\\\/p9qPc+xNnaniHppcCNyoQb3kz0Tqo2Othl5XjHHiC1PFJ9ol0t8dfYqMLG+1nsYEutSYPmWdjVQzl0Tq5ox5tHljtPGK4EGI\\\\\\\/PPW0EXmv3sSdKGW3HLlfKAcRVjz27B85wJJaG0Dric1w4r4VrBEPCOo8Zc6JfSC2r2PiKpcv3BFImqhfKBTT07UPUttODZgC1xXlD3Q\\\\\\\/ntUcBwt56NG9\\\\\\\/VX+NIGiQLozCViBR9stna\\\\\\\/SQhwzuD5BK0sWLB2GuzC3QniW3eljeXJDSfDM3ws1f2hUTraALve130KMusqj3xHxFFCB8Zf23CbZlPzEIcJrAY546qCnD6L3C9xV8IQ0BkUTVOX6fG64QmV\\\\\\\/ie3GsEkSfluGnMp5ZHUvuFiUbWBQ3eR9NJcqmw9QEfxo\\\\\\\/tmtxNDXrYPhfcChd92WLk61+eVntKbPJ5ORgXd6TDiFR85M8lE2w3U6Ig5vNPAIl51qKgWwAqxvA3uwAZtIxQyLIGzKP7ei\\\\\\\/eNGg6dkXkCrGo7BhE870DcJO8h93Nw1aia39zbSu0Do2pX6t8NGQxT3xpihlZC5xmfNu0xOUhB67lUKNz9t5PLabXgTAfml0pA\\\\\\\/xHuZHn\\\\\\\/UZMdnR6yGjKMkpUT4oApU6UzUo8Tp4kzLbWbnPeqsVPTZdI9\\\\\\\/j02JQIvvWnv0EvdMtsIlb7gmffcDyl6EwgBc3\\\\\\\/fK8chigLVx0qwWCMovZR3tcu6kaoKlEmq2RbvZFCxM1a+aHdRcAhrpAUA5tOnHJgK1gVvtrMJ1bVvjq73hOSgDIZwj+0lko6NijavS8b9+\\\\\\\/9j3TiClatDY2yaZS2pZ0MnMhuw\\\\\\\/UeIxi41pRAT6TFfI4OWcBZJPeugZLuxWu8ljD9PfwBg3VEsFnYLGp4T9HD3FjkQMmnFmIDOFANhsnmV8ruHPFksYTDUmZ367PHWR7NvTW0nb\\\\\\\/WqVsXiAmrm5o6S7pQ\\\\\\\/dxM6crjY993Vx1Pq\\\\\\\/nkXVHmpE0kMsOo5y+ek2Q5osZw41mo\\\\\\\/mZxVTay2UsIWq5mA7euoBPLGfp5U3w+H8dEmEmcfXaQtmZV5Jg+yuQwZgYJRHuDdRMZadZkL8+bH3APFpHPbr+mEoqfbGw1chF08cLYeumfPWVqCifEFFLWXY\\\\\\\/5sihg48xM7PXecRBe2IRg8x12FX48WwsCTQqZlYhsdPUN4lptiyr3EudahnqrkDFxE2qKLmRX5bcHaWZztu+NviyruIw+m3gNuLwnu0FHQ\\\\\\\/HDk6BCuhjHQBmCSI63UyzkRJ7M7ZSgqzK56jXes\\\\\\\/pKTW9zyBeFuO24O8z8x1nFDKFGvfMyOdKBmYzt76jxZxs\\\\\\\/7dMsQo3oShDNtrOFt61tIOfWev4mruX9g4NJZ1zagbkCsHYUQbGIZnc0ij5WRs\\\\\\\/Z\\\\\\\/1DayrvzB9e9BNMrNIauCMyHQ2iekH7\\\\\\\/S\\\\\\\/pY5TjRbC3uecHLhA6pbQ0M22K4PXpIZysANde9fbWlAWRoOfJu7Mr6U1t72D9u8j49ZLQmkzpb2Pxm7wuuZj2yYXDlk+tOyjuJ4KPAUmkfXKAwaqS3Q\\\\\\\/zkflRLMpoHXZYN+wC4zTPN9dhYF+toHulBM43k1s7uLWWMsrRXyiltOVIE4\\\\\\\/o44P7p+dk3xUl7f4WSPsWoFcfgWL5wn3sMqbg1HYtPizci3L2oJvQZpIfhNnB\\\\\\\/e+QUR5C7f1qViL1DgtC1xFaJOATEx85XvR6ediMAIlMu1\\\\\\\/dlh+hGKp9yUyl0yVYYZIfnMAve6OJuN7fwUKiLWV2S8AKHssPutJvpmK9uQqqG9zs1WdG78gnZHJ4XNHCc4fQ+idq+Ubo5+BEzsSHasxxI03YMO7qAnRzHx7n6SZFRfDLeeyF4QWTst+6xpP2csDutIYrTuyO3vWmFrNzVfrVpPyX\\\\\\\/KPHkiTf0wPkpM2HjSxdsHGvxu9ri9uLTOLgctwYzf\\\\\\\/bdKtmTAnRV1KazEYs9TLv+AWVANRXEe\\\\\\\/jcaAQEe+Ej18xwzkm2cHcWH++O15gPRNcDFmR4FCXg2ZRRgKOHh5f9pv0XvTSl8bsvfo+cTcFuQcRgBDmRHURjp7qefkFxxIa\\\\\\\/mvwzXONZHAV8iSKOpCyM\\\\\\\/pQkNm7pTEYL48pPSUzIWe459tqcXTcNJMIGyQ\\\\\\\/1MeQbagELfePKDLgc6UG8U+DSQu\\\\\\\/vOU\\\\\\\/zdr\\\\\\\/ohTwP9HDEquppVVc\\\\\\\/yL1DkSr7VjiXlR2oSwVwMeo\\\\\\\/l4MSy5j+q5j8PWbV\\\\\\\/uzrP8MB10rpxRW4NVjASD6rFkEylAj5vaKlxin8Mz+YBXK4iS\\\\\\\/Vfu9B0TvRksa3aNZxJNkQANF7VvvGL6PJEGDf5aplw60HGSQWH0\\\\\\\/\\\\\\\/a68Q9EZFzbPMBPcftxi6mdAHguxBixW1t3fxPO\\\\\\\/Sn9csfDyi0zmexS3C6K\\\\\\\/ZYj5jT1SUrq7uDwtbWDyJC++fKc9sz++9EyLUs6hcdM+sgiJLQgGaOnV+DwNPp8W\\\\\\\/kMVo2QmLTy3pZ1iRUBm62DYxr9jzZmlFSf\\\\\\\/iQxsNK2wltPhzy9O+235UMczkgM6g5wvbsHKbB0\\\\\\\/iiTqmlgkxveZGGm\\\\\\\/KbC8C\\\\\\\/YTrgn2PpDz0RAh5MtQrBQBebMoBN17K4eAg\\\\\\\/MK8\\\\\\\/Ti+Pxj5gPOvlVwseKNLZMw1vpH4M02acdBYHpn\\\\\\\/XPCp8sYZHIc\\\\\\\/KloiMKg4h9\\\\\\\/YZpVYl0hK6w5nmuD\\\\\\\/oBpfVuCW9LGHikbcvCXDIHk8nBmn41u4ZaGYKKRaznXntI54wgkmXDi8tP6HWB92jlsQaWLYyYn2fkzTWYkqgsjeExuoxvdLiapA2B5q6gQwbk+rK6++RQuN75o1rbGWgKYOnAn6y845hDpvm1v4DMME7Ko5SlQQAb4PrbMvnKDN5kjPTZqEaoE55YTC8GlN0g4xoKW1mOG18gJm4Eqz7QcSHNcT\\\\\\\/6ywPpbuNX78h496kNj4R42JHBONQWNjAkQgObhQqJpDLkqfBOENgwPXY2j\\\\\\\/9yvT+04bIYGEzE+5hv9yxjQ5kVfghbRfl+XmgglZ\\\\\\\/AmkS7UfJ6qN6fkctvxOSt3fCBYt7SMsVlyQcERs\\\\\\\/mRThGbnY71KIb82TkTZp3jRn\\\\\\\/fQQP7hyP3Ka672S4k8WYhZTk5TohFzHLAp1zWXDpo2sLgXoDWBm7O7hGhf3CVP96S4KBtaLYkTOnzRg==\\\u0022,\\\u0022iv\\\u0022:\\\u00224ytep8\\\\\\\/5qgPeWyveDlCawA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022108b5a61f4c620dcb6cb4317b1a69d8950f1445efbf02d407312a0d46ecbbfba\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HhN+pHKxuxdY\\\\\\\/2oYuOV1qakAxpUU3DzOpPz4A3aVV6GEmDAFVwR32uEz8AgUhobxhk+GP9jNu84yLFjSfX5ZO3skVbKNRDbQNMhf\\\\\\\/S49j4IAPyRaBOchSSLv54lVQIMQAwlVf0bKdmTeoZAZ0ltL3wNU135mU1S5tjvdz8\\\\\\\/hN0Yw\\\\\\\/4aGsbz7IQG7o\\\\\\\/VRv0lvIkgq9vRMCGiDvrzvQ\\\\\\\/KDSrEazS27AH+5hWJW6YY3anwLJ+h1VGsoHV\\\\\\\/FtXYd9841GeNLCotGtlRMoJFy+xn3ZF5GPCoPiq0LIHSK1p+6mFBSLGmoJvWqETzYH4fpGVXgyBUG9YR2OorX7uRboa6mMIaHhqA6hF+lcbGLQ2qAkzquLfSJNTrVWFbLz2Kwk27+ZlteYJ+Qht5AC69mstccrbZLzhYVjJGEOEfTZnNsXqCEb6NnOvIQsnAdH4ajKjq5JhfpTOJlgbnUP1FqRsH9J3KqGh24djLXFVABJVeoXp2EeVQuptiKGDIzn3keCq5VPCRqS\\\\\\\/D\\\\\\\/3wXD\\\\\\\/Z1LZrjj53thSm+guhaAWAXzsG0kk4RsUYNCjRqTXZtb24VZUXoj2AeMPEXw9yluh8XPoXRaF\\\\\\\/CwVa5HByi0viRcdNegcZiDoEKacppM67z8uC3TgpGtapwnpv+eH1oIea5XoXs0CYurPz9FLu+BP5J98Bt5A6r1kO6fQ42ThdDRozdzQLQmaTqnfz7vGWBrxo\\\\\\\/qPKIh6CfkL\\\\\\\/k82ripEwVtPZwtc7KeaggJ4MlGH\\\\\\\/2NQV3OHtGqDeOTNmAvjiHK3DrlGMwRvZPFPZE++qmCjX2CDtmclotKBRzLIQgcOqdLj5cTYFE8QNkofHHDKuwTRNaxaot51R0Nr1SvaTR8GemN+eTToYZ0wQ5w2K9BTMWL6E4BdhhiITcTQ4tCU0zRDeGXa5Zxi2TMYaba93cJ6Vw5\\\\\\\/DYMrLiqr2wfzXQy\\\\\\\/2fVOdzHag+TVoIXdX25THnCA4Wpwl8LFBtO12qOdVB1Zu4RQwo+MZuA3ie3PvjJU63YDoIx06RNJCG6dJMhCPZZSIrP0C4JEsbrF7\\\\\\\/uiQlWcTuOcLXGF8MLFhrLSsfMmLg\\\\\\\/uay125F9fpCHm2MEpbzSsTN6utlI5j4NugrKh0dben6Bu1A\\\\\\\/ouvdWi3iFOhOfs7gp8p4rYjWcgpi\\\\\\\/\\\\\\\/D6w8xQ0OICdw1jQab66MDy\\\\\\\/KkRuSS3PgbWJEVqzrtrkZIvgfeIdXx2w+EVW2HN2ZM6m7Jg6+H6x\\\\\\\/X59OMv\\\\\\\/cmzHz6imfOizDeSZyhg40XFrpNSFz0a0y2MwGRFXc0pIv20W9cVtCqX7+9QGpkawUIzsFEwxNNC3YhquMaR6bCdSSiApl7fHO\\\\\\\/oZ8rWAiALHYAXDkpc+hpGKqO55qVozf80+yu+XVF8\\\\\\\/nDLJuCdWnQq49Cop96WGVysUK6LSJI0YY2jMlPlpKPvln+Sc5FUpkb22T005Ps\\\\\\\/czM+E0fnVPHe0Wc+oossGxfkAz30ujeiSulk3BVRj6j622Gpro1XelFkmdlohrK7mk1iBISudZ\\\\\\\/d+Am\\\\\\\/Vmb9+20bJnpmwaPVNrRYf5PVwBfLOFCJ5iKAUYLiTdXkpTgrJDv44u32tR36Kr+84z\\\\\\\/SF+vI9U3R7GULihwdrq5iWUNX7lwYr+SNVpa16dPfJGPnEefttXa\\\\\\\/JGJXgAPkYUjhkF8gCnnayLjaw0nCuRom3WXAb8b3O9\\\\\\\/R8RS302KlRgpX10blZo5wY\\\\\\\/CC9S50\\\\\\\/qStEXg5z9MzgOxWvgfpGjR3+fyWhl523hbaYeRoXPnI7bMeyKV2rDgLx2FQLKGoESOFuFOYEo6z3i7HksZGlciNcE4WvFSls6nOS4Ftig\\\\\\\/136qEccw3IVQRXDH4Ah7ajdDLItAE2YDv5LEiU3U5oy7A5TFHnsJekobrh6Cx4EyrjP7NNDlWoMQf6etLnzWLHGH3kKrMKZifUidKR8JnFiEUhbiCkw64SiUwOiHVVKpE1NXYJuZQpCBVK8RpTBrzoZudYkae4RB5gaIN3fP05vXPu3KvAlpyRQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022i3G0UdqDa8HESgJXh6\\\\\\\/hpQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f8f1ac9d045d0aa8a2fa95cb9343f10ece18b816ece33185449cc3a0946e8814\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495551 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genome-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/455" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-d-syHwz_HFz9FebcU__X086Hdy69xE0xoKIH2JAQoYU" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/455" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-kQGaQrJ1Kz9QfTA81Z7DJZqg1o4tVc6lECBxvfeJdbg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/2/443" title="Characterization of Embryonic Skin Transcriptome in Anser cygnoides at Three Feather Follicles Developmental Stages" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/2/467" title="CRISPR/Cas9-Mediated Integration of Large Transgene into Pig CEP112 Locus" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495551" id="node495551" data-pisa="ggg;10/2/455" data-pisa-master="ggg;g3.119.400913" data-apath="/ggg/10/2/455.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>De Novo</em> Genome Assembly of <em>Populus simonii</em> Further Supports That <em>Populus simonii</em> and <em>Populus trichocarpa</em> Belong to Different Sections</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-2890-9702" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hainan Wu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5937-3467" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dan Yao</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-4530-8653" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yuhua Chen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-5191-828X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Wenguo Yang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-7130-7670" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Wei Zhao</span>, <span class="highwire-citation-author" data-delta="5">Hua Gao</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-9795-211X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Chunfa Tong</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">455-466;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400913 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495551" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hainan Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Forest Genetics & Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hainan%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHainan%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2890-9702" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hainan Wu</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dan Yao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Forest Genetics & Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dan%2BYao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yao%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADan%2BYao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5937-3467" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dan Yao</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yuhua Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Forest Genetics & Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yuhua%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYuhua%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4530-8653" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yuhua Chen</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Wenguo Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Forest Genetics & Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wenguo%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWenguo%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5191-828X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Wenguo Yang</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Wei Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Forest Genetics & Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wei%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWei%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7130-7670" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Wei Zhao</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Hua Gao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Forest Genetics & Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hua%2BGao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gao%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHua%2BGao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Chunfa Tong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Forest Genetics & Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chunfa%2BTong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tong%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChunfa%2BTong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9795-211X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Chunfa Tong</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tongchf@njfu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">tongchf@njfu.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/2/455" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495551" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495551/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/2/455.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495551" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495551/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/2/455.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495551" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495551/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/2/455.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495551" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495551/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2"><em>Populus simonii</em> is an important tree in the genus <em>Populus</em>, widely distributed in the Northern Hemisphere and having a long cultivation history. Although this species has ecologically and economically important values, its genome sequence is currently not available, hindering the development of new varieties with wider adaptive and commercial traits. Here, we report a chromosome-level genome assembly of <em>P. simonii</em> using PacBio long-read sequencing data aided by Illumina paired-end reads and related genetic linkage maps. The assembly is 441.38 Mb in length and contain 686 contigs with a contig N50 of 1.94 Mb. With the linkage maps, 336 contigs were successfully anchored into 19 pseudochromosomes, accounting for 90.2% of the assembled genome size. Genomic integrity assessment showed that 1,347 (97.9%) of the 1,375 genes conserved among all embryophytes can be found in the <em>P. simonii</em> assembly. Genomic repeat analysis revealed that 41.47% of the <em>P. simonii</em> genome is composed of repetitive elements, of which 40.17% contained interspersed repeats. A total of 45,459 genes were predicted from the <em>P. simonii</em> genome sequence and 39,833 (87.6%) of the genes were annotated with one or more related functions. Phylogenetic analysis indicated that <em>P. simonii</em> and <em>Populus trichocarpa</em> should be placed in different sections, contrary to the previous classification according to morphology. The genome assembly not only provides an important genetic resource for the comparative and functional genomics of different <em>Populus</em> species, but also furnishes one of the closest reference sequences for identifying genomic variants in an F<sub>1</sub> hybrid population derived by crossing <em>P. simonii</em> with other <em>Populus</em> species.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genome-assembly" class="hw-term hw-article-keyword hw-article-keyword-genome-assembly" rel="nofollow">genome assembly</a></li><li class="kwd"><a href="/keyword/pacbio-sequencing" class="hw-term hw-article-keyword hw-article-keyword-pacbio-sequencing" rel="nofollow">PacBio sequencing</a></li><li class="kwd"><a href="/keyword/illumina-sequencing" class="hw-term hw-article-keyword hw-article-keyword-illumina-sequencing" rel="nofollow">Illumina sequencing</a></li><li class="kwd"><a href="/keyword/genetic-linkage-maps" class="hw-term hw-article-keyword hw-article-keyword-genetic-linkage-maps" rel="nofollow">genetic linkage maps</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/populus-simonii" class="hw-term hw-article-keyword hw-article-keyword-populus-simonii" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Populus simonii</em></a></li></ul><p id="p-3"><em>Populus simonii</em> (Salicaceae: <em>Populus</em>) is one of the most important native trees in northern China, which is mainly distributed from Qinghai to the east coast and from the Heilongjiang River to the Yangtze River (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Wei <em>et al.</em> 2013</a>). It is a primary tree species in the three northern regions of China (northwest, north, and northeast) and plays an important role in preventing desertification, reducing soil erosion, counteracting wind damage, and fixing sand dunes. Because of its drought resistance, barren tolerance, wide adaptability, strong rooting ability and interspecific cross-compatibility, <em>P. simonii</em> has been regarded by poplar breeders as one of the best parents for breeding poplar clone varieties (<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Zhu <em>et al.</em> 2018</a>). Based on this recognition, we established an F<sub>1</sub> hybrid population by crossing <em>P. deltoides</em> and <em>P. simonii</em> and used this population to construct genetic linkage maps and map quantitative trait loci (QTL) (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Tong <em>et al.</em> 2016</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Mousavi <em>et al.</em> 2016</a>).</p><p id="p-4">The acquisition of genomic sequences provides a solid basis for understanding the biological significance of individual species. In our previous study, paired-end (PE) reads from the Illumina Hiseq platform were used to construct a draft genome sequence of <em>P. simonii</em> as a reference sequence for generating single nucleotide polymorphism (SNP) markers that can be used to construct genetic linkage maps, but the assembly has relatively poor continuity and lacks annotated information on repetitive sequences and functional genes (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Mousavi <em>et al.</em> 2016</a>). In studies of the stress resistance of <em>P. simonii</em> (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chen <em>et al.</em> 2013</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Song <em>et al.</em> 2013</a>), some transcriptional regulators related to drought and cold tolerance were found by analyzing transcriptome sequencing data, but the lack of genomic information for <em>P. simonii</em> made it difficult to further study the molecular mechanism of stress resistance. Although the genome assemblies of <em>Populus deltoides</em> (<a href="http://www.phytozome.net">http://www.phytozome.net</a>), <em>Populus euphratica</em> (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Ma <em>et al.</em> 2013</a>), and <em>Populus trichocarpa</em> (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Tuskan <em>et al.</em> 2006</a>) are available online, there are significant differences in the genomes of different poplar species (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Yang <em>et al.</em> 2017</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Liu <em>et al.</em> 2017</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Ma <em>et al.</em> 2019</a>). Moreover, comparative genomics based on <em>de novo</em> assembly in poplar is still in its infancy. Further development of genomic resources for <em>P. simonii</em> will provide unique opportunities for comparative genomics and will also accelerate understanding of the evolutionary process underlying the phenotypic and adaptive differences exhibited by <em>P. simonii</em>. To improve the understanding of the <em>P. simonii</em> genome, explore its complex stress-resistance characteristics, and further study its genome through genetic linkage analysis and QTL mapping, it is necessary to obtain a genome assembly of <em>P. simonii</em> that possesses high continuity.</p><p id="p-5">To date, more than 200 plant genomes have been sequenced (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Vanburen <em>et al.</em> 2018</a>), including some highly heterozygous plant genomes such as Norway spruce (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Nystedt <em>et al.</em> 2013</a>) and <em>Liriodendron</em> (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chen <em>et al.</em> 2019</a>). Recently, some <em>de novo</em> assemblies of plant genomes such as <em>Ganoderma lucidum</em> (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Liu <em>et al.</em> 2012</a>), <em>Siraitia grosvenorii</em> (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Itkin <em>et al.</em> 2016</a>), and <em>Populus pruinosa</em> (<a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Yang <em>et al.</em> 2017</a>) were obtained using next-genenation sequencing (NGS) technology. Due to the complexity of plant genomes, the draft assembled sequences obtained with NGS platforms generally have poor continuity, and many gaps remain, resulting in the potential loss of a large number of important biological characteristics (<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Vanburen <em>et al.</em> 2018</a>). With recent advances in sequencing technology, several new platforms such as PacBio and Oxford Nanopore have been used to assemble large and complex genomes from tens of thousands of long individual reads. Several of these genomes, including those of opium poppy (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Guo <em>et al.</em> 2018</a>), <em>Panicum miliaceum</em> (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Shi <em>et al.</em> 2019</a>), maize (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Yang <em>et al.</em> 2019</a>), and strawberry (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Edger <em>et al.</em> 2019</a>), were assembled using reads generated on the PacBio sequencing platform, while others such as those of <em>Arabidopsis thaliana</em> (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Michael <em>et al.</em> 2018</a>), <em>Chrysanthemum nankingense</em> (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Song <em>et al.</em> 2018</a>) and <em>Tectona grandis</em> (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Yasodha <em>et al.</em> 2018</a>) were assembled using the Nanopore sequencing technology.</p><p id="p-6">In this study, we report a high-quality genome sequence of <em>P. simonii</em> that was mainly assembled using long reads generated by the PacBio Sequel system. These reads were incorporated with paired-end (PE) short reads from the Illumina HiSeq platform and with high-density genetic maps of <em>P. deltoides</em> and <em>P. simonii</em>. The draft genome sequence is of high quality in terms of both genome integrity and base quality. Genomic repetitive sequences were analyzed, and a large number of protein-coding genes were predicted and annotated. Concomitant phylogenetic analysis provided genome-level evidence for the placement of <em>P. simonii</em> and <em>P. trichocarpa</em> in different sections. The genome assembly of <em>P. simonii</em> provides a valuable resource for identifying genome variants among individuals, performing comparative genomics within and between species, and mining candidate genes that underlie ecologically and economically important traits.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Sampling and sequencing</h3><p id="p-7">As the male parent in an F<sub>1</sub> hybrid population, a single <em>P. simonii</em> tree was chosen from a forestland managed by the Luoning Forest Bureau of Henan Province, China and planted at the Xiashu Forest Farm of Nanjing Forestry University, Jurong, Jingsu Province, China (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Mousavi <em>et al.</em> 2016</a>). Fresh leaves were collected from the paternal tree, immediately transferred to liquid nitrogen, and stored until DNA extraction. Genomic DNA was extracted using the CTAB protocol (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Porebski <em>et al.</em> 1997</a>), and a library with an insert size of 20 kb was constructed using a BluePippin DNA size selection instrument (Sage Science, MA, USA) with a lower size limit of 10 kb. The prepared library was sequenced using P6/C4 chemistry according to the manufacturer’s protocols (Pacific Biosciences, USA). Single-molecule real-time (SMRT) sequencing of long reads was performed at BMK (Biomarker Technologies Corporation, Beijing, China) and at FTC (Frasergen Technologies Corporation, Wuhan, China) on the PacBio Sequel platform (Pacific Biosciences, USA). After the removal of adapter sequences using SMRTlink (v7.0.1; <a href="https://www.pacb.com/support/software-downloads">https://www.pacb.com/support/software-downloads</a>), the remaining sequences were used for genome assembly and error correction.</p><p id="p-8">In addition, using the same individual tree described above, we sequenced the whole genome of <em>P. simonii</em> by the Illumina HiSeq 2000 sequencing platform at BMK. The detailed procedure used for whole-genome sequencing is described in <a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Mousavi <em>et al.</em> (2016)</a>. The PE reads data are available from the NCBI SRA database under the accession number SRP071167. These whole-genome sequencing data were filtered using the NGC QC toolkit with default parameters (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Patel and Mukesh 2012</a>) to obtain high-quality data for correcting the contigs assembled with the PacBio sequencing data.</p><p id="p-9">For the purpose of genome annotation, we also performed RNA sequencing of fresh leaf tissue isolated using TRIzol Reagent (Invitrogen, USA). The RNA libraries were prepared using the TruSeq RNA Library Preparation Kit (Illumina, CA, USA), and PE sequencing with a read length of 90 bp was then conducted on the Illumina HiSeq 2000 platform at Beijing Genomics Institute (BGI). These transcriptome reads were already deposited in the SRA database at the NCBI with the accession number SRR9113443. The RNA sequencing data were also filtered using the NGC QC toolkit with default parameters (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Patel and Mukesh 2012</a>).</p></div><div id="sec-3" class="subsection"><h3>Genome assembly and correction</h3><p id="p-10">The genome size of <em>P. simonii</em> was estimated based on the k-mer method using the sequencing data from the Illumina DNA library (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lamichhaney <em>et al.</em> 2016</a>). High-quality reads were subjected to 17-mer distribution using the Jellyfish program (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Marçais and Kingsford 2011</a>). The genome size was estimated according to the formula G=(Ktotal-Kerror)/D, where Ktotal is the total number of K-mers, Kerror is the number of K-mers at low frequency (frequency &lt;=1), D is the depth of K-mer, and G is the final predicted genome size. In addition, the genome heterozygosity was estimated using GenomeScope software (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Vurture <em>et al.</em> 2017</a>).</p><p id="p-11">As shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, we first obtained the contigs of <em>P. simonii</em> by assembling the long reads sequenced from the PacBio Sequel platform. The reads data in BAM format were transformed into FASTA format by the standard PacBio SMRTlink software package after removal of adapter sequences. After further filtering out short sequences (&lt;500 bp), the remaining reads were used in the subsequent analysis. The contig assembly of <em>P. simonii</em> was conducted using the FALCON assembler (v0.3.1) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chin <em>et al.</em> 2016</a>). FALCON is a hierarchical genome assembly software package developed by PacBio that uses the following steps to generate a genome assembly from a set of sequencing reads: (1) Raw subreads overlapping for error correction; (2) Preassembly and error correction; (3) Overlapping detection of the error-corrected reads; (4) Overlap filtering; (5) Construction of a graph from the overlaps; and (6) Construction of contigs from the graph. Through adjusting the parameters of the FALCON software in error correction and assembly several times, the following best parameters for assembly were obtained: length_cutoff = 1000; length_cutoff_pr = 1000; pa_daligner_option = -e.70 -l1000 -k18 -h480 -w8 -s100; falcon_sense_option =–output-multi–min-cov-aln 4–min-idt 0.70–min-cov 4–max-n-read 200; overlap_filtering_setting =–max-diff 60–max-cov 100–min-cov 2. Due to the complexity of forest genomes (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Minio <em>et al.</em> 2019</a>), we further used FALCON-unzip with the default parameters to phase the reads based on heterozygous SNPs identified in the initial assembly. The purpose of this step was to produce a set of partially phased primary contigs and fully phased haplotigs (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Chin <em>et al.</em> 2016</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/455/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Integrated work-flow for sequencing, assembly and annotation of the Populus simonii genome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1899513578" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Integrated work-flow for sequencing, assembly and annotation of the &lt;em&gt;Populus simonii&lt;/em&gt; genome.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/455/F1.medium.gif" width="418" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/2/455/F1.medium.gif" width="418" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/455/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/455/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495727" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-12" class="first-child">Integrated work-flow for sequencing, assembly and annotation of the <em>Populus simonii</em> genome.</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">To improve the accuracy of the genome assembly, we polished the contigs described above in two ways. First, all the SMRT clean reads were aligned to the contigs of <em>P. simonii</em> using the command <em>blasr</em> in SMRTlink; then, Arrow software (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chin <em>et al.</em> 2013</a>) was used to correct the errors in the contigs. Next, second-round correction was performed using the high-quality Illumina short reads. The PE reads data were aligned to the contigs using the command <em>mem</em> in the software BWA (v0.7.15) (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Li and Durbin 2009</a>); SAMtools software (v1.8) (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Li <em>et al.</em> 2009</a>) was then used to convert the output files to BAM format. We applied the software Pilon (v1.22) (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Walker <em>et al.</em> 2014</a>) to the alignment results to further correct the contigs of the assembly under the default parameters.</p></div><div id="sec-4" class="subsection"><h3>Assembly integration with genetic linkage maps</h3><p id="p-14">Two parental linkage maps of <em>P. deltoides</em> and <em>P. simonii</em> were used to anchor the contigs into chromosomes using the software ALLMAPS (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Tang <em>et al.</em> 2015</a>). The linkage maps, which were constructed in a previous study (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Yao <em>et al.</em> 2018</a>), contained a total of 3,913 SNPs on the maternal map of <em>P deltoides</em> and 2,584 SNPs on the paternal map of <em>P. somonii</em>. To match each linkage group with the chromosome in <em>Populus</em>, we mapped a 41-bp sequence flanking each SNP to the reference genome of <em>P. trichocarpa</em> using the program <em>blastn</em> from NCBI (<a href="ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/2.8.1/">ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/2.8.1/</a>). If the majority of SNPs in a linkage group were mapped to a chromosome, the linkage group was considered to correspond to the chromosome. The sequence representing a SNP was also aligned to the contigs assembled above by setting the query coverage equal to 90%. After that, two csv-formatted files were prepared for each parental linkage map according to the requirement of ALLMAPS; these files contained records for each mapped SNP, including the contig identifier, the position on the contig, the linkage group identifier, and the position in the linkage group. Finally, the anchoring process was conducted using ALLMAPs with equal weights for the two linkage maps.</p></div><div id="sec-5" class="subsection"><h3>Assessment of the genome assembly</h3><p id="p-15">To evaluate the base level accuracy of the genome assembly, high-quality PE reads data were mapped to the assembled genome using BWA software to generate mapping ratio statistics. Then, the erroneous bases and homozygous SNP loci in the genome were identified using the software Freebayes (v1.2.0) (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Garrison and Marth 2012</a>) with the BAM files generated in BWA. We further evaluated the completeness of the genome assembly using Benchmarking Universal Single-Copy Orthologs software (BUSCO v3) (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Simão <em>et al.</em> 2015</a>) with the embryophyta_odb10 database (<a href="http://busco.ezlab.org">http://busco.ezlab.org</a>) and the parameter settings “-m genome -c 30”. The completeness of genome assembly was assessed on the basis of the proportion of the complete BUSCOs present in the genome.</p></div><div id="sec-6" class="subsection"><h3>Repetitive sequences and noncoding RNA annotation</h3><p id="p-16">Plant genomes, especially complex plant genomes in which many of the repetitive sequences are transposable elements (TEs), usually contain large numbers of repetitive sequences (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Xu <em>et al.</em> 2019</a>). To identify transposable elements, the software RepeatModeler (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Tarailo-Graovac and Chen 2009</a>) was used to identify <em>de novo</em> repeat types in <em>P. siomonii</em>. Then, combination of the results from RepeatModeler with the repeat sequence database Repbase (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bao <em>et al.</em> 2015</a>) was used as the final repeat sequence library for subsequent analysis with RepeatMasker (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Tarailo-Graovac and Chen 2009</a>). This step yields a more accurate annotation result for repetitive elements in <em>P. simonii</em>.</p><p id="p-17">In the process of annotating noncoding RNAs (ncRNA), we identified miRNA and snRNA genes in the genome of <em>P. simonii</em> by searching the Rfam database using INFERNAL software with default parameters (v1.1.2) (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kalvari <em>et al.</em> 2018</a>). Moreover, we used tRNAscan-SE software with default parameters for tRNA annotation (v1.3.1) (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lowe and Eddy 1997</a>) and RNAmmer for rRNA annotation (v1.2) (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lagesen <em>et al.</em> 2007</a>).</p></div><div id="sec-7" class="subsection"><h3>Gene prediction</h3><p id="p-18">We identified the protein-coding genes of <em>P. simonii</em> using <em>ab initio</em> prediction, homology-based prediction, and RNA-seq-assisted prediction methods. The transcripts were obtained by <em>de novo</em> assembling the RNA-seq data of <em>P. simonii</em> with Trinity (v2.4.0) (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Haas <em>et al.</em> 2013</a>). Before gene prediction, an initial gene model file was constructed by analyzing the transcriptomic data with the software PASA (v2.3.3) (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Campbell <em>et al.</em> 2006</a>). In addition, the repetitive sequences of the transcripts were screened according to the annotations of repetitive sequences to prevent them from interfering with the results of gene prediction. Next, the Augustus (v3.3.1) (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Stanke <em>et al.</em> 2006</a>), SNAP (v2006-07-28) (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Korf 2004</a>) and GeneMark (v4.46) (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Ter-Hovhannisyan <em>et al.</em> 2008</a>) software packages were applied with default parameters to create three <em>ab initio</em> training files incorporating the use of PASA for gene model training. For homology-based prediction, protein sequences from 4 closely related species, <em>Salix suchowensis</em> (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dai <em>et al.</em> 2014</a>), <em>Populus trichocarpa</em> (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Tuskan <em>et al.</em> 2006</a>), <em>Populus deltoides</em> (<a href="https://phytozome.jgi.doe.gov/pz/portal.html">https://phytozome.jgi.doe.gov/pz/portal.html</a>), and <em>Populus euphratica</em> (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Ma <em>et al.</em> 2013</a>), were aligned to the genome assembly of <em>P. simonii</em> using tblastn with the default parameters. Furthermore, Exonerate software (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Slater and Birney 2005</a>) was used to polish the blast hits to obtain exact intron and exon positions. Finally, all the predictions results were integrated using the software package MAKER (v3.01.02) (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Holt 2011</a>) to generate a consensus gene set, and the quality of the gene annotation was assessed using BUSCO (v3) with the parameters set as follows: “-c 30 –m proteins –l embryophyta_odb10”.</p></div><div id="sec-8" class="subsection"><h3>Functional annotation of protein-coding genes</h3><p id="p-19">Functional annotations of the protein-coding genes in <em>P. simonii</em> were obtained according to the best matches by aligning the predicted protein sequences to well-known databases using the local blastp command with an e-value threshold of 1e-5. The databases included Nonredundant protein sequences (Nr) as of March 12, 2019, eukaryotic orthologous groups of proteins (KOG) as of September 9, 2018 (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Tatusov <em>et al.</em> 2001</a>), the Kyoto Encyclopedia of Genes and Genomes (KEGG) as of July 1, 2019 (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Ogata <em>et al.</em> 2000</a>) and Swissprot as of August 10, 2019 (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bairoch and Boeckmann 1991</a>). To derive the annotation information for motifs and domains, we searched the protein sequences in the Pfam database (Pfam 28) as of May 20, 2015 (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Zdobnov and Apweiler 2001</a>) using the software hmmer (v3.1b2) (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Mistry <em>et al.</em> 2013</a>) with default parameters. The blast hits of the protein-coding genes were mapped to the Gene Ontology (GO) terms using the BLAST2GO pipeline (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Conesa <em>et al.</em> 2005</a>). InteractiVenn software (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Heberle <em>et al.</em> 2015</a>) was used to plot a Venn diagram of the number of protein-coding genes with functional annotations based on the six databases described above.</p></div><div id="sec-9" class="subsection"><h3>Comparative and evolutionary genomics</h3><p id="p-20">To cluster families from protein-coding genes, we downloaded the online protein data for eight other species, including three species in the genus <em>Populus</em>. We retrieved protein data of <em>P. trichocarpa</em> (v3.1), <em>P. deltoides</em> WV94 (v2.1), <em>Ricinus communis</em> (v0.1), <em>Oryza sativa</em> (v7), <em>Carica papaya</em> ASGPB (v0.4), <em>Manihot esculenta</em> (v6.1), and <em>Salix purpurea</em> (v1.0) from Phytozome (<a href="https://phytozome.jgi.doe.gov/pz/portal.html">https://phytozome.jgi.doe.gov/pz/portal.html</a>). The data for <em>P. euphratica</em> (v1.0) were downloaded from NCBI (<a href="https://www.ncbi.nlm.nih.gov/genome/">https://www.ncbi.nlm.nih.gov/genome/</a>). The <em>Salix suchowensis</em> (v4.1) data were obtained from the ftp site at Nanjing Forestry University (<a href="ftp://plantgenie.org/Data/PopGenIE/Salix-suchowensis">ftp://plantgenie.org/Data/PopGenIE/Salix-suchowensis</a>). To remove redundant sequences caused by alternative splicing variations, we retained only the gene model at each gene locus that encodes the longest potential protein sequence. We then employed DIAMOND software to identify potentially orthologous gene families among the filtered protein sequences in these species using a maximum e-value of 1e-5 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Buchfink <em>et al.</em> 2015</a>). The pipeline OrthoFinder was used to cluster gene families based on the aligned results under the default parameters (v2.3.3) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Emms and Kelly 2015</a>). STRIDE software (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Emms and Kelly 2017</a>) was used to identify gene duplication events based on the orthgroups generated by OrthoFinder.</p><p id="p-21">The <em>P. simonii</em> genome was compared to the <em>P. trichocarpa</em> genome (v3.1) using the MUMmer software package (v3.23) (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kurtz <em>et al.</em> 2004</a>). The chromosome-level sequences of <em>P. simonii</em> were aligned to the reference genome of <em>P. trichocarpa</em> with the PROmer script in the package followed a filtering process using the delta-filter script with parameter setting as “-i 100 -l 150”. A postscript file was then generated using the mummerplot script for visualization of the genomic structure comparison.</p><p id="p-22">To reveal the phylogenetic relationship between <em>P. simonii</em> and other <em>Populus</em> species, we conducted phylogenetic analysis using the orthologs from the single-copy gene families. MAFFT software (v7.158b) (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Katoh and Standley 2013</a>) was used to generate multiple sequence alignments of protein-coding sequences for each single-copy gene with an accurate option (LINS-i). The software trimAl (v1.2) (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Capella-Gutiérrez <em>et al.</em> 2009</a>) was used with the default parameters to remove poorly aligned positions and divergent regions. The alignments for all the single-copy genes were then concatenated to form a super matrix. The super matrix was used for phylogenetic tree construction by RAxML software (v8.2.12) (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Stamatakis 2014</a>) with the model JTT+G+F (determined by the BIC model selection criterion) and bootstrap replicates of 1000. Next, r8s software (v1.71) (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Sanderson 2003</a>) was applied to estimate the divergence time among species with selected parameter settings as follows: “blformat lengths=persite nsites=462057 ulrametric=no; set smoothing=100; divtime method=PL algorithm=TN” and the others as defaults. The divergence times of <em>Oryza sativa</em> (115-308 Mya), <em>Ricinus communis</em> (70-86 Mya), <em>and Salix purpurea</em> (12-48 Mya) obtained from the TimeTree database (<a href="http://www.timetree.org/">http://www.timetree.org/</a>) were used as the calibration points. Finally, the phylogenetic tree was visualized in FigTree (v1.4.3) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Drummond and Rambaut 2007</a>).</p><p id="p-23">To investigate whole-genome duplication (WGD) events in <em>P. simonii</em>, the distribution of synonymous substitution rate (Ks) was obtained from its CDS sequences, compared with <em>P. trichocarpa</em>. The pipline wgd (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Zwaenepoel and Van de Peer 2019</a>) was used to calculate the Ks distributions from paralogous sequences in each of the two species. We also used CAFÉ software (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">De Bie <em>et al.</em> 2006</a>) to analyze gene family evolution according to the phylogenetic relationships and divergence times, setting the parameters as “load –p 0.05 –t 20; lambda –s”.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-24">The data of the whole-genome PE short reads from the Illumina platform, the two batches of long reads from the PacBio system, and the transcriptome reads have been deposited in the SRA database at the National Center for Biotechnology Information (NCBI) with accession numbers of SRP071167, SRR9112943, SRR9887262, and SRR9113443, respectively. The genome assembly of <em>P. simonii</em> is available under the GenBank assembly accession number GCA_007827005.2. Statistical information on the PacBio Sequel raw data can be found in File S1. The repetitive sequence annotation was saved in File S2 in fasta format. Annotation of noncoding RNAs can be found in File S3. The protein sequences with the predicted genes are contained in File S4 in fasta format. The genome annotation was saved in File S5 in gff3 format. Gene functional annotation information can be found in File S6. The supplementary tables (Tables S1-S7) and figures (Figures S1-S11) can be found in File S7. Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.9905492">https://doi.org/10.25387/g3.9905492</a>.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Sequencing</h3><p id="p-25">SMRT sequencing of long reads was conducted on a PacBio Sequel platform with 6 cells. After filtering out the low quality and adapter sequences, a total of 3,412,303 reads were obtained (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and Figure S1). After further filtering out the short sequences (&lt;500 bp), 3,147,743 high-quality reads up to 29.65 Gb remained, with an average read length of 9,418 bp and a longest read length of 112,390 bp (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and File S1).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495639/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495639/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Clean data<em>#</em><sup><em><sup>a</sup></em></sup> generated by the PacBio sequel platform</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495758/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495758/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Statistics of high-quality reads data and library information</span><div class="sb-div caption-clear"></div></div></div><p id="p-29">The short fragments of the DNA and RNA libraries were sequenced using the Illumina HiSeq 2000 sequencing platform. A total of 326,922,690 PE reads representing 32.02 Gb were obtained by genomic sequencing. After filtering out the low-quality sequences, 256,124,774 high-quality reads representing 25.87 Gb of genomic data remained. For the transcriptome sequencing, we obtained approximately 8.50 Gb of RNA-Seq data after filtering out the low-quality reads (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>).</p></div><div id="sec-13" class="subsection"><h3>Genome size estimation, assembly and correction</h3><p id="p-30">To determine the amount of data needed for sequencing, we estimated the genome size of <em>P. simonii</em>. The genome size was predicted based on K-mer analysis using the software Jellyfish with the Illumina genomic sequencing data (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Marçais and Kingsford 2011</a>). Taking the k-mer size as 17 bp, a total of 21,770,605,790 k-mers were generated with 932,308 error kmers and a kmer depth of 53, leading to an estimated genome size of approximately 411 Mb and an estimated heterozygosity rate of 1.34% (Figure S2).</p><p id="p-31"><em>De novo</em> assembly of the <em>P. simonii</em> genome was performed using FALCON software with the PacBio long reads data. To improve the accuracy of the PacBio data, we first used the self-correcting program of FALCON to correct the HQ long reads, obtaining 1,690,300 reads up to 16.15 Gb (∼34x coverage) with an average length of 9,553 bp for <em>de novo</em> assembly. After performing the primary assembly, we obtained a draft genome size (G1) of 447 Mb with 911 contigs and a contig N50 of 1.89 Mb (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). We then used FALCON-unzip to phase the primary contigs, obtaining 440 Mb partially phased primary contigs (G2) and 328 Mb fully phased haplotigs with contig N50s of 1.93 Mb and 202 kb, respectively (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). Next, the assembled genome was error-corrected using Arrow software with the PacBio data to obtain a consensus sequence. This step corrected 946,062 insertions, 142,410 deletions, and 540,656 substitutions, leading to a corrected genome size of 441 Mb. To further improve the accuracy of the genome assembly, we also used Pilon software and the Illumina genomic reads to correct the consensus sequences again. This identified 181,316 insertion sites, 76,138 deletion sites, and 152,077 substitution sites that required correction. After the two rounds of error correction, the final genome assembly (G3) of <em>P. simonii</em> contained 686 contigs and had a total length of 441 Mb and a contig N50 of 1.94 Mb; the longest contig was equal to 8.12 Mb, and the GC content was 33.65% (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495755/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495755/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Statistics of the <em>Populus simonii</em> assembly</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Integration of the genome assembly using linkage maps</h3><p id="p-33">The two high-quality genetic linkage maps from our previous study (<a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Yao <em>et al.</em> 2018</a>) were used to anchor the contigs assembled above into chromosomes with genetic and genomic distances. Those contigs, each containing at least one marker on the linkage maps, were assigned to 19 chromosomes. The maternal and paternal genetic maps were integrated to form a consensus map using ALLMAPS. The resulting consensus map consisted of 5,971 unique markers, equivalent to an average physical marker density of 15.0 markers/Mb. Of 686 contigs in the draft genome assembly of <em>P. simonii</em>, 336 were aligned on the genetic maps by at least one marker, accounting for 90.2% of the assembled genome size (398 Mb of the 441 Mb assembly). In addition, 225 contigs (representing 75.9% of the anchored contigs and comprising 84.1% of the total genome length) were anchored with two or more markers so that their orientations on the chromosomes could be determined. Due to lack of markers or location conflicts, 350 contigs and 126 markers were not anchored to chromosomes; these represented only 9.8% of the total genome length (Figure S3 and <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). The adjacent contigs in each chromosome were filled with 100 ‘N’s. In summary, the final assembled genome of <em>P. simonii</em> contained 19 chromosome sequences and 350 unplaced contigs and had a total length of 441 Mb and a sequence N50 of 19.6 Mb (G4).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495652/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495652/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Summary of the consensus map</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Assembly evaluation</h3><p id="p-35">To evaluate the accuracy of our genome assembly, the 25.87 Gb Illumina genomic DNA sequencing data were aligned to the final assembled genome of <em>P. simonii</em> (G4) using BWA. The results showed that 97.56% of the reads were mapped to the genome assembly, indicating a high mapping ratio. We then used the software Freebayes to identify the erroneous bases in the <em>P. simonii</em> genome assembly. A total of 80,463 homozygous mutation sites were identified with a mutation rate of 1.82×10<sup>−4</sup>, indicating a high-quality genome assembly.</p><p id="p-36">We also evaluated the completeness of the genome assembly (G3, <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>) using BUSCO with the embryophyta_oda10 database. The results showed that 97.9% of the complete BUSCO genes were identified in the assembly (1,347 <em>vs.</em> 1,375 genes), with 0.5% partial BUSCOs identified and only 1.6% missed (Table S1). Overall, all the results suggest that the quality of the assembly is high with respect to the base level accuracy and the completeness of the assembly.</p></div><div id="sec-16" class="subsection"><h3>Repetitive sequence and noncoding RNA analysis</h3><p id="p-37">Repetitive sequence analysis of the assembly (G3, <a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>) showed that the repetitive sequences comprised approximately 183 Mb (41.47%) of the <em>P. simonii</em> genome (File S2); of these, interspersed repeats accounted for 40.17%, simple repeats occupied approximately 5,723,970 bp (1.30%), and DNA elements made up 3.99% (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). In the noncoding RNA analysis, we identified 1,153 miRNAs, 1,177 tRNAs, 290 rRNAs, and 618 snRNAs in the <em>P. simonii</em> genome (Table S2; File S3).</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495776/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495776/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Summary statistics of annotated repeats</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Gene prediction and functional annotation</h3><p id="p-38">We predicted genes with <em>ab initio</em>, homologous annotation and transcriptome sequencing-based methods using the MAKER software package. A total of 45,459 protein-coding genes were predicted in the <em>P. simonii</em> genome; they were distributed in 369 scaffolds (G4, <a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>) with an average of 123 genes per scaffold (Table S3; File S4). BUSCO analysis showed that 94.9% of the complete BUSCO genes were identified in the protein-coding genes (Table S1). All gene structure information for the genome is reported in File S5.</p><p id="p-39">To obtain motif, domain, pathway and other functional information on the predicted genes, the predicted protein sequences were searched in the publicly available databases, and the annotated results in each database were counted. By aligning to the KOG database, 10,257 proteins were assigned to 25 KOG categories; of these, the T function classification (Signal transduction mechanisms) contained approximately 1,538 predicted genes, and the R function classification (general function prediction only) contained 1,493 predicted genes (Figure S4). Moreover, 17,860 genes were assigned to corresponding GO terms, accounting for 39.3% of the total number of predicted proteins. The GO annotation classification includes three main categories: “biological process”, “cellular component” and “molecular function”. In the molecular function category, “catalytic activity” was the largest subcategory, followed by the “binding” subcategory. In the cellular component category, “cell” and “cell part” were the two main subcategories. In the biological process category, the majority of protein-coding genes were assigned to the “metabolic process” subcategory (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). To further enrich our understanding of the functions of the predicted proteins in <em>P. simonii</em>, we successfully annotated 11,866 putative proteins in the KEGG database. Detailed KEGG function classifications information is shown in Figure S5. Approximately 87.6% (39,833) of the protein-coding genes were functionally annotated using at least one public databases, including Nr, Swissport, Pfam, KOG, KEGG, and GO (Table S4; File S6). Figure S6 displays the Venn diagram of the protein-coding genes with functional annotations.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/455/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Gene Ontology (GO) function annotation of Populus simonii using WEGO 2.0 (Ye et al. 2018). The horizontal axis shows the GO classification types, and the vertical axis represents the number of annotated protein-coding genes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1899513578" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Gene Ontology (GO) function annotation of &lt;em&gt;Populus simonii&lt;/em&gt; using WEGO 2.0 (Ye &lt;em&gt;et al.&lt;/em&gt; 2018). The horizontal axis shows the GO classification types, and the vertical axis represents the number of annotated protein-coding genes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/455/F2.medium.gif" width="440" height="275"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/2/455/F2.medium.gif" width="440" height="275"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/455/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/455/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495517" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-40" class="first-child">Gene Ontology (GO) function annotation of <em>Populus simonii</em> using WEGO 2.0 (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Ye <em>et al.</em> 2018</a>). The horizontal axis shows the GO classification types, and the vertical axis represents the number of annotated protein-coding genes.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Comparative and evolutionary genomics</h3><p id="p-41">Among <em>P. simonii</em> and the related eight plant species described in Materials and Methods, a total of 26,181 gene families were constructed using the software OrthoFinder; of these, 966 were identified as single-copy orthologous gene families, and 10 were identified as unique to <em>P. simonii</em> (Tables S5 and S6). There were 24,955 gene families containing sequences from the four <em>Populus</em> species, of which 15,556 (62.3%), 4,451 (17.8%), 4,237 (17.0%), and 711 (2.8%) were shared by 4, 3, 2, and only one of these species, respectively (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Excluding the gene families shared by all four <em>Populus</em> species, we found that the desert tree species <em>P. euphratica</em> shared 996 gene families with any one or two of the three species <em>P. trichocarpa</em>, <em>P. deltoides</em>, and <em>P. simonii</em>, while 7,692 gene families were shared within the remaining three <em>Populus</em> species. The results show that the average number of gene families (2,564) shared within the three <em>Populus</em> species was almost 2.6 times the number of gene families shared between <em>P. euphratica</em> and any one or two of the other three <em>Populus</em> species. This indicates that <em>P. simonii</em> is more closely related to <em>P. trichocarpa</em> and <em>P. deltoides</em> than to <em>P. euphratica</em>. In particular, the genome structure of <em>P. simonii</em> was evaluated by comparing it with that of <em>P. trichocarpa</em>, performed by genomic alignments at the chromosome level using MUMmer software (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Kurtz <em>et al.</em> 2004</a>). As a result, <em>P. simonii</em> structure was globally similar to that of <em>P. trichocarpa</em>, without large rearrangements, invertions or translocations (Figure S7).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/455/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Shared gene families and their distribution per species. Venn diagram showing the shared gene families between the selected Populus species: Populus simonii, Populus trichocarpa, Populus deltoides, and Populus euphratica. The histogram represents the total number of gene families for each species. The numbers of gene families shared by 4, 3, 2, and only one of these species are presented at the bottom." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1899513578" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Shared gene families and their distribution per species. Venn diagram showing the shared gene families between the selected &lt;em&gt;Populus&lt;/em&gt; species: &lt;em&gt;Populus simonii&lt;/em&gt;, &lt;em&gt;Populus trichocarpa&lt;/em&gt;, &lt;em&gt;Populus deltoides&lt;/em&gt;, and &lt;em&gt;Populus euphratica&lt;/em&gt;. The histogram represents the total number of gene families for each species. The numbers of gene families shared by 4, 3, 2, and only one of these species are presented at the bottom.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/455/F3.medium.gif" width="338" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/2/455/F3.medium.gif" width="338" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/455/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/455/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495699" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-42" class="first-child">Shared gene families and their distribution per species. Venn diagram showing the shared gene families between the selected <em>Populus</em> species: <em>Populus simonii</em>, <em>Populus trichocarpa</em>, <em>Populus deltoides</em>, and <em>Populus euphratica</em>. The histogram represents the total number of gene families for each species. The numbers of gene families shared by 4, 3, 2, and only one of these species are presented at the bottom.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">The evolutionary relationship of <em>P. simonii</em> to the other eight species was constructed using the 966 single-copy orthologous genes, and the divergence time and the extrapolated calibration time point were estimated. The phylogenetic tree analysis showed that <em>P. simonii</em> is more closely related to <em>P. deltoides</em> and <em>P. trichocarpa</em> than to <em>P. euphratica</em> (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>), consistent with the above inference from the number of shared gene families. However, the analysis also showed that <em>P. trichocarpa</em> is more closely related to <em>P. deltoides</em> than to <em>P. simonii</em>, which is contrary to the traditional section classification according to morphology (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Eckenwalder 1996</a>) but consistent with the results in recent studies using molecular data (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Wang <em>et al.</em> 2014</a>; <a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Zong <em>et al.</em> 2019</a>); this suggests that <em>P. simonii</em> and <em>P. trichocarpa</em> should be assigned to different sections in the genus <em>Populus</em> (see more in Discussion). Moreover, it was estimated that <em>P. simonii</em> diverged approximately 4.36 million years ago from its common ancestor with <em>P. deltoides</em> and <em>P. trichocarpa</em> (Figure S8). The number of gene duplications was also estimated along each branch on the species tree; this indicated that there were 2,777 gene duplications in <em>P. simonii</em> (Figure S9). Furthermore, from the density curves of Ks distributions for the paralogs of <em>P. simonii</em> and <em>P. trichocarpa</em> (Figure S10), we observed that the two species shared similar peaks near the Ks value of 0.23. This indicated that <em>P. simonii</em> experienced the same WGD events as <em>P. trichocarpa</em> (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Ma <em>et al.</em> 2019</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Liu <em>et al.</em> 2019</a>). Additionally, the analysis of gene family expansion and contraction showed that 2,356 gene families were expanded and 5,224 families were contracted in the <em>P. simonii</em> genome compared to the other plant species (Figure S11 and Table S7).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/455/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic relationships of Populus simonii and related species. A maximum likelihood phylogenetic tree of P. simonii and 8 other plant species was constructed through the concatenated alignment of 966 1-to-1 single-copy orthologous genes and then using RAxML software with the model JTT+G+F. The number on the nodes represents the bootstrap support value estimated from 1000 bootstrap tests." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1899513578" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogenetic relationships of &lt;em&gt;Populus simonii&lt;/em&gt; and related species. A maximum likelihood phylogenetic tree of &lt;em&gt;P. simonii&lt;/em&gt; and 8 other plant species was constructed through the concatenated alignment of 966 1-to-1 single-copy orthologous genes and then using RAxML software with the model JTT+G+F. The number on the nodes represents the bootstrap support value estimated from 1000 bootstrap tests.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/455/F4.medium.gif" width="440" height="345"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/10/2/455/F4.medium.gif" width="440" height="345"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/455/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/455/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495510" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-44" class="first-child">Phylogenetic relationships of <em>Populus simonii</em> and related species. A maximum likelihood phylogenetic tree of <em>P. simonii</em> and 8 other plant species was constructed through the concatenated alignment of 966 1-to-1 single-copy orthologous genes and then using RAxML software with the model JTT+G+F. The number on the nodes represents the bootstrap support value estimated from 1000 bootstrap tests.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-19"><h2 class="">Discussion</h2><p id="p-45">High-quality genome assembly is particularly important for improving the annotation of gene models as well as for facilitating evolutionary and functional genomics analyses (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lin <em>et al.</em> 2018</a>). Here, we present a chromosome-scale genome assembly of <em>P. simonii</em> with fewer gaps and higher continuity. This assembly was obtained by combining the use of PacBio long reads with the use of Illumina short reads and the related genetic linkage maps. The assembly described here can be considered a high-quality draft genome of <em>P. simonii</em> because we attempted to maximize its quality by optimizing the parameters of FALCON, an undertaking that has not previously been reported, as well as by performing two polishing iterations using different sequencing technologies. The size of the assembly was approximately 441 Mb with a contig N50 of 1.94 Mb. Compared with previously published genomes in <em>Populus</em>, such as <em>P. alba</em> with contig N50 of 9.8 kb (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Ma <em>et al.</em> 2019</a>) and <em>P. pruinosa</em> with contig N50 of 14 kb (<a id="xref-ref-76-3" class="xref-bibr" href="#ref-76">Yang <em>et al.</em> 2017</a>), the contig N50 of the <em>P. simonii</em> genome obtained by SMRT data are greatly improved. Furthermore, the PacBio-based assembly contains more complete repetitive sequences than the Illumina-based assembly (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Winkler <em>et al.</em> 2018</a>). This confirms that the strategy of combining third-generation sequencing (used to obtain contig with higher continuity) and Illumina sequencing technology (used to obtain high-quality base sequences) can be effectively applied to the assembly of complex plant genomes (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Li <em>et al.</em> 2017</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jung <em>et al.</em> 2019</a>). However, the contig N50 value of our assembly is still far from that of the other plant genomes assembled using the PacBio Sequel platform, such as <em>Hordeum vulgare</em> (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Zeng <em>et al.</em> 2018</a>) and <em>Brassica rapa</em> (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Zhang <em>et al.</em> 2018</a>). This may be due to the higher heterozygosity of the genomes of forest trees and could be improved by increasing the sequencing depth of the PacBio long reads in <em>P. simonii</em> from the current ∼50X to 80X (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Lin <em>et al.</em> 2018</a>).</p><p id="p-46">On the other hand, the quality of a chromosome-level genome assembly largely depends on the genetic linkage map used to anchor the contigs. The key problem in creating a linkage map is whether the order of the molecular markers within each linkage group is sufficiently accurate. Errors in the order of markers in a linkage map will seriously affect the order of contigs in the anchoring process. We used two high-density genetic linkage maps to anchor 336 contigs into chromosomes with a size of 398 Mb, accounting for 90.2% of the genome in this study. The linkage maps were constructed using SNPs generated from the RADseq data across the mapping population in our previous study (<a id="xref-ref-78-3" class="xref-bibr" href="#ref-78">Yao <em>et al.</em> 2018</a>). Compared with previous genetic mapping studies in <em>Populus</em> (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bradshaw <em>et al.</em> 1994</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Yin <em>et al.</em> 2002</a>; <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Zhang <em>et al.</em> 2004</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Zhang <em>et al.</em> 2009</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Paolucci <em>et al.</em> 2010</a>), the linkage maps used in this study could be considered to be of high quality because they contained many more SNPs that were confirmed to be of high quality. Moreover, the number of divided linkage groups perfectly matched the karyotype of <em>Populus</em> under a wide range of LOD thresholds, which was rarely reported in the previous studies. This suggests that the SNPs were uniformly distributed on the genome and were of high quality. Undoubtedly, high-quality SNPs can improve the accuracy of ordering SNPs using the available mapping software, thus enhancing the quality of our chromosome-level genome assembly. However, there is still room to improve the algorithms used to order markers in the current mapping software; ordering tens or hundreds of markers in a linkage group belongs to the NP-hard category of problems (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Wu <em>et al.</em> 2008</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Monroe <em>et al.</em> 2017</a>), which involves choosing the optimal order of the markers among a huge number of possible orders. Advances in ordering markers would substantially increase the precision of genetic mapping and hence improve the accuracy of chromosome-level genome assembly.</p><p id="p-47">In the context of gene predictions and annotations, our assembled genome has some characteristics similar to those of previous genome assembly studies in <em>Populus</em>. The annotation of the <em>P. simonii</em> genome is almost complete, with 97.9% of the complete BUSCOs, higher than the completeness of annotation in <em>P. alba</em> (91.10%) (<a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Ma <em>et al.</em> 2019</a>), <em>P. trichocarpa</em> (93.95%) (<a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">Tuskan <em>et al.</em> 2006</a>) and <em>P. euphratica</em> (94.35%) (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Ma <em>et al.</em> 2013</a>). Approximately 41.47% of the genome was annotated as containing repetitive sequences, similar to the values estimated for other <em>Populus species</em> (<a id="xref-ref-66-4" class="xref-bibr" href="#ref-66">Tuskan <em>et al.</em> 2006</a>; <a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Ma <em>et al.</em> 2013</a>; <a id="xref-ref-76-4" class="xref-bibr" href="#ref-76">Yang <em>et al.</em> 2017</a>; <a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">Ma <em>et al.</em> 2019</a>). A total of 45,459 genes were predicted in the genome of <em>P. simonii</em>, slightly more than the number of genes predicted in <em>P. trichocarpa</em> (42,950 genes) and in <em>P. deltoides</em> (44,853 genes). The expansion in the number of predicted genes may be attributed to repeated gene duplication (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Yang <em>et al.</em> 2018</a>); this could be confirmed by analyzing the whole-genome duplications as well as the gene families. Among the predicted genes, 39,833 (87.6%) were annotated with relevant functional information, laying the foundation for further elucidation of the biological characteristics of <em>P. simonii</em>, such as its drought stress response, its barren tolerance and its developed root system. Overall, the acquisition of the genome of <em>P. simonii</em> provides an important genetic resource for comparative genomics in <em>Populus</em> and facilitates further studies on phylogenetic inference and whole-genome duplication events.</p><p id="p-48">We measured the phylogenetic affinity of <em>P. simonii</em> with three other <em>Populus</em> species, <em>P. deltoides</em>, <em>P. trichocarpa</em>, and <em>P. euphratica</em>, because these species provide the only genome data available to date in <em>Populus</em>. According to important morphological characteristics, <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Eckenwalder (1996)</a> assigned <em>P. simonii</em> and <em>P. trichocarpa</em> to the section <em>Tacamahaca</em>, <em>P. deltoides</em> to the section <em>Aigeiros</em>, and <em>P. euphratica</em> to the section <em>Turanga</em>, suggesting that <em>P. simonii</em> and <em>P. trichocarpa</em> are more closely related than any other two of the four species. This classification of the <em>Populus</em> species is not consistent with our current phylogenetic result, which shows that <em>P. deltoides</em> and <em>P. trichocarpa</em> are more closely related to each other than <em>P. simonii</em> is to either <em>P. trichocarpa</em> or <em>P. deltoides</em>. Interestingly, this is not the first report of an inconsistent phylogenetic relationship within the <em>Populus</em> genus based on the use of different methods. Using nuclear and plastid DNA sequences, <a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Wang <em>et al.</em> (2014)</a> found that the section <em>Tacamahaca</em> could be divided into two groups, with <em>P. simonii</em> belonging to section <em>Leucoides</em> or to a sister group to section <em>Aigeriros</em> and <em>P. trichocarpa</em> belonging to the other group as a subsection in <em>Tacamahaca</em>. More recently, <a id="xref-ref-88-2" class="xref-bibr" href="#ref-88">Zong <em>et al.</em> (2019)</a> divided <em>P. simonii</em> and <em>P. trichocarpa</em> into different clades but placed <em>P. deltoides</em> and <em>P. trichocarpa</em> in the same clade using whole chloroplast genome sequences. Overall, in addition to the fact that there is almost no controversy regarding the relationship of <em>P. euphratica</em> to other <em>Populus</em> species, the phylogenetic relationships among <em>P. simonii</em>, <em>P. deltoides</em>, and <em>P. trichocarpa</em> revealed by these two studies are consistent with the results obtained in the current study.</p></div><div class="section conclusions" id="sec-20"><h2 class="">Conclusions</h2><p id="p-49">Herein, we report the first chromosome-scale genome assembly for <em>P. simonii</em>; it was obtained through an assembly strategy that combined the use of PacBio long reads data with the use of Illumina short PE reads data and related genetic linkage maps. With the assembly data, phylogenetic analysis indicates that <em>P. simonii</em> and <em>P. trichocarpa</em> should be assigned to different sections in the genus <em>Populus</em>. The genome assembly, which includes predicted genes and functional annotations, not only provides an important resource for the comparative and functional genomics of <em>Populus</em> species, but also provides the closest reference sequences for identifying genomic variants in an F<sub>1</sub> hybrid population derived by crossing <em>P. simonii</em> and other <em>Populus</em> species. The assembly strategy used in this work can be applied to other species in <em>Populus</em> to generate chromosome-level genome assemblies in a fast and cost-effective manner.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-50">This work was supported by the National Natural Science Foundation of China (Grant No. 31870654 and 31270706) awarded to CT and the Priority Academic Program Development of Jiangsu Higher Education Institutions.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-6"><p id="p-51">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.9905492">https://doi.org/10.25387/g3.9905492</a>.</p></li><li class="fn" id="fn-7"><p id="p-52"><em>Communicating editor: R. Hernandez</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-09-23"><span class="received-label">Received </span>September 23, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-12-03"><span class="accepted-label">Accepted </span>December 3, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Wu <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.1" data-doi="10.1093/bioinformatics/bti610"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conesa</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Götz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">García-Gómez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Terol</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Talón</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">3674</span>–<span class="cit-lpage">3676</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bti610</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti610%26rft_id%253Dinfo%253Apmid%252F16081474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti610&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16081474&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231694600016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.2" data-doi="10.1093/nar/19.suppl.2247"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bairoch</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Boeckmann</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">The SWISS-PROT protein sequence data bank.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">2247</span>–<span class="cit-lpage">2249</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/19.suppl.2247</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F19.suppl.2247%26rft_id%253Dinfo%253Apmid%252F2041811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/19.suppl.2247&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2041811&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FM75600018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.3" data-doi="10.1186/s13100-015-0041-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. K.</span> <span class="cit-name-surname">Kojima</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Kohany</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Repbase Update, a database of repetitive elements in eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">Mob. DNA</abbr> <span class="cit-vol">6</span>: <span class="cit-lpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13100-015-0041-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMob.%2BDNA%26rft.volume%253D6%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13100-015-0041-9%26rft_id%253Dinfo%253Apmid%252F26045719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13100-015-0041-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26045719&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.4" data-doi="10.1007/BF00225137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradshaw</span>,  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Villar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. D.</span> <span class="cit-name-surname">Watson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. G.</span> <span class="cit-name-surname">Otto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Stewart</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Molecular genetics of growth and development in <em>Populus</em>. III. A genetic linkage map of a hybrid poplar composed of RFLP, STS, and RAPD markers.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">178</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00225137</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D89%26rft.spage%253D167%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00225137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00225137&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994PN76800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.5" data-doi="10.1038/nmeth.3176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buchfink</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Xie</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. H.</span> <span class="cit-name-surname">Huson</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fast and sensitive protein alignment using DIAMOND.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.3176</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D12%26rft.spage%253D59%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3176%26rft_id%253Dinfo%253Apmid%252F25402007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3176&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25402007&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.6" data-doi="10.1186/1471-2164-7-327"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>,  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Haas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Hamilton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Mount</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. R.</span> <span class="cit-name-surname">Buell</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Comprehensive analysis of alternative splicing in rice and comparative analyses with <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">343</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-7-327</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DCampbell%26rft.auinit1%253DM.%2BA.%26rft.volume%253D7%26rft.spage%253D327%26rft.epage%253D327%26rft.atitle%253DComprehensive%2Banalysis%2Bof%2Balternative%2Bsplicing%2Bin%2Brice%2Band%2Bcomparative%2Banalyses%2Bwith%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-7-327%26rft_id%253Dinfo%253Apmid%252F17194304%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-7-327&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17194304&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.7" data-doi="10.1093/bioinformatics/btp348"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Capella-Gutiérrez</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Silla-Martínez</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Gabaldón</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1972</span>–<span class="cit-lpage">1973</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp348</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp348%26rft_id%253Dinfo%253Apmid%252F19505945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp348&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505945&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268107100022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.8" data-doi="10.1038/s41477-018-0323-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Hao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Guang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Zhao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Wang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title"><em>Liriodendron</em> genome sheds light on angiosperm phylogeny and species–pair differentiation.</span> <abbr class="cit-jnl-abbrev">Nat. Plants</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">18</span>–<span class="cit-lpage">25</span>. Erratum: 328.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41477-018-0323-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BPlants%26rft.volume%253D5%26rft.spage%253D18%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41477-018-0323-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41477-018-0323-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.9" data-doi="10.1007/s11105-013-0563-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Song</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">He</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Zhang</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide analysis of gene expression in response to drought stress in <em>Populus simonii</em>.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol. Report.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">946</span>–<span class="cit-lpage">962</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11105-013-0563-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMol.%2BBiol.%2BReport.%26rft.volume%253D31%26rft.spage%253D946%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11105-013-0563-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11105-013-0563-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.10" data-doi="10.1038/nmeth.2474"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>,  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. H.</span> <span class="cit-name-surname">Alexander</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Marks</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Klammer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Drake</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">563</span>–<span class="cit-lpage">569</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.2474</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D563%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2474%26rft_id%253Dinfo%253Apmid%252F23644548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2474&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23644548&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319668700028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.11" data-doi="10.1038/nmeth.4035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>,  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Peluso</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. J.</span> <span class="cit-name-surname">Sedlazeck</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nattestad</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. T.</span> <span class="cit-name-surname">Concepcion</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Phased diploid genome assembly with single-molecule real-time sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1050</span>–<span class="cit-lpage">1054</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.4035</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D13%26rft.spage%253D1050%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4035%26rft_id%253Dinfo%253Apmid%252F27749838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.4035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27749838&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.12" data-doi="10.1038/cr.2014.83"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dai</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Hu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Cai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Feng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Ye</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The willow genome and divergent evolution from poplar after the common genome duplication.</span> <abbr class="cit-jnl-abbrev">Cell Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1274</span>–<span class="cit-lpage">1277</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/cr.2014.83</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRes.%26rft.volume%253D24%26rft.spage%253D1274%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2014.83%26rft_id%253Dinfo%253Apmid%252F24980958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2014.83&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24980958&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.13" data-doi="10.1093/bioinformatics/btl097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Bie</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Cristianini</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Demuth</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Hahn</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">CAFE: a computational tool for the study of gene family evolution.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1269</span>–<span class="cit-lpage">1271</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btl097</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl097%26rft_id%253Dinfo%253Apmid%252F16543274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btl097&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16543274&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237319300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.14" data-doi="10.1186/1471-2148-7-214"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drummond</span>,  <span class="cit-name-given-names">A. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Rambaut</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">BEAST: Bayesian evolutionary analysis by sampling trees.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">214</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2148-7-214</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DDrummond%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D7%26rft.spage%253D214%26rft.epage%253D214%26rft.atitle%253DBEAST%253A%2BBayesian%2Bevolutionary%2Banalysis%2Bby%2Bsampling%2Btrees.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-7-214%26rft_id%253Dinfo%253Apmid%252F17996036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-7-214&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17996036&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.455.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eckenwalder</span>,  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Systematics and evolution of Populus, pp</span>, pp. <span class="cit-fpage">7</span>–<span class="cit-lpage">32</span> in <span class="cit-source">Biology of Populus and its implications for management and conservation</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Stettler</span>,  <span class="cit-name-given-names">R. F.</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">H. D.</span> <span class="cit-name-surname">Bradshaw</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">P. E.</span> <span class="cit-name-surname">Heilman</span></span>, and <span class="cit-ed"> <span class="cit-name-given-names">T. M.</span> <span class="cit-name-surname">Hinckley</span></span>. <span class="cit-publ-name">NRC Research Press, National Council of Canada</span>, <span class="cit-publ-loc">Ottawa</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.16" data-doi="10.1038/s41588-019-0356-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edger</span>,  <span class="cit-name-given-names">P. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. J.</span> <span class="cit-name-surname">Poorten</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">VanBuren</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Hardigan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Colle</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Origin and evolution of the octoploid strawberry genome.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">541</span>–<span class="cit-lpage">547</span>. Erratum: 765.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41588-019-0356-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D51%26rft.spage%253D541%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-019-0356-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41588-019-0356-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.17" data-doi="10.1186/s13059-015-0721-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Emms</span>,  <span class="cit-name-given-names">D. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kelly</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">157</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-015-0721-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D157%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0721-2%26rft_id%253Dinfo%253Apmid%252F26243257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0721-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26243257&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.18" data-doi="10.1093/molbev/msx259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Emms</span>,  <span class="cit-name-given-names">D. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kelly</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">STRIDE: Species tree root inference from gene duplication events.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">3267</span>–<span class="cit-lpage">3278</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msx259</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsx259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msx259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-web" id="cit-10.2.455.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrison</span>,  <span class="cit-name-given-names">E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Marth</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> Haplotype-based variant detection from short-read sequencing. arXiv: 1207.3907.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.20" data-doi="10.1126/science.aat4096"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Winzer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Ning</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The opium poppy genome and morphinan production.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">362</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">347</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.aat4096</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aat4096%26rft_id%253Dinfo%253Apmid%252F30166436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjIvNjQxMi8zNDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzQ1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.21" data-doi="10.1038/nprot.2013.084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Alexie</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Moran</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Manfred</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. D.</span> <span class="cit-name-surname">Blood</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1494</span>–<span class="cit-lpage">1512</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nprot.2013.084</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc.%26rft.volume%253D8%26rft.spage%253D1494%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2013.084%26rft_id%253Dinfo%253Apmid%252F23845962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2013.084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23845962&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.22" data-doi="10.1186/s12859-015-0611-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heberle</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. V.</span> <span class="cit-name-surname">Meirelles</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. R.</span> <span class="cit-name-surname">da Silva</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. P.</span> <span class="cit-name-surname">Telles</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Minghim</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">InteractiVenn: a web-based tool for the analysis of sets through Venn diagrams.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">169</span>–<span class="cit-lpage">175</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-015-0611-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D16%26rft.spage%253D169%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-015-0611-3%26rft_id%253Dinfo%253Apmid%252F25994840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-015-0611-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25994840&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.23" data-doi="10.1186/1471-2105-12-491"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>,  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">MAKER2: an annotation pipeline and genome-database management tool for second-generation genome projects.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">504</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-491</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHolt%26rft.auinit1%253DC.%26rft.volume%253D12%26rft.spage%253D491%26rft.epage%253D491%26rft.atitle%253DMAKER2%253A%2Ban%2Bannotation%2Bpipeline%2Band%2Bgenome-database%2Bmanagement%2Btool%2Bfor%2Bsecond-generation%2Bgenome%2Bprojects.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-491%26rft_id%253Dinfo%253Apmid%252F22192575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-491&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22192575&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.24" data-doi="10.1073/pnas.1604828113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Itkin</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Davidovich-Rikanati</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Cohen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Portnoy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Doron-Faigenboim</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The biosynthetic pathway of the nonsugar, high-intensity sweetener mogroside V from <em>Siraitia grosvenorii</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">E7619</span>–<span class="cit-lpage">E7628</span>. Erratum: E3862.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1604828113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1604828113%26rft_id%253Dinfo%253Apmid%252F27821754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEzLzQ3L0U3NjE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi80NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.25" data-doi="10.1093/nar/gkt263"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mistry</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. D.</span> <span class="cit-name-surname">Finn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Eddy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Alex</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Marco</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Challenges in homology search: HMMER3 and convergent evolution of coiled-coil regions.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-elocation-id">e121</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkt263</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt263%26rft_id%253Dinfo%253Apmid%252F23598997%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt263&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23598997&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.26" data-doi="10.1016/j.tplants.2019.05.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Winefield</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bombarely</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Prentis</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Waterhouse</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Tools and strategies for long-read sequencing and <em>de novo</em> assembly of plant genomes.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">700</span>–<span class="cit-lpage">724</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2019.05.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D24%26rft.spage%253D700%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2019.05.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2019.05.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.27" data-doi="10.1093/nar/gkx1038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kalvari</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Argasinska</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Quinones-Olvera</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. P.</span> <span class="cit-name-surname">Nawrocki</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Rivas</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Rfam 13.0: shifting to a genome-centric resource for non-coding RNA families.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">D335</span>–<span class="cit-lpage">D342</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkx1038</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D46%26rft.spage%253DD335%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1038%26rft_id%253Dinfo%253Apmid%252F29112718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx1038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29112718&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.28" data-doi="10.1093/molbev/mst010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katoh</span>,  <span class="cit-name-given-names">K.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Standley</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">772</span>–<span class="cit-lpage">780</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/mst010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst010%26rft_id%253Dinfo%253Apmid%252F23329690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23329690&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000317002300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.29" data-doi="10.1186/1471-2105-5-59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korf</span>,  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Gene finding in novel genomes.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">67</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-5-59</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DKorf%26rft.auinit1%253DI.%26rft.volume%253D5%26rft.spage%253D59%26rft.epage%253D59%26rft.atitle%253DGene%2Bfinding%2Bin%2Bnovel%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-5-59%26rft_id%253Dinfo%253Apmid%252F15144565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-5-59&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15144565&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.30" data-doi="10.1186/gb-2004-5-2-r12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurtz</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Phillippy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Delcher</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Smoot</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Shumway</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Versatile and open software for comparing large genomes.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">R12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2004-5-2-r12</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DKurtz%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253DR12%26rft.epage%253DR12%26rft.atitle%253DVersatile%2Band%2Bopen%2Bsoftware%2Bfor%2Bcomparing%2Blarge%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2004-5-2-r12%26rft_id%253Dinfo%253Apmid%252F14759262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2004-5-2-r12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14759262&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.31" data-doi="10.1093/nar/gkm160"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lagesen</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Hallin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Rødland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.-H.</span> <span class="cit-name-surname">Staerfeldt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Rognes</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">RNAmmer: consistent and rapid annotation of ribosomal RNA genes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">3100</span>–<span class="cit-lpage">3108</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkm160</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm160%26rft_id%253Dinfo%253Apmid%252F17452365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkm160&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17452365&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247350800027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.32" data-doi="10.1038/ng.3430"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lamichhaney</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Fan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Widemo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Gunnarsson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. S.</span> <span class="cit-name-surname">Thalmann</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Structural genomic changes underlie alternative reproductive strategies in the ruff (<em>Philomachus pugnax</em>).</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">84</span>–<span class="cit-lpage">88</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3430</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3430&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.33" data-doi="10.3390/genes9010006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Lin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">An</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Wang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Huang</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome sequencing and assembly by long reads in plants.</span> <abbr class="cit-jnl-abbrev">Genes</abbr> <span class="cit-vol">9</span>: <span class="cit-issue">6</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3390/genes9010006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fgenes9010006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/genes9010006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.34" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Durbin</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows–Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.35" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Handsaker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wysoker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Fennell</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ruan</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment-Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.36" data-doi="10.1073/pnas.1801437115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">Y.-C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Delhomme</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Schiffthaler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Sundström</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Functional and evolutionary genomic inferences in <em>Populus</em> through genome and population sequencing of American and European aspen.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">E10970</span>–<span class="cit-lpage">E10978</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1801437115</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1801437115%26rft_id%253Dinfo%253Apmid%252F30373829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMzoiMTE1LzQ2L0UxMDk3MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvNDU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.37" data-doi="10.1371/journal.pone.0036146"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Gong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Dai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Kang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Huang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genome of <em>Ganoderma lucidum</em> provides insights into triterpenes biosynthesis and wood degradation.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e36146</span>. Erratum<span class="cit-pub-id-sep cit-pub-id-doi-sep">: </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/annotation/9f521140-7406-425a-bc90-d0f6075dc854</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0036146</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLiu%26rft.auinit1%253DD.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De36146%26rft.epage%253De36146%26rft.atitle%253DThe%2Bgenome%2Bof%2BGanoderma%2Blucidum%2Bprovides%2Binsights%2Binto%2Btriterpenes%2Bbiosynthesis%2Band%2Bwood%2Bdegradation%2B%255Bcorrected%255D.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0036146%26rft_id%253Dinfo%253Apmid%252F22567134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0036146&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22567134&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.38" data-doi="10.1007/s11427-018-9455-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y.-J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.-R.</span> <span class="cit-name-surname">Wang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.-Y.</span> <span class="cit-name-surname">Zeng</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">De novo assembly of white poplar genome and genetic diversity of white poplar population in Irtysh River basin in China.</span> <abbr class="cit-jnl-abbrev">Sci. China Life Sci.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">609</span>–<span class="cit-lpage">618</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11427-018-9455-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BChina%2BLife%2BSci.%26rft.volume%253D62%26rft.spage%253D609%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11427-018-9455-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11427-018-9455-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Ge</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Li</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Two highly similar poplar paleo-subgenomes suggest an autotetraploid ancestor of <em>Salicaceae</em> plants.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">571</span>–<span class="cit-lpage">581</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D8%26rft.spage%253D571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.40" data-doi="10.1093/nar/25.5.955"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>,  <span class="cit-name-given-names">T. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Eddy</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">955</span>–<span class="cit-lpage">964</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/25.5.955</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.5.955%26rft_id%253Dinfo%253Apmid%252F9023104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/25.5.955&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9023104&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.41" data-doi="10.1111/pbi.12989"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Wan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Duan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Bai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Bai</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Genome sequence and genetic transformation of a widely distributed and cultivated poplar.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">460</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbi.12989</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12989&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.42" data-doi="10.1038/ncomms3797"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Zhou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Yue</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Hu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic insights into salt adaptation in a desert poplar.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">2797</span>–<span class="cit-lpage">2806</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms3797</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D4%26rft.spage%253D2797%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms3797%26rft_id%253Dinfo%253Apmid%252F24256998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms3797&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24256998&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.43" data-doi="10.1093/bioinformatics/btr011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marçais</span>,  <span class="cit-name-given-names">G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Kingsford</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A fast, lock-free approach for efficient parallel counting of occurrences of k-mers.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">764</span>–<span class="cit-lpage">770</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr011%26rft_id%253Dinfo%253Apmid%252F21217122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21217122&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288277300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.44" data-doi="10.1093/nar/gkl200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stanke</span>.  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Keller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Gunduz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Hayes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Waack</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">AUGUSTUS: ab initio prediction of alternative transcripts.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">W435</span>–<span class="cit-lpage">W439</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkl200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DStanke%26rft.auinit1%253DM.%26rft.volume%253D34%26rft.issue%253DWeb%2BServer%2Bissue%26rft.spage%253DW435%26rft.epage%253DW439%26rft.atitle%253DAUGUSTUS%253A%2Bab%2Binitio%2Bprediction%2Bof%2Balternative%2Btranscripts.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl200%26rft_id%253Dinfo%253Apmid%252F16845043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkl200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16845043&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245650200087&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.45" data-doi="10.1038/s41467-018-03016-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michael</span>,  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Jupe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Bemm</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. T.</span> <span class="cit-name-surname">Motley</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Sandoval</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">High contiguity <em>Arabidopsis thaliana</em> genome assembly with a single nanopore flow cell.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">541</span>–<span class="cit-lpage">548</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41467-018-03016-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D9%26rft.spage%253D541%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-018-03016-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-018-03016-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.46" data-doi="10.1534/g3.119.400030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Minio</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Massonnet</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Figueroa-Balderas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Castro</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Cantu</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Diploid genome assembly of the wine grape Carménère.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1331</span>–<span class="cit-lpage">1337</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.119.400030</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.119.400030%26rft_id%253Dinfo%253Apmid%252F30923135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjkvNS8xMzMxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi80NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.47" data-doi="10.1186/s13040-017-0158-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Monroe</span>,  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. A.</span> <span class="cit-name-surname">Allen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Tanger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Mullen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. T.</span> <span class="cit-name-surname">Lovell</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">TSPmap, a tool making use of traveling salesperson problem solvers in the efficient and accurate construction of high-density genetic linkage maps.</span> <abbr class="cit-jnl-abbrev">BioData Min.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">38</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13040-017-0158-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioData%2BMin.%26rft.volume%253D10%26rft.spage%253D38%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13040-017-0158-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13040-017-0158-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.48" data-doi="10.1186/s12864-016-3003-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mousavi</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Tong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Tao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">De novo SNP discovery and genetic linkage mapping in poplar using restriction site associated DNA and whole-genome sequencing technologies.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">656</span>–<span class="cit-lpage">667</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-016-3003-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D656%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-016-3003-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-016-3003-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.49" data-doi="10.1038/nature12211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nystedt</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. R.</span> <span class="cit-name-surname">Street</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wetterbom</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Zuccolo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.-C.</span> <span class="cit-name-surname">Lin</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The Norway spruce genome sequence and conifer genome evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">497</span>: <span class="cit-fpage">579</span>–<span class="cit-lpage">584</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature12211</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D497%26rft.spage%253D579%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12211%26rft_id%253Dinfo%253Apmid%252F23698360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23698360&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319556100035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.50" data-doi="10.1093/nar/27.1.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ogata</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Goto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Sato</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Fujibuchi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Bono</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">KEGG: Kyoto Encyclopedia of Genes and Genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">34</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/27.1.29</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D27%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F27.1.29%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/27.1.29&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000077983000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.51" data-doi="10.1007/s11295-010-0297-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paolucci</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Gaudet</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Jorge</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Beritognolo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Terzoli</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic linkage maps of <em>Populus alba</em> L. and comparative mapping analysis of sex determination across <em>Populus</em> species.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">863</span>–<span class="cit-lpage">875</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11295-010-0297-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D6%26rft.spage%253D863%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-010-0297-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-010-0297-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000284421100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.52" data-doi="10.1371/journal.pone.0030619"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>,  <span class="cit-name-given-names">R. K.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Mukesh</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">NGS QC Toolkit: a toolkit for quality control of next generation sequencing data.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e30619</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0030619</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPatel%26rft.auinit1%253DR.%2BK.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De30619%26rft.epage%253De30619%26rft.atitle%253DNGS%2BQC%2BToolkit%253A%2Ba%2Btoolkit%2Bfor%2Bquality%2Bcontrol%2Bof%2Bnext%2Bgeneration%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0030619%26rft_id%253Dinfo%253Apmid%252F22312429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0030619&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22312429&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.53" data-doi="10.1007/BF02772108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Porebski</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. G.</span> <span class="cit-name-surname">Bailey</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. R.</span> <span class="cit-name-surname">Baum</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Modification of a CTAB DNA extraction protocol for plants containing high polysaccharide and polyphenol components.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol. Report.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">8</span>–<span class="cit-lpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02772108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMol.%2BBiol.%2BReport.%26rft.volume%253D15%26rft.spage%253D8%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02772108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02772108&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997WT59300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.54" data-doi="10.1093/bioinformatics/19.2.301"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanderson</span>,  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">r8s: inferring absolute rates of molecular evolution and divergence times in the absence of a molecular clock.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">302</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/19.2.301</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DSanderson%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D19%26rft.issue%253D2%26rft.spage%253D301%26rft.epage%253D302%26rft.atitle%253Dr8s%253A%2Binferring%2Babsolute%2Brates%2Bof%2Bmolecular%2Bevolution%2Band%2Bdivergence%2Btimes%2Bin%2Bthe%2Babsence%2Bof%2Ba%2Bmolecular%2Bclock%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F19.2.301%26rft_id%253Dinfo%253Apmid%252F12538260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/19.2.301&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12538260&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180913600026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.55" data-doi="10.1038/s41467-018-07876-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Ma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Liu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Chromosome conformation capture resolved near complete genome assembly of broomcorn millet.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">464</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41467-018-07876-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D10%26rft.spage%253D464%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-018-07876-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-018-07876-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.56" data-doi="10.1093/bioinformatics/btv351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simão</span>,  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Waterhouse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Panagiotis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. V.</span> <span class="cit-name-surname">Kriventseva</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Zdobnov</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3210</span>–<span class="cit-lpage">3212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv351</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv351%26rft_id%253Dinfo%253Apmid%252F26059717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26059717&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.57" data-doi="10.1186/1471-2105-6-31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slater</span>,  <span class="cit-name-given-names">G. S. C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Birney</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Automated generation of heuristics for biological sequence comparison.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">41</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-6-31</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DSlater%26rft.auinit1%253DG.%2BS.%26rft.volume%253D6%26rft.spage%253D31%26rft.epage%253D31%26rft.atitle%253DAutomated%2Bgeneration%2Bof%2Bheuristics%2Bfor%2Bbiological%2Bsequence%2Bcomparison.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-6-31%26rft_id%253Dinfo%253Apmid%252F15713233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-6-31&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15713233&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.58" data-doi="10.1016/j.molp.2018.10.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Song</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Dong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Zhao</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The <em>Chrysanthemum nankingense</em> genome provides insights into the evolution and diversification of chrysanthemum flowers and medicinal traits.</span> <abbr class="cit-jnl-abbrev">Mol. Plant</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1482</span>–<span class="cit-lpage">1491</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molp.2018.10.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%26rft.volume%253D11%26rft.spage%253D1482%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molp.2018.10.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molp.2018.10.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.59" data-doi="10.1007/s00299-013-1454-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Ci</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Zhang</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Transcriptome profiling reveals differential transcript abundance in response to chilling stress in <em>Populus simonii</em>.</span> <abbr class="cit-jnl-abbrev">Plant Cell Rep.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1407</span>–<span class="cit-lpage">1425</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00299-013-1454-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BRep.%26rft.volume%253D32%26rft.spage%253D1407%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00299-013-1454-x%26rft_id%253Dinfo%253Apmid%252F23652820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00299-013-1454-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23652820&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.60" data-doi="10.1093/bioinformatics/btu033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>,  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1312</span>–<span class="cit-lpage">1313</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24451623&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000336095100024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.61" data-doi="10.1186/s13059-014-0573-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Miao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Ming</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">ALLMAPS: robust scaffold ordering based on multiple maps.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">17</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-014-0573-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0573-1%26rft_id%253Dinfo%253Apmid%252F25583564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0573-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25583564&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.62" data-doi="10.1002/0471250953.bi0410s25"><div class="cit-metadata"><cite>Tarailo-Graovac, M., and N. Chen, 2009 Using RepeatMasker to identify repetitive elements in genomic sequences. Current Protocols in Bioinformatics 25: 4.10.11–14.10.14.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/0471250953.bi0410s25</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCurr%2BProtoc%2BBioinformatics%26rft.aulast%253DTarailo-Graovac%26rft.auinit1%253DM.%26rft.volume%253DChapter%2B4%26rft.spage%253DUnit%2B4.10%26rft.epage%253DUnit%2B4.10%26rft.atitle%253DUsing%2BRepeatMasker%2Bto%2Bidentify%2Brepetitive%2Belements%2Bin%2Bgenomic%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F0471250953.bi0410s25%26rft_id%253Dinfo%253Apmid%252F19274634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/0471250953.bi0410s25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19274634&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.63" data-doi="10.1093/nar/29.1.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tatusov</span>,  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Natale</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. V.</span> <span class="cit-name-surname">Garkavtsev</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. A.</span> <span class="cit-name-surname">Tatusova</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U. T.</span> <span class="cit-name-surname">Shankavaram</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The COG database: new developments in phylogenetic classification of proteins from complete genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">22</span>–<span class="cit-lpage">28</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/29.1.22</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DTatusov%26rft.auinit1%253DR.%2BL.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D22%26rft.epage%253D28%26rft.atitle%253DThe%2BCOG%2Bdatabase%253A%2Bnew%2Bdevelopments%2Bin%2Bphylogenetic%2Bclassification%2Bof%2Bproteins%2Bfrom%2Bcomplete%2Bgenomes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F29.1.22%26rft_id%253Dinfo%253Apmid%252F11125040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/29.1.22&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11125040&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166360300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.64" data-doi="10.1101/gr.081612.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ter-Hovhannisyan</span>,  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Lomsadze</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. O.</span> <span class="cit-name-surname">Chernoff</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Borodovsky</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Gene prediction in novel fungal genomes using an ab initio algorithm with unsupervised training.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1979</span>–<span class="cit-lpage">1990</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.081612.108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.081612.108%26rft_id%253Dinfo%253Apmid%252F18757608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMi8xOTc5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi80NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.65" data-doi="10.1371/journal.pone.0150692"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ou</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Construction of high-density linkage maps of <em>Populus deltoides</em> × <em>P. simonii</em> using restriction-site associated DNA sequencing.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e0150692</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0150692</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0150692%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0150692%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0150692&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.66" data-doi="10.1126/science.1128691"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tuskan</span>,  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Difazio</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Jansson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bohlmann</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Grigoriev</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The genome of black cottonwood, <em>Populus trichocarpa</em> (Torr. &amp; Gray).</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">313</span>: <span class="cit-fpage">1596</span>–<span class="cit-lpage">1604</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1128691</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTuskan%26rft.auinit1%253DG.%2BA.%26rft.volume%253D313%26rft.issue%253D5793%26rft.spage%253D1596%26rft.epage%253D1604%26rft.atitle%253DThe%2Bgenome%2Bof%2Bblack%2Bcottonwood%252C%2BPopulus%2Btrichocarpa%2B%2528Torr.%2B%2526amp%253B%2BGray%2529.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1128691%26rft_id%253Dinfo%253Apmid%252F16973872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTMvNTc5My8xNTk2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi80NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.67" data-doi="10.1093/gigascience/gix120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanBuren</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Wai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Colle</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sullivan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">A near complete, chromosome-scale assembly of the black raspberry (<em>Rubus occidentalis</em>) genome.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gigascience/giy094</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgigascience%252Fgix120%26rft_id%253Dinfo%253Apmid%252F29300887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gigascience/gix120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29300887&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.68" data-doi="10.1093/bioinformatics/btx153"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vurture</span>,  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. J.</span> <span class="cit-name-surname">Sedlazeck</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nattestad</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Underwood</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Fang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">GenomeScope: fast reference-free genome profiling from short reads.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">2202</span>–<span class="cit-lpage">2204</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btx153</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D33%26rft.spage%253D2202%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtx153%26rft_id%253Dinfo%253Apmid%252F28369201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btx153&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28369201&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.69" data-doi="10.1371/journal.pone.0112963"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Abeel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Shea</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Priest</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Abouelliel</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Pilon: An integrated tool for comprehensive microbial variant detection and genome assembly improvement.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e112963</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0112963</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De112963%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0112963%26rft_id%253Dinfo%253Apmid%252F25409509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0112963&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25409509&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.70" data-doi="10.1371/journal.pone.0103645"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Z. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. H.</span> <span class="cit-name-surname">Du</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Dayanandan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. S.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. F.</span> <span class="cit-name-surname">Zeng</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Phylogeny reconstruction and hybrid analysis of <em>Populus</em> (Salicaceae) based on nucleotide sequences of multiple single-copy nuclear genes and plastid fragments.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e103645</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0103645</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De103645%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0103645%26rft_id%253Dinfo%253Apmid%252F25116432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0103645&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25116432&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.71" data-doi="10.1007/s11105-012-0527-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Du</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Li</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Zhang</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic diversity and population structure in Chinese indigenous poplar (<em>Populus simonii</em>) populations using microsatellite markers.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol. Report.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">620</span>–<span class="cit-lpage">632</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11105-012-0527-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMol.%2BBiol.%2BReport.%26rft.volume%253D31%26rft.spage%253D620%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11105-012-0527-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11105-012-0527-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winkler</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. G.</span> <span class="cit-name-surname">Roscito</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Sameith</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hiller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Myers</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The genome of the tegu lizard <em>Salvator merianae</em>: combining Illumina, PacBio, and optical mapping data to generate a highly contiguous assembly.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">giy141</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.73" data-doi="10.1371/journal.pgen.1000212"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. R.</span> <span class="cit-name-surname">Bhat</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. J.</span> <span class="cit-name-surname">Close</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Lonardi</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient and accurate construction of genetic linkage maps from the minimum spanning tree of a graph.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-elocation-id">e1000212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000212</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWu%26rft.auinit1%253DY.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253De1000212%26rft.epage%253De1000212%26rft.atitle%253DEfficient%2Band%2Baccurate%2Bconstruction%2Bof%2Bgenetic%2Blinkage%2Bmaps%2Bfrom%2Bthe%2Bminimum%2Bspanning%2Btree%2Bof%2Ba%2Bgraph.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000212%26rft_id%253Dinfo%253Apmid%252F18846212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000212&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18846212&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.74" data-doi="10.1093/gigascience/giy164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">C.-Q.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.-S.</span> <span class="cit-name-surname">Zhou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.-X.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Zhao</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Genome sequence of <em>Malania oleifera</em>, a tree with great value for nervonic acid production.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">giy164</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gigascience/giy164</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D8%26rft.spage%253Dgiy164%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgigascience%252Fgiy164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gigascience/giy164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.75" data-doi="10.1038/s41588-019-0427-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Gao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Gui</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Chen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Genome assembly of a tropical maize inbred line provides insights into structural variation and crop improvement.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">1052</span>–<span class="cit-lpage">1059</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41588-019-0427-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D51%26rft.spage%253D1052%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-019-0427-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41588-019-0427-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.76" data-doi="10.1093/gigascience/giw015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Liu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The draft genome sequence of a desert tree <em>Populus pruinosa</em>.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gigascience/gix075</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D6%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgigascience%252Fgiw015%26rft_id%253Dinfo%253Apmid%252F29092041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gigascience/giw015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29092041&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.77" data-doi="10.1038/s41438-018-0108-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Yue</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Ding</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Chen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The chromosome-level quality genome provides insights into the evolution of the biosynthesis genes for aroma compounds of <em>Osmanthus fragrans</em>.</span> <abbr class="cit-jnl-abbrev">Hortic. Res.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">72</span>–<span class="cit-lpage">84</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41438-018-0108-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHortic.%2BRes.%26rft.volume%253D5%26rft.spage%253D72%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41438-018-0108-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41438-018-0108-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.78" data-doi="10.1093/bib/bby114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Wu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Gao</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">gmRAD: an integrated SNP calling pipeline for genetic mapping with RADseq across a hybrid population.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> bby:114.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bib/bby114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbby114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bby114&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.79" data-doi="10.1093/dnares/dsy013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yasodha</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Vasudeva</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Balakrishnan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Sakthi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Abel</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Draft genome of a high value tropical timber tree, Teak (<em>Tectona grandis</em> L. f): insights into SSR diversity, phylogeny and conservation.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">409</span>–<span class="cit-lpage">419</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/dnares/dsy013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft.volume%253D25%26rft.spage%253D409%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsy013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/dnares/dsy013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.80" data-doi="10.1093/nar/gky400"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Cui</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">WEGO 2.0: a web tool for analyzing and plotting GO annotations, 2018 update.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">W71</span>–<span class="cit-lpage">W75</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gky400</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D46%26rft.spage%253DW71%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky400%26rft_id%253Dinfo%253Apmid%252F29788377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky400&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29788377&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.81" data-doi="10.1139/g02-013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Huang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Zhuge</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Molecular linkage maps of the <em>Populus</em> genome.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">541</span>–<span class="cit-lpage">555</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/g02-013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DYin%26rft.auinit1%253DT.%26rft.volume%253D45%26rft.issue%253D3%26rft.spage%253D541%26rft.epage%253D555%26rft.atitle%253DMolecular%2Blinkage%2Bmaps%2Bof%2Bthe%2BPopulus%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg02-013%26rft_id%253Dinfo%253Apmid%252F12033623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g02-013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12033623&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.82" data-doi="10.1093/bioinformatics/17.9.847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zdobnov</span>,  <span class="cit-name-given-names">E. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Apweiler</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">InterProScan—an integration platform for the signature-recognition methods in InterPro.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">847</span>–<span class="cit-lpage">848</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/17.9.847</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DZdobnov%26rft.auinit1%253DE.%2BM.%26rft.volume%253D17%26rft.issue%253D9%26rft.spage%253D847%26rft.epage%253D848%26rft.atitle%253DInterProScan%2B-%2Ban%2Bintegration%2Bplatform%2Bfor%2Bthe%2Bsignature-recognition%2Bmethods%2Bin%2BInterPro%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F17.9.847%26rft_id%253Dinfo%253Apmid%252F11590104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/17.9.847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11590104&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171566400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.83" data-doi="10.1101/409136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Bai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Xu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Improved high-quality genome assembly and annotation of Qingke, Tibetan hulless barley.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1101/409136</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F409136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI0MDkxMzZ2MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvNDU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.84" data-doi="10.1007/s11295-009-0207-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Tong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Yin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Zhuge</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Detection of quantitative trait loci influencing growth trajectories of adventitious roots in <em>Populus</em> using functional mapping.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">539</span>–<span class="cit-lpage">552</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11295-009-0207-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D5%26rft.spage%253D539%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-009-0207-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-009-0207-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.85" data-doi="10.1007/s00122-003-1478-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Yang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Li</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genetic mapping in (<em>Populus tomentosa</em> × <em>Populus bolleana</em>) and <em>P. tomentosa</em> Carr. using AFLP markers.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">657</span>–<span class="cit-lpage">662</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-003-1478-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DD.%26rft.volume%253D108%26rft.issue%253D4%26rft.spage%253D657%26rft.epage%253D662%26rft.atitle%253DGenetic%2Bmapping%2Bin%2B%2528Populus%2Btomentosa%2Bx%2BPopulus%2Bbolleana%2529%2Band%2BP.%2Btomentosa%2BCarr.%2Busing%2BAFLP%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-003-1478-7%26rft_id%253Dinfo%253Apmid%252F14564399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-003-1478-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14564399&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.86" data-doi="10.1038/s41438-018-0071-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Cai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Grob</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Improved <em>Brassica rapa</em> reference genome by single-molecule sequencing and chromosome conformation capture technologies.</span> <abbr class="cit-jnl-abbrev">Hortic. Res.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">50</span>–<span class="cit-lpage">60</span>. Erratum: 124.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41438-018-0071-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHortic.%2BRes.%26rft.volume%253D5%26rft.spage%253D50%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41438-018-0071-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41438-018-0071-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.87" data-doi="10.3390/f9090516"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Tian</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Nie</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Variation of traits on seeds and germination derived from the hybridization between the sections Tacamahaca and Aigeiros of the genus <em>Populus</em>.</span> <abbr class="cit-jnl-abbrev">Forests</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">516</span>–<span class="cit-lpage">530</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3390/f9090516</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DForests%26rft.volume%253D9%26rft.spage%253D516%26rft_id%253Dinfo%253Adoi%252F10.3390%252Ff9090516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/f9090516&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.88" data-doi="10.3389/fpls.2019.00005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zong</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. H.</span> <span class="cit-name-surname">Gan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Zhou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X. L.</span> <span class="cit-name-surname">Zou</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Plastome sequences help to resolve deep-level relationships of <em>Populus</em> in the Family Salicaceae.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">5</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2019.00005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D10%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2019.00005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2019.00005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.455.89" data-doi="10.1093/bioinformatics/bty915"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zwaenepoel</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Van de Peer</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">wgd—simple command line tools for the analysis of ancient whole-genome duplications.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">2153</span>–<span class="cit-lpage">2155</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bty915</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D35%26rft.spage%253D2153%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty915&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/2/455.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/2/443" title="Characterization of Embryonic Skin Transcriptome in Anser cygnoides at Three Feather Follicles Developmental Stages" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/2/467" title="CRISPR/Cas9-Mediated Integration of Large Transgene into Pig CEP112 Locus" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/2"><span>Volume 10 Issue 2, February 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/2.cover-source.jpg?itok=LSawHOSr" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/455" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-xwKdwhKstBb39wsJD62M6YFAWZe0KneJsiPN5c6Lsv8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genome-report" class="" data-icon-position="" data-hide-link-title="0">Genome Report</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/2/455" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/2/455" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/2/455" class="active" data-icon-position="" data-hide-link-title="0">De Novo Genome Assembly of Populus simonii Further Supports That Populus simonii and Populus trichocarpa Belong to Different Sections</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495551" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-WxB8FRLmlD8kDXqPtJ7ZmtCKx1_Fg10s74gMzm3TlX0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/2/455.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/2/455" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-WX5_fyu0RqBW1tk0bjXYm2du3z0JNYj60aUOSqxX5oY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/2/455" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/2/455.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495551" id="node495551--2" data-pisa="ggg;10/2/455" data-pisa-master="ggg;g3.119.400913" data-apath="/ggg/10/2/455.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>De Novo</em> Genome Assembly of <em>Populus simonii</em> Further Supports That <em>Populus simonii</em> and <em>Populus trichocarpa</em> Belong to Different Sections</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-2890-9702" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hainan Wu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5937-3467" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dan Yao</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-4530-8653" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yuhua Chen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-5191-828X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Wenguo Yang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-7130-7670" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Wei Zhao</span>, <span class="highwire-citation-author" data-delta="5">Hua Gao</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-9795-211X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Chunfa Tong</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">455-466;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400913 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495551--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hainan Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Forest Genetics & Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hainan%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHainan%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2890-9702" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hainan Wu</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dan Yao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Forest Genetics & Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dan%2BYao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yao%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADan%2BYao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5937-3467" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dan Yao</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yuhua Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Forest Genetics & Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yuhua%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYuhua%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4530-8653" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yuhua Chen</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Wenguo Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Forest Genetics & Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wenguo%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWenguo%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5191-828X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Wenguo Yang</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Wei Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Forest Genetics & Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wei%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWei%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7130-7670" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Wei Zhao</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Hua Gao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Forest Genetics & Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hua%2BGao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gao%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHua%2BGao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Chunfa Tong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Forest Genetics & Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chunfa%2BTong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tong%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChunfa%2BTong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9795-211X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Chunfa Tong</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tongchf@njfu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">tongchf@njfu.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/2/455" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/2/455&amp;title=De%20Novo%20Genome%20Assembly%20of%20Populus%20simonii%20Further%20Supports%20That%20Populus%20simonii%20and%20Populus%20trichocarpa%20Belong%20to%20Different%20Sections" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/2/455&amp;title=De%20Novo%20Genome%20Assembly%20of%20Populus%20simonii%20Further%20Supports%20That%20Populus%20simonii%20and%20Populus%20trichocarpa%20Belong%20to%20Different%20Sections" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/2/455&amp;title=De%20Novo%20Genome%20Assembly%20of%20Populus%20simonii%20Further%20Supports%20That%20Populus%20simonii%20and%20Populus%20trichocarpa%20Belong%20to%20Different%20Sections" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/2/455&amp;text=De%20Novo%20Genome%20Assembly%20of%20Populus%20simonii%20Further%20Supports%20That%20Populus%20simonii%20and%20Populus%20trichocarpa%20Belong%20to%20Different%20Sections" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/2/455&amp;title=De%20Novo%20Genome%20Assembly%20of%20Populus%20simonii%20Further%20Supports%20That%20Populus%20simonii%20and%20Populus%20trichocarpa%20Belong%20to%20Different%20Sections" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/2/455&amp;t=De%20Novo%20Genome%20Assembly%20of%20Populus%20simonii%20Further%20Supports%20That%20Populus%20simonii%20and%20Populus%20trichocarpa%20Belong%20to%20Different%20Sections" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/2/455&amp;title=De%20Novo%20Genome%20Assembly%20of%20Populus%20simonii%20Further%20Supports%20That%20Populus%20simonii%20and%20Populus%20trichocarpa%20Belong%20to%20Different%20Sections" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/2/455&amp;title=De%20Novo%20Genome%20Assembly%20of%20Populus%20simonii%20Further%20Supports%20That%20Populus%20simonii%20and%20Populus%20trichocarpa%20Belong%20to%20Different%20Sections" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495551/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495551" id="node495551--3" data-pisa="ggg;10/2/455" data-pisa-master="ggg;g3.119.400913" data-apath="/ggg/10/2/455.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>De Novo</em> Genome Assembly of <em>Populus simonii</em> Further Supports That <em>Populus simonii</em> and <em>Populus trichocarpa</em> Belong to Different Sections</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-2890-9702" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hainan Wu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5937-3467" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dan Yao</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-4530-8653" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yuhua Chen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-5191-828X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Wenguo Yang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-7130-7670" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Wei Zhao</span>, <span class="highwire-citation-author" data-delta="5">Hua Gao</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-9795-211X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Chunfa Tong</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">455-466;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400913 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495551--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hainan Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Forest Genetics & Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hainan%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHainan%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2890-9702" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hainan Wu</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dan Yao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Forest Genetics & Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dan%2BYao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yao%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADan%2BYao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5937-3467" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dan Yao</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yuhua Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Forest Genetics & Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yuhua%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYuhua%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4530-8653" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yuhua Chen</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Wenguo Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Forest Genetics & Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wenguo%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWenguo%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5191-828X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Wenguo Yang</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Wei Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Forest Genetics & Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wei%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWei%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7130-7670" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Wei Zhao</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Hua Gao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Forest Genetics & Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hua%2BGao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gao%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHua%2BGao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Chunfa Tong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Forest Genetics & Biotechnology of Ministry of Education, Co-Innovation Center for Sustainable Forestry in Southern China, Nanjing Forestry University, Nanjing 210037, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chunfa%2BTong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tong%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChunfa%2BTong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9795-211X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Chunfa Tong</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tongchf@njfu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">tongchf@njfu.edu.cn</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=455&amp;rft.epage=466&amp;rft.atitle=De%20Novo%20Genome%20Assembly%20of%20Populus%20simonii%20Further%20Supports%20That%20Populus%20simonii%20and%20Populus%20trichocarpa%20Belong%20to%20Different%20Sections&amp;rft.volume=10&amp;rft.issue=2&amp;rft.date=2020-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Wu%2C+Hainan&amp;rft.au=Yao%2C+Dan&amp;rft.au=Chen%2C+Yuhua&amp;rft.au=Yang%2C+Wenguo&amp;rft.au=Zhao%2C+Wei&amp;rft.au=Gao%2C+Hua&amp;rft.au=Tong%2C+Chunfa"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495551/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495551/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495551/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495551/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495551/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495551/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495551/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495551/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495551/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495551/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495551/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495551/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496044" id="node496044" data-pisa="ggg;g3.119.400871v1" data-pisa-master="ggg;g3.119.400871" data-apath="/ggg/early/2020/03/17/g3.119.400871.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400871" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the <em>indica</em> Rice Variety IR64 Using Linked-Read Sequencing and Nanopore Sequencing</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496040" id="node496040" data-pisa="ggg;g3.119.400893v1" data-pisa-master="ggg;g3.119.400893" data-apath="/ggg/early/2020/03/12/g3.119.400893.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/12/g3.119.400893" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Understanding the Genetic Domestication History of the Jianchang Duck by Genotyping and Sequencing of Genomic Genes Under Selection</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496037" id="node496037" data-pisa="ggg;g3.120.401071v1" data-pisa-master="ggg;g3.120.401071" data-apath="/ggg/early/2020/03/11/g3.120.401071.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/11/g3.120.401071" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the Meadow Brown Butterfly, <em>Maniola jurtina</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genome%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genome Report</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/2/455" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/2/455.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/2/455.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/2/455.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
