<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/2/505" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/2/505.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/2/505.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/2/505.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/10/2/505/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/2/505.atom" />
<meta name="HW.pisa" content="ggg;10/2/505" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Few Stickleback Suffice for the Transport of Alleles to New Lakes" />
<meta name="DC.Identifier" content="10.1534/g3.119.400564" />
<meta name="DC.Date" content="2020-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Galloway et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Threespine stickleback populations provide a striking example of local adaptation to divergent habitats in populations that are connected by recurrent gene flow. These small fish occur in marine and freshwater habitats throughout the Northern Hemisphere, and in numerous cases the smaller freshwater populations have been established “de novo” from marine colonists. Independently evolved freshwater populations exhibit similar phenotypes that have been shown to derive largely from the same standing genetic variants. Geographic isolation prevents direct migration between the freshwater populations, strongly suggesting that these shared locally adaptive alleles are transported through the marine population. However it is still largely unknown how gene flow, recombination, and selection jointly impact the standing variation that might fuel this adaptation. Here we use individual-based, spatially explicit simulations to determine the levels of gene flow that best match observed patterns of allele sharing among habitats in stickleback. We aim to better understand how gene flow and local adaptation in large metapopulations determine the speed of adaptation and re-use of standing genetic variation. In our simulations we find that repeated adaptation uses a shared set of alleles that are maintained at low frequency by migration-selection balance in oceanic populations. This process occurs over a realistic range of intermediate levels of gene flow that match previous empirical population genomic studies in stickleback. Examining these simulations more deeply reveals how lower levels of gene flow leads to slow, independent adaptation to different habitats, whereas higher levels of gene flow leads to significant mutation load – but an increased probability of successful population genomic scans for locally adapted alleles. Surprisingly, we find that the genealogical origins of most freshwater adapted alleles can be traced back to the original generation of marine individuals that colonized the lakes, as opposed to subsequent migrants. These simulations provide deeper context for existing studies of stickleback evolutionary genomics, and guidance for future empirical studies in this model. More broadly, our results support existing theory of local adaptation but extend it by more completely documenting the genealogical history of adaptive alleles in a metapopulation." />
<meta name="DC.Contributor" content="Jared Galloway" />
<meta name="DC.Contributor" content="William A. Cresko" />
<meta name="DC.Contributor" content="Peter Ralph" />
<meta name="article:published_time" content="2020-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Few Stickleback Suffice for the Transport of Alleles to New Lakes" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Threespine stickleback populations provide a striking example of local adaptation to divergent habitats in populations that are connected by recurrent gene flow. These small fish occur in marine and freshwater habitats throughout the Northern Hemisphere, and in numerous cases the smaller freshwater populations have been established “de novo” from marine colonists. Independently evolved freshwater populations exhibit similar phenotypes that have been shown to derive largely from the same standing genetic variants. Geographic isolation prevents direct migration between the freshwater populations, strongly suggesting that these shared locally adaptive alleles are transported through the marine population. However it is still largely unknown how gene flow, recombination, and selection jointly impact the standing variation that might fuel this adaptation. Here we use individual-based, spatially explicit simulations to determine the levels of gene flow that best match observed patterns of allele sharing among habitats in stickleback. We aim to better understand how gene flow and local adaptation in large metapopulations determine the speed of adaptation and re-use of standing genetic variation. In our simulations we find that repeated adaptation uses a shared set of alleles that are maintained at low frequency by migration-selection balance in oceanic populations. This process occurs over a realistic range of intermediate levels of gene flow that match previous empirical population genomic studies in stickleback. Examining these simulations more deeply reveals how lower levels of gene flow leads to slow, independent adaptation to different habitats, whereas higher levels of gene flow leads to significant mutation load – but an increased probability of successful population genomic scans for locally adapted alleles. Surprisingly, we find that the genealogical origins of most freshwater adapted alleles can be traced back to the original generation of marine individuals that colonized the lakes, as opposed to subsequent migrants. These simulations provide deeper context for existing studies of stickleback evolutionary genomics, and guidance for future empirical studies in this model. More broadly, our results support existing theory of local adaptation but extend it by more completely documenting the genealogical history of adaptive alleles in a metapopulation.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/02/01" />
<meta name="citation_mjid" content="ggg;10/2/505" />
<meta name="citation_id" content="10/2/505" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/2/505" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/2/505.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/2/505.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/2/505.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400564" />
<meta name="citation_pmid" content="31801796" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="505" />
<meta name="citation_lastpage" content="514" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jared Galloway" />
<meta name="citation_author_institution" content="Institute of Ecology and Evolution and" />
<meta name="citation_author" content="William A. Cresko" />
<meta name="citation_author_institution" content="Institute of Ecology and Evolution and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3496-8074" />
<meta name="citation_author" content="Peter Ralph" />
<meta name="citation_author_institution" content="Institute of Ecology and Evolution and" />
<meta name="citation_author_institution" content="Department of Mathematics, University of Oregon" />
<meta name="citation_author_email" content="plr@uoregon.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9459-6866" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=R. D. H. Barrett;citation_author=D. Schluter;citation_title=Adaptation from standing genetic variation.;citation_pages=38-44;citation_volume=23;citation_year=2008;citation_issue=1;citation_pmid=18006185;citation_doi=10.1016/j.tree.2007.09.008" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=N. Barton;citation_author=B. O. Bengtsson;citation_title=The barrier to genetic exchange between hybridising populations.;citation_pages=357-376;citation_volume=57;citation_year=1986;citation_pmid=3804765;citation_doi=10.1038/hdy.1986.135" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=N. H. Barton;citation_title=The role of hybridization in evolution.;citation_pages=551-568;citation_volume=10;citation_year=2001;citation_issue=3;citation_pmid=11298968;citation_doi=10.1046/j.1365-294x.2001.01216.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Bassham;citation_author=J. Catchen;citation_author=E. Lescak;citation_author=F. A. von Hippel;citation_author=W. A. Cresko;citation_title=Repeated selection of alternatively adapted haplotypes creates sweeping genomic remodeling in stickleback.;citation_volume=209;citation_year=2018;citation_pmid=29794240;citation_doi=10.1534/genetics.117.300610" />
<meta name="citation_reference" content="citation_title=The Evolutionary Biology of the Three Spine Sticklebacks;citation_pages=1;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=B. Charlesworth;citation_title=A big book and a small book on selection.;citation_pages=1337-1338;citation_volume=51;citation_year=1997;citation_doi=10.1111/j.1558-5646.1997.tb03983.x" />
<meta name="citation_reference" content="citation_journal_title=Annual Review of Ecology and Systematics;citation_journal_abbrev=Annu. Rev. Ecol. Syst.;citation_author=B. Charlesworth;citation_author=D. Charlesworth;citation_author=N. H. Barton;citation_title=The effects of genetic and geographic structure on neutral variation.;citation_pages=99-125;citation_volume=34;citation_year=2003;citation_doi=10.1146/annurev.ecolsys.34.011802.132359" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=L.-M. Chevin;citation_author=G. Decorzent;citation_author=T. Lenormand;citation_title=Niche dimensionality and the genetics of ecological speciation.;citation_pages=1244-1256;citation_volume=68;citation_year=2014;citation_pmid=24410181;citation_doi=10.1111/evo.12346" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. F. Colosimo;citation_author=K. E. Hosemann;citation_author=S. Balabhadra;citation_author=G. Villarreal;citation_author=M. Dickson;citation_title=Widespread Parallel Evolution in Sticklebacks by Repeated Fixation of Ectodysplasin Alleles;citation_pages=1928-1933;citation_volume=307;citation_year=2005;citation_issue=5717;citation_pmid=15790847;citation_doi=10.1126/science.1107239" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=P. F. Colosimo;citation_author=C. L. Peichel;citation_author=K. Nereng;citation_author=B. K. Blackman;citation_author=M. D. Shapiro;citation_title=The genetic architecture of parallel armor plate reduction in threespine sticklebacks.;citation_pages=E109-E109;citation_volume=2;citation_year=2004;citation_issue=5;citation_pmid=15069472;citation_doi=10.1371/journal.pbio.0020109" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=W. A. Cresko;citation_author=A. Amores;citation_author=C. Wilson;citation_author=J. Murphy;citation_author=M. Currey;citation_title=Parallel genetic basis for repeated evolution of armor loss in Alaskan threespine stickleback populations.;citation_volume=101;citation_year=2004;citation_pmid=15069186;citation_doi=10.1073/pnas.0308479101" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_journal_abbrev=Genetica;citation_author=W. A. Cresko;citation_author=K. L. McGuigan;citation_author=P. C. Phillips;citation_author=J. H. Postlethwait;citation_title=Studies of threespine stickleback developmental evolution: progress and promise.;citation_pages=105-126;citation_volume=129;citation_year=2007;citation_issue=1;citation_pmid=16897450;citation_doi=10.1007/s10709-006-0036-z" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=J. DeFaveri;citation_author=P. R. Jonsson;citation_author=J. Merilä;citation_title=Heterogeneous genomic differentiation in marine threespine sticklebacks: Adaptation along an environmental gradient.;citation_pages=2530-2546;citation_volume=67;citation_year=2013;citation_pmid=24033165;citation_doi=10.1111/evo.12097" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=J. DeFaveri;citation_author=T. Shikano;citation_author=Y. Shimada;citation_author=A. Goto;citation_author=J. Merilä;citation_title=Global analysis of genes involved in freshwater adaptation in threespine sticklebacks (Gasterosteus aculeatus).;citation_pages=1800-1807;citation_volume=65;citation_year=2011;citation_issue=6;citation_pmid=21644964;citation_doi=10.1111/j.1558-5646.2011.01247.x" />
<meta name="citation_reference" content="citation_journal_title=Monographs in population biology;citation_journal_abbrev=Monogr Popul Biol;citation_author=J. Endler;citation_title=Geographic variation, speciation, and clines.;citation_pages=1-246;citation_volume=10;citation_year=1977;citation_pmid=409931;citation_doi=10.2307/2259234" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=P. G. D. Feulner;citation_author=F. J. J. Chain;citation_author=M. Panchal;citation_author=Y. Huang;citation_author=C. Eizaguirre;citation_title=Genomics of Divergence along a Continuum of Parapatric Population Differentiation.;citation_pages=e1004966;citation_volume=11;citation_year=2015;citation_pmid=25679225;citation_doi=10.1371/journal.pgen.1004966" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=S. M. Flaxman;citation_title=Surfing downhill: when should population range expansion be characterized by reductions in fitness?;citation_pages=5963-5965;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12564" />
<meta name="citation_reference" content="citation_title=Population Genetics;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Math. Proc. Camb. Philos. Soc.;citation_author=J. B. S. Haldane;citation_title=A mathematical theory of natural and artificial selection, part V: Selection and mutation.;citation_pages=838-844;citation_volume=23;citation_year=1927;citation_doi=10.1017/S0305004100015644" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour;citation_author=B. C. Haller;citation_author=J. Galloway;citation_author=J. Kelleher;citation_author=P. W. Messer;citation_author=P. L. Ralph;citation_title=Tree-sequence recording in SLiM opens new horizons for forward-time simulation of whole genomes.;citation_year=2019;citation_doi=10.1111/1755-0998.12968" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=B. C. Haller;citation_author=P. W. Messer;citation_title=SLiM 2: Flexible, interactive forward genetic simulations.;citation_volume=34;citation_year=2017;citation_pmid=27702775;citation_doi=10.1093/molbev/msw211" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=B. C. Haller;citation_author=P. W. Messer;citation_title=SLiM 3: Forward genetic simulations beyond the Wright-Fisher model.;citation_volume=36;citation_year=2019;citation_doi=10.1093/molbev/msy228" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=D. Hanson;citation_author=R. D. Barrett;citation_author=A. P. Hendry;citation_title=Testing for parallel allochronic isolation in lake-stream stickleback.;citation_pages=47-57;citation_volume=29;citation_year=2016;citation_doi=10.1111/jeb.12761" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=D. Hanson;citation_author=J. Hu;citation_author=A. P. Hendry;citation_author=R. D. H. Barrett;citation_title=Heritable gene expression differences between lake and stream stickleback include both parallel and antiparallel components.;citation_pages=339-348;citation_volume=119;citation_year=2017;citation_doi=10.1038/hdy.2017.50" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=R. G. Harrison;citation_author=E. L. Larson;citation_title=Heterogeneous genome divergence, differential introgression, and the origin and structure of hybrid zones.;citation_pages=2454-2466;citation_volume=25;citation_year=2016;citation_doi=10.1111/mec.13582" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Hermisson;citation_author=P. S. Pennings;citation_title=Soft sweeps.;citation_volume=169;citation_year=2005;citation_pmid=15716498;citation_doi=10.1534/genetics.104.036947" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. Hirase;citation_author=H. Ozaki;citation_author=W. Iwasaki;citation_title=Parallel selection on gene copy number variations through evolution of three-spined stickleback genomes.;citation_pages=735;citation_volume=15;citation_year=2014;citation_pmid=25168270;citation_doi=10.1186/1471-2164-15-735" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=S. Hoban;citation_author=J. L. Kelley;citation_author=K. E. Lotterhos;citation_author=M. F. Antolin;citation_author=G. Bradburd;citation_title=Finding the genomic basis of local adaptation: Pitfalls, practical solutions, and future directions.;citation_pages=379-397;citation_volume=188;citation_year=2016;citation_pmid=27622873;citation_doi=10.1086/688018" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. A. Hohenlohe;citation_author=S. Bassham;citation_author=P. D. Etter;citation_author=N. Stiffler;citation_author=E. A. Johnson;citation_title=Population genomics of parallel adaptation in threespine stickleback using sequenced RAD tags.;citation_pages=e1000862-e1000862;citation_volume=6;citation_year=2010;citation_issue=2;citation_pmid=20195501;citation_doi=10.1371/journal.pgen.1000862" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=G. Hunt;citation_author=M. A. Bell;citation_author=M. P. Travis;citation_title=Evolution toward a new adaptive optimum: phenotypic evolution in a fossil stickleback lineage.;citation_pages=700-710;citation_volume=62;citation_year=2008;citation_issue=3;citation_pmid=18081713;citation_doi=10.1111/j.1558-5646.2007.00310.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=F. C. Jones;citation_author=M. G. Grabherr;citation_author=Y. F. Chan;citation_author=P. Russell;citation_author=E. Mauceli;citation_title=The genomic basis of adaptive evolution in threespine sticklebacks.;citation_pages=55-61;citation_volume=484;citation_year=2012;citation_issue=7392;citation_pmid=22481358;citation_doi=10.1038/nature10944" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=J. Kelleher;citation_author=A. M. Etheridge;citation_author=G. McVean;citation_title=Efficient coalescent simulation and genealogical analysis for large sample sizes.;citation_pages=e1004842;citation_volume=12;citation_year=2016;citation_pmid=27145223;citation_doi=10.1371/journal.pcbi.1004842" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=J. Kelleher;citation_author=K. R. Thornton;citation_author=J. Ashander;citation_author=P. L. Ralph;citation_title=Efficient pedigree recording for fast population genetics simulation.;citation_pages=e1006581;citation_volume=14;citation_year=2018;citation_doi=10.1371/journal.pcbi.1006581" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y.-T. Lai;citation_author=C. K. L. Yeung;citation_author=K. E. Omland;citation_author=E.-L. Pang;citation_author=Y. Hao;citation_title=Standing genetic variation as the predominant source for adaptation of a songbird.;citation_volume=116;citation_year=2019;citation_pmid=30659151;citation_doi=10.1073/pnas.1813597116" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=A. Lambert;citation_title=Probability of fixation under weak selection: a branching process unifying approach.;citation_pages=419-441;citation_volume=69;citation_year=2006;citation_issue=4;citation_pmid=16504230;citation_doi=10.1016/j.tpb.2006.01.002" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=E. A. Lescak;citation_author=S. L. Bassham;citation_author=J. Catchen;citation_author=O. Gelmond;citation_author=M. L. Sherbick;citation_title=Evolution of stickleback in 50 years on earthquake-uplifted islands.;citation_volume=112;citation_year=2015;citation_pmid=26668399;citation_doi=10.1073/pnas.1512020112" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=K. E. Lotterhos;citation_author=M. C. Whitlock;citation_title=The relative power of genome scans to detect local adaptation depends on sampling design and statistical method.;citation_pages=1031-1046;citation_volume=24;citation_year=2015;citation_pmid=25648189;citation_doi=10.1111/mec.13100" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=D. A. Marques;citation_author=K. Lucek;citation_author=J. I. Meier;citation_author=S. Mwaiko;citation_author=C. E. Wagner;citation_title=Genomics of rapid incipient speciation in sympatric threespine stickleback.;citation_pages=e1005887;citation_volume=12;citation_year=2016;citation_pmid=26925837;citation_doi=10.1371/journal.pgen.1005887" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. M. Maynard Smith;citation_author=J. Haigh;citation_title=The hitch-hiking effect of a favourable gene.;citation_pages=23-35;citation_volume=23;citation_year=1974;citation_issue=1;citation_pmid=4407212;citation_doi=10.1017/S0016672300014634" />
<meta name="citation_reference" content="citation_journal_title=Evolution Letters;citation_author=T. C. Nelson;citation_author=W. A. Cresko;citation_title=Ancient genomic variation underlies repeated ecological adaptation in young stickleback populations.;citation_pages=9-21;citation_volume=2;citation_year=2018;citation_doi=10.1002/evl3.37" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=T. C. Nelson;citation_author=M. R. Jones;citation_author=J. P. Velotta;citation_author=A. S. Dhawanjewar;citation_author=R. M. Schweizer;citation_title=Unveiling connections between genotype, phenotype, and fitness in natural populations.;citation_pages=1866-1876;citation_volume=28;citation_year=2019;citation_doi=10.1111/mec.15067" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=P. Nosil;citation_author=D. J. Funk;citation_author=D. Ortiz-Barrientos;citation_title=Divergent selection and heterogeneous genomic divergence.;citation_pages=375-402;citation_volume=18;citation_year=2009;citation_issue=3;citation_pmid=19143936;citation_doi=10.1111/j.1365-294X.2008.03946.x" />
<meta name="citation_reference" content="citation_title=Quantitative genetic variation and heritability;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=H. A. Orr;citation_title=The genetic theory of adaptation: a brief history.;citation_pages=119-127;citation_volume=6;citation_year=2005;citation_issue=2;citation_pmid=15716908;citation_doi=10.1038/nrg1523" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=D. Petry;citation_title=The effect on neutral gene flow of selection at a linked locus.;citation_pages=300-313;citation_volume=23;citation_year=1983;citation_issue=3;citation_pmid=6623407;citation_doi=10.1016/0040-5809(83)90020-5" />
<meta name="citation_reference" content="citation_journal_title=Scientific American;citation_journal_abbrev=Scientific American;citation_author=J. K. Pritchard;citation_title=Talk to the machine. Speech-recognition programs are no longer clumsy exercises in futility.;citation_pages=40-40;citation_volume=303;citation_year=2010;citation_issue=6;citation_pmid=21141356;citation_doi=10.1038/scientificamerican1010-40" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=P. L. Ralph;citation_author=G. Coop;citation_title=The role of standing variation in geographic convergent adaptation.;citation_pages=S5-S23;citation_volume=186;citation_year=2015;citation_doi=10.1086/682948" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=M. Roesti;citation_author=S. Gavrilets;citation_author=A. P. Hendry;citation_author=W. Salzburger;citation_author=D. Berner;citation_title=The genomic signature of parallel adaptation from shared genetic variation.;citation_pages=3944-3956;citation_volume=23;citation_year=2014;citation_pmid=24635356;citation_doi=10.1111/mec.12720" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=M. Roesti;citation_author=A. P. Hendry;citation_author=W. Salzburger;citation_author=D. Berner;citation_title=Genome divergence during evolutionary diversification as revealed in replicate lake-stream stickleback population pairs.;citation_pages=2852-2862;citation_volume=21;citation_year=2012;citation_issue=12;citation_pmid=22384978;citation_doi=10.1111/j.1365-294X.2012.05509.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=K. Samuk;citation_author=G. L. Owens;citation_author=K. E. Delmore;citation_author=S. E. Miller;citation_author=D. J. Rennison;citation_title=Gene flow and selection interact to promote adaptive divergence in regions of low recombination.;citation_pages=4378-4390;citation_volume=26;citation_year=2017;citation_doi=10.1111/mec.14226" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. Schluter;citation_author=G. L. Conte;citation_title=Genetics and ecological speciation.;citation_volume=106;citation_year=2009;citation_pmid=19528639;citation_doi=10.1073/pnas.0901264106" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. R. Schrider;citation_author=A. D. Kern;citation_title=Soft sweeps are the dominant mode of adaptation in the human genome.;citation_pages=1863-1877;citation_volume=34;citation_year=2017;citation_pmid=28482049;citation_doi=10.1093/molbev/msx154" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Slatkin;citation_title=GENE FLOW AND SELECTION IN A TWO-LOCUS SYSTEM;citation_pages=787-802;citation_volume=81;citation_year=1975;citation_issue=4;citation_pmid=1213276" />
<meta name="citation_reference" content="citation_journal_title=Ecol. Evol.;citation_author=L. Southcott;citation_author=M. R. Kronforst;citation_title=A neutral view of the evolving genomic architecture of speciation.;citation_volume=7;citation_year=2017;citation_doi=10.1002/ece3.3190" />
<meta name="citation_reference" content="citation_journal_title=Nat. Ecol. Evol.;citation_author=Y. E. Stuart;citation_author=T. Veen;citation_author=J. N. Weber;citation_author=D. Hanson;citation_author=M. Ravinet;citation_title=Contrasting effects of environment and genetics generate a continuum of parallel evolution.;citation_pages=0158;citation_volume=1;citation_year=2017;citation_doi=10.1038/s41559-017-0158" />
<meta name="citation_reference" content="Svardal, H., C. Rueffler, and J. Hermisson, 2014 A general condition for adaptive genetic polymorphism in temporally and spatially heterogeneous environments. arxiv:1411.3709." />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=N. V. Terekhanova;citation_author=M. D. Logacheva;citation_author=A. A. Penin;citation_author=T. V. Neretina;citation_author=A. E. Barmintseva;citation_title=Fast evolution from precast bricks: Genomics of young freshwater populations of threespine stickleback Gasterosteus aculeatus.;citation_pages=e1004696;citation_volume=10;citation_year=2014;citation_pmid=25299485;citation_doi=10.1371/journal.pgen.1004696" />
<meta name="citation_reference" content="citation_journal_title=Ecology;citation_author=J. N. Thompson;citation_title=Evaluating the dynamics of coevolution among geographically structured populations.;citation_pages=1619-1623;citation_volume=78;citation_year=1997;citation_doi=10.2307/2266085" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=S. Yeaman;citation_author=M. C. Whitlock;citation_title=The genetic architecture of adaptation under migration-selection balance.;citation_pages=1897-1911;citation_volume=65;citation_year=2011;citation_issue=7;citation_pmid=21729046;citation_doi=10.1111/j.1558-5646.2011.01269.x" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Few Stickleback Suffice for the Transport of Alleles to New Lakes" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
<meta name="twitter:description" content="Threespine stickleback populations provide a striking example of local adaptation to divergent habitats in populations that are connected by recurrent gene flow. These small fish occur in marine and freshwater habitats throughout the Northern Hemisphere, and in numerous cases the smaller freshwater populations have been established “de novo” from marine colonists. Independently evolved freshwater populations exhibit similar phenotypes that have been shown to derive largely from the same standing genetic variants. Geographic isolation prevents direct migration between the freshwater populations, strongly suggesting that these shared locally adaptive alleles are transported through the marine population. However it is still largely unknown how gene flow, recombination, and selection jointly impact the standing variation that might fuel this adaptation. Here we use individual-based, spatially explicit simulations to determine the levels of gene flow that best match observed patterns of allele sharing among habitats in stickleback. We aim to better understand how gene flow and local adaptation in large metapopulations determine the speed of adaptation and re-use of standing genetic variation. In our simulations we find that repeated adaptation uses a shared set of alleles that are maintained at low frequency by migration-selection balance in oceanic populations. This process occurs over a realistic range of intermediate levels of gene flow that match previous empirical population genomic studies in stickleback. Examining these simulations more deeply reveals how lower levels of gene flow leads to slow, independent adaptation to different habitats, whereas higher levels of gene flow leads to significant mutation load – but an increased probability of successful population genomic scans for locally adapted alleles. Surprisingly, we find that the genealogical origins of most freshwater adapted alleles can be traced back to the original generation of marine individuals that colonized the lakes, as opposed to subsequent migrants. These simulations provide deeper context for existing studies of stickleback evolutionary genomics, and guidance for future empirical studies in this model. More broadly, our results support existing theory of local adaptation but extend it by more completely documenting the genealogical history of adaptive alleles in a metapopulation." />
<meta name="og-title" property="og:title" content="A Few Stickleback Suffice for the Transport of Alleles to New Lakes" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/2/505" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Threespine stickleback populations provide a striking example of local adaptation to divergent habitats in populations that are connected by recurrent gene flow. These small fish occur in marine and freshwater habitats throughout the Northern Hemisphere, and in numerous cases the smaller freshwater populations have been established “de novo” from marine colonists. Independently evolved freshwater populations exhibit similar phenotypes that have been shown to derive largely from the same standing genetic variants. Geographic isolation prevents direct migration between the freshwater populations, strongly suggesting that these shared locally adaptive alleles are transported through the marine population. However it is still largely unknown how gene flow, recombination, and selection jointly impact the standing variation that might fuel this adaptation. Here we use individual-based, spatially explicit simulations to determine the levels of gene flow that best match observed patterns of allele sharing among habitats in stickleback. We aim to better understand how gene flow and local adaptation in large metapopulations determine the speed of adaptation and re-use of standing genetic variation. In our simulations we find that repeated adaptation uses a shared set of alleles that are maintained at low frequency by migration-selection balance in oceanic populations. This process occurs over a realistic range of intermediate levels of gene flow that match previous empirical population genomic studies in stickleback. Examining these simulations more deeply reveals how lower levels of gene flow leads to slow, independent adaptation to different habitats, whereas higher levels of gene flow leads to significant mutation load – but an increased probability of successful population genomic scans for locally adapted alleles. Surprisingly, we find that the genealogical origins of most freshwater adapted alleles can be traced back to the original generation of marine individuals that colonized the lakes, as opposed to subsequent migrants. These simulations provide deeper context for existing studies of stickleback evolutionary genomics, and guidance for future empirical studies in this model. More broadly, our results support existing theory of local adaptation but extend it by more completely documenting the genealogical history of adaptive alleles in a metapopulation." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
    <title>A Few Stickleback Suffice for the Transport of Alleles to New Lakes | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"EjiNcdZyquxUAM60DCKGvT2K8hjtlJj-2YrinlyNcuQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495528","apath":"\/ggg\/10\/2\/505.atom","pisa":"ggg;10\/2\/505","ac":{"\/ggg\/10\/2\/505.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/2\/505.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/2\/505"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/2\/505"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/2\/505.abstract":true,"\/content\/10\/2\/505":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002260M7adBlWNFL3Y6aNAhyhfbmeBTfO7aAw2ef8mUsXs84vyoIY3u7q9zHuFlCxCVDD\\\\\\\/7JSXgBmVtYJICHsv4PjKTEWSQ6APeyW5+dQdp9KpzEd\\\\\\\/BpqHWSdW\\\\\\\/x1Vx60TDVTguC9lBTuaEKnXPcsB7zP2XCUkkBPC40NbcwPRPl78CRs\\\\\\\/cxnoxXQoAjbHP0OHglS8TDA593ipA8BD+0ZPSjOP1e5xDMs7dlDxehx+tcWgNcxP4OYVFSWOH\\\\\\\/s7gGZTH0X0dzlj7ZYPI7yAEYT55589NVe9w+TTDzZRAKb228IJEXUmLvTTKl3zMB55rR3iX2dYuM9mNYfjPGzk4mUS9Qhfsjeyprat5rCADSkpwm4rRyj7If9jO9vGqvrf2rX+4DG16VV\\\\\\\/bt6eTf9\\\\\\\/4j\\\\\\\/ufvppmHNjfyRe1xbreukkN7p9eLm3fzGSS5hr1\\\\\\\/HkGflTpcZchf0EJOGtgx0FjSauzKDpd1U1JG6uu2yY\\\\\\\/RYxu+iwlzCvNFHQMUh7DFEp98Wx3xzBhTvKY30vbUTGKwVdJ2K14Psud9QLBD0IfJN18e8rMECziJD2At8R6X71haQBIM2oO9n9gGQAlJEB3t0W+mafa7zJHpn4npnQIRgXJ7u1dVnA3h+1nXsiN1BiPO8Tz50KrWirqKd2QCL8qGCR+uBpZ\\\\\\\/1gqOycvOi\\\\\\\/g54HuAUny4XnlveRfkinZha2aAB8kovZqC\\\\\\\/Qg9BQuh6q67enIwG5yIL\\\\\\\/kkDo\\\\\\\/WbVXPIttbeKCAb2wBndx6gjo82Pld6Qnsir3DQCeobdIKfm0Mf305D2R1l654rzKqGUI3AcF3NY2iHAXJNYJbXQEJe67pmhpG01757Kvd3fqjdo6JjQ7CsA8\\\\\\\/wqlXWHHx0vZnCWI8dnFanvfk\\\\\\\/tsn2pd5dqwZ18qUZEaKyHI11oMktGzjlfVPTpaHvGWnAqVFW0aiC2XF0ueXpb1edbpCeoiTSeX\\\\\\\/geeXvuA1w0f05l9Ap\\\\\\\/0YChmQzoljpnL98uIADZe85kXsNGTpnWKBhJn3UcoSDir\\\\\\\/u7uy8rCPhs7ZxIryf+0DBsuDkOPzrEWmL8Nq9wCzPtmgyLndWJfPH2+Arg4HIwM42idYNLPWaNWkhQoodcwkwkkAe5mHES62cT8s6ggWWqh\\\\\\\/xh193swoDwEawWcsoPl3iFtoVDWvlEKXc7N3h6VJpfarQGpWlhFcY9Qpe9egR2MGiu9ORS0lDv1sFlVwL+1ZEzGDf4dc\\\\\\\/BzrkqI5OyGNocuXMxZ2Gdtc0Sr99WKoDO2v2fSVhxEeFzR+HaYJphBiFnCb\\\u0022,\\\u0022iv\\\u0022:\\\u0022t0TAlcXPljEQZlaLm2pgSQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225fa39680923979697da64776ee57c3ce6223fdc31a810f56b6125d901cfa5ba4\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022frbAyiJK5oWey\\\\\\\/ztxxKj9lkIbIiqSBFrsqGVeifqyYN3Q2Eh\\\\\\\/jIyTsdmGquURJDwfn26QpYaLmeQ+b5VSwbDCzdU1wTD8YN0WdG4Ceu8ys2rNml758EuEWhaBKr5dhASlginTE79zshcQ9j6ag+BvBQpIpzc6UumnJ9yI6LX3qmSwvZII97PbRy5bztEUyfTv87Cjt90aGBDeR8VsZ8HeyPh7LMcP3OAfSsF+0xJWR\\\\\\\/Zurza4P+34La3Wask\\\\\\\/hXVezm3IkC3aQHo3lDA+2MpZRU+QWNI8c0\\\\\\\/sku5VvMkiP1qiYkRAB8yIic629mJUTOpv+KExng75cA9MogUt3VfAr1CWGhb83k0vSgeqqW1L99q6bgGPkRYgWPOPGQYVos5dUEKAe0BwO6DCwJypYe2pPInrIk51qfR3r3B4M4pk\\\\\\\/+1SB2yhfhhuHsp8UreeBNEwm84ikuTybfWgi\\\\\\\/Ib0ebTp5AmQHhwmXYG\\\\\\\/qHRIYtk54EmqWxExTYz8x7VJQNSvIeT+yyeuetxH\\\\\\\/H9hnTOmwj20RkmbobK8VV3VW\\\\\\\/XOpz04GM0+kk14CAE2l9mJnnvq6exyi0yW3cA7M033VN1GiABrajUmHXVL7WP9zAGMa4jz8dG6gp8KOo+zj41c4j18xXLrYnq+q+mEOsTj0t6mfy5sqg9VUFHND70AqEnVbHhpi2Gg2ZpS8J1v2E2EZtFst2GxMl0Pc6JguDN0pACpc12G5cP2XEXyBj7JnXqvqNTRAT+NgCwFlmPVqmDO\\\\\\\/W0ul3YkQy3BhxF98YYuSakuKIZbhCYkf6jc\\\\\\\/w89dXzTChhZ+lWdIVOpTgkes279EJVW4FkP8djW9g5xm4vfE5SYkIP3wtV12iDbMlv4A36+ZYxHDNGMqn\\\\\\\/2\\\\\\\/ws3Jgx9KVxKQxPzN3WPvfxQHVNuXB57a8yds0iomeulS0TevtXtW6wJkiU6q1PYXmP4QRvAP3RUpxgI5znP5+rLwuyX3JqkOZagxpuvGfMzSssyw6qSALI\\\\\\\/5WZzaB4Hk9YglQIFCSBTxhOFibCj21lkG5hSHCuaDHjuQhynaQFz4mj+efaGwJUj3n95SR9cUqO7Vqus\\\\\\\/FB\\\\\\\/uz2bceH5VEnzYSiZP97Bj4P6or2891HtSmEBS9C2wOFV8RPj21uYnA8g6LFY\\\\\\\/0zUK6\\\\\\\/Nkjp4MlFDHd4+MdRpaTrRaA9BSGDbntYVJtnz24tTyWScSxAAjQBk1cx6dQIBUr8NnxkfADNUd9VuB0TV52iYhSuqIWwoQ3O9NKlnrU87F5e3RM4Mfav1yjytcnc47PJvZ9oICjFX1O2SZe2mCZ2VzctRpGqFE2GhQ7os+jjX2nOkci5A6Hy2Uaoq02MhxMhTzudotasUgIavVlV0ZjsUtOIFgGguIZ9jslEPIigD2QY+jwCqJ1jeSVbbJ7ucgebPT1lY9NrOZVN9+JiYIX\\\\\\\/m5IOzdFBoAdWUa+sSSauhHkq3SRzf8tybaUYc56GhJa0l0KYYaJGCPxXfvH8AlD3u\\\\\\\/5eS2khdNHrmRWtsROiuCSPm2FpxAf3S2ODuxW1MgJE2FclJWb4UuC9aFlMjrYU6b1km7lZkFUJpU7PxweZgmMYS8WClJjibI3Y67efeEpTiRAQHRtwi1C3gh6aFk5\\\\\\\/yiYvMTR0SFVzH9avVMcWi07Y+1TJIqEz3nh2hf\\\\\\\/tt4sXn8L0g\\\\\\\/HE1imvCCvQvvbG72bt9I7wiFlvSZeg3WY3msQIVtADOPsOnHuTWtrPN2prumCDnMGlyxTVEIHfSPieyq9s1jVddglUKUcjHyQk+aE4WxrE5n8q5jcKBc9WP0QiggrDwwe0PY4uvVJm+kUz5KSpGPdT0Xp7aFifbhMxBKexnRBuF8GupkeGM9CwVuvHdRmnF5PqFxv3swUzqCr8D2RLPpPjRiLO6PENBitiS5ioyuB\\\\\\\/EpJ9OfSIVORijEA68cleUss0EVhQoQDlhKCUEsZGRJ92WucOd1d9ANv3diKJNfZIu9d9xvO5GVc9GI0MKsOxar\\\\\\\/ewyHFLSZgYGfygvVk4owp4gt\\\\\\\/PfBVX+HGSyMBNeLSwrAZJLN9Tmky98Qi1XpMffSPmjl6l3ShxT7FeTe2LNTUSorM\\\\\\\/HiSnu240qPd3A+lCPqNkXTenK4UeozE5HheniDzs4\\\\\\\/pMmDoNa4HIoQYuWdJRwOM8n0uhyGAdTtstsJ7kmyn+03BG\\\\\\\/NaYAhceLF3FDluZYiM2TiudOKFOaYKunp8vEOji3YreNiRU2RH0oxGGG3OgyxAKwQd5XhhOdSxwsQ4O6t95+u2ewhyillbhIGnHUw4C59kbvtOMIcStqsgEFxMokl20F8KFXwjqbKgiIBv93YC1E38hE34Wh0bwwaFHoQrDk6wvhVBKM6dF+W8QV5gwS036QuuSyTBki\\\\\\\/tppdu3C5BHKWn+\\\\\\\/sQFLbTeoq4Nr\\\\\\\/N0I3jouyZtx\\\\\\\/YTuU2ieb28IWLDoDE1ac6O1WVOToLfqT3TsfOCIOPNfrm8RDgZsjXOON+roM9hZmQ+KAkmIuQ6Iyl3E4CRqPXSiCqCM+9UZZQeMB8BcoKIM3bXW8K29k4wvmfVFImZfVIWzueVqaDaWOQmmxo2urfQVBsNW45sN4XoIGeOAJflza4g\\\\\\\/6xaseehmlSq4JvmcH77lkD\\\\\\\/cfZZVVo9cw9488ss78j4bS2EbKi8odRHAHOopi2RE32u+Xo\\\\\\\/OlGIZV4jyowClMB\\\\\\\/ht8ZTLKmI6IFrSycYFvzPSTYspEoiDPcbKMRZzy4HerqbOc52c9CYBr2E\\\\\\\/MVkcID3uM2RUneeCAH81hzcyD\\\\\\\/emxifNrZN9oODjyf1STUKxMZeDRk8xt6qEnPUtAR5BwbxjovYGLyzF\\\\\\\/N+mTwE92Tv6uMJwvOFV\\\\\\\/U1C4D6yqiNzOKygi9CklKUnlRZpGiTsNZ+51j3+I2MP0k5o7UAcNtL0Dd6Qs4Byg0QwbHJEVmhLyyz\\\\\\\/GI4xEKDTvgAstNLEeFIKqS58L4IpStm9W2oYbT7QUl7tCd+BXMpm8abxgbrrXea4BeY+t8fvrr3ynK4M7JBdgtsHdz9P1DZ92WnhUr+uyHSysl9JPDIMc7l9uQEh70Xcw0pZRzumdZfP1fC2tzYJDoRUqmXjVNipnYvXmJmZWlXVkfzkSpmn41xKHl\\\\\\\/yPvrJeIetOptOgzL46C1MPMQXsLN3D\\\\\\\/yQitcR6zdEIb0EH5W3CzQh6BpMyMtYBlgxRBdedKCb4q8Kz4EKRHcWgYBIDlnYCHdNr8KMjF1FvvoRnNYfuzHanP4HP2F\\\\\\\/iMajWKTCgBa1p\\\\\\\/Zr6MFQ0u1had2R7GAU7J6pvYXtNcw3mokqG+nD078zcZ4ILnjZ2su4qee3ynh7YEMSleesl79w7ADxh+DkswUhX7SgsNxc9UbabLQZMOfP32f\\\\\\\/aKqaUnknEJY2N1Bx61hINSFbyICquuHSEVNyOJnhktxuKEQRE9\\\\\\\/5DAr9gSIijtukrGnA6AZEMYKn635399cjy3UNfNr0VvT95JWrUsZk+46b3Jr+w8EaEiXa6Y5Zbe8belTDKO46aeOZsxkfaMhC\\\\\\\/RzmQFE9aHjRdG34RMWG2zF\\\\\\\/axFP8io\\\\\\\/DwS9UYHSdtDWdB9yxXIHrYgWAlX1SNCVXGZwbgFxPCuArAcYaj7eY8komB6GbhdQG8iamQUJaIaKdk9cWWQJ73STRF9E6ogsgZp52vtWm42V3VIs1MRt0Y5KaHhaz33DYrBhKl6FGXvcS+OWH\\\\\\\/qHDjrUAEQ7QwrhqqorAXizrSqyN3GLJ2wFnJC1ifzE3HcyMGen7HvlBYVNkcXpysCChMWQhMNWimGKxjiFATcIDiBA3jl3HZ46r1AWZDYlPiaRPe3wqp0L8M0DJ3fyTLHSXGKBu3MpgSf5ikCIdRWwZqe9alLtEE+QJwLDlB4QkFHDogVEPGRrhoGgxB1UmPMcGmfwBp4W0i2YcpDe0AmIDO3mGPCIALGm1AEgBSqinN25cRE5BBFQmhoL0tzKUyPfFy7fJuLJvxmhUO6wUgoPWvVv7y3+mh\\\\\\\/L6b0nkr7RT9t7GgGlIIyXpA0tEFCKrNrqmv7my9vqTaqkMpYxh4o+qxSFzwsI90hprAD3fIyX\\\\\\\/kgR8KJZUSr0ehGDSb0RU5oZVkzGTEz29l2s3oJz4hMmzDR+xJd0M\\\\\\\/jNFKrDWcMX\\\\\\\/esSAYkMmqyQyX8nugjSvuPJ1UphYAqREyB2p9H2KamDVlaYc51ROKNSV3BTbYNf6CI1wQmHGbQJuV8enMRb5Lc+KHUcb+yLHzePb+wx1LHiRMYwRlemQfeHzX\\\\\\\/cEgF7nNIaLblb4AFFrZRymz2lPb41\\\\\\\/sKij0yNt8i1wdCTceqcNI7lGCnNJM0wVj1Zq4N9uUdNdVNuX3Bm9pU4y4IVmrD73Rmc\\\\\\\/Il5Fb6nHqszzYDEKAjqR9w0AvY2EjlFAGWHvmZacK4kjHR\\\\\\\/BbIgFTZQ3jbqMp9aQa253YFes\\\\\\\/TX2boOd29KPYISpk4wI7vjdLtY0asPDmvT6kY1wmjRPnwSElOTRDQhVQd+CYoQ4yAXio8Kk8r3b3ZpVFnNRuqDI6vR7iX2J7RU79HCV507gJeAe5tRa8EX\\\\\\\/HCCXKOZmqUbG77W9OKadc2NgkBK+ztxZlI4p5OXr50ErOjvw8nnYFfocsymDXxLnJsQJML7PURihwlHQ\\\\\\\/iMsPEBEctwDWLkg8OLm+\\\\\\\/wA1euD40sJRCDdSANdI\\\\\\\/Uj3n920dppU\\\\\\\/NCoj++0DEtSkNknyS3bVPv2YMPxN8MB3Xuj2XIAij8cA8GJQNYi1MOHMpp2cp8fSC9RrsTnq7Xd9b\\\\\\\/z3BommvlEVVDAUCgxBs5oFUMMYgwsziXmcHLG9egLvVht78O9ob41GEStb9oOyaMqY77Y0XnnbaWVv7SK43QQYS9pFso3v+1E013wP+agEUIvR++L5g2BJHUvo+T4jcnw58M2UUGCCf\\\\\\\/R67yEAdRyAtTglsYA0logdeYwC\\\\\\\/FzdhU5OFbZPJzHUtnBLG\\\\\\\/8DpcmJEauChBFAy5N2sGaNS1b7JYM1kqtqw2dbCvonXMvffnhoTT01Rn9P\\\\\\\/s0n\\\\\\\/8\\\\\\\/Merb3HzyHqjrRLktsebmmud0KEsfGHKn0Ie3mnqJ8JutED9FwndEYa89k4HL2XOI4aKezdF2D0ioehhLkIfw9ZvKhvRxGdmjJRhp\\\\\\\/uRg10ygjVyxqiRFDT8sgAjrNK1jZHOX9DQRKZ8mypZo9NCswaBmgQjZgCeegc8OPkwi+pHPXHSwECcKUj0eWSk3OPqtycCMeyFNWXNuu63H4Q\\\\\\\/9Em0+5z7fGhBXEe2mAMctFSR27LPhJQ\\\\\\\/alRQ2RLnsBPFrvaKSLYN4gR0\\\\\\\/Say0MRdtjZVPRO4hahyWM9sz1\\\\\\\/PgXyHAP03dDHO7JGAP0\\\\\\\/wIjNqwp8t0G\\\\\\\/Bt1YQUcLYmMrKdrvVVlKJaCnx8kptFQabiZtsD\\\\\\\/GE3XguA\\\\\\\/B7LjmNM2nmTupwJAJ5IibE59doaWdYUIQbJ+EvcLDYuvYPjOiAVGGoq\\\\\\\/C62p+DhfQC\\\\\\\/7asmRgBolueiwetPGIPHYno64TKayh4uYcPaLuG\\\\\\\/edgG+jjh31x\\\\\\\/XegglH9EokELdT0H80k04qGTJA40Mrs2WOqPIAWgaw+cJ32isnaaxTo6zmgMMdR4edRACzOgYjpUzU1umDPlYScN3EPCq5V91TLjnPvgGq2JJJY6wbQaYx\\\\\\\/0h7WYShKq\\\\\\\/OqTruSyrith8m\\\\\\\/2W\\\\\\\/4Km5lHIbWjNnpTUtf1lY0kaFZ2gLDN0mMuREKnIlNKCTlGj5HXuR3opVxONUP6PcBDHjT\\\\\\\/yNVrmjDCtO7NCq7yiG\\\\\\\/B+0ytswszo1CKB5VjRrJj3OOTNj7C9fJoxvKntDPa+HW6iWvgJQP1gst3qubnjrWDG2QhMR1KrvtXblGrBa4zc5Bf+vVKuSEmKOiRsf5fOob1fED9O6FQ9Nwkbc0CxCyhuwK1QQJi38Jw+lvUcvxA3u5YeqRve4qWwTSQak\\\\\\\/0fvem2EWhc0f3LyfYqgDiI\\\\\\\/xhXHaYuRp7Em6XJRnFoqGmDWIPLIkw1qcszADq54ig4rrHD8vhBJwecLFGRtXhPP1jbyBIye9shXtB+fyj2zyKwj37sAcOqwTn906giBff90M4Vv93LFzQHfy6k\\\\\\\/L1s9YqJuuosWg\\\\\\\/x76CU1LgVoZRu8d7gcFU\\\\\\\/qjetq1WwP3hfmN2QZpLSYmpfPnDj1sum0+i8X0MYzdljUaMU97re6OzlkF9FxlXaMVAq792d+fI0jCcnJBg3ay7Xi8Yb0qf5vld9a4pS7+y5ZePYG5AFFnpgSqaWszkDKVxb7lXQNmTUXmZE1xR7CV6WGaTtPJAcu1F\\\\\\\/zqglwIGMpRNOlkW9IU3U4voPf7wxpPOWQLyRWJ\\\\\\\/7b46PCFrtJxy3GN6FoQb430qWfB4uTh8P6tV3qsASRsJByQoENi+JW3Pskl0LqgFXmgXs+IKOGGOx1RnUO10PY20WFZEPQOFFepBEyCVJHh+7EY0WdBUWkIP9PKYXcQrr2OF8Wo5oXvXbROPMRDxGC5vRiCQqCIY9D77f73QjlrV1o9VAA+l+O6SiuBbzefTwpzbBnI\\\\\\\/HxozW+Ap6tLcRcdIvnD+cWpVRbjJzWemvsxn9Ehoao4BiPld7cxmcFSTimI8D6vIBJBQJdgFEDCh26X\\\\\\\/4r+fkCLjyXdUZvQ3Om26jOsmi0C\\\\\\\/phQc62O7WRCsiibkjoIM2tMDZiEMxaSphRnBw2C5r2MvgyC\\\\\\\/i7mg\\\\\\\/nAByyuRZJQLFG+w2MbNOw2eaP4zVrLwqthl4vR2RCs8sZFnWbbFqXnnQ801i+cDKvx9\\\\\\\/F521kqfk7jEiZGyyD0RXvysnMfBGaCcjTc2h0SF6NkfZYj6grOi8ndV\\\\\\\/jERyjLPeLzM7N0UzR93N4WvLvo4eOUMYDZNG\\\\\\\/F7Z4r1t\\\\\\\/FhYkTlnmIpy5FZ5q1cMoJosom4tf\\\\\\\/SJ198JwzMcYeAaIlwvjMNyBRiQ3AgaKaeSiAizhCRMgFNsQu0Kr4Wb\\\\\\\/A5jxz\\\\\\\/RV3xrylveU4\\\\\\\/n\\\\\\\/yu+Hon\\\\\\\/jb0DAGlZ4Zx88+JOSGRUI6OBaicLYWe5RiuJ\\\\\\\/F2p5omQqGGXbvunpAhH+axhcS6s1fXMKXssnD\\\\\\\/NTViWSG60MUO\\\\\\\/7YQwpcse6ESpTdEnh07JDllyAyt0uI9D5k5QRhLdpE1zURn2pXybKzSUaMb5j3Xn8qRHt5n\\\\\\\/x6BDqDkO8wY9tyU5sUQ3qsYrndQl7eqZ3nmiXYiB5srE8hVBnWifNosFFHkj\\\\\\\/mtOt\\\\\\\/sg6rJUPm4K7Jp70GXRSW53X\\\\\\\/AqzWPa\\\\\\\/eK1lHTBhIbOGYdKjnhQ\\\\\\\/ERN1ikghhLWFDG85GnlrR\\\\\\\/cXZ2apAiqeIVA5bIoIuaSrkeBNKSzxZGbiJDP0Pg3bnnGTXpmI5OY1iJLeSLfFuNsRrJUF5VkpD8N92yVmciWxkjnp+vQmdOas\\\\\\\/yg85Vu0+11HyMG5FWihE\\\\\\\/Co\\\\\\\/mLUsVEhBxhTuOInfaLKuj+VsDZNsjf78Uj8rP5JbuEKau6aPYKdIgwW+qaM5vnJVoupYenLS2kIto76xrTLDbVa1LlQ1GFu5jRUTePliGX4Mvp4KPzxJFiA24MgznLAY+23YNBzOtbuQ52KT3+mrdGZqHD6PkczpdlXVjabMwlHN3dbF0NUwDPrUq4UM6zTm4STmjWdBEKo4o4lFsXxvKNyNo6UBnCThZJuEi\\\\\\\/pSf1diXXnjzfd4GhDs+wkoOvfiCcef06iMnDG30Nq0TahUyiqjdT\\\\\\\/IHyuIxhscCV\\\\\\\/GdNh1vBKGvKrc5Rf3WZsowZ3jPDxaRA800SGZS0zNq9IAmDSZqeGs\\\\\\\/WOh1vHju4ZQchtteXCaP8qs4m9kV9qzijOMnh3+vHG8GLCLOH2FVVi63iNAfkRRe\\\\\\\/Q1D0IddhspNtaqW5mYecF+Ex+vjtKfGblx57mB1AbLZw5A31iMWBZe2RL7d4PrbQpr4dbe3MPD3bxCies7mehKqwiPeb7W9EwCOdQSf1NGrXaLDvShXgP6JmuAP9z6PQweUO6YoaxLxgCnIAB2a\\\\\\\/ExKO\\\\\\\/gjmMKACBfQ\\\\\\\/1g5ev4cX7DobRQKKv\\\\\\\/1kHy70\\\\\\\/I6YSA9S+2hroAazw4sDBbDPXZzr5t8VL8UJDrGBV3KuPWziXmYk76HtaCfJigCoy7G39ezHBEb\\\\\\\/gftwwcaPnr162cLNtZcJCiz0+F6ASTLtCgqiYejPFwWp3v+xowU6oxfNImx9PP4V68tJ79Y3NnDrTHLtojxLVNHOGdpIwdWRRruaoETGW\\\\\\\/TF5lqUxng+PiQCpsDKetPlGI6wiNc9kOUxeWFddy1\\\\\\\/RHVvj2FGUwKAVeGmFPkfnEee8qQehNCGDmFR4BJd5bcLRIDY25il4khT5nLz1fHiQeMlupOMDCYleJOlUwJTWF9JC\\\\\\\/ntSjpLJd1yPIvZynaaJKinaKmiU04l2vmkzRYwK4GUxAGI\\\\\\\/mADClJKt2p0LQhgTS8VS9KMl5q4KEntYr51bXcJZd+0FjyKYA7gI4C6YhrdLIf9YS0U+QZsexGwz+2DDG1QrKFQoz6tTOQGAeZimTxP7am4zs3dC80dEtnPfvpbY1WqMeXEjhzS5CyO5EoWnZRQo+dPJHXkFccK9gV1dPSM\\\\\\\/me678hHYkFFdd4N4diRNeeyH4qwN9pxGtMwmh\\\\\\\/rM1K20n6K7gR\\\\\\\/ua3bu+3BreRI5NoKyl4\\\\\\\/\\\\\\\/PwjFMkBha3ESMVgKAHYBQH6uNUWn\\\\\\\/2Y8v31PvTzu\\\\\\\/KkgD5npmXeb1QnVs8IHzyFPajhXpAkQ\\\\\\\/o\\\\\\\/5KoY8GO2w1jt+Qj+pC0lxTZfg4T1EuGHkAdQ4rLw5AAgoBEqifU3hBlCM5kqBXYSOKUPoVh8uKr3Ubjjd\\\\\\\/u8BWVnHq7vttyyiFasXd6SCA6br6gdoo2JXADvYsY6pXrbSBcpaX5Nj65fkF4sTXGquptkKJRNbUTwQcB+M10hYwZTFOmuozMP3v2RIZH9tsjb4yygAqNJDABwutc\\\\\\\/jPY\\\\\\\/nuq7pnuaZMjQAOMIybtMWx2N9FUqdivALBkodAizEB7+es1suBRj7\\\\\\\/yQFvlYPyKiETizwbm8GuyaP9lRuOnkr35sgLNxuRNWNpAAMyVLqzgxsmwBttqp\\\\\\\/J9Ce9dFrptO7ZoPU9EF700MfBZT2nABzEl3cCOPZIPuEPupxH6oaauV+IHLgv5vXfZB0qsUkYui4rEWVBCODiaKkW86bSPn8HCRO7qSP+Nclu8\\\\\\\/FKrbCik18uOJeQOVplKZNb57bYWpU9p0qnioYyJhUE5m9TSVVRUVlGmU5MSwivnHOpWqzMi61iKUa0h+clbVrdQeX7Nh8Q7bOusJCzEAcpC8vGATJZ2vNabUbkwYGi\\\\\\\/VOi2RZhq1eU1b2GrvqsSL4iAhtyaht6gWlpZfa5f7WesAF9Gz5IMfe6G+D2qqXDOrJv\\\\\\\/\\\\\\\/9n9Dc7nKvq029TreTyWbLaedGc53hxYZuTqc26eJ1Yabo4KowqRl1xkFZAhjk5qxn4pLGzIQvlH\\\\\\\/GZdHon5vEDDl3wrVxHiDkyRTIE2IV5ECPgKiURcmYIPa\\\\\\\/VRHcqhYxU0CYE5IryHFkoZnuU2aRW2DOTAePio4\\\\\\\/rkJ6euhspG1xcicOiR15ebbhT+HoHz3KNxU3RzRs8LNLDxFwKl+CkEgQwsY3nRLuF4ncdsqSxtD8u94CZNydvLXvoMeXmwpLT5tfq3vKRiI8BBQtX35DLRAoghLRi7BhG6e2MWuVncepXCPWeyO\\\\\\\/HHu58yccjtFWiRJXhUukr8PYV7wcjlOZ11R3i\\\\\\\/0k2FsouNj\\\\\\\/prRK2\\\\\\\/GB484YHwPU2Nqfh13r4wGGYNNpVgBJ\\\\\\\/4Qxf+BhrlmHN00IIgbGxOrjcr7ClW7SlsBuowAPvongrywkKIvIr\\\\\\\/ZyEM4xxayp+gST\\\\\\\/uqcye+hGxIL\\\\\\\/PLV9Wk94eh8OKH1DAMAYv3Cpysaf79rxJyeEgx2JtKBj\\\\\\\/qUey33f4wz4XM2\\\\\\\/E91LPtV0u2bLRDGIZQve9SBQ\\\\\\\/kkdkJIsPyu8j7n1nr+lkxbv\\\\\\\/bJsURSULIPNunm0ORbqWy7GsJXdelshG8IrH\\\\\\\/wGRFXJBlPtJy8wDu968COH8VWEFZh+Aw\\\\\\\/Lc1ZGgn0HZkSJsDhIsNtFKPq37MbRKwCPDUKdrEV07gUqriMGhkTS4r4kot2gD5G4N+Ehg0LDOgGHrdRYO8ScFX6ImE\\\\\\\/sy34QR0ngUQt2iwmQsmf+frg5xDDJGfggsOkzPmbeLx4NaWQGQTvsXNqBGw+eNpAWWS7Xz10CvQQE7LCYza7J1zLA3ZX2PhvBlqw4t9u7VDQgVBma9iFhtmoH1POtXPCT6iz16BJjVhI2ld9mbkYjru3p3ebH28zt0\\\\\\\/sLYUtT6GHrgNX7Et\\\\\\\/3cqXct7CU5vgn89W77vr4M\\\\\\\/LypeUlkeK9UT44oI0bhPQjbcic9TpULK335YKcEHFcUZUuaIi8KzQaXZuw3bc8IhuooFbiQp9UMktGnGQVglvcW0OKAbPSFc2RBODzpRXm2zJ9QdRtXWMdq9ca5Fvq1JxGjESiX2TvN0IH0F3cONLQZEUOSOAiefTeBGNKhMbUc0ZXTF6IFA3LyLxwe\\\\\\\/18xOrArPCUT3DZWxMiajd2CgNJ9e6eOiGFJhFZVmiQNPbIbuaAVW5xmytxArvEs\\\\\\\/orrvgAyPbOMneVz3TLVF9V4EwyP4DZnL86Sv1Sw4xnODYxRAAeePQ\\\\\\\/aeiO0EVQd\\\\\\\/VcG39HfcaNPG90DSc4WEIQjnnagFyDYss+IoqA3NETEeloI04ZNvibo4j6yYMNAMpn2e8cK4SNTivj5brVv8KjiktlJy0VWZtVqyfymNiOHE6R0gbyOl0exPfgLge1HqODaoJQWSSg+Jp6Et\\\\\\\/Nz6O2bHj1oKZO\\\\\\\/dsLjHOJ8sHs5cLUZ2iqd1+lQRZZ61tDqYMl4JOBRB2bvvWCYM4bmEzhdruzINlGi6z9lzq22qzujMpI12D4401wK4yBCQSiLDB4LCuWbm\\\\\\\/wPNbebvphdmhQ5vagDRNDMlebA8x\\\\\\\/KtlBK5z9A2NWLaQ1AuyLRRLIdgUBVFzF9o7hFvwSBaPvzMicrw37VzfNTuaSIdwDiHVTSUStgVDYOl2kcPQIYlZLMMOtpNyKFA6IHNFCg\\\\\\\/Gl6n7\\\\\\\/IGOqMNGeAhOMYANj3ZGgBDarmYBm5hAM0GJem75u6ELNUS1KaGiEAU7s6lZYjfkq0RRxyTsbUlbKYBQHpT1k3cg1ROhxTO3iJRk82m1ypp979t7raJe6\\\\\\\/Mf2lufwZv4MR7rbPuJuSjidCDARU6Qif09NJJfACV6JAbF9AWJ59IQ7xCXBS6SP4njN1nCpbaPf4zGa99WyAN915dOeFTk3YXTn5la2ef6FtGhSpcFo9m0asJiwK6dBfFRRZSQwyNGiNL0KfpKS\\\\\\\/Lq+3d+HIA21f6kEnOrznN1APSMZha6XVje52aqDQWOzmukwacg890M7RjvD+Tq+E+HGkP0kivcu6lql\\\\\\\/KXCVJJRDosiiOaJZVIZWsWaEDbRyE3dydsC\\\\\\\/qtVJH7tOIHr2FFDYkgqsBNivCUtQhLI4UXu+nXV01czgnIGjXwHyA0l4lGUXbPk\\\\\\\/wyjgyNTw25cFF2pzPnwC\\\\\\\/kcya1mzw3yvw\\\\\\\/6Xu0hiYh4rar+Xd78eTUObUjh4dgiIjkhEnwiWsrQoo8V\\\\\\\/UR1jVv8\\\\\\\/u9M25A6lyHCZbBrfbVwCpxrNBEUmeDJ8l3ICo8OFwTYpqBiGEzynVIcFiPqO+I36Wb\\\\\\\/toMWljvYl3C7P+TKROTnwRap8vk2j6Z\\\\\\\/4NZ28sKwhLdtBMllZPttLPix17SS2zcuyk0+cqzCAOAnoRTDe4Ldpmu+jrv6\\\\\\\/YY\\\\\\\/r3\\\\\\\/P42v4f7gbktGKk+uir0RTM8mYw8ywSkHIL1\\\\\\\/C0d1djH9iaHcG6\\\\\\\/aBE1y6OBV6jT2g+5s6HgaGx7CXmSEiHHDUTHxuuSlv+PhEJ687HXe2mMI0BRI\\\\\\\/AsZKhS4OHtLm4uHoMog4Gn9kkR8pkhk8xEMme5PxNK0fPNWPJHu0Ig8nL9DsnmXuNtqdK6FhRsosWDfse+WKHKq4sPN7tzOmPcbf5N4baH2gJdwuvtjI69ECRGy59mMF2645CE697hf4QRUtQ30KFcGr1OV8+eaV+ldUDX2SudM9JASkUzae4QyLp05QmPdbxKbL7pJgNWm+FboBsDX6dCz1YPo9FcQMOvFk9lx6hvn4rB15LXsvhiteGlMPIwEKiVqoN9sw0N7FtSAkd\\\\\\\/EDtNPTTP4wVx0hWxw3wSu9srFxNym1iv89EVVv4A0Ttpv9PdDEgQS3N0PrSz3ehTn1nzEwods6g3XBOrZGFON7OeFWxNwyx86QZ\\\\\\\/tsSW8mvfbkTef0OTiO69ScZBVLgDUcnWQZ9YpA+99tR8p8hGDvpnZ8hLx9uTprNk89DJcmiVc5cGnqZUk06hqSkPKfprjcql+RSXyC\\\\\\\/X7WQQdtBU5kj6ZQLx6+my81E7B9wFLB9u\\\\\\\/5zFPkvo1ZLPyLmtWCCDdMHWhm8fU70oHbfcO1QNK7IxFmLtM5itFP8DHppBG2ryJpRZgD31FptdXXM1SS4nD7F5lZk3lqz\\\\\\\/S8KxbEmZi7veLC7\\\\\\\/SENMQ2Ma37Nge7iXch5I48zKEbef0\\\\\\\/E8ZljwJVQiC0zCORqxuXrn1BtqxKwwqSE6DrJyh8sLoodllPZYO+7nUXYHaCgiFNhaGeWRxHpKUvxBFHSZRvg\\\\\\\/ZJie52pGsx9w0DzX7pWDwy1wvHhAMParoLqJVepje7c2EXSyr17w+KNVY77cNaJDWNQny3qWeNibTWE4ReabyTK0kQllbz8USJ6mQWR\\\\\\\/H8X3VoyTk0o78O4WGfKRhn2yOkU1s\\\\\\\/v5+jEP\\\\\\\/pr3aY8iyi9Ghwhyj7J1a5UfR3UhGBLEAtobpdX+ejRDYA53cLSX22Pz\\\\\\\/whPt7jXYGYSv9bjRDxhIqXtYOb52\\\\\\\/FWpIfdCaThlSdmJVFDhY6ZXzURS2mTKXOkuVGFQ96WQ9g2LkNYjeSWcRY9UC1rh5cw9EbmqT3p37wenPrNp7LOH3mqphDS7Wm6z+LAEgou7AWPqU3ghRHXo0UcCHqagagDIPDMTRAakndupzeApqd0JQoxe34pbG9Sa\\\\\\\/z8Jcs9Qs1\\\\\\\/liOncJ2+WUE0Bvhg+s9OsORG3XGJt5YCz4W5ttgXAowDzWO\\\\\\\/tzkfXD4zBj2teRTd5PyAz6vlbEUx6PPZKHuF2utzxp1U5onWBvi2fdlq9CbTZ1PMdbkUiI\\\\\\\/shoADP\\\\\\\/EAFZ7aL66TpNfhTpT3CdThcI+YwiM0n1pOJ7Kx8xDUfprnNPDaEfySHYiDpYFMrV+f8MDnMi7qDSeTKTu\\\\\\\/Rbkb9iO3w\\\\\\\/ICSQJ6E9YnUW6QMQRS5iJ01ctB6LYjYyDBFH9R\\\\\\\/az\\\\\\\/RcBE5ZDgu9GXXv8PV5ILQ9qYEGqkLuJLvV9Pxj7pzmUtx\\\\\\\/RxDRucwPNjSgo6K7Y\\\\\\\/+vHy57ZXstre236a6TgGKSXpabzo8GYQkmT1FkSoRC\\\\\\\/rZRjcjGLxNF15jgOeo4wLi6yYp\\\\\\\/aNKTxOCskoMuNWGqV1Bxjnhjcg8VmIK8Rw+4DLKO5WR3BHlbV7nUTFZdgS8FsFsnhNsmRsLcntxG\\\\\\\/hF8hnKFM\\\\\\\/842fZ2elolZ1zoPOGEIrWKwbegWC8rDrcGoVHU5XRNwdjN\\\\\\\/NlDUYgVn8mkndcJwoAxXpT7G\\\\\\\/QswE12AVBW1oFTUD1ZBUxYUQ59C1TQiTyXNXnRkSjuQpqW0orl\\\\\\\/E9zT94YmXnyAKqxgaXf3+RaFnff\\\\\\\/ep4KzPF8si9tR6s6nehXxG2z+WABmICTondcikbSQeNWay\\\\\\\/ouMTRxBOvpFhOKvwejiD0oNwGbaenIDiflEC72p9OfslOSptKeDihUsUMTHe6gtlanwcldL9xDOJJNscDOJ27zmCPy3wMu9qBf3n1e\\\\\\\/B0XEhdj3E5008Cx2wxX27UU\\\\\\\/EFcxM0DtHCNv3tZcjlOzJC4mAKxmSWGKOkOmBiSZ8Nrpg3SAXJs5+ivgLDNjC\\\\\\\/\\\\\\\/HTykwTUvNliTmGVw4o\\\\\\\/liKIvbKkOb7i63+Rve2k6wLlCu92fmMqVjvmlq8NUTF6NN4ApyPvOZjCuNKE\\\\\\\/ALxZG2v4V5rca2q2VPKXcNrl+Yc\\\\\\\/sLu6egwNIk7jyrt8ZvrpTuWOHDtquwhJpRyOEZGz2gyAlUajNQxpg9hLoCMDhfG2Lrh71pT4fXeVEDdjKKkT7oJCcnKg3ndxWIpnYnU\\\\\\\/5Es1egy1ga4vR5kt6+BeWTFn9sWWmOIlo1Es2K+43aurxGg4AgT\\\\\\\/hAX1MTC+y+LI4AQRkv72O9ZyO8xwERHwuBg\\\\\\\/xqMkq0rr05O1L52m041EIL2uEf+ECAAGhB\\\\\\\/6dDYXn2yfy2bPbvaC5VkymtJnMq0k1UT7TyTAbcULLTQL7+TJWtXIPED4mLf5JNEwDIQTTToJTI+3JOsPQcU2ZrFo+cYv9ab8G2ZTL88ZBtVmiBELdWHDqUInSh7gEEVZ2JQyy+hfDt8SFMFfRQGV9Th4gi2nUp03f3IM3WNCVLxKTn9o+TRaihWZk9DOOOq5zLyc5v+ZoDHPkP9rcMSs6t8Hkfy7M+lQvxjGnYNvXwIOUFZu9T5hv2tG1meKPHL2+Th\\\\\\\/cDcttRbl17eT6dLV3gojPkLRuRN4lowpPgZTF82x5OyxZLxzayuraZU0wRvBUbgr7\\\\\\\/rsCL1npj4dy1eboXMelP+gn55EXRQpRTjHWBdXQC5CieNSCB5LYEJJlnOOglA1UtyQDFkzSNC5D2y1qphiBnoM6LmzpLqQLeAYkjp23s4MKjZjPRFq2tkCSNq2hUHpyVJvHYdtfuouy7ts9Gxd+MN6sZJX34V7inV6L\\\\\\\/53Wo7Yr4SD2ePwOGeeVW01ZQNV5C5BsltCoxpd5kiy6dAYfGwZKqkbHrsmOXonlDWcjsMUFDNbpQ0WathNxXg\\\\\\\/y\\\\\\\/WbGkv56T3eVzzxGW3io8wiBf31zPL8JvFJ8QaMgoYiMNKKVzJZruAt\\\\\\\/ACLj1SJjgA\\\\\\\/1y7dsHeXrxFebFRGz3nsdUMfRtY5O4yRHXcvy0KJHPpsHXqdJl5B\\\\\\\/6dSw7rj+vYzXszP1BXAnq3EDSE0zw84ZCeDXqkHWEwXZJisEye\\\\\\\/i\\\\\\\/3yfdwrW0kqMrWF+IrfcA4bmoen25aQRrL\\\\\\\/QQK0kGjVVUhzF6MVebCPdaCNgi5lTDdk2\\\\\\\/mlXrvSr0Pb1xhn2hyaidsNiJ2GMWrt2KD2lRqC1lXQrtkuqMaA6FnG9cTvmtOjZzCVvhcMKTx\\\\\\\/sPoVNAXUopw3P9Vbva3XUTcWozjhuxxarRDTH4opmI7MPfQ1K841U0vF2Oe\\\\\\\/0EdYmjgstHZT+2V0Xbffa1+qyqD\\\\\\\/WSLnncnVS9F1mb17s7e5DE0Dq4dCbti7F1Nai\\\\\\\/XJm1iF7fqXKlPVkuDShJIWPDgKznTS17cFnvqrkAUoUJA0GvOjIP1AguGyJ71Ny+PUU8wQe+FhSYYtU20Q6ioNoqEVDTn7CgwHFZY8FjVTlMQAqMRcnRXINiLJwHJwMGVMLMaeYhU\\\\\\\/mIUcWq85UHnfPzOTnb5gX8xQzBfoeIwCcDuT9fBM86bwWRJdUvgefMMHWSS+YAhAn5pe8k9d8Ud9DMeLq4MTsz2TcvVkradRylgUNe1ErudGl6m8LKQkBm0S3hviCZhmsAH2VysaMYAys4gXJnThmbm95xOvexZD7jYjtAWrhLQqgVAIIzaDNHtr4pXm1\\\\\\\/ggtNJxxo6SiTUZRZzsDyMp4twaIvWBDZlTf3IKC3eoywPJRQw+\\\\\\\/yO1sq2UNm1WGtJOn\\\\\\\/dl2oJiAUQmH97ZiaLZFRRo1r88xsT1jVu2C5FCxXcPwQtgdzG+NX2F5W3vzbrXmJFlPvoqihUxUnNYb0I9OEB3nasVPHnT7qQQ0rzREjsdzNlA\\\\\\\/yZkxDmKhliqxxNVNrzndWh8yS1E+UNGavk6BxprxunQtRByybdRlVKCr6rNEy9D5QiePqmLt\\\\\\\/jashMn0+KxvRi6KYjEDpgWHrY9pP2\\\\\\\/beAebGun0QPGDbHQJqtGskmyJyPPESG+ZoMQFKFzb\\\\\\\/8ieCX9NdsavXqsdGlXXwaTXl\\\\\\\/kH8qiY0YwNGPZPJwAxtPIbHcSNDEFggaWjo0YqjhAvAok1TQdvnYSupCDSCGuclYcDkiPdVGdl5kl5K6RrzqcO+gtVJrerwxzbd0ITiHak\\\\\\\/GKMyyqDax4wMzMbt2eJ9HBgGJYj1XUyGNh6LBPXZit3wBwjoEjdoeeT0Ms+Q1v22yWzv7Nu2al4w\\\\\\\/HIwRYH2MFAXdkbiGPgeEawSRXo74kECn10x3vkToa2dxQWMP7yWn6apUG36klIe0VUPZFMYzDsg308cNZks4JrKNm7mgdA1NFVQbHPE1XpnDe+34QEe0GVHs9ubQP92wr\\\\\\\/sJgV4TUd2RKvSFNLDNbhd5G\\\\\\\/NAllwnRBqPJSDk\\\\\\\/hPFzRsUjdvjPWWhh\\\\\\\/sdnen8yRACBGp\\\\\\\/ybxILnFCkLG0kcrXsL2ZPkOxQwse7k\\\\\\\/8CfY029xTMEHc57z03pMWZ8EsH2MMy3unGZPMGQT0asISGuZMmYhCk7MIEd9444X23YYy+7yPsR4yYym8GFQ8jnSiqoxJ+m5GevK8e5ZlzJYbkD5BmIFipl4BMwPPbC\\\\\\\/2OEw519jlDwX0bAoLzjgncpEgjrP3GBtIDh9PKMF76FidFjKITSeWmmg+eslA2TuWfKTfUA0bSW00HpudEQu6PJePYsfL84KMibGHEvPoPhfw2GYQumJzG0\\\\\\\/RmHyDCdsdMqCnueTdbZxID7KaFIW7fQWAhhpToG1jmCCNJwGm\\\\\\\/SRaPg1ZPoha093ZHnmRzKBVad6Q3Xs9CtFT74an4ZfhLTgTBP2YTwyYfUqwBEaUi42YD\\\\\\\/hjP62nSQKhWCltvt08JA\\\\\\\/z+nJEJ7ojM0aGKpei+TkvcYu4dUIDLOdeuZy10RtfFgCXyl6PnYe\\\\\\\/kQrJXwS9PTu\\\\\\\/A20reQCE7Y39jj\\\\\\\/Na7Ptjn8RYw5oeMp49+5bDCkgTJqG6cIWggHEyPlBoWXmN3Bmne8o0M8n86ohLgLN7C07Tqrf9CXQ1G5\\\\\\\/aq9Y\\\\\\\/9GOWnREZa39bJnysTXCmQQWRNhgG1UXmAUOCuVqeejhmCGXPpLnts7ZY8lN5jyKpaEK\\\\\\\/VSFTLGpsBzMtFlBwsQAoi33w1Rs1xWKzyOHzGS31kD4HZkBOYecZMUpEL+Kcl9tNBEYVZT4XNPwm0IKjVs5TZOXSiT2w5q0ciM8ZEem3yTgWsS2VJJnOJBS\\\\\\\/PHEYOUAV+WkA3YpX2UQNBotCKjUCI\\\\\\\/5UxFQHNVQlF0kFUyKkjos+D9fbUH\\\\\\\/XYf\\\\\\\/NDKkF5MI76oAujRbm+2olBeviLudxa7vzS1jq\\\\\\\/1qRGjICgBdoRfCHNfmnFjGuqh9Hu7SdfeqlgAfuf5tXGZg9r+FTHW0EEc67xk1ik4G7EFtMoZKYcikRmgCBzAg9A0bIZG7k65NaXagYJtgBq4vI0az2y8DGRZ2dCnk8ikL\\\\\\\/X6UqlaLH7ju9HilsPJ+SSI7\\\\\\\/4JRvAX8sd0tbuew23Ou2+izmhLnPfxBkQcNLIdPqkT3dnsc3Yyhx6iq2y3EdePE79dS9mtoFHNtavmH\\\\\\\/yxm9AvXmr5Ux3C+ty+B4XgYxSS7v1LmHyM4+KGVDwKkYlY7fxktSJqh1qcX8iAHYkuGwmSSdJamAaftyVAbBba\\\\\\\/Gqc3hcr7GSHhiKXCclk4o7CUntNRTU2EMyyQjxq+oY3iNgE6Ih2wkd\\\\\\\/xMJT50ogc1K8odQ+4Ngc+v3fBukOabA9UA1eDJX8qsPNs8Gi+7JDE0rd23bVY3u\\\\\\\/ZFBr0usSO8v1PU6sj46F\\\\\\\/E4uO9BP5V6i8V9L5WSg0d9neFzUXuIP5nn5wVf0UUd6xf4XiMBN9b3yLcGgzEps1Hz4ZEdf\\\\\\\/H2q9DrI7UJYfIJlSR6E6SAi+OrqPlzslnIsVou1ziVpL9+BoiV+be5tXB9DvlClMov4RFJqf0YgQcuQpaKx48wYqZ4ynkiu3yadBNv2B9q0lpVqBmHW0qRqGBjuZCHGipcH6yXQqpLENG139XsQadY9uTbn7z3nxlZn2MRGO8ocrE9kemcWa8Bg\\\\\\\/thkJMltvBXnwfITuscZBU0L3Fk8BxpH88iKmCREJ5gapTKcnzll3gpdwp3m5rAiKA2YcPwXyGMX2hPkzXgaFFaWBWMu4HlKpOBpJ\\\\\\\/Vq7yiDUPJrilYq5SL1+c2FpVVnsSG1b9WKVppN3iBWSPKDgGR0Z3VxWDBPG5EinA4iHGKgBmNGm3Yg5+2AyxgQj64k+YdgWATX4JubWaxx3ovoY3EmhYkwyzAklToLsz6rwgF3U3+fufopTvXTXoQzB+YaYRJmCbAigHfgT5V+Sx22BqUIa2L83pwxQdglqjVmS7de4KrTZHc68CQcwGe8dbPUHFp\\\\\\\/sAIvy7lqf3lLarkViYOWKTX0rdhffWkAHAzNEG2D8pTppGzG7yyaP3x8NJFv84LMOLCcuNapAI7nVGvostuSurmDWIZg0d6Hjwh0NVaWF5z+g5tWe\\\\\\\/ZciWrkXGqWTqwIzxtVwTpUWYUqfr+b9Kj0uFYGyuf4dOgInmlYDyT3BRdkTUIpbZzSEMOe0ISg7DclWtzrtD6h2d8L0jpjbfr+X+Gm5J66PmXgRFyhl08IjvjhbG2ASL\\\\\\\/+VOJlPX4qUZq9fChKHtw7fNqMP1VHYYG6M021P3eZVR31oxNTtmlhimIZOxhI\\\\\\\/Tp9Lcs+9L1Q4Zi1XyPlA6tiSjTlQeBvgwdD71xd4qZOJT+bZbWTHXzce+llDJ8FWTflA2VPt1b8rgpT1fq+9Wv0aD3X40lnXZjvVr8PMfoHCZ7Kg1YPBS5CieB5j+ZyRoHDQsHLEec\\\\\\\/mXpsN6t5m0aV5CRaJt4SxkkyN8PkGC\\\\\\\/sCol+++06ur9NAKu4pQJeyK2rS5SWUzhPD6Mo6fNkGR3v3HPzENzi24Mj\\\\\\\/WMcw7b3PmNxx1xhHPmYSE3YrXR726AkPegOC3PzBMC88oDv7d0pJLFVMLym8tnwMwGzjl6Yk6snWSrlsL0CUIDbTF\\\\\\\/wUXHezmuhXEh6TjCRtiznVkt23NjEF5YHY93DpX46zu2FeOGAy9FM87CUMiues1y8zEQFcpOgklFHytzhGOzeOTrEpSvPr5cjYde7t1V+8sfoDMBR4vxSEedWg2pJPAO6X\\\\\\\/lX9lt5nh0yWYO7yKxWhIORtJBNrRFh03JMGsA8QQLXDPEQfE1978lc1JvP9vIXJN0+uEXpeVmqEY99PX\\\\\\\/lF5v4jyRrfbtFhSKR2\\\\\\\/OvlT2sozEMAUQIM27inJNiIApjVhdtR3fdGN8uj87Hc7W+FwvlLHRtbS\\\\\\\/3ge6f6pv5pPmNiFVb1ptfU54t74ukthK8YwfsYIvLix18SzRUDSMm8ck2O4kNeKlXspKsCsLd4vNu1Q2mQktXT5frYE7AFDUJgr84CxzMvWrCTTRgF3Qn+QY16TuAAuS2pWzhbvvfBQ7yqk3OXwChuzXyRwmxuFZKvRUtO2m8EG0tsUgj0tbcwFh\\\\\\\/xVYz9SEXvll2HMV13I6L9E6N7AQnn2oX5BEC\\\\\\\/AwVGsZ5IRR3BAQogfUSmJMJQgfYFqavxxdRdQeBkTHbF2cQ8lQRHaR1yn4SkNjI1zbHAnPRUuXSrAbHGXrFkk+yUfHaomQ6yfMt7QI9YZyryQa1Dm1tOrWfWrU+PZOT2\\\\\\\/iq3p2lXznqBjVP69PNALaGAicaaQ2rSw1a0jrQsb3o5BdTYURilsZCWnxULMScxLbxPFiOT\\\\\\\/9nIfEMEGFIuaWLpDpVu3\\\\\\\/r3ML1PSCQUPzKOEyYWl5hXKxdIP6jZFNM\\\\\\\/SxdNOkunm5YoGSzNwfcw54sssrPqub1O6WIFIrurly0+gVaRNQq9wjmRyIc8wmQ+raGcvjJalek92JYs4UYfhUEUY6w9r\\\\\\\/8EwppMA\\\\\\\/R\\\\\\\/Oz\\\\\\\/B7nA4UGNHz1k0tv578fYQLf7PKI\\\\\\\/Xers2sa1gPcjZBugX7dzadBKw\\\\\\\/VDwzXTMiVA8wVtDCBjGO91yc4H\\\\\\\/QHnmNF0Z05H+JYA2SXkJnPaDQOhG0PAM4Jn6AZ6AGzJd3AiMxpLSxM5etpzQG+sambuRphWOZ\\\\\\\/9ooKFG4ZCzka6ejzqT2s5Lr0\\\\\\\/8edZCGkG43fCkcwR+aOLQ16QEwSHjR\\\\\\\/l+25udylYqUnU76avXjMvvfbw0PeMdTDBZs3X0+LMg9bNbPsFMjhfkKw0rh0ZAnI1apYjrH1YQJQpPaZnABDDptaUnlEicDDcAlCkxdc4lUq5yDAPFJNhzmA6jnjCoVSDS13JdafCzabwt3jlqU1kKthA29oeWZNw5aiPrWJLVLPJ9jP2CzksVD5tHy5D2QmR6k7TZ4UNjKcYdhdvAOh7e895GWj2oh5GLCf359kpJPsBQKkIFIktcWuMjKuRwqpEkSCu9Cs+LcwTeJ0Yr8cB314nsV3bP+fOGcyqLl78x\\\\\\\/bmACqSCQC5V2imvnkqf9FlWfBiUQgjf7tdXHsoWyJ3xBjBigq0o3N19TqZZEdMx8XIJn9mem8aX7oJl4wOQoo2beGaIt+WwysGJuXC1sBcMe9ON6vQP9+Y0PUM289PDmJCgKNleKRf5XvsF7tnkm0PHC6A66IsmgZkFZfziPWqU48ZtOPaLMrh2SFpX2QFDh4QGUj4QbcHkdkInhRNpA8YEfAsWFoeazlY8KagW39kJgEAhPbYoZ6hrrzVXANzCM8RCLE4y4oPv4aY3k0RrUOyoYPEPYJGkSe975YflVnyNcUTrDewnP6iUvPeA3nGzKUr0OYIjVpKt4XgnVB0Mawf8w6BdwESf9NPahag1JPhYjsVUqdQlWrM931micwlVANV6B+QkWirs9e1gNPYa8DqDY\\\\\\\/uNzCm9xM7B2ahdMEjaRsKgwDUz0OvRBbkuyWWETqcCEpym0o9Z1TPst+rZfiYO\\\\\\\/j5\\\\\\\/0MCZVIz9FVbFyYgz6A5hjswcA\\\\\\\/xAEUYua9OMvPuIXetpmhU+QOcoBpRduchtUkrBrz+lgS9Vyl8nim6kwj1hSkOtLF\\\\\\\/YOsOs3uzeNEQljv7HVFF5N56G39UF390Rt1g8o8xdwwI1jSYEjxZX+pb9tBl1U2E4mf7aKeuO86lqrhy5vxa\\\\\\\/ZKLZ63prEcxqxo0rXsZmImA37D2Lfg6oQRhX79kKLMowOuNXji4enAPv5a7FHGx25Y7jiS0gade7cNpfVssUMzdXgYlA192P92XI8dzLHoeGbS9260XshuA+UJB6JxLklBWbcfPYfleu80Ihc0\\\\\\\/7dyD\\\\\\\/ojdhb8ctAmyDc3B+xHhNzldfprMdVVAPlKmcB4wW3y0u+H2h4d9f6\\\\\\\/QNOxalC7Wce0bds5psRMGnwuHpfi+3H2srL01IMnzXYwMtb5vFUmcKxUYY+KlyXJgaCy2qMsCPV9oVDVkNE5gvzA2Vv8P8v6bEgC9BnXblSyggkH29SJbWIHb3l9Cnynr9mZhS+xMaYhYORnVYzD\\\\\\\/GSn86j2h08cLpIwfvYg7uKYYXSnweYyr6arplhYchlMPEKwFj5rem7kMYcFiSM2tPex2lj47NJGX5JcRMLZeLdcIJeiAMZMVvXyAEf8Bacz\\\\\\\/nLIfqPfP7O1O8DOobWzVQlgGJJsRXpjJnmZgR8nvrTT\\\\\\\/Rcg8Ag2InCfYT6u7MZcZ+gZP2kvtjpz5nSYdFaxJZ28yk+PEextEKl2DwBR6KxXw6gCjy4Pb+Q2Ni+0z2Ms\\\\\\\/UL0nWbB5n\\\\\\\/0P8AEGgQaSVsVU\\\\\\\/yea2VlhOffYTdPJ\\\\\\\/47QWaOjZAyAhIFEqi1f7dqu52tHx\\\\\\\/dHiEXD6xEkUFVg3J7cxsRMNhs57B9g74SacdmOhu0VEaHf5DbqPI8qX4f18Q3sLCxJ9lS2Cih3EQuu2yXDlU3L9dCs67ttwJwkmUAtG0mq8N8QZw+tZPOOvX28xep2I26YPIbuQQh0YlRHGp1Ffh251OM5lmOiKjpgzFsRcSg4FZwd+FNw3rK7Jg9y2sGli9rkjNufp0R\\\\\\\/jPZm2wT8r9\\\\\\\/tya62h0RntnEd\\\\\\\/w6Gxh\\\\\\\/yJfgxHtA4tV1b7wmIP6lN5cXxy4evjGHagF6BjVxBT93BjQnzGAOy\\\\\\\/cTllkjZe\\\\\\\/KX7a1BxNxM\\\\\\\/HeboX9yybboLmJ0gOgXwuDXitaxVqiqXhTqXDIu2GAHcjSFqyBFV09k\\\\\\\/5AOoM35fXY4MWig31kiMaoDm9iLAu0+bPz0WHfJ0C0UhBSBvXQ\\\\\\\/r\\\\\\\/HE\\\\\\\/8ZO\\\\\\\/mxPvBVNXPl2LYdAdH4z2wpqf2l8KgL3LydBsM53JC7Ym6wYTREqkf9EmFkaBLNP0ZcLD93uFWyAr\\\\\\\/eiDAnakpje4fuLxdMDHNKEN53OTJ2nMftO38Zu7IrfVlPByUuf9ZUaUE5DTQTFOhZNNM+iC4TDj7sVK4DSWWN2VHcxLYGpI61TB9aPDLtpMCXt8bi34lO+V\\\\\\\/zFYRpBbd+jeF8P5EmuPeNJv8O0e33Nww3eY\\\\\\\/gZ1dztcbNPADbUE6AG9IhgqKR6zum+eJPLChTwFk6UQ90ue+V3o2v1K89y10bQa7QaTYD3lsocNV1zi1XfZHe9G7COCsPdgHtzNiOUf4X9AAb\\\\\\\/cKjiCzNHrO5O7vfvbAiyrnXbgObqm5AclFL1JYfZFapgwiK+EvMpIEQpw18J9Ys\\\\\\\/dzmnlCEypck4tnssKsk6CcgDmvASF8WgEMlumk1\\\\\\\/J7+KLwNjGxNswB0J\\\\\\\/+CxGyx4yta3WmrrlJi5461INqbmQMWinSK9ANrU4xvIMvyxAjiCeGIXtNg74DaTDQDZQOSDMmJYyyz9s82oAlEXdxHGHfThtmebD1hsfEDWL2MTrLwxSX1oYcsrQMPEHa5fPv4pfpJZWLlI1TB8sBxD1sAiHcIX8xfKdrxKjZBOjFLSia7ivtZJNxEMopsQhA0NZ5O5dI9C1gHOR8CFS55Sgw5lBd0Kf2kW0yOXq4ESW5kk55pde2G46tMnEK9klKMdZcnG10GGPAIrcTeO3a5TLA6trggEm70lj7YVPSIRqSzQCcyam8HiTYp2ABDWKGYRjdRupwlE+Okfqat1Mu0YTTYd+IqoKlctftfQobo3p8AQG9tfa0XdTJQaf49KnyvnBtTmKNJ+mZTzKw+iMDCozen7ECmjsMiQ9z6PJ3\\\\\\\/PiNaj577pC0jMq7VROLX1zp\\\\\\\/Www+JKhdA8tebQRbubNyv+ytxt\\\\\\\/f2WHMx+FtMqHu1hSvXvtpI4pPDCnN9cqsdqv0kf+dUJyr32CUZnlp5CrMSWyjtpcZjEk7XLdmznts\\\\\\\/TYZ5BnRqN6CrInR3FmrKqEW12fn9gbMroLUGHR86tXt3llka56LDxFRV4GGK3XgzbWzvWzR6RkMlmzVwgFTAfGD6D9arZyX4nGG\\\\\\\/lj6IPhNExnLjlxUZvcu11BVTfiZg7\\\\\\\/lyaH5R8zL1XQ4NiIObqwHXqsjrICo8nDnknGEyGBT1CgUSDt7hWIfcqX5HdMiQ53cdT7Hof814xi7SvWp0mpmujHt\\\\\\\/oh3\\\\\\\/URmFZpSXpkFoMLb3geLsQ4wqFLEo5yQLthBYvUA29PAXDuXvpG5FJYVoWf+iQEs5jGeHDQN80MqiJuqJ+6su\\\\\\\/W+2yqM7\\\\\\\/43NFTDxB2x1IvPbz+f3PPsQWjjgsBlNKN1pqUgUKQ4AC6CUsvwe4PZSg6owwHMtJRKSMHOrscne7YCZRlFPRQExmqxAmhxRbvgNDHup5qq5rRuLyh+gD60e+9RUR\\\\\\\/dQLXIkLzQ2zyk7fGCRDxjx0ur1XopquC6Jz0TRTTlW2yfGNzAyp6gZ7Nfp6jJH56zt1Jbtmmi\\\\\\\/Mepd2Wfk4jv5e+UR5OGcvRDQvh1WVKcjvcPIwCf5CG1Uuugt6hpXE\\\\\\\/LG\\\\\\\/ydgdEmVos8bg19fEIzerGwHNdRy1E4r9n3tGL8WfKVT4VtWMOJhAbHzM8+LMFzM08g+9vCiqLsEsVzC6gO4L4hzBbDaAMla92JbcpusBsKWo0+NQmpSWsc3JIqO7wrM\\\\\\\/f6N7KqT32UT8nQcUVNqhtqGz8R0q7jEuAy8HbsysFgCoJx3tcVpNcB3AZQ4Bvjt6JpPhlcfU1QvM3\\\\\\\/oBlA+ERhNTcDqVUYtfbsCcJ6xPteMdCIREo4OrU5OBzGwS7xc5J\\\\\\\/8j53U96d+aJYWCJKG\\\\\\\/2Y1ZNmLYRQw\\\\\\\/W5Cop7kyx1iYsYAWV9h3CAkXVvEkc0vu6vLebaevQI5YGhw8UpGyoOhfYlVZgFfOGgqfk+i15SxUo52y0Dc0MJ4MKf1DiepH\\\\\\\/fU+hvvKsP1n9yFMv8H6Lw9X4PeXaf9\\\\\\\/H\\\\\\\/0ir5Ey\\\\\\\/LoR8vZbXiqjvPLWXsmXUKKuXltA9U+wKEIHT65vEPcO2afLwU6THEjKHj1uaXZfdPg+MJPNHzjOHeVs3aztNn2WzSRkCzKbRtBbPbnK+zf5lNEs\\\\\\\/VTIkkVfC4Fl58a8oMstaLb85aYawAM\\\\\\\/Ioj01hWQrRDp948BcEClJ0fMwBbZXVEDn04Plzoeegw\\\\\\\/Ih27MU5iFSjxn\\\\\\\/tVJ7pbr6wumsMSCr8W9F7RI+8YFL6gAIdChXGkrAVYU8IbPR2tOnxsAPxyEsWFTaVOj3dp8IFiqh5rOCpJluc11R53rq7Q2K7iDX9hgwTVLCOduD0kxMGKm5Qee+Nxqc+2BXRvQdnjhKwDmW7a1jFpTg4CIhMOBooRCgItWRSGUojkVZJBTBv1k9p2Fct9DlfovhC3azEspamckJOBWfjr8c9GpTNbuqedoTdl8bFlrvzWbt8Eq4cBxEQ4F22lLUfiFer4UY2JNsYLiophexF+470sacxE87+gkkevFIJ2kXKV2aSvaf4CydBEj3atj9p5QznxDsPusbaqxBwf6TDKHB3oECe80d0CKlfBvPp1RswjLpkL1LTmgUIuCwVzBXUNyXYoDA3bZASxXtPvvHqweZXolypz3kuwpNvz7Dl6VTacHfCugDgTT3andJsNiIR9v1Tx56B91bBV+GuSxulvNuyRtzszueLA\\\\\\\/T9EOrUhl+AnGrhJaYmNGMDJrwZX+haIQmQO0doG1Tj2lk0yNLYiPHkIaaH8X5pBe+mQj\\\\\\\/t3+dRYuFjyODdpHpMXfJ7wsn0V8c2SNW4oVCcGk8hn34onQj4nY0jak5dd5WpRCK+z6llWCdroCtVW2fdZ19b4EpfqSoYLUUayuRZqt4He+5bmjFMMdJsws6Tj2bQC+dlU954pV\\\\\\\/kfn0OSFQUVZChJIEqudG6TUQy9HupqhzrQHFZvVye40wEHUuh+\\\\\\\/vQ8mkSev4j9b33CVsc2qEvbdcwC5tGSyQGrxk\\\\\\\/nc0pXOvCZwHdVUKKOZZt+uSTkgq+552gC8AwgfjPuCphtpiaNYLTJtpLOl5lUoblWfh8e\\\\\\\/FXf3ZvHKvS3XMZb1DSQtDAk\\\\\\\/urLNK1Eocavt8I\\\\\\\/N25oG5FJ7YV3AOt8QRYCNOr9Rt9Z+vBqJ+wR3567uwqWCjb3YFlJdsXPvqQaqyuA6+KOuyyoZiKPqJ7daaWPb9UquM87WEnQ9io\\\\\\\/\\\\\\\/+PK2CyvN3G384d\\\\\\\/s7YnRLUX25OxIoHuEXNrKulVJ+EGNBAruKYGy9ZWYYf39SYnhbM5MbE8n548S\\\\\\\/9\\\\\\\/\\\\\\\/LX73\\\\\\\/owoJ83C\\\\\\\/B37dMX2dCV211txRB4DqwdJ\\\\\\\/45d0Tpuq74iDtQTue0Vm24LLAXoG8KfJNnp8qMEO4RZD+MecvggBB7c6fpejPjN1fkswcN4Ey2Bsf1B1btmhynRuN4aJQojQfB3kdcqtFqIlr15s6szfIDW7wDSJGTewrB123\\\\\\\/BHEdCBwOV0TP8QyjXOkB2svTZdRYTKq\\\\\\\/HXB1Q9PVcTasvb1pL0bcUoqpD1nwXeTmzwl8ya+YGbCnN8OjsfGZTOATt7rNJ5sf4L9Pw00XTw3y9a7g4bGubrce1JzdJ+0YaOnSCJTh9aPYt7kqvBkVrazRkZLHRshIjcRy3cG0rYn14Wkv3BgQbf28pdGgilyIWqwZm09VeCGTuIbjRSiEnUoX0sl2PXQka7ydW99BgqnQnLn20MELgO2G8dZktcHxyI6C9SUKeVbjXhn7t78cjEo5VCBY7PWTDE9Dnc+oVc3KGE18xKqxQtJ4X26OMxrAcEhMW6RjddUSP06ZIx4BQS3pX0Mrz0y1a61jY6ABHwbNxvDUYUdwXOdY+qetsZNt7nTKwQrdpi3aIj8xyyvovqzJgHSPMgiA1r3GoBWzeSGve80+XkJFw+xKByB+wwymCbQkNz\\\\\\\/FSkkYUW\\\\\\\/7Wh9tqQfuTFeocJTs+q99Vp8pb0CJll71p+cgXTTtu9WMznl1WVs37r24yQRr+c5Ju5eEC47if1exUAJGnNop+JTCYPobGk9XP52FnOAxjXSTzkcBKKi3GX2yQxQPX0pmBEpo5xHMiEgecOGJou36xYS0mQfoHBqkKBOsuxP\\\\\\\/ej3L1IKa+h1\\\\\\\/kFo0Sr3FskCt1M4X9+\\\\\\\/y+ndGcqCUG1Isp+H2LONjuSxhnm7y1OMPiFUNpmOlEQ8a+qwecQMu43NoA9Hj0b0kPe3538YsK0JOHKqHzGJQurQ9D+MRbWQWfdrzmSf\\\\\\\/mbhaVbJrTkH2ArUW9Crd3V1eCol\\\\\\\/0aPBRu6nNuXwE2ZaA\\\\\\\/6SrYoMviAn4yZEzT5mokmWARj0Sh2Rs5JNThiTQKa2k2uCRCWNxkdtTUOQD\\\\\\\/6juHORA9rMb0otGneiP8w1q1jq0Hytzsu7YiswFxRB30SpWJsTzP\\\\\\\/wEQCE9q16ER3tDGusFlVVCc1OgcSJDCcTohiuJqnCuF2ED1VlxXtwQBZXEO25Z1yw1L80prNigfQl3xj5uoqJOX5pEJ66VpP0ma+0uaQv8PxdDX5jTl\\\\\\\/gKZHnGhtf\\\\\\\/caqGXi2MUNpoH1ui09W73ARufvtENNlQ\\\\\\\/rr0PEFolSE\\\\\\\/rfQXEPIPDOmgRkqi3Tud86BbCYwo9YsfVPnS42SHri69\\\\\\\/gc7XlzWvcA0OOuVQ5mis14DznxEN8KLUc\\\\\\\/FhHNxwCqBIBeoNQSsjEO+ZblrHnI4Hj2K8nA5aS7pDAEYGvEzoM1Uvu1oVFjBpRQYpUL9euvSyHFbEwZQIrekQspkEE\\\\\\\/CPmCA8JEaMLzKgXJ5luDgdMC1kCLmHb53K+kvWDsI6WHa98He2oWnTu95vdKLr3GfmPZRagd+ykDJTVrSM9VGDIA\\\\\\\/\\\\\\\/vkxv4Qi7KOq7FCk6wWmXP2ptKZdidkZL+1TfH6\\\\\\\/0B7f400gkY0SkgJFGabc8ts8AJZQ4YiWoG80r8sKRlVjpb8TdVJcs9iaa3FAlukfKcZQBvd0DtD+pKLSqFXdphnIWWyQYjL+REp0XjcwOnVEkOXUJdiYXeP6wgNbe\\\\\\\/baDbKVWLmG6YEIMTfIFvbPJWjlwaB78fXvlvh2GbabKc6Hpl4Icpsjtz3kB9Avfk2EyFJ4VTilMAqqHMfxhkOBQMAq87NIeTnwLSe3RhS+83XE69cbPfwGpG3eb58R2U2TJWr2w6MCj1\\\\\\\/v4vLnHzIZdS3Om3acDqCJ+1IlPOn26cdIB3lIMBrNc5ZXk1YBYN+e7X4mG6x8Wt4ve+psbmHX46MqAfXlr+5CTJrjqRhgBWFQQhk4eFTdo1bzEet4pXECZnGmR2vPBgmCJGkFBHcll34fgZH570g1ffR\\\\\\\/k6hSdTzbEc9q8Os8gyjIaJSDDAv4+JratEO1w1Di6ldSobxCU3BqlJZMaLBhXB2QflvjHeeXMa2YkOXibF0QzALFV1erk+cD5efGWqvE1xMsRfe32H0kVCH6UgRsfF2l61CuvzRn9+uzyoO7GTpPzBlyCd6rAgrJEJ3epSheXiqlX1dD0BmSvjlWH3aSueHasQzxW+F5Y9ZCAm+lP09IRyHN6F9cks\\\\\\\/xX6g+KR2+fHgnzoN7gtW\\\\\\\/bZ03zT49bdJH32gKs4cuKBXxcwGtaaYsavNKCf+yh0BFTf8YGd0H7AocxPHLG8NhaqM6fFZo56HFggid33NT9wGzrQsNlS+xIs9F4wE1VSO\\\\\\\/FJPjB\\\\\\\/94Y3\\\\\\\/CcoSBhsebMlJtw79yz\\\\\\\/JCIpLYDbPBGyMHUnSPaOWI455bL9S4rT3zUNoecTPwZG+K\\\\\\\/FPa5r3pTcqVrY48WequlKevtEumnE8XTGfF82YvWHiLyKl6v2Q86nGP4mNrZswq5I\\\\\\\/4jh5uZDq8pEIUYWWZOSgry4zgDgNUCcxUaCcJ+XKtc1etw3+cCiklBC+R8M6uqcj0UvqJtozhfmSWwT4Hnfb7Df04t6CwoMdyk8Q1+yoQup9TRtQZ2STA+oN3+c4\\\\\\\/6nVoo3CVQTE4k2ymIDIz8x94TsTzUTv5TbgI\\\\\\\/JktpZt+NjZLBpi5dx1rJMe3lfbZDaqPsm8DeSrMqVUZIe77hQzEZ2UbdGpdUzyO0TgutKdLyRaAnOvEiJb+OHesdVL6o3aHcqyGEgwb1mf6vKwOZMixfaTzgUlR2QJt4\\\\\\\/xfca2pf7K5F382bdGL42R4WY7GlZswhFmHv05qgUKsTXau7eM5D\\\u0022,\\\u0022iv\\\u0022:\\\u0022JM1BJsoHkfCdUPLWg\\\\\\\/rGYQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222fbac09b7b775b34f4e7c65f1297c1d21cca875b49fb14377efe580b7759c5e4\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PT6ct1RzJpu7hrv34nRQHDJBgAa73j0t8rNPjNx45DPEAAW0Ptx\\\\\\\/aqXoXxw3Ioix0VA8Lsu8BzXjcAPo5mnbbMdOegLY8fGZo+uUJH6yOiyoY1cDDxVKqIAV2PbvJjQl9Om39\\\\\\\/Ypi2cJ\\\\\\\/EZmywJIFBClEx36in84yNFhwuThAuvriwofSuoRDDc+UFddFyX44WSbggP5+Ut6p3UDxvoryekUrFUapuhSm9NlQY\\\\\\\/jd2NHDetsPtK7ccjQO1IWgUk27pDzA8JC7xmXsHliRsVf1lbONuJa7ThmwHfN0pUeX\\\\\\\/v8AWN77XgFPOwNvA5CBS4DaffVB3fS2NGeQfyYxMCQ8OXSrJLSLuPeXjTA7akjyehM++f7wH6F2JYnBWlB0V3TXciQanxr6xrNKAeECqiqy2RfRwrsu+XNVX5vucuA+Qoi8y\\\\\\\/A2RNE47AX32glrX6OoJGGR1lBLNgm6DBMJdCqTMUaHBoSQ9tK4QodKXY+vXfy4mS3hMc3RhKSCIXSzZU1D2C+KRJA6dVLst5BiFpmC5YdmjbZCTGJ\\\\\\\/dYOdpYQlgrvE2UJp5YqbfwoV8EITnO5H0ib1yHTfJpWGbCcM+fBF7ofsLTxmZcDggigFyrjXqHJwdTlQcG1sh70prstLmrslYEZU2bz14w8iTNl53jqHdgTYfgWVwvEKAPLyixbP3hOnG7YpSU7qlEi3tk0b2Q\\\\\\\/w44xMkMIv0geyZSSvxX2zzu+6Bm4iLZBpTXFXzAxXES34uz0Bb4C7U0hGawrS3LcO5qwgHnjA5wQQTDgSzce38EkqOmErAMdF\\\\\\\/Ob24+Dv7KqdknYJ+M3apPiTIWs7u7FaKIJlOMtP0HwJPaTfNpl\\\\\\\/EYerV7eBoGPCM\\\\\\\/d4fRvLNxAxd1tPF5d\\\\\\\/8Aq+igUhqzskYDPF29G9+ZGNhx4mo5ZtaAPqYki2jnfH33l7bw3IoSx4Efr5GN075Z1fL0QeRp4htVhTp7MzYxCAjct9Pnv68EVyQQ+vvhOKceNC8wGpVU6RX6x2uyfk4VyCx23EGPgUpfMin4RD4ecvrelPAqG+x6QEZjyChbhW2ZO2QDGWXRdZ8Be25IYZCXinlBVVL83vndErpFaxqSsPsi3duVoZvJfOXS\\\\\\\/bgyVi4mnQW7RkYW3I0N6USzR+5udY457XPRXR+nhP+8Xs2YlZKIVelhHjQt5XdAmW4DRJsFzuqNtcdXywSIMwfmzk9SZ4+jLxUKEXfFmryyC41lo8nao+EZLA\\\\\\\/\\\\\\\/q5hJlWnUj3Q2+k10dojUdKV0tJmRzIQ4cEVLlYBhFZzfd1qmTwhOPSbg91KnVf999A99BL902NTYF9Ekhzba2iXPPbwzU1sCn7lcAyq9LgPeqakVHmZ8EjOgV6RmIBX8NlkeJKMYIs9eeZIlM8G7sKmQicsedyN7\\\\\\\/6bRZ4MIdMs7TZdE0QH2pbylNO1cEfWo2aTrqfRO\\\\\\\/ldmFDX7UJDQ4nS\\\\\\\/3ycqb\\\\\\\/7TEBHgeiXAP22DFWWAkICms+NtyRA8rdKlDXYURv4Gi4ghFj7FSpsel4w+JET1x4PPr5fkyHnaCMmwrK7DyuFA18TXw0rTtc9QCOOSvpUX6l4AG4bgmXmLMs4tJpst8IDNUrmzHAYv+VHOQfMFmJVptabWAeJHPsEtFgZoh+\\\\\\\/Dks9f3lfvczYZRJ5CMyO1Hn9zWoQXCyLarqWV6voISiwIa8AX87Sb\\\\\\\/q0k8a\\\\\\\/eaJMgAbBkDruN3TjfCM5Yrm3xBXQX94vQeou2cSYhEVkJeYwXaoNsNlTOMUQnxPZgj04YuTnBa9YwXYn4Mt6GzXSovi8H+H8hq8+QLU4U9szHK9Qf0HMNDKGYXmNKmDZXabDqr2qIUo1JxwSzwSw5Owo0BfKYVL93BUaIeHP+MOyZ0PdzgjMNoSgwy3nCM9xsSFr8aLj5310I4p8SLnF\\\\\\\/J\\\\\\\/fLWtbzJ9jH8w3YOBHyt8Vo0lyY5nqy2crdaVB4cLp2tNzZL0\\\\\\\/9aHX0SeaAK7JplD9f5YPgDxjfEcGLij\\\\\\\/+hZj0u7GTiXRhIMHcJZqG9lTMmUbTZdHxPEMN6j\\\\\\\/Y=\\\u0022,\\\u0022iv\\\u0022:\\\u0022ErKrYROHUMj2D7ccldMnMg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002266a73e4d8befa20fa9b30baa53a5e1784bda1bce82ce4ac88ac8c13fd81a0a69\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495528 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/505" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-sWY8wP3gOF1945tEFDJgbGPKw4pdkKvbMVj-63Rcvzk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/505" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-e2p3nba6gMD4OQncfR9yUDgjoVdwuAenhYvrhGJeer8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/2/495" title="Demonstration of a Simple Epitope Tag Multimerization Strategy for Enhancing the Sensitivity of Protein Detection Using Drosophila vAChT" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/2/515" title="Accelerated Evolution of Limb-Related Gene Hoxd11 in the Common Ancestor of Cetaceans and Ruminants (Cetruminantia)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495528" id="node495528" data-pisa="ggg;10/2/505" data-pisa-master="ggg;g3.119.400564" data-apath="/ggg/10/2/505.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Few Stickleback Suffice for the Transport of Alleles to New Lakes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jared Galloway</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-3496-8074" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>William A. Cresko</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-9459-6866" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter Ralph</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">505-514;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400564 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495528" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jared Galloway </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Ecology and Evolution and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jared%2BGalloway%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Galloway%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJared%2BGalloway%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">William A. Cresko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Ecology and Evolution and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BA.%2BCresko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cresko%20WA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BA.%2BCresko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3496-8074" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for William A. Cresko</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Peter Ralph </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Ecology and Evolution and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematics, University of Oregon</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BRalph%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ralph%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BRalph%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9459-6866" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter Ralph</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:plr@uoregon.edu" class="" data-icon-position="" data-hide-link-title="0">plr@uoregon.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/2/505" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495528" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495528/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/2/505.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495528" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495528/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/2/505.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495528" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495528/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/2/505.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495528" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495528/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Threespine stickleback populations provide a striking example of local adaptation to divergent habitats in populations that are connected by recurrent gene flow. These small fish occur in marine and freshwater habitats throughout the Northern Hemisphere, and in numerous cases the smaller freshwater populations have been established “de novo” from marine colonists. Independently evolved freshwater populations exhibit similar phenotypes that have been shown to derive largely from the same standing genetic variants. Geographic isolation prevents direct migration between the freshwater populations, strongly suggesting that these shared locally adaptive alleles are transported through the marine population. However it is still largely unknown how gene flow, recombination, and selection jointly impact the standing variation that might fuel this adaptation. Here we use individual-based, spatially explicit simulations to determine the levels of gene flow that best match observed patterns of allele sharing among habitats in stickleback. We aim to better understand how gene flow and local adaptation in large metapopulations determine the speed of adaptation and re-use of standing genetic variation. In our simulations we find that repeated adaptation uses a shared set of alleles that are maintained at low frequency by migration-selection balance in oceanic populations. This process occurs over a realistic range of intermediate levels of gene flow that match previous empirical population genomic studies in stickleback. Examining these simulations more deeply reveals how lower levels of gene flow leads to slow, independent adaptation to different habitats, whereas higher levels of gene flow leads to significant mutation load – but an increased probability of successful population genomic scans for locally adapted alleles. Surprisingly, we find that the genealogical origins of most freshwater adapted alleles can be traced back to the original generation of marine individuals that colonized the lakes, as opposed to subsequent migrants. These simulations provide deeper context for existing studies of stickleback evolutionary genomics, and guidance for future empirical studies in this model. More broadly, our results support existing theory of local adaptation but extend it by more completely documenting the genealogical history of adaptive alleles in a metapopulation.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/local-adaptation" class="hw-term hw-article-keyword hw-article-keyword-local-adaptation" rel="nofollow">local adaptation</a></li><li class="kwd"><a href="/keyword/threespine-stickleback" class="hw-term hw-article-keyword hw-article-keyword-threespine-stickleback" rel="nofollow">threespine stickleback</a></li><li class="kwd"><a href="/keyword/genomic-simulation" class="hw-term hw-article-keyword hw-article-keyword-genomic-simulation" rel="nofollow">genomic simulation</a></li></ul><p id="p-3">The canonical model for the genetics of adaptation has long been the sequential fixation of new mutations (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Maynard Smith and Haigh 1974</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Endler 1977</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Orr 2005</a>). While it has proved to be a useful baseline for understanding the genetic variation we see in species today, this model is now rightfully understood as incomplete for many species in nature that have more complicated population structures (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lai <em>et al.</em> 2019</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Schrider and Kern 2017</a>). In particular, empirical studies have increasingly identified the need to more deeply incorporate standing genetic variation into adaptation dynamics for metapopulations inhabiting an array of habitats (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hermisson and Pennings 2005</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Barrett and Schluter 2008</a>). Populations experiencing diverse selective pressures while still exhibiting significant gene flow often result in more complex genomic signals that are still not fully understood (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Charlesworth 1997</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Charlesworth <em>et al.</em> 2003</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Nosil <em>et al.</em> 2009</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Flaxman 2013</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Samuk <em>et al.</em> 2017</a>). Concurrently, a growing number of empirical studies have identified instances of convergent evolution using standing genetic variation (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Jones <em>et al.</em> 2012</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Roesti <em>et al.</em> 2012</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Feulner <em>et al.</em> 2015</a>; <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Schrider and Kern 2017</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Barrett and Schluter 2008</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Nelson and Cresko 2018</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Nelson <em>et al.</em> 2019</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bassham <em>et al.</em> 2018</a>). However, it is still not clear how variation in evolutionary processes – such as gene flow, recombination, selection and mutation – can promote the maintenance and re-use of standing genetic variation, particularly during colonization and adaptation to new environments (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Nelson and Cresko 2018</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Pritchard 2010</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Yeaman and Whitlock 2011</a>; <a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Schrider and Kern 2017</a>). It is similarly unclear whether variation in these evolutionary processes can determine the genetic architecture of evolving traits via standing genetic variation. Theoretical work is mostly limited to one or two loci, or relies on approximations of uncertain validity (<em>e.g.</em>, <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Slatkin 1975</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Petry 1983</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Barton and Bengtsson 1986</a>; <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Yeaman and Whitlock 2011</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Ralph and Coop 2015</a>). Therefore, detailed simulation models of specific systems can provide an important complement to empirical studies in the lab and field, by helping us understand precisely how standing genetic variation might fuel local adaptation, and what genomic signals we can expect to see.</p><p id="p-4">The ancestral marine form of threespine stickleback fish (<em>Gasterosteus aculeatus</em>) has given rise to hundreds of thousands to millions of independently derived freshwater populations in recently de-glaciated regions around the Northern Hemisphere (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bell and Foster 1994</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Thompson 1997</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cresko <em>et al.</em> 2007</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hunt <em>et al.</em> 2008</a>). This model organism has provided some of the earliest data showing the heterogeneous nature of divergence across genomes and the much more extensive use of standing genetic variation than once thought (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Schluter and Conte 2009</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">DeFaveri <em>et al.</em> 2011</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">2013</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Roesti <em>et al.</em> 2014</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Nelson and Cresko 2018</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bassham <em>et al.</em> 2018</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Nelson <em>et al.</em> 2019</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hohenlohe <em>et al.</em> 2010</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Terekhanova <em>et al.</em> 2014</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Marques <em>et al.</em> 2016</a>). While geographic isolation often prevents direct migration between freshwater populations, threespine stickleback in them frequently evolve similar phenotypes (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cresko <em>et al.</em> 2004</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Colosimo <em>et al.</em> 2004</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Stuart <em>et al.</em> 2017</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hanson <em>et al.</em> 2016</a>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">2017</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hirase <em>et al.</em> 2014</a>). The most recent evolutionary genomic studies on stickleback document that while the overall dynamic of local adaptation to marine and freshwater habitats has been occurring for millions of years (<a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Nelson and Cresko 2018</a>; <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Nelson <em>et al.</em> 2019</a>), independent local adaptation of marine individuals to freshwater environments has been observed to take place in just tens of generations (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Terekhanova <em>et al.</em> 2014</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lescak <em>et al.</em> 2015</a>; <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Bassham <em>et al.</em> 2018</a>). For example, in 1964 the Great Alaskan Earthquake caused an uplift of many islands and coastal regions throughout the Gulf of Alaska. Studies of stickleback populations on uplifted Middleton Island showed that newly created freshwater ponds were invaded by the surrounding marine population of stickleback which evolved the freshwater syndrome of phenotypes in less than 50 years (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Lescak <em>et al.</em> 2015</a>; <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Bassham <em>et al.</em> 2018</a>). Amazingly, the portions of the genomes of these populations that showed increased divergence from the oceanic population mirrored those previously found to differ between ocean and freshwater populations that that have been geographically separated for thousands of years (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Hohenlohe <em>et al.</em> 2010</a>; <a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Bassham <em>et al.</em> 2018</a>).</p><p id="p-5">But how can evolution occur at such a rapid pace? Waiting for new mutations to arise in each lake or pond would take much longer than the decades since the Alaskan earthquake with any plausible target size. Even more improbable is having divergence cluster in such similar genomic regions across independent populations. An alternative hypothesis is that the majority of alleles important for freshwater adaptation are present in the marine individuals due to recurrent gene flow from freshwater back in to marine populations. <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Schluter and Conte (2009)</a> proposed a conceptual model they termed the “transporter” hypothesis to describe the process by which alleles beneficial in freshwater environments are maintained at migration-selection balance in the larger oceanic population and therefore available to be utilized during subsequent adaptation to new freshwater habitats. (The alleles conferring adaptation to freshwater environments are thereby “transported” to and reassembled in new lakes.) The first clear example of the global reuse of such alleles in stickleback was the gene <em>eda</em> which has been shown to be an important regulator for the number of bony lateral plates (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Colosimo <em>et al.</em> 2004</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Cresko <em>et al.</em> 2004</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Colosimo <em>et al.</em> 2005</a>). While the low lateral plate version of this gene arose millions of years ago, it is found in much younger freshwater ponds around the Northern Hemisphere (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">O’Brien <em>et al.</em> 2015</a>). More recently, genome-wide haplotype analyses have provided evidence that <em>most</em> regions of the genome that distinguish marine-freshwater genetic differences share this pattern (Nelson and Cresko 2017).</p><p id="p-6">While the growing body of population genomic data on stickleback evolution supports the transporter hypothesis, a number of questions remain. What are the actual population sizes, migration rates, and fitness differentials consistent with this hypothesis? How many differentially selected alleles exist, how many are used at any one time, and how are they arranged within the genome? A curious natural history observation underlying many of these questions is the fact that some newly formed freshwater habitats, such as the ponds on Middleton Island, are quite small and presumably the number of initial marine migrants is few. The variation carried by these few initial migrants might therefore be a small subset of the total variation, and thus be insufficient to fuel adaptation without subsequent influx of alleles from additional generations of marine migrants carrying the remaining freshwater adaptive alleles.</p><p id="p-7">Here, we use individual-based forward simulations implemented in SLiM that incorporates selection on a quantitative trait explicitly determined additively from a realistically long genome to model the stickleback metapopulation and address these questions (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Haller and Messer 2017</a>, 2018). We ask how variation in amount of gene flow affects the genetic architecture of local adaptation to newly created freshwater ponds. Because we record the entire genealogy of all alleles (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kelleher <em>et al.</em> 2018</a>), we can determine the distribution and abundance of haplotypes across all marine and freshwater populations to know the timing and proportion of potentially adaptive alleles that are actually utilized in each population. In addition, we can document how these adaptive alleles are distributed across the genomes, and as a consequence determine how this may affect the efficacy of genome scans for between-habitat differentiation.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><p id="p-8">To explore these questions, we used SLiM (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Haller and Messer 2017</a>, 2018) to implement forward-time simulation of populations of individuals with explicitly represented genomes in which selection acted upon a single continuous quantitative trait. The details of the model were motivated by current understanding of threespine stickleback evolutionary, history and demography. The following model included divergent selection in the two habitats which had substantial spatial structure. Certain aspects of the model remain simplistic due to computational constraints; in particular, total population sizes are much smaller than in reality, although we may capture a good picture of local dynamics.</p><div id="sec-2" class="subsection"><h3>Habitat and geography</h3><p id="p-9">Our simulations include two habitat types – marine and freshwater – defined by the nature of their selective pressures, each with 5,000 diploid individuals. The arrangement of these habitats, depicted in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, roughly models a set of freshwater habitats along a stretch of coastline. The marine habitat is a continuous, one-dimensional range of length 25 units, while the freshwater habitat is divided into 25 discrete subpopulations (which we call “lakes”), each connected to the marine habitat at regularly spaced intervals (positions <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-1.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-2.gif"/></noscript></span></span>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/505/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Diagram of simulated populations: a single, continuous, one-dimensional marine habitat (blue) is coupled to &#x201C;lakes&#x201D;, in which stickleback randomly mate, at discrete locations with arrows representing migration patterns. After an initial period of 100K generations with 25 lakes, an additional 25 lakes are added (at the same set of locations) and populated with marine individuals to simulate the appearance of newly accessible freshwater habitats colonized by marine stickleback. The marine habitat, and each set of 25 lakes, each contain 5,000 individuals at all times." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1073803408" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Diagram of simulated populations: a single, continuous, one-dimensional marine habitat (blue) is coupled to &#x201C;lakes&#x201D;, in which stickleback randomly mate, at discrete locations with arrows representing migration patterns. After an initial period of 100K generations with 25 lakes, an additional 25 lakes are added (at the same set of locations) and populated with marine individuals to simulate the appearance of newly accessible freshwater habitats colonized by marine stickleback. The marine habitat, and each set of 25 lakes, each contain 5,000 individuals at all times.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/505/F1.medium.gif" width="440" height="295"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/2/505/F1.medium.gif" width="440" height="295"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/505/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/505/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495513" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">Diagram of simulated populations: a single, continuous, one-dimensional marine habitat (blue) is coupled to “lakes”, in which stickleback randomly mate, at discrete locations with arrows representing migration patterns. After an initial period of 100K generations with 25 lakes, an additional 25 lakes are added (at the same set of locations) and populated with marine individuals to simulate the appearance of newly accessible freshwater habitats colonized by marine stickleback. The marine habitat, and each set of 25 lakes, each contain 5,000 individuals at all times.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">Divergent selection is mediated by a single quantitative trait with different optima in marine and freshwater habitats. This situation roughly models the cumulative effect of the various phenotypes thought to be under divergent selection between the habitats, such as armor morphology, body size, craniofacial variation and opercle shape. The optimal trait values in the marine and freshwater habitats are <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-3.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-4.gif"/></noscript></span></span> respectively, and fitness of a fish with trait value <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-5.gif"/></noscript></span></span> in a habitat with optimal value <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-6.gif"/></noscript></span></span> is determined by a Gaussian kernel with standard deviation 15, <em>i.e.</em>,</p><span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-7.gif"/></noscript></span></span><p id="p-12">Thus, the difference between an individual’s trait value and the optimum determines that individual’s fitness. We chose the difference between optima and strength of stabilizing selection in each habitat so that (a) around 10 (diploid, homozygous) mutations were sufficient to move from one optimum to the other, and (b) well-adapted fish from one habitat would have low, but nonzero, fitness in the other habitat.</p></div><div id="sec-3" class="subsection"><h3>Genetic architecture of the trait</h3><p id="p-13">Individuals are diploid, with a linear chromosome <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-8.gif"/></noscript></span></span> bp long, with recombination events occurring at a rate of <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-9.gif"/></noscript></span></span> per bp per generation. Mutations that can affect the trait under selection occur at rate <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-10.gif"/></noscript></span></span> per locus per generation in ten regions of <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-11.gif"/></noscript></span></span> loci each, spread evenly along the chromosome. This translates to a total adaptive target size of <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-12.gif"/></noscript></span></span> per individual per generation, or <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-13.gif"/></noscript></span></span> trait-affecting mutation, per population of 5,000 diploids, per generation. This architecture was chosen to model a moderate number of clusters of trait-affecting genes, each of which having about 1Kb of potentially adaptive target (since mutation rates are closer to <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-14.gif"/></noscript></span></span>, a mutation rate of <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-15.gif"/></noscript></span></span> is roughly equivalent to having potentially adaptive sites at one of every 100 bases). Each mutation in these regions is either additive, completely recessive, or completely dominant (with equal probability). Effect sizes for these mutations are chosen randomly from an exponential distribution with mean <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-16.gif"/></noscript></span></span>, either positive or negative with equal probability. Individual trait values (<span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-17.gif"/></noscript></span></span>) are determined additively from the diploid genotypes. Concretely, an individual that is heterozygous and homozygous for mutations at sets of loci <em>H</em> and <em>D</em> respectively has trait value <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-18.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-19.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-20.gif"/></noscript></span></span> are the dominance coefficient and the effect size of the mutation at locus <em>i</em>. Subsequent mutations at the same locus replace the previous allele.</p></div><div id="sec-4" class="subsection"><h3>Population dynamics</h3><p id="p-14">We use SLiM to simulate a Wright–Fisher population with non-overlapping generations and a fixed population size of 5,000 diploid individuals in each habitat. Each generation, the two parents of each new offspring are chosen proportional to their fitness (unlike actual stickleback, all individuals are hermaphroditic), and the contributing genomes are produced by Poisson recombination with an average of one crossover per chromosome per generation (<span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-21.gif"/></noscript></span></span> per locus per generation). Since the total population across <em>all</em> 25 lakes is fixed at 5000, and the Wright–Fisher model assumes unrealistic global population regulation, we normalize the fitnesses of each individual so that approximately 200 offspring are generated in each lake, each generation. (A simpler implementation would use SLiM’s “non-Wright-Fisher” model type, which was not available when we wrote the simulations; but we do not expect the difference to affect results.) To do this, we divide fitness values of each freshwater individual by the mean fitness in their lake, so that the mean fitnesses of all lakes are equal before selection happens.</p><p id="p-15">As depicted in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> the marine population is continuous (one-dimensional) while lakes are discrete. Dispersal occurs both locally along the coastline within the marine habitat, as well as between the marine habitat and the lakes, with a lake–ocean migration rate denoted <em>m</em>. No migration occurs directly between the freshwater populations. All individual dispersal events can be thought of as occurring at the juvenile stage in the life cycle of the simulation. Each new individual in each habitat has parents from the other habitat with probability <em>m</em> (in which case we call it a “migrant”), and parents from the same habitat with probability <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-22.gif"/></noscript></span></span>. The first parent of each non-migrant individual in the freshwater habitat is chosen from the freshwater habitat proportional to fitness, and a mate is chosen from the same lake as the first, also proportional to fitness. The offspring then lives in the same lake as the parents. Parents for each non-migrant marine individual are chosen similarly: first, a single parent is chosen proportionally to fitness in the marine habitat, and then a mate is chosen, also proportionally to fitness but re-weighted by a Gaussian function of the distance separating the two, with standard deviation <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-23.gif"/></noscript></span></span>. Concretely, if the first parent is marine individual <em>i</em>, then marine individual <em>j</em> is chosen as the mate with probability proportional to <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-24.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-25.gif"/></noscript></span></span> is the fitness of individual <em>j</em> and <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-26.gif"/></noscript></span></span> is the distance between the two locations. Finally, each new marine offspring is given a position displaced from the first parent’s position by a random Gaussian distance with mean 0 and standard deviation 0.5, and reflected to stay within the habitat range. Parents for each freshwater migrant are chosen in the same way as for non-migrant marine individuals, and are assigned to the lake nearest to the position of the first marine parent. Similarly, parents for each marine migrant are both chosen from the same lake as before, and the offspring is given a spatial location in the marine habitat at the location of the parent’s lake.</p></div><div id="sec-5" class="subsection"><h3>Colonization of newly formed lakes</h3><p id="p-16">To study how marine-derived populations adapt after colonizing newly appearing freshwater habitats, we introduce a new set of 25 lakes after 100,000 generations. These new lakes are populated with marine individuals to emulate a freshwater lake being colonized by oceanic stickleback that had previously been exchanging alleles with older freshwater populations. This creates two <em>sets</em> of lakes, which along with the marine population have a total of 15,000 individuals. Since this introduction of lakes doubles the number of lake-to-marine immigrants, the probability that a new marine individual has freshwater parents is <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-27.gif"/></noscript></span></span> instead of <em>m</em>.</p></div><div id="sec-6" class="subsection"><h3>Recording genealogical history</h3><p id="p-17">We used SLiM’s ability to record <em>tree sequences</em> (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Haller <em>et al.</em> 2019</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kelleher <em>et al.</em> 2016</a>) to output the genealogical history of all individuals at the time of introduction of new lakes, at the time of adaptation, and at the end of the simulation. This allowed us to directly query the true origins of adaptive alleles. In addition, it allowed for much larger simulations by avoiding the computationally expensive task of simulating neutral mutations which were retroactively added to the gene trees at a rate of <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-28.gif"/></noscript></span></span> per locus per generation, as described in <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Kelleher <em>et al.</em> (2018)</a>. The tree sequence output by each simulation allows us to explore the origin of the genetic basis of adaptation in the new lakes. To do this, we constructed the genealogical tree relating all extant chromosomes at each locus along the genome. Using these trees we classified each adaptive allele, in each genome in the new lakes at the time of adaptation, into four categories:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-18"><em>a “De novo” allele:</em> deriving from a new mutation that occurred in a new lake.</p></li><li id="list-item-2"><p id="p-19"><em>a “Migrant” allele:</em> deriving from a migrant not in the initial generation that colonized the lake</p></li><li id="list-item-3"><p id="p-20"><em>a “Captured” allele:</em> present in initial colonists of the new lake, and both common (above 50%) in the original lakes, and uncommon (below 50%) in the ocean.</p></li><li id="list-item-4"><p id="p-21"><em>a “Marine” allele:</em> present in initial colonists of the new lake, but not a “captured” allele.</p></li></ol><p id="p-22">The proportion of trait-affecting alleles in new lakes that fall in these categories measures the degree to which selection in the new environments made use of (1) new mutation, (2) post-colonization migration, (3) standing variation at migration–selection balance, and (4) standing variation at mutation–selection balance.</p><p id="p-23">We used neutral mutations to calculate <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-29.gif"/></noscript></span></span> for each locus, which we then averaged in windows as a measure of between-population relative differentiation. Concretely, if <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-30.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-31.gif"/></noscript></span></span> are the frequencies of a given mutant allele in the freshwater and marine habitats, respectively, and <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-32.gif"/></noscript></span></span>, then we compute <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-33.gif"/></noscript></span></span> for that mutation as <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-34.gif"/></noscript></span></span> (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gillespie 1998</a>).</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-24">The source code used to run and analyze the simulations in this paper are available at <a href="https://github.com/jgallowa07/SticklebackPaper">https://github.com/jgallowa07/SticklebackPaper</a>. Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.10876052">https://doi.org/10.25387/g3.10876052</a>.</p></div></div><div class="section results-discussion" id="sec-8"><h2 class="">Results &amp; Discussion</h2><p id="p-25">To observe the impact of gene flow on selection in the new lakes, we varied the ocean–lake migration rate, <em>m</em>, across separate simulations from <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-35.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-36.gif"/></noscript></span></span>. Below, we often refer to these migration rates in terms of the number of migrants per lake, per generation, which we denote <em>M</em>. Since each lake contains 200 individuals, <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-37.gif"/></noscript></span></span>. Many aspects of adaptation changed substantially across this range, including the speed of adaptation, degree of sharing of adaptive alleles among lakes, and the population genetic signals left behind. At very low rates of gene flow, each new lake’s population adapted almost completely independently through <em>de novo</em> mutation, which took a very long time (<span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-38.gif"/></noscript></span></span> generations). At very high rates of gene flow, local adaptation was constrained by the large influx of locally maladaptive alleles. Between these two extremes, genetic variation that allowed adaptation to freshwater habitats could move relatively easily between lakes. Perhaps surprisingly, only a few migrants per generation from the lakes to the ocean were needed to maintain sufficient genetic variation in the ocean to dramatically accelerate adaptation in new lakes.</p><div id="sec-9" class="subsection"><h3>Rapid local adaptation at intermediate gene flow rates</h3><p id="p-26">Local adaptation occurred in all simulations with the exception of the highest gene flow at which half of each population was composed of migrants, as can be seen by the mean trait values of <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figures 3</a> and S3 show how mean trait values in freshwater and marine populations diverged over time, until the trait means were close to the optimal values in each habitat. Establishment of adaptive alleles in the old lakes during the first few thousand generations manifests as jumps in the mean trait value – which move the trait by an amount of order 1 every few hundred generations (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). At the lowest rate of gene flow, <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-39.gif"/></noscript></span></span>, differences at around 20 commonly polymorphic sites (about 10 that shift the trait in each direction) were responsible for most of the adaptive differences between freshwater and marine habitats. As expected, increasing migration rate decreased differentiation between habitats: as seen in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-40.gif"/></noscript></span></span> between marine and freshwater habitats at neutral sites steadily declines as migration increases.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/505/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of mean individual trait values across generations of the simulation (recording started after introduction of the new lakes at 100,000 generations), for different migration rates. The dashed yellow and gray lines at  give the optimum phenotypes in the marine and freshwater environments, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1073803408" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of mean individual trait values across generations of the simulation (recording started after introduction of the new lakes at 100,000 generations), for different migration rates. The dashed yellow and gray lines at &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-41&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; give the optimum phenotypes in the marine and freshwater environments, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/505/F2.medium.gif" width="440" height="211"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/2/505/F2.medium.gif" width="440" height="211"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/505/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/505/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495680" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-27" class="first-child">Distribution of mean individual trait values across generations of the simulation (recording started after introduction of the new lakes at 100,000 generations), for different migration rates. The dashed yellow and gray lines at <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/F2/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/F2/embed/mml-math-41.gif"/></noscript></span></span> give the optimum phenotypes in the marine and freshwater environments, respectively.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/505/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mean individual trait values in the marine habitat (red line), the original lakes (tan lines; average thicker), and the new lakes (black lines; average thicker), across the course of two simulations, with migration rates of (A) and (B) migrants per lake per generation, respectively. Optimal trait values in the two habitats are at . Analogous plots for other migration rates are shown in Figure S3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1073803408" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Mean individual trait values in the marine habitat (red line), the original lakes (tan lines; average thicker), and the new lakes (black lines; average thicker), across the course of two simulations, with migration rates of (A)&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-42&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and (B)&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-43&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; migrants per lake per generation, respectively. Optimal trait values in the two habitats are at &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-44&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. Analogous plots for other migration rates are shown in Figure S3.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/505/F3.medium.gif" width="440" height="316"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/2/505/F3.medium.gif" width="440" height="316"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/505/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/505/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495757" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-28" class="first-child">Mean individual trait values in the marine habitat (red line), the original lakes (tan lines; average thicker), and the new lakes (black lines; average thicker), across the course of two simulations, with migration rates of (A)<span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/F3/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/F3/embed/mml-math-42.gif"/></noscript></span></span> and (B)<span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/F3/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/F3/embed/mml-math-43.gif"/></noscript></span></span> migrants per lake per generation, respectively. Optimal trait values in the two habitats are at <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/F3/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/F3/embed/mml-math-44.gif"/></noscript></span></span>. Analogous plots for other migration rates are shown in Figure S3.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/505/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average  in windows of 500bp between: (top) marine habitat and old lakes; (middle) marine habitat and new lakes; and (bottom) old and new lakes. The columns are ordered by M (the number of migrants per lake, per generation) and show  values for a separate simulation,  is calculated between all marine individuals, and all lakes pooled together. All locations of pre-existing freshwater adapted alleles have been highlighted by partially transparent pink dashed vertical lines, so darker shades of pink show more freshwater adapted alleles at that location." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1073803408" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Average &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-45&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; in windows of 500bp between: (top) marine habitat and old lakes; (middle) marine habitat and new lakes; and (bottom) old and new lakes. The columns are ordered by &lt;em&gt;M&lt;/em&gt; (the number of migrants per lake, per generation) and show &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-46&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; values for a separate simulation, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-47&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is calculated between all marine individuals, and all lakes pooled together. All locations of pre-existing freshwater adapted alleles have been highlighted by partially transparent pink dashed vertical lines, so darker shades of pink show more freshwater adapted alleles at that location.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/505/F4.medium.gif" width="440" height="292"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/10/2/505/F4.medium.gif" width="440" height="292"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/505/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/505/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495712" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-29" class="first-child">Average <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/F4/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/F4/embed/mml-math-45.gif"/></noscript></span></span> in windows of 500bp between: (top) marine habitat and old lakes; (middle) marine habitat and new lakes; and (bottom) old and new lakes. The columns are ordered by <em>M</em> (the number of migrants per lake, per generation) and show <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/F4/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/F4/embed/mml-math-46.gif"/></noscript></span></span> values for a separate simulation, <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/F4/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/F4/embed/mml-math-47.gif"/></noscript></span></span> is calculated between all marine individuals, and all lakes pooled together. All locations of pre-existing freshwater adapted alleles have been highlighted by partially transparent pink dashed vertical lines, so darker shades of pink show more freshwater adapted alleles at that location.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">Adaptation occurred much more quickly at higher migration rates, both in the old and new sets of lakes. We measured this “time to adaptation” as the number of generations until average trait values in old and new lakes were within 0.5 of each other, shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> for different rates of gene flow. Adaptation of new lakes took over 18,000 generations at the lowest rate of <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-48.gif"/></noscript></span></span>, while at <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-49.gif"/></noscript></span></span> migrant per lake per generation, new lakes adapted in just under 60 generations.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/505/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Time to adaptation as a function of migration rate. The time to adaptation is measured as the number of generations until the introduced population&#x2019;s mean phenotype comes within 0.5 of the original lakes average phenotype. Each point represents a single simulation run. (Adaptation did not occur at the highest rate of gene flow.)" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1073803408" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Time to adaptation as a function of migration rate. The time to adaptation is measured as the number of generations until the introduced population&#x2019;s mean phenotype comes within 0.5 of the original lakes average phenotype. Each point represents a single simulation run. (Adaptation did not occur at the highest rate of gene flow.)&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/505/F5.medium.gif" width="440" height="264"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/10/2/505/F5.medium.gif" width="440" height="264"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/505/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/505/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495778" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-31" class="first-child">Time to adaptation as a function of migration rate. The time to adaptation is measured as the number of generations until the introduced population’s mean phenotype comes within 0.5 of the original lakes average phenotype. Each point represents a single simulation run. (Adaptation did not occur at the highest rate of gene flow.)</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Widespread allele sharing among lakes</h3><p id="p-32">Adaptive alleles were shared between lakes at many rates of gene flow, but not at the lowest. At low migration rates, the <em>initial</em> period of adaptation takes roughly 25 times longer for lakes than it does for the ocean. This difference occurs because in the absence of gene flow, each lake must wait for its own novel mutations to arise in order to adapt. Because the marine habitat is continuous, with 25 times more individuals than any one lake, there is a much larger influx of new mutations to be selected upon. At higher migration rates, greater mixing allows the initial lakes to share alleles instead of developing their own genetic basis for adaptation.</p><p id="p-33">To investigate in more depth how locally adaptive alleles found in the original lakes are shared among newly derived lakes, as well as how they spread to the new lakes, we defined and tracked the distribution of “pre-existing freshwater adapted alleles” at the beginning of each generation. To be considered in this category, an allele must participate in the genetic basis of local adaptation for at least one of the original lakes. Concretely, these are any non-neutral mutations whose frequency is above 50% in at least one original lake and below 50% in the marine habitat. (So, “captured” alleles as defined above are pre-existing, but not all pre-existing alleles are captured.) <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a> shows the distribution of the number of these alleles across generations. At <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-50.gif"/></noscript></span></span>, each lake has a private set of about 10 mutations nearly fixed in that lake but not elsewhere: new lakes independently acquire new adaptive alleles rather than pre-existing ones. At <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-51.gif"/></noscript></span></span>, we again observe the original lakes adapting nearly independently from each other, but now the new lakes adapt using pre-existing alleles present in the original set of lakes. Concurrently, the average marine individual carries <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-52.gif"/></noscript></span></span> pre-existing freshwater adapted alleles, standing variation which was nearly absent at <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-53.gif"/></noscript></span></span>. As migration rate increases past this, the total number of pre-existing freshwater adapted alleles declines (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6C</a>). Interestingly, the frequency of these alleles in the ocean stays relatively constant across the reasonable rates of gene flow. We explored the possibility of dominance being a good predictor of how long an adaptive allele can be maintained in the marine environment, but found no effect strong enough to be seen with our relatively small sample size of adaptive alleles.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/505/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Amount of standing freshwater variation by habitat, across migration rates. Each plot counts &#x201C;pre-existing freshwater adapted alleles&#x201D;, that are common in the original lakes but rare in the ocean (see text for definition). (A) Mean number of these alleles per individual. (B) Mean percentage of these alleles per individual. (C) Total number of these alleles (so, ). The number of alleles meeting these conditions changes over the course of the simulation, and each plot shows distributions of these values throughout each simulations. The horizontal axis shows M, the mean number of migrants per lake per generation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1073803408" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Amount of standing freshwater variation by habitat, across migration rates. Each plot counts &#x201C;pre-existing freshwater adapted alleles&#x201D;, that are common in the original lakes but rare in the ocean (see text for definition). (A) Mean number of these alleles per individual. (B) Mean percentage of these alleles per individual. (C) Total number of these alleles (so, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-54&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). The number of alleles meeting these conditions changes over the course of the simulation, and each plot shows distributions of these values throughout each simulations. The horizontal axis shows &lt;em&gt;M&lt;/em&gt;, the mean number of migrants per lake per generation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/505/F6.medium.gif" width="440" height="232"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/10/2/505/F6.medium.gif" width="440" height="232"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/505/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/505/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495698" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-34" class="first-child">Amount of standing freshwater variation by habitat, across migration rates. Each plot counts “pre-existing freshwater adapted alleles”, that are common in the original lakes but rare in the ocean (see text for definition). (A) Mean number of these alleles per individual. (B) Mean percentage of these alleles per individual. (C) Total number of these alleles (so, <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/F6/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/F6/embed/mml-math-54.gif"/></noscript></span></span>). The number of alleles meeting these conditions changes over the course of the simulation, and each plot shows distributions of these values throughout each simulations. The horizontal axis shows <em>M</em>, the mean number of migrants per lake per generation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35"><a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a> shows the distribution, through time, of the mean <em>percentage</em> of currently-defined freshwater adapted alleles that each genome in each of the populations carries. If all individuals across lakes carried the same set of alleles determining their trait value, this would be 100%. At the lowest migration rate (<span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-55.gif"/></noscript></span></span>), each genome in the original lakes have almost exactly <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-56.gif"/></noscript></span></span> of the total number of pre-existing freshwater adapted alleles – this is because each one of the 25 lakes has adapted with a unique set of alleles. Since these are <em>pre-existing</em> alleles, the value is zero for introduced lakes. <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6A</a> shows us that at 0.1 migrants per lake per generation and above, the average individual across the new lakes has nearly the same amount of pre-existing freshwater adapted alleles as individuals across the old lakes. As expected, the genetic basis of the freshwater phenotype seems to simplify as migration increases – higher rates of migration allow adaptive alleles of larger effect to travel more efficiently through the population, even though they are deleterious in the ocean.</p><p id="p-36">The numbers in <a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6</a> strongly suggest that the dramatic increase in speed of local adaptation we observed above occurs because higher gene flow allows sharing of freshwater alleles among populations. We confirmed this finding by using recorded tree sequences to identify the origin of each trait-affecting allele in each individual in the new lakes, as defined in the Methods. <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> shows that at the lowest rate of gene flow the majority of adaptive alleles are derived from “<em>de novo</em>” mutation. As gene flow increases, a larger fraction of adaptive alleles derive from pre-existing variation in the marine population at the time of introduction. In other words, greater mixing at higher migration rates allows lakes to share alleles instead of developing their own genetic basis of adaptation.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/505/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(Origin of adaptive alleles:) Each bar plot shows the origins of all trait-affecting alleles above frequency 50% in at least one new lake, classified as (red) new mutations, (black) post-colonization migrants, (tan) &#x201C;captured&#x201D; from pre-existing lakes, or (grey) standing marine variation. See Methods for precise definitions of these categories." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1073803408" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(Origin of adaptive alleles:) Each bar plot shows the origins of all trait-affecting alleles above frequency 50% in at least one new lake, classified as (red) new mutations, (black) post-colonization migrants, (tan) &#x201C;captured&#x201D; from pre-existing lakes, or (grey) standing marine variation. See Methods for precise definitions of these categories.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/505/F7.medium.gif" width="440" height="236"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/10/2/505/F7.medium.gif" width="440" height="236"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/505/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/505/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495665" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-37" class="first-child">(Origin of adaptive alleles:) Each bar plot shows the origins of all trait-affecting alleles above frequency 50% in at least one new lake, classified as (red) new mutations, (black) post-colonization migrants, (tan) “captured” from pre-existing lakes, or (grey) standing marine variation. See Methods for precise definitions of these categories.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">While increased migration allows sharing of adaptive alleles among lakes, at <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-57.gif"/></noscript></span></span> migrants per lake per generation the constant influx of alleles between the habitats creates substantial migration load. This level of gene flow only replaces 5% of each population each generation with migrants from the other habitat, but is sufficient to shift the mean trait values to nearly half their optimal values, as seen in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>.</p><p id="p-39">We find that, unsurprisingly, sharing of alleles across lakes leads to a very simple genetic basis – only tens of loci, at <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-58.gif"/></noscript></span></span>. This suggests that all else being equal, systems with higher migration should have fewer loci responsible for local adaptation. However, “all else is equal” is rarely plausible: the actual number of locally adaptive alleles in a particular system depends on many other factors, including the strength of selection, degree of pleiotropy, and the genetic basis of the trait. A different genetic basis could well lead to a much larger number of alleles, even at high migration rates: but, could these be reliably identified? Next, we look at how well adaptive alleles are identified with genome scans.</p></div><div id="sec-11" class="subsection"><h3>The efficacy of genomic scans of selection depends on gene flow</h3><p id="p-40">Here, we take a closer look at the genomic architecture of local adaptation between the two habitats. Can measures of local differentiation such as <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-59.gif"/></noscript></span></span> be used to identify the causal loci? This question has been extensively studied, both with simulation (<em>e.g.</em>, <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Southcott and Kronforst 2017</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lotterhos and Whitlock 2015</a>) and in the wild (reviewed in <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Harrison and Larson 2016</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hoban <em>et al.</em> 2016</a>); our simulations provide a detailed look where the truth is known. <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> shows plots along the genome of average per-locus <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-60.gif"/></noscript></span></span> values in 500bp windows between the marine habitat and all freshwater habitats pooled together. These <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-61.gif"/></noscript></span></span> values are calculated using all polymorphic loci, including both those that affect phenotype as well as neutral variants. Higher rates of dispersal showed more distinct <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-62.gif"/></noscript></span></span> peaks over polymorphic loci, while “background” levels of <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-63.gif"/></noscript></span></span> increase as gene flow decreases, swamping out this signal until the regions under selection are indistinguishable. This is likely the contribution of two separate forces of natural selection: first, stronger genetic drift with less migration leads to higher background <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-64.gif"/></noscript></span></span>, and second, greater sharing of adaptive alleles providing a shared signal across populations.</p><p id="p-41">At first glance, this suggests that genome scans for local adaptation based purely on measures of differentiation will only be successful given enough migration between habitats. But how many of these peaks are actually underlying trait differences that form the basis for local adaptation? To quantify this, Figure S1 shows the power and true positive rates that would be obtained by an <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-65.gif"/></noscript></span></span> cutoff that declared everything above a certain value to be a causal locus.</p><p id="p-42">What we observe most plainly in this graph is that <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-66.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-67.gif"/></noscript></span></span> migrants per lake per generation are the only rates of dispersal at which a large percentage of peaks (above <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-68.gif"/></noscript></span></span>) actually lie on top of regions which impact trait differences. Unfortunately, the low statistical power for all rates of <em>M</em> would seem to suggest that many regions which impact trait differences do not appear as <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-69.gif"/></noscript></span></span> peaks. In other words, with high rates of dispersal between population, <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-70.gif"/></noscript></span></span> peaks may reliably identify causal loci, but not all causal loci will appear as <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-71.gif"/></noscript></span></span> peaks. It’s important to note that <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-72.gif"/></noscript></span></span> in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figures 4</a> and S1 was calculated after pooling together all lakes and all marine individuals. For comparison, Figures S2 and S4 show <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-73.gif"/></noscript></span></span> calculated with individuals only in the first fifth of the range (5 lakes and corresponding marine habitat). These show a substantially lower true positive rate, as population-specific noise swamps any signal between the lakes.</p><p id="p-43">As has been suggested elsewhere, introgression aids identification of locally adaptive loci from scans for differentiation. This is generally because shared adaptive alleles provide something akin to independent replicates, allowing signal to emerge above the noise created by genetic drift. However, the required sample sizes and conditions on levels of admixture may be restrictive in practice. More generally, the difficulty of correctly identifying causal loci in even our most ideal situation calls for caution (and proof-of-concept simulations) in designing and carrying out scans for alleles underlying locally adaptive, polygenic traits.</p></div><div id="sec-12" class="subsection"><h3>Alleles for freshwater adaptation are mostly present in the initial generation</h3><p id="p-44">We have thus far found that the speed of adaptation depends strongly on the degree to which alleles can be shared between populations. However, the <em>origin</em> of the alleles underlying the phenotype is still unknown. In our simulations migration from marine individuals into the new lakes continues throughout introduction and adaptation: but is consistent influx of standing genetic variation a necessity for rapid adaptation of the population? To answer this, we traced the genealogy of <em>all</em> adaptive alleles in <em>all</em> individuals from the introduced lakes, after local adaptation. Surprisingly, for all cases with the exception of <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-74.gif"/></noscript></span></span>, we found that the majority of adaptive allele origins traced back to the original generation of inhabitants in the lake (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>). In our simulations, any reasonable subset of the ancestral population has the potential for rapid adaptation without the need post-colonization hybridization events.</p><p id="p-45">If effectively capturing standing genetic variants is the key to rapid adaptation, as we have presented thus far, why is there a large difference in speed of adaptation between dispersal rates of <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-75.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-76.gif"/></noscript></span></span> seen in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>? This is surprising because we see a similar quantity of allele sharing at <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-77.gif"/></noscript></span></span>, but much more rapid adaptation (<span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-78.gif"/></noscript></span></span> <em>vs.</em> 2000 generations).</p><p id="p-46">A possible explanation is that at lower rates of gene flow, freshwater alleles present as standing variation in the marine habitat are more tightly linked to marine alleles. This implies that adaptation in the new lakes must wait for recombination to separate freshwater and marine adapted alleles. A freshwater and a marine allele (of roughly equal and opposite effects) in close proximity could form an essentially neutral locus, which would persist in the marine population much longer than a freshwater allele not “masked” in this way. This suggests that a reasonable portion of freshwater alleles in the marine population would be linked to marine alleles. This genetic variation would be more difficult to access, however, so if there is sufficient additive genetic variation available to a new population in unlinked alleles, this will be used first. However, if the genetic variation initially available is not sufficient for a new population to adapt, then adaptation might need to wait for recombination to separate these haplotypes. Similar effects have been noted before in simulation (<em>e.g.</em>,<a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Yeaman and Whitlock 2011</a>).</p><p id="p-47">In other words, perhaps higher dispersal from freshwater to the ocean maintains relatively intact freshwater haplotypes that can be more easily rebuilt in the marine environments. Recall that trait-affecting mutations only occur in relatively small regions of <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-79.gif"/></noscript></span></span> loci in which recombination occurs only once in every one thousand meioses. This implies that even if there is a sufficient amount of variation in the initial population of a lake to shift the trait from the marine optimum (<span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-80.gif"/></noscript></span></span>) to the freshwater optimum (<span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-81.gif"/></noscript></span></span>), rebuilding the most beneficial haplotype may prove to be a non-trivial task for selection. For example, suppose there are 10 variants segregating at low frequency with effect size <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-82.gif"/></noscript></span></span> each, but each is paired with a compensatory allele with effect size <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-83.gif"/></noscript></span></span>. Each local haplotype is therefore neutral. This might also explain why at <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-84.gif"/></noscript></span></span>, marine individuals still hold on average several alleles that shift the trait in the freshwater direction (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6A</a>).</p><p id="p-48">To quantify the genetic variation available <em>without</em> recombination within the ten genomic regions, we first found, within each population, the haplotype with the largest net negative effect at each of the ten genomic regions. Summing these ten numbers, we get the maximum amount that selection could move the population in the freshwater direction without recombining within these regions. The mean of this value across the 25 lake populations is shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> – typical populations at <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-85.gif"/></noscript></span></span> migrants per lake per generation could shift to a phenotype of -12.02 (and so have sufficient variation to adapt without recombination), but the “best” haplotypes in the populations at <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-86.gif"/></noscript></span></span> have effect sizes nearly twice as big at a mean total of -21.02. The amount of variation available at <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-87.gif"/></noscript></span></span> is lower (only -7.02), presumably because of migration load, while at <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-88.gif"/></noscript></span></span> almost no alleles with negative effect are present. Note that some of these haplotypes will likely be lost to drift – indeed, if they did all fix, then populations at <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-89.gif"/></noscript></span></span> would adapt much more quickly. However, this calculation supports our explanation above: it appears that adaptation at <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-90.gif"/></noscript></span></span> is slower because it must wait for intralocus recombination to free up genetic variation present but masked in the marine population.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495764/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495764/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Haplotypic variation present in the new lakes at time of colonization, across rates of gene flow, “Best” quantified as the most negative trait value achievable with intact haplotypes, averaged across populations (see text for details)</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Simulation results align with theoretical expectations</h3><p id="p-49">How do our results compare to what is expected from population genetics theory? Our simulations included many loci under selection and in linkage to many other selected alleles, which makes precise calculation impossible. Nonetheless, rough calculations based on simple population genetics theory – with the benefit of hindsight – turn out to describe qualitatively most of the aspects of adaptation we observed above. Because we model stabilizing selection on an additive trait controlled by a moderately large number of loci within each population, more precise expectations might be obtained through quantitative genetics (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Svardal <em>et al.</em> 2014</a>) or even Fisher’s geometric model (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barton 2001</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chevin <em>et al.</em> 2014</a>), but doing so is beyond the scope of this paper. The arguments we use here are standard in population genetics (see <em>e.g.</em>,<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Gillespie 1998</a>).</p><p id="p-50">One of the most useful things that theory tells us is about the fate of a new allele in a lake, that has appeared by either migration or mutation. If the allele has fitness advantage <em>s</em> – <em>i.e.</em>, when it is rare but present in <em>n</em> copies, the expected number of copies in the next generation is <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-91.gif"/></noscript></span></span> – then the probability that it escapes demographic stochasticity to become common in the population is approximately <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-92.gif"/></noscript></span></span> (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lambert 2006</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Haldane 1927</a>), assuming Poisson reproduction, as we roughly have here. Since we are studying a quantitative trait, the fitness effect of each allele depends on the population context: if the individuals in the population all have trait values <em>z</em> units above the optimum, and the allele has effect size <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-93.gif"/></noscript></span></span> in heterozygotes, then the fitness advantage of the allele is the ratio of fitnesses with and without the allele. As we calculate fitness here, this is <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-94.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-95.gif"/></noscript></span></span>. This tells us two things: (1) the rate of adaptation decreases as the population approaches the optimum, and (2) larger mutations (in the right direction) are more likely to fix.</p><div id="sec-14" class="subsection"><h4>New mutations:</h4><p id="p-51">The total rate of appearance of new mutations per lake is <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-96.gif"/></noscript></span></span> per generation, and these are divided evenly in six categories: additive, dominant, and recessive, in either direction. Therefore, a new additive or dominant effect mutation appears once every 75 generations, on average. The effect size of each new mutation is randomly drawn from an Exponential distribution with mean <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-97.gif"/></noscript></span></span>, and so averaging the probability of establishment over this distribution, we get that the probability that a dominant mutation manages to establish in a population differing from the optimum by <em>z</em> is roughly <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-98.gif"/></noscript></span></span>. Multiplying the rate of appearance of these mutations with the probability they establish predicts that the rate of establishment of dominant mutations is <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-99.gif"/></noscript></span></span>, <em>i.e.</em>, about one such mutation every <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-100.gif"/></noscript></span></span> generations. During the initial phase of adaptation, the populations begin at around distance <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-101.gif"/></noscript></span></span> from the optimum, and <em>z</em> decreases as adaptation progresses. Mutations that successfully established are more likely to be strong: the distribution of the effect sizes of these successfully established mutations has density proportional to <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-102.gif"/></noscript></span></span>, which is a Gamma distribution with mean 1 and shape parameter 2. There are also additive alleles: these have half the effect in heterozygotes, and so roughly half the probability of establishment. Combining these facts, we expect adaptive alleles to appear through mutation within lakes at first on a time scale of 3,000 generations, with the time between local fixation of new alleles increasing as adaptation progresses, and each to move the trait by a distance of order 1. This agrees roughly with what we see in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figures 3</a> and S3.</p></div><div id="sec-15" class="subsection"><h4>Standing variation:</h4><p id="p-52">How are freshwater alleles maintained in the ocean, where they are deleterious? Assuming that fish in the marine environment are close to their phenotypic optimum, an allele that when heterozygous moves the trait <em>u</em> units in the freshwater direction, has fitness roughly <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-103.gif"/></noscript></span></span>, <em>i.e.</em>, a fitness differential of <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-104.gif"/></noscript></span></span>. The product of population size and fitness differential in the marine environment for a mutation with <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-105.gif"/></noscript></span></span> is therefore <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-106.gif"/></noscript></span></span>, implying that these alleles are strongly selected against but might drift to moderate frequency if recessive. The average frequency of such an allele in the marine environment at migration-selection equilibrium is equal to the proportion of individuals in the ocean replaced by migrants per generation divided by the selective disadvantage, <em>i.e.</em>, around <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-107.gif"/></noscript></span></span>. Each new lake is likely to contain a few copies of alleles at frequency above 1/400 (since each new lake is initialized with 400 randomly selected genomes). For a typical allele with effect size <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-108.gif"/></noscript></span></span>, the equilibrium migration-selection frequency is greater than this threshold if <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-109.gif"/></noscript></span></span>. Since <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-110.gif"/></noscript></span></span>, if <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-111.gif"/></noscript></span></span> this suggests that there is a good chance that any particular lake-adapted allele that is present in all pre-existing lakes will appear at least once in the fish that colonize a new lake. However, an allele with effect size <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-112.gif"/></noscript></span></span> only has probability of around 1/20 of establishing locally, and so must be present in about 20 copies to ensure establishment. Putting these calculations together, we expect migration-selection balance to maintain sufficient genetic variation for new lakes to adapt if <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-113.gif"/></noscript></span></span>, which corresponds to <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-114.gif"/></noscript></span></span>. This is in good agreement with our observations. However, this calculation treats each allele independently; in practice we found that standing freshwater variation in the ocean might be masked by linkage to compensatory marine variants.</p></div><div id="sec-16" class="subsection"><h4>Migration:</h4><p id="p-53">The key quantity regulating the amount of standing variation in the ocean is the <em>downstream</em> migration rate, from lakes to the ocean. How important is the upstream migration rate? If sufficient genetic variation is not present in a new lake initially, it must appear either by new mutation or by migration. Since a proportion <em>m</em> of each lake is composed of migrants each generation, it takes <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-115.gif"/></noscript></span></span> generations until the genetic variation introduced by migrants equals the amount initially present at colonization. This implies a dichotomy: either (a) migration is high, and adaptation is possible using variants present at colonization or arriving shortly thereafter, or (b) migration is low, so adaptation takes many multiples of <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-116.gif"/></noscript></span></span> generations. Since in our model lower migration also reduces the amount of variation available in the ocean, we expect very little contribution of subsequent migration across any value of <em>m</em>, as seen in <a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7</a>.</p><p id="p-54">We expect at least two other aspects of colonization and migration to be relevant. First, the size of the initial population of marine individuals colonizing each new lake affects how much of the marine genetic diversity is present. In our simulations, new lakes began with a full set of individuals, as may happen in uplifted tidal habitats. If instead each new lake began with only a few individuals (inducing an initial bottleneck), subsequent migration of alleles would likely become more relevant. If in addition, there was no subsequent “upstream” migration, adaptation might need to rely on new mutations, as at our lowest migration rates. However, exactly where these boundaries lie will require additional theoretical or simulation work.</p></div><div id="sec-17" class="subsection"><h4>Population size:</h4><p id="p-55">In our simulations, the marine and freshwater populations are of equal size. More generally, we expect the downstream migration rate that is required to maintain standing variation there to scale with the size of the marine population, since the same number of migrants make up a smaller proportion of a larger population. So, if the marine population was well-mixed and tenfold larger than all freshwater populations combined, the number of migrants per lake per generation would need to be ten times larger than in our simulations to maintain sufficient standing genetic variation. If the marine population was a thousandfold larger, adaptation from standing genetic variation that is deleterious in the ocean might be impossible. The fact that repeated adaptation from standing genetic variation has been shown to occur in real world threespine stickleback populations argues that the population sizes of marine and combined freshwater stickleback are not that different. While this might at first seem implausible given the ocean is large and lakes are small, there are hundreds of thousands to millions of lakes inhabited by threespine stickleback in coastal regions of the Northern Hemisphere, and the limiting habitat in the ocean constraining marine stickleback population sizes is likely to be coastal nesting areas. Together, these facts make the equivalency of marine and freshwater stickleback population sizes much more plausible.</p></div></div></div><div class="section conclusions" id="sec-18"><h2 class="">Conclusion</h2><p id="p-56">In this paper, we have analyzed a relatively realistic simulation study of a coastal meta-population to understand how repeated local adaptation operates on complex standing variation across a wide range of gene flow rates. We have shown that rapid and parallel adaptation similar to that documented in natural stickleback populations occurs over a realistic range of parameter values. These simulations therefore provide an in-depth look at (and quantitative proof-of-concept of) the “transporter” hypothesis suggested by <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Schluter and Conte (2009)</a>. Selection is able to rebuild the freshwater haplotype at a rapid pace (in tens of generations) from variation present at migration-selection balance in marine populations A finding of practical consequence is that the efficacy of <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/505/embed/mml-math-117.gif"/></noscript></span></span>-based genomic scans varies significantly with migration rate. Perhaps surprisingly, we found that the majority of adaptive variant can be traced back to the original generation of new habitats, with little role for post-colonization migrant alleles. In other words, a randomly chosen subset of marine stickleback carry the capacity to rapidly adapt in freshwater habitats without <em>any</em> continued connection to the rest of of the species. Thus, the large marine population was able to harbor and distribute alleles that were deleterious in the ocean habitat, but adaptive in freshwater habitats, to surrounding freshwater populations with only a few migrants per population, per generation. A few good stickleback suffice.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-57">We'd like to thank Thom Nelson and the entire Cresko lab, as well as Jeff Ross-Ibarra and several anonymous reviewers for useful input. Thank you toBen Haller for suggesting SLiM model improvements and interpretation. Thiswork benefited from access to the University of Oregon high performancecomputer, Talapas. </p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-58">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.10876052">https://doi.org/10.25387/g3.10876052</a>.</p></li><li class="fn" id="fn-2"><p id="p-59"><em>Communicating editor: J. Ross-Ibarra</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-07-22"><span class="received-label">Received </span>July 22, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-11-22"><span class="accepted-label">Accepted </span>November 22, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Galloway <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.1" data-doi="10.1016/j.tree.2007.09.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>,  <span class="cit-name-given-names">R. D. H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Schluter</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Adaptation from standing genetic variation.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">38</span>–<span class="cit-lpage">44</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tree.2007.09.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DBarrett%26rft.auinit1%253DR.%2BD.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D38%26rft.epage%253D44%26rft.atitle%253DAdaptation%2Bfrom%2Bstanding%2Bgenetic%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2007.09.008%26rft_id%253Dinfo%253Apmid%252F18006185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2007.09.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18006185&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252767300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.2" data-doi="10.1038/hdy.1986.135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>,  <span class="cit-name-given-names">N.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. O.</span> <span class="cit-name-surname">Bengtsson</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">The barrier to genetic exchange between hybridising populations.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">376</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.1986.135</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D57%26rft.spage%253D357%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1986.135%26rft_id%253Dinfo%253Apmid%252F3804765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1986.135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3804765&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986F469700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.3" data-doi="10.1046/j.1365-294x.2001.01216.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>,  <span class="cit-name-given-names">N. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The role of hybridization in evolution.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">551</span>–<span class="cit-lpage">568</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1365-294x.2001.01216.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DBarton%26rft.auinit1%253DN.%2BH.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253D551%26rft.epage%253D568%26rft.atitle%253DThe%2Brole%2Bof%2Bhybridization%2Bin%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-294x.2001.01216.x%26rft_id%253Dinfo%253Apmid%252F11298968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-294x.2001.01216.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11298968&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168367400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.4" data-doi="10.1534/genetics.117.300610"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Catchen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Lescak</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. A.</span> <span class="cit-name-surname">von Hippel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. A.</span> <span class="cit-name-surname">Cresko</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Repeated selection of alternatively adapted haplotypes creates sweeping genomic remodeling in stickleback.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">209</span>: <span class="cit-fpage">921</span>–<span class="cit-lpage">939</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300610%26rft_id%253Dinfo%253Apmid%252F29794240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA5LzMvOTIxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi81MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.505.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Foster</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-source">The Evolutionary Biology of the Three Spine Sticklebacks</span>. pp. <span class="cit-fpage">1</span>–<span class="cit-fpage">27</span>. Edited by Susan Adlai Foster. Oxford: Oxford University Press, 1994</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.6" data-doi="10.1111/j.1558-5646.1997.tb03983.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">A big book and a small book on selection.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">1337</span>–<span class="cit-lpage">1338</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.1997.tb03983.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D51%26rft.spage%253D1337%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.1997.tb03983.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.1997.tb03983.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.7" data-doi="10.1146/annurev.ecolsys.34.011802.132359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Charlesworth</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. H.</span> <span class="cit-name-surname">Barton</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The effects of genetic and geographic structure on neutral variation.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">99</span>–<span class="cit-lpage">125</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.ecolsys.34.011802.132359</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%2Band%2BSystematics%26rft.stitle%253DAnnu.%2BRev.%2BEcol.%2BSyst.%26rft.volume%253D34%26rft.spage%253D99%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.34.011802.132359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.34.011802.132359&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000220102000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.8" data-doi="10.1111/evo.12346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chevin</span>,  <span class="cit-name-given-names">L.-M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Decorzent</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Lenormand</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Niche dimensionality and the genetics of ecological speciation.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">1244</span>–<span class="cit-lpage">1256</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/evo.12346</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D68%26rft.spage%253D1244%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.12346%26rft_id%253Dinfo%253Apmid%252F24410181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.12346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24410181&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.9" data-doi="10.1126/science.1107239"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Colosimo</span>,  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. E.</span> <span class="cit-name-surname">Hosemann</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Balabhadra</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Villarreal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Dickson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Widespread parallel evolution in sticklebacks by repeated fixation of ectodysplasin alleles.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">307</span>: <span class="cit-fpage">1928</span>–<span class="cit-lpage">1933</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1107239</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DColosimo%26rft.auinit1%253DP.%2BF.%26rft.volume%253D307%26rft.issue%253D5717%26rft.spage%253D1928%26rft.epage%253D1933%26rft.atitle%253DWidespread%2BParallel%2BEvolution%2Bin%2BSticklebacks%2Bby%2BRepeated%2BFixation%2Bof%2BEctodysplasin%2BAlleles%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1107239%26rft_id%253Dinfo%253Apmid%252F15790847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDcvNTcxNy8xOTI4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi81MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.10" data-doi="10.1371/journal.pbio.0020109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Colosimo</span>,  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. L.</span> <span class="cit-name-surname">Peichel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Nereng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. K.</span> <span class="cit-name-surname">Blackman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Shapiro</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The genetic architecture of parallel armor plate reduction in threespine sticklebacks.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-elocation-id">E109</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.0020109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DColosimo%26rft.auinit1%253DP.%2BF.%26rft.volume%253D2%26rft.issue%253D5%26rft.spage%253DE109%26rft.epage%253DE109%26rft.atitle%253DThe%2Bgenetic%2Barchitecture%2Bof%2Bparallel%2Barmor%2Bplate%2Breduction%2Bin%2Bthreespine%2Bsticklebacks.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0020109%26rft_id%253Dinfo%253Apmid%252F15069472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0020109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15069472&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.11" data-doi="10.1073/pnas.0308479101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>,  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Amores</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Wilson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Murphy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Currey</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Parallel genetic basis for repeated evolution of armor loss in Alaskan threespine stickleback populations.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">6050</span>–<span class="cit-lpage">6055</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0308479101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0308479101%26rft_id%253Dinfo%253Apmid%252F15069186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzE2LzYwNTAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzUwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.12" data-doi="10.1007/s10709-006-0036-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>,  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. L.</span> <span class="cit-name-surname">McGuigan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. C.</span> <span class="cit-name-surname">Phillips</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Postlethwait</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Studies of threespine stickleback developmental evolution: progress and promise.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">126</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10709-006-0036-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.stitle%253DGenetica%26rft.aulast%253DCresko%26rft.auinit1%253DW.%2BA.%26rft.volume%253D129%26rft.issue%253D1%26rft.spage%253D105%26rft.epage%253D126%26rft.atitle%253DStudies%2Bof%2Bthreespine%2Bstickleback%2Bdevelopmental%2Bevolution%253A%2Bprogress%2Band%2Bpromise.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-006-0036-z%26rft_id%253Dinfo%253Apmid%252F16897450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-006-0036-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16897450&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242816200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.13" data-doi="10.1111/evo.12097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DeFaveri</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. R.</span> <span class="cit-name-surname">Jonsson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Merilä</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Heterogeneous genomic differentiation in marine threespine sticklebacks: Adaptation along an environmental gradient.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">2530</span>–<span class="cit-lpage">2546</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/evo.12097</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D67%26rft.spage%253D2530%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.12097%26rft_id%253Dinfo%253Apmid%252F24033165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.12097&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24033165&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.14" data-doi="10.1111/j.1558-5646.2011.01247.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DeFaveri</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Shikano</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Shimada</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Goto</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Merilä</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Global analysis of genes involved in freshwater adaptation in threespine sticklebacks (gasterosteus aculeatus).</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">1800</span>–<span class="cit-lpage">1807</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.2011.01247.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DDefaveri%26rft.auinit1%253DJ.%26rft.volume%253D65%26rft.issue%253D6%26rft.spage%253D1800%26rft.epage%253D1807%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Bgenes%2Binvolved%2Bin%2Bfreshwater%2Badaptation%2Bin%2Bthreespine%2Bsticklebacks%2B%2528Gasterosteus%2Baculeatus%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2011.01247.x%26rft_id%253Dinfo%253Apmid%252F21644964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2011.01247.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21644964&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291270300021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.15" data-doi="10.2307/2259234"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endler</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Geographic variation, speciation and clines.</span> <abbr class="cit-jnl-abbrev">Monogr. Popul. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMonographs%2Bin%2Bpopulation%2Bbiology%26rft.stitle%253DMonogr%2BPopul%2BBiol%26rft.aulast%253DEndler%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D10%26rft.spage%253D1%26rft.epage%253D246%26rft.atitle%253DGeographic%2Bvariation%252C%2Bspeciation%252C%2Band%2Bclines.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2259234%26rft_id%253Dinfo%253Apmid%252F409931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2259234&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=409931&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.16" data-doi="10.1371/journal.pgen.1004966"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feulner</span>,  <span class="cit-name-given-names">P. G. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. J. J.</span> <span class="cit-name-surname">Chain</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Panchal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Huang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Eizaguirre</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomics of Divergence along a Continuum of Parapatric Population Differentiation.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e1004966</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004966</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1004966%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004966%26rft_id%253Dinfo%253Apmid%252F25679225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004966&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25679225&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.17" data-doi="10.1111/mec.12564"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flaxman</span>,  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Surfing downhill: when should population range expansion be characterized by reductions in fitness?</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">5963</span>–<span class="cit-lpage">5965</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12564</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D5963%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12564&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000327823700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.505.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gillespie</span>,  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source">Population Genetics</span>, <span class="cit-publ-name">The Johns Hopkins University Press</span>, <span class="cit-publ-loc">Baltimore, MD</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.19" data-doi="10.1017/S0305004100015644"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haldane</span>,  <span class="cit-name-given-names">J. B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1927</span> <span class="cit-article-title">A mathematical theory of natural and artificial selection, part V: Selection and mutation.</span> <abbr class="cit-jnl-abbrev">Math. Proc. Camb. Philos. Soc.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">838</span>–<span class="cit-lpage">844</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0305004100015644</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMath.%2BProc.%2BCamb.%2BPhilos.%2BSoc.%26rft.volume%253D23%26rft.spage%253D838%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0305004100015644%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0305004100015644&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.20" data-doi="10.1111/1755-0998.12968"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haller</span>,  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Galloway</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Kelleher</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. W.</span> <span class="cit-name-surname">Messer</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. L.</span> <span class="cit-name-surname">Ralph</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Tree-sequence recording in SLiM opens new horizons for forward-time simulation of whole genomes.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour</abbr>. 19: 552–566.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12968</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12968&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.21" data-doi="10.1093/molbev/msw211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haller</span>,  <span class="cit-name-given-names">B. C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. W.</span> <span class="cit-name-surname">Messer</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">SLiM 2: Flexible, interactive forward genetic simulations.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">230</span>–<span class="cit-lpage">240</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msw211</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw211%26rft_id%253Dinfo%253Apmid%252F27702775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27702775&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.2.505.22" data-doi="10.1093/molbev/msy228"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haller</span>,  <span class="cit-name-given-names">B. C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. W.</span> <span class="cit-name-surname">Messer</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">SLiM 3: Forward genetic simulations beyond the Wright-Fisher model.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">632</span>–<span class="cit-lpage">637</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msy228</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsy228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msy228&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.23" data-doi="10.1111/jeb.12761"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hanson</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. D.</span> <span class="cit-name-surname">Barrett</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Hendry</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Testing for parallel allochronic isolation in lake-stream stickleback.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">57</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jeb.12761</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D29%26rft.spage%253D47%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjeb.12761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jeb.12761&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.24" data-doi="10.1038/hdy.2017.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hanson</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Hendry</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. D. H.</span> <span class="cit-name-surname">Barrett</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Heritable gene expression differences between lake and stream stickleback include both parallel and antiparallel components.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">339</span>–<span class="cit-lpage">348</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2017.50</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D119%26rft.spage%253D339%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2017.50%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2017.50&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.25" data-doi="10.1111/mec.13582"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harrison</span>,  <span class="cit-name-given-names">R. G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. L.</span> <span class="cit-name-surname">Larson</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Heterogeneous genome divergence, differential introgression, and the origin and structure of hybrid zones.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2454</span>–<span class="cit-lpage">2466</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13582</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D2454%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13582&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.26" data-doi="10.1534/genetics.104.036947"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hermisson</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. S.</span> <span class="cit-name-surname">Pennings</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Soft sweeps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">2335</span>–<span class="cit-lpage">2352</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.036947</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.036947%26rft_id%253Dinfo%253Apmid%252F15716498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS80LzIzMzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzUwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.27" data-doi="10.1186/1471-2164-15-735"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hirase</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Ozaki</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Iwasaki</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Parallel selection on gene copy number variations through evolution of three-spined stickleback genomes.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">735</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-735</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D735%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-735%26rft_id%253Dinfo%253Apmid%252F25168270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-735&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25168270&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.28" data-doi="10.1086/688018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoban</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Kelley</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. E.</span> <span class="cit-name-surname">Lotterhos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. F.</span> <span class="cit-name-surname">Antolin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Bradburd</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Finding the genomic basis of local adaptation: Pitfalls, practical solutions, and future directions.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">379</span>–<span class="cit-lpage">397</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/688018</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D188%26rft.spage%253D379%26rft_id%253Dinfo%253Adoi%252F10.1086%252F688018%26rft_id%253Dinfo%253Apmid%252F27622873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/688018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27622873&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.29" data-doi="10.1371/journal.pgen.1000862"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>,  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Bassham</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. D.</span> <span class="cit-name-surname">Etter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Stiffler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Johnson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Population genomics of parallel adaptation in threespine stickleback using sequenced RAD tags.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e1000862</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000862</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHohenlohe%26rft.auinit1%253DP.%2BA.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253De1000862%26rft.epage%253De1000862%26rft.atitle%253DPopulation%2Bgenomics%2Bof%2Bparallel%2Badaptation%2Bin%2Bthreespine%2Bstickleback%2Busing%2Bsequenced%2BRAD%2Btags.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000862%26rft_id%253Dinfo%253Apmid%252F20195501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000862&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20195501&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.30" data-doi="10.1111/j.1558-5646.2007.00310.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Bell</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Travis</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Evolution toward a new adaptive optimum: Phenotypic evolution in a fossil stickleback lineage.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">700</span>–<span class="cit-lpage">710</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.2007.00310.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DHunt%26rft.auinit1%253DG.%26rft.volume%253D62%26rft.issue%253D3%26rft.spage%253D700%26rft.epage%253D710%26rft.atitle%253DEvolution%2Btoward%2Ba%2Bnew%2Badaptive%2Boptimum%253A%2Bphenotypic%2Bevolution%2Bin%2Ba%2Bfossil%2Bstickleback%2Blineage.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2007.00310.x%26rft_id%253Dinfo%253Apmid%252F18081713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2007.00310.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18081713&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253758600018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.31" data-doi="10.1038/nature10944"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. G.</span> <span class="cit-name-surname">Grabherr</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. F.</span> <span class="cit-name-surname">Chan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Russell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Mauceli</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genomic basis of adaptive evolution in threespine sticklebacks.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">484</span>: <span class="cit-fpage">55</span>–<span class="cit-lpage">61</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature10944</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJones%26rft.auinit1%253DF.%2BC.%26rft.volume%253D484%26rft.issue%253D7392%26rft.spage%253D55%26rft.epage%253D61%26rft.atitle%253DThe%2Bgenomic%2Bbasis%2Bof%2Badaptive%2Bevolution%2Bin%2Bthreespine%2Bsticklebacks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10944%26rft_id%253Dinfo%253Apmid%252F22481358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10944&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22481358&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302343400033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.32" data-doi="10.1371/journal.pcbi.1004842"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelleher</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Etheridge</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">McVean</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Efficient coalescent simulation and genealogical analysis for large sample sizes.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-elocation-id">e1004842</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1004842</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D12%26rft.spage%253De1004842%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1004842%26rft_id%253Dinfo%253Apmid%252F27145223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1004842&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27145223&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.33" data-doi="10.1371/journal.pcbi.1006581"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelleher</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. R.</span> <span class="cit-name-surname">Thornton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ashander</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. L.</span> <span class="cit-name-surname">Ralph</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Efficient pedigree recording for fast population genetics simulation.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-elocation-id">e1006581</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1006581</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D14%26rft.spage%253De1006581%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1006581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1006581&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.34" data-doi="10.1073/pnas.1813597116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>,  <span class="cit-name-given-names">Y.-T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. K. L.</span> <span class="cit-name-surname">Yeung</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. E.</span> <span class="cit-name-surname">Omland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.-L.</span> <span class="cit-name-surname">Pang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Hao</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Standing genetic variation as the predominant source for adaptation of a songbird.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">2152</span>–<span class="cit-lpage">2157</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1813597116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1813597116%26rft_id%253Dinfo%253Apmid%252F30659151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTE2LzYvMjE1MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvNTA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.35" data-doi="10.1016/j.tpb.2006.01.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lambert</span>,  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Probability of fixation under weak selection: a branching process unifying approach.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">441</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tpb.2006.01.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DLambert%26rft.auinit1%253DA.%26rft.volume%253D69%26rft.issue%253D4%26rft.spage%253D419%26rft.epage%253D441%26rft.atitle%253DProbability%2Bof%2Bfixation%2Bunder%2Bweak%2Bselection%253A%2Ba%2Bbranching%2Bprocess%2Bunifying%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tpb.2006.01.002%26rft_id%253Dinfo%253Apmid%252F16504230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tpb.2006.01.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16504230&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237757100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.36" data-doi="10.1073/pnas.1512020112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lescak</span>,  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Bassham</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Catchen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Gelmond</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Sherbick</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Evolution of stickleback in 50 years on earthquake-uplifted islands.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">E7204</span>–<span class="cit-lpage">E7212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1512020112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1512020112%26rft_id%253Dinfo%253Apmid%252F26668399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEyLzUyL0U3MjA0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi81MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.37" data-doi="10.1111/mec.13100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lotterhos</span>,  <span class="cit-name-given-names">K. E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Whitlock</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The relative power of genome scans to detect local adaptation depends on sampling design and statistical method.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1031</span>–<span class="cit-lpage">1046</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13100</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D24%26rft.spage%253D1031%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13100%26rft_id%253Dinfo%253Apmid%252F25648189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13100&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25648189&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.38" data-doi="10.1371/journal.pgen.1005887"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marques</span>,  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Lucek</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. I.</span> <span class="cit-name-surname">Meier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Mwaiko</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. E.</span> <span class="cit-name-surname">Wagner</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomics of rapid incipient speciation in sympatric threespine stickleback.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-elocation-id">e1005887</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005887</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1005887%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005887%26rft_id%253Dinfo%253Apmid%252F26925837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005887&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26925837&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.39" data-doi="10.1017/S0016672300014634"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maynard Smith</span>,  <span class="cit-name-given-names">J. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Haigh</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The hitch-hiking effect of a favourable gene.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">35</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300014634</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DSmith%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D35%26rft.atitle%253DThe%2Bhitch-hiking%2Beffect%2Bof%2Ba%2Bfavourable%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300014634%26rft_id%253Dinfo%253Apmid%252F4407212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300014634&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4407212&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1974T216900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.40" data-doi="10.1002/evl3.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>,  <span class="cit-name-given-names">T. C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. A.</span> <span class="cit-name-surname">Cresko</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Ancient genomic variation underlies repeated ecological adaptation in young stickleback populations.</span> <abbr class="cit-jnl-abbrev">Evolution Letters</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/evl3.37</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2BLetters%26rft.volume%253D2%26rft.spage%253D9%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fevl3.37%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/evl3.37&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.41" data-doi="10.1111/mec.15067"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>,  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Jones</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Velotta</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. S.</span> <span class="cit-name-surname">Dhawanjewar</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Schweizer</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Unveiling connections between genotype, phenotype, and fitness in natural populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1866</span>–<span class="cit-lpage">1876</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.15067</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D28%26rft.spage%253D1866%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.15067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.15067&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.42" data-doi="10.1111/j.1365-294X.2008.03946.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nosil</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Funk</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Ortiz-Barrientos</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Divergent selection and heterogeneous genomic divergence.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">402</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2008.03946.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DNosil%26rft.auinit1%253DP.%26rft.volume%253D18%26rft.issue%253D3%26rft.spage%253D375%26rft.epage%253D402%26rft.atitle%253DDivergent%2Bselection%2Band%2Bheterogeneous%2Bgenomic%2Bdivergence.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2008.03946.x%26rft_id%253Dinfo%253Apmid%252F19143936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2008.03946.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19143936&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262517400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.505.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Brien</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hunt</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wolf</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-source">Quantitative genetic variation and heritability</span>, <span class="cit-publ-name">Oxford Bibliographies in Evolutionary Biology</span>. <a href="https://doi.org/10.1093/OBO/9780199941728-0064">https://doi.org/10.1093/OBO/9780199941728-0064</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.44" data-doi="10.1038/nrg1523"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orr</span>,  <span class="cit-name-given-names">H. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The genetic theory of adaptation: a brief history.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">127</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1523</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DOrr%26rft.auinit1%253DH.%2BA.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253D119%26rft.epage%253D127%26rft.atitle%253DThe%2Bgenetic%2Btheory%2Bof%2Badaptation%253A%2Ba%2Bbrief%2Bhistory.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1523%26rft_id%253Dinfo%253Apmid%252F15716908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1523&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15716908&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226721300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.45" data-doi="10.1016/0040-5809(83)90020-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petry</span>,  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">The effect on neutral gene flow of selection at a linked locus.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">300</span>–<span class="cit-lpage">313</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0040-5809(83)90020-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DPetry%26rft.auinit1%253DD.%26rft.volume%253D23%26rft.issue%253D3%26rft.spage%253D300%26rft.epage%253D313%26rft.atitle%253DThe%2Beffect%2Bon%2Bneutral%2Bgene%2Bflow%2Bof%2Bselection%2Bat%2Ba%2Blinked%2Blocus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252883%252990020-5%26rft_id%253Dinfo%253Apmid%252F6623407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(83)90020-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6623407&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.46" data-doi="10.1038/scientificamerican1010-40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>,  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">How we are evolving.</span> <abbr class="cit-jnl-abbrev">Sci. Am.</abbr> <span class="cit-vol">303</span>: <span class="cit-fpage">40</span>–<span class="cit-lpage">47</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/scientificamerican1010-40</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientific%2BAmerican%26rft.stitle%253DScientific%2BAmerican%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D303%26rft.issue%253D6%26rft.spage%253D40%26rft.epage%253D40%26rft.atitle%253DTalk%2Bto%2Bthe%2Bmachine.%2BSpeech-recognition%2Bprograms%2Bare%2Bno%2Blonger%2Bclumsy%2Bexercises%2Bin%2Bfutility.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fscientificamerican1010-40%26rft_id%253Dinfo%253Apmid%252F21141356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/scientificamerican1010-40&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21141356&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284314300024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.47" data-doi="10.1086/682948"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ralph</span>,  <span class="cit-name-given-names">P. L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Coop</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The role of standing variation in geographic convergent adaptation.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">S5</span>–<span class="cit-lpage">S23</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/682948</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D186%26rft.spage%253DS5%26rft_id%253Dinfo%253Adoi%252F10.1086%252F682948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/682948&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.48" data-doi="10.1111/mec.12720"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roesti</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Gavrilets</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Hendry</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Salzburger</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Berner</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The genomic signature of parallel adaptation from shared genetic variation.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">3944</span>–<span class="cit-lpage">3956</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12720</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D3944%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12720%26rft_id%253Dinfo%253Apmid%252F24635356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12720&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24635356&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.49" data-doi="10.1111/j.1365-294X.2012.05509.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roesti</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Hendry</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Salzburger</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Berner</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome divergence during evolutionary diversification as revealed in replicate lake–stream stickleback population pairs.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2852</span>–<span class="cit-lpage">2862</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2012.05509.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DRoesti%26rft.auinit1%253DM.%26rft.volume%253D21%26rft.issue%253D12%26rft.spage%253D2852%26rft.epage%253D2862%26rft.atitle%253DGenome%2Bdivergence%2Bduring%2Bevolutionary%2Bdiversification%2Bas%2Brevealed%2Bin%2Breplicate%2Blake-stream%2Bstickleback%2Bpopulation%2Bpairs.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05509.x%26rft_id%253Dinfo%253Apmid%252F22384978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05509.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22384978&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304903100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.50" data-doi="10.1111/mec.14226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Samuk</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. L.</span> <span class="cit-name-surname">Owens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. E.</span> <span class="cit-name-surname">Delmore</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Miller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Rennison</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Gene flow and selection interact to promote adaptive divergence in regions of low recombination.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">4378</span>–<span class="cit-lpage">4390</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.14226</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D26%26rft.spage%253D4378%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.14226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.14226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.51" data-doi="10.1073/pnas.0901264106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schluter</span>,  <span class="cit-name-given-names">D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. L.</span> <span class="cit-name-surname">Conte</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetics and ecological speciation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">9955</span>–<span class="cit-lpage">9962</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0901264106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0901264106%26rft_id%253Dinfo%253Apmid%252F19528639%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoyMToiMTA2L1N1cHBsZW1lbnRfMS85OTU1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi81MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.52" data-doi="10.1093/molbev/msx154"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>,  <span class="cit-name-given-names">D. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Kern</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Soft sweeps are the dominant mode of adaptation in the human genome.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">1863</span>–<span class="cit-lpage">1877</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msx154</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D34%26rft.spage%253D1863%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsx154%26rft_id%253Dinfo%253Apmid%252F28482049%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msx154&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28482049&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Gene flow and selection in a two-locus system.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">787</span>–<span class="cit-lpage">802</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSlatkin%26rft.auinit1%253DM.%26rft.volume%253D81%26rft.issue%253D4%26rft.spage%253D787%26rft.epage%253D802%26rft.atitle%253DGENE%2BFLOW%2BAND%2BSELECTION%2BIN%2BA%2BTWO-LOCUS%2BSYSTEM%26rft_id%253Dinfo%253Apmid%252F1213276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiODEvNC83ODciO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzUwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.54" data-doi="10.1002/ece3.3190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Southcott</span>,  <span class="cit-name-given-names">L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Kronforst</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A neutral view of the evolving genomic architecture of speciation.</span> <abbr class="cit-jnl-abbrev">Ecol. Evol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">6358</span>–<span class="cit-lpage">6366</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/ece3.3190</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fece3.3190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/ece3.3190&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.55" data-doi="10.1038/s41559-017-0158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stuart</span>,  <span class="cit-name-given-names">Y. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Veen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. N.</span> <span class="cit-name-surname">Weber</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Hanson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ravinet</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Contrasting effects of environment and genetics generate a continuum of parallel evolution.</span> <abbr class="cit-jnl-abbrev">Nat. Ecol. Evol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">0158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41559-017-0158</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BEcol.%2BEvol.%26rft.volume%253D1%26rft.spage%253D0158%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41559-017-0158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41559-017-0158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-web" id="cit-10.2.505.56"><div class="cit-metadata unstructured">Svardal, H., C. Rueffler, and J. Hermisson, 2014 A general condition for adaptive genetic polymorphism in temporally and spatially heterogeneous environments. arxiv:1411.3709.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.57" data-doi="10.1371/journal.pgen.1004696"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Terekhanova</span>,  <span class="cit-name-given-names">N. V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Logacheva</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Penin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. V.</span> <span class="cit-name-surname">Neretina</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Barmintseva</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Fast evolution from precast bricks: Genomics of young freshwater populations of threespine stickleback <em>Gasterosteus aculeatus</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-elocation-id">e1004696</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004696</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004696%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004696%26rft_id%253Dinfo%253Apmid%252F25299485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004696&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25299485&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.58" data-doi="10.2307/2266085"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>,  <span class="cit-name-given-names">J. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Evaluating the dynamics of coevolution among geographically structured populations.</span> <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">1619</span>–<span class="cit-lpage">1623</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1890/0012-9658(1997)078[1619:ETDOCA]2.0.CO;2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D78%26rft.spage%253D1619%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2266085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2266085&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997XU40900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.505.59" data-doi="10.1111/j.1558-5646.2011.01269.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yeaman</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Whitlock</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The genetic architecture of adaptation under migration-selection balance.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">1897</span>–<span class="cit-lpage">1911</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.2011.01269.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DYeaman%26rft.auinit1%253DS.%26rft.volume%253D65%26rft.issue%253D7%26rft.spage%253D1897%26rft.epage%253D1911%26rft.atitle%253DThe%2Bgenetic%2Barchitecture%2Bof%2Badaptation%2Bunder%2Bmigration-selection%2Bbalance.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2011.01269.x%26rft_id%253Dinfo%253Apmid%252F21729046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2011.01269.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21729046&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292447000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/2/505.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/2/495" title="Demonstration of a Simple Epitope Tag Multimerization Strategy for Enhancing the Sensitivity of Protein Detection Using Drosophila vAChT" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/2/515" title="Accelerated Evolution of Limb-Related Gene Hoxd11 in the Common Ancestor of Cetaceans and Ruminants (Cetruminantia)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/2"><span>Volume 10 Issue 2, February 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/2.cover-source.jpg?itok=LSawHOSr" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/505.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Ro4KkbX4vMcYDFXjkMViaV71Prhe0glS5TSQSi3SzYg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/2/505" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/2/505" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/2/505" class="active" data-icon-position="" data-hide-link-title="0">A Few Stickleback Suffice for the Transport of Alleles to New Lakes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495528" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-BG4bLEsZTccUMEyZ8IRlFc3quB5GrqsMZVNo3MBeXw0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/2/505.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/2/505" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-pn2PyvheKnDTvVF2FJ-qdXj_41y_WeLZjY4U9eYT3K0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/2/505" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/2/505.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495528" id="node495528--2" data-pisa="ggg;10/2/505" data-pisa-master="ggg;g3.119.400564" data-apath="/ggg/10/2/505.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Few Stickleback Suffice for the Transport of Alleles to New Lakes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jared Galloway</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-3496-8074" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>William A. Cresko</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-9459-6866" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter Ralph</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">505-514;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400564 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495528--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jared Galloway </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Ecology and Evolution and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jared%2BGalloway%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Galloway%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJared%2BGalloway%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">William A. Cresko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Ecology and Evolution and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BA.%2BCresko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cresko%20WA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BA.%2BCresko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3496-8074" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for William A. Cresko</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Peter Ralph </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Ecology and Evolution and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematics, University of Oregon</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BRalph%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ralph%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BRalph%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9459-6866" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter Ralph</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:plr@uoregon.edu" class="" data-icon-position="" data-hide-link-title="0">plr@uoregon.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/2/505" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/2/505&amp;title=A%20Few%20Stickleback%20Suffice%20for%20the%20Transport%20of%20Alleles%20to%20New%20Lakes" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/2/505&amp;title=A%20Few%20Stickleback%20Suffice%20for%20the%20Transport%20of%20Alleles%20to%20New%20Lakes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/2/505&amp;title=A%20Few%20Stickleback%20Suffice%20for%20the%20Transport%20of%20Alleles%20to%20New%20Lakes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/2/505&amp;text=A%20Few%20Stickleback%20Suffice%20for%20the%20Transport%20of%20Alleles%20to%20New%20Lakes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/2/505&amp;title=A%20Few%20Stickleback%20Suffice%20for%20the%20Transport%20of%20Alleles%20to%20New%20Lakes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/2/505&amp;t=A%20Few%20Stickleback%20Suffice%20for%20the%20Transport%20of%20Alleles%20to%20New%20Lakes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/2/505&amp;title=A%20Few%20Stickleback%20Suffice%20for%20the%20Transport%20of%20Alleles%20to%20New%20Lakes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/2/505&amp;title=A%20Few%20Stickleback%20Suffice%20for%20the%20Transport%20of%20Alleles%20to%20New%20Lakes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495528/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495528" id="node495528--3" data-pisa="ggg;10/2/505" data-pisa-master="ggg;g3.119.400564" data-apath="/ggg/10/2/505.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Few Stickleback Suffice for the Transport of Alleles to New Lakes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jared Galloway</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-3496-8074" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>William A. Cresko</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-9459-6866" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter Ralph</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">505-514;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400564 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495528--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jared Galloway </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Ecology and Evolution and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jared%2BGalloway%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Galloway%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJared%2BGalloway%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">William A. Cresko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Ecology and Evolution and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BA.%2BCresko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cresko%20WA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BA.%2BCresko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3496-8074" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for William A. Cresko</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Peter Ralph </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Ecology and Evolution and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematics, University of Oregon</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BRalph%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ralph%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BRalph%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9459-6866" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter Ralph</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:plr@uoregon.edu" class="" data-icon-position="" data-hide-link-title="0">plr@uoregon.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=505&amp;rft.epage=514&amp;rft.atitle=A%20Few%20Stickleback%20Suffice%20for%20the%20Transport%20of%20Alleles%20to%20New%20Lakes&amp;rft.volume=10&amp;rft.issue=2&amp;rft.date=2020-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Galloway%2C+Jared&amp;rft.au=Cresko%2C+William+A.&amp;rft.au=Ralph%2C+Peter"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495528/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495528/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495528/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495528/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495528/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495528/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495528/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495528/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495528/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495528/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495528/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495528/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/2/505" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results &amp; Discussion</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/2/505.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/2/505.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/2/505.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
