<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/2/677" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/2/677.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/2/677.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/2/677.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/10/2/677/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/2/677.atom" />
<meta name="HW.pisa" content="ggg;10/2/677" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Contingent Convergence: The Ability To Detect Convergent Genomic Evolution Is Dependent on Population Size and Migration" />
<meta name="DC.Identifier" content="10.1534/g3.119.400970" />
<meta name="DC.Date" content="2020-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Whiting and Fraser. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Outlier scans, in which the genome is scanned for signatures of selection, have become a prominent tool in studies of local adaptation, and more recently studies of genetic convergence in natural populations. However, such methods have the potential to be confounded by features of demographic history, such as population size and migration, which are considerably varied across natural populations. In this study, we use forward-simulations to investigate and illustrate how several measures of genetic differentiation commonly used in outlier scans (FST, DXY and Δπ) are influenced by demographic variation across multiple sampling generations. In a factorial design with 16 treatments, we manipulate the presence/absence of founding bottlenecks (N of founding individuals), prolonged bottlenecks (proportional size of diverging population) and migration rate between two populations with ancestral and diverged phenotypic optima. Our results illustrate known constraints of individual measures associated with reduced population size and a lack of migration; but notably we demonstrate how relationships between measures are similarly dependent on these features of demography. We find that false-positive signals of convergent evolution (the same simulated outliers detected in independent treatments) are attainable as a product of similar population size and migration treatments (particularly for DXY), and that outliers across different measures (for e.g. , FST and DXY) can occur with little influence of selection. Taken together, we show how underappreciated, yet quantifiable measures of demographic history can influence commonly employed methods for detecting selection." />
<meta name="DC.Contributor" content="James R. Whiting" />
<meta name="DC.Contributor" content="Bonnie A. Fraser" />
<meta name="article:published_time" content="2020-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Contingent Convergence: The Ability To Detect Convergent Genomic Evolution Is Dependent on Population Size and Migration" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Outlier scans, in which the genome is scanned for signatures of selection, have become a prominent tool in studies of local adaptation, and more recently studies of genetic convergence in natural populations. However, such methods have the potential to be confounded by features of demographic history, such as population size and migration, which are considerably varied across natural populations. In this study, we use forward-simulations to investigate and illustrate how several measures of genetic differentiation commonly used in outlier scans (F&lt;sub&gt;ST&lt;/sub&gt;, D&lt;sub&gt;XY&lt;/sub&gt; and Δπ) are influenced by demographic variation across multiple sampling generations. In a factorial design with 16 treatments, we manipulate the presence/absence of founding bottlenecks (N of founding individuals), prolonged bottlenecks (proportional size of diverging population) and migration rate between two populations with ancestral and diverged phenotypic optima. Our results illustrate known constraints of individual measures associated with reduced population size and a lack of migration; but notably we demonstrate how relationships between measures are similarly dependent on these features of demography. We find that false-positive signals of convergent evolution (the same simulated outliers detected in independent treatments) are attainable as a product of similar population size and migration treatments (particularly for D&lt;sub&gt;XY&lt;/sub&gt;), and that outliers across different measures (for &lt;i&gt;e.g.&lt;/i&gt;, F&lt;sub&gt;ST&lt;/sub&gt; and D&lt;sub&gt;XY&lt;/sub&gt;) can occur with little influence of selection. Taken together, we show how underappreciated, yet quantifiable measures of demographic history can influence commonly employed methods for detecting selection.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/02/01" />
<meta name="citation_mjid" content="ggg;10/2/677" />
<meta name="citation_id" content="10/2/677" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/2/677" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/2/677.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/2/677.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/2/677.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400970" />
<meta name="citation_pmid" content="31871215" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="17" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="677" />
<meta name="citation_lastpage" content="693" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="James R. Whiting" />
<meta name="citation_author_institution" content="Department of Biosciences, University of Exeter, Geoffrey Pope Building, Exeter, EX4 4QD" />
<meta name="citation_author_email" content="j.whiting2@exeter.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8936-4991" />
<meta name="citation_author" content="Bonnie A. Fraser" />
<meta name="citation_author_institution" content="Department of Biosciences, University of Exeter, Geoffrey Pope Building, Exeter, EX4 4QD" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5905-9056" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=C. W. Ahrens;citation_author=P. D. Rymer;citation_author=A. Stow;citation_author=J. Bragg;citation_author=S. Dillon;citation_title=The search for loci under selection: trends, biases and progress.;citation_pages=1342-1356;citation_volume=27;citation_year=2018;citation_pmid=29524276;citation_doi=10.1111/mec.14549" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=C. Bank;citation_author=G. B. Ewing;citation_author=A. Ferrer-Admettla;citation_author=M. Foll;citation_author=J. D. Jensen;citation_title=Thinking too positive? Revisiting current methods of population genetic selection inference.;citation_pages=540-546;citation_volume=30;citation_year=2014;citation_pmid=25438719;citation_doi=10.1016/j.tig.2014.09.010" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=S. M. Van Belleghem;citation_author=C. Vangestel;citation_author=K. De Wolf;citation_author=Z. De Corte;citation_author=M. Most;citation_title=Evolution at two time frames: Polymorphisms from an ancient singular divergence event fuel contemporary parallel evolution.;citation_pages=e1007796;citation_volume=14;citation_year=2018;citation_doi=10.1371/journal.pgen.1007796" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=R. Burri;citation_title=Linked selection, demography and the evolution of correlated genomic landscapes in birds and beyond.;citation_pages=3853-3856;citation_volume=26;citation_year=2017;citation_doi=10.1111/mec.14167" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. Charlesworth;citation_title=Background selection and patterns of genetic diversity in Drosophila melanogaster.;citation_pages=131-149;citation_volume=68;citation_year=1996;citation_issue=2;citation_pmid=8940902;citation_doi=10.1017/S0016672300034029" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=B. Charlesworth;citation_title=Fundamental concepts in genetics: effective population size and patterns of molecular evolution and variation.;citation_pages=195-205;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19204717;citation_doi=10.1038/nrg2526" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=B. Charlesworth;citation_title=Measures of divergence between populations and the effect of forces that reduce variability;citation_pages=538-543;citation_volume=15;citation_year=1998;citation_issue=5;citation_pmid=9580982;citation_doi=10.1093/oxfordjournals.molbev.a025953" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Charlesworth;citation_author=B. Charlesworth;citation_author=M. T. Morgan;citation_title=The pattern of neutral molecular variation under the background selection model.;citation_pages=1619-1632;citation_volume=141;citation_year=1995;citation_issue=4;citation_pmid=8601499" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=B. Charlesworth;citation_author=M. T. Morgan;citation_author=D. Charlesworth;citation_title=The effect of deleterious mutations on neutral molecular variation.;citation_pages=1289-1303;citation_volume=134;citation_year=1993;citation_issue=4;citation_pmid=8375663;citation_doi=10.1111/j.0014-3820.2002.tb00188.x" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. Charlesworth;citation_author=M. Nordborg;citation_author=D. Charlesworth;citation_title=The effects of local selection, balanced polymorphism and background selection on equilibrium patterns of genetic diversity in subdivided populations.;citation_pages=155-174;citation_volume=70;citation_year=1997;citation_issue=2;citation_pmid=9449192;citation_doi=10.1017/S0016672397002954" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=S. Collins;citation_author=J. De Meaux;citation_title=Adaptation to different rates of environmental change in Chlamydomonas.;citation_pages=2952-2965;citation_volume=63;citation_year=2009;citation_issue=11;citation_pmid=19619223;citation_doi=10.1111/j.1558-5646.2009.00770.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=E. A. Cooper;citation_author=J. A. C. Uy;citation_title=Genomic evidence for convergent evolution of a key trait underlying divergence in island birds.;citation_pages=3760-3774;citation_volume=26;citation_year=2017;citation_doi=10.1111/mec.14116" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=T. E. Cruickshank;citation_author=M. W. Hahn;citation_title=Reanalysis suggests that genomic islands of speciation are due to reduced diversity, not reduced gene flow.;citation_pages=3133-3157;citation_volume=23;citation_year=2014;citation_pmid=24845075;citation_doi=10.1111/mec.12796" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=A. D. Cutter;citation_author=B. A. Payseur;citation_title=Genomic signatures of selection at linked sites: Unifying the disparity among species.;citation_pages=262-274;citation_volume=14;citation_year=2013;citation_pmid=23478346;citation_doi=10.1038/nrg3425" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=E. L. Dittmar;citation_author=C. G. Oakley;citation_author=J. K. Conner;citation_author=B. A. Gould;citation_author=D. W. Schemske;citation_title=Factors influencing the effect size distribution of adaptive substitutions.;citation_volume=283;citation_year=2016;citation_pmid=27053750;citation_doi=10.1098/rspb.2015.3065" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=B. M. Van Doren;citation_author=L. Campagna;citation_author=B. Helm;citation_author=J. C. Illera;citation_author=I. J. Lovette;citation_title=Correlated patterns of genetic diversity and differentiation across an avian family.;citation_pages=3982-3997;citation_volume=26;citation_year=2017;citation_pmid=28256062;citation_doi=10.1111/mec.14083" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=L. Dutoit;citation_author=N. Vijay;citation_author=C. F. Mugal;citation_author=C. M. Bossu;citation_author=R. Burri;citation_title=Covariation in levels of nucleotide diversity in homologous regions of the avian genome long after completion of lineage sorting.;citation_volume=284;citation_year=2017;citation_pmid=28202815;citation_doi=10.1098/rspb.2016.2756" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=H. Ellegren;citation_author=N. Galtier;citation_title=Determinants of genetic diversity.;citation_pages=422-433;citation_volume=17;citation_year=2016;citation_pmid=27265362;citation_doi=10.1038/nrg.2016.58" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=H. Ellegren;citation_author=J. B. W. Wolf;citation_title=Parallelism in genomic landscapes of differentiation, conserved genomic features and the role of linked selection.;citation_pages=1516-1518;citation_volume=30;citation_year=2017;citation_doi=10.1111/jeb.13113" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=A. D. Foote;citation_author=N. Vijay;citation_author=M. C. Ávila-Arcos;citation_author=R. W. Baird;citation_author=J. W. Durban;citation_title=Genome-culture coevolution promotes rapid divergence of killer whale ecotypes.;citation_pages=11693;citation_volume=7;citation_year=2016;citation_pmid=27243207;citation_doi=10.1038/ncomms11693" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=R. Frankham;citation_title=Effective population size/adult population size ratios in wildlife: a review.;citation_pages=95-107;citation_volume=66;citation_year=1995;citation_pmid=18976539;citation_doi=10.1017/S0016672300034455" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=B. A. Fraser;citation_author=A. Künstner;citation_author=D. N. Reznick;citation_author=C. Dreyer;citation_author=D. Weigel;citation_title=Population genomics of natural and experimental populations of guppies (Poecilia reticulata).;citation_pages=389-408;citation_volume=24;citation_year=2015;citation_doi=10.1111/mec.13022" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. I. Gossmann;citation_author=M. Woolfit;citation_author=A. Eyre-Walker;citation_title=Quantifying the variation in the effective population size within a genome.;citation_volume=189;citation_year=2011;citation_pmid=21954163;citation_doi=10.1534/genetics.111.132654" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=B. C. Haller;citation_author=P. W. Messer;citation_title=SLiM 3: Forward genetic simulations beyond the Wright–Fisher model.;citation_volume=36;citation_year=2019;citation_doi=10.1093/molbev/msy228" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=T. Hämälä;citation_author=O. Savolainen;citation_title=Genomic patterns of local adaptation under gene flow in Arabidopsis lyrata.;citation_pages=2557-2571;citation_volume=36;citation_year=2019;citation_doi=10.1093/molbev/msz149" />
<meta name="citation_reference" content="Hartl, D. L., A. G. Clark, and A. G. Clark, 1997 Principles of population genetics. Sinauer associates Sunderland." />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=W. G. Hill;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=Data and theory point to mainly additive genetic variance for complex traits.;citation_pages=e1000008-e1000008;citation_volume=4;citation_year=2008;citation_issue=2;citation_pmid=18454194;citation_doi=10.1371/journal.pgen.1000008" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=S. Hoban;citation_author=J. L. Kelley;citation_author=K. E. Lotterhos;citation_author=M. F. Antolin;citation_author=G. Bradburd;citation_title=Finding the Genomic Basis of Local Adaptation: Pitfalls, Practical Solutions, and Future Directions.;citation_pages=379-397;citation_volume=188;citation_year=2016;citation_pmid=27622873;citation_doi=10.1086/688018" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=A. Hodgkinson;citation_author=A. Eyre-Walker;citation_title=Variation in the mutation rate across mammalian genomes.;citation_pages=756-766;citation_volume=12;citation_year=2011;citation_issue=11;citation_pmid=21969038;citation_doi=10.1038/nrg3098" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. A. Hohenlohe;citation_author=S. Bassham;citation_author=P. D. Etter;citation_author=N. Stiffler;citation_author=E. A. Johnson;citation_title=Population genomics of parallel adaptation in threespine stickleback using sequenced RAD tags.;citation_pages=e1000862-e1000862;citation_volume=6;citation_year=2010;citation_issue=2;citation_pmid=20195501;citation_doi=10.1371/journal.pgen.1000862" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. R. Hudson;citation_author=M. Slatkin;citation_author=W. P. Maddison;citation_title=Estimation of levels of gene flow from DNA sequence data.;citation_pages=583-589;citation_volume=132;citation_year=1992;citation_issue=2;citation_pmid=1427045;citation_doi=10.1371/journal.pgen.1005397" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=D. E. Irwin;citation_author=M. Alcaide;citation_author=K. E. Delmore;citation_author=J. H. Irwin;citation_author=G. L. Owens;citation_title=Recurrent selection explains parallel evolution of genomic regions of high relative but low absolute differentiation in a ring species.;citation_pages=4488-4507;citation_volume=25;citation_year=2016;citation_doi=10.1111/mec.13792" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=F. C. Jones;citation_author=Y. F. Chan;citation_author=J. Schmutz;citation_author=J. Grimwood;citation_author=S. D. Brady;citation_title=A genome-wide SNP genotyping array reveals patterns of global and repeated species-pair divergence in sticklebacks.;citation_pages=83-90;citation_volume=22;citation_year=2012;citation_issue=1;citation_pmid=22197244;citation_doi=10.1016/j.cub.2011.11.045" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=F. C. Jones;citation_author=M. G. Grabherr;citation_author=Y. F. Chan;citation_author=P. Russell;citation_author=E. Mauceli;citation_title=The genomic basis of adaptive evolution in threespine sticklebacks.;citation_pages=55-61;citation_volume=484;citation_year=2012;citation_issue=7392;citation_pmid=22481358;citation_doi=10.1038/nature10944" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Bus. Innov. Res.;citation_author=T. Kess;citation_author=J. Galindo;citation_author=E. G. Boulding;citation_title=Genomic divergence between Spanish Littorina saxatilis ecotypes unravels limited admixture and extensive parallelism associated with population history.;citation_pages=8311-8327;citation_volume=17;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Kirkpatrick;citation_author=N. Barton;citation_title=Chromosome inversions, local adaptation and speciation.;citation_volume=173;citation_year=2006;citation_pmid=16204214;citation_doi=10.1534/genetics.105.047985" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. Künstner;citation_author=M. Hoffmann;citation_author=B. A. Fraser;citation_author=V. A. Kottler;citation_author=E. Sharma;citation_title=The genome of the trinidadian guppy, Poecilia reticulata, and variation in the Guanapo population.;citation_pages=e0169087;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0169087" />
<meta name="citation_reference" content="Malinsky, M., R. J. Challis, A. M. Tyers, S. Schiffels, Y. Terai et al., 2015 Genomic islands of speciation separate cichlid ecomorphs in an East African crater lake. Science (80). 350: 1493–1498. https://doi.org/10.1126/science.aac9927" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=R. Matthey‐Doret;citation_author=M. C. Whitlock;citation_title=Background selection and FST: consequences for detecting local adaptation.;citation_volume=28;citation_year=2019;citation_doi=10.1111/mec.15197" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. I. Meier;citation_author=D. A. Marques;citation_author=C. E. Wagner;citation_author=L. Excoffier;citation_author=O. Seehausen;citation_title=Genomics of parallel ecological speciation in Lake Victoria cichlids.;citation_volume=35;citation_year=2018;citation_doi=10.1093/molbev/msy051" />
<meta name="citation_reference" content="Morales, H. E., R. Faria, K. Johannesson, T. Larsson, M. Panova et al., 2019 Genomic architecture of parallel ecological divergence: beyond a single environmental contrast. Sci. Adv. 5: eaav9963. 10.1126/sciadv.aav9963" />
<meta name="citation_reference" content="citation_title=Molecular evolutionary genetics;citation_author=M. Nei;citation_year=1987;citation_doi=10.7312/nei-92038" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=H. Nishikawa;citation_author=T. Iijima;citation_author=R. Kajitani;citation_author=J. Yamaguchi;citation_author=T. Ando;citation_title=A genetic mechanism for female-limited Batesian mimicry in Papilio butterfly.;citation_pages=405-409;citation_volume=47;citation_year=2015;citation_pmid=25751626;citation_doi=10.1038/ng.3241" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=P. Nosil;citation_author=D. J. Funk;citation_author=D. Ortiz-Barrientos;citation_title=Divergent selection and heterogeneous genomic divergence.;citation_pages=375-402;citation_volume=18;citation_year=2009;citation_issue=3;citation_pmid=19143936;citation_doi=10.1111/j.1365-294X.2008.03946.x" />
<meta name="citation_reference" content="citation_journal_title=Methods Ecol. Evol.;citation_author=C. S. Quilodrán;citation_author=K. Ruegg;citation_author=A. T. Sendell‐Price;citation_author=E. C. Anderson;citation_author=T. Coulson;citation_title=The multiple population genetic and demographic routes to islands of genomic divergence.;citation_year=2019;citation_doi=10.1111/2041-210X.13324" />
<meta name="citation_reference" content="R Core Team, 2016 R: A Language and Environment for Statistical Computing. R Found. Stat. Comput. Vienna Austria 0: {ISBN} 3–900051–07–0." />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=M. Ravinet;citation_author=R. Faria;citation_author=R. K. Butlin;citation_author=J. Galindo;citation_author=N. Bierne;citation_title=Interpreting the genomic landscape of speciation: a road map for finding barriers to gene flow.;citation_pages=1450-1477;citation_volume=30;citation_year=2017;citation_doi=10.1111/jeb.13047" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=M. Ravinet;citation_author=A. Westram;citation_author=K. Johannesson;citation_author=R. Butlin;citation_author=C. André;citation_title=Shared and nonshared genomic divergence in parallel ecotypes of Littorina saxatilis at a local scale.;citation_pages=287-305;citation_volume=25;citation_year=2016;citation_doi=10.1111/mec.13332" />
<meta name="citation_reference" content="Reid, N. M., D. A. Proestou, B. W. Clark, W. C. Warren, J. K. Colbourne et al., 2016 The genomic landscape of rapid repeated evolutionary adaptation to toxic pollution in wild fish. Science (80). 354: 1305–1308. https://doi.org/10.1126/science.aah4993" />
<meta name="citation_reference" content="citation_journal_title=Evolution (N. Y.);citation_author=D. Reznick;citation_author=J. A. Endler;citation_title=The impact of predation on life history evolution in Trinidadian guppies (Poecilia reticulata).;citation_pages=160-177;citation_volume=36;citation_year=1982" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=F. Roda;citation_author=L. Ambrose;citation_author=G. M. Walter;citation_author=H. L. Liu;citation_author=A. Schaul;citation_title=Genomic evidence for the parallel evolution of coastal forms in the Senecio lautus complex.;citation_pages=2941-2952;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12311" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. Romiguier;citation_author=P. Gayral;citation_author=M. Ballenghien;citation_author=A. Bernard;citation_author=V. Cahais;citation_title=Comparative population genomics in animals uncovers the determinants of genetic diversity.;citation_pages=261-263;citation_volume=515;citation_year=2014;citation_pmid=25141177;citation_doi=10.1038/nature13685" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=Q. Rougemont;citation_author=P. Gagnaire;citation_author=C. Perrier;citation_author=C. Genthon;citation_author=A. Besnard;citation_title=Inferring the demographic history underlying parallel genomic divergence among pairs of parasitic and nonparasitic lamprey ecotypes.;citation_volume=26;citation_year=2017;citation_doi=10.1111/mec.13664" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=G. Sella;citation_author=D. A. Petrov;citation_author=M. Przeworski;citation_author=P. Andolfatto;citation_title=Pervasive natural selection in the Drosophila genome?;citation_pages=e1000495-e1000495;citation_volume=5;citation_year=2009;citation_issue=6;citation_pmid=19503600;citation_doi=10.1371/journal.pgen.1000495" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. M. Smith;citation_author=J. Haigh;citation_title=The hitch-hiking effect of a favourable gene.;citation_pages=23-35;citation_volume=23;citation_year=1974;citation_issue=1;citation_pmid=4407212;citation_doi=10.1017/S0016672300014634" />
<meta name="citation_reference" content="Soria-Carrasco, V., Z. Gompert, A. A. Comeault, T. E. Farkas, T. L. Parchman et al., 2014 Stick insect genomes reveal natural selection’s role in parallel speciation. Science (80). 344: 738–742. https://doi.org/10.1126/science.1252136" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=S. Stankowski;citation_author=M. A. Chase;citation_author=A. M. Fuiten;citation_author=M. F. Rodrigues;citation_author=P. L. Ralph;citation_title=Widespread selection and gene flow shape the genomic landscape during a radiation of monkeyflowers.;citation_pages=e3000391;citation_volume=17;citation_year=2019;citation_doi=10.1371/journal.pbio.3000391" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=M. Tine;citation_author=H. Kuhl;citation_author=P.-A. Gagnaire;citation_author=B. Louro;citation_author=E. Desmarais;citation_title=European sea bass genome and its variation provide insights into adaptation to euryhalinity and speciation.;citation_pages=5770;citation_volume=5;citation_year=2014;citation_pmid=25534655;citation_doi=10.1038/ncomms6770" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=R. Torres;citation_author=Z. A. Szpiech;citation_author=R. D. Hernandez;citation_title=Human demographic history has amplified the effects of background selection across the genome.;citation_pages=e1007387;citation_volume=14;citation_year=2018;citation_doi=10.1371/journal.pgen.1007387" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=E. Trucchi;citation_author=B. Frajman;citation_author=T. H. A. Haverkamp;citation_author=P. Schönswetter;citation_author=O. Paun;citation_title=Genomic analyses suggest parallel ecological divergence in Heliosperma pusillum (Caryophyllaceae).;citation_pages=267-278;citation_volume=216;citation_year=2017;citation_doi=10.1111/nph.14722" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=T. L. Turner;citation_author=M. W. Hahn;citation_author=S. V. Nuzhdin;citation_title=Genomic islands of speciation in Anopheles gambiae.;citation_pages=e285-e285;citation_volume=3;citation_year=2005;citation_issue=9;citation_pmid=16076241;citation_doi=10.1371/journal.pbio.0030285" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=N. Vijay;citation_author=M. Weissensteiner;citation_author=R. Burri;citation_author=T. Kawakami;citation_author=H. Ellegren;citation_title=Genomewide patterns of variation in genetic diversity are shared among populations, species and higher-order taxa.;citation_pages=4284-4295;citation_volume=26;citation_year=2017;citation_doi=10.1111/mec.14195" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=M. D. Waterhouse;citation_author=L. P. Erb;citation_author=E. A. Beever;citation_author=M. A. Russello;citation_title=Adaptive population divergence and directional gene flow across steep elevational gradients in a climate‐sensitive mammal.;citation_pages=2512-2528;citation_volume=27;citation_year=2018;citation_doi=10.1111/mec.14701" />
<meta name="citation_reference" content="citation_journal_title=Evolution (N. Y.);citation_author=B. S. Weir;citation_author=C. C. Cockerham;citation_title=Estimating F‐statistics for the analysis of population structure.;citation_pages=1358-1370;citation_volume=38;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=A. M. Westram;citation_author=J. Galindo;citation_author=M. Alm Rosenblad;citation_author=J. W. Grahame;citation_author=M. Panova;citation_title=Do the same genes underlie parallel phenotypic divergence in different Littorina saxatilis populations?;citation_pages=4603-4616;citation_volume=23;citation_year=2014;citation_doi=10.1111/mec.12883" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=S. Yeaman;citation_author=S. P. Otto;citation_title=Establishment and maintenance of adaptive genetic divergence under migration, selection, and drift.;citation_pages=2123-2129;citation_volume=65;citation_year=2011;citation_issue=7;citation_pmid=21729066;citation_doi=10.1111/j.1558-5646.2011.01277.x" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=K. Zeng;citation_title=A coalescent model of background selection with recombination, demography and variation in selection coefficients.;citation_pages=363-371;citation_volume=110;citation_year=2013;citation_pmid=23188176;citation_doi=10.1038/hdy.2012.102" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Contingent Convergence: The Ability To Detect Convergent Genomic Evolution Is Dependent on Population Size and Migration" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
<meta name="twitter:description" content="Outlier scans, in which the genome is scanned for signatures of selection, have become a prominent tool in studies of local adaptation, and more recently studies of genetic convergence in natural populations. However, such methods have the potential to be confounded by features of demographic history, such as population size and migration, which are considerably varied across natural populations. In this study, we use forward-simulations to investigate and illustrate how several measures of genetic differentiation commonly used in outlier scans (FST, DXY and Δπ) are influenced by demographic variation across multiple sampling generations. In a factorial design with 16 treatments, we manipulate the presence/absence of founding bottlenecks (N of founding individuals), prolonged bottlenecks (proportional size of diverging population) and migration rate between two populations with ancestral and diverged phenotypic optima. Our results illustrate known constraints of individual measures associated with reduced population size and a lack of migration; but notably we demonstrate how relationships between measures are similarly dependent on these features of demography. We find that false-positive signals of convergent evolution (the same simulated outliers detected in independent treatments) are attainable as a product of similar population size and migration treatments (particularly for DXY), and that outliers across different measures (for e.g. , FST and DXY) can occur with little influence of selection. Taken together, we show how underappreciated, yet quantifiable measures of demographic history can influence commonly employed methods for detecting selection." />
<meta name="og-title" property="og:title" content="Contingent Convergence: The Ability To Detect Convergent Genomic Evolution Is Dependent on Population Size and Migration" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/2/677" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Outlier scans, in which the genome is scanned for signatures of selection, have become a prominent tool in studies of local adaptation, and more recently studies of genetic convergence in natural populations. However, such methods have the potential to be confounded by features of demographic history, such as population size and migration, which are considerably varied across natural populations. In this study, we use forward-simulations to investigate and illustrate how several measures of genetic differentiation commonly used in outlier scans (FST, DXY and Δπ) are influenced by demographic variation across multiple sampling generations. In a factorial design with 16 treatments, we manipulate the presence/absence of founding bottlenecks (N of founding individuals), prolonged bottlenecks (proportional size of diverging population) and migration rate between two populations with ancestral and diverged phenotypic optima. Our results illustrate known constraints of individual measures associated with reduced population size and a lack of migration; but notably we demonstrate how relationships between measures are similarly dependent on these features of demography. We find that false-positive signals of convergent evolution (the same simulated outliers detected in independent treatments) are attainable as a product of similar population size and migration treatments (particularly for DXY), and that outliers across different measures (for e.g. , FST and DXY) can occur with little influence of selection. Taken together, we show how underappreciated, yet quantifiable measures of demographic history can influence commonly employed methods for detecting selection." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2.cover-source.jpg" />
    <title>Contingent Convergence: The Ability To Detect Convergent Genomic Evolution Is Dependent on Population Size and Migration | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"a3m5sQXvKNjl_Q7bAj0K3Ceq6Mjuo4xdpTxCPZKkWiA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495540","apath":"\/ggg\/10\/2\/677.atom","pisa":"ggg;10\/2\/677","ac":{"\/ggg\/10\/2\/677.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/2\/677.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/2\/677"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/2\/677"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/2\/677":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022GLadU1sRpyC6vBXk\\\\\\\/ab1x9Auzm\\\\\\\/1BYAPRJaSUUD1J7hm1OyHrpUhARFU8MpT44lN8h2B125i3EISYnlTXPD\\\\\\\/viUINwmYGmIsJj3g6\\\\\\\/nK8noC8oE16\\\\\\\/hM3uaH7nBasVDxVycMSGrI\\\\\\\/SOtWYCnoBEr0F6gzncYsTBpx5yyku0Mn\\\\\\\/9nxXmsFu6bvkz\\\\\\\/qeMG1zjpkN33mul0Pz5brdxSPdc5pKZv0MPpAveD0WzOSY3Gs0tMO2KYvSp3c0hfLgqVsQ4VAHNkVDDIA1Im6ut599XI\\\\\\\/sOWODXufwz75gEeYX\\\\\\\/zb4O7QH5Yl8M1+7+87jwQmBEV91aez06RSYkNpnmiZJog5D4+zMbXcmkaYWBBOuhAnPsK\\\\\\\/xhQp1nvLzyW+gjdzgxNGzoImy0bLxHH1NY3t63v34yQv1z0aENuqFqifPULImtlDSB\\\\\\\/rq2ylhx5Bvk9UZyI4jfhDaWkUBoLNcO6ig1rzGhiSA8koruvJ55UjiprRf0CVnro0CMtwafsMWAMbgVwGslijbHIDUaRv1n58Ejd5+kO6nPb+inJoEsYahACD\\\\\\\/FuGQi6SMjKafPkLRCWXmGZWyN9aY4\\\\\\\/4rbDL9fDiRDpYGyYhTY1q\\\\\\\/uk6zgy6uj58vSeMteYu6frMIceueAoYHjsjD9NzJbGLtnXqSQnbpMMhJLQKsTVAspAMpf4qAr7pe+Bva41R0NpMrpth9UJMnM8lTAW79uZw8iGI+HkBuwkMSWICoHudCf3XAk1HxaXPph4szvOPq+Zu70XJhPjl02tlGlOMBsEP22m57e\\\\\\\/MOOI\\\\\\\/KSrqXNbwq8I5TOTMNvwf0uvozwpeL0OchC3gnpGDCUlbteZK0WSt3ePEFdngf+RJlV9OqdfmpTpCgoCCJ4FsQmAoIQOCmFuynu9xzBGvvPm26ZCbRuFZ\\\\\\\/7XeHG+FvMrK\\\\\\\/zyL1O6N30VwE0Ii7ylmmxN6nwoYJpskYQXkgzRupzoZBxfXei2XL2BSJQeqSYdADVeQshIaOF\\\\\\\/DhqxkZydqb8hRn5pBb6NHH3CYlf\\\\\\\/mnsuu3E2Hx54dIo29hlwp+u2LWF+8D7T\\\\\\\/grck132K4UftZ9\\\\\\\/HuJqaZzmiiN9E9fBJwGQMh\\\\\\\/\\\\\\\/MNzhmQ\\\\\\\/X1oghqIPjJlkl1V\\\\\\\/5a7kfsyYAOpEiALO8aYmFT7I\\\\\\\/wBJH24c6PukONM6EZbQtS2VgJm0IZILoX1qJmYof7qQlnjXQQchSfjHEdfIYC8FOIK7x5LivYxuUGHlBS+fgT\\\\\\\/NLbcNsELtdIijIkIkzW0MU0Pw2lu2LgW8UE5t2hnK8JceSRKucMQ3i\\\u0022,\\\u0022iv\\\u0022:\\\u0022oFyVzKXgax0BAo8ynWoFpQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a314ba17ad66e963518de14d652eb2de9cdcd56240d3535c35b3c2db629321e2\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PfL6DbniVm2cEZIFKl9W\\\\\\\/zW20EikptA8l2P0JVSMF1\\\\\\\/Y\\\\\\\/Z5Ugd9v+Fng7jibVAI+b7iAT3DBj9TqZ2Ezx2zjc2dmETdCVcY+PbznG5SzHOZ5qu3Aw19HWdB3Z11GIrCQI7fzz6bvuR\\\\\\\/9JUcUhYOlbTdSRJY9DoYEx4fAostSoHR+p1W+lOORLjvx8YkgQrMAoAYGCl9bnIlzmpTfBSFZmzJ0Qr3tg67i2LtH1klegHoTnqTyE5YIzb9xXDTj4LbQTnOjesyohEXb\\\\\\\/drJI5L+1U0aIXopBHzidQr75WHyz7VCDT5mItmT8grGdXlyZzVT5g0bJotRjBkDmi8CA6SpxLMdQpDtp7zeKsU6e\\\\\\\/OInMhUinEo0z9xzNrpjF6EmjZBt6nkmWSrnra3iAT\\\\\\\/UpTanP\\\\\\\/B4NHU\\\\\\\/OUop+RCSFFZi0WucabM+6otsJu+9GRbkrCCJOVXK3wzBKy14tE8tps\\\\\\\/CCMDG7Z5aBP1g7GPj2LyhsqzIoUb1vI0I7JkjOWeXDW9a3qARBhxY4Kk4+BDfXXuOedO6gP4GU533VXrRHkMw2QtzbyKirQStbiqRBsF0LWKPHnlp1snYM56Zah9FwfhfK2Q5cahhJoTIKGZxRnYXPJlV3wo+s\\\\\\\/9SYE7ofuFv9Dk6Mq7rd8eH8bKGOcPQNZNVJoTD03N3esHkecUOTl+DDZ9d1tzIowCmFTJ8gT9GyGji9c7hdscmegB4V80wmW+DN\\\\\\\/lCmRNcBt6jhwZtNpZVRjJc5oxO4CgQ6S16qB4lni4ag0ZRnoeR5UttOesHlSjH9K24FKLDQdDYHbUoaK5BS+3ZZ\\\\\\\/Q67IILDMrFN8OcBWjohxfkqW3zSVS3blS+k\\\\\\\/vQkY\\\\\\\/inIeSklZ8d2OjCySSXtRsqj6rBxHx4E49hugulJYYJr2GUE1XmKAo0GSU54yf3A5w4lhH5z7L0Mm0RYcNvOLzOKxYn4ZJLNBrS2rq8cRzma\\\\\\\/atfpW9HrdeyNA+Pu\\\\\\\/sK+4FrlaQOmhItA1s6wkRF7Mw1IXHT3VU9bs\\\\\\\/Lg+ik6qwG6tbfa+iGBaqFq2jBHuARXoKX3gvKhppkO9kOwuwVyMK6cM8s7nmqSzLcZuhgy74W1JOU5JDsjCbNyxPn0X4IIUMnS7Z+gq9DYbb60\\\\\\\/iKyAjHHxF\\\\\\\/drbEKkGZ\\\\\\\/jJcfJGXmJaqP7HvtOx12nO1jmx5LDD1vYj2dDAHzTBbQsKdl4hPTxJ+7cwycubs7dSPBFPWZvLr8mxePaWJ66VWZhCoDAsnbY2i7AAwL72\\\\\\\/LutV7vk6iw14vhEDguFPAUGB8TnhFWIcOGNCMSmjgTIpGj7vm89TfQLbF8+ZsPMw3W8McIbk6fD\\\\\\\/3YaW3aO6gpo24JnJ3\\\\\\\/tSQqamcP72rHejEGsHtlaDhVXGzz7I6zTnxpCPO4e2Im+4mgMJ2BNKrWbS4VX\\\\\\\/KFG4rBlA3s0d41C4F9WZdyDBH3dhnT9ObiCzrGJM4eWyWjMm2Xequi4EonxZ\\\\\\\/6yEbC+QGXenGrIND+i\\\\\\\/XMJxktIudV67Dhdawn9lx\\\\\\\/Ib47BJEgjILPRZT8g3OcoP0MlfIDyq\\\\\\\/IEcUgbmDUpI8GjQ5xZW+UaGaJeLiXNwLlIURkulHwwQFG0MwkE+BSbUuWLop0EtSyxcEwcgiMVnHv4fqRZ5F9azq8035y\\\\\\\/fFBgb37z\\\\\\\/zf1hw3v+Ua6jQ36iyeYt6dbpPWnWA+BCCWv1PepbFcGptPQCZX\\\\\\\/ZuT1BlyTaVPo20C0oZm7lX5QSXT8mClKI+xtlv2XSaW6paMklqES53R5+jAFd9EH0uU3o4hICHp2l2jAXYqlwYlR0r1dCMjRVtfgg1n9ZCH+p1iz6EyLFsGr4N4e2atd49ZMhZ1Ew3sF0zz40iBEjqsCQwkdAZ1FuSBSOUEnPPXgLvzNsgYVCpv+stkxOfLe677AsTl2GRVMuGPrIcH0Sv7wWyciUc5d\\\\\\\/fBxvh2XsFvlgOBGP7NLkaDtoUh8ukKLsPz9fZY0pL4Klvoxmk5eeCjkMt5EsXhNYIJ+2MvQaIstJguLM5\\\\\\\/r3Cc1gcLhVIarEUWwlU5llOAqI\\\\\\\/NvRFy8wNMlp7jkS1GzdTaSlECiRLAYszumN3IIV92jwHZY3xIGSA\\\\\\\/+7LMIt28UINpc2BuFnCy0GGzIGf5Ydv338GsJe+caSZB4x9hJ9clxVEwiCDdqCPJ5nRwROkzxB2NukJlZvMY6h7RpCq+iexJue6SuAmZ7ADbv7rVzmv1kh5XI+MNM\\\\\\\/V7SNw4VPDOPPc4FopHDdt8IEAHuCOurhuHh89hPpRQUD5vr0v4j1ToSvLgbg4p4gnWzErKq8ve0HHv9AgW9Mnt30lOeE\\\\\\\/Ie2eM0Ulcw7Z\\\\\\\/R+soTZ6reocbt1mlJfHDjJsolmg+mhmRVVWXAadd3FRlv0qtcRuKNmkQwweObwmFdBGcyF9gb+0GcW5UpcJnVlPJJODFb8qgRljeeEN+j25G2vfloc9lv+O+t9tzZZHwXENJlkigkv2jjnQ0l3XjDhAScOFjdC+Abts1YG15f3RTsKAsVCZLV3DaGgF8B6+j57xfBcbgU1isJin1Z3TR1rowdv\\\\\\\/xQxVQ4e1OMh0Zt1oQRS3cPIXyRKt1u2GmmRIgl8uIcBGJ35a4G5Nv4OsD+g4BlfnGDmdE944OyB6zWhvpQ1ALiOatlVDWMXdWKwUs9U7plZAnCPLfYZKrTniM8SthlbRNEnjliFstM09MQsmOHjG6t85jJ9g+mizcoGy1JirmWVmEXRvLD9gz4OaNU0Psz0ooDH9gZRnnTkW5XgI+AnkEeUYkJXkMbrHBy9Jl5WXiWPmBClK7UU1ANdIJP9TNNEdrhayjKOEkb16pmc+jnuL3sG9KifasKOYmpaLjdeadlAI0cv5Kq74fyGLFWZ5Am\\\\\\\/iakAGEgiEpjSQXpI+nM4Pa3rekJpD6eg6CENefBZuGVsG2pLrfyTV2UV31z9fF+9cwrsOdhwRigZCLJiW1tEvpHRtGK\\\\\\\/4J7aFAh1tbz50X5wb8RamD4h6dql4q1dLe8vmYAHdtqzcx7JB4TBg26JEQ8TjmWDV\\\\\\\/vGY14t0LkO2txuFbRzi2+gtLu+M3+yXV4Q8ElzWUgdIASTUG7vEq3FNbQSBAKeAEAwCuAeghWXOe91YlFd97mOY65geb6QRxIrrPyNbVkgq99ZUk9fJu\\\\\\\/mX8YHXFY4WiPiOfounURxF3ICeiMG3wGD+Khs2nqA+zJhg\\\\\\\/To++YDWNq4dshBLSOAQmNYK\\\\\\\/h3R6yGqNB+H+RPgTnURgNbfR7low3zxpT5KdqoOfVsrW1YkX8tSqfY1TLlfB+ctZ2wBqfhXZ8yA4jcqD36vF8PhEj+gmQiHPaMlitXdj74UI6oHIVUuBir5ZAE876LeMJieddtl0hqi+WWsindhEunB\\\\\\\/ufKr96nIrLsDo6YqH8\\\\\\\/UA4ZVYP6a530dQXfCxv\\\\\\\/TNJo2UGSLpKn8A66d229yem+y9hhSLOxkbrZbLcFXXgEX2zP9juYSdv6tHXirdPyC1dd4vqUYmF3rRYi9qmR4eiPRRcDJn30tOY\\\\\\\/DsgimV+aU49R8lbDgumrHtSIyUwNOsBErRf18hN2PUQZLINBQZRTWAmkNRr8b+v+CTxTEPnjD\\\\\\\/CXKCs44m0WX4v+\\\\\\\/cvGwPSfA1W1ah5S+uEVEPeYS+ieYxjNawBMaAKcT4P6TkyDTeJgSU71E4Efv+G0+KfAjC69aIwS5G2cn3hamPiklCNQIuX6ZnulmMh+1it\\\\\\\/gnocxMYAE7vKCaPMcpwyLB\\\\\\\/4ofyW6yspibViT5sPB\\\\\\\/2oPN16HsOWuoMLHau1zCjkvJ8YOHdTdKTPoIQl1zKVwZ5NYm6952F3jJIN8xCWtMkkYJOD1+WGh6Otxyh6y+2hggI61uqb5Ct\\\\\\\/ehCyzUJI0a3cfHcSBq8oAwjTFb7aRA6K2gtJ8uF+XdDNazRL7LV12uKDNOQC9LbYr\\\\\\\/CbNw\\\\\\\/FZOm0ZhqAD+TY6PkHFJmbLNqI4a30zor\\\\\\\/15LHqM1MUxENuD0Slw4hO88ptWUXL8EIZNfc0DiKbdYWLx3MYEyofrTbKMA9snvoJKmjUx\\\\\\\/i8B8exSGFTmQbgocAIzG6QcT+TzUrpPt5wRLmZFyZ\\\\\\\/XCwUCr9\\\\\\\/MWBbXlcuMaajRqcIy8Y30p9+itLODLi7jCzgxrACmcLCO6MJ3injrRb1b6URzkxQzsoWm6prCw+IWIZEUkTOIA2Dp1cNRjrS6KTf98tQUL46RopzvyrcGYCPcEA9phhKUyQDx8Wv3LNbnYeJQw8IWZla11F\\\\\\\/hAGPlY\\\\\\\/W0rKbxpZua3gWYgLXjtlN\\\\\\\/nXhoYPji\\\\\\\/Xq385fH03bfznWsB1UNynDU6S1YA+1OjdOEmtMbu6h2ZAu34Jzjrh\\\\\\\/zwatGyeX2zsU7MFyFbBM+vOJh242gFbiO6JRZyOEVFhsFkPfOFVF8aSoKAuseid5jitzCy9DrmAk+Falhi2DmrmXPzdEVtzDSg1ilHVQ\\\\\\\/OAjhFH11RkbHB2XZXv0inQKvXx3nIOv68u7zACb0W0NgX3nmncfJLnt5fYMtegXTH1OKLzptBXcPPs8q+ZWUHc6GGPSymtHlIXnigkkSIgt8ql+oRcS0ZWtj49A22eZGKJgJcoEELrJGXKptInbs3uZno9K+9UKfwlnKNotnb1yBTjgvp1krUK+EO3A9att1CQeDVrnBkbGJ7lQgw0S4QzdsaI\\\\\\\/qjbe4v4bdJvZ2pmdkZas2nMcbYAP6JrhRQANdKKyd9dHqj3DxBq2k147GUgN8NAqgGCHLuk3YDmAx9SRDXLKFTavEkjG03\\\\\\\/0IvgMAviPefrsjrFE8K2dgxapAAa6oGnmTXtW7KsaoJDCYlOTK\\\\\\\/Wnzqsvg+yf3mVS2iQV0TYul2wHfrSUO7cCPKpdVq3B+IcUmvDWmdhONzdTSnQEwztNYK\\\\\\\/f80CP+nNfSxdUfAhPNGdOhFTkzcr05mkoNghO0vA9mGg8YIBffvVBCQbWGaMK0efIpQp4Hv8Qp89BpfWqi9Q+ZbiBm1quGUMfR91nZdazt4+U0qoyRJanXh3kguEWyMsJi7IMr24tkmqBRZPQb\\\\\\\/zt4vunY8BIRpHI8OBV5rBMn6Kzc8Sq3Qa00QnnhrNjTOB7AzcnhSMHENm1A4uCQ7qs809YX0D3lCAUCdwxT95lx+H6BG1GGfyJJUQ\\\\\\\/FzpBOLfRWC1ux9S1ICUYTMbXgDKUuPIksiz78HWXCLWKsBkGAa3aTkF2Fw0NJVzy\\\\\\\/TK6jHWu5bLYoM+BCkwWlFg+Qfvf4VZR5rXrkGthL0HMblrRw9CSsxrytTat5rtqATyAmahwGPTNsJAgX2vTei2QnzriowguvB1yHVhpJe3e9g1qIrxQ9jGHhJTXeulSMN2iXgUn8Bb8odBm8PnfIZytqHFL1Ox1bT8hZv4eYknbhjs0RP6RmdlaroCjxAgdGfNV2tjNVzYVSAxyRueGPuI0yQKXw25tWTjQhGjrPNHJ2\\\\\\\/ayMT9SuGBE5VINjjXrWDXfFWM6pmMDwbJuju9igdDtUPglr0V5xFqVogaepo7+hnzbSyduSRMu0OizWOeA5bvGpN6ciZSFpisTG9hPHt+hUzEd1C8Up7LpuGbThvUUUHPPAAYIrp8OYlZuruM\\\\\\\/AGZD6h7x4E9b0GVVMcY5zOqtTEkOjWQ0y4XUr9cIAKHng+GYwEx3orNXb6PRuqdE8WHzqq2xB1hq8cz90+qN4eg4byUFOa\\\\\\\/rAGNQvZ7fvh6j9XsJrZPOreNdDLNf7ponhzyZTuDxSBiuxyZ4tux4ACcZUZaEu0zJVkiIAZHpqOPMMZdODjleId\\\\\\\/g4U3EaplKaxgGxNav9Ba4zGhAHDAixf5oOgWcmY\\\\\\\/q59ACmrSWCe9zU\\\\\\\/i17j2LezrqPyInS9SLcl80arSrh+l0bYKsTW+qrhhxTS78xbuwkzsiK41VvZXO5HcYcdZ606KCq+CbMmxW2HGmCBY0pS69SIyC9fH6vFWKCAk2KSdpQWQSsfUbQAhd2riYRePEvxRDouKYt5E5ZwK6cz+0ziPKBvyyHXk8bzBlB\\\\\\\/Qz9ep6kUwsLtB\\\\\\\/89H+fpueE8buHvkYt279VidLF404yaWxz5mG8Aq4By2ks4AFyBh+6sjauPhrQRaKt5YIfiyAhECkAZtoQe4m+\\\\\\\/Af2QxKmGcFZCc4Wn6zkfBE4dW36nG31iFKxR5sw3Ah24JLpplLouSXGxagAW2Vt53I2WwDbGtz+ex+SwVPNlTZzLb2bYI27uq67l5H9ZARYSe2K4L1vMsFp4ghEzb1PronDav+fiFaSgUPeJc49Dp9Xh4BgyrXPH7l7ckdKF\\\\\\\/00TEYyt7uCa3vV0CMOXu2T1uIHoN8bF3babPjpuxP8bbf3Bgf+rwPqJ+neQhwV3BW3txpNA+QapfCpLHNCcxI9RMbYAf3OB0PSMR5iVX3j1e7rZN6GB7YrzDrSNJX36modH0IjXQsibTDC3SKdQLFlJzLXpPiwzDLuwRDeMgXtqj\\\\\\\/RZBMT7TQmAcfopfI6s+fcXx8kvBuje4gywaFNTryfRFiSuTDD6g37tprghO18BKTfU7oNRPBYQxTwhYWy8PWap\\\\\\\/uczHwr751LZvqc0cbrzOnecaltBw01jtSCyCVpyU2fbJDOd9cGkGoDwFKH8fZsPQqhbnISeow05mGsMGIzqVM\\\\\\\/nLFuZ3zExRMGQdx6OuQ3Ux3iUkLlITaZ5dJk9sCFSPIEmpd1aZyX+W64G2raROrqyiGW8ALMzk0X8nXfj8Q4geXg05donKOtNuAac9KEGkgXVLdWHOR4SF7ZynuAnrwuuwmHRALSQxYkbjR1oG\\\\\\\/N5D\\\\\\\/m0XU\\\\\\\/fR05SoE1egfd4l1hTI7kptWERAstGRcrfUQ4NNN65PJIGUTqdiUZpDQGddUOKavxYlrt91tb7G4ruSdlviwZYisait7OI+uLQKI71+29XNw\\\\\\\/mf7zXBAP2sapd5cnPqq07DWGM7SuF8\\\\\\\/31uRdi2ebmEJ9nHth\\\\\\\/DlbGrkgtq8j9Elpel7gyWAKVju0bQ46IDhOL07N5zIfv2H0BEJRqn3AZEk1\\\\\\\/ocyGobIPV8AJ2A+tEcVx+2kbGI+fZCl1Zv\\\\\\\/+dOlmVNubxy2ZBikoARqOLsHMckc86+9MwfzJDpjs7KoDBbC1hKacNjSXskNeOvlMVpouzqtL+l+wK5s+dvcxt47nVuLrEpShfubUlA+Pl3fmubbeTvQal4INmGD3RboH\\\\\\\/ImGHMZJ42kTIFDxgfSSwXDeuzsB4u5RXVK3bdmvXfWIipg+pUX8VlL10Pu6l7K760A92ts7DLF4UxLbn1\\\\\\\/xzScUrybSvVlOEGN53QYHqySQ7uxdocueulOJIKH7KKNXMfgcw9u5eNXQWqv5gShXxW0LXNDIYBSkpCpm05FW8zUCPXV41SAFf7LqqrrCJyXMFoLgMZg3OGOV2oE\\\\\\\/pLuIAnl9AjixzvNcIETkBYSEExaOMaChvuSz0pxNXBT6pAgAhvg7kfpi9NnTAxQW5mFij3\\\\\\\/ZIgXelS4r1KkyX2zwoGyrM7jR6AieqyDBgLLX7pSjfA3K0V6tQX7trxBNA69oZcqZqAfSBuGgUqGVJVI+OBRHmu9k\\\\\\\/5Cw2uozd9sm2VHyCY5fmBelYqxog6\\\\\\\/kDGxjObw53EJTsp\\\\\\\/8xqrBkxiQDZUCX1EmrZt08qDtkuFTMEmlhI\\\\\\\/fBczeo8DZBtE74Z2RR3VNT5ZE32cAMxOEX\\\\\\\/rtlDKJJ+rO10KrQDpecdazSV5H6kjFR9ttD7Sz4Gi3MYsv2tgB\\\\\\\/\\\\\\\/K8qTVS6nrUk2\\\\\\\/TBvgLH\\\\\\\/9s4hHnvClXPoqqzHRkVY0Fo5HsO\\\\\\\/HGgZwTkyVBlFIS2HberVvjGF3P0hue8sA\\\\\\\/w9ansflCMk8SC0tEf3tKXZbfwVOzyEwgk\\\\\\\/\\\\\\\/non4t9TJtq9j3YxZlxxjAby9+9cP+vnaa87udMVugXbzxGyOo2G7KbSRnIAdkXzOMj5Seu7HzCCtNrxSMN2hXSG3peKTzo12tXAXWPFH7SUgi+Z9GtWCTfD4Ia0bRIib6H7gTFI4pawuI7WNHX3Xw5h+lg2m1nEoR2GeIOIwFKa2vB0jrL\\\\\\\/RJmV64GlSD0nxv+fkD9Y+RHAINS3qvhMLL69ZH0dt\\\\\\\/it5JhNYeeW2fizmPsBPBgJOnVAz1429ld0Q3Fm8ng\\\\\\\/JflHdzLgB6e0G291XKbT4GWXqBBvoZyC6U34HxaZ8xEI3XnUH5D\\\\\\\/cu1stKAeNeD4UmPEw4C\\\\\\\/v1mde6Q9l+nS6BnBKtXfyagjVAFDAztpUS1pJgV2W28yB2hjDuwf94PEq5ljYXZkEgCKhsYh8ZPOlqs5UgnRaVxsnWJAtrILinhMTgBzpIgkKu4IOT\\\\\\\/hyUUvFWg7BNmtDDSEFcIqff+xR1Ji0IWXvpv+7wjliRTzvnO6bLmvMDTDJ+5uHYjknTIRBRR04KmnMBUTei93CQ1iYm7gSyr7S10HqyeolslCmZdff3DeZZW5c4UTmDokThxGiyS7b7ttEAn\\\\\\\/drSQcQgmSNeFfGJTm5KFi\\\\\\\/qqk9eSR+J7bIbE5YmGmp8l4J+R3weZ4sL81r21DRyvbgUpwx\\\\\\\/NDyPEIe7TNauAkIncsLLVg9EdiFIM5KHAOcB2v\\\\\\\/9iQY7yhExBasFkme8tYuaBybiB3HC76FbLe5Gop4vwnejPlutd+Ftx\\\\\\\/m3Oug60AWBc\\\\\\\/z2l49u4mz8Pi9rFvurJD5G9zvwK5p1APPxI7QC\\\\\\\/dxka9BzpCArzx8JH6phmdjKXXi1ixRDeOO0ykT1G0VngZAldubKhxNpdtgjcy0\\\\\\\/6zvuTOITXd1blJDPkXRyKAS5ubjkmYY+KyafbAe2v\\\\\\\/c+Xui+MmAy42oH8OQAWEIzrlPp\\\\\\\/laSXkuYy+sEN\\\\\\\/F3pPpM+9LhIS4Gapz9En3vwSx1GCZvLRlx3SHykSCyXKhPV1HIeUz0qkVI1Q5GHk+11qVfSKSHa\\\\\\\/jPh38s23\\\\\\\/KsbGN108Rd\\\\\\\/ptmFIF2Uv475wIwtK3FNXmRqH5OpsspSC3NT2Q2LN1XdrSuFzfhTDaQyrLF4VRx5mQrudz5+uj9GOB9+8PxRzzKEouci3aMc2QecpuIdwrW5DVCBCxBJPGL5u0czedjIT527olf0AwqvoYdcQAc4lLujO765gtr2n1mAdo8VjplK1aHd4PFp06wUeoVQObZR5NnjDWsqYM\\\\\\\/Hj8D9PtSfvHyIvHlSlOMrjckjAgVDBiiVUTqjWvsTcnZH4Z3CIXqUPiKcNLbm9CMBBotBgD4MVuOwOVUuPHeWspAILz3NM1Nn57m8xTFHp\\\\\\\/ClyljNYNPuI\\\\\\\/VvGxUVGfjxNubxdyBhISeTWdTl9eWXzSsU96FbHjHIAKvA6c66YePVjHUX2Vr5LuUjdham83W7loxX9lYIyptSGlBcNjNeewwGsZydEWBSRHQsmunMQVEs\\\\\\\/vfyYklTzPHfXbY\\\\\\\/dltXIU7aHLfuQkaE07+FBGnRXOHtRKeqtUeL5nIJRMSHXBGZSCB2cSKEiwmeBIIsnbFDEfqzRCBjlFauDs+TFnkci+kzrPXy5466kQiXDZ2hSf5ZT6xcXX6LTdj1LTVGDCATbFE+yrqoXyU+lftL5yG0TIQDbmH0PR6HGc4qdy+x8DUP0n7mLj++2cPVHjYSFCOTiYMYUyahZpd6F+yfST4yeFC0ECbOe\\\\\\\/v+W5PZQjKtHP1caOrjNg7H2jDmqOO+kWaF17n4aF6X6A3wfqf8yFjY\\\\\\\/V3P6KoWDSRiRXMTReySw5FkGnsQ0x\\\\\\\/xY\\\\\\\/vgNMBVYwfpOkJMfYgcuEE9Lkmxf6\\\\\\\/AHKVqxG1QsMT4NzNTehDR2Qukng4wgpZUlX25W+5w4M8sMiZWn\\\\\\\/Vu\\\\\\\/y\\\\\\\/HHqTpnTFQJVOXkU5WEYK\\\\\\\/jHG2oA4jf3AvCIAiql+iGofXNI3ZZelM1\\\\\\\/vVW\\\\\\\/Kdco\\\\\\\/crtheMXizAqf43FWWjsew9CHndRoIwguyDH4jHdwWazybCp0V+lQ0Hw8qMaGL8aSIN++ii4c3bsQxR5xzIN59W6tH+w\\\\\\\/tYu0F783fpYAo9IrI+KypnoSQ2R8AF0BzC9kowFEE9+XocJyNOTpBEmskILB+DyfyNr61DAph+HhQWMGSLai5AZ0DaWVAmH7Q3Uyfo2OdLRLTm2klByBmSJHBW8q1I91rA11R72RdNJP+McKVE54Wl24tc8d38MHH54CqTS9q7vRSAkjBhiu4wDbHfKmeGY7QrmVnR3iqvNV7OBOR0SPzB8NfVz5PhaPMXMnLEngQSEgWbsKTpUME2\\\\\\\/Q5\\\\\\\/+8o5ZkDU9Cdyzg7HweiD8ffROLWVY9Oe0BNlRlPyAl1IFMmJ5lb3kKaso7iWvoGiOk7\\\\\\\/J7iVsuMwEF02\\\\\\\/Ydxv1wSmHqQBMhybp\\\\\\\/kdbJhy1UvL7y0zdkD9JUWxppo6xCBDfxVWXa9fBI2IE+ZL5p7bW+e\\\\\\\/z0osH7+RGdw3SroVtxW7JZ9XWJX7qY7coS+97rlEmAS+YjvHmsE+QGmYXQPcLmEQcSkLlfyV56HXFNa+4k7mY5lcqGGeWSev+isybC7Su3OpUc39IP9a7BbrlKQZ5UukPtCBcMN7DiHfkN3pgePThGSaQ2QpL9jv2l9+o\\\\\\\/lNk\\\\\\\/ylDhad8N+FHTSY4qKmdwnBEtlA+axVSg\\\\\\\/rtdF\\\\\\\/KCzJ50wssOee1c97l4yGCD2Mp5XMila5QJNPO3wnPN2BaRBPU9d8S5Ogmj7I4Um+\\\\\\\/4z7TtiWcT6g\\\\\\\/KEb7tishcGTTweE+ngG7XNLtIdb8dzXyE3+A7gp0Y6BtOgtNTHmr+Movf+pKX1HGnpmt8KupW4u5nUbykaWTPl7Ocmk1UIvUP1XgasnbN6rofCAteEFm+G+mx1M7BctZFUnNMs9X6Pl9hOe4uckgXm0OTE7NGC4Tdvq0Oof7I+0h+L2XpEbopgcRq\\\\\\\/cQ8FZngGm0eUXGBSQmIVJCvxh9TxhhNGn4XRPBgBdGrS5zi6F7GOkqMaUw8s\\\\\\\/9Ru4f54JYAFj\\\\\\\/7qXdRKBQf601MfZGSJpkq2LrZoWzouNndgz6aCS41bUVSZcjmIwp1b1322zC6WSe9GSV7HAmEtedmOOcTYMxDjjEycEkDPwiLWi6oWFuchiw2ObiYO4QS40VCfb16BW+UWY4hxIpW+aMukibkVQPrCpI0oPbrmdTtPpB9N9rVDZsPxPDkz5ikCBQ5epcC28iCU9pQ8h2Tvvxxa\\\\\\\/cEVNQcSIC0wOdSOhrgYPG1LP3e2HvHBZjBOAyNmVoLhS7APtex\\\\\\\/5uMFkecSCkRp+4AoDNyc0XCyGWD0FYykoSmZ0LG9RAOPOXnv+0PFuCRyrl1sRlyedx+\\\\\\\/Sba068LQeOtjWsAik56cYi8YC5sMVCAMtTXh5h4fIdv1IXxd6M1zyhj7MAgwRvY0ofmN3VqFjV0Qdpi2vyHEHyssMAcEF7cvKLHoOi2tJQ04qlowDBXxzrsLZYQ1GDijxXc4aLcZmmphY+Q3Ov9rIXYS5KhMfer6r5BhE3ihSrnzG53kQ3laJksA5mUMhFcuiBUv8kysxvjUvwHn5YPmwhjOdXJtrP3SAD6QbIS8kFGL7rK3NEGwuFFi7Rq89pnSch1nqkRFv13m6uiLa7itJUt36zGQfceuRXgT9rzscHL7IhuOyzd9XeGKfRezkiHAvBnWq6eO9X5yrkWFoC48O3v5MuByHUbiGnqxY5ntkiLEN9jhysOkjs0IKJxc1\\\\\\\/8ubtyLb\\\\\\\/uORoi0pCaQnumihdCmSEiOGf9jWtasQeaD8GVTG9mgmgFrBHwqMmldTuq1kj4gEp5hWcHEWhc5Qbt4yJtcU9KaundZT78I0BY0wuVUd7E9ll4dsElX0CN+UGFAIcxQo7EvypAJ7fuFSKSVWKu8aqYYRp1Uiq8BmDbZFcVZEBjGHKfGHY53f5XG6EjSO5JhAWv+N\\\\\\\/nsAqCJcFzizZglPxhZRnoviJj+JYEU0dIhv1WKn5lE8G4ITkgwp6oJzZurU4sAwvXrHIAd2NgEI5YTjoCiaup7rupRrB6G83Q0TExbnQ72bd9Ufwr4YMhGD\\\\\\\/B+fKBD\\\\\\\/fMSJOIHdrXSS3UPtO0DlLGsBI\\\\\\\/9Ihq4tUFK6xZSKgM2ZES9ubO7S8o2CRrZDXnZ+LTpN0ZC7Bbmx38E+AqCPrfYtBdFHK8QsLnL1RJ1fb0EHoADLtv1jDsw98v7Dk0814Ogb4BmsqQkonjyOgpf8KAW0vWJ8QmTy569lm2PTDZjJWKsBO6S7jE4SbPBbuGd20hTuYdOwPfroiTXl+\\\\\\\/48GdXZ+k8Qf70H49oI3LubXhsgdKAJZA0MJ33kggcx4nJoSVi4kShWvk9mlcPn10+VDJhzmZYQncjw829aUi6fNufbjJXk3v7zFaxEZXPTDAKQAIqAOSRwAVG5UxvlB5S\\\\\\\/E8N3yamRQ3zw1NPArUIQ5B9ER651NLZtjqIvrpblKez88DJev5wf\\\\\\\/WryWp1Ej6k3iOOdw48g9hhQKZvDZUhwxG6cVh+nG9g8xDhP+kv5G01areoJ0hzv64ir7x5V+ytm53qMUb8F4UC5i+h+S4Y46h+9oOmR+GJiv3DzRO5fY2Kqib5Lm\\\\\\\/mNb5bkmNGkvIJlvl1+sZvKf9qznOiTI18wnX2ybMMLvKXceUj4\\\\\\\/cQFeMSdEJ4Niv++o4fbsZ7\\\\\\\/g4bjk+HTtl3XKE13nZ8aituXP9fc21C0eJdqODkck9YHrar65fAv+G\\\\\\\/argH\\\\\\\/NXz0JJo4HgnlGgK\\\\\\\/Fy0LkRRpK8KCTlFDuKwqdUM0VK2BVn46xeuKmVgycOO3CNNO1+XQ9JxZdHnW9VrsV\\\\\\\/2IEALQsGdXt1SjGN7SCTTbTRM+ScuVg0Z+Y+mFxJVyJRWMrM\\\\\\\/oYPdXKZIDuVkMPxtHrrdXDhISUumg5D0fWutdKxHs\\\\\\\/02HSTH1\\\\\\\/hn1VP8OrJaOGptxyI2nLRqy\\\\\\\/GkpHfUVcyi3eKwQFmXBS2TjObd6cchef\\\\\\\/ObRjOmn1vUidYGipD33h\\\\\\\/aZkG6Q+61H2ROYaRVUQPJ7DIRs8\\\\\\\/us+4ug4wAheBKGEjJiYIxIJeyxRxM3\\\\\\\/Kvn0gjrRyIPo6tMCuTfC\\\\\\\/L17SuiNWQect0DBdPhjKl93YYqw5EW1NPlsAOCQWfgU\\\\\\\/FBsOw8edLBhfS5Sza1AtZRCCq6+mgYnb0esVz6nQS\\\\\\\/Hjh0G65Uqg3RuC5sUPb0qQCGATUs0iRx0yOcjm+tAduFURuQ1YDunnZMb4d4v9JABDomKkyiSLol\\\\\\\/oNWxOpGb85YkI4ZKS+QRVpssTcFVooP0jJ+9RAewlEeCfY3O1Zzk8oOhpr4oB0tPVxVWvhXCIuKnr4FsetVRvOI3+N7szw+NgAtLwRFGW8BaEJWWEc5kcmtFd69qrfxCZcvWP\\\\\\\/CZKxsZvnx\\\\\\\/ZkENeheA6BkFQG4t7\\\\\\\/jcAZ+T6ySdQZj+qwDZcrw2Zir5PslSjFbZ8IId643p8fTvA6m3YxfqXaJRvpBP6RxKp9wTyfbuLVNTwmj2qy2UwNYNIXoBlWYuAmsCtADsD4gf8drHdTEitQgfkY5z1s8HzQ8GbTz4Q4Z+j7v7F9nR9a4gmSWxvCRLZrRUEOvJYKYUE+XpDGpC4ipczX1soeXH3o7V9ME+K8FtZkAqYRUvfp6H6GKyRCMcHqdI3yQqic3iP5MMh\\\\\\\/OC9bIDq8gyQogbucIFBEeGAhZBFxDUbXb4RCkQ0Tjxx61d9L3rs3FBxPNNb0xpnuUefvaAum5223Bh8PX21Oo4W+aZ18SLfEXuNzvgIy\\\\\\\/psSX8O5vx4oEIeWTgWfIXwnY6Om9Zpl4nzPBXewF\\\\\\\/9o4cYtrPPHvB32P7DwJM4f0ULor+VOrfUYO\\\\\\\/nhERPRzMWx2gbl7vr\\\\\\\/eETuFzHPITROzXrAQAWNOLbHGkemfg7fSinckiL20pokL5k75jxOVlBKzB9pfPVPVLuJBh2uE2Pu6eb1zzcaR61wqAqX9OxV5dLqEpvCt7vmHPE3BPtqFp1W\\\\\\\/SPDEnojeGYs+GrCPmCwujjx9r2BecFJGxAN7GOMMe1zp9ShbglCJex+rzN1hZz1g1xiQ\\\\\\\/3dnpaF275geUXHqPsA9LdbF\\\\\\\/bJJGr8Kk2pQYmF3kdZw7\\\\\\\/hSDtiESvtvU0C6WhAhGdl1m1mPkwCgiqsqvqUYrKA8DPDGPBRKxTpL1OA4w4myfPFpCOIfil3vOj1tPpca9gjNAezSKgbjMnugC5eqmbmULGy+iKUGvNXQXfWjmZEO9jGixlQlcduTBfIlpz6QZk9RD+Hz\\\\\\\/Ly3jylCXxWAERwA3iw2JhuH29y\\\\\\\/kb9ZQpSBXV0w2qYoc+S9Guj8JasG4nDWfasz3DmncB8qtkRcp+l9uRZAFAYALQexouMsOR\\\\\\\/WHMdTMLdbpkYLdmXkc5j0geF3kJ4rlN3g4QRXvEQ6pfzPjN4oW3+68U1v3xge4YTgv23e6c0PRIT++zJLFzENj+OEULHouoscHw0H95Ssa2USwAXH0n7JlFYImgarI+lD8qPlCJSeARNAQeTozKq+5PqKEwwLe3qNwawYW0K90RZKTKl6UYxeC0Vy1ORrkIhcohwIjqHXDxjfQllBSVv4+mQSxMtA19wW8obKdPpyR0mOG6VwizpVPOZk6KlFL\\\\\\\/OxFZwYK6Kx7hYtQnYmWi+rFeAiFhbOyaG1qEHLBeoH82kSVCva7KQWT7KqbqHni27Tm0cv5MVj2n9oXIzkuPSiX51lofqX2wEkzyOkfekw3vXZ7vDMoeOBU0\\\\\\\/CcQNaORoHzqKWB05dkDwscNBTTAUVdo4cUNyzpVuWgNdd8biOZhxnLAY6baY8ApVf8n4K7dcb2Qn3onvzCSU30gnGrS87K7Ehj2mYAVuOCr2oVjw9Pa9mjtc07K3ELV4xJJvtsoBalP2jgcnXESGm2feH2dV+1vHBt9O7kzLUwuuyDGZwYV04hNA\\\\\\\/XEcrpGq4NULSzIk96Og7eX4JEewYj9ep050hcx0\\\\\\\/bNteEZQV\\\\\\\/QjY9zGgEKeCvkaJ9KRFVd3EnMp6dJxdiZ\\\\\\\/0sbvOxbhFgjXmXIz9fWsyyjIv1E5\\\\\\\/KPIk3Lj2WqtlvBRxOlTR0obuj6Neg1m\\\\\\\/+1QUO8Vn69kEPUQD+DdMdEKix4agZwU5bkDhZSAvmCsZSPPyJjs3vc19vYXFDhHCIzwM1Ariyfxd4GQ1wobNccWdwB\\\\\\\/4lcTcga0nRqbYJLHb0AeHH\\\\\\\/dkEbAwd1JJPHfUB93+BNZh7X8pKoQ4TaMnJzipeM2vgIxcCB7o1Dhaa8u+\\\\\\\/gVKtWOzrmu6ABhm2DJwYPcTLKRRSOoCOG8S6UvlPiT8s7dxYe5BhgRY4\\\\\\\/Zp58MKT24Ft3o5Tg4xbBpbjxC8Ae7H4KiDTJ\\\\\\\/ImXY3xS5L0ywYgXkO2jIhBn6MhbV7PNoqK3xFQIgHY3HiXdaKWYJp340hXHjiSuw6QB\\\\\\\/Pkc63SiBM7LS5szOy3duCrRVj8Rr\\\\\\\/Hxe14sZx+iXHFhUTS2SuJxQzB1yo4mGR+MPMVh8jPeu3EXwgOeBdP6VOPCmf7RDJsL8pFSQ5N58I73NDBjIFhLh1wFLdAlWJMFxX4PIN6Q7XblvWM3Tzmj0nbTlFizZ\\\\\\\/tGNQpWhAcCCVtoMA5Sl33rHwxYkbK2lBrftqm+ikolrvtEcRFFrsu9qsAiUBEWs3l2A5qhBqPdizhabAF8yRK6Jzg7BqezBFiUaRUVLz491kn9FF7RnJIZTTAq4ZBIuyldfnjYv+81EGKqs0M8y6dRPW\\\\\\\/PH\\\\\\\/dDOkYN7Xf7\\\\\\\/uN4pkr2d4UrICup0vJqRIR8Ha3HqfXm5QDrEAk7qBuXjxWbk1knF7O09EgmFB4aELI64mPuQIzVmQL5ssq7GqCJbYS6scYHM+k5O4\\\\\\\/1hFTcboewF5CFGzuv2mAxM3KV5djOzrPkMLEQeSZKhr8gIsqIif0qG8rVICnDxZihl6\\\\\\\/yXDtQ0nk0i2qw9JqOrbbbhLUo4X0ct0TaYdrgtthQ6vDG\\\\\\\/FpkWdy+dU91il+rcPs2I\\\\\\\/uteWRUoayJ0vwJbTCpsTYlxrPDNzw3PpFF4wgYo1Cez2FT1wuL+Q7dLu1eSAX0cCwZW8KiVZt5J5lAfpVTRF2TU2xjtv4SSXfV38aQ+Ra+06Ngy5I\\\\\\\/vOjS5La9VN\\\\\\\/XB5427UWGH5IKukUlzyp8qLcCEph3TYti20N26Z6jb9QGnz8tEfJY+1cl+Ptvv0ukp838r8MCckFPnPYySnna8pLP39NB\\\\\\\/HKgVCJZuxrtqyOa+8RXC2Yq1ZzpSVYBXn0McmW2Sk7Jw65aBVoC2r3E+c9dpcQI8F\\\\\\\/zvEhy44KHtD2Nf0ai4Nbs7iqxQyYOTLve4xMt\\\\\\\/Fr6ssAr54HMR606ZbsgQX4pxv9YXW1SwfHfCo46tIwJltYdlg0Pr2tNvvqMv7Pwd2kjB6r3MkEYwL7oOExsSoAPnnksqnQQP7roJ+DiGiV2ha\\\\\\\/6Z8QgDN\\\\\\\/9fQLA59JZ66J6LT0zud1r0dvLqhH93dvI8plhjcG5gci396PhFsOdWiTar99CZT3jbuiAI4XxazNHsYx9REkoD3KgsUZjnsKgqgQGpTaK86nMwUBoJHe5JH2phB0Ueg8v5bnbtRfstbVJiGvaZz3MJAg\\\\\\\/tcuLR3LfebPwe\\\\\\\/UO1GzA+E\\\\\\\/vjpSkZOn2Jk3+RamaAhS4pYfhyQQHJWpxpOT3uLUG0gVK9xU9oFRhDR+VErO1uF+lNm+tpOEdtYj+XgUqfpiZjS8nxqwAAiIqU\\\\\\\/4P83GiDrhdHVBJ6iJjH1VCs9\\\\\\\/J9AbotCfpaR2GM7NIkdw1oMKqFQOpFp8AMPUwuuaMPzGBqwPjiIMcRD6Zcd74yBIHtxnRk7tfzL45VqKUoofMxA\\\\\\\/Q3Pfsil5S6De5BkfSlHbg953jxiSjt5m+AoPBvlmQDgbmy2RSv6THWVDNkzVw9Vyzvzqcg4kwHCO2AVQZgZwkrhbu7Oo3Hv6JPC9w3cjpO5ib9lIKhnMkFqwMBR78iTUkvqi+ontQ7uzsefLrC9378iGTDte\\\\\\\/2UKjJP1LKpKB6B4V\\\\\\\/4p3PwBR1ke+HxqaRlqjZqtoandByXhZfKJwglgzHPHXFL6tl7ZhFtPcx6YEzMNiQS08OY7omkg9gvlpfIR28biEH7B04GVHM9z1x7baAtozf2ZnobBTGpbpdX3lwr+OULzogElba2\\\\\\\/WDu3nmxEXcE56\\\\\\\/TrgsNN18DFsaH3abYNfjCQOk8f2KU1WxHVrHlTHmoYWKAMZkLbhWSxQHMfLdqlm9QiHnjli+cUVLW4gVKdpA2Nag82SQGuZo7QR69OI2cG8R5QaNRdR1emEdVCFoeaMAIKMyhdW+pqWU9YNR70j44pbAfSyarOqiXPHoW2X0zcyDbhq6Sk7moG1JTFoToa4XxOBSRFMgRcu7zE7c+AIZNjyaafB7NqIdzcr0N8pbY78PijzqYffAbY9isxGC80xXXPhZ1pPsv6U8HlXnWPOo9A78DHoz+98uamJnHJzYpbJq8MwFm+1nTT2LokFKV6Zu05IqDYI8H4WpQfA8dA9GEM\\\\\\\/sIwwpfIadkNJKkD1a+N\\\\\\\/gwfNAU7+IZz1U9AeROuA266agLqhidaAbrJqYTuTnpPfmU5GYYo8USWUOAU1xyxytBkqzA0NuKENQHMEGhup8642Ue+d1qRtMXBxsqHoCZiwhCda4cEQhfruFXbAVPfasLmNTXYWBEvtC0q6ISCAXclZ8D+drLiF4hw4cxiUWUi+jNeKFRvwVO3+zNEXQKp8RQQK3S8g4n0kveeKq6U\\\\\\\/zhXKZ+gRVauvR+RutPUhlNgOwhyAt8cJTU2BfPGmYxkb6DrDFKpu2n2j1NLpTvINxqR1yu\\\\\\\/h2rqn4j4nC74r+9DIR07AQtc8YqpXoOOAavbokuAv\\\\\\\/5Ud7hlOHisST\\\\\\\/15AEo3qQq5BdTi35y0C0SCYlBBMf96wAsiZOTdrYq29J6osix5zy\\\\\\\/xr5XLuVngV9Onodi9fUiVbCWifmVMvIBU2VSUAJtdxGX6q9rJXS63JCGbPEfojpwt6hFQ2Xw12dPfiYWPTaKMIylbSuLK7Uwws40EiP7SUwLhR790fxOVBeWBbT670DBZo4b+\\\\\\\/GwLrXeOMK\\\\\\\/SmhRny3C5VjTWIGuglySZUkOcGe0GZDNhkwqwx7JkQnKbGf8e9I9s87qDn86qG0c\\\\\\\/IYIKj1hipKhe4y08NKOVqimVWxQg4CtLEAyfqqTiQahq2jY8eCMPYSrlQ924QZ1d\\\\\\\/ooYy\\\\\\\/eEHd9ip5PMduRYB3hmtpf5AB1UKAof0Sx+zecUQhGFIlQRw5RtIlYqpikvdxcQqJXlzW0hpG8wD8nUQuqzXlS\\\\\\\/rZN7CMxLRdRr7862PReP5GH1GZ1QTe7TgHDJMxssKSTfr2ziEI9J7QTmZ8aySwKI0bUKBjNWkJtZVaH+uDEUBsWF6Fj4zGkc4t43sNtI+zKoOV6yu0UZcG5kkkDEnIjTMaytsJFIk1ZbY3JwaeAeepfKVRkiHO\\\\\\\/KYDuPJP0iBEJ5Zx5htYu413ymyKqparQ8075fwTAOrGPDO3WMyrHnZj\\\\\\\/U8BsSQRDSWasfR8RugupQnisEibkt0TR9717sZz9Wgu1Nei\\\\\\\/2DyfNjLkKKg0cAT92wNmBKgZQ9KP1x+MnrccXu2S4ChGME3TgmvBkc8gUJq8BLtgb4oZYFNXjcudnINKyeviAgaJ4wP9eOakDJDRaFVWTXbixahCiQQQ6m01qEA6UhpkBC3Q12NsPP3DA8TBrjcyEtY2Ead0oBLZae0QNBe719VQTxeCvshUv9zGPKBMYkj3AW+b6ucTe2q43AZGAWxYd3gm1DWQc5ehZBndThW\\\\\\\/lVwxa\\\\\\\/YrgxG5wvNzeJVdTlWy2k8CUWRLpRXt1DGBTFPRqah\\\\\\\/4QPMx9gGxNal6AAIwAtgF1ULgEuT8VrwG153t7phUHpbYx1aG8OV+S02ZjCqDhENbqa1iE5bdNDjkUJSN+dqUNn\\\\\\\/7Xpcuy8qJGdVItYg11F9uXXAEMAV6MjILZsVM6LsE+lMRJBWGaMbg23NNTcFs0NOjyJI\\\\\\\/bm6BmB+0GYFqNCib73\\\\\\\/c5K\\\\\\\/xuOgUxg5zzLGF\\\\\\\/JBTLFjgjm5mw4uGQonJfhORepyQs+vNaV2I\\\\\\\/iW+0MJbr40vylNg2gpScYNx5sNaWNKK1YvkceNQ1\\\\\\\/QUXH06c\\\\\\\/uEWiPbX3igKgHYUnyC983ku42ZizmYyzzYo1kRFqPsHhrxz2Kg5ng2QDU4mRQ3EHACEF8\\\\\\\/1kq82w1qgro\\\\\\\/DHu7mOZokXdI+9xL+CHEjw7BAx1G2kHAqhf7znGJ2y6CuI0JpMrHdZFuiUFzfKE45AR641wZ3PoFEkG4TRnjkY6DEidB79ZGycFxHMh3NKBFf5NTiV\\\\\\\/UtHenly5mrvc16sQRTbgMTvpqokE0r6\\\\\\\/gNjPXlWQFkNcFCSGhXx1Ede0mJ0Bya4JGnQ4ok3zRLjJKSHS\\\\\\\/H\\\\\\\/zr+PrjBPWMCcf7A5nFTEYO+\\\\\\\/00lsxcNzK6fmrP7r1tlU+M1Lj\\\\\\\/UcHM5lI1kBppIjwN8juLfVKVj6j9Oz2UuFn2BJgE2IXBKCrSf3\\\\\\\/T8CjAIxWK3eTbJxfYFr6q+fxFavW849nybTG47sduY2ATLPcxNi44yIBA\\\\\\\/yVCK+sZbwKu2eNh6yGRo62sQ16hiSiVIY9+jYDbGFCHyK1FJxGsvjracXGjI2Zr6KedOBeMeEkPILcoxGkG19JhelDjrfeDaEVBzLspYSV\\\\\\\/RAdcpNMkSunY5kILkU9aNIwClV4T\\\\\\\/MZ\\\\\\\/mdIyjlA6BmyXD1M7SIQKqEP0bN6wZYHfDk\\\\\\\/LBuDGlMlE85ldgX6jzkNUhFwfnlaR14w0ZJvq5z7IJSGuBH7IsINxzi\\\\\\\/0ZP1R\\\\\\\/PZ601gtEWO9dOxkRwWPOId61HbSDqMDPf1RYwKBBNh6DF2c7uP14\\\\\\\/kItuz5fMorG\\\\\\\/ZDJckjH4XhkMUUkQJbxsUxYsmP6OL8EuAY8khhCJjkqQf3cEYIzGVUwSJfpo9eF6gOqfu6rFMyewbifUhbCYdMXfNnULp14Geb6k9kR1hvpBYgg+xp8KauQABKTAzeMInmxd\\\\\\\/92Kb6mdV8Og1uKXe1jtNkVeonqvX2yDHAP+6gE5WqbVEpHAOyt5AC2JFGZ+KTkmcDZAXXb2xMSAr0i5\\\\\\\/d9pI19j+iFgsV6Bnah5FBoNEeaUnrsBbfa9agEg8mfoMb37xfpurzIyvUzwnHjLWNHmI8vxkgw2gSSgn4pXi34Wd7+D8OY1GSyNIjCampqUd0Dzz4A5xOYiXm4\\\\\\\/ECMOUy\\\\\\\/4a8BCra9GIaiSBlHNfzxCGpCtRsKEZ+fFAjVl7BzeAQb3jT\\\\\\\/CBvC0c4NdwCdcZhma8HmoHtde7Z6V2M93n3yVEOvkNHfelkhWczAX6hBlwBmpUVakcz91OwhBCuqNnVoXZXlpoGdWlO6+r7FCkQ5JnzYpiyLskWNtuKrezdNjesumSsM668w4SQqOZy\\\\\\\/AHm3GKnfpxDfWTKtJB461EPoZFL6vAF4uJea1AsrBBPUvIPL6bAbJ3wsz9bk7uRmyNqAEQasXj\\\\\\\/2lvvfsehYoueYJwNW4b+XZVS0dR9SHP4RZ31ggBkhZMvjg7RuzgMIFPEaCzCHcL7h00yJcrd0TkTrAu69Eh5lWX+eMbKeLnHYD6dxU+kegpUucDMjZWgwAf+aOtisU82Qgz7OrPdO0R2S1LW9VxKxmlrziuuasdiTLYKUVCbfL8OEUdWPSIF+chz13fwGQIAzLE5KheZ5k6NnKvOz6Ko3VvtMN9IfQZKV+GgVoL9Wy4v\\\\\\\/jGKiWxiKtVhNRQ3q8enRYwhHuPYhFec5WOIJ\\\\\\\/9b7xHqMNWMiNPxC95UpNsSVZBqNGrVlnEBSZ5qMzVxFk714b4EzocX19sVRMYHnqb\\\\\\\/WpWTjE+wwF7tA9XyhW1pcxr7UJxkF0oEdrMRrpfqLj2hRfC1x3SYoXhMnzOhR8Z1Bzgt+948ketTIUVR\\\\\\\/HBRNDkurXeyTB7P7IBWLXWrllyNRvnm1A7\\\\\\\/sMEe4Awx5iSoGyXgok5J\\\\\\\/ddCaQeboR+Ddo0cHe0OJSNmDoktJaFmjIMF0lKHy0z9NQcihlxMUf1bVir59ApjJF4IZl1gFlHrUYqZJmzavGTm\\\\\\\/xnJX5+BZNC8ccjHdobOdwLpld3q\\\\\\\/nQ1gMxQj2MHBnpHuOhlXerAtKEZh+eJCX3W01tYQ2TmoCusEJFSdcPi7Yn\\\\\\\/7dT4ExfgPIrqHiAWISLE0uNx2chvTIlgBOMeMgYbpbbnU3sUed6wpqTIdzXxMkRUgjwzG\\\\\\\/hnCDq\\\\\\\/LcyymNntw+SGFY5NSkrYcdMDh+Bn2nU\\\\\\\/saX8FZeMFuvJB7VNTLgOMO4lejD5oAmt1ZszP4Qg62t+ljW7+XzAFj+xBbTwGUU8F1czMghyf4QCihBkoav6LHL6U\\\\\\\/a7XklV+iCFFmccnSi60l9lc3zowMnUsXrjPNTjy82G3+pUQKBUwMVgINU7MKpUE03kCYlY3P0+9dm7g13xGjicqpNPPxvsSEXBEZIMxsRssnhvOqtBy4Q5x3sFo2m3iTlxRPb0RgVgE9Iuz5rdk8KEnp81wAxPVScQIL4uQLh1zIi+EPEUH+wmDnZ8YsOwNzZSLww\\\\\\\/shf3XrU2BhMcGfuzcjQ9KrJb1e1M795cWBfT2SnVstm3q\\\\\\\/44lkQPNLoDkVEJLf8CVMkV\\\\\\\/KYg\\\\\\\/+Nr2p2Zf9neAGRJPSv1maRUk8zZAzNn1KwbWAvPa4PESFgMHqzhfgyJip0LBELGBGCuRs5nbp5snbh6nzOXtFR\\\\\\\/SGrrMPAbczmmMFNPT5XMwj\\\\\\\/2B+qqQd1+zAEp\\\\\\\/GXEAbddSNY2srTIo+9g60A\\\\\\\/K2J6j0jkuvs37Kr\\\\\\\/Ht01pOcdm3pOlRaCs3wnG1OU8gcyyGQVBiHhXk00mliDdL0Y8BhVDBPGGOhqVWHwVFG1rGibXkDrT493qbXqQN7u6QgzLJC2\\\\\\\/C2sgKbyTZpQF0dO6NrwQLPdoPxllvRt\\\\\\\/ph0D9ZBowpYncZrnzzWE\\\\\\\/cfx+mq6oZnyvvZqBjjUNExySfLwmICtWycv2Rms9fi1FZZcqsS3CuEfq8VDhbCQ3Y6InVuBOscHVM0vTm4ExJF7YUGwPTLunKoqXsrW+RMrNmkcdyl3nfCKTq+OU2RLJCaF97E+hP94ElP9q+tn97pDN4JTSyA7Z43fphLJkrGt5XJFvAeRVagba5ctNGY4c8oG5HhOJG1BWLTge0fgswxs4fDVOREqi7e+dCQenFNV4PKqsm9NrwVgS12kv1nH6sWm3TWCMCyUww73vmPZOlrgTMR+583N1w2odon5CzQt7tazqdT3dycFLb+f0KF+G5tBba8j4\\\\\\\/RiZ1EDv8a1X9K62ht\\\\\\\/SSZ8vSyO\\\\\\\/owNHEadyCFGV8hgr13uf+Lr3ZlB4fiw9RbMOAc5spbY1aDSY19zTDx1OnnNmbYEVLKOfMQcMH1+tky9KWX37KygqlxBpxxM8l2PTA3OVbdIMynhq7dpDi8ndc6zReM0t9TivLqVffV9oM9dMB53To+33BDy\\\\\\\/BXiJMxqJjweuC+JRLbpw8O9SZARsu8p1\\\\\\\/zp0AKoD3Vcc3xHvWru0ZBwZifKw2NX9T\\\\\\\/XYlBuaycIlhztk0pHsm7RwSbncUDGMrFMT15UeNKQXOCXHpFftKyQqnJbjgkDXZu3rV6F9KNztqegUuqq1iEC\\\\\\\/PGRtf3zL46HQhDT4QtyCE4StW3C8P4Nfxtgdln9NJB9PD3EOieIbfdvuM+++qHcl8TifWi5k31GcnTKRBL3vWNUf6\\\\\\\/w62ANhfo27vmJ0vClFLVBjyT1Zdvo05ybS5j8cwwJvn4t8\\\\\\\/5PhOOEA9Va6jETNBBKIiOglU9\\\\\\\/czaryzuhKKE8ntAexlTRrIThJ6Wgvgat6hwjkWcKWzz4g3uF8gSkKmgy7g419wqZI9q+Zq3UW1Fqg2r2rEfLBLbCBbJGNE\\\\\\\/lypwISgaBxWnghPjxnBUfH40ROU+56sRdVWiJ7WlRrVafilbyTJkV\\\\\\\/W\\\\\\\/Ibi0gy\\\\\\\/6jFXPQM+Y5FGP\\\\\\\/JJYN6G3g8yQgOvcouZTiqw+wu5YIzwTIxEBY4O6ezq4hs0J49FzDW1pce6oYQY9ttsfjmqADB94cElk2LRjp+1zJXBt9h3jv50JXlPRe67tooZDoMGKUn7osqNOOMoAPkFJUm26VGZoLzsJ6xAEXZzc2LtbEdmTJpMSpy184pGONCW0nPjP7VUrWhqQEgC2DlZ3wEWswiEKGgC6gL9Y8istuX3NSLEBNBxiraP\\\\\\\/tdE+mszHpgom88UTiWWS\\\\\\\/nsoNPpR7AGFLF9H0jnj\\\\\\\/xHEFSveQyzQ29EdUYo4XzGDhY+JKfPL1rIqf+\\\\\\\/LwrV8\\\\\\\/yL+iM+YltFgIIH29gL2MNdY5pZC7agveB6AImUl\\\\\\\/2Gj4EbuolkAZINywLiHDA6ovaxXUIyVO3FsZEF5Q59cB3hsWv\\\\\\\/zVDV9v79ks78xisJpgDOKSmffs8I+FF\\\\\\\/Zacdw\\\\\\\/zYxjo\\\\\\\/xDMOIfdVR6wQEKDnSmwNkiX1TjSY5m+hPhvGDPgXumdZAJmAzdFhnIPyDdSZ6RUWbGvlqFobFmNwninAs3+i\\\\\\\/Qes7LF6e4rxyoRaE\\\\\\\/PjYxWnyokhAb3A++W6m6CTK9AVdmHEjb3kKDEEw9COTOZRsfEOq7b7UjehbYHBd9v1qsA6wXubXxUN7o3y1Hl56VxGQcJHX4AqKE\\\\\\\/UzShYeWjxTf\\\\\\\/klhhf77cMYUDZhkxvNPcXFxsD8cNwzLFglbQ+x+lj3VBhE+XQl8WEL1au9DCcppDZnOg1RYRJNbe3eGaEWLV0vi8oL82UN55RFYbrVYpw8zL2j3VI+2mawUgItF864eujYmTmDhNGpNwOG5wkpJD6NR9m+ZpXCg1yYri3s4uLporvJw6fUWPPPri6kJhMvHL2YWR8761KzNEv3NrJYrixJOcUHxdi9Vn5qLXT9QpY2r2yNdxTae3fC5xd9N2BRt62Hn5St7x6g0aKhPftoEKDNEwaL5wQZBNY1g7t\\\\\\\/34Njqh1\\\\\\\/phqMYyLqJKxSm4l1fbzCnQxV+W1Y0uIfEQ4Lx6FVEOBg6QfvoaYp+sWGpwtGQiVD6ZEfHntImsgLXjDLigSZiKworOYA1yVnIEORWzFdydin7n5k0paieoVnGnbRaa8F\\\\\\\/KWR58pYXpyCAAGXpd0sr0swvpRq6SAgPP\\\\\\\/54oP3ZF45+2EFXx3J+D95ZbytLjMUHIHCiCZW12Iu\\\\\\\/6gXCBLjMwYNpL+z+Oscf9FDid+7uYyQ81JqeZSREIIC10z0+z1XjpKVZS\\\\\\\/nqc01UN\\\\\\\/BkUSEiNTzaKLF4DqHYyA3aaGeNo6dd985pHQbAi\\\\\\\/0mLd+24RVl1v3ot6+3UmFBudo0QGTqXeKjfZSuD3cCVe+fMsI+05cwhFZB5R4qXrU0jkPtSwDyuBSW4Xh9yIv5l+P3W6nXrkPUcI2lgTViUbksS2qabJ7muY71znwJHg0pkOM52eJJvGLqST8Ou0gqq5gWmqMnrtSO49cUcFznOmCyMQwot8RHMUcg59oSNr\\\\\\\/FWRxq\\\\\\\/\\\\\\\/KcAkX99RX1IUQS+2UOiaPIZuN7vqL1i9lwmv+tTkspPT2AiS4Pwc89m8XKLkvUvIR5FKiDyYb\\\\\\\/MSAZ7Wz9tcj\\\\\\\/KRpcXoySa6HpKNe0eTKJOkHb7U9D2byl24zMN+GDf82peVSw\\\\\\\/nH5oXWPH3L9c4RnYb3+Y5fCfWqc\\\\\\\/60y+RjjYoBNeBoM7u5fL6S71x8MCg+IIKdTKzbw0v8x3adxC+S1j2LPHkipwARiK4AIXsWjwpIgkkRjxyHNAqhRS5OzJD89kU3rH98WZAk1D\\\\\\\/7IpUmOzOJHsH7O\\\\\\\/y2IDKtaGufa7yb06tPHSjwDXEnrhbEJSTskAN0FxDJREtBOLuHWUJ4zjV4L+PTV8gGn4jTzZ3fX5QBXN1F28LkUTFHvB8wvvLApXZGpJ1nev3IVIGLpmNUYK60XaVdHd2OqAOc8rL\\\\\\\/HTnMnAb4hQzjlctkzOPz+6Q5js33m+vi3VK+o3mKAx6fdCfFbliM6E0sFBvz8MvMXbjEB37+vB\\\\\\\/oyGfCZ4QVXGfHsnFjcSS\\\\\\\/NCyDOj7Z+Vyu73qbXqNJP7D\\\\\\\/s3GX64D1KzCdXrQTeOuJzrXaWssbsCRLm0Zqwf+HpPDkrd9vWw71uRYiOhGCSKjkvWHHcAeBv\\\\\\\/JdcynP8M9Dpr8ScHsWlqbf+kcubhtWZFRhFaRczHaCUnylyg6S0e5cCQv2bXvUg5U0rZGNY\\\\\\\/e+oaHqH5xMz\\\\\\\/DIL0MCsk94jyE50uynepOkW0OhHmFK+sB1wLbwZkzD1bNHwh9uCgkm40z6kb7dptziEC6DonK8ZlzgvXNtg2ETNLfr878GDSIkwR15VdDx18Gd\\\\\\\/AobGODrO+h\\\\\\\/ZWpae0A6WCQNVWclz5dmocZAYJch4Qj13sI1kPkqYqhHEh+pGg02OKBdJxJ\\\\\\\/Q9oD4L7NvKXTqOD2aiHyhIQB2a+h0MjP\\\\\\\/XRNcxV7qzwk7YdmEyGzJX2vtsqNm9WcMInDce+QUddt0OZYST5ufGFmemGIAJdndKyzftSFfNrQVrDsywKNUcVrCxYB7fkyvTfnK5+YXTSipg+5JoKEWeu66XuRUFxotXijSxi6tIrjXJW0YL4KZmwfYEly9u9ZMBfaoiWBurJR08ihCmxvZb6wRrAylWJhauxF2VEhLC6g4cFGqOs7S23XiSAH96MjLqCxd0m\\\\\\\/5z3wDcw8VgujkMhbfQqeAad4NtgKAqmU6MmuzMWhLYIfjnStqCbOpNhQ+n3Fi+4VFurx6AkSNnVac00YJBPpHYqgo3DG6Z3db\\\\\\\/pAaVXmKSmjOUjdcGEUgF2Mc8U955BelVLi8N4U5F+xMd74bFVRbBvG\\\\\\\/gai8hg0Iepqhsa2S8+ngD1Ez4rPTIp\\\\\\\/Bzlp7UTrZZh0R\\\\\\\/9ahNRCcIxv5kbxm6tmlDLregsd+88K9vAUc7gaQE3c7FA8xUyDtPyMSiPGT6j3l5OhZGzaNAQmxPi9BO2bk0X9Me8z7YS7O5hd8VMCtVS9T1Lad9oJF6Nrj6nXYFkAbU4934DghOJY9h4zbo01ivH\\\\\\\/ukm5UNiFrRVx3lLvzkQIQi2vRwvJePx9krc3dZx8DyysjPWrL97mzb0jeI66pA2ka6YaKjzmWKnJ0GJrBY5OAG6nid9bVP\\\\\\\/0IUndTXtHyZJi0XOWads+vdWnsvmlL0tiPfpUVCYKBg3vcJy83JjfCXz2FcyLP0VeRi95qZ9T+xxu6kZxohkYJus4fiEhBhyp16\\\\\\\/fMo74qcmYouHG\\\\\\\/A4yqWAOg8cmpZNwd6+sjSHkV6RD5MIf8WpnpemfalYFQBmFKQDHXesVnxYNqVF93kdXOxG\\\\\\\/li1PaMBxw4gMTpgn9dNPPaqDn\\\\\\\/ORVhSPPutHVLTUo6PgeKSZ4N3GypszwYi0iapWnxO3ZdsVIw\\\\\\\/xcKHkKxwgu+IYZuk7M4HgMBCx++KkntznM2O3hyjAzwb08g8pOP+dKSyKgtJlJPOLz7cW\\\\\\\/Cfu4G2IXaGvaOhn2i79dOpiECWfl\\\\\\\/oOfSz+tigyTl5Oz88xdNBFeqynXh3JEZsUGvXAqLKr0okw489E3LY6AkRA22Dpl4f+l9FcFxGlBGB3ISrTgl40bG4bR3NaUrP+zva0\\\\\\\/mOeKFIVBu+A57VRMscB65qsPmehJdRifQyOYSEdNHx6s1pt9wRdfsXe\\\\\\\/k0F\\\\\\\/XUBDbnz1zSsskjWhqgnnJKE79tBg0W3R1rFKZBOio2kzrMf6PwubNtjPUoY30m9BGpnujwOeZE+nkUsvizN4bLB1azjYu\\\\\\\/yfazaBfuu+3NKwxOH\\\\\\\/9u3xIj5dqsvXOPIHZ+qmvjgYpb6uqcMBFacRGST0agk4UHCAnxbMTMSegpnkI2Oggxy5zhaZxJGm+1NDkSG5lhfGDViHGO97KtjWND073X0j33Qw0znS9pJuwIWZAcPJY0BxYejmXj\\\\\\\/waHk0MqRhQUA8w8BhZoCHzyHxJnVq5eL6Vy+KSr22aeWj947fi+wdDLhlRJxpuTxuT9vyY42yaPMFgVJbe\\\\\\\/K70Gd+CYkXgc4aa0Ti6LyJ8No2DVMVTy8ahjeBPFE\\\\\\\/neX2Z4+l85SJd39DSUsDVlBdxwX\\\\\\\/XiXMKHr4hjQv0MG2+x5CP2Qc8sOSNEL\\\\\\\/kZ8NqjutIfDy9jrSWBm4pGTOVmWCin+oS83Mr3sGdUyFfXQGcMu53DUGkM\\\\\\\/ZiFy4hFaEyNns+P5PEySQug9Qpsz5ibtMqRW6YKh99IeYcyG83Nnce5l7ptB69v1ekMQK6Z0adQ3vVha3\\\\\\\/tlQnnkZTuZtnibd20pWuzRmJhjNi0PEHb2aALdx\\\\\\\/D+t\\\\\\\/462FyPhMxQAVQLpvEzDXBBZAkH8t2CK57JV+R9GDTNi+Hf8g24U+Bk3z5EgG7TU0LSgcnmP2M9UtfR1KImFZGTmoUp8tD2P1OAQLuZugR6IjMuvUSsTKCRI9\\\\\\\/v4JXAtoaipCm7TsrGoobfPTlMO7AOv6PufJIy6uSzNxRkUPAObqc53ngISrA1HhK6m5iIXMPiDAZ6Rjt841UiPrqACpEpA6IJdcvrNGSqAO8GhvZAxBIeE54ujfpRM3lwohRApbW\\\\\\\/Gsd\\\\\\\/Mk\\\\\\\/48iW2j3DbywVNf7LoVDRZI3f7BNDTTejKP9J84UwltF13d3RvVX8WgC6fN1ubW0SkIscixFDAcUjN7gR6miDL9Z3StnDEW127N4Ie2cO4cTbKYy1YYx\\\\\\\/Uo+ufyrT9u8KSrdCE+MglrP9nSEb6O2X2DKS2OwA02xYr8P\\\\\\\/H045nGeevWv27ns+j77CK5FAN9wRyAhsmQm1dcQdx8olhqctss2sbRJ+iN2IECbnPFadliQIy75HaE7tskGtccT4NToJnH1ocDEDU26abZclFIj+Z88BlysSm3jTw96vwcj\\\\\\\/\\\\\\\/0x8i6t4P3mAJqBy8dGDINNVqRS94Jdw+qbyoMfdEmaBY0CraA29l8IZQT+Lg\\\\\\\/wZbb5ANwrtQ+DCmQPSIxMyefIgHm8edqggY7wZ5cnb2\\\\\\\/qKQy5GjasnZWKph8cfgmQwLXr\\\\\\\/zSas4TkpBfysDTGhD0GQLSVLls3WikrjyxqbXS3Prm2MJAAX\\\\\\\/FLHdTCZlWyZZ+DC4DxBY2YCp5ogLMgcGhGVeJySFxFTTz7v4D8AtyysESrQAumr3Fc8orSU+Pc91tpVTgvCVC8Krlwi0YBgp1hbx\\\\\\\/tI4wE2kfXWDBmXCB\\\\\\\/MUmD+RfaXESQBjsBxSjHRmbMWgeCrfOjfzd+\\\\\\\/BFP4OURi2jvUfe257UkY+M3DPcCKayDii0qRtTYv75R0t1SbjBWOVLOyEgH05E115q\\\\\\\/Hk7SLEN0j\\\\\\\/mwkKC5vUMsequheZ+D6BH5XkOlcvNnvAK9T9+Mha6SVP2IONgmuU\\\\\\\/pR6GXN5ZyCWjFzmM0Y+3pdDVptlILSQG6S3LDOp0mwlPQJvpB68P+mGcazc9sVPS9SWOIbXwUPlOIT+IXa\\\\\\\/tr4E2sbQtAw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022MryiV\\\\\\\/lPqvpZDIUNVfA1RQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022371514540ae0f71790df01a5d7a80ec58e5b8e104f0fd00a87010634d7182c0f\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022spfqoQ4Omwo\\\\\\\/HpwLfIzC48V1op8wmxXqiDtyCEIayblYZF65WRKMEkmfzjPSDQiQtQDy+6g5CjQ7Z\\\\\\\/ILVuJKYZO\\\\\\\/99xgKlg58Nbms4zMDP4zhm1jvV\\\\\\\/jVqMMWRp8\\\\\\\/Rjzg43AH\\\\\\\/LIblWengK\\\\\\\/T5bNouvb48\\\\\\\/Bzxz8UBkDitrq+YoAogxDcaQTHZgBEnyk\\\\\\\/1wFesfnvIMYVpFU9qphVzCipfRUL13SNGWxDmRg3Lp0I3s7elJoTl4Yu7FRY5KHNTTneSGRzx+sSTBc6RJHxY54zS7tm+C3879Aotr+SKcPHoe91a4qefdS4Fx2D3dW6Q8TxaSLj+RG8fIax09g4fvvJ1TcWqtI0wf58aW89Wf2jvuKvwxtj58BE7Q12\\\\\\\/5MBCE6baqz4Vi5ROUY7ADFxDLORodLRTpdd4s+PxS1Q09yRBapd5Oy04TquQXTiOpF94IeAPORAIIPm3x9bxjmF8J\\\\\\\/upOLz0msSALZkbnDmrtYX0FPtfwi8VXBVUMRXIMiyZ0I6dLBfY7Mh16qaIshlI39lA2gaXkSrnkZ1q+AhcpxbqPWM9yoyDvYV4qhiMeEKiIygqepuSBZVP7fnOuRBpgKuR6mwjGakUHThT6bplAfJnTWTnMs8Yu4IqsD13HfbZN8EFwe+uAMGao7rlps3yOX\\\\\\\/cki+OP\\\\\\\/ITwDSV8xBpw6i0lkyZN7s+S85FoT6qYg2MbcuBvNkiLd9vbPl\\\\\\\/ZfdZ4CAfNiLbUo3Q8G80uDcyGRdgjvS5Xa3e6Lr5v2niS\\\\\\\/D+b\\\\\\\/vAib4HxYnLC0f6HAygDdAHCJ9ENdrOQsvW9KLV5TYVTO\\\\\\\/ivXOlEEzDui+lV0I66gKQ9rVa2\\\\\\\/Y6JPixM6IyK1+Gh2wIUCnCi1nWoH3ysJKLIG9r+1UUr6GaGKjU+kQ3BD1SOkO4tOFT+uLfTivf81keHntrAAEWwbz55hAZgl2CPOwbBdOidLf5SpB3p2Ba7oA+B5b9mZXeSxrzSv+U6zPgmERJSovxS1CDJ8ziyMVmXQ5nlp+ll8r5K\\\\\\\/+NA0zy+Sn7BrY8SSTB13bP7t9PLLON2qZnvJm1PXLL39HKZs5PCjmLlY2rPPv67r9T3F4FW4YJiyl1528s4YI6YhaZHm1gRHxtIAP3s\\\\\\\/VZU0JTQH278jXim9iisS+hXIUg\\\\\\\/a2uXyND39sc\\\\\\\/+VoqMm9NgZjNxcEpJElaVrZizu7kbiczXgGD+gpe3XaJ6A7B8eeAAQ5p8bxjFEFUfpxxbNylNnfFep38ta7UCGbXyBoUCNgyNw9LC8xAXuK\\\\\\\/PkTzN8Bt5Hvr12uvsINCP8meXg\\\\\\\/hjtN0FUQlvbLYZWq6oiPQEbxFJDoypS2NfGjdHy0d8N78k4ttOFCb9HPNTRXN2kq3DVkPTb+Zkk1Jt8lPjuwoaaDFjoBC5byJmrRdaRdWwfKhifktntTYjY5B9TL7kDsALyO+4HM7qDxdEiRKbjUdBgWI8lbdjYBFAc0jZbGWJpO+7rF4kdyxUTNfNfRwoYGVe0Sq5iCIN4\\\\\\\/CDuAM2cBk00NWVhsmg\\\\\\\/py6vvQx5tCurxOlTGM1+38gHyB\\\\\\\/mp4ap34bfyn\\\\\\\/zDm+Uk9eHGzPu0\\\\\\\/nyLHFT9FGcIwkDLEIIE9eeltQEfy1hKPKCDZGMkadXzc\\\\\\\/kXpLAVfFEp8laa2nX55rG7DG8bTjKPZIB3ZkR1z9RAilCBcuryHygbnUF6UmVFZJHsddlw4Bq\\\\\\\/hkGISPAv5Zk9+CD7Zx9ycD\\\\\\\/1miQ\\\\\\\/zYySU8mzH64QiqDs2Mh6iH5RK8C\\\\\\\/GHPDNE6ms5tvAc1eGkcn6pXfY5gTKyizVEnAn\\\\\\\/Wl8CIEHMrD5UoQY8SKTKmxy7LPmkNCNsGzqTBNgks8YCmBX1SlioHxpYrV5nhhwp5NaNqC0Ndq1uu5QmgnXT8daQds44Qw8wSacCtx139QcxHGT+SROHQnh1D6UFMH3pzmaZd3dxu8Dhh+4kpr7XFurkpSzlNPUC3fKAxyWIVG9x8pXbagpAnms0VL5ORo9pGVucqbwalgJ2MhZgpRZnYTTO1wQwuA==\\\u0022,\\\u0022iv\\\u0022:\\\u00220Sms\\\\\\\/BBes2UfYQaBGW0yYQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022253dfaa8ba369634695fc65be446db35449cb1db49325c60e22c613fff482ab2\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495540 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/677" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-GmegY18LR906k0pAsmgKptRgUdyYBreYOzYZZe4CU-E" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/677" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tpHel7AFqpnls2qu9-mTluAr0iHh-oK59XfpL5icEo8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/2/665" title="Quantitative Genomic Dissection of Soybean Yield Components" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/2/695" title="Genomic Predictive Ability for Foliar Nutritive Traits in Perennial Ryegrass" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495540" id="node495540" data-pisa="ggg;10/2/677" data-pisa-master="ggg;g3.119.400970" data-apath="/ggg/10/2/677.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Contingent Convergence: The Ability To Detect Convergent Genomic Evolution Is Dependent on Population Size and Migration</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-8936-4991" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>James R. Whiting</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5905-9056" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bonnie A. Fraser</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">677-693;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400970 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495540" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">James R. Whiting </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biosciences, University of Exeter, Geoffrey Pope Building, Exeter, EX4 4QD</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BR.%2BWhiting%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whiting%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJames%2BR.%2BWhiting%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8936-4991" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for James R. Whiting</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.whiting2@exeter.ac.uk" class="" data-icon-position="" data-hide-link-title="0">j.whiting2@exeter.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bonnie A. Fraser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biosciences, University of Exeter, Geoffrey Pope Building, Exeter, EX4 4QD</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bonnie%2BA.%2BFraser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fraser%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABonnie%2BA.%2BFraser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5905-9056" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bonnie A. Fraser</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/2/677" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495540" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495540/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/2/677.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495540" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495540/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/2/677.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495540" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495540/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/2/677.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495540" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495540/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Outlier scans, in which the genome is scanned for signatures of selection, have become a prominent tool in studies of local adaptation, and more recently studies of genetic convergence in natural populations. However, such methods have the potential to be confounded by features of demographic history, such as population size and migration, which are considerably varied across natural populations. In this study, we use forward-simulations to investigate and illustrate how several measures of genetic differentiation commonly used in outlier scans (F<sub>ST</sub>, D<sub>XY</sub> and Δπ) are influenced by demographic variation across multiple sampling generations. In a factorial design with 16 treatments, we manipulate the presence/absence of founding bottlenecks (N of founding individuals), prolonged bottlenecks (proportional size of diverging population) and migration rate between two populations with ancestral and diverged phenotypic optima. Our results illustrate known constraints of individual measures associated with reduced population size and a lack of migration; but notably we demonstrate how relationships between measures are similarly dependent on these features of demography. We find that false-positive signals of convergent evolution (the same simulated outliers detected in independent treatments) are attainable as a product of similar population size and migration treatments (particularly for D<sub>XY</sub>), and that outliers across different measures (for <em>e.g.</em>, F<sub>ST</sub> and D<sub>XY</sub>) can occur with little influence of selection. Taken together, we show how underappreciated, yet quantifiable measures of demographic history can influence commonly employed methods for detecting selection.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/outlier-scans" class="hw-term hw-article-keyword hw-article-keyword-outlier-scans" rel="nofollow">Outlier scans</a></li><li class="kwd"><a href="/keyword/demographic-history" class="hw-term hw-article-keyword hw-article-keyword-demographic-history" rel="nofollow">demographic history</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/fst" class="hw-term hw-article-keyword hw-article-keyword-fst" rel="nofollow">F<sub xmlns:default="http://www.w3.org/1999/xhtml">ST</sub></a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/dxy" class="hw-term hw-article-keyword hw-article-keyword-dxy" rel="nofollow">D<sub xmlns:default="http://www.w3.org/1999/xhtml">XY</sub></a></li><li class="kwd"><a href="/keyword/false-positives" class="hw-term hw-article-keyword hw-article-keyword-false-positives" rel="nofollow">false-positives</a></li><li class="kwd"><a href="/keyword/convergence" class="hw-term hw-article-keyword hw-article-keyword-convergence" rel="nofollow">convergence</a></li></ul><p id="p-3">Studies assessing adaptation and evolution across the genome are increasing in popularity with the availability of modern sequencing technologies. These studies often center around quantifying patterns of variation in genome-wide SNPs, which can be used to highlight regions or genes having experienced selection relative to the neutral backdrop of the rest of the genome. These analyses, which we refer to here as outlier scans, have become a common tool in population genetics and have been useful across diverse, natural systems in identifying candidate genes associated with the evolution of a range of adaptive traits (reviewed recently by (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ahrens <em>et al.</em> 2018</a>)). More recently, the method of overlapping outlier scans across independent lineages has been employed to test whether the same regions are involved in independent adaptation events (<em>i.e.</em>, genetic convergent evolution) (reviewed by (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Fraser and Whiting 2019</a>)). This study seeks to investigate how different outlier scan methods are influenced by demographic variation in natural populations, and how this may lead to overlapping false-positives.</p><p id="p-4">Recent discussions have highlighted the propensity of outlier scans to yield false-positives, given that outliers caused by heterogeneous genomic landscapes are commonplace irrespective of selection (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Ellegren and Wolf 2017</a>). For example, background selection (BGS), whereby linkage between neutral and deleterious variants reduces local diversity (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Charlesworth <em>et al.</em> 1993</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bank <em>et al.</em> 2014</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Burri 2017</a>), has been invoked to offer alternative explanations for patterns at first attributed to directional selection (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cruickshank and Hahn 2014</a>). Furthermore, the influence of neutral processes such as genetic drift (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Charlesworth 2009</a>) can similarly produce elevated genetic divergence and false-positive signatures of selection.</p><p id="p-5">The strength of these processes is dependent on demography. For example, the influence of neutral processes and BGS should be more pronounced in smaller populations (low N<sub>e</sub>) (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Charlesworth <em>et al.</em> 1995</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Charlesworth 2009</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Yeaman and Otto 2011</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Cutter and Payseur 2013</a>); as has been demonstrated in humans (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Torres <em>et al.</em> 2018</a>) and <em>Drosophila</em> (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Charlesworth 1996</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Sella <em>et al.</em> 2009</a>). This relationship, however, may not be simply linear, with additional simulation evidence suggesting the effects of BGS are strongest at intermediate-low N<sub>e</sub>, and weaker at very low or high N<sub>e</sub> (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Zeng 2013</a>).</p><p id="p-6">Lack of connectivity among populations may also elevate measures of genetic differentiation, if a large global population is composed of smaller, isolated, sub-divided populations that each experience the effects of reduced N<sub>e</sub> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Charlesworth <em>et al.</em> 1997</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hoban <em>et al.</em> 2016</a>). In an attempt to mitigate the likelihood of false-positives, some have advocated using multiple measures of population differentiation and divergence to identify regions of the genome likely to be under selection (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Charlesworth 1998</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Cruickshank and Hahn 2014</a>). However, how these measures are correlated with each other, and with selection under different demographic scenarios, has not been explored.</p><p id="p-7">A diverse array of measures of genetic differentiation and divergence have been employed for outlier scans, but here we focus on two of the most common measures of relative differentiation (F<sub>ST</sub> and changes in nucleotide diversity [Δπ]) and a measure of absolute divergence (D<sub>XY</sub>). The fixation-index F<sub>ST</sub> (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Weir and Cockerham 1984</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Hudson <em>et al.</em> 1992</a>), which measures the relative amount of within- and between-population variance. This measure is therefore maximized when genomic regions exhibit the lowest within- and highest between-population variance. F<sub>ST</sub> outliers at the right-tail of the distribution are considered candidates for adaptation because they reflect regions with large differences in allele frequency or high substitution rate (large between-population variance) and/or low nucleotide variation (low within-population variance) relative to the rest of the genome. Changes in nucleotide diversity (Δπ) are another indicator of adaptation, as selection on a beneficial allele limits variation within a population resulting in selective sweeps (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Smith and Haigh 1974</a>). Comparisons of the ratio of π between diverging populations reveal regions under selection, as local π is reduced in one population in comparison to the other. While similar to F<sub>ST</sub>, Δπ does not discriminate between which copy of a polymorphism is fixed, such that a substitution between populations is equivalent to a common non-polymorphic site. Δπ outliers therefore represent regions of the genome with reduced π in either population relative to the rest of the genome. As a measure of absolute genetic divergence, D<sub>XY</sub> (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Nei 1987</a>) does not consider the relative frequencies of polymorphisms within populations (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Charlesworth 1998</a>; <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Cruickshank and Hahn 2014</a>). D<sub>XY</sub> can be quantified as the average number of pairwise differences between sequence comparisons between two populations. This measure is therefore influenced by ancestral π and the substitution rate, so D<sub>XY</sub> outliers highlight regions that are highly variable ancestrally, or in either population (large π), or exhibit many substitutions and thus increased sequence divergence.</p><p id="p-8">Because each measure of differentiation/divergence (hereafter referred to collectively as measures of divergence) quantifies genetic variation between populations in a slightly different way, each has a unique relationship with demography. While we can predict how individual measures are influenced by demography, and subsequently neutral processes, we know little about how different relationships between measures of divergence and demography affect their combined usage. We expect then that the utility of using multiple measures of divergence to detect selection may vary with demography.</p><p id="p-9">This complex interplay between divergence measures and demography may be further exacerbated in studies that compare scans from multiple populations to identify convergent genomic evolution. Here, researchers use measures of population divergence across independent pairs of evolutionary replicates with outlier loci compared across results. This strategy has been employed extensively across diverse taxa, including: birds (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cooper and Uy 2017</a>), fish (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hohenlohe <em>et al.</em> 2010</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Jones <em>et al.</em> 2012a</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Fraser <em>et al.</em> 2015</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Reid <em>et al.</em> 2016</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Rougemont <em>et al.</em> 2017</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Meier <em>et al.</em> 2018</a>), insects (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Soria-Carrasco <em>et al.</em> 2014</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Van Belleghem <em>et al.</em> 2018</a>), mammals (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Waterhouse <em>et al.</em> 2018</a>), molluscs (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Westram <em>et al.</em> 2014</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Ravinet <em>et al.</em> 2016</a>) and plants (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Roda <em>et al.</em> 2013</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Trucchi <em>et al.</em> 2017</a>). For the sake of consistency, it is common to infer outlier loci within each replicate pair through a common method, but if replicates differ in their demographic histories then the applicability/power of that common method will also vary accordingly. This begs the question, can demographic variation among replicates alone explain signals of, or lack of, convergence?</p><p id="p-10">Here, we used forward-simulations to investigate the effects of different demographic histories on the relationships between measures of divergence with selection. We varied demography through manipulating the number of founding individuals (founding bottlenecks), the population size of the diverging population (prolonged bottlenecks) and the presence/absence of migration. We simulated the effects of selection on 25kb regions, each with a gene designed from features taken from the guppy (<em>Poecilia reticulata</em>) genome assembly, a prominent model system for studies of convergent evolution (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Reznick and Endler 1982</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Fraser <em>et al.</em> 2015</a>). Moreover, we measured genetic divergence at 12 set time points through F<sub>ST</sub>, D<sub>XY</sub> and Δπ between diverging populations to examine temporal relationships. Our aim was to investigate the significance of founding/prolonged bottlenecks and migration between populations when employing outlier scans, and highlight demographic scenarios that are susceptible to false-positives. We also aimed to test the occurrence of common outliers across measures, and whether overlapping outliers are consistently good indicators of selection. We sought to answer the following questions: 1) How do these demographic factors influence measures of divergence through time? 2) How well do measures of divergence identify regions of the genome under strong selection through time and across demographic factors? 3) How are the different measures of divergence related through time and across demographic factors? 4) Where do we detect the strongest signals of convergence (<em>i.e.</em>, overlapping outliers using single or multiple measures), and are these consistent with selection?</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><p id="p-11">The forward-simulation software SLiM 3.0 (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Haller and Messer 2019</a>) was used to simulate population divergence under contrasting demographic treatments in a fully factorial design between a population with an ancestral phenotype (AP) of N = 1000 and population with a diverged phenotype (DP), with a mutation rate based on the guppy genome (4.89e<sup>-8</sup>) (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Künstner <em>et al.</em> 2016</a>) and scaled 100-fold over three raw mutation rates (4.89e<sup>-5</sup>, 4.89e<sup>-6</sup>, 4.89e<sup>-7</sup>) to ensure robustness of results across different, more realistic values of θ (4N<sub>e</sub>μ) (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). These scaled mutation rates therefore represent effective population sizes of 10-, 100-, and 1000-times greater than the number of individuals in our simulations (N = 1000), in line with estimates from other species (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Charlesworth 2009</a>). The main text reflects results for the intermediate mutation rate 4.89e<sup>-6</sup>, with others presented in supplementary figures. SLiM employs a classic Wright-Fisher model to simulate populations, in which a population of diploid hermaphrodites proceeds through generations such that an individual’s contribution toward the next generation is proportional to its relative fitness.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495754/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495754/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Simulation Parameters</span><div class="sb-div caption-clear"></div></div></div><p id="p-12">Demographic treatments included reducing DP size (prolonged bottlenecks) relative to the AP size (N = 1000) (0.01, 0.1, 0.5, 1.0), migration as a proportion of individuals exchanged between populations (0.0, 0.002) and founding bottlenecks as the number of individuals sampled from the burn-in population to construct DP genomes (N = 100 or 1000). For example, a demographic history with founding bottleneck = 100 individuals, DP size = 0.01, and migration = 0.002 would represent the following scenario: 1) At generation 1, following a burn-in period of 10,000 generations to reach mutation-selection balance, populations split as 100 genomes are sampled from the burn-in population of 1000 to form DP, while AP is formed by sampling all 1000 burn-in genomes; 2) At the next generation (and for the remainder of the simulation), DP size reflects the prolonged bottleneck treatment, in this case 10 (0.01 of 1000); 3) AP and DP experience migration in both directions for the remainder of the simulation. Thus, each of our treatments can be thought of as a manipulation of the following: Founding bottlenecks limit the amount of variation within the burn-in population that is available to found DP; Prolonged bottlenecks limit the size of DP, reducing mutational input and moderating the strength of neutral evolution and efficacy of selection; and migration dictates the presence of migration between AP and DP (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The total N of individuals (AP + DP) within simulations varies between 1010 and 2000 depending on DP size parameter, however this potential expansion does not impact our results (Supporting Information).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/677/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Experimental design for simulations. A) Examples of selection treatments experienced by genes from across the range, illustrated as the relationship between relative fitness of an individual and its phenotype. Each facet represents a different fitness landscape modified through editing the standard deviation of the normal distribution of fitness consequences in DP (blue, dashed). Facet labels constitute S, which were transformed as 10-S to give fitness function standard deviations (S&#x3C3;). The x-axes represent phenotype as calculated through non-synonymous mutations and the y-axes represent relative fitness of individuals. The ancestral phenotype of AP (red, solid) is the same in all treatments (mean = 0, &#x3C3; = 1), while DPs have a diverged phenotypic optimum (mean = 10, &#x3C3; = S&#x3C3;). B) Distribution of S values applied to genomic regions (1 per region) C) Demographic representation of treatment factors. D) Representation of simulation timeline for treatments, illustrating that all treatments share an ancestral burn-in population before splitting into 16 replicated &#x201C;AP&#x201D; (solid) and &#x201C;DP&#x201D; (dotted) population pairs. Red, dashed lines denote sampling generations at which FST, DXY and &#x394;&#x3C0; are calculated and averaged across the preceding 20 generations. The purpose of this averaging was to achieve a general sense of population differentiation at sampling points, such that values represent stable patterns rather than stochastic generation to generation variation. E) Examples of two simulated 25kb regions, showing central genes that function as QTL, and illustrating how regions vary in gene length, exon N and selection target (%). F) Representation of 100 simulated 25kb regions dataset per iteration (N = 20) per treatment group (N = 16)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1717965380" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Experimental design for simulations. &lt;strong&gt;A)&lt;/strong&gt; Examples of selection treatments experienced by genes from across the range, illustrated as the relationship between relative fitness of an individual and its phenotype. Each facet represents a different fitness landscape modified through editing the standard deviation of the normal distribution of fitness consequences in DP (blue, dashed). Facet labels constitute &lt;em&gt;S&lt;/em&gt;, which were transformed as 10&lt;sup&gt;-&lt;/sup&gt;&lt;em&gt;&lt;sup&gt;S&lt;/sup&gt;&lt;/em&gt; to give fitness function standard deviations (&lt;em&gt;S&lt;sub&gt;&#x3C3;&lt;/sub&gt;&lt;/em&gt;). The x-axes represent phenotype as calculated through non-synonymous mutations and the y-axes represent relative fitness of individuals. The ancestral phenotype of AP (red, solid) is the same in all treatments (mean = 0, &#x3C3; = 1), while DPs have a diverged phenotypic optimum (mean = 10, &#x3C3; = &lt;em&gt;S&lt;sub&gt;&#x3C3;&lt;/sub&gt;&lt;/em&gt;). &lt;strong&gt;B)&lt;/strong&gt; Distribution of &lt;em&gt;S&lt;/em&gt; values applied to genomic regions (1 per region) &lt;strong&gt;C)&lt;/strong&gt; Demographic representation of treatment factors. &lt;strong&gt;D)&lt;/strong&gt; Representation of simulation timeline for treatments, illustrating that all treatments share an ancestral burn-in population before splitting into 16 replicated &#x201C;AP&#x201D; (solid) and &#x201C;DP&#x201D; (dotted) population pairs. Red, dashed lines denote sampling generations at which F&lt;sub&gt;ST&lt;/sub&gt;, D&lt;sub&gt;XY&lt;/sub&gt; and &#x394;&#x3C0; are calculated and averaged across the preceding 20 generations. The purpose of this averaging was to achieve a general sense of population differentiation at sampling points, such that values represent stable patterns rather than stochastic generation to generation variation. &lt;strong&gt;E)&lt;/strong&gt; Examples of two simulated 25kb regions, showing central genes that function as QTL, and illustrating how regions vary in gene length, exon N and selection target (%). &lt;strong&gt;F)&lt;/strong&gt; Representation of 100 simulated 25kb regions dataset per iteration (N = 20) per treatment group (N = 16).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/677/F1.medium.gif" width="440" height="399"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/2/677/F1.medium.gif" width="440" height="399"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/677/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/677/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495697" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-13" class="first-child">Experimental design for simulations. <strong>A)</strong> Examples of selection treatments experienced by genes from across the range, illustrated as the relationship between relative fitness of an individual and its phenotype. Each facet represents a different fitness landscape modified through editing the standard deviation of the normal distribution of fitness consequences in DP (blue, dashed). Facet labels constitute <em>S</em>, which were transformed as 10<sup>-</sup><em><sup>S</sup></em> to give fitness function standard deviations (<em>S<sub>σ</sub></em>). The x-axes represent phenotype as calculated through non-synonymous mutations and the y-axes represent relative fitness of individuals. The ancestral phenotype of AP (red, solid) is the same in all treatments (mean = 0, σ = 1), while DPs have a diverged phenotypic optimum (mean = 10, σ = <em>S<sub>σ</sub></em>). <strong>B)</strong> Distribution of <em>S</em> values applied to genomic regions (1 per region) <strong>C)</strong> Demographic representation of treatment factors. <strong>D)</strong> Representation of simulation timeline for treatments, illustrating that all treatments share an ancestral burn-in population before splitting into 16 replicated “AP” (solid) and “DP” (dotted) population pairs. Red, dashed lines denote sampling generations at which F<sub>ST</sub>, D<sub>XY</sub> and Δπ are calculated and averaged across the preceding 20 generations. The purpose of this averaging was to achieve a general sense of population differentiation at sampling points, such that values represent stable patterns rather than stochastic generation to generation variation. <strong>E)</strong> Examples of two simulated 25kb regions, showing central genes that function as QTL, and illustrating how regions vary in gene length, exon N and selection target (%). <strong>F)</strong> Representation of 100 simulated 25kb regions dataset per iteration (N = 20) per treatment group (N = 16).</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">Combining these treatment levels in a fully factorial experimental design generated a total of 16 different, independent demographic histories that all 25kb gene regions (see below for architecture) experienced. This factorial design allows us to examine the relative influence of founding bottlenecks, prolonged bottlenecks and migration, but our study is limited to these features of demographic history and does not extend to population expansions, more complex cases of migration, or demographic fluctuations through time. In some cases, the influence of prolonged bottlenecks renders the effects of founding bottlenecks unnecessary. However, when DP size is greater than 100, the inclusion of the founding bottleneck allows us to compare populations with equivalent mutational input but different standing variation.</p><p id="p-15">SLiM runs were performed independently over 25kb genomic regions with a central ‘gene’ that functioned as a QTL and varied in length and exon content (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1E</a>); with exon number, lengths, and intron lengths drawn at random from the guppy genome gene annotation file (gff). In total, we simulated a dataset of 100 25kb regions (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1F</a>). Recombination rate was scaled alongside mutation rate from a human-derived <em>r</em> = 1e<sup>-8</sup> to 1e<sup>-6</sup>.</p><p id="p-16">Selection (<em>S</em>) in our model was represented as the fitness consequences incurred through distance from a phenotypic optimum in a one-dimensional fitness landscape. Selection in Wright-Fisher models is soft, in that low fitness individuals are not removed but have a lower likelihood of contributing to the next generation. Phenotypic optima were maintained over the course of simulations; thus, selection was constant throughout. This setup can be considered as analogous to two environments with contrasting optimum trait values for a single trait. Intensity of selection was manipulated by modifying the standard deviation (<em>S</em><sub>σ</sub>) of the normal distribution curve from which the density distribution was calculated through the “dnorm” function in SLiM. Values for <em>S</em> were drawn from a continuous distribution between -1.00 and 1.00 and transformed such that <em>S</em><sub>σ</sub> = 10<sup>-</sup><em><sup>S</sup></em>, yielding values between 0.1 and 10, with 0.1 representing the steepest fitness peak (<em>S</em> = 1) and 10 the shallowest (<em>S</em> = -1) (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>). Phenotypes were calculated per individual, per region, as the additive phenotypic effects of exonic non-synonymous mutations, which appeared at a rate of 7/3 relative to synonymous mutations (assuming that most mutations in the third base of a codon do not alter the amino acid). Additive genetic variance was assessed due to its prevalence in complex traits in nature (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hill <em>et al.</em> 2008</a>). Effect sizes for mutations with phenotypic effects were drawn from a Gaussian distribution with mean = 0 and σ = 1. The remaining synonymous exon mutations, and mutations in introns and outside of genes, had no effect on fitness.</p><p id="p-17">For each simulation, populations were seeded with 1000 individuals and allowed to proceed for a burn-in period of 5*2N (10,000) generations to reach mutation-selection-migration balance. During this period, burn-in populations evolved toward the ancestral phenotypic optimum of 0, defined as a normal distribution with mean = 0 and σ = 1.0 (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1A</a>). Burn-in populations were then subjected to each demographic treatment to simulate the founding of multiple populations from a shared ancestral state. During this ‘divergence period’, AP continued to evolve around the ancestral optimum of 0, while DP’s phenotypic optimum was centered around 10, with fitness consequences defined according to <em>S<sub>σ</sub></em>. Individuals also experienced fitness costs associated with phenotypic proximity to other individuals within the population as a proxy for competition and to ensure a realistic amount of phenotypic variation persisted within populations. Fitness costs due to competitive proximity were scaled to a maximum value of 1 with σ = 0.4 and occurred reciprocally between local individuals with phenotypes with a difference of ≤ 1.2 (3 * 0.4).</p><p id="p-18">Simulations were sampled at 100, 500, 1000, and then every 1000 generations up to 10,000 (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1C</a>). F<sub>ST</sub> was calculated across the 25kb region based on the proportion of subpopulation heterozygosity (<em>H<sub>S</sub></em>) relative to total heterozygosity (<em>H<sub>T</sub></em>) (according to <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Hudson, Slatkin and Maddison (1992)</a>): <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/677/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/677/embed/mml-math-1.gif"/></noscript></span></span>. D<sub>XY</sub> was calculated as the sum of nucleotide differences (<em>d<sub>ij</sub></em>) between the <em>i</em><sup>th</sup> haplotype from AP and the <em>j</em><sup>th</sup> haplotype from DP (according to <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Nei (1987)</a>): <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/677/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/677/embed/mml-math-2.gif"/></noscript></span></span>. Mean heterozygosity (π) for each population was calculated as a single measure across the 25kb region. At each sampling point, each measure was calculated and averaged across the preceding 20 generations. Averaging was performed such that measures would not be dramatically biased by events occurring within individual generations. The change in mean heterozygosity between AP and DP (Δπ) was calculated as the ratio of π<sub>AP</sub> to π<sub>DP</sub>, such that reduced diversity in the DP population increases the value of Δπ: <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/677/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/2/677/embed/mml-math-3.gif"/></noscript></span></span>. Statistics were calculated over all monomorphic and polymorphic sites of 25kb regions. This has been designed to replicate genome scans that use window-approaches, with each 25kb region analogous to an independent window. In total, 100 unique 25kb regions were simulated across 16 demographic treatments across three mutation rates, with results for the intermediate mutation rate presented in the main text. To account for stochastic noise in the simulation, each 25kb region was iterated 20 times for each demographic treatment. Simulations with divergent AP and DP phenotypes are referred to as “Pheno<sub>Div</sub>” simulations. Additional simulations were performed in which both populations shared the ancestral phenotype (“Pheno<sub>Null</sub>” simulations) and in which all sites were neutral with no selection imposed (“Neutral” simulations). The former of these was used to assess whether patterns associated with selection were driven by phenotypic divergence or variable stabilizing selection within populations, while the latter was used to disentangle effects of selection and neutral processes such as drift. A common set of 100 25kb regions were used for all simulations. Outliers for simulations were taken as upper 95% quantiles.</p><p id="p-19">All data analysis was performed in R (3.5) (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">R Core Team 2016</a>). To assess relationships between divergence measures and selection, data were grouped by sampling generation within each treatment group (N = 16) and Pearson’s correlation coefficients were calculated between measures of divergence and selection (<em>S</em>) at each sampling point for all regions (N = 100). Correlation coefficients were then grouped within specific treatment levels (<em>e.g.</em>, DP size = 0.5, or migration = 0.002) and averaged to give a coefficient reflecting each specific treatment level. These correlation coefficients were calculated for each iteration (N = 20) and averaged over to give final values.</p><p id="p-20">To assess the effects of treatments on detecting outliers, we compared distributions and 95% cut-offs within each treatment for each measure of divergence for Pheno<sub>Div</sub>, Pheno<sub>Null</sub>, and Neutral simulations. We limited this analysis to early (100, 500), intermediate (3000) and late (10,000) sampling generations. Here, data from all iterations of each 25kb region were pooled. To calculate false positive (FPR) and false negative rates (FNR), we pooled Neutral simulation data within treatment groups (20 iterations of 100 genes, N = 2000), removed a random set of iterations (N = 100), and replaced it with an assortment of random iterations of each gene (<em>e.g.</em>, Gene 1/Iteration 2, Gene 2/Iteration 14... Gene 100/Iteration 4) from Pheno<sub>Div</sub> data. We calculated FPR as the proportion of data above the 95% quantile for each measure of divergence/differentiation that came from neutral simulations. For single measures, FPR = FNR as 5% of data in each permutation comes from Pheno<sub>Div</sub>. We combined outlier sets across all combinations of F<sub>ST</sub>, D<sub>XY</sub> and Δπ and examined neutral proportions within outlier sets to determine FPR. FNR for combined outlier sets corresponded to the proportion of Pheno<sub>Div</sub> data not recovered in the combined outlier sets. These permutations were performed 100 times with results averaged. Proportional overlap of outlier sets was also calculated and compared across demographic treatment groups to examine convergence of results across treatments. Overlap was calculated within each permutation, averaged over, and visualized using heatmaps with hierarchical clustering of axes.</p><p id="p-21">To examine how simulated gene features influenced patterns of genetic divergence, we used a linear mixed modeling (LMM) approach with gene ID and treatment group as random factors. Gene features modeled as independent factors were: number of exons (Exon N), gene size, the proportion of gene that is coding (selection target %), selection applied to each gene (<em>S</em>), and the generation at which the optimum phenotype was reached (Pheno Gen).</p><div id="sec-2" class="subsection other1"><h3>Data availability</h3><p id="p-22">A full set of scripts including all bash, Eidos and R scripts necessary to repeat this analysis can be downloaded from Github (<a href="https://github.com/JimWhiting91/Contingent_Convergence_Pipeline">https://github.com/JimWhiting91/Contingent_Convergence_Pipeline</a>). Supplementary figures (S1-49) have been uploaded through the GSA figshare portal. Supplementary figures include results for different mutation/recombination rates, Pheno<sub>Null</sub> and neutral data across sampling generations along with additional figures. Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.11323088">https://doi.org/10.25387/g3.11323088</a>.</p></div></div><div class="section results" id="sec-3"><h2 class="">Results</h2><div id="sec-4" class="subsection"><h3>Demography modifies measures of divergence</h3><p id="p-23">Founding bottlenecks, simulated by reducing the number of possible founding genomes to a random 10% of the ancestral population, had little measurable effect on F<sub>ST</sub>, D<sub>XY</sub> or Δπ, producing minimal variance between treatments over all sampling points (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/677/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effects of demographic treatments on measures of genetic divergence across all sampling generations. Point color and shape denote treatment groups for founding bottlenecks (A), prolonged bottlenecks (B) and migration (C). Each point represents values of divergence averaged across all genes within individual treatments groups, averaged within treatment levels, and averaged over 20 iterations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1717965380" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Effects of demographic treatments on measures of genetic divergence across all sampling generations. Point color and shape denote treatment groups for founding bottlenecks (&lt;strong&gt;A&lt;/strong&gt;), prolonged bottlenecks (&lt;strong&gt;B&lt;/strong&gt;) and migration (&lt;strong&gt;C&lt;/strong&gt;). Each point represents values of divergence averaged across all genes within individual treatments groups, averaged within treatment levels, and averaged over 20 iterations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/677/F2.medium.gif" width="440" height="290"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/2/677/F2.medium.gif" width="440" height="290"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/677/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/677/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495660" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-24" class="first-child">Effects of demographic treatments on measures of genetic divergence across all sampling generations. Point color and shape denote treatment groups for founding bottlenecks (<strong>A</strong>), prolonged bottlenecks (<strong>B</strong>) and migration (<strong>C</strong>). Each point represents values of divergence averaged across all genes within individual treatments groups, averaged within treatment levels, and averaged over 20 iterations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Prolonged bottlenecks, simulated by modifying the stable number of individuals within DP, had pronounced and variable effects on all measures. F<sub>ST</sub> increased with reductions in DP size. This effect was generally consistent through time, although variance between treatments increased gradually through time (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). A similar effect was observed for Δπ; however, this measure was particularly susceptible to inflation under the most extreme reductions in DP size, with substantially more elevated values observed between DP size proportions of 0.01 and 0.1 compared with 0.1 and either 0.5 or 1.0. This pattern was broadly consistent through sampling points. Such observations are unsurprising given that both F<sub>ST</sub> and Δπ increase with processes that reduce within-population genetic variance. D<sub>XY</sub> was generally robust to prolonged bottlenecks, but in contrast to F<sub>ST</sub> and Δπ, D<sub>XY</sub> decreased when DP sizes were reduced (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>).</p><p id="p-26">The inclusion of migration reduced absolute values of all measures of divergence. For F<sub>ST</sub> and D<sub>XY</sub>, variance between migration treatments increased across the simulation, however D<sub>XY</sub> was generally more consistent across migration treatments. Δπ was also reduced in the presence of migration, although the effect of migration on Δπ was generally consistent across all generations and did not increase through time, as was the case for F<sub>ST</sub> and D<sub>XY</sub>.</p><p id="p-27">While F<sub>ST</sub> and D<sub>XY</sub> increased generally over time, Δπ peaked around generation 500 and declined thereafter. This peak corresponded to the median generation that DP replicates reached their optimum phenotype (median across all data = 493), suggesting this peak reflects selective sweeps. The majority of these patterns were apparent in the Pheno<sub>Null</sub> (Figure S1) and neutral (Figure S2) simulations, however divergence for all measures was reduced and ultimately negligible by the removal of divergent phenotypes when migration was present.</p></div><div id="sec-5" class="subsection"><h3>Demography moderates the association between measures of divergence and selection</h3><p id="p-28">Again, founding bottlenecks had a minimal effect on the correlations observed between strength of selection and measures of divergence with minimal variance observed between bottleneck treatments in all comparisons (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/677/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effects of demographic treatments on the relationship between selection and measures of genetic divergence across all sampling generations. Point color and shape denote treatment groups for founding bottlenecks (A), prolonged bottlenecks (B) and migration (C). Each point represents correlation coefficients calculated across all genes within individual treatments groups, averaged within treatment levels, and averaged over 20 iterations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1717965380" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Effects of demographic treatments on the relationship between selection and measures of genetic divergence across all sampling generations. Point color and shape denote treatment groups for founding bottlenecks (&lt;strong&gt;A&lt;/strong&gt;), prolonged bottlenecks (&lt;strong&gt;B&lt;/strong&gt;) and migration (&lt;strong&gt;C&lt;/strong&gt;). Each point represents correlation coefficients calculated across all genes within individual treatments groups, averaged within treatment levels, and averaged over 20 iterations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/677/F3.medium.gif" width="440" height="290"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/2/677/F3.medium.gif" width="440" height="290"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/677/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/677/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495779" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-29" class="first-child">Effects of demographic treatments on the relationship between selection and measures of genetic divergence across all sampling generations. Point color and shape denote treatment groups for founding bottlenecks (<strong>A</strong>), prolonged bottlenecks (<strong>B</strong>) and migration (<strong>C</strong>). Each point represents correlation coefficients calculated across all genes within individual treatments groups, averaged within treatment levels, and averaged over 20 iterations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">Prolonged bottlenecks had substantial effects on relative (F<sub>ST</sub> and Δπ) measures and marginal effects on absolute (D<sub>XY</sub>) measures of divergence (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). F<sub>ST</sub> correlations with selection became consistently weaker as DP size reduced. There was minimal difference between Δπ correlations with selection except for the most extreme reductions in DP size. Effects for both were generally consistent through time. For D<sub>XY</sub>, correlations with selection were broadly consistent with minimal variance across DP size reductions (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>).</p><p id="p-31">Expectedly, the absence of migration largely precluded the ability of measures of divergence to predict strength of selection, with notable variance observed between no migration (0.0) and minimal migration (0.002) treatments emerging rapidly for all divergence measures (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>). Both F<sub>ST</sub> and D<sub>XY</sub> variance between migration treatments was greatest at the 10,000 generations sampling point, whereas similar variance was observed between migration treatments for Δπ across simulations. This observation again highlights the significance of temporal differences between measures. Interestingly, correlations between Δπ and selection persisted, albeit weakly, in the absence of migration, which was not the case for F<sub>ST</sub> and D<sub>XY</sub>. Further, at larger population scaling (μ = 4.89e<sup>-5</sup>, <em>r</em> = 1e<sup>-5</sup>) D<sub>XY</sub> correlations with selection were negative (although became more positive over time with migration) (Figure S6), most likely due to a stronger influence of mutational input. In larger populations, positive correlations were observed between F<sub>ST</sub> and selection without migration, but were weaker than with migration (Figure S6).</p><p id="p-32">Prolonged bottlenecks had a minimal effect on how measures of divergence correlated with selection in Pheno<sub>Null</sub> data (Figure S5). Both F<sub>ST</sub> and D<sub>XY</sub> became negatively correlated with selection over time without divergent selection, likely due to stronger selection on common alleles shared between AP and DP. Negative correlations were stronger for D<sub>XY</sub>, consistent with reductions in π<sub>DP</sub> with stronger selection. This suggests positive associations between selection and D<sub>XY</sub> in Pheno<sub>Div</sub> simulations are likely more dependent on adaptive substitutions in order to overcome this effect. Δπ was generally positively associated with selection in Pheno<sub>Null</sub> simulations regardless of migration, but was slightly reduced when migration was absent.</p><p id="p-33">By examining the correlation coefficients of all 16 unique demographic histories, we can investigate the combined effect of migration and prolonged bottlenecks and directly compare effectiveness of individual measures across time (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). F<sub>ST</sub> consistently outperforms D<sub>XY</sub> in terms of associating with selection under most demographic treatments, particularly when DP sizes are larger and migration is present. By 10,000 generations however, the relative dominance of F<sub>ST</sub> appears to subside, with the trend through time suggesting a relative improvement in D<sub>XY</sub> in treatments with migration (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Δπ is similarly more informative than D<sub>XY</sub> across sampling generations under most demographic treatments. Interestingly at sampling generation 10,000, reductions in prolonged bottlenecks produce the biggest bias toward Δπ (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). The resilience of Δπ under no-migration treatments is also apparent in F<sub>ST</sub> - Δπ comparisons, such that at 3,000 generations Δπ is more informative than F<sub>ST</sub> in the absence of migration. Consistent with its rapid response to sweeps around 500 generation, Δπ slightly outperforms F<sub>ST</sub> under most demographic scenarios in early generations. By 10,000 generations, however, F<sub>ST</sub> performs as well as Δπ without migration and outperforms Δπ with migration.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/677/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pairwise comparisons between the correlation coefficients of selection with FST, DXY and &#x394;&#x3C0; across four sampling points. Each data point represents a unique demographic treatment with points colored according to DP population size and shaped according to migration level. Correlation 1 refers to the first measure listed in the comparison and Correlation 2 to the second. The y = x line is plotted within each facet to illustrate biases toward one measure. Points below the line are biased toward Correlation 1, while points above the line are biased toward Correlation 2. Each point represents correlation coefficients calculated across all genes within individual treatments groups and averaged over 20 iterations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1717965380" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Pairwise comparisons between the correlation coefficients of selection with F&lt;sub&gt;ST&lt;/sub&gt;, D&lt;sub&gt;XY&lt;/sub&gt; and &#x394;&#x3C0; across four sampling points. Each data point represents a unique demographic treatment with points colored according to DP population size and shaped according to migration level. Correlation 1 refers to the first measure listed in the comparison and Correlation 2 to the second. The y = x line is plotted within each facet to illustrate biases toward one measure. Points below the line are biased toward Correlation 1, while points above the line are biased toward Correlation 2. Each point represents correlation coefficients calculated across all genes within individual treatments groups and averaged over 20 iterations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/677/F4.medium.gif" width="440" height="310"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/10/2/677/F4.medium.gif" width="440" height="310"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/677/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/677/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495508" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">Pairwise comparisons between the correlation coefficients of selection with F<sub>ST</sub>, D<sub>XY</sub> and Δπ across four sampling points. Each data point represents a unique demographic treatment with points colored according to DP population size and shaped according to migration level. Correlation 1 refers to the first measure listed in the comparison and Correlation 2 to the second. The y = x line is plotted within each facet to illustrate biases toward one measure. Points below the line are biased toward Correlation 1, while points above the line are biased toward Correlation 2. Each point represents correlation coefficients calculated across all genes within individual treatments groups and averaged over 20 iterations.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Demography moderates the shape and tail end of divergence distributions</h3><p id="p-35">By comparing distributions across simulations with divergent (Pheno<sub>Div</sub>) and stabilizing (Pheno<sub>Null</sub>) selection with neutral runs, we can examine the effect of demographic treatments on the ability of each measure of divergence to discriminate between them (Figure S11-13; <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). There are few differences between distributions of F<sub>ST</sub> for the three simulation types when migration is absent between AP and DP replicates, highlighting an increased likelihood of false-positives. The exceptions occur in early sampling points at 100 and 500 generations (Figure S12) when sweeps are most common. With migration, Pheno<sub>Null</sub> F<sub>ST</sub> is marginally elevated compared with neutral F<sub>ST</sub>, but distributions are broadly similar. Pheno<sub>Div</sub> F<sub>ST</sub> distributions however become more positive and flattened, according to variable selection, with the majority of Pheno<sub>Div</sub> F<sub>ST</sub> above the 95% quantiles of Pheno<sub>Div</sub> and neutral F<sub>ST</sub> by 10,000 generations (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/677/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distributions of FST under each of the 16 unique demographic treatments under three selection regimes: PhenoDiv (divergent selection), PhenoNull (stabilizing selection) and Neutral, after 10,000 generations. Upper 5% quantiles are highlighted for each distribution, with linetype corresponding to selection: Solid = Divergent, Dashed = Stabilizing, Dotted = Neutral. Each distribution represents data pooled from 20 iterations of 100 25kb regions (N = 2000)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1717965380" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distributions of F&lt;sub&gt;ST&lt;/sub&gt; under each of the 16 unique demographic treatments under three selection regimes: Pheno&lt;sub&gt;Div&lt;/sub&gt; (divergent selection), Pheno&lt;sub&gt;Null&lt;/sub&gt; (stabilizing selection) and Neutral, after 10,000 generations. Upper 5% quantiles are highlighted for each distribution, with linetype corresponding to selection: Solid = Divergent, Dashed = Stabilizing, Dotted = Neutral. Each distribution represents data pooled from 20 iterations of 100 25kb regions (N = 2000).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/677/F5.medium.gif" width="440" height="366"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/10/2/677/F5.medium.gif" width="440" height="366"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/677/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/677/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495672" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-36" class="first-child">Distributions of F<sub>ST</sub> under each of the 16 unique demographic treatments under three selection regimes: Pheno<sub>Div</sub> (divergent selection), Pheno<sub>Null</sub> (stabilizing selection) and Neutral, after 10,000 generations. Upper 5% quantiles are highlighted for each distribution, with linetype corresponding to selection: Solid = Divergent, Dashed = Stabilizing, Dotted = Neutral. Each distribution represents data pooled from 20 iterations of 100 25kb regions (N = 2000).</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">Similar patterns were observed for D<sub>XY</sub> distributions (Figure S14-17), with little to discriminate between in treatments without migration. However, without migration, neutral D<sub>XY</sub> was generally reduced relative to Pheno<sub>Null</sub> and Pheno<sub>Div</sub> D<sub>XY</sub> at earlier sampling points. With migration, like F<sub>ST</sub>, Pheno<sub>Div</sub> D<sub>XY</sub> was readily distinguishable from Pheno<sub>Null</sub> and neutral distributions, but Pheno<sub>Null</sub> D<sub>XY</sub> was also generally more positive than neutral D<sub>XY</sub>. These patterns also emerged more slowly than for F<sub>ST</sub>. In contrast, Pheno<sub>Div</sub> Δπ (Figure S18-21) was elevated according to DP size, such that at 500 generations the majority of 25kb regions under divergent selection exhibited Δπ above neutral and Pheno<sub>Null</sub> 95% cut-offs for all treatments with DP size ≥ 0.5.</p><p id="p-38">We quantified false-positive rates (FPR) by permuting over merged data comprised of randomly sampled 5% Pheno<sub>Div</sub> regions and 95% neutral regions and observing the upper 5% quantile (Table S1). By 100 generations, F<sub>ST</sub> FPR ranged between 0.06 and 0.91, and was lower with increased DP size and lower in treatments without migration (Table S1). By 500 generations (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>), FPR rates were lower with migration and higher without, but only for treatments with DP sizes of 0.5 and 1.0. F<sub>ST</sub> FPR remained high (&gt; 0.81) for all treatments with smaller DP sizes. By 3,000 generations, migration was the most important demographic factor for F<sub>ST</sub> FPR. With migration, FPR ranged from 0.15 – 0.61, and decreased with increasing DP size. Without migration, FPR were high (0.88 - 0.97), close to the random proportion of neutral (0.95) data. By the end of simulations, F<sub>ST</sub> FPR was as low as 0.13, and was no greater than 0.27 with migration and DP size ≥ 0.1. Without migration, FPR was &gt; 0.94.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495773/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495773/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">False positive (FPR) and false negative rates (FNR) calculated across all measures of divergence and their combined use. Estimates were calculated by combining 5% of data under divergent selection with 95% neutral data and taking upper 5% cut-offs. For single measures, outlier N is always 100 (5% OF 2,000) and FNR = FPR</span><div class="sb-div caption-clear"></div></div></div><p id="p-39">Initial D<sub>XY</sub> FPR were generally high irrespective of demographic treatment, ranging between 0.78 and 0.93. FPR were largely similar after 500 generations, but by 3,000 generations there was a distinction between treatments with (0.07 ≤ FPR ≤ 0.71) and without (0.85 ≤ FPR ≤ 0.88) migration. Interestingly, here FPR rates were lower (0.07 ≤ FPR ≤ 0.24) when DP were smaller (size = 0.01, 0.1) rather than larger (0.47 ≤ FPR ≤ 0.71). This pattern was also observed at the end of simulations, with FPR lower without migration (0.01 ≤ FPR ≤ 0.34) and lowest with DP size = 0.1.</p><p id="p-40">Δπ FPR rates were generally higher across all sampling generations, with FPR not falling below 0.28. There was a clear distinction based on DP size in earlier (100 and 500) sampling points, with FPR lower with larger DP size. However, at later sampling generations (3,000 and 10,000), FPR were generally high (≥ 0.68) regardless of treatment.</p><p id="p-41">Taken together, there are clear effects of DP size and migration on distributions of genetic variation and upper quantiles of interest. DP size appears initially most important in the first few hundred generations for F<sub>ST</sub> and Δπ, but these effects are later swamped by the effect of migration for F<sub>ST</sub> and are simply eroded for Δπ. D<sub>XY</sub> exhibits similar patterns to F<sub>ST</sub>, but these develop after many more generations, and while gene flow increases the informativeness of D<sub>XY</sub> for detecting divergent selection, as it does for F<sub>ST</sub>, D<sub>XY</sub> and F<sub>ST</sub> experience opposing effects of increases to DP size.</p></div><div id="sec-7" class="subsection"><h3>Demography moderates relationships between measures of divergence</h3><p id="p-42">Given F<sub>ST</sub>, D<sub>XY</sub> and Δπ are all measures of population genetic divergence, there is an assumption that positive correlations should exist between them. We employed the same analysis as above for correlations with selection, but instead examined correlations between individual measures. Founding bottlenecks had minimal effects on the correlations observed between all measures of divergence (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/677/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effects of demographic treatments on the relationship between measures of genetic divergence across all sampling generations. Point color and shape denote treatment groups for founding bottlenecks (A), prolonged bottlenecks (B) and migration (C). Each point represents correlation coefficients calculated across all genes within individual treatments groups, averaged within treatment levels, and averaged over 20 iterations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1717965380" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Effects of demographic treatments on the relationship between measures of genetic divergence across all sampling generations. Point color and shape denote treatment groups for founding bottlenecks (&lt;strong&gt;A&lt;/strong&gt;), prolonged bottlenecks (&lt;strong&gt;B&lt;/strong&gt;) and migration (&lt;strong&gt;C&lt;/strong&gt;). Each point represents correlation coefficients calculated across all genes within individual treatments groups, averaged within treatment levels, and averaged over 20 iterations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/677/F6.medium.gif" width="440" height="288"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/10/2/677/F6.medium.gif" width="440" height="288"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/677/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/677/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495690" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-43" class="first-child">Effects of demographic treatments on the relationship between measures of genetic divergence across all sampling generations. Point color and shape denote treatment groups for founding bottlenecks (<strong>A</strong>), prolonged bottlenecks (<strong>B</strong>) and migration (<strong>C</strong>). Each point represents correlation coefficients calculated across all genes within individual treatments groups, averaged within treatment levels, and averaged over 20 iterations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">Positive correlations between F<sub>ST</sub> and D<sub>XY</sub> emerged rapidly irrespective of DP size, but smaller DP sizes generally increased the correlation (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>), with variance between treatments generally decreasing over time. Similarly, F<sub>ST</sub> and Δπ were generally positively correlated, however reductions in DP size reduced correlation coefficients. By 4,000 generations, F<sub>ST</sub> - Δπ correlations for DP sizes ≥ 0.1 stabilized around 0.4, but correlations under extreme prolonged bottlenecks continued to decline to a low of 0.19 (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a>). D<sub>XY</sub> - Δπ correlations were generally weaker than other comparisons across the course of simulations, but were minimally affected by prolonged bottlenecks.</p><p id="p-45">Migration induced substantial variance between correlations of divergence measures, with effects dependent on sampling point (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6C</a>). In the absence of migration, F<sub>ST</sub> and D<sub>XY</sub> were more strongly correlated for the first 4,000 generations than in treatments with migration. However, from here until 10,000 generations this pattern reversed and F<sub>ST</sub> - D<sub>XY</sub> correlations increased with migration and deteriorated in allopatry. F<sub>ST</sub> - Δπ correlations were strong initially, but a lack of migration weakened the correlation over time until measures were uncorrelated by around 4,000 generations. In contrast, in the presence of migration, positive correlations between F<sub>ST</sub> and Δπ were strong and relatively stable (R<sup>2</sup> = 0.75 – 0.61 over the whole simulation period). Interestingly, without migration, D<sub>XY</sub> and Δπ were largely uncorrelated, but migration induced a positive correlation between D<sub>XY</sub> and Δπ that emerged after 2,000 generations and continued to increase through time.</p><p id="p-46">Contextualised by our previous demonstrations of associations with selection, these results highlight that selection can induce positive correlations between measures. By 10,000 generations, all pairwise comparisons of divergence measures become positively correlated with migration, which we know is when variation is most strongly associated with selection. Crucially however, this relationship only emerges after several thousand generations, before which we observe positive correlations in migration-absent treatments when associations with selection are weak for all measures (F<sub>ST</sub> - D<sub>XY</sub> in particular). Extreme reductions in DP size also increase positive correlations between F<sub>ST</sub> and D<sub>XY</sub> despite poor associations with selection in these treatments. These results highlight that positive correlations between statistics are also achievable in the absence of divergent selection. It is also interesting to note the decay of correlations with Δπ and both F<sub>ST</sub> and D<sub>XY</sub> in the absence of migration. These observations are most likely driven by the substitution rate. Substitutions that are not linked to selection (drift with ineffective selection) likely drive increased F<sub>ST</sub> and D<sub>XY</sub> but not Δπ.</p><p id="p-47">In Pheno<sub>Null</sub> simulations, F<sub>ST</sub> and D<sub>XY</sub> were positively correlated in all treatments, but stronger associations linked with demographic treatments with effective selection did not emerge (Figure S22). This was also true for neutral simulations (Figure S23), but F<sub>ST</sub> - D<sub>XY</sub> correlations were slightly larger than for Pheno<sub>Div</sub> and Pheno<sub>Null</sub> when selection was ineffective, reaching a maximum of R<sup>2</sup> = 0.81 without migration (Figure S23C). This demonstrates that the positive correlations in Pheno<sub>Div</sub> simulations are driven in part by divergently adaptive allele frequency shifts and substitutions when selection is effective, but these correlations can also emerge under stabilizing selection or neutrality. Specifically, strong positive correlations with Δπ were dependent on divergent selection, and F<sub>ST</sub> – Δπ correlations became negative over time in treatments without migration under neutrality. D<sub>XY</sub> – Δπ correlations became negative in Pheno<sub>Null</sub> data with migration and were unassociated without. Thus, these results highlight that the relationships between measures of divergence are highly dependent on migration, population size, time, and selection experienced over a genomic region.</p><p id="p-48">We then examined FPR and false negative rates (FNR) when combining outliers across F<sub>ST</sub>, D<sub>XY</sub> and Δπ (Table S1; summaries from 500 and 10,000 generations in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Combined F<sub>ST</sub> - D<sub>XY</sub> outliers exhibited FPR rates that were highly variable (0.00 - 0.95) and similar to or slightly lower than F<sub>ST</sub> alone at generations 100 and 500, suggesting some improvement in reducing FPR. During this period, FNRs were also high (≥ 0.78), suggesting most regions with divergent selection could not be detected on a neutral backdrop. Combined F<sub>ST</sub> - D<sub>XY</sub> outlier sets performed well at generations 3,000 and 10,000 for treatments with migration, in some cases dropping to 0 although FNR were highly variable (0.26 ≤ FNR ≤ 1.00). High FPR (0.84 ≤ FPR ≤ 0.99) and high FNR (0.93 ≤ FPNR ≤ 1.00) were observed without migration, highlighting most common outliers between F<sub>ST</sub> and D<sub>XY</sub> to be neutral, and a failure to detect almost all divergent regions.</p><p id="p-49">Combined F<sub>ST</sub> - Δπ outliers tended to outperform F<sub>ST</sub> and Δπ outliers according to FPR with DP sizes of 0.5 or 1.0 and in earlier (100 and 500) sampling generations. Here, FPR dropped to a low of 0 but FNR were reasonable through this time (≥ 0.35). Beyond this (sampling generations 3,000 and 10,000), FPR again dropped to 0, however these were generally alongside high FNR of up to 1.0 without migration, highlighting a failure to detect any common outliers at all. A good example of improvement over singular measures was observed at generation 500, with a founding bottleneck, equal sized populations, and migration. Here, an average of ∼66% of divergent regions were detected with an average FPR of 0. This is compared with: singular F<sub>ST</sub>, where FPR/FNR = 0.18; and singular Δπ where FPR/FNR = 0.29. By 10,000 generations, combined outlier sets of F<sub>ST</sub> - Δπ performed poorer than singular F<sub>ST</sub> with migration present, but generally returned low numbers of false positives when migration was absent, unlike F<sub>ST</sub> - D<sub>XY</sub>.</p><p id="p-50">Combined D<sub>XY</sub> - Δπ outliers performed poorly across all treatments and all sampling generations, with FNR failing to fall below 0.71. There were, however, some benefits in terms of low FPR for treatments with larger DP sizes (0.5 and 1.0) across all sampling points, suggesting high confidence in outliers (although most divergent regions are missed). This discordance between D<sub>XY</sub> and Δπ and large FNR also limited the combined usage of all three measures together, with similarly high FNR largely precluding their combined usage. All three statistics did exhibit low FPR with larger DP populations at 100, and 500 generations despite migration. However, at later sampling generations performance of all three combined outlier sets was poor (high FPR/FNR) if migration was absent.</p><p id="p-51">These results therefore highlight that combining measures can help reduce FPR, but usually at the cost of increased FNR (expectedly), and only under certain demographies. Our observation that high FPR are prevalent among combined outlier sets from statistics, particularly in the absence of migration, suggests their usage must be dependent on a knowledge of disparity in population size and connectivity of populations. These findings also highlight that the strong correlations that emerge between measures of divergence under scenarios with ineffective selection or even under neutrality do extend to the tail-ends of distributions.</p></div><div id="sec-8" class="subsection"><h3>Demography drives signals of convergence irrespective of selection</h3><p id="p-52">Clusters of overlapping outliers developed steadily over time (Figure S26-28). By sampling generation 10,000, significant proportions of overlapping outliers were recovered across different demographic treatment groups (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). For F<sub>ST</sub> and D<sub>XY</sub>, clustering of treatments was driven by the presence/absence of migration, with the highest proportions of convergent outliers observed between treatments with migration.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/2/677/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The proportional overlap of outliers above the 95% quantile, averaged across 100 downsampled datasets consisting of 95% Neutral and 5% PhenoDiv data for each of the 16 demographic treatments after 10,000 generations. Axis orderings were determined through hierarchical clustering. Heatmaps are shown for single measures of FST, DXY and &#x394;&#x3C0; in the first column, and combined measures in the second column. Heatmaps are colored according to a common scale of 0 to 1. Treatments are labeled with founding bottleneck (Bot), DP population size (Pop2), and migration (Mig) values." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1717965380" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The proportional overlap of outliers above the 95% quantile, averaged across 100 downsampled datasets consisting of 95% Neutral and 5% Pheno&lt;sub&gt;Div&lt;/sub&gt; data for each of the 16 demographic treatments after 10,000 generations. Axis orderings were determined through hierarchical clustering. Heatmaps are shown for single measures of F&lt;sub&gt;ST&lt;/sub&gt;, D&lt;sub&gt;XY&lt;/sub&gt; and &#x394;&#x3C0; in the first column, and combined measures in the second column. Heatmaps are colored according to a common scale of 0 to 1. Treatments are labeled with founding bottleneck (Bot), DP population size (Pop2), and migration (Mig) values.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/2/677/F7.medium.gif" width="307" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/10/2/677/F7.medium.gif" width="307" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/2/677/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/2/677/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495667" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-53" class="first-child">The proportional overlap of outliers above the 95% quantile, averaged across 100 downsampled datasets consisting of 95% Neutral and 5% Pheno<sub>Div</sub> data for each of the 16 demographic treatments after 10,000 generations. Axis orderings were determined through hierarchical clustering. Heatmaps are shown for single measures of F<sub>ST</sub>, D<sub>XY</sub> and Δπ in the first column, and combined measures in the second column. Heatmaps are colored according to a common scale of 0 to 1. Treatments are labeled with founding bottleneck (Bot), DP population size (Pop2), and migration (Mig) values.</p><div class="sb-div caption-clear"></div></div></div><p id="p-54">Interestingly, for D<sub>XY</sub> reasonable proportions of convergent outliers were also recovered across no migration treatments, and the same was true of F<sub>ST</sub> by 3,000 generations (Figure S28), and for both measures at 10,000 generations in ‘smaller’ populations with reduced scaling (μ = 4.89e<sup>-7</sup>, <em>r</em> = 1e<sup>-7</sup>; Figure S37). This is despite these treatment groups lacking effective selection. Importantly, there was little overlap between migration and no-migration clusters, suggesting different convergent outliers within each. Combining outliers from F<sub>ST</sub> and D<sub>XY</sub> reduced overlap among no-migration treatments, but did not remove overlapping outliers altogether.</p><p id="p-55">There were minimal convergent outliers observed for Δπ outliers, however combining Δπ with F<sub>ST</sub> and D<sub>XY</sub> did appear somewhat effective in removing convergent outliers found between no-migration treatments. However, for both combination of Δπ with F<sub>ST</sub> and D<sub>XY</sub>, the highest proportional overlap was observed for treatments with the smallest DP size.</p><p id="p-56">Interestingly, the clustering of F<sub>ST</sub> - D<sub>XY</sub> outliers in the presence of migration was greatly reduced when divergent selection was removed in both Pheno<sub>Null</sub> (Figure S29-32) and neutral data (Figure S33-36), but clusters of outliers in the absence of migration were still apparent.</p><p id="p-57">Because migration was the dominant factor in clustering of treatments with convergent outlier overlap, we sought to investigate what features of simulated genes drove variance in measures of divergence with treatments separated by migration factor using linear mixed models at the final sampling generation. With migration between AP and DP, selection had by far the strongest effect on F<sub>ST</sub> (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>) in the expected positive direction. We also observed a weaker positive association with selection target (% coding) of gene (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>), and weaker negative associations with Pheno Gen (generation DP optimum reached) (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). These fixed effects explained 48.96% of variance in F<sub>ST</sub> with migration.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495636/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495636/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">LMM results for models of measures of divergence explained by features of simulated regions. For all models, random variables included gene ID and demographic treatment</span><div class="sb-div caption-clear"></div></div></div><p id="p-58">Conversely, fixed effects explained minimal variance (1.10%) of F<sub>ST</sub> in treatments without migration. These fixed effects were significant, but had weak positive associations with selection, exon N and selection target %, and a weak negative association with Pheno Gen.</p><p id="p-59">D<sub>XY</sub> was similarly most strongly positively associated with selection in treatments with migration (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>), but the strength of this effect relative to the other model effects was not as large as observed for F<sub>ST</sub>. D<sub>XY</sub> was also negatively associated with Pheno Gen (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>), as was F<sub>ST</sub>, but negatively associated with Exon N (<a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>). This model however explained less variance of D<sub>XY</sub> with migration (9.53%) than F<sub>ST</sub> with migration. In treatments without migration, selection target % was strongly negatively associated with D<sub>XY</sub> (<a id="xref-table-wrap-3-7" class="xref-table" href="#T3">Table 3</a>), and this fixed effect explained 30.70% of D<sub>XY</sub> variance without migration.</p><p id="p-60">Selection was the most important fixed effect for Δπ regardless of migration (<a id="xref-table-wrap-3-8" class="xref-table" href="#T3">Table 3</a>), however both models explained minimal variance (8.27% with migration, 0.59% without migration). This is consistent with the previous demonstration of the erosion of Δπ over time (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>).</p><p id="p-61">Removing divergent selection (<em>i.e.</em>, in Pheno<sub>Null</sub> simulations) modified models for F<sub>ST</sub> and D<sub>XY</sub> for treatments with migration. Selection and selection target remained the most important model effects, but had more similar sized effects, and ultimately variance explained dropped from 48.96 to 3.60%. Conversely, the model for Pheno<sub>Null</sub> D<sub>XY</sub> with migration increased variance explained from 9.53 to 13.10% compared with Pheno<sub>Div</sub> D<sub>XY</sub>. Selection target had a strongly significant negative association alongside strength of selection in this model. Δπ models were largely unchanged. Together, these results confirm that divergent selection, and not stabilizing selection within DP, drive F<sub>ST</sub> and D<sub>XY</sub> variation in Pheno<sub>Div</sub> simulations.</p></div></div><div class="section discussion" id="sec-9"><h2 class="">Discussion</h2><div id="sec-10" class="subsection"><h3>Summary of results</h3><p id="p-62">Here, we show that features of demography can have dramatic effects on how measures of population divergence identify regions of the genome under selection. Effects are also strongly time-dependent. Using simulated populations, we have demonstrated the relative influences of founding bottlenecks, prolonged bottlenecks, and migration on three commonly used measures of genetic divergence (F<sub>ST</sub>, D<sub>XY</sub>, Δπ), while demonstrating the relative usefulness of each measure for informing on selection when population sizes and connectivity vary.</p><p id="p-63">We find that founding bottlenecks have little effect on population divergence measures, potentially either because populations quickly recover (before first sampling after 100 generations), or founding bottlenecks of 10% (100 individuals) were not extreme enough. Prolonged bottlenecks (reductions in DP size) however, artificially inflate F<sub>ST</sub> and Δπ but reduce D<sub>XY,</sub> and can erase the relationship of F<sub>ST</sub> and D<sub>XY</sub> with selection under the most extreme reductions in population size. Relative measures are, in part, driven by intra-population changes in allele frequency, which become exaggerated in smaller populations as a product of drift (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Charlesworth 2009</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Ellegren and Galtier 2016</a>). As a consequence, we observe inflated measures of relative divergence as allele frequencies drift in smaller DP replicates.</p><p id="p-64">In contrast, D<sub>XY</sub> increases with larger DP size, as a product of the relationship between the number of segregating sites and the population-level mutation rate (4N<sub>e</sub>μ) (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hartl <em>et al.</em> 1997</a>). D<sub>XY</sub> is a measure of sequence divergence and is averaged across all sites (although similar statistics limit averaging to segregating sites only), which results in higher D<sub>XY</sub> as segregating sites are introduced into either population at a rate of 4N<sub>e</sub>μ. This relationship with the number of segregating sites can be observed by examining the positive relationships between D<sub>XY</sub> and π<sub>DP</sub> (Figure S38). Overall, the relationship between D<sub>XY</sub> and selection is less affected by prolonged bottlenecks than F<sub>ST</sub> and Δπ, likely due to the lack of allele frequency relevance. Consider for example, two SNPs with minor allele frequencies of 0/0.5 (SNP 1) and 0.5/0.5 (SNP 2) in AP/DP. Each locus contributes equally toward D<sub>xy</sub> (SNP 1 = [0 × 0.5] + [1 × 0.5] = 0.5; SNP 2 = [0.5 × 0.5] + [0.5 × 0.5] = 0.5), whereas the reduction of within-population variance observed for SNP 1 inflates F<sub>ST</sub> and Δπ. However, we also see evidence of D<sub>XY</sub> FPR increasing with increased DP size (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>), suggesting this relationship with increased acquisition of segregating sites may conflict with increased efficacy of selection.</p><p id="p-65">Migration, even at the relatively modest rate of 0.2% employed here, substantially reduced absolute values for all measures of divergence. However, while absolute values were reduced, their informativeness of selection coefficients increased dramatically (both in terms of their overall relationship with selection and in identifying outliers). Such a result is expected given the role of gene flow in homogenizing neutral loci (reducing measures of divergence), while retaining population divergence around adaptive loci (increasing informativeness). The well-known ‘genomic islands of divergence’ model is often invoked to explain this pattern of heterogenous genomic divergence in studies of speciation-with-gene-flow (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Turner <em>et al.</em> 2005</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Nosil <em>et al.</em> 2009</a>).</p><p id="p-66">Migration also exhibited interesting temporal patterns that are useful for discussing the discrepancies observed between Δπ and both F<sub>ST</sub> and D<sub>XY</sub>. Of the measures considered here, Δπ uniquely disregards SNP substitutions in its calculation, as fixed substitutions have no influence on intra-population heterozygosity beyond reducing the heterozygosity of proximate SNPs in the aftermath of a hard sweep. This characteristic explains why Δπ responds to selection more rapidly than F<sub>ST</sub> and D<sub>xy</sub> and is influenced by migration in a constant manner across time. In addition, the inability of Δπ to account for adaptive substitutions is likely why the informativeness of Δπ peaked at around 500 generations (approximate median for sweeps), decayed, and then stabilized. In contrast, the contributions of adaptive substitutions to F<sub>ST</sub> and D<sub>xy</sub> over time increases their predictive power. This characteristic of Δπ, however, also makes it unable to differentiate between divergent and stabilizing selection. This temporal observation has important implications for studies of rapid adaptation on the scale of 10s to 100s of generations, in which our simulations suggest Δπ may be the most informative measure of divergence, while D<sub>xy</sub> is initially uninformative for several thousand generations.</p><p id="p-67">Examining the effects of bottlenecks and migration on the associations between measures of divergence themselves is a novel element to this study. The relative weaknesses of individual measures of divergences has prompted a recent movement within the literature to employ multiple measures of divergence to avoid false-positives (for <em>e.g.</em>, <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Tine <em>et al.</em> 2014</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Malinsky <em>et al.</em> 2015</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hämälä and Savolainen 2019</a>). Our results provide some support for this strategy, with reductions in FPR in treatments with gene flow generally, and low FPR when combining either F<sub>ST</sub> or D<sub>XY</sub> with Δπ (albeit at a reasonably high FNR). However, we also find large numbers of overlapping outliers across combined F<sub>ST</sub> - D<sub>XY</sub> with a large FPR across treatments without migration. Further, these false-positives persisted in Pheno<sub>Null</sub> and neutral data, whereas clusters of treatments with overlapping outliers (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>) and with low FPR were restricted to simulations with divergent phenotypes. It is clear then that combining outlier sets of F<sub>ST</sub> and D<sub>XY</sub> only improves analyses under certain demographic histories, in particular when populations are connected by migration.</p><p id="p-68"><a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Cruickshank and Hahn (2014)</a> suggested that a disagreement between F<sub>ST</sub> and D<sub>XY</sub> outliers in genomic-islands-of-divergence tests highlights a particular susceptibility of F<sub>ST</sub> to BGS (although see (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Matthey‐Doret and Whitlock 2019</a>)). Our results are in line with the notion that D<sub>XY</sub> may be more resistant to false positives due to BGS, given that reductions in DP size resulted in minimal variance in D<sub>XY</sub> (assuming reductions in population size are analogous to different rates of BGS across the genome exhibit reduced N<sub>e</sub>). However, BGS was not specifically manipulated in these simulations, and results are difficult to disentangle with variation in efficacy of removing deleterious mutations. Further, this minimal variance may be explained by the opposing forces of selection efficacy and acquisition of segregating sites discussed previously. We also found that a greater proportion of variance could be explained by the size of selection target for D<sub>XY</sub> (30.7%) in the absence of migration than for F<sub>ST</sub> (1.1%). We found that F<sub>ST</sub> and D<sub>XY</sub> are positively correlated under several demographic scenarios, but this strong association only reflects selection when gene flow is present and only after several thousand generations, consistent with previous simulation work (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Ravinet <em>et al.</em> 2017</a>). When migration is absent, and selection ineffective, F<sub>ST</sub> and D<sub>XY</sub> are also positively correlated; but this relationship decays over time (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6C</a>), with empirical evidence (see below) suggesting a negative relationship is likely to emerge without gene flow. No-migration treatments are consistent with Isolation-By-Distance demographic histories and, similarly, comparisons between reproductively-isolated populations or species. Indeed, negative correlations between F<sub>ST</sub> and D<sub>XY</sub> within and between clades of birds (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Irwin <em>et al.</em> 2016</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Vijay <em>et al.</em> 2017</a>), within a radiation event of monkeyflowers (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Stankowski <em>et al.</em> 2019</a>) and between speciating orca populations (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Foote <em>et al.</em> 2016</a>) support a declining relationship between these measures over long periods of time in isolation. In agreement, we find that without migration F<sub>ST</sub> and D<sub>XY</sub> exhibit opposite associations with the proportion of regions made up of coding elements. Mechanistically, a larger selection target increases the rate of deleterious mutation, reducing local π directly through loss of polymorphic deleterious sites, or indirectly through the loss of linked neutral variants under a BGS model. Reductions in local π increase F<sub>ST</sub> and decrease D<sub>XY.</sub> Over time, associations between F<sub>ST</sub> and D<sub>XY</sub> should stabilize, given π is generally well-conserved in stable populations even across long time periods (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Romiguier <em>et al.</em> 2014</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Dutoit <em>et al.</em> 2017</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Van Doren <em>et al.</em> 2017</a>).</p><p id="p-69">By comparing our results to a second dataset that lacked divergent selection (Pheno<sub>Null</sub>), we found consistent support that positive associations between D<sub>XY</sub> and selection are strongly dependent on the inclusion of a divergent phenotype. Positive associations with F<sub>ST</sub> are attainable only in the absence of migration, and are weakly negative without, and Δπ patterns are primarily driven by variable stabilizing selection and made weaker by the inclusion of phenotypic divergence. These comparisons are useful in highlighting the relative roles of adaptive allele frequency changes and substitutions in driving patterns of genetic differentiation. It is also of interest to note that overlapping outliers are readily attainable (most strongly for D<sub>XY</sub>) across no-migration treatments in Pheno<sub>Null</sub> (Figure S32) and neutral simulations (Figure S36), but not for treatments with migration. This confirms that overlapping outliers in no-migration treatments occur due to common non-divergent processes within genomic regions, whereas overlapping outliers in treatments with migration are driven by the effects of divergent selection. The discrepancies between our Pheno<sub>Div</sub> and other simulated datasets highlight the necessity in quantifying phenotypic differences or environmental selection pressure when interpreting patterns of variation across the genome.</p></div><div id="sec-11" class="subsection"><h3>Detecting genetic convergence</h3><p id="p-70">In addition to understanding how outlier detection in individual pairs were affected by bottlenecks and migration, we wanted to explore how studies looking at overlapping outliers in multiple pairs (<em>i.e.</em>, detecting genetic convergence) were affected by these aspects of demography. Our simulation design, in which an ancestral burn-in population is used to found 16 independent AP-DP pairs is analogous to replicated ecotype population pairs in model systems, such as various ecotype pairs of the three-spined stickleback, <em>Gasterosteus aculeatus</em> (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Hohenlohe <em>et al.</em> 2010</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Jones <em>et al.</em> 2012a</a>); high/low predation Trinidadian guppies, <em>Poecilia reticulata</em> (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Fraser <em>et al.</em> 2015</a>); crab/wave ecotype periwinkles, <em>Littorina saxitilis</em> (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Westram <em>et al.</em> 2014</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Ravinet <em>et al.</em> 2016</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Kess <em>et al.</em> 2018</a>), and alpine/montane ecotypes of <em>Heliosperma pusillum</em> (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Trucchi <em>et al.</em> 2017</a>). We found overlapping outliers between demographic treatments and thus, that signals of convergent outliers are attainable for singularly used F<sub>ST</sub> and D<sub>XY</sub>, and combined outlier sets for F<sub>ST</sub> – Δπ, D<sub>XY</sub> – Δπ and F<sub>ST</sub> - D<sub>XY</sub>. Clustering of outliers was predominantly driven by the presence or absence of migration (with minimal overlap between clusters) (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7</a>).</p><p id="p-71">The overlap of quantile-based outliers between demographic treatments without migration and ineffective selection may be explained in part probabilistically. With migration restricted, AP and DP exhibit significantly elevated measures of divergence, as seen in <a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1</a>. This increase results in a normal distribution of F<sub>ST</sub> and D<sub>XY</sub> across neutral simulations without migration. In contrast, with migration, drift is limited and random recombination influences gene flow and subsequently variation in divergence. Distributions of divergence with migration under neutrality are therefore are heavily right tail-skewed (Figure S39). Spread of data increases with longer right tails, and density of data in each overlapping distribution is limited to the median and lower quantiles.</p><p id="p-72">We also expect some effect of different amounts of starting variation among genome regions following burn-ins. With gene flow restricted, this variation may promote overlap under neutral conditions given all demographic treatments are founded from a common burn-in. However, this feature of our simulations is analogous to the conserved landscapes of variation observed in natural genomes (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Burri 2017</a>; <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Vijay <em>et al.</em> 2017</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Stankowski <em>et al.</em> 2019</a>).</p><p id="p-73">Empirically, the influence of migration on outlier overlap has been observed in replicate pairs of parasitic and non-parasitic lampreys. As we show here, when comparing outliers from disconnected and connected parasitic/non-parasitic pairs, <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Rougemont <em>et al.</em> (2017)</a> recover greater numbers of overlapping outliers among comparisons of disconnected pairs than connected pairs. Overlapping outliers among connected pairs are however better correlated, which the authors suggest reflects selection.</p></div><div id="sec-12" class="subsection"><h3>Limitations of simulations</h3><p id="p-74">In our analyses, we grouped genomic regions within 16 unique demographic treatments, assuming the effects of reductions in population size and migration are equivalent for all regions. This may be unrepresentative of genomes sampled from the wild, in which gene flow and effective population size can vary across genomic regions through structural variation, variable recombination rate and BGS (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gossmann <em>et al.</em> 2011</a>). A prominent example of such a mechanism is the well-characterized consequence of reduced gene flow within inversions that carry locally adapted alleles. Assuming inversion variants are fixed between populations, gene flow across the locus is limited by the prevention of recombinant haplotypes and resistance to introgression (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kirkpatrick and Barton 2006</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Ravinet <em>et al.</em> 2017</a>). Recent genome scan studies have highlighted convergent outliers within inversions (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Jones <em>et al.</em> 2012b</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Nishikawa <em>et al.</em> 2015</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Morales <em>et al.</em> 2019</a>), confirming theoretical models regarding their role in shielding adaptive haplotypes from introgression during the adaptation process. Our simulations suggest that genes with reduced migration, and reduced N<sub>e</sub> as a consequence, will have inflated measures of divergence and differentiation relative to other genomic regions. Therefore, we predict this may have led to their over-representation in genome scan outliers, and increased potential to overlap across replicate populations. Thus, caution should be taken regarding the adaptive significance of these outliers relative to absolutely lower values of genetic divergence attained from regions outside of chromosomal rearrangements.</p><p id="p-75">By choosing to use a factorial design here, we have increased our understanding of the interplay between specific features of demographic history and multiple measures of population divergence. However, computational limitations constrained absolute population sizes to a maximum of 1,000 individuals and generations to a maximum of 10,000 (with 10,000 generation burn-in). To mitigate these constraints, we repeated the analysis over multiple mutation rates (and scaled recombination rate) to illustrate patterns over 100-fold variation of θ. In general, most trends were consistent, suggesting that results should be consistent across taxa of variable effective population size. However, certain patterns were exaggerated or dampened with increased or decreased mutation rate respectively. For instance, the overlap of outliers between no-migration treatments was exaggerated at our smallest level of population scaling (Figure S37), suggesting false-positive signals of convergence may be more likely with reduced N<sub>e</sub>. Further, patterns associated with D<sub>XY</sub> and selection vary according to population size, appearing delayed in smaller populations (Figure S7) and swamped by mutational input in larger populations (Figure S6). This latter effect can induce negative associations between selection and D<sub>XY</sub> with effective selection, as selection reduces local genetic variation that would otherwise increase D<sub>XY</sub>. It is well-documented that both N<sub>e</sub> (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Frankham 1995</a>) and mutation rate (μ) (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hodgkinson and Eyre-Walker 2011</a>) are highly variable across taxa, which suggests that applying knowledge of the relationships between measures of population differentiation will vary in nature.</p><p id="p-76">In addition, temporal variation within our simulations is confounded by the time at which there was a major shift in the DP population phenotype (Figure S40), and by extension when selective sweeps occur. This variation in timing is random with respect to adaptive mutations arising <em>de novo</em>, but is also influenced by demographic treatment. For example, treatments that experience founding bottlenecks are less likely to evolve using variation in the founder, increasing dependence on <em>de novo</em> mutations for adaptation. Additionally, variation in our DP size parameter modifies the per population number of new mutations per generation. This is also true for features of simulated genes, such as size of selection target. Predictable temporal variation in the time at which adaptation is likely to occur is a probable source of variance between measures of divergence and is particularly clear for Δπ, but this was controlled for in later modeling analyses as a fixed effect.</p><p id="p-77">A further consideration for these simulations concerns the architecture of the phenotype. Results reported here pertain to mutation effect sizes drawn from a distribution centered at 0 with σ = 1. This produces mutations of typically large effect, but was selected on the basis of phenotypic optima being reasonably distant, with a difference of 10. Thus, 99% of mutations in simulations produce phenotypic differences of less than a third the divergence distance of AP and DP phenotypes. There are numerous factors that influence the distribution of mutation effect sizes in nature, including: selection, mutation, drift, gene flow, extent of pleiotropy and distance to phenotypic optima (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Dittmar <em>et al.</em> 2016</a>), with no single expectation for natural systems as a result. The relatively large distance between optima in our simulations, as well as the rapid change in optima implemented in simulations (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Collins and De Meaux 2009</a>), likely gives increased importance to mutations of large effect. The interactions between mutation effect size and the results presented here are beyond the scope of the current study, but we did investigate the effect of reducing μ<sub>σ</sub> of mutation effect distributions to 0.1 (Supporting Information; Figures S40-45). Briefly, we see reductions in the strength of correlations and associations with selection with decreasing phenotypic effects of mutation, consistent with the notion of softer sweeps on small-effect loci. We also see increased variance in the amount of time taken for simulations to reach the Pheno<sub>Div</sub> optimum, which agrees with the probable importance of large effect loci in our standard dataset. We, however, retain strong positive associations between measures such as F<sub>ST</sub> and D<sub>XY</sub>, as we see in neutral simulations, as well as overlapping outliers linked to selection at the tail ends of Pheno<sub>Div</sub> distributions. Running the simulations in this way suggests that many of the patterns described here may be robust to scenarios with reduced mutation effect sizes.</p><p id="p-78">It is also important when translating these results to genomic data to consider how correlations between measures of divergence depend on the selection type used in simulations. For example, F<sub>ST</sub> and D<sub>XY</sub> are strongly positively correlated under neutrality without migration, but under the same demographic scenarios we observe a decay in the relationship between F<sub>ST</sub> and D<sub>XY</sub> when divergent selection is involved. Genomes of natural populations will include regions that are neutral or nearly neutral, under stabilizing selection around a common phenotypic optimum, or divergent between populations. Thus, the patterns described here may not apply to all genomic regions pooled together.</p></div></div><div class="section other2" id="sec-13"><h2 class="">Concluding remarks</h2><p id="p-79">We have used forward-in-time simulations to perform a factorial experiment in which we explored the relationships between three measures of genetic divergence, selection, and features of demographic history that are commonly variable in natural populations. In agreement with previous theoretical work, we found the reliance of measures of genetic divergence to identify loci under selection are dependent on population size and migration, and variable through time, with a notable lag in D<sub>XY</sub>. Furthermore, we provided novel comparisons between measures of genetic divergence that call into question the use of multiple measures to rule out false-positives. We also demonstrated that signals of convergent evolution across independent replicates can be driven by similar features of demographic history with minimal influences of selection, specifically, across replicate populations pairs that lack migration. Therefore, we strongly advise those using overlapping outlier scans to carefully consider the demographic context of their system to avoid false-positives. In particular, the presence or absence of migration between diverging populations is a key factor determining the informativeness of genetic variation for selection, and importantly shapes our expectations of outlier overlap among replicate population pairs. It is tempting to assume that replication in study design or analysis in the form of taking multiple measures of genetic divergence can reduce the risk of attaining false-positives. We hope to emphasize in this study that this is not always the case, as false-positives (<em>i.e.</em>, genome scan outliers that are not associated with regions under divergent selection) can be driven by non-random genomic or common demographic features that cannot be bypassed through replication. Moreover, many of the patterns we observe are variable through time, such that the relevant pitfalls of analyses will depend on the age of the populations being considered. It should thus also be important to estimate population splits, as a young replicate pair and older replicate pair with similar demographic histories should be expected to exhibit potentially different patterns of genetic variation.</p><p id="p-80">Recent simulation work by <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Quilodrán <em>et al.</em> (2019)</a>, has also emphasized the influence of genomic features and demography, and includes simulation software for estimating the distribution of genetic variation over user-defined chromosomes. Such an approach is particularly useful for systems with chromosome-level genome assemblies in order to gain a sense for how features such as recombination, gene density, and selection targets may produce false-positives under certain demographies. The software employed here, SLiM (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Haller and Messer 2019</a>), may also be used to this end, and the scripts accompanying this study will facilitate similar analyses over system-specific genome regions. Further, recent work on genomic landscapes of linked selection (<a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Stankowski <em>et al.</em> 2019</a>) has highlighted that much of the total variance of genetic divergence such as F<sub>ST</sub>, D<sub>XY,</sub> and π can be explained by the major principal component (PC1) over numerous pairwise comparisons. These population comparisons need not reflect divergent phenotypes, as PC1 reflects genomic features associated with diversity landscapes. Adopting this approach may be useful for systems lacking a chromosome-level genome assembly by estimating SNPs or regions with non-random elevated measures of divergence associated with genome features. Such SNPs or regions may be particularly prone to false-positives under certain demographic histories.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-81">The authors wish to thank all members of the Fraser lab for insightful discussions on this project, as well as members of the University of Exeter’s EB theme and the University of Sussex’s EBE group. Thanks to the University of Exeter’s ISCA HPC infrastructure for performing simulations. The authors would also like to thank Kai Zeng and Adam Eyre-Walker for feedback on earlier copies of this manuscript, and two anonymous reviewers for inciteful and helpful comments. This work was funded as part of a European Research Council grant (758382 GUPPYCon) awarded to BAF.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-82">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.11323088">https://doi.org/10.25387/g3.11323088</a>.</p></li><li class="fn" id="fn-2"><p id="p-83"><em>Communicating editor: J. Ross-Ibarra</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-10-18"><span class="received-label">Received </span>October 18, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-12-19"><span class="accepted-label">Accepted </span>December 19, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Whiting and Fraser</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.1" data-doi="10.1111/mec.14549"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ahrens</span>,  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. D.</span> <span class="cit-name-surname">Rymer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Stow</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bragg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Dillon</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The search for loci under selection: trends, biases and progress.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1342</span>–<span class="cit-lpage">1356</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.14549</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D27%26rft.spage%253D1342%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.14549%26rft_id%253Dinfo%253Apmid%252F29524276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.14549&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29524276&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.2" data-doi="10.1016/j.tig.2014.09.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bank</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. B.</span> <span class="cit-name-surname">Ewing</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ferrer-Admettla</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Foll</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Jensen</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Thinking too positive? Revisiting current methods of population genetic selection inference.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">540</span>–<span class="cit-lpage">546</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2014.09.010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D30%26rft.spage%253D540%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2014.09.010%26rft_id%253Dinfo%253Apmid%252F25438719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2014.09.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25438719&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.3" data-doi="10.1371/journal.pgen.1007796"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Belleghem</span>,  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Vangestel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">De Wolf</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">De Corte</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Most</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Evolution at two time frames: Polymorphisms from an ancient singular divergence event fuel contemporary parallel evolution.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-elocation-id">e1007796</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1007796</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D14%26rft.spage%253De1007796%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1007796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1007796&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.4" data-doi="10.1111/mec.14167"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burri</span>,  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Linked selection, demography and the evolution of correlated genomic landscapes in birds and beyond.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">3853</span>–<span class="cit-lpage">3856</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.14167</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D26%26rft.spage%253D3853%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.14167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.14167&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.5" data-doi="10.1017/S0016672300034029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Background selection and patterns of genetic diversity in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">149</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300034029</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB.%26rft.volume%253D68%26rft.issue%253D2%26rft.spage%253D131%26rft.epage%253D149%26rft.atitle%253DBackground%2Bselection%2Band%2Bpatterns%2Bof%2Bgenetic%2Bdiversity%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300034029%26rft_id%253Dinfo%253Apmid%252F8940902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300034029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8940902&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VU51800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.6" data-doi="10.1038/nrg2526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Effective population size and patterns of molecular evolution and variation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">205</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2526</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253D195%26rft.epage%253D205%26rft.atitle%253DFundamental%2Bconcepts%2Bin%2Bgenetics%253A%2Beffective%2Bpopulation%2Bsize%2Band%2Bpatterns%2Bof%2Bmolecular%2Bevolution%2Band%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2526%26rft_id%253Dinfo%253Apmid%252F19204717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19204717&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263457000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.7" data-doi="10.1093/oxfordjournals.molbev.a025953"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Measures of divergence between populations and the effect of forces that reduce variability.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">538</span>–<span class="cit-lpage">543</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/oxfordjournals.molbev.a025953</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB%26rft.volume%253D15%26rft.issue%253D5%26rft.spage%253D538%26rft.epage%253D543%26rft.atitle%253DMeasures%2Bof%2Bdivergence%2Bbetween%2Bpopulations%2Band%2Bthe%2Beffect%2Bof%2Bforces%2Bthat%2Breduce%2Bvariability%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a025953%26rft_id%253Dinfo%253Apmid%252F9580982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a025953&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9580982&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000073343800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Charlesworth</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. T.</span> <span class="cit-name-surname">Morgan</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">The pattern of neutral molecular variation under the background selection model.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">1619</span>–<span class="cit-lpage">1632</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCharlesworth%26rft.auinit1%253DD.%26rft.volume%253D141%26rft.issue%253D4%26rft.spage%253D1619%26rft.epage%253D1632%26rft.atitle%253DThe%2Bpattern%2Bof%2Bneutral%2Bmolecular%2Bvariation%2Bunder%2Bthe%2Bbackground%2Bselection%2Bmodel.%26rft_id%253Dinfo%253Apmid%252F8601499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0MS80LzE2MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzY3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.9" data-doi="10.1111/j.0014-3820.2002.tb00188.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. T.</span> <span class="cit-name-surname">Morgan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Charlesworth</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">The effect of deleterious mutations on neutral molecular variation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">1289</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB.%26rft.volume%253D134%26rft.issue%253D4%26rft.spage%253D1289%26rft.epage%253D1303%26rft.atitle%253DThe%2Beffect%2Bof%2Bdeleterious%2Bmutations%2Bon%2Bneutral%2Bmolecular%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0014-3820.2002.tb00188.x%26rft_id%253Dinfo%253Apmid%252F8375663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNC80LzEyODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzY3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.10" data-doi="10.1017/S0016672397002954"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nordborg</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Charlesworth</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The effects of local selection, balanced polymorphism and background selection on equilibrium patterns of genetic diversity in subdivided populations.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">174</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672397002954</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB.%26rft.volume%253D70%26rft.issue%253D2%26rft.spage%253D155%26rft.epage%253D174%26rft.atitle%253DThe%2Beffects%2Bof%2Blocal%2Bselection%252C%2Bbalanced%2Bpolymorphism%2Band%2Bbackground%2Bselection%2Bon%2Bequilibrium%2Bpatterns%2Bof%2Bgenetic%2Bdiversity%2Bin%2Bsubdivided%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672397002954%26rft_id%253Dinfo%253Apmid%252F9449192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672397002954&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9449192&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071240500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.11" data-doi="10.1111/j.1558-5646.2009.00770.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">De Meaux</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Adaptation to different rates of environmental change in Chlamydomonas.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">2952</span>–<span class="cit-lpage">2965</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.2009.00770.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DCollins%26rft.auinit1%253DS.%26rft.volume%253D63%26rft.issue%253D11%26rft.spage%253D2952%26rft.epage%253D2965%26rft.atitle%253DAdaptation%2Bto%2Bdifferent%2Brates%2Bof%2Benvironmental%2Bchange%2Bin%2BChlamydomonas.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2009.00770.x%26rft_id%253Dinfo%253Apmid%252F19619223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2009.00770.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19619223&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271031900015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.12" data-doi="10.1111/mec.14116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>,  <span class="cit-name-given-names">E. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A. C.</span> <span class="cit-name-surname">Uy</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic evidence for convergent evolution of a key trait underlying divergence in island birds.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">3760</span>–<span class="cit-lpage">3774</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.14116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D26%26rft.spage%253D3760%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.14116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.14116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.13" data-doi="10.1111/mec.12796"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cruickshank</span>,  <span class="cit-name-given-names">T. E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Hahn</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Reanalysis suggests that genomic islands of speciation are due to reduced diversity, not reduced gene flow.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">3133</span>–<span class="cit-lpage">3157</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12796</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D3133%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12796%26rft_id%253Dinfo%253Apmid%252F24845075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12796&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24845075&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000338014900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.14" data-doi="10.1038/nrg3425"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cutter</span>,  <span class="cit-name-given-names">A. D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. A.</span> <span class="cit-name-surname">Payseur</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic signatures of selection at linked sites: Unifying the disparity among species.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">262</span>–<span class="cit-lpage">274</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3425</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D14%26rft.spage%253D262%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3425%26rft_id%253Dinfo%253Apmid%252F23478346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3425&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23478346&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.15" data-doi="10.1098/rspb.2015.3065"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dittmar</span>,  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. G.</span> <span class="cit-name-surname">Oakley</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Conner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. A.</span> <span class="cit-name-surname">Gould</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. W.</span> <span class="cit-name-surname">Schemske</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Factors influencing the effect size distribution of adaptive substitutions.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">283</span>: <span class="cit-elocation-id">20153065</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rspb.2015.3065</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2015.3065%26rft_id%253Dinfo%253Apmid%252F27053750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2015.3065&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27053750&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.16" data-doi="10.1111/mec.14083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Doren</span>,  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Campagna</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Helm</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Illera</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. J.</span> <span class="cit-name-surname">Lovette</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Correlated patterns of genetic diversity and differentiation across an avian family.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">3982</span>–<span class="cit-lpage">3997</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.14083</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D26%26rft.spage%253D3982%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.14083%26rft_id%253Dinfo%253Apmid%252F28256062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.14083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28256062&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.17" data-doi="10.1098/rspb.2016.2756"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dutoit</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Vijay</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. F.</span> <span class="cit-name-surname">Mugal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Bossu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Burri</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Covariation in levels of nucleotide diversity in homologous regions of the avian genome long after completion of lineage sorting.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">284</span>: <span class="cit-elocation-id">20162756</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rspb.2016.2756</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2016.2756%26rft_id%253Dinfo%253Apmid%252F28202815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2016.2756&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28202815&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.18" data-doi="10.1038/nrg.2016.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>,  <span class="cit-name-given-names">H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Galtier</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Determinants of genetic diversity.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">422</span>–<span class="cit-lpage">433</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg.2016.58</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D17%26rft.spage%253D422%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg.2016.58%26rft_id%253Dinfo%253Apmid%252F27265362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg.2016.58&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27265362&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.19" data-doi="10.1111/jeb.13113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>,  <span class="cit-name-given-names">H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B. W.</span> <span class="cit-name-surname">Wolf</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Parallelism in genomic landscapes of differentiation, conserved genomic features and the role of linked selection.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1516</span>–<span class="cit-lpage">1518</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jeb.13113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D30%26rft.spage%253D1516%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjeb.13113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jeb.13113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.20" data-doi="10.1038/ncomms11693"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foote</span>,  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Vijay</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Ávila-Arcos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. W.</span> <span class="cit-name-surname">Baird</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Durban</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-culture coevolution promotes rapid divergence of killer whale ecotypes.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">11693</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms11693</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D7%26rft.spage%253D11693%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms11693%26rft_id%253Dinfo%253Apmid%252F27243207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms11693&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27243207&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.21" data-doi="10.1017/S0016672300034455"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frankham</span>,  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Effective population size/adult population size ratios in wildlife: a review.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">107</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300034455</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D66%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300034455%26rft_id%253Dinfo%253Apmid%252F18976539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300034455&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18976539&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995TD89900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.22" data-doi="10.1111/mec.13022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>,  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Künstner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. N.</span> <span class="cit-name-surname">Reznick</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Dreyer</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Weigel</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Population genomics of natural and experimental populations of guppies (<em>Poecilia reticulata</em>).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">408</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13022</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D24%26rft.spage%253D389%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>,  <span class="cit-name-given-names">B. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Whiting</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> What can be learned by scanning the genome for molecular convergence in wild populations?<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nyas.14177</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.24" data-doi="10.1534/genetics.111.132654"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gossmann</span>,  <span class="cit-name-given-names">T. I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Woolfit</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Eyre-Walker</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Quantifying the variation in the effective population size within a genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">1389</span>–<span class="cit-lpage">1402</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.132654</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132654%26rft_id%253Dinfo%253Apmid%252F21954163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4OS80LzEzODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8yLzY3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.25" data-doi="10.1093/molbev/msy228"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haller</span>,  <span class="cit-name-given-names">B. C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. W.</span> <span class="cit-name-surname">Messer</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">SLiM 3: Forward genetic simulations beyond the Wright–Fisher model.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">632</span>–<span class="cit-lpage">637</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msy228</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsy228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msy228&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.26" data-doi="10.1093/molbev/msz149"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hämälä</span>,  <span class="cit-name-given-names">T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Savolainen</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Genomic patterns of local adaptation under gene flow in Arabidopsis lyrata.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">2557</span>–<span class="cit-lpage">2571</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msz149</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D36%26rft.spage%253D2557%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsz149%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msz149&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.27"><div class="cit-metadata unstructured">Hartl, D. L., A. G. Clark, and A. G. Clark, 1997 <em>Principles of population genetics</em>. Sinauer associates Sunderland.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.28" data-doi="10.1371/journal.pgen.1000008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>,  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Visscher</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Data and theory point to mainly additive genetic variance for complex traits.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-elocation-id">e1000008</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253De1000008%26rft.epage%253De1000008%26rft.atitle%253DData%2Band%2Btheory%2Bpoint%2Bto%2Bmainly%2Badditive%2Bgenetic%2Bvariance%2Bfor%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000008%26rft_id%253Dinfo%253Apmid%252F18454194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18454194&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.29" data-doi="10.1086/688018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoban</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Kelley</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. E.</span> <span class="cit-name-surname">Lotterhos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. F.</span> <span class="cit-name-surname">Antolin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Bradburd</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Finding the Genomic Basis of Local Adaptation: Pitfalls, Practical Solutions, and Future Directions.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">379</span>–<span class="cit-lpage">397</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/688018</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D188%26rft.spage%253D379%26rft_id%253Dinfo%253Adoi%252F10.1086%252F688018%26rft_id%253Dinfo%253Apmid%252F27622873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/688018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27622873&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.30" data-doi="10.1038/nrg3098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hodgkinson</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Eyre-Walker</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Variation in the mutation rate across mammalian genomes.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">756</span>–<span class="cit-lpage">766</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3098</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DHodgkinson%26rft.auinit1%253DA.%26rft.volume%253D12%26rft.issue%253D11%26rft.spage%253D756%26rft.epage%253D766%26rft.atitle%253DVariation%2Bin%2Bthe%2Bmutation%2Brate%2Bacross%2Bmammalian%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3098%26rft_id%253Dinfo%253Apmid%252F21969038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3098&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21969038&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.31" data-doi="10.1371/journal.pgen.1000862"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>,  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Bassham</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. D.</span> <span class="cit-name-surname">Etter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Stiffler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Johnson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Population genomics of parallel adaptation in threespine stickleback using sequenced RAD tags.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e1000862</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000862</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHohenlohe%26rft.auinit1%253DP.%2BA.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253De1000862%26rft.epage%253De1000862%26rft.atitle%253DPopulation%2Bgenomics%2Bof%2Bparallel%2Badaptation%2Bin%2Bthreespine%2Bstickleback%2Busing%2Bsequenced%2BRAD%2Btags.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000862%26rft_id%253Dinfo%253Apmid%252F20195501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000862&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20195501&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.32" data-doi="10.1371/journal.pgen.1005397"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>,  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Slatkin</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. P.</span> <span class="cit-name-surname">Maddison</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Estimation of levels of gene flow from DNA sequence data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">583</span>–<span class="cit-lpage">589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHudson%26rft.auinit1%253DR.%2BR.%26rft.volume%253D132%26rft.issue%253D2%26rft.spage%253D583%26rft.epage%253D589%26rft.atitle%253DEstimation%2Bof%2Blevels%2Bof%2Bgene%2Bflow%2Bfrom%2BDNA%2Bsequence%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005397%26rft_id%253Dinfo%253Apmid%252F1427045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTMyLzIvNTgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi82NzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.33" data-doi="10.1111/mec.13792"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Irwin</span>,  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Alcaide</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. E.</span> <span class="cit-name-surname">Delmore</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Irwin</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. L.</span> <span class="cit-name-surname">Owens</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Recurrent selection explains parallel evolution of genomic regions of high relative but low absolute differentiation in a ring species.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">4488</span>–<span class="cit-lpage">4507</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13792</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D4488%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13792&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.34" data-doi="10.1016/j.cub.2011.11.045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. F.</span> <span class="cit-name-surname">Chan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Schmutz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Grimwood</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. D.</span> <span class="cit-name-surname">Brady</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">A genome-wide SNP genotyping array reveals patterns of global and repeated species-pair divergence in sticklebacks.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">90</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cub.2011.11.045</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DJones%26rft.auinit1%253DF.%2BC.%26rft.volume%253D22%26rft.issue%253D1%26rft.spage%253D83%26rft.epage%253D90%26rft.atitle%253DA%2Bgenome-wide%2BSNP%2Bgenotyping%2Barray%2Breveals%2Bpatterns%2Bof%2Bglobal%2Band%2Brepeated%2Bspecies-pair%2Bdivergence%2Bin%2Bsticklebacks.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2011.11.045%26rft_id%253Dinfo%253Apmid%252F22197244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2011.11.045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22197244&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.35" data-doi="10.1038/nature10944"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. G.</span> <span class="cit-name-surname">Grabherr</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. F.</span> <span class="cit-name-surname">Chan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Russell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Mauceli</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">The genomic basis of adaptive evolution in threespine sticklebacks.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">484</span>: <span class="cit-fpage">55</span>–<span class="cit-lpage">61</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature10944</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJones%26rft.auinit1%253DF.%2BC.%26rft.volume%253D484%26rft.issue%253D7392%26rft.spage%253D55%26rft.epage%253D61%26rft.atitle%253DThe%2Bgenomic%2Bbasis%2Bof%2Badaptive%2Bevolution%2Bin%2Bthreespine%2Bsticklebacks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10944%26rft_id%253Dinfo%253Apmid%252F22481358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10944&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22481358&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302343400033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kess</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Galindo</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. G.</span> <span class="cit-name-surname">Boulding</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genomic divergence between Spanish <em>Littorina saxatilis</em> ecotypes unravels limited admixture and extensive parallelism associated with population history.</span> <abbr class="cit-jnl-abbrev">Int. J. Bus. Innov. Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">8311</span>–<span class="cit-lpage">8327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BBus.%2BInnov.%2BRes.%26rft.volume%253D17%26rft.spage%253D8311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.37" data-doi="10.1534/genetics.105.047985"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Barton</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Chromosome inversions, local adaptation and speciation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">434</span>. Erratum 208: 433.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.047985</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.047985%26rft_id%253Dinfo%253Apmid%252F16204214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTczLzEvNDE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMi82NzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.38" data-doi="10.1371/journal.pone.0169087"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Künstner</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hoffmann</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. A.</span> <span class="cit-name-surname">Fraser</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. A.</span> <span class="cit-name-surname">Kottler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Sharma</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The genome of the trinidadian guppy, Poecilia reticulata, and variation in the Guanapo population.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e0169087</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0169087</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0169087%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0169087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0169087&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.39"><div class="cit-metadata unstructured">Malinsky, M., R. J. Challis, A. M. Tyers, S. Schiffels, Y. Terai <em>et al.</em>, 2015 Genomic islands of speciation separate cichlid ecomorphs in an East African crater lake. Science (80). 350: 1493–1498. <a href="https://doi.org/10.1126/science.aac9927">https://doi.org/10.1126/science.aac9927</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.40" data-doi="10.1111/mec.15197"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matthey‐Doret</span>,  <span class="cit-name-given-names">R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Whitlock</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Background selection and FST: consequences for detecting local adaptation.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">3902</span>–<span class="cit-lpage">3914</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.15197</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.15197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.15197&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.41" data-doi="10.1093/molbev/msy051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meier</span>,  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Marques</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. E.</span> <span class="cit-name-surname">Wagner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Excoffier</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Seehausen</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genomics of parallel ecological speciation in Lake Victoria cichlids.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">1489</span>–<span class="cit-lpage">1506</span>. Erratum: 2594.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msy051</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsy051%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msy051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.42" data-doi="10.1126/sciadv.aav9963"><div class="cit-metadata"><cite>Morales, H. E., R. Faria, K. Johannesson, T. Larsson, M. Panova <em>et al.</em>, 2019 Genomic architecture of parallel ecological divergence: beyond a single environmental contrast. Sci. Adv. 5: eaav9963.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/sciadv.aav9963</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DMorales%26rft.auinit1%253DH.%2BE.%26rft.volume%253D5%26rft.issue%253D12%26rft.spage%253Deaav9963%26rft.epage%253Deaav9963%26rft.atitle%253DGenomic%2Barchitecture%2Bof%2Bparallel%2Becological%2Bdivergence%253A%2BBeyond%2Ba%2Bsingle%2Benvironmental%2Bcontrast%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.aav9963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMzoiNS8xMi9lYWF2OTk2MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzIvNjc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-book" id="cit-10.2.677.43" data-doi="10.7312/nei-92038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-source">Molecular evolutionary genetics</span>, <span class="cit-publ-name">Columbia university press</span>, <span class="cit-publ-loc">NY</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7312/nei-92038</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bevolutionary%2Bgenetics%26rft_id%253Dinfo%253Adoi%252F10.7312%252Fnei-92038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7312/nei-92038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.44" data-doi="10.1038/ng.3241"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nishikawa</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Iijima</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Kajitani</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Yamaguchi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Ando</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A genetic mechanism for female-limited Batesian mimicry in <em>Papilio</em> butterfly.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">405</span>–<span class="cit-lpage">409</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3241</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D405%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3241%26rft_id%253Dinfo%253Apmid%252F25751626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3241&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25751626&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.45" data-doi="10.1111/j.1365-294X.2008.03946.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nosil</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Funk</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Ortiz-Barrientos</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Divergent selection and heterogeneous genomic divergence.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">402</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2008.03946.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DNosil%26rft.auinit1%253DP.%26rft.volume%253D18%26rft.issue%253D3%26rft.spage%253D375%26rft.epage%253D402%26rft.atitle%253DDivergent%2Bselection%2Band%2Bheterogeneous%2Bgenomic%2Bdivergence.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2008.03946.x%26rft_id%253Dinfo%253Apmid%252F19143936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2008.03946.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19143936&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262517400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.46" data-doi="10.1111/2041-210X.13324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quilodrán</span>,  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Ruegg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. T.</span> <span class="cit-name-surname">Sendell‐Price</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Anderson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Coulson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">The multiple population genetic and demographic routes to islands of genomic divergence.</span> <abbr class="cit-jnl-abbrev">Methods Ecol. Evol.</abbr> 11: 762658.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/2041-210X.13324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1111%252F2041-210X.13324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/2041-210X.13324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.47"><div class="cit-metadata unstructured">R Core Team, 2016 R: A Language and Environment for Statistical Computing. R Found. Stat. Comput. Vienna Austria 0: {ISBN} 3–900051–07–0.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.48" data-doi="10.1111/jeb.13047"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ravinet</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Faria</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. K.</span> <span class="cit-name-surname">Butlin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Galindo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Bierne</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Interpreting the genomic landscape of speciation: a road map for finding barriers to gene flow.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1450</span>–<span class="cit-lpage">1477</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jeb.13047</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D30%26rft.spage%253D1450%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjeb.13047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jeb.13047&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.49" data-doi="10.1111/mec.13332"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ravinet</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Westram</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Johannesson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Butlin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">André</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Shared and nonshared genomic divergence in parallel ecotypes of <em>Littorina saxatilis</em> at a local scale.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">305</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13332</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D287%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13332&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.50"><div class="cit-metadata unstructured">Reid, N. M., D. A. Proestou, B. W. Clark, W. C. Warren, J. K. Colbourne <em>et al.</em>, 2016 The genomic landscape of rapid repeated evolutionary adaptation to toxic pollution in wild fish. Science (80). 354: 1305–1308. <a href="https://doi.org/10.1126/science.aah4993">https://doi.org/10.1126/science.aah4993</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reznick</span>,  <span class="cit-name-given-names">D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Endler</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">The impact of predation on life history evolution in Trinidadian guppies (<em>Poecilia reticulata</em>).</span> <abbr class="cit-jnl-abbrev">Evolution (N. Y.)</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">160</span>–<span class="cit-lpage">177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2B%2528N.%2BY.%2529%26rft.volume%253D36%26rft.spage%253D160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.52" data-doi="10.1111/mec.12311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roda</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Ambrose</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. M.</span> <span class="cit-name-surname">Walter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. L.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Schaul</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic evidence for the parallel evolution of coastal forms in the <em>Senecio lautus</em> complex.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2941</span>–<span class="cit-lpage">2952</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12311</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D2941%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12311&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000319675300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.53" data-doi="10.1038/nature13685"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Romiguier</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Gayral</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ballenghien</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bernard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Cahais</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparative population genomics in animals uncovers the determinants of genetic diversity.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">515</span>: <span class="cit-fpage">261</span>–<span class="cit-lpage">263</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature13685</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D515%26rft.spage%253D261%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13685%26rft_id%253Dinfo%253Apmid%252F25141177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13685&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25141177&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.54" data-doi="10.1111/mec.13664"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rougemont</span>,  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Gagnaire</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Perrier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Genthon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Besnard</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Inferring the demographic history underlying parallel genomic divergence among pairs of parasitic and nonparasitic lamprey ecotypes.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">142</span>–<span class="cit-lpage">162</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13664</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13664&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.55" data-doi="10.1371/journal.pgen.1000495"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sella</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Petrov</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Przeworski</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Andolfatto</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Pervasive natural selection in the <em>Drosophila</em> genome?</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-elocation-id">e1000495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000495</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSella%26rft.auinit1%253DG.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253De1000495%26rft.epage%253De1000495%26rft.atitle%253DPervasive%2Bnatural%2Bselection%2Bin%2Bthe%2BDrosophila%2Bgenome%253F%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000495%26rft_id%253Dinfo%253Apmid%252F19503600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000495&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19503600&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.56" data-doi="10.1017/S0016672300014634"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">J. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Haigh</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The hitch-hiking effect of a favourable gene.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">35</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300014634</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DSmith%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D35%26rft.atitle%253DThe%2Bhitch-hiking%2Beffect%2Bof%2Ba%2Bfavourable%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300014634%26rft_id%253Dinfo%253Apmid%252F4407212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300014634&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4407212&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1974T216900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.57"><div class="cit-metadata unstructured">Soria-Carrasco, V., Z. Gompert, A. A. Comeault, T. E. Farkas, T. L. Parchman <em>et al.</em>, 2014 Stick insect genomes reveal natural selection’s role in parallel speciation. Science (80). 344: 738–742. <a href="https://doi.org/10.1126/science.1252136">https://doi.org/10.1126/science.1252136</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.58" data-doi="10.1371/journal.pbio.3000391"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stankowski</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Chase</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Fuiten</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. F.</span> <span class="cit-name-surname">Rodrigues</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. L.</span> <span class="cit-name-surname">Ralph</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Widespread selection and gene flow shape the genomic landscape during a radiation of monkeyflowers.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-elocation-id">e3000391</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.3000391</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D17%26rft.spage%253De3000391%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.3000391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.3000391&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.59" data-doi="10.1038/ncomms6770"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tine</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Kuhl</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.-A.</span> <span class="cit-name-surname">Gagnaire</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Louro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Desmarais</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">European sea bass genome and its variation provide insights into adaptation to euryhalinity and speciation.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">5770</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms6770</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D5770%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms6770%26rft_id%253Dinfo%253Apmid%252F25534655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms6770&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25534655&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.60" data-doi="10.1371/journal.pgen.1007387"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Torres</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. A.</span> <span class="cit-name-surname">Szpiech</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. D.</span> <span class="cit-name-surname">Hernandez</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Human demographic history has amplified the effects of background selection across the genome.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-elocation-id">e1007387</span>. Erratum 15: e1007898.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1007387</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D14%26rft.spage%253De1007387%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1007387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1007387&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.61" data-doi="10.1111/nph.14722"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trucchi</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Frajman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. H. A.</span> <span class="cit-name-surname">Haverkamp</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Schönswetter</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Paun</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic analyses suggest parallel ecological divergence in <em>Heliosperma pusillum</em> (Caryophyllaceae).</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">216</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">278</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.14722</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D216%26rft.spage%253D267%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.14722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.14722&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.62" data-doi="10.1371/journal.pbio.0030285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>,  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Hahn</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. V.</span> <span class="cit-name-surname">Nuzhdin</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genomic islands of speciation in <em>Anopheles gambiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-elocation-id">e285</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.0030285</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DTurner%26rft.auinit1%253DT.%2BL.%26rft.volume%253D3%26rft.issue%253D9%26rft.spage%253De285%26rft.epage%253De285%26rft.atitle%253DGenomic%2Bislands%2Bof%2Bspeciation%2Bin%2BAnopheles%2Bgambiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0030285%26rft_id%253Dinfo%253Apmid%252F16076241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0030285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16076241&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.63" data-doi="10.1111/mec.14195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vijay</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Weissensteiner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Burri</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Kawakami</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Ellegren</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomewide patterns of variation in genetic diversity are shared among populations, species and higher-order taxa.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">4284</span>–<span class="cit-lpage">4295</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.14195</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D26%26rft.spage%253D4284%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.14195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.14195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.64" data-doi="10.1111/mec.14701"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>,  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. P.</span> <span class="cit-name-surname">Erb</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Beever</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Russello</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Adaptive population divergence and directional gene flow across steep elevational gradients in a climate‐sensitive mammal.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2512</span>–<span class="cit-lpage">2528</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.14701</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D27%26rft.spage%253D2512%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.14701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.14701&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.65"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>,  <span class="cit-name-given-names">B. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Cockerham</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Estimating F‐statistics for the analysis of population structure.</span> <abbr class="cit-jnl-abbrev">Evolution (N. Y.)</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1358</span>–<span class="cit-lpage">1370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2B%2528N.%2BY.%2529%26rft.volume%253D38%26rft.spage%253D1358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.66" data-doi="10.1111/mec.12883"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Westram</span>,  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Galindo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Alm Rosenblad</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Grahame</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Panova</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Do the same genes underlie parallel phenotypic divergence in different <em>Littorina saxatilis</em> populations?</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">4603</span>–<span class="cit-lpage">4616</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12883</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D4603%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12883&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.67" data-doi="10.1111/j.1558-5646.2011.01277.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yeaman</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. P.</span> <span class="cit-name-surname">Otto</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Establishment and maintenance of adaptive genetic divergence under migration, selection, and drift.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">2123</span>–<span class="cit-lpage">2129</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.2011.01277.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DYeaman%26rft.auinit1%253DS.%26rft.volume%253D65%26rft.issue%253D7%26rft.spage%253D2123%26rft.epage%253D2129%26rft.atitle%253DEstablishment%2Band%2Bmaintenance%2Bof%2Badaptive%2Bgenetic%2Bdivergence%2Bunder%2Bmigration%252C%2Bselection%252C%2Band%2Bdrift.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2011.01277.x%26rft_id%253Dinfo%253Apmid%252F21729066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2011.01277.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21729066&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292447000026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-10.2.677.68" data-doi="10.1038/hdy.2012.102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>,  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A coalescent model of background selection with recombination, demography and variation in selection coefficients.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">363</span>–<span class="cit-lpage">371</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2012.102</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D110%26rft.spage%253D363%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2012.102%26rft_id%253Dinfo%253Apmid%252F23188176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2012.102&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23188176&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F2%2F677.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316552300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/2/677.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/2/665" title="Quantitative Genomic Dissection of Soybean Yield Components" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/2/695" title="Genomic Predictive Ability for Foliar Nutritive Traits in Perennial Ryegrass" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/2"><span>Volume 10 Issue 2, February 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/2.cover-source.jpg?itok=LSawHOSr" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/2/677" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-6BO1_yCc7FBJFkbPR4bZgptggPHTdAbL1kB-RKmhNLk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/2/677" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/2/677" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/2/677" class="active" data-icon-position="" data-hide-link-title="0">Contingent Convergence: The Ability To Detect Convergent Genomic Evolution Is Dependent on Population Size and Migration</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495540" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-QnO1J8tZxIV8je3_5x6VeAPYWX5U-D0ylOMFYgYBwNM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/2/677.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/2/677" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-MejuFVEUnDO7BOo3nqE_-YuabZRfpUHpLviq2PPApOE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/2/677" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/2/677.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495540" id="node495540--2" data-pisa="ggg;10/2/677" data-pisa-master="ggg;g3.119.400970" data-apath="/ggg/10/2/677.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Contingent Convergence: The Ability To Detect Convergent Genomic Evolution Is Dependent on Population Size and Migration</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-8936-4991" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>James R. Whiting</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5905-9056" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bonnie A. Fraser</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">677-693;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400970 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495540--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">James R. Whiting </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biosciences, University of Exeter, Geoffrey Pope Building, Exeter, EX4 4QD</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BR.%2BWhiting%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whiting%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJames%2BR.%2BWhiting%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8936-4991" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for James R. Whiting</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.whiting2@exeter.ac.uk" class="" data-icon-position="" data-hide-link-title="0">j.whiting2@exeter.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bonnie A. Fraser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biosciences, University of Exeter, Geoffrey Pope Building, Exeter, EX4 4QD</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bonnie%2BA.%2BFraser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fraser%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABonnie%2BA.%2BFraser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5905-9056" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bonnie A. Fraser</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/2/677" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/2/677&amp;title=Contingent%20Convergence%3A%20The%20Ability%20To%20Detect%20Convergent%20Genomic%20Evolution%20Is%20Dependent%20on%20Population%20Size%20and%20Migration" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/2/677&amp;title=Contingent%20Convergence%3A%20The%20Ability%20To%20Detect%20Convergent%20Genomic%20Evolution%20Is%20Dependent%20on%20Population%20Size%20and%20Migration" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/2/677&amp;title=Contingent%20Convergence%3A%20The%20Ability%20To%20Detect%20Convergent%20Genomic%20Evolution%20Is%20Dependent%20on%20Population%20Size%20and%20Migration" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/2/677&amp;text=Contingent%20Convergence%3A%20The%20Ability%20To%20Detect%20Convergent%20Genomic%20Evolution%20Is%20Dependent%20on%20Population%20Size%20and%20Migration" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/2/677&amp;title=Contingent%20Convergence%3A%20The%20Ability%20To%20Detect%20Convergent%20Genomic%20Evolution%20Is%20Dependent%20on%20Population%20Size%20and%20Migration" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/2/677&amp;t=Contingent%20Convergence%3A%20The%20Ability%20To%20Detect%20Convergent%20Genomic%20Evolution%20Is%20Dependent%20on%20Population%20Size%20and%20Migration" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/2/677&amp;title=Contingent%20Convergence%3A%20The%20Ability%20To%20Detect%20Convergent%20Genomic%20Evolution%20Is%20Dependent%20on%20Population%20Size%20and%20Migration" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/2/677&amp;title=Contingent%20Convergence%3A%20The%20Ability%20To%20Detect%20Convergent%20Genomic%20Evolution%20Is%20Dependent%20on%20Population%20Size%20and%20Migration" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495540/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495540" id="node495540--3" data-pisa="ggg;10/2/677" data-pisa-master="ggg;g3.119.400970" data-apath="/ggg/10/2/677.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Contingent Convergence: The Ability To Detect Convergent Genomic Evolution Is Dependent on Population Size and Migration</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-8936-4991" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>James R. Whiting</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5905-9056" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bonnie A. Fraser</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">677-693;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400970 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495540--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">James R. Whiting </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biosciences, University of Exeter, Geoffrey Pope Building, Exeter, EX4 4QD</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BR.%2BWhiting%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whiting%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJames%2BR.%2BWhiting%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8936-4991" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for James R. Whiting</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.whiting2@exeter.ac.uk" class="" data-icon-position="" data-hide-link-title="0">j.whiting2@exeter.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bonnie A. Fraser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biosciences, University of Exeter, Geoffrey Pope Building, Exeter, EX4 4QD</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bonnie%2BA.%2BFraser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fraser%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABonnie%2BA.%2BFraser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5905-9056" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bonnie A. Fraser</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=677&amp;rft.epage=693&amp;rft.atitle=Contingent%20Convergence%3A%20The%20Ability%20To%20Detect%20Convergent%20Genomic%20Evolution%20Is%20Dependent%20on%20Population%20Size%20and%20Migration&amp;rft.volume=10&amp;rft.issue=2&amp;rft.date=2020-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Whiting%2C+James+R.&amp;rft.au=Fraser%2C+Bonnie+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495540/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495540/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495540/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495540/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495540/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495540/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495540/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495540/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495540/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495540/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495540/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495540/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/2/677" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Concluding remarks</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/2/677.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/2/677.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/2/677.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
