<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/1/2/131" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/1/2/131.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/1/2/131.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/1/2/131.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2.cover.gif" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/1/2/131/embed/mml-math-13.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/1/2/131.atom" />
<meta name="HW.pisa" content="ggg;1/2/131" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Quantitative Genetics of Genomic Imprinting: A Comparison of Simple Variance Derivations, the Effects of Inbreeding, and Response to Selection" />
<meta name="DC.Identifier" content="10.1534/g3.111.000042" />
<meta name="DC.Date" content="2011-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2011 Santure, Spencer. This is an open access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The level of expression of an imprinted gene is dependent on the sex of the parent from which it was inherited. As a result, reciprocal heterozygotes in a population may have different mean phenotypes for quantitative traits. Using standard quantitative genetic methods for deriving breeding values, population variances, and covariances between relatives, we demonstrate that although these approaches are equivalent under Mendelian expression, this equivalence is lost when genomic imprinting is acting. Imprinting introduces both parent-of-origin-dependent and generation-dependent effects that result in differences in the way additive and dominance effects are defined for the various approaches. Further, imprinting creates a covariance between additive and dominance terms absent under Mendelian expression, but the expression for this covariance cannot be derived using a number of the standard approaches for defining additive and dominance terms. Inbreeding also generates such a covariance, and we demonstrate that a modified method for partitioning variances can easily accommodate both inbreeding and imprinting. As with inbreeding, the concept of breeding values has no useful meaning for an imprinted trait. Finally, we derive the expression for the response to selection under imprinting, and conclude that the response to selection for an imprinted trait cannot be predicted from the breeder’s equation, even when there is no dominance." />
<meta name="DC.Contributor" content="Anna W. Santure" />
<meta name="DC.Contributor" content="Hamish G. Spencer" />
<meta name="article:published_time" content="2011-07-01" />
<meta name="article:section" content="Investigation" />
<meta name="citation_title" content="Quantitative Genetics of Genomic Imprinting: A Comparison of Simple Variance Derivations, the Effects of Inbreeding, and Response to Selection" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The level of expression of an imprinted gene is dependent on the sex of the parent from which it was inherited. As a result, reciprocal heterozygotes in a population may have different mean phenotypes for quantitative traits. Using standard quantitative genetic methods for deriving breeding values, population variances, and covariances between relatives, we demonstrate that although these approaches are equivalent under Mendelian expression, this equivalence is lost when genomic imprinting is acting. Imprinting introduces both parent-of-origin-dependent and generation-dependent effects that result in differences in the way additive and dominance effects are defined for the various approaches. Further, imprinting creates a covariance between additive and dominance terms absent under Mendelian expression, but the expression for this covariance cannot be derived using a number of the standard approaches for defining additive and dominance terms. Inbreeding also generates such a covariance, and we demonstrate that a modified method for partitioning variances can easily accommodate both inbreeding and imprinting. As with inbreeding, the concept of breeding values has no useful meaning for an imprinted trait. Finally, we derive the expression for the response to selection under imprinting, and conclude that the response to selection for an imprinted trait cannot be predicted from the breeder’s equation, even when there is no dominance.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2011/07/01" />
<meta name="citation_mjid" content="ggg;1/2/131" />
<meta name="citation_id" content="1/2/131" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/1/2/131" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/1/2/131.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/1/2/131.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/1/2/131.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.000042" />
<meta name="citation_pmid" content="22384325" />
<meta name="citation_volume" content="1" />
<meta name="citation_issue" content="2" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigation" />
<meta name="citation_firstpage" content="131" />
<meta name="citation_lastpage" content="142" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Anna W. Santure" />
<meta name="citation_author_institution" content="Allan Wilson Centre for Molecular Ecology and Evolution, University of Otago, Dunedin, New Zealand" />
<meta name="citation_author_institution" content="Department of Zoology, University of Otago, Dunedin, New Zealand" />
<meta name="citation_author_email" content="asanture@gmail.com" />
<meta name="citation_author" content="Hamish G. Spencer" />
<meta name="citation_author_institution" content="Allan Wilson Centre for Molecular Ecology and Evolution, University of Otago, Dunedin, New Zealand" />
<meta name="citation_author_institution" content="Department of Zoology, University of Otago, Dunedin, New Zealand" />
<meta name="citation_author_institution" content="National Research Centre for Growth and Development, University of Otago, Dunedin, New Zealand" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=F. Dai;citation_author=D. E. Weeks;citation_title=Ordered genotypes: an extended ITO method and a general formula for genetic covariance.;citation_pages=1035-1045;citation_volume=78;citation_year=2006;citation_issue=6;citation_pmid=16685653;citation_doi=10.1086/504045" />
<meta name="citation_reference" content="citation_journal_title=Livest. Prod. Sci.;citation_author=D.-J. de Koning;citation_author=A. P. Rattink;citation_author=B. Harlizius;citation_author=M. A. M. Groenen;citation_author=E. W. Brascamp;citation_title=Detection and characterization of quantitative trait loci for growth and reproduction traits in pigs.;citation_pages=185-198;citation_volume=72;citation_year=2001;citation_doi=10.1016/S0301-6226(01)00226-3" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D.-J. de Koning;citation_author=A. P. Rattink;citation_author=B. Harlizius;citation_author=J. A. M. van Arendonk;citation_author=E. W. Brascamp;citation_title=Genome-wide scan for body composition in pigs reveals important role of imprinting.;citation_volume=97;citation_year=2000;citation_pmid=10859367;citation_doi=10.1073/pnas.140216397" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. G. de Vries;citation_author=R. Kerr;citation_author=B. Tier;citation_author=T. Long;citation_author=T. H. E. Meuwissen;citation_title=Gametic imprinting effects on rate and composition of pig growth.;citation_pages=1037-1042;citation_volume=88;citation_year=1994;citation_pmid=24186259;citation_doi=10.1007/BF00220813" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=T. M. DeChiara;citation_author=E. J. Robertson;citation_author=A. Efstratiadis;citation_title=Parental imprinting of the mouse insulin-like growth factor II gene.;citation_pages=849-859;citation_volume=64;citation_year=1991;citation_issue=4;citation_pmid=1997210;citation_doi=10.1016/0092-8674(91)90513-X" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=T. H. Engellandt;citation_author=B. Tier;citation_title=Genetic variances due to imprinted genes in cattle.;citation_pages=154-165;citation_volume=119;citation_year=2002;citation_doi=10.1046/j.1439-0388.2002.00323.x" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=A. Essl;citation_author=K. Voith;citation_title=Genomic imprinting effects on dairy- and fitness-related traits in cattle.;citation_pages=182-189;citation_volume=119;citation_year=2002;citation_doi=10.1046/j.1439-0388.2002.00334.x" />
<meta name="citation_reference" content="citation_title=Mathematical Population Genetics;citation_author=W. J. Ewens;citation_year=1979" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=D. S. Falconer;citation_title=A note on Fisher's 'average effect' and 'average excess'.;citation_pages=337-347;citation_volume=46;citation_year=1985;citation_issue=3;citation_pmid=4092925;citation_doi=10.1017/S0016672300022825" />
<meta name="citation_reference" content="citation_title=Introduction to Quantitative Genetics;citation_author=D. S. Falconer;citation_author=T. F. C. Mackay;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Trans. R. Soc. Edinb.;citation_author=R. A. Fisher;citation_title=The correlation between relatives on the supposition of Mendelian inheritance.;citation_pages=399-433;citation_volume=52;citation_year=1918;citation_doi=10.1017/S0080456800012163" />
<meta name="citation_reference" content="citation_journal_title=Ann. Eugen.;citation_author=R. A. Fisher;citation_title=Average excess and average effect of a gene substitution.;citation_pages=53-63;citation_volume=11;citation_year=1941;citation_doi=10.1111/j.1469-1809.1941.tb02272.x" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=C. J. Goodnight;citation_title=Quantitative trait loci and gene interaction: the quantitative genetics of metapopulations.;citation_pages=587-598;citation_volume=84;citation_year=2000;citation_pmid=10849084;citation_doi=10.1046/j.1365-2540.2000.00698.x" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=R. Hager;citation_author=J. M. Cheverud;citation_author=J. B. Wolf;citation_title=Relative contribution of additive, dominance, and imprinting effects to phenotypic variation in body size and growth between divergent selection lines of mice.;citation_pages=1118-1128;citation_volume=63;citation_year=2009;citation_pmid=19187253;citation_doi=10.1111/j.1558-5646.2009.00638.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. L. Harris;citation_title=GENOTYPIC COVARIANCES BETWEEN INBRED RELATIVES;citation_pages=1319-1348;citation_volume=50;citation_year=1964;citation_issue=6;citation_pmid=14239792" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=J. S. Heywood;citation_title=An exact form of the breeder’s equation for the evolution of a quantitative trait under natural selection.;citation_pages=2287-2298;citation_volume=59;citation_year=2005;citation_pmid=16396170;citation_doi=10.1111/j.0014-3820.2005.tb00939.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=H. Hirooka;citation_author=D.-J. de Koning;citation_author=B. Harlizius;citation_author=J. A. M. van Arendonk;citation_author=A. P. Rattink;citation_title=A whole-genome scan for quantitative trait loci affecting teat number in pigs;citation_pages=2320-2326;citation_volume=79;citation_year=2001;citation_issue=9;citation_pmid=11583418" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. A. Knott;citation_author=L. Marklund;citation_author=C. S. Haley;citation_author=K. Andersson;citation_author=W. Davies;citation_title=Multiple marker mapping of quantitative trait loci in a cross between outbred wild boar and large white pigs.;citation_pages=1069-1080;citation_volume=149;citation_year=1998;citation_pmid=9611214" />
<meta name="citation_reference" content="citation_journal_title=Asian-Australas. J. Anim. Sci.;citation_author=H. K. Lee;citation_author=S. S. Lee;citation_author=T. H. Kim;citation_author=G. J. Jeon;citation_author=H. W. Jung;citation_title=Detection of imprinted Quantitative Trait Loci (QTL) for growth traits in pigs.;citation_pages=1087-1092;citation_volume=16;citation_year=2003;citation_doi=10.5713/ajas.2003.1087" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=C. C. Li;citation_author=L. Sacks;citation_title=BIOMETRICS;citation_pages=347-360;citation_volume=10;citation_year=1954;citation_doi=10.2307/3001590" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=P. P. Luedi;citation_author=F. S. Dietrich;citation_author=J. R. Weidman;citation_author=J. M. Bosko;citation_author=R. L. Jirtle;citation_title=Computational and experimental identification of novel human imprinted genes.;citation_volume=17;citation_year=2007;citation_pmid=18055845;citation_doi=10.1101/gr.6584707" />
<meta name="citation_reference" content="citation_title=Genetics and Analysis of Quantitative Traits;citation_author=M. Lynch;citation_author=B. Walsh;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=D. Milan;citation_author=J.-P. Bidanel;citation_author=N. Iannuccelli;citation_author=J. Riquet;citation_author=Y. Amigues;citation_title=Detection of quantitative trait loci for carcass composition traits in pigs.;citation_pages=705-728;citation_volume=34;citation_year=2002;citation_issue=6;citation_pmid=12486399;citation_doi=10.1051/gse:2002026" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=I. M. Morison;citation_author=J. P. Ramsay;citation_author=H. G. Spencer;citation_title=A census of mammalian imprinting.;citation_pages=457-465;citation_volume=21;citation_year=2005;citation_issue=8;citation_pmid=15990197;citation_doi=10.1016/j.tig.2005.06.008" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=T. Nagylaki;citation_author=Y. Lou;citation_title=Evolution under multiallelic migration-selection models.;citation_pages=21-40;citation_volume=72;citation_year=2007;citation_issue=1;citation_pmid=17470373;citation_doi=10.1016/j.tpb.2007.02.005" />
<meta name="citation_reference" content="citation_journal_title=Curr. Genomics;citation_author=A. K. Naumova;citation_author=S. Croteau;citation_title=Mechanisms of epigenetic variation: polymorphic imprinting.;citation_pages=417-429;citation_volume=5;citation_year=2004;citation_doi=10.2174/1389202043349183" />
<meta name="citation_reference" content="citation_journal_title=Development genes and evolution;citation_journal_abbrev=Dev Genes Evol;citation_author=M. J. O’Neill;citation_author=R. S. Ingram;citation_author=P. B. Vrana;citation_author=S. M. Tilghman;citation_title=Allelic expression of IGF2 in marsupials and birds.;citation_pages=18-20;citation_volume=210;citation_year=2000;citation_issue=1;citation_pmid=10603082;citation_doi=10.1007/PL00008182" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. R. Price;citation_title=Selection and covariance.;citation_pages=520-521;citation_volume=227;citation_year=1970;citation_issue=5257;citation_pmid=5428476;citation_doi=10.1038/227520a0" />
<meta name="citation_reference" content="citation_journal_title=Annals of human genetics;citation_journal_abbrev=Ann Hum Genet;citation_author=G. R. Price;citation_title=Extension of covariance selection mathematics.;citation_pages=485-490;citation_volume=35;citation_year=1972;citation_issue=4;citation_pmid=5073694;citation_doi=10.1111/j.1469-1809.1957.tb01874.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=R. Quintanilla;citation_author=D. Milan;citation_author=J.-P. Bidanel;citation_title=A further look at quantitative trait loci affecting growth and fatness in a cross between Meishan and Large White pig populations.;citation_pages=193-210;citation_volume=34;citation_year=2002;citation_issue=2;citation_pmid=12081807;citation_doi=10.1051/gse:2002003" />
<meta name="citation_reference" content="citation_journal_title=Journal of cellular biochemistry;citation_journal_abbrev=J Cell Biochem;citation_author=E. Rand;citation_author=H. Cedar;citation_title=Regulation of imprinting: A multi-tiered process.;citation_pages=400-407;citation_volume=88;citation_year=2003;citation_issue=2;citation_pmid=12520543;citation_doi=10.1002/jcb.10352" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=A. P. Rattink;citation_author=D. J. de Koning;citation_author=M. Faivre;citation_author=B. Harlizius;citation_author=J. A. M. van Arendonk;citation_title=Fine mapping and imprinting analysis for fatness trait QTLs in pigs.;citation_pages=656-661;citation_volume=11;citation_year=2000;citation_issue=8;citation_pmid=10920235;citation_doi=10.1007/s003350010117" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=I. Sandovici;citation_author=S. Kassovska-Bratinova;citation_author=J. C. Loredo-Osti;citation_author=M. Leppert;citation_author=A. Suarez;citation_title=Interindividual variability and parent of origin DNA methylation differences at specific human Alu elements.;citation_volume=14;citation_year=2005;citation_pmid=15972727;citation_doi=10.1093/hmg/ddi218" />
<meta name="citation_reference" content="citation_journal_title=Human Molecular Genetics;citation_journal_abbrev=Hum Mol Genet;citation_author=I. Sandovici;citation_author=M. Leppert;citation_author=P. R. Hawk;citation_author=A. Suarez;citation_author=Y. Linares;citation_title=Familial aggregation of abnormal methylation of parental alleles at the IGF2/H19 and IGF2R differentially methylated regions;citation_pages=1569-1578;citation_volume=12;citation_year=2003;citation_issue=13;citation_pmid=12812984;citation_doi=10.1093/hmg/ddg167" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. W. Santure;citation_author=H. G. Spencer;citation_title=Influence of mom and dad: Quantitative genetic models for maternal effects and genomic imprinting.;citation_volume=173;citation_year=2006;citation_pmid=16751674;citation_doi=10.1534/genetics.105.049494" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=L. R. Schaeffer;citation_author=B. W. Kennedy;citation_author=J. P. Gibson;citation_title=The Inverse of the Gametic Relationship Matrix;citation_pages=1266-1272;citation_volume=72;citation_year=1989;citation_issue=5;citation_doi=10.3168/jds.S0022-0302(89)79231-6" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. G. Spencer;citation_title=The Correlation Between Relatives on the Supposition of Genomic Imprinting;citation_pages=411-417;citation_volume=161;citation_year=2002;citation_issue=1;citation_pmid=12019254" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=A. Stella;citation_author=K. J. Stalder;citation_author=A. M. Saxton;citation_author=P. J. Boettcher;citation_title=Estimation of variances for gametic effects on litter size in Yorkshire and Landrace swine;citation_pages=2171-2178;citation_volume=81;citation_year=2003;citation_issue=9;citation_pmid=12968691" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=A. R. Templeton;citation_title=The general relationship between average effect and average excess.;citation_pages=69-70;citation_volume=49;citation_year=1987;citation_issue=1;citation_pmid=3569910;citation_doi=10.1017/S0016672300026756" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=B. Tier;citation_author=J. Solkner;citation_title=Analysing gametic variation with an animal model.;citation_pages=868-872;citation_volume=85;citation_year=1993;citation_pmid=24196062" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Quantitative Genetics of Genomic Imprinting: A Comparison of Simple Variance Derivations, the Effects of Inbreeding, and Response to Selection" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2.cover.gif" />
<meta name="twitter:description" content="The level of expression of an imprinted gene is dependent on the sex of the parent from which it was inherited. As a result, reciprocal heterozygotes in a population may have different mean phenotypes for quantitative traits. Using standard quantitative genetic methods for deriving breeding values, population variances, and covariances between relatives, we demonstrate that although these approaches are equivalent under Mendelian expression, this equivalence is lost when genomic imprinting is acting. Imprinting introduces both parent-of-origin-dependent and generation-dependent effects that result in differences in the way additive and dominance effects are defined for the various approaches. Further, imprinting creates a covariance between additive and dominance terms absent under Mendelian expression, but the expression for this covariance cannot be derived using a number of the standard approaches for defining additive and dominance terms. Inbreeding also generates such a covariance, and we demonstrate that a modified method for partitioning variances can easily accommodate both inbreeding and imprinting. As with inbreeding, the concept of breeding values has no useful meaning for an imprinted trait. Finally, we derive the expression for the response to selection under imprinting, and conclude that the response to selection for an imprinted trait cannot be predicted from the breeder’s equation, even when there is no dominance." />
<meta name="og-title" property="og:title" content="Quantitative Genetics of Genomic Imprinting: A Comparison of Simple Variance Derivations, the Effects of Inbreeding, and Response to Selection" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/1/2/131" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The level of expression of an imprinted gene is dependent on the sex of the parent from which it was inherited. As a result, reciprocal heterozygotes in a population may have different mean phenotypes for quantitative traits. Using standard quantitative genetic methods for deriving breeding values, population variances, and covariances between relatives, we demonstrate that although these approaches are equivalent under Mendelian expression, this equivalence is lost when genomic imprinting is acting. Imprinting introduces both parent-of-origin-dependent and generation-dependent effects that result in differences in the way additive and dominance effects are defined for the various approaches. Further, imprinting creates a covariance between additive and dominance terms absent under Mendelian expression, but the expression for this covariance cannot be derived using a number of the standard approaches for defining additive and dominance terms. Inbreeding also generates such a covariance, and we demonstrate that a modified method for partitioning variances can easily accommodate both inbreeding and imprinting. As with inbreeding, the concept of breeding values has no useful meaning for an imprinted trait. Finally, we derive the expression for the response to selection under imprinting, and conclude that the response to selection for an imprinted trait cannot be predicted from the breeder’s equation, even when there is no dominance." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2.cover.gif" />
    <title>Quantitative Genetics of Genomic Imprinting: A Comparison of Simple Variance Derivations, the Effects of Inbreeding, and Response to Selection | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__34Arr8QetTzezDWW4Etyv7gbgH-x-ZTNo3yO15Q-gHU__Mk-lyyG5WoK4SRy56eUq5eomxN7OGXjpDv-Ob1a1Q1I__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"CWINZtkNbiEjzG38689KM4RDzHj91D2mgZqAM2u8vPk","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459163","apath":"\/ggg\/1\/2\/131.atom","pisa":"ggg;1\/2\/131","ac":{"\/ggg\/1\/2\/131.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/1\/2\/131.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;1\/2\/131"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;1\/2\/131"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/1\/2\/131.abstract":true,"\/content\/1\/2\/131":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022xyL4yoH7sG+w3xtBY9a+Jeoz+swz8mDO6erkP0JmTRblTvofeWBAks8KUDrWSTPDvGrK\\\\\\\/S066x22fh+3O7ivOmBN9BNV\\\\\\\/O+oOB\\\\\\\/xNmWQKwN1QpdNVnNDvKJo1A93C2Rf5u3\\\\\\\/IG0431NsOdOCxN7gk0ZPAq8NjiciKUvkYgfQzoYVZgfvw9dZ+SKXCfNTnqwxFxqYIQd8xALILAQToXY\\\\\\\/9KaWTnZpsz\\\\\\\/70QnPZBJ9b054HehAxTyqHYWUGlsFjZAMQbzvfCBCwsUznGUkN8IG5PwmNMdsdh97WCGx1jc59qncjvw7J6Xrji0IR5u\\\\\\\/3BB68VUgIptUOBb6AUDXeXyOmR4o3FhvmS18N+S0cceuGYYF5Sq1WsVdOEiOOth7Mp9y\\\\\\\/JSmqCTEmVu7naxGNqIlxyTmgjA+phBbqLpuTU4J+xlTeoaCutLfncor6W5TwR1GhQVI2pde+L1ApxVS8QtlERQNIvbWKWucmeCX6wWBGVqIVGH962hgTT9poYfFpBLPfBoeq9qeq4OSuHujyinDiW4RgxcI8nUQPlfHMZnicuSDHqtw2kcJcELbvarkDtsk\\\\\\\/TGQOANRAl+GLw7LpTp8e3tUhEFefB2qIVOLlY+9YjEAwKUwlEX61tU0riWTsdRwyZoIqIjakiGPkF5nGEX44uMJNRxLhatxDNiv\\\\\\\/lPt1Stx1fHmADEV6LrmE33Njdth8ygFFRQYEEHUOYs95CreCbewXGxthSA3hSuAMCc8armOWmZb8TyXfmgf9XXQvnEK+TO3tLf6QAU5ralfHsm6ymHNx3c9Xk3ljAhkYcsXy7Y4zGr9Q2\\\\\\\/Gc8v2I7bFp7ujKL46yFRFWaJzbamcE+yD9ZwP+6nyUOgu7knHGaPVJUewlZdhNJj+wL8n3Vd0PtpwvuX2BnAIq2YRDnxACvp+jRWkSdvp6RMmocj+vjKINt3rweVnlpS2\\\\\\\/\\\\\\\/rETTlsmTqicuMHkZ6FGJDkD3E5Nq8\\\\\\\/YS9ZBw0nF6z0y\\\\\\\/W8XqEH5aOGxQXxVfdSJy0st5e1rWUfz+Zo\\\\\\\/xSbpbTMjxn5zLQ+lrX\\\\\\\/ol910oAyewebR4Q8kTycIMo99lJsirIkH79a1rLw2WdO1JFIkvV0kugISYAC\\\\\\\/lDvS2bFHd+HqQTXR6CQuJmBP\\\\\\\/CEMet5gqLPCJVdkfV\\\\\\\/UkbKBw6z8OCxZ7L9TKHN2mUQpiV8OiWfxO7nFHH1kyjQTl+Qqf1\\\\\\\/5xkdqDyXqI2WCTpYHUTW4ctjiURpvuVdi2HRTILdJYabo7rHiM0gUaUISusvrlbf+iErDezG6R5S\\\u0022,\\\u0022iv\\\u0022:\\\u0022H8n9CFB+sdpInNU\\\\\\\/ADGOkg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002274c3fd2c9fc9673b43ca706c70cb5cacc0ec5acaa13039a1678244c713708e68\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LIUQdhohUjVnOa9CJg3din33KjKDxAsL8ewHiklxGrMNB6eT\\\\\\\/LsmlTJQylgXvrEZ0JAEdjkvcbSxeK0qEJ8s4tAP62IgPk37rr2iDrMlkZ1L4GErhB\\\\\\\/NjnXVSaBmvj8RIm8vovXhcs102moMBfwlhQkNw\\\\\\\/1\\\\\\\/lmoqnwt8BYZD5BIUC8Haku0YOLvFSnktur1hJJqN+OW+cQvs5JKEl0ccH8gSn0HVKV7h5U+2xKmlLvh6I+jGcakRgiMXLjjHMdN\\\\\\\/h6ycnmy2NlKt7MZwrQvHAyyCUROrZ6F9EgSppnycii4p+WL7unGyPr474WP\\\\\\\/dlHPm43WuIj6csxjoiJ5pNRWzio6kPwwjffRtU5SWMKgvlerffMY7iI21u4513bzh4yXipeE7OUKbyblDigMPCA\\\\\\\/nb6KHbDD45UyNZW0suKgFeyJufumbg+KQnxRkHxJJ6XS6WsNtohyVakhendMzbUBKf2LX5z1MrsbHgdMou6XFqQutkIuegGC1L0H3odqgphA\\\\\\\/WY0voPQDnieOZRQa\\\\\\\/YdW0HM5rTZinWqtpx6OJvu977fWdBUekcIeGr+jfTGl1NIvXvkxXyAY3dwyhyedFygm4JcAnGLhnnu4HNwWl6WEDt3axK7B0XvxqV2IZN55r3s5kro2LTupnBNF8VP\\\\\\\/EWWyFW0ZK4iuTTGCLkeeeuo9BRGm3kiLlEmTsliWp9R\\\\\\\/HmSVrLt6hhgjp1JOyDKtpTMdBa9quogY6CFz9+Jze1j5kLtDuGBQg1IucSxu+NzyBYLXaieRCRJU63PHVMfkoL6X095Ur8pfg78XBvlJC65s6VkAOhXklD5F5EN2EFq9NLgJSfrjhM8pVqZhaOosz98y+yoB2LMpyYXzHsXcFp0qrOmINAq05srRB88V6pwnjzQyY1A0bvhmFwzNiLdf290nGTV3jCJ5ing6MlhbS6S2W0w6nGVheWfWSOH3h7+6qXD\\\\\\\/dqmjlc\\\\\\\/SCldn5Ts8XGw7cWrRbQML6CnSUBoH5bivwZTPKVw2N5BglPPnMjMh+qvroTZh8X+OlL8TQuspbkJWYkMSkBig+r57TxpZ7gOZPbmAhhM7WfWz2m5PbAZx0UO69p4MdqfEy395K6KytO+ehUT9jQE13bJjTjM3Fc5Zn4+ZmLGMgriY3LoykIB26vIoolZy1YHAymUaKXNiKp5IlmjZtlQPm\\\\\\\/RSpTuhv0gy4IoF7\\\\\\\/D0iQBVHRgaMC9Eh\\\\\\\/18pkLR2lssGU4Oh\\\\\\\/XX57+4Jxu12sOpegE9Lka2DxeLpwQzdXoyH5BUhcIdtuX5jwUBb6Fnn+0JqemOv63goR7qIDTQE4eZV4\\\\\\\/0VPpS2K+F\\\\\\\/4Oz9NU629dSQKJHuiK256hCNCZ8BHaVX5Dakl62nOSGvClcDhX\\\\\\\/Xa92M8xbANi8TK2Kdxv16O+x4803funvA5uC+u+vaSAgbOOXFIn15BeyPiQeMUCrZYPCXZsrWiVZ7CDRDJDrBNoH8PWGlY2rPtnCRTTTb53kChp0APxwA5qvy\\\\\\\/JnzJ+GFYeMOw7UuuA7CWvK6lUAq0RfGR6bDHkxJJT88eoJJvOLtqOaQTSH0wzgfX8PrYQhkI95sy8J\\\\\\\/cqtnb3ocP1so0Oh6xOKe41Qhf24ZBm2JyjqkaNr6mxu\\\\\\\/6\\\\\\\/9JyJsT1Jc+BME0MhYIxyQZXd8JBJn6e4\\\\\\\/oRpR1mwIzeQRidqnPEfEidZoqzXeOqBMDsBQ5vrCVDxbKTixSOqwF9JvScki+yNZpsbI5OI+fmjX5q8b7I\\\\\\\/5ghJ9bKk963oStVBM09tPVXFBsqpnKvOS39BDzLDiA1KuibhXtIodYfq1SwZUUeTLqJNTj339tM8PNg9+eDj7JDXsg3XOpgYHcm8rsd2b+De5Z7uZLv857ZuzOOrzi2DnIL4EdyZkpc+iueoAspHPlITBsAb\\\\\\\/ca9FkXDrs7ZKeZ1LqzH4FuuIdmhtcj\\\\\\\/xUEOcMnv0L2n\\\\\\\/OVN1Ej0QtU+i7Pdd6PsqB\\\\\\\/4clLHKgWqX2+dv\\\\\\\/lrNp3PF+WD+d4yj9Cmredf6YxgthDg3wUa0240CgF\\\\\\\/4s5NiaJoPT134NzhB6IqJ3IhKxazw99aL8Jxt85Umt5ioZIB4+86snV\\\\\\\/udEFa9gCRZsBbRr4v2svAeF9VMy+nFoGfrRtYjsP7r2zHmGr4bccS2oTi7cGpVXkYMYwEt8FizUVthN16El+VoFHQ\\\\\\\/\\\\\\\/HCFCtZ2LpjDbPRHebzWeapOxc+3HbWUWbmvpjsAVkVOqhM6yX\\\\\\\/Y5UwYrYjVap+vBhbbL9f2ecjHZ+3IFktzF917ddJyvLKKeNQkcVjbazNWVPdQcMgVu0FkEjsmK3nILMnW9E6suxTNIXcinzI1h2TwD\\\\\\\/l4VzxIkWHXoK0rRcVjdbICZ9p3hIC5e\\\\\\\/wWu9fuMlWO9dvltLu0D8G7PC9AFRcgbdIjoBdn70hBGF1kP6ysC0lJ+Z9KjUC1BxKFNxr2dvPhm0PATd9JupY7FdQ8Tn4yaqc2jbAoKz7wPLujNy4vNLwnuRiUyRsarMtZdQiFq4nnycUjZ+bWnDn1jYEvkOzkeZ2b2cNLyGk+k15oXH05NdzW+3e99sDn4A0QPzPp9ePws3xHOSLbolB6CfwV6wi2\\\\\\\/zTbPV7BohSCPJqd4oHrQP8FlXeCItdpaE7FOWj96fbl7vYf5FV7iYcFcAzKmsjo8taQk\\\\\\\/RK3347FOKne397zhauJGxuZvQ47chk\\\\\\\/8ktvvYRWUqVhlBYyoGlz\\\\\\\/zMkrNY5kby\\\\\\\/RYsPPyEEj69RkcS81P8SUoJZAVTtwbxgZzhG7nNA2jJ3ilwKBm6hfTQI\\\\\\\/l4ieMdp2Gvkb7M\\\\\\\/56oNSMd2hkxMh4JVGKS855C9zTSfxNifMTMngCIKdmfSYjRvJWMxIEw1cYMGbfeGHuT+AMi7KppfB1bcAJTLasN+GGME\\\\\\\/EMjp0h\\\\\\\/AifbP1dUY6q1L+ir9rpP0h+BWmyindAAE1Dn+YJMTUNMNnKQTmpa\\\\\\\/PA70d7Kz0+qGNYtiS54dd6SxzIndxXuh7uxmusDukFOac7Sskll9osjhQ7GpmeOT12pz+zD4DgyrySvYKUSam24b6qB\\\\\\\/4hjjjsOIkCegxpD4ae386QYWIMdi1V43aDrKXeV9qyz+Q2UPrZbKg5nwgl33jRuM4Ij0\\\\\\\/JAhYkBPQdZcqUjJnrkMukB5ArIP3d6vb5mLrW+k+AVBmshjohClJ2WMxrdxDDuaNmVc6a3y9N1c4h9zAvJTI9VxhOSCnHXjFPOCjpgbaXM4V\\\\\\\/UEicu0QNIatt52rr+q7hBR4Lr+3ADTijOSDmrTCmT\\\\\\\/CkL\\\\\\\/qkf\\\\\\\/Vs1Jeatir43id\\\\\\\/MDWCmRUFFq\\\\\\\/TaoKk+uJkDMA6\\\\\\\/J6rQktubBDO9qhCuV0aAwhile9qS49vxhLgs5E3xzuTEuP\\\\\\\/swJ\\\\\\\/LomjPiR1JGNBDwXY8jxq\\\\\\\/7BAnmzmH5S1guyyk6T2Wj+zH0+Q20lfmymP0c3+IkLUx\\\\\\\/Oq2MNGiDE9DK62rPfWwWP6k9gsefO8XS7Cbjmc2A6IVIq6YrhoXB1BMAfzMF5NUlyrVmMZa1ZCr7J8I7BAYmKogZL+RxasEg47WCmarOC6mv0mDQ9811nr9gOrET3RoGWoSVRlFGCp2ujmuOgzubloWW0WzQ3j9aE6m4eFGvxJISmFEj5Y009IXJBM6q3AfBDKwv0RlYBHhA1sCwcbmyn0EFxZOHxUMxAhkCFw9jLugb9rLGhxL\\\\\\\/jXq76QietgEg\\\\\\\/\\\\\\\/fMdCAYv5hT6oYSjaqzflMTj5H\\\\\\\/hie9zz1k\\\\\\\/PMQAMJDUK0sRv1J+5yO24uU7DTwSVCxbc+si7UWy5IM0Jsv+ma9H31ycvIZJ+\\\\\\\/YeixxbhH+O08seurT9mA1frzkZPtwHd6JSThuKEo2OovTDwWqNTHlXJMqxe3YcabXSr\\\\\\\/B04DDPPvG+yw6FBnUHywO6Js91z+k+3TnzbKOms+DgtrOebxCquG7QiXjfaBSzON4XPSrE0Pua\\\\\\\/pS4\\\\\\\/oBLgsNV40f9BuABB9jkvZM+cnyL+2V727RJmuyDPyBQRyonS1GgGfnBzKnbZoqZbbBr7YNIiOdHFsFVE7MCKbw0e0L+ygsqxA5STpuqvhkNsLNPsORbcY3bGy14gDDF848PzpsXpAujyPdud0MMqmh1pZxYgonizXtU4GyknHdSSqtEWV8idI40I011Qqi1xOdGeqCfZ9taK6apKTVW9n3\\\\\\\/9tBM2z1IPXfRrqbPTqYUyPHgLZ2vLNWTsHkKOYfNXAf+uLqGZiLjfS6kk4zrjeSY1nNAtemc9qheij7byyd9ywyuTvjxjz8wvm4p1FS9Uff2CeMpdpiSrLLcmxvZYJLMyOGjETh8fE5WjJcsnSS+Vl6w6041tOfLO1MZpAX+cloKwCsHLVI4fmm3V6ow1zoxonHQ6ATv1Cbie\\\\\\\/m\\\\\\\/AAAqD8ZrWaswJISdHc4W18y\\\\\\\/I0pt\\\\\\\/5Gbz6d+ugq6ORY421FA39GkMpiV5SyEdVBjhMapnRL5yGnOVw\\\\\\\/ViXcw5gLG14pYmKLTvMEn9gMR0b1Tk4qCNR5SN8sNiejBBWwE0kENAHf4n689zxPJqycH+d5xovmhaZ5i+DfORzXdyT3RMOndso6eEPpsU6m1VoQE\\\\\\\/aGO1aqLwF9Qd7zUC+MynlajOnF1QhJY1uXEky4nl0I1t7FudJJls2dbpB1l7XOI6QfrtaqK8OIIvszpdVdvp7RTYn0G6vZVbYRwsPwW6EBx47WJ00flxJ\\\\\\\/WwCI1BPj995Cc2h9bQ5nL0VvlA4Nj7KY2RpBLBKoAIHeMoHE7dC98OHLx7DJ+8iDIgKI2vbuaZHCUf\\\\\\\/Zo+yXhu4NCcJ0EgptayDUFaH24wKZeWlsb\\\\\\\/s\\\\\\\/aF3P2XzfpDyaXG7EuTYQPPpLXwFA9gsM0sdZ87RzuR6cSNBLCbVCKvPtp4rS8oKlDQ8Vj6VIHU2hJCD4DPSJFnaoGwGZ1PEJ5v1mRHFcTl8Csid88j17IRcfheJ8Cc0UnJNtXxOKztBvrRv7Y4TE+sWAdeaJX9oT1JGn9Y+ZiDbWBM\\\\\\\/MfK2WeomNi7CJLpRHFelgtbLBJ1RsO+6HikUiL0a48U8B3kmDrjXKnag6JaVt+Yui7xmfDdE\\\\\\\/CQu8oo+Yi7dVzhVGDr4dmmLMxpr7o+qW2eCahvRxgClSaEl4sa0lGGrVZ9G9yNKFFn3Ll2HWX6vtj2W48VJgvokE+Ki6J9+nH8HtTLemyNudPjubXWxa+TaMNBpT6Fko2wpdjtU4lbfofnmmtIj\\\\\\\/ldRL3NQpO4GBcGLIYJREeN5cphT70\\\\\\\/yvVk9FkSjE6Lhm095RNu9yfx+NobeVcMUK3uor2ReLN+lCu3jGmngVlvs4U7YTgeyPDdE6CzI2\\\\\\\/984E24eFfoc0rW061xGDfmfAKC3TeUA3Iv1\\\\\\\/96JcVYCaOvO8PVag9Fu2NfwwgGscGRiKTKkcgdodiXUYVCUPLuRDmYfCNGZG9P48hH8D\\\\\\\/pNsKq69ea8ClSa3GFaAKxkUR0OwFIqmlaYPhrlcV06l4EwYGEuFrbcbCm2Kh0mkqqvae2SaI403Awqsq9VvHzD9QmJrfNXNksurx++NbF0NlNNdnQa2xXb7GgkDY\\\\\\\/hi8eqNVASdlYjuQLprGTucvUmpL7W4nXH2VkuUs37GRD4Rf8CV0SYYMcgjx87MPr0PWkZhhlo82GLv055BuQBoAzh+gJI3D3KyI7MBf3vEgRQToGgMHNUyr24rxjxMut8IngyHJM46LVSTdN1ihqjgAgumtr9NQwDm3U0f0e0sI1eW7izyRGf2Z\\\\\\\/9hSzYlcOs5Jgz2J4Z\\\\\\\/SEIvshWGThmrza16dZ3EOoTLM\\\\\\\/BVrCCDT2Wk8uKXW8enZhGWpU0ctjeW+iFnuTkiLYXj1qgXKf\\\\\\\/+saDA1AR8I45o7H5Gz6Gk0jIRL4wHuHErb2chJCKxHuQxbu6kFo5nMdP67L6Z2hRckaVsHTbjaQ9rUdZkuIO2VJ7GELPSHQSfzYLlWGd8gxKcgw2\\\\\\\/gSL4e\\\\\\\/HG+wE+gcgqvfHh9GHlRO67kaoew9jkkazDrr6\\\\\\\/odfH8VlhOE7FLug6AjudrVrE34WTqXxRpo3wwKsqdqze+1N2jQk6daID6+gmebBV4NoH\\\\\\\/F1wqo2hzF4hw7bkcatPVwHmkT\\\\\\\/cDm+E05mRgg7\\\\\\\/3\\\\\\\/CIN12L1QI3OBlrn0S\\\\\\\/QrfS4AqesFK\\\\\\\/\\\\\\\/b45FFsU5RjC6XV2WU2nkRq5kD01i3vkfFLaFU1I42SK7PWb6rOLTm1kR3Ud8gSYD2AMEF\\\\\\\/mdSGMS6bXKLPIrRoB4JU9yaBVXyfTlPgq3iYrCw6AY3t45An3ZguHSyVTz7gp2BhbpQIbPHJQLM7kgnU+TEmrhGdDB7+LZuvvBaojEN8SAOYP9CteatirO18Ro0zFp2Yj4wp+\\\\\\\/FN4hfxEf4\\\\\\\/4FI1A0+Sm7VCbZ0FmSC3+xKOGE4YLhOICMKSiJKzSWg5EvitYdMBoBYyza9LoPdV0EyWs6A7pBbktHtmND3vR\\\\\\\/1NPMOepe7rOXX\\\\\\\/R\\\\\\\/gqdRtKagTRVW7bYMjobSWoH9+g3cveBfcDWKl0rpTo74paHN1hOr0h\\\\\\\/SEkRiJgbCIc0s0j9tm72eqV8EDNV095azUhI0WojJTv+pGmMawJ4Z5Vump0\\\\\\\/bQq3X4gT0tuWDVY\\\\\\\/wBqdoSdKvRIL3u+h9bboU\\\\\\\/lEIqRYAx8G3pvXuLDCBXexGxdvRWrbsX5Ia7ymmvD+mJgE9kxkBg7qgoQvQaZiCIbTCkouZtMLbUCdorF64ReFRCkxGDXLZ8h7+uD9q+KxN8Q4e4A4eA7UH2HViko9fyiaYzDpFuiInOrS7WQYKXFY\\\\\\\/76g27aUZdhOh6wTUzQGHncf0nougVGUuQ4hX71l03Osio0cQWBEo\\\\\\\/2\\\\\\\/dITfp1krP05W0LTlAh2l+fOs0zZBxsOjZIwRQP7qs6YZ\\\\\\\/GV+ZOiuvse6o\\\\\\\/HkyxzhupB9MbcYzB554wohyulpWv6PPEKeVL\\\\\\\/USEAYH5N5j3cFb6dqOX1B5TPm9jmuATincV0MrDUec1\\\\\\\/Mth4saXePoo59FFDbowaQKTW+Xzxycucj+BWx2w5PvH6dE5+HzRwfbVJRrBiG+6khNxF3fUiLvFGR9Z8hOPDWxotkdFpSta6MDSwZdXvT5hAsc4BoUEDtekfteZMZ\\\\\\\/SIBw+vH7b3yKCUszp4qBFaV7L+VzxS61BHNYFNa3bD9h2rWYIMvtT3aixSbqq984du00QBm2NXJr+Hi9+9cKKy5f3H5r3SuOyOoSgu7ilgRfZSY7DW10JeKc0aJKqlWWA5YxZ89L77vYhVIYYhn3lucglhNc92Sff7l\\\\\\\/kp80eJ1A8GQyBWNv5uQ5qgFRLYCTFYJ1YLtDrz9ML1Ob6DEyNZI54+i2wk5YGiWB1WtjnJBJ8KnwiZ+thR3Gn3uBpiUMhHBxcILhM7nEx4ZijA7Xz03QurvktSPLzYlAGMhyvcheRsv1blo2BS9MH0JyepHi7bFlwd7Hp6P10x\\\\\\\/O27s0aNSIOF8mPI5KsHBqCQQSlx+7LKOZVtOdTEpD8RgE7SeHxXurGTtQzBiW\\\\\\\/Dix4PgPxkcyPbYQGT5mA5Ag+NP6VJTloRi0p7jwMp4Pl9D7Qs0f7ZH+LJ8Qu2kYAEsZPwNLPsn+wVcMdsvyxlLkw7KLeUUSmR3dOvGGp8ytd8A9gkKRnJLKKZ68bWnOlJ1j6kQLuRXpnBpd17VRUycRwUBTPe5ONiEjyNIdr+5+hWGVPcyw3sTvAzWQ2p+Hz65+ml5kF1c5S6yy4KUOlc1CuTL5smQjVApwNfwdMZWK3MUx6gt8uoH80SRaNVG4FRtpksPDjJyQzuAyeaUUnxRe2VCC89EIlricrTsFSmcKk2M0tghxTbDnNQk4TkDWFOKUqZmGFCq+c2cPYmGzSAbFqxeTC7w5zs6r5orL29HCkkESyvveBzeRTExA26sq1spMe8jOFDzgzqH3aORKecuwwtU8oDwO6xuE0N9S1KAYpb\\\\\\\/0S3AZlAG5Jcm\\\\\\\/sPtgfPBsmI+26OatioNxupYd9t0pENKTiSo8OaHlXTr2XxriZ9XaDAL\\\\\\\/FdQUCegF14+NPZ8Esjoo08u6cT9dTn1rzvBYU84eIWccot\\\\\\\/RvMQa1vHSy7E7LaAw\\\\\\\/vYr8noArgoXew9eyw\\\\\\\/pRy5aqbcG0e3FRwVZ+yQvR8lDBNdxQ8XV\\\\\\\/8TNsyLyx77NempU+M1skoxDWBRkXNGdkH1ICVoAFuyvV6gLhxZs0tzgLi8mLs\\\\\\\/1eMO1zuXpa5Hctlf0vsnNh1w\\\\\\\/Crnay5MK2OiOoR83N\\\\\\\/hXCH6+l20vL3eTDvznoTvov\\\\\\\/hcvN\\\\\\\/lBr5jcLrPk9WrUBHLWQLV1bFelqbAN7frSZAC0hKiKtiw2orHw3FcEibePMSwmEAoGrvLwos2uSjuMHjWpdCahUlGrGtzB5tXoNYJXYcel9Se4x\\\\\\\/kncOqwjGlugmToIEtrPbvLvZkmka9VFjUHyfmUWuge\\\\\\\/ELQVDWrLohxTK6\\\\\\\/Ukyli21i8BWSkA7qdMb\\\\\\\/kkEWLyDfxIOgpnSgNHHf48RZ4Wd92H9TaU3SBXNfzs7Fjy5QYDf\\\\\\\/5RxAe\\\\\\\/\\\\\\\/5lt4\\\\\\\/fcK5MkpZTFnv47rRV4RTQ7iOP89YoKDTG2loihpCkqh+vcmez\\\\\\\/2gnsDqBN3Bij5GjcawMFpB0cBnqz3LYCmuFjYWBe4olMUSksIfEK2jYoW0gKEkJzXQJmXvjnk5emAvIrInY2fxumkMf35odYsie5I764+MecHMPJwP8hN4cnSxZbMSdYHt36mDQC8fhxRMeTOSiupOgefUBoACRL5wKJHcAydOWRU5wUslWGMt1tuX24un\\\\\\\/Gk2Uzt8Hk8Z\\\\\\\/N9jFW3d8YGnUaQu6m4jZkm\\\\\\\/qpGC3DSBMsmNyBf9oysoodzzCsYQdi4yheJaMtoF2t7QkIqQvN+XOhqbkFeGQh8rxcEN9DQQaZ4a8vYtIC34xW1pT2tXymN335n93oIzDz8Lrux1n4SUi2M0ONNVKqNQmBXZRwrzP\\\\\\\/htjgQLH0CfRUcwf8yLjHNKaPZvCaaEtEGEHtNgC8l9HIl4UuetnS4o+60iPStlXnP2YTIyGwi7bdC6\\\\\\\/+E2LfFkT+sJfzq2OuIqdF+2vFQnafdkinEwn0eBA\\\\\\\/w+wsqhA9uEieKfpnQQ8pfDhlmmrQEXaeGw1Lv4QH9mdOKj7sskhDMWAY3WNy4GxSNUZB2rGbE+t99HzhUgShgsABnXakTUkglCsbVakqsOlLlwDRc0+BmZ5KUvITEA9ouXpSiaut9Wrwz9wa1nN54h1i52kxHV8lRcbZ9D0SbpheaCS5ryAIeEjuWMIlFNawKaf36IgTOVMLm\\\\\\\/C9WnQOkbyj0evxrQKX2JYZnhx\\\\\\\/6wCAK6pn\\\\\\\/CnbCtD3IqT5CTnGtkev3Q9+EszC5Wef6SLiSrNcUhqpzTL2S90ytQCAhP6vbgShLjXWfyrhCjMuIYT8jWErx4N8wIFg+qUx7u1XMBT+ZR0nEVGNjffNyrSNftWRgYZmQZ2+DQpl27ym4nL7gFJckKQ2t5se1YKNjdQ+dPbnLcdX41uxOCPqM5r3wJz1WA+iQBJH2IGYlK3kd1\\\\\\\/uW6zJfb7jUHnNWUxFIgEq\\\\\\\/L0haSV2UM0sdFf9+8TGdgWLCgfenysnD+p5TbLRzHOzhpt7fRcdhyrnlnIysLrohBLj7LZg9IW+hG+1NjzVDrk\\\\\\\/j4VExQ2epJ4B53e5sdKLSaQEPcJJDC6Vu4Zf\\\\\\\/Oc7c2m9FcTPekpO4PhqhjsNvtvxZ48DReSxdNJqqLmkg8lksEBn5gPebnWRxkby8\\\\\\\/GkX3pLcVKptgWWTAY885J5\\\\\\\/UFHrHbZShn+SmI4axqu8pz1ETD6Z766gNduX7t21Khbx+n6t44nhukbQ2XDw8UtXOdWCJSI5cGYCNkLsAAVe2o1ssdp64ErUyrWGDh+OGHbtQonESdlGw+I1C0r8GC92wu8c12bCCoEDGy1\\\\\\\/lYlnVbnICDA8DLj3z\\\\\\\/7MKZ5bDArpVb4trRc4SVLfegSXn3NvwBHZ0P3ldp+BPTNp9L7rQYYlBT9LrmKXVGzf3EuPkwadr4c8rPOp6ZSnCr0+ei25\\\\\\\/vFMzTOGvFC29Tpl5LD17422KEpE7+vfK+j34kdPzzFEGw12e56jxeby6+kearfZTsPRFy4HWvgRhuKWJvDogcpcP3nM7iRJReUP50bpT6STyCNrJLg69roS79u1o5tJur5reqZheYb5a36YxDT3EN1OtixJLp5Fc2\\\\\\\/K8ZgGdWjmpUZOJoVHXC+4KanUTRZ0rBRmgRumwGi1\\\\\\\/LiU+HxC+2nN6WKl8jUQMF1SYOB8MX6WJOMAuT3JFzEZ7UEFMOgaX\\\\\\\/6jGFXF1Og6wktpJvZJvsHxlm7IqtcPEpYsUkNuMFz6IE\\\\\\\/UAGfituVzZvCt9d2TQ0E1cdpwhBc1w6DhpDStqsR7sjAR5JacqY74FgMCb5F8XuHxxC9KMuLqccqTyB1FJNsMGX550ZgEGaHM\\\\\\\/gyqHPNnyi07WSNW0oC5YP2r16NZX22cklDTmZ30vLJ5uMNDRpIGYczxXadT8e8KyS1MknMc09LtFIqVQ2IAOnN4TbdXSeTjeU7OkAqtooqI0TCUucGsMJGihVcFcJaRArhEUBU0Hq4wE7YFCKkWqXxpukx+xHUMHH3asx5uc6fPkVs+nQNQpvpytH0m6znToSl1ny0Ie0D3IfwEWEjr3nrWxD+lVLvdeX6z3rUPmYCTChqE6UGzHcuCgFttehj+R7sATP7ZvBayLHASN9mKUz0bbV2LMPpwY0BL1TDti7SgrfQ1BvMryV83OgiQm00E+dbC1yRa3Fozoat17HmOjBNLomh4FjQoPm3C67kaONzi4+ET2evGeSGk2vzBwooM+ekiyvVa6pWzuzLEVeb0+ysYWZ5NSDI4Y7hWjXtUBY3fKT76yGqRGzJ7ET+5bKCdkP+bjVNoc5oxI9m7omEZ+9\\\\\\\/vUMPq420\\\\\\\/qehYOJ17Zt1e1M0l6IWKP50ltBmGa\\\\\\\/w+GpuABE1Ll94yv4kHnyNS2U0bH6WW5O8HvZsiwhkoO3IzCgGlu04hSfvVUMzDip7Mfqq5Vo3GRhVT0WtTHRL4Kx4KPrEvUnS9EIpmKs3qgd923KXF7yBDHmshqJpQUe7kpfT\\\\\\\/1tqoWShgWgIu+5gieYA0jNneziXoYbcigjhb6A5LiGJ\\\\\\\/btlsNt4XtHiuDDjC\\\\\\\/Oz\\\\\\\/qkJzGM1EKZwOksBkVCMwY66z0qliCG+xcDVeYGidi0WTCUXM0mL81jbNNzFrPpxdXpENe1L05zmYRyp6Hv1ju76Wyqw6ivZEr+BvDmFEL9\\\\\\\/CcRxCuX1M4l5RPqOS2xVphGRK7iiT4H+AcCc5Cp5HysUwOJ80jShdbOye9ZddJk+L3A+WINodKmKBLdxBHPXxQV7ILedV126+Q2vBfn2AJhoN8MuucckA7agbPSejpXIS+xP7PfsALEiz0FO1MTMSyRncbCIe0pf\\\\\\\/hgPL52bqq0D7H8R040n9C5UL2+Okr55Lx5wZ9r+jgPpo+vwvXKOqCRRHqL9vwMfH+fM30\\\\\\\/xjzu+AK2AGJhnfD3cEZi2MsWkDzM6yFtaa81jPldAsQJZcQVFcPPGtMVmWsHxmjlDCUWRIRI22ZSVnbYNgEqjp4SFBXMeFnu85jgdxF7\\\\\\\/vysDHfFMaHua7bBd59CFHS9ujOQlktJRAo4KuTB8RZwpdLKlP6x93go49ouATrY5xKlG+b0hyu3Bzdut3SfYyoL4Anq2P4ROB+Wjfnar4+nkZ5b0sxObXIU3dGge6Lc8sdC\\\\\\\/Kcen\\\\\\\/4g0ZW826cun4QSlL+8OO0XmacRB0eFFgA3oFEfLlGQ7ZMEF9ahjlBoKuLid4SSd4leO0q8ONk\\\\\\\/354CLO\\\\\\\/OGhmWseZy\\\\\\\/tYGVL91DjubtGvFuDUkakTQNZU5HcaNlwzg7HCOD8TLU6CHa2rng0hkEb8nD6MF\\\\\\\/p9hrlfb+KKnYdGKzHkQImFNObnRm4+M9lPp5JwR1lAfW0LUUMNE82swGumAIE11uZsuISsEKw+\\\\\\\/XJtYA5P5SfddijlAlJ0jq3amejiNu6zMoZAZEHDoQrGSuJpADeDsy\\\\\\\/38xY1qq\\\\\\\/GQtMr0ZnHZxFgkYHKACm8SnHR3OCZ3ZLMqkBycKK56xc0G7OOY7CcKjjQMlQaNWu\\\\\\\/HtjZ64ZDbZUUAfiQwmrm7Gss4L6KxQnBfyCvKiQIPRVX7o+FmP59r8VFIXqqeJsB4yeRx154CEz2ksr8LZsWrhuu1XE99e2\\\\\\\/CLV7Wf8Gw5qt+ILcDJCYRbIJFdyd6aLZQKsrYTf5UFOWOdSRkDjXifSbmBYm1GuGNAdS41iUIlBRrhN9WxIHlrU8TOo7\\\\\\\/KbuNbSQXkMLiZqMQDqufjb9ti2i6kAPmVjghZ2PipXOVMNwcU+UZ27BNbKyqBZU74hNcqXZJTOBZjtU7SPGzJu+WKLj+jpuLC8g3TiTEK29uzyzIhLXmNY5bKCaHPu0IaQcIp4zVYTJhjtPhblMdPYLHRUNAuv3Q0LdpU4E0U\\\\\\\/u1t2q34E33DiQRmg8bUrjSoaJGJ+qXX9+IhGVyIdjGX8xB+86KmDOCsA6p5JkfKOBsuTe\\\\\\\/F1mE4bqZr46MZhN6ZhfxEETmxr3X6iJJ2tjo3xESKqdfO7FxUGDL6RhpxgjrIjOxjxIU2Xk04XKr0Xo+C3+9I2\\\\\\\/tg4aZYteZ\\\\\\\/WWqcW+pdbhPw2RlE4f6fIMQTPAlRCwIFT5DXcrvqYd3NDrVtQiJO4WK8lHPUlrlXoyPrPb2BIRx1K71NR5Ycler+AiPWwUlLhjqz3fjEd3E39PU2n03qLClmo2QBtDWsnHYH3yVXoARPoktUcoftJJ5EjC8Fkn96Ou4ML3tCzc7XN3\\\\\\\/uVwqh1FBS3Saoue0ZMsvTCIGie7wtAVBMNS7IjXzSUbFefqSuk4Si6ZMmxyigEesmunAaiNE+eAiQRmF6+5mSVS\\\\\\\/PSrtTiIFrTx6gPwdYcRX2ZcGdWl+Ong9feNX3xZ9DLAov8RZ\\\\\\\/SzDpS4sZWMOgEi1lEbwkGcfpHLmjn2RD6MGJLNhz3724WYG\\\\\\\/aTksF2Wn4ngbIPgBfWtyf\\\\\\\/XGptbM2FXnEpSEEblQ8AmHY0LWK2ekLgJvB0ebjJXPSXaWspsR4VW0j0S87snzpFCgYkYbPf336JV7FQ3z\\\\\\\/hp7srdEoYWxgtga\\\\\\\/Cq7naaUp2fNHeCBy6IR1ToDpUiE6DnxyfYUnBnz75QJTbKuMtK4asCWJVyJY4bhe0HIP9m+beSEp1Jw5UmZYvBMOfX670XvrHnmRHtkIOps5OSrVGQaDcxS5Fz\\\\\\\/WL9UVhV9sUC58pQwfK639mAeBs+2fq\\\\\\\/+zPeK+h0MfkvT3l87SIOYoCRfItANrCVyY28LAvCdAoYBnJJ7ykfhbGKTmMzXvyBins1JCDzMWcjgNTP9L9lCdtKBzbey2VKygON\\\\\\\/s\\\\\\\/6Ll2boaZ5ChXioQ\\\\\\\/pNDAfISOc7VsHvWpAqkx9PSchlT1cR5B2WQhRePRL3oe5TWqxmd09akhimizhuvjyx0VIf7QuSpkp5932t6mkqcO7xUGmIh+\\\\\\\/tY6vucPXqxmxSiH3sPikIXj19qOZI2grWYk2tNphPrTM\\\\\\\/lS9USehk0+ZycmlgZ\\\\\\\/4Nloh\\\\\\\/92NIKn7b4K2kfDETC29dkxxEkDUeQCvLKCfoMtUll1yP0snlrYXhAeL2POxXrUds\\\\\\\/3kafY3rCRlmewWc\\\\\\\/+Bd69\\\\\\\/Z0ulQgMYQZaLA8tdIeLGikUlY7Zeide6H5qno4mPYQaGmT9IjaaflaqASAeUZUmpSclj34vH5UxnoT9KZw09K6fNU0Onlwg86KWcWGTvsXf7XVkbDyMuBbX\\\\\\\/3obC6RL66q0D1B\\\\\\\/IJHCsEBYEs4axO5fcyiJuE8g8kKinbT\\\\\\\/MZDZK1rgEDUw9xaiH6WhEDUXzv6s+NMGwMlzklYN44w6uYLuL1Zpwmrb5HQIJC+4vjV9aihf\\\\\\\/6yL9+01VoqXPZVau40f9diTPAemauOC6oC8dLXwBxzZFX8TAU\\\\\\\/0Td+OfGHhe07aHTrdiKcxlqYEqxhvtB2Uh1DUeRIz+Cj3esR++6ldGgFhAqrmtOQFM7ue4YmlfBmDW\\\\\\\/quKUA04LfAwTmVLFUQyJW7PHbarfarKcA5ziLKMq2PjttGDsLIz8zRIF\\\\\\\/ST0rxgCOlQ6CYn+6kkhRspD1eioDQAlJLoWoclxOt35lFW5xGAa\\\\\\\/defh+8pQ9iNh46kdw3\\\\\\\/MgCK8FgMzlsPrttfklFNOzcJUj1tGvHgsf7iUFlSqil2BEaB1wNCSPjLMah+mcAJaSJPuXG61Vf\\\\\\\/vnpDzbnw2M8pF4dndcCt8wXP6cnaXFzAmA9+RwDBdV7470hJJHy83mAtOBTyMab1sNz74kNrX7pGmZeGQ72u\\\\\\\/hhVNEaaTxz+2042yPAQq38SA88udx7iuWXnbD8St8njVUQbOmPdHrNLrZZVfTm+KXzVWi81A5NjIzTXHYPUbfDaj2NxFO9AJUpPpvSkc87hwRIHX5lGqGH0iyV+fkFMfQAgdiXnwOvs\\\\\\\/q6P5AOj8uAoG6INYkPovER2VSgddFiKnOTkyZ1ROysEIXfyL4GsFenYCX4LK0NR6uoWkh9zK7ySaaCkq1j\\\\\\\/GdrhLSJ3Hr9vvwo2+xtNFkVGWC8zyJvu\\\\\\\/T62rMp32TAcNT\\\\\\\/Xx97kTMuWrodaWleWIj5dlHY\\\\\\\/q2s+Hf489QF+ZJ5z+Cn5c8QuMBspaphbmE4z13Ai+G\\\\\\\/fmy2uz6DiYuqsYndE0xWhOD9HwptmUJd\\\\\\\/wRzHjIoQEtlBQd+Z2XcUS1NeAUd7tkRXkIwAE6DAwctlYJomJK0BC8Dd3F4W8DR7nJPOY+LDaOOoMkZJVszpClm7sF09WUYwEPGabtZqAsEe6H3omNe5cpCAfFItsXNyTsBcvPcLYw8\\\\\\\/x4WtmhoC2wcGZQTO6p2Y4LoM8XpzxydNf6tlqAnfkEZ\\\\\\\/IbuVfmaq3NY8hpIdsljBYw8k+sxLyM9ILUXGaGCRr5qCS4z2iPmYQ8R014MY29Fu7s9iTJtwR\\\\\\\/eL+7lT56hH25nDF3mRsZxPCyth8rksrspVZtl\\\\\\\/M6EppeoLxrvMpi9DZae50vU9bzN9t6a4mLt\\\\\\\/e1+cF\\\\\\\/vsNVtExoFIKB8Ii7xCwBQlFl5llv+6inip5nG6UbpZkKbVf+aTl1t2kHrcYZwOsoauHsZELvvgmPJcp7RXWtNCSslyhBzcI7hZVLLXON3JJCrh0sPo8hJuExHf8exqZzETWkwSlFlWtEYFqpzL4qSHvf9BPxceL6Xx+VGqPhVA7doiKQnSqoA8l\\\\\\\/HTSg60swSJicyFqhrhqBS9IQ4XcKa4gadWCqpymaWtVUyb7DSeyUZUuQLHra8Oag57UW+4VafTWkJL00kOEwFIlZh5E01+999bfg915etFUIRKl3kTLn+oFUaAGVQBEXpjmrWJMuFVybpL+yIupjob8IeAVaNqNmjOiJJx0sUtMrDUBzrjCt8H+T\\\\\\\/g0ppxB2ITYzGyPi\\\\\\\/qjWApxaImv901mu2T374VgP8CDrKgK16bqF9z40zbFzCG59ycu+JECqLryD9u\\\\\\\/9UdIBH0FjtgJwU4lLLw5gac371IIYb4ktJUOS9LpXILrvjn6eHZpoTaO29L3mSLPTWCIag++6PL+UwQ\\\\\\\/ojbg+oi6P\\\\\\\/wtMIqWbXYFai4zuCQDbXShlF4kQ18guiGwLrFLrWfRe8mqsy0twEFAHAXoaarCQeeNRHbvPZb7i1rzqthboct\\\\\\\/43rFeNnfMoe8uMlnF+A4KggC\\\\\\\/5ccA+KdK\\\\\\\/ZNH3o2Haxu8lNRLgjg7ATXR60Ogch2QOsQohww3xjg8E6RkIrWuHnQcG3qpWZ1YY6KjFIXuTfz9JXhCJhq6pcBEqkCmvR0SHlHD4mrQ\\\\\\\/F20aWI0o6KdQXMUllLqykMzUYxEapqwCHNvv2GnG40S2P87RShuF9xGRO0fIYEYTvd7K4VtzYhFzhDvOd6U6WtrB3aQFV2N9Z\\\\\\\/ZU8b\\\\\\\/UgAFkinzShTpiq+njStTxCr7bN4gwJbldGmrP2zFJWIaQyDw67Q9CFpz6CcTG\\\\\\\/IqLq4TJr3yQLhtFvYby1Zo0qgqnnVwsQHMMAwH1BmjeB03ALmjvh6lWaZLpdEsI0d6OehXATXqVt69myx0w+Dm3GhQ4\\\\\\\/Cz6PR\\\\\\\/0TzYOhCJH9jJFjnelUckGNU+DhzOqkDc28rUkJUa9q3oUTPBnKhGffL+Ri9Z+ro8wrHwGDSx9sBkn0d\\\\\\\/82ir4ednq2cX5YU6FZkD08ePelC3WYA1EKkCoI+eq8djX89hy7miQBgJaEcYLrr4I3CEfpEqRWIGGG\\\\\\\/IWfgoZ+eTbZcEsnrfMUFCi1rqWpd79nn27kECUrmcpwEAkQKDaZuOyQZRrERKGQxat6fjc6RFWcuDlKXxh8QyWJU3cWiib4mViS32ysyPl9yN9BBEulwFEPfIpqibBPB4PeQiaKcvQsauXONXKJIP\\\\\\\/ylC6esXYgB5AEMr2vK+EpjMu9N7kJ713DXRrcl2lTU5e38Yx\\\\\\\/K8S8ZnJR0oI4EvJcmakgT7V3MkEe9Kf8dzJnWV6KNWo3vw2nOyKa7czt5r\\\\\\\/sjYz8t7dH8oXm33NJD9FRQPprf8Jx6cmtydxpydSdwuHzajPWYVWBdGURwkjWW5QSwuaSNTE3GlqARKJ\\\\\\\/JP9tJVfh2hGE4KfOWyBo3ZhxfvvBiQhXl+bHsA29jLz92cawl4\\\\\\\/XpdBHKvsatLyafSAeVVsCOz5OmPOUBpRrh9ilO6f\\\\\\\/pVu65+atp\\\\\\\/K2e5YBRVaJk6Cl5pfGOa0nFot7xig7sHdewve088BVUB\\\\\\\/NHvxlRyHwFniQMltnsvgAxxTrLe8p0dAiTPnwM0Ye3F0wSSOGz35sCkyQbxf5xK3F5EPtAOO+40UC+We43vJGKHeNkSz4iIVS7iAtddmL7PdmOkp5D78+luhqZATxci\\\\\\\/2vA3aL96FIFYPzLKVX7v4oQ9UL7kw3MIOD57gJ34Z\\\\\\\/\\\\\\\/M8f+TaO8bpPnmrpQlxHiJBPJTdJqQVaOuhW5KQpk0wwsS759PNDs06c\\\\\\\/A\\\\\\\/jAJ\\\\\\\/XaXVOvuNpNgf9msohDTiIr3qmiE18CtKWDBAhKUEatvS62NC3zJW6SZarD8B3EDXUf8pL+Xc7lXQEw2cYSMijo8ZWDI2DNFqF4HaHJjfAeFxRa8o94fuiTzV5i+c3KexyBA6VdT\\\\\\\/QBcVcO03lquW4kDYoNunYStNH2uVxej9Yr7BwEu69y1aIBhQIFC\\\\\\\/gybQmMQcEaalUECB7+q8JwfjmJ2nxeuZUV+GsD6wlhIDGTGYvSW2K9eYaa+tOxyAgEkPunBo+oePGK8JThY8ozrWnIzbdgQhJQnA17qCPoOX30pwpOqMFPxB64Utmfd8CjNSHuwFfLsrnElXPMO1gnLjr1Y4d\\\\\\\/trJ0hCh4NErufWbqghg+OtlIGQwdtnCMd3ZAiJaz84m69Qqq5LyhrKBMD4Db3r8fTetczwCK1pQWnZi6h9DyLT5kW7QFVYiNVSrqBmUKstqDv60\\\\\\\/kqi2tq7vk9X6gCtCyvQgnoGbN2H7PGmTXKqWHCCC9GML+V\\\\\\\/Mh+ErOWd9MujeOBlrlp44qthTMOjrg9goRXiL4a\\\\\\\/4KOP6cTRVtLQ\\\\\\\/49BfSlVG06ayk2HOieEAgnxEJkk6vdzMH4\\\\\\\/GHxpbBZa42t20wiaYr\\\\\\\/Wyj0I7oSnAPxJWrKQc\\\\\\\/03wyQXHUrLg391o2kmXoRzf1L3PscW7u8ru1acScorhBOIYbwmM5yRzGJbgXUB+FwMo3tKQBMxX69ZyexuD8gcb9y598Os7zqzcAtKrEJBYvPOqoqkY+NPHZpviteQ7UPPICWhiEr005U+RxSmXeCR6R8244Xa\\\\\\\/8JAEhfkfQJ07\\\\\\\/OfqyasdE5mhewg51Rhop1rma8vygI4dMfJi2jDkXOXShmQvZfMGs0Cqstvj1h7gKNjMRG8Zmpb4KcIi\\\\\\\/8wm\\\\\\\/t0LHhG857i+k8BAhimm4LKSFI2D7Y6NZ32OkmeV42PJjuzU68jg1bPX\\\\\\\/h1fi8uLkvW4MtsyJgrCsQOLdfrCOORNAOn\\\\\\\/hb4RNbBeYP8n7a2roLgoeIdGKMw1kbu0AfkFZTyLKPK3mWY8Bkfc8Z+2vpCdFA7z5oFrGLLMGLndqw4SoLw+ZmNFduuSei4\\\\\\\/MI6650hGSJcV+SZh646Gqgh7YMwroomM7hEaiJ6+eiNGea4tfn0SUqyzVpGmcKaH26T3RdtaKxmUhi117yfm1RVR2IXNv0WpvaYxMY546bOxFye5oeYw+c2POPHwuHnOWuBCapmV0iKRpZyD5wNoIvKgPPz7Q3co6dncWTpQ5uYOtprI0LoS8bH6Q0ws7EiDXoqWu+rvs+Mq7a7xQuPk0BgPtHAdLLRqOhn73+wGDx74Wthwc\\\\\\\/MVZLjiUM\\\\\\\/vOD+vR7f\\\\\\\/EDH5AWgsX42eoDmMbjxfih2WVsOfA33D88d457LtJbFh31Qb90RdF34MnYYdBsdIyRINewCi7SrWEdksjyggudSIEbzv6zhGdLhlc67EEfWP0UGciqc6NGZbcjZBslrOgGNAUKkJu3y1biSUytru5NSXnOo5yN+FydYk8ynu582Hh4DQ2ih7g6TEcq+PYUx1BcAvgFOt8+0YLY1c0lHLbHux3kwgiqZoo4l7uw7mWfnNWTHTQ97ur3M\\\\\\\/fheFlJL160cpUgXilivwdLHJia1fsg98X+FBCm4dl3kLSZib80JMTRQrjDuVsdNsuAeOT8XDh9Kw7zhIhx8GnnkbIE6s1qdb7yUP\\\\\\\/rEshBQuK2lxc66NYwPWPOnl0Bx33ukFJXs790aMlWHFaGENhNbDk5yMmCy136tPUdI+lHJ\\\\\\\/WR0ZWnqQiipdca0M5GSCiKZ1XYcshXTxwTtraxDdCSw7OdPkNLmigTmyi9cn0ZeFVxyYZ7hbwXeVnmoZNHntoaDk4+syn4yRunvk45xj0Cj4Rrn2aRrH0J+n37bxeq1QakXG6PqAzMA4alazXZeYSMaglaIyZitAAsEyt4zz10ygK9dADf3xCoivTJM\\\\\\\/8iu5hP6qbBIIPQFW\\\\\\\/oO9bYtCpohbrh6TP\\\\\\\/70wRBpchgNfKNqGuWnCXNho7nOxD+QkqCl79oBMlOMMRZ37OT5FoadZ1ebYk+r8CwsgL4qoidS\\\\\\\/JGfWf6E58HjoKd9tdSQdaJObwCeviQ9hNcHq13TUplen2XMnfLshrxM\\\\\\\/MkGSRfSuZeeBWzLb5vo5CmenbP3YeM3mcVjeO\\\\\\\/aF9x0GsFObHkX+yx\\\\\\\/obkToztXIJDC6Ub4K6iA1PKgG\\\\\\\/LL2IixE8VwKAd6mySQzqwkledyqXp66MCmWhVAXmfrxdRSVJbHZm4spy3xLHCnOXopOsuFdTQ0kvCxIMOzvcMDbybC59\\\\\\\/T0AFNVgwkAA292TX435eFlf8x45fPEhlqXuR27mWIjglWit59xoRZU6Cl8a3l1QSABvpN3V6jHiHUZOjGdhfK9NMyOSs0IBE4ep899yf8c+MpdYE6C9bkO3Wockpo36NWDogDqhRPyWP\\\\\\\/C1vF5+K7pod9AjrNQqJF7Klq\\\\\\\/7VzI2U30dxwv2XpqcjazW2qkaqrfTJNe2SfFdI79bdj\\\\\\\/MKNqmXtcq2UL7jUhjBdTWpXZp2CB1owco0xPCef4KWJ8WTPbr2dE2kSm5Q\\\\\\\/GfxC949+TnJ8idqZK50NBLXmgYovkYjY+5YfN7rTQYSe24xPobD84PpgVhMgMW3OiuhFm9NrzwqcBG\\\\\\\/Hs4n7GQSHCWzWr8YZfRb1qocbJucdo\\\\\\\/MF0zeTmw9CL60Q7GJXyLIyj62rhwM2hxd4AOF7mnTcZtIZ3VDdXxxLKBbkJEhmi3EwOkxzTGMG8JGpaHQdisa+5jIogsWBwC+I9uECVED7v8fsI4pBtNIgKLHKFkzctTpMhuT4v2cUzTUXbKt+4v+1OyCToryS017WzfHZO58emgtW5A69ds0Uu\\\\\\\/tAqUdW5mdw6Qk4RLTgYVCAJS1nNOrDyPyWVGejKVfxRpByELqmpTBd\\\\\\\/RkbzdfsRoQ\\\\\\\/xC37ym3\\\\\\\/PENwPYA68SLG8FMEW4KR2SYrKiRyIyFY8lEjLHgyDLrIV37GrbqJVOYjCFqfHc1gpWnwDUm3Lljgi26cBkdE8czETN3cehFLF8VbQOOB8\\\\\\\/wjKoe6VzoqJRLOhQn6Zb5XloMJDU6r1NXeuxur5T53ewoj6NbRkD6F760zIEskgeVI8UInMiIfFab3HX6DTu6mw1bb9lgDzDwdMwJF5z4qPM3r7W\\\\\\\/LR4YwDbA\\\\\\\/TcAgxujzoxj3djRwnC9PUQUjZS\\\\\\\/rbH30NIDj\\\\\\\/J3oXc\\\\\\\/6AcgvlVO2Wi+0eNzc6L4lbJ5gDvw\\\\\\\/6lQtYnfAH+k2\\\\\\\/E9RB25xyXPuy8NJ+wfZK7woshib\\\\\\\/d6n5Cz7i9AerHGXahZFcPH69KCbmQHMRYxgC+9brLz5XKjBItCHP59uizOOb4Ak7lA88nbYArn3FHfc7smgzImffLDDsHtkypkQg2nEJmh65nI5OI+vxonVUEmrWW3pebsz0pEtoPecYjmr1PL2o\\\\\\\/lXECbg0t40zUb4nZ7i0vgunRxyMRWOJJea65cYPijL8eSrI+poCVlpLzoWmcWeHnXnzkRdtM\\\\\\\/WnHLwZW7xj0XAbpSoCVHfP5b\\\\\\\/p+\\\\\\\/vMSwwEZ2Mc5ai5EHvFY+KCfglHsPLW0k4WZ34JACwuyZKcOxyK9yD93LJLAq0o7gc9nTlG7Q4XVGv7BbFEHW26j+SOvwtXjZEUNRa8VDVmOziNiJ6iVmyWOzmxPKOUQSFwH3uRAS4cmQ1iUnbysRDJSwxx5qGxWAlT0YhAW3cB7lpM5wCIuzaXMHWsCZQ\\\\\\\/hWDEEvm3fLB4vmoxb0DopplZeZv+atHOF7wxEC5\\\\\\\/oW7kZISr\\\\\\\/1Y+9fc8mmjZiPdykUyQmk+0yYUBqyH63v336uH1Cfjl+ABL6WvgdWyT0hbG3n28ICMIkgigJHD2Dvd36kYb9w2as8h8fYIytUxSvmpEOgDP+HL4VXNAz7T7orJujEUhnb1PPh9FfR6Vm85gP225i3Na2vy54skFErcIMdvTJcD6EoDTCelMoRjDTasts5hLjDA0iabr2ghHpAA\\\\\\\/2XdMvdzwBZN0G3zELFF888\\\\\\\/KTDoiQAgkthgUQjYmAN7izFkf5fKX23CMTG10N3I3PNkWDMnaLdsMYojrdJoMPvOPzdO2NoRfMnHRyLxJaaqcD3ZnSMXPxGqSTZ9aHOXG8KXq6MwC\\\\\\\/bstzMYcuYP5CQDBZuvfEOCPFoVu1PLdr4H74WYyn\\\\\\\/BlyO\\\\\\\/oYZqKjrTXknOuW5+WowSqgqPrxApvzoFsNb5YiuIz+ccWY0pSsCD6n+iWbkXYWCOFJSCWk11FqGM13\\\\\\\/5WV1jdX\\\\\\\/RP9p4yoZ0Bc5JkcgNREQhwCKY7kaCnkqNFHQiIAab+NTu9It5d1t91NMChMgGwqyApt47Uh0ZM\\\\\\\/lnM3HASo9jOBIb5IILA9m6IwfVwK7UQxG+NQ\\\\\\\/YBwON8OIoVOZPiEYHBU1er5Bnz6VbOU3Iik5K3S9Vrx+uBYTwTZUJTnaAMnUwiWFbYa65gRgbeZX4GiHH176APX+B0I9jMcDcehmTkW1HAze0NaGA5vQanfliRYVETzS1kA5mYfzhyvbn2Yk+JFHq64gAt2ZCuZ3OWRbaQcM9UHstp3A28VrpCpW2Lw7VAwrItfw5NQbDhNIgSjpAuXnxBhj2ulDiW73RkIB+Qk7L6jd8ddooBnfUWjwOlVf9M3kD4c8aaqOHFDjMLpp9ys453TsrkZzrtMFS+F2ksfBPVfCWUpWM\\\\\\\/vl835RY68Xq0IlwD4+NHHW7gsc\\\\\\\/5URAzgTaLa+dNSHBXSsVui\\\\\\\/uRJJbpjJUoPOI0l2zSHm\\\\\\\/Fj1DVmvnOf2\\\\\\\/rzqUJxZ0ljn\\\\\\\/RBF6+xUiQQ5VeJxTiTplHsa4ojfRSudJ65FF6Uhg4MMsoKCiy2nzX0tYwGI1\\\\\\\/io4T3fdYNsSrLWyQxtWVjWzSJ8pBLSJbtWJC3CBbqXt19\\\\\\\/LmVoLbqm\\\\\\\/huLnp+RkjxWsoyPOLpy2kAdlAGEsf+sYcz+OhXlRFRhV5zAIanGHZnf33dFmDm9DQB4hz8kinqz6k5OUu\\\\\\\/Xz5osl763TZ1ImbhAnvfKxAMXwhlJD0bKWSW3y35hZl\\\\\\\/1XZmXLcE8RSuyXed9cxXqKCQ98mcndsZEM8mNCQe58hajGNmvlgFliaP9iWpkoH7pOxjHJxf+8mfEE3L6lgRPeSYpNB+\\\\\\\/oSWL55eIg9\\\\\\\/i8ntN4P5gihg\\\\\\\/9wmVPiyEejN4siP34Q\\\\\\\/y3rhD0cb2etye6sS0jUCaD8N9xFF09dvUgxQy1M+58Y9B5M+6i60KjA6QGke02YQPzl+33m6Mvf4utmaI2FZT\\\\\\\/+VGFsRTR5HnfB0vhytAcv10mfWMSTELAvh34+zOble5ejK+pdzsASlkWm05pFv4SIuSrtToC+BlFJgr04uTQ1qDDOYVzhHTPuxUyAhn4Lo7WWHYUlwqnf9dGvqrY+FdtxE8rxuoBbdyZFLRNYTxk7lEFyMJ2r4LDWfOrXfxrCguzFX33KQPtBosDBtw5HO1Eb\\\\\\\/hixN+z\\\\\\\/U8+oebdeZTUiIqkRxAdcmmaV0ortTxwn75uMPpP2dmyEAJLQVbfLqg4A5+iF8ZWFKvtqtlj6o6ddTIg0G\\\\\\\/4qhLtqw9fiaOCKcXT+EMjL6XownZ5PDPsYsFP4sNP+dhdirApnnF6RRV4\\\\\\\/nA1sBQnvQVSrhOhDubebPfsbS6HCCjagzRndizoLI34fzo09K6uQ2hbzYGkB6hG\\\\\\\/bn3Wp2fVHdzTLD8r5C\\\\\\\/UAXfLpyirnbIrJQWtwtB05JyCypvHjqzo1pU6\\\\\\\/G3HIT87xdfTIu6zZ0GqewPneRdNAlwkY5xKbQn7LbARqG8ViRFzH3KHRRzS711XEabEM06obohwHcfBvnkIYbr+8SWUlfm85nxXWE1c9HM+6hi57pFLbShHve+SVMm1PbPmwF\\\\\\\/l0ufPjCx9WEgd5TX2EduOq+0u1WrdLon4zfy\\\\\\\/cYQ5Zou3iiJOCL8VWaWPSyS9EOx\\\\\\\/YPI7Szn4LTcH3tp2m5h6xrDFvNrLdBJxQ3yendzbM988hteixovsHbibpdk8PVYSqA0w8T45KM7wFjIhLFQmcYq51PAyGvhWk+ELdnDMAp1bvxJG11E3on5CieDgF4l9zQD5qXdf5jMeo\\\\\\\/eHxMrRWAbeiNp3TkYAvojxdm\\\\\\\/taIb9QhJIlXoayOioNQL2zerPZ3uTfBwV\\\\\\\/xL5oRdJwGHpS0NdvHXH6oUyH4evxsqaB8QYrHGYtxJA6Tk\\\\\\\/L0x8gHIzRrXntXrm3IgvQuKOkdjWvFeJtwIvtgP67aVRYKehf4LKvHDER63egFF4PIRylYiCTF8TYApoDC7v15gqgS7VaeD3mmfu2dEwWgQIVmJ\\\\\\\/kwETBmgNz0NZ6W+7xRxQFI1uyb6T5FZEe7jihM592+sILvDcvsyZgPu2sX6qH8nglY1pSTg9x0M3FI9vUNpDFlV1Ybkh+yMrCajHEvPybpJYhL2KZYqvLwCxcYY\\\\\\\/S23pICuAhnMvm1\\\\\\\/lwEI7P0LnWxYqxXYW5H9I\\\\\\\/oc8acddjAOG32HVkL0XpMEgxFNxmoP24gwzzCfR2RUaiW2qBw3lmqhJaGrpvwAl2DLvGyGX+bNoLCPC2xMg8esqnktUBTeMLvlAK01bltvLSuZqxQKJ74E1RnB28aHcWDxlbTt1Fkstuoe0leZn4E\\\\\\\/lKwTpPc+fgckxqB3po4gTDyOyUnd0VhMbPK78K+svlS8eD\\\\\\\/n07Bv7OD6qM4KKQ\\\\\\\/9fwobTxei3mDBL4ATMYvw3XqhTjWCKaP4PvpbjScBirqgIQoDxm5w+e0723CAZWNicXjoobj5E65EN8fD47EOj54xYWRKOz05wPMQPV6O5igBVf1dD8TOJ7VoC+b+37TK8zWEvYToKBdR95ZOSpVmg4WEVEGby\\\\\\\/sfUFIkn0OA6FduhGKbyr48VRyTf78IN9O3wxFWZ0p3tVaIw4HZIAFDSmAmedChI843+gh1WIFZgPeUERdeFGAY5QusGTimhmyrrLsIB4BoRa4jAdRfIbzonMfegxOy6800lMi8YZbpZ87zfj0o\\\\\\\/+WOGoV6UtBX38IfUq4uwZRYsUVhU+fXNPtsKXKJK9iwx5NNSoDvDpdAve18z6aIeHHJg\\\\\\\/sTB1Fgxuz7mysBPLW8ZpQ5KPRrAjpnEHRBndX5ApKSXIbChRyIe3hXC7OfPODsiuWOW77Rz8DoT1MkMuBM5zTrAQM7ZL1zWw2nM2kJ7wCRL2NqqgzJbx7dFXD90HrLkV1KmCTh4t+fDkeHKODArvWIozWTljVRtZRmtOuxgLAT\\\\\\\/e3ZwkQhoMNOh6LRPq+7h4w1A89rz5Zn4CBgi00u6llN\\\\\\\/g4D09r60pAr\\\\\\\/ml3ZjxaFjC8bd0+gJy4gHzxtjcbTLxIEQ41o4phKrZHf8OsJ6CEj1G0NSSly6+5dDorN1SlB6CLBXWhNnLNcBQjUMb128ZuCR1ygy+xfDOJxbvBeAOvh0RxxU5vcepC56GNrSk9ZmHCDZ4PHDaQR6X0uw1yojJKT1Qc1Na3Vd9iM5z3OMDMjxp1\\\\\\\/whvwW8JVidsfTG0D2Kk3ZPYnbLKBPCE5L8rhEgsHiAWZzvFQKa9OfF6gtrzeWS5PQcgbsFmNSoaQGw2nvJ\\\\\\\/lsYxix\\\\\\\/MFaT9s9Pp687zXdbgy\\\\\\\/vVMZLrcKrP+zjtXgXea\\\\\\\/HD5rCNCON8N2xfDHQTH5VgzLsviiUvLfYjqzjV03XLTp5zT7Au6tqxLzDrZcWNgv6l8svKK7Ne3eU3AM7APVIecpeyb\\\\\\\/kZTcrPm8pcghsql\\\\\\\/3mhxctdvUiKkcU\\\\\\\/S7x5A4IE2x2BtocgtPzh6OQtUigmN\\\\\\\/RTnLBXvDPUbj5EiMrUSix+YtO5vROH9hlU2tPZwwb1b0kaejQpJMJi8R8lTfg\\\\\\\/F2x3Co89v7S5MO5JLPUQ1pYbgwPGpJd2Yb8be1jYyLpfSxvRg2QyBOUBon3mB5RxKsVSmf0o+nqFg14L94sZwVm5QBffqwNznt4xGrTWM+5KJUMhGRoRhz2zUG\\\\\\\/lHekDqEVAyvagN24eGSxGBDc4IB1V+LEJDPeJxCsu5A7lv5kGZy01Jn42sJ4dTQOpewDQKJvdGTZtEB5Ji\\\\\\\/8qgd4AOWYly6OSPPcbmDedQ\\\\\\\/QdWnkw6yZFohSH+uWaE2QkkgthPlFP62uf8GZmu\\\\\\\/yUNzSFqJ2KANAcfO6\\\\\\\/9J+4jMSkttkJRr15UFGrxvbJkn6Sj4VpwuPaRxklPq\\\\\\\/oz0mn6m9gbrC6Dd4Hb6d\\\\\\\/V8u4U\\\\\\\/jYeYpgdKgBBXmihe8ND0aoDZGrux6wPn20jXYON\\\\\\\/UtAKfHEcQRensn\\\\\\\/94dwO\\\\\\\/z9atwLvIfDcfD\\\\\\\/LCFQuRZ\\\\\\\/ueLF\\\\\\\/UnaZiDs8YqeAvSPJ\\\\\\\/OZ6XbEohJ\\\\\\\/WCSyCxFmuhetO3WH7xKVvoDyOrdUqy2pgCS2UtsCqFF23PrK4Dpgl23MKTkZOEkY1mqsChTI2AbF266kXsf+YbrEXsB6XlKmiyKPBF+Tw93YWV+FpbJgl7ZjmX7LZIRBiX\\\\\\\/Vha1VAcQ8LS5g9pFeCwLBaUX0DPtzcn8tePqce8CdInduyeFS\\\\\\\/niXRHxrkm9xYDH\\\\\\\/p2WuXNBzcBq7O8Wkv0d\\\\\\\/VSAUr796+fttNn+2Z2s+LpKfQpYVzQ1riHwBcFpc+thdtjmFfA1AM2EnYxZF5BhGFEygFDkpqjRBB7jVdIHvBCxtDkrRK+XjkEhUw7yimWyhPa1PXV6HrVdCVPSltnSRkCfpVSBDspO1J7\\\\\\\/KY1LGfg7E7RtQjfH5vgUwI54Pf6L\\\\\\\/wikoqSpK71nIXd7lY5\\\\\\\/IDx2Q3sIwDTob6J1UBxuMGeJngTJ9hMpcId+C7IH43r92PGlrcyso4+bws4jZc8Rl\\\\\\\/EExoGNvucjPbT+FxLe+pgql2\\\\\\\/msUSfDtzH0S3\\\\\\\/86niQssPLJYNzmB6T9K1wceKmQ916\\\\\\\/Bfi8XA1zNXjGwROFkScvu5V5qvPwaNEzkM5zktaTZPq+U4T1PvPSUBJ9j0ZZC8MFQqPLnZuHHzbvQnC\\\\\\\/0eNshfX6+GNJ0f96BddJLCGcdJX1qXN7jOammywvxIqQsShi6x4ZaHyLCizX6JQvSMwCGhAaVt1h6GLdxX3vHHpzZFmBSFlh5RkFNBqmGFzQjpLzc1Cywr4WrAcAZ1ZDgoKusWMTKELnC+0YDTU+HBD8HWoi66pEvEv8Xz0sMSnrIx1\\\\\\\/5rzHN8Pi+lBpcqoVCQfiIh71FdvA4xESgUxbNJutYrIv3b\\\\\\\/lMDLLxPKUpFrylXWJ9MLT0\\\\\\\/Jb70P5Lkt7qsTq8Xgvb0noiVAjcXW+qPQmzmxn5tsry2TFU\\\\\\\/GDrrbSFuU4v\\\\\\\/2\\\\\\\/HjUESjtguXFplWXpgMciKaPDvfhhLJzFPReM9k4q5xuYmTbSIQP06w53J2\\\\\\\/su7DCnDijACFDvZwBJ4\\\\\\\/uVKeuRKk11gKRvCQdXl4BcFgiJauwgBFir+zZbuDBx+RBTSg+gsJn8bpTivrlQzz9BYDZCM6go\\\\\\\/L258g60NBC+aj3oly9KiCrYoGfNvIuxAVoTH7ttp5ZUH+Qq3yx6MbeY14PZ2WiEL+I3h1SNukwQmzEA5x7POUN1I6qlNF+iOuuMBEtvZWilBUjnxp+bWdakuE9DnGhlbebjawmnMcfIVV5KflJp5BAaOSUaJFK\\\\\\\/e6rCqnyvTj5JFwQdrudxo8cm+Hcr62BrPBljjetn9pX7ickELlqo\\\\\\\/Nnf6wkwzz65S7iuVMVrCygkHAB\\\\\\\/sFCgF7jdRWqTsdvuHeU8lq47m1g+g\\\\\\\/jjSXPq5w3Nd4Mjv+whXI6eFF7apKBrt5zLZCQCy\\\\\\\/weiXIAeCIohVkhL19tK0Xaw90ENH4Td\\\\\\\/GWq272bcQOBE9ugdWMoTxlCa5nTmhWOcoQYoMYv7obNzIIXQNjgBn+ie88FJZFj+epZD2vQ6vUff4aujtWOvhYkf1D0CxqNwNlGAZky4Eujy\\\\\\\/Y2FM9UwwwcqRAFC6rGcbQj6Ziuz\\\\\\\/3LwKMisvldgtIq80jY5KIbzMHCwuaKsITX8s5OtxLzqk82yyWNqpvuBBtWjs\\\\\\\/YRc2sq491ZZuHPuecmBuKKbyCva74DhMcnLviWpDarR\\\\\\\/AhfoTk28gJ50xHQqVJFeXPFSXPhCR9oyoZwd3C0XSeYpnCbQM4ZCOBEy0uDTkzvXpHT6pIuz8hfYG0ca0iyex7REoXvjRO8UPNFyFE\\\\\\\/LyLYYH9RsCsBXqpRpVjdgADzyUzyE\\\\\\\/i6GACtjt0j\\\\\\\/2fjnQh5Cf1ioTQ6GhTLo+s8Brx1iYLSDzeiMsQtOX0FOOVbTME+P2JTc544DaFHFKGZ7FtyVV\\\\\\\/7\\\\\\\/nbUqcQFaYBHj1Uu0Asq7H5yR5odowGWFCYUF9dIFOjYbSs3aE7cEVIVSrHQnXy1sbG4uS5D12RqaPaEJiywFIVdnGsMECfRgNouZ4pmYEWsBxSAT1CajIwJIvfZqRV4cF\\\\\\\/fLSTQqO92a43evNdPoB8Wj9BfU2thLppmev9eJG0uY7LeaN\\\\\\\/iCkivyRd9HFovWHivVRl0\\\\\\\/xfIFYn1CS+QAZyX1RILkdBEdwjUeHmspsjk4l\\\\\\\/S58wN+n1gtqU9WhBdjW7YoR8Nnt7KOuyBvL74Q1HECUANeYqvd2IXVKZUQXDApgcmGDkCyM8ihXbaTk10ELUZQz8MCwIPhcLRH5VYkJgBvpOODLy5TWhnI20BZU6npkc0QD0bPOxE4Y3cuGzCI9pH4dJXg0Yzr44md44LTagpNz3EQeFxalceMtGklurW38weNA2+8ZxspFpc\\\\\\\/UOejwgFBvFoO9pKsqTb2j9EQy76kk9XoJQjtnke9SkFVgdToj5lVsYkBklL9mdJSUI\\\\\\\/bRiLifYju08H4sNj4GZ1fWuLXodzqNI2UAzESXNks3rwjj7SLPYgD2mjyCsLLYo2i1Bf6BHkhAFRTuT9UHWUbAPyL2rjwTv2qI40YW2pAX3ozuiMTv40VaIP\\\\\\\/EyxWF\\\\\\\/s3q3qFmzIk1fRKuvYlYS0LwxYb1tha61KCUszCILQMUi7S7ggtII5Dk+1IJJZhgl81BFQPv+aGzx0obCG0vcDcF9pznj8UgCx3iypEp9IS1mtN6nJ+oEiNtJF0tW54H6wklgoWpeeKAqmU6n9YmCd3beTOKLtyn\\\\\\\/ogI+sSgldqlJhoggwAGle0u7rqQgDWvSNbNDUM1MoMf5VL3e0rp0tyV2aNnE8mgARvXAB5Xbvn\\\\\\\/FqOY2Oqct1\\\\\\\/Pw8PcuAi8t7xuhkmmbzo7af5Nu6uwZRZ+ikc0vnoSBpIrGamOIoSqe8gC+DuB+yolDiYrBjpE5yqzmZIoi0Ih6+XlySsmqpjJpY9Ivs8DCZBdp6GXDw6qYX44JfRIhbkXgg0fd6qq\\\\\\\/w9VaDQKCi89gZv+1EwYKW+LjPfTqYWAJw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022kBGKPz5\\\\\\\/LYdHqldjktVLGg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226970960a40dc47d83024355a6de851c40f6a707a460f549f68e92f57e307e71c\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CQMLaM9\\\\\\\/jCBYlMcAjyq7c0VuHaggW2nOzfMPqeiR3m33+zz83UvcqATHQY6eNTZnCjd+YiwSt6snzWyfnowJyAKe0LVCb9ZMnoG\\\\\\\/Us838D7imE9hsNDdjjHn77YTpRPcam15h\\\\\\\/0ixc8M0dKNvlhkoSQTOoLiGokEG1aSUzT37WiYqlf1yPGmCDQMYGhhI6N5PzzHpgBynBfx6imTjyCtIH6GOPhNELWu5UNMPgK1pJoevfFFHeVVHraZeLtzSCtunTrc96ec21ZJLFc6bze7Rfe+YgVrEIdJBmfWQCjMmgp7hDAzyJolGesmT7tvoqJjez2da38qkRauKs5v+uraN\\\\\\\/UVCWFda62wuKZcb9CTo0gkVMu+Hy2Y9SOuqMmAdQCEEZjV5UvlLycNsQUorYUtYXsagWjHqKx5TONTHbcshR+rKK0Iyj0swxYbshGGB6IT9TtyqTPnhGDyBZrjQ2Tjop2bDnT5K6073CcYGtah9IirLc\\\\\\\/DMmqI028vdTgsy6pxa6bZYKug9jBn8d2yjNmMEzAFeHwQqUcT7vft1SaNekJT6EK5+KYERnA8UohQcIa82cvsGReZDUf+sRjsbKTieSr\\\\\\\/mM8yyUBt9IqZi8yAI97mJXRMjmtXRgxN3U88ZgmZobqlJM2WQWBPSdraAK4WoonbAl7sPrkqn6CD41Wpn3F8Qo\\\\\\\/nXZKhesI9A8glh73fb9tOeb8yF57pRppP2D+ugwjTjZ3pgdi4a8+NTDi9C1bCr8rPXWFw6OXWNB4ZLSDQj\\\\\\\/EC4H2hyN45Wz1yo6xyK0cEH54rDHh6NmjjUAG+dRvMQaqL\\\\\\\/c\\\\\\\/WxUxAg1Aut8EbfOZWrn3+P875EmocNe2m4ncA0KWpkHot61jhYethedxCpbC1YZwX0YQ7rDJ8o1Fi+tDo54aMhEM0wnys8VlFHqLeLrizMa\\\\\\\/3l7VbFBzC8MetZbXXz5W92s16dlmuzGbHBkV5i0PTwvU4Tvvug61C\\\\\\\/OLaJePTyjzPmFd31mo5FHLUSxEHWakMIklv54YY6dP0aNH+RZFMVfTkvRecz8MoQ5OWVk4rtDoaXrcNlO9EXWgCz3AK7vQVHHE0GM3sWLQT+56PbUW6k2tTqyYCVryrPeGxl9jEsOaw8gaLGRPZ6nPauHsjcDyP2ZbhzuXsputjpSdFJzbGFGfjhvbW1tZ3ibqZDJ3JqSc78HaeCMyPs3HxDIqVSt9j0X\\\\\\\/ipLgMNg8\\\\\\\/NwVJ9z+Vd2kLGzEOHMZHBKMFs3XKwU3wOs6vB6d8\\\\\\\/epPZaxhLzg6RhByT\\\\\\\/zo06CHB\\\\\\\/d082ZaRSvLlWWQy6bBxHY2JQ4gJDbuIdhBkIpbF9YKD0dWQdlbIz0FBWb4R6TqWvPcuDcVC9sOrSP1sM3XFDcTLwXae90BKXW1VpNv0PDl\\\\\\\/bPAX7I0cagHV4ZOPOQmzUrfngLPaCaWYJqL9FVFA3T4ICd3j2vFqNDoKC\\\\\\\/K93gooiWvKEA9Pb4BCznbERSifVWjygVwawcu+f\\\\\\\/3X2G7RDwA9JMcx5RysEdDDdcWEdqnpLnZA\\\\\\\/zytnmZXWCtv2OjA4Buuj4jf1QnxSK41e49XfiCbgHn+ZCgqwkEBY5\\\\\\\/flHKjj1aPD1Pyov+R\\\\\\\/HuZbs0VTEeo4DAMmy+gJJtZZUg1YBO59o3r8P3wbUlwBRgZ8J4OAWG3DTKEMQ70WLJqNmbggSL4EwkGFqunSJ8QHNI5d0fiBSq5x+DTbT0jHdQdA2dX4s1JxrB69Ig77fez90v\\\\\\\/XJ0sPMoZKGYFUWEH8BOwHW0ca4mDyjSdSDVZ9L\\\\\\\/Y1QcFCgsNiUeb+MLhyQ\\\\\\\/M3NhpvmoQV5hKf8RK+KrS4Jq60Itq\\\\\\\/DvQy\\\\\\\/o6hSBcWDEQLjMFTX7WB\\\\\\\/hdnRCy580b5hgyFaFQ\\\\\\\/04JpYZ0+XVTkLUScqg8A4TZWJMxN+tg1HtHQAQ9sgw6Eo7sO6bKZf0083OSO7BT8LO4GHMl7yEqSkxNlm57KLx7zDslcCJGmXjGcKxpLSyuCANzVeDk\\\\\\\/mbkQhgaorHfpLiZxwKtEGEwABi1w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022roKhNyUfXuiIGpOaDg0iHA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222197f78f615a535a49ea1f6ed1f2a66cbe035b02a28cf3879e4e01177f6d2241\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459163 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigation">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/2/131" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qTm4FP7ev5vqK_jBB5kUfcGEF-HUDVbIj2JALMH6Cxw" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/2/131" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--1JzQLggN3ldF_soQwLKdEDsyDVg0pxbHltxlTxLiQE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/2/117" title="Fluorescence In Situ Hybridization–Based Karyotyping of Soybean Translocation Lines" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/2/143" title="A Screen for Germination Mutants in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459163" id="node459163" data-pisa="ggg;1/2/131" data-pisa-master="ggg;g3.111.000042" data-apath="/ggg/1/2/131.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Quantitative Genetics of Genomic Imprinting: A Comparison of Simple Variance Derivations, the Effects of Inbreeding, and Response to Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Anna W. Santure</span> and <span class="highwire-citation-author" data-delta="1">Hamish G. Spencer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">131-142;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000042 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459163" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anna W. Santure </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Allan Wilson Centre for Molecular Ecology and Evolution, University of Otago, Dunedin, New Zealand</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Zoology, University of Otago, Dunedin, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BW.%2BSanture%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Santure%20AW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnna%2BW.%2BSanture%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:asanture@gmail.com" class="" data-icon-position="" data-hide-link-title="0">asanture@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Hamish G. Spencer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Allan Wilson Centre for Molecular Ecology and Evolution, University of Otago, Dunedin, New Zealand</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Zoology, University of Otago, Dunedin, New Zealand</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>National Research Centre for Growth and Development, University of Otago, Dunedin, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hamish%2BG.%2BSpencer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spencer%20HG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHamish%2BG.%2BSpencer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/1/2/131" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459163" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459163/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/2/131.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459163" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459163/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/2/131.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459163" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459163/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/1/2/131.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459163" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459163/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">The level of expression of an imprinted gene is dependent on the sex of the parent from which it was inherited. As a result, reciprocal heterozygotes in a population may have different mean phenotypes for quantitative traits. Using standard quantitative genetic methods for deriving breeding values, population variances, and covariances between relatives, we demonstrate that although these approaches are equivalent under Mendelian expression, this equivalence is lost when genomic imprinting is acting. Imprinting introduces both parent-of-origin-dependent and generation-dependent effects that result in differences in the way additive and dominance effects are defined for the various approaches. Further, imprinting creates a covariance between additive and dominance terms absent under Mendelian expression, but the expression for this covariance cannot be derived using a number of the standard approaches for defining additive and dominance terms. Inbreeding also generates such a covariance, and we demonstrate that a modified method for partitioning variances can easily accommodate both inbreeding and imprinting. As with inbreeding, the concept of breeding values has no useful meaning for an imprinted trait. Finally, we derive the expression for the response to selection under imprinting, and conclude that the response to selection for an imprinted trait cannot be predicted from the breeder’s equation, even when there is no dominance.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/breeder%E2%80%99s-equation" class="hw-term hw-article-keyword hw-article-keyword-breeder&#x2019;s-equation" rel="nofollow">breeder’s equation</a></li><li class="kwd"><a href="/keyword/correlations-between-relatives" class="hw-term hw-article-keyword hw-article-keyword-correlations-between-relatives" rel="nofollow">correlations between relatives</a></li><li class="kwd"><a href="/keyword/breeding-value" class="hw-term hw-article-keyword hw-article-keyword-breeding-value" rel="nofollow">breeding value</a></li><li class="kwd"><a href="/keyword/additive-effect" class="hw-term hw-article-keyword hw-article-keyword-additive-effect" rel="nofollow">additive effect</a></li><li class="kwd"><a href="/keyword/parental-effect" class="hw-term hw-article-keyword hw-article-keyword-parental-effect" rel="nofollow">parental effect</a></li></ul><p id="p-4">A gene is imprinted when its level of expression is dependent on the sex of the parent from which it was inherited. For example, insulin-like growth factor 2 (<em>Igf</em>2) is expressed only from the paternal allele in most fetal tissues of eutherian and marsupial mammals, while the maternally inherited allele is inactivated (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">DeChiara <em>et al.</em> 1991</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">O’Neill <em>et al.</em> 2000</a>). More generally, imprinting results in nonequivalence of reciprocal heterozygotes, where inheriting an <em>A</em><sub>1</sub> allele from one’s mother and an <em>A</em><sub>2</sub> allele from one’s father gives a different phenotype, on average, than the reverse inheritance pattern. Complex processes of epigenetic regulation are necessary for the repression of one allele while the other is expressed. These processes include allele-specific modifications such as differential DNA methylation, chromatin structure and histone packing, and differences in replication timing of the maternally and paternally inherited genomes (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Rand and Cedar 2003</a>).</p><p id="p-5">Approximately 234 imprinted genes have been identified in mammals, including 68 in humans, and many of these genes are thought to be involved in traits such as growth and development (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Morison <em>et al.</em> 2005</a>). A publication predicting imprinted genes based on sequence characteristics suggests that imprinted loci in the human genome number as high as 156 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Luedi <em>et al.</em> 2007</a>). Recent years have seen an increasing number of statistical methods developed that aim to identify imprinting in quantitative traits. Using QTL mapping, for example, imprinting has been suggested for quantitative traits as diverse as carcass composition, growth, coat color and reproductive traits (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">de Koning <em>et al.</em> 2001</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">de Koning <em>et al.</em> 2000</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hager <em>et al.</em> 2009</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hirooka <em>et al.</em> 2001</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Knott <em>et al.</em> 1998</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lee <em>et al.</em> 2003</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Milan <em>et al.</em> 2002</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Quintanilla <em>et al.</em> 2002</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Rattink <em>et al.</em> 2000</a>), while general mixed models have demonstrated the involvement of imprinting in traits such as milk yield, litter size, and growth (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">de Vries <em>et al.</em> 1994</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Engellandt and Tier 2002</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Essl and Voith 2002</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kaiser <em>et al.</em> 1998</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Schaeffer <em>et al.</em> 1989</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Stella <em>et al.</em> 2003</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Tier and Solkner 1993</a>).</p><p id="p-6">The inclusion of imprinting effects in these genetic methods highlights the significance of imprinting to a range of economically important livestock production traits and to human health and disease, as well as the importance of understanding the effect imprinting may have on traditional approaches to modeling quantitative genetic traits. Quantitative traits may be influenced by many genes, the environment, and any number of interactions between them, and models for these traits are correspondingly complex. Nevertheless, we here employ a one-locus, two-allele quantitative genetic model to demonstrate the differences in a number of standard approaches for theoretically defining breeding values, genotypic variance and covariances between relatives. In doing so, we show that genomic imprinting may have a significant effect on the assumptions made in these most minimal models, and is therefore likely to also influence more complex models involving many alleles and multiple genetic loci. We also compare the effects of imprinting and inbreeding on quantitative genetic parameters and predict the response to selection for an imprinted trait.</p><p id="p-7">In <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, we list and define the important symbols used in this paper, following the convention of <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Nagylaki and Lou (2007)</a>. The reference is either to the equation closest to the definition of each symbol [thus (7), (7)+, and (7)- would mean equation (7), the text below (7), and the text above (7), respectively], or to or the relevant approach or table. The table is ordered alphabetically and split into Roman letters, Greek letters, equation simplifications, and subscripts and superscripts.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460826/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460826/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1 </span> <span class="caption-title">Important symbols used in this paper</span><div class="sb-div caption-clear"></div></div></div><div class="section other1" id="sec-1"><h2 class="">The Model</h2><p id="p-8">We here present an overview of a number of approaches for deriving quantitative genetic models for imprinting at one locus. Such models are the basis for many quantitative genetic approaches for dissecting genetic and environmental effects in quantitative traits. Following the approach of <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Spencer (2002)</a>, consider an autosomal diallelic locus subject to imprinting, with alleles <em>A</em><sub>1</sub> and <em>A</em><sub>2</sub> at frequency <em>p</em><sub>1</sub> and <em>p</em><sub>2</sub> (= 1 − <em>p</em><sub>1</sub>) respectively in the population. Note that the population under consideration is static, without selection, migration or mutation operating. Assume that on some suitable scale, the genotypic value (<em>G<sub>ij</sub></em> for genotype <em>A<sub>i</sub>A<sub>j</sub></em>) of <em>A</em><sub>1</sub><em>A</em><sub>1</sub> homozygotes is 0 and <em>A</em><sub>2</sub><em>A</em><sub>2</sub> homozygotes is 2<em>a</em>. Assuming no maternal effects, writing the maternally inherited allele first, <em>A</em><sub>2</sub><em>A</em><sub>1</sub> heterozygotes have genotypic value <em>a</em>(1+<em>k</em><sub>1</sub>) and <em>A</em><sub>1</sub><em>A</em><sub>2</sub> heterozygotes have value <em>a</em>(1+<em>k</em><sub>2</sub>), following the notation of <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Santure and Spencer (2006)</a> (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/2/131/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genotypic values (Gij) for genotypes AiAj under genomic imprinting." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1684317246" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genotypic values (&lt;em&gt;G&lt;sub&gt;ij&lt;/sub&gt;&lt;/em&gt;) for genotypes &lt;em&gt;A&lt;sub&gt;i&lt;/sub&gt;A&lt;sub&gt;j&lt;/sub&gt;&lt;/em&gt; under genomic imprinting.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1.&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/2/131/F1.medium.gif" width="440" height="69"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1.&#x2002;" src="https://www.g3journal.org/content/ggg/1/2/131/F1.medium.gif" width="440" height="69"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/2/131/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1.&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/2/131/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460781" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1. </span> <p id="p-9" class="first-child">Genotypic values (<em>G<sub>ij</sub></em>) for genotypes <em>A<sub>i</sub>A<sub>j</sub></em> under genomic imprinting.</p><div class="sb-div caption-clear"></div></div></div><p id="p-10">In general, imprinting is thought of as complete inactivation of one allele dependent on parental origin, corresponding to <em>k</em><sub>1</sub> = −1 and <em>k</em><sub>2</sub> = 1 (complete silencing of the maternal allele), or <em>k</em><sub>1</sub> = 1 and <em>k</em><sub>2</sub> = −1 (complete silencing of the paternal allele). More recently, however, imprinting has been treated as a quantitative trait, which implies that maternal or paternal alleles may only be partially inactivated (see, <em>e.g.</em>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Naumova and Croteau 2004</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Sandovici <em>et al.</em> 2005</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Sandovici <em>et al.</em> 2003</a>), and <em>k</em><sub>1</sub> and <em>k</em><sub>2</sub> may take any values in the range [−1,1]. Note that if <em>k</em><sub>1</sub> = <em>k</em><sub>2</sub> = 0 then the trait is purely additive, and both reciprocal heterozygotes have a genotypic value midway between the homozygotes. If <em>k</em><sub>1</sub> and <em>k</em><sub>2</sub> are equal but of opposite sign (for example, <em>k</em><sub>1</sub> = −1 and <em>k</em><sub>2</sub> = 1, giving complete maternal inactivation) then the locus is subject only to imprinting. However, in the most general case, where <em>k</em><sub>1</sub> and <em>k</em><sub>2</sub> take any values in the range [−1,1], we might consider that both imprinting and dominance are acting on the locus, as the mean genotypic value of heterozygotes is not the mean of the homozygotes.</p><p id="p-11">With the help of <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, the mean genotypic value over the population is<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-13.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>and the total genetic variance is<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-14.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>where<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-15.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span>and<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-16.gif"/></noscript></span> <span class="disp-formula-label">(4)</span></span>(<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Spencer 2002</a>). We follow a number of approaches in calculating breeding values, components of variance and covariances between relatives. Doing so illustrates that various assumptions made in these approaches are not valid in the presence of imprinting.</p></div><div class="section other2" id="sec-2"><h2 class="">Approaches</h2><p id="p-12">Five approaches are outlined in the Appendix, and the results of their partitioning breeding values, the corresponding calculation of variances and covariances, and the derivation of covariances between relatives are shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Tables 2</a>–<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">4</a>. In the absence of imprinting, all of these approaches give identical breeding values, variance components, and covariances between relatives. The expressions for these terms in the absence of imprinting are obtained by setting <em>k</em><sub>1</sub> = <em>k</em><sub>2</sub> = <em>k</em>. Importantly, it can be seen that it is only by modifying the least squares regression approach (Approach 2b) can the sex-specific additive and dominance values derived by <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Spencer (2002)</a> be recovered (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Santure and Spencer 2006</a>). The other three approaches (Approaches 2a, 3a, and 3b) fail to incorporate sex effects, and give incorrect results when partitioning the variance components and calculating covariances between relatives.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460786/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460786/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2 </span> <span class="caption-title">Summary of breeding values for all approaches</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460837/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460837/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3 </span> <span class="caption-title">Summary of variance components for all approaches</span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460773/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460773/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4 </span> <span class="caption-title">Summary of covariances between relatives for all approaches</span><div class="sb-div caption-clear"></div></div></div></div><div class="section other3" id="sec-3"><h2 class="">Inbreeding and Imprinting</h2><p id="p-14">An interesting aspect of the above variance decompositions is the similarity between inbreeding and imprinting, as inbreeding also introduces a covariance between additive and dominance effects (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Harris 1964</a>) that may not be partitioned if an incorrect method is used. To investigate this similarity, we incorporate inbreeding into Approach 2b. We represent an inbred population by dividing the population into two groups: a group that represents the expected Hardy-Weinberg proportions, comprising an overall proportion of (1 − <em>f</em>), and a completely homozygous group with no heterozygotes, comprising a proportion <em>f</em> of the population. Thus genotypic frequencies for <em>A</em><sub>1</sub><em>A</em><sub>1</sub>, <em>A</em><sub>2</sub><em>A</em><sub>1</sub> and <em>A</em><sub>1</sub><em>A</em><sub>2</sub>, and <em>A</em><sub>2</sub><em>A</em><sub>2</sub> genotypes are <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-72.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-73.gif"/></noscript></span></span> each and <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-74.gif"/></noscript></span></span>, respectively. Now the overall population mean incorporating both inbreeding (<em>I</em>) and imprinting is</p><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-75.gif"/></noscript></span> <span class="disp-formula-label">(5)</span></span><p id="p-15">When there is no inbreeding, <em>f</em> = 0, the population is in Hardy-Weinberg proportions, and the mean reduces to <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-76.gif"/></noscript></span></span> as expected. With no imprinting, the mean reduces to <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-77.gif"/></noscript></span></span>. We assume that the inbreeding coefficient <em>f</em> is stable across generations, so the proportion of heterozygotes does not change.</p><p id="p-16">As in Approach 2b, male and female additive and dominance deviations may be calculated separately (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Santure and Spencer 2006</a>). For example, the additive effect of inheriting an <em>A</em><sub>1</sub> allele maternally is<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-78.gif"/></noscript></span></span>The remaining additive effects are</p><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-79.gif"/></noscript></span></span><p id="p-17">Breeding values and dominance deviations may be calculated as in Approach 2b.</p><div id="sec-4" class="subsection"><h3>Genetic variance components</h3><p id="p-18">The total variance for an inbred population with imprinting is<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-80.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-81.gif"/></noscript></span></span> is the total genetic variance for the case of imprinting only (2). When there is complete inbreeding (<em>f</em> = 1), the total variance is<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-82.gif"/></noscript></span> <span class="disp-formula-label">(6)</span></span>and for no inbreeding (<em>f</em> = 0), we recover<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-83.gif"/></noscript></span>. <span class="disp-formula-label">(7)</span></span>The total variance may also be rewritten as<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-84.gif"/></noscript></span> <span class="disp-formula-label">(8)</span></span>and for the case of no imprinting (<em>k</em><sub>1</sub> = <em>k</em><sub>2</sub> = <em>k</em>), the total genetic variance becomes<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-85.gif"/></noscript></span> <span class="disp-formula-label">(9)</span></span>(<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Harris, 1964</a>), where<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-86.gif"/></noscript></span>,</span><span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-87.gif"/></noscript></span> <span class="disp-formula-label">(10)</span></span>and</p><span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-88.gif"/></noscript></span>.</span><p id="p-19">Comparing equations (6) and (7), we can see that in the absence of imprinting and dominance (<em>k</em><sub>1</sub> = <em>k</em><sub>2</sub> = <em>f</em> = 0), the total variance for an inbred population is twice that of an outbred population (<em>k</em><sub>1</sub> = <em>k</em><sub>2</sub> = 0, <em>f</em> = 1), and indeed the effect of inbreeding is linear; an inbreeding coefficient of <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-89.gif"/></noscript></span></span> yields a total variance <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-90.gif"/></noscript></span></span> times the variance with no inbreeding. However, for dominance but no imprinting (9), the effect of increasing inbreeding is nonlinear; the population variance may increase or decrease relative to an outbred population depending on the allele frequencies and the value of the dominance coefficient. A similar effect is evident with imprinting; if there is no dominance (<em>i.e.</em>, <em>k</em><sub>1</sub> = −<em>k</em><sub>2</sub>), the population variance linearly increases with increasing inbreeding, while with both dominance and imprinting <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-91.gif"/></noscript></span></span> the population variance is a quadratic function of <em>f</em>. Thus, it is dominance but not imprinting which determines the relationship of the population variance with increasing inbreeding.</p><p id="p-20">For a highly selfing species, the degree of imprinting may have a large effect on the total population variance. For example, consider a population with <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-92.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-93.gif"/></noscript></span></span>. Setting <em>k</em><sub>1</sub> = −<em>k</em><sub>2</sub> (imprinting, no dominance), the total variance is 0.20 for <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-94.gif"/></noscript></span></span> and 0.25 for <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-95.gif"/></noscript></span></span>. Interestingly, the effect of imprinting becomes less pronounced as inbreeding levels increase; for <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-96.gif"/></noscript></span></span>, the total variance increases from 0.18 for <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-97.gif"/></noscript></span></span> to 0.25 for <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-98.gif"/></noscript></span></span>, while for <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-99.gif"/></noscript></span></span> the variance increases from 0.23 for <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-100.gif"/></noscript></span></span> to 0.25 for <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-101.gif"/></noscript></span></span>.</p><p id="p-21">The female and male additive variances are<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-102.gif"/></noscript></span> <span class="disp-formula-label">(11)</span></span>and<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-103.gif"/></noscript></span> <span class="disp-formula-label">(12)</span></span>where <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-104.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-105.gif"/></noscript></span></span> are the female and male additive variances calculated from Approaches 1 and 2b (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). The female and male dominance variances are<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-106.gif"/></noscript></span> <span class="disp-formula-label">(13)</span></span>and<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-107.gif"/></noscript></span> <span class="disp-formula-label">(14)</span></span>where <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-108.gif"/></noscript></span></span> is the dominance variance (Approaches 1 and 2b, <a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>). Interestingly, and unlike the pure imprinting case, the dominance variance is different for males and females. The female and male covariances between additive and dominance terms are<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-109.gif"/></noscript></span> <span class="disp-formula-label">(15)</span></span>and<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-110.gif"/></noscript></span> <span class="disp-formula-label">(16)</span></span>where <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-111.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-112.gif"/></noscript></span></span> are the female and male covariances between additive and dominance effects (Approaches 1 and 2b, <a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>). When there is no imprinting, the female and male covariances reduce to the same value:</p><span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-113.gif"/></noscript></span> <span class="disp-formula-label">(17)</span></span><p id="p-22">Considering that <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-114.gif"/></noscript></span></span> we can see that <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-115.gif"/></noscript></span></span> and hence the covariance is strictly negative under inbreeding alone. Recall that under imprinting alone<span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-116.gif"/></noscript></span></span>and<span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-117.gif"/></noscript></span></span>(<a id="xref-table-wrap-3-7" class="xref-table" href="#T3">Table 3</a>). Now we may rearrange <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-118.gif"/></noscript></span></span> to give <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-119.gif"/></noscript></span></span>, so that<span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-120.gif"/></noscript></span></span>and hence <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-121.gif"/></noscript></span></span>, so the average of male and female covariances under inbreeding is also strictly negative. However, if <em>k</em><sub>1</sub> and <em>k</em><sub>2</sub> are of opposite sign, then one of <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-122.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-123.gif"/></noscript></span></span> may be positive. Thus, although both imprinting and inbreeding introduce a covariance between additive and dominance effects, it is only the presence of imprinting that allows the covariance in one sex to be positive. Imprinting can therefore have a significant effect on the total genetic variance and on the sex-specific components of variance of an inbred population.</p></div></div><div class="section other4" id="sec-5"><h2 class="">Response to Selection</h2><p id="p-23">We follow the approach of <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Heywood (2005)</a> to investigate the response of an imprinted quantitative trait to natural selection. To include selection, let the absolute fitness of parent genotype <em>A<sub>i</sub>A<sub>j</sub></em> be <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-124.gif"/></noscript></span></span> and define the relative fitness <em>w<sub>ij</sub></em> as <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-125.gif"/></noscript></span></span> where<span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-126.gif"/></noscript></span> <span class="disp-formula-label">(18)</span></span>is the mean fitness. Following <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Heywood (2005)</a>, we consider the special case with the linear fitness function <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-127.gif"/></noscript></span></span>, which gives <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-128.gif"/></noscript></span></span>. We denote mean offspring genotypic values after selection as <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-129.gif"/></noscript></span></span>. We can now write the change in mean trait value from the parent to the offspring generation (the response to selection; <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-130.gif"/></noscript></span></span> as<span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-131.gif"/></noscript></span> <span class="disp-formula-label">(19)</span></span>where <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-132.gif"/></noscript></span></span>, the selection differential, is the covariance between parent relative fitness and genotypic value, <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-133.gif"/></noscript></span></span> is the change in mean trait value from parent to offspring, and the expectation is taken over parents (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Heywood 2005</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Price 1970</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Price 1972</a>).</p><p id="p-24"><a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Heywood (2005)</a> defines <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-134.gif"/></noscript></span></span> as the mean genotypic value of offspring from parent <em>A<sub>i</sub>A<sub>j</sub></em> before selection, then sets <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-135.gif"/></noscript></span></span> and, after some algebra, restates (19) as<span class="disp-formula" id="disp-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-136.gif"/></noscript></span> <span class="disp-formula-label">(20)</span></span>or, alternatively,<span class="disp-formula" id="disp-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-137.gif"/></noscript></span> <span class="disp-formula-label">(21)</span></span>(<a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">Heywood 2005</a>). We now apply this approach to an imprinted quantitative trait. As usual, we need to define both male (paternal) and female (maternal) terms. The absolute fitnesses of the four genotypes are<span class="disp-formula" id="disp-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-138.gif"/></noscript></span></span>and</p><span class="disp-formula" id="disp-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-139.gif"/></noscript></span> <span class="disp-formula-label">(22)</span></span><p id="p-25">Relative fitnesses are shown in <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>, along with the frequency of each genotype and average values of offspring before and after selection. Note that the population mean, variances and covariances <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-140.gif"/></noscript></span></span> are the same as derived for Approaches 1 and 2b (<a id="xref-table-wrap-3-8" class="xref-table" href="#T3">Table 3</a>).</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460822/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460822/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5 </span> <span class="caption-title">Population values under selection model</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">Now the allele frequencies after selection are<span class="disp-formula" id="disp-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-177.gif"/></noscript></span> <span class="disp-formula-label">(23)</span></span>and</p><span class="disp-formula" id="disp-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-178.gif"/></noscript></span> <span class="disp-formula-label">(24)</span></span><p id="p-27">For both female and male parents, the mean genotypic value of offspring before selection is equal to the mean genotypic value:</p><span class="disp-formula" id="disp-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-179.gif"/></noscript></span> <span class="disp-formula-label">(25)</span></span><span class="disp-formula" id="disp-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-180.gif"/></noscript></span> <span class="disp-formula-label">(26)</span></span><p id="p-28">The mean values of offspring after selection for female and male parents are:</p><span class="disp-formula" id="disp-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-181.gif"/></noscript></span></span><p id="p-29">The difference between male and female offspring means after selection is<span class="disp-formula" id="disp-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-182.gif"/></noscript></span> <span class="disp-formula-label">(27)</span></span>which is zero when there is no imprinting (<em>k</em><sub>1</sub> = <em>k</em><sub>2</sub> = <em>k</em>). This result clearly demonstrates the difference between female and male parents in their effect on offspring means.</p><p id="p-30">We derive the full set of covariances and expectations required for equations (20) and (21) in the Appendix. Now, the response to selection is<span class="disp-formula" id="disp-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-183.gif"/></noscript></span> <span class="disp-formula-label">(28)</span></span>where<span class="disp-formula" id="disp-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-184.gif"/></noscript></span> <span class="disp-formula-label">(29)</span></span>and</p><span class="disp-formula" id="disp-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-185.gif"/></noscript></span> <span class="disp-formula-label">(30)</span></span><p id="p-31">It is clear, therefore, that the response to selection is the same for males and females, and is, as expected, related to the population variances and covariances in addition to the selection coefficient <em>t</em>. In the absence of imprinting <em>k</em><sub>1</sub> = <em>k</em><sub>2</sub> = <em>k</em>, <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-186.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-187.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-188.gif"/></noscript></span></span>, and our total response to selection becomes<span class="disp-formula" id="disp-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-189.gif"/></noscript></span> <span class="disp-formula-label">(31)</span></span>(<a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">Heywood 2005</a>) where<span class="disp-formula" id="disp-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-190.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-191.gif"/></noscript></span></span>and</p><span class="disp-formula" id="disp-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-192.gif"/></noscript></span></span><p id="p-32">How does the magnitude of the response to selection compare to what we would predict if imprinting is ignored, and reciprocal heterozygotes are assumed to have the same genotypic value? Substituting <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-193.gif"/></noscript></span></span> into (31), we find that the expression for the response to selection is identical to the full expression derived with separate <em>k</em><sub>1</sub> and <em>k</em><sub>2</sub> terms (28). This suggests that even if imprinting is acting, the predicted response to selection is the same whether calculated using separate genotypic values, or using the average of the genotypic values for the two reciprocal heterozygotes. If <em>k</em><sub>1</sub> = −<em>k</em><sub>2</sub> so there is imprinting but no dominance (as the mean heterozygote genotypic value is midway between the homozygote genotypic values; <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-194.gif"/></noscript></span></span>), expressions (28) and (31) become</p><span class="disp-formula" id="disp-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-195.gif"/></noscript></span> <span class="disp-formula-label">(32)</span></span><div id="sec-6" class="subsection"><h3>Comparison to breeder’s equation</h3><p id="p-33">The response to selection according to the breeder’s equation is<span class="disp-formula" id="disp-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-196.gif"/></noscript></span> <span class="disp-formula-label">(33)</span></span>where the narrow sense heritability, <em>h</em><sup>2</sup>, is the ratio between the additive and total genetic variance and <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-197.gif"/></noscript></span></span> as previously. For the case of imprinting, we can see that the breeder’s equation<span class="disp-formula" id="disp-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-198.gif"/></noscript></span> <span class="disp-formula-label">(34)</span></span>is only equal to the response to selection<span class="disp-formula" id="disp-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-199.gif"/></noscript></span> <span class="disp-formula-label">(28)</span></span>when the dominance variance and the male and female covariances between the additive and dominance terms are zero, which requires <em>k</em><sub>1</sub> = <em>k</em><sub>2</sub> = 0. For the case of no imprinting, the breeder’s equation becomes<span class="disp-formula" id="disp-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-200.gif"/></noscript></span> <span class="disp-formula-label">(35)</span></span>and is equal to the response to selection (31) when <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-201.gif"/></noscript></span></span>. Therefore, we can see the well-known result that the response to selection and the breeder’s equation are equal only when the dominance variance is zero, and hence the breeder’s equation only predicts the response to selection in the absence of dominance, whether the locus is imprinted or not.</p><p id="p-34">The difference between the breeder’s equation (34) and the predicted response to selection (28) is a function dependent on <em>a</em>, <em>t</em>, <em>k</em><sub>1</sub>, <em>k</em><sub>2</sub> and <em>p</em><sub>1</sub> (= 1 − <em>p</em><sub>2</sub>). For a dominant trait with no imprinting <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-202.gif"/></noscript></span></span> the true response to selection (31) is strictly less than that predicted by the breeder’s equation (35). Similarly, if <em>k</em><sub>1</sub> = −<em>k</em><sub>2</sub> so there is imprinting but no dominance (as the mean heterozygote genotypic value is midway between the homozygote genotypic values), the breeder’s equation becomes<span class="disp-formula" id="disp-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-203.gif"/></noscript></span> <span class="disp-formula-label">(36)</span></span>while the true response to selection is</p><span class="disp-formula" id="disp-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-204.gif"/></noscript></span> <span class="disp-formula-label">(37)</span></span><p id="p-35">Comparing equations (36) and (37), we can see that the breeder’s equation again overestimates the response to selection for the special case of imprinting but no dominance. For the case of complete inactivation of the maternal or paternal allele <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-205.gif"/></noscript></span></span>, the breeder’s equation predicts a response double that of the true response.</p><p id="p-36">If we include both imprinting and dominance, and let <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-206.gif"/></noscript></span></span>, <em>t</em> = 1, <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-207.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-208.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-209.gif"/></noscript></span></span>, the response to selection (28) is also generally less than that predicted by the breeder’s equation. However, it is interesting to note that if the difference between <em>k</em><sub>1</sub> and <em>k</em><sub>2</sub> is less than <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-210.gif"/></noscript></span></span>, then the predicted response to selection may be the same as or slightly more than that predicted by the breeder’s equation. Therefore, very small differences in the genotypic values of reciprocal heterozygotes may result in the breeder’s equation underestimating the response to selection.</p><p id="p-37">These results contrast with the derivation of <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">de Vries <em>et al.</em> (1994)</a>, who from the covariance of parents and offspring predicted the response to selection for an imprinted trait as<span class="disp-formula" id="disp-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-211.gif"/></noscript></span> <span class="disp-formula-label">(38)</span></span>where <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-212.gif"/></noscript></span></span> is defined as the variance due to imprinted genes.</p></div></div><div class="section discussion" id="sec-7"><h2 class="">Discussion</h2><p id="p-38">We have demonstrated that a simple one-locus two-allele model of genomic imprinting produces large differences in predictions for additive (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>) and dominance terms from a number of standard approaches for partitioning the genotypic value of an individual. These approaches are equivalent in the absence of imprinting under standard Mendelian expression (where heterozygotes have equivalent genotypic values and hence <em>k</em><sub>1</sub> = <em>k</em><sub>2</sub>). Although all approaches give identical total genetic variance, there are differences in the partitioning of the genetic variance into additive, dominance and covariance terms (<a id="xref-table-wrap-3-9" class="xref-table" href="#T3">Table 3</a>).</p><p id="p-39">The major differences in the approaches arise due to differences in how breeding values and additive effects are defined. Approaches 1 and 2b incorporate both sex- and generation-dependent terms, and breeding values are equivalent for these approaches (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). However, Approaches 2a and the regression methods (Approaches 3a and 3b) are unable to partition separate male and female terms. Consider how breeding values are calculated for the different approaches. Approach 1 defines breeding values in terms of allelic contribution to offspring, and breeding values are the same for reciprocal heterozygotes. Genotypic values in Approach 2b are defined in terms of the male or female effect they pass on to offspring, and so include the same sex-specific generation effect as Approach 1. Breeding values are consequently equivalent for reciprocal heterozygotes. The single regression Approach 3a similarly forces genotypic equivalence for the predicted value of reciprocal heterozygotes. In contrast, the other two approaches define breeding values in terms of an individual’s own genotype and the parental origin of alleles in that genotype. As a consequence of imprinting, the parental origin of these alleles has an effect on the genotypic value of individuals and hence reciprocal heterozygotes have different breeding values (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-40">Under standard Mendelian expression, breeding values are expected to be equivalent whether defined as the sum of additive allelic effects (Approaches 2 and 3) or from the means of offspring (Approach 1). However, differences have been noted where alleles in the population are not in Hardy-Weinberg equilibrium (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Ewens 1979</a>), in relation to populations with nonrandom mating and inbreeding (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Falconer 1985</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Fisher 1941</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Templeton 1987</a>), and as a result of population subdivision (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Goodnight 2000</a>). Genomic imprinting represents a distinct phenomenon causing differences in the definition of additive effects between the approaches we have investigated. The difference of these approaches in predicted breeding values mirrors the conclusion of <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Falconer (1985)</a>, who found that, “the concept of breeding value [has] no useful meaning when mating is not random.” In addition, genomic imprinting introduces a covariance between breeding values and dominance deviations (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Spencer 2002</a>). This covariance between additive and dominance effects has only been noted previously when a population is inbred (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Harris 1964</a>).</p><p id="p-41">In comparing these approaches, we assumed that Approach 1 gives us “correct” values for population parameters. Approach 1 is the most time-intensive method for partitioning genetic variance because it requires derivation of mating tables to give offspring mean values. However, this approach does allow separate calculation of male and female variances and covariances, which is of great value when considering offspring-parent and halfsib covariances in real populations.</p><p id="p-42">Approach 2a was able to retrieve the additive variance, but the true additive-by-dominance covariance was included in the expression for the dominance variance. By defining additive terms specific to male and female inheritance, we were able to “rescue” this method to include separate breeding values and dominance deviations, and their corresponding variances, for the two sexes (Approach 2b). Of particular note is that Approach 2b was the only approach able to recover the Approach 1 covariance between additive and dominance effects. Defining separate male and female dominance terms <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-213.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-213.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-214.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-214.gif"/></noscript></span></span> includes a “generation” effect that is not accounted for in Approaches 2a and 3. Approach 1 is based on calculating breeding values and dominance deviations that relate to the following generation because we use offspring means in their calculation. The equivalence of Approach 1 with Approach 2b is a reassurance that defining separate male and female dominance terms is an appropriate measure to include a sex and generation effect in this approach. A closer investigation of Approaches 1 and 2b is presented for a model including maternal genetic effects and genomic imprinting (<a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Santure and Spencer 2006</a>).</p><p id="p-43">It is well known that parental effects may have a significant effect on the phenotype of offspring. It is important for methods to include such effects, but it is not easy to imagine how the linear regression models (Approaches 3a and 3b) could be extended to allow for parental effects such as imprinting and maternal genetic effects.</p><p id="p-44">It is interesting to assess how different these approaches are in their estimation of variance and covariance components. The numerical examples in <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a> contrast genetic variance components and resemblances between relatives for the different approaches for two scenarios, one where alleles are largely paternally inactivated, and one where maternally inherited alleles are largely inactivated. We assume that phenotypic (and hence genotypic) values range from 0 to 1 <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-215.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-215.gif"/></noscript></span></span>. We can see that, as one would expect, paternal inactivation increases the covariance between mothers and offspring and half sibs sharing a mother, relative to fathers and offspring and half sibs sharing a father respectively (and vice versa) (from the correct expressions using Approaches 1 and 2b). Approach 3a underestimates the true additive variance, while Approaches 2a, 3a, and 3b all underestimate the dominance variance. As discussed previously, Approaches 2a, 3a, and 3b are not able to calculate the covariance between additive and dominance effects (<a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>). This covariance between breeding values and dominance deviations is included in the expressions for resemblance between parents and offspring and full sibs and is likely to play a large role in identifying quantitative traits that are influenced by imprinted loci (<a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Spencer 2002</a>).</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460835/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460835/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6 </span> <span class="caption-title">Values of variances and covariances for all approaches, given paternal and maternal inactivation</span><div class="sb-div caption-clear"></div></div></div><p id="p-45">By using Approach 2b, we were able to extend the imprinting model to include inbreeding. As previously noted, inbreeding also creates differences in how breeding values are defined (see <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Falconer 1985</a>) and creates a covariance between additive and dominance effects that is not present in a randomly mating population (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Harris 1964</a>). Interestingly, we have demonstrated that in the presence of both inbreeding and imprinting, the dominance variance is different for males and females. The covariance between additive and dominance terms is strictly negative under inbreeding alone, and is on average negative when averaged over males and females under imprinting alone. However, it is only imprinting that allows the covariance in one sex to be positive. The sex-based differences introduced by imprinting represent an important difference between the effects of inbreeding and imprinting on the derivation of quantitative genetic models.</p><p id="p-46">Finally, we derived the full expression for the response to selection of an imprinted trait. For an imprinted trait, the breeder’s equation generally overestimates the true response to selection, a result well established when a trait is known to exhibit dominance. Excitingly, we have demonstrated that even in the absence of dominance, where on average reciprocal heterozygotes have a genotypic value midway between the two homozygotes, the breeder’s equation does not predict the true response to selection. This result has very great significance for predicting the reaction to selection in natural populations—if heterozygotes are not distinguished and we only measure additive variance, we are very likely to overestimate the expected change in mean trait values between generations.</p><p id="p-47">Detecting genomic imprinting of a quantitative trait using, for example, covariances between relatives, is likely to be difficult given the large sampling variance of such covariances and the possibility of maternal effects increasing the covariance of offspring with their mothers (<a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">Santure and Spencer 2006</a>; <a id="xref-ref-38-6" class="xref-bibr" href="#ref-38">Spencer 2002</a>). However, the derivations above do suggest that a number of different quantities may provide indicators for the influence of imprinting, such that if one approach lacks power to distinguish imprinting from nonimprinting, another avenue may provide fruitful. For example, 1) large differences in the covariance of offspring with their mothers compared to fathers (particularly if the covariance with fathers is greater), 2) the existence of a non-zero covariance between additive effects and dominance deviations (particularly if there is a difference in sign between male and female covariances), and 3) a smaller than expected response to selection based on the breeder’s equation (particularly when there is little evidence for dominance) all provide good evidence for the influence of genomic imprinting on a quantitative trait. A large range of methods is presently available for assessing the role of imprinting in complex and quantitative traits. These methods follow the broad spectrum of genetic approaches for dissecting complex traits, from general mixed models, use of covariances between relatives and identification of parent of origin effects in phenotype inheritance for traits without genotypic information available; to the marker-based approaches of linkage mapping, association studies and QTL mapping. A number of these approaches utilize variance component estimation, resemblances between relatives or differences in the phenotypic values of heterozygotes; quantities discussed in this manuscript. Such approaches are invaluable in the dissection of quantitative traits, and we encourage researchers to employ an approach that can successfully incorporate genomic imprinting into a model of the quantitative trait of interest.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-48">The authors thank B. L. Harris and K. G. Dodds for discussion and Mark Johnston and an anonymous reviewer for helpful comments to improve the manuscript. A.W.S. was the recipient of a Bright Future Enterprise Doctoral Scholarship, supported by Livestock Improvement Corporation, the New Zealand Foundation for Research, Science and Technology and the Tertiary Education Commission.</p></div><div class="section app" id="app-1"><h2 class="">Appendix</h2><div class="section" id="sec-8"><h3>Approach 1 (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Falconer and Mackay 1996</a>)</h3><p id="p-50">This approach is based on using the genotypic values of parents and offspring to calculate genotypic deviations, population breeding values and dominance deviations, components of variance, and covariances between relatives. The genotypic deviation of a genotype is the difference between its genotypic value (<em>G<sub>ij</sub></em>) and the population mean <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-218.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-218.gif"/></noscript></span></span>. The breeding value is defined as twice the difference between the mean genotypic value of the class’s offspring and the population mean, and can be derived separately for males and females (<a id="xref-ref-38-7" class="xref-bibr" href="#ref-38">Spencer 2002</a>). The dominance deviation for a genotypic class is the difference between the genotypic deviation and the breeding value. The genetic variance of the population is the variance of the genotypic deviations, male and female additive genetic variances are the respective variances of their breeding values and the male and female dominance genetic variances are the variances of the male and female dominance deviations (<a id="xref-ref-38-8" class="xref-bibr" href="#ref-38">Spencer 2002</a>). Resemblances between relatives are calculated from first principles with the help of a mating table.</p></div><div class="section" id="sec-9"><h3>Approach 2a (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lynch and Walsh 1998</a>)</h3><p id="p-51">Based on a general least squares approach to calculate population breeding values, dominance deviations, components of variance and covariances between relatives. The genotypic value <em>G<sub>ij</sub></em> for genotype <em>A<sub>i</sub>A<sub>j</sub></em> is the sum of the population mean <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-219.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-219.gif"/></noscript></span></span> plus additive <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-220.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-220.gif"/></noscript></span></span> and dominance <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-221.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-221.gif"/></noscript></span></span> effects:</p><span class="disp-formula" id="disp-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-222.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-222.gif"/></noscript></span></span><p id="p-52">where <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-223.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-223.gif"/></noscript></span></span> is the average additive effect of inheriting an <em>A<sub>i</sub></em> allele from the mother, <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-224.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-224.gif"/></noscript></span></span> is the average effect of inheriting an <em>A<sub>j</sub></em> allele from the father and <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-225.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-225.gif"/></noscript></span></span> is the remaining dominance term (also see <a id="xref-ref-36-6" class="xref-bibr" href="#ref-36">Santure and Spencer 2006</a>). Note that here “<span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-226.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-226.gif"/></noscript></span></span>” represents either of an <em>A</em><sub>1</sub> or <em>A</em><sub>2</sub> allele in the genotype. Breeding values are defined as the sum of additive effects of alleles for each genotype, for example the breeding value of the <em>A</em><sub>1</sub><em>A</em><sub>2</sub> genotype is <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-227.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-227.gif"/></noscript></span></span>. The additive variance is the variance of the additive allelic effects, while the dominance variance is the variance of the dominance deviations. By definition, the covariance between additive allelic and dominance effects is zero.</p><p id="p-53">In the absence of separate female and male variances, we follow <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fisher (1918)</a> and define the covariances between relatives as sums of additive and dominance variances.</p></div><div class="section" id="sec-10"><h3>Approach 2b (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Lynch and Walsh 1998</a>)</h3><p id="p-54">Approach 2a (above, and in <a id="xref-ref-36-7" class="xref-bibr" href="#ref-36">Santure and Spencer 2006</a>) calculated total additive and dominance effects and did not allow separate calculation of female and male additive and dominance variances as was possible in Approach 1. By treating individuals as parents in terms of the alleles that they will pass onto offspring in the next generation, we can redefine the genotypic value of an individual as the sum of additive effects inherited by its offspring, plus the population mean and a sex-specific dominance deviation (<a id="xref-ref-36-8" class="xref-bibr" href="#ref-36">Santure and Spencer 2006</a>). In using these definitions, we partition the additive and dominance terms into those specific to male and female inheritance.</p><p id="p-55">Now the partitioning of the genotypic value becomes different for males and females:</p><span class="disp-formula" id="disp-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-228.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-228.gif"/></noscript></span></span><p id="p-56">where the extra subscript on <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-229.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-229.gif"/></noscript></span></span> indicates female (<em>f</em>) and male (<em>m</em>) dominance effects, defined as</p><span class="disp-formula" id="disp-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-230.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-230.gif"/></noscript></span></span><p id="p-57">and</p><span class="disp-formula" id="disp-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-231.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-231.gif"/></noscript></span></span><p id="p-58">Male and female breeding values are then defined as the sum of male and female additive effects;</p><span class="disp-formula" id="disp-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-232.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-232.gif"/></noscript></span></span><p id="p-59">The male and female additive genetic variances are the variances of male and female additive effects, dominance genetic variances are the variances of the dominance deviations, and the covariance between dominance deviations and breeding values is similarly calculated separately for males and females. Covariances between relatives are then calculated following <a id="xref-ref-38-9" class="xref-bibr" href="#ref-38">Spencer (2002)</a> as sums of additive, dominance and covariance terms.</p></div><div class="section" id="sec-11"><h3>Approach 3a (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Fisher 1918</a>; <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Lynch and Walsh 1998</a>)</h3><p id="p-60">An alternative approach is to follow a regression method, expressing the genotypic value <em>G<sub>ij</sub></em> of the <em>A<sub>i</sub>A<sub>j</sub></em> genotype using least squares regression (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Fisher 1918</a>): based on the relationship between the number of copies of the <em>A</em><sub>2</sub> allele in the genotype and the genotypic value, we may define <em>G<sub>ij</sub></em> as the sum of a predicted regression value <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-233.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-233.gif"/></noscript></span></span> and a residual error corresponding to a dominance deviation <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-234.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-234.gif"/></noscript></span></span>. The predicted regression value may be further decomposed into the mean of the genotypes <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-235.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-235.gif"/></noscript></span></span> plus additive effects <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-236.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-236.gif"/></noscript></span></span>, where additive effects are linear terms dependent on the number of <em>A</em><sub>1</sub> and <em>A</em><sub>2</sub> alleles in the genotype (<em>N</em><sub>1</sub> and <em>N</em><sub>2</sub> = (2 − <em>N</em><sub>1</sub>) respectively), so that</p><span class="disp-formula" id="disp-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-237.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-237.gif"/></noscript></span></span><p id="p-61">Breeding values, dominance terms, variances, and covariances are calculated as in Approach 2a. By definition, the covariance between additive and dominance terms is zero.</p></div><div class="section" id="sec-12"><h3>Approach 3b (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Lynch and Walsh 1998</a>)</h3><p id="p-62">Alternatively, we may extend to a multiple regression approach to dissect the genotypic value into additive and dominance components. Using matrix notation, we can express the genotypic value as</p><span class="disp-formula" id="disp-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-238.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-238.gif"/></noscript></span></span><p id="p-63">where <em>G<sub>ij</sub></em> is the matrix of genotypic values, X is an incidence matrix, <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-239.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-239.gif"/></noscript></span></span> is the vector of the intercept (<span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-240.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-240.gif"/></noscript></span></span>) and the two parental partial regression coefficients <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-241.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-241.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-242.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-242.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-243.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-243.gif"/></noscript></span></span> is the vector of dominance effects:</p><span class="disp-formula" id="disp-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-244.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-244.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-245.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-245.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-246.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-246.gif"/></noscript></span></span><p id="p-64">The terms <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-247.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-247.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-248.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-248.gif"/></noscript></span></span> may then be estimated using a generalized least squares approach, so that</p><span class="disp-formula" id="disp-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-249.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-249.gif"/></noscript></span></span><p id="p-65">where</p><span class="disp-formula" id="disp-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-250.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-250.gif"/></noscript></span></span><p id="p-66">is the matrix of genotypic frequencies. Additive effects for each genotype are defined as the difference between the genotypic value and the sum of the population mean and dominance effect. Breeding values, dominance terms, variances and covariances are calculated as in Approach 2a. By definition, the covariance between additive and dominance terms is zero.</p></div><div class="section" id="sec-13"><h3>Response to selection</h3><p id="p-67">We here derive the full set of covariances and expectations which, in addition to equations (22)–(26), are required to describe the response to selection of an imprinted trait [equations (20) and (21)].</p><p id="p-68">The covariances between genotypic values before and after selection are<span class="disp-formula" id="disp-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-251.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-251.gif"/></noscript></span></span>and</p><span class="disp-formula" id="disp-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-252.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-252.gif"/></noscript></span></span><p id="p-69">The covariance between selection coefficients and genotypic values after selection are</p><span class="disp-formula" id="disp-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-253.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-253.gif"/></noscript></span></span><p id="p-70">Thus, although the mean values of offspring after selection for female and male parents differ, the relationship between selection coefficients and the difference in genotypic values before and after selection are the same for the offspring of female and male parents. Other covariances are shown below</p><span class="disp-formula" id="disp-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-254.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-254.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-255.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-255.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-256.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-256.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-257.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-257.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-258.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-258.gif"/></noscript></span></span><p id="p-71">Now we can find the components of the response to selection. Recalling equations (20) and (21), the components of equation (20) for females are<span class="disp-formula" id="disp-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-259.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-259.gif"/></noscript></span> <span class="disp-formula-label">(39)</span></span><span class="disp-formula" id="disp-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-260.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-260.gif"/></noscript></span> <span class="disp-formula-label">(40)</span></span><span class="disp-formula" id="disp-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-261.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-261.gif"/></noscript></span> <span class="disp-formula-label">(41)</span></span><span class="disp-formula" id="disp-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-262.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-262.gif"/></noscript></span> <span class="disp-formula-label">(42)</span></span><span class="disp-formula" id="disp-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-263.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-263.gif"/></noscript></span> <span class="disp-formula-label">(43)</span></span>and similarly, the components of (20) for males are</p><span class="disp-formula" id="disp-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-264.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-264.gif"/></noscript></span> <span class="disp-formula-label">(44)</span></span><span class="disp-formula" id="disp-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-265.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-265.gif"/></noscript></span> <span class="disp-formula-label">(45)</span></span><span class="disp-formula" id="disp-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-266.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-266.gif"/></noscript></span> <span class="disp-formula-label">(46)</span></span><span class="disp-formula" id="disp-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-267.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-267.gif"/></noscript></span> <span class="disp-formula-label">(47)</span></span><span class="disp-formula" id="disp-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-268.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-268.gif"/></noscript></span> <span class="disp-formula-label">(48)</span></span><p id="p-72">Interestingly, <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-269.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-269.gif"/></noscript></span></span> —the covariance between selection coefficients and the change in mean genetic value before and after selection—is the same for offspring of male and female parents. Then we find that the male and female sum of equation (20) components are<span class="disp-formula" id="disp-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-270.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-270.gif"/></noscript></span> <span class="disp-formula-label">(49)</span></span>where<span class="disp-formula" id="disp-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-271.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-271.gif"/></noscript></span> <span class="disp-formula-label">(50)</span></span>and</p><span class="disp-formula" id="disp-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-272.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-272.gif"/></noscript></span> <span class="disp-formula-label">(51)</span></span><p id="p-73">Hence</p><span class="disp-formula" id="disp-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-273.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-273.gif"/></noscript></span></span><p id="p-74">For equation (21), the extra terms we need to define are <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-274.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-274.gif"/></noscript></span></span> for males and females, and <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-275.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-275.gif"/></noscript></span></span><strong>:</strong><span class="disp-formula" id="disp-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-276.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-276.gif"/></noscript></span> <span class="disp-formula-label">(52)</span></span><span class="disp-formula" id="disp-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-277.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-277.gif"/></noscript></span> <span class="disp-formula-label">(53)</span></span>and<span class="disp-formula" id="disp-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-278.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-278.gif"/></noscript></span> <span class="disp-formula-label">(54)</span></span>and as expected, the sum of equation (21) components for females and males is</p><span class="disp-formula" id="disp-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-279.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2/131/embed/mml-math-279.gif"/></noscript></span></span></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-49"><em>Communicating editor: Mark Johnston</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2010-11-09"><span class="received-label">Received </span>November 9, 2010.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-03-13"><span class="accepted-label">Accepted </span>March 13, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2011 Santure, Spencer</li></ul><div class="license" id="license-1"><p id="p-2">This is an open access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.2.131.1" data-doi="10.1086/504045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dai</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weeks</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Ordered genotypes: An extended ITO method and a general formula for genetic covariance.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">1035</span>–<span class="cit-lpage">1045</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DDai%26rft.auinit1%253DF.%26rft.volume%253D78%26rft.issue%253D6%26rft.spage%253D1035%26rft.epage%253D1045%26rft.atitle%253DOrdered%2Bgenotypes%253A%2Ban%2Bextended%2BITO%2Bmethod%2Band%2Ba%2Bgeneral%2Bformula%2Bfor%2Bgenetic%2Bcovariance.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F504045%26rft_id%253Dinfo%253Apmid%252F16685653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/504045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16685653&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.2" data-doi="10.1016/S0301-6226(01)00226-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Koning</span>  <span class="cit-name-given-names">D.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rattink</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harlizius</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groenen</span>  <span class="cit-name-given-names">M. A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brascamp</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Detection and characterization of quantitative trait loci for growth and reproduction traits in pigs.</span> <abbr class="cit-jnl-abbrev">Livest. Prod. Sci.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BProd.%2BSci.%26rft.volume%253D72%26rft.spage%253D185%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0301-6226%252801%252900226-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0301-6226(01)00226-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.3" data-doi="10.1073/pnas.140216397"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Koning</span>  <span class="cit-name-given-names">D.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rattink</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harlizius</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Arendonk</span>  <span class="cit-name-given-names">J. A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brascamp</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genome-wide scan for body composition in pigs reveals important role of imprinting.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">7947</span>–<span class="cit-lpage">7950</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.140216397%26rft_id%253Dinfo%253Apmid%252F10859367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTcvMTQvNzk0NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvMi8xMzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.4" data-doi="10.1007/BF00220813"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Vries</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerr</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tier</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Gametic imprinting effects on rate and composition of pig growth.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">1037</span>–<span class="cit-lpage">1042</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D88%26rft.spage%253D1037%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00220813%26rft_id%253Dinfo%253Apmid%252F24186259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00220813&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24186259&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.5" data-doi="10.1016/0092-8674(91)90513-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DeChiara</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Efstratiadis</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Parental imprinting of the mouse insulin-like growth factor-II gene.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">849</span>–<span class="cit-lpage">859</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DDeChiara%26rft.auinit1%253DT.%2BM.%26rft.volume%253D64%26rft.issue%253D4%26rft.spage%253D849%26rft.epage%253D859%26rft.atitle%253DParental%2Bimprinting%2Bof%2Bthe%2Bmouse%2Binsulin-like%2Bgrowth%2Bfactor%2BII%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252891%252990513-X%26rft_id%253Dinfo%253Apmid%252F1997210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(91)90513-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1997210&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991EZ47800019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.6" data-doi="10.1046/j.1439-0388.2002.00323.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Engellandt</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tier</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genetic variances due to imprinted genes in cattle.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">154</span>–<span class="cit-lpage">165</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D119%26rft.spage%253D154%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1439-0388.2002.00323.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1439-0388.2002.00323.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.7" data-doi="10.1046/j.1439-0388.2002.00334.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Essl</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voith</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genomic imprinting effects on dairy- and fitness-related traits in cattle.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">182</span>–<span class="cit-lpage">189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D119%26rft.spage%253D182%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1439-0388.2002.00334.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1439-0388.2002.00334.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-book" id="cit-1.2.131.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ewens</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-source">Mathematical Population Genetics</span>. <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">Berlin</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.9" data-doi="10.1017/S0016672300022825"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">A note on Fisher’s “average effect” and “average excess”.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">347</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DFalconer%26rft.auinit1%253DD.%2BS.%26rft.volume%253D46%26rft.issue%253D3%26rft.spage%253D337%26rft.epage%253D347%26rft.atitle%253DA%2Bnote%2Bon%2BFisher%2527s%2B%2527average%2Beffect%2527%2Band%2B%2527average%2Bexcess%2527.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300022825%26rft_id%253Dinfo%253Apmid%252F4092925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300022825&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4092925&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-1.2.131.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source">Introduction to Quantitative Genetics</span>, <span class="cit-edition">Ed. 4</span>. <span class="cit-publ-name">Longman</span>, <span class="cit-publ-loc">Harlow, Essex, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.11" data-doi="10.1017/S0080456800012163"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1918</span> <span class="cit-article-title">The correlation between relatives on the supposition of Mendelian inheritance.</span> <abbr class="cit-jnl-abbrev">Trans. R. Soc. Edinb.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrans.%2BR.%2BSoc.%2BEdinb.%26rft.volume%253D52%26rft.spage%253D399%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0080456800012163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0080456800012163&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.12" data-doi="10.1111/j.1469-1809.1941.tb02272.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1941</span> <span class="cit-article-title">Average excess and average effect of a gene substitution.</span> <abbr class="cit-jnl-abbrev">Ann. Eugen.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BEugen.%26rft.volume%253D11%26rft.spage%253D53%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1941.tb02272.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1941.tb02272.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.13" data-doi="10.1046/j.1365-2540.2000.00698.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodnight</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Quantitative trait loci and gene interaction: the quantitative genetics of metapopulations.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">587</span>–<span class="cit-lpage">598</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D84%26rft.spage%253D587%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2540.2000.00698.x%26rft_id%253Dinfo%253Apmid%252F10849084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2540.2000.00698.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10849084&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.14" data-doi="10.1111/j.1558-5646.2009.00638.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hager</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheverud</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Relative contribution of additive, dominance, and imprinting effects to phenotypic variation in body size and growth between divergent selection lines of mice.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">1118</span>–<span class="cit-lpage">1128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D63%26rft.spage%253D1118%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2009.00638.x%26rft_id%253Dinfo%253Apmid%252F19187253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2009.00638.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19187253&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">Genotypic covariances between inbred relatives.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1319</span>–<span class="cit-lpage">1348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHarris%26rft.auinit1%253DD.%2BL.%26rft.volume%253D50%26rft.issue%253D6%26rft.spage%253D1319%26rft.epage%253D1348%26rft.atitle%253DGENOTYPIC%2BCOVARIANCES%2BBETWEEN%2BINBRED%2BRELATIVES%26rft_id%253Dinfo%253Apmid%252F14239792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo5OiI1MC82LzEzMTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzIvMTMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.16" data-doi="10.1111/j.0014-3820.2005.tb00939.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heywood</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">An exact form of the breeder’s equation for the evolution of a quantitative trait under natural selection.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">2287</span>–<span class="cit-lpage">2298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D59%26rft.spage%253D2287%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0014-3820.2005.tb00939.x%26rft_id%253Dinfo%253Apmid%252F16396170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0014-3820.2005.tb00939.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16396170&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hirooka</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Koning</span>  <span class="cit-name-given-names">D.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harlizius</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Arendonk</span>  <span class="cit-name-given-names">J. A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rattink</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A whole-genome scan for quantitative trait loci affecting teat number in pigs.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">2320</span>–<span class="cit-lpage">2326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DHirooka%26rft.auinit1%253DH.%26rft.volume%253D79%26rft.issue%253D9%26rft.spage%253D2320%26rft.epage%253D2326%26rft.atitle%253DA%2Bwhole-genome%2Bscan%2Bfor%2Bquantitative%2Btrait%2Bloci%2Baffecting%2Bteat%2Bnumber%2Bin%2Bpigs%26rft_id%253Dinfo%253Apmid%252F11583418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11583418&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171208900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-other" id="cit-1.2.131.18"><div class="cit-metadata"><cite><span class="cit-comment">Kaiser, C. J., M. E. Goddard, and A. Reverter, 1998 Analysis of gametic imprinting effects for test day milk yield in Australian Holstein cattle. Proceedings of the 6th World Congress on Genetics Applied to Livestock Production, Armidale, NSW, Australia, Vol. 23, pp. 355–358</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marklund</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davies</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Multiple marker mapping of quantitative trait loci in a cross between outbred wild boar and large white pigs.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">1069</span>–<span class="cit-lpage">1080</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D149%26rft.spage%253D1069%26rft_id%253Dinfo%253Apmid%252F9611214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9611214&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000074028400050&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.20" data-doi="10.5713/ajas.2003.1087"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeon</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">H. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Detection of imprinted Quantitative Trait Loci (QTL) for growth traits in pigs.</span> <abbr class="cit-jnl-abbrev">Asian-Australas. J. Anim. Sci.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1087</span>–<span class="cit-lpage">1092</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAsian-Australas.%2BJ.%2BAnim.%2BSci.%26rft.volume%253D16%26rft.spage%253D1087%26rft_id%253Dinfo%253Adoi%252F10.5713%252Fajas.2003.1087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5713/ajas.2003.1087&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.2.131.21" data-doi="10.2307/3001590"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sacks</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1954</span> <span class="cit-article-title">The derivation of joint distribution and correlation between relatives by the use of stochastic matrices.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D10%26rft.spage%253D347%26rft.atitle%253DBIOMETRICS%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3001590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3001590&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1954UG27000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.22" data-doi="10.1101/gr.6584707"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luedi</span>  <span class="cit-name-given-names">P. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dietrich</span>  <span class="cit-name-given-names">F. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weidman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bosko</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jirtle</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Computational and experimental identification of novel human imprinted genes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1723</span>–<span class="cit-lpage">1730</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6584707%26rft_id%253Dinfo%253Apmid%252F18055845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNy8xMi8xNzIzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS8yLzEzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-1.2.131.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source">Genetics and Analysis of Quantitative Traits</span>. <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.24" data-doi="10.1051/gse:2002026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Milan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bidanel</span>  <span class="cit-name-given-names">J.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iannuccelli</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riquet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amigues</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Detection of quantitative trait loci for carcass composition traits in pigs.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">705</span>–<span class="cit-lpage">728</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DMilan%26rft.auinit1%253DD.%26rft.volume%253D34%26rft.issue%253D6%26rft.spage%253D705%26rft.epage%253D728%26rft.atitle%253DDetection%2Bof%2Bquantitative%2Btrait%2Bloci%2Bfor%2Bcarcass%2Bcomposition%2Btraits%2Bin%2Bpigs.%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A2002026%26rft_id%253Dinfo%253Apmid%252F12486399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:2002026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12486399&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179842800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.25" data-doi="10.1016/j.tig.2005.06.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morison</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramsay</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spencer</span>  <span class="cit-name-given-names">H. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A census of mammalian imprinting.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">457</span>–<span class="cit-lpage">465</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DMorison%26rft.auinit1%253DI.%2BM.%26rft.volume%253D21%26rft.issue%253D8%26rft.spage%253D457%26rft.epage%253D465%26rft.atitle%253DA%2Bcensus%2Bof%2Bmammalian%2Bimprinting.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2005.06.008%26rft_id%253Dinfo%253Apmid%252F15990197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2005.06.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15990197&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231209200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.26" data-doi="10.1016/j.tpb.2007.02.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagylaki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lou</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Evolution under multiallelic migration–selection models.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DNagylaki%26rft.auinit1%253DT.%26rft.volume%253D72%26rft.issue%253D1%26rft.spage%253D21%26rft.epage%253D40%26rft.atitle%253DEvolution%2Bunder%2Bmultiallelic%2Bmigration-selection%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tpb.2007.02.005%26rft_id%253Dinfo%253Apmid%252F17470373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tpb.2007.02.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17470373&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.27" data-doi="10.2174/1389202043349183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Naumova</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Croteau</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Mechanisms of epigenetic variation: polymorphic imprinting.</span> <abbr class="cit-jnl-abbrev">Curr. Genomics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">429</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BGenomics%26rft.volume%253D5%26rft.spage%253D417%26rft_id%253Dinfo%253Adoi%252F10.2174%252F1389202043349183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2174/1389202043349183&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000222098000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.28" data-doi="10.1007/PL00008182"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Neill</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ingram</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vrana</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tilghman</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Allelic expression of IGF2 in marsupials and birds.</span> <abbr class="cit-jnl-abbrev">Dev. Genes Evol.</abbr> <span class="cit-vol">210</span>: <span class="cit-fpage">18</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%2Bgenes%2Band%2Bevolution%26rft.stitle%253DDev%2BGenes%2BEvol%26rft.aulast%253DO%2527Neill%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D210%26rft.issue%253D1%26rft.spage%253D18%26rft.epage%253D20%26rft.atitle%253DAllelic%2Bexpression%2Bof%2BIGF2%2Bin%2Bmarsupials%2Band%2Bbirds.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FPL00008182%26rft_id%253Dinfo%253Apmid%252F10603082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/PL00008182&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10603082&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084010900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.29" data-doi="10.1038/227520a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Selection and Covariance.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">227</span>: <span class="cit-fpage">520</span>–<span class="cit-lpage">521</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPrice%26rft.auinit1%253DG.%2BR.%26rft.volume%253D227%26rft.issue%253D5257%26rft.spage%253D520%26rft.epage%253D521%26rft.atitle%253DSelection%2Band%2Bcovariance.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F227520a0%26rft_id%253Dinfo%253Apmid%252F5428476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/227520a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5428476&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1970G904200067&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.30" data-doi="10.1111/j.1469-1809.1957.tb01874.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Extension of covariance selection mathematics.</span> <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">485</span>–<span class="cit-lpage">490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAnn%2BHum%2BGenet%26rft.aulast%253DPrice%26rft.auinit1%253DG.%2BR.%26rft.volume%253D35%26rft.issue%253D4%26rft.spage%253D485%26rft.epage%253D490%26rft.atitle%253DExtension%2Bof%2Bcovariance%2Bselection%2Bmathematics.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1957.tb01874.x%26rft_id%253Dinfo%253Apmid%252F5073694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1957.tb01874.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5073694&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1972M377200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.31" data-doi="10.1051/gse:2002003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quintanilla</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bidanel</span>  <span class="cit-name-given-names">J.-P.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A further look at quantitative trait loci affecting growth and fatness in a cross between Mieshan and Large White pig populations.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DQuintanilla%26rft.auinit1%253DR.%26rft.volume%253D34%26rft.issue%253D2%26rft.spage%253D193%26rft.epage%253D210%26rft.atitle%253DA%2Bfurther%2Blook%2Bat%2Bquantitative%2Btrait%2Bloci%2Baffecting%2Bgrowth%2Band%2Bfatness%2Bin%2Ba%2Bcross%2Bbetween%2BMeishan%2Band%2BLarge%2BWhite%2Bpig%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A2002003%26rft_id%253Dinfo%253Apmid%252F12081807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:2002003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12081807&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.32" data-doi="10.1002/jcb.10352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rand</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cedar</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Regulation of imprinting: A multi-tiered process.</span> <abbr class="cit-jnl-abbrev">J. Cell. Biochem.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">400</span>–<span class="cit-lpage">407</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcellular%2Bbiochemistry%26rft.stitle%253DJ%2BCell%2BBiochem%26rft.aulast%253DRand%26rft.auinit1%253DE.%26rft.volume%253D88%26rft.issue%253D2%26rft.spage%253D400%26rft.epage%253D407%26rft.atitle%253DRegulation%2Bof%2Bimprinting%253A%2BA%2Bmulti-tiered%2Bprocess.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcb.10352%26rft_id%253Dinfo%253Apmid%252F12520543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcb.10352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12520543&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180479100029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.33" data-doi="10.1007/s003350010117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rattink</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Koning</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faivre</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harlizius</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Arendonk</span>  <span class="cit-name-given-names">J. A. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Fine mapping and imprinting analysis for fatness trait QTLs in pigs.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">656</span>–<span class="cit-lpage">661</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DRattink%26rft.auinit1%253DA.%2BP.%26rft.volume%253D11%26rft.issue%253D8%26rft.spage%253D656%26rft.epage%253D661%26rft.atitle%253DFine%2Bmapping%2Band%2Bimprinting%2Banalysis%2Bfor%2Bfatness%2Btrait%2BQTLs%2Bin%2Bpigs.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs003350010117%26rft_id%253Dinfo%253Apmid%252F10920235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s003350010117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10920235&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088453300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.34" data-doi="10.1093/hmg/ddi218"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sandovici</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kassovska-Bratinova</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loredo-Osti</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leppert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suarez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Interindividual variability and parent of origin DNA methylation differences at specific human Alu elements.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2135</span>–<span class="cit-lpage">2143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddi218%26rft_id%253Dinfo%253Apmid%252F15972727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/hmg/ddi218&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15972727&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230725000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.35" data-doi="10.1093/hmg/ddg167"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sandovici</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leppert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hawk</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suarez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linares</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Familial aggregation of abnormal methylation of parental alleles at the IGF2/H19 and IGF2R differentially methylated regions.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1569</span>–<span class="cit-lpage">1578</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2BMolecular%2BGenetics%26rft.stitle%253DHum%2BMol%2BGenet%26rft.aulast%253DSandovici%26rft.auinit1%253DI.%26rft.volume%253D12%26rft.issue%253D13%26rft.spage%253D1569%26rft.epage%253D1578%26rft.atitle%253DFamilial%2Baggregation%2Bof%2Babnormal%2Bmethylation%2Bof%2Bparental%2Balleles%2Bat%2Bthe%2BIGF2%252FH19%2Band%2BIGF2R%2Bdifferentially%2Bmethylated%2Bregions%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddg167%26rft_id%253Dinfo%253Apmid%252F12812984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/hmg/ddg167&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12812984&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183663800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.36" data-doi="10.1534/genetics.105.049494"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Santure</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spencer</span>  <span class="cit-name-given-names">H. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Influence of mom and dad: Quantitative genetic models for maternal effects and genomic imprinting.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">2297</span>–<span class="cit-lpage">2316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.049494%26rft_id%253Dinfo%253Apmid%252F16751674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My80LzIyOTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzIvMTMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.37" data-doi="10.3168/jds.S0022-0302(89)79231-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kennedy</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">The inverse of the gametic relationship matrix.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">1266</span>–<span class="cit-lpage">1272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DSchaeffer%26rft.auinit1%253DL.%2BR.%26rft.volume%253D72%26rft.issue%253D5%26rft.spage%253D1266%26rft.epage%253D1272%26rft.atitle%253DThe%2BInverse%2Bof%2Bthe%2BGametic%2BRelationship%2BMatrix%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252889%252979231-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(89)79231-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spencer</span>  <span class="cit-name-given-names">H. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The correlation between relatives on the supposition of genomic imprinting.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">411</span>–<span class="cit-lpage">417</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSpencer%26rft.auinit1%253DH.%2BG.%26rft.volume%253D161%26rft.issue%253D1%26rft.spage%253D411%26rft.epage%253D417%26rft.atitle%253DThe%2BCorrelation%2BBetween%2BRelatives%2Bon%2Bthe%2BSupposition%2Bof%2BGenomic%2BImprinting%26rft_id%253Dinfo%253Apmid%252F12019254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYxLzEvNDExIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS8yLzEzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stella</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stalder</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saxton</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boettcher</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Estimation of variances for gametic effects on litter size in Yorkshire and Landrace swine.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">2171</span>–<span class="cit-lpage">2178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DStella%26rft.auinit1%253DA.%26rft.volume%253D81%26rft.issue%253D9%26rft.spage%253D2171%26rft.epage%253D2178%26rft.atitle%253DEstimation%2Bof%2Bvariances%2Bfor%2Bgametic%2Beffects%2Bon%2Blitter%2Bsize%2Bin%2BYorkshire%2Band%2BLandrace%2Bswine%26rft_id%253Dinfo%253Apmid%252F12968691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12968691&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.40" data-doi="10.1017/S0016672300026756"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Templeton</span>  <span class="cit-name-given-names">A. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The general relationship between average effect and average excess.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DTempleton%26rft.auinit1%253DA.%2BR.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D69%26rft.epage%253D70%26rft.atitle%253DThe%2Bgeneral%2Brelationship%2Bbetween%2Baverage%2Beffect%2Band%2Baverage%2Bexcess.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300026756%26rft_id%253Dinfo%253Apmid%252F3569910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300026756&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3569910&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987G654900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.131.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tier</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solkner</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Analysing gametic variation with an animal model.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">868</span>–<span class="cit-lpage">872</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D85%26rft.spage%253D868%26rft_id%253Dinfo%253Apmid%252F24196062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24196062&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/1/2/131.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/2/117" title="Fluorescence In Situ Hybridization–Based Karyotyping of Soybean Translocation Lines" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/2/143" title="A Screen for Germination Mutants in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/1/2"><span>Volume 1 Issue 2, July 2011</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/1/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/1/2.cover.gif?itok=hCLGYhoa" width="248" height="330" alt="G3: Genes, Genomes, Genetics: 1 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/2/131.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-HlaXlytPkZXiPdFLd_QbhVQugp-xkhTxOnSKRcLfoGQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigation-1" class="" data-icon-position="" data-hide-link-title="0">Investigation</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/1/2/131" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/1/2/131" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/1/2/131" class="active" data-icon-position="" data-hide-link-title="0">Quantitative Genetics of Genomic Imprinting: A Comparison of Simple Variance Derivations, the Effects of Inbreeding, and Response to Selection</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459163" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Kp87_rfIGTyOqFD2VBCahbebLNaU1IptyCTTG98F6RU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/2/131.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/1/2/131" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Px8OCLoAD6KKjpmhYmAD1quWARrZ-DfERoAjZGnoS-I" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/1/2/131" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/2/131.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459163" id="node459163--2" data-pisa="ggg;1/2/131" data-pisa-master="ggg;g3.111.000042" data-apath="/ggg/1/2/131.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Quantitative Genetics of Genomic Imprinting: A Comparison of Simple Variance Derivations, the Effects of Inbreeding, and Response to Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Anna W. Santure</span> and <span class="highwire-citation-author" data-delta="1">Hamish G. Spencer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">131-142;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000042 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459163--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anna W. Santure </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Allan Wilson Centre for Molecular Ecology and Evolution, University of Otago, Dunedin, New Zealand</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Zoology, University of Otago, Dunedin, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BW.%2BSanture%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Santure%20AW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnna%2BW.%2BSanture%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:asanture@gmail.com" class="" data-icon-position="" data-hide-link-title="0">asanture@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Hamish G. Spencer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Allan Wilson Centre for Molecular Ecology and Evolution, University of Otago, Dunedin, New Zealand</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Zoology, University of Otago, Dunedin, New Zealand</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>National Research Centre for Growth and Development, University of Otago, Dunedin, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hamish%2BG.%2BSpencer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spencer%20HG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHamish%2BG.%2BSpencer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/1/2/131" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/1/2/131&amp;title=Quantitative%20Genetics%20of%20Genomic%20Imprinting%3A%20A%20Comparison%20of%20Simple%20Variance%20Derivations%2C%20the%20Effects%20of%20Inbreeding%2C%20and%20Response%20to%20Selection" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/1/2/131&amp;title=Quantitative%20Genetics%20of%20Genomic%20Imprinting%3A%20A%20Comparison%20of%20Simple%20Variance%20Derivations%2C%20the%20Effects%20of%20Inbreeding%2C%20and%20Response%20to%20Selection" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/1/2/131&amp;title=Quantitative%20Genetics%20of%20Genomic%20Imprinting%3A%20A%20Comparison%20of%20Simple%20Variance%20Derivations%2C%20the%20Effects%20of%20Inbreeding%2C%20and%20Response%20to%20Selection" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/1/2/131&amp;text=Quantitative%20Genetics%20of%20Genomic%20Imprinting%3A%20A%20Comparison%20of%20Simple%20Variance%20Derivations%2C%20the%20Effects%20of%20Inbreeding%2C%20and%20Response%20to%20Selection" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/1/2/131&amp;title=Quantitative%20Genetics%20of%20Genomic%20Imprinting%3A%20A%20Comparison%20of%20Simple%20Variance%20Derivations%2C%20the%20Effects%20of%20Inbreeding%2C%20and%20Response%20to%20Selection" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/1/2/131&amp;t=Quantitative%20Genetics%20of%20Genomic%20Imprinting%3A%20A%20Comparison%20of%20Simple%20Variance%20Derivations%2C%20the%20Effects%20of%20Inbreeding%2C%20and%20Response%20to%20Selection" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/1/2/131&amp;title=Quantitative%20Genetics%20of%20Genomic%20Imprinting%3A%20A%20Comparison%20of%20Simple%20Variance%20Derivations%2C%20the%20Effects%20of%20Inbreeding%2C%20and%20Response%20to%20Selection" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/1/2/131&amp;title=Quantitative%20Genetics%20of%20Genomic%20Imprinting%3A%20A%20Comparison%20of%20Simple%20Variance%20Derivations%2C%20the%20Effects%20of%20Inbreeding%2C%20and%20Response%20to%20Selection" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459163/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459163" id="node459163--3" data-pisa="ggg;1/2/131" data-pisa-master="ggg;g3.111.000042" data-apath="/ggg/1/2/131.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Quantitative Genetics of Genomic Imprinting: A Comparison of Simple Variance Derivations, the Effects of Inbreeding, and Response to Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Anna W. Santure</span> and <span class="highwire-citation-author" data-delta="1">Hamish G. Spencer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">131-142;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000042 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459163--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anna W. Santure </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Allan Wilson Centre for Molecular Ecology and Evolution, University of Otago, Dunedin, New Zealand</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Zoology, University of Otago, Dunedin, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BW.%2BSanture%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Santure%20AW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnna%2BW.%2BSanture%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:asanture@gmail.com" class="" data-icon-position="" data-hide-link-title="0">asanture@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Hamish G. Spencer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Allan Wilson Centre for Molecular Ecology and Evolution, University of Otago, Dunedin, New Zealand</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Zoology, University of Otago, Dunedin, New Zealand</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>National Research Centre for Growth and Development, University of Otago, Dunedin, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hamish%2BG.%2BSpencer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spencer%20HG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHamish%2BG.%2BSpencer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=131&amp;rft.epage=142&amp;rft.atitle=Quantitative%20Genetics%20of%20Genomic%20Imprinting%3A%20A%20Comparison%20of%20Simple%20Variance%20Derivations%2C%20the%20Effects%20of%20Inbreeding%2C%20and%20Response%20to%20Selection&amp;rft.volume=1&amp;rft.issue=2&amp;rft.date=2011-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Santure%2C+Anna+W.&amp;rft.au=Spencer%2C+Hamish+G."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459163/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459163/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459163/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459163/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459163/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459163/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459163/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459163/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459163/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459163/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459163/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459163/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496051" id="node496051" data-pisa="ggg;g3.120.400867v1" data-pisa-master="ggg;g3.120.400867" data-apath="/ggg/early/2020/03/26/g3.120.400867.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/26/g3.120.400867" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Comparative Cytology of Female Meiosis 1 Among <em>Drosophila</em> Species</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496050" id="node496050" data-pisa="ggg;g3.120.401055v1" data-pisa-master="ggg;g3.120.401055" data-apath="/ggg/early/2020/03/26/g3.120.401055.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/26/g3.120.401055" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Anchor Away - A Fast, Reliable and Reversible Technique To Inhibit Proteins in <em>Drosophila melanogaster</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496048" id="node496048" data-pisa="ggg;g3.119.400937v1" data-pisa-master="ggg;g3.119.400937" data-apath="/ggg/early/2020/03/25/g3.119.400937.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/25/g3.119.400937" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tunable Transcriptional Interference at the Endogenous Alcohol Dehydrogenase Gene Locus in <em>Drosophila melanogaster</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigation" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigation</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/1/2/131" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">The Model</a></li><li class="odd"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Approaches</a></li><li class="even"><a href="#sec-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Inbreeding and Imprinting</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Response to Selection</a></li><li class="even"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgements</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/1/2/131.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/1/2/131.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/1/2/131.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
