<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/1/2/93" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/1/2/93.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/1/2/93.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/1/2/93.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2.cover.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/1/2/93.atom" />
<meta name="HW.pisa" content="ggg;1/2/93" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Translational Genomics in Legumes Allowed Placing In Silico 5460 Unigenes on the Pea Functional Map and Identified Candidate Genes in Pisum sativum L." />
<meta name="DC.Identifier" content="10.1534/g3.111.000349" />
<meta name="DC.Date" content="2011-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2011 Bordat et al.. This is an open access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="To identify genes involved in phenotypic traits, translational genomics from highly characterized model plants to poorly characterized crop plants provides a valuable source of markers to saturate a zone of interest as well as functionally characterized candidate genes. In this paper, an integrated view of the pea genetic map was developed. A series of gene markers were mapped and their best reciprocal homologs were identified on M. truncatula , L. japonicus , soybean, and poplar pseudomolecules. Based on the syntenic relationships uncovered between pea and M. truncatula , 5460 pea Unigenes were tentatively placed on the consensus map. A new bioinformatics tool, [http://www.thelegumeportal.net/pea\_mtr\_translational_toolkit][1], was developed that allows, for any gene sequence, to search its putative position on the pea consensus map and hence to search for candidate genes among neighboring Unigenes. As an example, a promising candidate gene for the hypernodulation mutation nod3 in pea was proposed based on the map position of the likely homolog of Pub1 , a M. truncatula gene involved in nodulation regulation. A broader view of pea genome evolution was obtained by revealing syntenic relationships between pea and sequenced genomes. Blocks of synteny were identified which gave new insights into the evolution of chromosome structure in Papillionoids and Eudicots. The power of the translational genomics approach was underlined.

 [1]: http://www.thelegumeportal.net/pea_mtr_translational_toolkit" />
<meta name="DC.Contributor" content="Amandine Bordat" />
<meta name="DC.Contributor" content="Vincent Savois" />
<meta name="DC.Contributor" content="Marie Nicolas" />
<meta name="DC.Contributor" content="Jérome Salse" />
<meta name="DC.Contributor" content="Aurélie Chauveau" />
<meta name="DC.Contributor" content="Michael Bourgeois" />
<meta name="DC.Contributor" content="Jean Potier" />
<meta name="DC.Contributor" content="Hervé Houtin" />
<meta name="DC.Contributor" content="Céline Rond" />
<meta name="DC.Contributor" content="Florent Murat" />
<meta name="DC.Contributor" content="Pascal Marget" />
<meta name="DC.Contributor" content="Grégoire Aubert" />
<meta name="DC.Contributor" content="Judith Burstin" />
<meta name="article:published_time" content="2011-07-01" />
<meta name="article:section" content="Investigation" />
<meta name="citation_title" content="Translational Genomics in Legumes Allowed Placing In Silico 5460 Unigenes on the Pea Functional Map and Identified Candidate Genes in Pisum sativum L." />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;To identify genes involved in phenotypic traits, translational genomics from highly characterized model plants to poorly characterized crop plants provides a valuable source of markers to saturate a zone of interest as well as functionally characterized candidate genes. In this paper, an integrated view of the pea genetic map was developed. A series of gene markers were mapped and their best reciprocal homologs were identified on &lt;i&gt;M. truncatula&lt;/i&gt;, &lt;i&gt;L. japonicus&lt;/i&gt;, soybean, and poplar pseudomolecules. Based on the syntenic relationships uncovered between pea and &lt;i&gt;M. truncatula&lt;/i&gt;, 5460 pea Unigenes were tentatively placed on the consensus map. A new bioinformatics tool, http://www.thelegumeportal.net/pea_mtr_translational_toolkit, was developed that allows, for any gene sequence, to search its putative position on the pea consensus map and hence to search for candidate genes among neighboring Unigenes. As an example, a promising candidate gene for the hypernodulation mutation &lt;i&gt;nod3&lt;/i&gt; in pea was proposed based on the map position of the likely homolog of &lt;i&gt;Pub1&lt;/i&gt;, a &lt;i&gt;M. truncatula&lt;/i&gt; gene involved in nodulation regulation. A broader view of pea genome evolution was obtained by revealing syntenic relationships between pea and sequenced genomes. Blocks of synteny were identified which gave new insights into the evolution of chromosome structure in Papillionoids and Eudicots. The power of the translational genomics approach was underlined.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2011/07/01" />
<meta name="citation_mjid" content="ggg;1/2/93" />
<meta name="citation_id" content="1/2/93" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/1/2/93" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/1/2/93.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/1/2/93.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/1/2/93.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.000349" />
<meta name="citation_pmid" content="22384322" />
<meta name="citation_volume" content="1" />
<meta name="citation_issue" content="2" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigation" />
<meta name="citation_firstpage" content="93" />
<meta name="citation_lastpage" content="103" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Amandine Bordat" />
<meta name="citation_author_institution" content="INRA, UMRLEG, 21065 Dijon Cedex, France" />
<meta name="citation_author" content="Vincent Savois" />
<meta name="citation_author_institution" content="INRA, UMRLEG, 21065 Dijon Cedex, France" />
<meta name="citation_author" content="Marie Nicolas" />
<meta name="citation_author_institution" content="INRA, UMRLEG, 21065 Dijon Cedex, France" />
<meta name="citation_author" content="Jérome Salse" />
<meta name="citation_author_institution" content="INRA, UMR1095, 63100 Clermont-Ferrand, France" />
<meta name="citation_author" content="Aurélie Chauveau" />
<meta name="citation_author_institution" content="INRA, UMRLEG, 21065 Dijon Cedex, France" />
<meta name="citation_author" content="Michael Bourgeois" />
<meta name="citation_author_institution" content="INRA, UMRLEG, 21065 Dijon Cedex, France" />
<meta name="citation_author" content="Jean Potier" />
<meta name="citation_author_institution" content="INRA, UMRLEG, 21065 Dijon Cedex, France" />
<meta name="citation_author" content="Hervé Houtin" />
<meta name="citation_author_institution" content="INRA, UMRLEG, 21065 Dijon Cedex, France" />
<meta name="citation_author" content="Céline Rond" />
<meta name="citation_author_institution" content="INRA, UMRLEG, 21065 Dijon Cedex, France" />
<meta name="citation_author" content="Florent Murat" />
<meta name="citation_author_institution" content="INRA, UMR1095, 63100 Clermont-Ferrand, France" />
<meta name="citation_author" content="Pascal Marget" />
<meta name="citation_author_institution" content="INRA, UMRLEG, 21065 Dijon Cedex, France" />
<meta name="citation_author" content="Grégoire Aubert" />
<meta name="citation_author_institution" content="INRA, UMRLEG, 21065 Dijon Cedex, France" />
<meta name="citation_author" content="Judith Burstin" />
<meta name="citation_author_institution" content="INRA, UMRLEG, 21065 Dijon Cedex, France" />
<meta name="citation_author_email" content="judith.burstin@dijon.inra.fr" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=M. Abrouk;citation_author=F. Murat;citation_author=C. Pont;citation_author=J. Messing;citation_author=S. Jackson;citation_title=Palaeogenomics of plants: synteny-based modelling of extinct ancestors.;citation_pages=479-487;citation_volume=15;citation_year=2010;citation_issue=9;citation_pmid=20638891;citation_doi=10.1016/j.tplants.2010.06.001" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=G. Aubert;citation_author=J. Morin;citation_author=J. Jacquin;citation_author=K. Loridon;citation_author=M. C. Quillet;citation_title=Functional mapping in pea, as an aid to the candidate gene selection and for investigating synteny with the model legume Medicago truncatula.;citation_pages=1024-1041;citation_volume=112;citation_year=2006;citation_issue=6;citation_pmid=16416153;citation_doi=10.1007/s00122-005-0205-y" />
<meta name="citation_reference" content="citation_journal_title=Agri. Hort. Genet.;citation_author=S. Blixt;citation_title=Mutation genetics in Pisum.;citation_pages=1-293;citation_volume=30;citation_year=1972" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=J. Burstin;citation_author=P. Marget;citation_author=M. Huart;citation_author=A. Moessner;citation_author=B. Mangin;citation_title=Developmental genes have pleiotropic effects on plant morphology and source capacity, eventually impacting on seed protein content and productivity in pea.;citation_volume=144;citation_year=2007;citation_pmid=17449650;citation_doi=10.1104/pp.107.096966" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. B. Cannon;citation_author=L. Sterck;citation_author=S. Rombauts;citation_author=S. Sato;citation_author=F. Cheung;citation_title=Legume genome evolution viewed through the Medicago truncatula and Lotus japonicus genomes.;citation_volume=103;citation_year=2006;citation_pmid=17003129;citation_doi=10.1073/pnas.0603228103" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. K. Choi;citation_author=J. H. Mun;citation_author=D. J. Kim;citation_author=H. Zhu;citation_author=J. M. Baek;citation_title=Estimating genome conservation between crop and model legume species.;citation_volume=101;citation_year=2004;citation_pmid=15489274;citation_doi=10.1073/pnas.0402251101" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. De Givry;citation_author=M. Bouchez;citation_author=P. Chabrier;citation_author=D. Milan;citation_author=T. Schiex;citation_title=CARTHAGENE: multipopulation integrated genetic and radiated hybrid mapping.;citation_volume=21;citation_year=2005;citation_pmid=15598829;citation_doi=10.1093/bioinformatics/bti222" />
<meta name="citation_reference" content="citation_journal_title=Plant Mol. Biol. Rep.;citation_author=S. L. Dellaporta;citation_author=J. Wood;citation_author=J. B. Hicks;citation_title=A plant DNA minipreparation: Version II.;citation_pages=19-21;citation_volume=1;citation_year=1983;citation_doi=10.1007/BF02712670" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=C. Deulvot;citation_author=H. Charrel;citation_author=A. Marty;citation_author=F. Jacquin;citation_author=C. Donnadieu;citation_title=Highly-multiplexed SNP genotyping for genetic mapping and germplasm diversity studies in pea.;citation_pages=468-468;citation_volume=11;citation_year=2010;citation_pmid=20701750;citation_doi=10.1186/1471-2164-11-468" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=J. J. Doyle;citation_author=M. A. Luckow;citation_title=The Rest of the Iceberg. Legume Diversity and Evolution in a Phylogenetic Context;citation_pages=900-910;citation_volume=131;citation_year=2003;citation_issue=3;citation_pmid=12644643;citation_doi=10.1104/pp.102.018150" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=T. H. N. Ellis;citation_author=S. J. Poyser;citation_title=An integrated and comparative view of pea genetic and cytogenetic maps.;citation_pages=17-25;citation_volume=153;citation_year=2002;citation_doi=10.1046/j.0028-646X.2001.00302.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Endre;citation_author=A. Kereszt;citation_author=Z. Kevei;citation_author=S. Mihacea;citation_author=P. Kaló;citation_title=A receptor kinase gene regulating symbiotic nodule development.;citation_pages=962-966;citation_volume=417;citation_year=2002;citation_issue=6892;citation_pmid=12087406;citation_doi=10.1038/nature00842" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=J. Forment;citation_author=F. Gilabert;citation_author=A. Robles;citation_author=V. Conejero;citation_author=F. Nuez;citation_title=EST2uni: an open, parallel tool for automated EST analysis and database creation, with a data mining web interface and microarray expression data integration.;citation_pages=5-5;citation_volume=9;citation_year=2008;citation_pmid=18179701;citation_doi=10.1186/1471-2105-9-5" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=C. H. Galeano;citation_author=A. C. Fernández;citation_author=M. Gómez;citation_author=M. B. Blair;citation_title=Single strand conformation polymorphism based SNP and Indel markers for genetic mapping and synteny analysis of common bean (Phaseolus vulgaris L.).;citation_pages=629-629;citation_volume=10;citation_year=2009;citation_pmid=20030833;citation_doi=10.1186/1471-2164-10-629" />
<meta name="citation_reference" content="citation_journal_title=Biochemical Journal;citation_journal_abbrev=Biochem. J.;citation_author=J. A. Gatehouse;citation_author=I. M. Evans;citation_author=D. Bown;citation_author=R. R. Croy;citation_author=D. Boulter;citation_title=Control of storage-protein synthesis during seed development in pea (Pisum sativum L.);citation_pages=119-127;citation_volume=208;citation_year=1982;citation_issue=1;citation_pmid=6897609;citation_doi=10.1042/bj2080119" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=J. George;citation_author=T. I. Sawbridge;citation_author=N.O.I. Cogan;citation_author=A.R. Gendall;citation_author=K. F. Smith;citation_title=Comparison of genome structure between white clover and Medicago truncatula supports homoeologous group nomenclature based on conserved synteny.;citation_pages=905-911;citation_volume=51;citation_year=2008;citation_issue=11;citation_pmid=18956023" />
<meta name="citation_reference" content="citation_journal_title=Journal of the American College of Nutrition;citation_journal_abbrev=J. Am. Coll. Nutr.;citation_author=M. A. Grusak;citation_title=Enhancing Mineral Content in Plant Food Products;citation_pages=178S-183;citation_volume=21;citation_year=2002;citation_issue=90003;citation_pmid=12071302;citation_doi=10.1080/07315724.2002.10719263" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=G. Gualtieri;citation_author=O. Kulikova;citation_author=E. Limpens;citation_author=D. J. Kim;citation_author=D. R. Cook;citation_title=Microsynteny between pea and Medicago truncatula in the SYM2 region.;citation_pages=225-235;citation_volume=50;citation_year=2002;citation_issue=2;citation_pmid=12175015;citation_doi=10.1023/A:1016085523752" />
<meta name="citation_reference" content="citation_journal_title=Br. J. Nutr.;citation_author=F. Guillon;citation_author=M. M. J. Champ;citation_title=Carbohydrates fractions of legumes: uses in human nutrition and potential for health.;citation_pages=293-306;citation_volume=88;citation_year=2002;citation_doi=10.1079/BJN2002720" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=X. Huang;citation_author=A. Madan;citation_title=CAP3: A DNA Sequence Assembly Program;citation_pages=868-877;citation_volume=9;citation_year=1999;citation_issue=9;citation_pmid=10508846;citation_doi=10.1101/gr.9.9.868" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=K. J. Hall;citation_author=J. S. Parker;citation_author=T. H. N. Ellis;citation_author=L. Turner;citation_author=M. R. Knox;citation_title=The relationship between genetic and cytogenetic maps of pea. II. Physical maps of linkage mapping populations.;citation_pages=755-769;citation_volume=40;citation_year=1997;citation_issue=5;citation_pmid=18464863" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=V. Hecht;citation_author=F. Foucher;citation_author=C. Ferrandiz;citation_author=K. Macknight;citation_author=C. Navarro;citation_title=Conservation of Arabidopsis flowering genes in model legumes.;citation_volume=137;citation_year=2005;citation_pmid=15778459;citation_doi=10.1104/pp.104.057018" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. K. Hougaard;citation_author=L. H. Madsen;citation_author=N. Sandal;citation_author=M. D. Moretzsohn;citation_author=J. Fredslund;citation_title=Legume anchor markers link syntenic regions between Phaseolus vulgaris, Lotus japonicus, Medicago truncatula and Arachis;citation_pages=2299-2312;citation_volume=179;citation_year=2008;citation_pmid=18689902;citation_doi=10.1534/genetics.108.090084" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Jing;citation_author=R. Johnson;citation_author=A. Seres;citation_author=G. Kiss;citation_author=M. J. Ambrose;citation_title=Gene-based sequence diversity analysis of field pea (Pisum).;citation_pages=2263-2275;citation_volume=177;citation_year=2007;citation_pmid=18073431;citation_doi=10.1534/genetics.107.081323" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=P. Kaló;citation_author=A. Seres;citation_author=S. A. Taylor;citation_author=J. Jakab;citation_author=Z. Kevei;citation_title=Comparative mapping between Medicago sativa and Pisum sativum.;citation_pages=235-246;citation_volume=272;citation_year=2004;citation_issue=3;citation_pmid=15340836;citation_doi=10.1007/s00438-004-1055-z" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=V. Katoch;citation_author=S. Sharma;citation_author=S. Pathania;citation_author=D. K. Banayal;citation_author=S. K. Sharma;citation_title=Molecular mapping of pea powdery mildew resistance gene er2 to pea linkage group III.;citation_pages=229-237;citation_volume=25;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=T. Kroj;citation_author=G. Savino;citation_author=C. Valon;citation_author=J. Giraudat;citation_author=F. Parcy;citation_title=Regulation of storage protein gene expression in Arabidopsis;citation_pages=6065-6073;citation_volume=130;citation_year=2003;citation_issue=24;citation_pmid=14597573;citation_doi=10.1242/dev.00814" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=L. Krusell;citation_author=L. H. Madsen;citation_author=S. Sato;citation_author=G. Aubert;citation_author=A. Genua;citation_title=Shoot control of root development and nodulation is mediated by a receptor-like kinase.;citation_pages=422-426;citation_volume=420;citation_year=2002;citation_issue=6914;citation_pmid=12442170;citation_doi=10.1038/nature01207" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=P. Lara;citation_author=L. Oñate-Sánchez;citation_author=Z. Abraham;citation_author=C. Ferrándiz;citation_author=I. Díaz;citation_title=Synergistic activation of seed storage protein gene expression in Arabidopsis by ABI3 and Two bZIPs Related to OPAQUE2.;citation_volume=278;citation_year=2003;citation_pmid=12657652;citation_doi=10.1074/jbc.M210538200" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=V. Laucou;citation_author=K. Haurogné;citation_author=N. Ellis;citation_author=C. Rameau;citation_title=Genetic mapping in pea. 1. RAPD-based genetic linkage map of Pisum sativum.;citation_pages=905-915;citation_volume=97;citation_year=1998;citation_doi=10.1007/s001220050971" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=J. Lévy;citation_author=C. Bres;citation_author=R. Geurts;citation_author=B. Chalhoub;citation_author=O. Kulikova;citation_title=A putative Ca2+ and calmodulin-dependent protein kinase required for bacterial and fungal symbioses.;citation_volume=303;citation_year=2004;citation_pmid=14963335;citation_doi=10.1126/science.1093038" />
<meta name="citation_reference" content="citation_journal_abbrev=J Integr Plant Biol;citation_author=X. Li;citation_author=L. L. Zhuang;citation_author=M. A. M. Phil;citation_author=C. Rameau;citation_author=X. H. Hu;citation_title=Genetic analysis of ele mutants and comparative mapping of ele1 locus in the control of organ internal asymmetry in garden pea.;citation_pages=528-535;citation_volume=52;citation_year=2010;citation_issue=6;citation_pmid=20590983;citation_doi=10.1111/j.1744-7909.2010.00949.x" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=E. Limpens;citation_author=C. Franken;citation_author=P. Smit;citation_author=J. Willemse;citation_author=T. Bisseling;citation_title=LysM domain receptor kinases regulating rhizobial Nod factor-induced infection.;citation_volume=302;citation_year=2003;citation_pmid=12947035;citation_doi=10.1126/science.1090074" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=K. Loridon;citation_author=K. McPhee;citation_author=J. Morin;citation_author=P. Dubreuil;citation_author=M. L. Pilet-Nayel;citation_title=Microsatellite marker polymorphism and mapping in pea (Pisum sativum L.).;citation_pages=1022-1031;citation_volume=111;citation_year=2005;citation_issue=6;citation_pmid=16133320;citation_doi=10.1007/s00122-005-0014-3" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. Macas;citation_author=P. Neumann;citation_author=A. Navrátilová;citation_title=Repetitive DNA in the pea (Pisum sativum L.) genome: comprehensive characterization using 454 sequencing and comparison to soybean and Medicago truncatula.;citation_pages=427-427;citation_volume=8;citation_year=2007;citation_pmid=18031571;citation_doi=10.1186/1471-2164-8-427" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=M. Mbengue;citation_author=S. Camut;citation_author=F. de Carvalho-Niebel;citation_author=L. Deslandes;citation_author=S. Froidure;citation_title=The Medicago truncatula E3 ubiquitin ligase PUB1 interacts with the LYK3 symbiotic receptor and negatively regulates infection and nodulation.;citation_volume=22;citation_year=2010;citation_pmid=20971894;citation_doi=10.1105/tpc.110.075861" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=P. E. McClean;citation_author=S. Mamidi;citation_author=M. McConnell;citation_author=S. Chikara;citation_author=R. Lee;citation_title=Synteny mapping between common bean and soybean reveals extensive blocks of shared loci.;citation_pages=184-184;citation_volume=11;citation_year=2010;citation_pmid=20298570;citation_doi=10.1186/1471-2164-11-184" />
<meta name="citation_reference" content="citation_journal_title=Planta;citation_author=D. W. Meinke;citation_author=J. Chen;citation_author=N. J. Beachy;citation_title=Expression of storage-protein genes during soybean seed development.;citation_pages=130-139;citation_volume=153;citation_year=1981;citation_doi=10.1007/BF00384094" />
<meta name="citation_reference" content="citation_journal_abbrev=Planta;citation_author=R. K. Mishra;citation_author=S. Chaudhary;citation_author=A. Kumar;citation_author=S. Kumar;citation_title=Effects of MULTIFOLIATE-PINNA, AFILA, TENDRIL-LESS and UNIFOLIATA genes on leafblade architecture in Pisum sativum.;citation_pages=177-190;citation_volume=230;citation_year=2009;citation_issue=1;citation_pmid=19404676;citation_doi=10.1007/s00425-009-0931-5" />
<meta name="citation_reference" content="citation_journal_title=Journal of the American Dietetic Association;citation_journal_abbrev=J Am Diet Assoc;citation_author=D. C. Mitchell;citation_author=F. R. Lawrence;citation_author=T. J. Hartman;citation_author=J. M. Curran;citation_title=Consumption of dry beans, peas, and lentils could improve diet quality in the US population.;citation_pages=909-913;citation_volume=109;citation_year=2009;citation_issue=5;citation_pmid=19394480;citation_doi=10.1016/j.jada.2009.02.029" />
<meta name="citation_reference" content="citation_journal_title=Planta;citation_author=G. Mönke;citation_author=L. Altschmied;citation_author=A. Tewes;citation_author=W. Reidt;citation_author=H. P. Mock;citation_title=Seed-specific transcription factors ABI3 and FUS3: molecular interaction with DNA.;citation_pages=158-166;citation_volume=19;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=S. N. Nayak;citation_author=H. Zhu;citation_author=N. Varghese;citation_author=S. Datta;citation_author=H. K. Choi;citation_title=Integration of novel SSR and gene-based SNP marker loci in the chickpea genetic map and establishment of new anchor points with Medicago truncatula genome.;citation_pages=1415-1441;citation_volume=120;citation_year=2010;citation_issue=7;citation_pmid=20098978;citation_doi=10.1007/s00122-010-1265-1" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=F. Parcy;citation_author=C. Valon;citation_author=A. Kohara;citation_author=S. Miséra;citation_author=J. Giraudat;citation_title=The ABSCISIC ACID-INSENSITIVE3, FUSCA3, and LEAFY COTYLEDON1 Loci Act in Concert to Control Multiple Aspects of Arabidopsis Seed Development;citation_pages=1265-1277;citation_volume=9;citation_year=1997;citation_issue=8;citation_pmid=9286105;citation_doi=10.1105/tpc.9.8.1265" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=G. Pertea;citation_author=X. Huang;citation_author=F. Liang;citation_author=V. Antonescu;citation_author=R. Sultana;citation_title=TIGR Gene Indices clustering tools (TGICL): a software system for fast clustering of large EST datasets;citation_pages=651-652;citation_volume=19;citation_year=2003;citation_issue=5;citation_pmid=12651724;citation_doi=10.1093/bioinformatics/btg034" />
<meta name="citation_reference" content="citation_journal_title=Funct. Plant Biol.;citation_author=H. T. T. Phan;citation_author=S. R. Ellwood;citation_author=R. Ford;citation_author=S. Thomas;citation_author=R. Oliver;citation_title=Differences in syntenic complexity between Medicago truncatula with Lens culinaris and Lupinus albus;citation_pages=775-782;citation_volume=33;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=S. Prioul-Gervais;citation_author=G. Deniot;citation_author=E. M. Receveur;citation_author=A. Frankewitz;citation_author=M. Fourmann;citation_title=Candidate genes for quantitative resistance to Mycosphaerella pinodes in pea (Pisum sativum L.).;citation_pages=971-984;citation_volume=114;citation_year=2007;citation_issue=6;citation_pmid=17265025" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C. Rameau;citation_author=D. Dénoue;citation_author=F. Fraval;citation_author=K. Haurogné;citation_author=J. Josserand;citation_title=Genetic mapping in pea. 2. Identification of RAPD and SCAR markers linked to genes affecting plant architecture.;citation_pages=916-928;citation_volume=97;citation_year=1998;citation_doi=10.1007/s001220050972" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Salse;citation_author=M. Abrouk;citation_author=S. Bolot;citation_author=N. Guilhot;citation_author=E. Courcelle;citation_title=Reconstruction of monocotelydoneous proto-chromosomes reveals faster evolution in plants than in animals.;citation_volume=106;citation_year=2009;citation_pmid=19706486;citation_doi=10.1073/pnas.0902350106" />
<meta name="citation_reference" content="citation_journal_title=Briefings in Bioinf.;citation_author=J. Salse;citation_author=M. Abrouk;citation_author=F. Murat;citation_author=U. Masood Quraishi;citation_author=C. Feuillet;citation_title=Improved standards and new comparative genomics tools provide new insights into grasses paleogenomics.;citation_volume=10;citation_year=2009;citation_pmid=19720678;citation_doi=10.1093/bib/bbp037" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=S. Sato;citation_author=Y. Nakamura;citation_author=E. Asamizu;citation_author=S. Isobe;citation_author=S. Tabata;citation_title=Genome Sequencing and Genome Resources in Model Legumes;citation_pages=588-593;citation_volume=144;citation_year=2007;citation_issue=2;citation_pmid=17556522;citation_doi=10.1104/pp.107.097493" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=S. Sato;citation_author=S. Isobe;citation_author=S. Tabata;citation_title=Structural analyses of the genomes in legumes.;citation_pages=146-152;citation_volume=13;citation_year=2010;citation_issue=2;citation_pmid=20071214" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. J. Schmidt;citation_author=F. A. Burr;citation_author=M. J. Auckerman;citation_author=B. Burr;citation_title=Maize regulatory gene opaque-2 encodes a protein with a &quot;leucine-zipper&quot; motif that binds to zein DNA.;citation_pages=46-50;citation_volume=87;citation_year=1990;citation_issue=1;citation_pmid=2296602;citation_doi=10.1073/pnas.87.1.46" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=R. J. Schmidt;citation_author=M. Ketudat;citation_author=M. J. Aukerman;citation_author=G. Hoschek;citation_title=Opaque-2 Is a Transcriptional Activator That Recognizes a Specific Target Site in 22-kD Zein Genes;citation_pages=689-700;citation_volume=4;citation_year=1992;citation_issue=6;citation_pmid=1392590;citation_doi=10.1105/tpc.4.6.689" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Schmutz;citation_author=J. B. Cannon;citation_author=J. Schlueter;citation_author=J. Ma;citation_author=T. Mitros;citation_title=Genome sequence of the palaeopolyploid soybean.;citation_pages=178-183;citation_volume=463;citation_year=2010;citation_issue=7278;citation_pmid=20075913;citation_doi=10.1038/nature08670" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=C. J. Simon;citation_author=F. J. MuehIbauer;citation_title=Construction of a chickpea linkage map and its comparison with maps of pea and lentil.;citation_pages=115-119;citation_volume=88;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Pisum Genetics;citation_author=A. A. Sinjushin;citation_author=F. A. Konovalov;citation_author=S. A. Gostimskii;citation_title=A gene for stem fasciation is localized on linkage group III;citation_pages=19-20;citation_volume=38;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=S. Stracke;citation_author=S. Sato;citation_author=N. Sandal;citation_author=M. Koyama;citation_author=T. Kaneko;citation_title=Exploitation of colinear relationships between the genomes of Lotus japonicus, Pisum sativum and Arabidopsis thaliana, for positional cloning of a legume symbiosis gene.;citation_pages=442-449;citation_volume=108;citation_year=2004;citation_issue=3;citation_pmid=14557858;citation_doi=10.1007/s00122-003-1438-2" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=H. Tang;citation_author=J. E. Bowers;citation_author=X. Wang;citation_author=R. Ming;citation_author=M. Alam;citation_title=Synteny and Collinearity in Plant Genomes;citation_pages=486-488;citation_volume=320;citation_year=2008;citation_issue=5875;citation_pmid=18436778;citation_doi=10.1126/science.1153917" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=M. Tegeder;citation_author=C. E. Offler;citation_author=W. B. Frommer;citation_author=J. W. Patrick;citation_title=Amino Acid Transporters Are Localized to Transfer Cells of Developing Pea Seeds;citation_pages=319-326;citation_volume=122;citation_year=2000;citation_issue=2;citation_pmid=10677425;citation_doi=10.1104/pp.122.2.319" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=T. Ueda;citation_author=W. Waverczak;citation_author=K. Ward;citation_author=N. Sher;citation_author=M. Ketudat;citation_title=Mutations of the 22- and 27-kD Zein Promoters Affect Transactivation by the Opaque-2 Protein;citation_pages=701-709;citation_volume=4;citation_year=1992;citation_issue=6;citation_pmid=1392591;citation_doi=10.1105/tpc.4.6.701" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=R. E. Voorrips;citation_title=MapChart: Software for the Graphical Presentation of Linkage Maps and QTLs;citation_pages=77-78;citation_volume=93;citation_year=2002;citation_issue=1;citation_pmid=12011185;citation_doi=10.1093/jhered/93.1.77" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Z. Wang;citation_author=Y. Luo;citation_author=X. Li;citation_author=L. Wang;citation_author=S. Xu;citation_title=Genetic control of floral zygomorphy in pea (Pisum sativum L.).;citation_volume=105;citation_year=2008;citation_pmid=18650395;citation_doi=10.1073/pnas.0803291105" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=N. F. Weeden;citation_author=F. J. Muehlbauer;citation_author=G. Ladizinsky;citation_title=Extensive Conservation of Linkage Relationships Between Pea and Lentil Genetic Maps;citation_pages=123-129;citation_volume=83;citation_year=1992;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=Pisum Genet.;citation_author=N. F. Weeden;citation_author=T. H. N. Ellis;citation_author=G. M. Timmerman-Vaughan;citation_author=W. K. Swiecicki;citation_author=S. M. Rozov;citation_title=A consensus linkage map for Pisum sativum.;citation_pages=1-3;citation_volume=30;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=N. D. Young;citation_author=S. B. Cannon;citation_author=S. Sato;citation_author=D. Kim;citation_author=D. R. Cook;citation_title=Sequencing the Genespaces of Medicago truncatula and Lotus japonicus;citation_pages=1174-1181;citation_volume=137;citation_year=2005;citation_issue=4;citation_pmid=15824279;citation_doi=10.1104/pp.104.057034" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=H. Zhu;citation_author=H. K. Choi;citation_author=D. R. Cook;citation_author=R. C. Shoemaker;citation_title=Bridging Model and Crop Legumes through Comparative Genomics;citation_pages=1189-1196;citation_volume=137;citation_year=2005;citation_issue=4;citation_pmid=15824281;citation_doi=10.1104/pp.104.058891" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Zhu;citation_author=B. K. Riely;citation_author=N. J. Burns;citation_author=J. M. Ané;citation_title=Tracing nonlegume orthologs of legume genes required for nodulation and arbuscular mycorhizal symbiose.;citation_pages=2491-2499;citation_volume=172;citation_year=2006;citation_pmid=16452143;citation_doi=10.1534/genetics.105.051185" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Translational Genomics in Legumes Allowed Placing In Silico 5460 Unigenes on the Pea Functional Map and Identified Candidate Genes in Pisum sativum L." />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2.cover.gif" />
<meta name="twitter:description" content="To identify genes involved in phenotypic traits, translational genomics from highly characterized model plants to poorly characterized crop plants provides a valuable source of markers to saturate a zone of interest as well as functionally characterized candidate genes. In this paper, an integrated view of the pea genetic map was developed. A series of gene markers were mapped and their best reciprocal homologs were identified on M. truncatula , L. japonicus , soybean, and poplar pseudomolecules. Based on the syntenic relationships uncovered between pea and M. truncatula , 5460 pea Unigenes were tentatively placed on the consensus map. A new bioinformatics tool, [http://www.thelegumeportal.net/pea\_mtr\_translational_toolkit][1], was developed that allows, for any gene sequence, to search its putative position on the pea consensus map and hence to search for candidate genes among neighboring Unigenes. As an example, a promising candidate gene for the hypernodulation mutation nod3 in pea was proposed based on the map position of the likely homolog of Pub1 , a M. truncatula gene involved in nodulation regulation. A broader view of pea genome evolution was obtained by revealing syntenic relationships between pea and sequenced genomes. Blocks of synteny were identified which gave new insights into the evolution of chromosome structure in Papillionoids and Eudicots. The power of the translational genomics approach was underlined.

 [1]: http://www.thelegumeportal.net/pea_mtr_translational_toolkit" />
<meta name="og-title" property="og:title" content="Translational Genomics in Legumes Allowed Placing In Silico 5460 Unigenes on the Pea Functional Map and Identified Candidate Genes in Pisum sativum L." />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/1/2/93" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="To identify genes involved in phenotypic traits, translational genomics from highly characterized model plants to poorly characterized crop plants provides a valuable source of markers to saturate a zone of interest as well as functionally characterized candidate genes. In this paper, an integrated view of the pea genetic map was developed. A series of gene markers were mapped and their best reciprocal homologs were identified on M. truncatula , L. japonicus , soybean, and poplar pseudomolecules. Based on the syntenic relationships uncovered between pea and M. truncatula , 5460 pea Unigenes were tentatively placed on the consensus map. A new bioinformatics tool, [http://www.thelegumeportal.net/pea\_mtr\_translational_toolkit][1], was developed that allows, for any gene sequence, to search its putative position on the pea consensus map and hence to search for candidate genes among neighboring Unigenes. As an example, a promising candidate gene for the hypernodulation mutation nod3 in pea was proposed based on the map position of the likely homolog of Pub1 , a M. truncatula gene involved in nodulation regulation. A broader view of pea genome evolution was obtained by revealing syntenic relationships between pea and sequenced genomes. Blocks of synteny were identified which gave new insights into the evolution of chromosome structure in Papillionoids and Eudicots. The power of the translational genomics approach was underlined.

 [1]: http://www.thelegumeportal.net/pea_mtr_translational_toolkit" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2.cover.gif" />
    <title>Translational Genomics in Legumes Allowed Placing In Silico 5460 Unigenes on the Pea Functional Map and Identified Candidate Genes in Pisum sativum L. | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"26Cj2n7j2eMT-vrveiM6zhN7Gbnw7cGB_zQ9XdpuwQU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459160","apath":"\/ggg\/1\/2\/93.atom","pisa":"ggg;1\/2\/93","ac":{"\/ggg\/1\/2\/93.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/1\/2\/93.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;1\/2\/93"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;1\/2\/93"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/1\/2\/93":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+R8JW+ztl6w+6NhFO5DC40Ei6Hvsb6V4PVlcPfswzm7AD\\\\\\\/HzlFVrct\\\\\\\/5vlNOVObcskh1lFhi\\\\\\\/91Yo1PzGp97mQaZwOOijWTVLSu29jjZCCuafFt4tI9W03o2dOKluWNZ92CqpdMCrWBgcQnqKSba4wQRoNm18nKpdC+Q4FEs4svBj300r2UXYIM89Gm9N3NBO7kmoZsukSJQSlm2wYywKkRNTKuQw6LKch8QNpaDLJEiW7N33Em2gz6DOvEnxOmcq9QuJEfxa4CtgrjEO+qsSsspq64HrtOq\\\\\\\/iFKzHtvsQ506f6Xq1+javwTctRaSQP0Wapf0ahoe7ztfblekb5e2rvHXhH25sNxaqv3UXC2pNlAdkmCarLD7spFjXwNfbuP48ukzYvoyQRWHKx0GEv\\\\\\\/AFGV47H21qn3HWlJR9HxDDgCCJXzjUCt0LliuPm1vfKWf6RW+iS+tdTH\\\\\\\/ZzbGz4An88Xtz+AIx8GB\\\\\\\/j\\\\\\\/e72B8o\\\\\\\/Ip74PzZHbhUTdcc8oMWsD5abjVpCvD8NykDdZ2X8ldrGqevjEcdth0LQKG7nJ97TwXENC3fV3\\\\\\\/26HESArW8q6juz\\\\\\\/EqFmm4xG132FxGyXCLMQgD\\\\\\\/dGgvCBq\\\\\\\/fHe60y9pQGjgXacWi014+Cmnj6SUHOJfO\\\\\\\/eHI4z8Nbfm2M19zjWYG+d9oG28BHXptOSV1XtCOo9nV7aw34Vf0A+f+HlwVGe8+iF8rvWxXROizX15sEtCBaecsHnI3bpYEKM9+KDeVLMoLMsWc4p7WS2VJQTRwCH76r0lMQmNUk15kZ\\\\\\\/F0Cpn8BNks9i4oYlUUrESiaqcuL4XKIiWERQ\\\\\\\/uiovn5MQcil2BWP91xZsLXHAXbwPIBcJfVx\\\\\\\/9WhFUukCtByGArq3FAuls7lWr4T2mq1L3vGgNkqcvWJpYcU5IOnlKL532+kNwdzGjGaG7cvk98aVGbjqJb18u2ADYd7FZDkyVXa7B4\\\\\\\/eZgkF9nm0bUkMeeWQREUoTX3UEx6DfLcppyEUDIjcC99TdVZiicmdRFavFKdo28KRoniX24uWN6bXQiJSjz21i7IiFinAxZN9Y7gCd63f3EKnlKitzGRrncye0nczw3Mgkx7Nx09Qyix1sR9Jpebw0oKpYoTUtgQcAMNDghW0hoyAytjE88SBuZ+uFOeMNmPuTwZi5n4nmWWCHwm6WatBi0XEEAGCfNojQ54AHi3j\\\\\\\/\\\\\\\/sS\\\\\\\/plBNl\\\\\\\/VrdU5iCfjGdEW5po9gwwGuxZJg+sfP3NzJdDqzOb2BHolwdNxkDqAxoMp7NP3wmEp6xJ3rpY7M\\\u0022,\\\u0022iv\\\u0022:\\\u00220wHWs6OhIHIx\\\\\\\/YpubJStYA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222015842c0bda7e6e9ae77ae4537d98dd2386c03b18368a0421a775c94d0d3aad\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022R\\\\\\\/kGFdWQubrDYK9M8Ib0aIoT\\\\\\\/Z6dgt+rBrHfGkXygpR39JZmMZ8Mq8dgo98PUZtA910X\\\\\\\/8Xxd6dZTzW\\\\\\\/7nKyLMmgzkWNQiUbJhhddFbnzDq0p1qDlKACSFw6AfSQU\\\\\\\/5wUjI0uivf6x1pzhNC1aT\\\\\\\/X+tbeE5ihM0EXdMcOpZ2MqqVk9qr8jJelkl0QyUzyGzzYULyQTHJFB2Tapty9lDkmghrMyBB64fntESX7gPqKTMZQW+b92gmFBc40NymjR9hEFnGIAoRkvjbUn7czURNTIibPBIcQYRVkeushwpKPibFKiTR2pmNt8R9FjZ30OWDhq3mCxZ2UDWuAVVTCneLWXB6WrO2seofTtuGK4IzfgkpYJhiKh6l6mYRrbvMVkazj5B6\\\\\\\/nyr1LGpeOSYxUzm8NejVfTxc+G7El\\\\\\\/ZHVcpWNX\\\\\\\/9X1D9xLNXo9NuBX7O2YE5mXavrOOzrxPswFir466R01Wzwnbi7oRMudF48oWjWwOuRejSIRm7uUpZ7vSMHwolmdPiBHQCcTt6ojMbnxxs6B\\\\\\\/VPjJlxS5n309AZyZWYgBn16l9gG7AjQST4HvaX3+kG7O5sVKUZQP0LAx6qJRObHjAU+VqQ6cXAyuxdlLbpbRLM0GE1M5lykUIXK5ws0dJB7lQ2LTBLcFtLL9SIk1lnZv57Yblhb0GcwzjI2aQm\\\\\\\/7E7kxb8eb6vwyEzdba6070FZqnceNXzxPr0AyF6ulrbMIMyaG9jdORcJZxVosv0vyHGmx11x+m9ti2OfxoEZcd6KD9k9MkZBqlAL+vPi3Uu+Df3OHwuPNqWH2Un8FZ5X+Mb+vKsrrpPh1rQxyXWi6aOgbZc2PKNdiPyHjVYZnfif6VCFDdwV+E0lB1LAh8ARLFhdyRTyB1lwmu\\\\\\\/mKLnQSLVbstRW\\\\\\\/3UF0YbKNHgCynXzBFmWSJxPk7ZHbZMH0m8ReEOx2P9CZPVuTkPe9ZWxAoX2ydpMrqKSoC5ANvQpIesBjuNnLi+hc3WlCVidHJvhJVNucMyvYdkeLzyoizm7FCFlZmSWLrazsJdZdHlxozGt9qp1iNU8S65u8v0750di3zFI\\\\\\\/yufXk38G5JxCATr1PIWrBMkcGZvMhvm2DB97sAuieVl\\\\\\\/gPyotfV8THEHL5nEOuemswM6liszXbypX7sE0yL5RICS9gXfjeEAT3aSXEMTH\\\\\\\/1zEIY8xJHRq\\\\\\\/FVpWwlYw8QGwvXVb6HE6Nwva\\\\\\\/hN6PgTTlSw\\\\\\\/QpiMN\\\\\\\/SY3b8UD6KmCIndJtLfZyTh+iMxaahUTcbnI\\\\\\\/OSiUrkRLKRjYxRVpK3Xz23QcCLe\\\\\\\/Sd9TsCSWMyVy6hv0n93NlyN9NRInap1AfZymhOC7dSdrWV3RYopZjxiCsJTMHncfFzALvc+wAH\\\\\\\/V6Ya6HL2pqDpJfFbWqkIDq71HFVJBJD80\\\\\\\/cZgMPr2TCYE6r+VQlAOMgu44oloOIgts5roaZ\\\\\\\/9H124686DDbH\\\\\\\/YXbL5p\\\\\\\/ZEc1fKF+9V3IiQLr2e4QMSD8SJp8+Gu5HRhqJANkf0spVhP4QByEh1clMzbig3+KtE\\\\\\\/hLPNABMqy6JDSlyzY90fRwNNSHhGJnyqA80LIRGlW1pf+HBHm2jx0qqxLQ8rT3ZorjXxcRfzbs974j9yVhxqeYlsWtVc8MRUiXpQQMuYFZPa1cQxNsVS870SQDvkGRLiOnFZdnKXRvNycDZP0UPYWenOWq3kYH2\\\\\\\/e00WIrY30i8JsfHBtMEtpbOFFbg\\\\\\\/dvZs6sHhxU9xQa8zO4QFUhFBbctpdUds6TgO+4q0xbEjhIGoC+NiZjdSnl8jAJ\\\\\\\/EFBjZBHwaNR0blZAqxcvJhUFDfLglPAqTpm5oc0n\\\\\\\/hpcQZR+FAxtOrgBF6\\\\\\\/KZtOTM7izx4BayZmwNJ1KB3xwDr85pqMfMs\\\\\\\/NLLbLsh9xiXZJAPXAs4osiiSoyGxzwB95kQ3TAVOhOFJT043QY3Wblt\\\\\\\/isRks5wNSQgBCN4law+Cx8zocbrQXRU9q6l4C0NNaoLwYSVNOV7h3Xz\\\\\\\/QulFHDLaPP3NpE0nqxb1brcxdlr2cF23QuZH70ANYMJVha6brMxFliMK\\\\\\\/WuzkPH4XYw01wYzzNPg0gNBYhJHOxKbXZILT+kgUx\\\\\\\/FnUSlCn3df3cDV2Q\\\\\\\/WgI4Tw21TavjRxTAY4wCFqAg3HGDEOsmWffI1E+xYNHj\\\\\\\/iO5n+oxY1CHVhZyOyyyPzdiuwFfV+SY2QikHLFDYfRqf3wBzpKICtN75mkT4QKlS4bIduFKHfXlDsZGOK7iVS2CbtwlU6otPySp4SpHlNaKq0WItavSurpBBacqzg4DPWjJTNTidJujBYuwHcsVrM8s+ZNfKY4slzpxMtHE2jZZUIgwTJgF3DjfSE5uMiMtm60dGKb9FX2a6p+dR4aDBcNml1l2EXyt4XI\\\\\\\/uf25wONy9SCQf3p9fHFCNqaHcjOzZfrvGmm4pPXaX44+3h0qdhwsFzYVzfuhwbnVjVgM7pQWjK36lOCtvBAZYTuSgP9cpXbY++FygvbRdl\\\\\\\/eMSEypjIRsNVZZmI2OZ6PnavIxG1q1TmqG1x5LB0tTVpB7wxyt36qEp+AvL3nM2ranUYCVQFlb7jGRdLkQ7kROpAZznEgO6X\\\\\\\/zfGi+isHWiKFt984AUtmzKgkYmNerZFZMk0rHE1mkvPjE5KWzSlxe+ZiMvvf2mfSugAnpRB2G5DIt7KweNxgwnLkqJz53eDH2MdYIw6Xttyw0lcPgGp1D1Ez87hjZEPzDuVCGhjIb0\\\\\\\/7z9Hv8lriMO9303r25UQUEuApSCxUFXe1kq3B6zVfKVqd7cgJWUERKF1wVw10bxlSa0JdT+mKGUxIJFWHZ4N3+WUzKSJZQpSc9XFsKzKE72xLAigtRj\\\\\\\/GjrtMabuKpQeR8kqvDHLIGxIfRhD\\\\\\\/v+T5+GRKHPU7ZP2evV9f5ZjKaPaF0\\\\\\\/cPEVLqekYpw2Tv8W8D5\\\\\\\/oZzW2qWvw96hS41KZOHtCpoeVa\\\\\\\/wgqjdAUXEeChpHLpbGA5Fey6bk4SByh+eST8UGbjuurzbSxEcxm2lzZHIDRhY6uMklSu+MMvOlENRPAt5VYBwG8tCC4QJyAMKbhKrHhc6BPMAeoffAtZ+rJ\\\\\\\/YCvp9Ih2ja2sAKnoC\\\\\\\/v+tomGFZfgPnHzK5LcO1UEDvEcFsknYDPi\\\\\\\/1RSU44O5ZxqPI5f2BrBk2yLivSL0Yzu2ZA+aqLA4n22+FNdQwIDZMYt\\\\\\\/zSPcNdH5r4Cr+kH5L\\\\\\\/WRoq\\\\\\\/1rPvgnfOdg7uWQXSKiv5phyZSX8AvYN5TgO2UdpkOPVc0ryfXxAS0kxQYNhePqJfOdoSf+1ZueTVS8pl3VrNW6sG4BUQc\\\\\\\/alwrg69cBoFMyBT\\\\\\\/7laRPHV1qMg4fh0N0oxvYbcaJMpDmyXqHbEGZNiHmzyW6vGD7SHn9n4qDTs5nifeOEo+0epfviYN6Ry3D3caTnCOlO4Dqo7Hrps2UafyOTUSn0p11eZrW1RvOGMfprlDWd7+3I1Pgf4o15U58bCr33c6Csmjs\\\\\\\/A7dAYqxen69vY2Qaese39DDDHQJK\\\\\\\/Yr2Za5gYgT53MX25uqtPpZNo8vHymoFR9sIwTVu19sJZn5tqR7hZbgp6WcTFjXbwlhSXvXXbj+O3rwgWmZb9bFNP5MaeNHOfMviKK0E7YZc76d\\\\\\\/2uITpT3jyf1G1T06rVjN1of8GpqC6CBpv+OJ4jO6ytI4GkPnbRK09C4o1zBm9fsTe9Mx9HXguJDBHVDNlg7w\\\\\\\/2BoO8fwSGGqNSj1FUKPNevq305\\\\\\\/N2HFnIBcvmKhMWHTOh8AQfHKecpPu8zlQhv8oaXkxE1TGK84zfOTDGutpoNurAB6E2pjihLzmN0r5rnej5Y1A+PlHZ3ZiTrpjCU0beJ2LqtUy5NAdw5L72qDEg05WaEk8I6SoluxyLGm2BLDJPpns8t0pxPjfc5HmVGE0Sg8b\\\\\\\/Gc4NdKxrgMqNa0T\\\\\\\/RqqOjpsnWcSOIgq3U\\\\\\\/rWH9YlGVDMPakGc6IPIkXFKZJkdaJ1hFmBPZDQcA\\\\\\\/U8ZoFkzpAxlDdesbiq2Xg3\\\\\\\/5QZHgdi1RB9v\\\\\\\/WYI4bBuK9WzQG5yaU\\\\\\\/Wbb\\\\\\\/nXsEh+9SSa7raRiHjkzyFtGoGsmXKM8mqKBMUmYY6PTrJyg\\\\\\\/L9gUMRh0XdH35fGrkXPi9FyXJ0xUUoeHB0yTjfSB41h7eQ7UkIxSlCcxLSWp0RhDTWybx66Z0SFHnKrG2PfTYujcFvzsgO3fhawFyDJgCslpHhHJx8ZOxEPgo3elrUcDNYPnKfh3YWn4uU9bswkqgGbQ3MH46NfYIr3VGy0tjTFlOVjAyS7boyv5zRPv2LvQyeK4FrV6B2hMzI29duCL\\\\\\\/5+YzB\\\\\\\/qBdXClEIXJ5BVwunrjCldZJS0RH6RijZX49V8Svvj3UbqYcrcVvEGx51JiE7AHYAgG8Yjnx++G+IuDpMXm0n5WycBie\\\\\\\/hyu3ihfE1BcPSCEzEz1Biqgm96gV5xAw\\\\\\\/ZJmxahnTJtVsq7BeSGuMZGVbmGrFpoAXq\\\\\\\/exAt7hNeTFi3h1zbJm8mob4r5u1w02wkre1O0OqTv\\\\\\\/T4URhUSmECNd0iDgykOVxswTPMrAjrFwy1JIu+MlFbBBCVsJ6l8QPs3813Llp+QgR6z0a+Xs76QFt4qsxrODOpZ3TCSbu9bGbQ51osJAmUbSMLqoaxAmIEZ6kRHibzbFqDB7T\\\\\\\/LGwZ6fW161EH66kXCIhtVy1m\\\\\\\/suv0s0nhV\\\\\\\/0\\\\\\\/2Bzh12bq8oFXCkzxAUGWv4IBC2DDGj7Uewe6rJOuCYNRUTrONYKMMEc2x\\\\\\\/ZYHBwwVVB20M702ho0sKYA0Wfv6xPOurtuKqx7N5ZkiOeU6+WholLj8V3rAtVo2O1zCV8SMiGWhua4e9ZNHH7DAVLmVcK8iADZnY4hXa2iFUk8GA9cBuI97hpflnRl8C7dmXKYCECbb0BCw9P67o3cPDBJDtTRqOmZl2zHCLpelK8mRLakeAmXSjhv9HS4DZOeBhdfzT181EbZnK89dp767g6tR7uwnvN7tr+VfT2kENu\\\\\\\/ElU36ySRnnyY4vDDVQ1oUEAFp713dM6FRS1qdTcDuQn+eb0NIBJqalWzXcaQJGYKcSH\\\\\\\/OscIAnviAlxUXSL0fcOngoc\\\\\\\/qNW846OEPkoV2cttX6OtqzNfSrRJe2I7GJ+U6rqwqQGkgyTj02dofNjUOd75RhoA+U952Hq9JOtd6GcqOngrcWeT0s5kaEPciw0+VQ\\\\\\\/i6TvsjG0X1M0Pk1qBJRkSlad2m5l4\\\\\\\/D+rnh5LWkp6zrSgw75CctCl2Nxuya1S2LckygB7EslCvMtYUU1M6iOJtEej9cJ\\\\\\\/cwZwvWD1knMSocMDXxwGk9OvfAursMkB8j79OgQ7kJdtg5zTKX6jjCl+QFAKUAUnc3tmSX4qllsoo4c1a0Q8+iGIboxndniRPyqGTe6kj1l+EzVEdls3uCyPkeq4wnwcSgnP8N2mWQE6E4hVO+PReQrnJL6NuPeqI7wN6xdG+i5fVD3bxIS5KBfb0rkv8yxGWBBOdI93wfh2\\\\\\\/QHUi0MpkVrLqL9ONj9nYUFv8I8M3XHSXZjIc7SrbHcZMsGaCkkjkFDSA53VWq1oZexhFVYJrLtZ4ged0hEO7XbPLv95LbnkIRyV1QPmx85Wfn\\\\\\\/vB+ec3MWS\\\\\\\/NSFiS\\\\\\\/\\\\\\\/m7O4S\\\\\\\/TcJIPkKxN2LdY3ptGDGQdciuVXz2L1OZgeU4imoG5INCGsaboNt\\\\\\\/mqhwB70ExpXtNXe\\\\\\\/R3IIpZ2h4d4X6SByPtrw9DKEaLjjSfjbW8NnR2PmW3SJo0PL5448lscnn844oLAno3jeYeCHHqTuEm\\\\\\\/WkpX3JKG9fGJKUZoZXbR10QpDr60qcAEFxW+htauSz5nehNOG4\\\\\\\/kSuOntQ\\\\\\\/nYpapMJ\\\\\\\/Ae2FSPKPymF\\\\\\\/UMroMMFlfDHC6Fze\\\\\\\/UmXbGtJfSJfVmi0r97A5cGBgqdBoDeGUDpF84s2rkIzTxlqh3lsAMEL7goKTOClji+sFp10qDsWyaTHTUNGXbTLcuLjN0e0hvgrzOEKvoAnU0sqZfq6VYqD8mjDEC7tUoLaYLepjXh2lHpmkF3lv5lOAw+sbk0qHUWZJsK9Yz9HU8l3wF4gUMfms4qqx6BacGP+AdY0zmQ44MZkPuP+gwDg6z8WYxp2xzaUMCkQcixXOIIWzamrfEAIF0OgSlofMHcZmxxtSDuA3VsdPiD3uV9t8kjh0vsGTRFmHBtXCdjPkfdWGed0yRkA2Dyr6O1m2MAEQmRKUL0QqeeLTf6kcAVT\\\\\\\/XQjNKDGNFOV+lI5vgh3YbDNdsMB0TCG9FfnEKZJSkhtF8R4EKUhzLbCG6dxPgv302bA+kcoKiraZQ172y43dmRSVwosnXLh7mv03PW4D6vdc3qCzz9wasn34fYR4ZqKaVQSJ4KSSnHELaQhY6lxUyG7n1WK+fO1SM8PEvVYsIUh8ds2zArfyxk13KmWKpZ6BKq5PBM3ZqFtXwzKER0Jng9s9LfF7rY3FSONsK7OZguksim+GXwJsu+E0I9sOm7XcsAXthVpay6UHiI4j8zdH+UdsaCYBLKBiQ2w8wOEAUzyfHDw1jZU32xwLkSbpmUvPILJ96lD8gwKv4Wbg0y1t7Ei90JJfq5UYusp\\\\\\\/APdowTqZLlg1Dkbus2bXg6KhqweodAS7x+5qiL\\\\\\\/GI44bxxT1YdwfaTCHbnJS8h1ryKhoMQgmQGbi0gMwenv9QQQ3j1E7S3WkjWqeaiRk+bpAmAhI4qvgyy3uwsZUJxGZezfXn8bMSK2rffTnDQwxPjLJsGTj6X0XS\\\\\\\/D6GQu1L8OBsxQG6pIjLpwR7NoWkf\\\\\\\/\\\\\\\/VUsb\\\\\\\/FgymozgYEluh3lS2sNbpITLqAHk4GUXDzg2rg4ZAhTe+VFDpa6CB7fv3jLfPcvfDZ7Wjqq44b3FxxDdwrTE9meQ0lWLcPXGpn1aBVgcyVSFuRFoZtpkhvNA\\\\\\\/qYFsQT\\\\\\\/VBq8ITQ6HwdhilyRc6zO3MDC9\\\\\\\/e4RvWDH0vXXphl8Hj+PofuLp1E+dNa5VG3eHZUw4UVkZeczNHyD2UlTvChm6YiTkLkDxHpXie81RYIA66jG+byNY2whWERSQbzqASgUU9hMVQoSO\\\\\\\/nCoycqcoibTgI6w4PybO2CdOwHgRKAv83qyEeOO8vNYtu4w0WepYlRRVbHhVTHMiUTt1rj7TUwlAIFfa42jHrrGYUf\\\\\\\/tHcvk5d0ts1gqjNIvmikjjBm2n8YVlHK9M4awj5Q3lZVuF5owa5XoIkb0gAB2q8a0sIFAWlC+w68nFQf\\\\\\\/LZCionHuzYeNeMgEhvWwfUANCGQoAtYSnqbQwYPTlHhHZIXnmMRyitiCwou\\\\\\\/L4LEm\\\\\\\/tQ\\\\\\\/sUGK2Gtv5D+FyzWP\\\\\\\/fDKL9xy4IriEhjGEbJ\\\\\\\/QeHuep8D14bZE12Blbww8b+yqxCQz6Oc5k3xRumz1C0gAJ2jlkKoXchIBrMX8wfxLNKPJ58JPGlX7FIPkqKVkcEdL6dHuH9or0eRBE01\\\\\\\/XKBk4s1EtdFClxg410vYHEKYpp070ZIthJ4rnqkm+FRZa9PLsDU7QGBEVyXx4gBA1NrjBaEguXlrhzrVpKyAtm9OeBHAsLndWNrly5uQleeKHe7E\\\\\\\/grm3YxncCVA5pIt3D+v9R\\\\\\\/inai0ln\\\\\\\/JqSppE5tQ7Qb4JQmqhOqGzXWgOV62t\\\\\\\/yTuzr5HCWruT0MSoTMF8WKTbYKygxw9wLYoPxScuFdt95DnjGIXwYaQ9o249REDUjvMVcLDMQ15jNlSjH0dK7wed3Bxs7Oxcxc52HDm6O2xEkohsGrQs+92VBz842w8NEUt9LMcuIpBK3iluLV8x9oXu2yvN07ZqsfabmpdiJrxr+DMyVByjz9Pvg+XxW0vR+3h3bXeGD8x7X4lOqzjsz6zF05ExGV26ImrfpWDXlijl\\\\\\\/afIy1hcqT0E+Atm4as\\\\\\\/hGHrg7KvTFtaspQWpfKQGQJdt+l1ymUN9Rz+Oag2V5N7bUEcBWBGTVkr1E6gnhDON4YFoykop8KbrYKSeT1a4oeOlDze5iNpG3ZWRPLSvJ6jq7YxteWp3ovQWfy2Mz7OVCQrzcnUsquUfdPZG4MFwFBbXL4lF3FXAtzYlwzT79a3mVe4WbHersocT9vz07gU8UnaTd1f8\\\\\\\/N2jQwnXjgNGM6NlOq\\\\\\\/fg2AD7JbB4FC+143FZiUdHAeGn2Spc2zOaAhIxSkp\\\\\\\/mPWuLclsnqG\\\\\\\/h1veJTTsC\\\\\\\/U0KAFLBACDFyP4EpPnWQjX\\\\\\\/7trKPqCJTa8sd4T9U1mGeWkK4G63TA04DuS2NWrwlpaNgUytwukzHbzamqYzce6eb5fPK5lbkKuMRmaMfqvXVN7Ec8h9hKhhrkV0mWhId54aGGDTpgsueYHuLVgMSE+SEqafnY5kqHrl6B\\\\\\\/xuSMg4AkYHLHma2o4VprKCSLFuTlUIUYwI\\\\\\\/7RKvhBNiwhHdmahXmwmGl\\\\\\\/\\\\\\\/5s18+TMGIjyrMBaDMtpTSk7E9o\\\\\\\/IyYqDTnb9CeiOoke5nB5ddZLgsD0U1CaZvr5QM43QdF6pnN8+vW5PSzgv4J+APKSaxMXBcV5y3MeBacvVkgx\\\\\\\/nD6bJdOZ\\\\\\\/qgufgUKFiXlmIcAgvB+40BgVvxLLFnbWaFloVXZFOZyTYGnmOgXQonkiwaznvL69nk+YWnOft74C540ZLiLoeDvXxanfNrJRGHoLpntXSP6+tNR9gRLpwgK79hjOUCI4\\\\\\\/5ieTEXwJIJrxLATTxiCiPY39fxuO4+zOkYV1pxqkIhkeATIFcaqaPUqzlnUkVU4WMfBDx4v1xx6flbrXM09YAE7bz\\\\\\\/3I\\\\\\\/O8Z8\\\\\\\/bTIWiMrQBSS+hDW0H2QpOntyiu+EO\\\\\\\/jzB6EkKgOh5vL+NruLr4uzxMdP\\\\\\\/+CqFZP2q2GOb9vj7b0q4g+LaCvp+9asjKlrIbz7GfHr+ibIc6qqhFqpEZEsTvxGxtiL5GEwEn49rI5tsJHr\\\\\\\/q1X7Cj5L2zWkf1j8qAcGuJ\\\\\\\/F\\\\\\\/39dAsja8BTuHqqLy1YmvgwVAyx\\\\\\\/LLv0BxbUyoB8\\\\\\\/7Ys+Vbg7PDLA07asMp6jFrgGSVpo4HjfnssX9vnUr71QxeESBndFe8JEQ0KLnVzZImqx1luwGLhGdCTLxMGVwXT85RdSPFS0CmqDpVeAzR8ByotJrzu4RjReQ34PL1CpHAXYVxq2ktjvneWUgh4T7SCV1X6AhePmlSO5piAFAjJmjCzq04n1HTHriNaK6Xg2kRr9nBYxMQ2umN756g\\\\\\\/lXQvFcGteLY9DwovRrVGB8xSi4KbwhYwTiUGASmiNjDCl8yjoAlQPedIQzOvACPYUg5Bnohc28t2bO0VCLbWd2qF102C5GwxChogS7u+9119\\\\\\\/nwTjRyvMmnpA4GdWeVoTtYZ2mjgNqUYugdPQwwvXUQFIyoaEukJIL9VvepY1TwG9WponLFttEzGPqLH1NzzUPCYdCI91SJzu7TKS6\\\\\\\/mKT2TZkhtCQD5jM8VloenemC1oyug1w8n68JJVIxltk7mHfJBRmX5qmpJyhyn03f+zu7mZBo7fuS1cNL8gwwgVeg6ezJJWb\\\\\\\/6fktsJ1nctWu+gPAW4ktl76bpJGQJGVycNpGO7l\\\\\\\/HjS5wyGxZId603c6mgsdWLoulBUsktI\\\\\\\/Pvne73M60P8mfgcBkmNBS\\\\\\\/TuFSx1tssEqWxUZkpexdyVhBAVkxi7FN+IgCBwtgKi\\\\\\\/md7CJmS+cvURqGvs+7PEZjXLW8wA3\\\\\\\/OWiDOeHcMyCMYz+hNkanc\\\\\\\/QhguiKNPZ1CUSHoZh8V\\\\\\\/g7imQlTkGqBcXPeWMzBEIa3sKThNwkYmedOUtRx01U1KtzMfes8TdbFmWL2eUs3FaeMjaC4s8vnXowUkTxYMXAp6qA8nP7tHUUUe0eGxpINYbaeyMaZYZ70LXA3N1aCwllpvGDT9uiJ13XHFPhblI68UjDhStmkFov83Urrle3ZpFKptlmtG48iEszoV2hR\\\\\\\/LptwR7qR+qSgKx717kBfjCutr\\\\\\\/X1lZMkxokY79nJpcsYnlOv1TT6unNsO344wqicMiKlAUBVnju0Xb6JVUaA9iHQdADli57J19eoDGTWs0PDyk4QAMQkk2FMaVXyxpTKXbOVj22MNqfSiXw6ftrPI8Yh0hnJIXnasqPU16VpVrKoL7X6J92ZRXwKcJpXEKKVJsq0+eWbFI2vFJ3MhNfhSfzJGYPAowJ4y5gjwlb9+X2BXbFu9gTVZ+\\\\\\\/BMjdm7xTVeVvMMwzf+QgTZAl4bclS3+Td0iRMaZ9d+ven\\\\\\\/zgdWNQO1Oo5naIG8Vpnmx3v\\\\\\\/uEQILE2te+pNGY9ba07d\\\\\\\/1HKNXtl8I0j9cLVl0dSHhLdENCCcudchmLC5TC8qJRS9JJffkBOO+zuF1lZkfwT\\\\\\\/NuFIJB2t0771B16YV2YWVRHeGa6dPLuM9ErRR8qtyftohkA1vuj6MIGfy9DAd4\\\\\\\/IGwS1JuCoxZsP2RO4geGMeYnHPjhPGBSEXKdhK5+DpgtyvIxsoCRtffJT7WnxrKmbenxfjl5gd\\\\\\\/3IlNIPoHm3kNfoQrC\\\\\\\/ERPrjWFJ2mSLxwFkweJtv05QwwxYfRE6v3u4tXlSTgI1uY2lRnilFwK2+eRnCxXXbsDsxU6EWEZ8wQe0iLfmdtpGt\\\\\\\/2YYjNAs6Z4BFbCDKUISOpm5uJ0YMo2+l1sOaf37CkvFXjgAk2BKqH1EXUls4cuzZedDTHVd6N27UC2uYodqyPqhEcuyQZe04qV6BubmvY3AJpnrPwvm\\\\\\\/t1rBn1WyXcTdw5hSSz4lW5eDBxgEfhSZ\\\\\\\/62LJWZOoM6shFpqGcboTWQ5EyiQ6ar\\\\\\\/kMZLjBqoUatcIa7vifaOtDHojMn+Sq\\\\\\\/XIK1WZW7xnpo\\\\\\\/e9WHgoRXAKYI0Tp01VwhRnSfqus5+lCE4nyJztMwg9r7Rrlu76GFip9LgY4H8Abl8h7T5xdi6976em32b8VFqgJYqyEBC6oGHtxxXicuWHd7VWctr5WBfg\\\\\\\/WTUc+XFNxtyPFnABKMa+7cgTtHSivOcjz3m72YMjvOWdxvU5JrPZ0fy8iPQT25Fenan+\\\\\\\/OBpSGGkPxE4lzEcTW4EmkGP91BhUj862+hx95CDpgEyA+cZ12nt4Zb9A8EMV+aC0rJYdoDzpRAP8KhaecaNNIS0WZt32XJFwMqABOy8+drXeAtw4VJD3Np3\\\\\\\/FkA\\\\\\\/aHSlS5YVn9svzmZv5Glsqx3jn97bLRedVTWSdmPbsbpAxiIDqltlRpqZKD4Lq1AySOB6+Ibez26AanINMbL3BjqsCtHw\\\\\\\/7YD5seeTQAT4oAaleOBiH6mhT4Z9Xzy9cIBTfdNhsc\\\\\\\/RssQXiOQ3M\\\\\\\/cFqA7lje\\\\\\\/\\\\\\\/DY7en3OpqLKY3Ym+j+\\\\\\\/YxFE8N30btbp5OWVALDjFcRk4l2OFsXTdG2rZ71NBSUN56FnxflquZg6vCJdD17h3NHX991o7gBbvvIioDZVfhYmOSP41ibvxpWNCth4cKlfollM53PEtwo2zjv0VtB8IZkFcsIRfQZ3FRUzBzUjTYgNXKEgQeDm88nOlw0N1Guh\\\\\\\/vIJ0o3wonrOz6vcev5PGl09TrTu2cBrdEtaoP6UF973N1f3TUJ1livcdIAxn6EoBhyc3Sq1rWu3J98262WQ9zoSAh5eIwS2bZxSJ\\\\\\\/phjPGuEBk7NBLixJHp1D\\\\\\\/5lsaeDqTW19UqEyXBaMsYRAIBeM\\\\\\\/iw3Sdp7BTROcy0R01SGAK8iqWMY64ktqlkCQuHhp8FQGNSIVyof422cot1ve4g5H3tAiEPiFLuwAec0Odjv0rtz1jwkfx7oE2NGR7rr4e1jAg6OwUeUw9pz+C\\\\\\\/hK7Bkj0VqhR22tIKMI+xjMvsgZOTBnkAGWy+tJJwKkHzuamwyHVVu7BAjddpXuxe++82DiqnWscE0rbEV9AeYkNMaLEJWGkn\\\\\\\/AzL6m5k2QGqeUo\\\\\\\/lMWwKhyVz7+tZsZWHAwE8tbC43eSwvFQBH3\\\\\\\/P9xgbB6k83JCntYTcdjsnhfmbvER6KNrnnkJfsfaGTmPw23eoW4u0ILlyBxjZmPK3eBk1p4Lm3qj7u6rmfgqGVQxZwCGa7cwUSKro475rKLfeShRM+DZBHS8+kPSk0BkIZ+Lm8mgOFna7VSUyg+9FglPmv2ZJPkdRmRNdS6ojhmDulybMLLp6iWBz3Jemz9Bs4kpsaQ8x+jBj15lqDf3Cr1BDsq3VW3wFnXP0ZY3iZv+eIvySGDm0pFLdW8cxlXZQLuZosHSrNRx6vNDQb5qdDYfxFjRF+K5BU5vGkea+n4dFJvm0by1Y+qViWZGaIctd4wQgl2WZyI23IpiTsiaHlc74tZ0fWTh01u9AYu3eItgs1KyxZYgNnbiVK0voKhVJQrDio40AmaSVdtdFOzFURX1h7TRYjd1CfHLKIS8ZduO09N+0Q9CnN8eStc\\\\\\\/Ec8Dri67vaYbVm4Iq61ZPJQngxv+0AfVitVpU7EN9ErS\\\\\\\/cGY8vo42QnnSHpqfEP44SfOD\\\\\\\/+21wUeBGNLnEZsHeCygx2UXxJ1YKZHwBpik7sELCnG2UVgB1hQkHJ98BtaJMtRzXY8MfDTPsnMzru\\\\\\\/Ux326N0xisrwR5Ziq0y9ggtNxswhV80sli2QS8tLs2XSvsywPi+jurrfbkP+ZE+n53uWlbMCS0\\\\\\\/61SnxVubetAbc8GHFihCgjRusT96qBZz8+7Ge+eF8G\\\\\\\/K7yG27x76mbNFWRcT9+5E+0c09hoP6WjEK0UU0Vqs3bHbWDHHMb1nU+pf8cJzb0BrXnQleExGVIl5Tw+ilg79FXCtls6OgCDU7b0SnZlo1l6OoX\\\\\\\/WYE9lZ++BanmAupySeAS0HkASM2ewZBgMA1PN6hBQJIuiRCmJlg5Bb8xN8DxCXk5n2dVMfvdXxe3KvQ7K2iqB2f3tmdj\\\\\\\/CUfLdH4DnNgbgxo9OVCAyMDpKNOXmxg859b+F06CJ\\\\\\\/KG2qgJSWgLE5xSFz6Z0lA60\\\\\\\/IC7DZuAQ5g4seP0znLAbxiGizJeWP7CTrKmb0vB7Mz5j9eaYfYx0feuhUlV1ssiyFp350+XQ5gtde2CI4O6Civ1INBdFufTvkXBaRzuqF0WUnlYruLl71hGvbWC\\\\\\\/CelboPJ+Rq65CLCeyYrvV32\\\\\\\/T5WCxtJ3CB9a9\\\\\\\/evMYMjM+kpaoJhnKP7B3L5T9zrT3BckR8Xx\\\\\\\/5Fba7ijUVWeuHFjQYxk9T\\\\\\\/Mb3ynfOJPX37k\\\\\\\/42lgzAynWsLKPXWa9\\\\\\\/K4duoKg3b8o9Trl1GfJlIZI1HXV0BwiGI\\\\\\\/xVdL\\\\\\\/bAzruDkotz0S9h\\\\\\\/tVEQOFt3pfHxmpUUPzi64kMxWMmC4NdVcW1GoMCF4ja9IRMgIZJsmu7tFYvA8Rmy+zcLhWtIkwINZw\\\\\\\/u\\\\\\\/EKVYKiO6EQC7XIeMNMA0CJYdeFngWZI36B1A6VhJjPVT4jNxVsYFJzsqC7taw1HriecVcM7QMhK2HZMJNbaT9O\\\\\\\/PsgYU2C9Iw7dzAb0XUQxp0qGUrcbKZeADO0SgO32oaiVnDSQzHJoHGZJva7rKraYKQEkTO09j+LGCsMj5q2LmqgqomosU3DJGD1EgppisrwCO73XIEqOqKnpXACUI5sz4WAjVOfKsBGZDxlu2Ct3qIQnWj5NGpQlnfkvwkgaRH2nW9Iy8EmPB7z8vpxHFcmO1o9fLHZkPffbYIt\\\\\\\/zCOVlNJGVS0Gq5Cj2TziqPbHGpiY9H5uRR1Smdeyq1iMA40BjfQ+y\\\\\\\/tiO45QL77YmksosTeo1NITIAnv8pQqMxQVYY6qv\\\\\\\/mAKP1ogEBvsePygf8B2W7O+Krwc3DXgOhCzVWDWR6IPQEkoiKtIgr2j1DWkXVMfBdWS3v9QaYpxfCyWHebMZXvtpL8ix0hTu37+\\\\\\\/X2IMwioyoHuU\\\\\\\/lm65qqHYwDpbuqqCYYTxP\\\\\\\/jsu4Gpp1JjyZqy5t5Z0qChwneNY8Yd\\\\\\\/DZTwOb6VaQwnKCAGsb6V1pEyzGBr+BEZ4d2E1MUJp4vIVrJFX2HUAbhx\\\\\\\/TNH0s68ovKe\\\\\\\/OaL4b6BP0nwb\\\\\\\/2Xj85tflPIWu6+hZ1zMRg654goLlFoil3ynk4ypbKGKG05a8vvZKdzJzV\\\\\\\/D+xlxtcKln4B49TXCWTaTM2AmWSzUGGNi6AjRKZdu7TkJxuq89Nl0RTterrdzsQI\\\\\\\/nWHbAqgARzG2miK7p1tK9pFhCTb\\\\\\\/cswxQ5ajN5wcU4Cbp08wBbgwm8AuLW8LnlW15L8oJZwi0HOTh3G1uQFJYKcfncQAwfPRo13RY0HWZDbQiRukJsQpsOOD5h3EcCJiiX4YIovfdo4eNGEmMGYTE+E2ByYt3MkCE7LrjP9Zw5WS9TAx+TfcH9iNq1wQeQcB1hRnDWjkHkzQ8L5LvTK8fY9kGZsQHPlCwrXKFq+OtAWrqk9nGrW\\\\\\\/CHtbhtDSmGm4ZS3uDRrWD2h7ho4VmEnjaD5elWZ4Z3f+qd6tRKoc4jJ8KjAqKvgiTXtzrzlsNoaywoW0eHaQ1WRGMUPInjvwXwlJ8t08y5rJ\\\\\\\/nIJQbdAOFupykihRkqtp7vmjS1Kw2Qv6tPlAOvkR9Jd\\\\\\\/Sr2UWnaKUTTbTj+ZeU1+6hGlZHgGkimJwWJZx6SdDGp2GzQRuq5ru\\\\\\\/g1K9Oye9S9LbVZDiJK45vNLDpnodsUqBKH0SiTZV1oAwu5TlZSuo0Z+n35h\\\\\\\/3ck8TFW2rYbHv58Ya3azmBW1Jt2Yis+90KU6VuN3eWxI\\\\\\\/1gvmoHUTRBV3nAQCS2ztxuGPBt7whaAHpXaZdIApSMevlTUtQViLGQOU+lEb75uN8afStGcP+F6sei612EFRVfjSRoFB5pUCg0LvUtzdwMnhI+h\\\\\\\/L1OtQ7Q9Z0VSmKtCD5B\\\\\\\/4qd9goXeB2LFkDwBaWjzNvawtJcVRdjz771CXUnfNslwbfrS0nz0b7r56pNmmZZezpiuUVUNteQL2+dJYvps++qM3KPMmQeewfhD\\\\\\\/zplDGGO64Xd1chAmUwhrhqWqaX8Fd5Sw6Gim3GFheSAL6uwvV72FW4ph4xr7391mGZHL\\\\\\\/vjOJ\\\\\\\/rh+kZ4qtURl57AZYnp9nkGrNcGsS2PBU6Q2UbMyqo73ayrs\\\\\\\/vJxT4lakmkllTd6nF\\\\\\\/0LljEi3mpcMrYJujbA6DB7JVOagGuDoH0YjKb5Yfi\\\\\\\/cSpfOgsa3ZZQzXqr4Jf3vCycKZj8QK4hVLeXq3X6yuIW4OIagAHfnu+ZcqW+5bvOjY45EW2yWhrlnP8mSOp\\\\\\\/KcLtsIyRASO+ALRnusDHsCVeT4oZiGSTLkH0vm\\\\\\\/eetbOvdD2Rn5VzpWkTKAITsAZQcD91+Go2f1StbNSfU8PDSU7g432P0\\\\\\\/NzyxttAsVD9ZeSuUe4ymePnCcmKbuDtkOa5pgljqXLkWifoLQVjZjZ0Ae6afHhLWWewKx\\\\\\\/Lv7GbcmoyRwplhZ3GaUBuPwRq6VzXFbqQ54934Z9YU6HCGxJSUccClBiUbCh3ejLydttStV2Y4wlQ+AJnlMRjIMiTskVXq1J4PAeQoYFQuYWcSD3xS7Nl+PKmGufKUTh8rEj9wcaog+QIsviHkuuRzlbyqLdKn50kzTrugUwUqdv6rd2rWsRXQhgjmY2fcZIeQ2sLQ2eN4jFJh0ld3K9H8zpBjh5qGgPV39eqEHk5gI3RI\\\\\\\/PBrcsY8fEbK\\\\\\\/ag+G9oEuK5srHmQcsQXdOm4BQoh\\\\\\\/x11a9RiC1pILx3QHjHFl9BnTO0kZItmtcv4vl79Na3arDI5NaEOabgcbu+fzZrfG2VJRpRLooa+Lo8Ayo3f9dkhPxmX2gXb+UqucrGfVFqgYSoFujJDc966B5YkKkBgLHOxl1sL2GSVOAA5xCZ7823QU6fh5ofL3JLWL20XSMJWBsZlLvJipkNDMPxArn4d39fssI6yuhigz9ebLpYI1sH7bHAHyXoBUlO6Y+WrI6nfcOYQdPTBiAP0W1EBJoxsjjK1DQc7YPVubQ1o91pTtiRNBs0bhQ4XIW08GlJBEe7vClPfAhKES7cQ8PlRJRUx372XHRASXXKnG16GZTjMicoRQXpYrBnZ+LjJThE12bohKGqbsBQLk8PGapyaH4haHMf7UfRE9yPHPkV4r9mD+B2y0xF6CHaEfL7CSTxDdgRa0tE7gKgMqepO0KKF71+qVWJthX8njDdyKhqpRJIRy0WgDQt8uo76QS3wxhAdWb\\\\\\\/LQJu+2BuNj2zgKMbGKf7hYerMiWzHvbOi4NWpHLFcOYB7Bcp7PfJgGm\\\\\\\/uESZ6qp7HJM9v5mEFkeT\\\\\\\/xfNTcWT+INIU9LToZFrZodhYkzA4NBmC48nzhPtzV18huiu90mgBw3cLVrZKBEOx5LSQmdCcAhQL\\\\\\\/Fh898XmsYTC69rfMxfhLoS+mWTy36LFg012q\\\\\\\/rfFTRGjz0\\\\\\\/U9G\\\\\\\/kajP6e7mB3IwzAoqJlTv+EnCB535n\\\\\\\/TMc2\\\\\\\/YlyH2d5H8OIgTqtgBzc00hyrRmqtPFcPQlE5PgloxjjtOBLmGVqVbYM6rj3A9S1UEYn5thbl1MoZE1y8cOcxKIejBNjxQegFYdHDC9dKDx1P291+oN5p45zSPVZSEseVqd+tE3VcwBS4QkbmARv3QoYG02iesWhXvoRDaZsSBR+OtlwaUxJByUVl+2H+bRcBcvQ4KnEWWDaTWiJxpnsrefsA0JNwJM4wL2cAxlQJqCT2tYlU0KgvGALbJvJGYAS6SCy9wi8lZumbpo6hWoM12BXgy8uDMUnG4OycFiMMl87AzZBTWrPVglNCoV8I9pll9xuJbornCRQoXgHsGusJEp96LkS6FFDc\\\\\\\/ODAleQUDf5jHgq6UblZl7KCQGxl0YGzW3zlBUaJ9Wdu7oKvY4gihbRrZ6+aosxs6qRc4LN\\\\\\\/l7njiKXqjM155YXvRjXMFRjJjSMG3hw5gqoDXuScbJKGO51b+sCHuR66E+dRrpwVCBIoQe2qTZbcw0h+tvtw2IUS8Op9DSorS\\\\\\\/LPo9OvsJz8uELma0nTF2s5xAkJEIlf6sT4loCSu0oHelYqYKPrl1\\\\\\\/0iJ+5VDQF7P8ElAR23XqjO8OVnDWQBB51iICN0zCH47TNheCGbaMPlITGulQ3PQ6fGw+WWs9r1IPoSHcJtoQ+zOuGVCY2jfxr7yJmYtsgqdOtVG2o6jHHPxXY0k0ZWi4nQj3Mf87WXUmyYCDq7VY+BejoyAda79cxzFkao8kcv+kZeTvqSX7HuVdMIqr\\\\\\\/lPx+tb0TJNUy39X0j7SD4wN6soe0PJb\\\\\\\/HVF8OgC6xY0O\\\\\\\/f2MIteiqblzOi5CPu25Lk\\\\\\\/p5s68Z7Mrjyf0UUyvij1+YLfz9gHT4DU5sxVn3IZo88mRylqtc+DKG3BxVO3no8\\\\\\\/fo+vRD\\\\\\\/W70WS3RzugYOEL6aw9xDXmruoM0Qhg+cDSDML41swQ8w05pSWsiyyJrVXuoVLTPQEAeJRueM6g0+WyTv8LQKQaH56EbqvTWPLsFMockhxxQHEvvDRBlvJ9w++0fWKOTBMGKLqN0mFIQnDV1SZFN8V83wYMnATZZA1+9iAbmCV249Rhz9+p5er5KZK4nbcRQ8S79QM9q4p62EBVig75wNAMf\\\\\\\/kkqEY281L\\\\\\\/SNCwLm3m7+7QmMdUqksdHVxZvqm08b22GRgqos4UFL2XKfS316ntVeeIoRX74Qk4b9hiS1fqg\\\\\\\/45B0ANqtkUMWOaL1m1rgu\\\\\\\/qsWcmSDSDtR0G9DOTFyWM4ygBeNE5WTVSW8xU\\\\\\\/2qpfweeY4FnV6N+F+aftEY8oC+JBL8TsHlENoCJpMVQFmSH+zHkm5T0SCiJ1UN1hp0uxT5oYGRoKH\\\\\\\/Ei6GvA2PTfMbEo5ltnIw\\\\\\\/AFCkktizTgh9aqaU49bSOL4v8b35DIPUt3h0CRlUXQd6\\\\\\\/yxDzKWv7ZAN7nM57veJPfU2t7Yahc+uFobhQ9D9ejS9bmt0aRVgzd+xCQv1IseTBhgETii3V4NQHoy0lNwMdihjOHsitnOwKoGSHeu24U+2qKx5eK8ClGxWdDrvP67xotsReAqiTvTF3QoFyapP7KclbsQb9JelZNC9p7NiLO3HVinX7qK0H08dWMW51T2+6QjnAi61Kwlhaw8DNLzmlFzUcx6aIhEu+GyuVCU3FwIcyq4gmcmCVOxUPM9abTD\\\\\\\/CCI+2mJmx0m1jSpaync8AVMfSg7sO3CL9w5wypVKv70kFxnRmA6OjvVQPUDGN8aHpkX5kQsP8wCDu5NUqANfn4XwrOkXCwwHrbUsaeFM5GjbwucSdcyh92d\\\\\\\/+uEmcLdt45YPgZjYNbTSGVAyr\\\\\\\/YvTjRf4mIYjUp7B3ckn2J8PGsmhB2k4DoENdnRzzKPNbQnzMRSv5az+5zfp0QHp8AJxsnO\\\\\\\/TvoZzb4+OTflry6XvBYINylmJaXR1hwwyB17sBGUYYAc0rxmFvV0c2Krc+BWkmOvfTxXzXXrj4pQeBxdk1TmncyuKbBLV+KFhLR4mMcQhqRkPmUcY8VzJDG+z7+znEz9uMrj2iE8zwvzIc1lj3FciloaGHLDF1B1NPitPshq+PAw+FLf+355FvXhA8TpB\\\\\\\/+KQa1hXeU6SY8TjICbzNTH0KiMqvnEEJzvnE8KSbiUFA1I32Ul0GjPPq+Lm1SRWFnnRiKExWO3TQgN80vCG+OgwZ5paKv31KZ5ujJuwDDm5h4NUeb9BhKhCmNb37H\\\\\\\/kXIpjoVhq45EYCxAYzqauztwhL0x1G84xdfG\\\\\\\/UC8vH0ceDS7Q8VbqQ0i8i8Z5zb6mF2OwC2yD+g4F3dwM0tfE9WMh+\\\\\\\/dCzSnG2uGfP9oY7ZBUFkB8HhVzk6dmbof+xQP0W7K3WB0U+5R1EylqWlM0tftxYUZPIgMQTlWjnkupjPY\\\\\\\/gPVHTkeBoH4z6TkdD0YO\\\\\\\/ASOX5nolerPtZQhV797eXM+0AiX2ZX6CWFtnHl0SkfYtBkPcxKwJ43TTq\\\\\\\/nuJnZl8UAoXgOjJktcWlBLMiuVq8QcvpeNLBZ+44TwA9P39OTKmq8lyXO8alhzX0dpQ2yqA084tuPEvXYlZukWEFDsk6JTxEhFajYxor7zsIL7iNHyCVDbljX7VRJdMDBjPqSJqJU8JFsvFZo9pi+TyLKXGc8FtESj80U458p+8ZhJUwpy9R2O6CntxYGkasnxxXQI+\\\\\\\/QryfQsU3UY4WTd1KJo7ttMTLaiNtk6bkW0HBhfoffBzuBwfrhWBlXMMyQL75g\\\\\\\/\\\\\\\/JIXA1LsTPrb1vzdBLQ39PfD4K2yw8+gCBLTxlH\\\\\\\/yHVTUYvD92VFiZPmJPvUzJ\\\\\\\/zNxmpFuq8XSE7RAeLInkP\\\\\\\/NMZryUpeoyn8kTZc41qVMasYmV7wiEPZ2EWO5JYBfKgYU1zDHtCnbd8zFb5djwFh9i4Wl3embsRczGF9LQZEGX44o19iLRxw4TyaJE+t\\\\\\\/aeu49gCQl8hVSAIlyg4Mi5m\\\\\\\/33dlEug4TjQaGIPxkPBSL\\\\\\\/fjwWwOBLJYaksQ6l6XR2P7KgdKM7CHuQ\\\\\\\/yjn7zFhjBk+1IwqWIv3iRvpr+pqY5Z1oVVvTh7lNrfRg4HVFwSNczlvHqovAZxv6dwqXOvwTdFcubkiMrTZ+tSIpvUNSfFOSm543axG2GmXvy\\\\\\\/psDpr5IoPHju9fN980iSCJHLE5e7AKHT3vnWp5N0HSnTAyo78N7A6dEw6X4hruuefKu60kOoJvP4TNSig+G6mF1zhrnX0z51NY4xWAX2H8ipHELytdMkHPxQHLFk61NqcxyxzImvET1ZNmXCXHDHNV6qeOxKwhZKi5U71pE2tkn2Gtqy4BD2HXVeyAmbL0YrEghktOd7AlpPjd2DGeYgaZsLVJai84PKdAuCGzJ06ncuouubXIIsgld9h30K3\\\\\\\/bsb7e2s5ZBfgaUyfANk6SJOMCLaYopzQEHeU6rMMsJNgCAxCVaRFossvsPlpYgUe+6vte5kezTXrpeimxyUuGRgKadZFOj9uo0FiaBcqHXCv\\\\\\\/LhxlYyn8FcCqboVIfJoDvrvIIgBtHHxwP9nt2jMkYGj6vZUOAwPRLgi7TCNA7pN\\\\\\\/Zbe2pyjo+nLiRcPnCcQCGLnjtD4PQGrKDO73L93rfekKazwxPsF1FlyGkEmLdRjG76Z6PZJO5nW8vlzTiP2kk2fjAV9WZFebiVthtY8iCjkMjOW67loXmZRrMmNV9c\\\\\\\/MiXueAOcWKsWOeZjfMThBHbQslRqZ8HV6SPSp7h5GT0GxL4Pe\\\\\\\/5msIIERWk+llswF7wjqHXjyEkqiRBM5GuBktAuxHJkke8NczePamS11cyw\\\\\\\/99lhCUNIHF2kbUzq7V6CE8nzjoNUniNYNVlajOGccs7Cj7EUyoePk7IE\\\\\\\/QrmzwhYIPqt8i1DDw\\\\\\\/Cornw9ag9UkA74SeKJt50brjcQGWcCIzkCt4iUaxSK\\\\\\\/QQRrN6uCn1rBHEzXWeyo2nx4\\\\\\\/FLKmJP5eCFFpJTm5gudPlLzjhSR7tMPa8qBbnAyog\\\\\\\/eJu8ZCm51vlhpigy3IDYdOZ3GNMMmvMQxdH5N2UgX1i7sEOK9nf\\\\\\\/e7wbZ\\\\\\\/LUK6mVDEfxa5QYNbCJJ9\\\\\\\/vgkHYYURbBqvOcGxH7Z7Hl2PTxmx3TfbyjJCR8S426eoOzZ7BI6RYWK04PpmiZcVd3P2RTjP3FO\\\\\\\/Vo0UieA0P0f\\\\\\\/KwEH2iINqFvUVW7j8mDj3cEAi5LYzLl7Lx+Y3dFyaHy2Y91AOS96QcBSvU841hZbGCQlWZjxbbdB7eMEum+o98vHjWrlTfprxjkPFS0SNDf4JBTbYm+26sqtSwgMxIzqpR\\\\\\\/iHEQ9n\\\\\\\/\\\\\\\/2ZR891FtclmeLbhzBG8MhpRUh+4XqNYxWxnzKaovF8iTRM85zGL0rw6Ka2WvXvfF\\\\\\\/xiko0ezni7sBNrq5O1Fsp\\\\\\\/3Y4dMH1jsBj37TkJidXxy\\\\\\\/BbhnTQhiFbW7iedDvlEuEHEcq5VIoSZsQ4nWS9rZosnsVqB5zNqxcGJbzMfEV1eZ\\\\\\\/pC6OeUxFvnhGxqolHk5hr+GZ4TiQE8hRu3eoOZVkHgt0nJ0FUzjT6\\\\\\\/04RORDrrMrnmmFhvMA2pRzWmf\\\\\\\/VcwQOU91S9As0aNIxeL6mOEizMREZXtJEqo7jfi\\\\\\\/\\\\\\\/L+87W6B2cEKwNcAUIz\\\\\\\/f96eVqnr\\\\\\\/HI2kcHeQjc5px804rAuOGveC\\\\\\\/j\\\\\\\/sArkgnlUWOVPBCDOJebE+Mgf0q7NUKj\\\\\\\/fhyAPG4u7z\\\\\\\/jPJuJn+6kR3ls0lSLG9e3fopQDfqVF8j9qhi8iwC\\\\\\\/nRU40Vizxsg4IQXt7dtLX3tjcVFA6J7QSSGJlkKdodHVrI5hYv39ehbFbdQmO50iohXOAvAypS78gIB\\\\\\\/Z\\\\\\\/U4bQ9QaPn5+Cg0MmGsOBYP4aa4RX4hS+DKPzgbwi97absgksrNIntDNhLb78f0DJESxZrgfWK416uXyTiy\\\\\\\/By+No5Nq+YMaNvahxuxQI\\\\\\\/1IzdmuCegLpIE3BzU7+x8Tzgjkq0MZq\\\\\\\/DS721RQRvzkGo2xOx5Y9Rj\\\\\\\/uffdALwITdOtNmN5Y6f3sSCwUfb7EFF1jlZ6MP\\\\\\\/m+nxAkbxOLoME7ZefaGCtku4H7yTRvBoxZLDtaXuL5FeuOU44HRizBs0CQUixqG7UUt\\\\\\\/BMFx16y4f5HLD9w2RwSpXbU8pOvxTTaUS4IukyNR5iyBgO8ecLgvKvq3vJ+gmKTF85kCUhRsj\\\\\\\/Xp708HTcJSQX4du9uRpyrXviwdNuUZv0I4uoF4sdzd4cpoVKYJwL9z2Z+dG0zjWTBB+vMde5x61Gc3XnuXaS12O2ym0pudbMnpShFgKDdBsgVXCIvYESQabK5bRxd6Jynhi0VrqQqrj8y\\\\\\\/cImrftbVgOCqRccbJdd5Oz7hXodcFfni3BOfxh9F8CmR2BkD9veIupsdzL8+FzR0Ijj4FSqRhFX358xcdD2tYQnp8TFm9ISEUgl9kL014FGCw+bfLPT974aGt6JQF3HZ2FCOtn8dquD4QkLC61gsD+lWY3z\\\\\\\/uOtk4a1ne8A8cZgJSYBTTchXXiNJz+Vke19RZWWAgxeY+MKiUBJvmpaMMMupm3WRO+iTfqpH1Aj5c1r1FOFAUojDvaMeg88lply7F+RGm152Ckk8nGuefyTQR3gxs1UQVd+UZT92I48ILRBfheLEN+VGTlcN+bLkjC2fWlvZWRlVjgCdDa2yzxJPnQKVZPfu8pkvMa2JUFREf1ujbJ\\\\\\\/y+UW22OER3sBKxWddG8rqH8NO15JIKfRFL265rv7h3C5xsJL9wGtX7hWAehE5SEW4icr\\\\\\\/4wwQYqXYW+Xrpqs2JJdoZ4IIrUxpoDV9ZdxJdhjQ\\\\\\\/Obp7kiOaBJ6NechTb+IxMXkLc1VYsdeEaYJ60FjmPNymUp3jCf6qc4difWRWJmN2R\\\\\\\/q3cdW0VyzCKy\\\\\\\/1KQnc+aH6UfhSO\\\\\\\/0ad7GVy6\\\\\\\/ETtrLvZOlIqTE8FmUhuAf1ry2aQRwu4COupo1gRohny743Tjc+yOpbt4sdJRx8cwBQyPfgLR1jGB0hwT1Ir322\\\\\\\/1EEQ\\\\\\\/Eepp9aKK3oqj9Bqyg72fvdmuKHxG5tTjn8RY+4lt5DJrEmajZw89an5LfONyOflB8a511BOqGaakoYU521aHQKIAflqXOXb4UKn\\\\\\\/229\\\\\\\/LhVMG6\\\\\\\/Ze4uwF7sj5N+xXQM21r3QUSN0xNifz8FF\\\\\\\/JeB74GDwFHG8i\\\\\\\/I62oqQhtdSGR4oO25ReOPMegxTerv\\\\\\\/3sn5rxkvZwsHsoYHHLAcbVwi2M1Xa+f8n2cSzkdCbTrlI1fScGuuRgjS4BhGMsdFdMv1tUsXaEbI6BRJHLhAGOCS+ImomN4fvr4yOm1NhKcr+lHL0PWTDwoS\\\\\\\/BgXeFVGwfCTr+dWl+4cwJuQCkNlNczFmGk7kbz+2By70fNlQH0o7lzX9wXgTqOaPF8Yrucehr4DH5b\\\\\\\/TZLG9TXvBgmgCxZLqQETSfbfquczIf8vNlNm5GS0Ye2eFEcbqJuVNyB1XubrM+jmizwB46v3CjgZme9JE3saJhsYhlUXyOTCNB8I46KWm2A8MezOPc9Zr7DMq8yGePz\\\\\\\/Itiq+TsoglY1JC9KnSVDp0VzkF3Huu9HjWbXw6oo5tp4OBwiKsTAjMenysZOtf9PSfzaUaOyhF+phdMFytwKAz4lJOsN4UhP\\\\\\\/ajOnYVZOwte7UC2JeEl1XEapHbCR81ku\\\\\\\/4qx5ByJswrYDw8OqXgD7C8rd0rujCV7M7TIucnlIz8Sr7ra7mjmtvXcc6zQxDGsWx9OTOc+MO6aERb8Fp69gauCVtyoj9Wf4N3FVuU\\\\\\\/C3LrPSve22Uad+RQJKDiFmyIp\\\\\\\/3\\\\\\\/A5mNrmt1KsW+MGsljq7N4sYPqbeVxdzSl8bxX3zBLWBeFJDii7dbEsJIhmu+VHeT3tXxTWv3yM0CufbUWRTCHhyYl9WNOHSgzF\\\\\\\/9kdMGPzdtWvK1xuEHrrZvJvjiC1bf5v+PXgd80IEvUooBnQY7cZu6ueggXmYJxlu4QGyc062PuZdc7mBilbz8Pe744VyhF\\\\\\\/QNJ7fIcRZKp3lDJFVGDeRco0djyFac6VMVVbueqS5fZHVzpdLnYSg6gCTNy7uebLJnYIH977jVT+WJhHajAMpyIGTwAG2KMmY1d7ZL\\\\\\\/COfImCDi+mJsOFIgclCXFqkfyzimPiNVOma2K3r39iLlAQzTECBRsa\\\\\\\/wOfndfzqGwRsly27I5bDixJ9SNfTibPi0B0RKAWI+OTpqEj2dbQWtEZpp2V5lKsc1vysUkdMq4fz4R8\\\\\\\/TUN4Pr4NVYQyaQFlRkGv4zhQ7mvBOQir4NUWTbiNfaIUyCClFgtKHZLCFMV7M3S56Niz6TkfgUgNWL+Ev18cRUOq+\\\\\\\/6sEJbum+L9Szy22CGrA3zU+xfydzIMNZ7L9XsCSfUCkR2a6d5mMSRAFbZJ65t6z\\\\\\\/yb35yEm8UOODkn6Jvs8Bx8F+Axu3bLuWvH7FEb5ii2k8cSLWjRRS5HHrSOSsomm+A6xBeKA3MMODJzGTpe1byhxH0rEsxawIuMYFIE0PyU78w00xnaSlf56nYwQ6VkZVUVsmac51vEnNQcDfVZKf+wnnp3cY+1h2DOYhHBOMyeZc06tEP3DGJNAvYqOoysZNmQGdJqDo9Oocgj3Ecy4ZO+EVdOd5h5ydAAcIKWMqSCTt8kFfVVqvEbGGDtDMfrJG4v6NvFTmX6fqL4ObsQOz1pU7Na19z\\\\\\\/n72tsCv6iGZRumziwVSpOWNpDGflSuZPkBvtFDGFK5XfTRjZZAkPWkn2gGnkSsKQcE+MF7q0ucC+TEvpY7bROjbskZXL0yMCd2HzH04MJHZ23YxzNjSBdNdCtmAPy+PhKTncxLue1a78HU46LwpTKe78gs7EG3Ac+eati7iPeU12vs9q12T4n6oriu8znDBpqYHeJ6WFj4BHFm+I2ac7hh9MH6XIR1BpDRZP+E3KIRmLZGuZ7RkcuZkHpYozqpQGJt94v+o8MVPTn3RjQHzDyo7PPhYONYcrVIfqYG\\\\\\\/qaYJzdPxOwrYtXLvzS9RQaRCiXxcGPRpaHWyDddaQmUgoJeRKEhsk5Ll3eJAkYyWV2SW8J1nmhtKdBVHuvbJBZVGROJqKVpMVbJwS4fwTZ3sW4n02aQkXBjUmeLxDjpVZPKku4G1jtToXraJoQb8VDMFFzcjnQ4rNbKmN6On7JLIzSkNa3j4dBF4JnaIGRkSR3ynK7VTqHk8eDkhOrhRpUIGDMCtQu06MMs6NDtbWkTdfal7SzJ445qYC\\\\\\\/+V6gDs+5+hGfYCC\\\\\\\/3WkC2gVsl3EhDLLgOQ9goVFte2f+FRh1YAzJikGmX3kuSWRe\\\\\\\/BbkP7tJeyDoaWxetTf6JFhpDzgaMcXZOe5JDLvqa8b47RCEyy2OMqpFprWqNURVZDdhfm4tQKic3689fsmTeWUIvBmbT\\\\\\\/2Wdxr4LdjjMOUZayncdo+vnayeofHuC5BGthwqRDUdXaKb4c\\\\\\\/MgWy4Q0BiXvN28CkcWOKuLlAQ33NeH3OevM8aG67RRoe+TM6UcKiNfkTrcJOJqUbZFeHVOwieHzLEVaGlWLNgO6fD+6ylpS\\\\\\\/epjGlmdndQ31N6wZy+fl6+\\\\\\\/HL2iHd6g3syE4kkrIGa67BbnpG4KywJuNfKrvna5FtEL6IuVMgnXwoghi\\\\\\\/6l8+bwbv\\\\\\\/WXcSsbCieqLCzsFEprlq7fp2sMJnic0qUhz+z56pj6YFV9JjwFdRgFnC9f5lnVXN17Mvls5i69iFbh4fZJQ6XVuXPLMxTAwrcDNRaVC8hiMjwBjTyFs\\\\\\\/grULkYqlxfUox27Q7w+g+E2XbfKxP9tNQtKQ8roDVDBvmKmoBqoioS0aRY9vsTSYPk9zWH8qr1Gx+10UUjJ\\\\\\\/Bv\\\\\\\/8okR0mo\\\\\\\/ynXiRMD3keFzU82hzBcVqvY5BzELSbTWkzAFhdZlRRJ9BEobK4CdfSbr04HWwKkSE8tNZ2UrotJTVvy61vsYAtioSOIH2EGL4lK8hRHMNuuwcMw07w\\\\\\\/K+QQlIO2XtHKzncm91ZJ6lTMutpZPMRjaIlTNH7naW6wSCK+Jwd7DNqT99hIsCmcy4qn5GaWsmcIHzw4zsWINOeVq3oMMQobRKQ8WWdpUEL5y7Vb2aDBaMD4IxCIHWDnnMDvLNbPXUOvvhK1WTG2ZMMwL27gXPKjhQ7M1MR2pdInG3TxglpmoBiMqLuBbrCzluIa\\\\\\\/vC+IUEXtAg8+OzQExSn2SMCOretTkGb2wiJJtl4LM7jlqRMZScjkC+a2LukuZSrDEFA+KwiCo\\\\\\\/PFNEZsa9RZz\\\\\\\/HhxgBgqk6vVehzbKce+Kj4emgBuuXX+On5dOqvaoMxpP4hi7867g6TpRCyUt5qZtH3XZjPfoMjQsQkGJPa4v5BQs\\\\\\\/tbuEan273XquVuVicnzUgEf9ZRHrtRfJvAYUIfWeaNla7FhRuehItz\\\\\\\/2ITR+g1gNyNXkM8l8Jw\\\\\\\/NtVlcTzj2iTxJFd4CPKnHRc91coi1\\\\\\\/9bnoq9SHcJlerSSZT2k5i+LAuzMkK0u83eqFJb83M\\\\\\\/JCXJJxMGR39o9lsDFpdWV1JmibrWCjVieRlpuv7iilsuTBYCYj0afrlFJg72DtD4Z34fCBAxOAr1FVaqVLA2QvxxbjAQ\\\\\\\/IaWEirrFGEevcEzv6i8zyIm4PYB7CbU6cvxbKIx++1QFXXvMzgSiDORSecF3Emo72LWQ0YwgZf1az4CFoQqQznV24PvZQ4T6963GIpdt3UlYT\\\\\\\/A8PPMm9BqioYp+SySgW4KW9\\\\\\\/dJVQnUacpAu\\\\\\\/Y7NniHb+ZCI9HDeuHryQBoyDBUkq\\\\\\\/qDzcb\\\\\\\/XhSHRFclJntSenLNHNyX8uDNipA+TrselK13XJvC\\\\\\\/8uGk1WhagoWbLaLIXCm5Yd6LFL4L1ZFpPpcJTDdepc\\\\\\\/1vemeqI0tfFoqdFGCD\\\\\\\/2jCug0Jag2KMWr+hGSQwEjsYUCtRI1EdwpIVE+zJmIxWaOzM8GRSDDRm4f3AnktVX8KklX6afSrYZ0OkQwZOxa2jJirbJqPjWDJFBsK20LsQ3716xfkPk7BW7ah\\\\\\\/kt8hY7uHGHSUVUv9\\\\\\\/iWusVVH8b7bbq9NqUGfsSEJzj+4c9l1j8+gr6DJIx2FMJM2eOR\\\\\\\/1tGuPYzsdek6ueMSVjZNanaKax2LecrLthNtfsna9Zcekn7xUr80LyNiVlCH2Zsb\\\\\\\/RuiXtmXKfMtNSjbs3KLpcnRqygFT2WUE0FHwpVt3PKxNy2AnlZHC9fm48MJ+Jmf1GZhoX2HpEYGrJTLTlHD7w344bgaUS9OZcWJHUs56EK3MrcM8atEuLnHRffgRme2O66cwLt1wrs\\\\\\\/WWU1B\\\\\\\/GT8FiyUPqDyayXI150LMm1iuHgqu8CJEC6HsvuStQUiFT7uMaTt6xSEY0qlaLNPpyXrLIxvw4xxM1HKH+R2rObSgqyNLNgVCbIL1xfBdEySGttxMMPcAO\\\\\\\/ooLcJG5bN9ywKln1RDeD8xWZEDc1g61\\\\\\\/wLEYDdyU8ob6zBBitTYx4kIxBa44eEC\\\\\\\/Iwy86sY5nrrIjdEElT2YtpUsJGl8y3PfdsV12Ax\\\\\\\/OH\\\\\\\/cHpA9ulYmKiRNAMIYXRSJ9kk5lKXWV5QNXuCWSYOiZdF8F97OnrmCwdSxAU77sd4JDtYTq47zPL49PZnk74qLUCtxP\\\\\\\/KRi08os85of9fRaF6W69JKPA6xMSm4vYBOl5Czl7V82r2MApICog1TTMUdCLXDhx+aMtcBKroVyJj0CyZeCzyt3Bwtxv9jZc67iBwuxJKujuRGaredG\\\\\\\/UpOtUffD+le7PN0ZwPTHST4NFLLoNFlTLEc6AxvBOemf0pIWSrLjFVECaXCwfmo4yRJTvvh2oBYnbVN2Bff+uzQc0QloDND3RELokcIuzT2P5bgvc+9I04ZqJuD9dLcbckTZm\\\\\\\/jguQFPns6Ui\\\\\\\/ilWNj+AuXg+\\\\\\\/b5vwJ3A5DdZIcG9hNcKPdp8O+eLlWxNJ\\\\\\\/t8ZNkrKASkGtFm1OZ27Osw\\\\\\\/i+2D+y\\\\\\\/pTr9S9UDQwhRvt8CziuTDEYbwCaxxaaJLC0YjeuLnQ5eluTYICVeO6XSWZcaDcc9jxErHD3O0geOklaLOXFRm9Qw8P5i4v6Wl9XZw00jYKYiG89PinWPm5AEmjCrJU+ppVPaZKv3N2gUEps3x8Xme2nAioSWlq3gudKzT6hHaRL9Pe8lrqSTLze6T4+VcsQJ6L+5F8npvGcPquzJ\\\\\\\/Nm\\\\\\\/kiUzanx1vcltBvcQbfvcUgUNEnMr9DyJqoNVXRdwUCWHqGhs1jwZDB9HypMgkpnF8jNqRM3X1pWpM8fdIZsBEXOCpJDwPYarM+KzCa7dwmh\\\\\\\/d5KNAG5Zbi0SyY4UG4q6Mcr+WGMxJGbiq\\\\\\\/CtPXz0OTnD+ElRnVMZjh3IQTAVt4wPUUBaTfqeGhGwIvvoaDz9UUxVQxdDOLi\\\\\\\/rDDmeIqbHk5n4Vo15YmKy+mzocc4K7lPtPapRHQIPQ\\\\\\\/TkElVccHqZbDTjaY6GiJbXC8nuh8IZmiazrZCJpzG9f1etknloclTafpFdJuVLwXLrBV3FDAOdM80cdQ4YD37GClbXhLzlP\\\\\\\/tG42ybiDaYdkRrTYKoOKPuJc6r7do\\\\\\\/GRldkuJxtDNdyugP\\\\\\\/iBDYjO+kvmv\\\\\\\/yytYcRAROWX6VzpjrWlXgqOZ5UcD8U3DHBMLF9lrVLLkNvCV4TsmudLIC3UoJ5USJJm+fpXUKzZmh\\\\\\\/3XuO+3mBwI2D1N2n0rdAe5ItJxGg\\\\\\\/fnajNCU54v+sVR7ljSvtzf3jNv2n\\\\\\\/FiZ1Dvg5oTyrKktwiAFI6ssuGuXHXMRjybw46xw9yKECdX4IIJ+tI3WsZGRuzCoHL89pJGdDFIoG4wCVlL1k4ijzwqsLn3KKHGT2W+uty30n6mH5GNNlGQqJdvaLDezm14on6MshhBDIOZ3jez7I+6Jj78lXrcJCI0ULbEFWPgdZVbxxD0Xs1gkpQIMe4fFHaJVUImt1DraQM9DSj87I2rWVBcoNlqaAgX2SIJxw+hna7d4LzTiA0nDPw==\\\u0022,\\\u0022iv\\\u0022:\\\u00229r1c\\\\\\\/bl+0PtbX6Rhy+dNWQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c4a766d0627ce3001e9302b8a80159cd62a89ac9175e96ac59f1a0c1afd3299b\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022UgPEK7V9QyRTYRsaeJk1cJOVI542X5t+KKD+XPRGJF33R+1zmcg1NaR\\\\\\\/jmTeuzL4amY3sN850bhTt+gRTJuRxB1U1Rut4TNdA0FpOWiMXWJlwhN+kPZsoPhqk\\\\\\\/vQ\\\\\\\/03WDAEnLYGQ9aDxUIDY0rZSJk9jT\\\\\\\/DXLrK\\\\\\\/knB9SjtjIEK6o7kcCEJwvW4rmbROzBnE\\\\\\\/27ySsG9aZOblQXiLJy+qgllgqCakOrU1aXNf+TamcqiM\\\\\\\/\\\\\\\/M2BJ8Bf4KDj1I5IZVnNsHBKSUQxED+twla0F2+jKXpdZoIk0AldMcWTiZPpjTo6fx1PhX2Rl9aQHst3yWFUOGL9ScKJ6GiLf23q5cTkM4RSNuV3w7jthEdw7IuriBvhQuehOtaHgKW9JI\\\\\\\/3jtZQRnKXhmxkD8iIA7MpJcqD6Ycj6W0WPwsOQW5hUiyvPLKR34wBZqn7nFdix71p8eaLFxUiiyTyjYXAvBblixgoZluov7nMkundpHFUNXXiveSZuhd6Cb3OpiUVrzaVCd8GlzIsiPQUmIDVBBgTVaxVOD8jz6ykI1O4S9\\\\\\\/rrnqC7uDswz4RzwF+pefNzgAsJOKsb3MUd5E6YhM4h1ZUAZgmr8ChZaPKnR8KFbglQEMgZ0Tq3wjJ+MuDZIhEVYllR\\\\\\\/2Ym4pBnnVSlF32+2BPU0a1OAQ6YbOoXnEQemRekHO\\\\\\\/yuCQT79le4qLvO9\\\\\\\/tIPYICB9CR6gCqu\\\\\\\/Z+Tbt2aEWuC9t9kGvtdo98MhU3p62PNjSoFfhBPAHMPdBuIASrmPG9Dp4a3QNRXTeh9jh+T74hpsCfTziyzCsFAsX7f8syQU3LVezaMzqiOz0B1CTzbW3w7XjDpiGqPi8L+Y3nr8JSHKy6fX4qNW9LzEjrWd1PE6TesXCIAMVg4p699yNtwsYEqof3JcKMZ2ZJ0ItiVOg3NHOAgjTSGFdM\\\\\\\/tsHwGY5361KTeeEMG5WfTdkROt0Hmde1zfV390V0zuH\\\\\\\/a8HZROSL4f695tISR0UafgZT+1yyH0afBx\\\\\\\/TQB39b5Tg3hZn\\\\\\\/8QDS6hereozzmtRvUSguWhw+MaHr+OzI481axGuyW912FUPGdDFDkNkmuIgFjG70nEnvjQlh2Fs5wp5EX8OSxHCTM5Cfxz0icR\\\\\\\/uePn2tv\\\\\\\/\\\\\\\/\\\\\\\/93WA8h92UpHermwc\\\\\\\/6dk0PoMO1XqIJUU9uqJfIxVBmOmEbNwZVOAIi8dG2rDsuyQdE6vWGs1GZUJ5q1mR5Zie6C2DSu5LrzM78kPUduegWtm4YCmjVxnjOJplVLaUqrVlTt5oC+6LIUzfYyo4pFj0uwnn598AWNnif4qOVK5k6AjQ74oXTyTWiBUEU+Tc\\\\\\\/7OMHZP2qnd9YiS6WJOTBIfQs1dtIS8hkjyF5sFZOkbnXfubLw7w1hjDdvI57U7NY2X1Jc2OT9CDdqAwEYKFjoGqbHibdgzXiKizM\\\\\\\/6bBdLvtAxhvLoyyuj5iehEWjhJkfmObggPT7\\\\\\\/BquS7WO9NF+LLWhnOogtdLcmZm8HlKRCRQwIS+6j0p9ijKSydCyOqtZVB3W1MfPIOX5doZb9fC3NeNtpSfwWvFb2\\\\\\\/GHGO+3p9yI8WV\\\\\\\/zBldVEezsok3mNdjLyKsLALbqkuevvv00M35REibUQRWyxiQ\\\\\\\/T6icqHefAa5k+\\\\\\\/j5N3lVGHk6OwcCmqIYl\\\\\\\/gzrYfB3uX12ICrmOqhioc9kQNDqFby35gXKY7QDcLUkDICl0YZP2elHplUmM6cT6302ZnAFefrMgs4TCX\\\\\\\/ydp4W9jLUoMbeEVYJBrh\\\\\\\/re7OLMMQBTnhxYXhisb7hJ2HFc7ZUoNBWt29zILEraxDUgSf42B\\\\\\\/zPDpL47LOATDNaYCt99le8S664fUFuHwg6G5Vx8l2KkWWSfzDV2QvmcH9yYgWfgN3XNwHFWBdwcZzoo7RQjQ3sKUhPoRHtSt85nB6JuTliskmc3uvpep7t3THlc7ytUPObdx2MqCjKyjMxogPV\\\\\\\/LUIEyzrlqS5qRcpWjTYRWtb7hkT+yFym9WkhunOHHuMZ8g2m4aiT8xgb21w==\\\u0022,\\\u0022iv\\\u0022:\\\u00227OXnJ28FUYRY2eBNUwFRaA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224c14c8cea8f125d00a7d3bd70a13975357cd0ccfa02100e336ed5ac95ce55c32\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459160 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigation">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/2/93" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-dlMKy75jWB0xQZhhRGqLPdvsDommnQTnsODpl7aFy7c" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/2/93" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-M_fEfwR6q2d-W12aCLUyLUWxLAVlhzDEtQs8-P72uLo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/2/85" title="Organization and Evolution of Subtelomeric Satellite Repeats in the Potato Genome" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/2/105" title="A QTL Study for Regions Contributing to Arabidopsis thaliana Root Skewing on Tilted Surfaces" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459160" id="node459160" data-pisa="ggg;1/2/93" data-pisa-master="ggg;g3.111.000349" data-apath="/ggg/1/2/93.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Translational Genomics in Legumes Allowed Placing <em>In Silico</em> 5460 Unigenes on the Pea Functional Map and Identified Candidate Genes in <em>Pisum sativum</em> L.</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Amandine Bordat</span>, <span class="highwire-citation-author" data-delta="1">Vincent Savois</span>, <span class="highwire-citation-author" data-delta="2">Marie Nicolas</span>, <span class="highwire-citation-author" data-delta="3">Jérome Salse</span>, <span class="highwire-citation-author" data-delta="4">Aurélie Chauveau</span>, <span class="highwire-citation-author" data-delta="5">Michael Bourgeois</span>, <span class="highwire-citation-author" data-delta="6">Jean Potier</span>, <span class="highwire-citation-author" data-delta="7">Hervé Houtin</span>, <span class="highwire-citation-author" data-delta="8">Céline Rond</span>, <span class="highwire-citation-author" data-delta="9">Florent Murat</span>, <span class="highwire-citation-author" data-delta="10">Pascal Marget</span>, <span class="highwire-citation-author" data-delta="11">Grégoire Aubert</span> and <span class="highwire-citation-author" data-delta="12">Judith Burstin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">93-103;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000349 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459160" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Amandine Bordat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amandine%2BBordat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bordat%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmandine%2BBordat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Vincent Savois </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vincent%2BSavois%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Savois%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVincent%2BSavois%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marie Nicolas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie%2BNicolas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nicolas%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie%2BNicolas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jérome Salse </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>INRA, UMR1095, 63100 Clermont-Ferrand, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%A9rome%2BSalse%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Salse%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ%25C3%25A9rome%2BSalse%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Aurélie Chauveau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aur%C3%A9lie%2BChauveau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chauveau%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAur%25C3%25A9lie%2BChauveau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Michael Bourgeois </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BBourgeois%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bourgeois%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BBourgeois%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jean Potier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean%2BPotier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Potier%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean%2BPotier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Hervé Houtin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Herv%C3%A9%2BHoutin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Houtin%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHerv%25C3%25A9%2BHoutin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Céline Rond </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C%C3%A9line%2BRond%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rond%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AC%25C3%25A9line%2BRond%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Florent Murat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>INRA, UMR1095, 63100 Clermont-Ferrand, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Florent%2BMurat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Murat%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFlorent%2BMurat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Pascal Marget </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pascal%2BMarget%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marget%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APascal%2BMarget%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Grégoire Aubert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gr%C3%A9goire%2BAubert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aubert%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGr%25C3%25A9goire%2BAubert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Judith Burstin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Judith%2BBurstin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burstin%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJudith%2BBurstin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:judith.burstin@dijon.inra.fr" class="" data-icon-position="" data-hide-link-title="0">judith.burstin@dijon.inra.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/1/2/93" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459160" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459160/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/2/93.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459160" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459160/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/2/93.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459160" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459160/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/2/93.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459160" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459160/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/1/2/93.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459160" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459160/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-6">To identify genes involved in phenotypic traits, translational genomics from highly characterized model plants to poorly characterized crop plants provides a valuable source of markers to saturate a zone of interest as well as functionally characterized candidate genes. In this paper, an integrated view of the pea genetic map was developed. A series of gene markers were mapped and their best reciprocal homologs were identified on <em>M. truncatula</em>, <em>L. japonicus</em>, soybean, and poplar pseudomolecules. Based on the syntenic relationships uncovered between pea and <em>M. truncatula</em>, 5460 pea Unigenes were tentatively placed on the consensus map. A new bioinformatics tool, <a href="http://www.thelegumeportal.net/pea_mtr_translational_toolkit">http://www.thelegumeportal.net/pea_mtr_translational_toolkit</a>, was developed that allows, for any gene sequence, to search its putative position on the pea consensus map and hence to search for candidate genes among neighboring Unigenes. As an example, a promising candidate gene for the hypernodulation mutation <em>nod3</em> in pea was proposed based on the map position of the likely homolog of <em>Pub1</em>, a <em>M. truncatula</em> gene involved in nodulation regulation. A broader view of pea genome evolution was obtained by revealing syntenic relationships between pea and sequenced genomes. Blocks of synteny were identified which gave new insights into the evolution of chromosome structure in Papillionoids and Eudicots. The power of the translational genomics approach was underlined.</p></div><span class="kwd-group-title">Keywords</span><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/pisum-sativum" class="hw-term hw-article-keyword hw-article-keyword-pisum-sativum" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Pisum sativum</em></a></li><li class="kwd"><a href="/keyword/functional-consensus-map" class="hw-term hw-article-keyword hw-article-keyword-functional-consensus-map" rel="nofollow">functional consensus map</a></li><li class="kwd"><a href="/keyword/synteny" class="hw-term hw-article-keyword hw-article-keyword-synteny" rel="nofollow">synteny</a></li><li class="kwd"><a href="/keyword/model-legume-species" class="hw-term hw-article-keyword hw-article-keyword-model-legume-species" rel="nofollow">model legume species</a></li><li class="kwd"><a href="/keyword/translational-genomics" class="hw-term hw-article-keyword hw-article-keyword-translational-genomics" rel="nofollow">translational genomics</a></li></ul><p id="p-7">Throughout history, navigators and explorers have established and progressively refined maps to discover new locations in the world. Similarly, precise genetic maps are required as a first step to locate and identify major genes as well as quantitative trait loci (QTL) involved in traits of interest. Functional genetic maps locate genes involved in physiological processes potentially underlying traits and allow identifying functional and positional candidate genes through translational genomics. Closely related species usually display syntenic regions in their genomes, in which several genes share similar map orders (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Young <em>et al.</em> 2005</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Zhu <em>et al.</em> 2005</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cannon <em>et al.</em> 2006</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Tang <em>et al.</em> 2008</a>). Positional candidate genes for QTL or mutations in the crop species can thus be identified in the syntenic regions of model species genomes. In legumes, excellent examples of this strategy are given by the identification of genes involved in the control of N-fixing symbiosis between pea and rhizobia through their <em>Medicago truncatula</em> or <em>Lotus japonicus</em> counterparts (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Endre <em>et al.</em> 2002</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Krusell <em>et al.</em> 2002</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Limpens <em>et al.</em> 2003</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lévy <em>et al.</em> 2004</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Stracke <em>et al.</em> 2004</a>). In order to systematically take advantage of genomic data available in closely related species, a good knowledge of the conservation of genome organization between model and crop species should be gained.</p><p id="p-8">The legume family is wide and incredibly diverse (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Doyle and Luckow 2003</a>). It includes numerous species that have been part of human diets since the dawn of agriculture and represents a valuable source of dietary proteins, as well as fibers and micronutrients (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Grusak 2002</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Guillon and Champ 2002</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Mitchell <em>et al.</em> 2009</a>). Pea is an important plant protein source in temperate regions of the world (10.4 M tons dry peas produced worldwide in 2009, <a href="http://faostat.fao.org/">http://faostat.fao.org/</a>). As for other protein-rich legume crop species, rapid genetic improvement of this species is needed to meet the increasing demand for protein food sources in the world. In the Leguminosae family, total or euchromatic genome sequences are now available for three species. All three species belong to the Papilionoideae subfamily. <em>Medicago truncatula</em> is taxonomically the closest model species to cool-season legume crops such as pea, lentil, faba bean, and chickpea. They belong to the Inverted Repeat Loss Clade of the Hologalegina clade of the Papilionoidae subfamily (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Doyle and Luckow 2003</a>). The other legume model species, <em>Lotus japonicus</em>, belongs to the closely related robinioid clade of the Hologalegina clade. The third species is soybean (<em>Glycine max</em>), an economically important crop belonging to the more distant milletioid clade of the Papilionoidae subfamily. Extensive genomics resources are available for the three species and the sequencing of the euchromatic regions of the <em>M. truncatula</em> and <em>L. japonicus</em> genomes is close to completion (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Cannon <em>et al.</em> 2006</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Sato <em>et al.</em> 2007</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Schmutz <em>et al.</em> 2010</a>). Intra- and intergenome comparisons identified syntenic blocks between <em>M. truncatula</em> and <em>L. japonicus</em> genomes and revealed that the three legume genomes hold traces of an ancient whole genome duplication (WGD) that occurred ca. ∼59 Mya, probably after the separation of the Fabaceae from Eurosid I group including the Salicaceae (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Doyle and Luckow 2003</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Young <em>et al.</em> 2005</a>; <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Cannon <em>et al.</em> 2006</a>), while a more recent WGD occurred in the soybean lineage ∼13 Mya (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Schmutz <em>et al.</em> 2010</a>). Existing pea linkage maps allowed a draft evaluation of macrosynteny between pea and <em>M. truncatula</em>, <em>L. japonicus</em>, <em>G. max</em>, and other legume linkage maps (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aubert <em>et al.</em> 2006</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kaló <em>et al.</em> 2004</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Choi <em>et al.</em> 2004</a>) suggesting a significant colinearity among the genomes of these species.</p><p id="p-9">In this paper, we established a new pea functional consensus map using recently developed SNP markers. This allowed us to assess in more detail the macrosynteny between pea and the three sequenced legume species by systematically searching for the best reciprocal homologs of pea Unigenes in <em>M. truncatula</em>, <em>L. japonicus</em> and <em>G. max</em> genome databases. We also used the pea genome sequence information to integrate it into previous paleo-genomics analysis in order to unravel the pea evolutionary paleo-history leading to its actual seven-chromosome structure. We used an original and robust method devoted to the identification of orthologous regions between plant genomes as well as for the detection of duplications within genomes based on integrative sequence alignment criteria combined with a statistical validation (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Salse <em>et al.</em> 2009a</a>). This method identified seven paleo-duplications in Monocots and Eudicots and allowed proposing a common ancestor with five and seven chromosomes for the Monocots and Eudicots, respectively (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Salse <em>et al.</em> 2009b</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abrouk <em>et al.</em> 2010</a>). Finally, taking advantage of the high level of synteny between pea and <em>M. truncatula</em>, we established a pea–<em>M. truncatula</em> translational toolkit allowing for any gene sequence to search its putative position on the pea consensus map, and from a position on the pea consensus map, to search for candidate genes among the neighboring placed Unigenes.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Development of the new pea functional consensus map</h3><div id="sec-3" class="subsection"><h4>Plant material:</h4><p id="p-10">In order to increase the number of loci mapped and get more precision about their localization in our consensus map, we used six different recombinant inbred line (RIL) populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000349/-/DC1/000349SI.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000349/-/DC1/TableS1.pdf">Table S1</a>). We used data obtained for three RIL populations previously described: Térèse × K586 (Pop1, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Laucou <em>et al.</em> 1998</a>), Térèse × Champagne (Pop2, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Loridon <em>et al.</em> 2005</a>) and Caméor× China (Pop9, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Deulvot <em>et al.</em> 2010</a>). Furthermore, Cameor, VavD265, Ballet, three pea (<em>Pisum sativum</em> L.) lines showing variability in protein content and seed protein composition, were selected to create interconnected RIL populations by single seed descent from the crosses between Cameor × VavD265 (Pop3, 211 F6:8), Cameor × Ballet (Pop4, 207 F6:8), Ballet × VavD265 (Pop5, 211 F6:8). Leaf tissues were harvested from the F6 plant and then from a bulk of eight F7 or F8 plants for further DNA extractions. Total DNA was extracted from leaf tissues according to <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dellaporta <em>et al.</em> (1983)</a>. A total of 1022 lines were used to build the consensus map.</p></div><div id="sec-4" class="subsection"><h4>Marker selection, development, and genotyping:</h4><p id="p-11">Microsatellite markers (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000349/-/DC1/TableS2.xls">Table S2</a>) were selected based on their polymorphism and map information (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Loridon <em>et al.</em> 2005</a>) to build the framework maps of the three new populations. SSR were genotyped as described in <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Loridon <em>et al.</em> (2005)</a>. Moreover, 75 genes already mapped in Pop1, Pop2, Pop9 were genotyped in Pop3, Pop4, and/or Pop5 as gene-anchored bridge markers (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Aubert <em>et al.</em> 2006</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Deulvot <em>et al.</em> 2010</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000349/-/DC1/TableS2.xls">Table S2</a>) and 34 new gene markers were developed for Pop3, Pop4 or Pop5 and presented in this paper. Additionally, 7 and 14 new gene markers scored in Pop1 (C. Rameau, personal communication) and in Pop2 (I. Lejeune-Hénaut, personal communication) were added on the consensus map. PCR amplifications of the selected genes in parents of mapping populations were carried out as described in <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Aubert <em>et al.</em> (2006)</a>. PCR products were directly sequenced by Millegen (Labège, France) and polymorphisms were searched. Different types of markers were then designed according to the type of polymorphism identified (INDELs, SNPs or null alleles). Genotyping conditions are summarized in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000349/-/DC1/TableS3.xls">Table S3</a>. Scoring data are available at <a href="http://www.thelegumeportal.net/pea_genotype_scores">http://www.thelegumeportal.net/pea_genotype_scores</a>.</p></div><div id="sec-5" class="subsection"><h4>Map construction:</h4><p id="p-12">A map was built for each RIL population as described in <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Aubert <em>et al.</em> (2006)</a>. Then, the consensus map was developed for the six populations (Pop1, 2, 3, 4, 5, 9). Population data were merged and the consensus map was built using CarthaGene (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">De Givry <em>et al.</em> 2005</a>). The inherent difficulty of defining groups for such a large dataset was overcome by analyzing groups obtained with increasing LOD scores, from 15 to 30, using a maximum distance criterion of 30 cM. Then, markers were added to the consensus map on the basis of their relative position to common markers shared by at least two RIL populations. The map order was refined using the annealing and flips procedures of CarthaGene. The best map obtained, (<em>i.e.</em>, the map presenting the maximum log-likelihood and minimum length) was finally drawn using MapChart version 2.1 software (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Voorrips 2002</a>).</p></div></div><div id="sec-6" class="subsection"><h3>Search for syntenic relationships between pea and <em>M. truncatula</em>, <em>L. japonicus</em>, soybean, and poplar genomes</h3><p id="p-13">In order to enhance the comparative mapping and assess the conservation of synteny between the <em>P. sativum</em> and <em>M. truncatula</em>, <em>L. japonicus</em>, soybean, and poplar genomes, we searched for the best reciprocal homologs of all pea gene sequences available in predicted gene sequences of the genomes of the four species.</p><div id="sec-7" class="subsection"><h4>Developing the pea Unigene set:</h4><p id="p-14">30156 pea sequences were retrieved from public databases on January 27<sup>th</sup> 2010: 2227 pea CDS from public EMBLCDS database (<a href="http://www.ebi.ac.uk/embl/-">http://www.ebi.ac.uk/embl/-</a> <a href="ftp://ftp.ebi.ac.uk/pub/databases/embl/cds">ftp://ftp.ebi.ac.uk/pub/databases/embl/cds</a>), 18552 pea EST from GenBank (dbEST), and 9377 pea EST from IPK Gatersleben (<a href="http://www.ipk-gatersleben.de/">www.ipk-gatersleben.de/</a>). Repetitive elements were masked with Repeat Masker (Smit AFA, Hubley R &amp; Green P. /RepeatMasker Open-3.0/. 1996-2010 [<a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>]) with the parameter “-species = fabaceae” to mask both common (176) and fabaceae specific (291) repetitive elements. Low complexity regions like poly-A tails or other low quality ends were trimmed with SeqClean (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Pertea <em>et al.</em> 2003</a>). Unexpected vector sequences were removed with Seqclean using NCBI's Univec database. All the sequences under 100 bp were removed. Sequence cleaning and contig clustering were processed with EST2Uni package (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Forment <em>et al.</em> 2008</a>). Clustering has been done with the TIGR Gene Indices clustering tools (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Pertea <em>et al.</em> 2003</a>) software (TGICL) using CAP3 assembly program (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Huang and Madan, 1999</a>) with a minimum percent identity for overlaps of 95% for both programs (parameters: 'O “p95” -p 95 -n 16000'; default values: -O “p93” -p 94 -v 30 -n 1000). The annotation of the resulting Unigenes (contigs and singletons) was made based on homology to known sequences using BLASTN or tBLASTn. The e-value threshold was 1e<sup>−15</sup> with a minimum identity percentage of 70%. The Unigenes were blasted against two databases in the following order: 1) the UniprotKB/Swissprot database (v57.9), 2) the predicted genes from release 3 (IMGA-gene-v3) of the Medicago Genome Sequence Consortium (MGSC) (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Cannon <em>et al.</em> 2006</a>), 3) and the <em>M. truncatula</em> Gene Index release 9 (MtGI9) from TIGR. Annotation was characterized as “very similar” if the e-value was below 1e<sup>-20</sup> and “highly similar” below 1e<sup>−50</sup>.</p></div><div id="sec-8" class="subsection"><h4>Searching for the best reciprocal homologs of the pea sequences in <em>M. truncatula</em>, <em>L. japonicus</em>, soybean, and poplar genomes:</h4><p id="p-15">The search for putative orthologs of pea genes in <em>M. truncatula</em> was carried out by performing reciprocal BLASTs between the Unigenes and the <em>M. truncatula</em> sequence database containing the sequences of IMGA-gene-v3, MtGI9, and those of the BACs release 3 from MGSC. The threshold e-value was 1e<sup>−20</sup> for both BLASTN. If for a given Unigene the best hit was a Medicago gene predicted sequence from IMGA-gene-v3, or an EST contig from MtGI9, a reciprocal BLASTN was performed on the pea database. If the best hit was a BAC sequence, a reciprocal BLASTN was done between a sequence containing the matching zone plus 2 kb at both 5′ and the 3′ side of the matched sequence. The best reciprocal homologs were searched using a perl script (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000349/-/DC1/FileS1.pdf">File S1</a>). If the best hit was the original sequence of the pea Unigene, the Medicago gene sequence was considered as the best reciprocal homolog of that pea sequence. Then, the positions of the best reciprocal homologs of pea Unigenes were searched using <em>M. truncatula</em> gene position data from the genome assembly v3 of the MGSC (<a href="http://www.medicago.org/">http://www.medicago.org/</a>). If the best reciprocal homolog was a BAC sequence or a predicted gene from IMGA-gene-v3, the position was given by its position on the pseudo-chromosome. If the best hit was a TC from TIGR, the position was obtained, if possible, by finding the best homolog among the predicted genes from IMGA-gene-v3 and retrieving the predicted gene position on the pseudo-chromosome. The search for putative orthologs of pea genes in <em>L. japonicus</em>, soybean (<em>G. max</em>), and poplar (<em>P. trichocarpa</em>) was achieved by reciprocal BLASTs as described for <em>M. truncatula</em>, except that only publicly available predicted gene sequences for these species were used. For <em>L. japonicus</em>, we used coding sequences from the Kazusa DNA Research Institute (<a href="http://www.kazusa.or.jp/lotus">http://www.kazusa.or.jp/lotus</a>). For <em>G. max</em>, high confidence protein-coding sequences from the soybean genome were used (<a href="http://www.phytozome.net/soybean">http://www.phytozome.net/soybean</a>). For <em>P. trichocarpa</em>, the coding sequences were retrieved from the sequences of the <em>P. trichocarpa</em> sequencing project (<a href="http://www.phytozome.net/poplar">http://www.phytozome.net/poplar</a>). Finally, we compared the position of best reciprocal homologs on the pea consensus map on the one hand, and on the <em>M. truncatula</em>, <em>L. japonicus</em>, soybean and poplar pseudo-chromosomes on the other hand, using dot-plots and comparative maps. The dot-plots were built using Excel.</p></div><div id="sec-9" class="subsection"><h4>Pea genome evolution survey:</h4><p id="p-16">The grape, <em>Arabidopsis</em>, <em>Medicago</em>, <em>Lotus</em>, soybean, poplar, and papaya sequence databases (<a href="http://www.phytozome.org/">http://www.phytozome.org/</a>, <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Abrouk <em>et al.</em> 2010</a>) were used to perform a synteny analysis using three parameters recently defined by <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Salse <em>et al.</em> (2009a)</a>. These parameters increase the stringency and significance of BLAST sequence alignment by parsing BLASTX results and rebuilding HSPs (High Scoring Pairs) or pairwise sequence alignments to identify accurate paralogous and orthologous relationships. This analysis allowed searching for traces of Eudicot paleo-duplication in the pea genome.</p></div><div id="sec-10" class="subsection"><h4>The Pea_Medicago_translational_toolkit:</h4><p id="p-17">The “Pea_Medicago_translational_toolkit” is a cgi/perl webpage hosted on an Apache webserver (<a href="http://www.thelegumeportal.net/pea_mtr_translational_toolkit">http://www.thelegumeportal.net/pea_mtr_translational_toolkit</a>). This toolkit allows two actions: 1) searching for the putative position of a gene on the pea consensus map and 2) searching for putative candidate genes in the neighborhood of a pea genetic locus. When searching for putative gene position, the user inputs a pea sequence in the dedicated box, this sequence is BLASTed against the pea Unigene database at 1e<sup>−20</sup> threshold, and the positions of the five best hits, if any, are displayed. These positions are inferred from the positions of <em>Medicago</em> best homologs of all pea Unigenes on <em>M. truncatula</em> pseudo-chromosomes relatively to <em>Medicago</em> best reciprocal homologs of the pea marker genes. When searching positional candidate genes at a locus, the user inputs either one or two pea markers, or a position in cM on the pea consensus map. The output is a list of Unigenes putatively located near the markers or the position entered by the user.</p></div></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>The new <em>Pisum sativum L. consensus</em> map</h3><p id="p-18">The map presented in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> results from a consensus mapping procedure for six different recombinant inbred line (RIL) populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000349/-/DC1/TableS1.pdf">Table S1</a>). Four new mapping populations were added to the ones used for the previous pea functional map (<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Aubert <em>et al.</em> 2006</a>). Microsatellite and gene markers were mapped in populations Pop3, Pop4, and Pop5, and only gene markers were mapped in Pop9 (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Deulvot <em>et al.</em> 2010</a>). Three genomic regions displayed marker segregation distortion (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000349/-/DC1/TableS4.xls">Table S4</a>): between AB33 and <em>Gibbi</em> on the top of LGII in Pop3 and Pop9, near <em>Le</em> at the bottom of LGIII in Pop5, and between AC10_1 and AD60 on LGVI in Pop3. The small distorted area found in Pop5 next to <em>Le</em> is probably due to the effect of this developmental gene driving seed number per plant (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Burstin <em>et al.</em> 2007</a>). As described in <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Loridon <em>et al.</em> (2005)</a> and <a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Aubert <em>et al.</em> (2006)</a>, marker ranks and positions were generally consistent across populations, except local rearrangements (at the top of LGI in Pop5, C20b, Agps2, AA228 and Nin are reversed; in the middle of LGII in Pop4 and of LGIII in Pop3, very close markers are reversed).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/2/93/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The new Pisum sativum L. consensus functional map. Gene markers are in bold on the right of linkage groups. Known mutations and protein or gene markers placed according to previous maps (Laucou et al. 1998; Hall et al. 1997; Weeden et al. 1998; Ellis and Poyser 2002; Wang et al. 2008; Sinjushin et al. 2006; Mishra et al. 2009; Li et al. 2010) are in bold on the left of linkage groups. Known mutations are in blue type. *Mendel&#x2019;s genes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-495845934" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The new &lt;em&gt;Pisum sativum&lt;/em&gt; L. consensus functional map. Gene markers are in bold on the right of linkage groups. Known mutations and protein or gene markers placed according to previous maps (Laucou &lt;em&gt;et al.&lt;/em&gt; 1998; Hall &lt;em&gt;et al.&lt;/em&gt; 1997; Weeden &lt;em&gt;et al.&lt;/em&gt; 1998; Ellis and Poyser 2002; Wang &lt;em&gt;et al.&lt;/em&gt; 2008; Sinjushin &lt;em&gt;et al&lt;/em&gt;. 2006; Mishra &lt;em&gt;et al.&lt;/em&gt; 2009; Li &lt;em&gt;et al.&lt;/em&gt; 2010) are in bold on the left of linkage groups. Known mutations are in blue type. &lt;sup&gt;*&lt;/sup&gt;Mendel&#x2019;s genes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/2/93/F1.medium.gif" width="440" height="330"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/1/2/93/F1.medium.gif" width="440" height="330"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/2/93/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/2/93/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460818" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-19" class="first-child">The new <em>Pisum sativum</em> L. consensus functional map. Gene markers are in bold on the right of linkage groups. Known mutations and protein or gene markers placed according to previous maps (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Laucou <em>et al.</em> 1998</a>; Hall <em>et al.</em> 1997; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Weeden <em>et al.</em> 1998</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Ellis and Poyser 2002</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Wang <em>et al.</em> 2008</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Sinjushin <em>et al</em>. 2006</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Mishra <em>et al.</em> 2009</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Li <em>et al.</em> 2010</a>) are in bold on the left of linkage groups. Known mutations are in blue type. <sup>*</sup>Mendel’s genes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">Then, the consensus map was built using bridge markers as framework markers (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000349/-/DC1/TableS2.xls">Table S2</a>). Out of the 536 markers placed on the consensus map, 56% were genotyped in at least two mapping populations, and between 16 and 154 markers were shared per pair of RIL populations (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The new consensus functional map covers 1389 cM with 97% of marker intervals below 10 cM. This map includes 3 morphological markers, 180 SSR markers, 133 RAPD, 6 RFLP, and 214 gene-based markers belonging to different functional classes (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000349/-/DC1/TableS3.xls">Table S3</a>). Fifty-one new genes were mapped in the present study and added to gene markers previously published in <a id="xref-ref-2-7" class="xref-bibr" href="#ref-2">Aubert <em>et al.</em> (2006)</a> and <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Deulvot <em>et al.</em> (2010)</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460667/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460667/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1 </span> <span class="caption-title">Number of markers shared per pair of RIL population</span><div class="sb-div caption-clear"></div></div></div><p id="p-21">This map has the advantage of incorporating markers used for different published maps (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Weeden <em>et al.</em> 1998</a>, <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Laucou <em>et al.</em> 1998</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Choi <em>et al.</em> 2004</a>, <a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">Loridon <em>et al.</em> 2005</a>; <a id="xref-ref-2-8" class="xref-bibr" href="#ref-2">Aubert <em>et al.</em> 2006</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jing <em>et al.</em> 2007</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Prioul-Gervais <em>et al.</em> 2007</a>). These links allowed us to tentatively place 48 known mutations and 15 protein or gene markers on the consensus map (Hall <em>et al.</em> 1997; <a id="xref-ref-64-3" class="xref-bibr" href="#ref-64">Weeden <em>et al.</em> 1998</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Rameau <em>et al.</em> 1998</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Ellis and Poyser 2002</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hecht <em>et al.</em> 2005</a>; <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Wang <em>et al.</em> 2008</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Mishra <em>et al.</em> 2009</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Sinjushin <em>et al</em>. 2006</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Katoch <em>et al.</em> 2009</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Li <em>et al.</em> 2010</a>) (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). A total of 70 known mutations were placed on the map (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>) including the 48 above-mentioned mutations and 22 mutations corresponding to mapped genes.</p></div><div id="sec-13" class="subsection"><h3>Definition of the pea Unigene set</h3><p id="p-22">From 30,156 pea sequences retrieved from public databases, 541 have been removed and 13,381 have been trimmed after cleaning and low complexity region removal, leading to 29,615 high quality sequences. Sequences ranged from 101 to 6789 bp and had a mean length of 623 bp. The mean expressed sequence tag (EST) length was 520 bp, and the mean gene coding sequence (CDS) length was 1188 bp. The clustering produced 13,747 Unigenes: 4792 contigs and 8955 singletons. Contigs had an average length of 807 bp and an average depth of 4.3 sequences: about half of the contigs (2397) were made of two sequences, and 14% (678) were made of more than five sequences. Singletons had an average length of 525 bp. Surprisingly, 4% of singletons (346) were CDS but had no corresponding EST. A total of 10,416 (75%) of the Unigenes were annotated: 2993 with Swissprot database, 852 with Uniprot database and the remainder with the composite <em>M. truncatula</em> database. As expected, the deepest contigs, including most of ESTs, were found for the 26S ribosomal RNA, provicilin and legumin A genes.</p></div><div id="sec-14" class="subsection"><h3>Identification of putative orthologs of pea Unigenes in <em>M. truncatula</em>, <em>L. japonicus</em>, soybean, and poplar sequenced genomes</h3><p id="p-23">In the search for putative <em>Medicago</em> orthologs, pea Unigenes were BLASTed on IMGA predicted genes, TIGR TC and BAC sequences (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Out of the 13,747 pea Unigene sequences, 11,166 (81%) had a unidirectional match under defined conditions and 8375 (61%) had a reciprocal best hit with the starting Unigene. Out of these 8375 best bidirectional homolog sequences, 5460 could be placed on the <em>M. truncatula</em> pseudo-chromosomes. Out of the 214 gene-based markers located on the new consensus map, 121 had a best reciprocal <em>M. truncatula</em> homolog and 19 had been defined directly from <em>M. truncatula</em> sequences. In total, 140 sequences corresponding to pea gene markers could be placed on the <em>M. truncatula</em> genome. This represents a substantial increase in the number of pea–<em>M. truncatula</em> bridges compared with <a id="xref-ref-2-9" class="xref-bibr" href="#ref-2">Aubert <em>et al.</em> (2006)</a> and <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Choi <em>et al.</em> (2004)</a>, who reported 45 and 57 links, respectively, and allowed a more precise assessment of synteny (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000349/-/DC1/FigureS1.pdf">Figure S1</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460666/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460666/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2 </span> <span class="caption-title">Summary of the BLAST search for putative <em>Medicago</em> orthologs performed for the 13,747 pea Unigene sequences</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/2/93/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dot-plots of syntenic relationships between the P. sativum linkage groups (LG) and M. truncatula (a), L. japonicus (b), G. max (c), and P. trichocarpa (d) pseudo-chromosomes. The best reciprocal homologs are placed on the dot-plot according to their position on the pea LG (Y-axis) and their position on the pseudo-chromosomes (x-axis). Synteny conservation is evidenced when homologs symbols are placed on diagonal lines. Rearrangements are circled. Syntenic blocks are highlighted according to pea LG (blue diamonds: gene mapped on PsI; yellow squares: PsII; green triangles: PsIII; pink squares: PsIV; red diamonds: PsV; purple circles: PsVI; pink triangles: LGVII); symbols are color-coded when at least three best reciprocal homologs are found between a pea LG and M. truncatula, L .japonicus, G. max, or P. trichocarpa pseudo-chromosomes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-495845934" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Dot-plots of syntenic relationships between the &lt;em&gt;P. sativum&lt;/em&gt; linkage groups (LG) and &lt;em&gt;M. truncatula&lt;/em&gt; (a), &lt;em&gt;L. japonicus&lt;/em&gt; (b), &lt;em&gt;G. max&lt;/em&gt; (c), and &lt;em&gt;P. trichocarpa&lt;/em&gt; (d) pseudo-chromosomes. The best reciprocal homologs are placed on the dot-plot according to their position on the pea LG (Y-axis) and their position on the pseudo-chromosomes (x-axis). Synteny conservation is evidenced when homologs symbols are placed on diagonal lines. Rearrangements are circled. Syntenic blocks are highlighted according to pea LG (blue diamonds: gene mapped on PsI; yellow squares: PsII; green triangles: PsIII; pink squares: PsIV; red diamonds: PsV; purple circles: PsVI; pink triangles: LGVII); symbols are color-coded when at least three best reciprocal homologs are found between a pea LG and &lt;em&gt;M. truncatula&lt;/em&gt;, &lt;em&gt;L .japonicus&lt;/em&gt;, &lt;em&gt;G. max&lt;/em&gt;, or &lt;em&gt;P. trichocarpa&lt;/em&gt; pseudo-chromosomes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/2/93/F2.medium.gif" width="154" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/1/2/93/F2.medium.gif" width="154" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/2/93/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/2/93/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460687" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-25" class="first-child">Dot-plots of syntenic relationships between the <em>P. sativum</em> linkage groups (LG) and <em>M. truncatula</em> (a), <em>L. japonicus</em> (b), <em>G. max</em> (c), and <em>P. trichocarpa</em> (d) pseudo-chromosomes. The best reciprocal homologs are placed on the dot-plot according to their position on the pea LG (Y-axis) and their position on the pseudo-chromosomes (x-axis). Synteny conservation is evidenced when homologs symbols are placed on diagonal lines. Rearrangements are circled. Syntenic blocks are highlighted according to pea LG (blue diamonds: gene mapped on PsI; yellow squares: PsII; green triangles: PsIII; pink squares: PsIV; red diamonds: PsV; purple circles: PsVI; pink triangles: LGVII); symbols are color-coded when at least three best reciprocal homologs are found between a pea LG and <em>M. truncatula</em>, <em>L .japonicus</em>, <em>G. max</em>, or <em>P. trichocarpa</em> pseudo-chromosomes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">A simpler method was used for searching the best reciprocal homolog in <em>L. japonicus</em>, soybean and poplar, where only predicted gene sequences were taken into account. Best reciprocal homologs were identified to pea Unigenes, and 76, 151, and 98 were placed on the <em>L. japonicus</em>, soybean, and poplar genomes, respectively (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460681/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460681/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3 </span> <span class="caption-title">Summary of reciprocal BLAST between pea Unigenes and <em>M. truncatula</em>, <em>L. japonicus</em>, soybean, and poplar predicted gene sequences</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Marker synteny between the <em>Pisum sativum</em> and <em>M. truncatula</em>, <em>L. japonicus</em>, soybean and poplar genomes</h3><p id="p-32">Dotplots (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>) and the summary view of the chromosome relationships among the five species (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>) illustrate the varying levels of conserved macrosynteny between pea and the four species analyzed. For all pea linkage groups, clear blocks of synteny linked pea and <em>M. truncatula</em> genomes with varying levels of rearrangements among the syntenic blocks. Pea linkage group I (PsI) is related to <em>M. truncatula</em> chromosome 5 (Mt5), with a small inversion of gene order in the middle, like PsIV with Mt8. PsII is related to Mt1 with an inversion at the top of PsII, and PsIII is related both to Mt2 and Mt3 with minor rearrangement along the synteny block, like PsVII with Mt4. To a lesser extent, the conservation of synteny is also striking with <em>L. japonicus</em> and soybean. For all pea linkage groups except PsVI, blocks of synteny could be identified with <em>L. japonicus</em>, with a few rearrangements (<em>e.g.</em>, the inversions in the middle of the group for PsI and PsII). As expected, synteny was more fragmented with soybean. However, clear blocks of synteny can be identified for all pea linkage groups, with from one to four syntenic blocks in soybean for each pea linkage group. Long segments of colinearity were conserved, as for example between PsV and Gm9 and 19, or PsVII with Gm11. In contrast, no clear pattern of synteny emerges from the dot-plot between pea and <em>P. trichocarpa</em> chromosomes. Comparative maps allow a closer look at the conservation of synteny blocks. <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> shows syntenic regions for PsIII and PsV. While for PsV, synteny encompasses the whole linkage group, for PsIII, intra- and interchromosome rearrangements increase complexity. The pea–<em>M. truncatula</em> comparative maps (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000349/-/DC1/FigureS1.pdf">Figure S1</a>) also inform about variations in genetic and genomic distances: for example, the longer Mt5 pseudo-molecule (42.8 Mb) corresponds to the smaller pea linkage group (PsI, 142.4 cM), suggesting a restriction of recombination for this pea chromosome.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/2/93/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary view of P. sativum, M. truncatula, L. japonicus, soybean (G. max), and poplar (P. trichocarpa) genomes, phylogenetic relationships, and molecular characteristics. Genomes are depicted through best reciprocal homogue genes, color-coded and numbered according to the position of the P. sativum homolog on linkage groups of the consensus functional map." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-495845934" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Summary view of &lt;em&gt;P. sativum&lt;/em&gt;, &lt;em&gt;M. truncatula&lt;/em&gt;, &lt;em&gt;L. japonicus&lt;/em&gt;, soybean (G. max), and poplar (P. trichocarpa) genomes, phylogenetic relationships, and molecular characteristics. Genomes are depicted through best reciprocal homogue genes, color-coded and numbered according to the position of the &lt;em&gt;P. sativum&lt;/em&gt; homolog on linkage groups of the consensus functional map.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/2/93/F3.medium.gif" width="440" height="349"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/1/2/93/F3.medium.gif" width="440" height="349"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/2/93/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/2/93/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460811" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-33" class="first-child">Summary view of <em>P. sativum</em>, <em>M. truncatula</em>, <em>L. japonicus</em>, soybean (G. max), and poplar (P. trichocarpa) genomes, phylogenetic relationships, and molecular characteristics. Genomes are depicted through best reciprocal homogue genes, color-coded and numbered according to the position of the <em>P. sativum</em> homolog on linkage groups of the consensus functional map.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/2/93/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparative maps between P. sativum and M. truncatula, L. japonicus, soybean for (A) LGII and (B) LGV." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-495845934" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparative maps between &lt;em&gt;P. sativum&lt;/em&gt; and &lt;em&gt;M. truncatula&lt;/em&gt;, &lt;em&gt;L. japonicus&lt;/em&gt;, soybean for (A) LGII and (B) LGV.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/2/93/F4.medium.gif" width="440" height="245"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/1/2/93/F4.medium.gif" width="440" height="245"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/2/93/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/2/93/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460831" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-34" class="first-child">Comparative maps between <em>P. sativum</em> and <em>M. truncatula</em>, <em>L. japonicus</em>, soybean for (A) LGII and (B) LGV.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Pea genome evolution</h3><p id="p-35">Using the alignment parameters and statistical tests described previously by <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Salse <em>et al.</em> (2009a</a>,<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">b</a>), we analyzed the syntenic relationships between pea, grape, <em>Arabidopsis</em>, <em>Medicago</em>, <em>Lotus</em>, soybean, poplar, and papaya genomes. Using grape as the reference genome, 149 robust orthologous relationships have been identified covering 37% of the pea map. Integration of the synteny relationships established independently between pea and the seven Eudicot sequenced genomes led to the precise characterization, in pea, of the seven paleo-triplications proposed recently as the basis of the definition of seven ancestral chromosomal groups in Eudicots (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Abrouk <em>et al.</em> 2010</a>). The putative origin of shared ancestral duplications found in pea, are shown on <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> using a seven color code indicating the seven common ancestor chromosomes of Eudicots. In pea, the seven ancestral shared triplications proposed recently in Eudicots (<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Abrouk <em>et al.</em> 2010</a>) are thus characterized for the first time in the present paper. Based on the ancestral (γ) as well as the lineage specific (α, β) whole genome duplications reported for the Eudicots, it becomes possible to propose an evolutionary scenario that has shaped the seven pea chromosomes from the seven chromosome Eudicot ancestor and more specifically from the 21 paleo-hexaploid intermediate (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). We suggest from the 21 chromosome-intermediate ancestor at least 25 major chromosome fusions (CF) to obtain the current seven chromosome structure (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/2/93/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pea genome paleo-history. The pea genome (bottom) is represented with a seven-color code to illuminate the evolution of segments from a common ancestor with seven chromosomes (A1&#x2013;A7, top). Colored lines indicate the evolution of segments from a seven-chromosome common ancestor of the Eudicots to reach the modern pea genome structure. The 25 chromosomal fusions (CF) are highlighted with colored arrows. At the bottom of the figure is shown the actual pea genome structure. The ancestral shared duplications can be compared with the seven ancestral paleo-triplications reported in grape (Vitis vinifera, Abrouk et al. 2010): V. vinifera chromosomes are indicated by Vv, P. sativum linkage groups by Ps. 1Vv1-Vv14-Vv17/ PsIII-PsIV-PsV-PsVI-PsVII (yellow), Vv2-Vv12-Vv15-Vv16/PsI-PsII-PsV-PsVII (blue), Vv3-Vv4-Vv7-Vv18/PsI-PsII-PsIII-PsVII (green), Vv4-Vv9-Vv11/PsII-PsIII-PsIV-PsVI (light blue), Vv5-Vv7-Vv14/PsII-PsV (pink), Vv6-Vv8-Vv13/PsII-PsV-PsVII (brown), Vv10-Vv12-Vv19/PsI-PsIII-PsIV-PsVI-PsVII (red)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-495845934" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Pea genome paleo-history. The pea genome (bottom) is represented with a seven-color code to illuminate the evolution of segments from a common ancestor with seven chromosomes (A1&#x2013;A7, top). Colored lines indicate the evolution of segments from a seven-chromosome common ancestor of the Eudicots to reach the modern pea genome structure. The 25 chromosomal fusions (CF) are highlighted with colored arrows. At the bottom of the figure is shown the actual pea genome structure. The ancestral shared duplications can be compared with the seven ancestral paleo-triplications reported in grape (&lt;em&gt;Vitis vinifera&lt;/em&gt;, Abrouk &lt;em&gt;et al.&lt;/em&gt; 2010): &lt;em&gt;V. vinifera&lt;/em&gt; chromosomes are indicated by Vv, &lt;em&gt;P. sativum&lt;/em&gt; linkage groups by Ps. 1Vv1-Vv14-Vv17/ PsIII-PsIV-PsV-PsVI-PsVII (yellow), Vv2-Vv12-Vv15-Vv16/PsI-PsII-PsV-PsVII (blue), Vv3-Vv4-Vv7-Vv18/PsI-PsII-PsIII-PsVII (green), Vv4-Vv9-Vv11/PsII-PsIII-PsIV-PsVI (light blue), Vv5-Vv7-Vv14/PsII-PsV (pink), Vv6-Vv8-Vv13/PsII-PsV-PsVII (brown), Vv10-Vv12-Vv19/PsI-PsIII-PsIV-PsVI-PsVII (red).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/2/93/F5.medium.gif" width="440" height="353"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/1/2/93/F5.medium.gif" width="440" height="353"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/2/93/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/2/93/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460672" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-36" class="first-child">Pea genome paleo-history. The pea genome (bottom) is represented with a seven-color code to illuminate the evolution of segments from a common ancestor with seven chromosomes (A1–A7, top). Colored lines indicate the evolution of segments from a seven-chromosome common ancestor of the Eudicots to reach the modern pea genome structure. The 25 chromosomal fusions (CF) are highlighted with colored arrows. At the bottom of the figure is shown the actual pea genome structure. The ancestral shared duplications can be compared with the seven ancestral paleo-triplications reported in grape (<em>Vitis vinifera</em>, <a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">Abrouk <em>et al.</em> 2010</a>): <em>V. vinifera</em> chromosomes are indicated by Vv, <em>P. sativum</em> linkage groups by Ps. 1Vv1-Vv14-Vv17/ PsIII-PsIV-PsV-PsVI-PsVII (yellow), Vv2-Vv12-Vv15-Vv16/PsI-PsII-PsV-PsVII (blue), Vv3-Vv4-Vv7-Vv18/PsI-PsII-PsIII-PsVII (green), Vv4-Vv9-Vv11/PsII-PsIII-PsIV-PsVI (light blue), Vv5-Vv7-Vv14/PsII-PsV (pink), Vv6-Vv8-Vv13/PsII-PsV-PsVII (brown), Vv10-Vv12-Vv19/PsI-PsIII-PsIV-PsVI-PsVII (red).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>A Pea-Medicago translational toolkit</h3><p id="p-37">Because the conservation of gene colinearity was generally good between pea and <em>M. truncatula</em>, we designed a Pea-Medicago translational toolkit based on their comparative map. If one wants to locate a gene sequence on the pea consensus map, one can enter the nucleic acid sequence of interest; this sequence will be BLASTed onto our Unigene database, andits position on the pea consensus map will be inferred from the position of this gene’s best homolog on the <em>M. truncatula</em> chromosomes. A reverse way to use this translational toolkit is to search the Unigene database for putative candidate genes in the neighborhood of a pea genetic locus of interest to the user. Of course, the validity of the predicted position will be more reliable in regions where pea–<em>M. truncatula</em> colinearity is good. It depends also on the type of gene investigated and the possibility to find the best homologs in our pea Unigene database. The translational toolkit provides a worked example. Other tools investigate synteny among legume species (Legume Information System at <a href="http://www.comparative-legumes.org/">http://www.comparative-legumes.org/</a> or the <em>Medicago truncatula</em> GBrowse at <a href="http://medtr.comparative-legumes.org/gb2/gbrowse/3.5.1/">http://medtr.comparative-legumes.org/gb2/gbrowse/3.5.1/</a>). Our tool has the advantage to allow for any sequence of interest, the search of its putative position on the pea map.</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-38"><em>Pisum sativum</em> has been a case study for geneticists since Mendel’s pioneering studies. Since then, significant efforts have been put into the development of genetic maps for this species (<a id="xref-ref-64-4" class="xref-bibr" href="#ref-64">Weeden <em>et al.</em> 1998</a>; <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Ellis and Poyser 2002</a>; <a id="xref-ref-2-10" class="xref-bibr" href="#ref-2">Aubert <em>et al.</em> 2006</a>).</p><div id="sec-19" class="subsection"><h3>A tool for translational genomics</h3><p id="p-39">Many mutations have been described in pea. In 1972, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Blixt (1972)</a> published a mutation map including 169 morphological markers. Some gene markers of the present consensus map correspond to known mutations <em>e.g.</em>, the <em>A</em>, <em>I</em>, <em>Le</em> and <em>R</em> genes studied by Mendel. Other previously published mutations were placed on the functional map (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>) by using bridge markers. The underlying genes responsible for many mutant phenotypes are still unknown. Our translational toolkit may help researchers 1) to find candidate genes for traits of interest, whether in pea or in a related species for which syntenic regions are identified, or 2) to define well-located gene markers in the vicinity of the gene or QTL they want to identify. For example, a root-expressed E3 ubiquitin ligase gene (PUB1) was recently shown to negatively regulate nodulation in <em>Medicago truncatula</em> (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Mbengue <em>et al.</em> 2010</a>). The translational toolkit enabled us to place <em>Pub1</em>’s closest pea homolog on the top of pea LGI, in the region of a hypernodulation mutant, <em>Nod3</em> (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gualtieri <em>et al.</em> 2002</a>). This makes the pea ortholog of <em>Pub1</em> a very good candidate for <em>Nod3</em>.</p></div><div id="sec-20" class="subsection"><h3>New features of the <em>P. sativum</em> consensus map</h3><p id="p-40">In this paper, we present the new consensus linkage map of <em>P. sativum</em> including 214 functional markers. Diverse functional classes are represented: development, carbohydrate metabolism, amino acid metabolism, transport, transcriptional regulation (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000349/-/DC1/TableS2.xls">Table S2</a>). Because transcription factors (TF) are important key regulators of gene expression in eukaryotes and particularly seed storage protein gene expression (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Meinke <em>et al.</em> 1981</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gatehouse <em>et al.</em> 1982</a>), we mapped putative TF genes selected by homology to <em>M. truncatula</em>, <em>Arabidopsis thaliana</em>, or maize TF sequences. An OPAQUE2-LIKE gene was located on <em>P. sativum</em> LGII (PsII). In cereals, OPAQUE2 gene encodes a basic leucine zipper (bZIP) transcription factor that binds to a promoter element in the 22-kD zein genes to activate their expression (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Schmidt <em>et al.</em> 1990</a>, <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">1992</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Ueda <em>et al.</em> 1992</a>). Coincidentally, a legumin gene cluster was mapped in this region (Hall <em>et al.</em> 1997) as well as <em>Vpe</em>, a gene encoding for a protein involved in the maturation of storage proteins, <em>RNApol2</em> which encodes for an RNA polymerase, and <em>PsDof7</em> another transcription factor. ABSCISIC ACID-INSENSITIVE3 (ABI3) is also a seed-specific TF that has key regulatory functions during seed development, particularly in the expression of seed storage protein genes (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Parcy <em>et al.</em> 1997</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kroj <em>et al.</em> 2003</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Mönke <em>et al.</em> 2004</a>) and acts in synergy with other bZIP factors (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lara <em>et al.</em> 2003</a>). <em>Abi3</em> was mapped on LGV near four other TF genes: <em>ColA</em>, <em>ArfB3</em>, <em>Aux1</em>, and <em>Tir1</em>, and close to a previously mapped vicilin gene cluster (Hall <em>et al.</em> 1997). The genetic clustering of potentially interacting genes could favor their optimal interaction. Altogether, the functional markers located on this map might be good candidates for seed traits in pea and should be tested for their colocation with seed productivity and quality trait QTL. As in our previous work (<a id="xref-ref-34-6" class="xref-bibr" href="#ref-34">Loridon <em>et al.</em> 2005</a>; <a id="xref-ref-2-11" class="xref-bibr" href="#ref-2">Aubert <em>et al.</em> 2006</a>), we privileged easily transferable markers to allow their wide use both by academics and breeders.</p></div><div id="sec-21" class="subsection"><h3>Synteny among <em>P. sativum</em>, <em>M. truncatula</em>, <em>L. japonicus</em>, and <em>P. trichocarpa</em></h3><p id="p-41">In legumes, early studies reported colinearity among cool-season legume linkage maps (pea and lentil: <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Weeden <em>et al.</em> 1992</a>, pea and chickpea: Simon and Muehibauer 1997). Since then, other studies have reported a high level of macrosynteny between the pea and <em>Medicago</em> genomes (<a id="xref-ref-2-12" class="xref-bibr" href="#ref-2">Aubert <em>et al.</em> 2006</a>; <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Choi <em>et al.</em> 2004</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Kaló <em>et al.</em> 2004</a>) as well as among several legume species (<a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Choi <em>et al.</em> 2004</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Zhu <em>et al.</em> 2006</a>). With the advent of whole genome sequences, syntenic relationships can be revisited in a more detailed way in order to identify candidate genes in sequenced species for traits in crop species. In the present study, pea was put at the center of a search for syntenic relationship with four other species, from the closely related <em>M. truncatula</em> to the more distant <em>P. trichocarpa</em>. As summarized in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>, the five species not only differ in their level of phylogenetic relatedness, but also in their genome size and chromosome numbers. The genome size difference is 10-fold from <em>L. japonicus</em> and <em>M. truncatula</em> small genomes (ca. 500 Mb, <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Sato <em>et al.</em> 2010</a>), through soybean (ca. 1100 Mb, <a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Schmutz <em>et al.</em> 2010</a>), to pea (ca. 5000 Mb, <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Ellis and Poyser 2002</a>). While soybean genome has undergone a recent whole genome duplication (WGD) event (ca. 13 Mya, <a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">Schmutz <em>et al.</em> 2010</a>), the pea genome is probably the richest in retro-transposons (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Ellis and Poyser 2002</a>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Macas <em>et al.</em> 2007</a>). Despite these marked differences, a striking conservation of gene synteny and gene order was shown in this study.</p><p id="p-42">Identifying and aligning the best-in-genome homologous sequences provided a clearer picture of syntenic relationships. The highest rate of best reciprocal homolog identification for the 13,747 pea Unigenes was found for soybean (6626 putative orthologs identified, <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>) for which close to 100% of the genome sequence is available. The second highest score was for <em>M. truncatula</em>, the closest species to pea but for which only ∼60% of the genome is covered by available sequences. The estimated number of pea Unigenes for which a putative ortholog could be expected to be found in the four other species, if 100% of the genome sequence was available (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>), was closely related to the phylogenetic relatedness among species, with the highest expected rate of ortholog discovery for <em>M. truncatula</em> (71%), then <em>L. japonicus</em> (60%), soybean (49%), and finally <em>P. trichocarpa</em> (22%). Similarly, decreasing levels of colinearity were correlated with higher phylogenetic distances (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). Despite the recent WGD having occurred in the soybean lineage, clear traces of synteny and coancestry were seen between the pea and soybean genomes. And although overall synteny is blurred with P. <em>trichocarpa</em>, some interesting relics merit further inspection.</p><p id="p-43">A few markers broke the synteny (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figures 2</a> and <a id="xref-fig-3-4" class="xref-fig" href="#F3">3</a>). It may be that the best reciprocal homolog found in the genome of the sequenced species is not the ortholog, as the <em>Medicago</em> and <em>Lotus</em> genome sequencing are still in progress. Other possible sources of synteny breakage are gene translocation, for example following a transposition event, or ortholog gene loss making paralogs appear as best reciprocal homologs. Finally, regarding marker order rearrangements, it should be noted that the consensus map is the best statistical model obtained, but errors in marker ordering can be generated following the merge of data from different populations due to 1) translocation events as commonly reported in different pea stocks (<a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Ellis and Poyser 2002</a>) and 2) missing data for markers genotyped in some populations and not others, depending on their polymorphism.</p><p id="p-44">Finally, this pea functional consensus map also provides a platform tool for converting markers from pea to allied species for which few functional markers are available. Through the use of <em>M. truncatula</em> as bridge species, our data relates to recent publications on lentil (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Phan <em>et al.</em> 2006</a>), white clover (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">George <em>et al.</em> 2008</a>), <em>Phaseolus vulgaris</em> (McClean <em>et al.</em> 2010; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Galeano <em>et al.</em> 2009</a>; Hougaard <em>et al.</em> 2008), and chickpea (Nayak <em>et al.</em> 2010).</p></div><div id="sec-22" class="subsection"><h3>Pea genome evolution</h3><p id="p-45">By increasing the number of mapped genes on the pea functional map, this study also significantly increased the number of bridge markers to the sequenced genomes. This allowed to put forward some hypotheses as regards to the ancient Papillionoid or Eurosid I chromosomes. The structure of the proto-papillionoid chromosome seems generally to be retained, with a few deviations. The most puzzling case involves PsIII and PsVI. When comparing pea and <em>M. truncatula</em>, it appears that PsIII is split in two in <em>M. truncatula</em>: orthologs of genes located on PsIII (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>) and PsVI are found in Mt2. In <em>L. japonicus</em>, some PsIII gene orthologs are associated with PsVI gene orthologs to form part of Lj3. In <em>G. max</em>, some PsIII gene orthologs are associated with PsVI gene orthologs in Gm4, 6, 12, and 15. Finally, even in <em>P. trichocarpa</em>, some gene orthologs of PsIII and PsVI are merged for example in Pt15. Thus, it appears that PsIII and PsVI could have been part of a same chromosome before the radiation between the Leguminosae and the other Eurosid I including the Salicaceae. Two duplicated genes give another view of PsIII and PsVI evolution: while <em>PsAAP1</em> (#71 in <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>) and <em>PsAAP2</em> (#69) are located in neighboring positions on PsIII probably following a tandem duplication (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Tegeder <em>et al.</em> 2000</a>), the <em>PGK1</em> (#92) and <em>PGK2</em> (#182) genes (<a id="xref-ref-2-13" class="xref-bibr" href="#ref-2">Aubert <em>et al.</em> 2006</a>) are mapped on PsIII and PsVI.</p><p id="p-46">In order to further unravel the pea genome paleo-history, we identified and characterized shared paleo-duplications based on the integration of orthologous relationships identified between the 7 pea chromosomes and the grape, <em>Arabidopsis</em>, <em>Medicago</em>, <em>Lotus</em>, soybean, poplar and papaya genomes. We proposed an evolutionary scenario that may have shaped the seven pea chromosomes from a seven chromosome paleo-hexaploid Eudicot ancestor (<a id="xref-ref-1-6" class="xref-bibr" href="#ref-1">Abrouk <em>et al.</em> 2010</a>). Under this hypothesis, the pea genome underwent at least one triplication event and 25 chromosome fusions (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>). Interestingly, the two shorter linkage groups correspond to chromosomes having experienced few chromosomal fusion events (three for PsI and two for PsVI) during evolution, while longer linkage groups correspond to chromosomes having experienced more fusions (at least six for PsIII and four for PsV), suggesting a link between genome evolution and recombination distribution. This approach also allowed defining potential paralogous regions in the genome. For example, PsIV and PsVI are likely to share paralogous genes resulting from ancestral triplication in the eudicot ancestor. Finally, compared with the other Eudicots analyzed, pea appears to have evolved a simple genome structure, intermediate between the grape (with an ancestral-type modern genome structure) and soybean (with an intensively rearranged modern genome structure). This approach may be of general interest to identify regions potentially carrying ancient paralog genes.</p><p id="p-47">In conclusion, the new consensus functional map of pea presented herein represents a significant step toward a better understanding of legume evolution of genomes and traits. Legumes are fascinating plants, establishing vital symbiosis with soil microflora, developing highly nutritive seeds, and showing an astonishing diversity of forms. Among these species, pea has a special historical interest and its genome seems to have retained a large part of the ancestral papillionoideae chromosome structure. It is thus urgent to develop more genomic resources for this species, an effort that is currently underway in different ongoing projects throughout the world.</p></div></div><div class="section" id="sec-23"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-48">We thank P. Dubreuil for the development of SSR markers, K. Gallardo and R. Thompson for helpful discussion and comments on the manuscript, C. Rameau and I. Lejeune-Hénaut for sharing unpublished results, and an anonymous reviewer for improving the manuscript. We are grateful to the MGSC for making available the v3 of Mt genome assembly.This work was supported by the French national programs Genoplante GOP-PEAC2. M. N. was appointed on grants from GOP-PEAC2. RIL populations were produced in the framework of GOP-PEAC2. This work was also supported by the FP6 EU project “Grain Legumes Integrated Project” (FOOD-CT-2004-506223). A. B. and M. B. were appointed on grants from GLIP.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-11"><p id="p-49"><em>Communicating editor: Brenda J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-01-07"><span class="received-label">Received </span>January 7, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-05-06"><span class="accepted-label">Accepted </span>May 6, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2011 Bordat <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-5">This is an open access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.1" data-doi="10.1016/j.tplants.2010.06.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abrouk</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murat</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pont</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messing</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Palaeogenomics of plants: synteny-based modelling of extinct ancestors.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">487</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DAbrouk%26rft.auinit1%253DM.%26rft.volume%253D15%26rft.issue%253D9%26rft.spage%253D479%26rft.epage%253D487%26rft.atitle%253DPalaeogenomics%2Bof%2Bplants%253A%2Bsynteny-based%2Bmodelling%2Bof%2Bextinct%2Bancestors.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2010.06.001%26rft_id%253Dinfo%253Apmid%252F20638891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2010.06.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20638891&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282403000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.2" data-doi="10.1007/s00122-005-0205-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aubert</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacquin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loridon</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quillet</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Functional mapping in pea, as an aid to the candidate gene selection and for investigating synteny with the model legume <em>Medicago truncatula</em>.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">1024</span>–<span class="cit-lpage">1041</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DAubert%26rft.auinit1%253DG.%26rft.volume%253D112%26rft.issue%253D6%26rft.spage%253D1024%26rft.epage%253D1041%26rft.atitle%253DFunctional%2Bmapping%2Bin%2Bpea%252C%2Bas%2Ban%2Baid%2Bto%2Bthe%2Bcandidate%2Bgene%2Bselection%2Band%2Bfor%2Binvestigating%2Bsynteny%2Bwith%2Bthe%2Bmodel%2Blegume%2BMedicago%2Btruncatula.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-0205-y%26rft_id%253Dinfo%253Apmid%252F16416153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-0205-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16416153&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236486600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blixt</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Mutation genetics in <em>Pisum</em>.</span> <abbr class="cit-jnl-abbrev">Agri. Hort. Genet.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgri.%2BHort.%2BGenet.%26rft.volume%253D30%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.4" data-doi="10.1104/pp.107.096966"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burstin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marget</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huart</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moessner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mangin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Developmental genes have pleiotropic effects on plant morphology and source capacity, eventually impacting on seed protein content and productivity in pea.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">768</span>–<span class="cit-lpage">781</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.107.096966%26rft_id%253Dinfo%253Apmid%252F17449650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNDQvMi83NjgiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8xLzIvOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.5" data-doi="10.1073/pnas.0603228103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cannon</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sterck</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rombauts</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Legume genome evolution viewed through the <em>Medicago truncatula</em> and <em>Lotus japonicus</em> genomes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">14959</span>–<span class="cit-lpage">14964</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0603228103%26rft_id%253Dinfo%253Apmid%252F17003129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzQwLzE0OTU5IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMS8yLzkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.6" data-doi="10.1073/pnas.0402251101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mun</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baek</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Estimating genome conservation between crop and model legume species.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">15289</span>–<span class="cit-lpage">15294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0402251101%26rft_id%253Dinfo%253Apmid%252F15489274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzQzLzE1Mjg5IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMS8yLzkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.7" data-doi="10.1093/bioinformatics/bti222"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Givry</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouchez</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chabrier</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schiex</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">CARTHAGENE: multipopulation integrated genetic and radiated hybrid mapping.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1703</span>–<span class="cit-lpage">1704</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti222%26rft_id%253Dinfo%253Apmid%252F15598829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti222&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15598829&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228401800062&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.8" data-doi="10.1007/BF02712670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dellaporta</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hicks</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">A plant DNA minipreparation: Version II.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol. Rep.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMol.%2BBiol.%2BRep.%26rft.volume%253D1%26rft.spage%253D19%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02712670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02712670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.9" data-doi="10.1186/1471-2164-11-468"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deulvot</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charrel</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marty</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacquin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnadieu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Highly-multiplexed SNP genotyping for genetic mapping and germplasm diversity studies in pea.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DDeulvot%26rft.auinit1%253DC.%26rft.volume%253D11%26rft.spage%253D468%26rft.epage%253D468%26rft.atitle%253DHighly-multiplexed%2BSNP%2Bgenotyping%2Bfor%2Bgenetic%2Bmapping%2Band%2Bgermplasm%2Bdiversity%2Bstudies%2Bin%2Bpea.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-468%26rft_id%253Dinfo%253Apmid%252F20701750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-468&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20701750&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.10" data-doi="10.1104/pp.102.018150"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doyle</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luckow</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The rest of the iceberg. Legume diversity and evolution in a phylogenetic context.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">900</span>–<span class="cit-lpage">910</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DDoyle%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D131%26rft.issue%253D3%26rft.spage%253D900%26rft.epage%253D910%26rft.atitle%253DThe%2BRest%2Bof%2Bthe%2BIceberg.%2BLegume%2BDiversity%2Band%2BEvolution%2Bin%2Ba%2BPhylogenetic%2BContext%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.102.018150%26rft_id%253Dinfo%253Apmid%252F12644643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxMzEvMy85MDAiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8xLzIvOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.11" data-doi="10.1046/j.0028-646X.2001.00302.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">T. H. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poyser</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">An integrated and comparative view of pea genetic and cytogenetic maps.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">17</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D153%26rft.spage%253D17%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.0028-646X.2001.00302.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.0028-646X.2001.00302.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000172980200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.12" data-doi="10.1038/nature00842"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endre</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kereszt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kevei</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mihacea</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaló</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A receptor kinase gene regulating symbiotic nodule development.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">417</span>: <span class="cit-fpage">962</span>–<span class="cit-lpage">966</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DEndre%26rft.auinit1%253DG.%26rft.volume%253D417%26rft.issue%253D6892%26rft.spage%253D962%26rft.epage%253D966%26rft.atitle%253DA%2Breceptor%2Bkinase%2Bgene%2Bregulating%2Bsymbiotic%2Bnodule%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature00842%26rft_id%253Dinfo%253Apmid%252F12087406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature00842&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12087406&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176441200042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.13" data-doi="10.1186/1471-2105-9-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forment</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilabert</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robles</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conejero</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nuez</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">EST2uni: an open, parallel tool for automated EST analysis and database creation, with a data mining web interface and microarray expression data integration.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DForment%26rft.auinit1%253DJ.%26rft.volume%253D9%26rft.spage%253D5%26rft.epage%253D5%26rft.atitle%253DEST2uni%253A%2Ban%2Bopen%252C%2Bparallel%2Btool%2Bfor%2Bautomated%2BEST%2Banalysis%2Band%2Bdatabase%2Bcreation%252C%2Bwith%2Ba%2Bdata%2Bmining%2Bweb%2Binterface%2Band%2Bmicroarray%2Bexpression%2Bdata%2Bintegration.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-5%26rft_id%253Dinfo%253Apmid%252F18179701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18179701&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.14" data-doi="10.1186/1471-2164-10-629"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galeano</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernández</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gómez</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blair</span>  <span class="cit-name-given-names">M. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Single strand conformation polymorphism based SNP and indel markers for genetic mapping and synteny analysis of common bean (<em>Phaseolus vulgaris</em> L.).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">629</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DGaleano%26rft.auinit1%253DC.%2BH.%26rft.volume%253D10%26rft.spage%253D629%26rft.epage%253D629%26rft.atitle%253DSingle%2Bstrand%2Bconformation%2Bpolymorphism%2Bbased%2BSNP%2Band%2BIndel%2Bmarkers%2Bfor%2Bgenetic%2Bmapping%2Band%2Bsynteny%2Banalysis%2Bof%2Bcommon%2Bbean%2B%2528Phaseolus%2Bvulgaris%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-629%26rft_id%253Dinfo%253Apmid%252F20030833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-629&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20030833&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.15" data-doi="10.1042/bj2080119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gatehouse</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bown</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Croy</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boulter</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Control of storage-protein synthesis during seed development in pea (<em>Pisum sativum</em> L.).</span> <abbr class="cit-jnl-abbrev">Biochem. J.</abbr> <span class="cit-vol">208</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2BJournal%26rft.stitle%253DBiochem.%2BJ.%26rft.aulast%253DGatehouse%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D208%26rft.issue%253D1%26rft.spage%253D119%26rft.epage%253D127%26rft.atitle%253DControl%2Bof%2Bstorage-protein%2Bsynthesis%2Bduring%2Bseed%2Bdevelopment%2Bin%2Bpea%2B%2528Pisum%2Bsativum%2BL.%2529%26rft_id%253Dinfo%253Adoi%252F10.1042%252Fbj2080119%26rft_id%253Dinfo%253Apmid%252F6897609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InBwYmlvY2hlbWoiO3M6NToicmVzaWQiO3M6OToiMjA4LzEvMTE5IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMS8yLzkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">George</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sawbridge</span>  <span class="cit-name-given-names">T. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cogan</span>  <span class="cit-name-given-names">N.O.I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gendall</span>  <span class="cit-name-given-names">A.R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Comparison of genome structure between white clover and <em>Medicago truncatula</em> supports because group nomenclature based on conserved synteny.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">905</span>–<span class="cit-lpage">911</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DGeorge%26rft.auinit1%253DJ.%26rft.volume%253D51%26rft.issue%253D11%26rft.spage%253D905%26rft.epage%253D911%26rft.atitle%253DComparison%2Bof%2Bgenome%2Bstructure%2Bbetween%2Bwhite%2Bclover%2Band%2BMedicago%2Btruncatula%2Bsupports%2Bhomoeologous%2Bgroup%2Bnomenclature%2Bbased%2Bon%2Bconserved%2Bsynteny.%26rft_id%253Dinfo%253Apmid%252F18956023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18956023&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.17" data-doi="10.1080/07315724.2002.10719263"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grusak</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Enhancing mineral content in plant food products.</span> <abbr class="cit-jnl-abbrev">J. Am. Coll. Nutr.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">178S</span>–<span class="cit-lpage">183S</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BAmerican%2BCollege%2Bof%2BNutrition%26rft.stitle%253DJ.%2BAm.%2BColl.%2BNutr.%26rft.aulast%253DGrusak%26rft.auinit1%253DM.%2BA.%26rft.volume%253D21%26rft.issue%253D90003%26rft.spage%253D178S%26rft.epage%253D183%26rft.atitle%253DEnhancing%2BMineral%2BContent%2Bin%2BPlant%2BFood%2BProducts%26rft_id%253Dinfo%253Adoi%252F10.1080%252F07315724.2002.10719263%26rft_id%253Dinfo%253Apmid%252F12071302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/07315724.2002.10719263&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12071302&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.18" data-doi="10.1023/A:1016085523752"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gualtieri</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulikova</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Limpens</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Microsynteny between pea and Medicago truncatula in the SYM2 region</span>. <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DGualtieri%26rft.auinit1%253DG.%26rft.volume%253D50%26rft.issue%253D2%26rft.spage%253D225%26rft.epage%253D235%26rft.atitle%253DMicrosynteny%2Bbetween%2Bpea%2Band%2BMedicago%2Btruncatula%2Bin%2Bthe%2BSYM2%2Bregion.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1016085523752%26rft_id%253Dinfo%253Apmid%252F12175015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1016085523752&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12175015&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176462800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.19" data-doi="10.1079/BJN2002720"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guillon</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Champ</span>  <span class="cit-name-given-names">M. M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Carbohydrates fractions of legumes: uses in human nutrition and potential for health.</span> <abbr class="cit-jnl-abbrev">Br. J. Nutr.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">293</span>–<span class="cit-lpage">306</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBr.%2BJ.%2BNutr.%26rft.volume%253D88%26rft.spage%253D293%26rft_id%253Dinfo%253Adoi%252F10.1079%252FBJN2002720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1079/BJN2002720&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.20" data-doi="10.1101/gr.9.9.868"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madan</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">CAP3: a DNA sequence assembly program.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">868</span>–<span class="cit-lpage">877</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DHuang%26rft.auinit1%253DX.%26rft.volume%253D9%26rft.issue%253D9%26rft.spage%253D868%26rft.epage%253D877%26rft.atitle%253DCAP3%253A%2BA%2BDNA%2BSequence%2BAssembly%2BProgram%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.9.9.868%26rft_id%253Dinfo%253Apmid%252F10508846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjkvOS84NjgiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8xLzIvOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.2.93.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">T. H. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knox</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The relationship between genetic and cytogenetic maps of pea. II. Physical maps of linkage mapping populations.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">755</span>–<span class="cit-lpage">769</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DHall%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D40%26rft.issue%253D5%26rft.spage%253D755%26rft.epage%253D769%26rft.atitle%253DThe%2Brelationship%2Bbetween%2Bgenetic%2Band%2Bcytogenetic%2Bmaps%2Bof%2Bpea.%2BII.%2BPhysical%2Bmaps%2Bof%2Blinkage%2Bmapping%2Bpopulations.%26rft_id%253Dinfo%253Apmid%252F18464863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18464863&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.22" data-doi="10.1104/pp.104.057018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hecht</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foucher</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrandiz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macknight</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navarro</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Conservation of Arabidopsis flowering genes in model legumes.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">1420</span>–<span class="cit-lpage">1434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.104.057018%26rft_id%253Dinfo%253Apmid%252F15778459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTM3LzQvMTQyMCI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzEvMi85My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.2.93.23" data-doi="10.1534/genetics.108.090084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hougaard</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madsen</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandal</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moretzsohn</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fredslund</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Legume anchor markers link syntenic regions between Phaseolus vulgaris, Lotus japonicus, Medicago truncatula and Arachis</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">2299</span>–<span class="cit-lpage">2312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D179%26rft.spage%253D2299%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.090084%26rft_id%253Dinfo%253Apmid%252F18689902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.108.090084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18689902&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258591200046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.24" data-doi="10.1534/genetics.107.081323"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jing</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seres</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kiss</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ambrose</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Gene-based sequence diversity analysis of field pea (<em>Pisum</em>).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2263</span>–<span class="cit-lpage">2275</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D177%26rft.spage%253D2263%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081323%26rft_id%253Dinfo%253Apmid%252F18073431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.107.081323&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18073431&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251949800026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.25" data-doi="10.1007/s00438-004-1055-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaló</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seres</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jakab</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kevei</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Comparative mapping between <em>Medicago sativa</em> and <em>Pisum sativum</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">272</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DKal%25C3%25B3%26rft.auinit1%253DP.%26rft.volume%253D272%26rft.issue%253D3%26rft.spage%253D235%26rft.epage%253D246%26rft.atitle%253DComparative%2Bmapping%2Bbetween%2BMedicago%2Bsativa%2Band%2BPisum%2Bsativum.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-004-1055-z%26rft_id%253Dinfo%253Apmid%252F15340836%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-004-1055-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15340836&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224752000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katoch</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pathania</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banayal</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Molecular mapping of pea powdery mildew resistance gene <em>er2</em> to pea linkage group III.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">229</span>–<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D25%26rft.spage%253D229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.27" data-doi="10.1242/dev.00814"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kroj</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Savino</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valon</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giraudat</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parcy</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Regulation of storage protein gene expression in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">6065</span>–<span class="cit-lpage">6073</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DKroj%26rft.auinit1%253DT.%26rft.volume%253D130%26rft.issue%253D24%26rft.spage%253D6065%26rft.epage%253D6073%26rft.atitle%253DRegulation%2Bof%2Bstorage%2Bprotein%2Bgene%2Bexpression%2Bin%2BArabidopsis%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.00814%26rft_id%253Dinfo%253Apmid%252F14597573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTMwLzI0LzYwNjUiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8xLzIvOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.28" data-doi="10.1038/nature01207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krusell</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madsen</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aubert</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Genua</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Shoot control of root development and nodulation is mediated by a receptor-like kinase.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">420</span>: <span class="cit-fpage">422</span>–<span class="cit-lpage">426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKrusell%26rft.auinit1%253DL.%26rft.volume%253D420%26rft.issue%253D6914%26rft.spage%253D422%26rft.epage%253D426%26rft.atitle%253DShoot%2Bcontrol%2Bof%2Broot%2Bdevelopment%2Band%2Bnodulation%2Bis%2Bmediated%2Bby%2Ba%2Breceptor-like%2Bkinase.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01207%26rft_id%253Dinfo%253Apmid%252F12442170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01207&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12442170&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179494700041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.29" data-doi="10.1074/jbc.M210538200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lara</span>  <span class="cit-name-given-names">P</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oñate-Sánchez</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abraham</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrándiz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Díaz</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Synergistic activation of seed storage protein gene expression in <em>Arabidopsis</em> by ABI3 and Two bZIPs Related to OPAQUE2.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">278</span>: <span class="cit-fpage">21003</span>–<span class="cit-lpage">21011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M210538200%26rft_id%253Dinfo%253Apmid%252F12657652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzgvMjMvMjEwMDMiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8xLzIvOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.30" data-doi="10.1007/s001220050971"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laucou</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haurogné</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rameau</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Genetic mapping in pea. 1. RAPD-based genetic linkage map of <em>Pisum sativum</em>.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">905</span>–<span class="cit-lpage">915</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D97%26rft.spage%253D905%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050971&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000076924400029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.31" data-doi="10.1126/science.1093038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lévy</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bres</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geurts</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chalhoub</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulikova</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A putative Ca<sup>2+</sup> and calmodulin-dependent protein kinase required for bacterial and fungal symbioses.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">303</span>: <span class="cit-fpage">1361</span>–<span class="cit-lpage">1364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1093038%26rft_id%253Dinfo%253Apmid%252F14963335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDMvNTY2Mi8xMzYxIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMS8yLzkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.32" data-doi="10.1111/j.1744-7909.2010.00949.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhuang</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phil</span>  <span class="cit-name-given-names">M. A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rameau</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">X. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic analysis of ele mutants and comparative mapping of ele1 locus in the control of organ internal asymmetry in garden pea.</span> <abbr class="cit-jnl-abbrev">J. Integr. Plant Biol.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">528</span>–<span class="cit-lpage">535</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BIntegr%2BPlant%2BBiol%26rft.aulast%253DLi%26rft.auinit1%253DX.%26rft.volume%253D52%26rft.issue%253D6%26rft.spage%253D528%26rft.epage%253D535%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Bele%2Bmutants%2Band%2Bcomparative%2Bmapping%2Bof%2Bele1%2Blocus%2Bin%2Bthe%2Bcontrol%2Bof%2Borgan%2Binternal%2Basymmetry%2Bin%2Bgarden%2Bpea.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1744-7909.2010.00949.x%26rft_id%253Dinfo%253Apmid%252F20590983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1744-7909.2010.00949.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20590983&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.33" data-doi="10.1126/science.1090074"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Limpens</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franken</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smit</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willemse</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bisseling</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">LysM domain receptor kinases regulating rhizobial Nod factor-induced infection.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">302</span>: <span class="cit-fpage">630</span>–<span class="cit-lpage">633</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1090074%26rft_id%253Dinfo%253Apmid%252F12947035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDIvNTY0NS82MzAiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8xLzIvOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.34" data-doi="10.1007/s00122-005-0014-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loridon</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McPhee</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubreuil</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pilet-Nayel</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Microsatellite marker polymorphism and mapping in pea (<em>Pisum sativum</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">1022</span>–<span class="cit-lpage">1031</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLoridon%26rft.auinit1%253DK.%26rft.volume%253D111%26rft.issue%253D6%26rft.spage%253D1022%26rft.epage%253D1031%26rft.atitle%253DMicrosatellite%2Bmarker%2Bpolymorphism%2Band%2Bmapping%2Bin%2Bpea%2B%2528Pisum%2Bsativum%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-0014-3%26rft_id%253Dinfo%253Apmid%252F16133320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-0014-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16133320&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232500000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.35" data-doi="10.1186/1471-2164-8-427"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Macas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neumann</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navrátilová</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Repetitive DNA in the pea (<em>Pisum sativum</em> L.) genome comprehensive characterization using 454 sequencing and comparison to soybean and <em>Medicago truncatula</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMacas%26rft.auinit1%253DJ.%26rft.volume%253D8%26rft.spage%253D427%26rft.epage%253D427%26rft.atitle%253DRepetitive%2BDNA%2Bin%2Bthe%2Bpea%2B%2528Pisum%2Bsativum%2BL.%2529%2Bgenome%253A%2Bcomprehensive%2Bcharacterization%2Busing%2B454%2Bsequencing%2Band%2Bcomparison%2Bto%2Bsoybean%2Band%2BMedicago%2Btruncatula.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-8-427%26rft_id%253Dinfo%253Apmid%252F18031571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-8-427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18031571&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.36" data-doi="10.1105/tpc.110.075861"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mbengue</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camut</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Carvalho-Niebel</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deslandes</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Froidure</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The <em>Medicago truncatula</em> E3 ubiquitin ligase PUB1 interacts with the LYK3 symbiotic receptor and negatively regulates infection and nodulation.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3474</span>–<span class="cit-lpage">3488</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.110.075861%26rft_id%253Dinfo%253Apmid%252F20971894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xMC8zNDc0IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMS8yLzkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.2.93.37" data-doi="10.1186/1471-2164-11-184"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McClean</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mamidi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McConnell</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chikara</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Synteny mapping between common bean and soybean reveals extensive blocks of shared loci.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMcClean%26rft.auinit1%253DP.%2BE.%26rft.volume%253D11%26rft.spage%253D184%26rft.epage%253D184%26rft.atitle%253DSynteny%2Bmapping%2Bbetween%2Bcommon%2Bbean%2Band%2Bsoybean%2Breveals%2Bextensive%2Bblocks%2Bof%2Bshared%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-184%26rft_id%253Dinfo%253Apmid%252F20298570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-184&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20298570&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.38" data-doi="10.1007/BF00384094"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meinke</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beachy</span>  <span class="cit-name-given-names">N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Expression of storage-protein genes during soybean seed development.</span> <abbr class="cit-jnl-abbrev">Planta</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">130</span>–<span class="cit-lpage">139</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlanta%26rft.volume%253D153%26rft.spage%253D130%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00384094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00384094&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1981MP93100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.39" data-doi="10.1007/s00425-009-0931-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mishra</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaudhary</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Effects of MULTIFOLIATE-PINNA, AFILA, TENDRIL-LESS and UNIFOLIATA genes on leafblade architecture in <em>Pisum sativum</em>.</span> <abbr class="cit-jnl-abbrev">Planta</abbr> <span class="cit-vol">230</span>: <span class="cit-fpage">177</span>–<span class="cit-lpage">190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlanta%26rft.aulast%253DMishra%26rft.auinit1%253DR.%2BK.%26rft.volume%253D230%26rft.issue%253D1%26rft.spage%253D177%26rft.epage%253D190%26rft.atitle%253DEffects%2Bof%2BMULTIFOLIATE-PINNA%252C%2BAFILA%252C%2BTENDRIL-LESS%2Band%2BUNIFOLIATA%2Bgenes%2Bon%2Bleafblade%2Barchitecture%2Bin%2BPisum%2Bsativum.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00425-009-0931-5%26rft_id%253Dinfo%253Apmid%252F19404676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00425-009-0931-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19404676&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266486600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.40" data-doi="10.1016/j.jada.2009.02.029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawrence</span>  <span class="cit-name-given-names">F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartman</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curran</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Consumption of dry beans, peas, and lentils could improve diet quality in the US population.</span> <abbr class="cit-jnl-abbrev">J. Am. Diet. Assoc.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">909</span>–<span class="cit-lpage">913</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BAmerican%2BDietetic%2BAssociation%26rft.stitle%253DJ%2BAm%2BDiet%2BAssoc%26rft.aulast%253DMitchell%26rft.auinit1%253DD.%2BC.%26rft.volume%253D109%26rft.issue%253D5%26rft.spage%253D909%26rft.epage%253D913%26rft.atitle%253DConsumption%2Bof%2Bdry%2Bbeans%252C%2Bpeas%252C%2Band%2Blentils%2Bcould%2Bimprove%2Bdiet%2Bquality%2Bin%2Bthe%2BUS%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jada.2009.02.029%26rft_id%253Dinfo%253Apmid%252F19394480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jada.2009.02.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19394480&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mönke</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altschmied</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tewes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reidt</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mock</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Seed-specific transcription factors ABI3 and FUS3: molecular interaction with DNA.</span> <abbr class="cit-jnl-abbrev">Planta</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">158</span>–<span class="cit-lpage">166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlanta%26rft.volume%253D19%26rft.spage%253D158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.2.93.42" data-doi="10.1007/s00122-010-1265-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nayak</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varghese</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Datta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Integration of novel SSR and gene-based SNP marker loci in the chickpea genetic map and establishment of new anchor points with <em>Medicago truncatula</em> genome.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">1415</span>–<span class="cit-lpage">1441</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DNayak%26rft.auinit1%253DS.%2BN.%26rft.volume%253D120%26rft.issue%253D7%26rft.spage%253D1415%26rft.epage%253D1441%26rft.atitle%253DIntegration%2Bof%2Bnovel%2BSSR%2Band%2Bgene-based%2BSNP%2Bmarker%2Bloci%2Bin%2Bthe%2Bchickpea%2Bgenetic%2Bmap%2Band%2Bestablishment%2Bof%2Bnew%2Banchor%2Bpoints%2Bwith%2BMedicago%2Btruncatula%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1265-1%26rft_id%253Dinfo%253Apmid%252F20098978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1265-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20098978&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276674500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.43" data-doi="10.1105/tpc.9.8.1265"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parcy</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valon</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohara</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miséra</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giraudat</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The <em>ABSCISIC ACID-lNSENSITIVE3</em>, <em>FUSCA3</em>, and <em>LEAFY COTYLEDON1</em> loci act in concert to control multiple aspects of <em>Arabidopsis</em> seed development.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1265</span>–<span class="cit-lpage">1277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DParcy%26rft.auinit1%253DF.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D1265%26rft.epage%253D1277%26rft.atitle%253DThe%2BABSCISIC%2BACID-INSENSITIVE3%252C%2BFUSCA3%252C%2Band%2BLEAFY%2BCOTYLEDON1%2BLoci%2BAct%2Bin%2BConcert%2Bto%2BControl%2BMultiple%2BAspects%2Bof%2BArabidopsis%2BSeed%2BDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.9.8.1265%26rft_id%253Dinfo%253Apmid%252F9286105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjkvOC8xMjY1IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMS8yLzkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.44" data-doi="10.1093/bioinformatics/btg034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antonescu</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sultana</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">TIGR Gene Indices clustering tools (TGICL): a software system for fast clustering of large EST datasets.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">651</span>–<span class="cit-lpage">652</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DPertea%26rft.auinit1%253DG.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D651%26rft.epage%253D652%26rft.atitle%253DTIGR%2BGene%2BIndices%2Bclustering%2Btools%2B%2528TGICL%2529%253A%2Ba%2Bsoftware%2Bsystem%2Bfor%2Bfast%2Bclustering%2Bof%2Blarge%2BEST%2Bdatasets%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg034%26rft_id%253Dinfo%253Apmid%252F12651724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12651724&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181964600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phan</span>  <span class="cit-name-given-names">H. T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellwood</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ford</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Differences in syntenic complexity between <em>Medicago truncatula</em> with <em>Lens culinaris</em> and <em>Lupinus albus</em></span>. <abbr class="cit-jnl-abbrev">Funct. Plant Biol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">775</span>–<span class="cit-lpage">782</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunct.%2BPlant%2BBiol.%26rft.volume%253D33%26rft.spage%253D775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prioul-Gervais</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deniot</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Receveur</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frankewitz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fourmann</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Candidate genes for quantitative resistance to <em>Mycosphaerella pinodes</em> in pea (<em>Pisum sativum</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">971</span>–<span class="cit-lpage">984</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DPrioul-Gervais%26rft.auinit1%253DS.%26rft.volume%253D114%26rft.issue%253D6%26rft.spage%253D971%26rft.epage%253D984%26rft.atitle%253DCandidate%2Bgenes%2Bfor%2Bquantitative%2Bresistance%2Bto%2BMycosphaerella%2Bpinodes%2Bin%2Bpea%2B%2528Pisum%2Bsativum%2BL.%2529.%26rft_id%253Dinfo%253Apmid%252F17265025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17265025&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.47" data-doi="10.1007/s001220050972"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rameau</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dénoue</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraval</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haurogné</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Josserand</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Genetic mapping in pea. 2. Identification of RAPD and SCAR markers linked to genes affecting plant architecture.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">916</span>–<span class="cit-lpage">928</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D97%26rft.spage%253D916%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050972&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000076924400030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.48" data-doi="10.1073/pnas.0902350106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salse</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abrouk</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolot</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guilhot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Courcelle</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span> <span class="cit-article-title">Reconstruction of monocotelydoneous proto-chromosomes reveals faster evolution in plants than in animals.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">14908</span>–<span class="cit-lpage">14913</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0902350106%26rft_id%253Dinfo%253Apmid%252F19706486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzM1LzE0OTA4IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMS8yLzkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.49" data-doi="10.1093/bib/bbp037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salse</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abrouk</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murat</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masood Quraishi</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feuillet</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span> <span class="cit-article-title">Improved standards and new comparative genomics tools provide new insights into grasses paleogenomics.</span> <abbr class="cit-jnl-abbrev">Briefings in Bioinf.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">619</span>–<span class="cit-lpage">630</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBriefings%2Bin%2BBioinf.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbp037%26rft_id%253Dinfo%253Apmid%252F19720678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbp037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19720678&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272086700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.50" data-doi="10.1104/pp.107.097493"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asamizu</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isobe</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tabata</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome sequencing and genome resources in model legumes.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">588</span>–<span class="cit-lpage">593</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DSato%26rft.auinit1%253DS.%26rft.volume%253D144%26rft.issue%253D2%26rft.spage%253D588%26rft.epage%253D593%26rft.atitle%253DGenome%2BSequencing%2Band%2BGenome%2BResources%2Bin%2BModel%2BLegumes%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.107.097493%26rft_id%253Dinfo%253Apmid%252F17556522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNDQvMi81ODgiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8xLzIvOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isobe</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tabata</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Structural analyses of the genomes in legumes.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">146</span>–<span class="cit-lpage">152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DSato%26rft.auinit1%253DS.%26rft.volume%253D13%26rft.issue%253D2%26rft.spage%253D146%26rft.epage%253D152%26rft.atitle%253DStructural%2Banalyses%2Bof%2Bthe%2Bgenomes%2Bin%2Blegumes.%26rft_id%253Dinfo%253Apmid%252F20071214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20071214&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.52" data-doi="10.1073/pnas.87.1.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burr</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auckerman</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burr</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Maize regulatory gene Opaque-2 encodes a protein with a “leucine-zipper” motif that binds to zein DNA.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">46</span>–<span class="cit-lpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSchmidt%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D87%26rft.issue%253D1%26rft.spage%253D46%26rft.epage%253D50%26rft.atitle%253DMaize%2Bregulatory%2Bgene%2Bopaque-2%2Bencodes%2Ba%2Bprotein%2Bwith%2Ba%2B%2522leucine-zipper%2522%2Bmotif%2Bthat%2Bbinds%2Bto%2Bzein%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.87.1.46%26rft_id%253Dinfo%253Apmid%252F2296602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo3OiI4Ny8xLzQ2IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMS8yLzkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.53" data-doi="10.1105/tpc.4.6.689"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ketudat</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aukerman</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoschek</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Opaque-2 is a transcriptional activator that recognizes a specific target site in 22-kD zein genes.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">689</span>–<span class="cit-lpage">700</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DSchmidt%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253D689%26rft.epage%253D700%26rft.atitle%253DOpaque-2%2BIs%2Ba%2BTranscriptional%2BActivator%2BThat%2BRecognizes%2Ba%2BSpecific%2BTarget%2BSite%2Bin%2B22-kD%2BZein%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.4.6.689%26rft_id%253Dinfo%253Apmid%252F1392590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjc6IjQvNi82ODkiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8xLzIvOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.54" data-doi="10.1038/nature08670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmutz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cannon</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlueter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitros</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome sequence of the palaeopolyploid soybean.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">463</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSchmutz%26rft.auinit1%253DJ.%26rft.volume%253D463%26rft.issue%253D7278%26rft.spage%253D178%26rft.epage%253D183%26rft.atitle%253DGenome%2Bsequence%2Bof%2Bthe%2Bpalaeopolyploid%2Bsoybean.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08670%26rft_id%253Dinfo%253Apmid%252F20075913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20075913&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273582700025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.2.93.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MuehIbauer</span>  <span class="cit-name-given-names">F. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Construction of a chickpea linkage map and its comparison with maps of pea and lentil.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft.volume%253D88%26rft.spage%253D115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sinjushin</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Konovalov</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gostimskii</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A gene for stem fasciation is localized on linkage group III</span>. <abbr class="cit-jnl-abbrev">Pisum Genetics</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPisum%2BGenetics%26rft.volume%253D38%26rft.spage%253D19%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.57" data-doi="10.1007/s00122-003-1438-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stracke</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandal</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koyama</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaneko</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Exploitation of colinear relationships between the genomes of <em>Lotus japonicus</em>, <em>Pisum sativum</em> and <em>Arabidopsis thaliana</em>, for positional cloning of a legume symbiosis gene.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">442</span>–<span class="cit-lpage">449</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DStracke%26rft.auinit1%253DS.%26rft.volume%253D108%26rft.issue%253D3%26rft.spage%253D442%26rft.epage%253D449%26rft.atitle%253DExploitation%2Bof%2Bcolinear%2Brelationships%2Bbetween%2Bthe%2Bgenomes%2Bof%2BLotus%2Bjaponicus%252C%2BPisum%2Bsativum%2Band%2BArabidopsis%2Bthaliana%252C%2Bfor%2Bpositional%2Bcloning%2Bof%2Ba%2Blegume%2Bsymbiosis%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-003-1438-2%26rft_id%253Dinfo%253Apmid%252F14557858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-003-1438-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14557858&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188747800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.58" data-doi="10.1126/science.1153917"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ming</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alam</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Synteny and collinearity in plant genomes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>: <span class="cit-fpage">486</span>–<span class="cit-lpage">488</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTang%26rft.auinit1%253DH.%26rft.volume%253D320%26rft.issue%253D5875%26rft.spage%253D486%26rft.epage%253D488%26rft.atitle%253DSynteny%2Band%2BCollinearity%2Bin%2BPlant%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1153917%26rft_id%253Dinfo%253Apmid%252F18436778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjAvNTg3NS80ODYiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8xLzIvOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.59" data-doi="10.1104/pp.122.2.319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tegeder</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Offler</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frommer</span>  <span class="cit-name-given-names">W. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patrick</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Amino acid transporters are localized to transfer cells of developing pea seeds.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DTegeder%26rft.auinit1%253DM.%26rft.volume%253D122%26rft.issue%253D2%26rft.spage%253D319%26rft.epage%253D326%26rft.atitle%253DAmino%2BAcid%2BTransporters%2BAre%2BLocalized%2Bto%2BTransfer%2BCells%2Bof%2BDeveloping%2BPea%2BSeeds%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.122.2.319%26rft_id%253Dinfo%253Apmid%252F10677425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxMjIvMi8zMTkiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8xLzIvOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.60" data-doi="10.1105/tpc.4.6.701"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ueda</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waverczak</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sher</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ketudat</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Mutations of the 22- and 27-kD zein promoters affect transactivation by the opaque-2 protein.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">701</span>–<span class="cit-lpage">709</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DUeda%26rft.auinit1%253DT.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253D701%26rft.epage%253D709%26rft.atitle%253DMutations%2Bof%2Bthe%2B22-%2Band%2B27-kD%2BZein%2BPromoters%2BAffect%2BTransactivation%2Bby%2Bthe%2BOpaque-2%2BProtein%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.4.6.701%26rft_id%253Dinfo%253Apmid%252F1392591%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjc6IjQvNi83MDEiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8xLzIvOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.61" data-doi="10.1093/jhered/93.1.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">MapChart: software for the graphical presentation of linkage maps and QTLs.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DVoorrips%26rft.auinit1%253DR.%2BE.%26rft.volume%253D93%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D78%26rft.atitle%253DMapChart%253A%2BSoftware%2Bfor%2Bthe%2BGraphical%2BPresentation%2Bof%2BLinkage%2BMaps%2Band%2BQTLs%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F93.1.77%26rft_id%253Dinfo%253Apmid%252F12011185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/93.1.77&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12011185&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175627300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.62" data-doi="10.1073/pnas.0803291105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic control of floral zygomorphy in pea (<em>Pisum sativum</em> L.).</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">10414</span>–<span class="cit-lpage">10419</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0803291105%26rft_id%253Dinfo%253Apmid%252F18650395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzMwLzEwNDE0IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMS8yLzkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weeden</span>  <span class="cit-name-given-names">N. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muehlbauer</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ladizinsky</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Extensive conservation of linkage relationships between pea and lentil genetic maps.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DWeeden%26rft.auinit1%253DN.%2BF.%26rft.volume%253D83%26rft.issue%253D2%26rft.spage%253D123%26rft.epage%253D129%26rft.atitle%253DExtensive%2BConservation%2Bof%2BLinkage%2BRelationships%2BBetween%2BPea%2Band%2BLentil%2BGenetic%2BMaps%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1992HR24100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weeden</span>  <span class="cit-name-given-names">N. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">T. H. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Timmerman-Vaughan</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swiecicki</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozov</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">A consensus linkage map for <em>Pisum sativum</em>.</span> <abbr class="cit-jnl-abbrev">Pisum Genet.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPisum%2BGenet.%26rft.volume%253D30%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.65" data-doi="10.1104/pp.104.057034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cannon</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Sequencing the genespaces of <em>Medicago truncatula</em> and <em>Lotus japonicus</em>.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">1174</span>–<span class="cit-lpage">1181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DYoung%26rft.auinit1%253DN.%2BD.%26rft.volume%253D137%26rft.issue%253D4%26rft.spage%253D1174%26rft.epage%253D1181%26rft.atitle%253DSequencing%2Bthe%2BGenespaces%2Bof%2BMedicago%2Btruncatula%2Band%2BLotus%2Bjaponicus%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.104.057034%26rft_id%253Dinfo%253Apmid%252F15824279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTM3LzQvMTE3NCI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzEvMi85My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.66" data-doi="10.1104/pp.104.058891"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Bridging model and crop legumes through comparative genomics.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">1189</span>–<span class="cit-lpage">1196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DZhu%26rft.auinit1%253DH.%26rft.volume%253D137%26rft.issue%253D4%26rft.spage%253D1189%26rft.epage%253D1196%26rft.atitle%253DBridging%2BModel%2Band%2BCrop%2BLegumes%2Bthrough%2BComparative%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.104.058891%26rft_id%253Dinfo%253Apmid%252F15824281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTM3LzQvMTE4OSI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzEvMi85My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.93.67" data-doi="10.1534/genetics.105.051185"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riely</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burns</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ané</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Tracing nonlegume orthologs of legume genes required for nodulation and arbuscular mycorhizal symbiose.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">2491</span>–<span class="cit-lpage">2499</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D172%26rft.spage%253D2491%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.051185%26rft_id%253Dinfo%253Apmid%252F16452143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.105.051185&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16452143&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F93.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237225800039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/1/2/93.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/2/85" title="Organization and Evolution of Subtelomeric Satellite Repeats in the Potato Genome" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/2/105" title="A QTL Study for Regions Contributing to Arabidopsis thaliana Root Skewing on Tilted Surfaces" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/1/2"><span>Volume 1 Issue 2, July 2011</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/1/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/1/2.cover.gif?itok=hCLGYhoa" width="248" height="330" alt="G3: Genes, Genomes, Genetics: 1 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/2/93" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-GpOBL7Hxd3ngKVs3DZJeDYPaRGnUmCEquRjaaGHLDiA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigation-1" class="" data-icon-position="" data-hide-link-title="0">Investigation</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/1/2/93" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/1/2/93" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/1/2/93" class="active" data-icon-position="" data-hide-link-title="0">Translational Genomics in Legumes Allowed Placing In Silico 5460 Unigenes on the Pea Functional Map and Identified Candidate Genes in Pisum sativum L.</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459160" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-gEB1WqnCi4L6ucHvG4bavcAgtGdWud2y-YJEbjmbx4Q" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/2/93.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/1/2/93" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-bgsIpqg9ARD_Owb46ULge-A8drGjb-PlUvZ2JNiDcK0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/1/2/93" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/2/93.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459160" id="node459160--2" data-pisa="ggg;1/2/93" data-pisa-master="ggg;g3.111.000349" data-apath="/ggg/1/2/93.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Translational Genomics in Legumes Allowed Placing <em>In Silico</em> 5460 Unigenes on the Pea Functional Map and Identified Candidate Genes in <em>Pisum sativum</em> L.</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Amandine Bordat</span>, <span class="highwire-citation-author" data-delta="1">Vincent Savois</span>, <span class="highwire-citation-author" data-delta="2">Marie Nicolas</span>, <span class="highwire-citation-author" data-delta="3">Jérome Salse</span>, <span class="highwire-citation-author" data-delta="4">Aurélie Chauveau</span>, <span class="highwire-citation-author" data-delta="5">Michael Bourgeois</span>, <span class="highwire-citation-author" data-delta="6">Jean Potier</span>, <span class="highwire-citation-author" data-delta="7">Hervé Houtin</span>, <span class="highwire-citation-author" data-delta="8">Céline Rond</span>, <span class="highwire-citation-author" data-delta="9">Florent Murat</span>, <span class="highwire-citation-author" data-delta="10">Pascal Marget</span>, <span class="highwire-citation-author" data-delta="11">Grégoire Aubert</span> and <span class="highwire-citation-author" data-delta="12">Judith Burstin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">93-103;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000349 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459160--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Amandine Bordat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amandine%2BBordat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bordat%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmandine%2BBordat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Vincent Savois </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vincent%2BSavois%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Savois%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVincent%2BSavois%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marie Nicolas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie%2BNicolas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nicolas%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie%2BNicolas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jérome Salse </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>INRA, UMR1095, 63100 Clermont-Ferrand, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%A9rome%2BSalse%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Salse%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ%25C3%25A9rome%2BSalse%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Aurélie Chauveau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aur%C3%A9lie%2BChauveau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chauveau%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAur%25C3%25A9lie%2BChauveau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Michael Bourgeois </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BBourgeois%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bourgeois%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BBourgeois%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jean Potier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean%2BPotier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Potier%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean%2BPotier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Hervé Houtin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Herv%C3%A9%2BHoutin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Houtin%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHerv%25C3%25A9%2BHoutin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Céline Rond </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C%C3%A9line%2BRond%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rond%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AC%25C3%25A9line%2BRond%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Florent Murat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>INRA, UMR1095, 63100 Clermont-Ferrand, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Florent%2BMurat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Murat%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFlorent%2BMurat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Pascal Marget </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pascal%2BMarget%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marget%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APascal%2BMarget%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Grégoire Aubert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gr%C3%A9goire%2BAubert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aubert%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGr%25C3%25A9goire%2BAubert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Judith Burstin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Judith%2BBurstin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burstin%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJudith%2BBurstin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:judith.burstin@dijon.inra.fr" class="" data-icon-position="" data-hide-link-title="0">judith.burstin@dijon.inra.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/1/2/93" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/1/2/93&amp;title=Translational%20Genomics%20in%20Legumes%20Allowed%20Placing%20In%20Silico%205460%20Unigenes%20on%20the%20Pea%20Functional%20Map%20and%20Identified%20Candidate%20Genes%20in%20Pisum%20sativum%20L." id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/1/2/93&amp;title=Translational%20Genomics%20in%20Legumes%20Allowed%20Placing%20In%20Silico%205460%20Unigenes%20on%20the%20Pea%20Functional%20Map%20and%20Identified%20Candidate%20Genes%20in%20Pisum%20sativum%20L." id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/1/2/93&amp;title=Translational%20Genomics%20in%20Legumes%20Allowed%20Placing%20In%20Silico%205460%20Unigenes%20on%20the%20Pea%20Functional%20Map%20and%20Identified%20Candidate%20Genes%20in%20Pisum%20sativum%20L." id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/1/2/93&amp;text=Translational%20Genomics%20in%20Legumes%20Allowed%20Placing%20In%20Silico%205460%20Unigenes%20on%20the%20Pea%20Functional%20Map%20and%20Identified%20Candidate%20Genes%20in%20Pisum%20sativum%20L." id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/1/2/93&amp;title=Translational%20Genomics%20in%20Legumes%20Allowed%20Placing%20In%20Silico%205460%20Unigenes%20on%20the%20Pea%20Functional%20Map%20and%20Identified%20Candidate%20Genes%20in%20Pisum%20sativum%20L." id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/1/2/93&amp;t=Translational%20Genomics%20in%20Legumes%20Allowed%20Placing%20In%20Silico%205460%20Unigenes%20on%20the%20Pea%20Functional%20Map%20and%20Identified%20Candidate%20Genes%20in%20Pisum%20sativum%20L." id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/1/2/93&amp;title=Translational%20Genomics%20in%20Legumes%20Allowed%20Placing%20In%20Silico%205460%20Unigenes%20on%20the%20Pea%20Functional%20Map%20and%20Identified%20Candidate%20Genes%20in%20Pisum%20sativum%20L." id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/1/2/93&amp;title=Translational%20Genomics%20in%20Legumes%20Allowed%20Placing%20In%20Silico%205460%20Unigenes%20on%20the%20Pea%20Functional%20Map%20and%20Identified%20Candidate%20Genes%20in%20Pisum%20sativum%20L." id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459160/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459160" id="node459160--3" data-pisa="ggg;1/2/93" data-pisa-master="ggg;g3.111.000349" data-apath="/ggg/1/2/93.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Translational Genomics in Legumes Allowed Placing <em>In Silico</em> 5460 Unigenes on the Pea Functional Map and Identified Candidate Genes in <em>Pisum sativum</em> L.</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Amandine Bordat</span>, <span class="highwire-citation-author" data-delta="1">Vincent Savois</span>, <span class="highwire-citation-author" data-delta="2">Marie Nicolas</span>, <span class="highwire-citation-author" data-delta="3">Jérome Salse</span>, <span class="highwire-citation-author" data-delta="4">Aurélie Chauveau</span>, <span class="highwire-citation-author" data-delta="5">Michael Bourgeois</span>, <span class="highwire-citation-author" data-delta="6">Jean Potier</span>, <span class="highwire-citation-author" data-delta="7">Hervé Houtin</span>, <span class="highwire-citation-author" data-delta="8">Céline Rond</span>, <span class="highwire-citation-author" data-delta="9">Florent Murat</span>, <span class="highwire-citation-author" data-delta="10">Pascal Marget</span>, <span class="highwire-citation-author" data-delta="11">Grégoire Aubert</span> and <span class="highwire-citation-author" data-delta="12">Judith Burstin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">93-103;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000349 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459160--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Amandine Bordat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amandine%2BBordat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bordat%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmandine%2BBordat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Vincent Savois </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vincent%2BSavois%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Savois%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVincent%2BSavois%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marie Nicolas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie%2BNicolas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nicolas%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie%2BNicolas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jérome Salse </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>INRA, UMR1095, 63100 Clermont-Ferrand, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%A9rome%2BSalse%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Salse%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ%25C3%25A9rome%2BSalse%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Aurélie Chauveau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aur%C3%A9lie%2BChauveau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chauveau%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAur%25C3%25A9lie%2BChauveau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Michael Bourgeois </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BBourgeois%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bourgeois%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BBourgeois%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jean Potier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean%2BPotier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Potier%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean%2BPotier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Hervé Houtin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Herv%C3%A9%2BHoutin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Houtin%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHerv%25C3%25A9%2BHoutin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Céline Rond </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C%C3%A9line%2BRond%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rond%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AC%25C3%25A9line%2BRond%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Florent Murat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>INRA, UMR1095, 63100 Clermont-Ferrand, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Florent%2BMurat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Murat%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFlorent%2BMurat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Pascal Marget </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pascal%2BMarget%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marget%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APascal%2BMarget%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Grégoire Aubert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gr%C3%A9goire%2BAubert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aubert%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGr%25C3%25A9goire%2BAubert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Judith Burstin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA, UMRLEG, 21065 Dijon Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Judith%2BBurstin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burstin%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJudith%2BBurstin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:judith.burstin@dijon.inra.fr" class="" data-icon-position="" data-hide-link-title="0">judith.burstin@dijon.inra.fr</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=93&amp;rft.epage=103&amp;rft.atitle=Translational%20Genomics%20in%20Legumes%20Allowed%20Placing%20In%20Silico%205460%20Unigenes%20on%20the%20Pea%20Functional%20Map%20and%20Identified%20Candidate%20Genes%20in%20Pisum%20sativum%20L.&amp;rft.volume=1&amp;rft.issue=2&amp;rft.date=2011-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Bordat%2C+Amandine&amp;rft.au=Savois%2C+Vincent&amp;rft.au=Nicolas%2C+Marie&amp;rft.au=Salse%2C+J%C3%A9rome&amp;rft.au=Chauveau%2C+Aur%C3%A9lie&amp;rft.au=Bourgeois%2C+Michael&amp;rft.au=Potier%2C+Jean&amp;rft.au=Houtin%2C+Herv%C3%A9&amp;rft.au=Rond%2C+C%C3%A9line&amp;rft.au=Murat%2C+Florent&amp;rft.au=Marget%2C+Pascal&amp;rft.au=Aubert%2C+Gr%C3%A9goire&amp;rft.au=Burstin%2C+Judith"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459160/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459160/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459160/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459160/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459160/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459160/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459160/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459160/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459160/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459160/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459160/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459160/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496048" id="node496048" data-pisa="ggg;g3.119.400937v1" data-pisa-master="ggg;g3.119.400937" data-apath="/ggg/early/2020/03/25/g3.119.400937.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/25/g3.119.400937" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tunable Transcriptional Interference at the Endogenous Alcohol Dehydrogenase Gene Locus in <em>Drosophila melanogaster</em></span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496046" id="node496046" data-pisa="ggg;g3.120.401048v1" data-pisa-master="ggg;g3.120.401048" data-apath="/ggg/early/2020/03/23/g3.120.401048.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/23/g3.120.401048" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tools for the Genetic Manipulation of <em>Herpetomonas muscarum</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496045" id="node496045" data-pisa="ggg;g3.119.400884v1" data-pisa-master="ggg;g3.119.400884" data-apath="/ggg/early/2020/03/17/g3.119.400884.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400884" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study for Maize Leaf Cuticular Conductance Identifies Candidate Genes Involved in the Regulation of Cuticle Development</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigation" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigation</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/1/2/93" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/1/2/93.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/1/2/93.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/1/2/93.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/1/2/93.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
