<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/1/2/117" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/1/2/117.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/1/2/117.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/1/2/117.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2.cover.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/1/2/117.atom" />
<meta name="HW.pisa" content="ggg;1/2/117" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Fluorescence In Situ Hybridization–Based Karyotyping of Soybean Translocation Lines" />
<meta name="DC.Identifier" content="10.1534/g3.111.000034" />
<meta name="DC.Date" content="2011-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2011 Findley et al.. This is an open access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Soybean ( Glycine max [L.] Merr.) is a major crop species and, therefore, a major target of genomic and genetic research. However, in contrast to other plant species, relatively few chromosomal aberrations have been identified and characterized in soybean. This is due in part to the difficulty of cytogenetic analysis of its small, morphologically homogeneous chromosomes. The recent development of a fluorescence in situ hybridization –based karyotyping system for soybean has enabled our characterization of most of the chromosomal translocation lines identified to date. Utilizing genetic data from existing translocation studies in soybean, we identified the chromosomes and approximate breakpoints involved in five translocation lines." />
<meta name="DC.Contributor" content="Seth D. Findley" />
<meta name="DC.Contributor" content="Allison L. Pappas" />
<meta name="DC.Contributor" content="Yaya Cui" />
<meta name="DC.Contributor" content="James A. Birchler" />
<meta name="DC.Contributor" content="Reid G. Palmer" />
<meta name="DC.Contributor" content="Gary Stacey" />
<meta name="article:published_time" content="2011-07-01" />
<meta name="article:section" content="Investigation" />
<meta name="citation_title" content="Fluorescence In Situ Hybridization–Based Karyotyping of Soybean Translocation Lines" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Soybean (&lt;i&gt;Glycine max&lt;/i&gt; [L.] Merr.) is a major crop species and, therefore, a major target of genomic and genetic research. However, in contrast to other plant species, relatively few chromosomal aberrations have been identified and characterized in soybean. This is due in part to the difficulty of cytogenetic analysis of its small, morphologically homogeneous chromosomes. The recent development of a fluorescence &lt;i&gt;in situ&lt;/i&gt; hybridization –based karyotyping system for soybean has enabled our characterization of most of the chromosomal translocation lines identified to date. Utilizing genetic data from existing translocation studies in soybean, we identified the chromosomes and approximate breakpoints involved in five translocation lines.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2011/07/01" />
<meta name="citation_mjid" content="ggg;1/2/117" />
<meta name="citation_id" content="1/2/117" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/1/2/117" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/1/2/117.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/1/2/117.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/1/2/117.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.000034" />
<meta name="citation_pmid" content="22384324" />
<meta name="citation_volume" content="1" />
<meta name="citation_issue" content="2" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigation" />
<meta name="citation_firstpage" content="117" />
<meta name="citation_lastpage" content="129" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Seth D. Findley" />
<meta name="citation_author_institution" content="National Center for Soybean Biotechnology, Division of Plant Sciences, University of Missouri, Columbia, Missouri 65211" />
<meta name="citation_author" content="Allison L. Pappas" />
<meta name="citation_author_institution" content="USDA-ARS, Iowa State University, Ames, Iowa 50011" />
<meta name="citation_author" content="Yaya Cui" />
<meta name="citation_author_institution" content="National Center for Soybean Biotechnology, Division of Plant Sciences, University of Missouri, Columbia, Missouri 65211" />
<meta name="citation_author" content="James A. Birchler" />
<meta name="citation_author_institution" content="Division of Biological Sciences, University of Missouri, Columbia, Missouri 65211" />
<meta name="citation_author" content="Reid G. Palmer" />
<meta name="citation_author_institution" content="USDA-ARS, Iowa State University, Ames, Iowa 50011" />
<meta name="citation_author" content="Gary Stacey" />
<meta name="citation_author_institution" content="National Center for Soybean Biotechnology, Division of Plant Sciences, University of Missouri, Columbia, Missouri 65211" />
<meta name="citation_author_email" content="staceyg@missouri.edu" />
<meta name="citation_reference" content="citation_journal_title=Cytologia (Tokyo);citation_author=Q. N. Ahmad;citation_author=E. J. Britten;citation_author=D. E. Byth;citation_title=The karyotype of Glycine soja and its relationship to that of the soybean, Glycine max.;citation_pages=645-658;citation_volume=49;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. S. Akkaya;citation_author=A. B. Arvind;citation_author=P. B. Cregan;citation_title=Length polymorphisms of simple sequence repeat DNA in soybean.;citation_pages=1131-1139;citation_volume=132;citation_year=1992;citation_pmid=1459432" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=S. L. Broich;citation_author=R. G. Palmer;citation_title=A cluster analysis of wild and domesticated soybean phenotypes.;citation_pages=23-32;citation_volume=29;citation_year=1980;citation_doi=10.1007/BF00037246" />
<meta name="citation_reference" content="citation_title=Discussions in Cytogenetics.;citation_author=C. R. Burnham;citation_year=1962" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=Y. Chen;citation_author=R. L. Nelson;citation_title=Genetic Variation and Relationships among Cultivated, Wild, and Semiwild Soybean;citation_pages=316-325;citation_volume=44;citation_year=2004;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_title=Crit. Rev. Plant Sci.;citation_author=G. Chung;citation_author=R. J. Singh;citation_title=Broadening the genetic base of soybean: A multidisciplinary approach.;citation_pages=295-341;citation_volume=27;citation_year=2008;citation_doi=10.1080/07352680802333904" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=P. B. Cregan;citation_author=T. Jarvik;citation_author=A. L. Bush;citation_author=R. C. Shoemaker;citation_author=K. G. Lark;citation_title=An Integrated Genetic Linkage Map of the Soybean Genome;citation_pages=1464-1490;citation_volume=39;citation_year=1999;citation_issue=5;citation_doi=10.2135/cropsci1999.3951464x" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=P. B. Cregan;citation_author=K. P. Kollipara;citation_author=S. J. Xu;citation_author=R. J. Singh;citation_author=S. E. Fogarty;citation_title=Primary Trisomics and SSR Markers as Tools to Associate Chromosomes with Linkage Groups in Soybean;citation_pages=1262-1267;citation_volume=41;citation_year=2001;citation_issue=4" />
<meta name="citation_reference" content="citation_title=Screening of Soybean (Glycine max) Accessions and G. soja Accessions for Chromosome Interchanges and Inversions. Agronomy Abstracts;citation_author=X. Delannay;citation_author=T. C. Kilen;citation_author=R. G. Palmer;citation_pages=63;citation_year=1982" />
<meta name="citation_reference" content="citation_journal_title=Journal of applied genetics;citation_journal_abbrev=J Appl Genet;citation_author=L. De Lorenzi;citation_author=L. Molteni;citation_author=P. Parma;citation_title=FISH mapping in cattle (Bos taurus L.) is not yet out of fashion.;citation_pages=497-499;citation_volume=51;citation_year=2010;citation_issue=4;citation_pmid=21063067" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=N. Diwan;citation_author=P. B. Cregan;citation_title=Automated sizing of fluorescent-labeled simple sequence repeat (SSR) markers to assay genetic variation in soybean.;citation_pages=723-733;citation_volume=95;citation_year=1997;citation_doi=10.1007/s001220050618" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=J. D. Durrant;citation_author=B. W. Gardunia;citation_author=K. D. Livingstone;citation_author=M. R. Stevens;citation_author=E. N. Jellen;citation_title=An algorithm for analyzing linkages affected by heterozygous translocations: QuadMap.;citation_volume=97;citation_year=2006;citation_pmid=16394257;citation_doi=10.1093/jhered/esj002" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. E. Endrizzi;citation_title=Alternate -1 and alternate -2 disjunction in heterozygous reciprocal translocations.;citation_pages=55-60;citation_volume=77;citation_year=1974;citation_pmid=17248653" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. D. Findley;citation_author=S. Cannon;citation_author=K. Varala;citation_author=J. Du;citation_author=J. Ma;citation_title=A fluorescence in situ hybridization system for karyotyping soybean.;citation_pages=727-744;citation_volume=185;citation_year=2010;citation_pmid=20421607;citation_doi=10.1534/genetics.109.113753" />
<meta name="citation_reference" content="citation_journal_title=Nucleus;citation_author=W. Gottschalk;citation_title=The performance of translocation homozygous Pisum lines in comparison with translocation heterozygous plants.;citation_pages=29-34;citation_volume=21;citation_year=1978" />
<meta name="citation_reference" content="citation_title=Botanical Histochemistry.;citation_author=W. A. Jensen;citation_year=1962" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=K. K. Kato;citation_author=R. G. Palmer;citation_title=Molecular mapping of four ovule lethal mutants in soybean.;citation_pages=577-585;citation_volume=108;citation_year=2004;citation_issue=4;citation_pmid=14610643;citation_doi=10.1007/s00122-003-1482-y" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=E. S. Lander;citation_author=P. Green;citation_author=J. Abrahamson;citation_author=A. Barlow;citation_author=M. J. Daly;citation_title=MAPMAKER: an interactive computer package for constructing primary genetic linkage maps of experimental and natural populations.;citation_pages=174-181;citation_volume=1;citation_year=1987;citation_issue=2;citation_pmid=3692487;citation_doi=10.1016/0888-7543(87)90010-3" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. D. Lee;citation_author=J. K. Yu;citation_author=Y. H. Hwang;citation_author=S. Blake;citation_author=Y. S. So;citation_title=Genetic Diversity of Wild Soybean (Glycine soja Sieb. and Zucc.) Accessions from South Korea and Other Countries;citation_pages=606-616;citation_volume=48;citation_year=2008;citation_issue=2;citation_doi=10.2135/cropsci2007.05.0257" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J.-Y. Lin;citation_author=B. Hass-Jacobus;citation_author=P. Sanmiguel;citation_author=J. G. Walling;citation_author=Y. Yuan;citation_title=Molecular and cytogenetic characterization of gene-poor, heterochromatic regions of Glycine max L. Merr.;citation_pages=1221-1230;citation_volume=170;citation_year=2005;citation_pmid=15879505;citation_doi=10.1534/genetics.105.041616" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=K. D. Livingstone;citation_author=G. Churchill;citation_author=M. K. Jahn;citation_title=Linkage mapping in populations with karyotypic rearrangements;citation_pages=423-428;citation_volume=91;citation_year=2000;citation_issue=6;citation_pmid=11218078;citation_doi=10.1093/jhered/91.6.423" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=P.E. McClean;citation_author=S. Mamidi;citation_author=M. McConnell;citation_author=S. Chikara;citation_author=R. Lee;citation_title=Synteny mapping between common bean and soybean reveals extensive blocks of shared loci.;citation_pages=184-184;citation_volume=11;citation_year=2010;citation_pmid=20298570;citation_doi=10.1186/1471-2164-11-184" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. U.S.A.;citation_author=M. Kim;citation_author=S. Lee;citation_author=K. Van;citation_author=T. Kim;citation_author=S. Jeong;citation_title=Whole-genome sequencing and intensive analysis of the undomesticated soybean (Glycine soja Sieb. and Zucc.) genome.;citation_volume=107;citation_year=2010;citation_pmid=21131573;citation_doi=10.1073/pnas.1009526107" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=A. A. Mahama;citation_author=L. M. Deaderick;citation_author=K. Sadanaga;citation_author=K. E. Newhouse;citation_author=R. G. Palmer;citation_title=Cytogenetic analysis of translocations in Soybean;citation_pages=648-653;citation_volume=90;citation_year=1999;citation_issue=6;citation_doi=10.1093/jhered/90.6.648" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=A. A. Mahama;citation_author=R. G. Palmer;citation_title=Translocation Breakpoints in Soybean Classical Genetic Linkage Groups 6 and 8;citation_pages=1602-1609;citation_volume=43;citation_year=2003;citation_issue=5;citation_doi=10.2135/cropsci2003.1602" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=D. M. Nichols;citation_author=W. Lianzheng;citation_author=Y. Pei;citation_author=K. D. Glover;citation_author=B. W. Diers;citation_title=Variability among Chinese Glycine soja and Chinese and North American Soybean Genotypes;citation_pages=1289-1298;citation_volume=47;citation_year=2007;citation_issue=3;citation_doi=10.2135/cropsci2006.09.0605" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=R. G. Palmer;citation_author=H. E. Heer;citation_title=Agronomic characteristics and genetics of a chromosome interchange in soybean.;citation_pages=651-663;citation_volume=33;citation_year=1984;citation_doi=10.1007/BF00021893" />
<meta name="citation_reference" content="citation_title=Soybeans: Improvement, Production, and Uses;citation_title=Qualitative genetics and cytogenetics;citation_author=R. G. Palmer;citation_author=T. C. Kilen;citation_pages=135-209;citation_year=1987" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=R. G. Palmer;citation_author=K. E. Newhouse;citation_author=R. A. Graybosch;citation_author=X. Delannay;citation_title=Chromosome structure of the wild soybean: Accessions from China and the Soviet Union of Glycine soja Sieb. &amp; Zucc.;citation_pages=243-247;citation_volume=78;citation_year=1987;citation_issue=4" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. G. Palmer;citation_author=H. Sun;citation_author=L. M. Zhao;citation_title=Genetics and Cytology of Chromosome Inversions in Soybean Germplasm;citation_pages=683-687;citation_volume=40;citation_year=2000;citation_issue=3" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=S. A. Peters;citation_author=E. Datema;citation_author=D. Szinay;citation_author=M. J. Vanstaveren;citation_author=E. G. Schijlen;citation_title=Solanum lycopersicum cv. Heinz 1706 chromosome 6: distribution and abundance of genes and retrotransposable elements.;citation_pages=857-869;citation_volume=58;citation_year=2009;citation_issue=5;citation_pmid=19207213" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=K. Sadanaga;citation_author=R. L. Grindeland;citation_title=Locating the w1 Locus on the Satellite Chromosome in Soybean;citation_pages=147-151;citation_volume=24;citation_year=1984;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_title=Soybean Genetics Newsletter;citation_author=K. Sadanaga;citation_author=K. E. Newhouse;citation_title=Identifying translocations in soybeans.;citation_pages=129-130;citation_volume=9;citation_year=1982" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Schmutz;citation_author=S. B. Cannon;citation_author=J. A. Schlueter;citation_author=J. Ma;citation_author=T. Mitros;citation_title=Genome sequence of the palaeopolyploid soybean.;citation_pages=178-183;citation_volume=463;citation_year=2010;citation_issue=7278;citation_pmid=20075913;citation_doi=10.1038/nature08670" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=R. C. Shoemaker;citation_author=P. M. Hatfield;citation_author=R. G. Palmer;citation_author=A. G. Atherly;citation_title=Chloroplast DNA variation in the genus Glycine subgenus Soja;citation_pages=26-30;citation_volume=77;citation_year=1986;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=R. J. Singh;citation_author=T. Hymowitz;citation_title=The genomic relationship between Glycine max (L.) Merr. and G. soja Sieb. and Zucc. as revealed by pachytene chromosome analysis.;citation_pages=705-711;citation_volume=76;citation_year=1988;citation_pmid=24232348;citation_doi=10.1007/BF00303516" />
<meta name="citation_reference" content="citation_journal_title=History Sect.;citation_author=B. V. Skvortzow;citation_title=The soybean - wild and cultivated in Eastern Asia. Proc. Manchurian Res. Soc. Publ. Ser. A. Natural History.;citation_pages=1-8;citation_volume=22;citation_year=1927" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=S. M. Stack;citation_author=S. M. Royer;citation_author=L. A. Shearer;citation_author=S. B. Chang;citation_author=J. J. Giovannoni;citation_title=Role of fluorescence in situ hybridization in sequencing the tomato genome.;citation_pages=339-350;citation_volume=124;citation_year=2009;citation_issue=3-4;citation_pmid=19556785;citation_doi=10.1159/000218137" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=K. Swaminathan;citation_author=K. Varala;citation_author=M. E. Hudson;citation_title=Global repeat discovery and estimation of genomic copy number in a large, complex genome using a high-throughput 454 sequence survey.;citation_pages=132-132;citation_volume=8;citation_year=2007;citation_pmid=17524145;citation_doi=10.1186/1471-2164-8-132" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=D. Szinay;citation_author=S. B. Chang;citation_author=L. Khrustaleva;citation_author=S. Peters;citation_author=E. Schijlen;citation_title=High-resolution chromosome mapping of BACs using multi-colour FISH and pooled-BAC FISH as a backbone for sequencing tomato chromosome 6.;citation_pages=627-637;citation_volume=56;citation_year=2008;citation_issue=4;citation_pmid=18643986;citation_doi=10.1111/j.1365-313X.2008.03626.x" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=M. Vahedian;citation_author=L. Shi;citation_author=T. Zhu;citation_author=R. Okimoto;citation_author=K. Danna;citation_title=Genomic organization and evolution of the soybean SB92 satellite sequence.;citation_pages=857-862;citation_volume=29;citation_year=1995;citation_issue=4;citation_pmid=8541510;citation_doi=10.1007/BF00041174" />
<meta name="citation_reference" content="citation_journal_title=Bot. Gaz.;citation_author=C. Veatch;citation_title=Chromosomes of the soy bean.;citation_pages=189;citation_volume=96;citation_year=1934" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=R. E. Voorrips;citation_title=MapChart: Software for the Graphical Presentation of Linkage Maps and QTLs;citation_pages=77-78;citation_volume=93;citation_year=2002;citation_issue=1;citation_pmid=12011185;citation_doi=10.1093/jhered/93.1.77" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. F. Williams;citation_title=A spermatogonial map of chromosome 3 of Drosophila melanogaster.;citation_pages=131-131;citation_volume=33;citation_year=1948;citation_issue=1;citation_pmid=18903882" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Fluorescence In Situ Hybridization–Based Karyotyping of Soybean Translocation Lines" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2.cover.gif" />
<meta name="twitter:description" content="Soybean ( Glycine max [L.] Merr.) is a major crop species and, therefore, a major target of genomic and genetic research. However, in contrast to other plant species, relatively few chromosomal aberrations have been identified and characterized in soybean. This is due in part to the difficulty of cytogenetic analysis of its small, morphologically homogeneous chromosomes. The recent development of a fluorescence in situ hybridization –based karyotyping system for soybean has enabled our characterization of most of the chromosomal translocation lines identified to date. Utilizing genetic data from existing translocation studies in soybean, we identified the chromosomes and approximate breakpoints involved in five translocation lines." />
<meta name="og-title" property="og:title" content="Fluorescence In Situ Hybridization–Based Karyotyping of Soybean Translocation Lines" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/1/2/117" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Soybean ( Glycine max [L.] Merr.) is a major crop species and, therefore, a major target of genomic and genetic research. However, in contrast to other plant species, relatively few chromosomal aberrations have been identified and characterized in soybean. This is due in part to the difficulty of cytogenetic analysis of its small, morphologically homogeneous chromosomes. The recent development of a fluorescence in situ hybridization –based karyotyping system for soybean has enabled our characterization of most of the chromosomal translocation lines identified to date. Utilizing genetic data from existing translocation studies in soybean, we identified the chromosomes and approximate breakpoints involved in five translocation lines." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/2.cover.gif" />
    <title>Fluorescence In Situ Hybridization–Based Karyotyping of Soybean Translocation Lines | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"tuzYj2Pa233ePALGVCtRQuItdmgjypPy4iNPdDe2j34","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459165","apath":"\/ggg\/1\/2\/117.atom","pisa":"ggg;1\/2\/117","ac":{"\/ggg\/1\/2\/117.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/1\/2\/117.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;1\/2\/117"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;1\/2\/117"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/1\/2\/117":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00225KEeXsKy1VlMpYUFbUHAZs+Jw\\\\\\\/Kqspz5RhlEgputfNmBVNiWxWsqNpS15tQ+iTa\\\\\\\/RC7PK03XzdpebWfTSBsHsGo4jociJno0BuOOrBqc9eJI60NzQkxfTcIf5U3CF4YKw45qeSmJHIRkw5NZXp7R+H9GM9ZFkvSsgsss+4kO1Rnm6HBp\\\\\\\/Va+Ik9+yaWn4aS2+Db3Cl0xxtwFrCkvR0DqCiVqSRZPkCqJV1LoEq\\\\\\\/rMQEqjRX+PHV8kKUUcUb7IG9gUQN3DHces4NxSXhVsGfU4+n+kCNpL3vWhXcFfR7ZiaORW7GNV7QLHr7BTDzN1\\\\\\\/xswEEss0sXFNHRy9OHsjqtvExbh+xgMA\\\\\\\/1aYkoDrL02ZfUmGIdNqbw6G0jAzEm5N7ALz3nLAXGkb+p+\\\\\\\/UevV7D9yC\\\\\\\/lB24lYsIVsPzlrDvogVvzqoA+w\\\\\\\/5MAigmyKrVeBiKSrWNaxzUS8LwIPr++T9mI8Cu8C2mvm5p4jXHAL3bibBmwEblm6WNH2YXHF7msmxfd3o84Bcfl\\\\\\\/fNb3JLTzXdZsxYszYq4Tl64USht1f4G7fupVHVM+Vr\\\\\\\/tVKqEx7zptQ3hUG3uOgHcxAfj7mOo8CeamZinVkHmHG60qeKhz8KKhRRstpOYb5NDVP3vkp53qgT+tHG+9VirnhQIg8OBpamSE\\\\\\\/0tvLvq0WYyr9wwRRBlbPoX+LKVBLBRaP7X882gWkWWnINfgsRLPWYcQnXuU5NNoT2CVWE3UT1jSz5z4Cs7wLUEKbDWevANKRPA722dNQV8vBBhbd4NB1bwDci6GPrzZx59W2WLPvfR\\\\\\\/FahIL9IKQqkMr\\\\\\\/phgkdX5RX9jD9guSJgR7ezjfUwsLl6T+acvQtykoS+elfIDQH6P0WSc\\\\\\\/TLbv9EqXZgK2nVoNwnGT6j\\\\\\\/R4oZ59GTUnvMUhPpYlaQi+h0eMG23\\\\\\\/pRd2I2DCFw0tvj6BJxNEbdq4KHPy\\\\\\\/od5JkCniNHZ2hB17YQEF7ZWtrYYlsQPwybF5RY3CriPm8mkJMDvUcZS4VsUw\\\\\\\/hL1rSnQ6Aw2HGsaRvIXsHK8OctlxBNl9SSMGA19zz9kPBsUWR5dqzITNskPwv7i3P4DW7K00S+lmXwZXOxrZNBtijt6R9hGIRtu6HiGf8vB\\\\\\\/fQgZtrGv4Nyf81nHmMElkhVSxZSeU8jYbMpFzGyvmPiKB\\\\\\\/zK1vXqpKCIus75yunYr4SHxantTnb5\\\\\\\/nhAua7V1m\\\\\\\/Fc6BCepHIHFGH89zibLMUSIMRFI6VW9uz6a4ZxCvHvn3Ij\\\\\\\/JyIPe2Bk13htyCruCpbP4\\\u0022,\\\u0022iv\\\u0022:\\\u0022TpvzIKpFDP7x2dAc4IeS5w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a00d1f17fb3742946e6974e74177b37a9f63d605d9b73ff9fdfe7927077f3af2\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022YgX60PyUvzWrX3h+c5684DcyJigDhuRod3hHbXpMkXAudLOldhSIyR30S6QfkpXN1AGxvNwhI\\\\\\\/\\\\\\\/IFamL3swqX2+kWJSHexkek2FdbM4oBwOi75vOc16bKiX0z8FlrY08CQk8QtY8Os8JfZ8Ff559zytUzZau+YNbnCDC1xlYMk\\\\\\\/o4XrSfCcNtN9TqITIuu5kdsbcpfxfqdSA+pfb5NWT7DTx\\\\\\\/ww03v2hkl\\\\\\\/vfVR+he8CkZUlDLoBh819qEwrZ9Mq68Q0Dsb0RNH5p62adWFCsRCH5BQG65kX45wnrTkBvR+aoScTDTcpIUqNx\\\\\\\/vVgwj4kMkuSa9pWRQVCyhrgmTTHPW0t8PGdkmMLEgLsvk6Aelr2dHMzK3vSVJ+R2Oj4YPhjBbEyz6unBwBdu9nEVXY0g1jYZLVpuECFcv1JbSxOBa4v7WrCT0CQN6EZI+lRDaWFDnZT9RzvqlKnqVI4BK3jnn1HQ+ZGtZZgcF4OZZ6MUb7jv6O8QP+tBFqICoCLjyledQY5QEJBK5aARZ464CmJwcTsIhCH1EwHspTf4EqCJZ3U3tdN2ERG\\\\\\\/hFMh9Y0Fp+YavedbOrb5cPEvD5CCEXSHqIaQwl8Wf0oMCpi4VYBjdDSbPvYfFNw+2u\\\\\\\/GWMX3QbPfppGccQN\\\\\\\/wQyF8YHiL+Mt6A7TfeG+nIhnOKZcZxTkJxoO3crMHGZAU7ZqgaeCotkIpc1NadW4Fs4RoWPzjO3f17nflC2Besv\\\\\\\/1LsDVIXlzLZhz0hVNk0uboPNwrrnuT8HVblGD3WOt\\\\\\\/CIfGKms7csQpWAS1EsVxgAcuDZGyoseFTYsQYbwMbtD3BCmR5uYCFS7lhGOmQC6pqsfBuPgRvc+EekZkdylMKSGyj5rrbydq+bcIsxW13e+FxihR3p\\\\\\\/ZQUAxtKa4p5dDQgP1C+7nO3UJ1dV0W\\\\\\\/1H2lo6srBdqp0J\\\\\\\/k3B34Krn4N2rMlL94vNeX3dbRmw0dteLcDdNdM8vEbv764\\\\\\\/3cg8Ndl41kahjESiLSDrtwXAjFaCgTdG+A\\\\\\\/cG90T4+A1fcr4IhfRtsNslvTTilwpXa\\\\\\\/h7uAC748AIrJhc\\\\\\\/f0PKHE9xvS75tUkFwDWialptikUHwCcKaGZFnm\\\\\\\/illiRvfGMUTNwHIsNTixv9WS99iIXPpPsz71urnFIFSVeWbQyXcwBKmK\\\\\\\/JtCRwY8ZEZANxPl4kNJo9btGxqfAL\\\\\\\/mvj\\\\\\\/sC79FQ1edzX5szWSxcED7MYUrMP4P455TZ3DazZLuiUCuAzxdQKSCneUO44aGMcT7vN3NAUjJhEPW97wsZeX4\\\\\\\/287Pif74LoWoMBdL4EVsihNIXssygfWxQtbtkz93GjZ+W2pDFH0fh92g79ytB7dhw0PnNPP5Tj3xLO63cfUx4fWKP2HkjNeLGlxWuLeMyNdvPWCklo6Eo9+iP8atgHYhm\\\\\\\/Cbh8VpRksjUbzVQ5gf5srhTqgyg1KlS+OlQe2m7sTXLCqtrZpAEOge68gZQq05zhH6F4b2u\\\\\\\/KbkMBL841j+IjNF1E7b94j5mzDIEUJMTB0jmVT8EHNstWiVZ8TUMzPNHg1\\\\\\\/XcZKYmvsNaLM\\\\\\\/wCNgeJJxR7LxhfjoBeSbUa5Jrmri1JHgIYvxVzhWV47a0UoyA44wSvN+0uoh67tjy6\\\\\\\/3O1P7hWAyxjEpVh05DI9Mn9BfXUZtQk2tCpi\\\\\\\/YPvmzZN1Q6pXuxpdbiWlLT5lSuxgPY6P4bHEHwFT4Y\\\\\\\/omkug6aCd5BdNdaIOtJ3Wcxz4XGdd7GF7yXjbMSz2vJabo8O4ztmQK6GVVzVM4+U8ydMS8WkvYYr7Gg0f7wu2T+Vr+m65aUOtOA+O0Mx8VFVXc5nocSndYQM3nMymgW8deysObcfzZjDuUo513wcDginCrDAgTpIL6d1Ir5u1OR\\\\\\\/I4\\\\\\\/0BwS7amPvAzClcv4utGBX4\\\\\\\/V6QPRIVazZqKyG4WLJmNFdhqISGEhlTS9UJFQaz5eaxUN+MKFjAniWILRm+9+jmDrvTykNNyqQeGSjlW68SZ8d2\\\\\\\/W\\\\\\\/ADkeSV7hk1f\\\\\\\/VjzJXawa5z5T3pHLoJBkNjTPwLW6vVc1wm1NYK5LWPaBIMo5hji9zg2ynmtsWVprlOhzvDqKAfkZAigpovj1HvUG1KP2FwUj81JTMC49HiY9fXjOY\\\\\\\/8LDxH50m9Hdi7+71z6IHZkRzG43rIHTrPOF9Zyw01V9mpZ2j0TkNGO5p1QkT9N7RKJE7qiuxXev1cfYqPzziLRDa22sreiGin7H5klNhO\\\\\\\/hh7QaPbuCReY330oZucJojWHKu9CrDpk0A96YMTYV8STbtYGa2vEv7O5+R5eQ1TDpTM7M5CnTdTISoAf0HSGrmNluoOw0NAEvYdvTuGSjwuqzpRXRD53qZReUJQ7Rmy5A8P2wgTSStLYN+D66Wl9k+pf+Bcw+LmI\\\\\\\/r3Z9E+6iNuH6B3jnNHQ+gM\\\\\\\/2pYTkWC\\\\\\\/k1QPWWSjmVH1wXVkVH2QMGJC\\\\\\\/ZD6IWIIW6B4AdZDG5\\\\\\\/Tv2uDvfZYHeoSV6057tvvirTAfSt4kq0tHPGCHR9JJ49dluvxsnL93fpb6Ufk3FJ3AAVXYqOAXOwfKR5DcX9AFk68DoEG1TCG5Mocigdq8o2JvYMunsd\\\\\\\/HUgmcei3VPbsQwfXJtQVRi6JuUWEW35sDle3i\\\\\\\/LBTkGGlRGQa1dku06vSZRAe\\\\\\\/lIpcG70V7hbnTDldnbg+L0VGwTL9RmuUT1xLTkqOCeDGfGvEnFTueMlh9TSSsTTDpTUYmJXV4BvSYml6ATiVXz7EEpSianaGkD68+O5wmKQWzIABjE4A\\\\\\\/UglTLkP\\\\\\\/WxYbAXfyIptcxBgeNuWG6e7rYDUFLBikO0HMH+qqGTXC6uU\\\\\\\/MnIFVDkMaIvMmX99mRyo2aFh61bNawLClj\\\\\\\/vLHiXbEta5Uxnj4ondhqov78h5fejTWKvYoEYYSAMWuyeuialtyoYWbjP1cenLfINtl77YxnZ928Nesv62tEpAUgjn2rr285RN\\\\\\\/x6t2g7hfDdlavedV+eYjQdFl935Wh9s8KPi97RI80ljJlO\\\\\\\/0PU+hy\\\\\\\/PvDwZ11\\\\\\\/Y9ID6L8UHmvj+aQHssMR15edYpH0ZLUJElwWtJ0dVTE+vIERmXfPDsB36kIFl2gRZxgcwldEilK94kVJkfL2ovefuwGBCaM2S8Ug8t19\\\\\\\/MnsmOHKSWn\\\\\\\/fuHJzi6p2Uv59x2ljDkiMS8Y7adHsbK\\\\\\\/+VHXD0JJp1Cia+3BvROWfhtrYFdtH0\\\\\\\/0zJlU1HcKi0rmnIvSmreN0EHsh8x20i9F3B+7uW1LTo5KMwR6I4lUzrqqUIsdTckXTcJI\\\\\\\/6yBLG60h7SdEuy2AK9HY5094ORQkzwk1K8xRsyj\\\\\\\/uKCPA6mNzD8FgRiqsd6lqbzcSHbsjLs6fUKxpEo0nm0OqdeIIENW1xw5xUFGzp\\\\\\\/GJ1M1l5TM946R6OEjdd59NfKWSEeQpEJ7RsYij4IK4WsUhIIL1OsU6TtjHsGzW0j\\\\\\\/lP1b65KjHWGfjozJxnoktXwAQ36iiLyRhBLBmuGMIP5dF2bLyWa\\\\\\\/UXSs+916Q76g6AAQtfBICbeLOEe4DM7NBcX48QLrkiehNP7jbE1Ck28PjGijOEP\\\\\\\/PKC1zpqK3hFNRc+KA5UPG0Wuok85gp16Pd8+Cul8S+HGKdW8dljWe33CDVGqbmBir3ruMw+\\\\\\\/eIvnN080QEZJUgf+hWJDnaJyZIACIKXBuaaC8WdMHraYvnUTTBLtDAqjxJ+VppLE3jZuhDlXMk8FZBxpEpRmwXhoa8LIRZNbD5wt5XwC3NtyFfKLNP6fwZKpFcHKtCtozsWUrhM3GW6QfGL4beS5QWW6GefjsYgfcLkkPz7xuk2cD6NsNKZbrITmE5H39K0dtAVWRC3Bfdy4B294nzTezH1Rer1sYINyw2WaWh48t0Xr85S1VjX5RCDdGySD9xd948XDL+N+6miUIdP0gaR7KR4EbDPj5AC0xp1qwQwK7\\\\\\\/2E9jGdV0+uptNNDtHj3GMjwmdg0I4RfEYmtbOt8Cs38Q8xLqiHcaY86+NWpLM0EPcrfSKLk9lP83kkV6d7ivGxQpp1MH9geEbDR51ZN8bJ56XruRZqMy9KYC5JjJMKckzOAdVPRj05\\\\\\\/fTL5sXbnvUGNCZ9epduVQtshAZ6IU\\\\\\\/KfzqUcPXF8vc0t8m3mXENw8LPpB3ZJdT\\\\\\\/qkePd+kV7RruGzkF\\\\\\\/1Q0X5v0bG3Ogk9DKOs4C2TDCh5fQRGFO1hpKguNK+JrlM7VLe02ijI1sgt4Yxw0XjZqCJeCTAPs2DJSOV7ULYJ6a\\\\\\\/BheBIRRnDU4r2sLu+RUsHbkAVtSb\\\\\\\/tXh1h8xQrugCWWeN5gD\\\\\\\/Du0AfBAlgcQV12C2Zzq5TNk8BPwWBxTBW0C289bNuAO5dy6S9wlCPJz7l2EmWI5ShWPkQHjucDlN9+lduCxNsShZE6t9Rw0qZ+ubfs\\\\\\\/38+L7B0ljxt4nLsUsiY\\\\\\\/PPS8qqf6yeg388KITcM607XtqWsaeEUwmUMqYFauS5vpyWsa\\\\\\\/OqNh3CHSyQZ1zdSi3YmG3DGLxbIvKMKzbTjlBUmRjlzcyMCl72wNOBmpOscEKjbaE5AdKZmlhrSI6BGKdthwvVYDXvIfQ7+5YTjlzW9N4jQ5Y4nsiIGZUe+9ZlwYfEBGLudYd7KJCwbTnlSW0rvQ82Y1F185f0DjBaMHX3\\\\\\\/YA7oPWsGmNNT5Ox0ZCjC2hDT82FbpcX1lXkIDhsszcOyeMmTtd8xDvf\\\\\\\/ViiR5VMLqWTbpfnl1rSnmvaj2ugYCGkt+LpVJmDpM3OAcwG0Iy9KSVJ+iPCC8s7W0e41yy\\\\\\\/VEZiJ7RBmKzRbzKKGSI0DoGMBMjvL3aZTyVp+bB\\\\\\\/6TAMnfC\\\\\\\/oHLFPrZXIaN7jb0zYXODpVbel9SoLR+RW\\\\\\\/9pNNk5GyPMF9gYCCDNoNeBqUojxpadCzo94gyXsFgSNyTJdkpqx\\\\\\\/rgShXCB6ly3Mi4bgSkY95UVkcPFl1w7UFTiMeHhw+RQi+NIqdy3EJ71+FRrKgQV5168z37xzCR9u9XB83zsJXhGwUC0uJQPtZgC3W8nFJByz5rsGLBoFTrXsa5A6AwHdz8rJ+g9xsLcgVn91YRiI1tOLQbrM77OQGjcQEe99CCW3CPB6vJM1qxoLefIoOIhs+A9uIc418T2KtNESwYqOm4M2azqKbAMTMuVQxVI9mPceKgJ3M\\\\\\\/5uAMuQn+rad+QeuPWKvySWtpvomB71uOQcDi1AqzG\\\\\\\/4QRuera2RHxPb1yat\\\\\\\/h8nVbva7eMJ5xYQ5Dg79uLyFN4NtE4qrDije9lZvtSQwko4sX38mbOTRuvGWf0scTkDM2nkOcyR0YgYVY8GmBAcQuI\\\\\\\/mparag7ToJ5uChUMHRpn\\\\\\\/V3vh1qckSDk6HqInSKz7wyEQ3m0YNOH2S40j3ZBWBIsNLB0y9M\\\\\\\/alF9wdr0WY\\\\\\\/TiH37mckPwq\\\\\\\/0\\\\\\\/DlxkYlgFHviOETYYP4Cebi5qTt6JMpWp5\\\\\\\/3priBWqmssNqnivaLuPUCbEKTnbkPMwtppbRSgLYqLZoaUrn4YcSGg497+Zo4Koyzo5Xup+k61VBwe69bhs7aPbqHACP8EChxWu\\\\\\\/3IIsza7XbtNRMnG0JNAuKqGce3wlDJg1WxwPwWwS\\\\\\\/NrLcrLLwyzdVwDE\\\\\\\/epWYg6KxvSrm+1f40ew\\\\\\\/u8CYTD9o09K\\\\\\\/LAKdCTHVN0owapk4UzCV3kp9us35ylsRs3D6hepZB\\\\\\\/0YmlJnbEGGRtD8po\\\\\\\/gjEK0iE3sE14CJGU6e0b0ouPDIL9WhyCE0jPgdbCcKAuqO2YqCvOhlgQ4snTEa7Wyay7K\\\\\\\/bVnmxnHkbK3etubw6WKgnkbX3mIj1J\\\\\\\/kwlq5B2JX4kI9MDw\\\\\\\/PCICYshwuZVn\\\\\\\/aGwGyS4iQAz0cCtH6WlEZDc7tRC77xbjnWKYThHoMMY9ovaB2RpTdQ5z+JBUQb0PU2Fu+S3E1Y45XzpoUWVvtJk5OlQkyYhU\\\\\\\/5\\\\\\\/ZX+M5Iqr\\\\\\\/NEKGxgbaAhVjo1yc0ilvwto8zjCe8evC3Xmu5Z\\\\\\\/tOV0llphvBNCtmQI34KMOI5mUiMksuU7f5cPYrw2m2JUcEKjnhHsxfi7lLiMx9TsCbpvnUzKXwohDj22pnhD7vBifxo0Wjv9R9Ezu8GAbr3YALDpWFanmdwQbtjxT\\\\\\\/MoRjZemmJ\\\\\\\/a3QiCUlUeHbcwfX9RVss\\\\\\\/MH6PBNfSXnyCKZ\\\\\\\/yj8f+dvLF4GMRSMHv78Y8XUdDUIwltbAjy6gYLowl4oJBXQilSipHBZJHY2bGcbmABeIFZXXCHZZZoeBldqBQrgabAWDF7TE3\\\\\\\/QF1JWqbdYtjBx2OyVNYLgeVPDJENiUqh8ZmKrDihbpUMaSWyGGtf+gfx52oIuX7GQor5dzyn3GdgUiM+Z\\\\\\\/a25udycDWFxjYPWnrKLCDTfcb5IfEonDOeNlPtROBRiW5cRJTR6wWE8zTugNwCo7Dcef1DKvwdFakkIoeFbiN3CCVhM\\\\\\\/LtG5RxorTLBUB1WF4NxAnIqu6MC5jHzOEFaptqVpuFrwMcU3M0\\\\\\\/caCPhnVUPMUEIiXT2CprHobubMKh78LPsUSgom8xpQyT7wlV53D\\\\\\\/v7ykRML097U6eqOC42BIsPmkkADDHsW66+2pG6KlH1c2T1OayQuEhb\\\\\\\/LkiFJMi+A6Q3pFc8uZTKEobJ014OSu+0Tzem5sDN422OpoR\\\\\\\/ey3FguxSP8CoVaR7ggOZMDe7plqec8FF\\\\\\\/eNhl9ri5Lp1lC85nblpCRnduKNdpr4pFthrHRaPScZyYswsOyhtuR9MOlHM7pSrdUFt2GRIIFAPcSZPQ3CSF+iwA6y3LWJh+HVvQ6MjQ+ZcqKzdsz47klTFSot9HdEFdOQWx0v5yMG1e0ioyjyhefeGiWhyXE6FhXwbOQKp8YBuhXmoaasdcONuqAQ2UBqQY0scZKhEo6RLzI6X7W15awHwc4xWadKm7aF3AuflRNm\\\\\\\/zu2V6Ub+FS5hbsqsvz5\\\\\\\/04cV2KPa13FPiBeHbM0iwUOUTO\\\\\\\/tPAmiX0sHdVgUQ7dC9H3972EgIdB0LFa6LCvUJo2pThIKaExiLfrtC+csVpxefVR2eSZJF65mlBFgGiA2EyKy8P5V+halb1PZPfIzAPq5p6Z\\\\\\\/7VmAaadqStl0qgbVmJBXGM0LQJdx+8PVI8HdWQGPUHUYWleuMaJwBq5AmgKhKZQI+S3Y+JQBK+rmn+ujKxsGrQfjkbVnWv2evfXGG6k1ew9ISI4f9GHHpq7cj1n+BrCXtjkKsxR6X23gpcEf1QP53n0Kgc3z1QgNiRYryUUasaMY94W5DUvGBAGFOmwjzUeSlOWxE2pLQWgNzWEonygDcmjiZV3zTjfC4q8cRenXnf7NvpuR6mSN86uFnS9ZlEznzpYAEEIiAjqZyjAItx5D3rR4262ja3sTHl7FTmsA9pUA6R1bvTnVs+41oqc78zs7NWfHnJwxAk2nhGrYzJuRMncWOZJNE5iX5DrqkmFnP9UYJe17EPxeTRKp7GEhFLJDAbrx5K0olfOTQigMZ8QFrIMOrj08PkIxgRzcj31IGpZL7vdh82EjKboFbi6g0EYHQR8lSF6X97h8krsnvKfpxh8m+Mg\\\\\\\/fKnV8ZU6WvawEMw3hauDIWxSRnRIL5JNxa9wUI7CjocjIqNd+HKYgBiySGDi+nTAe5XV6AvoqGSXrHbPZykJUXEJy9FtQypifxYQJ1WvQ+9bl1oGv2Evar+0V77CUN9VB+rKoDmzPfYcjPe9LehNEy5sdRLU5B7VKjT5zgmP9AGHQsl79cVsaRRNgJTnpruaDzoOTLgGcYOoXM97TmE7GasbSzq6SlZrphk7ZkdrmukfacqR31pfUylDYkQZGZT1GRdNWNLzeGoUPytXRIdgIIpj8v+sqh+WBG53ZLKuhixSD9dQC4q7eIZBk03QORqY7pl9kvfMKpXed+AGQ7Q5wpPrCdympSrY0gtKPBKdSQUjDkr6adBUIveAqfMV\\\\\\\/jvP5yu\\\\\\\/x+s5nlopoInFTYHoQMWm\\\\\\\/dhtGsBH0FxhogiSNYuxvXzzLBMmfZFFH2yjFDvw3tyglbFmnAwNPk5ycZ83aSnO23Ky52t1B6H6vShHuC8fjNB8+5dZSelHS058rt+4uDXibwZBW4e3qQGc9\\\\\\\/nMByG4uUYWDmHbReJd0GFuUnK5b+jgSfuUMrDS0gEemBn2h2DMj+I9xJreOoBmVUuRM5ezbTLemziMpFqBktVpRrbyVw5e+wmBMbMkFdqd4gX4AAUR6PZuYPWZgNjgUTBfcftgfiWj3VkqTIynfUP8Qb6N92d2njuO51lSWEMC5LMtg+Y1yjJWeMvhW9C0tGK\\\\\\\/k0x0pS3taZG+fJJJj7Aihrw3TOyhqtLLGoiamT5Zw8m\\\\\\\/e5oggo6W44IjAZR2I2gJiL6gvUmzyXBiUq2\\\\\\\/TIeF\\\\\\\/A\\\\\\\/vZb3tLyTMINrh9Y86I\\\\\\\/YoG2ws7KNP+Uw7AQztLXFGvWgYB5QWfICW6+nyaddP+GVCauJw40ZNauhDMaYme1ovCre\\\\\\\/UFSlGrdy9LMDhckTyHx1DxuhXDKwMw187ygMmkPIZBa\\\\\\\/BXa464v7F\\\\\\\/B6ZeuPVTYgUuLMoC3+ZXCmadb31p8xSwW2e+MsQRNdgU1GonRkASYzjvEVPhEwTmHp9WeRe2pXb+LIsAxhCvmcSRyCOrukgRCT48UJBUQm\\\\\\\/X1VeB9Yf7X03w+rApl+0Njn9hpjxhrn7NLbjKsYhIaFSQQwPX43cqOlR7rN+tDQbgv8TVeAD95wSlUbrGaxa2uHyDrRbAypDSG\\\\\\\/oAcmmQXS8otR2yqOnpl3ro0P0N5oTi0VdKTPAcX1PyZ57VdpV4ZZ5PUzSh8xRsqSuYn+QgBVBXzbGweJ9f1i8IBasaN\\\\\\\/ebpHEHZ2p8zM0mXXTHTkTgDszQ4RnlhJDwE57a7F4gfifRvagjAo4Igvbzx7jcS7xgFUzT\\\\\\\/jLJseiW8Got6Rlv77tl3O7dcmOFEf1ElW+vL9EZ\\\\\\\/31aJ2w5fHCREs5eXROlPlI5JaiP42XxQMqJAgcErk2sdmtMWMC0TZ8dnHFNppUNClpQL\\\\\\\/twhQ2dWObBeg8LTVmxJUxv9yjb9\\\\\\\/X3fAezM+bL1l5PplEp2d9qAp7pXrehJtPBZsN\\\\\\\/x2NkHnGlf0BMs2aEhcHpX\\\\\\\/ir8XCKoOR73bfaSCrRE\\\\\\\/SIFHebhomzyG+824rCg01kPH6dOmbE6m4Kpn9zLJiA62ofLrYnuBdeAjsAHBQEDP2yCsDYD+OuwYBVGCqeYSRuhpyHp4k3FuA2TizGQOtqwx1k5Lp2BFhXiVHp3r6uVvv\\\\\\\/UVO+WzA9B6qw2FVXYfYSKkaAQlzgW7nJ4pNzd+0TBrePlT7iRQkTyUgtrapZhmFI6de1tVuJvFsH2InkHU7W8NgK24\\\\\\\/lGaut\\\\\\\/XekW85xAKPM\\\\\\\/UCn9pAcb+ALe9ILCE+SVeTrkR2tQyTvnHJ5ZthyNMwR46gQvA+E8Cu8hxQUBmm4G+Waogugwip7jAfrYwQWDi\\\\\\\/RiXlK8d4ztRD0New17Y1elq0yJd2yOzaTtJPYlStb2wzlJKBNQ8ol2ZqyTWQgbJBjrIYyhDWqvda3FB28wBJ5UdNPQZbAxP8c8EoRcCusb2lF4AyhnvD9\\\\\\\/2EfuFq8GH8L1rP8mhCdvG2K0UdTnaXgPciKuMlz4uJF4X0X\\\\\\\/zaGwM5jneojZ44BNsYNcbjRmKzmFvVyT0oySdnOBx1KAXJ+Ojw3NIgg4Oi6dGVa2DIBWShWKUIT8ICWe+4kXNrznR99773Pg2pQuktO5fxTCf0lAhcwEqVN8dGxIPUx0Uh9r\\\\\\\/VubOrFWZ+0ey+jI5\\\\\\\/BkgUMvRjEFbmxc0TGFk7oOzsiKbk51l64jT1e43rYoKYrf\\\\\\\/hwThHLqCYNEAKlGa9ziQhvvlsZXjxHWPruDC6AOpSpbxLL\\\\\\\/JTt0fhnfsJwN42kvqATB+E9qb4xkvPMlkz09IZQylUxDc9\\\\\\\/MR9ZH4LbsgPho0F+HKAqrWFpfqlhfB+d2jU+VSTf7OSpHwnGDyglsqvMzqwlZFqUP6O8\\\\\\\/R9lejCn9eYRPAA0ToAfiv+6QINuHkWB1mAbCBqfu9\\\\\\\/T5jqG4DFk8YOKOiKD7BN92Om4SKay9Vx6Ei1RdUvDiDa6DNm5aU2K1281rRqF2O6ljdhI\\\\\\\/2bpxTYZ3le4B\\\\\\\/d8WC0Oc398ana09QkqSwP4M8uW8Aq5wtR3uuwtJ1ohAh2XI2hfFlNQJzZlIeERjZfXadCtfcI0J5gfanD5dmv02bqow+IDOw4g+Pi7k+yKE+0MLs0pChgUTGg7OmTBZoIyjOGyP1T4oFG69Dsf8FW\\\\\\\/qjo9HScTu3OXFEgJG4Hs7FhSHIGEzf0UAskQUnnU5A4Z3Qqy6U38r33\\\\\\\/WjGBqTNMwf9Fp0u2fVVh\\\\\\\/IQBZQUq82zSoUAvNXy4f9Ghnaq4PKNtDpGN+ZRPOJgp2loVyK+6yneunV8yQqjLMjBJ6ueAzK8w2WajEkPCxe1+SddYPNJa\\\\\\\/xCr6oXDatSpqsq3g7pjhIIOOPJ0Y2oJPrESvHLJxsJjn79Flx5v2OMkhEUwOE87RCgb\\\\\\\/kyvNEVgAYJx4uZ7+qFdgTyKVO2Ww3qA\\\\\\\/8H4cX1lUH07V6OU3CIJawr1KapqoDQgfWGrsfLymhnG0IL0JDkt9YnK0PgvO0f1BWK9tW07hPOnRkn687AQf1arWxkswU3\\\\\\\/ngOaPnycJ\\\\\\\/7dlM\\\\\\\/tipEelXlqazvtL829LJ9bcOGhbsepewylHz6xMKG+b7uOaPwdFJnmuqfOyfX2kfWna5vvw0IiRTFc6JtBi5LpGj\\\\\\\/+2lxR1BE+RoSpUg+tp5Uhsy2J7lVjlEw1udCQHHl+GvdlC7dwos8Bw936ZqAl4mz2+M7zMCpiOOdDIJtX4kLI3JVS0PIXghYA8ZQT5WFqfTexbFaF5bd2ha28MpUYoHXwtvE9gxbkmDptFpsSk5KMTOCTjIcpPpPcDTzxZmrTauGpufn1c8kE+iJkTmjyk0RmbaYuFbAjBlwRL6oxiwJDQm1FgeU17EosfROqQtT3hXHQS9b2wSMpSZwraOkWX1N6Lqt8xAcm8DHn0PpDfoKUjN7n6o5h8Igl\\\\\\\/LBqY1z3MDpVi4MhBFBTVuiECY7IPYNTaAuSmleXSqozy9wtEKlKuBogQhvOx3Quzx6AesR0c4a6GJ+twrjKafN0y61PJmmDFPTFuJeC+AQ7mS\\\\\\\/Fb1WznAX4yU3ML6SG15naz7+9l9lI5EO8aFgtsSKt\\\\\\\/tcMt4T6LKYfzyn6s0vwVwgdlzfrgiS3KzDdRtbtnaRyUWcTGRANxLYxdufVExTh+cGgFspkT2EdfOWn9RPcmngo2tcu062u+UGxqi84ucEzFJIJd8BZzNvUjo8on+yKvmmu6fmQwVKlCBb+Jf9Jy9pjpHfmL7RP+CPu1ppbccLMofhGx34TdANfecTFevh7PEQjXULM7dQvVnQBXpS8kIgPlzjjzwagHdGi1rH9GTLcWbo4vKskWJnpzhHYaetHKbhEvpbcxV7ZCTZhW1BTzFZP+beFeCmsXhZNdIalnFjKN+zLNWJw9B4pEV3HzuQQqlp70FtexxikPvW50iGYh8IKKHW8oZHor4l0daintZPrlMsHX12tlN2UD8o\\\\\\\/0aIe7Jv7a4KrVuP52O8mfciOTG16xPQ0aOLrwWNNg3Ntulvqrf1dmfwJVuAbSIBjfOLIyubPaphQLehE9Qy1EmI+UnCSFJBzxX81Bh6mmVf1weQwzP75ssNDGYJfcR4j4haYMZQY6DYfJrrrBpKJCjV\\\\\\\/IrMywaHhfcQBwxliKofPRHs1zY7kxClWWIV8LVU7ejtYKEtBw1xLjIclefRQPOSVfukWxyqFIPePUxhZwpX7yxzfb9p\\\\\\\/Q5g2+wTK0KTGlBwn8A9ZIPng2+KqufEDsoEMaZ2EXP8yXtWaMq3C68z5kd4zlTKPvMKmjpJgBvdtFOaq19ndc9Rfsfwl7uZi0kwfU1Gn9GmdMrxSjK0CFPgIax2CcmSln6+GkCFapXwyeEni6E0M7c4PTrOX23aRiHSPQoZUC7OhYQv8JWCifTj4GTkqdSoYyTi1uw+fGGW\\\\\\\/Dja9bUeUDV0+Efr8lkTo8bhv\\\\\\\/7B89RNrvViOriNZFsgMnO\\\\\\\/U19WM0vG\\\\\\\/3T2Fu57JmI\\\\\\\/dKtHpBDk21raGwqC8b5THawSdyWBStN7w6UdZ1LZSH\\\\\\\/BEwtHV6NcFTg74JtO9wGdu8mlqRaCuZyXB5m6coebxGNJ8Y3D4VZII65SI7Pd0IN95uESxalr6TJxPJ23DQtPAMp3NzHk\\\\\\\/Ev1vmjRR+L7rb5n0USpnBrU8NnsIZUiHp9GtWWAVnFDZcFnmLN7zyBen3D6dQmzsLrN8M0tpcxtmLKMpPVQBHjk563Bv5eLkQF1f\\\\\\\/VsLthPQ7vw9sWnDF155ba1xt202oeRS1CWfqvLsnSd6u0Ywa6HwyyQYitwDPel7ujsZoh3emLiSmjfrk2UuHzdF0VMzhB6SoE4\\\\\\\/2qAAhGMMI5k1V\\\\\\\/w1ZXx1FxYV2CNYOlXcxktbIhqLjQtzWWy12QFqTY6xbFrcRmyAC4mVIOWJgK\\\\\\\/Pf2Z\\\\\\\/spN7LMZhD597R3AfX01Y8xnc9cmkVTeJLC40u9vpkbuS+SayV5fNzkFZqOlv6uqCJfb33\\\\\\\/nddAXdFSbH\\\\\\\/UJzKT1gmSCPVyWHl8HDvbQwQOmT\\\\\\\/O+DoHoRkivzUjvzML0H\\\\\\\/u5qY12RK9HrIkrc\\\\\\\/iD3ZYt5PfMl6YMZMk8qCHj80I2aR8iV0fMPRMifMCXhhovOJl+ArX3U+H2tFPuCZbHQkqEf7tgEUfCwG4V\\\\\\\/q86ubqiebHX6jecNpQx9cRA4REoKuSmN0Y+ymxx6JsLpbxGBuzU950zHHz1DLaxfCoFkt8foOEPj6mEFlkxzes0qSkdc39SfXSm715Utcyn\\\\\\\/OzYIxoAzs\\\\\\\/JpAoWXomCE00oxAzed3fZ4JuBTMnr\\\\\\\/ZO4OVXVUgaXAVrvT+Q\\\\\\\/a23pHeEk33AzFy3M+J0C5dEldNd9AQJC4imjd2b92XWfS8QaGsHzUW99F5MveFq6zMN3Go6WAgIrpyNbYzmrXFd3dHox5kaMvRjneCQlVcvBu4IngLIlVBa7IevBPFdKaNiuDDi1XZeR\\\\\\\/MGB9M9vOj50UNViPApxRi7o6Rqk9IXOEfW6sI7BGDNE+5vye1R7qpvXkbkQhpuK\\\\\\\/abfgjxQStKu58\\\\\\\/rgPBmgqBfsdMFpny7iuyn5l7RPLcUfHU6t7PNGy\\\\\\\/v2p6zLD39L23HPJvjusoPh5y3VtflXeUefqNRRp3nmbdZF\\\\\\\/49is\\\\\\\/CSNpEc+pcKqXUQkhsD+mMY56KOmHAXAtfD8q7Bgb0l9eRsvWmymbY4BxNblV9XZo7Ye1AX+dYS4oxoqZLB8hm\\\\\\\/MyZt0f47SrlV3kFHX3crr1uFzKe0e95+Y8FgZjxYxdheTuyKkZDEScRvTokdsAVP3u+6hacv2sennIYAHmfjull8XXaaLlzUX6Xt4qPCzDLlfFZWx40y\\\\\\\/Bj5IbKvKSY93e+D1HjuwjjwOfMbYurDAkiBnWe\\\\\\\/N83up7h1zk5YkwKt20wmBL+bGPJUwX5ddghebtGFTAQdtep4XK6\\\\\\\/7VYz1AKRA9B3hCYev4rHyCe6rpcbO0XMrHvBrlmusovwuGcdfy8VMk58RIeBWX0rABtwjiW4qOif0pcupS37lAd7G2Mjpu1cwPoymN2thAJ+DasQSpcUisF5gncHnOcqvY+gyNY0EkncsMb0AAI3bMeC65VJ5f5oH77zYO0g3sWiYGhKZvJrouDE3AMZkLXzmDKoY2e2BFXy9P2n2GXSYitFVp907GJkQ3B4FEQSE8tmFXhqWnE6K\\\\\\\/FgDiR1UiC88gro27+jF6+DMOOa5LD+YJAHDwnncxipNYuMzKZPaY5fmIIIV3A2qmOPxlBM24DHDhMXXpJZGbQdUrw0PELhECA\\\\\\\/MjYaqCacEPC+yNDqi3unVCr67qRMPAT65oAlZ8atLYxt8bKGGub8HIn8+k7trfYvOGVfRaHYIgqNEpt2\\\\\\\/\\\\\\\/kxTV0RCQ6CZ3hk0kGZ8\\\\\\\/Ei4ghukxrYlQjk53N7CT06HlEaTCKFjiHD17CElPnHrL2AeuneS34XylEHMOrhBQG0sgCD\\\\\\\/Q30HMzE6qVW5Vfz2rpd9ZMnwlVU2SuIg0xYq\\\\\\\/VOAEKuLhT6XILpbJBtuOI1CTmCWVlz3plDsvYj\\\\\\\/uHEP7sGUQnIroFOv5eH3MDgIYR5ZIQnQhtfRtXna9uxaV+OQ7Y4r+XAjFYIFh1l6gMJC8iJ0QEnPKfdlxopGobEFGYJ1twf7k1Oj2cU4dtjt7MytSiGQ8b5hqDBdUxRsZekmjkKYyQ9h6yknJC3Fa3vU7kT41u2oNgHunOiH+IzNKTyj9LGBYmE9p8NtzOYof93gW+vYGYTIIYtY1bzsCIXjVMlH\\\\\\\/L4+1faDEIzXodSDDdCRAzhbxWpiIKDPLFvDTjXJ7WgiQM\\\\\\\/RJREB\\\\\\\/7T38oRzxaMAaW8plbqkjOqIJAxDjKVYXWPyWN8Exfmo+HkvGWTUH1RL0XNKIap5RyGl4ZkdFZkX2Tgby\\\\\\\/igT5V3srQSBkGHeg4QwP\\\\\\\/55210z0nvtuN55KyMu0rX4gjI64oLTOaQL7nBxNo4zFRP7AddxpgVfzTVLwFzr84Fygs7tUvWpz576oS4ie\\\\\\\/Ido8O65NSK7R4JKgR9rFPwIRvuhzDCvZx1f9962+10OG6Ri2q6+kk0cgf0nKKk9S8jwh2A\\\\\\\/G32gyezpYChWLGOojJEYsoAvWhhdTOz3gnpGqBrnJctUNJ\\\\\\\/u8UWb0EnYK5sECVYkcQJkgEUUX5uCmtJq+F5ffy1Rsnt1SNrDg6U07iRuPmWlZB4R38tUZQxT1AtL\\\\\\\/veVpN121Ocdsf2v\\\\\\\/2gE+frz1GuGqx5cz1ciFtHtbqc8EvQZ+ZLk+jaQl\\\\\\\/+AXUY7V7ewBqJdMcB7sOcJzsRWZh2SMChmTheVP29y\\\\\\\/wJvRgOn1xxHp69wNonej9uAgMKD7rofBMVHbqcC9v54UySheVcuZ+jskyjQNvK4slSOGi2ZnWu9sCBZ\\\\\\\/rbvroHoPRH+SmoPGh4fIfWiVxFh66TYA2B9fOneZgSem+\\\\\\\/HObUcBrJpcxc7a5Qv4ZZ1W5asVreeOqKYjM5RE9nfZnrYCdHTKgXTYA74+INw4\\\\\\\/q\\\\\\\/a9LqFGtTpNI6DNxYDmm1tl1jLgXZeC8GFfAxvLFb7M006h9ql133s6KyhPPACkGcn9uxylalTd8t1m6zoqv\\\\\\\/hIK+1lMGuIYC8mi+FHKmNZXetfsbIF6EvZcIqqJF0TOKec3pbv3ymUeVZyDVSXz1tbk++biehQpCDTMKcXDPJj27l8UBEYkeq2gwSe8GrpZvq1+7xGEre1FhtD0expBe0z++MFb4VesHJWzzGEYKdkM\\\\\\\/QqPGKJJROFlQo10kiRAsZPPHTVaAzDqwH4uLAjdMmfk3f4Ba\\\\\\\/MbalrbqMqt7oMATMsaTLkW5DSnKIj6i5hts+5s+2bV9ZXALOF5quSft6DomQsFRjh\\\\\\\/k7yiDK5bxBLngSsXe0M5+1eDccAr0VJutDHsXaa88\\\\\\\/4zpHFQZycnI\\\\\\\/blblLRV9W6H66lQongWgCK+TkEQOQVKR8OTaeQ9y9jENUsajWECR1aslvD77NCTz17cG\\\\\\\/nI7jOCxc2BxwUHYWHb7WvKzqA40K8lHlxBD5b177zmVwKqTh4NRFCA+bfGMvU+3w3cmz1DUprD4WvMv7U8ojAWjEV20aooDHWPKMPmPrMQ4lPNF9\\\\\\\/cxkJNsXNeQnZEJKPpIwekmCwAQHCnEU7k6QG7e\\\\\\\/pXXt0c7msps5j88igOdb3RlG8goDWqivAKU7AfxvD9f61qSuTk0RxjDs7+AA1nfKFxibbnVI2SIbCzs3m+ns3g2K9bGnDF3+t6vq3L7oryUr\\\\\\\/zB6GEJIjoTlucPc6o2SsFuMFdc29PIHMAIJf5EBGftMztud1tTsVSwuhpX+8ijIsEpzDMpFiJGFlx7Wb5j\\\\\\\/mCw8Dy3WrZiby\\\\\\\/14y7PwEdNKGcRVpBrA3GQSKOEStNserg4GkOIoBiUkBNXmjkIE0VOAyLJDuRvEVT8b2dhpROl+2h9kubHOwECHJrop5n2n2xoQOvH4ZUWOrk+c6pcbJOhSeJ+citqm7I3X7nPoPnSvKpSv09p0rp+wGdIYY6QSLHr6U9+CmU8J98cfDAzMtSH6a4W3MXsqrfGssGGHQcIcFOapqH8nsV\\\\\\\/kUU7AHir8WjSe8dWLDMROv4V2mr9+bW+9wIUi83RG1sRjWyKF4pCriKKRqpQSFK1EskR1eTaacr2vpI93u4xhrkuTkhhRq6effsqnmRyYwDTryKlq5x4LJdXlFNnRizWLO5YWgWW9nr9y2KWBf21UEQ6O9Yj2hYCv2cnTasXgAJcv8m6gXT6Cd+i28D9MxQe0gwcYrrCQHPwa78H+vKyKHXuWKlJFUqv2QhK2o8GF07IxCW1osz8Pbe1aAdXXbXS2zo2sNTKMe9b9xjyzL8kXhYXCLruc+6TMCauLosGXXX1sSQSErluYS7HlQVCwB4gaVIU0t43czhYjWwYfRdhVOS7+tWeLW1CrkmlDvysgWPgQpTFebu92P9Dm5UAhZUA95l9Nqd9tTCq+AbcZPB4ROOt3eO+SibAq\\\\\\\/QFeQSPyDOkNhsSQeJP\\\\\\\/6eppt2Y+\\\\\\\/K9ToEgJt8vWf1BU766oR7vD3QOIYmHygDziEFwWtdwxiworM28Mco2r1\\\\\\\/\\\\\\\/jHg+eNG2uq4TBDNVAujwCsVpK9KSws0abouxJEwTY1Cr7lmBI\\\\\\\/zaKzwig2qBih70TMVXgtcDZPYbVhr95z+2Cu5HbVRLF0dzsVaGLp3cWUTiSc\\\\\\\/e6Sa7CzDt58CmitERL\\\\\\\/YcRiVECPc9GL2LXh3Zd1gV3FFxPMhKl6\\\\\\\/zIFCmaITP61GeWIpmKGi4sVhlHyzeHup9GAFzPbvqAjd\\\\\\\/TBOKKHDattKfbL9vwPcqWUq4u5WnYzoBbzCGfIEMscXgQ4aFeF12kXnxeczThjKTbUX9IyIszR3piMgo9rQa\\\\\\\/\\\\\\\/TokEcqcJD8B48u65aIk8R0kFRMkhe6l8ym8qlKlRTJs54\\\\\\\/h0NBsITSvaJVT1Ifu1P7sli3OIXRtQoPdTt\\\\\\\/3XZHeqPDqV2T5bvhLakCWnlaqNx+LkU8h+p\\\\\\\/LYZyHMtco1YVWkMm1sGEkxUL4lnnUGcA+qQTvUnnEdZ53SGYppRlqMW49QBP6DGqhJf1oEXjmPC7tKqQyYIn2ekBkEdXv60EUWfzz5c4tYOuF5ylZd9VhDvlMlRoKA7i+2dAw4y1Uhm3AB9T+jIEXjX84qow3WN0SVcC3gYkPr750G8GzCi79Kd27NZpMs6Bxlm6UsUEk5byWkI5Az7dzU1PvQcsHkuLdKDTfBlNFvAMlPSrbv1fAbE9kYyeueNLdi2m3iggk7olLZsgs6e5p+tpUZ\\\\\\\/MXEnpP4zGDGp5fOUhBaiaQz0ukx+BgpgE7k8KzvCCKMXC3UKxBh8Iw\\\\\\\/MgvBl4Ak8EjI4ohDQboibvs+7ZI6uYBnW60YnZI0Lgsnio8Z+NSBlKCLIeXEaBYe+bQte9XdJbzhr1vqY2NCF+9G5XCMXpCmiZv4QoT5f\\\\\\\/5aEzN4DPJ1BLzyfh40POhiMhdirAwOsLnhgMDfhPKUfV4TUmHWy1yAJ7sHXMMGH4HMDiD+4eZrBLRWOKCFIAl2LaYDi9MraDiTNotH7\\\\\\\/JvNVhI2ACGn4LDu\\\\\\\/shBIG2aCRh3tpeUsrylOTadaYhEry+jRCBHbAfg8AtPzQxwkq8af0P9mRQv5zY2UML42zcKi59j5\\\\\\\/TbXyliGWxpnMdknRc\\\\\\\/ZkgqQcSDdIIW2ebs3oywlZS8ixXj5C50\\\\\\\/dBQGcq9Tto1CSsX\\\\\\\/e7tgo7OQU5KaIML8wP7HTiftnPZCHtGQXjs1sndC4znQnAF8hbSNuYatQrytoJpdAa96T\\\\\\\/VCwBz0KH+C4Auy5BAoip0Qd1oHoQHuZAKvRC0KeBGPtnFrzGTu3z2qbNZIwY99tEKItFrRbfGh04uFrnDSLjU9vwtv8soYr1D6wWkKbEksSt2D+yLy2rVv4SPG4ynueOvAQaA9FuY7eElJQkv\\\\\\\/4cNz5jGhYYg+qDeY8\\\\\\\/Lmfe6TxSNFEcJ4Wx4sTJu4KCavnwM1+1SAOGg792\\\\\\\/5QhnYftW2o+HOm3ovi2aRnLyCRkGwCtRL6fkcdjmtxAdAu+EQDvMy1LBDwP+w625gTFX2aCR0GFkRo8QAN7cxKNCIqu+9gB\\\\\\\/Q0UUQ7uCRYh88fI4g3R37KRzXwb4hme\\\\\\\/UpikT76YqwUjL0AEgNlbaUhzztC5Uen0WyBTma6u4v9wUbfW4O\\\\\\\/Q5\\\\\\\/vrn2YBA6jNCvzLdb6tpwXQyNcy30d0wJgGxOUIk6N3EJCBKbohwY2t89UfIHZcfesKs2Iz6opBp1+0Js9QJrRtSNC4cC9zLLMHl7cd\\\\\\\/z3yt7VLZx4X1BgYWgsruDkmVwmF4vaTJOAMG4tiwpkjM0Qk1WkrLe5WbZeznsi6CjunkdbN+uucTMrr\\\\\\\/l4jpM9KG8DXk0XjVKHy2ka2dwc7gFhpY3ZBHkgPbfnSQf9AC9mhQ6xng7fGT9h4Vaf9d9w1P1ZaUp8vmwLyLCxpFvmfkA\\\\\\\/nzEeEtHRfWEoYl6TzsXLiBHWRL9NBlCyc0qx7HFCRZR7dAO5zTvjwl4IoCYBaNJ+Nz3NLl8GMhT8v5feiONgfDTfDSbSshAhk8C4apD8FAkUncKUentIxtn488GGEewlf1kxMSdhuLCHhmyAvW3thy24+vuBJjdiIkroz9GYFHbP7tE4lxlmAgRoHCJRLw9cayUd47+USjrbRgqCQkBOFM+SqB2mffqLGYXfH8M5Qea1yxqIdGTPtuVqapMIUWOMznqcgxDaPBMj8yqm5W8USwVUA9FKbnbxM4GB5RL9i9rF0UBol3atjW2Kt0dcTzDVYQ7G58gw8\\\\\\\/jipq2yTfwct47eZ\\\\\\\/E\\\\\\\/cdANUIR61ivSStUtdZeNKMvljQd+d87PHdhrqTrgvvgvMK2jFuMqst6ZKuItufBA0JSZD6MtrIoroNYLkvDCCkkjJuRRZgzsbioBDVu\\\\\\\/z6VNmTLQBt5L6YUTkM9mFUhjUCWl+1Qlm9NbrsK\\\\\\\/ZVijLChj3+qgW1vVCZsAMZHCslRaLsSTX3ZYp+KgvL+1t2MmofNtMX3t3zo11AXCW\\\\\\\/dDsc\\\\\\\/qr0fGykPgazwhDmoePIfp3tQppc0dEHaVtcpB8OKqmd23FXDNYDR9fVlttsXkfv4HjBUDzflgyVfCKIEUxSWiwbgflSXTGQS5mjxUcY7EJsHM3YNQ50LTFxIqhiGG\\\\\\\/TzgrgRz9psT7qO0\\\\\\\/M54ja0ZYjgPn5WWOBkBUOcgHsJ\\\\\\\/q8\\\\\\\/TMvh57Yztx7x94p8DEzoMTvSWo3XKrXW4JfbUPPw917XZRoM7gg4obaxGabcbDxKP47jqH1JkCIBRGiydo1NSQo93yZi\\\\\\\/mGx3kUxiifbg4jy3paYHoYNAiL7HqT5DAjywCbhN2nQSvko1jjiPcZ\\\\\\\/ZmueBI1OfFDaeX2lm5qdE5Zmlvhrn6wEd7o0CC1RbhtwCpU66iIgt8ZQ5r2HI1XmAjXxb2JWCoqMD0p20BEikc6gwp5zQ2aOjmU1MB733om4wk7vepi2yJ5y7jg7f+qJfSl3c2zOixwJQfbqlC1adMW4tNDL08J56Hsr7v1D00rO+8Xv4rDeTDcGFvrVjcAtOQGBeOTKUol3OptLZJSj8icMyw2T8wrnOxoSfeyFElXIKlR\\\\\\\/v5jN2df3flpwX+tDjaJAh26lTO3vFfEMt8ZZJMzqVLt5A\\\\\\\/vVkeMBse4dwzA8vT6Dx29dNe1zZdmW9UpAI3LnmckQ4+PL7pIwix0FiJiV2KnSnZ+F7LsnS9glW6MSl1nXCFpQ2qq7xlNeAvVGGjHHwoRmyjzw9CPB6QEo+lra3\\\\\\\/HLG8B+w0jffioLzx4J44JermSyS\\\\\\\/LyN8VpPQRkcnMv+qmpbxUwPt5QU\\\\\\\/mdOKtY0wWN8W8+AWp\\\\\\\/9S5X+lP95xOytS8KTRAw2VehdErqRVv1UEkSnqhVX3gQLtk+dMeNEEBnc78k85M+TxLknY25G096L6CwYXsHIyAc35wAi7zpLv8eW6h8lxCtIYIxS7vc3aZYJf5Pfmh3yahlXaVgOOYXk\\\\\\\/6+Cr4WBSgBD80xtESkDD3yY6YwonhUizLrBA2I1QNEqMjPzrVObYHv+o09mIL4TkXFNUKfJ4nI9RVLrh4VUn90GZCeqEtPRA242iqmJ6muQWBB4iNaKNJUcBL8nJNl7UYtJ+M9uvky3pTvA3Mo54zrSoOSFuHiYk0rkMpAvtJz4p5Z\\\\\\\/lOEefwYV1JIpENKIiiXk3ZOamdT0kYVea5E4p4Pe\\\\\\\/hZ7pXTt8d5vBteoDLa9Wic9Mhfk6abYRLV5DXOt2o7lQP6toM+PWgRO9A4nGxsEhGlDHO4hLg9lksYgWIEXbWP0SeMKWNf+1ljVPJuSIE1v10prNwXODlQEaP7zfYifNiGJOgK6AvcfDF3DcXeELrDWSvO7j22JVmFSZvg1a+iOW7P3fR\\\\\\\/vSVDcX5kDR8lZiZxcEnqtnxfpmyBEMOgypk9dYejt45ZD0ZJqEYPaudtORp84rmyj+n8PFklZ7R1IaAyv0RmwgLuilQwej6iZEyQXxukBkWQ30Gce5A45H7q4ZDcJypur5KomyUgokDCD4icxzj2nOH9G8cc9wTDsX8Xd5KQGXu4GA+lcYgn0MWHRAZLJdcwbj8+DlcWCL7nsgV7KC4JJhoOi\\\\\\\/R\\\\\\\/Fu0f4YMle3I9DJ+5PVOaQ0+UIQq\\\\\\\/RtVbPRPOZz4ymYV2jEsrJXHKOTRArq55tiD0v0oq9BS6TxF6pkFPHmgrz3B1BItvbx9mYnpq65YMpGPx+59K6NMG3tTZiMWbH1PZLNKxqDeBO6J+L0oIHNehDFe3+kClLvHvv3vlxs3sNkmKGjax2xIQWhViuZFev\\\\\\\/mPLTB9xtDf9CdFhV\\\\\\\/C0xnVo7R1NAk1sbQuqfZr4km3yi25l0B3LunWfvNDaHLJwPwk3gnV67L6TNxHvRljnB67Mu29PAJC7s8a25b1aGaXeg0Ovh+TuaQCXIB99iWxONUtEObS8xNhYsFMdA6PDmr\\\\\\\/Y\\\\\\\/dyBKT1cgY0ilOx2gza+8aXDEanhfyn8gfk5gkL27vt+iw3naWrIy19Gkve2DfhAdM41Mtktnh4D1NSVEbh18LvdeDGRYE\\\\\\\/03vtuk7Z4aHq+fFPGspD4M3g1PHsgVtb8tl3OsbhIENadvc\\\\\\\/gQsD4cP26rK2Y4ElXvV4kdwuACqYdz1Dc3SS7cdzaAH2Vflm5WE83ZjXc+k8sYptOVWevE8O0WkoIqN3TSDfF1zjmYEdpJasbCoP4ismvDH31pA0Nw\\\\\\\/SwMMWBWKevJ4xOSoSc13MTE0dD0Bd\\\\\\\/7hi357HbTkc+Dv4mv8XCiexv5rsVv7fpdh5ALIXxguU\\\\\\\/a2gH8IIju67irI9PcgMERi9M9cPkpRyf51A+X4y7FIJcWQASP8MgNZbKlg03Ng8GMYlrG8HGRqhr27G4IhNAcQSFbvsHLWSZZmuuznuOeWXI2HcW1qc3z2kLOotRvtog6xZ04\\\\\\\/OWGyQc6KuJPMAYV5sGBlGUmFgCvyzN8N94Smb3EuuiGwKdoc5LoLfJNkPVd9VO+HkwaUDYTRsndk5hmuHkBOhIiE8cHURnPcsPal\\\\\\\/uRK0KbqGbFlj+A\\\\\\\/zaP9Qo77u34TElR\\\\\\\/tkFaYCcdHiWAZkMEGoX0YWnZODX7Y5Vbkwk\\\\\\\/rB2RR869isVp5VS5JYgnu2+tRG7xmRHOlcwqBfiwT70yyT+P0pBqgVFCGo\\\\\\\/glkqox6KIrJfSZN+YrmT6Dtt97vKBx7ww6TrddvPJjRjdL9IVhJZFG0r\\\\\\\/UFNd99Eo1X1iM\\\\\\\/Buty5Fw\\\\\\\/aHujDAmxf9S6mufimaoBMXMb1PS18rIp\\\\\\\/tclEKYuT4KhJK0ZWnXAjM0Q8fSRhzTtqPNSniJqE9FFgKuyoI6wgikjvxIk1sJF\\\\\\\/DB0ORcd9qEjrGeubWsrsUs7JO0r8Es2AiLfuA5j7eUwiv\\\\\\\/Fy1UL3i4Gtlk\\\\\\\/7C025+c+GUYDeyhH1gu+I7uLwuiq6EXGKkrGnDxvsTdU7X5MDDh8wUQcn1Fra4WvOp07LaVJa66yMx6p7AS41fksAJXsKQn\\\\\\\/D3J5MSymZ8XzLYbceearV7+mjSqFJoRQUe6wINQiZYaYvDAvjVg4dpn3AIQF4va5bPgM02DpW5+QCOgjsE3HJoqUU1hga8Y3Z+xqBuGmPCzxPtSpTKOjyB96Vy8YY8ZSYxdO6H5GxrNfug2rBB8GK7m3kZxCFajQsXTeU1KkRC8UJ1CX4TtoLY2vBpfre0360r8aVI9DcCkaJwsLuF\\\\\\\/s3jMBfcyJtih8khoASrxIZ9IY8Y\\\\\\\/dM2h\\\\\\\/aiu6bXxi\\\\\\\/kEyEbUmb5LDOqIBY03TGnnBShE5KuKhqIpjIdbEuYWNdE5Jw\\\\\\\/3jsZWnY3FzxjUY4HF6f\\\\\\\/QO7JZd1hebSG5PMyn8AfmYBYOa5Hb8+ia46F9eAYqAz8rTp0KXMbbbog6ta2Z9bD8WVpvKCvFapMMVY+v7bK0WIIDfEkg6XSLEtIkCWgLwNFMhqCHdDfDkBp8LUS\\\\\\\/Wwr6c7aWXE2q2Nsa4kE7hHrtmRCgc\\\\\\\/7LhXyHH+3tyJKM4+302rdZ4W3oaj\\\\\\\/mjnD1RRyDhTYklL7oB7uqU\\\\\\\/wxeE4D0Fs6BBW099dP5jV1r5ofFAl73GVLk3ELT+wJ7qT2VUL5j39FzkfGWtqL7QsGDL8eWe9hU4D2uM9XLfddmbe4uMOFlVyiqhyyv53uJKnL0K4qFON7ETJItL9jsht5Q3vJdsfrD7coUVEwyy\\\\\\\/TiGlF22HWPj4wpcHHGRjVGn34YYmYCqw+8HUqCZbfDNLvDMrJ81Dy+yQkrbTMBuuKo3DrmMtrOnCHQ5F3pl4f5RKLQXJHLTUDRqx2oUWv5DdaruUR21GcdAU3uXuB9qzvJlYYiXk\\\\\\\/a0m\\\\\\\/TK1M1M\\\\\\\/4o4gez8UA5C52+roLEtNMVaczPsEU3YelZuqlKNgHfSPu\\\\\\\/3asvXtVRVr0svYyd+R3y0Yj2hgBqMt+\\\\\\\/gzyyKmdd7OlfrgdE6RxwSAahBHQIyNo6WzzAb1J\\\\\\\/OBhW65VGXx8v5fM1hzvgXqLZhYXkqxgd2A8k2oh0K9DSPRW\\\\\\\/2WRtAATZIxbeJX28AYcuL4K9sckLyr8GofA9RLz4mKDmvknb1sdwkg54LiDNUo3SZxHNyxgecIXhcbjXBNKwK9ZS+FWu2c\\\\\\\/pIqwFP71InJwuRBCUEJ36cbmEm5uxFyK+YApVJeqVh79HVeuOxHZ75N1jQWSMMqyCyGamIAL+UoGsRnpOXIGcn4IClpIlrqgca9awTZaS6J+d4Od\\\\\\\/wt\\\\\\\/BpJ6HWLdxAkfFf\\\\\\\/0lHdJnWj0UC7JyOxczNwYSLrKfUibmIe15J5jZXcRNKAv6cJpFo58pTmWVrpxyTvlFsiZSddouqo9aiKPmeUNJf4retdu1GDo\\\\\\\/RIfmU6lqdpvUmK2MGH6TXIDuKmP1MwrJ97Evyn+hUr2rMT0REZ+fF5jqZNymAkMiMWB1wd8iVBboKa25NQXgWW8ODQ1aVB7Vq35UKyerNYpp\\\\\\\/yka2RnTOnxz3e6Ah+02X+byJLFiiGLZqTsCPdp7SrLbIrSepWQdx4BuDBriKMHfj1nlwspE7AoHvmJHITWO1La35CRpOCoVA0Tfa1cOuGC1xSkXy6wjsBF96hXOjlzhwBKrgzwto9ITnvZhy1p048tvq4GmakeU4MUgMDCRrk\\\\\\\/YHFmLHNrXnE3I9WM6nK9ERVg8Bv38gPYzZZ2Orh9jC8MNRQFah5z7fUEjj30uQlr6RAqURU\\\\\\\/ZIxj9USpoJriAXLLl4w74x5OQAlxQhcb6yyT9EQMNGPrNrPGAHv19hQdN8WIlBvpHVsBWYF4xk3tQfrp7el1oHksnz6v4Wn8bwM3C3ksQZQLMYU4IBTx0Z9AwKDGQ8J\\\\\\\/H4f1YG1UzsvhYvPLP9nyan6log4+nY23iCtOWjCxtJupt8AwohoRaaDkX1h1diy\\\\\\\/Yd7IdycI488+v91kJrRhtKBi8L6qPKvXOCEBEVnjELbCw22Z0pzeVnhvre2Y\\\\\\\/GPXLoyNdcf0sUE2PSIhHC8fq2aet+xQ7fNMxsZV8i0gAU8iEY6DNBNh8FfrSQ\\\\\\\/yg9uGwmV5ziko6I7K1OQ8wh1BFs\\\\\\\/pf+S8iOBd5HUkW1xUHCQ2VFlbMjPZhh1BH1xVMQ4ogydXYjS00slh4ThIoI3DfZtfDKSUZTiDS+MeWyKAej6mYnW0JZSSlEpF2\\\\\\\/2Xs8DYgcxA\\\\\\\/T7z1ZThP2Xh\\\\\\\/Vo3\\\\\\\/JKRFrEEpwc3TCfkcHctjEqJnTh8EtCCL4V0Ul2MA77dZmapznDKEKyEcMbyLDDUmAN5DkM2wmvKZ3AVE7DLvKkGN8RKS4pAI1fWuP4JQfLXhkr0qbVJ\\\\\\\/KVhwLbcK1MdYjx+Uh5BmZhhg9URTpj6YouvSppZ2fwpO8zMvxVj\\\\\\\/Rbi2LrimSNXs7iG09NWnhNXyWW+s64hkKsOsbUDjmKKg6ZWb\\\\\\\/zlMv4zNOjRDPGVKsiyKH4IJMCwK9rarvof51OWD5gK9xk6VP9BKkRpcnoFn2rIFS31JRNbfTtanVKmoTJxLjt5ZDRYFjbespgNp6YIH+GPCKefZ4PxKslVmJ4p\\\\\\\/Ms9StWprQ9n5Yt1g5fM+oIIDqWcUV9iuuAnEb4w55YzL1qkvQjnSwcjAwRW2ZH8xkyJCW1eIOQYvTA2d0aWfmsyRAT21CoTijgci2oPGApPM3o9sYTu39jTGLG8B1WeJgiRW7E00NtHP4qdFezbToA5cOJPYFzcL3edzHFPUNFgKy6IQZ0nbHkg8UrzlQkdwgCKAAi62HQaI3IAV\\\\\\\/j6EqxCc81C4sOcxnyUi4ZvkFCT9mrAKdXyWYWB4ZNCLlEa7AKn+asiS0LU\\\\\\\/h3d8ZvrfAuRyNr8O6LTkA8xe8W5zZ+xiAF5vcr0QitEfjxICmNKvYW1CgQ0a4uDKYpIuC3Q2AIn7fSmrvMABWPSlDgr\\\\\\\/maz+Pd2eWa\\\\\\\/uFslGIaJXnwP\\\\\\\/x3nZczUpWmXSU7v4+o6q2IPhh6VZ9yOOuHmUoNyDBhso6eRVdx6OCsBb\\\\\\\/RHBhkmBzrOHTisD0OXk0a\\\\\\\/tYvd3jDtHINGxfBwKFTBXQnhg35e3Kws5pLNK7Zd3Jam47jFOJvMiKSLRIpOwzp3t2yBmp0R9u10wf6N3O8LFfOoDsupiRS4u3K6BXcO\\\\\\\/YvLM1i7s+J2vKCgjQbr47TZT35lwbd7OjGf3Ob\\\\\\\/Ei1enhkJjrNzaVgq2emsWURuj+eZmRP+i9EvcflZk4MstiTqtAaRpT2CEWp1UUxIBV4BmzdPukwvEnDZyoGI3WiSanZu+1LRykU+ZJwRJ2Vw5VKIqni9LqVYgkDpDQsXbKE\\\\\\\/VB73zJ\\\\\\\/XPLzM9HBV47YP6pX4\\\\\\\/6I\\\\\\\/XF+QC0hg9GwzHiSd3uO1eK91VZV3HFEAt9oyGidR7nAUIql6iPm4+axLwJ9i\\\\\\\/1HFIsAclWPmmBcNHrLZdCgKYArEDmN\\\\\\\/fkft+w7D9jI3ZFtDpFoYIBm\\\\\\\/UIPGNF3DAXdblNHBAXqknx3iCrZeO95JN8SkItHkAV0UmwDBIbN\\\\\\\/qEtbg+7bsFDZLzPp44S3Ag7t5MAFjmG31GFZMj0ZLETUOwx0K4xCW23z6PdvJmBi0SNPS3BGbDQ+\\\\\\\/uYsIfi99qG5uPSd9ldBAeGX\\\\\\\/Sqdy5NVJZv09xP9F981h7ZIOR+lHwYfaL4r85+1RN0bjx19vciIa\\\\\\\/X7vCc0d92zxiuigdAfzTA5ioTGq6NrQx8\\\\\\\/VTZf+HJRtxJYNEhKEm5lBGJs3SHq9zQJtou89LBe963HOjIp21E\\\\\\\/r\\\\\\\/Bk53WB8BTu8qZikQ1lXdZJKx6MWaQxVKJ3L+XnpRdGSNEpRbBv51UNMnINBb9eNSWaI8cLqL0Llk1SB4\\\\\\\/CVJIkUxC4gtw+Uavxz9mv8yFlrDxJmzky3l53I2E4vjIiLT6lDuAfHg\\\\\\\/yt1x3m\\\\\\\/0i45Hczs9gydcQaxwR9I60fwcK0MprFBxJyJH9nsVU2BGJGO7nXMvI5IQUlZlaGGVJpPmEyV3G4HLrdd+LVAPhAN9pC2uWRj+iNhXT4wRy6Caa\\\\\\\/u2sMWT9gHaEqv++MUUkJG0NWbNl0kCyDPZkHzJfdUKef4Pzt0tXFquDtciPO4KKipIJT1KY6PbigqeaF+xOqpCRwXcA8veIjE2fIxvWQbEPK+SL5zjPaiga859Oikbv3KmpmZRPH1i\\\\\\\/aBCPdahu1Vis0pyAh4ZL32cR1ZVF3M0WnHo69iQJe8THTAssXpwbshdPOcU0i7RuaHlUcLrINBFrGERP33zHoCDM\\\\\\\/5SPaaL1Hd\\\\\\\/yICrePo1K2RjuqhJst6B2y8d0E7P7MMlyzxS8+hKaXYaKkNB74Aq40NJ9BTY4uIweebW3BbQi\\\\\\\/tAgjZX3ZKnLnn5IAeH39KyXe08QwV7aeYcGh9uZac9y7qjgIhAP0uU2eHuybJu42ICNyj4iFvmCbMOAzqeqyiRIVLaTLNxvpiSuNuReT\\\\\\\/CO3CkqiBVARyWLBtpjZMSIV1EHpo1udejnkYQNusvnjzfokruPbPWwIPWtMaYDpSCdup6TRPr0ndaZOrdg3xDiz\\\\\\\/Y0oRdZJDHGyMXH+50av6E0XPkXzE557tga\\\\\\\/rOiWRCB2+6WUdSWmv8qONnI8eMtErm9vU+jHMUH\\\\\\\/8zmZV+hz91fjNfE+ih3SAx6h\\\\\\\/Cx3botr+FnF\\\\\\\/Wxx0M+bQp92DNGLbdW45x4LIpjiqI4\\\\\\\/TknfrrG2rWEH6U5vGSsoZFWO\\\\\\\/kvhQdMF4ol1hnaIbFsQHC9MXDBEqrssdDMDHYyDSEe2YfMEJgphtIlKnZFkz8XRwwzbHTHsQ5Lrw\\\\\\\/3UvUN50fiJ3\\\\\\\/N0LUrE025IwJRV3il2NV0pEz6517yHqED2WlravfDqb\\\\\\\/kuOBM\\\\\\\/YmGY8j7gI14tPzJbi5MhDqigj6dV2Y87BS1+1I1BejFCe8Zaz4b589htFGmB3Od+fRbMfDyYv23vEwzmP3l\\\\\\\/\\\\\\\/ib0L6gchoJ35h66EsqyUZIpLGGFXb0ATGhuSCmbaTFgBC5wJLkYOIhvWEVkMS\\\\\\\/CR5CWWyIaOmVkS81vDCOvOAK4IfkhRJbhVk3iTR4YoESq3z6j5BUhpCx68N\\\\\\\/kKaD4d\\\\\\\/KznEy1CE8vOwzIsV7hxd0SCYzoHxkCNG4sfT7iqu+abRqYOL+idBnVq3a0BV0sAJUVohSh\\\\\\\/BqQYR6hjNp78ZSWL6c2DDgHA3gIXUl+agJt2HbqAA1hlEOLqx7Wj\\\\\\\/YGD0lAXwm9pU2XDN7CI\\\\\\\/z88MvlggJc2zdIAMRJkVBAtp0hM9aTMddF0pNtovyHm8MWtHdiztSmrvKJP\\\\\\\/3ABEglbXNtt44UYaK41FNoEQHa3VcrlwpuL98UkIDTX8i9x73yy6eTUayTzCr5W7WohvO50G+sIiPXalZ5tPkyakwRESmH52FV5+6h8IRPzGit4kkoy3sSXH9eWOVayZmRzYPPVh3LIJae0kbf118rmXf\\\\\\\/IrmqeaNx7v5oqZ6XB4eWwlsHR0hipbp4OaJJu7IWdlbznKxOD\\\\\\\/39WRZF8yF7PkeLbeW2wxgpK5y3+vW3ud4udkWB473ypyujmDZg\\\\\\\/mvhvA1QW1AYOws02HdqBO6q7iptGAYSbvve507cdHSJBgmOWvtfEyF6WXqnrLxOQUX4s\\\\\\\/WkaGQTw9V8skneNBtv84JnGczWCCImJtANoQTM08Fv1\\\\\\\/tEpNyAG5PmwtnSwltn401odS2iILfE\\\\\\\/Cs2c8NhBcPMa+7\\\\\\\/3LIp0LBC9CnOO\\\\\\\/6tKwGemaQynfKP2Oqdb\\\\\\\/iZOrU0sQsM+q7xV\\\\\\\/\\\\\\\/pXaLRJrycVOMfZifj1xDO5CDZSLx4z+uWYjH7T8iOm3i71woDXmlqgyPhc3dOoLP4q38GX1KYvmXFbU9QHPegdKQ9IdPy33VuRLAeZC7Umh5s1xSVqJ8GkFFbo0bfeD1H43CDQ0bQkoOK+jGRqT3tSnabFps5BsEVlm1ktbRGeQUI+GLbTUimrxg6fwLbXmhC69Oew2Hvzfu1z5ldU8dtg4v4Qd36SpqESp1iVh5buXsmbIATXIYScXQsFCFh28RS01WSj9UI3CsXjQlpycRLVLmknDa5dgtFYO+qANd1ZjQZH4H3Zu1rW2pCK5WpFtcb8DPI2jf70XMFUg7CNKs3Di+Hdd\\\\\\\/nKd\\\\\\\/HQPdp9ClbtUSfFAWL2TDOkA0\\\\\\\/mdg9HyKDpMZ8g5YS3hTazYFZfQJ6zwp8Qk5Ea8LEWNutZXmmMC363lL\\\\\\\/6iBiDUWDxENWSvJfcTyCk1e24SMECFMFfseQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00221RBevXKe5egXSa6yPipQJw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f1e7fc6b82006f51fee84f636e963ed5fe192f9fafb7717295489002c94ac9b9\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229dy05+rUso1dQhWt83V8pSx\\\\\\\/VIsB08mnkZzC4BU5m3fZSPcKHnxuK2d9P7tsOv7kPnbSp7Gt5uzdhugfVI6zX\\\\\\\/O+SEQgwbU1VlyP4TaXq4hEGCN4cvFGvWDj+0hhMWbEXaKXpsOc3iVhX540RjXbTpsjZuREbUG1DgAHZ7vNt4JcfIEpCHA6HDg5GsZ4AUzCmL44v7PhNfpI7v\\\\\\\/jph4g26hUNx6Az46eHfCkmd6LNJDC0Wh+7UqgbIXlZvUAKamgG0Zm7+t8xHMZvMWYe8ifCc33zQ3CBecQLHEJWmGdthHDApA7uIi34p2YEhr+1vePkW9rIOfQDaVb5PEfsB21dQn4ZuIEUkJ05fLI1QP6kVea6nD3Io1lm1gsiy3UNIrXgc7or2+qkiqGaVPWaLDSfco+Ph1StUZV64g2eGs8ZldqzyaYsriw0xC\\\\\\\/FZK9i1ytLkgQ\\\\\\\/5zw+d1jZAdHYYI6yQQxdoTatSZfqzrZsqN5dQSS4DpGYT0q41lt5UdAAbRZf9nfKpZE8GPFVwn03DZFUV+54hK8uC93V6nGjSLfmURvtwBK93nQfQiQ69oGwK4l6Kx4Bv5+zM3Pgj8iAh4z1CUcLAbdk7ypnsTmKxGOrOW2K7npqvMn5s6CHtSsStpkcs4rakQxh\\\\\\\/OIKJc6hIjJ3OATJSaHtBH2dlPOaxkleH2ZQRCfChWAMDyVsIyJ1zh4VHcwepIvzawEeaUPL30vFD55i2ER0IegkvY63TL1A\\\\\\\/SV8I8WturywapXJLpmR7NZnSPzpU3N48Jzdtyy\\\\\\\/IFOzVB+jeWt07VXZ44qgOvfnEm9J6d2nkSkd\\\\\\\/nm2Ax4VKlb8Cuwez8pzIj5Rht8RfhtMakfaSKqxG+Ee0bsZZsJA4NroigpA\\\\\\\/NTo2l5smmcqOLqBWYKjR9Q\\\\\\\/stgAIr9FzjzM\\\\\\\/tlIIIoSa1dicumkZdNPrvVu5MxQS4HzCsvL07cjSrsLKKkCvrpFhXJ9OB8cnx+RySmmKX1gLUwhMA2TPhN0PTnnlt96U9K6CoMDxdH+IgmuFKan6vThQwzrtMALLCzcXeTRFVdNdsTQqzADJ8rPNg2gzwx7TyIELrS5KcZDIDhgBWUUaIQEYIJQ\\\\\\\/B\\\\\\\/DD5tnTP9HbyYzSDZ3giJE2CoImr3U6lT9TIcjwgIJpB5K234QaeLBH3ixoMwTBnr3nxRXBRFAc2DS4GQWq+ySWC1cWkYQO+ywh4P5ZxlIo6bX3U85Zli4xPTFH+\\\\\\\/7zIBv3gbbslXIJN9lL3+0ig7TqU4clu+WAMLxJIiRnaTVFGg9sOR9sYd6kph0yh8HTgBY328kORUf9wTq2acMNpe24hEGaKiXTo68cZ8AEXkGc4E1JvmzlXSLerikruQTtuFAs6vVQdA\\\\\\\/fVtZmPJ\\\\\\\/fOfw\\\\\\\/aN3OTbf67TBtV8+IntlZCazI7rQLCMHnaD9vkknMye77NoFGN4Vw+geNy+XweQ\\\\\\\/IVqIWvQfSivOUjwra2TmOSjVWGu3YJvPaih9eL0Nwja9nNaaiiAfvsOqpccqPkDR\\\\\\\/64Ddw8WtwXTlOIQeXxkJwljEGLvkvpNnIlg4PsgeoX\\\\\\\/tITkDUFkajFXXVI+\\\\\\\/egXxNZmciPxkQ6nx0NxGE\\\\\\\/LUnHX4Lks0joxzW1MZuTNSwGTseCmApIjKV+LxgQSZbFh+Q1EryesM02hSr62xsAzXZqAvDkB9XALvEuGoV5wNzGbsQPhf3tSbsW0DzJODQPlDzdgGJSggfgEHHjMXTlYuNTOkOL0W+VrrRsZLv0ZvUbslCKVtYxPSyoPSXjtebBaPd0EbdwSY4NPtk4gAmWU\\\\\\\/L6LwVIyooQYMbGd4Q5nQ7JKPzbz2svUvftmLvDTLUJYglshVw9i6nxRWWc\\\\\\\/U9qrrzifZgfXHInMc3Gi3ams2aGZsSlRIjEDjXq8I\\\\\\\/P0DWe2bpWl8vpmte9qPhzDwQH9iJMtjIkMv3vf0gLyDJpjP22U9kcwoNe+Lm7S1B46tslWcEBTZYOVfdsP3kZYFuFC8QTHpVuj912A3gIRkzZ2g0\\\\\\\/Ghya0A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022hMsyx809pG0P14TCVUjkIg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002238cef9d09ffc834ca7c84895aa5ca297def53d912ef569c29296ec3191ceb2db\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459165 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigation">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/2/117" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Gi8E5GisvYzuLydwFjrOv8T25PUevT7jPqGhGA12qmE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/2/117" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ueCfvO5oiIpq8VTAT3P-r-LQJKhtu87xR7xalXBpwHM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/2/105" title="A QTL Study for Regions Contributing to Arabidopsis thaliana Root Skewing on Tilted Surfaces" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/2/131" title="Quantitative Genetics of Genomic Imprinting: A Comparison of Simple Variance Derivations, the Effects of Inbreeding, and Response to Selection" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459165" id="node459165" data-pisa="ggg;1/2/117" data-pisa-master="ggg;g3.111.000034" data-apath="/ggg/1/2/117.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fluorescence <em>In Situ</em> Hybridization–Based Karyotyping of Soybean Translocation Lines</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Seth D. Findley</span>, <span class="highwire-citation-author" data-delta="1">Allison L. Pappas</span>, <span class="highwire-citation-author" data-delta="2">Yaya Cui</span>, <span class="highwire-citation-author" data-delta="3">James A. Birchler</span>, <span class="highwire-citation-author" data-delta="4">Reid G. Palmer</span> and <span class="highwire-citation-author" data-delta="5">Gary Stacey</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">117-129;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000034 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459165" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Seth D. Findley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Soybean Biotechnology, Division of Plant Sciences, University of Missouri, Columbia, Missouri 65211</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Seth%2BD.%2BFindley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Findley%20SD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASeth%2BD.%2BFindley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Allison L. Pappas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA-ARS, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allison%2BL.%2BPappas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pappas%20AL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAllison%2BL.%2BPappas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yaya Cui </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Soybean Biotechnology, Division of Plant Sciences, University of Missouri, Columbia, Missouri 65211</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yaya%2BCui%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cui%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYaya%2BCui%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">James A. Birchler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Division of Biological Sciences, University of Missouri, Columbia, Missouri 65211</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BA.%2BBirchler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Birchler%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BA.%2BBirchler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Reid G. Palmer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA-ARS, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Reid%2BG.%2BPalmer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palmer%20RG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AReid%2BG.%2BPalmer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gary Stacey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Soybean Biotechnology, Division of Plant Sciences, University of Missouri, Columbia, Missouri 65211</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BStacey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stacey%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGary%2BStacey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:staceyg@missouri.edu" class="" data-icon-position="" data-hide-link-title="0">staceyg@missouri.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/1/2/117" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459165" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459165/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/2/117.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459165" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459165/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/2/117.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459165" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459165/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/2/117.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459165" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459165/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/1/2/117.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459165" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459165/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Soybean (<em>Glycine max</em> [L.] Merr.) is a major crop species and, therefore, a major target of genomic and genetic research. However, in contrast to other plant species, relatively few chromosomal aberrations have been identified and characterized in soybean. This is due in part to the difficulty of cytogenetic analysis of its small, morphologically homogeneous chromosomes. The recent development of a fluorescence <em>in situ</em> hybridization –based karyotyping system for soybean has enabled our characterization of most of the chromosomal translocation lines identified to date. Utilizing genetic data from existing translocation studies in soybean, we identified the chromosomes and approximate breakpoints involved in five translocation lines.</p></div><span class="kwd-group-title">Keywords</span><ul class="kwd-group"><li class="kwd"><a href="/keyword/soybean" class="hw-term hw-article-keyword hw-article-keyword-soybean" rel="nofollow">soybean</a></li><li class="kwd"><a href="/keyword/chromosome-translocation" class="hw-term hw-article-keyword hw-article-keyword-chromosome-translocation" rel="nofollow">chromosome translocation</a></li><li class="kwd"><a href="/keyword/fish" class="hw-term hw-article-keyword hw-article-keyword-fish" rel="nofollow">FISH</a></li><li class="kwd"><a href="/keyword/karyotype" class="hw-term hw-article-keyword hw-article-keyword-karyotype" rel="nofollow">karyotype</a></li><li class="kwd"><a href="/keyword/cytogenetics" class="hw-term hw-article-keyword hw-article-keyword-cytogenetics" rel="nofollow">cytogenetics</a></li></ul><p id="p-3">Domesticated soybean (<em>Glycine max</em> [L.] Merr.) is a major crop species and the target of substantial investment of resources toward development of genetic and molecular maps, including the recently completed soybean genome sequencing project (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Schmutz <em>et al.</em> 2010</a>). However, the study of both classical and molecular cytogenetics of soybean has been relatively difficult and thus slower to develop. Very few large-scale structural aberrations have been characterized in soybean chromosomes (reviewed by <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chung and Singh 2008</a>): a few chromosomal inversion lines have been identified (reviewed by <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Palmer <em>et al.</em> 2000</a>), and only seven accessions are known to involve the exchange of fragments between nonhomologous chromosomes (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Palmer and Kilen 1987</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Sellner, 1990</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Mahama <em>et al.</em> 1999</a>). The difficulty in characterizing even large-scale chromosomal aberrations in soybean can be attributed to their relatively large number (2n = 40; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Veatch 1934</a>), small size (∼2 µm), and morphological homogeneity (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ahmad <em>et al.</em> 1984</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Singh and Hymowitz 1988</a>). The inability to directly distinguish between different soybean chromosomes has meant that aberrations had to be characterized primarily by genetics, in combination with cytogenetics, to detect alterations in patterns of meiotic chromosome pairing. The best characterized reciprocal translocation line in soybean is <em>Glycine soja</em> P.I. 101404B, originally identified by Williams (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Williams 1948</a>) and genetically characterized by <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Palmer and Heer (1984)</a>. One breakpoint was positioned with respect to genetic markers on one of the two chromosomes involved (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Sadanaga and Grindeland 1984</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Mahama and Palmer 2003</a>), chromosome 13 (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cregan <em>et al.</em> 1999</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cregan <em>et al.</em> 2001</a>). This translocation appears to predominate in Chinese <em>G. soja</em> accessions (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Palmer <em>et al.</em> 1987</a>) and is indistinguishable from the exchange characterized in <em>G. soja</em> P.I. 464890B (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Findley <em>et al.</em> 2010</a>, discussed below).</p><p id="p-4">The ability to rapidly and definitively address chromosome structure in soybean is now possible due to the development of a fluorescence <em>in situ</em> hybridization (FISH)-based karyotyping system (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Findley <em>et al.</em> 2010</a>), which utilizes genetically anchored bacterial artificial chromosomes (BACs; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Schmutz <em>et al.</em> 2010</a>) and variation in genomic repeats to identify each soybean chromosome. Using these tools, the translocation in <em>G. soja</em> P.I. 464890B was determined to involve the reciprocal exchange of a (&gt;17.9 Mb) segment of chromosome 13 with a ∼4.2 Mb segment of chromosome 11 (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Findley <em>et al.</em> 2010</a>). That study was the first detailed structural analysis of any soybean chromosome rearrangement in which both the chromosomes involved and breakpoints were defined.</p><p id="p-5">Fortunately, the inability to directly identify translocation chromosomes has not precluded determining whether a given soybean accession contains them, because the presence of rearranged chromosomes can be detected through their interaction with their nonrearranged (normal) counterparts in meiosis. In soybean, such interaction has been detected directly, through chromosome pairing patterns in meiosis in heterozygous chromosome translocation plants (<em>e.g.</em>, <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Singh and Hymowitz 1988</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Mahama <em>et al.</em> 1999</a>), or indirectly, through analysis of pollen or ovule abortion in heterozygous chromosome translocation plants (<em>e.g.</em>, <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Palmer <em>et al.</em> 1987</a>). Such heterozygous soybean plants have about 50% pollen and ovule sterility (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Palmer and Heer 1984</a>). This is typical of a large number of plants that have equally frequent alternate and adjacent chromosome segregation (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Burnham 1962</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Endrizzi 1974</a>). Soybean (2n = 40) normally forms 20 bivalents in metaphase I of meiosis (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Veatch 1934</a>; <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Singh and Hymowitz 1988</a>). In F1 plants heterozygous for a reciprocal translocation, in addition to 18 bivalents, interaction between the normal and rearranged chromosomes can generate a single chain, or ring of four (the “☉4” conformation) chromosomes (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Mahama <em>et al.</em> 1999</a>). These aberrant, homology-based associations are thus diagnostic in confirming suspected translocations.</p><p id="p-6">The power of meiotic cytogenetic analysis also can be harnessed to determine whether a <em>particular</em> chromosome is rearranged in two <em>different</em> reciprocal translocation lines (<em>i.e.</em> whether one or both exchange chromosomes are shared between two different lines, each containing a pair of reciprocally-exchanged chromosomes). For example, if line A has a translocation between chromosomes 1 and 2, and line B has a translocation between chromosomes 2 and 3, in meiosis of transheterozygous F1 plants, chains or rings of six chromosomes (☉6 conformation) will form (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Burnham 1962</a>). The presence of a ☉6 conformation in meiosis in F1 progeny generated from two different reciprocal translocation lines was a key diagnostic criterion in the analysis several soybean translocation lines (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Mahama <em>et al.</em> 1999</a>). Through meiotic chromosome interaction studies, combined with pollen/ovule sterility analyses in translocation combinations, <a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">Mahama <em>et al.</em> (1999)</a> determined which of the seven extant soybean translocation lines had particular translocation chromosomes in common. Percentage pollen abortion, ovule abortion, and reduction in seed set were higher in F<sub>1</sub> plants from crosses among homozygous translocated lines (a greater number of interchanged chromosomes) than from crosses between homozygous translocation lines (only two nonhomologous interchanged chromosomes) (<a id="xref-ref-25-6" class="xref-bibr" href="#ref-25">Mahama <em>et al.</em> 1999</a>). A similar inverse relationship in <em>Pisum</em> was reported by <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gottschalk (1978)</a>. Despite the power of this approach, the translocation chromosomes could not be determined in these lines because the identities of the aberrant chromosomes themselves were unknown.</p><p id="p-7">FISH-based soybean karyotyping (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Findley <em>et al.</em> 2010</a>) has enabled us to capitalize on genetic interaction data to characterize the remaining uncharacterized translocation lines described in (<a id="xref-ref-25-7" class="xref-bibr" href="#ref-25">Mahama <em>et al.</em> 1999</a>). In the present study, we focused on the following six soybean translocation lines: Clark T/T, renamed here as translocation line-1 (TL-1), which has the translocation chromosomes from <em>Glycine soja</em> P.I. 101404B introgressed into <em>G. max</em> cv. Clark; P.I. 189866 (TL-2), a naturally occurring translocation in an accession of <em>Glycine gracilis</em> Skvortz. (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Skvortzow 1927</a>), which is a weedy form of soybean from Northeast China (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Broich and Palmer 1980</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Shoemaker <em>et al.</em> 1986</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chen and Nelson 2004</a>); KS175-7-3 (TL-3), KS172-11-3 (TL-4) and KS171-31-2 (TL-5), were each identified from fast-neutron irradiated <em>G. max</em> populations; and finally, L75-2083-4 (TL-6), a spontaneous translocation identified in progeny of a cross between <em>G. max</em> cultivars (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Sadanaga and Newhouse 1982</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Palmer and Kilen 1987</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Sellner 1990</a>; <a id="xref-ref-25-8" class="xref-bibr" href="#ref-25">Mahama <em>et al.</em> 1999</a>). Our analysis of these translocation lines involved sequential unraveling of the relationships detected in the cytological analysis of meiotic chromosome structures formed in plants transheterozygous for these various chromosomes.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Fluorescence <em>in situ</em> hybridization</h3><p id="p-8">Sample preparation, FISH experiments, and image processing were performed precisely as described in <a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Findley <em>et al.</em> (2010)</a>. The Cent-Gm repeat-based FISH cocktail is based on differential chromosome painting with a cocktail of fluorophore-tagged oligonucleotides whose sequences target four variants of Cent-Gm, the soybean centromeric repeat (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Vahedian <em>et al.</em> 1995</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Swaminathan <em>et al.</em> 2007</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gill <em>et al.</em>, 2009</a>; <a id="xref-ref-14-6" class="xref-bibr" href="#ref-14">Findley <em>et al.</em> 2010</a>). The cocktail used in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figures 3</a>–<a id="xref-fig-5-1" class="xref-fig" href="#F5">5</a> utilized the following fluorophore-conjugated oligonucleotide probes (amount per slide): CY5-CentGm-2-M (7.5 ng), fluorescein-CentGm-2-E (20 ng), Texas red-615-CentGm-1-AF (0.01 ng), fluorescein-CentGm-1-G (20 ng), and CY5-SB86-C (20 ng). A fluorescein-SB86-C oligonucleotide (10 ng) was used for FISH in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, panels D and H. Images in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, panels K-L, utilized two fluorophore-conjugated Cent-Gm oligonucleotide probes (amount per slide): fluorescein-CentGm-2-E (10 ng) plus fluorescein-CentGm-1-G (10 ng) (see <a id="xref-ref-14-7" class="xref-bibr" href="#ref-14">Findley <em>et al.</em> 2010</a> for oligonucleotide sequences). For all FISH and 4’,6-diamidino-2-phenylindole (DAPI) images, chromosomes presented in pairs (<em>e.g.</em> <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>, panels H3 and H4) were obtained from a single image of a single chromosome spread that contained both chromosome types. Exceptions were the following images that only required single chromosome types, which were obtained individually: Figures 2D; 2H; 3C; 4, C and F; 4, D and G; 4, K1–K10; 4, L1–L10; 5E3; 5F3; 5, G3–G6; 5K3; 5L4; and 5, M5–M6. Seeds germinated for isolation of root-tip chromosomes from lines characterized in the present study were derived from plants homozygous for their respective translocations. <em>Glycine max</em> (L.) Merr. cv W82 (<em>G. max</em> W82) was used as control.</p><p id="p-9">As described in (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Schmutz <em>et al.</em> 2010</a>), homeologous sequences exist for the majority of regions from which BAC probes are derived; therefore, secondary hybridization signals are also frequently detectable in FISH images (<a id="xref-ref-14-8" class="xref-bibr" href="#ref-14">Findley <em>et al.</em> 2010</a>). However, these secondary hybridization signals are as a rule less bright, and thus distinguishable from primary signals. The hybridization signals for BAC probes in the “tester” set developed for individual chromosome identification (<a id="xref-ref-14-9" class="xref-bibr" href="#ref-14">Findley <em>et al.</em> 2010</a>) are well characterized. Additional BAC probes for breakpoint mapping studies were used in combination with well-characterized BACs, thereby ensuring proper chromosome identification.</p></div><div id="sec-3" class="subsection"><h3>Mapping populations</h3><p id="p-10">The CS mutant was found in a cross of cv. SRF350 Pike in 1976 by Carol Schoener, a graduate student with Dr. W.R. Fehr at Iowa State University at Ames, Iowa. P.I. 189866 is a plant introduction from northeastern China identified as <em>G. gracilis</em> Skvortzov. It is homozygous for a chromosome translocation (X. Delannay and R.G. Palmer, unpublished results). Fertile plants in entry A04-19 (CS mutant line) segregating fertile and sterile plants were used as female parent in manual cross-pollinations with A04-152 (PI189866). The F<sub>1</sub> seed was advanced to the F<sub>2</sub> generation at the University of Puerto Rico/Iowa State University soybean nursery near Isabela, Puerto Rico, as entry 4. All F<sub>1</sub> plants were heterozygous for the chromosome translocation and exhibited about 50% pollen sterility and reduced seed set. The F<sub>1</sub> plants were single-plant threshed. The F<sub>2</sub> seed were planted at the Bruner Farm near Ames, Iowa in summer 2005. Two F<sub>2</sub> families (A05-187 and A05-188) segregated for the chromosome translocation and for the CS sterile that were used for the mapping study. Mature flower buds were collected from each plant separately and placed into vials containing 70% (v/v) ethanol. The anthers were squashed and pollen dispersed in a drop of 1% (v/v) I<sub>2</sub>KI solution (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Jensen 1962</a>). Plants were classified as fertile, about 50% pollen sterile, or male sterile by observations of pollen grain stain intensity at 100× magnification. At maturity, F<sub>2</sub> plants were single-plant threshed. The F<sub>2:3</sub> progenies were grown at the Bruner Farm in 2006. Approximately 50 to 60 seeds per progeny row were tractor planted in 3-m-long rows, spaced 0.7 m apart. Progeny rows were classified for segregation of the CS sterile mutant and the semisterility of the heterozygous chromosome translocation. The genotype of each individual F<sub>2</sub> plant was rationalized based upon the segregation pattern of its F<sub>2:3</sub> progeny row.</p></div><div id="sec-4" class="subsection"><h3>SSR analysis</h3><p id="p-11">Ten grams of young trifoliates were collected from each F<sub>2</sub> plant and stored in individual plastic bags and freeze-dried for 48 to 72 hr. Dry leaves were transferred to 15 mL sterile propylene tubes, ground to powder using glass beads, and kept at −80°C until DNA extraction. The protocol for DNA extractions of <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kato and Palmer (2004)</a> was followed. PCR reactions were performed under one of two conditions. In one set of reactions, 30 µl reactions contained 1X Reaction Buffer (10 m<span class="sc">M</span> Tris-HCl (pH 9.0 at 25°C), 50 m<span class="sc">M</span> KCl, 0.1% Triton X-100; Promega), 0.15 m<span class="sc">M</span> each dNTP, 1.75 m<span class="sc">M</span> MgCl<sub>2</sub>, 1.7 ng/µl genomic DNA, 0.15 µM forward primer, 0.15 µM reverse primer, and 0.1 U Taq polymerase (Promega). DNA was amplified for 32 cycles with denaturation at 94°C for 45 sec, annealing at 47°C for 45 sec, and extension at 68°C for 45 sec. In the other set of reactions, 20 µl reactions contained 1X Thermopol Reaction Buffer (20 m<span class="sc">M</span> Tris-HCl, 10 m<span class="sc">M</span> (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 10 m<span class="sc">M</span> KCl, 2 m<span class="sc">M</span> MgSO<sub>4</sub>, 0.1% Triton X-100, pH 8.8 at 25°C; New England Biolabs), 0.20 m<span class="sc">M</span> each dNTP, 2.5 ng/µl genomic DNA, 0.5 µM forward primer, 0.5 µM reverse primer, and 0.025 U/µl Taq polymerase (New England Biolabs). DNA initially was denatured at 95°C for 2 min, followed by 30 cycles of denaturation at 95°C for 30 sec, annealing at 45°C for 30 sec, and extension at 68°C for 30 sec, and a final extension step at 68°C for 7 min. A PTC-100 Programmable Thermal Controller (MJ Research) was used for all reactions.</p><p id="p-12">PCR amplicons were analyzed on 3%–5% High Resolution Blend agarose (Amresco) containing 0.20 µg/µl ethidium bromide in 1X TBE buffer (0.089 M Tris base, 0.089 M boric acid, and 2 m<span class="sc">M</span> ethylene-diaminetetraacetic acid). Gels were run at a constant 80-100 V for 1.5-4 hr and visualized using UV-transillumination. Amplicons from F2 plants were noted as having either a parental or nonparental genotype. Linkage groups (including pseudo-linkage groups) were determined using QuadMap (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Durrant <em>et al.</em> 2006</a>) and MapMaker 3.0 (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Lander <em>et al.</em> 1987</a>). MapMaker 3.0 was further used to determine marker order and distances. Graphical representations of the maps were generated by MapChart 2.1 (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Voorrips 2002</a>).</p></div></div><div class="section results" id="sec-5"><h2 class="">Results</h2><div id="sec-6" class="subsection"><h3>Analysis of translocation line TL-2</h3><p id="p-13">TL-2 was reported as a homozygous chromosome translocation line by <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Delannay <em>et al.</em> (1982)</a>. A cross of cv. Minsoy (PI27890) x CS mutant was used to molecularly map the CS mutant with SSRs to Classical Genetic Linkage Group 3 (<em>i.e.</em>, CGLG3; Pappas and Palmer, unpublished results). CGLG3 corresponds to Molecular Linkage group D1a (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Cregan <em>et al.</em> 1999</a>) and the pseudomolecule for <em>G. max</em> chromosome 1 (Gm01) (<a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Schmutz <em>et al.</em> 2010</a>). The CS locus was previously found to be linked to the translocation breakpoint in the TL-2 line (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Mahama and Palmer 2003</a>). While this information identified the involvement of chromosome 1 in the translocation, the identity of the other chromosome had to be determined by other means. To identify SSR markers that could be used for molecular mapping, the TL-2 line and CS mutant were screened for polymorphisms at several SSR loci on each chromosome (data not shown). These results allowed for the identification of <em>Glycine gracilis</em> (Gg) linkage groups between markers on Gg01 and Gg08. PCR amplification of polymorphic SSR markers from Gg01 and Gg08 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000034/-/DC1/000034SI.xlsx">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000034/-/DC1/TableS1.xlsx">Table S1</a>) was performed on genomic DNA from 174 F2 progeny.</p><p id="p-14">Several SSR markers from Gg01 and Gg08 were found to be linked. Initial analysis involved evaluating linkage groups using the program QuadMap (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Durrant <em>et al.</em> 2006</a>), which identifies linkage groups in lines with heterozygous translocations. QuadMap was expected to identify four linkage groups, each corresponding to chromosome segments on either side of the translocation breakpoints in the TL-2 line; however, QuadMap was unable to tease apart the pseudolinkage groups, including linkage of several SSR markers from the short arms of Gg01 and Gg08. The inability of QuadMap to identify the translocated segments of TL-2 prevented localization of the translocation breakpoint using molecular mapping alone, as several different maps with similar LOD scores could be generated in MapMaker. To correctly identify the region of the translocation breakpoints in Gg01 and Gg08 (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>), it was necessary to use information about centromere association supplied from the FISH experiments (described below). We generated maps that show linkage of the long arm markers from chromosomes one and eight with a distance of ∼19 cM between the two closest SSRs (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, right). Perhaps most striking was the close linkage between Satt_482 and Satt_333 from the short arms of chromosome one and eight, respectively (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>, left). A 1.5-cM distance between these two SSRs reflects their pericentromeric locations on the same segment.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/2/117/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic linkage maps of Gg01 and Gg08 from G. gracilis. SSR markers from Gg01 are in gray; SSR markers from Gg08 are in black. Several markers from the short arms of Gg01 and Gg08 are closely linked. Using information about centromere association from the FISH analysis, all short arm markers from Gg01 and Gg08 were mapped to the same chromosome (left map) as were all long arms markers (right map). Map distances are shown in centiMorgans (cM)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-498101292" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genetic linkage maps of Gg01 and Gg08 from &lt;em&gt;G. gracilis&lt;/em&gt;. SSR markers from Gg01 are in gray; SSR markers from Gg08 are in black. Several markers from the short arms of Gg01 and Gg08 are closely linked. Using information about centromere association from the FISH analysis, all short arm markers from Gg01 and Gg08 were mapped to the same chromosome (left map) as were all long arms markers (right map). Map distances are shown in centiMorgans (cM).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/2/117/F1.medium.gif" width="321" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/1/2/117/F1.medium.gif" width="321" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/2/117/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/2/117/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460800" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-15" class="first-child">Genetic linkage maps of Gg01 and Gg08 from <em>G. gracilis</em>. SSR markers from Gg01 are in gray; SSR markers from Gg08 are in black. Several markers from the short arms of Gg01 and Gg08 are closely linked. Using information about centromere association from the FISH analysis, all short arm markers from Gg01 and Gg08 were mapped to the same chromosome (left map) as were all long arms markers (right map). Map distances are shown in centiMorgans (cM).</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">Based on both cytology (Figure 2D1; <a id="xref-ref-14-10" class="xref-bibr" href="#ref-14">Findley <em>et al.</em> 2010</a>) and genome sequence assembly (<a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">Schmutz <em>et al.</em> 2010</a>), Gm01 is a nearly acrocentric chromosome, whose short arm (Gm01-SA) and long arm (Gm01-LA) correspond to the 5′ and 3′ ends, respectively, of the corresponding pseudomolecule (see <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). In metaphase chromosome spreads of TL-2 (data not shown), a single chromosome pair appeared aberrantly large (Figure 2H1). To determine whether this chromosome was related to chromosome 1 (Gg01) in TL-2 (<em>Glycine gracilis</em>) and to investigate its structure, we used two Gm01-specific FISH probes. This first was a Gm01 pseudomolecule-derived BAC (Gm08-BAC-E; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000034/-/DC1/TableS2.xlsx">Table S2</a>) that hybridizes to the distal end of Gm01-LA in <em>G. max</em> W82 (Figure 2D1). The second was an oligonucleotide-based probe targeting an 86-basepair genomic repeat (SB86) that uniquely hybridizes to a centromere-proximal position on Gm01-LA (Figure 2D1; <a id="xref-ref-14-11" class="xref-bibr" href="#ref-14">Findley <em>et al.</em> 2010</a>). In TL-2, both probes hybridized to the same arm of the aberrant chromosome, suggesting that much of the Gm01-LA-homologous segment was intact in this accession (Figure 2H1). However, the opposite chromosome arm was much longer in TL-2 in comparison to the Gm01-SA (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, D1 and H1).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/2/117/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="FISH-based characterization of translocation line TL-2. (A) Diagram of Gm01 and Gm08 in G. max W82. Numbers indicate positions (in bp or Mb) in the pseudomolecule of each chromosome. For Gm01, the short arm (&#x201C;SA&#x201D;) is comprised of the 5&#x2032; end (bp &#x201C;1&#x201D;) through the centromere region (at &#x223C;16.6 Mb); the long arm (&#x201C;LA&#x201D;) extends from the centromere to the 3&#x2032; end (at 55.9 Mb in the pseudomolecule). For Gm08, the long arm (&#x201C;LA&#x201D;) is comprised of the 5&#x2032; end (bp &#x201C;1&#x201D;) through the centromere region (at &#x223C;28.8 Mb); the short arm (&#x201C;SA&#x201D;) extends from the centromere to the 3&#x2032; end (at 47.0 Mb). The yellow shading depicts the approximate extent of high-repeat content sequence that defines pericentromeric chromatin. The paired red dots indicate positions of BAC probes used in mapping experiments (representative FISH images are shown; see Table S2 for complete series). The blue triangles indicate the approximate breakpoint positions for TL-2; the gray outlines represent chromosome structure unaffected by the translocation. The blue shading indicates the chromosome sections that the BACs were mapped relative to in (D&#x2013;K). (B) Model of the TL-2 translocation chromosomes. Gg08 (Glycine gracilis chromosome 8) is comprised of sequences homologous to Gm08-LA, the Gm08 centromere, and the Gm01-LA; Gg01 is comprised of sequences homologous to Gm01-SA, the Gm01 centromere and the Gm01-SA. (C) Data summary of the probe sets used in FISH for G. max W82 (D&#x2013;G) and TL-2 (H&#x2013;K). (D1) G. max Gm01 hybridization with Gm01-LA markers: SB86 repeat probe (&#x201C;D&#x201D; in diagram, green hybridization signal in FISH panel) and the terminal Gm01-LA BAC probe, Gm01-BAC-E (&#x201C;E&#x201D; in diagram, red hybridization signal in FISH panel). (H1) Hybridization to the aberrantly large TL-2 chromosome with Gm01-LA markers: SB86 repeat probe (&#x201C;D&#x201D; in diagram, green hybridization signal in FISH panel) and the terminal Gm01-LA BAC probe, Gm01-BAC-E (&#x201C;E&#x201D; in diagram, red hybridization signal in FISH panel). The arrowhead indicates the aberrantly large chromosome arm associated with the Gm01-LA-homologous probes in the mutant. The DAPI signal in (D1) and (H1) is inverted to black to accentuate chromosome structure. In the G. max W82 BAC mapping control experiments (E&#x2013;G), the upper panels are diagrams of the FISH data (middle panels), and corresponding DAPI-stained chromosomes (lower panels). In the TL-2 BAC mapping experiments (I&#x2013;K), the lower panels are diagrams of the FISH data (uppermost panels), and corresponding DAPI-stained chromosomes (middle panels). In (E&#x2013;G) and (I&#x2013;K), three FISH probe hybridization signals are detected: the BAC probe tested (e.g., Gm01-BAC-A= &#x201C;A&#x201D; in diagrams, with red hybridization signal, indicated by arrowhead), a Gm01-LA marker (&#x201C;D&#x201D; in diagrams, indicating the SB86 repeat probe, with blue hybridization signal) and a Gm08-SA marker (&#x201C;7&#x201D; in diagrams, indicating Gm08-BAC-7, with green hybridization signal). (E and I) The Gm01-SA probes (Gm01-BAC-A, -B, and -C) hybridized to Gm01-SA in G. max W82 (E1, E3, and E5, respectively), but to the arm opposite the Gm08-SA-homologous arm in TL-2 (I2, I4, and I6, respectively). (F and J) The Gm08-LA probes (Gm08-BAC-1, -2, and -3) hybridized to Gm08-LA in G. max W82 (F2, F4, and F6, respectively), but to the arm opposite the Gm01-LA-homologous arm in TL-2 (J1, J3, and J5, respectively). (G and K) The Gm08-SA probes (Gm08-BAC-4, -5, and -6) hybridized to Gm08-SA in G. max W82 (G2, G4, and G6, respectively), and also to the Gm08-SA-homologous arm in TL-2 (K2, K4, and K6, respectively). The high hybridization background of the red probes used in the experiments in (E5), (I5), (G1), and (K1) obscures the blue SB86 hybridization signal, which more evident when the blue channel is examined individually. The 2-&#xB5;m scale bar in (H1) is valid for FISH and chromosome panels (D1). The 2-&#xB5;m scale bar in (K6) is valid for (E&#x2013;G) and (I&#x2013;K)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-498101292" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;FISH-based characterization of translocation line TL-2. (A) Diagram of Gm01 and Gm08 in &lt;em&gt;G. max&lt;/em&gt; W82. Numbers indicate positions (in bp or Mb) in the pseudomolecule of each chromosome. For Gm01, the short arm (&#x201C;SA&#x201D;) is comprised of the 5&#x2032; end (bp &#x201C;1&#x201D;) through the centromere region (at &#x223C;16.6 Mb); the long arm (&#x201C;LA&#x201D;) extends from the centromere to the 3&#x2032; end (at 55.9 Mb in the pseudomolecule). For Gm08, the long arm (&#x201C;LA&#x201D;) is comprised of the 5&#x2032; end (bp &#x201C;1&#x201D;) through the centromere region (at &#x223C;28.8 Mb); the short arm (&#x201C;SA&#x201D;) extends from the centromere to the 3&#x2032; end (at 47.0 Mb). The yellow shading depicts the approximate extent of high-repeat content sequence that defines pericentromeric chromatin. The paired red dots indicate positions of BAC probes used in mapping experiments (representative FISH images are shown; see Table S2 for complete series). The blue triangles indicate the approximate breakpoint positions for TL-2; the gray outlines represent chromosome structure unaffected by the translocation. The blue shading indicates the chromosome sections that the BACs were mapped relative to in (D&#x2013;K). (B) Model of the TL-2 translocation chromosomes. Gg08 (&lt;em&gt;Glycine gracilis&lt;/em&gt; chromosome 8) is comprised of sequences homologous to Gm08-LA, the Gm08 centromere, and the Gm01-LA; Gg01 is comprised of sequences homologous to Gm01-SA, the Gm01 centromere and the Gm01-SA. (C) Data summary of the probe sets used in FISH for &lt;em&gt;G. max&lt;/em&gt; W82 (D&#x2013;G) and TL-2 (H&#x2013;K). (D1) &lt;em&gt;G. max&lt;/em&gt; Gm01 hybridization with Gm01-LA markers: SB86 repeat probe (&#x201C;D&#x201D; in diagram, green hybridization signal in FISH panel) and the terminal Gm01-LA BAC probe, Gm01-BAC-E (&#x201C;E&#x201D; in diagram, red hybridization signal in FISH panel). (H1) Hybridization to the aberrantly large TL-2 chromosome with Gm01-LA markers: SB86 repeat probe (&#x201C;D&#x201D; in diagram, green hybridization signal in FISH panel) and the terminal Gm01-LA BAC probe, Gm01-BAC-E (&#x201C;E&#x201D; in diagram, red hybridization signal in FISH panel). The arrowhead indicates the aberrantly large chromosome arm associated with the Gm01-LA-homologous probes in the mutant. The DAPI signal in (D1) and (H1) is inverted to black to accentuate chromosome structure. In the &lt;em&gt;G. max&lt;/em&gt; W82 BAC mapping control experiments (E&#x2013;G), the upper panels are diagrams of the FISH data (middle panels), and corresponding DAPI-stained chromosomes (lower panels). In the TL-2 BAC mapping experiments (I&#x2013;K), the lower panels are diagrams of the FISH data (uppermost panels), and corresponding DAPI-stained chromosomes (middle panels). In (E&#x2013;G) and (I&#x2013;K), three FISH probe hybridization signals are detected: the BAC probe tested (&lt;em&gt;e.g.&lt;/em&gt;, Gm01-BAC-A= &#x201C;A&#x201D; in diagrams, with red hybridization signal, indicated by arrowhead), a Gm01-LA marker (&#x201C;D&#x201D; in diagrams, indicating the SB86 repeat probe, with blue hybridization signal) and a Gm08-SA marker (&#x201C;7&#x201D; in diagrams, indicating Gm08-BAC-7, with green hybridization signal). (E and I) The Gm01-SA probes (Gm01-BAC-A, -B, and -C) hybridized to Gm01-SA in &lt;em&gt;G. max&lt;/em&gt; W82 (E1, E3, and E5, respectively), but to the arm opposite the Gm08-SA-homologous arm in TL-2 (I2, I4, and I6, respectively). (F and J) The Gm08-LA probes (Gm08-BAC-1, -2, and -3) hybridized to Gm08-LA in &lt;em&gt;G. max&lt;/em&gt; W82 (F2, F4, and F6, respectively), but to the arm opposite the Gm01-LA-homologous arm in TL-2 (J1, J3, and J5, respectively). (G and K) The Gm08-SA probes (Gm08-BAC-4, -5, and -6) hybridized to Gm08-SA in &lt;em&gt;G. max&lt;/em&gt; W82 (G2, G4, and G6, respectively), and also to the Gm08-SA-homologous arm in TL-2 (K2, K4, and K6, respectively). The high hybridization background of the red probes used in the experiments in (E5), (I5), (G1), and (K1) obscures the blue SB86 hybridization signal, which more evident when the blue channel is examined individually. The 2-&#xB5;m scale bar in (H1) is valid for FISH and chromosome panels (D1). The 2-&#xB5;m scale bar in (K6) is valid for (E&#x2013;G) and (I&#x2013;K).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/2/117/F2.medium.gif" width="440" height="301"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/1/2/117/F2.medium.gif" width="440" height="301"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/2/117/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/2/117/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460662" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-17" class="first-child">FISH-based characterization of translocation line TL-2. (A) Diagram of Gm01 and Gm08 in <em>G. max</em> W82. Numbers indicate positions (in bp or Mb) in the pseudomolecule of each chromosome. For Gm01, the short arm (“SA”) is comprised of the 5′ end (bp “1”) through the centromere region (at ∼16.6 Mb); the long arm (“LA”) extends from the centromere to the 3′ end (at 55.9 Mb in the pseudomolecule). For Gm08, the long arm (“LA”) is comprised of the 5′ end (bp “1”) through the centromere region (at ∼28.8 Mb); the short arm (“SA”) extends from the centromere to the 3′ end (at 47.0 Mb). The yellow shading depicts the approximate extent of high-repeat content sequence that defines pericentromeric chromatin. The paired red dots indicate positions of BAC probes used in mapping experiments (representative FISH images are shown; see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000034/-/DC1/TableS2.xlsx">Table S2</a> for complete series). The blue triangles indicate the approximate breakpoint positions for TL-2; the gray outlines represent chromosome structure unaffected by the translocation. The blue shading indicates the chromosome sections that the BACs were mapped relative to in (D–K). (B) Model of the TL-2 translocation chromosomes. Gg08 (<em>Glycine gracilis</em> chromosome 8) is comprised of sequences homologous to Gm08-LA, the Gm08 centromere, and the Gm01-LA; Gg01 is comprised of sequences homologous to Gm01-SA, the Gm01 centromere and the Gm01-SA. (C) Data summary of the probe sets used in FISH for <em>G. max</em> W82 (D–G) and TL-2 (H–K). (D1) <em>G. max</em> Gm01 hybridization with Gm01-LA markers: SB86 repeat probe (“D” in diagram, green hybridization signal in FISH panel) and the terminal Gm01-LA BAC probe, Gm01-BAC-E (“E” in diagram, red hybridization signal in FISH panel). (H1) Hybridization to the aberrantly large TL-2 chromosome with Gm01-LA markers: SB86 repeat probe (“D” in diagram, green hybridization signal in FISH panel) and the terminal Gm01-LA BAC probe, Gm01-BAC-E (“E” in diagram, red hybridization signal in FISH panel). The arrowhead indicates the aberrantly large chromosome arm associated with the Gm01-LA-homologous probes in the mutant. The DAPI signal in (D1) and (H1) is inverted to black to accentuate chromosome structure. In the <em>G. max</em> W82 BAC mapping control experiments (E–G), the upper panels are diagrams of the FISH data (middle panels), and corresponding DAPI-stained chromosomes (lower panels). In the TL-2 BAC mapping experiments (I–K), the lower panels are diagrams of the FISH data (uppermost panels), and corresponding DAPI-stained chromosomes (middle panels). In (E–G) and (I–K), three FISH probe hybridization signals are detected: the BAC probe tested (<em>e.g.</em>, Gm01-BAC-A= “A” in diagrams, with red hybridization signal, indicated by arrowhead), a Gm01-LA marker (“D” in diagrams, indicating the SB86 repeat probe, with blue hybridization signal) and a Gm08-SA marker (“7” in diagrams, indicating Gm08-BAC-7, with green hybridization signal). (E and I) The Gm01-SA probes (Gm01-BAC-A, -B, and -C) hybridized to Gm01-SA in <em>G. max</em> W82 (E1, E3, and E5, respectively), but to the arm opposite the Gm08-SA-homologous arm in TL-2 (I2, I4, and I6, respectively). (F and J) The Gm08-LA probes (Gm08-BAC-1, -2, and -3) hybridized to Gm08-LA in <em>G. max</em> W82 (F2, F4, and F6, respectively), but to the arm opposite the Gm01-LA-homologous arm in TL-2 (J1, J3, and J5, respectively). (G and K) The Gm08-SA probes (Gm08-BAC-4, -5, and -6) hybridized to Gm08-SA in <em>G. max</em> W82 (G2, G4, and G6, respectively), and also to the Gm08-SA-homologous arm in TL-2 (K2, K4, and K6, respectively). The high hybridization background of the red probes used in the experiments in (E5), (I5), (G1), and (K1) obscures the blue SB86 hybridization signal, which more evident when the blue channel is examined individually. The 2-µm scale bar in (H1) is valid for FISH and chromosome panels (D1). The 2-µm scale bar in (K6) is valid for (E–G) and (I–K).</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">Having confirmed the involvement of chromosome 1 in the TL-2 translocation, we sought to characterize the segment juxtaposed to the Gm01-LA-homologous segment in this accession by utilizing the collection of chromosome-assigned BAC probes developed for FISH-based karyotyping of <em>G. max</em> and <em>G. soja</em> (<a id="xref-ref-14-12" class="xref-bibr" href="#ref-14">Findley <em>et al.</em> 2010</a>). To test for a novel association of other chromosome sequences with the Gm01-LA-homologous segment in TL-2, we screened one or two BAC probes derived from each Gm chromosome by FISH. Of the 24 (of 40 possible) chromosome termini probed by these BACs, only the BAC derived from the distal end of the Gm08-LA (BAC GM_WBb0096N20; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000034/-/DC1/TableS2.xlsx">Table S2</a>, line 31) hybridized to the chromosome containing the SB86 repeat probe target (data not shown; see <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>, J1, J3, and J5 for equivalent localization patterns). Thus, in TL-2, a segment homologous to Gm08-LA is associated with a chromosome segment homologous to Gm01-LA. To determine whether the translocation is reciprocal, we used positioned BAC clones (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2, A and C</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000034/-/DC1/TableS2.xlsx">Table S2</a>) to assess associations between chromosome segments homologous to Gm01-SA, Gm01-LA, Gm08-LA, and Gm08-SA in TL-2.</p></div><div id="sec-7" class="subsection"><h3>Gm01-SA BACs</h3><p id="p-19">Having determined that much of the Gm01-LA-homologous region was intact in TL-2, we tested whether sequences corresponding to Gm01-SA are associated with the segment homologous to Gm01-LA. While all <em>G. max</em> chromosomes are characterized by large pericentromeric regions defined by a high content of repetitive DNA elements (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Lin <em>et al.</em> 2005</a>; <a id="xref-ref-35-6" class="xref-bibr" href="#ref-35">Schmutz <em>et al.</em> 2010</a>), Gm01 has an unusually extended high-repeat region (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2A</a>), which significantly constrained BAC probe selection. We tested eight Gm01-SA BACs (including Gm01-BAC-A through -C, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000034/-/DC1/TableS2.xlsx">Table S2</a>; <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2C</a>) in combination with two other probes: SB86 and a Gm08-SA terminal BAC (Gm08-BAC-7). In <em>G. max</em> W82, each of the Gm01-SA-derived BACs hybridized to Gm01-SA (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a>, E1, E3, and E5). In contrast, in TL-2, each of these probes hybridized to the arm opposite to the one hybridizing to Gm08-BAC-7 (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2, I</a><a id="xref-fig-2-10" class="xref-fig" href="#F2">2, I</a><a id="xref-fig-4-2" class="xref-fig" href="#F4">4</a>, and I6), thus indicating a <em>reciprocal</em> translocation between chromosomes 1 and 8 in TL-2 in which the Gg01-SA is associated with the Gg08-SA, and the Gg01-LA is associated with the Gg08-LA.</p></div><div id="sec-8" class="subsection"><h3>Gm08-LA BACs</h3><p id="p-20">To determine the extent of Gm08-LA sequences associated with the Gm01-LA-homologous segment, we tested 15 Gm08-LA-derived BACs, (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2, A and C</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000034/-/DC1/TableS2.xlsx">Table S2</a>). In W82, these BACs hybridized to Gm08-LA at expected positions (<a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2</a>, F2, F4, and F6, diagrammed in <a id="xref-fig-2-13" class="xref-fig" href="#F2">Figure 2C</a> and data not shown). In contrast, in TL-2, each of these probes hybridized to the chromosome arm opposite to the one hybridizing to the SB86 probe (<a id="xref-fig-2-14" class="xref-fig" href="#F2">Figure 2</a>, J1, J3, and J5, diagrammed in <a id="xref-fig-2-15" class="xref-fig" href="#F2">Figure 2C</a> and data not shown). Thus, in TL-2, the majority of the segment homologous to Gm08-LA is associated with the chromosome segment homologous to Gm01-LA, thereby explaining the greater overall length of the aberrant chromosome in this line.</p></div><div id="sec-9" class="subsection"><h3>Gm08-SA BACs</h3><p id="p-21">Finally, we determined the extent of Gm08-SA homologous sequences that were associated with the Gm01-SA-related fragment in TL-2. We tested eight BACs (including Gm08-BAC-4 through -7, <a id="xref-fig-2-16" class="xref-fig" href="#F2">Figure 2C</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000034/-/DC1/TableS2.xlsx">Table S2</a>) spanning Gm08-SA. In <em>G. max</em> W82, each of these BACs hybridized to as expected (<a id="xref-fig-2-17" class="xref-fig" href="#F2">Figure 2</a>, G2, G4, and G6 and data not shown). Similarly, each Gm08-SA probe hybridized to the chromosome arm targeted by the Gm08-SA-terminal probe (Gm08-BAC-7) in TL-2 (<a id="xref-fig-2-18" class="xref-fig" href="#F2">Figure 2</a>, K2, K4, and K6 and data not shown). Therefore, most Gm08-SA-homologous sequences are associated with the tested Gm01-SA-homologous sequences in TL-2.</p><p id="p-22">These experiments indicate a reciprocal translocation in TL-2 between chromosome 1 and chromosome 8 in which a segment homologous to Gm01-SA is associated with a segment homologous to Gm08-SA; whereas a segment homologous to Gm08-LA is associated with a segment homologous to Gm01-LA, resulting in the two chromosome forms shown in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-fig-2-19" class="xref-fig" href="#F2">2B</a>. More precise definition of the translocation breakpoints in chromosomes 1 and 8 was not possible using BAC clones, due to the high repeat content of the pericentromeric regions. However, we could identify translocation chromosomes based on their centromere labeling, because chromosomes in numerous <em>G. soja</em> and <em>G. max</em> lines share similar coloration/intensity of Cent-Gm cocktail hybridization (<a id="xref-ref-14-13" class="xref-bibr" href="#ref-14">Findley <em>et al.</em> 2010</a> and unpublished observations). Using this approach (data not shown), we found that the Gm01-SA-BAC signals to be associated with a chromosome whose centromere color (aqua) resembled Gm01, suggesting that one breakpoint of the translocation (relative to <em>G. max</em>) occurred on the Gm01-LA (<a id="xref-fig-2-20" class="xref-fig" href="#F2">Figure 2A</a>), resulting in the chromosome that we designate Gg01 (<em>Glycine gracilis</em> chromosome 1). However, the centromere color of the second translocation chromosome was not similar to either Gm08 or Gm01, suggesting that the centromere repeats of Gg08 are divergent. Molecular mapping of SSR markers supports breakpoints near the centromeres of both Gg01 and Gg08, with the close linkage of Satt482 (from chromosome 1) and Satt333 (from chromosome 8) being particularly striking (∼1 cM apart; <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>).</p></div><div id="sec-10" class="subsection"><h3>Analysis of translocation line TL-4</h3><p id="p-23">TL-4 formed the ☉6 conformation in translocation heterozygotes with TL-1; because TL-1 involves a translocation between Gm11 and Gm13 (<a id="xref-ref-14-14" class="xref-bibr" href="#ref-14">Findley <em>et al.</em> 2010</a>), it seemed likely that TL-4 would involve a translocation in one of these two chromosomes. Our strategy for characterizing this and other translocations involved three steps: first, assessment of overall chromosome morphology by examining the mutant karyotype; second, identification of candidate translocation chromosomes using the Cent-Gm cocktail in combination with chromosome-specific BAC probes; and finally, translocation chromosome verification and breakpoint estimation using positioned BAC probes (<a id="xref-ref-35-7" class="xref-bibr" href="#ref-35">Schmutz <em>et al.</em> 2010</a>). TL-4 chromosome spreads (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>) revealed two aberrant chromosome pairs: one that was much longer than typical soybean chromosomes (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>, arrowheads; enlarged chromosome in panel E), and a second, much shorter satellite chromosome (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3B</a>, arrows; enlarged chromosome in panel D). Cent-Gm cocktail hybridization (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3A</a>, arrowheads; enlarged chromosome in panel F) indicated that the shorter chromosome was likely a truncated form of Gm13; whereas the blue-green color of the larger chromosome centromere (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3A</a>, arrows; enlarged in panel G) suggested that it might be a modified form of Gm02, Gm04, Gm10, or Gm17. FISH with paired, chromosome-specific, terminal-position BAC probes for Gm02, Gm10, and Gm17 labeled opposite termini of single chromosome pairs with more typical morphology, indicating that these chromosomes were intact in TL-4 (data not shown). In contrast, the Gm04 terminal BAC probes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000034/-/DC1/TableS2.xlsx">Table S2</a> and Figure 3H4), both labeled the aberrantly large TL-4 chromosome (Figure 3H6). The signal for the 5′ Gm04 BAC (labeled “1” in <a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3</a>, H1, H2, and K1) labeled near a chromosome terminus in TL-4 (Figure 3H6, red signal, indicated by arrow), as it did in <em>G. max</em> W82 (Figure 3H4, red signal, indicated by arrow), indicating that the 5′ end of TL-4 Gm04 was intact. In contrast, the 3′ Gm04 BAC (labeled “2” in <a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3</a>, H1, H2, and K1) hybridized internally within the aberrantly long arm, suggesting a distal addition to an otherwise intact Gm04 in this mutant.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/2/117/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="FISH-based characterization of translocation line TL-4. (A) Cent-Gm&#x2013;based FISH karyotyping of a TL-4 chromosome spread; (B) DAPI channel of the chromosomes in (A). The two outline-arrows in (A and B) indicate the aberrantly small chromosome pair; the two arrowheads indicate the aberrantly large chromosome pair. (C&#x2013;E) Enlarged images of DAPI-stained chromosomes: (C) a normal Gm13 in G. max W82, (D) the aberrantly small TL-4 chromosome, (E) the aberrantly large TL-4 chromosome. (F) and (G) show Cent-Gm&#x2013;based FISH hybridization of the chromosomes in (D) and (E), respectively. (H&#x2013;K) FISH utilizing pseudomolecule-derived BAC probes to map the TL-4 translocation. The G. max W82 chromosome diagrams in each column 1 panel correspond to the G. max W82 FISH images in columns 3 and 4; the TL-4 chromosome diagrams in each column 2 panel correspond to the TL-4 FISH images in columns 5 and 6. In columns 3&#x2013;6, FISH hybridization panels are shown above their corresponding DAPI-stained counterparts. (Row H) In G. max W82 (H1), Gm04-BAC-1 (labeled &#x201C;1&#x201D; in column 1) hybridized to one end of Gm04 (red hybridization signal in H4, indicated by arrow); Gm04-BAC-2 (labeled &#x201C;2&#x201D; in column 1) hybridized to the opposite end of Gm04 (green hybridization signal in H4, indicated by arrowhead). In TL-4 (H2), Gm04-BAC-1 (labeled &#x201C;1&#x201D; in column 2) hybridized to the short arm of the large chromosome (red hybridization signal in H4, indicated by arrow); Gm04-BAC-2 (labeled &#x201C;2&#x201D; in column 2) hybridized internally, within the longer arm of this chromosome (green hybridization signal in H6, indicated by arrowhead). (Row I) In G. max W82 (I1), Gm13-BAC-C (labeled &#x201C;C&#x201D; in column 1) hybridized to the distal long arm of Gm13 (red hybridization signal in I3, indicated by arrow); whereas Gm04-BAC-2 (labeled &#x201C;2&#x201D; in column 1) hybridized to the 3&#x2032; end of Gm04 (green hybridization signal in I4, indicated by arrowhead). In TL-4 (I2), Gm13-BAC-C (labeled &#x201C;C&#x201D; in column 2) hybridized to a distal position on the longer arm of the large chromosome (red hybridization signal in I6, indicated by arrow); Gm04-BAC-2 (labeled &#x201C;2&#x201D; in column 2) hybridized internally, to the long arm of the same chromosome (green hybridization signal in I6, indicated by arrowhead). (Row J) In G. max W82 (J1), Gm13-BAC-B (labeled &#x201C;B&#x201D; in column 1) hybridized to a centromere-proximal position on the long arm of Gm13 (red hybridization signal in J3, indicated by arrow); Gm04-BAC-2 (labeled &#x201C;2&#x201D; in column 1) hybridized to the 3&#x2032; end of Gm04 (green hybridization signal in J4, indicated by arrowhead). In TL-4 (J2), Gm13-BAC-B (labeled &#x201C;B&#x201D; in column 2) hybridized internally within longer arm of the large chromosome (red hybridization signal in J6, indicated by arrow), close to the position of Gm04-BAC-2 (labeled &#x201C;2&#x201D; in column 2) on the long arm of the same chromosome (green hybridization signal in J6, indicated by arrowhead). (Row K) Translocation model for TL-4. (K1) A segment of Gm13 containing at least those sequences corresponding to Gm13-BAC-A through Gm13-BAC-C translocated (blue shading with arrow) to the 3&#x2032; end of Gm04, resulting in the TL-4 chromosome forms shown in (K2). (K3) Additional TL-4 FISH image with the most centromere-proximal Gm13 probe, Gm13-BAC-A (labeled &#x201C;A&#x201D; in column 2), which hybridized internally within longer arm of the large chromosome (red hybridization signal in K6, indicated by arrow), close to the position of Gm04-BAC-2 (labeled &#x201C;2&#x201D; in column 2) on the long arm of the same chromosome (green hybridization signal in K6, indicated by arrowhead). In columns H1&#x2013;K1, the diagram for Gm04 is inverted so that the 5&#x2032; end of the chromosome (labeled &#x201C;LO&#x201D;) is on the bottom, whereas the 3&#x2032; end (labeled &#x201C;HI&#x201D;) is on the top. Unless otherwise indicated in this way in subsequent figures, the top of a chromosome corresponds to the 5&#x2032; end of the corresponding pseudomolecule. The 2-&#xB5;m scale bar in (B) is also valid for (A). The 2-&#xB5;m scale bar in (G) is valid for (C&#x2013;G). The 2-&#xB5;m scale bar in (J6) is valid for (H&#x2013;K)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-498101292" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;FISH-based characterization of translocation line TL-4. (A) Cent-Gm&#x2013;based FISH karyotyping of a TL-4 chromosome spread; (B) DAPI channel of the chromosomes in (A). The two outline-arrows in (A and B) indicate the aberrantly small chromosome pair; the two arrowheads indicate the aberrantly large chromosome pair. (C&#x2013;E) Enlarged images of DAPI-stained chromosomes: (C) a normal Gm13 in &lt;em&gt;G. max&lt;/em&gt; W82, (D) the aberrantly small TL-4 chromosome, (E) the aberrantly large TL-4 chromosome. (F) and (G) show Cent-Gm&#x2013;based FISH hybridization of the chromosomes in (D) and (E), respectively. (H&#x2013;K) FISH utilizing pseudomolecule-derived BAC probes to map the TL-4 translocation. The &lt;em&gt;G. max&lt;/em&gt; W82 chromosome diagrams in each column 1 panel correspond to the &lt;em&gt;G. max&lt;/em&gt; W82 FISH images in columns 3 and 4; the TL-4 chromosome diagrams in each column 2 panel correspond to the TL-4 FISH images in columns 5 and 6. In columns 3&#x2013;6, FISH hybridization panels are shown above their corresponding DAPI-stained counterparts. (Row H) In &lt;em&gt;G. max&lt;/em&gt; W82 (H1), Gm04-BAC-1 (labeled &#x201C;1&#x201D; in column 1) hybridized to one end of Gm04 (red hybridization signal in H4, indicated by arrow); Gm04-BAC-2 (labeled &#x201C;2&#x201D; in column 1) hybridized to the opposite end of Gm04 (green hybridization signal in H4, indicated by arrowhead). In TL-4 (H2), Gm04-BAC-1 (labeled &#x201C;1&#x201D; in column 2) hybridized to the short arm of the large chromosome (red hybridization signal in H4, indicated by arrow); Gm04-BAC-2 (labeled &#x201C;2&#x201D; in column 2) hybridized internally, within the longer arm of this chromosome (green hybridization signal in H6, indicated by arrowhead). (Row I) In &lt;em&gt;G. max&lt;/em&gt; W82 (I1), Gm13-BAC-C (labeled &#x201C;C&#x201D; in column 1) hybridized to the distal long arm of Gm13 (red hybridization signal in I3, indicated by arrow); whereas Gm04-BAC-2 (labeled &#x201C;2&#x201D; in column 1) hybridized to the 3&#x2032; end of Gm04 (green hybridization signal in I4, indicated by arrowhead). In TL-4 (I2), Gm13-BAC-C (labeled &#x201C;C&#x201D; in column 2) hybridized to a distal position on the longer arm of the large chromosome (red hybridization signal in I6, indicated by arrow); Gm04-BAC-2 (labeled &#x201C;2&#x201D; in column 2) hybridized internally, to the long arm of the same chromosome (green hybridization signal in I6, indicated by arrowhead). (Row J) In &lt;em&gt;G. max&lt;/em&gt; W82 (J1), Gm13-BAC-B (labeled &#x201C;B&#x201D; in column 1) hybridized to a centromere-proximal position on the long arm of Gm13 (red hybridization signal in J3, indicated by arrow); Gm04-BAC-2 (labeled &#x201C;2&#x201D; in column 1) hybridized to the 3&#x2032; end of Gm04 (green hybridization signal in J4, indicated by arrowhead). In TL-4 (J2), Gm13-BAC-B (labeled &#x201C;B&#x201D; in column 2) hybridized internally within longer arm of the large chromosome (red hybridization signal in J6, indicated by arrow), close to the position of Gm04-BAC-2 (labeled &#x201C;2&#x201D; in column 2) on the long arm of the same chromosome (green hybridization signal in J6, indicated by arrowhead). (Row K) Translocation model for TL-4. (K1) A segment of Gm13 containing at least those sequences corresponding to Gm13-BAC-A through Gm13-BAC-C translocated (blue shading with arrow) to the 3&#x2032; end of Gm04, resulting in the TL-4 chromosome forms shown in (K2). (K3) Additional TL-4 FISH image with the most centromere-proximal Gm13 probe, Gm13-BAC-A (labeled &#x201C;A&#x201D; in column 2), which hybridized internally within longer arm of the large chromosome (red hybridization signal in K6, indicated by arrow), close to the position of Gm04-BAC-2 (labeled &#x201C;2&#x201D; in column 2) on the long arm of the same chromosome (green hybridization signal in K6, indicated by arrowhead). In columns H1&#x2013;K1, the diagram for Gm04 is inverted so that the 5&#x2032; end of the chromosome (labeled &#x201C;LO&#x201D;) is on the bottom, whereas the 3&#x2032; end (labeled &#x201C;HI&#x201D;) is on the top. Unless otherwise indicated in this way in subsequent figures, the top of a chromosome corresponds to the 5&#x2032; end of the corresponding pseudomolecule. The 2-&#xB5;m scale bar in (B) is also valid for (A). The 2-&#xB5;m scale bar in (G) is valid for (C&#x2013;G). The 2-&#xB5;m scale bar in (J6) is valid for (H&#x2013;K).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/2/117/F3.medium.gif" width="440" height="253"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/1/2/117/F3.medium.gif" width="440" height="253"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/2/117/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/2/117/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460660" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-24" class="first-child">FISH-based characterization of translocation line TL-4. (A) Cent-Gm–based FISH karyotyping of a TL-4 chromosome spread; (B) DAPI channel of the chromosomes in (A). The two outline-arrows in (A and B) indicate the aberrantly small chromosome pair; the two arrowheads indicate the aberrantly large chromosome pair. (C–E) Enlarged images of DAPI-stained chromosomes: (C) a normal Gm13 in <em>G. max</em> W82, (D) the aberrantly small TL-4 chromosome, (E) the aberrantly large TL-4 chromosome. (F) and (G) show Cent-Gm–based FISH hybridization of the chromosomes in (D) and (E), respectively. (H–K) FISH utilizing pseudomolecule-derived BAC probes to map the TL-4 translocation. The <em>G. max</em> W82 chromosome diagrams in each column 1 panel correspond to the <em>G. max</em> W82 FISH images in columns 3 and 4; the TL-4 chromosome diagrams in each column 2 panel correspond to the TL-4 FISH images in columns 5 and 6. In columns 3–6, FISH hybridization panels are shown above their corresponding DAPI-stained counterparts. (Row H) In <em>G. max</em> W82 (H1), Gm04-BAC-1 (labeled “1” in column 1) hybridized to one end of Gm04 (red hybridization signal in H4, indicated by arrow); Gm04-BAC-2 (labeled “2” in column 1) hybridized to the opposite end of Gm04 (green hybridization signal in H4, indicated by arrowhead). In TL-4 (H2), Gm04-BAC-1 (labeled “1” in column 2) hybridized to the short arm of the large chromosome (red hybridization signal in H4, indicated by arrow); Gm04-BAC-2 (labeled “2” in column 2) hybridized internally, within the longer arm of this chromosome (green hybridization signal in H6, indicated by arrowhead). (Row I) In <em>G. max</em> W82 (I1), Gm13-BAC-C (labeled “C” in column 1) hybridized to the distal long arm of Gm13 (red hybridization signal in I3, indicated by arrow); whereas Gm04-BAC-2 (labeled “2” in column 1) hybridized to the 3′ end of Gm04 (green hybridization signal in I4, indicated by arrowhead). In TL-4 (I2), Gm13-BAC-C (labeled “C” in column 2) hybridized to a distal position on the longer arm of the large chromosome (red hybridization signal in I6, indicated by arrow); Gm04-BAC-2 (labeled “2” in column 2) hybridized internally, to the long arm of the same chromosome (green hybridization signal in I6, indicated by arrowhead). (Row J) In <em>G. max</em> W82 (J1), Gm13-BAC-B (labeled “B” in column 1) hybridized to a centromere-proximal position on the long arm of Gm13 (red hybridization signal in J3, indicated by arrow); Gm04-BAC-2 (labeled “2” in column 1) hybridized to the 3′ end of Gm04 (green hybridization signal in J4, indicated by arrowhead). In TL-4 (J2), Gm13-BAC-B (labeled “B” in column 2) hybridized internally within longer arm of the large chromosome (red hybridization signal in J6, indicated by arrow), close to the position of Gm04-BAC-2 (labeled “2” in column 2) on the long arm of the same chromosome (green hybridization signal in J6, indicated by arrowhead). (Row K) Translocation model for TL-4. (K1) A segment of Gm13 containing at least those sequences corresponding to Gm13-BAC-A through Gm13-BAC-C translocated (blue shading with arrow) to the 3′ end of Gm04, resulting in the TL-4 chromosome forms shown in (K2). (K3) Additional TL-4 FISH image with the most centromere-proximal Gm13 probe, Gm13-BAC-A (labeled “A” in column 2), which hybridized internally within longer arm of the large chromosome (red hybridization signal in K6, indicated by arrow), close to the position of Gm04-BAC-2 (labeled “2” in column 2) on the long arm of the same chromosome (green hybridization signal in K6, indicated by arrowhead). In columns H1–K1, the diagram for Gm04 is inverted so that the 5′ end of the chromosome (labeled “LO”) is on the bottom, whereas the 3′ end (labeled “HI”) is on the top. Unless otherwise indicated in this way in subsequent figures, the top of a chromosome corresponds to the 5′ end of the corresponding pseudomolecule. The 2-µm scale bar in (B) is also valid for (A). The 2-µm scale bar in (G) is valid for (C–G). The 2-µm scale bar in (J6) is valid for (H–K).</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/2/117/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="FISH-based characterization of translocation line TL-3. (A) FISH with Cent-Gm cocktail supplemented with an 18S-rDNA probe of a TL-3 chromosome spread. (B) The DAPI channel of the chromosomes in (A). The two outline-arrows in (A) and (B) indicate the aberrantly large chromosome pair that also hybridized to the rDNA probe; whereas the two arrowheads indicate the aberrantly small chromosome pair. (C&#x2013;E) Enlarged images of DAPI-stained chromosomes: (C) a normal Gm13 in G. max W82, (D) the aberrantly large TL-3 chromosome, (E) the aberrantly small TL-3 chromosome. (F) and (G) show 18S-rDNA probe hybridization (red signal) of the chromosomes in (C) and (D), respectively; whereas the aberrantly small TL-3 chromosome in (H) is hybridized to the Cent-Gm probe cocktail. Panels (I) and (K) are diagrammatic summaries of the FISH experiments performed in (K-L) using a series of Gm13-LA BAC probes (Gm13-BAC-1 through -11; see Table S2). (K1&#x2013;K10) The upper, middle, and lower panels are the chromosome diagram, FISH signal, and DAPI signal, respectively, for G. max W82 Gm13. The chromosome in each FISH panel has signal derived from three probes: a single tested Gm13-LA BAC (red hybridization signal, indicated in the K1 FISH panel by the arrow), plus Gm13-BAC-11 (green, chromosome-terminal hybridization signal, indicated in the K1 FISH panel by the arrowhead) and the Cent-Gm cocktail (the green, centromeric hybridization signal, indicated in the K1 FISH panel by the dot). In (K1&#x2013;K10), the Gm13 BAC probes hybridized to positions spanning from Gm13-LA-proximal (K1) to Gm13-LA-terminal (K10). (L1&#x2013;L10) The upper, middle, and lower panels are the FISH signal, DAPI signal, and the chromosome diagram, respectively, for TL-3 Gm13. The chromosome in each FISH panel has signal derived from three probes, as described for (K1&#x2013;K10). In (L1&#x2013;L10), Gm13-BAC-11 hybridized to a terminal position on the shorter arm of the aberrant Gm13 form in TL-3. (L1&#x2013;L3) Only the three most centromere-proximal Gm13-BAC probes (Gm-BAC-1 through -3) hybridized near the centromere of the longer arm of the mutant Gm13. (L4&#x2013;L10) Each of the remaining Gm13 BAC probes (Gm13-BAC-4 through -10) hybridized sequentially more distally, on the shorter arm of TL-3 Gm13. (M and N) FISH utilizing pseudomolecule-derived BAC probes to map the TL-3 translocation. The G. max W82 chromosome diagrams in each column 1 panel correspond to the G. max W82 FISH images in columns 3 and 4; the TL-3 chromosome diagrams in each column 2 panel correspond to the TL-3 FISH images in columns 5 and 6. In columns 3-6, FISH hybridization panels are shown above their corresponding DAPI-stained chromosomes. (Row M) In G. max W82 (M1), Gm05-BAC-A (labeled &#x201C;A&#x201D; in column 1) hybridized to one end of Gm05 (red hybridization signal in M4, indicated by arrow); Gm05-BAC-B (labeled &#x201C;B&#x201D; in column 1) hybridized to the opposite end of Gm05 (green hybridization signal in M4, indicated by arrowhead). In TL-3, Gm05-BAC-A (labeled &#x201C;A&#x201D; in column 2) hybridized to a centromere-distal position on the arm of the aberrantly small chromosome (green hybridization signal in H6, indicated by arrowhead); Gm05-BAC-B (labeled &#x201C;B&#x201D; in column 2) hybridized to the distal end of the longer arm of the aberrant Gm13 (red hybridization signal in (H5), indicated by arrow). (Row N) In G. max W82, Gm13-BAC-11 (labeled &#x201C;11&#x201D; in column 1) hybridized to the distal end of the Gm13-LA (red hybridization signal in (N3), indicated by arrow); Gm05-BAC-B (labeled &#x201C;B&#x201D; in column 1) hybridized to the 3&#x2032; end of Gm05 (green hybridization signal in (N4), indicated by arrowhead). In TL-3 (N2), Gm13-BAC-11 (labeled &#x201C;11&#x201D; in column 2) hybridized to a distal position on the shorter arm of the aberrant Gm13 (red hybridization signal in (N5), indicated by arrow); Gm04-BAC-B (labeled &#x201C;B&#x201D; in column 2) hybridized near the terminus of the longer arm of the same chromosome (green hybridization signal in (N5), indicated by arrowhead). (Row O) Model of the events that generated the chromosome forms in TL-3. (O1) Diagram of the normal Gm13. &#x201C;Position&#x201D; indicates the position, in Mb of the central bp of a given BAC in the Gm13 pseudomolecule (e.g., Gm13-BAC-3 is at 28.6 Mb in the Gm13 pseudomolecule; see Table S2 for details); whereas &#x201C;Interval&#x201D; numbers represent the window in Mb between BAC probe &#x201C;Position&#x201D; numbers. Thus, in the first event, a chromosome break occurred in the 2.9 Mb interval between sequences corresponding to Gm13-BAC-3 and Gm13-BAC-4, thereby releasing a &#x223C;12.9 Mb segment of the Gm13-LA. This segment then became associated with the NOR-containing satellite arm (O2, left chromosome). The yellow arrow in (O1&#x2013;O3) indicates the polarity of the translocated Gm13-LA fragment; the blue box indicates the possibility that Gm13 may have experienced a pericentromeric inversion involving two breaks. Next, a fragment comprising most of 3&#x2032; end of Gm05 (O2, right chromosome) was translocated to the broken end of the Gm13-LA to generate the chromosome forms in O3. The gray arrow in (O2 and O3) indicates the polarity of the translocated Gm05 fragment. The 2-&#xB5;m scale bar in (A) is also valid for (B). The 2-&#xB5;m scale bar in (G) is valid for (C&#x2013;H). The 2-&#xB5;m scale bar in (K10) is valid for (K&#x2013;L). The 2-&#xB5;m scale bar in (N6) is valid for (M) and (N)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-498101292" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;FISH-based characterization of translocation line TL-3. (A) FISH with Cent-Gm cocktail supplemented with an 18S-rDNA probe of a TL-3 chromosome spread. (B) The DAPI channel of the chromosomes in (A). The two outline-arrows in (A) and (B) indicate the aberrantly large chromosome pair that also hybridized to the rDNA probe; whereas the two arrowheads indicate the aberrantly small chromosome pair. (C&#x2013;E) Enlarged images of DAPI-stained chromosomes: (C) a normal Gm13 in &lt;em&gt;G. max&lt;/em&gt; W82, (D) the aberrantly large TL-3 chromosome, (E) the aberrantly small TL-3 chromosome. (F) and (G) show 18S-rDNA probe hybridization (red signal) of the chromosomes in (C) and (D), respectively; whereas the aberrantly small TL-3 chromosome in (H) is hybridized to the Cent-Gm probe cocktail. Panels (I) and (K) are diagrammatic summaries of the FISH experiments performed in (K-L) using a series of Gm13-LA BAC probes (Gm13-BAC-1 through -11; see Table S2). (K1&#x2013;K10) The upper, middle, and lower panels are the chromosome diagram, FISH signal, and DAPI signal, respectively, for &lt;em&gt;G. max&lt;/em&gt; W82 Gm13. The chromosome in each FISH panel has signal derived from three probes: a single tested Gm13-LA BAC (red hybridization signal, indicated in the K1 FISH panel by the arrow), plus Gm13-BAC-11 (green, chromosome-terminal hybridization signal, indicated in the K1 FISH panel by the arrowhead) and the Cent-Gm cocktail (the green, centromeric hybridization signal, indicated in the K1 FISH panel by the dot). In (K1&#x2013;K10), the Gm13 BAC probes hybridized to positions spanning from Gm13-LA-proximal (K1) to Gm13-LA-terminal (K10). (L1&#x2013;L10) The upper, middle, and lower panels are the FISH signal, DAPI signal, and the chromosome diagram, respectively, for TL-3 Gm13. The chromosome in each FISH panel has signal derived from three probes, as described for (K1&#x2013;K10). In (L1&#x2013;L10), Gm13-BAC-11 hybridized to a terminal position on the shorter arm of the aberrant Gm13 form in TL-3. (L1&#x2013;L3) Only the three most centromere-proximal Gm13-BAC probes (Gm-BAC-1 through -3) hybridized near the centromere of the longer arm of the mutant Gm13. (L4&#x2013;L10) Each of the remaining Gm13 BAC probes (Gm13-BAC-4 through -10) hybridized sequentially more distally, on the shorter arm of TL-3 Gm13. (M and N) FISH utilizing pseudomolecule-derived BAC probes to map the TL-3 translocation. The &lt;em&gt;G. max&lt;/em&gt; W82 chromosome diagrams in each column 1 panel correspond to the &lt;em&gt;G. max&lt;/em&gt; W82 FISH images in columns 3 and 4; the TL-3 chromosome diagrams in each column 2 panel correspond to the TL-3 FISH images in columns 5 and 6. In columns 3-6, FISH hybridization panels are shown above their corresponding DAPI-stained chromosomes. (Row M) In &lt;em&gt;G. max&lt;/em&gt; W82 (M1), Gm05-BAC-A (labeled &#x201C;A&#x201D; in column 1) hybridized to one end of Gm05 (red hybridization signal in M4, indicated by arrow); Gm05-BAC-B (labeled &#x201C;B&#x201D; in column 1) hybridized to the opposite end of Gm05 (green hybridization signal in M4, indicated by arrowhead). In TL-3, Gm05-BAC-A (labeled &#x201C;A&#x201D; in column 2) hybridized to a centromere-distal position on the arm of the aberrantly small chromosome (green hybridization signal in H6, indicated by arrowhead); Gm05-BAC-B (labeled &#x201C;B&#x201D; in column 2) hybridized to the distal end of the longer arm of the aberrant Gm13 (red hybridization signal in (H5), indicated by arrow). (Row N) In &lt;em&gt;G. max&lt;/em&gt; W82, Gm13-BAC-11 (labeled &#x201C;11&#x201D; in column 1) hybridized to the distal end of the Gm13-LA (red hybridization signal in (N3), indicated by arrow); Gm05-BAC-B (labeled &#x201C;B&#x201D; in column 1) hybridized to the 3&#x2032; end of Gm05 (green hybridization signal in (N4), indicated by arrowhead). In TL-3 (N2), Gm13-BAC-11 (labeled &#x201C;11&#x201D; in column 2) hybridized to a distal position on the shorter arm of the aberrant Gm13 (red hybridization signal in (N5), indicated by arrow); Gm04-BAC-B (labeled &#x201C;B&#x201D; in column 2) hybridized near the terminus of the longer arm of the same chromosome (green hybridization signal in (N5), indicated by arrowhead). (Row O) Model of the events that generated the chromosome forms in TL-3. (O1) Diagram of the normal Gm13. &#x201C;Position&#x201D; indicates the position, in Mb of the central bp of a given BAC in the Gm13 pseudomolecule (&lt;em&gt;e.g.&lt;/em&gt;, Gm13-BAC-3 is at 28.6 Mb in the Gm13 pseudomolecule; see Table S2 for details); whereas &#x201C;Interval&#x201D; numbers represent the window in Mb between BAC probe &#x201C;Position&#x201D; numbers. Thus, in the first event, a chromosome break occurred in the 2.9 Mb interval between sequences corresponding to Gm13-BAC-3 and Gm13-BAC-4, thereby releasing a &#x223C;12.9 Mb segment of the Gm13-LA. This segment then became associated with the NOR-containing satellite arm (O2, left chromosome). The yellow arrow in (O1&#x2013;O3) indicates the polarity of the translocated Gm13-LA fragment; the blue box indicates the possibility that Gm13 may have experienced a pericentromeric inversion involving two breaks. Next, a fragment comprising most of 3&#x2032; end of Gm05 (O2, right chromosome) was translocated to the broken end of the Gm13-LA to generate the chromosome forms in O3. The gray arrow in (O2 and O3) indicates the polarity of the translocated Gm05 fragment. The 2-&#xB5;m scale bar in (A) is also valid for (B). The 2-&#xB5;m scale bar in (G) is valid for (C&#x2013;H). The 2-&#xB5;m scale bar in (K10) is valid for (K&#x2013;L). The 2-&#xB5;m scale bar in (N6) is valid for (M) and (N).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/2/117/F4.medium.gif" width="231" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/1/2/117/F4.medium.gif" width="231" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/2/117/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/2/117/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460807" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-25" class="first-child">FISH-based characterization of translocation line TL-3. (A) FISH with Cent-Gm cocktail supplemented with an 18S-rDNA probe of a TL-3 chromosome spread. (B) The DAPI channel of the chromosomes in (A). The two outline-arrows in (A) and (B) indicate the aberrantly large chromosome pair that also hybridized to the rDNA probe; whereas the two arrowheads indicate the aberrantly small chromosome pair. (C–E) Enlarged images of DAPI-stained chromosomes: (C) a normal Gm13 in <em>G. max</em> W82, (D) the aberrantly large TL-3 chromosome, (E) the aberrantly small TL-3 chromosome. (F) and (G) show 18S-rDNA probe hybridization (red signal) of the chromosomes in (C) and (D), respectively; whereas the aberrantly small TL-3 chromosome in (H) is hybridized to the Cent-Gm probe cocktail. Panels (I) and (K) are diagrammatic summaries of the FISH experiments performed in (K-L) using a series of Gm13-LA BAC probes (Gm13-BAC-1 through -11; see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000034/-/DC1/TableS2.xlsx">Table S2</a>). (K1–K10) The upper, middle, and lower panels are the chromosome diagram, FISH signal, and DAPI signal, respectively, for <em>G. max</em> W82 Gm13. The chromosome in each FISH panel has signal derived from three probes: a single tested Gm13-LA BAC (red hybridization signal, indicated in the K1 FISH panel by the arrow), plus Gm13-BAC-11 (green, chromosome-terminal hybridization signal, indicated in the K1 FISH panel by the arrowhead) and the Cent-Gm cocktail (the green, centromeric hybridization signal, indicated in the K1 FISH panel by the dot). In (K1–K10), the Gm13 BAC probes hybridized to positions spanning from Gm13-LA-proximal (K1) to Gm13-LA-terminal (K10). (L1–L10) The upper, middle, and lower panels are the FISH signal, DAPI signal, and the chromosome diagram, respectively, for TL-3 Gm13. The chromosome in each FISH panel has signal derived from three probes, as described for (K1–K10). In (L1–L10), Gm13-BAC-11 hybridized to a terminal position on the shorter arm of the aberrant Gm13 form in TL-3. (L1–L3) Only the three most centromere-proximal Gm13-BAC probes (Gm-BAC-1 through -3) hybridized near the centromere of the longer arm of the mutant Gm13. (L4–L10) Each of the remaining Gm13 BAC probes (Gm13-BAC-4 through -10) hybridized sequentially more distally, on the shorter arm of TL-3 Gm13. (M and N) FISH utilizing pseudomolecule-derived BAC probes to map the TL-3 translocation. The <em>G. max</em> W82 chromosome diagrams in each column 1 panel correspond to the <em>G. max</em> W82 FISH images in columns 3 and 4; the TL-3 chromosome diagrams in each column 2 panel correspond to the TL-3 FISH images in columns 5 and 6. In columns 3-6, FISH hybridization panels are shown above their corresponding DAPI-stained chromosomes. (Row M) In <em>G. max</em> W82 (M1), Gm05-BAC-A (labeled “A” in column 1) hybridized to one end of Gm05 (red hybridization signal in M4, indicated by arrow); Gm05-BAC-B (labeled “B” in column 1) hybridized to the opposite end of Gm05 (green hybridization signal in M4, indicated by arrowhead). In TL-3, Gm05-BAC-A (labeled “A” in column 2) hybridized to a centromere-distal position on the arm of the aberrantly small chromosome (green hybridization signal in H6, indicated by arrowhead); Gm05-BAC-B (labeled “B” in column 2) hybridized to the distal end of the longer arm of the aberrant Gm13 (red hybridization signal in (H5), indicated by arrow). (Row N) In <em>G. max</em> W82, Gm13-BAC-11 (labeled “11” in column 1) hybridized to the distal end of the Gm13-LA (red hybridization signal in (N3), indicated by arrow); Gm05-BAC-B (labeled “B” in column 1) hybridized to the 3′ end of Gm05 (green hybridization signal in (N4), indicated by arrowhead). In TL-3 (N2), Gm13-BAC-11 (labeled “11” in column 2) hybridized to a distal position on the shorter arm of the aberrant Gm13 (red hybridization signal in (N5), indicated by arrow); Gm04-BAC-B (labeled “B” in column 2) hybridized near the terminus of the longer arm of the same chromosome (green hybridization signal in (N5), indicated by arrowhead). (Row O) Model of the events that generated the chromosome forms in TL-3. (O1) Diagram of the normal Gm13. “Position” indicates the position, in Mb of the central bp of a given BAC in the Gm13 pseudomolecule (<em>e.g.</em>, Gm13-BAC-3 is at 28.6 Mb in the Gm13 pseudomolecule; see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000034/-/DC1/TableS2.xlsx">Table S2</a> for details); whereas “Interval” numbers represent the window in Mb between BAC probe “Position” numbers. Thus, in the first event, a chromosome break occurred in the 2.9 Mb interval between sequences corresponding to Gm13-BAC-3 and Gm13-BAC-4, thereby releasing a ∼12.9 Mb segment of the Gm13-LA. This segment then became associated with the NOR-containing satellite arm (O2, left chromosome). The yellow arrow in (O1–O3) indicates the polarity of the translocated Gm13-LA fragment; the blue box indicates the possibility that Gm13 may have experienced a pericentromeric inversion involving two breaks. Next, a fragment comprising most of 3′ end of Gm05 (O2, right chromosome) was translocated to the broken end of the Gm13-LA to generate the chromosome forms in O3. The gray arrow in (O2 and O3) indicates the polarity of the translocated Gm05 fragment. The 2-µm scale bar in (A) is also valid for (B). The 2-µm scale bar in (G) is valid for (C–H). The 2-µm scale bar in (K10) is valid for (K–L). The 2-µm scale bar in (N6) is valid for (M) and (N).</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">Due to the short arm length of the satellite chromosome in TL-4 (compare <a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3D</a> [TL-4] with panel C [<em>G. max</em> W82]), we hypothesized that a fragment of Gm13-LA might be translocated to Gm04 in this line. Indeed, Gm13-BAC-C (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000034/-/DC1/TableS2.xlsx">Table S2</a>, line 3), which hybridizes near the terminus of Gm13-LA in <em>G. max</em> (Figure 3I3), hybridized to the terminus of the longer arm of the aberrant Gm04 in TL-4 (Figure 3I6). This result both confirmed the involvement of Gm13 in the translocation and indicated the polarity of the segment translocated; specifically, the Gm13-LA terminus now represents the terminus of Gm04-LA in TL-4. To determine how much of the Gm13-LA was translocated onto Gm04 in TL-4, we used Gm13-LA BAC probes (<a id="xref-ref-14-15" class="xref-bibr" href="#ref-14">Findley <em>et al.</em> 2010</a>). Gm13-BAC-B represents the most centromere-proximal BAC of a continuous series of BACs that validated the genome sequence assembly of Gm13-LA (<a id="xref-fig-3-11" class="xref-fig" href="#F3">Figure 3</a>, J1 and J3; <a id="xref-ref-14-16" class="xref-bibr" href="#ref-14">Findley <em>et al.</em> 2010</a>). In TL-4, this BAC also hybridized to the longer arm of the mutant Gm04, near to the Gm04 3′ terminal probe (<a id="xref-fig-3-12" class="xref-fig" href="#F3">Figure 3</a>, J2 and J6). Finally, although not contiguous with the other Gm13 BAC probes, Gm13-BAC-A is one of the most 5′ terminal BACs available for the Gm13 pseudomolecule; in TL-4, this probe also hybridized very near to the signal for Gm04-BAC-2, on the longer arm of the mutant Gm04 (Figure 3K6).</p><p id="p-27">Altogether, our FISH mapping suggests a model (<a id="xref-fig-3-13" class="xref-fig" href="#F3">Figure 3</a>, K1 and K2) for a translocation between Gm13 and Gm04 in which a large fragment (&gt;17 Mb) of Gm13 has translocated to the 3′ terminus of Gm04 in TL-4. Because the most terminal Gm04 BAC tested (Gm04-BAC-2) is positioned at 44.1 Mb of the 49.2 Mb <em>G. max</em> W82 pseudomolecule, and also due to the lack of more centromere-proximal Gm13-LA probes, our analysis does not rule out the possibility of a reciprocal translocation in which a small fragment of Gm04 was transferred to the truncated TL-4 Gm13-LA.</p></div><div id="sec-11" class="subsection"><h3>Analysis of translocation line TL-3</h3><p id="p-28">Because TL-3 formed the ☉6 conformation in translocation heterozygote combinations with either TL-1 (which involves Gm13 and Gm11; <a id="xref-ref-14-17" class="xref-bibr" href="#ref-14">Findley <em>et al.</em> 2010</a>) or TL-4 (which involves Gm13 and Gm04, discussed above), it seemed likely that TL-3 would have a rearranged Gm13. TL-3 had two aberrant chromosomes: one that was truncated at or near to a centromere, so that it appeared to contain only single chromosome arms (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>, arrowheads; enlarged in <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4E</a>), and second that was longer than any other (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a>, arrows; enlarged in <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4D</a>). Furthermore, because a normal Gm13, with its characteristic NOR-containing knob (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4C</a>), was not evident in TL-3 chromosome spreads, we hypothesized that this large chromosome was an aberrant form of Gm13. We confirmed this in FISH performed using the Cent-Gm cocktail supplemented with an 18S-rDNA probe (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4A</a>). The centromere color (pale green) of the large chromosome was typical of Gm13 in other <em>G. max</em> cultivars (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4A</a>; enlargement not shown). More definitively, the central, low-intensity DAPI-stained region of the chromosome was targeted by the rDNA probe (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4, D and G</a>; compare to W82 in <a id="xref-fig-4-11" class="xref-fig" href="#F4">Figure 4, C and F</a>), indicating that it was the NOR. Thus, the larger aberrant chromosome in TL-3 was a rearranged form of Gm13.</p></div><div id="sec-12" class="subsection"><h3>Analysis of Gm13 in TL-3</h3><p id="p-29">To assess the chromosome structure of the TL-3 Gm13, we tested a series of BACs (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000034/-/DC1/TableS2.xlsx">Table S2</a>) that span most of Gm13-LA at intervals of ∼1 Mb. These probes are shown diagrammatically for <em>G. max</em> W82 in <a id="xref-fig-4-12" class="xref-fig" href="#F4">Figure 4, I and K</a> (upper), and by FISH in <a id="xref-fig-4-13" class="xref-fig" href="#F4">Figure 4</a>, K1–K10 (middle). The first three, most centromere-proximal probes hybridized to proximal positions along the longer arm of TL-3 Gm13 (<a id="xref-fig-4-14" class="xref-fig" href="#F4">Figure 4, L</a><a id="xref-fig-1-6" class="xref-fig" href="#F1">1</a>–L<a id="xref-fig-3-14" class="xref-fig" href="#F3">3</a>), as they do in <em>G. max</em> W82 (<a id="xref-fig-4-15" class="xref-fig" href="#F4">Figure 4</a>, K1–K3). This preservation of colinearity of probe targets: (NOR)/(centromere)/(Gm13-BACs1-3) in TL-3 indicates a normal chromosome structure around the centromere of the aberrant Gm13 (<a id="xref-fig-4-16" class="xref-fig" href="#F4">Figure 4O</a>, blue boxed regions). In contrast, each of the remaining Gm13-LA probes hybridized to the opposite, shorter chromosome arm of TL-3 Gm13 (<a id="xref-fig-4-17" class="xref-fig" href="#F4">Figure 4, L</a><a id="xref-fig-4-18" class="xref-fig" href="#F4">4</a>–L10). The fact that Gm13-BAC-4 hybridized closest to the centromere, and Gm13-BAC-11 nearest to the terminus (Figure 4L4) of the shorter arm suggests that an otherwise intact Gm13-LA fragment was broken (between sequences corresponding to Gm13-BAC-3 and -BAC-4) and transposed to the NOR-containing chromosome arm in TL-3 (Figure 4O2). An alternative explanation to account for this novel arrangement would be an inversion (involving two chromosome breaks) of the chromosome region surrounding the Gm13 centromere. The continuity of the NOR signal in the mutant requires that one (the 5′) break of the inversion would have had to occur in the small knob region at the 5′ terminus of Gm13.</p></div><div id="sec-13" class="subsection"><h3>Identification of the translocation partner in TL-3</h3><p id="p-30">Because the identity of the chromosome segment distal to the point at which Gm13-BAC-3 hybridized in TL-3 Gm13 (<a id="xref-fig-4-19" class="xref-fig" href="#F4">Figure 4, L</a><a id="xref-fig-3-15" class="xref-fig" href="#F3">3</a> and O2) was not resolved by the above analysis, we hypothesized that it might represent the missing arm of the aberrantly small chromosome in the mutant karyotype (<a id="xref-fig-4-20" class="xref-fig" href="#F4">Figure 4, B and E</a>). The bright orange centromere labeling of the small chromosome (<a id="xref-fig-4-21" class="xref-fig" href="#F4">Figure 4, A and H</a>) indicated that it could be a form of Gm05, or possibly Gm02 or Gm16, each of which is painted orange by the Cent-Gm probe cocktail in multiple <em>G. max</em> and <em>G. soja</em> accessions (<a id="xref-ref-14-18" class="xref-bibr" href="#ref-14">Findley <em>et al.</em> 2010</a>, and data not shown). BAC probes derived from near the 5′ end (Gm05-BAC-A) and 3′ end (Gm05-BAC-B) of the Gm05, which label opposite ends of W82 Gm05 (Figure 4M4), hybridized to two different chromosomes in TL-3. Gm05-BAC-A hybridized to the aberrantly small chromosome (Figure 4M6); whereas Gm05-BAC-B hybridized to the longer arm of the aberrant Gm13 (Figure 4M5). The fact that the Gm05-BAC-A–hybridizing chromosome arm, corresponding to the 5′ end of the Gm05 pseudomolecule, is still associated with the centromere in the mutant indicates that this chromosome was broken on the opposite arm, but close to the Gm05 centromere. Translocation of this Gm05 fragment to a rearranged Gm13 thus accounts for the longer Gm13 arm in TL-3. The orientation of the fragment in Gm13 is indicated by hybridization of the Gm05-BAC-B probe; this probe hybridizes to the distal end of the 3′ end of a normal Gm05, but hybridized to the distal end of the TL-3 Gm13-LA. This hypothesis was confirmed using Gm13-BAC-11 in combination with Gm05-BAC-B, BACs that represent 3′ positions in their respective chromosomes in <em>G. max</em> W82 (<a id="xref-fig-4-22" class="xref-fig" href="#F4">Figure 4</a>, N3 and N4), but opposite ends of Gm13 in TL-3 (Figure 4N5). Collectively, these data are consistent with a two-step translocation model (<a id="xref-fig-4-23" class="xref-fig" href="#F4">Figure 4</a>, O1–O3). In the first step, a distal fragment of Gm13-LA was transposed to the end of the NOR-containing Gm13-SA. In the second step, most of the 5′ arm of Gm05 was transposed to the remaining Gm13-LA, resulting in the chromosome forms shown in Figure 4O3.</p></div><div id="sec-14" class="subsection"><h3>Analysis of translocation line TL-5</h3><p id="p-31">TL-5 formed the ☉6 conformation and exhibited partial pollen sterility in translocation heterozygotes with TL-2, a translocation that we determined (above) involves Gm01 and Gm08. An aberration in one of these two chromosomes was thus likely in TL-5. TL-5 had two aberrant chromosomes: an aberrantly small pair (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>, arrow; enlarged in <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a>) and an aberrantly large pair (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5A</a>, arrowhead; enlarged in <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5C</a>). FISH mapping with Gm01 terminal BACs indicated Gm01 to be intact in TL-5 (data not shown). In contrast, Gm08 terminal probes, Gm08-BAC-A (Gm08-LA, Gm08 5′ end) and Gm08-BAC-E (Gm08-SA, Gm08 3′ end), did not hybridize to a single TL-5 chromosome (compare <a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5</a>, E5 and E6 with Figure 5E3). In TL-5, Gm08-BAC-A hybridized near the terminus of the long arm of the aberrantly large chromosome; whereas Gm08-BAC-E hybridized to the longer arm of the aberrantly short chromosome. This indicates the involvement of Gm08 (and not Gm01) in the TL-5 translocation and also accounts for the two aberrant chromosome forms.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/2/117/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="FISH-based characterization of translocation lines TL-5 and TL-6. (A) DAPI channel of a TL-5 chromosome spread. (B) Corresponding Cent-Gm&#x2013;based FISH karyotyping of the chromosomes in (A). The two outline-arrows in (A) and (B) indicate the aberrantly small chromosome pair; the two arrowheads indicate the aberrantly large chromosome pair. (C) Enlarged DAPI image containing one each of the aberrant chromosome types in (A). (D) Corresponding FISH image for (C), in which the two Gm08-related BAC probes hybridized to two different chromosomes. The aberrantly large chromosome, with the bright orange centromere, contains hybridization signal (green spots) for Gm08-BAC-A; the aberrantly small chromosome, with the yellowish centromere, contains hybridization signal (red spots) for Gm08-BAC-B. (E&#x2013;G) FISH utilizing pseudomolecule-derived BAC probes to map the TL-5 translocation. The G. max W82 chromosome diagrams in each column 1 panel correspond to the G. max W82 FISH images in columns 3 and 4; the TL-5 chromosome diagrams in each column 2 panel correspond to the TL-5 FISH images in columns 5 and 6. (E4) and (F4) contain no images. In columns 3&#x2013;6, FISH hybridization panels are shown above the corresponding DAPI-stained chromosomes. (Row E) In G. max W82, Gm08-BAC-A (labeled &#x201C;A&#x201D; in column 1) hybridized to Gm08-LA (green hybridization signal in (E3), indicated by the arrowhead); Gm08-BAC-E (labeled &#x201C;E&#x201D; in column 1) hybridized to Gm08-SA (red hybridization signal in (E3), indicated by the arrow). In TL-5, the two Gm08-BAC probes hybridized to two different chromosomes. Gm08-BAC-A (labeled &#x201C;A&#x201D; in column 2) hybridized to the longer arm of the aberrantly large chromosome (green hybridization signal in (E6), indicated by the arrowhead); whereas Gm08-BAC-E (labeled &#x201C;E&#x201D; in column 2) hybridized to the longer arm of the aberrantly short TL-5 chromosome (red hybridization signal in (E5), indicated by the arrow). (Row F) In G. max W82, three Gm08-LA probes (Gm08-BAC-B through -D, labeled &#x201C;B&#x201D; through &#x201C;D&#x201D; in column 1) hybridized to the Gm08-LA (red hybridization signals in (F3), indicated by the arrow), and the Gm08-SA probe (Gm08-BAC-E, labeled &#x201C;E&#x201D; in column 1) hybridized to the Gm08-SA (green hybridization signals in (F3), indicated by the arrowhead). (F6) In TL-5, the three Gm08-LA probes hybridized to the long arm of the aberrantly large TL-5 chromosome (red hybridization signals in (F6), indicated by the arrow), and the Gm08-SA probe (Gm08-BAC-E, labeled &#x201C;E&#x201D; in column 2) hybridized to the longer arm of the aberrantly short TL-5 chromosome (green hybridization signals in (F5), indicated by the arrowhead). (Row G) In G. max W82, the Gm08-LA probe (Gm08-BAC-A, labeled &#x201C;A&#x201D; in column 1, left) and Gm08-SA probe (Gm08-BAC-E, labeled &#x201C;E&#x201D; in column 1, left) hybridized to opposite arms of Gm08. Panel (G3) is duplicated from (E3). Similarly, the Gm02-5&#x2032; probe (Gm02-BAC-1, labeled &#x201C;1&#x201D; in column 1, right) and Gm02-3&#x2032; probe (Gm02-BAC-2, labeled &#x201C;2&#x201D; in column 1, left) hybridized to opposite arms of Gm02 (E4). In contrast, in TL-5, the Gm02-3&#x2032; probe (Gm02-BAC-2, labeled &#x201C;2&#x201D; in column 2, left) hybridized (G5) to the chromosome arm opposite to the one with the Gm08-SA probe signal (Gm08-BAC-E, labeled &#x201C;E,&#x201D; in column 2, left). This centromere color of this chromosome indicates that it is a rearranged Gm08. Conversely, the Gm08-LA&#x2032; probe (Gm08-BAC-A, labeled &#x201C;A&#x201D; in column 2, right) hybridized (G6) to the chromosome arm opposite to the one with the Gm02-5&#x2032; probe signal (Gm02-BAC-1, labeled &#x201C;1&#x201D; in column 2, right). This centromere color of this chromosome indicates that it is a rearranged Gm02. (Row H) Translocation model for TL-5. (H1) A segment of Gm08-LA containing at least sequences corresponding to Gm08-BAC-A through Gm08-BAC-D has was reciprocally exchanged (blue boxes and arrow) with a segment of the 3&#x2032; end of Gm02, resulting in the TL-5 chromosome forms shown in (H2). The yellow arrows in the Gm08 diagrams indicates the polarity of the translocated fragment as determined by these experiments; the orientation of the arrows in Gm02 are conjecture. (I) DAPI channel of a TL-6 chromosome spread. (J) Corresponding Cent-Gm-based FISH karyotyping of the chromosomes in (I). The two outline-arrows in (I) and (J) indicate the mutant Gm02 pair; the two arrowheads indicate the mutant Gm11 pair; see (K5) and (K6) for enlargements. (K&#x2013;M) FISH utilizing pseudomolecule-derived BAC probes to map the TL-6 translocation. The G. max W82 chromosome diagrams in each column 1 panel correspond to the G. max W82 FISH images in columns 3 and 4; the TL-6 chromosome diagrams in each column 2 panel correspond to the TL-6 FISH images in columns 5 and 6. Panels (K4), (L3), (M3), and (M4) contain no images. In columns 3&#x2013;6, FISH hybridization panels are shown above the corresponding DAPI-stained chromosomes. (Row K) In G. max W82, Gm02-5&#x2032; BAC probe (Gm02-BAC-A, labeled &#x201C;A&#x201D; in column 1; red signal, indicated by arrow in K3) and Gm02-3&#x2032; BAC probe (Gm02-BAC-B, labeled &#x201C;B&#x201D; in column 1; green signal, indicated by arrowhead in K3) hybridized to opposite ends of Gm02. In TL-6, the two Gm02-BAC probes hybridized to two different chromosomes. Gm02-BAC-A (labeled &#x201C;A&#x201D; in column 2) hybridized to a chromosome with an orange centromere (red hybridization signal in (K5), indicated by the arrowhead); Gm02-BAC-B (labeled &#x201C;B&#x201D; in column 2) hybridized to a chromosome with a dark green centromere (green hybridization signal in (K6), indicated by the arrow). (Row L) In G. max W82 (L4), Gm11-LA BAC probe (Gm11-BAC-1, labeled &#x201C;1&#x201D; in column 1; green signal, indicated by arrowhead in L4) and Gm11-SA BAC probe (Gm11-BAC-2, labeled &#x201C;2&#x201D; in column 1; red signal, indicated by arrow in L4) hybridized to opposite ends of Gm11. In TL-6, the two Gm11-BAC probes hybridized to two different chromosomes. Gm11-BAC-1 (labeled &#x201C;1&#x201D; in column 2) hybridized to a chromosome with an orange centromere (green hybridization signal in (L5), indicated by the arrowhead); Gm11-BAC-2 (labeled &#x201C;2&#x201D; in column 2) hybridized to a chromosome with a dark green centromere (red hybridization signal in (L6), indicated by the arrowhead). (Row M) In TL-6, the Gm02-5&#x2032; probe (Gm02-BAC-A, labeled &#x201C;A&#x201D; in column 2, left) hybridized (M5) to the chromosome arm opposite to the one with the Gm11-LA probe signal (Gm11-BAC-1, labeled &#x201C;1&#x201D; in column 2, left). This centromere color of this chromosome indicates that it is a rearranged Gm02. Conversely, the Gm11-SA&#x2032; probe (Gm11-BAC-2, labeled &#x201C;2&#x201D; in column 2, right) hybridized (M6) to the chromosome arm opposite to the one with the Gm02-3&#x2032; probe signal (Gm02-BAC-B, labeled &#x201C;B&#x201D; in column 2, right). This centromere color of this chromosome indicates that it is a rearranged Gm11. (Row N) Translocation model for TL-6. (N1) A segment of 3&#x2032; Gm02 containing sequences corresponding to Gm02-BAC-B was reciprocally exchanged with a segment of Gm11-LA, resulting in the TL-6 chromosome forms shown in (N2). The yellow and gray arrows in all diagrams indicate the presumed, but unverified polarity. In addition, the gray hatching in all TL-6 chromosome diagrams indicates the uncertainty regarding chromosome break positions. The 2-&#xB5;m scale bar in (A) is also valid for (B). The 2-&#xB5;m scale bar in (C) is also valid for (D). The 2-&#xB5;m scale bar in (I) is valid for (J). The 2-&#xB5;m scale bar in (E6) is valid for (E&#x2013;G). The 2-&#xB5;m scale bar in (M6) is valid for (K&#x2013;M)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-498101292" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;FISH-based characterization of translocation lines TL-5 and TL-6. (A) DAPI channel of a TL-5 chromosome spread. (B) Corresponding Cent-Gm&#x2013;based FISH karyotyping of the chromosomes in (A). The two outline-arrows in (A) and (B) indicate the aberrantly small chromosome pair; the two arrowheads indicate the aberrantly large chromosome pair. (C) Enlarged DAPI image containing one each of the aberrant chromosome types in (A). (D) Corresponding FISH image for (C), in which the two Gm08-related BAC probes hybridized to two different chromosomes. The aberrantly large chromosome, with the bright orange centromere, contains hybridization signal (green spots) for Gm08-BAC-A; the aberrantly small chromosome, with the yellowish centromere, contains hybridization signal (red spots) for Gm08-BAC-B. (E&#x2013;G) FISH utilizing pseudomolecule-derived BAC probes to map the TL-5 translocation. The &lt;em&gt;G. max&lt;/em&gt; W82 chromosome diagrams in each column 1 panel correspond to the &lt;em&gt;G. max&lt;/em&gt; W82 FISH images in columns 3 and 4; the TL-5 chromosome diagrams in each column 2 panel correspond to the TL-5 FISH images in columns 5 and 6. (E4) and (F4) contain no images. In columns 3&#x2013;6, FISH hybridization panels are shown above the corresponding DAPI-stained chromosomes. (Row E) In &lt;em&gt;G. max&lt;/em&gt; W82, Gm08-BAC-A (labeled &#x201C;A&#x201D; in column 1) hybridized to Gm08-LA (green hybridization signal in (E3), indicated by the arrowhead); Gm08-BAC-E (labeled &#x201C;E&#x201D; in column 1) hybridized to Gm08-SA (red hybridization signal in (E3), indicated by the arrow). In TL-5, the two Gm08-BAC probes hybridized to two different chromosomes. Gm08-BAC-A (labeled &#x201C;A&#x201D; in column 2) hybridized to the longer arm of the aberrantly large chromosome (green hybridization signal in (E6), indicated by the arrowhead); whereas Gm08-BAC-E (labeled &#x201C;E&#x201D; in column 2) hybridized to the longer arm of the aberrantly short TL-5 chromosome (red hybridization signal in (E5), indicated by the arrow). (Row F) In &lt;em&gt;G. max&lt;/em&gt; W82, three Gm08-LA probes (Gm08-BAC-B through -D, labeled &#x201C;B&#x201D; through &#x201C;D&#x201D; in column 1) hybridized to the Gm08-LA (red hybridization signals in (F3), indicated by the arrow), and the Gm08-SA probe (Gm08-BAC-E, labeled &#x201C;E&#x201D; in column 1) hybridized to the Gm08-SA (green hybridization signals in (F3), indicated by the arrowhead). (F6) In TL-5, the three Gm08-LA probes hybridized to the long arm of the aberrantly large TL-5 chromosome (red hybridization signals in (F6), indicated by the arrow), and the Gm08-SA probe (Gm08-BAC-E, labeled &#x201C;E&#x201D; in column 2) hybridized to the longer arm of the aberrantly short TL-5 chromosome (green hybridization signals in (F5), indicated by the arrowhead). (Row G) In &lt;em&gt;G. max&lt;/em&gt; W82, the Gm08-LA probe (Gm08-BAC-A, labeled &#x201C;A&#x201D; in column 1, left) and Gm08-SA probe (Gm08-BAC-E, labeled &#x201C;E&#x201D; in column 1, left) hybridized to opposite arms of Gm08. Panel (G3) is duplicated from (E3). Similarly, the Gm02-5&#x2032; probe (Gm02-BAC-1, labeled &#x201C;1&#x201D; in column 1, right) and Gm02-3&#x2032; probe (Gm02-BAC-2, labeled &#x201C;2&#x201D; in column 1, left) hybridized to opposite arms of Gm02 (E4). In contrast, in TL-5, the Gm02-3&#x2032; probe (Gm02-BAC-2, labeled &#x201C;2&#x201D; in column 2, left) hybridized (G5) to the chromosome arm opposite to the one with the Gm08-SA probe signal (Gm08-BAC-E, labeled &#x201C;E,&#x201D; in column 2, left). This centromere color of this chromosome indicates that it is a rearranged Gm08. Conversely, the Gm08-LA&#x2032; probe (Gm08-BAC-A, labeled &#x201C;A&#x201D; in column 2, right) hybridized (G6) to the chromosome arm opposite to the one with the Gm02-5&#x2032; probe signal (Gm02-BAC-1, labeled &#x201C;1&#x201D; in column 2, right). This centromere color of this chromosome indicates that it is a rearranged Gm02. (Row H) Translocation model for TL-5. (H1) A segment of Gm08-LA containing at least sequences corresponding to Gm08-BAC-A through Gm08-BAC-D has was reciprocally exchanged (blue boxes and arrow) with a segment of the 3&#x2032; end of Gm02, resulting in the TL-5 chromosome forms shown in (H2). The yellow arrows in the Gm08 diagrams indicates the polarity of the translocated fragment as determined by these experiments; the orientation of the arrows in Gm02 are conjecture. (I) DAPI channel of a TL-6 chromosome spread. (J) Corresponding Cent-Gm-based FISH karyotyping of the chromosomes in (I). The two outline-arrows in (I) and (J) indicate the mutant Gm02 pair; the two arrowheads indicate the mutant Gm11 pair; see (K5) and (K6) for enlargements. (K&#x2013;M) FISH utilizing pseudomolecule-derived BAC probes to map the TL-6 translocation. The &lt;em&gt;G. max&lt;/em&gt; W82 chromosome diagrams in each column 1 panel correspond to the &lt;em&gt;G. max&lt;/em&gt; W82 FISH images in columns 3 and 4; the TL-6 chromosome diagrams in each column 2 panel correspond to the TL-6 FISH images in columns 5 and 6. Panels (K4), (L3), (M3), and (M4) contain no images. In columns 3&#x2013;6, FISH hybridization panels are shown above the corresponding DAPI-stained chromosomes. (Row K) In &lt;em&gt;G. max&lt;/em&gt; W82, Gm02-5&#x2032; BAC probe (Gm02-BAC-A, labeled &#x201C;A&#x201D; in column 1; red signal, indicated by arrow in K3) and Gm02-3&#x2032; BAC probe (Gm02-BAC-B, labeled &#x201C;B&#x201D; in column 1; green signal, indicated by arrowhead in K3) hybridized to opposite ends of Gm02. In TL-6, the two Gm02-BAC probes hybridized to two different chromosomes. Gm02-BAC-A (labeled &#x201C;A&#x201D; in column 2) hybridized to a chromosome with an orange centromere (red hybridization signal in (K5), indicated by the arrowhead); Gm02-BAC-B (labeled &#x201C;B&#x201D; in column 2) hybridized to a chromosome with a dark green centromere (green hybridization signal in (K6), indicated by the arrow). (Row L) In &lt;em&gt;G. max&lt;/em&gt; W82 (L4), Gm11-LA BAC probe (Gm11-BAC-1, labeled &#x201C;1&#x201D; in column 1; green signal, indicated by arrowhead in L4) and Gm11-SA BAC probe (Gm11-BAC-2, labeled &#x201C;2&#x201D; in column 1; red signal, indicated by arrow in L4) hybridized to opposite ends of Gm11. In TL-6, the two Gm11-BAC probes hybridized to two different chromosomes. Gm11-BAC-1 (labeled &#x201C;1&#x201D; in column 2) hybridized to a chromosome with an orange centromere (green hybridization signal in (L5), indicated by the arrowhead); Gm11-BAC-2 (labeled &#x201C;2&#x201D; in column 2) hybridized to a chromosome with a dark green centromere (red hybridization signal in (L6), indicated by the arrowhead). (Row M) In TL-6, the Gm02-5&#x2032; probe (Gm02-BAC-A, labeled &#x201C;A&#x201D; in column 2, left) hybridized (M5) to the chromosome arm opposite to the one with the Gm11-LA probe signal (Gm11-BAC-1, labeled &#x201C;1&#x201D; in column 2, left). This centromere color of this chromosome indicates that it is a rearranged Gm02. Conversely, the Gm11-SA&#x2032; probe (Gm11-BAC-2, labeled &#x201C;2&#x201D; in column 2, right) hybridized (M6) to the chromosome arm opposite to the one with the Gm02-3&#x2032; probe signal (Gm02-BAC-B, labeled &#x201C;B&#x201D; in column 2, right). This centromere color of this chromosome indicates that it is a rearranged Gm11. (Row N) Translocation model for TL-6. (N1) A segment of 3&#x2032; Gm02 containing sequences corresponding to Gm02-BAC-B was reciprocally exchanged with a segment of Gm11-LA, resulting in the TL-6 chromosome forms shown in (N2). The yellow and gray arrows in all diagrams indicate the presumed, but unverified polarity. In addition, the gray hatching in all TL-6 chromosome diagrams indicates the uncertainty regarding chromosome break positions. The 2-&#xB5;m scale bar in (A) is also valid for (B). The 2-&#xB5;m scale bar in (C) is also valid for (D). The 2-&#xB5;m scale bar in (I) is valid for (J). The 2-&#xB5;m scale bar in (E6) is valid for (E&#x2013;G). The 2-&#xB5;m scale bar in (M6) is valid for (K&#x2013;M).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/2/117/F5.medium.gif" width="440" height="306"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/1/2/117/F5.medium.gif" width="440" height="306"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/2/117/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/2/117/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460689" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-32" class="first-child">FISH-based characterization of translocation lines TL-5 and TL-6. (A) DAPI channel of a TL-5 chromosome spread. (B) Corresponding Cent-Gm–based FISH karyotyping of the chromosomes in (A). The two outline-arrows in (A) and (B) indicate the aberrantly small chromosome pair; the two arrowheads indicate the aberrantly large chromosome pair. (C) Enlarged DAPI image containing one each of the aberrant chromosome types in (A). (D) Corresponding FISH image for (C), in which the two Gm08-related BAC probes hybridized to two different chromosomes. The aberrantly large chromosome, with the bright orange centromere, contains hybridization signal (green spots) for Gm08-BAC-A; the aberrantly small chromosome, with the yellowish centromere, contains hybridization signal (red spots) for Gm08-BAC-B. (E–G) FISH utilizing pseudomolecule-derived BAC probes to map the TL-5 translocation. The <em>G. max</em> W82 chromosome diagrams in each column 1 panel correspond to the <em>G. max</em> W82 FISH images in columns 3 and 4; the TL-5 chromosome diagrams in each column 2 panel correspond to the TL-5 FISH images in columns 5 and 6. (E4) and (F4) contain no images. In columns 3–6, FISH hybridization panels are shown above the corresponding DAPI-stained chromosomes. (Row E) In <em>G. max</em> W82, Gm08-BAC-A (labeled “A” in column 1) hybridized to Gm08-LA (green hybridization signal in (E3), indicated by the arrowhead); Gm08-BAC-E (labeled “E” in column 1) hybridized to Gm08-SA (red hybridization signal in (E3), indicated by the arrow). In TL-5, the two Gm08-BAC probes hybridized to two different chromosomes. Gm08-BAC-A (labeled “A” in column 2) hybridized to the longer arm of the aberrantly large chromosome (green hybridization signal in (E6), indicated by the arrowhead); whereas Gm08-BAC-E (labeled “E” in column 2) hybridized to the longer arm of the aberrantly short TL-5 chromosome (red hybridization signal in (E5), indicated by the arrow). (Row F) In <em>G. max</em> W82, three Gm08-LA probes (Gm08-BAC-B through -D, labeled “B” through “D” in column 1) hybridized to the Gm08-LA (red hybridization signals in (F3), indicated by the arrow), and the Gm08-SA probe (Gm08-BAC-E, labeled “E” in column 1) hybridized to the Gm08-SA (green hybridization signals in (F3), indicated by the arrowhead). (F6) In TL-5, the three Gm08-LA probes hybridized to the long arm of the aberrantly large TL-5 chromosome (red hybridization signals in (F6), indicated by the arrow), and the Gm08-SA probe (Gm08-BAC-E, labeled “E” in column 2) hybridized to the longer arm of the aberrantly short TL-5 chromosome (green hybridization signals in (F5), indicated by the arrowhead). (Row G) In <em>G. max</em> W82, the Gm08-LA probe (Gm08-BAC-A, labeled “A” in column 1, left) and Gm08-SA probe (Gm08-BAC-E, labeled “E” in column 1, left) hybridized to opposite arms of Gm08. Panel (G3) is duplicated from (E3). Similarly, the Gm02-5′ probe (Gm02-BAC-1, labeled “1” in column 1, right) and Gm02-3′ probe (Gm02-BAC-2, labeled “2” in column 1, left) hybridized to opposite arms of Gm02 (E4). In contrast, in TL-5, the Gm02-3′ probe (Gm02-BAC-2, labeled “2” in column 2, left) hybridized (G5) to the chromosome arm opposite to the one with the Gm08-SA probe signal (Gm08-BAC-E, labeled “E,” in column 2, left). This centromere color of this chromosome indicates that it is a rearranged Gm08. Conversely, the Gm08-LA′ probe (Gm08-BAC-A, labeled “A” in column 2, right) hybridized (G6) to the chromosome arm opposite to the one with the Gm02-5′ probe signal (Gm02-BAC-1, labeled “1” in column 2, right). This centromere color of this chromosome indicates that it is a rearranged Gm02. (Row H) Translocation model for TL-5. (H1) A segment of Gm08-LA containing at least sequences corresponding to Gm08-BAC-A through Gm08-BAC-D has was reciprocally exchanged (blue boxes and arrow) with a segment of the 3′ end of Gm02, resulting in the TL-5 chromosome forms shown in (H2). The yellow arrows in the Gm08 diagrams indicates the polarity of the translocated fragment as determined by these experiments; the orientation of the arrows in Gm02 are conjecture. (I) DAPI channel of a TL-6 chromosome spread. (J) Corresponding Cent-Gm-based FISH karyotyping of the chromosomes in (I). The two outline-arrows in (I) and (J) indicate the mutant Gm02 pair; the two arrowheads indicate the mutant Gm11 pair; see (K5) and (K6) for enlargements. (K–M) FISH utilizing pseudomolecule-derived BAC probes to map the TL-6 translocation. The <em>G. max</em> W82 chromosome diagrams in each column 1 panel correspond to the <em>G. max</em> W82 FISH images in columns 3 and 4; the TL-6 chromosome diagrams in each column 2 panel correspond to the TL-6 FISH images in columns 5 and 6. Panels (K4), (L3), (M3), and (M4) contain no images. In columns 3–6, FISH hybridization panels are shown above the corresponding DAPI-stained chromosomes. (Row K) In <em>G. max</em> W82, Gm02-5′ BAC probe (Gm02-BAC-A, labeled “A” in column 1; red signal, indicated by arrow in K3) and Gm02-3′ BAC probe (Gm02-BAC-B, labeled “B” in column 1; green signal, indicated by arrowhead in K3) hybridized to opposite ends of Gm02. In TL-6, the two Gm02-BAC probes hybridized to two different chromosomes. Gm02-BAC-A (labeled “A” in column 2) hybridized to a chromosome with an orange centromere (red hybridization signal in (K5), indicated by the arrowhead); Gm02-BAC-B (labeled “B” in column 2) hybridized to a chromosome with a dark green centromere (green hybridization signal in (K6), indicated by the arrow). (Row L) In <em>G. max</em> W82 (L4), Gm11-LA BAC probe (Gm11-BAC-1, labeled “1” in column 1; green signal, indicated by arrowhead in L4) and Gm11-SA BAC probe (Gm11-BAC-2, labeled “2” in column 1; red signal, indicated by arrow in L4) hybridized to opposite ends of Gm11. In TL-6, the two Gm11-BAC probes hybridized to two different chromosomes. Gm11-BAC-1 (labeled “1” in column 2) hybridized to a chromosome with an orange centromere (green hybridization signal in (L5), indicated by the arrowhead); Gm11-BAC-2 (labeled “2” in column 2) hybridized to a chromosome with a dark green centromere (red hybridization signal in (L6), indicated by the arrowhead). (Row M) In TL-6, the Gm02-5′ probe (Gm02-BAC-A, labeled “A” in column 2, left) hybridized (M5) to the chromosome arm opposite to the one with the Gm11-LA probe signal (Gm11-BAC-1, labeled “1” in column 2, left). This centromere color of this chromosome indicates that it is a rearranged Gm02. Conversely, the Gm11-SA′ probe (Gm11-BAC-2, labeled “2” in column 2, right) hybridized (M6) to the chromosome arm opposite to the one with the Gm02-3′ probe signal (Gm02-BAC-B, labeled “B” in column 2, right). This centromere color of this chromosome indicates that it is a rearranged Gm11. (Row N) Translocation model for TL-6. (N1) A segment of 3′ Gm02 containing sequences corresponding to Gm02-BAC-B was reciprocally exchanged with a segment of Gm11-LA, resulting in the TL-6 chromosome forms shown in (N2). The yellow and gray arrows in all diagrams indicate the presumed, but unverified polarity. In addition, the gray hatching in all TL-6 chromosome diagrams indicates the uncertainty regarding chromosome break positions. The 2-µm scale bar in (A) is also valid for (B). The 2-µm scale bar in (C) is also valid for (D). The 2-µm scale bar in (I) is valid for (J). The 2-µm scale bar in (E6) is valid for (E–G). The 2-µm scale bar in (M6) is valid for (K–M).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Analysis of Gm08 probes in TL-5</h3><p id="p-33">To characterize Gm08 in TL-5 and to identify its translocation partner, we examined the mutant karyotype using the two Gm08 BACs in combination with the Cent-Gm cocktail (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5B</a>). The Gm08-BAC-E probe hybridized to a chromosome with a yellow-green labeled centromere (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5D</a>), a color typical of Gm08 in other <em>G. max</em> cultivars (<a id="xref-ref-14-19" class="xref-bibr" href="#ref-14">Findley <em>et al.</em> 2010</a>). This result indicated that the smaller chromosome in the TL-5 karyotype is Gm08, because the Gm08 centromere is still associated with the 5′ end of Gm08 in the mutant, and also that the breakpoint was on the opposite arm. We then determined how much of the Gm08-LA was translocated to the aberrantly large chromosome using BACs derived from Gm08-LA positions 11.0, 14.9, and 23.3 Mb. When tested individually in <em>G. max</em> W82, each of these probes hybridized only to the Gm08-LA (data not shown); when used in combination, 5-6 distinct hybridization signals are detectable on Gm08-LA (Figure 5F3). FISH in TL-5 also produced 5-6 spots, which were restricted to the long arm of the aberrantly large chromosome (Figure 5F6). Hybridization of Gm08-BAC-A near the terminus of the long arm of the large chromosome (<a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5D</a>) indicates that the fragment containing the 5′ end of Gm08-LA represents the chromosome terminus.</p></div><div id="sec-16" class="subsection"><h3>Identification of the Gm08 translocation partner in TL-5</h3><p id="p-34">Because the TL-5 Gm08-SA is intact (due to contiguity between the Gm08-SA probe hybridization and the Gm08 centromere), and also that a large proportion of Gm08-LA is translocated in the mutant, we next determined the identity of the Gm08 translocation partner. The Cent-Gm cocktail painting of the aberrantly large, Gm08-LA-containing chromosome was bright orange (<a id="xref-fig-5-10" class="xref-fig" href="#F5">Figure 5D</a>), a color typical for Gm02, or possibly Gm05 or Gm16. We first assessed the structure of TL-5 Gm02 using BAC probes Gm02-BAC-1 and Gm02-BAC-2, which are derived from 5′ and 3′ Gm02 positions, respectively (<a id="xref-ref-14-20" class="xref-bibr" href="#ref-14">Findley <em>et al.</em> 2010</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000034/-/DC1/TableS2.xlsx">Table S2</a> and data not shown). In <em>G. max</em> W82, these probes hybridized to opposite ends of Gm02 (Figure 5G4). In TL-5, the signals for these BACs hybridized to two different chromosomes: the aberrantly large chromosome and the aberrantly small Gm08. Specifically, the Gm02-BAC-1 signal hybridized to the short arm of the aberrantly large chromosome (Figure 5G6); whereas Gm02-BAC-2 hybridized to the shorter arm of the mutant Gm08 (Figure 5G5). Thus, the translocation partner for Gm08 in TL-5 is Gm02, and furthermore, the translocation is reciprocal. Despite the short length of the truncated Gm08-LA in TL-5, Gm02-BAC-1, which is derived from the 3′ end of Gm02, still hybridized to the truncated Gm08 (Figure 5G5). Due to its small size, however, we did not assess the polarity of the translocated Gm02 fragment. Overall, our analysis is consistent with the translocation model shown in <a id="xref-fig-5-11" class="xref-fig" href="#F5">Figure 5</a>, H1 and H2, in which a large fragment of Gm08-LA is reciprocally exchanged with a fragment of the 3′ end of Gm02.</p></div><div id="sec-17" class="subsection"><h3>Analysis of translocation line TL-6</h3><p id="p-35">Because TL-6 formed the ☉6 conformation with TL-5, candidate TL-6 translocation chromosomes included Gm02 or Gm08. However, the overall karyotype of TL-6 (<a id="xref-fig-5-12" class="xref-fig" href="#F5">Figure 5I</a>) revealed no obvious aberrations, so we relied on FISH with the Cent-Gm cocktail and BAC probes to characterize the mutant. We first tested terminal BAC clones: Gm02-BAC-A and Gm02-BAC-B, which are derived from 5′ and 3′ Gm02 positions, respectively (Figure 5K3 and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000034/-/DC1/TableS2.xlsx">Table S2</a>). In TL-6, Gm02-BAC-A hybridized to a chromosome with a bright orange-labeled centromere (<a id="xref-fig-5-13" class="xref-fig" href="#F5">Figure 5, J</a> and K5), which is typical for <em>G. max</em> Gm02; this also suggested that the 5′ end of Gm02 was still associated the Gm02 centromere in the mutant. In contrast, Gm02-BAC-B hybridized to a dark green centromere-containing chromosome (Figure 5K6), which might be Gm11. We therefore assessed the structure of Gm11 in TL-6 using 5′ pseudomolecule-positioned Gm11-BAC-1 in combination with 3′ pseudomolecule-positioned Gm11-BAC-2. In <em>G. max</em> W82, these two probes label opposite ends of Gm11 (Figure 5L4). In contrast, in TL-6, the two BACs hybridized to two different chromosomes: Gm11-BAC-2 was associated with the dark green centromere (Figure 5L6), but the Gm11-BAC-2 signal was associated with the aberrant Gm02 (Figure 5L5). Thus, a fragment of Gm11-LA was translocated to the 5-end of Gm02, but the Gm11-SA was still associated with the Gm11 centromere. To verify this, we simultaneously probed the mutant with Gm02- and Gm08-derived BACs (<a id="xref-fig-5-14" class="xref-fig" href="#F5">Figure 5</a>, M5 and M6). Indeed, the 5′ Gm11-derived signal (Gm11-BAC-1) was associated with the 5′ Gm02-derived signal (Gm02-BAC-A) on a chromosome with an orange centromere (Figure 5M5). Conversely, the 3′ Gm11-derived signal (Gm11-BAC-2) was associated with 3′ Gm02-derived signal (Gm02-BAC-B) on a chromosome with dark green centromere (Figure 5M6). These data suggest a model (<a id="xref-fig-5-15" class="xref-fig" href="#F5">Figure 5</a>, N1 and N2) for a reciprocal translocation in TL-6 between Gm02 and Gm11, in which a section of the 5′ end-associated Gm02 arm has exchanged with a section of 5′ end-associated Gm11-LA.</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-36">A simple but effective FISH-based karyotyping cocktail in combination with genetically anchored, pseudomolecule-derived BACs has enabled us to rapidly characterize five soybean translocations. In the process, we validated all previously detected genetic interactions between elements of a translocation interaction network (<a id="xref-ref-25-9" class="xref-bibr" href="#ref-25">Mahama <em>et al.</em>, 1999</a>). Critical to this effort was the prior characterization of two elements of the network, TL-1 and TL-2. At the start of our study, we knew that TL-1 involved an exchange between chromosomes 11 and 13 (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Palmer and Heer 1984</a>; Mahama <em>et al.</em> 2003; <a id="xref-ref-14-21" class="xref-bibr" href="#ref-14">Findley <em>et al.</em> 2010</a>); we also knew that TL-2 involved a rearrangement in chromosome 1 (Gg01; Pappas and Palmer, unpublished results). The sequential identification of individual translocation chromosomes in these and subsequent examined lines permitted us to characterize all translocation chromosomes in the network.</p><p id="p-37">For TL-2, FISH-based karyotyping verified the involvement of Gg01 and identified its translocation partner, Gg08. We also determined the involvement of both chromosomes using molecular mapping of SSRs; however, identifying the chromosomal breakpoints using this method was not possible. Pseudo-linkage groups identified by MapMaker included the linkage of the short arm SSRs of both Gg01 and Gg08 (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>), in support of the FISH data. While QuadMap uses an algorithm for assessing linkage groups in lines containing heterozygous translocations (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Livingstone <em>et al.</em> 2000</a>; <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Durrant <em>et al.</em> 2006</a>), it was unable to identify the translocated segments in TL-2. To use molecular mapping and QuadMap to locate the translocation breakpoints, it may be necessary to use a recombinant inbred line of TL-2. Although SSR analysis has long been an important tool for constructing genetic linkage maps (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akkaya <em>et al.</em> 1992</a>; <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Cregan <em>et al.</em> 1999</a>) and assessing soybean cultivars (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Diwan and Cregan 1997</a>), its usefulness in identifying the translocation breakpoints in the TL-2 line is limited. Considering the time and money required to produce a population for mapping, screening SSR loci for polymorphisms, and assessing SSR genotypes, obtaining a plethora of alternative linkage maps identifying several different possible translocation breakpoints can be frustrating. Our FISH analysis of TL-2 circumvented the need for growing a mapping population and facilitated the unambiguous identification of chromosomal segments involved in the TL-2 translocation. The involvement of chromosomes homologous to Gm01 and Gm08 validated the observed lack of genetic interaction of TL-2 with TL-1, TL-3, and TL-4 (<a id="xref-ref-25-10" class="xref-bibr" href="#ref-25">Mahama <em>et al.</em>, 1999</a>), none of which involve Gm01 or Gm08, and enabled our subsequent identification of the translocation chromosomes in TL-2 interacting lines TL-5 and TL-6.</p><p id="p-38">Genetic interaction of TL-5 with TL-2 (Gg01 and Gg08) permitted identification of the first TL-5 translocation chromosome (Gm08, not Gm01); the second (Gm02) was identified through Cent-Gm–based FISH karyotyping. Of the fast-neutron mutants studied, TL-5 appears to be the only one involving a simple reciprocal translocation. The chromosomes identified (Gm02 and Gm08) also validated genetic and cytological data (<a id="xref-ref-25-11" class="xref-bibr" href="#ref-25">Mahama <em>et al.</em> 1999</a>), which suggested that TL-5 shared a translocation chromosome (Gm08) with TL-2 (Gm08 and Gm01), but not with TL-1 (Gm13 and Gm11), TL-3 (Gm13 and Gm05) or TL-4 (Gm13 and Gm04).</p><p id="p-39">For TL-4, genetic interaction with TL-1 (involving Gm11 and Gm13) permitted identification of the first translocation chromosome (Gm13, and not Gm11); the second (Gm04) was identified through Cent-Gm–based FISH karyotyping. The involvement of Gm13 in the translocation also validated the genetic interaction of TL-4 with TL-3 (below), which also involves Gm13, as well as the lack of interaction (<a id="xref-ref-25-12" class="xref-bibr" href="#ref-25">Mahama <em>et al.</em>, 1999</a>) with any of the other translocation lines studied (none of which involve Gm04).</p><p id="p-40">The genetic interaction (<a id="xref-ref-25-13" class="xref-bibr" href="#ref-25">Mahama <em>et al.</em>, 1999</a>) of TL-3 with TL-1 (Gm11 and Gm13) also permitted identification of the first translocation chromosome (Gm13, not Gm11); the second chromosome (Gm05) was identified through Cent-Gm–based FISH karyotyping. This analysis validated the lack of genetic interaction of TL-3 with TL-2, TL-5, and TL-6 (<a id="xref-ref-25-14" class="xref-bibr" href="#ref-25">Mahama <em>et al.</em>, 1999</a>), none of which involve either Gm05 or Gm13. Because TL-3 chromosomes were derived from fast-neutron–bombarded plants, the (minimally) two chromosome breaks (one in Gm13 and one in Gm05) required for this translocation are likely to have occurred simultaneously in the same cell, followed by the action of nonhomologous end-joining repair mechanisms that generated the chromosome forms in TL-3. Of the translocations studied by (<a id="xref-ref-25-15" class="xref-bibr" href="#ref-25">Mahama <em>et al.</em> 1999</a>), TL-6, genetically interacted with only one other line, TL-5. Thus, characterization of TL-6 required the prior, sequential characterization of the mutants (translocation chromosomes): TL-5 (Gm08 and Gm02) and TL-2 (Gm08 and Gm01).</p><p id="p-41">As demonstrated by these experiments, molecular cytogenetics represents a powerful complement to both classical genetics and genomics. Mitotic and meiotic chromosome FISH continues to be invaluable in genome sequencing efforts. FISH played important roles, for example, in the Solanaceae Genome Project, both in the direction of sequencing efforts, through BAC-based anchoring of contigs (<em>e.g.</em>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Stack <em>et al.</em> 2009</a>), a critical step development of minimal tiling path of BACs in development of chromosome pseudomolecules, and in the detection and closure of numerous and significant gaps, in even euchromatic regions of large genomes (<em>e.g.</em>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Szinay <em>et al.</em> 2008</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Peters <em>et al.</em> 2009</a>). BAC-based FISH can also facilitate integration of molecular marker-based genetic maps with physical maps, because the technique can readily validate (or dispute) chromosome models (<em>e.g.</em>, De Lorenzi <em>et al.</em> 2010), which can be weak in domains high in genomic repeat, or in regions with a low density of molecular markers (reviewed in <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Szinay <em>et al.</em> 2008</a>). The availability of soybean-based karyotyping resources, including anchored BACs (<a id="xref-ref-35-8" class="xref-bibr" href="#ref-35">Schmutz <em>et al.</em> 2010</a>) and a repeat-based karyotyping cocktail (<a id="xref-ref-14-22" class="xref-bibr" href="#ref-14">Findley <em>et al.</em> 2010</a>) should facilitate the numerous array-based comparative genomic hybridization studies currently underway in soybean. For example, in the analysis of fast-neutron mutagenesis and TILLING (targeted induced local lesions in genomes) populations, FISH can validate deletions and trace the fate (integrity and locations) of duplicated sequences. Soybean-based FISH tools, particularly BACs, due to their potential for significant cross-species hybridization, may also be useful in comparative studies (<em>e.g.</em>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Mcclean <em>et al.</em> 2010</a>) of related <em>Glycine</em> species (<em>e.g.</em>, wild perennial Glycine species), which may not be sequenced in the near future. Numerous soybean genome resequencing efforts are underway, including those targeting the diversity (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Nichols <em>et al.</em> 2007</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lee <em>et al.</em> 2008</a>) in soybean’s wild progenitor, <em>Glycine soja</em>. However, because large-scale chromosome rearrangements may not be easily detectable by genome resequencing (<em>e.g.</em>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kim <em>et al.</em> 2010</a>), FISH-based cytogenetics will undoubtedly maintain its importance by its ability to rapidly assess chromosome structure in the largely uncharacterized wild soybean germplasm collections.</p></div><div class="section" id="sec-19"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-42">We thank Jesse Riordan (USDA ARS) and Scott Kalberer (USDA ARS) for help with the chromosome mapping. We thank Scott Jackson, the Arizona Genomics Institute and the Clemson University Genomics Institute for BAC clones. We thank Jeongmin Choi, Patrice Albert, and members of the Birchler laboratory for helpful discussions. Finally, we are grateful to Jason C. Anderson, Steven B. Cannon, and anonymous reviewers for their helpful critiques. S.D.F. and G.S. received funding from the US Department of Agriculture (grant 2008-34555-19305) and the National Center for Soybean Biotechnology. J.A.B received funding from the National Science Foundation (grant DBI0423898). R.G.P. and A.L.P. gratefully acknowledge funding from BRDC.</p><p id="p-43">This is a joint contribution from the Iowa Agriculture and Home Economics Experiment Station, Ames, Iowa, Project No. 4403; and from the United States Department of Agriculture, Agricultural Research Service, Corn Insects and Crop Genetics Research Unit. The mention of a trademark or proprietary product does not constitute a guarantee or warranty of the product by Iowa State University or the United States Department of Agriculture, and the use of the name by Iowa State University or the United States Department of Agriculture implies no approval of the product to the exclusion of others that may also be suitable.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-44"><em>Communicating editor: Scott A. Jackson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-03-04"><span class="received-label">Received </span>March 4, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-05-07"><span class="accepted-label">Accepted </span>May 7, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2011 Findley <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-1">This is an open access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ahmad</span>  <span class="cit-name-given-names">Q. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Britten</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byth</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">The karyotype of <em>Glycine soja</em> and its relationship to that of the soybean, <em>Glycine max</em>.</span> <abbr class="cit-jnl-abbrev">Cytologia (Tokyo)</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">645</span>–<span class="cit-lpage">658</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytologia%2B%2528Tokyo%2529%26rft.volume%253D49%26rft.spage%253D645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akkaya</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arvind</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cregan</span>  <span class="cit-name-given-names">P. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Length polymorphisms of simple sequence repeat DNA in soybean.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">1131</span>–<span class="cit-lpage">1139</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D132%26rft.spage%253D1131%26rft_id%253Dinfo%253Apmid%252F1459432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=1459432&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JZ83200024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.3" data-doi="10.1007/BF00037246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broich</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">A cluster analysis of wild and domesticated soybean phenotypes.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D29%26rft.spage%253D23%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00037246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00037246&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1980JV21600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-1.2.117.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burnham</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1962</span> <span class="cit-source">Discussions in Cytogenetics.</span> <span class="cit-publ-name">Burgess Publ. Com</span>, <span class="cit-publ-loc">Minneapolis, MN</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genetic variation and relationships among cultivated, wild, and semiwild soybean.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">316</span>–<span class="cit-lpage">325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DChen%26rft.auinit1%253DY.%26rft.volume%253D44%26rft.issue%253D1%26rft.spage%253D316%26rft.epage%253D325%26rft.atitle%253DGenetic%2BVariation%2Band%2BRelationships%2Bamong%2BCultivated%252C%2BWild%252C%2Band%2BSemiwild%2BSoybean%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000187633900043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.6" data-doi="10.1080/07352680802333904"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chung</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Broadening the genetic base of soybean: A multidisciplinary approach.</span> <abbr class="cit-jnl-abbrev">Crit. Rev. Plant Sci.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrit.%2BRev.%2BPlant%2BSci.%26rft.volume%253D27%26rft.spage%253D295%26rft_id%253Dinfo%253Adoi%252F10.1080%252F07352680802333904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/07352680802333904&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.7" data-doi="10.2135/cropsci1999.3951464x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cregan</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarvik</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bush</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lark</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">An integrated genetic linkage map of the soybean genome.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1464</span>–<span class="cit-lpage">1490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCregan%26rft.auinit1%253DP.B.%26rft.volume%253D39%26rft.issue%253D5%26rft.spage%253D1464%26rft.epage%253D1490%26rft.atitle%253DAn%2BIntegrated%2BGenetic%2BLinkage%2BMap%2Bof%2Bthe%2BSoybean%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1999.3951464x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1999.3951464x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000082813800030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cregan</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kollipara</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fogarty</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Primary trisomics and SSR markers as tools to associate chromosomes with linkage groups in soybean.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">1262</span>–<span class="cit-lpage">1267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCregan%26rft.auinit1%253DP.%2BB.%26rft.volume%253D41%26rft.issue%253D4%26rft.spage%253D1262%26rft.epage%253D1267%26rft.atitle%253DPrimary%2BTrisomics%2Band%2BSSR%2BMarkers%2Bas%2BTools%2Bto%2BAssociate%2BChromosomes%2Bwith%2BLinkage%2BGroups%2Bin%2BSoybean%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-1.2.117.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Delannay</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kilen</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-source">Screening of Soybean (Glycine max) Accessions and G. soja Accessions for Chromosome Interchanges and Inversions. Agronomy Abstracts</span>, p. <span class="cit-fpage">63</span>. <span class="cit-publ-name">American Society of Agronomy</span>, <span class="cit-publ-loc">Madison, WI</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.2.117.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Lorenzi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molteni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parma</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">FISH mapping in cattle (<em>Bos taurus</em> L.) is not yet out of fashion.</span> <abbr class="cit-jnl-abbrev">J. Appl. Genet.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">497</span>–<span class="cit-lpage">499</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bapplied%2Bgenetics%26rft.stitle%253DJ%2BAppl%2BGenet%26rft.aulast%253DLorenzi%26rft.auinit1%253DL.%26rft.volume%253D51%26rft.issue%253D4%26rft.spage%253D497%26rft.epage%253D499%26rft.atitle%253DFISH%2Bmapping%2Bin%2Bcattle%2B%2528Bos%2Btaurus%2BL.%2529%2Bis%2Bnot%2Byet%2Bout%2Bof%2Bfashion.%26rft_id%253Dinfo%253Apmid%252F21063067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21063067&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.11" data-doi="10.1007/s001220050618"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Diwan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cregan</span>  <span class="cit-name-given-names">P. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Automated sizing of fluorescent-labeled simple sequence repeat (SSR) markers to assay genetic variation in soybean.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">723</span>–<span class="cit-lpage">733</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D95%26rft.spage%253D723%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050618&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997YF71500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.12" data-doi="10.1093/jhered/esj002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durrant</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardunia</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Livingstone</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jellen</span>  <span class="cit-name-given-names">E. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">An algorithm for analyzing linkages affected by heterozygous translocations: QuadMap.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">62</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesj002%26rft_id%253Dinfo%253Apmid%252F16394257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/esj002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16394257&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endrizzi</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Alternate -1 and alternate -2 disjunction in heterozygous reciprocal translocations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">55</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D77%26rft.spage%253D55%26rft_id%253Dinfo%253Apmid%252F17248653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17248653&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1974T582100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.14" data-doi="10.1534/genetics.109.113753"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Findley</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cannon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varala</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A fluorescence in situ hybridization system for karyotyping soybean.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">727</span>–<span class="cit-lpage">744</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D185%26rft.spage%253D727%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.113753%26rft_id%253Dinfo%253Apmid%252F20421607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.109.113753&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20421607&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281906800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-other" id="cit-1.2.117.15"><div class="cit-metadata"><cite><span class="cit-comment">Gill, N., S. Findley, J. G. Walling, C. Hans, J. Ma <em>et al.</em>, 2009 Molecular and chromosomal evidence for allopolyploidy in soybean, <em>Glycine max</em> (L.) Merr. Plant Physiol. DOI: 10.1104/pp.109.137935</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gottschalk</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">The performance of translocation homozygous <em>Pisum</em> lines in comparison with translocation heterozygous plants.</span> <abbr class="cit-jnl-abbrev">Nucleus</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleus%26rft.volume%253D21%26rft.spage%253D29%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-1.2.117.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1962</span> <span class="cit-source">Botanical Histochemistry.</span> <span class="cit-publ-name">Freeman</span>, <span class="cit-publ-loc">San Francisco, CA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.18" data-doi="10.1007/s00122-003-1482-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kato</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Molecular mapping of four ovule lethal mutants in soybean.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">577</span>–<span class="cit-lpage">585</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DKato%26rft.auinit1%253DK.%2BK.%26rft.volume%253D108%26rft.issue%253D4%26rft.spage%253D577%26rft.epage%253D585%26rft.atitle%253DMolecular%2Bmapping%2Bof%2Bfour%2Bovule%2Blethal%2Bmutants%2Bin%2Bsoybean.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-003-1482-y%26rft_id%253Dinfo%253Apmid%252F14610643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-003-1482-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14610643&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000189207100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.19" data-doi="10.1016/0888-7543(87)90010-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abrahamson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barlow</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">MAPMAKER: an interactive computer package for constructing primary genetic linkage maps of experimental and natural populations.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">174</span>–<span class="cit-lpage">181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D1%26rft.issue%253D2%26rft.spage%253D174%26rft.epage%253D181%26rft.atitle%253DMAPMAKER%253A%2Ban%2Binteractive%2Bcomputer%2Bpackage%2Bfor%2Bconstructing%2Bprimary%2Bgenetic%2Blinkage%2Bmaps%2Bof%2Bexperimental%2Band%2Bnatural%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0888-7543%252887%252990010-3%26rft_id%253Dinfo%253Apmid%252F3692487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0888-7543(87)90010-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3692487&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.20" data-doi="10.2135/cropsci2007.05.0257"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hwang</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blake</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">So</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic diversity of wild soybean (<em>Glycine soja</em> Sieb. and Zucc.) accessions from South Korea and other countries.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">606</span>–<span class="cit-lpage">616</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DLee%26rft.auinit1%253DJ.-D.%26rft.volume%253D48%26rft.issue%253D2%26rft.spage%253D606%26rft.epage%253D616%26rft.atitle%253DGenetic%2BDiversity%2Bof%2BWild%2BSoybean%2B%2528Glycine%2Bsoja%2BSieb.%2Band%2BZucc.%2529%2BAccessions%2Bfrom%2BSouth%2BKorea%2Band%2BOther%2BCountries%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2007.05.0257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2007.05.0257&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000254855500024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.21" data-doi="10.1534/genetics.105.041616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">J.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hass-Jacobus</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanmiguel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walling</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Molecular and cytogenetic characterization of gene-poor, heterochromatic regions of <em>Glycine max</em> L. Merr.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">1221</span>–<span class="cit-lpage">1230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D170%26rft.spage%253D1221%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.041616%26rft_id%253Dinfo%253Apmid%252F15879505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.105.041616&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15879505&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231097700021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.22" data-doi="10.1093/jhered/91.6.423"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Livingstone</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jahn</span>  <span class="cit-name-given-names">M. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Linkage mapping in populations with karyotypic rearrangements.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">428</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DLivingstone%26rft.auinit1%253DK.%2BD.%26rft.volume%253D91%26rft.issue%253D6%26rft.spage%253D423%26rft.epage%253D428%26rft.atitle%253DLinkage%2Bmapping%2Bin%2Bpopulations%2Bwith%2Bkaryotypic%2Brearrangements%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F91.6.423%26rft_id%253Dinfo%253Apmid%252F11218078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/91.6.423&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11218078&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.23" data-doi="10.1186/1471-2164-11-184"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McClean</span>  <span class="cit-name-given-names">P.E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mamidi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McConnell</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chikara</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Synteny mapping between common bean and soybean reveals extensive blocks of shared loci.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMcClean%26rft.auinit1%253DP.%2BE.%26rft.volume%253D11%26rft.spage%253D184%26rft.epage%253D184%26rft.atitle%253DSynteny%2Bmapping%2Bbetween%2Bcommon%2Bbean%2Band%2Bsoybean%2Breveals%2Bextensive%2Bblocks%2Bof%2Bshared%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-184%26rft_id%253Dinfo%253Apmid%252F20298570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-184&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20298570&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.24" data-doi="10.1073/pnas.1009526107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Whole-genome sequencing and intensive analysis of the undomesticated soybean (<em>Glycine soja</em> Sieb. and Zucc.) genome.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U.S.A.</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">22032</span>–<span class="cit-lpage">22037</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.S.A.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1009526107%26rft_id%253Dinfo%253Apmid%252F21131573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzUxLzIyMDMyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS8yLzExNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.25" data-doi="10.1093/jhered/90.6.648"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mahama</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deaderick</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sadanaga</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newhouse</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Cytogenetic analysis of translocations in soybean.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">648</span>–<span class="cit-lpage">653</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DMahama%26rft.auinit1%253DA.%2BA.%26rft.volume%253D90%26rft.issue%253D6%26rft.spage%253D648%26rft.epage%253D653%26rft.atitle%253DCytogenetic%2Banalysis%2Bof%2Btranslocations%2Bin%2BSoybean%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F90.6.648%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/90.6.648&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000083923200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.26" data-doi="10.2135/cropsci2003.1602"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mahama</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Translocation breakpoints in soybean classical genetic linkage groups 6 and 8.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1602</span>–<span class="cit-lpage">1609</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DMahama%26rft.auinit1%253DA.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D1602%26rft.epage%253D1609%26rft.atitle%253DTranslocation%2BBreakpoints%2Bin%2BSoybean%2BClassical%2BGenetic%2BLinkage%2BGroups%2B6%2Band%2B8%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2003.1602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2003.1602&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000185174300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.27" data-doi="10.2135/cropsci2006.09.0605"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nichols</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lianzheng</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pei</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glover</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diers</span>  <span class="cit-name-given-names">B. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Variability among Chinese <em>Glycine soja</em> and Chinese and North American soybean genotypes.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1289</span>–<span class="cit-lpage">1298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DNichols%26rft.auinit1%253DD.%2BM.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D1289%26rft.epage%253D1298%26rft.atitle%253DVariability%2Bamong%2BChinese%2BGlycine%2Bsoja%2Band%2BChinese%2Band%2BNorth%2BAmerican%2BSoybean%2BGenotypes%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.09.0605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.09.0605&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.28" data-doi="10.1007/BF00021893"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heer</span>  <span class="cit-name-given-names">H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Agronomic characteristics and genetics of a chromosome interchange in soybean.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">651</span>–<span class="cit-lpage">663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D33%26rft.spage%253D651%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00021893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00021893&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1984AEX0700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-book" id="cit-1.2.117.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kilen</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Qualitative genetics and cytogenetics</span>, pp. <span class="cit-fpage">135</span>–<span class="cit-lpage">209</span> in <span class="cit-source"><em>Soybeans: Improvement</em>, <em>Production</em>, <em>and Uses</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Wilcox</span>  <span class="cit-name-given-names">J. R.</span></span>. <span class="cit-publ-name">ASA, CSSA, and SSSS</span>, <span class="cit-publ-loc">Madison, WI</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newhouse</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graybosch</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delannay</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Chromosome structure of the wild soybean: Accessions from China and the Soviet Union of <em>Glycine soja</em> Sieb. &amp; Zucc.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DPalmer%26rft.auinit1%253DR.%2BG.%26rft.volume%253D78%26rft.issue%253D4%26rft.spage%253D243%26rft.epage%253D247%26rft.atitle%253DChromosome%2Bstructure%2Bof%2Bthe%2Bwild%2Bsoybean%253A%2BAccessions%2Bfrom%2BChina%2Band%2Bthe%2BSoviet%2BUnion%2Bof%2BGlycine%2Bsoja%2BSieb.%2B%2526%2BZucc.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1987J524200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genetics and cytology of chromosome inversions in soybean germplasm.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">683</span>–<span class="cit-lpage">687</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DPalmer%26rft.auinit1%253DR.G.%26rft.volume%253D40%26rft.issue%253D3%26rft.spage%253D683%26rft.epage%253D687%26rft.atitle%253DGenetics%2Band%2BCytology%2Bof%2BChromosome%2BInversions%2Bin%2BSoybean%2BGermplasm%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000089410400016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peters</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Datema</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szinay</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vanstaveren</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schijlen</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title"><em>Solanum lycopersicum</em> cv. Heinz 1706 chromosome 6: distribution and abundance of genes and retrotransposable elements.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">857</span>–<span class="cit-lpage">869</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DPeters%26rft.auinit1%253DS.%2BA.%26rft.volume%253D58%26rft.issue%253D5%26rft.spage%253D857%26rft.epage%253D869%26rft.atitle%253DSolanum%2Blycopersicum%2Bcv.%2BHeinz%2B1706%2Bchromosome%2B6%253A%2Bdistribution%2Band%2Babundance%2Bof%2Bgenes%2Band%2Bretrotransposable%2Belements.%26rft_id%253Dinfo%253Apmid%252F19207213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19207213&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sadanaga</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grindeland</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Locating the <em>w1</em> locus on the satellite chromosome in soybean.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">147</span>–<span class="cit-lpage">151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DSadanaga%26rft.auinit1%253DK.%26rft.volume%253D24%26rft.issue%253D1%26rft.spage%253D147%26rft.epage%253D151%26rft.atitle%253DLocating%2Bthe%2Bw1%2BLocus%2Bon%2Bthe%2BSatellite%2BChromosome%2Bin%2BSoybean%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sadanaga</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newhouse</span>  <span class="cit-name-given-names">K. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Identifying translocations in soybeans.</span> <abbr class="cit-jnl-abbrev">Soybean Genetics Newsletter</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">129</span>–<span class="cit-lpage">130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSoybean%2BGenetics%2BNewsletter%26rft.volume%253D9%26rft.spage%253D129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.35" data-doi="10.1038/nature08670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmutz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cannon</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlueter</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitros</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome sequence of the paleopolyploid soybean.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">463</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSchmutz%26rft.auinit1%253DJ.%26rft.volume%253D463%26rft.issue%253D7278%26rft.spage%253D178%26rft.epage%253D183%26rft.atitle%253DGenome%2Bsequence%2Bof%2Bthe%2Bpalaeopolyploid%2Bsoybean.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08670%26rft_id%253Dinfo%253Apmid%252F20075913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20075913&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273582700025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-other" id="cit-1.2.117.36"><div class="cit-metadata"><cite><span class="cit-comment">Sellner, L. M., 1990 A genetic study of seven independent translocations in soybean. M.S. Thesis. Iowa State University, Ames, Iowa</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hatfield</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atherly</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Chloroplast DNA variation in the genus <em>Glycine</em> subgenus <em>soja</em>.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">26</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DShoemaker%26rft.auinit1%253DR.%2BC.%26rft.volume%253D77%26rft.issue%253D1%26rft.spage%253D26%26rft.epage%253D30%26rft.atitle%253DChloroplast%2BDNA%2Bvariation%2Bin%2Bthe%2Bgenus%2BGlycine%2Bsubgenus%2BSoja%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1986A238800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.38" data-doi="10.1007/BF00303516"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hymowitz</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">The genomic relationship between <em>Glycine max</em> (L.) Merr. and <em>G. soja</em> Sieb. and Zucc. as revealed by pachytene chromosome analysis.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">705</span>–<span class="cit-lpage">711</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D76%26rft.spage%253D705%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00303516%26rft_id%253Dinfo%253Apmid%252F24232348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00303516&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24232348&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988R001900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Skvortzow</span>  <span class="cit-name-given-names">B. V.</span></span></li></ol><cite>, <span class="cit-pub-date">1927</span> <span class="cit-article-title">The soybean - wild and cultivated in Eastern Asia. Proc. Manchurian Res. Soc. Publ. Ser. A. Natural History.</span> <abbr class="cit-jnl-abbrev">History Sect.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHistory%2BSect.%26rft.volume%253D22%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.40" data-doi="10.1159/000218137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stack</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Royer</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shearer</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giovannoni</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Role of fluorescence in situ hybridization in sequencing the tomato genome.</span> <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">339</span>–<span class="cit-lpage">350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253DStack%26rft.auinit1%253DS.%2BM.%26rft.volume%253D124%26rft.issue%253D3-4%26rft.spage%253D339%26rft.epage%253D350%26rft.atitle%253DRole%2Bof%2Bfluorescence%2Bin%2Bsitu%2Bhybridization%2Bin%2Bsequencing%2Bthe%2Btomato%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000218137%26rft_id%253Dinfo%253Apmid%252F19556785%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000218137&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19556785&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267648000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.41" data-doi="10.1186/1471-2164-8-132"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swaminathan</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varala</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Global repeat discovery and estimation of genomic copy number in a large, complex genome using a high-throughput 454 sequence survey.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DSwaminathan%26rft.auinit1%253DK.%26rft.volume%253D8%26rft.spage%253D132%26rft.epage%253D132%26rft.atitle%253DGlobal%2Brepeat%2Bdiscovery%2Band%2Bestimation%2Bof%2Bgenomic%2Bcopy%2Bnumber%2Bin%2Ba%2Blarge%252C%2Bcomplex%2Bgenome%2Busing%2Ba%2Bhigh-throughput%2B454%2Bsequence%2Bsurvey.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-8-132%26rft_id%253Dinfo%253Apmid%252F17524145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-8-132&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17524145&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.42" data-doi="10.1111/j.1365-313X.2008.03626.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szinay</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khrustaleva</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peters</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schijlen</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">High-resolution chromosome mapping of BACs using multi-colour FISH and pooled-BAC FISH as a backbone for sequencing tomato chromosome 6.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">627</span>–<span class="cit-lpage">637</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DSzinay%26rft.auinit1%253DD.%26rft.volume%253D56%26rft.issue%253D4%26rft.spage%253D627%26rft.epage%253D637%26rft.atitle%253DHigh-resolution%2Bchromosome%2Bmapping%2Bof%2BBACs%2Busing%2Bmulti-colour%2BFISH%2Band%2Bpooled-BAC%2BFISH%2Bas%2Ba%2Bbackbone%2Bfor%2Bsequencing%2Btomato%2Bchromosome%2B6.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2008.03626.x%26rft_id%253Dinfo%253Apmid%252F18643986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2008.03626.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18643986&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260824700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.43" data-doi="10.1007/BF00041174"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vahedian</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okimoto</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danna</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Genomic organization and evolution of the soybean SB92 satellite sequence.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">857</span>–<span class="cit-lpage">862</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DVahedian%26rft.auinit1%253DM.%26rft.volume%253D29%26rft.issue%253D4%26rft.spage%253D857%26rft.epage%253D862%26rft.atitle%253DGenomic%2Borganization%2Band%2Bevolution%2Bof%2Bthe%2Bsoybean%2BSB92%2Bsatellite%2Bsequence.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00041174%26rft_id%253Dinfo%253Apmid%252F8541510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00041174&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8541510&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995TL29900021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Veatch</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1934</span> <span class="cit-article-title">Chromosomes of the soy bean.</span> <abbr class="cit-jnl-abbrev">Bot. Gaz.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBot.%2BGaz.%26rft.volume%253D96%26rft.spage%253D189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.45" data-doi="10.1093/jhered/93.1.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">MapChart: Software for the graphical presentation of linkage maps and QTLs.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DVoorrips%26rft.auinit1%253DR.%2BE.%26rft.volume%253D93%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D78%26rft.atitle%253DMapChart%253A%2BSoftware%2Bfor%2Bthe%2BGraphical%2BPresentation%2Bof%2BLinkage%2BMaps%2Band%2BQTLs%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F93.1.77%26rft_id%253Dinfo%253Apmid%252F12011185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/93.1.77&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12011185&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175627300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-1.2.117.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">L. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1948</span> <span class="cit-article-title">Inheritance in a species cross in soybean.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWhittinghill%26rft.auinit1%253DM.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D131%26rft.epage%253D131%26rft.atitle%253DA%2Bspermatogonial%2Bmap%2Bof%2Bchromosome%2B3%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Apmid%252F18903882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18903882&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F2%2F117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/1/2/117.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/2/105" title="A QTL Study for Regions Contributing to Arabidopsis thaliana Root Skewing on Tilted Surfaces" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/2/131" title="Quantitative Genetics of Genomic Imprinting: A Comparison of Simple Variance Derivations, the Effects of Inbreeding, and Response to Selection" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/1/2"><span>Volume 1 Issue 2, July 2011</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/1/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/1/2.cover.gif?itok=hCLGYhoa" width="248" height="330" alt="G3: Genes, Genomes, Genetics: 1 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/2/117" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-QgYROO1o1XtgsYXzzJYSU_iM_BAksyuWOerNr0kSAmE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigation-1" class="" data-icon-position="" data-hide-link-title="0">Investigation</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/1/2/117" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/1/2/117" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/1/2/117" class="active" data-icon-position="" data-hide-link-title="0">Fluorescence In Situ Hybridization–Based Karyotyping of Soybean Translocation Lines</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459165" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-hwlOxiSunj0U1LSomUYTIXKYit07ncZ9ED-wBl8Zqr4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/2/117.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/1/2/117" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ITvh55eVjQmfjqAE55VSrJRds6kljPUbTAfElqJcKFI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/1/2/117" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/2/117.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459165" id="node459165--2" data-pisa="ggg;1/2/117" data-pisa-master="ggg;g3.111.000034" data-apath="/ggg/1/2/117.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fluorescence <em>In Situ</em> Hybridization–Based Karyotyping of Soybean Translocation Lines</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Seth D. Findley</span>, <span class="highwire-citation-author" data-delta="1">Allison L. Pappas</span>, <span class="highwire-citation-author" data-delta="2">Yaya Cui</span>, <span class="highwire-citation-author" data-delta="3">James A. Birchler</span>, <span class="highwire-citation-author" data-delta="4">Reid G. Palmer</span> and <span class="highwire-citation-author" data-delta="5">Gary Stacey</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">117-129;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000034 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459165--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Seth D. Findley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Soybean Biotechnology, Division of Plant Sciences, University of Missouri, Columbia, Missouri 65211</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Seth%2BD.%2BFindley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Findley%20SD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASeth%2BD.%2BFindley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Allison L. Pappas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA-ARS, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allison%2BL.%2BPappas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pappas%20AL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAllison%2BL.%2BPappas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yaya Cui </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Soybean Biotechnology, Division of Plant Sciences, University of Missouri, Columbia, Missouri 65211</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yaya%2BCui%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cui%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYaya%2BCui%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">James A. Birchler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Division of Biological Sciences, University of Missouri, Columbia, Missouri 65211</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BA.%2BBirchler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Birchler%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BA.%2BBirchler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Reid G. Palmer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA-ARS, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Reid%2BG.%2BPalmer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palmer%20RG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AReid%2BG.%2BPalmer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gary Stacey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Soybean Biotechnology, Division of Plant Sciences, University of Missouri, Columbia, Missouri 65211</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BStacey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stacey%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGary%2BStacey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:staceyg@missouri.edu" class="" data-icon-position="" data-hide-link-title="0">staceyg@missouri.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/1/2/117" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/1/2/117&amp;title=Fluorescence%20In%20Situ%20Hybridization%E2%80%93Based%20Karyotyping%20of%20Soybean%20Translocation%20Lines" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/1/2/117&amp;title=Fluorescence%20In%20Situ%20Hybridization%E2%80%93Based%20Karyotyping%20of%20Soybean%20Translocation%20Lines" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/1/2/117&amp;title=Fluorescence%20In%20Situ%20Hybridization%E2%80%93Based%20Karyotyping%20of%20Soybean%20Translocation%20Lines" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/1/2/117&amp;text=Fluorescence%20In%20Situ%20Hybridization%E2%80%93Based%20Karyotyping%20of%20Soybean%20Translocation%20Lines" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/1/2/117&amp;title=Fluorescence%20In%20Situ%20Hybridization%E2%80%93Based%20Karyotyping%20of%20Soybean%20Translocation%20Lines" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/1/2/117&amp;t=Fluorescence%20In%20Situ%20Hybridization%E2%80%93Based%20Karyotyping%20of%20Soybean%20Translocation%20Lines" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/1/2/117&amp;title=Fluorescence%20In%20Situ%20Hybridization%E2%80%93Based%20Karyotyping%20of%20Soybean%20Translocation%20Lines" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/1/2/117&amp;title=Fluorescence%20In%20Situ%20Hybridization%E2%80%93Based%20Karyotyping%20of%20Soybean%20Translocation%20Lines" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459165/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459165" id="node459165--3" data-pisa="ggg;1/2/117" data-pisa-master="ggg;g3.111.000034" data-apath="/ggg/1/2/117.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fluorescence <em>In Situ</em> Hybridization–Based Karyotyping of Soybean Translocation Lines</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Seth D. Findley</span>, <span class="highwire-citation-author" data-delta="1">Allison L. Pappas</span>, <span class="highwire-citation-author" data-delta="2">Yaya Cui</span>, <span class="highwire-citation-author" data-delta="3">James A. Birchler</span>, <span class="highwire-citation-author" data-delta="4">Reid G. Palmer</span> and <span class="highwire-citation-author" data-delta="5">Gary Stacey</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">117-129;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000034 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459165--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Seth D. Findley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Soybean Biotechnology, Division of Plant Sciences, University of Missouri, Columbia, Missouri 65211</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Seth%2BD.%2BFindley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Findley%20SD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASeth%2BD.%2BFindley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Allison L. Pappas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA-ARS, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allison%2BL.%2BPappas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pappas%20AL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAllison%2BL.%2BPappas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yaya Cui </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Soybean Biotechnology, Division of Plant Sciences, University of Missouri, Columbia, Missouri 65211</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yaya%2BCui%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cui%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYaya%2BCui%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">James A. Birchler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Division of Biological Sciences, University of Missouri, Columbia, Missouri 65211</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BA.%2BBirchler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Birchler%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BA.%2BBirchler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Reid G. Palmer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA-ARS, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Reid%2BG.%2BPalmer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palmer%20RG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AReid%2BG.%2BPalmer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gary Stacey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Soybean Biotechnology, Division of Plant Sciences, University of Missouri, Columbia, Missouri 65211</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BStacey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stacey%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGary%2BStacey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:staceyg@missouri.edu" class="" data-icon-position="" data-hide-link-title="0">staceyg@missouri.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=117&amp;rft.epage=129&amp;rft.atitle=Fluorescence%20In%20Situ%20Hybridization%E2%80%93Based%20Karyotyping%20of%20Soybean%20Translocation%20Lines&amp;rft.volume=1&amp;rft.issue=2&amp;rft.date=2011-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Findley%2C+Seth+D.&amp;rft.au=Pappas%2C+Allison+L.&amp;rft.au=Cui%2C+Yaya&amp;rft.au=Birchler%2C+James+A.&amp;rft.au=Palmer%2C+Reid+G.&amp;rft.au=Stacey%2C+Gary"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459165/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459165/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459165/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459165/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459165/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459165/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459165/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459165/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459165/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459165/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459165/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459165/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496056" id="node496056" data-pisa="ggg;g3.120.401188v1" data-pisa-master="ggg;g3.120.401188" data-apath="/ggg/early/2020/03/27/g3.120.401188.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/27/g3.120.401188" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">MYCs and PIFs Act Independently in Arabidopsis Growth Regulation</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496055" id="node496055" data-pisa="ggg;g3.120.401106v1" data-pisa-master="ggg;g3.120.401106" data-apath="/ggg/early/2020/03/27/g3.120.401106.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/27/g3.120.401106" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Identifying Small Molecules That Promote Quasipalindrome-Associated Template-Switch Mutations in <em>Escherichia coli</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496053" id="node496053" data-pisa="ggg;g3.120.401160v1" data-pisa-master="ggg;g3.120.401160" data-apath="/ggg/early/2020/03/27/g3.120.401160.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/27/g3.120.401160" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Evaluation of STAR and Kallisto on Single Cell RNA-Seq Data Alignment</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigation" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigation</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/1/2/117" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/1/2/117.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/1/2/117.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/1/2/117.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/1/2/117.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
