<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/7/803" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/7/803.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/7/803.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/7/803.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/7/803.atom" />
<meta name="HW.pisa" content="ggg;2/7/803" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Impact of Sampling Schemes on Demographic Inference: An Empirical Study in Two Species with Different Mating Systems and Demographic Histories" />
<meta name="DC.Identifier" content="10.1534/g3.112.002410" />
<meta name="DC.Date" content="2012-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 St. Onge et al." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Most species have at least some level of genetic structure. Recent simulation studies have shown that it is important to consider population structure when sampling individuals to infer past population history. The relevance of the results of these computer simulations for empirical studies, however, remains unclear. In the present study, we use DNA sequence datasets collected from two closely related species with very different histories, the selfing species Capsella rubella and its outcrossing relative C. grandiflora , to assess the impact of different sampling strategies on summary statistics and the inference of historical demography. Sampling strategy did not strongly influence the mean values of Tajima’s D in either species, but it had some impact on the variance. The general conclusions about demographic history were comparable across sampling schemes even when resampled data were analyzed with approximate Bayesian computation (ABC). We used simulations to explore the effects of sampling scheme under different demographic models. We conclude that when sequences from modest numbers of loci (&lt;60) are analyzed, the sampling strategy is generally of limited importance. The same is true under intermediate or high levels of gene flow (4Nm &gt; 2–10) in models in which global expansion is combined with either local expansion or hierarchical population structure. Although we observe a less severe effect of sampling than predicted under some earlier simulation models, our results should not be seen as an encouragement to neglect this issue. In general, a good coverage of the natural range, both within and between populations, will be needed to obtain a reliable reconstruction of a species’s demographic history, and in fact, the effect of sampling scheme on polymorphism patterns may itself provide important information about demographic history." />
<meta name="DC.Contributor" content="K. R. St. Onge" />
<meta name="DC.Contributor" content="A. E. Palmé" />
<meta name="DC.Contributor" content="S. I. Wright" />
<meta name="DC.Contributor" content="M. Lascoux" />
<meta name="article:published_time" content="2012-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Impact of Sampling Schemes on Demographic Inference: An Empirical Study in Two Species with Different Mating Systems and Demographic Histories" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Most species have at least some level of genetic structure. Recent simulation studies have shown that it is important to consider population structure when sampling individuals to infer past population history. The relevance of the results of these computer simulations for empirical studies, however, remains unclear. In the present study, we use DNA sequence datasets collected from two closely related species with very different histories, the selfing species &lt;i&gt;Capsella rubella&lt;/i&gt; and its outcrossing relative &lt;i&gt;C. grandiflora&lt;/i&gt;, to assess the impact of different sampling strategies on summary statistics and the inference of historical demography. Sampling strategy did not strongly influence the mean values of Tajima’s D in either species, but it had some impact on the variance. The general conclusions about demographic history were comparable across sampling schemes even when resampled data were analyzed with approximate Bayesian computation (ABC). We used simulations to explore the effects of sampling scheme under different demographic models. We conclude that when sequences from modest numbers of loci (&amp;lt;60) are analyzed, the sampling strategy is generally of limited importance. The same is true under intermediate or high levels of gene flow (4Nm &amp;gt; 2–10) in models in which global expansion is combined with either local expansion or hierarchical population structure. Although we observe a less severe effect of sampling than predicted under some earlier simulation models, our results should not be seen as an encouragement to neglect this issue. In general, a good coverage of the natural range, both within and between populations, will be needed to obtain a reliable reconstruction of a species’s demographic history, and in fact, the effect of sampling scheme on polymorphism patterns may itself provide important information about demographic history.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/07/01" />
<meta name="citation_mjid" content="ggg;2/7/803" />
<meta name="citation_id" content="2/7/803" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/7/803" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/7/803.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/7/803.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/7/803.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.002410" />
<meta name="citation_pmid" content="22870403" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="7" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="803" />
<meta name="citation_lastpage" content="814" />
<meta name="citation_author" content="K. R. St. Onge" />
<meta name="citation_author_institution" content="Department of Ecology and Genetics, Uppsala University, SE-75236 Uppsala, Sweden" />
<meta name="citation_author_email" content="K.R.St.Onge@uu.nl" />
<meta name="citation_author" content="A. E. Palmé" />
<meta name="citation_author_institution" content="Department of Ecology and Genetics, Uppsala University, SE-75236 Uppsala, Sweden" />
<meta name="citation_author" content="S. I. Wright" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology and Centre for Analysis of Genome Evolution and Function, University of Toronto, Toronto, Ontario M5S 3B2, Canada, and" />
<meta name="citation_author" content="M. Lascoux" />
<meta name="citation_author_institution" content="Department of Ecology and Genetics, Uppsala University, SE-75236 Uppsala, Sweden" />
<meta name="citation_author_institution" content="Laboratory of Evolutionary Genomics, CAS-MPG Partner Institute for Computational Biology, Chinese Academy of Sciences, Shanghai 200031, China" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Chikhi;citation_author=V. C. Sousa;citation_author=P. Luisi;citation_author=B. Goossens;citation_author=M. A. Beaumont;citation_title=The confounding effects of population structure, genetic diversity and the sampling scheme on the detection and quantification of population size changes;citation_pages=983-995;citation_volume=186;citation_year=2010;citation_pmid=20739713;citation_doi=10.1534/genetics.110.118661" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=R. I. Colautti;citation_author=C. G. Eckert;citation_author=S. C. H. Barrett;citation_title=Evolutionary constraints on adaptive evolution during range expansion in an invasive plant;citation_volume=277;citation_year=2010;citation_pmid=20164098;citation_doi=10.1098/rspb.2009.2231" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. De;citation_author=R. Durrett;citation_title=Stepping-stone spatial structure causes slow decay of linkage disequilibrium and shifts the site frequency spectrum;citation_pages=969-981;citation_volume=176;citation_year=2007;citation_pmid=17409067" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. De Mita;citation_author=J. Ronfort;citation_author=H. I. McKhann;citation_author=C. Poncet;citation_title=Investigation of the demographic and selective forces shaping the nucleotide diversity of genes involved in nod factor signalling in Medicago truncatula;citation_pages=2123-2133;citation_volume=177;citation_year=2007;citation_pmid=18073426;citation_doi=10.1534/genetics.107.076943" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=G. Evanno;citation_author=S. Regnaut;citation_author=J. Goudet;citation_title=Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study.;citation_pages=2611-2620;citation_volume=14;citation_year=2005;citation_issue=8;citation_pmid=15969739;citation_doi=10.1111/j.1365-294X.2005.02553.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Excoffier;citation_author=P. E. Smouse;citation_author=J. M. Quattro;citation_title=Analysis of molecular variance inferred from metric distances among DNA haplotypes: application to human mitochondrial DNA restriction data;citation_pages=479-491;citation_volume=131;citation_year=1992;citation_pmid=1644282" />
<meta name="citation_reference" content="citation_journal_title=Evol. Bioinform.;citation_author=L. Excoffier;citation_author=G. Laval;citation_author=S. Schneider;citation_title=Arlequin (version 3.0): an integrated software package for population genetics data analysis;citation_pages=47-50;citation_volume=1;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. C. Fay;citation_author=C.-I. Wu;citation_title=Hitchhiking under positive Darwinian selection;citation_pages=1405-1413;citation_volume=155;citation_year=2000;citation_pmid=10880498" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. P. Foxe;citation_author=T. Slotte;citation_author=E. A. Stahl;citation_author=B. Neuffer;citation_author=H. Hurka;citation_title=Recent speciation associated with the evolution of selfing in Capsella;citation_pages=5241-5245;citation_volume=10;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. X. Fu;citation_author=W. H. Li;citation_title=Statistical tests of neutrality of mutations;citation_pages=693-709;citation_volume=133;citation_year=1993;citation_pmid=8454210" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y.-L. Guo;citation_author=J. S. Bechsgaard;citation_author=T. Slotte;citation_author=B. Neuffer;citation_author=M. Lascoux;citation_title=Recent speciation of Capsella rubella from Capsella grandiflora, associated with loss of self-incompatibility and an extreme bottleneck;citation_volume=106;citation_year=2009;citation_pmid=19307580;citation_doi=10.1073/pnas.0808012106" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=J. L. Hamrick;citation_author=M. J. W. Godt;citation_title=Effects of Life History Traits on Genetic Diversity in Plant Species;citation_pages=1291-1298;citation_volume=351;citation_year=1996;citation_issue=1345;citation_doi=10.1098/rstb.1996.0112" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=R. E. Kass;citation_author=A. E. Raftery;citation_pages=773-795;citation_volume=90;citation_year=1995;citation_pmid=10783804;citation_doi=10.2307/2291091" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. C. Lewontin;citation_title=Twenty-five years ago in Genetics: electrophoresis in the development of evolutionary genetics: milestone or millstone?;citation_pages=657-662;citation_volume=128;citation_year=1991;citation_issue=4;citation_pmid=1916239" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Y. Li;citation_author=M. Stocks;citation_author=S. Hemmilä;citation_author=T. Kallman;citation_author=H. Zhu;citation_title=Demographic histories of four spruce (Picea) species of the Qinghai-Tibetan Plateau and neighboring areas inferred from multiple nuclear loci;citation_volume=27;citation_year=2010;citation_pmid=20031927;citation_doi=10.1093/molbev/msp301" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Librado;citation_author=J. Rozas;citation_title=DnaSP v5: a software for comprehensive analysis of DNA polymorphism data;citation_volume=25;citation_year=2009;citation_pmid=19346325;citation_doi=10.1093/bioinformatics/btp187" />
<meta name="citation_reference" content="citation_journal_title=Infect. Genet. Evol.;citation_author=J. S. Lopes;citation_author=M. A. Beaumont;citation_title=ABC: a useful Bayesian tool for the analysis of population data;citation_pages=825-832;citation_volume=10;citation_year=2010;citation_doi=10.1016/j.meegid.2009.10.010" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=H. Nybom;citation_title=Comparison of different nuclear DNA markers for estimating intraspecific genetic diversity in plants.;citation_pages=1143-1155;citation_volume=13;citation_year=2004;citation_issue=5;citation_pmid=15078452;citation_doi=10.1111/j.1365-294X.2004.02141.x" />
<meta name="citation_reference" content="citation_journal_title=Perspect. Plant Ecol. Evol. Syst.;citation_author=H. Nybom;citation_author=I. V. Bartish;citation_title=Effects of life history traits and sampling strategies on genetic diversity estimates obtained with RAPD markers in plants;citation_pages=93-114;citation_volume=3;citation_year=2000;citation_doi=10.1078/1433-8319-00006" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=B. M. Peter;citation_author=D. Wegmann;citation_author=L. Excoffier;citation_title=Distinguishing between population bottleneck and population subdivision by a Bayesian model choice procedure.;citation_pages=4648-4660;citation_volume=19;citation_year=2010;citation_issue=21;citation_pmid=20735743;citation_doi=10.1111/j.1365-294X.2010.04783.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of population structure using multilocus genotype data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_pmid=10835412" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Przeworski;citation_title=The signature of positive selection at randomly chosen loci;citation_pages=1179-1189;citation_volume=160;citation_year=2002;citation_pmid=11901132" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=S. E. Ptak;citation_author=M. Przeworski;citation_title=Evidence for population growth in humans is confounded by fine-scale population structure.;citation_pages=559-563;citation_volume=18;citation_year=2002;citation_issue=11;citation_pmid=12414185;citation_doi=10.1016/S0168-9525(02)02781-6" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=S. Qiu;citation_author=K. Zeng;citation_author=T. Slotte;citation_author=S. I. Wright;citation_author=D. Charlesworth;citation_title=Reduced efficacy of natural selection on codon usage bias in selfing Arabidopsis and Capsella species;citation_volume=3;citation_year=2011;citation_pmid=21856647;citation_doi=10.1093/gbe/evr085" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=N. Ray;citation_author=M. Currat;citation_author=L. Excoffier;citation_title=Intra-Deme Molecular Diversity in Spatially Expanding Populations;citation_pages=76-86;citation_volume=20;citation_year=2003;citation_issue=1;citation_pmid=12519909;citation_doi=10.1093/molbev/msg009" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Robertson;citation_title=Gene frequency distributions as a test of selective neutrality;citation_pages=775-785;citation_volume=81;citation_year=1975;citation_pmid=1213275" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. L. Simonsen;citation_author=G. A. Churchill;citation_author=C. F. Aquadro;citation_title=Properties of statistical tests of neutrality for DNA polymorphism data;citation_pages=413-429;citation_volume=141;citation_year=1995;citation_pmid=8536987" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=T. Slotte;citation_author=J. P. Foxe;citation_author=K. M. Hazzouri;citation_author=S. I. Wright;citation_title=Genome-wide evidence for efficient positive and purifying selection in Capsella grandiflora, a plant species with a large effective population size;citation_volume=27;citation_year=2010;citation_pmid=20194429;citation_doi=10.1093/molbev/msq062" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=K. St. Onge;citation_author=T. Källman;citation_author=T. Slotte;citation_author=M. Lascoux;citation_author=A. E. Palmé;citation_title=Contrasting demographic history and population structure in Capsella rubella and Capsella grandiflora, two closely related species with different mating systems.;citation_pages=3306-3320;citation_volume=20;citation_year=2011;citation_issue=16;citation_pmid=21777317;citation_doi=10.1111/j.1365-294X.2011.05189.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol. 10.1093/molbev/mss024;citation_title=Coalescent-based analysis distinguishes between allo- and autopolyploid origin in shepherd’s purse (Capsella bursa-pastoris);citation_author=K. St. Onge;citation_author=J. P. Foxe;citation_author=L. Junrui;citation_author=L. Haipeng;citation_author=K. Holm;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. Städler;citation_author=B. Haubold;citation_author=C. Merino;citation_author=W. Stephan;citation_author=P. Pfaffelhuber;citation_title=The impact of sampling schemes on the site frequency spectrum in nonequilibrium subdivided populations;citation_volume=182;citation_year=2009;citation_pmid=19237689;citation_doi=10.1534/genetics.108.094904" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=A comparison of bayesian methods for haplotype reconstruction from population genotype data.;citation_pages=1162-1169;citation_volume=73;citation_year=2003;citation_issue=5;citation_pmid=14574645;citation_doi=10.1086/379378" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. Stephens;citation_author=N. J. Smith;citation_author=P. Donnelly;citation_title=A new statistical method for haplotype reconstruction from population data.;citation_pages=978-989;citation_volume=68;citation_year=2001;citation_issue=4;citation_pmid=11254454;citation_doi=10.1086/319501" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=F. Tajima;citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism;citation_pages=585-595;citation_volume=123;citation_year=1989;citation_pmid=2513255" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. Thornton;citation_title=libsequence: a C++ class library for evolutionary genetic analysis;citation_pages=2325-2327;citation_volume=19;citation_year=2003;citation_issue=17;citation_pmid=14630667;citation_doi=10.1093/bioinformatics/btg316" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Thornton;citation_author=P. Andolfatto;citation_title=Approximate Bayesian inference reveals evidence for a recent, severe bottleneck in a Netherlands population of Drosophila melanogaster;citation_pages=1607-1619;citation_volume=172;citation_year=2006;citation_pmid=16299396;citation_doi=10.1534/genetics.105.048223" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Wakeley;citation_title=Nonequilibrium migration in human history;citation_pages=1863-1871;citation_volume=153;citation_year=1999;citation_pmid=10581291" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=J. Wakeley;citation_title=The coalescent in an island model of population subdivision with variation among demes.;citation_pages=133-144;citation_volume=59;citation_year=2001;citation_issue=2;citation_pmid=11302758;citation_doi=10.1006/tpbi.2000.1495" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Wakeley;citation_author=N. Aliacar;citation_title=Gene genealogies in a metapopulation;citation_pages=893-905;citation_volume=159;citation_year=2001;citation_pmid=11606561" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=G. A. Watterson;citation_title=On the number of segregating sites in genetical models without recombination.;citation_pages=256-276;citation_volume=7;citation_year=1975;citation_issue=2;citation_pmid=1145509;citation_doi=10.1016/0040-5809(75)90020-9" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=C. W. Wheat;citation_author=C. R. Haag;citation_author=J. H. Marden;citation_author=I. Hanski;citation_author=M. J. Frilander;citation_title=Nucleotide polymorphism at a gene (Pgi) under balancing selection in a butterfly metapopulation;citation_volume=27;citation_year=2010;citation_pmid=19793833;citation_doi=10.1093/molbev/msp227" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Wright;citation_title=Evolution in Mendelian populations;citation_pages=97-159;citation_volume=16;citation_year=1931;citation_pmid=17246615" />
<meta name="citation_reference" content="citation_journal_title=Ann. Eugen.;citation_author=S. Wright;citation_title=The genetical structure of populations;citation_pages=323-354;citation_volume=15;citation_year=1951;citation_pmid=24540312" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=S. Wright;citation_title=The interpretation of population structure by F-statistics with special regard to systems of mating;citation_pages=395-420;citation_volume=19;citation_year=1965;citation_doi=10.2307/2406450" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Zeng;citation_author=B. Charlesworth;citation_title=The joint effects of background selection and genetic recombination on local gene genealogies;citation_pages=251-266;citation_volume=189;citation_year=2011;citation_pmid=21705759;citation_doi=10.1534/genetics.111.130575" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Zeng;citation_author=Y.-X. Fu;citation_author=S. Shi;citation_author=C.-I. Wu;citation_title=Statistical tests for detecting positive selection by utilizing high-frequency variants;citation_pages=1431-1439;citation_volume=174;citation_year=2006;citation_pmid=16951063;citation_doi=10.1534/genetics.106.061432" />
<meta name="twitter:title" content="Impact of Sampling Schemes on Demographic Inference: An Empirical Study in Two Species with Different Mating Systems and Demographic Histories" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7.cover-source.jpg" />
<meta name="twitter:description" content="Most species have at least some level of genetic structure. Recent simulation studies have shown that it is important to consider population structure when sampling individuals to infer past population history. The relevance of the results of these computer simulations for empirical studies, however, remains unclear. In the present study, we use DNA sequence datasets collected from two closely related species with very different histories, the selfing species Capsella rubella and its outcrossing relative C. grandiflora , to assess the impact of different sampling strategies on summary statistics and the inference of historical demography. Sampling strategy did not strongly influence the mean values of Tajima’s D in either species, but it had some impact on the variance. The general conclusions about demographic history were comparable across sampling schemes even when resampled data were analyzed with approximate Bayesian computation (ABC). We used simulations to explore the effects of sampling scheme under different demographic models. We conclude that when sequences from modest numbers of loci (&lt;60) are analyzed, the sampling strategy is generally of limited importance. The same is true under intermediate or high levels of gene flow (4Nm &gt; 2–10) in models in which global expansion is combined with either local expansion or hierarchical population structure. Although we observe a less severe effect of sampling than predicted under some earlier simulation models, our results should not be seen as an encouragement to neglect this issue. In general, a good coverage of the natural range, both within and between populations, will be needed to obtain a reliable reconstruction of a species’s demographic history, and in fact, the effect of sampling scheme on polymorphism patterns may itself provide important information about demographic history." />
<meta name="og-title" property="og:title" content="Impact of Sampling Schemes on Demographic Inference: An Empirical Study in Two Species with Different Mating Systems and Demographic Histories" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/7/803" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Most species have at least some level of genetic structure. Recent simulation studies have shown that it is important to consider population structure when sampling individuals to infer past population history. The relevance of the results of these computer simulations for empirical studies, however, remains unclear. In the present study, we use DNA sequence datasets collected from two closely related species with very different histories, the selfing species Capsella rubella and its outcrossing relative C. grandiflora , to assess the impact of different sampling strategies on summary statistics and the inference of historical demography. Sampling strategy did not strongly influence the mean values of Tajima’s D in either species, but it had some impact on the variance. The general conclusions about demographic history were comparable across sampling schemes even when resampled data were analyzed with approximate Bayesian computation (ABC). We used simulations to explore the effects of sampling scheme under different demographic models. We conclude that when sequences from modest numbers of loci (&lt;60) are analyzed, the sampling strategy is generally of limited importance. The same is true under intermediate or high levels of gene flow (4Nm &gt; 2–10) in models in which global expansion is combined with either local expansion or hierarchical population structure. Although we observe a less severe effect of sampling than predicted under some earlier simulation models, our results should not be seen as an encouragement to neglect this issue. In general, a good coverage of the natural range, both within and between populations, will be needed to obtain a reliable reconstruction of a species’s demographic history, and in fact, the effect of sampling scheme on polymorphism patterns may itself provide important information about demographic history." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7.cover-source.jpg" />
    <title>Impact of Sampling Schemes on Demographic Inference: An Empirical Study in Two Species with Different Mating Systems and Demographic Histories | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ig8OCNyvzw6aw6vpfru2aRiJQcCISFvcm2MbQyLzIhU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459302","apath":"\/ggg\/2\/7\/803.atom","pisa":"ggg;2\/7\/803","ac":{"\/ggg\/2\/7\/803.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/7\/803.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/7\/803"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/7\/803"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/7\/803":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Q95j\\\\\\\/MlJLMnw1+rjADZJzeLT519QArkFMSNubVoR4pS2cNAmcJjJCx03fLfyu+TA68NZt3a9Bv2XXUJ5vYZGynSP+pbPBllihZwlfPOdAdyuvIff6Nie+ofCB\\\\\\\/2wJlimuz+Zya\\\\\\\/AolE0gkEUyjQADuTngeTSIkXRheG6Crb5lxafCl0H3YmxiajDJuAFmwIkKzLULZ4G+SngZ+YwmgYeVGZ\\\\\\\/8jFgfLEFwLiDWzUxPMrcSv3n7vHC8C0+60\\\\\\\/Is2aDGSg8xZ892DsIrO3tGs7Gg3nvx8faRHaiBvE9A3pRXCDABTsBBmlrmRtl7tRRqh+xs6mQNkK\\\\\\\/trMqVUI5MVc8YWkmW43ilRvm6ByXGY8R4x2Q62kdDmfNYiIR718JmQTlgv0tYoo39bFMjFHPWfrQfSM0c7OxJ8KtXQtWeUSuBwsgbppyezcThGwE7sNoHoXKFIoxaW7qpET4AQrvWGzULdhdyAdx+2gOxlF6lsLYLNAoihMiXzWeK74EyXk9GeLD5k3bSb38jZvNL0VTlPzVizkKWWyBa3vK\\\\\\\/GozvgljUNfNeAmfg5xNFxGeASpSwpjOqrXKXv5jNYFG0Vn21xwjYA8feo1BYrxFbV8XnWlYayUPGShtABr374PtvPLOCpOirgLuf39U\\\\\\\/ri7LteKcKIQjGtmlOi38vM8Wqnwes\\\\\\\/yb6vIgmA7x2vMSx9RwmAONaYj29Lm5MAXW5UeCs78qlOTBFRXHoZY8\\\\\\\/FcTl60Flr0qTjfe0fj0WwezFlNmYvvGb8fb37aHpOj7mScPuJE0pG3agWWc0lBnoZxskcjGOIwRBKnno6harQ\\\\\\\/ppHmYeL+XyAQBuVqNJWwks9SrerhLgtb0Cduv2lNhFSjrPbGaEegoiS3pD9c5IpECQDMlwpxl2m4uMMn3YDkh\\\\\\\/NSKBGIv8irwqXyzC78gcr5+nkR9Pf3NoDY2BbvYemRc+sQ8bwXOdNr6M34jFII6U7vljRJCV9yVZfqWuOStXvW+gsjaJlmZ+KvdxWXZG4D3D3x2UVFE8lJEibm0wlXgHOM+lgVpJolXxG0KXxQ5K2C3DzooyljW+\\\\\\\/02lYRaiAj7jNAC4IUJEO1YUTRtGmnE1OnCPV1Uqb3YfR5SyRlJtlh4jA+yEmuLyGtbQ3P9KA2w\\\\\\\/\\\\\\\/5ha4dg4jkCfv6bkqdCWUiZBdqisSb+meGuJD9sSOOnMw6XQ3cqOHa0E5pSGXllWdMg+yeDZHzV5U7CF0x0CmPmf7+GZ310Hz5Uo5nMDDdauGdvTVB8LLc78pf0YWbKWWLMoDXCvMv\\\u0022,\\\u0022iv\\\u0022:\\\u0022pcNepq3ojS7eMQ6\\\\\\\/PZqiIg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224d53c72089460d0cbbe29dd2e597bd6fa8ec68f947b64e5639aeada22709b433\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ZrMyu1TW0LG9UJvZqPygOpOwbXyGJ34FgfoMroFy2itkbFZ9n3gKVo4QNqlrTAlFNlBgTrKPq+wx+9Z01AE6t2EZRRwcrMgVnMtExFHuxhtOL\\\\\\\/shjzAzaTxraHA\\\\\\\/Yo7EpideZDThNXYUubgzxTL84AJx\\\\\\\/dL5OLqrfDk1AIEj6Nu5PLy\\\\\\\/RCXZtC5C2qdUG1JYXMb7gH8QCwXOldkzqRrJV3EYA6GexurSE8euY1kk2rsah5Ki57W3NeW+Xr3PSN5O93ox7SqMgLvaqGfg20ulIs2rVnFvZRMvj3wtLIpz0Aasu04HT0wN2NO+E2ghyT8CN4PymKLVvO1Gwin2ByfAw6b2UuSKwyaBjMJ46m9CVvNDp9VhTitXheBkDeIgT3xsS3cmUDxD\\\\\\\/6by1suMYkSawebJuNpy7qnU5NP0AhZ93Ki\\\\\\\/TVe4AOi7wCf2a1slNqmeLBdAPaUzzlThpsELMSelNj12KenriK4SNkf4bfQVKtz7CSfGdlXusjW5ghF5TCs\\\\\\\/Vp4gG8wrhVusQMwaCzUyiHpshNpqwSsd5PCql\\\\\\\/FigNvxdMAUxfP0Yl0N1QiD1HGjImTiJzosMX3HQ+m4gQfyAWYoqDYdANc\\\\\\\/LMLCKbiLpNYXnQxjmzyzMbmQmiYEIViXKkpX1fW\\\\\\\/mZRUlXLPqtJ7ZyVBdVIzS\\\\\\\/tEfZykMHSwU1X\\\\\\\/F0a4ZhKereIcv74Sxw29O4FeBnKTs12KkFbQOkxMe8iVYLPHj7AVZpweIpEIG9E1jFmD5Wc0I9uR8bvqxGxBCBVfb422WNS\\\\\\\/cijC7BgMIi25PZhgUpqcSnUeuugFmKTGVYnFvzm5yJ6LLm6fGUEpOWdqw+RGBmk6e373moLfBb43kr8+oiw0ARhS79L8R3BGT\\\\\\\/PVT4ngmWbZd06Zreq0q\\\\\\\/hf+RwRyo4j7+f1IaG3ef2E5Ahk\\\\\\\/ncOiOFvA5yQVPT1mGS7XUL6EFdHos6hqh\\\\\\\/6QdHk3Q5iMnRBtkl1\\\\\\\/DN7R2GkffiZOKTAFwXDB\\\\\\\/PqPzqLitVw11nFnmf5YtMv5PdAdFULyBDLciUxDRYYnJpbUs2VhOWqnofZS6uI2VXy+2Z7jTk84yJjOij+lZoMzKy4uCZAzNaA3ARhYoJSsSTFgua4yNNyd2vyc9NNfcKEB\\\\\\\/0Flxuit0XSxnHPrzFLaMIWnyiJ6Lo+GPgvOlCaCiMh1ccTwrnYqlt89wC7G5ZrIv1G5NWSVxNhJJe4vHcNasGnUHlrX1fFeyyAfHWg14LfcUB52MubIm0EtDndiaN1lMzhyPHq3j90Jxl8uErsBXluZLmcoJzpoRd5VjAzMNqMr8Q8d7m0rlVhpnTJjm0jrvF\\\\\\\/TwNqeC7Qn204KeVJK22UPettOlKiY8AwHlG6fZ+kdASTjzj4dsI+kB0wtMkcVdlXw2yyANWDwe7vIWC9GziQuAqylBDPzI9uOCX+Mofw9l1c5l+NZQPlrN\\\\\\\/XYDVye6o8+kNWFEGI32FSFvd0l+e1TxzyIQUhGG4M5xV7aOCe9IQoDuIOA322MjWE08yPL3U\\\\\\\/+V95+lb19d6Y\\\\\\\/FpxW3O7TaVWRoLUTCHS3NQA7p1jnAvQVdbRqaKaLcxkNpX2MlrikfmFCFFeNeeFKqCCOsZzq9yp+Q9SQMN4sYF3+39I8INDApJjeL41+Dg6QL2owAUYb4\\\\\\\/xngxnoU7ctHYR650DFHOA08t3bTk2tLXMlIAGk4X0ZDlGYphr16SmFgrYYBwURPconJ1u27aeRsbgMAWLCkBGgVL+huhvG8f9kVtU2lFZrGX3Xjn7DJexZvwz7qDnmi6WXGHTbqUCsSzqtkuAr9Ebzm8Ks9H6efp2X6uecX5dMqilPEPG+Mj\\\\\\\/0qqoouzJuuHN7xihJk4rwXfcyFq3gR7g8U48yi5WwZ\\\\\\\/wiKTXzoquCzoURgpR\\\\\\\/KyulyDkJRoxr7whrzG6AppcsbrmpUcFcU4Fhhe3LIH9MoH0XRf3K9pcs9w83O2wSQuOakuCJ1y+ayQO0WwYKYU0STe61zDJ6m82sTCbgpsRNBwPwattXID3FfY\\\\\\\/dRxBMgcKxW0yIv\\\\\\\/iepAOs8E1OY00A4iWArmJkCvh0DWT761Dq3FQVZJhKjJgX+Kipb7z+vNReomaoJVbNypCIkB8zFojBuavYzV3BOfNHVkcl\\\\\\\/YjfxZXxZtl9Gnv6n7sAUZkCoDI8aUN3K6OM8J565LE0jSAfAnV0qyL4xfJj1cyxcMOUFgdNpBdy5lfJjTQBByWadSlzXhtvL7bCbqjJjbr+DiICLxUuggjfuMJNHM+\\\\\\\/0StAMRshZEfXjvdkjmyqAcdtAncC5JAV4\\\\\\\/tOSrKS5UNDHpC723\\\\\\\/\\\\\\\/7P9q0VGeHBVbeWW2ezwf0tG\\\\\\\/dbOf3FeBowlojbABiTE73zAvDzWri6AZvCyIuPzJqZyvtEPh\\\\\\\/lWVKVszWs5nXGhkr9ttxFVNGNhSMtCJgJCYxvjfEDiDOsf6SOlHmYgDc1wJUXs0rouDpxHd\\\\\\\/3bkjVu6TXtXC4Jrl\\\\\\\/TmX692WReh5nmM39CC2kLJxuKEQPSfz3Z16YDmQ+fyA3wZJSNKnb4Nz7EH+Msz70pR2htIU7XPSOSW0GsBjmm40itf4lX0TjDPbRat71VLP3fq8mFxlqAvDZCiCQgymrNWv8wVgNiFac3wUJJ26L9RbGmcuoe\\\\\\\/9aRwAKwgQ\\\\\\\/abeYaHiVDNV39i4DrFGz+qGBKAwUV8yTOCEY4PeAk4lrd2yv0FLKQ9x\\\\\\\/E45a1G6MxWGDAXFIKFOIaEcPeJMkcdX54jwwpovds4IgVlMVGz04Y1EHL3stxlHY6to5rgBY\\\\\\\/kYMwwnLhSjixkopRNTAiqpoBEKQ7TrS+iqda03jBM5+ArsGbNU8EcEm8PCfBw609j1Xfy7YAelEOE7Tte+LdCh5fHJBSks2lJcZ7cbXG8u14hE5\\\\\\\/cBDyezvdi7M3IXSdeJo0al0fQDBamlrfaS\\\\\\\/oiIwhOlm1L6VMQBsn4X+wroiqpPPUJ\\\\\\\/Uku0qngYXb2n3qzxaH5GU7U8hI7rAJDNGa9fV7W4jnO6t42l8InrzeHpns0ficjrOP2h8GlGVNo56mBx0Rc2XoLFdKnFRxVAotexlitFnqV05fDchW7bWXn4zZSQw6x6UvtREMIL4+LOzaxRYSAWK8UPpsrHRNqV87B2fHbaczQVzpV+ntJ5Vsz8r43+qNuQJBlsavumF1yJgh+pM\\\\\\\/n4OuEmkTR0Ug0Np5gewiQYz39JKhfPkHhwjJiv9i+QBOcq5oAOUWyZLv7nblKhqMh6ZXxp0HxSz1BPnoT+0aqf3Z\\\\\\\/UNF54iyYRMXH0xjHb+Pd1vXT9DHD9SS\\\\\\\/Bc0aEhSC+OUezDIJ8A8oaDP0bUpRSMlaZcYgluNsh0YyNOZa0ehOjFBQdGKLZhPl2zrH3n9oAGkuFL474M68fouhfblOmdadZmL3QjF63HK8gCYoaYvjThHrjuSFxW3foSRRqPT41LB16NjR7vBWcnrwQLBdoI4+bFboNm13xEf3Hdmv8w68RYt\\\\\\\/39MwqmQAEQnQAMaCnui4MsBijIfyTlByX6gLD4HiHreeGgcz+PDwj\\\\\\\/4CgYP1LzaRqM8cwQEeNHZKojgPFNvyHskeN1WZc60MitOq7L8yQuhifKh7Fs1eANr6wvcUw8broOw\\\\\\\/1LV5028GqzSXp53cDKMhZ77J2b16nLc028DrYfRInOCW5TJz7WpFF7krIDyrq3kRwB0yILyMWmyo4Gf9itxALQOQNfgpFmyzWTxl5+dcMXHo\\\\\\\/WCnbqo0SQyV8vEqT70P0yuerHwFZZTVnNYPjuo5\\\\\\\/kD+txjdVgqtxBBxGdt5JONRhcqBqYq\\\\\\\/ZPYmtkhIi6qswwpzryC2dvmuiP5xXZ+0xEOp27hGvyamDHwZLoowub55ZONyy+1XjreiMkz3cvqE\\\\\\\/vTxkMDHgvrVCl\\\\\\\/y+9Dk1mHoGqfn4rL9on9xzXYBOhjioMtzpXgis1W2X0MbM6y6dtZqd13plfFSmvXhxL7LpxsTx+JYKntg8EceGyYY5zIWua94y2oOEuu2tuTIgX3JaXUeUpbkRCoN1ryVH1EGn1FgOWsmlIIMbehhxlEScWvMJRftrlxwoi9pUSjjzuTi0n+J\\\\\\\/k2hNM4GnVG9z37sGgs65Uljez\\\\\\\/KZe8J1S64S\\\\\\\/aUbE2YtZAjrq0q7ohqAygANb1OVJ9lcEI4IuLEFCGEbAcMlEA04Lh77kREd4V42jO6+k\\\\\\\/x9OS13Me\\\\\\\/iBIqleG0oY86ke7tvAfvi6Wkrh68d+qSWAXDFZfk98zDVbmfOhXx7U4RtVuDz5kfg8dyOW6DVM6hesYEGdeye6eidZ3cEuJlmwcHsNoZViQmY6wQtYZKTAntp7SF4FEInb46TnVNkuapyvuuDl2JWzBw95Dw3g10m1UKF1KyOpAYqItz1WC5ZmBNcSM4oT4HreBat\\\\\\\/ilVYVpCyP3QK4ehpcsY5beE1yngoquDPgi9VXD161YIdIM2jz3ogrYQyoxEa049AvHB2Idd+14p23BWERvvF3ZvkaKQzp9F6+xoX2YtJqiWa5RkHX16KqE4oZkm2ekihYHOt9YTJgLi4meZNNihpqrUIloyQETe2LYcyqVLjLy\\\\\\\/Eq\\\\\\\/Kjv60HO399OQz0Eq6DX\\\\\\\/b6PUchzUpCbQ3YeaEOBqUqqo4\\\\\\\/7Ilr8uXROvaQryBNPiwnpPbmhnAhyR+Oi\\\\\\\/tpKpacTuBZBxHcW+EDntPsMj1DO+T8GM2g25AXABeYg3isGwsbvm9g+KjQpi0oKoiKwr7Fezv+s4a+KNvmeeXM6V\\\\\\\/RAhX805syGh75LUi4Wlk4JJiWhwOrBqEOxIwTj1RUeTQPRpeYKG\\\\\\\/T7WAGbiRTnabfrQ\\\\\\\/imeYQsPfmS0ALdyOHc9C4wmKrkwRue2wsd993jMcTox0j8LaSuM9IJIpdOtWjBpEH6PfP3Ni0IRhEtt9nFlEp81Yy3MY1K8kQ+QAl6FCVzTMddYHZlw4zXD6e0lqCL8aHF6\\\\\\\/p6FjdW+RvAimQ7pdwFNFujzzs6k+Cl5VSuZf6+iIHelieh84Kmt1E1uNCR+VlUV3wbnEZqZPaXlAvi2HLibtw1AAT7uk4X0UtWHsFzK8MBwv5KiDW3ciFZtnej1ITYNmYlpQaozypPnMD+XEBMnh6EBELm5dsvcKXEy4Dk13sdMQSAsdzSxBTi2UVl4OuCljCyFsd94rybGlXqCltgFFf9Uq\\\\\\\/84vZuNuYKUe1IvK1yo\\\\\\\/+8kbHiqTZrr5BSf3QYU2DQOy7g5+6r7V1sHKOY7qSBvuuKCjO61pR+0hjHJpQIJaiAmGMoKqUmffOfH7iFLOzGvbZprRqTdbDwswEkq5lNBA8teUcccQyEYez0AwP\\\\\\\/\\\\\\\/Zz7Ux3j5nCzEA9FNnxxtcU3PBQBYoFC0q6hbd+UtJ2GpblxN1tFMv6sdlCpKl30lKHcO3VObXMVmoEmpN9rZdlg6HCEZmzGNVpW6n8BVaqyZwJK\\\\\\\/valPibsVVz2tNxj3BV9f6UP07G8ap7NLG6uLsFkqmT9OFH3lPkQNmnIRWi7AHuaG\\\\\\\/Q1Gm6hu3XaWPfznc5Li9VGF3T35XCPXRBGNJNPg07syG+whhfHlj5MCCen9XqRoCaaRYGIMXfzWB\\\\\\\/DAisQ8Myk5rfw1+fiD8bTGibGbu3iKtIn7YGoSyx5r9sL7OwSxfvm6ud05APsDmk+dvebx6F50QkjmpN7HTJTsRkzriJ+XOHvyZPT93WSqpnOJO+rSgcaXmaM3\\\\\\\/3JpgEJuxJNtjYN7BUvh\\\\\\\/o4POyKa4+1pEzUP6p0gR8tWNeKHsLRWRbuhFuBbY+oYuF6pJZ59Uf\\\\\\\/NZq1lWZa1KpHIN1UB4a+vIXzoKue0e+aGcgIiZ8UfhdZEfHwn0ctaMG4+zPUbv+\\\\\\\/C\\\\\\\/GgEpemhJIcVcfxu7YdWKKZ0K8OwJNUuo5AohOhrFL5CowewovtULLkkQhwgapMPhtsNLResdpv5JyzYd6R0W21f\\\\\\\/tKf5I\\\\\\\/FKkKc9HemcfAgC8OReg0KVwUqhi\\\\\\\/6zuHZsMD4O3+45mKOGM+LaaPwgVla4A20ns33UcVzNiIuAlqtkqBXhHWRxbN7Hw3cLV7J62x1ZDdK0IDtAihuAvufLqEjD3zXoxYPQH6Bqiu6cpXf\\\\\\\/+eF3e3kmc5WcD04Fvhlgaxf7MbjcDSz+WtVUs1ZLReSJ0pGi9FWh3QpNgrtNupIUSqslcj5kzx9EhJKcB4awR8aJa1todqmn83DJEZXvSdCG6637Q9ZX7gBrU\\\\\\\/SLYFv1JKha8JtjviExhK4lPEC+jhQgpscpVYc\\\\\\\/MbiHQH7tqgLP+Ny1vWVOkCsvCjvqOhhJZsBiPJwZQZFbbv+LXxwP3uh+4psAR0ABnAUhkYBkGlzfJ12utPZBH\\\\\\\/riQk1G8R1VnTYBFzDUVFJFLaYvH1r9kWR1j2XmfH1u2C\\\\\\\/2xFjcRsjpgOd47kXKpfmWgKqJMROYB28rLC68VoiFYuWs+43oIfIpJ77NA6WjlPnzbyrqFaYYfk+vmJP0guBnG29kUMHOg26\\\\\\\/2UKA97BpyQ14oaChS50yip5rTgFKhJIzwil1hzvEyuO59Po6ywHvpR\\\\\\\/D71an457Uyq8FpN4dkIDjkKwv5wnPZkVS6fN8YUo+synw8F7RivrGLQHMh3SeiA7C4vYGAI5ciLRcKdYWjXPljqlnBJc+hfPFEsMK+qix1ie\\\\\\\/FuFKRY9YJ4LixMZzt4DQuSsx9njGyFpDeAEbhjmP7GkJ5b4exG8ewKEJmnE7L2ZSWwkTppu4YgafLxwsBy\\\\\\\/BikWJ+iN+CrlFmaZ9VmW1az73CTQdcZ+8R5n2hSfrIwHB5cpWBeQ7uuoH2e9X5Ixjrarc2230SJyIfNMto01dR90TjoRWePcA8D2TAaz7rwPpy1VsE27U7Hr9MIHApNHsy5if8CvOOu\\\\\\\/7P20BmXgKb38VSbViClqq0CNGehSykkj5hoRBd4Ijmxub1fYOzdUhl5VR9iA7DJp+ctupihHPj6gay00Sq0ZyHdY7TWHaZidsOwsQmw0b9Hyznv+SpyiI1LalgBIaSUtp6+UN\\\\\\\/JrYvJ580j4DS8zhy6Vdm7lnw+XRnlKoSrs7+Bwn6zlQPInMOJB4EGAV+lnAORFk7PM2pzvsAruZuqkBP8spE4l6TGWKE44yz3HFxzCncU\\\\\\\/YXsgVF9i2SWNbPIGy8vwbrIgcxNNLxZTC4HjTroLs1VJdpyMkPCeY2TvSXOZCjTgFq67UTgrvv7ouaNu96Gpz3K\\\\\\\/4cowIkDJ2VlPZOcoNwh2\\\\\\\/jxsvzFd22axlNxtBuAklsw6iWLTKGpAeSYKD1BJ0admF+JnNS7ew3cyZIS9YL+Baclaaycf8dZMGDDev7kY9nE0KkmbWRiDSIYFLpVY4UFY56asPKVsOAgUd4aFdjI2dkf8yPZRIK6EWMVuysYvyCZkQmSgNVvtXQG7Yp7OlP59BlgoJmTnNhkZy0XgvDRd0apSgwcimNE4sYwAfqh1IG6ikuVb9dj1QQcHUgKtXCXjDMrA2Ck0JUtLzHByfm+dFry3s1PIcqosQthzUPYjX\\\\\\\/+OdsOoC3zhYEk8P5w1eugFkwhhi4W2L2mcZlZfzNqF9WI6yqrzZKhtqP50gM2xBOG49RZVccuWGK3p4PAinno78dnf77mnFJBoeKkDyqp\\\\\\\/FgEJDATJKJCsEk+Kg7u9iBm\\\\\\\/RIrEpmfzfTwN7NrSMH\\\\\\\/zFnSfmwmXVfp\\\\\\\/6RPsq71mBcMZOlurwuOZTZMgSLtHMjs2BxVrVSpMilGTlLR2Cd1d2yz6dpBX962zdRmv53Ni4JiNlcZLlq77RwE5AXaE5UBH5gU6X7vcIMmMzw8cmWcgathGMlKLiU4N6Kh\\\\\\\/UbhnDz7r0PkLbh87lKl5hhp1190BCsFFERgp53wDr8g7FdfbpjbwqqzwU7LMb8c3U2zWj3oj0YZosqIL21ISrO0Ia5WpQWoeXxF8TljzmLsEa2950\\\\\\\/CJ2i9\\\\\\\/O3qCnfPtgGqOgo4\\\\\\\/wYMOI93GzP5BQlcd\\\\\\\/m5A2GDhxtDL6Jz6dzJAJ4Bnczafb8Leko2TTgeF6HNYjwuwzenhXKu7k4PbgXiy\\\\\\\/dqSUfdbma8kgTjrk70oTfRvu13fc4b6Et6zdW3L9PxT9U0ssGKrIjHbq9\\\\\\\/\\\\\\\/BSdF\\\\\\\/3YfcDiyISkvuiZdc1lCq93xkP+t+rWso7WDTPu9l6CxlLN4x6AXos8d25aTClo0jwTqRvbhedwJfCz0vkc7s1+ampqpsAbOO023BFh4xI5hlBg9AOQkZ+G1xYkIMt1ZoCJPI9Pk5\\\\\\\/\\\\\\\/tw0XKJ87eT8Q7qxTSjajdwbzdEdJz0MqjOp5CdJ8G9\\\\\\\/ucUtK0HWoC8EdlJYVRekNwqIYmByvHoCzDKBcYAc0ULylwn8dIHBpge\\\\\\\/YBG1RlxSYk3bMZqiOlrCVt2IkXAzq5ygBzfdyiUQVIWbkY4sJJ+KpFAydmDIlpoiFwlNJtEwmTK4jkbDhxb\\\\\\\/So1xfNuqjz0edLui99yGjH4RDC3s2nSO46wUgTP7oYUNKe0OCh5u1Ooyp42VopWI2ZzJfnCD4+r2M4GP8FVt2fhbY2AfCjTyTqzY64vp9bgsAB86LEUzpjkqA31eYYEuqVXbagRAMYlsew9SQ\\\\\\\/pk4uSJPJfnsMBCYhrxSJZkLxFE3IQYKVwtk1rE9Wmlwnz6EA276rXKpGjXwI7Vs8NQZ1KOvWOO9xlTn3XFcAs1X1Fe4kE5vsW0LayQTL5nGG2YFdTeNJz6RmAiGtdqBquKo0VZamT18G7oNg2kEleFr6a+QYvJEK2VWTmRjNh\\\\\\\/yssuyqWKa\\\\\\\/ksmhhL0d3d58Vn3vNwdBtuQtPZqTs9cBxkBEtrYbC6cSH6UX+cu6WQMAu4D9ZlE75T4QgywqopbRXUxXPg31qvB5nLx+OrTEIZhp7rHO26aGIYQCG58RDIrL4y0r4uYnZ9DqulVzKhJb2TPrVV4drddhWRpM+9PPawr2NXf+6AaFBxjzMxj8JlRpJRjxzPIfhzsVziXeI3AZQyX4vdKF52mUpw+TEFZgZvpqWvbybHVhHeliu8seTJDe7HA3s\\\\\\\/X1aqf0zV0mqYiftmukVfR5NMzGE3okJLIRQYVTd\\\\\\\/A2Sx77dy8yQxRC\\\\\\\/lGe8Yw+B64YW7Q5fgPTR6y1BgrAf5UqOTD\\\\\\\/kf73+Wh4ka2CtFg7ZwrWyzw\\\\\\\/Je2JsDjrwwBOmCqItYlMkMXCe0XuxWpPeILELPyCfzPrdBFtSZhYINiSl59lpJdvn77HF\\\\\\\/90bMvGUDtYrO9jHo7ZUMyPeA8ntrSgZ8Yo+ATQkwvbSin3SApycPBoC2YeFJIm4m6rpEEXEdR2OtwDJUFT+HVEQpsCMY0pmF+lhxW5EhXo4YeXREodmpOdthkeBHxPqdJcKE4lhklRmdR25HlRvrE6RdnBbtqJoSKc3u1hbiTrtd7VaaF+X12GwkGACqYFl\\\\\\\/MR0sL+\\\\\\\/\\\\\\\/6lrqCpFzOeIaP0jXL44GTdylyR1BL7xwfcWLiST8OdT2d6OK7Maez7Zpa\\\\\\\/17fBrdU9YnKesXGniZ0Qq6BzI0XOtMDE+Ia79VGUMkimeNO6\\\\\\\/MIFaCnxnPh1oatu+XcCZrjrLtWNqnAWTyYPsgplMO6JeOrtJcFZ3mBbYKcDLXlUmbDvP8sz7E74ISqSjIWyyTK6zMOAHFCkkbFKbpsj8Q8zET8n23lg7\\\\\\\/bgTizs88BoTcgMp\\\\\\\/4Oz9tj6tbcZ3db6nIRmysRVO+SUGPJDtax7R9yCc1cym9EzS\\\\\\\/gLs9\\\\\\\/L8p0\\\\\\\/won04gx71WfIyt0895j44+ZEgVWDa2VrbQdeZYMrCpnVFbEVuKWkHjQmi7q6mUX9r5fs\\\\\\\/fDo+Sg1DeSZyuP4ggAjQkXrAqfvL9QdPkGmWCBdyhNyEcxKkO8hbv5nVMBBah34nMGzNfq0VlkA6ZgcyixwCe8LUi+qJCEF8\\\\\\\/V+SzxGGcbjv0iXSFk6r17qSjkvWKzpH6To8mrfUSIJTJUoDzNlYDbplmhi1BWd9g17S0qbF6s0ojTc51yJCqlG+QU3uccKpSaDca98LA4f62lQpWekJv+Hz4zyvSZ8SsabiH0e6Q8nnuPX2AkktyRkztsoUvVyCZkWp0WaherMtROdXUJCTMxwZHcz5tADHyMFpsT001yDI7nzXauYMW1z4UhFM5q+rQ0tnFM0G3jli1T6EhRHqgnmO8nCdOrM+nZA+PK\\\\\\\/OwrM1gr+9w62c4qUT\\\\\\\/x\\\\\\\/kl91uiMbhoZbTh0hIBGsxj6xpuTs4w+jPKeye4TNYl5OG9ODNO5Pbp7m0KqDaytCDz3um4PJWgDvKLnlQ2eRXZxmv7GTqEKxnmlvusqLJOQmb7G8UhMR3BL\\\\\\\/Lw7QCFQNZeF3R5NFXJtWL7kdPTOe5vUTPu2M\\\\\\\/5PrRCZuVGwMjcYy4THVK0FtfX1uX5cuAs+SswzyIN4d0yVYNjfBPAiRlsjTFwBCZvFDXERHBvI8hhHKpc41DoNDNFZzCa8VWZhMNTzBeyvTJCWbBfhhv7XRalg7d7+iU9+y2YKNYzzSv5QIngi9v70gSb0VpN9uKGb4Qg6Yk4CeA0Z+q1tOqHwOKALqQRq3iksrdSsh5MauwaiKlfkPza1CGCJkkfkLjO9c7qoruj5Y\\\\\\\/lBdMUC\\\\\\\/h\\\\\\\/Df2Sd5U+sb\\\\\\\/TYhbQ\\\\\\\/7mGXRcS1wVPpaSzmcq3GFe03+htL5F7t4cEbNmb6rJOi9ndyrCQpsVBmyNFJSxi+XXVTLZHjqXEvIej7VpL788jKZ92POkFpVlVETYOCvDyj69IAzj5KTG9I9QidU2esYt7jJDuSytCPKYfmjlDPL4q+EMBf3BgEBvu6jVVtu60ii+Jol8xNyDewlaNgB1Xyo4AOojSRXrtp1sXeRYBIvP66nC\\\\\\\/Mu1hoUhfuchAjFgLNmqWFtK4BcSYvywNq1eORF8wrNwCNI\\\\\\\/NLafHquoT1RsNL5NKS+80koI+uwLMzlaWM4mAEmdBdC9zjEuUSDuMnlVDBI\\\\\\\/FlWAu9iZs\\\\\\\/wBg0beLiWhqFWr1X5mfsuXsnuR8wMxGijnjUQ1eWIYXgznrzAYmGeBst8RgClJ0lI9HuhdVAFTr1gEJzqpXsZJ6DlE\\\\\\\/2ArMqjMpiIQO+cGJlO+tMnKE++AQuw5IGwelVb6WiZx3cXye58c8uSxBfUDnifNLeLLLkUJgIdqJM+q8iJH+xeEMalKNBazyAdqbV8Fmg1mlOkB3rQnZJJMnvuQLDs1WiSt8L4Aq6gr3qTQF\\\\\\\/z7uzyQiheTPZmuMouZfl0DdMnumdkjq4ZwYv2MhTeh8K9FzlOcqHycKanrLqyLDJvfx\\\\\\\/uYve19RfgsjHJ1med6lc2kxcumlGEUBrhz\\\\\\\/hAgykNbHHR6Tb5xSh7IRVUl0I1nrD3PI+ZhJX+69ebI\\\\\\\/SrX75TGAnxLR3q1fxMp9BO35oSp2Ar4\\\\\\\/9Y84ns6z7y9ZB+niIe81she4b8GWmntLkHn5voveKj46u5lJ7ZG8B6pWMs751bCdOQCqWf9Ala3eVwkrGgkXzXTe6ONFAZeH\\\\\\\/FSUAo4oeNBn55jNM+DvYylSBd+1\\\\\\\/TlwB44PZETeOzvAPqftaPfXErEVQhVoy0UxHSJT8+b0lKD4Ch+yPfxpLZgGLWbqqdz62IoCiiQ8C+g6MlDhbMyMjzJ0j+I1UDQsmMYlUf3Yvsq59nbCASFFmho6AwxT6qJb7jRiequtGBAgQN\\\\\\\/NQ+AbfP6nq3dwc5etgUlQHpGOrwCGpEv109sFcdW4fi7VMgH0fweCYETkjEmGqNiYfRVzojC0ylzQwyrbDYw02EPUEEDrcbEAp07c+gko0wWqoWqjeXzWT\\\\\\\/38xRvhwSwkftvghOQHxbrf8KIL+ERgeOfJ4RZBPmmpVkj6TxDU1PguR1kq9+DNFMruA70mE3Cq+KD\\\\\\\/jk2CyZ4p0IUPAFjpQsgSNEQ\\\\\\\/TVSeZ\\\\\\\/aWA5jt1gt0INU5E2Ey\\\\\\\/Tbb8HC2\\\\\\\/bxwDm8VMxiSZbgdb\\\\\\\/r7GuIIJS9emrkHyCtQW3DmtKt9Qgy6fLw4gvMREuxtUxk6yQgva7bYZLYvRf8Krw9q1fPc2KhCgFoY02Iwhwr0sOMjdWKvyi5v6UbfKCu7UoRPAFPxkXJtFxqj6VpbCPSDQ1lEOXJzvDjnfs8ALes+E3bBHKARh5ycQMx4BnSPpy50cRZp3iPmdsyJrDDODQ3fs0HgW+UgrSjTScRvMz6nNDSRuq5Ojp\\\\\\\/1T5CGT4mI2Nhpl5Vmesz7aj\\\\\\\/YD7wQySKxkuo\\\\\\\/+zKHX3Zz7UcQvyWtTBBO\\\\\\\/nNDvH3xhyGYpv3\\\\\\\/OCTPdUI2Z+nUc6qwBIOh0n1yU8jfhLpqQpEPYu0H8ZrA72Sevl5n3+je6Lm2B1065t5qs6nTRJ4yhSOvBO5EnW0bqqB+xCj68IRgdi3VZ5dRmib281CFJD\\\\\\\/CM8B\\\\\\\/l0hbyOLHTzxzwL3DjNl8r9Nf+LCLOHx\\\\\\\/Wn0S2U4q7pM2ndiOcoiI2PcCu2\\\\\\\/w+sCazGgQUPr5LrnWI7nAAv\\\\\\\/BpYhGvdsm\\\\\\\/OKfOmJym0MpeQc9q6H+1y41jifuvYmefgdu4i7NcVuJjGu8Qi\\\\\\\/HW4prDVz63ibfhhtNaklfj44TsdrQHcvaIVhQV5+I10sjzoPJe4H0+h3wfwdLjjEOnbz4t4VdXd8XffZ7KaBDG+LVb\\\\\\\/UTLThQ0JMpAv4axFJQ\\\\\\\/2dto6uK9rNqn\\\\\\\/kXxOuAVQq9+YAcmhjTXGIJnmq80brsGt9OskmjJdRZNXcQZecEf4xzypa93LzQNAbuWupdLKIRu+psWRkddaN+UNPp7biR2muQcymKLlGCPfcmvOKy6eN8DmbXn4naVbqf3bMyZrDnOUJCYlQvNsWLLtfVpgHSLV9wy6G+yTr7zwQczuWZdk8mWbweBpQkPE9MO9EJNnFCzeGSbOgtO1HVQ\\\\\\\/w+VMaR2DxOK3TproAzskyayIOmVfkgjd1+Uyx2Af2oFXVvCR\\\\\\\/bkOxpT2XwrBLgc+cSNMnPDkhNdMDiyUt8DP6bZ4k6LvMvOPFTg+GbNffrxnqdBaQSQtN0Uk3PQiYknrSl3jIhki0g9OrXJ362iU4+hdeJntd03VSmKgw\\\\\\\/zCtSrlxAbw5E3x0KZXVlP4po4+AbGJZrg316uY5eObGAUts1xylv0hQdoZIvADlSWEcmVtp6gXBmjC0ZO3BU\\\\\\\/7VIT2Stpx\\\\\\\/z3dxkHbHYU0fjYscrx8xN3sM\\\\\\\/HGoW2ANfqthzTEGWf\\\\\\\/UZylKvJXwpE9L16ZH0nB2zk38+Nd+N8Af03h7zPF9vXI68wtOn17ChoHaMYSHAySQpCsZvJaHH\\\\\\\/Fh0Dkd7+Ufx65SF6LpIaJpuS7lAGmu0c1sD9sQUwhRs3eVdK3ceBlEvIBMCcfy+WCwlcBboaVkzlB5+z\\\\\\\/AGt9T0m3tKHo6ERldUojdRhUf\\\\\\\/KSM8\\\\\\\/ZnU5p18EhsGwecScWcY4EZcPD21wL3OVH+0gGXn77xDzI+Ip6PeXUcZldUykGOWwbazn\\\\\\\/I2Y8H5wq3qGO3QXC1LIW3iPrgBK+i3pjLvp8LqaH064UdXDYn5kHdGgxotjSgVTlw9TJb6lsvvmzGGNd4c1xbo6GUDZSLKsmGO4enu1KsVa1cfwo1hyZxpCHnwbNzpE\\\\\\\/5a369s4q60pdfTGxnTjfeYOCfAwxk5oz4q\\\\\\\/EoqhPqs0vSwQUc0O5MLvp6MB2O4nOJ21\\\\\\\/Trw6wh1ePxZpIvZE1OsLC62mQskLWgJvjf+q0J4KjxVH8W3ZiXTEqOc5\\\\\\\/mTgmBXGQbLRVtq7R7CFx9t1nbY5shl2MSHTuuDO+3Ngrd8yHLF0jUn4VIM+IqrYqFkNw+ntEAOiC7sbo2gJh6HfYLw\\\\\\\/GLsmFvTOYycqBi41OLb5NmGKO0vsverzajGHaQdQfpw6rxPMc+FmEpaaepyUhmqahXbvQViy\\\\\\\/U6W51pEd+Et9uLVdaZBRvg1ehAI0w6oHXv\\\\\\\/hkpP4AOkMaX4P+e4VTUTCd8NDrYXUQVQj5IfD405rkPsNlp5ruF73MNKZlVipqDU4YDeMXt8SCItlVFO9HwmXW1GXSV1YV+o7v8ZPdBnkU\\\\\\\/c2E40b4aR3SQKUuJ+5E7tds14KEkmlLc1K0hoixwV77183gWCXg+T77wcZg3XsR53jsmakEptHxA9TZBnRqOU4eMni2YpVZ7mPEmVPDj1w4u3\\\\\\\/pMn\\\\\\\/ghbcUrQ+OfBwKASFQPEAhFumrd2i6uRIm2+dKOUhELSYIwnYKQdpa\\\\\\\/bn7jQyVfoUEaqP69fAQI+qgFi0xqgO+57Uo2p7DEJC5Ol0JaxLemQkbZQYp2hoxefhBFeedCg4ZbqLe3hhfWXSda\\\\\\\/oSNO0rar+\\\\\\\/S0cz9rGQdF3uZxkJVh7c28e6dn4qRY1SLuWkojkSp6DjeAAsj6gv7ZP0u4dF0IosUTJLkxzlrOjb+T6GjFOXkwEZxpKxBPQ7OsUUOJK+GvwXdHi301LHmsrplb6uCNLpC5Bf\\\\\\\/kBZhj4zhB2xKiK\\\\\\\/WoChP4P1eB0CrS\\\\\\\/LjzvNKZJNb6PysC6Y\\\\\\\/HqGPU9raSsmRiLcBhmnSPIiMoujnkYwZpaj2Xk+pA0MgeqU\\\\\\\/rAMwp+0NLNlO3tGoOYis0yyboegD7Go4kP6CukNeNxcRwchhBcJvDqXENc35qGXKJGJOJ5Nk9UVBNu94fB53CkakXAhU1U7LjarEvnTpmUvVsn8cRRtRsuQ6OHtWTqNdn0ranqGH+jXpmhiAXwni2U0fNYBO7FmwLk0OY\\\\\\\/IsIfMwzC0Hycm4CyQLCtosGWk6p0uS8itSEvLRfJQTMWRh8XQRsCqiDcWQkRk0A4M6iD\\\\\\\/UdR2lXkXPfsbX5ootrVcMt2wqti\\\\\\\/VviDckTQy7P9S6Hd3a4ZQsSuOQWYPJgEIIDPFDNGPKljDlyDiYr25GfHWPvsEWtm9FG004xHtgeUee49sqjQ9zG7HM5gG5sdkme3I7aHdz2DzYsPAyvsNUBnLp+\\\\\\\/7hAXLSfWuBD8EfUpaCS2tLaFjSzMTk8UL8ef+vgZ35VbZUNcooWZpGP+1d++FL8hoVhPh7psULa4mAjPDcTUWNkOVenE1H7V+6+UJMLq4H0Jirir8w3oyMXao2Ef6YiQXI5NH4ektF8xTiLH4S4X9pwWh\\\\\\\/fLspTQkMRbs8OaJyO1AShF2z8KOBLuefBOIKhXR98AiZOcpVjFpMhCcxSATtNo29ZrmLNS84ybfaoc77TWZdT+3V1hBGacJODqe7LL93RT3CKmZVOXBE2VvZOqt\\\\\\\/i+lQU\\\\\\\/VP4UlUnhYElMGRKKIugLyjgyzdYVc1KpjaUU9FP3a3f3QLSMBZp+JIT8gAIVU4d4CikNyuep3nYv9\\\\\\\/e+MOHtVnbJ1Xcz+TuMRJ\\\\\\\/DqN5brFrXpKXjo\\\\\\\/8uBdWKycJQJQkrWm21AdTVYKU1qNbXrR226fUNpaE9GghoMZea74yo4AIqogIO2d34iarUu2iJQIezWbK2VKLP9lXSKNcoqmWSS1YpMCNpX6BxkydXrZcnfAt3hRJizBPeU9GejjR0m1UJzodiXwQsoTVc1aSegczJxzkd63awiLobHjJI4WSVVd8oLdN2oseVd62nTu2G\\\\\\\/EDhHrHGmi5YfKoyNJCJqHIUpyg7bplq6BFjqp2NT1Kki+AB+\\\\\\\/tRasoIBxCs9dNyF+XAth5y2cnHjmsmps14w2PDDxyIZuzKCI2TeecfFPMX7smsJkVDsoUCv14yVWlDn3z5qG9MBn9TtSlw+AWsBsWaxIVR4pBXwwMVrHvdbRwlYrcQY24AjQCbyYFozPji\\\\\\\/ad9h+u6Rqe9LVPWn+tVju8DgRVmUDqP0NOmJ1wwdxO9UgUb8YGzRguPo51jiUcQdKqSvemGxd5QNu2BdG9gi\\\\\\\/O\\\\\\\/dcnUK2yT\\\\\\\/cyeREVCdiKU6g2Ky0Xu8VeAc7Y3sak1H7DeYODSjvcXC1VG0XGRP\\\\\\\/PmBzUBKjXDKxuz+MfzIpKsT2+abI3kuZeElYFHWD0a134uiwMuTCXsPYpEe0+BecFxWzpkNrdhunreUqnKd0d8mBtibX+xAwLi9HTV7kqSbcnZ5bn6p3kiDDN+T65wXxdbsALHsR1\\\\\\\/TE+OME7c7h1Xg8xJ3GhfQTSYEYNWfDetSS1vF9rwED9SVOgizW9tpwlpaAHn4wZB7pL\\\\\\\/UwC6fecwCHhAgsZ\\\\\\\/+NsjSqpJFA2wP8\\\\\\\/5ijErcnvw2Oi4wN9UkieWZy0KbPW3XJxvW5fzOfOEVfEDkMGWON8nbzwah4AVfEebFRhICBd3gNjTxOv2MC+rM5RXma+7yqTnFEs0EmftjkNHuaT\\\\\\\/SdQqZzafRv\\\\\\\/fUeZvHPcrvkz4RtjcU+B6kSC1qDVdBxYU+fjUQj8kzoQk\\\\\\\/Mb3MEAmUWxmjtFSsVJdWZzoXZ4O53lTM3t951leV6pE9EXDw4QiQbfQ7QOGH5bIjWgUwtqhTjziJ6MHEztBDiG0p3ZUkmN98ARjx4InBsvuLqSWxTQzssWW0vpFmascF29IikAx6IXcajOOC5SZrWbcGbX5HupSUSw6HKI7ao70F+z3sB7\\\\\\\/gINhYbRR1xsrFPG87BnXgLvcqyDg3JfTGKiGkLNTh9ZaZIvsycoEjywgdto3AxZODYhxuBUV3cw5uyXMeFzd9vGunU8IU4Abjll4XLEQzy5taFateGP8JWbpjJodfMeFqirwiUfTZAEVIpJWNPBr9ddO0bpZOiO6tcUsn08jjKXgRAooOUCvQrgYNtDRHliskxeVDfeD2kZJocBHmhl7GY8Ip1cjGlKElqLw554Ads+ehUdkSYQwt3G4N2TZ93uRajPjjh0FIDr7sv1ysI8XPSRevzfTYmsCdt0YZ3p11AqIbrOCIIixzf4Qai6rfJ0p7g5eFbfki6690hB2gfd2B+I3yk86M4drj0PpVSxP1UB\\\\\\\/EeNkoEGq7NJFnhE9xD5g+oYynl73Wc7bZEA\\\\\\\/samTuYCaZU5KXSaaPuBhSE3P5LovCBEdDcTFGIW9LqcdFbsAS17EF6KWS22zIIoAW4FizwMgf1KzRQMv8p7B69xJKt2I42kRlm9P273LbDvg6Dv2bBdSDRFnKVJJDG+EW3r38pJZy4Jny6lW\\\\\\\/bP7zcA2NRVeRdBLi6iBfoMA8Io5V8oolvp90vNN+9sehYIMfVKJrANSADQQE32kBrDWTpwsngQDqfDFhrjkqbzuN1wbfTMaFZ6aXCzBXRlklOQeHuitoUV8aoVgoNHd2o5a\\\\\\\/BBMHDN5rGnW\\\\\\\/WDsSoB079AJZdOAwLRjs\\\\\\\/4atcUhnuMn1wV5bCkvQEtnYUc3DJ4lG3TKTG9AFJuQNlKKdrmKKBniLKa1jMIB2PdgN2S851zl300BtiCJ8ojrO7mnHYlpRtp\\\\\\\/7rgk79bGxPMcxuxOVBWToJguS61gkfSIJJ85Rdjzpa8zTM7T15od0lL6Ib0\\\\\\\/gHDVWvfKSBopBtKkwtBR+I4F9kZVXERAiApb5QUuBHO0tbg+tZijHhstZi1usRTQ2eYA0qKB05PYviKgNrmHD32RaJCXWTiPqdCYx5RAefHjxL88ySuSqfe\\\\\\\/SLc3bqft85gwYOOEKsbMbHl6fAtqKLkGrlHCPTbFgzaIiuxyeD09mQh938rdMvPRPajuk6uAatFcXWyZ8M42iB1367ig7ew3EnwMvnIS7L\\\\\\\/xuKnLcWoKoPUOvT\\\\\\\/0gp5WXOBuTnJm\\\\\\\/CAdicAUskjgDNZPniZW7pZn3OqcQoDJAOoiLj8JLuxaMjuJf4l4Et+EvHIOzqlAnWSSBgTkQ9oNBzxrrhxzB6sCDJ9xg95yvyk0VX1o\\\\\\\/4eV5L9xUASgXb3o2mJ79Ytw5jYG6cAEsP86J1L06FtvLifZzvtt0h78l895pVqGa\\\\\\\/RhGbyS0dWB0MXTOuEgPSHZbGLf1P+s+bWOMkd5F3tuXhPro0Yv\\\\\\\/QbA0hQhSNE0GeB9+7EoVa\\\\\\\/Lc5\\\\\\\/UqYEhp8LMnNZTKXJG02DVOMMpBTTs8zlvD6qik\\\\\\\/OzfCTQ1XjZgG+ebbZ1ghdzDEHD7HWWYVqc6KfjUzMj7lWdx3f96TjqXmh2hojXyZIeVoKgCWu4eo42AmPv22jUWXOnl6+RDZP03oasoCVpj7hRD40nTpSGOoODVVDG6sSanbCCD8CgtCt7UL\\\\\\\/rSMCVkGiEzjtLq3lRLCoVgi6MhQOX7zQM\\\\\\\/GzsCjVf0M0wfUsHOAXFPGLbZB2u6RmUIaCfCE7NJad50LKrcxM90l8UkJqbPEwICN+pauPuqn25bmwbCRctChp5ynr81DjXCxBne0t+Bv0+CaI5v9BZe7l+boR54dKXPZzskqQI68QXlCxtKTr8XmYkpHPmwm8OsxS9rnZh2seETLDEfLyAM8Pcr7Tj1ssPzSnNLh94bYw1PrB0g+GxmFGigAO7\\\\\\\/CzcRv7vEH3bzc0HU9PSZ8v8h\\\\\\\/fUweRtgjGOLiVJjF6NDZpBwYgZldD0P+x7XSkgGYboJnJxKOzcT2aQdojjMnymg7fFmQudJdhuFzFdgv\\\\\\\/efED+7e7rCI0VGZGYhgdCiNYSkc6SEWj6C5epEZZah3VuPM4osvlX\\\\\\\/V7RG5BMw1GpyOJuE55hwDGuvXKINFfVusDwvTAgirq9maubz0rMtmhDBImiZYzzvNDbPt5ud\\\\\\\/p3dwlOgfiB59BxVSch3CKdPeOvKAEWHKJpSgzlsInQc0LJ9WWDH1MwGoeHnGjLoYfAajupkgkuJnsovQEtOssRbicf2k67MVIN5Afn4ow1I49mzIUuRWMk9J7s18DEwZL7NEJX7c9\\\\\\\/1Nmdco1WQ\\\\\\\/qDK6D0QhLQ4ozmNxZ+3YtDTgfcZiCSCg5bjceFhyHj+mAzJpwqs3FHNnWaXQPTeRcaFnSPoq43fhFO5UF1C5Jy9qgNqFaJX3SiDSQokZn9jf+MyEKuEGC\\\\\\\/xeAiWAYIfxVXAmUgIQAol2wV1FKoU\\\\\\\/+T6RWUu4VWY95JKyfd+PimP8v5YwZTkOzNuOgJxlhst0D2rsAs9o4vy4tY6CbkRMW7Il8wbar\\\\\\\/dkskb3Ot\\\\\\\/2wvW2NZfYRKdVy20p6ckrymtNAfc8YT\\\\\\\/R98FZw4xzdCMQYqkHFfXltY5vEilbqfZuc2goi37YGNU7uHO69\\\\\\\/usqsjmJguqwR58\\\\\\\/j3DxNir+NLhHTVYoIKHQVWMi2rm3bDeEwyMQfmPFHH1Wivbcb4Y836p812ZOOQ65jlKtZ6ItFWXJ23LQPaFGYxf6bo4rqEuXX8kNRfREhVmDO6cwQk6tRmH77svUt4v\\\\\\\/SF4OCYh7fxq0yb4WfFULTIXhCZACLF3GG2l2ubFLitgLTpWQYIcP3MGNUutLVC3ipJBNxXbf\\\\\\\/mHVkuJVc44Gz7COXckWi+f8XAZ3qlY9v29U44fIWI3KCDU1oFxXiddM+hLGAOF5mxFqJ2Li4ZYOLaNJfdG7XJ8\\\\\\\/3y3FId3lLFw0uJXACkCQiCD8lcox5DhOfCEjqu\\\\\\\/TU2gVDpmhXmy7B\\\\\\\/QC0jcHCGrUKqJ5imiZWICPWCU2\\\\\\\/Z1FwhqFn4vx3m4aAzw6jSlDqBYuJFZSSWUwPvJ5fm6luZ9wHkvraOFX4zeCsPsyM39CjUqNrFG3E4xoWNEcMcrKT+OsljyXZRo4+yz3imeJtEPC\\\\\\\/iaV6BVHNxVFnHz7+XO9EEQqQ7+lqshXEp+42OfmscVZe2KggkVj+BQN\\\\\\\/cZZMEHg9gwuIcCILkpMWLaOXE6qNgP1YJKFs00+KX1mVLz9H3U8NqrnkT605PokLag0xqrPNOeeM70ZRgwBSWKKw8M228D1by86Rfg01JvdA6\\\\\\\/iWSjeV0+\\\\\\\/i8z35+wZwgPVYuswL1CQpn3v9kvfd1DC+qUG3mTo9WLTHF7xpnoKiDpJixWxG9Newj+BgLOB8M6i0aTHqWYvumWKK2CV0HYd4DG36\\\\\\\/7mN2tSuc+hsGw1jkM1o6r+QrSSl\\\\\\\/pzr5AQvnLUK+06QNtkVbs4J9XaVOTJJb3Y+T8s78fg3QbdxkfC\\\\\\\/62ZHS25Tgiabho2lqORbbiPLnLmReuY7Lr6uSyi4AG+2UlyLUGe641u7T5Y7REqfXVkBtjCTLhssBVonzQCnr9x8xg2WOey1qgYcM6OlCiZl7Cl9rBqIDCyDlOUTJkgeIVsbEnyL1GLjPlmlP6cCdN5nDTaJBUdJmPIbho1YxTn4N5SPxeYl9AwH89gSfuE51IT+H0pA5oUbGtLnxNNlUmuJZNMMEF5qA\\\\\\\/+MpinnMkXzmPjx3nhxTZ0rO5okcFQW9FwwhJXQsIXrR7QPUmyVymZiMl\\\\\\\/iLTUlFT1QNfzl1ufkpPOW+3cuAqBrTmgltPZvLgzQA5m7dG17DqERescN\\\\\\\/o\\\\\\\/5+KZFs7vH+sLNW7y\\\\\\\/e\\\\\\\/QnKezXY+cbnGqMsD2FhwJbMt715isUKnCA3qlFAMy5ll+lUBxkI\\\\\\\/gFlwWUh0eahwhnjJzpaUCbcLLMA7X6+ZiD6na2ax\\\\\\\/6krCGvrWkzv8LkJdPIH2BwHP0fBi8xCTZOijh0BsXX53JwoWyeImhCxP8fdsbp4HZpgwMGwHt0W9hryTskWZ6bdCrYTKf2j7Qm2vd\\\\\\\/gLUeV6y+ruv8IdgSgC2yXBX1roVZ+xgBr2zpIi9GyYnC8e1k4rOqy4X3vXkDIM4Ou5Ol+gUWtVwhdBLOIUvuCa91oQjGnDpUsi2ikE\\\\\\\/hykqv+54LmjYihwhfEO6e4vcEpBDc5tKSK5e8EwbIsZNSH3Q4mT4h8CMYW\\\\\\\/f87K7\\\\\\\/3cU0PpiT+MJ8FVsAfXqfNbPXGTIf5\\\\\\\/\\\\\\\/rtjIoQwHtlx5NjR9onVH6wA+lGz0FA+\\\\\\\/jJtSeMpDhu+Mzs2qe0mPDxjO4+Zu9TKmdPz7qz8baW\\\\\\\/ekk78doBqBRP4HKKU\\\\\\\/Oywdf0vtMqMiBXe3dV1FMGsfDGmiSrv8eyhzFAVx0hrKy4rqWauIL0Ka11wKJD2qFiaI7uYKZX4TKjoRNuFUpLPHI+USxTq+PR+dbuJt3peqrMDAsjVpXoiacDTxFWoV3rc21wPjGldWlMpku\\\\\\\/NUeB0UCOtBL6Zybk4nIHb3HlL21s\\\\\\\/xerpS\\\\\\\/Z5VPUnho88PrnV6SDa+ikphlPOUfImxhgCN2xFYd\\\\\\\/OqnrdDhTba5+pBqlqjciYzsdoYVfivUvOU9GV6IMk8mPpdPxeKvUjwzxel5UdlD9zk6YWTgk\\\\\\\/mG3o4V8jBpQKj9Mz2G2+pW4zMah5UuyhLXdp9tHJ40epDZ6WUqPPlOg+eN1PTRI11zJGf16JzusEuybmLGptSgST7qXxK5f7w3ncrtXbnwgcH57MRQaDQgsO\\\\\\\/kTb\\\\\\\/xGGwSbuiLgnTZA3wxLsnayXw9BfHCMSKFrpX2HWbnmqDx0ROKL8hJuxfuPI1ILuzuaTmuhB8RQoFnaHcW7kixtgnlO1USI7JjNoZhtNraePtap9ZtYJjOag\\\\\\\/B25KSZqpsJr5jK48+N7VxhXRb1YLy6o+0haZ9Gd+eOlvplxsW4zBoFN71Fmj4YgYKgxZRWNQEPhOW7eSzJwDSe4wYSB7oiYA4lFit2hcjObnzxk6TejhGgYX4Xa83H1GRstNa9czTW3UhVl7I3a7rW6nf4sbDA1ImVtKAOw+Ckp4NkKyOpI6k1vpWljFhM4Couz5IOmK0oBhVhs5Q0Vh5TAKnKAQzbU6y1XxmV4mPG9ogc4A1hh1BnjkbZtTqm2dzq9A2ZV1nQQxLQlgGRkuuxvd4OjHspcwwD8JIV9iTAilnH44i7e0ZPRdeaJSQnE9ZqjkaUPRVMIAUkTaPE1xyZId7c4aQjDiLstHLISfzY3kKCpwHWKCmAjCSl8PMu4EMiUAPNeNVBrVPyRijdsBMl9cJjogYMy\\\\\\\/P6VjtRWFjv3m9w+9Lu28E5SU5DRiKuX3pMBeWJpnSODGXmAh80+MGT9ls0rEZH5fWcflrTY8TCLBfeeKNTLdw9tNVeBK6zuYMZzTbDAOd0CPffr7NOl4o9hjnEbJ3IA6FlMdDilNKR0YCmaU\\\\\\\/PwOSe++DnLDiF1rBL8o89VpiApZ0fW2XZD0ekPRJq\\\\\\\/UkjR7iEpwIEaoIXYEwnM1dB51tw4rUXzKXg6xgzJEfqRZ2xWlOJDJ6GZ8q6gnSlthJFcsD1dweFfor0ZpAG2yhTDsJwfZJZFN80Htva+8pWwu6mlA53eSGLSLbHT09l2Wpsnw9S2hYEhLKHcG3xRTljCfipaqMpSs+AkYqMwC1dUjeig5LqtXumlXFmPNslEcIIwQmQ1rS6uTgyLtrs6tMufXXoHSEKBvAWUrHqWBYJuUdFs+\\\\\\\/DvbFVYUIr8r394r1tDIl0CMtsTCf+YlVLZ+\\\\\\\/ACZCQrMx325dJU0qjRORiaIt2AGzBx5gbDxBXJC8LkDGYaj0FvgwuCnBZyuKLxorNWvXVYmpa24hD5iKqz8KwHTwXXdtwgIH0Jy9ydwNQWJHpFHAS4Qrp8zzyJG\\\\\\\/dBqdubZtlSnH66JiMVuN4TV+4C2QjbChKrvqVqJm0pI4yvV4jkt2WNifUZnq0vJsHlroJK0eApmDvGeNBZu3NWKnJzK8OtpXswu3IAxYQvxDaV3q8ixaU9oD5wJXk9b+4l00XWkYFoaenDJNBTCEZNY0QkCiuvbJE3fY+3F3BRXSdeDQzIGk8juUHUApi+TnoV9r8RsTad3WBcnakMEydPFMwrWldePC+9JKPSmRNh3Diy6+g\\\\\\\/WVfHBVWMw8bHhh5gJmb+hWDVxu2kM5fgkK5rLuAqJYSI\\\\\\\/t77+2Wn395GCXmkGqt0Q5N0YBorc6gkz0eezIy4QZAfTLmXWPA6WhDtzUS4iPsbnhDWa93rKdXseFGUVfrlruADCTrCcwl0g2xTX\\\\\\\/3+7pZTgPP9XoJRTBlTfGCAd9Q6qUz\\\\\\\/BcAMjhnVJDBVXq\\\\\\\/htNxB7JBywv\\\\\\\/9De\\\\\\\/\\\\\\\/bLBDEwjbmb5jGr5DOeyNPFKKpDFp0sEFDgUYM2eug8bAa3Nv9bmSlwnnkk7qMujxtpTahVgsjTQRCC+z6E2VOHntVpRDLpb0b6X6xGDcdOPVgCGzvZ8nTGMcJaKGFXplzFKYDmWPL5EzZPSE5ni5Ghze6XmuKznJhuw4opMX6rhn3HfTbXmkJzzkjK6SmpMFJ5\\\\\\\/VKoxW0tijBDQgUbu4u\\\\\\\/+N3P5LYpDFiTCp2nimm2remAj5LCzyCIsX9nomiioXGfmLDL7NpwIeZ7URZZpkaIQ\\\\\\\/jr6FLwDD4KyailkGAFrTiC1fRi7J0wKhnVmySfzj0a+V49Vq9gUF1EpQA6nrOOwo\\\\\\\/NVHZOneQpiJhXKn4Nkx1c2V0VOzdbUS7\\\\\\\/TIPwZMhZ9HQCvL0j8+FAjbl4j6fcCew3Ip5nLu70pBEJ2jnwbtDCa31DFnzCftFbPpz788aRZsn6sJJUNmrX11Lu5v53r6Tcfh2TkFcEUAcMSQK5\\\\\\\/DrSOA0f+NvfIdIk+4\\\\\\\/WQsmSQwFVBk1+IKOvkVa9QmKrzBNgCpvaHsPgC3SJi5tRRfmhtveyqmooHmam3hwLIGtRDkKhLsK1xVbzaS4TqW7iLIglfm\\\\\\\/a2kZpX+LQ0HtM9j7VMUAjnDSx97wtyFC1XnbhcEftjP17qAu7nTrtk\\\\\\\/+vqQdYBabR41MEMr9mBS66B05CedPb\\\\\\\/unU61N+3nCDDT6X9Nuq6lMpE3eKkVfuvSsoPkkHxV4F1QuW+u7LkBJMSWHFAMnuJjyBiC4zoqnsYOThJoMf7S39s3XADyq1Nx3\\\\\\\/uQ4wJIYUgSODSM6tDd\\\\\\\/69Bw6Lq4wcQ2JnU+i\\\\\\\/trmKVeDKb3U8Un61AR6bISlUKAZJ1ELMgswiA5CvzWrKv2+9t\\\\\\\/6jgqHU9trTBAGRabpXUsb1zEWj1nJ9PWqwbYnqGMZaLhCMzvovTRW2UPOwXw9xKwNvN0j\\\\\\\/m4Sn1T9xRh17uqjutSQuwNx8TLu5+mQV\\\\\\\/oONRfdOLXQ4m\\\\\\\/QEnXfzxjKA7sV6XtaUR4KiSOdZuHkG6Q2\\\\\\\/yTUEzZikyADOaanwnqT6Grcmajo\\\\\\\/Vaj1nJeFZfRd++F70s7A62ImOkTEYzO0r3eVxAILvHBBvsF5CW8KZznJYlyrta89nThbvenZVRcu9DHxVRk5+D\\\\\\\/hPWjkti6TaWXJybOPSTBw8E5nZ6pb6sdrz+o+PY9kSid9TW+zDZoNDYdhOXaPzb9Gbyv96vVDNSDlVc7KLpfxU3Dzpa4lbQbJnmJ5P+TlqL\\\\\\\/gtAvcvO8oFCYgU1sHQ7WdGtC0i4XE7kxkqJzsW7YAN4z9h4cBv3B0WPskWxeoJFXUiz6YK\\\\\\\/hZtMMgn98NiBg+m223AuMz\\\\\\\/fq0s0sP6ZDeMvCAtenxsZd+E7oCkBmuUrREmOXVyWxBinl7wvgf90HEyXcPiro6SKcrQEze+dn1Cqp56Y\\\\\\\/Juy3iueBi58q1s8KoLRO1jrMuu1+u5eU6sEY3nrRqpSJemUba7s+K\\\\\\\/taep1u38zWjQsvkLib41E6Fxwwk5cV0xjmB4z0onPgoKhMcQLJPhvA8P0je5anzE5zLVc50QjAEXiQsaaZn7VXKjTrurc6WFcU5luW3ilJzZc67r+7PuXx5ai9KOhU05e6OWSZfkvpH4FJKCCtyTcM7VNXNq77TOd6JoRkCbh1z\\\\\\\/+5K7k4EwnttjkeJqG98zvelZJicgS5mYzIFOJlLhXBgOAfoswjhGhZIjnlfN2iO58Z6B36VTHb14DKKOcqRuutJjcHXidWPtZ5pmwW6Xh0W8kJoi1rez+Ed4EoSEs4fo7mBOanuZAIHA4oNX+RW2ekdYMA9ng9hgLol5IUMwYc7SqjS3vITgQG\\\\\\\/evKAvLTyaPvfCwLC4+JMP9DDEPWeecaKvfFJbePztdBK1vI8x1f8fxrl1f1K5AdA3E3SGLWXVqqWknxI\\\\\\\/DYjOuKoASDGmoVX9jRyXCKEPjcp1zVprqzZwizTrP3MsNzTRgz5p8AtKajJP+7P5sMq\\\\\\\/skE5ZRgd7X7tVS\\\\\\\/CXEV8+slLfOY6jC3Sb6me1reXrnio1YunksisKurHhrN8cdTeGqw+fE78n857Okz5YFFRannBYxPg5qFrySTrZouOpyxXxb\\\\\\\/Lbf\\\\\\\/zUeqzY0szX6ojTAnTr6QRHL81QtxQVyjcBuBajRj4oiRNpaE3Y41mZk+HpuHyd7GId5TopA\\\\\\\/wn8agS+5VEbMrwwISjJS5VHAXTRSSyAXskHa0pRreU8omlkszHceXkfUcqLh0s8pVl+Qx1Bc1kxEWdbf8mEPZ8LuKcubIskagwIKnAkorXhrE58HxLMmz2o8F0FaAFa0zSv0a1ZdcudxWK3eydff8K1\\\\\\\/OBNZBhXPGmgMVgBHeZcSSxffuiVcgCZUkbVnx5QJUGv6rj\\\\\\\/nVLy5gcTyTI2nMn48fh4SC7xo9BUfRxkHR55lPUv7LbFkdQ2ombYS4RC280XeHAgfi8kZ8pA3PuOoykfTVTozHHs3zv4yFjmaLKFYwl9aDCQX+pL63l4qS5735657wClTSuTwb3km0XKdEU8jf7alIXNPcAjToVCj60BjbM28MIpO5W1Pa7c5MOjXvquMNgW5IUimedU4IG\\\\\\\/byefaaRq\\\\\\\/cgeedJJ\\\\\\\/\\\\\\\/X4EkNC1zKswGsxfVwKbeb9OjdpuxmVo6z2u6+tRe\\\\\\\/yOayNLN\\\\\\\/hqVf\\\\\\\/8+oY\\\\\\\/K+VQPdheFV0PAoheJbezsypi5lJhk9Erjce8wY3eg5QDh1C2xBh48lq+jIdphidqCLdvyx5wFJYgaZuXmXK0B2gYRnXir4rxmMza9ILQCEprFrcwkOxR3yNWqcsVN1ssHQ\\\\\\\/lI5g5wz6QGY7KxwvJjeSdZCcADC7eboVvjccu73vsqkYuEM\\\\\\\/HB9LeI6M\\\\\\\/cVL7VHPV35dMIzw74yikWQdqs46ku06B9P3ZzNO5gJGL\\\\\\\/A+axXJIEpJy3vLx6UKeFIFfcl87h5SY\\\\\\\/ZhZ+1kO7gFXQzNRY2s+UtvW6WyNLWa1pXI\\\\\\\/3pe7hmyxDK1ZFMdAMD7xgXy0jKb6rpZYaqJuJj0Mro3CF749t1+DPIquZob4TpiQItymmZWgV39QLQFl0HP2+jwxX6OQtU7TkoAnVRN32lsub6AFsLHQV7oE\\\\\\\/xtlS6eEd7pn23qSCecEkIe8rPCyXW7dHzjdZFbF1lSdxVzrRpsx9g4askV5ebc3GbU\\\\\\\/BDSM1dmBQDxwx\\\\\\\/a84QAQKWL0+hpgmkvz96uoJJlev7xPtsFqybjvZ\\\\\\\/NlYzrtSgPZjeVPl0JRcCR5QUKFFfaU1SZCZR\\\\\\\/IN1Qn4\\\\\\\/sHtGXeFnndT+7MquAFazwmOIKr6\\\\\\\/m7+zzzbGx5po+Opf1gCBCS0Wvhvy8fEzQKWihDDQUptSmSwO0w8gGFJe9gntzkTRA6LL\\\\\\\/ukuK0sBbbM6jBtBxv6gl5N\\\\\\\/HnvDnPliweSnQVeGSAPPFImVBVn0WBQhsHWjrUityIXuQqaDjMZPDx3I8Np\\\\\\\/1zJ1cQirlgre\\\\\\\/WT8qpcu\\\\\\\/p7J0Wf8BTi2+YzyJVP8nqAAxX1Q5aYOwEnM9sAD\\\\\\\/LVSg4l0g+875yWtlOKd5gZF4C+I1bQgdwUshPFbouw0DEiDcyogbU1venwxAL4XYk64vZKakDdAZBpXW4R9E0oLver9Dk2zuTRqNbpnvTYvEMJ39ss8CjwYoBJsOOD\\\\\\\/jynETBscAteivlPlXs9GzJ+nIvs18GnM2xey1EZgntPwdw9qBtjfChSKxHn4J\\\\\\\/6r4TakgJIsGrGAjIvinLLOZftfFmVCeh3OsLp8dNjleIHMfXEtBR0L8RiE4r3glLAz7inptTrj0\\\\\\\/ZckvdzAhpNsukQ2sB9I1TZuQTZSMI4pOXISlkTVroVs8evSL3ByfsLJvlfDL0TyUYF5ARMZm2hUn5uxVwNZYc1xb78bItBKBIV7g0yiVs15wW1qGVha3sobh13ShGMeBvy63aj85wp74q9my7AMLsZEWnoGUaKyzeGNp2wEt9vFU8Tf0Cu79+IycHfIkktKxN24l+EeTN6qeWsPMmVFHajZENxWRws\\\\\\\/gtkVfHF68FSUOY4wqNgrGtcv30xAB7HqsNN2ggvJI583a4nWqegRfE7BTrFbdKEdyhHXpREpGmspep07c5FumxQI+27zuCVlr4CYPCIM8GYyNpG\\\\\\\/q8F+7VDZSuO64DPPuUUpH0iYjN+IOiiSkJhJeSLWAp4H0zqwkqVKCG9WH8tUnflMuk+e3sPL6X\\\\\\\/zKckYfEfpYIVR04cuFG2aMuRoj10JmF4iE71tZVpA2K9kYAROuNqE\\\\\\\/ecZopQdx8t\\\\\\\/iey2xQpw77W5kn1\\\\\\\/U5PaEsxkN4DnhgM7Y4e5tq1H3Z+oee\\\\\\\/lYzjqglZHjsMoc7xQE5SBk4B7gGQiplHNIkVu0drIGFAKyLIM4G1z3R4UZ9c1pf4kNqfJcFx8qfs+Y1M+gysOX57fHJXWZ6npx+dlfc6kAmSB0eC2lHQz+PHHT3TNJBHCld18nXYUEstUITVpsKVFkD8V7e3S\\\\\\\/zumI+AlE+h8OieROs4ePpJTQygYGUEV+TtBTH6AFC6U2lXmPhn1fqDuXqy9CabP32LSJeIdbZih6BFPuHEMDEhr2vSBdHV6ObJuXoiFlqFpiGaMXRKK+Xebevhb1pXCw59Il8bBgeQsMn2PVO9FpsgP5z5medMGXO0E3R4nOlw5FA4h0+rzW+swyC\\\\\\\/0dSJ4DVzSX2evLY9FEnvspdrHL5pxIuqXpEB9pMEpiy2cldH0CqaRdji772h\\\\\\\/jwmS\\\\\\\/rJ\\\\\\\/LOXTTy1uZopAvXPZ95aPxGiXxIdhrX5B6jBS1I0YI47aqM5KFWXzqQwK1gp458+5vvLtb0WMU+BaAFv95vUilYNPlcgpP4yISfsGtQFMoLgnLtGXRLf2okYhkFzqpowSUFMFh2lTnS5yd9KVQP8D8SZABvJvs62Et\\\\\\\/ChL1ZbKVOpTVTlYOiwkMukF4A0W\\\\\\\/dBnIeiLbIbUaut954HumVH\\\\\\\/RsIsjmUybVHscqoyBbwreZf62X6y1PzOsJ8CgzVpe+b1rKTfIu5\\\\\\\/ZR7vlV9EJHovJ5YGYxzlGIuhXdoHO2iulA7PNhSwTk2gXt4f9jbbAbyXxR3L\\\\\\\/XuWxF652fgmvMy3VrcA4RukDiVgXBOOweU8G6DOfBqp\\\\\\\/bLUBvNUSqda2gcPhmvV93T6EGT\\\\\\\/7SXji6QSZ9PINwGTMAsUzjploqlsaavu+1znY8BLZWmXNfsZ7\\\\\\\/zFTu3NnqJd3I+2GHongmbFvh6D7bvUK5GS0uvyc804ZzvKPSGMpSNRyq4P5avXPpqnsN8c1WPphEw1iabvBc0mX9eikrXGtI6M2Cwzni8lfPrKrvz6+egZMm+iSNsQ43y7zz6h6EpldZvW0blmIJkC8mvbYAz8DmS30Vu0ljoqdK1HhdhDPo4QTyaPCQDd08ik8QsWWI1vEgSN6NwerAV373IlGji3pR548ikUFOvpbOlyNuhsvIgvKq2wTzIdpEjr4k\\\\\\\/4H30mIx59rL75aOAp7XQ2AtWea+cG9W9S6sq7qixYIeBiOnJ4MMFoWeBVEL1aT\\\\\\\/48rS4iJyu6S89LQiH3+vpNAE1KgWvTGN8mAFiaC5RODWL1xBA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022dNrcqOQppDmz+La3GemWvg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ec311686cc5c3d7ffabbe2a43cf224743949cabf642ab530a86ceb065f48dde8\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Sw7u+eYZjbZA2ym8dYCerQ\\\\\\\/9Rx81bYUsEMsFuhpH\\\\\\\/TbhZQ76rhRgUyfZ5nAnoypqWl2fHgB3pvtRtZgu1IDKmF8tpBSAc3vv24k2eeEQAE3eP5NRAVrk+d0g8KA61UXa2aNSl6a6xvrquw3R670oRGTRnGCVKx5MPDWw1vV3H1+EJTbwSEdZgzrN1wpLQc1hlZyzexROyI\\\\\\\/WOEYo\\\\\\\/PBQOteEvJvhcXVOEDpHnMuH89ng8KrUAwVWd6xD9eb7ovs1FdWCpJ\\\\\\\/v6wmGfL9U3X3Amt9KTdHM3ZipcrFNC8yeu5aTzoIcVF5iPCOY5QBoEuY0kjt65AKBRPweUleWYFC7EMbKgdJzYb3\\\\\\\/EK+Wcsu\\\\\\\/6NrQuwp0H8Sv3kJTEDB4va3L6Woe9arzJLcdvx8FOPazYdxFa496lrREo0z7y0S427lnXB3sHFI5Xnx3Htx\\\\\\\/iaKSfKwppIrXSEoOEYdA23TIGRnczSMxkR3BC5GS314NWekcAh2MWTVQMKX+dTkVJEXmET055so4BBCVC+Iw2dXmxsg+bxebDHDEXx7yPpiIRiZ\\\\\\\/IUlXp2RiBxlPx1m89YJ0EG5fCsw\\\\\\\/zV\\\\\\\/89qjEQYNKjD34amCnAXbaXLMGL3cez8xrxRcRbxxdP3mTbtdT+NOe0l0p+PmmjDfqg5Lla0FhLw3GJ8s7Fmg9FTSNaKZLd+a6oVM+6K\\\\\\\/1zPGc3UJVTl86jPtcZ6sMxW1zqN+AXmtae3hbFN3hYbhhm9m1Ng519fA47k1MxZzVNZ5P3y8t9zp2Luqw8D4GYExgUE1fcSS0cHTNvY6pjTnUNOppow+26X6SSNfS06R96hBYFv\\\\\\\/dM6j+\\\\\\\/VgcJhY40+meLUqmwGO3vLgyDQstFbVL7okbtOQGoSIrXDtrCYgFjPKR5FlbWi4bgrZ9HF+qqx\\\\\\\/KbNTTdVyhiIz9EZM+8kVZ7FuuEOoEg8ncN2Vx3DGPz2y24FDXs+QYJA\\\\\\\/GTucVyjgQLfZqF73WELpRols4kI266p7WrFOGI2p6CT426DonZpngCnyR0AZ46XFYjG9v1mpHMFreaPADBg48sNygkj5tKSMS1iLUvoToldH1yRE0OKwSA1YIZEGq+Oy6AgQF3bN93wHAd5lN4b5FNn6M+k71nva+lR6JQohxcmxO+qcR4290Xbn3pWhgF+4B+506ayGJzHsMHyDKQ5MlnsUo\\\\\\\/IvuVRqNSS32rIKP4DwGXNZS+W73bAuuVtRBgzhXQJhNvrREmgRFCJpj4zSJTIU97H4hDo5eRqpBumjuNWSCv2MQ5m0Nl9dfHusv2Wy02GApYixXeLfwKf8cHYGphAwdVFZG5woJ+sm4GeZunEfbCGy26X4kjsGTpbl3DLYByTObwLcfWfPMjxFH\\\\\\\/kw1pjqGl6cVJUf+2jatHzJB+V6Uc3s7vpNUrfHNj4dvRcwxYyD5DniaeTSbgIW0e0cIkeXdqBwqlkMlS\\\\\\\/6zrcht7al1ANjEEv0aKCBJ\\\\\\\/GhqI+q1iOjX8cMEDicToAv6AdExlBX65tAuutE0K9U7jX3ScV0zlMKQIwJdQDZZSRK2577GAU6uflS6WQy9FrP4nQHWsaefCG3vprRQahlE49n5JLzimxlF4\\\\\\\/GQNEU+7QsYOHAqiLiUdQcmG26kdF\\\\\\\/r5oPehp1wcgjzoiIqTvUYdAzVyN9lY1a3J7ckfx53P051rDyR05cCHbxoeLT3chsvrmbmLbyXkZu0Dij57KiDUs91+KldydEpuAl\\\\\\\/s+KqrXDenab1SfRd6xrBgqH1o35V0\\\\\\\/fmLHYQxCn7TtMf2qurE0vrb9gqADIm2vWLcUbwwyQiK+1pXHGH8zigYNcxTh5UqBMbFNxuLqIoWTOOwIz5xdA6ObRbrB1SYshNWR+3DQVISqBNea1qX\\\\\\\/oCnMdZ6Xb3EfhX9hJLHcR8UoYaCINSCUXbuGCrfWlKNFfJ1jgQA+fu9A0+Y7r9MPLcIWri1rr1Txq9\\\\\\\/DJGXdvHtwqbGPBLJKUg5UDq6FYu2pf7mrCFx28vbsDdG850cEpGGe0BBZvwww==\\\u0022,\\\u0022iv\\\u0022:\\\u002254W6sH3vo+hSYs5aW9GFgg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022343a9e9e804e21c616718824b983184abf7221a3f3d638333137d229d3defe5e\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459302 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/7/803" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-JQro-_O3glqie4GoUy4d-mvknugl5CuT51sgxBj_CSs" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/7/803" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CT2Q-IwbIq6GHkhYrAcTqpCoixPq8cqy7x6ogrGJVyA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/7/789" title="Comparison of Parallel High-Throughput RNA Sequencing Between Knockout of TDP-43 and Its Overexpression Reveals Primarily Nonreciprocal and Nonoverlapping Gene Expression Changes in the Central Nervous System of Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/7/815" title="A Sequence-Anchored Linkage Map of the Plant–Parasitic Nematode Meloidogyne hapla Reveals Exceptionally High Genome-Wide Recombination" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459302" id="node459302" data-pisa="ggg;2/7/803" data-pisa-master="ggg;2/7/803" data-apath="/ggg/2/7/803.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Impact of Sampling Schemes on Demographic Inference: An Empirical Study in Two Species with Different Mating Systems and Demographic Histories</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">K. R. St. Onge</span>, <span class="highwire-citation-author" data-delta="1">A. E. Palmé</span>, <span class="highwire-citation-author" data-delta="2">S. I. Wright</span> and <span class="highwire-citation-author" data-delta="3">M. Lascoux</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">803-814;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002410 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;2/7/803" data-atom-uri="/ggg/2/7/803.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459302" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">K. R. St. Onge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, Uppsala University, SE-75236 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=K.%2BR.%2BSt.%2BOnge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=St.%20Onge%20KR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AK.%2BR.%2BSt.%2BOnge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:K.R.St.Onge@uu.nl" class="" data-icon-position="" data-hide-link-title="0">K.R.St.Onge@uu.nl</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">A. E. Palmé </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, Uppsala University, SE-75236 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=A.%2BE.%2BPalm%C3%A9%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palm%C3%A9%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AA.%2BE.%2BPalm%25C3%25A9%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">S. I. Wright </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology and Centre for Analysis of Genome Evolution and Function, University of Toronto, Toronto, Ontario M5S 3B2, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S.%2BI.%2BWright%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wright%20SI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AS.%2BI.%2BWright%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">M. Lascoux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, Uppsala University, SE-75236 Uppsala, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory of Evolutionary Genomics, CAS-MPG Partner Institute for Computational Biology, Chinese Academy of Sciences, Shanghai 200031, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BLascoux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lascoux%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM.%2BLascoux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/7/803" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459302" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459302/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/7/803.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459302" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459302/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/7/803.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459302" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459302/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/7/803.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459302" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459302/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/7/803.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459302" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459302/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Most species have at least some level of genetic structure. Recent simulation studies have shown that it is important to consider population structure when sampling individuals to infer past population history. The relevance of the results of these computer simulations for empirical studies, however, remains unclear. In the present study, we use DNA sequence datasets collected from two closely related species with very different histories, the selfing species <em>Capsella rubella</em> and its outcrossing relative <em>C. grandiflora</em>, to assess the impact of different sampling strategies on summary statistics and the inference of historical demography. Sampling strategy did not strongly influence the mean values of Tajima’s D in either species, but it had some impact on the variance. The general conclusions about demographic history were comparable across sampling schemes even when resampled data were analyzed with approximate Bayesian computation (ABC). We used simulations to explore the effects of sampling scheme under different demographic models. We conclude that when sequences from modest numbers of loci (&lt;60) are analyzed, the sampling strategy is generally of limited importance. The same is true under intermediate or high levels of gene flow (4Nm &gt; 2–10) in models in which global expansion is combined with either local expansion or hierarchical population structure. Although we observe a less severe effect of sampling than predicted under some earlier simulation models, our results should not be seen as an encouragement to neglect this issue. In general, a good coverage of the natural range, both within and between populations, will be needed to obtain a reliable reconstruction of a species’s demographic history, and in fact, the effect of sampling scheme on polymorphism patterns may itself provide important information about demographic history.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/population-structure" class="hw-term hw-article-keyword hw-article-keyword-population-structure" rel="nofollow">population structure</a></li><li class="kwd"><a href="/keyword/tajima%E2%80%99s-d" class="hw-term hw-article-keyword hw-article-keyword-tajima&#x2019;s-d" rel="nofollow">Tajima’s D</a></li><li class="kwd"><a href="/keyword/frequency-spectrum" class="hw-term hw-article-keyword hw-article-keyword-frequency-spectrum" rel="nofollow">frequency spectrum</a></li><li class="kwd"><a href="/keyword/capsella" class="hw-term hw-article-keyword hw-article-keyword-capsella" rel="nofollow">Capsella</a></li></ul><p id="p-5">Since the seminal work of Sewall <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Wright (1931)</a>, one of the main concerns of population genetics has been the description and explanation of genetic variation within and between populations and species (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Lewontin 1991</a>). To a large extent, this is still true, and the already immense body of literature on the subject continues to grow. Many recent studies have somewhat neglected population structure, concentrating instead on inference of past demographic events, such as bottlenecks or population expansion, through advanced coalescent simulations in single populations or assuming a single population (<em>e.g.</em> <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Thornton and Andolfatto 2006</a>). These inferences have often been based on “pooled” data. Typically, a rather limited number of chromosomes (at least compared with the number often employed in “classical” population genetics studies) are sampled in a few populations across the range of the species of interest and sequenced at a group of loci. When sequences are collected across the range, they are then pooled and analyzed. Generally, the analysis starts by calculating some summary of the site frequency spectrum such as Tajima’s D (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Tajima 1989</a>), Fu and Li’s D (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Fu and Li 1993</a>), Fay and Wu’s H (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Fay and Wu 2000</a>), or of the joint frequency spectrum, such as Wright fixation indices (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wright 1965</a>), when many populations are considered. Among these summaries of the site frequency spectrum, Tajima’s D was the first to be developed and has certainly been the most popular. It belongs to a large family of “neutrality” tests that compare different estimates of the population mutation rate, θ = 4N<sub>e</sub>μ (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Zeng <em>et al.</em> 2006</a>), where N<sub>e</sub> is the effective population size and μ is the mutation rate. The key idea behind these tests is that some estimators of θ = 4N<sub>e</sub>μ will be more sensitive than others to specific departures from the standard neutral model. Comparing two estimators of θ should then indicate the type of departure experienced by the population. For example, Tajima’s D is calculated by taking the difference between θπ, the nucleotide diversity, which is based on pairwise differences between sequences, and θ<sub>W</sub>, which is based on the number of segregating sites, the latter being more sensitive than the former to an excess of rare variants (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Tajima 1989</a>), resulting in a negative value when there is an excess of rare variants and a positive value when there is an excess of intermediate frequency variants. Tajima’s D is versatile and generally has more statistical power to detect a selective sweep than do other single tests based on the site frequency spectrum (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Simonsen <em>et al.</em> 1995</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Przeworski 2002</a>), though as most other such tests, it tends to have low power when considering a single locus. Today with the coming of age of powerful coalescent simulation tools, single-summary statistics such as Tajima’s D are rarely used alone to infer demographic history, but they remain important as input for approximate Bayesian computation (ABC) analysis and for the insights they provide on the underlying genealogical process.</p><p id="p-6">Recent papers (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> 2009</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Chikhi <em>et al.</em> 2010</a>) building on earlier theoretical work by <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Wakeley (1999</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">2001</a>), <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Wakeley and Aliacar (2001)</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Ray <em>et al.</em> (2003)</a>, and <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">De and Durrett (2007)</a> drew attention to the fact that pooling of data across populations can be a problem for demographic inferences. Briefly, the question is whether one can reasonably expect to recover the true species history from a sample of sequences originating from a single deme (local sampling) or pooled from a limited number of demes across the natural range (pooled sampling) when the species is geographically and genetically structured. Simulation studies of simple models of population structure, such as the island model and the stepping-stone model, suggest that local sampling or pooled sampling tends to result in a higher frequency of intermediate frequency polymorphisms than expected in a panmictic population, leading to positive Tajima’s D (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">De and Durrett 2007</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> 2009</a>). Collection of a single sample from each population (scattered sampling) or random sampling would, on the other hand, result in a frequency spectrum more similar to expectations under the standard neutral model, with a Tajima’s D around zero (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">De and Durrett 2007</a>; <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> 2009</a>).</p><p id="p-7">However, in many empirical datasets, pooled samples have a lower Tajima’s D than local samples (<em>e.g.</em> <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Ptak and Przeworski 2002</a>). <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Ptak and Przeworski (2002)</a> observed a trend of a more negative Tajima’s D as more ethnicities are included in human datasets. They interpreted this as the result of a combination of population structure and growth and confirmed with limited simulations that such a pattern can be created in a model including both population structure and expansion. Using more extensive simulations, <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> (2009)</a> verified that such a model can result in negative estimates of Tajima’s D in scattered and pooled samples under a wide range of migration rates and that the estimate based on local samples is either positive or at least less negative (4N<sub>0</sub>m = 0.5–100, where N<sub>0</sub> is the population size and m is the migration rate). This clearly shows that sampling can have an effect on the estimation of Tajima’s D and therefore can influence the interpretation of the data. For example, a positive Tajima’s D across genes is often interpreted as a sign of a population size decrease, but if local sampling in a structured population of stable size can result in positive values, it is easy to draw the wrong conclusion. This problem is not restricted to the estimation of Tajima’s D; sampling can also affect the inference of population history in full Bayesian methods or ABC methods (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Chikhi <em>et al.</em> 2010</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Peter <em>et al.</em> 2010</a>). Population structure, if not properly accounted for, can, for example, lead to the detection of spurious bottlenecks (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Chikhi <em>et al.</em> 2010</a>).</p><p id="p-8">Simulation studies thus indicate that sampling can in some circumstances have a serious impact on the inference of population history. However, by necessity, the demographic models implemented in coalescent simulations remain fairly simple. The basic model in <a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> (2009)</a>, which we build upon in this paper, involves an ancestral population that splits instantly into I demes at time τ in the past. Individual demes all have the same constant population size through time and are connected through migration. Both an n-island model and a stepping-stone model are considered. In the equilibrium model, τ is assumed to be very large, and the total size of the subdivided population is equal to the size of the ancestral population. In the global expansion model in which range expansion is simulated by fragmenting an ancestral population instantaneously into I demes, the global population size over the demes is larger than the ancestral size, thereby simulating global expansion; however, it does not include population expansion within demes. We note that this procedure can therefore result in an initial bottleneck of each single deme; the same is true for the equilibrium model of <a id="xref-ref-31-6" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> (2009)</a>. Additionally, the study did not consider hierarchical population structure, a rather common feature of natural populations with important implications for population genetics inferences (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Robertson 1975</a>). <a id="xref-ref-31-7" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> (2009)</a> tested the effect of the pooled and local sampling schemes on estimates of Tajima’s D and Fu and Li’s D from multilocus data from wild tomatoes, but unfortunately, the data were too limited to implement the scattered sampling scheme.</p><p id="p-9">Although the studies by <a id="xref-ref-31-8" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> (2009)</a> and <a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Chikhi <em>et al.</em> (2010)</a> are both timely and important reminders of the importance of population structure and sampling schemes and have already had an impact on the way empirical studies are conducted (<em>e.g.</em> <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Li <em>et al.</em> 2010</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wheat <em>et al.</em> 2010</a>), the general relevance of their results for empirical datasets remains untested. In the present study, our first goal is to explore the impact of sampling on demographic inferences within two empirical datasets with divergent population structure and history and with a number of loci in the same range as many sequence studies on natural populations of nonmodel organisms (13–16 loci). This contrasts with the simulations of <a id="xref-ref-31-9" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> (2009)</a>, which assumed 1000 independent loci. Second, we want to compare these results with comparable simulated datasets to explore the effect of hierarchical structure and expansion within demes, which was not considered previously. The two investigated species are <em>Capsella rubella</em> and <em>Capsella grandiflora</em>, two closely related species with contrasting population histories and population structures. The selfing species <em>C. rubella</em> derived recently from <em>C. grandiflora</em> (around 13,000 to 70,000 years ago) and underwent a severe bottleneck followed by a population expansion (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Foxe <em>et al.</em> 2009</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Guo <em>et al.</em> 2009</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">St. Onge <em>et al.</em> 2011</a>). <em>C. rubella</em> shows high values of Wright fixation index F<sub>ST</sub>, and Bayesian clustering methods give many clusters with a large amount of admixture (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">St. Onge <em>et al.</em> 2011</a>). In contrast, F<sub>ST</sub> values in <em>C. grandiflora</em> are low. but the species exhibits a clear population genetic structure with three well-delineated clusters showing either evidence of weak population growth or no departure from the standard neutral model (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">St. Onge <em>et al.</em> 2011</a>).</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Populations and subsampled datasets</h3><p id="p-10">Populations of <em>C. grandiflora</em> and <em>C. rubella</em>, previously described in <a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">St. Onge <em>et al.</em> (2011)</a>, were combined with additional populations: numbers 83, 86, 87, 133, 135 in <em>C. grandiflora</em> and numbers 32, 39, 4, 6, 8, 63 in <em>C. rubella</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002410/-/DC1/002410SI.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002410/-/DC1/TableS1.pdf">Table S1</a>). The total number of individuals was 79 and 95 in <em>C. grandiflora</em> and <em>C. rubella</em>, respectively. Briefly, the original populations (core populations) consisted of seven populations of each species, with 6 to 12 individuals each. Five of our <em>C. grandiflora</em> core populations originated from northwestern Greece, and two originated from the Greek island of Corfu. Six of our <em>C. rubella</em> core populations originated from southern Italy and Sicily, and one originated from northeastern Spain. Five additional populations were added to the <em>C. grandiflora</em> populations with 2 individuals each, 3 from northwestern Greece and two from southern Albania. Six additional populations, from Italy, Spain, France, and Luxembourg, were added to the <em>C. rubella</em> populations with 1 to 3 individuals each.</p><p id="p-11">These datasets were subsampled into 3 different datasets corresponding to three different sampling strategies (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), and this subsampling was replicated four times within each species, giving a total of 24 subsampled datasets for analysis. Following <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Wakeley (1999</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">2001</a>) and <a id="xref-ref-31-10" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> (2009)</a>, we defined three basic sampling schemes: local, pooled, and scattered (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Local samples contain <em>n</em> individuals from a single deme. Pooled samples contain several individuals sampled from several demes. Finally, scattered samples include single individuals from each of <em>n</em> different demes. In each species, the populations were randomly subsampled to create 4 random samples in each of the three sampling schemes, resulting in a total of 12 subsamples in each species. Each local subsample consisted of 12 individuals sampled from the same geographic population. Due to the limit in the size of the some populations, the 4 populations with the highest number of individuals were used as the local subsamples for each species. The pooled subsamples were obtained by sampling the 12 individuals from 4 populations (three individuals from each), and finally, the scattered subsamples consisted of 12 individuals sampled in as many populations. Individuals and populations were chosen randomly for both the pooled and scattered subsamples, except for populations with fewer than 3 individuals, which were excluded from the pooled subsamples to keep equal numbers of sampled individuals per population. In total for each species, we analyzed 12 subsamples each with sequence data from 12 individuals (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Because of the high levels of inbreeding in <em>C. rubella</em>, the data from this species was treated as haploid and the same was done for <em>C. grandiflora</em> for comparison of summary statistics.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461540/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461540/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Sampling schemes used in both <em>C. rubella</em> and <em>C. grandiflora</em></span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/7/803/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic representation of the three sampling schemes: (A) local, (B) scattered, and (C) pooled. Boxes represent sampled populations and dots represent sampled individuals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2050857728" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Schematic representation of the three sampling schemes: (A) local, (B) scattered, and (C) pooled. Boxes represent sampled populations and dots represent sampled individuals.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/7/803/F1.medium.gif" width="440" height="296"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/2/7/803/F1.medium.gif" width="440" height="296"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/7/803/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/7/803/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461573" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-12" class="first-child">Schematic representation of the three sampling schemes: (A) local, (B) scattered, and (C) pooled. Boxes represent sampled populations and dots represent sampled individuals.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Sequence data</h3><p id="p-13">Sequence data were collected for 16 and 13 genes in <em>C. grandiflora</em> and <em>C. rubella</em>, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002410/-/DC1/TableS2.pdf">Table S2</a>). Details of PCR and sequencing of these genes can be found in <a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">St. Onge <em>et al.</em> (2011)</a>. Briefly, PCR primers were designed using the <em>A. thaliana</em> reference genome to amplify 500–700 bp fragments, including both exons and introns. The forward and reverse strands of these fragments were sequenced directly at the Macrogen sequencing facility in Korea. The sequences were aligned and edited using CodonCode Aligner v2.0.6 (CodonCode, Dedham, MA). The software PHASE 2.1 ( <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Stephens <em>et al.</em> 2001</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Stephens and Donnelly 2003</a>), implemented in DNAsp v4.50 (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Librado and Rozas 2009</a>) was used to generate haplotypes. We also cloned and sequenced several PCR-products of each gene, except for AT3G62890 and AT4G08920, and this information was used to improve phase determination by including known haplotypes in PHASE.</p></div><div id="sec-4" class="subsection"><h3>Population structure</h3><p id="p-14">A previous analysis of genetic structure in the two species showed that 3 distinct genetic clusters are present in the seven <em>C. grandiflora</em> core populations, and that at least 5 clusters with some admixture are present in the seven <em>C. rubella</em> core populations (<a id="xref-ref-29-6" class="xref-bibr" href="#ref-29">St. Onge <em>et al.</em> 2011</a>). For this study, we reran the STRUCTURE analysis described in <a id="xref-ref-29-7" class="xref-bibr" href="#ref-29">St. Onge <em>et al.</em> (2011)</a> with the newly sequenced individuals from the additional populations. Briefly, we used the Bayesian clustering algorithm in the program STRUCTURE 2.2 (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Pritchard <em>et al.</em> 2000</a>) to investigate the probability of 1 to 12 clusters (K). For this analysis, we used the complete sequences of each sequenced locus as haplotypes, excluding singletons. For each value of K, the program was run 10 times using the admixture model with uncorrelated allele frequencies, burn-in period of 100,000, and run length of 500,000, at which stage convergence was achieved (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002410/-/DC1/FigureS3.pdf">Figure S3</a>). To evaluate the optimal number of genetic clusters, delta K was calculated according to <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Evanno <em>et al.</em> (2005)</a>. We also report the LnP(D) values in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002410/-/DC1/FileS1.pdf">File S1</a>. Because they are not informative, singletons were excluded as well as individuals with more than 50% missing data.</p><p id="p-15">To be able to compare our results with those reported in <a id="xref-ref-31-11" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> (2009)</a>, we also estimated Wright’s fixation indices (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wright 1951</a>, <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">1965</a>) using an AMOVA based on pairwise differences (Excoffier <em>et al.</em> 1992) implemented in Arlequin v3.5 (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Excoffier <em>et al.</em> 2005</a>). As there is some missing data, we followed the suggestion of the software authors and used a locus-by-locus AMOVA rather than a standard AMOVA. Significance of F<sub>ST</sub> was tested by 1000 permutations of haplotypes among populations.</p></div><div id="sec-5" class="subsection"><h3>Summary statistics</h3><p id="p-16">Watterson’s estimate of the population mutation rate (θ<sub>W</sub>) (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Watterson 1975</a>), the number of segregating sites (S), the nucleotide diversity (π), Tajima’s D (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Tajima 1989</a>), and Fu and Li’s D (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Fu and Li 1993</a>) were calculated for all sites using the libsequence C++ library (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Thornton 2003</a>). In <em>C. grandiflora</em>, we also considered only the synonymous sites to avoid potential effects of selection. Synonymous site were also considered in <em>C. rubella</em> for comparison with the synonymous statistics of <em>C. grandiflora</em>; however, it should be noted that data are very limited in the haploid synonymous dataset of <em>C. rubella</em> due to the low variation in this species. All calculations were done separately for each of the 12 subsamples and for the whole dataset in each species. Averages and variances for each sampling scheme were calculated by first averaging each locus across the four replicates and then averaging over all loci.</p></div><div id="sec-6" class="subsection"><h3>ABC analysis</h3><p id="p-17">To infer the demographic history of species, one seldom relies only upon raw summary statistics of the frequency spectrum. Instead, one uses them within ABC (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Lopes and Beaumont 2010</a>) analysis or likelihood models. We therefore conducted an ABC analysis using the program SEQLIB v1.0 (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">De Mita <em>et al.</em> 2007</a>) on all 12 subsamples of <em>C. rubella</em> and <em>C. grandiflora</em>. This analysis was conducted following <a id="xref-ref-29-8" class="xref-bibr" href="#ref-29">St. Onge <em>et al.</em> (2011)</a>, using synonymous sites for <em>C. grandiflora</em> and total sites for <em>C. rubella</em>. Analysis was done with somewhat shorter runs; 250,000 samples were generated in both the initial and ABC steps. Because this number of samples gave similar results to our previous study using this program, 250,000 samples were considered adequate. The summary statistics used here are the nucleotide diversity (π), Watterson’s estimator of the population mutation rate (θ<sub>W</sub>), and the number of unique haplotypes (excluding singletons) (K), and these statistics were calculated for 16 loci in <em>C. grandiflora</em> and 13 in <em>C. rubella</em>. In the initial step, 5% or 10% of samples in <em>C. rubella</em> and <em>C. grandiflora</em>, respectively, were kept on which the new prior ranges were based. The model probabilities were based on the acceptance rate from the ABC step using an absolute acceptance threshold of a Euclidean distance of 0.05 or less from the real data. We tested the same demographic models tested in <a id="xref-ref-29-9" class="xref-bibr" href="#ref-29">St. Onge <em>et al.</em> (2011)</a>: (i) a standard neutral model (SNM) that includes one parameter (θ, the population mutation rate); (ii) a population expansion model (PEM) with two parameters (θ and α, an exponential growth factor); (iii) an instant change model with three parameters [θ, the time of population size change (T), and the population mutation rate of the ancestral population (θA)]; and (iv) a bottleneck model (BNM) with four parameters [θ, time since end of bottleneck (T), duration of bottleneck (d), and size of the population during the bottleneck (f)]. See supplementary text in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002410/-/DC1/FileS1.pdf">File S1</a> for further details.</p></div><div id="sec-7" class="subsection"><h3>Simulations</h3><p id="p-18">To understand the nature of the discrepancy between our results (minimal effects of sampling on summary statistics) and those of <a id="xref-ref-31-12" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> (2009)</a> and thereby the robustness of the latter, we used computer simulations. The simulations and figures were produced with the program Sampling v0.5 (<a href="http://guanine.evolbio.mpg.de/sampling/">http://guanine.evolbio.mpg.de/sampling/</a>), the program used by <a id="xref-ref-31-13" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> (2009)</a> (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figures 4</a> and <a id="xref-fig-5-1" class="xref-fig" href="#F5">5</a>), or with slightly modified versions of it (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A, B</a>). The model implemented in this program considers a large ancestral population, which instantaneously splits into subdivided populations at time τ with migration between them, 4Nm, and an expansion factor β. We first used Sampling with and without expansion to compare simulated results to our empirical results by assuming 20 demes and 16 loci, the number of loci used in the present study. Using Sampling without expansion, we varied the number of loci to assess the robustness of the results to the number of loci used. Then using Sampling with expansion, we drew contour plots of Tajima’s D for different values of the time at which population expansion started (τ) and of the population expansion factor (β) under local and pooled samplings. Finally, the models implemented in <a id="xref-ref-31-14" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> (2009)</a> are rather simplistic, with an instantaneous shift from an ancestral population to subdivided populations that likely exacerbates the differences between the different sampling schemes more than a progressive shift toward an n-island model would do. To test the robustness of the model to departures from the patterns seen in <a id="xref-ref-31-15" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> (2009)</a>, we made two simple modifications to the basic model implemented by Sampling. First their global expansion model does not consider growth of individual populations but only a relative increase of the global population size compared with the ancestral population. We therefore added exponential growth (α = 25) to each subpopulation in the Wright island model with 100 demes without recombination. Twenty individuals were sampled. Second, we implemented a simple hierarchical model in which the ancestral population first splits into two main populations that then split into subpopulations, but otherwise use the same parameters as the <a id="xref-ref-31-16" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> (2009)</a> global expansion model. Looking backward, demes 1 to 50 merge into a single deme and population 51 to 100 into another deme of the same size at time τ = 1, and these two remaining demes merged into a single ancestral deme at time τ = 2. At each stage, we assume a Wright island model with no recombination, and 20 individuals were sampled. The final number of subpopulations was the same as in the model of <a id="xref-ref-31-17" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> (2009)</a>, and migration was possible between the two initial subpopulations as well as among the final subpopulations. This model included global expansion but no expansion within local populations. The modified Sampling code is available from M. Lascoux.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Genetic diversity and population structure</h3><p id="p-19">Summary statistics of the total dataset for our two species are reported in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002410/-/DC1/TableS2.pdf">Table S2</a>. Although the present dataset is somewhat larger than that presented in <a id="xref-ref-29-10" class="xref-bibr" href="#ref-29">St. Onge <em>et al.</em> (2011)</a>, diversity statistics are highly similar. Briefly, on average 151 chromosomes were sequenced over 12 populations of <em>C. grandiflora</em>, and 86 (haploid number) chromosomes were sequenced over 13 populations of <em>C. rubella</em>. Genetic diversity is reduced in <em>C. rubella</em> compared with <em>C. grandiflora</em> (total sites π = 0.007 and 0.005, synonymous sites π = 0.021 and 0.013, and average number of haplotypes per locus = 38.3 and 4.23 for the total datasets of <em>C. grandiflora</em> and <em>C. rubella</em>, respectively). Tajima’s D for the total datasets are −1.15 and 0.71 (−0.52 and 0.73 for synonymous) for <em>C. grandiflora</em> and <em>C. rubella</em>, respectively, suggesting population expansion in <em>C. grandiflora</em> and a population bottleneck in <em>C. rubella</em>.</p><p id="p-20">The results from clustering analysis in STRUCTURE generally agreed well with our expectations based on geography and previous analysis on a subset of these populations (<a id="xref-ref-29-11" class="xref-bibr" href="#ref-29">St. Onge <em>et al.</em> 2011</a>). In <em>C. grandiflora</em>, the clustering analysis supports the presence of three genetic clusters and the populations that have been added cluster with one or two of the neighboring original populations (for clustering and the plot of ΔK over the number of clusters, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002410/-/DC1/FigureS1.pdf">Figure S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002410/-/DC1/FigureS2.pdf">Figure S2</a>; for plots of LnP(D) over K, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002410/-/DC1/FileS1.pdf">File S1</a>). In general, geographically close populations are inferred to belong to the same genetic clusters. However, as the sample sizes of some populations are low, we have limited power to detect additional genetic clusters containing only such populations. At values of K = 4 and greater, admixture is added (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002410/-/DC1/FigureS2.pdf">Figure S2</a>).</p><p id="p-21">As in <a id="xref-ref-29-12" class="xref-bibr" href="#ref-29">St. Onge <em>et al.</em> (2011)</a>, the clustering analysis in <em>C. rubella</em> shows a large amount of admixture, and populations 22, 23, 49 and 50 show low levels of admixture (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002410/-/DC1/FigureS2.pdf">Figure S2</a>). At the most likely number of genetic clusters (K = 6, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002410/-/DC1/FigureS2.pdf">Figure S2</a>) and also for other values of K, population 6 is genetically similar to the geographically close population 50, whereas populations 3, 35, 39, and 4 show a high level of admixture.</p><p id="p-22">The average F<sub>ST</sub> over all loci and all populations were 0.14 and 0.54 in <em>C. grandiflora</em> and <em>C. rubella</em>, respectively. Permutation tests (1000 permutations) showed that both values are significantly different from zero (<em>P</em> &lt; 0.01).</p></div><div id="sec-10" class="subsection"><h3>Summary statistics</h3><p id="p-23">In both species, there was an increase in the average number of segregating sites from local to pooled to scattered sampling (4.1, 6.8, and 8.0 in <em>C. rubella</em>, and 9.9, 11.1, and 11.5 in <em>C. grandiflora</em> for local, pooled, and scattered, respectively, <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). A similar increase is also observed in Watterson’s estimate of θ, which is a direct function of S, and for π. In <em>C. grandiflora</em>, the differences between the three sampling schemes were not very pronounced, with overlapping interquartile ranges between local and pooled or pooled and scattered, but not between the local and scattered sampling schemes (paired <em>t</em>-test, <em>P</em> &gt; 0.05 for all test). Differences were greater between the <em>C. rubella</em> sampling types, where interquartile ranges were not overlapping between any of the sampling schemes (paired <em>t</em>-tests, <em>P</em> &gt; 0.05 for all tests except for local sampling values of S, θ, and π, which are significantly different from those of scattered and pooled sampling).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/7/803/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Box plots of summary statistics averaged over all loci for the different sampling strategies: number of segregating sites, S (top row); nucleotide diversity, &#x3C0; (middle row); and Watterson&#x2019;s estimate of the population mutation rate, &#x3B8;W (bottom row). Statistics are shown for both total site and synonymous sites. Gray lines indicate the average value for the total dataset. An asterisk (*) indicates total dataset values that lie far outside the range of the figure; these values are 30.5 and 13.7 for S in C. grandiflora for total sites and synonymous sites, respectively. Each box plot is calculated using n = 4, boxes range the upper and lower quartiles, the bold line indicates the median value, and whiskers indicate 1.5 times the quartile range." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2050857728" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Box plots of summary statistics averaged over all loci for the different sampling strategies: number of segregating sites, S (top row); nucleotide diversity, &#x3C0; (middle row); and Watterson&#x2019;s estimate of the population mutation rate, &#x3B8;&lt;sub&gt;W&lt;/sub&gt; (bottom row). Statistics are shown for both total site and synonymous sites. Gray lines indicate the average value for the total dataset. An asterisk (&lt;sup&gt;*&lt;/sup&gt;) indicates total dataset values that lie far outside the range of the figure; these values are 30.5 and 13.7 for S in &lt;em&gt;C. grandiflora&lt;/em&gt; for total sites and synonymous sites, respectively. Each box plot is calculated using n = 4, boxes range the upper and lower quartiles, the bold line indicates the median value, and whiskers indicate 1.5 times the quartile range.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/7/803/F2.medium.gif" width="362" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/2/7/803/F2.medium.gif" width="362" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/7/803/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/7/803/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461549" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-24" class="first-child">Box plots of summary statistics averaged over all loci for the different sampling strategies: number of segregating sites, S (top row); nucleotide diversity, π (middle row); and Watterson’s estimate of the population mutation rate, θ<sub>W</sub> (bottom row). Statistics are shown for both total site and synonymous sites. Gray lines indicate the average value for the total dataset. An asterisk (<sup>*</sup>) indicates total dataset values that lie far outside the range of the figure; these values are 30.5 and 13.7 for S in <em>C. grandiflora</em> for total sites and synonymous sites, respectively. Each box plot is calculated using n = 4, boxes range the upper and lower quartiles, the bold line indicates the median value, and whiskers indicate 1.5 times the quartile range.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">The average number of singletons also tends to increase from local to pooled to scattered samples, although this trend is less apparent than the increasing trend of the diversity statistics discussed above. In <em>C. grandiflora</em>, the average number of singletons of the scattered samples is actually slightly lower than the estimate of the pooled sampling. No great differences are seen between sampling strategies in <em>C. rubella</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002410/-/DC1/FigureS4.pdf">Figure S4</a>). Note that, for comparison, the average number of singletons was calculated for total sites due to a lack of data in <em>C. rubella</em> synonymous dataset.</p><p id="p-26">The average Tajima’s D is not strongly affected by the sampling strategies in either species. In <em>C. rubella</em> average Tajima’s D values are positive (0.53, 0.99, and 0.53 for local, pooled, and scattered samplings, respectively), whereas all estimates of average Tajima’s D are negative (−0.36, −0.43, and −0.50 for local, pooled, and scattered samplings, respectively) in <em>C. grandiflora</em>. The distribution of Tajima’s D estimates becomes narrower in the scattered sampling in <em>C. rubella</em> (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Similar patterns were observed for Fu and Li’s D (data not shown).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/7/803/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Box plots of Tajima&#x2019;s D averaged over all loci for different sampling strategies in C. grandiflora and C. rubella. Statistics are shown for both total site and synonymous sites. Gray lines indicate the average value for the total dataset. Each box plot is calculated using n = 4, boxes range the upper and lower quartiles, the bold line indicates the median value, and whiskers indicate 1.5 times the quartile range." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2050857728" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Box plots of Tajima&#x2019;s D averaged over all loci for different sampling strategies in &lt;em&gt;C. grandiflora&lt;/em&gt; and &lt;em&gt;C. rubella&lt;/em&gt;. Statistics are shown for both total site and synonymous sites. Gray lines indicate the average value for the total dataset. Each box plot is calculated using n = 4, boxes range the upper and lower quartiles, the bold line indicates the median value, and whiskers indicate 1.5 times the quartile range.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/7/803/F3.medium.gif" width="440" height="209"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/2/7/803/F3.medium.gif" width="440" height="209"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/7/803/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/7/803/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461673" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">Box plots of Tajima’s D averaged over all loci for different sampling strategies in <em>C. grandiflora</em> and <em>C. rubella</em>. Statistics are shown for both total site and synonymous sites. Gray lines indicate the average value for the total dataset. Each box plot is calculated using n = 4, boxes range the upper and lower quartiles, the bold line indicates the median value, and whiskers indicate 1.5 times the quartile range.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">Also note that, because we used total diversity in these analyses, Tajima’s D may be partly influenced by purifying selection on nonsynonymous sites (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Slotte <em>et al.</em> 2010</a>; Zeng and Charlesworth 2011). Indeed, although the same subtle trend of decreasing Tajima’s D is apparent in <em>C. grandiflora</em> when considering just synonymous sites, the average values of Tajima’s D tend to be closer to zero (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>).</p><p id="p-29">In comparing the summary statistics of the subsamples to the total dataset, which is a mixed dataset with many individuals sampled in some locations and few in others (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002410/-/DC1/TableS1.pdf">Table S1</a>), we find that for all statistics, the scattered sample of <em>C. grandiflora</em> is closest to the total dataset. Diversity is increased in the total dataset, and Tajima’s D is more negative than under any of the three sampling schemes (−1.15). For <em>C. rubella</em>, the comparison is not so straightforward; the number of segregating sites is larger in the total dataset, whereas the number of singletons is smaller than in the subsamples. This seems intuitive, considering the selfing habit of the species: as we increase the number of sampled populations, we increase the amount of variation we sample, and as we increase the number of individuals sampled per population, we increase the chance of sampling the same haplotype multiple times, thereby reducing the number of singletons. It follows that the estimates of θ<sub>W</sub> and π for the pooled samples are closest to those of the total dataset, whereas the estimates of Tajima’s D in the local and scattered samples are closest to that of the total dataset (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002410/-/DC1/TableS2.pdf">Table S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002410/-/DC1/TableS3.pdf">Table S3</a> and <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figures 2</a> and <a id="xref-fig-3-3" class="xref-fig" href="#F3">3</a>).</p></div><div id="sec-11" class="subsection"><h3>ABC analysis</h3><p id="p-30">To further investigate the robustness of the demographic inference, we conducted simulations using an Approximate Bayesian Computation (ABC, <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Lopes and Beaumont 2010</a>) approach described in <a id="xref-ref-29-13" class="xref-bibr" href="#ref-29">St. Onge <em>et al.</em> (2011)</a> on all 12 subsampled datasets in both species. Four models were tested: (i) a standard neutral model (SNM), (ii) a population expansion model (PEM), (iii) an instant population size change model (ICM), and (iv) a bottleneck model (BNM).</p><p id="p-31">In <em>C. rubella</em>, this analysis supports a recent and strong reduction in population size (ICM or BNM models) in all four subsamples of all three sampling strategies (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>), clearly demonstrating that the signature of the bottleneck at speciation in <em>C. rubella</em> is stronger than any effect that the sampling strategy may have. Because the acceptance rate was 0.0 for the SNM for all analyses, raw acceptance rates are presented rather than Bayes factors. From the instant change model, point estimates of effective population size, date of population size change, and the ancient population size ranged from 6000 to 26,000 individuals, from 8000 to 204,000 years ago, and from 5 to 31 times larger than the current size, respectively. Overall, these results suggest that major demographic events may simply overpower the effects of the sampling strategy. Posterior probability curves for these parameters are given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002410/-/DC1/FileS2.pdf">File S2</a>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461632/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461632/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Probabilities for the models tested in <em>C. rubella</em></span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461650/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461650/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Probabilities for the models in tested <em>C. grandiflora</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-34">In all 12 subsamples in <em>C. grandiflora</em>, probabilities of the three models are very similar (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>) and would give Bayes factors no larger than three in any subsample. This supports previous studies (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Foxe <em>et al.</em> 2009</a>; <a id="xref-ref-29-14" class="xref-bibr" href="#ref-29">St. Onge <em>et al.</em> 2011</a>) that find either no strong departure from the standard neutral model or evidence of weak expansion in <em>C. grandiflora</em>, and it also demonstrates that sampling strategy does not alter this conclusion. Posterior probability curves for this analysis are given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002410/-/DC1/FileS3.pdf">File S3</a>.</p><p id="p-35">Note that the main purpose of the ABC analysis here is to see whether different conclusions would be drawn under different sampling schemes and not to identify the model with the best fit to our data, as demography in these two species has previously been the focus of several independent studies (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Foxe <em>et al.</em> 2009</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Guo <em>et al.</em> 2009</a>; <a id="xref-ref-29-15" class="xref-bibr" href="#ref-29">St. Onge <em>et al.</em> 2011</a>)</p></div><div id="sec-12" class="subsection"><h3>Simulations</h3><p id="p-36"><a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> gives the mean of simulated Tajima’s D values under the three sampling schemes as a function of gene flow (4N<sub>0</sub>m) and assuming an equilibrium stepping-stone model equivalent to <a id="xref-ref-31-18" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> (2009)</a> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). The number of loci was equal to 16 in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>. In <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B–F</a>, the number of loci is doubled for each plot from panel B to panel F. Our estimate of F<sub>ST</sub> is equal to 0.54 in <em>C. rubella</em>. An F<sub>ST</sub> of 0.54 corresponds approximately to a value of 4Nm = 1 under a Wright island equilibrium model [F<sub>ST</sub> = 1/(4Nm + 1)]. However, <em>C. rubella</em> does not follow an equilibrium model and instead, like many other selfers, most likely went through bottlenecks during colonization (<a id="xref-ref-29-16" class="xref-bibr" href="#ref-29">St. Onge <em>et al.</em> 2011</a>). An F<sub>ST</sub> of 0.54 will therefore likely be obtained for a proportionally higher number of migrants (as there is more drift within populations) than under an equilibrium model, making our estimate of migration upwardly biased. Therefore, we expect that for a species at equilibrium, the same F<sub>ST</sub> value would be obtained for a lower level of migration and a 4Nm of slightly less than 1. If we then consider the 4Nm values between 0.5 and 1 in <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4A</a>, we see that there is a good qualitative fit with the <em>C. rubella</em> data: the local and pooled samplings yield similar values of Tajima’s D, and the scattered sampling yields a somewhat lower value although the difference is not significant. It is therefore clear that a simple model with divergence among subpopulations can explain at least part of the deviation from expectations under the standard neutral model (Tajima’s D = 0 under neutral expectations) observed in Tajima’s D in <em>C. rubella</em>. Note that the effect of sampling becomes clear only when the number of loci is approximately 100.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/7/803/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Tajima&#x2019;s D (DT) as a function of gene flow (4N0m) in an equilibrium stepping-stone model (without expansion). Simulations were run with Sampling v0.5 in which every plotted point is the average based on 1000 independently generated datasets and vertical lines are standard errors. In panel A, we assumed 20 demes and 16 loci, the number of loci used in the present study. Panels B&#x2013;F are the same as panel A with increasing number of loci 32, 64, 128, 256, and 512. See text for further details." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2050857728" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Tajima&#x2019;s D (D&lt;sub&gt;T&lt;/sub&gt;) as a function of gene flow (4N&lt;sub&gt;0&lt;/sub&gt;m) in an equilibrium stepping-stone model (without expansion). Simulations were run with Sampling v0.5 in which every plotted point is the average based on 1000 independently generated datasets and vertical lines are standard errors. In panel A, we assumed 20 demes and 16 loci, the number of loci used in the present study. Panels B&#x2013;F are the same as panel A with increasing number of loci 32, 64, 128, 256, and 512. See text for further details.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/7/803/F4.medium.gif" width="440" height="309"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/2/7/803/F4.medium.gif" width="440" height="309"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/7/803/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/7/803/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461537" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-37" class="first-child">Tajima’s D (D<sub>T</sub>) as a function of gene flow (4N<sub>0</sub>m) in an equilibrium stepping-stone model (without expansion). Simulations were run with <span class="sc">Sampling</span> v0.5 in which every plotted point is the average based on 1000 independently generated datasets and vertical lines are standard errors. In panel A, we assumed 20 demes and 16 loci, the number of loci used in the present study. Panels B–F are the same as panel A with increasing number of loci 32, 64, 128, 256, and 512. See text for further details.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">According to the results of <a id="xref-ref-31-19" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> (2009)</a>, it is possible to get a negative Tajima’s D under some circumstances with local and pooled sampling using a model combining expansion and population structure. To explore which parameter values of the global expansion model will produce simulations with Tajima’s D values similar to the observed values of Tajima’s D in our <em>C. grandiflora</em> dataset, we used Sampling v.0.5 to draw contour plots of Tajima’s D for different values of the time at which population expansion started (τ) and of the population expansion factor (β) under local and pooled samplings. We assumed an island model with 20 demes and 16 loci, 4Nm = 6 and θ = 4, which correspond roughly to the values per locus in <em>C. grandiflora</em>. The number of chromosomes sampled in single demes was 12, and the number of chromosome in the pooled sample was 48. The contour plots are given in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>. The observed mean values of Tajima’s D for <em>C. grandiflora</em> are −0.44 and −0.49 for the local and pooled sampling strategies, respectively. Such values can be obtained for relatively low values of τ (&lt;10) and moderate values of β (&gt;2 in pooled and &gt;6 in local; <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/7/803/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Contour plots showing average values of Tajima&#x2019;s D (DT) obtained in simulations designed to mimic the empirical data from C. grandiflora. (A) Single deme samples (n = 12 sequences) and (B) pooled samples (n = 48 sequences). We assumed an island model with 20 demes, 4Nm = 6, &#x3B8; = 4. The contours discussed in the text in relation to the C. grandiflora dataset are highlighted with gray." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2050857728" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Contour plots showing average values of Tajima&#x2019;s D (D&lt;sub&gt;T&lt;/sub&gt;) obtained in simulations designed to mimic the empirical data from &lt;em&gt;C. grandiflora&lt;/em&gt;. (A) Single deme samples (n = 12 sequences) and (B) pooled samples (n = 48 sequences). We assumed an island model with 20 demes, 4Nm = 6, &#x3B8; = 4. The contours discussed in the text in relation to the &lt;em&gt;C. grandiflora&lt;/em&gt; dataset are highlighted with gray.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/7/803/F5.medium.gif" width="440" height="251"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/2/7/803/F5.medium.gif" width="440" height="251"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/7/803/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/7/803/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461581" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-39" class="first-child">Contour plots showing average values of Tajima’s D (D<sub>T</sub>) obtained in simulations designed to mimic the empirical data from <em>C. grandiflora</em>. (A) Single deme samples (n = 12 sequences) and (B) pooled samples (n = 48 sequences). We assumed an island model with 20 demes, 4Nm = 6, θ = 4. The contours discussed in the text in relation to the <em>C. grandiflora</em> dataset are highlighted with gray.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">When local population growth was added to the global expansion model, negative Tajima’s D values are obtained for both local and scattered sampling schemes under low migration and not only under the scattered sampling strategy (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A</a>). Thus, if there is within-deme expansion, it can be detected by within-deme sampling, but this local expansion signal becomes obscured with pooled sampling. Conversely, this within-deme local sampling is not sufficient to detect global expansion if local samples are experiencing bottlenecking as a result of subdivision, as is the case in the <a id="xref-ref-31-20" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> (2009)</a> expansion model. Importantly, this only holds for strongly subdivided populations, as the three sampling schemes lead to similar values of Tajima’s D at intermediate and high migration rates (above 4Nm = 5 or 10).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/7/803/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Averages of Tajima&#x2019;s D (DT) as a function of migration rates between demes in a model including both species-wide expansion (&#x3B2; = 10, &#x3C4; = 2) and growth within demes (A) or hierarchical structure (B). As in Figure 2 in St&#xE4;dler et al. (2009), simulations were run with Sampling v0.5 and every plotted point is based on 1000 independently generated datasets, corresponding to 1000 unlinked loci; error bars represent standard errors. In panel A, the exponential growth rate within each deme was &#x3B1; = 25.We simulated a Wright island model with 100 demes without recombination. In panel B, demes 1 to 50 merge into one deme and demes 51 to 100 into another one at time &#x3C4; = 1, and then the two remaining demes merge into a single ancestral deme at time &#x3C4; = 2. At each stage, we assume a Wright island model with no recombination." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2050857728" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Averages of Tajima&#x2019;s D (D&lt;sub&gt;T&lt;/sub&gt;) as a function of migration rates between demes in a model including both species-wide expansion (&#x3B2; = 10, &#x3C4; = 2) and growth within demes (A) or hierarchical structure (B). As in Figure 2 in St&#xE4;dler &lt;em&gt;et al.&lt;/em&gt; (2009), simulations were run with Sampling v0.5 and every plotted point is based on 1000 independently generated datasets, corresponding to 1000 unlinked loci; error bars represent standard errors. In panel A, the exponential growth rate within each deme was &#x3B1; = 25.We simulated a Wright island model with 100 demes without recombination. In panel B, demes 1 to 50 merge into one deme and demes 51 to 100 into another one at time &#x3C4; = 1, and then the two remaining demes merge into a single ancestral deme at time &#x3C4; = 2. At each stage, we assume a Wright island model with no recombination.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/7/803/F6.medium.gif" width="440" height="250"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/2/7/803/F6.medium.gif" width="440" height="250"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/7/803/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/7/803/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461595" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-41" class="first-child">Averages of Tajima’s D (D<sub>T</sub>) as a function of migration rates between demes in a model including both species-wide expansion (β = 10, τ = 2) and growth within demes (A) or hierarchical structure (B). As in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a> in <a id="xref-ref-31-21" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> (2009)</a>, simulations were run with <span class="sc">Sampling</span> v0.5 and every plotted point is based on 1000 independently generated datasets, corresponding to 1000 unlinked loci; error bars represent standard errors. In panel A, the exponential growth rate within each deme was α = 25.We simulated a Wright island model with 100 demes without recombination. In panel B, demes 1 to 50 merge into one deme and demes 51 to 100 into another one at time τ = 1, and then the two remaining demes merge into a single ancestral deme at time τ = 2. At each stage, we assume a Wright island model with no recombination.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">When a simple hierarchical structure is added to the model, the scattered and pooled sampling strategies lead to similar and positive values of Tajima’s D, whereas for low migration rates, the local sampling strategy leads to values close to zero (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a>). For higher levels of migration (4Nm &gt; 2), again the three sampling strategies give similar results. Thus, at high or intermediate levels of migration, the estimation of Tajima’s D is not largely affected by the sampling strategy if there is either expansion within subpopulations or a hierarchical structure.</p></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-43">In the present study, we evaluated the impact of different sampling strategies on summary statistics of the site frequency spectrum and on ABC analyses in two related species with different mating systems and demographic histories. <em>C. rubella</em> is a young species that experienced a severe bottleneck when it derived from <em>C. grandiflora</em> (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Foxe <em>et al.</em> 2009</a>; <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Guo <em>et al.</em> 2009</a>; <a id="xref-ref-29-17" class="xref-bibr" href="#ref-29">St. Onge <em>et al.</em> 2011</a>), and it is geographically structured (<a id="xref-ref-29-18" class="xref-bibr" href="#ref-29">St. Onge <em>et al.</em> 2011</a>). In contrast, <em>C. grandiflora</em> experienced some limited population growth but showed no evidence of bottleneck (<a id="xref-ref-29-19" class="xref-bibr" href="#ref-29">St. Onge <em>et al.</em> 2011</a>). Its population genetic structure is clear, but the amount of population subdivision is low. The two species should therefore constitute a good empirical ground to test the relevance of the results presented by <a id="xref-ref-31-22" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> (2009)</a>, who showed that pooled sampling yielded values of the summary statistics that are intermediate between those of the local and scattered sampling schemes. Our interest was also in the possibility that, rather than simply viewing sampling as an “artifact,” looking at the effects of different sampling schemes on polymorphism patterns may actually provide important insights into demographic history. We reached two main conclusions. First, the trends across the three sampling schemes in our results fit quite well with the predictions of <a id="xref-ref-31-23" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> (2009)</a>. Second, the effect of the sampling schemes on the summary statistics, and the conclusions drawn from them, seem limited in the range of population structure and demographic history encompassed by the two species studied here, even though the expected trends in summary statistics over sampling schemes are clear. Below we discuss both aspects, and in the last section, we attempt to give some general conclusions.</p><p id="p-44">The overall trend observed in this dataset is that diversity increases from local to pooled to scattered sampling (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>), even though the sample size remains the same. That the sampling of more subpopulations results in the identification of more polymorphisms and larger differences among sequences is, of course, not surprising. Per definition, population structure implies that individuals within a subpopulation are more similar to each other than to individuals in other subpopulations. This trend of increasing variation with the number of subpopulations sampled is also expected to be stronger in selfing than in outcrossing species as, at equilibrium, population structure is expected to be more pronounced in the former than in the latter and a larger part of the variation is therefore due to population differences. In concordance with this, we observe a stronger increase in diversity (S, θ<sub>W</sub>, π) in <em>C. rubella</em> than in <em>C. grandiflora</em> (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>). Both theoretical and empirical studies indicate that more singletons or rare alleles should be present in samples from many populations than from few (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Ptak and Przeworski 2002</a>; <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">De and Durrett 2007</a>). However, we did not observe a significant pattern in this direction for <em>C. rubella</em>, even though there is a significant trend in the other diversity statistics (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>). This could perhaps be explained by the very low number of singletons in this species and a fairly large variance due to the severity of the population bottleneck.</p><div id="sec-14" class="subsection"><h3>Population structure and bottleneck explain Tajima’s D in <em>C. rubella</em></h3><p id="p-45">Simulations showed that a simple model with divergence among subpopulations can explain at least part of the deviation from expectations under the standard neutral model observed in Tajima’s D in <em>C. rubella</em>. So, is the inference of a population bottleneck in <em>C. rubella</em> incorrect? It is quite clear from other evidence that this is not the case. Three studies that used different population genetic approaches and sampling strategies, all concluded that <em>C. rubella</em> has gone through a bottleneck (<a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Foxe <em>et al.</em> 2009</a>; <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Guo <em>et al.</em> 2009</a>; <a id="xref-ref-29-20" class="xref-bibr" href="#ref-29">St. Onge <em>et al.</em> 2011</a>). However, they all used variations on the pooled sampling strategy and theoretical studies suggest that the best way to avoid, or at least minimize, the problem of sampling in a structured population is to use only a single sample from each population (<a id="xref-ref-31-24" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> 2009</a>; <a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">Chikhi <em>et al.</em> 2010</a>). We see in <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a> that even with the scattered sampling Tajima’s D is different from zero. In addition, the ABC analysis with different sampling strategies led to the same conclusion strongly suggesting that the deviation from the standard neutral model is not only due to population structure.</p></div><div id="sec-15" class="subsection"><h3>Population expansions could explain part of the deviation in Tajima’s D in <em>C. grandiflora</em></h3><p id="p-46">In <em>C. grandiflora</em>, all three sampling strategies result in negative Tajima’s D (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>). A negative Tajima’s D is not expected with any sampling strategy if the model only includes population structure (<a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">De and Durrett 2007</a>; <a id="xref-ref-31-25" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> 2009</a>). However, if a model includes both population structure and expansion, overall negative Tajima’s D values are expected under some circumstances (<a id="xref-ref-31-26" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> 2009</a>).</p><p id="p-47">Simulations showed that relatively low values of the time since expansion τ (&lt;10) and moderate values of the expansion factor β (&gt;2 in pooled and &gt;6 in local) can lead to values of Tajima’s D observed in <em>C. grandiflora</em>. A τ = 2, for example, corresponds to an expansion 8N<sub>0</sub> generations ago, or an expansion starting about 2.5–4 Mya, assuming N<sub>0</sub> = 300,000-500,000 and β = 6, which corresponds to a 6-fold increase of the ancestral population size. This seems consistent with the results of <a id="xref-ref-29-21" class="xref-bibr" href="#ref-29">St. Onge <em>et al.</em> (2011)</a> that detected no departure from the standard neutral model or evidence of limited population expansion in the three <em>C. grandiflora</em> clusters. Also the ABC analysis (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Tables 2</a> and <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">3</a>) indicates no strong departure from the standard neutral model, as probabilities across models for each subsample are very similar and none would confer a Bayes factor above 3 (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Kass and Raftery 1995</a>), which normally leads, because of convenience, to the acceptance of the simplest model. Compared with <em>C. rubella</em>, <em>C. grandiflora</em> is clearly much closer to the standard neutral model, and strong inferences about its demographic history are therefore more difficult to make. In such cases, it is possible that also a weak effect of sampling strategy could lead to erroneous conclusions. In our case, there is a trend toward higher probabilities for the bottleneck model under local and pooled sampling, which is absent with the scattered sampling (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Tables 2</a> and <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">3</a>). This is in line with the results of <a id="xref-ref-1-6" class="xref-bibr" href="#ref-1">Chikhi <em>et al.</em> (2010)</a> that population structure can result in false bottleneck signals. With higher power, the effect of sampling may overcome the weak demographic signal in <em>C. grandiflora</em>.</p><p id="p-48">The effects of background selection in <em>C. grandiflora</em> are clear when comparing the synonymous and total sites Tajima’s D values (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>). Tajima’s D in all three sampling schemes is closer to zero when considering only synonymous sites. This change is not seen in <em>C. rubella</em>; although the Tajima’s D values change somewhat between the total sites data and synonymous data, the values remain near 0.5 (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3</a>). This is likely a result of the enormous difference in effective population sizes between the two species. The effective population size of <em>C. grandiflora</em> is estimated to be at least an order of magnitude higher than that of <em>C. rubella</em> (<a id="xref-ref-9-6" class="xref-bibr" href="#ref-9">Foxe <em>et al.</em> 2009</a>, <a id="xref-ref-29-22" class="xref-bibr" href="#ref-29">St. Onge <em>et al.</em> 2011</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">2012</a>). Accordingly, studies of the difference in selective effects in these species have demonstrated efficient positive and purifying selection in <em>C. grandiflora</em> (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Slotte <em>et al.</em> 2010</a>) and a diminished efficiency of selection in <em>C. rubella</em> (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Qiu <em>et al.</em> 2011</a>). It is possible that part or all of the weak expansion signal we detect in <em>C.grandiflora</em> is a results of background selection on codon usage bias, which would affect synonymous sites (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Qiu <em>et al.</em> 2011</a>).</p></div><div id="sec-16" class="subsection"><h3>Sampling effect on Tajima’s D is limited with few loci</h3><p id="p-49">A difference between the simulations described above and those in <a id="xref-ref-31-27" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> (2009)</a> is the number of simulated loci. The simulations of <a id="xref-ref-31-28" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> (2009)</a> were based on 1000 unlinked loci (independent replicates), whereas our empirical tests used only 13–16 loci, and in our simulations we varied the number of loci. We wanted to make the simulations comparable not only with our empirical data but also with other studies. Even though sequencing data are increasingly easy and inexpensive to accumulate, most studies on nonmodel species are still conducted using a limited number of loci. As a lower number of loci naturally results in lower power, what might be seen as an important significant effect at 1000 loci might not be so at 10 or 20. The lack of significant differences between the different sampling strategies observed in our study (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a>) and previous empirical studies may thus simply reflect the low number of loci used (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Li <em>et al.</em> 2010</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Wheat <em>et al.</em> 2010</a>), suggesting that even though sampling can definitely have an impact in structured populations, it might not be crucial when the number of loci is low and will in several cases be overpowered by the effect of demography. This is demonstrated in our simulations in which we observe a large amount of variation in Tajima’s D at lower numbers of loci, and the estimations of Tajima’s D with different sampling strategies are largely overlapping. This pattern is found in simulations with 16 and 32 loci and, to a lesser extent, at a higher number of loci (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4</a>).</p></div><div id="sec-17" class="subsection"><h3>Hierarchical structure and expansion within subpopulations</h3><p id="p-50">We found that simple departures from the models implemented by <a id="xref-ref-31-29" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> (2009)</a> altered the conclusions. For example, with the inclusion of expansion within demes as well as global expansion, the power to detect expansion with local samples increases and negative Tajima’s D is expected at lower levels of migration (4Nm &lt; 5; <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6A</a>). In the simulations by <a id="xref-ref-31-30" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> (2009)</a> that only included global expansion, Tajima’s D is expected to be near zero or positive for the same levels of migration; in other words, local sampling is detecting signals only at a local level in strongly subdivided populations. In general, when we add local expansion or hierarchical structure to the global expansion model of <a id="xref-ref-31-31" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> (2009)</a>, the differences among the three different sampling strategies decrease at higher migration rates. At 4Nm above 10 in the local expansion model or 2 in the hierarchical model, the three sampling strategies result in very similar values of Tajima’s D. The range of populations to which this could apply should be rather large, as a migration rate of 4Nm = 2 or above should correspond to an F<sub>ST</sub> between 0 and 0.33 at equilibrium. F<sub>ST</sub>, or G<sub>ST</sub>, in this range is very common among outcrossers (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hamrick and Godt 1996</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Nybom and Bartish 2000</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Nybom 2004</a>).</p><p id="p-51">In the same vein, we note that the many-deme model may not always be appropriate; for example, STRUCTURE analysis of <em>C. grandiflora</em> provides evidence for only three major regional clusters, with no evidence in the present case of further clustering within regions. If this is accurate, “scattered” samples combining more “populations” within a region may rather be the equivalent of increasing the sampling within populations. If we would want true scattered sampling from genetically differentiated populations in this species, we would have to be content with a very low sample size, in this case, three. Even if there are more than three genetic populations, perhaps in unsampled regions or in regions where low sample sizes limit the power to detect new genetic clusters, the number of clusters is most likely small, as we have sampled a rather large part of the geographic distribution of <em>C. grandiflora</em>. The case of <em>C. grandiflora</em> exemplifies two potentially problematic aspects of using scattered sampling in natural populations. First, without prior information, it can be easy to design a scattered sampling scheme that turns out to be a pooled sampling scheme if you consider genetic populations. Second, if you want to base your sampling scheme on population genetic structure, you need a good dense sampling to have the power to detect the population genetic structure, and only then can you pick a smaller subsample according to the scattered sampling scheme.</p></div></div><div class="section conclusions" id="sec-18"><h2 class="">Conclusions</h2><p id="p-52">Both the empirical data, based on two datasets with markedly different population structure and demographic history, and the simulated data presented here suggest that the effect of sampling on the site frequency spectrum is limited in many cases. For instance, if populations have experienced large changes in demography, such as the recent large change in population size in <em>C. rubella</em>, the effect of demography overpowers the effect of sampling. In addition, the effect of sampling is expected to be small if migration is not too limited (F<sub>ST</sub> between 0 and 0.33) and there is a hierarchical population structure or local expansion (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6</a>), which should be common in natural populations. Additionally, the number of sequenced loci is important; as our simulations show the effect of sampling is low in datasets with &lt;60 loci (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4</a>).</p><p id="p-53">However, it might not always be easy to determine if sampling strategy will have an impact, as many of the factors determining this are not known in advance. In some parts of the parameter space under the models explored here, sampling will definitely have a large effect on Tajima’s D and, consequently, on inferences of population history or selection. Examples of what factors are important are not only the presence of expansion or population structure <em>per se</em>, but also the form of expansion (within deme or increase in the number of demes), the amount of population differentiation (migration rate), and type of population structure (island model/hierarchical model), and these factors are generally not prior knowledge. Thus. the results of <a id="xref-ref-31-32" class="xref-bibr" href="#ref-31">Städler <em>et al.</em> (2009)</a> may not invalidate the conclusions of earlier empirical studies, but they certainly encourage caution when choosing a sampling strategy.</p><p id="p-54">To conclude, even though we failed to detect an effect of sampling on the site frequency spectrum as strong as that suggested by earlier theoretical studies, we are not advocating the neglect of sampling issues. We would, however, like to caution against using only a scattered sampling strategy. Only a good coverage of the natural range, using both within- and between-population samples and a large number of loci, are likely to lead to reliable inferences of species population structure and demographic history and, thereby, to correct conclusions on adaptation and the evolutionary history of the species, especially in species having experienced recent range expansions (<em>e.g.</em> <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Colautti <em>et al.</em> 2010</a>). In fact, our simulation analyses highlight that contrasting diversity patterns with different sampling strategies can be quite useful when distinguishing equilibrium population structure with gene flow from nonequilibrium patterns of divergence and population size change. Future studies should further investigate how the patterns from multiple sampling strategies might be built into coalescent approaches fitting demographic models. Also, in species that are likely to have experienced significant intensity of selection, it would be better to restrict the analysis to the sites that are less likely to have been under purifying selection, such as synonymous sites or noncoding sequences far from genes. With the falling cost of genomic surveys, it will be easier to reach those goals, even in nonmodel species.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-55">We thank Aldo Musacchio, Domenico Gargano, Salvatore Cozzolino, Santiago González Martínez, Alexis Ducousso, Myriam Heuertz, Fazia Krouchi, and Tanja Slotte for help with sampling. This work was supported by Erik Philip-Sörensens Stiftelse, Nilsson-Ehle-fonden, Carl Tryggers stiftelse, FORMAS, and the Royal Swedish Academy of Sciences. M. Lascoux thanks the Chinese Academy of Sciences and the Swedish Research Council for financial support and Li Haipeng for his hospitality in Shanghai.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></li><li class="fn" id="fn-2"><p id="p-2">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002410/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002410/-/DC1</a></p></li><li class="fn" id="fn-6"><p id="p-56"><em>Communicating editor: Y. Kim</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-02-29"><span class="received-label">Received </span>February 29, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-05-10"><span class="accepted-label">Accepted </span>May 10, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 St. Onge <em>et al.</em></li></ul><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.1" data-doi="10.1534/genetics.110.118661"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chikhi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sousa</span>  <span class="cit-name-given-names">V. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luisi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goossens</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beaumont</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The confounding effects of population structure, genetic diversity and the sampling scheme on the detection and quantification of population size changes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">983</span>–<span class="cit-lpage">995</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D186%26rft.spage%253D983%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118661%26rft_id%253Dinfo%253Apmid%252F20739713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.110.118661&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20739713&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283996100017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.2" data-doi="10.1098/rspb.2009.2231"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Colautti</span>  <span class="cit-name-given-names">R. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">S. C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Evolutionary constraints on adaptive evolution during range expansion in an invasive plant</span>. <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">277</span>: <span class="cit-fpage">1799</span>–<span class="cit-lpage">1806</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2009.2231%26rft_id%253Dinfo%253Apmid%252F20164098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMzoiMjc3LzE2ODkvMTc5OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNy84MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durrett</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Stepping-stone spatial structure causes slow decay of linkage disequilibrium and shifts the site frequency spectrum</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">969</span>–<span class="cit-lpage">981</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D176%26rft.spage%253D969%26rft_id%253Dinfo%253Apmid%252F17409067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17409067&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247870700022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.4" data-doi="10.1534/genetics.107.076943"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Mita</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronfort</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKhann</span>  <span class="cit-name-given-names">H. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poncet</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Investigation of the demographic and selective forces shaping the nucleotide diversity of genes involved in nod factor signalling in Medicago truncatula</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2123</span>–<span class="cit-lpage">2133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D177%26rft.spage%253D2123%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.076943%26rft_id%253Dinfo%253Apmid%252F18073426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.107.076943&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18073426&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251949800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.5" data-doi="10.1111/j.1365-294X.2005.02553.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evanno</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regnaut</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2611</span>–<span class="cit-lpage">2620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DEvanno%26rft.auinit1%253DG.%26rft.volume%253D14%26rft.issue%253D8%26rft.spage%253D2611%26rft.epage%253D2620%26rft.atitle%253DDetecting%2Bthe%2Bnumber%2Bof%2Bclusters%2Bof%2Bindividuals%2Busing%2Bthe%2Bsoftware%2BSTRUCTURE%253A%2Ba%2Bsimulation%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2005.02553.x%26rft_id%253Dinfo%253Apmid%252F15969739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2005.02553.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15969739&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229961500029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2.7.803.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smouse</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quattro</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Analysis of molecular variance inferred from metric distances among DNA haplotypes: application to human mitochondrial DNA restriction data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D131%26rft.spage%253D479%26rft_id%253Dinfo%253Apmid%252F1644282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=1644282&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HW75900021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laval</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Arlequin (version 3.0): an integrated software package for population genetics data analysis</span>. <abbr class="cit-jnl-abbrev">Evol. Bioinform.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BBioinform.%26rft.volume%253D1%26rft.spage%253D47%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000207065900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C.-I.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Hitchhiking under positive Darwinian selection</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">1405</span>–<span class="cit-lpage">1413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D155%26rft.spage%253D1405%26rft_id%253Dinfo%253Apmid%252F10880498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10880498&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088056800036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foxe</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slotte</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stahl</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neuffer</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurka</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Recent speciation associated with the evolution of selfing in Capsella</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">5241</span>–<span class="cit-lpage">5245</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.volume%253D10%26rft.spage%253D5241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Statistical tests of neutrality of mutations</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">693</span>–<span class="cit-lpage">709</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D133%26rft.spage%253D693%26rft_id%253Dinfo%253Apmid%252F8454210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8454210&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993KP81600022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.11" data-doi="10.1073/pnas.0808012106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Y.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bechsgaard</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slotte</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neuffer</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lascoux</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Recent speciation of Capsella rubella from Capsella grandiflora, associated with loss of self-incompatibility and an extreme bottleneck</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">5246</span>–<span class="cit-lpage">5251</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0808012106%26rft_id%253Dinfo%253Apmid%252F19307580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA2LzEzLzUyNDYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzcvODAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.12" data-doi="10.1098/rstb.1996.0112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamrick</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Godt</span>  <span class="cit-name-given-names">M. J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Effects of life history traits on genetic diversity in plant species</span>. <abbr class="cit-jnl-abbrev">Philos. T. Roy. Soc. B</abbr> <span class="cit-vol">351</span>: <span class="cit-fpage">1291</span>–<span class="cit-lpage">1298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DHamrick%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D351%26rft.issue%253D1345%26rft.spage%253D1291%26rft.epage%253D1298%26rft.atitle%253DEffects%2Bof%2BLife%2BHistory%2BTraits%2Bon%2BGenetic%2BDiversity%2Bin%2BPlant%2BSpecies%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.1996.0112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEzOiIzNTEvMTM0NS8xMjkxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi83LzgwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.13" data-doi="10.2307/2291091"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kass</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raftery</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">773</span>–<span class="cit-lpage">795</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D90%26rft.spage%253D773%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2291091%26rft_id%253Dinfo%253Apmid%252F10783804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2291091&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10783804&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RA10400045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lewontin</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Twenty-five years ago in Genetics: electrophoresis in the development of evolutionary genetics: milestone or millstone?</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">657</span>–<span class="cit-lpage">662</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLewontin%26rft.auinit1%253DR.%2BC.%26rft.volume%253D128%26rft.issue%253D4%26rft.spage%253D657%26rft.epage%253D662%26rft.atitle%253DTwenty-five%2Byears%2Bago%2Bin%2BGenetics%253A%2Belectrophoresis%2Bin%2Bthe%2Bdevelopment%2Bof%2Bevolutionary%2Bgenetics%253A%2Bmilestone%2Bor%2Bmillstone%253F%26rft_id%253Dinfo%253Apmid%252F1916239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo5OiIxMjgvNC82NTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzcvODAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.15" data-doi="10.1093/molbev/msp301"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stocks</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hemmilä</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kallman</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Demographic histories of four spruce (Picea) species of the Qinghai-Tibetan Plateau and neighboring areas inferred from multiple nuclear loci</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1001</span>–<span class="cit-lpage">1014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp301%26rft_id%253Dinfo%253Apmid%252F20031927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI3LzUvMTAwMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNy84MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.16" data-doi="10.1093/bioinformatics/btp187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Librado</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozas</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">DnaSP v5: a software for comprehensive analysis of DNA polymorphism data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1451</span>–<span class="cit-lpage">1452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp187%26rft_id%253Dinfo%253Apmid%252F19346325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTEvMTQ1MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNy84MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.17" data-doi="10.1016/j.meegid.2009.10.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beaumont</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">ABC: a useful Bayesian tool for the analysis of population data</span>. <abbr class="cit-jnl-abbrev">Infect. Genet. Evol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">825</span>–<span class="cit-lpage">832</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfect.%2BGenet.%2BEvol.%26rft.volume%253D10%26rft.spage%253D825%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.meegid.2009.10.010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.meegid.2009.10.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.18" data-doi="10.1111/j.1365-294X.2004.02141.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nybom</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Comparison of different nuclear DNA markers for estimating intraspecific genetic diversity in plants</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1143</span>–<span class="cit-lpage">1155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DNybom%26rft.auinit1%253DH.%26rft.volume%253D13%26rft.issue%253D5%26rft.spage%253D1143%26rft.epage%253D1155%26rft.atitle%253DComparison%2Bof%2Bdifferent%2Bnuclear%2BDNA%2Bmarkers%2Bfor%2Bestimating%2Bintraspecific%2Bgenetic%2Bdiversity%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2004.02141.x%26rft_id%253Dinfo%253Apmid%252F15078452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2004.02141.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15078452&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221016300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.19" data-doi="10.1078/1433-8319-00006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nybom</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartish</span>  <span class="cit-name-given-names">I. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Effects of life history traits and sampling strategies on genetic diversity estimates obtained with RAPD markers in plants</span>. <abbr class="cit-jnl-abbrev">Perspect. Plant Ecol. Evol. Syst.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">93</span>–<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPerspect.%2BPlant%2BEcol.%2BEvol.%2BSyst.%26rft.volume%253D3%26rft.spage%253D93%26rft_id%253Dinfo%253Adoi%252F10.1078%252F1433-8319-00006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1078/1433-8319-00006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.20" data-doi="10.1111/j.1365-294X.2010.04783.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peter</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegmann</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Distinguishing between population bottleneck and population subdivision by a Bayesian model choice procedure</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">4648</span>–<span class="cit-lpage">4660</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DPeter%26rft.auinit1%253DB.%2BM.%26rft.volume%253D19%26rft.issue%253D21%26rft.spage%253D4648%26rft.epage%253D4660%26rft.atitle%253DDistinguishing%2Bbetween%2Bpopulation%2Bbottleneck%2Band%2Bpopulation%2Bsubdivision%2Bby%2Ba%2BBayesian%2Bmodel%2Bchoice%2Bprocedure.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2010.04783.x%26rft_id%253Dinfo%253Apmid%252F20735743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2010.04783.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20735743&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283163700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D155%26rft.spage%253D945%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10835412&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087475100039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Przeworski</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The signature of positive selection at randomly chosen loci</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">1179</span>–<span class="cit-lpage">1189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D160%26rft.spage%253D1179%26rft_id%253Dinfo%253Apmid%252F11901132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11901132&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174790600030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.23" data-doi="10.1016/S0168-9525(02)02781-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ptak</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przeworski</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Evidence for population growth in humans is confounded by fine-scale population structure</span>. <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">563</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DPtak%26rft.auinit1%253DS.%2BE.%26rft.volume%253D18%26rft.issue%253D11%26rft.spage%253D559%26rft.epage%253D563%26rft.atitle%253DEvidence%2Bfor%2Bpopulation%2Bgrowth%2Bin%2Bhumans%2Bis%2Bconfounded%2Bby%2Bfine-scale%2Bpopulation%2Bstructure.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252802%252902781-6%26rft_id%253Dinfo%253Apmid%252F12414185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(02)02781-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12414185&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178796300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.24" data-doi="10.1093/gbe/evr085"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slotte</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Reduced efficacy of natural selection on codon usage bias in selfing Arabidopsis and Capsella species</span>. <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">868</span>–<span class="cit-lpage">880</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevr085%26rft_id%253Dinfo%253Apmid%252F21856647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjMvMC84NjgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzcvODAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.25" data-doi="10.1093/molbev/msg009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ray</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currat</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Intra-Deme molecular diversity in spatially expanding populations</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">76</span>–<span class="cit-lpage">86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DRay%26rft.auinit1%253DN.%26rft.volume%253D20%26rft.issue%253D1%26rft.spage%253D76%26rft.epage%253D86%26rft.atitle%253DIntra-Deme%2BMolecular%2BDiversity%2Bin%2BSpatially%2BExpanding%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsg009%26rft_id%253Dinfo%253Apmid%252F12519909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjc6IjIwLzEvNzYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzcvODAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Gene frequency distributions as a test of selective neutrality</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">775</span>–<span class="cit-lpage">785</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D81%26rft.spage%253D775%26rft_id%253Dinfo%253Apmid%252F1213275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=1213275&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simonsen</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aquadro</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Properties of statistical tests of neutrality for DNA polymorphism data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">429</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D141%26rft.spage%253D413%26rft_id%253Dinfo%253Apmid%252F8536987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8536987&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RR17100035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.28" data-doi="10.1093/molbev/msq062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slotte</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foxe</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hazzouri</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide evidence for efficient positive and purifying selection in Capsella grandiflora, a plant species with a large effective population size</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1813</span>–<span class="cit-lpage">1821</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq062%26rft_id%253Dinfo%253Apmid%252F20194429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI3LzgvMTgxMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNy84MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.29" data-doi="10.1111/j.1365-294X.2011.05189.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">St. Onge</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Källman</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slotte</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lascoux</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmé</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Contrasting demographic history and population structure in Capsella rubella and Capsella grandiflora, two closely related species with different mating systems</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3306</span>–<span class="cit-lpage">3320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DSt%2BOnge%26rft.auinit1%253DK.%2BR.%26rft.volume%253D20%26rft.issue%253D16%26rft.spage%253D3306%26rft.epage%253D3320%26rft.atitle%253DContrasting%2Bdemographic%2Bhistory%2Band%2Bpopulation%2Bstructure%2Bin%2BCapsella%2Brubella%2Band%2BCapsella%2Bgrandiflora%252C%2Btwo%2Bclosely%2Brelated%2Bspecies%2Bwith%2Bdifferent%2Bmating%2Bsystems.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2011.05189.x%26rft_id%253Dinfo%253Apmid%252F21777317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2011.05189.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21777317&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293750000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">St. Onge</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foxe</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Junrui</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haipeng</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holm</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Coalescent-based analysis distinguishes between allo- and autopolyploid origin in shepherd’s purse (Capsella bursa-pastoris)</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol. 10.1093/molbev/mss024</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.31" data-doi="10.1534/genetics.108.094904"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Städler</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haubold</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merino</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfaffelhuber</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The impact of sampling schemes on the site frequency spectrum in nonequilibrium subdivided populations</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">205</span>–<span class="cit-lpage">216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.094904%26rft_id%253Dinfo%253Apmid%252F19237689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMjA1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi83LzgwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.32" data-doi="10.1086/379378"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A comparison of Bayesian methods for haplotype reconstruction from population genotype data</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">1162</span>–<span class="cit-lpage">1169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DStephens%26rft.auinit1%253DM.%26rft.volume%253D73%26rft.issue%253D5%26rft.spage%253D1162%26rft.epage%253D1169%26rft.atitle%253DA%2Bcomparison%2Bof%2Bbayesian%2Bmethods%2Bfor%2Bhaplotype%2Breconstruction%2Bfrom%2Bpopulation%2Bgenotype%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F379378%26rft_id%253Dinfo%253Apmid%252F14574645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/379378&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14574645&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186493400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.33" data-doi="10.1086/319501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A new statistical method for haplotype reconstruction from population data</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">978</span>–<span class="cit-lpage">989</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DStephens%26rft.auinit1%253DM.%26rft.volume%253D68%26rft.issue%253D4%26rft.spage%253D978%26rft.epage%253D989%26rft.atitle%253DA%2Bnew%2Bstatistical%2Bmethod%2Bfor%2Bhaplotype%2Breconstruction%2Bfrom%2Bpopulation%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F319501%26rft_id%253Dinfo%253Apmid%252F11254454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/319501&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11254454&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167666000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Statistical method for testing the neutral mutation hypothesis by DNA polymorphism</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D123%26rft.spage%253D585%26rft_id%253Dinfo%253Apmid%252F2513255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2513255&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989AX26700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.35" data-doi="10.1093/bioinformatics/btg316"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Libsequence: a C++ class library for evolutionary genetic analysis</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">2325</span>–<span class="cit-lpage">2327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DThornton%26rft.auinit1%253DK.%26rft.volume%253D19%26rft.issue%253D17%26rft.spage%253D2325%26rft.epage%253D2327%26rft.atitle%253Dlibsequence%253A%2Ba%2BC%252B%252B%2Bclass%2Blibrary%2Bfor%2Bevolutionary%2Bgenetic%2Banalysis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg316%26rft_id%253Dinfo%253Apmid%252F14630667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMTkvMTcvMjMyNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNy84MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.36" data-doi="10.1534/genetics.105.048223"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Approximate Bayesian inference reveals evidence for a recent, severe bottleneck in a Netherlands population of Drosophila melanogaster</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">1607</span>–<span class="cit-lpage">1619</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D172%26rft.spage%253D1607%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.048223%26rft_id%253Dinfo%253Apmid%252F16299396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.105.048223&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16299396&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236668100020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wakeley</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Nonequilibrium migration in human history</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">1863</span>–<span class="cit-lpage">1871</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D153%26rft.spage%253D1863%26rft_id%253Dinfo%253Apmid%252F10581291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10581291&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084079300032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.38" data-doi="10.1006/tpbi.2000.1495"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wakeley</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The coalescent in an island model of population subdivision with variation among demes</span>. <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DWakeley%26rft.auinit1%253DJ.%26rft.volume%253D59%26rft.issue%253D2%26rft.spage%253D133%26rft.epage%253D144%26rft.atitle%253DThe%2Bcoalescent%2Bin%2Ban%2Bisland%2Bmodel%2Bof%2Bpopulation%2Bsubdivision%2Bwith%2Bvariation%2Bamong%2Bdemes.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ftpbi.2000.1495%26rft_id%253Dinfo%253Apmid%252F11302758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/tpbi.2000.1495&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11302758&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168456000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wakeley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aliacar</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Gene genealogies in a metapopulation</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">893</span>–<span class="cit-lpage">905</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D159%26rft.spage%253D893%26rft_id%253Dinfo%253Apmid%252F11606561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11606561&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171744900039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.40" data-doi="10.1016/0040-5809(75)90020-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watterson</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">On the number of segregating sites in genetical models without recombination</span>. <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">256</span>–<span class="cit-lpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DWatterson%26rft.auinit1%253DG.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D256%26rft.epage%253D276%26rft.atitle%253DOn%2Bthe%2Bnumber%2Bof%2Bsegregating%2Bsites%2Bin%2Bgenetical%2Bmodels%2Bwithout%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252875%252990020-9%26rft_id%253Dinfo%253Apmid%252F1145509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(75)90020-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1145509&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975W295600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.41" data-doi="10.1093/molbev/msp227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wheat</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haag</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marden</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanski</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frilander</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Nucleotide polymorphism at a gene (Pgi) under balancing selection in a butterfly metapopulation</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp227%26rft_id%253Dinfo%253Apmid%252F19793833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI3LzIvMjY3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi83LzgwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1931</span> <span class="cit-article-title">Evolution in Mendelian populations</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D16%26rft.spage%253D97%26rft_id%253Dinfo%253Apmid%252F17246615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17246615&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1951</span> <span class="cit-article-title">The genetical structure of populations</span>. <abbr class="cit-jnl-abbrev">Ann. Eugen.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BEugen.%26rft.volume%253D15%26rft.spage%253D323%26rft_id%253Dinfo%253Apmid%252F24540312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24540312&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1951XY09200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.44" data-doi="10.2307/2406450"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1965</span> <span class="cit-article-title">The interpretation of population structure by F-statistics with special regard to systems of mating</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">395</span>–<span class="cit-lpage">420</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D19%26rft.spage%253D395%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2406450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2406450&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1965CLA4300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2.7.803.45" data-doi="10.1534/genetics.111.130575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The joint effects of background selection and genetic recombination on local gene genealogies</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D189%26rft.spage%253D251%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.130575%26rft_id%253Dinfo%253Apmid%252F21705759%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.111.130575&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21705759&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294721600020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.803.46" data-doi="10.1534/genetics.106.061432"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y.-X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C.-I.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Statistical tests for detecting positive selection by utilizing high-frequency variants</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">1431</span>–<span class="cit-lpage">1439</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D174%26rft.spage%253D1431%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.061432%26rft_id%253Dinfo%253Apmid%252F16951063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.106.061432&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16951063&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F803.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242532600031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/7/803.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/7/789" title="Comparison of Parallel High-Throughput RNA Sequencing Between Knockout of TDP-43 and Its Overexpression Reveals Primarily Nonreciprocal and Nonoverlapping Gene Expression Changes in the Central Nervous System of Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/7/815" title="A Sequence-Anchored Linkage Map of the Plant–Parasitic Nematode Meloidogyne hapla Reveals Exceptionally High Genome-Wide Recombination" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/7"><span>Volume 2 Issue 7, July 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/7.cover-source.jpg?itok=tq8UCk63" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/7/803" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jDrO3z8fU9aCvbjIwEgS5o1l3CWa5jpsXb9sSLKLLo4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/7/803" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/7/803" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/7/803" class="active" data-icon-position="" data-hide-link-title="0">Impact of Sampling Schemes on Demographic Inference: An Empirical Study in Two Species with Different Mating Systems and Demographic Histories</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459302" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-e6v_JIMxz74Q-Vc_ZEoJM6vgubVRiVUZ4ADubQz72T4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/7/803.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/7/803" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-1nGhv0dICavZsc10tJu7s0gYNvYPa6r-AvpLJYiSGwY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/7/803" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/7/803.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459302" id="node459302--2" data-pisa="ggg;2/7/803" data-pisa-master="ggg;2/7/803" data-apath="/ggg/2/7/803.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Impact of Sampling Schemes on Demographic Inference: An Empirical Study in Two Species with Different Mating Systems and Demographic Histories</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">K. R. St. Onge</span>, <span class="highwire-citation-author" data-delta="1">A. E. Palmé</span>, <span class="highwire-citation-author" data-delta="2">S. I. Wright</span> and <span class="highwire-citation-author" data-delta="3">M. Lascoux</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">803-814;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002410 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;2/7/803" data-atom-uri="/ggg/2/7/803.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459302--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">K. R. St. Onge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, Uppsala University, SE-75236 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=K.%2BR.%2BSt.%2BOnge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=St.%20Onge%20KR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AK.%2BR.%2BSt.%2BOnge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:K.R.St.Onge@uu.nl" class="" data-icon-position="" data-hide-link-title="0">K.R.St.Onge@uu.nl</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">A. E. Palmé </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, Uppsala University, SE-75236 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=A.%2BE.%2BPalm%C3%A9%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palm%C3%A9%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AA.%2BE.%2BPalm%25C3%25A9%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">S. I. Wright </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology and Centre for Analysis of Genome Evolution and Function, University of Toronto, Toronto, Ontario M5S 3B2, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S.%2BI.%2BWright%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wright%20SI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AS.%2BI.%2BWright%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">M. Lascoux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, Uppsala University, SE-75236 Uppsala, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory of Evolutionary Genomics, CAS-MPG Partner Institute for Computational Biology, Chinese Academy of Sciences, Shanghai 200031, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BLascoux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lascoux%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM.%2BLascoux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/7/803" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/7/803&amp;title=Impact%20of%20Sampling%20Schemes%20on%20Demographic%20Inference%3A%20An%20Empirical%20Study%20in%20Two%20Species%20with%20Different%20Mating%20Systems%20and%20Demographic%20Histories" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/7/803&amp;title=Impact%20of%20Sampling%20Schemes%20on%20Demographic%20Inference%3A%20An%20Empirical%20Study%20in%20Two%20Species%20with%20Different%20Mating%20Systems%20and%20Demographic%20Histories" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/7/803&amp;title=Impact%20of%20Sampling%20Schemes%20on%20Demographic%20Inference%3A%20An%20Empirical%20Study%20in%20Two%20Species%20with%20Different%20Mating%20Systems%20and%20Demographic%20Histories" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/7/803&amp;text=Impact%20of%20Sampling%20Schemes%20on%20Demographic%20Inference%3A%20An%20Empirical%20Study%20in%20Two%20Species%20with%20Different%20Mating%20Systems%20and%20Demographic%20Histories" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/7/803&amp;title=Impact%20of%20Sampling%20Schemes%20on%20Demographic%20Inference%3A%20An%20Empirical%20Study%20in%20Two%20Species%20with%20Different%20Mating%20Systems%20and%20Demographic%20Histories" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/7/803&amp;t=Impact%20of%20Sampling%20Schemes%20on%20Demographic%20Inference%3A%20An%20Empirical%20Study%20in%20Two%20Species%20with%20Different%20Mating%20Systems%20and%20Demographic%20Histories" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/7/803&amp;title=Impact%20of%20Sampling%20Schemes%20on%20Demographic%20Inference%3A%20An%20Empirical%20Study%20in%20Two%20Species%20with%20Different%20Mating%20Systems%20and%20Demographic%20Histories" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/7/803&amp;title=Impact%20of%20Sampling%20Schemes%20on%20Demographic%20Inference%3A%20An%20Empirical%20Study%20in%20Two%20Species%20with%20Different%20Mating%20Systems%20and%20Demographic%20Histories" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459302/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459302" id="node459302--3" data-pisa="ggg;2/7/803" data-pisa-master="ggg;2/7/803" data-apath="/ggg/2/7/803.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Impact of Sampling Schemes on Demographic Inference: An Empirical Study in Two Species with Different Mating Systems and Demographic Histories</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">K. R. St. Onge</span>, <span class="highwire-citation-author" data-delta="1">A. E. Palmé</span>, <span class="highwire-citation-author" data-delta="2">S. I. Wright</span> and <span class="highwire-citation-author" data-delta="3">M. Lascoux</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">803-814;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002410 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;2/7/803" data-atom-uri="/ggg/2/7/803.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459302--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">K. R. St. Onge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, Uppsala University, SE-75236 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=K.%2BR.%2BSt.%2BOnge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=St.%20Onge%20KR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AK.%2BR.%2BSt.%2BOnge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:K.R.St.Onge@uu.nl" class="" data-icon-position="" data-hide-link-title="0">K.R.St.Onge@uu.nl</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">A. E. Palmé </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, Uppsala University, SE-75236 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=A.%2BE.%2BPalm%C3%A9%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palm%C3%A9%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AA.%2BE.%2BPalm%25C3%25A9%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">S. I. Wright </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology and Centre for Analysis of Genome Evolution and Function, University of Toronto, Toronto, Ontario M5S 3B2, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S.%2BI.%2BWright%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wright%20SI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AS.%2BI.%2BWright%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">M. Lascoux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, Uppsala University, SE-75236 Uppsala, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory of Evolutionary Genomics, CAS-MPG Partner Institute for Computational Biology, Chinese Academy of Sciences, Shanghai 200031, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BLascoux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lascoux%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM.%2BLascoux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=803&amp;rft.epage=814&amp;rft.atitle=Impact%20of%20Sampling%20Schemes%20on%20Demographic%20Inference%3A%20An%20Empirical%20Study%20in%20Two%20Species%20with%20Different%20Mating%20Systems%20and%20Demographic%20Histories&amp;rft.volume=2&amp;rft.issue=7&amp;rft.date=2012-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=St.+Onge%2C+K.+R.&amp;rft.au=Palm%C3%A9%2C+A.+E.&amp;rft.au=Wright%2C+S.+I.&amp;rft.au=Lascoux%2C+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459302/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459302/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459302/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459302/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459302/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459302/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459302/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459302/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459302/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459302/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459302/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459302/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-ajax-pane-content pane-highwire-permission-link" >
  
        <h2 class="pane-title"><span class='panels-ajax-pane-title' data-pid='new-37'></span></h2>
    
  
  <div class="pane-content">
    <div class='panels-ajax-pane panels-ajax-pane-new-37' data-pid='new-37'></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/7/803" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/7/803.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/7/803.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/7/803.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/7/803.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
