<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/7/779" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/7/779.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/7/779.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/7/779.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/2/7/779/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/7/779.atom" />
<meta name="HW.pisa" content="ggg;2/7/779" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genotype by Environment Interaction of Quantitative Traits: A Case Study in Barley" />
<meta name="DC.Identifier" content="10.1534/g3.112.002980" />
<meta name="DC.Date" content="2012-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Zhao, Xu" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Genotype by environment interaction is a phenomenon that a better genotype in one environment may perform poorly in another environment. When the genotype refers to a quantitative trait locus (QTL), this phenomenon is called QTL by environment interaction, denoted by Q×E. Using a recently developed new Bayesian method and genome-wide marker information, we estimated and tested QTL main effects and Q×E interactions for a well-known barley dataset produced by the North American Barley Genome Mapping Project. This dataset contained seven quantitative traits collected from 145 doubled-haploid (DH) lines evaluated in multiple environments, which derived from a cross between two Canadian two-row barley lines, Harrington and TR306. Numerous main effects and Q×E interaction effects have been detected for all seven quantitative traits. However, main effects seem to be more important than the Q×E interaction effects for all seven traits examined. The number of main effects detected varied from 26 for the maturity trait to 75 for the heading trait, with an average of 61.86. The heading trait has the most detected effects, with a total of 98 (75 main, 29 Q×E). Among the 98 effects, 6 loci had both the main and Q×E effects. Among the total number of detected loci, on average, 78.5% of the loci show the main effects whereas 34.9% of the loci show Q×E interactions. Overall, we detected many loci with either the main or the Q×E effects, and the main effects appear to be more important than the Q×E interaction effects for all the seven traits. This means that most detected loci have a constant effect across environments. Another discovery from this analysis is that Q×E interaction occurs independently, regardless whether the locus has main effects." />
<meta name="DC.Contributor" content="Fuping Zhao" />
<meta name="DC.Contributor" content="Shizhong Xu" />
<meta name="article:published_time" content="2012-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genotype by Environment Interaction of Quantitative Traits: A Case Study in Barley" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genotype by environment interaction is a phenomenon that a better genotype in one environment may perform poorly in another environment. When the genotype refers to a quantitative trait locus (QTL), this phenomenon is called QTL by environment interaction, denoted by Q×E. Using a recently developed new Bayesian method and genome-wide marker information, we estimated and tested QTL main effects and Q×E interactions for a well-known barley dataset produced by the North American Barley Genome Mapping Project. This dataset contained seven quantitative traits collected from 145 doubled-haploid (DH) lines evaluated in multiple environments, which derived from a cross between two Canadian two-row barley lines, Harrington and TR306. Numerous main effects and Q×E interaction effects have been detected for all seven quantitative traits. However, main effects seem to be more important than the Q×E interaction effects for all seven traits examined. The number of main effects detected varied from 26 for the maturity trait to 75 for the heading trait, with an average of 61.86. The heading trait has the most detected effects, with a total of 98 (75 main, 29 Q×E). Among the 98 effects, 6 loci had both the main and Q×E effects. Among the total number of detected loci, on average, 78.5% of the loci show the main effects whereas 34.9% of the loci show Q×E interactions. Overall, we detected many loci with either the main or the Q×E effects, and the main effects appear to be more important than the Q×E interaction effects for all the seven traits. This means that most detected loci have a constant effect across environments. Another discovery from this analysis is that Q×E interaction occurs independently, regardless whether the locus has main effects.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/07/01" />
<meta name="citation_mjid" content="ggg;2/7/779" />
<meta name="citation_id" content="2/7/779" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/7/779" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/7/779.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/7/779.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/7/779.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.002980" />
<meta name="citation_pmid" content="22870401" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="7" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="779" />
<meta name="citation_lastpage" content="788" />
<meta name="citation_author" content="Fuping Zhao" />
<meta name="citation_author_institution" content="Department of Botany and Plant Sciences, University of California, Riverside, California 92521" />
<meta name="citation_author" content="Shizhong Xu" />
<meta name="citation_author_institution" content="Department of Botany and Plant Sciences, University of California, Riverside, California 92521" />
<meta name="citation_author_email" content="shizhong.xu@ucr.edu" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. P. Boer;citation_author=D. Wright;citation_author=L. Feng;citation_author=D. W. Podlich;citation_author=L. Luo;citation_title=A mixed-model quantitative trait loci (QTL) analysis for multiple-environment trial data using environmental covariables for QTL-by-environment interactions, with an example in maize;citation_pages=1801-1813;citation_volume=177;citation_year=2007;citation_pmid=17947443;citation_doi=10.1534/genetics.107.071068" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Plant Genomics;citation_title=Significance test and genome selection in Bayesian shrinkage analysis;citation_author=X. Che;citation_author=S. Xu;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=X. Chen;citation_author=F. Zhao;citation_author=S. Xu;citation_title=Mapping environment specific quantitative trait loci;citation_pages=1053-1066;citation_volume=186;citation_year=2010;citation_pmid=20805558;citation_doi=10.1534/genetics.110.120311" />
<meta name="citation_reference" content="citation_journal_title=Journal of Plant Breeding and Crop Science;citation_author=M. Cherif;citation_author=S. Rezgui;citation_author=P. Devaux;citation_author=M. Harrabi;citation_title=Genotype × environment interactions and heritability of quantitative resistance to net blotch in Tunisian barley;citation_pages=110-116;citation_volume=2;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=P. Hayes;citation_author=B. Liu;citation_author=S. Knapp;citation_author=F. Chen;citation_author=B. Jones;citation_title=Quantitative trait locus effects and environmental interaction in a sample of North American barley germ plasm;citation_pages=392-401;citation_volume=87;citation_year=1993;citation_pmid=24190268;citation_doi=10.1007/BF01184929" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=R. Jansen;citation_author=J. Ooijen;citation_author=P. Stam;citation_author=C. Lister;citation_author=C. Dean;citation_title=Genotype-by-environment interaction in genetic mapping of multiple quantitative trait loci;citation_pages=33-37;citation_volume=91;citation_year=1995;citation_pmid=24169664;citation_doi=10.1007/BF00220855" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Jiang;citation_author=Z. B. Zeng;citation_title=Multiple trait analysis of genetic mapping for quantitative trait loci;citation_pages=1111-1127;citation_volume=140;citation_year=1995;citation_pmid=7672582" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=C. Jiang;citation_author=Z. B. Zeng;citation_title=Mapping quantitative trait loci with dominant and missing markers in various crosses from two inbred lines;citation_pages=47-58;citation_volume=101;citation_year=1997;citation_pmid=9465409;citation_doi=10.1023/A:1018394410659" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. S. Lander;citation_author=D. Botstein;citation_title=Mapping Mendelian factors underlying quantitative traits using RFLP linkage maps;citation_pages=185-199;citation_volume=121;citation_year=1989;citation_pmid=2563713" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Methods;citation_author=Q. Li;citation_author=Z. Huang;citation_author=M. Xu;citation_author=C. Wang;citation_author=J. Gai;citation_title=Functional mapping of genotype-environment interactions for soybean growth by a semiparametric approach.;citation_pages=13-13;citation_volume=6;citation_year=2010;citation_pmid=20525184;citation_doi=10.1186/1746-4811-6-13" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=Y. Li;citation_author=C. Wu;citation_author=G. Jiang;citation_author=L. Wang;citation_author=Y. He;citation_title=Dynamic analyses of rice blast resistance for the assessment of genetic and environmental effects;citation_pages=541-547;citation_volume=126;citation_year=2007;citation_doi=10.1111/j.1439-0523.2007.01409.x" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=A. Paterson;citation_author=Y. Saranga;citation_author=M. Menz;citation_author=C. X. Jiang;citation_author=R. Wright;citation_title=QTL analysis of genotype x environment interactions affecting cotton fiber quality.;citation_pages=384-396;citation_volume=106;citation_year=2003;citation_issue=3;citation_pmid=12589538" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. P. Piepho;citation_title=A mixed-model approach to mapping quantitative trait loci in barley on the basis of multiple environment data;citation_pages=2043-2050;citation_volume=156;citation_year=2000;citation_pmid=11102394" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=H. P. Piepho;citation_title=Statistical tests for QTL and QTL-by-environment effects in segregating populations derived from line crosses.;citation_pages=561-566;citation_volume=110;citation_year=2005;citation_issue=3;citation_pmid=15655665;citation_doi=10.1007/s00122-004-1872-9" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=K. Pillen;citation_author=A. Zacharias;citation_author=J. Leon;citation_title=Advanced backcross QTL analysis in barley (Hordeum vulgare L.).;citation_pages=340-352;citation_volume=107;citation_year=2003;citation_issue=2;citation_pmid=12677407;citation_doi=10.1007/s00122-003-1253-9" />
<meta name="citation_reference" content="citation_author=N. Tinker;citation_author=D. Mather;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=N. Tinker;citation_author=D. Mather;citation_author=B. Rossnagel;citation_author=K. Kasha;citation_author=A. Kleinhofs;citation_title=Regions of the Genome that Affect Agronomic Performance in Two-Row Barley;citation_pages=1053-1062;citation_volume=36;citation_year=1996;citation_issue=4;citation_doi=10.2135/cropsci1996.0011183X003600040040x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Agricultural Genomics;citation_title=PLABQTL: a program for composite interval mapping of QTL;citation_author=H. Utz;citation_author=A. Melchinger;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. F. Utz;citation_author=A. E. Melchinger;citation_author=C. C. Schon;citation_title=Bias and sampling error of the estimated proportion of genotypic variance explained by quantitative trait loci determined from experimental data in maize using cross validation and validation with independent samples;citation_pages=1839-1849;citation_volume=154;citation_year=2000;citation_pmid=10866652" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. L. Yang;citation_author=R. L. Jing;citation_author=X. P. Chang;citation_author=W. Li;citation_title=Identification of quantitative trait loci and environmental interactions for accumulation and remobilization of water-soluble carbohydrates in wheat (Triticum aestivum L.) stems;citation_pages=571-584;citation_volume=176;citation_year=2007;citation_pmid=17287530;citation_doi=10.1534/genetics.106.068361" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. B. Zeng;citation_title=Precision mapping of quantitative trait loci;citation_pages=1457-1468;citation_volume=136;citation_year=1994;citation_pmid=8013918" />
<meta name="twitter:title" content="Genotype by Environment Interaction of Quantitative Traits: A Case Study in Barley" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7.cover-source.jpg" />
<meta name="twitter:description" content="Genotype by environment interaction is a phenomenon that a better genotype in one environment may perform poorly in another environment. When the genotype refers to a quantitative trait locus (QTL), this phenomenon is called QTL by environment interaction, denoted by Q×E. Using a recently developed new Bayesian method and genome-wide marker information, we estimated and tested QTL main effects and Q×E interactions for a well-known barley dataset produced by the North American Barley Genome Mapping Project. This dataset contained seven quantitative traits collected from 145 doubled-haploid (DH) lines evaluated in multiple environments, which derived from a cross between two Canadian two-row barley lines, Harrington and TR306. Numerous main effects and Q×E interaction effects have been detected for all seven quantitative traits. However, main effects seem to be more important than the Q×E interaction effects for all seven traits examined. The number of main effects detected varied from 26 for the maturity trait to 75 for the heading trait, with an average of 61.86. The heading trait has the most detected effects, with a total of 98 (75 main, 29 Q×E). Among the 98 effects, 6 loci had both the main and Q×E effects. Among the total number of detected loci, on average, 78.5% of the loci show the main effects whereas 34.9% of the loci show Q×E interactions. Overall, we detected many loci with either the main or the Q×E effects, and the main effects appear to be more important than the Q×E interaction effects for all the seven traits. This means that most detected loci have a constant effect across environments. Another discovery from this analysis is that Q×E interaction occurs independently, regardless whether the locus has main effects." />
<meta name="og-title" property="og:title" content="Genotype by Environment Interaction of Quantitative Traits: A Case Study in Barley" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/7/779" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genotype by environment interaction is a phenomenon that a better genotype in one environment may perform poorly in another environment. When the genotype refers to a quantitative trait locus (QTL), this phenomenon is called QTL by environment interaction, denoted by Q×E. Using a recently developed new Bayesian method and genome-wide marker information, we estimated and tested QTL main effects and Q×E interactions for a well-known barley dataset produced by the North American Barley Genome Mapping Project. This dataset contained seven quantitative traits collected from 145 doubled-haploid (DH) lines evaluated in multiple environments, which derived from a cross between two Canadian two-row barley lines, Harrington and TR306. Numerous main effects and Q×E interaction effects have been detected for all seven quantitative traits. However, main effects seem to be more important than the Q×E interaction effects for all seven traits examined. The number of main effects detected varied from 26 for the maturity trait to 75 for the heading trait, with an average of 61.86. The heading trait has the most detected effects, with a total of 98 (75 main, 29 Q×E). Among the 98 effects, 6 loci had both the main and Q×E effects. Among the total number of detected loci, on average, 78.5% of the loci show the main effects whereas 34.9% of the loci show Q×E interactions. Overall, we detected many loci with either the main or the Q×E effects, and the main effects appear to be more important than the Q×E interaction effects for all the seven traits. This means that most detected loci have a constant effect across environments. Another discovery from this analysis is that Q×E interaction occurs independently, regardless whether the locus has main effects." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7.cover-source.jpg" />
    <title>Genotype by Environment Interaction of Quantitative Traits: A Case Study in Barley | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"q9nIuFsfvwP_SZsLqFa5C95LW4d2mOAJn7fMLdliso4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459299","apath":"\/ggg\/2\/7\/779.atom","pisa":"ggg;2\/7\/779","ac":{"\/ggg\/2\/7\/779.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/7\/779.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/7\/779"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/7\/779"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/7\/779":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022dqzsSQ4rwgTTW8y2r4fdpoPTLaAN1CsRMb6f063D8SqTckW+z0HM7vWYmzc3iMUtWPBRtrhHM6A\\\\\\\/7HXldnzaJ26DlxevHPZTN6PPAuNp0Fd1A3Cf\\\\\\\/3XsU40MbSvo+B2hNKDyeIDQG7kJ\\\\\\\/33I\\\\\\\/024oKWsBCs7tt9AJlbayzRpDZ0B3OLFjpiJ9FKSlLSSQOziW6bac1uVVKd5EtJHvyuB7FY8QS7HeXpCuLrwaOgillPskukYU2gFV3SM4l8iONB6m0bexMYstMohCWiVCOHjwaaHDytC5Ub8YQa02AoMQhqGyTE74JwAOWGs94kNfYsdYzJr6nNJzLhk2ebCuzgN\\\\\\\/0NzWGWMsBvUvaU5S9LUBLYIUHXTYWDInedyfAo1ops4VJ99frFEY+NEtHwhyYaNLpeFS\\\\\\\/UDUMdZH5BbEhx5CyvaGNm6krJ86wyuFk0HS35FZVYMCJshPKUguDZcz5rkiDs+gH2ZhjfiGFZdL6aDXdOVL8mnOX3bd65FZQoujT8z6Re2I\\\\\\\/vGuGWhkwGFKGWFj9lTQ6rrCYyF8G3Xd5HdF2D+E1+n2n7RLT1A2V3mFx5brbkp1p5bTJGAxUlSzkhzH94VSje\\\\\\\/DnGDJw6eYYg9jo5EZb8JQBcPf\\\\\\\/gghCvKjCUjpctbo3FyEarZyJ1pAXqWaqDun92ZP2qvHnnnZSkwIl3MqHLVQzxwrR18P2+7wbuqPvTnNC3cCCZgJ74+gCwHXXV8jVe8v5RmcxHUZamRnR8\\\\\\\/brQUTR5ldAwW1QuPUidKzlKluTwfrWgEVbTGxPKk0Ose0W52iplcStsQW5HGN7WFCZz+oqaaGgF8Q4qH0A55vfJ9XJkOQPRHTo75q2heCcjxpooEwbW61JHcpwPkuPONC4tasOQjLGnLBzqeLgnNyYqyGyzQSl6C5qerMKCaFymY9k2DhrYtM6EqCyvkx4T6umeEa\\\\\\\/z0g9BlHt+PQO5oc4YIg48zgp\\\\\\\/4lvXlVUcDSRRwXUcVjddPGlScN8Ot56HqRpj11wJmtP+52hMPywyn9Lg\\\\\\\/oRlATGVNQtQZiOijGyPbiVCcnhDBFugSJWTYm4oYBXD4QM5dLKn7S\\\\\\\/OO\\\\\\\/82ib\\\\\\\/K+JbQIkJUf8PfJvTbcUxocp8UrdGbM7v4j18weUKD8JnjTsK4wQzWBNsTuHXZCCUvo0jN1WZFeV4J73S9c5xQI95Dt3BqCWl3FwmnPLZB5WbWPLeu9Fk+0LAMGmhR19jn+onr0rrcBIFPGLsR45WQlT6QVTDMz\\\\\\\/u8oaI7++NR\\\\\\\/fgTAEdcuNVM\\\\\\\/Ksip7r\\\\\\\/3\\\u0022,\\\u0022iv\\\u0022:\\\u0022HofQUAd2vzTeeG71iIWDSg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ab3928f5143efab686b8f1d5240d5c4888138e57b6a8672979df73ac8ef2c060\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PVvoXm\\\\\\\/FSYUqvodwv+Vs2x1uDATpFX65qu\\\\\\\/9gFKqkRcvhtjkUCm9\\\\\\\/OtCdm4gTAgG2E3pNaTyBprOtW87pz9OnglPVhcaQ67BMsdY7SFuKwOc39A7X5e9Xa\\\\\\\/156AS3fqNwJ1n+HTUO9kOEBMkwFpd5rrraCedTWfMHPi2lpjVD9l9XK2jVJanE9e7HM3OSuta1uxMtYNd9OtPmLfqgTx3uBDLQRec3luptxm\\\\\\\/mhQmICbyqVzEkQYk6p1K5oHJY+Ja+Cka7dolV9Mt7Q3pU3cKbQFdxZ1Toi2V8nPRuTZValpaHcog8uX53G3Lg1GwrBiLnP4vJ0rYTeBNrLpGi03uYpDLt8CqOCzmHR9HiKXGqzgHTHhs6Hi6AKjcuoLa8tChTtGs\\\\\\\/nhWDsi1TfmBi82Xe7c2+s+mx\\\\\\\/mxWncuH\\\\\\\/pXhV1LOMAcsCR02NRFEBfZDEmjDzpwW05ZEmO4Pv1YNXPU2MgGqaZ3HHfBaMvDwKhCu51kX\\\\\\\/nFU2somtJL24yEcMSfzKQcmMMleOtPwp1xG892EOOI1vqJ+AXOecgdtwNyyP2DEaMAWzxzzgJtzSiFBOh6llwhZFeU49XGoMiCCv0tp8btBZwj2x\\\\\\\/9nPn9GOZ7Rp9\\\\\\\/mOjhEVUDyOUCCH1N8iJrx9o4890xCD8o8hHn\\\\\\\/XbxxDwS7jTjwr97CJfZSOkkHC5IpmvaO53nZV9jP8zSpvXr\\\\\\\/44\\\\\\\/jlRK8rDt46bzDkZdhoLN7neVnnNnEOed857NvjHkrvulwj\\\\\\\/nS8CItMIUMWzJdMFbgj9W7mL\\\\\\\/fFtjRBOKuhx9IfgNIsHGHYFM19+u9AsTjtpC+OYJwDzeOJOf8v5QMBcRQEs59OrxHX5oOHipDDRf6J2QJNFYyfpEwVKtooWfmQd9wGB5SPLe55W8dHqLSZp2NmDvIgIpOQtfRfO1nNskvjyvBpcWeixsfqCD8Ij1d7PchPcBW+nkS3zRIIuE4cd15sRj2eYrmucgZngrhZ+o+MQNe9PtOsxEiWxaaAxvyL+ZDV4mqfOHKie7a5y0\\\\\\\/vWdRQ6Bj3pKRKfQ3tzO2CATiP1xCOQ71m+UWcNyN1O\\\\\\\/+0bQ3qpCFT5KtajXJ1OGtmTSUOtZdxPNkCpSlR1OPDr5Aqz7hhaEz7SbwgFjqQ7kf727f8op4RLJLCLsdARtWsfca95X\\\\\\\/3+sMosZdOAp26x8mqJ1vC6MMoNUlrnfouuF6lW8PxAGg4tj0S92ZaRjR0h\\\\\\\/vqrGvSxhtgLcx+zLVqkRfSPqMgfAS0HbPTWlySoZNWgqmur5Ba8zgxCCp8G0hz1SFPYhMfwcxKHnSuls7ROk+cX\\\\\\\/5ZY2sHs14tBwcYWMGplx7nnHPwMhlvClDUZFtKI+hG5yrv6dyloET4RNIbbNa0hjnVaMLiH+2AJNhr3M9o0SCqQbVOX4NsQmmYGHUrSvalYLdWevi+vPJg5mY83a5P4iIMqbJUpc9mQxua2s4uTLwcg+eJmXOjMo13Mr1PCzUfwvDlYbG+o465Lq+7GzLIoWTojUNJK0yx2mDCHuoX18qEjlkWShBx7LQDy18KZGP+r4izovohto6V8kdS0kp9mxhVvW+Ml+Vi8h1A\\\\\\\/cZU6IkSHzAmKxcKfk4Gnar6GJ9wqP0YRZpLMl9r8G2SGOiWuT7rpDHEbYxGRtXzQ\\\\\\\/wHfDfEI2\\\\\\\/HCS4v5mcTA\\\\\\\/glOhZh2LXZbQw97YoHEGAmYqj5oSPYy5bFCByTG5SAQH2ndH1kFTpglZX3V1fQZGkswOpjx68Dws1SScgAQSmlCT0rUyfq2B7wfWkDAldKzdwUndnneTVO4yzQz5AKQJG+DJ8zaNB4WUiXaclmV8s4Ydxh94SjMHdOLknhQh1R\\\\\\\/VrTBW1ae9ZS0+MtJQljbrqrfzntOU4wRr++v\\\\\\\/GieHgTfZ028UnzVzhX2v12ieTvn16xjSOyW6YQDDYVE\\\\\\\/HB5X8O9TcHhjpeKU3TMIXSxlAQuIp43ekibYBZINFUWg29zH6im1Jzxf3LUw8p07ivFhK4dQgospSjkkbo6FD01zcgSn2hOOp1mstX1vYMmPlTtbSpPm4CuSy1LC5W9UWuZu+Iob2QW3x7sMQdHVP8h3R6SmPTAqZlTZM6qjMGUy6hPQM9ecyvZAY2k1RBz9pYLMxuvhV7kf\\\\\\\/AVuPJBCUKeJjcL+CKuzpMl5rWJJlZ2UaXE8LwcEMQIc+rLldK58Snj9AUuco6hjvL6HN6iuFumV5OtsI\\\\\\\/rJsOggp7gcTLcXs7NiAR\\\\\\\/zwv8eQWLO\\\\\\\/VuXLs9nh4i9jsoNgoqvGoqbBmJ6nBGZiM3Bs+hRkTa7BzilyP41ZHt7RrNmJXw0rTaa16H5ihSILgQ6Sn638Oj\\\\\\\/D2bb42vxtfJtRYdmPhS45oVWUQHuebtJ2YmzGITEl6HIxkHDZac1Ze4ZvvlAjm08Ja1h49hWF1xNzeEm7x2Xa\\\\\\\/m6oYGXPfpYALaCEDADRV4+Ubx6HmZcMxtRa1FMIeq9kY44Har4HE1AlLr3liQsWp\\\\\\\/nmKQS\\\\\\\/1bhEEa++vX9bwuiqgK87V8E5zPtUv5ZDY01+jviTFW4UZPXM18LiPT4JBSZTf9jM0DeZAWxnYx3Rw7VxNpzH80oKWHRTx3T1Yq7eefJNIWCkuq28ipyILLJm9di\\\\\\\/ztY7TImIA1dG25sX5lQM5SONf06V7dVtAJQZYxwJajgx64+x\\\\\\\/SrbB5KwqwMBetMSr+WAv90oWbCHlxMnfKXrwJ\\\\\\\/JwT9zddULlfmsQA0t4D8uY2atjMiOkvnqfFA3UKAW7K5uOSTsaNtHEDLB9eldTCs4MRZs3fY1xc3JMHUDmeSPcyzSEhBtJw+aD1Ta+YuEab5XRlyShfOfBimhecBf7QwEnMtkgBq3HiAFL7CrXoH7hM82Hok3ce1ii9PNkY+WmRSXyPrt5nqsokFhvBQ03hdH2RsfV+uofxdMUru5DzDmJoqqQGX4iYcM7bGUq+yz+XTfetX4K6QM20QFgaSyMYghVIz4eGE4LMegrZwYGRUzdJfWYyxB1sIB8wRFyYAGwARtikuL\\\\\\\/ZBGIJBSTritCXftjBCxYHmw5W0hoHiL5wYYIAosqtzHfT7mEWd9CwEMKRtO8fTATjmZKUifG4sEfbk7bbwzckiDj4WnSXR1cVynQ7wDXgbXHK8r8gO\\\\\\\/vBS7OaZoNTedReA3\\\\\\\/3OkLBIpj9z7hDiXndtpYeR3zfxASp+p2SxhblFehwngn8+JXfGLjfHlzdW2UdE+vg+d0Y7y+lSRwCuILEzfTFolVvXypaugXxGd9zV1L2ImKTRWh9Kb9TZ1NQfG9cSTEOx62mBSqwrVopSiIAxn7I3gXRkaLuonftPpCU709AS57V0P6nntzeqz2QD7tNCMAYys7iAO2u\\\\\\\/Rkoic6wA+q5bH3b+MBa1y4Ojjkt0gYESG3jHwegZaSEhkbNL925Cw6Oi0UixUG44LrYnClJOpDS+d8cAH6CzZo4KAQPRXzHNbPNUDOIRGhDncvBouyEfLZ3RrSsdnrzKpHERvb9ucjA8y2mi4JCVR72S+KlW\\\\\\\/ghRxNbNSXjl0OJJMG1exitiz8WHHFcdibC9TEbAF\\\\\\\/bh9DfzpuHvf4+4ANSKC+JiEm\\\\\\\/OJi88YJm7jNp5oBFPOaaN+v1w\\\\\\\/OMD7GawDmXQ9PPbZkGCIWODy6tPf7XC0UANCQ4nVKFQDk7bUXOhXWHqrTNd+XhbkjuH9DE+A\\\\\\\/peZuUoboQYWZlU234xAtJ9YdAPWQ94V7hNHjdcfhGvWawk4rerSzh1eqYTaKWF3Lsz+2XdQdm\\\\\\\/m4kVnc\\\\\\\/zh4wG1AENvheCMVv0l40VWvGbY3CvjA4waXUb0RdoZZDNbVhBJchoCwApBb4nBqSCyAda6rCLNCEbMidxgheFiDXfy9D9JbuxoUlusRPpOXMibexf0l7Y67esu7Vnoz36heNJdDUd27Xx\\\\\\\/ycPsdwBiBD\\\\\\\/Z\\\\\\\/BN+odoj4MgFQa0FyqRifTXYNWo3oxicQNgdYI9AKEow556SwlVl59vkTy6axqaW+B67ZD02ZTUZpHWLkUyK7gngPYLU+iCKQCzIhImVPxwq\\\\\\\/Z+hIMdajGyga9nEb\\\\\\\/Zg6rvRw2sm10Ih7TkG4koH8f\\\\\\\/c\\\\\\\/U\\\\\\\/OJNTr8JLMYouNO\\\\\\\/64f6JzfLJ6CrRNgQ5a+x483nddWohVAR8TE+AsjysBcbwmrQOxvjQD7dRbCCnUOjhZrYeiLZxoyN5eVv7SlU+fCosxdVo\\\\\\\/Us80wiDwx6Ikw6J7ZkPF1O58NDB\\\\\\\/Ww\\\\\\\/ev1J\\\\\\\/mtCgqx9Wil4J0P\\\\\\\/0ryvJK0AoNp8tMXnuwjnyxWv\\\\\\\/bdnaZWPRYtjiLZkQC8JMimrKjTs4fyLzYb8\\\\\\\/1N0p1p7Yrjs3hSGS48pLba48\\\\\\\/K7eK+tbogAGj0DDrTe0N7Sdh8bem12owFfjsO6AbPSty9Q7AbANY2qvfBN5nIHebrPULVonj0r1wKq2ymp4KRX+bWz+P57W2TiiSmLcEFRcTwRR2+qm2zrioV\\\\\\\/B8uZUkkqNAV5k\\\\\\\/A6i77m109dFMVAJ99mEAmBanLlH+7GBsDvU7ZCFojg5uvn\\\\\\\/e3pFZxj2vdnGfT2BxV4jeOEvBkgJ4BmhgFbK6ZCKNr8lZna9Xw443lAdFYSYnT\\\\\\\/SmeaBrrFOhDP9++HklXK6BlC\\\\\\\/M96UOzdoqn\\\\\\\/B5jMkqL50ymSnGQL4BiQ4T9Rd0UjdTQTiWbTRfUwx+pZSKIK01NJoY8nDaaYYLeIyhwJqJovwxniqOZClt2E9pSIK33I3XFii+NbAWtMWPdqK4Mt+fZSxnhTRLuatLymFFOTwfHyWAOy8d3yNoJ7KTlzcES0+B7ZYrZX0CKg2ddmJmlepqqs7d7TdMSyuo77ZMv7cvu\\\\\\\/5Mx4mLU931R4vC9EYch9dSuQwVcbvkcvM0d++tODm3IdyV7tOd94yA5idbRbKU\\\\\\\/Zpep9KFAhxSBX2W0psUXiIRgP55PFnvHT1173pjvwI\\\\\\\/EHMUzE\\\\\\\/kpY+wfJzH+oNqRmCWZtt+unzX43EqKiYJOubJ92Op0v8xZSMxGcCE89Cogcxz9YJsByVzArqex8HnabdYhLVlUJHroABVvm6PoCMofvc2TXEROWIWdafbn4r7cNqLXEwVsl0A7Q+vnOA3ukLMCJVSAj4H7yz0WtGPg6VW0Heh\\\\\\\/IhwmCqUSNbQ4Q1FC\\\\\\\/t1bLOY9h3Z+4xIMXca6ar9ZJgN7w+i6pChSdpzdKyT25vLV9\\\\\\\/O94mR4pc1E6SEgXq4V3Scri9wfJW7H0V6Quj1WCUOQedCpCO4K5oARfHx7lVY403Jkoolk24FkafOnp4KCMW6ygb7p0kcgBIn7ftpS+s9DjHzj2DBLRreqJ7Y4i6a4jW6AAsahaQ0oSr+Kh\\\\\\\/LaBH+X4RAsRY2MXxxO5GxtJQ5oMkET1Jje3fq+dSTFiAqQHKBm13BdrAJpeQHCdOeluLn+Jm4mwXo4E\\\\\\\/qeUgR\\\\\\\/FhEq27zyLQswhY99giTSGMSJx3tfz5yaewD0o6jwIJjtYVpJXP0l5aTJRqhLqiBf\\\\\\\/0v\\\\\\\/U7aT2hTF5xda+e8R8ksi5U0sdx06q1\\\\\\\/R8pjPG+pqpb7dkpb78+mNoYmKUFSBK0l7Kyv3HRUnsKbCbRvI47ciPvs1Lx7X3m4hJ370KVcyfcqQBL\\\\\\\/CQRh4t+WgizdKCSef6l86m9EeVMyoab9GckG1sano6G3HsTXDS6eSHfdkemALHQTSknmCcShLSzj04zd2JDXpIvJhW5YbeR3zXkZzvKrxg6TCZUNec5MGJe2ncTtzRgtOupFGITxwkzJC7ThMGGGv5B+mZ0fRo5ld6lIFjjTs06qjeTscbZ0JCdofm54C53VKzyrLUDW0ilQ4KZRP1cUoJuzLGJGR1LedTlRWB\\\\\\\/NsC+4WkPz17mGyexZF4kd04YQ1ANwxRft2Ond3X6\\\\\\\/l8pE1RVNEsXK5qPS+mZE6LJ\\\\\\\/zz+B9KklUFvh0vLcaP5ypiO6H9G1pmo02xsNZ\\\\\\\/Lz\\\\\\\/6D+cYVwhVlO83Oy842bTspZcXhuLu3JEF6OagFuG1gnoDQRmBfzaE34jCx+mWH86UqzQSQgyyni\\\\\\\/JUUqG+4kSEnT7nGKDKtAkpqO+slp2yeCABC5unN0YVOJqoj9r01y+n4dejndNKt4ueqzmTZWiN4uU5bcRlQvhxtH\\\\\\\/6EQFJ2Py0CjhhlwXXsCW+sjrSR0\\\\\\\/aMRE2qjdw1SYWExKXXUVVq+MhmIDdsNPBsFM0MgE0roQn5JstdWcTuj2mP8ya1FNAXok6vkWifbXGoAx4sRvod8Io98gw9a59ztDGHz7FlqcLyofmyNeWYQeZzh5al0hKZ0i9+\\\\\\\/3khMWEYrBke3vvVkqn6mvT0KLc7RDrgVCH57MywhNkgs6Lk\\\\\\\/OGUiKw4vD36b9sORg2T5A0XdWtBQe8pjtwe30+whY2oqurnevFQ5FrsNF37Omty6PA4NcQYl45qSxbjQwCo4TFVoVXezT39icVjBM6aMGhblpOOBmjdzflKr51dvebhfrjySys7uY1jnbsL02HOGCv99INVlA8+2TIE984QJiEeN9ttoMhbIH4QD4wLxI8KJHh6Y10h6sjayJBIbhOY3y0Bl6v24OyXitjF9ABvDmwwCmIdabmPGBRmDL00LYag92C1VG6rkD3JqlC8DnLpOIrqgznHvjSvpQAS94xsIaYF1SbYMlM\\\\\\\/pKRMhSuVGfrC71DqruGopDmvBBcNDX+lMVWg3J62QAyU6JRftGzwJf5YQLrHwWNXqYBJfl+dcSHlFltXPx7A37SkV3cXfX5vDhH5uwASfmh0O\\\\\\\/6UbxsGSsB89RNUWDFtcggZGdE0NkRAQQlzHTOrvUKSawVMH+Vy\\\\\\\/tp2Nh2uka\\\\\\\/41l\\\\\\\/obXDH9C\\\\\\\/ri9OyXmxETbMngvR5tRrWAquggc714j9dDg6Laib2F37yG7DfqgjcT0FzA8kJfWL2lTvcpgDXfdcIOf7SdZEkk8BTYJvZKPtz9ow2RZ8DGhIA9yM7PGalEkpqpR3dwAaBGJQmam6MEhLdafVhX5l9bBn1n+5Z\\\\\\\/CPKableoOtVzhCCHH\\\\\\\/FD6J67atoyx+B\\\\\\\/WUyG5adn\\\\\\\/OFHETk74FFBOe0zQqrl9ann9B8uVAsSerigVZ62MYL7GrHR3yScTY10r6w\\\\\\\/OZMCa8gEjQ\\\\\\\/gYk25LS2GJuI60E1tDIlGEITHRwtMNSgq7G7ZBm82RAxlVh86DZIQNGHhIkXFSHU8cFtqruMU+dinBdY1Aon\\\\\\\/TLPNE4f7GB9yTocJRGjCwmSeajedxaTOkruXaoRIdfiYa7RxDFMc217zdyNi\\\\\\\/v3o4dnopo5uOnT7Eahx0wrSUtw9JSftuZQ+nENCE+ZEPmYZA+xeEpalIHNKeeR6pH1aRbFH6HBLcylLcxCsjaFjUwYzZ3vsqgcQTMluhbGiR75QXvZWXfi\\\\\\\/1Nl+TgTM86pSQeWu9FULmJXG9rS1NdhZVCrb37iZcCSCdfAbuK5l0r2\\\\\\\/WBafGLa3lGIbDcRo3lSb7c022T0\\\\\\\/7hGckuiXraotRshl8qQHFd8FeCop5mLRCFrGx\\\\\\\/ayQP45Mkj1V83wFbvwTmfxH+pqoik1V2GNi35gtP0uqNqQ9Ie8zFA65nJr3TivLCkpPGi8OiBRhYw5vM3yb7JSVY+aGoydIFr0V+9N13zDxz05UGdOhI6qn6FSIq4Jc+pn8qLXtd4jqEjBN1+7FztwwlV+C1shVU7bPxrulxuZ3ejKKh8KS2W9sHb1OFIKiSjIIv7PXm9XySgBskT5a4QgTxTMqULCVi7j2T91SwvxEeS7bMd35w0sBDpZzoXmiS2uA2yw\\\\\\\/Uiro922o\\\\\\\/Hnl8UPy5TL8EDAgE9QlPUPi0ZTJT0cdYUlyZpfO0hHhbN1mZ\\\\\\\/oaaPo4ptKxZiepdbYhz7qAche8qmoYJxWJX2hMAHQCrsx77Q3br2SfyVIiikuSLnPE3gkL9ppasqHmB+VNxCb39VeSLImTr6XD5MZBohF8jqy+WKxEtDGIurKaSgkVC2L7lv9uO7nHXqRPion0C1ypkH\\\\\\\/01\\\\\\\/L69KatZ3rwiCVMu3e1TSyPW8aG19KGDNNJF1Xi7MH38evLKFrXc8JVkyrQmk+tA1KuPX6iDUR1HQrFu8aj+yWlvD66IZ4oXD4ZZXIEiemHnwB42WI9agww5pt7GggmQVdnBFylZJGZS6RZFskNdZ6PZjiat81paNCHW2YJPvs\\\\\\\/01Q\\\\\\\/4CZGqpSFBNbHNZN73WoJiuuB28HV2momszcdd0ZNxlXoS1wOo0DixVXDkG6NShlSZYvEZJuGvvK6+H1KUhG0wmcod69TdmyO86OSH280cYz02MSquK8BDUF79KKsDg+Jg5zGIAKsO9JI+Mrx5gVv\\\\\\\/ayUJo6Q9rJe1VHcwV8lBTCe1gW1r9fytOZkbgXBwz5LsC49Xo+3Cilnwm1UywI8Axo99jLoQc5Yui0C17HMr1BaoCGItD4hka11g5llsr44t9aEoqpBsD54QwjqX9T8xx4JhPUFZYJ3e2FWLByNEXT+ZBh\\\\\\\/ypY6SxhdLQt5Z0V3N9E9yTOLeQcSpkfan5zh6tB8nZB6vJC4S8W\\\\\\\/YRS4Q0Nfau2b7H8mwKzxyY+IaB7wKpr0NEyEv1YdCIQk3YHszt4fJg9Sxyj93SnVUDaWhsfOs38woHm\\\\\\\/Zr6oMi\\\\\\\/wAjVMDPPjFOX1r7AXxmWIto12TQaJBcuouuZW\\\\\\\/dITmyi5CPaq+FdxQTsv1O\\\\\\\/DVQDrmYlVX7N\\\\\\\/Kx+xHceRLNQ5gAXL67mRZtg2SVO1iv2uLIABs50O287nvYwnxrQT3adq1X2\\\\\\\/UcJ9y8TTPuVRQMHio938mAt5tsbwrHxNc9GVc74p1gs3uT45ExjUs9yzaPIg0Yvlcv\\\\\\\/nV0p2HKT6qq1x+OJ17o6aVKqf\\\\\\\/ONYLPfkIYR3F0AakT0JtrZjxohcoY3hlVUYiTkDw4wJ\\\\\\\/H0DNWn2TXuU2+DkKGkNJs6N0zQkDwVxeXDHsMbolc7QpO+7U3SLXkOII\\\\\\\/tXEcyErnVxonLSmYiPZTMUuaCa5bmm57caFwZC0Hvzlrwx8Azsk0Qpdg2j2NvAOyPN9VmvkqQ4lzAjrbyB7RQ4tK5egsCM3vJbIL4Eq4uNBPGywartFIprjMCG5g89ficdgdT9cusmIDP6MKuRlKgd7gbBlinSy7ZMyXs8XOf1N9USvusc+ZTjfvvMteg8xYYqyJQrSP5WYURDjQnxjksgkRMt1SGdIwNplZgpjQKtfqleh78PoBn5Vxp2enBpxXh\\\\\\\/AUW5rMbsTsB6LbRKmq939FsIxgGvnigDIjN+HunwCQMwOEBYHpbG0R+1UnqqDJI+KExYrPULSrm5w0CFz5j0RCAFjAOxDLcjhc5EuwZDLmazqGzrDfQFUwfPcZXR+\\\\\\\/Zoc4w+7gc4hkMsH72jqy548wXxWkrVFouAM+JgIfUhRfBVlKL+G0XrnAaL83YTzPLwix7AHcmFUr5AvWIIx0kjGkRWKHNOODoZN8svQzz+EDfozIhlxDaFHggT8Szqh2QqAbTQ\\\\\\\/bXowhDpjAOtxXcQiwlUkj+MgkL+ugHfRu47dxQWU\\\\\\\/gqJdfy3XHTDSDM+IkvGVeymIBrLsDPMBFgB1ywEzFI+iuDNJf3Fm+u5cn\\\\\\\/vzCesJJO+aRhmxTf3BWqn4si18D8EElBF9zYYRe2TiFxkQEGc\\\\\\\/mafdWvAD0sfpSEbABlFEIs4AAq8DokZ\\\\\\\/JSEPuWIvQlbHYD5sZtDVroq2h4VUye9Dkw89Dlqqo\\\\\\\/0lccteV360VcU3JcNqW6+VY3HuAIVoGCbzsZsrQNr1IjGcJm8GDwSFamAhWViEVGylmM9iMRWkJIb23SZ+EhlAe6qoC1C9Zt\\\\\\\/LtidIfJ8z7\\\\\\\/enB6ndrgi\\\\\\\/aGzfw5e04lCzcXnWmpuIxMS1W+fUBzPL+oa5oVOqrnhngXslE\\\\\\\/MuFLXKEcS6nCV5soSyrwCTzu9v5TiVY4d059JDNYT+Yri7jpnJAIKpmU5y8M4hawIAF5i7KDa\\\\\\\/yB3PdggxIRgF8nWukwzITf7xFMrMwKPnTNx8XQu5oi2jcFrfrlyETP4Zi6WTSkgmJ5iNsyiQjmgT0MENMtrmAhk3uJ28Zj5ainxlDE5qRmVH6DFWBgHYLbvFzqBdPpEuSUCSJmosZHYfRCtYBi97wZqg4lDLVtL6Ld7IfTvAZ1dT5Nm44rnEcY69LUXsM6Ad7Ow3i6flv0zqB8FcJv7fWud1GGpQVt4LU35PBFQAajNwKpKyqAA12x1216N9A4FwC5lhjb16Lr3rKKZONSAAAOTgFmtnP3VP3ytXHLvQ9gpJbkk07qQTPn6TyaefBfO1XLVa7jGM57oUo8If+REhFs6BkXacGuu5L9xoy3DDfUp+TIyAg6OOT6BokDNWtnvQKbTJP9xLx7XQRTiOACu5k8OLKU0v5bSTTx2JSQSAB7UL+SAYgpMqgwQEZFrpGMtY9FySM5gPDe+D5g5MNE2rgd\\\\\\\/d2MLVlNKCAeqCjHfspB3eI2GOlzPtgweZ1\\\\\\\/iy3AXV22Y+NztN5XOW+8jinK1f3qT0belEQJ2tylcDEYqgMyHnGNXIv3bUF8wqOvY9Wk34cTgrHvYmFp9eekVHrmplvv6Q4Tl8va949E3cJdLib\\\\\\\/foTQHttY58wXK\\\\\\\/S46\\\\\\\/SYypiKz\\\\\\\/tQ1ldtG33gRtgY6NUUMaO1ifWhEzkLXXUxyF\\\\\\\/+SoXwFch7geViJOK9XC868Gxdi6MZQcvM0Bm+kBMBV5vKDAp7WWKxtbPKicI3L+D98wik9\\\\\\\/xdQ3g47XoI6QzRqDCIXy0FUwVmpFIkIGwKXL41a4v8U4hEhqetvit0otUrJIyA40YzLFotren0ElzcWzw902K+mruJo6eqF+iabQ7vaj2FUsaLq9GUccEisY+bud1gPJ5fk1PhMFrlf1YUGGWwE8nDaqJofXlB6rPuO78\\\\\\\/rReW03XnhuFdEzGwi8NfPsZB6VOluiXta66TrJFROMpoHLUgnitCYjLed+2DWdwxtqeJk9qCnGk+OYZQV5FWq1JC5Q0NaIbD9GlkfTEglaNOoQKoJTu47i5xjLGZupuVrdZQzaim89wQN3inMCxGQTKHSCDM1cls\\\\\\\/sB4JQUyFNGtroVy2hyAea1sU8iD3YQuCnPGNE3ll6k0vUH66erHfjF\\\\\\\/n2dBUvEVYtCvvfDFmpOYNqUWYLCPg9UogFK286Je+a7YMmaahYBNZAL5Pf\\\\\\\/cHIpxaUkoDaa+96EWl5fM6nSGE9ZcIb8Q792nSXYcM2D+XoIKhkhAqU58h++KnSXGZd6qQQn4wa5VC7vzK4Y8BEc\\\\\\\/i6V+wnQ9uuP5IeDE4r5dm7lwChb0Yn7hej3\\\\\\\/UbByE+sLquS0r+31FWxVtqflU5xNvw8TV0\\\\\\\/xpfQtEt+BoHNmB479rWjKb9BUFpxk3x8eo8\\\\\\\/eZ+qrHM2LqyR99ftmCogslxXNq+CDtf4lMzRLaGWCKQIjJDtapGtzuzzvOo+BJrMjz77\\\\\\\/w563WNS8eU6\\\\\\\/0Rx91yURrud8QkEsnaJ6yeb9+cZgGmsNBglJXzxgzGhn15Ys1OhPBZv7P+ha351RWUksg6BAk+YfzbXiQU9ZTGYtAadxiIU8PuTTIu14Cmf6ziUyyhxvFH+J2434a1MP4SX1SwHsgm8P9TzR8jfrTbhlctrPV11Hqx+TS732vxQipZRBzALK2x21PUZ8Jaldf9yXCrd3Zl5N4n5JAYczoi6UdJWr+rojR39nrSav1qhEY75vQ\\\\\\\/tRWjku8YwCgE88NVtgLp\\\\\\\/ukcmMCFAxpS7AfvTRzziwdweJdHlRPoV3IVpLOkDuJ\\\\\\\/MbjBAko84JtFd8e4E13glnYsbFTVaGb5+q\\\\\\\/F5QDHrBaHDDx\\\\\\\/Q9rA3nQgycbMS40ACHAAa8gQv3EAUSsHclxWfLg15+jYBHGzgnEAeb+c1F4pqs3R7iZMb80gjn+9Uxh6hAt5zMz43T0ILfcF1QRz6KUAfvvPN+mbT7VVAUom2axHlWAd0eMCAoz1zY4rDodCqPbf8bgWtQeeISE0Ua8XLadzk3OPlruWEIaEQBQC+oGyI0b+TJFPuKm+zB2utBac4uwRos5JLv\\\\\\\/DsATQYCnAA8AKNgK7c153YRK+trUmbXsYnFCZD49nethwtymdBYLZGishh4ITlQmtaq3ha+Y0H2\\\\\\\/Soa\\\\\\\/\\\\\\\/FYuTOI8v9QYQvPwqeVubXHPIoRr1oz8Qg9uIG2pNfUi3\\\\\\\/vwPgobPaueIE84WfNzEM5vpsRxE4v4jj71odMkNLEtTz5jCKIR+xDTVbP99U+0dBg8nnjRoSXww2N3XZwyQJOQhdjj7bzsfn23VZsAnolygltj9RvFtirvoHlLFIgdJ+12noI4V6EQO5WNwssxr2bnZiDft+iEmNQSvJpNHDURMJQCfmEmaBIQNGcb051CL\\\\\\\/KPccSDCTRxjV85lcM9jpYEXx1gzryrLyqU88fSoaux0dBQatQ1QNWFuoiUznujEyFYeXe9yG9Okwi1+wVT1YhtSkgNFPrtW6xl4fMDoRe\\\\\\\/7zN+nvS6QABRvDEp0CbLczBX3c8yhKMqB0JuXPvuQ+akRou7zfqcX9m70RX18+kEywcen7ubMkl78zhcT79y\\\\\\\/d\\\\\\\/K+B0Z16+eMAuGXBAERJAGteeKA6CjI7dqN+p0LoB2G4GiBSMb8+lM0xweBlmNjrkLQe3gRYUs7fUv6myc7wlxuQUqiJG7FihxHLiKlIJxkXewiN\\\\\\\/HtuRDr4351GRc5r1isD02EhCZ3aeBB0s+AHkTkB\\\\\\\/DnEkoRAmpdmZAYEdmuyHbEna09n4CRa09Hm+raCwl\\\\\\\/WgAaW7gzsU8EXSOKpoklrRqE0ARBdjojrrJUsdMsFS6DTYEFTdUa+NRqtE8UiYvRsnHSJ8nF23an6dWRgmng9RGfGbgCPqzn9UuyupU9niL6e+8ysiZauDrJap0AFaqjTBkumUn+ubTrnHMSZdWM0kh9q6NLldevFe0QN3O65huZB1y3067GPjNOZXdF\\\\\\\/jy477LjDz1mC\\\\\\\/zbNgsyj2+bQmwK2isZ+M4EceUqsZNbOvj8griCWtdBNtHEk3eZoqPvH\\\\\\\/P4HAIq0NyJcC8NJeyr2ei8U55wlOazPjlRBa+ZCXOUWJKq4WhIHVnQPvSFEfQMSw7klUwLDDE8rIqQW31rRzBRuvAH6fJD8hJu\\\\\\\/DxCgy6j8irIHM3s\\\\\\\/R7zgptIEVK5QwIahjCkqxSeMunf2q\\\\\\\/xdQuT52+4n2OO\\\\\\\/ZfLd0XSVHZTAoc1gGbm0KWdaU8z3zgjW7ko1vqLrGMyr1utaA84dtjFMwsCWCQ0qMPxrR4EypqflGcEcMooI2Hhg3pJpgj7Y2fzr+\\\\\\\/SAxRxWgUcasfhDU4uL0lLc48ZEl\\\\\\\/F6b3Y4ww3F3jVeQ46EROspLGfHyjPeH6D1L6X8lSmih1qJa\\\\\\\/GtZeEHfl7SVd+4PdBAq9vOAq9lsVfMMerJyGM04cWgqWjZNrU\\\\\\\/KpHgtgPp82DIUAxWXlBT4ruWjOm5xl6GwL2lvkt84Jcns4x5Sm7j1LTMEM3I2D\\\\\\\/\\\\\\\/JlA+gorxBwT\\\\\\\/jIjz\\\\\\\/dnkInuR5V547syMstZdlaIj7yIUyhJv7ymP+oq0kYkNSAjSPyebxBDufjwS+xKkjLI3idqqpk9gY4PFUGsTQvOOCSGPdT0Q2r6hQ+5HDVAMKwqxY+\\\\\\\/GqYOo3sNuL\\\\\\\/Vi5c1MqWfdjzkEJ+mkuUoRCCWtpPSfSTf\\\\\\\/+prpycUKIgf9deBIzl4y8klDuavueXl0nPjU6sG1KclQ1Z0YfLTDtRJezpaJJdpnoy+MMFMcVEztt8rCX0bx+CNEITSLPZCCY\\\\\\\/fEMS7eUoq31YzbuDm0WZYuYlG+PxaIzEiaW3x3XrMd6JgR3ZHhgQkwgzOg3JJ3ADNOcecsF5bzCBg1I38haaEy\\\\\\\/3qa0BmNpgNRQMV6ZZHeL6rx98uzePBDYqPYqB1ypkgiU44Z9qfV7qS\\\\\\\/FgWVb1L0ONaiSQkUrIernHEOOOf1L7OnsUZVaCJxzd5gdVLu9Pmy46bGLf\\\\\\\/deT6tbksO3VNOMAOyyghVyo8Qp8kTgz2ypnaZ\\\\\\\/Lxwrz1v5uLmnDCdTxLtMluV97ygC2M1aHFPb9ZyuN\\\\\\\/hHLkoPffGzPL+g0vfvcQWjOpXmx1XrNMkJSexejfD\\\\\\\/RSGlB63q9A5NNjbC7Lgi8qz7clDxfenuWQZZS1Q5Eq3dIvR7gCoZBd6qN4kgUGb4Hum8+iAWQ+Y3yXbpfNkiYZ6t8ff9xgIAxW0L7knJBjcDwZ7bXwteWaieAWBYILuZ7gguIK2EzbDpBZOhcELUOJztcZYPQ4p2l2BFp67fYjxf62OZI14dBYa\\\\\\\/blWGJhPupnHh\\\\\\\/nfqW2Ux+G7ug\\\\\\\/0tV\\\\\\\/tm33dYESzAqC58KBYiJJGf\\\\\\\/A10LbrrxjxJFKwtXY\\\\\\\/Wf+J9WHqRTQOM6JMxSeaVV\\\\\\\/BW1xuPuv7ZwjHJGByN\\\\\\\/3zJLJPQaTTvtcVwGktZRrOALo9TgW\\\\\\\/eHyxpJ0K+OHBGD2tz6FCY58dO+m\\\\\\\/4rsYsUFu70YAtWTqt+5RpxQOVu\\\\\\\/sEoD7CzCM88NPl9L2M7fP2vjQuIhhCkTvBfzVOLQngJGeFy8+Fs8j4VkbzAnZ\\\\\\\/heJT7YUKWpfNPJ1f+ONKCN1HJ0T\\\\\\\/PsgUi2XNocPt20wdZwtGDrjMYNXRKV6\\\\\\\/8LjhHLq4tEFSZwMktpso\\\\\\\/r65yIqAWEQz+ptWUJP0y3Cnj0gqZxDIHUd9hx7NscoAnLf2rrQHn9aflShnn1p+Xk5+dOjxfd3P\\\\\\\/gKP8+tI4n3Cxgql5ZyO5Giw5LB9XNmST5AGmuQu8\\\\\\\/QoD8Llq45deTsItceSRlYkabKXCF+8aRfcU6eGtOEF+cKm6m5aBg5kKvv4MkA7mjXvWi5\\\\\\\/zq0HQKvY+nuTGwORkc78auwGkPsIjVx1V4nG6NgCN5wLV0kXLPcZJrs4Ox9LSUXjk3Y56GgPy4y74f9J+3ZK1UCk4jOLFSlo2rzpajq1pQDmR78TKvPedIfhsrvjXDM9dZdg1OEJB+6jMiYadNXjvXdrpgvsgHv5gkeFOxgIKnumX\\\\\\\/VjhaJjShrIs6AQKPJp3U5UKfdLgr9ZsNCiKMh9LtQ2XKG22W48qAUihqIh+m1GDWibJnjFzEch2bTOIlgL+ZOx7fbjSFYxWi5y4DT7K7Co0c\\\\\\\/Dwsj5QKr8SmQZbnPUrSn6K+pHcXt9W3o\\\\\\\/rzoMyAOuTIw+LPjeq0SRLN2sGzb5PfB6BDzweG8ng7ZC+2yUSfkXKa5q3z75JYmXG2qj9R4n\\\\\\\/7GJAJIF6atyzF2jQgOMGDj6tEDEg0NYv\\\\\\\/M26350AgGM1ExQD1sE5bguH1fH7wEDBy\\\\\\\/jS182GxjZTMuhQ\\\\\\\/q68e0SkH1MubvcMMRyJCMFrvsfUOxkCNYsHVxJcKQ\\\\\\\/XfskIK4UNYqa5WceYbsHtdfDxiNZuIZy4WmxUZx8DIKKmK1a30pNMtpLMUQ56KCHIKdiIrOoRf7pJFZtOpByeryHlxLh6uy2f2CcXzCoNKv6\\\\\\\/ZXSw9Eo4TXauPp2AE4oUoRGaih0b118FgpsybrXeoM5BnEun7E7M5DVepVBPUsnFNJqVJlIdfvZPIv5k6vMRj9224NPI18F+P0qWhm9KH30e8pNRDAUzH\\\\\\\/Hhs6uLD1ZusuU7dWpdZmmkLTbwzzbor+m3\\\\\\\/SOFzhGWppQncNhBGO3X3UmCMh4UW08xWSXdyMjfQHSue8QfViYN5ohJNuoWUJ5GjknnCkEZgxFnO8FVNEjUH6o5sV05qA4ucJ8THqNb+BEkVd6438c63s+H767q3KR08a7FwNgGWXg28V1swGaDy79F4eprQwmcuWgvsmSUvRnxOYIz1d7VGQdLGD7xnzo7\\\\\\\/f9y5GU0zRGf9simwy9D1hOJ3gc5RsoWPN2yefv0zXbvKmW1UykvlLPgUik3mqwzvubEFJgyIgFVu0BaydGTFbKmS8ndbwkDdQrTHQHh742agDJJjwKuxV4DLDaGJnWWSVecIo5NldYlu1Otr8sw+82Il8n3Z4oFydXN0eq\\\\\\\/HUPoSfFfA3jspAGFo7YemjKzHIRdHGcpzokJU4q81kPnqgeFEa10Vq\\\\\\\/GjVGc9GXan8rNJi+aiYxJFhXSre0+Q3bg3k4XspfFGXS2rhHCUdIgs1wuZ\\\\\\\/no68sgiQVi3YAQ73ZjFPOe3riyPn2e8xlhTLxFJK8LCbbyBYPaW4TsnVuk9\\\\\\\/ViUB3D41K2HaGkxg52HSr3fPMdVgvUh9j16FRQSqojzUqeqhH1\\\\\\\/CeDEquCFaAxGXkoA\\\\\\\/v+akwlRasLIqHkeXFysIBu+aqvw1zzHgdXinbGB9GYQD93+DnmH07t+935TjosT19QUKUBaU2bJlfLBQWNCwiKz\\\\\\\/tD55qKspwPkfuPxzZilLvac494X5aDz6oLXCoaxEgO1dHNXZS\\\\\\\/XtFzgF6FntP+w7pu1jKcaFidK1ioHD5ToqdbDiivAjXnqTBGaWbXMYnQoQaz8CSa7+C\\\\\\\/9zu\\\\\\\/NHCWMfkVj8U1VZEywDnxv0aaZJJ\\\\\\\/68yKiF+R0HMAYFa3+gZaJHQV61Y7kKKGs6dO4zkJW+SpjT\\\\\\\/9vrLHcbQV8R7ZD8VE0lokRz2dLYmuR8lWu+f+LE47l+TvSDebd2l+D6uZyAdCP8es1iZ8bib8KDTgU9H0K5Cm6r8GmnAF4N5dOop3Ow5pnRWUFZut\\\\\\\/Cq7HRV6fkTxM8Q5r3GgrWllUR5uFHl3D6LhFcCX7HWgExfmIddCXp4cmz384IH33HHJjTnEj8b\\\\\\\/R1ECRW5jA\\\\\\\/deNZ9u1mRcRIUvbOe8KTg2CzZG81K+pvGuXQErBJBYb08LNFXcbr34vQz8tZTbku94Mou+p3+2a\\\\\\\/i2oKfasVfdDzGIDy\\\\\\\/4xVOnUcdYhAwYathi9sUviGmikQvc0BtLJ8EfFxJKXcEo0Z11XMeJI7tACrwVluI2KUJXO+RDJVhqrtoDMSQHb+eJUNot1MHi+xs7pLHmNvkx+\\\\\\\/XDee6y53C2UbPTjnmSnHS5\\\\\\\/+NG\\\\\\\/TDeZlLuZJQSHye6icH7BUXC9DphUdAJD9xeLVSzgD+T21kafvL\\\\\\\/wfzp3qimw0bL4yh8oD9mJo7nHd8hmLI1YKa9OkRfhQPCBrVKdObywb3xZtcL7Vu2a112+8IsuzA6NbLKZ1ku3KT9K0DAWqVNlzdbN17dQy08yY1OKXVSUGZScru8nl8mO8fjnbZiw68nfdaGPbanGtDGeTBP9qgrTuGIAIxSLCZPCO6htH9LEm2\\\\\\\/ewboAmL5NZoQQqMy128hsDh49bo\\\\\\\/9E73rso3+UVKosldSg27LkBgawUfeVuN64IQAWPwmUAZ0o8hNvglXeS+sjuYjsLBfG2eWe4eMpewkNDG56HiqB1j6eQxz0Pz3u\\\\\\\/TeSeqxoMRfDHluq87+oHR3U2cE49\\\\\\\/oe20IqUWBsvopRmK0hfZIefDGUN1PFltKy91yXJNkn1Sv2EqA\\\\\\\/DyNxaV\\\\\\\/Q7tKaYo4UPDQpZZmZPLJy2pzF4Hhi6CuPAZAcVNg5gdipperHIYTR6E6strdJH7vLuBV+RTQ7l6JdgLDET2c\\\\\\\/CwqSOwRjTQgISwWvqFwO3BHPSoHUSKhgMoP+ADp4sdQekc7bBK7mkZs0hjaS1NWjxo7r8UMmMwc\\\\\\\/aG+im+NS20hQFVOAmGz5R5xgfMgOs5ijqkCavE0XmCG+Hnj5GO2CeIbcUz3PHalr8QyLMH3vKpGL9uUQAVoOVwgCPQKeRAxMzk6XAGawh4UdheEG\\\\\\\/eOG3Kex4fUvHK6udsE4YScsD8VN1KkfQC6B4I58qFJVhUKk2g4x\\\\\\\/RmaRnzz05fGqZhiwkcZsHyW92pSv4\\\\\\\/3utOPr55AczQTxgA4Qg5PonJMu2mJo1Y5We1WyXpsFW\\\\\\\/ztwrQ7ZEGo6EHzyJ8HsJ+fbu+0lJiOI+PcNCjSb3r9YFDOHJTGcVPfcDlWz\\\\\\\/XMRlEieYY2F13euuYNrpkQ6QGJnjidVcUL7zUm+QNoVzkrKVImwGiuXXUCsbRmDLS8ztJj4SH0rk\\\\\\\/nJ6qKSOHjNZMexL6iqbOb0+s6RIzTfygbMrnWk3NYlDwWxnKTzt+AcMYZptNmbwg3ArA1OVvmyuJuwoPAfKYYsLYMHjX1aYbugxkIHLuDrsu1OADsr7+3wyEtUwx9q+Ovu+IACkFSEHYFL3JA6QhO+hORFePnOiWui+U5lc0ixe8tD+BWddkGEaa5oEBHxCjMt9i1aOTVH8sJldrjPapWrlBQzAuQeYXdYgdx+3VhNTCrrk3kxJpC9pXHjeWh7BFUUnEy1wo16pkl0oAUTei2FW3LwIPw+V2cDha3Fd35zP3gv7oTo29Q7ljDVyeXQ1cXBBlqP+3qaKlX1jlLIThnbrnQrLbduMyXkV0\\\\\\\/lN5heuZd\\\\\\\/R\\\\\\\/RN12pj44BjGm2NpkbaSiuKfTD6AHzW\\\\\\\/7SH8WHODB0QmwEiS2T9FRKKnvXsjq5hl5k9kYKZn5JCXZk69ZUmRZSnMQ1CQILYRKnxuuNDMr36ZYVQjunx8R61NmBp2\\\\\\\/\\\\\\\/xtSn6i6TDgfTiwXGAMnJHYFvwKEavWBP4x9tT6FUrDG0myZZHVystP5wFNKYh1DsmrR1BFtl\\\\\\\/YEXpxzPk2qep7UN7SwshDy9+rYIvqgWomEl6dWJZFbypfXbO6MST1cD8zt3kntyj0\\\\\\\/qAm92iDkaVJ7XbpJMTyy2460aBqA0ZxTRcieswbUM3w3biA28\\\\\\\/TcOnauV2C78yqJp9NDvNURwxC18cqjfGvkjTj5gwlo2UipBrCU9ic4DNZhA9qxAWJdcy4OgIoCj3NTPSCYAunjcX6PXGLUYoCDcsPlJE4wisFYRoi7cGFUFXlO\\\\\\\/XU9Ys+ygCYbb9wuLlECpXf2sOHK\\\\\\\/I7SjZSPaKjdo+dj6h7bb6W5o7LNvk2+0ThvoLa4Sj\\\\\\\/m92ooFV1F33pZki85s73gUut17XDN13jiWUDKGv0F3bxalevUdYKQPavPF+TYUQsEcM4QuShyvvfxyHQcIGELo0YebQvFGkN\\\\\\\/dJbd3uXZU8Yg2mTQdPHIV+IG97r1+ghGzrMXPcPfrPngN5o+DEzuAPkYp4Lk2IgdAGW\\\\\\\/wYUGy8aKo56Oiwwqqzoz1jjDHMbYLb2xG+EcE18+EsewbGrlwxswNVz9LHokg9uJeUr+aNneykvzI0FlwwSoVm\\\\\\\/xrv+0SlNqlQ9oECMp1E9flfaq5T8EHxG8urJ2H9IJT5Ruab57PwJV65tRkXqhc2YnK0XfVEEfWJHKKlMfeMjRNX3uKZq61tKiSOH1hDnx44m93xYY3Apb6gV1ROOCKQSvW6zRs5yET+mEPnjMDvY2c8qy3XECHws3CshRXg2dUH9DudMzfSHOQY0c4kNjct2eaxYTONbYylkUmpkD0lX39sp6wzoE+QxVdirrBJtUCINLxq9VPYEv9y8\\\\\\\/83FiJPH+qtITW2ZkB1\\\\\\\/mwpwS0vp1O8hZ2nysvQU4Cv9V89vghtr+FwXroEaPqcW5VxKVgk8kCBSL9RpE2JhSxvKjUbUlnQEIqarYFJ\\\\\\\/QRigUlWanAiFRXe2\\\\\\\/4bRQR7dbf1oTsDcIgLVdOojQOdcWiBe+l2lPTaHWmb5N1MZERo7CxQd17pR61iQV\\\\\\\/eANxDv689bt4HI1DzeQT6ZaEsT9coJNoyyQmvJ7UgJ7jVA2nvfPXLy65BnDokZNIBseNOcMzqlhoO8j4O3ZNvSoRGpxc9QhqxrWIFuoZdMt1d5+agfWV99BFeolKzZaxIrOHdS4zM+EnAqWu046EuF2FnG9Q4yTpiaKARhQ1hACNvChRnwC87OwIoeg3XK6OY8LW4MAmZIKxO9xZGmk6r0zOiTb5OOozuC4esVL7zOaZqlM6WYPpqTJXZpRZrNZzCyOrYDPSuApCdGsBLHsbrmomHoAzxWNn\\\\\\\/gHagI72CdQ8QrL1VwOH9CfQjIk++xGj1ZkRTYXvEin6jG4Jh8Spl1IEEnTGFAFOBgUzE4PuIE9fKK4FdI\\\\\\\/MMe0ny293jwhFRGiQBKpx0pk9JPxVRahkDR8O8WRBWeblF4DLfXN7sBhRmttbjAhHUAaZpvnDDgkeW4HKwFzamElERbPNHZYdM055DUlNX6tnPk+50j5RgIXpYKZQSMKBxCtxmHkn3g9oj+WV9OCr+9bIasRPq3\\\\\\\/hIFPzn8hEKUCV8mQQ2w6cNn9nZ5wbzVtkgSiPAgL68TCpcmbjPHgNtMPKLkFO5Py49zacwskrAO81zHh4XZXXWvEoJdd121U3CLZizVFyGYhChzM21DvPlIYQRtYRBNPNArRGGphZwaJGphDFwBE6+m3xUq12o2iBTr5LLv\\\\\\\/L2vJyC2iOaPWyrStMniej7O+ZGAhKKDt6Wnt7oLvsuCWKGYJiIJe\\\\\\\/c4kLK3jnkLJuJ4imOFokGoAFRou8cGNte9EA6BATBlHN0XIVIGh53srC2CpRSthuBk81boTBSSOQXQCGrZqm6rh9wE0kgo0STgpMzTTi5PONXDzIVZZ4qE+V9VKch9GtfaIv75EeLhWrIOaiHlGe2aXhyfKrDOharQ2YlbbBBujGLVn0iv1wbY49HYOUQ5XfWR050jHvWW6Sux1pjvzev0MOA2bpjlKkNHcW54H4yZ4qh9WheDe0KOndRkApHK4J+nhfnhxweBUSOfcS8MKq\\\\\\\/+Np8IWy4j7rP9NI99rX8PCdlErIYQEeZzby+kWNDF056sMzX\\\\\\\/02tZ4XNXJJXMCEPT7kFKYq10uleWzkMKX7OOJ1qaOOCtKl4SfVNPRynN3Q0XX2S8J2+nMXvOFl0sR2E8r9wLtG3L99gzYPDRBMDmEEBT9d7qt1y9kVQntNTnxlpcKr0eCNRgF5eIhXqHwXdCliQW2IPhUEwH7OD8I0a4liPw7ibcH0vF0oL1tKoxRD2r4RRK9c6qlGtkuijf6nD820k3luocgrof6wk0XXEh\\\\\\\/5D5LiHmzEsp8LqGJyTDQMGSELEuxsuV+mT417RFaxusl3GWZb8mZrdXsc7dcA+KHKkqCsl3yJNz1JNGZQm25ZIM\\\\\\\/U+LjBmn3DbnU8PvEcgZbpBy0RXCUeY8PcCs+jcVjT8ZvegOEV4lFgi1s2tJgox5dLGVuWZ+5V\\\\\\\/CPviwpQU0FdgVM5joNCKxWwPOPp3Wq6M9SVfua45NE5t1tK0P3mKmyk8wmTl4YcKg+KDjTIi7QcZmJXFbeuYM\\\\\\\/UVYrdVgakzy6Yquif3od35lgop98Sk+hGRwuVFww1nEfUOyJN4AS7q3J0Sq30k6jUVJrDClAinr0iQjlsbY1zFkmDfCnhkb20vRCPSfUv7uBifRICmJ9lBTQo6+1R2vL0Pf1WrXfJV3MhiPz\\\\\\\/KGrlwWZXDE5fKI6PCohdOdJDRRG9S3TGejhcmujk+aGAmmwPqOIpb13k0jrn\\\\\\\/\\\\\\\/PTmEOvjzkcTQjR\\\\\\\/YHO1gCFgFlHgFp90zNPU2PJL9Tl0HBJP2YWoc3+b50bY5f6P8FUhfLgrqnU97SCaQOzSi6cS41XlThTrv1SQIcAN0xL3IOMfKXaitQcPApOJaZ9yfKJ5iTxiy7yewfDnznQT6lKUi1+2r+Crz\\\\\\\/l3jkGZv0X\\\\\\\/zfL3gs1xFKjGpmG2+ZWNa0FvqTR1tnpU2oi5RhuZEXfITNJcDVYJWsLiXQkUouaAH+J6vgn7yd8vUtF9GX8vQY\\\\\\\/QOUpWwodfRcArO3EpOHgJWoJCbnxYajzSfSVtGToe\\\\\\\/1zWN8l\\\\\\\/g6E+25k+kuRxGzFNE7hgnHaZBXGGxP4vg8NWsJ9KjwCZBk30FQL8y2mP\\\\\\\/c5+opp8idZ1zn\\\\\\\/e8G8c3HOECrXyo5JrihDj8xZYEj+bDtCerkqjLcJJOEj0Yuqo2kYgqtiBXnHq+bt3dbxQvi99vr6cpWQhEY9lWu0Zqn+FcVhhWYXAe74bJilGVk4eLMtQ+sdsa7aB8\\\\\\\/kWXp841PSqhC6dXe4BjWhauYiV\\\\\\\/mCa5TSvP9GAkLS6p+U0NskxQG2woQtxV4\\\\\\\/T8+itAv7BgTVlqUvj3MGyI2uzeIsKVyvX0lmIE1j1hvCZJpSJravBpj+GK1Eky5A7klCmGF4HGGp41oFDfL7sq4PoSUM4BAfzgAecIlnDW4WirB+hFXQ6SRgm9v+4e6cP2IjkbR\\\\\\\/dPsgBG\\\\\\\/VwNuPo1xxi4WhWIeG+Qr4TkeiqwLcEr1UMatM45RSGMTbzVWC9vrVTwfp8cijNAmBjUBZ1XQ0XD6vYxx6pKzQotqeXywQRoX8cO61\\\\\\\/fat3VDfq7O\\\\\\\/ajC0qUZACehWTCj3aC5QdcF\\\\\\\/jp+KJnPsfmwOYXdZHME+nEJXBQDD\\\\\\\/qb5D8p4d\\\\\\\/JEUC2QFIZ98jCjHE2IcmGWVEko\\\\\\\/wqWVTBO6P7orJChypz9u\\\\\\\/ObZ4lrmcd+V\\\\\\\/5trKwLYX4cbKPdaYy0Sc+aIQjUMzu9sFE9zL\\\\\\\/xP3j1kcnMTUsWfrdnaTyjWlZfsBiOjroCdSOdNoTHB9+KJEqhbgf4++olCiJ5cfg+0GES3pVP40mj1KpRbwBY7RgAI9rmoPOpwM80ITH1cTxWGV9sbrywyX3YIMVSPHs61IDVZJXgHi28YqowmqL4XZ36LYVFKhudXsQWJ3NPUlNSXfOo8cvB3x\\\\\\\/tVF5LRFhfU3VDb5md9bvBeOytL2t7KR55QmOkakebhjtWYwXfX57mVQbdp8oc9oJXD1zjo+5OeA0iMR4CIJiwPUEDaSf6opdRMJuHdMsLMs\\\\\\\/eNxS2OvBKWhwS5iepYcDVLEJBwLbp1ObIZAGCmauJOPpPsyfO\\\\\\\/oCM3jkOIpSyQJ8YarXgTrMwun1T053tHQhjQuca23rlEx\\\\\\\/UVphNFygfCtpIxPBw9SsJVyk9ugQkLPotgFvCC3fn5TPsOIbAbnMzKbdWVKeq6sOR6kFw8F\\\\\\\/nuOmicGrbCe71uRva+QcTKEsZ560m79dCQtxhsr+GcBd3yOo2Q4ueUKdMOkgxjgo7bzPR9KmxSbTAb\\\\\\\/uHeq+t9APEk+0uHcHNVMb3nqJGiduqeXesWgm5eC6qfTeH4jWtxnSsTMVeddWDO+pQ\\\\\\\/Q2B4GP7uyPjFP16nmT0rnrryIxg6SwqagxsxmCLrerVrqgAnG8xPjL5EYZTHcQxgrkqFoNnY+pErepYvv7uZziIQEaLU9fnXTvgJ5\\\\\\\/p+\\\\\\\/xwl8y7DMYE6aeOIGY7wwx1WYz14b0eUV1aTXc2QiDVUVQaqG0D5ERv2hUqtSc9cv0FIcOTeR3zHNCdWujrzdFeHbtV4Vbs9jqIDB65qqwfffOI9DxCjHM1CvpPmJZlVsJJZh+lQV008IGlnuA6lkTqXH3TnbqL9izOl9MufTpJiChKX759woSwvKmyVGmHYViZnFMqCxfdf5ljzZIY+5NXrePMD8RcD484bmTo4dN\\\\\\\/vHilqLFWPMYvn4uar2NOvIT+VPl\\\\\\\/UBHmIAE7lfm8QC8GDHRHjwdAY+YsNKXu\\\\\\\/3In2InihS2GYNN3dpse4rcY6TFzaM8FlfvCEXUud\\\\\\\/8aHwVQB6Dzh\\\\\\\/iDgO7cvh57u\\\\\\\/mMbgsO0rFXzbbV4vTdLH4ARdC1tiOD+SHa0BYmNHTn5MJ2fdVMgXPNEQE\\\\\\\/Oe\\\\\\\/LHEwQiyDc1m4KEVqOlES6akB5FuvAIzivN9bEr45xU\\\\\\\/N0oJiZYyaemnTV6895FUiz4d+SXCDBaujyPPjUbHU9atkmRrk60yCeAWzlQDaJET3Qc1x34zvi9+onOstJHJz\\\\\\\/DIacaNxMr94mgvSKJi5L1SJgxdZb+Z0WWOEkk8m+HTBFNJpNtD3\\\\\\\/CgAL14CT292\\\\\\\/jICSRnAgCKFb38JqktOh+mH2efAvjNhLEvyfGpXQNkVgmZ6aRDl2wnKXeWCgGj2pI8jKzCU39J9eGftplP1j4CsJyE74xMaEJE6C\\\\\\\/vOaLmaVq\\\\\\\/ikLVUOksdiBYuOC7YJn9jF8ar5I3WqlzIayHPMFgwAtVodECiCyivZIfRc6Iq+lE4ohv9vnNAmfLa6EX2yXpPXP9OdQA0N\\\\\\\/SzgFf9y+1S\\\\\\\/KfKMuQpsZE\\\\\\\/n0\\\\\\\/\\\\\\\/+6i7H\\\\\\\/R\\\\\\\/pJfEW6HFL804WWLiq+RhKE+MF3DBEUuQ\\\\\\\/bce8qlnOdE5otrfuvLu0m\\\\\\\/IhTGVEnL44uxEsVNkKnntLFfZgPaKvoAtSXpph70sM046m9Zyz0NUA07lIhiFcxVYFATpaB92jW0o13S8qV07b0rxg3Rom9iyXfQPiUYh666+3rIEvk7U0r8VEYQbTQdxwV\\\\\\\/9KprWvPTGDlvJivxHOD9wCsV2MewY1qGFJOWlQcU4GI4\\\\\\\/hFt0wt3WWDCcu015L4U+YhY86dyF7aCp3nquloc8r2LBNtH8foagRWxMXzSG\\\\\\\/Cqj9EU3CGIlPI2WCTmG+67LQbvq9QGDsoCE+too0YNQyza+CgGoLmeGK2sbcvp9Bh52BxNcxbj\\\\\\\/jOv\\\\\\\/A0xdqyCUi6oQRoHHYBeWsKX2N4pUE2MitkhYq52csxpiPgu3ByOURXLFFqel3r3LMmM62g7rFCD9d10\\\\\\\/gS7sjySTho1gO1OW+dmY1OEBrnAD+R4lNkJd4qOEJ7H5Mdu7KXzFdNTzu1MwWaCsgDMGUexdH+S+\\\\\\\/47SbG\\\\\\\/5VBRGSoSaOGwgq7qdS9Xxx77l1G37qnaSltCp24tEaY\\\\\\\/NqkzKaYMdYnfArOUH38AgccWiFONGAgXmNG9Da0kJXozh0GiG7t9fxhOLNtiXOkkradBRcdJN2Tf75Eg3Sf2UM4otrNx+gZ8vWm7oaWaovFYw0\\\\\\\/GODX6\\\\\\\/YvQRznTulKiCjzCHibBKPctWQsPt8Q9cmx+FHOPZniYVtJXQ6gBRwWGnYHA6KcZiN\\\\\\\/4XQLGHzskhd8L9HVai5Owj+8641YfY0OAVJrDY7LjBrZCqZPbz\\\\\\\/cwQiXFlRsVmEsuizXcBYl0Hg2RK2jqKesvIbK+ujiFVySdrxxKq25QGvxLCKrOck75wJA3qieejhUKzbOa\\\\\\\/r1vVkJjQitrtrw3CB6nx1hAWjIVlbF1dAvJCeEXwMHm\\\\\\\/K9tIWEtQamNJO1GkqFqi3vj\\\\\\\/E6uJpHuiuOHmUtypG4rUV+CLD1W\\\\\\\/\\\\\\\/GtsJ4QC5knoL225zAv8ByokZLVMZdNg0BKkXdqydoux4bALu0tMPhFwT+AxHBae34+CoetNQcuh8wqwCArwy\\\\\\\/oLOyXtZdlIM6RuZw+Gnk1oe6qr7pvlJzehPMCYMq41O2VzATwvLLy1FxMi1p0jab8t0KcMWEeTqC0lQ0a4OV3WtRjDj+usPX2npB1tUCL5kqYSjlMjuTA1I1C4dhkSqe\\\\\\\/n7XB96gj1nT8mSsHG0fv0rCZnTMDeB9aVoFoox93HTMAvo+op2xp0Fhqe41ObHTPJnCiOP+KVkCzM78hMqcVPTgkMkj61gV4Os4sPM0YJtTchDc7GKG22Wt4lrjdfCQt7AJ2RDqHMtseU0QoJf0c+N877xDIJnIV5XE9lNNBvH+bwMcDv4pbTv22RWDWhNF3fcWqEgr2FOHHHQJg5oDZB2VbhddmTa8ayxOlXJlPa2FuRuYg5hCZe6LEONitcdbiLWg\\\\\\\/8pSwy\\\\\\\/OalE06VebcNUp6QzKBQ14s19OS0xw\\\\\\\/WVnNfRD73uB5XE6iD35i\\\\\\\/b7nOd6Sj+TYtYEuxscGanACI\\\\\\\/088Cqlz3BoF5scDzcqByhKzPijmIGlwnBIhG0T\\\\\\\/u7zFOhYRrqZpoYZH+DBfNUvVYXAJXAIy0s5aYj2f94nvZQthBC1gOG\\\\\\\/BaUNILG8hJ+ufzI\\\\\\\/FD4qqWxJbrw3VRSaXiItzGqxo0N91pPzrfaNu7bVgP2Bld8KyH3ifb+Rd93MfU3NAMyP7PWa1luRS0cRDswJnkROYM+voErwm8q+nvqaJNL7553SdcJ7ry1HFd2tmAYb42Avpwufqrp+FHhGjoTS8Kiv7OUICYKsGIRSfouRIPeu5cwRxdYZPLe9v68lZGLVKrMUUJAjQ7Lzs7HnbU\\\\\\\/EismLOvtR7aCKAcYMCE0DiXYdKjrqEZewFmuEyGgpLgFLzCS4U5H75L0rfM4wHSGvC0q47qwURso+NrAB32jA1NuTvcD4QYwe\\\\\\\/0XDLEBCk8p+7TPltxIb+VKLzZMvFDM3oLtM3BM48lWIVSl2NDs1hLS7rlQ3sT3pv7ciHmnN1a\\\\\\\/VNeZPeXWxR+jPCx4tmR09Xi0BQnsget0lFRibbLL7LWLBaCfd8uoeYoU7bX51L0Nbyvxk\\\\\\\/M5cIBkjzDKLrybdLLxxKXhpsclJ6ArRs1CFa1cdPhr6m+UpL+HDRq3NxxTPI2jb374zRn5v7GVqa70ZuJ61hZORlSArwDnnQn4UkbH7YK5kP25KAUNBRoP1H9qMLZPKEJrH54ZGPaTAdszI4BclAzQvA4P3hBK5Ez1GhweyKiVTIl+VbMUYTj77sAarSaQeLLeEOzj3YOtcEF4NNjFJ\\\\\\\/Lvcnd7zVFDOOWJoiKz+5DYNgoAWI+qVKjt13kuILXTffWOl\\\\\\\/2qHe7E2BD0IoX3QdFMUZmfpiIRlx4SX\\\\\\\/pORHRo4UN1CSauDrjt4nhguXqZCIEfe95q2IhllAzwLS3qgiv4drUz4+FRVUfC0f4\\\\\\\/D++tcCuTI3gfplssfz7Va1+lDkmxlJDaEtIvHpEvvsTFJxsAIzJagjydC3yXyehYuteNu3U3AuaPv3dOGLS8SgoZwiJLF7Qaa33kjtvRZ\\\\\\\/iNQJx8mRjkSG20h7nBjuED01h8\\\\\\\/v64gdzTYhu2\\\\\\\/amDVf+4rmx7PKg1UVGEWGVDSQIqer3uOXMmlEcFHs20SzNyX+bvY9xDbAfTogFXmNuaL3\\\\\\\/DxNXebKT8ab13ZqqinwS7otRD7QJ0WYz3HonlA4is2GgdHmam6RLVbZogd9mWt2EKyAyNC0IV\\\\\\\/ZCeBICz+zd2aYAhOvuZz+PHaQsvbL86eocyAVy4Bar6IKDGR58vsmX\\\\\\\/I3iFzbcxG4ish4nWtC1VJLuvcZw6yBkKirVm\\\\\\\/dlna5sXIK3+fW9ozKcSCKpCX5KrIe4mJliIwc3zG4GzV8+3DRcFsODlZNSzBD62OApovzodkabYIXajutbO\\\\\\\/dkLMIF2vCCd7JNY435LpwC\\\\\\\/S6yrUtJMrUZOW8l7AHFOpdNL4GjtuopaBGimW\\\\\\\/KkvSsQn7JFqZVUmByA\\\\\\\/Zb9bIGA7xVwH3RfSNi3s9k2TTV+0Sg72i+KM1OWHwCmOugy7fn4weahut4G5DmgJ0K6u8yPbmeOR89TnVyyM\\\\\\\/iKo9IoqelciqJWdsPkdfj\\\\\\\/quBODdTZmPSXjvziL614c67NmMp3V5JIGD8dIP4jPF24Hl6Uoo220Gr0P1K0u7ikYepP+7ULyEKJ3PlnDfvld3xTjbvPUhB4qefkEMLzzgoXRo7s2aX7nG9dRJ78Pz4ATeI9cRZNjShzy6sENSPWDB+PsoSmOdNqhnU+YQICrAyVtdxwLOLhlPG6L9ZvkMl+15lP2lIGgFD3GHDy93CS4LeQ73CPVfDIEzlSfeFZVlFbDq7TocxR7\\\\\\\/XB6sHCdw7fRRvPj1vOuQQF7C5ivKmDDyaaaOcgL\\\\\\\/HLYfgOKREpWgv4V77xitZTKxrKhkWPbQ+UxsWHt9OKjYHzOVs53UdB5lI6TJuLJ+Iew6w6FqA2hIvl7kab0SM7DveO2HbGK8Roi0aJSMRCA2oLREUrkVxn7\\\\\\\/E2uJ\\\\\\\/QnxdqKx\\\\\\\/L\\\\\\\/RNMJY3FX3GQugTBiNXS422LtMgnAb9p\\\\\\\/9DNApuNZi1XEi5Tml2tCrG\\\\\\\/zMbc217UyqN4Mc22cTm9otbAssNRWG5\\\\\\\/Ac6agTrWs\\\\\\\/2jcCdisBcwzfUYPijbqXwfjriefzrynS3jdRYosDuY797CbpdqJtkMF5Ffu4BKz4vKyJaOdQAScHvI0JSuB7IKJzeigOV4SrFbqrwats3\\\\\\\/7LMboeiwfr4qChJUI4w8+UZsndbqABwOdv5opZ5FCKuF5Arpuk0D5FD\\\\\\\/4y2R32QF6HCNW+xrbbhApzzhOfdPwbotoH+dvDUBlAuI653QtOoxo2Y4RtjbU0VLNqZtqFl7SFf0hWN+lBL955OmyOdH1JBjoxdpMLY0ValaKev6DnPB48XGaXQrWkIdp1o+uHvqcQK\\\\\\\/\\\\\\\/XXgeFbOVUq+rRSMrlWbFqQ8W61J1PW8wQudEIclPGM67OXsH2GVWGkkzCeLJ+gpBL4gnjWCdI3wgoYXClRx072X6T39nW5AvYfGh+Qefcgj+A+KiG+6f17YlTv88AlH4P07fRASs+0rx4vY68HV9nV4wXQyXwzr+PeQRqYvmWGrLyuNtNh98XkcEm7HKO49U3EYhUUOpjTx2WBcsA8s7F8\\\\\\\/W3u1Pk2HpZrBGJg2b5RICULdHmc5meHSJg6qU8oijTVAjmW3OKKMChi9mjisxi+Duf1UDkVjeTmCuLhbDJzzAx+BkYwJjXdVQgO7gS2D0OqvqqaPifaL8VmF1MQvuo4i737C\\\\\\\/E7CrUJ65bRMDUUb1RiyU\\\\\\\/0OOyIu5AhwyJwn\\\\\\\/VExH3iCW6ubj6JS0CT2mrJ1VBSvEx+U2+lLQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022b4flo2xisI7+ts6Jd7iRCQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222b661ec602cb8ebcf4eaae075df0b650ebb0eff2071f33aca1fb6a33dd3e42b8\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022fU4d1m2HkJc2qFGDMDDPkKY6pqRm9ZmhAipwKbuKFS0ZxrZ\\\\\\\/9EY9sGRA+BtvyewWge4HiSgCKPgUmlVYOUS+VuLUkLukEFak1kql17iZfGAnsQNpvMqVuc88Ho21sJ8gpjrCfE6jCQMDawFT\\\\\\\/sX8HCDubBUakAN4bBzWycWQzUXIt\\\\\\\/IcsEGQOUFOIBqqxJx2Qlmp8SgEemonz60NrwRUtiWAzxYGQ1FNVSYROttru0+Gt9urh1cHXRpLRk5otOxYZ1Qbn9dl\\\\\\\/yZ5Z2rms9uqcsn9HxzJ5anWsnkkrOd\\\\\\\/umMFwfIvFL5kmMfrT2OcpGcCr2Hw4VvIPRnNovRpixDw+JXCwAvG94+sc9w\\\\\\\/cT+4BBBRHgExvCI72s2xBQXBV9PlNETXblkWc27SPMz1s7hywdd246\\\\\\\/Dwhzemm9E9t72aIcTcMCmRVStkQ23Ine92bYKgtVE53Z0TH9Dkk1ugCABE1cYhxNR2itjWMFHheOHV51XgZf6dv9jHGHJygZyHJneRqhP3EmKwVcLjr\\\\\\\/BIILbEGvVmZtMcTiGpDGb6h3ZRz4eXyQNK3m2nTb7MRYGe8Y4dVT1wK8VY8iZ9iuTzoLThKfHYbSRZt5ju7pjJUF46lNgPQkDWJypnt+9C9H6wMdc3UbATm+uDQCZAy8H7748oZMdqzGrNn2I6R\\\\\\\/iyoRsMpLsQUp32WEEF3ySahyzRH6nKTGK7efEnkIoNU7WSUlNXo5PPEEDIyH+8XGhUC826RjPURLYxD6Ch+abwAOH+xKfCdpk4p9ZtqS3l1VNP47acBcwcquguomDu6X\\\\\\\/XviJNBLbUsRk8yJnphCCYHarYvhMDY6+4i39G6JkNBQoDSEkMBRZ4jwM\\\\\\\/PYwknTdHM4ZJRnVh2oYnRuLwQjo51h10odxjEiLKC4YSRrcd+Mf2rVdhDrX1TeQ6jyUgBKyV9jvwFT6meJ9BnKzYpP615Hpq4dQkZy42WaHBxdQK9Jg556J8Mg1sK8iVMmV9J1b9WPD9peXaCV3ApegjgPUd2uZlyqmPJ+3GzCH4shAcDX\\\\\\\/65QClmC1P7YL97KZVAeLqHtXstTU8UpS83yh\\\\\\\/fVYn186Ku3puUGiAfpGWHiztmRVkYVmtRUBQNTI8Jliy2tP2OMPToipyCgO2Ox7NtVDo5bxJePQOxHvW8E\\\\\\\/i0bdCkLZxzVIBGVW6VDiTdKDJ9Qk7X5bn+UHxQjLcCIiCaKXN3wyhYluoVh4mSdNL32+rqkVbYIu24R7Ch7iN97jYf05wpfHCgwCIa6Qz7i3JtMpV90vgTveImYqr4yOZwD8shKYIGgrHv3HNNQLHfwCxOIFyvEHDmQ6ouqQM98EpWg9W6AyAyb+ZaN1YKftrIkcWVg5c+5s1AsaDvXrYFUmGZ5LNp831zMRodkihE9m6NOKN7+9tHDrUzvz2UBsOPHhrqyhmjaJlKc2T7Ws6rWcXcW95B8i2bbTZjzuA6DRN3f3tSW582BZxd93aUGUxUp6dxOChg76jCzpErZscxv36OJamto\\\\\\\/6B9znn\\\\\\\/p7XBYZ6P24PSpNeQEKHX0rI9fDnHZprzHrLjFnVkUkcdND7tJ7Yjz32HSkZXtuhlylIglmhd7DnZi7WNV3cTuSHKS6shZVCHBeK9Gqn6n7E\\\\\\\/s6qu9rLjSzKhvuQvpNw0A4mED0LAeps8SCKfFL6CbQfxuvuQ7xLRWBB2ZLb8YYeglnoLjNpSm4dCEk8C1Uv6SmoSQjbNeNpPzNQKNTgb6+ovXSvQ2t\\\\\\\/eXRZF3cCGSvDcRchRBZ5sq6rsjuRkRv9FwY2ytN\\\\\\\/OzZ5TvzsPFUBZD63NW4WgmOABDbEkgNhQdFsTUu4lGOEszyZmH9XuqMBJ2fkvC176338T1WYYJENFNRwnq0cQIcvwIenXR9xCtJaSuqjSRkBxYKaayMxEqdb8P9r5xJpQGBs5TFM2bNnRVRaYqH3Yl5ZV8Um0kAjl7uvCpqLnzPJ\\\\\\\/BDudWpUeUf3Rb4\\\\\\\/ddZa06O8BVfhYcKrKoJ2zOQ6VW3PAgItkgFw==\\\u0022,\\\u0022iv\\\u0022:\\\u00225TqQ\\\\\\\/Gm9dVQu4cLGdtrD0w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226a3f507061c485c0fb9a9038205424f1b053ff8de6750b6cea260ff89cf54f14\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459299 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/7/779" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-mFIBex-p9xUk7BWhzp8MoTAyufK1M4l6SJYKJ_n6C6U" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/7/779" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-KHrqV6WkFaCgmUW6ZFzYhhOfjsbjbIV0WzDscDCiXAY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/7/769" title="Genetic Diversity and Population Structure of Chinese Foxtail Millet [Setaria italica (L.) Beauv.] Landraces" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/7/789" title="Comparison of Parallel High-Throughput RNA Sequencing Between Knockout of TDP-43 and Its Overexpression Reveals Primarily Nonreciprocal and Nonoverlapping Gene Expression Changes in the Central Nervous System of Drosophila" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459299" id="node459299" data-pisa="ggg;2/7/779" data-pisa-master="ggg;2/7/779" data-apath="/ggg/2/7/779.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genotype by Environment Interaction of Quantitative Traits: A Case Study in Barley</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Fuping Zhao</span> and <span class="highwire-citation-author" data-delta="1">Shizhong Xu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">779-788;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002980 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;2/7/779" data-atom-uri="/ggg/2/7/779.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459299" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Fuping Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Botany and Plant Sciences, University of California, Riverside, California 92521</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fuping%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFuping%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Shizhong Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Botany and Plant Sciences, University of California, Riverside, California 92521</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shizhong%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShizhong%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:shizhong.xu@ucr.edu" class="" data-icon-position="" data-hide-link-title="0">shizhong.xu@ucr.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/7/779" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459299" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459299/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/7/779.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459299" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459299/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/7/779.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459299" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459299/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/7/779.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459299" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459299/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/7/779.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459299" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459299/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Genotype by environment interaction is a phenomenon that a better genotype in one environment may perform poorly in another environment. When the genotype refers to a quantitative trait locus (QTL), this phenomenon is called QTL by environment interaction, denoted by Q×E. Using a recently developed new Bayesian method and genome-wide marker information, we estimated and tested QTL main effects and Q×E interactions for a well-known barley dataset produced by the North American Barley Genome Mapping Project. This dataset contained seven quantitative traits collected from 145 doubled-haploid (DH) lines evaluated in multiple environments, which derived from a cross between two Canadian two-row barley lines, Harrington and TR306. Numerous main effects and Q×E interaction effects have been detected for all seven quantitative traits. However, main effects seem to be more important than the Q×E interaction effects for all seven traits examined. The number of main effects detected varied from 26 for the maturity trait to 75 for the heading trait, with an average of 61.86. The heading trait has the most detected effects, with a total of 98 (75 main, 29 Q×E). Among the 98 effects, 6 loci had both the main and Q×E effects. Among the total number of detected loci, on average, 78.5% of the loci show the main effects whereas 34.9% of the loci show Q×E interactions. Overall, we detected many loci with either the main or the Q×E effects, and the main effects appear to be more important than the Q×E interaction effects for all the seven traits. This means that most detected loci have a constant effect across environments. Another discovery from this analysis is that Q×E interaction occurs independently, regardless whether the locus has main effects.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/barley" class="hw-term hw-article-keyword hw-article-keyword-barley" rel="nofollow">barley</a></li><li class="kwd"><a href="/keyword/markov-chain-monte-carlo" class="hw-term hw-article-keyword hw-article-keyword-markov-chain-monte-carlo" rel="nofollow">Markov chain Monte Carlo</a></li><li class="kwd"><a href="/keyword/mixed-model" class="hw-term hw-article-keyword hw-article-keyword-mixed-model" rel="nofollow">mixed model</a></li><li class="kwd"><a href="/keyword/quantitative-trait-locus" class="hw-term hw-article-keyword hw-article-keyword-quantitative-trait-locus" rel="nofollow">quantitative trait locus</a></li><li class="kwd"><a href="/keyword/q%C3%97e-interaction" class="hw-term hw-article-keyword hw-article-keyword-q&#xD7;e-interaction" rel="nofollow">Q×E interaction</a></li></ul><p id="p-5">Genotype by environment interaction is a phenomenon that genotype effects may significantly differ across environments. When the genotype refers to a quantitative trait locus (QTL), G×E interaction is renamed as QTL by environment interaction, denoted by Q×E. The magnitude of Q×E interaction for a particular locus is measured by the deviation of the estimated QTL effect in a specific environment from the average estimated QTL effect across environments. A QTL with a small Q×E interaction is more stable than a QTL with a larger Q×E interaction. A more stable QTL has a wider inference space and can be used in plant breeding across all environments. On the contrary, an unstable QTL can only be used in the environment from which it is detected.</p><p id="p-6">Q×E interactions for quantitative traits in crops across multiple environments have been ubiquitous. <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Boer <em>et al.</em> (2007)</a> reported that most of the detected QTL controlling grain yield and grain moisture of maize were influenced by temperature differences during critical stages of the growth. <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Li <em>et al.</em> (2010)</a> found that several QTL governing growth trajectories of soybean cause significant genotype-environment interaction for plant height. <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Paterson <em>et al.</em> (2003)</a> showed that genetic control of cotton fiber quality was markedly affected both by general difference between growing seasons and by specific differences in water management regimes. <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Yang <em>et al.</em> (2007)</a> found that majority of the QTL detected for nine quantitative traits of wheat interact with drought stress and well-watered conditions. As for barley, G×E or Q×E interactions have been reported by several investigators (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cherif <em>et al.</em> 2010</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Hayes <em>et al.</em> 1993</a>; Piepho 2000; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Tinker <em>et al.</em> 1996</a>).</p><p id="p-7">Three models have been proposed for detection of Q×E interaction. The simplest method is the standard two-way ANOVA (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Pillen <em>et al.</em> 2003</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Tinker and Mather 1995</a>), in which Q×E interaction is tested one marker at a time. A slightly more advanced method is the split-plot ANOVA (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Utz and Melchinger 1996</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Utz <em>et al.</em> 2000</a>), in which each genotype is considered to be a main plot, and observations from different environments play the role of split-plots. The common practice for Q×E analysis is to conduct interval mapping (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Lander and Botstein 1989</a>) or composite interval mapping (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Zeng 1994</a>) for each environment separately and then to compare the QTL mapping results for these environments (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Li <em>et al.</em> 2007</a>). If a QTL is detected in some environments but not in others, Q×E interaction is implied. This approach is subject to the same drawback as the ANOVA in terms of ignoring the covariance of the residuals because data are analyzed one environment at a time. Multivariate repeated measurement analysis is a more advanced method of Q×E analysis. Phenotypes of the same trait measured in different environments are treated as “different traits.” This multivariate approach to QTL mapping was first proposed by <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Jiang and Zeng (1995)</a>. The advantage of this method is that the variance-covariance matrix of the residuals can be incorporated into the model. So far, only unstructured covariance matrix (full positive definite matrix) has been considered under the multivariate QTL mapping. Recently, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Piepho (2005)</a> proposed a mixed-model approach to modeling highly structured covariance matrix and showed that the mixed-model approach outperforms all the methods described above.</p><p id="p-8"><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Chen <em>et al.</em> (2010)</a> recently developed a new Bayesian shrinkage method for estimating and testing Q×E interactions. This method estimates all main effects and Q×E interactions simultaneously in a single model. When the number of environments is large, <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Chen <em>et al.</em> (2010)</a> used the variance of the estimated QTL effects across multiple environments as a measure of the Q×E interaction. This approach has tremendously simplified Q×E studies. In addition, they incorporated the permutation analysis into the Q×E study and provided a significance test for each effect. <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Chen <em>et al.</em> (2010)</a> used the yield trait of barley as an example to demonstrate the application of the new method. Numerous main and Q×E study interaction effects were detected for that trait. This well-known barley data set was developed by the North American Barley Genome Mapping Project (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Tinker <em>et al.</em> 1996</a>). In the barley experiment, the investigators measured a total of seven quantitative traits from multiple environments and a total of 127 mapped markers that cover 1500 cM of the barley genome. Incorporating all 127 markers into a single model, <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Chen <em>et al.</em> (2010)</a> only analyzed one of these traits. To capture the whole-genome marker information, we inserted a pseudo marker every 5 cM for a marker interval greater than 5 cM, and the total number of loci was 304. Then we analyzed all seven traits with 304 markers using the new method to draw general conclusions and reported the results in this study. As the genome coverage of the markers was quite high, no QTL would be missed. A general conclusion about the relative importance of Q×E in these quantitative traits is drawn, which can help us understand the genetic architecture of the complex quantitative traits to guide practical breeding programs of barley aimed at improving crop production in difficult environments.</p><div class="section materials-methods" id="sec-1"><h2 class="">MATERIALS AND METHODS</h2><div id="sec-2" class="subsection"><h3>Experimental data</h3><p id="p-9">The data were retrieved from the North American Barley Genome Mapping Project (NABGMP) website (<a href="http://gnome.agrenv.mcgill.ca/">http://gnome.agrenv.mcgill.ca/</a>). The experimental design and results were reported by <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Tinker <em>et al.</em> (1996)</a>. For the article to be self-contained, we briefly summarized the experiment here. This experiment consisted of a doubled haploid population with <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-1.gif"/></noscript></span></span> lines derived from a cross between two Canadian two-row barley lines, Harrington and TR306. Seven traits were investigated in the project: yield, heading, maturity, height, lodging, kernel weight, and test weight. The 145 lines were grown in a range of environments: yield was measured in 28 environments, heading in 29, maturity in 15, height in 27, lodging in 17, kernel weight in 25, and test weight in 28 environments. A total of 127 mapped markers covered 1500 cM of the barley genome (seven chromosomes). The genotype of each marker was coded as −1 for the TR306 allele and +1 for the Harrington allele. A missing genotype was coded as 0. The data contained about ∼4.9% missing marker genotypes and a few missing phenotypes.</p><p id="p-10">We inserted a pseudo marker every 5 cM of the genome for a marker interval greater than 5 cM. Including the inserted pseudo markers, the total number of loci subject to analysis was 304. Missing marker and pseudo marker genotypes were inferred from flanking marker information using the multipoint method (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Jiang and Zeng 1997</a>). For example, the genotype indicator variable for line <em>j</em> at marker <em>k</em> is defined as<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>Let <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-3.gif"/></noscript></span></span> be the conditional probability of <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-4.gif"/></noscript></span></span> inferred from all markers of the current chromosome. The conditional expectation for a missing <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-5.gif"/></noscript></span></span> is<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-6.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>which will replace <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-7.gif"/></noscript></span></span> if <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-8.gif"/></noscript></span></span> is missing. It should be noted that each marker (true or pseudo) will be treated as a putative QTL in this research, <em>i.e.</em>, only the effects of markers (true and pseudo) will be estimated. If a QTL is located between two markers, its effect would be picked up by the two flanking markers. Hereafter, we use markers (including true and pseudo markers) and putative QTL interchangeably.</p><p id="p-11">The missing phenotype <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-9.gif"/></noscript></span></span> for line <em>j</em> in the <em>i</em>th environment was drawn from the normal distribution with mean <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-10.gif"/></noscript></span></span> and variance <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-11.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-12.gif"/></noscript></span></span> is the average of phenotypes of all lines in the <em>i</em>th environment and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-13.gif"/></noscript></span></span> is the variance of phenotypes across lines in the <em>i</em>th environment. The seven traits were analyzed separately in seven analyses. The numbers of environments whose data contribute to the analysis for all traits are given in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>.</p></div><div id="sec-3" class="subsection"><h3>Statistical method</h3><p id="p-12">Let <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-14.gif"/></noscript></span></span> be the phenotypic values for line <em>j</em> collected in <em>m</em> environments. The multivariate model for <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-15.gif"/></noscript></span></span> is<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-16.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-17.gif"/></noscript></span></span> is a vector of intercepts, <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-18.gif"/></noscript></span></span> is a numerically coded genotypic indicator variable for line <em>j</em> at locus <em>k</em>, for <em>k</em> = 1,<strong>…</strong>, <em>p</em>, where <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-19.gif"/></noscript></span></span> is the number of markers, including true and pseudo markers, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-20.gif"/></noscript></span></span> is an <em>m</em>×1 vector of QTL effects for the <em>m</em> environments and the residual error vector <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-21.gif"/></noscript></span></span> is assumed to be multivariate normal with density <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-22.gif"/></noscript></span></span>, where <em>θ</em> is an <em>m</em>×<em>m</em> variance –covariance matrix. The variance- covariance structures of residual errors considered are identity, diagonal and factor analytic structured matrices.</p><p id="p-13">For locus <em>k</em>, the mean of the marker effects cross the environments <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-23.gif"/></noscript></span></span> represents the main effects and the variance of the marker effects <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-24.gif"/></noscript></span></span> represents the <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-25.gif"/></noscript></span></span> interaction. We used<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-26.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>to represent the uniformity of the QTL effects cross environments. When there is no <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-27.gif"/></noscript></span></span> interaction, we expect that <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-28.gif"/></noscript></span></span> and thus <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-29.gif"/></noscript></span></span>. Since the method is a Bayesian method, prior and posterior distributions of parameters are required in the analysis. These distributions have been described in detail by <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Chen <em>et al.</em> (2010)</a> and thus are not provided here in this report.</p></div><div id="sec-4" class="subsection"><h3>MCMC sampling</h3><p id="p-14">The MCMC-implemented Bayesian analysis developed by <a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">Chen <em>et al.</em> (2010)</a> was used for parameter estimation. The overall Markov chain contained 160,000 iterations. The first 40,000 iterations were deleted from the post MCMC analysis (burn-in deletion). Thereafter, the observations were saved in every 40 iterations to reduce the autocorrelation among different observations of the posterior samples. Therefore, the final posterior sample for analysis contained 3000 observations. Posterior means and other posterior statistics were obtained from the posterior sample of the 3000 observations.</p></div><div id="sec-5" class="subsection"><h3>Model selection</h3><p id="p-15">We used five different variance-covariance structures to analyze the seven traits, which were (1) the homogeneous variance <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-30.gif"/></noscript></span></span>, (2) the heterogeneous variance <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-31.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-32.gif"/></noscript></span></span> is a diagonal matrix with each diagonal element representing the residual error variance for the trait in that environment, (3) the first-order factor analytic structure <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-33.gif"/></noscript></span></span>, where <em>B</em> is factor loading, (4) the second-order factor analytic structure <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-34.gif"/></noscript></span></span>, and (5) the third-order factor analytic structure <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-35.gif"/></noscript></span></span>. In order to select the optimal covariance structures for fitting the each trait, we used the Bayesian information criteria (BIC) to evaluate the performance of the five models. The BIC score was calculated using<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-36.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where <em>L</em> is the likelihood function, <em>p</em> is the number of parameters, and <em>n</em> is the sample size. The BIC scores for the seven traits are shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. As seen in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, the minimum BIC scores of the lodging and maturity traits are under the first-order factor scenario and those of the remaining traits with the heterogeneous residual variance structure model reach the minimum values. These indicate that the first-order factor analytic model outperforms the other models for lodging and maturity traits and the heterogeneous residual variance structure model for the remaining traits. Therefore, we will analyze the lodging and maturity traits using the first-order factor analytic model and the remaining traits with heterogeneous residual variance structure model in the subsequent studies.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461647/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461647/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">BIC scores of the five variance-covariance structures for the seven agronomic traits of barley in the Harrington×TR306 double haploid population</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Permutation test</h3><p id="p-16">To test the significance of the QTL effects, we conducted a permutation test to generate the null distribution of each main effect and each Q×E interaction. In the permutation analysis, we repeated the MCMC sampling method but randomly shuffled the phenotypes to artificially destroy the association between the markers and the phenotypes. This permutation test for Bayesian shrinkage analysis was proposed by <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Che and Xu (2010)</a>. We adopted this permutation analysis to draw the critical values for significance tests. The number of iterations for the permutation analysis was the same as that described for the original data analysis. The posterior distribution for each <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-37.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-38.gif"/></noscript></span></span> was inferred from the permuted posterior samples to draw the null distribution. An actual estimate of <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-39.gif"/></noscript></span></span> beyond the 0.5% and 99.5% interval indicates significant main effect while an actual estimate of <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-40.gif"/></noscript></span></span> larger than the 99% of the null distribution indicates significant <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-41.gif"/></noscript></span></span> interaction.</p></div></div><div class="section results" id="sec-7"><h2 class="">RESULTS</h2><p id="p-17">We wrote a SAS/IML program to implement the Bayesian analysis. The program took about 36∼54 CPU hours to complete the entire analysis for each trait on a Pentium PC with a 3.6-GHz processor and 3.00 GB RAM. The estimated QTL effect profiles, including the main effect and <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-42.gif"/></noscript></span></span> interaction, are depicted in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> for the trait yield. The figure also includes the 99% confidence interval profiles of the null distribution generated from the permutation analysis. The estimated QTL effect profiles for the remaining six traits are shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>, and <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>. QTL main and interaction effects have been detected for each of the seven traits. However, no regions of the genome show evidence of a locus affecting all traits. One region near the end of the plus arm of chromosome 7 affected six traits (excluding heading), but the magnitudes of effects varied across traits. The results of this analysis are summarized in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>. The number of main effects varied from 26 (maturity) to 75 (heading) with an average of 61.86. Apparently, heading has the most detected effects with a total of 98 effects (75 main and 29 Q×E effects). Among the 98 effects, 6 loci had both the main and Q×E effects. Among the total number of detected loci, on average, 78.5% of the loci show the main effects, and 34.9% of the loci show Q×E interaction. This means that main effects contribute more of the genetic variance than the Q×E interaction effects. Other summary statistics can be found in the last column of <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>. Overall, we detected many loci with either the main or the Q×E effects and the main effects appear to be more important than the Q×E interaction for all the seven traits. This means that most detected loci have a constant effect across environments. Another discovery from this analysis is that Q×E interaction occurs independently, regardless whether the locus has main effect or not. Traditional analysis that evaluates Q×E only on those loci that show significant main effects should be abandoned because that approach will fail to detect a large number of Q×E effects that show no main effects. <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figures 1</a>–<a id="xref-fig-7-2" class="xref-fig" href="#F7">7</a> give the summary of the discovery of this study. Detailed information regarding the actual estimated effects and locations of the significance loci are provided in the tables in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002980/-/DC1/FileS1.xls">File S1</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/7/779/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimated main QTL effects (upper panel) and Q&#xD7;E interaction effects (lower panel) across the barley genome for the yield trait. Chromosomes are separated by the vertical reference lines. The needles (in blue) represent the effects and the curves (in red) represent the 99% confidence intervals generated from a permutation analysis. The ticks on the horizontal axis indicate the marker positions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1083988261" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimated main QTL effects (upper panel) and Q&#xD7;E interaction effects (lower panel) across the barley genome for the yield trait. Chromosomes are separated by the vertical reference lines. The needles (in blue) represent the effects and the curves (in red) represent the 99% confidence intervals generated from a permutation analysis. The ticks on the horizontal axis indicate the marker positions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/7/779/F1.medium.gif" width="440" height="288"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/2/7/779/F1.medium.gif" width="440" height="288"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/7/779/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/7/779/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461564" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-18" class="first-child">Estimated main QTL effects (upper panel) and Q×E interaction effects (lower panel) across the barley genome for the yield trait. Chromosomes are separated by the vertical reference lines. The needles (in blue) represent the effects and the curves (in red) represent the 99% confidence intervals generated from a permutation analysis. The ticks on the horizontal axis indicate the marker positions.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/7/779/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimated main QTL effects (upper panel) and Q&#xD7;E interaction effects (lower panel) across the barley genome for the test weight trait. Chromosomes are separated by the vertical reference lines. The needles (in blue) represent the effects and the curves (in red) represent the 99% confidence intervals generated from a permutation analysis. The ticks on the horizontal axis indicate the marker positions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1083988261" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimated main QTL effects (upper panel) and Q&#xD7;E interaction effects (lower panel) across the barley genome for the test weight trait. Chromosomes are separated by the vertical reference lines. The needles (in blue) represent the effects and the curves (in red) represent the 99% confidence intervals generated from a permutation analysis. The ticks on the horizontal axis indicate the marker positions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/7/779/F2.medium.gif" width="440" height="298"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/2/7/779/F2.medium.gif" width="440" height="298"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/7/779/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/7/779/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461658" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-19" class="first-child">Estimated main QTL effects (upper panel) and Q×E interaction effects (lower panel) across the barley genome for the test weight trait. Chromosomes are separated by the vertical reference lines. The needles (in blue) represent the effects and the curves (in red) represent the 99% confidence intervals generated from a permutation analysis. The ticks on the horizontal axis indicate the marker positions.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/7/779/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimated main QTL effects (upper panel) and Q&#xD7;E interaction effects (lower panel) across the barley genome for the maturity trait. Chromosomes are separated by the vertical reference lines. The needles (in blue) represent the effects and the curves (in red) represent the 99% confidence intervals generated from a permutation analysis. The ticks on the horizontal axis indicate the marker positions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1083988261" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimated main QTL effects (upper panel) and Q&#xD7;E interaction effects (lower panel) across the barley genome for the maturity trait. Chromosomes are separated by the vertical reference lines. The needles (in blue) represent the effects and the curves (in red) represent the 99% confidence intervals generated from a permutation analysis. The ticks on the horizontal axis indicate the marker positions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/7/779/F3.medium.gif" width="440" height="292"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/2/7/779/F3.medium.gif" width="440" height="292"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/7/779/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/7/779/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461618" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-20" class="first-child">Estimated main QTL effects (upper panel) and Q×E interaction effects (lower panel) across the barley genome for the maturity trait. Chromosomes are separated by the vertical reference lines. The needles (in blue) represent the effects and the curves (in red) represent the 99% confidence intervals generated from a permutation analysis. The ticks on the horizontal axis indicate the marker positions.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/7/779/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimated main QTL effects (upper panel) and Q&#xD7;E interaction effects (lower panel) across the barley genome for the lodging trait. Chromosomes are separated by the vertical reference lines. The needles (in blue) represent the effects and the curves (in red) represent the 99% confidence intervals generated from a permutation analysis. The ticks on the horizontal axis indicate the marker positions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1083988261" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimated main QTL effects (upper panel) and Q&#xD7;E interaction effects (lower panel) across the barley genome for the lodging trait. Chromosomes are separated by the vertical reference lines. The needles (in blue) represent the effects and the curves (in red) represent the 99% confidence intervals generated from a permutation analysis. The ticks on the horizontal axis indicate the marker positions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/7/779/F4.medium.gif" width="440" height="293"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/2/7/779/F4.medium.gif" width="440" height="293"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/7/779/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/7/779/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461612" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-21" class="first-child">Estimated main QTL effects (upper panel) and Q×E interaction effects (lower panel) across the barley genome for the lodging trait. Chromosomes are separated by the vertical reference lines. The needles (in blue) represent the effects and the curves (in red) represent the 99% confidence intervals generated from a permutation analysis. The ticks on the horizontal axis indicate the marker positions.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/7/779/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimated main QTL effects (upper panel) and Q&#xD7;E interaction effects (lower panel) across the barley genome for the kernel weight. Chromosomes are separated by the vertical reference lines. The needles (in blue) represent the effects and the curves (in red) represent the 99% confidence intervals generated from a permutation analysis. The ticks on the horizontal axis indicate the marker positions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1083988261" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimated main QTL effects (upper panel) and Q&#xD7;E interaction effects (lower panel) across the barley genome for the kernel weight. Chromosomes are separated by the vertical reference lines. The needles (in blue) represent the effects and the curves (in red) represent the 99% confidence intervals generated from a permutation analysis. The ticks on the horizontal axis indicate the marker positions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/7/779/F5.medium.gif" width="440" height="291"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/2/7/779/F5.medium.gif" width="440" height="291"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/7/779/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/7/779/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461560" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-22" class="first-child">Estimated main QTL effects (upper panel) and Q×E interaction effects (lower panel) across the barley genome for the kernel weight. Chromosomes are separated by the vertical reference lines. The needles (in blue) represent the effects and the curves (in red) represent the 99% confidence intervals generated from a permutation analysis. The ticks on the horizontal axis indicate the marker positions.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/7/779/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimated main QTL effects (upper panel) and Q&#xD7;E interaction effects (lower panel) across the barley genome for plant height. Chromosomes are separated by the vertical reference lines. The needles (in blue) represent the effects and the curves (in red) represent the 99% confidence intervals generated from a permutation analysis. The ticks on the horizontal axis indicate the marker positions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1083988261" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimated main QTL effects (upper panel) and Q&#xD7;E interaction effects (lower panel) across the barley genome for plant height. Chromosomes are separated by the vertical reference lines. The needles (in blue) represent the effects and the curves (in red) represent the 99% confidence intervals generated from a permutation analysis. The ticks on the horizontal axis indicate the marker positions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/7/779/F6.medium.gif" width="440" height="294"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/2/7/779/F6.medium.gif" width="440" height="294"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/7/779/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/7/779/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461593" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-23" class="first-child">Estimated main QTL effects (upper panel) and Q×E interaction effects (lower panel) across the barley genome for plant height. Chromosomes are separated by the vertical reference lines. The needles (in blue) represent the effects and the curves (in red) represent the 99% confidence intervals generated from a permutation analysis. The ticks on the horizontal axis indicate the marker positions.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/7/779/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimated main QTL effects (upper panel) and Q&#xD7;E interaction effects (lower panel) across the barley genome for the date of heading. Chromosomes are separated by the vertical reference lines. The needles (in blue) represent the effects and the curves (in red) represent the 99% confidence intervals generated from a permutation analysis. The ticks on the horizontal axis indicate the marker positions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1083988261" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimated main QTL effects (upper panel) and Q&#xD7;E interaction effects (lower panel) across the barley genome for the date of heading. Chromosomes are separated by the vertical reference lines. The needles (in blue) represent the effects and the curves (in red) represent the 99% confidence intervals generated from a permutation analysis. The ticks on the horizontal axis indicate the marker positions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/7/779/F7.medium.gif" width="440" height="291"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/2/7/779/F7.medium.gif" width="440" height="291"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/7/779/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/7/779/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461666" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-24" class="first-child">Estimated main QTL effects (upper panel) and Q×E interaction effects (lower panel) across the barley genome for the date of heading. Chromosomes are separated by the vertical reference lines. The needles (in blue) represent the effects and the curves (in red) represent the 99% confidence intervals generated from a permutation analysis. The ticks on the horizontal axis indicate the marker positions.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461621/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461621/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary statistics for the seven agronomic traits of barley in the Harrington×TR306 double haploid population using Bayesian mapping</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">Responding to a reviewer’s comment, we also used the mixed-model approach of <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Piepho (2005)</a> to analyze these data for comparison. This method is single-point analysis. The linear model is<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-68.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-69.gif"/></noscript></span></span> is the general mean at the <em>k</em>th locus, <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-70.gif"/></noscript></span></span> is the QTL main effect at the <em>k</em>th locus (treated as fixed effect), <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-71.gif"/></noscript></span></span> is the <em>m</em>th environmental main effect at the <em>k</em>th locus (treated as random), <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-72.gif"/></noscript></span></span> is the <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-73.gif"/></noscript></span></span> interaction effect in the <em>m</em>th environment at the <em>k</em>th locus (treated as random).</p><p id="p-27">The mixed-model approach of <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Piepho (2005)</a> was implemented using the MIXED procedure of the SAS system (SAS Institute, Cary, NC). As an example, we only presented in detail the result of the trait named yield. The estimated QTL main effects and Q×E interaction effects are depicted in <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>, where the top panel gives the estimated QTL main effects and the bottom panel shows the estimated Q×E interaction effects. This approach validated most of loci with QTL and Q×E interaction effects detected by the Bayesian analysis except a few places where the Bayesian analysis had extra peaks. For example, the two QTL with the effects in opposite directions in the middle of chromosome 2 detected by the Bayesian analysis disappeared in the mixed-model approach of <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Piepho (2005)</a>. Moreover, the magnitudes of the estimated QTL main effects and Q×E interaction effects are larger than those estimated by the Bayesian analysis. This comparison showed that the Bayesian analysis may be advantageous over the mixed-model approach. Note that this study is to report the Bayesian analysis of the barley data, not for the purpose of comparing different methods.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/7/779/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimated main QTL effects (upper panel) and Q&#xD7;E interaction effects (lower panel) across the barley genome for the yield trait using the mixed method approach of Piepho (2005). Chromosomes are separated by the vertical reference lines. The needles (in blue) represent the effects." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1083988261" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimated main QTL effects (upper panel) and Q&#xD7;E interaction effects (lower panel) across the barley genome for the yield trait using the mixed method approach of Piepho (2005). Chromosomes are separated by the vertical reference lines. The needles (in blue) represent the effects.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/7/779/F8.medium.gif" width="440" height="291"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/2/7/779/F8.medium.gif" width="440" height="291"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/7/779/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/7/779/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461533" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-28" class="first-child">Estimated main QTL effects (upper panel) and Q×E interaction effects (lower panel) across the barley genome for the yield trait using the mixed method approach of <a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Piepho (2005)</a>. Chromosomes are separated by the vertical reference lines. The needles (in blue) represent the effects.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section discussion" id="sec-8"><h2 class="">DISCUSSION</h2><p id="p-29">We used the new Bayesian method developed by <a id="xref-ref-3-7" class="xref-bibr" href="#ref-3">Chen <em>et al.</em> (2010)</a> for Q×E interaction. In the original study, the authors used one trait (yield) as an example to demonstrate the method and test the computer program. In this study, we emphasize the application of this new method to Q×E detection for all seven traits measured in the experiment. This is an application study using an advanced statistical method. The 8 main effects and 18 Q×E interaction effects detected by <a id="xref-ref-3-8" class="xref-bibr" href="#ref-3">Chen <em>et al.</em> (2010)</a> for yield were all validated by this study. Furthermore, we detected more markers with main and Q×E interaction effects. The difference between the original study of <a id="xref-ref-3-9" class="xref-bibr" href="#ref-3">Chen <em>et al.</em> (2010)</a> and this study deserves further discussion. <a id="xref-ref-3-10" class="xref-bibr" href="#ref-3">Chen <em>et al.</em> (2010)</a> used 127 markers (all true markers), and they only estimated 127 main effects and 127 Q×E interaction effects. In this study, we used 304 markers (127 true markers and 177 pseudo markers) and estimated 304 main effects and 304 Q×E interaction effects. The additional effects we detected were largely contributed by the inserted pseudo markers. For example, many additional peaks that appeared in this study coincided with pseudo markers in large marker intervals. It was impossible for <a id="xref-ref-3-11" class="xref-bibr" href="#ref-3">Chen <em>et al.</em> (2010)</a> to detect these additional effects because they only analyzed the 127 true markers, not the 177 pseudo markers. Comparison of the yield trait between this study and that of <a id="xref-ref-3-12" class="xref-bibr" href="#ref-3">Chen <em>et al.</em> (2010)</a> clearly demonstrated the advantage of the pseudo marker insertion approach.</p><p id="p-30">In the original barley QTL mapping experiment, the authors (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Tinker <em>et al.</em> 1996</a>) provided some preliminary results about Q×E interactions using the interval mapping (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Lander and Botstein 1989</a>) and composite interval mapping approaches (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Zeng 1994</a>). The difference between the interval mapping and the Bayesian mapping in the context of Q×E is that the Bayesian method uses a multivariate (multiple environments in this case) and multiple QTL model. All parameters and significance tests are performed simultaneously within the same model. Advantages of the simultaneous analysis over the one-environment-and-one-marker analysis are 3-fold: (a) conclusions are drawn from the result of the same analysis, rather than from separate analyses; (b) significance tests are conducted simultaneously in one experiment, and thus no adjustment is required regarding the experiment-wise Type I error rate; and (c) more effects can be detected using the joint analysis. We examined the Q×E interaction effects detected from the <em>ad hoc</em> interval mapping studies by the original authors of the barley experiment (<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Tinker <em>et al.</em> 1996</a>) and found that almost all effects detected by <a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">Tinker <em>et al.</em> (1996)</a> were also detected in this study. The comparison is summarized in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. Note that the numbers of QTL main effects and Q×E effects were counted directly from <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> of <a id="xref-ref-17-7" class="xref-bibr" href="#ref-17">Tinker <em>et al.</em> (1996)</a>. In addition, we detected more Q×E effects (see the tables in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002980/-/DC1/FileS1.xls">File S1</a> for detailed information about the results).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461634/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461634/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">The numbers of QTL and Q×E interactions for the seven agronomic traits of barley in the Harrington×TR306 double haploid population detected by the interval mapping approach and validated by the Bayesian approach</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">In contrast to common Q×E studies (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Boer <em>et al.</em> 2007</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Cherif <em>et al.</em> 2010</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Hayes <em>et al.</em> 1993</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Jansen <em>et al.</em> 1995</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Li <em>et al.</em> 2007</a>, <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">2010</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Paterson <em>et al.</em> 2003</a>; Piepho 2000; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Tinker and Mather 1995</a>; <a id="xref-ref-17-10" class="xref-bibr" href="#ref-17">Tinker <em>et al.</em> 1996</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Yang <em>et al.</em> 2007</a>) in which QTL effects from each environment are estimated and tested, here we used the variance (a single value) of the QTL effects across all environments as a measurement of Q×E interaction. The test is now based on a single value of variance for each locus. This automatically converts the traditional fixed-model analysis into a random model analysis because we are now estimating and testing a variance component. This random model treatment has eliminated many problems related to the fixed-model analysis. For example, there is little concern about model saturation (overfitting) under the random model framework. When the number of environments increases, the number of parameters to be tested remains the same because there is still only one variance <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7/779/embed/mml-math-82.gif"/></noscript></span></span> per locus.</p><p id="p-33">In this research, we used the permutation tests proposed by <a id="xref-ref-3-13" class="xref-bibr" href="#ref-3">Chen <em>et al.</em> (2010)</a> for drawing the empirical critical values of the test statistics to decide the significance of the detected QTL effects. In theory, significance test are not required in Bayesian analysis. However, the purpose of this study was to evaluate the genome-wide QTL main effects and Q×E interaction effects. We expected that some estimated QTL main effects and Q×E interaction effects might be detected by chance and thus wanted to exclude the noisy effects. In addition, the Bayesian mapping of <a id="xref-ref-3-14" class="xref-bibr" href="#ref-3">Chen <em>et al.</em> (2010)</a> was developed based on a hierarchical model and it is a hybrid method between the Bayesian and the frequentist approaches. The asymptotic theory derived based on the frequentist approach may not apply to the hierarchical model. Therefore, we employed the permutation test to separate the true QTL from the spurious QTL.</p><p id="p-34">Understanding the relative importance of Q×E interaction is useful in plant breeding. The way we formulated the Q×E interaction is unique because it is measured as a variance component. The variance across environments reflects the stability of a QTL, in which a larger variance indicates a lower stability of the QTL. From the breeding point of view, understanding Q×E interaction is important in marker-assisted selection and breeding. A QTL with less or no Q×E interaction can be utilized in a broad range of conditions, whereas QTL with significant Q×E interaction can only be used in the specific environment in which it is detected.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-35">We greatly thank two anonymous reviewers for their useful comments and suggestions on an earlier version of the manuscript. This project was supported by the Agriculture and Food Research Initiative of U.S.D.A. National Institute of Food and Agriculture 2007-02784 (to S.X.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></li><li class="fn" id="fn-2"><p id="p-2">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002980/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002980/-/DC1</a></p></li><li class="fn" id="fn-6"><p id="p-36"><em>Communicating editor: B. S. Yandell</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-02-25"><span class="received-label">Received </span>February 25, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-05-07"><span class="accepted-label">Accepted </span>May 7, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Zhao, Xu</li></ul><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.779.1" data-doi="10.1534/genetics.107.071068"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boer</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Podlich</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A mixed-model quantitative trait loci (QTL) analysis for multiple-environment trial data using environmental covariables for QTL-by-environment interactions, with an example in maize</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">1801</span>–<span class="cit-lpage">1813</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D177%26rft.spage%253D1801%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.071068%26rft_id%253Dinfo%253Apmid%252F17947443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.107.071068&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17947443&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251368800043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.779.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Che</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Significance test and genome selection in Bayesian shrinkage analysis</span>. <abbr class="cit-jnl-abbrev">Int. J. Plant Genomics</abbr> 2010: Article ID 893206.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.779.3" data-doi="10.1534/genetics.110.120311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mapping environment specific quantitative trait loci</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">1053</span>–<span class="cit-lpage">1066</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D186%26rft.spage%253D1053%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.120311%26rft_id%253Dinfo%253Apmid%252F20805558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.110.120311&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20805558&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.779.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cherif</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rezgui</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devaux</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harrabi</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genotype × environment interactions and heritability of quantitative resistance to net blotch in Tunisian barley</span>. <abbr class="cit-jnl-abbrev">Journal of Plant Breeding and Crop Science</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">110</span>–<span class="cit-lpage">116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BPlant%2BBreeding%2Band%2BCrop%2BScience%26rft.volume%253D2%26rft.spage%253D110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.779.5" data-doi="10.1007/BF01184929"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knapp</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Quantitative trait locus effects and environmental interaction in a sample of North American barley germ plasm</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">392</span>–<span class="cit-lpage">401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D87%26rft.spage%253D392%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01184929%26rft_id%253Dinfo%253Apmid%252F24190268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01184929&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24190268&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993MK06900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.779.6" data-doi="10.1007/BF00220855"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ooijen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lister</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Genotype-by-environment interaction in genetic mapping of multiple quantitative trait loci</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D91%26rft.spage%253D33%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00220855%26rft_id%253Dinfo%253Apmid%252F24169664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00220855&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24169664&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RJ83400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.779.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Multiple trait analysis of genetic mapping for quantitative trait loci</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">1111</span>–<span class="cit-lpage">1127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D140%26rft.spage%253D1111%26rft_id%253Dinfo%253Apmid%252F7672582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=7672582&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RE86800023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.779.8" data-doi="10.1023/A:1018394410659"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Mapping quantitative trait loci with dominant and missing markers in various crosses from two inbred lines</span>. <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D101%26rft.spage%253D47%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1018394410659%26rft_id%253Dinfo%253Apmid%252F9465409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1018394410659&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9465409&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071700000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.779.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Mapping Mendelian factors underlying quantitative traits using RFLP linkage maps</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D121%26rft.spage%253D185%26rft_id%253Dinfo%253Apmid%252F2563713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2563713&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989R676700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.779.10" data-doi="10.1186/1746-4811-6-13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gai</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Functional mapping of genotype-environment interactions for soybean growth by a semiparametric approach</span>. <abbr class="cit-jnl-abbrev">Plant Methods</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BMethods%26rft.aulast%253DLi%26rft.auinit1%253DQ.%26rft.volume%253D6%26rft.spage%253D13%26rft.epage%253D13%26rft.atitle%253DFunctional%2Bmapping%2Bof%2Bgenotype-environment%2Binteractions%2Bfor%2Bsoybean%2Bgrowth%2Bby%2Ba%2Bsemiparametric%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1746-4811-6-13%26rft_id%253Dinfo%253Apmid%252F20525184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1746-4811-6-13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20525184&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.779.11" data-doi="10.1111/j.1439-0523.2007.01409.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Dynamic analyses of rice blast resistance for the assessment of genetic and environmental effects</span>. <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">541</span>–<span class="cit-lpage">547</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D126%26rft.spage%253D541%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0523.2007.01409.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0523.2007.01409.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.779.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saranga</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Menz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">C. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">QTL analysis of genotype× environment interactions affecting cotton fiber quality</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">384</span>–<span class="cit-lpage">396</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DPaterson%26rft.auinit1%253DA.%2BH.%26rft.volume%253D106%26rft.issue%253D3%26rft.spage%253D384%26rft.epage%253D396%26rft.atitle%253DQTL%2Banalysis%2Bof%2Bgenotype%2Bx%2Benvironment%2Binteractions%2Baffecting%2Bcotton%2Bfiber%2Bquality.%26rft_id%253Dinfo%253Apmid%252F12589538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12589538&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181521500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2.7.779.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A mixed-model approach to mapping quantitative trait loci in barley on the basis of multiple environment data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">2043</span>–<span class="cit-lpage">2050</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D156%26rft.spage%253D2043%26rft_id%253Dinfo%253Apmid%252F11102394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11102394&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165740100047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.779.14" data-doi="10.1007/s00122-004-1872-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Statistical tests for QTL and QTL-by-environment effects in segregating populations derived from line crosses</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">561</span>–<span class="cit-lpage">566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DPiepho%26rft.auinit1%253DH.%2BP.%26rft.volume%253D110%26rft.issue%253D3%26rft.spage%253D561%26rft.epage%253D566%26rft.atitle%253DStatistical%2Btests%2Bfor%2BQTL%2Band%2BQTL-by-environment%2Beffects%2Bin%2Bsegregating%2Bpopulations%2Bderived%2Bfrom%2Bline%2Bcrosses.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-004-1872-9%26rft_id%253Dinfo%253Apmid%252F15655665%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-004-1872-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15655665&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.779.15" data-doi="10.1007/s00122-003-1253-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pillen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zacharias</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leon</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Advanced backcross QTL analysis in barley (Hordeum vulgare L.)</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">340</span>–<span class="cit-lpage">352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DPillen%26rft.auinit1%253DK.%26rft.volume%253D107%26rft.issue%253D2%26rft.spage%253D340%26rft.epage%253D352%26rft.atitle%253DAdvanced%2Bbackcross%2BQTL%2Banalysis%2Bin%2Bbarley%2B%2528Hordeum%2Bvulgare%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-003-1253-9%26rft_id%253Dinfo%253Apmid%252F12677407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-003-1253-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12677407&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184029100018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-other" id="cit-2.7.779.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tinker</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mather</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-comment">Methods for QTL analysis with progeny replicated in multiple environments. Journal of Agricultural Genomics 1. Available at: <a href="http://wheat.pw.usda.gov/jag/papers95/paper195/jqtl15.html">http://wheat.pw.usda.gov/jag/papers95/paper195/jqtl15.html</a></span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.779.17" data-doi="10.2135/cropsci1996.0011183X003600040040x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tinker</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mather</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rossnagel</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasha</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleinhofs</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Regions of the genome that affect agronomic performance in two-row barley</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1053</span>–<span class="cit-lpage">1062</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DTinker%26rft.auinit1%253DN.%2BA.%26rft.volume%253D36%26rft.issue%253D4%26rft.spage%253D1053%26rft.epage%253D1062%26rft.atitle%253DRegions%2Bof%2Bthe%2BGenome%2Bthat%2BAffect%2BAgronomic%2BPerformance%2Bin%2BTwo-Row%2BBarley%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1996.0011183X003600040040x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1996.0011183X003600040040x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996UV54900040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.779.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Utz</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">PLABQTL: a program for composite interval mapping of QTL</span>. <abbr class="cit-jnl-abbrev">Journal of Agricultural Genomics</abbr>. <span class="cit-comment">Available at: <a href="http://wheat.pw.usda.gov/jag/papers96/paper196/indexp196.html">http://wheat.pw.usda.gov/jag/papers96/paper196/indexp196.html</a></span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.779.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Utz</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schon</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Bias and sampling error of the estimated proportion of genotypic variance explained by quantitative trait loci determined from experimental data in maize using cross validation and validation with independent samples</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">1839</span>–<span class="cit-lpage">1849</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D154%26rft.spage%253D1839%26rft_id%253Dinfo%253Apmid%252F10866652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10866652&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.779.20" data-doi="10.1534/genetics.106.068361"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jing</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">X. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Identification of quantitative trait loci and environmental interactions for accumulation and remobilization of water-soluble carbohydrates in wheat (Triticum aestivum L.) stems</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">571</span>–<span class="cit-lpage">584</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D176%26rft.spage%253D571%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.068361%26rft_id%253Dinfo%253Apmid%252F17287530%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.106.068361&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17287530&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246960900047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.779.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Precision mapping of quantitative trait loci</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">1457</span>–<span class="cit-lpage">1468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D136%26rft.spage%253D1457%26rft_id%253Dinfo%253Apmid%252F8013918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8013918&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994NB82100022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/7/779.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/7/769" title="Genetic Diversity and Population Structure of Chinese Foxtail Millet [Setaria italica (L.) Beauv.] Landraces" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/7/789" title="Comparison of Parallel High-Throughput RNA Sequencing Between Knockout of TDP-43 and Its Overexpression Reveals Primarily Nonreciprocal and Nonoverlapping Gene Expression Changes in the Central Nervous System of Drosophila" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/7"><span>Volume 2 Issue 7, July 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/7.cover-source.jpg?itok=tq8UCk63" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/7/779" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CrllmMSOCa-Z4f2Io47zoyiBxeLilOWH7VIW4yVjT4A" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/7/779" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/7/779" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/7/779" class="active" data-icon-position="" data-hide-link-title="0">Genotype by Environment Interaction of Quantitative Traits: A Case Study in Barley</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459299" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-SlsNEmuNmpruX-hiSBw9CbgzDbK-3cbiFurmBifh1wo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/7/779.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/7/779" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-nuWz0vy8btJWzZG4aNfNfHZ0qICO5gQ4yjGwDgEcgrA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/7/779" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/7/779.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459299" id="node459299--2" data-pisa="ggg;2/7/779" data-pisa-master="ggg;2/7/779" data-apath="/ggg/2/7/779.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genotype by Environment Interaction of Quantitative Traits: A Case Study in Barley</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Fuping Zhao</span> and <span class="highwire-citation-author" data-delta="1">Shizhong Xu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">779-788;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002980 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;2/7/779" data-atom-uri="/ggg/2/7/779.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459299--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Fuping Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Botany and Plant Sciences, University of California, Riverside, California 92521</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fuping%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFuping%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Shizhong Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Botany and Plant Sciences, University of California, Riverside, California 92521</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shizhong%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShizhong%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:shizhong.xu@ucr.edu" class="" data-icon-position="" data-hide-link-title="0">shizhong.xu@ucr.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/7/779" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/7/779&amp;title=Genotype%20by%20Environment%20Interaction%20of%20Quantitative%20Traits%3A%20A%20Case%20Study%20in%20Barley" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/7/779&amp;title=Genotype%20by%20Environment%20Interaction%20of%20Quantitative%20Traits%3A%20A%20Case%20Study%20in%20Barley" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/7/779&amp;title=Genotype%20by%20Environment%20Interaction%20of%20Quantitative%20Traits%3A%20A%20Case%20Study%20in%20Barley" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/7/779&amp;text=Genotype%20by%20Environment%20Interaction%20of%20Quantitative%20Traits%3A%20A%20Case%20Study%20in%20Barley" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/7/779&amp;title=Genotype%20by%20Environment%20Interaction%20of%20Quantitative%20Traits%3A%20A%20Case%20Study%20in%20Barley" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/7/779&amp;t=Genotype%20by%20Environment%20Interaction%20of%20Quantitative%20Traits%3A%20A%20Case%20Study%20in%20Barley" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/7/779&amp;title=Genotype%20by%20Environment%20Interaction%20of%20Quantitative%20Traits%3A%20A%20Case%20Study%20in%20Barley" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/7/779&amp;title=Genotype%20by%20Environment%20Interaction%20of%20Quantitative%20Traits%3A%20A%20Case%20Study%20in%20Barley" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459299/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459299" id="node459299--3" data-pisa="ggg;2/7/779" data-pisa-master="ggg;2/7/779" data-apath="/ggg/2/7/779.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genotype by Environment Interaction of Quantitative Traits: A Case Study in Barley</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Fuping Zhao</span> and <span class="highwire-citation-author" data-delta="1">Shizhong Xu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">779-788;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002980 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;2/7/779" data-atom-uri="/ggg/2/7/779.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459299--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Fuping Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Botany and Plant Sciences, University of California, Riverside, California 92521</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fuping%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFuping%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Shizhong Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Botany and Plant Sciences, University of California, Riverside, California 92521</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shizhong%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShizhong%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:shizhong.xu@ucr.edu" class="" data-icon-position="" data-hide-link-title="0">shizhong.xu@ucr.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=779&amp;rft.epage=788&amp;rft.atitle=Genotype%20by%20Environment%20Interaction%20of%20Quantitative%20Traits%3A%20A%20Case%20Study%20in%20Barley&amp;rft.volume=2&amp;rft.issue=7&amp;rft.date=2012-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Zhao%2C+Fuping&amp;rft.au=Xu%2C+Shizhong"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459299/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459299/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459299/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459299/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459299/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459299/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459299/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459299/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459299/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459299/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459299/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459299/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-ajax-pane-content pane-highwire-permission-link" >
  
        <h2 class="pane-title"><span class='panels-ajax-pane-title' data-pid='new-37'></span></h2>
    
  
  <div class="pane-content">
    <div class='panels-ajax-pane panels-ajax-pane-new-37' data-pid='new-37'></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/7/779" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">MATERIALS AND METHODS</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">RESULTS</a></li><li class="even"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">DISCUSSION</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/7/779.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/7/779.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/7/779.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/7/779.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
