<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/7/761" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/7/761.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/7/761.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/7/761.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/7/761.atom" />
<meta name="HW.pisa" content="ggg;2/7/761" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Impact of Loci Nature on Estimating Recombination and Mutation Rates in Chlamydia trachomatis" />
<meta name="DC.Identifier" content="10.1534/g3.112.002923" />
<meta name="DC.Date" content="2012-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Ferreira et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The knowledge of the frequency and relative weight of mutation and recombination events in evolution is essential for understanding how microorganisms reach fitted phenotypes. Traditionally, these evolutionary parameters have been inferred by using data from multilocus sequence typing (MLST), which is known to have yielded conflicting results. In the near future, these estimations will certainly be performed by computational analyses of full-genome sequences. However, it is not known whether this approach will yield accurate results as bacterial genomes exhibit heterogeneous representation of loci categories, and it is not clear how loci nature impacts such estimations. Therefore, we assessed how mutation and recombination inferences are shaped by loci with different genetic features, using the bacterium Chlamydia trachomatis as the study model. We found that loci assigning a high number of alleles and positively selected genes yielded nonconvergent estimates and incongruent phylogenies and thus are more prone to confound algorithms. Unexpectedly, for the model under evaluation, housekeeping genes and noncoding regions shaped estimations in a similar manner, which points to a nonrandom role of the latter in C. trachomatis evolution. Although the present results relate to a specific bacterium, we speculate that microbe-specific genomic architectures (such as coding capacity, polymorphism dispersion, and fraction of positively selected loci) may differentially buffer the effect of the confounding factors when estimating recombination and mutation rates and, thus, influence the accuracy of using full-genome sequences for such purpose. This putative bias associated with in silico inferences should be taken into account when discussing the results obtained by the analyses of full-genome sequences, in which the “one size fits all” approach may not be applicable." />
<meta name="DC.Contributor" content="Rita Ferreira" />
<meta name="DC.Contributor" content="Vítor Borges" />
<meta name="DC.Contributor" content="Alexandra Nunes" />
<meta name="DC.Contributor" content="Paulo Jorge Nogueira" />
<meta name="DC.Contributor" content="Maria José Borrego" />
<meta name="DC.Contributor" content="João Paulo Gomes" />
<meta name="article:published_time" content="2012-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Impact of Loci Nature on Estimating Recombination and Mutation Rates in Chlamydia trachomatis" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The knowledge of the frequency and relative weight of mutation and recombination events in evolution is essential for understanding how microorganisms reach fitted phenotypes. Traditionally, these evolutionary parameters have been inferred by using data from multilocus sequence typing (MLST), which is known to have yielded conflicting results. In the near future, these estimations will certainly be performed by computational analyses of full-genome sequences. However, it is not known whether this approach will yield accurate results as bacterial genomes exhibit heterogeneous representation of loci categories, and it is not clear how loci nature impacts such estimations. Therefore, we assessed how mutation and recombination inferences are shaped by loci with different genetic features, using the bacterium &lt;i&gt;Chlamydia trachomatis&lt;/i&gt; as the study model. We found that loci assigning a high number of alleles and positively selected genes yielded nonconvergent estimates and incongruent phylogenies and thus are more prone to confound algorithms. Unexpectedly, for the model under evaluation, housekeeping genes and noncoding regions shaped estimations in a similar manner, which points to a nonrandom role of the latter in &lt;i&gt;C. trachomatis&lt;/i&gt; evolution. Although the present results relate to a specific bacterium, we speculate that microbe-specific genomic architectures (such as coding capacity, polymorphism dispersion, and fraction of positively selected loci) may differentially buffer the effect of the confounding factors when estimating recombination and mutation rates and, thus, influence the accuracy of using full-genome sequences for such purpose. This putative bias associated with &lt;i&gt;in silico&lt;/i&gt; inferences should be taken into account when discussing the results obtained by the analyses of full-genome sequences, in which the “one size fits all” approach may not be applicable.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/07/01" />
<meta name="citation_mjid" content="ggg;2/7/761" />
<meta name="citation_id" content="2/7/761" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/7/761" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/7/761.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/7/761.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/7/761.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.002923" />
<meta name="citation_pmid" content="22870399" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="7" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="761" />
<meta name="citation_lastpage" content="768" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Rita Ferreira" />
<meta name="citation_author_institution" content="Department of Infectious Diseases, National Institute of Health, 1649-016 Lisbon, Portugal" />
<meta name="citation_author" content="Vítor Borges" />
<meta name="citation_author_institution" content="Department of Infectious Diseases, National Institute of Health, 1649-016 Lisbon, Portugal" />
<meta name="citation_author" content="Alexandra Nunes" />
<meta name="citation_author_institution" content="Department of Infectious Diseases, National Institute of Health, 1649-016 Lisbon, Portugal" />
<meta name="citation_author" content="Paulo Jorge Nogueira" />
<meta name="citation_author_institution" content="Institute of Preventive Medicine–Faculty of Medicine, University of Lisbon, 1649-028 Lisbon, Portugal, and" />
<meta name="citation_author_institution" content="General Directorate of Health, Lisbon, 1049-005 Lisbon, Portugal" />
<meta name="citation_author" content="Maria José Borrego" />
<meta name="citation_author_institution" content="Department of Infectious Diseases, National Institute of Health, 1649-016 Lisbon, Portugal" />
<meta name="citation_author" content="João Paulo Gomes" />
<meta name="citation_author_institution" content="Department of Infectious Diseases, National Institute of Health, 1649-016 Lisbon, Portugal" />
<meta name="citation_author_email" content="j.paulo.gomes@insa.min-saude.pt" />
<meta name="citation_reference" content="citation_journal_title=Annual review of microbiology;citation_journal_abbrev=Annu Rev Microbiol;citation_author=M. Achtman;citation_title=Evolution, population structure, and phylogeography of genetically monomorphic bacterial pathogens.;citation_pages=53-70;citation_volume=62;citation_year=2008;citation_pmid=18785837;citation_doi=10.1146/annurev.micro.62.081307.162832" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. Andolfatto;citation_title=Adaptive evolution of non-coding DNA in Drosophila.;citation_pages=1149-1152;citation_volume=437;citation_year=2005;citation_issue=7062;citation_pmid=16237443;citation_doi=10.1038/nature04107" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=P. Awadalla;citation_title=The evolutionary genomics of pathogen recombination.;citation_pages=50-60;citation_volume=4;citation_year=2003;citation_issue=1;citation_pmid=12509753;citation_doi=10.1038/nrg964" />
<meta name="citation_reference" content="citation_journal_title=Journal of Microbiological Methods;citation_journal_abbrev=Journal of Microbiological Methods;citation_author=V. Borges;citation_author=R. Ferreira;citation_author=A. Nunes;citation_author=P. J. Nogueira;citation_author=M. J. Borrego;citation_title=Normalization strategies for real-time expression data in Chlamydia trachomatis.;citation_pages=256-264;citation_volume=82;citation_year=2010;citation_issue=3;citation_pmid=20619305;citation_doi=10.1016/j.mimet.2010.06.013" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=E. C. Bush;citation_author=B. T. Lahn;citation_title=Selective constraint on noncoding regions of hominid genomes.;citation_pages=e73-e73;citation_volume=1;citation_year=2005;citation_issue=7;citation_pmid=16362073;citation_doi=10.1371/journal.pcbi.0010073" />
<meta name="citation_reference" content="citation_journal_title=Infection and Immunity;citation_journal_abbrev=Infect. Immun.;citation_author=J. H. Carlson;citation_author=S. F. Porcella;citation_author=G. McClarty;citation_author=H. D. Caldwell;citation_title=Comparative Genomic Analysis of Chlamydia trachomatis Oculotropic and Genitotropic Strains;citation_pages=6407-6418;citation_volume=73;citation_year=2005;citation_issue=10;citation_pmid=16177312;citation_doi=10.1128/IAI.73.10.6407-6418.2005" />
<meta name="citation_reference" content="citation_journal_title=Annals of the New York Academy of Sciences;citation_journal_abbrev=Ann. N. Y. Acad. Sci.;citation_author=I. N. Clarke;citation_title=Evolution of Chlamydia trachomatis.;citation_pages=E11-E18;citation_volume=1230;citation_year=2011;citation_pmid=22239534;citation_doi=10.1111/j.1749-6632.2011.06194.x" />
<meta name="citation_reference" content="citation_journal_title=Nature medicine;citation_journal_abbrev=Nat Med;citation_author=D. J. Conway;citation_author=D. R. Cavanagh;citation_author=K. Tanabe;citation_author=C. Roper;citation_author=Z. S. Mikes;citation_title=A principal target of human immunity to malaria identified by molecular population genetic and immunological analyses.;citation_pages=689-692;citation_volume=6;citation_year=2000;citation_issue=6;citation_pmid=10835687;citation_doi=10.1038/76272" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_journal_abbrev=Microbiology;citation_author=J. E. Cooper;citation_author=E. J. Feil;citation_title=The phylogeny of Staphylococcus aureus - which genes make the best intra-species markers?;citation_pages=1297-1305;citation_volume=152;citation_year=2006;citation_issue=Pt 5;citation_pmid=16622047;citation_doi=10.1099/mic.0.28620-0" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=A. E. Darling;citation_author=B. Mau;citation_author=F. R. Blattner;citation_author=N. T. Perna;citation_title=Mauve: Multiple Alignment of Conserved Genomic Sequence With Rearrangements;citation_pages=1394-1403;citation_volume=14;citation_year=2004;citation_issue=7;citation_pmid=15231754;citation_doi=10.1101/gr.2289704" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. E. Darling;citation_author=B. Mau;citation_author=N. T. Perna;citation_title=progressiveMauve: multiple genome alignment with gene gain, loss and rearrangement.;citation_pages=e11147-e11147;citation_volume=5;citation_year=2010;citation_issue=6;citation_pmid=20593022;citation_doi=10.1371/journal.pone.0011147" />
<meta name="citation_reference" content="citation_journal_title=Emerging infectious diseases;citation_journal_abbrev=Emerg Infect Dis;citation_author=D. Dean;citation_author=W. J. Bruno;citation_author=R. Wan;citation_author=J. P. Gomes;citation_author=S. Devignot;citation_title=Predicting phenotype and emerging strains among Chlamydia trachomatis infections.;citation_pages=1385-1394;citation_volume=15;citation_year=2009;citation_issue=9;citation_pmid=19788805;citation_doi=10.3201/eid1509.090272" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=H. C. den Bakker;citation_author=X. Didelot;citation_author=E. D. Fortes;citation_author=K. K. Nightingale;citation_author=M. Wiedmann;citation_title=Lineage specific recombination rates and microevolution in Listeria monocytogenes.;citation_pages=277-277;citation_volume=8;citation_year=2008;citation_pmid=18842152;citation_doi=10.1186/1471-2148-8-277" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=X. Didelot;citation_author=D. Falush;citation_title=Inference of bacterial microevolution using multilocus sequence data;citation_pages=1251-1266;citation_volume=175;citation_year=2007;citation_pmid=17151252;citation_doi=10.1534/genetics.106.063305" />
<meta name="citation_reference" content="citation_journal_title=Trends in microbiology;citation_journal_abbrev=Trends Microbiol;citation_author=X. Didelot;citation_author=M. C. Maiden;citation_title=Impact of recombination on bacterial evolution.;citation_pages=315-322;citation_volume=18;citation_year=2010;citation_issue=7;citation_pmid=20452218;citation_doi=10.1016/j.tim.2010.04.002" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=X. Didelot;citation_author=R. Bowden;citation_author=T. Street;citation_author=T. Golubchik;citation_author=C. Spencer;citation_title=Recombination and population structure in Salmonella enterica.;citation_pages=e1002191-e1002191;citation_volume=7;citation_year=2011;citation_issue=7;citation_pmid=21829375;citation_doi=10.1371/journal.pgen.1002191" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. Falush;citation_author=C. Kraft;citation_author=N. S. Taylor;citation_author=P. Correa;citation_author=J. G. Fox;citation_title=Recombination and mutation during long-term gastric colonization by Helicobacter pylori: estimates of clock rates, recombination size, and minimal age;citation_volume=98;citation_year=2001;citation_pmid=11742075;citation_doi=10.1073/pnas.251396098" />
<meta name="citation_reference" content="citation_journal_title=Annual review of microbiology;citation_journal_abbrev=Annu Rev Microbiol;citation_author=E. J. Feil;citation_author=B. G. Spratt;citation_title=RECOMBINATION AND THE POPULATION STRUCTURES OF BACTERIAL PATHOGENS;citation_pages=561-590;citation_volume=55;citation_year=2001;citation_issue=1;citation_pmid=11544367;citation_doi=10.1146/annurev.micro.55.1.561" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=E. J. Feil;citation_author=M. C. Maiden;citation_author=M. Achtman;citation_author=B. G. Spratt;citation_title=The relative contributions of recombination and mutation to the divergence of clones of Neisseria meningitidis;citation_pages=1496-1502;citation_volume=16;citation_year=1999;citation_issue=11;citation_pmid=10555280;citation_doi=10.1093/oxfordjournals.molbev.a026061" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=E. J. Feil;citation_author=E. C. Holmes;citation_author=D. E. Bessen;citation_author=M. S. Chan;citation_author=N. P. Day;citation_title=Recombination within natural populations of pathogenic bacteria: Short-term empirical estimates and long-term phylogenetic consequences;citation_pages=182-187;citation_volume=98;citation_year=2001;citation_issue=1;citation_pmid=11136255;citation_doi=10.1073/pnas.98.1.182" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=E. J. Feil;citation_author=J. E. Cooper;citation_author=H. Grundmann;citation_author=D. A. Robinson;citation_author=M. C. Enright;citation_title=How Clonal Is Staphylococcus aureus?;citation_pages=3307-3316;citation_volume=185;citation_year=2003;citation_issue=11;citation_pmid=12754228;citation_doi=10.1128/JB.185.11.3307-3316.2003" />
<meta name="citation_reference" content="citation_journal_title=Stat. Sci.;citation_author=A. Gelman;citation_author=D. B. Rubin;citation_title=Inference from iterative simulation using multiple sequences;citation_pages=457-511;citation_volume=7;citation_year=1992;citation_doi=10.1214/ss/1177011136" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=J. P. Gomes;citation_author=W. J. Bruno;citation_author=M. J. Borrego;citation_author=D. Dean;citation_title=Recombination in the Genome of Chlamydia trachomatis Involving the Polymorphic Membrane Protein C Gene Relative to ompA and Evidence for Horizontal Gene Transfer;citation_pages=4295-4306;citation_volume=186;citation_year=2004;citation_issue=13;citation_pmid=15205432;citation_doi=10.1128/JB.186.13.4295-4306.2004" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=J. P. Gomes;citation_author=A. Nunes;citation_author=W. J. Bruno;citation_author=M. J. Borrego;citation_author=C. Florindo;citation_title=Polymorphisms in the Nine Polymorphic Membrane Proteins of Chlamydia trachomatis across All Serovars: Evidence for Serovar Da Recombination and Correlation with Tissue Tropism;citation_pages=275-286;citation_volume=188;citation_year=2006;citation_issue=1;citation_pmid=16352844;citation_doi=10.1128/JB.188.1.275-286.2006" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. P. Gomes;citation_author=W. J. Bruno;citation_author=A. Nunes;citation_author=N. Santos;citation_author=C. Florindo;citation_title=Evolution of Chlamydia trachomatis diversity occurs by widespread interstrain recombination involving hotspots;citation_volume=17;citation_year=2007;citation_pmid=17090662;citation_doi=10.1101/gr.5674706" />
<meta name="citation_reference" content="citation_journal_title=Journal of Theoretical Biology;citation_journal_abbrev=Journal of Theoretical Biology;citation_author=W. P. Hanage;citation_author=C. Fraser;citation_author=B. G. Spratt;citation_title=The impact of homologous recombination on the generation of diversity in bacteria.;citation_pages=210-219;citation_volume=239;citation_year=2006;citation_issue=2;citation_pmid=16236325;citation_doi=10.1016/j.jtbi.2005.08.035" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. R. Harris;citation_author=I. N. Clarke;citation_author=H. M. Seth-Smith;citation_author=A. W. Solomon;citation_author=L. T. Cutcliffe;citation_title=Whole-genome analysis of diverse Chlamydia trachomatis strains identifies phylogenetic relationships masked by current clinical typing.;citation_pages=413-419;citation_volume=44;citation_year=2012;citation_issue=4;citation_pmid=22406642;citation_doi=10.1038/ng.2214" />
<meta name="citation_reference" content="citation_journal_abbrev=Biol Direct;citation_author=S. J. Joseph;citation_author=X. Didelot;citation_author=K. Gandhi;citation_author=D. Dean;citation_author=T. D. Read;citation_title=Interplay of recombination and selection in the genomes of Chlamydia trachomatis.;citation_pages=28-28;citation_volume=6;citation_year=2011;citation_pmid=21615910;citation_doi=10.1186/1745-6150-6-28" />
<meta name="citation_reference" content="citation_journal_title=Infect. Immun.;citation_author=B. M. Jeffrey;citation_author=R. J. Suchland;citation_author=K. L. Quinn;citation_author=J. R. Davidson;citation_author=W. E. Stamm;citation_title=Genome sequencing of recent clinical Chlamydia trachomatis strains identifies loci associated with tissue tropism and regions of apparent recombination;citation_volume=78;citation_year=2010;citation_pmid=20308297;citation_doi=10.1128/IAI.01324-09" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=N. L. Kaplan;citation_author=R. R. Hudson;citation_author=C. H. Langley;citation_title=The “hitchhiking effect” revisited;citation_pages=887-899;citation_volume=123;citation_year=1989;citation_pmid=2612899" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. Kennemann;citation_author=X. Didelot;citation_author=T. Aebischer;citation_author=S. Kuhn;citation_author=B. Drescher;citation_title=Helicobacter pylori genome evolution during human infection;citation_volume=108;citation_year=2011;citation_pmid=21383187;citation_doi=10.1073/pnas.1018444108" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=T. Lefébure;citation_author=M. J. Stanhope;citation_title=Evolution of the core and pan-genome of Streptococcus: positive selection, recombination, and genome composition.;citation_pages=R71-R71;citation_volume=8;citation_year=2007;citation_issue=5;citation_pmid=17475002;citation_doi=10.1186/gb-2007-8-5-r71" />
<meta name="citation_reference" content="citation_journal_title=Annual review of microbiology;citation_journal_abbrev=Annu Rev Microbiol;citation_author=M. C. Maiden;citation_title=Multilocus sequence typing of bacteria.;citation_pages=561-588;citation_volume=60;citation_year=2006;citation_pmid=16774461;citation_doi=10.1146/annurev.micro.59.030804.121325" />
<meta name="citation_reference" content="citation_journal_title=Journal of Clinical Microbiology;citation_journal_abbrev=J. Clin. Microbiol.;citation_author=E. Meats;citation_author=E. J. Feil;citation_author=S. Stringer;citation_author=A. J. Cody;citation_author=R. Goldstein;citation_title=Characterization of Encapsulated and Noncapsulated Haemophilus influenzae and Determination of Phylogenetic Relationships by Multilocus Sequence Typing;citation_pages=1623-1636;citation_volume=41;citation_year=2003;citation_issue=4;citation_pmid=12682154;citation_doi=10.1128/JCM.41.4.1623-1636.2003" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=K. L. Millman;citation_author=S. Tavare;citation_author=D. Dean;citation_title=Recombination in the ompA Gene but Not theomcB Gene of Chlamydia Contributes to Serovar-Specific Differences in Tissue Tropism, Immune Surveillance, and Persistence of the Organism;citation_pages=5997-6008;citation_volume=183;citation_year=2001;citation_issue=20;citation_pmid=11567000;citation_doi=10.1128/JB.183.20.5997-6008.2001" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=A. Nunes;citation_author=P. J. Nogueira;citation_author=M. J. Borrego;citation_author=J. P. Gomes;citation_title=Chlamydia trachomatis diversity viewed as a tissue-specific coevolutionary arms race.;citation_pages=R153-R153;citation_volume=9;citation_year=2008;citation_issue=10;citation_pmid=18947394;citation_doi=10.1186/gb-2008-9-10-r153" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=A. Nunes;citation_author=P. J. Nogueira;citation_author=M. J. Borrego;citation_author=J. P. Gomes;citation_title=Adaptive evolution of the Chlamydia trachomatis dominant antigen reveals distinct evolutionary scenarios for B- and T-cell epitopes: worldwide survey;citation_pages=e13171;citation_volume=5;citation_year=2010;citation_pmid=20957150;citation_doi=10.1371/journal.pone.0013171" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=H. Ochman;citation_author=J. G. Lawrence;citation_author=E. A. Groisman;citation_title=Lateral gene transfer and the nature of bacterial innovation.;citation_pages=299-304;citation_volume=405;citation_year=2000;citation_issue=6784;citation_pmid=10830951;citation_doi=10.1038/35012500" />
<meta name="citation_reference" content="citation_journal_title=Infection, genetics and evolution :  journal of molecular epidemiology and evolutionary genetics in infectious diseases;citation_journal_abbrev=Infect Genet Evol;citation_author=M. Pérez-Losada;citation_author=E. B. Browne;citation_author=A. Madsen;citation_author=T. Wirth;citation_author=R. P. Viscidi;citation_title=Population genetics of microbial pathogens estimated from multilocus sequence typing (MLST) data.;citation_pages=97-112;citation_volume=6;citation_year=2006;citation_issue=2;citation_pmid=16503511;citation_doi=10.1016/j.meegid.2005.02.003" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=L. Petersen;citation_author=J. P. Bollback;citation_author=M. Dimmic;citation_author=M. Hubisz;citation_author=R. Nielsen;citation_title=Genes under positive selection in Escherichia coli;citation_volume=17;citation_year=2007;citation_pmid=17675366;citation_doi=10.1101/gr.6254707" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=H. M. Seth-Smith;citation_author=S. R. Harris;citation_author=K. Persson;citation_author=P. Marsh;citation_author=A. Barron;citation_title=Co-evolution of genomes and plasmids within Chlamydia trachomatis and the emergence in Sweden of a new variant strain.;citation_pages=239-239;citation_volume=10;citation_year=2009;citation_pmid=19460133;citation_doi=10.1186/1471-2164-10-239" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=N. H. Smith;citation_author=J. Dale;citation_author=J. Inwald;citation_author=S. Palmer;citation_author=S. V. Gordon;citation_title=The population structure of Mycobacterium bovis in Great Britain: clonal expansion;citation_volume=100;citation_year=2003;citation_pmid=14657373;citation_doi=10.1073/pnas.2036554100" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in microbiology;citation_journal_abbrev=Curr Opin Microbiol;citation_author=B. G. Spratt;citation_author=W. P. Hanage;citation_author=E. J. Feil;citation_title=The relative contributions of recombination and point mutation to the diversification of bacterial clones.;citation_pages=602-606;citation_volume=4;citation_year=2001;citation_issue=5;citation_pmid=11587939;citation_doi=10.1016/S1369-5274(00)00257-5" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. S. Stephens;citation_author=S. Kalman;citation_author=C. Lammel;citation_author=J. Fan;citation_author=R. Marathe;citation_title=Genome Sequence of an Obligate Intracellular Pathogen of Humans: Chlamydia trachomatis;citation_pages=754-759;citation_volume=282;citation_year=1998;citation_issue=5389;citation_pmid=9784136;citation_doi=10.1126/science.282.5389.754" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. P. Stumpf;citation_author=G. A. McVean;citation_title=Estimating recombination rates from population-genetic data.;citation_pages=959-968;citation_volume=4;citation_year=2003;citation_issue=12;citation_pmid=14631356;citation_doi=10.1038/nrg1227" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=P. Supply;citation_author=R. M. Warren;citation_author=A. L. Bañuls;citation_author=S. Lesjean;citation_author=G. D. Van Der Spuy;citation_title=Linkage disequilibrium between minisatellite loci supports clonal evolution of Mycobacterium tuberculosis in a high tuberculosis incidence area.;citation_pages=529-538;citation_volume=47;citation_year=2003;citation_issue=2;citation_pmid=12519202;citation_doi=10.1046/j.1365-2958.2003.03315.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=N. R. Thomson;citation_author=M. T. Holden;citation_author=C. Carder;citation_author=N. Lennard;citation_author=S. J. Lockey;citation_title=Chlamydia trachomatis: genome sequence analysis of lymphogranuloma venereum isolates;citation_volume=18;citation_year=2008;citation_pmid=18032721;citation_doi=10.1101/gr.7020108" />
<meta name="citation_reference" content="citation_journal_title=Trends in microbiology;citation_journal_abbrev=Trends Microbiol;citation_author=M. Vos;citation_title=Why do bacteria engage in homologous recombination?;citation_pages=226-232;citation_volume=17;citation_year=2009;citation_issue=6;citation_pmid=19464181;citation_doi=10.1016/j.tim.2009.03.001" />
<meta name="citation_reference" content="citation_journal_abbrev=ISME J;citation_author=M. Vos;citation_author=X. Didelot;citation_title=A comparison of homologous recombination rates in bacteria and archaea.;citation_pages=199-208;citation_volume=3;citation_year=2009;citation_issue=2;citation_pmid=18830278;citation_doi=10.1038/ismej.2008.93" />
<meta name="citation_reference" content="citation_title=Population Genetics of Bacteria: Symposium 52 (Society for General Microbiology Symposia);citation_title=Genetic population structure and pathogenicity in enteric bacteria;citation_author=T. S. Whittam;citation_pages=217-245;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=FEBS Letters;citation_journal_abbrev=FEBS Letters;citation_author=A. Zomorodipour;citation_author=S. G. Andersson;citation_title=Obligate intracellular parasites: Rickettsia prowazekii and Chlamydia trachomatis.;citation_pages=11-15;citation_volume=452;citation_year=1999;citation_issue=1-2;citation_pmid=10376669;citation_doi=10.1016/S0014-5793(99)00563-3" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Impact of Loci Nature on Estimating Recombination and Mutation Rates in Chlamydia trachomatis" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7.cover-source.jpg" />
<meta name="twitter:description" content="The knowledge of the frequency and relative weight of mutation and recombination events in evolution is essential for understanding how microorganisms reach fitted phenotypes. Traditionally, these evolutionary parameters have been inferred by using data from multilocus sequence typing (MLST), which is known to have yielded conflicting results. In the near future, these estimations will certainly be performed by computational analyses of full-genome sequences. However, it is not known whether this approach will yield accurate results as bacterial genomes exhibit heterogeneous representation of loci categories, and it is not clear how loci nature impacts such estimations. Therefore, we assessed how mutation and recombination inferences are shaped by loci with different genetic features, using the bacterium Chlamydia trachomatis as the study model. We found that loci assigning a high number of alleles and positively selected genes yielded nonconvergent estimates and incongruent phylogenies and thus are more prone to confound algorithms. Unexpectedly, for the model under evaluation, housekeeping genes and noncoding regions shaped estimations in a similar manner, which points to a nonrandom role of the latter in C. trachomatis evolution. Although the present results relate to a specific bacterium, we speculate that microbe-specific genomic architectures (such as coding capacity, polymorphism dispersion, and fraction of positively selected loci) may differentially buffer the effect of the confounding factors when estimating recombination and mutation rates and, thus, influence the accuracy of using full-genome sequences for such purpose. This putative bias associated with in silico inferences should be taken into account when discussing the results obtained by the analyses of full-genome sequences, in which the “one size fits all” approach may not be applicable." />
<meta name="og-title" property="og:title" content="Impact of Loci Nature on Estimating Recombination and Mutation Rates in Chlamydia trachomatis" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/7/761" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The knowledge of the frequency and relative weight of mutation and recombination events in evolution is essential for understanding how microorganisms reach fitted phenotypes. Traditionally, these evolutionary parameters have been inferred by using data from multilocus sequence typing (MLST), which is known to have yielded conflicting results. In the near future, these estimations will certainly be performed by computational analyses of full-genome sequences. However, it is not known whether this approach will yield accurate results as bacterial genomes exhibit heterogeneous representation of loci categories, and it is not clear how loci nature impacts such estimations. Therefore, we assessed how mutation and recombination inferences are shaped by loci with different genetic features, using the bacterium Chlamydia trachomatis as the study model. We found that loci assigning a high number of alleles and positively selected genes yielded nonconvergent estimates and incongruent phylogenies and thus are more prone to confound algorithms. Unexpectedly, for the model under evaluation, housekeeping genes and noncoding regions shaped estimations in a similar manner, which points to a nonrandom role of the latter in C. trachomatis evolution. Although the present results relate to a specific bacterium, we speculate that microbe-specific genomic architectures (such as coding capacity, polymorphism dispersion, and fraction of positively selected loci) may differentially buffer the effect of the confounding factors when estimating recombination and mutation rates and, thus, influence the accuracy of using full-genome sequences for such purpose. This putative bias associated with in silico inferences should be taken into account when discussing the results obtained by the analyses of full-genome sequences, in which the “one size fits all” approach may not be applicable." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7.cover-source.jpg" />
    <title>Impact of Loci Nature on Estimating Recombination and Mutation Rates in Chlamydia trachomatis | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"8dkRHqk3BqMGOiB7_NcMtXnBGhvvXMP_MrdT0NX8qfY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459291","apath":"\/ggg\/2\/7\/761.atom","pisa":"ggg;2\/7\/761","ac":{"\/ggg\/2\/7\/761.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/7\/761.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/7\/761"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/7\/761"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/7\/761.abstract":true,"\/content\/2\/7\/761":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022nG9+wal0hyqs4\\\\\\\/bb71Oz4KNQ3XEXj7cDIzNXtio8hXl8iGvjPgMi34+t\\\\\\\/vbGbaNFXIAI5nLqazishlEccBSiJwst9UiWrxmoB52i7nhOOhY03tN778DXrB+CoDWHS33yEDDJmiTR0tBMCO+tgENbM3x7glcq9NSk9bSpsJjrJurUAzylUL9awBUknvkoSY8qKPRjybNYoV72RUARVsNjJpDH5DEXJS07JjesX3F+sbA4Yk+tx\\\\\\\/P\\\\\\\/Uj+HtGz3j31cY8vWHWiz1PZamnKIRRv0Kxe7UxZ5jKdOjtAgU604V0Du4oBeHHrcGEbNyu0U1FzU4N+0HeSDrxjDchrgEebx6dy9WY0ZUll2vCwepbtUQSXiANoLVEgi5xG73\\\\\\\/IobrsAgVinL1OVMpDsfppiRpV5xS2XLI66+QHU9YKLDHuEbNQ\\\\\\\/pYB1MJwcF64Gljjxpes+5gm\\\\\\\/mM0ORRNaP\\\\\\\/qCDylA52lcH2SNsc2x+yUedomVTxoLDd+CB9Jy8bgS3XVj1SIqyh21tLs+V66VM755g3Jqg2q1P006koNDkgpnHqx7BdzRXqKPQVIhM4YUCVTmPV2nQ5CuuVhCWmYZtZE3AjgSk7A2io\\\\\\\/0OTt8HKg69aYSZnxDSVtNWl0s6aYZWKgRF+zi5idkj8jQMIut\\\\\\\/lTZ81AoiUGmgpX6LxqmE3LpHTsN6B4nxZb2384x7b+zulCUug4ap+UiGsN9ixBsONir4maY2Bo0Ucroc1VR0Cy6SvguyMs+st4EKWX0gy0oPP5o2an0FEYPBX8OwdZR\\\\\\\/zvTOmmyGOhKsCUOXI1qN3K\\\\\\\/oj\\\\\\\/3VrckIMgkubROPIMyYzTOOE1atvdPxsKIN39hWCV5x48sZ5DQD9TX0lYHIm3yNg3aFNl++UOSzcxa6\\\\\\\/2cs9vfMIW6DF1f8O3W28OVqh+ANrifci\\\\\\\/2n7NfY4BU9EgJeDKlSpqi+\\\\\\\/NYRYPWEILmc3hbtJ\\\\\\\/PwfVRIiMSXhwZmvuyJY27A1pZhtk5ne9+tKB5hp1zHjSRuIyaUzJGw8uMvNzHTz3UAcg0Am3N51PH3t82XXjF+gTSF1YsAJzq3gdZq\\\\\\\/9V5veugp7HExC7y+phhQEDFbui6pWjeNb1cNR5XMmIrW93r6je1Ynmt\\\\\\\/HV0ylEXYccBwcmsDqKhWjLtMat\\\\\\\/EmWotlWPzVgxeeI\\\\\\\/MBkZVf+PsJaKJCsXgUakh8rOEOMLofkXYauna\\\\\\\/OztYOoIbGIKpoPSNMiq0o05qhX54HK7kCu9ajttQ4l12QAkZNJmn0VCCYn4\\\\\\\/BTfW4kz5SIFXYEcSFUvxY\\\u0022,\\\u0022iv\\\u0022:\\\u0022X5gpr3mz8gA5iwxMf4xQjg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220cb08f621d338aeb029a576df0037370be8759cc8e2106acb16df894fc72df22\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Ajflwn5igXnS0X9pLdYH\\\\\\\/ORBOpm0TQIVOpl2k8Vpy3LXOuxIp8NIM5k9SVHTbQUZCJ0PfF+D0XHgARDCrCyg1rSV39X57OEilla+W3pZ8rXz+NY7A+fFQpbrcdQM3+\\\\\\\/JV+WkSX4QetznxVBg4+9+emjNVVZpN9LFEhWVnRwbJm7+7vPmmj0i4E+jA2R8YyOqgu\\\\\\\/8GK3d9Wt9wmPBqA9dO4c3MoyhUR1mvOrZh1A3MKgYBAtsEPOd6gtToT1MuD5exY4CRsFKUO\\\\\\\/dXIaaPv9jieFhYeJdaXBWFto8ehU+OUq9sMCey2q1JW6wruBc4wc9MilqukXK43PDnFgWO2zPYV1l8hEBBooHBAnvHD3+x2H1PPcE4TWvwKO0lGlOOmqcLP2CiZKtWrjc\\\\\\\/SYzM\\\\\\\/6XEQDCR1cAStnFa9cfDmmgUtx4kgzaPqNnbBzE2xa9TaDtr3n3DR1gll1Gdq\\\\\\\/MEJDEm9rJQmO7WnKYXUJIq5GX8\\\\\\\/LmviHD\\\\\\\/Lw5ZbsH+Yyw0BrS9L1rc0z4Qn\\\\\\\/po4wlSQIR3NOuu7oXwS66700dVxdvZ3zHW8Aj93FxAE\\\\\\\/oK9EiGVsa72zFOGLX9boD6DxpGPP36PrgWrN1CRw63rnQsGgUTSNlo+3S+vqkiD2E0+WixOqiCu0wy5Jz6Bi2gXwrPy+ngioTpvH03DQV6Lc1iUwehmbGH0oH\\\\\\\/Zz6ViTle7g9LrvTFLVM\\\\\\\/JcgeBZq6+6dTqmAmhB44s8NI9D2x6cubCU\\\\\\\/L3etb9BmEsQlYOpEEzmdJm17SMirXToc1DcRH4X56jLvz4N7WJUTkP2tZ+TlGeo5oiPdfnZ8GW5dInMgtsnqW8LEiGLN3JAjiGCeobZxPqxxPIMYB3AFpbaLLyCm4VdOXJLqQbir9EgiIuUPf\\\\\\\/zRd+tVQeZeFuW+u7zzP9Klp3RrleEnOGsCzI6+p4+m8c77RDnlz0+7hcY9qzItbxP3z39WdT05wXgVfWRWY9FfkYvllmuq9byxDHxwk\\\\\\\/MZTjjL4OWurDkBjbRVezCLkPUK2cxyNqpDDtEgksCGIqdpoOZTna6dExuqbFRTGMLm9sT3pMoJuAzxrwQ7QWzEbgpGpKGxcfwSQSX+gaH2x89RCD2X\\\\\\\/lEG8M4EZs4J9Z9MHRuR0\\\\\\\/3Uosq5Luc28eLWY3St6xFDA8GSQrEvNZxX8paqccMd3WzayDVoo+GWBVZfX+si+3M7DEWeZHrOOCXVga\\\\\\\/nRyPTTUvp7d8lnf21Qvb8oTVipXzIusHh9oXfE9tvsrM5puFSTameo6aUKc5+IiAaHxapSW2loY9FEfJ3odcW+Yo8zGBANpHH+P\\\\\\\/UxbVZSKtX3iOcv57itv4n0ZoeGNLgjCOLKlm6biqxBcEBDwBkDY3HMacHpI2YgJFJgSvPhtX9\\\\\\\/jQ8y7LYhdjotBeig9H7HnByU6VAdz+cCzVO\\\\\\\/1M9i8nB3OpSLpekYvOF2OmDNEkzXjp9R9f+62699prCyxYE4e54X8J7xuQvAwhqaYymPVvvmyWV0nCMijL0H0wS7MYWm5dDcriLpSlIEPmHwpO74Y0jGCccfhhBcPkRJeljssRatoltBJIldLvG9O8MT54+mRf2DpPuCVYlfj+m7Xlm\\\\\\\/rp0KeQK1BrUsq116\\\\\\\/ybvfH4YKZ4v+Y\\\\\\\/4deBY4p59dQAjARnqALsh+PDlIeFLcW8gIrq1iyS7nGdOILAQmQOPQZZ9fj6SWzbfVc5ZJLmrNw\\\\\\\/kS\\\\\\\/nk2LwMc6sr29rq+sMqgN+5QUR5a4zM4sBFSyvEXCNuSGaFwZhiaj3WmdTm815Loq8vpRAjdvSYUc3JA5taQsA54lzvuBZlOH4+bl1\\\\\\\/ipRLq8i1NkVZ1Om09ruHwE1HiNiPbybf+3fpKRANN3Jjrhdbi3aUok1vNkIIgnxZ4CiBKbwSt3DFMaJKnIU6X6TNrP\\\\\\\/SMcm2kMcZmz+IsKbZcIy6DG2QWxx5PkpGQwjmTIW2C7gEleyCM\\\\\\\/40r0glqKPxSrWCTeFnqYOU34RCsduhibv6IJ1RRLVHU2wkjVjlMXIxnO11x+z5hPRIjSjNtLr3Ll4mzm50GWKQGEEk+WxPFDtBjav3Dua5yK75L9JVebnzWhZ+Sqgv\\\\\\\/RligSIYgAIfGdLT\\\\\\\/C50w8qnKvCyUjMYYGXo0lyMLTxL7+fxDn1BGbRCR9P5ZWqWdXijW2SKmUbIODmsgOY+hWtp+yXmG40TkaNQFs9IGTJ00ECfXIUkCRFinQRCTbzTW6aU2P12E3mIMXJrwuKQiY2u1S5pqu7NNpAVV6RYv6yqX5RWIzMBlzS2fACczDnFjRLe+nCw+zGwZFTbl5X9rJERxkHfsF0C2o5dgZiYU2wyFwrjsk+ZriDq8N\\\\\\\/mpthV56N8MOqUBmeF7B972ZJOjUm9riKAbvZfVGNOHMNu8DCryGT8UgQREvOQanpb9j0Vk+4bifl28h+35XZ+chbjU68Gx87ixt\\\\\\\/N1BLcuJ3\\\\\\\/PtEMuEamlEv8RA8s3mLKIbHEKsNJyfTNuOUYgO7904kjhjFV5VexuhA9bqW4NQw8GeqRrOQAREX0m3d54I+Jug3HPd4UOL2ENkgUpLH6WnInlle2cexgUVGmlCpbqxDH0JTFkfh5Zw1xxZ+fC8ZUB9cVfS1ShPt\\\\\\\/TAQLPQjrglPm0YPwg9HciLa4F5dqXnn459jbyPICeSTtLbffP3X6JHY08\\\\\\\/4PpUt9nQolRaqitTRjpqRYX9hnAQntvP\\\\\\\/OIztDiysU3cDMvrkHibaquhVXOxkHlUQ8KhbQJ8ljd1YDBf0uIgvZoKwn\\\\\\\/VLQu33HCJdjB6zd7\\\\\\\/YKviPUl9jp8GPDSarz\\\\\\\/Jq5hnqY63F03dr2qAJiBj3rW2RYvxYG36LyeBzUmeUGQVvoxJqsfIQ9elXqHz7XPPmJWAdQSH4BgyJ1zk2Bjz1yqPi6vowC\\\\\\\/rv3rz\\\\\\\/adniHLzEF0oTIZASPZGX522bxTiajB1CWb4lVyo+tZzUErlDk4cThNJ+W12j4dQ5E5zRGK16NcIwpMPC6GupVE6e4kfg6bZz6umUnYfGhkP3Obvf5NBGmUZPT2StyY\\\\\\\/E57rvzLhD5eZmX716MIeyTTDgOvShUtPeZpJUnzivd9FpqyUrGox106CIal8dofxYheys9mPh4cjDjauB316p4Y5CNqP7hVLOVwMgk93cGJcZBrZetRhCHPRebgnn3XFYs1SuevNRuG1nVqbyqnXvKF9t22tigCNkDP6hvEvTvG7sEFzOLusL\\\\\\\/ZVPrfvOXtTG6JgdpOQqnfZ9AccMYTmTNTKTGfIdLjPEmEfxDHM71fRL1Fw53uEAjLIZKxAD0JAjWocYS6EL+Jm65Fq8et5qttSco4Y19dUCdppgftz5RiuI7m5Sg+wLF9RvH48fwx8DcXqSIbFKtlenIXeFw6MrT+Ax\\\\\\\/Qg7EW7MQWAlxecx+I7H2pKFmQmDkCZ0wsFNZOi9rOMHLc0Fz5vKsbYkK4GhFeMrPgu4twEct4UD+m92xSJNT7tN53zWf1EkdU3B31ho9MUTc1c+jQoEedbHpUgS3bUIyJ4aZAN2+9AD\\\\\\\/TWlaudIknavEuxH2IqvOt7GLOwJLo376QWJeYXR+fKYmfIGGms+GVj5s3spgcM1eCeg5dNK9kY+8OvJdMNHFgJOR3i6XaVWJTFZvNWHGnt9uo0aXTXc9rqoTjnwgulCoF5ztfNKN5ti1KbjeLrbWs3yBICp7gz3SueJAWTC\\\\\\\/kkti6QU78d+iIFg4b9SWiY7ONtSzWXKOA\\\\\\\/Wi4zLdyd9kiDxZY60WIXLGQC3ykBwqkhF8WzdPfLwp0oHD8dvU1YYd1EC53YTHyIBgJrUHPkl6jyXEB9mBTTqAHAEM6lUY0Z8EwiHcbzwUZi4TxdNZsGVM1PwSoPjgll7nP4QDSuCfhXMmGHHcW3lUcdRxErPNi5eAV5U2j+kYqgH3J3PY5wWlqCFz1ktSoL1v08YA9cBw0cA2LqhdMx\\\\\\\/5VTlxdBTjruWTyWfMh0VZoeWNJjfNb2QUfBvBElPstzzPkNb80mVNjaponHKGWurOm0QNbaL+\\\\\\\/kV1ltEsU8Q5zuGt3UoEfgmhI+giMimjxm1yCrMHzJxorHEHTBSsJj0N33aleowy1rBb9g\\\\\\\/Trr+I9niwd5pTrif0NF\\\\\\\/8EAw6IIYaFno1jELdzd4rJiGc72P+EoyOcJabQoIZVT8xIz31QPAJQkiZKbwpNQoyKdx2hecelTt+YUlFtji1plagZcML7PW\\\\\\\/Jo1hX\\\\\\\/xIE2spoB2ipvIN+DAL7tyQDTNklBidpBJKrLOQRbqxxh\\\\\\\/X6OHedOvAyNidYgY9cHzzoCbOXhbug8tkygLobhFqkZ2DNU9q8\\\\\\\/tLSQMfBzdaRCiZ2ZOInYLOqm+Do\\\\\\\/7XuznJBbS+mv8M+115HOif8l73PMvzy0d3iRUrc9NH2y+XdAlHbjxmnyxETIM93dEhCX6Psu6KCumGWBai01ew7OfE4rIVf8lSONPwSPXRz\\\\\\\/M8BMyvLQwdYeW0h7vO96TIh7AcKpv+3dOv3DpQotn2zbl6VIB0envZ4gLlLMGXZcDWT80iqMWAXpD+uwW24VeEMm4sTTC71+nWcfN8TJcyrFtEZQKLeIf8V0i7+490\\\\\\\/8JeLT00KosJyL4m+FLh\\\\\\\/rqC4QGPt6409mgMsX8gzX1C6COOhWaIfKLAh87YO5WxnhfgeQucIja8ta5lnGaYBBbnJVuzWoMtAiMTS8Pp1wrxuMEpPkRzKA\\\\\\\/uEjm75LJ8ocmWjARVDNHm+m9mkoVh2i2h4On1EuxnJ6GgwkYJk3lURb3pYdLiKEClzmKxctzx6MTL4Gx+ImPyOGWvgeOxnmXHQvUbh7qJ\\\\\\\/oSp6oUc2nukNdyGAgfLbKhWaEOKhrbqko8KYMyn4YSxUj9bOKfWJZbwHQwWRbVAeuehBeWjY9xRn6Cgm5N1WS1EVrRb1VLeyxbSXKqZdnhaFga+1tYGyOFo2NXMJlbhdyGYW5QQMvjf1ceCAvlmIKUGVlGIeGAA6v8hATxbLWNR25HqA74Kt3T\\\\\\\/vFg\\\\\\\/S2nd2MLHLVfv\\\\\\\/TA\\\\\\\/p9EhHoNbe4UdyIx\\\\\\\/f5iwMQVBaK\\\\\\\/OfPZFatBc8ao1AgGIX5n5r71GrDE7iY59MKTjLLduIwyielM\\\\\\\/IBYGlXJY4gAV5irPtINCnou2VuwSl5FcWpipdFybIEWZcde1LmuAIOPEKPlXsl7dhPmfEwmL0Nks7IDsB4IclIkcNDIUxU\\\\\\\/\\\\\\\/\\\\\\\/FI3DDwAGTAquM3Mzw\\\\\\\/k+7mwy0hGRqEX9a\\\\\\\/HJ8OsSscaW233KZ85RMPB6GH7bWPE3USl5mtpkjkgFLs2v6qQkBHbQ4mfD8xIlHq41940N0Q+Ors0OfcQEESAr9nACaxj\\\\\\\/lGFy62f1Oz9RdKOkk5Ngr8KPx4DYlZHzONsOANFYJl4qyWvbhxx916UlSvXbZfwS0DidzBh8QNeti8KsG8Xd5nsp9kqnnG4blRSDYB1WcJL5YTBCfJfujh5lxi+aTdXHf\\\\\\\/QKTGdx7OrlEk+TMciuekHTK04R0gKWnU7Emn1vSoT874knxwa59guX0OTEbDcz6yLJXiVQ6QVVgXJ0ZdUWfU70S9MYqIocOu7GeNVuoPBGUJz94EDL7rgqmjJnR2Ag25zrRw9qpjDIcGauP82FrbeBigSpTs46n+UH1UWSDwt6izXx3Fx5XfxKFEEhRBTAnIf7nTXVcHHaY39WmpStvw5UaccG\\\\\\\/SAdp9Dbih3fBkciTHaOsMVZ5VBz3awIcni7+SbbvtPtijvCpD8IY81DAt0u87DSOGWHUClYmQL\\\\\\\/YPGnE8W9Lo9+NW6cBCbZaymzJ5TMbZuhZguFORE7ODzcTBr1NNf+0FHhkmGViKj28BMFrEYnmiqbAzBWgqcRPHeYuThH24BdVj0UKi+575C55cflHA7v4ZeXpP+AZtoBp50FavHw42ZnbdpBA8zZ9xQrLNq\\\\\\\/YT0iHK0q\\\\\\\/QMEKa2GWQ0KWhFtvgsNpDuz1szfOXc9nH55ursv7FStM2cwjGetqsAAzRyAfE3G\\\\\\\/VVr8ObWabVu5oFrM7amJb8X17WZP8jVpkSoXFH4XFOBONT+M4jRBug1PJEXhwVemGoI9Ki2HHQroE8ugGeODFZWFdvBwAV2NdzMlvnEji+Hn1jgemtPPyVzb83vYUKwIkBO\\\\\\\/6PkwPlbs\\\\\\\/7D8\\\\\\\/+TgGWi516FwUK56Z9kcauXGh0DKs+rZKUuppV37kML2zTqGBJ\\\\\\\/K4R36YbvRo+ghaUCyjlAHlFKdEFARdATQUq78hnTFHDPI4szt3eJ8zmRT1TopL17OEFW8cL8rodfL1oGXOBwreJFGXdIlLWwGkNj9nh6vEL4b+\\\\\\\/ttj1f7lCcCL7hfdVliHAIH0fpM4RpFD9UZuZ3LYFkcrpcbiakfG3aduVenITqXO\\\\\\\/OE8vkZN8qRQTXb4HfAzGrO9KtBFtMq8pG7exI1Fod9kYgDc1k2\\\\\\\/rumm7rlmvyDFZj+rDJQ+RrW9KZlr9msIGX7ZeWpIcfUJT3A4dI1hxqrJ0458c+qxX2y3U25CEoW4\\\\\\\/XjeFktvHfIB+tcyKvFxjOgg9LWK4ksZh20aZj3IJZJXmW\\\\\\\/dEfWFO\\\\\\\/NEtTbI\\\\\\\/s1KgOBPfnMltPHfcqFoSqxZMBPTemRklhNbu9OG9Ub5v0mB+YO9d+OFBLRx0hRFnP2b7xaK9SOYq4hZjYOYvPHyKWlr1CFwbXcnj+KmVIyMNwO+BtVQasitz6d2VY0p+kulDu7keatM+CDtHGz+H+imPjrQSBbcfjjEpAkbsms5wIRqXxeKHDYhNfTWMKyqsFMxpTShdJOoymN\\\\\\\/A83Db3A7JJTD\\\\\\\/DpLzbXfnKHLLqf3NBsZ+NS7CzLF9iAPbIb7oXviJyDJPCif6mp0ZEKy+Rk78lPEZILT+Z0hA9sBGOiI+IBvfJ\\\\\\\/Y\\\\\\\/LUAl8F6OufaNC7unuLIs3zcj6WBByzfBgaP21BarqbsTH3SMTnt+Hr6uTvQjgF3p\\\\\\\/HrwZY\\\\\\\/ebrAs72LQRV1Cim7Bq+QplnCol6LumgGS39yI031kU3kTfSQcipanJaTgycreOq1DJ\\\\\\\/jg7606ZkbpZmklO0V0pPUS8WDShI93qDnynoG3JSWyiPEssPOmZYvtWSwsfHoIzwjtl\\\\\\\/b9anxgCPI\\\\\\\/jOrybrIssolKUNscKkJnIx1obwnWGYMxDM94OhEM6ujDK\\\\\\\/hTNqAnxouvcY+vpBX1H8kHd9E+n4KtnedhmcwV+4K1jOjr\\\\\\\/DiITF7vexfMSYCmCHi\\\\\\\/pDCKq5F62ZG5yzeHpEmErxbbt3rgOkGo4s88ZGXqQCryvcg2p\\\\\\\/O8eqUbRpQvKOJ\\\\\\\/CeviKDPkR2LBpltSibmce4BtEjIMiLPmbku9Q2Wc1ew2kyt4t4S6QBlt06g0JDdgf+1GLTHofgnxnKfEVPsD5IbdL0dyw6rupzFBrI7tPwOMCu75ZujYJdQNSfO6GjVsc37Lgu9mwcbXrNzRzxcow0GM5r+20AY1YJd\\\\\\\/WtkJs3VGyDTMjFvW4MT6GoThAB9hNvqKKdowZ4rkiwjEBZcychmks+saWOUpp0aBNHH3P+T1sWfsrvLDgwyAqp\\\\\\\/JH3cQCWd8O8iwAHEO3qnqC+\\\\\\\/qDIEVRbCx5Zm0PuvnKxDqmnFkxori9Ts6G79keX48fBux6EnRlFA0o3p6edTlDweD3CtXZ+K0RJlyqGyqf0RnT\\\\\\\/kBF1uXKKcuy3gGYMD2S6Uk\\\\\\\/77XaT2cdWgyt31RjbEyFV2nRLfZWqg96CuoUrc86wlUaQcPSJPqi8Ys6UOOP7ZHhrFaEjFlDL9N7wJqZPn9AzPWEioUxgrH\\\\\\\/yWOIIeWU5Xdqc65gXIDuHoqYYGUz+1RE\\\\\\\/mqA\\\\\\\/iuouiYDxp8RaI\\\\\\\/BRbd1yBNzFoXW+u2luzgm11yFWd9j70CPkk7SrLIXZ9zcKlMR9qPKsOC7x7pbeLi9mcQlrThPQowf3VPvNCPSrHjX+ULeXmaqXBBxL2XDPm0RejfhDtF0JMg0AaRsDRV2SBySIaGdggDAXD4kBvUDEusvfUN7Vy0SOGfmUjICKgRY0of\\\\\\\/RAuN8OMBk23kC25zsjEp47OLQE5nLSt6pbbRd26d2lcCdvZMuMBlRPOA743lN46ETFIVfoWpmSHijRWtwLNBBG0YxX08a0cAzqpJYE2tUisWCFzLaMYcRGJX6zfpkggYCs9r5AKH4QLXbvmME4dwt\\\\\\\/AtAw1kGLrx\\\\\\\/8JZXNu2K3RRr9dIvc56SNcd9meFz6CQPoH9JHraLZ+5n8ODhCzKgTAwauMJ5pD6rkWc3VBmpNja0PFAXErw0ufHAuNxIN0lK\\\\\\\/ej\\\\\\\/9fCk03Qa72WNOdoV3\\\\\\\/T7MdvFiOtLMfLTv9xz1u4IOdseLGql2HrGz9JvxPk+EAB49bU0RPX8QA1JiukApm7XwZbhjZigUN7utoN3uctP1bpWEa5ozZsw2WzDT7j\\\\\\\/0pim8wJ93l+jIl\\\\\\\/ErFs6r6EX85MvdfRWNJ++KrwQm3HhBqf3w0D40HsCZKK9iNj\\\\\\\/jQvngU+TGYyQaEbyYSpgaKtWP\\\\\\\/rc8jiix\\\\\\\/mEnqpx17T+hDvnju3IT9e28bylWSplPZT1e\\\\\\\/LoPkcbnCi0muh1Py3foIMaHmmCynXJiZcfBwtE22QQm3pAr68NlyA\\\\\\\/dhP8ljb579Sbr8JHAltancO5xVzOZBOq1PVs9TDX26BVdhZWgG95KIu2caUbNbt8jd3LRUWTMat4odLlx9YknapFVvR9I6a3ySpHka2nNXLE3BueDudT91rR6CEJajtYjaugnG8xlOCwB1vd1LvX3BIDYdAWIihk09Bvd7dy93YBtnH70uJwb7kgiMkheDsqFMj4u\\\\\\\/Blc\\\\\\\/196we2nVazh5U21Dl2gp5vbAh+I+j\\\\\\\/S9Fr7tyQdEbq2zDgGqzm7Xl2hhh4Bynn3C5kzYt50eb7+cEeAaEY9etigHh24OZOFUMdZi59ujXqJ+XNF9uU5qwCUAVaSOOK6lL9MvTZHko0aWpGWw8rWfAw+iFTATV\\\\\\\/3SiLw47i35HcoovmZMwIi7cDPI9axgmI7oMCQkkS6nZ7DPiierqIqkE6GBcBjm6oQfzCwkqFlhDKHe0YeuNK8KD+XLiLZVPXg6ZNCGioQ4omb\\\\\\\/EwwzCFFszZ4Idub0K2c7MELV56pWbda6FecpeUvcJ1bzKawxdKD+JPktKy7LSMyh2e1ytiuhYB7xcOVNOiKyV7yueiILBHu\\\\\\\/4BsN\\\\\\\/ZE34oG3Bnv+RfLMP0mG4eRxA5V\\\\\\\/1czFEtTYNLvy32ZVZLUtQSxLE60ohjHyoP8jNy5OJDms7N0tAGjczB9vlWbJmKJpzwUbl7U5kNt9bhzdzvMPJfYhnRFjFmgentUo\\\\\\\/LONcOVlfW0amrabUTpitQdJw8duiIMP6g6av6WyS7fO9UNJEvN+MNrr+5HK0RhVR4gJr3gglYJzJ27R76vCYU6dCffAgfaPlMe1kMGK2RbgTasFiTXWJXDX1oKDBQOKQoR2bp66b+uS1K9fGx0\\\\\\\/HaDpdHHdqSR6LYNunP1LNhkay2Zz5Zlg2\\\\\\\/oQlFsrX2OSXq6aUS3kBXyPatmzjhULpNluGVaE1AV\\\\\\\/nesc4Za0BFo8au4QKGononWoraHZp8ETJO7VXe4slxaTy2V0Q3Ok+DfP\\\\\\\/nfh6Uyv0bN4gvcl2+jgtVLsQ9z6MHz7kkESmLzrDKNHCGfVBoFmDSz0WCWMcJBW3L4679+BfR7\\\\\\\/dDbworlQI8oNLrsQolhLQ5PLS8W1pyf7rIkiF0oPgHmIwdkTaIf30M9TmDaZ5KbQYcfq8pKaLe7O6OjYYmhKwdpMqMndDnjSwHWNP3SsERujeesIm9i\\\\\\\/c6nWC0Qpuld7dj0d3D9SqwQ+Lk7Jo5O7Rt1V4229HHxxm\\\\\\\/RQsa4Fo2Gd54P2QCV8D3vI2huTIHpA09gNQtOsUtR2fv\\\\\\\/O1TYAw9Dn5GR5YYnJ\\\\\\\/J2j9MT61aris8qr1BzvM5SmPekxi4OaoBJAMueAc6ib9SNjuoF5\\\\\\\/rLPvCRjFsR52uc50WvKovL3CWr2JKHKh+c+vE27fKoqd0i3niOdLQ\\\\\\\/nNb39W9jvc52ZsrPSfudhCXByjckQj8RzHhTvQNl+rXKuw3\\\\\\\/FhVxaF8mdGwgObSefTCNKNyOTvqbJ7s7W9jZY+5dH3Pp3FXRuxzgemFnB+P5TJ+yy4SOPktpJtar5OzlGoEgEKVvN0ZypbdLajHnrs+twQ8oQrXEXcN0dTiWUn\\\\\\\/0gh3GC3SxHL1IzdL2aKaxznBL2lyWYjmLnf25rcwDPB6qNPGYdbvkDL7\\\\\\\/LxBt0M1YcFJ8u12+5mKFpthUD9xyMU2PjBTD49XNbMbnwek5\\\\\\\/yoZc18\\\\\\\/hJsPuXLQQM1vCy2truypeQOdYwlYf2\\\\\\\/pNWRVXt2tRg9YZ8L0ZrClbbwjW9qeCO\\\\\\\/zFIuVJvMIp3riyoU49m3K1lkIkU3c1zTivinj4d0pAIa1lbBXgVjRLpf8O7PPmFYbXWoeYWYvbZZd3wdLTH7ywmCTWDle\\\\\\\/76HBRiNxlWosZhoQbBw0cd\\\\\\\/qV+aAdi42rHPgF+ZYmiV1VuxIy1GSFZz0oes9pL27pNpSrm63Fob5LrPvjp527n197pqfjfb\\\\\\\/krhrfLKLo\\\\\\\/WcIs6aeo0FANpxzR+oOtbeYuSKmmoYYQKLfIsWMmQ58LGdCn6HbZF6+FHM8xhGGepvRtvfQ4ypQB7QuAo0rsuB\\\\\\\/wUGEZGz3wTmGhjOm+OhQyz2uhaieeuyQiebj31oLcvftsF+ZPW\\\\\\\/mlnUqO0pUrNvO98oHmcTN2P5\\\\\\\/9BFzOxuyQjddcq4\\\\\\\/H36bkGPhhXiDaMUZiPYklqcqmd+ZX+aUlth4yxNf5aZ7qOyHLTEDqAEO2SPjc+XPmN49Blp5FeE\\\\\\\/rw4UPIcMWGH2T704e7ZRxuk4CgWGeXMM1C\\\\\\\/WWDTnaBsmVycmCnGZt3BLEitudgQRtpTa1m7DYCY6+k5J2Viu1wnWpWOzbmPRDLJK+IxNCMSny2\\\\\\\/NjIc6rNPo1O986LmM7ao7iiILWUWWT1CuJ6u5AiABuExq8jKwTu+qJOVaD36MNwO7AjsGkYQYTFhTUHyVQyqGvSARsxlRoGc9jB1o+N8XePiJ7ntKNH3J7UHCVNIL2w24wyK5IdLahZx+8NtILFKDvk109\\\\\\\/dQ4s6BjaT0ozOCX\\\\\\\/wsr3X3mwPHshXvtweErz4oDPpDewOadbhLRQmY70o56Ea1YdJxmlY5y6CWLPuaZ6y7Ih\\\\\\\/6x0YVuC\\\\\\\/j8OTFwHh1UE6pjAduiWugddi+thbPHhRD6jTFJWBHF+pzB8rUniGoxzcx06huf1U8S1noagY6dvfud2KKef4aLSRpBkmu7l0KBA\\\\\\\/60rrJUuMWI1HUTIc9fHhgZHM+KkPufrwohHx4ogHM4Se6w\\\\\\\/NzZPLBsCMW8hqH5jkVk4SuM5d\\\\\\\/REZlV6okekRkzWbruRgpMCOCpduawDMqvWHI+aGofKcJg9x1kxc1pFym6gYv53RzLlB03R+iBzh6QKLwmMAsw8ojnYCsSvutdN8NNshGXu97GuzSJ7WmeO6Z7zN\\\\\\\/ZCaGLz25+9qERAtTZdudj3A6re5gjFsEAzhu6w7bDvlxWChdtPbM6XZgaDy40yJ9vTiuQSnehVH7Vh+14kVB920pROIwBHALRaa2GFj9lyTKBmiTHj6r1yiNadBPgeZMbQDeIeiv86hzVj\\\\\\\/MrIWq4Ftlg0gVsMWwD8NwtcH8vanfyq4MLy8M28XRwCQupUvHW2Tzlg4ZxSYXdX8hfTxJMNAQVAa4Myn2hSpsCgq+yX4jy7sZfdNQFBDwQzoOoOPJHLN3vwBtl1GkuoJFHuYfqd7+AbapJfkvi8oTd+H7\\\\\\\/EFIHcXj1mybfpdx8stOpZLQQE2YKkmbgJbJE\\\\\\\/mhtsOHnt9FgGYL++rV+IJg4eiMPx1itY4kAuLjmHZsZ26ADfaJ6sfkC4\\\\\\\/tDFatL2Irfw7IOKsg4n+yKky9nDJmbuLkq8wU94eDAv5rVFyTM4ZYzOmg2430fci43b9msgDzlm\\\\\\\/Q0Gbp4nWW1o4SZ6vwiPLXQGHUKmUiFBbCl7+r5f9LFyICopdsoGhTBqhLfMVJe8GejKUHEMiZNdt2Zs8AqiEMWwMznEBygOY4Xe5N+gt2o1ox\\\\\\\/vck9Ob15QZTeyP2SR3CBgN+QvCX8NBuCXGGwHCABhgiFqNNDat7Q\\\\\\\/TR9gveAkRepQBIcU0Rl4BRyMbuI11MregoVdpT2zsCfQZjgMRf5A9OpCJIm5HTaNQ0LDOa9o3XfKqcQShK2AoWqlQ\\\\\\\/+Jm4LebbmL8bFpILDc6loVqmXj+dDU\\\\\\\/dWID0aSLhhiVCsy77N90mkezKRpJg+DFvODeGQdIIigASRM9rdSTmnRcymiY5FdM7rp\\\\\\\/SUV4rXkVQY75I67sPlikTOc+uJU0rGZAGy7wo8r+knwyGs+6okheeMv6\\\\\\\/8+tv+uPKNKL6BdRGyU9W4ZGOW04rUxUEjv1lVrGqO3wNCqGz732QWKK9u+Su96H05LWtwoO4zrLlRf7JeKcIAyOvofZvMBn2j7T3aJyj6irdl3KigdFh1YxGPCXwjlZZQ+B7sW\\\\\\\/hX2OhUJ\\\\\\\/55RpTgqWxUVj5BrhzWrZolzzxZgPlpk9wh0lMm0isll8d7YkNfZbAj5JEk9OnsESiKqHdASfFSXeju8NooXHthVEZIcUs1KqWNsnvQJC5aG3fL7PpXhgi4QU6bHXaKLDrX9O5hjf7PQKu7nzAEU7w5AUEbkVF5YLq3x2OH8pzpoGwJXfBEJs6cwpZRIw96tk3Hh6m\\\\\\\/E1ucFORD9ocPf54ULrXgqoNTBxCp7+jibSzLKA\\\\\\\/B3o8OGiFbpMviWJuEeuywFZqDnFopb7iO9ix33nJF2RXlz5uA7oXZShh2MciWuX9fbqa7fDjPC3UUCfY9IioB9q5dbtTIPV0SMwdvRRCfM7zSpeDY9EmLK8Jqfw545HYgEpmn\\\\\\\/ADrYcCv6DMtfrGn2zMArfbxrIHwYI1c4eO8tanpcT1BlUdqhHgZ7XIaaNHmQGHA9roLg6jKonrtJDlF75DXvgj0mRGxSSncFauV91hAVbzrKPDd1d0bswNE7yjcIskB9r0sa\\\\\\\/AB6L2pk6sbYakursZh7EkC3KjUmwWmQLWJMflBxlYY+HBuXJ7Dfe8jo6l8iE1ChUB9MCvjEnkI4Mj\\\\\\\/yZCku8kbagG8+n+MKmSdV8q+RrKNxHx1wLlI2WWbQ7JahtijviJ\\\\\\\/VqwuDjiqYI2hL3OAWnFehd70Df2sba3g7Tb\\\\\\\/nxR6hhd\\\\\\\/HK1EilrkzX6\\\\\\\/20sDLo98k7phNk\\\\\\\/gCd7FAOzxrDFDb\\\\\\\/R+Z1ZaKBvjYuPz+WtdzYvUgU6PoXCk9mMomNYo2fSXxo6IgcD0DFw\\\\\\\/9fR5kyxsXDpBviQudlzDLltNutZh\\\\\\\/Yky4q\\\\\\\/LLFajFFy973Ppit2O41VF50mdngMSfjLWGj+o0s4GNJmKCPatqa40RvnY6ZeoI5rava3\\\\\\\/SNI85Pug+O9vuJhpzpwuDGMzQ6gM\\\\\\\/hM9kshmrO6BL1bztTOcDwDqJgXtD25WSi6aJ\\\\\\\/VdedtiqpBHkSLe+QzLHcuuLMWA9h9jVJxZkKBlGXFz4jguMxH2zzh7Ea+dvbYt\\\\\\\/ScQB1eR3WUq4+tJ\\\\\\\/GXmFXoQ6lpwEZHVCyLcKu1avT203vj6g+N243vPGGzWhr6BPPNHW1A+UiU5zqFCUJM+A\\\\\\\/WKiWbzS7vka\\\\\\\/G9ECmmQ+YUEyGvh4KDXLesIj8r0JnXpQ\\\\\\\/EK5OX12hJCqWZJFO2IRZfzm5URtYwXNf+zR5VldjBLi6HC+NmNksLOxg+Hs1ZQa8cPL+JaYCKtDsQRc\\\\\\\/dHTuqmtMeOLW7PgaKhv+2cedDAHZQfoH3GvDj9E5kanWmovvlnWZNX1oHaSi1t7fxwf1irVlac9QLdXcDoO4y5F+aFvSDWWs\\\\\\\/e7oq736Ubg8dp0PnZ\\\\\\\/OMN3lNxKwddVriQWzlNZXUYTrHmnhJRtJezXA6wgizCqOMlPtNY\\\\\\\/JFR2qX\\\\\\\/7nIAiYAsy+ymHPCus99u+cwtJKeuF5DpQjm0CktIWe7OEjvEikPngLMyIcA1kxsUcX6tJsGn1+kBFqzpVPsZchQDdwtK8tnvi3E8rBv90sPhHXMmk+0AVkMDQgkYH7\\\\\\\/RRgiTlE1aXNx4snVsg+DYrSizIv3xaz+TsREGT4k3CPLjESTB5gweXMRtjAOMFuC0I5BswsUbsm1O6mNCnCc7e6VNoY1WQARF1zyQnTsMA6Cx6CRl1MdJwOjAKL9cM0BfrNzn\\\\\\\/HLMYTLXZK4aQVKdjHxZQj4GjVXWIEzzyAm2JFalIRApnk9e2izasKc5saqBDJnPX8Vqb1zafPh4WozyzL2IwhCilssOaSyxNPJAXh8bFgmxL\\\\\\\/t32YN6H5wSSNHy1cxiV+XzB7c+\\\\\\\/AbDM6IZunRXZqMz5peNG0gqt2WyiqXT1GJOGTrG31GU1ELfAqD2f2wEzO9YQ4mKxyXHKiGw5dptCM4qru8e7N0elR+BDfrrrRxLDFsc\\\\\\\/WRPV7VJBjgUZNn27xOTIbOWGAf8ZiK6UM7p8VftgGwme4ebMOxqDKN1z1t\\\\\\\/0ISAz5FvW9\\\\\\\/jDWt3bhgwwaDWoeyn40gXt9NmcfvmAQrDGWcWajYGxNpqrKwkHoAjY9eZ0MqpAQBbi5iANGhzmOblVGUB2VwpKkEkS3fERzcs9Pw5bVk2JJqWmIq0hKRAoU+FcaTSfkNfBsv0K02DM4TVX6Skcdb424N3PqAT5E7ZN4NRJuVJN1\\\\\\\/9N3Zl0x21eepQD6JbDcaMK50A\\\\\\\/2FhO\\\\\\\/o6+bcVF3vGS7ge0oic7u\\\\\\\/MrTQ2LN5zCtOZ5mWT3+uY4ompswSl5ft4ab9IYW9PGvfK3sMDvyCnodAF56wr2gAh\\\\\\\/XWeROJ2TeprOeDfjlhrzWrwddj+fJvvCg18A5Vxs15CtzmqSa311FXw++uLuShZADOBTvNuBBrwtFGC\\\\\\\/Scfh9QIXfvRCIQ01Yj1zn1mu0azVvKYaNC4bA\\\\\\\/Ky7kF0QY4\\\\\\\/+lUAO1MA7B\\\\\\\/NoqoK4hu6fNBQUv2hO+cwmb5ZXXuoZYWJjsJ9Wk26oTpysXLI26aYVTCeI8I1wKdAKJjyZblQEhwdOmCyri4JoDhTBGoJRKtMqGV7Ew7EZf6ekYla8N8RcSfHR9PebkMDvrxpAFECXkFdXrkWONM52qv0qpotQH9og5aeDsx281TKE02TpyZkm4nx9K\\\\\\\/kd7n4bjvrTF\\\\\\\/e5u9de2rdnP0fHCy4yVbTSB6J1fGRlMOjiQy1xynngM38d1+0pBX14yvAXhTp4aBrL\\\\\\\/w3yZpWowVOu22KO7hfLNb51\\\\\\\/0zU+QjlFSgQUGdE3jGFMc5dzupJYA+UZr7PktSi9FR0Osc7hJFUJov8YKLAdWM6G+Z99OEKo3DU+YqJXthWSeKJWKXAYOOe8qZta6\\\\\\\/pk68mG9YkXkZPDdsAz9QFAkiG+kDNs2PkUuofeopEarH7TViozro0KXpA2wDormuW1fnZrSPXHPplfT8kj8gVI1tbEdg68ZwceKS7b0lBAVFkD68obRfBkimRz3Vp4UizROUfmJuFPj+ryCiQDxsPNRNP0LYeQpIFd8MnCrm7Cn2qnzhTw28We9cEH5y3WwcvpesJzCXVokUcCVlRlRTJmVpU73nBclvYNQZiHiRqRQiPX\\\\\\\/oEO0w54vv\\\\\\\/VOB8Ve6gJ\\\\\\\/Pv\\\\\\\/8xKMflTM8NvGfqzrSQfVgxmtGRcEweHwaM9o2dSV7wZRAp3\\\\\\\/aYG1SCeuSDrDwOj47t1XHIwpGIWd2G6OM685vFapA+HZ+4TAVH3PvTVSNuycZH1RreaO1XN6RveV6wIcgky0JgsY9KrJobUGeOGQbOJbrsM\\\\\\\/8YIrttfKjjBWez1XrRkD4KlQ2Gq31V2\\\\\\\/owbwq4Erued4aV5sdFsHA7QtIjsvno1ZGz2AD8gwXwuQM0EiWt7\\\\\\\/eYXiJdLoiL\\\\\\\/dpP0\\\\\\\/M5UnjXvWiMIqqtus89QEI7H96zr0yE5tCKR1XAE2Jo3x17\\\\\\\/JIdJc0oUvWDNuBPSf4jrxrvj+Qbapd0QSXkeXCNtaYouj9dwQo\\\\\\\/hLa\\\\\\\/8D1wfsIu+so2E8GWmBuWs5+bAWm3WnUZ3KZPgj6lUhvtoiwnu3yJuzEab677GSys6CH8859cWjEsSpK0ZoeqvpotwBftUeHHYDbeft6Swmjt6Tw5P4\\\\\\\/UdxsoYndFpZZjhgt2aM7921h\\\\\\\/1w1ez9RlpvQiEUYhI5VXOS5uA\\\\\\\/zzttjR3vvsfSNydkNU5PDSjeFl+x\\\\\\\/yTUGlQSoRwfGURwouAYqKabNkCDk2h5\\\\\\\/EGpl2G0V2TscFTCJ6XJ3L6WLAQLMUFh3zb\\\\\\\/wb+aiDGu7avC9Noh9T2uZg\\\\\\\/zh6jfwpejjOZnzdGLD9hZ6wpoVzoTQUTIHTZ1L10XPddzqLlf\\\\\\\/iKMXHbbezyyhuXe2rKWz5hYJp0sK2oJUldRK5hV+dsE5PTbg+NBBFvRnoXib2Ez+t7a\\\\\\\/n4VttflKiWTzByBmnXfhXmd1RUpM7eFd1Dy0FH688raj3OvLqT5auE2ZjCNv6QuTsmv5mydsKIhIWEpQkmCnAq7A\\\\\\\/9pDz1yFT3fhnDS9quveTVs32l29tRbtoDdeQ6y0Udf7BVz04if4ns4WqFrDIDax9\\\\\\\/xAuZ8nc5wqDlSTIqhIj81OK4VHUSihtKRnIcMXNbj8IKfUlmSWC0PTBofVJkpSpCVPJmMGlVEpV1NOGnj\\\\\\\/1UN6qs2GiDA7W2GpoRfaPRsLm9oWwzl81Qq\\\\\\\/\\\\\\\/yCYfzn2hk6noi0a3mVAh0JoDFIIHHvXJ+Yu3+n+P9awbhM2aAcLmmX71VUEMudvPYK+7efgZglfyJgwPA+vL1znnL7HZ1p\\\\\\\/N+4KQ0vAtxzz55prhkIlLbdBfSg7hnaeyLbETHlxM\\\\\\\/PWEfvbIeas9oE9JqBTefcLWvZCLCVxbI4CgKicXwY0okSmQ5v5HvyLPg1t5DOLIvvEGi0FCxIHCpYtpjxZUO\\\\\\\/EICncoz7aAshI6QESJRk9cSznelJOkywN24nTE6+vzlDmJUmwGoUDakUkCj55u6Kh+z8CijHLZLZb4hDHt+lKYaR6RaXNtfllwfS+2\\\\\\\/jGCd5eMQTmkFjmdalcNbouVBMOBsw9E\\\\\\\/KXIB3t8pA1UDlodIYlyrGOQH9ArsRKUA\\\\\\\/PSahfyapfJ+ong2Xf1ukqzLL3E2qBkh5UR+Clzu9MLVwn+D8NnQtHX1P7wRTaU9R8KP3Yr9Nf0aI463FRe5m4owXP1VQkcl2fSa109jREO2xTe2vH8Db8qazt5pm\\\\\\\/kzw71s2Fwh8+mvN8sxtsl6vFuUMnKbU2H\\\\\\\/U0FDtee6IQY\\\\\\\/d1fZA5pgKDvS1DkFJLvwGzh3QiwDcR\\\\\\\/Sq3CksALJSHWQt\\\\\\\/YPRK0Gev\\\\\\\/Wv7nZsh2KMGXtnspDjX4xz2vWPcYO3OhNizzp+\\\\\\\/ZcMQ+EArBi3RenXafBmepCcyCsRu711C3sYLXH4qRsorGiHwkOMZdmyoMH9xDHDqGDXYApvUm88eCj3i+zKry+Afn3CARw3Ufefk5XQImK\\\\\\\/WyF1L2btV2Pb+Jt3lho9v2OHCFau4weo5hr1CiPnJ+uQjFE2\\\\\\\/SEsm\\\\\\\/eyZrn92ZN8bccsY\\\\\\\/9PnNji6WD+82GxFr4ykXzFOjvrpc\\\\\\\/Rb3jDnaWPZ7oRzTrTRbYMknAb2VQ4Xqe6\\\\\\\/ErcQeRK1EuAJajR0383aLzE2L2T8z09zHf0h2SJ\\\\\\\/QjFwgdBbgiqXx2eAJGjuyyj4w5TDeqNVyXDRXtNM6cOZ\\\\\\\/II9m721+nA6jzPY7CWxSsmxBvBr25ilPgiUVhRL7XgN+Yuzj7Z1hbYgnuUrFmEqwqGz60OqyC3OQnxDMP54XgVt\\\\\\\/jvTFpvg4BUW1V1qyg4LUBV\\\\\\\/+0HfSu72nlqwVimecOFYhFR5afe9Hw59icJzIibTXjh0S4jhnO3E1cCZfrSqqVx\\\\\\\/xkGlpbLK4YWEoBTNRmZQQwtw9TR4a31jy0fRSn2ye4\\\\\\\/W2wjbv4KHKIJAXudUxPsdlQ\\\\\\\/U\\\\\\\/Guc+ZtCG1cI+mLakSLQdP13KwRAqfNTmVeenawUp+VLBW2PK6O5sxHvyzXuakZxfcCzT\\\\\\\/IMhQ3fhrbFEx83QrJ5A\\\\\\\/q\\\\\\\/3nVwLR5xVXlQtbZHDnI9t8tW1zPKBqbF0b7FwFPmY6sdTU06Du4OrnFpIXHKaK5sUGzdXPzcKEanJPQbRdSP3h\\\\\\\/V8rIqtrFwFD4ygnDN2SoH\\\\\\\/HTXjfQ2uWU6y0fAWFp2o\\\\\\\/oei+jabhjlcB32WJQUvS8F2fdvxQcFjrlykQFTlfNCul57A28p5QZt31BJmLSScJn400FBwwtEDy49w1FTkIibpy5s8gJIsNSzjJ6uuoMyV9c4MeDzNKkjTxkv+r2sCl7lUyi3ffp10VNL+71LUirIB8nSbCVX3BxbMasgFh+jrvb0SJYZsnf6HKzaMV7ej6dPpdJmhsgT3NDZpDeRVjlDEPbFQyh2wA\\\\\\\/HuyWXqXQlG8908keQjjcAI8w04sd1HofBs+7zeciON7UqYY1Xmb4lO+\\\\\\\/VAmSGAqD0hUOwkGag7rZHq2gkmH9yJrjF9k\\\\\\\/ftrK0kcm353+AqWLDkZ17P6JTEcZ+4GojiCwUzqWmhrvfQISN6rsdOhjLHTBwH1h7CPjrctHXk5y0xEkFl9Mm4681dDAlyOW\\\\\\\/j+eW0aMNeXGps2GCrE4IjAqStgvTzto1iezPfKOUdIAaPdh\\\\\\\/hP9StGGVtrwfZLEUSqjDpIJdDpI8ENVkAgy6fLkNHmEAsRT8mW9BY8s1ASb+l9BRg02d8w5SHXFCrkGpagJReetFzKdDRSUsQ06RKWhqbby26RHHKTaUSW8HnuIxOZ15qT3yoIyAUWXg6uVVcEO0MudLOClNGRwTM0K+qRI5U0c4HS7IhuSYG8t0\\\\\\\/eMXuUfxsmBpUXAF8D2I9smCM7nmktRY4LfCUS2DhllY+leHPHPsOxXuHwZ3mZ8uQda6aGmus+qPAtx3697yuA2Bb8juhOr3z2W5JBtfkq5gQ8CJW23TVTesmijE56LTpxj6MH0tijLEaUCJQ6VFf7UDZm1PlCuAnk58m5Hb0i8wnd\\\\\\\/uK8dgvvNsz+nAiq7w\\\\\\\/cCO2YHuNix2S3Ja0U1K\\\\\\\/RqdLrsGsdMGiCT+juAP4i2H1oswWHK+kqlZMTOZ94r69f4H+xcRSCkteYoXBuTmFwz8wyJz7I5Ja90anUSDMKS3JWEwAygorTeY2rk6X+lQ1X3lc9h+JTB0hXWMlnElIykha+MIBvb25iwTIwBSGE+vWeqhKHc0EsCX3ccH1vr+n2ZWyDvQwywFvEcFEPl4nsAj+QzeiM1ZPZ+6341LTs9OI\\\\\\\/Q7\\\\\\\/6YQJj3T4S14Geb46BOTXjZS+Wix5\\\\\\\/IJpnXSLKTP4JI3k541aRWiAMZfzcknoeAq0q5zzCFxqPb3c77kNMZMrfLhL4J3l0J2qfcd9b6bLm26F82aqrd\\\\\\\/fByND6seVg6xcL9g2PlI0ALxzTRE9NixxwWRmn0nlRQzqBdlN+5bXlGkJ9eZoKPTuBauFQZFBamwx\\\\\\\/QdLqP+yO4qsMgFgwDdQfDEoGBoszHUflo8kY1y0tCYl1I9LXd7dymP+zffZ0PAM4kGoMbAd7WPAL1X\\\\\\\/8MTxGir8CVmJoZQaLYORrQfWkWTAnTAiIZu\\\\\\\/igmAkYWqwQCEZiFiQu4Xj4Vid0YviZ1wgqrFQiO9oScU7XmYkbnoPgiaF1ZhS4fJBYthS7Q8mbIpw\\\\\\\/qx5dg6g7zdAXALCxyhthma0at3ZwMHU6QccagxyX4fdyl1kLV44bt\\\\\\\/7myf+GfY5L2YK5JicX\\\\\\\/PjwyDDniOgqgejLT1L3ZyEMJF0vNikl+42kD81\\\\\\\/0BB4kkgFKir3RIkP+azrmYZr3JcfI2HLp7ud1O5fwLuLFJds35eIuCYSOnT15zYQzlvUqUmbk62vuB+sK34+PrSQJvxjqTPShEy7EJz9\\\\\\\/3vObEfK\\\\\\\/SKnHHV39R6G6IkRVuUQrK+7HcfFK6JeE+ctx14IwUqQ0Te7A0Ojg8b2j5biLNr3DYu4b9FOIzIbYiBhQqtcp4B+sqUNMkOmIvOFXKL2w6eX05SMClQ6cQhJB7W7hqpM+\\\\\\\/ba1Ym7wiL4eVeE4Y2iTKraPQ9+ri4yNLN4yEKK3SLZWNSmKURJmAGqwy56Rip8k9+f+ohhfvcKsBntFMcUUWADdKPxfPysDcxYY68J0dwbI8GfUEProddTWcXW+szoyXFJ1C39XJ+GMEsLB+Ae7ll7AyK9lKpFWWYCgdW6wqu4xTCm7LzG4KLyRlwl5vts4c7m0mWH98ENsBKAiKB139TRgsYVPj99QxM3I1M3G0hKkZJitzli+B0hpViTPsby9yWwajkKqwGM+OefTrgZVBLaxYLHsd3bYCvuD+t1c3mFxDLB2V06ANgdnR7n8ETXwPxY4YXpyqHEWvUQvMHwZBoqxgqE8cI13Banltkfg5cclEIn7yUda3hpOPpG5R0mioB8kyLtrk153PJjbLsiMIRupJsWFrbviqhcU699NSPsrT+4NEHvPi815Fqd3SCSsoAV8M5F1ni0wpYwcTGoXy0dMRFRQg8xi1fkKgE6xmkVfDCn0WvSEYIxafaIk0frn41o+fHrLe+aDxLyiBAcG5oD8LRztFGssI6olVbj0UrSU\\\\\\\/CUTH\\\\\\\/jKYczekPttO41q31wOkwiyv89tpbwPR+A07RjzyRlnD9p+6+8JgwSipOcSlhP6GJ1wkCQyAae14KQlzZZZo2E837BNhOFA2Eu0GD5nqVyVu35DAsnz+ziMgSoV1cWC2Ef5+h6yobUvzIYfxz7sOGcCu5wQItUG+LoYpJOWw2TNT6lzMTGz3jbFH7n09bKc9SKnZRY7rtKBIS6V1dV+7WoQAV3jxs51pt+TUaOYiFwi1\\\\\\\/CUTcj28wK3T82r\\\\\\\/r1ZtQGlzJ\\\\\\\/Nou0VW+5GkG0nH+EbBeCSK6rp2hODbWYEpmF7veBHqcYw2HY1o+29yuOd0F5tQp76YTaErcqmhMNxKVXojBP4XcPuAoH9PxwPkpvu57wV8RY9soXC5E0eUH5h+LTPHylStwho0mWumUwbVNJ7oZsAmTwPyXHIfWzbH1sxNui2mPkjpqpc\\\\\\\/1BDdrfSx2HHEyVLZV5CeHGKw1LtGLsNKJqxy2LKcw1yA+\\\\\\\/sl4Ql+B\\\\\\\/fcNS6EAQzAVlK9Q7Ej0kl\\\\\\\/L9ljB8dyR4qRNsUCYp\\\\\\\/ux9r9dOri9qYiX21tiaj0z9ktgzcS\\\\\\\/2lvY4aM3G2gq7lBf86SURznEtuBu8f6Qxzh9Lrj96m4PelU\\\\\\\/U2Poe\\\\\\\/ivC1pyY2n5+afjTcUfVAeXzl7ni1AaRyLWaRJZ69ScxYNTA2AhAmm0CaxAtH9eZJZv9RQeE+p45yYuhnoAi3xhGKvSRRkxVeJgOEGzKjTc5gkDgdpbXZT1vS6ai9ueXzn1eVlMwq818eY7vmTSwBrzkCU8GaZax\\\\\\\/zYO5U+YFo3V1C6ZroWsUOvwrXR+4m485gcSIstvGjcvHjnCQWpAkTdGWoz58GqjxzRMyfMHxpm5KS\\\\\\\/G5snG72FgedmAa0stlY9fHdHxUtnSwMUN5dk1ESyufvsPLWKVF1YDurIL\\\\\\\/T2upHNLov8aexcgaOwyhhluFYvm9nn9OLdDgmEVWbWP7\\\\\\\/PRQeNuhLnPr9uHE0ql31TU9raoHSZHJWgnqng6SoxgwQuXdxHxfG+duKIi7LW5sxAjewccby9IHyYoYGx1QQuwtVdwNH4pjB8co7YB8RK+EBIdeQ9Cw584BY\\\\\\\/Ac1DCMQT\\\\\\\/Vh8Fkt6XngMRODWR\\\\\\\/BZclbQZ7gwHpdT9OxO6Y+yctGFNF+gjLwICvzL\\\\\\\/4kqoF1PbdXmqY+\\\\\\\/m4DiWeOpv3sU+Q3mlzHe7PBgifKEAulk1BZO7xle\\\\\\\/\\\\\\\/RSgh+iH\\\\\\\/DBHDbkOjrt+bF08zJ2WZ\\\\\\\/CJZzCsAHqVep78yeQ2SI+jTcSLFP12gdA3IYjB6exXXKph8YDlv3VLArW7s2++4pVSN67SV1bufnulgsx+X03ogd+EvLzEElRDp+LnyGS4CSJig8tKEz2OYJwzlHjOSj6nhQ861JFcTfazs9t7Ya\\\\\\\/WCtBMQkhO\\\\\\\/\\\\\\\/MOrJorY7ThJoXGHc7bsj+4sKQv3M0BQhN3LsOoybb60SwNGFX7hrnF48UaSXTyXSkx+B1jhIwuXHu\\\\\\\/MWZat\\\\\\\/G3tHgqHhKTkitQBuFJ2eOGgZ47BG2LQ0r7z6eC7Mw7X9pvzNCzn7fQZQyDa69C\\\\\\\/Fynvj7jEB\\\\\\\/5VKZs8JzI\\\\\\\/YdGoikpbyH2CZD1OKm8EQCISrKMk8WxHjbOAEC652fXsJvRv71sfuyPjLZtoAP2suY6mKch6JT9zpS3HS5TfaSyEjoVREHgxnEOoWdnXXpNTuAW1zZjrmFb4ZfX3HRNlTKYI5vbYJByaOTs5AyUqyaj6M\\\\\\\/a8pra8nKx5dn612MtI1nIn629dZugGwc2XJncldwRFZkOzAzgg5iUtdQ7ItPTxIgMhmwn5TNccu8U1ZVcQ1KeIcwijJ4s3X1yOEBAv07QT3nxDg6YRvU3hHxl3NqwmAqntCpUdEfTEH72fxMHUME+QHtMHAFR+q3KTjOqsi1RPi7alNTkokLpgsfmDPQlbWWWkJVCTe56MI89wmQjlycp\\\\\\\/KzPmliRBf9W9JaTeAF\\\\\\\/\\\\\\\/uO0tWTpolX2il5z3kcIm+kbDGUW0nZZMd0ecICxlHWeFX7LNSe\\\\\\\/l\\\\\\\/YZBRdDMCFVJYmI6ntbbG0rEiyd4\\\\\\\/P2blpPs8xtTNm0EXM\\\\\\\/NilaD+BeUYF\\\\\\\/yuCw\\\\\\\/QNFY4f2IB6s3JKzVbRfbKzEqb55uqYybjus9IYQL3IaUYQtmR4ELLpT1+HeAMMrXEhBaBNBXZVoNezXYRdpnPAHt\\\\\\\/pjztpV5z2UG0fL+yUWLo3x0Mj6moVnrHpqflIUu3owi0kQ0pdqmKZf5oCsryrnnzA6QvwwTNuajoWjr\\\\\\\/B4ZBOIPUO6NqSxmMtvc+3gD2dsr4ntn27lN3Dds5dxk9kXRPYHJroD08OkI+npejbnIHLnRdwjlTccF1TerFfstbMPAa+tHtE5SsgHRJ4OjVy2z7NQmTxnwK47MNsFkn\\\\\\\/bl4KYxgrABz3rPHnwg8S1FJjcxmqESseNgl2xNcaK9NXhjGJb6XtBpx2PZszrX3Ht1LIWaRn7WZVpS7jkJ7lGMQwm3O21geFcujTuQ4Y0k+3Od1wVQ8RztbvcJGn07EAt2Zvb3feGkDxBo4loHa\\\\\\\/Eruvq6qt\\\\\\\/aqAUXEzDOQzgVB8O7XALs5FgrqcUx9LnExrJFJEnPpkOTXv8VUmIze945TCT+bW47W3e0RnVuOC0hSO3P2WGH1bz9X2fXSCO3kYtM8VHYm0Vlt2qGzpVQ46RpdfOH53fEe0JSLi5ccYfWd6\\\\\\\/cgh2PYnjGi0hOAnNhQ2VGpvEhUkeBRbfM1f8UTr\\\\\\\/PpFZ3MGp\\\\\\\/ocObM+a603gcIX7eGjt9Xu68+mVxe9O80QKxSm7sQdWT8PctHgyKCo6YBNiIHRaDyD4tCEHatBVmIyaI9JgDUUJMcx+NYEMYhvD7ujn8VA369RQTZRoTPROQzDdxpDMpbxjop0sicD3K4r81bFB879QxKD+MyiLtbLFtw1V2BQEM9btCWpkKt0Hf65NTJP+LsSP9Wfx4MzLrMGrLmMaXiijLa18gBUcBV4L2vPxKl6Cx7tp0kE5dK735jMYULlt\\\\\\\/zt6uzxm6NpxhmS\\\\\\\/f+43FCGPWbZXroe9vsQv6YzHh7sMkASkLiohxwKDctQ7dG59iDAizGibkOWWfMceBbvxEfEdsHOON42gGccogYP17vkRw034WB1YIJMYZGXEmTBr4IxaSQjd1xx7337u8HEOrmEDF9ns2tPvvjucdQ0OJ6twxn\\\\\\\/rmGXc75CAtIXS3i7GOjZq1IwIG8ymx2608E6Ep0BCjsJwifJYqjFWt30NGxkKTURD0sA1HtLc7OGVTVuS6i97qE1UsN+x6JQ847pwzrZqL0Hjeq0oHqhgUHCOSXut\\\\\\\/bmCatNXseP8MKbTSogsmEeKa5rMfunuH1MDSiC5Y6+2xwHnUCQ8iWGIVRV38rI9U8B4kjgP5e7DM3H\\\\\\\/nnA9zTC3cPNSL5kIqH5SdiTTX45+jEr7tVEBJ9YYnRfTFSoNmPclVF4z1IvchmBu2epk1loZvP+VMzNTupna45CBnLicrUo70NTtpzZI5UikMsmVz2eoqLsbvSFQmCcW4DYFqPDYzfvzfSA98osGXyBqD7Q5tiMdB0K5zPrq4FW0XXbPgjw2Ps8RR3t26ltp90ztGzX0HG2gImvBreGl+PyzUWjWmVL0+OqVVL5uH6y\\\\\\\/HCtnx97AwKmmD0Ci555QEk\\\\\\\/9HS660h26Xff0H4Bqy2ci33yX7C\\\\\\\/NnOPko5vcVQUYWfKlzkBmXB4oqv\\\\\\\/ObeKMhfVBOBOjASpPcAKR6Sfliqc6vzEQOpS4e3LdruYq7sfl1BDRlBlnPQxNztxieWwz5AWGbrc1eTC1Va1qSblhYXKj7pHSUXuRJxQnQwDyRvfRahWiLs68gkX4oTQ5L\\\\\\\/ogUjlA7Ywg51HbJqLBMJQsmfGBCAt4Jl6CuyfLE0CUACSOFSslZbfdLVFVH01ITFpSY3v\\\\\\\/mjZnuxKtNvwDDI8NpXLG1E4+isawOJEpYcs\\\\\\\/3iEPEFZURnif3O9kmUSeTYymINbygA+cp9jKevLS14AQDQnX6V809h6nL9hyaF1C2SjK1zWWVJBi3F2NatTCrNTgaqgfEKEctdAi0PXJlvuC4CxNIOkezQhSxKRF\\\\\\\/LECvMNVEFCIL5L3G3kZSHRd+uXVsam18H+Ve4YcDXNWbiu3mAyoLuqvcAJGMOZb+bgt7sT6NA76GB45W\\\\\\\/2LEqqGh5ivEEIjfZkWErYAfJ\\\\\\\/oEJxwTci5scd1j7s2uFXHG1WcOBHX2SdbVQbmHOw45TmCShCtrI75WBRV3USiAr1WVgLeOAtpMowVUC7KLSwV8SyHeAjwqtWy3Lb6p3owQi7TszlsSfzDXMdEoJWk41WxJUkWmi9lr7X2Q6nkIUJztVkVrO3vpogCVB34IVxjVe9dPUxeaQd7CZkCWyFW1lcdVIua+Lf6RTZccowug8RaqFgN0jEN0G6Uwzt0or5OOTReciwztPX7PYnJ8cW\\\\\\\/4Os5H5lWkFUlrWG8o06+XcwbrFDVPSDChTPoTv1NH7GnkoXBrsaFuVlufXGhGO2lpQPOO3KBwHnSZRLoz69RRzXSxSSb2miikxLhp23vPKN\\\\\\\/YT30kO7YqJzSBGvwDsid+H14xOnfnPOTDS0PWVM1napoaWDnZsFkKBr3AvR+umoH4B3iGxCLI5pGzRCUiZpVgQb07PgvDIeh00bC6rsFqD1sSJYOVhOcNkgWVV7Cct8A+HPzmHSAmb74WOusBKq+zOdf5sp39Z8G72Gu5qw7RzWkDuaTOI2mEmQC8t+FiZUADDHi\\\\\\\/MUZ38Ez6jkG7at2wVrR7jivN1dA67MdXuWCfKB122F\\\\\\\/dTHaKgvmSGmVqOrf5ZtyucTjQjZgyAT5OCT3plSxJ7ff9xDGWhD0TWLw4Ap0FaN4ViOH55a0oRvEib7pQLAf5NRfJBaMh\\\\\\\/PlIJeEYnGuEZptycZQ\\\\\\\/1rKRVd4VS7hzCjtiLFLVGHLysJPTnXR14C8kltdbmLcfAVyQSCgGA7dQ3ySx6R2YFg5f\\\\\\\/COvBWB1MGT6p9oDVhTO\\\\\\\/MytgZqO5UVoGyv6EDbXD+FG7LXyYtLGE8xBQ267ETOndvyQA6AGdass5Q6uInevu1xyI0fbu4t3PShwDkhpLz9RNs7BtJLb23WjzFfTAoX3LNj0RsC9KytOuXk37Tl\\\\\\\/V9GKDue3em6xL8jFuWJkNcXxfmi5EGRsYNgvVST+9+OYHuLch9hzjEP2yv0plfEikZBT4BXQKPL10IWtVEAN8KpBjdNy3K7dTiT7X+LYelu4504ZxFdtohCp2D9O7XWLjkbmiKIV8GDHIFFScUxpam6CSU4xogSe1jrUQiclviLfDJuMR3H6rWhcxzdQVNwM993pps\\\\\\\/ItHd8ALNFWDoRpHzSCwysbXuVIdKwjAVaK1CLRcTOOske\\\\\\\/WuBC0uHmzUo4y7CThHkwoB7srxH++W76LvE6ddEXX\\\\\\\/mM\\\\\\\/9vUA9hUFy53vCqHml97GHvybre3wf+x+rDhJp67UstSu0gPGPmv7fdGLzNcU8oAXA\\\\\\\/O7h1MZa6m9GOTWDy5MQuf65l2KWbAxUnVVWPwUmmLIeOlc\\\\\\\/I9liMZot5tHuN7aES0ZB8qu7FuBUAce1bqg\\\\\\\/Vd2ZiWltLdG5k2IEIrIBqJGfkJAoTxEa7yOeYO6bzzALYANOYi0XmiaLueNe5+xj49h0l1R942fpbAZKxyirRW7YAMBVPGe39jzXy3mKN3HhFCpnyiCxho4nbqi+nHTpXg56gYaKHXGG+MhpEaPh0Xd0cfYFJK9ZDxxqmdi36chyBmVr+8iNyR51tRBNGssKL1bTJQ9v3KD7nhvnkxMKICGdsIr2N8+Sy70RsV\\\\\\\/FnTMTzP9t5kbKrjQYQlC6UIDSlw487zV1br21iz9K04eMBbFWnU7Cj1C+4e3kEOCSgxOhLNNcXMUcusvWKBDgdMIxv+eWQTqGZM2SqY4qf9CAx5ontH2PQhD8w7zPUjjiZAsSQHrJEf9NdjYm94Q96XGHr8ApdvBcuUP7+YJ3OOlauzOraafZCuCxOXSBYuf5dp1jqXNTlhk\\\\\\\/vl5U\\\\\\\/0fG542kMRn3l0\\\\\\\/IyvjJF5WJqNRfbbylK68nT7NJL5sTIFEEN3xXwOgAfm5VUenUzWanBy9rn1bL1+uBRKCrLXSI0pzycrMCsuGdMwdIRZdbrBONMlxqyy3Hy4QLjzh1JOXFY\\\\\\\/jZHU0Kio0jXqk1iQ9dRj9AcCpLLGTkbmq9z00FMbWm30d2tjtIZKDKkGJoddMYFW6OV7LXWHpCvY7B7I9TTtrs6XtPAGW5DxYyae5xCnCzfIIiyqx6QG6QnZW0vgFX8LGfV4n109BtL949ps1Csnemu86YIJLzAyEy8u16MVmsR7BifJawXOky9qjzGe6CGPzHlSt9\\\\\\\/Bq\\\\\\\/4p3X6dSrEPwCoDSvebL5WI\\\\\\\/yPTY3SIDZvupDDvlvMutSSDE8Z7jDvni651gn7C\\\\\\\/tIp2+loxrWcyLJWH3pWkBPABfgBwmolf7k9dRGlK95mS4QqBIf45cygLvHBnJQlI+uI5uADRYnDnomECvPLIHYpTDPstvL\\\\\\\/CSuS13sI2WsWhcMNqJXdeGDojx5l3n7A45vExhUEWuAAUE\\\\\\\/MW6lusDoY3MZWxCibH\\\\\\\/gmHiNk1M6lJROUkWMJeCGK9bzYff+rX6\\\\\\\/ooj9mZYOG\\\\\\\/gJPyw6iEmNQyThFVaNvXKGqynQjD3SVcPxJb5ZPGqAGO0+c7nnLwibYhfDJUFxdJzW9+qZ0Sd0+XTzlK61JmjYg0wRfM35o8IOUMzn+BYYXiqbenzR9hQmGvSqTUxZR1aJ3uIKF77RLSNe8AUqPWOJ9M2SwykXWxbWp0uMVtxXN0ToBOkC8UKJDPJkZtZmI5K3WnzjdOZ4p9y1gsfV3IZkWGlhMSV7Yj48iyKWi\\\\\\\/LWqr0JuQSQ+ll1jxlJO38qus9ApUdd5yf7SM4yg18jeoS3MLP\\\\\\\/+i4Su1FftM8Fguvl+OnKH\\\\\\\/zgEzaGSyc+ImEtrmexVw52jSeAn8uykXiENGiZgEYE8H3Y8Yp7f4\\\\\\\/aD9yhEga7KuY3RDAtdwZJbwXD0ELyd1q2mwxWmiix2zDBOEjBn5wlRpNe2SsskNbhl2D2ptGEtfftIuj6NL6\\\\\\\/P6oCcVAiJ3SpbzGb1FRj4PKx6597WexvfmsYlMk39yS4BhY+E2Ix5qppLjsjh4RLfuy5vwFRWQCyOlsmyTHaAGGWyIlivSDP81RpL0IPkeRAZLopiPpm3d23d0T\\\\\\\/KLnCITg3elnr2eprRG4VrXsZjgHN0S4i11QZ2gvTGOcFLEvy3hM0RkuFR9qJWdaBgqzJzpnz71KsYtZYZfv7jgS7OHhtpEh67SYbgPdDG2rz\\\\\\\/VpCyPNlKsZgBbhUncr75Zs5j94KiRzATdeShYxCGFtshNbObfWXM+4bZWGPS8782GfmMjAteTuyyg7TU00v540Gp0i2jGxdONH93mpZ5z8IWJt\\\\\\\/T9f2MJL3tKEotBhIMB0a0mj0w0N0sLo\\\\\\\/ngh\\\\\\\/A6TUGOu6lOz+L6XFeOXmkosUcs4fAVEBle5chJS3Mwb\\\\\\\/cOJN7xjyWfo9kd4Zt4jbq6b663KKyy2MgKRl1PdoXDNhEvlWoV1GIVEMqT+fFNjDvTIQNOd2ToKv3p1F02ZdljAeehBvGUcGjGl0azVTffsKTtXC0U4MREhfmHIMjN0A5cx3hY0WZCaf4I13hukT8cgnpZKD4hpJSUwhkuW5+2kBp1dwDxUQmplqaAqSZbHXL2LyOM9oTEPeUNL0gqRoxcc\\\\\\\/RuxcggSn3y6cGIZsy7qOfdvM4IVRquVtm9IqU07DmSCXYXoFaXyH4OrU\\\u0022,\\\u0022iv\\\u0022:\\\u0022iPuwWuF8cB+nzg32VePHng==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022dcf0fd88fe1c8de0afc45a9c566a14ec481a23193398f17f0810de4b750ca92d\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002218nJ+GXv0Q1fFho7gVRwaoHQ02U3\\\\\\\/v1T+tERF+qmUUoz\\\\\\\/11qOCFYcBARZ0iKutfZPnebp8EGq9nO5u4oIb4I2hOXn8SkIvoDDe5BA1aWg5jYjD2MM9uEBZFbY3prxAiRl4xP3OdY0Fc50JlNJSDiZ9eSQtOp5+nNXVJ1h8bVgF1ehiyocPBvkg5dVRqqFbxkCIsnc5GFlmMd31RMUvnRdQsypYxUHHj5u7HVy2MQk2xJQk3ZUURxKjZeP5C2B6pt1SntNodnfdStW8bheW2QQtSDvAFE9JKYmD9cj6bq0vbq7CYPBkhM2yucp8kbPg2bogBlzTbfi2\\\\\\\/ewfuHX5yTwcbEMVuBn5jJdmTUQBn+ZA4x26I8+MGRJcUdA0eh4xYZ\\\\\\\/vtSvH1kDcN4Osy3\\\\\\\/rBd+rxMFiamjZ4gMsyPal+IwJgru41kdw6ars\\\\\\\/dajGp+gegyyQHZpil55rjin3pAEOZ0yh1FiCNckCJW1+jhpnWl7CvG4ZvkdQa\\\\\\\/Paz673d6K1AehSX9Jq4axNlCK9OO5Th50hxw3r2Nf8wfgGhblPhpHwFdkIo7XoTwGLu2xao5VsnasuJjXmZZy9hlJQ6Cra2nj65uIbeAfCYoy6uKOyCEfg04JCGbWnUVYoqYMQhUS89u+Am1oECt0Mod3XSJ32mU8lcBvvff1NPc4R3k8M6FTfXmfXOLujbSjiM4aS2onFcbAPl6zlWZxMeSLaaZzmnuyBfM617eWEuzA0SdGMcWBRyjPe2sbdVCdjOxKkAYeRRR5ks6EbuqcbcqjyEoIs7kBXhoEmwK9rTjWag1KuUb53riGZmUcNuq+ixf2G+pFOusS+5d0c6icytS4WwkiBm6EdQ3axW7UvtP3Yh1TSCRypQOwZQ0f4xOm8ivrPylDndgxcnkXK27NFFTT85rFaH7+1aSalAlxDEzlWmxaXHYxxDt6gGINheYE+aPf4c73D0W6RV4J8+aWbPIK2AjO\\\\\\\/p3AnkM0iNp1GiGMjVF0UefOTbmr8wwJS\\\\\\\/8fiPOi64hJkEjPm87DI8RjuyhNemLK1sgDI9CFF\\\\\\\/ARmND2Ai4QnCukA8kF9+FREoyKfGjulVDNvoT6O1fwlZK37sxLmpXysH6JsU2KuluwPOUzsjKwsGrvv9S1nbTPT4PvYG9UHw8YatTZSt2XP27IOoC5UcvUDlSfavMY79FlpYsi4kinxUxatZcQfnF\\\\\\\/GOdP8FOONqDcZmIGhEohpRT1WNrhmgQUzqbyV7Z5pYAVx2gxHTyV+GtwU0QQ82gp6bXGx0KSvSxbsqurLXtNpZ5Rvs7pJ8mGyVdk7gQ\\\\\\\/dYjMHphQoIiZwPG5S0Sqlxh4SkhmQK5cjGWei0pTw7KlONQmhauDLaK8nBQipLTRwOv10Mzu+dVWIKosdjnkLxVGZnsAWknShXPiHgY9CkUk3E7vlVX80xskHHrZ3QXHX5ago8cCWSHu91BEOKXMZEzTOuGft2MMBdYckxbNANejI68ES\\\\\\\/m0RdEYXdiBQ42tU1095cZVyNA22d1e0TelWz5aobUROsZnra4jCwxc4qYABoCsmI5+QgW7Imtmg7Faq+qQTMQZROs6eEPpOOWxUKIhh03kUEFxhHriEPO4DSJZrufNd9\\\\\\\/8l5UarCpEP0q2AlPsyhSmyJnY0o7QjnpjQUNH6rcLjoN81UQl2F0ENvtao+GQ+LEAgl4\\\\\\\/uYLAgk2DGsW17f0iXfA0acyzgFypKvxDetbSRoQeizSYuUqcU5JgxlE+PdikY8vXXvdQZUpT9nkyheLOWJZ3ruOU8+jG2k5Y86r2Wo+Bfj5otN+sramluQ1QtCVnWRyE9EotV2xLJNMbhnXo5hOE0vynreOOn55jReQUUQKu+enqdOYB4LPPs2HC6ExPCpiLMRMwVtaOe7\\\\\\\/8UloMIxv4sfsKP1lHVQN71nGmqpH5Wf1O4tLegw3PLdGN4vJl+4wv6wo5JegzcG2nra09mNW5TCIgkzkE26de3RrGDSXi1O3sB7ea4YtvA9zLPdRupHhVkSNDTGGfBVmeCedAlJIVuooTo=\\\u0022,\\\u0022iv\\\u0022:\\\u0022DZUnCvezkhUgxX7P3DAJ\\\\\\\/A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228b16e888c17d69777f40f4d9f367af3d827a29089f62c47b0c2a2fde8df9c19b\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459291 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/7/761" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-5uAvKKQ3food9-0bnWcwNGMAKQ5Mnjf9JfaBOihbhPg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/7/761" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-97735ddELdHmVXhwxJ1-FNaKdpYihil2edX3bhQgdW0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/7/753" title="A Noncomplementation Screen for Quantitative Trait Alleles in Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/7/769" title="Genetic Diversity and Population Structure of Chinese Foxtail Millet [Setaria italica (L.) Beauv.] Landraces" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459291" id="node459291" data-pisa="ggg;2/7/761" data-pisa-master="ggg;2/7/761" data-apath="/ggg/2/7/761.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Impact of Loci Nature on Estimating Recombination and Mutation Rates in <em>Chlamydia trachomatis</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Rita Ferreira</span>, <span class="highwire-citation-author" data-delta="1">Vítor Borges</span>, <span class="highwire-citation-author" data-delta="2">Alexandra Nunes</span>, <span class="highwire-citation-author" data-delta="3">Paulo Jorge Nogueira</span>, <span class="highwire-citation-author" data-delta="4">Maria José Borrego</span> and <span class="highwire-citation-author" data-delta="5">João Paulo Gomes</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">761-768;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002923 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459291" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Rita Ferreira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Infectious Diseases, National Institute of Health, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rita%2BFerreira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferreira%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARita%2BFerreira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Vítor Borges </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Infectious Diseases, National Institute of Health, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=V%C3%ADtor%2BBorges%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Borges%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AV%25C3%25ADtor%2BBorges%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alexandra Nunes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Infectious Diseases, National Institute of Health, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexandra%2BNunes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nunes%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexandra%2BNunes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paulo Jorge Nogueira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Preventive Medicine–Faculty of Medicine, University of Lisbon, 1649-028 Lisbon, Portugal, and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>General Directorate of Health, Lisbon, 1049-005 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulo%2BJorge%2BNogueira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nogueira%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulo%2BJorge%2BNogueira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Maria José Borrego </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Infectious Diseases, National Institute of Health, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BJos%C3%A9%2BBorrego%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Borrego%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BJos%25C3%25A9%2BBorrego%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">João Paulo Gomes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Infectious Diseases, National Institute of Health, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jo%C3%A3o%2BPaulo%2BGomes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gomes%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJo%25C3%25A3o%2BPaulo%2BGomes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.paulo.gomes@insa.min-saude.pt" class="" data-icon-position="" data-hide-link-title="0">j.paulo.gomes@insa.min-saude.pt</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/7/761" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459291" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459291/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/7/761.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459291" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459291/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/7/761.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459291" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459291/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/7/761.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459291" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459291/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/7/761.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459291" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459291/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">The knowledge of the frequency and relative weight of mutation and recombination events in evolution is essential for understanding how microorganisms reach fitted phenotypes. Traditionally, these evolutionary parameters have been inferred by using data from multilocus sequence typing (MLST), which is known to have yielded conflicting results. In the near future, these estimations will certainly be performed by computational analyses of full-genome sequences. However, it is not known whether this approach will yield accurate results as bacterial genomes exhibit heterogeneous representation of loci categories, and it is not clear how loci nature impacts such estimations. Therefore, we assessed how mutation and recombination inferences are shaped by loci with different genetic features, using the bacterium <em>Chlamydia trachomatis</em> as the study model. We found that loci assigning a high number of alleles and positively selected genes yielded nonconvergent estimates and incongruent phylogenies and thus are more prone to confound algorithms. Unexpectedly, for the model under evaluation, housekeeping genes and noncoding regions shaped estimations in a similar manner, which points to a nonrandom role of the latter in <em>C. trachomatis</em> evolution. Although the present results relate to a specific bacterium, we speculate that microbe-specific genomic architectures (such as coding capacity, polymorphism dispersion, and fraction of positively selected loci) may differentially buffer the effect of the confounding factors when estimating recombination and mutation rates and, thus, influence the accuracy of using full-genome sequences for such purpose. This putative bias associated with <em>in silico</em> inferences should be taken into account when discussing the results obtained by the analyses of full-genome sequences, in which the “one size fits all” approach may not be applicable.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/mutation-rate" class="hw-term hw-article-keyword hw-article-keyword-mutation-rate" rel="nofollow">mutation rate</a></li><li class="kwd"><a href="/keyword/recombination-rate" class="hw-term hw-article-keyword hw-article-keyword-recombination-rate" rel="nofollow">recombination rate</a></li><li class="kwd"><a href="/keyword/evolutionary-inference" class="hw-term hw-article-keyword hw-article-keyword-evolutionary-inference" rel="nofollow">evolutionary inference</a></li><li class="kwd"><a href="/keyword/clonalframe" class="hw-term hw-article-keyword hw-article-keyword-clonalframe" rel="nofollow">ClonalFrame</a></li></ul><p id="p-5">The ecological success of bacteria relies on their constant ability to diversify their genetic background to reach better-fitted phenotypes through selection. In this regard, point mutations and recombination events are especially relevant as they may be the basis for antigenic polymorphism, virulence dissimilarities, and differential tissue tropism (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Ochman <em>et al.</em> 2000</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Spratt <em>et al.</em> 2001</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Nunes <em>et al.</em> 2010</a>). As for mutation events, in which bacteria range from monomorphic (<em>e.g. Yersinia pestis</em>) to highly polymorphic (<em>e.g.</em> <em>Helicobacter pylori</em>) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Achtman 2008</a>), recombination is not equally important for all microorganisms. Indeed, they range from strictly clonal (lack or extremely low rates of recombination), such as <em>Mycobacterium</em> species or <em>Staphylococcus aureus</em> (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Smith <em>et al.</em> 2003</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Supply <em>et al.</em> 2003</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Vos and Didelot 2009</a>), to typical recombinants, such as <em>Helicobacter pylori</em> or <em>Neisseria gonorrheae</em> (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Falush <em>et al.</em> 2001</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Feil and Spratt 2001</a>). In the middle, there are microorganisms with a moderate recombination background that generate new genomic mosaic structures more fitted to deal with the environment, yielding new successful clones through a never-ending evolutionary process.</p><p id="p-6">The influence of allelic exchange in the evolution of bacterial pathogens has been measured by calculating the relative weight of recombination and mutation rates. Traditionally, these calculations have been performed on multilocus sequence typing (MLST) data resulting from the analysis of housekeeping genes (HK). However, the use of MLST data has yielded strikingly different results within the same species when estimations are performed with dissimilar MLST loci, strain samples, or analytical methodologies (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Didelot and Maiden 2010</a>). The rational for using this strategy relies on several arguments. On the one hand, large data sets are available for molecular typing purposes, and HKs are commonly dispersed around the chromosome, which prevents more than one gene from being affected by a single recombination event. Moreover, the use of HKs intends to avoid biased results because the accumulation of mutations may be confounded with the exchange of alleles by recombination when we employ loci that are either “highly polymorphic” or “too conserved”, multicopy or under positive selection (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Maiden 2006</a>). Nevertheless, this may not be a straightforward assumption as, except for the fixation of beneficial mutations through positive selection, the occurrence of point mutations exactly in the same genomic position simultaneously for several strains (homoplasy) likely results from recombination within the population (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Awadalla 2003</a>). Another question when employing MLST data to infer recombination is the use of a low number of HKs (usually seven), which may not accurately represent the genomic variability. Indeed, a previous study on bacteria found no justifiable reason for applying HKs when inferring intraspecies phylogenetic relationships, and it pointed out that the major concern when choosing candidate loci should rely on their genetic variability (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cooper and Feil 2006</a>). Thus, a wider approach based on using full-genome sequences has been recently applied, as it is expected that biasing effects from “inconvenient” loci are diluted. However, there is a multiplicity of bacterial species in which genomes have a highly heterogeneous representation of loci with different traits, such as polymorphism degree, size of intergenic regions, and selective pressures. Thus, it should be assessed how loci nature shapes the estimation parameters for understanding microbial evolution.</p><p id="p-7">One microorganism that may constitute a good model for evaluating the bias associated with the calculation of recombination and mutation rates through the analysis of different types of loci is the obligate intracellular human pathogen <em>Chlamydia trachomatis</em> due to its singular genomic features. Indeed, the core and the pan genomes of the 15 serological variants (serovars) of this pathogen are nearly identical, indicating that horizontal gene transfer is not relevant in <em>C. trachomatis</em> evolution. Moreover, the genome similarity among serovars is about 99%, in which major polymorphism is provided by few highly variable loci dispersed throughout the chromosome (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Thomson <em>et al.</em> 2008</a>), with evidence of positive selection for some of them (V. Borges, A. Nunes, R. Ferreira, M. J. Borrego, and J. P. Gomes, unpublished data; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Joseph <em>et al.</em> 2011</a>). Also, <em>C. trachomatis</em> is under the final stages of the evolutionary process of genome reduction (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Zomorodipour and Andersson 1999</a>), containing few nonessential genes and pseudogenes. Therefore, intergenic regions (IGR) likely contain regulatory domains of essential genes, which make IGRs putative targets of selection. In fact, it has been shown that several IGRs exhibit the same phylogenetic signal as neighboring genes (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Nunes <em>et al.</em> 2008</a>). Finally, although mutation events likely constitute the <em>C. trachomatis</em> major evolutionary driving force (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Nunes <em>et al.</em> 2008</a>), phenomena of genome-dispersed recombination have been recently described, seemingly related to tissue tropism and antigenic variability (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Millman <em>et al.</em> 2001</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gomes <em>et al.</em> 2007</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jeffrey <em>et al.</em> 2010</a>). Accordingly, we applied the widely used robust bioinformatic platform ClonalFrame (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Didelot and Falush 2007</a>) to several data sets encompassing loci that may differently impact the estimation of recombination and mutation rates, namely, (i) HKs from a recently developed MLST scheme (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dean <em>et al.</em> 2009</a>); (ii) positively selected genes (PSG); (iii) five groups of loci strictly ranked by their number of alleles; and (iv) intergenic regions. The results from these data sets were compared with data generated through a wide genomic approach. The present study gets insights on the bias introduced when loci with different genetic features are used to estimate recombination and mutation rates. Our approach differs from previous evaluations (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Cooper and Feil 2006</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kennemann <em>et al.</em> 2011</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Pérez-Losada <em>et al</em>. 2006</a>) as we have assessed the individual weight of each group of loci. We believe our results may help to elucidate how the evolutionary parameters are shaped, which will certainly be essential for the comprehension and validation of the data generated through the computational analyses of full-genome sequences.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Chlamydial culture</h3><p id="p-8">By the time this work was performed, only four (A/Har13, B/Jali20, D/UW3, and L2/434) out of the 15 <em>C. trachomatis</em> prototype strains (representing the 15 existing serovars) had been fully sequenced (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Stephens <em>et al.</em> 1998</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Carlson <em>et al.</em> 2005</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Thomson <em>et al.</em> 2008</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Seth-Smith <em>et al.</em> 2009</a>). To obtain sequences for <em>in silico</em> analysis, we propagated prototype strains from the remaining serovars (Ba/Apache-2, C/TW3, E/Bour, F/IC-Cal3, G/UW57, H/UW43, I/UW12, J/UW36, K/UW31, L1/440, and L3/404). Our strategy relied on using the 15 prototype strains representing all serovars because tropism differences are well defined at the serovar level, and recent phylogenetic analysis showed that the chosen strains are likely representative of the major genetic variability within the species (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Harris <em>et al.</em> 2012</a>). Indeed, it is known that differences between same-serovar strains may be as low as 20 single nucleotide polymorphisms (SNP) (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Clarke 2011</a>). Cell culture was performed through standard techniques as previously described (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Borges <em>et al.</em> 2010</a>). Briefly, T<sub>25</sub> cm<sup>2</sup> flasks of confluent HeLa 229 cell monolayers were independently inoculated with each strain, and cultures were allowed to grow at 37°, 5% CO<sub>2</sub> for about 48 hours. After bacterial growth, infected cells were harvested by scraping, sonicating, and centrifuging, and the obtained bacterial pellet was subjected to DNA extraction by using the QIAamp DNA Mini Kit (Qiagen) according to manufacturer’s instructions, and then stored at −80° until use. We then amplified and sequenced the selected genomic regions (see below) for the propagated serovars. PCR primers are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002923/-/DC1/002923SI.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002923/-/DC1/TableS1.pdf">Table S1</a>. Sequencing was performed as previously described (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gomes <em>et al.</em> 2006</a>).</p></div><div id="sec-3" class="subsection"><h3>Loci selection and grouping strategies</h3><p id="p-9">Considering the high genomic similarity among the <em>C. trachomatis</em> serovars (about 99%) (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Thomson <em>et al.</em> 2008</a>), we used comparative genomics over the four fully sequenced serovars to select informative genomic regions for inferring evolutionary parameters. We were able to select a set of 136 chromosome-scattered and functionally diverse genomic regions (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002923/-/DC1/TableS2.pdf">Table S2</a>), which include 56 IGRs and 80 genes. The selected genomic regions are highly representative of the <em>C. trachomatis</em> serovar variability as they comprise about 55% of the total SNPs in just one tenth of the chromosomal length (<em>P</em> &lt; 10<sup>−7</sup>) (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002923/-/DC1/TableS3.pdf">Table S3</a>). These regions were then differently grouped according to specific characteristics. First, for each serovar, we created a group encompassing all 136 regions by compiling their sequences while maintaining the relative order of loci in the <em>C. trachomatis</em> chromosome. Throughout the text, the strategy using this first data set will be referred to as the wide genomic approach. The second data set, termed HK-MLST, is constituted by the seven HKs that compose a MLST system (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Dean <em>et al.</em> 2009</a>). Subsequently, we created five additional data sets by dividing the 80 selected genes according to the number of alleles that each gene defines among the 15 <em>C. trachomatis</em> serovars: 1 to 5 (17 genes), 6 and 7 (17 genes), 8 and 9 (18 genes), 10 and 11 (15 genes), and 12 to 15 alleles (13 genes) (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002923/-/DC1/TableS2.pdf">Table S2</a>). Finally, we intended to evaluate the impact of using PSGs and IGRs, which are loci categories commonly not recommended when performing this type of analysis, although their potential confounding effects lack experimental support. Thus, we created two data sets composed of 11 PSGs and 56 IGRs, respectively. The use of the IGR data set also relies on recent evidence indicating that noncoding regions may also be affected by selection (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Andolfatto 2005</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bush and Lahn 2005</a>) and recombination (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Gomes <em>et al.</em> 2007</a>), which suggests that there is no apparent reason to completely rule out their use for evolutionary inferences. All studied loci are represented in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/7/761/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Chromosomal mapping of studied loci. The two outer lanes represent the DNA strands of the C. trachomatis chromosome of D/UW3 strain (GenBank accession number NC_000117), where the 80 genes (from the total 136 genomic regions evaluated) are shown in black. Each data set is represented by inner circles: HK-MLST (light blue), alleles 1 to 5 (green), alleles 6 and 7 (pink), alleles 8 and 9 (dark blue), alleles 10 and 11 (red), alleles 12 to 15 (purple), PSG (orange) and IGR (gray). The central circle shows the G/C skew plot. The precise identification of the loci is shown in Table S2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-165708165" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Chromosomal mapping of studied loci. The two outer lanes represent the DNA strands of the &lt;em&gt;C. trachomatis&lt;/em&gt; chromosome of D/UW3 strain (GenBank accession number NC_000117), where the 80 genes (from the total 136 genomic regions evaluated) are shown in black. Each data set is represented by inner circles: HK-MLST (light blue), alleles 1 to 5 (green), alleles 6 and 7 (pink), alleles 8 and 9 (dark blue), alleles 10 and 11 (red), alleles 12 to 15 (purple), PSG (orange) and IGR (gray). The central circle shows the G/C skew plot. The precise identification of the loci is shown in Table S2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/7/761/F1.medium.gif" width="440" height="385"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/2/7/761/F1.medium.gif" width="440" height="385"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/7/761/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/7/761/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461602" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-10" class="first-child">Chromosomal mapping of studied loci. The two outer lanes represent the DNA strands of the <em>C. trachomatis</em> chromosome of D/UW3 strain (GenBank accession number <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=NC_000117&amp;atom=%2Fggg%2F2%2F7%2F761.atom">NC_000117</a>), where the 80 genes (from the total 136 genomic regions evaluated) are shown in black. Each data set is represented by inner circles: HK-MLST (light blue), alleles 1 to 5 (green), alleles 6 and 7 (pink), alleles 8 and 9 (dark blue), alleles 10 and 11 (red), alleles 12 to 15 (purple), PSG (orange) and IGR (gray). The central circle shows the G/C skew plot. The precise identification of the loci is shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002923/-/DC1/TableS2.pdf">Table S2</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>progressiveMauve alignments</h3><p id="p-11">Mauve software (<a href="http://asap.ahabs.wisc.edu/mauve/">http://asap.ahabs.wisc.edu/mauve/</a>) allows the construction of multiple genome alignments for the identification of conserved regions, SNPs, indel events, inversions, and other rearrangements (and their breakpoints location) across the aligned genomes (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Darling <em>et al.</em> 2004</a>). We aligned the sequences of the 15 prototype strains of each data set through the progressiveMauve algorithm (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Darling <em>et al.</em> 2010</a>) of the Mauve software v2.3.1. As the sequences length of different data sets were below 1 Mbp, we used a conservative seed weight value (match seed weight = 11) to improve the alignment by reducing noisy matching. The resulting alignments were manually confirmed, and the output files were subsequently used in ClonalFrame software. Although Mauve is particularly useful for aligning full-genome sequences, we used this application as it generates reliable alignments in a compatible format for ClonalFrame.</p></div><div id="sec-5" class="subsection"><h3>ClonalFrame analysis</h3><p id="p-12">ClonalFrame (<a href="http://www.xavierdidelot.xtreemhost.com/clonalframe.htm">http://www.xavierdidelot.xtreemhost.com/clonalframe.htm</a>) is a widely applied software for inferring the bacterial evolutionary parameters and events underlying DNA sequence variation either from full genomes or from independent regions (such as MLST data sets). The computational cost of the analysis is greatly reduced when the inference is applied to unlinked regions rather than to full genomes, by reconstructing the clonal genealogy and further analyzing each region separately. This is a viable strategy as unlinked regions of the genome are assumed approximately independent given the clonal genealogy of a sample. The ClonalFrame inference is performed in a Bayesian framework, assuming a standard neutral coalescent model (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Didelot and Falush 2007</a>).</p><p id="p-13">In this study, the ClonalFrame software v1.2 was used for estimating mutation and recombination rates of dissimilar data sets to evaluate the impact of loci nature on these estimations. Considering the aim of the present study, the ClonalFrame options were selected to (i) estimate the mutation rate (θ), the rate of new polymorphism introduced by recombination (ν), the average tract length of a recombination event (δ), and the recombination rate (R) during each run; (ii) construct a uniformly chosen coalescent tree; (iii) assume a constant population size model; (iv) generate a random seed value for each independent run; and (v) perform the branch swapping attempts in at least half of the time of each iteration. For each data set, two independent ClonalFrame runs were performed. When alignment artifacts hampered the correct function of the software, we manually removed the gap regions while maintaining the genetic variability among <em>C. trachomatis</em> serovars, and both new Mauve alignments and ClonalFrame runs were performed. All simulations were carried out using a Linux server.</p><p id="p-14">As different numbers of iterations may yield deviating results, we conducted an analysis of the ClonalFrame reproducibility by performing two independent runs of the wide genomic data set, using a wide range of iterations (30,000, 100,000, 300,000, 500,000, and 1,000,000). For all runs, the first half of the iterations was discarded as burn-ins, and parameters were sampled every 100 iterations during the second half. The optimal number of iterations determined was applied for the subsequent analyses.</p><p id="p-15">We also assessed the convergence of the estimated parameters (θ, R, δ, and ν) from independent runs on the same data set and with the same options by applying the method of <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gelman and Rubin (1992)</a> implemented in the Graphical User Interface of the ClonalFrame software. We assumed replicate runs to be convergent only when the calculated test statistic was adequate (<em>i.e.</em> below 1.1) for all parameters. Additionally, we performed a fine-tune analysis using the ClonalFrame phylogenetic tree comparison tool, which allows the visualization of the level of confidence (based on a color scale) in each node of the consensus tree of a first run according to the output data of a second run. Each node is given a color code according to the level of confidence; white and black indicate no confidence or total confidence, respectively. On this basis, we attributed a score to each node [ranging from zero (white nodes) to three (black nodes)] (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002923/-/DC1/FigureS1.pdf">Figure S1</a>) to achieve a numerical comparison between the runs of different data sets. The sum of the scores of all nodes of each tree was then divided by the respective number of nodes to calculate an average concordance score. Finally, we evaluated the confidence on the estimates of r/m (measure of the weight of recombination on diversification relative to mutation) and ρ/θ (measure of the frequency of occurrence of recombination relative to mutation events) obtained for each data set.</p></div><div id="sec-6" class="subsection"><h3>Nucleotide sequence accession numbers</h3><p id="p-16">The sequences of all <em>C. trachomatis</em> loci determined in this study were submitted to GenBank under the accession numbers <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=JQ066324&amp;atom=%2Fggg%2F2%2F7%2F761.atom">JQ066324</a>–<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=JQ066356&amp;atom=%2Fggg%2F2%2F7%2F761.atom">JQ066356</a> and <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=JQ066367&amp;atom=%2Fggg%2F2%2F7%2F761.atom">JQ066367</a>–<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=JQ066722&amp;atom=%2Fggg%2F2%2F7%2F761.atom">JQ066722</a>.</p></div></div><div class="section results-discussion" id="sec-7"><h2 class="">Results and Discussion</h2><p id="p-17">The analysis of the evolutionary history of bacteria relies on deciphering genetic differences that arose from several mechanisms, of which point mutations and recombination events are among the most relevant driving forces. The knowledge of the frequency and the relative weight of these two mechanisms is crucial for understanding the biology and the genealogy of microorganisms. This is generally achieved by calculating the ratio ρ/θ, which determines the relative frequency of occurrence of recombination and mutation events, and the ratio r/m, which measures the relative impact of recombination and mutation in genetic diversification. In fact, the estimation of these basic population parameters for microbial pathogens has proved useful, for instance, in explaining the dynamics of drug resistance and pathogenicity and may indicate which epidemiological process should be targeted for disease control (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Conway <em>et al.</em> 2000</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Awadalla 2003</a>). Nevertheless, identifying and determining the exact extent of recombination events is not a simple and straightforward procedure, as there is no ideal methodology for establishing relationships for all bacteria, from strictly clonal to highly recombining microorganisms (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Stumpf and McVean 2003</a>). <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Didelot and Falush (2007)</a> developed a robust computational platform, ClonalFrame, which has yielded valuable results in the inference of both the population structure and the role of the recombination process in several microorganisms, such as <em>Helicobacter pylori</em> (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Kennemann <em>et al.</em> 2011</a>), <em>Listeria monocytogenes</em> (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Den Bakker <em>et al.</em> 2008</a>), and <em>Salmonella enterica</em> (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Didelot <em>et al.</em> 2011</a>). Although most inferences have been generated by using MLST data, it is expected that the analysis of full-genome sequences will be the most applied strategy in the near future. However, loci of different natures are heterogeneously represented in bacterial genomes, and it is not known if they differently impact evolutionary inferences. In the present study, we evaluated how loci nature shapes ρ/θ and r/m estimates, and we used the generated data to speculate about the validity of using full-genome sequences as the approach to estimate such parameters.</p><div id="sec-8" class="subsection"><h3>Wide genomic approach</h3><p id="p-18">We compiled loci sequences for all 15 existing serovars, encompassing about 55% of all chromosome SNPs (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002923/-/DC1/TableS3.pdf">Table S3</a>), which is expected to better represent the <em>C. trachomatis</em> intraspecies genetic variability. This wide genomic data set was preliminarily used for the assessment of the accuracy of the ClonalFrame analysis by evaluating whether different numbers of iterations (<em>i.e.</em> different durations of the simulation period) yield variable results. In fact, the optimization of the number of iterations is a critical step when performing ClonalFrame analysis. The software was run with 30,000, 100,000, 300,000, 500,000, and 1,000,000 iterations for evaluating their impact in both r/m and ρ/θ ratio estimations. We found that the highest dispersion of the estimates of both parameters was obtained for the runs using 30,000 and 100,000 iterations, which noticeably affected the mean values, revealing that for a low number of iterations, small variations may markedly bias the estimation of the evolutionary parameters (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). By increasing the number of iterations, there was a tendency toward the stability of the results, as similar values were detected when using 500,000 and 1,000,000 iterations. These runs were also the most reproducible and reliable; thus, all subsequent analyses were run by using 1,000,000 iterations to decrease the putative bias strictly associated with simulation duration. We believe that a preliminary step of optimization is critical and mandatory, despite its large computational cost (&gt;50% of the 972 CPU hours dispended in all performed simulations).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/7/761/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Accuracy assessment of r/m and &#x3C1;/&#x3B8; estimations by varying the number of iterations. The figure illustrates the impact of the number of iterations on the estimations of the ratios r/m (A) and &#x3C1;/&#x3B8; (B) inferred from the wide genomic data set. The graphs present the values and respective 95% confidence intervals of the two independent runs performed with the same number of iterations. The stability (graph plateau), reproducibility (the proximity of the mean estimates from replicate runs), and high levels of confidence (narrower error bars) of both r/m and &#x3C1;/&#x3B8; values were reached only for runs using 500,000 and 1,000,000 iterations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-165708165" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Accuracy assessment of r/m and &#x3C1;/&#x3B8; estimations by varying the number of iterations. The figure illustrates the impact of the number of iterations on the estimations of the ratios r/m (A) and &#x3C1;/&#x3B8; (B) inferred from the wide genomic data set. The graphs present the values and respective 95% confidence intervals of the two independent runs performed with the same number of iterations. The stability (graph plateau), reproducibility (the proximity of the mean estimates from replicate runs), and high levels of confidence (narrower error bars) of both r/m and &#x3C1;/&#x3B8; values were reached only for runs using 500,000 and 1,000,000 iterations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/7/761/F2.medium.gif" width="375" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/2/7/761/F2.medium.gif" width="375" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/7/761/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/7/761/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461662" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-19" class="first-child">Accuracy assessment of r/m and ρ/θ estimations by varying the number of iterations. The figure illustrates the impact of the number of iterations on the estimations of the ratios r/m (A) and ρ/θ (B) inferred from the wide genomic data set. The graphs present the values and respective 95% confidence intervals of the two independent runs performed with the same number of iterations. The stability (graph plateau), reproducibility (the proximity of the mean estimates from replicate runs), and high levels of confidence (narrower error bars) of both r/m and ρ/θ values were reached only for runs using 500,000 and 1,000,000 iterations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">Another critical stage when estimating r/m and ρ/θ relies on ensuring that independent runs yield convergent estimates for all parameters (θ, R, δ, and ν) and thus sustain similar results. For the wide genomic data set, we observed a convergence scenario for all estimated parameters by using the Gelman-Rubin test implemented in the software (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, <a href="http://g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/TableS4.xls">Table S4</a>). As a fine-tune evaluation of convergence, we also used the phylogenetic tree comparison tool, which assesses the degree of concordance between trees from replicate runs (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, <a href="http://g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/TableS4.xls">Table S4</a>). It is worth noting that the inferred tree for the wide genomic data set had total confidence in all nodes (average concordance score = 3), which, in addition to the accuracy (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>) and convergence assessment steps, supports that the ratios r/m and ρ/θ were correctly inferred through the analysis of this data set. The mean estimates of r/m and ρ/θ ratios were 0.21 and 0.01, respectively (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, <a href="http://g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/TableS4.xls">Table S4</a>), which seem plausible concerning the unique biology of this bacterium. The low ρ/θ value was expected due to the obligate intracellular life style of <em>C. trachomatis</em>. Thus, recombination requires a host-cell coinfection by distinct strains [which is expected to occur at a frequency of 1% (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Clarke 2011</a>)] followed by the fusion of the inclusion vacuoles where this pathogen replicates. With respect to the low r/m value, the high genomic similarity degree of different serovars (about 99%) implies that, except for well-described situations (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Millman <em>et al.</em> 2001</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gomes <em>et al.</em> 2004</a>, <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">2006</a>, <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">2007</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Jeffrey <em>et al.</em> 2010</a>), a recombinant fragment introduces little diversity in the recipient microorganism. Our estimates using 15 prototype strains are similar to those obtained by <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Joseph <em>et al.</em> (2011)</a> based on four prototype and eight clinical strains (r/m = 0.71 and ρ/θ = 0.07), in which the minor differences may be due to the dissimilar sample sets. Indeed, both results place <em>C. trachomatis</em> in the same position (among organisms with low recombination rates) of a r/m “scale” (from 0.02 to 63.6) presented in a previous study that focused on a broad set of bacteria and archaea (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Vos and Didelot 2009</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/7/761/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Convergence assessment of the parameters &#x3B8;, &#x3BD;, &#x3B4;, and R. For each data set, the graph shows the convergence values from two independent simulations for the estimated parameters &#x3B8;, &#x3BD;, &#x3B4;, and R. The shaded region of the graph (amplified on the right) indicates the satisfactory range of values (below 1.1) of the test statistic for all parameters according to the Gelman-Rubin test. For the data sets PSG (orange), &#x201C;8 and 9 alleles&#x201D; (dark blue), &#x201C;10 and 11 alleles&#x201D; (red), and &#x201C;12 to 15 alleles&#x201D; (purple), convergence was not observed for at least one parameter." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-165708165" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Convergence assessment of the parameters &#x3B8;, &#x3BD;, &#x3B4;, and R. For each data set, the graph shows the convergence values from two independent simulations for the estimated parameters &#x3B8;, &#x3BD;, &#x3B4;, and R. The shaded region of the graph (amplified on the right) indicates the satisfactory range of values (below 1.1) of the test statistic for all parameters according to the Gelman-Rubin test. For the data sets PSG (orange), &#x201C;8 and 9 alleles&#x201D; (dark blue), &#x201C;10 and 11 alleles&#x201D; (red), and &#x201C;12 to 15 alleles&#x201D; (purple), convergence was not observed for at least one parameter.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/7/761/F3.medium.gif" width="440" height="305"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/2/7/761/F3.medium.gif" width="440" height="305"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/7/761/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/7/761/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461523" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-21" class="first-child">Convergence assessment of the parameters θ, ν, δ, and R. For each data set, the graph shows the convergence values from two independent simulations for the estimated parameters θ, ν, δ, and R. The shaded region of the graph (amplified on the right) indicates the satisfactory range of values (below 1.1) of the test statistic for all parameters according to the Gelman-Rubin test. For the data sets PSG (orange), “8 and 9 alleles” (dark blue), “10 and 11 alleles” (red), and “12 to 15 alleles” (purple), convergence was not observed for at least one parameter.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/7/761/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Concordance score between phylogenetic trees. The chart presents the average concordance scores between trees of replicate runs calculated for each data set. More external values correspond to higher concordance between trees, and the outer line represents the maximum average score (score = 3). Values were obtained by using the tree comparison tool of the ClonalFrame, which ranks each node of the first consensus tree according to the level of confidence found between the respective nodes of both trees from replicate runs. The color-based qualitative representation of this tool (see Figure S1) was converted into a quantitative approach as described in Materials and Methods to permit the concordance evaluation at the whole-tree level. Only the wide genomic data set reached the maximum average concordance score." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-165708165" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Concordance score between phylogenetic trees. The chart presents the average concordance scores between trees of replicate runs calculated for each data set. More external values correspond to higher concordance between trees, and the outer line represents the maximum average score (score = 3). Values were obtained by using the tree comparison tool of the ClonalFrame, which ranks each node of the first consensus tree according to the level of confidence found between the respective nodes of both trees from replicate runs. The color-based qualitative representation of this tool (see Figure S1) was converted into a quantitative approach as described in &lt;em&gt;Materials and Methods&lt;/em&gt; to permit the concordance evaluation at the whole-tree level. Only the wide genomic data set reached the maximum average concordance score.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/7/761/F4.medium.gif" width="440" height="343"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/2/7/761/F4.medium.gif" width="440" height="343"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/7/761/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/7/761/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461626" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-22" class="first-child">Concordance score between phylogenetic trees. The chart presents the average concordance scores between trees of replicate runs calculated for each data set. More external values correspond to higher concordance between trees, and the outer line represents the maximum average score (score = 3). Values were obtained by using the tree comparison tool of the ClonalFrame, which ranks each node of the first consensus tree according to the level of confidence found between the respective nodes of both trees from replicate runs. The color-based qualitative representation of this tool (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002923/-/DC1/FigureS1.pdf">Figure S1</a>) was converted into a quantitative approach as described in <em>Materials and Methods</em> to permit the concordance evaluation at the whole-tree level. Only the wide genomic data set reached the maximum average concordance score.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/7/761/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimates of r/m and &#x3C1;/&#x3B8;. The graphs show the estimates of r/m (A) and &#x3C1;/&#x3B8; (B) ratios calculated by the ClonalFrame software. For each data set, the results (mean and respective 95% CIs) of the two independent runs performed with 1,000,000 iterations are shown. The data sets that yielded nonconvergent runs assessed by the Gelman-Rubin test (see Figure 3) are shaded in gray." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-165708165" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimates of r/m and &#x3C1;/&#x3B8;. The graphs show the estimates of r/m (A) and &#x3C1;/&#x3B8; (B) ratios calculated by the ClonalFrame software. For each data set, the results (mean and respective 95% CIs) of the two independent runs performed with 1,000,000 iterations are shown. The data sets that yielded nonconvergent runs assessed by the Gelman-Rubin test (see Figure 3) are shaded in gray.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/7/761/F5.medium.gif" width="267" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/2/7/761/F5.medium.gif" width="267" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/7/761/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/7/761/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461545" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-23" class="first-child">Estimates of r/m and ρ/θ. The graphs show the estimates of r/m (A) and ρ/θ (B) ratios calculated by the ClonalFrame software. For each data set, the results (mean and respective 95% CIs) of the two independent runs performed with 1,000,000 iterations are shown. The data sets that yielded nonconvergent runs assessed by the Gelman-Rubin test (see <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>) are shaded in gray.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>HK-MLST</h3><p id="p-24">Although the MLST data has been widely used for estimating recombination rates of several bacteria, nonconsensual results have been published (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Whittam 1995</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Feil <em>et al.</em> 1999</a>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">2001</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">2003</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Meats <em>et al.</em> 2003</a>), and they may be strikingly conflicting, as illustrated for <em>Bacillus cereus</em> in which different studies reported recombination rates differing up to two orders of magnitude (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hanage <em>et al.</em> 2006</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Pérez-Losada <em>et al.</em> 2006</a>). For <em>C. trachomatis</em>, a previous study determined a r/m mean estimate of 0.3 based on MLST data (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Vos and Didelot 2009</a>), which is in agreement with our estimation using the wide genomic approach, although the authors reported wide 95% CIs (0.0–1.8). Therefore, we decided to test a more recent MLST system (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Dean <em>et al.</em> 2009</a>) for comparison purposes. We obtained r/m mean values of 1.09 and 0.91, and ρ/θ mean values of 0.12 and 0.14 (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>, <a href="http://g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/TableS4.xls">Table S4</a>) from convergent and reproducible replicate runs according to both the Gelman-Rubin test (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>, <a href="http://g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/TableS4.xls">Table S4</a>) and the phylogenetic tree comparison (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>, <a href="http://g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/TableS4.xls">Table S4</a>), despite the wide CIs that hamper precise estimations (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>, <a href="http://g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/TableS4.xls">Table S4</a>). Three major issues may underlie the dissimilarity between MLST-based analyses: analytical methodology, strain sampling, and loci selection. As these two analyses using MLST schemes were performed based on ClonalFrame and employed the same set of serovars, we speculate that the loci nature is the major factor influencing estimations. Therefore, MLST data should be applied with prudence when performing this type of evolutionary inference (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Achtman 2008</a>), as only a residual proportion of the genome is analyzed [usually 6 to 10 loci of approximately 400 to 600 bp in length (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Maiden 2006</a>)], which implies that the whole genetic diversity may not be guaranteed (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Didelot and Maiden 2010</a>). This is especially relevant in monomorphic organisms, in which the maximum level of variability is extremely low (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Achtman 2008</a>). Nevertheless, the relevance of the application of MLST systems for the characterization of bacterial isolates at the molecular level remains unquestionable.</p></div><div id="sec-10" class="subsection"><h3>Allelic profile</h3><p id="p-25">MLST systems usually employ genes that assign a low number of alleles. Therefore, we evaluated the impact of increasing the number of alleles per locus on the estimation of mutation and recombination rates, as the level of polymorphism could shape the results differently. Independent runs were not convergent with the three data sets involving loci that define the highest number of alleles (8 and 9, 10 and 11, and 12 to 15) (<em>i.e.</em> Gelman-Rubin statistic above 1.1 for at least one parameter) (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>, <a href="http://g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/TableS4.xls">Table S4</a>), and thus the parameters are poorly estimated by the software, resulting in inaccurate inferences of r/m and ρ/θ ratios (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>, <a href="http://g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/TableS4.xls">Table S4</a>). For the two groups of genes assigning a low number of alleles (1 to 5, and 6 and 7), the replicate simulations were convergent and reproducible, but they yielded a high dispersion of both ratios estimates. Moreover, these results contrasted with our estimations using the wide genomic data set and pointed to an implausible scenario of an excessive weight of recombination on genetic diversity of <em>C. trachomatis</em> (r/m mean ratios higher than 4 for the two groups) (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>, <a href="http://g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/TableS4.xls">Table S4</a>). Globally, we found that the level of polymorphism definitely affects the estimations of r/m and ρ/θ at both heterogeneity of results and confidence level. In particular, loci presenting high mutation rates are more prone to confound the estimations, which makes sense considering that an excessive polymorphism is expected to mask the haplotype structures that have evolved over time, making it difficult to analyze the presence or absence of recombination (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Awadalla 2003</a>).</p></div><div id="sec-11" class="subsection"><h3>Positively selected genes</h3><p id="p-26">The detection of genes under positive selection has been of great importance for clarifying the evolutionary history of bacteria, as they encrypt adaptive signatures that may underlie phenotypic differences, such as those related to pathogenicity (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Petersen <em>et al.</em> 2007</a>; V. Borges, A. Nunes, R. Ferreira, M. J. Borrego, and J. P. Gomes, unpublished data). However, it has been assumed that PSGs should not be used to infer recombination rates, in spite of the fact that their unsuitability has not been validated experimentally. The rational for their exclusion is that PSGs likely present an unusual number of changes, and the fixation of mutations due to selection could be confounded with their acquisition through a transferred recombining fragment (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Maiden 2006</a>). In fact, recombining fragments may bring together beneficial mutations that allow a faster increase in fitness in the presence of major environmental changes instead of solely accumulating point mutations through positive selection (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Vos 2009</a>). It is also known that recombination is increased in the proximity of positively selected regions (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Vos 2009</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Petersen <em>et al.</em> 2007</a>), as demonstrated, for instance, for the genus <em>Streptococcus</em> (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lefébure and Stanhope 2007</a>). In the present study, we tested a data set composed exclusively of genes putatively under positive selection (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Joseph <em>et al.</em> 2011</a>; V. Borges, A. Nunes, R. Ferreira, M. J. Borrego, and J. P. Gomes, unpublished data). The evaluation of accuracy revealed lack of convergence for all parameters (values highly above the acceptable cut-off) (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>, <a href="http://g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/TableS4.xls">Table S4</a>), and the PSG data set was the bottom-ranked group in analysis of the concordance between trees from independent runs (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>, <a href="http://g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/TableS4.xls">Table S4</a>). Consequently, we found that this data set presented unreliable (wide 95% CIs) and the least reproducible results, which is reflected by the discrepant mean estimate values between runs differing up to two orders of magnitude (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5</a>, <a href="http://g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/TableS4.xls">Table S4</a>). These results suggest that, for genomes subjected to strong selective pressures, estimations of recombination rates may be biased by the presence of a high fraction of PSGs. Nevertheless, because it is known that PSGs are also targets of recombination (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Vos 2009</a>), we believe that, for the majority of the bacterial genomic contexts, the use of wide genome approaches will likely buffer the confounding effects of PSGs on estimations. In fact, in the present study, the inclusion of PSGs in the wide genomic approach did not hamper the accurate inferences of the evolutionary parameters.</p></div><div id="sec-12" class="subsection"><h3>Intergenic regions</h3><p id="p-27">The IGRs have been excluded for inferring evolutionary histories of organisms, although they are known to carry promoter regions, ribosome binding sites, as well as transcription factor and regulator binding regions, which play critical roles in regulation of gene transcription. Recent studies demonstrated that noncoding regions are subject to significant selective constraints (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Andolfatto 2005</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bush and Lahn 2005</a>). For <em>C. trachomatis</em>, we previously detected recombination hotspots involving IGRs (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Gomes <em>et al.</em> 2007</a>), and we observed phylogenies of IGRs revealing the clustering of strains with the same disease outcomes (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Nunes <em>et al.</em> 2008</a>), which suggest selection or hitchhiking events (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kaplan <em>et al.</em> 1989</a>) involving these regions. This evidence, together with the knowledge that the small genome of <em>C. trachomatis</em> likely retains only the indispensable genes (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Zomorodipour and Andersson 1999</a>), points to a relevant role of IGRs in <em>C. trachomatis</em> evolution. Thus, we estimated rates of recombination and mutation using 56 IGRs because the accumulation of mutations in these regions may not be a random process and because they are heterogeneously represented in different genomes. We obtained &gt;90% of concordance between trees, and a Gelman-Rubin test statistic below 1.1 for all parameters (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figures 3</a> and <a id="xref-fig-4-4" class="xref-fig" href="#F4">4</a>, <a href="http://g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/TableS4.xls">Table S4</a>), indicating convergence. The r/m and ρ/θ mean estimates (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5</a>, <a href="http://g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/TableS4.xls">Table S4</a>) are about 1-log above those obtained for the wide genomic data set, but they are similar to the HK-MLST data set estimates, which suggests that this large set of noncoding regions and these specific HKs shape these evolutionary parameters in a similar fashion for the model under evaluation.</p></div></div><div class="section conclusions" id="sec-13"><h2 class="">Conclusion</h2><p id="p-28">We used a specific human pathogen with well-defined genomic characteristics as a model to study bias associated with the estimation of evolutionary parameters by computational simulations. Our results show that the estimation of mutation and recombination rates in <em>C. trachomatis</em> is influenced by the characteristics of the loci used for such calculations. Although the use of full-genome sequences to infer recombination and mutation rates is suitable for most microorganisms, we anticipate that soon a greater proportion of highly polymorphic or positively selected loci can make it an inaccurate approach. Thus, the correctness of the final output will depend on the dilution effect of these confounding factors by the remaining portions of the genome with dissimilar architectures. As data from population genetics has contributed to a better understanding of the biology and pathogenicity of organisms, the clarification of the putative bias associated with <em>in silico</em> inferences is of great interest for deciphering evolutionary traits.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-29">This work was supported by a grant, ERA-PTG/0004/2010, from Fundação para a Ciência e a Tecnologia (FCT) (to J.P.G.), in the frame of ERA-NET PathoGenoMics. R.F. and V.B. are recipients of Ph.D. fellowships (SFRH/BD/68532/2010 and SFRH/BD/68527/2010, respectively) from FCT. A.N. is a recipient of a post-doctoral fellowship (SFRH/BPD/75295/2010) from FCT. We are grateful to Karol Dobrzanski for providing the Linux server.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002923/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002923/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">Sequence data from this article have been deposited with the EMBL/GenBank Data Libraries under accession nos. JQ066324–JQ066356 and JQ066367–JQ066722.</p></li><li class="fn" id="fn-3"><p id="p-30"><em>Communicating editor: K. S. McKim</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-03-09"><span class="received-label">Received </span>March 9, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-05-03"><span class="accepted-label">Accepted </span>May 3, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Ferreira <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.1" data-doi="10.1146/annurev.micro.62.081307.162832"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Achtman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Evolution, population structure, and phylogeography of genetically monomorphic bacterial pathogens</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Microbiol.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bmicrobiology%26rft.stitle%253DAnnu%2BRev%2BMicrobiol%26rft.aulast%253DAchtman%26rft.auinit1%253DM.%26rft.volume%253D62%26rft.spage%253D53%26rft.epage%253D70%26rft.atitle%253DEvolution%252C%2Bpopulation%2Bstructure%252C%2Band%2Bphylogeography%2Bof%2Bgenetically%2Bmonomorphic%2Bbacterial%2Bpathogens.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.micro.62.081307.162832%26rft_id%253Dinfo%253Apmid%252F18785837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.micro.62.081307.162832&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18785837&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259968000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.2" data-doi="10.1038/nature04107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Adaptive evolution of non-coding DNA in <em>Drosophila</em></span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">437</span>: <span class="cit-fpage">1149</span>–<span class="cit-lpage">1152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAndolfatto%26rft.auinit1%253DP.%26rft.volume%253D437%26rft.issue%253D7062%26rft.spage%253D1149%26rft.epage%253D1152%26rft.atitle%253DAdaptive%2Bevolution%2Bof%2Bnon-coding%2BDNA%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04107%26rft_id%253Dinfo%253Apmid%252F16237443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16237443&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232660500043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.3" data-doi="10.1038/nrg964"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Awadalla</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The evolutionary genomics of pathogen recombination</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">50</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DAwadalla%26rft.auinit1%253DP.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D50%26rft.epage%253D60%26rft.atitle%253DThe%2Bevolutionary%2Bgenomics%2Bof%2Bpathogen%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg964%26rft_id%253Dinfo%253Apmid%252F12509753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg964&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12509753&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180171200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.4" data-doi="10.1016/j.mimet.2010.06.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borges</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nunes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nogueira</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borrego</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Normalization strategies for real-time expression data in <em>Chlamydia trachomatis</em></span>. <abbr class="cit-jnl-abbrev">J. Microbiol. Methods</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">256</span>–<span class="cit-lpage">264</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMicrobiological%2BMethods%26rft.stitle%253DJournal%2Bof%2BMicrobiological%2BMethods%26rft.aulast%253DBorges%26rft.auinit1%253DV.%26rft.volume%253D82%26rft.issue%253D3%26rft.spage%253D256%26rft.epage%253D264%26rft.atitle%253DNormalization%2Bstrategies%2Bfor%2Breal-time%2Bexpression%2Bdata%2Bin%2BChlamydia%2Btrachomatis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mimet.2010.06.013%26rft_id%253Dinfo%253Apmid%252F20619305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mimet.2010.06.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20619305&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282113000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.5" data-doi="10.1371/journal.pcbi.0010073"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bush</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lahn</span>  <span class="cit-name-given-names">B. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Selective constraint on noncoding regions of hominid genomes</span>. <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">e73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DBush%26rft.auinit1%253DE.%2BC.%26rft.volume%253D1%26rft.issue%253D7%26rft.spage%253De73%26rft.epage%253De73%26rft.atitle%253DSelective%2Bconstraint%2Bon%2Bnoncoding%2Bregions%2Bof%2Bhominid%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.0010073%26rft_id%253Dinfo%253Apmid%252F16362073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.0010073&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16362073&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.6" data-doi="10.1128/IAI.73.10.6407-6418.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Porcella</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McClarty</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caldwell</span>  <span class="cit-name-given-names">H. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Comparative genomic analysis of <em>Chlamydia trachomatis</em> oculotropic and genitotropic strains</span>. <abbr class="cit-jnl-abbrev">Infect. Immun.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">6407</span>–<span class="cit-lpage">6418</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%2Band%2BImmunity%26rft.stitle%253DInfect.%2BImmun.%26rft.aulast%253DCarlson%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D73%26rft.issue%253D10%26rft.spage%253D6407%26rft.epage%253D6418%26rft.atitle%253DComparative%2BGenomic%2BAnalysis%2Bof%2BChlamydia%2Btrachomatis%2BOculotropic%2Band%2BGenitotropic%2BStrains%26rft_id%253Dinfo%253Adoi%252F10.1128%252FIAI.73.10.6407-6418.2005%26rft_id%253Dinfo%253Apmid%252F16177312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaWFpIjtzOjU6InJlc2lkIjtzOjEwOiI3My8xMC82NDA3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi83Lzc2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.7" data-doi="10.1111/j.1749-6632.2011.06194.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">I. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evolution of <em>Chlamydia trachomatis</em></span>. <abbr class="cit-jnl-abbrev">Ann. N. Y. Acad. Sci.</abbr> <span class="cit-vol">1230</span>: <span class="cit-fpage">E11</span>–<span class="cit-lpage">E18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bthe%2BNew%2BYork%2BAcademy%2Bof%2BSciences%26rft.stitle%253DAnn.%2BN.%2BY.%2BAcad.%2BSci.%26rft.aulast%253DClarke%26rft.auinit1%253DI.%2BN.%26rft.volume%253D1230%26rft.spage%253DE11%26rft.epage%253DE18%26rft.atitle%253DEvolution%2Bof%2BChlamydia%2Btrachomatis.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1749-6632.2011.06194.x%26rft_id%253Dinfo%253Apmid%252F22239534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1749-6632.2011.06194.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22239534&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294181100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.8" data-doi="10.1038/76272"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conway</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanabe</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roper</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mikes</span>  <span class="cit-name-given-names">Z. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A principal target of human immunity to malaria identified by molecular population genetic and immunological analyses</span>. <abbr class="cit-jnl-abbrev">Nat. Med.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">689</span>–<span class="cit-lpage">692</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bmedicine%26rft.stitle%253DNat%2BMed%26rft.aulast%253DConway%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D6%26rft.issue%253D6%26rft.spage%253D689%26rft.epage%253D692%26rft.atitle%253DA%2Bprincipal%2Btarget%2Bof%2Bhuman%2Bimmunity%2Bto%2Bmalaria%2Bidentified%2Bby%2Bmolecular%2Bpopulation%2Bgenetic%2Band%2Bimmunological%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F76272%26rft_id%253Dinfo%253Apmid%252F10835687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/76272&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10835687&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087438300041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.9" data-doi="10.1099/mic.0.28620-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feil</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The phylogeny of <em>Staphylococcus aureus</em> - which genes make the best intra-species markers?</span> <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.stitle%253DMicrobiology%26rft.aulast%253DCooper%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D152%26rft.issue%253DPt%2B5%26rft.spage%253D1297%26rft.epage%253D1305%26rft.atitle%253DThe%2Bphylogeny%2Bof%2BStaphylococcus%2Baureus%2B-%2Bwhich%2Bgenes%2Bmake%2Bthe%2Bbest%2Bintra-species%2Bmarkers%253F%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fmic.0.28620-0%26rft_id%253Dinfo%253Apmid%252F16622047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/mic.0.28620-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16622047&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237618000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.10" data-doi="10.1101/gr.2289704"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darling</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mau</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blattner</span>  <span class="cit-name-given-names">F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perna</span>  <span class="cit-name-given-names">N. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Mauve: multiple alignment of conserved genomic sequence with rearrangements</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1394</span>–<span class="cit-lpage">1403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DDarling%26rft.auinit1%253DA.%2BC.E.%26rft.volume%253D14%26rft.issue%253D7%26rft.spage%253D1394%26rft.epage%253D1403%26rft.atitle%253DMauve%253A%2BMultiple%2BAlignment%2Bof%2BConserved%2BGenomic%2BSequence%2BWith%2BRearrangements%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.2289704%26rft_id%253Dinfo%253Apmid%252F15231754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE0LzcvMTM5NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNy83NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.11" data-doi="10.1371/journal.pone.0011147"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darling</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mau</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perna</span>  <span class="cit-name-given-names">N. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">progressiveMauve: multiple genome alignment with gene gain, loss, and rearrangement</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e11147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DDarling%26rft.auinit1%253DA.%2BE.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253De11147%26rft.epage%253De11147%26rft.atitle%253DprogressiveMauve%253A%2Bmultiple%2Bgenome%2Balignment%2Bwith%2Bgene%2Bgain%252C%2Bloss%2Band%2Brearrangement.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0011147%26rft_id%253Dinfo%253Apmid%252F20593022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0011147&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20593022&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.12" data-doi="10.3201/eid1509.090272"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruno</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomes</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devignot</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting phenotype and emerging strains among <em>Chlamydia trachomatis</em> infections</span>. <abbr class="cit-jnl-abbrev">Emerg. Infect. Dis.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1385</span>–<span class="cit-lpage">1394</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DDean%26rft.auinit1%253DD.%26rft.volume%253D15%26rft.issue%253D9%26rft.spage%253D1385%26rft.epage%253D1394%26rft.atitle%253DPredicting%2Bphenotype%2Band%2Bemerging%2Bstrains%2Bamong%2BChlamydia%2Btrachomatis%2Binfections.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1509.090272%26rft_id%253Dinfo%253Apmid%252F19788805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid1509.090272&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19788805&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.13" data-doi="10.1186/1471-2148-8-277"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">den Bakker</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didelot</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fortes</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nightingale</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiedmann</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Lineage specific recombination rates and microevolution in <em>Listeria monocytogenes</em></span>. <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253Dden%2BBakker%26rft.auinit1%253DH.%2BC.%26rft.volume%253D8%26rft.spage%253D277%26rft.epage%253D277%26rft.atitle%253DLineage%2Bspecific%2Brecombination%2Brates%2Band%2Bmicroevolution%2Bin%2BListeria%2Bmonocytogenes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-8-277%26rft_id%253Dinfo%253Apmid%252F18842152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-8-277&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18842152&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.14" data-doi="10.1534/genetics.106.063305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Didelot</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falush</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Inference of bacterial microevolution using multilocus sequence data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">1251</span>–<span class="cit-lpage">1266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D175%26rft.spage%253D1251%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.063305%26rft_id%253Dinfo%253Apmid%252F17151252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.106.063305&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17151252&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245590600024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.15" data-doi="10.1016/j.tim.2010.04.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Didelot</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maiden</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Impact of recombination on bacterial evolution</span>. <abbr class="cit-jnl-abbrev">Trends Microbiol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmicrobiology%26rft.stitle%253DTrends%2BMicrobiol%26rft.aulast%253DDidelot%26rft.auinit1%253DX.%26rft.volume%253D18%26rft.issue%253D7%26rft.spage%253D315%26rft.epage%253D322%26rft.atitle%253DImpact%2Bof%2Brecombination%2Bon%2Bbacterial%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2010.04.002%26rft_id%253Dinfo%253Apmid%252F20452218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2010.04.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20452218&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280054200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.16" data-doi="10.1371/journal.pgen.1002191"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Didelot</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowden</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Street</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golubchik</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spencer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Recombination and population structure in <em>Salmonella enterica</em></span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DDidelot%26rft.auinit1%253DX.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De1002191%26rft.epage%253De1002191%26rft.atitle%253DRecombination%2Band%2Bpopulation%2Bstructure%2Bin%2BSalmonella%2Benterica.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002191%26rft_id%253Dinfo%253Apmid%252F21829375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21829375&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.17" data-doi="10.1073/pnas.251396098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falush</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kraft</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Correa</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Recombination and mutation during long-term gastric colonization by <em>Helicobacter pylori</em>: estimates of clock rates, recombination size, and minimal age</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">15056</span>–<span class="cit-lpage">15061</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.251396098%26rft_id%253Dinfo%253Apmid%252F11742075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTgvMjYvMTUwNTYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzcvNzYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.18" data-doi="10.1146/annurev.micro.55.1.561"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feil</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spratt</span>  <span class="cit-name-given-names">B. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Recombination and the population structures of bacterial pathogens</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Microbiol.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">561</span>–<span class="cit-lpage">590</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bmicrobiology%26rft.stitle%253DAnnu%2BRev%2BMicrobiol%26rft.aulast%253DFeil%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D55%26rft.issue%253D1%26rft.spage%253D561%26rft.epage%253D590%26rft.atitle%253DRECOMBINATION%2BAND%2BTHE%2BPOPULATION%2BSTRUCTURES%2BOF%2BBACTERIAL%2BPATHOGENS%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.micro.55.1.561%26rft_id%253Dinfo%253Apmid%252F11544367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.micro.55.1.561&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11544367&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171732600022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.19" data-doi="10.1093/oxfordjournals.molbev.a026061"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feil</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maiden</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Achtman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spratt</span>  <span class="cit-name-given-names">B. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The relative contributions of recombination and mutation to the divergence of clones of <em>Neisseria meningitidis</em></span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1496</span>–<span class="cit-lpage">1502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DFeil%26rft.auinit1%253DE.%26rft.volume%253D16%26rft.issue%253D11%26rft.spage%253D1496%26rft.epage%253D1502%26rft.atitle%253DThe%2Brelative%2Bcontributions%2Bof%2Brecombination%2Band%2Bmutation%2Bto%2Bthe%2Bdivergence%2Bof%2Bclones%2Bof%2BNeisseria%2Bmeningitidis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a026061%26rft_id%253Dinfo%253Apmid%252F10555280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIxNi8xMS8xNDk2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi83Lzc2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.20" data-doi="10.1073/pnas.98.1.182"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feil</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bessen</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Day</span>  <span class="cit-name-given-names">N. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Recombination within natural populations of pathogenic bacteria: short-term empirical estimates and long-term phylogenetic consequences</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">182</span>–<span class="cit-lpage">187</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DFeil%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D98%26rft.issue%253D1%26rft.spage%253D182%26rft.epage%253D187%26rft.atitle%253DRecombination%2Bwithin%2Bnatural%2Bpopulations%2Bof%2Bpathogenic%2Bbacteria%253A%2BShort-term%2Bempirical%2Bestimates%2Band%2Blong-term%2Bphylogenetic%2Bconsequences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.98.1.182%26rft_id%253Dinfo%253Apmid%252F11136255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI5OC8xLzE4MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNy83NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.21" data-doi="10.1128/JB.185.11.3307-3316.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feil</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grundmann</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Enright</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">How clonal is <em>Staphylococcus aureus</em>?</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">3307</span>–<span class="cit-lpage">3316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DFeil%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D185%26rft.issue%253D11%26rft.spage%253D3307%26rft.epage%253D3316%26rft.atitle%253DHow%2BClonal%2BIs%2BStaphylococcus%2Baureus%253F%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.185.11.3307-3316.2003%26rft_id%253Dinfo%253Apmid%252F12754228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE4NS8xMS8zMzA3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi83Lzc2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.22" data-doi="10.1214/ss/1177011136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gelman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Inference from iterative simulation using multiple sequences</span>. <abbr class="cit-jnl-abbrev">Stat. Sci.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">457</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BSci.%26rft.volume%253D7%26rft.spage%253D457%26rft_id%253Dinfo%253Adoi%252F10.1214%252Fss%252F1177011136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/ss/1177011136&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.23" data-doi="10.1128/JB.186.13.4295-4306.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gomes</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruno</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borrego</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Recombination in the genome of <em>Chlamydia trachomatis</em> involving the polymorphic membrane protein C gene relative to <em>ompA</em> and evidence for horizontal gene transfer</span>. <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">4295</span>–<span class="cit-lpage">4306</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DGomes%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D186%26rft.issue%253D13%26rft.spage%253D4295%26rft.epage%253D4306%26rft.atitle%253DRecombination%2Bin%2Bthe%2BGenome%2Bof%2BChlamydia%2Btrachomatis%2BInvolving%2Bthe%2BPolymorphic%2BMembrane%2BProtein%2BC%2BGene%2BRelative%2Bto%2BompA%2Band%2BEvidence%2Bfor%2BHorizontal%2BGene%2BTransfer%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.186.13.4295-4306.2004%26rft_id%253Dinfo%253Apmid%252F15205432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE4Ni8xMy80Mjk1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi83Lzc2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.24" data-doi="10.1128/JB.188.1.275-286.2006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gomes</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nunes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruno</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borrego</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Florindo</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Polymorphisms in the nine polymorphic membrane proteins of <em>Chlamydia trachomatis</em> across all serovars: evidence for serovar Da recombination and correlation with tissue tropism</span>. <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">275</span>–<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DGomes%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D188%26rft.issue%253D1%26rft.spage%253D275%26rft.epage%253D286%26rft.atitle%253DPolymorphisms%2Bin%2Bthe%2BNine%2BPolymorphic%2BMembrane%2BProteins%2Bof%2BChlamydia%2Btrachomatis%2Bacross%2BAll%2BSerovars%253A%2BEvidence%2Bfor%2BSerovar%2BDa%2BRecombination%2Band%2BCorrelation%2Bwith%2BTissue%2BTropism%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.188.1.275-286.2006%26rft_id%253Dinfo%253Apmid%252F16352844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTg4LzEvMjc1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi83Lzc2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.25" data-doi="10.1101/gr.5674706"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gomes</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruno</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nunes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santos</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Florindo</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Evolution of <em>Chlamydia trachomatis</em> diversity occurs by widespread interstrain recombination involving hotspots</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">50</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5674706%26rft_id%253Dinfo%253Apmid%252F17090662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjE3LzEvNTAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzcvNzYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.26" data-doi="10.1016/j.jtbi.2005.08.035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hanage</span>  <span class="cit-name-given-names">W. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spratt</span>  <span class="cit-name-given-names">B. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The impact of homologous recombination on the generation of diversity in bacteria</span>. <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr> <span class="cit-vol">239</span>: <span class="cit-fpage">210</span>–<span class="cit-lpage">219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DHanage%26rft.auinit1%253DW.%2BP.%26rft.volume%253D239%26rft.issue%253D2%26rft.spage%253D210%26rft.epage%253D219%26rft.atitle%253DThe%2Bimpact%2Bof%2Bhomologous%2Brecombination%2Bon%2Bthe%2Bgeneration%2Bof%2Bdiversity%2Bin%2Bbacteria.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jtbi.2005.08.035%26rft_id%253Dinfo%253Apmid%252F16236325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jtbi.2005.08.035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16236325&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236340700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.27" data-doi="10.1038/ng.2214"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">I. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seth-Smith</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solomon</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cutcliffe</span>  <span class="cit-name-given-names">L. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Whole-genome analysis of diverse <em>Chlamydia trachomatis</em> strains identifies phylogenetic relationships masked by current clinical typing</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">419</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHarris%26rft.auinit1%253DS.%2BR.%26rft.volume%253D44%26rft.issue%253D4%26rft.spage%253D413%26rft.epage%253D419%26rft.atitle%253DWhole-genome%2Banalysis%2Bof%2Bdiverse%2BChlamydia%2Btrachomatis%2Bstrains%2Bidentifies%2Bphylogenetic%2Brelationships%2Bmasked%2Bby%2Bcurrent%2Bclinical%2Btyping.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2214%26rft_id%253Dinfo%253Apmid%252F22406642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2214&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22406642&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.28" data-doi="10.1186/1745-6150-6-28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Joseph</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didelot</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gandhi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Read</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Interplay of recombination and selection in the genomes of <em>Chlamydia trachomatis</em></span>. <abbr class="cit-jnl-abbrev">Biol. Direct</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBiol%2BDirect%26rft.aulast%253DJoseph%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D6%26rft.spage%253D28%26rft.epage%253D28%26rft.atitle%253DInterplay%2Bof%2Brecombination%2Band%2Bselection%2Bin%2Bthe%2Bgenomes%2Bof%2BChlamydia%2Btrachomatis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1745-6150-6-28%26rft_id%253Dinfo%253Apmid%252F21615910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1745-6150-6-28&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21615910&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.29" data-doi="10.1128/IAI.01324-09"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jeffrey</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suchland</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quinn</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stamm</span>  <span class="cit-name-given-names">W. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome sequencing of recent clinical <em>Chlamydia trachomatis</em> strains identifies loci associated with tissue tropism and regions of apparent recombination</span>. <abbr class="cit-jnl-abbrev">Infect. Immun.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">2544</span>–<span class="cit-lpage">2553</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfect.%2BImmun.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FIAI.01324-09%26rft_id%253Dinfo%253Apmid%252F20308297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaWFpIjtzOjU6InJlc2lkIjtzOjk6Ijc4LzYvMjU0NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNy83NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langley</span>  <span class="cit-name-given-names">C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">The “hitchhiking effect” revisited</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">887</span>–<span class="cit-lpage">899</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D123%26rft.spage%253D887%26rft_id%253Dinfo%253Apmid%252F2612899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2612899&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989CC60200024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.31" data-doi="10.1073/pnas.1018444108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kennemann</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didelot</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aebischer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drescher</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title"><em>Helicobacter pylori</em> genome evolution during human infection</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">5033</span>–<span class="cit-lpage">5038</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1018444108%26rft_id%253Dinfo%253Apmid%252F21383187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzEyLzUwMzMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzcvNzYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.32" data-doi="10.1186/gb-2007-8-5-r71"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lefébure</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stanhope</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Evolution of the core and pan-genome of <em>Streptococcus</em>: positive selection, recombination, and genome composition</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">R71</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLefebure%26rft.auinit1%253DT.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253DR71%26rft.epage%253DR71%26rft.atitle%253DEvolution%2Bof%2Bthe%2Bcore%2Band%2Bpan-genome%2Bof%2BStreptococcus%253A%2Bpositive%2Bselection%252C%2Brecombination%252C%2Band%2Bgenome%2Bcomposition.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2007-8-5-r71%26rft_id%253Dinfo%253Apmid%252F17475002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2007-8-5-r71&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17475002&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.33" data-doi="10.1146/annurev.micro.59.030804.121325"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maiden</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Multilocus sequence typing of bacteria</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Microbiol.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">561</span>–<span class="cit-lpage">588</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bmicrobiology%26rft.stitle%253DAnnu%2BRev%2BMicrobiol%26rft.aulast%253DBarnhart%26rft.auinit1%253DM.%2BM.%26rft.volume%253D60%26rft.spage%253D561%26rft.epage%253D588%26rft.atitle%253DMultilocus%2Bsequence%2Btyping%2Bof%2Bbacteria.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.micro.59.030804.121325%26rft_id%253Dinfo%253Apmid%252F16774461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.micro.59.030804.121325&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16774461&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242032500025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.34" data-doi="10.1128/JCM.41.4.1623-1636.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meats</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feil</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stringer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cody</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Characterization of encapsulated and noncapsulated <em>Haemophilus influenzae</em> and determination of phylogenetic relationships by multilocus sequence typing</span>. <abbr class="cit-jnl-abbrev">J. Clin. Microbiol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">1623</span>–<span class="cit-lpage">1636</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BMicrobiology%26rft.stitle%253DJ.%2BClin.%2BMicrobiol.%26rft.aulast%253DMeats%26rft.auinit1%253DE.%26rft.volume%253D41%26rft.issue%253D4%26rft.spage%253D1623%26rft.epage%253D1636%26rft.atitle%253DCharacterization%2Bof%2BEncapsulated%2Band%2BNoncapsulated%2BHaemophilus%2Binfluenzae%2Band%2BDetermination%2Bof%2BPhylogenetic%2BRelationships%2Bby%2BMultilocus%2BSequence%2BTyping%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.41.4.1623-1636.2003%26rft_id%253Dinfo%253Apmid%252F12682154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjk6IjQxLzQvMTYyMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNy83NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.35" data-doi="10.1128/JB.183.20.5997-6008.2001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Millman</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tavare</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Recombination in the <em>ompA</em> gene but not the <em>omcB</em> gene of <em>Chlamydia</em> contributes to serovar-specific differences in tissue tropism, immune surveillance, and persistence of the organism</span>. <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">5997</span>–<span class="cit-lpage">6008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DMillman%26rft.auinit1%253DK.%2BL.%26rft.volume%253D183%26rft.issue%253D20%26rft.spage%253D5997%26rft.epage%253D6008%26rft.atitle%253DRecombination%2Bin%2Bthe%2BompA%2BGene%2Bbut%2BNot%2BtheomcB%2BGene%2Bof%2BChlamydia%2BContributes%2Bto%2BSerovar-Specific%2BDifferences%2Bin%2BTissue%2BTropism%252C%2BImmune%2BSurveillance%252C%2Band%2BPersistence%2Bof%2Bthe%2BOrganism%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.183.20.5997-6008.2001%26rft_id%253Dinfo%253Apmid%252F11567000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE4My8yMC81OTk3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi83Lzc2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.36" data-doi="10.1186/gb-2008-9-10-r153"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nunes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nogueira</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borrego</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomes</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title"><em>Chlamydia trachomatis</em> diversity viewed as a tissue-specific coevolutionary arms race</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">R153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DNunes%26rft.auinit1%253DA.%26rft.volume%253D9%26rft.issue%253D10%26rft.spage%253DR153%26rft.epage%253DR153%26rft.atitle%253DChlamydia%2Btrachomatis%2Bdiversity%2Bviewed%2Bas%2Ba%2Btissue-specific%2Bcoevolutionary%2Barms%2Brace.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2008-9-10-r153%26rft_id%253Dinfo%253Apmid%252F18947394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2008-9-10-r153&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18947394&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.37" data-doi="10.1371/journal.pone.0013171"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nunes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nogueira</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borrego</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomes</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Adaptive evolution of the <em>Chlamydia trachomatis</em> dominant antigen reveals distinct evolutionary scenarios for B- and T-cell epitopes: worldwide survey</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e13171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D5%26rft.spage%253De13171%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0013171%26rft_id%253Dinfo%253Apmid%252F20957150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0013171&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20957150&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.38" data-doi="10.1038/35012500"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ochman</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawrence</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groisman</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Lateral gene transfer and the nature of bacterial innovation</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">405</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">304</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DOchman%26rft.auinit1%253DH.%26rft.volume%253D405%26rft.issue%253D6784%26rft.spage%253D299%26rft.epage%253D304%26rft.atitle%253DLateral%2Bgene%2Btransfer%2Band%2Bthe%2Bnature%2Bof%2Bbacterial%2Binnovation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35012500%26rft_id%253Dinfo%253Apmid%252F10830951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35012500&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10830951&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087085700035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.39" data-doi="10.1016/j.meegid.2005.02.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Losada</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browne</span>  <span class="cit-name-given-names">E. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wirth</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viscidi</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Population genetics of microbial pathogens estimated from multilocus sequence typing (MLST) data</span>. <abbr class="cit-jnl-abbrev">Infect. Genet. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%252C%2Bgenetics%2Band%2Bevolution%2B%253A%2B%2Bjournal%2Bof%2Bmolecular%2Bepidemiology%2Band%2Bevolutionary%2Bgenetics%2Bin%2Binfectious%2Bdiseases%26rft.stitle%253DInfect%2BGenet%2BEvol%26rft.aulast%253DPerez-Losada%26rft.auinit1%253DM.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253D97%26rft.epage%253D112%26rft.atitle%253DPopulation%2Bgenetics%2Bof%2Bmicrobial%2Bpathogens%2Bestimated%2Bfrom%2Bmultilocus%2Bsequence%2Btyping%2B%2528MLST%2529%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.meegid.2005.02.003%26rft_id%253Dinfo%253Apmid%252F16503511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.meegid.2005.02.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16503511&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236245400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.40" data-doi="10.1101/gr.6254707"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petersen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bollback</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dimmic</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genes under positive selection in <em>Escherichia coli</em></span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1336</span>–<span class="cit-lpage">1343</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6254707%26rft_id%253Dinfo%253Apmid%252F17675366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE3LzkvMTMzNiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNy83NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.41" data-doi="10.1186/1471-2164-10-239"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seth-Smith</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Persson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marsh</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barron</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Co-evolution of genomes and plasmids within <em>Chlamydia trachomatis</em> and the emergence in Sweden of a new variant strain</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DSeth-Smith%26rft.auinit1%253DH.%2BM.%26rft.volume%253D10%26rft.spage%253D239%26rft.epage%253D239%26rft.atitle%253DCo-evolution%2Bof%2Bgenomes%2Band%2Bplasmids%2Bwithin%2BChlamydia%2Btrachomatis%2Band%2Bthe%2Bemergence%2Bin%2BSweden%2Bof%2Ba%2Bnew%2Bvariant%2Bstrain.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-239%26rft_id%253Dinfo%253Apmid%252F19460133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-239&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19460133&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.42" data-doi="10.1073/pnas.2036554100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dale</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inwald</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The population structure of <em>Mycobacterium bovis</em> in Great Britain: clonal expansion</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">15271</span>–<span class="cit-lpage">15275</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.2036554100%26rft_id%253Dinfo%253Apmid%252F14657373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAwLzI1LzE1MjcxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi83Lzc2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.43" data-doi="10.1016/S1369-5274(00)00257-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spratt</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanage</span>  <span class="cit-name-given-names">W. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feil</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The relative contributions of recombination and point mutation to the diversification of bacterial clones</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Microbiol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">602</span>–<span class="cit-lpage">606</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bmicrobiology%26rft.stitle%253DCurr%2BOpin%2BMicrobiol%26rft.aulast%253DSpratt%26rft.auinit1%253DB.%2BG.%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253D602%26rft.epage%253D606%26rft.atitle%253DThe%2Brelative%2Bcontributions%2Bof%2Brecombination%2Band%2Bpoint%2Bmutation%2Bto%2Bthe%2Bdiversification%2Bof%2Bbacterial%2Bclones.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1369-5274%252800%252900257-5%26rft_id%253Dinfo%253Apmid%252F11587939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1369-5274(00)00257-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11587939&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171860900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.44" data-doi="10.1126/science.282.5389.754"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lammel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marathe</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Genome sequence of an obligate intracellular pathogen of humans: <em>Chlamydia trachomatis</em></span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">754</span>–<span class="cit-lpage">759</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DStephens%26rft.auinit1%253DR.%2BS.%26rft.volume%253D282%26rft.issue%253D5389%26rft.spage%253D754%26rft.epage%253D759%26rft.atitle%253DGenome%2BSequence%2Bof%2Ban%2BObligate%2BIntracellular%2BPathogen%2Bof%2BHumans%253A%2BChlamydia%2Btrachomatis%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.282.5389.754%26rft_id%253Dinfo%253Apmid%252F9784136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODIvNTM4OS83NTQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzcvNzYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.45" data-doi="10.1038/nrg1227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stumpf</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Estimating recombination rates from population-genetic data</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">959</span>–<span class="cit-lpage">968</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DStumpf%26rft.auinit1%253DM.%2BP.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253D959%26rft.epage%253D968%26rft.atitle%253DEstimating%2Brecombination%2Brates%2Bfrom%2Bpopulation-genetic%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1227%26rft_id%253Dinfo%253Apmid%252F14631356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1227&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14631356&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186893100014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.46" data-doi="10.1046/j.1365-2958.2003.03315.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Supply</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warren</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bañuls</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lesjean</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Der Spuy</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Linkage disequilibrium between minisatellite loci supports clonal evolution of <em>Mycobacterium tuberculosis</em> in a high tuberculosis incidence area</span>. <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">529</span>–<span class="cit-lpage">538</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DSupply%26rft.auinit1%253DP.%26rft.volume%253D47%26rft.issue%253D2%26rft.spage%253D529%26rft.epage%253D538%26rft.atitle%253DLinkage%2Bdisequilibrium%2Bbetween%2Bminisatellite%2Bloci%2Bsupports%2Bclonal%2Bevolution%2Bof%2BMycobacterium%2Btuberculosis%2Bin%2Ba%2Bhigh%2Btuberculosis%2Bincidence%2Barea.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.2003.03315.x%26rft_id%253Dinfo%253Apmid%252F12519202%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.2003.03315.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12519202&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180692100021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.47" data-doi="10.1101/gr.7020108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holden</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carder</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lennard</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lockey</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title"><em>Chlamydia trachomatis</em>: genome sequence analysis of lymphogranuloma venereum isolates</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">161</span>–<span class="cit-lpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.7020108%26rft_id%253Dinfo%253Apmid%252F18032721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE4LzEvMTYxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi83Lzc2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.48" data-doi="10.1016/j.tim.2009.03.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vos</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Why do bacteria engage in homologous recombination?</span> <abbr class="cit-jnl-abbrev">Trends Microbiol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">226</span>–<span class="cit-lpage">232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmicrobiology%26rft.stitle%253DTrends%2BMicrobiol%26rft.aulast%253DVos%26rft.auinit1%253DM.%26rft.volume%253D17%26rft.issue%253D6%26rft.spage%253D226%26rft.epage%253D232%26rft.atitle%253DWhy%2Bdo%2Bbacteria%2Bengage%2Bin%2Bhomologous%2Brecombination%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2009.03.001%26rft_id%253Dinfo%253Apmid%252F19464181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2009.03.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19464181&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267449400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.49" data-doi="10.1038/ismej.2008.93"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vos</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didelot</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A comparison of homologous recombination rates in bacteria and archaea</span>. <abbr class="cit-jnl-abbrev">ISME J.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">199</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DISME%2BJ%26rft.aulast%253DVos%26rft.auinit1%253DM.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253D199%26rft.epage%253D208%26rft.atitle%253DA%2Bcomparison%2Bof%2Bhomologous%2Brecombination%2Brates%2Bin%2Bbacteria%2Band%2Barchaea.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fismej.2008.93%26rft_id%253Dinfo%253Apmid%252F18830278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ismej.2008.93&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18830278&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263261800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-book" id="cit-2.7.761.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whittam</span>  <span class="cit-name-given-names">T. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Genetic population structure and pathogenicity in enteric bacteria</span>, pp. <span class="cit-fpage">217</span>–<span class="cit-lpage">245</span> <span class="cit-comment">in</span> <span class="cit-source">Population Genetics of Bacteria: Symposium 52 (Society for General Microbiology Symposia)</span>, <span class="cit-comment">edited by</span> <span class="cit-ed"><span class="cit-name-surname">Baumberg</span>  <span class="cit-name-given-names">S.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">J. P. W.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Wellington</span>  <span class="cit-name-given-names">E. M. H.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Saunders</span>  <span class="cit-name-given-names">J. R.</span></span>. <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.761.51" data-doi="10.1016/S0014-5793(99)00563-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zomorodipour</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">S. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Obligate intracellular parasites: <em>Rickettsia prowazekii</em> and <em>Chlamydia trachomatis</em></span>. <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">452</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DZomorodipour%26rft.auinit1%253DA.%26rft.volume%253D452%26rft.issue%253D1-2%26rft.spage%253D11%26rft.epage%253D15%26rft.atitle%253DObligate%2Bintracellular%2Bparasites%253A%2BRickettsia%2Bprowazekii%2Band%2BChlamydia%2Btrachomatis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0014-5793%252899%252900563-3%26rft_id%253Dinfo%253Apmid%252F10376669%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0014-5793(99)00563-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10376669&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000080837700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/7/761.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/7/753" title="A Noncomplementation Screen for Quantitative Trait Alleles in Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/7/769" title="Genetic Diversity and Population Structure of Chinese Foxtail Millet [Setaria italica (L.) Beauv.] Landraces" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/7"><span>Volume 2 Issue 7, July 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/7.cover-source.jpg?itok=tq8UCk63" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/7/761.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-eR1HtdQfcVPDAS1-UUAe3eqWbojFNP1v9JBLbM5MdMc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/7/761" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/7/761" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/7/761" class="active" data-icon-position="" data-hide-link-title="0">Impact of Loci Nature on Estimating Recombination and Mutation Rates in Chlamydia trachomatis</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459291" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-QUQrwpklgryl_3gaiWfzeA_kd7RRD59TBQuUxG2GMZw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/7/761.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/7/761" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-lAzh3VAyFQtyrU1htT-3gmBHwQ86J-DSoVhOUf7VVaw" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/7/761" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/7/761.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459291" id="node459291--2" data-pisa="ggg;2/7/761" data-pisa-master="ggg;2/7/761" data-apath="/ggg/2/7/761.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Impact of Loci Nature on Estimating Recombination and Mutation Rates in <em>Chlamydia trachomatis</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Rita Ferreira</span>, <span class="highwire-citation-author" data-delta="1">Vítor Borges</span>, <span class="highwire-citation-author" data-delta="2">Alexandra Nunes</span>, <span class="highwire-citation-author" data-delta="3">Paulo Jorge Nogueira</span>, <span class="highwire-citation-author" data-delta="4">Maria José Borrego</span> and <span class="highwire-citation-author" data-delta="5">João Paulo Gomes</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">761-768;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002923 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459291--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Rita Ferreira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Infectious Diseases, National Institute of Health, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rita%2BFerreira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferreira%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARita%2BFerreira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Vítor Borges </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Infectious Diseases, National Institute of Health, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=V%C3%ADtor%2BBorges%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Borges%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AV%25C3%25ADtor%2BBorges%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alexandra Nunes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Infectious Diseases, National Institute of Health, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexandra%2BNunes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nunes%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexandra%2BNunes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paulo Jorge Nogueira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Preventive Medicine–Faculty of Medicine, University of Lisbon, 1649-028 Lisbon, Portugal, and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>General Directorate of Health, Lisbon, 1049-005 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulo%2BJorge%2BNogueira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nogueira%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulo%2BJorge%2BNogueira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Maria José Borrego </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Infectious Diseases, National Institute of Health, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BJos%C3%A9%2BBorrego%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Borrego%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BJos%25C3%25A9%2BBorrego%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">João Paulo Gomes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Infectious Diseases, National Institute of Health, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jo%C3%A3o%2BPaulo%2BGomes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gomes%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJo%25C3%25A3o%2BPaulo%2BGomes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.paulo.gomes@insa.min-saude.pt" class="" data-icon-position="" data-hide-link-title="0">j.paulo.gomes@insa.min-saude.pt</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/7/761" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/7/761&amp;title=Impact%20of%20Loci%20Nature%20on%20Estimating%20Recombination%20and%20Mutation%20Rates%20in%20Chlamydia%20trachomatis" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/7/761&amp;title=Impact%20of%20Loci%20Nature%20on%20Estimating%20Recombination%20and%20Mutation%20Rates%20in%20Chlamydia%20trachomatis" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/7/761&amp;title=Impact%20of%20Loci%20Nature%20on%20Estimating%20Recombination%20and%20Mutation%20Rates%20in%20Chlamydia%20trachomatis" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/7/761&amp;text=Impact%20of%20Loci%20Nature%20on%20Estimating%20Recombination%20and%20Mutation%20Rates%20in%20Chlamydia%20trachomatis" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/7/761&amp;title=Impact%20of%20Loci%20Nature%20on%20Estimating%20Recombination%20and%20Mutation%20Rates%20in%20Chlamydia%20trachomatis" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/7/761&amp;t=Impact%20of%20Loci%20Nature%20on%20Estimating%20Recombination%20and%20Mutation%20Rates%20in%20Chlamydia%20trachomatis" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/7/761&amp;title=Impact%20of%20Loci%20Nature%20on%20Estimating%20Recombination%20and%20Mutation%20Rates%20in%20Chlamydia%20trachomatis" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/7/761&amp;title=Impact%20of%20Loci%20Nature%20on%20Estimating%20Recombination%20and%20Mutation%20Rates%20in%20Chlamydia%20trachomatis" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459291/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459291" id="node459291--3" data-pisa="ggg;2/7/761" data-pisa-master="ggg;2/7/761" data-apath="/ggg/2/7/761.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Impact of Loci Nature on Estimating Recombination and Mutation Rates in <em>Chlamydia trachomatis</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Rita Ferreira</span>, <span class="highwire-citation-author" data-delta="1">Vítor Borges</span>, <span class="highwire-citation-author" data-delta="2">Alexandra Nunes</span>, <span class="highwire-citation-author" data-delta="3">Paulo Jorge Nogueira</span>, <span class="highwire-citation-author" data-delta="4">Maria José Borrego</span> and <span class="highwire-citation-author" data-delta="5">João Paulo Gomes</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">761-768;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002923 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459291--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Rita Ferreira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Infectious Diseases, National Institute of Health, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rita%2BFerreira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferreira%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARita%2BFerreira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Vítor Borges </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Infectious Diseases, National Institute of Health, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=V%C3%ADtor%2BBorges%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Borges%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AV%25C3%25ADtor%2BBorges%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alexandra Nunes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Infectious Diseases, National Institute of Health, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexandra%2BNunes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nunes%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexandra%2BNunes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paulo Jorge Nogueira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Preventive Medicine–Faculty of Medicine, University of Lisbon, 1649-028 Lisbon, Portugal, and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>General Directorate of Health, Lisbon, 1049-005 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulo%2BJorge%2BNogueira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nogueira%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulo%2BJorge%2BNogueira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Maria José Borrego </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Infectious Diseases, National Institute of Health, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BJos%C3%A9%2BBorrego%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Borrego%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BJos%25C3%25A9%2BBorrego%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">João Paulo Gomes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Infectious Diseases, National Institute of Health, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jo%C3%A3o%2BPaulo%2BGomes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gomes%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJo%25C3%25A3o%2BPaulo%2BGomes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.paulo.gomes@insa.min-saude.pt" class="" data-icon-position="" data-hide-link-title="0">j.paulo.gomes@insa.min-saude.pt</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=761&amp;rft.epage=768&amp;rft.atitle=Impact%20of%20Loci%20Nature%20on%20Estimating%20Recombination%20and%20Mutation%20Rates%20in%20Chlamydia%20trachomatis&amp;rft.volume=2&amp;rft.issue=7&amp;rft.date=2012-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Ferreira%2C+Rita&amp;rft.au=Borges%2C+V%C3%ADtor&amp;rft.au=Nunes%2C+Alexandra&amp;rft.au=Nogueira%2C+Paulo+Jorge&amp;rft.au=Borrego%2C+Maria+Jos%C3%A9&amp;rft.au=Gomes%2C+Jo%C3%A3o+Paulo"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459291/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459291/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459291/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459291/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459291/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459291/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459291/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459291/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459291/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459291/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459291/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459291/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/7/761" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/7/761.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/7/761.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/7/761.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/7/761.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
