<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/8/1241" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/8/1241.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/8/1241.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/8/1241.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/8/1241/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/8/1241.atom" />
<meta name="HW.pisa" content="ggg;3/8/1241" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Animal Models and Integrated Nested Laplace Approximations" />
<meta name="DC.Identifier" content="10.1534/g3.113.006700" />
<meta name="DC.Date" content="2013-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Holand et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Animal models are generalized linear mixed models used in evolutionary biology and animal breeding to identify the genetic part of traits. Integrated Nested Laplace Approximation (INLA) is a methodology for making fast, nonsampling-based Bayesian inference for hierarchical Gaussian Markov models. In this article, we demonstrate that the INLA methodology can be used for many versions of Bayesian animal models. We analyze animal models for both synthetic case studies and house sparrow ( Passer domesticus ) population case studies with Gaussian, binomial, and Poisson likelihoods using INLA. Inference results are compared with results using Markov Chain Monte Carlo methods. For model choice we use difference in deviance information criteria (DIC). We suggest and show how to evaluate differences in DIC by comparing them with sampling results from simulation studies. We also introduce an R package, AnimalINLA, for easy and fast inference for Bayesian Animal models using INLA." />
<meta name="DC.Contributor" content="Anna Marie Holand" />
<meta name="DC.Contributor" content="Ingelin Steinsland" />
<meta name="DC.Contributor" content="Sara Martino" />
<meta name="DC.Contributor" content="Henrik Jensen" />
<meta name="article:published_time" content="2013-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Animal Models and Integrated Nested Laplace Approximations" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Animal models are generalized linear mixed models used in evolutionary biology and animal breeding to identify the genetic part of traits. Integrated Nested Laplace Approximation (INLA) is a methodology for making fast, nonsampling-based Bayesian inference for hierarchical Gaussian Markov models. In this article, we demonstrate that the INLA methodology can be used for many versions of Bayesian animal models. We analyze animal models for both synthetic case studies and house sparrow (&lt;i&gt;Passer domesticus&lt;/i&gt;) population case studies with Gaussian, binomial, and Poisson likelihoods using INLA. Inference results are compared with results using Markov Chain Monte Carlo methods. For model choice we use difference in deviance information criteria (DIC). We suggest and show how to evaluate differences in DIC by comparing them with sampling results from simulation studies. We also introduce an R package, AnimalINLA, for easy and fast inference for Bayesian Animal models using INLA.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/08/01" />
<meta name="citation_mjid" content="ggg;3/8/1241" />
<meta name="citation_id" content="3/8/1241" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/8/1241" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/8/1241.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/8/1241.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/8/1241.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.006700" />
<meta name="citation_pmid" content="23708299" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1241" />
<meta name="citation_lastpage" content="1251" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Anna Marie Holand" />
<meta name="citation_author_institution" content="Centre for Biodiversity Dynamics, Department of Biology, NTNU, NO-7491 Trondheim, Norway" />
<meta name="citation_author_email" content="anna.m.holand@ntnu.no" />
<meta name="citation_author" content="Ingelin Steinsland" />
<meta name="citation_author_institution" content="Centre for Biodiversity Dynamics, Department of Mathematical Sciences, NTNU, NO-7491 Trondheim, Norway" />
<meta name="citation_author" content="Sara Martino" />
<meta name="citation_author_institution" content="Centre for Biodiversity Dynamics, Department of Mathematical Sciences, NTNU, NO-7491 Trondheim, Norway" />
<meta name="citation_author" content="Henrik Jensen" />
<meta name="citation_author_institution" content="Centre for Biodiversity Dynamics, Department of Biology, NTNU, NO-7491 Trondheim, Norway" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=B. Bolker;citation_author=M. Brooks;citation_author=C. J. Clark;citation_author=S. W. Geange;citation_author=J. R. Poulsen;citation_title=Generalized linear mixed models: a practical guide for ecology and evolution.;citation_pages=127-135;citation_volume=24;citation_year=2009;citation_issue=3;citation_pmid=19185386;citation_doi=10.1016/j.tree.2008.10.008" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. R. Dempster;citation_author=I. M. Lerner;citation_title=HERITABILITY OF THRESHOLD CHARACTERS;citation_pages=212-236;citation_volume=35;citation_year=1950;citation_issue=2;citation_pmid=17247344" />
<meta name="citation_reference" content="citation_journal_title=Scand. J. Stat.;citation_author=J. Eidsvik;citation_author=S. Martino;citation_author=H. Rue;citation_title=Approximate Bayesian inference in spatial generalized linear mixed models.;citation_pages=1-22;citation_volume=36;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. Foerster;citation_author=T. Coulson;citation_author=B. C. Sheldon;citation_author=J. M. Pemberton;citation_author=T. H. Clutton-Brock;citation_title=Sexually antagonistic genetic variation for fitness in red deer.;citation_pages=1107-1110;citation_volume=447;citation_year=2007;citation_issue=7148;citation_pmid=17597758;citation_doi=10.1038/nature05912" />
<meta name="citation_reference" content="citation_journal_title=Biostatistics;citation_author=Y. Fong;citation_author=H. Rue;citation_author=J. Wakefield;citation_title=Bayesian inference for generalized linear mixed models.;citation_volume=11;citation_year=2010;citation_pmid=19966070;citation_doi=10.1093/biostatistics/kxp053" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. L. Foulley;citation_author=D. Gianola;citation_author=S. Im;citation_title=Genetic evaluation of traits distributed as Poisson-binomial with reference to reproductive characters.;citation_pages=870;citation_volume=73;citation_year=1987;citation_pmid=24241297;citation_doi=10.1007/BF00289392" />
<meta name="citation_reference" content="citation_journal_title=Bayesian Anal.;citation_author=A. Gelman;citation_title=Prior distributions for variance parameters in hierarchical models.;citation_pages=515-533;citation_volume=1;citation_year=2006;citation_doi=10.1214/06-BA117A" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=D. Gianola;citation_author=R. L. Fernando;citation_title=Bayesian Methods in Animal Breeding Theory;citation_pages=217-244;citation_volume=63;citation_year=1986;citation_issue=1;citation_doi=10.2527/jas1986.631217x" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. E. Goddard;citation_author=B. C. Hayes;citation_title=Mapping genes for complex traits in domestic animals and their use in breeding programmes.;citation_pages=381-391;citation_volume=10;citation_year=2009;citation_issue=6;citation_pmid=19448663;citation_doi=10.1038/nrg2575" />
<meta name="citation_reference" content="citation_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=J. D. Hadfield;citation_title=Estimating evolutionary parameters when viability selection is operating;citation_pages=723-734;citation_volume=275;citation_year=2008;citation_issue=1635;citation_pmid=18211873;citation_doi=10.1098/rspb.2007.1013" />
<meta name="citation_reference" content="citation_journal_abbrev=J Stat Softw;citation_author=J. D. Hadfield;citation_title=Regularization Paths for Generalized Linear Models via Coordinate Descent.;citation_pages=1-22;citation_volume=33;citation_year=2010;citation_issue=1;citation_pmid=20808728;citation_doi=10.1111/j.1467-9868.2005.00503.x" />
<meta name="citation_reference" content="citation_journal_abbrev=Am Nat;citation_author=J. D. Hadfield;citation_author=A. J. Wilson;citation_author=D. Garant;citation_author=B. C. Sheldon;citation_author=L. E. B. Kruuk;citation_title=The misuse of BLUP in ecology and evolution.;citation_pages=116-125;citation_volume=175;citation_year=2010;citation_issue=1;citation_pmid=19922262;citation_doi=10.1086/648604" />
<meta name="citation_reference" content="citation_journal_title=Evol. Biol.;citation_author=T. F. Hansen;citation_author=C. Pélabon;citation_author=D. Houle;citation_title=Heritability is not evolvability.;citation_pages=258-277;citation_volume=38;citation_year=2011;citation_doi=10.1007/s11692-011-9127-6" />
<meta name="citation_reference" content="citation_journal_title=Ann. Math. Stat.;citation_author=C. Henderson;citation_title=Estimation of genetic parameters.;citation_pages=309-310;citation_volume=21;citation_year=1950" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=H. Jensen;citation_author=I. Steinsland;citation_author=T. H. Ringsby;citation_author=B.-E. Sæther;citation_title=Evolutionary dynamics of a sexual ornament in the house sparrow (Passer domesticus): the role of indirect selection within and between sexes.;citation_pages=1275-1293;citation_volume=62;citation_year=2008;citation_issue=6;citation_pmid=18384654;citation_doi=10.1111/j.1558-5646.2008.00395.x" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=J. S. Jones;citation_title=The heritability of fitness: Bad news for 'good genes'?;citation_pages=35-38;citation_volume=2;citation_year=1987;citation_issue=2;citation_pmid=21227813;citation_doi=10.1016/0169-5347(87)90096-6" />
<meta name="citation_reference" content="citation_journal_title=Phil. Trans. R. Soc. B;citation_author=L. Kruuk;citation_title=Estimating genetic parameters in natural populations using the ’animal model’.;citation_pages=874-890;citation_volume=359;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Ecol. Evol. Syst.;citation_author=L. E. B. Kruuk;citation_author=J. Slate;citation_author=A. J. Wilson;citation_title=New answers for old questions: The evolutionary quantitative genetics of wild animal populations.;citation_pages=525-548;citation_volume=39;citation_year=2008;citation_doi=doi:10.1146/annurev.ecolsys.39.110707.173542" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=R. Lande;citation_author=S. J. Arnold;citation_title=The measurement of selection on correlated characters.;citation_pages=1210-1226;citation_volume=37;citation_year=1983;citation_doi=10.2307/2408842" />
<meta name="citation_reference" content="citation_journal_title=Stat. Comput.;citation_author=D. Lunn;citation_author=A. Thomas;citation_author=N. Best;citation_author=D. Spiegelhalter;citation_title=WinBUGS—a Bayesian modelling framework: concepts, structure, and extensibility.;citation_pages=325-337;citation_volume=10;citation_year=2000;citation_doi=10.1023/A:1008929526011" />
<meta name="citation_reference" content="citation_title=Genetics and Analysis of Quantitative Traits;citation_author=M. Lynch;citation_author=B. Walsh;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Scand. J. Stat.;citation_author=S. Martino;citation_author=R. Akerkar;citation_author=H. Rue;citation_title=Approximate Bayesian inference for survival models.;citation_pages=514-528;citation_volume=38;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=C. Matos;citation_author=D. L. Thomas;citation_author=D. Gianola;citation_author=M. Perez-Enciso;citation_author=L. D. Young;citation_title=Genetic analysis of discrete reproductive traits in sheep using linear and nonlinear models: II. Goodness of fit and predictive ability;citation_pages=88-94;citation_volume=75;citation_year=1997;citation_issue=1;citation_pmid=9027552" />
<meta name="citation_reference" content="citation_journal_abbrev=Am Nat;citation_author=J. Merilä;citation_author=B. C. Sheldon;citation_title=Lifetime Reproductive Success and Heritability in Nature.;citation_pages=301-310;citation_volume=155;citation_year=2000;citation_issue=3;citation_pmid=10718727;citation_doi=10.1086/303330" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=K. Meyer;citation_title=Estimating variances and covariances for multivariate animal models by restricted maximum likelihood.;citation_pages=67-83;citation_volume=23;citation_year=1991;citation_doi=10.1186/1297-9686-23-1-67" />
<meta name="citation_reference" content="citation_journal_title=Journal of evolutionary biology;citation_journal_abbrev=J Evol Biol;citation_author=R. B. O’Hara;citation_author=J. M. Cano;citation_author=O. Ovaskainen;citation_author=C. Teplitsky;citation_author=J. S. Alho;citation_title=Bayesian approaches in evolutionary quantitative genetics.;citation_pages=949-957;citation_volume=21;citation_year=2008;citation_issue=4;citation_pmid=18373658;citation_doi=10.1111/j.1420-9101.2008.01529.x" />
<meta name="citation_reference" content="citation_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=O. Ovaskainen;citation_author=J. M. Cano;citation_author=J. Merilä;citation_title=A Bayesian framework for comparative quantitative genetics;citation_pages=669-678;citation_volume=275;citation_year=2008;citation_issue=1635;citation_pmid=18211881;citation_doi=10.1098/rspb.2007.0949" />
<meta name="citation_reference" content="citation_journal_title=Journal of evolutionary biology;citation_journal_abbrev=J Evol Biol;citation_author=E. Postma;citation_title=Implications of the difference between true and predicted breeding values for the study of natural selection and micro-evolution.;citation_pages=309-320;citation_volume=19;citation_year=2006;citation_issue=2;citation_pmid=16599906;citation_doi=10.1111/j.1420-9101.2005.01007.x" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Ecol;citation_author=H. Pärn;citation_author=H. Jensen;citation_author=T. H. Ringsby;citation_author=B.-E. Sæther;citation_title=Sex-specific fitness correlates of dispersal in a house sparrow metapopulation.;citation_pages=1216-1225;citation_volume=78;citation_year=2009;citation_issue=6;citation_pmid=19674180;citation_doi=10.1111/j.1365-2656.2009.01597.x" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=R. Quaas;citation_title=Computing the diagonal elements and inverse of a large numerator relationship matrix.;citation_pages=949-953;citation_volume=32;citation_year=1976;citation_doi=10.2307/2529279" />
<meta name="citation_reference" content="citation_journal_title=Ecology;citation_author=T. H. Ringsby;citation_author=B.-E. Sæther;citation_author=J. Tufto;citation_author=H. Jensen;citation_author=E. Solberg;citation_title=Asynchronous spatiotemporal demography of a house sparrow metapopulation in a correlated environment.;citation_pages=561-569;citation_volume=83;citation_year=2002;citation_doi=10.1890/0012-9658(2002)083[0561:ASDOAH]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Bayesian Anal.;citation_author=M. Roos;citation_author=L. Held;citation_title=Sensitivity analysis in bayesian generalized linear mixed models for binary data.;citation_pages=259-278;citation_volume=6;citation_year=2011;citation_doi=10.1214/11-BA609" />
<meta name="citation_reference" content="citation_title=Gaussian Markov Random Fields: Theory and Applications;citation_author=H. Rue;citation_author=L. Held;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=J. Statist. Plann. Inference;citation_author=H. Rue;citation_author=S. Martino;citation_title=Approximate Bayesian inference for hierarchical Gaussian Markov random field models.;citation_pages=3177-3192;citation_volume=137;citation_year=2007;citation_doi=10.1016/j.jspi.2006.07.016" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc., B;citation_author=H. Rue;citation_author=S. Martino;citation_author=N. Chopin;citation_title=Approximate Bayesian inference for latent Gaussian models using integrated nested Laplace approximations.;citation_pages=319-392;citation_volume=71;citation_year=2009;citation_doi=10.1111/j.1467-9868.2008.00700.x" />
<meta name="citation_reference" content="citation_journal_title=J. Roy. Stat. Soc.;citation_author=B. Schrödle;citation_author=L. Held;citation_author=A. Riebler;citation_author=J. Danuser;citation_title=Using integrated nested Laplace approximations for the evaluation of veterinary surveillance data from Switzerland: a case-study.;citation_pages=261-279;citation_volume=60;citation_year=2011" />
<meta name="citation_reference" content="citation_title=Genetic Improvement of Cattle and Sheep;citation_author=G. Simm;citation_year=1998" />
<meta name="citation_reference" content="citation_title=Generalized Latent Variable Modeling: Multilevel, Longitudinal, and Structural Equation Models;citation_author=A. Skrondal;citation_author=S. Rabe-Hesketh;citation_year=2004" />
<meta name="citation_reference" content="citation_title=Likelihood, Bayesian and MCMC Methods in Quantitative Genetics;citation_author=D. Sorensen;citation_author=D. Gianola;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=D. Sorensen;citation_author=C. S. Wang;citation_author=J. Jensen;citation_author=D. Gianola;citation_title=Bayesian analysis of genetic change due to selection using Gibbs sampling.;citation_pages=333-360;citation_volume=26;citation_year=1994;citation_doi=10.1051/gse:19940403" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B.;citation_author=D. J. Spiegelhalter;citation_author=N. G. Best;citation_author=B. P. Carlin;citation_author=A. van der Linde;citation_title=Bayesian measures of model complexity and fit (with discussion).;citation_pages=583-639;citation_volume=64;citation_year=2002;citation_doi=10.1111/1467-9868.00353" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=I. Steinsland;citation_author=H. Jensen;citation_title=Utilizing Gaussian Markov random field properties of Bayesian animal models.;citation_pages=763-771;citation_volume=66;citation_year=2010;citation_pmid=19817739;citation_doi=10.1111/j.1541-0420.2009.01336.x" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=R. J. Tempelman;citation_author=D. Gianola;citation_title=Assessment of a Poisson animal model for embryo yield in a simulated multiple ovulation-embryo transfer scheme.;citation_pages=263-290;citation_volume=26;citation_year=1994;citation_doi=10.1186/1297-9686-26-3-263" />
<meta name="citation_reference" content="citation_journal_title=Biostatistics;citation_author=M. A. Van De Wiel;citation_author=G. G. Leday;citation_title=Bayesian analysis of RNA sequencing data by estimating multiple shrinkage priors.;citation_volume=14;citation_year=2013;citation_pmid=22988280;citation_doi=10.1093/biostatistics/kxs031" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=A. I. Vazquez;citation_author=D. Gianola;citation_author=D. Bates;citation_author=K. A. Weigel;citation_author=B. Heringstad;citation_title=Assessment of Poisson, logit, and linear models for genetic analysis of clinical mastitis in Norwegian Red cows;citation_pages=739-748;citation_volume=92;citation_year=2009;citation_issue=2;citation_pmid=19164686;citation_doi=10.3168/jds.2008-1325" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=P. M. Visscher;citation_author=W. G. Hill;citation_author=N. R. Wray;citation_title=Heritability in the genomics era--concepts and misconceptions.;citation_pages=255-266;citation_volume=9;citation_year=2008;citation_issue=4;citation_pmid=18319743;citation_doi=10.1038/nrg2322" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Ecol.;citation_author=A. J. Wilson;citation_author=D. Réale;citation_author=M. N. Clements;citation_author=M. M. Morrissey;citation_author=E. Postma;citation_title=An ecologist's guide to the animal model.;citation_pages=13-26;citation_volume=79;citation_year=2009;citation_doi=10.1111/j.1365-2656.2009.01639.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Yang;citation_author=R. J. Tempelman;citation_title=A Bayesian antedependence model for whole genome prediction.;citation_volume=190;citation_year=2012;citation_pmid=22135352;citation_doi=10.1534/genetics.111.131540" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Animal Models and Integrated Nested Laplace Approximations" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8.cover-source.jpg" />
<meta name="twitter:description" content="Animal models are generalized linear mixed models used in evolutionary biology and animal breeding to identify the genetic part of traits. Integrated Nested Laplace Approximation (INLA) is a methodology for making fast, nonsampling-based Bayesian inference for hierarchical Gaussian Markov models. In this article, we demonstrate that the INLA methodology can be used for many versions of Bayesian animal models. We analyze animal models for both synthetic case studies and house sparrow ( Passer domesticus ) population case studies with Gaussian, binomial, and Poisson likelihoods using INLA. Inference results are compared with results using Markov Chain Monte Carlo methods. For model choice we use difference in deviance information criteria (DIC). We suggest and show how to evaluate differences in DIC by comparing them with sampling results from simulation studies. We also introduce an R package, AnimalINLA, for easy and fast inference for Bayesian Animal models using INLA." />
<meta name="og-title" property="og:title" content="Animal Models and Integrated Nested Laplace Approximations" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/8/1241" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Animal models are generalized linear mixed models used in evolutionary biology and animal breeding to identify the genetic part of traits. Integrated Nested Laplace Approximation (INLA) is a methodology for making fast, nonsampling-based Bayesian inference for hierarchical Gaussian Markov models. In this article, we demonstrate that the INLA methodology can be used for many versions of Bayesian animal models. We analyze animal models for both synthetic case studies and house sparrow ( Passer domesticus ) population case studies with Gaussian, binomial, and Poisson likelihoods using INLA. Inference results are compared with results using Markov Chain Monte Carlo methods. For model choice we use difference in deviance information criteria (DIC). We suggest and show how to evaluate differences in DIC by comparing them with sampling results from simulation studies. We also introduce an R package, AnimalINLA, for easy and fast inference for Bayesian Animal models using INLA." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8.cover-source.jpg" />
    <title>Animal Models and Integrated Nested Laplace Approximations | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"DDRH-KkcbR9Ei0J__sX8Os4nq9arZ9_tkqBW1vadjOQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459516","apath":"\/ggg\/3\/8\/1241.atom","pisa":"ggg;3\/8\/1241","ac":{"\/ggg\/3\/8\/1241.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/8\/1241.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/8\/1241"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/8\/1241"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/8\/1241":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022uPv9QrLD8Utcz7QcT8malKn6oUjje+eAREU02QB0DoYZWQ+BMgIWB+m7AqI7wUxMOUNtwEPJylr2nby3rKydn9mnaMeDLFIrk3V941bpi5UwB4I9T8x46uD+Lvva7ewlrkp4X7B58XRYgshUVVJ8eDSXQUnlrzy5\\\\\\\/YZ2iPHEd02LY2TuvGJCW+WMod7c84APOvEsRjL4Dqn7CbCa\\\\\\\/NHtGhSQdqf+7Py1a+CRCxzqFArgiGVpszLm1XMTTbwO8ZLbsI13SmQICFem99Kcb80uQKhz+AoMmmycQUrtK5\\\\\\\/ztA2SLK81mvrfLjjgbaGKloXZ8XKZZGdomT8w3bZrolDIvKf4bgnivDzvIlzy1G2nJxF7JF8bE8g1pKVhYgoE9yLceiA0PotkTtvsUlKbVLaW7U2CNmMpvbuX+7POo1FBZ5A635pU8jmdc\\\\\\\/vibW9Qfv5zoZfohZtmAQCQtTW5Gug\\\\\\\/Nkc6jB1ysCkURmLm\\\\\\\/m9nBXvUoMfNDOg07ZsXrtzSxp5d17d9bqglZ0BoykbvFObrxD88HJSWHMzZbtBWRSmPpQ0iXBPiDvbULxctXQXrLqTiKghxuPIXKP0MUlmENAKQqbbgvEeXoYUO\\\\\\\/KnypVDOgzn\\\\\\\/eshZOrJo3fGlIfXfLX65Xgefn6Jdv5pZvf2w0Gob6b+1t71P5HFJpJAuJznNlQ5DARvOCQHZwRiHAUXIftsGomLZGaQ5pGB\\\\\\\/L42D7N2AEN2gQ4dgsX2wFIc249AhLSMaQoaRMxco\\\\\\\/iwFaw3S1RGpuLCU1Ts0LF\\\\\\\/UvRhxY6N+T+LNCPS6iJV\\\\\\\/7Vinr\\\\\\\/XxRLG1Lve2Ppjba58a7bS7HBdFa6FWPU3gZtzlTWxGemJ01e\\\\\\\/q5HGyhcTbK7law18YaZgBgZoRdAV6zUWdi3xH\\\\\\\/9keL\\\\\\\/YhECn1+Avyjo25ZwrPgE6zj+TkTnBeA1lUO0xJbQvzKxyHOwqNhfgPeheBqGRkSw4Zk+i0AfmkJjjcG\\\\\\\/xsi9gWM87RUfyUc2R+SrfNxtOcQe8eBfVA5HwRYraBOwvqKjk5XWSb4Ku9h7O7pvXjP24AMvVydRDAH16J+cAeZ0nBIFv9wbNNQ\\\\\\\/\\\\\\\/cjsY0rRtOPM8jOnSOsq+fH3wUKqLeaL4sIlffTzqz8TONgBX5ZZPeqTR\\\\\\\/SRRKRq4gaM44pKuxSb88s5vSV+rgE95G8bbTggEtVQL4YYwWYzSSohzFm9l1FzqVSOvVUugNiTy\\\\\\\/lXCqnMEzGZAyuTD9cdOoRIihgCdZ7N2kuFmR0cnCj9bCVqDgEc9umWo4aFi5BvVGh26X\\\u0022,\\\u0022iv\\\u0022:\\\u0022pOgPT+wPuYRZ++Pg1FkYFw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c8cbd6d169e010d44fb0a9cfb7fd7a5a711d852ea5c418e51843625aa72e3426\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022gLayFKQ7lKL7n3iatLLvPkwMmMA8gqE5TfWaWjuW1RLhGVRiyk+zwueC49WnBl5K+ZPYPVndOaOWPH96ET6v0z7hi1UBrutvVSd7J1wievcroXwR6oY+ES28VpPag5cLYlG9syD0WG4QcwC0Y6CeOA4tOYgvlwcLztjAT0YHNiqWgQYxrP4mx1kz+zLnlb\\\\\\\/nd73bkOr1K2cbfesux1ijr6ZkW1B2AwU8Ql5w3utCmkD0RsbL0WoQ4WNLSGTQpU5Uk299w1JQ\\\\\\\/z44Euvq6yI1dtcWRz0foLPs5fHlK5MElD7jVlYnfMXlbGWPSMF5lP45cZFSJdv0jja\\\\\\\/JzrV8b9woj71HgYX0efvAmAXNtk3LdhyzXrZGg9wbczKD8IgI2DL9MyrUa+ukIydi9iMpQ\\\\\\\/8ORTjtvxGGINs1unew+n3iT\\\\\\\/5i6vv85700QiX8R8Wk\\\\\\\/qFHZYDqKvGJ1oqQT1B3XSWKZZh2njfKcSKLGIvUoS5uOBg7OaVlcMaSkXK3iEY+BURUGY1xnYJBXQ1gPUT0jbx+Y3T5W5SjBdoI2DMRDa5UDGzn1d89s35tf\\\\\\\/V+DtzUOgDfy2jqPc\\\\\\\/NwrbpCq5ikXT8Mpo7IRMcGNZV3F\\\\\\\/Xkqp\\\\\\\/I4nj6wy8VQ5Q5fjiG5yqXb\\\\\\\/wlAKMvxRZ8lpOgHqv+cgY3+2DaxUl22xAUZQIJmGKW5Cji45DeApJkZOhKCeds1XHqDX4lBGJwdsrjO6BpGCSbgpykL+33oukAknjEtnkAKrUCHbFOtVhgZy9KjlpA\\\\\\\/opoZaTCb\\\\\\\/VcX+6eU137rSum1\\\\\\\/0B7xjjosjCWv\\\\\\\/FlWvJtM\\\\\\\/YJfClkvnzPhedXwlS6TKZMIuJj2ON3EPCYj1g9F\\\\\\\/6E+6UO0hPztKBPCqO1nB7vKD6vdVAeGTFSz2OwY07p519q3KjFSGHQwjOM0NCDtXQJ5HypGXpcM4TZbbvHuQNvh2tlGk4laJM4VD2yxZR+P+A4Xpg3v9RXpsoAta\\\\\\\/WNhwLJWYvOsJHBRexRzKYk5nYIzdiku4d26q+1dRBSIaLGCCV2bXmI1H7cqg6aqdMeY7seB6xwhDHUczg1HCzL2tReZ35yfq5TByBF4eyD6KouLOXC01XVPZC0r3cw\\\\\\\/SdLmybiuD4f9mDmD8UqtHy6T8mnHta6fn5vk984hksIlKuCN3FHUXoc9blBeU2vPbRHbpPZuK01w6052VQZAv4W135wCwyhigBmeB5wRWa8yVHH\\\\\\\/d1vusFZQ31IVlTm5un4iXfo6BKHBLxndwQOQt\\\\\\\/1b2XX5ko948S63GQKF7EH+\\\\\\\/Z0qWkclV+dVsdWOU2qkfFGB8esy7\\\\\\\/zAgpnlTnlRzrDFnJhT5d+xHWy7jmdyV51wF5dApWDCcr8BX7tGOSlotPoA5ynBWuitq4VxPu2CGIeTIJXGPWVqtbez6rXsEJhfwgkVQLMZbH2ENgYUxjVB7bmZkPcKwyjNX7vZW36R5tNPdAj9XY5b4cTh4r9AAPzbm9NPpWBH2skVBNcC+T2tC4AlTEZMjB5u7nn66\\\\\\\/PtCuYMZhkVUBZBOQkrkjsD0S\\\\\\\/7bB0NwP6\\\\\\\/RbPE0Juexd9yjg+8f9XBoHcKLxAH48n\\\\\\\/Qrle0UEcGCVrivxtr68AgEegL3GXAN34mpLnguwuTT+raSWxG2IMVQljJF05x1GGOdTns6hZKyV\\\\\\\/eJlQeBO3g9Qwe5JrROYHPmGdMs3yrCLuyXNZbBiSszM\\\\\\\/Z\\\\\\\/LewH0XE4w9o3qEtShvPGRN1Ld7wZYuMDG0WSCWZE\\\\\\\/kImPzGkADsDg15mxE9B+GcgLfxGGSEYD9pNIDhf0kzG\\\\\\\/0OwgSJfTsfu5aAr1juGzWtvpNs7oX\\\\\\\/2h3GfdmJ+3D3n+v4yNVZ+PkE0Pgcgehng8yk6+1PW2iKpLvmXgVBSI110gJYRXgPGiiah6ZGgvhwjaWHwBAYgPHXz6PMSmW+SWiP+yIxoX9Y5YiuRtaxa2oLXSZ6V1Zvu6agmMZeZWtwDcfuEtXKXX5fRRFEoTpcUW5q1yg6P3sGcD0T5tU7DD1ftM7REihO4XbF0IJzltPTqrwVwSlr4I53ishViKUseo6nsDD9kqAXwgSLg0z+gNBGEwtZwfkex75iz4jcUxkcfUwjAmxDFIdMqEpWdx418rLJi4dk5HZVv4rQvdNsMXN3EoYh0y96WkW326aNxuJgb22I4VoISkifI6YQijDMBxa67k5ExzD7B+s9t+pzrTvkFAOII0ZkcLWro0YzdCz0pEgiXJyKvUX3t1XVxXgotbc+X3B\\\\\\\/iSvK9kCEsxfWRM66LaWWVdyXkem4VqiiEEq0zGnpuhV5rFZ9AgxItS1DiTGXUEiT\\\\\\\/XaQMDVjK4apnQrb+zCBS9JOqIksO9xVbzKdWHCFpo7P+aCgkSxmAed5lkFnA4aiKpS9Du7viEaRcyz2Kp23v+YkFXMwm6JGnx7fdfmd+7ozsAryxZRUueERPZYw9\\\\\\\/zNlcvn4upG2zPOErL8XfK6\\\\\\\/+gSWoMCwW+zff2uKuk6fWl5UujBb\\\\\\\/vMMXPbC1ybaXvqkg8e1MhtYw12pf0r8uWt4BaEMcqy2WrUEyB1PEa0SRdsXrWScUMsEb\\\\\\\/MQ+YDHOJFUG7rFf0OABKJw8Sxws3BM2lZrWcKxjw9DtIiE+wPHqmR2HG3nxMUWrAdMpK4A+HXD9iqHS9Jc1hvSaIpxgC4ZPUrlrF7c3GF0Jiyf57tf1pAFTPtUfn+DYPB5zyGELOQhcFq\\\\\\\/NlfuRxtQmkyHHGRw+6gg9xPGqSl2zB1u0FBaSw+iYmwtSmJRcDFZeePbBlfsoDe4gTsVi\\\\\\\/wa2GEV2Th8paLMQ\\\\\\\/ygSlnS1SJ935zq+YQxeNkFq2Ou3uk4W4OIA\\\\\\\/WMEMLAYtaHOpMhLH+uAbMllgtvCcdZbULvOt4yBVErOzc\\\\\\\/tYaUZHRWiGVeXGCdaHDDkYe1Ihlq+lSBVV\\\\\\\/kzHOgKPX\\\\\\\/pXOQeeOhQL6\\\\\\\/VVQOb1xdu2zhpUZOcqawJ6mvn7N2MOkivjaX7bwmrb43eMfcv9HKsT8b3+3F52qzZvbA\\\\\\\/3Nq68wgkkoUIjteraeGWqfjtOrKdZfdgfKw3wpjWZuqIEAnY\\\\\\\/OY8fONu\\\\\\\/1nN56dBN10vRrYwpWItc1uD6pbIJYu3J16cgQAcszxwNlGDQc+WOM4aRAvq5r0ne58KCxgcuZXRNdMiqTiWpR4rw12Q\\\\\\\/J5aWxZnYnHvNY2eI\\\\\\\/VqTtkfX9a4IWA\\\\\\\/DJA0AAoPkOJQb29yFSCkaaFL6dGNfHl5vwqMJRmCgKV2j7wbC0b\\\\\\\/HZ+LCzb\\\\\\\/dbu63eqbONQBc+7Hzszdres19HduXWN4TbuBGlJKBzv\\\\\\\/V8KJT52fqtkq21MpvEZkX4o4otFloiu1eYXabNVIuJl5ZJZ2NvHiK6F5tFi2epiu9CRMm53WEJuwkrbK2y9LFOvGc4FklSDjQ\\\\\\\/rRD7AtUO7ZGfN5GKb30c8\\\\\\\/BOhKt1OJU\\\\\\\/VZMr1ywSfa9WBTy66ceo\\\\\\\/FWRdvY0CTl0r7hkGYYV\\\\\\\/oVzrON7u16vjWhrqWkXRO6Abk6zmjKiSvTU+bxRJuR3ODx0VPGU+oPwqxcsJu5gVtfyLNspAWnvyMfxubsQYd5vs0uLKqvQMtro9+jyHSD\\\\\\\/lnPLkdAcGByKBy70F\\\\\\\/NfkOAJwtZgvtJ7ewsttxl8MOCaCd6nCHoXQw67QTZ5XUfFPTaJU0FJ202ZMik0iCLif5n+64C4MET52xDZAyQHEOMkRg8ZQRR7G4skO\\\\\\\/x69k09Eg79U0DDsRAOxipByBd08byb2hI6yZUzevCZ4LWcgeOVRSoY4q2tZZReWhDBI9\\\\\\\/EViOuFL4LM5JvkVmWk54KoludOJmyiY0NVob+IVM0bioZ88qGaZy4sYk083CXAamFRpkYMIozpnA5mwVIZhRpYKfxW3KBqBk2mZAQ185JWJ+QM\\\\\\\/ZLikfJG4ZmqCUtnRBkxCJ8jsYyASJ4TE3uTdr2yM5YG6j92eY16QknRBCmQbrmWruP9ZO3+UQMLk+tLHRDcwXA8o7FvyuLJcTdu3F1cXWqP1FGBrS8qy1nIVNn\\\\\\\/8jalboOLm5B9DX2yPXROuMbD4vskhLiq5\\\\\\\/iB4gvP9RCsECv4FI1thD\\\\\\\/j7GFRFrj91x5BAGlJVFu3Tgy8P5ifRvswFbaTpAxmtYx8r30pnXQCb16JkR5++Fi2kKEM2TjZ0rq4MOwOjT\\\\\\\/dcmhO\\\\\\\/XCDoNYW1Y9tIJ8ngxzhn0xKmZJyUXlqd1VC00H1W3S1QtyYO8RattNIL9k8nqmianfso6IddHfkun73BFJDFApV6WAjLdyR5WCey4NXTsv21roKzt71VFDc8+HjYBbKQcsRtKUuiI2IePE45DLM3+Hgju0jGq6bLCIl186BM93wcxNXdrM5A81Wqx\\\\\\\/lG4dLvjggY4GJioj84S4bY9cmiRq\\\\\\\/KGlYsa5cmH4yc1n3+RkYEMqjC7iT5JnRBUXJKaLukqPGpUMs9O\\\\\\\/l4sBQm7aVrOtf6Dso\\\\\\\/R3AzBVeeOotrtcWizg\\\\\\\/VbSsUV985nnrr8EaeeJQrI7D7krlepLUlhu2k2EjkAeQlV90cQVr9FeoVCX0WENtCqo3UV2wqwOhP1Pq93seFPZluWfFy7ZAiD7aUlWZ2Wh\\\\\\\/AOg4yaiotm\\\\\\\/lqdd1tzm\\\\\\\/XwqUMKLWQPd1H9ttR8oXVAgZIvNPsgfOv8CEiE6vugdVeNPcVH0TMQQDiC3tSK9SEUvv069fPX1Z7yaG2+JfzfAToJod1FHedHDakojt5qYWjOFhD3bhWupESMwY+xD9vcJQVF7l3n+fiG7fVtonvHz7DB3cTKuCyJ4Kr1yMcDuuZrC8Hy19JkrtsYYby2o8\\\\\\\/y71jyhkEaVqktBx5csyKQ\\\\\\\/fzRs1\\\\\\\/lVOFLAb7LrXoLfRomVljWDrt+Xsi9Hs50bfpIbxvxt1LncVCrzJg1\\\\\\\/qStwVzbL6WYzrEi2nsRqeRSThsaf+lnPn39fU3cAUYwoQdJM4YCrS7kuC40\\\\\\\/L4a80EWecZormOKX77qQ+NCFdc3wOX4wMW4fzn6gE1pOzh7LLynRtsE9fNJQmATsC3Ar8u+EvArA9RBMGyvsnikwdYVLD5HnJmQz7KcGYF0EUZJ8BD+WLASB195kCrY\\\\\\\/GIwTsFNJ1XvtYYk2Arj8+qMdm6KJEU5fgBLFk1+JhviLpu6tYaUijO\\\\\\\/lvwBNd2QZckenchkCkqq5ZbnqCz15oVG\\\\\\\/2f9Au4f9T7rZfNCv23rkrdh57QadhINm77NrQEHvoVdfwSOWE1vfjgNWnxIDc0Z+kFWXItSiKsYYZS6WEjADCsXcxufAmAoj4\\\\\\\/oPgScT68kjM4r+VPQP64D1KaW4\\\\\\\/4x1niS\\\\\\\/8VjfE8smNUrl6BuUC9447JiMTjx0iS5sqGOLx9VQe7jH2o7cI6M5zmR6ckkncIbUQDQd9csBQsnl3PJqO3fSfvPMDrB5e2RWFK2R4yecPPD\\\\\\\/YQPi4UTIoMiHgYXx9oxA6Fbb11acB+GeWkC0a55Vw1GwS8HPsRB3qbzkdrs5sqtyVLHtN2E7XlkWpwXbaiZyWWmruMrTsR18EeYVo5BHwZyswfDQs15qBJAKBZISCJFQbYz0RVGqwOfbUHDrAbZfjUbErTeS+kn7MgBApXwrJA04Jtyr6\\\\\\\/aviLe+GknISKu\\\\\\\/AzgRB3OYB5bSjcsEywub0r9\\\\\\\/RKHspSm3tqG8HtgZADsj\\\\\\\/hLXN8Fbsl6Ayv7OQ5yJznYxuOxSn1yMfp9x+nkZJ167y7qfOCVU3SjfzPfN1aL3uygy3MAcBn5wt4fxmC0Dd7ItcohNBzaq+eVUo9PSbwCjSqdlozD+wjiUN2iYXvedvGFLJRBA+ZLvd8aLf8ZOgnXNzs\\\\\\\/DwJhtoaZQ9jzywID6dRwrZT7O1NANqqqOkLduXPbwj20PLtNWikC0TZDeLpSZsLZ5WzvY+glTBe7x1oP1PRfiURN0ZNh+FAEF2F2Nxi395vM6pmwxu96DllNl8UuYBV4RiNMyy16E+YPUCjnuTNeOhikwe7l2m5dl2gkESB\\\\\\\/wVU1Pe3seN7JOojAX95V9CqRnXXJxcq7tkqolYPdUJJwttbiqz0dqQ3znPT9qzhzy0y7ucldqR6r0ncp824DugexeGIIvMIEY8r\\\\\\\/0DGqLXQbEF05MAo7PNxPPKeOhCoqxw5sGFiAf3grJsFx46rRHg5y12+VzIa+JnorkyzTdWgOyhDGT8mzVl85RpDKuRU1aGO1VHaxEfmY9Q8Qq8ZdiV4TKmiRFtG7b9YQx+d8YvBeud\\\\\\\/TO6EnXGMN1n0If2pjysx5Wc9prfZHQ74r9XcZjT5hQ8dYikc9f29DMJYYjSnUAUROs3w5pyQ8iw\\\\\\\/iBIuk8TUM9+zHO6b7DIWShEJvW1RvFaEmzkNrjyBs4aFMdMJUnCHybyT2GJ5Uih13KF59SyNLqNkCXLrYhWEkJ6fcC8trgoYolQ5\\\\\\\/GpmbWlSrb3YR7GPmaAJxG3ULoqh079GQ3CpN3dOSaF6yb9sOypZiTzynCBGfnOIg6OVKq1oBvuoR6F3UREYXw2Ab5+TBqzSdkVtqEM0rODjtREelgMxbtoWi\\\\\\\/PXRDTOK0pUyy2FbEBDoKqlXZDe\\\\\\\/uPSKIJUulC03aecg0M1RomO8MEwa2o1rR1EIFN7bSDAw9jWUlDG4gv8iljJ730PkUPa5XT+Qf9LrzLEW6wSpoVv6Z\\\\\\\/aNXfO6sn9UgZ6EmpqY9BvIzmRLnM+M6+TuQ2u7\\\\\\\/8\\\\\\\/3\\\\\\\/nexfL8a1p8lK\\\\\\\/pAH+pj4lpXNW2EizE0ne16mnnF6VKwvYIbFn+6eu7g4b5pvxA\\\\\\\/6DXEN8BYJcsNYTk50Ov1LmHtDEhOAu1SsIvDqINB6zV8yULB3Xb6dHY6xM\\\\\\\/LTAS2CAAXyPvwEZ3wJ7VqsUG5ts13xVTe2DFEttBWGe8ZzeEPtTupmRR0E0UCY2J4x4446j3TupiuX\\\\\\\/uubnyDNo+6L6WsxGd39WcKiNrXkjPb46oyyjWl3g3l1onxtZ0qgnRFxZu2S11equ5zoBu\\\\\\\/7gFTC6sOKRNb9geU6wVHQbedrW83r1FEGOl+fTg9L8lkH\\\\\\\/R3D8s2mFNSF26lzZCxeOFjVbl7vFw8e0yXJjlF6L4JOlpkNEEXABfPOj3Xuj3t0t9LYbZGCG9PPnORAzllYfH1tKoSF7S+2IroCj93yGxM1XrzNFRo5pgu+2DF3d0UKeGoeGM113gCGef+Z\\\\\\\/iqysH3x0TkwWva6lrjHciySyoggHaGhU0JWVb+4A58jv3Yr9TP+LdDYiDoaXihK0Nz\\\\\\\/coU\\\\\\\/LN4uAPDjOVYXDZ0H7gzydrUb\\\\\\\/TRMRWeLeHdvdmY4TNPd+ldm49dvnISSeznE6R8uya\\\\\\\/buqRS0jgX5MzHDvw48i0tHQIw66Iwsme49k7jSEY+iXv6IGXsbt0AwGoDnj\\\\\\\/Aw8+MnhNz+MQibHrYzpFFCmZUkEQcxMzzKIEq6fNq+IY7CK5lCVx69D01ENfw\\\\\\\/AHNlxG1w8KXQniYq1yEVz5a2qfWjIcDpSva41gBJ8Z+ADbc\\\\\\\/VL3b9Tw7fs9CaQ3EiDyVV77UYX41uMnfVudPkST87rt+TjjbZIU7QSlfAGYy6bAJQ4oKYy0F6ZCLIFKilVeYxb\\\\\\\/RyfUpTyXgU9U8An3\\\\\\\/4U\\\\\\\/Ty6PuhLbtg2WUQsf+azCkwxdcVDa3KyRtej7ACRkN+fKHDtSkyBApJ32OzEvXMl0npboAdNh9JGmpbk1hGUeS1jCefKrstsoTHCqM6EzY0GOp+Ekx9caZLXQflsQosqLMZ4X5UY++JZMZZiwBajPAjZeso67pr1yoPhQXlBffQ4cn8aBxAc227W6lP82Rs5hLkiwaX2grxfYnpNoOY1iKiKevXu4aL6IQ0asOtS6SLGAB6m0I\\\\\\\/WRbV0d9lAb0aachwo9wHMtnV3rc75np\\\\\\\/G2kcoEoH5w0yCpcl4Eax\\\\\\\/lyyLNvvxatRduG0KtYvc0ZP20O7UF2kUybZUuBSd27u2QF6GSNt7eM4ZF26UmXP2aWJ5q5wbKHnlNzfw1TRgeyottHrS0HbOlXFw8vExv+KGfK85Y1GGaPPzs19mFCYJN4Hl+lkL9yLgRAy7C\\\\\\\/QsU2+UQtias+JfAPi4fdzwwWOJlbOoputTnwSIvX+WyCotoXyC39IoMomG3sHq1G8yYMzx0ZMl2gOEH5qcq3rBwGeeYJCO8I21+3DkxRHwGZPn\\\\\\\/fpYPfGuEmUlfU81CQW1hqhiTbdxw+yzqCRTywONA\\\\\\\/DmY7j9KsTjJeG1MvnCoaJW6ZbtcXUshX7kErzdWPFAP5IcoOxmq8q4MhwdSxnpzwxYlnk+ATrixUSN\\\\\\\/SoZMNXluhhTebgYBrcofIZkA4yHVkMM7FgrlYnT51LGl1JpYwaiCkrcSVppo\\\\\\\/3jWIpatxS0M2QCEpvo2MWJCtiCXxyT2hN\\\\\\\/q8cU+2ISu1k\\\\\\\/nyQ9f6OSgYrFZIQHbyGD6XMx3Xcfbso61KWNH8dIZMyqzOwOXxwiuWHiEeKQuGzBg\\\\\\\/ZOrDkJYi9+wDWHdgZSKQD7e\\\\\\\/9X8Zzg9KlTciqrOCsu+Jbt4aaTXVj99rVwMa3a3gXYKaz1kmDTYeT9HMHh7rOfZ1rpL6aZQ\\\\\\\/dGl4ThYDjh4U6x2U5xvuCutAD09X1JVGHQoTYqXL+RR7Cv77FIyIzOtbSfs\\\\\\\/jH\\\\\\\/fXtz6sdDtNTyWtuuMXByo0LHPb6dBGHvIMDC9XWxwx6bzJlxc5GTTm6oyKhcUK6hMIJi6l4m9Tx47My++eaff6TqNnQteqfoEP\\\\\\\/L6qvC3WOsLt65Ok92VnI1v1B44iYbKmA1cr1OYXBSY7Ej8l3KPYlFgId0hOR861lFugcIXMd9fYkI97jeMM36CDflEEGNc7q7BxxaEyk+r74LjRyh6KBVCH9wj3RK3zdYtWQ2FH2CBZv+dZxCZYKdgabxyNKr62ES1QZbiCW\\\\\\\/Dpngjm+lyHq+rl0OV8FRwNnblXKzxFyvXdaSn8V0JeIm3e+Z6xwdHUpcDyf1Q77TIPq+utRZQKZKcRKDQcFFVVAVpXTkjQJDPMNqjNUnDkk40SXY066Q3Y+m3jIR\\\\\\\/2MkRS5wDzGVFFEXoDKWyEgI7tGYuUGSpPOO4MInDIaqO9WKuHZoPBipDgx0HMIvrmdhh8SQePOEqcdmd9i\\\\\\\/sOUAwQvPuXO18auTFdcmXQXbJ5ek\\\\\\\/3KvkUZV+kNWrmMTWdkKHigRVteK02dIDn9rryQBKNUY1EJTDg6hJ3me7TIAgchs\\\\\\\/56j8LvaAIcBUGzpHtcfGjUMn6lvXLBpS5pAOBcJ9CZfKg4Czt0xbxHFA4+80MI2W8UbQ1DSl3B65xyA9r+OsG2NqComjq\\\\\\\/ZL1DCw7glaoY\\\\\\\/Gxf\\\\\\\/MUVxMr6JJXhfPwPlrLLKInwP9hU9h39gz3s0tEOBAmP9MvY1pEZQ8bj13nV9Ch0DnP35aDwtAuqXkNWuaLyClTDsF5eHNX\\\\\\\/mlstQids3K6YC1M6Fn6fzSD9fYCMxuPo\\\\\\\/aygan8\\\\\\\/mbauP5l\\\\\\\/80hig4sWFqd8EKWUWRGhGXI2SZbqp\\\\\\\/tPd9lOKZfmIrPnpb5rpeGXpdCgYdIUHw3o7H09t+3KTDw\\\\\\\/3HLNgCd8ZkhfbzHWUkNdBpcxvdJtUIBPRCR9TARtm4Q3tsiz2HO\\\\\\\/6ylZO1iM+cMuFM2aZfTMo50b8NwSunpo2F8OaT+4YZrGcDlDiJFxhsD\\\\\\\/mnlLTX0P+xoDS\\\\\\\/+0CDLND3GZvY7hagNyIvrugMJP8wq3vR\\\\\\\/7Ux3jnATwPrc2Pzie7uWsec+itf8sxlP+t\\\\\\\/UbbF2yWHxQr+hPZSd8V4pSes6ow6zDOYLbjWHlZHadX9\\\\\\\/6txO6vWsXmtDg1ktk3INe7yyjW84SQQPGqKU\\\\\\\/NytJv7DP5KfOgwrXYk1LxGbMHn562Q5bQ+Pfy9ZjcVMc8lAxqhWMcz0QZ1vf7lilIFCd4e\\\\\\\/JXdkxAcnEDGvkGgH1I1OXeO3bkJ5SzfRr4DcB8CR59gwVYxatrD1f06GrfheIDHx6cSg08ycccsX+8W9yUm5J+iVUJWfFYu7pPoP2c4mEWDWea7XKKyJkVTYjFvPmdZao+ummlaGu4YHEIrBoNzLgTcy4ynSJRAhhq1b3UIuN0RiRjCEheu4dTFTol5FH9AUy0wY4f81ViZ8\\\\\\\/n4mMt9FChkcv7B66YniVTh98i\\\\\\\/CCVlnBM\\\\\\\/oEMHyTCO3isMwiMNigO2XVe\\\\\\\/Pv4PXO2JF2F6Lb+RrQeKsHJ4WqeoXXLGnoakz57nXlki940IJxR\\\\\\\/cBeRseO2iVilEC\\\\\\\/GDodvV\\\\\\\/oc8WbxKLLTIqeqHtsiHRgjlF1JxGu\\\\\\\/Jf5G6NM3ogY8ByaUVgUSyW3td7anIsQ\\\\\\\/yfQMP2uvYO6JXANiQNVzo53\\\\\\\/ZKXG7Rua8pMHdPOU+t8yyK5WyudzlNervjrdkCmTCwI9htufuHKZn3mFObkpRQLotwVFTKrVkkHihBn5OshUCytvk9wzPq+ZTP9wYCVrEJbt7vehQltf3\\\\\\\/0hRiZaq7q91r0FpbjyA8Kuc514JXKegSvadGVIeUFLF2sRHtDsgB5hh6DWkowjIgwe6eL78LC9RautjocgJodFxCHQsX\\\\\\\/4N\\\\\\\/3urgeUVG1eqIw8\\\\\\\/TVdVKBJMkKumuOOISNaFuKHb2ZPuI0pNR8Q7tn8XiC6X01XAvMyFduWaXe36WXRN5fUNwhAfhXqC8vKw1FlDhsqLJuoKFhUN441+NTmU0vh1Nrya1K5zd7lBjHcdtvr56fSv4g5n7EWt8xAwEuv7uDJZ4ymWjruNNb4LyTfnRA+Juwcv1nqGLASYwRlkhFM6vaIvwwa8y3IEfGaMuFZNyR+eDkUCpczLsUHZMPdcKwucyxLan10jMaAP853W7FiUv1qkDMGTzA1C0fYSxECiUWWAAXXWr83biGA4Ra9mEr6xbIdtEAvqIwNdzWWNQ5b+BuslYLsM26ml4sRIUvK56ySqOBWUDkrebHvImJ2EbVJ9Qvi1uWeu49TIJoXYfXPPF44Q3e2r\\\\\\\/iQqOmZE77a5a5X2PbudJcNRhwfsexmrzJWW8dzxLrC6sWuZ+mxfpvcPYn4ZiNduXP8COMWAcJ6iqmQVgwh90HK3TO8sw26KSIpr9wa3FKPZKFSN6PHsjCX7w37Hmx32zGIr677H5ra8F7AYB0PPUqn7VYNuD95DZBeutfpKiFkyqOFDTvQAn06eZkLN90+z8WIbcHxwPNsCt+3UL\\\\\\\/sVYGFPNz3Z6VgbSxYSZNSAd3X5aeaMFawf4ywUG5yrSENtk7G8FdOkNROLJdFWZfV5bxgb8kkQ89ggndE0y\\\\\\\/QjV3GOEWIsl1gQvj++b3Q1a5ulkkrF7PF+4tQQfWSdMbfEj9R1i5mJuMwBwegYDvjHS72WZG3f31LFrFEZOPLULE2wDPDUryzzSfzlhfmfld9Tb+QkjqofjmSSqIIQ9ZhSf9CB+Fl0fUI2++t1q\\\\\\\/5f9++1LM8N2WXN595I1L1Tyqm2yNq4byQHa\\\\\\\/H\\\\\\\/3jxmxBaXEFT2lzPQ2RPEx2kxe8unAJjl3pnSZAbi0iHD\\\\\\\/6rjIdZfbRYevMLFzQ2JdWQBMI1TFrpw7d\\\\\\\/8sstaj3iLUj33vl4Ux2hLgU1vDbPjL0vZXJWojgZd0CdOq250rwx3i\\\\\\\/hIbg2FWSeslugIuM4EYsO91lLFKYYeLxcUkuGdQGvPZx6+lYlWxbkR3q1cx1NpEIR83HifKN1XWXRHrk0IwUCaFmvjWq8dOimqD04BKtx8Bv3iS6\\\\\\\/oOpktvQt9sHtwjLqsj4UWd3iMHWNYb43hskBlXbvkAgIlizceEOoAg6ekfS9qFiFHCg52pg4CD\\\\\\\/QS8bSupIbOuY+OeUFJxx4d+tvQiSaU8CrQZ8Lplj8Ddbk1+M667vXDnq3bEnrSEAcYDA+sCvR1w+uw\\\\\\\/C1R8TSzQW3YQTxfyhA\\\\\\\/5CL4pQwjWKnzA4e6cMuaAy54coVsNDJecfGw0BqESwOIu2Ypg5RqPVCDUjOrRwMVLn5VfCFPJahGRdROfZ8EM3+GK9ACCOIzmwCNxm6Bou5xerCLe2i3LX2Ug0n46tiKAeGj7tUJwYJlIPrDwoADAKkxGoTP5s7ZECeHTcQu5QDv\\\\\\\/oUxsiSXYxTptTJZ\\\\\\\/prLC3FV54NW+2ZGc06T0ANtqM9Yar5QBiSWWjlJG2tTs6FkRAtkvxih5wwyKTgKEsLsgYz2BYHvosHA9C7Iqeq5qEve4l6aQT6Ui2+z0yYCev1krbh0guqsPrsAasf83oueWjQ3ROzUorqiDawof0dq10UVI2me5C1vR6xxwmHeuNSMUg0e2axH5twdfLNUG03KNDMLACGRIab\\\\\\\/itNdw3E8FWBnHoctDaBf4KpXItnVtUj31GyltNuXNy\\\\\\\/BfdC3PKMBKuaeldxegFTVvYncQYBz4DZI98kcLCW8W8AQ12Ry9E9nGNqLSmVEcjBf5VLDsxAot7K7R86B8LVJYiaIerFysJn8W3+jxkRRckhvdyljsGOAs1K29FteNsG4aeAH2mm0OMoPU2WhxwojQ7XAQJtQMvF5\\\\\\\/j7PelJYXR3iAUjuiNMrfXGnp+8x3xKDGOVFzZuGqLGAc0qBdjOYYuv7n9QfEv8jA2ALQF0EGCA0JzoZ2RXi1bXkePzCMoQnTxvkR8x0Hv75hrDsTvQQu5knpYk1\\\\\\\/cMTFS8vtdjcZf0SdrDuJlc41c2AqGGuPVeKFCv6BRsZgufR3J\\\\\\\/j9WJThuPQXj0VxDgL4NJH0294U2w+oEC0bgxCG+ocQT6Ht7t4bBblGJCyjxRS7xvjnmh9f4sQZJbaUNuUmHWA4M8xMOT7IxKfx8OEpOc1T1YXjRym9yloHO6rKKNSLWFuam0HS0r23mvEvP\\\\\\\/wLCQ\\\\\\\/co0PDrUhpzwz0poJBbuMT1HOXvZPO9u56X\\\\\\\/OuO07EDTvAYcZSBRYUgo1YD6tvyQECTruG1zzcxusz1kzUR7m5SFUDL0+iFQeiNnvv4OBLrGmZY\\\\\\\/y2mPsDNLshigSoST3oEsx9verENJJ5IiZgmUTkvrFlMSSl2TKcaXVdf8D7qUdwiHJFcVNZ78OV1swAnJkSof\\\\\\\/pm9\\\\\\\/NuHJM5z\\\\\\\/gYM4HESxtFH5fwgw\\\\\\\/tE6NnTE1vTwNaPw9s5dvR+26UkYu8j2TV9+8qJB25vDEoxuL+4dXSSt45QK1P1ZdyL9k1erQ9S5YJAkR9E\\\\\\\/lQ0+wrn+GfflW3GUbnZnlPZDEXjMjXd7egvKB85IT4iU7RIE6GrIRgQLXdkVNvbcJ8qEPBxwPh6AEbEwO8WAdtF4l9DDoJeSx8PDmX\\\\\\\/mJct1xmipeNP6PUeZF\\\\\\\/UEK4YSnivZehMnnaUpFmaKWugv\\\\\\\/28QxxKL\\\\\\\/MW27d6zZFxDLFWrAPzJuQ2Db2tJfErWL\\\\\\\/UG7JTqy+UGifi1eqKYwSeI+WmHD4q9MRCYKmv7SfzVhiMX6r4nXeK81h\\\\\\\/YOpqsSB9cWAquCPMgw5M8HaZqzDJPkeS8ItZokqRxohSDwPeMHD6E35e9SuqgkgtdfVd7ieBJLco03y7iJNAGpxQAPI+lcX0Eb\\\\\\\/UbUrdQ6DI2DuNJtqnEfcOTOGWrkGFd6k3xtCQacY3YXmT1jFG61KiO7kG8n5+m44GaMcNCGRSKTUHOHhpVqtx1sykQFJInCpng197MinmdWCh1LxWTSPraF9vhfDkjWBqQ\\\\\\\/EK+B7NPUoPvPLq3SBuOwEGeaT3k16KXIZBof33xX5dNC2AZog9A0KvMLq8Z7tLn5\\\\\\\/Mzb2LU2yZUfFHmJhpD08C5ViB4yRU6fApz1GMpC21vAwUku78ZABw+a0b6k4\\\\\\\/gOZff7oD3XR0jkvkTOtiNJt0chb+OB9U22XG2Rf45hsVYqH2XRjL9H4lXICgFbn+L2E2GCKqDNHH0QZQ1clAGcvw9HJ+52MilsH6Nxo3Qex1bzTXBzExWpsH9AiXtkjMYrNliiYPxFWdTjiMQL01PZNvgp\\\\\\\/3NF7wj3tk5b8FpRKVEmdR2p0S2r9Yw7ToBzlcRAV03BJyit\\\\\\\/jFXbQi3Mj+NbwVkS3WhQ345IORcDFG0+EWrQc4nY6beC54Fq0g8YAU840mn+vvVuqFoZjckWhegn5XeSl+AveOVNi2HvT+paV4bwK6USJrMDGAXGFnz8+SkseCg3wh1Apfry7yEOSTmVT6T5IndfQkFY57Mics2yDKVgCnBZIZGvCq7Wg42yxHywSHSfCbDcLS1lif27MD6VyzIWUSD7e4lbL+w9pwwPQ6YlQdtzYPPEw8\\\\\\\/UeF6NDpKI4WB\\\\\\\/WXnSHw8ApDJFK9G\\\\\\\/rp3ddF8SzuepZ8z0y1H6t\\\\\\\/VJCO7oYiIzprR4ZvZDNNji3xxoeqIrFJZWBo2CHL6itd1peUZtGZ56mjB4PYpm7g70x8xlca6ws+OWxyGS6AG4v45HeecGT50RDdof\\\\\\\/rhxHXBNOWz5zSlgSLu\\\\\\\/9ILDqTC9TP4f9054d4A7B9KnQklwZtdrvpbtQF1WsOhzaRbyEL7QFbMbNBihbElGPgg+hFXV4ixxVhxJ1SMkhqqjNz9ba8+WtRUTVAEMa3KEGepWvjAbKZLYeR1o+XoDDXeNTZN\\\\\\\/xURGy9KtP6RS5EfzHroGtY\\\\\\\/fhvLoUf\\\\\\\/mxO8TNZrrKQxWRbV4gaX1Iy8F4x1QZ41H4j\\\\\\\/KIJ8eSWluQLMipBRkR1DVI8iwVBbrTgzsp9NefKJGrk948vMzXK8eEU+V26OCoCTgkNBXH5IXvwouX3BRWAjvX\\\\\\\/AYdzJfa0BxYuUWpiBOXj0ZGCtuzsH8UaVoR7k4I0m5SJu+D8+F7OpTVwWE+4x\\\\\\\/8CJvDhEfXKp3J9nhqlPiHMm+sZswzG9tOPzW5lK\\\\\\\/VP0jYywZZe+Is0ozJcOroGDP75e\\\\\\\/LRLlSuuiWCwbhl1AHbTp5iYXnKpofTlP6KImlVQj\\\\\\\/lVHN3Prs4uMD97ESofJkF7lzb+Z85KxeK9kRJWVrtPyaS\\\\\\\/q14xcKfBPAoR71xeO4lVE0kIwZEA7MduG6\\\\\\\/Ukk7\\\\\\\/QN9fMNLTEvOi8dZEMfme95ZB7U4fcFliy1Vh7wTgUHsflehu1497CLgJC\\\\\\\/SaAacOIjvtcczv8az\\\\\\\/vfo1bacXU\\\\\\\/JpmGETkorCkHa8dwAFLMv7utQDdEeAAqGELNT8wrAqa64gTvhQ9gkDLikBPUZ8Y6e7lTCpCyjllCLrd6q4twXt1n68UC5EzLeQXChByhm5B2PwLpp\\\\\\\/mgC0O6uP\\\\\\\/XwbvT2JFA+6fXzUJaJbCI62FkNtT9nu99Zc4QLvpxHYAtv6hLRqKaEDgm\\\\\\\/r3STwwMMewUovN\\\\\\\/SnqA1FSrwCD5XFgdB5ttcaj18upgSyI\\\\\\\/8CFjs5oGMfVQxHQMeemm\\\\\\\/zD3b4HQ0g98hwU3pnpFW6+Z94\\\\\\\/j\\\\\\\/B4x3yQH0\\\\\\\/zcoftoIXaB\\\\\\\/SNmOXKfP+FC4iyNlNklnKEI9o3rEHw2WKXl2NwX5cfNUjgu5RAHCZjzM2b7H\\\\\\\/b5uC2U2FPFNRkmhRYP9K434q0yAIfP98Iym+rZOXh1CWBKTqd1gBAcTzHRT5smGb2nA8Vv5YjhmHUcsA0tAWIuuyqyv9FBW+q1jzmNg9L7qW6xcKJhEnbUHNtCv1RxpkBv3eZKzIuJ3toeMBCX+J+UOaEpScuiVMUViM93ULH2mDb4W99S7REjWtWZIVh+RX84RjgO5rtsHniaLZaevvQE116M81LCjFpyH881668Nl+3u44dbfj+90GYeYTVZzJF+ChCwC9U0IDQnCFG\\\\\\\/SRltw4EBZAJG7JK8xUdGbfd5EoTrICODizsy1fEgcR2TF+9ippvPiLYqbCc0GxjcRHUNN\\\\\\\/eu59J2ihTJgQtdAqarVBKT4sgf6l\\\\\\\/PNXwBYU2UpamNuwdMG3ggP9d5utl54hmFBuOE7AFoEk6KSyXJp59LXVMlZ6HQN2+GOsnBswRgo1FesxUKauiHj0zaXyNUASu1KyZf7O40pgWC2IOjKg05ZuAgt1Kqh14BYZut7JQdrvG3+RS2WrG28bH0tK37sgnSjy+WB2lQNozz5uBHTQTatIiBKhcTtPRchl8J0YQ+Ete6wn+y9EhOd\\\\\\\/BNX77EfKXZYmiMhPDyqWJQmmJcB8MHRksAc+0SwaJ9lIPriSLgetfglqKLcTdGfSCpgz4v4BRztWygZETq1VcI84pPGDlhE6tb+Lgrd6yX0L1TgQcb2reBpitNSIA77oRabSmycvUc9sKtFlN7YS1xpPTJj\\\\\\\/7AiOY5vUm89Mu1ssE835e3gD+8CINsHpJFHwCM6ndfNgowZHTMLqj5LpL+oIA2kP7F7x2nvdbQdrfCHPdZGTXfqDmNiBoYBsN4zQqd6rFJ44yzZQQVHMTlAFt8Fx6cf1PEo1c2e\\\\\\\/UVxQ2sJNBW7uEm2JLM\\\\\\\/ExCeGleU5e+Q854KsGCnBSQF0Z0khbFm5qMSJmTEnIfFdW9+lyQMifCX1yhdTcooEaNAAtjFiMGZBnAjCa2n7pFynVNZbpejy5jOzfX71Eo5qEzQCf1MTnFp1uZzdk5gN7Y1Nu9yr3K9L2a+dZcau+HvbpniaWLhmKDbuWQJ74TkrP+xNs077WaaCULV84QNOvMMexHugXLhqoB9bh6jB+3sU8allFd8H+V4jDLh+KeBTmGm+2T6L2H0HKr+iRyf2Ge2cGgkALiAhUxczhzJjedqg4GllVoH0zwwPJ1D1jlOl7QW2CyIMSroKoKTZ440ENPOvqlAI\\\\\\\/UNrd7tPQfzUQlkzmlrZInkB\\\\\\\/0JweUn7TvuoA9EK7b8ZYI43DH7seXK7iBRNgFRr0KUbpBvOct7v91pHlOEnSf3We3reWYMo9UGEcpAYcx5idRnMPQJh2QJagAcofuhZX6AjqdDsmqGaJSRaYbi2mpNajhbW72ocR1GtIeKWk5jJHjve5iVTh6rc1fQdTxuoz8dnIJSi7gk3VUV6Mm8canqmM27\\\\\\\/7Gip5+YCsMS1TAC+s9T5r1JKc5y96SJVUnlfdCF00wdVk\\\\\\\/LXIakB1IM3UZcXJP1Ny2bw\\\\\\\/od3YXO+xgugzCYfQ4654UgVKC5tdgd2QUZ59kQjiuufLXkcuEDXAov3XOqO6MxyGw7i1LVGijvztIJvVySFPw4SSW2PuPl42KqqS0wf\\\\\\\/xfEaJs4Ri2INF8LMWK\\\\\\\/X3gfHNae8dc5PomATfRDkb\\\\\\\/sGXRH1PM3iMfiOuGYSpvxpGhoHLFSomleuZWg8DpYAJmwMnzpxS4b077G0TO+e0AWDaC2QftcZy80CuxzUKN9333oE4YfNvKsNlzd0FvEZQQzFRxtzO1J+8NGXT\\\\\\\/78SwvFNuTFaj\\\\\\\/QJq0cfjZr5ZCWSJ45fszcgluOmy1RL3UMH9KHDFHj3QKowdSCTXAaMd1YtjKJoKW5HvKQhNy3EB2YeG2J4k8fBezw4lIU1OaPtbA1ChXXaZ1eGuh+SYHSdceZJRPp250Suje\\\\\\\/z+LDubJHrvVcUd5MQp14NrskLePeMebvyBY8tY8KI\\\\\\\/IbJBRo4n1LILr\\\\\\\/Df8aQsM6oZlRFgktns4Zu9TZeyQ16lYSKBZSxoRAsAaFQOHhGodGtx60MMdC+SYDSw+HIbQY6am\\\\\\\/eP4w\\\\\\\/96w88RyWQfYpeUjhtHj9W3fV3o0oclP2xWDLtG1fmpB00t7YnVKy9HDUwNbDy2vSzsLzWgd60J6h+Pq0th5WNxNVidMMwLj0OlqnBumqItHQ65Aocxdf57gSIwDttFSIJNfz48hjkUTnsAmD\\\\\\\/\\\\\\\/+K68oVdb\\\\\\\/qbZuuivNZd6bcyoHRdXgMpm7mIkdQ84KB5nvXsFZ210kzeBlOsoT0pk2M7iLPAeX9sz1X7xIRRkw+aEb3nuvjAFwEMXUFuGFVfK3q1F9FyWTR7umJvOnOQRWqRgMRAT+h1dpVy\\\\\\\/uCkXgTuIYW76uVCd6SRXns9N7Pd0\\\\\\\/5Is+\\\\\\\/gaJDcHlJeo\\\\\\\/WvNfZ6QmtJRMoLL8rB2ngt4p8C650AQikEIVgl4fGT69E6XDNDExQ2Llc0vk72NIt7mbSDF26H01CsYm\\\\\\\/k5UR2DkXkg19xcQ+mE9ilcGRmAAAos2OafwsKj8WOgnFVr2siwe9QzqYx1\\\\\\\/dO+9xIczWTuDShj\\\\\\\/iDTMphFQPHiLjQbBjByilCXxnyL\\\\\\\/UCW4zC+Opw\\\\\\\/2nLiY07IftF2DWDHF\\\\\\\/slaW+CoNTcmiSAczTz85LA3GBJLEhIo799kXHIl4FhWuzj2MB0UlazpNrBhkg2hWTg\\\\\\\/eH+VTQdnYE75xBhS+CriIIPUJfS7wrfamK9RJYWTWSaaJ9FQZY5B7avUbE3zH\\\\\\\/GZy4rGiC2qY5AwWJYGao7\\\\\\\/rUetzwgFzXpdFYyo7pHovd+wyUvBKCnL2tm951r\\\\\\\/ADbKjyqMk83O6oi8kj59KnMuT3ovkzkkmRO2DIwEygVYRnLEocryoHFRC866NzhON2wxUHeqSP09+nkf42W+83\\\\\\\/3+etMtHfeN41WJffctbXeRpOQfH2byRgSqjdZykM8DldVq5nYPLZ4LfDLBZ7LK+xaWKSUiC+usgaInEOLRTwobq31SvYzAK8SruVyaEx\\\\\\\/Gs9BRXx2y3MgzCNX0wJA9P\\\\\\\/QXX\\\\\\\/gWIE88HOnsv1Hv98nypfC5zpo7j8FOaXvSSGa17T+GlupDJTH\\\\\\\/lu5nTmrN9w3bFc++IWrRGSi5PInQRqqHwAGWpSO7pyGPraF6D0FcqKshrHbTMhGx+UV2tk\\\\\\\/3OUcx3nZsJnq8fvTK1WgNqqHLTzvdIIno7RBrwM1mf\\\\\\\/FcY7LYJFxYhAf\\\\\\\/\\\\\\\/8o9jN9XlPJ535vOsHulh4pxc8m8K1zA11ZHBbOhtKIS8gbnrVd0oLJFXOhrZgVUvLCEEMeWejb7TSJKEb\\\\\\\/I6k2Ci90JCwCRDlaAUBt8a3Q3nDDS8UDmOgKVrSUR2M9hZFqtq0dS\\\\\\\/+aBPsijJMlrDK6ckf\\\\\\\/i6HwkkBhSMgAptTV\\\\\\\/lvPYe+s\\\\\\\/W4C51+Gqf+6NWUt7gOm4LbgGwkiR+EJswB+BDUaalydxaBnWLEo\\\\\\\/EKi8va+oSMCWGWfh\\\\\\\/UraYaXwMoevJsFoxnd7FIM89UbrFUgxoGfNc8QiVvMZDS23xL+97R2IM2\\\\\\\/0xnmwoMxJ8Ylah\\\\\\\/sTYATXw\\\\\\\/g1nDkX1i+ZPQq1lSL5oV5ojFAyX7xESGLy5dU5oP\\\\\\\/baBl+n3a6Z7+y4X4o5nTFjYHxwRngTZRrhJp3VeHP86pH3z1oas3LhdfGlLfqKMiz4+TOautnsDICXC5fW3JNao6ygFvwpSLYdAf2Eed7kH\\\\\\\/RajaUoRA2Wzx8lv05gqA20GBqZHY9H345lf8d7BCxUVGZRy1CbEp5SzIKZ2zo4BT7N9cKzplcwxr2tLoLar2w+VUifrpfMf1snECqhU9s6ovB7OrVCsJBvbWRvnt8VzuHl1p2khoNfkBYYJRh9z4nb61lMCPX0ir5xebz\\\\\\\/vJwBCo2IsGyO6hQ0QjE7MkUPRnSR8MOzJINaSlSsiEWyhVo5rSGDkaYC+ndKb6v4MEKOzZahXvuXsYHlUoRXO8BEtWyu5JnHinJuNjgoWkyQHG9jR9R3Hq1i+IeolajQcs8QqPpOJkVos795LcCP6fw2uIQnsYPJgia10b4wl52sml7Gzbbb5ZwrFXyRnXGz0hYrYhUXJYRzo56DcKy+mq6zUdRbwp5Fqbg3\\\\\\\/q3FZqNpUv2pNc5wMzAP1kzFu4xzLamqZGd+r8mtUzWDRtOIITpitGrN0Iz5wr99UpbHinnSi6MnUjwjuBO+\\\\\\\/bzujQs6aXpEOaaw\\\\\\\/qO+IqY+b9Xg75yd8L9t6fDOpyBxnnH43G1UkbFEfL5Nlkn+Sfe0D0v1SZx2u+cHHOnhc+YcNL2vJhanyPR\\\\\\\/UKYT0IFHSyPlFc8p3b51mZgD8MtXFCq1cK3Cqs0UF9xU6IrM2sDHwrW0yy3d7FmWSTmD21bl7iLAppg9idNLOksSLvdkmXphNzrQHngv9RSo9IwGJ8okYtLxwAfr3rMyJdcFnn4UCdCVGyDUahWUhIO6v4uuP2aqBLzTlftvlVKPAipOF5PEzicIFJ+VOAfursogn9ruFJ0C3NxaA+hjZh+Go5hQ+58e+RrgUUeD1XSa9i4rUBmV\\\\\\\/GjVtGz9b4W4lKJO0Fk1TK2yDTw6ZDspdBVsgEMwJfnwzYJfWUEyjLGd8PyDrbzkcaQvHLGdR+SLa+X4AvfhwbNXcUL1gDus6olqtJac5f6UDFX92Ylu6ExbH\\\\\\\/7+nvq7nCcxyJXNHiBQ\\\\\\\/hpKYxJljHe9COGzdmhzHZwvupnvuter309bgbdwQ+1WUdlZaFDDXgqseA\\\\\\\/Mpy6ClMxwUs8V8tx8oOgozkPZ18ur7rXWa02vDmxYKsgGQY\\\\\\\/fY\\\\\\\/1\\\\\\\/vyAbSgvQSdW0WW\\\\\\\/QWBhgbAJuHK+er26VDfuvGR2WtI17fAGvFlamN0yAi3jfPeF6Zh4\\\\\\\/+UTkCvKWPPvSnP3IAlivUSS6+p4p5zqslm0ovruv0gjgOBGuoRkPb2eQ1BIGFMKnHwIG\\\\\\\/JF3g+dUoCBTpLyhVgRzlnPvcWI2Ed7fECeaM3fwgw+04BhexfBIW\\\\\\\/eG8pBR5FCm4yf6qRUNHlGFWcQZ0p68mzPkuosQKxv1mcQHCgi524BpiMVgpRvrZC5b0FBDX77NNkxQjiKjX0l2mK\\\\\\\/pdvE+zC\\\\\\\/gSJEq91Qm0MtvJGanlHbBxD2Ft+gfzHqoSpc8mKKxjh9UsATjw\\\\\\\/qWiGljpm4a0Pt5SLCVm9iw0Ymt1QemQpkG5+MUhW3JEiDbbFH0AAw\\\\\\\/FnkWDIsSHt+I0k+HqjTmGPNT+Tzmu5hLfBjpMquvhY87dD64AxK7Aga0uPSOMlTJ+s\\\\\\\/m9fPxZXkrFdOz9wTZpRPAq\\\\\\\/o7OjjN2aX7JJo3qbL7b2npzekur+z+RgIkjxhChUFOjt9ZtqVpCBbinB1oAnS9kQ2nW7m6Iz1sWx1WMG\\\\\\\/i9vCqp00N4nnvunMOZsgyhvGRi2BaJj9wJDTtbi8dWwSEL1chuXwoz3\\\\\\\/WBjKA2B\\\\\\\/tl\\\\\\\/JwvDkN8ccE0+yHJYp3Wy93D6nuXxhMr4g8gwVRfH3Gv\\\\\\\/QO3CDz457dM2U57rfTod9ziXlhkeZX230RiwYQ+5Wzt+P1CUyiPLaje2d10wZnjnuZxCFJ7+dMSR2OR80g3ZBF7owIYqhPyzJ9WKCQwlD8nYSI6qE1SHIV6+P+LDBuqmGdPq8pSdUpmciLpvOKYQLbtkVwoPli6GU\\\\\\\/Omn+fS7Knrv7lImqOyZ01j0FPM0N30v4COldaKfPZXfGSCOU405t830lqBMTrvOMDNNBMVvwDzardLgO2bUnApiPCh6iNez5rc2\\\\\\\/zXGP26oMNVEI3egbfJYmfwnP073B7E9WswZe8Kvvhf08B\\\\\\\/+fBXql29o\\\\\\\/zwBYqY+O\\\\\\\/4VQ5Z8XmJEva+hkvT4fDMXBF7oHIKACzzT9758U\\\\\\\/7TGQRGPYDu6DRKr1eELBLphZ5quIKyiS5vRSzyeeMuU2x\\\\\\\/SFyzq\\\\\\\/nWgEYbPMHCJep6duDBlslfeN78GAvPPDi1WZ+hsKyiWYVcMhhgkj1eAVPp0aHBcgeWj\\\\\\\/+wmhIcG0g0Cx65VPfUUChsSajaxVPihbr+VwvwqocObIT+sE+Rvll95+eaUBVSOa9IVIVCW0VO5ivJ0ClBB6DcPznIjQaG0swSYgtQWheIrXks9PAkJcBoicFTcSOhZ3iELSK1uXCKLOW0yew673krLMx6Pb1XVSSDqcAqlRp0xoR4FN40we8v0iPXzYHWFMt6A6AwB4ikiMW\\\\\\\/nj2zGIICGh3WArQKV0qba5TQZyrIsILWwJbkE6U8qhMraoz4jUgS8TlHFHZ3imJFyxJw3lQ63BVxj0zgzUI3otbOeufdetf6+xhbk7aKzNQj+gPEFKglmTqBnPZaGtFZIErny23Qec+jTOtUvcJfymUn1brLWm6AdmVMK4ssnfgWmYEA+NxHuV1ZlF+MgqhViDs6iX+qTpprhM\\\\\\\/m\\\\\\\/qYaJ\\\\\\\/hu2k7WEGdEwh67xhAC2Ny\\\\\\\/RDryMJjU7rIDMYkKur+pXfPa8Of+QW6gMxTd+w5YLOfbG8RjSDvR0D2ckcSKlrLeR\\\\\\\/yzFBOtw0RH6cmyTlYgWKeeu6d2GMGBe486P8M5ZOgLbzW0M9oQ1wFOZTZv8\\\\\\\/OdSToynxnE2E+giSQ96xNOXrV\\\\\\\/ve7zff\\\\\\\/+uwr\\\\\\\/0thZg2Itw0wYTVzeMBNfdsIaiuAW5jfqVW\\\\\\\/3LFmRrqYokgHkY6UoYY0Cn31OqRP7BFjTL1NoQ3czj6UD8U0VQ+fYgzG0oFHH3Y3kvZG8UchyYpxSoSUncC+pOWpfy+6t2IGRKgbyEWft6VXWGFTK4+jLBvNIaCvrYUNiUi6Lw+XtQTyQF4m4ESI45GiarAAt5Na9ok0\\\\\\\/b6k2P1F+IxNpHEnkNnPUNOHf\\\\\\\/Nqw0Mjtbx2aSUJrfz6+GXjEiDRO\\\\\\\/dtGi\\\\\\\/B8xInJtos9T4rC6dbsUJoO+oMeZQLd9FKCJP4Fc6P1eKudNoEoOF2NC3NE\\\\\\\/+Fariekh0xSGFzah7HeGZfi92jRt9mzlVbZyqtxxk61bzVPEvr77kGba7ZVUTqzRYHVO1czya9U02ZNSzOoPTffIXYjYrgDNkGSpKdJ3PkZaK7pBMmzt5WWDONO0kPDL2bxT3qjFetKWQhqshWAWF0\\\\\\\/tb3q54ew3kZzKm+rUbBPF6OgXLIO1TaIyePOlR5UdPqgItRPkxMOwtd0BIYla\\\\\\\/1zh2JMVf2aVkbjVZYG6x++iSkbODArtAJxJ7mt7YxmHm0nZ1LQi12rQlwYunRZYdPTixcvNH6Bo9GEZiHCPmpuGzw1vgZ7uDu\\\\\\\/gBc4hD1KerDAf\\\\\\\/nJP\\\\\\\/bdNaAIB2MBqAj9R\\\\\\\/uyHknmRiJk38isp7G3kBLrQ62uVSZR6CgDuR1j9eiyieo3g\\\\\\\/5a+Wv9cg+V4Wj2UamBujYATzOF+k8l8drik2TIYQ1NVe0oi2UvpV8QOqv58SLKGdW\\\\\\\/AqkJ65070ee26MZpGLAmhbeRTnNqU5o1UFH6SHwJJEUSXtXvw8kJJaNLCI13ABFX+W2a5iNvIkSMshrEklZ3uDXHN3OzDjD+W6S9sw5gkifJwoNVh1pInoiQNJfqocuvwDU0UJSwctkSPDz05YtBycx5oTJjP2ntw0OrMSTkO4A8IeTdIv6b5O6EUGN4ami0QZt2USUbb4tQ9dnZePhqHZTuEk9H3qZtYXcpFS8pPfZCgl0nM+1t\\\\\\\/FwpCN8ieA3YWBo9rQOwTHHQOP2hPc8si9qQYbKTzoShmWFQ0+ZmRvcrDEGADam6gPlTGWV6rlgvIyJBHJGA1p0yXv6Ebo77i9ZdPtQMkpu9m93LUEqXv9YKlQSGigs9UB7YZyYGr\\\\\\\/WemNFgtxOUe2SrLxYLAwR58CpQ0qhY270hTm+fm5D3IIsgatEWv4iUSa\\\\\\\/JufhQvRvzsEjMXHe68rV2Wc64UNOR4+I02Rdo7h2kWMDJAS23tuXO0gLYLO88N++\\\\\\\/nR8dxKkTRVjFtToI\\\\\\\/7mZihG\\\\\\\/v7LBtq4V4j3k7+sYMjxQ2KtZxRccP6oGBWigmHmlOZXiyP\\\\\\\/AV3JWHu88US6T9SFyP8+y37XLeg0bpc+U\\\\\\\/KAWfWDQFLIkDAlOWnfbUV65J+\\\\\\\/ExYD9iBR69REvcleEwp6AXhPWWGrOABBnRsDk4wCPDQsVasrr9aen6ynilcBjGS49paSAIkIdDSTWU9ftjgjULFw\\\\\\\/12vqDVq1xc0nKyUVDNJd1yGZrHXrRcncpIarJ2+QS9a0fPIcWleOp0TDLyiyUGrZdumTL9CcZim6tmxlfoAcgV2W+d9L42ZOJl\\\\\\\/nLGYhDmiRGUArHMk9YC3\\\\\\\/2VNWr4N15m9b184JcS9fAfbddcr36l1YgGCMRYd2fhbslfL3oXj6UyS7+1LymXeYbngeFUETkucgdqxNvoITClZWYtrJMbwweZAEwpTyFNMri1id2O25xKtW+AHHtBYRimU48YBXNJTW47KBXJRa5G6rCx9N\\\\\\\/ayEHpKD8oh3HTxpsJNZ3ic+\\\\\\\/ZpuJLTweXYZTEH8ZHpDfxVXI8L265dd0fN6fL1gdpTrfSB6+XiBEqjLyMkQ0KBM2sYYUGUOEsx+oelUj62YXaYfJ2AojlPZorjlAPuSB\\\\\\\/U4RS6gLjTKe0s8\\\\\\\/PV7GbhadaOYwUd2kmr57QxkqinRKOtRJoh9HWK4lMOjKy87UkwCumhn4y7Ctl\\\\\\\/2ih4VkmE+FcU8hAOK\\\\\\\/GDcuLaK7sWn9VSoQvKJsULNmnW5QHccVyMOqKnkMZ+VoFSc9AD+J6ZvENrGZL7cQxQjpcCY6drQxUEIeK+Mn3QoVDnG+kw6znAwyJkCuQtYMxYaYFgxCUQ0ps7Q2b9hVu68Z1pvgzF88jf6vRYSWPyfGW7gwDAVmSj4nYPncRO\\\\\\\/fgoVEhJJR\\\\\\\/XW8X7zrisN\\\\\\\/i5qR48SDxCHXZhqS5XSW\\\\\\\/O4frecV9rULHjdIUMUHVF8aK0sJnOT0MXpiYgIMhnC3ncCUK8utd3gO9lCyT9GVH3r0d\\\\\\\/YCucqdssd0co+0uAOnKMo2JxhtO4dU3hof5hEDMmSuBlH5vKc6jDPcJ3iPjUfCMg5DR4aj4vUmKjuw+i5OmmzJg00V5sZ1x3w20mp8Wzti1mKGip5WRO\\\\\\\/z2KUiYbAYfOWu59H41GNZEuK3w224OuPUWizqH5yiPf\\\\\\\/4g60JaMlGpN8jU6iQM6TEpRYs2D1kzdaWkOMhAEWq1QnKYmZesGg36H6lAOQLFgG\\\\\\\/J1Lv+vyzCLMRzTLiqsQWtsrBjAio4UyiLlIjHyZ+Nlv1WhH\\\\\\\/o28q40auKRdM13BFYbxM\\\\\\\/9TMVjhhCbaUxW7Q5caMhHY1ulxc9bBOJYfGzN7PuUyi3Y2bG7mZqeKQW4fhDEkWYjuGpkE5IOop+7bMtKr4NpEVDyrdvTD+sG0uxpCK3rgIN1DnkjI3ad\\\\\\\/Y+uNe1GWRjRd9Ds2Yh\\\\\\\/j1BFxuSb0MUxPJb3a9UySYzYx0ibJNZPoG2zAVtyYvMrfzIOzuI1Uj10NKSG2XCTF\\\\\\\/dKG6tlxRBIMN0KICPt8Q3Y2ms8fjPWq0i+8LjfYrEV3ZwQ9qhZtao48NcCvAgXBgBYrFXlnqF2lcNPJaryd90CXfZbX4lLyz6PoLEK89CFBZ4Fetq56mbuPBIzNC965UBcAiOs9aP6nmYjVmfD238V\\\\\\\/au2udMnEFXeSWXyti5RkfxzkUsEQkpuxbUYYi1I45O05izrF8UOFqxMIzN2ZGjgz5H\\\\\\\/zjapwRL++n9RnqW1\\\\\\\/mcn0oDPlAiH7ugD4M+0G3v+0TS1ai5dLe5vCAef0q+Zsu\\\\\\\/p5KVEXonSp49juQou3qIcRJ0DCqcy5bC+Au8YyM\\\\\\\/EVdxY5bqtUlOSmp3\\\\\\\/4R6sVFek7Atj793nygNxjiA9hqBMMB6CkXHOMAwZIeO5jfqTpI1R\\\\\\\/eoM\\\\\\\/XYW20mord1WHE+cHx6wj4x2WQEv+nk2zuwVXum2G1SfFWx8oK5DOulJicliQ3Pj8cWxx7zkgV\\\\\\\/CqYLSvZDqY9aTZ\\\\\\\/Em0NGYHnd+q60mNKnIPs0fZGZyl3HQUXJiT9UZMqBHPDsDMNRuunWNJ74SMPBQvodfHTbpeucTtpr09R+pcdGHsXeWSLH0J1vQ\\\\\\\/rbmzZ\\\\\\\/54jEZN9BH2X5VO1YkKUWo4JX3PWQcoyrpTwFxxbC9Pfh9yEUNpBr6zL75Hhs06tUTzf+eezWp3ivXdFPhcBPyKw60P4TStyU3alpW6gGLGglx8v\\\\\\\/q5rpKrtZarrxDmw1Z7dvArM9SgxGdFI1JbCYgh6chsqQHY9Ln78FXkK8RuwO90L+cWNDXCKFePOvgWDQUmLobsAxmKn1yzzS7hOYNCsg1SPJezZJQ+kcQQYDFycXKwBoFiFXs0FSDWF9hcpJtqDt71vw5fBOYYhiYY+BG43UfCOQBPHp3q5qoM7Q6u4N3QXa6SH4r3gCDM3TcnmE1KJm4j\\\\\\\/PifguPwzBMYNw4\\\\\\\/GsXxKoBR6GCHG62nU7bZMIiVZudyoq0QImulV\\\\\\\/xWe4VC2wsk5GbVvxiD3cHBrZxdiOh10OINVYJJ2XsoY\\\\\\\/KASx3ImP2sobcz3VNdYGGyD5Mm5B2d\\\\\\\/pwVkxnlH55v+dEBwyhPT2ZPDlWmd07N62NaYPAedVM\\\\\\\/2AstdPSUZrzALIIulhTezCHyWF7vIvCsRCsEpsscx+T6fJHGEncFuu3Euql9mgq+jFuzFFdEs7jgC4To\\\\\\\/82vdwkQHvajxrfzql\\\\\\\/mZU7WL0PP9SM8nhFRnCjAGnL\\\\\\\/mOoaYBVNqrn6+3Kb\\\\\\\/tjcmuytydjYCbZ8ggEO2WKwx3OVOmsYrywvmXmxCa\\\\\\\/xSPXEkfeeLUJkfcooKoGh3kdgHPhyqp8nN3ZjZSEatscj4umAN8PzumUKlwHi\\\\\\\/nE1FmHtPCO\\\\\\\/COT6tCmeg9Kx62Fhu8mAG46uSGYyZqUFOooe4jI6A\\\\\\\/b79I1YaW\\\\\\\/XBiEfC7\\\\\\\/3tjXKfhL5ZQ2La+AaZoNsl\\\\\\\/JYTOqDtcfooA+5iqhn+DmMx3\\\\\\\/bGJ45H1AD6sJS3YiaOw9YBKIOrAx2l1An6grJ+AY+22qqKbe+Spwl5M+N8ZaQShy0U68ZOHfQHbw\\\\\\\/a1MgFAvfyysMMoY3v+K32wODg4fjaj\\\\\\\/xX9lYo8GHPSGSfkSdxuOLBDSu2aNtqZxtrcV56SIJGXes8X63gzG2v3NDR5hvDG7BfjpBSlg\\\\\\\/75YGQAv3z3AZDRQigCp38yKsYU1L9jOQOf3rEy7vgv3hBd19xn3Ae68bo65\\\\\\\/XFogLGPQej\\\\\\\/MYpyJuTXUYvu+IvLoq84YjVHpRh7gE99EgWGbjZOIuh8qwaR4Vmy\\\\\\\/XD4dxqb6NjqNhtwgMoWsgW0s7pDYUZEiVNxhdQnMQXFd50e5Loq2jK1Mvjj6rU4WGVl\\\\\\\/N2JTHIyT78pUzvrQySviQXg6nuAv0IBKa5\\\\\\\/8yL49bapODxo1X\\\\\\\/Q4aXilf+LHRy7fw9izBvLu8A7JpnypHA9uJCtGNPar3D2y\\\\\\\/2Gl5U0aDMsvEA1CTqBo+AiH1CfiQUV4zJ1sQoG4V3OVNIEGOJmq+w3j0JMBnA46qGxoAD\\\\\\\/rbT4SYsUQUpJkJjdozUv+A70kFKz1bTnc5Kq9MrcGZwGXlROmW7xnqsILNaFlwyY\\\\\\\/TNHkMOZ1iLnbEM5k1gyhSpzsQGPe6TkFuEuHA3QlJ7XUgzwiUPn9dhzuWo\\\\\\\/zXHTqB1v0x8iUwfHVo6heY8T6zRlnni2MKYVUlyFSzhZ2fAOa4aXwjei11bz8rNoADX1xa1pAKljFan9xGnq5OPCiE5kNvh4QmFzbCQAPDddQIVcQeAWC+IW\\\\\\\/YK9WpZpkU+XSaviwjkZfQ1Fst6cFtZx6VOWu6GT9q+c0swIf0fZbNjluRvRsrk9GpvyKj+rYFfiaZksUTxK67aD87STodnBGVDEz88mcVBy5Q619IsRsUDMpuyLz5p\\\\\\\/VluiYyIfZ5AOZBqufLGPHGo77MFmZAgzRqHVCWV26mct9YqnCn1AmXOowc82jaJYJ34R8BX5jiGmvuS6678HZ4m3giV4gL\\\\\\\/wzzaazQGi+Nx+u9lXY9oTOqJKK28\\\\\\\/QSdW5vSXR5KmXdzx9Ezu\\\\\\\/fAFmic+f19k7nN+IG0888eKicGwwk9FvA5pNdIheOt5ziWOBqaNtUH4DNY5HjDytqGixpopPsoSU4IvCGlY\\\\\\\/Oxyq3VvDrnvsrWQPc4xqSStJGT\\\\\\\/mA0sES3QnxQQ570xJNasXbg\\\\\\\/6pTN7qDEYTuWRQ76ZGHkPEU+MmvCkyjpLvDDryV0bmb5ZAYpt7KOpuVEGbQVfpYh4apc1wc+0jrMmaog5+btsdnwhJmxB8E17dDgSqhCe+ApfSxDziWA07tEliHwc48xASdwJFHHJqRM8L0JHzeY9dweEA085khQP8UqQK28ocYcjU9WuymQK5YW0Acq4\\\\\\\/mBghxM+UCmkAipZFbz0zgbh0PIYGuNw8zM+E669x2kBwW9uT4UDiynb1JOZLnS9J32tVt8rNrPMK+0Erzi4Uy3ZiMJGQjw2hmWZaGMcbvXYLvC62R8\\\\\\\/F6fXcESEIXK3ZUwDt70Jm2L1DQMdmd6wq4N0Yoe2vSJUrRGX55ZiXOj1puONwN\\\\\\\/tGvT1xSivK8TthipxDh+4y22EOyn\\\\\\\/9M02lsaoKA1Qu2y8z5K6G\\\\\\\/iVIk9UBZ9kk5e0jRF2K4WC2d+lA3AAPK59DcCdlXxEi04WyygQFI\\\\\\\/AMMeeA7D+XL2pr+y9unXxslsC8vbPR6cmSWz1I1xeDhKpcCjBCwezr3GeH0hSAu0GWAKZ7CEypA8qIUsRMUu2Q4u7Y97VAAb\\\\\\\/eO\\\\\\\/CePu31nPVQad2PMosnLpsu8bYDClRZFAxCjD3t8B7AxkgbE6RVQvdoJMbFeD1b378BMRipxDoQ1b7VEHQgltwPAZtUkOrZ08oxHFjSAqMRrU51fW+wBX8K5ksFL5ktGxC78uv9cHxK3pGCK1NSk2KoTg8NtMUuyhSoIj8t+TihaKQYrVq1IvBwYP+d5VejV\\\\\\\/wRdxczolcO41yVoglr\\\\\\\/A3GOjkkErobk2nuEJt5cSQH24Nd\\\\\\\/gyjnA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022mU1vmWtTmEOXmfNd79B0lg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ad4af1b7ce923f5919e597b6eebc2f049431917c3e749703e5f605e6d6808e44\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022dtUTC6KpfbxfTgjArxvAqk8P2rfpFnoQW\\\\\\\/zU9tJzLBVyMHCca119fQA+nSSQZIzfDj5GvOIegHRmesi5xr6QylEduJsWgU18+\\\\\\\/vuPREYkQsk72TlNbKurP5ckVWcwc3M5BNLnyHmRWiUP9Xx7JH5blHIlN6PckbKNZ0vLoXvd\\\\\\\/M5iylE7exR5JhUzunOyjlEr+\\\\\\\/53gttqUEtviypVAAkHSIOncpE06BY6g5SSvHhu\\\\\\\/Nw14rG7tORLADT1VHiRSgNBn2ajrXwSLoevXId3VKlLsk7GVPGm7dGaavtAriFWOTYGFRkEK53G9sI4ZZAbs8iT\\\\\\\/aQ5WUPh5kAX7V8MY4xS+dWW08495AyS2CMgmx0kbA03Ser+PRE4RVbfwKmXHe6yV1pVayPA19DtbfdQsXRiK\\\\\\\/zJ0B6Cnbut8WWcN7kCz05PkI6xhOlqKLvkP97Eus2S8spMXp2ViD4VMSj6+qXWs6fH46urFyXP0742sBO+fXGnH7NZ0HUJaEBVrY5TOYZ2C8Y7vXnxl+M\\\\\\\/uoas4I58FOwP5rLjMdhAyC5ZwAfmPYAFlgFfJ\\\\\\\/CY8j3eUqNcMkzN2z1zjpLJI2AeRvY8T9eUUowlPazAxxTVLHg06w7BHxXU44x\\\\\\\/yrd3MMANGxkeMzT+iuX3MVr5SEJet0D5Ie2JjENKQzPGmT6i9824qaYKHywo+HK2fwC4fyPU\\\\\\\/otbWgrFdH7+VoW1M30W5xO1D9i\\\\\\\/VAxwMx6gmsE3sqsy\\\\\\\/1CcS0k2LnsCGxNWn7S85U+M4iYt8tW6K0F72FVMD66V4oJtcIJzwYnMQYmjK7mUV7Hm9Oznnx23OJVEEzD1w5lh4+fadN9CeFK3ikghKAB5R23VZqwro2qgzz2u6fZWdgE+4C\\\\\\\/3kRGMX6EqkS+kqbFego+wUKMCqQK1V+xDEMRmkHYwN3bfnm4Ii0UO3pSbbpxmFoef9b84VEfLh5FLuXFAgXRtdZ6HidZuP0usM\\\\\\\/a8WYQswxOSZVrqVfWbWZgtMUDwfGAabiRjgDQzGvG3DxYhwvQkPW4+c5IUaiF3FaN+9PM8HhUwv8jewBu1b76gJ5cPuWCJKeRok6STHQzo078tv71csIvxrDS+svxh8737w+yNJTGtHigXKMhUlc+6y1RcubUM+VZj2Zcyg7dW8zTH8T1G\\\\\\\/dChL1ySA1PzwODWe\\\\\\\/XRmbe5+AMT1oi0sQ\\\\\\\/xMEE5Gw2XaO6u6h+ooCKtQ\\\\\\\/DS4vuYmjfLXHuZwOr7bbTTf+wYvutgE7jgqcckVYIVdLqQR3kqSgvFC8fvewCRIT5LAKH2K61U0TUXn36Xl8u8ucm6cl0krH8UF\\\\\\\/lK+XNHNo8hYWczEUVjOasJEJJLKqQmyIcM\\\\\\\/4x4HnLO5p\\\\\\\/HcX4i76GBYPmz87JaVQ0b+ZGyr+FqPoqOh+WS0IfccZ94xrI10A6fp\\\\\\\/nje8GOXED5LvXx8vahZUKjZbZUAzo13GehxQqJLAesca4fsm6mmM0cV40+uAI7OpOP7yvHB\\\\\\\/V\\\\\\\/iQqsYDiW4Ywq3sdjT+8heyTyR1dX2aTG5tzidxMjb+xGU20JCNSWAHdnSWDodW6Kt7Jv9wSt5BTiRseSB8F1mI3GiOXQ5eqf5\\\\\\\/M+LSlICGQJqsnCub+bVSWMuFZL7raYcAAn9SW9f7xv5JUBv26QN9hsCVgs8c9Qfex6ztVqBSVts0Y1+1eAFgXssG\\\\\\\/AZyeBtAfN4gxlz6UnVQqdAR1zkFOoU6Hp4ifJQU9qzEIi9sAb+oIdzumaoBJ8zCd\\\\\\\/Zw3jhfvMar\\\\\\\/yXlapxMxIrDSM9VVwCXnpQJ\\\\\\\/n\\\\\\\/ELYg7t2wCLF9AoJzo6OyyETqX0rmdh\\\\\\\/WdIASmijEMW\\\\\\\/IAdM\\\\\\\/15MkR\\\\\\\/B+280aARMQ0c1Royv6IWTKpWN3WNKA\\\\\\\/ZZOMeQllDn2jMz\\\\\\\/5M61yTeHuP+eiSFGY5zjt6bwUIxbdN01ol7rzbTXwwJMMGM4WTRLPfOUiRlKHg\\\\\\\/WcidD4jgEM+Dg1Ryc2uD6LkyCad8zOGTWr3ZNfLFo+aDU1MHXTA7SWUOVlaEbfU3VpklQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022lbW8ui7HMWqYDZIv3RWPdA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222f56749f514198e080efc1048b8c4a38a0526f37f634e951161a6768239f49fa\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459516 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/8/1241" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-kss-geJbJHBYjUndrMJmMrpBY_dL2N3ofpd2l7b28pY" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/8/1241" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-we8dMAIS-OVI-mdhYmmL8vPPb17L4G9TyuwAvNzx70o" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/8/1225" title="Genomes of Ashbya Fungi Isolated from Insects Reveal Four Mating-Type Loci, Numerous Translocations, Lack of Transposons, and Distinct Gene Duplications" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/8/1253" title="Identification of Null Alleles and Deletions from SNP Genotypes for an Intercross Between Domestic and Wild Chickens" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459516" id="node459516" data-pisa="ggg;3/8/1241" data-pisa-master="ggg;g3.113.006700" data-apath="/ggg/3/8/1241.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Animal Models and Integrated Nested Laplace Approximations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Anna Marie Holand</span>, <span class="highwire-citation-author" data-delta="1">Ingelin Steinsland</span>, <span class="highwire-citation-author" data-delta="2">Sara Martino</span> and <span class="highwire-citation-author" data-delta="3">Henrik Jensen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1241-1251;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006700 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459516" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anna Marie Holand </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Biodiversity Dynamics, Department of Biology, NTNU, NO-7491 Trondheim, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BMarie%2BHoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Holand%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnna%2BMarie%2BHoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anna.m.holand@ntnu.no" class="" data-icon-position="" data-hide-link-title="0">anna.m.holand@ntnu.no</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ingelin Steinsland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre for Biodiversity Dynamics, Department of Mathematical Sciences, NTNU, NO-7491 Trondheim, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ingelin%2BSteinsland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steinsland%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIngelin%2BSteinsland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sara Martino </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre for Biodiversity Dynamics, Department of Mathematical Sciences, NTNU, NO-7491 Trondheim, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sara%2BMartino%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martino%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASara%2BMartino%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Henrik Jensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Biodiversity Dynamics, Department of Biology, NTNU, NO-7491 Trondheim, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Henrik%2BJensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jensen%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHenrik%2BJensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/8/1241" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459516" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459516/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/8/1241.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459516" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459516/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/8/1241.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459516" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459516/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/8/1241.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459516" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459516/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/8/1241.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459516" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459516/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Animal models are generalized linear mixed models used in evolutionary biology and animal breeding to identify the genetic part of traits. Integrated Nested Laplace Approximation (INLA) is a methodology for making fast, nonsampling-based Bayesian inference for hierarchical Gaussian Markov models. In this article, we demonstrate that the INLA methodology can be used for many versions of Bayesian animal models. We analyze animal models for both synthetic case studies and house sparrow (<em>Passer domesticus</em>) population case studies with Gaussian, binomial, and Poisson likelihoods using INLA. Inference results are compared with results using Markov Chain Monte Carlo methods. For model choice we use difference in deviance information criteria (DIC). We suggest and show how to evaluate differences in DIC by comparing them with sampling results from simulation studies. We also introduce an R package, AnimalINLA, for easy and fast inference for Bayesian Animal models using INLA.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/additive-genetic-models" class="hw-term hw-article-keyword hw-article-keyword-additive-genetic-models" rel="nofollow">additive genetic models</a></li><li class="kwd"><a href="/keyword/approximate-bayesian-inference" class="hw-term hw-article-keyword hw-article-keyword-approximate-bayesian-inference" rel="nofollow">approximate Bayesian inference</a></li><li class="kwd"><a href="/keyword/heritability" class="hw-term hw-article-keyword hw-article-keyword-heritability" rel="nofollow">heritability</a></li><li class="kwd"><a href="/keyword/quantitative-genetics" class="hw-term hw-article-keyword hw-article-keyword-quantitative-genetics" rel="nofollow">quantitative genetics</a></li><li class="kwd"><a href="/keyword/animalinla" class="hw-term hw-article-keyword hw-article-keyword-animalinla" rel="nofollow">AnimalINLA</a></li></ul><p id="p-5">To estimate the additive genetic variance (and thus the heritability) of different kinds of traits, biologists and animal breeders often use a generalized linear mixed model (GLMM), called an <em>animal model</em>. In an animal model individual <em>i</em>’s trait <em>y<sub>i</sub></em> has a genetic part, <em>u<sub>i</sub></em>. The value <em>u<sub>i</sub></em> is known as the breeding value of individual <em>i</em>. From the assumption that the breeding value is the sum of effects of many genes and from the central limit theorem, the breeding values are assumed to have a Gaussian distribution with a dependence structure given by the pedigree. Since early 1980s, animal breeders have successfully used a frequentist approach with restricted maximum likelihood (REML), to for example increase meat or milk yield in cattle (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Simm 1998</a>). However, inference with REML is not trivial for GLMM models. Models for non-Gaussian traits especially are challenging in regard to uncertainty in breeding values and other parameter estimates (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Tempelman and Gianola 1994</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Sorensen and Gianola 2002</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bolker <em>et al.</em> 2009</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Fong <em>et al.</em> 2010</a>). A popular approach is best linear unbiased prediction (BLUP) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Henderson 1950</a>) for calculating breeding values (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Wilson <em>et al.</em> 2009</a>). However, BLUP ignores all the uncertainty associated with the estimation and are not suitable for hypothesis testing in evolutionary questions (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Postma 2006</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Wilson <em>et al.</em> 2009</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hadfield <em>et al.</em> 2010</a>). Another approach is to perform modeling in a Bayesian framework. All parameters are then considered random variables, and it is (in theory) straightforward to account for all uncertainty jointly in parameter estimates. This solves the problems making inference for non-Gaussian traits (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Tempelman and Gianola 1994</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Fong <em>et al.</em> 2010</a>) and accounting for estimation uncertainty in the breeding values. Further, Bayesian modeling also solves many of the issues regarding analysis of breeding values discussed in <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Postma (2006)</a>, <a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Wilson <em>et al.</em> (2009)</a>, and <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Hadfield <em>et al.</em> (2010)</a> as both breeding values and functions of breeding values (<em>e.g.</em>, mean breeding values over hatch years) are considered random variables, and hence both uncertainty and dependencies are accounted for. This flexibility of the Bayesian framework has made Bayesian animal models increasingly popular. They have been used in animal breeding since early 1990s, whereas they only recently have been introduced to evolutionary biology (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kruuk <em>et al.</em> 2008</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">O’Hara <em>et al.</em> 2008</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Ovaskainen <em>et al.</em> 2008</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hadfield 2010</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Steinsland and Jensen 2010</a>). See <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gianola and Fernando (1986)</a> and <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Sorensen and Gianola (2002)</a> for a discussion of Bayesian animal models.</p><p id="p-6">Except in a few special cases, Bayesian models do not have closed form analytic expression for quantities of typical interest, <em>e.g.</em>, posterior means. Hence, numerical approximations are needed. The traditional approximation procedure for Bayesian models is Markov Chain Monte Carlo (MCMC) (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Sorensen and Gianola 2002</a>, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Van De Wiel <em>et al.</em> 2013</a>). MCMC is a very flexible methodology that can be used to make inference for any Bayesian model, and we can get posterior estimates for any random variable or parameter; marginally, jointly, or functions of them. Setting up a good MCMC algorithm (quick convergence, good mixing, and computationally fast) and evaluating it (convergence and mixing) is challenging for a nonspecialist. Recently, this has improved for animal models as there are now packages available for doing inference for these models with MCMC in R (MCMCglmm; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Hadfield 2010</a>) and in BUGS (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lunn <em>et al.</em> 2000</a>).</p><p id="p-7">For hierarchical latent Gaussian Markov random field models, a nonsampling-based numerical approximation procedure, the Integrated Nested Laplace Approximation (INLA) has recently been introduced (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Rue <em>et al.</em> 2009</a>). Using INLA we can calculate marginal posteriors for all parameters and each random effect, as well as the posterior for linear combinations of random effects. INLA is based on direct numerical integration instead of simulations. <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Rue and Martino (2007)</a> show for several models and datasets that INLA is much faster than MCMC and more accurate for a given computation budget. Faster inference encourages applied researchers to explore more models. Furthermore, this also opens new opportunities to do simulation studies, which for example can be used to explore identifiability issues and to set up tests of specific hypotheses. To perform model selection between GLMMs is not a trivial task (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Skrondal and Rabe-Hesketh 2004</a>), and using difference in deviance information criterion (DIC) has been questioned (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Fong <em>et al.</em> 2010</a>). We suggest using a simulation study to evaluate whether DIC is an appropriate measure for model selection. Fast simulation and inference methods are essential for simulation studies to be computationally feasible. INLA has been used in several fields of statistics, <em>e.g.</em>, survival analysis (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Martino <em>et al.</em> 2011</a>), for spatial GLMM (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Eidsvik <em>et al.</em> 2009</a>) and in disease mapping (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Roos and Held 2011</a>, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Schrödle <em>et al.</em> 2011</a>). This paper contributes to easier and faster Bayesian inference for both Gaussian and several non-Gaussian animal models by demonstrating that these models fit the INLA-framework and by providing an R-package, AnimalINLA, for doing the inference.</p><p id="p-8">In the section <em>Materials and Methods</em>, we introduce the data used in the case studies. Then we briefly revise relevant requirements for using INLA and the possibilities INLA gives, and fully specify the animal models we use. We also present a framework for simulation based testing of the ability of difference in DIC to choose between models with and without genetic effects. Next, results from the synthetic case studies and the house sparrow case studies are presented. Inference is carried out using INLA and for some cases results are compared with MCMC. The article ends with a <em>Discussion</em> and <em>Conclusion</em>, where the results as well as opportunities and limitations of the INLA framework in quantitative genetics are discussed.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Data</h3><p id="p-9">For the case studies we use data from a natural metapopulation of house sparrow (<em>Passer domesticus</em>) on six islands off the coast of Helgeland, Northern Norway (66°N, 13°E). From adults and juveniles (<em>i.e.</em>, birds born the same summer) a small blood sample was collected and from adults several morphological traits were measured (including bill depth). The blood samples were used to determine genetic parenthood, and a genetic pedigree for the birds on the study islands could be established. This study system has many qualities for providing data on morphology and fitness-related traits as more than 90% of all birds on the six main study islands were individually ringed. Intensive observation and capture protocols each year gave good estimates of the lifespan of individual birds (a bird was considered dead when it was no longer captured or observed). For a more thorough description of the field work, study area and genetic parenthood analyses, see (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Ringsby <em>et al.</em> 2002</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Jensen <em>et al.</em> 2008</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Pärn <em>et al.</em> 2009</a>) and references therein.</p><p id="p-10">For all case studies we used 1993 to 2002 as our study period, and we used the same pedigree, which consisted of the <em>n<sub>p</sub></em> = 3574 individuals that were present on the six main study islands in this period. The pedigree spanned up to seven generations. For our case studies we used individual data on (1) bill depth, (2) breeding season success, and (3) average reproductive intensity (ARI). For all birds, sex, hatch year, and hatch island were available. The animal model implicitly assumes that missing phenotype observations are missing at random, and hence we (implicitly) have assumed this. However, if the process behind the missing data are connected to the trait of interest, this could result in biased inference of additive genetic variance (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Hadfield 2008</a>).</p><p id="p-11">In case study one we considered bill depth. This trait was measured each year (<em>i.e.</em>, age) for most individuals over the course of a lifetime. The proportion of individuals that have more than one measurement was 0.4, ranging from two to nine measurements in total. Bill depths were approximately Gaussian distributed (see <a href="http://www.g3journal.org/content/suppl/2013/05/24/g3.113.006700.DC1/006700SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2013/05/24/g3.113.006700.DC1/FigureS1.pdf">Figure S1</a>), and we have measurements for <em>n<sub>d</sub></em> = 1025 birds. Many individuals in the pedigree had missing data for this trait because the bird did not survive until it was 1 year of age. We standardized the data to have mean 0 and variance 1.</p><p id="p-12">In case study two, we considered breeding season success. If at least one of the offspring of an adult bird produced in a given breeding season survived until recruitment, we defined its breeding season a success. A recruit is an offspring that survives up to adult age, <em>i.e.</em>, 1 year of age in the house sparrow. Otherwise the breeding season was a failure. The breeding season could be a failure either because the bird did not produce any offspring, or because all its offspring died before recruitment. The data consist of pairs of values (<em>n<sub>i</sub></em>, <em>y<sub>i</sub></em>), where <em>n<sub>i</sub></em> is the number of breeding seasons individual <em>i</em> had during the study period (<em>i.e.</em>, it was alive and adult) and <em>y<sub>i</sub></em> is the number of successful breeding seasons, <em>y<sub>i</sub></em> ≤ <em>n<sub>i</sub></em>. Individuals that died before their first breeding season (did not recruit) or that emigrated to an island not among the six main study islands have no data. There are <em>n<sub>d</sub></em> = 1182 individuals with data. Of these approximately 71% did not have any successful breeding seasons.</p><p id="p-13">In case study three, we considered data on ARI, <em>i.e.</em>, the average number of recruits an individual produced over its lifetime. Data take the form (<em>n<sub>i</sub></em>, <em>y<sub>i</sub></em>) where <em>n<sub>i</sub></em> is identical to <em>n<sub>i</sub></em> in case study two, and <em>y<sub>i</sub></em> is the total number of recruits produced in the study period. For this trait we had data for the same <em>n<sub>d</sub></em> = 1182 individuals as in case study two. <em>y<sub>i</sub></em> ranged from 0 to 10, with mean 0.64. 71% produced no recruits, and about 46% of the 344 individuals that produced one or more recruits produced only one. The datasets are available in <a href="http://www.g3journal.org/content/suppl/2013/05/24/g3.113.006700.DC1/FileS5.pdf">File S5</a>.</p></div><div id="sec-3" class="subsection"><h3>Latent Gaussian models and INLA</h3><p id="p-14">In this section we give a brief introduction to latent Gaussian models and how INLAs can be used to make approximations for posterior marginals for these models. In general, latent Gaussian models are hierarchical models in which we assume a <em>n<sub>p</sub></em>-dimensional latent field <strong><em>x</em></strong> to be point-wise observed through <em>n<sub>d</sub></em> ≤ <em>n<sub>p</sub></em> data <strong><em>y</em></strong>, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-1.gif"/></noscript></span></span>. The latent field <em>x</em> includes both random and fixed effects and is assumed to have a Gaussian density conditional on hyperparameters <strong><em>θ</em><sub>1</sub></strong>: <strong><em>x</em></strong>|<strong><em>θ</em><sub>1</sub></strong> ∼ <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-2.gif"/></noscript></span></span> (0, <em><strong>Q</strong></em><sup>−1</sup>(<strong><em>θ</em><sub>1</sub></strong>)).</p><p id="p-15">The data <strong><em>y</em></strong> are assumed to be conditionally independent given the latent field <strong><em>x</em></strong> and, possibly, some additional hyperparameters <em>θ</em><sub>2</sub>. The model definition is completed by assigning a prior density to the hyperparameters <strong><em>θ</em></strong> = {<strong><em>θ</em></strong><sub>1</sub>, <strong><em>θ</em></strong><sub>2</sub>}. In addition, some linear constraints of the form <strong><em>Bx</em></strong> = <em>e</em>, where the matrix <strong><em>B</em></strong> has rank <em>k</em>, may be imposed (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Rue <em>et al.</em> 2009</a>).</p><p id="p-16">INLA provides a recipe for computing in a fast and accurate way, approximations to marginal posterior densities for the hyperparameters <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-3.gif"/></noscript></span></span> and for the latent variables <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-4.gif"/></noscript></span></span>. Such approximations are based on a smart use of Laplace or other related analytical approximations and of numerical integration schemes. As a by-product of the main computations INLA can also compute the DIC (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Spiegelhalter <em>et al.</em> 2002</a>). DIC is calculated as the expectation of the deviance over the posterior distribution (<em>E</em><strong><em><sup>x</sup></em></strong><sup>,</sup><strong><em><sup>θ</sup></em></strong>(<em>D</em>(<strong><em>θ</em></strong>, <strong><em>x</em></strong>))) plus the effective number of parameters (<em>p<sub>D</sub></em>), DIC = <em>E<sup><strong>x</strong></sup></em><sup>,</sup><em><sup><strong><em>θ</em></strong></sup></em>(<em>D</em>(<strong><em>θ</em></strong>, <strong><em>x</em></strong>)) + <em>p<sub>D</sub></em>. For the class of latent Gaussian models INLA is constructed for the deviance can be expressed as <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-5.gif"/></noscript></span></span>, where the sum is over all observations. The posterior mean of the deviance <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-6.gif"/></noscript></span></span> can therefore be calculated using INLA (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Rue <em>et al.</em> 2009</a>). <em>p<sub>D</sub></em> is approximated to <em>n</em> − <em>tr</em>{<strong><em>Q</em></strong>(<em><strong>θ</strong><sup>m</sup></em>)<strong><em>Q</em></strong>*(<em><strong>θ</strong><sup>m</sup></em>)<sup>−1</sup>}, where <em>n</em> is the number of observations and <em><strong>θ</strong><sup>m</sup></em> denotes the posterior median of <em>π</em>(<strong><em>θ</em></strong>|<strong><em>y</em></strong>). It is worth to note that another common definition of DIC is to use the posterior mean (instead of median) for the parameters in the deviance. This definition is used in MCMCglmm.</p><p id="p-17">For the INLA methodology to work in a fast and efficient way, latent Gaussian models have to satisfy some additional properties. First, the latent Gaussian model <strong><em>x</em></strong>, often of large dimension, admits conditional independence properties. That is, it is a Gaussian Markov random field (GMRF) with a sparse precision matrix <em>Q</em> (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Rue and Held 2005</a>). The efficiency of INLA relies, in fact, on efficient algorithms for sparse matrices. Second, because INLA needs to integrate over the hyperparameter space, the dimension of non-Gaussian <strong><em>θ</em></strong> should not be too large, say ≤ 14, due to the numerical integration scheme and optimization methods used. Finally, each data point <em>y<sub>i</sub></em> depends on the latent Gaussian field only through the linear predictor <em>η<sub>i</sub></em> = <em>g</em>(<em>μ<sub>i</sub></em>) where <em>g</em>(⋅) is a known link function and <em>μ<sub>i</sub></em> = <em>E</em>(<em>y<sub>i</sub></em>|<strong><em>x</em></strong>, <strong><em>θ</em></strong>), <em>i.e.</em>, <em>π</em>(<em>y<sub>i</sub></em>|<strong><em>x</em></strong>, <strong><em>θ</em></strong>) = <em>π</em>(<em>y<sub>i</sub></em>|<em>η<sub>i</sub></em>, <strong><em>θ</em></strong>). INLA presents several advantages over MCMC based inference: it provides accurate results in just a fraction of the time needed by smart MCMC algorithms, and it does not require convergence diagnostics. Moreover, the R-INLA package (available at <a href="http://www.r-inla.org">www.r-inla.org</a>) makes inference from GRMF models using the INLA methodology easy.</p></div><div id="sec-4" class="subsection"><h3>Animal models</h3><p id="p-18">In this section we show that animal models are latent GMRF models, which fits into the INLA framework (see the section <em>Latent Gaussian models and INLA</em>). Moreover, we describe in detail the different versions of animal models for which we are interested. Because the focus of this article is to emphasize INLA as a method of inference for animal models, the models in the case studies and synthetic studies are kept simple. For a more in depth introduction to animal models, see (<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Sorensen and Gianola 2002</a>).</p><p id="p-19">In general, an animal model is a generalized linear mixed model; the observed trait <em>y<sub>i</sub></em>, <em>i</em> = 1, …, <em>n<sub>d</sub></em> belongs to an exponential family<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-7.gif"/></noscript></span></span>where the expected value <em>μ<sub>i</sub></em> = <em>E</em>(<em>Y<sub>i</sub></em>) is linked to a linear predictor <em>η<sub>i</sub></em> through a known link function <em>g</em>(⋅), so that <em>g</em>(<em>μ<sub>i</sub></em>) = <em>η<sub>i</sub></em>. The linear predictor <em>η<sub>i</sub></em> accounts for the effects of various covariates and the breeding value in an additive way;<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-8.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-9.gif"/></noscript></span></span> is an intercept, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-10.gif"/></noscript></span></span> are <em>fixed effects</em>, <em>u<sub>i</sub></em> individual <em>i</em>’s breeding value, <em>ε<sub>i</sub></em> its residual effect, and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-11.gif"/></noscript></span></span> a known incidence vector. The <em>fixed effects</em> (in a frequentist’s framework) account for group-specific effects such as <em>e.g.</em>, sex, year of birth, and locality or subpopulation. In a Bayesian framework all parameters are treated as random variables, but out of convenience we refer to <strong><em>β</em></strong>s as fixed effects. The breeding values are genetically linked random effects also known as additive genetic effects. The residual effects are unstructured Gaussian random effects, often named the environmental effect in quantitative genetics. We assign a Gaussian prior to <strong><em>β</em></strong>: <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-12.gif"/></noscript></span></span>, where <strong><em>I</em></strong> is the identity matrix. The residual effects are <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-13.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-14.gif"/></noscript></span></span> is often referred to as environmental variance.</p><p id="p-20">The breeding values for the population, <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-15.gif"/></noscript></span></span>, are assumed to have a dependency structure corresponding to the pedigree<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-16.gif"/></noscript></span></span>where <strong><em>A</em></strong> is the relationship matrix and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-17.gif"/></noscript></span></span> is the additive genetic variance (see <em>e.g.</em>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Lynch and Walsh 1998</a>, <a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">Sorensen and Gianola 2002</a>). A GMRF is a multivariate Gaussian model with a conditional independence structure, also known as a Markov structure. The pedigree imposes a Markov structure. If we are interested in individual <em>i</em>’s breeding value, and we know its parents, offspring and the other parent(s) of its offspring, other individuals’ breeding value do not give us any extra information. Because of the fact that the breeding values forms a GMRF, the inverse of the relationship matrix, <strong><em>A</em><sup>−1</sup></strong>, is a sparse matrix (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Steinsland and Jensen 2010</a>). <strong><em>A</em><sup>−1</sup></strong> can be calculated from the pedigree (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Quaas 1976</a>).</p><p id="p-21">Note that there might be more individuals in the pedigree than individuals with observations, <em>n<sub>d</sub></em> ≤ <em>n<sub>p</sub></em>, and we have assumed an indexing such that <em>u<sub>i</sub></em> corresponds to <em>y<sub>i</sub></em>. The hyperparameters <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-18.gif"/></noscript></span></span> are assigned inverse gamma priors. Furthermore, to avoid identification problems we include a common intercept and constrain the levels of each factors to sum to zero (see <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Steinsland and Jensen 2010</a>).</p><p id="p-22">The animal model as described previously is a latent GMRF model where the latent field is <strong><em>x</em></strong> = (<strong><em>β</em></strong>, <strong><em>u</em></strong>) and the hyperparameter vector <em>θ</em> includes the variances <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-19.gif"/></noscript></span></span> and, possibly, the parameters in the likelihood function. The precision matrix for the latent field <em>x</em> is sparse because the inverse of <strong><em>A</em></strong> is sparse. Moreover, the likelihood of each data point depends on the latent field only through the linear predictor <em>η<sub>i</sub></em> defined in equation 1. Therefore, INLA can be applied to the animal model.</p><p id="p-23">In our analyses we might be interested in marginal posterior for individual breeding values, <em>u<sub>i</sub></em>, fixed effects <strong><em>β</em></strong>, the additive genetic variance <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-20.gif"/></noscript></span></span>, the residual variance <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-21.gif"/></noscript></span></span>, the heritability <em>h</em><sup>2</sup>, or to evaluate the model using DIC. The heritability is loosely speaking the proportion of the variability the genes account for in a phenotypic trait. Precise definitions of heritability are given in subsequent subsections. In addition, it might be interesting to look at linear combinations of breeding values <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-22.gif"/></noscript></span></span>, where <em>w<sub>i</sub></em> are weights, for example the mean of breeding values for different cohorts.</p><div id="sec-5" class="subsection"><h4>Animal model for Gaussian data:</h4><p id="p-24">For many continuous traits, such as the bill depth of house sparrows, it is natural to assume a Gaussian likelihood with an identity link function, <em>η<sub>i</sub></em> = <em>μ<sub>i</sub></em>. The animal model can then be written as: <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-23.gif"/></noscript></span></span>, where the linear predictor is modeled as in (1) and the variance <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-24.gif"/></noscript></span></span> is the variance of the residual effects. The model can be formulated in the INLA framework with likelihood <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-25.gif"/></noscript></span></span> and latent field <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-26.gif"/></noscript></span></span>.</p><p id="p-25">In many datasets there are repeated measurements for some individuals. A common modeling approach is then to include an individual specific random effect <em>ind<sub>i</sub></em> for each individual. Let <em>y<sub>ij</sub></em> denote measurement <em>j</em> of individual <em>i</em>. The likelihood is <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-27.gif"/></noscript></span></span>, and the latent field <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-28.gif"/></noscript></span></span>. This redefines the variance interpretation, and we can interpret <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-29.gif"/></noscript></span></span> as a special environmental variance (variation of the individuals’ trait values through life), and <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-30.gif"/></noscript></span></span> as the measurement error or unexplained (environmental) variance (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Lynch and Walsh 1998</a>).</p><p id="p-26">In general, and in the Gaussian case, the narrow sense heritability is defined as the proportion of the phenotypic variance that is caused by additive genetic variance (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Lynch and Walsh 1998</a>)<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-31.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>Although it is easy to use the INLA methodology to compute posterior marginals of hyperparameters, posteriors for functions of more than one hyperparameter, <em>e.g.</em>, <em>h</em><sup>2</sup>, become computationally inconvenient. This can be solved by reformulating the model in the INLA framework (see <a href="http://www.g3journal.org/content/suppl/2013/05/24/g3.113.006700.DC1/FileS1.pdf">File S1</a> for this model formulation, and how to use it).</p></div><div id="sec-6" class="subsection"><h4>Animal model for binomial data:</h4><p id="p-27">With binomial data, the animal model is defined as: <em>y<sub>i</sub></em> ~ Bin(<em>n<sub>i</sub></em>, <em>p<sub>i</sub></em>) <em>i</em> = 1, …, <em>n<sub>d</sub></em>, where <em>n<sub>i</sub></em> is the numer of trials and <em>p<sub>i</sub></em> is the probability of success. Moreover, we assume a logit link function, so that the linear predictor is defined as: <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-32.gif"/></noscript></span></span>. The linear predictor is then modeled as in (1). In the binary case (<em>n<sub>i</sub></em> = 1) the variance of the non-structured random effect <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-33.gif"/></noscript></span></span> is confounded with the link, and is not identifiable (<a id="xref-ref-39-6" class="xref-bibr" href="#ref-39">Sorensen and Gianola 2002</a>) because the individual effects are already accounted for through the link and the likelihood. Therefore, we omit <em>ε</em> from the linear predictor and use this linear predictor for all binomial models</p><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-34.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span><p id="p-28">For binomial data, it is not immediately obvious how to define the heritability of the trait. The most common definition is derived from the idea that there exists a latent (unobserved) continuous trait called liability <em>l<sub>i</sub></em> such that we register a success if <em>l<sub>i</sub></em> &lt; 0 and a failure if <em>l<sub>i</sub></em> &gt; 0 (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Dempster and Lerner 1950</a>). The definition of heritability depends also on the type of the link function and in the case of the logistic function it is<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-35.gif"/></noscript></span> <span class="disp-formula-label">(4)</span></span>were <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-36.gif"/></noscript></span></span> is the variance of a logistic variable (see <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Vazquez <em>et al.</em> 2009</a>). Note that the heritability on the latent scale does not correspond to the proportion of explained variance in the phenotype, <em>e.g.</em>, the binomial data. For a discussion on heritability for non-Gaussian traits, see (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Dempster and Lerner 1950</a>, <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Visscher <em>et al.</em> 2008</a>). The binomial animal model is a latent Gaussian model with only one non-Gaussian hyperparameter, <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-37.gif"/></noscript></span></span>. The heritability, as defined in (4), is a function of only one random variable, <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-38.gif"/></noscript></span></span>, and can therefore easily be calculated from <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-39.gif"/></noscript></span></span>’s marginal posterior distribution.</p></div><div id="sec-7" class="subsection"><h4>Animal model for (zero-inflated) Poisson data:</h4><p id="p-29">Count data are often modeled as Poisson distributed: <em>y<sub>i</sub></em> ~ Poisson (<em>μ<sub>i</sub></em>) with <em>μ<sub>i</sub></em> = <em>E<sub>i</sub>λ<sub>i</sub></em>, where <em>E<sub>i</sub></em> is the known exposure, <em>e.g.</em>, the lifetime, and <em>λ<sub>i</sub></em> is the intensity, <em>e.g.</em>, the annual reproductive success. We assume an exponential link function <em>η<sub>i</sub></em> = log(<em>λ<sub>i</sub></em>), and model the linear predictor <strong><em>η</em></strong> as in (3).</p><p id="p-30">Datasets which are almost Poisson, but have too many zero-observations, often occur. Then a zero-inflated Poisson (ZIP) distribution might be useful. ZIP models are a mixture of a Poisson distribution and a distribution with point mass one at zero. There are several versions of zero-inflated Poisson, we will use <em>ZIP</em>(<em>p</em>, <em>μ<sub>i</sub></em>) defined as: Prob(<em>y</em>|…) = <em>p</em> × 1<sub>[</sub><em><sub>y</sub></em><sub>=0]</sub> + (1 − <em>p</em>) × Poisson(<em>y</em>; <em>μ<sub>i</sub></em>), where 1<sub>[</sub><em><sub>y</sub></em> <sub>= 0]</sub> is an indicator function and Poisson(<em>y</em>; <em>μ<sub>i</sub></em>) indicates the Poisson likelihood with mean <em>μ<sub>i</sub></em>, and <em>p</em> is the proportion of extra zeros. Poisson and zero-inflated Poisson animal models are latent Gaussian fields with hyperparameter vectors <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-40.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-41.gif"/></noscript></span></span>, respectively.</p><p id="p-31">In the Poisson case it has been proposed that the heritability on the log scale can be defined as (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Foulley <em>et al.</em> 1987</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Matos <em>et al.</em> 1997</a>, <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Vazquez <em>et al.</em> 2009</a>)<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-42.gif"/></noscript></span> <span class="disp-formula-label">(5)</span></span>where <em>λ</em> is the average intensity; <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-43.gif"/></noscript></span></span>.</p><p id="p-32">The heritability (5) is then a function of one hyper-parameter and the random variable <em>λ</em> which is a linear combination of functions of predictors <em>η<sub>i</sub></em>. Such a quantity is (at least currently) not possible to compute using R-INLA. An approximated estimate of <em>h</em><sup>2</sup> can be computed by using a point estimate for <em>λ</em> together with the marginal posterior of <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-44.gif"/></noscript></span></span>. The point estimate can either be calculated directly from data, or by plugging in point estimates for the predictors <strong><em>η</em></strong>. With this model we calculate the heritability of the intensity, <em>e.g.</em>, ARI, although the data are individual lifetime reproductive success. If the heritability of lifetime reproductive success (LRS) is of interest, this can be estimated by setting the exposure <em>E<sub>i</sub></em> = 1 (and only using individuals that are uncensored at either end of the study period).</p></div><div id="sec-8" class="subsection"><h4>Simulation-based evaluation of DIC:</h4><p id="p-33">We often want to check whether an additive genetic effect should be included in the model or not. In a classical approach to hypothesis testing, this corresponds to a null hypothesis of no genetic effects (no heritability), and an alternative hypothesis of some genetic effect (heritability). This can be seen as a model selection between a model without genetic effects and an animal model. That is, in our formulation from the section <em>Animal Models</em>, between a model with some likelihood and latent field,<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-45.gif"/></noscript></span> <span class="disp-formula-label">(6)</span></span>or</p><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-46.gif"/></noscript></span> <span class="disp-formula-label">(7)</span></span><p id="p-34">To evaluate the ability of the difference in DIC to correctly choose between these models we suggest the use of simulations. We use the pedigree, explanatory variables, priors, and missing structure of the dataset and a model we want to compare. To find an estimate of the probability of type I error, concluding that there are genetic effects when in truth there is none, we sample <em>S</em> new data sets from a model without genetic effects, <em>i.e.</em>, with (6), and impose the same missing structure as in the original data set. For each of these <em>S</em> data sets we fit both models, <em>i.e.</em>, with and without genetic effects, and find the difference in DIC, ΔDIC = DIC model (6) − DIC model (7). The resulting <em>S</em> values of ΔDIC is an approximation to the sampling distribution of ΔDIC under the null hypothesis. If we use the recommended limit of ΔDIC &gt; 10 to reject the null-hypothesis, we can find an estimate of the corresponding significance level from the proportion of the <em>S</em> ΔDIC values larger than 10. We can also chose a significance level <em>α</em>, and find the corresponding limit of ΔDIC from the simulations.</p><p id="p-35">The other important quantity regarding hypothesis tests is the power function of the test, <em>i.e.</em>, the probability the <em>H</em><sub>0</sub> is rejected when there are some genetic effects. To estimate the power for a specific value of <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-47.gif"/></noscript></span></span> or <em>h</em><sup>2</sup>, we follow the same simulation approach, except that simulations are now done from a model with genetic effects, <em>i.e.</em>, from (7) with <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-48.gif"/></noscript></span></span> as chosen. The proportion of these <em>S</em> Δ<em>DIC</em> values larger than our chosen limit is an estimate of the power for a given <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-49.gif"/></noscript></span></span>.</p></div></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Synthetic case studies</h3><p id="p-36">In this section we illustrate the INLA methodology using a series of synthetic case studies for the models (described in the section <em>Animal models</em>). We report here results for the Gaussian and the Binomial model. For corresponding results for the Poisson model see <a href="http://www.g3journal.org/content/suppl/2013/05/24/g3.113.006700.DC1/TableS1.pdf">Table S1</a>.</p><p id="p-37">To make our simulated data set as realistic as possible we do the following: first, we simulate data based on the pedigree of the house sparrow dataset with <em>n<sub>p</sub></em> = 3574 individuals (as described in the section <em>Data</em>). Second, we replicate in the simulated data set the same missing data structure that we find in the house sparrow data set. Inference is done using the AnimalINLA package. See <a href="http://www.g3journal.org/content/suppl/2013/05/24/g3.113.006700.DC1/FileS3.pdf">File S3</a> for R code. As priors for <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-50.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-51.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-52.gif"/></noscript></span></span> we use inverse gamma distribution InvGamma(<em>a</em>, <em>b</em>) (parametrized such that it has mean <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-53.gif"/></noscript></span></span> and variance <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-54.gif"/></noscript></span></span>). InvGamma(0.5, 0.5) for <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-55.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-56.gif"/></noscript></span></span> and InvGamma(<em>e</em>, <em>e</em><sup>−10</sup>) for <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-57.gif"/></noscript></span></span>.</p><div id="sec-11" class="subsection"><h4>Synthetic Gaussian case study:</h4><p id="p-38">In our first experiment we simulated Gaussian data from:<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-58.gif"/></noscript></span> <span class="disp-formula-label">(8)</span></span><span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-59.gif"/></noscript></span> <span class="disp-formula-label">(9)</span></span>where <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-60.gif"/></noscript></span></span>, and <strong><em>A</em><sup>−1</sup></strong> is computed from the house sparrow pedigree.</p><p id="p-39">We simulate data for <em>β</em><sub>0</sub> = 0 and values of <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-61.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-62.gif"/></noscript></span></span> between 0 and 1 such that <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-63.gif"/></noscript></span></span>. Moreover, we assume as missing all measurements that are missing for bill depth in the house sparrow data set. We follow <a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Steinsland and Jensen (2010)</a> and fit the model assuming a sum to zero constraint on the breeding values, <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-64.gif"/></noscript></span></span>. In this experiment we are interested in the variance parameters and use the model formulation described in the section <em>Animal model for Gaussian data</em> (<em>i.e.</em>, MF1 in <a href="http://www.g3journal.org/content/suppl/2013/05/24/g3.113.006700.DC1/FileS1.pdf">File S1</a>).</p><p id="p-40"><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a> shows the estimated posterior mean together with standard deviations, and the 95% credible interval (CI) for <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-65.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-66.gif"/></noscript></span></span>. The posterior means are quite close to the true values of <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-67.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-68.gif"/></noscript></span></span>, with small standard deviations and 95% CIs that contain the true value. However, for small values for <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-69.gif"/></noscript></span></span> (&lt;0.1) the posterior mean of the genetic variance seems to be systematically different from the parameter value used in the simulations. We will refer to this as a systematic error. It is caused by influential priors (discussed in the house sparrow case studies). As the likelihood is Gaussian, the Laplace approximation is exact and hence its accuracy comes from the numerical integration scheme.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/8/1241/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results from the synthetic Gaussian case study. (A) Posterior mean (solid lines) with 95% CI (dashed lines) for  (black and open squares) and  (gray and closed squares) from the Gaussian synthetic case study against the value of  used in the simulations (together with ). The power of a model selection test using &#x394;DIC &gt; 10 as limit is plotted as x and solid lines. The power is estimated using the simulation approach described in the text. (B) Boxplots of simulated values of &#x394;DIC against the value of  used in the simulations (together with ). The values of &#x394;DIC from the synthetic case study are plotted as stars. &#x394;DIC equal to 10 is indicated by a horizontal line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1385294554" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Results from the synthetic Gaussian case study. (A) Posterior mean (solid lines) with 95% CI (dashed lines) for &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-70&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (black and open squares) and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-71&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (gray and closed squares) from the Gaussian synthetic case study against the value of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-72&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; used in the simulations (together with &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-73&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). The power of a model selection test using &#x394;DIC &gt; 10 as limit is plotted as x and solid lines. The power is estimated using the simulation approach described in the text. (B) Boxplots of simulated values of &#x394;DIC against the value of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-74&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; used in the simulations (together with &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-75&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). The values of &#x394;DIC from the synthetic case study are plotted as stars. &#x394;DIC equal to 10 is indicated by a horizontal line.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/8/1241/F1.medium.gif" width="246" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/8/1241/F1.medium.gif" width="246" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/8/1241/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/8/1241/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463089" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-41" class="first-child">Results from the synthetic Gaussian case study. (A) Posterior mean (solid lines) with 95% CI (dashed lines) for <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/F1/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/F1/embed/mml-math-70.gif"/></noscript></span></span> (black and open squares) and <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/F1/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/F1/embed/mml-math-71.gif"/></noscript></span></span> (gray and closed squares) from the Gaussian synthetic case study against the value of <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/F1/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/F1/embed/mml-math-72.gif"/></noscript></span></span> used in the simulations (together with <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/F1/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/F1/embed/mml-math-73.gif"/></noscript></span></span>). The power of a model selection test using ΔDIC &gt; 10 as limit is plotted as x and solid lines. The power is estimated using the simulation approach described in the text. (B) Boxplots of simulated values of ΔDIC against the value of <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/F1/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/F1/embed/mml-math-74.gif"/></noscript></span></span> used in the simulations (together with <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/F1/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/F1/embed/mml-math-75.gif"/></noscript></span></span>). The values of ΔDIC from the synthetic case study are plotted as stars. ΔDIC equal to 10 is indicated by a horizontal line.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">For each simulated data set we also fit a model without genetic effect, hence where the model in (8) and (9) simplifies to:<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-76.gif"/></noscript></span> <span class="disp-formula-label">(10)</span></span><span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-77.gif"/></noscript></span> <span class="disp-formula-label">(11)</span></span>We now test, using ΔDIC whether a model with or without genetic affects will be chosen. The ΔDIC results are presented as stars (∗) in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>, and we find that using a limit ΔDIC &gt; 10 we chose the animal model for <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-78.gif"/></noscript></span></span>.</p><p id="p-43">To evaluate the ability of ΔDIC to choose between models, we use the simulation methodology suggested in the section <em>Simulation-based evaluation of DIC</em>. We use the same sets of <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-79.gif"/></noscript></span></span> as in the synthetic case study and simulated <em>S</em> = 100 synthetic data sets for each parameter sets. Boxplots of the corresponding ΔDIC are found in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>, whereas the power of the test for the different sets of <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-80.gif"/></noscript></span></span> are given in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>. We first notice that the significance level (the power for <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-81.gif"/></noscript></span></span>) is approximately 0.18. The power rapidly increases, and for <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-82.gif"/></noscript></span></span> it is 0.77, and already for <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-83.gif"/></noscript></span></span> it is 0.98. Hence, for Gaussian data with pedigree and missing structure as the house sparrow case study using difference in DIC as model selection criteria we have a good chance of identifying additive genetic variance above 0.1.</p></div><div id="sec-12" class="subsection"><h4>Synthetic binomial case study:</h4><p id="p-44">Binomial data can be challenging to analyze, especially when the number of trials <em>n<sub>i</sub></em> is very low (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Fong <em>et al.</em> 2010</a>). To analyze the performance of INLA for binomial data we have carried out different simulation studies and compared the estimates obtained with INLA with those obtained using MCMC (MCMCglmm; <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Hadfield 2010</a>). We simulate data from the model <em>y<sub>i</sub></em>|<em>p<sub>i</sub></em> ~ Bin(<em>n<sub>i</sub></em>, <em>p<sub>i</sub></em>) with a logit link function <em>η<sub>i</sub></em> = logit(<em>p<sub>i</sub></em>) = α + <em>u<sub>i</sub></em>. Where <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-84.gif"/></noscript></span></span>, and <strong><em>A</em><sup>−1</sup></strong> is computed from the house sparrow pedigree. We simulate data for <em>α</em> = 0 and values of <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-85.gif"/></noscript></span></span> such that the corresponding heritability, computed as in equation (4), varies between 0 and 1.</p><p id="p-45">In our first experiment we let <em>n<sub>i</sub></em> = 1, ∀<em>i</em> = 1, …, <em>n<sub>p</sub></em>, hence we have binary data for all the individuals in the pedigree. This case is, in general, particularly difficult, because with no replicates for any of the individuals the genetic variance is not identifiable (<a id="xref-ref-39-7" class="xref-bibr" href="#ref-39">Sorensen and Gianola 2002</a>). When we look at the posterior estimate for <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-86.gif"/></noscript></span></span>, we see that the performance of INLA is quite bad (see <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). Looking at <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>, we find that the posterior of the heritability differs between MCMC and INLA, MCMC follows the true value, INLA does not. INLA is based on a Gaussian approximation of the log-likelihood functions which, in this case, has a very non-Gaussian distribution, and the Laplace approximation is poor. Although MCMC follows the true value is has quite large credible intervals. The dependency structure induced by the house sparrow pedigree is not strong enough to allow for a precise estimation of the genetic variance. In practice, it is almost impossible to distinguish between cases with high and low heritability of the binary trait. We further find that for small heritability the estimates have systematic errors, as in the Gaussian case.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/8/1241/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results from the synthetic Binomial case study. True vs. estimated heritability: posterior mean (solid line) and 95% credible intervals (dashed lines) for INLA (black, open squares) and MCMC (gray, closed squares). The number of trials is in (A) 1, (B) 2, (C) uniform between 1 and 9, and (D) distributed as in the house sparrow data set." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1385294554" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Results from the synthetic Binomial case study. True &lt;em&gt;vs.&lt;/em&gt; estimated heritability: posterior mean (solid line) and 95% credible intervals (dashed lines) for INLA (black, open squares) and MCMC (gray, closed squares). The number of trials is in (A) 1, (B) 2, (C) uniform between 1 and 9, and (D) distributed as in the house sparrow data set.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/8/1241/F2.medium.gif" width="440" height="395"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/8/1241/F2.medium.gif" width="440" height="395"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/8/1241/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/8/1241/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463092" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-46" class="first-child">Results from the synthetic Binomial case study. True <em>vs.</em> estimated heritability: posterior mean (solid line) and 95% credible intervals (dashed lines) for INLA (black, open squares) and MCMC (gray, closed squares). The number of trials is in (A) 1, (B) 2, (C) uniform between 1 and 9, and (D) distributed as in the house sparrow data set.</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">The performance of INLA improves very quickly with increasing number of trials. In our second experiment we let <em>n<sub>i</sub></em> = 2, ∀<em>i</em> = 1, …, <em>n<sub>p</sub></em>, hence we have two trials for each individual in the pedigree. In this case, the presence of replicated measures makes it possible to estimate the genetic variance more accurately. <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a> shows that the posterior means computed by INLA are very close to those computed using MCMC and close to the true value of <em>h</em><sup>2</sup>. We still see a small systematic error for small values of <em>h</em><sup>2</sup> but not such that it would be problematic in a real data scenario. Even better estimates are obtained in the third experiment where the number of trials <em>n<sub>i</sub></em> changes from individual to individual in the pedigree and is randomly sampled between 1 and 9 (see <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>).</p><p id="p-48">In the last experiment the number of trials <em>n<sub>i</sub></em> is as in the house sparrow breeding season success data set (see the section <em>Data</em>). Moreover, in the simulated data we reproduce the same missing structure as in the real data set. In this experiment the number of trials is sampled uniformly between 1 and 9, and there are 1182 individuals with data. That is, for more than 65% of the individuals in the pedigree the trait under consideration was not recorded. Results shown in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2D</a>, are similar to those for the two previous cases. The estimates seem to be rather accurate with a small systematic error for very small values of the heritability. Moreover, results from INLA agree well with those from MCMC. In this experiment we have larger CI around the posterior mean when compared to the one in <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2C</a>. This is attributable to the presence of missing data.</p><p id="p-49">To test prior sensitivity, we performed a sensitivity analysis for all three likelihoods and for both no heritability (<em>h</em><sup>2</sup> = 0), and high heritabilities. Each dataset was analyzed with five different priors for <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-87.gif"/></noscript></span></span> and, when relevant, <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-88.gif"/></noscript></span></span> ; InvGamma(<em>a</em>, <em>b</em>) with a = b = 0.0001, 0.01, 0.5, 1, 10. The sensitivity analyses are described in <a href="http://www.g3journal.org/content/suppl/2013/05/24/g3.113.006700.DC1/FileS2.pdf">File S2</a> and results are shown in <a href="http://www.g3journal.org/content/suppl/2013/05/24/g3.113.006700.DC1/FigureS3.pdf">Figure S3</a>. The general findings are that for low heritabilities the results are very prior sensitive, while for higher heritabilities only extremely informative priors (<em>a</em> = <em>b</em> = 10) change the posterior considerably.</p></div></div><div id="sec-13" class="subsection"><h3>House sparrow case studies</h3><p id="p-50">In this section we analyze the data introduced in the section <em>Data</em> using the animal models in the section <em>Animal models</em>. To perform inference we use INLA, described in the section <em>Latent Gaussian models and INLA</em>. We have three case studies; bill depth, breeding season success, and ARI. For each case study, we first do model comparison using DIC to choose which fixed effects (sex, hatch year, and hatch island) and random effect (additive genetic effect) to include in our model. For the best model we do further analysis according to the chosen model and the case study. This includes estimating parameters, heritability and mean breeding values for each cohort.</p><p id="p-51">For all models we use these priors: <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-89.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-90.gif"/></noscript></span></span> and (when needed) <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-91.gif"/></noscript></span></span>. To choose the best model, we start with the full model and remove one variable at the time in a stepwise manner. In each step all nested models are examined, but only the one with lowest DIC (<em>i.e.</em>, the best one at each step) is reported in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463099/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463099/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Model selection in house sparrow case studies</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Bill depth</h3><p id="p-53">Bill depth is a Gaussian trait, and we use the animal model described in the section <em>Animal model for Gaussian data</em>. We first looked at bill depth when first (time) measured and age of the individual at that time. The full model can be written as; <em>η<sub>i</sub></em> = <em>β</em><sub>0</sub> + <em>β</em><sub>sex(</sub><em><sub>i</sub></em><sub>)</sub> + <em>β</em><sub>year(</sub><em><sub>i</sub></em><sub>)</sub> + <em>β</em><sub>island(</sub><em><sub>i</sub></em><sub>)</sub> + <em>β</em><sub>age</sub><em>x</em><sub>age</sub>(<em>i</em>) + <em>u<sub>i</sub></em>, where <em>sex</em>, <em>year</em>, and <em>island</em> are group-level factors and <em>x</em><sub>age</sub>(<em>i</em>) is a linear covariate with same prior as for <strong><em>β</em></strong>. The results from the model selection procedure are presented in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>. We see that the full model without age as a linear effect turns out to be the best;<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-92.gif"/></noscript></span> <span class="disp-formula-label">(12)</span></span>Our further analyses for bill depth are based on this model.</p><p id="p-54">We find the marginal posterior distribution for the variances; <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-93.gif"/></noscript></span></span> has posterior mean 0.24 (SD 0.05) and 95% CI 0.15−0.35. For <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-94.gif"/></noscript></span></span> we get a posterior mean 0.47 (SD 0.05) with 95% CI 0.39−0.58. We also calculate the marginal posterior of <em>h</em><sup>2</sup> using MF2 (see <a href="http://www.g3journal.org/content/suppl/2013/05/24/g3.113.006700.DC1/FileS1.pdf">File S1</a>); mean 0.35 (SD = 0.07) with 95% CI 0.21−0.48. The posteriors for <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-95.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-96.gif"/></noscript></span></span>, and <em>h</em><sup>2</sup> are plotted in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3, A and B</a>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/8/1241/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results from house sparrow case study on bill depth. All posterior estimates are obtained using INLA. (A) Posterior distribution of  (solid line) and  (dotted line). (B) Posterior distribution of heritabililty h2. (C) Left: Mean phenotopic (standardized) bill depth for each hatch year (with 95% confidence interval). (C) Right: Posterior mean (with 95% credible interval) of the levels &#x3B2;year for the factor year. (D) Posterior mean (with 95% credible interval) for average breeding values for each hatch year." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1385294554" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Results from house sparrow case study on bill depth. All posterior estimates are obtained using INLA. (A) Posterior distribution of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-97&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (solid line) and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-98&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (dotted line). (B) Posterior distribution of heritabililty &lt;em&gt;h&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt;. (C) Left: Mean phenotopic (standardized) bill depth for each hatch year (with 95% confidence interval). (C) Right: Posterior mean (with 95% credible interval) of the levels &lt;em&gt;&#x3B2;&lt;sub&gt;year&lt;/sub&gt;&lt;/em&gt; for the factor &lt;em&gt;year&lt;/em&gt;. (D) Posterior mean (with 95% credible interval) for average breeding values for each hatch year.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/8/1241/F3.medium.gif" width="440" height="381"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/8/1241/F3.medium.gif" width="440" height="381"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/8/1241/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/8/1241/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463106" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-55" class="first-child">Results from house sparrow case study on bill depth. All posterior estimates are obtained using INLA. (A) Posterior distribution of <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/F3/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/F3/embed/mml-math-97.gif"/></noscript></span></span> (solid line) and <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/F3/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/F3/embed/mml-math-98.gif"/></noscript></span></span> (dotted line). (B) Posterior distribution of heritabililty <em>h</em><sup>2</sup>. (C) Left: Mean phenotopic (standardized) bill depth for each hatch year (with 95% confidence interval). (C) Right: Posterior mean (with 95% credible interval) of the levels <em>β<sub>year</sub></em> for the factor <em>year</em>. (D) Posterior mean (with 95% credible interval) for average breeding values for each hatch year.</p><div class="sb-div caption-clear"></div></div></div><p id="p-56">To explore interesting features and evolutionary processes that may influence our study system we investigate trends in the breeding values over years (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Sorensen <em>et al.</em> 1994</a>), by estimating linear combinations of breeding values. We find the posterior distribution of average breeding values for each hatch year <em>year</em> (<em>i.e.</em>, cohort); <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-99.gif"/></noscript></span></span>, where <em>n<sub>year</sub></em> is the number of individuals with hatch year <em>year</em>, and the sum is over all these individuals. We fit the model specified in equation 12 (which includes hatch year as a factor), and the results are given in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3C</a> (posterior of <em>β<sub>year</sub></em>) and <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3D</a> (posteriors of mean breeding values for each cohort). We also calculate the posterior of the difference in average breeding values between the first (1993) and last (2002) cohorts in the study; <em>a<sub>diff</sub></em> = <em>a</em><sub>1993</sub> − <em>a</em><sub>2002</sub>, which gave a posterior mean −0.025 (SD 0.068) and 95% CI −0.161 – 0.108. The posterior marginal of the difference is given in <a href="http://www.g3journal.org/content/suppl/2013/05/24/g3.113.006700.DC1/FigureS5.pdf">Figure S5</a>. From <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3, C and D</a> we see that almost all the differences in the phenotype seems to be explained by the year specific fixed factor <em>β<sub>year</sub></em>, and from <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3D</a> there seems to be no trends in the breeding values, and hence no microevolution is going on. This is further supported by the posterior of the difference in breeding values between 1993 and 2002 cohorts <em>a</em><sub>diff</sub>|<em>y</em>, where 0 is well within a 95% credible interval.</p><p id="p-57">We also model an individual specific random effect <em>ind<sub>i</sub></em>, to include repeated measurement <em>j</em> for individual <em>i</em>, with latent field <em>η<sub>i</sub></em> = <em>β</em><sub>0</sub> + <em>β</em><sub>sex(</sub><em><sub>i</sub></em><sub>)</sub> + <em>β</em><sub>year(</sub><em><sub>i</sub></em><sub>)</sub> + <em>β</em><sub>island(</sub><em><sub>i</sub></em><sub>)</sub> + <em>u<sub>i</sub></em> + <em>ind<sub>i</sub></em>, where <em>ind<sub>i</sub></em> is a independent random effect and <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-100.gif"/></noscript></span></span>. When including individual as a random effect, we find that the marginal posterior distribution for the variances; <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-101.gif"/></noscript></span></span> has posterior mean 0.25 (SD .05) and 95% CI 0.17−0.37. For <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-102.gif"/></noscript></span></span> we get a posterior mean 0.42 (SD 0.02) with 95% CI 0.38−0.46. For <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-103.gif"/></noscript></span></span> we get the posterior mean 0.13 (SD 0.04) and 95% CI 0.07−0.21.</p></div><div id="sec-15" class="subsection"><h3>Breeding season success</h3><p id="p-58">Breeding season success is the number of breeding seasons that is a success, <em>i.e.</em>, results in at least one recruit. These data are in nature binomial, and are analyzed using the animal model in the section <em>Animal model for Binomial data</em>. We start with the full model:<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-104.gif"/></noscript></span> <span class="disp-formula-label">(13)</span></span>where the elements are described under equation 12. Results from the model selection procedure are reported in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>. We find that the best model does not include linear additive genetic effects, and hence that the inherited part of breeding season success probably is very close to zero.</p><p id="p-59">However, if we use the full model (equation 13) to estimate <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-105.gif"/></noscript></span></span> we get posterior mean 0.13, SD 0.05, and 95% CI 0.07−0.24. Furthermore, using (4) gives posterior heritability with mean 0.04 (SD 0.01) and 95% CI 0.02−0.07. These estimates are similar to those from the synthetic dataset when heritability is equal or close to zero in the section <em>Synthetic binomial case study</em> (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>).</p></div><div id="sec-16" class="subsection"><h3>Average reproductive intensity</h3><p id="p-60">ARI for an individual is the average number of recruits it produces during its lifetime. Thus, we are modeling data on LRS in such a way that lifetime is controlled for and the likelihood of any estimate of heritability will be for the ARI. This is count data, and we analyzed this trait by using the animal model in the section <em>Animal model for (zero-inflated) Poisson data</em> with <em>E<sub>i</sub></em> = <em>n<sub>i</sub></em>, where <em>n<sub>i</sub></em> is the number of breeding seasons individual <em>i</em> was alive during the study period. Because of the large number of zeros, we suspected that we needed a model that accounts for overdispersion. Therefore, we first fitted the full model as in equation 13 with two different likelihood models; Poisson (DIC = 2421.465) and zero-inflated Poisson (DIC = 2275.140). Because zero-inflated Poisson gave lowest DIC, we proceeded with this likelihood when choosing which fixed and random effects to include in the model. Also the histogram of LRS divided by lifespan indicated a zero-inflated Poisson distribution (see <a href="http://www.g3journal.org/content/suppl/2013/05/24/g3.113.006700.DC1/FigureS6.pdf">Figure S6</a>). The model with lowest DIC is the full model (equation 13), although very close in DIC to the model without additive genetic effects (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). We proceeded with the full model in our analysis of ARI. Hence, the results suggest that annual reproductive intensity might be heritable. Accordingly, the posterior for <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-106.gif"/></noscript></span></span> is 0.11 (SD 0.03) with 95% CI 0.06−0.18. To obtain the posterior of <em>h</em><sup>2</sup> defined as in (5), we plugged in the point estimate <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-107.gif"/></noscript></span></span>. This gives a posterior mean of the heritability of 0.03 (SD 0.01), 95% CI 0.02−0.05. Posterior distributions for <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-108.gif"/></noscript></span></span> and <em>h</em><sup>2</sup> are given in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/8/1241/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results from house sparrow case study on ARI. Posterior distribution of the heritability (h2) and additive genetic variance  of a zero-inflated Poisson distributed trait, average reproductive success, in Norwegian house sparrows." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1385294554" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Results from house sparrow case study on ARI. Posterior distribution of the heritability (&lt;em&gt;h&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt;) and additive genetic variance &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-109&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; of a zero-inflated Poisson distributed trait, average reproductive success, in Norwegian house sparrows.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/8/1241/F4.medium.gif" width="440" height="370"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/3/8/1241/F4.medium.gif" width="440" height="370"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/8/1241/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/8/1241/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463080" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-61" class="first-child">Results from house sparrow case study on ARI. Posterior distribution of the heritability (<em>h</em><sup>2</sup>) and additive genetic variance <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/F4/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/F4/embed/mml-math-109.gif"/></noscript></span></span> of a zero-inflated Poisson distributed trait, average reproductive success, in Norwegian house sparrows.</p><div class="sb-div caption-clear"></div></div></div><p id="p-62">The models used in the case studies can be extended. For example when modeling breeding season success, we might want year as a specific explanatory variable. This can be done by modeling it as repeated binary trait (each breeding season). When one uses binomial and Poisson likelihoods overdispersion is often a challenge. Common solutions are to include a random effect to the latent field or to use beta-binomial and negative binomial likelihoods, respectively, instead. All these options are available in INLA and R-codes for how to implement a random effect for Gaussian, binomal and Poisson likelihoods are presented in <a href="http://www.g3journal.org/content/suppl/2013/05/24/g3.113.006700.DC1/FileS4.pdf">File S4</a>. For small values of <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-110.gif"/></noscript></span></span> we observed systematic errors and prior sensitivity in all case studies. Priors for variances are discussed in <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Gelman (2006)</a>, and this topic should be further investigated, but it is outside the scope of this work.</p></div><div id="sec-17" class="subsection"><h3>Computation time</h3><p id="p-63">To compare the computation time used by INLA and MCMC, inference for the Gaussian case study and for the synthetic Gaussian case study for a large pedigree was performed with both MCMCglmm and INLA. All computation times reported are for a dual-core 2.67-GHz laptop. We visually inspect the posteriors of <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-111.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-112.gif"/></noscript></span></span> of INLA and MCMCglmm for an increasing number of iterations ((10.000, 100.000, 200.000) for the Gaussian case study and (10.000, 100.000, 500.000) for the synthetic Gaussian case study). MCMCglmm gave the same estimates as INLA (see <a href="http://www.g3journal.org/content/suppl/2013/05/24/g3.113.006700.DC1/FigureS2.pdf">Figure S2</a> and <a href="http://www.g3journal.org/content/suppl/2013/05/24/g3.113.006700.DC1/FigureS4.pdf">Figure S4</a>). For the Gaussian case study, the computation time for INLA was 7 sec for both model formulations for Gaussian data. For 200,000 iterations MCMCglmm used 17 min to achieve about the same accuracy as INLA (the Monte Carlo error is, however, still visible).</p><p id="p-64">To demonstrate the fast inference of INLA, we created a large pedigree from the existing house sparrow pedigree by merging 28 identical pedigrees and simulated data based on this pedigree with <em>n<sub>p</sub></em> = 100,072 individuals. We simulated Gaussian data for <em>β</em><sub>0</sub> = 0 and for <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-113.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-114.gif"/></noscript></span></span> as in section <em>Synthetic Gaussian case study</em>, with data for all individuals as in the pedigree. The computation time for INLA was 7.4 min. To achieve approximately the same accuracy as INLA, 500,000 iterations in MCMCglmm were needed; this took 17.9 hr, which is 145 times longer than INLA. As shown in <a href="http://www.g3journal.org/content/suppl/2013/05/24/g3.113.006700.DC1/FigureS4.pdf">Figure S4</a>, we found that even for 500,000 iterations, the Monte Carlo error was still visible.</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-65">We compared inference obtained for animal models by using INLA and MCMC. The general conclusion is that INLA is a fast and accurate approximation method that gives us the opportunity to perform simulation studies to explore models and identifiability issues. However, INLA is less flexible than MCMC methods, and we experienced this in case study three (ARI-data, Poisson likelihood), for which we were not able to calculate the heritability as defined in (5) using INLA. Although an approximated estimate could be obtained, in the Gaussian case heritability estimates can be obtained with INLA by using a tailored reparametrization. In the synthetic case study of binary traits, we experienced that INLA performed poorly for the additive genetic variance <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-115.gif"/></noscript></span></span> (for the pedigree we have used). Hence, we recommend that one should not use INLA for a binary animal model unless a simulation study suggests that INLA gives correct results for the pedigree and missing data structure of the case in question.</p><p id="p-66">Our study of average breeding value for bill depth (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3D</a>) did not indicate any change across cohorts. The posteriors of the levels of the factor <em>year</em>; (<em>β<sub>year</sub></em>, <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3C</a>) suggests that the observed phenotypic change is influenced by changes in the environment. Note that the estimates of linear combinations we obtain here take into account dependencies and uncertainties of breeding values and parameters. Hence, they do not suffer from the same systematic errors as when using regression on BLUP estimates obtained from REML-based analyses as discussed in <a id="xref-ref-47-4" class="xref-bibr" href="#ref-47">Wilson <em>et al.</em> (2009)</a> and <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Hadfield <em>et al.</em> (2010)</a>.</p><p id="p-67">When we extend the animal model for bill depth to account for repeated measurements and use individual as a random effect, we find that <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-116.gif"/></noscript></span></span> decreases. Some of the variance can now be explained by the individual effects, and the residual error can be interpreted as a measurement error and/or individual changes in bill depth during a house sparrow's lifetime. The additive genetic variance did however remain the same.</p><p id="p-68">The evolutionary process of selection may act on multiple (genetically linked) traits simultaneously (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Lande and Arnold 1983</a>). Hence, genetic correlations may impose constraints on the evolution of a given trait. A multivariate animal model which incorporates multiple traits simultaneously to estimate the genetic correlation between traits (<em>i.e.</em>, the additive genetic variance-covariance matrix) is therefore often of interest (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Meyer 1991</a>; <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Lynch and Walsh 1998</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kruuk 2004</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Jensen <em>et al.</em> 2008</a>). This extension of the animal model is in principle possible using the INLA methodology, but is limited by the number of hyperparameters and is not yet implemented in the software.</p><p id="p-69">A growing interest in quantitative genetics is for instance the use of genome-based data, such as high-density single-nucleotide polymorphism, in combination with a pedigree to be able to do better predictions of phenotypes (<em>e.g.</em>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Goddard and Hayes 2009</a>). The model by <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Yang and Tempelman (2012)</a> uses pedigree information together with a single-nucleotide polymorphism model that takes into account the within chromosome dependency. With some adjustments it will fit the INLA framework, but this requires further research.</p><p id="p-70">Both breeding season success and annual reproductive success are traits closely related to fitness. Fitness related traits have previously been found to be largely influenced by the environment and thus have low heritability (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Merilä and Sheldon 2000</a>). In our study the breeding season success was not found to be heritable, and annual reproductive success had very low heritability. Consequently, our results coincide with other studies in natural populations (see <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Jones 1987</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Merilä and Sheldon 2000</a>), finding low heritability for fitness-related traits. Note however that the environmental variance of fitness related traits usually is large, and that this will result in a low heritability of such traits even if they have some additive genetic variance (<em>e.g.</em>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Foerster <em>et al.</em> 2007</a>). Consequently, despite their low heritability these traits may still have evolutionary potential (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hansen <em>et al.</em> 2011</a>).</p><p id="p-71">In this work we demonstrated that INLA provides a suitable methodology for performing inference for a range of animal models. We have showed that because of the fast and accurate inference of INLA, we are able to use simulation studies for large pedigrees to examine different models and demonstrated the applicability of examining the difference in DIC as a method for model selection when using simulation studies. In our case studies we considered models with additive genetic effects (breeding values <strong><em>u</em></strong>), individual effects (environmental effects <strong><em>ε</em></strong>), and several fixed effects. These case studies required animal models with Gaussian, Binomial (with logit link), Poisson, and zero-inflated Poisson (with log link) likelihoods. Animal models might have a range of likelihoods. The R-INLA software also support different zero-inflated Gaussian and Binomial likelihoods, survival models (exponential, Weibull, and Cox likelihoods), Student’s <em>t</em>, and skew-normal likelihoods (see <a href="http://www.r-inla.org">www.r-inla.org</a>). It is also straightforward to make inference with INLA for animal models extended with other additive random effects, such as maternal effects or litter effects, as well as covariates.</p><p id="p-72">The R-package AnimalINLA has been developed for performing inference using INLA for animal models with likelihoods applied in this paper. It can be downloaded at <a href="http://www.r-inla.org">www.r-inla.org</a>. This package includes functionality for calculating the inverse of the relationship matrix <strong><em>A</em></strong> from a pedigree and simulation of data from pedigree. Furthermore, there are tailored functions for finding posteriors for <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-117.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-118.gif"/></noscript></span></span>, the heritability for Gaussian, binomial and Poisson likelihoods and linear combinations such as <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1241/embed/mml-math-119.gif"/></noscript></span></span>. These functions use R-INLA with suitable default settings. The R-INLA code is also included to give a good starting point to users who wants to make modifications, <em>e.g.</em>, other likelihoods or more random effects. Through providing easy to use software which gives results fast we hope Bayesian animal models become accessible to a wider audience of biologists and animal breeders.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-73">We thank Håvard Rue for customizing the R-INLA and Håvard Rue, Gregor Gorjanc, and Andrew Finley for fruitful discussions. We also thank colleagues at the Centre for Biodiversity Dynamics for help with collecting the house sparrow data. Part of the work was performed while A.M.H. and I.S. visited the Statistical and Applied Mathematical Sciences Institute. This study was supported by Norwegian Research Council (Storforsk, Strategic University Program in Conservation Biology, and grants no. 191847 and 221956).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006700/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006700/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-74"><em>Communicating editor: D.-J. De Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-01-30"><span class="received-label">Received </span>January 30, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-05-13"><span class="accepted-label">Accepted </span>May 13, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Holand <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.1" data-doi="10.1016/j.tree.2008.10.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geange</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poulsen</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Generalized linear mixed models: a practical guide for ecology and evolution.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">127</span>–<span class="cit-lpage">135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DBolker%26rft.auinit1%253DB.%2BM.%26rft.volume%253D24%26rft.issue%253D3%26rft.spage%253D127%26rft.epage%253D135%26rft.atitle%253DGeneralized%2Blinear%2Bmixed%2Bmodels%253A%2Ba%2Bpractical%2Bguide%2Bfor%2Becology%2Band%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2008.10.008%26rft_id%253Dinfo%253Apmid%252F19185386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2008.10.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19185386&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264615200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dempster</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lerner</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1950</span> <span class="cit-article-title">Heritability of threshold characters.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">212</span>–<span class="cit-lpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDempster%26rft.auinit1%253DE.%2BR.%26rft.volume%253D35%26rft.issue%253D2%26rft.spage%253D212%26rft.epage%253D236%26rft.atitle%253DHERITABILITY%2BOF%2BTHRESHOLD%2BCHARACTERS%26rft_id%253Dinfo%253Apmid%252F17247344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIzNS8yLzIxMiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvOC8xMjQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eidsvik</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martino</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rue</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Approximate Bayesian inference in spatial generalized linear mixed models.</span> <abbr class="cit-jnl-abbrev">Scand. J. Stat.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScand.%2BJ.%2BStat.%26rft.volume%253D36%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.4" data-doi="10.1038/nature05912"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foerster</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulson</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheldon</span>  <span class="cit-name-given-names">B. C</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clutton-Brock</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Sexually antagonistic genetic variation for fitness in red deer.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">447</span>: <span class="cit-fpage">1107</span>–<span class="cit-lpage">1119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DFoerster%26rft.auinit1%253DK.%26rft.volume%253D447%26rft.issue%253D7148%26rft.spage%253D1107%26rft.epage%253D1110%26rft.atitle%253DSexually%2Bantagonistic%2Bgenetic%2Bvariation%2Bfor%2Bfitness%2Bin%2Bred%2Bdeer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05912%26rft_id%253Dinfo%253Apmid%252F17597758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05912&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17597758&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247564600036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.5" data-doi="10.1093/biostatistics/kxp053"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fong</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rue</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wakefield</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Bayesian inference for generalized linear mixed models.</span> <abbr class="cit-jnl-abbrev">Biostatistics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">397</span>–<span class="cit-lpage">412</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiostatistics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252Fkxp053%26rft_id%253Dinfo%253Apmid%252F19966070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYmlvc3RzIjtzOjU6InJlc2lkIjtzOjg6IjExLzMvMzk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy84LzEyNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.6" data-doi="10.1007/BF00289392"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foulley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Im</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Genetic evaluation of traits distributed as Poisson-binomial with reference to reproductive characters.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">870</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D73%26rft.spage%253D870%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00289392%26rft_id%253Dinfo%253Apmid%252F24241297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00289392&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24241297&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.7" data-doi="10.1214/06-BA117A"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gelman</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Prior distributions for variance parameters in hierarchical models.</span> <abbr class="cit-jnl-abbrev">Bayesian Anal.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">515</span>–<span class="cit-lpage">533</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBayesian%2BAnal.%26rft.volume%253D1%26rft.spage%253D515%26rft_id%253Dinfo%253Adoi%252F10.1214%252F06-BA117A%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/06-BA117A&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000207447000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.8" data-doi="10.2527/jas1986.631217x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Bayesian methods in animal breeding theory.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DGianola%26rft.auinit1%253DD.%26rft.volume%253D63%26rft.issue%253D1%26rft.spage%253D217%26rft.epage%253D244%26rft.atitle%253DBayesian%2BMethods%2Bin%2BAnimal%2BBreeding%2BTheory%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas1986.631217x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas1986.631217x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1986D043100027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.9" data-doi="10.1038/nrg2575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mapping genes for complex traits in domestic animals and their use in breeding programmes.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BE.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D381%26rft.epage%253D391%26rft.atitle%253DMapping%2Bgenes%2Bfor%2Bcomplex%2Btraits%2Bin%2Bdomestic%2Banimals%2Band%2Btheir%2Buse%2Bin%2Bbreeding%2Bprogrammes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2575%26rft_id%253Dinfo%253Apmid%252F19448663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2575&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19448663&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266095200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-confproc" id="cit-3.8.1241.10" data-doi="10.1098/rspb.2007.1013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hadfield</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Estimating evolutionary parameters when viability selection is operating.</span> <span class="cit-source">Proc. R. Soc. B. Biol. Sci.</span> <span class="cit-vol">275</span>: <span class="cit-fpage">723</span>–<span class="cit-lpage">734</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DHadfield%26rft.auinit1%253DJ.%2BD%26rft.volume%253D275%26rft.issue%253D1635%26rft.spage%253D723%26rft.epage%253D734%26rft.atitle%253DEstimating%2Bevolutionary%2Bparameters%2Bwhen%2Bviability%2Bselection%2Bis%2Boperating%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2007.1013%26rft_id%253Dinfo%253Apmid%252F18211873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMjoiMjc1LzE2MzUvNzIzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy84LzEyNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.11" data-doi="10.1111/j.1467-9868.2005.00503.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hadfield</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">MCMC methods for multi-response generalized linear mixed models: the MCMCglmm R package.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BStat%2BSoftw%26rft.aulast%253DFriedman%26rft.auinit1%253DJ.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D22%26rft.atitle%253DRegularization%2BPaths%2Bfor%2BGeneralized%2BLinear%2BModels%2Bvia%2BCoordinate%2BDescent.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-9868.2005.00503.x%26rft_id%253Dinfo%253Apmid%252F20808728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-9868.2005.00503.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20808728&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275203200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.12" data-doi="10.1086/648604"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hadfield</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garant</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheldon</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruuk</span>  <span class="cit-name-given-names">L. E. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The misuse of BLUP in ecology and evolution.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">116</span>–<span class="cit-lpage">125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAm%2BNat%26rft.aulast%253DHadfield%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D175%26rft.issue%253D1%26rft.spage%253D116%26rft.epage%253D125%26rft.atitle%253DThe%2Bmisuse%2Bof%2BBLUP%2Bin%2Becology%2Band%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F648604%26rft_id%253Dinfo%253Apmid%252F19922262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/648604&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19922262&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272372200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.13" data-doi="10.1007/s11692-011-9127-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pélabon</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houle</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Heritability is not evolvability.</span> <abbr class="cit-jnl-abbrev">Evol. Biol.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">258</span>–<span class="cit-lpage">277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BBiol.%26rft.volume%253D38%26rft.spage%253D258%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11692-011-9127-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11692-011-9127-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1950</span> <span class="cit-article-title">Estimation of genetic parameters.</span> <abbr class="cit-jnl-abbrev">Ann. Math. Stat.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">309</span>–<span class="cit-lpage">310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BMath.%2BStat.%26rft.volume%253D21%26rft.spage%253D309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.15" data-doi="10.1111/j.1558-5646.2008.00395.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinsland</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ringsby</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sæther</span>  <span class="cit-name-given-names">B.-E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Evolutionary dynamics of a sexual ornament in the house sparrow (<em>Passer domesticus</em>): the role of indirect selection within and between sexes.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">1275</span>–<span class="cit-lpage">1293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DJensen%26rft.auinit1%253DH.%26rft.volume%253D62%26rft.issue%253D6%26rft.spage%253D1275%26rft.epage%253D1293%26rft.atitle%253DEvolutionary%2Bdynamics%2Bof%2Ba%2Bsexual%2Bornament%2Bin%2Bthe%2Bhouse%2Bsparrow%2B%2528Passer%2Bdomesticus%2529%253A%2Bthe%2Brole%2Bof%2Bindirect%2Bselection%2Bwithin%2Band%2Bbetween%2Bsexes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2008.00395.x%26rft_id%253Dinfo%253Apmid%252F18384654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2008.00395.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18384654&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256356300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.16" data-doi="10.1016/0169-5347(87)90096-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The heritability of fitness: bad news for ‘good genes’?</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">35</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DJones%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D2%26rft.issue%253D2%26rft.spage%253D35%26rft.epage%253D38%26rft.atitle%253DThe%2Bheritability%2Bof%2Bfitness%253A%2BBad%2Bnews%2Bfor%2B%2527good%2Bgenes%2527%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0169-5347%252887%252990096-6%26rft_id%253Dinfo%253Apmid%252F21227813%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0169-5347(87)90096-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21227813&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987G120600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kruuk</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Estimating genetic parameters in natural populations using the ’animal model’.</span> <abbr class="cit-jnl-abbrev">Phil. Trans. R. Soc. B</abbr> <span class="cit-vol">359</span>: <span class="cit-fpage">874</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhil.%2BTrans.%2BR.%2BSoc.%2BB%26rft.volume%253D359%26rft.spage%253D874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.18" data-doi="doi:10.1146/annurev.ecolsys.39.110707.173542"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kruuk</span>  <span class="cit-name-given-names">L. E. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slate</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">New answers for old questions: The evolutionary quantitative genetics of wild animal populations.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">525</span>–<span class="cit-lpage">548</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BEvol.%2BSyst.%26rft.volume%253D39%26rft.spage%253D525%26rft_id%253Dinfo%253Adoi%252Fdoi%253A10.1146%252Fannurev.ecolsys.39.110707.173542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=doi:10.1146/annurev.ecolsys.39.110707.173542&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000261725500025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.19" data-doi="10.2307/2408842"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lande</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">The measurement of selection on correlated characters.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">1210</span>–<span class="cit-lpage">1226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D37%26rft.spage%253D1210%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408842&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1983RX46700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.20" data-doi="10.1023/A:1008929526011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lunn</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Best</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spiegelhalter</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">WinBUGS—a Bayesian modelling framework: concepts, structure, and extensibility.</span> <abbr class="cit-jnl-abbrev">Stat. Comput.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">325</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BComput.%26rft.volume%253D10%26rft.spage%253D325%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1008929526011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1008929526011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000089242200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-book" id="cit-3.8.1241.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source">Genetics and Analysis of Quantitative Traits</span>. <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland, MA.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martino</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akerkar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rue</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Approximate Bayesian inference for survival models.</span> <abbr class="cit-jnl-abbrev">Scand. J. Stat.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">514</span>–<span class="cit-lpage">528</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScand.%2BJ.%2BStat.%26rft.volume%253D38%26rft.spage%253D514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000294120500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matos</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez-Enciso</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Genetic analysis of discrete reproductive traits in sheep using linear and nonlinear models: II. Goodness of fit and predictive ability.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">88</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DMatos%26rft.auinit1%253DC.%2BA.%26rft.volume%253D75%26rft.issue%253D1%26rft.spage%253D88%26rft.epage%253D94%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Bdiscrete%2Breproductive%2Btraits%2Bin%2Bsheep%2Busing%2Blinear%2Band%2Bnonlinear%2Bmodels%253A%2BII.%2BGoodness%2Bof%2Bfit%2Band%2Bpredictive%2Bability%26rft_id%253Dinfo%253Apmid%252F9027552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9027552&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WG57900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.24" data-doi="10.1086/303330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Merilä</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheldon</span>  <span class="cit-name-given-names">B. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Lifetime reproductive success and heritability in nature.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAm%2BNat%26rft.aulast%253DMeril%25C3%25A4%26rft.auinit1%253DJ.%26rft.volume%253D155%26rft.issue%253D3%26rft.spage%253D301%26rft.epage%253D310%26rft.atitle%253DLifetime%2BReproductive%2BSuccess%2Band%2BHeritability%2Bin%2BNature.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F303330%26rft_id%253Dinfo%253Apmid%252F10718727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/303330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10718727&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086444100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.25" data-doi="10.1186/1297-9686-23-1-67"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Estimating variances and covariances for multivariate animal models by restricted maximum likelihood.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D23%26rft.spage%253D67%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-23-1-67%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-23-1-67&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1991FE39200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.26" data-doi="10.1111/j.1420-9101.2008.01529.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Hara</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cano</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ovaskainen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teplitsky</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alho</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Bayesian approaches in evolutionary quantitative genetics.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">949</span>–<span class="cit-lpage">957</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.stitle%253DJ%2BEvol%2BBiol%26rft.aulast%253DO%2527Hara%26rft.auinit1%253DR.%2BB.%26rft.volume%253D21%26rft.issue%253D4%26rft.spage%253D949%26rft.epage%253D957%26rft.atitle%253DBayesian%2Bapproaches%2Bin%2Bevolutionary%2Bquantitative%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1420-9101.2008.01529.x%26rft_id%253Dinfo%253Apmid%252F18373658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1420-9101.2008.01529.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18373658&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-confproc" id="cit-3.8.1241.27" data-doi="10.1098/rspb.2007.0949"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ovaskainen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cano</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merilä</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A Bayesian framework for comparative quantitative genetics.</span> <span class="cit-source">Proc. R. Soc. B Biol. Sci.</span> <span class="cit-vol">275</span>: <span class="cit-fpage">669</span>–<span class="cit-lpage">678</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DOvaskainen%26rft.auinit1%253DO.%26rft.volume%253D275%26rft.issue%253D1635%26rft.spage%253D669%26rft.epage%253D678%26rft.atitle%253DA%2BBayesian%2Bframework%2Bfor%2Bcomparative%2Bquantitative%2Bgenetics%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2007.0949%26rft_id%253Dinfo%253Apmid%252F18211881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMjoiMjc1LzE2MzUvNjY5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy84LzEyNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.28" data-doi="10.1111/j.1420-9101.2005.01007.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Postma</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Implications of the difference between true and predicted breeding values for the study of natural selection and micro-evolution.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">309</span>–<span class="cit-lpage">320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.stitle%253DJ%2BEvol%2BBiol%26rft.aulast%253DPostma%26rft.auinit1%253DE.%26rft.volume%253D19%26rft.issue%253D2%26rft.spage%253D309%26rft.epage%253D320%26rft.atitle%253DImplications%2Bof%2Bthe%2Bdifference%2Bbetween%2Btrue%2Band%2Bpredicted%2Bbreeding%2Bvalues%2Bfor%2Bthe%2Bstudy%2Bof%2Bnatural%2Bselection%2Band%2Bmicro-evolution.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1420-9101.2005.01007.x%26rft_id%253Dinfo%253Apmid%252F16599906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1420-9101.2005.01007.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16599906&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235985400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.29" data-doi="10.1111/j.1365-2656.2009.01597.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pärn</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ringsby</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sæther</span>  <span class="cit-name-given-names">B.-E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Sex-specific fitness correlates of dispersal in a house sparrow metapopulation.</span> <abbr class="cit-jnl-abbrev">J. Anim. Ecol.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">1216</span>–<span class="cit-lpage">1225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BEcol%26rft.aulast%253DParn%26rft.auinit1%253DH.%26rft.volume%253D78%26rft.issue%253D6%26rft.spage%253D1216%26rft.epage%253D1225%26rft.atitle%253DSex-specific%2Bfitness%2Bcorrelates%2Bof%2Bdispersal%2Bin%2Ba%2Bhouse%2Bsparrow%2Bmetapopulation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2656.2009.01597.x%26rft_id%253Dinfo%253Apmid%252F19674180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2656.2009.01597.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19674180&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270785600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.30" data-doi="10.2307/2529279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quaas</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Computing the diagonal elements and inverse of a large numerator relationship matrix.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">949</span>–<span class="cit-lpage">953</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D32%26rft.spage%253D949%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2529279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2529279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1976CQ86300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.31" data-doi="10.1890/0012-9658(2002)083[0561:ASDOAH]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ringsby</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sæther</span>  <span class="cit-name-given-names">B.-E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tufto</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solberg</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Asynchronous spatiotemporal demography of a house sparrow metapopulation in a correlated environment.</span> <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">561</span>–<span class="cit-lpage">569</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D83%26rft.spage%253D561%26rft_id%253Dinfo%253Adoi%252F10.1890%252F0012-9658%25282002%2529083%255B0561%253AASDOAH%255D2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/0012-9658(2002)083[0561:ASDOAH]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000173538800022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.32" data-doi="10.1214/11-BA609"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roos</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Held</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Sensitivity analysis in bayesian generalized linear mixed models for binary data.</span> <abbr class="cit-jnl-abbrev">Bayesian Anal.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">259</span>–<span class="cit-lpage">278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBayesian%2BAnal.%26rft.volume%253D6%26rft.spage%253D259%26rft_id%253Dinfo%253Adoi%252F10.1214%252F11-BA609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/11-BA609&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-book" id="cit-3.8.1241.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rue</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Held</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source">Gaussian Markov Random Fields: Theory and Applications</span>. <span class="cit-publ-name">Chapman &amp; Hall</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.34" data-doi="10.1016/j.jspi.2006.07.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rue</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martino</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Approximate Bayesian inference for hierarchical Gaussian Markov random field models.</span> <abbr class="cit-jnl-abbrev">J. Statist. Plann. Inference</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">3177</span>–<span class="cit-lpage">3192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStatist.%2BPlann.%2BInference%26rft.volume%253D137%26rft.spage%253D3177%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jspi.2006.07.016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jspi.2006.07.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.35" data-doi="10.1111/j.1467-9868.2008.00700.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rue</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martino</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chopin</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Approximate Bayesian inference for latent Gaussian models using integrated nested Laplace approximations.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc., B</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">392</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%252C%2BB%26rft.volume%253D71%26rft.spage%253D319%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-9868.2008.00700.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-9868.2008.00700.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrödle</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Held</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riebler</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danuser</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Using integrated nested Laplace approximations for the evaluation of veterinary surveillance data from Switzerland: a case-study.</span> <abbr class="cit-jnl-abbrev">J. Roy. Stat. Soc.</abbr> <abbr class="cit-jnl-abbrev">C-App.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">261</span>–<span class="cit-lpage">279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BRoy.%2BStat.%2BSoc.%26rft.volume%253D60%26rft.spage%253D261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-3.8.1241.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simm</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source">Genetic Improvement of Cattle and Sheep</span>. <span class="cit-publ-name">Farming Press</span>, <span class="cit-publ-loc">Ipswich, U.K.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-book" id="cit-3.8.1241.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Skrondal</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabe-Hesketh</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-source"><em>Generalized Latent Variable Modeling: Multilevel</em>, <em>Longitudinal</em>, <em>and Structural Equation Models</em></span>. <span class="cit-publ-name">Chapman &amp; Hall/CRC</span>, <span class="cit-publ-loc">Boca Raton, FL</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-3.8.1241.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-source"><em>Likelihood</em>, <em>Bayesian and MCMC Methods in Quantitative Genetics</em></span>. <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.40" data-doi="10.1051/gse:19940403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Bayesian analysis of genetic change due to selection using Gibbs sampling.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D26%26rft.spage%253D333%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A19940403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:19940403&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994PG93000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.41" data-doi="10.1111/1467-9868.00353"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spiegelhalter</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Best</span>  <span class="cit-name-given-names">N. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlin</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Linde</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Bayesian measures of model complexity and fit (with discussion).</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">583</span>–<span class="cit-lpage">639</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB.%26rft.volume%253D64%26rft.spage%253D583%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1467-9868.00353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1467-9868.00353&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.42" data-doi="10.1111/j.1541-0420.2009.01336.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steinsland</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Utilizing Gaussian Markov random field properties of Bayesian animal models.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">763</span>–<span class="cit-lpage">771</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D66%26rft.spage%253D763%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1541-0420.2009.01336.x%26rft_id%253Dinfo%253Apmid%252F19817739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1541-0420.2009.01336.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19817739&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.43" data-doi="10.1186/1297-9686-26-3-263"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tempelman</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Assessment of a Poisson animal model for embryo yield in a simulated multiple ovulation-embryo transfer scheme.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D26%26rft.spage%253D263%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-26-3-263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-26-3-263&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.44" data-doi="10.1093/biostatistics/kxs031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van De Wiel</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leday</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Bayesian analysis of RNA sequencing data by estimating multiple shrinkage priors.</span> <abbr class="cit-jnl-abbrev">Biostatistics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiostatistics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252Fkxs031%26rft_id%253Dinfo%253Apmid%252F22988280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYmlvc3RzIjtzOjU6InJlc2lkIjtzOjg6IjE0LzEvMTEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy84LzEyNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.45" data-doi="10.3168/jds.2008-1325"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heringstad</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Assessment of Poisson, logit, and linear models for genetic analysis of clinical mastitis in Norwegian Red cows.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">739</span>–<span class="cit-lpage">748</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVazquez%26rft.auinit1%253DA.%2BI.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D739%26rft.epage%253D748%26rft.atitle%253DAssessment%2Bof%2BPoisson%252C%2Blogit%252C%2Band%2Blinear%2Bmodels%2Bfor%2Bgenetic%2Banalysis%2Bof%2Bclinical%2Bmastitis%2Bin%2BNorwegian%2BRed%2Bcows%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1325%26rft_id%253Dinfo%253Apmid%252F19164686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1325&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19164686&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262654900034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.46" data-doi="10.1038/nrg2322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wray</span>  <span class="cit-name-given-names">N. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Heritability in the genomics era— concepts and misconception.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DKruglyak%26rft.auinit1%253DL.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D255%26rft.epage%253D266%26rft.atitle%253DHeritability%2Bin%2Bthe%2Bgenomics%2Bera--concepts%2Band%2Bmisconceptions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2322%26rft_id%253Dinfo%253Apmid%252F18319743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18319743&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254084100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.47" data-doi="10.1111/j.1365-2656.2009.01639.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Réale</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clements</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrissey</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Postma</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">An ecologist's guide to the animal model.</span> <abbr class="cit-jnl-abbrev">J. Anim. Ecol.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BEcol.%26rft.volume%253D79%26rft.spage%253D13%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2656.2009.01639.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2656.2009.01639.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1241.48" data-doi="10.1534/genetics.111.131540"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tempelman</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A Bayesian antedependence model for whole genome prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1491</span>–<span class="cit-lpage">1501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.131540%26rft_id%253Dinfo%253Apmid%252F22135352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzE0OTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzgvMTI0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/8/1241.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/8/1225" title="Genomes of Ashbya Fungi Isolated from Insects Reveal Four Mating-Type Loci, Numerous Translocations, Lack of Transposons, and Distinct Gene Duplications" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/8/1253" title="Identification of Null Alleles and Deletions from SNP Genotypes for an Intercross Between Domestic and Wild Chickens" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/8"><span>Volume 3 Issue 8, August 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/8.cover-source.jpg?itok=Dmhq6EsG" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/8/1241" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-_iSljHa3jGlAn51kFrgZazoT_QcR9yjif4eoLc6KJso" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/8/1241" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/8/1241" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/8/1241" class="active" data-icon-position="" data-hide-link-title="0">Animal Models and Integrated Nested Laplace Approximations</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459516" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-XCDzMmCk_bIr_JM5qGfvY9W3WYMScekCg7Lfj5VygZg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/8/1241.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/8/1241" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Gc_PPGmGcUkEGirWuEV-aZh9kYimnmnJJr7V0tBt-bo" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/8/1241" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/8/1241.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459516" id="node459516--2" data-pisa="ggg;3/8/1241" data-pisa-master="ggg;g3.113.006700" data-apath="/ggg/3/8/1241.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Animal Models and Integrated Nested Laplace Approximations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Anna Marie Holand</span>, <span class="highwire-citation-author" data-delta="1">Ingelin Steinsland</span>, <span class="highwire-citation-author" data-delta="2">Sara Martino</span> and <span class="highwire-citation-author" data-delta="3">Henrik Jensen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1241-1251;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006700 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459516--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anna Marie Holand </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Biodiversity Dynamics, Department of Biology, NTNU, NO-7491 Trondheim, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BMarie%2BHoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Holand%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnna%2BMarie%2BHoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anna.m.holand@ntnu.no" class="" data-icon-position="" data-hide-link-title="0">anna.m.holand@ntnu.no</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ingelin Steinsland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre for Biodiversity Dynamics, Department of Mathematical Sciences, NTNU, NO-7491 Trondheim, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ingelin%2BSteinsland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steinsland%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIngelin%2BSteinsland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sara Martino </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre for Biodiversity Dynamics, Department of Mathematical Sciences, NTNU, NO-7491 Trondheim, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sara%2BMartino%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martino%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASara%2BMartino%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Henrik Jensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Biodiversity Dynamics, Department of Biology, NTNU, NO-7491 Trondheim, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Henrik%2BJensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jensen%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHenrik%2BJensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/8/1241" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/8/1241&amp;title=Animal%20Models%20and%20Integrated%20Nested%20Laplace%20Approximations" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/8/1241&amp;title=Animal%20Models%20and%20Integrated%20Nested%20Laplace%20Approximations" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/8/1241&amp;title=Animal%20Models%20and%20Integrated%20Nested%20Laplace%20Approximations" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/8/1241&amp;text=Animal%20Models%20and%20Integrated%20Nested%20Laplace%20Approximations" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/8/1241&amp;title=Animal%20Models%20and%20Integrated%20Nested%20Laplace%20Approximations" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/8/1241&amp;t=Animal%20Models%20and%20Integrated%20Nested%20Laplace%20Approximations" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/8/1241&amp;title=Animal%20Models%20and%20Integrated%20Nested%20Laplace%20Approximations" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/8/1241&amp;title=Animal%20Models%20and%20Integrated%20Nested%20Laplace%20Approximations" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459516/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459516" id="node459516--3" data-pisa="ggg;3/8/1241" data-pisa-master="ggg;g3.113.006700" data-apath="/ggg/3/8/1241.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Animal Models and Integrated Nested Laplace Approximations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Anna Marie Holand</span>, <span class="highwire-citation-author" data-delta="1">Ingelin Steinsland</span>, <span class="highwire-citation-author" data-delta="2">Sara Martino</span> and <span class="highwire-citation-author" data-delta="3">Henrik Jensen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1241-1251;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006700 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459516--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anna Marie Holand </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Biodiversity Dynamics, Department of Biology, NTNU, NO-7491 Trondheim, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BMarie%2BHoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Holand%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnna%2BMarie%2BHoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anna.m.holand@ntnu.no" class="" data-icon-position="" data-hide-link-title="0">anna.m.holand@ntnu.no</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ingelin Steinsland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre for Biodiversity Dynamics, Department of Mathematical Sciences, NTNU, NO-7491 Trondheim, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ingelin%2BSteinsland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steinsland%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIngelin%2BSteinsland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sara Martino </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre for Biodiversity Dynamics, Department of Mathematical Sciences, NTNU, NO-7491 Trondheim, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sara%2BMartino%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martino%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASara%2BMartino%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Henrik Jensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Biodiversity Dynamics, Department of Biology, NTNU, NO-7491 Trondheim, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Henrik%2BJensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jensen%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHenrik%2BJensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1241&amp;rft.epage=1251&amp;rft.atitle=Animal%20Models%20and%20Integrated%20Nested%20Laplace%20Approximations&amp;rft.volume=3&amp;rft.issue=8&amp;rft.date=2013-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Holand%2C+Anna+Marie&amp;rft.au=Steinsland%2C+Ingelin&amp;rft.au=Martino%2C+Sara&amp;rft.au=Jensen%2C+Henrik"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459516/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459516/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459516/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459516/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459516/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459516/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459516/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459516/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459516/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459516/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459516/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459516/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/8/1241" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/8/1241.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/8/1241.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/8/1241.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/8/1241.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
